<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="carcin"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4146422/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of a melanoma susceptibility locus and somatic mutation in TET2
</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4146422/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Carcinogenesis"><meta name="citation_title" content="Identification of a melanoma susceptibility locus and somatic mutation in TET2
"><meta name="citation_authors" content="Fengju Song, Christopher I. Amos, Jeffrey E. Lee, Christine G. Lian, Shenying Fang, Hongliang Liu, Stuart MacGregor, Mark M. Iles, Matthew H. Law, Neal I. Lindeman, Grant W. Montgomery, David L. Duffy, Anne E. Cust, Mark A. Jenkins, David C. Whiteman, Richard F. Kefford, Graham G. Giles, Bruce K. Armstrong, Joanne F. Aitken, John L. Hopper, Kevin M. Brown, Nicholas G. Martin, Graham J. Mann, D.Timothy Bishop, Julia A.Newton Bishop, the GenoMEL consortium, Peter Kraft, Abrar A. Qureshi, Peter A. Kanetsky, Nicholas K. Hayward, David J. Hunter, Qingyi Wei, Jiali Han"><meta name="citation_date" content="September 2014"><meta name="citation_issue" content="9"><meta name="citation_volume" content="35"><meta name="citation_firstpage" content="2097"><meta name="citation_doi" content="10.1093/carcin/bgu140"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4146422/?report=abstract"><meta name="citation_pmid" content="24980573"><meta name="DC.Title" content="Identification of a melanoma susceptibility locus and somatic mutation in TET2
"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Fengju Song"><meta name="DC.Contributor" content="Christopher I. Amos"><meta name="DC.Contributor" content="Jeffrey E. Lee"><meta name="DC.Contributor" content="Christine G. Lian"><meta name="DC.Contributor" content="Shenying Fang"><meta name="DC.Contributor" content="Hongliang Liu"><meta name="DC.Contributor" content="Stuart MacGregor"><meta name="DC.Contributor" content="Mark M. Iles"><meta name="DC.Contributor" content="Matthew H. Law"><meta name="DC.Contributor" content="Neal I. Lindeman"><meta name="DC.Contributor" content="Grant W. Montgomery"><meta name="DC.Contributor" content="David L. Duffy"><meta name="DC.Contributor" content="Anne E. Cust"><meta name="DC.Contributor" content="Mark A. Jenkins"><meta name="DC.Contributor" content="David C. Whiteman"><meta name="DC.Contributor" content="Richard F. Kefford"><meta name="DC.Contributor" content="Graham G. Giles"><meta name="DC.Contributor" content="Bruce K. Armstrong"><meta name="DC.Contributor" content="Joanne F. Aitken"><meta name="DC.Contributor" content="John L. Hopper"><meta name="DC.Contributor" content="Kevin M. Brown"><meta name="DC.Contributor" content="Nicholas G. Martin"><meta name="DC.Contributor" content="Graham J. Mann"><meta name="DC.Contributor" content="D.Timothy Bishop"><meta name="DC.Contributor" content="Julia A.Newton Bishop"><meta name="DC.Contributor" content="the GenoMEL consortium"><meta name="DC.Contributor" content="Peter Kraft"><meta name="DC.Contributor" content="Abrar A. Qureshi"><meta name="DC.Contributor" content="Peter A. Kanetsky"><meta name="DC.Contributor" content="Nicholas K. Hayward"><meta name="DC.Contributor" content="David J. Hunter"><meta name="DC.Contributor" content="Qingyi Wei"><meta name="DC.Contributor" content="Jiali Han"><meta name="DC.Date" content="2014 Sep"><meta name="DC.Identifier" content="10.1093/carcin/bgu140"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of a melanoma susceptibility locus and somatic mutation in TET2
"><meta property="og:type" content="article"><meta property="og:description" content="Although genetic studies have reported a number of loci associated with melanoma risk, the complex genetic architecture of the disease is not yet fully understood. We sought to identify common genetic variants associated with melanoma risk in a genome-wide ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4146422/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-carcin.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4146422/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4146422/pdf/bgu140.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D00A0DD56DB0100000000067F03DD.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4146422%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Carcinogenesis%22[journal]" class="navlink">Carcinogenesis</a></li><li class="accid">PMC4146422</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-carcin.png" alt="Logo of carcin" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/carcin" target="pmc_ext" ref="reftype=publisher&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Carcinogenesis</a></span>. 2014 Sep; 35(9): 2097–2101. </span></div><div><span class="fm-vol-iss-date">Published online 2014 Jun 30. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fcarcin%2Fbgu140" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/carcin/bgu140</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4146422</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24980573">24980573</a></div></div></div></div><h1 class="content-title">Identification of a melanoma susceptibility locus and somatic mutation in <em>TET2</em>
</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Song%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681100138240" co-class="co-affbox">Fengju Song</a>,<sup>
1
</sup> <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681105902000" co-class="co-affbox">Christopher I. Amos</a>,<sup>
2
</sup> <a href="/pubmed/?term=Lee%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681100134720" co-class="co-affbox">Jeffrey E. Lee</a>,<sup>
3
</sup> <a href="/pubmed/?term=Lian%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681105445536" co-class="co-affbox">Christine G. Lian</a>,<sup>
4
</sup> <a href="/pubmed/?term=Fang%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681105443168" co-class="co-affbox">Shenying Fang</a>,<sup>
3
</sup> <a href="/pubmed/?term=Liu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681105440800" co-class="co-affbox">Hongliang Liu</a>,<sup>
5
</sup> <a href="/pubmed/?term=MacGregor%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681106674608" co-class="co-affbox">Stuart MacGregor</a>,<sup>
6
</sup> <a href="/pubmed/?term=Iles%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681106672240" co-class="co-affbox">Mark M. Iles</a>,<sup>
7
</sup> <a href="/pubmed/?term=Law%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681101078704" co-class="co-affbox">Matthew H. Law</a>,<sup>
6
</sup> <a href="/pubmed/?term=Lindeman%20NI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681101076336" co-class="co-affbox">Neal I. Lindeman</a>,<sup>
4
</sup> <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681101073968" co-class="co-affbox">Grant W. Montgomery</a>,<sup>
6
</sup> <a href="/pubmed/?term=Duffy%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681105049248" co-class="co-affbox">David L. Duffy</a>,<sup>
6
</sup> <a href="/pubmed/?term=Cust%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681105046880" co-class="co-affbox">Anne E. Cust</a>,<sup>
8
,</sup><sup>
9
</sup> <a href="/pubmed/?term=Jenkins%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681100830528" co-class="co-affbox">Mark A. Jenkins</a>,<sup>
8
</sup> <a href="/pubmed/?term=Whiteman%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681100828160" co-class="co-affbox">David C. Whiteman</a>,<sup>
6
</sup> <a href="/pubmed/?term=Kefford%20RF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681100825792" co-class="co-affbox">Richard F. Kefford</a>,<sup>
10
</sup> <a href="/pubmed/?term=Giles%20GG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681105655536" co-class="co-affbox">Graham G. Giles</a>,<sup>
11
</sup> <a href="/pubmed/?term=Armstrong%20BK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681105653168" co-class="co-affbox">Bruce K. Armstrong</a>,<sup>
9
</sup> <a href="/pubmed/?term=Aitken%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681108025280" co-class="co-affbox">Joanne F. Aitken</a>,<sup>
12
</sup> <a href="/pubmed/?term=Hopper%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681108023024" co-class="co-affbox">John L. Hopper</a>,<sup>
8
</sup> <a href="/pubmed/?term=Brown%20KM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681108020656" co-class="co-affbox">Kevin M. Brown</a>,<sup>
13
</sup> <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681107284768" co-class="co-affbox">Nicholas G. Martin</a>,<sup>
6
</sup> <a href="/pubmed/?term=Mann%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681107282400" co-class="co-affbox">Graham J. Mann</a>,<sup>
10
</sup> <a href="/pubmed/?term=Bishop%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681107280032" co-class="co-affbox">D.Timothy Bishop</a>,<sup>
7
</sup> <a href="/pubmed/?term=Bishop%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681106036640" co-class="co-affbox">Julia A.Newton Bishop</a>,<sup>
7
</sup> the GenoMEL consortium, <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681106033648" co-class="co-affbox">Peter Kraft</a>,<sup>
14
</sup> <a href="/pubmed/?term=Qureshi%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681142365504" co-class="co-affbox">Abrar A. Qureshi</a>,<sup>
15
,</sup><sup>
16
</sup> <a href="/pubmed/?term=Kanetsky%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681142362016" co-class="co-affbox">Peter A. Kanetsky</a>,<sup>
17
</sup> <a href="/pubmed/?term=Hayward%20NK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681101491648" co-class="co-affbox">Nicholas K. Hayward</a>,<sup>
18
</sup> <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681101489280" co-class="co-affbox">David J. Hunter</a>,<sup>
14
</sup> <a href="/pubmed/?term=Wei%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681101486912" co-class="co-affbox">Qingyi Wei</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>
5
</sup> and  <a href="/pubmed/?term=Han%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573" class="affpopup" co-rid="_co_idm139681100108096" co-class="co-affbox">Jiali Han</a><sup>
1
,</sup><sup>
14
,</sup><sup>
15
,</sup><sup>
16
,</sup><sup>
19
,</sup><sup>
20
,</sup><sup>
21
,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139681100138240"><h3 class="no_margin">Fengju Song</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, Key Laboratory of Cancer Prevention and Therapy, National Clinical Research Center of Cancer, Tianjin Medical University Cancer Institute and Hospital, Tianjin 300060, P.R. China,</p><div>Find articles by <a href="/pubmed/?term=Song%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Fengju Song</a></div></div><div id="_co_idm139681105902000"><h3 class="no_margin">Christopher I. Amos</h3><p><sup>2</sup>Department of Community and Family Medicine, Center for Genomic Medicine, Geisel School of Medicine, Dartmouth College, Lebanon, NH 03755, USA,</p><div>Find articles by <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Christopher I. Amos</a></div></div><div id="_co_idm139681100134720"><h3 class="no_margin">Jeffrey E. Lee</h3><p><sup>3</sup>Department of Surgical Oncology, MD Anderson Cancer Center, Houston, TX 77030, USA,</p><div>Find articles by <a href="/pubmed/?term=Lee%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Jeffrey E. Lee</a></div></div><div id="_co_idm139681105445536"><h3 class="no_margin">Christine G. Lian</h3><p><sup>4</sup>Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Lian%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Christine G. Lian</a></div></div><div id="_co_idm139681105443168"><h3 class="no_margin">Shenying Fang</h3><p><sup>3</sup>Department of Surgical Oncology, MD Anderson Cancer Center, Houston, TX 77030, USA,</p><div>Find articles by <a href="/pubmed/?term=Fang%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Shenying Fang</a></div></div><div id="_co_idm139681105440800"><h3 class="no_margin">Hongliang Liu</h3><p><sup>5</sup>Duke Cancer Institute, Duke University Medical Center, Durham, NC 27710, USA,</p><div>Find articles by <a href="/pubmed/?term=Liu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Hongliang Liu</a></div></div><div id="_co_idm139681106674608"><h3 class="no_margin">Stuart MacGregor</h3><p><sup>6</sup>Queensland Institute of Medical Research, Brisbane, Queensland 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=MacGregor%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Stuart MacGregor</a></div></div><div id="_co_idm139681106672240"><h3 class="no_margin">Mark M. Iles</h3><p><sup>7</sup>Leeds Institute of Cancer and Pathology, University of Leeds, Leeds LS9 7TF, UK,</p><div>Find articles by <a href="/pubmed/?term=Iles%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Mark M. Iles</a></div></div><div id="_co_idm139681101078704"><h3 class="no_margin">Matthew H. Law</h3><p><sup>6</sup>Queensland Institute of Medical Research, Brisbane, Queensland 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=Law%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Matthew H. Law</a></div></div><div id="_co_idm139681101076336"><h3 class="no_margin">Neal I. Lindeman</h3><p><sup>4</sup>Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Lindeman%20NI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Neal I. Lindeman</a></div></div><div id="_co_idm139681101073968"><h3 class="no_margin">Grant W. Montgomery</h3><p><sup>6</sup>Queensland Institute of Medical Research, Brisbane, Queensland 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Grant W. Montgomery</a></div></div><div id="_co_idm139681105049248"><h3 class="no_margin">David L. Duffy</h3><p><sup>6</sup>Queensland Institute of Medical Research, Brisbane, Queensland 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=Duffy%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">David L. Duffy</a></div></div><div id="_co_idm139681105046880"><h3 class="no_margin">Anne E. Cust</h3><p><sup>8</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, School of Population Health, University of Melbourne, Melbourne, Victoria 3052, Australia,</p><p><sup>9</sup>Cancer Epidemiology and Services Research, Sydney School of Public Health, University of Sydney, Sydney, New South Wales 2006, Australia,</p><div>Find articles by <a href="/pubmed/?term=Cust%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Anne E. Cust</a></div></div><div id="_co_idm139681100830528"><h3 class="no_margin">Mark A. Jenkins</h3><p><sup>8</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, School of Population Health, University of Melbourne, Melbourne, Victoria 3052, Australia,</p><div>Find articles by <a href="/pubmed/?term=Jenkins%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Mark A. Jenkins</a></div></div><div id="_co_idm139681100828160"><h3 class="no_margin">David C. Whiteman</h3><p><sup>6</sup>Queensland Institute of Medical Research, Brisbane, Queensland 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=Whiteman%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">David C. Whiteman</a></div></div><div id="_co_idm139681100825792"><h3 class="no_margin">Richard F. Kefford</h3><p><sup>10</sup>Westmead Institute of Cancer Research, University of Sydney at Westmead Millennium Institute and Melanoma Institute Australia, Westmead, New South Wales 2145, Australia,</p><div>Find articles by <a href="/pubmed/?term=Kefford%20RF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Richard F. Kefford</a></div></div><div id="_co_idm139681105655536"><h3 class="no_margin">Graham G. Giles</h3><p><sup>11</sup>Cancer Epidemiology Centre, The Cancer Council Victoria, Carlton, Victoria 3053, Australia,</p><div>Find articles by <a href="/pubmed/?term=Giles%20GG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Graham G. Giles</a></div></div><div id="_co_idm139681105653168"><h3 class="no_margin">Bruce K. Armstrong</h3><p><sup>9</sup>Cancer Epidemiology and Services Research, Sydney School of Public Health, University of Sydney, Sydney, New South Wales 2006, Australia,</p><div>Find articles by <a href="/pubmed/?term=Armstrong%20BK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Bruce K. Armstrong</a></div></div><div id="_co_idm139681108025280"><h3 class="no_margin">Joanne F. Aitken</h3><p><sup>12</sup>Viertel Centre for Research in Cancer Control, Cancer Council Queensland, Brisbane, Queensland 4004, Australia,</p><div>Find articles by <a href="/pubmed/?term=Aitken%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Joanne F. Aitken</a></div></div><div id="_co_idm139681108023024"><h3 class="no_margin">John L. Hopper</h3><p><sup>8</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, School of Population Health, University of Melbourne, Melbourne, Victoria 3052, Australia,</p><div>Find articles by <a href="/pubmed/?term=Hopper%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">John L. Hopper</a></div></div><div id="_co_idm139681108020656"><h3 class="no_margin">Kevin M. Brown</h3><p><sup>13</sup>Laboratory of Translational Genomics, Division of Cancer Epidemiology and Genetics, National Cancer Institute, Bethesda, MD 20852, USA,</p><div>Find articles by <a href="/pubmed/?term=Brown%20KM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Kevin M. Brown</a></div></div><div id="_co_idm139681107284768"><h3 class="no_margin">Nicholas G. Martin</h3><p><sup>6</sup>Queensland Institute of Medical Research, Brisbane, Queensland 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Nicholas G. Martin</a></div></div><div id="_co_idm139681107282400"><h3 class="no_margin">Graham J. Mann</h3><p><sup>10</sup>Westmead Institute of Cancer Research, University of Sydney at Westmead Millennium Institute and Melanoma Institute Australia, Westmead, New South Wales 2145, Australia,</p><div>Find articles by <a href="/pubmed/?term=Mann%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Graham J. Mann</a></div></div><div id="_co_idm139681107280032"><h3 class="no_margin">D.Timothy Bishop</h3><p><sup>7</sup>Leeds Institute of Cancer and Pathology, University of Leeds, Leeds LS9 7TF, UK,</p><div>Find articles by <a href="/pubmed/?term=Bishop%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">D.Timothy Bishop</a></div></div><div id="_co_idm139681106036640"><h3 class="no_margin">Julia A.Newton Bishop</h3><p><sup>7</sup>Leeds Institute of Cancer and Pathology, University of Leeds, Leeds LS9 7TF, UK,</p><div>Find articles by <a href="/pubmed/?term=Bishop%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Julia A.Newton Bishop</a></div></div><div id="_co_idm139681106033648"><h3 class="no_margin">Peter Kraft</h3><p><sup>14</sup>Department of Epidemiology, Harvard School of Public Health, Boston, MA 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Peter Kraft</a></div></div><div id="_co_idm139681142365504"><h3 class="no_margin">Abrar A. Qureshi</h3><p><sup>15</sup>Channing Division of Network Medicine,  and</p><p><sup>16</sup>Department of Dermatology, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Qureshi%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Abrar A. Qureshi</a></div></div><div id="_co_idm139681142362016"><h3 class="no_margin">Peter A. Kanetsky</h3><p><sup>17</sup>Department of Biostatistics and Epidemiology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA,</p><div>Find articles by <a href="/pubmed/?term=Kanetsky%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Peter A. Kanetsky</a></div></div><div id="_co_idm139681101491648"><h3 class="no_margin">Nicholas K. Hayward</h3><p><sup>18</sup>Oncogenomics Laboratory, QIMR Berghofer Medical Research Institute, Brisbane, Queensland 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=Hayward%20NK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Nicholas K. Hayward</a></div></div><div id="_co_idm139681101489280"><h3 class="no_margin">David J. Hunter</h3><p><sup>14</sup>Department of Epidemiology, Harvard School of Public Health, Boston, MA 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">David J. Hunter</a></div></div><div id="_co_idm139681101486912"><h3 class="no_margin">Qingyi Wei</h3><p><sup>5</sup>Duke Cancer Institute, Duke University Medical Center, Durham, NC 27710, USA,</p><div>Find articles by <a href="/pubmed/?term=Wei%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Qingyi Wei</a></div></div><div id="_co_idm139681100108096"><h3 class="no_margin">Jiali Han</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, Key Laboratory of Cancer Prevention and Therapy, National Clinical Research Center of Cancer, Tianjin Medical University Cancer Institute and Hospital, Tianjin 300060, P.R. China,</p><p><sup>14</sup>Department of Epidemiology, Harvard School of Public Health, Boston, MA 02115, USA,</p><p><sup>15</sup>Channing Division of Network Medicine,  and</p><p><sup>16</sup>Department of Dermatology, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA 02115, USA,</p><p><sup>19</sup>Department of Epidemiology, Fairbanks School of Public Health, Indiana University, Indianapolis, IN 46202, USA,</p><p><sup>20</sup>Simon Cancer Center, Indiana University, Indianapolis, IN 46202, USA,  and</p><p><sup>21</sup>Department of Dermatology, School of Medicine, Indiana University, Indianapolis, IN 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Han%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24980573">Jiali Han</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139681141482544_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139681141482544_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139681141482544_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139681141482544_ai" style="display:none"><div class="fm-affl" id="AF0001" lang="en"><sup>1</sup>Department of Epidemiology and Biostatistics, Key Laboratory of Cancer Prevention and Therapy, National Clinical Research Center of Cancer, Tianjin Medical University Cancer Institute and Hospital, Tianjin 300060, P.R. China,</div><div class="fm-affl" id="AF0002" lang="en"><sup>2</sup>Department of Community and Family Medicine, Center for Genomic Medicine, Geisel School of Medicine, Dartmouth College, Lebanon, NH 03755, USA,</div><div class="fm-affl" id="AF0003" lang="en"><sup>3</sup>Department of Surgical Oncology, MD Anderson Cancer Center, Houston, TX 77030, USA,</div><div class="fm-affl" id="AF0004" lang="en"><sup>4</sup>Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA,</div><div class="fm-affl" id="AF0005" lang="en"><sup>5</sup>Duke Cancer Institute, Duke University Medical Center, Durham, NC 27710, USA,</div><div class="fm-affl" id="AF0006" lang="en"><sup>6</sup>Queensland Institute of Medical Research, Brisbane, Queensland 4029, Australia,</div><div class="fm-affl" id="AF0007" lang="en"><sup>7</sup>Leeds Institute of Cancer and Pathology, University of Leeds, Leeds LS9 7TF, UK,</div><div class="fm-affl" id="AF0008" lang="en"><sup>8</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, School of Population Health, University of Melbourne, Melbourne, Victoria 3052, Australia,</div><div class="fm-affl" id="AF0009" lang="en"><sup>9</sup>Cancer Epidemiology and Services Research, Sydney School of Public Health, University of Sydney, Sydney, New South Wales 2006, Australia,</div><div class="fm-affl" id="AF0010" lang="en"><sup>10</sup>Westmead Institute of Cancer Research, University of Sydney at Westmead Millennium Institute and Melanoma Institute Australia, Westmead, New South Wales 2145, Australia,</div><div class="fm-affl" id="AF0011" lang="en"><sup>11</sup>Cancer Epidemiology Centre, The Cancer Council Victoria, Carlton, Victoria 3053, Australia,</div><div class="fm-affl" id="AF0012" lang="en"><sup>12</sup>Viertel Centre for Research in Cancer Control, Cancer Council Queensland, Brisbane, Queensland 4004, Australia,</div><div class="fm-affl" id="AF0013" lang="en"><sup>13</sup>Laboratory of Translational Genomics, Division of Cancer Epidemiology and Genetics, National Cancer Institute, Bethesda, MD 20852, USA,</div><div class="fm-affl" id="AF0014" lang="en"><sup>14</sup>Department of Epidemiology, Harvard School of Public Health, Boston, MA 02115, USA,</div><div class="fm-affl" id="AF0015" lang="en"><sup>15</sup>Channing Division of Network Medicine,  and</div><div class="fm-affl" id="AF0016" lang="en"><sup>16</sup>Department of Dermatology, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA 02115, USA,</div><div class="fm-affl" id="AF0017" lang="en"><sup>17</sup>Department of Biostatistics and Epidemiology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA,</div><div class="fm-affl" id="AF0018" lang="en"><sup>18</sup>Oncogenomics Laboratory, QIMR Berghofer Medical Research Institute, Brisbane, Queensland 4029, Australia,</div><div class="fm-affl" id="AF0019" lang="en"><sup>19</sup>Department of Epidemiology, Fairbanks School of Public Health, Indiana University, Indianapolis, IN 46202, USA,</div><div class="fm-affl" id="AF0020" lang="en"><sup>20</sup>Simon Cancer Center, Indiana University, Indianapolis, IN 46202, USA,  and</div><div class="fm-affl" id="AF0021" lang="en"><sup>21</sup>Department of Dermatology, School of Medicine, Indiana University, Indianapolis, IN 46202, USA</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="c1">*To whom correspondence should be addressed. Tel: +1 317 278 0370; Fax: +1 317 274 3443; Email: <a href="mailto:dev@null" data-email="ude.ui@nahlaij" class="oemail">ude.ui@nahlaij</a></div><div id="c2">Correspondence may also be addressed to Qingyi Wei. Tel: +1 919 660 0562; Fax: +1 919 684 0902; Email: <a href="mailto:dev@null" data-email="ude.ekud@iew.iygniq" class="oemail">ude.ekud@iew.iygniq</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139681141482544_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 Mar 13; Revised 2014 May 16; Accepted 2014 Jun 19.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139681141482544_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2014. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4146422/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC4146422/bin/supp_35_9_2097__index.html" data-ga-action="click_feat_suppl">supp_35_9_2097__index.html</a><span style="color:gray"> (991 bytes)</span></div><div class="small guid">GUID:&nbsp;487FC81C-124B-422D-BCFA-F6786D9819D8</div>
<div><a href="/pmc/articles/PMC4146422/bin/supp_bgu140_GWAS_supplementary.doc" data-ga-action="click_feat_suppl">supp_bgu140_GWAS_supplementary.doc</a><span style="color:gray"> (240K)</span></div><div class="small guid">GUID:&nbsp;BFB7EEFD-A587-4939-86A0-5DF2CE9DE5BF</div>
</div></dd></dl></div><div id="idm139681101224016" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681101224016title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Although genetic studies have reported a number of loci associated with melanoma risk, the complex genetic architecture of the disease is not yet fully understood. We sought to identify common genetic variants associated with melanoma risk in a genome-wide association study (GWAS) of 2298 cases and 6654 controls. Thirteen of 15 known loci were replicated with nominal significance. A total of 69 single-nucleotide polymorphisms (SNPs) were selected for <em>in silico</em> replication in two independent melanoma GWAS datasets (a total of 5149 cases and 12 795 controls). Seven novel loci were nominally significantly associated with melanoma risk. These seven SNPs were further genotyped in 234 melanoma cases and 238 controls. The SNP rs4698934 was nominally significantly associated with melanoma risk. The combined odds ratio per T allele = 1.18; 95% confidence interval (1.10–1.25); combined <em>P</em> = 7.70 × 10<sup>−</sup>
<sup>7</sup>. This SNP is located in the intron of the <em>TET2</em> gene on chromosome 4q24. In addition, a novel somatic mutation of <em>TET2</em> was identified by next-generation sequencing in 1 of 22 sporadic melanoma cases. <em>TET2</em> encodes a member of TET family enzymes that oxidizes 5-methylcytosine to 5-hydroxymethylcytosine (5hmC). It is a putative epigenetic biomarker of melanoma as we previously reported, with observation of reduced <em>TET2</em> transcriptional expression. This study is the first to implicate <em>TET2</em> genetic variation and mutation in melanoma.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">Introduction</h2><p id="__p3" class="p p-first-last">Melanoma is the leading cause of skin cancer-related mortality and, in advanced stages, is characterized by poor prognosis. The incidence of melanoma has steadily increased worldwide (<a href="#CIT0001" rid="CIT0001" class=" bibr popnode">1</a>,<a href="#CIT0002" rid="CIT0002" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000143">2</a>). Much of this increase has been seen in relatively young adults, and consequently, the number of life years lost per melanoma death is higher than that of most other solid tumors (<a href="#CIT0003" rid="CIT0003" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000118">3</a>). Susceptibility to melanoma is determined by both environmental risk factors, including excessive exposure to ultraviolet radiation and genetically controlled phenotypic traits such as nevus propensity, red or blonde hair, light-colored eyes, fair skin, and limited tanning ability (<a href="#CIT0004" rid="CIT0004" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000137">4–6</a>). Common low-risk predisposing alleles probably account for most of the melanoma burden (<a href="#CIT0007" rid="CIT0007" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000120">7</a>,<a href="#CIT0008" rid="CIT0008" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000116">8</a>). Many putative risk alleles involved in cellular pathways such as pigmentation, DNA repair, telomere maintenance, oxidation stress, apoptosis, cell growth, and melanocyte differentiation and migration have been implicated in melanoma susceptibility (<a href="#CIT0009" rid="CIT0009" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000134">9–11</a>). Genome-wide association studies (GWASs) have propelled this field forward by identifying many novel low- to moderate-risk loci that may cause sporadic melanoma. Within the last several years, several GWASs have reported their findings on melanoma risk (<a href="#CIT0011" rid="CIT0011" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000131">11–18</a>), and most of the loci identified in earlier GWASs were within pigmentation genes (<a href="#CIT0019" rid="CIT0019" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000139">19</a>,<a href="#CIT0020" rid="CIT0020" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000125">20</a>), such as <em>MC1R</em> and <em>TYR</em>. Recent melanoma GWASs identified new loci not related to pigmentation, including <em>ATM</em>, <em>MX2</em>, <em>CASP8</em>, <em>PARP1</em>, <em>CCND1</em>, and a locus (probably <em>SETDB1</em> and <em>ARNT</em>) at 1q21.3 (<a href="#CIT0015" rid="CIT0015" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000121">15–17</a>). However, all currently identified loci are believed to account for only a small proportion of the genetic susceptibility to melanoma (<a href="#CIT0021" rid="CIT0021" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000122">21</a>,<a href="#CIT0022" rid="CIT0022" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000123">22</a>). To identify additional melanoma risk alleles, we conducted a multistage melanoma GWAS in Caucasians. In addition, somatic gene mutational status was detected by next-generation sequencing in sporadic melanoma samples.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">Materials and methods</h2><div id="s3" class="sec sec-first"><h3 id="s3title">Harvard cohorts</h3><div id="s4" class="sec sec-first"><p></p><h4 id="s4title" class="inline">Description of study populations. </h4><p id="__p4" class="p p-first">
<em>Nurses’ Health Study.</em> The Nurses’ Health Study (NHS) was established in 1976, when 121700 female registered nurses between the ages of 30 and 55 years residing in 11 larger US states completed and returned an initial self-administered questionnaire on their medical histories and baseline health-related exposures. Biennial questionnaires with collection of exposure information on risk factors have been collected prospectively. Every 2 years, along with exposures, outcome data with appropriate follow-up of reported disease events are collected. Overall, follow-up has been high; after more than 20 years, ~90% of participants continue to complete questionnaires. From May 1989 through September 1990, we collected blood samples from 32 826 participants in the NHS. Information on melanoma development was first collected in the 1984 questionnaire.</p><p id="__p5" class="p p-last">
<em>Health Professionals Follow-up Study</em>. In 1986, 51 529 men from all 50 US states in health professions (dentists, pharmacists, optometrists, osteopath physicians, podiatrists, and veterinarians) aged 40–75 years answered a detailed mailed questionnaire, forming the basis of the study. The average follow-up rate for this cohort over 10 years is &gt;90%. On each biennial questionnaire, we obtained disease- and health-related information. Between 1993 and 1994, 18 159 study participants provided blood samples by overnight courier. Information on melanoma development was first collected in the 1986 questionnaire. Description of the study population can be found elsewhere (<a href="#CIT0023" rid="CIT0023" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000146">23</a>,<a href="#CIT0024" rid="CIT0024" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000129">24</a>).</p></div><div id="s5" class="sec"><p></p><h4 id="s5title" class="inline">Melanoma cases and controls in the discovery set. </h4><p id="__p6" class="p p-first">Eligible cases in the NHS and the Health Professionals Follow-up Study (HPFS) consisted of participants with pathologically confirmed invasive melanoma, diagnosed any time after baseline up to the 2008 follow-up cycle for both cohorts. All subjects were United States non-Hispanic Caucasians.</p><p id="__p7" class="p">We have previously conducted several GWASs on different disease outcomes (NHS: breast cancer, coronary heart disease, type 2 diabetes, kidney stone, pancreatic cancer, and glaucoma; HPFS: coronary heart disease, type 2 diabetes, kidney stone, advanced prostate cancer, and glaucoma). The study description for eight GWAS sets of the discovery set is presented in <a href="http://carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgu140/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods</a>, available at <em>Carcinogenesis</em> Online. We included only controls in each GWAS, except for the kidney stone GWAS, in which we used both cases and controls. Participants without melanoma diagnosis were the controls in the current study, and those with melanoma diagnosis were the cases. In addition, we genotyped the rest of the melanoma cases in both cohorts who were not included in these previous GWASs. A detailed description of the numbers of cases and controls included and the platform for each study are shown in <a href="/pmc/articles/PMC4146422/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>. Finally, we included 494 melanoma cases and 5628 controls.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table I.</h3><!--caption a7--><div class="caption"><p id="__p8">Melanoma cases and controls (SNPs genotyped by studies in the NHS and HPFS of the discovery phase)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681107002464" class="xtable"><table frame="vsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">NHS</th><th rowspan="1" colspan="1" align="left">Control</th><th rowspan="1" colspan="1" align="left">Case</th><th rowspan="1" colspan="1" align="left">Genotyped SNPs</th><th rowspan="1" colspan="1" align="left">Platform</th><th rowspan="1" colspan="1" align="left">HPFS</th><th rowspan="1" colspan="1" align="left">Control</th><th rowspan="1" colspan="1" align="left">Case</th><th rowspan="1" colspan="1" align="left">Genotyped SNPs</th><th rowspan="1" colspan="1" align="left">Platform</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Melanoma</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">264</td><td rowspan="1" colspan="1">620901</td><td rowspan="1" colspan="1">Illumina 610</td><td rowspan="1" colspan="1">Melanoma</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">136</td><td rowspan="1" colspan="1">620901</td><td rowspan="1" colspan="1">Illumina 610</td></tr><tr><td rowspan="1" colspan="1">Breast cancer</td><td rowspan="1" colspan="1">840</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">546646</td><td rowspan="1" colspan="1">Illumina 550</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td></tr><tr><td rowspan="1" colspan="1">Coronary heart disease</td><td rowspan="1" colspan="1">557</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">721316</td><td rowspan="1" colspan="1">Affymetrix 6.0</td><td rowspan="1" colspan="1">Coronary heart disease</td><td rowspan="1" colspan="1">636</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">724881</td><td rowspan="1" colspan="1">Affymetrix 6.0</td></tr><tr><td rowspan="1" colspan="1">Glaucoma</td><td rowspan="1" colspan="1">337</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">495161</td><td rowspan="1" colspan="1">Illumina 660</td><td rowspan="1" colspan="1">Glaucoma</td><td rowspan="1" colspan="1">197</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">495161</td><td rowspan="1" colspan="1">Illumina 660</td></tr><tr><td rowspan="1" colspan="1">Kidney stone</td><td rowspan="1" colspan="1">359</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">620901</td><td rowspan="1" colspan="1">Illumina 610</td><td rowspan="1" colspan="1">Kidney stone</td><td rowspan="1" colspan="1">410</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">620901</td><td rowspan="1" colspan="1">Illumina 610</td></tr><tr><td rowspan="1" colspan="1">Type 2 diabetes</td><td rowspan="1" colspan="1">1220</td><td rowspan="1" colspan="1">23</td><td rowspan="1" colspan="1">704409</td><td rowspan="1" colspan="1">Affymetrix 6.0</td><td rowspan="1" colspan="1">Type 2 diabetes</td><td rowspan="1" colspan="1">867</td><td rowspan="1" colspan="1">23</td><td rowspan="1" colspan="1">706040</td><td rowspan="1" colspan="1">Affymetrix 6.0</td></tr><tr><td rowspan="1" colspan="1">Pancreatic cancer</td><td rowspan="1" colspan="1">64</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">559865</td><td rowspan="1" colspan="1">Illumina 550</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td></tr><tr><td rowspan="1" colspan="1">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1">Advanced prostate cancer</td><td rowspan="1" colspan="1">141</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">573612</td><td rowspan="1" colspan="1">Illumina 610</td></tr><tr><td rowspan="1" colspan="1">Total</td><td rowspan="1" colspan="1">3377</td><td rowspan="1" colspan="1">317</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1">Total</td><td rowspan="1" colspan="1">2251</td><td rowspan="1" colspan="1">177</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td></tr></tbody></table></div><div id="largeobj_idm139681107002464" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4146422/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="s6" class="sec"><p></p><h4 id="s6title" class="inline">Laboratory assays. </h4><p id="__p9" class="p p-first-last">
<em>Genotyping in eight GWASs of the discovery set.</em> We performed genotyping in the breast cancer GWAS in NHS using the Illumina HumanHap550 array, as part of the National Cancer Institute’s Cancer Genetic Markers of Susceptibility Project. For the coronary heart disease and type 2 diabetes GWASs of the discovery set, we performed genotyping using the Affymetrix 6.0 array. For the glaucoma GWAS, we performed genotyping using the Illumina HumanHap660 array. For the kidney stone, advanced prostate cancer, and melanoma GWASs, we performed genotyping using the Illumina HumanHap610 array. The quality control procedures for eight GWAS sets of the discovery set are presented in <a href="http://carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgu140/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods</a>, available at <em>Carcinogenesis</em> Online.</p></div><div id="s7" class="sec sec-last"><p></p><h4 id="s7title" class="inline">Imputation and meta-analysis. </h4><p id="__p10" class="p p-first-last">On the basis of the genotyped single-nucleotide polymorphisms (SNPs) and haplotype information in the National Center for Biotechnology Information build 35 of phase II Hapmap CEU data, we imputed genotypes for &gt;2.5 million SNPs using the program MACH (<a href="#CIT0025" rid="CIT0025" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000138">25</a>).Only SNPs with imputation quality <em>R</em>
<sup>2</sup> &gt; 0.95 in each study were included in the final analysis. A total of 1 579 307 SNPs were included in the final meta-analysis of the NHS and HPFS (<a href="/pmc/articles/PMC4146422/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>). Betas from each study of the discovery set were combined in a meta-analysis with weights proportional to the inverse variance of the beta in each study.</p></div></div><div id="s8" class="sec"><h3 id="s8title">MD Anderson Cancer Center</h3><p id="__p11" class="p p-first-last">The study participants for the discovery analysis were from a hospital-based case–control study of melanoma, for which cases were recruited from among non-Hispanic white patients at MD Anderson between March 1998 and August 2008. Samples and data were available from 931 melanoma patients and 1026 cancer-free controls (friends of other patients reporting to clinics), who were frequency matched on age and sex, completed a comprehensive skin lifestyle questionnaire, and passed quality control filters for genotyping. This questionnaire was administered by an interviewer to 70% of patients and controls and was self-administered for the remaining 30%. An additional case series comprising 873 individuals presenting for treatment for melanoma at MD Anderson was also included, bringing the total number of melanoma patients to 1804. The study protocols were approved by the Institutional Review Board at MD Anderson, and informed consent was obtained from all participants. Tissue samples were collected as whole blood, with various DNA extraction methods (including Gentra, Qiagen, and phenol–chloroform). DNA samples for the first-stage GWAS were genotyped using the Illumina Omnil-Quad array and were called using the Bead Studio algorithm at the John Hopkins University Center for Inherited Disease Research. Detailed information about the MD Anderson melanoma GWAS can be found elsewhere (<a href="#CIT0017" rid="CIT0017" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000128">17</a>).</p></div><div id="s9" class="sec"><h3 id="s9title">Melanoma GWASs in the replication set</h3><p id="__p12" class="p p-first-last">The study description and quality control procedures for the replication sets from GenoMEL (<a href="#CIT0015" rid="CIT0015" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000144">15</a>), Australia (<a href="#CIT0016" rid="CIT0016" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000148">16</a>), and the NHS II are presented in <a href="http://carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgu140/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods</a>, available at <em>Carcinogenesis</em> Online.</p></div><div id="s10" class="sec"><h3 id="s10title">Statistical methods</h3><p id="__p13" class="p p-first-last">We regressed a binary coding for melanoma case or control (0 or 1) on each SNP (dosage file) that passed quality control filters. The five largest principal components of genetic variation were nominally significantly associated with melanoma (<em>P</em> &lt; 0.05), and we adjusted for them along with age. These principal components were calculated for all individuals on the basis of ~10,000 unlinked markers using the EIGENSTRAT software. We performed a meta-analysis to combine the replication sets and the discovery set. From the discovery stage, SNPs with <em>P</em> values &lt;1×10<sup>−4</sup> were selected for replication, and in the replication stages, <em>P</em> values &lt; .05 were considered nominally statistically significant.</p></div><div id="s11" class="sec sec-last"><h3 id="s11title">Somatic mutation detection by targeted next-generation sequencing in sporadic melanoma cases</h3><p id="__p14" class="p p-first-last">DNA was isolated from formalin-fixed melanoma samples using standard methods. In brief, samples were incubated in proteinase K overnight, followed by subsequent purification of the DNA according to manufacturer’s instructions (QIAamp DNA Mini Kit, QIAGEN, Gaithersburg, MD). Then, DNA concentration was assessed using PicoGreen dsDNA detection (Life Technologies, Carlsbad, CA). Targeted next-generation sequencing was performed using cancer genomic assay to detect mutations in 275 cancer target genes. The complete coding sequence of the target genes, plus selected introns for 30 of the genes, was captured using a custom-designed Agilent SureSelect hybrid capture kit (Agilent Technologies, Santa Clara, CA) and massively parallel sequencing was performed on an Illumina HiSeq 2500 sequencer (Illumina, San Diego, CA). Data analysis was performed with a bioinformatics pipeline that included open-sourced software (Mutect and GATK, Broad Institute, Cambridge, MA) as well as internally developed software (VisCap Cancer, Breakmer).</p></div></div><div id="s12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s12title">Results</h2><p id="__p15" class="p p-first">We conducted a GWAS to identify common genetic variants associated with melanoma risk. The GWAS discovery set was a meta-analysis of the GWAS on melanoma risk in Harvard cohorts (NHS and Health Professionals Follow-up Study) and the MD Anderson GWAS on melanoma risk, including data for 2 449 178 SNPs that passed quality control procedures on a total of 8952 individuals (2298 cases and 6654 controls; (<a href="/pmc/articles/PMC4146422/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>).</p><p id="__p16">In the discovery set of the GWAS, we further selected the 69 SNPs most significantly associated with melanoma risk (<em>P</em> &lt; 5.0 × 10<sup>−5</sup>) and not in high linkage disequilibrium (<em>R</em>
<sup>2</sup> &lt; 0.8) with each other or with published melanoma risk SNPs for <em>in silico</em> replication in two independent GWAS datasets (Australian GWAS on melanoma risk and GenoMEL GWAS on melanoma risk; 5149 cases and 12 795 controls in total). Seven SNPs were nominally significantly associated with melanoma risk in the combined replication set (<a href="http://carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgu140/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 1</a>, available at <em>Carcinogenesis</em> Online). The most significant SNP identified was on chromosome 3, rs13097028, <em>P</em> = 9.99 × 10<sup>−7</sup> in the combined set. Another SNP identified on chromosome 3 was rs1031925, <em>P</em> = 4.27 × 10<sup>−6</sup>. Two SNPs on chromosome 2 were identified, rs11901831 (<em>P</em> = 1.48 × 10<sup>−5</sup>) and rs13404035 (<em>P</em> = 4.15 × 10<sup>−6</sup>). One SNP on chromosome 4 was identified, rs4698934 (<em>P</em> = 5.21 × 10<sup>−6</sup>). One SNP on chromosome 6 was identified, rs1889497 (<em>P</em> = 2.45×10<sup>−6</sup>). One SNP on chromosome 13 was also identified, rs4773180 (<em>P</em> = 5.69 × 10<sup>−5</sup>). None of the other 62 SNPs reached nominal significance in the combined replication set (<a href="http://carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgu140/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 1</a>, available at <em>Carcinogenesis</em> Online).</p><p id="__p17">We further genotyped these seven SNPs in a nested case–control study within the NHS II of 234 melanoma cases and 238 controls. The SNP rs4698934 was replicated with a <em>P</em> value of 0.009. The OR per T allele was 1.18 (95% CI: 1.10–1.25, combined <em>P</em> = 7.70 × 10<sup>−7</sup>) (<a href="/pmc/articles/PMC4146422/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table II</span></a>). Regional association plot of SNPs at the <em>TET2</em> locus is displayed in <a href="/pmc/articles/PMC4146422/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table II.</h3><!--caption a7--><div class="caption"><p id="__p18">Association between seven SNPs and melanoma risk</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681100359216" class="xtable"><table frame="vsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="7" rowspan="1" align="left">SNP information</th><th colspan="2" rowspan="1" align="left">Discovery set (Harvard and MD Anderson)</th><th colspan="2" rowspan="1" align="left">Replication 1 (GenoMEL and Australia)</th><th colspan="2" rowspan="1" align="left">Replication 2 (NHS II)</th><th colspan="2" rowspan="1" align="left">Meta-analysis</th></tr><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">CHR</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="left">A1</th><th rowspan="1" colspan="1" align="left">A2</th><th rowspan="1" colspan="1" align="left">
<em>R</em>
<sup>2</sup>
</th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left">
<em>P</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs11901831</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">212225520</td><td rowspan="1" colspan="1">
<em>ERBB4</em>
</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">1.96</td><td rowspan="1" colspan="1">1.39 (1.20–1.61)</td><td rowspan="1" colspan="1">1.1×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">1.12 (1.01–1.25)</td><td rowspan="1" colspan="1"> 0.030</td><td rowspan="1" colspan="1">1.13 (0.74–1.74)</td><td rowspan="1" colspan="1">0.563</td><td rowspan="1" colspan="1">1.21 (1.11–1.31)</td><td rowspan="1" colspan="1">1.2×10<sup>−5</sup>
</td></tr><tr><td rowspan="1" colspan="1">rs13404035</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">215121966</td><td rowspan="1" colspan="1">
<em>SPAG16</em>
</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">1.99</td><td rowspan="1" colspan="1">0.84 (0.78–0.92)</td><td rowspan="1" colspan="1">3.7×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">0.92 (0.86–0.98)</td><td rowspan="1" colspan="1"> 0.008</td><td rowspan="1" colspan="1">1.22 (0.91–1.63)</td><td rowspan="1" colspan="1">0.182</td><td rowspan="1" colspan="1">0.90 (0.86–0.94)</td><td rowspan="1" colspan="1">2.0×10<sup>−5</sup>
</td></tr><tr><td rowspan="1" colspan="1">rs1031925</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">51379274</td><td rowspan="1" colspan="1">
<em>DOCK3</em>
</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">1.97</td><td rowspan="1" colspan="1">1.29 (1.15–1.44)</td><td rowspan="1" colspan="1">2.1×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">1.11 (1.03–1.19)</td><td rowspan="1" colspan="1"> 0.005</td><td rowspan="1" colspan="1">0.92 (0.64–1.33)</td><td rowspan="1" colspan="1">0.662</td><td rowspan="1" colspan="1">1.15 (1.08–1.22)</td><td rowspan="1" colspan="1">8.1×10<sup>−6</sup>
</td></tr><tr><td rowspan="1" colspan="1">rs13097028</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">169464942</td><td rowspan="1" colspan="1">
<em>ACTRT3</em>
</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">1.99</td><td rowspan="1" colspan="1">0.85 (0.78–0.92)</td><td rowspan="1" colspan="1">4.9×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">0.92 (0.87–0.97)</td><td rowspan="1" colspan="1"> 0.002</td><td rowspan="1" colspan="1">0.88 (0.67–1.16)</td><td rowspan="1" colspan="1">0.376</td><td rowspan="1" colspan="1">0.89 (0.85–0.93)</td><td rowspan="1" colspan="1">6.7×10<sup>−7</sup>
</td></tr><tr><td rowspan="1" colspan="1">rs4698934</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">106139387</td><td rowspan="1" colspan="1">
<em>TET2</em>
</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">1.98</td><td rowspan="1" colspan="1">1.30 (1.16–1.45)</td><td rowspan="1" colspan="1">5.2×10<sup>−6</sup>
</td><td rowspan="1" colspan="1">1.10 (1.02–1.19)</td><td rowspan="1" colspan="1"> 0.018</td><td rowspan="1" colspan="1">1.67 (1.13–2.45)</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">1.18 (1.10–1.25)</td><td rowspan="1" colspan="1">7.7×10<sup>−7</sup>
</td></tr><tr><td rowspan="1" colspan="1">rs1889497</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">65432283</td><td rowspan="1" colspan="1">
<em>EYS</em>
</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1">1.99</td><td rowspan="1" colspan="1">0.80 (0.72–0.88)</td><td rowspan="1" colspan="1">7.0×10<sup>−6</sup>
</td><td rowspan="1" colspan="1">0.91 (0.84–0.98)</td><td rowspan="1" colspan="1"> 0.011</td><td rowspan="1" colspan="1">0.82 (0.59–1.14)</td><td rowspan="1" colspan="1">0.245</td><td rowspan="1" colspan="1">0.87 (0.82–0.92)</td><td rowspan="1" colspan="1">1.6×10<sup>−6</sup>
</td></tr><tr><td rowspan="1" colspan="1">rs4773180</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">111062015</td><td rowspan="1" colspan="1">
<em>COL4A2</em>
</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1">1.96</td><td rowspan="1" colspan="1">1.32 (1.15–1.50)</td><td rowspan="1" colspan="1">3.9×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">1.10 (1.00–1.21)</td><td rowspan="1" colspan="1"> 0.044</td><td rowspan="1" colspan="1">1.31 (0.82–2.10)</td><td rowspan="1" colspan="1">0.260</td><td rowspan="1" colspan="1">1.17 (1.09–1.26)</td><td rowspan="1" colspan="1">3.2×10<sup>−5</sup>
</td></tr></tbody></table></div><div id="largeobj_idm139681100359216" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4146422/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139681100232480"><p id="__p19" class="p p-first-last">A1, allele1, test allele; A2, allele2, reference allele; CHR, chromosome; CI, confidence interval; OR, odds ratio; Position, genome build 37.3; <em>R</em>
<sup>2</sup>, imputation <strong><em>R</em></strong>
<sup><strong>2</strong></sup> in Harvard data.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4146422/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681107274752" class="figure"><a href="/pmc/articles/PMC4146422/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4146422_carcin_bgu140_f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is carcin_bgu140_f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4146422/bin/carcin_bgu140_f0001.jpg"></a></div><div id="largeobj_idm139681107274752" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4146422/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4146422/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1.</a></div><!--caption a7--><div class="caption"><p id="__p20">Regional association plot of SNPs with melanoma risk. The vertical axis represents the −log10 <em>P</em> values. Recombination rates in this region are plotted in the background. LD is represented by R2 as five levels: 0–0.2, 0.2–0.4, 0.4–0.6, 0.6–0.8, and 0.8–1.0. The significance level of the SNP rs4698934 was shown for both discovery set and discovery and replication combined. The plot was generated based on Hapmap Build 37.3</p></div></div></div><p id="__p21" class="p">We examined the results of the previously identified melanoma susceptibility loci in the discovery set, and they were comparable with previous findings (<a href="/pmc/articles/PMC4146422/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table III</span></a>). All but two (<em>CASP8</em> and <em>CCND1</em>) of the known loci were nominally replicated. For the <em>CASP8</em> locus, the direction of association was consistent with prior publications. Among the novel SNPs we indentified in this study, SNPs at <em>TET2</em> locus is the most significantly associated with melanoma susceptibility. As we reported previously, <em>TET2</em>-mediated global loss of 5-hmC in melanoma genome is an epigenetic hallmark event (<a href="#CIT0026" rid="CIT0026" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000142">26</a>). Then, we sequenced 22 melanoma samples to a depth of 110 reads on average for each melanoma sample. A c.650C&gt;T nucleotide mutation was identified in one melanoma sample at an allelic fraction of 15.5%. The variant affects Serine nucleotide 217 to Phenylalanine, resulting from the substitution to missense of amino acid p.S217F (<a href="/pmc/articles/PMC4146422/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table III.</h3><!--caption a7--><div class="caption"><p id="__p22">Effect sizes for previously published melanoma risk loci in our melanoma GWAS</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681101452112" class="xtable"><table frame="vsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Ref</th><th rowspan="2" colspan="1" align="left">Gene</th><th rowspan="2" colspan="1" align="left">CHR</th><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="left">MA</th><th colspan="2" rowspan="1" align="left">Other studies</th><th colspan="2" rowspan="1" align="left">Our study</th></tr><tr><th rowspan="1" colspan="1" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" align="left">OR</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Barrett <em>et al.</em> (15)</td><td rowspan="1" colspan="1">
<em>—</em>
</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs7412746</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">9.0×10<sup>−11</sup>
</td><td rowspan="1" colspan="1">0.89 (0.85–0.95)</td><td rowspan="1" colspan="1">1.9×10<sup>−4</sup>
</td><td rowspan="1" colspan="1">0.86 (0.79–0.93)</td></tr><tr><td rowspan="1" colspan="1">Barrett <em>et al.</em> (15)</td><td rowspan="1" colspan="1">
<em>PARP1</em>
</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs3219090</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1">9.3×10<sup>−8</sup>
</td><td rowspan="1" colspan="1">0.82 (NA)</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">0.89 (0.82–0.97)</td></tr><tr><td rowspan="1" colspan="1">Falchi. <em>et al.</em> (14)</td><td rowspan="1" colspan="1">
<em>CASP8</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs13016963</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1">8.6×10<sup>−10</sup>
</td><td rowspan="1" colspan="1">1.14 (1.09–1.19)</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">1.06 (0.98–1.15)</td></tr><tr><td rowspan="1" colspan="1">Amos <em>et al.</em> (17)</td><td rowspan="1" colspan="1">
<em>CLPTM1L/TERT</em>
</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">rs401681</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">0.73 (0.59–0.91)</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.93 (0.85–1.00)</td></tr><tr><td rowspan="1" colspan="1">Brown <em>et al.</em> (12)</td><td rowspan="1" colspan="1">
<em>MTAP/CDKN2A</em>
</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs7023329</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1">4.0×10<sup>−7</sup>
</td><td rowspan="1" colspan="1">0.85 (0.80–0.91)</td><td rowspan="1" colspan="1">2.3×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">0.85 (0.78–0.91)</td></tr><tr><td rowspan="1" colspan="1">Bishop. <em>et al.</em> (13)</td><td rowspan="1" colspan="1">
<em>MTAP</em>
</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs10757257</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1">3.4×10<sup>−8</sup>
</td><td rowspan="1" colspan="1">1.23 (1.15–1.30)</td><td rowspan="1" colspan="1">3.9×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">1.18 (1.09–1.28)</td></tr><tr><td rowspan="1" colspan="1">Nan <em>et al.</em> (10)</td><td rowspan="1" colspan="1">
<em>TYR</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs1126809*</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1">2.8×10<sup>−7</sup>
</td><td rowspan="1" colspan="1">1.21 (1.13–1.30)</td><td rowspan="1" colspan="1">5.6×10<sup>−8</sup>
</td><td rowspan="1" colspan="1">1.25 (1.15–1.35)</td></tr><tr><td rowspan="1" colspan="1">Brown <em>et al.</em> (12)</td><td rowspan="1" colspan="1">
<em>TYR</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs1393350</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1">2.4×10<sup>−14</sup>
</td><td rowspan="1" colspan="1">1.29 (1.21–1.38)</td><td rowspan="1" colspan="1">6.7×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">1.20 (1.10–1.32)</td></tr><tr><td rowspan="1" colspan="1">Falchi. <em>et al.</em> (14)</td><td rowspan="1" colspan="1">
<em>CCND1</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs1485933</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1">0.0012</td><td rowspan="1" colspan="1">1.11 (1.04–1.18)</td><td rowspan="1" colspan="1" align="center">/</td><td rowspan="1" colspan="1">/</td></tr><tr><td rowspan="1" colspan="1">Falchi. <em>et al.</em> (14)</td><td rowspan="1" colspan="1">
<em>ATM</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs1801516</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1">3.4×10<sup>−9</sup>
</td><td rowspan="1" colspan="1">0.84 (0.78–0.90)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.88 (0.78–0.99)</td></tr><tr><td rowspan="1" colspan="1">Brown <em>et al.</em> (12)</td><td rowspan="1" colspan="1">
<em>MC1R</em>
</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">rs258322</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1">2.5×10<sup>−27</sup>
</td><td rowspan="1" colspan="1">1.67 (1.52–1.83)</td><td rowspan="1" colspan="1">1.6×10<sup>−9</sup>
</td><td rowspan="1" colspan="1">1.50 (1.31–1.71)</td></tr><tr><td rowspan="1" colspan="1">Gudbjartsson <em>et al.</em> (11)</td><td rowspan="1" colspan="1">
<em>PIGU</em>
</td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">rs910873</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">9.9×10<sup>−16</sup>
</td><td rowspan="1" colspan="1">1.75 (1.53–2.01)</td><td rowspan="1" colspan="1">9.8×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">1.42 (1.19–1.69)</td></tr><tr><td rowspan="1" colspan="1">Gudbjartsson <em>et al.</em> (11)</td><td rowspan="1" colspan="1">
<em>MYH7B</em>
</td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">rs1885120</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1">9.8×10<sup>−16</sup>
</td><td rowspan="1" colspan="1">1.78 (1.54–2.04)</td><td rowspan="1" colspan="1">7.1×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">1.43 (1.20–1.69)</td></tr><tr><td rowspan="1" colspan="1">Falchi. <em>et al.</em> (14)</td><td rowspan="1" colspan="1">
<em>MX2</em>
</td><td rowspan="1" colspan="1">21</td><td rowspan="1" colspan="1">rs45430</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1">2.9×10<sup>−9</sup>
</td><td rowspan="1" colspan="1">0.88 (0.85–0.92)</td><td rowspan="1" colspan="1">0.035</td><td rowspan="1" colspan="1">0.90 (0.81–0.99)</td></tr><tr><td rowspan="1" colspan="1">Bishop. <em>et al.</em> (13)</td><td rowspan="1" colspan="1">
<em>PLA2G6</em>
</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">rs132985</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">2.6×10<sup>−7</sup>
</td><td rowspan="1" colspan="1">1.23 (1.15–1.30)</td><td rowspan="1" colspan="1">2.8×10<sup>−5</sup>
</td><td rowspan="1" colspan="1">1.18 (1.10–1.28)</td></tr></tbody></table></div><div id="largeobj_idm139681101452112" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4146422/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139681104550672"><p id="__p23" class="p p-first-last">CHR, chromosome; MA, minor allele and also test allele; NA, not available; OR, odds ratio; our study, Harvard and MD Anderson melanoma GWAS; Ref, reference. </p></div><div id="idm139681104549632"><p id="__p24" class="p p-first-last">*SNP rs1126809 was not included in our GWAS, and we used SNP rs1847134 as substitution, the LD between the two SNPs is 0.96. SNP rs1485933 was not included in our GWAS, nor any SNPs in high linkage disequilibrium with it.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4146422/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681104545488" class="figure"><a href="/pmc/articles/PMC4146422/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4146422_carcin_bgu140_f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is carcin_bgu140_f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4146422/bin/carcin_bgu140_f0002.jpg"></a></div><div id="largeobj_idm139681104545488" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4146422/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4146422/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2.</a></div><!--caption a7--><div class="caption"><p id="__p25">Somatic mutation of <em>TET2</em> detection by next-generation sequencing in sporadic melanoma. Plot of copy number variation by chromosomes, sex chromosomes are excluded from the analysis. The vertical axis is the ratio of number of reads for this specimen and a panel of normal in log base 2 scales. A value of 0 denotes no difference from normal (diploid). When the sample contains 100% tumor cells, a value of −1 equals to 1 copy loss and 0.58 is 1 copy gain.</p></div></div></div></div><div id="s13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s13title">Discussion</h2><p id="__p26" class="p p-first">We replicated most of the known loci, and we identified several novel loci predisposing to melanoma. The SNP rs13097028 on chromosome 3q26 locus was the most significant, but was not replicated in the NHS II. The SNP rs4698934 locus on chromosome 4q24 was replicated. The NHS II has a modest sample size, and the statistical power for this replication is limited. The SNP rs13097028 is not located in gene region, and genes near this locus are <em>ACTRT3, MYNN, LRRC34, LRRIQ4, LRRC31</em>, and <em>MECOM</em>.</p><p id="__p27">The SNP rs4698934 is within the intron of <em>TET2</em> gene at position of 106139387. Interestingly, the somatic mutation of exon 3 at position of 106155749 is identified in our study. <em>TET2</em> (tet oncogene family member 2, or Ten-Eleven Translocation 2) is a family member of genes with enzymatic activities and plays important roles in epigenetic DNA demethylation (<a href="#CIT0027" rid="CIT0027" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000140">27</a>). TET proteins have enzymatic activity for the conversion from 5-methylcytosine (5-mC) to 5-hydroxymethylcytosine (5-hmC), which is a key intermediate of DNA demethylation. Mutations to the <em>TET</em> gene have been reported to be the most frequently mutated gene in myelodysplastic syndrome and tightly associated with reduced overall survival in hematologic malignancies (<a href="#CIT0028" rid="CIT0028" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000135">28</a>,<a href="#CIT0029" rid="CIT0029" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000145">29</a>). TET-mediated 5-hmC loss recently also has been reported in various solid malignancies, including breast cancer (<a href="#CIT0030" rid="CIT0030" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000124">30</a>,<a href="#CIT0031" rid="CIT0031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000115">31</a>), oral squamous cell carcinoma (<a href="#CIT0032" rid="CIT0032" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000141">32</a>), gastrointestinal stromal tumor (<a href="#CIT0033" rid="CIT0033" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000136">33</a>,<a href="#CIT0034" rid="CIT0034" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000132">34</a>), hepatocellular carcinoma (<a href="#CIT0035" rid="CIT0035" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000126">35</a>), and brain tumor (<a href="#CIT0036" rid="CIT0036" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000117">36</a>). In melanoma, several groups and we reported that the <em>TET2</em>-mediated 5-hmC loss is associated with prognosis and as a putative biomarker (<a href="#CIT0026" rid="CIT0026" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000127">26</a>,<a href="#CIT0037" rid="CIT0037" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000119">37–39</a>). In addition, we reported that increasing 5-hmC levels via overexpressing <em>TET2</em> reversed the genome-wide 5-hmC loss toward a benign nevus-like pattern (<a href="#CIT0026" rid="CIT0026" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000130">26</a>). We functionally characterized the significant impact of <em>TET2</em>-mediated 5-hmC loss in melanoma progression with diminished <em>TET2</em> gene expression as a possible molecular mechanisms underlying global loss of 5-hmC previously (<a href="#CIT0026" rid="CIT0026" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000133">26</a>). The findings of germline and somatic mutations in this study provide further evidence for a possible novel mechanism of 5-hmC loss is due to inactivation of <em>TET2</em> enzymatic function due to <em>TET2</em> gene mutation.</p><p id="__p28" class="p p-last">The major limitation of this study is its modest statistical power, which prevents detection of the modest effects of some genetic variants. With the current sample size, we have 80% power to detect an effect of 1.44. Furthermore, constitutional and environmental risk factors for melanoma, such as pigmentation and sun exposure, vary among European populations. These may underlie previously noted differences between the US and Australian or GenoMEL study findings (<a href="#CIT0017" rid="CIT0017" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_397000147">17</a>). Further meta-analysis of the existing GWASs is warranted to identify additional melanoma susceptibility loci. Although the identification of low-risk alleles has stimulated much enthusiasm in the field, gaining a deeper understanding of the biological consequences and clinical utility of these findings remains challenging. Future prospective studies to elucidate whether germline <em>TET2</em> SNP correlates with somatic <em>TET2</em> mutation in melanoma patients will provide critical insight, which will require larger clinical study with multiple institutions.</p></div><div id="s14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s14title">Supplementary material</h2><p id="__p29" class="p p-first-last">
<a href="http://carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgu140/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods and Supplementary Table 1</a> can be found at <a href="http://carcin.oxfordjournals.org/">http://carcin.oxfordjournals.org/</a>.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Funding</h2><p id="__p30" class="p p-first-last">
National Institutes of Health (<span>CA122838</span>;, <span>CA87969</span>;, <span>CA49449</span>;, <span>CA176726</span>;, <span>CA67262</span>;, <span>CA167552</span>; and <span>5P50CA093459-10</span>).</p></div><div id="__sec2" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm139681104209680"><a href="/pmc/articles/PMC4146422/bin/supp_35_9_2097__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm139681104206144" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681104206144title">Acknowledgements</h2><div class="sec"><p id="__p31">We thank the participants and staff of the Nurses’ Health Study and the Health Professionals Follow-up Study for their valuable contributions as well as the following state cancer registries for their help: AL, AZ, AR, CA, CO, CT, DE, FL, GA, ID, IL, IN, IA, KY, LA, ME, MD, MA, MI, NE, NH, NJ, NY, NC, ND, OH, OK, OR, PA, RI, SC, TN, TX, VA, WA, WY. In addition, this study was approved by the Connecticut Department of Public Health (DPH) Human Investigations Committee. Certain data used in this publication were obtained from the DPH. The authors assume full responsibility for analyses and interpretation of these data.</p><p id="__p32">
<em>Conflict of Interest Statement:</em> None declared.</p></div></div><div id="idm139681104203488" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681104203488title">Glossary</h2><div class="bk-sec"><div><h4>Abbreviations:</h4><table class="default-table glossary"><tbody><tr><td>GWAS</td><td>genome-wide association study</td></tr><tr><td>HPFS</td><td>Health Professionals Follow-up Study</td></tr><tr><td>NHS</td><td>Nurses’ Health Study</td></tr><tr><td>SNP</td><td>single-nucleotide polymorphism.</td></tr></tbody></table></div></div></div><div id="idm139681104593216" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681104593216title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CIT0001">1. <span class="mixed-citation">
Lian C.G., et al. 
(2014). 
<span class="ref-title">Skin cancer</span>. In Stewart B.W, editor. 
et al. (eds) <span class="ref-journal">World Cancer Report 2014</span>. World Health Organization, Geneva, Switzerland, pp. 495–503 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=World+Cancer+Report+2014&amp;author=C.G.+Lian&amp;publication_year=(2014)&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0002">2. <span class="mixed-citation">
Jemal A., et al. 
(2011). 
<span class="ref-title">Global cancer statistics</span>. <span class="ref-journal">CA Cancer J. Clin</span>., <span class="ref-vol">61</span>, 69–90 [<a href="/pubmed/21296855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CA+Cancer+J.+Clin&amp;title=Global+cancer+statistics&amp;author=A.+Jemal&amp;volume=61&amp;publication_year=(2011)&amp;pages=69-90&amp;pmid=21296855&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0003">3. <span class="mixed-citation">
MacKie R.M., et al. 
(2009). 
<span class="ref-title">Epidemiology of invasive cutaneous melanoma</span>. <span class="ref-journal">Ann. Oncol</span>., <span class="ref-vol">20</span> (<span class="ref-iss">suppl. 6</span>, vi1–vi7 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2712590/">PMC free article</a>]</span> [<a href="/pubmed/19617292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Oncol&amp;title=Epidemiology+of+invasive+cutaneous+melanoma&amp;author=R.M.+MacKie&amp;volume=20&amp;issue=suppl.+6&amp;publication_year=(2009)&amp;pages=vi1-vi7&amp;pmid=19617292&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0004">4. <span class="mixed-citation">
Tucker M.A., et al. 
(2003). 
<span class="ref-title">Melanoma etiology: where are we?</span>
<span class="ref-journal">Oncogene</span>, <span class="ref-vol">22</span>, 3042–3052 [<a href="/pubmed/12789279" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Melanoma+etiology:+where+are+we?&amp;author=M.A.+Tucker&amp;volume=22&amp;publication_year=(2003)&amp;pages=3042-3052&amp;pmid=12789279&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0005">5. <span class="mixed-citation">
Gandini S., et al. 
(2005). 
<span class="ref-title">Meta-analysis of risk factors for cutaneous melanoma. III. Family history, actinic damage and phenotypic factors</span>. <span class="ref-journal">Eur. J. Cancer</span>, <span class="ref-vol">41</span>, 2040–2059 [<a href="/pubmed/16125929" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Cancer&amp;title=Meta-analysis+of+risk+factors+for+cutaneous+melanoma.+III.+Family+history,+actinic+damage+and+phenotypic+factors&amp;author=S.+Gandini&amp;volume=41&amp;publication_year=(2005)&amp;pages=2040-2059&amp;pmid=16125929&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0006">6. <span class="mixed-citation">
Gandini S., et al. 
(2005). 
<span class="ref-title">Meta-analysis of risk factors for cutaneous melanoma. I. Common and atypical naevi</span>. <span class="ref-journal">Eur. J. Cancer</span>, <span class="ref-vol">41</span>, 28–44 [<a href="/pubmed/15617989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Cancer&amp;title=Meta-analysis+of+risk+factors+for+cutaneous+melanoma.+I.+Common+and+atypical+naevi&amp;author=S.+Gandini&amp;volume=41&amp;publication_year=(2005)&amp;pages=28-44&amp;pmid=15617989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0007">7. <span class="mixed-citation">
Meyle K.D., et al. 
(2009). 
<span class="ref-title">Genetic risk factors for melanoma</span>. <span class="ref-journal">Hum. Genet</span>., <span class="ref-vol">126</span>, 499–510 [<a href="/pubmed/19585149" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Genetic+risk+factors+for+melanoma&amp;author=K.D.+Meyle&amp;volume=126&amp;publication_year=(2009)&amp;pages=499-510&amp;pmid=19585149&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0008">8. <span class="mixed-citation">
Lin J., et al. 
(2008). 
<span class="ref-title">Genetics of melanoma predisposition</span>. <span class="ref-journal">Br. J. Dermatol</span>., <span class="ref-vol">159</span>, 286–291 [<a href="/pubmed/18547303" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Dermatol&amp;title=Genetics+of+melanoma+predisposition&amp;author=J.+Lin&amp;volume=159&amp;publication_year=(2008)&amp;pages=286-291&amp;pmid=18547303&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0009">9. <span class="mixed-citation">
Fargnoli M.C., et al. 
(2006). 
<span class="ref-title">High- and low-penetrance cutaneous melanoma susceptibility genes</span>. <span class="ref-journal">Expert Rev. Anticancer Ther</span>., <span class="ref-vol">6</span>, 657–670 [<a href="/pubmed/16759158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Rev.+Anticancer+Ther&amp;title=High-+and+low-penetrance+cutaneous+melanoma+susceptibility+genes&amp;author=M.C.+Fargnoli&amp;volume=6&amp;publication_year=(2006)&amp;pages=657-670&amp;pmid=16759158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0010">10. <span class="mixed-citation">
Nan H., et al. 
(2011). 
<span class="ref-title">Genetic variants in telomere-maintaining genes and skin cancer risk</span>. <span class="ref-journal">Hum. Genet</span>., <span class="ref-vol">129</span>, 247–253 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3443196/">PMC free article</a>]</span> [<a href="/pubmed/21116649" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Genetic+variants+in+telomere-maintaining+genes+and+skin+cancer+risk&amp;author=H.+Nan&amp;volume=129&amp;publication_year=(2011)&amp;pages=247-253&amp;pmid=21116649&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0011">11. <span class="mixed-citation">
Gudbjartsson D.F., et al. 
(2008). 
<span class="ref-title">ASIP and TYR pigmentation variants associate with cutaneous melanoma and basal cell carcinoma</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">40</span>, 886–891 [<a href="/pubmed/18488027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=ASIP+and+TYR+pigmentation+variants+associate+with+cutaneous+melanoma+and+basal+cell+carcinoma&amp;author=D.F.+Gudbjartsson&amp;volume=40&amp;publication_year=(2008)&amp;pages=886-891&amp;pmid=18488027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0012">12. <span class="mixed-citation">
Brown K.M., et al. 
(2008). 
<span class="ref-title">Common sequence variants on 20q11.22 confer melanoma susceptibility</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">40</span>, 838–840 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2755512/">PMC free article</a>]</span> [<a href="/pubmed/18488026" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+sequence+variants+on+20q11.22+confer+melanoma+susceptibility&amp;author=K.M.+Brown&amp;volume=40&amp;publication_year=(2008)&amp;pages=838-840&amp;pmid=18488026&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0013">13. <span class="mixed-citation">
Bishop D.T., et al. 
(2009). 
<span class="ref-title">Genome-wide association study identifies three loci associated with melanoma risk</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">41</span>, 920–925 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2741419/">PMC free article</a>]</span> [<a href="/pubmed/19578364" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+identifies+three+loci+associated+with+melanoma+risk&amp;author=D.T.+Bishop&amp;volume=41&amp;publication_year=(2009)&amp;pages=920-925&amp;pmid=19578364&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0014">14. <span class="mixed-citation">
Falchi M., et al. 
(2009). 
<span class="ref-title">Genome-wide association study identifies variants at 9p21 and 22q13 associated with development of cutaneous nevi</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">41</span>, 915–919 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080738/">PMC free article</a>]</span> [<a href="/pubmed/19578365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+identifies+variants+at+9p21+and+22q13+associated+with+development+of+cutaneous+nevi&amp;author=M.+Falchi&amp;volume=41&amp;publication_year=(2009)&amp;pages=915-919&amp;pmid=19578365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0015">15. <span class="mixed-citation">
Barrett J.H., et al.  ; GenoMEL Consortium. (2011). 
<span class="ref-title">Genome-wide association study identifies three new melanoma susceptibility loci</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">43</span>, 1108–1113 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3251256/">PMC free article</a>]</span> [<a href="/pubmed/21983787" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+identifies+three+new+melanoma+susceptibility+loci&amp;author=J.H.+Barrett&amp;volume=43&amp;publication_year=(2011)&amp;pages=1108-1113&amp;pmid=21983787&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0016">16. <span class="mixed-citation">
Macgregor S., et al. 
(2011). 
<span class="ref-title">Genome-wide association study identifies a new melanoma susceptibility locus at 1q21.3</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">43</span>, 1114–1118 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3227560/">PMC free article</a>]</span> [<a href="/pubmed/21983785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+identifies+a+new+melanoma+susceptibility+locus+at+1q21.3&amp;author=S.+Macgregor&amp;volume=43&amp;publication_year=(2011)&amp;pages=1114-1118&amp;pmid=21983785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0017">17. <span class="mixed-citation">
Amos C.I., et al.  ; GenoMEL Investigators; Q-Mega Investigators; AMFS Investigators. (2011). 
<span class="ref-title">Genome-wide association study identifies novel loci predisposing to cutaneous melanoma</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">20</span>, 5012–5023 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3298855/">PMC free article</a>]</span> [<a href="/pubmed/21926416" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Genome-wide+association+study+identifies+novel+loci+predisposing+to+cutaneous+melanoma&amp;author=C.I.+Amos&amp;volume=20&amp;publication_year=(2011)&amp;pages=5012-5023&amp;pmid=21926416&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0018">18. <span class="mixed-citation">
Rafnar T., et al. 
(2009). 
<span class="ref-title">Sequence variants at the TERT-CLPTM1L locus associate with many cancer types</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">41</span>, 221–227 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4525478/">PMC free article</a>]</span> [<a href="/pubmed/19151717" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Sequence+variants+at+the+TERT-CLPTM1L+locus+associate+with+many+cancer+types&amp;author=T.+Rafnar&amp;volume=41&amp;publication_year=(2009)&amp;pages=221-227&amp;pmid=19151717&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0019">19. <span class="mixed-citation">
Udayakumar D., et al. 
(2009). 
<span class="ref-title">Moderate- to low-risk variant alleles of cutaneous malignancies and nevi: lessons from genome-wide association studies</span>. <span class="ref-journal">Genome Med</span>., <span class="ref-vol">1</span>, 95. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2784308/">PMC free article</a>]</span> [<a href="/pubmed/19863770" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Med&amp;title=Moderate-+to+low-risk+variant+alleles+of+cutaneous+malignancies+and+nevi:+lessons+from+genome-wide+association+studies&amp;author=D.+Udayakumar&amp;volume=1&amp;publication_year=(2009)&amp;pages=95&amp;pmid=19863770&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0020">20. <span class="mixed-citation">
Saint-Martin C., et al.  ; French Group of Familial Myeloproliferative Disorders. (2009). 
<span class="ref-title">Analysis of the ten-eleven translocation 2 (TET2) gene in familial myeloproliferative neoplasms</span>. <span class="ref-journal">Blood</span>, <span class="ref-vol">114</span>, 1628–1632 [<a href="/pubmed/19564637" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Analysis+of+the+ten-eleven+translocation+2+(TET2)+gene+in+familial+myeloproliferative+neoplasms&amp;author=C.+Saint-Martin&amp;volume=114&amp;publication_year=(2009)&amp;pages=1628-1632&amp;pmid=19564637&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0021">21. <span class="mixed-citation">
Stacey S.N., et al. 
(2008). 
<span class="ref-title">Common variants on 1p36 and 1q42 are associated with cutaneous basal cell carcinoma but not with melanoma or pigmentation traits</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">40</span>, 1313–1318 [<a href="/pubmed/18849993" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+variants+on+1p36+and+1q42+are+associated+with+cutaneous+basal+cell+carcinoma+but+not+with+melanoma+or+pigmentation+traits&amp;author=S.N.+Stacey&amp;volume=40&amp;publication_year=(2008)&amp;pages=1313-1318&amp;pmid=18849993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0022">22. <span class="mixed-citation">
Duffy D.L., et al. 
(2010). 
<span class="ref-title">Multiple pigmentation gene polymorphisms account for a substantial proportion of risk of cutaneous malignant melanoma</span>. <span class="ref-journal">J. Invest. Dermatol</span>., <span class="ref-vol">130</span>, 520–528 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3672059/">PMC free article</a>]</span> [<a href="/pubmed/19710684" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Invest.+Dermatol&amp;title=Multiple+pigmentation+gene+polymorphisms+account+for+a+substantial+proportion+of+risk+of+cutaneous+malignant+melanoma&amp;author=D.L.+Duffy&amp;volume=130&amp;publication_year=(2010)&amp;pages=520-528&amp;pmid=19710684&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0023">23. <span class="mixed-citation">
Nan H., et al. 
(2011). 
<span class="ref-title">Genome-wide association study identifies novel alleles associated with risk of cutaneous basal cell carcinoma and squamous cell carcinoma</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">20</span>, 3718–3724 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3159556/">PMC free article</a>]</span> [<a href="/pubmed/21700618" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Genome-wide+association+study+identifies+novel+alleles+associated+with+risk+of+cutaneous+basal+cell+carcinoma+and+squamous+cell+carcinoma&amp;author=H.+Nan&amp;volume=20&amp;publication_year=(2011)&amp;pages=3718-3724&amp;pmid=21700618&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0024">24. <span class="mixed-citation">
Nan H., et al. 
(2011). 
<span class="ref-title">Genome-wide association study identifies nidogen 1 (NID1) as a susceptibility locus to cutaneous nevi and melanoma risk</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">20</span>, 2673–2679 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3110001/">PMC free article</a>]</span> [<a href="/pubmed/21478494" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Genome-wide+association+study+identifies+nidogen+1+(NID1)+as+a+susceptibility+locus+to+cutaneous+nevi+and+melanoma+risk&amp;author=H.+Nan&amp;volume=20&amp;publication_year=(2011)&amp;pages=2673-2679&amp;pmid=21478494&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0025">25. <span class="mixed-citation">
Biernacka J.M., et al. 
(2009). 
<span class="ref-title">Assessment of genotype imputation methods</span>. <span class="ref-journal">BMC Proc</span>., <span class="ref-vol">3</span> (<span class="ref-iss">suppl. 7</span>, S5. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2795949/">PMC free article</a>]</span> [<a href="/pubmed/20018042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Proc&amp;title=Assessment+of+genotype+imputation+methods&amp;author=J.M.+Biernacka&amp;volume=3&amp;issue=suppl.+7&amp;publication_year=(2009)&amp;pages=S5&amp;pmid=20018042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0026">26. <span class="mixed-citation">
Lian C.G., et al. 
(2012). 
<span class="ref-title">Loss of 5-hydroxymethylcytosine is an epigenetic hallmark of melanoma</span>. <span class="ref-journal">Cell</span>, <span class="ref-vol">150</span>, 1135–1146 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3770275/">PMC free article</a>]</span> [<a href="/pubmed/22980977" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Loss+of+5-hydroxymethylcytosine+is+an+epigenetic+hallmark+of+melanoma&amp;author=C.G.+Lian&amp;volume=150&amp;publication_year=(2012)&amp;pages=1135-1146&amp;pmid=22980977&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0027">27. <span class="mixed-citation">
Ito S., et al. 
(2010). 
<span class="ref-title">Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">466</span>, 1129–1133 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3491567/">PMC free article</a>]</span> [<a href="/pubmed/20639862" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Role+of+Tet+proteins+in+5mC+to+5hmC+conversion,+ES-cell+self-renewal+and+inner+cell+mass+specification&amp;author=S.+Ito&amp;volume=466&amp;publication_year=(2010)&amp;pages=1129-1133&amp;pmid=20639862&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0028">28. <span class="mixed-citation">
Saint-Martin C., et al.  ; French Group of Familial Myeloproliferative Disorders. (2009). 
<span class="ref-title">Analysis of the ten-eleven translocation 2 (TET2) gene in familial myeloproliferative neoplasms</span>. <span class="ref-journal">Blood</span>, <span class="ref-vol">114</span>, 1628–1632 [<a href="/pubmed/19564637" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Analysis+of+the+ten-eleven+translocation+2+(TET2)+gene+in+familial+myeloproliferative+neoplasms&amp;author=C.+Saint-Martin&amp;volume=114&amp;publication_year=(2009)&amp;pages=1628-1632&amp;pmid=19564637&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0029">29. <span class="mixed-citation">
Abdel-Wahab O., et al. 
(2009). 
<span class="ref-title">Genetic characterization of TET1, TET2, and TET3 alterations in myeloid malignancies</span>. <span class="ref-journal">Blood</span>, <span class="ref-vol">114</span>, 144–147 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2710942/">PMC free article</a>]</span> [<a href="/pubmed/19420352" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Genetic+characterization+of+TET1,+TET2,+and+TET3+alterations+in+myeloid+malignancies&amp;author=O.+Abdel-Wahab&amp;volume=114&amp;publication_year=(2009)&amp;pages=144-147&amp;pmid=19420352&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0030">30. <span class="mixed-citation">
Haffner M.C., et al. 
(2011). 
<span class="ref-title">Global 5-hydroxymethylcytosine content is significantly reduced in tissue stem/progenitor cell compartments and in human cancers</span>. <span class="ref-journal">Oncotarget</span>, <span class="ref-vol">2</span>, 627–637 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3248214/">PMC free article</a>]</span> [<a href="/pubmed/21896958" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Global+5-hydroxymethylcytosine+content+is+significantly+reduced+in+tissue+stem/progenitor+cell+compartments+and+in+human+cancers&amp;author=M.C.+Haffner&amp;volume=2&amp;publication_year=(2011)&amp;pages=627-637&amp;pmid=21896958&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0031">31. <span class="mixed-citation">
Sun M., et al. 
(2013). 
<span class="ref-title">HMGA2/TET1/HOXA9 signaling pathway regulates breast cancer growth and metastasis</span>. <span class="ref-journal">Proc. Natl Acad. Sci. USA</span>, <span class="ref-vol">110</span>, 9920–9925 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3683728/">PMC free article</a>]</span> [<a href="/pubmed/23716660" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=HMGA2/TET1/HOXA9+signaling+pathway+regulates+breast+cancer+growth+and+metastasis&amp;author=M.+Sun&amp;volume=110&amp;publication_year=(2013)&amp;pages=9920-9925&amp;pmid=23716660&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0032">32. <span class="mixed-citation">
Jäwert F., et al. 
(2013). 
<span class="ref-title">Loss of 5-hydroxymethylcytosine and TET2 in oral squamous cell carcinoma</span>. <span class="ref-journal">Anticancer Res</span>., <span class="ref-vol">33</span>, 4325–4328 [<a href="/pubmed/24122999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anticancer+Res&amp;title=Loss+of+5-hydroxymethylcytosine+and+TET2+in+oral+squamous+cell+carcinoma&amp;author=F.+Jäwert&amp;volume=33&amp;publication_year=(2013)&amp;pages=4325-4328&amp;pmid=24122999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0033">33. <span class="mixed-citation">
Mason E.F., et al. 
(2013). 
<span class="ref-title">Succinate dehydrogenase deficiency is associated with decreased 5-hydroxymethylcytosine production in gastrointestinal stromal tumors: implications for mechanisms of tumorigenesis</span>. <span class="ref-journal">Mod. Pathol</span>., <span class="ref-vol">26</span>, 1492–1497 [<a href="/pubmed/23743927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mod.+Pathol&amp;title=Succinate+dehydrogenase+deficiency+is+associated+with+decreased+5-hydroxymethylcytosine+production+in+gastrointestinal+stromal+tumors:+implications+for+mechanisms+of+tumorigenesis&amp;author=E.F.+Mason&amp;volume=26&amp;publication_year=(2013)&amp;pages=1492-1497&amp;pmid=23743927&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0034">34. <span class="mixed-citation">
Zhang L.T., et al. 
(2013). 
<span class="ref-title">Quantification of the sixth DNA base 5-hydroxymethylcytosine in colorectal cancer tissue and C-26 cell line</span>. <span class="ref-journal">Bioanalysis</span>, <span class="ref-vol">5</span>, 839–845 [<a href="/pubmed/23534428" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioanalysis&amp;title=Quantification+of+the+sixth+DNA+base+5-hydroxymethylcytosine+in+colorectal+cancer+tissue+and+C-26+cell+line&amp;author=L.T.+Zhang&amp;volume=5&amp;publication_year=(2013)&amp;pages=839-845&amp;pmid=23534428&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0035">35. <span class="mixed-citation">
Liu C., et al. 
(2013). 
<span class="ref-title">Decrease of 5-hydroxymethylcytosine is associated with progression of hepatocellular carcinoma through downregulation of TET1</span>. <span class="ref-journal">PLoS One</span>, <span class="ref-vol">8</span>, e62828. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3650038/">PMC free article</a>]</span> [<a href="/pubmed/23671639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Decrease+of+5-hydroxymethylcytosine+is+associated+with+progression+of+hepatocellular+carcinoma+through+downregulation+of+TET1&amp;author=C.+Liu&amp;volume=8&amp;publication_year=(2013)&amp;pages=e62828&amp;pmid=23671639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0036">36. <span class="mixed-citation">
Kraus T.F., et al. 
(2012). 
<span class="ref-title">Low values of 5-hydroxymethylcytosine (5hmC), the “sixth base,” are associated with anaplasia in human brain tumors</span>. <span class="ref-journal">Int. J. Cancer</span>, <span class="ref-vol">131</span>, 1577–1590 [<a href="/pubmed/22234893" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=Low+values+of+5-hydroxymethylcytosine+(5hmC),+the+“sixth+base,”+are+associated+with+anaplasia+in+human+brain+tumors&amp;author=T.F.+Kraus&amp;volume=131&amp;publication_year=(2012)&amp;pages=1577-1590&amp;pmid=22234893&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0037">37. <span class="mixed-citation">
Gambichler T., et al. 
(2013). 
<span class="ref-title">Loss of 5-hydroxymethylcytosine and ten-eleven translocation 2 protein expression in malignant melanoma</span>. <span class="ref-journal">Melanoma Res</span>., <span class="ref-vol">23</span>, 218–220 [<a href="/pubmed/23458759" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Melanoma+Res&amp;title=Loss+of+5-hydroxymethylcytosine+and+ten-eleven+translocation+2+protein+expression+in+malignant+melanoma&amp;author=T.+Gambichler&amp;volume=23&amp;publication_year=(2013)&amp;pages=218-220&amp;pmid=23458759&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0038">38. <span class="mixed-citation">
Larson A.R., et al. 
(2014). 
<span class="ref-title">Loss of 5-hydroxymethylcytosine correlates with increasing morphologic dysplasia in melanocytic tumors</span>. <span class="ref-journal">Mod. Pathol</span>.<span class="ref-vol">, 27</span>, 936–944 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4077910/">PMC free article</a>]</span> [<a href="/pubmed/24390216" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mod.+Pathol&amp;title=Loss+of+5-hydroxymethylcytosine+correlates+with+increasing+morphologic+dysplasia+in+melanocytic+tumors&amp;author=A.R.+Larson&amp;volume=,+27&amp;publication_year=(2014)&amp;pages=936-944&amp;pmid=24390216&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0039">39. <span class="mixed-citation">
Uchiyama R., et al. 
(2014). 
<span class="ref-title">5-Hydroxymethylcytosine as a useful marker to differentiate between malignant melanomas and benign melanocytic nevi</span>. <span class="ref-journal">J. Dermatol. Sci</span>., <span class="ref-vol">73</span>, 161–163 [<a href="/pubmed/24169492" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Dermatol.+Sci&amp;title=5-Hydroxymethylcytosine+as+a+useful+marker+to+differentiate+between+malignant+melanomas+and+benign+melanocytic+nevi&amp;author=R.+Uchiyama&amp;volume=73&amp;publication_year=(2014)&amp;pages=161-163&amp;pmid=24169492&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4146422&amp;issue-id=242088&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139681101224016title">Abstract</a></li><li><a href="#s1title">Introduction</a></li><li><a href="#s2title">Materials and methods</a></li><li><a href="#s12title">Results</a></li><li><a href="#s13title">Discussion</a></li><li><a href="#s14title">Supplementary material</a></li><li><a href="#__sec1title">Funding</a></li><li><a href="#__sec2title">Supplementary Material</a></li><li><a href="#idm139681104206144title">Acknowledgements</a></li><li><a href="#idm139681104203488title">Glossary</a></li><li><a href="#idm139681104593216title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Carcinogenesis</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4146422/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4146422/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4146422/pdf/bgu140.pdf">PDF (714K)</a></li> | <li><a href="#" data-citationid="PMC4146422" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4146422%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4146422%2F&amp;text=Identification%20of%20a%20melanoma%20susceptibility%20locus%20and%20somatic%20mutation%20in%20TET2%0A" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4146422%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24980573" data-aiid="4146422" data-aid="4146422" data-iid="242088" data-domainid="724" data-domain="carcin" data-accid="PMC4146422" data-md5="f780f64e4cb6dac2e6d0abb3e8d7d58c"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:25:44-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D00A0DD56DB0100000000067F03DD&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4146422%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=carcin&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4146422/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>