<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4829354/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>GWAS in an Amerindian ancestry population reveals novel systemic lupus erythematosus risk loci and the role of European admixture</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4829354/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Arthritis &amp; rheumatology (Hoboken, N.J.)"><meta name="citation_title" content="GWAS in an Amerindian ancestry population reveals novel systemic lupus erythematosus risk loci and the role of European admixture"><meta name="citation_authors" content="Marta E. Alarcón-Riquelme, Julie T. Ziegler, Julio Molineros, Timothy D. Howard, Andrés Moreno-Estrada, Elena Sánchez-Rodríguez, Hannah C. Ainsworth, Patricia Ortiz-Tello, Mary E. Comeau, Astrid Rasmussen, Jennifer A. Kelly, Adam Adler, Eduardo Acevedo-Vázquez, Jorge Mariano Cucho, Ignacio García-De la Torre, Mario H. Cardiel, Pedro Miranda, Luis Catoggio, Marco Maradiaga-Ceceña, Patrick Gaffney, Timothy Vyse, Lindsey A. Criswell, Betty P. Tsao, Kathy L. Sivils, Sang-Cheol Bae, Judith A. James, Robert Kimberly, Ken Kaufman, John B. Harley, Jorge Esquivel-Valerio, José F. Moctezuma, Mercedes A. García, Guillermo Berbotto, Alejandra Babini, Hugo Scherbarth, Sergio Toloza, Vicente Baca, Swapan K. Nath, Carlos Aguilar Salinas, Lorena Orozco, Teresa Tusié-Luna, Raphael Zidovetzki, Bernardo A. Pons-Estel, Carl D. Langefeld, Chaim O. Jacob"><meta name="citation_date" content="April 2016"><meta name="citation_issue" content="4"><meta name="citation_volume" content="68"><meta name="citation_firstpage" content="932"><meta name="citation_doi" content="10.1002/art.39504"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4829354/?report=abstract"><meta name="citation_pmid" content="26606652"><meta name="DC.Title" content="GWAS in an Amerindian ancestry population reveals novel systemic lupus erythematosus risk loci and the role of European admixture"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Marta E. Alarcón-Riquelme"><meta name="DC.Contributor" content="Julie T. Ziegler"><meta name="DC.Contributor" content="Julio Molineros"><meta name="DC.Contributor" content="Timothy D. Howard"><meta name="DC.Contributor" content="Andrés Moreno-Estrada"><meta name="DC.Contributor" content="Elena Sánchez-Rodríguez"><meta name="DC.Contributor" content="Hannah C. Ainsworth"><meta name="DC.Contributor" content="Patricia Ortiz-Tello"><meta name="DC.Contributor" content="Mary E. Comeau"><meta name="DC.Contributor" content="Astrid Rasmussen"><meta name="DC.Contributor" content="Jennifer A. Kelly"><meta name="DC.Contributor" content="Adam Adler"><meta name="DC.Contributor" content="Eduardo Acevedo-Vázquez"><meta name="DC.Contributor" content="Jorge Mariano Cucho"><meta name="DC.Contributor" content="Ignacio García-De la Torre"><meta name="DC.Contributor" content="Mario H. Cardiel"><meta name="DC.Contributor" content="Pedro Miranda"><meta name="DC.Contributor" content="Luis Catoggio"><meta name="DC.Contributor" content="Marco Maradiaga-Ceceña"><meta name="DC.Contributor" content="Patrick Gaffney"><meta name="DC.Contributor" content="Timothy Vyse"><meta name="DC.Contributor" content="Lindsey A. Criswell"><meta name="DC.Contributor" content="Betty P. Tsao"><meta name="DC.Contributor" content="Kathy L. Sivils"><meta name="DC.Contributor" content="Sang-Cheol Bae"><meta name="DC.Contributor" content="Judith A. James"><meta name="DC.Contributor" content="Robert Kimberly"><meta name="DC.Contributor" content="Ken Kaufman"><meta name="DC.Contributor" content="John B. Harley"><meta name="DC.Contributor" content="Jorge Esquivel-Valerio"><meta name="DC.Contributor" content="José F. Moctezuma"><meta name="DC.Contributor" content="Mercedes A. García"><meta name="DC.Contributor" content="Guillermo Berbotto"><meta name="DC.Contributor" content="Alejandra Babini"><meta name="DC.Contributor" content="Hugo Scherbarth"><meta name="DC.Contributor" content="Sergio Toloza"><meta name="DC.Contributor" content="Vicente Baca"><meta name="DC.Contributor" content="Swapan K. Nath"><meta name="DC.Contributor" content="Carlos Aguilar Salinas"><meta name="DC.Contributor" content="Lorena Orozco"><meta name="DC.Contributor" content="Teresa Tusié-Luna"><meta name="DC.Contributor" content="Raphael Zidovetzki"><meta name="DC.Contributor" content="Bernardo A. Pons-Estel"><meta name="DC.Contributor" content="Carl D. Langefeld"><meta name="DC.Contributor" content="Chaim O. Jacob"><meta name="DC.Date" content="2016 Apr"><meta name="DC.Identifier" content="10.1002/art.39504"><meta name="DC.Language" content="en"><meta property="og:title" content="GWAS in an Amerindian ancestry population reveals novel systemic lupus erythematosus risk loci and the role of European admixture"><meta property="og:type" content="article"><meta property="og:description" content="Systemic lupus erythematosus (SLE) is a chronic autoimmune disease with a strong genetic component. Our aim was to perform the first genome-wide association study on individuals from the Americas enriched for Native American heritage.We analyzed 3,710 ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4829354/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4829354/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4829354/pdf/nihms774837.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D4A91DD58D1010000000002CD01B4.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4829354%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4829354</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Arthritis Rheumatol</a></span>. Author manuscript; available in PMC 2016 Oct 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26606652" target="pmc_ext" ref="reftype=publisher&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Arthritis Rheumatol. 2016 Apr; 68(4): 932–943. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fart.39504" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/art.39504</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4829354</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS774837</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26606652">26606652</a></div></div></div></div><h1 class="content-title">GWAS in an Amerindian ancestry population reveals novel systemic lupus erythematosus risk loci and the role of European admixture</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Alarc%26%23x000f3%3Bn-Riquelme%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319467217232" co-class="co-affbox">Marta E. Alarcón-Riquelme</a>,<sup>1</sup> <a href="/pubmed/?term=Ziegler%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319462368480" co-class="co-affbox">Julie T. Ziegler</a>,<sup>2</sup> <a href="/pubmed/?term=Molineros%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319455892416" co-class="co-affbox">Julio Molineros</a>,<sup>1</sup> <a href="/pubmed/?term=Howard%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319544889776" co-class="co-affbox">Timothy D. Howard</a>,<sup>2</sup> <a href="/pubmed/?term=Moreno-Estrada%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319544887792" co-class="co-affbox">Andrés Moreno-Estrada</a>,<sup>3</sup> <a href="/pubmed/?term=S%26%23x000e1%3Bnchez-Rodr%26%23x000ed%3Bguez%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319464787840" co-class="co-affbox">Elena Sánchez-Rodríguez</a>,<sup>4</sup> <a href="/pubmed/?term=Ainsworth%20HC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319502683664" co-class="co-affbox">Hannah C. Ainsworth</a>,<sup>2</sup> <a href="/pubmed/?term=Ortiz-Tello%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319502681680" co-class="co-affbox">Patricia Ortiz-Tello</a>,<sup>3</sup> <a href="/pubmed/?term=Comeau%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319501175056" co-class="co-affbox">Mary E. Comeau</a>,<sup>2</sup> <a href="/pubmed/?term=Rasmussen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319537084288" co-class="co-affbox">Astrid Rasmussen</a>,<sup>1</sup> <a href="/pubmed/?term=Kelly%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319537082304" co-class="co-affbox">Jennifer A. Kelly</a>,<sup>1</sup> <a href="/pubmed/?term=Adler%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319463177264" co-class="co-affbox">Adam Adler</a>,<sup>1</sup> <a href="/pubmed/?term=Acevedo-V%26%23x000e1%3Bzquez%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319470293504" co-class="co-affbox">Eduardo Acevedo-Vázquez</a>,<sup>5</sup> <a href="/pubmed/?term=Cucho%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319470291520" co-class="co-affbox">Jorge Mariano Cucho</a>,<sup>5</sup> <a href="/pubmed/?term=Garc%26%23x000ed%3Ba-De%20la%20Torre%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319455272864" co-class="co-affbox">Ignacio García-De la Torre</a>,<sup>6</sup> <a href="/pubmed/?term=Cardiel%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319468854976" co-class="co-affbox">Mario H. Cardiel</a>,<sup>7</sup> <a href="/pubmed/?term=Miranda%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319468852992" co-class="co-affbox">Pedro Miranda</a>,<sup>8</sup> <a href="/pubmed/?term=Catoggio%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319536465008" co-class="co-affbox">Luis Catoggio</a>,<sup>9</sup> <a href="/pubmed/?term=Maradiaga-Cece%26%23x000f1%3Ba%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319469635696" co-class="co-affbox">Marco Maradiaga-Ceceña</a>,<sup>10</sup> <a href="/pubmed/?term=Gaffney%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319469633712" co-class="co-affbox">Patrick Gaffney</a>,<sup>1</sup> <a href="/pubmed/?term=Vyse%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319464766080" co-class="co-affbox">Timothy Vyse</a>,<sup>11</sup> <a href="/pubmed/?term=Criswell%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319465919296" co-class="co-affbox">Lindsey A. Criswell</a>,<sup>12</sup> <a href="/pubmed/?term=Tsao%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319465917312" co-class="co-affbox">Betty P. Tsao</a>,<sup>13</sup> <a href="/pubmed/?term=Sivils%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319466569424" co-class="co-affbox">Kathy L. Sivils</a>,<sup>1</sup> <a href="/pubmed/?term=Bae%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319466591904" co-class="co-affbox">Sang-Cheol Bae</a>,<sup>14</sup> <a href="/pubmed/?term=James%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319466589920" co-class="co-affbox">Judith A. James</a>,<sup>1,</sup><sup>15</sup> <a href="/pubmed/?term=Kimberly%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319502576384" co-class="co-affbox">Robert Kimberly</a>,<sup>16</sup> <a href="/pubmed/?term=Kaufman%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319468826480" co-class="co-affbox">Ken Kaufman</a>,<sup>17</sup> <a href="/pubmed/?term=Harley%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319499628352" co-class="co-affbox">John B. Harley</a>,<sup>17</sup> <a href="/pubmed/?term=Esquivel-Valerio%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319499626368" co-class="co-affbox">Jorge Esquivel-Valerio</a>,<sup>18</sup> <a href="/pubmed/?term=Moctezuma%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319466551056" co-class="co-affbox">José F. Moctezuma</a>,<sup>19</sup> <a href="/pubmed/?term=Garc%26%23x000ed%3Ba%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319467389360" co-class="co-affbox">Mercedes A. García</a>,<sup>20</sup> <a href="/pubmed/?term=Berbotto%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319467387376" co-class="co-affbox">Guillermo Berbotto</a>,<sup>21</sup> <a href="/pubmed/?term=Babini%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319460974080" co-class="co-affbox">Alejandra Babini</a>,<sup>22</sup> <a href="/pubmed/?term=Scherbarth%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319469026912" co-class="co-affbox">Hugo Scherbarth</a>,<sup>23</sup> <a href="/pubmed/?term=Toloza%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319469024928" co-class="co-affbox">Sergio Toloza</a>,<sup>24</sup> <a href="/pubmed/?term=Baca%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319456750736" co-class="co-affbox">Vicente Baca</a>,<sup>25</sup> <a href="/pubmed/?term=Nath%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319465073760" co-class="co-affbox">Swapan K. Nath</a>,<sup>1</sup> <a href="/pubmed/?term=Salinas%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319465071776" co-class="co-affbox">Carlos Aguilar Salinas</a>,<sup>26</sup> <a href="/pubmed/?term=Orozco%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319465696688" co-class="co-affbox">Lorena Orozco</a>,<sup>27</sup> <a href="/pubmed/?term=Tusi%26%23x000e9%3B-Luna%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319460144608" co-class="co-affbox">Teresa Tusié-Luna</a>,<sup>26,</sup><sup>28</sup> <a href="/pubmed/?term=Zidovetzki%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319469486336" co-class="co-affbox">Raphael Zidovetzki</a>,<sup>29,</sup><sup>30</sup> <a href="/pubmed/?term=Pons-Estel%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319469483728" co-class="co-affbox">Bernardo A. Pons-Estel</a>,<sup>31</sup> <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319470447488" co-class="co-affbox">Carl D. Langefeld</a>,<sup>2</sup> and  <a href="/pubmed/?term=Jacob%20CO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652" class="affpopup" co-rid="_co_idm140319469790672" co-class="co-affbox">Chaim O. Jacob</a><sup>29</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140319467217232"><h3 class="no_margin">Marta E. Alarcón-Riquelme</h3><p><sup>1</sup>Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK</p><div>Find articles by <a href="/pubmed/?term=Alarc%26%23x000f3%3Bn-Riquelme%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Marta E. Alarcón-Riquelme</a></div></div><div id="_co_idm140319462368480"><h3 class="no_margin">Julie T. Ziegler</h3><p><sup>2</sup>Center for Public Health Genomics, Wake Forest School of Medicine, Winston-Salem, 27157, NC</p><div>Find articles by <a href="/pubmed/?term=Ziegler%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Julie T. Ziegler</a></div></div><div id="_co_idm140319455892416"><h3 class="no_margin">Julio Molineros</h3><p><sup>1</sup>Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK</p><div>Find articles by <a href="/pubmed/?term=Molineros%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Julio Molineros</a></div></div><div id="_co_idm140319544889776"><h3 class="no_margin">Timothy D. Howard</h3><p><sup>2</sup>Center for Public Health Genomics, Wake Forest School of Medicine, Winston-Salem, 27157, NC</p><div>Find articles by <a href="/pubmed/?term=Howard%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Timothy D. Howard</a></div></div><div id="_co_idm140319544887792"><h3 class="no_margin">Andrés Moreno-Estrada</h3><p><sup>3</sup>Department of Genetics, Stanford University School of Medicine, 94305, San Francisco, CA and Laboratorio Nacional de Genómica para la Biodiversidad, Centro de Investigación y de Estudios Avanzados del IPN, Irapuato, 36821, Guanajuato, Mexico</p><div>Find articles by <a href="/pubmed/?term=Moreno-Estrada%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Andrés Moreno-Estrada</a></div></div><div id="_co_idm140319464787840"><h3 class="no_margin">Elena Sánchez-Rodríguez</h3><p><sup>4</sup>Department of Neurology, Icahn School of Medicine at Mount Sinai, New York, NY</p><div>Find articles by <a href="/pubmed/?term=S%26%23x000e1%3Bnchez-Rodr%26%23x000ed%3Bguez%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Elena Sánchez-Rodríguez</a></div></div><div id="_co_idm140319502683664"><h3 class="no_margin">Hannah C. Ainsworth</h3><p><sup>2</sup>Center for Public Health Genomics, Wake Forest School of Medicine, Winston-Salem, 27157, NC</p><div>Find articles by <a href="/pubmed/?term=Ainsworth%20HC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Hannah C. Ainsworth</a></div></div><div id="_co_idm140319502681680"><h3 class="no_margin">Patricia Ortiz-Tello</h3><p><sup>3</sup>Department of Genetics, Stanford University School of Medicine, 94305, San Francisco, CA and Laboratorio Nacional de Genómica para la Biodiversidad, Centro de Investigación y de Estudios Avanzados del IPN, Irapuato, 36821, Guanajuato, Mexico</p><div>Find articles by <a href="/pubmed/?term=Ortiz-Tello%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Patricia Ortiz-Tello</a></div></div><div id="_co_idm140319501175056"><h3 class="no_margin">Mary E. Comeau</h3><p><sup>2</sup>Center for Public Health Genomics, Wake Forest School of Medicine, Winston-Salem, 27157, NC</p><div>Find articles by <a href="/pubmed/?term=Comeau%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Mary E. Comeau</a></div></div><div id="_co_idm140319537084288"><h3 class="no_margin">Astrid Rasmussen</h3><p><sup>1</sup>Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK</p><div>Find articles by <a href="/pubmed/?term=Rasmussen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Astrid Rasmussen</a></div></div><div id="_co_idm140319537082304"><h3 class="no_margin">Jennifer A. Kelly</h3><p><sup>1</sup>Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK</p><div>Find articles by <a href="/pubmed/?term=Kelly%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Jennifer A. Kelly</a></div></div><div id="_co_idm140319463177264"><h3 class="no_margin">Adam Adler</h3><p><sup>1</sup>Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK</p><div>Find articles by <a href="/pubmed/?term=Adler%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Adam Adler</a></div></div><div id="_co_idm140319470293504"><h3 class="no_margin">Eduardo Acevedo-Vázquez</h3><p><sup>5</sup>Hospital Nacional Guillermo Almenara Irigoyen, Lima 41, Perú</p><div>Find articles by <a href="/pubmed/?term=Acevedo-V%26%23x000e1%3Bzquez%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Eduardo Acevedo-Vázquez</a></div></div><div id="_co_idm140319470291520"><h3 class="no_margin">Jorge Mariano Cucho</h3><p><sup>5</sup>Hospital Nacional Guillermo Almenara Irigoyen, Lima 41, Perú</p><div>Find articles by <a href="/pubmed/?term=Cucho%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Jorge Mariano Cucho</a></div></div><div id="_co_idm140319455272864"><h3 class="no_margin">Ignacio García-De la Torre</h3><p><sup>6</sup>Hospital General de Occidente, Zapopan, 45170, Jalisco, Mexico</p><div>Find articles by <a href="/pubmed/?term=Garc%26%23x000ed%3Ba-De%20la%20Torre%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Ignacio García-De la Torre</a></div></div><div id="_co_idm140319468854976"><h3 class="no_margin">Mario H. Cardiel</h3><p><sup>7</sup>Centro de Investigación Clínica de Morelia, Morelia, 58070, Michoacán, Mexico</p><div>Find articles by <a href="/pubmed/?term=Cardiel%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Mario H. Cardiel</a></div></div><div id="_co_idm140319468852992"><h3 class="no_margin">Pedro Miranda</h3><p><sup>8</sup>Centro de Estudios Reumatológicos, Santiago de Chile, Chile</p><div>Find articles by <a href="/pubmed/?term=Miranda%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Pedro Miranda</a></div></div><div id="_co_idm140319536465008"><h3 class="no_margin">Luis Catoggio</h3><p><sup>9</sup>Hospital Italiano de Buenos Aires, 1181, Buenos Aires, Argentina</p><div>Find articles by <a href="/pubmed/?term=Catoggio%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Luis Catoggio</a></div></div><div id="_co_idm140319469635696"><h3 class="no_margin">Marco Maradiaga-Ceceña</h3><p><sup>10</sup>Hospital General de Culiacán, Sinaloa, 80220, Mexico</p><div>Find articles by <a href="/pubmed/?term=Maradiaga-Cece%26%23x000f1%3Ba%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Marco Maradiaga-Ceceña</a></div></div><div id="_co_idm140319469633712"><h3 class="no_margin">Patrick Gaffney</h3><p><sup>1</sup>Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK</p><div>Find articles by <a href="/pubmed/?term=Gaffney%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Patrick Gaffney</a></div></div><div id="_co_idm140319464766080"><h3 class="no_margin">Timothy Vyse</h3><p><sup>11</sup>King’s College, London, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Vyse%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Timothy Vyse</a></div></div><div id="_co_idm140319465919296"><h3 class="no_margin">Lindsey A. Criswell</h3><p><sup>12</sup>University of California San Francisco, San Francisco, CA</p><div>Find articles by <a href="/pubmed/?term=Criswell%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Lindsey A. Criswell</a></div></div><div id="_co_idm140319465917312"><h3 class="no_margin">Betty P. Tsao</h3><p><sup>13</sup>University of California Los Angeles, LA, CA.</p><div>Find articles by <a href="/pubmed/?term=Tsao%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Betty P. Tsao</a></div></div><div id="_co_idm140319466569424"><h3 class="no_margin">Kathy L. Sivils</h3><p><sup>1</sup>Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK</p><div>Find articles by <a href="/pubmed/?term=Sivils%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Kathy L. Sivils</a></div></div><div id="_co_idm140319466591904"><h3 class="no_margin">Sang-Cheol Bae</h3><p><sup>14</sup>Department of Rheumatology, Hanyang University Hospital for Rheumatic Diseases, Seoul 133-792, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Bae%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Sang-Cheol Bae</a></div></div><div id="_co_idm140319466589920"><h3 class="no_margin">Judith A. James</h3><p><sup>1</sup>Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK</p><p><sup>15</sup>University of Alabama at Birmingham AL, 35294</p><div>Find articles by <a href="/pubmed/?term=James%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Judith A. James</a></div></div><div id="_co_idm140319502576384"><h3 class="no_margin">Robert Kimberly</h3><p><sup>16</sup>University of Oklahoma Health Sciences Center, Oklahoma City, 73104, OK</p><div>Find articles by <a href="/pubmed/?term=Kimberly%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Robert Kimberly</a></div></div><div id="_co_idm140319468826480"><h3 class="no_margin">Ken Kaufman</h3><p><sup>17</sup>Children’s Hospital, Cincinnati, OH</p><div>Find articles by <a href="/pubmed/?term=Kaufman%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Ken Kaufman</a></div></div><div id="_co_idm140319499628352"><h3 class="no_margin">John B. Harley</h3><p><sup>17</sup>Children’s Hospital, Cincinnati, OH</p><div>Find articles by <a href="/pubmed/?term=Harley%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">John B. Harley</a></div></div><div id="_co_idm140319499626368"><h3 class="no_margin">Jorge Esquivel-Valerio</h3><p><sup>18</sup>Hospital Universitario “Dr. José Eleuterio González” Universidad Autonoma de Nuevo León, 64020, Monterrey, México</p><div>Find articles by <a href="/pubmed/?term=Esquivel-Valerio%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Jorge Esquivel-Valerio</a></div></div><div id="_co_idm140319466551056"><h3 class="no_margin">José F. Moctezuma</h3><p><sup>19</sup>Hospital General de México, Mexico D.F., Mexico</p><div>Find articles by <a href="/pubmed/?term=Moctezuma%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">José F. Moctezuma</a></div></div><div id="_co_idm140319467389360"><h3 class="no_margin">Mercedes A. García</h3><p><sup>20</sup>H.I.G.A. General San Martin, La Plata, 1900, Argentina</p><div>Find articles by <a href="/pubmed/?term=Garc%26%23x000ed%3Ba%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Mercedes A. García</a></div></div><div id="_co_idm140319467387376"><h3 class="no_margin">Guillermo Berbotto</h3><p><sup>21</sup>Hospital Eva Perón, S2152EDD, Granadero Baigorria, Argentina</p><div>Find articles by <a href="/pubmed/?term=Berbotto%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Guillermo Berbotto</a></div></div><div id="_co_idm140319460974080"><h3 class="no_margin">Alejandra Babini</h3><p><sup>22</sup>Hospital Italiano de Córdoba, Córdoba, Argentina</p><div>Find articles by <a href="/pubmed/?term=Babini%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Alejandra Babini</a></div></div><div id="_co_idm140319469026912"><h3 class="no_margin">Hugo Scherbarth</h3><p><sup>23</sup>H.I.G.A. Oscar E. Alende, Mar del Plata, 7600, Argentina</p><div>Find articles by <a href="/pubmed/?term=Scherbarth%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Hugo Scherbarth</a></div></div><div id="_co_idm140319469024928"><h3 class="no_margin">Sergio Toloza</h3><p><sup>24</sup>Hospital Interzonal San Juan Bautista, Catamarca, 4700, Argentina</p><div>Find articles by <a href="/pubmed/?term=Toloza%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Sergio Toloza</a></div></div><div id="_co_idm140319456750736"><h3 class="no_margin">Vicente Baca</h3><p><sup>25</sup>Centro Médico Nacional Siglo XXI, Instituto Mexicano del Seguro Social, Mexico City, 06720, Mexico</p><div>Find articles by <a href="/pubmed/?term=Baca%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Vicente Baca</a></div></div><div id="_co_idm140319465073760"><h3 class="no_margin">Swapan K. Nath</h3><p><sup>1</sup>Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK</p><div>Find articles by <a href="/pubmed/?term=Nath%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Swapan K. Nath</a></div></div><div id="_co_idm140319465071776"><h3 class="no_margin">Carlos Aguilar Salinas</h3><p><sup>26</sup>Instituto Nacional de Ciencias Médicas y Nutrición Salvador Zubirán, Mexico D.F., Mexico</p><div>Find articles by <a href="/pubmed/?term=Salinas%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Carlos Aguilar Salinas</a></div></div><div id="_co_idm140319465696688"><h3 class="no_margin">Lorena Orozco</h3><p><sup>27</sup>Instituto Nacional de Medicina Genómica, México City, 14610, México</p><div>Find articles by <a href="/pubmed/?term=Orozco%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Lorena Orozco</a></div></div><div id="_co_idm140319460144608"><h3 class="no_margin">Teresa Tusié-Luna</h3><p><sup>26</sup>Instituto Nacional de Ciencias Médicas y Nutrición Salvador Zubirán, Mexico D.F., Mexico</p><p><sup>28</sup>Instituto de Investigaciones Biomédicas de la Universidad Nacional Autónoma de México, Mexico City, Mexico</p><div>Find articles by <a href="/pubmed/?term=Tusi%26%23x000e9%3B-Luna%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Teresa Tusié-Luna</a></div></div><div id="_co_idm140319469486336"><h3 class="no_margin">Raphael Zidovetzki</h3><p><sup>29</sup>The Lupus Genetics Group, Department of Medicine, University of Southern California, Los Angeles, CA</p><p><sup>30</sup>University of California, 92521, Riverside</p><div>Find articles by <a href="/pubmed/?term=Zidovetzki%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Raphael Zidovetzki</a></div></div><div id="_co_idm140319469483728"><h3 class="no_margin">Bernardo A. Pons-Estel</h3><p><sup>31</sup>Sanatorio Parque, Rosario, Argentina</p><div>Find articles by <a href="/pubmed/?term=Pons-Estel%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Bernardo A. Pons-Estel</a></div></div><div id="_co_idm140319470447488"><h3 class="no_margin">Carl D. Langefeld</h3><p><sup>2</sup>Center for Public Health Genomics, Wake Forest School of Medicine, Winston-Salem, 27157, NC</p><div>Find articles by <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Carl D. Langefeld</a></div></div><div id="_co_idm140319469790672"><h3 class="no_margin">Chaim O. Jacob</h3><p><sup>29</sup>The Lupus Genetics Group, Department of Medicine, University of Southern California, Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Jacob%20CO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26606652">Chaim O. Jacob</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140319498793168_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140319498793168_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140319498793168_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Center for Public Health Genomics, Wake Forest School of Medicine, Winston-Salem, 27157, NC</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Genetics, Stanford University School of Medicine, 94305, San Francisco, CA and Laboratorio Nacional de Genómica para la Biodiversidad, Centro de Investigación y de Estudios Avanzados del IPN, Irapuato, 36821, Guanajuato, Mexico</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Neurology, Icahn School of Medicine at Mount Sinai, New York, NY</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Hospital Nacional Guillermo Almenara Irigoyen, Lima 41, Perú</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Hospital General de Occidente, Zapopan, 45170, Jalisco, Mexico</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Centro de Investigación Clínica de Morelia, Morelia, 58070, Michoacán, Mexico</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Centro de Estudios Reumatológicos, Santiago de Chile, Chile</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Hospital Italiano de Buenos Aires, 1181, Buenos Aires, Argentina</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Hospital General de Culiacán, Sinaloa, 80220, Mexico</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>King’s College, London, United Kingdom</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>University of California San Francisco, San Francisco, CA</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>University of California Los Angeles, LA, CA.</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Department of Rheumatology, Hanyang University Hospital for Rheumatic Diseases, Seoul 133-792, Republic of Korea</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>University of Alabama at Birmingham AL, 35294</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>University of Oklahoma Health Sciences Center, Oklahoma City, 73104, OK</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Children’s Hospital, Cincinnati, OH</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Hospital Universitario “Dr. José Eleuterio González” Universidad Autonoma de Nuevo León, 64020, Monterrey, México</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Hospital General de México, Mexico D.F., Mexico</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>H.I.G.A. General San Martin, La Plata, 1900, Argentina</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Hospital Eva Perón, S2152EDD, Granadero Baigorria, Argentina</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Hospital Italiano de Córdoba, Córdoba, Argentina</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>H.I.G.A. Oscar E. Alende, Mar del Plata, 7600, Argentina</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Hospital Interzonal San Juan Bautista, Catamarca, 4700, Argentina</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Centro Médico Nacional Siglo XXI, Instituto Mexicano del Seguro Social, Mexico City, 06720, Mexico</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Instituto Nacional de Ciencias Médicas y Nutrición Salvador Zubirán, Mexico D.F., Mexico</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Instituto Nacional de Medicina Genómica, México City, 14610, México</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Instituto de Investigaciones Biomédicas de la Universidad Nacional Autónoma de México, Mexico City, Mexico</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>The Lupus Genetics Group, Department of Medicine, University of Southern California, Los Angeles, CA</div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>University of California, 92521, Riverside</div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Sanatorio Parque, Rosario, Argentina</div><div id="FN1">Corresponding author: Marta E. Alarcón-Riquelme, Oklahoma Medical Research Foundation, 825 NE 13<sup>th</sup> St, Oklahoma City, OK, USA and GENYO. Centro de Genómica e Investigación Oncológica Pfizer-Universidad de Granada-Junta de Andalucía, PTS Granada, 18016, Granada, Spain. <a href="mailto:dev@null" data-email="gro.frmo@mnocrala" class="oemail">gro.frmo@mnocrala</a>, <a href="mailto:dev@null" data-email="se.oyneg@nocrala.atram" class="oemail">se.oyneg@nocrala.atram</a> or <a href="mailto:dev@null" data-email="es.ik@nocrala.atram" class="oemail">es.ik@nocrala.atram</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140319498793168_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26606652" target="pmc_ext" ref="reftype=publisher&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Arthritis Rheumatol</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4829354/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp.</strong></div><div><a href="/pmc/articles/PMC4829354/bin/NIHMS774837-supplement-Supp.pdf" data-ga-action="click_feat_suppl">NIHMS774837-supplement-Supp.pdf</a><span style="color:gray"> (12M)</span></div><div class="small guid">GUID:&nbsp;08E02547-A2AE-4462-91B4-60B7D32EED70</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supp table S3.</strong></div><div><a href="/pmc/articles/PMC4829354/bin/NIHMS774837-supplement-supp_table_S3.pdf" data-ga-action="click_feat_suppl">NIHMS774837-supplement-supp_table_S3.pdf</a><span style="color:gray"> (74K)</span></div><div class="small guid">GUID:&nbsp;77C2F6C4-130D-47EB-A114-F19DC8C56483</div></div></dd></dl></div><div id="idm140319461162608" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140319461162608title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">OBJECTIVES</h3><p id="P1" class="p p-first-last">Systemic lupus erythematosus (SLE) is a chronic autoimmune disease with a strong genetic component. Our aim was to perform the first genome-wide association study on individuals from the Americas enriched for Native American heritage.</p></div><div id="S2" class="sec"><h3 id="S2title">MATERIALS and METHODS</h3><p id="P2" class="p p-first-last">We analyzed 3,710 individuals from four countries of Latin America and the Unites States diagnosed with SLE and healthy controls. Samples were genotyped with the HumanOmni1 BeadChip. Data of out-of-study controls was obtained for the HumanOmni2.5. Statistical analyses were performed using SNPTEST and SNPGWA. Data was adjusted for genomic control and FDR. Imputation was done using IMPUTE2, and HiBAG for classical HLA alleles.</p></div><div id="S3" class="sec"><h3 id="S3title">RESULTS</h3><p id="P3" class="p p-first-last">The <em>IRF5-TNPO3</em> region showed the strongest association and largest odds ratio (OR) (rs10488631, <em>P<sub>gcadj</sub> =</em> 2.61×10<sup>−29</sup>, OR = 2.12, 95% CI: 1.88–2.39) followed by the HLA class II on the <em>DQA2-DQB1</em> loci (rs9275572, <em>P<sub>gcadj</sub></em> = 1.11 × 10<sup>−16</sup>, OR = 1.62, 95% CI: 1.46–1.80; rs9271366, <em>P<sub>gcadj</sub></em>=6.46 × 10<sup>−12</sup>, OR = 2.06, 95% CI: 1.71–2.50). Other known SLE loci associated were <em>ITGAM</em>, <em>STAT4</em>, <em>TNIP1</em>, <em>NCF2</em> and <em>IRAK1</em>. We identified a novel locus on 10q24.33 (rs4917385, <em>P<sub>gcadj</sub></em> =1.4×10<sup>−8</sup>) with a eQTL effect (<em>P<sub>eqtl</sub></em>=8.0 × 10<sup>−37</sup> at <em>USMG5/miR1307</em>), and describe novel loci. We corroborate SLE-risk loci previously identified in European and Asians. Local ancestry estimation showed that HLA allele risk contribution is of European ancestral origin. Imputation of HLA alleles suggested that autochthonous Native American haplotypes provide protection.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">CONCLUSIONS</h3><p id="P4" class="p p-first-last">Our results show the insight gained by studying admixed populations to delineate the genetic architecture that underlies autoimmune and complex diseases.</p></div></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P5" class="p p-first">“Hispanics” or “Mestizos” in the Americas are a heterogeneous collection of populations with a complex history of Native American, European, African or Asian admixture, resulting from varying geographic origins and individual lineages<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444960">1</a>; <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444964">2</a></sup>. The influence of genetic factors is important to better understand these populations and provide an opportunity to leverage the ancestral contributions to identify genetic factors that predispose to complex genetic traits.</p><p id="P6">Systemic Lupus Erythematosus (SLE) (OMIM # 152700) is the prototypical systemic autoimmune disease with a complex genetic influence. Genome-wide association studies (GWAS) in European and European-American populations have identified over 40 genetic loci<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444970">3</a>; <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444986">4</a></sup>. Hispanics are more likely to develop severe SLE, showing an earlier age at onset and severe renal disease<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444988">5</a>; <a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527445000">6</a></sup>. In fact, global genetic proportions of Native American ancestry, is associated with increased, and European ancestry is associated with decreased risk of developing SLE with severe renal disease<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444973">6</a>; <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444959">7</a></sup>. Consistent with this observation, we have shown that the number of risk alleles for SLE increases with increased global genetic Native American ancestry<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444969">8</a></sup>. These patterns suggest that ancestry contributes to genetic susceptibility through an increased genetic load of risk alleles, and/or that genetic background of Native American ancestry interacts with European risk loci in disease development<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444981">8</a></sup>.</p><p id="P7" class="p p-last">With the aim of identifying new genetic risk loci for SLE contributed by the Native heritage of the Americas we report the first large-scale GWAS of individuals from four countries of Latin America and the Unites States diagnosed with SLE. We compared their allele frequencies across the HumanOmni1-Quad v1.0 BeadArray with healthy controls from the same geographical areas. To improve statistical power, we used a genotyped set of out-of-study controls from Mexico. We imputed classic HLA four-digit alleles and analyzed their impact on risk using univariate and stepwise regression. Based on local ancestry estimates, we attempted to identify the ethnic origins of the risk loci. Finally, we examined the risk loci to determine whether any of these are expression quantitative trait loci (eQTL) from publicly available databases. Together the known and novel loci identified in this GWAS provide a unique picture of the genetic architecture of SLE contributed by the Native American ancestry in its admixture with Europeans and define the HLA as a risk factor contributed by the European admixture.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Samples</h3><p id="P8" class="p p-first-last">All SLE patients were recruited from specialist rheumatology clinics and fulfilled the 1982 American College of Rheumatology criteria<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444977">9</a></sup>. We used two strategies to enrich samples for Native American ancestry. First, previously genotyped Hispanic, Latin American and US Native American individuals with quality control and a set of 253 ancestry informative markers (AIMs) that were part of the Lupus Large Association 2 Study (LLAS2) were included<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444956">6</a>; <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444954">8</a>; <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444990">10</a></sup>. Overall, only thirty five percent of these samples had primarily European and Native American admixture, while the great majority also had Asian, African and &lt;20% Native American ancestry, showing that the Hispanic U.S. and Mestizo Latin American populations are highly admixed. We selected samples with &lt;10% global African or Asian ancestries and at least 20% Native American, as estimated by <em>STRUCTURE</em> and using as reference HapMap data and an in-house reference set of 40 individuals of Nahua origin from Mexico<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444966">8</a>; <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444963">11</a></sup>. Second, we recruited patients from countries historically and demographically known to have a lower frequency of African or Asian ancestries<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444980">12</a></sup>.</p></div><div id="S8" class="sec"><h3 id="S8title">Genotyping and laboratory quality control</h3><p id="P9" class="p p-first">We genotyped all samples on the HumanOmni1-Quad v1.0 BeadChip, using manifest H, at the Oklahoma Medical Research Foundation (OMRF) genotyping core facility. We used the Illumina clustering algorithm with GenomeStudio v2011.1, and the Illumina provided scatter profile. Samples and SNPs were required to pass two-step quality control (QC) measurements. Samples with a SNP call rate below 90% were removed from the study. SNPs with a call rate below 90% or Hardy-Weinberg equilibrium (HWE) p-value &lt;0.0001 were re-clustered using Genome Studio automatic clustering. The cluster profile of SNPs generating a case-control chi square p-value &lt;0.0001 were manually inspected and adjusted to increase accuracy of genotype calls. Genotyping module v1.9.4 was used to complete the genotype clustering and calling. Cluster plots were examined and samples were excluded if they had a SNP call rate &lt;0.90. In addition we integrated a set of 1,432 out-of-study controls genotyped on the Illumina HumanOmni2.5-Quad from the SIGMA type 2 diabetes study in Mexicans<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444974">13</a></sup>. Analyses were completed with data aligned to the positive strand.</p><p id="P10" class="p p-last">In addition, 125 samples were genotyped for classical HLA A, B, C, DRB1, DQA1, DQB1 alleles using the LUMINEX system and data used to evaluate the performance of the imputation of classical HLA alleles (see below).</p></div><div id="S9" class="sec"><h3 id="S9title">Statistical quality control</h3><p id="P11" class="p p-first-last">We considered SNPs to be of high quality if they had call rates &gt;95%, no evidence of differential missingness between cases and controls (p&lt;0.05), and no evidence of departure from Hardy-Weinberg Expectation proportions (controls p&lt;0.01, cases p&lt;0.000001). We made primary inference based on those SNPs with minor allele frequency (MAF) greater than 1%. Based on SNPs passing the quality control thresholds, we removed samples if there was inconsistency between recorded and genetically inferred gender or excess autosomal heterozygosity. We removed duplicates and first- or second-degree relatives based on identity-by-descent statistics computed by the program KING<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444994">14</a></sup>. We performed a principal component (PC) analysis using Eigensoft v3.0, after merging with HapMap phase 3 individuals (CEU, YRI, CHB and MEX) as reference. The PC analysis was performed on a subset of SNPs with MAF ≥ 0.05 outside of the known SLE loci and flanking linked regions, after LD-prunning (r<sup>2</sup>≥0.2). Admixture estimates were computed using ADMIXTURE<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444955">15</a></sup> and genetic outliers removed based on the admixture estimates and first five PCs.</p></div><div id="S10" class="sec"><h3 id="S10title">Statistical analysis</h3><p id="P12" class="p p-first">To test for association between a SNP and case/control status, a logistic regression analysis was computed including the first two PCs as covariates; the two PCs reduced the inflation factor sufficiently so that adding additional PCs as covariates did not further reduced it. The primary analysis reported is the joint analysis of North and South America samples. We confirmed the results were consistent with results from the meta-analysis of North and South American samples computed separately, weighting by sample size and using the program METAL<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444987">16</a></sup>. Similarly, we found comparable results using GEMMA<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444968">17</a></sup>. No heterogeneity was observed between North and South American data (see results), supporting the use of the joint analysis. The primary inference was based on the additive genetic model, unless there was significant lack-of-fit to the additive model (p-value&lt;0.05). If there was evidence of departure from an additive model, then inference was based on the most significant result among the dominant, additive and recessive genetic models. The additive and recessive models were computed only if there were at least 10 and 30 individuals homozygous for the minor allele, respectively. For analysis of the X chromosome, the data analysis was first stratified by gender and then meta-analyzed across gender. The results of the GWAS reported are adjusted for the genomic control inflation factor (λ<sub>GC</sub>). These statistical analyses were performed using SNPTEST and SNPGWA version 4.0 as described below. We visually inspected the cluster plots for the most associated SNPs in the regions to confirm the genotyping quality. To determine how many independent associations were within a genomic region, a manual stepwise conditional logistic regression procedure (i.e., forward selection with backward elimination, entry and exit criteria of p-value&lt;0.0001) was computed. Specifically, for each region that reached genome-wide significance, the top SNP was included as a covariate and the association statistics were re-calculated. SNPs were allowed to enter and exit models in this stepwise fashion until no additional SNPs met a significance threshold of p-value &lt;0.0001. To adjust for the number of tests actually computed, we calculated the false discovery rate (FDR) adjusted p-values using the Benjamini-Hochberg procedure on the genomic control adjusted p values (<em>P<sub>gcadj</sub></em>). We denote these FDR-adjusted p-values as P<sub>FDR</sub><sup><a href="#R18" rid="R18" class=" bibr popnode">18</a></sup></p><p id="P13">The cumulative variance explained by common SNPs was estimated using a variance component model and restricted maximum likelihood estimation as implemented in the program GCTA and adjusting for the PCs as covariates using Yang’s correction factor (c=0 from formula 9) for imperfect LD with causal variants. Estimates are based on SNPs that had &lt;1% missing genotypes and stringent relatedness threshold of 0.025.</p><p id="P14" class="p p-last">We constructed regional plots of association for regions of interest using the program Locuszoom.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Imputation</h3><p id="P15" class="p p-first">We computed SNP genotype imputation across the genome using the 1000Genomes reference panel to localize the top statistical associations. Specifically, we used the program SHAPEIT to pre-phase separately the Illumina HumanOmni1-Quad genotype data and HumanOmni2.5M genotype data for SNPs that passed quality control. We used IMPUTE2 with the 1000 Genomes Phase 1 integrated reference panel to impute the SNP genotypes for the HumanOmni2.5M. We then used IMPUTE2<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444984">19</a>; <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444958">20</a></sup> to impute the SNP genotypes for the HumanOmni1-Quad with the 1000 Genomes Phase 1 integrated reference panel and the previously imputed HumanOmni2.5M chip. We computed tests of association, ORs, 95% CI’s and other summary statistics on those SNPs that had information scores (&gt; 0.5) and confidence scores (&gt; 0.9). We tested for association between imputed SNPs and SLE status using IMPUTE2’s companion program SNPTEST; SNPTEST explicitly accounts for estimated imputation uncertainty (i.e., the posterior probability) in the test of association.</p><div id="S12" class="sec"><p></p><h4 id="S12title" class="inline">Classical human leukocyte antigen (HLA) allele imputation </h4><p id="P16" class="p p-first-last">We imputed the classical human leukocyte antigen (HLA) alleles at HLA A, B, C, DPB1, DQA1, DQB1 and DRB1 using the program HiBAG<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444965">21</a></sup> and its corresponding Hispanic reference dataset. HiBAG uses an ensemble classifier and bagging techniques to arrive at an average posterior probability for the four-digit classical HLA alleles. Because 37.8% of the reference SNPs used by HiBAG were missing genotype data, we repeated the HLA imputation after filling-in missing genotype data with the “best guess” imputed SNP data from the 1000 genomes imputation described above. By using the “best guess” genotype data that had a posterior probability &gt;0.90, the percent of missing SNPs in the reference set was reduced to 0.6% (see <a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a> for details). However, and as should be expected, we observed that the posterior probabilities for the HLA alleles were comparable for the two methods and we are reporting the HLA imputation based on the imputed data incorporating the imputation posterior probability.</p></div><div id="S13" class="sec"><p></p><h4 id="S13title" class="inline">Estimation of Local Ancestry </h4><p id="P17" class="p p-first">Local ancestry assignment was performed using PCAdmix<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444995">22</a></sup> at <em>K=</em>3 ancestral groups. This approach relies on phased data from reference panels and the admixed individuals. To construct our reference panel we combined publicly available data from 26 West African (YRI), 26 European-descent (CEU) HapMap3 trios<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444962">23</a>; <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444961">24</a></sup> and 40 of the Nahua individuals. After merging with all Latino individuals and pruning for missing values at 5% and highly linked markers with r<sup>2</sup>&gt;0.8, the resulting data set comprised 417,563 overlapping SNPs. Individual genotypes from admixed and continental reference samples were phased using BEAGLE 3.0 software <sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444957">25</a></sup>. An assessment of the accuracy of this approach is given in<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444982">26</a></sup>.</p><p id="P18" class="p p-last">For each local ancestry window we performed a Welch’s two sample t-test between cases and controls to assess if there was significant contribution of any given ancestral group to each window’s disease association.</p></div><div id="S14" class="sec sec-last"><p></p><h4 id="S14title" class="inline">Bioinformatic Analysis of eQTL Locus </h4><p id="P19" class="p p-first-last">GENEVAR was used to analyze gene expression variation of the novel locus rs7911488 on <em>UMSG5</em>. Data from HapMap lymphoblasts from individuals of several ancestries was incorporated in the analysis. In addition, ENCODE v3 data was used to investigate transcription factor binding information for the locus. Transfac<sup>™</sup> and Match<sup>™</sup> interface was used to determine if SNP alleles modified binding of the transcription factors to the target DNA sequence. Only high quality vertebrate matrixes were used, with cut-offs minimizing the sum of the false positive and negatives. Prediction of the RNA folding for each of the alleles was done using the RNAfold. Finally, biogps was used to inform about the cellular expression of <em>USMG5</em>.</p></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Results</h2><p id="P20" class="p p-first">The present GWAS contained 3,710 genotyped individuals (1393 SLE cases, 886 controls) and 1431 out-of-study control genotype data from the SIGMA study<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558055421">27</a></sup>. The 1393 SLE cases were composed of 862 individuals of enriched Native American ancestry from Mexico, Hispanics and Native Americans from the United States, collectively denoted North American (NA); and, 531 individuals of enriched Native American ancestry from Argentina, Chile and Peru, collectively denoted as South American (SA). Parallel numbers for within-study-controls were 542 and 344, respectively; all out-of-study controls were of NA origin. The OMNI1 and OMNI2.5 had 996,672 and 1,469,575 SNPs that passed quality control filters, respectively. A total of 580,483 SNPs on both chips are the foundation of the analyses reported here. After quality control, 88.7% of the NA SLE cases and 78.8% NA controls were female; including the NA out-of-study controls reduce that percentage to 65.9% female. Similarly, 91.7% and 89.5% of the SA SLE cases and controls, respectively, were female.</p><p id="P21" class="p">Principal components (PC) analysis identified two PCs that significantly reduced the inflation of our test statistic to 1.18 and were supported by the PC scree plot. The addition of more PCs did not meaningfully impact the inflation. The PC plot shows that two PCs differentiated the populations diverging with a vertex at the European ancestry cluster and NA samples at the top and SA below (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S1</a>). These corresponded to a South American population group (<em>N</em> = 875), and a North American group (<em>N</em> = 2,835). The association analysis probability-probability plot is provided as <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a>. Thus, the analyses reported adjust for two PCs and then use a genomic control adjustment. The joint analysis of NA and SA versus the meta-analysis of NA and SA was carefully examined. Clearly, and because of the out-of-study controls were of NA origin, the statistical power was higher for the NA analysis. As expected, the results were comparable with the joint analysis allowing for a more robust sample size to consider additive and recessive genetic models (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S3</a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>). There were no SNPs with evidence for differential effects between NA and SA that met genome-wide significance or FDR thresholds of significance.</p><div id="S16" class="sec"><h3 id="S16title">Non-HLA Associations</h3><p id="P22" class="p p-first">Results are partitioned into three tiers: Tier 1 P&lt;5×10<sup>−8</sup>, Tier 2, 5×10<sup>−8</sup>&lt;P&lt;1×10<sup>−6</sup>, and Tier 3, not P&gt;1×10<sup>−6</sup> and P<sub>FDR</sub>&lt;0.05. All associations in these tiers meet significance adjusting for the number of tests computed via the Benjamini-Hochberg FDR method. All associations reported have linkage disequilibrium support for the top associations (<a href="/pmc/articles/PMC4829354/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). All loci were subjected to stepwise analysis (conditional logistic regression analysis) following imputation. When the result of this analysis surpassed that of the GC adjusted p value, it is shown in the last column of <a href="/pmc/articles/PMC4829354/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, and the subscript <em>i,</em> shown by the SNP rs number, when the SNP has been imputed. In total, 7 regions met Tier 1 criteria, 4 regions met Tier 2 criteria and 31 additional regions met Tier 3 criteria (<a href="/pmc/articles/PMC4829354/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Table S2</a>, and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Materials LocusZoom plots Figure S4</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p22">Overview of the loci associated in the GWAS with FDR values of &lt;0.05</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140319465077760" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="top" align="left">SNP</th><th rowspan="2" colspan="1" valign="top" align="center">Chr</th><th rowspan="2" colspan="1" valign="top" align="center">Position</th><th rowspan="2" colspan="1" valign="top" align="center">Region <sup>*</sup></th><th rowspan="2" colspan="1" valign="top" align="center">Gene</th><th rowspan="2" colspan="1" valign="top" align="center">RA <sup>°</sup></th><th colspan="2" rowspan="1" valign="bottom" align="center">RAF
<hr></th><th rowspan="2" colspan="1" valign="top" align="center">Best p-value GC</th><th rowspan="2" colspan="1" valign="top" align="center">Best p-value FDR</th><th rowspan="2" colspan="1" valign="top" align="center">OR(CI)</th><th rowspan="2" colspan="1" valign="top" align="center">Stepwise p-value imputation</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">Case</th><th rowspan="1" colspan="1" valign="top" align="center">Control</th></tr></thead><tbody><tr><td colspan="12" rowspan="1" valign="top" align="left"><strong>Tier 1 (Best p-value GC &lt; 5.0E-08)</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12736195<em><sub>i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">1q25</td><td rowspan="1" colspan="1" valign="top" align="center">173307901</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">LOC100506023</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.210</td><td rowspan="1" colspan="1" valign="top" align="center">0.279</td><td rowspan="1" colspan="1" valign="top" align="center">7.31E-11</td><td rowspan="1" colspan="1" valign="top" align="center">1.08E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.66(0.59–0.74)</td><td rowspan="1" colspan="1" valign="top" align="center">1.36E-11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17346550</td><td rowspan="1" colspan="1" valign="top" align="center">1q25</td><td rowspan="1" colspan="1" valign="top" align="center">173315625</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">LOC100506023</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.202</td><td rowspan="1" colspan="1" valign="top" align="center">0.269</td><td rowspan="1" colspan="1" valign="top" align="center">5.59E-10</td><td rowspan="1" colspan="1" valign="top" align="center">5.40E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.66(0.58–0.74)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><sup>†</sup> rs13306575</td><td rowspan="1" colspan="1" valign="top" align="center">1q25</td><td rowspan="1" colspan="1" valign="top" align="center">183532437</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">NCF2</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.117</td><td rowspan="1" colspan="1" valign="top" align="center">0.105</td><td rowspan="1" colspan="1" valign="top" align="center">3.21E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1.62E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.62(1.39–1.91)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11889341</td><td rowspan="1" colspan="1" valign="top" align="center">2q32</td><td rowspan="1" colspan="1" valign="top" align="center">191943742</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">STAT4</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.489</td><td rowspan="1" colspan="1" valign="top" align="center">0.407</td><td rowspan="1" colspan="1" valign="top" align="center">7.02E-13</td><td rowspan="1" colspan="1" valign="top" align="center">1.05E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1.5(1.36–1.66)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7708392<em><sub>r,i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">5q33</td><td rowspan="1" colspan="1" valign="top" align="center">150457485</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">TNIP1</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.536</td><td rowspan="1" colspan="1" valign="top" align="center">0.499</td><td rowspan="1" colspan="1" valign="top" align="center">2.23E-11</td><td rowspan="1" colspan="1" valign="top" align="center">3.42E-07</td><td rowspan="1" colspan="1" valign="top" align="center">1.76(1.50–2.07)</td><td rowspan="1" colspan="1" valign="top" align="center">1.34E-11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10036748<em><sub>r</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">5q33</td><td rowspan="1" colspan="1" valign="top" align="center">150458146</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">TNIP1</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.534</td><td rowspan="1" colspan="1" valign="top" align="center">0.498</td><td rowspan="1" colspan="1" valign="top" align="center">9.56E-11</td><td rowspan="1" colspan="1" valign="top" align="center">1.01E-06</td><td rowspan="1" colspan="1" valign="top" align="center">1.72(1.46–2.02)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4728142</td><td rowspan="1" colspan="1" valign="top" align="center">7q32</td><td rowspan="1" colspan="1" valign="top" align="center">128573967</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">IRF5</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.498</td><td rowspan="1" colspan="1" valign="top" align="center">0.387</td><td rowspan="1" colspan="1" valign="top" align="center">1.06E-23</td><td rowspan="1" colspan="1" valign="top" align="center">1.22E-18</td><td rowspan="1" colspan="1" valign="top" align="center">1.76(1.59–1.95)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10488631</td><td rowspan="1" colspan="1" valign="top" align="center">7q32</td><td rowspan="1" colspan="1" valign="top" align="center">128594183</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">TNPO3</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.295</td><td rowspan="1" colspan="1" valign="top" align="center">0.223</td><td rowspan="1" colspan="1" valign="top" align="center">2.61E-29</td><td rowspan="1" colspan="1" valign="top" align="center">6.04E-24</td><td rowspan="1" colspan="1" valign="top" align="center">2.12(1.88–2.39)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12539741<em><sub>i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">7q32</td><td rowspan="1" colspan="1" valign="top" align="center">128596805</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">TNPO3</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.295</td><td rowspan="1" colspan="1" valign="top" align="center">0.222</td><td rowspan="1" colspan="1" valign="top" align="center">6.24E-31</td><td rowspan="1" colspan="1" valign="top" align="center">1.32E-25</td><td rowspan="1" colspan="1" valign="top" align="center">2.11(1.88–2.37)</td><td rowspan="1" colspan="1" valign="top" align="center">1.26E-34</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4917385<em><sub>i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">10q24</td><td rowspan="1" colspan="1" valign="top" align="center">105003721</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center"><a id="__tag_789274885" class="tag_hotlink" href="/nuccore/BC040734" ref="/nuccore/BC040734"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"BC040734","term_id":"26251903","term_text":"BC040734"}}</span>BC040734</a></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.294</td><td rowspan="1" colspan="1" valign="top" align="center">0.352</td><td rowspan="1" colspan="1" valign="top" align="center">1.39E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1.04E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.72(0.65–0.80)</td><td rowspan="1" colspan="1" valign="top" align="center">9.98E-10</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs34572943</td><td rowspan="1" colspan="1" valign="top" align="center">16p11</td><td rowspan="1" colspan="1" valign="top" align="center">31272353</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">ITGAM</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.150</td><td rowspan="1" colspan="1" valign="top" align="center">0.068</td><td rowspan="1" colspan="1" valign="top" align="center">6.52E-19</td><td rowspan="1" colspan="1" valign="top" align="center">3.78E-14</td><td rowspan="1" colspan="1" valign="top" align="center">2.25(1.91–2.66)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1143679<em><sub>i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">16p11</td><td rowspan="1" colspan="1" valign="top" align="center">31276811</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">ITGAM</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.158</td><td rowspan="1" colspan="1" valign="top" align="center">0.071</td><td rowspan="1" colspan="1" valign="top" align="center">3.26E-21</td><td rowspan="1" colspan="1" valign="top" align="center">2.92E-16</td><td rowspan="1" colspan="1" valign="top" align="center">2.30(1.97–2.70)</td><td rowspan="1" colspan="1" valign="top" align="center">8.82E-24</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1059702</td><td rowspan="1" colspan="1" valign="top" align="center">Xq28</td><td rowspan="1" colspan="1" valign="top" align="center">153284192</td><td rowspan="1" colspan="1" valign="top" align="center">X1</td><td rowspan="1" colspan="1" valign="top" align="center">IRAK1</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.416</td><td rowspan="1" colspan="1" valign="top" align="center">0.500</td><td rowspan="1" colspan="1" valign="top" align="center">8.42E-14</td><td rowspan="1" colspan="1" valign="top" align="center">1.45E-09</td><td rowspan="1" colspan="1" valign="top" align="center">0.58(0.49–0.68)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="12" rowspan="1" valign="top" align="left"><strong>Tier 2 (Best p-value GC &lt; 1.0E-06)</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs643955<em><sub>r,i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">8p23</td><td rowspan="1" colspan="1" valign="top" align="center">9891254</td><td rowspan="1" colspan="1" valign="top" align="center">39</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.759</td><td rowspan="1" colspan="1" valign="top" align="center">0.748</td><td rowspan="1" colspan="1" valign="top" align="center">3.92E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.0016</td><td rowspan="1" colspan="1" valign="top" align="center">1.49(1.28–1.73)</td><td rowspan="1" colspan="1" valign="top" align="center">3.06E-07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7911488</td><td rowspan="1" colspan="1" valign="top" align="center">10q24</td><td rowspan="1" colspan="1" valign="top" align="center">105154089</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">USMG5</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.296</td><td rowspan="1" colspan="1" valign="top" align="center">0.350</td><td rowspan="1" colspan="1" valign="top" align="center">2.48E-07</td><td rowspan="1" colspan="1" valign="top" align="center">9.26E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.74(0.66–0.82)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11231824<em><sub>i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">11q13</td><td rowspan="1" colspan="1" valign="top" align="center">64354795</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">SLC22A12</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.146</td><td rowspan="1" colspan="1" valign="top" align="center">0.111</td><td rowspan="1" colspan="1" valign="top" align="center">7.02E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.0026</td><td rowspan="1" colspan="1" valign="top" align="center">1.51(1.3–1.76)</td><td rowspan="1" colspan="1" valign="top" align="center">8.99E-07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1878186<em><sub>d,i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">15q21</td><td rowspan="1" colspan="1" valign="top" align="center">48508400</td><td rowspan="1" colspan="1" valign="top" align="center">30</td><td rowspan="1" colspan="1" valign="top" align="center">SLC12A1</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.072</td><td rowspan="1" colspan="1" valign="top" align="center">0.047</td><td rowspan="1" colspan="1" valign="top" align="center">1.49E-07</td><td rowspan="1" colspan="1" valign="top" align="center">7.24E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.82(1.45–2.28)</td><td rowspan="1" colspan="1" valign="top" align="center">2.49E-07</td></tr><tr><td colspan="12" rowspan="1" valign="top" align="left"><strong>Tier 3 (FDR p-value &lt; 0.05, selected results)</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17301013<em><sub>d</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">1q25</td><td rowspan="1" colspan="1" valign="top" align="center">174312813</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">RABGAP1L</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.377</td><td rowspan="1" colspan="1" valign="top" align="center">0.417</td><td rowspan="1" colspan="1" valign="top" align="center">8.41E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.0155</td><td rowspan="1" colspan="1" valign="top" align="center">0.70(0.61–0.81)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10254284<em><sub>i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">7p15</td><td rowspan="1" colspan="1" valign="top" align="center">28167391</td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">JAZF1</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.613</td><td rowspan="1" colspan="1" valign="top" align="center">0.583</td><td rowspan="1" colspan="1" valign="top" align="center">8.21E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.0156</td><td rowspan="1" colspan="1" valign="top" align="center">1.29(1.16–1.42)</td><td rowspan="1" colspan="1" valign="top" align="center">4.06E-07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4641121<em><sub>d,i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">9p24</td><td rowspan="1" colspan="1" valign="top" align="center">7071706</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">KDM4C</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.144</td><td rowspan="1" colspan="1" valign="top" align="center">0.149</td><td rowspan="1" colspan="1" valign="top" align="center">8.39E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.0159</td><td rowspan="1" colspan="1" valign="top" align="center">0.68(0.58–0.80)</td><td rowspan="1" colspan="1" valign="top" align="center">4.78E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2928402<em><sub>i</sub></em></td><td rowspan="1" colspan="1" valign="top" align="center">10q11</td><td rowspan="1" colspan="1" valign="top" align="center">50018165</td><td rowspan="1" colspan="1" valign="top" align="center">25</td><td rowspan="1" colspan="1" valign="top" align="center">WDFY4</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.235</td><td rowspan="1" colspan="1" valign="top" align="center">0.182</td><td rowspan="1" colspan="1" valign="top" align="center">1.20E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.0039</td><td rowspan="1" colspan="1" valign="top" align="center">1.39(1.23–1.57)</td><td rowspan="1" colspan="1" valign="top" align="center">1.58E-07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs931127</td><td rowspan="1" colspan="1" valign="top" align="center">11q13</td><td rowspan="1" colspan="1" valign="top" align="center">65405300</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">PCNXL3</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.721</td><td rowspan="1" colspan="1" valign="top" align="center">0.703</td><td rowspan="1" colspan="1" valign="top" align="center">6.71E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.0132</td><td rowspan="1" colspan="1" valign="top" align="center">1.33(1.18–1.48)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7892586</td><td rowspan="1" colspan="1" valign="top" align="center">Xp22</td><td rowspan="1" colspan="1" valign="top" align="center">12833100</td><td rowspan="1" colspan="1" valign="top" align="center">X2</td><td rowspan="1" colspan="1" valign="top" align="center">PRPS2</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.212</td><td rowspan="1" colspan="1" valign="top" align="center">0.288</td><td rowspan="1" colspan="1" valign="top" align="center">3.18E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.0076</td><td rowspan="1" colspan="1" valign="top" align="center">0.73(0.60–0.88)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140319465077760" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4829354/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p23" class="p p-first-last">P-value is based on additive genetic model unless subscript <em>d</em> (dominant) or <em>r</em> (recessive) is present.</p></div><div id="TFN2"><p id="__p24" class="p p-first-last">P-value is based on direct genotypes unless subscript <em>i</em> (imputed) is present.</p></div><div id="TFN3"><sup>†</sup>Within-study versus out-of-study control comparison p-value&lt;0.01</div><div id="TFN4"><sup>*</sup>Region is the order of statistical significance of the hit, beginning with the best p-value across the genome but excluding the MHC.</div><div id="TFN5"><sup>°</sup>RA: Risk allele.</div></div></div><p id="P23">Surprisingly, the most significant association across the entire genome, including the HLA region, was that implicating <em>IRF5</em>/<em>TNPO3</em> (<a href="/pmc/articles/PMC4829354/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> and <a href="/pmc/articles/PMC4829354/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Specifically, rs12539741 (<em>P<sub>gcadj</sub> =</em> 6.24×10<sup>−31</sup>, <em>P<sub>FDR</sub></em> = 1.32×10<sup>−25</sup>, OR = 2.11, 95% CI: 1.88–2.37) together with rs10488631 (<em>P<sub>gcadj</sub> =</em> 2.61×10<sup>−29</sup>, <em>P<sub>FDR</sub></em> = 6.04×10<sup>−24</sup>, OR = 2.12, 95% CI: 1.88–2.39) were the top associations.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4829354/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140319467646992" class="figure"><a href="/pmc/articles/PMC4829354/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4829354_nihms774837f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms774837f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4829354/bin/nihms774837f1.jpg"></a></div><div id="largeobj_idm140319467646992" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4829354/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4829354/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Annotated Manhattan plot of the GWAS using a joint analysis</strong><p id="__p19">The plot shown represents the joint additive association analysis of North American and South American sets of cases and controls.</p></div></div></div><p id="P24">Multiple previously reported non-HLA SLE-predisposing loci in NA-enriched Hispanic samples are supported in these samples (<a href="/pmc/articles/PMC4829354/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Among the most prominent are <em>ITGAM</em> (rs34572943, P<em><sub>gcadj</sub></em> = 6.52×10<sup>−19</sup>; OR = 2.25, 95% CI: 1.91–2.66), <em>STAT4</em> (rs11889341, P<em><sub>gcadj</sub></em> = 3.02×10<sup>−13</sup>; OR = 1.5, 95% CI: 1.36–1.66)<em>, TNIP1</em> (rs7708392, P<em><sub>gcadj</sub></em> = 2.23×10<sup>−11</sup>; OR = 1.76, 95% CI: 1.50–2.07)<em>, NCF2</em> (rs13306575, P<em><sub>gcadj</sub></em> = 3.21×10<sup>−8</sup>; OR = 1.62, 95% CI: 1.39–1.91)<em>,</em> and <em>IRAK1</em> (rs1059702, P<em><sub>gcadj</sub></em> = 8.2×10<sup>−14</sup>; OR = 0.58, 95% CI: 0.49–0.68) in the X chromosome (<a href="/pmc/articles/PMC4829354/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). A locus in proximity to <em>TNFSF4</em> was detected at tier 1 level (rs17346550, LOC100506023, P<em><sub>gcadj</sub></em> = 5.59×10<sup>−10</sup>; OR = 0.66, 95% CI: 0.58–0.74) (<a href="/pmc/articles/PMC4829354/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). A region previously associated in Europeans that was associated, for the first time in individuals with enriched Native American ancestry in our study at tier 3 level was <em>JAZF1</em> (<a href="/pmc/articles/PMC4829354/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S4</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Table S2</a>). A region previously associated in Asians and associated with good support for the first time in a second population was <em>WDFY4</em> at tier level 3, as well as <em>RABGAP1L</em> previously associated as a deletion variant in Koreans<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444999">28</a></sup>.</p><p id="P25">Our GWAS identified a number of novel loci associated with SLE. The region on 10q24.33 (rs4917385, <em>P<sub>gcadj</sub></em> = 1.39 × 10<sup>−8</sup>; OR=0.72, 95% CI: 0.65–0.80; and rs7911488, <em>P<sub>gcadj</sub></em> = 2.48 × 10<sup>−7</sup><em>,</em> OR = 0.74, 95% CI: 0.66–0.82) had 27 Tier 1 and 2 SNPs (i.e., p-values &lt;10<sup>−6</sup>). The association in this region covers several genes (<em>INA, CALHM1, USMG5, PDCD11, PCGF6, NT5C2, COL17A1</em>, <em>CNNM2</em> and <em>miR1307</em>) (<a href="/pmc/articles/PMC4829354/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Using stepwise analysis following imputation, the strongest hit explaining the entire association was rs4917385 (<em>P</em> = 9.98 × 10<sup>−10</sup>) located on <em>RPEL1</em>. Strong LD was observed across the region, and further fine mapping will be needed. Considering the ancestral proximity between Native American and Asian populations, we used data from an ongoing SLE GWAS done on Korean individuals (Lessard, et al., <em>submitted</em>). The start and stop positions on the meta-analysis were based on SNPs that were +/−500kb from rs7911488, with the actual coordinates (b37): chr10:104,661,484 -105,643,134 bp. Importantly, albeit weakly, exactly the same genotyped SNPs showed association in this population (rs11191642, P<em><sub>dom</sub></em>= 1,02 × 10<sup>−2</sup>, and rs7911488, P<em><sub>dom</sub></em>= 1,33 × 10<sup>−2</sup>, <a href="/pmc/articles/PMC4829354/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>, mid panel) as in the Amerindian enriched individuals (rs11191642, P<em><sub>dom</sub></em>= 3.97 × 10<sup>−6</sup>, and rs7911488, P<em><sub>dom</sub></em>= 2,80 × 10<sup>−6</sup>, <a href="/pmc/articles/PMC4829354/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>, top panel) increasing the significance slightly for this locus (rs11191642, P<em><sub>dom</sub></em>= 8.66 × 10<sup>−7</sup>, and rs7911488, P<em><sub>dom</sub></em>= 1,01 × 10<sup>−6</sup>
<a href="/pmc/articles/PMC4829354/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>, bottom panel; <a href="#SD2" rid="SD2" class=" supplementary-material">Table S3</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4829354/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140319470657584" class="figure"><a href="/pmc/articles/PMC4829354/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4829354_nihms774837f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms774837f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4829354/bin/nihms774837f2.jpg"></a></div><div id="largeobj_idm140319470657584" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4829354/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4829354/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>LocusZoom plot of the new locus in chromosome 10</strong><p id="__p20">The data was obtained from the joint additive association analysis using only genotype data covering positions 105Mb to 105.3Mb. SNP rs7911488 lies at <em>USMG5</em> and MIR1307.</p></div></div></div><p id="P26">Two novel regions with tier 2 significance level were <em>SLC22A12</em> and <em>SLC12A1</em> (<a href="/pmc/articles/PMC4829354/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Three novel regions reach tier 3 level of significance and these are <em>KDM4C</em>, <em>PCNXL3</em> and <em>PRPS2</em> in the X chromosome.</p><p id="P27" class="p p-last">Using the FDR threshold of significance of 0.05 we identify other 17 novel and suggestive loci in our study among which are <em>TIPRL, DSTYK, UNC80, ABTB2, MS4A1, MACROD1, NRXN2, RAD51B, CRIP1, CTXN2, CABLES1, CDCD2, PGPEP1</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S2</a>).</p></div><div id="S17" class="sec"><h3 id="S17title">eQTL Analysis of the major novel locus</h3><p id="P28" class="p p-first">The chromosome 10q24.33 region houses <em>USMG5</em> and <em>miR1307</em>, where the associated SNP rs7911488 is located in the loop of the hairpin structure of the pre-miRNA of miR1307<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444992">29</a></sup>. SNP rs7911488 shows a <em>cis</em> eQTL effect (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5</a>) for all populations available<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444996">29</a>; <a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444997">30</a></sup>. Particularly for the populations of interest, values were, for CEU and MEX, rs7911488, <em>P<sub>eqtl</sub></em> = 8.0 × 10<sup>−37</sup> and <em>P<sub>eqtl</sub></em> = 5.1 × 10<sup>−15</sup>, respectively, using HapMap lymphoblastoid cell lines<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444979">30</a></sup>. The data indicated significantly lower <em>USMG5</em> levels in individuals with the AA genotype and higher levels in those with the GG genotype with heterozygotes intermediate between the two. Examination of the ENCODE transcription factor binding data (v3) revealed that rs7911488 lies within the binding region of several transcription factors. Using high quality Transfac<sup>™</sup> matrices we selected the two best transcription factor candidates to bind to the sequence of <em>USMG5</em>. The A variant of rs7911488 exhibited binding sites for two transcription factors: COMP1 (matrix sequence cgagtcGATTGgcaacacagacga, with matrix core in capitals), and RFX1 (gagtcgattgGCAACac). The G variant retained only the binding site for COMP1. Binding only to RFX1 was lost when the risk variant (G) of rs7911488 was present.</p><p id="P29" class="p p-last">The second intriguing mechanism of action for rs7911488 involves the microRNA-1307 (<em>miR1307</em>) gene, which overlaps exon 2 of <em>USMG5</em>. While this microRNA is listed as “provisional” in RefSeq, there is evidence that it is expressed<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444991">29</a></sup>. rs7911488 lies in the middle of this gene, and while it does not alter the sequence of the mature miR-1307 products, it appears to considerably change the structure of the precursor molecule in a way that would likely render it inactive. Similar to <em>USMG5</em>, miR-1307 is encoded from the negative DNA strand. Therefore, the A/G SNP designation, based on the reference sequence, would be the complement, T/C in miR-1307. Based on the predicted folding of RNA from the two alleles, the T allele forms the double-stranded structure of a precursor microRNA molecule (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6</a>, left). This would result in the molecule binding to the Dicer complex and eventual release of the single-stranded microRNAs. The presence of the C allele is predicted to completely alter the secondary structure of the molecule (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6</a>, right), making it unlikely to bind to the Dicer complex required for trimming. These data suggest that only the T allele would result in an active miR-1307, which would affect all of the downstream targets of this molecule.</p></div><div id="S18" class="sec"><h3 id="S18title">MHC Region Association</h3><p id="P30" class="p p-first-last">The HLA region showed several SNPs primarily confined to the class II region, with one peak on the <em>DQA2-DQB1</em> loci (rs9275572, <em>P<sub>gcadj</sub></em> = 1.11 × 10<sup>−16</sup>, OR = 1.62, 95% CI: 1.46–1.80) and a second peak on <em>DRB1-DQA1</em> loci (rs9271366, <em>P<sub>gcadj</sub></em> = 6.46×10<sup>−12</sup> OR = 2.06, 95% CI: 1.71–2.50). The latter SNP is tag for the DR2 haplotype of the HLA allele <em>DRB1*15:01</em> in Europeans<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup>. We observed no association with the tag for DR3 in Europeans, rs2187668 (<em>DRB1*03:01</em>), present in the OMNI1 and 2.5 BeadArrays and passing QC, suggesting that the haplotype structure is different to that of Europeans.</p></div><div id="S19" class="sec"><h3 id="S19title">Imputed Classic HLA Allele Associations</h3><p id="P31" class="p p-first">The imputation of the HLA alleles identified both known and novel four-digit allele associations with SLE. <a href="/pmc/articles/PMC4829354/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> provides the HLA allele associations of <em>P<sub>imp</sub></em>&lt;0.0001 or strongest association by locus. The strongest associations were with <em>DQA1*01:02</em>, <em>DQB1*06:02</em> and <em>DRB1*15:01</em>, known to conform the common European extended haplotype for DR2, and <em>DQA1*05:01</em>, <em>DQB1*02:01</em> and <em>DRB1*03:01</em>, known to conform, in turn the European extended haplotype for DR3<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444953">32</a></sup>. Several alleles frequently found in Native American populations were also observed. For instance, alleles frequent in the Mexican population, but not exclusive to it, such as <em>DQB1*03:01</em> and <em>DQB1*03:02</em> (<a href="/pmc/articles/PMC4829354/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Table S4</a>), appeared to confer protection. Particularly, <em>DQB1*03:01</em> was strongly associated, individually (<em>P<sub>imp</sub></em> = 8.4×10<sup>−12</sup>; OR = 0.60, 95% CI 0.52–0.69). This <em>DQB1</em> allele is found in known Native American haplotypes associated with the <em>DRB1*14:06</em>, <em>DRB1*16:02</em> and <em>DRB1*14:02</em> alleles (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S4</a>). A similar effect was observed for <em>DQB1*03:02</em>, which showed a weaker protective association (<em>P<sub>imp</sub></em> = 2.2×10<sup>−3</sup>; OR = 0.82, 95% CI 0.72–0.93). <em>DQB1*03:02</em> is found in Native American haplotypes when associated with the <em>DRB1*04:07</em> or <em>DRB1*04:11</em> alleles. These <em>DRB1</em> alleles were also weakly associated with a protective effect (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S3</a>). These <em>DRB1</em> alleles also showed individually, weaker associations, all protective, suggesting that Native American haplotypes, in aggregate contribute primarily to protection. Two associations were observed with <em>DRB1</em> alleles that confer protection and that are found in haplotypes of European ancestry, <em>DRB1*11:04</em> and <em>DRB1*11:01</em>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p28">Association Results<sup>*</sup> of Imputed Single-HLA Alleles by Locus</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140319464495920" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="left">HLA-Allele</th><th colspan="2" rowspan="1" valign="bottom" align="left">Dosage Frequency</th><th colspan="2" rowspan="1" valign="bottom" align="left">Best Guess Counts</th><th rowspan="2" colspan="1" valign="bottom" align="left">OR (95%)</th><th rowspan="2" colspan="1" valign="bottom" align="left">P-value</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Cases</th><th rowspan="1" colspan="1" valign="bottom" align="left">Controls</th><th rowspan="1" colspan="1" valign="bottom" align="left">Cases</th><th rowspan="1" colspan="1" valign="bottom" align="left">Controls</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">A<sup>*</sup>01:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.0936</td><td rowspan="1" colspan="1" valign="top" align="left">0.0542</td><td rowspan="1" colspan="1" valign="top" align="left">263</td><td rowspan="1" colspan="1" valign="top" align="left">241</td><td rowspan="1" colspan="1" valign="top" align="left">1.58 (1.30 – 1.93)</td><td rowspan="1" colspan="1" valign="top" align="left">5.52E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">B<sup>*</sup>08:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.0658</td><td rowspan="1" colspan="1" valign="top" align="left">0.0299</td><td rowspan="1" colspan="1" valign="top" align="left">189</td><td rowspan="1" colspan="1" valign="top" align="left">142</td><td rowspan="1" colspan="1" valign="top" align="left">2.06 (1.61 – 2.65)</td><td rowspan="1" colspan="1" valign="top" align="left">1.48E-08</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">C<sup>*</sup>07:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.1087</td><td rowspan="1" colspan="1" valign="top" align="left">0.0746</td><td rowspan="1" colspan="1" valign="top" align="left">308</td><td rowspan="1" colspan="1" valign="top" align="left">348</td><td rowspan="1" colspan="1" valign="top" align="left">1.29 (1.08 – 1.55)</td><td rowspan="1" colspan="1" valign="top" align="left">5.22E-03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DPB1<sup>*</sup>04:02</td><td rowspan="1" colspan="1" valign="top" align="left">0.2779</td><td rowspan="1" colspan="1" valign="top" align="left">0.3522</td><td rowspan="1" colspan="1" valign="top" align="left">819</td><td rowspan="1" colspan="1" valign="top" align="left">1808</td><td rowspan="1" colspan="1" valign="top" align="left">0.84 (0.74 – 0.95)</td><td rowspan="1" colspan="1" valign="top" align="left">5.19E-03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DQA1<sup>*</sup>01:02</td><td rowspan="1" colspan="1" valign="top" align="left">0.1433</td><td rowspan="1" colspan="1" valign="top" align="left">0.0811</td><td rowspan="1" colspan="1" valign="top" align="left">426</td><td rowspan="1" colspan="1" valign="top" align="left">382</td><td rowspan="1" colspan="1" valign="top" align="left">1.94 (1.63 – 2.31)</td><td rowspan="1" colspan="1" valign="top" align="left">1.64E-13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DQA1<sup>*</sup>05:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.1021</td><td rowspan="1" colspan="1" valign="top" align="left">0.0502</td><td rowspan="1" colspan="1" valign="top" align="left">284</td><td rowspan="1" colspan="1" valign="top" align="left">230</td><td rowspan="1" colspan="1" valign="top" align="left">1.91 (1.58 – 2.33)</td><td rowspan="1" colspan="1" valign="top" align="left">6.39E-11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DQA1<sup>*</sup>05:05</td><td rowspan="1" colspan="1" valign="top" align="left">0.0890</td><td rowspan="1" colspan="1" valign="top" align="left">0.1284</td><td rowspan="1" colspan="1" valign="top" align="left">251</td><td rowspan="1" colspan="1" valign="top" align="left">583</td><td rowspan="1" colspan="1" valign="top" align="left">0.59 (0.49 – 0.70)</td><td rowspan="1" colspan="1" valign="top" align="left">1.00E-08</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DQB1<sup>*</sup>06:02</td><td rowspan="1" colspan="1" valign="top" align="left">0.0928</td><td rowspan="1" colspan="1" valign="top" align="left">0.0469</td><td rowspan="1" colspan="1" valign="top" align="left">278</td><td rowspan="1" colspan="1" valign="top" align="left">227</td><td rowspan="1" colspan="1" valign="top" align="left">2.16 (1.75 – 2.67)</td><td rowspan="1" colspan="1" valign="top" align="left">1.10E-12</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DQB1<sup>*</sup>03:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.1413</td><td rowspan="1" colspan="1" valign="top" align="left">0.2090</td><td rowspan="1" colspan="1" valign="top" align="left">409</td><td rowspan="1" colspan="1" valign="top" align="left">1006</td><td rowspan="1" colspan="1" valign="top" align="left">0.60 (0.52 – 0.69)</td><td rowspan="1" colspan="1" valign="top" align="left">8.46E-12</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DQB1<sup>*</sup>02:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.1019</td><td rowspan="1" colspan="1" valign="top" align="left">0.0502</td><td rowspan="1" colspan="1" valign="top" align="left">283</td><td rowspan="1" colspan="1" valign="top" align="left">231</td><td rowspan="1" colspan="1" valign="top" align="left">1.92 (1.58 – 2.33)</td><td rowspan="1" colspan="1" valign="top" align="left">6.66E-11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DRB1<sup>*</sup>15:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.0899</td><td rowspan="1" colspan="1" valign="top" align="left">0.0450</td><td rowspan="1" colspan="1" valign="top" align="left">269</td><td rowspan="1" colspan="1" valign="top" align="left">220</td><td rowspan="1" colspan="1" valign="top" align="left">2.10 (1.70 – 2.59)</td><td rowspan="1" colspan="1" valign="top" align="left">6.82E-12</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DRB1<sup>*</sup>03:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.1018</td><td rowspan="1" colspan="1" valign="top" align="left">0.0496</td><td rowspan="1" colspan="1" valign="top" align="left">284</td><td rowspan="1" colspan="1" valign="top" align="left">230</td><td rowspan="1" colspan="1" valign="top" align="left">1.94 (1.59 – 2.35)</td><td rowspan="1" colspan="1" valign="top" align="left">3.68E-11</td></tr></tbody></table></div><div id="largeobj_idm140319464495920" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4829354/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><sup>*</sup>P-value of 1.0×10<sup>−4</sup> or smallest P-value within an HLA locus. Adjusted for PC1 and PC2</div></div></div><p id="P32">The HLA A and B genes also provided evidence of association and supported the presence of European-derived haplotypes (<a href="/pmc/articles/PMC4829354/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Specifically, <em>A*01:01</em> (<em>P<sub>imp</sub></em> = 5.52×10<sup>−6</sup>, OR=1.58), <em>B*08:01</em> (<em>P<sub>imp</sub></em> = 1.48×10<sup>−8</sup>, OR= 2.06) conferred risk in these Amerindian ancestry-enriched samples. Both are associated in the European haplotype including <em>DRB1*03:01</em>. The <em>C*07:01</em> is also part of this haplotype, however its association was much weaker (<em>P<sub>imp</sub></em> = 5.22 × 10<sup>−3</sup>, OR = 1.29) than for other class I and class II alleles. The European HLA class I gene alleles found in the extended <em>DRB1*15:01</em> haplotype, namely <em>A*03:01</em>, <em>B*07:02</em> and <em>C*07:02</em> were not associated despite the strong association of the corresponding class II alleles.</p><p id="P33" class="p p-last">The joint modeling of classic HLA alleles identified eight alleles spanning the five genes. Some changes were observed from the univariate results: the association of <em>DQA1*01:02</em> became much stronger (<a href="/pmc/articles/PMC4829354/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>), while associations of <em>A*01:01</em>, <em>DRB1*0301</em> and <em>DQB1*03:01</em> became weaker and <em>DQB1*06:02</em> turned from risk to non-risk. The remaining three alleles exhibited only minor modifications from the univariate analysis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p30">Stepwise Analysis using HIBAG-imputed HLA Alleles</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140319502824656" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="left">HLA-Allele</th><th colspan="2" rowspan="1" valign="bottom" align="left">Dosage Frequency</th><th colspan="2" rowspan="1" valign="bottom" align="left">Best Guess Counts</th><th colspan="2" rowspan="1" valign="bottom" align="left">Univariate Results</th><th colspan="2" rowspan="1" valign="bottom" align="left">Stepwise Model</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Cases</th><th rowspan="1" colspan="1" valign="bottom" align="left">Controls</th><th rowspan="1" colspan="1" valign="bottom" align="left">Cases</th><th rowspan="1" colspan="1" valign="bottom" align="left">Controls</th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (CI 95%)</th><th rowspan="1" colspan="1" valign="bottom" align="left">P-value</th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (CI 95%)</th><th rowspan="1" colspan="1" valign="bottom" align="left">P-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">DQA1*01:02</td><td rowspan="1" colspan="1" valign="top" align="left">0.1433</td><td rowspan="1" colspan="1" valign="top" align="left">0.0811</td><td rowspan="1" colspan="1" valign="top" align="left">426</td><td rowspan="1" colspan="1" valign="top" align="left">382</td><td rowspan="1" colspan="1" valign="top" align="left">1.94 (1.63 – 2.31)</td><td rowspan="1" colspan="1" valign="top" align="left">1.64E-13</td><td rowspan="1" colspan="1" valign="top" align="left">2.41 (1.96–2.95)</td><td rowspan="1" colspan="1" valign="top" align="left">3.96E-17</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DRB1*03:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.1018</td><td rowspan="1" colspan="1" valign="top" align="left">0.0496</td><td rowspan="1" colspan="1" valign="top" align="left">284</td><td rowspan="1" colspan="1" valign="top" align="left">230</td><td rowspan="1" colspan="1" valign="top" align="left">1.94 (1.59 – 2.35)</td><td rowspan="1" colspan="1" valign="top" align="left">3.68E-11</td><td rowspan="1" colspan="1" valign="top" align="left">1.98 (1.61–2.45)</td><td rowspan="1" colspan="1" valign="top" align="left">1.75E-10</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DQB1*03:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.1413</td><td rowspan="1" colspan="1" valign="top" align="left">0.2090</td><td rowspan="1" colspan="1" valign="top" align="left">409</td><td rowspan="1" colspan="1" valign="top" align="left">1006</td><td rowspan="1" colspan="1" valign="top" align="left">0.60 (0.52 – 0.69)</td><td rowspan="1" colspan="1" valign="top" align="left">8.46E-12</td><td rowspan="1" colspan="1" valign="top" align="left">0.74 (0.64–0.87)</td><td rowspan="1" colspan="1" valign="top" align="left">2.71E-04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DQB1*06:02</td><td rowspan="1" colspan="1" valign="top" align="left">0.0928</td><td rowspan="1" colspan="1" valign="top" align="left">0.0469</td><td rowspan="1" colspan="1" valign="top" align="left">278</td><td rowspan="1" colspan="1" valign="top" align="left">227</td><td rowspan="1" colspan="1" valign="top" align="left">2.16 (1.75 – 2.67)</td><td rowspan="1" colspan="1" valign="top" align="left">1.10E-12</td><td rowspan="1" colspan="1" valign="top" align="left">0.42 (0.26–0.69)</td><td rowspan="1" colspan="1" valign="top" align="left">5.21E-04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">A*01:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.0936</td><td rowspan="1" colspan="1" valign="top" align="left">0.0542</td><td rowspan="1" colspan="1" valign="top" align="left">263</td><td rowspan="1" colspan="1" valign="top" align="left">241</td><td rowspan="1" colspan="1" valign="top" align="left">1.58 (1.30 – 1.93)</td><td rowspan="1" colspan="1" valign="top" align="left">5.52E-06</td><td rowspan="1" colspan="1" valign="top" align="left">1.41 (1.15–1.74)</td><td rowspan="1" colspan="1" valign="top" align="left">1.16E-03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DRB1*08:02</td><td rowspan="1" colspan="1" valign="top" align="left">0.1192</td><td rowspan="1" colspan="1" valign="top" align="left">0.1270</td><td rowspan="1" colspan="1" valign="top" align="left">402</td><td rowspan="1" colspan="1" valign="top" align="left">702</td><td rowspan="1" colspan="1" valign="top" align="left">1.22 (1.02 – 1.45)</td><td rowspan="1" colspan="1" valign="top" align="left">2.65E-02</td><td rowspan="1" colspan="1" valign="top" align="left">1.36 (1.13–1.64)</td><td rowspan="1" colspan="1" valign="top" align="left">1.21E-03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">B*42:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.0054</td><td rowspan="1" colspan="1" valign="top" align="left">0.0025</td><td rowspan="1" colspan="1" valign="top" align="left">20</td><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">4.54 (1.54 – 13.40)</td><td rowspan="1" colspan="1" valign="top" align="left">6.16E-03</td><td rowspan="1" colspan="1" valign="top" align="left">6.05 (2.01–18.27)</td><td rowspan="1" colspan="1" valign="top" align="left">1.40E-03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">A*11:01</td><td rowspan="1" colspan="1" valign="top" align="left">0.0511</td><td rowspan="1" colspan="1" valign="top" align="left">0.0335</td><td rowspan="1" colspan="1" valign="top" align="left">141</td><td rowspan="1" colspan="1" valign="top" align="left">153</td><td rowspan="1" colspan="1" valign="top" align="left">1.40 (1.09 – 1.79)</td><td rowspan="1" colspan="1" valign="top" align="left">7.62E-03</td><td rowspan="1" colspan="1" valign="top" align="left">1.49 (1.15–1.92)</td><td rowspan="1" colspan="1" valign="top" align="left">2.22E-03</td></tr></tbody></table></div><div id="largeobj_idm140319502824656" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4829354/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN7"><p id="__p31" class="p p-first-last">SLS/SLE=0.01 and adjusted for PC1 and PC2. Sorted by Stepwise P-value.</p></div></div></div></div><div id="S20" class="sec sec-last"><h3 id="S20title">Local ancestry analysis of the HLA locus</h3><p id="P34" class="p p-first-last">To define if any genetic association could involve particular continental ancestries, we performed local ancestry estimations using <em>PCAdmix</em><sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444967">22</a></sup>. The posterior probabilities can be depicted for each phased chromosome (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S7</a>). We analyzed each associated locus for SNP windows showing association between cases and controls in any given ancestry. Confirming our imputation data, the <em>HLA</em> region in chromosome 6 (29.3–33.3Mbp) had 4 continuous windows with 863 SNPs with significant differences in the European ancestry between cases and controls (<a href="/pmc/articles/PMC4829354/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). The reminder of the loci had no specific significant differences in any particular ancestry, suggesting that most loci composed of common variants are found across all continental ancestry borders.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4829354/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140319465080336" class="figure"><a href="/pmc/articles/PMC4829354/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4829354_nihms774837f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms774837f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4829354/bin/nihms774837f3.jpg"></a></div><div id="largeobj_idm140319465080336" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4829354/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4829354/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Average European Ancestry proportion at the HLA Region</strong><p id="__p21">Local ancestry was estimated in each of four windows. The average distribution of local European HLA ancestry was significantly different between cases and controls (P=2.2×10<sup>−6</sup>) across all windows, and ancestry was highest in the window reflecting the region 32: 32.3–33.3M containing the HLA DRB1 to DQA2, the region of the MHC with the main genetic association (see Figure 3). The regions depicted cover the HLA class I, III and II, in order from 29–33Mb. The European ancestry within the HLA conferred a higher risk for SLE. Numbers below each notched bar are: 1: controls, 2: cases.</p></div></div></div></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Discussion</h2><p id="P35" class="p p-first">This is the first GWAS in SLE patients enriched with Native American ancestry. The opportunity to perform a complete GWAS provides an interesting view of the genetic structure of the population. In this study we have several findings: we clearly show a different pattern of genetic association where the genetic association within the <em>IRF5-TNPO3</em> locus surpasses that of the MHC, the main and major association for autoimmune disease in Europeans and in most populations. Using two approaches, local ancestry estimation and imputation of classical HLA alleles show that risk for lupus within the HLA region is given by the European ancestral alleles involving <em>DRB1*03:01</em> and <em>DRB1*15:01</em>, while all autochthonous Amerindian ancestral alleles identified were protective<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444998">32</a></sup>. It is feasible that the degree of diversity of the Native American populations is such that there are several potential, but low frequency risk alleles or novel functional alleles that require much more power to be consistently associated with SLE and remain to be discovered.</p><p id="P36">Our present results and our previous observations that individuals with higher Native American global ancestry had increased numbers of risk alleles seem to be contradictory. However, the most plausible explanation is that the number of alleles required to attain a level of risk for SLE in the Native American population is higher.</p><p id="P37">The imputation of HLA alleles using HiBAG in a primarily Native American population may be a controversial issue due to the scant information available and the complexity of the admixture. To ascertain the fidelity of our data, we performed a comparison of imputed and genotyped alleles in 87 individuals. Agreement between two- and four-digit HLA genotyping with that of HIBAG imputation is high (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure S8</a>) reinforcing our results and allowing for several conclusions.</p><p id="P38">Interestingly, the classical haplotype in Europeans for <em>DRB1*15:01</em>, class I loci (<em>A*03:01</em>, <em>B*07:02</em> and <em>C*07:02</em>), was not associated in this population. This finding suggests that the class II associations are independent and would support a major role for class II and not class I alleles in SLE. This is in agreement with other studies that performed trans-ancestral mapping of the HLA region in SLE<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444972">32</a></sup>.</p><p id="P39">We identify 23 completely new suggestive loci. Also, as recent admixture can produce extended blocks of linkage disequilibrium, several of these regions need to be analyzed individually. For example, <em>PCNXL3</em> is located close to <em>RNASEH2C</em>, a gene implicated in the Aicardi-Goutieres interferonopathy<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444978">33</a></sup>, a monogenic disease resembling SLE in infants, and <em>PRPS2</em> is located close to TLR7 in the X chromosome. In one study, the genetic association with TLR7 was found exclusively in males, but a second study found the association in males and females<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444976">34</a></sup>. We do not observe a male tendency in our study (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S5</a>). We observe the association in females, while, as expected, the frequency in males, is reduced.</p><p id="P40">One interesting finding is the strong association of the <em>TNPO3-IRF5</em> locus. This association surpassed even the MHC suggesting major alternative mechanisms towards disease development. We had previously reported this association in Mexicans<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444989">35</a></sup>. This locus shows a stronger effect than that for published European lineages<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444975">36</a></sup>. The reason for this is unclear. <em>IRF5</em> is a transcription factor clearly involved in the type I interferon and pro-inflammatory signaling, and risk alleles for this locus associate with increased expression of <em>IRF5</em>. Most importantly, studies by Niewold, et al have shown that there are specific associations of <em>IRF5-TNPO3</em> haplotypes with anti-RBP antibodies (the collective name for anti-Sm, anti-Ro and anti-nRNP antibodies). Both, the <em>IRF5</em> haplotypes and the serology strongly associated with increased levels of IFNα in serum<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444985">37</a></sup>, a key feature of SLE. The role of <em>TNPO3</em>, if any, is not understood, but a recent article by Kottyan, et al where both genes were carefully analyzed in several ethnicities showed an independent haplotype that included both genes tagged by SNP rs12534421 that originated in European or European-admixed populations but not found in Africans from HapMap or 1000 genomes. Another haplotype was found in all ethnicities tagged by SNP rs4728142 and located in the promoter of <em>IRF5</em>. Local admixture estimations done showed increased European admixture at the <em>IRF5-TNPO3</em> locus in African-American cases<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444983">38</a></sup>. Our local ancestry analysis in our admixed Amerindian-European sample does not specifically provide evidence for increased European ancestry within the <em>IRF5-TNPO3</em> locus although this possibility appears much likely and would further support a role of European ancestry in the inheritance of <em>IRF5</em> SNPs in disease susceptibility, as we previously proposed<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444971">35</a></sup>.</p><p id="P41">Local ancestry analysis provided information on the impact of other ancestries in our loci, and interestingly, none with the exception of the HLA showed a clear ancestral “bias”. This could require a larger sample size. Fine structural consequences of admixture and their impact on the development of disease in rapidly expanding and admixed populations is important to understand so we can eventually define differences that might have an impact on the mechanisms of disease development and the design of new individual therapies.</p><p id="P42">Our data suggesting the involvement of the <em>USMG5</em> gene and the miR-1307 requires further investigation. Whether the expression of this microRNA may impact numerous targets of importance in lupus needs to be experimentally tested. <em>USMG5</em> is widely expressed in the central nervous system, but also in CD105+ endothelial cells, CD34+ hematopoietic stem cells, dendritic, myeloid and NK cells. RFX1, the transcription factor whose binding was abolished according to Transfac analysis, is known to regulate expression of the HLA class II molecules. RFX1 was recently found to regulate the expression of CD11a and CD70 in T cells from SLE patients<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527444993">39</a></sup>.</p><p id="P43" class="p p-last">In conclusion, we suggest the existence of 23 novel loci for lupus in this population that need replication and one completely novel locus in chromosome 10 replicated in Asians. Several of the major genes for lupus previously confirmed in Europeans and Asians are also associated in this population showing that the major genes for this disease are clearly established and associated across ethnicities.</p></div><div id="S22" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supp</h4><div class="sup-box half_rhythm" id="d37e1697"><a href="/pmc/articles/PMC4829354/bin/NIHMS774837-supplement-Supp.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(12M, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>supp table S3</h4><div class="sup-box half_rhythm" id="d37e1701"><a href="/pmc/articles/PMC4829354/bin/NIHMS774837-supplement-supp_table_S3.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(74K, pdf)</sup></div></div></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">Acknowledgments</h2><div class="sec"><p id="__p1"><strong>Funding</strong></p><p id="__p2">Funding was obtained from the U.S. National Institutes of Health (R01CA141700 and RC1AR058621) to MEA-R, the SLEGEN Consortium (P01AI083194 directed by JBH, where MEA-R was PI and COJ Co-PI of the project), COJ (R01AR057172), the Alliance for Lupus Research (COJ,BPT), P01AR049084 to RK, R01AR060366, R21AI107176 to SN, U19A1082714, U01A101934, US4GM104938, P30AR053483 and P30GM103510 to JAJ, RO1AR043814, R21AR065626 to BPT.</p><p id="__p3">The authors would like to acknowledge the following: Wake Forest Center for Public Health Genomics for personnel support and computing support, Ms Laura Riba, who passed away 2014, for her excellent help in preparation and organization of Mexican of samples, Ms Farideh Movafagh and Ms Rosario Rodríguez-Guillén for technical assistance.</p></div></div><div id="idm140319470652624" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140319470652624title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"> </p><p id="__p4" class="p p-first"><strong>Conflicts of Interest</strong></p><p id="__p5" class="p p-last">The authors declare no conflicts of interest</p><p></p><p class="fn sec" id="FN3"></p><p id="__p6" class="p p-first"><strong>Web Resources</strong></p><p id="__p7" class="p">KING (<a href="http://people.virginia.edu/~wc9c/KING/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://people.virginia.edu/~wc9c/KING/</a>)</p><p id="__p8" class="p">Eigensoft v3.0 (<a href="http://www.hsph.harvard.edu/faculty/alkes-price/software/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.hsph.harvard.edu/faculty/alkes-price/software/</a>)</p><p id="__p9" class="p">ADMIXTURE (<a href="http://www.genetics.ucla.edu/software/admixture/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genetics.ucla.edu/software/admixture/</a>)</p><p id="__p10" class="p">SNPGWA version 4.0 (<a href="http://www.phs.wfubmc.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.phs.wfubmc.edu</a>)</p><p id="__p11" class="p">Locuszoom (<a href="http://csg.sph.umich.edu/locuszoom/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/locuszoom/</a>)</p><p id="__p12" class="p">SHAPEIT (<a href="http://www.shapeit.fr/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.shapeit.fr/</a>)</p><p id="__p13" class="p">IMPUTE2 (<a href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/impute/impute_v2.html</a>)</p><p id="__p14" class="p">SNPTEST (<a href="https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html</a>)</p><p id="__p15" class="p">GENEVAR(<a href="http://www.sanger.ac.uk/resources/software/genevar/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sanger.ac.uk/resources/software/genevar/</a>)</p><p id="__p16" class="p">Transfac<sup>™</sup> and Match<sup>™</sup>: <a href="http://www.gene-regulation.com/cgi-bin/pub/programs/match/bin/match.cgi" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gene-regulation.com/cgi-bin/pub/programs/match/bin/match.cgi</a></p><p id="__p17" class="p">RNAfold Webserver (<a href="http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi</a>)</p><p id="__p18" class="p p-last">biogps (<a href="http://biogps.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://biogps.org</a>)</p><p></p></div></div><div id="idm140319500525856" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140319500525856title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Pons-Estel BA, Catoggio LJ, Cardiel MH, Soriano ER, Gentiletti S, Villa AR, Abadi I, Caeiro F, Alvarellos A, Alarcon-Segovia D. The GLADEL multinational Latin American prospective inception cohort of 1,214 patients with systemic lupus erythematosus: ethnic and disease heterogeneity among “Hispanics” <span><span class="ref-journal">Medicine (Baltimore) </span>2004;<span class="ref-vol">83</span>:1–17.</span> [<a href="/pubmed/14747764" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Medicine+(Baltimore)&amp;title=The+GLADEL+multinational+Latin+American+prospective+inception+cohort+of+1,214+patients+with+systemic+lupus+erythematosus:+ethnic+and+disease+heterogeneity+among+“Hispanics”&amp;author=BA+Pons-Estel&amp;author=LJ+Catoggio&amp;author=MH+Cardiel&amp;author=ER+Soriano&amp;author=S+Gentiletti&amp;volume=83&amp;publication_year=2004&amp;pages=1-17&amp;pmid=14747764&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Auton A, Bryc K, Boyko AR, Lohmueller KE, Novembre J, Reynolds A, Indap A, Wright MH, Degenhardt JD, Gutenkunst RN, et al.  Global distribution of genomic diversity underscores rich complex history of continental human populations. <span><span class="ref-journal">Genome research. </span>2009;<span class="ref-vol">19</span>:795–803.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2675968/">PMC free article</a>]</span> [<a href="/pubmed/19218534" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+research&amp;title=Global+distribution+of+genomic+diversity+underscores+rich+complex+history+of+continental+human+populations&amp;author=A+Auton&amp;author=K+Bryc&amp;author=AR+Boyko&amp;author=KE+Lohmueller&amp;author=J+Novembre&amp;volume=19&amp;publication_year=2009&amp;pages=795-803&amp;pmid=19218534&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Harley JB, Alarcon-Riquelme ME, Criswell LA, Jacob CO, Kimberly RP, Moser KL, Tsao BP, Vyse TJ, Langefeld CD. Genome-wide association scan in women with systemic lupus erythematosus identifies susceptibility variants in ITGAM, PXK, KIAA1542 and other loci. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:204–210.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3712260/">PMC free article</a>]</span> [<a href="/pubmed/18204446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+scan+in+women+with+systemic+lupus+erythematosus+identifies+susceptibility+variants+in+ITGAM,+PXK,+KIAA1542+and+other+loci&amp;author=JB+Harley&amp;author=ME+Alarcon-Riquelme&amp;author=LA+Criswell&amp;author=CO+Jacob&amp;author=RP+Kimberly&amp;volume=40&amp;publication_year=2008&amp;pages=204-210&amp;pmid=18204446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Hom G, Graham RR, Modrek B, Taylor KE, Ortmann W, Garnier S, Lee AT, Chung SA, Ferreira RC, Pant PV, et al.  Association of systemic lupus erythematosus with C8orf13-BLK and ITGAM-ITGAX. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">358</span>:900–909.</span> [<a href="/pubmed/18204098" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Association+of+systemic+lupus+erythematosus+with+C8orf13-BLK+and+ITGAM-ITGAX&amp;author=G+Hom&amp;author=RR+Graham&amp;author=B+Modrek&amp;author=KE+Taylor&amp;author=W+Ortmann&amp;volume=358&amp;publication_year=2008&amp;pages=900-909&amp;pmid=18204098&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Calvo-Alen J, Reveille JD, Rodriguez-Valverde V, McGwin G, Jr, Baethge BA, Friedman AW, Alarcon GS. Clinical, immunogenetic and outcome features of Hispanic systemic lupus erythematosus patients of different ethnic ancestry. <span><span class="ref-journal">Lupus. </span>2003;<span class="ref-vol">12</span>:377–385.</span> [<a href="/pubmed/12765301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lupus&amp;title=Clinical,+immunogenetic+and+outcome+features+of+Hispanic+systemic+lupus+erythematosus+patients+of+different+ethnic+ancestry&amp;author=J+Calvo-Alen&amp;author=JD+Reveille&amp;author=V+Rodriguez-Valverde&amp;author=G+McGwin&amp;author=BA+Baethge&amp;volume=12&amp;publication_year=2003&amp;pages=377-385&amp;pmid=12765301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Sanchez E, Rasmussen A, Riba L, Acevedo-Vasquez E, Kelly JA, Langefeld CD, Williams AH, Ziegler JT, Comeau ME, Marion MC, et al.  Impact of genetic ancestry and sociodemographic status on the clinical expression of systemic lupus erythematosus in American Indian-European populations. <span><span class="ref-journal">Arthritis and Rheumatism. </span>2012;<span class="ref-vol">64</span>:3687–3694.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3485439/">PMC free article</a>]</span> [<a href="/pubmed/22886787" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+Rheumatism&amp;title=Impact+of+genetic+ancestry+and+sociodemographic+status+on+the+clinical+expression+of+systemic+lupus+erythematosus+in+American+Indian-European+populations&amp;author=E+Sanchez&amp;author=A+Rasmussen&amp;author=L+Riba&amp;author=E+Acevedo-Vasquez&amp;author=JA+Kelly&amp;volume=64&amp;publication_year=2012&amp;pages=3687-3694&amp;pmid=22886787&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Richman IB, Taylor KE, Chung SA, Trupin L, Petri M, Yelin E, Graham RR, Lee A, Behrens TW, Gregersen PK, et al.  European genetic ancestry is associated with a decreased risk of lupus nephritis. <span><span class="ref-journal">Arthritis and Rheumatism. </span>2012;<span class="ref-vol">64</span>:3374–3382.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3865923/">PMC free article</a>]</span> [<a href="/pubmed/23023776" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+Rheumatism&amp;title=European+genetic+ancestry+is+associated+with+a+decreased+risk+of+lupus+nephritis&amp;author=IB+Richman&amp;author=KE+Taylor&amp;author=SA+Chung&amp;author=L+Trupin&amp;author=M+Petri&amp;volume=64&amp;publication_year=2012&amp;pages=3374-3382&amp;pmid=23023776&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Sanchez E, Webb RD, Rasmussen A, Kelly JA, Riba L, Kaufman KM, Garcia-de la Torre I, Moctezuma JF, Maradiaga-Cecena MA, Cardiel-Rios MH, et al.  Genetically determined Amerindian ancestry correlates with increased frequency of risk alleles for systemic lupus erythematosus. <span><span class="ref-journal">Arthritis and Rheumatism. </span>2010;<span class="ref-vol">62</span>:3722–3729.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3078084/">PMC free article</a>]</span> [<a href="/pubmed/20848568" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+Rheumatism&amp;title=Genetically+determined+Amerindian+ancestry+correlates+with+increased+frequency+of+risk+alleles+for+systemic+lupus+erythematosus&amp;author=E+Sanchez&amp;author=RD+Webb&amp;author=A+Rasmussen&amp;author=JA+Kelly&amp;author=L+Riba&amp;volume=62&amp;publication_year=2010&amp;pages=3722-3729&amp;pmid=20848568&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Tan EM, Cohen AS, Fries JF, Masi AT, McShane DJ, Rothfield NF, Schaller JG, Talal N, Winchester RJ. The 1982 revised criteria for the classification of systemic lupus erythematosus. <span><span class="ref-journal">Arthritis Rheum. </span>1982;<span class="ref-vol">25</span>:1271–1277.</span> [<a href="/pubmed/7138600" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=The+1982+revised+criteria+for+the+classification+of+systemic+lupus+erythematosus&amp;author=EM+Tan&amp;author=AS+Cohen&amp;author=JF+Fries&amp;author=AT+Masi&amp;author=DJ+McShane&amp;volume=25&amp;publication_year=1982&amp;pages=1271-1277&amp;pmid=7138600&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Sanchez E, Nadig A, Richardson BC, Freedman BI, Kaufman KM, Kelly JA, Niewold TB, Kamen DL, Gilkeson GS, Ziegler JT, et al.  Phenotypic associations of genetic susceptibility loci in systemic lupus erythematosus. <span><span class="ref-journal">Annals of the rheumatic diseases. </span>2011;<span class="ref-vol">70</span>:1752–1757.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232181/">PMC free article</a>]</span> [<a href="/pubmed/21719445" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+the+rheumatic+diseases&amp;title=Phenotypic+associations+of+genetic+susceptibility+loci+in+systemic+lupus+erythematosus&amp;author=E+Sanchez&amp;author=A+Nadig&amp;author=BC+Richardson&amp;author=BI+Freedman&amp;author=KM+Kaufman&amp;volume=70&amp;publication_year=2011&amp;pages=1752-1757&amp;pmid=21719445&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Gomez M, Clark RM, Nath SK, Bhatti S, Sharma R, Alonso E, Rasmussen A, Bidichandani SI. Genetic admixture of European FRDA genes is the cause of Friedreich ataxia in the Mexican population. <span><span class="ref-journal">Genomics. </span>2004;<span class="ref-vol">84</span>:779–784.</span> [<a href="/pubmed/15475256" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Genetic+admixture+of+European+FRDA+genes+is+the+cause+of+Friedreich+ataxia+in+the+Mexican+population&amp;author=M+Gomez&amp;author=RM+Clark&amp;author=SK+Nath&amp;author=S+Bhatti&amp;author=R+Sharma&amp;volume=84&amp;publication_year=2004&amp;pages=779-784&amp;pmid=15475256&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Wang S, Ray N, Rojas W, Parra MV, Bedoya G, Gallo C, Poletti G, Mazzotti G, Hill K, Hurtado AM, et al.  Geographic patterns of genome admixture in Latin American Mestizos. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000037.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265669/">PMC free article</a>]</span> [<a href="/pubmed/18369456" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Geographic+patterns+of+genome+admixture+in+Latin+American+Mestizos&amp;author=S+Wang&amp;author=N+Ray&amp;author=W+Rojas&amp;author=MV+Parra&amp;author=G+Bedoya&amp;volume=4&amp;publication_year=2008&amp;pages=e1000037&amp;pmid=18369456&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Williams AL, Jacobs SB, Moreno-Macias H, Huerta-Chagoya A, Churchhouse C, Marquez-Luna C, Garcia-Ortiz H, Gomez-Vazquez MJ, Burtt NP, Aguilar-Salinas CA, et al.  Sequence variants in SLC16A11 are a common risk factor for type 2 diabetes in Mexico. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">506</span>:97–101.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4127086/">PMC free article</a>]</span> [<a href="/pubmed/24390345" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Sequence+variants+in+SLC16A11+are+a+common+risk+factor+for+type+2+diabetes+in+Mexico&amp;author=AL+Williams&amp;author=SB+Jacobs&amp;author=H+Moreno-Macias&amp;author=A+Huerta-Chagoya&amp;author=C+Churchhouse&amp;volume=506&amp;publication_year=2014&amp;pages=97-101&amp;pmid=24390345&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Manichaikul A, Mychaleckyj JC, Rich SS, Daly K, Sale M, Chen WM. Robust relationship inference in genome-wide association studies. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2867–2873.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025716/">PMC free article</a>]</span> [<a href="/pubmed/20926424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Robust+relationship+inference+in+genome-wide+association+studies&amp;author=A+Manichaikul&amp;author=JC+Mychaleckyj&amp;author=SS+Rich&amp;author=K+Daly&amp;author=M+Sale&amp;volume=26&amp;publication_year=2010&amp;pages=2867-2873&amp;pmid=20926424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Alexander DH, Novembre J, Lange K. Fast model-based estimation of ancestry in unrelated individuals. <span><span class="ref-journal">Genome research. </span>2009;<span class="ref-vol">19</span>:1655–1664.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2752134/">PMC free article</a>]</span> [<a href="/pubmed/19648217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+research&amp;title=Fast+model-based+estimation+of+ancestry+in+unrelated+individuals&amp;author=DH+Alexander&amp;author=J+Novembre&amp;author=K+Lange&amp;volume=19&amp;publication_year=2009&amp;pages=1655-1664&amp;pmid=19648217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Zhou X, Stephens M. Genome-wide efficient mixed-model analysis for association studies. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:821–824.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3386377/">PMC free article</a>]</span> [<a href="/pubmed/22706312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+efficient+mixed-model+analysis+for+association+studies&amp;author=X+Zhou&amp;author=M+Stephens&amp;volume=44&amp;publication_year=2012&amp;pages=821-824&amp;pmid=22706312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Wright SP. Adjusted P-values for Simultaneous Inference. <span><span class="ref-journal">Biometrics. </span>1992;<span class="ref-vol">48</span>:1005–1013.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Adjusted+P-values+for+Simultaneous+Inference&amp;author=SP+Wright&amp;volume=48&amp;publication_year=1992&amp;pages=1005-1013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Hancock DB, Levy JL, Gaddis NC, Bierut LJ, Saccone NL, Page GP, Johnson EO. Assessment of genotype imputation performance using 1000 Genomes in African American studies. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e50610.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3511547/">PMC free article</a>]</span> [<a href="/pubmed/23226329" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Assessment+of+genotype+imputation+performance+using+1000+Genomes+in+African+American+studies&amp;author=DB+Hancock&amp;author=JL+Levy&amp;author=NC+Gaddis&amp;author=LJ+Bierut&amp;author=NL+Saccone&amp;volume=7&amp;publication_year=2012&amp;pages=e50610&amp;pmid=23226329&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nature Genetics. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B+Howie&amp;author=C+Fuchsberger&amp;author=M+Stephens&amp;author=J+Marchini&amp;author=GR+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Zheng X, Shen J, Cox C, Wakefield JC, Ehm MG, Nelson MR, Weir BS. HIBAG-HLA genotype imputation with attribute bagging. <span><span class="ref-journal">The pharmacogenomics journal. </span>2014;<span class="ref-vol">14</span>:192–200.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772955/">PMC free article</a>]</span> [<a href="/pubmed/23712092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+pharmacogenomics+journal&amp;title=HIBAG-HLA+genotype+imputation+with+attribute+bagging&amp;author=X+Zheng&amp;author=J+Shen&amp;author=C+Cox&amp;author=JC+Wakefield&amp;author=MG+Ehm&amp;volume=14&amp;publication_year=2014&amp;pages=192-200&amp;pmid=23712092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Brisbin A, Bryc K, Byrnes J, Zakharia F, Omberg L, Degenhardt J, Reynolds A, Ostrer H, Mezey JG, Bustamante CD. PCAdmix: principal components-based assignment of ancestry along each chromosome in individuals with admixed ancestry from two or more populations. <span><span class="ref-journal">Human biology. </span>2012;<span class="ref-vol">84</span>:343–364.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3740525/">PMC free article</a>]</span> [<a href="/pubmed/23249312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+biology&amp;title=PCAdmix:+principal+components-based+assignment+of+ancestry+along+each+chromosome+in+individuals+with+admixed+ancestry+from+two+or+more+populations&amp;author=A+Brisbin&amp;author=K+Bryc&amp;author=J+Byrnes&amp;author=F+Zakharia&amp;author=L+Omberg&amp;volume=84&amp;publication_year=2012&amp;pages=343-364&amp;pmid=23249312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">The International HapMap Project. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">426</span>:789–796.</span> [<a href="/pubmed/14685227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+International+HapMap+Project&amp;volume=426&amp;publication_year=2003&amp;pages=789-796&amp;pmid=14685227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Altshuler DM, Gibbs RA, Peltonen L, Dermitzakis E, Schaffner SF, Yu F, Bonnen PE, de Bakker PI, Deloukas P, Gabriel SB, et al.  Integrating common and rare genetic variation in diverse human populations. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:52–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173859/">PMC free article</a>]</span> [<a href="/pubmed/20811451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrating+common+and+rare+genetic+variation+in+diverse+human+populations&amp;author=DM+Altshuler&amp;author=RA+Gibbs&amp;author=L+Peltonen&amp;author=E+Dermitzakis&amp;author=SF+Schaffner&amp;volume=467&amp;publication_year=2010&amp;pages=52-58&amp;pmid=20811451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Browning SR, Browning BL. Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:1084–1097.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265661/">PMC free article</a>]</span> [<a href="/pubmed/17924348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Rapid+and+accurate+haplotype+phasing+and+missing-data+inference+for+whole-genome+association+studies+by+use+of+localized+haplotype+clustering&amp;author=SR+Browning&amp;author=BL+Browning&amp;volume=81&amp;publication_year=2007&amp;pages=1084-1097&amp;pmid=17924348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Kidd JM, Gravel S, Byrnes J, Moreno-Estrada A, Musharoff S, Bryc K, Degenhardt JD, Brisbin A, Sheth V, Chen R, et al.  Population genetic inference from personal genome data: impact of ancestry and admixture on human genomic variation. <span><span class="ref-journal">American Journal of Human Genetics. </span>2012;<span class="ref-vol">91</span>:660–671.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3484644/">PMC free article</a>]</span> [<a href="/pubmed/23040495" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Population+genetic+inference+from+personal+genome+data:+impact+of+ancestry+and+admixture+on+human+genomic+variation&amp;author=JM+Kidd&amp;author=S+Gravel&amp;author=J+Byrnes&amp;author=A+Moreno-Estrada&amp;author=S+Musharoff&amp;volume=91&amp;publication_year=2012&amp;pages=660-671&amp;pmid=23040495&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Sequence variants in SLC16A11 are a common risk factor for type 2 diabetes in Mexico. <span><span class="ref-journal">Nature </span>2013</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4127086/">PMC free article</a>]</span> [<a href="/pubmed/24390345" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Sequence+variants+in+SLC16A11+are+a+common+risk+factor+for+type+2+diabetes+in+Mexico&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Kim JH, Jung SH, Bae JS, Lee HS, Yim SH, Park SY, Bang SY, Hu HJ, Shin HD, Bae SC, et al.  Deletion variants of RABGAP1L, 10q21.3, and C4 are associated with the risk of systemic lupus erythematosus in Korean women. <span><span class="ref-journal">Arthritis and Rheumatism. </span>2013;<span class="ref-vol">65</span>:1055–1063.</span> [<a href="/pubmed/23335107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+Rheumatism&amp;title=Deletion+variants+of+RABGAP1L,+10q21.3,+and+C4+are+associated+with+the+risk+of+systemic+lupus+erythematosus+in+Korean+women&amp;author=JH+Kim&amp;author=SH+Jung&amp;author=JS+Bae&amp;author=HS+Lee&amp;author=SH+Yim&amp;volume=65&amp;publication_year=2013&amp;pages=1055-1063&amp;pmid=23335107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Zhu JY, Pfuhl T, Motsch N, Barth S, Nicholls J, Grasser F, Meister G. Identification of novel Epstein-Barr virus microRNA genes from nasopharyngeal carcinomas. <span><span class="ref-journal">J Virol. </span>2009;<span class="ref-vol">83</span>:3333–3341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2655542/">PMC free article</a>]</span> [<a href="/pubmed/19144710" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Virol&amp;title=Identification+of+novel+Epstein-Barr+virus+microRNA+genes+from+nasopharyngeal+carcinomas&amp;author=JY+Zhu&amp;author=T+Pfuhl&amp;author=N+Motsch&amp;author=S+Barth&amp;author=J+Nicholls&amp;volume=83&amp;publication_year=2009&amp;pages=3333-3341&amp;pmid=19144710&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Stranger BE, Montgomery SB, Dimas AS, Parts L, Stegle O, Ingle CE, Sekowska M, Smith GD, Evans D, Gutierrez-Arcelus M, et al.  Patterns of cis regulatory variation in diverse human populations. <span><span class="ref-journal">PLoS genetics. </span>2012;<span class="ref-vol">8</span>:e1002639.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3330104/">PMC free article</a>]</span> [<a href="/pubmed/22532805" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=Patterns+of+cis+regulatory+variation+in+diverse+human+populations&amp;author=BE+Stranger&amp;author=SB+Montgomery&amp;author=AS+Dimas&amp;author=L+Parts&amp;author=O+Stegle&amp;volume=8&amp;publication_year=2012&amp;pages=e1002639&amp;pmid=22532805&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Fernando MMA, Goyettem P, Oksenberg J, Behrens T, Hauser SL, Rioux JD, Vyse TJ. Dissecting the MHC contribution to SLE. <span><span class="ref-journal">Ann Rheum Dis. </span>2008;<span class="ref-vol">67</span>:113.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Rheum+Dis&amp;title=Dissecting+the+MHC+contribution+to+SLE&amp;author=MMA+Fernando&amp;author=P+Goyettem&amp;author=J+Oksenberg&amp;author=T+Behrens&amp;author=SL+Hauser&amp;volume=67&amp;publication_year=2008&amp;pages=113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Zuniga J, Yu N, Barquera R, Alosco S, Ohashi M, Lebedeva T, Acuna-Alonzo V, Yunis M, Granados-Montiel J, Cruz-Lagunas A, et al.  HLA class I and class II conserved extended haplotypes and their fragments or blocks in Mexicans: implications for the study of genetic diversity in admixed populations. <span><span class="ref-journal">PLoS ONE. </span>2013;<span class="ref-vol">8</span>:e74442.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3781075/">PMC free article</a>]</span> [<a href="/pubmed/24086347" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=HLA+class+I+and+class+II+conserved+extended+haplotypes+and+their+fragments+or+blocks+in+Mexicans:+implications+for+the+study+of+genetic+diversity+in+admixed+populations&amp;author=J+Zuniga&amp;author=N+Yu&amp;author=R+Barquera&amp;author=S+Alosco&amp;author=M+Ohashi&amp;volume=8&amp;publication_year=2013&amp;pages=e74442&amp;pmid=24086347&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Crow YJ. Type I interferonopathies: a novel set of inborn errors of immunity. <span><span class="ref-journal">Annals of the New York Academy of Sciences. </span>2011;<span class="ref-vol">1238</span>:91–98.</span> [<a href="/pubmed/22129056" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+the+New+York+Academy+of+Sciences&amp;title=Type+I+interferonopathies:+a+novel+set+of+inborn+errors+of+immunity&amp;author=YJ+Crow&amp;volume=1238&amp;publication_year=2011&amp;pages=91-98&amp;pmid=22129056&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Deng Y, Zhao J, Sakurai D, Kaufman KM, Edberg JC, Kimberly RP, Kamen DL, Gilkeson GS, Jacob CO, Scofield RH, et al.  MicroRNA-3148 Modulates Allelic Expression of Toll-Like Receptor 7 Variant Associated with Systemic Lupus Erythematosus. <span><span class="ref-journal">PLoS genetics. </span>2013;<span class="ref-vol">9</span>:e1003336.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3585142/">PMC free article</a>]</span> [<a href="/pubmed/23468661" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=MicroRNA-3148+Modulates+Allelic+Expression+of+Toll-Like+Receptor+7+Variant+Associated+with+Systemic+Lupus+Erythematosus&amp;author=Y+Deng&amp;author=J+Zhao&amp;author=D+Sakurai&amp;author=KM+Kaufman&amp;author=JC+Edberg&amp;volume=9&amp;publication_year=2013&amp;pages=e1003336&amp;pmid=23468661&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Reddy MV, Velazquez-Cruz R, Baca V, Lima G, Granados J, Orozco L, Alarcon-Riquelme ME. Genetic association of IRF5 with SLE in Mexicans: higher frequency of the risk haplotype and its homozygozity than Europeans. <span><span class="ref-journal">Hum Genet. </span>2007;<span class="ref-vol">121</span>:721–727.</span> [<a href="/pubmed/17476532" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Genetic+association+of+IRF5+with+SLE+in+Mexicans:+higher+frequency+of+the+risk+haplotype+and+its+homozygozity+than+Europeans&amp;author=MV+Reddy&amp;author=R+Velazquez-Cruz&amp;author=V+Baca&amp;author=G+Lima&amp;author=J+Granados&amp;volume=121&amp;publication_year=2007&amp;pages=721-727&amp;pmid=17476532&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Graham RR, Kozyrev SV, Baechler EC, Reddy MV, Plenge RM, Bauer JW, Ortmann WA, Koeuth T, Escribano MF, et al.  Collaborative Groups, T.A. A common haplotype of interferon regulatory factor 5 (IRF5) regulates splicing and expression and is associated with increased risk of systemic lupus erythematosus. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:550–555.</span> [<a href="/pubmed/16642019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+common+haplotype+of+interferon+regulatory+factor+5+(IRF5)+regulates+splicing+and+expression+and+is+associated+with+increased+risk+of+systemic+lupus+erythematosus&amp;author=RR+Graham&amp;author=SV+Kozyrev&amp;author=EC+Baechler&amp;author=MV+Reddy&amp;author=RM+Plenge&amp;volume=38&amp;publication_year=2006&amp;pages=550-555&amp;pmid=16642019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Niewold TB, Kelly JA, Kariuki SN, Franek BS, Kumar AA, Kaufman KM, Thomas K, Walker D, Kamp S, Frost JM, et al.  IRF5 haplotypes demonstrate diverse serological associations which predict serum interferon alpha activity and explain the majority of the genetic association with systemic lupus erythematosus. <span><span class="ref-journal">Ann Rheum Dis. </span>2012;<span class="ref-vol">71</span>:463–468.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3307526/">PMC free article</a>]</span> [<a href="/pubmed/22088620" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Rheum+Dis&amp;title=IRF5+haplotypes+demonstrate+diverse+serological+associations+which+predict+serum+interferon+alpha+activity+and+explain+the+majority+of+the+genetic+association+with+systemic+lupus+erythematosus&amp;author=TB+Niewold&amp;author=JA+Kelly&amp;author=SN+Kariuki&amp;author=BS+Franek&amp;author=AA+Kumar&amp;volume=71&amp;publication_year=2012&amp;pages=463-468&amp;pmid=22088620&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Kottyan LC, Zoller EE, Bene J, Lu X, Kelly JA, Rupert AM, Lessard CJ, Vaughn SE, Marion M, Weirauch MT, et al.  The IRF5-TNPO3 association with systemic lupus erythematosus has two components that other autoimmune disorders variably share. <span><span class="ref-journal">Hum Mol Genet. </span>2015;<span class="ref-vol">24</span>:582–596.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4275071/">PMC free article</a>]</span> [<a href="/pubmed/25205108" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=The+IRF5-TNPO3+association+with+systemic+lupus+erythematosus+has+two+components+that+other+autoimmune+disorders+variably+share&amp;author=LC+Kottyan&amp;author=EE+Zoller&amp;author=J+Bene&amp;author=X+Lu&amp;author=JA+Kelly&amp;volume=24&amp;publication_year=2015&amp;pages=582-596&amp;pmid=25205108&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Zhao M, Wu X, Zhang Q, Luo S, Liang G, Su Y, Tan Y, Lu Q. RFX1 regulates CD70 and CD11a expression in lupus T cells by recruiting the histone methyltransferase SUV39H1. <span><span class="ref-journal">Arthritis Res Ther. </span>2010;<span class="ref-vol">12</span>:R227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3046540/">PMC free article</a>]</span> [<a href="/pubmed/21192791" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Res+Ther&amp;title=RFX1+regulates+CD70+and+CD11a+expression+in+lupus+T+cells+by+recruiting+the+histone+methyltransferase+SUV39H1&amp;author=M+Zhao&amp;author=X+Wu&amp;author=Q+Zhang&amp;author=S+Luo&amp;author=G+Liang&amp;volume=12&amp;publication_year=2010&amp;pages=R227&amp;pmid=21192791&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4829354&amp;issue-id=268022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140319461162608title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Methods</a></li><li><a href="#S15title">Results</a></li><li><a href="#S21title">Discussion</a></li><li><a href="#S22title">Supplementary Material</a></li><li><a href="#S23title">Acknowledgments</a></li><li><a href="#idm140319470652624title">Footnotes</a></li><li><a href="#idm140319500525856title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4829354/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4829354/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4829354/pdf/nihms774837.pdf">PDF (899K)</a></li> | <li><a href="#" data-citationid="PMC4829354" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4829354%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4829354%2F&amp;text=GWAS%20in%20an%20Amerindian%20ancestry%20population%20reveals%20novel%20systemic%20lupus%20erythematosus%20risk%20loci%20and%20the%20role%20of%20European%20admixture" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4829354%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26606652" data-aiid="4829354" data-aid="4829354" data-iid="268022" data-domainid="319" data-domain="nihpa" data-accid="PMC4829354" data-md5="ae436d8d086d80f2f8958774b5520f58"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:18:59-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D4A91DD58D1010000000002CD01B4&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4829354%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4829354/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>