<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4289640/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>The National Longitudinal Study of Adolescent to Adult Health (Add Health) Sibling Pairs Genome-Wide Data</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4289640/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Behavior genetics"><meta name="citation_title" content="The National Longitudinal Study of Adolescent to Adult Health (Add Health) Sibling Pairs Genome-Wide Data"><meta name="citation_authors" content="Matthew B. McQueen, Jason D. Boardman, Benjamin W. Domingue, Andrew Smolen, Joyce Tabor, Ley Killeya-Jones, Carolyn T. Halpern, Eric A. Whitsel, Kathleen MullanHarris"><meta name="citation_date" content="January 2015"><meta name="citation_issue" content="1"><meta name="citation_volume" content="45"><meta name="citation_firstpage" content="12"><meta name="citation_doi" content="10.1007/s10519-014-9692-4"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4289640/?report=abstract"><meta name="citation_pmid" content="25378290"><meta name="DC.Title" content="The National Longitudinal Study of Adolescent to Adult Health (Add Health) Sibling Pairs Genome-Wide Data"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Matthew B. McQueen"><meta name="DC.Contributor" content="Jason D. Boardman"><meta name="DC.Contributor" content="Benjamin W. Domingue"><meta name="DC.Contributor" content="Andrew Smolen"><meta name="DC.Contributor" content="Joyce Tabor"><meta name="DC.Contributor" content="Ley Killeya-Jones"><meta name="DC.Contributor" content="Carolyn T. Halpern"><meta name="DC.Contributor" content="Eric A. Whitsel"><meta name="DC.Contributor" content="Kathleen MullanHarris"><meta name="DC.Date" content="2015 Jan"><meta name="DC.Identifier" content="10.1007/s10519-014-9692-4"><meta name="DC.Language" content="en"><meta property="og:title" content="The National Longitudinal Study of Adolescent to Adult Health (Add Health) Sibling Pairs Genome-Wide Data"><meta property="og:type" content="article"><meta property="og:description" content="Here we provide a detailed description of the genome-wide information available on the National Longitudinal Study of Adolescent to Adult Health (Add Health) sibling pair subsample (Harris et al., 2012). A total of 2020 samples were genotyped (including ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4289640/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4289640/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4289640/pdf/nihms-641008.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8DEB6ADD57B90100000000013600BF.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4289640%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4289640</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Behav Genet</a></span>. Author manuscript; available in PMC 2016 Jan 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25378290" target="pmc_ext" ref="reftype=publisher&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Behav Genet. 2015 Jan; 45(1): 12–23. </span></a></div><span class="fm-vol-iss-date">Published online 2014 Nov 7. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1007%2Fs10519-014-9692-4" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s10519-014-9692-4</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4289640</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS641008</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25378290">25378290</a></div></div></div></div><h1 class="content-title">The National Longitudinal Study of Adolescent to Adult Health (Add Health) Sibling Pairs Genome-Wide Data</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=McQueen%20MB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290" class="affpopup" co-rid="_co_idm139780396346896" co-class="co-affbox">Matthew B. McQueen</a>,<sup>a,</sup><sup>*</sup> <a href="/pubmed/?term=Boardman%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290" class="affpopup" co-rid="_co_idm139780389963408" co-class="co-affbox">Jason D. Boardman</a>,<sup>b</sup> <a href="/pubmed/?term=Domingue%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290" class="affpopup" co-rid="_co_idm139780389945904" co-class="co-affbox">Benjamin W. Domingue</a>,<sup>b</sup> <a href="/pubmed/?term=Smolen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290" class="affpopup" co-rid="_co_idm139780394852672" co-class="co-affbox">Andrew Smolen</a>,<sup>c</sup> <a href="/pubmed/?term=Tabor%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290" class="affpopup" co-rid="_co_idm139780394850912" co-class="co-affbox">Joyce Tabor</a>,<sup>d</sup> <a href="/pubmed/?term=Killeya-Jones%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290" class="affpopup" co-rid="_co_idm139780395891488" co-class="co-affbox">Ley Killeya-Jones</a>,<sup>d</sup> <a href="/pubmed/?term=Halpern%20CT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290" class="affpopup" co-rid="_co_idm139780390232064" co-class="co-affbox">Carolyn T. Halpern</a>,<sup>d,</sup><sup>e</sup> <a href="/pubmed/?term=Whitsel%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290" class="affpopup" co-rid="_co_idm139780465061696" co-class="co-affbox">Eric A. Whitsel</a>,<sup>f,</sup><sup>g</sup> and  <a href="/pubmed/?term=MullanHarris%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290" class="affpopup" co-rid="_co_idm139780395971728" co-class="co-affbox">Kathleen MullanHarris</a><sup>d,</sup><sup>h,</sup><sup>i</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139780396346896"><h3 class="no_margin">Matthew B. McQueen</h3><p><sup>a</sup>Department of Integrative Physiology, University of Colorado Boulder</p><div>Find articles by <a href="/pubmed/?term=McQueen%20MB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290">Matthew B. McQueen</a></div></div><div id="_co_idm139780389963408"><h3 class="no_margin">Jason D. Boardman</h3><p><sup>b</sup>Institute of Behavioral Science, University of Colorado Boulder</p><div>Find articles by <a href="/pubmed/?term=Boardman%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290">Jason D. Boardman</a></div></div><div id="_co_idm139780389945904"><h3 class="no_margin">Benjamin W. Domingue</h3><p><sup>b</sup>Institute of Behavioral Science, University of Colorado Boulder</p><div>Find articles by <a href="/pubmed/?term=Domingue%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290">Benjamin W. Domingue</a></div></div><div id="_co_idm139780394852672"><h3 class="no_margin">Andrew Smolen</h3><p><sup>c</sup>Institute for Behavioral Genetics, University of Colorado Boulder</p><div>Find articles by <a href="/pubmed/?term=Smolen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290">Andrew Smolen</a></div></div><div id="_co_idm139780394850912"><h3 class="no_margin">Joyce Tabor</h3><p><sup>d</sup>Carolina Population Center, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Tabor%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290">Joyce Tabor</a></div></div><div id="_co_idm139780395891488"><h3 class="no_margin">Ley Killeya-Jones</h3><p><sup>d</sup>Carolina Population Center, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Killeya-Jones%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290">Ley Killeya-Jones</a></div></div><div id="_co_idm139780390232064"><h3 class="no_margin">Carolyn T. Halpern</h3><p><sup>d</sup>Carolina Population Center, University of North Carolina at Chapel Hill</p><p><sup>e</sup>Department of Maternal and Child Health, Gillings School of Global Public Health, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Halpern%20CT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290">Carolyn T. Halpern</a></div></div><div id="_co_idm139780465061696"><h3 class="no_margin">Eric A. Whitsel</h3><p><sup>f</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina at Chapel Hill </p><p><sup>g</sup>Department of Medicine, School of Medicine, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Whitsel%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290">Eric A. Whitsel</a></div></div><div id="_co_idm139780395971728"><h3 class="no_margin">Kathleen MullanHarris</h3><p><sup>d</sup>Carolina Population Center, University of North Carolina at Chapel Hill</p><p><sup>h</sup>Department of Sociology, University of North Carolina at Chapel Hill</p><p><sup>i</sup>Carolina Center for Genome Sciences, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=MullanHarris%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25378290">Kathleen MullanHarris</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139780389021376_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139780389021376_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139780389021376_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>a</sup>Department of Integrative Physiology, University of Colorado Boulder</div><div class="fm-affl" id="A2" lang="en"><sup>b</sup>Institute of Behavioral Science, University of Colorado Boulder</div><div class="fm-affl" id="A3" lang="en"><sup>c</sup>Institute for Behavioral Genetics, University of Colorado Boulder</div><div class="fm-affl" id="A4" lang="en"><sup>d</sup>Carolina Population Center, University of North Carolina at Chapel Hill</div><div class="fm-affl" id="A5" lang="en"><sup>e</sup>Department of Maternal and Child Health, Gillings School of Global Public Health, University of North Carolina at Chapel Hill</div><div class="fm-affl" id="A6" lang="en"><sup>f</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina at Chapel Hill </div><div class="fm-affl" id="A7" lang="en"><sup>g</sup>Department of Medicine, School of Medicine, University of North Carolina at Chapel Hill</div><div class="fm-affl" id="A8" lang="en"><sup>h</sup>Department of Sociology, University of North Carolina at Chapel Hill</div><div class="fm-affl" id="A9" lang="en"><sup>i</sup>Carolina Center for Genome Sciences, University of North Carolina at Chapel Hill</div><div id="CR1"><sup>*</sup>Address all correspondence to: Matthew B. McQueen, University of Colorado Boulder, Department of Integrative Physiology, 354 UCB; ph: 303-735-5158; fx: 303-492-4009. <a href="mailto:dev@null" data-email="ude.odaroloc@neeuqcm.ttam" class="oemail">ude.odaroloc@neeuqcm.ttam</a>.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139780389021376_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25378290" target="pmc_ext" ref="reftype=publisher&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Behav Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4289640/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>10519_2014_9692_MOESM1_ESM.</strong></div><div><a href="/pmc/articles/PMC4289640/bin/NIHMS641008-supplement-10519_2014_9692_MOESM1_ESM.docx" data-ga-action="click_feat_suppl">NIHMS641008-supplement-10519_2014_9692_MOESM1_ESM.docx</a><span style="color:gray"> (1.5M)</span></div><div class="small guid">GUID:&nbsp;4FA0DD26-1D98-4B05-8654-45F8A0783A2D</div></div></dd></dl></div><div id="idm139780432197504" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139780432197504title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Here we provide a detailed description of the genome-wide information available on the National Longitudinal Study of Adolescent to Adult Health (Add Health) sibling pair subsample (Harris et al., 2012). A total of 2020 samples were genotyped (including duplicates) arising from 1946 Add Health individuals from the sibling pairs subsample. After various steps for quality control (QC) and quality assurance (QA), we have high quality genome-wide data available on 1,888 individuals. In this report, we first highlight theQC and QA steps that were taken to prune the data of poorly performing samples and genetic markers. We further estimate the pairwise biological relationships using genome-wide data and compare those estimates to the assumed relationships in Add Health. Additionally, using genome-wide data from knownregional reference populations from Europe, West Africa, North and South America, Japan and China, weestimate the relative genetic ancestry of the respondents. Finally, rather than conducting a traditional cross-sectional genome-wide association study (GWAS) of body mass index (BMI), we opted to utilize the extensivepublicly available genome-wide information to conduct a weighted genome-wide association study (GWAS) of longitudinal BMI while accounting for both family and ethnic variation.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">National Longitudinal Study of Adolescent to Adult Health, Add Health, GWAS, BMI, Obesity, Sibling Pairs</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">The National Longitudinal Study of Adolescent to Adult Health (Add Health) is a nationally representative longitudinal study including over 20,000 adolescents originally sampled in Grades 7-12 in the United States between 1994 and 1995. Add Health respondents have been followed through adolescence and into early adulthood with four in-home interviews (1995, 1996, 2001-2002 and 2008-2009). The Add Health design included the oversampling of approximately 3,000 pairs of individuals who were raised in the same household. These pairs of individuals are biologically related to varying degrees including monozygotic (MZ) and dizygotic (DZ) twins, full siblings, half siblings and unrelated. For further details on the study design and sampling scheme for the Add Health Sibling Pairs Sample, including phenotypic, environmental and biological assessments, see <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821011">Harris et al. (2013)</a>. During the fourth in-home visit (Wave IV; 2008-2009), Add Health collected saliva on the entire sample of Add Health respondents (N=15,701), including the sibling pairs subsample. Consent rates (consent to provide saliva for DNA extraction) among the sibling pairs subsample for Wave IV saliva collection was an impressive 96%, which was similar to the consent rate for the entire Add Health sample. See <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821007">Harris et al. (2013)</a> for additional information on the Add Health Study design and genetic data.</p><p id="P3">Genome-wide association studies have largely been conducted using case-control and/or cross-sectional study designs primarily due to efficiency and ease of collection. The integration of genome-wide data into well-characterized longitudinal and prospective cohort studies that <em>include biological relationships</em> such as the Add Health sibling pair subsample has been much more limited. Notable exceptions include the Framingham Heart Study [NHLBI SNP-Health Association Resource (SHARe)] that follows multi-generational samples prospectively, and the Health and Retirement Study (HRS) that surveys a representative sample of individuals over the age of 50 every two years and follows them prospectively.Utilizing genetic data from longitudinal and prospective cohort studies has many potential advantages including refinement of phenotypic endpoints, phenotypic change and trajectory. Within the context of the ethnically diverse Add Health sibling pairs subsample of adolescents and young adults, there are additional advantages including family-based assessment and utilizing measured environmental and social factors collected over time.</p><p id="P4" class="p p-last">Here, we provide a description of the genome-wide data that were generated on the Add Health sibling pairs subsample. In particular, we focus on describing the targeted sample for genotyping, the quality control (QC) and quality assurance (QA) steps that were taken and how putative biological relationships were assessed. Using genome-wide data from known reference populations, we alsoshow the genetic ancestry of the Add Health sibling pairs subsample. We also explore the genetic heritability of body mass index (BMI) using the genome-wide data from the Add Health sibling pairs subsample. Finally, rather than conducting a traditional cross-sectional genome-wide association study (GWAS) of BMI, we opt to utilize the rich genome-wide information publicly available to conduct a weightedgenome-wide association study (GWAS) of longitudinal BMI while accounting for both family and ethnic variation. Funding for the genotyping of the sibling pairs subsample was provided by the National Institutes of Child Health and Human Development (R01 HD060726).</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials andMethods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Quality Control and Quality Assurance</h3><p id="P5" class="p p-first-last">The QA/QC Report for the Add Health Sibling Pairs Sample is provided in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Materials</a>. Briefly, there we describe how the sample was selected, prepared and genotyped, the number of markers removed, the number of samples removed, sex checks and duplicate concordance. The number of individual samples deemed of high quality for subsequent relationship testing, ancestry estimation and genome-wide analysis is N=1,888. The number of SNP markers (chromosomes 1-22 and X) with a genotyping call rate of at least 95% is N=940,862.</p></div><div id="S4" class="sec"><h3 id="S4title">Computer Software</h3><p id="P6" class="p p-first-last">For biological relationship testing, PLINK (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820983">Purcell et al., 2007</a>) and Kinship-based Inference for GWAS (KING; <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821002">Manichaikul et al., 2010</a>) were used. For genetic ancestry estimation, we used KING (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821012">Manichaikul et al., 2010</a>) and ADMIXTURE (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821008">Alexander et al., 2009</a>). R (R Core Team, 2013) was used for graphical display of ancestry information. For the estimation of heritability using genome-wide data, genome-wide complex trait analysis (GCTA; <a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820970">Yang et al., 2011</a>) was used. For the genome-wide association study (GWAS) we used SAS 9.3 (SAS Institute, Cary, NC, USA) and R. Once again, R was usedfor graphical display of the genome-wide association results.</p></div><div id="S5" class="sec"><h3 id="S5title">Estimation of Genetic Relatedness</h3><p id="P7" class="p p-first-last">Using information from chromosomes 1-22 (919,509 SNP markers) on the clean set of 1,888individual samples, we estimated Identity by State (IBS) and Identity by Descent (IBD) using PLINK (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820989">Purcell et al., 2007</a>) as well asthe Kinship Coefficient using KING (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820998">Manichaikul et al., 2010</a>). These measures are used to test duplicate concordance, confirm expected biological relationships, identify unknown or cryptic relatedness in the sample and provide the information necessary to assess genetic ancestry. The relationship measures are calculated pairwise for all individuals in the dataset. As generally recommended, we pruned autosomal SNPs to establish an approximately independent set of SNP markers to be used for IBS, IBD and Kinship Coefficient estimation. We used a linkage disequilibrium threshold (<em>r<sup>2</sup></em>) of 0.20 with a SNP window size of 50 and number of SNPs to shift window at each step of 5 <em>(PLINK command: --indep-pairwise 50 5 0.20</em>). After pruning, a set of 231,649 autosomal SNP markers in approximate linkage equilibrium was used to estimate the relationship measures. Pairwise mean IBD was estimated using PLINK (“PI_HAT”). However, PLINK’s estimates of IBD may be biased in stratified (multiethnic) samples (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821009">Manichaikul et al., 2010</a> and <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821001">Thorton et al., 2012</a>). Therefore, we relied upon the KING package to provide estimates of relationship (Kinship) that are robust to stratification.</p></div><div id="S6" class="sec"><h3 id="S6title">Estimation of Genetic Ancestry</h3><p id="P8" class="p p-first">We explored genetic ancestry in two different ways. Note that the sample of N=1,888 individuals with clean genotypes includes two MZ twin pairs. For the purposes of estimating genetic ancestry, we removed one individual (randomly) from each of the two MZ twin pairs resulting in a final analysis sample of N=1,886. For our first approach to estimating genetic ancestry, we used KING (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820996">Manichaikul et al., 2010</a>) to identify clusters of individuals based upon genetic similarity. KING uses multidimensional scaling (MDS) with Euclidean distance to generate principal coordinates (PCs) that can be used to identify population substructure. For the KING procedure, we used the same set of 231,649autosomal SNP markers in approximate linkage equilibrium that was used for the estimation of genetic relatedness.</p><p id="P9" class="p p-last">Second, we explored genetic ancestry using the software package, ADMIXTURE (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820974">Alexander et al., 2009</a>). ADMIXTURE uses an efficient likelihood model-based estimation of genetic ancestry using genome-wide data. For the ADMIXTURE procedure, we opted for a supervised analysis utilizing a series of known genetic ancestry populations as fixed groups to estimate the proportion of ancestry that individuals from the Add Health sibling pairs subsample share with each ancestral reference population. The ancestral populations used were derived from the Human Genome Diversity Project (HGDP; <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820977">Li et al., 2008</a>) and International Haplotype Map Project (HapMap; International HapMap 3 Consortium, 2010). Specifically, we utilized 108 samples from the HGDP to represent the Americas (Surui, Maya, Karitiana, Pima and Colombian), and 402 samples from HapMap to represent Europe (CEU), Africa (YRI), China (CHB) and Japan (JPT). In all, we identified 257,035 SNP markers that overlap across the Add Health sibling pairs subsample, the HGDP sample and the HapMap sample. For efficiency using the program ADMIXTURE, we created an autosomal SNP marker set that was in approximate linkage equilibrium (123,198 SNPs) to estimate ancestry.</p></div><div id="S7" class="sec"><h3 id="S7title">GCTA Heritability of BMI</h3><p id="P10" class="p p-first-last">We used the GCTA software (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820993">Yang et al., 2011</a>) to estimate heritability of body mass index (BMI) as measured in Add Health as part of the Wave 2, Wave III and Wave IV data collection. BMI was calculated using the standard formula of mass (kg) divided by height (m)squared (kg/m<sup>2</sup>) for each respondent. GCTA works by first estimating the genetic relatedness between all possible pairs of individuals. The genetic relatedness measures are known to be sensitive to population stratification, so for this application, we restricted the analysis to white respondents only. The subsequent step in the GCTA process is the estimation of a random effects model, where the random effects have a covariance structure based on the estimated genetic relatedness values. The percentage of total variance associated with the genetic random effects is considered the estimated heritability. For this particular study, we removed all pairwise relationship measures above 0.025.</p></div><div id="S8" class="sec"><h3 id="S8title">Genome-Wide Association Approach</h3><p id="P11" class="p p-first">To conduct SNP-by-SNP genome-wide association analysis of BMI, we started initially with919,509 autosomal markers with a genotyping call rate greater than 95%. Further steps involved removing SNP markers that show evidence of deviation from Hardy-Weinberg Equilibrium (HWE) in 492 unrelated, self-identified white individuals extracted from the entire sample. These 492 individuals were selected via a two-step process. First, we focused on the homogenous self-identified white sample followed by the random selection of one individual from each biological relationship pair. In all, 6,237 autosomal SNPs were flagged for potential deviation from HWE (p &lt; 0.001) and removed from the genome-wide association analysis. On the basis of minor allele frequency (MAF), we further removed SNP markers with an MAF &lt; 0.01 (32,313). Therefore, the final genome-wide association marker set includes 880,959 autosomal SNPs.As noted previously, the sample of N=1,888 individuals with clean genotypes includes two MZ twin pairs. For the purposes of the genome-wide association analysis, we removed one individual (randomly) from each of the two MZ twins pair resulting in a final analysis sample of N=1,886.</p><p id="P12" class="p p-last">To optimize statistical power, rather than conduct a traditional family-based association analysis on the related sets of individuals, we opted for a more flexible linear mixed effects model (Bates et al., 2014). This approach allowed us to model longitudinal measures of BMI (Waves II, III and IV) from all 1,886 individuals while accounting for biological relationships (if present) and within-individual variation in BMI as well as controlling for age, sex and MDS-derived components of ancestry. BMI measures from women who were pregnant were excluded from this analysis. Note that only 5 respondents did not have BMI measures across all three waves.</p></div><div id="S9" class="sec sec-last"><h3 id="S9title">Weighted Association</h3><p id="P13" class="p p-first">A major issue plaguing genome-wide studies is multiple testing that arises from testing hundreds of thousands (if not millions) of SNP markers for association with the disease or trait of interest. In response to this issue, many investigators have advocated the use of a Bonferroni-correction to limit the probability of committing type-I errors. However, this comes at a cost of simultaneously increasing the probability of committing type-II errors, thereby diminishing the opportunity of detecting true association signals. This is particularly true of smaller genome-wide association datasets such as the sibling pairs samples. One solution is to utilize prior information into the association scan. In this study, we use a weighted association approach as implemented by Roeder et al., 2006 to accomplish this. While there are a variety of ways to construct weights, there are only two criteria that must be met. First, each weight must be greater than 0 and the mean of the weights must be 1. There are numerous sources of prior information that can motivate the weighting scheme including linkage scans, bioinformatics information, as well as previously conducted (and independent) genome-wide association signals (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820988">Roeder et al., 2007</a>; <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820990">Roeder &amp; Wasserman, 2009</a>). Further, the prior information can be in the form of test statistics (i.e. LOD scores, Z scores) or p-values (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820999">Roeder et al., 2006</a>).</p><p id="P14">The weights for this study were derived from the GWAS on BMI as conducted by the Genetic Investigation of Anthropometric Traits (GIANT) consortium (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821010">Speliotes et al., 2010</a>). Details on the sample and the analysis procedures can be found elsewhere (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820991">Speliotes et al., 2010</a>). Briefly, the GIANT consortium conducting a GWAS on BMI using 249,796 individuals and made the association signals for each of the ~2.8M SNP markers available to the public. In particular, the p-values from the GWAS served as the prior information used to devise the weighting scheme for the genome-wide association scan from this study. In the original introduction to this approach, <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820986">Roeder et al. (2006)</a> introduced exponential and cumulative weighting procedures. We opted for a cumulated weighting scheme that can be less sensitive to large prior association signals and we also used a scaling factor (B) of 2 (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820972">Roeder et al., 2006</a>).</p><p id="P15" class="p p-last">We focused on markers that either overlapped between the GIANT consortium and this study or GIANT consortium markers that were in reasonable linkage disequilibrium (<em>r<sup>2</sup></em>&gt; 0.80)with SNP markers from this study. In all, we identified 717,411 markers to be tested for association that also have corresponding weights from the GIANT consortium. As a result, p-values from this study may be up-weighted or down-weighted depending upon the association signal from the GIANT GWAS. More specifically, the unweighted (nominal) p-values from this study are divided by the weights as assigned through the GIANT GWAS to generate the weighted p-values. The weighting procedure was conducted in R using the “weighted_FDR.R” script that can be found at <a href="http://www.wpic.pitt.edu/wpiccompgen/fdr/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wpic.pitt.edu/wpiccompgen/fdr/</a>.</p></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">Results</h2><div id="S11" class="sec sec-first"><h3 id="S11title">Biological Relationships</h3><p id="P16" class="p p-first-last">Table 1uses the Kinship Coefficient generated from KING to tabulate the observed pairwise relationship status based upon genome-wide data versus the expected pairwise relationships based upon information from Add Health (using the Add Health variable, “sibcl4”). The sibcl4 variable is one of the classification variables available for the Add Health sibling pairs subsample. This particular classification designates pairs of respondents into monozygotic twin pair (MZ), dizygotic twin pair (DZ), full sibling pair (FS), half-sibling pair (HS), cousin pair (CO), unrelated pair (UN) and undetermined relationship (UD). A total of 1,781,328 (<em><sub>1888</sub>C<sub>2</sub></em>) pairwise relationship comparisons were conducted. As recommended by the authors of KING (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820995">Manichaikul et al., 2010</a>), a Kinship Coefficient greater than 0.354 is categorized as an MZ twin pair (duplicates have been removed), between 0.177 and 0.354 as 1<sup>st</sup> degree relationship (DZ twin pairsand full sibling pairs - no parent-offspring are present), between 0.0884 and 0.177 as 2<sup>nd</sup> degree relationship (half-sibling pair and avuncular), between 0.0442 and 0.0884 as 3<sup>rd</sup> degree relationship (half-avuncular and first cousin) and less than 0.0442 as not related (NR).Note that the expected values for the Kinship Coefficient are 0.50, 0.25, 0.125, 0.0625 and 0.0 for MZ, DZ/FS, HS, CO and UN respectively and the boundaries suggested by the authors of KING are to account for the variability in the estimated Kinship based upon genome-wide data. As can be seen in <a href="/pmc/articles/PMC4289640/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>, the vast majority of expected relationships are consistent with the observed genetic relationships. However, there are notable discrepancies. For example, there are 33 expected full sibling pairs that are consistent with being half-sibling pairs according to the genetic data. Further, there are two pairs of MZ twins - one set of MZ twin pairs were thought to be a DZ twin pair while the other set were unknown prior to this study. Additionally, a pair of individuals who were thought to be an MZ twin pair is likely an DZ twin pair.As can also been seen in the table we have detected 266 pairs of individuals thought to be unrelated who are at least distally related.The majority of these pairs (264) are 3<sup>rd</sup> degree relationships (i.e., cousins). In total, there are 664 full sibling/DZ twin pairs (1328 individuals) that would be utilized for studies employing a traditional sibling pair family-based design. Based upon these relatedness measures, we have created a new variable available in the Add Health data sources called “sibcl<strong>g</strong>” that specifies the biological relationship based upon the genome-wide information as presented here.This variable will be made available to Add Health researchers through the Restricted-Use Data mechanism of Add Health and will be included with other variables related to the sibling pair data.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Observed versus Expected Relationship Status</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139780376896352" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="8" rowspan="1" valign="middle" align="center">
<strong>Observed Relationship (KING Kinship Coefficient)<sup>*</sup></strong>
</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">MZ</th><th rowspan="1" colspan="1" valign="middle" align="center">1<sup>st</sup> Degree</th><th rowspan="1" colspan="1" valign="middle" align="center">2<sup>nd</sup> Degree</th><th rowspan="1" colspan="1" valign="middle" align="center">3<sup>rd</sup> Degree</th><th rowspan="1" colspan="1" valign="middle" align="center">NR</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>TOTAL</em>
</th></tr></thead><tbody><tr><td rowspan="9" colspan="1" valign="middle" align="left">Expected Relationship<sup>*</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">MZ</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">DZ</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">173</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">177</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">FS</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">480</td><td rowspan="1" colspan="1" valign="middle" align="center">33</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">519</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">HS</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">38</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="center">51</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">AV</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">CO</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="center">18</td><td rowspan="1" colspan="1" valign="middle" align="center">38</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">NR</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">264</td><td rowspan="1" colspan="1" valign="middle" align="center">1,780,270</td><td rowspan="1" colspan="1" valign="middle" align="center">1,780,536</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">UD</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="right">
<em>TOTAL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">664</td><td rowspan="1" colspan="1" valign="middle" align="center">74</td><td rowspan="1" colspan="1" valign="middle" align="center">287</td><td rowspan="1" colspan="1" valign="middle" align="center">1,780,301</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>1,781,328</strong>
</td></tr></tbody></table></div><div id="largeobj_idm139780376896352" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4289640/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup><em>Note: MZ=monozygotic twin pair, DZ=dizygotic twin pair, FS=full sibling pair, HS=half-sibling pair, AV=avuncular pair, CO=cousin pair, NR=not related and UD=undetermined</em>.</div></div></div></div><div id="S12" class="sec"><h3 id="S12title">Self-Reported Ethnicity and Multidimensional Scaling (MDS)</h3><p id="P17" class="p p-first-last">We used the Add Health “ah_race” variable for self-report ethnicity. This variable includes five categories: White, Black, Native American, Asian and Hispanic. Add Health respondents who self-reported as Hispanic were included in the Hispanic category irrespective of whether they also self-reported as White, Black, etc. Of the 1,886 individuals included in this analysis, there are 917 who self-identify as White, 677 who self-identify as Black, 209 who self-identify as Hispanic, 73 who self-identify as Asian, 8 who self-identify as Native American; 2 individuals have unknown self-identified ethnicity (see <a href="/pmc/articles/PMC4289640/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>). <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental figure 3</a> shows the first 5 principal coordinate (PC) MDS estimates from KING, color-coded by self-identified ancestry. As can be seen in <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental figure 3A</a>, the first principal coordinate (PC1) distinguishes the European ancestry from African ancestry while the second principal coordinate (PC2) captures variation between European, Asian and to an extent, Hispanic ancestry. In <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental figure 3B</a> we see variation from Hispanic samples including a small set of self-identified Native American individuals. <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental figures 3C and 3D</a> further distinguish between and within self-identified ethnic groups.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">Self-Identified Race and Proportion of Genetic Ancestry<sup>*</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139780386916336" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="5" rowspan="1" valign="middle" align="center">Ancestral Population<sup>**</sup></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Self-Identified<br>Race<sup>***</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Europe</em>
<br>
<em>(CEU)</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Africa</em>
<br>
<em>(YRI)</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Americas</em>
<br>
<em>(AMR)</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>China</em>
<br>
<em>(CHB)</em>
</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>Japan</em>
<br>
<em>(JPT)</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left"><em>White</em><br>N=917 (48.6%)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.983</td><td rowspan="1" colspan="1" valign="middle" align="center">0.007</td><td rowspan="1" colspan="1" valign="middle" align="center">0.004</td><td rowspan="1" colspan="1" valign="middle" align="center">0.003</td><td rowspan="1" colspan="1" valign="middle" align="center">0.003</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"><em>Black</em><br>N=677 (35.9%)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.178</td><td rowspan="1" colspan="1" valign="middle" align="center">0.803</td><td rowspan="1" colspan="1" valign="middle" align="center">0.008</td><td rowspan="1" colspan="1" valign="middle" align="center">0.007</td><td rowspan="1" colspan="1" valign="middle" align="center">0.004</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"><em>Hispanic</em><br>N=209 (11.1%)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.612</td><td rowspan="1" colspan="1" valign="middle" align="center">0.105</td><td rowspan="1" colspan="1" valign="middle" align="center">0.251</td><td rowspan="1" colspan="1" valign="middle" align="center">0.017</td><td rowspan="1" colspan="1" valign="middle" align="center">0.015</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"><em>Asian</em> N=73 (3.9%)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.068</td><td rowspan="1" colspan="1" valign="middle" align="center">0.005</td><td rowspan="1" colspan="1" valign="middle" align="center">0.001</td><td rowspan="1" colspan="1" valign="middle" align="center">0.807</td><td rowspan="1" colspan="1" valign="middle" align="center">0.120</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"><em>Native American</em><br>N=8 (0.4%)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.338</td><td rowspan="1" colspan="1" valign="middle" align="center">0.280</td><td rowspan="1" colspan="1" valign="middle" align="center">0.332</td><td rowspan="1" colspan="1" valign="middle" align="center">0.021</td><td rowspan="1" colspan="1" valign="middle" align="center">0.029</td></tr></tbody></table></div><div id="largeobj_idm139780386916336" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4289640/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><sup>*</sup>Proportion of genetic ancestry as estimated by ADMIXTURE.</div><div id="TFN3"><sup>**</sup>Ancestral populations derived from the HGDP and HapMap reference populations.</div><div id="TFN4"><sup>***</sup>As defined by the Add Health Race ("ah_race") variable (2 respondents coded as unknown).</div></div></div></div><div id="S13" class="sec"><h3 id="S13title">Self-Reported Ethnicity and Genetic Ancestry</h3><p id="P18" class="p p-first-last"><a href="/pmc/articles/PMC4289640/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental figure 4</a> displays the proportion of ancestry shared with each of the reference populations of each individual from the Add Health sibling pairs sample. As can be seen in <a href="/pmc/articles/PMC4289640/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental figure 4A</a>, the vast majority of individuals who self-identify as White have predominately European ancestry (CEU). <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental figure 4B</a> illustrates the admixed ancestry typical of African Americans, self-identifying as Black. Self-identified Black individuals have a varying degree of African (YRI), European (CEU) and American (AMR) ancestry as can be seen in <a href="/pmc/articles/PMC4289640/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>. Likewise, self-identified Hispanic and Native American individuals (<a href="#SD1" rid="SD1" class=" supplementary-material">supplemental figures 4C and 4D</a>) display an admixed ancestry largely comprised of American (AMR), European (CEU), African (YRI) and to a lesser extent, Chinese (CHB) and Japanese (JPT) ancestry. Finally, <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental figure 4E</a> includes self-identified Asians. As can also be seen in <a href="/pmc/articles/PMC4289640/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a> the ancestry of this subgroup is primarily of Chinese origin (CHB) but with measurable admixture of Japanese (JPT) and European (CEU).</p></div><div id="S14" class="sec sec-last"><h3 id="S14title">GCTA Heritability and Weighted Genome-Wide Association Analysis of BMI</h3><p id="P19" class="p p-first">The GCTA heritability of Wave II, Wave III and Wave IV BMI based upon a sample of whiterespondents was estimated to be 0.82 (SE=0.081), 0.71 (SE=0.091), and0.67 (SE=0.084) respectively. Using a bivariate approach, the GCTA genetic correlation between Waves II/III, Waves II/IV and Waves III/IV were estimated to be 0.95 (SE=0.031), 0.85 (SE=0.042) and 0.99 (SE=0.033) respectively. The degree of relationship among the 1,886 individuals used in the genome-wide association analysis varies. In all, there are 614 individuals who are not 1<sup>st</sup> degree relatives (siblings) of others in the sample (N=614), 609 sibling pairs (N=1,218 individuals) and 18 sibling trios (N=54). <a href="/pmc/articles/PMC4289640/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a> and <a href="/pmc/articles/PMC4289640/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3</span></span><span>3</span></a> provide the characteristics of the sample used for the genome-wide analysis. As noted previously and seen in <a href="/pmc/articles/PMC4289640/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>, approximately half of the sample self-identifies as White, 36% Black, 11% Hispanic, 4% Asian and less than 1% as Native American. <a href="/pmc/articles/PMC4289640/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> shows the frequency of males (48%) and females (52%) as well as the mean age and BMI of the sample at each wave of collection. Consistent with other studies, the mean (and standard deviation) of BMI increases throughout young adulthood.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p6">Characteristics of the Genome-Wide Association Sample</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139780396109920" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Characteristic</th><th rowspan="1" colspan="1" valign="middle" align="center"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>Biological Sex</em></strong>
<sup>*</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
 
 
<em>Male</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">905 (48%)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
 
 
<em>Female</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">981 (52%)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>Age</em></strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
 
 
<em>Wave II (N=1761)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">16.4 (1.7)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
 
 
<em>Wave III (N=1634)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">22.4 (1.7)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
 
 
<em>Wave IV (N=1886)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">28.9 (1.7)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>Body Mass Index (BMI)</em></strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
 
 
<em>Wave II (N=1688)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">23.4 (5.1)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
 
 
<em>Wave III (N=1562)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">26.6 (6.4)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
 
 
<em>Wave IV (N=1859)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">29.4 (7.9)</td></tr></tbody></table></div><div id="largeobj_idm139780396109920" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4289640/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><sup>*</sup>Values expressed as sample size (relative percentage).</div><div id="TFN6"><sup>**</sup>N refers to the number of non-missing values for each wave and variable. Values expressed as mean (standard deviation).</div></div></div><p id="P20" class="p p-last">To assess for the presence of systematic biases in genome-wide analyses, we generated a quantile-quantile(Q-Q) plot of the unweighted p-values (<a href="#SD1" rid="SD1" class=" supplementary-material">supplemental figure 5</a>). As can be seen in the Q-Q plot, there is no evidence of widespread bias that is generating the associations.To illustrate the distribution of the weights as derived from the GIANT consortium, we provide asimple histogram (<a href="/pmc/articles/PMC4289640/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>figure 1</span></a>). As can be seen in <a href="/pmc/articles/PMC4289640/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>figure 1</span></a>, the vast majority of the p-values genome-wide are effectively down-weighted (&lt;1; gray bar, N=522,093) while a fair number of p-values are substantially up-weighted (&gt;1; colored bars, N=195,318). These weights were applied to each of the nominal p-values generated via the linear mixed effect model of longitudinal BMI.The resulting −log<sub>10</sub> weighted p-values are displayed using a traditional Manhattan Plot (<a href="/pmc/articles/PMC4289640/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2</span></a>). The red horizontal line on <a href="/pmc/articles/PMC4289640/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2</span></a> represents a genome-wide <em>significant</em> threshold (<em>p</em>=5×10<sup>−8</sup>) while the blue horizontal line represents a threshold of <em>p</em>=5×10<sup>−5</sup>. <a href="/pmc/articles/PMC4289640/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a> displays the 39 SNP associations achieving<em>p</em>&lt; 5.0×10<sup>−5</sup> ordered by genomic location. We chose a threshold of <em>p</em>&lt; 5.0×10<sup>−5</sup> as a reasonable, albeit arbitrary, threshold for association signals that warrants potential follow-up as other studies have done (e.g. see <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820969">Carty et al., 2012</a>). In the table, we report the SNP marker name, chromosome, base pair location, and the nearest gene and where that SNP is located relative to that gene. We also report the allele conferring risk (increasing BMI units), the frequency of that allele and the other allele present in the data (reported as forward strand). Finally, we report the results of the analysis including the linear mixed model coefficient (corresponding 95% confidence interval), the unweighted p-value, the weight applied to that SNP marker and the corresponding weighted <em>p</em>-value. In this analysis, the linear mixed model coefficient may be interpreted as the additive effect of the risk allele on body mass index in units of kg/m<sup>2</sup>. For example, the C allele of SNP rs1421085 (FTO gene) is associated with a 0.27 kg/m<sup>2</sup> increase in BMI. Overall, the list of 39 SNPs include some marker pairs that are likely in high linkage disequilibrium (LD) from the same genic region. Additionally, the influence of the weighting scheme can clearly be seen in the table. Using weights in this way allows for strongly significant markers to sift to the top even when down-weighted. There are 6 SNP markers that were down-weighted, yet still achieved a genome-wide suggestive level. However, the remaining 30 SNP markers were up-weighted. Therefore, this is largely a list of SNPs that have been pushed towards the top of the association signals as they are SNPs with prior information indicating evidence of association with BMI (GIANT consortium) and achieved at least nominal significance in the Add Health sample. A notable signal includes the highly replicable FTO gene region (chromosome 16) providing evidence that the Add Health sibling pairs sample is an informative genetic dataset for future use.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4289640/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139780390742592" class="figure"><a href="/pmc/articles/PMC4289640/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4289640_nihms-641008-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-641008-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4289640/bin/nihms-641008-f0001.jpg"></a></div><div id="largeobj_idm139780390742592" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4289640/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4289640/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Histogram of the weights derived from the GIANT consortium applied to the genome-wide association p-values.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4289640/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139780394116960" class="figure"><a href="/pmc/articles/PMC4289640/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4289640_nihms-641008-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-641008-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4289640/bin/nihms-641008-f0002.jpg"></a></div><div id="largeobj_idm139780394116960" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4289640/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4289640/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Manhattan Plot of the Weighted GWAS Association Signals</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p7">Top weighted genome-wide association signals (<em>p &lt; 5×10<sup>5</sup></em>) ordered by genomic location.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139780386145712" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="4" rowspan="1" valign="bottom" align="left">Marker Characteristics<sup>*</sup></th><th colspan="3" rowspan="1" valign="bottom" align="center">Allele<br>Information<sup>**</sup></th><th colspan="4" rowspan="1" valign="bottom" align="center">Analysis Results<sup>***</sup></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="right">Chr</th><th rowspan="1" colspan="1" valign="bottom" align="right">Position</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene (loc)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Risk</th><th rowspan="1" colspan="1" valign="bottom" align="center">Oth</th><th rowspan="1" colspan="1" valign="bottom" align="center">Freq</th><th rowspan="1" colspan="1" valign="bottom" align="center">Coef (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P-Value</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">Wght</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P<sub>W</sub>-Value</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs199950</td><td rowspan="1" colspan="1" valign="bottom" align="right">1</td><td rowspan="1" colspan="1" valign="bottom" align="right">181590858</td><td rowspan="1" colspan="1" valign="bottom" align="left">CACNA1E (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.31</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.00 (0.58,1.42)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000003</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.65</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000004</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs199939</td><td rowspan="1" colspan="1" valign="bottom" align="right">1</td><td rowspan="1" colspan="1" valign="bottom" align="right">181595979</td><td rowspan="1" colspan="1" valign="bottom" align="left">CACNA1E (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.29</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.96(0.53,1.40)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000013</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.45</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000009</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs7415921</td><td rowspan="1" colspan="1" valign="bottom" align="right">1</td><td rowspan="1" colspan="1" valign="bottom" align="right">205910883</td><td rowspan="1" colspan="1" valign="bottom" align="left">SLC26A9 (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.49</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.77(0.37,1.16)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000135</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.86</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000015</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs12047830</td><td rowspan="1" colspan="1" valign="bottom" align="right">1</td><td rowspan="1" colspan="1" valign="bottom" align="right">205916699</td><td rowspan="1" colspan="1" valign="bottom" align="left">SLC26A9<br>FAM72A (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.45</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.77(0.38,1.15)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000103</td><td rowspan="1" colspan="1" valign="bottom" align="center">6.08</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000017</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs1358136</td><td rowspan="1" colspan="1" valign="bottom" align="right">2</td><td rowspan="1" colspan="1" valign="bottom" align="right">6183596</td><td rowspan="1" colspan="1" valign="bottom" align="left">LOC400940 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.54</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.76(1.19,0.34)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000436</td><td rowspan="1" colspan="1" valign="bottom" align="center">10.05</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000043</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs11686766</td><td rowspan="1" colspan="1" valign="bottom" align="right">2</td><td rowspan="1" colspan="1" valign="bottom" align="right">55385259</td><td rowspan="1" colspan="1" valign="bottom" align="left">RTN4 C2orf63 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.06</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.66(0.89,2.43)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000021</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.13</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000007</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs13032294</td><td rowspan="1" colspan="1" valign="bottom" align="right">2</td><td rowspan="1" colspan="1" valign="bottom" align="right">55404883</td><td rowspan="1" colspan="1" valign="bottom" align="left">C2orf63 (C-N)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.06</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.60 (0.83,2.37)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000050</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.34</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000021</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs33500</td><td rowspan="1" colspan="1" valign="bottom" align="right">3</td><td rowspan="1" colspan="1" valign="bottom" align="right">42427191</td><td rowspan="1" colspan="1" valign="bottom" align="left">LOC100287105<br>LYZL4 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.13</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.09(0.55,1.64)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000086</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.32</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000010</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs1587734</td><td rowspan="1" colspan="1" valign="bottom" align="right">4</td><td rowspan="1" colspan="1" valign="bottom" align="right">120809455</td><td rowspan="1" colspan="1" valign="bottom" align="left">LOC730456<br>MAD2L1 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.75</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.82(1.25,0.38)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000223</td><td rowspan="1" colspan="1" valign="bottom" align="center">7.22</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000031</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs1879685</td><td rowspan="1" colspan="1" valign="bottom" align="right">4</td><td rowspan="1" colspan="1" valign="bottom" align="right">142484038</td><td rowspan="1" colspan="1" valign="bottom" align="left">ZNF330<br>LOC100286983</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.22</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.96(0.49,1.42)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000054</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.45</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000037</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs58644</td><td rowspan="1" colspan="1" valign="bottom" align="right">5</td><td rowspan="1" colspan="1" valign="bottom" align="right">141791827</td><td rowspan="1" colspan="1" valign="bottom" align="left">SPRY4<br>FGF1 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.26</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.93(0.48,1.39)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000065</td><td rowspan="1" colspan="1" valign="bottom" align="center">4.96</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000013</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs652462</td><td rowspan="1" colspan="1" valign="bottom" align="right">6</td><td rowspan="1" colspan="1" valign="bottom" align="right">71024741</td><td rowspan="1" colspan="1" valign="bottom" align="left">COL9A1<br>FAM135A (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.40</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.80 (0.40,1.21)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000103</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.16</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000048</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs156192</td><td rowspan="1" colspan="1" valign="bottom" align="right">6</td><td rowspan="1" colspan="1" valign="bottom" align="right">104917557</td><td rowspan="1" colspan="1" valign="bottom" align="left">FLJ10088<br>HACE1 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.62</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.81(1.21,0.42)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000058</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.06</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000028</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs1468298</td><td rowspan="1" colspan="1" valign="bottom" align="right">7</td><td rowspan="1" colspan="1" valign="bottom" align="right">8800995</td><td rowspan="1" colspan="1" valign="bottom" align="left">NXPH1<br>PER4 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.13</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.14(0.60,1.68)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000036</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.73</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000013</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs4722037</td><td rowspan="1" colspan="1" valign="bottom" align="right">7</td><td rowspan="1" colspan="1" valign="bottom" align="right">21586595</td><td rowspan="1" colspan="1" valign="bottom" align="left">DNAH11 (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.62</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.96(1.40,0.52)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000017</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.71</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000006</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs634010</td><td rowspan="1" colspan="1" valign="bottom" align="right">7</td><td rowspan="1" colspan="1" valign="bottom" align="right">111698108</td><td rowspan="1" colspan="1" valign="bottom" align="left">DOCK4 (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.24</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.04(0.59,1.48)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000005</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.28</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000017</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs7013912</td><td rowspan="1" colspan="1" valign="bottom" align="right">8</td><td rowspan="1" colspan="1" valign="bottom" align="right">40200810</td><td rowspan="1" colspan="1" valign="bottom" align="left">C8orf4<br>ZMAT4 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.88</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.15(1.72,0.59)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000061</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.14</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000028</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs10090663</td><td rowspan="1" colspan="1" valign="bottom" align="right">8</td><td rowspan="1" colspan="1" valign="bottom" align="right">97979470</td><td rowspan="1" colspan="1" valign="bottom" align="left">PGCP (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.08</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.44(0.77,2.10)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000025</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.77</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000009</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs10976070</td><td rowspan="1" colspan="1" valign="bottom" align="right">9</td><td rowspan="1" colspan="1" valign="bottom" align="right">7150975</td><td rowspan="1" colspan="1" valign="bottom" align="left">KDM4C (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.51</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.84(1.22,0.46)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000017</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.89</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000019</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs6475914</td><td rowspan="1" colspan="1" valign="bottom" align="right">9</td><td rowspan="1" colspan="1" valign="bottom" align="right">26553400</td><td rowspan="1" colspan="1" valign="bottom" align="left">TUSC1<br>C9orf82 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.07</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.43(0.69,2.17)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000155</td><td rowspan="1" colspan="1" valign="bottom" align="center">6.32</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000025</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs7898695</td><td rowspan="1" colspan="1" valign="bottom" align="right">10</td><td rowspan="1" colspan="1" valign="bottom" align="right">3184507</td><td rowspan="1" colspan="1" valign="bottom" align="left">PITRM1 (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.57</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.72(1.09,0.35)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000149</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.96</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000038</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs11002893</td><td rowspan="1" colspan="1" valign="bottom" align="right">10</td><td rowspan="1" colspan="1" valign="bottom" align="right">81012071</td><td rowspan="1" colspan="1" valign="bottom" align="left">ZMIZ1 (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.81</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.86(1.32,0.40)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000236</td><td rowspan="1" colspan="1" valign="bottom" align="center">6.19</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000038</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs885567</td><td rowspan="1" colspan="1" valign="bottom" align="right">10</td><td rowspan="1" colspan="1" valign="bottom" align="right">115357179</td><td rowspan="1" colspan="1" valign="bottom" align="left">NRAP (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.73</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.92(1.33,0.51)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000013</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.50</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000026</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs686407</td><td rowspan="1" colspan="1" valign="bottom" align="right">12</td><td rowspan="1" colspan="1" valign="bottom" align="right">69744104</td><td rowspan="1" colspan="1" valign="bottom" align="left">LYZ (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.62</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.85(1.30,0.40)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000209</td><td rowspan="1" colspan="1" valign="bottom" align="center">4.53</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000046</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs3803155</td><td rowspan="1" colspan="1" valign="bottom" align="right">12</td><td rowspan="1" colspan="1" valign="bottom" align="right">125834395</td><td rowspan="1" colspan="1" valign="bottom" align="left">TMEM132B (C-N)</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.35</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.81(0.40,1.22)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000104</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.73</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000028</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs2210727</td><td rowspan="1" colspan="1" valign="bottom" align="right">13</td><td rowspan="1" colspan="1" valign="bottom" align="right">84492665</td><td rowspan="1" colspan="1" valign="bottom" align="left">SLITRK1<br>SLITRK6 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.52</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.81(1.20,0.41)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000061</td><td rowspan="1" colspan="1" valign="bottom" align="center">4.66</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000013</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs7327886</td><td rowspan="1" colspan="1" valign="bottom" align="right">13</td><td rowspan="1" colspan="1" valign="bottom" align="right">84518116</td><td rowspan="1" colspan="1" valign="bottom" align="left">SLITRK1<br>SLITRK6 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.53</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.74(1.14,0.33)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000356</td><td rowspan="1" colspan="1" valign="bottom" align="center">9.66</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000037</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs3752854</td><td rowspan="1" colspan="1" valign="bottom" align="right">15</td><td rowspan="1" colspan="1" valign="bottom" align="right">38158906</td><td rowspan="1" colspan="1" valign="bottom" align="left">MEIS2 TMCO5A (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.07</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.60 (0.86,2.34)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000021</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.34</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000015</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs2243716</td><td rowspan="1" colspan="1" valign="bottom" align="right">16</td><td rowspan="1" colspan="1" valign="bottom" align="right">10088149</td><td rowspan="1" colspan="1" valign="bottom" align="left">GRIN2A (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.15</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.14(0.62,1.65)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000014</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.55</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000025</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs1421085</td><td rowspan="1" colspan="1" valign="bottom" align="right">16</td><td rowspan="1" colspan="1" valign="bottom" align="right">53800954</td><td rowspan="1" colspan="1" valign="bottom" align="left">FTO (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.27</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.84(0.40,1.29)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000190</td><td rowspan="1" colspan="1" valign="bottom" align="center">11.83</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000016</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs12149832</td><td rowspan="1" colspan="1" valign="bottom" align="right">16</td><td rowspan="1" colspan="1" valign="bottom" align="right">53842908</td><td rowspan="1" colspan="1" valign="bottom" align="left">FTO (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.28</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.79(0.35,1.23)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000482</td><td rowspan="1" colspan="1" valign="bottom" align="center">11.83</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000041</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs16958392</td><td rowspan="1" colspan="1" valign="bottom" align="right">16</td><td rowspan="1" colspan="1" valign="bottom" align="right">82770272</td><td rowspan="1" colspan="1" valign="bottom" align="left">CDH13 (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.10</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.25(0.62,1.88)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000107</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.25</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000048</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs12926503</td><td rowspan="1" colspan="1" valign="bottom" align="right">16</td><td rowspan="1" colspan="1" valign="bottom" align="right">84031482</td><td rowspan="1" colspan="1" valign="bottom" align="left">NECAB2 (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.85</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.23(1.77,0.69)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000009</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.41</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000022</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs2269457</td><td rowspan="1" colspan="1" valign="bottom" align="right">17</td><td rowspan="1" colspan="1" valign="bottom" align="right">38254689</td><td rowspan="1" colspan="1" valign="bottom" align="left">NR1D1 (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.34</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.79(0.38,1.20)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000156</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.80</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000041</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs877128</td><td rowspan="1" colspan="1" valign="bottom" align="right">18</td><td rowspan="1" colspan="1" valign="bottom" align="right">13911628</td><td rowspan="1" colspan="1" valign="bottom" align="left">MC2R (I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.18</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.98(0.49,1.46)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000076</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.71</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000028</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs8105895<sup>P</sup></td><td rowspan="1" colspan="1" valign="bottom" align="right">19</td><td rowspan="1" colspan="1" valign="bottom" align="right">22215457</td><td rowspan="1" colspan="1" valign="bottom" align="left">ZNF208<br>ZNF257 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.14</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.26(0.73,1.79)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000003</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.86</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000002</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs17450788</td><td rowspan="1" colspan="1" valign="bottom" align="right">19</td><td rowspan="1" colspan="1" valign="bottom" align="right">22224755</td><td rowspan="1" colspan="1" valign="bottom" align="left">ZNF208<br>ZNF257 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">C</td><td rowspan="1" colspan="1" valign="bottom" align="center">T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.13</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.22(0.67,1.77)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000015</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.33</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000012</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs6016086</td><td rowspan="1" colspan="1" valign="bottom" align="right">20</td><td rowspan="1" colspan="1" valign="bottom" align="right">38142337</td><td rowspan="1" colspan="1" valign="bottom" align="left">LOC33956<br>MAFB (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.18</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.99(0.49,1.48)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000093</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.82</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000033</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs460976<sup>P</sup></td><td rowspan="1" colspan="1" valign="bottom" align="right">21</td><td rowspan="1" colspan="1" valign="bottom" align="right">42835494</td><td rowspan="1" colspan="1" valign="bottom" align="left">MX1 TMPRSS2 (IG)</td><td rowspan="1" colspan="1" valign="bottom" align="center">G</td><td rowspan="1" colspan="1" valign="bottom" align="center">A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.96</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.22(3.20,1.23)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000011</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.23</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.000046</td></tr></tbody></table></div><div id="largeobj_idm139780386145712" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4289640/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN7"><sup>*</sup>SNP markers denoted with a <sup>p</sup> were derived from proxy SNPs in high LD with SNPs from the GIANT consortium. Gene codes: I=intron, IG=intergenetic, C-N=Coding, nonsynonymous.</div><div id="TFN8"><sup>**</sup>Risk refers to the allele that corresponds to an increase in BMI units while Oth refers to the other (non-risk) allele. Freq corresponds to the allele frequency of the risk allele.</div><div id="TFN9"><sup>***</sup>Coeff (95% CI) is the beta coefficient and 95% confidence interval as estimated through a linear mixed effects model. The <em>P-Value</em> is the unweighted nominal p-value, the Wght is the GIANT-derived weights and <em>P<sub>w</sub>-value</em> is the weighted p-value</div></div></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Discussion</h2><p id="P21" class="p p-first">The primary focus of this study was to introduce the Add Health sibling pairs subsample genome-wide association data and conduct initial analyses to demonstrate the scientific potential of the data as a resource to the Add Health community of researchers. Given the unconventional (among traditional genome-wide studies) Add Health pairs subsample, we adopted a relatively unconventional approach to carry out the genome-wide analysis. First, it is estimated that 96% of all genome-wide studies have been conducted on people of European descent (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820980">Bustamante et al., 2011</a>). The reasons and explanations for focusing so exclusively on samples of European descent range from convenience and efficiency (using existing cohort studies that focus on subjects of European descent) to minimizing sources of genetic heterogeneity (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820976">Pulit et al., 2010</a>; <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821005">Bustamante et al., 2011</a>). However, recently, there has been a series of studies that have empirically demonstrated a critical role of multiethnic studies in genome research of complex disease(<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820997">Pulit et al., 2010</a>; Masunuru et al., 2012; <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469258429">Carlson et al., 2013</a>; <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821004">Gong et al., 2013</a>; <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820985">Manichaikul et al., 2012</a>; <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820973">Manku et al., 2013</a>; <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821006">Marigorta et al., 2013</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820979">Sabater-Lleal et al., 2013</a>). Often, these multiethnic studies will conduct genome-wide analyses within a relatively homogenous European descent sample and simultaneously conduct a genome-wide analysis among a more genetically diverse sample such as African-Americans before combining the association signals using meta-analysis. An alternative approach, and one that was chosen for the present study, conducts the genome-wide analysis on the entire sample across multiple ethnic backgrounds. This approach has been successfully conducted in other studies of complex disease (for examples, see <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820971">Kurreeman et al., 2012</a> and <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821003">Xu et al., 2013</a>). However, rather than use the Add Health sample as a discovery sample (often requiring very large sample sizes) we adopted a weighting scheme based upon the GIANT consortium (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820987">Speliotes et al., 2010</a>) that is comprised of a series of European descent samples. Therefore, the approach taken for the present study is one that explores the extent to which the variants discovered in European descent GIANT consortium may also be of relevance to the multiethnic Add Health pairs subsample.</p><p id="P22">We note here that there are a multitude of valid and reasonable approaches that investigators may take when conducting a genetic study of a multiethnic, family-based sample with longitudinal measures of phenotype, behaviors and the environment. For example, Add Health researchers may be interested in imputing genotypes for purposes of combining association signals across multiple data sources genotyped on different platforms. Add Health researchers may also be interested in incorporating Add Health sampling weights and/or taking into account phenotypic clustering particularly when studying genetic risk factors within an environmental context. Furthermore, the family-based structure of the data would allow for more specific analyses using informative sibling pair family units through a variety of family-based association approaches (e.g. FBAT, <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820975">Laird et al., 2000</a>).</p><p id="P23">Through the weighted genome-wide association analysis, we observed association signals that align with previous and in some cases, established genetic variants associated with BMI. For example, we were able to identify the FTO region that has been previously identified and replicated (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820981">Frayling et al., 2007</a>). Additionally, we were able to identify variants that are upweighted through the GIANT consortium results, but do not achieve genome-wide significance in either GIANT or the Add Health sibling pairs subsample. These variants may be of particular interest for researchers who wish to explore GxE interactions in the Add Health sample to further explain the variability of the effect of these variants on BMI over time (age and development), behaviors and under particular environmental contexts.</p><p id="P24">The Add Health study is unique because of the explicit emphasis on properly characterizing the multilevel and multidimensional aspects of adolescents’ lives as they transition to adulthood. This design in conjunction with the related and unrelated pairs data (see <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820978">Harris et al., 2013</a>) has expanded the scope of the gene-environment interaction perspective to a multilevel perspective in which environmental influences are measured at the level of the state (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820982">Boardman 2009</a>), neighborhood (<a href="#R15" rid="R15" class=" bibr popnode">Cleveland et al., 2003</a>; Beaver et al., 2012), and schools (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469258428">Boardman et al., 2012</a>). Most importantly, the research design enables the measurement of factors such as social norms (Boardman et al., 2008) that are otherwise difficult to assess. The assessment of these contextual factors has been highlighted as a critical area for future research in gene-environment interplay (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_638290364">Spittel et al., 2013</a>) and the utilization of genome-wide data in conjunction with this social environmental backdrop may provide important insights in the etiology of complex morbidities such as obesity (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821000">Boone-Heinonen and Gordon-Larsen, 2012</a>).</p><p id="P25">As described elsewhere (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820984">Boardman et al., 2013</a>), the existing gene-environment interaction typology includes models in which genetic risk may be the most evident in the least risky, the most risky, or the typical environments. Depending on the anticipated GxE relationship and the specific phenotype, environments may either trigger or control genetic expression in a causal manner, or they may simply mask otherwise small genetic associations. Without a representation of the full range of environments, one may conclude that a specific polymorphism is either protective, risky, or not associated with a particular phenotype. Belksy and Pluess (2009) make a very strong case for the differential susceptibility hypothesis that argues that environmentally sensitive loci will be protective in the most enriching environments but deleterious in unhealthy environments. This cross-over association cannot be identified without a representative sample from the full continuum of environments that is, again, why the representativeness of the Add Health study is such an important resource in conjunction with the pairs data.</p><p id="P26" class="p">Finally, it is important to note that GWAS is but one use of genome wide data. For instance, the genome-wide relationship models discussed above (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420821013">Yang et al., 2010</a>) can be extended to incorporate these multilevel design features so that the contextual variation in the heritability of health behaviors can be examined using measured genetic similarity rather than assumed similarity from sibling-based models. Similarly, sibling fixed effects approaches can take advantage of the “random assignment” of risk alleles to examine siblings residing and socializing in similar environments compared to those in very different social contexts (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_420820992">Fletcher et al., 2011</a>). These methods provide unique and new possibilities to identify causal models and have thus far not been extended to the genome-wide level.</p><div id="S16" class="sec sec-last"><h3 id="S16title">Add Health Sibling Pairs Subsample Data Access</h3><p id="P27" class="p p-first-last">The genome-wide data and phenotype measures used in this study will be made available to the scientific community through the NIH database of Genotypes and Phenotypes (dbGaP) by January 2015. Researchers interested in using the Add Health sibling pairs subsample genome-wide data will be required to access genotype data through the dbGaPauthorized access system. Once genotype data are available through the dbGaP and access has been granted, researchers who request other phenotypic data not in dbGaP will be able to apply for a Genome-wide Data Restricted Access Agreement through Add Health beginning in 2015 (<a href="http://www.cpc.unc.edu/projects/addhealth" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cpc.unc.edu/projects/addhealth</a>).This process will allow approved investigators access to the entire Add Health sibling pairs subsample longitudinal data in addition to the genome-wide data.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>10519_2014_9692_MOESM1_ESM</h4><div class="sup-box half_rhythm" id="d36e630"><a href="/pmc/articles/PMC4289640/bin/NIHMS641008-supplement-10519_2014_9692_MOESM1_ESM.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.5M, docx)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="__p1">This research uses data from Add Health, a program project directed by Kathleen Mullan Harris and designed by J. Richard Udry, Peter S. Bearman, and Kathleen Mullan Harris at the University of North Carolina at Chapel Hill, and funded by grant P01-HD31921 from the Eunice Kennedy Shriver National Institute of Child Health and Human Development, with cooperative funding from 23 other federal agencies and foundations. Information on how to obtain the Add Health data files is available on the Add Health website (<a href="http://www.cpc.unc.edu/addhealth" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cpc.unc.edu/addhealth</a>). The genome-wide data generated for the Add Health sibling pairs subsample as well investigator effort for MBM, JDB, KMH, BD and AS was funded by grant R01-HD060726 from the Eunice Kennedy Shriver National Institute of Child Health and Human Development.</p></div></div><div id="idm139780375803728" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139780375803728title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Alexander DH, Novembre J, Lange K. Fast model-based estimation of ancestry in unrelated individuals. <span><span class="ref-journal">Genome Res. </span>2009;<span class="ref-vol">19</span>(9):1655–1664.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2752134/">PMC free article</a>]</span> [<a href="/pubmed/19648217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Fast+model-based+estimation+of+ancestry+in+unrelated+individuals&amp;author=DH+Alexander&amp;author=J+Novembre&amp;author=K+Lange&amp;volume=19&amp;issue=9&amp;publication_year=2009&amp;pages=1655-1664&amp;pmid=19648217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Alexander DH, Lange K. Enhancements to the ADMIXTURE algorithm for individual ancestry estimation. <span><span class="ref-journal">BMC Bioinformatics. </span>2011;<span class="ref-vol">12</span>:246.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3146885/">PMC free article</a>]</span> [<a href="/pubmed/21682921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=Enhancements+to+the+ADMIXTURE+algorithm+for+individual+ancestry+estimation&amp;author=DH+Alexander&amp;author=K+Lange&amp;volume=12&amp;publication_year=2011&amp;pages=246&amp;pmid=21682921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Altshuler DM, Gibbs RA, Peltonen L, Altshuler DM, Gibbs RA, Peltonen L, Dermitzakis E, Schaffner SF, Yu F, Peltonen L, Dermitzakis E, Bonnen PE, Altshuler DM, Gibbs RA, de Bakker PI, Deloukas P, Gabriel SB, Gwilliam R, Hunt S, Inouye M, Jia X, Palotie A, Parkin M, Whittaker P, Yu F, Chang K, Hawes A, Lewis LR, Ren Y, Wheeler D, Gibbs RA, Muzny DM, Barnes C, Darvishi K, Hurles M, Korn JM, Kristiansson K, Lee C, McCarrol SA, Nemesh J, Dermitzakis E, Keinan A, Montgomery SB, Pollack S, Price AL, Soranzo N, Bonnen PE, Gibbs RA, Gonzaga-Jauregui C, Keinan A, Price AL, Yu F, Anttila V, Brodeur W, Daly MJ, Leslie S, McVean G, Moutsianas L, Nguyen H, Schaffner SF, Zhang Q, Ghori MJ, McGinnis R, McLaren W, Pollack S, Price AL, Schaffner SF, Takeuchi F, Grossman SR, Shlyakhter I, Hostetter EB, Sabeti PC, Adebamowo CA, Foster MW, Gordon DR, Licinio J, Manca MC, Marshall PA, Matsuda I, Ngare D, Wang VO, Reddy D, Rotimi CN, Royal CD, Sharp RR, Zeng C, Brooks LD, McEwen JE. Integrating common and rare genetic variation in diverse human populations. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>(7311):52–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173859/">PMC free article</a>]</span> [<a href="/pubmed/20811451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrating+common+and+rare+genetic+variation+in+diverse+human+populations&amp;author=DM+Altshuler&amp;author=RA+Gibbs&amp;author=L+Peltonen&amp;author=DM+Altshuler&amp;author=RA+Gibbs&amp;volume=467&amp;issue=7311&amp;publication_year=2010&amp;pages=52-58&amp;pmid=20811451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Bates D, Maechler M, Bolker B. <span><span class="ref-journal">lme4: Linear mixed-effects models using S4 classes. R package version 0.999999-0.Computer software] </span>2012:e0lme4.</span> Retrieved from  <a href="http://CRAN.R-project.org/packag" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://CRAN.R-project.org/packag</a>. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=lme4:+Linear+mixed-effects+models+using+S4+classes.+R+package+version+0.999999-0.Computer+software]&amp;author=D+Bates&amp;author=M+Maechler&amp;author=B+Bolker&amp;publication_year=2012&amp;pages=e0lme4&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Beaver KM, Wright JP, Delisi M, Daigle LE, Swatt ML, Gibson CL. Evidence of a gene × environment interaction in the creation of victimization: results from a longitudinal sample of adolescents. <span><span class="ref-journal">Int J Offender Ther Comp Criminol. </span>2007;<span class="ref-vol">51</span>(6):620–645.</span> [<a href="/pubmed/17636204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Offender+Ther+Comp+Criminol&amp;title=Evidence+of+a+gene+×+environment+interaction+in+the+creation+of+victimization:+results+from+a+longitudinal+sample+of+adolescents&amp;author=KM+Beaver&amp;author=JP+Wright&amp;author=M+Delisi&amp;author=LE+Daigle&amp;author=ML+Swatt&amp;volume=51&amp;issue=6&amp;publication_year=2007&amp;pages=620-645&amp;pmid=17636204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Belsky J, Pluess M. Beyond diathesis stress: differential susceptibility to environmental influences. <span><span class="ref-journal">Psychol Bull. </span>2009;<span class="ref-vol">135</span>(6):885–908.</span> [<a href="/pubmed/19883141" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychol+Bull&amp;title=Beyond+diathesis+stress:+differential+susceptibility+to+environmental+influences&amp;author=J+Belsky&amp;author=M+Pluess&amp;volume=135&amp;issue=6&amp;publication_year=2009&amp;pages=885-908&amp;pmid=19883141&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Boardman JD. State-level moderation of genetic tendencies to smoke. <span><span class="ref-journal">Am J Public Health. </span>2009;<span class="ref-vol">99</span>(3):480–486.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2661451/">PMC free article</a>]</span> [<a href="/pubmed/19150910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Public+Health&amp;title=State-level+moderation+of+genetic+tendencies+to+smoke&amp;author=JD+Boardman&amp;volume=99&amp;issue=3&amp;publication_year=2009&amp;pages=480-486&amp;pmid=19150910&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Boardman JD, Domingue BW, Fletcher JM. How social and genetic factors predict friendship networks. <span><span class="ref-journal">ProcNatlAcadSci U S A. </span>2012;<span class="ref-vol">109</span>(43):17377–17381.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3491494/">PMC free article</a>]</span> [<a href="/pubmed/23045663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ProcNatlAcadSci+U+S+A&amp;title=How+social+and+genetic+factors+predict+friendship+networks&amp;author=JD+Boardman&amp;author=BW+Domingue&amp;author=JM+Fletcher&amp;volume=109&amp;issue=43&amp;publication_year=2012&amp;pages=17377-17381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Boardman JD, Daw J, Freese J. Defining the environment in gene-environment research: lessons from social epidemiology. <span><span class="ref-journal">Am J Public Health. </span>2013;<span class="ref-vol">103</span>(Suppl 1):S64–S72.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3786759/">PMC free article</a>]</span> [<a href="/pubmed/23927514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Public+Health&amp;title=Defining+the+environment+in+gene-environment+research:+lessons+from+social+epidemiology&amp;author=JD+Boardman&amp;author=J+Daw&amp;author=J+Freese&amp;volume=103&amp;issue=Suppl+1&amp;publication_year=2013&amp;pages=S64-S72&amp;pmid=23927514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Boardman JD, Domingue BW, Blalock CL, Haberstick BC, Harris KM, McQueen MB. Is the Gene-Environment Interaction Paradigm Relevant to Genome-Wide Studies? The Case of Education and Body Mass Index. <span><span class="ref-journal">Demography. </span>2013</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4035460/">PMC free article</a>]</span> [<a href="/pubmed/24281739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Demography&amp;title=Is+the+Gene-Environment+Interaction+Paradigm+Relevant+to+Genome-Wide+Studies?+The+Case+of+Education+and+Body+Mass+Index&amp;author=JD+Boardman&amp;author=BW+Domingue&amp;author=CL+Blalock&amp;author=BC+Haberstick&amp;author=KM+Harris&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Boone-Heinonen J, Gordon-Larsen P. Obesogenic environments in youth: concepts and methods from a longitudinal national sample. <span><span class="ref-journal">Am J Prev Med. </span>2012;<span class="ref-vol">42</span>(5):e37–e46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3382037/">PMC free article</a>]</span> [<a href="/pubmed/22516502" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Prev+Med&amp;title=Obesogenic+environments+in+youth:+concepts+and+methods+from+a+longitudinal+national+sample&amp;author=J+Boone-Heinonen&amp;author=P+Gordon-Larsen&amp;volume=42&amp;issue=5&amp;publication_year=2012&amp;pages=e37-e46&amp;pmid=22516502&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Bustamante CD, Burchard EG, De la Vega FM. Genomics for the world. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">475</span>(7355):163–165.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3708540/">PMC free article</a>]</span> [<a href="/pubmed/21753830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genomics+for+the+world&amp;author=CD+Bustamante&amp;author=EG+Burchard&amp;author=FM+De+la+Vega&amp;volume=475&amp;issue=7355&amp;publication_year=2011&amp;pages=163-165&amp;pmid=21753830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Carlson CS, Matise TC, North KE, Haiman CA, Fesinmeyer MD, Buyske S, Schumacher FR, Peters U, Franceschini N, Ritchie MD, Duggan DJ, Spencer KL, Dumitrescu L, Eaton CB, Thomas F, Young A, Carty C, Heiss G, Le Marchand L, Crawford DC, Hindorff LA, Kooperberg CL, PAGE Consortium  Generalization and dilution of association results from European GWAS in populations of non-European ancestry: the PAGE study. <span><span class="ref-journal">PLoSBiol. </span>2013;<span class="ref-vol">11</span>(9):e1001661.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3775722/">PMC free article</a>]</span> [<a href="/pubmed/24068893" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoSBiol&amp;title=Generalization+and+dilution+of+association+results+from+European+GWAS+in+populations+of+non-European+ancestry:+the+PAGE+study&amp;author=CS+Carlson&amp;author=TC+Matise&amp;author=KE+North&amp;author=CA+Haiman&amp;author=MD+Fesinmeyer&amp;volume=11&amp;issue=9&amp;publication_year=2013&amp;pages=e1001661&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Carty CL, Johnson NA, Hutter CM, Reiner AP, Peters U, Tang H, Kooperberg C. Genome-wide association study of body height in African Americans: the Women’s Health Initiative SNP Health Association Resource (SHARe) <span><span class="ref-journal">Hum Mol Genet. </span>2012;<span class="ref-vol">21</span>(3):711–720.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3259012/">PMC free article</a>]</span> [<a href="/pubmed/22021425" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+study+of+body+height+in+African+Americans:+the+Women’s+Health+Initiative+SNP+Health+Association+Resource+(SHARe)&amp;author=CL+Carty&amp;author=NA+Johnson&amp;author=CM+Hutter&amp;author=AP+Reiner&amp;author=U+Peters&amp;volume=21&amp;issue=3&amp;publication_year=2012&amp;pages=711-720&amp;pmid=22021425&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Cleveland HH. Disadvantaged neighborhoods and adolescent aggression: Behavioral genetic evidence of contextual effects. <span><span class="ref-journal">Journal of Research on Adolescence. </span>2003;<span class="ref-vol">13</span>(2):211–238.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Research+on+Adolescence&amp;title=Disadvantaged+neighborhoods+and+adolescent+aggression:+Behavioral+genetic+evidence+of+contextual+effects&amp;author=HH+Cleveland&amp;volume=13&amp;issue=2&amp;publication_year=2003&amp;pages=211-238&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Fletcher JM, Lehrer SF. Genetic lotteries within families. <span><span class="ref-journal">J Health Econ. </span>2011;<span class="ref-vol">30</span>(4):647–659.</span> [<a href="/pubmed/21664708" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Health+Econ&amp;title=Genetic+lotteries+within+families&amp;author=JM+Fletcher&amp;author=SF+Lehrer&amp;volume=30&amp;issue=4&amp;publication_year=2011&amp;pages=647-659&amp;pmid=21664708&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Frayling TM, Timpson NJ, Weedon MN, Zeggini E, Freathy RM, Lindgren CM, Perry JR, Elliott KS, Lango H, Rayner NW, Shields B, Harries LW, Barrett JC, Ellard S, Groves CJ, Knight B, Patch AM, Ness AR, Ebrahim S, Lawlor DA, Ring SM, Ben-Shlomo Y, Jarvelin MR, Sovio U, Bennett AJ, Melzer D, Ferrucci L, Loos RJ, Barroso I, Wareham NJ, Karpe F, Owen KR, Cardon LR, Walker M, Hitman GA, Palmer CN, Doney AS, Morris AD, Smith GD, Hattersley AT, McCarthy MI. A common variant in the FTO gene is associated with body mass index and predisposes to childhood and adult obesity. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>(5826):889–894.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2646098/">PMC free article</a>]</span> [<a href="/pubmed/17434869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+variant+in+the+FTO+gene+is+associated+with+body+mass+index+and+predisposes+to+childhood+and+adult+obesity&amp;author=TM+Frayling&amp;author=NJ+Timpson&amp;author=MN+Weedon&amp;author=E+Zeggini&amp;author=RM+Freathy&amp;volume=316&amp;issue=5826&amp;publication_year=2007&amp;pages=889-894&amp;pmid=17434869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Galanter JM, Gignoux CR, Torgerson DG, Roth LA, Eng C, Oh SS, Nguyen EA, Drake KA, Huntsman S, Hu D, Sen S, Davis A, Farber HJ, Avila PC, Brigino-Buenaventura E, Lenoir MA, Meade K, Serebrisky D, Borrell LN, Rodríguez-Cintrón W, Estrada AM, Mendoza KS, Winkler CA, Klitz W, Romieu I, London SJ, Gilliland F, Martinez F, Bustamante C, Williams LK, Kumar R, Rodríguez-Santana JR, Burchard EG. Genome-wide association study and admixture mapping identify different asthma-associated loci in Latinos: The Genes-environments &amp; Admixture in Latino Americans study. <span><span class="ref-journal">J Allergy ClinImmunol. </span>2014</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4085159/">PMC free article</a>]</span> [<a href="/pubmed/24406073" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+ClinImmunol&amp;title=Genome-wide+association+study+and+admixture+mapping+identify+different+asthma-associated+loci+in+Latinos:+The+Genes-environments+&amp;+Admixture+in+Latino+Americans+study&amp;author=JM+Galanter&amp;author=CR+Gignoux&amp;author=DG+Torgerson&amp;author=LA+Roth&amp;author=C+Eng&amp;publication_year=2014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Gong J, Schumacher F, Lim U, Hindorff LA, Haessler J, Buyske S, Carlson CS, Rosse S, Bůzková P, Fornage M, Gross M, Pankratz N, Pankow JS, Schreiner PJ, Cooper R, Ehret G, Gu CC, Houston D, Irvin MR, Jackson R, Kuller L, Henderson B, Cheng I, Wilkens L, Leppert M, Lewis CE, Li R, Nguyen KD, Goodloe R, Farber-Eger E, Boston J, Dilks HH, Ritchie MD, Fowke J, Pooler L, Graff M, Fernandez-Rhodes L, Cochrane B, Boerwinkle E, Kooperberg C, Matise TC, Le Marchand L, Crawford DC, Haiman CA, North KE, Peters U. Fine Mapping and Identification of BMI Loci in African Americans. <span><span class="ref-journal">Am J Hum Genet. </span>2013;<span class="ref-vol">93</span>(4):661–671.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3791273/">PMC free article</a>]</span> [<a href="/pubmed/24094743" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Fine+Mapping+and+Identification+of+BMI+Loci+in+African+Americans&amp;author=J+Gong&amp;author=F+Schumacher&amp;author=U+Lim&amp;author=LA+Hindorff&amp;author=J+Haessler&amp;volume=93&amp;issue=4&amp;publication_year=2013&amp;pages=661-671&amp;pmid=24094743&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Harris KM, Halpern CT, Haberstick BC, Smolen A. The National Longitudinal Study of Adolescent Health (Add Health) sibling pairs data. <span><span class="ref-journal">Twin Res Hum Genet. </span>2013;<span class="ref-vol">16</span>(1):391–398.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3574787/">PMC free article</a>]</span> [<a href="/pubmed/23231780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res+Hum+Genet&amp;title=The+National+Longitudinal+Study+of+Adolescent+Health+(Add+Health)+sibling+pairs+data&amp;author=KM+Harris&amp;author=CT+Halpern&amp;author=BC+Haberstick&amp;author=A+Smolen&amp;volume=16&amp;issue=1&amp;publication_year=2013&amp;pages=391-398&amp;pmid=23231780&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Johnson AD, Handsaker RE, Pulit SL, Nizzari MM, O’Donnell CJ, de Bakker PI. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. <span><span class="ref-journal">Bioinformatics. </span>2008;<span class="ref-vol">24</span>(24):2938–2939.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720775/">PMC free article</a>]</span> [<a href="/pubmed/18974171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNAP:+a+web-based+tool+for+identification+and+annotation+of+proxy+SNPs+using+HapMap&amp;author=AD+Johnson&amp;author=RE+Handsaker&amp;author=SL+Pulit&amp;author=MM+Nizzari&amp;author=CJ+O’Donnell&amp;volume=24&amp;issue=24&amp;publication_year=2008&amp;pages=2938-2939&amp;pmid=18974171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Kurreeman FA, Stahl EA, Okada Y, Liao K, Diogo D, Raychaudhuri S, Freudenberg J, Kochi Y, Patsopoulos NA, Gupta N, Sandor C, Bang SY, Lee HS, Padyukov L, Suzuki A, Siminovitch K, Worthington J, Gregersen PK, Hughes LB, Reynolds RJ, Bridges SL, Bae SC, Yamamoto K, Plenge RM. Use of a multiethnic approach to identify rheumatoid- arthritis-susceptibility loci, 1p36 and 17q12. <span><span class="ref-journal">Am J Hum Genet. </span>2012;<span class="ref-vol">90</span>(3):524–532.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3309197/">PMC free article</a>]</span> [<a href="/pubmed/22365150" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Use+of+a+multiethnic+approach+to+identify+rheumatoid-+arthritis-susceptibility+loci,+1p36+and+17q12&amp;author=FA+Kurreeman&amp;author=EA+Stahl&amp;author=Y+Okada&amp;author=K+Liao&amp;author=D+Diogo&amp;volume=90&amp;issue=3&amp;publication_year=2012&amp;pages=524-532&amp;pmid=22365150&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Laird NM, Horvath S, Xu X. Implementing a unified approach to family-based tests of association. <span><span class="ref-journal">Genet Epidemiol. </span>2000;<span class="ref-vol">19</span>(Suppl 1):S36–S42.</span> [<a href="/pubmed/11055368" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Implementing+a+unified+approach+to+family-based+tests+of+association&amp;author=NM+Laird&amp;author=S+Horvath&amp;author=X+Xu&amp;volume=19&amp;issue=Suppl+1&amp;publication_year=2000&amp;pages=S36-S42&amp;pmid=11055368&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Li JZ, Absher DM, Tang H, Southwick AM, Casto AM, Ramachandran S, Cann HM, Barsh GS, Feldman M, Cavalli-Sforza LL, Myers RM. Worldwide human relationships inferred from genome-wide patterns of variation. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">319</span>(5866):1100–1104.</span> [<a href="/pubmed/18292342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Worldwide+human+relationships+inferred+from+genome-wide+patterns+of+variation&amp;author=JZ+Li&amp;author=DM+Absher&amp;author=H+Tang&amp;author=AM+Southwick&amp;author=AM+Casto&amp;volume=319&amp;issue=5866&amp;publication_year=2008&amp;pages=1100-1104&amp;pmid=18292342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Manichaikul A, Mychaleckyj JC, Rich SS, Daly K, Sale M, Chen WM. Robust relationship inference in genome-wide association studies. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>(22):2867–2873.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025716/">PMC free article</a>]</span> [<a href="/pubmed/20926424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Robust+relationship+inference+in+genome-wide+association+studies&amp;author=A+Manichaikul&amp;author=JC+Mychaleckyj&amp;author=SS+Rich&amp;author=K+Daly&amp;author=M+Sale&amp;volume=26&amp;issue=22&amp;publication_year=2010&amp;pages=2867-2873&amp;pmid=20926424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Manichaikul A, Palmas W, Rodriguez CJ, Peralta CA, Divers J, Guo X, Chen WM, Wong Q, Williams K, Kerr KF, Taylor KD, Tsai MY, Goodarzi MO, Sale MM, Diez-Roux AV, Rich SS, Rotter JI, Mychaleckyj JC. Population structure of Hispanics in the United States: the multi-ethnic study of atherosclerosis. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>(4):e1002640.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3325201/">PMC free article</a>]</span> [<a href="/pubmed/22511882" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+of+Hispanics+in+the+United+States:+the+multi-ethnic+study+of+atherosclerosis&amp;author=A+Manichaikul&amp;author=W+Palmas&amp;author=CJ+Rodriguez&amp;author=CA+Peralta&amp;author=J+Divers&amp;volume=8&amp;issue=4&amp;publication_year=2012&amp;pages=e1002640&amp;pmid=22511882&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">Manku H, Langefeld CD, Guerra SG, Malik TH, Alarcon-Riquelme M, Anaya JM, Bae SC, Boackle SA, Brown EE, Criswell LA, Freedman BI, Gaffney PM, Gregersen PA, Guthridge JM, Han SH, Harley JB, Jacob CO, James JA, Kamen DL, Kaufman KM, Kelly JA, Martin J, Merrill JT, Moser KL, Niewold TB, Park SY, Pons-Estel BA, Sawalha AH, Scofield RH, Shen N, Stevens AM, Sun C, Gilkeson GS, Edberg JC, Kimberly RP, Nath SK, Tsao BP, Vyse TJ. Trans-ancestral studies fine map the SLE-susceptibility locus TNFSF4. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>(7):e1003554.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3715547/">PMC free article</a>]</span> [<a href="/pubmed/23874208" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Trans-ancestral+studies+fine+map+the+SLE-susceptibility+locus+TNFSF4&amp;author=H+Manku&amp;author=CD+Langefeld&amp;author=SG+Guerra&amp;author=TH+Malik&amp;author=M+Alarcon-Riquelme&amp;volume=9&amp;issue=7&amp;publication_year=2013&amp;pages=e1003554&amp;pmid=23874208&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Marigorta UM, Navarro A. High trans-ethnic replicability of GWAS results implies common causal variants. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>(6):e1003566.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3681663/">PMC free article</a>]</span> [<a href="/pubmed/23785302" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=High+trans-ethnic+replicability+of+GWAS+results+implies+common+causal+variants&amp;author=UM+Marigorta&amp;author=A+Navarro&amp;volume=9&amp;issue=6&amp;publication_year=2013&amp;pages=e1003566&amp;pmid=23785302&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Morrison J. Characterization and correction of error in genome-wide IBD estimation for samples with population structure. <span><span class="ref-journal">Genet Epidemiol. </span>2013;<span class="ref-vol">37</span>(6):635–641.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4001853/">PMC free article</a>]</span> [<a href="/pubmed/23740691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Characterization+and+correction+of+error+in+genome-wide+IBD+estimation+for+samples+with+population+structure&amp;author=J+Morrison&amp;volume=37&amp;issue=6&amp;publication_year=2013&amp;pages=635-641&amp;pmid=23740691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">Musunuru K, Romaine SP, Lettre G, Wilson JG, Volcik KA, Tsai MY, Taylor HA, Schreiner PJ, Rotter JI, Rich SS, Redline S, Psaty BM, Papanicolaou GJ, Ordovas JM, Liu K, Krauss RM, Glazer NL, Gabriel SB, Fornage M, Cupples LA, Buxbaum SG, Boerwinkle E, Ballantyne CM, Kathiresan S, Rader DJ. Multi-ethnic analysis of lipid-associated loci: the NHLBI CARe project. <span><span class="ref-journal">PLoS One. </span>2012;<span class="ref-vol">7</span>(5):e36473.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3357427/">PMC free article</a>]</span> [<a href="/pubmed/22629316" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Multi-ethnic+analysis+of+lipid-associated+loci:+the+NHLBI+CARe+project&amp;author=K+Musunuru&amp;author=SP+Romaine&amp;author=G+Lettre&amp;author=JG+Wilson&amp;author=KA+Volcik&amp;volume=7&amp;issue=5&amp;publication_year=2012&amp;pages=e36473&amp;pmid=22629316&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="element-citation">Nelson SC, Doheny KF, Laurie CC, Mirel DB. Is ‘forward’ the same as ‘plus’?...and other adventures in SNP allele nomenclature. <span><span class="ref-journal">Trends Genet. </span>2012;<span class="ref-vol">28</span>(8):361–363.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6099125/">PMC free article</a>]</span> [<a href="/pubmed/22658725" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=Is+‘forward’+the+same+as+‘plus’?...and+other+adventures+in+SNP+allele+nomenclature&amp;author=SC+Nelson&amp;author=KF+Doheny&amp;author=CC+Laurie&amp;author=DB+Mirel&amp;volume=28&amp;issue=8&amp;publication_year=2012&amp;pages=361-363&amp;pmid=22658725&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="element-citation">Pulit SL, Voight BF, de Bakker PI. Multiethnic genetic association studies improve power for locus discovery. <span><span class="ref-journal">PLoS One. </span>2010;<span class="ref-vol">5</span>(9):e12600.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935880/">PMC free article</a>]</span> [<a href="/pubmed/20838612" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Multiethnic+genetic+association+studies+improve+power+for+locus+discovery&amp;author=SL+Pulit&amp;author=BF+Voight&amp;author=PI+de+Bakker&amp;volume=5&amp;issue=9&amp;publication_year=2010&amp;pages=e12600&amp;pmid=20838612&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>(3):559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="element-citation">Roeder K, Bacanu SA, Wasserman L, Devlin B. Using linkage genome scans to improve power of association in genome scans. <span><span class="ref-journal">Am J Hum Genet. </span>2006;<span class="ref-vol">78</span>(2):243–252.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1380233/">PMC free article</a>]</span> [<a href="/pubmed/16400608" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Using+linkage+genome+scans+to+improve+power+of+association+in+genome+scans&amp;author=K+Roeder&amp;author=SA+Bacanu&amp;author=L+Wasserman&amp;author=B+Devlin&amp;volume=78&amp;issue=2&amp;publication_year=2006&amp;pages=243-252&amp;pmid=16400608&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="element-citation">Roeder K, Devlin B, Wasserman L. Improving power in genome-wide association studies: weights tip the scale. <span><span class="ref-journal">Genet Epidemiol. </span>2007;<span class="ref-vol">31</span>(7):741–747.</span> [<a href="/pubmed/17549760" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Improving+power+in+genome-wide+association+studies:+weights+tip+the+scale&amp;author=K+Roeder&amp;author=B+Devlin&amp;author=L+Wasserman&amp;volume=31&amp;issue=7&amp;publication_year=2007&amp;pages=741-747&amp;pmid=17549760&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="element-citation">Roeder K, Wasserman L. Genome-Wide Significance Levels and Weighted Hypothesis Testing. <span><span class="ref-journal">Stat Sci. </span>2009;<span class="ref-vol">24</span>(4):398–413.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2920568/">PMC free article</a>]</span> [<a href="/pubmed/20711421" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Sci&amp;title=Genome-Wide+Significance+Levels+and+Weighted+Hypothesis+Testing&amp;author=K+Roeder&amp;author=L+Wasserman&amp;volume=24&amp;issue=4&amp;publication_year=2009&amp;pages=398-413&amp;pmid=20711421&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="element-citation">Sabater-Lleal M, Huang J, Chasman D, Naitza S, Dehghan A, Johnson AD, Teumer A, Reiner AP, Folkersen L, Basu S, Rudnicka AR, Trompet S, Mälarstig A, Baumert J, Bis JC, Guo X, Hottenga JJ, Shin SY, Lopez LM, Lahti J, Tanaka T, Yanek LR, Oudot-Mellakh T, Wilson JF, Navarro P, Huffman JE, Zemunik T, Redline S, Mehra R, Pulanic D, Rudan I, Wright AF, Kolcic I, Polasek O, Wild SH, Campbell H, Curb JD, Wallace R, Liu S, Eaton CB, Becker DM, Becker LC, Bandinelli S, Räikkönen K, Widen E, Palotie A, Fornage M, Green D, Gross M, Davies G, Harris SE, Liewald DC, Starr JM, Williams FM, Grant PJ, Spector TD, Strawbridge RJ, Silveira A, Sennblad B, Rivadeneira F, Uitterlinden AG, Franco OH, Hofman A, van Dongen J, Willemsen G, Boomsma DI, Yao J, Swords Jenny N, Haritunians T, McKnight B, Lumley T, Taylor KD, Rotter JI, Psaty BM, Peters A, Gieger C, Illig T, Grotevendt A, Homuth G, Völzke H, Kocher T, Goel A, Franzosi MG, Seedorf U, Clarke R, Steri M, Tarasov KV, Sanna S, Schlessinger D, Stott DJ, Sattar N, Buckley BM, Rumley A, Lowe GD, McArdle WL, Chen MH, Tofler GH, Song J, Boerwinkle E, Folsom AR, Rose LM, Franco-Cereceda A, Teichert M, Ikram MA, Mosley TH, Bevan S, Dichgans M, Rothwell PM, Sudlow CL, Hopewell JC, Chambers JC, Saleheen D, Kooner JS, Danesh J, Nelson CP, Erdmann J, Reilly MP, Kathiresan S, Schunkert H, Morange PE, Ferrucci L, Eriksson JG, Jacobs D, Deary IJ, Soranzo N, Witteman JC, de Geus EJ, Tracy RP, Hayward C, Koenig W, Cucca F, Jukema JW, Eriksson P, Seshadri S, Markus HS, Watkins H, Samani NJ, Wallaschofski H, Smith NL, Tregouet D, Ridker PM, Tang W, Strachan DP, Hamsten A, O’Donnell CJ. Multiethnic meta-analysis of genome-wide association studies in &gt;100 000 subjects identifies 23 fibrinogen-associated Loci but no strong evidence of a causal association between circulating fibrinogen and cardiovascular disease. <span><span class="ref-journal">Circulation. </span>2013;<span class="ref-vol">128</span>(12):1310–1324.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3842025/">PMC free article</a>]</span> [<a href="/pubmed/23969696" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Multiethnic+meta-analysis+of+genome-wide+association+studies+in+>100+000+subjects+identifies+23+fibrinogen-associated+Loci+but+no+strong+evidence+of+a+causal+association+between+circulating+fibrinogen+and+cardiovascular+disease&amp;author=M+Sabater-Lleal&amp;author=J+Huang&amp;author=D+Chasman&amp;author=S+Naitza&amp;author=A+Dehghan&amp;volume=128&amp;issue=12&amp;publication_year=2013&amp;pages=1310-1324&amp;pmid=23969696&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="element-citation">Shi G, Rice TK, Gu CC, Rao DC. Application of three-level linear mixed-effects model incorporating gene-age interactions for association analysis of longitudinal family data. <span><span class="ref-journal">BMC Proc. </span>2009;<span class="ref-vol">3</span>(Suppl 7):S89.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2795992/">PMC free article</a>]</span> [<a href="/pubmed/20018085" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Proc&amp;title=Application+of+three-level+linear+mixed-effects+model+incorporating+gene-age+interactions+for+association+analysis+of+longitudinal+family+data&amp;author=G+Shi&amp;author=TK+Rice&amp;author=CC+Gu&amp;author=DC+Rao&amp;volume=3&amp;issue=Suppl+7&amp;publication_year=2009&amp;pages=S89&amp;pmid=20018085&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R39"><span class="element-citation">Speliotes EK, Willer CJ, Berndt SI, Monda KL, Thorleifsson G, Jackson AU, Lango Allen H, Lindgren CM, Luan J, Mägi R, Randall JC, Vedantam S, Winkler TW, Qi L, Workalemahu T, Heid IM, Steinthorsdottir V, Stringham HM, Weedon MN, Wheeler E, Wood AR, Ferreira T, Weyant RJ, Segrè AV, Estrada K, Liang L, Nemesh J, Park JH, Gustafsson S, Kilpeläinen TO, Yang J, Bouatia-Naji N, Esko T, Feitosa MF, Kutalik Z, Mangino M, Raychaudhuri S, Scherag A, Smith AV, Welch R, Zhao JH, Aben KK, Absher DM, Amin N, Dixon AL, Fisher E, Glazer NL, Goddard ME, Heard-Costa NL, Hoesel V, Hottenga JJ, Johansson A, Johnson T, Ketkar S, Lamina C, Li S, Moffatt MF, Myers RH, Narisu N, Perry JR, Peters MJ, Preuss M, Ripatti S, Rivadeneira F, Sandholt C, Scott LJ, Timpson NJ, Tyrer JP, van Wingerden S, Watanabe RM, White CC, Wiklund F, Barlassina C, Chasman DI, Cooper MN, Jansson JO, Lawrence RW, Pellikka N, Prokopenko I, Shi J, Thiering E, Alavere H, Alibrandi MT, Almgren P, Arnold AM, Aspelund T, Atwood LD, Balkau B, Balmforth AJ, Bennett AJ, Ben-Shlomo Y, Bergman RN, Bergmann S, Biebermann H, Blakemore AI, Boes T, Bonnycastle LL, Bornstein SR, Brown MJ, Buchanan TA, Busonero F, Campbell H, Cappuccio FP, Cavalcanti-Proença C, Chen YD, Chen CM, Chines PS, Clarke R, Coin L, Connell J, Day IN, den Heijer M, Duan J, Ebrahim S, Elliott P, Elosua R, Eiriksdottir G, Erdos MR, Eriksson JG, Facheris MF, Felix SB, Fischer-Posovszky P, Folsom AR, Friedrich N, Freimer NB, Fu M, Gaget S, Gejman PV, Geus EJ, Gieger C, Gjesing AP, Goel A, Goyette P, Grallert H, Grässler J, Greenawalt DM, Groves CJ, Gudnason V, Guiducci C, Hartikainen AL, Hassanali N, Hall AS, Havulinna AS, Hayward C, Heath AC, Hengstenberg C, Hicks AA, Hinney A, Hofman A, Homuth G, Hui J, Igl W, Iribarren C, Isomaa B, Jacobs KB, Jarick I, Jewell E, John U, Jørgensen T, Jousilahti P, Jula A, Kaakinen M, Kajantie E, Kaplan LM, Kathiresan S, Kettunen J, Kinnunen L, Knowles JW, Kolcic I, König IR, Koskinen S, Kovacs P, Kuusisto J, Kraft P, Kvaløy K, Laitinen J, Lantieri O, Lanzani C, Launer LJ, Lecoeur C, Lehtimäki T, Lettre G, Liu J, Lokki ML, Lorentzon M, Luben RN, Ludwig B, Manunta P, Marek D, Marre M, Martin NG, McArdle WL, McCarthy A, McKnight B, Meitinger T, Melander O, Meyre D, Midthjell K, Montgomery GW, Morken MA, Morris AP, Mulic R, Ngwa JS, Nelis M, Neville MJ, Nyholt DR, O’Donnell CJ, O’Rahilly S, Ong KK, Oostra B, Paré G, Parker AN, Perola M, Pichler I, Pietiläinen KH, Platou CG, Polasek O, Pouta A, Rafelt S, Raitakari O, Rayner NW, Ridderstråle M, Rief W, Ruokonen A, Robertson NR, Rzehak P, Salomaa V, Sanders AR, Sandhu MS, Sanna S, Saramies J, Savolainen MJ, Scherag S, Schipf S, Schreiber S, Schunkert H, Silander K, Sinisalo J, Siscovick DS, Smit JH, Soranzo N, Sovio U, Stephens J, Surakka I, Swift AJ, Tammesoo ML, Tardif JC, Teder-Laving M, Teslovich TM, Thompson JR, Thomson B, Tönjes A, Tuomi T, van Meurs JB, van Ommen GJ, Vatin V, Viikari J, Visvikis-Siest S, Vitart V, Vogel CI, Voight BF, Waite LL, Wallaschofski H, Walters GB, Widen E, Wiegand S, Wild SH, Willemsen G, Witte DR, Witteman JC, Xu J, Zhang Q, Zgaga L, Ziegler A, Zitting P, Beilby JP, Farooqi IS, Hebebrand J, Huikuri HV, James AL, Kähönen M, Levinson DF, Macciardi F, Nieminen MS, Ohlsson C, Palmer LJ, Ridker PM, Stumvoll M, Beckmann JS, Boeing H, Boerwinkle E, Boomsma DI, Caulfield MJ, Chanock SJ, Collins FS, Cupples LA, Smith GD, Erdmann J, Froguel P, Grönberg H, Gyllensten U, Hall P, Hansen T, Harris TB, Hattersley AT, Hayes RB, Heinrich J, Hu FB, Hveem K, Illig T, Jarvelin MR, Kaprio J, Karpe F, Khaw KT, Kiemeney LA, Krude H, Laakso M, Lawlor DA, Metspalu A, Munroe PB, Ouwehand WH, Pedersen O, Penninx BW, Peters A, Pramstaller PP, Quertermous T, Reinehr T, Rissanen A, Rudan I, Samani NJ, Schwarz PE, Shuldiner AR, Spector TD, Tuomilehto J, Uda M, Uitterlinden A, Valle TT, Wabitsch M, Waeber G, Wareham NJ, Watkins H, Wilson JF, Wright AF, Zillikens MC, Chatterjee N, McCarroll SA, Purcell S, Schadt EE, Visscher PM, Assimes TL, Borecki IB, Deloukas P, Fox CS, Groop LC, Haritunians T, Hunter DJ, Kaplan RC, Mohlke KL, O’Connell JR, Peltonen L, Schlessinger D, Strachan DP, van Duijn CM, Wichmann HE, Frayling TM, Thorsteinsdottir U, Abecasis GR, Barroso I, Boehnke M, Stefansson K, North KE, McCarthy MI, Hirschhorn JN, Ingelsson E, Loos RJ. Association analyses of 249,796 individuals reveal 18 new loci associated with body mass index. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>(11):937–948.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014648/">PMC free article</a>]</span> [<a href="/pubmed/20935630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Association+analyses+of+249,796+individuals+reveal+18+new+loci+associated+with+body+mass+index&amp;author=EK+Speliotes&amp;author=CJ+Willer&amp;author=SI+Berndt&amp;author=KL+Monda&amp;author=G+Thorleifsson&amp;volume=42&amp;issue=11&amp;publication_year=2010&amp;pages=937-948&amp;pmid=20935630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="element-citation">Spittel ML, Spotts EL, Deeds BG. Integration of behavioral, social science and genetics research: exploring public health significance. <span><span class="ref-journal">Am J Public Health. </span>2013;<span class="ref-vol">103</span>(Suppl 1):S5–S7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5320918/">PMC free article</a>]</span> [<a href="/pubmed/23927547" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Public+Health&amp;title=Integration+of+behavioral,+social+science+and+genetics+research:+exploring+public+health+significance&amp;author=ML+Spittel&amp;author=EL+Spotts&amp;author=BG+Deeds&amp;volume=103&amp;issue=Suppl+1&amp;publication_year=2013&amp;pages=S5-S7&amp;pmid=23927547&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R41"><span class="element-citation">Thornton T, Tang H, Hoffmann TJ, Ochs-Balcom HM, Caan BJ, Risch N. Estimating kinship in admixed populations. <span><span class="ref-journal">Am J Hum Genet. </span>2012;<span class="ref-vol">91</span>(1):122–138.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3397261/">PMC free article</a>]</span> [<a href="/pubmed/22748210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Estimating+kinship+in+admixed+populations&amp;author=T+Thornton&amp;author=H+Tang&amp;author=TJ+Hoffmann&amp;author=HM+Ochs-Balcom&amp;author=BJ+Caan&amp;volume=91&amp;issue=1&amp;publication_year=2012&amp;pages=122-138&amp;pmid=22748210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R42"><span class="element-citation">Xu H, Yang W, Perez-Andreu V, Devidas M, Fan Y, Cheng C, Pei D, Scheet P, Burchard EG, Eng C, Huntsman S, Torgerson DG, Dean M, Winick NJ, Martin PL, Camitta BM, Bowman WP, Willman CL, Carroll WL, Mullighan CG, Bhojwani D, Hunger SP, Pui CH, Evans WE, Relling MV, Loh ML, Yang JJ. Novel susceptibility variants at 10p12.31-12.2 for childhood acute lymphoblastic leukemia in ethnically diverse populations. <span><span class="ref-journal">J Natl Cancer Inst. </span>2013;<span class="ref-vol">105</span>(10):733–742.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3691938/">PMC free article</a>]</span> [<a href="/pubmed/23512250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Novel+susceptibility+variants+at+10p12.31-12.2+for+childhood+acute+lymphoblastic+leukemia+in+ethnically+diverse+populations&amp;author=H+Xu&amp;author=W+Yang&amp;author=V+Perez-Andreu&amp;author=M+Devidas&amp;author=Y+Fan&amp;volume=105&amp;issue=10&amp;publication_year=2013&amp;pages=733-742&amp;pmid=23512250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R43"><span class="element-citation">Yang J, Benyamin B, McEvoy BP, Gordon S, Henders AK, Nyholt DR, Madden PA, Heath AC, Martin NG, Montgomery GW, Goddard ME, Visscher PM. Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>(7):565–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J+Yang&amp;author=B+Benyamin&amp;author=BP+McEvoy&amp;author=S+Gordon&amp;author=AK+Henders&amp;volume=42&amp;issue=7&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R44"><span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>(1):76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;issue=1&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289640&amp;issue-id=247453&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139780432197504title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials andMethods</a></li><li><a href="#S10title">Results</a></li><li><a href="#S15title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#idm139780375803728title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4289640/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4289640/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4289640/pdf/nihms-641008.pdf">PDF (580K)</a></li> | <li><a href="#" data-citationid="PMC4289640" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4289640%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4289640%2F&amp;text=The%20National%20Longitudinal%20Study%20of%20Adolescent%20to%20Adult%20Health%20(Add%20Health)%20Sibling%20Pairs%20Genome-Wide%20Data" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4289640%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25378290" data-aiid="4289640" data-aid="4289640" data-iid="247453" data-domainid="319" data-domain="nihpa" data-accid="PMC4289640" data-md5="965848f034ffd1e7cc36d422f6a77a2d"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:52:47-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8DEB6ADD57B90100000000013600BF&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4289640%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4289640/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>