<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4107465/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Common genetic variants and gene expression associated with white matter microstructure in the human brain</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4107465/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="NeuroImage"><meta name="citation_title" content="Common genetic variants and gene expression associated with white matter microstructure in the human brain"><meta name="citation_authors" content="Emma Sprooten, Emma E Knowles, D Reese McKay, Harald HH Göring, Joanne E Curran, Jack W Kent, Jr, Melanie A Carless, Thomas D Dyer, Eugene I Drigalenko, Rene L Olvera, Peter T Fox, Laura Almasy, Ravi Duggirala, Peter Kochunov, John Blangero, David C Glahn"><meta name="citation_date" content="15 August 2014"><meta name="citation_volume" content="97"><meta name="citation_firstpage" content="252"><meta name="citation_doi" content="10.1016/j.neuroimage.2014.04.021"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4107465/?report=abstract"><meta name="citation_pmid" content="24736177"><meta name="DC.Title" content="Common genetic variants and gene expression associated with white matter microstructure in the human brain"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Emma Sprooten"><meta name="DC.Contributor" content="Emma E Knowles"><meta name="DC.Contributor" content="D Reese McKay"><meta name="DC.Contributor" content="Harald HH Göring"><meta name="DC.Contributor" content="Joanne E Curran"><meta name="DC.Contributor" content="Jack W Kent, Jr"><meta name="DC.Contributor" content="Melanie A Carless"><meta name="DC.Contributor" content="Thomas D Dyer"><meta name="DC.Contributor" content="Eugene I Drigalenko"><meta name="DC.Contributor" content="Rene L Olvera"><meta name="DC.Contributor" content="Peter T Fox"><meta name="DC.Contributor" content="Laura Almasy"><meta name="DC.Contributor" content="Ravi Duggirala"><meta name="DC.Contributor" content="Peter Kochunov"><meta name="DC.Contributor" content="John Blangero"><meta name="DC.Contributor" content="David C Glahn"><meta name="DC.Date" content="2014 Aug 15"><meta name="DC.Identifier" content="10.1016/j.neuroimage.2014.04.021"><meta name="DC.Language" content="en"><meta property="og:title" content="Common genetic variants and gene expression associated with white matter microstructure in the human brain"><meta property="og:type" content="article"><meta property="og:description" content="Identifying genes that contribute to white matter microstructure should provide insights into the neurobiological processes that regulate white matter development, plasticity and pathology. We detected five significant SNPs using genome-wide association ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4107465/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4107465/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4107465/pdf/nihms606427.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C3D0CDD5669D100000000085E04F6.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4107465%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4107465</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Neuroimage</a></span>. Author manuscript; available in PMC 2015 Aug 15.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=24736177" target="pmc_ext" ref="reftype=publisher&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Neuroimage. 2014 Aug 15; 97: 252–261. </span></a></div><span class="fm-vol-iss-date">Published online 2014 Apr 13. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.neuroimage.2014.04.021" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.neuroimage.2014.04.021</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4107465</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS606427</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24736177">24736177</a></div></div></div></div><h1 class="content-title">Common genetic variants and gene expression associated with white matter microstructure in the human brain</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sprooten%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811719756096" co-class="co-affbox">Emma Sprooten</a>, PhD,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Knowles%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811723104528" co-class="co-affbox">Emma E Knowles</a>, PhD,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=McKay%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811755070912" co-class="co-affbox">D Reese McKay</a>, PhD,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=G%26%23x000f6%3Bring%20HH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811710749824" co-class="co-affbox">Harald HH Göring</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Curran%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811714188240" co-class="co-affbox">Joanne E Curran</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Kent%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811714186000" co-class="co-affbox">Jack W Kent, Jr</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Carless%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811719139440" co-class="co-affbox">Melanie A Carless</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Dyer%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811710963904" co-class="co-affbox">Thomas D Dyer</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Drigalenko%20EI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811710961664" co-class="co-affbox">Eugene I Drigalenko</a>,<sup>3</sup> <a href="/pubmed/?term=Olvera%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811708635408" co-class="co-affbox">Rene L Olvera</a>, MD,<sup>4</sup> <a href="/pubmed/?term=Fox%20PT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811718883088" co-class="co-affbox">Peter T Fox</a>, MD,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Almasy%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811708563440" co-class="co-affbox">Laura Almasy</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Duggirala%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811722368176" co-class="co-affbox">Ravi Duggirala</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Kochunov%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811722365936" co-class="co-affbox">Peter Kochunov</a>, PhD,<sup>7</sup> <a href="/pubmed/?term=Blangero%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811756773216" co-class="co-affbox">John Blangero</a>, PhD,<sup>3</sup> and  <a href="/pubmed/?term=Glahn%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177" class="affpopup" co-rid="_co_idm139811719938944" co-class="co-affbox">David C Glahn</a>, PhD<sup>1,</sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139811719756096"><h3 class="no_margin">Emma Sprooten</h3><p><sup>1</sup>Department of Psychiatry, Yale University School of Medicine, 300 George Street, New Haven, CT, USA</p><p><sup>2</sup>Olin Neuropsychiatry Research Center, Institute of Living, Hartford Hospital, 200 Retreat Avenue, CT, USA</p><div>Find articles by <a href="/pubmed/?term=Sprooten%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Emma Sprooten</a></div></div><div id="_co_idm139811723104528"><h3 class="no_margin">Emma E Knowles</h3><p><sup>1</sup>Department of Psychiatry, Yale University School of Medicine, 300 George Street, New Haven, CT, USA</p><p><sup>2</sup>Olin Neuropsychiatry Research Center, Institute of Living, Hartford Hospital, 200 Retreat Avenue, CT, USA</p><div>Find articles by <a href="/pubmed/?term=Knowles%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Emma E Knowles</a></div></div><div id="_co_idm139811755070912"><h3 class="no_margin">D Reese McKay</h3><p><sup>1</sup>Department of Psychiatry, Yale University School of Medicine, 300 George Street, New Haven, CT, USA</p><p><sup>2</sup>Olin Neuropsychiatry Research Center, Institute of Living, Hartford Hospital, 200 Retreat Avenue, CT, USA</p><div>Find articles by <a href="/pubmed/?term=McKay%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">D Reese McKay</a></div></div><div id="_co_idm139811710749824"><h3 class="no_margin">Harald HH Göring</h3><p><sup>3</sup>Department of Genetics, Texas Biomedical Research Institute, PO Box 760549, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=G%26%23x000f6%3Bring%20HH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Harald HH Göring</a></div></div><div id="_co_idm139811714188240"><h3 class="no_margin">Joanne E Curran</h3><p><sup>3</sup>Department of Genetics, Texas Biomedical Research Institute, PO Box 760549, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Curran%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Joanne E Curran</a></div></div><div id="_co_idm139811714186000"><h3 class="no_margin">Jack W Kent, Jr</h3><p><sup>3</sup>Department of Genetics, Texas Biomedical Research Institute, PO Box 760549, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Kent%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Jack W Kent, Jr</a></div></div><div id="_co_idm139811719139440"><h3 class="no_margin">Melanie A Carless</h3><p><sup>3</sup>Department of Genetics, Texas Biomedical Research Institute, PO Box 760549, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Carless%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Melanie A Carless</a></div></div><div id="_co_idm139811710963904"><h3 class="no_margin">Thomas D Dyer</h3><p><sup>3</sup>Department of Genetics, Texas Biomedical Research Institute, PO Box 760549, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Dyer%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Thomas D Dyer</a></div></div><div id="_co_idm139811710961664"><h3 class="no_margin">Eugene I Drigalenko</h3><p><sup>3</sup>Department of Genetics, Texas Biomedical Research Institute, PO Box 760549, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Drigalenko%20EI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Eugene I Drigalenko</a></div></div><div id="_co_idm139811708635408"><h3 class="no_margin">Rene L Olvera</h3><p><sup>4</sup>Department of Psychiatry, University of Texas Health Science Center San Antonio, 7703 Floyd Curl Drive, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Olvera%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Rene L Olvera</a></div></div><div id="_co_idm139811718883088"><h3 class="no_margin">Peter T Fox</h3><p><sup>5</sup>Research Imaging Institute, University of Texas Health Science Center San Antonio, 8403 Floyd Curl Drive, San Antonio, TX, USA</p><p><sup>6</sup>South Texas Veterans Health System, 7400 Merton Minter, San Antonio, Texas 78229</p><div>Find articles by <a href="/pubmed/?term=Fox%20PT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Peter T Fox</a></div></div><div id="_co_idm139811708563440"><h3 class="no_margin">Laura Almasy</h3><p><sup>3</sup>Department of Genetics, Texas Biomedical Research Institute, PO Box 760549, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Almasy%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Laura Almasy</a></div></div><div id="_co_idm139811722368176"><h3 class="no_margin">Ravi Duggirala</h3><p><sup>3</sup>Department of Genetics, Texas Biomedical Research Institute, PO Box 760549, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Duggirala%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Ravi Duggirala</a></div></div><div id="_co_idm139811722365936"><h3 class="no_margin">Peter Kochunov</h3><p><sup>7</sup>Maryland Psychiatric Research Center, Department of Psychiatry, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Kochunov%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">Peter Kochunov</a></div></div><div id="_co_idm139811756773216"><h3 class="no_margin">John Blangero</h3><p><sup>3</sup>Department of Genetics, Texas Biomedical Research Institute, PO Box 760549, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Blangero%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">John Blangero</a></div></div><div id="_co_idm139811719938944"><h3 class="no_margin">David C Glahn</h3><p><sup>1</sup>Department of Psychiatry, Yale University School of Medicine, 300 George Street, New Haven, CT, USA</p><p><sup>2</sup>Olin Neuropsychiatry Research Center, Institute of Living, Hartford Hospital, 200 Retreat Avenue, CT, USA</p><div>Find articles by <a href="/pubmed/?term=Glahn%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24736177">David C Glahn</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139811721451424_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139811721451424_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139811721451424_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Psychiatry, Yale University School of Medicine, 300 George Street, New Haven, CT, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Olin Neuropsychiatry Research Center, Institute of Living, Hartford Hospital, 200 Retreat Avenue, CT, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Genetics, Texas Biomedical Research Institute, PO Box 760549, San Antonio, TX, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Psychiatry, University of Texas Health Science Center San Antonio, 7703 Floyd Curl Drive, San Antonio, TX, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Research Imaging Institute, University of Texas Health Science Center San Antonio, 8403 Floyd Curl Drive, San Antonio, TX, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>South Texas Veterans Health System, 7400 Merton Minter, San Antonio, Texas 78229</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Maryland Psychiatric Research Center, Department of Psychiatry, University of Maryland School of Medicine, Baltimore, MD, USA</div><div id="cor1"><strong>Corresponding Author</strong>: Emma Sprooten Ph.D., Olin Neuropsychiatry Research Center, Institute of Living/Hartford Hospital, 200 Retreat Avenue, CT, 06106, USA;, <a href="mailto:dev@null" data-email="ude.elay@netoorps.amme" class="oemail">ude.elay@netoorps.amme</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139811721451424_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN1">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=24736177" target="pmc_ext" ref="reftype=publisher&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Neuroimage</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4107465/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="4" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC4107465/bin/NIHMS606427-supplement-1.pdf" data-ga-action="click_feat_suppl">NIHMS606427-supplement-1.pdf</a><span style="color:gray"> (34K)</span></div><div class="small guid">GUID:&nbsp;15893A59-1D22-4823-A528-289A93616C9B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC4107465/bin/NIHMS606427-supplement-2.pdf" data-ga-action="click_feat_suppl">NIHMS606427-supplement-2.pdf</a><span style="color:gray"> (628K)</span></div><div class="small guid">GUID:&nbsp;D0D77275-4568-4670-926E-BDF7CDF03912</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>3.</strong></div><div><a href="/pmc/articles/PMC4107465/bin/NIHMS606427-supplement-3.docx" data-ga-action="click_feat_suppl">NIHMS606427-supplement-3.docx</a><span style="color:gray"> (17K)</span></div><div class="small guid">GUID:&nbsp;B088135D-24DC-4FAF-A7DA-C752F2B8DD90</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>4.</strong></div><div><a href="/pmc/articles/PMC4107465/bin/NIHMS606427-supplement-4.docx" data-ga-action="click_feat_suppl">NIHMS606427-supplement-4.docx</a><span style="color:gray"> (21K)</span></div><div class="small guid">GUID:&nbsp;48F19B35-2E74-4280-AB63-715E79220EAF</div></div></dd></dl></div><div id="idm139811717441424" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139811717441424title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Identifying genes that contribute to white matter microstructure should provide insights into the neurobiological processes that regulate white matter development, plasticity and pathology. We detected five significant SNPs using genome-wide association analysis on a global measure of fractional anisotropy in 776 individuals from large extended pedigrees. Genetic correlations and genome-wide association results indicated that the genetic signal was largely homogeneous across white matter regions. Using RNA transcripts derived from lymphocytes in the same individuals, we identified two genes (<em>GNA13</em> and <em>CCDC91</em>) that are likely to be <em>cis</em>-regulated by top SNPs, and whose expression levels were also genetically correlated with fractional anisotropy. A transcript of <em>HTR7</em> was phenotypically associated with FA, and was associated with an intronic genome-wide significant SNP. These results encourage further research in the mechanisms by which <em>GNA13, HTR7</em> and <em>CCDC91</em> influence brain structure, and emphasize a role for g-protein signaling in the development and maintenance of white matter microstructure in health and disease.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Diffusion tensor imaging, white matter, genome-wide association, gene expression, transcripts, extended pedigrees</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">1. Introduction</h2><p id="P2" class="p p-first">Studies in twins and extended pedigrees have established that white matter microstructure, as measured <em>in vivo</em> by diffusion tensor imaging (DTI), is heritable (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896316">Chiang et al., 2011b</a>; <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896312">Jahanshad et al., 2013a</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896301">Kochunov et al., 2010</a>). However, the genetic variants contributing to this heritability are unknown and little is understood about the mechanisms that govern the development, maintenance, plasticity and pathology of white matter microstructure. White matter plays an important a role in several neurological diseases (<a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896331">Stebbins and Murphy, 2009</a>) and psychiatric disorders (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896298">Kubicki et al., 2007</a>; <a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896335">Mahon et al., 2010</a>), which are phenotypes that also have substantial but poorly characterized, genetic components. There is increasing evidence that compromised white matter microstructure is part of the inherited risk for these disorders, as indicated by reduced FA in unaffected relatives (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896268">Gold et al., 2012</a>; <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896274">Hoptman et al., 2008</a>; <a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_482631778">Sprooten et al., 2013a</a>; <a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896302">Sprooten et al., 2011a</a>), and polygenic risk score analysis (<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_482631781">Whalley et al., 2013</a>). Therefore, identifying genes that influence white-matter microstructure could provide a biological anchor for disentangling basic molecular mechanisms that predispose to these debilitating disorders, potentially leading to novel treatment agents and prevention strategies.</p><p id="P3">DTI is a magnetic resonance imaging technique that is based on the orientation and magnitude of the motion of water molecules, and its restriction by surrounding tissue. Because of the parallel alignment of white matter fibers that restrict motion primarily in directions perpendicular to the fibers, DTI is ideally suited to measure properties of white matter microstructure (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896315">Beaulieu, 2002</a>). Fractional anisotropy (FA) is an index of the extent to which this motion is directionally constrained and, as validated in animal (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896306">Li et al., 2011</a>) and post-mortem research (<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896278">Schmierer et al., 2007</a>), it reflects a combination of myelin thickness, fiber coherence and axon integrity. Studies using <em>a priori</em> selected candidate genes and SNPs have associated FA with genetic variation in <em>NRG1</em> (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896294">McIntosh et al., 2008</a>; <a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896313">Sprooten et al., 2009</a>; <a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896269">Winterer et al., 2008</a>), <em>ErbB4</em> (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896261">Konrad et al., 2009</a>; <a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654415443">Zuliani et al., 2011</a>), <em>DISC1</em> (<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896308">Sprooten et al., 2011b</a>), <em>NTRK1</em> (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896329">Braskie et al., 2012</a>), <em>BDNF</em> (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896276">Chiang et al., 2011a</a>) and <em>APOE</em> (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896344">Jahanshad et al., 2012</a>), amongst others. However, FA is a complex, polygenic phenotype and for most complex phenotypes data-driven GWA have not implicated a priori candidate variants in their top results (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896309">Flint and Munafo, 2013</a>; <a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896283">Stein et al., 2012</a>), hence many more novel SNP-associations contributing to variation in FA could be discovered using GWA.</p><p id="P4">Numerous common variants correlated with complex disease risks have been reported using GWA (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896318">Hindorff et al., 2009</a>; <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896319">Hirschhorn and Daly, 2005</a>; <a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896321">Ripke et al., 2013</a>), but the effect size of individual common variants on complex phenotypes tend to be small (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896265">Flint and Munafo, 2013</a>; <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896292">Hindorff et al., 2009</a>). Significant genome-wide association reflects the presence of a relevant functional variant in the surrounding genomic region and thus is indicative of causal gene localization but not the identification of underlying biological mechanism, which is the ultimate goal of complex disease genetics. It is difficult to infer a specific gene’s involvement in trait variance solely based upon a statistically significant association, since the polymorphisms tagged in GWA rarely influence gene function directly and the effect of a tagging SNP reflects, in addition to the effect that it exerts, the effects of all SNPs within the surrounding linkage disequilibrium (LD) block, which may span many genes any one (or combination) of which could be driving the observed association. Examining complementary biological information, such as RNA expression, can refine inferences made from GWA and identify potential genes through which the associated SNPs are likely to exert their effect.</p><p id="P5">In the present paper, we aim to characterize the common variation contributing to the genetics of white matter microstructure. Firstly, to identify common variants affecting white matter microstructure we performed GWA of a global FA measure in a sample of 776 Mexican-American members of extended pedigrees. Secondly, to propose genes that may be responsible for the effects of these common variants, we correlated lymphocyte-derived RNA transcripts of nearby genes both with genetic variation in the genome-wide significant SNPs and with white matter microstructure.</p><p id="P6" class="p p-last">We chose a global index of white matter microstructure, namely average FA across the white matter skeleton derived from tract-based spatial statistics (TBSS), as our primary phenotype. This phenotype was previously shown to be heritable in multiple cohorts (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896304">Jahanshad et al., 2013a</a>), and relevant to genetic risk for bipolar disorder (<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_482631776">Sprooten et al., 2013a</a>; <a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896310">Sprooten et al., 2011a</a>) and major depressive disorder (<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_482631779">Whalley et al., 2013</a>). To examine the neuroanatomical specificity of genetic effects on FA, we performed voxel-wise analyses for our top SNPs, and used the family structure and additional GWA results of regional FA to investigate the degree to which additive genetic effects across the genome are anatomically homogeneous throughout the brain.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">2. Materials and Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">2.1 Participants</h3><p id="P7" class="p p-first">Participants were individuals of Mexican American ancestry who took part in the Genetics of Brain Structure and Function Study (GOBS) (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896290">Olvera et al., 2011</a>), which is an extension of the San Antonio Family study (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896291">Mitchell et al., 1996</a>; <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896288">Puppala et al., 2006</a>). Individuals in this cohort have actively participated in research for over 18 years and were randomly selected from the community with the only constraints that they are of Mexican-American ancestry, part of a large family and live within the San Antonio region. For the present analysis, subjects were excluded if they had MRI contraindications, documented history of neurological illness, or any major neurological event visible on the structural T1 or DTI scans (see also section 2.3 below). Of the participants in the final analysis, 16 self-reported history of a neurological event, but excluding these individuals did not change the pattern of results presented. Individuals with a history of psychiatric illness were included in the main analyses (psychosis: N=18, bipolar disorder: N=8, anxiety disorder: N=162, major depression: N=249), but all results remained significant at similar levels when covarying for history of these psychiatric diagnoses. Additional recruitment and demographic details of the sample are available elsewhere (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896320">Glahn et al., 2012</a>; <a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896327">Olvera et al., 2011</a>).</p><p id="P8" class="p p-last">After thorough data quality checks (see section 2.3), 776 individuals contributed both high-quality DTI scans and blood samples. This sample included 65 pedigrees, varying in size from 2 to 82 individuals, and 49 unrelated individuals. Sixty percent (N = 466) of the participants were female and their age ranged from 18 to 85 years (mean 44.37). All participants provided written informed consent on form approved by the IRBs at Yale University and the University of Texas Health Science Center in San Antonio (UTHSCSA).</p></div><div id="S4" class="sec"><h3 id="S4title">2.2 Diffusion tensor imaging acquisition</h3><p id="P9" class="p p-first-last">Diffusion imaging data were acquired on a Siemens 3T Trio scanner located at the Research Imaging Institute, UTHSCSA (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896322">Kochunov et al., 2010</a>; <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896325">Kochunov et al., 2007</a>). A single-shot, echo-planar, single refocusing spin-echo weighted sequence with TE/TR=87/8000ms was applied to obtain 50 contiguous axial slices field-of-view=200 × 200mm, thickness 3mm, resolution 1.7mm × 1.7mm) in 55 isotropically distributed diffusion weighted directions (b=0 and b=700s/mm<sup>2</sup>). Three non-diffusion weighted (b=0) images were acquired at the start of each sequence.</p></div><div id="S5" class="sec"><h3 id="S5title">2.3 DTI processing and extraction of FA values</h3><p id="P10" class="p p-first">Diffusion imaging data were converted to NIFTI format using MRIcron and pre-processed using standard tools available in FSL (<a href="http://www.fmrib.ox.ac.uk/fsl" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.fmrib.ox.ac.uk/fsl</a>) (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896287">Behrens et al., 2003</a>). First, for each subject, the 55 diffusion-weighted images were corrected for subject motion and eddy currents by linearly aligning them to the first b<sub>0</sub> volume within each individual. Next, non-brain tissue was removed using the brain extraction tool (“bet”) with a fractional intensity threshold of 0.1. Then, diffusion eigenvectors, eigenvalues and FA were calculated for each voxel using “dtifit”.</p><p id="P11">TBSS (<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896337">Smith et al., 2006</a>), a standardized and validated procedure for DTI analysis tailored to white matter anatomy, was applied to extract a global measure of whole-brain white matter microstructure. First FA images were slightly eroded and non-linearly registered to standard MNI space. FA maps were masked with a study-specific mask, which was created such that only voxels with FA&gt;0 in every participant were included.</p><p id="P12">The average FA map was created and fed into the TBSS skeletonization algorithm, and thresholded at FA&gt;0.2, resulting in a sample-specific skeleton template (<a href="/pmc/articles/PMC4107465/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Next, within each individual FA map the nearest maximum FA voxel was projected onto the skeleton template, to obtain one skeleton image per subject, representing the central structure of each subject’s white matter. The mean FA within this skeleton was extracted for each individual, hereafter referred to as “global FA”.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4107465/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139811708658032" class="figure"><a href="/pmc/articles/PMC4107465/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4107465_nihms606427f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms606427f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4107465/bin/nihms606427f1.jpg"></a></div><div id="largeobj_idm139811708658032" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4107465/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4107465/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="P53">Voxel-wise heritability of FA across the TBSS skeleton (green). Voxels where the heritability of FA was 0.2 or greater are colored according to the red-yellow color bar below.</p></div></div></div><p id="P13">The FA maps were visually inspected before and after registration to the MNI template. From the total of 796 DTI scans, 20 subjects were excluded from further analysis (7 for anatomical abnormalities, 5 for scanner and 8 for registration artifacts).</p><p id="P14" class="p p-last">In additional to global FA, 31 regional FA values were derived from the TBSS skeletons based on the John Hopkins University White Matter Labels Atlas, which is integrated in FSL. FA was extracted for all regions within this atlas, apart from for the uncinate fasciculi, the fornix and the cortico-spinal tracts because of incomplete coverage by the atlas and/or previous indications of low and unreliable heritability estimates in these regions (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896330">Jahanshad et al.; 2013a</a>). Masks derived from the atlas were smoothed with a kernel of 1.1 mm FWHM, thresholded at &gt;0.2 and binarized. Accurate overlap with the TBSS skeleton was visually verified for each region. Each mask was multiplied by the TBSS skeleton mask to obtain a parcellation of the center of each region-of-interest and mean FA values within each region were extracted for each subject.</p></div><div id="S6" class="sec"><h3 id="S6title">2.4 Heritability estimation</h3><p id="P15" class="p p-first-last">All statistics were calculated in SOLAR (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896336">Almasy and Blangero, 1998</a>), which applies maximum likelihood methods to decompose the variance of a trait into genetic and environmental components by modeling the covariance between individuals as a function of their genetic proximity. More specifically, to estimate heritability the covariance matrix was decomposed into the following: the observed additive coefficient of relationship matrix (reflecting genetic proximity), a set of covariates (sex, age, age<sup>2</sup>, age x sex, age<sup>2</sup> x sex), and a random environment component. The significance of each variance component is assessed by a chi-squared test comparing the final model to the model without the component of interest. This method was applied to calculate heritability for global FA and for each of the regions of interests in separate models. As SOLAR assumes a multivariate normal distribution, FA measures were transformed using an inverse normalization transformation prior to running variance decomposition analysis.</p></div><div id="S7" class="sec"><h3 id="S7title">2.5 Genotyping and genome-wide association</h3><p id="P16" class="p p-first">DNA was extracted from lymphocytes and genotyped using either the Illumina Human1M-Duo Beadchip or the Illumina HumanHap550 BeadChip in tandem with the supplemental HumanHap450S Beadchip. Both methods result in ∼ 1 million tagging SNPs, capturing approximately 90% of the common variation in humans. Genotype data were checked for accordance with Mendelian consistency using SimWalk2 (<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896264">Sobel and Lange, 1996</a>). Missing genotypes were imputed using a procedure specifically developed for extended pedigrees in MERLIN (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896281">Burdick et al., 2006</a>). After excluding the 2407 SNPs that deviated significantly from the Hardy-Weinberg equilibrium (Χ<sup>2</sup> &gt; 10), SNPs with a call rate &lt; 95% and 820 SNPs with a minor allele frequency &lt; 0.001, a total of 929,187 SNPs remained for further analysis.</p><p id="P17" class="p p-last">For genome-wide association analysis of global FA, each SNP dosage was coded as 0, 1 or 2, or for missing genotypes as the weighted value based on the imputation. Each SNP is added as a “covariate” to the heritability model described above, one at a time and assuming an additive effect of alleles. The significance of each SNP is adjusted for family structure by comparing the null model without the SNP, but with the coefficient of relationship matrix and the covariates, to the model of interest with the SNP. To account for population stratification, the first four principal components of the genome-wide SNP data were also added to this model. The fast exact likelihood calculation method described by Blangero <em>et al</em>. (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896262">Blangero et al., 2013</a>) was employed for all analyses. P-values &lt; 5*10<sup>−8</sup> were considered genome-wide significant (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896260">Pe'er et al., 2008</a>). The genotypes of the five top SNPs were checked in the original Illumina cluster files, and there were no obvious artifacts at these specific loci.</p></div><div id="S8" class="sec"><h3 id="S8title">2.6 Transcript processing and analysis</h3><p id="P18" class="p p-first">Whole-genome expression profiles were derived from untransformed lymphocytes which were collected during the clinic visit when the MRI scans were taken. The methods used to generate gene expression levels are similar to those described in detail elsewhere (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896307">Goring et al., 2007</a>). In brief, lymphocytes were isolated from whole-blood samples that were collected in the morning after a night of fasting, and stored in liquid nitrogen. Lymphocyte RNA was isolated and anti-sense RNA was synthesized, amplified, purified and hybridized to Illumina BeadChips version 3 according to standard protocols. 647 samples (including controls and duplicates) were processed in a single batch and analyzed using Illumina BeadScan and BeadStudio software. The samples were evaluated based on the average number of probes with significant expression (at a detection p-value ≤ 0.05; see below), mean raw “average signal” across probes, and average correlation (across probes) against all other samples. 595 high quality, unique samples were identified. Of these, 360 individuals also had high-quality DTI and GWA data (i.e. overlapped with the sample on which we did the GWA). Significantly expressed probes were identified as follows: (1) for each probe, samples were classified as significantly or not significantly expressed at a "detection p-value" (which is defined by the Illumina software based on comparisons with null probes) of p &lt;= 0.05; (2) for each probe, binomial tests were conducted on the number of samples in which the probes was "detected" or "not detected"; and (3) finally, considering all p-values resulting from the binomial tests for all probes, probes were accepted to be significantly expressed at a 5% false-discovery rate (FDR). As a result, 25,663 probes were significantly expressed above the baseline levels at 5% FDR. Subsequently, expression levels were processed by background noise subtraction (shifting values upwards so that the minimum raw “average signal” for a probe in a sample was 1), log 2 transformation, following by quantile normalization. Outliers of expression levels (&gt; 3 standard deviations from the mean) were removed before further analyses (between 0 and 9 cases per transcript).</p><p id="P19">To investigate whether any of the top SNPs (or SNPs in LD with them) could potentially have an effect on the function of a nearby gene, we entered the standardized, normalized expression levels of genes within a 500kb window (according to the NCBI SNP database) as traits in a polygenic model in SOLAR, with SNP as a covariate. The resulting p-values were corrected for multiple comparisons across transcripts at a false-discovery rate of 5%, and those that were significant were further investigated for any associations with FA. To quantify this relationship between FA and expression levels, the global FA measure and the gene’s transcript levels were both entered as traits in bivariate polygenic models in SOLAR to calculate their genetic correlation. In these models, the pedigree structure is used to decompose the phenotypic correlation (ρ<sub>p</sub>) between two traits (i and j) into genetic (ρ<sub>g</sub>) and environmental correlations (ρ<sub>e</sub>) so that 
</p><div class="disp-formula" id="FD1"><div class="f">ρ<sub>p</sub> =&nbsp;ρ<sub>g</sub> √(h<sup>2</sup><sub>j*</sub>h<sup>2</sup><sub>i</sub>) +&nbsp;ρ<sub>e</sub> √((1&nbsp;−&nbsp;h<sup>2</sup><sub>j</sub>)(1&nbsp;−&nbsp;h<sup>2</sup><sub>i</sub>))</div></div><p> while the ρ<sub>p</sub> quantifies the overal relationship between two traits, ρ<sub>g</sub> represents the amount of genetic variance shared between two traits, or pleiotropy (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896280">Almasy et al., 1997</a>; <a href="#R62" rid="R62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896275">Williams et al., 1999</a>). These parameters were estimated in polygenic models in SOLAR using all available pedigree information, and their significance was tested by comparing the log-likelihood of this polygenic model against the log-likelihood of the null model (where ρ<sub>p</sub> and ρ<sub>g</sub>, are respectively constrained equal to 0). The resulting likelihood ratio test is distributed as a chi-square with a single degree of freedom. As before, prior to running these models, FA and transcriptional measures were normalized using an inverse Guassion transformation, and for all polygenic models, age, sex, age squared and their interactions were included as (additional) covariates.</p><p id="P20" class="p p-last">The Allen Human Brain Atlas (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_482631777">Hawrylycz et al., 2012</a>) (<a href="http://human.brain-map.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://human.brain-map.org/</a>) was utilized to examine the expression patterns of genes that were significantly genetically correlated with FA in human white matter. For each gene, we downloaded the z-scores of all probes, of all available individuals for the corpus callosum, the cingulum and the central glial system. Since the transcript data in the Allen Brain Atlas are normalized within each batch, and within each subject across the brain, this z-score reflects the number of standard deviations away from the overall mean expression across probes, across brain regions, thus indicating the relative local preference of the gene’s expression across brain regions. The Allen Brain Atlas contained six samples of the corpus callosum, four of the central glial substance and one of the cingulum bundle.</p></div><div id="S9" class="sec sec-last"><h3 id="S9title">2.7 Anatomic uniformity of genetic signal across white matter regions</h3><p id="P21" class="p p-first">The question, whether the genetic influence is uniform across the brain’s white matter locations, was approached in several ways. Firstly, explore the extent to which the effects of genome-wide significant SNPs were regionally specific, additional GWAs were performed on FA values in each of the 119,563 voxels within the TBSS skeleton, using the same model as above, for the genome-wide significant SNPs only.</p><p id="P22">Secondly, extended pedigrees enable the computation of genetic correlations (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896324">Almasy et al., 1997</a>), which are a powerful way to quantify the extent to which two traits are influenced by the same additive genetic variation (also see 2.6 above). In contrast to genome-wide association studies, this method is most sensitive to rare genetic variants within pedigrees, rather than to common variation across unrelated individuals. Substantial heritability was regarded as a pre-requisite for the presence of genetic correlations, so only traits with h<sup>2</sup> &gt; 0.2 were considered for further analysis. Next, genetic correlations between global FA and FA within each of the regions were calculated by adding both as traits in a polygenic model in SOLAR, so that the covariance matrix between the two traits is decomposed into genetic and environmental correlations, in addition to covariance due to each of the covariates (age, sex, sex*age, age<sup>2</sup> and sex*age<sup>2</sup>).</p><p id="P23">Thirdly, to be more in line with GWAS’s sensitivity for common genetic variation, we computed Spearman’s rank correlation coefficients between the betas of global FA, and the betas of mean FA within each of the tracts. To minimize the influence of extreme outliers, beta values lower than −3 and higher than 3 (6 for global FA, 1 for the left ALIC, 2 for the right posterior corona radiate, 3 for the right superior cerebellar peduncle, and 1 for the left UF) were excluded for this analysis.</p><p id="P24">Furthermore, because the majority of SNPs across the whole genome are not strongly associated with the phenotype, they can be assumed to randomly vary around 0. For this reason we tested whether SNPs significantly associated with global FA (according to five thresholds: p&lt;0.1, p&lt;0.05, p&lt;0.01, p&lt;0.005, p&lt;0.001) were also more likely to be nominally associated with mean FA in the separate regions. To this end, chi-squared tests were performed between SNPs significant in the global FA GWAS and in each of the sub-region GWASs.</p><p id="P25" class="p p-last">To consider the reverse scenario – to what extent is FA in each of the regions determined by independent genetic variation – polygenic models were run for each region including global FA as a covariate. This estimates the heritability of the variance unique to each region.</p></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">3. Results</h2><div id="S11" class="sec sec-first"><h3 id="S11title">3.1 Heritability of FA</h3><p id="P26" class="p p-first">Consistent with previous reports including a subset of our data (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896339">Jahanshad et al., 2013a</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896258">Kochunov et al., 2010</a>), the global FA measure was significantly heritable with h<sup>2</sup> = 52% (p = 8.66*10<sup>−11</sup>). In the same model, age co-varied significantly with global FA (Χ<sup>2</sup> = 141.64; p = 1.17*10<sup>−32</sup>) as did sex but to a lesser extent (Χ<sup>2</sup> =4.05; p = 0.04), while neither age<sup>2</sup> nor any of the interactions were significant. Together the covariates explained 31% of the variance in FA. Re-running the model without the non-significant higher-order covariates resulted in an almost identical heritability estimate.</p><p id="P27">Similarly, average FA values for all tracts were significantly and substantially heritable (all h<sup>2</sup> &gt; 0.26, all p &lt; 0.0012). Again, in line with previous reports in our sample (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896297">Jahanshad et al., 2013a</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896332">Kochunov et al.; 2010</a>), the highest heritability estimates was found for the right superior longitudinal fasciculus, with h<sup>2</sup>=0.65 (p=8.49*10<sup>−14</sup>) and the splenium of the corpus callosum (h<sup>2</sup>=0.63; p=2.9*10<sup>−14</sup>). Any slight differences in these estimates between the two analyses are due to the use of a sample-specific template in the present study, and a small reduction in sample size because of missing genotyping data. See <a href="#SD3" rid="SD3" class=" supplementary-material">supplementary Table S1</a> for the heritability estimates of FA in each tract separately.</p><p id="P28" class="p p-last"><a href="/pmc/articles/PMC4107465/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> shows the results of a voxel-wise analysis of the heritability of FA. In line with the tract-wise analysis, the highest heritability was found for voxels in the splenium and genu of the corpus callosum, the superior longitudinal fasciculi and in the internal and external capsules. In 58,202 (49%) of the voxels, the heritability estimate of FA was greater than 0.20, and in 7% (8,600 voxels) the heritability estimate was 0.50 or greater.</p></div><div id="S12" class="sec"><h3 id="S12title">3.2 Genome-wide significant SNPs associated with global FA</h3><p id="P29" class="p p-first">GWA analysis of global FA was conducted in SOLAR, taking into account pedigree structure, age, sex, and four multi-dimensional scaling (MDS) components to account for population stratification. The genomic inflation factor was λ=1.03, indicating that any remaining population stratification effects after MDS regression, were negligible. The quantile-quantile (QQ) plot showed that p-values below p = 10<sup>−4</sup> were more significant than predicted by chance (<a href="/pmc/articles/PMC4107465/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). Five SNPs were associated with global FA at the genome-wide significant level (p &lt; 5*10<sup>−8</sup>; <a href="/pmc/articles/PMC4107465/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="/pmc/articles/PMC4107465/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4107465/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139811710011440" class="figure"><a href="/pmc/articles/PMC4107465/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4107465_nihms606427f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms606427f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4107465/bin/nihms606427f2.jpg"></a></div><div id="largeobj_idm139811710011440" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4107465/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4107465/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="P54">Quantile-quantile plot of GWA results on global FA, demonstrating population stratification effects were minimal, with genomic inflation factor λ=1.03.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4107465/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139811710008064" class="figure"><a href="/pmc/articles/PMC4107465/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4107465_nihms606427f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms606427f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4107465/bin/nihms606427f3.jpg"></a></div><div id="largeobj_idm139811710008064" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4107465/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4107465/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="P55">Manhattan plot of GWA results on global FA, showing five SNPs significant at the genome-wide significance level of &lt; 5 ×10<sup>−8</sup>.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P57">Genome-wide significant SNPs on mean FA in the entire TBSS skeleton</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139811719800944" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Χ<sup>2</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">p</th><th rowspan="1" colspan="1" valign="middle" align="center">MAF</th><th rowspan="1" colspan="1" valign="middle" align="left">Chr</th><th rowspan="1" colspan="1" valign="middle" align="left">SNP function</th><th rowspan="1" colspan="1" valign="middle" align="left">Genes<br>(&lt;50kb)</th><th rowspan="1" colspan="1" valign="middle" align="center">HWE Χ<sup>2</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs10853057</td><td rowspan="1" colspan="1" align="center">39.56</td><td rowspan="1" colspan="1" align="center">3.18E-10</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="left">17</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">GNA13</td><td rowspan="1" colspan="1" align="center">2.78</td></tr><tr><td rowspan="1" colspan="1" align="left">rs12249377</td><td rowspan="1" colspan="1" align="center">39.51</td><td rowspan="1" colspan="1" align="center">3.26E-10</td><td rowspan="1" colspan="1" align="center">0.31</td><td rowspan="1" colspan="1" align="left">10</td><td rowspan="1" colspan="1" align="left">Intron</td><td rowspan="1" colspan="1" align="left">HTR7</td><td rowspan="1" colspan="1" align="center">6.32</td></tr><tr><td rowspan="1" colspan="1" align="left">rs258415</td><td rowspan="1" colspan="1" align="center">37.77</td><td rowspan="1" colspan="1" align="center">7.97E-10</td><td rowspan="1" colspan="1" align="center">0.30</td><td rowspan="1" colspan="1" align="left">12</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left"><em>NA</em></td><td rowspan="1" colspan="1" align="center">0.98</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1991867</td><td rowspan="1" colspan="1" align="center">36.03</td><td rowspan="1" colspan="1" align="center">1.94E-09</td><td rowspan="1" colspan="1" align="center">0.40</td><td rowspan="1" colspan="1" align="left">16</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left"><em>NA</em></td><td rowspan="1" colspan="1" align="center">0.40</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1361277</td><td rowspan="1" colspan="1" align="center">31.72</td><td rowspan="1" colspan="1" align="center">2.00E-08</td><td rowspan="1" colspan="1" align="center">0.29</td><td rowspan="1" colspan="1" align="left">1</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left"><em>NA</em></td><td rowspan="1" colspan="1" align="center">1.16</td></tr></tbody></table></div><div id="largeobj_idm139811719800944" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4107465/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P58" class="p p-first-last">MAF = Minor Allele Frequency; HWE = Hardy-Weinberg Equilibrium; Chr = Chromosome</p></div></div></div><p id="P30">Four of the five genome-wide significant SNPs were intergenic (<a href="/pmc/articles/PMC4107465/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), while rs12249377 was located in an intron of serotonin receptor gene <em>HTR7</em>. SNP rs10853057 is located 7kb away from a g-protein coupled receptor gene, <em>GNA13</em>. The other SNPs were further than 50kb from any genes. Scatterplots showing the effects of these SNPs on FA per individual, and mean FA values grouped per genotype, are available in <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary Figure S1</a>. All five SNPs remained genome-wide significant after covarying for history of major depression, bipolar disorder, anxiety disorder, and psychosis (all p &lt; 10<sup>−9</sup>); and after covarying for within-scanner motion (total amount translation and total rotation parameters; all p &lt; 5*10<sup>−8</sup>).</p><p id="P31" class="p p-last">Voxel-wise associations revealed that, for all five top SNPs, the majority of voxels in the TBSS skeleton contributed to the genome-wide signal of global FA: the number of voxels with the same direction of effect (the same allele being associated with increased FA) ranged from 89% (rs1361277) to 91% (rs10853057) of voxels in the white matter skeleton. The voxel-wise results of SNPs rs10853057 and rs258415 are shown in <a href="/pmc/articles/PMC4107465/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>; other voxel-wise maps are available in <a href="#SD2" rid="SD2" class=" supplementary-material">supplementary Figure S2</a>. For all SNPs, the most significantly influenced tracts with p-values &lt; 0.001, were the genu and splenium of the corpus callosum, the internal capsules, parts of the superior longitudinal fasciculus, and parahippocampal white matter.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC4107465/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139811710519664" class="figure"><a href="/pmc/articles/PMC4107465/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4107465_nihms606427f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms606427f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4107465/bin/nihms606427f4.jpg"></a></div><div id="largeobj_idm139811710519664" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4107465/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC4107465/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="P56">Voxel-wise association results for rs10853057 and rs258415 overlayed on top of the mean FA image. Voxels in red-yellow are significantly associated with the SNP at p &lt; 0.001 (uncorrected) in the same direction as the SNP’s association with global FA. To further illustrate the degree of spatial homogeneity of the effect, the remaining voxels (with p &gt; 0.001) are colored blue if the allelic effect was in the same direction as for the global FA measure. Voxels are colored green if the SNP was associated with FA in direction opposite of its effect on global FA.</p></div></div></div></div><div id="S13" class="sec"><h3 id="S13title">3.3 Gene expression</h3><p id="P32" class="p p-first">To infer which genes might be considered responsible for the effects of the genome-wide significant SNPs, we examined lymphocyte-derived gene expression profiles of genes within a 500kb interval on either side of each tag SNPs. Transcript data were available for eleven nearby genes, corresponding to 15 Illumina transcript probes (<a href="/pmc/articles/PMC4107465/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Nine of these transcripts were significantly associated with their respective SNPs (<a href="/pmc/articles/PMC4107465/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) after correcting for the total number of transcripts tested (FDR=0.05).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P59">Effects of genotypes at top SNPs on expression level of nearby genes</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139811719425472" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="center">Transcript</th><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="left">Nearby top SNP</th><th rowspan="1" colspan="1" align="center">h<sup>2</sup></th><th rowspan="1" colspan="1" align="left">P (h<sup>2</sup>)</th><th rowspan="1" colspan="1" align="left">P (SNP)</th><th rowspan="1" colspan="1" align="left">P<sub>FDR</sub> (SNP)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center">ILMN_1676523</td><td rowspan="1" colspan="1" align="left">CCDC91</td><td rowspan="1" colspan="1" align="left">rs258415</td><td rowspan="1" colspan="1" align="center">0.38</td><td rowspan="1" colspan="1" align="left">4.00E-07</td><td rowspan="1" colspan="1" align="left">1.96E-06</td><td rowspan="1" colspan="1" align="left">7.35E-06<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_2190051</td><td rowspan="1" colspan="1" align="left">CCDC91</td><td rowspan="1" colspan="1" align="left">rs258415</td><td rowspan="1" colspan="1" align="center">0.21</td><td rowspan="1" colspan="1" align="left">0.009</td><td rowspan="1" colspan="1" align="left">0.06</td><td rowspan="1" colspan="1" align="left">0.09</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1758906</td><td rowspan="1" colspan="1" align="left">GNA13</td><td rowspan="1" colspan="1" align="left">rs10853057</td><td rowspan="1" colspan="1" align="center">0.42</td><td rowspan="1" colspan="1" align="left">1.00E-07</td><td rowspan="1" colspan="1" align="left">6.18E-11</td><td rowspan="1" colspan="1" align="left">9.27E-10<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_2176037</td><td rowspan="1" colspan="1" align="left">GNA13</td><td rowspan="1" colspan="1" align="left">rs10853057</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="left">0.006</td><td rowspan="1" colspan="1" align="left">7.52E-10</td><td rowspan="1" colspan="1" align="left">5.64E-09<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1744217</td><td rowspan="1" colspan="1" align="left">HTR7</td><td rowspan="1" colspan="1" align="left">rs12249377</td><td rowspan="1" colspan="1" align="center">0.09</td><td rowspan="1" colspan="1" align="left">0.16</td><td rowspan="1" colspan="1" align="left">0.006</td><td rowspan="1" colspan="1" align="left">0.01<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1652732</td><td rowspan="1" colspan="1" align="left">HTR7</td><td rowspan="1" colspan="1" align="left">rs12249377</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="left">0.01</td><td rowspan="1" colspan="1" align="left">0.73</td><td rowspan="1" colspan="1" align="left">0.78</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1727134</td><td rowspan="1" colspan="1" align="left">KLHDC5</td><td rowspan="1" colspan="1" align="left">rs258415</td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="left">0.01</td><td rowspan="1" colspan="1" align="left">0.03</td><td rowspan="1" colspan="1" align="left">0.04<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1781360</td><td rowspan="1" colspan="1" align="left">MPHOSPH6</td><td rowspan="1" colspan="1" align="left">rs1991867</td><td rowspan="1" colspan="1" align="center">0.32</td><td rowspan="1" colspan="1" align="left">1.24E-05</td><td rowspan="1" colspan="1" align="left">0.003</td><td rowspan="1" colspan="1" align="left">0.006<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_2189993</td><td rowspan="1" colspan="1" align="left">MRPS35</td><td rowspan="1" colspan="1" align="left">rs258415</td><td rowspan="1" colspan="1" align="center">0.55</td><td rowspan="1" colspan="1" align="left">4.77E-11</td><td rowspan="1" colspan="1" align="left">0.003</td><td rowspan="1" colspan="1" align="left">0.006<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1788059</td><td rowspan="1" colspan="1" align="left">PCGF5</td><td rowspan="1" colspan="1" align="left">rs12249377</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="left">0.007</td><td rowspan="1" colspan="1" align="left">2.45E-08</td><td rowspan="1" colspan="1" align="left">1.23E-07<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1754468</td><td rowspan="1" colspan="1" align="left">RGS9</td><td rowspan="1" colspan="1" align="left">rs10853057</td><td rowspan="1" colspan="1" align="center">0.26</td><td rowspan="1" colspan="1" align="left">0.003</td><td rowspan="1" colspan="1" align="left">0.06</td><td rowspan="1" colspan="1" align="left">0.09</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_2389984</td><td rowspan="1" colspan="1" align="left">RGS9</td><td rowspan="1" colspan="1" align="left">rs10853057</td><td rowspan="1" colspan="1" align="center">0.20</td><td rowspan="1" colspan="1" align="left">0.002</td><td rowspan="1" colspan="1" align="left">0.28</td><td rowspan="1" colspan="1" align="left">0.32</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1792283</td><td rowspan="1" colspan="1" align="left">RPP30</td><td rowspan="1" colspan="1" align="left">rs12249377</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="left">0.03</td><td rowspan="1" colspan="1" align="left">0.09</td><td rowspan="1" colspan="1" align="left">0.12</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1760798</td><td rowspan="1" colspan="1" align="left">RYR2</td><td rowspan="1" colspan="1" align="left">rs1361277</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="left">0.01</td><td rowspan="1" colspan="1" align="left">0.86</td><td rowspan="1" colspan="1" align="left">0.86</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1675429</td><td rowspan="1" colspan="1" align="left">SMURF2</td><td rowspan="1" colspan="1" align="left">rs10853057</td><td rowspan="1" colspan="1" align="center">0.12</td><td rowspan="1" colspan="1" align="left">0.05</td><td rowspan="1" colspan="1" align="left">0.0002</td><td rowspan="1" colspan="1" align="left">6.89E-04<sup>*</sup></td></tr></tbody></table></div><div id="largeobj_idm139811719425472" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4107465/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><sup>*</sup>Significant at FDR &lt; 0.05</div></div></div><p id="P33">Correlations with FA are reported in <a href="/pmc/articles/PMC4107465/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>. While six transcripts were significantly phenotypically correlated with global FA, two transcripts, one of <em>GNA13</em> and one of <em>CCDC91</em>, showed significant genetic correlations, providing circumstantial evidence that these genes influence global FA. Transcripts of <em>HTR7</em>, <em>PCGF5</em> and <em>SMURF2</em> tended to be less heritable and showed significant phenotypic, but not genotypic, correlations with global FA, suggesting lower measurement reliability and/or shared environmental influences. Both the associations with SNPs and the genetic correlations with FA remained similar when covarying for history of major psychiatric disorders; and after covarying for 6 parameters of within scanner movement (3 for translation and 3 for rotation).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="P61">Genetic correlations between global FA and nine transcripts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139811715716784" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="center">Transcript</th><th rowspan="1" colspan="1" align="center">Gene</th><th rowspan="1" colspan="1" align="center">ρ<sub>p</sub> with FA</th><th rowspan="1" colspan="1" align="center">P (ρ <sub>p</sub>)</th><th rowspan="1" colspan="1" align="center">P<sub>FDR</sub>(ρ<sub>p</sub>)</th><th rowspan="1" colspan="1" align="center">ρ<sub>g</sub> with FA</th><th rowspan="1" colspan="1" align="center">P (ρg)</th><th rowspan="1" colspan="1" align="center">P<sub>FDR</sub> (ρ<sub>g</sub>)<sup>1</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center">ILMN_1676523</td><td rowspan="1" colspan="1" align="center">CCDC91</td><td rowspan="1" colspan="1" align="center">0.21</td><td rowspan="1" colspan="1" align="center">0.0002</td><td rowspan="1" colspan="1" align="center">0.0006</td><td rowspan="1" colspan="1" align="center">0.41</td><td rowspan="1" colspan="1" align="center">0.006</td><td rowspan="1" colspan="1" align="center">0.048<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1758906</td><td rowspan="1" colspan="1" align="center">GNA13</td><td rowspan="1" colspan="1" align="center">0.15</td><td rowspan="1" colspan="1" align="center">0.01</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center">0.33</td><td rowspan="1" colspan="1" align="center">0.42</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_2176037</td><td rowspan="1" colspan="1" align="center">GNA13</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">0.0001</td><td rowspan="1" colspan="1" align="center">0.0005</td><td rowspan="1" colspan="1" align="center">0.42</td><td rowspan="1" colspan="1" align="center">0.01</td><td rowspan="1" colspan="1" align="center">0.048<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1744217</td><td rowspan="1" colspan="1" align="center">HTR7</td><td rowspan="1" colspan="1" align="center">−0.17</td><td rowspan="1" colspan="1" align="center">0.004</td><td rowspan="1" colspan="1" align="center">0.007</td><td rowspan="1" colspan="1" align="center">−0.27</td><td rowspan="1" colspan="1" align="center">0.27</td><td rowspan="1" colspan="1" align="center">0.41</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1727134</td><td rowspan="1" colspan="1" align="center">KLHDC5</td><td rowspan="1" colspan="1" align="center">0.07</td><td rowspan="1" colspan="1" align="center">0.24</td><td rowspan="1" colspan="1" align="center">0.31</td><td rowspan="1" colspan="1" align="center">0.05</td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">0.87</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1781360</td><td rowspan="1" colspan="1" align="center">MPHOSPH6</td><td rowspan="1" colspan="1" align="center">0.04</td><td rowspan="1" colspan="1" align="center">0.52</td><td rowspan="1" colspan="1" align="center">0.58</td><td rowspan="1" colspan="1" align="center">0.26</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">0.31</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_2189993</td><td rowspan="1" colspan="1" align="center">MRPS35</td><td rowspan="1" colspan="1" align="center">0.00</td><td rowspan="1" colspan="1" align="center">0.95</td><td rowspan="1" colspan="1" align="center">0.95</td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center">0.39</td><td rowspan="1" colspan="1" align="center">0.44</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1788059</td><td rowspan="1" colspan="1" align="center">PCGF5</td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">0.003</td><td rowspan="1" colspan="1" align="center">0.007</td><td rowspan="1" colspan="1" align="center">0.22</td><td rowspan="1" colspan="1" align="center">0.21</td><td rowspan="1" colspan="1" align="center">0.37</td></tr><tr><td rowspan="1" colspan="1" align="center">ILMN_1675429</td><td rowspan="1" colspan="1" align="center">SMURF2</td><td rowspan="1" colspan="1" align="center">−0.25</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">−0.40</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">0.31</td></tr></tbody></table></div><div id="largeobj_idm139811715716784" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4107465/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>1</sup>Only transcripts that significantly associated with top SNPs (<a href="/pmc/articles/PMC4107465/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>) were considered; ρ<sub>p</sub> = phenotypic correlation ρ<sub>g</sub> = genotypic correlation;</div><div id="TFN4"><sup>*</sup>Significant at FDR &lt; 0.05</div></div></div><p id="P34" class="p p-last">For each of <em>CCDC91</em> and <em>GNA13</em>, transcript data were available for 3 probes.<sup><sup><a href="#FN3" rid="FN3" class=" fn">1</a></sup></sup> Expression of <em>GNA13</em> was above average for all three probes in the corpus callosum (means across 6 samples: z<sub>probe1</sub>=2.44; z<sub>probe2</sub>=1.74, z<sub>probe3</sub>=1.22), in the cingulum bundle (z<sub>probe1</sub>=2.93; z<sub>probe2</sub>=1.99, z<sub>probe3</sub>=1.78), and in the central glial substance (means across 4 samples: z<sub>probe1</sub>=2.68; z<sub>probe2</sub>=2.00, z<sub>probe3</sub>=2.32). <em>CCDC91</em> is highly expressed in the cingulum bundle (z<sub>probe1</sub>=2.17; z<sub>probe2</sub>=1.72, z<sub>probe3</sub>=2.49), but did not demonstrate a clear preference for the corpus callocum or the cingulum bundles (mean z ranging from −0.04 to 0.70 and from −0.44 to 1,18; respectively).</p></div><div id="S14" class="sec sec-last"><h3 id="S14title">3.4 Homogeneity of additive genetic effects across white matter regions</h3><p id="P35" class="p p-first"><a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table S2</a> contains an overview of the statistics of the homogeneity of genetic effects across white matter regions derived from the Johns Hopkins University Labels atlas.</p><p id="P36">Phenotypic correlations with global FA were very high, between 0.48 and 0.82. Genetic correlations varied between 0.38 (for the left tapetum) and 0.94 (for the left anterior limb of the internal capsule and the corona radiata bilaterally). Of note, 29 of the 31 sub-regions had a genetic correlation of 0.50 or higher with the global FA measure. Similarly, when including global FA as a covariate in polygenic models for each of the tracts, global FA explained between 20% (left tapetum) and 61% (splenium of corpus callosum and left anterior limb of internal capsule) of the variance in each of the sub-regions.</p><p id="P37">Spearman’s rank correlation coefficients between the betas resulting from the GWAs for global FA and for each of the sub-regions were all very high: between 0.42 (left tapetum) and 0.84 (splenium of corpus callosum). Hence the ranks of the betas of global FA explained between 18% and 70% of the rankings of the regional betas. All these correlations were highly significant at p &lt; 2.2*10<sup>−16</sup>), but it should be noted that such genome-wide correlations are statistically over-powered because of the large number of measurement points across the genome.</p><p id="P38">In the five analyses of statistical enrichment for nominally significant SNPs of the global FA GWA within the sets of nominally significant SNPs of each of the regional GWAs(p &lt; 0.1, p &lt; 0.05, p &lt; 0.01, p &lt; 0.005, p &lt; 0.001 ), chi squared statistics ranged from 949 for the left tapetum (at p &lt; 0.001) to 250,452 for FA in the splenium (at p &lt; 0.1), and all were highly significant (all p &lt; 2.2*10<sup>−16</sup>).</p><p id="P39" class="p p-last">Despite this high dependence of the regional measures on global FA, co-varying for global FA in the polygenic models revealed that the unique variance of most regions is also heritable. With h<sup>2</sup> between 0.21 and 0.70 all the regions remained significantly heritable after statistically correcting for global FA. What is more, for some regions the heritability estimate increased, indicating a significant amount of genetic variation is affecting those regions specifically.</p></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">4. Discussion</h2><p id="P40" class="p p-first">We localized five genome-wide significant SNPs for global FA, indicating that variation in FA is sensitive to common genetic variation. Three of these SNPs were associated with nine lymphocyte-derived RNA expression levels from nearby genes, indicating that these variants (or variants in close LD) may regulate the expression of these transcripts. Transcript levels of <em>GNA13</em> and <em>CCDC91</em> were significantly genetically correlated with global FA, suggesting that the protein products of the<em>se</em> genes are involved in processes governing white matter microstructure.</p><p id="P41">Previous studies using <em>a priori</em> selected candidate genes and SNPs have associated FA with genetic variation in <em>DISC1</em> (<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896286">Sprooten et al., 2011b</a>), and <em>BDNF</em> (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896300">Chiang et al., 2011a</a>), amongst others. However, white matter microstructure is a complex trait with a polygenic genetic architecture. To date, few GWA have been performed on DTI-derived traits (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896277">Chiang et al., 2012</a>; <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896273">Jahanshad et al., 2013b</a>; <a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896295">Lopez et al., 2012</a>; <a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896341">Sprooten et al., 2013b</a>). <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896311">Jahanshad et al., (2013b)</a> found a genome-wide significant SNP significant (in <em>SPON1</em>) in association with the number of tractography streamlines between the left posterior cingulate gyrus and superior parietal lobe (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896272">Jahanshad et al., 2013b</a>). In the same sample, <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896263">Chiang et al. (2012)</a> detected 24 SNPs in association with extracted FA from 18 regions of interest, including variants in genes involved in lipid metabolism (<em>LPIN2</em>, <em>HADH</em>) and cell adhesion (<em>OPCML</em>, <em>KAZN</em>).</p><p id="P42">Our top-associated SNP rs10853057 is located only 7kb downstream from the gene encoding the g-protein alpha 13 subunit (<em>GNA13</em>). This SNP lies within in a DNase I hyper-sensitivity region and transcription factor binding site (<a href="http://genome.ucsc.edu/cgi-bin/hgGateway" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.ucsc.edu/cgi-bin/hgGateway</a>; (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896270">Consortium et al., 2012</a>), indicating high regulatory potential. This supports our finding that <em>GNA13</em> transcript levels were significantly associated with both our top SNP and with global FA suggesting that the g-protein alpha 13 subunit mediates an effect of rs10853057 on white matter development or maintenance. There are several conceivable mechanisms through which variation in <em>GNA13</em> could affect white matter microstructure. For example, the g-proteins defined by the alpha 13 and 12 subunits interact directly with RhoGEFs, which in turn interact with RhoA (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896267">Kvachnina et al., 2005</a>; <a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896314">Ponimaskin et al., 2002</a>; <a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896296">Siehler, 2009</a>), which is thought to regulate myelination (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896323">Feltri et al., 2008</a>) and axon growth (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896289">Gross et al., 2007</a>). From a broader perspective, Rho-proteins serve multiple other functions including focal adhesion, and cytoskeleton repair and maintenance (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896279">Dhanasekaran and Dermott, 1996</a>), that could affect white matter organization.</p><p id="P43">The second strongest genome-wide association, rs12249377, was located in an intron of the serotonin receptor 7 gene (<em>HTR7</em>). We also found a significant association between rs12249377 and the expression of one of the <em>HTR7</em> transcripts suggesting that this marker may be detecting cis-acting regulatory effects upon the <em>HTR7</em> gene. However, there was only a phenotypic – but no genotypic – correlation between the transcript and FA, suggesting that <em>HTR7</em> expression influences white matter microstructure, but it is unclear whether this effect is driven by secondary genetic or environmental factors (for example circadian rhythms). Our observation may be due to our lack of power to detect genetic correlation between <em>HTR7</em> expression levels and global FA due to the relatively low heritability of <em>HTR7</em> gene expression in lymphocytes. The serotonin system has been a major interest in behavioral neuroscience and neuropharmacology, and observations of knock-out animals and experiments with specific antagonists have demonstrated that HTR7 is involved in mood regulation, circadian rhythms, and depressive behavior (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896266">Hedlund and Sutcliffe, 2004</a>). Interestingly, HTR7 couples to g-protein alpha s and alpha 12 subunits, the amino acid sequence of the latter being very similar to the alpha 13 subunit, with a similar potential to activate RhoA proteins (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896340">Kvachnina et al., 2005</a>). Thus, <em>HTR7</em> and <em>GNA13</em> share their involvement in a molecular pathway that is critical for axon growth and myelination.</p><p id="P44">FA-associated SNP rs258415 was significantly associated with the expression of a gene 393kb away, namely <em>CCDC91</em>, which was in turn also genetically correlated with FA. Little is known about the function of <em>CCDC91</em>, although several studies indicate that it plays a role in brain structure and pathology. It is highly expressed in the central nervous system and it is involved in sub-cellular trafficking and localization of proteins to the Golgi complex. One study showed that there is an increased incidence of copy-number variants in <em>CCDC1</em> in patients with bipolar disorder (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896299">Chen et al., 2010</a>). Also, its interaction partners, GGA1 and GGA2, have been linked to the Alzheimer pathophysiology by interacting with beta-amyloid precursors (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896271">Herskowitz et al., 2012</a>).</p><p id="P45">It may be surprising that our top SNPs do not implicate existing candidate genes for white matter structure specifically. None of the genes identified in the present paper would have been hypothesized as being associated with FA <em>a priori</em>. However, a review of other GWA studies of complex traits suggests that firstly, the absence of candidate genes in the results is the rule rather than the exception (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896259">Flint and Munafo, 2013</a>; <a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896293">Seifuddin et al., 2012</a>), and secondly the representation of intergenic regions in the results has been unexpectedly large (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896284">Hindorff et al., 2009</a>). These two observations from GWA studies are likely due to several factors. Firstly, the polygenic nature of complex traits and the extremely high statistical penalty for multiple testing across the genome undoubtedly leads to a large number of unreported false negative SNPs, which likely include some previously hypothesized variants in candidate genes. Secondly, candidate genes are often based on animal experiments or linkage studies which are more suited to detecting rare variation of large effects. Third, the <em>a priori</em> information is incomplete, with many poorly characterized genes, regions transcribing non-coding RNAs that can regulate expression (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896338">Maurano et al., 2012</a>), and epigenetic effects of chromatin structure (<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896326">Mitchell et al., 2013</a>).</p><p id="P46">A well-known, but important limitation of GWA is that it is not sensitive to rare genetic variation. In addition, replication of top GWA hits across studies has proven difficult, and even more so across populations (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896317">Ioannidis, 2007</a>; <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896285">Johnson et al., 2011</a>). We were unable to acquire an independent replication sample from the same ancestry, since large imaging genetics samples of Mexican-American ethnicity are extremely rare to date. However, the observed correlations of transcript levels with top SNPs and global FA provided an internal replication, from independently acquired data, for the associated SNPs or their LD blocks in the present sample.</p><p id="P47" class="p p-last">Another limitation of the present study is that it is impossible to directly measure expression from brain tissue <em>in vivo</em>. Here, gene expression data were derived from lymphocytes, and the extent to which these transcripts are similarly expressed in the brain is difficult to ascertain (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390896334">Cai et al., 2010</a>). Since the correspondence of gene expression in distal brain tissue varies depending on the gene, we confirmed the presence of the transcripts of interest in a tissue specific expression atlas (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_482631780">Hawrylycz et al., 2012</a>).</p></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">5. Conclusions</h2><p id="P48" class="p p-first-last">Our estimate of global FA is a sensitive quantitative phenotype for genetic analysis of common genetic variation. In a large GWAS of 776 individuals of Mexican-American ancestry, we detected five genome-wide significant SNPs that may influence white matter microstructure in the general population. Analysis of gene transcript levels suggest a potential mechanism for the effects of two SNPs on white matter microstructure, via the expression of <em>GNA13</em> and <em>CCDC91</em>, that encourage further investigation into the biological functions of these genes and their relation to brain structure and function in health and disease.</p></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="idm139811715000256title">Highlights</h3></div><ul class="unordered" style="list-style-type:disc" id="L1"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="P64">Five genome-wide significant SNPs were detected in association with FA</p></li><li><p id="P65"><em>HTR7</em> expression is associated with FA and with intronic genome-wide significant hit</p></li><li><p id="P66"><em>GNA13</em> and <em>CCDC91</em> expression may mediate effects of genome-wide significant SNPs</p></li><li><p id="P67">Additive genetic effects on FA are largely homogeneous across white matter tracts</p></li></ul></div></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d36e1165"><a href="/pmc/articles/PMC4107465/bin/NIHMS606427-supplement-1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(34K, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><div class="sup-box half_rhythm" id="d36e1169"><a href="/pmc/articles/PMC4107465/bin/NIHMS606427-supplement-2.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(628K, pdf)</sup></div></div><div class="sec suppmat" id="SD3"><h4>3</h4><div class="sup-box half_rhythm" id="d36e1173"><a href="/pmc/articles/PMC4107465/bin/NIHMS606427-supplement-3.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(17K, docx)</sup></div></div><div class="sec suppmat" id="SD4"><h4>4</h4><div class="sup-box half_rhythm" id="d36e1177"><a href="/pmc/articles/PMC4107465/bin/NIHMS606427-supplement-4.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(21K, docx)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgements</h2><div class="sec"><p id="P52">Financial support for this study was provided by the National Institute of Mental Health Grants MH0708143 (Principal Investigator [PI]: DCG), <a id="__tag_786357375" class="tag_hotlink" href="/nuccore/MH078111" ref="/nuccore/MH078111"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH078111","term_id":"1365233409","term_text":"MH078111"}}</span>MH078111</a> (PI: JB), and <a id="__tag_786357374" class="tag_hotlink" href="/nuccore/MH083824" ref="/nuccore/MH083824"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH083824","term_id":"1365501178","term_text":"MH083824"}}</span>MH083824</a> (PI: DCG &amp; JB). SOLAR is supported by National Institute of Mental Health Grant MH59490 (PI: JB) ) and by NIBIB grant EB015611 (PI: PK).</p></div></div><div id="idm139811715268688" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139811715268688title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P49" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p><p class="fn sec" id="FN2"> </p><p id="P50" class="p p-first-last"><strong>Conflicts of Interest:</strong> The authors declare no competing financial interests.</p><p></p><p class="fn sec" id="FN3"><sup>1</sup>These data can be downloaded and visualized at the following 2 links, For GNA13 and CCDC91 respectively: <a href="http://human.brain-map.org/microarray/search/show?exact_match=false&amp;search_term=GNA13&amp;search_type=gene" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://human.brain-map.org/microarray/search/show?exact_match=false&amp;search_term=GNA13&amp;search_type=gene</a>
<a href="http://human.brain-map.org/microarray/search/show?exact_match=false&amp;search_term=ccdc91&amp;search_type=gene" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://human.brain-map.org/microarray/search/show?exact_match=false&amp;search_term=ccdc91&amp;search_type=gene</a></p></div></div><div id="idm139811720489792" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139811720489792title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Almasy L, Blangero J. Multipoint quantitative-trait linkage analysis in general pedigrees. <span><span class="ref-journal">Am J Hum Genet. </span>1998;<span class="ref-vol">62</span>:1198–1211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377101/">PMC free article</a>]</span> [<a href="/pubmed/9545414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Multipoint+quantitative-trait+linkage+analysis+in+general+pedigrees&amp;author=L+Almasy&amp;author=J+Blangero&amp;volume=62&amp;publication_year=1998&amp;pages=1198-1211&amp;pmid=9545414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Almasy L, Dyer TD, Blangero J. Bivariate quantitative trait linkage analysis: pleiotropy versus co-incident linkages. <span><span class="ref-journal">Genet Epidemiol. </span>1997;<span class="ref-vol">14</span>:953–958.</span> [<a href="/pubmed/9433606" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Bivariate+quantitative+trait+linkage+analysis:+pleiotropy+versus+co-incident+linkages&amp;author=L+Almasy&amp;author=TD+Dyer&amp;author=J+Blangero&amp;volume=14&amp;publication_year=1997&amp;pages=953-958&amp;pmid=9433606&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Beaulieu C. The basis of anisotropic water diffusion in the nervous system - a technical review. <span><span class="ref-journal">NMR Biomed. </span>2002;<span class="ref-vol">15</span>:435–455.</span> [<a href="/pubmed/12489094" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=NMR+Biomed&amp;title=The+basis+of+anisotropic+water+diffusion+in+the+nervous+system+-+a+technical+review&amp;author=C+Beaulieu&amp;volume=15&amp;publication_year=2002&amp;pages=435-455&amp;pmid=12489094&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Behrens TE, Woolrich MW, Jenkinson M, Johansen-Berg H, Nunes RG, Clare S, Matthews PM, Brady JM, Smith SM. Characterization and propagation of uncertainty in diffusion-weighted MR imaging. <span><span class="ref-journal">Magn Reson Med. </span>2003;<span class="ref-vol">50</span>:1077–1088.</span> [<a href="/pubmed/14587019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Magn+Reson+Med&amp;title=Characterization+and+propagation+of+uncertainty+in+diffusion-weighted+MR+imaging&amp;author=TE+Behrens&amp;author=MW+Woolrich&amp;author=M+Jenkinson&amp;author=H+Johansen-Berg&amp;author=RG+Nunes&amp;volume=50&amp;publication_year=2003&amp;pages=1077-1088&amp;pmid=14587019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Blangero J, Diego VP, Dyer TD, Almeida M, Peralta J, Kent JW, Jr, Williams JT, Almasy L, Goring HH. A kernel of truth: statistical advances in polygenic variance component models for complex human pedigrees. <span><span class="ref-journal">Adv Genet. </span>2013;<span class="ref-vol">81</span>:1–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4019427/">PMC free article</a>]</span> [<a href="/pubmed/23419715" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv+Genet&amp;title=A+kernel+of+truth:+statistical+advances+in+polygenic+variance+component+models+for+complex+human+pedigrees&amp;author=J+Blangero&amp;author=VP+Diego&amp;author=TD+Dyer&amp;author=M+Almeida&amp;author=J+Peralta&amp;volume=81&amp;publication_year=2013&amp;pages=1-31&amp;pmid=23419715&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Braskie MN, Jahanshad N, Stein JL, Barysheva M, Johnson K, McMahon KL, de Zubicaray GI, Martin NG, Wright MJ, Ringman JM, Toga AW, Thompson PM. Relationship of a variant in the NTRK1 gene to white matter microstructure in young adults. <span><span class="ref-journal">J Neurosci. </span>2012;<span class="ref-vol">32</span>:5964–5972.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3393752/">PMC free article</a>]</span> [<a href="/pubmed/22539856" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Relationship+of+a+variant+in+the+NTRK1+gene+to+white+matter+microstructure+in+young+adults&amp;author=MN+Braskie&amp;author=N+Jahanshad&amp;author=JL+Stein&amp;author=M+Barysheva&amp;author=K+Johnson&amp;volume=32&amp;publication_year=2012&amp;pages=5964-5972&amp;pmid=22539856&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Burdick JT, Chen WM, Abecasis GR, Cheung VG. In silico method for inferring genotypes in pedigrees. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:1002–1004.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3005330/">PMC free article</a>]</span> [<a href="/pubmed/16921375" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=In+silico+method+for+inferring+genotypes+in+pedigrees&amp;author=JT+Burdick&amp;author=WM+Chen&amp;author=GR+Abecasis&amp;author=VG+Cheung&amp;volume=38&amp;publication_year=2006&amp;pages=1002-1004&amp;pmid=16921375&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Cai C, Langfelder P, Fuller TF, Oldham MC, Luo R, van den Berg LH, Ophoff RA, Horvath S. Is human blood a good surrogate for brain tissue in transcriptional studies? <span><span class="ref-journal">BMC Genomics. </span>2010;<span class="ref-vol">11</span>:589.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3091510/">PMC free article</a>]</span> [<a href="/pubmed/20961428" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Is+human+blood+a+good+surrogate+for+brain+tissue+in+transcriptional+studies?&amp;author=C+Cai&amp;author=P+Langfelder&amp;author=TF+Fuller&amp;author=MC+Oldham&amp;author=R+Luo&amp;volume=11&amp;publication_year=2010&amp;pages=589&amp;pmid=20961428&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Chen X, Li X, Wang P, Liu Y, Zhang Z, Zhao G, Xu H, Zhu J, Qin X, Chen S, Hu L, Kong X. Novel association strategy with copy number variation for identifying new risk Loci of human diseases. <span><span class="ref-journal">PLoS One. </span>2010;<span class="ref-vol">5</span>:e12185.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2924882/">PMC free article</a>]</span> [<a href="/pubmed/20808825" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Novel+association+strategy+with+copy+number+variation+for+identifying+new+risk+Loci+of+human+diseases&amp;author=X+Chen&amp;author=X+Li&amp;author=P+Wang&amp;author=Y+Liu&amp;author=Z+Zhang&amp;volume=5&amp;publication_year=2010&amp;pages=e12185&amp;pmid=20808825&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Chiang MC, Barysheva M, McMahon KL, de Zubicaray GI, Johnson K, Montgomery GW, Martin NG, Toga AW, Wright MJ, Shapshak P, Thompson PM. Gene network effects on brain microstructure and intellectual performance identified in 472 twins. <span><span class="ref-journal">J Neurosci. </span>2012;<span class="ref-vol">32</span>:8732–8745.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3420968/">PMC free article</a>]</span> [<a href="/pubmed/22723713" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Gene+network+effects+on+brain+microstructure+and+intellectual+performance+identified+in+472+twins&amp;author=MC+Chiang&amp;author=M+Barysheva&amp;author=KL+McMahon&amp;author=GI+de+Zubicaray&amp;author=K+Johnson&amp;volume=32&amp;publication_year=2012&amp;pages=8732-8745&amp;pmid=22723713&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Chiang MC, Barysheva M, Toga AW, Medland SE, Hansell NK, James MR, McMahon KL, de Zubicaray GI, Martin NG, Wright MJ, Thompson PM. BDNF gene effects on brain circuitry replicated in 455 twins. <span><span class="ref-journal">Neuroimage. </span>2011a;<span class="ref-vol">55</span>:448–454.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3192852/">PMC free article</a>]</span> [<a href="/pubmed/21195196" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=BDNF+gene+effects+on+brain+circuitry+replicated+in+455+twins&amp;author=MC+Chiang&amp;author=M+Barysheva&amp;author=AW+Toga&amp;author=SE+Medland&amp;author=NK+Hansell&amp;volume=55&amp;publication_year=2011a&amp;pages=448-454&amp;pmid=21195196&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Chiang MC, McMahon KL, de Zubicaray GI, Martin NG, Hickie I, Toga AW, Wright MJ, Thompson PM. Genetics of white matter development: a DTI study of 705 twins and their siblings aged 12 to 29. <span><span class="ref-journal">Neuroimage. </span>2011b;<span class="ref-vol">54</span>:2308–2317.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3197836/">PMC free article</a>]</span> [<a href="/pubmed/20950689" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Genetics+of+white+matter+development:+a+DTI+study+of+705+twins+and+their+siblings+aged+12+to+29&amp;author=MC+Chiang&amp;author=KL+McMahon&amp;author=GI+de+Zubicaray&amp;author=NG+Martin&amp;author=I+Hickie&amp;volume=54&amp;publication_year=2011b&amp;pages=2308-2317&amp;pmid=20950689&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Consortium EP, Bernstein BE, Birney E, Dunham I, Green ED, Gunter C, Snyder M. An integrated encyclopedia of DNA elements in the human genome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:57–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3439153/">PMC free article</a>]</span> [<a href="/pubmed/22955616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+encyclopedia+of+DNA+elements+in+the+human+genome&amp;author=EP+Consortium&amp;author=BE+Bernstein&amp;author=E+Birney&amp;author=I+Dunham&amp;author=ED+Green&amp;volume=489&amp;publication_year=2012&amp;pages=57-74&amp;pmid=22955616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Dhanasekaran N, Dermott JM. Signaling by the G12 class of G proteins. <span><span class="ref-journal">Cell Signal. </span>1996;<span class="ref-vol">8</span>:235–245.</span> [<a href="/pubmed/8842523" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Signal&amp;title=Signaling+by+the+G12+class+of+G+proteins&amp;author=N+Dhanasekaran&amp;author=JM+Dermott&amp;volume=8&amp;publication_year=1996&amp;pages=235-245&amp;pmid=8842523&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Feltri ML, Suter U, Relvas JB. The function of RhoGTPases in axon ensheathment and myelination. <span><span class="ref-journal">Glia. </span>2008;<span class="ref-vol">56</span>:1508–1517.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2615182/">PMC free article</a>]</span> [<a href="/pubmed/18803320" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glia&amp;title=The+function+of+RhoGTPases+in+axon+ensheathment+and+myelination&amp;author=ML+Feltri&amp;author=U+Suter&amp;author=JB+Relvas&amp;volume=56&amp;publication_year=2008&amp;pages=1508-1517&amp;pmid=18803320&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Flint J, Munafo MR. Candidate and non-candidate genes in behavior genetics. <span><span class="ref-journal">Curr Opin Neurobiol. </span>2013;<span class="ref-vol">23</span>:57–61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3752971/">PMC free article</a>]</span> [<a href="/pubmed/22878161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Neurobiol&amp;title=Candidate+and+non-candidate+genes+in+behavior+genetics&amp;author=J+Flint&amp;author=MR+Munafo&amp;volume=23&amp;publication_year=2013&amp;pages=57-61&amp;pmid=22878161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Glahn DC, Curran JE, Winkler AM, Carless MA, Kent JW, Jr, Charlesworth JC, Johnson MP, Goring HH, Cole SA, Dyer TD, Moses EK, Olvera RL, Kochunov P, Duggirala R, Fox PT, Almasy L, Blangero J. High dimensional endophenotype ranking in the search for major depression risk genes. <span><span class="ref-journal">Biol Psychiatry. </span>2012;<span class="ref-vol">71</span>:6–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3230692/">PMC free article</a>]</span> [<a href="/pubmed/21982424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=High+dimensional+endophenotype+ranking+in+the+search+for+major+depression+risk+genes&amp;author=DC+Glahn&amp;author=JE+Curran&amp;author=AM+Winkler&amp;author=MA+Carless&amp;author=JW+Kent&amp;volume=71&amp;publication_year=2012&amp;pages=6-14&amp;pmid=21982424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Gold BT, Johnson NF, Powell DK, Smith CD. White matter integrity and vulnerability to Alzheimer's disease: preliminary findings and future directions. <span><span class="ref-journal">Biochim Biophys Acta. </span>2012;<span class="ref-vol">1822</span>:416–422.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3213315/">PMC free article</a>]</span> [<a href="/pubmed/21803153" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=White+matter+integrity+and+vulnerability+to+Alzheimer's+disease:+preliminary+findings+and+future+directions&amp;author=BT+Gold&amp;author=NF+Johnson&amp;author=DK+Powell&amp;author=CD+Smith&amp;volume=1822&amp;publication_year=2012&amp;pages=416-422&amp;pmid=21803153&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Goring HH, Curran JE, Johnson MP, Dyer TD, Charlesworth J, Cole SA, Jowett JB, Abraham LJ, Rainwater DL, Comuzzie AG, Mahaney MC, Almasy L, MacCluer JW, Kissebah AH, Collier GR, Moses EK, Blangero J. Discovery of expression QTLs using large-scale transcriptional profiling in human lymphocytes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1208–1216.</span> [<a href="/pubmed/17873875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Discovery+of+expression+QTLs+using+large-scale+transcriptional+profiling+in+human+lymphocytes&amp;author=HH+Goring&amp;author=JE+Curran&amp;author=MP+Johnson&amp;author=TD+Dyer&amp;author=J+Charlesworth&amp;volume=39&amp;publication_year=2007&amp;pages=1208-1216&amp;pmid=17873875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Gross RE, Mei Q, Gutekunst CA, Torre E. The pivotal role of RhoA GTPase in the molecular signaling of axon growth inhibition after CNS injury and targeted therapeutic strategies. <span><span class="ref-journal">Cell Transplant. </span>2007;<span class="ref-vol">16</span>:245–262.</span> [<a href="/pubmed/17503736" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Transplant&amp;title=The+pivotal+role+of+RhoA+GTPase+in+the+molecular+signaling+of+axon+growth+inhibition+after+CNS+injury+and+targeted+therapeutic+strategies&amp;author=RE+Gross&amp;author=Q+Mei&amp;author=CA+Gutekunst&amp;author=E+Torre&amp;volume=16&amp;publication_year=2007&amp;pages=245-262&amp;pmid=17503736&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Hawrylycz MJ, Lein ES, Guillozet-Bongaarts AL, Shen EH, Ng L, Miller JA, van de Lagemaat LN, Smith KA, Ebbert A, Riley ZL, Abajian C, Beckmann CF, Bernard A, Bertagnolli D, Boe AF, Cartagena PM, Chakravarty MM, Chapin M, Chong J, Dalley RA, Daly BD, Dang C, Datta S, Dee N, Dolbeare TA, Faber V, Feng D, Fowler DR, Goldy J, Gregor BW, Haradon Z, Haynor DR, Hohmann JG, Horvath S, Howard RE, Jeromin A, Jochim JM, Kinnunen M, Lau C, Lazarz ET, Lee C, Lemon TA, Li L, Li Y, Morris JA, Overly CC, Parker PD, Parry SE, Reding M, Royall JJ, Schulkin J, Sequeira PA, Slaughterbeck CR, Smith SC, Sodt AJ, Sunkin SM, Swanson BE, Vawter MP, Williams D, Wohnoutka P, Zielke HR, Geschwind DH, Hof PR, Smith SM, Koch C, Grant SG, Jones AR. An anatomically comprehensive atlas of the adult human brain transcriptome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:391–399.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4243026/">PMC free article</a>]</span> [<a href="/pubmed/22996553" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+anatomically+comprehensive+atlas+of+the+adult+human+brain+transcriptome&amp;author=MJ+Hawrylycz&amp;author=ES+Lein&amp;author=AL+Guillozet-Bongaarts&amp;author=EH+Shen&amp;author=L+Ng&amp;volume=489&amp;publication_year=2012&amp;pages=391-399&amp;pmid=22996553&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Hedlund PB, Sutcliffe JG. Functional, molecular and pharmacological advances in 5-HT7 receptor research. <span><span class="ref-journal">Trends Pharmacol Sci. </span>2004;<span class="ref-vol">25</span>:481–486.</span> [<a href="/pubmed/15559250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Pharmacol+Sci&amp;title=Functional,+molecular+and+pharmacological+advances+in+5-HT7+receptor+research&amp;author=PB+Hedlund&amp;author=JG+Sutcliffe&amp;volume=25&amp;publication_year=2004&amp;pages=481-486&amp;pmid=15559250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Herskowitz JH, Offe K, Deshpande A, Kahn RA, Levey AI, Lah JJ. GGA1-mediated endocytic traffic of LR11/SorLA alters APP intracellular distribution and amyloid-beta production. <span><span class="ref-journal">Mol Biol Cell. </span>2012;<span class="ref-vol">23</span>:2645–2657.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3395654/">PMC free article</a>]</span> [<a href="/pubmed/22621900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Cell&amp;title=GGA1-mediated+endocytic+traffic+of+LR11/SorLA+alters+APP+intracellular+distribution+and+amyloid-beta+production&amp;author=JH+Herskowitz&amp;author=K+Offe&amp;author=A+Deshpande&amp;author=RA+Kahn&amp;author=AI+Levey&amp;volume=23&amp;publication_year=2012&amp;pages=2645-2657&amp;pmid=22621900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Hindorff LA, Sethupathy P, Junkins HA, Ramos EM, Mehta JP, Collins FS, Manolio TA. Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2009;<span class="ref-vol">106</span>:9362–9367.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687147/">PMC free article</a>]</span> [<a href="/pubmed/19474294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Potential+etiologic+and+functional+implications+of+genome-wide+association+loci+for+human+diseases+and+traits&amp;author=LA+Hindorff&amp;author=P+Sethupathy&amp;author=HA+Junkins&amp;author=EM+Ramos&amp;author=JP+Mehta&amp;volume=106&amp;publication_year=2009&amp;pages=9362-9367&amp;pmid=19474294&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Hirschhorn JN, Daly MJ. Genome-wide association studies for common diseases and complex traits. <span><span class="ref-journal">Nat Rev Genet. </span>2005;<span class="ref-vol">6</span>:95–108.</span> [<a href="/pubmed/15716906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+for+common+diseases+and+complex+traits&amp;author=JN+Hirschhorn&amp;author=MJ+Daly&amp;volume=6&amp;publication_year=2005&amp;pages=95-108&amp;pmid=15716906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Hoptman MJ, Nierenberg J, Bertisch HC, Catalano D, Ardekani BA, Branch CA, Delisi LE. A DTI study of white matter microstructure in individuals at high genetic risk for schizophrenia. <span><span class="ref-journal">Schizophr Res. </span>2008;<span class="ref-vol">106</span>:115–124.</span> [<a href="/pubmed/18804959" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Schizophr+Res&amp;title=A+DTI+study+of+white+matter+microstructure+in+individuals+at+high+genetic+risk+for+schizophrenia&amp;author=MJ+Hoptman&amp;author=J+Nierenberg&amp;author=HC+Bertisch&amp;author=D+Catalano&amp;author=BA+Ardekani&amp;volume=106&amp;publication_year=2008&amp;pages=115-124&amp;pmid=18804959&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">Ioannidis JP. Non-replication and inconsistency in the genome-wide association setting. <span><span class="ref-journal">Hum Hered. </span>2007;<span class="ref-vol">64</span>:203–213.</span> [<a href="/pubmed/17551261" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Non-replication+and+inconsistency+in+the+genome-wide+association+setting&amp;author=JP+Ioannidis&amp;volume=64&amp;publication_year=2007&amp;pages=203-213&amp;pmid=17551261&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Jahanshad N, Kochunov PV, Sprooten E, Mandl RC, Nichols TE, Almasy L, Blangero J, Brouwer RM, Curran JE, de Zubicaray GI, Duggirala R, Fox PT, Hong LE, Landman BA, Martin NG, McMahon KL, Medland SE, Mitchell BD, Olvera RL, Peterson CP, Starr JM, Sussmann JE, Toga AW, Wardlaw JM, Wright MJ, Hulshoff Pol HE, Bastin ME, McIntosh AM, Deary IJ, Thompson PM, Glahn DC. Multi-site genetic analysis of diffusion images and voxelwise heritability analysis: A pilot project of the ENIGMA-DTI working group. <span><span class="ref-journal">Neuroimage. </span>2013a;<span class="ref-vol">81</span>:455–469.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3729717/">PMC free article</a>]</span> [<a href="/pubmed/23629049" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Multi-site+genetic+analysis+of+diffusion+images+and+voxelwise+heritability+analysis:+A+pilot+project+of+the+ENIGMA-DTI+working+group&amp;author=N+Jahanshad&amp;author=PV+Kochunov&amp;author=E+Sprooten&amp;author=RC+Mandl&amp;author=TE+Nichols&amp;volume=81&amp;publication_year=2013a&amp;pages=455-469&amp;pmid=23629049&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Jahanshad N, Rajagopalan P, Hua X, Hibar DP, Nir TM, Toga AW, Jack CR, Jr, Saykin AJ, Green RC, Weiner MW, Medland SE, Montgomery GW, Hansell NK, McMahon KL, de Zubicaray GI, Martin NG, Wright MJ, Thompson PM, Alzheimer's Disease Neuroimaging I. Genome-wide scan of healthy human connectome discovers SPON1 gene variant influencing dementia severity. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2013b;<span class="ref-vol">110</span>:4768–4773.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3606977/">PMC free article</a>]</span> [<a href="/pubmed/23471985" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Genome-wide+scan+of+healthy+human+connectome+discovers+SPON1+gene+variant+influencing+dementia+severity&amp;author=N+Jahanshad&amp;author=P+Rajagopalan&amp;author=X+Hua&amp;author=DP+Hibar&amp;author=TM+Nir&amp;volume=110&amp;publication_year=2013b&amp;pages=4768-4773&amp;pmid=23471985&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">Jahanshad N, Valcour VG, Nir TM, Kohannim O, Busovaca E, Nicolas K, Thompson P. Disrupted brain networks in the aging HIV+ population. <span><span class="ref-journal">Brain Connect. </span>2012</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3621327/">PMC free article</a>]</span> [<a href="/pubmed/23240599" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Connect&amp;title=Disrupted+brain+networks+in+the+aging+HIV++population&amp;author=N+Jahanshad&amp;author=VG+Valcour&amp;author=TM+Nir&amp;author=O+Kohannim&amp;author=E+Busovaca&amp;publication_year=2012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="element-citation">Johnson C, Drgon T, Walther D, Uhl GR. Genomic regions identified by overlapping clusters of nominally-positive SNPs from genome-wide studies of alcohol and illegal substance dependence. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>:e19210.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3144872/">PMC free article</a>]</span> [<a href="/pubmed/21818250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Genomic+regions+identified+by+overlapping+clusters+of+nominally-positive+SNPs+from+genome-wide+studies+of+alcohol+and+illegal+substance+dependence&amp;author=C+Johnson&amp;author=T+Drgon&amp;author=D+Walther&amp;author=GR+Uhl&amp;volume=6&amp;publication_year=2011&amp;pages=e19210&amp;pmid=21818250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="element-citation">Kochunov P, Glahn DC, Lancaster JL, Winkler AM, Smith S, Thompson PM, Almasy L, Duggirala R, Fox PT, Blangero J. Genetics of microstructure of cerebral white matter using diffusion tensor imaging. <span><span class="ref-journal">Neuroimage. </span>2010;<span class="ref-vol">53</span>:1109–1116.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2888778/">PMC free article</a>]</span> [<a href="/pubmed/20117221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Genetics+of+microstructure+of+cerebral+white+matter+using+diffusion+tensor+imaging&amp;author=P+Kochunov&amp;author=DC+Glahn&amp;author=JL+Lancaster&amp;author=AM+Winkler&amp;author=S+Smith&amp;volume=53&amp;publication_year=2010&amp;pages=1109-1116&amp;pmid=20117221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="element-citation">Kochunov P, Thompson PM, Lancaster JL, Bartzokis G, Smith S, Coyle T, Royall DR, Laird A, Fox PT. Relationship between white matter fractional anisotropy and other indices of cerebral health in normal aging: tract-based spatial statistics study of aging. <span><span class="ref-journal">Neuroimage. </span>2007;<span class="ref-vol">35</span>:478–487.</span> [<a href="/pubmed/17292629" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Relationship+between+white+matter+fractional+anisotropy+and+other+indices+of+cerebral+health+in+normal+aging:+tract-based+spatial+statistics+study+of+aging&amp;author=P+Kochunov&amp;author=PM+Thompson&amp;author=JL+Lancaster&amp;author=G+Bartzokis&amp;author=S+Smith&amp;volume=35&amp;publication_year=2007&amp;pages=478-487&amp;pmid=17292629&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="element-citation">Konrad A, Vucurevic G, Musso F, Stoeter P, Dahmen N, Winterer G. ErbB4 genotype predicts left frontotemporal structural connectivity in human brain. <span><span class="ref-journal">Neuropsychopharmacology. </span>2009;<span class="ref-vol">34</span>:641–650.</span> [<a href="/pubmed/18668031" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&amp;title=ErbB4+genotype+predicts+left+frontotemporal+structural+connectivity+in+human+brain&amp;author=A+Konrad&amp;author=G+Vucurevic&amp;author=F+Musso&amp;author=P+Stoeter&amp;author=N+Dahmen&amp;volume=34&amp;publication_year=2009&amp;pages=641-650&amp;pmid=18668031&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="element-citation">Kubicki M, McCarley R, Westin CF, Park HJ, Maier S, Kikinis R, Jolesz FA, Shenton ME. A review of diffusion tensor imaging studies in schizophrenia. <span><span class="ref-journal">J Psychiatr Res. </span>2007;<span class="ref-vol">41</span>:15–30.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2768134/">PMC free article</a>]</span> [<a href="/pubmed/16023676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Psychiatr+Res&amp;title=A+review+of+diffusion+tensor+imaging+studies+in+schizophrenia&amp;author=M+Kubicki&amp;author=R+McCarley&amp;author=CF+Westin&amp;author=HJ+Park&amp;author=S+Maier&amp;volume=41&amp;publication_year=2007&amp;pages=15-30&amp;pmid=16023676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="element-citation">Kvachnina E, Liu G, Dityatev A, Renner U, Dumuis A, Richter DW, Dityateva G, Schachner M, Voyno-Yasenetskaya TA, Ponimaskin EG. 5-HT7 receptor is coupled to G alpha subunits of heterotrimeric G12-protein to regulate gene transcription and neuronal morphology. <span><span class="ref-journal">J Neurosci. </span>2005;<span class="ref-vol">25</span>:7821–7830.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6725246/">PMC free article</a>]</span> [<a href="/pubmed/16120784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=5-HT7+receptor+is+coupled+to+G+alpha+subunits+of+heterotrimeric+G12-protein+to+regulate+gene+transcription+and+neuronal+morphology&amp;author=E+Kvachnina&amp;author=G+Liu&amp;author=A+Dityatev&amp;author=U+Renner&amp;author=A+Dumuis&amp;volume=25&amp;publication_year=2005&amp;pages=7821-7830&amp;pmid=16120784&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="element-citation">Li J, Li XY, Feng DF, Gu L. Quantitative evaluation of microscopic injury with diffusion tensor imaging in a rat model of diffuse axonal injury. <span><span class="ref-journal">Eur J Neurosci. </span>2011;<span class="ref-vol">33</span>:933–945.</span> [<a href="/pubmed/21385236" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Neurosci&amp;title=Quantitative+evaluation+of+microscopic+injury+with+diffusion+tensor+imaging+in+a+rat+model+of+diffuse+axonal+injury&amp;author=J+Li&amp;author=XY+Li&amp;author=DF+Feng&amp;author=L+Gu&amp;volume=33&amp;publication_year=2011&amp;pages=933-945&amp;pmid=21385236&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="element-citation">Lopez LM, Bastin ME, Maniega SM, Penke L, Davies G, Christoforou A, Valdes Hernandez MC, Royle NA, Tenesa A, Starr JM, Porteous DJ, Wardlaw JM, Deary IJ. A genome-wide search for genetic influences and biological pathways related to the brain's white matter integrity. <span><span class="ref-journal">Neurobiol Aging. </span>2012;<span class="ref-vol">33</span>(1847):e1841–e1814.</span> [<a href="/pubmed/22425255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol+Aging&amp;title=A+genome-wide+search+for+genetic+influences+and+biological+pathways+related+to+the+brain's+white+matter+integrity&amp;author=LM+Lopez&amp;author=ME+Bastin&amp;author=SM+Maniega&amp;author=L+Penke&amp;author=G+Davies&amp;volume=33&amp;issue=1847&amp;publication_year=2012&amp;pages=e1841-e1814&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R39"><span class="element-citation">Mahon K, Burdick KE, Szeszko PR. A role for white matter abnormalities in the pathophysiology of bipolar disorder. <span><span class="ref-journal">Neurosci Biobehav Rev. </span>2010;<span class="ref-vol">34</span>:533–554.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2847441/">PMC free article</a>]</span> [<a href="/pubmed/19896972" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurosci+Biobehav+Rev&amp;title=A+role+for+white+matter+abnormalities+in+the+pathophysiology+of+bipolar+disorder&amp;author=K+Mahon&amp;author=KE+Burdick&amp;author=PR+Szeszko&amp;volume=34&amp;publication_year=2010&amp;pages=533-554&amp;pmid=19896972&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="element-citation">Maurano MT, Humbert R, Rynes E, Thurman RE, Haugen E, Wang H, Reynolds AP, Sandstrom R, Qu H, Brody J, Shafer A, Neri F, Lee K, Kutyavin T, Stehling-Sun S, Johnson AK, Canfield TK, Giste E, Diegel M, Bates D, Hansen RS, Neph S, Sabo PJ, Heimfeld S, Raubitschek A, Ziegler S, Cotsapas C, Sotoodehnia N, Glass I, Sunyaev SR, Kaul R, Stamatoyannopoulos JA. Systematic localization of common disease-associated variation in regulatory DNA. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">337</span>:1190–1195.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3771521/">PMC free article</a>]</span> [<a href="/pubmed/22955828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Systematic+localization+of+common+disease-associated+variation+in+regulatory+DNA&amp;author=MT+Maurano&amp;author=R+Humbert&amp;author=E+Rynes&amp;author=RE+Thurman&amp;author=E+Haugen&amp;volume=337&amp;publication_year=2012&amp;pages=1190-1195&amp;pmid=22955828&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R41"><span class="element-citation">McIntosh AM, Moorhead TW, Job D, Lymer GK, Munoz Maniega S, McKirdy J, Sussmann JE, Baig BJ, Bastin ME, Porteous D, Evans KL, Johnstone EC, Lawrie SM, Hall J. The effects of a neuregulin 1 variant on white matter density and integrity. <span><span class="ref-journal">Mol Psychiatry. </span>2008;<span class="ref-vol">13</span>:1054–1059.</span> [<a href="/pubmed/17925794" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=The+effects+of+a+neuregulin+1+variant+on+white+matter+density+and+integrity&amp;author=AM+McIntosh&amp;author=TW+Moorhead&amp;author=D+Job&amp;author=GK+Lymer&amp;author=S+Munoz+Maniega&amp;volume=13&amp;publication_year=2008&amp;pages=1054-1059&amp;pmid=17925794&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R42"><span class="element-citation">Mitchell AC, Bharadwaj R, Whittle C, Krueger W, Mirnics K, Hurd Y, Rasmussen T, Akbarian S. The Genome in Three Dimensions: A New Frontier in Human Brain Research. <span><span class="ref-journal">Biol Psychiatry. </span>2013</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3925763/">PMC free article</a>]</span> [<a href="/pubmed/23958183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=The+Genome+in+Three+Dimensions:+A+New+Frontier+in+Human+Brain+Research&amp;author=AC+Mitchell&amp;author=R+Bharadwaj&amp;author=C+Whittle&amp;author=W+Krueger&amp;author=K+Mirnics&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R43"><span class="element-citation">Mitchell BD, Kammerer CM, Blangero J, Mahaney MC, Rainwater DL, Dyke B, Hixson JE, Henkel RD, Sharp RM, Comuzzie AG, VandeBerg JL, Stern MP, MacCluer JW. Genetic and environmental contributions to cardiovascular risk factors in Mexican Americans. The San Antonio Family Heart Study. <span><span class="ref-journal">Circulation. </span>1996;<span class="ref-vol">94</span>:2159–2170.</span> [<a href="/pubmed/8901667" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Genetic+and+environmental+contributions+to+cardiovascular+risk+factors+in+Mexican+Americans.+The+San+Antonio+Family+Heart+Study&amp;author=BD+Mitchell&amp;author=CM+Kammerer&amp;author=J+Blangero&amp;author=MC+Mahaney&amp;author=DL+Rainwater&amp;volume=94&amp;publication_year=1996&amp;pages=2159-2170&amp;pmid=8901667&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R44"><span class="element-citation">Olvera RL, Bearden CE, Velligan DI, Almasy L, Carless MA, Curran JE, Williamson DE, Duggirala R, Blangero J, Glahn DC. Common genetic influences on depression, alcohol, and substance use disorders in Mexican-American families. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2011;<span class="ref-vol">156B</span>:561–568.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3112290/">PMC free article</a>]</span> [<a href="/pubmed/21557468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Common+genetic+influences+on+depression,+alcohol,+and+substance+use+disorders+in+Mexican-American+families&amp;author=RL+Olvera&amp;author=CE+Bearden&amp;author=DI+Velligan&amp;author=L+Almasy&amp;author=MA+Carless&amp;volume=156B&amp;publication_year=2011&amp;pages=561-568&amp;pmid=21557468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R45"><span class="element-citation">Pe'er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe'er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R46"><span class="element-citation">Ponimaskin EG, Profirovic J, Vaiskunaite R, Richter DW, Voyno-Yasenetskaya TA. 5-Hydroxytryptamine 4(a) receptor is coupled to the Galpha subunit of heterotrimeric G13 protein. <span><span class="ref-journal">J Biol Chem. </span>2002;<span class="ref-vol">277</span>:20812–20819.</span> [<a href="/pubmed/11923294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=5-Hydroxytryptamine+4(a)+receptor+is+coupled+to+the+Galpha+subunit+of+heterotrimeric+G13+protein&amp;author=EG+Ponimaskin&amp;author=J+Profirovic&amp;author=R+Vaiskunaite&amp;author=DW+Richter&amp;author=TA+Voyno-Yasenetskaya&amp;volume=277&amp;publication_year=2002&amp;pages=20812-20819&amp;pmid=11923294&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R47"><span class="element-citation">Puppala S, Dodd GD, Fowler S, Arya R, Schneider J, Farook VS, Granato R, Dyer TD, Almasy L, Jenkinson CP, Diehl AK, Stern MP, Blangero J, Duggirala R. A genomewide search finds major susceptibility loci for gallbladder disease on chromosome 1 in Mexican Americans. <span><span class="ref-journal">Am J Hum Genet. </span>2006;<span class="ref-vol">78</span>:377–392.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1380282/">PMC free article</a>]</span> [<a href="/pubmed/16400619" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genomewide+search+finds+major+susceptibility+loci+for+gallbladder+disease+on+chromosome+1+in+Mexican+Americans&amp;author=S+Puppala&amp;author=GD+Dodd&amp;author=S+Fowler&amp;author=R+Arya&amp;author=J+Schneider&amp;volume=78&amp;publication_year=2006&amp;pages=377-392&amp;pmid=16400619&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R48"><span class="element-citation">Ripke S, O'Dushlaine C, Chambert K, Moran JL, Kahler AK, Akterin S, Bergen SE, Collins AL, Crowley JJ, Fromer M, Kim Y, Lee SH, Magnusson PK, Sanchez N, Stahl EA, Williams S, Wray NR, Xia K, Bettella F, Borglum AD, Bulik-Sullivan BK, Cormican P, Craddock N, de Leeuw C, Durmishi N, Gill M, Golimbet V, Hamshere ML, Holmans P, Hougaard DM, Kendler KS, Lin K, Morris DW, Mors O, Mortensen PB, Neale BM, O'Neill FA, Owen MJ, Milovancevic MP, Posthuma D, Powell J, Richards AL, Riley BP, Ruderfer D, Rujescu D, Sigurdsson E, Silagadze T, Smit AB, Stefansson H, Steinberg S, Suvisaari J, Tosato S, Verhage M, Walters JT, Multicenter Genetic Studies of Schizophrenia C, Levinson DF, Gejman PV, Kendler KS, Laurent C, Mowry BJ, O'Donovan MC, Owen MJ, Pulver AE, Riley BP, Schwab SG, Wildenauer DB, Dudbridge F, Holmans P, Shi J, Albus M, Alexander M, Campion D, Cohen D, Dikeos D, Duan J, Eichhammer P, Godard S, Hansen M, Lerer FB, Liang KY, Maier W, Mallet J, Nertney DA, Nestadt G, Norton N, O'Neill FA, Papadimitriou GN, Ribble R, Sanders AR, Silverman JM, Walsh D, Williams NM, Wormley B, Psychosis Endophenotypes International C, Arranz MJ, Bakker S, Bender S, Bramon E, Collier D, Crespo-Facorro B, Hall J, Iyegbe C, Jablensky A, Kahn RS, Kalaydjieva L, Lawrie S, Lewis CM, Lin K, Linszen DH, Mata I, McIntosh A, Murray RM, Ophoff RA, Powell J, Rujescu D, Van Os J, Walshe M, Weisbrod M, Wiersma D, Wellcome Trust Case Control C, Management C, Donnelly P, Barroso I, Blackwell JM, Bramon E, Brown MA, Casas JP, Corvin AP, Deloukas P, Duncanson A, Jankowski J, Markus HS, Mathew CG, Palmer CN, Plomin R, Rautanen A, Sawcer SJ, Trembath RC, Viswanathan AC, Wood NW, Data Analysis G, Spencer CC, Band G, Bellenguez C, Freeman C, Hellenthal G, Giannoulatou E, Pirinen M, Pearson RD, Strange A, Su Z, Vukcevic D, Donnelly P, Dna GDQC, Informatics G, Langford C, Hunt SE, Edkins S, Gwilliam R, Blackburn H, Bumpstead SJ, Dronov S, Gillman M, Gray E, Hammond N, Jayakumar A, McCann OT, Liddle J, Potter SC, Ravindrarajah R, Ricketts M, Tashakkori-Ghanbaria A, Waller MJ, Weston P, Widaa S, Whittaker P, Barroso I, Deloukas P, Publications C, Mathew CG, Blackwell JM, Brown MA, Corvin AP, McCarthy MI, Spencer CC, Bramon E, Corvin AP, O'Donovan MC, Stefansson K, Scolnick E, Purcell S, McCarroll SA, Sklar P, Hultman CM, Sullivan PF. Genome-wide association analysis identifies 13 new risk loci for schizophrenia. <span><span class="ref-journal">Nat Genet. </span>2013</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3827979/">PMC free article</a>]</span> [<a href="/pubmed/23974872" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analysis+identifies+13+new+risk+loci+for+schizophrenia&amp;author=S+Ripke&amp;author=C+O'Dushlaine&amp;author=K+Chambert&amp;author=JL+Moran&amp;author=AK+Kahler&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R49"><span class="element-citation">Schmierer K, Wheeler-Kingshott CA, Boulby PA, Scaravilli F, Altmann DR, Barker GJ, Tofts PS, Miller DH. Diffusion tensor imaging of post mortem multiple sclerosis brain. <span><span class="ref-journal">Neuroimage. </span>2007;<span class="ref-vol">35</span>:467–477.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1892244/">PMC free article</a>]</span> [<a href="/pubmed/17258908" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Diffusion+tensor+imaging+of+post+mortem+multiple+sclerosis+brain&amp;author=K+Schmierer&amp;author=CA+Wheeler-Kingshott&amp;author=PA+Boulby&amp;author=F+Scaravilli&amp;author=DR+Altmann&amp;volume=35&amp;publication_year=2007&amp;pages=467-477&amp;pmid=17258908&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R50"><span class="element-citation">Seifuddin F, Mahon PB, Judy J, Pirooznia M, Jancic D, Taylor J, Goes FS, Potash JB, Zandi PP. Meta-analysis of genetic association studies on bipolar disorder. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2012;<span class="ref-vol">159B</span>:508–518.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3582382/">PMC free article</a>]</span> [<a href="/pubmed/22573399" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Meta-analysis+of+genetic+association+studies+on+bipolar+disorder&amp;author=F+Seifuddin&amp;author=PB+Mahon&amp;author=J+Judy&amp;author=M+Pirooznia&amp;author=D+Jancic&amp;volume=159B&amp;publication_year=2012&amp;pages=508-518&amp;pmid=22573399&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R51"><span class="element-citation">Siehler S. Regulation of RhoGEF proteins by G12/13-coupled receptors. <span><span class="ref-journal">Br J Pharmacol. </span>2009;<span class="ref-vol">158</span>:41–49.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2795247/">PMC free article</a>]</span> [<a href="/pubmed/19226283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Pharmacol&amp;title=Regulation+of+RhoGEF+proteins+by+G12/13-coupled+receptors&amp;author=S+Siehler&amp;volume=158&amp;publication_year=2009&amp;pages=41-49&amp;pmid=19226283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R52"><span class="element-citation">Smith SM, Jenkinson M, Johansen-Berg H, Rueckert D, Nichols TE, Mackay CE, Watkins KE, Ciccarelli O, Cader MZ, Matthews PM, Behrens TE. Tract-based spatial statistics: voxelwise analysis of multi-subject diffusion data. <span><span class="ref-journal">Neuroimage. </span>2006;<span class="ref-vol">31</span>:1487–1505.</span> [<a href="/pubmed/16624579" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Tract-based+spatial+statistics:+voxelwise+analysis+of+multi-subject+diffusion+data&amp;author=SM+Smith&amp;author=M+Jenkinson&amp;author=H+Johansen-Berg&amp;author=D+Rueckert&amp;author=TE+Nichols&amp;volume=31&amp;publication_year=2006&amp;pages=1487-1505&amp;pmid=16624579&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R53"><span class="element-citation">Sobel E, Lange K. Descent graphs in pedigree analysis: applications to haplotyping, location scores, and marker-sharing statistics. <span><span class="ref-journal">Am J Hum Genet. </span>1996;<span class="ref-vol">58</span>:1323–1337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1915074/">PMC free article</a>]</span> [<a href="/pubmed/8651310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Descent+graphs+in+pedigree+analysis:+applications+to+haplotyping,+location+scores,+and+marker-sharing+statistics&amp;author=E+Sobel&amp;author=K+Lange&amp;volume=58&amp;publication_year=1996&amp;pages=1323-1337&amp;pmid=8651310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R54"><span class="element-citation">Sprooten E, Brumbaugh MS, Knowles EEM, McKay DR, Lewis J, Barrett J, Landau S, Cyr L, Kochunov P, Winkler AM, Pearlson GD, Glahn DC. Reduced white matter integrity in sibling pairs discordant for bipolar disorder. <span><span class="ref-journal">Am Journal of Psychiatry. </span>2013a</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4119087/">PMC free article</a>]</span> [<a href="/pubmed/24185242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+Journal+of+Psychiatry&amp;title=Reduced+white+matter+integrity+in+sibling+pairs+discordant+for+bipolar+disorder&amp;author=E+Sprooten&amp;author=MS+Brumbaugh&amp;author=EEM+Knowles&amp;author=DR+McKay&amp;author=J+Lewis&amp;publication_year=2013a&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R55"><span class="element-citation">Sprooten E, Fleming KM, Thomson PA, Bastin ME, Whalley HC, Hall J, Sussmann JE, McKirdy J, Blackwood D, Lawrie SM, McIntosh AM. White matter integrity as an intermediate phenotype: exploratory genome-wide association analysis in individuals at high risk of bipolar disorder. <span><span class="ref-journal">Psychiatry Res. </span>2013b;<span class="ref-vol">206</span>:223–231.</span> [<a href="/pubmed/23218918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychiatry+Res&amp;title=White+matter+integrity+as+an+intermediate+phenotype:+exploratory+genome-wide+association+analysis+in+individuals+at+high+risk+of+bipolar+disorder&amp;author=E+Sprooten&amp;author=KM+Fleming&amp;author=PA+Thomson&amp;author=ME+Bastin&amp;author=HC+Whalley&amp;volume=206&amp;publication_year=2013b&amp;pages=223-231&amp;pmid=23218918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R56"><span class="element-citation">Sprooten E, Lymer GK, Munoz Maniega S, McKirdy J, Clayden JD, Bastin ME, Porteous D, Johnstone EC, Lawrie SM, Hall J, McIntosh AM. The relationship of anterior thalamic radiation integrity to psychosis risk associated neuregulin-1 variants. <span><span class="ref-journal">Mol Psychiatry. </span>2009;<span class="ref-vol">14</span>:237–238.</span> 233. [<a href="/pubmed/19229203" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=The+relationship+of+anterior+thalamic+radiation+integrity+to+psychosis+risk+associated+neuregulin-1+variants&amp;author=E+Sprooten&amp;author=GK+Lymer&amp;author=S+Munoz+Maniega&amp;author=J+McKirdy&amp;author=JD+Clayden&amp;volume=14&amp;publication_year=2009&amp;pages=237-238&amp;pmid=19229203&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R57"><span class="element-citation">Sprooten E, Sussmann JE, Clugston A, Peel A, McKirdy J, Moorhead TW, Anderson S, Shand AJ, Giles S, Bastin ME, Hall J, Johnstone EC, Lawrie SM, McIntosh AM. White matter integrity in individuals at high genetic risk of bipolar disorder. <span><span class="ref-journal">Biol Psychiatry. </span>2011a;<span class="ref-vol">70</span>:350–356.</span> [<a href="/pubmed/21429475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=White+matter+integrity+in+individuals+at+high+genetic+risk+of+bipolar+disorder&amp;author=E+Sprooten&amp;author=JE+Sussmann&amp;author=A+Clugston&amp;author=A+Peel&amp;author=J+McKirdy&amp;volume=70&amp;publication_year=2011a&amp;pages=350-356&amp;pmid=21429475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R58"><span class="element-citation">Sprooten E, Sussmann JE, Moorhead TW, Whalley HC, Ffrench-Constant C, Blumberg HP, Bastin ME, Hall J, Lawrie SM, McIntosh AM. Association of white matter integrity with genetic variation in an exonic DISC1 SNP. <span><span class="ref-journal">Mol Psychiatry. </span>2011b;<span class="ref-vol">16</span>(685):688–689.</span> [<a href="/pubmed/21358711" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Association+of+white+matter+integrity+with+genetic+variation+in+an+exonic+DISC1+SNP&amp;author=E+Sprooten&amp;author=JE+Sussmann&amp;author=TW+Moorhead&amp;author=HC+Whalley&amp;author=C+Ffrench-Constant&amp;volume=16&amp;issue=685&amp;publication_year=2011b&amp;pages=688-689&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R59"><span class="element-citation">Stebbins GT, Murphy CM. Diffusion tensor imaging in Alzheimer's disease and mild cognitive impairment. <span><span class="ref-journal">Behav Neurol. </span>2009;<span class="ref-vol">21</span>:39–49.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3010401/">PMC free article</a>]</span> [<a href="/pubmed/19847044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behav+Neurol&amp;title=Diffusion+tensor+imaging+in+Alzheimer's+disease+and+mild+cognitive+impairment&amp;author=GT+Stebbins&amp;author=CM+Murphy&amp;volume=21&amp;publication_year=2009&amp;pages=39-49&amp;pmid=19847044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R60"><span class="element-citation">Stein JL, Medland SE, Vasquez AA, Hibar DP, Senstad RE, Winkler AM, Toro R, Appel K, Bartecek R, Bergmann O, Bernard M, Brown AA, Cannon DM, Chakravarty MM, Christoforou A, Domin M, Grimm O, Hollinshead M, Holmes AJ, Homuth G, Hottenga JJ, Langan C, Lopez LM, Hansell NK, Hwang KS, Kim S, Laje G, Lee PH, Liu X, Loth E, Lourdusamy A, Mattingsdal M, Mohnke S, Maniega SM, Nho K, Nugent AC, O'Brien C, Papmeyer M, Putz B, Ramasamy A, Rasmussen J, Rijpkema M, Risacher SL, Roddey JC, Rose EJ, Ryten M, Shen L, Sprooten E, Strengman E, Teumer A, Trabzuni D, Turner J, van Eijk K, van Erp TG, van Tol MJ, Wittfeld K, Wolf C, Woudstra S, Aleman A, Alhusaini S, Almasy L, Binder EB, Brohawn DG, Cantor RM, Carless MA, Corvin A, Czisch M, Curran JE, Davies G, de Almeida MA, Delanty N, Depondt C, Duggirala R, Dyer TD, Erk S, Fagerness J, Fox PT, Freimer NB, Gill M, Goring HH, Hagler DJ, Hoehn D, Holsboer F, Hoogman M, Hosten N, Jahanshad N, Johnson MP, Kasperaviciute D, Kent JW, Jr, Kochunov P, Lancaster JL, Lawrie SM, Liewald DC, Mandl R, Matarin M, Mattheisen M, Meisenzahl E, Melle I, Moses EK, Muhleisen TW, Nauck M, Nothen MM, Olvera RL, Pandolfo M, Pike GB, Puls R, Reinvang I, Renteria ME, Rietschel M, Roffman JL, Royle NA, Rujescu D, Savitz J, Schnack HG, Schnell K, Seiferth N, Smith C, Steen VM, Valdes Hernandez MC, Van den Heuvel M, van der Wee NJ, Van Haren NE, Veltman JA, Volzke H, Walker R, Westlye LT, Whelan CD, Agartz I, Boomsma DI, Cavalleri GL, Dale AM, Djurovic S, Drevets WC, Hagoort P, Hall J, Heinz A, Jack CR, Jr, Foroud TM, Le Hellard S, Macciardi F, Montgomery GW, Poline JB, Porteous DJ, Sisodiya SM, Starr JM, Sussmann J, Toga AW, Veltman DJ, Walter H, Weiner MW, Alzheimer's Disease Neuroimaging I, Consortium E, Consortium I, Saguenay Youth Study G, Bis JC, Ikram MA, Smith AV, Gudnason V, Tzourio C, Vernooij MW, Launer LJ, DeCarli C, Seshadri S, Cohorts for H, Aging Research in Genomic Epidemiology C, Andreassen OA, Apostolova LG, Bastin ME, Blangero J, Brunner HG, Buckner RL, Cichon S, Coppola G, de Zubicaray GI, Deary IJ, Donohoe G, de Geus EJ, Espeseth T, Fernandez G, Glahn DC, Grabe HJ, Hardy J, Hulshoff Pol HE, Jenkinson M, Kahn RS, McDonald C, McIntosh AM, McMahon FJ, McMahon KL, Meyer-Lindenberg A, Morris DW, Muller-Myhsok B, Nichols TE, Ophoff RA, Paus T, Pausova Z, Penninx BW, Potkin SG, Samann PG, Saykin AJ, Schumann G, Smoller JW, Wardlaw JM, Weale ME, Martin NG, Franke B, Wright MJ, Thompson PM, Enhancing Neuro Imaging Genetics through Meta-Analysis C. Identification of common variants associated with human hippocampal and intracranial volumes. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:552–561.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3635491/">PMC free article</a>]</span> [<a href="/pubmed/22504417" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Identification+of+common+variants+associated+with+human+hippocampal+and+intracranial+volumes&amp;author=JL+Stein&amp;author=SE+Medland&amp;author=AA+Vasquez&amp;author=DP+Hibar&amp;author=RE+Senstad&amp;volume=44&amp;publication_year=2012&amp;pages=552-561&amp;pmid=22504417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R61"><span class="element-citation">Whalley HC, Sprooten E, Hackett S, Hall L, Blackwood DH, Glahn DC, Bastin M, Hall J, Lawrie SM, Sussmann JE, McIntosh AM. Polygenic risk and white matter integrity in individuals at high risk of mood disorder. <span><span class="ref-journal">Biol Psychiatry. </span>2013;<span class="ref-vol">74</span>:280–286.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4185278/">PMC free article</a>]</span> [<a href="/pubmed/23453289" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Polygenic+risk+and+white+matter+integrity+in+individuals+at+high+risk+of+mood+disorder&amp;author=HC+Whalley&amp;author=E+Sprooten&amp;author=S+Hackett&amp;author=L+Hall&amp;author=DH+Blackwood&amp;volume=74&amp;publication_year=2013&amp;pages=280-286&amp;pmid=23453289&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R62"><span class="element-citation">Williams JT, Van Eerdewegh P, Almasy L, Blangero J. Joint multipoint linkage analysis of multivariate qualitative and quantitative traits. I. Likelihood formulation and simulation results. <span><span class="ref-journal">Am J Hum Genet. </span>1999;<span class="ref-vol">65</span>:1134–1147.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1288247/">PMC free article</a>]</span> [<a href="/pubmed/10486333" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Joint+multipoint+linkage+analysis+of+multivariate+qualitative+and+quantitative+traits.+I.+Likelihood+formulation+and+simulation+results&amp;author=JT+Williams&amp;author=P+Van+Eerdewegh&amp;author=L+Almasy&amp;author=J+Blangero&amp;volume=65&amp;publication_year=1999&amp;pages=1134-1147&amp;pmid=10486333&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R63"><span class="element-citation">Winterer G, Konrad A, Vucurevic G, Musso F, Stoeter P, Dahmen N. Association of 5' end neuregulin-1 (NRG1) gene variation with subcortical medial frontal microstructure in humans. <span><span class="ref-journal">Neuroimage. </span>2008;<span class="ref-vol">40</span>:712–718.</span> [<a href="/pubmed/18255317" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Association+of+5'+end+neuregulin-1+(NRG1)+gene+variation+with+subcortical+medial+frontal+microstructure+in+humans&amp;author=G+Winterer&amp;author=A+Konrad&amp;author=G+Vucurevic&amp;author=F+Musso&amp;author=P+Stoeter&amp;volume=40&amp;publication_year=2008&amp;pages=712-718&amp;pmid=18255317&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R64"><span class="element-citation">Zuliani R, Moorhead TW, Bastin ME, Johnstone EC, Lawrie SM, Brambilla P, O'Donovan MC, Owen MJ, Hall J, McIntosh AM. Genetic variants in the ErbB4 gene are associated with white matter integrity. <span><span class="ref-journal">Psychiatry Res. </span>2011;<span class="ref-vol">191</span>:133–137.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5372351/">PMC free article</a>]</span> [<a href="/pubmed/21232925" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychiatry+Res&amp;title=Genetic+variants+in+the+ErbB4+gene+are+associated+with+white+matter+integrity&amp;author=R+Zuliani&amp;author=TW+Moorhead&amp;author=ME+Bastin&amp;author=EC+Johnstone&amp;author=SM+Lawrie&amp;volume=191&amp;publication_year=2011&amp;pages=133-137&amp;pmid=21232925&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4107465&amp;issue-id=240676&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139811717441424title">Abstract</a></li><li><a href="#S1title">1. Introduction</a></li><li><a href="#S2title">2. Materials and Methods</a></li><li><a href="#S10title">3. Results</a></li><li><a href="#S15title">4. Discussion</a></li><li><a href="#S16title">5. Conclusions</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S17title">Acknowledgements</a></li><li><a href="#idm139811715268688title">Footnotes</a></li><li><a href="#idm139811720489792title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4107465/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4107465/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4107465/pdf/nihms606427.pdf">PDF (1.0M)</a></li> | <li><a href="#" data-citationid="PMC4107465" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4107465%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4107465%2F&amp;text=Common%20genetic%20variants%20and%20gene%20expression%20associated%20with%20white%20matter%20microstructure%20in%20the%20human%20brain" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4107465%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24736177" data-aiid="4107465" data-aid="4107465" data-iid="240676" data-domainid="319" data-domain="nihpa" data-accid="PMC4107465" data-md5="44c50d3874193831e82d5cfcc1311f97"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:03:18-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C3D0CDD5669D100000000085E04F6&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4107465%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4107465/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>