<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajrccm"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4214107/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Interrogation of Longitudinal FEV1 in Children with Asthma</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4214107/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Respiratory and Critical Care Medicine"><meta name="citation_title" content="Genome-wide Interrogation of Longitudinal FEV1 in Children with Asthma"><meta name="citation_authors" content="Kehua Wu, Eric R. Gamazon, Hae Kyung Im, Paul Geeleher, Steven R. White, Julian Solway, George L. Clemmer, Scott T. Weiss, Kelan G. Tantisira, Nancy J. Cox, Mark J. Ratain, R. Stephanie Huang"><meta name="citation_date" content="15 September 2014"><meta name="citation_issue" content="6"><meta name="citation_volume" content="190"><meta name="citation_firstpage" content="619"><meta name="citation_doi" content="10.1164/rccm.201403-0460OC"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4214107/?report=abstract"><meta name="citation_pmid" content="25221879"><meta name="DC.Title" content="Genome-wide Interrogation of Longitudinal FEV1 in Children with Asthma"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Thoracic Society"><meta name="DC.Contributor" content="Kehua Wu"><meta name="DC.Contributor" content="Eric R. Gamazon"><meta name="DC.Contributor" content="Hae Kyung Im"><meta name="DC.Contributor" content="Paul Geeleher"><meta name="DC.Contributor" content="Steven R. White"><meta name="DC.Contributor" content="Julian Solway"><meta name="DC.Contributor" content="George L. Clemmer"><meta name="DC.Contributor" content="Scott T. Weiss"><meta name="DC.Contributor" content="Kelan G. Tantisira"><meta name="DC.Contributor" content="Nancy J. Cox"><meta name="DC.Contributor" content="Mark J. Ratain"><meta name="DC.Contributor" content="R. Stephanie Huang"><meta name="DC.Date" content="2014 Sep 15"><meta name="DC.Identifier" content="10.1164/rccm.201403-0460OC"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Interrogation of Longitudinal FEV1 in Children with Asthma"><meta property="og:type" content="article"><meta property="og:description" content="Rationale: Most genomic studies of lung function have used phenotypic data derived from a single time-point (e.g., presence/absence of disease) without considering the dynamic progression of a chronic disease.Objectives: To characterize lung function ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4214107/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajrccm.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4214107/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4214107/pdf/rccm.201403-0460OC.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8AE755DD576D910000000002AB01A6.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4214107%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Am J Respir Crit Care Med%22[journal]" class="navlink">Am J Respir Crit Care Med</a></li><li class="accid">PMC4214107</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajrccm.gif" alt="Logo of ajrccm" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="338,19,460,34" alt="Issue Featuring Article" title="Issue Featuring Article" href="http://ajrccm.atsjournals.org/content/vol190/issue6" ref="reftype=publisher&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="338,3,494,18" alt="Publisher's Version of Article" title="Publisher's Version of Article" href="//dx.doi.org/10.1164/rccm.201403-0460OC" ref="reftype=publisher&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"><area shape="rect" coords="338,36,411,49" alt="Submissions" title="Submissions" href="http://mc.manuscriptcentral.com/atsjournals" ref="reftype=publisher&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,3,318,29" alt="American Thoracic Society" title="American Thoracic Society" href="http://www.thoracic.org/" ref="reftype=publisher&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="1,1,52,55" alt="American Thoracic Society" title="American Thoracic Society" href="http://www.thoracic.org/" ref="reftype=publisher&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,56,499,75" alt="American Journal of Respiratory and Critical Care Medicine" title="American Journal of Respiratory and Critical Care Medicine" href="http://ajrccm.atsjournals.org/" ref="reftype=publisher&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Respir Crit Care Med</a></span>. 2014 Sep 15; 190(6): 619–627. </span></div><div><span class="fm-vol-iss-date">Published online 2014 Sep 15. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1164%2Frccm.201403-0460OC" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1164/rccm.201403-0460OC</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4214107</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25221879">25221879</a></div></div></div></div><h1 class="content-title">Genome-wide Interrogation of Longitudinal FEV<sub>1</sub> in Children with Asthma</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Wu%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074028111696" co-class="co-affbox">Kehua Wu</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Gamazon%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074028043648" co-class="co-affbox">Eric R. Gamazon</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Im%20HK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074028041472" co-class="co-affbox">Hae Kyung Im</a>,<sup>3</sup> <a href="/pubmed/?term=Geeleher%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074034069488" co-class="co-affbox">Paul Geeleher</a>,<sup>1</sup> <a href="/pubmed/?term=White%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074031369456" co-class="co-affbox">Steven R. White</a>,<sup>1</sup> <a href="/pubmed/?term=Solway%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074033300832" co-class="co-affbox">Julian Solway</a>,<sup>1</sup> <a href="/pubmed/?term=Clemmer%20GL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074033298720" co-class="co-affbox">George L. Clemmer</a>,<sup>4</sup> <a href="/pubmed/?term=Weiss%20ST%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074028003152" co-class="co-affbox">Scott T. Weiss</a>,<sup>4</sup> <a href="/pubmed/?term=Tantisira%20KG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074020785072" co-class="co-affbox">Kelan G. Tantisira</a>,<sup>4</sup> <a href="/pubmed/?term=Cox%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074028673040" co-class="co-affbox">Nancy J. Cox</a>,<sup>1</sup> <a href="/pubmed/?term=Ratain%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074028670928" co-class="co-affbox">Mark J. Ratain</a>,<sup>1</sup> and  <a href="/pubmed/?term=Huang%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879" class="affpopup" co-rid="_co_idm140074030246480" co-class="co-affbox">R. Stephanie Huang</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140074028111696"><h3 class="no_margin">Kehua Wu</h3><p><sup>1</sup>Department of Medicine and</p><p><sup>2</sup>State Key Laboratory of Natural and Biomimetic Drugs, Peking University, Beijing, China; and</p><div>Find articles by <a href="/pubmed/?term=Wu%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">Kehua Wu</a></div></div><div id="_co_idm140074028043648"><h3 class="no_margin">Eric R. Gamazon</h3><p><sup>1</sup>Department of Medicine and</p><div>Find articles by <a href="/pubmed/?term=Gamazon%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">Eric R. Gamazon</a></div></div><div id="_co_idm140074028041472"><h3 class="no_margin">Hae Kyung Im</h3><p><sup>3</sup>Department of Health Studies, The University of Chicago, Chicago, Illinois</p><div>Find articles by <a href="/pubmed/?term=Im%20HK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">Hae Kyung Im</a></div></div><div id="_co_idm140074034069488"><h3 class="no_margin">Paul Geeleher</h3><p><sup>1</sup>Department of Medicine and</p><div>Find articles by <a href="/pubmed/?term=Geeleher%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">Paul Geeleher</a></div></div><div id="_co_idm140074031369456"><h3 class="no_margin">Steven R. White</h3><p><sup>1</sup>Department of Medicine and</p><div>Find articles by <a href="/pubmed/?term=White%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">Steven R. White</a></div></div><div id="_co_idm140074033300832"><h3 class="no_margin">Julian Solway</h3><p><sup>1</sup>Department of Medicine and</p><div>Find articles by <a href="/pubmed/?term=Solway%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">Julian Solway</a></div></div><div id="_co_idm140074033298720"><h3 class="no_margin">George L. Clemmer</h3><p><sup>4</sup>Channing Division of Network Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts</p><div>Find articles by <a href="/pubmed/?term=Clemmer%20GL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">George L. Clemmer</a></div></div><div id="_co_idm140074028003152"><h3 class="no_margin">Scott T. Weiss</h3><p><sup>4</sup>Channing Division of Network Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts</p><div>Find articles by <a href="/pubmed/?term=Weiss%20ST%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">Scott T. Weiss</a></div></div><div id="_co_idm140074020785072"><h3 class="no_margin">Kelan G. Tantisira</h3><p><sup>4</sup>Channing Division of Network Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts</p><div>Find articles by <a href="/pubmed/?term=Tantisira%20KG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">Kelan G. Tantisira</a></div></div><div id="_co_idm140074028673040"><h3 class="no_margin">Nancy J. Cox</h3><p><sup>1</sup>Department of Medicine and</p><div>Find articles by <a href="/pubmed/?term=Cox%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">Nancy J. Cox</a></div></div><div id="_co_idm140074028670928"><h3 class="no_margin">Mark J. Ratain</h3><p><sup>1</sup>Department of Medicine and</p><div>Find articles by <a href="/pubmed/?term=Ratain%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">Mark J. Ratain</a></div></div><div id="_co_idm140074030246480"><h3 class="no_margin">R. Stephanie Huang</h3><p><sup>1</sup>Department of Medicine and</p><div>Find articles by <a href="/pubmed/?term=Huang%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25221879">R. Stephanie Huang</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140074034085856_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140074034085856_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140074034085856_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140074034085856_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Medicine and</div><div class="fm-affl" id="aff2" lang="en"><sup>3</sup>Department of Health Studies, The University of Chicago, Chicago, Illinois</div><div class="fm-affl" id="aff3" lang="en"><sup>2</sup>State Key Laboratory of Natural and Biomimetic Drugs, Peking University, Beijing, China; and</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Channing Division of Network Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="fn1"><sup>*</sup>These authors contributed equally to this work.</div><div id="idm140074031031760">Correspondence and requests for reprints should be addressed to R. Stephanie Huang, Ph.D., 900 East 57th Street, KCBD Room 7148, The University of Chicago, Chicago, IL 60637. E-mail: <a href="mailto:dev@null" data-email="ude.ogacihcu.dsb.enicidem@gnauhr" class="oemail">ude.ogacihcu.dsb.enicidem@gnauhr</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140074034085856_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 Mar 11; Accepted 2014 Aug 3.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140074034085856_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2014 by the American Thoracic Society</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4214107/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140074030770640" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140074030770640title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first"><strong>Rationale:</strong> Most genomic studies of lung function have used phenotypic data derived from a single time-point (e.g., presence/absence of disease) without considering the dynamic progression of a chronic disease.</p><p id="__p3"><strong>Objectives:</strong> To characterize lung function change over time in subjects with asthma and identify genetic contributors to a longitudinal phenotype.</p><p id="__p4"><strong>Methods:</strong> We present a method that models longitudinal FEV<sub>1</sub> data, collected from 1,041 children with asthma who participated in the Childhood Asthma Management Program. This longitudinal progression model was built using population-based nonlinear mixed-effects modeling with an exponential structure and the determinants of age and height.</p><p id="__p5"><strong>Measurements and Main Results:</strong> We found ethnicity was a key covariate for FEV<sub>1</sub> level. Budesonide-treated children with asthma had a slight but significant effect on FEV<sub>1</sub> when compared with those treated with placebo or nedocromil (<em>P</em> &lt; 0.001). A genome-wide association study identified seven single-nucleotide polymorphisms nominally associated with longitudinal lung function phenotypes in 581 white Childhood Asthma Management Program subjects (<em>P</em> &lt; 10<sup>−4</sup> in the placebo [“discovery”] and <em>P</em> &lt; 0.05 in the nedocromil treatment [“replication”] group). Using ChIP-seq and RNA-seq data, we found that some of the associated variants were in strong enhancer regions in human lung fibroblasts and may affect gene expression in human lung tissue. Genetic mapping restricted to genome-wide enhancer single-nucleotide polymorphisms in lung fibroblasts revealed a highly significant variant (rs6763931; <em>P</em> = 4 × 10<sup>−6</sup>; false discovery rate &lt; 0.05).</p><p id="__p6" class="p p-last"><strong>Conclusions:</strong> This study offers a strategy to explore the genetic determinants of longitudinal phenotypes, provide a comprehensive picture of disease pathophysiology, and suggest potential treatment targets.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">asthma, NONMEM, longitudinal model, FEV<sub>1</sub></span></div></div><div id="idm140074018505632" class="tsec sec headless whole_rhythm"><div class="boxed-text-box whole_rhythm hide-overflow"><div class="sec"><h3 id="__sec1title">At a Glance Commentary</h3><div class="sec"><h4 id="__sec2title">Scientific Knowledge on the Subject</h4><p id="__p7" class="p p-first-last">Genome-wide association studies of single–time-point phenotypes have been conducted for the discovery of genetic determinants of lung function and/or asthma risk in large human populations. Several single-nucleotide polymorphisms have been found to be reproducibly associated with pulmonary function.</p></div><div class="sec"><h4 id="__sec3title">What This Study Adds to the Field</h4><p id="__p8" class="p p-first-last">We developed an integrative method that combines mixed-effects longitudinal modeling and a genome-wide association study. This approach may facilitate a better understanding of the progression of a complex phenotype over time and enable improved discovery of genetic variants associated with a dynamic phenotype.</p></div></div></div><p id="__p9" class="p p-first">Genome-wide association studies (GWASs) have greatly contributed to the identification of genes and genetic variants conferring susceptibility to complex diseases and other heritable traits, such as lung function (<a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562578">1</a>–<a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562575">4</a>). For example, a metaanalysis of GWASs has implicated several independent loci for association with forced expiratory volume in 1 second (FEV<sub>1</sub>) in the general population (<a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562586">5</a>, <a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562582">6</a>). Some genomic loci were found to influence pulmonary function in several populations of patients with asthma (<a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562580">7</a>). A subsequent study demonstrated that genes involved in airway remodeling were associated with lung function both in general populations and in patients with asthma (<a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562594">8</a>). Using GWAS to identify the genetic factors associated with lung function may, in addition, lead to a greater understanding of asthma pathophysiology. However, in most such studies conducted to date, the phenotype is typically defined assuming a single time-point (e.g., the binary classification with/without disease at a given time or response/no-response to treatment after one cycle). This binary “snapshot” phenotype reflects a single depiction of the dynamic biology, but fails to capture the progression of a disease over time of a chronic condition, such as asthma.</p><p id="__p10">Longitudinal modeling of disease trajectory is a useful tool to characterize the disease progression and to identify disease-modifying effects of drugs (<a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562591">9</a>, <a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562587">10</a>). In asthma, the most commonly used lung function indicator is FEV<sub>1</sub>. This parameter is highly dependent on physiologic factors (e.g., age, body weight, height [<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562588">11</a>–<a href="#bib14" rid="bib14" class=" bibr popnode">14</a>]) and pathologic conditions (e.g., severity of asthma), and is in addition partially genetically determined (<a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562563">5</a>). Longitudinal modeling of FEV<sub>1</sub> allows for estimation of an individual’s baseline lung function and change in lung function over time, which is independent of potentially confounding stature-related factors. To our knowledge, only a handful of studies have attempted longitudinal modeling of FEV<sub>1</sub>. All were performed in healthy subjects (<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562604">11</a>–<a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562608">13</a>), with the exception of one study that included patients with asthma (age range, 6–88 yr). None of these modeling studies performed a genetic analysis (<a href="#bib14" rid="bib14" class=" bibr popnode">14</a>).</p><p id="__p11" class="p">In this study, we develop a method that applies nonlinear mixed-effects modeling (NONMEM) to longitudinal FEV<sub>1</sub> observations with the goal of describing lung function in children with asthma in the presence and absence of treatment. Furthermore, GWAS was performed to identify genetic predictors of baseline lung function and the rate of change in FEV<sub>1</sub> level over time. In our study, the “longitudinal phenotypes” used for GWAS were the estimates of the FEV<sub>1</sub> baseline level and the rate of change in FEV<sub>1</sub> with age for each patient derived from the population-based model. (We use the term “longitudinal phenotypes” throughout to refer to the two phenotypes even though the FEV<sub>1</sub> baseline level is defined at a particular [initial] time-point; “longitudinal” refers to the fact that these phenotypes were derived from a longitudinal modeling approach.) These phenotypes were independent of confounding anthropometric factors and characterize the development of lung function in these patients. We hypothesized that our method, which integrates GWAS and longitudinal modeling, may facilitate a better understanding of lung function in children with asthma over time than a standard analysis of a snapshot phenotype as currently used.</p></div><div id="__sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec4title">Methods</h2><div id="__sec5" class="sec sec-first"><h3 id="__sec5title">Study Design</h3><p id="__p12" class="p p-first">This study was performed using the Childhood Asthma Management Program (CAMP) study dataset (<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562568">15</a>). Demographic information for the CAMP study participants is shown in <a href="/pmc/articles/PMC4214107/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>. The CAMP study was approved by the Institutional Review Board in all eight study sites. Informed consent and assent were obtained from the participants and their guardians before enrolment (<a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562592">16</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p13">Demographic Information of the Patients at the Time of Enrollment</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140074067284048" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Characteristic</th><th rowspan="1" colspan="1" align="center">Placebo</th><th rowspan="1" colspan="1" align="center">Nedocromil</th><th rowspan="1" colspan="1" align="center">Budesonide</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Number</td><td rowspan="1" colspan="1" align="center">418</td><td rowspan="1" colspan="1" align="center">312</td><td rowspan="1" colspan="1" align="center">311</td></tr><tr><td rowspan="1" colspan="1" align="left">Age, yr</td><td rowspan="1" colspan="1" align="center">9.0 ± 2.2</td><td rowspan="1" colspan="1" align="center">8.8 ± 2.1</td><td rowspan="1" colspan="1" align="center">9.0 ± 2.1</td></tr><tr><td rowspan="1" colspan="1" align="left">Sex, male/female</td><td rowspan="1" colspan="1" align="center">234/184</td><td rowspan="1" colspan="1" align="center">206/106</td><td rowspan="1" colspan="1" align="center">181/130</td></tr><tr><td rowspan="1" colspan="1" align="left">Race, n (%)</td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td rowspan="1" colspan="1" align="center">&nbsp;</td></tr><tr><td rowspan="1" colspan="1" align="left"> White</td><td rowspan="1" colspan="1" align="center">292 (69.9)</td><td rowspan="1" colspan="1" align="center">218 (69.9)</td><td rowspan="1" colspan="1" align="center">201 (64.6)</td></tr><tr><td rowspan="1" colspan="1" align="left"> African American</td><td rowspan="1" colspan="1" align="center">56 (13.4)</td><td rowspan="1" colspan="1" align="center">38 (12.2)</td><td rowspan="1" colspan="1" align="center">44 (14.1)</td></tr><tr><td rowspan="1" colspan="1" align="left"> Mexican American</td><td rowspan="1" colspan="1" align="center">37 (8.9)</td><td rowspan="1" colspan="1" align="center">29 (9.3)</td><td rowspan="1" colspan="1" align="center">32 (10.3)</td></tr><tr><td rowspan="1" colspan="1" align="left"> Others</td><td rowspan="1" colspan="1" align="center">33 (7.9)</td><td rowspan="1" colspan="1" align="center">27 (8.7)</td><td rowspan="1" colspan="1" align="center">34 (10.9)</td></tr><tr><td rowspan="1" colspan="1" align="left">Height, cm</td><td rowspan="1" colspan="1" align="center">55.3 ± 28.8</td><td rowspan="1" colspan="1" align="center">56.0 ± 28.7</td><td rowspan="1" colspan="1" align="center">56.8 ± 28.0</td></tr><tr><td rowspan="1" colspan="1" align="left">Body weight, kg</td><td rowspan="1" colspan="1" align="center">42.0 ± 16.0</td><td rowspan="1" colspan="1" align="center">42.2 ± 16.0</td><td rowspan="1" colspan="1" align="center">42.9 ± 16.8</td></tr><tr><td rowspan="1" colspan="1" align="left">Age of first symptoms, yr</td><td rowspan="1" colspan="1" align="center">3.0 ± 2.6</td><td rowspan="1" colspan="1" align="center">3.1 ± 2.4</td><td rowspan="1" colspan="1" align="center">3.1 ± 2.3</td></tr><tr><td rowspan="1" colspan="1" align="left">Year since diagnosis of asthma</td><td rowspan="1" colspan="1" align="center">4.9 ± 2.7</td><td rowspan="1" colspan="1" align="center">5.0 ± 2.7</td><td rowspan="1" colspan="1" align="center">5.2 ± 2.6</td></tr><tr><td rowspan="1" colspan="1" align="left">Maternal asthma, no/yes</td><td rowspan="1" colspan="1" align="center">301/102</td><td rowspan="1" colspan="1" align="center">224/81</td><td rowspan="1" colspan="1" align="center">225/79</td></tr><tr><td rowspan="1" colspan="1" align="left">Paternal asthma, no/yes</td><td rowspan="1" colspan="1" align="center">301/80</td><td rowspan="1" colspan="1" align="center">243/55</td><td rowspan="1" colspan="1" align="center">219/73</td></tr><tr><td rowspan="1" colspan="1" align="left">Vitamin D levels</td><td rowspan="1" colspan="1" align="center">36.7 ± 15.2</td><td rowspan="1" colspan="1" align="center">37.1 ± 16.8</td><td rowspan="1" colspan="1" align="center">39.9 ± 14.9</td></tr><tr><td rowspan="1" colspan="1" align="left">Mother smoked while pregnant, no/yes</td><td rowspan="1" colspan="1" align="center">349/65</td><td rowspan="1" colspan="1" align="center">266/46</td><td rowspan="1" colspan="1" align="center">271/39</td></tr><tr><td rowspan="1" colspan="1" align="left">Mother, dad, or other smoked after birth, no/yes</td><td rowspan="1" colspan="1" align="center">57/361</td><td rowspan="1" colspan="1" align="center">42/270</td><td rowspan="1" colspan="1" align="center">37/274</td></tr></tbody></table></div><div id="largeobj_idm140074067284048" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4214107/table/tbl1/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec6" class="sec"><h3 id="__sec6title">Lung Function over Time and Drug Effect Model Development</h3><p id="__p14" class="p p-first">Longitudinal FEV<sub>1</sub> (data collected every 2–4 mo in 4-yr period of time, before administering albuterol) was fitted to a nonlinear mixed-effects model with extended least squares regression using the NONMEM (Ellicott City, MD) program. Details regarding model development and covariate selection can be found in the M<span class="small-caps">ethods</span> section of the online supplement.</p><p id="__p15">Interindividual variability on FEV<sub>1</sub> was evaluated using an additive error model:</p><div class="disp-formula" id="fd1"><div class="f"><em>P</em><sub><em>i</em><em>j</em></sub> =&nbsp;<em>P</em><sub><em>T</em><em>V</em><em>j</em></sub> +&nbsp;<em>η</em><sub><em>i</em><em>j</em></sub></div><div class="l">(1)</div></div><p>where <em>P</em><sub><em>i</em><em>j</em></sub> is the true value of the <em>j</em>th parameter for the <em>i</em>th subject. <em>P</em><sub><em>T</em><em>V</em><em>j</em></sub> is the population typical value (TV) for the <em>j</em>th parameter, and <em>η</em><sub><em>ij</em></sub> is an interindividual random effect, which quantifies the deviation of <em>P</em><sub><em>i</em><em>j</em></sub> from <em>P</em><sub><em>T</em><em>V</em><em>j</em></sub> and is assumed to follow a normal distribution with mean of 0 and variance of <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="m6" overflow="scroll"><mrow><msubsup><mi>ω</mi><mi>j</mi><mn>2</mn></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-1-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="m6" overflow="scroll"><mrow><msubsup><mi>ω</mi><mi>j</mi><mn>2</mn></msubsup></mrow></math></script>.</p><p id="__p16" class="p p-last">Intraindividual variability was evaluated using a combined proportional and additive error model as follows:</p><div class="disp-formula" id="fd2"><div class="f">FEV<sub>1<sub>obs</sub></sub> =&nbsp;FEV<sub>1<sub>pred</sub></sub>&nbsp;⋅&nbsp;(1 +&nbsp;<em>ε</em><sub>1</sub>) +&nbsp;<em>ε</em><sub>2</sub></div><div class="l">(2)</div></div><p>where FEV<sub>1<sub>obs</sub></sub> are the observations and FEV<sub>1<sub>pred</sub></sub> is the corresponding model prediction. <em>ε</em><sub>1</sub> and <em>ε</em><sub>2</sub> are independent and normally distributed random variables with zero mean and variance of <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="m12" overflow="scroll"><mrow><msubsup><mi>σ</mi><mn>1</mn><mn>2</mn></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-2-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="m12" overflow="scroll"><mrow><msubsup><mi>σ</mi><mn>1</mn><mn>2</mn></msubsup></mrow></math></script> and <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="m13" overflow="scroll"><mrow><msubsup><mi>σ</mi><mn>2</mn><mn>2</mn></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-3-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="m13" overflow="scroll"><mrow><msubsup><mi>σ</mi><mn>2</mn><mn>2</mn></msubsup></mrow></math></script>, respectively, that account for the residual unexplained variability. The final model was evaluated by a nonparametric bootstrap and the visual predictive check to assess the predictive performance and robustness (<a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562583">17</a>, <a href="#bib18" rid="bib18" class=" bibr popnode">18</a>).</p></div><div id="__sec7" class="sec"><h3 id="__sec7title">GWAS</h3><p id="__p17" class="p p-first-last">The FEV<sub>1</sub> for each patient was calculated with equation 1 using NONMEM. The individual-level parameters (theta<sub>1</sub> and theta<sub>3</sub>; <em>see</em> equation 3 in R<span class="small-caps">esults</span> section) derived from the longitudinal model were the primary phenotypes used, which quantified the rate of change in FEV<sub>1</sub> with age and the baseline FEV<sub>1</sub> level for each child with asthma. Genotype-phenotype associations were calculated assuming an additive genetic model. Details regarding the association analyses can be found in the M<span class="small-caps">ethods</span> section of the online supplement. For comparison with the longitudinal phenotypes, we also conducted GWAS on the single-time-point phenotypes (FEV<sub>1</sub> at 48 mo).</p></div><div id="__sec8" class="sec"><h3 id="__sec8title">Functional Evaluation of Top Single-Nucleotide Polymorphism Associations</h3><p id="__p18" class="p p-first">We annotated the GWAS-identified variants with chromatin status using ChromHMM (<a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562603">19</a>). The WashU Epigenome Browser was used for visualization. Using the R package <em>limma</em>, we reanalyzed GEO data <a id="__tag_410562609" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE18965" ref="/geo/query/acc.cgi?acc=GSE18965"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE18965","term_id":"18965"}}</span>GSE18965</a> from a microarray study of differential expression between (atopic) airway epithelial cells from subjects with asthma and control subjects to determine candidate target genes for the implicated enhancers (<a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562585">20</a>). To evaluate the relative expression of implicated genes in a variety of human tissues, we used RNA-Seq data from the Illumina Human BodyMap 2.0 project (<a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562567">21</a>). For details, see the M<span class="small-caps">ethods</span> section of the online supplement.</p><p id="__p19" class="p p-last">We compared, in a quantile–quantile (Q-Q) plot, the expected and observed distribution of <em>P</em> values from the pooled analysis for those variants mapping to strong enhancers in lung fibroblast. We also performed simulations (n = 100) using single-nucleotide polymorphisms (SNPs) that match the allele frequency and distance to nearest gene of the SNPs that overlap with the strong enhancers and generated the corresponding Q-Q plot for each simulation. We used a false discovery rate (FDR)-based multiple testing correction (<a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562607">22</a>); FDR less than 0.05 was used to declare a significant association.</p></div><div id="__sec9" class="sec sec-last"><h3 id="__sec9title"><strong>Genetic Association with Longitudinal versus Single–Time-Point Phenotype Using Chromatin Profiling Data and the</strong> National Human Genome Research Institute <strong>Catalog</strong></h3><p id="__p20" class="p p-first">Using the SNPs overlapping the chromatin states in normal human lung fibroblast (NHLF), we evaluated the gain in statistical power to detect a quantitative trait locus from the pooled analysis of the longitudinal phenotypes in relation to the single–time-point phenotypes.</p><p id="__p21" class="p p-last">We also compared the <em>P</em> values from each set of phenotypes for the SNPs that have been found to be reproducibly associated with lung function as curated in the National Human Genome Research Institute (NHGRI) catalog of published GWAS.</p></div></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">Results</h2><div id="__sec11" class="sec sec-first"><h3 id="__sec11title">Longitudinal Model of FEV<sub>1</sub> in Children with Asthma</h3><p id="__p22" class="p p-first">Both exponential and linear structural models were evaluated using the CAMP prebronchodilator FEV<sub>1</sub> data. Before testing potential clinical covariates, we established a base model by evaluating age, body weight, height, sex, and body mass index as potential determinants of FEV<sub>1</sub> change over time. Akaike information criterion (AIC) was used to compare the reference structural models (<em>see</em> Table E1). The best prediction for FEV<sub>1</sub> was achieved by an exponential function of age and height:</p><div class="disp-formula" id="fd3"><div class="f">FEV<sub>1</sub> =&nbsp;exp(theta<sub>1</sub>&nbsp;×&nbsp;age +&nbsp;theta<sub>2</sub>&nbsp;×&nbsp;height&nbsp;−&nbsp;theta<sub>3</sub>) +&nbsp;theta<sub>drugeffect</sub></div><div class="l">(3)</div></div><p>In equation 3, theta<sub>1</sub> and theta<sub>2</sub> are the rate of change in FEV<sub>1</sub> associated with age and height, respectively. The theta<sub>3</sub> refers to a baseline level for FEV<sub>1</sub> (i.e., the FEV<sub>1</sub> level at birth assuming the model is applicable to that age range). We evaluated the model that assumes a different definition of baseline level (defined at the mean age of 9, rather than age at birth) and found that the original theta<sub>3</sub> and the new theta<sub>3</sub> were significantly correlated (Spearman correlation of 0.97; <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). A comparison of the two models can be found in the M<span class="small-caps">ethods</span> section of the online supplement and Table E2.</p><p id="__p23">The fit of the base model (including age and height) was not improved by the addition of sex, body weight, and body mass index in children with asthma (<em>P</em> &gt; 0.05). Using the base model, additional covariates (listed in <a href="/pmc/articles/PMC4214107/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>) were evaluated. Among them, only race was found to be a key covariate. The FEV<sub>1</sub> level was similar between whites and Mexican Americans, but it was significantly lower in African American children with asthma. Furthermore, we analyzed the model that assumes an interaction between age and sex. Based on AIC, this interaction model did not perform as well as the base model (<em>see</em> Table E1).</p><p id="__p24">Evaluating the different treatment arms, we observed that the additive drug effect model fit the data better than a proportional model, with lower values of AIC (<em>see</em> Table E1) and objective function value (OFV). We found that budesonide had a minor but statistically significant effect on the prebronchodilator FEV<sub>1</sub> (additive model theta<sub>drugeffect</sub>, <em>P</em> &lt; 0.001). The FEV<sub>1</sub> with long-term treatment of budesonide was predicted to be 0.103 ± 0.129 higher than FEV<sub>1</sub> in the placebo group. Nedocromil did not show any significant effect on FEV<sub>1</sub> when compared with the placebo group. We observed no significant treatment effect on the rate of change in FEV<sub>1</sub> with age. We tested the interaction between age and treatment (<em>see</em> Table E1) and found this model, based on AIC, to perform less optimally than our selected model.</p><p id="__p25">The interindividual variability of theta<sub>2</sub> was very small (&lt;10<sup>−6</sup>), suggesting the rate of change in FEV<sub>1</sub> with height was similar among the subjects. Therefore, in our final model, theta<sub>2</sub>’s interindividual variability was fixed to zero. <a href="/pmc/articles/PMC4214107/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a> illustrates the relationship between the observed and population-predicted FEV<sub>1</sub>, and the relationship between observations and individual predicted FEV<sub>1</sub> values using our final model. Most of the conditional weighted residuals were evenly distributed around 0 (<a href="/pmc/articles/PMC4214107/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1C</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC4214107/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140074026446640" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is rccm.201403-0460OC_f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is rccm.201403-0460OC_f1.jpg" src="/pmc/articles/PMC4214107/bin/rccm.201403-0460OC_f1.jpg"></div></a><div id="largeobj_idm140074026446640" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4214107/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC4214107/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC4214107/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p26">Goodness of fit of final model. (<em>A</em>) Relationship between observed FEV<sub>1</sub> and population typical FEV<sub>1</sub> predictions. (<em>B</em>) Relationship between observed FEV<sub>1</sub> and individual FEV<sub>1</sub> predictions. (<em>C</em>) Conditional weighted residuals (WRES) versus age. Most conditional WRES evenly distributed around 0. The <em>solid lines</em> are <em>diagonal lines</em> of identity in <em>A</em> and <em>B</em>.</p></div></div></div><p id="__p27" class="p">To assess the stability of our model, bootstrapping validation analysis was performed. This analysis showed that the median parameter values and the corresponding relative standard error resulting from the bootstrapping agreed with the estimates from our final model (<a href="/pmc/articles/PMC4214107/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>), suggesting that the final model fitted FEV<sub>1</sub> observations reasonably well and was stable. Most of the observed FEV<sub>1</sub> fell within the 5th–95th percent prediction intervals (<a href="/pmc/articles/PMC4214107/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a>), with less than 10% of the observations lying outside the intervals, suggesting that the final model adequately described most of the observed data. Therefore, we calculated, for each CAMP subject, the individual-level parameters (theta<sub>1</sub> and theta<sub>3</sub>) generated by our final model as given by equation 1. These individual-level parameters were used in the subsequent GWAS analysis as phenotypes to explore the relationship between genetic variants and baseline FEV<sub>1</sub> level or the increase in FEV<sub>1</sub> in children with asthma.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p28">The Estimates of the Parameters from the Final Model (Equation 3)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140074020825072" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Parameters</th><th rowspan="2" colspan="1" align="center">Description</th><th colspan="2" rowspan="1" align="center">NONMEM<hr></th><th colspan="2" rowspan="1" align="center">Bootstrap<hr></th></tr><tr><th rowspan="1" colspan="1" align="center">Estimate</th><th rowspan="1" colspan="1" align="center">RSE (<em>%</em>)<sup>*</sup></th><th rowspan="1" colspan="1" align="center">Estimate</th><th rowspan="1" colspan="1" align="center">RSE (<em>%</em>)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Theta<sub>1</sub></td><td rowspan="1" colspan="1" align="left">Rate of change associated with age</td><td rowspan="1" colspan="1" align="center">0.0137</td><td rowspan="1" colspan="1" align="center">16.1</td><td rowspan="1" colspan="1" align="center">0.0138</td><td rowspan="1" colspan="1" align="center">16.8</td></tr><tr><td rowspan="1" colspan="1" align="left">Theta<sub>2</sub></td><td rowspan="1" colspan="1" align="left">Rate of change associated with height</td><td rowspan="1" colspan="1" align="center">0.0169</td><td rowspan="1" colspan="1" align="center">2.20</td><td rowspan="1" colspan="1" align="center">0.0169</td><td rowspan="1" colspan="1" align="center">2.30</td></tr><tr><td rowspan="1" colspan="1" align="left">Theta<sub>3</sub></td><td rowspan="1" colspan="1" align="left">Intercept among the ethnic groups</td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td rowspan="1" colspan="1" align="center">&nbsp;</td></tr><tr><td colspan="2" rowspan="1" align="left"> White/Mexican American<hr></td><td rowspan="1" colspan="1" align="center">1.89</td><td rowspan="1" colspan="1" align="center">1.78</td><td rowspan="1" colspan="1" align="center">1.89</td><td rowspan="1" colspan="1" align="center">1.84</td></tr><tr><td colspan="2" rowspan="1" align="left"> African American<hr></td><td rowspan="1" colspan="1" align="center">2.04</td><td rowspan="1" colspan="1" align="center">0.575</td><td rowspan="1" colspan="1" align="center">2.04</td><td rowspan="1" colspan="1" align="center">0.641</td></tr><tr><td colspan="2" rowspan="1" align="left"> Others<hr></td><td rowspan="1" colspan="1" align="center">1.95</td><td rowspan="1" colspan="1" align="center">0.742</td><td rowspan="1" colspan="1" align="center">1.94</td><td rowspan="1" colspan="1" align="center">0.787</td></tr><tr><td rowspan="1" colspan="1" align="left">Theta<sub>drugeffect</sub></td><td rowspan="1" colspan="1" align="left">Drug effect of budesonide</td><td rowspan="1" colspan="1" align="center">0.103</td><td rowspan="1" colspan="1" align="center">14.1</td><td rowspan="1" colspan="1" align="center">0.108</td><td rowspan="1" colspan="1" align="center">14.4</td></tr><tr><td colspan="6" rowspan="1" align="left">Interindividual variability (SD) (shrinkage %)<sup>†</sup><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"> Theta<sub>1</sub><hr></td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td colspan="2" rowspan="1" align="center">0.00722 (32.9)<hr></td><td colspan="2" rowspan="1" align="center">0.00723<hr></td></tr><tr><td rowspan="1" colspan="1" align="left"> Theta<sub>3</sub><hr></td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td colspan="2" rowspan="1" align="center">0.0912 (26.4)<hr></td><td colspan="2" rowspan="1" align="center">0.0915<hr></td></tr><tr><td rowspan="1" colspan="1" align="left"> Theta<sub>drugeffect</sub><hr></td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td colspan="2" rowspan="1" align="center">0.129 (67.3)<hr></td><td colspan="2" rowspan="1" align="center">0.123<hr></td></tr><tr><td colspan="6" rowspan="1" align="left">Intraindividual variability (shrinkage %)<sup>‡</sup><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"> Proportional (CV%)<hr></td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td colspan="2" rowspan="1" align="center">5.91 (4.52)<hr></td><td colspan="2" rowspan="1" align="center">5.91<hr></td></tr><tr><td rowspan="1" colspan="1" align="left"> Additive (SD)</td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td colspan="2" rowspan="1" align="center">0.0863 (4.52)</td><td colspan="2" rowspan="1" align="center">0.0859</td></tr></tbody></table></div><div id="largeobj_idm140074020825072" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4214107/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140074029899312"><p id="__p29" class="p p-first-last"><em>Definition of abbreviations</em>: CV = coefficient of variation; NONMEM = nonlinear mixed-effects modeling; RSE = relative standard error.</p></div><div id="tblfn1"><sup>*</sup>Percent RSE (100% × standard error/estimate).</div><div id="tblfn2"><sup>†</sup>The SD across the subjects for each parameter.</div><div id="tblfn3"><sup>‡</sup>The CV and SD for the proportional and additive residual unexplained variability, respectively.</div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC4214107/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140074024667744" class="figure"><a href="/pmc/articles/PMC4214107/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4214107_rccm.201403-0460OC_f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is rccm.201403-0460OC_f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4214107/bin/rccm.201403-0460OC_f2.jpg"></a></div><div id="largeobj_idm140074024667744" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4214107/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC4214107/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p33">Visual predictive check of the final model. The median, the 5th, and 95th percent prediction intervals from visual predictive check simulation were superimposed with the observations. (<em>A</em>) Placebo and nedocromil treatment groups. (<em>B</em>) Budesonide treatment group. The medians of model simulations are shown by <em>solid lines</em> and 95% prediction intervals are encompassed by the <em>broken lines</em>. The <em>gray circles</em> refer to the observed FEV<sub>1</sub>.</p></div></div></div></div><div id="__sec12" class="sec"><h3 id="__sec12title">GWAS</h3><p id="__p34" class="p p-first">Because of the observed ancestry effect on FEV<sub>1</sub> level and the potential for population stratification (and the small sample size of the African-American and Hispanic cohorts), we chose to focus on white subjects enrolled in the CAMP study for genome-wide association analysis. A GWAS was performed on the pooled samples with treatment as a covariate and in the separate placebo (292 subjects) and nedocromil (218 subjects) groups, because there were no observed differences in longitudinal FEV<sub>1</sub> between these two groups. Association analyses between 473,680 genotyped SNPs (which passed quality control) and the two modeled parameters (theta<sub>1</sub> and theta<sub>3</sub>, equation 3) were performed. Forty-six and 53 SNPs were nominally associated with theta<sub>1</sub> (<em>P</em> &lt; 1 × 10<sup>−4</sup>) and theta<sub>3</sub> (<em>P</em> &lt; 1 × 10<sup>−4</sup>), respectively, in the placebo group. For replication, these SNPs were further evaluated (<em>P</em> &lt; 0.05 and concordance of effect) in the nedocromil group. Six SNPs for theta<sub>3</sub> (baseline FEV<sub>1</sub> level) and one SNP (rs17161791) for theta<sub>1</sub> (rate of change in FEV<sub>1</sub> with age) showed associations at <em>P</em> less than 0.05 (<a href="/pmc/articles/PMC4214107/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>) in the nedocromil treatment group. Among them, two intronic SNPs (rs347412, <em>P</em> = 1.39 × 10<sup>−5</sup>; rs238349, <em>P</em> = 1.42 × 10<sup>−5</sup>), on chromosome 13 in the <em>DGKH</em> gene, showed the most significant associations with theta<sub>3</sub>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p35">GWAS Results</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140074029204704" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="center">Chr (Location)</th><th rowspan="2" colspan="1" align="center">Allele<sup>*</sup> (Frequency)</th><th rowspan="2" colspan="1" align="center">Gene</th><th colspan="3" rowspan="1" align="center"><em>P</em> Value (Beta)<hr></th></tr><tr><th rowspan="1" colspan="1" align="center">Placebo</th><th rowspan="1" colspan="1" align="center">Nedocromil</th><th rowspan="1" colspan="1" align="center">Placebo + Nedocromil</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs347412</td><td rowspan="1" colspan="1" align="center">13 (intron)</td><td rowspan="1" colspan="1" align="center"><strong>A</strong>|G (0.5435)</td><td rowspan="1" colspan="1" align="center"><em>DGKH</em></td><td rowspan="1" colspan="1" align="center">1.39 × 10<sup>−5</sup> (0.033)</td><td rowspan="1" colspan="1" align="center">0.0249 (0.019)</td><td rowspan="1" colspan="1" align="center">1.42 × 10<sup>−6</sup> (0.026)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs238349</td><td rowspan="1" colspan="1" align="center">13 (intron)</td><td rowspan="1" colspan="1" align="center"><strong>C</strong>|A (0.5213)</td><td rowspan="1" colspan="1" align="center"><em>DGKH</em></td><td rowspan="1" colspan="1" align="center">1.42 × 10<sup>−5</sup> (0.034)</td><td rowspan="1" colspan="1" align="center">0.0497 (0.016)</td><td rowspan="1" colspan="1" align="center">3.54 × 10<sup>−6</sup> (0.025)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs559389</td><td rowspan="1" colspan="1" align="center">11 (intergenic)</td><td rowspan="1" colspan="1" align="center">T|<strong>C</strong> (0.5762)</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">5.28 × 10<sup>−5</sup> (−0.030)</td><td rowspan="1" colspan="1" align="center">0.0428 (−0.016)</td><td rowspan="1" colspan="1" align="center">9.28 × 10<sup>−6</sup> (−0.023)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9366309</td><td rowspan="1" colspan="1" align="center">6 (intergenic)</td><td rowspan="1" colspan="1" align="center">C|<strong>T</strong> (0.6333)</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">3.32 × 10<sup>−5</sup> (−0.033)</td><td rowspan="1" colspan="1" align="center">0.0348 (−0.017)</td><td rowspan="1" colspan="1" align="center">7.05 × 10<sup>−6</sup> (−0.024)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs6763931</td><td rowspan="1" colspan="1" align="center">3 (intron)</td><td rowspan="1" colspan="1" align="center">G|<strong>A</strong> (0.5675)</td><td rowspan="1" colspan="1" align="center"><em>ZBTB38</em></td><td rowspan="1" colspan="1" align="center">5.90 × 10<sup>−5</sup> (0.031)</td><td rowspan="1" colspan="1" align="center">0.0107 (0.020)</td><td rowspan="1" colspan="1" align="center">4.05 × 10<sup>−6</sup> (0.024)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2304725</td><td rowspan="1" colspan="1" align="center">3 (synonymous)</td><td rowspan="1" colspan="1" align="center">T|<strong>C</strong> (0.7091)</td><td rowspan="1" colspan="1" align="center"><em>SLC6A11</em></td><td rowspan="1" colspan="1" align="center">3.87 × 10<sup>−5</sup> (0.033)</td><td rowspan="1" colspan="1" align="center">0.0270 (0.019)</td><td rowspan="1" colspan="1" align="center">3.04 × 10<sup>−6</sup> (0.026)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs17161791<sup>†</sup></td><td rowspan="1" colspan="1" align="center">7 (intergenic)</td><td rowspan="1" colspan="1" align="center">T|<strong>C</strong> (0.7303)</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">3.01 × 10<sup>−5</sup> (0.002)</td><td rowspan="1" colspan="1" align="center">0.0249 (0.001)</td><td rowspan="1" colspan="1" align="center">1.56 × 10<sup>−6</sup> (0.002)</td></tr></tbody></table></div><div id="largeobj_idm140074029204704" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4214107/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140074069953584"><p id="__p36" class="p p-first-last"><em>Definition of abbreviations</em>: GWAS = genome-wide association studies; SNP = single-nucleotide polymorphism.</p></div><div id="tblfn4"><sup>*</sup>The first allele is the common allele in whites followed by its frequency; bolded allele indicates effect allele.</div><div id="tblfn5"><sup>†</sup>The last SNP is associated with theta<sub>1</sub>, the other six SNPs are associated with theta<sub>3</sub>.</div></div></div><p id="__p39" class="p p-last">The variant allele for each of these four SNPs (rs347412, rs238349, rs6763931 [<em>see</em> Figure E1], and rs2304725) was associated with higher theta<sub>3</sub> and, therefore, lower FEV<sub>1</sub>. For the other two SNPs (rs559389 and rs9366309), the variant allele was found to be associated with lower theta<sub>3</sub> leading to higher FEV<sub>1</sub>. The variant allele of rs17161791 was associated with higher theta<sub>1</sub> resulting in higher FEV<sub>1</sub> increase rate. Not surprisingly, the association <em>P</em> values for the model parameters (theta<sub>1</sub> and theta<sub>3</sub>) at all seven SNPs were improved in the pooled data analysis (placebo + nedocromil) relative to the placebo-alone analysis (<a href="/pmc/articles/PMC4214107/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>).</p></div><div id="__sec13" class="sec"><h3 id="__sec13title">Functional Evaluation of Top GWAS Associations</h3><p id="__p40" class="p p-first">To functionally characterize our top SNP associations, we used ChromHMM (<a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562590">23</a>) applied to ENCODE data (<a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562570">19</a>, <a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562562">24</a>) from NHLF and a lymphoblastoid cell line (GM12878). We found that rs6763931 (intronic to <em>ZBTB38</em> gene) overlaps a strong enhancer state in both NHLF and GM12878 (<a href="/pmc/articles/PMC4214107/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure 3</span></a>). Consistent with this, we observed that the same SNP coincides with an active transcription start site in fetal lung fibroblast cells (IMR90; <em>see</em> Figure E2). Differential expression analysis of the genes at this locus between children with atopy and with asthma and nonatopic healthy individuals identified a nearby gene (103 kb away), <em>RASA2</em>, that was highly differentially expressed (<em>P</em> = 0.002; <em>see</em> Figure E3); in contrast, the host gene (<em>ZBTB38</em>) showed no evidence of differential expression (<em>P</em> = 0.21; <em>see</em> Figure E4) (<a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562596">20</a>). (In all, four genes [<em>RASA2</em>, <em>ZBTB38</em>, <em>RNF7</em>, and <em>SLC25A36</em>] in a 1-Mb region centered at the SNP rs6763931 were tested; thus the differential expression of <em>RASA2</em> meets Bonferroni significance.) The nongenic SNP rs559389, another top association, is in strong linkage disequilibrium (<em>r</em><sup><em>2</em></sup>&gt;0.80 in 1,000 Genomes EUR) with variants (rs538322, rs3018303, and rs12366105) that also overlap regions of strong enhancer histone marks in NHLF.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC4214107/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140074070322704" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is rccm.201403-0460OC_f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is rccm.201403-0460OC_f3.jpg" src="/pmc/articles/PMC4214107/bin/rccm.201403-0460OC_f3.jpg"></div></a><div id="largeobj_idm140074070322704" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4214107/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a target="object" href="/pmc/articles/PMC4214107/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC4214107/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure 3.</a></div><!--caption a7--><div class="caption"><p id="__p41">Regulatory function of rs6763931. The single-nucleotide polymorphism rs6763931 (located in an intron of <em>ZBTB38</em>, <em>black arrow</em>) overlaps a strong enhancer in normal human lung fibroblast (NHLF) and in a lymphoblastoid cell line (GM12878).</p></div></div></div><p id="__p42">Finally, we tested our top SNPs for association with expression in a variety of tissues. We found that rs238349 is a cis-acting expression quantitative trait locus in lung for diacylglycerol kinase, eta (<em>DGKH</em>; <em>P</em> = 5.2 × 10<sup>−5</sup>), using public GTEx RNA-seq data (Broad) (<a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562584">25</a>). <em>DGKH</em> is most highly expressed in prostate and lung in a comparison of 16 human tissues (<em>see</em> Table E3).</p><p id="__p43" class="p p-last">Taken together, these results provide strong evidence that our top SNPs are likely to mediate their phenotypic effect via transcriptional mechanisms in lung fibroblast and/or an immune-related tissue.</p></div><div id="__sec14" class="sec sec-last"><h3 id="__sec14title">Genetic Association with Longitudinal versus Single–Time-Point Phenotype</h3><div id="__sec15" class="sec sec-first"><p></p><h4 id="__sec15title" class="inline">Comparison for genetic variants in enhancer regions in lung fibroblast cells. </h4><p id="__p44" class="p p-first">Because of the small sample size, we did not expect any SNPs to reach genome-wide significance according to a conservative Bonferroni adjustment. Remarkably, the Q-Q plot in the pooled analysis restricted to regions enriched for functional SNPs (e.g., see <a href="/pmc/articles/PMC4214107/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure 4</span></a> for the Q-Q plot of SNPs overlapping strong enhancer regions in human lung fibroblast; n = 10,751 interrogated SNPs) showed a highly significant association (rs6763931; FDR &lt;0.05; <em>P</em> = 4.05 × 10<sup>−6</sup>) with theta<sub>3</sub>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC4214107/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140074029928144" class="figure"><a href="/pmc/articles/PMC4214107/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4214107_rccm.201403-0460OC_f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is rccm.201403-0460OC_f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4214107/bin/rccm.201403-0460OC_f4.jpg"></a></div><div id="largeobj_idm140074029928144" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4214107/figure/fig4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC4214107/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Figure 4.</a></div><!--caption a7--><div class="caption"><p id="__p45">Quantile–quantile plot for the single-nucleotide polymorphisms (SNPs) overlapping strong enhancers. A highly significant association (rs6763931, FDR &lt; 0.05) was identified using SNPs in regulatory regions in human lung fibroblast. The <em>P</em> values obtained from associations of the enhancer SNPs with theta<sub>3</sub> are shown as <em>circles</em>. The <em>horizontal line</em> at 0.05/N (significance level after Bonferroni correction) is also shown. FDR = false discovery rate.</p></div></div></div><p id="__p46" class="p">As expected, GWAS of single–time-point phenotypes in the pooled dataset yielded no genome-wide significant findings, nor was there a Bonferroni-adjusted significant association with any single–time-point phenotypes among the SNPs in strong enhancer regions in NHLF. <a href="/pmc/articles/PMC4214107/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a>, for example, shows a comparison of the association results between our longitudinal phenotypes and FEV<sub>1</sub> at 48 months (single time-point) for SNPs that intersect strong enhancer states in NHLF. Furthermore, none of the simulated datasets (n = 100) showed a significant association (FDR &lt; 0.05).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>Table 4.</h3><!--caption a7--><div class="caption"><p id="__p47">Comparison between Longitudinal Phenotype and Single–Time-Point Phenotype for SNPs Located in Strong Enhancer States in Human Lung Fibroblast and SNPs Known to Be Reproducibly Associated with Lung Function as Curated in the NHGRI Catalog</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140074025666640" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Analysis Using Enhancer SNPs in Lung Fibroblast</th><th rowspan="1" colspan="1" align="center">FDR &lt; 0.05</th><th rowspan="1" colspan="1" align="center">FDR &lt; 0.10</th><th rowspan="1" colspan="1" align="center">Bonferroni (0.05/N)</th><th rowspan="1" colspan="1" align="center">Analysis Using Known Lung Function-associated SNPs in the NHGRI Catalog</th><th rowspan="1" colspan="1" align="center"><em>P</em> &lt; 0.05</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Longitudinal</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">Longitudinal</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left">FEV<sub>1</sub> at 48 mo</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">FEV<sub>1</sub> at 48 mo</td><td rowspan="1" colspan="1" align="center">0</td></tr><tr><td rowspan="1" colspan="1" align="left">Permuted longitudinal data (n = 100)</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td rowspan="1" colspan="1" align="center">&nbsp;</td></tr></tbody></table></div><div id="largeobj_idm140074025666640" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4214107/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140074028346320"><p id="__p48" class="p p-first-last"><em>Definition of abbreviations</em>: FDR = false discovery rate; NHGRI = National Human Genome Research Institute; SNP = single-nucleotide polymorphism.</p></div></div></div></div><div id="__sec16" class="sec sec-last"><p></p><h4 id="__sec16title" class="inline">Comparison for pulmonary function associated SNPs curated in the NHGRI catalog. </h4><p id="__p49" class="p p-first-last">The NHGRI GWAS catalog (<a href="#bib26" rid="bib26" class=" bibr popnode">26</a>) lists more than 50 SNPs that have been found to be associated with at least one of the pulmonary functional terms (represented by FEV<sub>1</sub> or FEV<sub>1</sub>/FVC or forced expiratory flow). Of these, 23 were genotyped in the CAMP dataset (<em>see</em> Table E4). Consistent with the observed improvement to detect a significant association with a longitudinal phenotype, but not with a single–time-point phenotype, using SNPs that overlap regulatory regions, no reproducible lung function–associated SNP as curated in NHGRI catalog (<a href="#bib26" rid="bib26" class=" bibr popnode">26</a>) showed a nominally significant association (<em>P</em> &lt; 0.05) (<a href="/pmc/articles/PMC4214107/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a>) with the single–time-point phenotypes; in contrast, we observed five SNP associations with our longitudinal phenotypes. These include rs4762767 and rs58667 for theta<sub>1</sub> and rs1291183, rs12984174, and rs2571445 for theta<sub>3</sub>.</p></div></div></div><div id="__sec17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec17title">Discussion</h2><p id="__p50" class="p p-first">In this study, we developed an integrative method that combines population-based mixed-effects modeling and GWAS to identify SNPs that may contribute to baseline FEV<sub>1</sub> or rate of change in FEV<sub>1</sub> with age in children with asthma. Current approaches to finding disease susceptibility loci are primarily based on single-time-point phenotypes yielding results that reflect only a snapshot of the dynamic biology of disease. These approaches, which focus on limited observations, are prone to bias. Population-based mixed-effects modeling, used in this study, considers all FEV<sub>1</sub> observations by concurrently fitting them. The random errors can be appropriately accounted for by repeated measures per subjects (<a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562564">9</a>). Additionally, FEV<sub>1</sub> is highly dependent on the pathophysiologic condition in children (<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562581">11</a>–<a href="#bib14" rid="bib14" class=" bibr popnode">14</a>). A significant association between the genotype and FEV<sub>1</sub> observed at a selected time-point may be related to a confounding factor (e.g., transient bronchospasm) rather than lung function. The individual-level parameters from our model (used as phenotypes here), which are defined independently of the clinical and pathophysiologic terms, reflect baseline lung function level and lung function progression in this disease setting resulting in a more appropriate and comprehensive understanding of lung function in children with asthma. The approach developed here can be extended to other diseases and/or drug effect (<a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562598">27</a>) with longitudinal data. Sikorska and coworkers (<a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562573">28</a>) proposed a two-step integrative method using the classical linear mixed-effect model as reference. Our method applies nonlinear mixed-effect modeling, and both linear and nonlinear models were tested.</p><p id="__p51">Demonstrating the benefit of using longitudinal modeling of disease, we observed that budesonide had a small but significant effect on FEV<sub>1</sub> with ΔOFV of 19.34 when using mixed-effects modeling in FEV<sub>1</sub> over a period of up to 6 years. This treatment benefit was not reported in the previous CAMP study [<a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562595">2</a>], when % FEV<sub>1</sub> change after bronchodilator at a selected time-point was compared with that of baseline. The initial CAMP study nevertheless concluded that inhaled corticosteroids, such as budesonide, are still useful, because they provide better control of asthma resulting in fewer hospitalizations and urgent care visits to a caregiver and reduced albuterol treatment for symptoms (<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562574">15</a>). Our study found the justification for the conclusion, because we observed a significant treatment effect of budesonide on FEV<sub>1</sub> level in children with asthma by using disease progression modeling. In our study, we did not detect any drug effect for nedocromil; indeed, the inclusion of drug effect did not give a significant change in OFV. This is consistent with the previous report based on the same dataset (<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562569">15</a>). The longitudinal modeling described here may provide a powerful tool to capture the long-term drug effect over time. When applied in clinical trials, our approach may detect additional drug effects that would be missed by the standard single–time-point strategy. This is consistent with some other previous reports in both healthy subjects and subjects with asthma (<a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562577">13</a>, <a href="#bib14" rid="bib14" class=" bibr popnode">14</a>).</p><p id="__p52">We have demonstrated that age and height are the essential physiologic determinants of lung function growth in children with asthma, and race is a key covariate for FEV<sub>1</sub> level. Hankinson and coworkers (<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562599">11</a>) reported lung function reference values in nonsmokers between the ages of 8 and 80 years. They also found age and height were predictive variables for FEV<sub>1</sub> function in children, consistent with the model developed in this study. Interestingly, in their study, whites and Mexican Americans had similar FVC and FEV<sub>1</sub>. The values for FVC and FEV<sub>1</sub> were higher than in African Americans, which was corroborated by our results. Another study reported that age, height, and body weight were predictors of FEV<sub>1</sub> in patients with asthma ranging in age from 6 to 88 (<a href="#bib14" rid="bib14" class=" bibr popnode">14</a>). However, in our study, body weight did not improve the goodness of fit (ΔOFV &lt;3.84; <em>P</em> &gt; 0.05 after addition) likely because of the high correlation between height and body weight in children. Both exponential and linear models were used to describe the time course of FEV<sub>1</sub> in previous studies (<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562600">11</a>–<a href="#bib14" rid="bib14" class=" bibr popnode">14</a>). Exponential function had the best fit for our model (<em>see</em> Table E1). Sex was also included in a previous report (<a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562602">12</a>), although it did not show a significant difference in our study (ΔOFV &lt;3.84, after addition).</p><p id="__p53">We identified seven SNPs nominally associated with the modeled FEV<sub>1</sub> parameters in both placebo and nedocromil groups. Of these, two intronic SNPs (rs347412 and rs238349 in the <em>DGKH</em> gene) and another SNP (rs2304725 in <em>SLC6A11</em> gene) have been reported to be associated with smoking cessation (<a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562597">29</a>). Another two SNPs have been reported to be associated with height, namely rs347412 (<a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562565">30</a>) and rs6763931 (<em>P</em> = 5.90 × 10<sup>−5</sup>) (<a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562579">31</a>–<a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562606">33</a>). The latter one has also been implicated in growth impairment (<a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562601">34</a>). We interpret these findings to indicate that the SNPs we identified may also be implicated in physiologic growth on a macro scale in children, an observation that needs to be further validated.</p><p id="__p54">GWAS of single–time-point phenotypes have been conducted for the discovery of genetic determinants of lung function and/or asthma risk in large human populations. Of the SNPs associated with pulmonary function in the NHGRI catalog (<a href="#bib26" rid="bib26" class=" bibr popnode">26</a>), 23 were genotyped in the CAMP dataset. Of these, we found five SNPs were suggestively associated with baseline level and progression of FEV<sub>1</sub> (<em>P</em> &lt; 0.05) in our study: rs4762767 and rs58667 for theta<sub>1</sub> and rs1291183, rs12984174, and rs2571445 for theta<sub>3</sub>. Specifically, rs1291183, located within the gene <em>YES1</em> on chromosome 18, was previously reported to be associated with percent predicted FEV<sub>1</sub> and percent predicted FVC (<em>P</em> = 3.54×10<sup>−6</sup> and 5.47×10<sup>−5</sup>, respectively) in populations of European descent with asthma (<a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562572">8</a>). This SNP was associated with our longitudinal phenotype (theta<sub>3</sub> at <em>P</em> &lt; 0.05) in placebo and nedocromil treatment groups. In addition, the SNPs rs2571445 and rs58667 were reported to be associated with FEV<sub>1</sub> or percent predicted FEV<sub>1</sub> with <em>P</em> = 1.11×10<sup>−12</sup> and 3.95×10<sup>−7</sup> in individuals of European ancestry (with rs58667, the association was seen in patients with asthma of European ancestry) (<a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562589">5</a>, <a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562593">8</a>). rs12984174 was previously reported to be associated with percent predicted FVC (8.89×10<sup>−6</sup>) in subjects with asthma (<a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562576">8</a>), as was rs4762767 with pulmonary function as measured by FEV<sub>1</sub>/FVC (<a href="#bib35" rid="bib35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562571">35</a>).</p><p id="__p55">Besides replicating these five previously reported SNPs, our study discovered seven additional loci that may be linked to lung function progression in asthma, suggesting improved biologic discovery from a longitudinal modeling approach. Indeed, trait mapping using a longitudinal phenotype, but not the single–time-point phenotypes, restricted to genetic variants that overlap enhancer regions in lung fibroblast identified a highly significant association. We should note that we were still underpowered to detect associations with the rate of change in FEV<sub>1</sub> versus the baseline level even with the use of functional data. However, both theta<sub>1</sub> and theta<sub>3</sub> showed improved replication relative to single–time-point phenotypes with respect to previously identified loci (as found in the NHGRI catalog). Longitudinal molecular-level and gene expression investigations in relevant cell types may further improve biologic discovery.</p><p id="__p56">One limitation of our study, common in asthma genetic studies, is the small sample size, which results in only nominally significant findings from the GWAS. However, the use of functional and epigenomic datasets in relevant cell types to prioritize genetic variants allowed us to discover loci that pass Bonferroni significance (<a href="/pmc/articles/PMC4214107/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure 4</span></a>), and our longitudinal approach (in contrast to the use of single–time-point phenotypes) enabled us to confirm lung function loci previously identified by other GWAS. One notable finding from the molecular and epigenomic datasets used here is the differential expression in children with atopy with asthma versus control subjects of an adjacent gene (<em>RASA2</em>), and not the host gene (<em>ZBTB38</em>), to the regulatory variant rs6763931 that overlaps an enhancer region in lung fibroblast. Although the exact mechanism for this connection remains to be fully elucidated, this finding is consistent with several recent studies (<a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562605">36</a>, <a href="#bib37" rid="bib37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_410562566">37</a>) showing the distal regulatory effects (and proposing potential mechanisms), at several hundred kilobases, of (noncoding) enhancer SNPs associated with complex human phenotypes. Future studies on the functional connection between rs6763931 and <em>RASA2</em> are warranted.</p><p id="__p57" class="p p-last">In summary, our study developed a genetic locus mapping approach that combines nonlinear mixed-effects longitudinal modeling of phenotype and GWAS. This integrative approach allows us to identify new SNPs associated with longitudinal lung function in childhood asthma. These may offer insights into the mechanism underlying pulmonary function regulation in subjects with asthma and may further indicate potential treatment targets.</p></div><div id="idm140074069534432" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140074069534432title">Acknowledgment</h2><div class="sec"><div id="__sec18" class="sec sec-first"><h3 id="__sec18title"></h3><p id="__p58" class="p p-first-last">The authors thank the National Institutes of Health GWAS Data Repository; the Contributing Investigators who contributed the phenotype data and DNA samples from their original study; and the NHLBI in collaboration with CARE, CAMP, and ACRN Clinical Network Investigators and the associated Data Coordinating Centers for their support to these studies.</p></div></div></div><div id="idm140074069532816" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140074069532816title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140074030191280"></p><p id="__p59" class="p p-first-last">Supported by the National Center for Advancing Translational Sciences of the National Institutes of Health (NIH; <span>UL1RR024999</span>), NIH/NIGMS grant <span>U01GM61393</span>, and by NIH/NIGMS grant <span>K08GM089941</span>. R.S.H. also received support from NIH/NCI grant <span>R21 CA139278</span>, University of Chicago Cancer Center Support Grant (#P30 CA14599), and Breast Cancer SPORE Career Development Award (CA125183). E.R.G. and N.J.C. are supported by R01 MH101820 and R01 MH090937. S.T.W. and K.G.T. are supported by U01 HL065899 and R01 NR013391. The Genotype-Tissue Expression (GTEx) Project was supported by the Common Fund of the Office of the Director of the NIH (commonfund.nih.gov/GTEx). Additional funds were provided by the NCI, NHGRI, NHLBI, NIDA, NIMH, and NINDS. Donors were enrolled at Biospecimen Source Sites funded by NCI\SAIC-Frederick, Inc. (SAIC-F) subcontracts to the National Disease Research Interchange (10XS170), Roswell Park Cancer Institute (10XS171), and Science Care, Inc. (X10S172). The Laboratory, Data Analysis, and Coordinating Center was funded through a contract (HHSN268201000029C) to the Broad Institute, Inc. Biorepository operations were funded through an SAIC-F subcontract to Van Andel Institute (10ST1035). Additional data repository and project management were provided by SAIC-F (HHSN261200800001E). The Brain Bank was supported by a supplement to University of Miami grant DA006227. The datasets used for the analyses described in this manuscript were obtained from dbGaP through dbGaP accession number phs000424.v3.p1.c1 on 12/13/2013.</p><p></p><p class="fn sec" id="idm140074068552208"></p><p id="__p60" class="p p-first-last">Author contributions: Conceived and designed the study, K.W., E.R.G., H.K.I., M.J.R., and R.S.H. Performed the study, K.W., E.R.G., P.G., and R.S.H. Contributed reagents/materials/analysis tools, S.R.W., J.S., S.T.W., G.L.C., K.G.T., N.J.C., and M.J.R. Wrote the manuscript, K.W., E.R.G., and R.S.H. Edited and approved the manuscript, K.W., E.R.G., H.K.I., P.G., S.R.W., J.S., G.L.C., S.T.W., K.G.T., N.J.C., M.J.R., and R.S.H. Supervised the project, N.J.C., M.J.R., and R.S.H.</p><p></p><p class="fn sec" id="idm140074020491232"></p><p id="__p61" class="p p-first-last">This article has an online supplement, which is accessible from this issue's table of contents at <a href="http://www.atsjournals.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.atsjournals.org</a></p><p></p><p class="fn sec" id="idm140074020489760"></p><p id="__p62" class="p p-first-last">Originally Published in Press as DOI: <a href="http://dx.doi.org/10.1164/rccm.201403-0460OC" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">10.1164/rccm.201403-0460OC</a> on August 5, 2014</p><p></p><p class="fn sec" id="idm140074034075872"></p><p id="__p63" class="p p-first-last"><span class="underline"><strong><a href="http://www.atsjournals.org/doi/suppl/10.1164/rccm.201403-0460OC/suppl_file/disclosures.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Author disclosures</a></strong></span> are available with the text of this article at <a href="http://www.atsjournals.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.atsjournals.org</a>.</p><p></p></div></div><div id="idm140074034073040" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140074034073040title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1.  <span class="element-citation">Covar RA, Spahn JD, Murphy JR, Szefler SJ Childhood Asthma Management Program Research Group. Progression of asthma measured by lung function in the childhood asthma management program. <span><span class="ref-journal"><em>Am J Respir Crit Care Med</em>. </span>2004;<span class="ref-vol">170</span>:234–241.</span> [<a href="/pubmed/15028558" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Respir+Crit+Care+Med&amp;title=Progression+of+asthma+measured+by+lung+function+in+the+childhood+asthma+management+program&amp;author=RA+Covar&amp;author=JD+Spahn&amp;author=JR+Murphy&amp;author=SJ+Szefler&amp;volume=170&amp;publication_year=2004&amp;pages=234-241&amp;pmid=15028558&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2.  <span class="element-citation">Hancock DB, Romieu I, Shi M, Sienra-Monge JJ, Wu H, Chiu GY, Li H, del Rio-Navarro BE, Willis-Owen SA, Weiss ST, et al.  Genome-wide association study implicates chromosome 9q21.31 as a susceptibility locus for asthma in Mexican children. <span><span class="ref-journal"><em>PLoS Genet</em>. </span>2009;<span class="ref-vol">5</span>:e1000623.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2722731/">PMC free article</a>]</span> [<a href="/pubmed/19714205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+implicates+chromosome+9q21.31+as+a+susceptibility+locus+for+asthma+in+Mexican+children&amp;author=DB+Hancock&amp;author=I+Romieu&amp;author=M+Shi&amp;author=JJ+Sienra-Monge&amp;author=H+Wu&amp;volume=5&amp;publication_year=2009&amp;pages=e1000623&amp;pmid=19714205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3.  <span class="element-citation">Hirota T, Takahashi A, Kubo M, Tsunoda T, Tomita K, Doi S, Fujita K, Miyatake A, Enomoto T, Miyagawa T, et al.  Genome-wide association study identifies three new susceptibility loci for adult asthma in the Japanese population. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2011;<span class="ref-vol">43</span>:893–896.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4310726/">PMC free article</a>]</span> [<a href="/pubmed/21804548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+three+new+susceptibility+loci+for+adult+asthma+in+the+Japanese+population&amp;author=T+Hirota&amp;author=A+Takahashi&amp;author=M+Kubo&amp;author=T+Tsunoda&amp;author=K+Tomita&amp;volume=43&amp;publication_year=2011&amp;pages=893-896&amp;pmid=21804548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4.  <span class="mixed-citation">Li X, Howard TD, Zheng SL, Haselkorn T, Peters SP, Meyers DA, Bleecker ER. <span class="ref-title">Genome-wide association study of asthma identifies RAD50–IL13 and HLA-DR/DQ regions </span><span class="ref-journal"><em>J Allergy Clin Immunol</em></span> 2010<span class="ref-vol">125</span>328–335.e311 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2824608/">PMC free article</a>]</span> [<a href="/pubmed/20159242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=Genome-wide+association+study+of+asthma+identifies+RAD50–IL13+and+HLA-DR/DQ+regions&amp;author=X+Li&amp;author=TD+Howard&amp;author=SL+Zheng&amp;author=T+Haselkorn&amp;author=SP+Peters&amp;volume=125&amp;publication_year=2010&amp;pages=328-335&amp;pmid=20159242&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5.  <span class="element-citation">Repapi E, Sayers I, Wain LV, Burton PR, Johnson T, Obeidat M, Zhao JH, Ramasamy A, Zhai G, Vitart V, et al.  Wellcome Trust Case Control Consortium; NSHD Respiratory Study Team. Genome-wide association study identifies five loci associated with lung function. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2010;<span class="ref-vol">42</span>:36–44.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2862965/">PMC free article</a>]</span> [<a href="/pubmed/20010834" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+five+loci+associated+with+lung+function&amp;author=E+Repapi&amp;author=I+Sayers&amp;author=LV+Wain&amp;author=PR+Burton&amp;author=T+Johnson&amp;volume=42&amp;publication_year=2010&amp;pages=36-44&amp;pmid=20010834&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6.  <span class="element-citation">Hancock DB, Eijgelsheim M, Wilk JB, Gharib SA, Loehr LR, Marciante KD, Franceschini N, van Durme YM, Chen TH, Barr RG, et al.  Meta-analyses of genome-wide association studies identify multiple loci associated with pulmonary function. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2010;<span class="ref-vol">42</span>:45–52.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2832852/">PMC free article</a>]</span> [<a href="/pubmed/20010835" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analyses+of+genome-wide+association+studies+identify+multiple+loci+associated+with+pulmonary+function&amp;author=DB+Hancock&amp;author=M+Eijgelsheim&amp;author=JB+Wilk&amp;author=SA+Gharib&amp;author=LR+Loehr&amp;volume=42&amp;publication_year=2010&amp;pages=45-52&amp;pmid=20010835&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7.  <span class="element-citation">Li X, Howard TD, Moore WC, Ampleford EJ, Li H, Busse WW, Calhoun WJ, Castro M, Chung KF, Erzurum SC, et al.  Importance of hedgehog interacting protein and other lung function genes in asthma. <span><span class="ref-journal"><em>J Allergy Clin Immunol</em>. </span>2011;<span class="ref-vol">127</span>:1457–1465.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3105202/">PMC free article</a>]</span> [<a href="/pubmed/21397937" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=Importance+of+hedgehog+interacting+protein+and+other+lung+function+genes+in+asthma&amp;author=X+Li&amp;author=TD+Howard&amp;author=WC+Moore&amp;author=EJ+Ampleford&amp;author=H+Li&amp;volume=127&amp;publication_year=2011&amp;pages=1457-1465&amp;pmid=21397937&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8.  <span class="mixed-citation">Li X, Hawkins GA, Ampleford EJ, Moore WC, Li H, Hastie AT, Howard TD, Boushey HA, Busse WW, Calhoun WJ, et al.  <span class="ref-title">Genome-wide association study identifies TH1 pathway genes associated with lung function in asthmatic patients </span><span class="ref-journal"><em>J Allergy Clin Immunol</em></span> 2013<span class="ref-vol">132</span>313–320.e15 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3746327/">PMC free article</a>]</span> [<a href="/pubmed/23541324" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=Genome-wide+association+study+identifies+TH1+pathway+genes+associated+with+lung+function+in+asthmatic+patients&amp;author=X+Li&amp;author=GA+Hawkins&amp;author=EJ+Ampleford&amp;author=WC+Moore&amp;author=H+Li&amp;volume=132&amp;publication_year=2013&amp;pages=313-320&amp;pmid=23541324&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9.  <span class="element-citation">Mould DR. Models for disease progression: new approaches and uses. <span><span class="ref-journal"><em>Clin Pharmacol Ther</em>. </span>2012;<span class="ref-vol">92</span>:125–131.</span> [<a href="/pubmed/22617225" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Models+for+disease+progression:+new+approaches+and+uses&amp;author=DR+Mould&amp;volume=92&amp;publication_year=2012&amp;pages=125-131&amp;pmid=22617225&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10.  <span class="element-citation">Mould DR, Denman NG, Duffull S. Using disease progression models as a tool to detect drug effect. <span><span class="ref-journal"><em>Clin Pharmacol Ther</em>. </span>2007;<span class="ref-vol">82</span>:81–86.</span> [<a href="/pubmed/17507925" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Using+disease+progression+models+as+a+tool+to+detect+drug+effect&amp;author=DR+Mould&amp;author=NG+Denman&amp;author=S+Duffull&amp;volume=82&amp;publication_year=2007&amp;pages=81-86&amp;pmid=17507925&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11.  <span class="element-citation">Hankinson JL, Odencrantz JR, Fedan KB. Spirometric reference values from a sample of the general U.S. population. <span><span class="ref-journal"><em>Am J Respir Crit Care Med</em>. </span>1999;<span class="ref-vol">159</span>:179–187.</span> [<a href="/pubmed/9872837" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Respir+Crit+Care+Med&amp;title=Spirometric+reference+values+from+a+sample+of+the+general+U.S.+population&amp;author=JL+Hankinson&amp;author=JR+Odencrantz&amp;author=KB+Fedan&amp;volume=159&amp;publication_year=1999&amp;pages=179-187&amp;pmid=9872837&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12.  <span class="element-citation">Hibbert ME, Lannigan A, Landau LI, Phelan PD. Lung function values from a longitudinal study of healthy children and adolescents. <span><span class="ref-journal"><em>Pediatr Pulmonol</em>. </span>1989;<span class="ref-vol">7</span>:101–109.</span> [<a href="/pubmed/2797918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr+Pulmonol&amp;title=Lung+function+values+from+a+longitudinal+study+of+healthy+children+and+adolescents&amp;author=ME+Hibbert&amp;author=A+Lannigan&amp;author=LI+Landau&amp;author=PD+Phelan&amp;volume=7&amp;publication_year=1989&amp;pages=101-109&amp;pmid=2797918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13.  <span class="element-citation">Kristufek P, Brezina M, Ciutti P, Strmen J, Mayer M. Reference values and modelling of lung function development as a transcendent function of age, body height and mass. <span><span class="ref-journal"><em>Bull Eur Physiopathol Respir</em>. </span>1987;<span class="ref-vol">23</span>:139–147.</span> [<a href="/pubmed/3607328" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bull+Eur+Physiopathol+Respir&amp;title=Reference+values+and+modelling+of+lung+function+development+as+a+transcendent+function+of+age,+body+height+and+mass&amp;author=P+Kristufek&amp;author=M+Brezina&amp;author=P+Ciutti&amp;author=J+Strmen&amp;author=M+Mayer&amp;volume=23&amp;publication_year=1987&amp;pages=139-147&amp;pmid=3607328&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14.  <span class="mixed-citation">Ma G, Olsson B, Rosenborg J, Karlsson M.Quantifying lung function progression in asthma. Presented at the Annual Meeting of the Population Approach Group in Europe. June 9, 2009, St. Petersburg, Russia, p. 18</span></div><div class="ref-cit-blk half_rhythm" id="bib15">15.  <span class="element-citation">The Childhood Asthma Management Program Research Group. Long-term effects of budesonide or nedocromil in children with asthma. <span><span class="ref-journal"><em>N Engl J Med</em>. </span>2000;<span class="ref-vol">343</span>:1054–1063.</span> [<a href="/pubmed/11027739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Long-term+effects+of+budesonide+or+nedocromil+in+children+with+asthma&amp;volume=343&amp;publication_year=2000&amp;pages=1054-1063&amp;pmid=11027739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16.  <span class="element-citation">Childhood Asthma Management Program Research Group. The Childhood Asthma Management Program (CAMP): design, rationale, and methods. <span><span class="ref-journal"><em>Control Clin Trials</em>. </span>1999;<span class="ref-vol">20</span>:91–120.</span> [<a href="/pubmed/10027502" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Control+Clin+Trials&amp;title=The+Childhood+Asthma+Management+Program+(CAMP):+design,+rationale,+and+methods&amp;volume=20&amp;publication_year=1999&amp;pages=91-120&amp;pmid=10027502&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17.  <span class="element-citation">Yano Y, Beal SL, Sheiner LB. Evaluating pharmacokinetic/pharmacodynamic models using the posterior predictive check. <span><span class="ref-journal"><em>J Pharmacokinet Pharmacodyn</em>. </span>2001;<span class="ref-vol">28</span>:171–192.</span> [<a href="/pubmed/11381569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pharmacokinet+Pharmacodyn&amp;title=Evaluating+pharmacokinetic/pharmacodynamic+models+using+the+posterior+predictive+check&amp;author=Y+Yano&amp;author=SL+Beal&amp;author=LB+Sheiner&amp;volume=28&amp;publication_year=2001&amp;pages=171-192&amp;pmid=11381569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18.  <span class="mixed-citation">Karlsson M, Holford N.A tutorial on visual predictive checks. Presented at the Annual Meeting of the Population Approach Group in Europe. June 20, 2008, Marseille, France. p. 17</span></div><div class="ref-cit-blk half_rhythm" id="bib19">19.  <span class="element-citation">Ernst J, Kheradpour P, Mikkelsen TS, Shoresh N, Ward LD, Epstein CB, Zhang X, Wang L, Issner R, Coyne M, et al.  Mapping and analysis of chromatin state dynamics in nine human cell types. <span><span class="ref-journal"><em>Nature</em>. </span>2011;<span class="ref-vol">473</span>:43–49.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3088773/">PMC free article</a>]</span> [<a href="/pubmed/21441907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mapping+and+analysis+of+chromatin+state+dynamics+in+nine+human+cell+types&amp;author=J+Ernst&amp;author=P+Kheradpour&amp;author=TS+Mikkelsen&amp;author=N+Shoresh&amp;author=LD+Ward&amp;volume=473&amp;publication_year=2011&amp;pages=43-49&amp;pmid=21441907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20.  <span class="element-citation">Kicic A, Hallstrand TS, Sutanto EN, Stevens PT, Kobor MS, Taplin C, Paré PD, Beyer RP, Stick SM, Knight DA. Decreased fibronectin production significantly contributes to dysregulated repair of asthmatic epithelium. <span><span class="ref-journal"><em>Am J Respir Crit Care Med</em>. </span>2010;<span class="ref-vol">181</span>:889–898.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2862303/">PMC free article</a>]</span> [<a href="/pubmed/20110557" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Respir+Crit+Care+Med&amp;title=Decreased+fibronectin+production+significantly+contributes+to+dysregulated+repair+of+asthmatic+epithelium&amp;author=A+Kicic&amp;author=TS+Hallstrand&amp;author=EN+Sutanto&amp;author=PT+Stevens&amp;author=MS+Kobor&amp;volume=181&amp;publication_year=2010&amp;pages=889-898&amp;pmid=20110557&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21.  <span class="element-citation">Asmann YW, Necela BM, Kalari KR, Hossain A, Baker TR, Carr JM, Davis C, Getz JE, Hostetter G, Li X, et al.  Detection of redundant fusion transcripts as biomarkers or disease-specific therapeutic targets in breast cancer. <span><span class="ref-journal"><em>Cancer Res</em>. </span>2012;<span class="ref-vol">72</span>:1921–1928.</span> [<a href="/pubmed/22496456" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Detection+of+redundant+fusion+transcripts+as+biomarkers+or+disease-specific+therapeutic+targets+in+breast+cancer&amp;author=YW+Asmann&amp;author=BM+Necela&amp;author=KR+Kalari&amp;author=A+Hossain&amp;author=TR+Baker&amp;volume=72&amp;publication_year=2012&amp;pages=1921-1928&amp;pmid=22496456&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22.  <span class="element-citation">Storey JD, Tibshirani R. Statistical significance for genomewide studies. <span><span class="ref-journal"><em>Proc Natl Acad Sci USA</em>. </span>2003;<span class="ref-vol">100</span>:9440–9445.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC170937/">PMC free article</a>]</span> [<a href="/pubmed/12883005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Statistical+significance+for+genomewide+studies&amp;author=JD+Storey&amp;author=R+Tibshirani&amp;volume=100&amp;publication_year=2003&amp;pages=9440-9445&amp;pmid=12883005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23.  <span class="element-citation">Ernst J, Kellis M. ChromHMM: automating chromatin-state discovery and characterization. <span><span class="ref-journal"><em>Nat Methods</em>. </span>2012;<span class="ref-vol">9</span>:215–216.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3577932/">PMC free article</a>]</span> [<a href="/pubmed/22373907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=ChromHMM:+automating+chromatin-state+discovery+and+characterization&amp;author=J+Ernst&amp;author=M+Kellis&amp;volume=9&amp;publication_year=2012&amp;pages=215-216&amp;pmid=22373907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24.  <span class="element-citation">Bernstein BE, Birney E, Dunham I, Green ED, Gunter C, Snyder M ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. <span><span class="ref-journal"><em>Nature</em>. </span>2012;<span class="ref-vol">489</span>:57–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3439153/">PMC free article</a>]</span> [<a href="/pubmed/22955616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+encyclopedia+of+DNA+elements+in+the+human+genome&amp;author=BE+Bernstein&amp;author=E+Birney&amp;author=I+Dunham&amp;author=ED+Green&amp;author=C+Gunter&amp;volume=489&amp;publication_year=2012&amp;pages=57-74&amp;pmid=22955616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25.  <span class="element-citation">GTEx Consortium. The Genotype-Tissue Expression (GTEx) project. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2013;<span class="ref-vol">45</span>:580–585.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4010069/">PMC free article</a>]</span> [<a href="/pubmed/23715323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+Genotype-Tissue+Expression+(GTEx)+project&amp;volume=45&amp;publication_year=2013&amp;pages=580-585&amp;pmid=23715323&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26.  <span class="mixed-citation">Hindorff L, MacArthur J, Morales J, Junkins H, Hall P, Klemm A, Manolio T.A catalog of published genome-wide association studies [accessed 2014 Mar 12]. Available from: <a href="http://www.genome.gov/gwastudies" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">www.genome.gov/gwastudies</a></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27.  <span class="element-citation">Wu R, Tong C, Wang Z, Mauger D, Tantisira K, Szefler SJ, Chinchilli VM, Israel E. A conceptual framework for pharmacodynamic genome-wide association studies in pharmacogenomics. <span><span class="ref-journal"><em>Drug Discov Today</em>. </span>2011;<span class="ref-vol">16</span>:884–890.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4029408/">PMC free article</a>]</span> [<a href="/pubmed/21920452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Discov+Today&amp;title=A+conceptual+framework+for+pharmacodynamic+genome-wide+association+studies+in+pharmacogenomics&amp;author=R+Wu&amp;author=C+Tong&amp;author=Z+Wang&amp;author=D+Mauger&amp;author=K+Tantisira&amp;volume=16&amp;publication_year=2011&amp;pages=884-890&amp;pmid=21920452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28.  <span class="element-citation">Sikorska K, Rivadeneira F, Groenen PJ, Hofman A, Uitterlinden AG, Eilers PH, Lesaffre E. Fast linear mixed model computations for genome-wide association studies with longitudinal data. <span><span class="ref-journal"><em>Stat Med</em>. </span>2013;<span class="ref-vol">32</span>:165–180.</span> [<a href="/pubmed/22911890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=Fast+linear+mixed+model+computations+for+genome-wide+association+studies+with+longitudinal+data&amp;author=K+Sikorska&amp;author=F+Rivadeneira&amp;author=PJ+Groenen&amp;author=A+Hofman&amp;author=AG+Uitterlinden&amp;volume=32&amp;publication_year=2013&amp;pages=165-180&amp;pmid=22911890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29.  <span class="element-citation">Rose JE, Behm FM, Drgon T, Johnson C, Uhl GR. Personalized smoking cessation: interactions between nicotine dose, dependence and quit-success genotype score. <span><span class="ref-journal"><em>Mol Med</em>. </span>2010;<span class="ref-vol">16</span>:247–253.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896464/">PMC free article</a>]</span> [<a href="/pubmed/20379614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Med&amp;title=Personalized+smoking+cessation:+interactions+between+nicotine+dose,+dependence+and+quit-success+genotype+score&amp;author=JE+Rose&amp;author=FM+Behm&amp;author=T+Drgon&amp;author=C+Johnson&amp;author=GR+Uhl&amp;volume=16&amp;publication_year=2010&amp;pages=247-253&amp;pmid=20379614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30.  <span class="element-citation">Okada Y, Kamatani Y, Takahashi A, Matsuda K, Hosono N, Ohmiya H, Daigo Y, Yamamoto K, Kubo M, Nakamura Y, et al.  A genome-wide association study in 19 633 Japanese subjects identified LHX3-QSOX2 and IGF1 as adult height loci. <span><span class="ref-journal"><em>Hum Mol Genet</em>. </span>2010;<span class="ref-vol">19</span>:2303–2312.</span> [<a href="/pubmed/20189936" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=A+genome-wide+association+study+in+19+633+Japanese+subjects+identified+LHX3-QSOX2+and+IGF1+as+adult+height+loci&amp;author=Y+Okada&amp;author=Y+Kamatani&amp;author=A+Takahashi&amp;author=K+Matsuda&amp;author=N+Hosono&amp;volume=19&amp;publication_year=2010&amp;pages=2303-2312&amp;pmid=20189936&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31.  <span class="element-citation">Gudbjartsson DF, Walters GB, Thorleifsson G, Stefansson H, Halldorsson BV, Zusmanovich P, Sulem P, Thorlacius S, Gylfason A, Steinberg S, et al.  Many sequence variants affecting diversity of adult human height. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2008;<span class="ref-vol">40</span>:609–615.</span> [<a href="/pubmed/18391951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Many+sequence+variants+affecting+diversity+of+adult+human+height&amp;author=DF+Gudbjartsson&amp;author=GB+Walters&amp;author=G+Thorleifsson&amp;author=H+Stefansson&amp;author=BV+Halldorsson&amp;volume=40&amp;publication_year=2008&amp;pages=609-615&amp;pmid=18391951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32.  <span class="element-citation">Soranzo N, Rivadeneira F, Chinappen-Horsley U, Malkina I, Richards JB, Hammond N, Stolk L, Nica A, Inouye M, Hofman A, et al.  Meta-analysis of genome-wide scans for human adult stature identifies novel loci and associations with measures of skeletal frame size. <span><span class="ref-journal"><em>PLoS Genet</em>. </span>2009;<span class="ref-vol">5</span>:e1000445.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2661236/">PMC free article</a>]</span> [<a href="/pubmed/19343178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Meta-analysis+of+genome-wide+scans+for+human+adult+stature+identifies+novel+loci+and+associations+with+measures+of+skeletal+frame+size&amp;author=N+Soranzo&amp;author=F+Rivadeneira&amp;author=U+Chinappen-Horsley&amp;author=I+Malkina&amp;author=JB+Richards&amp;volume=5&amp;publication_year=2009&amp;pages=e1000445&amp;pmid=19343178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33.  <span class="element-citation">Zhao J, Li M, Bradfield JP, Zhang H, Mentch FD, Wang K, Sleiman PM, Kim CE, Glessner JT, Hou C, et al.  The role of height-associated loci identified in genome wide association studies in the determination of pediatric stature. <span><span class="ref-journal"><em>BMC Med Genet</em>. </span>2010;<span class="ref-vol">11</span>:96.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2894790/">PMC free article</a>]</span> [<a href="/pubmed/20546612" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=The+role+of+height-associated+loci+identified+in+genome+wide+association+studies+in+the+determination+of+pediatric+stature&amp;author=J+Zhao&amp;author=M+Li&amp;author=JP+Bradfield&amp;author=H+Zhang&amp;author=FD+Mentch&amp;volume=11&amp;publication_year=2010&amp;pages=96&amp;pmid=20546612&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34.  <span class="element-citation">Lee JJ, Essers JB, Kugathasan S, Escher JC, Lettre G, Butler JL, Stephens MC, Ramoni MF, Grand RJ, Hirschhorn J. Association of linear growth impairment in pediatric Crohn’s disease and a known height locus: a pilot study. <span><span class="ref-journal"><em>Ann Hum Genet</em>. </span>2010;<span class="ref-vol">74</span>:489–497.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955808/">PMC free article</a>]</span> [<a href="/pubmed/20846217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Association+of+linear+growth+impairment+in+pediatric+Crohn’s+disease+and+a+known+height+locus:+a+pilot+study&amp;author=JJ+Lee&amp;author=JB+Essers&amp;author=S+Kugathasan&amp;author=JC+Escher&amp;author=G+Lettre&amp;volume=74&amp;publication_year=2010&amp;pages=489-497&amp;pmid=20846217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35.  <span class="element-citation">Soler Artigas M, Loth DW, Wain LV, Gharib SA, Obeidat M, Tang W, Zhai G, Zhao JH, Smith AV, Huffman JE, et al.  Genome-wide association and large-scale follow up identifies 16 new loci influencing lung function. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2011;<span class="ref-vol">43</span>:1082–1090.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3267376/">PMC free article</a>]</span> [<a href="/pubmed/21946350" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+and+large-scale+follow+up+identifies+16+new+loci+influencing+lung+function&amp;author=M+Soler+Artigas&amp;author=DW+Loth&amp;author=LV+Wain&amp;author=SA+Gharib&amp;author=M+Obeidat&amp;volume=43&amp;publication_year=2011&amp;pages=1082-1090&amp;pmid=21946350&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36.  <span class="element-citation">Smemo S, Tena JJ, Kim KH, Gamazon ER, Sakabe NJ, Gómez-Marín C, Aneas I, Credidio FL, Sobreira DR, Wasserman NF, et al.  Obesity-associated variants within FTO form long-range functional connections with IRX3. <span><span class="ref-journal"><em>Nature</em>. </span>2014;<span class="ref-vol">507</span>:371–375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4113484/">PMC free article</a>]</span> [<a href="/pubmed/24646999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Obesity-associated+variants+within+FTO+form+long-range+functional+connections+with+IRX3&amp;author=S+Smemo&amp;author=JJ+Tena&amp;author=KH+Kim&amp;author=ER+Gamazon&amp;author=NJ+Sakabe&amp;volume=507&amp;publication_year=2014&amp;pages=371-375&amp;pmid=24646999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37.  <span class="element-citation">Sur IK, Hallikas O, Vähärautio A, Yan J, Turunen M, Enge M, Taipale M, Karhu A, Aaltonen LA, Taipale J. Mice lacking a Myc enhancer that includes human SNP rs6983267 are resistant to intestinal tumors. <span><span class="ref-journal"><em>Science</em>. </span>2012;<span class="ref-vol">338</span>:1360–1363.</span> [<a href="/pubmed/23118011" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mice+lacking+a+Myc+enhancer+that+includes+human+SNP+rs6983267+are+resistant+to+intestinal+tumors&amp;author=IK+Sur&amp;author=O+Hallikas&amp;author=A+Vähärautio&amp;author=J+Yan&amp;author=M+Turunen&amp;volume=338&amp;publication_year=2012&amp;pages=1360-1363&amp;pmid=23118011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4214107&amp;issue-id=244670&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140074030770640title">Abstract</a></li><li><a href="#__sec4title">Methods</a></li><li><a href="#__sec10title">Results</a></li><li><a href="#__sec17title">Discussion</a></li><li><a href="#idm140074069534432title">Acknowledgment</a></li><li><a href="#idm140074069532816title">Footnotes</a></li><li><a href="#idm140074034073040title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Respiratory and Critical Care Medicine</span> are provided here courtesy of <strong>American Thoracic Society</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4214107/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4214107/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4214107/pdf/rccm.201403-0460OC.pdf">PDF (894K)</a></li> | <li><a href="#" data-citationid="PMC4214107" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4214107%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4214107%2F&amp;text=Genome-wide%20Interrogation%20of%20Longitudinal%20FEV1%20in%20Children%20with%20Asthma" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4214107%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25221879" data-aiid="4214107" data-aid="4214107" data-iid="244670" data-domainid="465" data-domain="ajrccm" data-accid="PMC4214107" data-md5="45338317f3adb93753fcbe8c094962eb"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:41:29-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8AE755DD576D910000000002AB01A6&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4214107%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajrccm&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4214107/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size1, sans-serif;"></div></div></body></html>