<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajrccm"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4872654/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>High-Density Genetic Mapping Identifies New Susceptibility Variants in Sarcoidosis Phenotypes and Shows Genomic-driven Phenotypic Differences</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4872654/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Respiratory and Critical Care Medicine"><meta name="citation_title" content="High-Density Genetic Mapping Identifies New Susceptibility Variants in Sarcoidosis Phenotypes and Shows Genomic-driven Phenotypic Differences"><meta name="citation_authors" content="Natalia V. Rivera, Marcus Ronninger, Klementy Shchetynsky, Andre Franke, Markus M. Nöthen, Joachim Müller-Quernheim, Stefan Schreiber, Indra Adrianto, Bekir Karakaya, Coline H. M. van Moorsel, Zdenka Navratilova, Vitezslav Kolek, Benjamin A. Rybicki, Michael C. Iannuzzi, Martin Petrek, Jan C. Grutters, Courtney Montgomery, Annegret Fischer, Anders Eklund, Leonid Padyukov, Johan Grunewald"><meta name="citation_date" content="1 May 2016"><meta name="citation_issue" content="9"><meta name="citation_volume" content="193"><meta name="citation_firstpage" content="1008"><meta name="citation_doi" content="10.1164/rccm.201507-1372OC"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4872654/?report=abstract"><meta name="citation_pmid" content="26651848"><meta name="DC.Title" content="High-Density Genetic Mapping Identifies New Susceptibility Variants in Sarcoidosis Phenotypes and Shows Genomic-driven Phenotypic Differences"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Thoracic Society"><meta name="DC.Contributor" content="Natalia V. Rivera"><meta name="DC.Contributor" content="Marcus Ronninger"><meta name="DC.Contributor" content="Klementy Shchetynsky"><meta name="DC.Contributor" content="Andre Franke"><meta name="DC.Contributor" content="Markus M. Nöthen"><meta name="DC.Contributor" content="Joachim Müller-Quernheim"><meta name="DC.Contributor" content="Stefan Schreiber"><meta name="DC.Contributor" content="Indra Adrianto"><meta name="DC.Contributor" content="Bekir Karakaya"><meta name="DC.Contributor" content="Coline H. M. van Moorsel"><meta name="DC.Contributor" content="Zdenka Navratilova"><meta name="DC.Contributor" content="Vitezslav Kolek"><meta name="DC.Contributor" content="Benjamin A. Rybicki"><meta name="DC.Contributor" content="Michael C. Iannuzzi"><meta name="DC.Contributor" content="Martin Petrek"><meta name="DC.Contributor" content="Jan C. Grutters"><meta name="DC.Contributor" content="Courtney Montgomery"><meta name="DC.Contributor" content="Annegret Fischer"><meta name="DC.Contributor" content="Anders Eklund"><meta name="DC.Contributor" content="Leonid Padyukov"><meta name="DC.Contributor" content="Johan Grunewald"><meta name="DC.Date" content="2016 May 1"><meta name="DC.Identifier" content="10.1164/rccm.201507-1372OC"><meta name="DC.Language" content="en"><meta property="og:title" content="High-Density Genetic Mapping Identifies New Susceptibility Variants in Sarcoidosis Phenotypes and Shows Genomic-driven Phenotypic Differences"><meta property="og:type" content="article"><meta property="og:description" content="Rationale: Sarcoidosis is a multisystem disease of unknown cause. Löfgren’s syndrome (LS) is a characteristic subgroup of sarcoidosis that is associated with a good prognosis in sarcoidosis. However, little is known about its genetic architecture ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4872654/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajrccm.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4872654/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4872654/pdf/rccm.201507-1372OC.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8AD78ADD58F53100000000018A00E2.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4872654%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Am J Respir Crit Care Med%22[journal]" class="navlink">Am J Respir Crit Care Med</a></li><li class="accid">PMC4872654</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajrccm.gif" alt="Logo of ajrccm" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="338,19,460,34" alt="Issue Featuring Article" title="Issue Featuring Article" href="http://ajrccm.atsjournals.org/content/vol193/issue9" ref="reftype=publisher&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="338,3,494,18" alt="Publisher's Version of Article" title="Publisher's Version of Article" href="//dx.doi.org/10.1164/rccm.201507-1372OC" ref="reftype=publisher&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"><area shape="rect" coords="338,36,411,49" alt="Submissions" title="Submissions" href="http://mc.manuscriptcentral.com/atsjournals" ref="reftype=publisher&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,3,318,29" alt="American Thoracic Society" title="American Thoracic Society" href="http://www.thoracic.org/" ref="reftype=publisher&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="1,1,52,55" alt="American Thoracic Society" title="American Thoracic Society" href="http://www.thoracic.org/" ref="reftype=publisher&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,56,499,75" alt="American Journal of Respiratory and Critical Care Medicine" title="American Journal of Respiratory and Critical Care Medicine" href="http://ajrccm.atsjournals.org/" ref="reftype=publisher&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Respir Crit Care Med</a></span>. 2016 May 1; 193(9): 1008–1022. </span></div><div><span class="fm-vol-iss-date">Published online 2016 May 1. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1164%2Frccm.201507-1372OC" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1164/rccm.201507-1372OC</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4872654</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26651848">26651848</a></div></div></div></div><h1 class="content-title">High-Density Genetic Mapping Identifies New Susceptibility Variants in Sarcoidosis Phenotypes and Shows Genomic-driven Phenotypic Differences</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Rivera%20NV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913874794384" co-class="co-affbox">Natalia V. Rivera</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Ronninger%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913860611072" co-class="co-affbox">Marcus Ronninger</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Shchetynsky%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913869210688" co-class="co-affbox">Klementy Shchetynsky</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913864118960" co-class="co-affbox">Andre Franke</a>,<sup>4</sup> <a href="/pubmed/?term=N%26%23x000f6%3Bthen%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913864116848" co-class="co-affbox">Markus M. Nöthen</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=M%26%23x000fc%3Bller-Quernheim%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913865401728" co-class="co-affbox">Joachim Müller-Quernheim</a>,<sup>7</sup> <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913862093424" co-class="co-affbox">Stefan Schreiber</a>,<sup>4,</sup><sup>8,</sup><sup>9</sup> <a href="/pubmed/?term=Adrianto%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913874764848" co-class="co-affbox">Indra Adrianto</a>,<sup>10</sup> <a href="/pubmed/?term=Karakaya%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913873500240" co-class="co-affbox">Bekir Karakaya</a>,<sup>11</sup> <a href="/pubmed/?term=van%20Moorsel%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913907542704" co-class="co-affbox">Coline H. M. van Moorsel</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Navratilova%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913869918880" co-class="co-affbox">Zdenka Navratilova</a>,<sup>13</sup> <a href="/pubmed/?term=Kolek%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913869916768" co-class="co-affbox">Vitezslav Kolek</a>,<sup>14</sup> <a href="/pubmed/?term=Rybicki%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913870731632" co-class="co-affbox">Benjamin A. Rybicki</a>,<sup>15</sup> <a href="/pubmed/?term=Iannuzzi%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913863100352" co-class="co-affbox">Michael C. Iannuzzi</a>,<sup>16</sup> <a href="/pubmed/?term=Petrek%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913874394080" co-class="co-affbox">Martin Petrek</a>,<sup>13</sup> <a href="/pubmed/?term=Grutters%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913874391968" co-class="co-affbox">Jan C. Grutters</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Montgomery%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913875578512" co-class="co-affbox">Courtney Montgomery</a>,<sup>10</sup> <a href="/pubmed/?term=Fischer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913862045520" co-class="co-affbox">Annegret Fischer</a>,<sup>4</sup> <a href="/pubmed/?term=Eklund%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913876941920" co-class="co-affbox">Anders Eklund</a>,<sup>1</sup> <a href="/pubmed/?term=Padyukov%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913876939808" co-class="co-affbox">Leonid Padyukov</a>,<sup>2,</sup><sup>3</sup> and  <a href="/pubmed/?term=Grunewald%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848" class="affpopup" co-rid="_co_idm139913861995616" co-class="co-affbox">Johan Grunewald</a><sup>2,</sup><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139913874794384"><h3 class="no_margin">Natalia V. Rivera</h3><p><sup>1</sup>Respiratory Unit, Department of Medicine, Karolinska Institutet, Karolinska University Hospital, Stockholm, Sweden</p><p><sup>2</sup>Center for Molecular Medicine, and</p><div>Find articles by <a href="/pubmed/?term=Rivera%20NV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Natalia V. Rivera</a></div></div><div id="_co_idm139913860611072"><h3 class="no_margin">Marcus Ronninger</h3><p><sup>1</sup>Respiratory Unit, Department of Medicine, Karolinska Institutet, Karolinska University Hospital, Stockholm, Sweden</p><p><sup>2</sup>Center for Molecular Medicine, and</p><div>Find articles by <a href="/pubmed/?term=Ronninger%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Marcus Ronninger</a></div></div><div id="_co_idm139913869210688"><h3 class="no_margin">Klementy Shchetynsky</h3><p><sup>2</sup>Center for Molecular Medicine, and</p><p><sup>3</sup>Rheumatology Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Shchetynsky%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Klementy Shchetynsky</a></div></div><div id="_co_idm139913864118960"><h3 class="no_margin">Andre Franke</h3><p><sup>4</sup>Institute of Clinical Molecular Biology, University Hospital Schleswig-Holstein, Campus Kiel and Kiel University, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Andre Franke</a></div></div><div id="_co_idm139913864116848"><h3 class="no_margin">Markus M. Nöthen</h3><p><sup>5</sup>Institute of Human Genetics, and</p><p><sup>6</sup>Department of Genomics, Life &amp; Brain Center, University of Bonn, Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=N%26%23x000f6%3Bthen%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Markus M. Nöthen</a></div></div><div id="_co_idm139913865401728"><h3 class="no_margin">Joachim Müller-Quernheim</h3><p><sup>7</sup>Department of Pneumology, University of Freiburg, Freiburg, Germany</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000fc%3Bller-Quernheim%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Joachim Müller-Quernheim</a></div></div><div id="_co_idm139913862093424"><h3 class="no_margin">Stefan Schreiber</h3><p><sup>4</sup>Institute of Clinical Molecular Biology, University Hospital Schleswig-Holstein, Campus Kiel and Kiel University, Kiel, Germany</p><p><sup>8</sup>Popgen Biobank and</p><p><sup>9</sup>Clinic of Internal Medicine I, University Hospital Schleswig-Holstein, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Stefan Schreiber</a></div></div><div id="_co_idm139913874764848"><h3 class="no_margin">Indra Adrianto</h3><p><sup>10</sup>Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, Oklahoma</p><div>Find articles by <a href="/pubmed/?term=Adrianto%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Indra Adrianto</a></div></div><div id="_co_idm139913873500240"><h3 class="no_margin">Bekir Karakaya</h3><p><sup>11</sup>Center of Interstitial Lung Diseases, St. Antonius Hospital, Nieuwegein, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Karakaya%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Bekir Karakaya</a></div></div><div id="_co_idm139913907542704"><h3 class="no_margin">Coline H. M. van Moorsel</h3><p><sup>11</sup>Center of Interstitial Lung Diseases, St. Antonius Hospital, Nieuwegein, the Netherlands</p><p><sup>12</sup>Division of Heart &amp; Lungs, University Medical Center Utrecht, Utrecht, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Moorsel%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Coline H. M. van Moorsel</a></div></div><div id="_co_idm139913869918880"><h3 class="no_margin">Zdenka Navratilova</h3><p><sup>13</sup>Department of Pathological Physiology and</p><div>Find articles by <a href="/pubmed/?term=Navratilova%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Zdenka Navratilova</a></div></div><div id="_co_idm139913869916768"><h3 class="no_margin">Vitezslav Kolek</h3><p><sup>14</sup>Department of Respiratory Medicine, Faculty of Medicine and Dentistry, Palacky University Olomouc, Olomouc, Czech Republic</p><div>Find articles by <a href="/pubmed/?term=Kolek%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Vitezslav Kolek</a></div></div><div id="_co_idm139913870731632"><h3 class="no_margin">Benjamin A. Rybicki</h3><p><sup>15</sup>Department of Public Health Sciences, Henry Ford Health Systems, Detroit, Michigan; and</p><div>Find articles by <a href="/pubmed/?term=Rybicki%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Benjamin A. Rybicki</a></div></div><div id="_co_idm139913863100352"><h3 class="no_margin">Michael C. Iannuzzi</h3><p><sup>16</sup>State University of New York Upstate Medical University Syracuse, Syracuse, New York</p><div>Find articles by <a href="/pubmed/?term=Iannuzzi%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Michael C. Iannuzzi</a></div></div><div id="_co_idm139913874394080"><h3 class="no_margin">Martin Petrek</h3><p><sup>13</sup>Department of Pathological Physiology and</p><div>Find articles by <a href="/pubmed/?term=Petrek%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Martin Petrek</a></div></div><div id="_co_idm139913874391968"><h3 class="no_margin">Jan C. Grutters</h3><p><sup>11</sup>Center of Interstitial Lung Diseases, St. Antonius Hospital, Nieuwegein, the Netherlands</p><p><sup>12</sup>Division of Heart &amp; Lungs, University Medical Center Utrecht, Utrecht, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Grutters%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Jan C. Grutters</a></div></div><div id="_co_idm139913875578512"><h3 class="no_margin">Courtney Montgomery</h3><p><sup>10</sup>Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, Oklahoma</p><div>Find articles by <a href="/pubmed/?term=Montgomery%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Courtney Montgomery</a></div></div><div id="_co_idm139913862045520"><h3 class="no_margin">Annegret Fischer</h3><p><sup>4</sup>Institute of Clinical Molecular Biology, University Hospital Schleswig-Holstein, Campus Kiel and Kiel University, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Fischer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Annegret Fischer</a></div></div><div id="_co_idm139913876941920"><h3 class="no_margin">Anders Eklund</h3><p><sup>1</sup>Respiratory Unit, Department of Medicine, Karolinska Institutet, Karolinska University Hospital, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Eklund%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Anders Eklund</a></div></div><div id="_co_idm139913876939808"><h3 class="no_margin">Leonid Padyukov</h3><p><sup>2</sup>Center for Molecular Medicine, and</p><p><sup>3</sup>Rheumatology Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Padyukov%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Leonid Padyukov</a></div></div><div id="_co_idm139913861995616"><h3 class="no_margin">Johan Grunewald</h3><p><sup>2</sup>Center for Molecular Medicine, and</p><p><sup>3</sup>Rheumatology Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Grunewald%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26651848">Johan Grunewald</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139913873045376_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139913873045376_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139913873045376_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139913873045376_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Respiratory Unit, Department of Medicine, Karolinska Institutet, Karolinska University Hospital, Stockholm, Sweden</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Center for Molecular Medicine, and</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Rheumatology Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Institute of Clinical Molecular Biology, University Hospital Schleswig-Holstein, Campus Kiel and Kiel University, Kiel, Germany</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Institute of Human Genetics, and</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Department of Genomics, Life &amp; Brain Center, University of Bonn, Bonn, Germany</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Department of Pneumology, University of Freiburg, Freiburg, Germany</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Popgen Biobank and</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Clinic of Internal Medicine I, University Hospital Schleswig-Holstein, Kiel, Germany</div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, Oklahoma</div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>Center of Interstitial Lung Diseases, St. Antonius Hospital, Nieuwegein, the Netherlands</div><div class="fm-affl" id="aff12" lang="en"><sup>12</sup>Division of Heart &amp; Lungs, University Medical Center Utrecht, Utrecht, the Netherlands</div><div class="fm-affl" id="aff13" lang="en"><sup>13</sup>Department of Pathological Physiology and</div><div class="fm-affl" id="aff14" lang="en"><sup>14</sup>Department of Respiratory Medicine, Faculty of Medicine and Dentistry, Palacky University Olomouc, Olomouc, Czech Republic</div><div class="fm-affl" id="aff15" lang="en"><sup>15</sup>Department of Public Health Sciences, Henry Ford Health Systems, Detroit, Michigan; and</div><div class="fm-affl" id="aff16" lang="en"><sup>16</sup>State University of New York Upstate Medical University Syracuse, Syracuse, New York</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="idm139913866083760">Correspondence and requests for reprints should be addressed to Natalia V. Rivera, Ph.D., M.Sc., Respiratory Unit, Department of Medicine, Karolinska Institutet, Karolinska University Hospital Solna L4:01, SE-171 76 Stockholm, Sweden. E-mail: <a href="mailto:dev@null" data-email="es.ik@arevir.ailatan" class="oemail">es.ik@arevir.ailatan</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139913873045376_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2015 Jul 15; Accepted 2015 Dec 9.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139913873045376_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2016 by the American Thoracic Society</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>See "<a href="/pubmed/27128702" class="int-reflink">Two Faces of Sarcoidosis. A Genetic Insight?</a>" in volume 193 on&nbsp;page&nbsp;942.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>This article has been <a href="/pmc/articles/PMC4872654/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm139913866216992" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139913866216992title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first"><strong>Rationale:</strong> Sarcoidosis is a multisystem disease of unknown cause. Löfgren’s syndrome (LS) is a characteristic subgroup of sarcoidosis that is associated with a good prognosis in sarcoidosis. However, little is known about its genetic architecture or its broader phenotype, non-LS sarcoidosis.</p><p id="__p2"><strong>Objectives:</strong> To address the genetic architecture of sarcoidosis phenotypes, LS and non-LS.</p><p id="__p3"><strong>Methods:</strong> An association study in a white Swedish cohort of 384 LS, 664 non-LS, and 2,086 control subjects, totaling 3,134 subjects using a fine-mapping genotyping platform was conducted. Replication was performed in four independent cohorts, three of white European descent (Germany, n = 4,975; the Netherlands, n = 613; and Czech Republic, n = 521), and one of black African descent (United States, n = 1,657), totaling 7,766 subjects.</p><p id="__p4"><strong>Measurements and Main Results:</strong> A total of 727 LS-associated variants expanding throughout the extended major histocompatibility complex (MHC) region and 68 non-LS–associated variants located in the MHC class II region were identified and confirmed. A shared overlap between LS and non-LS defined by 17 variants located in the MHC class II region was found. Outside the MHC region, two LS-associated loci, in <em>ADCY3</em> and between <em>CSMD1</em> and <em>MCPH1</em>, were observed and replicated.</p><p id="__p5" class="p p-last"><strong>Conclusions:</strong> Comprehensive and integrative analyses of genetics, transcription, and pathway modeling on LS and non-LS indicates that these sarcoidosis phenotypes have different genetic susceptibility, genomic distributions, and cellular activities, suggesting distinct molecular mechanisms in pathways related to immune response with a common region.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genetic epidemiology of sarcoidosis, Löfgren’s syndrome, non-Löfgren’s syndrome, genome-wide associations, Immunochip</span></div></div><div id="idm139913874367440" class="tsec sec headless whole_rhythm"><div class="boxed-text-box whole_rhythm hide-overflow"><div class="sec"><h3 id="__sec1title">At a Glance Commentary</h3><div class="sec"><h4 id="__sec2title">Scientific Knowledge on the Subject</h4><p id="__p6" class="p p-first-last">The genetics in subgroups of patients with sarcoidosis have not been previously investigated.</p></div><div class="sec"><h4 id="__sec3title">What This Study Adds to the Field</h4><p id="__p7" class="p p-first-last">This study delineates the genetic architecture of the specific subgroup with Löfgren’s syndrome.</p></div></div></div><p id="__p8" class="p p-first">Sarcoidosis is a systematic inflammatory disease of unknown cause that is characterized by noncaseating granulomas. The main affected organ is the lung, which is engaged in more than 90% of patients and about a quarter develop pulmonary fibrosis with permanent dysfunction (<a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102643">1</a>). Löfgren’s syndrome (LS) is an acute form of sarcoidosis, characterized by bilateral lymphadenopathy on chest radiography (in some cases with parenchymal infiltrates), erythema nodosum, and/or bilateral ankle arthritis or distinct periarticular inflammation (<a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102657">2</a>). Patients with LS have a better prognosis compared with patients without LS, as evaluated 2 years after disease onset. In particular, patients with LS carrying the <em>HLA-DRB1*03</em> allele run a good disease course (<a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102663">3</a>).</p><p id="__p9">Sarcoidosis has a peak incidence in the third and fourth decade of life, with a higher disease rate in females, and has major effects on quality of life, the ability to work, and basic activities of daily living (<a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102633">4</a>, <a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102650">5</a>). In Sweden, the incidence rate of sarcoidosis has been estimated at approximately 64 per 100,000 persons (<a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102636">6</a>) and is highly represented by LS (<a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102646">7</a>) with an estimate of 15 per 100,000 persons yearly as observed in our respiratory unit.</p><p id="__p10" class="p">In the present study, we aimed to identify genetic variants linked specifically with two sarcoidosis phenotypes, LS and non-LS, in a sample of white Swedish persons. Replication analysis of identified genetic variants was conducted in independent cohorts with similar phenotypic characterization of European ancestry from Germany, the Netherlands, and Czech Republic and of African American ancestry from the United States. A genome-wide association approach was adopted using a high-density genetic mapping technique that has been used across a range of autoimmune and inflammatory diseases (<a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102660">8</a>–<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102652">11</a>) but so far not for sarcoidosis phenotypes.</p></div><div id="__sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec4title">Methods</h2><div id="__sec5" class="sec sec-first"><h3 id="__sec5title">Study Design and Participants</h3><p id="__p11" class="p p-first-last">All studies had protocols approved by local institutional review boards. All participants in Sweden, Germany, the Netherlands, Czech Republic, and United States provided written informed consent and gave permission to use their DNA for research purposes. Baseline measures of clinical and demographic characteristics were measured at the time of the participants’ enrollment in each study. Details of study cohorts, genotyping, and quality control of genetic data are available in the M<span class="small-caps">ethods</span> section of the online supplement.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Statistical Analysis</h3><div id="__sec7" class="sec sec-first"><p></p><h4 id="__sec7title" class="inline">Association study of LS and non-LS </h4><p id="__p12" class="p p-first-last">For both LS and non-LS, logistic regression analysis of genotyped and imputed single-nucleotide polymorphisms (SNPs) was performed with expected genotype count using the posterior probabilities. Analyses were adjusted for sex and were performed using PLINK v1.07 (<a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102635">12</a>) on Immunochip data. Genome-wide significance was defined as genomic control-corrected <em>P</em><sub><em>GC</em></sub> less than 5 × 10<sup>−8</sup> and a suggestive significance threshold of <em>P</em><em><sub>GC</sub></em> greater than 5 × 10<sup>−8</sup> and less than 5 × 10<sup>−5</sup> was established for identifying susceptible variants. The suggestive <em>P</em> value was chosen based on published works (<a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102651">9</a>, <a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102637">11</a>, <a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102653">13</a>) using Immunochip data and proposed threshold significance <em>P</em><sub><em>Ichip</em></sub> less than 10<sup>−4</sup>. Moreover, conditional analysis on the lead SNP was performed as to evaluate for other potential variants. Additionally, conditional analysis on <em>HLA-DRB1</em> alleles was also performed. Each <em>HLA-DRB1</em> allele was analyzed as a covariate along with sex and was coded as positive (assigned a “1”) or negative (assigned a “0”) for the allele in question.</p></div><div id="__sec8" class="sec"><p></p><h4 id="__sec8title" class="inline">Replication of significant SNPs </h4><p id="__p13" class="p p-first-last">Replication of associated SNPs was conducted in four independent cohorts, three of white European descent and one of black African descent, totaling 7,766 subjects. Specifically, in samples from Germany (64 LS, 413 non-LS, and 4,498 control subjects) and from African Americans (781 non-LS and 876 control subjects), SNPs identified at <em>P</em><sub><em>GC</em></sub> less than 5 × 10<sup>−5</sup> in the discovery cohort were extracted from an available dataset typed on the Immunochip. In subjects from the Netherlands (241 LS, 180 non-LS, and 192 control subjects) and Czech Republic (47 LS, 263 non-LS, and 211 control subjects), the top SNP associated with LS and non-LS, respectively, was genotyped by TaqMan assay. A nominal <em>P</em> value threshold for replication significance was set to <em>P</em> less than 0.05 with Bonferroni correction.</p></div><div id="__sec9" class="sec"><p></p><h4 id="__sec9title" class="inline">Meta-analysis of associated SNPs </h4><p id="__p14" class="p p-first-last">We conducted meta-analysis by combining association results from all cohorts. An inverse variance weighting approach on regression estimates and standard errors was used as implemented in the Metafor (<a href="#bib14" rid="bib14" class=" bibr popnode">14</a>) R-package for high and suggestive signals associated with LS and non-LS, respectively. We also applied genomic control to the discovery cohort (given that the inflation factor, lambda, was &gt;1.00 [i.e., 1.16 in LS, and 1.17 in non-LS]) by multiplying standard errors by the square root of lambda. For both phenotypes, we calculated a meta-analysis log odds ratio (OR), 95% confidence interval (CI), meta-<em>P</em> value, along with model statistic measures, amount of heterogeneity in the true log effect size (τ<sup>2</sup>), heterogeneity <em>P</em> value (τ<sup>2</sup>
<em>P</em>), and total variability in effect size (<em>I</em><sup>2</sup>). Fixed-effect and random-effect models were used for carrying out meta-analysis. We established a significance threshold of meta-<em>P</em> less than 5 × 10<sup>−8</sup> as genome-wide significant.</p></div><div id="__sec10" class="sec sec-last"><p></p><h4 id="__sec10title" class="inline">Gene-centric analysis </h4><p id="__p15" class="p p-first-last">To explore the genetic effects of variants located within a gene including those in introns, coding, 5′ untranslated region (5′ UTR), and 3′ UTR, we performed a genome-wide association gene-based approach for LS and non-LS, respectively, using the versatile gene-based association test as implemented in VEGAS (Versatile Gene-based Association Study) (<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102642">15</a>) software, which requires genic SNPs and individual marker <em>P</em> values from genome-wide association test to compute gene-based <em>P</em> value. From the association summary statistics of LS and non-LS, we extracted 48,740 SNPs tagging 6,840 genes across the genome and used the genomic-controlled <em>P</em> value (<em>P</em><sub><em>GC</em></sub>) as <em>P</em> value entry. To account for LD structure, the HapMap phase 2 CEU population (Utah residents with ancestry from northern and western Europe) was used as reference along with the top SNP option. A Bonferroni-corrected threshold of <em>P</em> less than 7.3 × 10<sup>−6</sup> (0.05/6840) was used for genome-wide significance and <em>P</em> less than 0.05 for nominal significance.</p></div></div><div id="__sec11" class="sec sec-last"><h3 id="__sec11title">Integrative Analyses on Significant Findings</h3><div id="__sec12" class="sec sec-first"><p></p><h4 id="__sec12title" class="inline">Bioinformatic analysis for refinement of associated variants </h4><p id="__p16" class="p p-first-last">To explore the potential functionality of genetic variants associated with LS and non-LS and to substantiate their associations with the phenotypes in question, we evaluated genetic variant involvement in expression quantitative trait loci (eQTL), which are known to be directly linked to DNA variation. Using public/published eQTL databases (<a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102658">16</a>–<a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102661">25</a>), we classified a SNP as <em>cis-</em>eQTL SNP if it was found associated with eQTL at <em>P</em> less than 0.05. For SNPs with no direct <em>cis-</em>eQTL association, we computed the linkage disequilibrium (LD; defined by <em>R</em><sup><em>2</em></sup> &gt; 0.8 and Dʹ = 1) using HapMap and 1,000 Genomes datasets to establish whether the SNP was near a <em>cis-</em>eQTL SNP. Additionally, to investigate whether a SNP had a role in machineries of gene regulation, we used available datasets from ENCODE (the Encyclopedia of DNA Elements) and the National Institutes of Health Roadmap Epigenomics Mapping Consortium. Using RegulomeDB (<a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102656">26</a>) and HaploReg v2 (<a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102645">27</a>) databases, we obtained measured/predicted data for all SNPs in question across various tissues and cell types. We classified a SNP as a regulatory SNP if it had at least one measured/predicted regulatory function present across collected data.</p></div><div id="__sec13" class="sec sec-last"><p></p><h4 id="__sec13title" class="inline">Enrichment and pathway analyses of associated variants </h4><p id="__p17" class="p p-first-last">Enrichment and pathway analyses were conducted based on significant confirmed variants using MetaCore (Thomson Reuters, New York, NY) integrated software. Herein, we followed two different approaches: examination of biologic implication by associated variants; and examination of biologic implication by pooling together associated variants and differentially expressed genes. In the latter, three public available expression datasets from sarcoidosis (GEO accession numbers <a id="__tag_534102665" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE18781" ref="/geo/query/acc.cgi?acc=GSE18781"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE18781","term_id":"18781"}}</span>GSE18781</a> [<a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102649">28</a>], <a id="__tag_534102664" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE19314" ref="/geo/query/acc.cgi?acc=GSE19314"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE19314","term_id":"19314"}}</span>GSE19314</a> [<a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102662">29</a>], and <a id="__tag_534102666" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE32887" ref="/geo/query/acc.cgi?acc=GSE32887"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE32887","term_id":"32887"}}</span>GSE32887</a> [<a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102641">30</a>]) were considered. For each Gene Expression Omnibus (GEO-GSE) dataset, we selected sarcoidosis cases and control subjects at a minimum 1:1 ratio (details are provided in Table E7 in the online supplement) and performed expression analysis using the online GEOR2 tool (<a href="http://www.ncbi.nlm.nih.gov/geo/geo2r/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/geo/geo2r/</a>). Subsequently, results were downloaded and mapped to chromosome/position using the Affymetrix-Human-Genome-U133-Plus-2.0-Array annotation-data (chip_hgu133plus2) Bioconductor R-package (Affymetrix, Inc., Santa Clara, CA). After mapping, a subset of probe-IDs with false discovery rate less than 0.05 identified as differentially expressed was extracted and uploaded onto MetaCore.</p></div></div></div><div id="__sec14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec14title">Results</h2><div id="__sec15" class="sec sec-first"><h3 id="__sec15title">Association Findings</h3><div id="__sec16" class="sec sec-first"><p></p><h4 id="__sec16title" class="inline">LS versus healthy control subjects </h4><p id="__p18" class="p p-first">The association analysis on LS (345 cases vs. 2,025 control subjects) revealed rs3130288 (OR, 4.01; 95% CI, 3.26–4.95; <em>P</em><sub><em>GC</em></sub> = 6.47 × 10<sup>−34</sup>) located in the 3′ UTR of <em>ATF6B</em> (also known as <em>CREBL1</em>), which localizes in major histocompatibility complex (MHC) class III region as the most significant finding. A large number of high signals (1,246 SNPs at <em>P</em><sub><em>GC</em></sub> &lt; 5 × 10<sup>−8</sup>) was identified spanning from the extended MHC class I to class II regions (from <em>LRRC16</em>, chr6:25558005 to <em>HLA-DPB2</em>, chr6:33191175). The top 25 significant SNPs are provided in <a href="/pmc/articles/PMC4872654/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>. The entire list of LS-associated SNPs can be found as Table E1. Outside the MHC region, rs13407913 (OR, 1.48; 95% CI, 1.25–1.74; <em>P</em><sub><em>GC</em></sub> = 2.31 × 10<sup>−5</sup>) located in an intron of <em>ADCY3</em> on chromosome 2 was observed. Manhattan plot of association statistics is shown in <a href="/pmc/articles/PMC4872654/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1A</span></a>. A close-up association regional plot is shown in <a href="/pmc/articles/PMC4872654/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a>. Genomic control showed small dispersion for LS with an inflation factor lambda of 1.16. Q-Q plot for the observed versus expected <em>P</em> values and distribution of significant SNPs (defined by <em>P</em><sub><em>GC</em></sub> &lt; 5 × 10<sup>−8</sup>) by variant location are shown in Figures E1 and E2, respectively. Suggestive signals (692 SNPs at 5 × 10<sup>−8</sup> &lt; <em>P</em><sub><em>GC</em></sub> &lt; 5 × 10<sup>−5</sup>) enriching the extended MHC region were also identified and are provided in Table E1.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p19">Results of Immunochip Analysis on LS versus Healthy Control Subjects Highlighting the Top 25 SNPs at Significance <em>P</em> Less Than 5 × 10<sup>−8</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913871511952" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="center">CHR</th><th rowspan="2" colspan="1" align="center">BP</th><th rowspan="2" colspan="1" align="center">Gene (Nearby Genes)</th><th rowspan="2" colspan="1" align="center">CA/NCA</th><th rowspan="2" colspan="1" align="center">CAF</th><th colspan="3" rowspan="1" align="center">Discovery Cohort (Sweden, European Ancestry) (<em>n</em> = <em>2,470</em>)<hr></th><th colspan="2" rowspan="1" align="center">Replicating Cohort (Germany, European Ancestry) (<em>n</em> = <em>4,498</em>)<hr></th></tr><tr><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value</th><th rowspan="1" colspan="1" align="center"><em>P</em><sub><em>GC</em></sub></th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs3130288</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32203979</td><td rowspan="1" colspan="1" align="center"><em>CREBL1</em></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.345</td><td rowspan="1" colspan="1" align="center">4.02 (3.26–4.95)</td><td rowspan="1" colspan="1" align="center">3.29E-39</td><td rowspan="1" colspan="1" align="center">6.47E-34</td><td rowspan="1" colspan="1" align="center">4.13 (2.83–6.02)</td><td rowspan="1" colspan="1" align="center">1.66E-15</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129927</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32441805</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.343</td><td rowspan="1" colspan="1" align="center">3.99 (3.24–4.92)</td><td rowspan="1" colspan="1" align="center">1.09E-38</td><td rowspan="1" colspan="1" align="center">1.82E-33</td><td rowspan="1" colspan="1" align="center">4.19 (2.87–6.11)</td><td rowspan="1" colspan="1" align="center">7.79E-16</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2187668</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32713862</td><td rowspan="1" colspan="1" align="center"><em>HLA-DQA1</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.359</td><td rowspan="1" colspan="1" align="center">3.97 (3.22–4.89)</td><td rowspan="1" colspan="1" align="center">1.80E-38</td><td rowspan="1" colspan="1" align="center">2.79E-33</td><td rowspan="1" colspan="1" align="center">3.84 (2.65–5.57)</td><td rowspan="1" colspan="1" align="center">2.93E-14</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2854275</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32736406</td><td rowspan="1" colspan="1" align="center"><em>HLA-DQB1</em></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.357</td><td rowspan="1" colspan="1" align="center">3.96 (3.21–4.88)</td><td rowspan="1" colspan="1" align="center">2.64E-38</td><td rowspan="1" colspan="1" align="center">3.87E-33</td><td rowspan="1" colspan="1" align="center">3.74 (2.57–5.44)</td><td rowspan="1" colspan="1" align="center">1.42E-13</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129950</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32466179</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center">0.342</td><td rowspan="1" colspan="1" align="center">3.98 (3.22–4.91)</td><td rowspan="1" colspan="1" align="center">5.22E-38</td><td rowspan="1" colspan="1" align="center">6.96E-33</td><td rowspan="1" colspan="1" align="center">4.33 (2.98–6.31)</td><td rowspan="1" colspan="1" align="center">7.89E-17</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9268219</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32392086</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.334</td><td rowspan="1" colspan="1" align="center">3.97 (3.22–4.90)</td><td rowspan="1" colspan="1" align="center">5.59E-38</td><td rowspan="1" colspan="1" align="center">7.38E-33</td><td rowspan="1" colspan="1" align="center">4.34 (2.98–6.33)</td><td rowspan="1" colspan="1" align="center">1.14E-16</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129716</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32765414</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DQB1, HLA-DQA2)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.356</td><td rowspan="1" colspan="1" align="center">3.92 (3.19–4.83)</td><td rowspan="1" colspan="1" align="center">5.78E-38</td><td rowspan="1" colspan="1" align="center">7.60E-33</td><td rowspan="1" colspan="1" align="center">3.73 (2.57–5.43)</td><td rowspan="1" colspan="1" align="center">1.61E-13</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2856674</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32767623</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DQB1, HLA-DQA2)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.357</td><td rowspan="1" colspan="1" align="center">3.93 (3.19–4.84)</td><td rowspan="1" colspan="1" align="center">6.08E-38</td><td rowspan="1" colspan="1" align="center">7.93E-33</td><td rowspan="1" colspan="1" align="center">3.74 (2.57–5.44)</td><td rowspan="1" colspan="1" align="center">1.49E-13</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2395149</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32433540</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.343</td><td rowspan="1" colspan="1" align="center">3.92 (3.19–4.83)</td><td rowspan="1" colspan="1" align="center">7.82E-38</td><td rowspan="1" colspan="1" align="center">9.85E-33</td><td rowspan="1" colspan="1" align="center">4.19 (2.87–6.11)</td><td rowspan="1" colspan="1" align="center">7.79E-16</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9268235</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32398186</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.343</td><td rowspan="1" colspan="1" align="center">3.91 (3.18–4.82)</td><td rowspan="1" colspan="1" align="center">1.06E-37</td><td rowspan="1" colspan="1" align="center">1.28E-32</td><td rowspan="1" colspan="1" align="center">4.18 (2.86–6.09)</td><td rowspan="1" colspan="1" align="center">8.86E-16</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1265757</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32410360</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.343</td><td rowspan="1" colspan="1" align="center">3.91 (3.18–4.82)</td><td rowspan="1" colspan="1" align="center">1.06E-37</td><td rowspan="1" colspan="1" align="center">1.28E-32</td><td rowspan="1" colspan="1" align="center">4.18 (2.87– 6.10)</td><td rowspan="1" colspan="1" align="center">8.31E-16</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3135394</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32516475</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.344</td><td rowspan="1" colspan="1" align="center">3.84 (3.13–4.72)</td><td rowspan="1" colspan="1" align="center">1.19E-37</td><td rowspan="1" colspan="1" align="center">1.41E-32</td><td rowspan="1" colspan="1" align="center">4.21 (2.89–6.14)</td><td rowspan="1" colspan="1" align="center">6.03E-16</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3130490</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31847099</td><td rowspan="1" colspan="1" align="center"><em>C6orf27</em></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.339</td><td rowspan="1" colspan="1" align="center">3.83 (3.12–4.70)</td><td rowspan="1" colspan="1" align="center">1.84E-37</td><td rowspan="1" colspan="1" align="center">2.05E-32</td><td rowspan="1" colspan="1" align="center">3.79 (2.58–5.56)</td><td rowspan="1" colspan="1" align="center">2.97E-13</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7775397</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32369230</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.343</td><td rowspan="1" colspan="1" align="center">3.87 (3.14–4.76)</td><td rowspan="1" colspan="1" align="center">2.45E-37</td><td rowspan="1" colspan="1" align="center">2.63E-32</td><td rowspan="1" colspan="1" align="center">4.18 (2.87–6.10)</td><td rowspan="1" colspan="1" align="center">8.42E-16</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9268177</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32382860</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.343</td><td rowspan="1" colspan="1" align="center">3.87 (3.14–4.76)</td><td rowspan="1" colspan="1" align="center">2.56E-37</td><td rowspan="1" colspan="1" align="center">2.73E-32</td><td rowspan="1" colspan="1" align="center">4.18 (2.87–6.10)</td><td rowspan="1" colspan="1" align="center">8.31E-16</td></tr><tr><td rowspan="1" colspan="1" align="left">rs389884</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32048876</td><td rowspan="1" colspan="1" align="center"><em>STK19</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.342</td><td rowspan="1" colspan="1" align="center">3.82 (3.11–4.69)</td><td rowspan="1" colspan="1" align="center">3.80E-37</td><td rowspan="1" colspan="1" align="center">3.84E-32</td><td rowspan="1" colspan="1" align="center">4.09 (2.80–5.96)</td><td rowspan="1" colspan="1" align="center">2.76E-15</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1150753</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32167845</td><td rowspan="1" colspan="1" align="center"><em>TNXB</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.341</td><td rowspan="1" colspan="1" align="center">3.83 (3.11–4.71)</td><td rowspan="1" colspan="1" align="center">4.27E-37</td><td rowspan="1" colspan="1" align="center">4.24E-32</td><td rowspan="1" colspan="1" align="center">4.12 (2.82–6.01)</td><td rowspan="1" colspan="1" align="center">1.88E-15</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3132971</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32338234</td><td rowspan="1" colspan="1" align="center"><em>(NOTCH4, C6orf10)</em></td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.341</td><td rowspan="1" colspan="1" align="center">3.86 (3.13–4.75)</td><td rowspan="1" colspan="1" align="center">8.76E-37</td><td rowspan="1" colspan="1" align="center">7.86E-32</td><td rowspan="1" colspan="1" align="center">4.18 (2.86–6.09)</td><td rowspan="1" colspan="1" align="center">8.97E-16</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4143332</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31456344</td><td rowspan="1" colspan="1" align="center">&nbsp;</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.337</td><td rowspan="1" colspan="1" align="center">3.78 (3.07–4.65)</td><td rowspan="1" colspan="1" align="center">1.56E-36</td><td rowspan="1" colspan="1" align="center">1.29E-31</td><td rowspan="1" colspan="1" align="center">3.64 (2.50–5.31)</td><td rowspan="1" colspan="1" align="center">6.88E-13</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3131618</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31542600</td><td rowspan="1" colspan="1" align="center"><em>(HCP5, 3.8-1)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.340</td><td rowspan="1" colspan="1" align="center">3.81 (3.09–4.69)</td><td rowspan="1" colspan="1" align="center">2.40E-36</td><td rowspan="1" colspan="1" align="center">1.87E-31</td><td rowspan="1" colspan="1" align="center">3.72 (2.55–5.42)</td><td rowspan="1" colspan="1" align="center">2.69E-13</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3132449</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31733992</td><td rowspan="1" colspan="1" align="center"><em>(APOM, C6orf47)</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.340</td><td rowspan="1" colspan="1" align="center">3.71 (3.03–4.55)</td><td rowspan="1" colspan="1" align="center">2.46E-36</td><td rowspan="1" colspan="1" align="center">1.91E-31</td><td rowspan="1" colspan="1" align="center">3.94 (2.70–5.75)</td><td rowspan="1" colspan="1" align="center">1.73E-14</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3117577</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31835453</td><td rowspan="1" colspan="1" align="center"><em>MSH5</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.339</td><td rowspan="1" colspan="1" align="center">3.70 (3.02–4.54)</td><td rowspan="1" colspan="1" align="center">2.85E-36</td><td rowspan="1" colspan="1" align="center">2.17E-31</td><td rowspan="1" colspan="1" align="center">3.96 (2.72–5.77)</td><td rowspan="1" colspan="1" align="center">1.36E-14</td></tr><tr><td rowspan="1" colspan="1" align="left">rs519417</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31986412</td><td rowspan="1" colspan="1" align="center"><em>(ZBTB12, C2)</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.340</td><td rowspan="1" colspan="1" align="center">3.71 (3.03–4.56)</td><td rowspan="1" colspan="1" align="center">3.26E-36</td><td rowspan="1" colspan="1" align="center">2.43E-31</td><td rowspan="1" colspan="1" align="center">4.01 (2.75–5.85)</td><td rowspan="1" colspan="1" align="center">6.91E-15</td></tr><tr><td rowspan="1" colspan="1" align="left">rs497309</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32000463</td><td rowspan="1" colspan="1" align="center"><em>(ZBTB12, C2)</em></td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.340</td><td rowspan="1" colspan="1" align="center">3.71 (3.02–4.55)</td><td rowspan="1" colspan="1" align="center">3.79E-36</td><td rowspan="1" colspan="1" align="center">2.77E-31</td><td rowspan="1" colspan="1" align="center">4.03 (2.76–5.88)</td><td rowspan="1" colspan="1" align="center">5.71E-15</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1270942</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32026839</td><td rowspan="1" colspan="1" align="center"><em>CFB</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.339</td><td rowspan="1" colspan="1" align="center">3.72 (3.03–4.56)</td><td rowspan="1" colspan="1" align="center">4.06E-36</td><td rowspan="1" colspan="1" align="center">2.94E-31</td><td rowspan="1" colspan="1" align="center">4.05 (2.78–5.91)</td><td rowspan="1" colspan="1" align="center">4.43E-15</td></tr></tbody></table></div><div id="largeobj_idm139913871511952" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4872654/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139913869175264"><p id="__p20" class="p p-first-last"><em>Definition of abbreviations</em>: BP = base pairs; CA = coded allele; CAF = coded allele frequency; CHR = chromosome; CI = confidence interval; LS = Löfgren’s syndrome; NCA = noncoded allele; OR = odds ratio; <em>P</em> = unadjusted <em>P</em> value; <em>P</em><sub><em>GC</em></sub> = adjusted <em>P</em> value by genomic control (lambda = 1.16); SNP = single-nucleotide polymorphism.</p></div><div id="idm139913870774432"><p id="__p21" class="p p-first-last">Genetic effects (OR) adjusted according to sex, coded allele. OR is the risk on LS (0/1) per allele based on the additive genetic model. 95% CI is the 95% CI for the OR.</p></div><div id="idm139913870773616"><p id="__p22" class="p p-first-last">The entire list of LS-associated SNPs is provided in Table E2.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC4872654/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913874770656" class="figure"><a href="/pmc/articles/PMC4872654/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4872654_rccm.201507-1372OC_f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is rccm.201507-1372OC_f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4872654/bin/rccm.201507-1372OC_f1.jpg"></a></div><div id="largeobj_idm139913874770656" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4872654/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC4872654/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p23">Manhattan plot of all markers from the Immunochip. Each <em>dot</em> represents the <em>P</em><sub><em>GC</em></sub> value of association analysis [−log<sub>10</sub>(<em>P<sub>GC</sub></em>)] on the <em>y-axis</em> and position in chromosome on the <em>x-axis</em> (<em>P</em><sub><em>GC</em></sub> is genomic-controlled <em>P</em> value). (<em>Top</em>) Löfgren’s syndrome (LS) versus control subjects and (<em>bottom</em>) non-LS versus control subjects. <em>Red line</em> indicates <em>P</em><sub><em>GC</em></sub> &lt; 5 × 10<sup>−8</sup>; <em>blue line</em> is <em>P</em><sub>GC</sub> &lt; 5 × 10<sup>−5</sup>. HC = healthy control subjects.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC4872654/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913863324304" class="figure"><a href="/pmc/articles/PMC4872654/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4872654_rccm.201507-1372OC_f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is rccm.201507-1372OC_f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4872654/bin/rccm.201507-1372OC_f2.jpg"></a></div><div id="largeobj_idm139913863324304" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4872654/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC4872654/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p24">Regional association plots of chr6:25527073–33191175. Löfgren’s syndrome (LS)-associated single-nucleotide polymorphisms (SNPs) (<em>top</em>) at <em>P</em><sub><em>GC</em></sub> &lt; 5 × 10<sup>−8</sup> highlight region chr6:28910128–3295881. Non-LS–associated SNPs (<em>bottom</em>) at <em>P</em><sub><em>GC</em></sub> &lt; 5 × 10<sup>−8</sup> highlight region chr6: 32447326–32789970. <em>P</em><sub><em>GC</em></sub> = genomic-controlled <em>P</em> value.</p></div></div></div><p id="__p25">Results after conditioning on the top signal rs3130288 (<em>see</em> Table E2) showed that residual associations in the MHC classical class II locus were predominant, as indicated by leading SNP rs4642516 (<em>P</em><sub><em>cond,GC</em></sub> = 6.69 × 10<sup>−7</sup>) located between <em>HLA-DQB1</em> and <em>HLA-DQA2</em>. Additionally, findings at <em>P</em><sub><em>cond,GC</em></sub> less than 5 × 10<sup>−5</sup> were also noted in non-MHC genomic loci, 55 SNPs located near <em>ERAP1</em>, in <em>ERAP2</em> and <em>LNPEP</em>, and near <em>LIX1</em>, all on chromosome 5, and one SNP between <em>CSMD1</em> and <em>MCPH1</em> on chromosome 8. The Manhattan and Q-Q plots after conditioning are shown in Figure E3. Regional association plots for LS-associated SNPs on chromosome 6 at <em>P</em><sub><em>GC</em></sub> less than 5 × 10<sup>−8</sup> before and after conditioning on the top signal are illustrated in Figures E4A and E4B. Regional association plot for suggestive finding is shown in Figure E5.</p><p id="__p26" class="p p-last">Additionally, because <em>HLA-DRB1*03</em> plays a strong role in the course of the disease, which is particularly seen in patients with LS, we stratified our LS dataset (cases and control subjects) by <em>HLA-DRB1*03</em> carriers and noncarriers and performed Immunochip analysis on these subgroups. The association test in <em>HLA-DRB1*03</em> carriers (513 healthy control subjects and 253 LS) identified rs2248462 (<em>P</em><sub><em>GC</em></sub> = 1.20 × 10<sup>−4</sup>), an intergenic SNP located <em>HCP5</em> and <em>MICB</em> on chr6:31554775 as the top signal. Similarly, in <em>HLA-DBR1*03</em> noncarriers (1,573 healthy control subjects and 131 LS), the test revealed rs3104407 (<em>P</em><sub><em>GC</em></sub> = 1.25 × 10<sup>−5</sup>), an intergenic SNP located between <em>HLA-DQB1</em> and <em>HLA-DQA2</em> on chr6:32790430 as the top signal (further details on these analyses and results are given in the online supplement N<span class="small-caps">ote</span>). Notably, Immunochip results for <em>HLA-DRB1*03</em> stratified groups were consistent with Immunochip results for conditioning on <em>HLA-DRB1*03</em> (<em>see</em> the N<span class="small-caps">ote</span> in the online supplement and Figure E6), namely all identified associations for LS substantially attenuate (with <em>P</em> &gt; 5 × 10<sup>−8</sup>) when the effect of <em>HLA-DRB1*03</em> is considered.</p></div><div id="__sec17" class="sec sec-last"><p></p><h4 id="__sec17title" class="inline">Non-LS versus healthy control subjects </h4><p id="__p27" class="p p-first">Results from association analysis on non-LS (568 cases vs. 2,086 control subjects) showed rs1964995 (OR, 0.55; 95% CI, 0.47–0.64; <em>P</em><sub><em>GC</em></sub> = 2.57 × 10<sup>−13</sup>) located between <em>HLA-DRA</em> and <em>HLA-DRB5</em>, which localizes to MHC class II locus as the most significant signal. Fifty-two high signals located between <em>C6orf10</em> and <em>HLA-DQA2</em> (chr6:32447326–32789970) and between <em>PLAC9</em> and <em>ANXA11</em> (chr10:8191705–81916682) at <em>P</em><sub><em>GC</em></sub> &lt; 5 × 10<sup>−8</sup> were identified. The top 25 significant SNPs are provided in <a href="/pmc/articles/PMC4872654/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>. The entire list of non-LS–associated SNPs can be found in Table E3. Manhattan plot for Immunochip associations is illustrated in <a href="/pmc/articles/PMC4872654/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1B</span></a>. A close-up association regional plot is shown in <a href="/pmc/articles/PMC4872654/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a>. Genomic control for non-LS defined by inflation factor lambda was 1.17. The Q-Q plot and distribution of significant SNPs (<em>P</em><sub><em>GC</em></sub> &lt; 5 × 10<sup>−8</sup>) by variant type are illustrated in Figures E7 and E8, respectively. Suggestive signals (122 SNPs included in Table E3) at <em>P</em><em><sub>GC</sub></em> greater than 5 × 10<sup>−8</sup> and less than 5 × 10<sup>−5</sup> revealed additional signals in the MHC region, particularly clustering between classes III and II regions and outside the extended MHC on chromosome 10 in <em>ANXA11</em> and near <em>PLAC9</em>, <em>DYDC2</em>, <em>C10orf58</em>, and <em>TSPAN14</em>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p28">Results of Immunochip Analysis on Non-LS versus Healthy Control Subjects Highlighting the Top 25 SNPs at Significance <em>P</em> Less Than 5 × 10<sup>−8</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913871313312" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="center">CHR</th><th rowspan="2" colspan="1" align="center">BP</th><th rowspan="2" colspan="1" align="center">Gene (Nearby Genes)</th><th rowspan="2" colspan="1" align="center">CA/NCA</th><th rowspan="2" colspan="1" align="center">CAF</th><th colspan="3" rowspan="1" align="center">Discovery Cohort (Sweden, European Ancestry) (<em>n</em> = <em>2,750</em>)<hr></th><th colspan="2" rowspan="1" align="center">Replicating Cohort (Germany, European Ancestry) (<em>n</em> = <em>4,911</em>)<hr></th><th colspan="2" rowspan="1" align="center">Replicating Cohort (African American Ancestry) (<em>n</em> = <em>1,657</em>)<hr></th></tr><tr><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value</th><th rowspan="1" colspan="1" align="center"><em>P</em><sub><em>GC</em></sub></th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs1964995</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32557389</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.296</td><td rowspan="1" colspan="1" align="center">0.55 (0.47–0.64)</td><td rowspan="1" colspan="1" align="center">2.47E-15</td><td rowspan="1" colspan="1" align="center">2.57E-13</td><td rowspan="1" colspan="1" align="center">0.56 (0.48–0.66)</td><td rowspan="1" colspan="1" align="center">1.19E-12</td><td rowspan="1" colspan="1" align="center">0.70 (0.59–0.82)</td><td rowspan="1" colspan="1" align="center">1.95E-05</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9269110</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32551247</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.389</td><td rowspan="1" colspan="1" align="center">1.74 (1.51–2.01)</td><td rowspan="1" colspan="1" align="center">5.78E-14</td><td rowspan="1" colspan="1" align="center">3.82E-12</td><td rowspan="1" colspan="1" align="center">1.34 (1.16–1.56)</td><td rowspan="1" colspan="1" align="center">6.20E-05</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">—</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9269081</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32549078</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.376</td><td rowspan="1" colspan="1" align="center">1.71 (1.48–1.97)</td><td rowspan="1" colspan="1" align="center">4.01E-13</td><td rowspan="1" colspan="1" align="center">2.01E-11</td><td rowspan="1" colspan="1" align="center">1.33 (1.14–1.54)</td><td rowspan="1" colspan="1" align="center">1.63E-04</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">—</td></tr><tr><td rowspan="1" colspan="1" align="left">rs6923504</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32536164</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">G/C</td><td rowspan="1" colspan="1" align="center">0.381</td><td rowspan="1" colspan="1" align="center">1.66 (1.44–1.91)</td><td rowspan="1" colspan="1" align="center">4.05E-12</td><td rowspan="1" colspan="1" align="center">1.46E-10</td><td rowspan="1" colspan="1" align="center">1.35 (1.16–1.56)</td><td rowspan="1" colspan="1" align="center">6.09E-05</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">—</td></tr><tr><td rowspan="1" colspan="1" align="left">rs6903608</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32536263</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.381</td><td rowspan="1" colspan="1" align="center">1.66 (1.44–1.91)</td><td rowspan="1" colspan="1" align="center">4.05E-12</td><td rowspan="1" colspan="1" align="center">1.46E-10</td><td rowspan="1" colspan="1" align="center">1.36 (1.17–1.57)</td><td rowspan="1" colspan="1" align="center">3.55E-05</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">—</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9268882</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32539601</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.381</td><td rowspan="1" colspan="1" align="center">1.66 (1.44–1.91)</td><td rowspan="1" colspan="1" align="center">4.54E-12</td><td rowspan="1" colspan="1" align="center">1.61E-10</td><td rowspan="1" colspan="1" align="center">1.36 (1.17–1.57)</td><td rowspan="1" colspan="1" align="center">3.86E-05</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">—</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3998158</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32789970</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DQB1, HLA-DQA2)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.168</td><td rowspan="1" colspan="1" align="center">0.53 (0.45–0.64)</td><td rowspan="1" colspan="1" align="center">5.35E-12</td><td rowspan="1" colspan="1" align="center">1.85E-10</td><td rowspan="1" colspan="1" align="center">0.60 (0.49–0.74)</td><td rowspan="1" colspan="1" align="center">9.45E-07</td><td rowspan="1" colspan="1" align="center">1.19 (0.98–1.43)</td><td rowspan="1" colspan="1" align="center">7.47E-02</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9275582</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32788048</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DQB1, HLA-DQA2)</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.172</td><td rowspan="1" colspan="1" align="center">0.53 (0.45–0.64)</td><td rowspan="1" colspan="1" align="center">5.56E-12</td><td rowspan="1" colspan="1" align="center">1.91E-10</td><td rowspan="1" colspan="1" align="center">0.59 (0.48–0.72)</td><td rowspan="1" colspan="1" align="center">2.42E-07</td><td rowspan="1" colspan="1" align="center">1.29 (1.08–1.54)</td><td rowspan="1" colspan="1" align="center">4.29E-03</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9275580</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32787440</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DQB1, HLA-DQA2)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.178</td><td rowspan="1" colspan="1" align="center">0.54 (0.46–0.65)</td><td rowspan="1" colspan="1" align="center">9.71E-12</td><td rowspan="1" colspan="1" align="center">3.08E-10</td><td rowspan="1" colspan="1" align="center">0.60 (0.50–0.74)</td><td rowspan="1" colspan="1" align="center">4.99E-07</td><td rowspan="1" colspan="1" align="center">1.42 (1.20–1.67)</td><td rowspan="1" colspan="1" align="center">3.98E-05</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2395153</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32453573</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center">0.273</td><td rowspan="1" colspan="1" align="center">0.59 (0.51–0.69)</td><td rowspan="1" colspan="1" align="center">1.46E-11</td><td rowspan="1" colspan="1" align="center">4.38E-10</td><td rowspan="1" colspan="1" align="center">0.68 (0.58–0.79)</td><td rowspan="1" colspan="1" align="center">8.61E-07</td><td rowspan="1" colspan="1" align="center">0.63 (0.53–0.75)</td><td rowspan="1" colspan="1" align="center">2.12E-07</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129948</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32462622</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.390</td><td rowspan="1" colspan="1" align="center">1.63 (1.41–1.88)</td><td rowspan="1" colspan="1" align="center">4.11E-11</td><td rowspan="1" colspan="1" align="center">1.06E-09</td><td rowspan="1" colspan="1" align="center">1.35 (1.17–1.56</td><td rowspan="1" colspan="1" align="center">5.66E-05</td><td rowspan="1" colspan="1" align="center">1.35 (1.14–1.60)</td><td rowspan="1" colspan="1" align="center">6.34E-04</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3117098</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32466491</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.390</td><td rowspan="1" colspan="1" align="center">1.59 (1.38–1.84)</td><td rowspan="1" colspan="1" align="center">2.37E-10</td><td rowspan="1" colspan="1" align="center">4.78E-09</td><td rowspan="1" colspan="1" align="center">1.35 (1.16–1.56)</td><td rowspan="1" colspan="1" align="center">6.45E-05</td><td rowspan="1" colspan="1" align="center">1.36 (1.14–1.61)</td><td rowspan="1" colspan="1" align="center">4.63E-04</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129954</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32473558</td><td rowspan="1" colspan="1" align="center"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.390</td><td rowspan="1" colspan="1" align="center">1.59 (1.38–1.84)</td><td rowspan="1" colspan="1" align="center">2.37E-10</td><td rowspan="1" colspan="1" align="center">4.78E-09</td><td rowspan="1" colspan="1" align="center">1.35 (1.16–1.56)</td><td rowspan="1" colspan="1" align="center">6.10E-05</td><td rowspan="1" colspan="1" align="center">1.34 (1.13–1.59)</td><td rowspan="1" colspan="1" align="center">8.91E-04</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4424066</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32462406</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.314</td><td rowspan="1" colspan="1" align="center">0.62 (0.54–0.72)</td><td rowspan="1" colspan="1" align="center">2.54E-10</td><td rowspan="1" colspan="1" align="center">5.06E-09</td><td rowspan="1" colspan="1" align="center">0.70 (0.60–0.81)</td><td rowspan="1" colspan="1" align="center">3.09E-06</td><td rowspan="1" colspan="1" align="center">0.75 (0.64–0.87)</td><td rowspan="1" colspan="1" align="center">1.97E-04</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129961</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32486918</td><td rowspan="1" colspan="1" align="center"><em>(BTNL2, HLA-DRA)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.390</td><td rowspan="1" colspan="1" align="center">1.59 (1.38–1.84)</td><td rowspan="1" colspan="1" align="center">2.82E-10</td><td rowspan="1" colspan="1" align="center">5.53E-09</td><td rowspan="1" colspan="1" align="center">1.34 (1.16–1.55)</td><td rowspan="1" colspan="1" align="center">7.52E-05</td><td rowspan="1" colspan="1" align="center">1.35 (1.14–1.60)</td><td rowspan="1" colspan="1" align="center">4.18E-04</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129955</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32473818</td><td rowspan="1" colspan="1" align="center"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.385</td><td rowspan="1" colspan="1" align="center">1.59 (1.38–1.84)</td><td rowspan="1" colspan="1" align="center">3.49E-10</td><td rowspan="1" colspan="1" align="center">6.64E-09</td><td rowspan="1" colspan="1" align="center">1.35 (1.17–1.56)</td><td rowspan="1" colspan="1" align="center">5.19E-05</td><td rowspan="1" colspan="1" align="center">1.34 (1.13–1.59)</td><td rowspan="1" colspan="1" align="center">8.24E-04</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4373382</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32458846</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.313</td><td rowspan="1" colspan="1" align="center">0.63 (0.54–0.73)</td><td rowspan="1" colspan="1" align="center">4.16E-10</td><td rowspan="1" colspan="1" align="center">7.72E-09</td><td rowspan="1" colspan="1" align="center">0.69 (0.59–0.80)</td><td rowspan="1" colspan="1" align="center">1.58E-06</td><td rowspan="1" colspan="1" align="center">0.75 (0.65–0.87)</td><td rowspan="1" colspan="1" align="center">1.73E-04</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3817973</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32469089</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.314</td><td rowspan="1" colspan="1" align="center">0.63 (0.54–0.73)</td><td rowspan="1" colspan="1" align="center">4.35E-10</td><td rowspan="1" colspan="1" align="center">8.02E-09</td><td rowspan="1" colspan="1" align="center">0.69 (0.59–0.80)</td><td rowspan="1" colspan="1" align="center">1.57E-06</td><td rowspan="1" colspan="1" align="center">0.75 (0.65–0.88)</td><td rowspan="1" colspan="1" align="center">2.68E-04</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2076529</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32471933</td><td rowspan="1" colspan="1" align="center"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.314</td><td rowspan="1" colspan="1" align="center">0.63 (0.54–0.73)</td><td rowspan="1" colspan="1" align="center">4.35E-10</td><td rowspan="1" colspan="1" align="center">8.02E-09</td><td rowspan="1" colspan="1" align="center">0.69 (0.59–0.80)</td><td rowspan="1" colspan="1" align="center">1.48E-06</td><td rowspan="1" colspan="1" align="center">0.75 (0.65–0.88)</td><td rowspan="1" colspan="1" align="center">2.56E-04</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9268472</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32463583</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.314</td><td rowspan="1" colspan="1" align="center">0.63 (0.54–0.73)</td><td rowspan="1" colspan="1" align="center">4.61E-10</td><td rowspan="1" colspan="1" align="center">8.43E-09</td><td rowspan="1" colspan="1" align="center">0.69 (0.59–0.80)</td><td rowspan="1" colspan="1" align="center">1.57E-06</td><td rowspan="1" colspan="1" align="center">0.76 (0.65–0.89)</td><td rowspan="1" colspan="1" align="center">4.44E-04</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9271588</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32698931</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRB1, HLA-DQA1)</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.351</td><td rowspan="1" colspan="1" align="center">0.64 (0.56–0.74)</td><td rowspan="1" colspan="1" align="center">8.35E-10</td><td rowspan="1" colspan="1" align="center">1.40E-08</td><td rowspan="1" colspan="1" align="center">0.67 (0.58–0.78)</td><td rowspan="1" colspan="1" align="center">2.40E-07</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">—</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2858332</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32789139</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DQB1, HLA-DQA2)</em></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.354</td><td rowspan="1" colspan="1" align="center">0.65 (0.56–0.75)</td><td rowspan="1" colspan="1" align="center">1.47E-09</td><td rowspan="1" colspan="1" align="center">2.28E-08</td><td rowspan="1" colspan="1" align="center">1.36 (1.18–1.57)</td><td rowspan="1" colspan="1" align="center">3.07E-05</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">—</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3830135</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32656442</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRB1</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.054</td><td rowspan="1" colspan="1" align="center">0.42 (0.32–0.56)</td><td rowspan="1" colspan="1" align="center">1.82E-09</td><td rowspan="1" colspan="1" align="center">2.74E-08</td><td rowspan="1" colspan="1" align="center">0.56 (0.42–0.74)</td><td rowspan="1" colspan="1" align="center">3.16E-05</td><td rowspan="1" colspan="1" align="center">—</td><td rowspan="1" colspan="1" align="center">—</td></tr></tbody></table></div><div id="largeobj_idm139913871313312" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4872654/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139913862935696"><p id="__p29" class="p p-first-last"><em>Definition of abbreviations</em>: BP = base pairs; CA = coded allele; CAF = coded allele frequency; CHR = chromosome; CI = confidence interval; LS = Löfgren’s syndrome; NCA = noncoded allele; OR = odds ratio; <em>P</em> = unadjusted <em>P</em> value; <em>P</em><sub><em>GC</em></sub> = adjusted <em>P</em> value by genomic control (lambda = 1.17); SNP = single-nucleotide polymorphism.</p></div><div id="idm139913870133920"><p id="__p30" class="p p-first-last">Genetic effects (OR) adjusted according to sex, coded allele. OR is the risk on LS (0/1) per allele based on the additive genetic model. 95% CI is the 95% CI for the OR.</p></div><div id="idm139913866011296"><p id="__p31" class="p p-first-last">The entire list of non-LS–associated SNPs is provided in Table E3.</p></div></div></div><p id="__p32">Results after conditioning on top signal rs1964995 (<em>see</em> Table E4) stressed rs1049550 (<em>P</em><sub><em>cond,GC</em></sub> = 2.19 × 10<sup>−7</sup>) located in a coding region of <em>ANXA11</em> as the highest peak, followed by four intergenic SNPs, adjacent to previously observed genes. Suggestive signals were also observed, of which most (intergenic SNPs) were clustered together among the HLA class II genes (chr6:32411670–32873563). Manhattan and Q-Q plots after conditioning are shown in Figure E9. Regional association plots for non-LS–associated SNPs on chromosome 6 at <em>P</em><sub><em>GC</em></sub> less than 5 × 10<sup>−8</sup> before and after conditioning on the top signal are illustrated in Figures E10A and E10B. Regional association plot for suggestive signals at <em>P</em><sub><em>GC</em></sub> less than 5 × 10<sup>−5</sup> is illustrated in Figure E11. Association results from evaluating both phenotypes, LS versus non-LS including conditioning on the top signal and on <em>HLA-DRB1</em> alleles, and replication results are provided in the N<span class="small-caps">ote</span> in the online supplement.</p><p id="__p33" class="p p-last">Additionally, considering that the non-LS phenotype may be confounded by phenotype heterogeneity because of the presence of extrapulmonary manifestations as often seen in patients without LS, we reran Immunochip analysis by excluding patients without LS with confirmed extrapulmonary manifestations. Herein, 451 non-LS cases without extrapulmonary manifestations and 2,086 healthy control subjects were reanalyzed. Immunochip results showed rs1964995 (OR, 0.50; 95% CI, 0.42–0.59; <em>P</em><sub><em>GC</em></sub> = 4.31 × 10<sup>−13</sup>) as the top association along with similar high and suggested signals as identified in the main Immunochip for non-LS, thus implying that the same dominating gene associations are important in both non-LS groups with or without extrapulmonary manifestations. Results from Immunochip for non-LS without pulmonary manifestation are adjoined with main Immunochip findings for non-LS as given in Table E3.</p></div></div><div id="__sec18" class="sec"><h3 id="__sec18title">Replication of Significant SNPs</h3><p id="__p34" class="p p-first">Results from the association analysis on LS (64 LS vs. 4,498 control subjects) conducted in an independent German cohort confirmed the association of many signals. In the LS group, out of 1,246 high signals identified in the discovery cohort, 671 were confirmed with Bonferroni-adjusted <em>P</em> less than 4 × 10<sup>−5</sup> (0.05/1,246) to account for multiple testing. The top signal rs3130288 identified in the discovery cohort was well replicated (Germany: OR, 4.13; 95% CI, 2.83–6.02; <em>P</em> = 1.66 × 10<sup>−15</sup>) (the Netherlands: OR, 4.54; 95% CI, 2.97–6.93; <em>P</em> = 2.82 × 10<sup>−12</sup>) (Czech Republic: OR, 4.94; 95% CI, 2.53–9.65; <em>P</em> = 2.88 × 10<sup>−6</sup>). Confirmed signals localized to the extended MHC region spanning from <em>ZNF184</em> to <em>HLA-DPB2</em> (chr6:27521903–33191175), as illustrated in the regional association plot (<a href="/pmc/articles/PMC4872654/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a>). Replication for assessing 692 suggestive signals identified with <em>P</em><em><sub>GC</sub></em> greater than 5 × 10<sup>−8</sup> and less than 5 × 10<sup>−5</sup> in the Swedish population resulted in the confirmation of 56 suggested signals with Bonferroni-adjusted <em>P</em> less than 7.22 × 10<sup>−5</sup> (0.05/692), which localized to MHC class I and class II regions. Non-MHC suggested signals located in <em>LRRC16A</em> (rs6921589, <em>P</em> = 6.54 × 10<sup>−2</sup> and rs9295661, <em>P</em> = 6.61 × 10<sup>−2</sup>) were borderline significant. Signals in <em>ADCY3</em> (rs13407913, rs6752378, and rs10182181) were not confirmed in the German cohort, possibly because of few number of LS cases. Results from this analysis are given in Table E1.</p><p id="__p35">In the non-LS group, out of 38 high signals identified in the discovery cohort, 27 were confirmed in the German cohort and 20 in the African American cohort with Bonferroni-adjusted <em>P</em> less than 1.32 × 10<sup>−3</sup> (0.05/38). The lead signal rs1964995 was well replicated in all participating cohorts (Germany: OR, 0.56; 95% CI, 0.49–0.66; <em>P</em> = 1.19 × 10<sup>−12</sup>) (African American: OR, 0.70; 95% CI, 0.59–0.82; <em>P</em> = 1.95 × 10<sup>−5</sup>) (the Netherlands: OR, 0.71; 95% CI, 0.51–0.97; <em>P</em> = 3.31 × 10<sup>−2</sup>) (Czech Republic: OR, 0.55; 95% CI, 0.39–0.77; <em>P</em> = 4.05 × 10<sup>−4</sup>). All confirmed signals localized to MHC class II region, clustering among <em>C6orf10</em>, <em>BTNL2</em>, <em>HLA-DRA</em>, <em>HLA-DRB5, HLA-DRB1</em>, <em>HLA-DQA1</em>, and <em>HLA-DQA2</em>, as illustrated in the regional association plot (<a href="/pmc/articles/PMC4872654/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>). Replication assessment of 135 suggestive signals confirmed 41 suggestive signals in the Germany cohort and 30 suggested signals in the USA-AA cohort with Bonferroni-adjusted <em>P</em> less than 3.70 × 10<sup>−4</sup> (0.05/135). These localized in the same regions as confirmed high signals, thus enriching the density of the associated region. Out of the seven signals located outside the MHC, three signals (rs1953600, rs2573346, and rs1049550; in strong LD, <em>R</em><sup><em>2</em></sup> ≥ 0.87, <em>D′</em>≥ 0.93) located between <em>PLAC9</em> and <em>ANXA11</em> (intergenic), and inside <em>ANXA11</em> (intronic and coding) were replicated with nominal significance <em>P</em> less than 0.05. Results from this analysis are given in Table E3.</p><p id="__p36">Results from evaluating the association of six SNPs located outside the MHC region and conducted in two independent cohorts from Czech Republic and the Netherlands led to the confirmation of two SNPs. For LS, rs10108612 (discovery <em>P</em><sub><em>GC</em></sub> = 1.97 × 10<sup>−2</sup>) located between <em>CSMD1</em> and <em>MCPH1</em> on chromosome 8 was marginally significant (<em>P</em> = 5.81 × 10<sup>−2</sup>) in the Czech cohort with and nonsignificant in the Dutch cohort. Similarly, rs13407913 (discovery <em>P</em><sub><em>GC</em></sub> = 2.31 × 10<sup>−5</sup>) located in an intron of <em>ADCY3</em> on chromosome 2 was marginally significant (<em>P</em> = 7.8 × 10<sup>−2</sup>) in the Dutch LS group and nonsignificant in the Czech LS group. For non-LS, no associations among the tested SNPs were found in the either Czech or Dutch samples.</p><p id="__p37" class="p p-last">In summary, combining replicated high and suggestive signals, we observed that 710 confirmed variants were associated exclusively to LS compared with control subjects, 51 to non-LS compared with control subjects, and 17 were associated to both LS and non-LS. Whereas in a population of African ancestry, out of 50 signals associated with non-LS, 17 were LS-associated signals found in populations of European ancestry. In populations with European ancestry, the shared genomic loci associated to both LS and non-LS spanned between two intergenic regions between <em>C6orf10</em> and <em>BTNL2</em> and between <em>HLA-DRA</em> and <em>HLA-DRB5</em> (chr6:32453573–32528157), whereas in a population with African ancestry it extended along chromosome 6 to an intergenic region located between <em>HLA-DQB1</em> and <em>HLA-DQA2</em> (chr6: 32453573–32778286).</p></div><div id="__sec19" class="sec"><h3 id="__sec19title">Meta-analysis of Associated SNPs</h3><p id="__p38" class="p p-first">In the meta-analysis, results for estimating the overall effect size (OR) of lead SNPs associated with LS and non-LS, respectively, were highly significant (rs3130288 meta-<em>P</em> =  4.3 × 10<sup>−66</sup> and rs1964995 meta-<em>P</em> =  2.5 × 10<sup>−32</sup>) (<a href="/pmc/articles/PMC4872654/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure 3</span></a>). Likewise, results for SNPs located outside the MHC region showed a similar pattern: for LS, rs13407913 (meta-<em>P</em> = 3.8 × 10<sup>−3</sup>), rs9295661 (meta-<em>P</em> = 8.1 × 10<sup>−10</sup>), and rs10108612 (meta-<em>P</em> = 2.3 × 10<sup>−2</sup>); and for non-LS, rs1953600 (meta-<em>P</em> = 1.3 × 10<sup>−11</sup>), rs2573346 (meta-<em>P</em> = 1.9 × 10<sup>−10</sup>), and rs1049550 (meta-<em>P</em> = 2.6 × 10<sup>−1a</sup>). Forest plots for SNPs outside the MHC are illustrated in <a href="/pmc/articles/PMC4872654/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure 4</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC4872654/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913873394032" class="figure"><a href="/pmc/articles/PMC4872654/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4872654_rccm.201507-1372OC_f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is rccm.201507-1372OC_f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4872654/bin/rccm.201507-1372OC_f3.jpg"></a></div><div id="largeobj_idm139913873394032" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4872654/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC4872654/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure 3.</a></div><!--caption a7--><div class="caption"><p id="__p39">Forest plots showing Löfgren’s syndrome (LS) and non-LS lead single-nucleotide polymorphism (SNP) associations inside the major histocompatibility complex. Plots show the study-specific association estimates (odds ratio) and 95% confidence intervals (CIs) for discovery set (Swedish) and all replication studies (German, African American [USA-AA], Dutch, and Czech). Rs3130288 (<em>A</em>) and rs1964995 (<em>B</em>) are the lead SNPs associated with LS and non-LS, respectively. The summary estimate includes the association estimate and CI under fixed-effect model for the meta-analysis combining the discovery and replicating cohort shown as a <em>diamond</em>. <em>I</em><sup><em>2</em></sup> indicates the percentage of total variation in study estimates because of heterogeneity in the meta-analysis.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC4872654/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913864578736" class="figure"><a href="/pmc/articles/PMC4872654/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4872654_rccm.201507-1372OC_f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is rccm.201507-1372OC_f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4872654/bin/rccm.201507-1372OC_f4.jpg"></a></div><div id="largeobj_idm139913864578736" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4872654/figure/fig4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC4872654/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Figure 4.</a></div><!--caption a7--><div class="caption"><p id="__p40">Forest plots showing Löfgren’s syndrome (LS) and non-LS single-nucleotide polymorphism (SNP) associations outside the major histocompatibility complex. Plots show the study-specific association estimates (odds ratio) and 95% confidence intervals (CIs) for discovery set (Swedish) and replication studies (German, African American [USA-AA], Dutch, Czech). Results for LS-associated SNPs rs133409913, rs9295661, and rs10108612 are shown in the <em>top</em>. Results for non-LS–associated SNP rs1953600, rs2573346, and rs1049550 are shown in the <em>bottom</em>. The association estimate and CI (estimated under fixed-effect model) for the meta-analysis combining the discovery and replicating cohort are shown as a <em>diamond</em>. <em>Blank spaces</em> indicate occasions in which a particularly study was not able to provide results for a given SNP. <em>I</em><sup><em>2</em></sup> indicates the percentage of total variation in study estimates because of heterogeneity in the meta-analysis.</p></div></div></div><p id="__p41" class="p">Combining association summary statistics from both Swedish and German cohorts, results from meta-analysis on 1,938 high and suggested LS signals revealed 1,499 signals (defining 267 loci; 22/1,499 signals had heterogeneity <em>P</em> &lt; 0.05 and <em>I</em><sup><em>2</em></sup> = 0%) under the fixed-effects model and 1,320 signals (defining 241 loci; 116/1,320 had <em>I</em><sup><em>2</em></sup> = [ 0.7%, 62.4%]) under the random-effects model with strong evidence for association at meta-<em>P</em> less than 5 × 10<sup>−8</sup>. <a href="/pmc/articles/PMC4872654/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a> provides the top 25 LS-associated findings. The entire list for LS-associated SNPs can be found in Table E5. Similarly, meta-analysis combining association summary statistics from Swedish, German, and African American cohorts for evaluating 173 high and suggested non-LS signals showed 47 signals (defining 12 loci) with strong evidence for association under the fixed-effects model and 37 signals (defining eight loci) under the random-effects model, with strong evidence for association at meta-<em>P</em> less than 5 × 10<sup>−8</sup>. Non-LS signals were selected with heterogeneity <em>P</em> greater than 0.05. <a href="/pmc/articles/PMC4872654/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a> provides the 25 top non-LS–associated signals. The entire list for non-LS–associated SNPs can be found in Table E6.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p42">Results from Combined Meta-analysis on LS-associated SNPs at Significance Meta-<em>P</em> Less Than 5 × 10<sup>−8</sup> (Top 25 Findings)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913865653920" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="center">CHR</th><th rowspan="2" colspan="1" align="center">BP</th><th rowspan="2" colspan="1" align="center">CA/NCA</th><th rowspan="2" colspan="1" align="center">Gene (Nearby Genes)</th><th rowspan="2" colspan="1" align="center">SNP Location</th><th rowspan="2" colspan="1" align="center">Cohorts</th><th colspan="2" rowspan="1" align="center">Fixed-Effects Model<hr></th><th colspan="2" rowspan="1" align="center">Random-Effects Model<hr></th></tr><tr><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center">Meta-<em>P</em> Value</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center">Meta-<em>P</em> Value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs3130288</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32203979</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center"><em>CREBL1</em></td><td rowspan="1" colspan="1" align="center">3′ UTR</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">4.16 (3.54–4.90)</td><td rowspan="1" colspan="1" align="center">4.30E-66</td><td rowspan="1" colspan="1" align="center">4.16 (3.54–4.90)</td><td rowspan="1" colspan="1" align="center">4.30E-66</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129927</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32441805</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">4.04 (3.36–4.84)</td><td rowspan="1" colspan="1" align="center">1.29E-50</td><td rowspan="1" colspan="1" align="center">4.04 (3.36–4.84)</td><td rowspan="1" colspan="1" align="center">1.29E-50</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9268219</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32392086</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">4.05 (3.37–4.87)</td><td rowspan="1" colspan="1" align="center">1.51E-50</td><td rowspan="1" colspan="1" align="center">4.05 (3.37–4.87)</td><td rowspan="1" colspan="1" align="center">1.51E-50</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129950</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32466179</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">4.06 (3.38–4.88)</td><td rowspan="1" colspan="1" align="center">1.79E-50</td><td rowspan="1" colspan="1" align="center">4.06 (3.38–4.88)</td><td rowspan="1" colspan="1" align="center">1.79E-50</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2395149</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32433540</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.98 (3.32–4.77)</td><td rowspan="1" colspan="1" align="center">3.55E-50</td><td rowspan="1" colspan="1" align="center">3.98 (3.32–4.77)</td><td rowspan="1" colspan="1" align="center">3.55E-50</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3135394</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32516475</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.92 (3.27–4.70)</td><td rowspan="1" colspan="1" align="center">7.66E-50</td><td rowspan="1" colspan="1" align="center">3.92 (3.27–4.70)</td><td rowspan="1" colspan="1" align="center">7.66E-50</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1265757</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32410360</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.97 (3.31–4.77)</td><td rowspan="1" colspan="1" align="center">8.02E-50</td><td rowspan="1" colspan="1" align="center">3.97 (3.31–4.77)</td><td rowspan="1" colspan="1" align="center">8.02E-50</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9268235</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32398186</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.97 (3.31–4.76)</td><td rowspan="1" colspan="1" align="center">8.22E-50</td><td rowspan="1" colspan="1" align="center">3.97 (3.31–4.76)</td><td rowspan="1" colspan="1" align="center">8.22E-50</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7775397</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32369230</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">Coding</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.94 (3.28–4.73)</td><td rowspan="1" colspan="1" align="center">2.94E-49</td><td rowspan="1" colspan="1" align="center">3.94 (3.28–4.73)</td><td rowspan="1" colspan="1" align="center">2.94E-49</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9268177</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32382860</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.94 (3.28–4.73)</td><td rowspan="1" colspan="1" align="center">2.98E-49</td><td rowspan="1" colspan="1" align="center">3.94 (3.28–4.73)</td><td rowspan="1" colspan="1" align="center">2.98E-49</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2187668</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32713862</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>HLA-DQA1</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.94 (3.28–4.73)</td><td rowspan="1" colspan="1" align="center">3.29E-49</td><td rowspan="1" colspan="1" align="center">3.94 (3.28–4.73)</td><td rowspan="1" colspan="1" align="center">3.29E-49</td></tr><tr><td rowspan="1" colspan="1" align="left">rs389884</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32048876</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>STK19</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.88 (3.24–4.65)</td><td rowspan="1" colspan="1" align="center">4.28E-49</td><td rowspan="1" colspan="1" align="center">3.88 (3.24–4.65)</td><td rowspan="1" colspan="1" align="center">4.28E-49</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3132971</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32338234</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center"><em>(NOTCH4, C6orf10)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.93 (3.28–4.72)</td><td rowspan="1" colspan="1" align="center">6.55E-49</td><td rowspan="1" colspan="1" align="center">3.93 (3.28–4.72)</td><td rowspan="1" colspan="1" align="center">6.55E-49</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1794282</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32774504</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DQB1, HLA-DQA2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.92 (3.26–4.70)</td><td rowspan="1" colspan="1" align="center">9.82E-49</td><td rowspan="1" colspan="1" align="center">3.92 (3.26–4.70)</td><td rowspan="1" colspan="1" align="center">9.82E-49</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1150753</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32167845</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>TNXB</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.89 (3.25–4.67)</td><td rowspan="1" colspan="1" align="center">1.26E-48</td><td rowspan="1" colspan="1" align="center">3.89 (3.25–4.67)</td><td rowspan="1" colspan="1" align="center">1.26E-48</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129716</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32765414</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DQB1, HLA-DQA2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.88 (3.23–4.65)</td><td rowspan="1" colspan="1" align="center">1.77E-48</td><td rowspan="1" colspan="1" align="center">3.88 (3.23–4.65)</td><td rowspan="1" colspan="1" align="center">1.77E-48</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2854275</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32736406</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center"><em>HLA-DQB1</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.91 (3.25–4.69)</td><td rowspan="1" colspan="1" align="center">2.39E-48</td><td rowspan="1" colspan="1" align="center">3.91 (3.25–4.69)</td><td rowspan="1" colspan="1" align="center">2.39E-48</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2856674</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32767623</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DQB1, HLA-DQA2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.88 (3.24–4.66)</td><td rowspan="1" colspan="1" align="center">2.80E-48</td><td rowspan="1" colspan="1" align="center">3.88 (3.24–4.66)</td><td rowspan="1" colspan="1" align="center">2.80E-48</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3130490</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31847099</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center"><em>C6orf27</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.82 (3.19–4.58)</td><td rowspan="1" colspan="1" align="center">6.92E-48</td><td rowspan="1" colspan="1" align="center">3.82 (3.19–4.58)</td><td rowspan="1" colspan="1" align="center">6.92E-48</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1270942</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32026839</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>CFB</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.79 (3.17–4.54)</td><td rowspan="1" colspan="1" align="center">7.12E-48</td><td rowspan="1" colspan="1" align="center">3.79 (3.17–4.54)</td><td rowspan="1" colspan="1" align="center">7.12E-48</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3132449</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31733992</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>(APOM, C6orf47)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.76 (3.14–4.50)</td><td rowspan="1" colspan="1" align="center">1.11E-47</td><td rowspan="1" colspan="1" align="center">3.76 (3.14–4.50)</td><td rowspan="1" colspan="1" align="center">1.11E-47</td></tr><tr><td rowspan="1" colspan="1" align="left">rs519417</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31986412</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>(ZBTB12, C2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.78 (3.16–4.52)</td><td rowspan="1" colspan="1" align="center">1.34E-47</td><td rowspan="1" colspan="1" align="center">3.78 (3.16–4.52)</td><td rowspan="1" colspan="1" align="center">1.34E-47</td></tr><tr><td rowspan="1" colspan="1" align="left">rs497309</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32000463</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center"><em>(ZBTB12, C2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.78 (3.16–4.53)</td><td rowspan="1" colspan="1" align="center">1.82E-47</td><td rowspan="1" colspan="1" align="center">3.78 (3.16–4.53)</td><td rowspan="1" colspan="1" align="center">1.82E-47</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3117577</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31835453</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>MSH5</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.76 (3.14–4.49)</td><td rowspan="1" colspan="1" align="center">1.96E-47</td><td rowspan="1" colspan="1" align="center">3.76 (3.14–4.49)</td><td rowspan="1" colspan="1" align="center">1.96E-47</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3117582</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31728499</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center"><em>(BAT3, APOM)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">GER, SWE</td><td rowspan="1" colspan="1" align="center">3.73 (3.12–4.46)</td><td rowspan="1" colspan="1" align="center">2.53E-47</td><td rowspan="1" colspan="1" align="center">3.73 (3.12–4.46)</td><td rowspan="1" colspan="1" align="center">2.53E-47</td></tr></tbody></table></div><div id="largeobj_idm139913865653920" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4872654/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139913868794944"><p id="__p43" class="p p-first-last"><em>Definition of abbreviations</em>: BP = base pairs; CA = coded allele; CHR = chromosome; CI = confidence interval; GER = Germany; LS = Löfgren’s syndrome; NCA = noncoded allele; OR = odds ratio; SNP = single-nucleotide polymorphism; SWE = Sweden; UTR = untranslated region.</p></div><div id="idm139913868793280"><p id="__p44" class="p p-first-last">The entire list of LS-associated SNPs is provided in Table E5.</p></div><div id="idm139913868792560"><p id="__p45" class="p p-first-last">The overall association estimates (OR) and 95% CI for LS-associated SNPs under fixed-effect and random-effect models, respectively. Cohorts from SWE and GER are of European ancestry.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>Table 4.</h3><!--caption a7--><div class="caption"><p id="__p46">Results from Combined Meta-analysis on Non-LS–associated SNPs at Significance Meta-<em>P</em> Less Than 5 × 10<sup>−8</sup> (Top 25 Findings)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913869895840" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="center">CHR</th><th rowspan="2" colspan="1" align="center">BP</th><th rowspan="2" colspan="1" align="center">CA/NCA</th><th rowspan="2" colspan="1" align="center">Gene (Nearby Genes)</th><th rowspan="2" colspan="1" align="center">SNP Location</th><th rowspan="2" colspan="1" align="center">Cohorts</th><th colspan="2" rowspan="1" align="center">Fixed-Effects Model<hr></th><th colspan="2" rowspan="1" align="center">Random-Effects Model<hr></th><th rowspan="1" colspan="1" align="center">&nbsp;</th><th rowspan="1" colspan="1" align="center">&nbsp;</th></tr><tr><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center">Meta-<em>P</em> Value</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center">Meta-<em>P</em> Value</th><th rowspan="1" colspan="1" align="center">τ<sup>2</sup><em>P</em> Value</th><th rowspan="1" colspan="1" align="center"><em>I</em><sup>2</sup> (<em>%</em>)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs1964995</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32557389</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.60 (0.55–0.65)</td><td rowspan="1" colspan="1" align="center">2.53E-32</td><td rowspan="1" colspan="1" align="center">0.61 (0.54–0.68)</td><td rowspan="1" colspan="1" align="center">5.99E-17</td><td rowspan="1" colspan="1" align="center">1.55E-01</td><td rowspan="1" colspan="1" align="center">41.79</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2213585</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32521128</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.51 (1.39–1.63)</td><td rowspan="1" colspan="1" align="center">8.82E-23</td><td rowspan="1" colspan="1" align="center">1.51 (1.34–1.70)</td><td rowspan="1" colspan="1" align="center">3.36E-11</td><td rowspan="1" colspan="1" align="center">1.11E-01</td><td rowspan="1" colspan="1" align="center">54.69</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2227139</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32521437</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.50 (1.39–1.63)</td><td rowspan="1" colspan="1" align="center">8.96E-23</td><td rowspan="1" colspan="1" align="center">1.51 (1.33–1.70)</td><td rowspan="1" colspan="1" align="center">6.87E-11</td><td rowspan="1" colspan="1" align="center">1.03E-01</td><td rowspan="1" colspan="1" align="center">56.15</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2213586</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32521072</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.50 (1.38–1.63)</td><td rowspan="1" colspan="1" align="center">1.29E-22</td><td rowspan="1" colspan="1" align="center">1.50 (1.33–1.70)</td><td rowspan="1" colspan="1" align="center">4.58E-11</td><td rowspan="1" colspan="1" align="center">1.09E-01</td><td rowspan="1" colspan="1" align="center">54.98</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7192</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32519624</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="center">Coding</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.50 (1.39–1.63)</td><td rowspan="1" colspan="1" align="center">1.31E-22</td><td rowspan="1" colspan="1" align="center">1.51 (1.34–1.69)</td><td rowspan="1" colspan="1" align="center">8.78E-12</td><td rowspan="1" colspan="1" align="center">1.27E-01</td><td rowspan="1" colspan="1" align="center">51.57</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2395153</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32453573</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.63 (0.58–0.69)</td><td rowspan="1" colspan="1" align="center">1.76E-22</td><td rowspan="1" colspan="1" align="center">0.63 (0.58–0.69)</td><td rowspan="1" colspan="1" align="center">1.76E-22</td><td rowspan="1" colspan="1" align="center">5.17E-01</td><td rowspan="1" colspan="1" align="center">0.00</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7195</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32520517</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="center">UTR</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.50 (1.38–1.63)</td><td rowspan="1" colspan="1" align="center">1.77E-22</td><td rowspan="1" colspan="1" align="center">1.50 (1.34–1.68)</td><td rowspan="1" colspan="1" align="center">9.57E-13</td><td rowspan="1" colspan="1" align="center">1.52E-01</td><td rowspan="1" colspan="1" align="center">46.70</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3763327</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32521808</td><td rowspan="1" colspan="1" align="center">G/C</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, USA-AA</td><td rowspan="1" colspan="1" align="center">1.57 (1.42–1.73)</td><td rowspan="1" colspan="1" align="center">7.36E-19</td><td rowspan="1" colspan="1" align="center">1.57 (1.34–1.85)</td><td rowspan="1" colspan="1" align="center">2.75E-08</td><td rowspan="1" colspan="1" align="center">1.08E-01</td><td rowspan="1" colspan="1" align="center">61.20</td></tr><tr><td rowspan="1" colspan="1" align="left">rs477515</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32677669</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRB1, HLA-DQA1)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.64 (0.58–0.71)</td><td rowspan="1" colspan="1" align="center">3.93E-18</td><td rowspan="1" colspan="1" align="center">0.63 (0.54–0.75)</td><td rowspan="1" colspan="1" align="center">5.32E-08</td><td rowspan="1" colspan="1" align="center">7.79E-02</td><td rowspan="1" colspan="1" align="center">61.89</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2395182</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32521295</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.49 (1.36–1.62)</td><td rowspan="1" colspan="1" align="center">7.05E-18</td><td rowspan="1" colspan="1" align="center">1.47 (1.22–1.78)</td><td rowspan="1" colspan="1" align="center">4.95E-05</td><td rowspan="1" colspan="1" align="center">1.48E-02</td><td rowspan="1" colspan="1" align="center">76.86</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4373382</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32458846</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.69 (0.63–0.75)</td><td rowspan="1" colspan="1" align="center">1.17E-17</td><td rowspan="1" colspan="1" align="center">0.69 (0.62–0.76)</td><td rowspan="1" colspan="1" align="center">5.81E-13</td><td rowspan="1" colspan="1" align="center">2.48E-01</td><td rowspan="1" colspan="1" align="center">28.87</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129952</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32467741</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, USA-AA</td><td rowspan="1" colspan="1" align="center">1.64 (1.46–1.84)</td><td rowspan="1" colspan="1" align="center">1.54E-17</td><td rowspan="1" colspan="1" align="center">1.64 (1.46–1.84)</td><td rowspan="1" colspan="1" align="center">1.54E-17</td><td rowspan="1" colspan="1" align="center">4.18E-01</td><td rowspan="1" colspan="1" align="center">0.00</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4424066</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32462406</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.69 (0.63–0.75)</td><td rowspan="1" colspan="1" align="center">1.73E-17</td><td rowspan="1" colspan="1" align="center">0.69 (0.62–0.76)</td><td rowspan="1" colspan="1" align="center">3.44E-12</td><td rowspan="1" colspan="1" align="center">2.28E-01</td><td rowspan="1" colspan="1" align="center">32.82</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2076529</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32471933</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="center">Coding</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.69 (0.63–0.75)</td><td rowspan="1" colspan="1" align="center">1.74E-17</td><td rowspan="1" colspan="1" align="center">0.69 (0.62–0.76)</td><td rowspan="1" colspan="1" align="center">1.70E-12</td><td rowspan="1" colspan="1" align="center">2.37E-01</td><td rowspan="1" colspan="1" align="center">30.92</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3817973</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32469089</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.69 (0.63–0.75)</td><td rowspan="1" colspan="1" align="center">1.96E-17</td><td rowspan="1" colspan="1" align="center">0.69 (0.62–0.76)</td><td rowspan="1" colspan="1" align="center">2.39E-12</td><td rowspan="1" colspan="1" align="center">2.34E-01</td><td rowspan="1" colspan="1" align="center">31.62</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2239802</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32519824</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.48 (1.35–1.61)</td><td rowspan="1" colspan="1" align="center">1.97E-17</td><td rowspan="1" colspan="1" align="center">1.46 (1.22–1.76)</td><td rowspan="1" colspan="1" align="center">5.85E-05</td><td rowspan="1" colspan="1" align="center">1.56E-02</td><td rowspan="1" colspan="1" align="center">76.65</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9268472</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32463583</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.69 (0.63–0.75)</td><td rowspan="1" colspan="1" align="center">3.84E-17</td><td rowspan="1" colspan="1" align="center">0.69 (0.62–0.77)</td><td rowspan="1" colspan="1" align="center">3.06E-11</td><td rowspan="1" colspan="1" align="center">2.04E-01</td><td rowspan="1" colspan="1" align="center">37.34</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3177928</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32520413</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="center">UTR</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.53 (0.46–0.62)</td><td rowspan="1" colspan="1" align="center">8.37E-17</td><td rowspan="1" colspan="1" align="center">0.53 (0.46–0.62)</td><td rowspan="1" colspan="1" align="center">8.37E-17</td><td rowspan="1" colspan="1" align="center">6.77E-01</td><td rowspan="1" colspan="1" align="center">0.00</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2076530</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32471794</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="center">Coding</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.69 (0.64–0.76)</td><td rowspan="1" colspan="1" align="center">1.21E-16</td><td rowspan="1" colspan="1" align="center">0.69 (0.64–0.76)</td><td rowspan="1" colspan="1" align="center">1.21E-16</td><td rowspan="1" colspan="1" align="center">4.96E-01</td><td rowspan="1" colspan="1" align="center">0.00</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9268969</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32542327</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"><em>(HLA-DRA, HLA-DRB5)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.66 (0.60–0.73)</td><td rowspan="1" colspan="1" align="center">1.29E-16</td><td rowspan="1" colspan="1" align="center">0.66 (0.60–0.73)</td><td rowspan="1" colspan="1" align="center">1.29E-16</td><td rowspan="1" colspan="1" align="center">6.60E-01</td><td rowspan="1" colspan="1" align="center">0.00</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129888</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32519704</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.48 (1.35–1.62)</td><td rowspan="1" colspan="1" align="center">2.07E-16</td><td rowspan="1" colspan="1" align="center">1.47 (1.25–1.74)</td><td rowspan="1" colspan="1" align="center">6.22E-06</td><td rowspan="1" colspan="1" align="center">4.00E-02</td><td rowspan="1" colspan="1" align="center">68.94</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129948</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32462622</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.44 (1.32–1.58)</td><td rowspan="1" colspan="1" align="center">2.30E-16</td><td rowspan="1" colspan="1" align="center">1.44 (1.27–1.63)</td><td rowspan="1" colspan="1" align="center">1.46E-08</td><td rowspan="1" colspan="1" align="center">1.31E-01</td><td rowspan="1" colspan="1" align="center">50.83</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2294878</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32475773</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="center">Intron</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">0.70 (0.64–0.76)</td><td rowspan="1" colspan="1" align="center">3.66E-16</td><td rowspan="1" colspan="1" align="center">0.70 (0.64–0.76)</td><td rowspan="1" colspan="1" align="center">3.66E-16</td><td rowspan="1" colspan="1" align="center">9.24E-01</td><td rowspan="1" colspan="1" align="center">0.00</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3117098</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32466491</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>(C6orf10, BTNL2)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.44 (1.32–1.57)</td><td rowspan="1" colspan="1" align="center">6.96E-16</td><td rowspan="1" colspan="1" align="center">1.43 (1.28–1.60)</td><td rowspan="1" colspan="1" align="center">4.11E-10</td><td rowspan="1" colspan="1" align="center">2.01E-01</td><td rowspan="1" colspan="1" align="center">38.74</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3129961</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">32486918</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center"><em>(BTNL2, HLA-DRA)</em></td><td rowspan="1" colspan="1" align="center">Intergenic</td><td rowspan="1" colspan="1" align="center">SWE, GER, USA-AA</td><td rowspan="1" colspan="1" align="center">1.43 (1.31–1.56)</td><td rowspan="1" colspan="1" align="center">1.02E-15</td><td rowspan="1" colspan="1" align="center">1.43 (1.28–1.60)</td><td rowspan="1" colspan="1" align="center">6.67E-10</td><td rowspan="1" colspan="1" align="center">1.95E-01</td><td rowspan="1" colspan="1" align="center">39.66</td></tr></tbody></table></div><div id="largeobj_idm139913869895840" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4872654/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139913872756672"><p id="__p47" class="p p-first-last"><em>Definition of abbreviations</em>: BP = base pairs; CA = coded allele; CHR = chromosome; CI = confidence interval; GER = Germany; LS = Löfgren’s syndrome; NCA = noncoded allele; OR = odds ratio; SNP = single-nucleotide polymorphism; SWE = Sweden; USA-AA = US African American; UTR = untranslated region.</p></div><div id="idm139913872754960"><p id="__p48" class="p p-first-last">The entire list of non-LS–associated SNPs is provided in Table E6.</p></div><div id="idm139913872754240"><p id="__p49" class="p p-first-last">The overall association estimates (OR) and 95% CI for non-LS–associated SNPs under fixed-effect and random-effect models, respectively.</p></div><div id="idm139913872753392"><p id="__p50" class="p p-first-last">τ<sup>2</sup>
<em>P</em> represents the <em>P</em> value significance for the amount of heterogeneity in the true log odds; the <em>I</em><sup>2</sup> statistic represents how much of the total variability in the effect size estimate can be attributed to heterogeneity among the true effects. Cohorts from SWE and GER are of European ancestry; the USA-AA cohort is of African ancestry.</p></div></div></div></div><div id="__sec20" class="sec"><h3 id="__sec20title">Gene-centric Analysis</h3><p id="__p51" class="p p-first">A gene-based approach identified five genomic loci (<em>BTNL2</em>, <em>HLA-DRA</em>, and <em>HLA-DRB1</em> on chr6, and <em>PLAC9</em> and <em>ANXA11</em> on chr10) associated with non-LS and 179 genomic loci (from <em>SCGN</em> to <em>HLA-DPB1</em> localizing to extended MHC on chr6) associated with LS as exceeding a Bonferroni-corrected threshold of <em>P</em> less than 7.3 × 10<sup>−6</sup>. Remarkably, significant gene associations for both LS and non-LS identified by this method were similar to those identified by associated SNPs from Immunochip analyses, and thus strengthened our findings. <a href="/pmc/articles/PMC4872654/table/tbl5/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl5" rid-ob="ob-tbl5" co-legend-rid=""><span>Table 5</span></a> lists the 15 most significant LS- and non-LS–associated genes obtained from VEGAS. Results for LS and non-LS at nominal <em>P</em> less than 0.05 are provided in Tables E7 and E8, respectively.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl5"><h3>Table 5.</h3><!--caption a7--><div class="caption"><p id="__p52">VEGAS Results for the 15 Most Significant Genes from Immunochip Analysis on LS and non-LS (48,740 SNPs and 6,840 Genes)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913865791728" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">CHR</th><th rowspan="1" colspan="1" align="center">Gene</th><th rowspan="1" colspan="1" align="center">nSNPs</th><th rowspan="1" colspan="1" align="center">Start Position</th><th rowspan="1" colspan="1" align="center">Stop Position</th><th rowspan="1" colspan="1" align="center">Test Statistic</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value</th><th rowspan="1" colspan="1" align="center">Best SNP</th><th rowspan="1" colspan="1" align="center">SNP <em>P</em> Value</th></tr></thead><tbody><tr><td colspan="9" rowspan="1" align="left">LS vs. HC<hr></td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>TNXB</em></td><td rowspan="1" colspan="1" align="center">50</td><td rowspan="1" colspan="1" align="center">32116910</td><td rowspan="1" colspan="1" align="center">32185129</td><td rowspan="1" colspan="1" align="center">1206.77</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3130288</td><td rowspan="1" colspan="1" align="center">6.47E-34</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>CREBL1</em></td><td rowspan="1" colspan="1" align="center">51</td><td rowspan="1" colspan="1" align="center">32191022</td><td rowspan="1" colspan="1" align="center">32203995</td><td rowspan="1" colspan="1" align="center">1409.21</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3130288</td><td rowspan="1" colspan="1" align="center">6.47E-34</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>FKBPL</em></td><td rowspan="1" colspan="1" align="center">49</td><td rowspan="1" colspan="1" align="center">32204461</td><td rowspan="1" colspan="1" align="center">32206045</td><td rowspan="1" colspan="1" align="center">1464.31</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3130288</td><td rowspan="1" colspan="1" align="center">6.47E-34</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>PRRT1</em></td><td rowspan="1" colspan="1" align="center">61</td><td rowspan="1" colspan="1" align="center">32224117</td><td rowspan="1" colspan="1" align="center">32227698</td><td rowspan="1" colspan="1" align="center">2290.62</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3130288</td><td rowspan="1" colspan="1" align="center">6.47E-34</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>PPT2</em></td><td rowspan="1" colspan="1" align="center">68</td><td rowspan="1" colspan="1" align="center">32229278</td><td rowspan="1" colspan="1" align="center">32239430</td><td rowspan="1" colspan="1" align="center">2779.84</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3130288</td><td rowspan="1" colspan="1" align="center">6.47E-34</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>EGFL8</em></td><td rowspan="1" colspan="1" align="center">69</td><td rowspan="1" colspan="1" align="center">32240382</td><td rowspan="1" colspan="1" align="center">32244040</td><td rowspan="1" colspan="1" align="center">2728.04</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3130288</td><td rowspan="1" colspan="1" align="center">6.47E-34</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>AGPAT1</em></td><td rowspan="1" colspan="1" align="center">84</td><td rowspan="1" colspan="1" align="center">32243966</td><td rowspan="1" colspan="1" align="center">32253820</td><td rowspan="1" colspan="1" align="center">3085.42</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3130288</td><td rowspan="1" colspan="1" align="center">6.47E-34</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>C6orf10</em></td><td rowspan="1" colspan="1" align="center">149</td><td rowspan="1" colspan="1" align="center">32368452</td><td rowspan="1" colspan="1" align="center">32447634</td><td rowspan="1" colspan="1" align="center">5678.59</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3129927</td><td rowspan="1" colspan="1" align="center">1.82E-33</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="center">109</td><td rowspan="1" colspan="1" align="center">32470490</td><td rowspan="1" colspan="1" align="center">32482878</td><td rowspan="1" colspan="1" align="center">3946.51</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3129927</td><td rowspan="1" colspan="1" align="center">1.82E-33</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRB1</em></td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">32654524</td><td rowspan="1" colspan="1" align="center">32665540</td><td rowspan="1" colspan="1" align="center">144.87</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs2187668</td><td rowspan="1" colspan="1" align="center">2.79E-33</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>HLA-DQA1</em></td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">32713160</td><td rowspan="1" colspan="1" align="center">32719407</td><td rowspan="1" colspan="1" align="center">288.31</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs2187668</td><td rowspan="1" colspan="1" align="center">2.79E-33</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>HLA-DQB1</em></td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">32735634</td><td rowspan="1" colspan="1" align="center">32742444</td><td rowspan="1" colspan="1" align="center">288.31</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs2187668</td><td rowspan="1" colspan="1" align="center">2.79E-33</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="center">61</td><td rowspan="1" colspan="1" align="center">32515624</td><td rowspan="1" colspan="1" align="center">32520802</td><td rowspan="1" colspan="1" align="center">2278.93</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3135394</td><td rowspan="1" colspan="1" align="center">1.41E-32</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>C2</em></td><td rowspan="1" colspan="1" align="center">50</td><td rowspan="1" colspan="1" align="center">32003472</td><td rowspan="1" colspan="1" align="center">32021427</td><td rowspan="1" colspan="1" align="center">1027.18</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs389884</td><td rowspan="1" colspan="1" align="center">3.84E-32</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>CFB</em></td><td rowspan="1" colspan="1" align="center">45</td><td rowspan="1" colspan="1" align="center">32021699</td><td rowspan="1" colspan="1" align="center">32027840</td><td rowspan="1" colspan="1" align="center">831.90</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs389884</td><td rowspan="1" colspan="1" align="center">3.84E-32</td></tr><tr><td colspan="9" rowspan="1" align="left">Non-LS vs. HC<hr></td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="center">61</td><td rowspan="1" colspan="1" align="center">32515624</td><td rowspan="1" colspan="1" align="center">32520802</td><td rowspan="1" colspan="1" align="center">671.61</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3129882</td><td rowspan="1" colspan="1" align="center">5.33E-10</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="center">109</td><td rowspan="1" colspan="1" align="center">32470490</td><td rowspan="1" colspan="1" align="center">32482878</td><td rowspan="1" colspan="1" align="center">935.64</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3129882</td><td rowspan="1" colspan="1" align="center">5.33E-10</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>HLA-DRB1</em></td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">32654524</td><td rowspan="1" colspan="1" align="center">32665540</td><td rowspan="1" colspan="1" align="center">36.76</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs3830135</td><td rowspan="1" colspan="1" align="center">2.74E-08</td></tr><tr><td rowspan="1" colspan="1" align="left"> 10</td><td rowspan="1" colspan="1" align="center"><em>ANXA11</em></td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">81904859</td><td rowspan="1" colspan="1" align="center">81955308</td><td rowspan="1" colspan="1" align="center">82.55</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs1049550</td><td rowspan="1" colspan="1" align="center">1.07E-07</td></tr><tr><td rowspan="1" colspan="1" align="left"> 10</td><td rowspan="1" colspan="1" align="center"><em>PLAC9</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">81882237</td><td rowspan="1" colspan="1" align="center">81894764</td><td rowspan="1" colspan="1" align="center">60.30</td><td rowspan="1" colspan="1" align="center">1.00E-06</td><td rowspan="1" colspan="1" align="center">rs1049550</td><td rowspan="1" colspan="1" align="center">1.07E-07</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>HLA-DPB1</em></td><td rowspan="1" colspan="1" align="center">58</td><td rowspan="1" colspan="1" align="center">33151737</td><td rowspan="1" colspan="1" align="center">33162954</td><td rowspan="1" colspan="1" align="center">363.18</td><td rowspan="1" colspan="1" align="center">9.40E-05</td><td rowspan="1" colspan="1" align="center">rs9277357</td><td rowspan="1" colspan="1" align="center">1.61E-05</td></tr><tr><td rowspan="1" colspan="1" align="left"> 18</td><td rowspan="1" colspan="1" align="center"><em>PTPN2</em></td><td rowspan="1" colspan="1" align="center">54</td><td rowspan="1" colspan="1" align="center">12775479</td><td rowspan="1" colspan="1" align="center">12874334</td><td rowspan="1" colspan="1" align="center">317.12</td><td rowspan="1" colspan="1" align="center">9.70E-05</td><td rowspan="1" colspan="1" align="center">rs2852151</td><td rowspan="1" colspan="1" align="center">1.55E-04</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>HLA-DPA1</em></td><td rowspan="1" colspan="1" align="center">46</td><td rowspan="1" colspan="1" align="center">33140771</td><td rowspan="1" colspan="1" align="center">33149356</td><td rowspan="1" colspan="1" align="center">326.14</td><td rowspan="1" colspan="1" align="center">1.28E-04</td><td rowspan="1" colspan="1" align="center">rs9277357</td><td rowspan="1" colspan="1" align="center">1.61E-05</td></tr><tr><td rowspan="1" colspan="1" align="left"> 12</td><td rowspan="1" colspan="1" align="center"><em>PAH</em></td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">1.02E+08</td><td rowspan="1" colspan="1" align="center">1.02E+08</td><td rowspan="1" colspan="1" align="center">18.94</td><td rowspan="1" colspan="1" align="center">1.85E-04</td><td rowspan="1" colspan="1" align="center">rs2242381</td><td rowspan="1" colspan="1" align="center">1.10E-04</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>LST1</em></td><td rowspan="1" colspan="1" align="center">61</td><td rowspan="1" colspan="1" align="center">31661949</td><td rowspan="1" colspan="1" align="center">31664665</td><td rowspan="1" colspan="1" align="center">249.78</td><td rowspan="1" colspan="1" align="center">2.11E-04</td><td rowspan="1" colspan="1" align="center">rs2239704</td><td rowspan="1" colspan="1" align="center">1.53E-05</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>TNF</em></td><td rowspan="1" colspan="1" align="center">60</td><td rowspan="1" colspan="1" align="center">31651328</td><td rowspan="1" colspan="1" align="center">31654091</td><td rowspan="1" colspan="1" align="center">297.74</td><td rowspan="1" colspan="1" align="center">2.92E-04</td><td rowspan="1" colspan="1" align="center">rs2239704</td><td rowspan="1" colspan="1" align="center">1.53E-05</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>LTB</em></td><td rowspan="1" colspan="1" align="center">61</td><td rowspan="1" colspan="1" align="center">31656314</td><td rowspan="1" colspan="1" align="center">31658181</td><td rowspan="1" colspan="1" align="center">269.43</td><td rowspan="1" colspan="1" align="center">3.19E-04</td><td rowspan="1" colspan="1" align="center">rs2239704</td><td rowspan="1" colspan="1" align="center">1.53E-05</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>NCR3</em></td><td rowspan="1" colspan="1" align="center">60</td><td rowspan="1" colspan="1" align="center">31664650</td><td rowspan="1" colspan="1" align="center">31668741</td><td rowspan="1" colspan="1" align="center">225.25</td><td rowspan="1" colspan="1" align="center">3.71E-04</td><td rowspan="1" colspan="1" align="center">rs2239704</td><td rowspan="1" colspan="1" align="center">1.53E-05</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center"><em>PSMB9</em></td><td rowspan="1" colspan="1" align="center">87</td><td rowspan="1" colspan="1" align="center">32929915</td><td rowspan="1" colspan="1" align="center">32935606</td><td rowspan="1" colspan="1" align="center">357.37</td><td rowspan="1" colspan="1" align="center">4.05E-04</td><td rowspan="1" colspan="1" align="center">rs3819717</td><td rowspan="1" colspan="1" align="center">2.43E-06</td></tr><tr><td rowspan="1" colspan="1" align="left"> 6</td><td rowspan="1" colspan="1" align="center">PSMB8</td><td rowspan="1" colspan="1" align="center">87</td><td rowspan="1" colspan="1" align="center">32916471</td><td rowspan="1" colspan="1" align="center">32920690</td><td rowspan="1" colspan="1" align="center">357.37</td><td rowspan="1" colspan="1" align="center">4.25E-04</td><td rowspan="1" colspan="1" align="center">rs3819717</td><td rowspan="1" colspan="1" align="center">2.43E-06</td></tr></tbody></table></div><div id="largeobj_idm139913865791728" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4872654/table/tbl5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139913868782320"><p id="__p53" class="p p-first-last"><em>Definition of abbreviations</em>: CHR = chromosome; HC = healthy control subjects; LS = Löfgren’s syndrome; nSNPs = number of SNPs included within a gene; SNP = single-nucleotide polymorphism; VEGAS = Versatile Gene-based Association Study.</p></div><div id="idm139913868780768"><p id="__p54" class="p p-first-last">The entire lists of LS and non-LS genic SNPs at nominal <em>P</em> &lt; 0.05 are provided in Tables E7 and E8, respectively.</p></div></div></div></div><div id="__sec21" class="sec sec-last"><h3 id="__sec21title">Integrative Analyses on Significant Findings</h3><div id="__sec22" class="sec sec-first"><p></p><h4 id="__sec22title" class="inline">Bioinformatic analysis for refinement of associated variants </h4><p id="__p55" class="p p-first">Results from studying the potential influence of confirmed variants on gene expression levels in various tissues and cell types revealed 564 of 710 LS-associated SNPs, 51 of 68 non-LS–associated SNPs, and 16 of 17 mutually associated SNPs as <em>cis-</em>eQTL SNPs (<a href="/pmc/articles/PMC4872654/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Figure 5</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC4872654/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913868766144" class="figure"><a href="/pmc/articles/PMC4872654/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4872654_rccm.201507-1372OC_f5.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is rccm.201507-1372OC_f5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4872654/bin/rccm.201507-1372OC_f5.jpg"></a></div><div id="largeobj_idm139913868766144" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4872654/figure/fig5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC4872654/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Figure 5.</a></div><!--caption a7--><div class="caption"><p id="__p58">Distribution of <em>cis-</em>expression quantitative trait loci (eQTL) and regulatory single-nucleotide polymorphisms (SNPs) among associated confirmed variants of Löfgren’s syndrome (LS), non-LS, and both. Percentages are derived from the total number in each group, respectively. Genes enclosed in <em>parentheses</em> denote intergenic regions. HC = healthy control subjects; MHC = major histocompatibility complex; TF = transcription factor.</p></div></div></div><p id="__p56">Advanced query on unclassified SNPs using extended LD information uncovered 105 LS-associated SNPs and 12 non-LS–associated SNPs positioned near a <em>cis-</em>eQTL SNP located on the same haplotype and in high LD (<em>R</em><sup>2</sup> &gt; 0.8). Moreover, 41 LS-associated SNPs and three non-LS–associated SNPs were found with no <em>cis-</em>eQTL mapping.</p><p id="__p57" class="p">Results from assessing confirmed variants in elements of transcription regulation using datasets from ENCODE and the National Institutes of Health Roadmap Epigenomics Mapping Consortium identified 675 LS-associated SNPs, 49 non-LS–associated SNPs, and all 17 mutually SNPs as regulatory SNPs. A distribution diagram of confirmed variants by transcription-element classification is illustrated in <a href="/pmc/articles/PMC4872654/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Figure 5</span></a>.</p></div><div id="__sec23" class="sec sec-last"><p></p><h4 id="__sec23title" class="inline">Enrichment and pathway analyses of associated variants </h4><p id="__p59" class="p p-first">Results from enrichment analysis based on 727 LS-associated confirmed variants at false discovery rate less than 0.05 identified pathway maps related to immune response of T-cell regulation and processes, and cytoskeleton remodeling. Pooling together 727 LS-associated SNPs and differentially expressed genes (from each GSE dataset tested) yielded similar pathway maps (<em>see</em> Table E9). In non-LS, assessment based on the 68 confirmed variants yielded no pathway maps. However, when combined with differentially expressed genes, identification of pathway maps related to immune response, differentiation, and regulation of T cells (<em>see</em> Table E10) was detected.</p><p id="__p60" class="p p-last">Noteworthy, many of the pathway maps that were identified in LS and non-LS, respectively, were overlapping by sharing genes with other pathways, and in some cases were subsets of each other. For example, in LS, the pathway maps labeled as a, b, c, d, f, g, i, and k (<em>see</em> Table E9) shared three common components: MHC class II, nuclear factor-κB, and CD86. Likewise, in non-LS, pathway maps labeled 1, 2, 4, 6, 8, 11, 13, and 14 (<em>see</em> Table E10) shared one common component: PCL-gamma 1. Further analysis on enrichment results from both phenotypes pinpointed the differences and shared pathway maps among these. Explicitly, pathway maps b-c-e-g-h (<em>see</em> Table E9) were unique to LS; pathway maps 1-2-3-4-6-7-8-9-11-12-13-14 (<em>see</em> Table E10) were unique to non-LS; and pathway maps a-10, c-17, d-5, i-16, k-15 were common to both. In the latter, the nuclear factor-κB was unanimously present across the shared pathway maps.</p></div></div></div><div id="__sec24" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec24title">Discussion</h2><p id="__p61" class="p p-first">In this study, we report for the first time results of Immunochip analysis of LS and non-LS segregated as two different sarcoidosis entities in well-phenotyped cohorts with replicated and meta-analysis results. In this work, we aimed to investigate the genetic contributions to LS and non-LS by using the custom Illumina Infinium array that leveraged the remarkable genetic overlap of 186 loci associated with autoimmunity. Results from various approaches in LS (using populations of European ancestry) and non-LS (using populations of European and African ancestry) show that genetic susceptibility of these phenotypes is different and that only a small number of loci are shared. Thus, we were able to capture genetic differences between LS and non-LS.</p><p id="__p62">LS-associated variants expand through the extended MHC gene map (<a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102659">31</a>), ranging from <em>ZNF184</em> to <em>HLA-DPB2</em>. Outside the extended MHC region, variants in <em>ADCY3</em>, <em>LRR16A</em>, and between <em>CSMD1</em> and <em>MCPH1</em> shall be further studied to explain their association, which may be linked to eQTL, as elucidated by <em>cis-</em>eQTL SNPs in gene expression of immunity cells (<a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102634">18</a>, <a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102644">21</a>, <a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102654">26</a>, <a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102647">32</a>). The susceptibility of non-LS localized to a smaller region of the MHC class II, ranging from <em>C6orf10</em> to <em>HLA-DQA2</em>. Noteworthy, outside this region, confirmed variants highlighted exclusive associations with <em>PLAC9</em> and <em>ANXA11 (</em>a previously discovered sarcoidosis-associated locus) on chromosome 10. An observed overlap between LS and non-LS defined by 17 variants that localized to MHC class II region is worth further exploration. Our efforts to gain insights on the role of <em>HLA-DRB1</em> alleles in non-LS and LS phenotypes reinforced these findings (results and further discussion are available in the N<span class="small-caps">ote</span> in the online supplement). Likewise, results from a genome-wide gene-centric approach underlined the significance of LS and non-LS identified gene regions, which may well establish the groundwork for investigating these phenotypes independently.</p><p id="__p63">Individual assessment of each gene variant based on bioinformatic exploration provides an extensive knowledge for supporting variant association by highlighting potential regulatory and epigenetic mechanisms ensuing within the associated regions.</p><p id="__p64">As reported from many genome-wide association studies (<a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102640">33</a>), common genetic variants can only explain a relatively small portion of the phenotype variability (explained variance in LS, 4.1–4.7%; non-LS, 1.28–2.13%) and heritability thus stressing the plausibility for involvement of other genetic and regulatory elements to account for the portion of phenotypic variance. In non-LS, the results showed a presence of non-HLA genes, thus raising the possibility for <em>gene x factor</em> (a factor is defined as a gene, regulatory/epigenetic element, or protein) interactions across chromosomes, resembling a polygenic phenotype. Thus if so, the phenotype variability may be more likely explained by genes than rather by phenotype heterogeneity within non-LS (i.e., ocular, skin, heart, neurologic sarcoidosis, and many more), as we have demonstrated. Whereas in LS, the results show a higher presence of HLA-genes, suggesting that the phenotype variability may be explained by HLA-genes and <em>gene x factor</em> interactions located within one chromosome.</p><p id="__p65">The presence of eQTL SNPs, regulatory elements, and epigenetic factors in the genomic regions associated with LS and non-LS, prompt new research questions that urge to be undertaken. eQTL genes have been proposed to act as master <em>cis</em>- and <em>trans-</em>regulators (<a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102648">34</a>), whereas epigenetic elements alter chromatin structure and influence gene expression imposing specific heritable patterns on progeny cells (<a href="#bib35" rid="bib35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102639">35</a>–<a href="#bib39" rid="bib39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102655">39</a>). Based on this knowledge, it is not surprising to observe a predominant presence of <em>cis-</em>eQTL SNPs among the genetics of LS and non-LS. Analogously, the presence of epigenetic factors in the associated regions is expected given the numerous essential interactions between HLA and non-HLA genes. Hence, the observed implications and differences in eQTLs and epigenetics events among LS and non-LS support our hypothesis that mechanisms underlying the genetic structure and pathogenesis of these phenotypes are distinct. The limitation of this bioinformatic analysis is the lack of expression and epigenetic data available from our datasets, which could facilitate an advanced understanding of biologic implications for the genomic-associated regions.</p><p id="__p66">With the present collective information comprising associated variants and differently expressed genes in sarcoidosis, we are able to present a better understanding of the functional mechanisms that may shed light on the pathogenesis of LS and non-LS, and on the overlap between these phenotypes. The molecular similarities between both phenotypes may be elucidated by potential sharing mechanisms that were captured on pathway maps that identified several overlapping pathways underlining immune responses. For example, pathways maps highlighting immune response to naive CD4<sup>+</sup> T, Th1, and Th2 cells shall be further studied to understand their correlation with these phenotypes. Particularly, in LS, pathway maps involving Th9, Th17, and Th22 cells and dendritic cells shall be favorably considered for investigation. Given that, significant involvement of Th17 cells determined by higher levels of IL-17 in bronchoalveolar lavage fluid of patients with LS has been reported (<a href="#bib40" rid="bib40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_534102638">40</a>). Naturally, to facilitate such investigations additional omics studies, such as transcriptomics and epigenomics, are necessary before functional studies take place.</p><p id="__p67" class="p p-last">In summary, we present a comprehensive genetic study of LS and non-LS phenotypes of sarcoidosis in populations of European and African ancestry, thus focusing mainly on the genetic structure of each phenotype. For the first time, the genetic architecture of LS and non-LS is analyzed and presented, independently, identifying 175 loci associated with LS and 11 loci associated with non-LS by replication in independent cohorts. Moreover, out of the total number of loci identified, five are common to both phenotypes. We provide evidence to sustain our claim, that LS and non-LS are two different phenotypes with a shared genetic overlap and a reasonable number of specific factors. Through various analyses, we demonstrated that LS and non-LS have different genetic susceptibility and thus different genomic distributions that comprise distinctive gene regulatory elements associated with various relevant pathway maps of immune responses.</p></div><div id="idm139913871559216" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139913871559216title">Acknowledgment</h2><div class="sec"><div id="__sec25" class="sec sec-first"><h3 id="__sec25title"></h3><p id="__p68" class="p p-first-last">The authors thank the EIRA group for information regarding control subjects in the Swedish cohort. They thank Dr. F. Mrazek (University Hospital Olomouc) for assistance in providing samples of control subjects in the Czech Republic cohort. They thank Professor Wolfgang Lieb (PopGen), Professor Carsten Büning, and Professor Stephan Brand for providing German control samples.</p></div></div></div><div id="idm139913871557408" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139913871557408title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm139913871557280"></p><p id="__p69" class="p p-first-last">The Swedish cohort was funded by Swedish Heart-Lung Foundation, by the Swedish Medical Research Council, and through the regional agreement on medical training and clinical research between Stockholm County Council and the Karolinska Institutet. The Czech Republic cohort was funded by grant IGA_LF_2015_020 (M.P.). The German study was supported by the Federal Ministry for Education and Research in Germany through the National Genome Research Network, within the framework of the e:Med (sysINFLAME) research and funding concept (01ZX1306A), the Genome Research Network “MooDS,” and the PopGen 2.0 network (01EY1103); by the German Research Foundation through grants FI 1935/1-1, MU 692/8-1, and BR 1912/6-1; through the Clusters of Excellence “Inflammation at Interfaces” and “ImmunoSensation”; by the Heinz Nixdorf Foundation; and by the Else-Kröner-Fresenius-Stiftung (Else Kröner-Exzellenzstipendium 2010_EKES.32). The African American study was supported by National Institutes of Health grant P30GM110766.</p><p></p><p class="fn sec" id="idm139913871555504"></p><p id="__p70" class="p p-first-last">Author Contributions: Substantial contribution to design of work and clinical/genetic data for replication and meta-analysis: K.S., A. Franke, M.M.N., J.M.-Q., S.S., A. Fischer, C.H.M.v.M., B.K., J.C.G., Z.N., V.K., M.P., I.A., B.A.R., M.C.I., and C.M. Drafting the work and/or revising it critically for intellectual content: N.V.R., K.S., M.R., A.E., L.P., J.G., A. Fischer, M.P., and C.H.M.v.M. Accountability for all aspects of the work in ensuring that questions related to the accuracy/integrity of any part of the work are appropriately investigated and resolved: N.V.R., L.P., and J.G. All authors have given final approval of the version submitted for publication.</p><p></p><p class="fn sec" id="idm139913871554128"></p><p id="__p71" class="p p-first-last">This article has an online supplement, which is accessible from this issue's table of contents at <a href="http://www.atsjournals.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.atsjournals.org</a></p><p></p><p class="fn sec" id="idm139913871552560"></p><p id="__p72" class="p p-first-last">Originally Published in Press as DOI: <a href="http://dx.doi.org/10.1164/rccm.201507-1372OC" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">10.1164/rccm.201507-1372OC</a> on December 10, 2015</p><p></p><p class="fn sec" id="idm139913871550976"></p><p id="__p73" class="p p-first-last"><span class="underline"><strong><a href="http://www.atsjournals.org/doi/suppl/10.1164/rccm.201507-1372OC/suppl_file/disclosures.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Author disclosures</a></strong></span> are available with the text of this article at <a href="http://www.atsjournals.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.atsjournals.org</a>.</p><p></p></div></div><div id="idm139913871547984" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139913871547984title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1.  <span class="element-citation">Iannuzzi MC, Rybicki BA, Teirstein AS. Sarcoidosis. <span><span class="ref-journal"><em>N Engl J Med</em>. </span>2007;<span class="ref-vol">357</span>:2153–2165.</span> [<a href="/pubmed/18032765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Sarcoidosis&amp;author=MC+Iannuzzi&amp;author=BA+Rybicki&amp;author=AS+Teirstein&amp;volume=357&amp;publication_year=2007&amp;pages=2153-2165&amp;pmid=18032765&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2.  <span class="element-citation">Bauer HJ, Löfgren S. International study of pulmonary sarcoidosis in mass chest radiography. <span><span class="ref-journal"><em>Acta Med Scand Suppl</em>. </span>1964;<span class="ref-vol">425</span>:103–105.</span> [<a href="/pubmed/5884450" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Med+Scand+Suppl&amp;title=International+study+of+pulmonary+sarcoidosis+in+mass+chest+radiography&amp;author=HJ+Bauer&amp;author=S+Löfgren&amp;volume=425&amp;publication_year=1964&amp;pages=103-105&amp;pmid=5884450&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3.  <span class="element-citation">Grunewald J, Eklund A. Löfgren’s syndrome: human leukocyte antigen strongly influences the disease course. <span><span class="ref-journal"><em>Am J Respir Crit Care Med</em>. </span>2009;<span class="ref-vol">179</span>:307–312.</span> [<a href="/pubmed/18996998" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Respir+Crit+Care+Med&amp;title=Löfgren’s+syndrome:+human+leukocyte+antigen+strongly+influences+the+disease+course&amp;author=J+Grunewald&amp;author=A+Eklund&amp;volume=179&amp;publication_year=2009&amp;pages=307-312&amp;pmid=18996998&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4.  <span class="element-citation">De Vries J, Van Heck GL, Drent M. Gender differences in sarcoidosis: symptoms, quality of life, and medical consumption. <span><span class="ref-journal"><em>Women Health</em>. </span>1999;<span class="ref-vol">30</span>:99–114.</span> [<a href="/pubmed/10881761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Women+Health&amp;title=Gender+differences+in+sarcoidosis:+symptoms,+quality+of+life,+and+medical+consumption&amp;author=J+De+Vries&amp;author=GL+Van+Heck&amp;author=M+Drent&amp;volume=30&amp;publication_year=1999&amp;pages=99-114&amp;pmid=10881761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5.  <span class="element-citation">Drent M, Marcellis R, Lenssen A, De Vries J. Association between physical functions and quality of life in sarcoidosis. <span><span class="ref-journal"><em>Sarcoidosis</em>. </span>2014;<span class="ref-vol">31</span>:117–128.</span> [<a href="/pubmed/25078639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sarcoidosis&amp;title=Association+between+physical+functions+and+quality+of+life+in+sarcoidosis&amp;author=M+Drent&amp;author=R+Marcellis&amp;author=A+Lenssen&amp;author=J+De+Vries&amp;volume=31&amp;publication_year=2014&amp;pages=117-128&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6.  <span class="element-citation">Belfer MH, Stevens RW. Sarcoidosis: a primary care review. <span><span class="ref-journal"><em>Am Fam Physician</em>. </span>1998;<span class="ref-vol">58</span>:2041–2050, 55–6.</span> [<a href="/pubmed/9861878" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+Fam+Physician&amp;title=Sarcoidosis:+a+primary+care+review&amp;author=MH+Belfer&amp;author=RW+Stevens&amp;volume=58&amp;publication_year=1998&amp;pages=2041-2050,+55–6&amp;pmid=9861878&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7.  <span class="element-citation">Bauer HJ, Löfgren S. International study of pulmonary sarcoidosis in mass chest radiography. <span><span class="ref-journal"><em>Acta Med Scand Suppl</em>. </span>1964;<span class="ref-vol">425</span>:103–105.</span> [<a href="/pubmed/5884450" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Med+Scand+Suppl&amp;title=International+study+of+pulmonary+sarcoidosis+in+mass+chest+radiography&amp;author=HJ+Bauer&amp;author=S+Löfgren&amp;volume=425&amp;publication_year=1964&amp;pages=103-105&amp;pmid=5884450&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8.  <span class="element-citation">Trynka G, Hunt KA, Bockett NA, Romanos J, Mistry V, Szperl A, Bakker SF, Bardella MT, Bhaw-Rosun L, Castillejo G, et al.  Spanish Consortium on the Genetics of Coeliac Disease (CEGEC); PreventCD Study Group; Wellcome Trust Case Control Consortium (WTCCC) Dense genotyping identifies and localizes multiple common and rare variant association signals in celiac disease. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2011;<span class="ref-vol">43</span>:1193–1201.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3242065/">PMC free article</a>]</span> [<a href="/pubmed/22057235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Dense+genotyping+identifies+and+localizes+multiple+common+and+rare+variant+association+signals+in+celiac+disease&amp;author=G+Trynka&amp;author=KA+Hunt&amp;author=NA+Bockett&amp;author=J+Romanos&amp;author=V+Mistry&amp;volume=43&amp;publication_year=2011&amp;pages=1193-1201&amp;pmid=22057235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9.  <span class="element-citation">Eyre S, Bowes J, Diogo D, Lee A, Barton A, Martin P, Zhernakova A, Stahl E, Viatte S, McAllister K, et al.  Biologics in Rheumatoid Arthritis Genetics and Genomics Study Syndicate; Wellcome Trust Case Control Consortium. High-density genetic mapping identifies new susceptibility loci for rheumatoid arthritis. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2012;<span class="ref-vol">44</span>:1336–1340.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3605761/">PMC free article</a>]</span> [<a href="/pubmed/23143596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=High-density+genetic+mapping+identifies+new+susceptibility+loci+for+rheumatoid+arthritis&amp;author=S+Eyre&amp;author=J+Bowes&amp;author=D+Diogo&amp;author=A+Lee&amp;author=A+Barton&amp;volume=44&amp;publication_year=2012&amp;pages=1336-1340&amp;pmid=23143596&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10.  <span class="element-citation">Tsoi LC, Spain SL, Knight J, Ellinghaus E, Stuart PE, Capon F, Ding J, Li Y, Tejasvi T, Gudjonsson JE, et al.  Collaborative Association Study of Psoriasis (CASP); Genetic Analysis of Psoriasis Consortium; Psoriasis Association Genetics Extension; Wellcome Trust Case Control Consortium 2. Identification of 15 new psoriasis susceptibility loci highlights the role of innate immunity. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2012;<span class="ref-vol">44</span>:1341–1348.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3510312/">PMC free article</a>]</span> [<a href="/pubmed/23143594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Identification+of+15+new+psoriasis+susceptibility+loci+highlights+the+role+of+innate+immunity&amp;author=LC+Tsoi&amp;author=SL+Spain&amp;author=J+Knight&amp;author=E+Ellinghaus&amp;author=PE+Stuart&amp;volume=44&amp;publication_year=2012&amp;pages=1341-1348&amp;pmid=23143594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11.  <span class="element-citation">Beecham AH, Patsopoulos NA, Xifara DK, Davis MF, Kemppinen A, Cotsapas C, Shah TS, Spencer C, Booth D, Goris A, et al.  International Multiple Sclerosis Genetics Consortium (IMSGC); Wellcome Trust Case Control Consortium 2 (WTCCC2); International IBD Genetics Consortium (IIBDGC) Analysis of immune-related loci identifies 48 new susceptibility variants for multiple sclerosis. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2013;<span class="ref-vol">45</span>:1353–1360.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3832895/">PMC free article</a>]</span> [<a href="/pubmed/24076602" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Analysis+of+immune-related+loci+identifies+48+new+susceptibility+variants+for+multiple+sclerosis&amp;author=AH+Beecham&amp;author=NA+Patsopoulos&amp;author=DK+Xifara&amp;author=MF+Davis&amp;author=A+Kemppinen&amp;volume=45&amp;publication_year=2013&amp;pages=1353-1360&amp;pmid=24076602&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12.  <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal"><em>Am J Hum Genet</em>. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13.  <span class="element-citation">Ellinghaus D, Baurecht H, Esparza-Gordillo J, Rodríguez E, Matanovic A, Marenholz I, Hübner N, Schaarschmidt H, Novak N, Michel S, et al.  High-density genotyping study identifies four new susceptibility loci for atopic dermatitis. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2013;<span class="ref-vol">45</span>:808–812.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3797441/">PMC free article</a>]</span> [<a href="/pubmed/23727859" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=High-density+genotyping+study+identifies+four+new+susceptibility+loci+for+atopic+dermatitis&amp;author=D+Ellinghaus&amp;author=H+Baurecht&amp;author=J+Esparza-Gordillo&amp;author=E+Rodríguez&amp;author=A+Matanovic&amp;volume=45&amp;publication_year=2013&amp;pages=808-812&amp;pmid=23727859&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14.  <span class="mixed-citation">Viechtbauer W. <span class="ref-title">Conducting meta-analyses in R with the Metafor package </span><span class="ref-journal"><em>J Stat Softw</em></span> 2010<span class="ref-vol">36</span>3:1–48 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Stat+Softw&amp;title=Conducting+meta-analyses+in+R+with+the+Metafor+package&amp;author=W.+Viechtbauer&amp;volume=36&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15.  <span class="element-citation">Liu JZ, McRae AF, Nyholt DR, Medland SE, Wray NR, Brown KM, Hayward NK, Montgomery GW, Visscher PM, Martin NG, et al.  AMFS Investigators. A versatile gene-based test for genome-wide association studies. <span><span class="ref-journal"><em>Am J Hum Genet</em>. </span>2010;<span class="ref-vol">87</span>:139–145.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896770/">PMC free article</a>]</span> [<a href="/pubmed/20598278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+versatile+gene-based+test+for+genome-wide+association+studies&amp;author=JZ+Liu&amp;author=AF+McRae&amp;author=DR+Nyholt&amp;author=SE+Medland&amp;author=NR+Wray&amp;volume=87&amp;publication_year=2010&amp;pages=139-145&amp;pmid=20598278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16.  <span class="element-citation">Westra HJ, Peters MJ, Esko T, Yaghootkar H, Schurmann C, Kettunen J, Christiansen MW, Fairfax BP, Schramm K, Powell JE, et al.  Systematic identification of trans eQTLs as putative drivers of known disease associations. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2013;<span class="ref-vol">45</span>:1238–1243.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3991562/">PMC free article</a>]</span> [<a href="/pubmed/24013639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Systematic+identification+of+trans+eQTLs+as+putative+drivers+of+known+disease+associations&amp;author=HJ+Westra&amp;author=MJ+Peters&amp;author=T+Esko&amp;author=H+Yaghootkar&amp;author=C+Schurmann&amp;volume=45&amp;publication_year=2013&amp;pages=1238-1243&amp;pmid=24013639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17.  <span class="element-citation">Dimas AS, Deutsch S, Stranger BE, Montgomery SB, Borel C, Attar-Cohen H, Ingle C, Beazley C, Gutierrez Arcelus M, Sekowska M, et al.  Common regulatory variation impacts gene expression in a cell type-dependent manner. <span><span class="ref-journal"><em>Science</em>. </span>2009;<span class="ref-vol">325</span>:1246–1250.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2867218/">PMC free article</a>]</span> [<a href="/pubmed/19644074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Common+regulatory+variation+impacts+gene+expression+in+a+cell+type-dependent+manner&amp;author=AS+Dimas&amp;author=S+Deutsch&amp;author=BE+Stranger&amp;author=SB+Montgomery&amp;author=C+Borel&amp;volume=325&amp;publication_year=2009&amp;pages=1246-1250&amp;pmid=19644074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18.  <span class="element-citation">Dixon AL, Liang L, Moffatt MF, Chen W, Heath S, Wong KC, Taylor J, Burnett E, Gut I, Farrall M, et al.  A genome-wide association study of global gene expression. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2007;<span class="ref-vol">39</span>:1202–1207.</span> [<a href="/pubmed/17873877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+of+global+gene+expression&amp;author=AL+Dixon&amp;author=L+Liang&amp;author=MF+Moffatt&amp;author=W+Chen&amp;author=S+Heath&amp;volume=39&amp;publication_year=2007&amp;pages=1202-1207&amp;pmid=17873877&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19.  <span class="element-citation">Gaffney DJ, Veyrieras JB, Degner JF, Pique-Regi R, Pai AA, Crawford GE, Stephens M, Gilad Y, Pritchard JK. Dissecting the regulatory architecture of gene expression QTLs. <span><span class="ref-journal"><em>Genome Biol</em>. </span>2012;<span class="ref-vol">13</span>:R7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3334587/">PMC free article</a>]</span> [<a href="/pubmed/22293038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Dissecting+the+regulatory+architecture+of+gene+expression+QTLs&amp;author=DJ+Gaffney&amp;author=JB+Veyrieras&amp;author=JF+Degner&amp;author=R+Pique-Regi&amp;author=AA+Pai&amp;volume=13&amp;publication_year=2012&amp;pages=R7&amp;pmid=22293038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20.  <span class="element-citation">Montgomery SB, Sammeth M, Gutierrez-Arcelus M, Lach RP, Ingle C, Nisbett J, Guigo R, Dermitzakis ET. Transcriptome genetics using second generation sequencing in a caucasian population. <span><span class="ref-journal"><em>Nature</em>. </span>2010;<span class="ref-vol">464</span>:773–777.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3836232/">PMC free article</a>]</span> [<a href="/pubmed/20220756" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Transcriptome+genetics+using+second+generation+sequencing+in+a+caucasian+population&amp;author=SB+Montgomery&amp;author=M+Sammeth&amp;author=M+Gutierrez-Arcelus&amp;author=RP+Lach&amp;author=C+Ingle&amp;volume=464&amp;publication_year=2010&amp;pages=773-777&amp;pmid=20220756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21.  <span class="element-citation">Nica AC, Parts L, Glass D, Nisbet J, Barrett A, Sekowska M, Travers M, Potter S, Grundberg E, Small K, et al.  MuTHER Consortium. The architecture of gene regulatory variation across multiple human tissues: the MuTHER study. <span><span class="ref-journal"><em>PLoS Genet</em>. </span>2011;<span class="ref-vol">7</span>:e1002003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3033383/">PMC free article</a>]</span> [<a href="/pubmed/21304890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=The+architecture+of+gene+regulatory+variation+across+multiple+human+tissues:+the+MuTHER+study&amp;author=AC+Nica&amp;author=L+Parts&amp;author=D+Glass&amp;author=J+Nisbet&amp;author=A+Barrett&amp;volume=7&amp;publication_year=2011&amp;pages=e1002003&amp;pmid=21304890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22.  <span class="element-citation">Pickrell JK, Marioni JC, Pai AA, Degner JF, Engelhardt BE, Nkadori E, Veyrieras JB, Stephens M, Gilad Y, Pritchard JK. Understanding mechanisms underlying human gene expression variation with RNA sequencing. <span><span class="ref-journal"><em>Nature</em>. </span>2010;<span class="ref-vol">464</span>:768–772.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3089435/">PMC free article</a>]</span> [<a href="/pubmed/20220758" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Understanding+mechanisms+underlying+human+gene+expression+variation+with+RNA+sequencing&amp;author=JK+Pickrell&amp;author=JC+Marioni&amp;author=AA+Pai&amp;author=JF+Degner&amp;author=BE+Engelhardt&amp;volume=464&amp;publication_year=2010&amp;pages=768-772&amp;pmid=20220758&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23.  <span class="element-citation">Stranger BE, Nica AC, Forrest MS, Dimas A, Bird CP, Beazley C, Ingle CE, Dunning M, Flicek P, Koller D, et al.  Population genomics of human gene expression. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2007;<span class="ref-vol">39</span>:1217–1224.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683249/">PMC free article</a>]</span> [<a href="/pubmed/17873874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Population+genomics+of+human+gene+expression&amp;author=BE+Stranger&amp;author=AC+Nica&amp;author=MS+Forrest&amp;author=A+Dimas&amp;author=CP+Bird&amp;volume=39&amp;publication_year=2007&amp;pages=1217-1224&amp;pmid=17873874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24.  <span class="element-citation">Veyrieras JB, Kudaravalli S, Kim SY, Dermitzakis ET, Gilad Y, Stephens M, Pritchard JK. High-resolution mapping of expression-QTLs yields insight into human gene regulation. <span><span class="ref-journal"><em>PLoS Genet</em>. </span>2008;<span class="ref-vol">4</span>:e1000214.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2556086/">PMC free article</a>]</span> [<a href="/pubmed/18846210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=High-resolution+mapping+of+expression-QTLs+yields+insight+into+human+gene+regulation&amp;author=JB+Veyrieras&amp;author=S+Kudaravalli&amp;author=SY+Kim&amp;author=ET+Dermitzakis&amp;author=Y+Gilad&amp;volume=4&amp;publication_year=2008&amp;pages=e1000214&amp;pmid=18846210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25.  <span class="element-citation">Zeller T, Wild P, Szymczak S, Rotival M, Schillert A, Castagne R, Maouche S, Germain M, Lackner K, Rossmann H, et al.  Genetics and beyond: the transcriptome of human monocytes and disease susceptibility. <span><span class="ref-journal"><em>PLoS One</em>. </span>2010;<span class="ref-vol">5</span>:e10693.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2872668/">PMC free article</a>]</span> [<a href="/pubmed/20502693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Genetics+and+beyond:+the+transcriptome+of+human+monocytes+and+disease+susceptibility&amp;author=T+Zeller&amp;author=P+Wild&amp;author=S+Szymczak&amp;author=M+Rotival&amp;author=A+Schillert&amp;volume=5&amp;publication_year=2010&amp;pages=e10693&amp;pmid=20502693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26.  <span class="element-citation">Boyle APHE, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, Karczewski KJ, Park J, Hitz BC, Weng S, et al.  Annotation of functional variation in personal genomes using RegulomeDB. <span><span class="ref-journal"><em>Genome Res</em>. </span>2012;<span class="ref-vol">22</span>:1790–1797.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3431494/">PMC free article</a>]</span> [<a href="/pubmed/22955989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Annotation+of+functional+variation+in+personal+genomes+using+RegulomeDB&amp;author=APHE+Boyle&amp;author=EL+Hong&amp;author=M+Hariharan&amp;author=Y+Cheng&amp;author=MA+Schaub&amp;volume=22&amp;publication_year=2012&amp;pages=1790-1797&amp;pmid=22955989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27.  <span class="element-citation">Ward LD, Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. <span><span class="ref-journal"><em>Nucleic Acids Res</em>. </span>2012;<span class="ref-vol">40</span>:D930–D934.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245002/">PMC free article</a>]</span> [<a href="/pubmed/22064851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=HaploReg:+a+resource+for+exploring+chromatin+states,+conservation,+and+regulatory+motif+alterations+within+sets+of+genetically+linked+variants&amp;author=LD+Ward&amp;author=M+Kellis&amp;volume=40&amp;publication_year=2012&amp;pages=D930-D934&amp;pmid=22064851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28.  <span class="element-citation">Sharma SM, Choi D, Planck SR, Harrington CA, Austin CR, Lewis JA, Diebel TN, Martin TM, Smith JR, Rosenbaum JT. Insights in to the pathogenesis of axial spondyloarthropathy based on gene expression profiles. <span><span class="ref-journal"><em>Arthritis Res Ther</em>. </span>2009;<span class="ref-vol">11</span>:R168.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3003511/">PMC free article</a>]</span> [<a href="/pubmed/19900269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Res+Ther&amp;title=Insights+in+to+the+pathogenesis+of+axial+spondyloarthropathy+based+on+gene+expression+profiles&amp;author=SM+Sharma&amp;author=D+Choi&amp;author=SR+Planck&amp;author=CA+Harrington&amp;author=CR+Austin&amp;volume=11&amp;publication_year=2009&amp;pages=R168&amp;pmid=19900269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29.  <span class="element-citation">Koth LL, Solberg OD, Peng JC, Bhakta NR, Nguyen CP, Woodruff PG. Sarcoidosis blood transcriptome reflects lung inflammation and overlaps with tuberculosis. <span><span class="ref-journal"><em>Am J Respir Crit Care Med</em>. </span>2011;<span class="ref-vol">184</span>:1153–1163.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3262024/">PMC free article</a>]</span> [<a href="/pubmed/21852540" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Respir+Crit+Care+Med&amp;title=Sarcoidosis+blood+transcriptome+reflects+lung+inflammation+and+overlaps+with+tuberculosis&amp;author=LL+Koth&amp;author=OD+Solberg&amp;author=JC+Peng&amp;author=NR+Bhakta&amp;author=CP+Nguyen&amp;volume=184&amp;publication_year=2011&amp;pages=1153-1163&amp;pmid=21852540&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30.  <span class="element-citation">Judson MA, Marchell RM, Mascelli M, Piantone A, Barnathan ES, Petty KJ, Chen D, Fan H, Grund H, Ma K, et al.  Molecular profiling and gene expression analysis in cutaneous sarcoidosis: the role of interleukin-12, interleukin-23, and the T-helper 17 pathway. <span><span class="ref-journal"><em>J Am Acad Dermatol</em>. </span>2012;<span class="ref-vol">66</span>:901–910.</span> [<a href="/pubmed/21924794" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Dermatol&amp;title=Molecular+profiling+and+gene+expression+analysis+in+cutaneous+sarcoidosis:+the+role+of+interleukin-12,+interleukin-23,+and+the+T-helper+17+pathway&amp;author=MA+Judson&amp;author=RM+Marchell&amp;author=M+Mascelli&amp;author=A+Piantone&amp;author=ES+Barnathan&amp;volume=66&amp;publication_year=2012&amp;pages=901-910&amp;pmid=21924794&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31.  <span class="element-citation">Horton R, Wilming L, Rand V, Lovering RC, Bruford EA, Khodiyar VK, Lush MJ, Povey S, Talbot CC, Jr, Wright MW, et al.  Gene map of the extended human MHC. <span><span class="ref-journal"><em>Nat Rev Genet</em>. </span>2004;<span class="ref-vol">5</span>:889–899.</span> [<a href="/pubmed/15573121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Gene+map+of+the+extended+human+MHC&amp;author=R+Horton&amp;author=L+Wilming&amp;author=V+Rand&amp;author=RC+Lovering&amp;author=EA+Bruford&amp;volume=5&amp;publication_year=2004&amp;pages=889-899&amp;pmid=15573121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32.  <span class="element-citation">Ferreira MA, Mangino M, Brumme CJ, Zhao ZZ, Medland SE, Wright MJ, Nyholt DR, Gordon S, Campbell M, McEvoy BP, et al.  International HIV Controllers Study. Quantitative trait loci for CD4:CD8 lymphocyte ratio are associated with risk of type 1 diabetes and HIV-1 immune control. <span><span class="ref-journal"><em>Am J Hum Genet</em>. </span>2010;<span class="ref-vol">86</span>:88–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2801744/">PMC free article</a>]</span> [<a href="/pubmed/20045101" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Quantitative+trait+loci+for+CD4:CD8+lymphocyte+ratio+are+associated+with+risk+of+type+1+diabetes+and+HIV-1+immune+control&amp;author=MA+Ferreira&amp;author=M+Mangino&amp;author=CJ+Brumme&amp;author=ZZ+Zhao&amp;author=SE+Medland&amp;volume=86&amp;publication_year=2010&amp;pages=88-92&amp;pmid=20045101&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33.  <span class="element-citation">Manolio TA, Collins FS, Cox NJ, Goldstein DB, Hindorff LA, Hunter DJ, McCarthy MI, Ramos EM, Cardon LR, Chakravarti A, et al.  Finding the missing heritability of complex diseases. <span><span class="ref-journal"><em>Nature</em>. </span>2009;<span class="ref-vol">461</span>:747–753.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2831613/">PMC free article</a>]</span> [<a href="/pubmed/19812666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Finding+the+missing+heritability+of+complex+diseases&amp;author=TA+Manolio&amp;author=FS+Collins&amp;author=NJ+Cox&amp;author=DB+Goldstein&amp;author=LA+Hindorff&amp;volume=461&amp;publication_year=2009&amp;pages=747-753&amp;pmid=19812666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34.  <span class="element-citation">Small KS, Hedman AK, Grundberg E, Nica AC, Thorleifsson G, Kong A, Thorsteindottir U, Shin SY, Richards HB, Soranzo N, et al.  GIANT Consortium; MAGIC Investigators; DIAGRAM Consortium; MuTHER Consortium. Identification of an imprinted master trans regulator at the KLF14 locus related to multiple metabolic phenotypes. <span><span class="ref-journal"><em>Nat Genet</em>. </span>2011;<span class="ref-vol">43</span>:561–564.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3192952/">PMC free article</a>]</span> [<a href="/pubmed/21572415" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Identification+of+an+imprinted+master+trans+regulator+at+the+KLF14+locus+related+to+multiple+metabolic+phenotypes&amp;author=KS+Small&amp;author=AK+Hedman&amp;author=E+Grundberg&amp;author=AC+Nica&amp;author=G+Thorleifsson&amp;volume=43&amp;publication_year=2011&amp;pages=561-564&amp;pmid=21572415&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35.  <span class="element-citation">Wei G, Wei L, Zhu J, Zang C, Hu-Li J, Yao Z, Cui K, Kanno Y, Roh TY, Watford WT, et al.  Global mapping of H3K4me3 and H3K27me3 reveals specificity and plasticity in lineage fate determination of differentiating CD4+ T cells. <span><span class="ref-journal"><em>Immunity</em>. </span>2009;<span class="ref-vol">30</span>:155–167.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2722509/">PMC free article</a>]</span> [<a href="/pubmed/19144320" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Global+mapping+of+H3K4me3+and+H3K27me3+reveals+specificity+and+plasticity+in+lineage+fate+determination+of+differentiating+CD4++T+cells&amp;author=G+Wei&amp;author=L+Wei&amp;author=J+Zhu&amp;author=C+Zang&amp;author=J+Hu-Li&amp;volume=30&amp;publication_year=2009&amp;pages=155-167&amp;pmid=19144320&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36.  <span class="element-citation">Ansel KM, Lee DU, Rao A. An epigenetic view of helper T cell differentiation. <span><span class="ref-journal"><em>Nat Immunol</em>. </span>2003;<span class="ref-vol">4</span>:616–623.</span> [<a href="/pubmed/12830136" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=An+epigenetic+view+of+helper+T+cell+differentiation&amp;author=KM+Ansel&amp;author=DU+Lee&amp;author=A+Rao&amp;volume=4&amp;publication_year=2003&amp;pages=616-623&amp;pmid=12830136&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37.  <span class="element-citation">Merkenschlager M, Wilson CB. RNAi and chromatin in T cell development and function. <span><span class="ref-journal"><em>Curr Opin Immunol</em>. </span>2008;<span class="ref-vol">20</span>:131–138.</span> [<a href="/pubmed/18440793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Immunol&amp;title=RNAi+and+chromatin+in+T+cell+development+and+function&amp;author=M+Merkenschlager&amp;author=CB+Wilson&amp;volume=20&amp;publication_year=2008&amp;pages=131-138&amp;pmid=18440793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38.  <span class="element-citation">Zhou L, Chong MM, Littman DR. Plasticity of CD4+ T cell lineage differentiation. <span><span class="ref-journal"><em>Immunity</em>. </span>2009;<span class="ref-vol">30</span>:646–655.</span> [<a href="/pubmed/19464987" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Plasticity+of+CD4++T+cell+lineage+differentiation&amp;author=L+Zhou&amp;author=MM+Chong&amp;author=DR+Littman&amp;volume=30&amp;publication_year=2009&amp;pages=646-655&amp;pmid=19464987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39.  <span class="element-citation">Wilson CB, Rowell E, Sekimata M. Epigenetic control of T-helper-cell differentiation. <span><span class="ref-journal"><em>Nat Rev Immunol</em>. </span>2009;<span class="ref-vol">9</span>:91–105.</span> [<a href="/pubmed/19151746" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Immunol&amp;title=Epigenetic+control+of+T-helper-cell+differentiation&amp;author=CB+Wilson&amp;author=E+Rowell&amp;author=M+Sekimata&amp;volume=9&amp;publication_year=2009&amp;pages=91-105&amp;pmid=19151746&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40.  <span class="element-citation">Ostadkarampour M, Eklund A, Moller D, Glader P, Olgart Höglund C, Lindén A, Grunewald J, Wahlström J. Higher levels of interleukin IL-17 and antigen-specific IL-17 responses in pulmonary sarcoidosis patients with Löfgren’s syndrome. <span><span class="ref-journal"><em>Clin Exp Immunol</em>. </span>2014;<span class="ref-vol">178</span>:342–352.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4233383/">PMC free article</a>]</span> [<a href="/pubmed/24962673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Exp+Immunol&amp;title=Higher+levels+of+interleukin+IL-17+and+antigen-specific+IL-17+responses+in+pulmonary+sarcoidosis+patients+with+Löfgren’s+syndrome&amp;author=M+Ostadkarampour&amp;author=A+Eklund&amp;author=D+Moller&amp;author=P+Glader&amp;author=C+Olgart+Höglund&amp;volume=178&amp;publication_year=2014&amp;pages=342-352&amp;pmid=24962673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4872654&amp;issue-id=269719&amp;journal-id=465&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm139913866216992title">Abstract</a></li><li><a href="#__sec4title">Methods</a></li><li><a href="#__sec14title">Results</a></li><li><a href="#__sec24title">Discussion</a></li><li><a href="#idm139913871559216title">Acknowledgment</a></li><li><a href="#idm139913871557408title">Footnotes</a></li><li><a href="#idm139913871547984title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Respiratory and Critical Care Medicine</span> are provided here courtesy of <strong>American Thoracic Society</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4872654/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4872654/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4872654/pdf/rccm.201507-1372OC.pdf">PDF (1.2M)</a></li> | <li><a href="#" data-citationid="PMC4872654" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4872654%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4872654%2F&amp;text=High-Density%20Genetic%20Mapping%20Identifies%20New%20Susceptibility%20Variants%20in%20Sarcoidosis%20Phenotypes%20and%20Shows%20Genomic-driven%20Phenotypic%20Differences" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4872654%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26651848" data-aiid="4872654" data-aid="4872654" data-iid="269719" data-domainid="465" data-domain="ajrccm" data-accid="PMC4872654" data-md5="9f86b324db23edacb2675656d658ad0f"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:19:48-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8AD78ADD58F53100000000018A00E2&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4872654%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajrccm&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4872654/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>