<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4112126/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Frontotemporal dementia and its subtypes: a genome-wide association study</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4112126/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Lancet neurology"><meta name="citation_title" content="Frontotemporal dementia and its subtypes: a genome-wide association study"><meta name="citation_authors" content="Raffaele Ferrari, Dena G Hernandez, Michael A Nalls, Jonathan D Rohrer, Adaikalavan Ramasamy, John B J Kwok, Carol Dobson-Stone, William S Brooks, Peter R Schofield, Glenda M Halliday, John R Hodges, Olivier Piguet, Lauren Bartley, Elizabeth Thompson, Eric Haan, Isabel Hernández, Agustín Ruiz, Mercè Boada, Barbara Borroni, Alessandro Padovani, Carlos Cruchaga, Nigel J Cairns, Luisa Benussi, Giuliano Binetti, Roberta Ghidoni, Gianluigi Forloni, Daniela Galimberti, Chiara Fenoglio, Maria Serpente, Elio Scarpini, Jordi Clarimón, Alberto Lleó, Rafael Blesa, Maria Landqvist Waldö, Karin Nilsson, Christer Nilsson, Ian R A Mackenzie, Ging-Yuek R Hsiung, David M A Mann, Jordan Grafman, Christopher M Morris, Johannes Attems, Timothy D Griffiths, Ian G McKeith, Alan J Thomas, P Pietrini, Edward D Huey, Eric M Wassermann, Atik Baborie, Evelyn Jaros, Michael C Tierney, Pau Pastor, Cristina Razquin, Sara Ortega-Cubero, Elena Alonso, Robert Perneczky, Janine Diehl-Schmid, Panagiotis Alexopoulos, Alexander Kurz, Innocenzo Rainero, Elisa Rubino, Lorenzo Pinessi, Ekaterina Rogaeva, Peter St George-Hyslop, Giacomina Rossi, Fabrizio Tagliavini, Giorgio Giaccone, James B Rowe, J C M Schlachetzki, James Uphill, John Collinge, S Mead, Adrian Danek, Vivianna M Van Deerlin, Murray Grossman, John Q Trojanowsk, Julie van der Zee, William Deschamps, Tim Van Langenhove, Marc Cruts, Christine Van Broeckhoven, Stefano F Cappa, Isabelle Le Ber, Didier Hannequin, Véronique Golfier, Martine Vercelletto, Alexis Brice, Benedetta Nacmias, Sandro Sorbi, Silvia Bagnoli, Irene Piaceri, Jørgen E Nielsen, Lena E Hjermind, Matthias Riemenschneider, Manuel Mayhaus, Bernd Ibach, Gilles Gasparoni, Sabrina Pichler, Wei Gu, Martin N Rossor, Nick C Fox, Jason D Warren, Maria Grazia Spillantini, Huw R Morris, Patrizia Rizzu, Peter Heutink, Julie S Snowden, Sara Rollinson, Anna Richardson, Alexander Gerhard, Amalia C Bruni, Raffaele Maletta, Francesca Frangipane, Chiara Cupidi, Livia Bernardi, Maria Anfossi, Maura Gallo, Maria Elena Conidi, Nicoletta Smirne, Rosa Rademakers, Matt Baker, Dennis W Dickson, Neill R Graff-Radford, Ronald C Petersen, David Knopman, Keith A Josephs, Bradley F Boeve, Joseph E Parisi, William W Seeley, Bruce L Miller, Anna M Karydas, Howard Rosen, John C van Swieten, Elise G P Dopper, Harro Seelaar, Yolande AL Pijnenburg, Philip Scheltens, Giancarlo Logroscino, Rosa Capozzo, Valeria Novelli, Annibale A Puca, M Franceschi, Alfredo Postiglione, Graziella Milan, Paolo Sorrentino, Mark Kristiansen, Huei-Hsin Chiang, Caroline Graff, Florence Pasquier, Adeline Rollin, Vincent Deramecourt, Florence Lebert, Dimitrios Kapogiannis, Luigi Ferrucci, Stuart Pickering-Brown, Andrew B Singleton, John Hardy, Parastoo Momeni"><meta name="citation_date" content="July 2014"><meta name="citation_issue" content="7"><meta name="citation_volume" content="13"><meta name="citation_firstpage" content="686"><meta name="citation_doi" content="10.1016/S1474-4422(14)70065-1"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4112126/?report=abstract"><meta name="citation_pmid" content="24943344"><meta name="DC.Title" content="Frontotemporal dementia and its subtypes: a genome-wide association study"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Raffaele Ferrari"><meta name="DC.Contributor" content="Dena G Hernandez"><meta name="DC.Contributor" content="Michael A Nalls"><meta name="DC.Contributor" content="Jonathan D Rohrer"><meta name="DC.Contributor" content="Adaikalavan Ramasamy"><meta name="DC.Contributor" content="John B J Kwok"><meta name="DC.Contributor" content="Carol Dobson-Stone"><meta name="DC.Contributor" content="William S Brooks"><meta name="DC.Contributor" content="Peter R Schofield"><meta name="DC.Contributor" content="Glenda M Halliday"><meta name="DC.Contributor" content="John R Hodges"><meta name="DC.Contributor" content="Olivier Piguet"><meta name="DC.Contributor" content="Lauren Bartley"><meta name="DC.Contributor" content="Elizabeth Thompson"><meta name="DC.Contributor" content="Eric Haan"><meta name="DC.Contributor" content="Isabel Hernández"><meta name="DC.Contributor" content="Agustín Ruiz"><meta name="DC.Contributor" content="Mercè Boada"><meta name="DC.Contributor" content="Barbara Borroni"><meta name="DC.Contributor" content="Alessandro Padovani"><meta name="DC.Contributor" content="Carlos Cruchaga"><meta name="DC.Contributor" content="Nigel J Cairns"><meta name="DC.Contributor" content="Luisa Benussi"><meta name="DC.Contributor" content="Giuliano Binetti"><meta name="DC.Contributor" content="Roberta Ghidoni"><meta name="DC.Contributor" content="Gianluigi Forloni"><meta name="DC.Contributor" content="Daniela Galimberti"><meta name="DC.Contributor" content="Chiara Fenoglio"><meta name="DC.Contributor" content="Maria Serpente"><meta name="DC.Contributor" content="Elio Scarpini"><meta name="DC.Contributor" content="Jordi Clarimón"><meta name="DC.Contributor" content="Alberto Lleó"><meta name="DC.Contributor" content="Rafael Blesa"><meta name="DC.Contributor" content="Maria Landqvist Waldö"><meta name="DC.Contributor" content="Karin Nilsson"><meta name="DC.Contributor" content="Christer Nilsson"><meta name="DC.Contributor" content="Ian R A Mackenzie"><meta name="DC.Contributor" content="Ging-Yuek R Hsiung"><meta name="DC.Contributor" content="David M A Mann"><meta name="DC.Contributor" content="Jordan Grafman"><meta name="DC.Contributor" content="Christopher M Morris"><meta name="DC.Contributor" content="Johannes Attems"><meta name="DC.Contributor" content="Timothy D Griffiths"><meta name="DC.Contributor" content="Ian G McKeith"><meta name="DC.Contributor" content="Alan J Thomas"><meta name="DC.Contributor" content="P Pietrini"><meta name="DC.Contributor" content="Edward D Huey"><meta name="DC.Contributor" content="Eric M Wassermann"><meta name="DC.Contributor" content="Atik Baborie"><meta name="DC.Contributor" content="Evelyn Jaros"><meta name="DC.Contributor" content="Michael C Tierney"><meta name="DC.Contributor" content="Pau Pastor"><meta name="DC.Contributor" content="Cristina Razquin"><meta name="DC.Contributor" content="Sara Ortega-Cubero"><meta name="DC.Contributor" content="Elena Alonso"><meta name="DC.Contributor" content="Robert Perneczky"><meta name="DC.Contributor" content="Janine Diehl-Schmid"><meta name="DC.Contributor" content="Panagiotis Alexopoulos"><meta name="DC.Contributor" content="Alexander Kurz"><meta name="DC.Contributor" content="Innocenzo Rainero"><meta name="DC.Contributor" content="Elisa Rubino"><meta name="DC.Contributor" content="Lorenzo Pinessi"><meta name="DC.Contributor" content="Ekaterina Rogaeva"><meta name="DC.Contributor" content="Peter St George-Hyslop"><meta name="DC.Contributor" content="Giacomina Rossi"><meta name="DC.Contributor" content="Fabrizio Tagliavini"><meta name="DC.Contributor" content="Giorgio Giaccone"><meta name="DC.Contributor" content="James B Rowe"><meta name="DC.Contributor" content="J C M Schlachetzki"><meta name="DC.Contributor" content="James Uphill"><meta name="DC.Contributor" content="John Collinge"><meta name="DC.Contributor" content="S Mead"><meta name="DC.Contributor" content="Adrian Danek"><meta name="DC.Contributor" content="Vivianna M Van Deerlin"><meta name="DC.Contributor" content="Murray Grossman"><meta name="DC.Contributor" content="John Q Trojanowsk"><meta name="DC.Contributor" content="Julie van der Zee"><meta name="DC.Contributor" content="William Deschamps"><meta name="DC.Contributor" content="Tim Van Langenhove"><meta name="DC.Contributor" content="Marc Cruts"><meta name="DC.Contributor" content="Christine Van Broeckhoven"><meta name="DC.Contributor" content="Stefano F Cappa"><meta name="DC.Contributor" content="Isabelle Le Ber"><meta name="DC.Contributor" content="Didier Hannequin"><meta name="DC.Contributor" content="Véronique Golfier"><meta name="DC.Contributor" content="Martine Vercelletto"><meta name="DC.Contributor" content="Alexis Brice"><meta name="DC.Contributor" content="Benedetta Nacmias"><meta name="DC.Contributor" content="Sandro Sorbi"><meta name="DC.Contributor" content="Silvia Bagnoli"><meta name="DC.Contributor" content="Irene Piaceri"><meta name="DC.Contributor" content="Jørgen E Nielsen"><meta name="DC.Contributor" content="Lena E Hjermind"><meta name="DC.Contributor" content="Matthias Riemenschneider"><meta name="DC.Contributor" content="Manuel Mayhaus"><meta name="DC.Contributor" content="Bernd Ibach"><meta name="DC.Contributor" content="Gilles Gasparoni"><meta name="DC.Contributor" content="Sabrina Pichler"><meta name="DC.Contributor" content="Wei Gu"><meta name="DC.Contributor" content="Martin N Rossor"><meta name="DC.Contributor" content="Nick C Fox"><meta name="DC.Contributor" content="Jason D Warren"><meta name="DC.Contributor" content="Maria Grazia Spillantini"><meta name="DC.Contributor" content="Huw R Morris"><meta name="DC.Contributor" content="Patrizia Rizzu"><meta name="DC.Contributor" content="Peter Heutink"><meta name="DC.Contributor" content="Julie S Snowden"><meta name="DC.Contributor" content="Sara Rollinson"><meta name="DC.Contributor" content="Anna Richardson"><meta name="DC.Contributor" content="Alexander Gerhard"><meta name="DC.Contributor" content="Amalia C Bruni"><meta name="DC.Contributor" content="Raffaele Maletta"><meta name="DC.Contributor" content="Francesca Frangipane"><meta name="DC.Contributor" content="Chiara Cupidi"><meta name="DC.Contributor" content="Livia Bernardi"><meta name="DC.Contributor" content="Maria Anfossi"><meta name="DC.Contributor" content="Maura Gallo"><meta name="DC.Contributor" content="Maria Elena Conidi"><meta name="DC.Contributor" content="Nicoletta Smirne"><meta name="DC.Contributor" content="Rosa Rademakers"><meta name="DC.Contributor" content="Matt Baker"><meta name="DC.Contributor" content="Dennis W Dickson"><meta name="DC.Contributor" content="Neill R Graff-Radford"><meta name="DC.Contributor" content="Ronald C Petersen"><meta name="DC.Contributor" content="David Knopman"><meta name="DC.Contributor" content="Keith A Josephs"><meta name="DC.Contributor" content="Bradley F Boeve"><meta name="DC.Contributor" content="Joseph E Parisi"><meta name="DC.Contributor" content="William W Seeley"><meta name="DC.Contributor" content="Bruce L Miller"><meta name="DC.Contributor" content="Anna M Karydas"><meta name="DC.Contributor" content="Howard Rosen"><meta name="DC.Contributor" content="John C van Swieten"><meta name="DC.Contributor" content="Elise G P Dopper"><meta name="DC.Contributor" content="Harro Seelaar"><meta name="DC.Contributor" content="Yolande AL Pijnenburg"><meta name="DC.Contributor" content="Philip Scheltens"><meta name="DC.Contributor" content="Giancarlo Logroscino"><meta name="DC.Contributor" content="Rosa Capozzo"><meta name="DC.Contributor" content="Valeria Novelli"><meta name="DC.Contributor" content="Annibale A Puca"><meta name="DC.Contributor" content="M Franceschi"><meta name="DC.Contributor" content="Alfredo Postiglione"><meta name="DC.Contributor" content="Graziella Milan"><meta name="DC.Contributor" content="Paolo Sorrentino"><meta name="DC.Contributor" content="Mark Kristiansen"><meta name="DC.Contributor" content="Huei-Hsin Chiang"><meta name="DC.Contributor" content="Caroline Graff"><meta name="DC.Contributor" content="Florence Pasquier"><meta name="DC.Contributor" content="Adeline Rollin"><meta name="DC.Contributor" content="Vincent Deramecourt"><meta name="DC.Contributor" content="Florence Lebert"><meta name="DC.Contributor" content="Dimitrios Kapogiannis"><meta name="DC.Contributor" content="Luigi Ferrucci"><meta name="DC.Contributor" content="Stuart Pickering-Brown"><meta name="DC.Contributor" content="Andrew B Singleton"><meta name="DC.Contributor" content="John Hardy"><meta name="DC.Contributor" content="Parastoo Momeni"><meta name="DC.Date" content="2014 Jul"><meta name="DC.Identifier" content="10.1016/S1474-4422(14)70065-1"><meta name="DC.Language" content="en"><meta property="og:title" content="Frontotemporal dementia and its subtypes: a genome-wide association study"><meta property="og:type" content="article"><meta property="og:description" content="Frontotemporal dementia (FTD) is a complex disorder characterised by a broad range of clinical manifestations, differential pathological signatures, and genetic variability. Mutations in three genes—MAPT, GRN, and C9orf72—have been associated ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4112126/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4112126/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4112126/pdf/nihms599470.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8815A8DD5723D10000000002B8019D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/config.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/input/MathML/config.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/NativeMML/config.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/mml2jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathEvents.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathZoom.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathMenu.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/toMathML.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4112126%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4112126</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Lancet Neurol</a></span>. Author manuscript; available in PMC 2015 Jul 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=24943344" target="pmc_ext" ref="reftype=publisher&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Lancet Neurol. 2014 Jul; 13(7): 686–699. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2FS1474-4422(14)70065-1" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/S1474-4422(14)70065-1</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4112126</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS599470</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24943344">24943344</a></div></div></div></div><h1 class="content-title">Frontotemporal dementia and its subtypes: a genome-wide association study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Ferrari%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Raffaele Ferrari</a>,<sup>*</sup> <a href="/pubmed/?term=Hernandez%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Dena G Hernandez</a>,<sup>*</sup> <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Michael A Nalls</a>,<sup>*</sup> <a href="/pubmed/?term=Rohrer%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Jonathan D Rohrer</a>,<sup>*</sup> <a href="/pubmed/?term=Ramasamy%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Adaikalavan Ramasamy</a>, <a href="/pubmed/?term=Kwok%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">John B J Kwok</a>, <a href="/pubmed/?term=Dobson-Stone%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Carol Dobson-Stone</a>, <a href="/pubmed/?term=Brooks%20WS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">William S Brooks</a>, <a href="/pubmed/?term=Schofield%20PR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Peter R Schofield</a>, <a href="/pubmed/?term=Halliday%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Glenda M Halliday</a>, <a href="/pubmed/?term=Hodges%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">John R Hodges</a>, <a href="/pubmed/?term=Piguet%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Olivier Piguet</a>, <a href="/pubmed/?term=Bartley%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Lauren Bartley</a>, <a href="/pubmed/?term=Thompson%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Elizabeth Thompson</a>, <a href="/pubmed/?term=Haan%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Eric Haan</a>, <a href="/pubmed/?term=Hern%26%23x000e1%3Bndez%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Isabel Hernández</a>, <a href="/pubmed/?term=Ruiz%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Agustín Ruiz</a>, <a href="/pubmed/?term=Boada%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Mercè Boada</a>, <a href="/pubmed/?term=Borroni%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Barbara Borroni</a>, <a href="/pubmed/?term=Padovani%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Alessandro Padovani</a>, <a href="/pubmed/?term=Cruchaga%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Carlos Cruchaga</a>, <a href="/pubmed/?term=Cairns%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Nigel J Cairns</a>, <a href="/pubmed/?term=Benussi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Luisa Benussi</a>, <a href="/pubmed/?term=Binetti%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Giuliano Binetti</a>, <a href="/pubmed/?term=Ghidoni%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Roberta Ghidoni</a>, <a href="/pubmed/?term=Forloni%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Gianluigi Forloni</a>, <a href="/pubmed/?term=Galimberti%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Daniela Galimberti</a>, <a href="/pubmed/?term=Fenoglio%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Chiara Fenoglio</a>, <a href="/pubmed/?term=Serpente%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Maria Serpente</a>, <a href="/pubmed/?term=Scarpini%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Elio Scarpini</a>, <a href="/pubmed/?term=Clarim%26%23x000f3%3Bn%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Jordi Clarimón</a>, <a href="/pubmed/?term=Lle%26%23x000f3%3B%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Alberto Lleó</a>, <a href="/pubmed/?term=Blesa%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Rafael Blesa</a>, <a href="/pubmed/?term=Wald%26%23x000f6%3B%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Maria Landqvist Waldö</a>, <a href="/pubmed/?term=Nilsson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Karin Nilsson</a>, <a href="/pubmed/?term=Nilsson%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Christer Nilsson</a>, <a href="/pubmed/?term=Mackenzie%20IR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Ian R A Mackenzie</a>, <a href="/pubmed/?term=Hsiung%20GY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Ging-Yuek R Hsiung</a>, <a href="/pubmed/?term=Mann%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">David M A Mann</a>, <a href="/pubmed/?term=Grafman%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Jordan Grafman</a>, <a href="/pubmed/?term=Morris%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Christopher M Morris</a>, <a href="/pubmed/?term=Attems%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Johannes Attems</a>, <a href="/pubmed/?term=Griffiths%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Timothy D Griffiths</a>, <a href="/pubmed/?term=McKeith%20IG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Ian G McKeith</a>, <a href="/pubmed/?term=Thomas%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Alan J Thomas</a>, <a href="/pubmed/?term=Pietrini%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">P Pietrini</a>, <a href="/pubmed/?term=Huey%20ED%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Edward D Huey</a>, <a href="/pubmed/?term=Wassermann%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Eric M Wassermann</a>, <a href="/pubmed/?term=Baborie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Atik Baborie</a>, <a href="/pubmed/?term=Jaros%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Evelyn Jaros</a>, <a href="/pubmed/?term=Tierney%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Michael C Tierney</a>, <a href="/pubmed/?term=Pastor%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Pau Pastor</a>, <a href="/pubmed/?term=Razquin%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Cristina Razquin</a>, <a href="/pubmed/?term=Ortega-Cubero%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Sara Ortega-Cubero</a>, <a href="/pubmed/?term=Alonso%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Elena Alonso</a>, <a href="/pubmed/?term=Perneczky%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Robert Perneczky</a>, <a href="/pubmed/?term=Diehl-Schmid%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Janine Diehl-Schmid</a>, <a href="/pubmed/?term=Alexopoulos%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Panagiotis Alexopoulos</a>, <a href="/pubmed/?term=Kurz%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Alexander Kurz</a>, <a href="/pubmed/?term=Rainero%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Innocenzo Rainero</a>, <a href="/pubmed/?term=Rubino%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Elisa Rubino</a>, <a href="/pubmed/?term=Pinessi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Lorenzo Pinessi</a>, <a href="/pubmed/?term=Rogaeva%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Ekaterina Rogaeva</a>, <a href="/pubmed/?term=George-Hyslop%20PS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Peter St George-Hyslop</a>, <a href="/pubmed/?term=Rossi%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Giacomina Rossi</a>, <a href="/pubmed/?term=Tagliavini%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Fabrizio Tagliavini</a>, <a href="/pubmed/?term=Giaccone%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Giorgio Giaccone</a>, <a href="/pubmed/?term=Rowe%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">James B Rowe</a>, <a href="/pubmed/?term=Schlachetzki%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">J C M Schlachetzki</a>, <a href="/pubmed/?term=Uphill%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">James Uphill</a>, <a href="/pubmed/?term=Collinge%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">John Collinge</a>, <a href="/pubmed/?term=Mead%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">S Mead</a>, <a href="/pubmed/?term=Danek%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Adrian Danek</a>, <a href="/pubmed/?term=Van%20Deerlin%20VM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Vivianna M Van Deerlin</a>, <a href="/pubmed/?term=Grossman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Murray Grossman</a>, <a href="/pubmed/?term=Trojanowsk%20JQ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">John Q Trojanowsk</a>, <a href="/pubmed/?term=van%20der%20Zee%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Julie van der Zee</a>, <a href="/pubmed/?term=Deschamps%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">William Deschamps</a>, <a href="/pubmed/?term=Van%20Langenhove%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Tim Van Langenhove</a>, <a href="/pubmed/?term=Cruts%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Marc Cruts</a>, <a href="/pubmed/?term=Van%20Broeckhoven%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Christine Van Broeckhoven</a>, <a href="/pubmed/?term=Cappa%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Stefano F Cappa</a>, <a href="/pubmed/?term=Le%20Ber%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Isabelle Le Ber</a>, <a href="/pubmed/?term=Hannequin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Didier Hannequin</a>, <a href="/pubmed/?term=Golfier%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Véronique Golfier</a>, <a href="/pubmed/?term=Vercelletto%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Martine Vercelletto</a>, <a href="/pubmed/?term=Brice%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Alexis Brice</a>, <a href="/pubmed/?term=Nacmias%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Benedetta Nacmias</a>, <a href="/pubmed/?term=Sorbi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Sandro Sorbi</a>, <a href="/pubmed/?term=Bagnoli%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Silvia Bagnoli</a>, <a href="/pubmed/?term=Piaceri%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Irene Piaceri</a>, <a href="/pubmed/?term=Nielsen%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Jørgen E Nielsen</a>, <a href="/pubmed/?term=Hjermind%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Lena E Hjermind</a>, <a href="/pubmed/?term=Riemenschneider%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Matthias Riemenschneider</a>, <a href="/pubmed/?term=Mayhaus%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Manuel Mayhaus</a>, <a href="/pubmed/?term=Ibach%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Bernd Ibach</a>, <a href="/pubmed/?term=Gasparoni%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Gilles Gasparoni</a>, <a href="/pubmed/?term=Pichler%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Sabrina Pichler</a>, <a href="/pubmed/?term=Gu%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Wei Gu</a>, <a href="/pubmed/?term=Rossor%20MN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Martin N Rossor</a>, <a href="/pubmed/?term=Fox%20NC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Nick C Fox</a>, <a href="/pubmed/?term=Warren%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Jason D Warren</a>, <a href="/pubmed/?term=Spillantini%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Maria Grazia Spillantini</a>, <a href="/pubmed/?term=Morris%20HR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Huw R Morris</a>, <a href="/pubmed/?term=Rizzu%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Patrizia Rizzu</a>, <a href="/pubmed/?term=Heutink%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Peter Heutink</a>, <a href="/pubmed/?term=Snowden%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Julie S Snowden</a>, <a href="/pubmed/?term=Rollinson%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Sara Rollinson</a>, <a href="/pubmed/?term=Richardson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Anna Richardson</a>, <a href="/pubmed/?term=Gerhard%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Alexander Gerhard</a>, <a href="/pubmed/?term=Bruni%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Amalia C Bruni</a>, <a href="/pubmed/?term=Maletta%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Raffaele Maletta</a>, <a href="/pubmed/?term=Frangipane%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Francesca Frangipane</a>, <a href="/pubmed/?term=Cupidi%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Chiara Cupidi</a>, <a href="/pubmed/?term=Bernardi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Livia Bernardi</a>, <a href="/pubmed/?term=Anfossi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Maria Anfossi</a>, <a href="/pubmed/?term=Gallo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Maura Gallo</a>, <a href="/pubmed/?term=Conidi%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Maria Elena Conidi</a>, <a href="/pubmed/?term=Smirne%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Nicoletta Smirne</a>, <a href="/pubmed/?term=Rademakers%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Rosa Rademakers</a>, <a href="/pubmed/?term=Baker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Matt Baker</a>, <a href="/pubmed/?term=Dickson%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Dennis W Dickson</a>, <a href="/pubmed/?term=Graff-Radford%20NR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Neill R Graff-Radford</a>, <a href="/pubmed/?term=Petersen%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Ronald C Petersen</a>, <a href="/pubmed/?term=Knopman%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">David Knopman</a>, <a href="/pubmed/?term=Josephs%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Keith A Josephs</a>, <a href="/pubmed/?term=Boeve%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Bradley F Boeve</a>, <a href="/pubmed/?term=Parisi%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Joseph E Parisi</a>, <a href="/pubmed/?term=Seeley%20WW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">William W Seeley</a>, <a href="/pubmed/?term=Miller%20BL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Bruce L Miller</a>, <a href="/pubmed/?term=Karydas%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Anna M Karydas</a>, <a href="/pubmed/?term=Rosen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Howard Rosen</a>, <a href="/pubmed/?term=van%20Swieten%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">John C van Swieten</a>, <a href="/pubmed/?term=Dopper%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Elise G P Dopper</a>, <a href="/pubmed/?term=Seelaar%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Harro Seelaar</a>, <a href="/pubmed/?term=Pijnenburg%20YA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Yolande AL Pijnenburg</a>, <a href="/pubmed/?term=Scheltens%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Philip Scheltens</a>, <a href="/pubmed/?term=Logroscino%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Giancarlo Logroscino</a>, <a href="/pubmed/?term=Capozzo%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Rosa Capozzo</a>, <a href="/pubmed/?term=Novelli%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Valeria Novelli</a>, <a href="/pubmed/?term=Puca%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Annibale A Puca</a>, <a href="/pubmed/?term=Franceschi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">M Franceschi</a>, <a href="/pubmed/?term=Postiglione%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Alfredo Postiglione</a>, <a href="/pubmed/?term=Milan%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Graziella Milan</a>, <a href="/pubmed/?term=Sorrentino%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Paolo Sorrentino</a>, <a href="/pubmed/?term=Kristiansen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Mark Kristiansen</a>, <a href="/pubmed/?term=Chiang%20HH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Huei-Hsin Chiang</a>, <a href="/pubmed/?term=Graff%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Caroline Graff</a>, <a href="/pubmed/?term=Pasquier%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Florence Pasquier</a>, <a href="/pubmed/?term=Rollin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Adeline Rollin</a>, <a href="/pubmed/?term=Deramecourt%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Vincent Deramecourt</a>, <a href="/pubmed/?term=Lebert%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Florence Lebert</a>, <a href="/pubmed/?term=Kapogiannis%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Dimitrios Kapogiannis</a>, <a href="/pubmed/?term=Ferrucci%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Luigi Ferrucci</a>, <a href="/pubmed/?term=Pickering-Brown%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Stuart Pickering-Brown</a>, <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Andrew B Singleton</a>,<sup>†</sup> <a href="/pubmed/?term=Hardy%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">John Hardy</a>,<sup>†</sup> and  <a href="/pubmed/?term=Momeni%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24943344">Parastoo Momeni</a><sup>†</sup></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140164567936496_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140164567936496_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140164567936496_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">Laboratory of Neurogenetics, Department of Internal Medicine, Texas Tech University Health Science Center, Lubbock, Texas, USA (R Ferrari MSc, P Momeni PhD); Reta Lila Weston Research Laboratories, Department of Molecular Neuroscience, UCL Institute of Neurology, London, UK (R Ferrari, D G Hernandez MSc, J D Rohrer PhD, A Ramasamy PhD, J Hardy PhD); Laboratory of Neurogenetics, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA (D G Hernandez, M A Nalls PhD, A B Singleton PhD); Clinical Research Branch, National Institute on Aging, Baltimore, MD, USA (L Ferrucci MD); Institute of Brain, Behaviour and Mental Health, Faculty of Medical and Human Sciences, University of Manchester, Manchester, UK (J S Snowden PhD, S Rollinson PhD, S Pickering-Brown PhD); Neuroscience Research Australia, Sydney, NSW, Australia (J B J Kwok PhD, C Dobson-Stone PhD, W S Brooks MBBS, P R Schofield DSc, G M Halliday PhD, J R Hodges MD, O Piguet PhD, L Bartley MSc); University of New South Wales, Sydney, NSW, Australia (J B J Kwok, C Dobson-Stone, W S Brooks, P R Schofield, G M Halliday, J R Hodges, O Piguet); South Australian Clinical Genetics Service, SA Pathology at Women’s and Children’s Hospital, North Adelaide, SA, Australia (E Thompson MD, E Haan MBBS); Department of Paediatrics, University of Adelaide, Adelaide, SA, Australia (E Thompson, E Haan); Memory Clinic of Fundació ACE, Institut Català de Neurociències Aplicades, Barcelona, Spain (I Hernández MD, A Ruiz MD, M Boada MD); Hospital Universitari Vall d’Hebron–Institut de Recerca, Universitat Autonoma de Barcelona (VHIR-UAB), Barcelona, Spain (M Boada); Neurology Clinic, University of Brescia, Brescia, Italy (B Borroni MD, A Padovani MD); Department of Psychiatry (C Cruchaga PhD), Hope Center (C Cruchaga, N J Cairns PhD), Washington University School of Medicine, St Louis, Missouri, USA; Department of Pathology and Immunology, Washington University, St Louis, Missouri, USA (N J Cairns); NeuroBioGen Lab-Memory Clinic, IRCCS Istituto Centro San Giovanni di Dio Fatebenefratelli, Brescia, Italy (L Benussi PhD, G Binetti MD); Proteomics Unit, IRCCS Istituto Centro San Giovanni di Dio Fatebenefratelli, Brescia Italy (L Benussi, G Binetti); Biology of Neurodegenerative Disorders, IRCCS Istituto di Ricerche Farmacologiche Mario Negri, Milano, Italy (G Forloni PhD); University of Milan, Milan, Italy (D Galimberti PhD, C Fenoglio PhD, M Serpente PhD, E Scarpini MD); Fondazione Cà Granda, IRCCS Ospedale Maggiore Policlinico, Milan, Italy (D Galimberti, C Fenoglio, M Serpente, E Scarpini); Memory Unit, Neurology Department and Sant Pau Biomedical Research Institute, Hospital de la Santa Creu i Sant Pau, Universitat Autònoma de Barcelona, Barcelona, Spain (J Clarimón PhD, A Lleó MD, R Blesa MD); Center for Networker Biomedical Research in Neurodegenerative Diseases (CIBERNED), Madrid, Spain (J Clarimón, A Lleó, R Blesa, P Pastor MD, S Ortega-Cubero MD); Unit of Geriatric Psychiatry (M L Waldö MD, K Nilsson PhD), Clinical Memory Research Unit (C Nilsson PhD), Department of Clinical Sciences, Lund University, Sweden; Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, Canada (I R A Mackenzie MD); Division of Neurology, University of British Columbia, Vancouver, Canada (G-Y R Hsiung MD); Institute of Brain, Behaviour and Mental Health, University of Manchester, Salford Royal Hospital, Stott Lane, Salford, UK (D M A Mann PhD); Rehabilitation Institute of Chicago, Departments of Physical Medicine and Rehabilitation, Psychiatry, and Cognitive Neurology and Alzheimer’s Disease Center; Feinberg School of Medicine, Northwestern University, USA (J Grafman PhD, C M Morris PhD, J Attems MD, T D Griffiths FMedSci); Department of Psychology, Weinberg College of Arts and Sciences, Northwestern University, USA (J Grafman); Newcastle Brain Tissue Resource, Institute for Ageing and Health, Newcastle University, Newcastle upon Tyne, UK (C M Morris, J Attems MD, T D Griffiths FMedSci); Newcastle University, Institute for Ageing and Health, Campus for Ageing and Vitality, Newcastle upon Tyne, UK (C M Morris, J Attems MD, A J Thomas PhD, E Jaros PhD); Institute of Neuroscience, Newcastle University Medical School, Newcastle upon Tyne, UK (C M Morris, T D Griffiths FMedSci); Biomedical Research Building, Campus for Ageing and Vitality, Newcastle University, Newcastle upon Tyne, UK (I G McKeith MD); Clinical Psychology Branch, Pisa University Hospital, Pisa, Italy, Laboratory of Clinical Biochemistry and Molecular Biology, University of Pisa, Pisa, Italy (P Pietrini MD); Taub Institute, Departments of Psychiatry and Neurology, Columbia University, New York, NY, USA 10032 (E D Huey MD); Behavioral Neurology Unit, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD, USA (E M Wassermann MD, M C Tierney MSc); Neuropathology Department, Walton Centre FT, Liverpool, UK (A Baborie MD); Neuropathology/Cellular Pathology, Royal Victoria Infirmary, Newcastle upon Tyne, UK (E Jaros); Neurogenetics Laboratory, Division of Neurosciences, Center for Applied Medical Research, Universidad de Navarra, Pamplona, Spain (P Pastor, C Razquin PhD, S Ortega-Cubero, E Alonso BSc); Department of Neurology, Clínica Universidad de Navarra, University of Navarra School of Medicine, Pamplona, Spain (P Pastor); Neuroepidemiology and Ageing Research Unit, School of Public Health, Faculty of Medicine, The Imperial College of Science, Technology and Medicine, London, UK (R Perneczky MD); West London Cognitive Disorders Treatment and Research Unit, West London Mental Health Trust, London TW8 8 DS, UK (R Perneczky); Department of Psychiatry and Psychotherapy, Technische Universität München, Munich, Germany (R Perneczky, J Diehl-Schmid MD, P Alexopoulos MD, A Kurz MD); Neurology I, Department of Neuroscience, University of Torino, Italy, AO Città della Salute e della Scienza di Torino, Italy (I Rainero MD, E Rubino MD, L Pinessi MD); Tanz Centre for Research in Neurodegenerative Diseases and Department of Medicine, University of Toronto, Toronto, Ontario, Canada (E Rogaeva PhD, P St George-Hyslop MD); Cambridge Institute for Medical Research and the Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK (P St George-Hyslop); Division of Neurology V and Neuropathology, Fondazione IRCCS Istituto Neurologico Carlo Besta, Milano Italy (G Rossi PhD, F Tagliavini MD, G Giaccone MD); Cambridge University Department of Clinical Neurosciences, Cambridge, CB2 0SZ, UK (J B Rowe PhD); MRC Cognition and Brain Sciences Unit, Cambridge, UK (J B Rowe); Behavioural and Clinical Neuroscience Institute, Cambridge, UK (J B Rowe); Department of Psychiatry and Psychotherapy, University of Freiburg Medical School, Germany (J C M Schlachetzki MD); Department of Molecular Neurology, University Hospital Erlangen, Erlangen, Germany (J C M Schlachetzki); MRC Prion Unit, Department of Neurodegenerative Disease, UCL Institute of Neurology, London, UK (J Uphill BSc, J Collinge MD, S Mead PhD); Neurologische Klinik und Poliklinik, Ludwig-Maximilians-Universität, Munich, Germany (A Danek MD); German Center for Neurodegenerative Diseases (DZNE), Munich, Germany (A Danek); University of Pennsylvania Perelman School of Medicine, Department of Neurology and Penn Frontotemporal Degeneration Center, Philadelphia, PA, USA (V M Van Deerlin PhD, M Grossman MD, J Q Trojanowski PhD); Neurodegenerative Brain Diseases group, Department of Molecular Genetics, VIB, Antwerp, Belgium (J van der Zee PhD, W Deschamps MSc, T Van Langenhove MD, M Cruts PhD, C Van Broeckhoven PhD); Laboratory of Neurogenetics, Institute Born-Bunge, University of Antwerp, Antwerp, Belgium (J van der Zee, W Deschamps, T Van Langenhove, M Cruts, C Van Broeckhoven); Neurorehabilitation Unit, Deptartment Of Clinical Neuroscience, Vita-Salute University and San Raffaele Scientific Institute, Milan, Italy (S F Cappa MD); Inserm, UMR_S975, CRICM, F-75013; UPMC Univ Paris 06, UMR_S975, F-75013; and CNRS UMR 7225, F-75013, Paris, France (I Le Ber MD, A Brice MD); AP-HP, Hôpital de la Salpêtrière, Département de neurologie-centre de références des démences rares, F-75013, Paris, France (I Le Ber, A Brice); Service de Neurologie, Inserm U1079, CNR-MAJ, Rouen University Hospital, France (D Hannequin MD); Service de neurologie, CH Saint Brieuc, France (V Golfier MD); Service de Neurologie, CHU Nantes, France (M Vercelletto MD); Department of Neurosciences, Psychology, Drug Research and Child Health (NEUROFARBA) University of Florence, Florence, Italy (B Nacmias PhD, S Sorbi PhD, S Bagnoli PhD, I Piaceri PhD); Danish Dementia Research Centre, Neurogenetics Clinic, Department of Neurology, Rigshospitalet, Copenhagen University Hospital, Denmark (J E Nielsen MD, L E Hjermind MD); Department of Cellular and Molecular Medicine, Section of Neurogenetics, The Panum Institute, University of Copenhagen, Denmark (J E Nielsen MD, L E Hjermind MD); Saarland University Hospital, Department for Psychiatry and Psychotherapy, Homburg/Saar, Germany (M Riemenschneider MD); Saarland University, Laboratory for Neurogenetics, Kirrberger, Homburg/Saar, Germany (M Riemenschneider, M Mayhaus PhD, G Gasparoni PhD, S Pichler MSc, W Gu PhD); University Regensburg, Department of Psychiatry, Psychotherapy and Psychosomatics, Universitätsstr 84, Regensburg, Germany (B Ibach PhD); Luxembourg Centre For Systems Biomedicine (LCSB), University of Luxembourg, Luxembourg (W Gu); Dementia Research Centre, Department of Neurodegenerative Disease, UCL Institute of Neurology, Queen Square, London, UK (J D Rohrer, M N Rossor MD, N C Fox MD, J D Warren PhD); University of Cambridge, Department of Clinical Neurosciences, John Van Geest Brain Repair Centre, Cambridge, UK (M G Spillantini PhD); MRC Centre for Neuropsychiatric Genetics and Genomics, Cardiff University, School of Medicine, Cardiff, UK (H R Morris PhD); German Center of Neurodegenerative Diseases-Tübingen, Tübingen, Germany (P Rizzu PhD, P Heutink PhD); Salford Royal Foundation Trust, Faculty of Medical and Human Sciences, University of Manchester, UK (A Richardson MB); Institute of Brain, Behaviour and Mental Health, The University of Manchester, Withington, Manchester, UK (A Gerhard MD); Regional Neurogenetic Centre, ASPCZ, Lamezia Terme, Italy (A C Bruni MD, R Maletta MD, F Frangipane MD, C Cupidi MD, L Bernardi PhD, M Anfossi PhD, M Gallo PhD, M Elena Conidi PhD, N Smirne BSc); Department of Neuroscience (R Rademakers PhD, M Baker BSc, D W Dickson MD), Department of Neurology (N R Graff-Radford MD), Mayo Clinic Jacksonville, Jacksonville, FL, USA; Department of Neurology(R C Petersen MD, D Knopman MD, K A Josephs MD, B F Boeve MD), Department of Pathology (J E Parisi MD), Mayo Clinic Rochester, Rochester, MN USA; Department of Neurology (W W Seeley MD, B L Miller MD, A M Karydas BA, H Rosen MD), University of California, San Francisco, CA, USA; Department of Neurology, Erasmus Medical Centre, Rotterdam, The Netherlands (J C van Swieten MD, E G P Dopper MD, H Seelaar PhD); Department of Medical Genetics, VU university Medical Centre, Amsterdam, The Netherlands (J C van Swieten); Alzheimer Centre and Department of Neurology, VU University medical centre, Amsterdam, The Netherlands (Y A L Pijnenburg MD, P Scheltens MD); Department of Basic Medical Sciences, Neurosciences and Sense Organs of the Aldo Moro University of Bari, Italy (G Logroscino MD, R Capozzo MD); Department of Medical and Molecular Genetics, King<sup>1</sup>s College London, Guy’s Hospital, London, UK (A Ramasamy); Department of Molecular Cardiology, IRCCS Fondazione S Maugeri, Pavia, Italy (V Novelli PhD); Cardiovascular Research Unit, IRCCS Multimedica, Milan, Italy (An A Puca MD); Department of Medicine and Surgery, University of Salerno, Baronissi (SA), Italy (An A Puca); Neurology Department, IRCCS Multimedica, Milan, Italy (M Franceschi MD); Department of Clinical Medicine and Surgery, University of Naples Federico II, Naples, Italy (A Postiglione MD); Geriatric Center Frullone- ASL Napoli 1 Centro, Naples, Italy (G Milan MD, P Sorrentino MD); UCL Genomics, Institute of Child Health (ICH), UCL, London, UK (M Kristiansen PhD); Karolinska Institutet, Department NVS, KI-Alzheimer Disease Research Center, Stockholm, Sweden (H-H Chiang PhD, C Graff MD); Department of Geriatric Medicine, Genetics Unit, Karolinska Universtiy Hospital, Stockholm (H-H Chiang, C Graff); Université Lille Nord de France, Lille, France (F Pasquier MD, A Rollin MD, V Deramecourt MD, F Lebert MD); and National Institute on Aging (NIA/NIH),, Baltimore, MD, USA (D Kapogiannis MD)</div><div id="cor1">Correspondence to: Dr John Hardy, Reta Lila Weston Research Laboratories, Department of Molecular Neuroscience, University College London Institute of Neurology, London WC1N 3BG, UK, <a href="mailto:dev@null" data-email="ku.ca.lcu@ydrah.j" class="oemail">ku.ca.lcu@ydrah.j</a></div><div id="FN1"><sup>*</sup>Contributed equally</div><div id="FN2"><sup>†</sup>Joint last authors</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140164567936496_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=24943344" target="pmc_ext" ref="reftype=publisher&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Lancet Neurol</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4112126/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm140164637996784" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140164637996784title">Summary</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P3" class="p p-first-last">Frontotemporal dementia (FTD) is a complex disorder characterised by a broad range of clinical manifestations, differential pathological signatures, and genetic variability. Mutations in three genes—<em>MAPT, GRN,</em> and <em>C9orf72</em>—have been associated with FTD. We sought to identify novel genetic risk loci associated with the disorder.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P4" class="p p-first-last">We did a two-stage genome-wide association study on clinical FTD, analysing samples from 3526 patients with FTD and 9402 healthy controls. All participants had European ancestry. In the discovery phase (samples from 2154 patients with FTD and 4308 controls), we did separate association analyses for each FTD subtype (behavioural variant FTD, semantic dementia, progressive non-fluent aphasia, and FTD overlapping with motor neuron disease [FTD-MND]), followed by a meta-analysis of the entire dataset. We carried forward replication of the novel suggestive loci in an independent sample series (samples from 1372 patients and 5094 controls) and then did joint phase and brain expression and methylation quantitative trait loci analyses for the associated (p&lt;5 × 10<sup>−8</sup>) and suggestive single-nucleotide polymorphisms.</p></div><div id="S3" class="sec"><h3 id="S3title">Findings</h3><p id="P5" class="p p-first-last">We identified novel associations exceeding the genome-wide significance threshold (p&lt;5 × 10<sup>−8</sup>) that encompassed the <em>HLA</em> locus at 6p21.3 in the entire cohort. We also identified a potential novel locus at 11q14, encompassing <em>RAB38/CTSC</em>, for the behavioural FTD subtype. Analysis of expression and methylation quantitative trait loci data suggested that these loci might affect expression and methylation <em>incis</em>.</p></div><div id="S4" class="sec"><h3 id="S4title">Interpretation</h3><p id="P6" class="p p-first-last">Our findings suggest that immune system processes (link to 6p21.3) and possibly lysosomal and autophagy pathways (link to 11q14) are potentially involved in FTD. Our findings need to be replicated to better define the association of the newly identified loci with disease and possibly to shed light on the pathomechanisms contributing to FTD.</p></div><div id="S5" class="sec sec-last"><h3 id="S5title">Funding</h3><p id="P7" class="p p-first-last">The National Institute of Neurological Disorders and Stroke and National Institute on Aging, the Wellcome/ MRC Centre on Parkinson’s disease, Alzheimer’s Research UK, and Texas Tech University Health Sciences Center.</p></div></div></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Introduction</h2><p id="P8" class="p p-first">Frontotemporal dementia (FTD) is the second most common form of young-onset dementia after Alzheimer’s disease and comprises about 10–20% of all dementias worldwide.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775555">1</a></sup> FTD occurs in roughly three to 15 per 100 000 individuals aged between 55 years and 65 year<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775511">2</a></sup> The disease has an insidious onset: it is familial in 30–50% of patients and affects men and women almost equally.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775522">3</a></sup> The main clinical syndromes are the behavioural variant<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775550">1</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775533">4</a></sup> and the language variants (semantic dementia and progressive nonfluent aphasia).<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775500">1</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775509">5</a></sup> FTD also overlap with motor neuron disease (FTD-MND), and atypical parkinsonian disorders.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775536">3</a></sup> The molecular pathology is heterogeneous and based on the type of neuronal lesions and protein inclusions: 40% or more of patients have frontotemporal lobar degeneration (FTLD) with tau pathology (FTLD-tau), about 50% have TDP-43 (TAR DNA-binding protein 43) pathology (FTLD-TDP),<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775563">6</a></sup> and the remaining 10% have inclusions positive for fused in sarcoma (FUS; FTLD-FUS) or ubiquitin/p62 (FTLD-UPS [ubiquitin proteasome system]).<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775501">7</a></sup> Mutations in three main genes are commonly associated with FTD: the microtubule-associated protein tau (<em>MAPT</em>),<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775520">8</a></sup> granulin (<em>GRN</em>),<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775490">9</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775523">10</a></sup> and <em>C9orf72</em>.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775528">11</a>–<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775535">15</a></sup> Mutations in the charged multivesicular body protein 2B (<em>CHMP2B</em>), the valosin-containing protein (<em>VCP</em>), and ubiquilin 2 (<em>UBQLN2</em>) genes are rare causes of disease.<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a>,<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_483108613">16</a></sup> Findings from a previous genome-wide association study (GWAS) of neuropathologically confirmed FTLD-TDP (515 patients <em>vs</em> 2509) showed <em>TMEM106B</em> to be a disease risk factor.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775507">17</a></sup></p><p id="P9" class="p p-last">We did a larger GWAS in samples from people with clinical FTD, and we report results for the discovery, replication, and joint phase analyses, as well as for assessment of the effect on expression and methylation quantitative trait loci (QTL) exerted by associated or suggestive SNPs. We aimed to identify novel genetic risk loci associated with FTD and its subtypes.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Methods</h2><div id="S8" class="sec sec-first"><h3 id="S8title">Study population</h3><p id="P10" class="p p-first">44 international research groups (<a href="#FN3" rid="FN3" class=" fn">appendix</a>) contributed samples to this two-stage (discovery phase and replication phase) GWAS of clinical FTD. Investigators at every site obtained appropriate written informed consent from patients and control individuals. Every participating group provided consent for the use of these samples for the purposes of this study. The patients included in the discovery phase were diagnosed according to the Neary criteria<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775531">1</a></sup> for FTD, whereas those included in replication phase were diagnosed according to the Neary criteria,<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775486">1</a></sup> or the revised criteria for behavioural FTD<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775541">4</a></sup> and the language variants of FTD<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775557">5</a></sup> at every collaborative site. For each patient, the diagnosis was made by a neurologist with an interest in FTD or (the minority: &lt;5%) by pathological diagnosis. To cover the most relevant FTD clinical signatures, we included patients diagnosed with behavioural FTD, semantic dementia, progressive nonfluent aphasia, or FTD-MND.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775494">18</a></sup> We reviewed all patients with a diagnosis of language impairment to exclude cases of the logopenic variant of primary progressive aphasia,<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775489">5</a></sup> most of which are associated with Alzheimer’s disease pathology. Samples were obtained from North America (USA and Canada), UK, France, the Netherlands, Belgium, Germany, Denmark, Sweden, Spain, and Italy and all patients were of confirmed European ancestry.</p><p id="P11">DNA was collected at the three institutions leading this project: the Department of Molecular Neuroscience at University College London (UCL), UK; the Laboratory of Neurogenetics of the National Institute on Aging at the National Institutes of Health (NIH), MD, USA; and the Laboratory of Neurogenetics at the Texas Tech University Health Sciences Center (TTUHSC); TX, USA. All samples were anonymous and stored with a patientspecific coded identification number. Each DNA sample was assessed for quality with gel electrophoresis and DNA concentrations were assessed via spectrophotometer (Nanodrop; Wilmington, DE, USA) or fluorometer (Qubit; Life Technologies, Grand Island, NY, USA). Samples from non-overlapping patients were genotyped at the Laboratory of Neurogenetics of the National Institute on Aging, NIH (40%) or at the core facility at the Institute of Child Health, UCL (60%). We obtained standardised clinical, pathological, and genetic data for each patient from all the collaborating groups (<a href="#FN3" rid="FN3" class=" fn">appendix</a>). Sporadic cases along with probands from FTD families were included in the study.. We excluded carriers of mutations in <em>MAPT</em> and <em>GRN</em>. We did not exclude individuals with <em>C9orf72</em> expansions because this locus was identified subsequent to sample collection. After quality control of genotyping data and detailed assessment of the clinical diagnosis, we used 2154 and 1372 samples in the discovery phase and replication phase, respectively, for association analysis (<a href="/pmc/articles/PMC4112126/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>). In total, after quality control, we analysed 3526 FTD samples (<a href="/pmc/articles/PMC4112126/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>). Further details about cases included in the study are provided in the <a href="#FN3" rid="FN3" class=" fn">appendix</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P42">Sample characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140164557958736" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="top" align="left"></th><th colspan="3" rowspan="1" valign="top" align="left">Samples collected (n)</th><th colspan="3" rowspan="1" valign="top" align="left">Samples included in analysis (n)</th><th colspan="2" rowspan="1" valign="top" align="left">Samples from women (% [n/N])</th><th colspan="2" rowspan="1" valign="top" align="left">Mean age at onset</th></tr><tr><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left">Replication<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left">Total</th><th rowspan="1" colspan="1" valign="top" align="left">Discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left">Replication<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left">Total</th><th rowspan="1" colspan="1" valign="top" align="left">Discovery phase</th><th rowspan="1" colspan="1" valign="top" align="left">Replication phase</th><th rowspan="1" colspan="1" valign="top" align="left">Discovery phase</th><th rowspan="1" colspan="1" valign="top" align="left">Replication phase</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">USA</td><td rowspan="1" colspan="1" align="right">706</td><td rowspan="1" colspan="1" align="right">209</td><td rowspan="1" colspan="1" align="right">915</td><td rowspan="1" colspan="1" align="right">579</td><td rowspan="1" colspan="1" align="right">175</td><td rowspan="1" colspan="1" align="right">754</td><td rowspan="1" colspan="1" align="left">44% (257/579)</td><td rowspan="1" colspan="1" align="left">49% (85/174)</td><td rowspan="1" colspan="1" align="left">60 (23–85); 520</td><td rowspan="1" colspan="1" align="left">63 (24–93); 120</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Canada</td><td rowspan="1" colspan="1" align="right">25</td><td rowspan="1" colspan="1" align="right">37</td><td rowspan="1" colspan="1" align="right">62</td><td rowspan="1" colspan="1" align="right">24</td><td rowspan="1" colspan="1" align="right">29</td><td rowspan="1" colspan="1" align="right">53</td><td rowspan="1" colspan="1" align="left">52% (12/23)</td><td rowspan="1" colspan="1" align="left">57% (8/14)</td><td rowspan="1" colspan="1" align="left">64 (43–85); 15</td><td rowspan="1" colspan="1" align="left">59 (43–75); 9</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">UK</td><td rowspan="1" colspan="1" align="right">494</td><td rowspan="1" colspan="1" align="right">372</td><td rowspan="1" colspan="1" align="right">866</td><td rowspan="1" colspan="1" align="right">401</td><td rowspan="1" colspan="1" align="right">284</td><td rowspan="1" colspan="1" align="right">685</td><td rowspan="1" colspan="1" align="left">43% (171/400)</td><td rowspan="1" colspan="1" align="left">40% (108/272)</td><td rowspan="1" colspan="1" align="left">60 (23–83); 372</td><td rowspan="1" colspan="1" align="left">61 (35–86); 167</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Spain</td><td rowspan="1" colspan="1" align="right">100</td><td rowspan="1" colspan="1" align="right">330</td><td rowspan="1" colspan="1" align="right">430</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">309</td><td rowspan="1" colspan="1" align="right">309</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">43% (133/309)</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">65 (32–89); 308</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">France</td><td rowspan="1" colspan="1" align="right">238</td><td rowspan="1" colspan="1" align="right">54</td><td rowspan="1" colspan="1" align="right">292</td><td rowspan="1" colspan="1" align="right">205</td><td rowspan="1" colspan="1" align="right">42</td><td rowspan="1" colspan="1" align="right">247</td><td rowspan="1" colspan="1" align="left">44% (91/205)</td><td rowspan="1" colspan="1" align="left">48% (20/42)</td><td rowspan="1" colspan="1" align="left">62 (39–79); 190</td><td rowspan="1" colspan="1" align="left">NA</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Belgium</td><td rowspan="1" colspan="1" align="right">240</td><td rowspan="1" colspan="1" align="right">51</td><td rowspan="1" colspan="1" align="right">291</td><td rowspan="1" colspan="1" align="right">191</td><td rowspan="1" colspan="1" align="right">42</td><td rowspan="1" colspan="1" align="right">233</td><td rowspan="1" colspan="1" align="left">46% (88/191)</td><td rowspan="1" colspan="1" align="left">29% (12/42)</td><td rowspan="1" colspan="1" align="left">63 (29–90); 191</td><td rowspan="1" colspan="1" align="left">64 (43–84); 42</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Netherlands</td><td rowspan="1" colspan="1" align="right">333</td><td rowspan="1" colspan="1" align="right">93</td><td rowspan="1" colspan="1" align="right">426</td><td rowspan="1" colspan="1" align="right">250</td><td rowspan="1" colspan="1" align="right">77</td><td rowspan="1" colspan="1" align="right">327</td><td rowspan="1" colspan="1" align="left">52 (129/250)</td><td rowspan="1" colspan="1" align="left">40% (31/77)</td><td rowspan="1" colspan="1" align="left">58 (29–76); 250</td><td rowspan="1" colspan="1" align="left">61 (51–69); 59</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Denmark</td><td rowspan="1" colspan="1" align="right">35</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">35</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="left">71% (5/7)</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">57 (40–62); 7</td><td rowspan="1" colspan="1" align="left">NA</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Germany</td><td rowspan="1" colspan="1" align="right">349</td><td rowspan="1" colspan="1" align="right">34</td><td rowspan="1" colspan="1" align="right">383</td><td rowspan="1" colspan="1" align="right">320</td><td rowspan="1" colspan="1" align="right">33</td><td rowspan="1" colspan="1" align="right">353</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">50% (15/30)</td><td rowspan="1" colspan="1" align="left">61 (36–83); 243</td><td rowspan="1" colspan="1" align="left">57 (29–72); 30</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Sweden</td><td rowspan="1" colspan="1" align="right">26</td><td rowspan="1" colspan="1" align="right">112</td><td rowspan="1" colspan="1" align="right">138</td><td rowspan="1" colspan="1" align="right">18</td><td rowspan="1" colspan="1" align="right">98</td><td rowspan="1" colspan="1" align="right">116</td><td rowspan="1" colspan="1" align="left">56% (10/18)</td><td rowspan="1" colspan="1" align="left">61% (60/98)</td><td rowspan="1" colspan="1" align="left">57 (38–75); 16</td><td rowspan="1" colspan="1" align="left">62 (28–78); 93</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Italy</td><td rowspan="1" colspan="1" align="right">1035</td><td rowspan="1" colspan="1" align="right">563</td><td rowspan="1" colspan="1" align="right">1598</td><td rowspan="1" colspan="1" align="right">564</td><td rowspan="1" colspan="1" align="right">371</td><td rowspan="1" colspan="1" align="right">935</td><td rowspan="1" colspan="1" align="left">53% (297/561)</td><td rowspan="1" colspan="1" align="left">45% (168/371)</td><td rowspan="1" colspan="1" align="left">64 (31–83); 429</td><td rowspan="1" colspan="1" align="left">65 (31–87); 353</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Australia</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">138</td><td rowspan="1" colspan="1" align="right">138</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">121</td><td rowspan="1" colspan="1" align="right">121</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">36% (44/121)</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">59 (32–77); 112</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Meta-analysis</td><td rowspan="1" colspan="1" align="right">3581</td><td rowspan="1" colspan="1" align="right">1993</td><td rowspan="1" colspan="1" align="right">5574</td><td rowspan="1" colspan="1" align="right">2559 (2154<sup>*</sup>)</td><td rowspan="1" colspan="1" align="right">1581 (1372<sup>*</sup>)</td><td rowspan="1" colspan="1" align="right">4140 (3526<sup>*</sup>)</td><td rowspan="1" colspan="1" align="left">47% (1186/2552)</td><td rowspan="1" colspan="1" align="left">44% (684/1550)</td><td rowspan="1" colspan="1" align="left">61 (23–90); 2233</td><td rowspan="1" colspan="1" align="left">62 (24–93); 1293</td></tr></tbody></table></div><div id="largeobj_idm140164557958736" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4112126/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P43" class="p p-first-last">NA=.</p></div><div id="TFN2"><sup>*</sup>The number of the samples that passed genotyping data quality control and were used for association analyses.</div></div></div><p id="P12" class="p p-last">Control samples for the discovery phase were taken from studies previously done at the Laboratory of Neurogenetics of the National Institute on Aging at the NIH or at UCL. Control individuals were matched to patients on the basis of population ancestry and genotyping platform. Aggregate data for control samples were merged based on overlapping single-nucleotide polymorphisms (SNPs). The selected 7444 control samples were from the USA, UK, Italy, Germany, France, Sweden, and the Netherlands, and were used as controls in previous GWAS;<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775527">19</a></sup> all individuals had given consent for their samples to be used as controls. All were free of neurological illness at the time of sampling, but most had not been screened for the absence of a family history of FTD. For each patient, at least two controls were matched based on compatibility of genetic ancestry estimates by principal components analysis to accommodate the lack of precisely matched clinical controls. After quality control, we included 4308 control samples in this study. The genotyping of controls for the replication phase was done at the Laboratory of Neurogenetics of the National Institute on Aging, NIH (90%) and at the core facility at the Institute of Child Health, UCL (10%). All control samples used in the replication phase were collected from the groups participating in the study (5094 samples passed quality control) and were of European ancestry from the following countries: USA (European American), UK, Italy, France, Germany, Sweden, Spain, and the Netherlands.</p></div><div id="S9" class="sec"><h3 id="S9title">Procedures</h3><p id="P13" class="p p-first">For every sample, 2 µg of DNA extracted from either blood or the brain at each collaborative site was collected (whole genome amplified DNA samples were excluded). Samples were securely stored at −20°C. Every sample was first screened for integrity and purity by means of gel electrophoresis on 1% agarose gel, and concentrations were analysed by spectrophotometric (Nanodrop) or fluorometric (Qubit) quantification. The same procedure was implemented at NIH, UCL, and TTUHSC.</p><p id="P14">Samples from patients and control individuals included in the discovery phase were genotyped using Illumina human 370K, 550K, and 660K Quad Beadchips and Omni Express chips (Illumina Inc, CA, USA). We used Illumina NeuroX custom chips for all samples included in replication phase genotyping. The NeuroX chip is a partially custom-designed chip that specifically targets the main loci associated with several different neurological disorders obtained from GWAS or whole-exome sequencing data. The NeuroX chip holds about 267K SNPs, of which 3759 were FTD-specific, being selected from SNPs that had p values of less than 1 × 10<sup>−4</sup> during the discovery phase of the study. These SNPs were tag SNPs based on European ancestry linkage-disequilibrium patterns from the most up-to-date data for samples of European ancestry from the 1000 Genomes project.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775504">20</a></sup> For all GWAS significant hits and candidate SNPs, five linkage-disequilibrium-based proxies or technical replicates were included on the array per locus, tagging associations within +/−250 kb and <em>r</em><sup>2</sup> &gt;0·5 from the most strongly associated proximal SNP. To replicate each locus, we picked the tag SNP most significant in the discovery phase before beginning. If no linkage-disequilibrium-based proxies were available, technical replicates were included. All genotyping arrays (discovery phase and replication phase) were assayed on the Illumina Infinium platform (Illumina, San Diego, CA, USA) at the Laboratory of Neurogenetics of the National Institute on Aging, NIH and at the core facility at the Institute of Child Health, UCL. All genotypes for this project were called centrally using Illumina Genome Studio and all 3759 SNPs of interest for FTD were manually examined to ensure high-quality genotype clusters before data export.</p><p id="P15" class="p p-last">For the purpose of assessing possible biological relevance for any associated SNPs we used quantitative trait loci (QTL) data generated by the UK Brain Expression Consortium (UKBEC) and the North American Brain Expression Consortium (NABEC) for brain tissues assayed for genome-wide expression and methylation. Details about sample collection, RNA/DNA extraction, and genotyping are provided in the <a href="#FN3" rid="FN3" class=" fn">appendix</a>.</p></div><div id="S10" class="sec"><h3 id="S10title">Statistical analysis</h3><p id="P16" class="p p-first">We did standard quality control for GWAS data before association analyses. Briefly, for the discovery phase, we extracted overlapping SNPs across all Illumina arrays used. This was done as a means of dealing with the low numbers of matched cases and controls per study site or chip type to facilitate the FTD subtype analyses. We maximised sample size for the subtype analyses by pooling as many possible samples while sacrificing some array content, leaving 228 189 autosomal SNPs as a basis for imputation after the quality control was completed. We excluded samples possibly mismatched for sex by assessing X chromosome heterozygosity. Samples with a call rate of greater than 95% and SNPs with a minor allele frequency greater than 1% were filtered and included in the analyses. We calculated Hardy-Weinberg equilibrium p values (exclusion at p values &lt;1 × 10<sup>−5</sup>). We assessed non-random missingness per SNP by case-control status with exclusion at p values of less than 1 × 10<sup>−5</sup> and non-random missingness per SNP by haplotype at p values for exclusion &lt;1 × 10<sup>−5</sup>. We assessed the presence of relatedness by identifying and excluding first-degree relatives (through identity by descent for any pairwise with an estimate of less than 0·125) and verified European ancestry by principal components analysis compared with HapMap3 populations, with European ancestry ascertained at values for the first two eigenvectors less than six SDs from the population mean for the combined Europeans from Utah and Tuscans from Italy reference samples.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775515">21</a></sup> After preliminary quality assessment, principal components analysis as implemented in EIGENSTRAT<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775549">22</a></sup> was used to assess matching between cases and controls based on all available cases and controls. Custom coding in R was used to match cases to controls. We treated each subtype (behavioural FTD, semantic dementia, progressive nonfluent aphasia, and FTD-MND) as a separate group in which the two most genetically similar unique controls per case were selected based on eigenvectors 1 and 2 in order to compensate for a lack of precisely matched controls at recruitment. In this respect, matched controls were unique per case and non-redundant across subtype datasets. Thus, cases and controls were matched for each subtype (behavioural FTD, semantic dementia, progressive nonfluent aphasia, and FTD-MND) based on similarity of the first two eigenvectors from principal components analysis and did not overlap across subtypes. We used logistic regression based on imputed dosages to assess the association between each SNP and any of the FTD subtypes, adjusting for eigenvectors 1 and 2 from principal components analysis as covariates. Eigenvectors were generated separately for each subtype, as in the overall sample pool, parameter estimates for the first two were associated with case status at p values of less than 0·05. We did fixed-effects meta-analyses to combine results across subtypes and quantify heterogeneity across subtypes. Genomic inflation was minimal across subtypes and in the meta analysis across subtypes (λ&lt;1·05), therefore we did not use genomic control (see <a href="#FN3" rid="FN3" class=" fn">appendix</a> for quantile-quantile plots and λ values per discovery phase analysis). SNPs were imputed to August, 2010 release of the 1000 Genomes haplotypes using default settings of minimac and were excluded if their minor allele frequency was less than 0·01 or imputation quality (Rsq) was less than 0·30 across all samples, leaving 6 026 385 SNPs for analyses.</p><p id="P17" class="p p-last">For the replication phase, we did standard quality control as for the discovery phase with slight adjustments to account for the bias in NeuroX array content (candidate neurological or neurodegenerative disease SNPs and exonic content). Standard content variants included on the NeuroX array that were used for sample quality control were called using a publicly available cluster file based on more than 60 000 samples.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775556">23</a></sup> For quality control, variants with GenTrain scores greater than 0·70 (indicative of high-quality genotype clusters) were extracted first to calculate call rates. Samples with call rates greater than 95% were excluded, as were samples whose genetically determined sex conflicted with that from the clinical data and samples exhibiting excess heterozygosity. Next, SNPs overlapping with HapMap phase 3 samples were extracted from the previous subset and pruned for linkage disequilibrium (SNPs excluded if <em>r</em><sup>2</sup> &gt;0·50 within a 50 SNP sliding window), and SNPs with minor allele frequency less than 5%, Hardy-Weinberg equilibrium p values less than 1 × 10<sup>−5</sup>, and per SNP missingness rates greater than 5%. At this stage, we used pairwise identity-by-descent filtering to remove samples that were cryptically related and principal components analysis to identify samples to be excluded when genetic ancestry was not consistent with European descent based on comparisons with HapMap phase 3 reference populations. For replication analyses and due to an effort to maximise the restricted power of this phase compared to the discovery phase, analyses of each subtype included all control samples available, adjusting for the first five eigenvectors only from principal components analysis as covariates in the logistic regression model. No other adjustments were implemented. Additionally, we pooled the individual genotypes from different subsets in the replication phase to help increase statistical power. For details about QTL statistical analysis, see <a href="#FN3" rid="FN3" class=" fn">appendix</a>.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Role of the funding source</h3><p id="P18" class="p p-first-last">The sponsors of the study had no role in study design, data collection, data analysis, data interpretation, or writing of the report. No pharmaceutical company or other agency paid to write this article. The corresponding author had full access to all the data in the study and had final responsibility for the decision to submit for publication.</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Results</h2><p id="P19" class="p p-first">In the discovery phase, we analysed samples from 2154 patients (<a href="/pmc/articles/PMC4112126/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>) and 4308 controls. We first did separate association analyses for each subtype (behavioural FTD, semantic dementia, progressive nonfluent aphasia, and FTD-MND; <a href="/pmc/articles/PMC4112126/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>) and then undertook a meta-analysis of the entire dataset. Findings from the meta-analysis showed 29 SNPs (<a href="#FN3" rid="FN3" class=" fn">appendix</a>) exceeding genome-wide significance (p value &lt;5×10<sup>−8</sup>) at the <em>HLA</em> locus (6p21.3), encompassing the butyrophilinlike 2 (MHC class II associated) gene (<em>BTNL2</em>) and the major histocompatibility complex, class II, DR alpha (<em>HLA-DRA</em>), and DR beta 5 (<em>HLA-DRB5</em>; figure, <a href="/pmc/articles/PMC4112126/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>table 3</span></a>). To identify susceptibility loci for the behavioural FTD subtype we analysed 1377 patient samples (<a href="/pmc/articles/PMC4112126/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>) and 2754 control samples. Two non-coding SNPs at 11q14, locating to intron 1 of the gene <em>RAB38</em>, member RAS oncogene family (<em>RAB38</em>; rs302652) and encompassing <em>RAB38</em> and cathepsin C (<em>CTSC</em>; rs74977128), passed the genome-wide significance threshold (figure, <a href="/pmc/articles/PMC4112126/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>table 3</span></a>). Similarly, we did analyses on the other subtypes (<a href="/pmc/articles/PMC4112126/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>): 308 semantic dementia versus 616 controls, 269 progressive nonfluent aphasia versus 538 controls, and 200 FTD-MND versus 400 controls. No SNP reached genome-wide significance in either subtype, probably due to the small sample size. However, several SNPs (<a href="#FN3" rid="FN3" class=" fn">appendix</a>) showed suggestive associations (p values between 1 × 10<sup>−6</sup> and 1 × 10<sup>−7</sup>; figure) and warrant further investigation in future screenings.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P45">Sample characteristics, by subtype</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140164558258624" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="top" align="left"></th><th colspan="3" rowspan="1" valign="top" align="left">Behavioural frontotemporal<br>dementia</th><th colspan="3" rowspan="1" valign="top" align="left">Semantic dementia</th><th colspan="3" rowspan="1" valign="top" align="left">Progressive nonfluent aphasia</th><th colspan="3" rowspan="1" valign="top" align="left">Frontotemporal dementia/<br>myotrophic lateral sclerosis</th><th colspan="3" rowspan="1" valign="top" align="left">FTLD-U</th></tr><tr><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="right">Discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="right">Replication<br>phase</th><th rowspan="1" colspan="1" valign="top" align="right">Total</th><th rowspan="1" colspan="1" valign="top" align="right">Discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="right">Replication<br>phase</th><th rowspan="1" colspan="1" valign="top" align="right">Total</th><th rowspan="1" colspan="1" valign="top" align="right">Discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="right">Replication<br>phase</th><th rowspan="1" colspan="1" valign="top" align="right">Total</th><th rowspan="1" colspan="1" valign="top" align="right">Discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="right">Replication<br>phase</th><th rowspan="1" colspan="1" valign="top" align="right">Total</th><th rowspan="1" colspan="1" valign="top" align="left">Discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="right">Replication<br>phase</th><th rowspan="1" colspan="1" valign="top" align="right">Total</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">USA</td><td rowspan="1" colspan="1" align="right">315</td><td rowspan="1" colspan="1" align="right">25</td><td rowspan="1" colspan="1" align="right">340</td><td rowspan="1" colspan="1" align="right">147</td><td rowspan="1" colspan="1" align="right">12</td><td rowspan="1" colspan="1" align="right">159</td><td rowspan="1" colspan="1" align="right">81</td><td rowspan="1" colspan="1" align="right">15</td><td rowspan="1" colspan="1" align="right">96</td><td rowspan="1" colspan="1" align="right">36</td><td rowspan="1" colspan="1" align="right">21</td><td rowspan="1" colspan="1" align="right">57</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="right">102</td><td rowspan="1" colspan="1" align="right">102</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Canada</td><td rowspan="1" colspan="1" align="right">22</td><td rowspan="1" colspan="1" align="right">5</td><td rowspan="1" colspan="1" align="right">27</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">5</td><td rowspan="1" colspan="1" align="right">5</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="right">11</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">UK<sup>*</sup></td><td rowspan="1" colspan="1" align="right">207</td><td rowspan="1" colspan="1" align="right">152</td><td rowspan="1" colspan="1" align="right">359</td><td rowspan="1" colspan="1" align="right">75</td><td rowspan="1" colspan="1" align="right">53</td><td rowspan="1" colspan="1" align="right">128</td><td rowspan="1" colspan="1" align="right">69</td><td rowspan="1" colspan="1" align="right">44</td><td rowspan="1" colspan="1" align="right">113</td><td rowspan="1" colspan="1" align="right">50</td><td rowspan="1" colspan="1" align="right">16</td><td rowspan="1" colspan="1" align="right">66</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="right">19</td><td rowspan="1" colspan="1" align="right">19</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Spain</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">194</td><td rowspan="1" colspan="1" align="right">194</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">41</td><td rowspan="1" colspan="1" align="right">41</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">51</td><td rowspan="1" colspan="1" align="right">51</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">13</td><td rowspan="1" colspan="1" align="right">13</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="right">10</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">France</td><td rowspan="1" colspan="1" align="right">135</td><td rowspan="1" colspan="1" align="right">30</td><td rowspan="1" colspan="1" align="right">165</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="right">59</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">67</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">1</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Belgium</td><td rowspan="1" colspan="1" align="right">135</td><td rowspan="1" colspan="1" align="right">27</td><td rowspan="1" colspan="1" align="right">162</td><td rowspan="1" colspan="1" align="right">13</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">14</td><td rowspan="1" colspan="1" align="right">22</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">24</td><td rowspan="1" colspan="1" align="right">21</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">23</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="right">10</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Netherlands</td><td rowspan="1" colspan="1" align="right">159</td><td rowspan="1" colspan="1" align="right">37</td><td rowspan="1" colspan="1" align="right">196</td><td rowspan="1" colspan="1" align="right">47</td><td rowspan="1" colspan="1" align="right">31</td><td rowspan="1" colspan="1" align="right">78</td><td rowspan="1" colspan="1" align="right">24</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">30</td><td rowspan="1" colspan="1" align="right">20</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">23</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">0</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Denmark</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">0</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Germany</td><td rowspan="1" colspan="1" align="right">209</td><td rowspan="1" colspan="1" align="right">18</td><td rowspan="1" colspan="1" align="right">227</td><td rowspan="1" colspan="1" align="right">45</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">53</td><td rowspan="1" colspan="1" align="right">55</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">61</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">12</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">0</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Sweden</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">53</td><td rowspan="1" colspan="1" align="right">60</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">20</td><td rowspan="1" colspan="1" align="right">22</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="right">16</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">7</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Italy</td><td rowspan="1" colspan="1" align="right">443</td><td rowspan="1" colspan="1" align="right">186</td><td rowspan="1" colspan="1" align="right">629</td><td rowspan="1" colspan="1" align="right">28</td><td rowspan="1" colspan="1" align="right">22</td><td rowspan="1" colspan="1" align="right">50</td><td rowspan="1" colspan="1" align="right">69</td><td rowspan="1" colspan="1" align="right">86</td><td rowspan="1" colspan="1" align="right">155</td><td rowspan="1" colspan="1" align="right">24</td><td rowspan="1" colspan="1" align="right">16</td><td rowspan="1" colspan="1" align="right">40</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="right">61</td><td rowspan="1" colspan="1" align="right">61</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Australia<sup>*</sup></td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">56</td><td rowspan="1" colspan="1" align="right">56</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">26</td><td rowspan="1" colspan="1" align="right">26</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">19</td><td rowspan="1" colspan="1" align="right">19</td><td rowspan="1" colspan="1" align="right">NA</td><td rowspan="1" colspan="1" align="right">20</td><td rowspan="1" colspan="1" align="right">20</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">0</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Meta-analysis</td><td rowspan="1" colspan="1" align="right">1634<br>(1377<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">783<br>(690<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">2417<br>(2061<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">361<br>(308<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">215<br>(190<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">576<br>(495<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">335<br>(269<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">247<br>(221<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">582<br>(486<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">229<br>(200<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">115<br>(94<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">344<br>(294<sup>†</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" align="right">221<br>(177<sup>†</sup>)</td><td rowspan="1" colspan="1" align="right">221<br>(177<sup>†</sup>)</td></tr></tbody></table></div><div id="largeobj_idm140164558258624" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4112126/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="P46" class="p p-first-last">NA=.</p></div><div id="TFN4"><sup>*</sup>Used the same control samples.</div><div id="TFN5"><sup>†</sup>The number of the samples that passed genotyping data quality control and were used for association analyses.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="P49">Characteristics of single-nucleotide polymorphisms exceeding genome-wide significance in the discovery phase</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140164557896112" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" valign="top" align="right">Chromosome</th><th rowspan="1" colspan="1" valign="top" align="left">Base pair</th><th rowspan="1" colspan="1" valign="top" align="left">Candidate gene</th><th rowspan="1" colspan="1" valign="top" align="left">Minor<br>allele</th><th rowspan="1" colspan="1" valign="top" align="left">Major<br>allele</th><th rowspan="1" colspan="1" align="left">Frequency of<br>minor allele<br>(<em>r</em><sup>2</sup> when<br>applicable)</th><th rowspan="1" colspan="1" valign="top" align="left">Imputation<br>quality</th><th rowspan="1" colspan="1" valign="top" align="left">Odds ratio (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="left">Standard<br>error</th><th rowspan="1" colspan="1" valign="top" align="left">p value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"><strong>Discovery phase</strong></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left">Behavioural frontotemporal dementia</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs302652</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="left">87894831</td><td rowspan="1" colspan="1" align="left"><em>RAB38</em></td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">T</td><td rowspan="1" colspan="1" align="left">0·259</td><td rowspan="1" colspan="1" align="left">0·9296</td><td rowspan="1" colspan="1" align="left">0·730 (0·65–0·82)</td><td rowspan="1" colspan="1" align="left">0·057</td><td rowspan="1" colspan="1" align="left">2·02×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs74977128</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="left">87936874</td><td rowspan="1" colspan="1" align="left"><em>RAB38/CTSC</em></td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">T</td><td rowspan="1" colspan="1" align="left">0·118</td><td rowspan="1" colspan="1" align="left">0·4182</td><td rowspan="1" colspan="1" align="left">1·815 (1·48–2·24)</td><td rowspan="1" colspan="1" align="left">0·107</td><td rowspan="1" colspan="1" align="left">3·06×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">All frontotemporal dementia<sup>*</sup></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs9268877</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="left">32431147</td><td rowspan="1" colspan="1" align="left"><em>HLA-DRA/HLA-DRB5</em></td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">0·440</td><td rowspan="1" colspan="1" align="left">0·7783</td><td rowspan="1" colspan="1" align="left">1·331 (1·22–1·45)</td><td rowspan="1" colspan="1" align="left">0·045</td><td rowspan="1" colspan="1" align="left">1·65×10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs9268856</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="left">32429719</td><td rowspan="1" colspan="1" align="left"><em>HLA-DRA/HLA-DRB5</em></td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">0·251</td><td rowspan="1" colspan="1" align="left">0·8563</td><td rowspan="1" colspan="1" align="left">0·752 (0·68–0·83)</td><td rowspan="1" colspan="1" align="left">0·050</td><td rowspan="1" colspan="1" align="left">1·30×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs1980493</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="left">32363215</td><td rowspan="1" colspan="1" align="left"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">T</td><td rowspan="1" colspan="1" align="left">0·147</td><td rowspan="1" colspan="1" align="left">0·9642</td><td rowspan="1" colspan="1" align="left">0·720 (0·69–0·81)</td><td rowspan="1" colspan="1" align="left">0·060</td><td rowspan="1" colspan="1" align="left">4·94×10<sup>−8</sup></td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Replication phase</strong></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left">Behavioural frontotemporal dementia</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs302668 (proxy)</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="left">87876911</td><td rowspan="1" colspan="1" align="left"><em>RAB38</em></td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">T</td><td rowspan="1" colspan="1" align="left">0·325 (0·65)</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">0·877 (0.77–0.99)</td><td rowspan="1" colspan="1" align="left">0·064</td><td rowspan="1" colspan="1" align="left">0·041</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs16913634 (proxy)</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="left">87934068</td><td rowspan="1" colspan="1" align="left"><em>RAB38/CTSC</em></td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">0·104 (0·54)</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">0·964 (0.79–1.17)</td><td rowspan="1" colspan="1" align="left">0·098</td><td rowspan="1" colspan="1" align="left">0·710</td></tr><tr><td rowspan="1" colspan="1" align="left">All frontotemporal dementia<sup>*</sup></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs9268877</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="left">32431147</td><td rowspan="1" colspan="1" align="left"><em>HLA-DRA/HLA-DRB5</em></td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">0·449</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">1·080 (0·98–1·18)</td><td rowspan="1" colspan="1" align="left">0·047</td><td rowspan="1" colspan="1" align="left">0·104</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs9268856</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="left">32429719</td><td rowspan="1" colspan="1" align="left"><em>HLA-DRA/HLA-DRB5</em></td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">0·253</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">0·878 (0·79–0·97)</td><td rowspan="1" colspan="1" align="left">0·053</td><td rowspan="1" colspan="1" align="left">0·014</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs1980493</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="left">32363215</td><td rowspan="1" colspan="1" align="left"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">T</td><td rowspan="1" colspan="1" align="left">0·145</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">0.85 (0·75–0·97)</td><td rowspan="1" colspan="1" align="left">0·068</td><td rowspan="1" colspan="1" align="left">0·020</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"><strong>Discovery and replication combined</strong></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left">Behavioural frontotemporal dementia</td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs302668 (proxy)</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="left">87876911</td><td rowspan="1" colspan="1" align="left"><em>RAB38</em></td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">T</td><td rowspan="1" colspan="1" align="left">0·292 (0·65)</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">0·814 (0·71–0·92)</td><td rowspan="1" colspan="1" align="left">0·064</td><td rowspan="1" colspan="1" align="left">2·44×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs16913634 (proxy)<sup>†</sup></td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="left">87934068</td><td rowspan="1" colspan="1" align="left"><em>RAB38/CTSC</em></td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">0·111 (0·54)</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">1·248 (1·14–1·37)</td><td rowspan="1" colspan="1" align="left">0·049</td><td rowspan="1" colspan="1" align="left">8·15×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">All frontotemporal dementia<sup>*</sup></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs9268877<sup>†</sup></td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="left">32431147</td><td rowspan="1" colspan="1" align="left"><em>HLA-DRA/HLA-DRB5</em></td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">0·4445</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">1·204 (1·11–1·30)</td><td rowspan="1" colspan="1" align="left">0·039</td><td rowspan="1" colspan="1" align="left">1·05×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs9268856</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="left">32429719</td><td rowspan="1" colspan="1" align="left"><em>HLA-DRA/HLA-DRB5</em></td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">0·252</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">0·809 (0·76–0·86)</td><td rowspan="1" colspan="1" align="left">0·029</td><td rowspan="1" colspan="1" align="left">5·51×10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs1980493</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="left">32363215</td><td rowspan="1" colspan="1" align="left"><em>BTNL2</em></td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">T</td><td rowspan="1" colspan="1" align="left">0·146</td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">0·775 (0·69–0·86)</td><td rowspan="1" colspan="1" align="left">0·058</td><td rowspan="1" colspan="1" align="left">1·57×10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm140164557896112" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4112126/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><p id="P50" class="p p-first-last">Replication and joint analyses were assessed for the same single-nucleotide polymorphisms (SNPs) at 6p21.3, whereas proxy SNPs were used to assess the association at 11q14 (for which <em>r</em><sup>2</sup> values are included). The odds ratio is shown for the minor allele.</p></div><div id="TFN7"><sup>†</sup>NA=.</div><div id="TFN8"><sup>*</sup>Denotes only minimal cross-subtype heterogeneity, with heterogeneity p values ranging from 0·793 to 0·944 based on Cochran’s Q test.</div><div id="TFN9"><sup>†</sup>Heterogeneity p value &lt;0·01 in the meta-analyses of the discovery and replication phases combined.</div></div></div><p id="P20">In the replication phase, we analysed samples from 1372 patients (<a href="/pmc/articles/PMC4112126/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>) and 5094 controls. We assessed the associated SNPs at 6p21.3 (rs9268877, rs9268856, and rs1980493) in the whole replication cohort (<a href="/pmc/articles/PMC4112126/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>table 3</span></a>). <a href="/pmc/articles/PMC4112126/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> shows findings from the surrogate or proxy SNPs assessed for replication at 11q14 in 690 behavioural FTD cases: rs302668 and rs16913634. Combined analyses of discovery and replication phases showed genome-wide significant association at 6p21.3 for all SNPs (<a href="/pmc/articles/PMC4112126/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>table 3</span></a>). Joint p values of the SNPs at 11q14 only revealed suggestive association for rs302668 (<a href="/pmc/articles/PMC4112126/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>table 3</span></a>) possibly because of decreased power due to proxy-based replication (<em>r</em><sup>2</sup> of rs302652 to rs302668=0·65).</p><p id="P21">We then assessed biological relevance for the novel potential loci in human brain cortex tissues assayed for genome-wide expression and methylation. There was no eQTL in our dataset, but assessment of Zeller and colleagues’ dataset<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775521">38</a></sup> showed a <em>cis</em>-eQTL (p=5·05 × 10<sup>−32</sup>; <a href="#FN3" rid="FN3" class=" fn">appendix</a>) at 11q14 for rs302652 (chr11:87894881, risk allele T) causing a decreased expression of <em>RAB38</em> (Illumina ILMN_2134974 located on chr11:87846656-87846705) in monocytes. These data suggest a role in transcriptional processes <em>in cis</em> for this SNP. Furthermore, we identified significant <em>cis</em>-mQTL at 6p21.3 after multiple test correction for rs1980493 (risk allele T) that associated with changes in the methylation levels related to <em>HLADRA</em> in the frontal cortex (<a href="/pmc/articles/PMC4112126/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>table 4</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="P54">Summary of association of top hits with <em>cis</em>-methylation levels at 6p21.3</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140164552881296" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">CpG probe</th><th rowspan="1" colspan="1" valign="top" align="left">Single-<br>nucleotide<br>polymorphism</th><th rowspan="1" colspan="1" valign="top" align="left">Chromosome</th><th rowspan="1" colspan="1" valign="top" align="left">Position<br>(base pair)</th><th rowspan="1" colspan="1" valign="top" align="left">Reference<br>allele</th><th rowspan="1" colspan="1" valign="top" align="left">Alternate<br>allele</th><th rowspan="1" colspan="1" valign="top" align="left">Frequency<br>of<br>reference<br>allele</th><th rowspan="1" colspan="1" valign="top" align="left">Imputation<br>quality</th><th rowspan="1" colspan="1" align="left">Effect<br>estimate of<br>alternate<br>allele (in<br><em>Z</em> units)</th><th rowspan="1" colspan="1" valign="top" align="left">Standard<br>error</th><th rowspan="1" colspan="1" valign="top" align="left">p value</th><th rowspan="1" colspan="1" valign="top" align="left">FDR adjsuted<br>p value</th><th rowspan="1" colspan="1" valign="top" align="left">Probe start<br>(base pair)</th><th rowspan="1" colspan="1" valign="top" align="left">Probe end<br>(base pair)</th><th rowspan="1" colspan="1" valign="top" align="left">Symbol</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Frontal cortex<br>(CpG<br>methylation)</td><td rowspan="1" colspan="1" valign="top" align="left">cg21415604</td><td rowspan="1" colspan="1" valign="top" align="left">rs1980493</td><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">32363215</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·8361</td><td rowspan="1" colspan="1" valign="top" align="left">0·9888</td><td rowspan="1" colspan="1" valign="top" align="left">−0·463</td><td rowspan="1" colspan="1" valign="top" align="left">0·116</td><td rowspan="1" colspan="1" valign="top" align="left">0·0000701</td><td rowspan="1" colspan="1" valign="top" align="left">0·00834666</td><td rowspan="1" colspan="1" valign="top" align="left">31948433</td><td rowspan="1" colspan="1" valign="top" align="left">31948483</td><td rowspan="1" colspan="1" valign="top" align="left">C4B</td></tr><tr><td rowspan="1" colspan="1" align="left">Frontal cortex<br>(CpG<br>methylation)</td><td rowspan="1" colspan="1" valign="top" align="left">cg25764570</td><td rowspan="1" colspan="1" valign="top" align="left">rs1980493</td><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">32363215</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·8361</td><td rowspan="1" colspan="1" valign="top" align="left">0·9888</td><td rowspan="1" colspan="1" valign="top" align="left">−0·652</td><td rowspan="1" colspan="1" valign="top" align="left">0·116</td><td rowspan="1" colspan="1" valign="top" align="left">2·17×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·00000773</td><td rowspan="1" colspan="1" valign="top" align="left">32407239</td><td rowspan="1" colspan="1" valign="top" align="left">32407289</td><td rowspan="1" colspan="1" valign="top" align="left">HLA-DRA</td></tr><tr><td rowspan="1" colspan="1" align="left">Frontal cortex<br>(CpG<br>methylation)</td><td rowspan="1" colspan="1" valign="top" align="left">cg25764570</td><td rowspan="1" colspan="1" valign="top" align="left">rs9268856</td><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">32429719</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0·748</td><td rowspan="1" colspan="1" valign="top" align="left">0·9687</td><td rowspan="1" colspan="1" valign="top" align="left">−0·484</td><td rowspan="1" colspan="1" valign="top" align="left">0·1</td><td rowspan="1" colspan="1" valign="top" align="left">1·16×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·000207417</td><td rowspan="1" colspan="1" valign="top" align="left">32407239</td><td rowspan="1" colspan="1" valign="top" align="left">32407289</td><td rowspan="1" colspan="1" valign="top" align="left">HLA-DRA</td></tr></tbody></table></div><div id="largeobj_idm140164552881296" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4112126/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN10"><p id="P55" class="p p-first-last">Association is shown for rs1980493 and rs9268877, which indicates an involvement of methylation processes and patterns in relation to <em>HLA-DRA</em>.</p></div></div></div><p id="P22">To assess potential genetic overlap between FTD and closely related forms of neurodegenerative diseases we selected relevant SNPs for candidate loci and analysed them in our dataset. This analysis included published association studies for amyotrophic lateral sclerosis,<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775498">39</a></sup> progressive supranuclear palsy (PSP) and corticobasal degeneration (CBD),<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775506">40</a></sup> Alzheimer’s disease,<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775487">41</a></sup> and FTLD-TDP.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775551">17</a></sup> We also assessed whether the two loci identified through this study had also been reported previously in other studies of neurological disorders.</p><p id="P23">For the <em>C9orf72</em> locus (for amyotrophic lateral sclerosis), the SNP rs3849942 (effect allele A) was associated with the FTD-MND subtype, which was consistent with our post-hoc analyses (about 23% of expansion carriers in this subtype; <a href="/pmc/articles/PMC4112126/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>table 5</span></a>; <a href="#FN3" rid="FN3" class=" fn">appendix</a>). Association was modest in behavioural FTD (p=7·38×10<sup>−3</sup>, OR=1·155) as well as in the entire discovery cohort, but we saw no evidence for association in the semantic dementia or progressive nonfluent aphasia subtypes (<a href="/pmc/articles/PMC4112126/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>table 5</span></a>). These results confirm that the <em>C9orf72</em> locus associates mainly with FTD-MND and to a lesser extent with behavioural FTD (<a href="#FN3" rid="FN3" class=" fn">appendix</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="P56">Comparison with single-nucleotide polymorphisms and loci associated with other disorders</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140164555094528" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="5" colspan="1" valign="top" align="left"></th><th rowspan="5" colspan="1" valign="top" align="left">Reference<br>allele</th><th rowspan="5" colspan="1" valign="top" align="left">Alternate<br>allele</th><th colspan="3" rowspan="1" valign="top" align="left">Previous studies</th><th colspan="12" rowspan="1" valign="top" align="left">This study (discovery phase)</th></tr><tr><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="12" rowspan="1" valign="bottom" align="left"><hr></th></tr><tr><th rowspan="3" colspan="1" valign="top" align="left">Frequency<br>of reference<br>allele</th><th colspan="2" rowspan="1" valign="top" align="left">Reported<br>association</th><th rowspan="3" colspan="1" valign="top" align="left">Frequency<br>of reference<br>allele</th><th rowspan="3" colspan="1" valign="top" align="left">Imputation<br>quality<br>(RSQ)</th><th colspan="2" rowspan="1" valign="top" align="left">Meta-analysis (all<br>frontotemporal<br>dementia)</th><th colspan="2" rowspan="1" valign="top" align="left">Behavioural<br>frontotemporal<br>dementia</th><th colspan="2" rowspan="1" valign="top" align="left">Semantic dementia</th><th colspan="2" rowspan="1" valign="top" align="left">Progressive<br>nonfluent aphasia</th><th colspan="2" rowspan="1" valign="top" align="left">Frontotemporal<br>dementia/motor<br>neuron disease</th></tr><tr><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">p value<br>(joint)</th><th rowspan="1" colspan="1" valign="top" align="left">Odds<br>ratio</th><th rowspan="1" colspan="1" valign="top" align="left">p value</th><th rowspan="1" colspan="1" valign="top" align="left">Odds<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="left">p value</th><th rowspan="1" colspan="1" valign="top" align="left">Odds<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="left">p value</th><th rowspan="1" colspan="1" valign="top" align="left">Odds<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="left">p value</th><th rowspan="1" colspan="1" valign="top" align="left">Odds<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="left">p value</th><th rowspan="1" colspan="1" valign="top" align="left">Odds<br>ratio<br>(95% CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"><strong>Chromosome 17</strong></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td colspan="18" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td colspan="4" rowspan="1" align="left">Progressive supranuclear palsy or corticobasal degeneration<sup><a href="#R40" rid="R40" class=" bibr popnode">40</a></sup></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;<em>MAPT</em></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs8070723 (44081064)</td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">0·950 (stage 1); 0·940 (stage 2)</td><td rowspan="1" colspan="1" align="left">1·50×10<sup>−11</sup></td><td rowspan="1" colspan="1" align="left">5·46</td><td rowspan="1" colspan="1" align="left">0·765</td><td rowspan="1" colspan="1" align="left">0·8400</td><td rowspan="1" colspan="1" align="left">2·80×10<sup>−4</sup></td><td rowspan="1" colspan="1" align="left">1·201 (1·09–1·32)</td><td rowspan="1" colspan="1" align="left">3·14×10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1·201 (1·06–1·36)</td><td rowspan="1" colspan="1" align="left">4·34×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·103 (0·86–1·41)</td><td rowspan="1" colspan="1" align="left">8·72×10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1·471 (1·10−1·97)</td><td rowspan="1" colspan="1" align="left">5·82×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·091 (0·80–1·49)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs242557 (44019712)</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">0·470 (stage 1); 0·500 (stage 2)</td><td rowspan="1" colspan="1" align="left">4·20×10<sup>−70</sup></td><td rowspan="1" colspan="1" align="left">0·51</td><td rowspan="1" colspan="1" align="left">0·634</td><td rowspan="1" colspan="1" align="left">0·5246</td><td rowspan="1" colspan="1" align="left">4·82×10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">0·853 (1·05–1·31)</td><td rowspan="1" colspan="1" align="left">1·27×10<sup>−2</sup></td><td rowspan="1" colspan="1" align="left">0·841 (0·73–0·96)</td><td rowspan="1" colspan="1" align="left">3·20×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·867 (0·65–1·15)</td><td rowspan="1" colspan="1" align="left">2·02×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·815 (0·59–1·11)</td><td rowspan="1" colspan="1" align="left">8·23×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·961 (0·68–1·36)</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Chromosome 19</strong></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">Alzheimer’s disease<sup><a href="#R41" rid="R41" class=" bibr popnode">41</a></sup></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;<em>TOMM40/APOE</em></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs2075650 (45395619)</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">0·150</td><td rowspan="1" colspan="1" align="left">1·80×10<sup>−157</sup></td><td rowspan="1" colspan="1" align="left">2·53</td><td rowspan="1" colspan="1" align="left">0·141</td><td rowspan="1" colspan="1" align="left">0·9978</td><td rowspan="1" colspan="1" align="left">8·81×10<sup>−7</sup></td><td rowspan="1" colspan="1" align="left">1·304 (0·69–0·85)</td><td rowspan="1" colspan="1" align="left">1·37×10<sup>−6</sup></td><td rowspan="1" colspan="1" align="left">1·383 (0·63–0·82)</td><td rowspan="1" colspan="1" align="left">3·64×10−</td><td rowspan="1" colspan="1" align="left">1·326 (0·58–0·98)</td><td rowspan="1" colspan="1" align="left">8·69×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·976 (0·76–1·38)</td><td rowspan="1" colspan="1" align="left">2·06×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·252 (0·56–1·13)</td></tr><tr><td colspan="18" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Chromosome 9</strong></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td colspan="18" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left">Amyotrophic lateral sclerosis<sup><a href="#R39" rid="R39" class=" bibr popnode">39</a></sup></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;<em>C9orf72/MOB3B</em></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs3849942 (27543281)</td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">0·260</td><td rowspan="1" colspan="1" align="left">1·01×10<sup>−8</sup></td><td rowspan="1" colspan="1" align="left">1·20</td><td rowspan="1" colspan="1" align="left">0·253</td><td rowspan="1" colspan="1" align="left">0·9996</td><td rowspan="1" colspan="1" align="left">4·38×10<sup>−4</sup></td><td rowspan="1" colspan="1" align="left">1·166 (1·07–1·27)</td><td rowspan="1" colspan="1" align="left">7·38×10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1·155 (0·78–0·96)</td><td rowspan="1" colspan="1" align="left">9·89×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·010 (0·80–1·25)</td><td rowspan="1" colspan="1" align="left">9·03×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·990 (0·79–1·31)</td><td rowspan="1" colspan="1" align="left">2·12×10<sup>−6</sup></td><td rowspan="1" colspan="1" align="left">1·957 (0·39–0·68)</td></tr><tr><td colspan="18" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Chromosome 7</strong></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td colspan="18" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">FTLD-TDP<sup><a href="#R17" rid="R17" class=" bibr popnode">17</a></sup></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;<em>TMEM106B</em></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs1990622 (12283787)</td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">0·679</td><td rowspan="1" colspan="1" align="left">1·08×10<sup>−11</sup></td><td rowspan="1" colspan="1" align="left">1·64</td><td rowspan="1" colspan="1" align="left">0·600</td><td rowspan="1" colspan="1" align="left">0·9588</td><td rowspan="1" colspan="1" align="left">7·88×10<sup>−2</sup></td><td rowspan="1" colspan="1" align="left">1·080 (0·99–1·16)</td><td rowspan="1" colspan="1" align="left">5·85×10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1·144 (1·04–1·26)</td><td rowspan="1" colspan="1" align="left">8·36×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·978 (0·80–1·20)</td><td rowspan="1" colspan="1" align="left">8·98×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·985 (0·79–1·23)</td><td rowspan="1" colspan="1" align="left">3·11×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·876 (0·68–1·13)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs6966915 (12265988)</td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">T</td><td rowspan="1" colspan="1" align="left">0·679</td><td rowspan="1" colspan="1" align="left">1·63×10<sup>−11</sup></td><td rowspan="1" colspan="1" align="left">1·64</td><td rowspan="1" colspan="1" align="left">0·596</td><td rowspan="1" colspan="1" align="left">0·9675</td><td rowspan="1" colspan="1" align="left">1·21×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·070 (0·87–1·02)</td><td rowspan="1" colspan="1" align="left">5·74×10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1·144 (1·04–1·26)</td><td rowspan="1" colspan="1" align="left">5·27×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·936 (0·76–1·15)</td><td rowspan="1" colspan="1" align="left">7·26×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·961 (0·77–1·20)</td><td rowspan="1" colspan="1" align="left">3·62×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·888 (0·69–1·14)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs1020004 (12255778)</td><td rowspan="1" colspan="1" align="left">T</td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">0·767</td><td rowspan="1" colspan="1" align="left">5·00×10<sup>−11</sup></td><td rowspan="1" colspan="1" align="left">1·66</td><td rowspan="1" colspan="1" align="left">0·693</td><td rowspan="1" colspan="1" align="left">0·9538</td><td rowspan="1" colspan="1" align="left">4·59×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·030 (0·95–1·12)</td><td rowspan="1" colspan="1" align="left">5·71×10<sup>−2</sup></td><td rowspan="1" colspan="1" align="left">1·104 (1·00–1·22)</td><td rowspan="1" colspan="1" align="left">8·53×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·980 (0·79–1·21)</td><td rowspan="1" colspan="1" align="left">5·00×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·921 (0·72–1·17)</td><td rowspan="1" colspan="1" align="left">1·20×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·805 (0·61–1·06)</td></tr><tr><td colspan="18" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Chromosome 6</strong></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td colspan="18" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left">Multiple sclerosis<sup><a href="#R43" rid="R43" class=" bibr popnode">43</a>,<a href="#R44" rid="R44" class=" bibr popnode">44</a></sup></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;<em>RAB38</em></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs1386330 (87819427)</td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">T</td><td rowspan="1" colspan="1" align="left">0·130</td><td rowspan="1" colspan="1" align="left">2·00×10<sup>−6</sup></td><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="left">0·141</td><td rowspan="1" colspan="1" align="left">0·9694</td><td rowspan="1" colspan="1" align="left">3·35×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·050 (0·85–1·06)</td><td rowspan="1" colspan="1" align="left">6·09×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·040 (0·84–1·10)</td><td rowspan="1" colspan="1" align="left">7·60×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·040 (0·72–1·27)</td><td rowspan="1" colspan="1" align="left">6·97×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·060 (0·68–1·29)</td><td rowspan="1" colspan="1" align="left">3·00×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·829 (0·58–1·18)</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs3129871 (32406342)</td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">0·230</td><td rowspan="1" colspan="1" align="left">8·94×10<sup>−81</sup></td><td rowspan="1" colspan="1" align="left">1·99</td><td rowspan="1" colspan="1" align="left">0·131</td><td rowspan="1" colspan="1" align="left">0·9734</td><td rowspan="1" colspan="1" align="left">4·80×10<sup>−2</sup></td><td rowspan="1" colspan="1" align="left">1·122 (1·00–1·26)</td><td rowspan="1" colspan="1" align="left">2·10×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·095 (0·79–1·05)</td><td rowspan="1" colspan="1" align="left">1·25×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·254 (0·60–1·06)</td><td rowspan="1" colspan="1" align="left">5·02×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·120 (0·64–1·24)</td><td rowspan="1" colspan="1" align="left">6·10×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·105 (0·61–1·33)</td></tr><tr><td rowspan="1" colspan="1" align="left">Parkinson’s disease<sup><a href="#R45" rid="R45" class=" bibr popnode">45</a>,<a href="#R46" rid="R46" class=" bibr popnode">46</a></sup></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;<em>HLA-DRA</em></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs3129871 (32406342)</td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">C</td><td rowspan="1" colspan="1" align="left">0·504</td><td rowspan="1" colspan="1" align="left">5·70×10<sup>−15</sup></td><td rowspan="1" colspan="1" align="left">1·72</td><td rowspan="1" colspan="1" align="left">0·337</td><td rowspan="1" colspan="1" align="left">0·9379</td><td rowspan="1" colspan="1" align="left">3·43×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·961 (0·88–1·04)</td><td rowspan="1" colspan="1" align="left">3·15×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·949 (0·95–1·16)</td><td rowspan="1" colspan="1" align="left">4·94×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·078 (0·75–1·15)</td><td rowspan="1" colspan="1" align="left">8·24×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·974 (0·81–1·30)</td><td rowspan="1" colspan="1" align="left">2·72×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">0·859 (0·89–1·53)</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;rs3129882 (32409530)</td><td rowspan="1" colspan="1" align="left">G</td><td rowspan="1" colspan="1" align="left">A</td><td rowspan="1" colspan="1" align="left">0·450</td><td rowspan="1" colspan="1" align="left">1·90×10<sup>−10</sup></td><td rowspan="1" colspan="1" align="left">1·26</td><td rowspan="1" colspan="1" align="left">0·456</td><td rowspan="1" colspan="1" align="left">0·9992</td><td rowspan="1" colspan="1" align="left">3·36×10<sup>−2</sup></td><td rowspan="1" colspan="1" align="left">1·086 (0·85–0·99)</td><td rowspan="1" colspan="1" align="left">3·27×10<sup>−2</sup></td><td rowspan="1" colspan="1" align="left">1·106 (0·82–0·99)</td><td rowspan="1" colspan="1" align="left">7·52×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·033 (0·79–1·18)</td><td rowspan="1" colspan="1" align="left">5·74×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·065 (0·75–1·17)</td><td rowspan="1" colspan="1" align="left">7·07×10<sup>−1</sup></td><td rowspan="1" colspan="1" align="left">1·049 (0·74–1·22)</td></tr></tbody></table></div><div id="largeobj_idm140164555094528" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4112126/table/T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN11"><p id="P57" class="p p-first-last">NA=. FLTD-TDP=frontotemporal lobar degeneration with TDP43-positive inclusions. RSQ=</p></div></div></div><p id="P24">For the <em>MAPT</em> locus (PSP/CBD), the SNPs rs242557 (effect allele G) and rs8070723 (effect allele A)<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775540">40</a></sup> were significantly associated only within the entire cohort and in the behavioural FTD and progressive nonfluent aphasia subtypes (rs8070723 only; <a href="/pmc/articles/PMC4112126/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>table 5</span></a>). The effect was small in our study although in the same direction as in the GWAS for progressive supranuclear palsy (5·46<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775544">40</a></sup>
<em>vs</em> about 1·2–1·4 in our study; <a href="/pmc/articles/PMC4112126/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>table 5</span></a>). These results might have arisen because we excluded all known chromosome 17 mutation carriers and because tau pathology is a less common feature of sporadic FTD.</p><p id="P25">For the <em>TOMM40/APOE</em> locus (Alzheimer’s disease), the SNP rs2075650 (effect allele G) (<a href="/pmc/articles/PMC4112126/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>table 5</span></a>). Several Alzheimer’s disease GWASs reported association with the minor allele of this SNP with ORs greater than 2·5,<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775503">41</a></sup> but in our study the OR was about 1·3 (<a href="/pmc/articles/PMC4112126/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>table 5</span></a>). This suggestive association might be indicative of clinical overlap between patients with clinically diagnosed FTD and those with Alzheimer’s disease.<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775510">42</a></sup></p><p id="P26">For the <em>TMEM106B</em> locus (FTLD-TDP), we assessed the three associated SNPs reported by Van Deerlin and colleagues (rs1990622, effect allele A; rs6966915, effect allele C; rs1020004, effect allele T).<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775542">17</a></sup> All achieved modest p values in the entire dataset with lowest p values in the range of 10<sup>−2</sup>–10<sup>−3</sup> only in the behavioural FTD subtype (<a href="/pmc/articles/PMC4112126/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>table 5</span></a>). Van Deerlin and colleagues’ study<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775546">17</a></sup> was done on samples from patients with autopsy-confirmed FTLD-TDP, whereas our cohort is mainly clinically defined. Additionally, the previous study included many <em>GRN</em> mutation carriers, who frequently present with behavioural FTD;<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775518">17</a></sup> in our study, <em>GRN</em> mutation carriers were excluded. Biochemical evidence has suggested that <em>TMEM106B</em> is directly related to <em>GRN</em> metabolism,<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup> thus we regard our data as a limited replication of the original finding.</p><p id="P27" class="p p-last">Finally, the <em>RAB38</em> locus previously showed suggestive association in multiple sclerosis,<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_483108612">43</a></sup> but the <em>HLA</em> locus was reported to associate with multiple sclerosis,<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775517">44</a>,<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775508">45</a></sup> Parkinson’s disease,<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775525">19</a>,<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775558">46</a></sup> and Alzheimer’s disease.<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775514">47</a></sup> None of the SNPs reported in these studies, and which were assessed in our dataset (<a href="/pmc/articles/PMC4112126/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>table 5</span></a>),<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_483108614">43</a>–<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775559">46</a></sup> showed association with FTD, probably suggesting that different risk haplotype sub-structures at the same loci associate with distinctive phenotypes.</p></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Discussion</h2><p id="P28" class="p p-first">FTD is characterised by a broad range of clinical manifestations, differential pathological signatures, and substantial genetic variability, which imply a complex disease mechanisms.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775532">15</a></sup> In the search for novel disease risk loci associated with FTD we have done an extensive GWAS on a large cohort of mainly clinically diagnosed FTD samples from patients of European ancestry. Several limitations might apply to this study. In view of the phenotype heterogeneity of FTD, and considering that it is a rare neurodegenerative disorder,<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775529">2</a></sup> testing the hypothesis “common variant – common disease” for diseases of this kind is challenging and clearly benefits from large sample sizes. Additionally, our findings might indicate association with specific loci without necessarily implying causality; low heritability due to common variability can also apply. However, the QQ plots and associated λ values (<a href="#FN3" rid="FN3" class=" fn">appendix</a>) conformed to GWAS standards, lending support to our findings.</p><p id="P29">We included samples from more than 3500 patients and, thus, we know of no larger GWAS for FTD. We have identified two novel potential loci for FTD: 11q14, encompassing <em>RAB38/CTSC</em>, was suggestive for the behavioural FTD subtype, and 6p21.3, encompassing the <em>HLA</em> locus was statistically significant for the entire cohort.</p><p id="P30"><em>RAB38</em><sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775496">48</a></sup> encodes the transmembrane protein RAB38, which is expressed in the thyroid, in elements of the immune system, and in the brain. From a functional perspective, RAB38 has been shown to mediate protein trafficking to lysosomal-related organelles and maturation of phagosomes (panel).<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775553">49</a>,<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775552">50</a></sup>
<em>CTSC</em> is a lysosomal cysteine-proteinase that participates in the activation of serine proteinases in immune and inflammatory cells that are involved in immune and inflammatory processes including phagocytosis of pathogens and local activation and deactivation of inflammatory factors (<em>Online Mendelian Inheritance in Man</em> [OMIM] number 602365). The SNP rs302652 at the <em>RAB38/CTSC</em> locus shows an eQTL in monocytes<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775530">38</a></sup> associated with decreased expression of <em>RAB38</em>, possibly indicating that a decreased function of RAB38 might be the mechanism by which the association at this locus is mediated. Both <em>RAB38</em> and <em>CTSC</em> are implicated in lysosomal biology and an association with lysosomal and autophagic processes in FTD was previously suggested in two studies of <em>GRN</em><sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775516">51</a></sup> and <em>TMEM106B</em>.<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775493">52</a></sup> A role for autophagy has also been shown in Parkinson’s disease.<sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775497">53</a></sup> Our findings will need to be replicated in other FTD cohorts in follow-up studies (eg, fine-mapping studies) to lend support to the proposal that lysosomal biology and autophagy might be involved in the aetiology of FTD.<sup><a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775537">54</a></sup></p><p id="P31">The genetic association that we identified with the <em>HLA</em> locus supports the idea of a link between FTD and the immune system. Our mQTL data showed that risk at this locus is associated with <em>cis</em>-changes in methylation levels of <em>HLA-DRA</em> in the frontal cortex. <em>HLA</em> associations have been previously reported in Alzheimer’s disease,<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775513">47</a></sup> Parkinson’s disease,<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775545">19</a>,<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775495">46</a></sup> and multiple sclerosis.<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775538">44</a>,<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775526">45</a></sup> Additionally, a general involvement of the innate and the adaptive immune responses has been suggested in the pathogenesis of neurodegenerative diseases,<sup><a href="#R55" rid="R55" class=" bibr popnode">55</a>,<a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775524">56</a></sup> lending supporting to the idea that the immune system plays an important part within the spectrum of neurological disorders.</p><p id="P32" class="p p-last">Future studies should aim to replicate our findings and, in so doing, elucidate the functional basis of FTD. Additionally, our data indicate that common pathways and processes might underlie different forms of neurodegenerative disorders, including Alzheimer’s disease, Parkinson’s disease, multiple sclerosis, and FTD. Exploring the possibility of developing therapeutic measures targeting general damage responses could hold promise—after replication and validation of our findings—for the development and implementation of treatment options for these neurological disorders, including FTD.</p></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4112126/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140164561919280" class="figure"><a href="/pmc/articles/PMC4112126/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4112126_nihms599470f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms599470f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4112126/bin/nihms599470f1.jpg"></a></div><div id="largeobj_idm140164561919280" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4112126/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4112126/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Manhattan plots identifying regions with genome–wide significant associations</strong><p id="P40">(A) Manhattan plot for the entire dataset of the discovery phase depicts the associated regionat 6p21.3. Single–nucleotide polymorphisms (SNPs) with smallest p values and their location within or in proximity of the nearest genes are shown. (B) Manhattan plot for the behavioural frontotemporal dementia set in the discovery phase depicts the associated region at 11q14.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4112126/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140164561916192" class="figure"><a href="/pmc/articles/PMC4112126/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4112126_nihms599470f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms599470f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4112126/bin/nihms599470f2.jpg"></a></div><div id="largeobj_idm140164561916192" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4112126/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4112126/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Manhattan plots identifying regions with genome–wide significant associations</strong><p id="P41">SNPs with smallest p values and their location within or in proximity of the nearest genes are shown.Manhattan plots for semantic dementia (C), progressive nonfluent aphasia (D), and motor neurone disease (E) frontotemporal dementia sets.</p></div></div></div><div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="idm140164551812752title"><em>Panel</em>: Research in context</h3></div><div class="sec" id="S15"><h3 id="S15title">Systematic review</h3><p id="P58" class="p p-first-last">We searched PubMed for the most relevant research articles and review articles on frontotemporal dementia using the following terms: “FTD” and “genetics”, and “FTD” and “review”.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775492">1</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775554">4</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775512">5</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775499">8</a>–<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775505">17</a>,<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775539">54</a></sup> We compared our results to several previously published genome-wide association studies. We identified only one directly relevant study that investigated a pathologically defined subtype of frontotemporal dementia (frontotemporal lobar degeneration with TDP43-positive inclusions; FTLD-TDP).<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775543">17</a></sup> The other studies were of related diseases such as amyotrophic lateral sclerosis,<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775519">39</a></sup> Alzheimer’s disease,<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775502">41</a>,<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775491">47</a></sup> progressive supranuclear palsy and corticobasal degeneration,<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775561">40</a></sup> multiple sclerosis,<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_483108615">43</a>–<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775488">45</a></sup> and Parkinson’s disease.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775548">19</a>,<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_391775560">46</a></sup></p></div><div class="sec" id="S16"><h3 id="S16title">Interpretation</h3><p id="P59" class="p p-first-last">To the best of our knowledge, ours is the first genome-wide association study in samples from patients with clinical frontotemporal dementia. In view of the complexity and heterogeneity of the disease, mutations in only three main genes—<em>MAPT, GRN,</em> and <em>C9orf72</em>—have been associated with frontotemporal dementia, and these explain only a small proportion of cases. Most importantly, little is known about the mechanisms involved in the development of this disorder. Our findings suggest that common variability in loci that point to immune processes, and possibly to lysosomal biology and autophagy, are involved in the pathobiology of the disease. These findings provide a basis for future replication and functional studies.</p></div></div></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Acknowledgments</h2><div class="sec"><p id="P38">We received intramural funding from the National Institute of Neurological Disorders and Stroke (NINDS) and National Institute on Aging (NIA), the Wellcome/MRC Centre on Parkinson’s disease, Alzheimer’s Research UK (ARUK, Grant ARUK-PG2012-18), and by the office of the Dean of the School of Medicine, Department of Internal Medicine, at Texas Tech University Health Sciences Center. We thank Mike Hubank and Kerra Pearce at the Genomic core facility at the Institute of Child Health (ICH), UCL, for assisting RF in doing Illumina genotyping experiments (FTD-GWAS genotyping). The work done by the North American Brain Expression Consortium (NABEC) was supported in part by the Intramural Research Program of the National Institute on Aging, National Institutes of Health, part of the US Department of Health and Human Services (project number ZIA AG000932-04), and by a Research Grant from the Department of Defense (W81XWH-09-2-0128). Work done by the UK Brain Expression Consortium (UKBEC) was supported by the MRC through the MRC Sudden Death Brain Bank (CS), by a Project Grant (G0901254 to JH and MW), and by a Fellowship award (G0802462 to MR). DT was supported by the King Faisal Specialist Hospital and Research Centre, Saudi Arabia. Computing facilities used at King’s College London were supported by the National Institute for Health Research (NIHR) Biomedical Research Centre based at Guy’s and St Thomas’ NHS Foundation Trust and King’s College London. We thank AROS Applied Biotechnology AS company laboratories and Affymetrix for their valuable input. JBJK was supported by the National Health and Medical Resarch Council (NHMRC), Australia (project grants 510217 and 1005769). CDS was supported by NHMRC (project grants 630428 and 1005769). PRS was supported by NHMRC (project grants 510217 and 1005769) and acknowledges that DNA samples were prepared by Genetic Repositories Australia, supported by NHMRC Enabling Grant 401184. GMH was supported by NHMRC Research Fellowship 630434, Project Grant 1029538, and Program Grant 1037746. JRH was supported by the Australian Research Council Federation Fellowship, NHMRC Project Grant 1029538 and NHMRC Program Grant 1037746. OP was supported by NHMRC Career Development Fellowship 1022684, Project Grant 1003139. IH, AR, and MB acknowledge the patients and controls who participated in this project and the Trinitat Port-Carbó and her family who are supporting Fundació ACE research programmes. CC was supported by Grant P30-NS069329-01 and acknowledges that the recruitment and clinical characterisation of research participants at Washington University were supported by NIH P50 AG05681, P01 AG03991, and P01 AG026276. LB and GB were supported by the Ricerca Corrente, Italian Ministry of Health. RG was supported by Fondazione CARIPLO 2009-2633, Ricerca Corrente, Italian Ministry of Health. GF was supported by Fondazione CARIPLO 2009–2633. ES was supported by the Italian Ministry of Health. CF was supported by Fondazione Cariplo. MS was supported from the Italian Ministry of Health (Ricerca Corrente). MLW was supported by Government funding of clinical research within NHS Sweden (ALF). KN was supported by Thure Carlsson Foundation. CN was supported by Swedish Alzheimer Fund. IRAM and GYRH were supported by CIHR (grant 74580) PARF (grant C06-01). JG was supported by the NINDS intramural research funds for FTD research. CMM was supported by Medical Research Council UK, Brains for Dementia Research, Alzheimer’s Society, Alzheimer’s Research UK, National Institutes for Health Research, Department of Health, and Yvonne Mairy Bequest, and acknowledges that tissue samples made available for this study were provided by the Newcastle Brain Tissue Resource, which was funded in part by grants G0400074 and G1100540 from the UK MRC, the Alzheimer’s Research Trust and Alzheimer’s Society through the Brains for Dementia Research Initiative and an NIHR Biomedical Research Centre Grant in Ageing and Health, and NIHR Biomedical Research Unit in Lewy Body Disorders. CMM was supported by the UK Department of Health and Medical Research Council and the Research was supported by the National Institute for Health Research Newcastle Biomedical Research Centre based at Newcastle Hospitals Foundation Trust and Newcastle University and acknowledges that the views expressed are those of the authors and not necessarily those of the NHS, the NIHR or the Department of Health. JA was supported by MRC, Dunhill Medical Trust, and Alzheimer’s Research UK. TDG was supported by Wellcome Trust Senior Clinical Fellow. IGM was supported by NIHR Biomedical Research Centre and Unit on Ageing Grants and acknowledges the National Institute for Health Research Newcastle Biomedical Research Centre based at Newcastle Hospitals Foundation Trust and Newcastle University. The views expressed are those of the authors and not necessarily those of the NHS, the NIHR, or the Department of Health. AJT was supported by Medical Research Council, Alzheimer’s Society, Alzheimer’s Research UK, and the National Institutes for Health Research. EJ was supported by NIHR and Newcastle Biomedical Research Centre. PP, CR, SOC, and EA were supported partially by FIMA (Foundation for Applied Medical Research). PP acknowledges Manuel Seijo-Martínez (Department of Neurology, Hospital do Salnés, Pontevedra, Spain) and Ramon Rene, Jordi Gascon, and Jaume Campdelacreu (Department of Neurology, Hospital de Bellvitge, Barcelona, Spain) for providing FTD DNA samples. RP, JDS, PA, and AK were supported by the German Federal Ministry of Education and Research (BMBF; grant number FKZ 01GI1007A—German FTLD consortium). IR was supported by Ministero dell’Istruzione, dell’Università e della Ricerca (MIUR) of Italy. PStGH was supported by the Canadian Institutes of Health Research, Wellcome Trust, Ontario Research Fund. FT was supported by the Italian Ministry of Health (ricerca corrente) and MIUR grant RBAP11FRE9. GR and GG were supported by the Italian Ministry of Health (ricerca corrente). JBR was supported by Cambridge NIHR Biomedical Research Centre and Wellcome Trust (088324). JU, JC, and SM were supported by the MRC Prion Unit core funding and acknowledge MRC UK, UCLH Biomedical Research Centre, and Queen Square Dementia BRU. SM thanks John Beck, Tracy Campbell, Gary Adamson, Ron Druyeh, Jessica Lowe, and Mark Poulter. AD thanks Benedikt Bader, Manuela Neumann, Sigrun Roeber, Thomas Arzberger, and Hans Kretzschmar. VMVD and JQT were supported by grants AG032953, <a id="__tag_786366702" class="tag_hotlink" href="/nuccore/AG017586" ref="/nuccore/AG017586"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG017586","term_id":"55790122","term_text":"AG017586"}}</span>AG017586</a> and <a id="__tag_786366705" class="tag_hotlink" href="/nuccore/AG010124" ref="/nuccore/AG010124"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG010124","term_id":"3294400","term_text":"AG010124"}}</span>AG010124</a>. MG was supported by Grants AG032953, <a id="__tag_786366701" class="tag_hotlink" href="/nuccore/AG017586" ref="/nuccore/AG017586"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG017586","term_id":"55790122","term_text":"AG017586"}}</span>AG017586</a>, <a id="__tag_786366700" class="tag_hotlink" href="/nuccore/AG010124" ref="/nuccore/AG010124"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG010124","term_id":"3294400","term_text":"AG010124"}}</span>AG010124</a>, and NS044266. VMVD thanks EunRan Suh for assistance with sample handling and Elisabeth McCarty-Wood for help in selection of patients. JQT thanks Terry Schuck, John Robinson, and Kevin Raible for assistance with neuropathological assessment of patients. CVB and the Antwerp site were in part funded by the MetLife Foundation Award for Medical Research (to CVB), the Belgian Science Policy Office Interuniversity Attraction Poles programme; the Foundation for Alzheimer Research (SAO-FRA); the Medical Foundation Queen Elisabeth; the Flemish Government Methusalem Excellence award (to CVB); the Research Foundation Flanders (FWO); and the University of Antwerp Research Fund. JvdZ holds a postdoctoral fellowship of the FWO. CVB and the Antwerp site authors thanks neurologists S Engelborghs, PP De Deyn, A Sieben, and Rik Vandenberghe and neuropathologist JJ Martin for the clinical and pathological diagnoses. Isabelle Leber and Alexis Brice were supported by the programme Investissements d’avenir ANR-10-IAIHU-06 and acknowledges the contribution of The French Research Network on FTLD/FTLD-ALS for the contribution in samples collection. BN, SS, SB, and IP were supported by Prin 2010-prot.2010PWNJXK; Cassa di Rispario di Firenze e Cassa di Risparmio di Pistoia e Pescia. JEN was supported by the Novo Nordisk Foundation, Denmark. MR was supported by the German National Genome Network (NGFN) and the German Ministry for Education and Research Grant Number 01GS0465. JDR, MNR, NCF, and JDW were supported by an MRC programme grant, the NIHR Queen Square Dementia Biomedical Research Unit and the Leonard Wolfson Experimental Neurology Centre. MGS was supported by MRC grant n G0301152, Cambridge Biomedical Research Centre, and thanks K Westmore for extracting DNA. HM was supported by the Motor Neuron Disease Association (Grant 6057). RR was supported by P50 AG016574, R01 NS080882, R01 NS065782, P50 NS72187, and the Consortium for Frontotemporal Dementia. DWD was supported by P50NS072187, P50AG016574, State of Florida Alzheimer Disease Initiative, and CurePSP Inc. NRGR, JEP, RCP, DK, and BFB were supported by P50 AG016574. KAJ was supported by R01 AG037491. WWS was supported by NIH <a id="__tag_786366699" class="tag_hotlink" href="/nuccore/AG023501" ref="/nuccore/AG023501"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG023501","term_id":"7681676","term_text":"AG023501"}}</span>AG023501</a>, <a id="__tag_786366704" class="tag_hotlink" href="/nuccore/AG019724" ref="/nuccore/AG019724"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG019724","term_id":"6045668","term_text":"AG019724"}}</span>AG019724</a>, Consortium for Frontotemporal Dementia Research. BLM was supported by P50AG023501, P01AG019724, Consortium for FTD Research. HR was supported by <a id="__tag_786366703" class="tag_hotlink" href="/nuccore/AG032306" ref="/nuccore/AG032306"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG032306","term_id":"16559179","term_text":"AG032306"}}</span>AG032306</a>. JCvS was supported by Stichting Dioraphte Foundation (11 02 03 00), Nuts Ohra Foundation (0801-69), Hersenstichting Nederland (BG 2010-02), and Alzheimer Nederland. CG and HHC acknowledge families, patients, clinicians including Inger Nennesmo and Vesna Jelic, Laura Fratiglioni for control samples and Jenny Björkström, Håkan Thonberg, Charlotte Forsell, Anna-Karin Lindström, and Lena Lilius for sample handling. CG was supported by Swedish Brain Power (SBP), the Strategic Research Programme in Neuroscience at Karolinska Institutet (StratNeuro), the regional agreement on medical training and clinical research (ALF) between Stockholm County Council and Karolinska Institutet, Swedish Alzheimer Foundation, Swedish Research Council, Karolinska Institutet PhD-student funding, King Gustaf V, and Queen Victoria’s Free Mason Foundation. FP, AR, VD, and FL acknowledge Labex DISTALZ. RF acknowledges the help and support of June Howard at the Texas Tech University Health Sciences Center Office of Sponsored Programs for tremendous help in managing Material Transfer Agreement at TTUHSC.</p><p id="P39">CVB and MC are inventors on patent applications for GRN and C9orf72. PRS receives speaker fees from Janssen pharmaceutical. RR receives research support from the NIH (R01 NS080882, R01 NS065782, R01 AG026251, R01 NS076471, and P50 AG16574), the ALS Therapy Alliance, and the Consortium for Frontotemporal Degeneration Research, honoraria for lectures or educational activities not funded by industry. RR serves on the medical advisory board of the Association for Frontotemporal Degeneration and the board of directors of the International Society for Frontotemporal Dementia, and holds a patent on methods to screen for the hexanucleotide repeat expansion in the C9ORF72 gene. DWD serves on the editorial boards of the <em>American Journal of Pathology, Journal of Neuropathology and Experimental Neurology, Brain Pathology, Neurobiology of Aging, Journal of Neurology, Neurosurgery, and Psychiatry, Annals of Neurology, and Neuropathology</em>. DWD is supported by NIH grants (P50 AG16574, P50 NS72187, P01 AG03949), the Mangurian Foundation, CurePSP, and the Robert E Jacoby Professorship for Alzheimer’s Research. NRGR is on the Scientific Advisory Board for Codman, TauRzx multicenter study, Consultation for CYTOX. RCP chairs a Data Monitoring Committee for Pfizer and Janssen Alzheimer Immunotherapy, and is a consultant for GE Healthcare and Elan Pharmaceuticals. RCP receives royalties from Oxford University Press for Mild Cognitive Impairment. DK is Deputy Editor for <em>Neurology</em>. DK has served on a data safety monitoring board for Lilly Pharmaceuticals, as a consultant to TauRx, was an investigator in clinical trials sponsored by Baxter, Elan Pharmaceuticals, and Forest Pharmaceuticals in the past 2 years and receives research support from the NIH. BFB has served as an investigator for clinical trials sponsored by Cephalon Inc, Allon Pharmaceuticals, and GE Healthcare. BFB receives royalties from the publication of a book entitled Behavioral Neurology Of Dementia (Cambridge Medicine, 2009). BFB has received honoraria from the American Academy of Neurology. BFB serves on the Scientific Advisory Board of the Tau Consortium. BFB receives research support from the National Institute on Aging (P50 AG016574, U01 <a id="__tag_786366708" class="tag_hotlink" href="/nuccore/AG006786" ref="/nuccore/AG006786"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG006786","term_id":"3063075","term_text":"AG006786"}}</span>AG006786</a>, RO1 <a id="__tag_786366707" class="tag_hotlink" href="/nuccore/AG032306" ref="/nuccore/AG032306"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG032306","term_id":"16559179","term_text":"AG032306"}}</span>AG032306</a>, RO1 <a id="__tag_786366706" class="tag_hotlink" href="/nuccore/AG041797" ref="/nuccore/AG041797"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG041797","term_id":"16570522","term_text":"AG041797"}}</span>AG041797</a>) and the Mangurian Foundation. BLM is on the Board Membership of The Larry L Hillblom Foundation, The John Douglas French Foundation, The Tau Consortium, Sagol School of Neuroscience Tel Aviv University. BLM holds consultancy for Tau Rx lts—Chair, Scientific Advisory Board bvFTD Trial Allon Therapeutics—Steering Committee AL-108-231 Study, Bristol-Myers Squibb-Advisory Board, Progressive Supranuclear Palsy (PSP), Neurology Scientific Advisory Board Meeting Siemens Molecular Imaging, and Eli Lilly US Alzheimer’s Disease Advisory Board, and receives royalties from Cambridge University Press Guilford Publications Inc, Neurocase.</p></div></div><div id="idm140164555515072" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140164555515072title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="P33" class="p p-first-last">See Online for appendix</p><p></p><div><p id="P34" class="p p-first"><strong>Contributors</strong></p>JH, PM, ABS, MAN, RF, and JDR designed the study. JDR, RF and JH did the clinical quality checks. RF coordinated sample collection, received samples at UCL and TTUHSC, and did material quality control for discovery and replication phases. DGH received samples at NIH and coordinated material quality control at NIH. JDR, JBJK, CDS, PRS, WSB, JRH, GMH, OP, LB, ET, EH, IH, AR, MB BB, AP, LB, GB, RG, GF, DG, ES, CF, MS, JC, AL, RB, MLW, KN, CN, IRAM, GYRH, DMAM, JG, CMM, JA, TDG, IGM, AJT, PP, EDH, EMW, AB, EJ, MCT, PP, CR, SOC, EA, RP, JDS, PA, AK, IR, ER, LP, ER, PStGH, ER, GR, FT, GG, JBR, JCMS, JU, JC, SM, AD, VMVD, MG, JQT, JvdZ, TVL, CVB, WD, MC, SFC, ILB, AB, DH, VG, MV, BN, SS, SB, IP, JEN, LEH, MR, BI, MM, GG, SP, WG, MNR, NCF, JDW, MGS, HM, PR, PH, JSS, AG, AR, SR, ACB, RM, FF, CC, LB, MA, MG, MEC, NS, RR, MB, DWD, JEP, NRGR, RCP, DK, KAJ, BFB, WWS, BLM, AMK, HR, JCvS, EGPD, HS, YALP, PS, GL, RC, VN, AAP, MF, AP, GM, PS, HHC, CG, FP, AR, VD, FL, DK, LF, and SPB collected and characterised samples. MK was responsible for genotyping at ICH. JH, PM, ABS, and SPB provided funding for this study. JH, PM, and ABS supervised the study. MAN did statistical and association analyses. RF, MAN, and JH analysed and interpreted the data. AR helped in the interpretation of the e/mQTL data. RF, MAN, JH, and PM wrote the first draft of the paper. All other co-authors participated in preparation of the paper by reading and commenting on drafts before submission.</div><p class="fn sec" id="FN5"> </p><p id="P36" class="p p-first"><strong>Declaration of interests</strong></p><p id="P37" class="p p-last">RF, DGH, MAN, JDR, AR, JBJK, CDS, WSB, GMH, JRH, OP, LB, ET, EH, IH, AR, MB, BB, AP, CC, NJC, LB, GB, RG, GF, DG, CF, MS, ES, JC, AL, RB, MLW, KN, CN, IRAM, GYRH, DMAM, JG, CMM, JA, TDG, IGM, AJT, PP, EDH, EMW, AB, EJ, MCT, PP, CR, SOC, EA, RP, JDS, PA, AK, IR, ER, LP, ER, PStGH, GR, FT, GG, JBR, JCMS, JU, JC, SM, AD, VMVD, MG, JQT, JvdZ, WD, TVL, SFC, ILB, DH, VG, MV, AB, BN, SS, SB, IP, JEN, LEH, MR, MM, BI, GG, SP, WG, MNR, NCF, JDW, MGS, HRM, PR, PH, JSS, SR, AR, AG, ACB, RM, FF, CC, LB, MA, MG, MEC, NS, MB, KAJ, JEP, WWS, AMK, HR, JCvS, EGPD, HS, YALP, PS, GL, RC, VN, AAP, MF, AP, GM, PS, MK, HHC, CG, FP, AR, VD, FL, DK, LF, SPB, JH, PM, and ABS declare no competing interests.</p><p></p></div></div><div id="idm140164559889440" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140164559889440title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Neary D, Snowden JS, Gustafson L, et al.  Frontotemporal lobar degeneration: a consensus on clinical diagnostic criteria. <span><span class="ref-journal">Neurology. </span>1998;<span class="ref-vol">51</span>:1546–1554.</span> [<a href="/pubmed/9855500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Frontotemporal+lobar+degeneration:+a+consensus+on+clinical+diagnostic+criteria&amp;author=D+Neary&amp;author=JS+Snowden&amp;author=L+Gustafson&amp;volume=51&amp;publication_year=1998&amp;pages=1546-1554&amp;pmid=9855500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Rabinovici GD, Miller BL. Frontotemporal lobar degeneration: epidemiology, pathophysiology, diagnosis and management. <span><span class="ref-journal">CNS Drugs. </span>2010;<span class="ref-vol">24</span>:375–398.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2916644/">PMC free article</a>]</span> [<a href="/pubmed/20369906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CNS+Drugs&amp;title=Frontotemporal+lobar+degeneration:+epidemiology,+pathophysiology,+diagnosis+and+management&amp;author=GD+Rabinovici&amp;author=BL+Miller&amp;volume=24&amp;publication_year=2010&amp;pages=375-398&amp;pmid=20369906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Rohrer JD, Warren JD. Phenotypic signatures of genetic frontotemporal dementia. <span><span class="ref-journal">Curr Opin Neurol. </span>2011;<span class="ref-vol">24</span>:542–549.</span> [<a href="/pubmed/21986680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Neurol&amp;title=Phenotypic+signatures+of+genetic+frontotemporal+dementia&amp;author=JD+Rohrer&amp;author=JD+Warren&amp;volume=24&amp;publication_year=2011&amp;pages=542-549&amp;pmid=21986680&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Rascovsky K, Hodges JR, Knopman D, et al.  Sensitivity of revised diagnostic criteria for the behavioural variant of frontotemporal dementia. <span><span class="ref-journal">Brain. </span>2011;<span class="ref-vol">134</span>:2456–2477.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3170532/">PMC free article</a>]</span> [<a href="/pubmed/21810890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Sensitivity+of+revised+diagnostic+criteria+for+the+behavioural+variant+of+frontotemporal+dementia&amp;author=K+Rascovsky&amp;author=JR+Hodges&amp;author=D+Knopman&amp;volume=134&amp;publication_year=2011&amp;pages=2456-2477&amp;pmid=21810890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Gorno–Tempini ML, Hillis AE, Weintraub S, et al.  Classification of primary progressive aphasia and its variants. <span><span class="ref-journal">Neurology. </span>2011;<span class="ref-vol">76</span>:1006–1014.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3059138/">PMC free article</a>]</span> [<a href="/pubmed/21325651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Classification+of+primary+progressive+aphasia+and+its+variants&amp;author=ML+Gorno–Tempini&amp;author=AE+Hillis&amp;author=S+Weintraub&amp;volume=76&amp;publication_year=2011&amp;pages=1006-1014&amp;pmid=21325651&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Mackenzie IR, Neumann M, Baborie A, et al.  A harmonized classification system for FTLD–TDP pathology. <span><span class="ref-journal">Acta Neuropathol. </span>2011;<span class="ref-vol">122</span>:111–113.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3285143/">PMC free article</a>]</span> [<a href="/pubmed/21644037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Neuropathol&amp;title=A+harmonized+classification+system+for+FTLD–TDP+pathology&amp;author=IR+Mackenzie&amp;author=M+Neumann&amp;author=A+Baborie&amp;volume=122&amp;publication_year=2011&amp;pages=111-113&amp;pmid=21644037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Halliday G, Bigio EH, Cairns NJ, Neumann M, Mackenzie IR, Mann DM. Mechanisms of disease in frontotemporal lobar degeneration: gain of function versus loss of function effects. <span><span class="ref-journal">Acta Neuropathol. </span>2012;<span class="ref-vol">124</span>:373–382.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3445027/">PMC free article</a>]</span> [<a href="/pubmed/22878865" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Neuropathol&amp;title=Mechanisms+of+disease+in+frontotemporal+lobar+degeneration:+gain+of+function+versus+loss+of+function+effects&amp;author=G+Halliday&amp;author=EH+Bigio&amp;author=NJ+Cairns&amp;author=M+Neumann&amp;author=IR+Mackenzie&amp;volume=124&amp;publication_year=2012&amp;pages=373-382&amp;pmid=22878865&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Hutton M, Lendon CL, Rizzu P, et al.  Association of missense and 5′–splice–site mutations in tau with the inherited dementia FTDP–17. <span><span class="ref-journal">Nature. </span>1998;<span class="ref-vol">393</span>:702–705.</span> [<a href="/pubmed/9641683" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Association+of+missense+and+5′–splice–site+mutations+in+tau+with+the+inherited+dementia+FTDP–17&amp;author=M+Hutton&amp;author=CL+Lendon&amp;author=P+Rizzu&amp;volume=393&amp;publication_year=1998&amp;pages=702-705&amp;pmid=9641683&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Baker M, Mackenzie IR, Pickering-Brown SM, et al.  Mutations in progranulin cause tau–negative frontotemporal dementia linked to chromosome 17. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">442</span>:916–919.</span> [<a href="/pubmed/16862116" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mutations+in+progranulin+cause+tau–negative+frontotemporal+dementia+linked+to+chromosome+17&amp;author=M+Baker&amp;author=IR+Mackenzie&amp;author=SM+Pickering-Brown&amp;volume=442&amp;publication_year=2006&amp;pages=916-919&amp;pmid=16862116&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Cruts M, Gijselinck I, van der Zee J, et al.  Null mutations in progranulin cause ubiquitin-positive frontotemporal dementia linked to chromosome 17q21. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">442</span>:920–924.</span> [<a href="/pubmed/16862115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Null+mutations+in+progranulin+cause+ubiquitin-positive+frontotemporal+dementia+linked+to+chromosome+17q21&amp;author=M+Cruts&amp;author=I+Gijselinck&amp;author=J+van+der+Zee&amp;volume=442&amp;publication_year=2006&amp;pages=920-924&amp;pmid=16862115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">DeJesus–Hernandez M, Mackenzie IR, Boeve BF, et al.  Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS. <span><span class="ref-journal">Neuron. </span>2011;<span class="ref-vol">72</span>:245–256.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3202986/">PMC free article</a>]</span> [<a href="/pubmed/21944778" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Expanded+GGGGCC+hexanucleotide+repeat+in+noncoding+region+of+C9ORF72+causes+chromosome+9p-linked+FTD+and+ALS&amp;author=M+DeJesus–Hernandez&amp;author=IR+Mackenzie&amp;author=BF+Boeve&amp;volume=72&amp;publication_year=2011&amp;pages=245-256&amp;pmid=21944778&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Renton AE, Majounie E, Waite A, et al.  A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p21-linked ALS-FTD. <span><span class="ref-journal">Neuron. </span>2011;<span class="ref-vol">72</span>:257–268.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3200438/">PMC free article</a>]</span> [<a href="/pubmed/21944779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=A+hexanucleotide+repeat+expansion+in+C9ORF72+is+the+cause+of+chromosome+9p21-linked+ALS-FTD&amp;author=AE+Renton&amp;author=E+Majounie&amp;author=A+Waite&amp;volume=72&amp;publication_year=2011&amp;pages=257-268&amp;pmid=21944779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Ferrari R, Thumma A, Momeni P.  <span class="ref-journal">eLS.</span> Chichester: John Wiley &amp; Sons, Ltd; 2013. Molecular Genetics of Frontotemporal Dementia. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=eLS&amp;author=R+Ferrari&amp;author=A+Thumma&amp;author=P+Momeni&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">van der Zee J, Gijselinck I, Dillen L, et al.  A Pan-European Study of the C9orf72 Repeat Associated with FTLD: Geographic Prevalence, Genomic Instability, and Intermediate Repeats. <span><span class="ref-journal">Hum mutat. </span>2013;<span class="ref-vol">34</span>:363–373.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3638346/">PMC free article</a>]</span> [<a href="/pubmed/23111906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+mutat&amp;title=A+Pan-European+Study+of+the+C9orf72+Repeat+Associated+with+FTLD:+Geographic+Prevalence,+Genomic+Instability,+and+Intermediate+Repeats&amp;author=J+van+der+Zee&amp;author=I+Gijselinck&amp;author=L+Dillen&amp;volume=34&amp;publication_year=2013&amp;pages=363-373&amp;pmid=23111906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Rohrer JD, Rosen HJ. Neuroimaging in frontotemporal dementia. <span><span class="ref-journal">Int Rev Psychiatry. </span>2013;<span class="ref-vol">25</span>:221–229.</span> [<a href="/pubmed/23611351" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+Rev+Psychiatry&amp;title=Neuroimaging+in+frontotemporal+dementia&amp;author=JD+Rohrer&amp;author=HJ+Rosen&amp;volume=25&amp;publication_year=2013&amp;pages=221-229&amp;pmid=23611351&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Ling SC, Polymenidou M, Cleveland DW. Converging Mechanisms in ALS and FTD: Disrupted RNA and Protein Homeostasis. <span><span class="ref-journal">Neuron. </span>2013;<span class="ref-vol">79</span>:416–438.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4411085/">PMC free article</a>]</span> [<a href="/pubmed/23931993" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Converging+Mechanisms+in+ALS+and+FTD:+Disrupted+RNA+and+Protein+Homeostasis&amp;author=SC+Ling&amp;author=M+Polymenidou&amp;author=DW+Cleveland&amp;volume=79&amp;publication_year=2013&amp;pages=416-438&amp;pmid=23931993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Van Deerlin VM, Sleiman PM, Martinez–Lage M, et al.  Common variants at 7p21 are associated with frontotemporal lobar degeneration with TDP-43 inclusions. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:234–239.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2828525/">PMC free article</a>]</span> [<a href="/pubmed/20154673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+7p21+are+associated+with+frontotemporal+lobar+degeneration+with+TDP-43+inclusions&amp;author=VM+Van+Deerlin&amp;author=PM+Sleiman&amp;author=M+Martinez–Lage&amp;volume=42&amp;publication_year=2010&amp;pages=234-239&amp;pmid=20154673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Strong MJ, Grace GM, Freedman M, et al.  Consensus criteria for the diagnosis of frontotemporal cognitive and behavioural syndromes in amyotrophic lateral sclerosis. <span><span class="ref-journal">Amyotroph Lateral Scler. </span>2009;<span class="ref-vol">10</span>:131–146.</span> [<a href="/pubmed/19462523" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Amyotroph+Lateral+Scler&amp;title=Consensus+criteria+for+the+diagnosis+of+frontotemporal+cognitive+and+behavioural+syndromes+in+amyotrophic+lateral+sclerosis&amp;author=MJ+Strong&amp;author=GM+Grace&amp;author=M+Freedman&amp;volume=10&amp;publication_year=2009&amp;pages=131-146&amp;pmid=19462523&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">International Parkinson Disease Genomics Consortium. Nalls MA, Plagnol V, et al.  Imputation of sequence variants for identification of genetic risks for Parkinson’s disease: a meta-analysis of genome-wide association studies. <span><span class="ref-journal">Lancet. </span>2011;<span class="ref-vol">377</span>:641–649.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696507/">PMC free article</a>]</span> [<a href="/pubmed/21292315" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Imputation+of+sequence+variants+for+identification+of+genetic+risks+for+Parkinson’s+disease:+a+meta-analysis+of+genome-wide+association+studies&amp;author=MA+Nalls&amp;author=V+Plagnol&amp;volume=377&amp;publication_year=2011&amp;pages=641-649&amp;pmid=21292315&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">1000 Genomes Project Consortium. Abecasis GR, Auton A, et al.  An integrated map of genetic variation from 1092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1092+human+genomes&amp;author=GR+Abecasis&amp;author=A+Auton&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">International HapMap 3 Consortium. Altshuler DM, Gibbs RA, et al.  Integrating common and rare genetic variation in diverse human populations. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:52–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173859/">PMC free article</a>]</span> [<a href="/pubmed/20811451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrating+common+and+rare+genetic+variation+in+diverse+human+populations&amp;author=DM+Altshuler&amp;author=RA+Gibbs&amp;volume=467&amp;publication_year=2010&amp;pages=52-58&amp;pmid=20811451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nature Gen. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Gen&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Grove ML, Yu B, Cochran BJ, et al.  Best Practices and Joint Calling of the HumanExome BeadChip: The CHARGE Consortium. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>:e68095.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3709915/">PMC free article</a>]</span> [<a href="/pubmed/23874508" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Best+Practices+and+Joint+Calling+of+the+HumanExome+BeadChip:+The+CHARGE+Consortium&amp;author=ML+Grove&amp;author=B+Yu&amp;author=BJ+Cochran&amp;volume=8&amp;publication_year=2013&amp;pages=e68095&amp;pmid=23874508&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Millar T, Walker R, Arango JC, et al.  Tissue and organ donation for research in forensic pathology: the MRC Sudden Death Brainand Tissue Bank. <span><span class="ref-journal">J Pathol. </span>2007;<span class="ref-vol">213</span>:369–375.</span> [<a href="/pubmed/17990279" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pathol&amp;title=Tissue+and+organ+donation+for+research+in+forensic+pathology:+the+MRC+Sudden+Death+Brainand+Tissue+Bank&amp;author=T+Millar&amp;author=R+Walker&amp;author=JC+Arango&amp;volume=213&amp;publication_year=2007&amp;pages=369-375&amp;pmid=17990279&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Beach TG, Sue LI, Walker DG, et al.  The Sun Health Research Institute Brain Donation Program: description and experience, 1987–2007. <span><span class="ref-journal">Cell Tissue Bank. </span>2008;<span class="ref-vol">9</span>:229–245.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2493521/">PMC free article</a>]</span> [<a href="/pubmed/18347928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Tissue+Bank&amp;title=The+Sun+Health+Research+Institute+Brain+Donation+Program:+description+and+experience,+1987–2007&amp;author=TG+Beach&amp;author=LI+Sue&amp;author=DG+Walker&amp;volume=9&amp;publication_year=2008&amp;pages=229-245&amp;pmid=18347928&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Hawrylycz MJ, Lein ES, Guillozet–Bongaarts AL, et al.  An anatomically comprehensive atlas of the adult human brain transcriptome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:391–399.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4243026/">PMC free article</a>]</span> [<a href="/pubmed/22996553" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+anatomically+comprehensive+atlas+of+the+adult+human+brain+transcriptome&amp;author=MJ+Hawrylycz&amp;author=ES+Lein&amp;author=AL+Guillozet–Bongaarts&amp;volume=489&amp;publication_year=2012&amp;pages=391-399&amp;pmid=22996553&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Kang HJ, Kawasawa YI, Cheng F, et al.  Spatio-temporal transcriptome of the human brain. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">478</span>:483–489.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3566780/">PMC free article</a>]</span> [<a href="/pubmed/22031440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Spatio-temporal+transcriptome+of+the+human+brain&amp;author=HJ+Kang&amp;author=YI+Kawasawa&amp;author=F+Cheng&amp;volume=478&amp;publication_year=2011&amp;pages=483-489&amp;pmid=22031440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Roth RB, Hevezi P, Lee J, et al.  Gene expression analyses reveal molecular relationships among 20 regions of the human CNS. <span><span class="ref-journal">Neurogenetics. </span>2006;<span class="ref-vol">7</span>:67–80.</span> [<a href="/pubmed/16572319" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurogenetics&amp;title=Gene+expression+analyses+reveal+molecular+relationships+among+20+regions+of+the+human+CNS&amp;author=RB+Roth&amp;author=P+Hevezi&amp;author=J+Lee&amp;volume=7&amp;publication_year=2006&amp;pages=67-80&amp;pmid=16572319&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Trabzuni D, Ryten M, Walker R, et al.  Quality control parameters on a large dataset of regionally dissected human control brains for whole genome expression studies. <span><span class="ref-journal">J Neurochem. </span>2011;<span class="ref-vol">119</span>:275–282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3664422/">PMC free article</a>]</span> [<a href="/pubmed/21848658" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurochem&amp;title=Quality+control+parameters+on+a+large+dataset+of+regionally+dissected+human+control+brains+for+whole+genome+expression+studies&amp;author=D+Trabzuni&amp;author=M+Ryten&amp;author=R+Walker&amp;volume=119&amp;publication_year=2011&amp;pages=275-282&amp;pmid=21848658&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Irizarry RA, Hobbs B, Collin F, et al.  Exploration, normalization, and summaries of high density oligonucleotide array probe level data. <span><span class="ref-journal">Biostatistics. </span>2003;<span class="ref-vol">4</span>:249–264.</span> [<a href="/pubmed/12925520" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biostatistics&amp;title=Exploration,+normalization,+and+summaries+of+high+density+oligonucleotide+array+probe+level+data&amp;author=RA+Irizarry&amp;author=B+Hobbs&amp;author=F+Collin&amp;volume=4&amp;publication_year=2003&amp;pages=249-264&amp;pmid=12925520&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Ramasamy A, Trabzuni D, Gibbs JR, et al.  Resolving the polymorphism-in-probe problem is critical for correct interpretation of expression QTL studies. <span><span class="ref-journal">Nucleic Acids Res. </span>2013;<span class="ref-vol">41</span>:e88.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3627570/">PMC free article</a>]</span> [<a href="/pubmed/23435227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Resolving+the+polymorphism-in-probe+problem+is+critical+for+correct+interpretation+of+expression+QTL+studies&amp;author=A+Ramasamy&amp;author=D+Trabzuni&amp;author=JR+Gibbs&amp;volume=41&amp;publication_year=2013&amp;pages=e88&amp;pmid=23435227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">International Parkinson’s Disease Genomics Consortium (IPDGC); Wellcome Trust Case Control Consortium 2 (WTCCC2) A two-stage meta-analysis identifies several new loci for Parkinson’s disease. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002142.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128098/">PMC free article</a>]</span> [<a href="/pubmed/21738488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+two-stage+meta-analysis+identifies+several+new+loci+for+Parkinson’s+disease&amp;volume=7&amp;publication_year=2011&amp;pages=e1002142&amp;pmid=21738488&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Li Y, Willer C, Sanna S, Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Genomics+Hum+Genet&amp;title=Genotype+imputation&amp;author=Y+Li&amp;author=C+Willer&amp;author=S+Sanna&amp;author=G+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2010;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y+Li&amp;author=CJ+Willer&amp;author=J+Ding&amp;author=P+Scheet&amp;author=GR+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Gibbs JR, van der Brug MP, Hernandez DG, et al.  Abundant quantitative trait loci exist for DNA methylation and gene expression in human brain. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000952.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2869317/">PMC free article</a>]</span> [<a href="/pubmed/20485568" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Abundant+quantitative+trait+loci+exist+for+DNA+methylation+and+gene+expression+in+human+brain&amp;author=JR+Gibbs&amp;author=MP+van+der+Brug&amp;author=DG+Hernandez&amp;volume=6&amp;publication_year=2010&amp;pages=e1000952&amp;pmid=20485568&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Barbosa-Morais NL, Dunning MJ, Samarajiwa SA, et al.  A re-annotation pipeline for Illumina BeadArrays: improving the interpretation of gene expression data. <span><span class="ref-journal">Nucleic Acids Res. </span>2010;<span class="ref-vol">38</span>:e1.7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2817484/">PMC free article</a>]</span> [<a href="/pubmed/19923232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=A+re-annotation+pipeline+for+Illumina+BeadArrays:+improving+the+interpretation+of+gene+expression+data&amp;author=NL+Barbosa-Morais&amp;author=MJ+Dunning&amp;author=SA+Samarajiwa&amp;volume=38&amp;publication_year=2010&amp;pages=e1.7&amp;pmid=19854949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Shabalin AA. Matrix eQTL: Ultra fast eQTL analysis via large matrix operations. <span><span class="ref-journal">Bioinformatics. </span>2012;<span class="ref-vol">28</span>:1353–1358.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3348564/">PMC free article</a>]</span> [<a href="/pubmed/22492648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Matrix+eQTL:+Ultra+fast+eQTL+analysis+via+large+matrix+operations&amp;author=AA+Shabalin&amp;volume=28&amp;publication_year=2012&amp;pages=1353-1358&amp;pmid=22492648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Zeller T, Wild P, Szymczak S, et al.  Genetics and beyond the transcriptome of human monocytes and disease susceptibility. <span><span class="ref-journal">PLoS One. </span>2010;<span class="ref-vol">5</span>:e10693.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2872668/">PMC free article</a>]</span> [<a href="/pubmed/20502693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Genetics+and+beyond+the+transcriptome+of+human+monocytes+and+disease+susceptibility&amp;author=T+Zeller&amp;author=P+Wild&amp;author=S+Szymczak&amp;volume=5&amp;publication_year=2010&amp;pages=e10693&amp;pmid=20502693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">van Es MA, Veldink JH, Saris CG, et al.  Genome-wide association study identifies 19p13.3 (UNC13A) and 9p21.2 as susceptibility loci for sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1083–1087.</span> [<a href="/pubmed/19734901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+19p13.3+(UNC13A)+and+9p21.2+as+susceptibility+loci+for+sporadic+amyotrophic+lateral+sclerosis&amp;author=MA+van+Es&amp;author=JH+Veldink&amp;author=CG+Saris&amp;volume=41&amp;publication_year=2009&amp;pages=1083-1087&amp;pmid=19734901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Hölinger GU, Melhem NM, Dickson DW, et al.  Identification of common variants influencing risk of the tauopathy progressive supranuclear palsy. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:699–705.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3125476/">PMC free article</a>]</span> [<a href="/pubmed/21685912" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Identification+of+common+variants+influencing+risk+of+the+tauopathy+progressive+supranuclear+palsy&amp;author=GU+Hölinger&amp;author=NM+Melhem&amp;author=DW+Dickson&amp;volume=43&amp;publication_year=2011&amp;pages=699-705&amp;pmid=21685912&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Seshadri S, Fitzpatrick AL, Ikram MA, et al.  Genome-wide analysis of genetic loci associated with Alzheimer disease. <span><span class="ref-journal">JAMA. </span>2010;<span class="ref-vol">303</span>:1832–1840.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2989531/">PMC free article</a>]</span> [<a href="/pubmed/20460622" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Genome-wide+analysis+of+genetic+loci+associated+with+Alzheimer+disease&amp;author=S+Seshadri&amp;author=AL+Fitzpatrick&amp;author=MA+Ikram&amp;volume=303&amp;publication_year=2010&amp;pages=1832-1840&amp;pmid=20460622&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">van der Zee J, Sleegers K, Van Broeckhoven C. Invited article: the Alzheimer disease-frontotemporal lobar degeneration spectrum. <span><span class="ref-journal">Neurology. </span>2008;<span class="ref-vol">71</span>:1191–1197.</span> [<a href="/pubmed/18838666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Invited+article:+the+Alzheimer+disease-frontotemporal+lobar+degeneration+spectrum&amp;author=J+van+der+Zee&amp;author=K+Sleegers&amp;author=C+Van+Broeckhoven&amp;volume=71&amp;publication_year=2008&amp;pages=1191-1197&amp;pmid=18838666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Baranzini SE, Wang J, Gibson RA, et al.  Genome wide association analysis of susceptibility and clinical phenotype in multiple sclerosis. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:767–778.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4334814/">PMC free article</a>]</span> [<a href="/pubmed/19010793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome+wide+association+analysis+of+susceptibility+and+clinical+phenotype+in+multiple+sclerosis&amp;author=SE+Baranzini&amp;author=J+Wang&amp;author=RA+Gibson&amp;volume=18&amp;publication_year=2009&amp;pages=767-778&amp;pmid=19010793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">International Multiple Sclerosis Genetics Consortium. Hafler DA, Compston A, et al.  Risk alleles for multiple sclerosis identified by a genomewide study. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:851–862.</span> [<a href="/pubmed/17660530" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Risk+alleles+for+multiple+sclerosis+identified+by+a+genomewide+study&amp;author=DA+Hafler&amp;author=A+Compston&amp;volume=357&amp;publication_year=2007&amp;pages=851-862&amp;pmid=17660530&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Mero IL, Gustavsen MW, Sæther HS, et al.  Oligoclonal band status in Scandinavian multiple sclerosis patients is associated with specific genetic risk alleles. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>:e58352.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3589422/">PMC free article</a>]</span> [<a href="/pubmed/23472185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Oligoclonal+band+status+in+Scandinavian+multiple+sclerosis+patients+is+associated+with+specific+genetic+risk+alleles&amp;author=IL+Mero&amp;author=MW+Gustavsen&amp;author=HS+Sæther&amp;volume=8&amp;publication_year=2013&amp;pages=e58352&amp;pmid=23472185&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Hamza TH, Zabetian CP, Tenesa A, et al.  Common genetic variation in the HLA region is associated with late-onset sporadic Parkinson’s disease. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:781–785.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2930111/">PMC free article</a>]</span> [<a href="/pubmed/20711177" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+genetic+variation+in+the+HLA+region+is+associated+with+late-onset+sporadic+Parkinson’s+disease&amp;author=TH+Hamza&amp;author=CP+Zabetian&amp;author=A+Tenesa&amp;volume=42&amp;publication_year=2010&amp;pages=781-785&amp;pmid=20711177&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Lambert JC, Ibrahim-Verbaas CA, Harold D, et al.  Meta-analysis of 74046 individuals identifies 11 new susceptibility loci for Alzheimer’s disease. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:1452–1458.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3896259/">PMC free article</a>]</span> [<a href="/pubmed/24162737" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+74046+individuals+identifies+11+new+susceptibility+loci+for+Alzheimer’s+disease&amp;author=JC+Lambert&amp;author=CA+Ibrahim-Verbaas&amp;author=D+Harold&amp;volume=45&amp;publication_year=2013&amp;pages=1452-1458&amp;pmid=24162737&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Jäger D, Stockert E, Jäger E, et al.  Serological cloning of a melanocyte rab guanosine 5-prime-triphosphate-binding protein and a chromosome condensation protein from a melanoma complementary DNA library. <span><span class="ref-journal">Cancer Res. </span>2000;<span class="ref-vol">60</span>:3584–3359.</span> [<a href="/pubmed/10910072" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Serological+cloning+of+a+melanocyte+rab+guanosine+5-prime-triphosphate-binding+protein+and+a+chromosome+condensation+protein+from+a+melanoma+complementary+DNA+library&amp;author=D+Jäger&amp;author=E+Stockert&amp;author=E+Jäger&amp;volume=60&amp;publication_year=2000&amp;pages=3584-3359&amp;pmid=10910072&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Bultema JJ, Ambrosio AL, Burek CL, Di Pietro SM. BLOC-2, AP-3, and AP-1 proteins function in concert with Rab38 and Rab32 proteins to mediate protein trafficking to lysosome–related organelles. <span><span class="ref-journal">J Biol Chem. </span>2012;<span class="ref-vol">287</span>:19550–19563.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3365991/">PMC free article</a>]</span> [<a href="/pubmed/22511774" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=BLOC-2,+AP-3,+and+AP-1+proteins+function+in+concert+with+Rab38+and+Rab32+proteins+to+mediate+protein+trafficking+to+lysosome–related+organelles&amp;author=JJ+Bultema&amp;author=AL+Ambrosio&amp;author=CL+Burek&amp;author=SM+Di+Pietro&amp;volume=287&amp;publication_year=2012&amp;pages=19550-19563&amp;pmid=22511774&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Seto S, Tsujimura K, Koide Y. Rab GTPAses regulating phagosome maturation are differentially recruited to mycobacterial phagosomes. <span><span class="ref-journal">Traffic. </span>2011;<span class="ref-vol">12</span>:407–420.</span> [<a href="/pubmed/21255211" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Rab+GTPAses+regulating+phagosome+maturation+are+differentially+recruited+to+mycobacterial+phagosomes&amp;author=S+Seto&amp;author=K+Tsujimura&amp;author=Y+Koide&amp;volume=12&amp;publication_year=2011&amp;pages=407-420&amp;pmid=21255211&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Hu F, Padukkavidana T, Vægter CB, et al.  Sortilin–mediated endocytosis determines levels of the frontotemporal dementia protein, progranulin. <span><span class="ref-journal">Neuron. </span>2010;<span class="ref-vol">68</span>:654–667.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2990962/">PMC free article</a>]</span> [<a href="/pubmed/21092856" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Sortilin–mediated+endocytosis+determines+levels+of+the+frontotemporal+dementia+protein,+progranulin&amp;author=F+Hu&amp;author=T+Padukkavidana&amp;author=CB+Vægter&amp;volume=68&amp;publication_year=2010&amp;pages=654-667&amp;pmid=21092856&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Brady OA, Zheng Y, Murphy K, Huang M, Hu F. The frontotemporal lobar degeneration risk factor, TMEM106B, regulates lysosomal morphology and function. <span><span class="ref-journal">Hum Mol Genet. </span>2013;<span class="ref-vol">22</span>:685–695.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3554197/">PMC free article</a>]</span> [<a href="/pubmed/23136129" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=The+frontotemporal+lobar+degeneration+risk+factor,+TMEM106B,+regulates+lysosomal+morphology+and+function&amp;author=OA+Brady&amp;author=Y+Zheng&amp;author=K+Murphy&amp;author=M+Huang&amp;author=F+Hu&amp;volume=22&amp;publication_year=2013&amp;pages=685-695&amp;pmid=23136129&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Westbroek W, Gustafson AM, Sidransky E. Exploring the link between glucocerebrosidase mutations and parkinsonism. <span><span class="ref-journal">Trends Mol Med. </span>2011;<span class="ref-vol">17</span>:485–493.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3351003/">PMC free article</a>]</span> [<a href="/pubmed/21723784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Mol+Med&amp;title=Exploring+the+link+between+glucocerebrosidase+mutations+and+parkinsonism&amp;author=W+Westbroek&amp;author=AM+Gustafson&amp;author=E+Sidransky&amp;volume=17&amp;publication_year=2011&amp;pages=485-493&amp;pmid=21723784&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Hardy J1, Rogaeva E. Motor neuron disease and frontotemporal dementia: sometimes related, sometimes not. <span><span class="ref-journal">Exp Neurol. </span>2013</span> pii: S0014–4886(13)00330–0. [<a href="/pubmed/24246281" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp+Neurol&amp;title=Motor+neuron+disease+and+frontotemporal+dementia:+sometimes+related,+sometimes+not&amp;author=J1+Hardy&amp;author=E+Rogaeva&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Amor S, Woodroof N. Review series on immune responses in neurodegenerative diseases: innate and adaptive immune responses in neurodegeneration and repair. <span><span class="ref-journal">Immunology. </span>2013</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunology&amp;title=Review+series+on+immune+responses+in+neurodegenerative+diseases:+innate+and+adaptive+immune+responses+in+neurodegeneration+and+repair&amp;author=S+Amor&amp;author=N+Woodroof&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Träger U, Tabrizi SJ. Peripheral inflammation in neurodegeneration. <span><span class="ref-journal">J Mol Med. </span>2013;<span class="ref-vol">91</span>:673–681.</span> [<a href="/pubmed/23546523" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Med&amp;title=Peripheral+inflammation+in+neurodegeneration&amp;author=U+Träger&amp;author=SJ+Tabrizi&amp;volume=91&amp;publication_year=2013&amp;pages=673-681&amp;pmid=23546523&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4112126&amp;issue-id=240908&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140164637996784title">Summary</a></li><li><a href="#S6title">Introduction</a></li><li><a href="#S7title">Methods</a></li><li><a href="#S12title">Results</a></li><li><a href="#S13title">Discussion</a></li><li><a href="#S14title">Acknowledgments</a></li><li><a href="#idm140164555515072title">Footnotes</a></li><li><a href="#idm140164559889440title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4112126/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4112126/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4112126/pdf/nihms599470.pdf">PDF (829K)</a></li> | <li><a href="#" data-citationid="PMC4112126" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4112126%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4112126%2F&amp;text=Frontotemporal%20dementia%20and%20its%20subtypes%3A%20a%20genome-wide%20association%20study" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4112126%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24943344" data-aiid="4112126" data-aid="4112126" data-iid="240908" data-domainid="319" data-domain="nihpa" data-accid="PMC4112126" data-md5="83ee88155e50ad88c09e24f1662afe5e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:21:09-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE8815A8DD5723D10000000002B8019D&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4112126%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4112126/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>