<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4480365/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Meta-analysis of Genome-Wide Association Studies Identifies Novel Loci Associated With Optic Disc Morphology</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4480365/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Genetic epidemiology"><meta name="citation_title" content="Meta-analysis of Genome-Wide Association Studies Identifies Novel Loci Associated With Optic Disc Morphology"><meta name="citation_authors" content="Henriët Springelkamp, Aniket Mishra, Pirro G. Hysi, Puya Gharahkhani, René Höhn, Chiea-Chuen Khor, Jessica N. Cooke Bailey, Xiaoyan Luo, Wishal D. Ramdas, Eranga Vithana, Victor Koh, Seyhan Yazar, Liang Xu, Hannah Forward, Lisa S. Kearns, Najaf Amin, Adriana I. Iglesias, Kar-Seng Sim, Elisabeth M. van Leeuwen, Ayse Demirkan, Sven van der Lee, Seng-Chee Loon, Fernando Rivadeneira, Abhishek Nag, Paul G. Sanfilippo, Arne Schillert, Paulus T. V. M. de Jong, Ben A. Oostra, André G. Uitterlinden, Albert Hofman, NEIGHBORHOOD Consortium, Tiger Zhou, Kathryn P. Burdon, Timothy D. Spector, Karl J. Lackner, Seang-Mei Saw, Johannes R. Vingerling, Yik-Ying Teo, Louis R. Pasquale, Roger C. W. Wolfs, Hans G. Lemij, E-Shyong Tai, Jost B. Jonas, Ching-Yu Cheng, Tin Aung, Nomdo M. Jansonius, Caroline C. W. Klaver, Jamie E. Craig, Terri L. Young, Jonathan L. Haines, Stuart MacGregor, David A. Mackey, Norbert Pfeiffer, Tien-Yin Wong, Janey L. Wiggs, Alex W. Hewitt, Cornelia M. van Duijn, Christopher J. Hammond"><meta name="citation_date" content="March 2015"><meta name="citation_issue" content="3"><meta name="citation_volume" content="39"><meta name="citation_firstpage" content="207"><meta name="citation_doi" content="10.1002/gepi.21886"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4480365/?report=abstract"><meta name="citation_pmid" content="25631615"><meta name="DC.Title" content="Meta-analysis of Genome-Wide Association Studies Identifies Novel Loci Associated With Optic Disc Morphology"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Henriët Springelkamp"><meta name="DC.Contributor" content="Aniket Mishra"><meta name="DC.Contributor" content="Pirro G. Hysi"><meta name="DC.Contributor" content="Puya Gharahkhani"><meta name="DC.Contributor" content="René Höhn"><meta name="DC.Contributor" content="Chiea-Chuen Khor"><meta name="DC.Contributor" content="Jessica N. Cooke Bailey"><meta name="DC.Contributor" content="Xiaoyan Luo"><meta name="DC.Contributor" content="Wishal D. Ramdas"><meta name="DC.Contributor" content="Eranga Vithana"><meta name="DC.Contributor" content="Victor Koh"><meta name="DC.Contributor" content="Seyhan Yazar"><meta name="DC.Contributor" content="Liang Xu"><meta name="DC.Contributor" content="Hannah Forward"><meta name="DC.Contributor" content="Lisa S. Kearns"><meta name="DC.Contributor" content="Najaf Amin"><meta name="DC.Contributor" content="Adriana I. Iglesias"><meta name="DC.Contributor" content="Kar-Seng Sim"><meta name="DC.Contributor" content="Elisabeth M. van Leeuwen"><meta name="DC.Contributor" content="Ayse Demirkan"><meta name="DC.Contributor" content="Sven van der Lee"><meta name="DC.Contributor" content="Seng-Chee Loon"><meta name="DC.Contributor" content="Fernando Rivadeneira"><meta name="DC.Contributor" content="Abhishek Nag"><meta name="DC.Contributor" content="Paul G. Sanfilippo"><meta name="DC.Contributor" content="Arne Schillert"><meta name="DC.Contributor" content="Paulus T. V. M. de Jong"><meta name="DC.Contributor" content="Ben A. Oostra"><meta name="DC.Contributor" content="André G. Uitterlinden"><meta name="DC.Contributor" content="Albert Hofman"><meta name="DC.Contributor" content="NEIGHBORHOOD Consortium"><meta name="DC.Contributor" content="Tiger Zhou"><meta name="DC.Contributor" content="Kathryn P. Burdon"><meta name="DC.Contributor" content="Timothy D. Spector"><meta name="DC.Contributor" content="Karl J. Lackner"><meta name="DC.Contributor" content="Seang-Mei Saw"><meta name="DC.Contributor" content="Johannes R. Vingerling"><meta name="DC.Contributor" content="Yik-Ying Teo"><meta name="DC.Contributor" content="Louis R. Pasquale"><meta name="DC.Contributor" content="Roger C. W. Wolfs"><meta name="DC.Contributor" content="Hans G. Lemij"><meta name="DC.Contributor" content="E-Shyong Tai"><meta name="DC.Contributor" content="Jost B. Jonas"><meta name="DC.Contributor" content="Ching-Yu Cheng"><meta name="DC.Contributor" content="Tin Aung"><meta name="DC.Contributor" content="Nomdo M. Jansonius"><meta name="DC.Contributor" content="Caroline C. W. Klaver"><meta name="DC.Contributor" content="Jamie E. Craig"><meta name="DC.Contributor" content="Terri L. Young"><meta name="DC.Contributor" content="Jonathan L. Haines"><meta name="DC.Contributor" content="Stuart MacGregor"><meta name="DC.Contributor" content="David A. Mackey"><meta name="DC.Contributor" content="Norbert Pfeiffer"><meta name="DC.Contributor" content="Tien-Yin Wong"><meta name="DC.Contributor" content="Janey L. Wiggs"><meta name="DC.Contributor" content="Alex W. Hewitt"><meta name="DC.Contributor" content="Cornelia M. van Duijn"><meta name="DC.Contributor" content="Christopher J. Hammond"><meta name="DC.Date" content="2015 Mar"><meta name="DC.Identifier" content="10.1002/gepi.21886"><meta name="DC.Language" content="en"><meta property="og:title" content="Meta-analysis of Genome-Wide Association Studies Identifies Novel Loci Associated With Optic Disc Morphology"><meta property="og:type" content="article"><meta property="og:description" content="Primary open-angle glaucoma is the most common optic neuropathy and an important cause of irreversible blindness worldwide. The optic nerve head or optic disc is divided in two parts: a central cup (without nerve fibers) surrounded by the neuroretinal ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4480365/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4480365/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4480365/pdf/nihms700469.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C5394DD5764F10000000008160501.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4480365%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4480365</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Genet Epidemiol</a></span>. Author manuscript; available in PMC 2016 Mar 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25631615" target="pmc_ext" ref="reftype=publisher&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Genet Epidemiol. 2015 Mar; 39(3): 207–216. </span></a></div><span class="fm-vol-iss-date">Published online 2015 Jan 28. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fgepi.21886" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/gepi.21886</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4480365</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS700469</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25631615">25631615</a></div></div></div></div><h1 class="content-title">Meta-analysis of Genome-Wide Association Studies Identifies Novel Loci Associated With Optic Disc Morphology</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Springelkamp%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451260432" co-class="co-affbox">Henriët Springelkamp</a>,<sup>1,</sup><sup>2,</sup><sup>‡</sup> <a href="/pubmed/?term=Mishra%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153521648720" co-class="co-affbox">Aniket Mishra</a>,<sup>3,</sup><sup>‡</sup> <a href="/pubmed/?term=Hysi%20PG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451256320" co-class="co-affbox">Pirro G. Hysi</a>,<sup>4,</sup><sup>‡</sup> <a href="/pubmed/?term=Gharahkhani%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153452052448" co-class="co-affbox">Puya Gharahkhani</a>,<sup>3,</sup><sup>‡</sup> <a href="/pubmed/?term=H%26%23x000f6%3Bhn%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153452049728" co-class="co-affbox">René Höhn</a>,<sup>5,</sup><sup>6,</sup><sup>‡</sup> <a href="/pubmed/?term=Khor%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153452046272" co-class="co-affbox">Chiea-Chuen Khor</a>,<sup>7,</sup><sup>8,</sup><sup>‡</sup> <a href="/pubmed/?term=Cooke%20Bailey%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153449696192" co-class="co-affbox">Jessica N. Cooke Bailey</a>,<sup>9,</sup><sup>‡</sup> <a href="/pubmed/?term=Luo%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153449693472" co-class="co-affbox">Xiaoyan Luo</a>,<sup>10</sup> <a href="/pubmed/?term=Ramdas%20WD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153449691488" co-class="co-affbox">Wishal D. Ramdas</a>,<sup>2</sup> <a href="/pubmed/?term=Vithana%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153450734064" co-class="co-affbox">Eranga Vithana</a>,<sup>7,</sup><sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Koh%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153450730608" co-class="co-affbox">Victor Koh</a>,<sup>7</sup> <a href="/pubmed/?term=Yazar%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153450728624" co-class="co-affbox">Seyhan Yazar</a>,<sup>13</sup> <a href="/pubmed/?term=Xu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153452951344" co-class="co-affbox">Liang Xu</a>,<sup>14,</sup><sup>15</sup> <a href="/pubmed/?term=Forward%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153452948624" co-class="co-affbox">Hannah Forward</a>,<sup>13</sup> <a href="/pubmed/?term=Kearns%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153452946640" co-class="co-affbox">Lisa S. Kearns</a>,<sup>16</sup> <a href="/pubmed/?term=Amin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153452944656" co-class="co-affbox">Najaf Amin</a>,<sup>1</sup> <a href="/pubmed/?term=Iglesias%20AI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153453313456" co-class="co-affbox">Adriana I. Iglesias</a>,<sup>1</sup> <a href="/pubmed/?term=Sim%20KS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153453311472" co-class="co-affbox">Kar-Seng Sim</a>,<sup>8</sup> <a href="/pubmed/?term=van%20Leeuwen%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153453309488" co-class="co-affbox">Elisabeth M. van Leeuwen</a>,<sup>1</sup> <a href="/pubmed/?term=Demirkan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153453307504" co-class="co-affbox">Ayse Demirkan</a>,<sup>1,</sup><sup>17</sup> <a href="/pubmed/?term=van%20der%20Lee%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153452899776" co-class="co-affbox">Sven van der Lee</a>,<sup>1</sup> <a href="/pubmed/?term=Loon%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153452897792" co-class="co-affbox">Seng-Chee Loon</a>,<sup>7</sup> <a href="/pubmed/?term=Rivadeneira%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153452895808" co-class="co-affbox">Fernando Rivadeneira</a>,<sup>1,</sup><sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=Nag%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153444593056" co-class="co-affbox">Abhishek Nag</a>,<sup>4</sup> <a href="/pubmed/?term=Sanfilippo%20PG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153444591072" co-class="co-affbox">Paul G. Sanfilippo</a>,<sup>13,</sup><sup>16</sup> <a href="/pubmed/?term=Schillert%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153444588352" co-class="co-affbox">Arne Schillert</a>,<sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=de%20Jong%20PT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153445291456" co-class="co-affbox">Paulus T. V. M. de Jong</a>,<sup>22,</sup><sup>23,</sup><sup>24</sup> <a href="/pubmed/?term=Oostra%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153445288000" co-class="co-affbox">Ben A. Oostra</a>,<sup>25</sup> <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153445286016" co-class="co-affbox">André G. Uitterlinden</a>,<sup>1,</sup><sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451807152" co-class="co-affbox">Albert Hofman</a>,<sup>1,</sup><sup>19</sup> NEIGHBORHOOD Consortium,<sup>†</sup> <a href="/pubmed/?term=Zhou%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451803072" co-class="co-affbox">Tiger Zhou</a>,<sup>26</sup> <a href="/pubmed/?term=Burdon%20KP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451801088" co-class="co-affbox">Kathryn P. Burdon</a>,<sup>27</sup> <a href="/pubmed/?term=Spector%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451799104" co-class="co-affbox">Timothy D. Spector</a>,<sup>4</sup> <a href="/pubmed/?term=Lackner%20KJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153445590208" co-class="co-affbox">Karl J. Lackner</a>,<sup>6</sup> <a href="/pubmed/?term=Saw%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153445588224" co-class="co-affbox">Seang-Mei Saw</a>,<sup>7,</sup><sup>11,</sup><sup>12,</sup><sup>28</sup> <a href="/pubmed/?term=Vingerling%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153445584032" co-class="co-affbox">Johannes R. Vingerling</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Teo%20YY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451126112" co-class="co-affbox">Yik-Ying Teo</a>,<sup>28,</sup><sup>29</sup> <a href="/pubmed/?term=Pasquale%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451123392" co-class="co-affbox">Louis R. Pasquale</a>,<sup>30,</sup><sup>31</sup> <a href="/pubmed/?term=Wolfs%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451120672" co-class="co-affbox">Roger C. W. Wolfs</a>,<sup>2</sup> <a href="/pubmed/?term=Lemij%20HG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451118688" co-class="co-affbox">Hans G. Lemij</a>,<sup>32</sup> <a href="/pubmed/?term=Tai%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153450544016" co-class="co-affbox">E-Shyong Tai</a>,<sup>28,</sup><sup>29,</sup><sup>33</sup> <a href="/pubmed/?term=Jonas%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153450540560" co-class="co-affbox">Jost B. Jonas</a>,<sup>34</sup> <a href="/pubmed/?term=Cheng%20CY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153450538576" co-class="co-affbox">Ching-Yu Cheng</a>,<sup>7,</sup><sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Aung%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153450535120" co-class="co-affbox">Tin Aung</a>,<sup>7,</sup><sup>12</sup> <a href="/pubmed/?term=Jansonius%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153442678368" co-class="co-affbox">Nomdo M. Jansonius</a>,<sup>35</sup> <a href="/pubmed/?term=Klaver%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153442676384" co-class="co-affbox">Caroline C. W. Klaver</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Craig%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153442673664" co-class="co-affbox">Jamie E. Craig</a>,<sup>26</sup> <a href="/pubmed/?term=Young%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153442671680" co-class="co-affbox">Terri L. Young</a>,<sup>10</sup> <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153448540096" co-class="co-affbox">Jonathan L. Haines</a>,<sup>9</sup> <a href="/pubmed/?term=MacGregor%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153448538112" co-class="co-affbox">Stuart MacGregor</a>,<sup>3</sup> <a href="/pubmed/?term=Mackey%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153448536128" co-class="co-affbox">David A. Mackey</a>,<sup>13,</sup><sup>27</sup> <a href="/pubmed/?term=Pfeiffer%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153448533408" co-class="co-affbox">Norbert Pfeiffer</a>,<sup>5,</sup><sup>§</sup> <a href="/pubmed/?term=Wong%20TY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451505472" co-class="co-affbox">Tien-Yin Wong</a>,<sup>7,</sup><sup>11,</sup><sup>12,</sup><sup>§</sup> <a href="/pubmed/?term=Wiggs%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451501280" co-class="co-affbox">Janey L. Wiggs</a>,<sup>31,</sup><sup>§</sup> <a href="/pubmed/?term=Hewitt%20AW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153451498560" co-class="co-affbox">Alex W. Hewitt</a>,<sup>16,</sup><sup>27,</sup><sup>§</sup> <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153449946592" co-class="co-affbox">Cornelia M. van Duijn</a>,<sup>1,</sup><sup>*</sup><sup>§</sup> and  <a href="/pubmed/?term=Hammond%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615" class="affpopup" co-rid="_co_idm140153449943136" co-class="co-affbox">Christopher J. Hammond</a><sup>4,</sup><sup>§</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140153451260432"><h3 class="no_margin">Henriët Springelkamp</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><p><sup>2</sup>Department of Ophthalmology, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Springelkamp%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Henriët Springelkamp</a></div></div><div id="_co_idm140153521648720"><h3 class="no_margin">Aniket Mishra</h3><p><sup>3</sup>Statistical Genetics, QIMR Berghofer Medical Research Institute, Royal Brisbane Hospital, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Mishra%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Aniket Mishra</a></div></div><div id="_co_idm140153451256320"><h3 class="no_margin">Pirro G. Hysi</h3><p><sup>4</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, London, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Hysi%20PG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Pirro G. Hysi</a></div></div><div id="_co_idm140153452052448"><h3 class="no_margin">Puya Gharahkhani</h3><p><sup>3</sup>Statistical Genetics, QIMR Berghofer Medical Research Institute, Royal Brisbane Hospital, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Gharahkhani%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Puya Gharahkhani</a></div></div><div id="_co_idm140153452049728"><h3 class="no_margin">René Höhn</h3><p><sup>5</sup>Department of Ophthalmology, University Medical Center Mainz, Mainz, Germany</p><p><sup>6</sup>Klinik Pallas Olten, Olten, Switzerland</p><div>Find articles by <a href="/pubmed/?term=H%26%23x000f6%3Bhn%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">René Höhn</a></div></div><div id="_co_idm140153452046272"><h3 class="no_margin">Chiea-Chuen Khor</h3><p><sup>7</sup>Department of Ophthalmology, National University of Singapore and National University Health System, Singapore, Singapore</p><p><sup>8</sup>Division of Human Genetics, Genome Institute of Singapore, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Khor%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Chiea-Chuen Khor</a></div></div><div id="_co_idm140153449696192"><h3 class="no_margin">Jessica N. Cooke Bailey</h3><p><sup>9</sup>Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, Ohio, United States of America</p><div>Find articles by <a href="/pubmed/?term=Cooke%20Bailey%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Jessica N. Cooke Bailey</a></div></div><div id="_co_idm140153449693472"><h3 class="no_margin">Xiaoyan Luo</h3><p><sup>10</sup>Duke University Eye Center, Durham, North Carolina, United States of America</p><div>Find articles by <a href="/pubmed/?term=Luo%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Xiaoyan Luo</a></div></div><div id="_co_idm140153449691488"><h3 class="no_margin">Wishal D. Ramdas</h3><p><sup>2</sup>Department of Ophthalmology, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Ramdas%20WD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Wishal D. Ramdas</a></div></div><div id="_co_idm140153450734064"><h3 class="no_margin">Eranga Vithana</h3><p><sup>7</sup>Department of Ophthalmology, National University of Singapore and National University Health System, Singapore, Singapore</p><p><sup>11</sup>Duke-National University of Singapore, Graduate Medical School, Singapore, Singapore</p><p><sup>12</sup>Singapore Eye Research Institute, Singapore National Eye Centre, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Vithana%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Eranga Vithana</a></div></div><div id="_co_idm140153450730608"><h3 class="no_margin">Victor Koh</h3><p><sup>7</sup>Department of Ophthalmology, National University of Singapore and National University Health System, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Koh%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Victor Koh</a></div></div><div id="_co_idm140153450728624"><h3 class="no_margin">Seyhan Yazar</h3><p><sup>13</sup>Centre for Ophthalmology and Visual Science, Lions Eye Institute, University of Western Australia, Perth, Australia</p><div>Find articles by <a href="/pubmed/?term=Yazar%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Seyhan Yazar</a></div></div><div id="_co_idm140153452951344"><h3 class="no_margin">Liang Xu</h3><p><sup>14</sup>Beijing Institute of Ophthalmology, Beijing Tongren Eye Center, Beijing Tongren Hospital, Capital Medical University, Beijing, China</p><p><sup>15</sup>Beijing Ophthalmology and Visual Science Key Lab, Beijing, China</p><div>Find articles by <a href="/pubmed/?term=Xu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Liang Xu</a></div></div><div id="_co_idm140153452948624"><h3 class="no_margin">Hannah Forward</h3><p><sup>13</sup>Centre for Ophthalmology and Visual Science, Lions Eye Institute, University of Western Australia, Perth, Australia</p><div>Find articles by <a href="/pubmed/?term=Forward%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Hannah Forward</a></div></div><div id="_co_idm140153452946640"><h3 class="no_margin">Lisa S. Kearns</h3><p><sup>16</sup>Centre for Eye Research Australia (CERA), University of Melbourne, Royal Victorian Eye and Ear Hospital, Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Kearns%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Lisa S. Kearns</a></div></div><div id="_co_idm140153452944656"><h3 class="no_margin">Najaf Amin</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Amin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Najaf Amin</a></div></div><div id="_co_idm140153453313456"><h3 class="no_margin">Adriana I. Iglesias</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Iglesias%20AI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Adriana I. Iglesias</a></div></div><div id="_co_idm140153453311472"><h3 class="no_margin">Kar-Seng Sim</h3><p><sup>8</sup>Division of Human Genetics, Genome Institute of Singapore, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Sim%20KS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Kar-Seng Sim</a></div></div><div id="_co_idm140153453309488"><h3 class="no_margin">Elisabeth M. van Leeuwen</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Leeuwen%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Elisabeth M. van Leeuwen</a></div></div><div id="_co_idm140153453307504"><h3 class="no_margin">Ayse Demirkan</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><p><sup>17</sup>Department of Human Genetics, Leiden University Medical Center, Leiden, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Demirkan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Ayse Demirkan</a></div></div><div id="_co_idm140153452899776"><h3 class="no_margin">Sven van der Lee</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20der%20Lee%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Sven van der Lee</a></div></div><div id="_co_idm140153452897792"><h3 class="no_margin">Seng-Chee Loon</h3><p><sup>7</sup>Department of Ophthalmology, National University of Singapore and National University Health System, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Loon%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Seng-Chee Loon</a></div></div><div id="_co_idm140153452895808"><h3 class="no_margin">Fernando Rivadeneira</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><p><sup>18</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam, the Netherlands</p><p><sup>19</sup>Netherlands Consortium for Healthy Ageing, Netherlands Genomics Initiative, the Hague, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Rivadeneira%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Fernando Rivadeneira</a></div></div><div id="_co_idm140153444593056"><h3 class="no_margin">Abhishek Nag</h3><p><sup>4</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, London, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Nag%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Abhishek Nag</a></div></div><div id="_co_idm140153444591072"><h3 class="no_margin">Paul G. Sanfilippo</h3><p><sup>13</sup>Centre for Ophthalmology and Visual Science, Lions Eye Institute, University of Western Australia, Perth, Australia</p><p><sup>16</sup>Centre for Eye Research Australia (CERA), University of Melbourne, Royal Victorian Eye and Ear Hospital, Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Sanfilippo%20PG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Paul G. Sanfilippo</a></div></div><div id="_co_idm140153444588352"><h3 class="no_margin">Arne Schillert</h3><p><sup>20</sup>DZHK (German Centre for Cardiovascular Research), partner site Hamburg/Kiel/Lübeck, Lübeck, Germany</p><p><sup>21</sup>Institute of Medical Biometry and Statistics, University of Lübeck, Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=Schillert%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Arne Schillert</a></div></div><div id="_co_idm140153445291456"><h3 class="no_margin">Paulus T. V. M. de Jong</h3><p><sup>22</sup>Department of Ophthalmology, Academic Medical Center, Amsterdam, the Netherlands</p><p><sup>23</sup>Department of Ophthalmology, Leiden University Medical Center, Leiden, the Netherlands</p><p><sup>24</sup>Department of Retinal Signal Processing, Netherlands Institute for Neuroscience, Amsterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=de%20Jong%20PT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Paulus T. V. M. de Jong</a></div></div><div id="_co_idm140153445288000"><h3 class="no_margin">Ben A. Oostra</h3><p><sup>25</sup>Department of Clinical Genetics, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Oostra%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Ben A. Oostra</a></div></div><div id="_co_idm140153445286016"><h3 class="no_margin">André G. Uitterlinden</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><p><sup>18</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam, the Netherlands</p><p><sup>19</sup>Netherlands Consortium for Healthy Ageing, Netherlands Genomics Initiative, the Hague, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">André G. Uitterlinden</a></div></div><div id="_co_idm140153451807152"><h3 class="no_margin">Albert Hofman</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><p><sup>19</sup>Netherlands Consortium for Healthy Ageing, Netherlands Genomics Initiative, the Hague, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Albert Hofman</a></div></div><div id="_co_idm140153451803072"><h3 class="no_margin">Tiger Zhou</h3><p><sup>26</sup>Department of Ophthalmology, Flinders University, Adelaide, Australia</p><div>Find articles by <a href="/pubmed/?term=Zhou%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Tiger Zhou</a></div></div><div id="_co_idm140153451801088"><h3 class="no_margin">Kathryn P. Burdon</h3><p><sup>27</sup>School of Medicine, Menzies Research Institute Tasmania, University of Tasmania, Hobart, Australia</p><div>Find articles by <a href="/pubmed/?term=Burdon%20KP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Kathryn P. Burdon</a></div></div><div id="_co_idm140153451799104"><h3 class="no_margin">Timothy D. Spector</h3><p><sup>4</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, London, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Spector%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Timothy D. Spector</a></div></div><div id="_co_idm140153445590208"><h3 class="no_margin">Karl J. Lackner</h3><p><sup>6</sup>Klinik Pallas Olten, Olten, Switzerland</p><div>Find articles by <a href="/pubmed/?term=Lackner%20KJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Karl J. Lackner</a></div></div><div id="_co_idm140153445588224"><h3 class="no_margin">Seang-Mei Saw</h3><p><sup>7</sup>Department of Ophthalmology, National University of Singapore and National University Health System, Singapore, Singapore</p><p><sup>11</sup>Duke-National University of Singapore, Graduate Medical School, Singapore, Singapore</p><p><sup>12</sup>Singapore Eye Research Institute, Singapore National Eye Centre, Singapore, Singapore</p><p><sup>28</sup>Saw Swee Hock School of Public Health, National University of Singapore and National University Health System, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Saw%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Seang-Mei Saw</a></div></div><div id="_co_idm140153445584032"><h3 class="no_margin">Johannes R. Vingerling</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><p><sup>2</sup>Department of Ophthalmology, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Vingerling%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Johannes R. Vingerling</a></div></div><div id="_co_idm140153451126112"><h3 class="no_margin">Yik-Ying Teo</h3><p><sup>28</sup>Saw Swee Hock School of Public Health, National University of Singapore and National University Health System, Singapore, Singapore</p><p><sup>29</sup>Department of Statistics and Applied Probability, National University of Singapore, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Teo%20YY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Yik-Ying Teo</a></div></div><div id="_co_idm140153451123392"><h3 class="no_margin">Louis R. Pasquale</h3><p><sup>30</sup>Channing Division of Network Medicine, Brigham and Women’s Hospital, Boston, Massachusetts, United States of America</p><p><sup>31</sup>Department of Ophthalmology, Harvard Medical School and Massachusetts Eye and Ear Infirmary, Boston, Massachusetts, United States of America</p><div>Find articles by <a href="/pubmed/?term=Pasquale%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Louis R. Pasquale</a></div></div><div id="_co_idm140153451120672"><h3 class="no_margin">Roger C. W. Wolfs</h3><p><sup>2</sup>Department of Ophthalmology, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Wolfs%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Roger C. W. Wolfs</a></div></div><div id="_co_idm140153451118688"><h3 class="no_margin">Hans G. Lemij</h3><p><sup>32</sup>Glaucoma Service, The Rotterdam Eye Hospital, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Lemij%20HG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Hans G. Lemij</a></div></div><div id="_co_idm140153450544016"><h3 class="no_margin">E-Shyong Tai</h3><p><sup>28</sup>Saw Swee Hock School of Public Health, National University of Singapore and National University Health System, Singapore, Singapore</p><p><sup>29</sup>Department of Statistics and Applied Probability, National University of Singapore, Singapore, Singapore</p><p><sup>33</sup>Department of Medicine, National University of Singapore and National University Health System, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Tai%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">E-Shyong Tai</a></div></div><div id="_co_idm140153450540560"><h3 class="no_margin">Jost B. Jonas</h3><p><sup>34</sup>Department of Ophthalmology, Medical Faculty Mannheim of the Ruprecht-Karls-University of Heidelberg, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Jonas%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Jost B. Jonas</a></div></div><div id="_co_idm140153450538576"><h3 class="no_margin">Ching-Yu Cheng</h3><p><sup>7</sup>Department of Ophthalmology, National University of Singapore and National University Health System, Singapore, Singapore</p><p><sup>11</sup>Duke-National University of Singapore, Graduate Medical School, Singapore, Singapore</p><p><sup>12</sup>Singapore Eye Research Institute, Singapore National Eye Centre, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Cheng%20CY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Ching-Yu Cheng</a></div></div><div id="_co_idm140153450535120"><h3 class="no_margin">Tin Aung</h3><p><sup>7</sup>Department of Ophthalmology, National University of Singapore and National University Health System, Singapore, Singapore</p><p><sup>12</sup>Singapore Eye Research Institute, Singapore National Eye Centre, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Aung%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Tin Aung</a></div></div><div id="_co_idm140153442678368"><h3 class="no_margin">Nomdo M. Jansonius</h3><p><sup>35</sup>Department of Ophthalmology, University of Groningen, University Medical Center Groningen, Groningen, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Jansonius%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Nomdo M. Jansonius</a></div></div><div id="_co_idm140153442676384"><h3 class="no_margin">Caroline C. W. Klaver</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><p><sup>2</sup>Department of Ophthalmology, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Klaver%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Caroline C. W. Klaver</a></div></div><div id="_co_idm140153442673664"><h3 class="no_margin">Jamie E. Craig</h3><p><sup>26</sup>Department of Ophthalmology, Flinders University, Adelaide, Australia</p><div>Find articles by <a href="/pubmed/?term=Craig%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Jamie E. Craig</a></div></div><div id="_co_idm140153442671680"><h3 class="no_margin">Terri L. Young</h3><p><sup>10</sup>Duke University Eye Center, Durham, North Carolina, United States of America</p><div>Find articles by <a href="/pubmed/?term=Young%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Terri L. Young</a></div></div><div id="_co_idm140153448540096"><h3 class="no_margin">Jonathan L. Haines</h3><p><sup>9</sup>Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, Ohio, United States of America</p><div>Find articles by <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Jonathan L. Haines</a></div></div><div id="_co_idm140153448538112"><h3 class="no_margin">Stuart MacGregor</h3><p><sup>3</sup>Statistical Genetics, QIMR Berghofer Medical Research Institute, Royal Brisbane Hospital, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=MacGregor%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Stuart MacGregor</a></div></div><div id="_co_idm140153448536128"><h3 class="no_margin">David A. Mackey</h3><p><sup>13</sup>Centre for Ophthalmology and Visual Science, Lions Eye Institute, University of Western Australia, Perth, Australia</p><p><sup>27</sup>School of Medicine, Menzies Research Institute Tasmania, University of Tasmania, Hobart, Australia</p><div>Find articles by <a href="/pubmed/?term=Mackey%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">David A. Mackey</a></div></div><div id="_co_idm140153448533408"><h3 class="no_margin">Norbert Pfeiffer</h3><p><sup>5</sup>Department of Ophthalmology, University Medical Center Mainz, Mainz, Germany</p><div>Find articles by <a href="/pubmed/?term=Pfeiffer%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Norbert Pfeiffer</a></div></div><div id="_co_idm140153451505472"><h3 class="no_margin">Tien-Yin Wong</h3><p><sup>7</sup>Department of Ophthalmology, National University of Singapore and National University Health System, Singapore, Singapore</p><p><sup>11</sup>Duke-National University of Singapore, Graduate Medical School, Singapore, Singapore</p><p><sup>12</sup>Singapore Eye Research Institute, Singapore National Eye Centre, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Wong%20TY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Tien-Yin Wong</a></div></div><div id="_co_idm140153451501280"><h3 class="no_margin">Janey L. Wiggs</h3><p><sup>31</sup>Department of Ophthalmology, Harvard Medical School and Massachusetts Eye and Ear Infirmary, Boston, Massachusetts, United States of America</p><div>Find articles by <a href="/pubmed/?term=Wiggs%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Janey L. Wiggs</a></div></div><div id="_co_idm140153451498560"><h3 class="no_margin">Alex W. Hewitt</h3><p><sup>16</sup>Centre for Eye Research Australia (CERA), University of Melbourne, Royal Victorian Eye and Ear Hospital, Melbourne, Victoria, Australia</p><p><sup>27</sup>School of Medicine, Menzies Research Institute Tasmania, University of Tasmania, Hobart, Australia</p><div>Find articles by <a href="/pubmed/?term=Hewitt%20AW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Alex W. Hewitt</a></div></div><div id="_co_idm140153449946592"><h3 class="no_margin">Cornelia M. van Duijn</h3><p><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Cornelia M. van Duijn</a></div></div><div id="_co_idm140153449943136"><h3 class="no_margin">Christopher J. Hammond</h3><p><sup>4</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, London, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Hammond%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25631615">Christopher J. Hammond</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140153451950992_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140153451950992_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140153451950992_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Ophthalmology, Erasmus Medical Center, Rotterdam, the Netherlands</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Statistical Genetics, QIMR Berghofer Medical Research Institute, Royal Brisbane Hospital, Brisbane, Australia</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, London, United Kingdom</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Ophthalmology, University Medical Center Mainz, Mainz, Germany</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Klinik Pallas Olten, Olten, Switzerland</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Ophthalmology, National University of Singapore and National University Health System, Singapore, Singapore</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Division of Human Genetics, Genome Institute of Singapore, Singapore, Singapore</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, Ohio, United States of America</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Duke University Eye Center, Durham, North Carolina, United States of America</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Duke-National University of Singapore, Graduate Medical School, Singapore, Singapore</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Singapore Eye Research Institute, Singapore National Eye Centre, Singapore, Singapore</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Centre for Ophthalmology and Visual Science, Lions Eye Institute, University of Western Australia, Perth, Australia</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Beijing Institute of Ophthalmology, Beijing Tongren Eye Center, Beijing Tongren Hospital, Capital Medical University, Beijing, China</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Beijing Ophthalmology and Visual Science Key Lab, Beijing, China</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Centre for Eye Research Australia (CERA), University of Melbourne, Royal Victorian Eye and Ear Hospital, Melbourne, Victoria, Australia</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Department of Human Genetics, Leiden University Medical Center, Leiden, the Netherlands</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam, the Netherlands</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Netherlands Consortium for Healthy Ageing, Netherlands Genomics Initiative, the Hague, the Netherlands</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>DZHK (German Centre for Cardiovascular Research), partner site Hamburg/Kiel/Lübeck, Lübeck, Germany</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Institute of Medical Biometry and Statistics, University of Lübeck, Lübeck, Germany</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Ophthalmology, Academic Medical Center, Amsterdam, the Netherlands</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Department of Ophthalmology, Leiden University Medical Center, Leiden, the Netherlands</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Department of Retinal Signal Processing, Netherlands Institute for Neuroscience, Amsterdam, the Netherlands</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Department of Clinical Genetics, Erasmus Medical Center, Rotterdam, the Netherlands</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Department of Ophthalmology, Flinders University, Adelaide, Australia</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>School of Medicine, Menzies Research Institute Tasmania, University of Tasmania, Hobart, Australia</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Saw Swee Hock School of Public Health, National University of Singapore and National University Health System, Singapore, Singapore</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Department of Statistics and Applied Probability, National University of Singapore, Singapore, Singapore</div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Channing Division of Network Medicine, Brigham and Women’s Hospital, Boston, Massachusetts, United States of America</div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Department of Ophthalmology, Harvard Medical School and Massachusetts Eye and Ear Infirmary, Boston, Massachusetts, United States of America</div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Glaucoma Service, The Rotterdam Eye Hospital, Rotterdam, the Netherlands</div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Department of Medicine, National University of Singapore and National University Health System, Singapore, Singapore</div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>Department of Ophthalmology, Medical Faculty Mannheim of the Ruprecht-Karls-University of Heidelberg, Heidelberg, Germany</div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>Department of Ophthalmology, University of Groningen, University Medical Center Groningen, Groningen, the Netherlands</div><div id="FN1"><sup>*</sup>Correspondence to: Cornelia M. van Duijn, Erasmus Medisch Centrum, Cornelia van Duijn room NA-2714, Postbus 2040, 3000 CA Rotterdam, the Netherlands. <a href="mailto:dev@null" data-email="ln.cmsumsare@njiudnav.c" class="oemail">ln.cmsumsare@njiudnav.c</a></div><div id="FN2"><sup>†</sup>Membership of the NEIGHBORHOOD Consortium is listed in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>.</div><div id="FN3"><sup>‡</sup>These authors contributed equally to this work.</div><div id="FN4"><sup>§</sup>These authors jointly directed this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140153451950992_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25631615" target="pmc_ext" ref="reftype=publisher&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Genet Epidemiol</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4480365/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplement.</strong></div><div><a href="/pmc/articles/PMC4480365/bin/NIHMS700469-supplement-supplement.docx" data-ga-action="click_feat_suppl">NIHMS700469-supplement-supplement.docx</a><span style="color:gray"> (9.7M)</span></div><div class="small guid">GUID:&nbsp;242D3B23-5FA5-4AD1-8816-1829192713EC</div></div></dd></dl></div><div id="idm140153445368640" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140153445368640title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Primary open-angle glaucoma is the most common optic neuropathy and an important cause of irreversible blindness worldwide. The optic nerve head or optic disc is divided in two parts: a central cup (without nerve fibers) surrounded by the neuroretinal rim (containing axons of the retinal ganglion cells). The International Glaucoma Genetics Consortium conducted a meta-analysis of genome-wide association studies consisting of 17,248 individuals of European ancestry and 6,841 individuals of Asian ancestry. The outcomes of the genome-wide association studies were disc area and cup area. These specific measurements describe optic nerve morphology in another way than the vertical cup-disc ratio, which is a clinically used measurement, and may shed light on new glaucoma mechanisms. We identified 10 new loci associated with disc area (<em>CDC42BPA</em>, <em>F5</em>, <em>DIRC3</em>, <em>RARB</em>, <em>ABI3BP</em>, <em>DCAF4L2</em>, <em>ELP4</em>, <em>TMTC2</em>, <em>NR2F2</em>, and <em>HORMAD2</em>) and another 10 new loci associated with cup area (<em>DHRS3</em>, <em>TRIB2</em>, <em>EFEMP1</em>, <em>FLNB</em>, <em>FAM101</em>, <em>DDHD1</em>, <em>ASB7</em>, <em>KPNB1</em>, <em>BCAS3</em>, and <em>TRIOBP</em>). The new genes participate in a number of pathways and future work is likely to identify more functions related to the pathogenesis of glaucoma.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">GWAS, disc area, cup area, glaucoma</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first-last">The optic nerve is a white matter tract approximately 55 millimeters in length that transmits visual information from the eye to the brain. Various diseases—the most common of which is glaucoma—affect the optic nerve morphology and function. There are many types of glaucoma and in this manuscript we focus on primary open-angle or simple glaucoma, which is one of the leading causes of irreversible blindness worldwide. The optic nerve head, often referred to as the optic disc, is the place where the retinal ganglion cell axons leave the eye and bundle together to form the optic nerve. It is visible at the back of the eye by ophthalmoscopy and is valuable in the assessment of optic nerve-related diseases. Additionally, the optic nerve morphology is a major target of imaging devices (including the Heidelberg Retina Tomography and Optical Coherence Tomography) in screening and follow-up of glaucoma-suspect persons and glaucoma patients. The optic disc consists of two morphologically distinct parts: the cup in the center of the disc, without nerve fibers, and the (neuroretinal) rim, carrying the axons of the retinal ganglion cells. There is a small, age-related decline in the number of axons during life: the decrease is about one third of axons in 100 years [<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029370">Jonas et al., 1990</a>; <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029392">Jonas et al., 1992</a>]. Glaucoma is characterized by an accelerated loss of retinal ganglion cell axons, resulting in an enlarged cup and a reduced rim area. The heritability of optic nerve morphological features is estimated to be 52–83% for the disc area, 66–77% for the cup area, and 34–39% for the rim area [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029371">Sanfilippo et al., 2010</a>; <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029396">van Koolwijk et al., 2007</a>]. The majority of genetic studies has focused on the vertical cup-disc ratio (VCDR), which is a measure used to assess glaucoma clinically. However, different mechanisms (growth vs. degeneration) may underlie the disc, cup, and rim area. This raises the question whether gene discovery focusing on other measures (parameters) describing the optic disc than only the VCDR may shed light on the development and pathogenesis and mechanisms of diseases of the optic nerve. To date, genome-wide association studies (GWAS) have identified four loci for disc area within or near to the genes <em>ATOH7, CARD10</em>, <em>CDC7/TGFBR3,</em> and <em>SALL1</em> and one locus for rim area (<em>RERE</em>) [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029389">Axenovich et al., 2011</a>; <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029390">Khor et al., 2011</a>; <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029395">Macgregor et al., 2010</a>; <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029376">Ramdas et al., 2010</a>]. We performed a meta-analysis of GWAS for these disc area parameters within the International Glaucoma Genetics Consortium (IGGC).</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Study design</h3><p id="P3" class="p p-first-last">We performed a meta-analysis on directly genotyped and imputed SNPs from individuals of European ancestry in seven studies, with a total of 17,248 individuals (stage 1). Subsequently, we evaluated significantly associated SNPs in 6,841 subjects of Asian origin including four different studies (stage 2) and performed a meta-analysis on all individual studies from stage 1 and stage 2 (stage 3).</p></div><div id="S4" class="sec"><h3 id="S4title">Participants and Phenotyping</h3><p id="P4" class="p p-first">All studies included in this meta-analysis are part of the International Glaucoma Genetics Consortium (IGGC). The ophthalmic examination of each study included an assessment of the optic nerve head (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S1B</a>).</p><p id="P5">The meta-analysis of stage 1 was based on seven studies of European ancestry: Brisbane Adolescent Twin Study (BATS), Erasmus Rucphen Family (ERF) Study, Gutenberg Health Study (GHS I/GHS II), Raine Study, Rotterdam Study (RS-I/RS-II/RS-III), Twins Eye Study in Tasmania (TEST), and TwinsUK. Stage 2 comprised four Asian studies: the Beijing Eye Study (BES), Singapore Chinese Eye Study (SCES), Singapore Malay Eye Study (SIMES), and Singapore Indian Eye Study (SINDI).</p><p id="P6" class="p p-last">Information on general methods, demographics, phenotyping, and genotyping methods of the study cohorts can be found in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables S1 and S2</a> and the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>. All studies were performed with the approval of their local medical ethics committee, and written informed consent was obtained from all participants in accordance with the Declaration of Helsinki.</p></div><div id="S5" class="sec"><h3 id="S5title">Genotyping and Imputation</h3><p id="P7" class="p p-first-last">Information on genotyping in each cohort, the particular platforms used to perform genotyping and the methods of imputation can be found in more detail in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S1C</a>. To produce consistent datasets and enable a meta-analysis of studies across different genotyping platforms, the studies performed genomic imputation on available HapMap Phase 2 genotypes with MACH [<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029397">Li et al., 2010</a>] or IMPUTE [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029374">Marchini et al., 2007</a>], using the appropriate ancestry groups as templates. Each study applied stringent quality control procedures for imputation (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). For the metaanalysis, only single nucleotide polymorphisms (SNPs) with minor allele frequency ≥ 1%, Hardy Weinberg Equilibrium <em>P</em>-value &gt; 10<sup>−6</sup>, and imputation quality scores ≥ 0.3 (proper-info of IMPUTE) or R2 ≥ 0.3 (MACH) were included.</p></div><div id="S6" class="sec"><h3 id="S6title">Statistical Analysis</h3><p id="P8" class="p p-first">As the rim area is the difference between the disc area and cup area, there are two independent variables. Of these, we selected (essentially arbitrarily from a mathematical point of view) disc area and cup area. Moreover, disc and cup area are clearly correlated (Pearson correlation coefficient is 0.59 in Rotterdam Study I). For that reason, we analyzed (1) disc area and (2) cup area adjusted for disc area. We used the mean of the measurements of both eyes. Unreliable optic nerve head data were excluded (e.g., images with standard deviation &gt; 50 for the Heidelberg Retina Tomograph). In cases of missing or unreliable data for one eye, data for the other eye were taken. Each individual study did a linear regression model between the outcomes and approximately 2.5 million HapMap stage 2 SNPs under the assumption of an additive model for the effect of the risk allele. Analyses were adjusted for age, sex, and the first two principal components (for population-based studies) or family structure (for family-based studies) to correct for population substructure. Adding additional principal components did not appreciably change the lambda. Glaucoma is characterized by an increased cupping independent of the size of the disc. Therefore, in the linear regression analysis with cup area as outcome, we used disc area as an extra covariate.</p><p id="P9">All study effect estimates were oriented to the positive strand of the NCBI Build 36 reference. Positioning and annotations for the SNPs were done using the NCBI Build 37 reference. We performed an inverse variance weighted fixed-effect meta-analysis using METAL software [<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029388">Willer et al., 2010</a>]. We used the ‘genomic control’ option in METAL that estimates the inflation of the test statistic of each individual study and corrects the standard error of each individual study for the inflation.</p><p id="P10" class="p p-last">In stage 1, a <em>P</em>-value &lt; 5.0 × 10<sup>−8</sup> (the genome-wide threshold of association) was considered significant. In the replication stage 2, a nominal <em>P</em>-value &lt; 0.05 was considered significant given the already high prior probabilities of association from stage 1. Genome-wide significant SNPs for disc area were tested for cup area, and vice versa. In total, there were 36 independent SNPs. Therefore, our Bonferonni corrected significant threshold for this analysis was 0.05/36 = 1.39 × 10<sup>−3</sup>. Manhattan, regional, and forest plots were made using R (<a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>), LocusZoom [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029384">Pruim et al., 2010</a>], and Stata/SE 12.0 (StataCorp LP, College Station, TX, USA), respectively.</p></div><div id="S7" class="sec"><h3 id="S7title">Gene-Based Test Using VEGAS</h3><p id="P11" class="p p-first-last">Gene-based testing was performed using VEGAS software [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029393">Liu et al., 2010</a>], which combines the test statistics of all SNPs present within and 50 kb upstream or downstream of each gene. Linkage disequilibrium (LD) between the markers was accounted for through simulations from the multivariate normal distribution, based on estimates of LD from reference populations. Because Asian and European ancestry populations show different LD patterns, we performed separate gene-based tests for each population. Hapmap 2 CEU population was used as a reference to calculate LD for European ancestry data, whereas Hapmap 2 JPT and CHB combined population was used as a reference for Asian ancestry data. After calculation of gene-based test statistics for Asian and European ancestry populations separately, meta-analysis was conducted using Fisher’s method for combining <em>P</em>-values. VEGAS was applied to the summary data from the full disc and cup area analysis (as in <a href="/pmc/articles/PMC4480365/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a> and <a href="/pmc/articles/PMC4480365/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2</span></span><span>2</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p34">Summary of SNPs that showed genome-wide significant (<em>P</em> &lt; 5 × 10<sup>−8</sup>) association with disc area in the combined analysis (<em>n</em> = 24,089 subjects with European and Asian descent)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140153446837456" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="2" colspan="1" valign="bottom" align="right">Chr.</th><th rowspan="2" colspan="1" valign="bottom" align="right">Position</th><th rowspan="2" colspan="1" valign="bottom" align="left">Annotation</th><th rowspan="2" colspan="1" valign="bottom" align="left">Nearest Gene</th><th rowspan="2" colspan="1" valign="bottom" align="center">A1/A2</th><th rowspan="2" colspan="1" valign="bottom" align="center">Europeans<br>MAF</th><th rowspan="2" colspan="1" valign="bottom" align="center">Asians<br>MAF</th><th colspan="5" rowspan="1" valign="bottom" align="center">Combined (<em>n</em> = 24,089)
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Heterogeneity
<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">Cup area combined<br>(<em>n</em> = 23,831)
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center"><em>β</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">SE</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P-</em>value</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>β</em> (R)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P-</em>value (R)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P-</em>value</th><th rowspan="1" colspan="1" valign="bottom" align="center">I<sup>2</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>β</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">SE</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P-</em>value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1192419</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">92,080,059</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CDC7/TGFBR3</em></td><td rowspan="1" colspan="1" valign="top" align="center">a/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">0.087</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td><td rowspan="1" colspan="1" valign="top" align="center">7.98 × 10<sup>−56</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.087</td><td rowspan="1" colspan="1" valign="top" align="center">7.98 × 10<sup>−56</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.51 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">4.58×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6671926</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">227,386,971</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CDC42BPA</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">a/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">−0.067</td><td rowspan="1" colspan="1" valign="top" align="center">0.009</td><td rowspan="1" colspan="1" valign="top" align="center">3.69 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.065</td><td rowspan="1" colspan="1" valign="top" align="center">9.21 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.95 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.025</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">2.33×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9969524</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="right">88,746,846</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DCAF4L2</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">t/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">0.030</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">1.54 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.032</td><td rowspan="1" colspan="1" valign="top" align="center">4.36 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.79 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">19.0</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">4.24×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1900004</td><td rowspan="1" colspan="1" valign="top" align="right">10</td><td rowspan="1" colspan="1" valign="top" align="right">70,000,881</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ATOH7</em></td><td rowspan="1" colspan="1" valign="top" align="center">t/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">−0.097</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">1.13 × 10<sup>−73</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.097</td><td rowspan="1" colspan="1" valign="top" align="center">7.27 × 10<sup>−39</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.43 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">40.4</td><td rowspan="1" colspan="1" valign="top" align="center">−0.016</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">4.92×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1362756</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="right">51,458,290</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SALL1</em></td><td rowspan="1" colspan="1" valign="top" align="center">c/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.033</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">9.27 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.033</td><td rowspan="1" colspan="1" valign="top" align="center">5.84 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.91 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.6</td><td rowspan="1" colspan="1" valign="top" align="center">0.011</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">4.66×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12406092</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">169,543,131</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>F5</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">a/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.028</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">3.32 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.028</td><td rowspan="1" colspan="1" valign="top" align="center">3.3 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.60 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">2.88×10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1549733</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">218,472,172</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DIRC3</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">t/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.031</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">4.03 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.031</td><td rowspan="1" colspan="1" valign="top" align="center">4.03 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.74 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">−0.001</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">6.99×10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11129176</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">25,049,310</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RARB</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">a/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.026</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">1.74 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.026</td><td rowspan="1" colspan="1" valign="top" align="center">1.74 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.99 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.011</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">1.12×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9860250</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">100,637,871</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ABI3BP</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">g/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">−0.036</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td><td rowspan="1" colspan="1" valign="top" align="center">2.42 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.036</td><td rowspan="1" colspan="1" valign="top" align="center">2.42 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.59 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">−0.004</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">2.05×10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11031436</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="right">31,663,882</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ELP4</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">t/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">0.033</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">6.43 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.032</td><td rowspan="1" colspan="1" valign="top" align="center">3.63 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.47 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">19.4</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">3.26×10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1511589</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="right">84,061,431</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TMTC2</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">a/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">−0.028</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">1.08 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.028</td><td rowspan="1" colspan="1" valign="top" align="center">5.40 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.22 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">12.0</td><td rowspan="1" colspan="1" valign="top" align="center">−0.012</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">5.34×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs8034595</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="right">96,719,229</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NR2F2</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">a/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">−0.026</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">2.54 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.026</td><td rowspan="1" colspan="1" valign="top" align="center">2.54 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.99 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">2.20×10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2412970</td><td rowspan="1" colspan="1" valign="top" align="right">22</td><td rowspan="1" colspan="1" valign="top" align="right">30,486,826</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HORMAD2</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">g/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.024</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">3.40 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.024</td><td rowspan="1" colspan="1" valign="top" align="center">3.40 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.75 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.007</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">3.52×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9607469</td><td rowspan="1" colspan="1" valign="top" align="right">22</td><td rowspan="1" colspan="1" valign="top" align="right">37,919,267</td><td rowspan="1" colspan="1" valign="top" align="left">Upstream gene variant</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CARD10</em></td><td rowspan="1" colspan="1" valign="top" align="center">a/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.041</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td><td rowspan="1" colspan="1" valign="top" align="center">2.29 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.041</td><td rowspan="1" colspan="1" valign="top" align="center">1.31 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.17 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">32.3</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">1.03×10<sup>−1</sup></td></tr></tbody></table></div><div id="largeobj_idm140153446837456" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4480365/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p35" class="p p-first-last">We tested for heterogeneous effects, for which <em>P</em>-values and I<sup>2</sup> are shown. Results for the combined meta-analysis of cup area are shown. The first five SNPs were genome-wide significant in stage 1 (meta-analysis of subjects with European descent); the last nine SNPs reached genome-wide significance in stage 3 (meta-analysis of subjects with European and Asian descent). SNP, single nucleotide polymorphism; nearest gene, reference NCBI build 37; A1, reference allele; A2, other allele; MAF, average minor allele frequency; NA, not available; <em>β</em>, effect size on disc area based on allele A1 based on a fixed-effect meta-analysis; <em>β</em> (R), effect size on disc area based on allele A1 based on a random-effect meta-analysis; <em>P</em>-value (R) is the <em>P</em>-value based on a random-effect meta-analysis; SE, standard error of the effect size.</p></div><div id="TFN2"><sup>a</sup>Are the newly identified loci.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p37">Summary of SNPs that showed genome-wide significant (<em>P</em> &lt; 5 × 10<sup>−8</sup>) association with cup area (adjusted for disc area) in the combined analysis (<em>n</em> = 23,831 subjects with European and Asian descent)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140153444370240" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="2" colspan="1" valign="bottom" align="right">Chr.</th><th rowspan="2" colspan="1" valign="bottom" align="right">Position</th><th rowspan="2" colspan="1" valign="bottom" align="left">Annotation</th><th rowspan="2" colspan="1" valign="bottom" align="left">Nearest gene</th><th rowspan="2" colspan="1" valign="bottom" align="center">A1/A2</th><th rowspan="2" colspan="1" valign="bottom" align="center">Europeans<br>MAF</th><th rowspan="2" colspan="1" valign="bottom" align="center">Asians<br>MAF</th><th colspan="5" rowspan="1" valign="bottom" align="center">Combined (<em>n</em> = 23,831)
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Heterogeneity
<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">Disc area combined (<em>n</em> = 24,089)
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center"><em>β</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">SE</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P-</em>value</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>β</em> (R)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P-</em>value (R)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P-</em>value</th><th rowspan="1" colspan="1" valign="bottom" align="center">I<sup>2</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>β</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">SE</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P-</em>value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs301801</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">8,495,945</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RERE</em></td><td rowspan="1" colspan="1" valign="top" align="center">c/t</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.016</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">4.55 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.016</td><td rowspan="1" colspan="1" valign="top" align="center">4.55 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.56 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">3.80 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3924048</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">12,614,848</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DHRS3</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">g/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.40</td><td rowspan="1" colspan="1" valign="top" align="center">0.58</td><td rowspan="1" colspan="1" valign="top" align="center">−0.016</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">1.34 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.016</td><td rowspan="1" colspan="1" valign="top" align="center">1.34 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.55 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">−0.003</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">4.68 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2113818</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">12,890,860</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TRIB2</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">t/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.015</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">1.19 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.015</td><td rowspan="1" colspan="1" valign="top" align="center">1.19 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.90 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">2.70 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2623325</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">99,131,755</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>COL8A1</em></td><td rowspan="1" colspan="1" valign="top" align="center">a/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.025</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">4.36 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.024</td><td rowspan="1" colspan="1" valign="top" align="center">1.33 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.71 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">27.9</td><td rowspan="1" colspan="1" valign="top" align="center">0.025</td><td rowspan="1" colspan="1" valign="top" align="center">0.008</td><td rowspan="1" colspan="1" valign="top" align="center">1.92 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7865618</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="right">22,031,005</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CDKN2B-AS1</em></td><td rowspan="1" colspan="1" valign="top" align="center">g/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">−0.023</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">1.37 × 10<sup>−21</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.022</td><td rowspan="1" colspan="1" valign="top" align="center">9.36 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.12 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">12.9</td><td rowspan="1" colspan="1" valign="top" align="center">−0.010</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">2.01 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3858145</td><td rowspan="1" colspan="1" valign="top" align="right">10</td><td rowspan="1" colspan="1" valign="top" align="right">70,011,838</td><td rowspan="1" colspan="1" valign="top" align="left">Regulatory</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ATOH7</em></td><td rowspan="1" colspan="1" valign="top" align="center">g/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">−0.015</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">7.83 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">1.56 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.6 5× 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">18.6</td><td rowspan="1" colspan="1" valign="top" align="center">−0.090</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">5.14× 10<sup>−75</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1346</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="right">65,337,251</td><td rowspan="1" colspan="1" valign="top" align="left">Upstream</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SSSCA1</em></td><td rowspan="1" colspan="1" valign="top" align="center">t/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">−0.019</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">1.78 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.019</td><td rowspan="1" colspan="1" valign="top" align="center">7.50 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.02 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.4</td><td rowspan="1" colspan="1" valign="top" align="center">−0.020</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td><td rowspan="1" colspan="1" valign="top" align="center">3.84 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7972528</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="right">84,131,036</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TMTC2</em></td><td rowspan="1" colspan="1" valign="top" align="center">t/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">4.03 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.011</td><td rowspan="1" colspan="1" valign="top" align="center">9.34 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.02 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">55.4</td><td rowspan="1" colspan="1" valign="top" align="center">−0.020</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">4.24 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9546434</td><td rowspan="1" colspan="1" valign="top" align="right">13</td><td rowspan="1" colspan="1" valign="top" align="right">36,694,391</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DCLK1</em></td><td rowspan="1" colspan="1" valign="top" align="center">t/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">0.021</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">1.98 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.020</td><td rowspan="1" colspan="1" valign="top" align="center">2.11 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.38 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">−0.012</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td><td rowspan="1" colspan="1" valign="top" align="center">2.80 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10483727</td><td rowspan="1" colspan="1" valign="top" align="right">14</td><td rowspan="1" colspan="1" valign="top" align="right">61,072,875</td><td rowspan="1" colspan="1" valign="top" align="left">Upstream</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SIX6</em></td><td rowspan="1" colspan="1" valign="top" align="center">t/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.40</td><td rowspan="1" colspan="1" valign="top" align="center">0.71</td><td rowspan="1" colspan="1" valign="top" align="center">0.026</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">2.10 × 10<sup>−27</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.025</td><td rowspan="1" colspan="1" valign="top" align="center">2.41 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.80 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">56.6</td><td rowspan="1" colspan="1" valign="top" align="center">−0.023</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">1.14 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11646917</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="right">51,428,908</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SALL1</em></td><td rowspan="1" colspan="1" valign="top" align="center">t/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">−0.018</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">1.71 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.018</td><td rowspan="1" colspan="1" valign="top" align="center">1.84 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.36 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.2</td><td rowspan="1" colspan="1" valign="top" align="center">−0.020</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td><td rowspan="1" colspan="1" valign="top" align="center">8.53 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11870935</td><td rowspan="1" colspan="1" valign="top" align="right">17</td><td rowspan="1" colspan="1" valign="top" align="right">45,732,605</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>KPNB1</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">g/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">0.013</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">1.57 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.013</td><td rowspan="1" colspan="1" valign="top" align="center">1.57 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.81 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.001</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">8.82 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11651885</td><td rowspan="1" colspan="1" valign="top" align="right">17</td><td rowspan="1" colspan="1" valign="top" align="right">59,286,263</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>BCAS3</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">g/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">−0.018</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">4.49 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.017</td><td rowspan="1" colspan="1" valign="top" align="center">1.09 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">9.27 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">54.6</td><td rowspan="1" colspan="1" valign="top" align="center">0.012</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">1.75×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6054383</td><td rowspan="1" colspan="1" valign="top" align="right">20</td><td rowspan="1" colspan="1" valign="top" align="right">6,584,604</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>BMP2</em></td><td rowspan="1" colspan="1" valign="top" align="center">t/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.61</td><td rowspan="1" colspan="1" valign="top" align="center">−0.015</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">2.13× 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">2.51 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.04 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">6.88 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1033667</td><td rowspan="1" colspan="1" valign="top" align="right">22</td><td rowspan="1" colspan="1" valign="top" align="right">2,9130,300</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CHEK2</em></td><td rowspan="1" colspan="1" valign="top" align="center">t/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">−0.023</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">1.13 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.024</td><td rowspan="1" colspan="1" valign="top" align="center">5.48 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.19 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">32.1</td><td rowspan="1" colspan="1" valign="top" align="center">−0.020</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">6.18 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1346786</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">56,108,333</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>EFEMP1</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">t/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">0.63</td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">4.26 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">4.26 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.20 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.008</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">9.66 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6764184</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">58,006,266</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FLNB</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">t/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">0.015</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">1.10 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.015</td><td rowspan="1" colspan="1" valign="top" align="center">2.48 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.94 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">14.6</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">5.25 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1402538</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="right">122,388,851</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HSF2</em></td><td rowspan="1" colspan="1" valign="top" align="center">a/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.40</td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">4.33 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">4.33 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.51 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">−0.015</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">8.57 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10846617</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="right">124,662,131</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FAM101A</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">c/g</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">7.17 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">7.17 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.71 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">−0.003</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">5.31 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10130556</td><td rowspan="1" colspan="1" valign="top" align="right">14</td><td rowspan="1" colspan="1" valign="top" align="right">53,970,675</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DDHD1/BMP4</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">g/c</td><td rowspan="1" colspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">4.53 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td rowspan="1" colspan="1" valign="top" align="center">4.53× 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.46 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">2.07 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11247230</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="right">101,197,005</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ASB7</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">g/a</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.71</td><td rowspan="1" colspan="1" valign="top" align="center">0.014</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">1.34 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.014</td><td rowspan="1" colspan="1" valign="top" align="center">1.34 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.46 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.017</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">1.41 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs5756813</td><td rowspan="1" colspan="1" valign="top" align="right">22</td><td rowspan="1" colspan="1" valign="top" align="right">38,175,477</td><td rowspan="1" colspan="1" valign="top" align="left">Downstream</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TRIOBP</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">g/t</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.014</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">1.49 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.014</td><td rowspan="1" colspan="1" valign="top" align="center">1.49 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">9.61 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0.013</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">2.39 × 10<sup>−3</sup></td></tr></tbody></table></div><div id="largeobj_idm140153444370240" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4480365/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="__p38" class="p p-first-last">We tested for heterogeneous effects, for which <em>P</em>-values and I<sup>2</sup> are shown. Results for the combined meta-analysis for disc area are shown. The first 15 SNPs were genome-wide significant in stage 1 (meta-analysis of subjects with European descent); the last seven SNPs reached genome-wide significance in stage 3 (meta-analysis of subjects with European and Asian descent). SNP, single nucleotide polymorphism; nearest gene, reference NCBI build 37; A1, reference allele; A2, other allele; MAF, average minor allele frequency; NA, not available; <em>β</em>, effect size on disc area based on allele A1 based on a fixed-effect meta-analysis; <em>β</em> (R), effect size on cup area based on allele A1 based on a random-effect meta-analysis; <em>P-</em>value (R) is the <em>P</em>-value based on a random-effect meta-analysis; SE, standard error of the effect size.</p></div><div id="TFN4"><sup>a</sup>Are the newly identified loci.</div></div></div></div><div id="S8" class="sec"><h3 id="S8title">Phenotypic Variability</h3><p id="P12" class="p p-first-last">To evaluate whether the different optic nerve head area parameters have a shared genetic component with primary open-angle glaucoma, two genetic risk scores were calculated based on the GWAS results for disc area and cup area. The genotyped SNPs from the discovery cohort were categorized into 17 categories according to <em>P</em>-values, and risk scores for each category were calculated in the ANZRAG study consisting of 1,155 glaucoma cases and 1,992 controls and NEIGHBOR consisting of 2,131 glaucoma cases and 2,290 controls as target cohorts (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a>). To maximize the overlap between the genotyped SNPs from the discovery cohort and the SNPs included in the target cohorts, the imputed SNPs with imputation quality score &gt; 0.8 in the target cohort were used for risk score calculation to replace SNPs that were not genotyped in the target cohort. For each individual, the score for a particular SNP was calculated as the effect estimate of the SNP multiplied by the dosage of the effect allele of that SNP. The risk score was defined as the mean of the scores for all SNPs. Logistic regression analyses with glaucoma as outcome adjusted for sex as covariate were performed to calculate the Nagelkerke R-square for the two risk scores (disc area and cup area).</p></div><div id="S9" class="sec sec-last"><h3 id="S9title">Pathway-Analysis Using Pathway-VEGAS</h3><p id="P13" class="p p-first-last">Prespecified pathways from the Gene Ontology database with size ranging in 5–500 genes were used to perform pathway analysis. Pathway-VEGAS combines VEGAS gene-based test statistics based on prespecified biological pathways [<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029383">Lu et al., 2013</a>]. Pathway <em>P</em>-values were computed by summing χ<sup>2</sup> test statistics derived from VEGAS <em>P</em>-values. Empirical “VEGAS-pathway” <em>P</em>-values for each pathway were computed by comparing the real data summed χ<sup>2</sup> test statistics with 500,000 simulations where the relevant number (as per size of pathway) of randomly drawn χ<sup>2</sup> test statistics was summed. To ensure clusters of genes did not adversely affect the result, gene-sets were pruned such that each gene was &gt;500 kb from all other genes in the pathway. When genes were clustered, only one of the clustered genes was included for that pathway. Pathway-VEGAS was performed separately for European and Asian ancestry datasets. Meta-analysis was conducted using Fisher’s method for combining <em>P</em>-values.</p></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">Results</h2><p id="P14" class="p p-first">This work followed two parallel directions that corresponded to multistage meta-analyses of two phenotypes of interest. Although there were superimpositions in the genetic risk of each of these phenotypes leading to regulation of optic disc morphology in the populations, results will be broken down and reported individually for each. As described in the Methods, we tested for association using linear regression models adjusting for age, sex, and two principal components or family structure.</p><div id="S11" class="sec"><h3 id="S11title">Disc Area</h3><p id="P15" class="p p-first">Stage 1 included 17,248 individuals of European ancestry. We analyzed approximately 2.5 million directly genotyped or imputed (HapMap) SNPs. The inflation factors (λ) varied between 0.98 and 1.06 (1.10 for the meta-analysis), implying adequate within-study control of population substructure (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S2</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. S2A, B, and C</a>). This analysis yielded 296 genome-wide significant (<em>P</em> &lt; 5.0 × 10<sup>−8</sup>) SNPs located across five chromosomal regions (<em>CDC7/TGFBR3, CDC42BPA, DCAF4L2, ATOH7,</em> and <em>SALL1</em>) (<a href="/pmc/articles/PMC4480365/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. S1A</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S3</a>).</p><p id="P16">Stage 2 included 6,841 individuals of Asian ancestry. The λ varied between 1.00 and 1.03. Of the most significantly associated SNPs at each of the five chromosomal regions in Europeans, three reached nominal significance (<em>P</em> &lt; 0.05) in the Asians: <em>CDC7/TGFBR3, CDC42BPA,</em> and <em>ATOH7</em>. The SNP with the most significant association at the chromosome 8 region (<em>DCAF4L2</em>) in stage 1 was not imputed in the Asian population. The second most associated SNP in Europeans (rs12547416, <em>β</em> = −0.03, <em>P</em> = 3.25 × 10<sup>−8</sup>) at this region was significant in the Asian population (<em>β</em> = −0.03, <em>P</em> = 2.95 × 10<sup>−4</sup>).</p><p id="P17">The combined analysis in stage 3 (overall λ 1.10) resulted in nine additional genome-wide significant chromosomal regions. The results of these SNPs were genome-wide suggestive (<em>P</em> &lt; 5.0 × 10<sup>−5</sup>) in the individuals of European ancestry and nominally significant in individuals of Asian ancestry (<em>P</em> &lt; 0.05). Of the 14 associated regions (five associated in Europeans and Asians and nine identified using all cohorts), 10 were not previously related to disc area: <em>CDC42BPA</em> (chr. 1) and <em>DCAF4L2</em> (chr. 8) identified in stage 1, and <em>F5</em> (chr.1), <em>DIRC3</em> (chr. 2), <em>RARB</em> (chr.3), <em>ABI3BP</em> (chr. 3), <em>ELP4</em> (chr. 11), <em>TMTC2</em> (chr. 12), <em>NR2F2</em> (chr. 15), and <em>HORMAD2</em> (chr. 22) identified in stage 3.</p><p id="P18" class="p p-last">In order to identify new loci that were not found through per-SNP test, we performed a gene-based test using VE-GAS software. Because of the smaller number of genes tested (17,872), our gene-based significance threshold p<sub>gene-based</sub> was 2.80 × 10<sup>−6</sup> (0.05/17,872). <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S5</a> shows 23 genes with a <em>P</em>-value below 2.80 × 10<sup>−6</sup> for the gene-based test. Of these 23 genes, 22 genes were located in loci identified by the GWAS. In addition to the loci already identified, we found a gene-based significant association of <em>PAX6</em> with disc area (gene-based test <em>P</em> = 5.15 × 10<sup>−8</sup>).</p></div><div id="S12" class="sec"><h3 id="S12title">Cup Area</h3><p id="P19" class="p p-first">Stage 1 included 17,218 individuals of European ancestry, with λ-values varying between 0.98 and 1.06 (1.10 for the meta-analysis), implying adequate within-study control of population substructure (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S2</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. S6A, B, and C</a>). In total, 342 SNPs located across 15 chromosomal regions were genome-wide significant (<a href="/pmc/articles/PMC4480365/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. S5A</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S4</a>).</p><p id="P20">Stage 2 consisted of 6,613 individuals of Asian ancestry (λ 1.01–1.03). Nine of the 15 most associated SNPs across the 15 chromosomal regions were nominal significant in this Asian population. The most significantly associated SNP on chromosome 16 in Europeans could not be imputed with sufficient accuracy for use in individuals of Asian ancestry (<em>SALL1</em>). The second most significant associated SNP in the Europeans (rs4238758, <em>β</em> = −0.02, <em>P</em> = 4.83 × 10<sup>−8</sup>) did not replicate in individuals of Asian ancestry (<em>β</em> = −0.02, <em>P</em> = 3.11 × 10<sup>−1</sup>).</p><p id="P21">In stage 3, the combined analysis (meta-analysis λ 1.10) yielded seven additional genome-wide significant loci. Of the 22 (15 + 7) chromosomal regions, 12 were previously genome-wide significant associated with the VCDR, the clinically used optic disc parameter [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029369">Springelkamp et al., 2014</a>]. The VCDR is highly correlated to cup area (<em>r</em> = 0.78, calculated in the Rotterdam Study I). The other 10 loci were new: <em>DHRS3</em> (chr.1), <em>TRIB2</em> (chr.2), <em>KPNB1</em> (chr.17), and <em>BCAS3</em> (chr.17) identified in stage 1, and <em>EFEMP1</em> (chr. 2), <em>FLNB</em> (chr.3), <em>FAM101A</em> (chr.12), <em>DDHD1</em> (chr.14), <em>ASB7</em> (chr.15), and <em>TRIOBP</em> (chr.22) identified in stage 3.</p><p id="P22" class="p p-last">In the gene-based analysis, <em>FAT4</em> was significantly associated with cup area, but this association disappeared after correction for disc area. This gene is also associated with disc area (nominal significant; <em>P</em> = 6.69 × 10<sup>−3</sup>) suggesting that <em>FAT4</em> acts primarily through its effect on disc area. For the cup area adjusted for disc area analysis, 27 genes were significant but all of them are located in regions identified by the GWAS.</p></div><div id="S13" class="sec"><h3 id="S13title">From Genes to Glaucoma</h3><p id="P23" class="p p-first-last">To investigate the relevance of the disc area and cup area SNPs in the clinical disease glaucoma, we calculated the explained variance of glaucoma in ANZRAG and NEIGHBOR. The top SNPs from the disc area analysis (<em>P</em> &lt; 10<sup>−8</sup>) explained 0.1% (ANZRAG) and 0.07% (NEIGHBOR) of the variance of glaucoma (<a href="/pmc/articles/PMC4480365/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The top SNPs from the cup area analyses (<em>P</em> &lt; 10<sup>−8</sup>), explained 2.1% (ANZRAG), and 3.2% (NEIGHBOR) of the variance. The top SNPs mainly consisted of SNPs in <em>CDKN2B-AS1</em> and <em>SIX6.</em> To investigate the effect of other SNPs, we removed SNPs within 1 MB from <em>CDKN2B-AS1</em> and <em>SIX6</em> in ANZRAG. The explained variance of glaucoma decreased from 1.5% to 1.0% (SNPs <em>P</em> &lt; 0.1), but was still significant (<em>P</em> = 1.36 × 10<sup>−6</sup>). In the Rotterdam Study I, the 10 new cup area SNPs explained an additional 0.9% of the VCDR phenotypic variability compared to known VCDR SNPs [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029375">Springelkamp et al., 2014</a>].</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p40">The explained variance (Nagelkerke <em>R</em><sup>2</sup>) for glaucoma in ANZRAG (1,155 cases and 1,992 controls) and NEIGHBOR (2,131 cases and 2,290 controls) determined by the single nucleotide polymorphisms from the genome-wide association analysis for disc area and cup area</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140153444835952" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="left"><em>P</em>-value threshold</th><th rowspan="3" colspan="1" valign="bottom" align="right">number of tested SNPs</th><th colspan="5" rowspan="1" valign="bottom" align="center">Disc area
<hr></th><th rowspan="3" colspan="1" valign="bottom" align="right">Number of tested SNPs</th><th colspan="5" rowspan="1" valign="bottom" align="center">Cup area
<hr></th></tr><tr><th colspan="3" rowspan="1" valign="bottom" align="center">ANZRAG
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">NEIGHBOR
<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">ANZRAG
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">NEIGHBOR
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">Beta</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>R</em><sup>2</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em>-value</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>R</em><sup>2</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em>-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Beta</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>R</em><sup>2</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em>-value</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>R</em><sup>2</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="right">63</td><td rowspan="1" colspan="1" valign="top" align="center">−0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.0011</td><td rowspan="1" colspan="1" valign="top" align="center">1.08 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0007</td><td rowspan="1" colspan="1" valign="top" align="center">1.25 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="right">55</td><td rowspan="1" colspan="1" valign="top" align="center">1.96</td><td rowspan="1" colspan="1" valign="top" align="center">0.0207</td><td rowspan="1" colspan="1" valign="top" align="center">4.08 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0316</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;2.00 × 10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="right">85</td><td rowspan="1" colspan="1" valign="top" align="center">−0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.0010</td><td rowspan="1" colspan="1" valign="top" align="center">1.24 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0007</td><td rowspan="1" colspan="1" valign="top" align="center">1.34 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="right">87</td><td rowspan="1" colspan="1" valign="top" align="center">1.82</td><td rowspan="1" colspan="1" valign="top" align="center">0.0207</td><td rowspan="1" colspan="1" valign="top" align="center">4.22 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0353</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;2.00 × 10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="right">106</td><td rowspan="1" colspan="1" valign="top" align="center">−0.12</td><td rowspan="1" colspan="1" valign="top" align="center">0.0009</td><td rowspan="1" colspan="1" valign="top" align="center">1.52 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0008</td><td rowspan="1" colspan="1" valign="top" align="center">1.02 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="right">124</td><td rowspan="1" colspan="1" valign="top" align="center">1.56</td><td rowspan="1" colspan="1" valign="top" align="center">0.0179</td><td rowspan="1" colspan="1" valign="top" align="center">1.23 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0355</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;2.00 × 10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">173</td><td rowspan="1" colspan="1" valign="top" align="center">−0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.0005</td><td rowspan="1" colspan="1" valign="top" align="center">2.62 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0014</td><td rowspan="1" colspan="1" valign="top" align="center">2.95 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="right">183</td><td rowspan="1" colspan="1" valign="top" align="center">1.43</td><td rowspan="1" colspan="1" valign="top" align="center">0.0190</td><td rowspan="1" colspan="1" valign="top" align="center">3.30 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0348</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;2.00 × 10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="right">340</td><td rowspan="1" colspan="1" valign="top" align="center">−0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.0001</td><td rowspan="1" colspan="1" valign="top" align="center">5.87 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0019</td><td rowspan="1" colspan="1" valign="top" align="center">1.29 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="right">428</td><td rowspan="1" colspan="1" valign="top" align="center">1.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.0180</td><td rowspan="1" colspan="1" valign="top" align="center">1.11 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0313</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;2.00 × 10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="right">1,315</td><td rowspan="1" colspan="1" valign="top" align="center">−0.01</td><td rowspan="1" colspan="1" valign="top" align="center">0.0000</td><td rowspan="1" colspan="1" valign="top" align="center">9.22 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0034</td><td rowspan="1" colspan="1" valign="top" align="center">7.72 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="right">1,410</td><td rowspan="1" colspan="1" valign="top" align="center">0.88</td><td rowspan="1" colspan="1" valign="top" align="center">0.0211</td><td rowspan="1" colspan="1" valign="top" align="center">2.48 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0316</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;2.00 × 10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="right">7,731</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.0015</td><td rowspan="1" colspan="1" valign="top" align="center">6.67 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0040</td><td rowspan="1" colspan="1" valign="top" align="center">2.85 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="right">7,782</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">0.0183</td><td rowspan="1" colspan="1" valign="top" align="center">7.84 ×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0284</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;2.00 × 10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;0.1</td><td rowspan="1" colspan="1" valign="top" align="right">56,542</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.0055</td><td rowspan="1" colspan="1" valign="top" align="center">3.71 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0038</td><td rowspan="1" colspan="1" valign="top" align="center">3.87 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="right">58,133</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.0151</td><td rowspan="1" colspan="1" valign="top" align="center">3.42 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0165</td><td rowspan="1" colspan="1" valign="top" align="center">1.52 × 10<sup>−13</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;0.2</td><td rowspan="1" colspan="1" valign="top" align="right">106,818</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">0.0048</td><td rowspan="1" colspan="1" valign="top" align="center">8.39 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0049</td><td rowspan="1" colspan="1" valign="top" align="center">5.73 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">108,666</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.0132</td><td rowspan="1" colspan="1" valign="top" align="center">3.14 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0149</td><td rowspan="1" colspan="1" valign="top" align="center">2.44 × 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;0.3</td><td rowspan="1" colspan="1" valign="top" align="right">155,892</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.0038</td><td rowspan="1" colspan="1" valign="top" align="center">2.94 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0054</td><td rowspan="1" colspan="1" valign="top" align="center">2.54 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">157,742</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.0114</td><td rowspan="1" colspan="1" valign="top" align="center">2.94 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0142</td><td rowspan="1" colspan="1" valign="top" align="center">7.10 × 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;0.4</td><td rowspan="1" colspan="1" valign="top" align="right">204,028</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.0041</td><td rowspan="1" colspan="1" valign="top" align="center">2.20 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0053</td><td rowspan="1" colspan="1" valign="top" align="center">2.76 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">205,288</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.0107</td><td rowspan="1" colspan="1" valign="top" align="center">6.76 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0137</td><td rowspan="1" colspan="1" valign="top" align="center">1.87 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;0.5</td><td rowspan="1" colspan="1" valign="top" align="right">252,003</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.0042</td><td rowspan="1" colspan="1" valign="top" align="center">1.97 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0057</td><td rowspan="1" colspan="1" valign="top" align="center">1.42 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">252,595</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.0110</td><td rowspan="1" colspan="1" valign="top" align="center">4.51 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0127</td><td rowspan="1" colspan="1" valign="top" align="center">1.01 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;0.6</td><td rowspan="1" colspan="1" valign="top" align="right">299,085</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.0042</td><td rowspan="1" colspan="1" valign="top" align="center">1.93 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0059</td><td rowspan="1" colspan="1" valign="top" align="center">1.05 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">299,291</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.0112</td><td rowspan="1" colspan="1" valign="top" align="center">3.60 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0123</td><td rowspan="1" colspan="1" valign="top" align="center">1.83 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;0.7</td><td rowspan="1" colspan="1" valign="top" align="right">345,745</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.0042</td><td rowspan="1" colspan="1" valign="top" align="center">1.96 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0058</td><td rowspan="1" colspan="1" valign="top" align="center">1.30 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">346,007</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.0110</td><td rowspan="1" colspan="1" valign="top" align="center">4.82 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0126</td><td rowspan="1" colspan="1" valign="top" align="center">1.19 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;0.8</td><td rowspan="1" colspan="1" valign="top" align="right">392,318</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.0042</td><td rowspan="1" colspan="1" valign="top" align="center">1.83 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0058</td><td rowspan="1" colspan="1" valign="top" align="center">1.19 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">392,623</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.0111</td><td rowspan="1" colspan="1" valign="top" align="center">4.26 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0127</td><td rowspan="1" colspan="1" valign="top" align="center">9.37 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;0.9</td><td rowspan="1" colspan="1" valign="top" align="right">438,618</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.0042</td><td rowspan="1" colspan="1" valign="top" align="center">1.96 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0058</td><td rowspan="1" colspan="1" valign="top" align="center">1.15 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">438,802</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.0109</td><td rowspan="1" colspan="1" valign="top" align="center">4.96 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0125</td><td rowspan="1" colspan="1" valign="top" align="center">1.23 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&lt;1.0</td><td rowspan="1" colspan="1" valign="top" align="right">484,974</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.0041</td><td rowspan="1" colspan="1" valign="top" align="center">2.04 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0058</td><td rowspan="1" colspan="1" valign="top" align="center">1.15 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">484,974</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.0110</td><td rowspan="1" colspan="1" valign="top" align="center">4.73 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0126</td><td rowspan="1" colspan="1" valign="top" align="center">1.19 × 10<sup>−10</sup></td></tr></tbody></table></div><div id="largeobj_idm140153444835952" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4480365/table/T3/?report=objectonly">Open in a separate window</a></div></div></div><div id="S14" class="sec sec-last"><h3 id="S14title">Pathway Analysis</h3><p id="P24" class="p p-first-last">To test whether the genes found through the VEGAS gene-based approach were enriched in 4,628 prespecified Gene Ontology pathways we performed a pathway analysis using Pathway-VEGAS [<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029382">Lu et al., 2013</a>]. We used a pathway-wide significance threshold of 1.08 × 10<sup>−5</sup> (0.05/4,628). One pathway exceeded the pathway-wide significance level for disc area: “Entrainment of circadian clock” (<em>P</em> = 8.00 × 10<sup>−6</sup>). This pathway result was driven by the strong association signal at <em>ATOH7</em>. For cup area (unadjusted for disc area), the top pathway is “Negative regulation of cyclin-dependent protein kinase activity” that is also associated with VCDR [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029379">Springelkamp et al., 2014</a>]. After adjustment for disc area, the top pathway for cup area was “G1/S transition checkpoint” (<em>P</em> = 4.66 × 10<sup>−5</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S6</a>). The known POAG gene <em>CDKN2B-AS1</em> is part of this pathway.</p></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Discussion</h2><p id="P25" class="p p-first">This study identified new genetic loci associated with two parameters describing the morphology of the optic nerve head. In total, we identified 10 new disc area loci and 10 new cup area loci. Gene-based analysis identified one additional region associated with disc area.</p><p id="P26">Of the 10 new disc area loci, two were identified in stage 1 and did replicate in stage 2. The eight other new loci were identified in stage 3 and therefore replication is missing because of lack of samples. In the cup area analysis, four new loci were identified in stage 1 and the <em>KPNB1</em> and <em>BCAS3</em> SNPs did not replicate in stage 2. However, the effect estimates are similar and in the same direction in Caucasian and Asian populations so this might be due to lack of power since we included less samples in stage 2. For the six other new loci from stage 3, replication is also missing. Although there is lack of replication for the new loci identified in stage 3, the <em>P</em>-values of the associations from stage 1 are low for these SNPs and the effect estimates are similar and in the same direction in stage 2, suggesting that these new loci are real new loci. Some SNPs showed heterogeneity. Therefore, we ran also a random-effect meta-analysis. For the new loci, most effect estimates and <em>P</em>-values remain similar after the random-effect meta-analysis. Only the <em>P</em>-value for <em>BCAS3</em> (cup area) decreased from 4.49 × 10<sup>−11</sup> to 9.27 × 10<sup>−3</sup>, but the effect estimate remained similar (−0.018 vs. −0.017), which is compatible with the heterogeneity as measured with the I<sup>2</sup>.</p><p id="P27">We investigated the expression of the genes implicated in the parameters for optic nerve head areas by these analyses in various eye tissues using published literature or human ocular gene expression databases (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables S7 and S8</a>) [<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029387">Bowes Rickman et al., 2006</a>; <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029368">Liu et al., 2011</a>; <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029386">Wagner et al., 2013</a>; <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029385">Young et al., 2013</a>]. The highest expression in the optic nerve was found for <em>ABI3BP</em>. Most of the other genes were also expressed in the optic nerve or other glaucoma-related eye tissues like the trabecular meshwork and the cornea.</p><p id="P28">The genes in the new disc area loci have different functions. An interesting gene is <em>RARB</em>, which limits cell growth by regulating gene expression. Also <em>NR2F2</em> plays a role in gene regulation. <em>PAX6</em> was identified by gene-based analysis. Although <em>PAX6</em> is a neighboring gene (with linkage disequilibrium extending across this region) to <em>ELP4</em>, which was associated with disc area in the GWAS, the strong biological relevance of <em>PAX6</em> to eye development (it is expressed in developing eyes, and rare mutations cause aniridia, a rare developmental eye disorder [<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029378">Jordan et al., 1992</a>]) suggests that genetic variation in this region more likely influences the regulation of <em>PAX6</em> rather than other genes in the region.</p><p id="P29">Our study shows that studies of optic nerve head parameters may shed light on clinical outcomes. The genetic overlap between disc area and glaucoma is small, but the direction of the significant risk scores (<em>P</em> threshold &lt; 0.1 and higher) might suggest that a larger disc area increases the risk of glaucoma. There is a strong genetic overlap between the cup area and glaucoma: 2.1% and 3.2% of the variance of glaucoma is explained by the most significant SNPs for cup area in two independent glaucoma case–control studies (ANZRAG and NEIGHBOR, respectively). This is mostly explained by the known genes <em>CDKN2B-AS1</em> and <em>SIX6</em>, but SNPs in other genes explain also 1.0% of the variance, based on a polygenic risk score comprising all SNPs associated at <em>P</em> &lt; 0.1 with cup area. The loci that are associated with cup area are also associated with VCDR (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S9</a>). The region on chromosome 22 (with the top SNP rs5756813) contains the <em>CARD10</em> gene that was previously reported to be associated with disc area. However, it seems that the <em>TRIOBP</em> gene is responsible for the association with cup area. Its protein interacts with trio, which is interesting because of the role of trio in neural tissue development [<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029377">Seipel et al., 2001</a>]. The nearest gene to the top SNP on chromosome 14 (rs10130566) is <em>DDHD1</em>, but the association might be explained by the <em>BMP4</em> gene. This gene is a member of the bone morphogenetic protein family, which is part of the transforming growth factor-beta superfamily. Another member of this family is <em>BMP2</em>, which is also associated with VCDR [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029373">Springelkamp et al., 2014</a>]. While the top SNP on chromosome 15 is located near to the <em>ASB4</em> gene, the <em>ADAMTS17</em> in this region may contribute more to disease susceptibility. This gene belongs to the same family of <em>ADAMTS8,</em> which is associated with VCDR [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029398">Springelkamp et al., 2014</a>]. Furthermore, <em>ADAMTS17</em> is already linked to some forms of (syndromal) glaucoma [<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029391">Morales et al., 2009</a>]. Pathway analysis implicated that cell growth and death is an important mechanism associated with cup area.</p><p id="P30"><a href="/pmc/articles/PMC4480365/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> shows the overlap between the different optic nerve head area parameters. Overall, most loci were only associated with disc area or cup area. <em>ATOH7</em> was associated with disc and cup area as well as with glaucoma [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029380">Ramdas et al., 2011</a>]. <em>SIX6</em> and <em>CDKN2B-AS1</em> were associated with cup area and glaucoma [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029394">Burdon et al., 2011</a>; <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029381">Wiggs et al., 2012</a>]. The figure shows only genome-wide significant SNPs, but it is likely that other SNPs affect also more than one trait, including rs11129176 (<em>RARB</em>), which is genome-wide significant in disc area and reached a <em>P</em>-value of 1.12 × 10<sup>−5</sup> in the cup area analysis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4480365/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140153447533888" class="figure"><a href="/pmc/articles/PMC4480365/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4480365_nihms700469f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms700469f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4480365/bin/nihms700469f3.jpg"></a></div><div id="largeobj_idm140153447533888" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4480365/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4480365/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p33">Overlap between the different optic nerve head parameters. Genes that reached genome-wide significance are shown. VCDR = vertical cup-disc ratio, * = genes that have been previously associated with glaucoma. All SNPs associated with cup area, are also associated with VCDR (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S9</a>).</p></div></div></div><p id="P31" class="p p-last">In summary, we found 20 new loci associated with optic nerve head area and/or cupping which explain a further proportion of the missing heritability of glaucoma. These results showed that investigation of more refined measurements of optic nerve head morphology, especially the cup area, is a fruitful approach to discover new glaucoma-related loci, in addition to the crude VCDR linear measurement commonly used in clinical practice and previously investigated [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_458029372">Springelkamp et al., 2014</a>]. The new loci contain many genes with different functions, and while there appears to be one strong candidate causal gene in some regions, there are several possible candidate genes in others. Further research including exome sequencing and functional studies is necessary to unravel the causative associations in the gene-dense regions and the mechanism of these genes in the pathophysiology of glaucoma. Our findings are an important step toward a better understanding of the disease.</p></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4480365/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140153447543920" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms700469f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms700469f1.jpg" src="/pmc/articles/PMC4480365/bin/nihms700469f1.jpg"></div></a><div id="largeobj_idm140153447543920" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4480365/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC4480365/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4480365/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p31">Manhattan plot of the GWAS meta-analysis for disc area in the combined analysis (<em>n</em> = 24,089 subjects of European and Asian ancestry). The plot shows −log10-transformed <em>P</em>-values for all single nucleotide polymorphisms. The red dotted horizontal line represents the genome-wide significance threshold of <em>P</em> &lt; 5.0 × 10<sup>−8</sup>; the blue dotted line indicates <em>P</em>-value of 1 × 10<sup>−5</sup>. Gene loci in bold have been previously associated with disc area.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4480365/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140153447537616" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms700469f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms700469f2.jpg" src="/pmc/articles/PMC4480365/bin/nihms700469f2.jpg"></div></a><div id="largeobj_idm140153447537616" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4480365/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC4480365/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4480365/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p32">Manhattan plot of the GWAS meta-analysis for cup area (adjusted for disc area) in the combined analysis (<em>n</em> = 23,831 subjects of European and Asian ancestry). The plot shows −log10-transformed <em>P</em>-values for all single nucleotide polymorphisms. The red dotted horizontal line represents the genome-wide significance threshold of <em>P</em> &lt; 5.0 × 10<sup>−8</sup>; the blue dotted line indicates <em>P</em>-value of 1 × 10<sup>−5</sup>. Gene loci in bold have been previously associated with cup area.</p></div></div></div></div></div><div id="S16" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>supplement</h4><div class="sup-box half_rhythm" id="d36e1555"><a href="/pmc/articles/PMC4480365/bin/NIHMS700469-supplement-supplement.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(9.7M, docx)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="__p4">We would like to thank the contributions of all study participants and staff at the recruitment centers. Complete funding information and acknowledgments for each individual study can be found in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a>.</p></div></div><div id="APP1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP1title">NEIGHBORHOOD Consortium membership list</h2><!--/article/back/app-group/app/--><p id="P32">R. Rand Allingham<sup><sup><a href="#FN5" rid="FN5" class=" fn">1</a></sup></sup>, Murray H. Brilliant<sup><sup><a href="#FN6" rid="FN6" class=" fn">2</a></sup></sup>, Donald L. Budenz<sup><sup><a href="#FN7" rid="FN7" class=" fn">3</a></sup></sup>, Jessica N. Cooke Bailey<sup><sup><a href="#FN8" rid="FN8" class=" fn">4</a></sup></sup>, William G. Christen<sup><sup><a href="#FN9" rid="FN9" class=" fn">5</a></sup></sup>, John Fingert<sup><sup><a href="#FN10" rid="FN10" class=" fn">6</a></sup>,<sup><a href="#FN11" rid="FN11" class=" fn">7</a></sup></sup>, Douglas Gaasterland<sup><sup><a href="#FN12" rid="FN12" class=" fn">8</a></sup></sup>, Terry Gaasterland<sup><sup><a href="#FN13" rid="FN13" class=" fn">9</a></sup></sup>, Jonathan L Haines<sup><sup><a href="#FN8" rid="FN8" class=" fn">4</a></sup></sup>, Michael A Hauser<sup><sup><a href="#FN5" rid="FN5" class=" fn">1</a></sup>,<sup><a href="#FN14" rid="FN14" class=" fn">10</a></sup></sup>, Jae H. Kang<sup><sup><a href="#FN15" rid="FN15" class=" fn">11</a></sup></sup>, Peter Kraft<sup><sup><a href="#FN16" rid="FN16" class=" fn">12</a></sup></sup>, Richard K Lee<sup><sup><a href="#FN17" rid="FN17" class=" fn">13</a></sup></sup>, Paul A Lichter<sup><sup><a href="#FN18" rid="FN18" class=" fn">14</a></sup></sup>, Yutao Liu<sup><sup><a href="#FN5" rid="FN5" class=" fn">1</a></sup>,<sup><a href="#FN14" rid="FN14" class=" fn">10</a></sup></sup>, Stephanie J Loomis<sup><sup><a href="#FN19" rid="FN19" class=" fn">15</a></sup></sup>, Sayoko E Moroi<sup><sup><a href="#FN18" rid="FN18" class=" fn">14</a></sup></sup>, Louis R Pasquale<sup><sup><a href="#FN15" rid="FN15" class=" fn">11</a></sup>,<sup><a href="#FN19" rid="FN19" class=" fn">15</a></sup></sup>, Margaret A Pericak-Vance<sup><sup><a href="#FN20" rid="FN20" class=" fn">16</a></sup></sup>, Anthony Realini<sup><sup><a href="#FN21" rid="FN21" class=" fn">17</a></sup></sup>, Julia E Richards<sup><sup><a href="#FN18" rid="FN18" class=" fn">14</a></sup></sup>, Joel S Schuman<sup><sup><a href="#FN22" rid="FN22" class=" fn">18</a></sup></sup>, William K Scott<sup><sup><a href="#FN20" rid="FN20" class=" fn">16</a></sup></sup>, Kuldev Singh<sup><sup><a href="#FN23" rid="FN23" class=" fn">19</a></sup></sup>, Arthur J Sit<sup><sup><a href="#FN24" rid="FN24" class=" fn">20</a></sup></sup>, Douglas Vollrath<sup><sup><a href="#FN25" rid="FN25" class=" fn">21</a></sup></sup>, Robert N Weinreb<sup><sup><a href="#FN26" rid="FN26" class=" fn">22</a></sup></sup>, Janey L Wiggs<sup><sup><a href="#FN19" rid="FN19" class=" fn">15</a></sup></sup>, Gadi Wollstein<sup><sup><a href="#FN22" rid="FN22" class=" fn">18</a></sup></sup>, Donald J Zack<sup><sup><a href="#FN27" rid="FN27" class=" fn">23</a></sup></sup>, Kang Zhang<sup><sup><a href="#FN26" rid="FN26" class=" fn">22</a></sup></sup>.</p></div><div id="idm140153443578512" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140153443578512title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN5"><sup>1</sup>Department of Ophthalmology, Duke University Medical Center, Durham, North Carolina, USA.</p><p class="fn sec" id="FN6"><sup>2</sup>Center for Human Genetics, Marshfield Clinic Research Foundation, Marshfield, Wisconsin, USA.</p><p class="fn sec" id="FN7"><sup>3</sup>Department of Ophthalmology, University of North Carolina, Chapel Hill, North Carolina, USA.</p><p class="fn sec" id="FN8"><sup>4</sup>Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, Ohio, USA.</p><p class="fn sec" id="FN9"><sup>5</sup>Department of Medicine, Brigham and Women’s Hospital, Boston, Massachusetts, USA.</p><p class="fn sec" id="FN10"><sup>6</sup>Department of Anatomy/Cell Biology, College of Medicine, University of Iowa, Iowa City, Iowa, USA.</p><p class="fn sec" id="FN11"><sup>7</sup>Department of Ophthalmology, College of Medicine, University of Iowa, Iowa City, Iowa, USA.</p><p class="fn sec" id="FN12"><sup>8</sup>Eye Doctors of Washington, Chevy Chase, Maryland, USA.</p><p class="fn sec" id="FN13"><sup>9</sup>Scripps Genome Center, University of California at San Diego, San Diego, California, USA.</p><p class="fn sec" id="FN14"><sup>10</sup>Department of Medicine, Duke University Medical Center, Durham, North Carolina, USA.</p><p class="fn sec" id="FN15"><sup>11</sup>Channing Division of Network Medicine, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts, USA.</p><p class="fn sec" id="FN16"><sup>12</sup>Department of Biostatistics, Harvard School of Public Health, Boston, Massachusetts, USA.</p><p class="fn sec" id="FN17"><sup>13</sup>Bascom Palmer Eye Institute, University of Miami Miller School of Medicine, Miami, Florida, USA.</p><p class="fn sec" id="FN18"><sup>14</sup>Department of Ophthalmology and Visual Sciences, University of Michigan, Ann Arbor, Michigan, USA.</p><p class="fn sec" id="FN19"><sup>15</sup>Department of Ophthalmology, Harvard Medical School and Massachusetts Eye and Ear Infirmary, Boston, Massachusetts, USA.</p><p class="fn sec" id="FN20"><sup>16</sup>Institute for Human Genomics, University of Miami Miller School of Medicine, Miami, Florida, USA.</p><p class="fn sec" id="FN21"><sup>17</sup>Department of Ophthalmology, WVU Eye Institute, Morgantown, West Virginia, USA.</p><p class="fn sec" id="FN22"><sup>18</sup>Department of Ophthalmology, UPMC Eye Center, University of Pittsburgh, Pittsburgh, Pennsylvania, USA.</p><p class="fn sec" id="FN23"><sup>19</sup>Department of Ophthalmology, Stanford University, Palo Alto, California, USA.</p><p class="fn sec" id="FN24"><sup>20</sup>Department of Ophthalmology, Mayo Clinic, Rochester, Minnesota, USA.</p><p class="fn sec" id="FN25"><sup>21</sup>Department of Genetics, Stanford University, Palo Alto, California, USA.</p><p class="fn sec" id="FN26"><sup>22</sup>Department of Ophthalmology, Hamilton Eye Center, University of California, San Diego, California, USA.</p><p class="fn sec" id="FN27"><sup>23</sup>Wilmer Eye Institute, John Hopkins University, Baltimore, Maryland, USA.</p><p class="fn sec" id="FN28"></p><p id="__p28" class="p p-first"><strong>Conflict of interest</strong></p><p id="__p29" class="p p-last">The authors declare no competing financial interests.</p><p></p><p class="fn sec" id="FN29"></p><p id="__p30" class="p p-first-last"><a href="#SD1" rid="SD1" class=" supplementary-material">Supporting Information</a> is available in the online issue at <a href="http://wileyonlinelibrary.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">wileyonlinelibrary.com</a>.</p><p></p></div></div><div id="idm140153483258000" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140153483258000title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Axenovich T, Zorkoltseva I, Belonogova N, van Koolwijk LM, Borodin P, Kirichenko A, Babenko V, Ramdas WD, Amin N, Despriet DD, et al.  Linkage and association analyses of glaucoma related traits in a large pedigree from a Dutch genetically isolated population. <span><span class="ref-journal">J Med Genet. </span>2011;<span class="ref-vol">48</span>(12):802–809.</span> [<a href="/pubmed/22058429" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Linkage+and+association+analyses+of+glaucoma+related+traits+in+a+large+pedigree+from+a+Dutch+genetically+isolated+population&amp;author=T+Axenovich&amp;author=I+Zorkoltseva&amp;author=N+Belonogova&amp;author=LM+van+Koolwijk&amp;author=P+Borodin&amp;volume=48&amp;issue=12&amp;publication_year=2011&amp;pages=802-809&amp;pmid=22058429&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Bowes Rickman C, Ebright JN, Zavodni ZJ, Yu L, Wang T, Daiger SP, Wistow G, Boon K, Hauser MA. Defining the human macula transcriptome and candidate retinal disease genes using EyeSAGE. <span><span class="ref-journal">Invest Ophthalmol Vis Sci. </span>2006;<span class="ref-vol">47</span>(6):2305–2316.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2813776/">PMC free article</a>]</span> [<a href="/pubmed/16723438" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest+Ophthalmol+Vis+Sci&amp;title=Defining+the+human+macula+transcriptome+and+candidate+retinal+disease+genes+using+EyeSAGE&amp;author=C+Bowes+Rickman&amp;author=JN+Ebright&amp;author=ZJ+Zavodni&amp;author=L+Yu&amp;author=T+Wang&amp;volume=47&amp;issue=6&amp;publication_year=2006&amp;pages=2305-2316&amp;pmid=16723438&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Burdon KP, Macgregor S, Hewitt AW, Sharma S, Chidlow G, Mills RA, Danoy P, Casson R, Viswanathan AC, Liu JZ, et al.  Genome-wide association study identifies susceptibility loci for open angle glaucoma at TMCO1 and CDKN2B-AS1. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>(6):574–578.</span> [<a href="/pubmed/21532571" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+susceptibility+loci+for+open+angle+glaucoma+at+TMCO1+and+CDKN2B-AS1&amp;author=KP+Burdon&amp;author=S+Macgregor&amp;author=AW+Hewitt&amp;author=S+Sharma&amp;author=G+Chidlow&amp;volume=43&amp;issue=6&amp;publication_year=2011&amp;pages=574-578&amp;pmid=21532571&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Jonas JB, Muller-Bergh JA, Schlotzer-Schrehardt UM, Naumann GO. Histomorphometry of the human optic nerve. <span><span class="ref-journal">Invest Ophthalmol Vis Sci. </span>1990;<span class="ref-vol">31</span>(4):736–744.</span> [<a href="/pubmed/2335441" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest+Ophthalmol+Vis+Sci&amp;title=Histomorphometry+of+the+human+optic+nerve&amp;author=JB+Jonas&amp;author=JA+Muller-Bergh&amp;author=UM+Schlotzer-Schrehardt&amp;author=GO+Naumann&amp;volume=31&amp;issue=4&amp;publication_year=1990&amp;pages=736-744&amp;pmid=2335441&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Jonas JB, Schmidt AM, Muller-Bergh JA, Schlotzer-Schrehardt UM, Naumann GO. Human optic nerve fiber count and optic disc size. <span><span class="ref-journal">Invest Ophthalmol Vis Sci. </span>1992;<span class="ref-vol">33</span>(6):2012–2018.</span> [<a href="/pubmed/1582806" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest+Ophthalmol+Vis+Sci&amp;title=Human+optic+nerve+fiber+count+and+optic+disc+size&amp;author=JB+Jonas&amp;author=AM+Schmidt&amp;author=JA+Muller-Bergh&amp;author=UM+Schlotzer-Schrehardt&amp;author=GO+Naumann&amp;volume=33&amp;issue=6&amp;publication_year=1992&amp;pages=2012-2018&amp;pmid=1582806&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Jordan T, Hanson I, Zaletayev D, Hodgson S, Prosser J, Seawright A, Hastie N, van Heyningen V. The human PAX6 gene is mutated in two patients with aniridia. <span><span class="ref-journal">Nat Genet. </span>1992;<span class="ref-vol">1</span>(5):328–332.</span> [<a href="/pubmed/1302030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+human+PAX6+gene+is+mutated+in+two+patients+with+aniridia&amp;author=T+Jordan&amp;author=I+Hanson&amp;author=D+Zaletayev&amp;author=S+Hodgson&amp;author=J+Prosser&amp;volume=1&amp;issue=5&amp;publication_year=1992&amp;pages=328-332&amp;pmid=1302030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Khor CC, Ramdas WD, Vithana EN, Cornes BK, Sim X, Tay WT, Saw SM, Zheng Y, Lavanya R, Wu R, et al.  Genome-wide association studies in Asians confirm the involvement of ATOH7 and TGFBR3, and further identify CARD10 as a novel locus influencing optic disc area. <span><span class="ref-journal">Hum Mol Genet. </span>2011;<span class="ref-vol">20</span>(9):1864–1872.</span> [<a href="/pubmed/21307088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+studies+in+Asians+confirm+the+involvement+of+ATOH7+and+TGFBR3,+and+further+identify+CARD10+as+a+novel+locus+influencing+optic+disc+area&amp;author=CC+Khor&amp;author=WD+Ramdas&amp;author=EN+Vithana&amp;author=BK+Cornes&amp;author=X+Sim&amp;volume=20&amp;issue=9&amp;publication_year=2011&amp;pages=1864-1872&amp;pmid=21307088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2010;<span class="ref-vol">34</span>(8):816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y+Li&amp;author=CJ+Willer&amp;author=J+Ding&amp;author=P+Scheet&amp;author=GR+Abecasis&amp;volume=34&amp;issue=8&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Liu JZ, McRae AF, Nyholt DR, Medland SE, Wray NR, Brown KM, Investigators A, Hayward NK, Montgomery GW, Visscher PM, et al.  A versatile gene-based test for genome-wide association studies. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>(1):139–145.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896770/">PMC free article</a>]</span> [<a href="/pubmed/20598278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+versatile+gene-based+test+for+genome-wide+association+studies&amp;author=JZ+Liu&amp;author=AF+McRae&amp;author=DR+Nyholt&amp;author=SE+Medland&amp;author=NR+Wray&amp;volume=87&amp;issue=1&amp;publication_year=2010&amp;pages=139-145&amp;pmid=20598278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Liu Y, Munro D, Layfield D, Dellinger A, Walter J, Peterson K, Rickman CB, Allingham RR, Hauser MA. Serial analysis of gene expression (SAGE) in normal human trabecular meshwork. <span><span class="ref-journal">Mol Vis. </span>2011;<span class="ref-vol">17</span>:885–893.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3081805/">PMC free article</a>]</span> [<a href="/pubmed/21528004" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Vis&amp;title=Serial+analysis+of+gene+expression+(SAGE)+in+normal+human+trabecular+meshwork&amp;author=Y+Liu&amp;author=D+Munro&amp;author=D+Layfield&amp;author=A+Dellinger&amp;author=J+Walter&amp;volume=17&amp;publication_year=2011&amp;pages=885-893&amp;pmid=21528004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Lu Y, Vitart V, Burdon KP, Khor CC, Bykhovskaya Y, Mirshahi A, Hewitt AW, Koehn D, Hysi PG, Ramdas WD, et al.  Genome-wide association analyses identify multiple loci associated with central corneal thickness and keratoconus. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>(2):155–163.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3720123/">PMC free article</a>]</span> [<a href="/pubmed/23291589" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analyses+identify+multiple+loci+associated+with+central+corneal+thickness+and+keratoconus&amp;author=Y+Lu&amp;author=V+Vitart&amp;author=KP+Burdon&amp;author=CC+Khor&amp;author=Y+Bykhovskaya&amp;volume=45&amp;issue=2&amp;publication_year=2013&amp;pages=155-163&amp;pmid=23291589&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Macgregor S, Hewitt AW, Hysi PG, Ruddle JB, Medland SE, Henders AK, Gordon SD, Andrew T, McEvoy B, Sanfilippo PG, et al.  Genome-wide association identifies ATOH7 as a major gene determining human optic disc size. <span><span class="ref-journal">Hum Mol Genet. </span>2010;<span class="ref-vol">19</span>(13):2716–2724.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2883339/">PMC free article</a>]</span> [<a href="/pubmed/20395239" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+identifies+ATOH7+as+a+major+gene+determining+human+optic+disc+size&amp;author=S+Macgregor&amp;author=AW+Hewitt&amp;author=PG+Hysi&amp;author=JB+Ruddle&amp;author=SE+Medland&amp;volume=19&amp;issue=13&amp;publication_year=2010&amp;pages=2716-2724&amp;pmid=20395239&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>(7):906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;issue=7&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Morales J, Al-Sharif L, Khalil DS, Shinwari JM, Bavi P, Al-Mahrouqi RA, Al-Rajhi A, Alkuraya FS, Meyer BF, Al Tassan N. Homozygous mutations in ADAMTS10 and ADAMTS17 cause lenticular myopia, ectopia lentis, glaucoma, spherophakia, and short stature. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">85</span>(5):558–568.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775842/">PMC free article</a>]</span> [<a href="/pubmed/19836009" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Homozygous+mutations+in+ADAMTS10+and+ADAMTS17+cause+lenticular+myopia,+ectopia+lentis,+glaucoma,+spherophakia,+and+short+stature&amp;author=J+Morales&amp;author=L+Al-Sharif&amp;author=DS+Khalil&amp;author=JM+Shinwari&amp;author=P+Bavi&amp;volume=85&amp;issue=5&amp;publication_year=2009&amp;pages=558-568&amp;pmid=19836009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, Boehnke M, Abecasis GR, Willer CJ. LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>(18):2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;issue=18&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Ramdas WD, van Koolwijk LM, Ikram MK, Jansonius NM, de Jong PT, Bergen AA, Isaacs A, Amin N, Aulchenko YS, Wolfs RC, et al.  A genome-wide association study of optic disc parameters. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>(6):e1000978.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2883590/">PMC free article</a>]</span> [<a href="/pubmed/20548946" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+association+study+of+optic+disc+parameters&amp;author=WD+Ramdas&amp;author=LM+van+Koolwijk&amp;author=MK+Ikram&amp;author=NM+Jansonius&amp;author=PT+de+Jong&amp;volume=6&amp;issue=6&amp;publication_year=2010&amp;pages=e1000978&amp;pmid=20548946&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Ramdas WD, van Koolwijk LM, Lemij HG, Pasutto F, Cree AJ, Thorleifsson G, Janssen SF, Jacoline TB, Amin N, Rivadeneira F, et al.  Common genetic variants associated with open-angle glaucoma. <span><span class="ref-journal">Hum Mol Genet. </span>2011;<span class="ref-vol">20</span>(12):2464–2471.</span> [<a href="/pubmed/21427129" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Common+genetic+variants+associated+with+open-angle+glaucoma&amp;author=WD+Ramdas&amp;author=LM+van+Koolwijk&amp;author=HG+Lemij&amp;author=F+Pasutto&amp;author=AJ+Cree&amp;volume=20&amp;issue=12&amp;publication_year=2011&amp;pages=2464-2471&amp;pmid=21427129&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Sanfilippo PG, Hewitt AW, Hammond CJ, Mackey DA. The heritability of ocular traits. <span><span class="ref-journal">Surv Ophthalmol. </span>2010;<span class="ref-vol">55</span>(6):561–583.</span> [<a href="/pubmed/20851442" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Surv+Ophthalmol&amp;title=The+heritability+of+ocular+traits&amp;author=PG+Sanfilippo&amp;author=AW+Hewitt&amp;author=CJ+Hammond&amp;author=DA+Mackey&amp;volume=55&amp;issue=6&amp;publication_year=2010&amp;pages=561-583&amp;pmid=20851442&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Seipel K, O’Brien SP, Iannotti E, Medley QG, Streuli M. Tara, a novel F-actin binding protein, associates with the Trio guanine nucleotide exchange factor and regulates actin cytoskeletal organization. <span><span class="ref-journal">J Cell Sci. </span>2001;<span class="ref-vol">114</span>(Pt 2):389–399.</span> [<a href="/pubmed/11148140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Sci&amp;title=Tara,+a+novel+F-actin+binding+protein,+associates+with+the+Trio+guanine+nucleotide+exchange+factor+and+regulates+actin+cytoskeletal+organization&amp;author=K+Seipel&amp;author=SP+O’Brien&amp;author=E+Iannotti&amp;author=QG+Medley&amp;author=M+Streuli&amp;volume=114&amp;issue=Pt+2&amp;publication_year=2001&amp;pages=389-399&amp;pmid=11148140&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Springelkamp H, Hohn R, Mishra A, Hysi PG, Khor CC, Loomis SJ, Bailey JN, Gibson J, Thorleifsson G, Janssen SF, et al.  Meta-analysis of genome-wide association studies identifies novel loci that influence cupping and the glaucomatous process. <span><span class="ref-journal">Nat Commun. </span>2014;<span class="ref-vol">5</span>:4883.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4199103/">PMC free article</a>]</span> [<a href="/pubmed/25241763" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Meta-analysis+of+genome-wide+association+studies+identifies+novel+loci+that+influence+cupping+and+the+glaucomatous+process&amp;author=H+Springelkamp&amp;author=R+Hohn&amp;author=A+Mishra&amp;author=PG+Hysi&amp;author=CC+Khor&amp;volume=5&amp;publication_year=2014&amp;pages=4883&amp;pmid=25241763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">van Koolwijk LM, Despriet DD, van Duijn CM, Pardo Cortes LM, Vingerling JR, Aulchenko YS, Oostra BA, Klaver CC, Lemij HG. Genetic contributions to glaucoma: heritability of intraocular pressure, retinal nerve fiber layer thickness, and optic disc morphology. <span><span class="ref-journal">Invest Ophthalmol Vis Sci. </span>2007;<span class="ref-vol">48</span>(8):3669–3676.</span> [<a href="/pubmed/17652737" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest+Ophthalmol+Vis+Sci&amp;title=Genetic+contributions+to+glaucoma:+heritability+of+intraocular+pressure,+retinal+nerve+fiber+layer+thickness,+and+optic+disc+morphology&amp;author=LM+van+Koolwijk&amp;author=DD+Despriet&amp;author=CM+van+Duijn&amp;author=LM+Pardo+Cortes&amp;author=JR+Vingerling&amp;volume=48&amp;issue=8&amp;publication_year=2007&amp;pages=3669-3676&amp;pmid=17652737&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Wagner AH, Anand VN, Wang WH, Chatterton JE, Sun D, Shepard AR, Jacobson N, Pang IH, Deluca AP, Casavant TL, et al.  Exon-level expression profiling of ocular tissues. <span><span class="ref-journal">Exp Eye Res. </span>2013;<span class="ref-vol">111</span>:105–111.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3664108/">PMC free article</a>]</span> [<a href="/pubmed/23500522" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp+Eye+Res&amp;title=Exon-level+expression+profiling+of+ocular+tissues&amp;author=AH+Wagner&amp;author=VN+Anand&amp;author=WH+Wang&amp;author=JE+Chatterton&amp;author=D+Sun&amp;volume=111&amp;publication_year=2013&amp;pages=105-111&amp;pmid=23500522&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Wiggs JL, Yaspan BL, Hauser MA, Kang JH, Allingham RR, Olson LM, Abdrabou W, Fan BJ, Wang DY, Brodeur W, et al.  Common variants at 9p21 and 8q22 are associated with increased susceptibility to optic nerve degeneration in glaucoma. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>(4):e1002654.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3343074/">PMC free article</a>]</span> [<a href="/pubmed/22570617" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Common+variants+at+9p21+and+8q22+are+associated+with+increased+susceptibility+to+optic+nerve+degeneration+in+glaucoma&amp;author=JL+Wiggs&amp;author=BL+Yaspan&amp;author=MA+Hauser&amp;author=JH+Kang&amp;author=RR+Allingham&amp;volume=8&amp;issue=4&amp;publication_year=2012&amp;pages=e1002654&amp;pmid=22570617&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>(17):2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;issue=17&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Young TL, Hawthorne F, Feng S, Luo X, St Germain E, Wang M, Metlapally R. Whole genome expression profiling of normal human fetal and adult ocular tissues. <span><span class="ref-journal">Exp Eye Res. </span>2013;<span class="ref-vol">116</span>:265–278.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3875233/">PMC free article</a>]</span> [<a href="/pubmed/24016867" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp+Eye+Res&amp;title=Whole+genome+expression+profiling+of+normal+human+fetal+and+adult+ocular+tissues&amp;author=TL+Young&amp;author=F+Hawthorne&amp;author=S+Feng&amp;author=X+Luo&amp;author=E+St+Germain&amp;volume=116&amp;publication_year=2013&amp;pages=265-278&amp;pmid=24016867&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4480365&amp;issue-id=254567&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140153445368640title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Methods</a></li><li><a href="#S10title">Results</a></li><li><a href="#S15title">Discussion</a></li><li><a href="#S16title">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#APP1title">NEIGHBORHOOD Consortium membership list</a></li><li><a href="#idm140153443578512title">Footnotes</a></li><li><a href="#idm140153483258000title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4480365/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4480365/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4480365/pdf/nihms700469.pdf">PDF (776K)</a></li> | <li><a href="#" data-citationid="PMC4480365" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4480365%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4480365%2F&amp;text=Meta-analysis%20of%20Genome-Wide%20Association%20Studies%20Identifies%20Novel%20Loci%20Associated%20With%20Optic%20Disc%20Morphology" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4480365%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25631615" data-aiid="4480365" data-aid="4480365" data-iid="254567" data-domainid="319" data-domain="nihpa" data-accid="PMC4480365" data-md5="fc4056f47a792451dee80e6000650c1c"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:15:50-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C5394DD5764F10000000008160501&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4480365%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4480365/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>