<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4266741/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association analysis demonstrates the highly polygenic character of age-related hearing impairment</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4266741/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="Genome-wide association analysis demonstrates the highly polygenic character of age-related hearing impairment"><meta name="citation_authors" content="Erik Fransen, Sarah Bonneux, Jason J Corneveaux, Isabelle Schrauwen, Federica Di Berardino, Cory H White, Jeffrey D Ohmen, Paul Van de Heyning, Umberto Ambrosetti, Matthew J Huentelman, Guy Van Camp, Rick A Friedman"><meta name="citation_date" content="January 2015"><meta name="citation_issue" content="1"><meta name="citation_volume" content="23"><meta name="citation_firstpage" content="110"><meta name="citation_doi" content="10.1038/ejhg.2014.56"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4266741/?report=abstract"><meta name="citation_pmid" content="24939585"><meta name="DC.Title" content="Genome-wide association analysis demonstrates the highly polygenic character of age-related hearing impairment"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Erik Fransen"><meta name="DC.Contributor" content="Sarah Bonneux"><meta name="DC.Contributor" content="Jason J Corneveaux"><meta name="DC.Contributor" content="Isabelle Schrauwen"><meta name="DC.Contributor" content="Federica Di Berardino"><meta name="DC.Contributor" content="Cory H White"><meta name="DC.Contributor" content="Jeffrey D Ohmen"><meta name="DC.Contributor" content="Paul Van de Heyning"><meta name="DC.Contributor" content="Umberto Ambrosetti"><meta name="DC.Contributor" content="Matthew J Huentelman"><meta name="DC.Contributor" content="Guy Van Camp"><meta name="DC.Contributor" content="Rick A Friedman"><meta name="DC.Date" content="2015 Jan"><meta name="DC.Identifier" content="10.1038/ejhg.2014.56"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association analysis demonstrates the highly polygenic character of age-related hearing impairment"><meta property="og:type" content="article"><meta property="og:description" content="We performed a genome-wide association study (GWAS) to identify the genes responsible for age-related hearing impairment (ARHI), the most common form of hearing impairment in the elderly. Analysis of common variants, with and without adjustment for stratification ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4266741/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4266741/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4266741/pdf/ejhg201456a.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B7681DD56D5210000000005F2038F.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266741%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/246534/">v.23(1); 2015 Jan</a></li><li class="accid">PMC4266741</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2015 Jan; 23(1): 110–115. </span></div><div><span class="fm-vol-iss-date">Published online 2014 Jun 18. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fejhg.2014.56" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ejhg.2014.56</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4266741</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24939585">24939585</a></div></div></div></div><h1 class="content-title">Genome-wide association analysis demonstrates the highly polygenic character of age-related hearing impairment</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Fransen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160369994144" co-class="co-affbox">Erik Fransen</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Bonneux%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160367605792" co-class="co-affbox">Sarah Bonneux</a>,<sup>1</sup> <a href="/pubmed/?term=Corneveaux%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160367603808" co-class="co-affbox">Jason J Corneveaux</a>,<sup>3</sup> <a href="/pubmed/?term=Schrauwen%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160366080688" co-class="co-affbox">Isabelle Schrauwen</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Di%20Berardino%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160370000448" co-class="co-affbox">Federica Di Berardino</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=White%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160376528336" co-class="co-affbox">Cory H White</a>,<sup>6</sup> <a href="/pubmed/?term=Ohmen%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160376526352" co-class="co-affbox">Jeffrey D Ohmen</a>,<sup>6</sup> <a href="/pubmed/?term=Van%20de%20Heyning%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160368470064" co-class="co-affbox">Paul Van de Heyning</a>,<sup>7</sup> <a href="/pubmed/?term=Ambrosetti%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160368468080" co-class="co-affbox">Umberto Ambrosetti</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Huentelman%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160375603808" co-class="co-affbox">Matthew J Huentelman</a>,<sup>3</sup> <a href="/pubmed/?term=Van%20Camp%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160375601824" co-class="co-affbox">Guy Van Camp</a>,<sup>1,</sup><sup>*</sup> and  <a href="/pubmed/?term=Friedman%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585" class="affpopup" co-rid="_co_idm140160397254240" co-class="co-affbox">Rick A Friedman</a><sup>6,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140160369994144"><h3 class="no_margin">Erik Fransen</h3><p><sup>1</sup>Center for Medical Genetics, University of Antwerp, Antwerp, Belgium</p><p><sup>2</sup>StatUa Center for Statistics, University of Antwerp, Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Fransen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Erik Fransen</a></div></div><div id="_co_idm140160367605792"><h3 class="no_margin">Sarah Bonneux</h3><p><sup>1</sup>Center for Medical Genetics, University of Antwerp, Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Bonneux%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Sarah Bonneux</a></div></div><div id="_co_idm140160367603808"><h3 class="no_margin">Jason J Corneveaux</h3><p><sup>3</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, USA</p><div>Find articles by <a href="/pubmed/?term=Corneveaux%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Jason J Corneveaux</a></div></div><div id="_co_idm140160366080688"><h3 class="no_margin">Isabelle Schrauwen</h3><p><sup>1</sup>Center for Medical Genetics, University of Antwerp, Antwerp, Belgium</p><p><sup>2</sup>StatUa Center for Statistics, University of Antwerp, Antwerp, Belgium</p><p><sup>3</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, USA</p><div>Find articles by <a href="/pubmed/?term=Schrauwen%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Isabelle Schrauwen</a></div></div><div id="_co_idm140160370000448"><h3 class="no_margin">Federica Di Berardino</h3><p><sup>4</sup>Audiology Unit, Department of Clinical Sciences and Community Health, University of Milan, Milan, Italy</p><p><sup>5</sup>Fondazione IRCCS Ca' Granda Ospedale Maggiore Policlinico, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Di%20Berardino%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Federica Di Berardino</a></div></div><div id="_co_idm140160376528336"><h3 class="no_margin">Cory H White</h3><p><sup>6</sup>Cell Biology and Genetics Division, House Research Institute, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=White%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Cory H White</a></div></div><div id="_co_idm140160376526352"><h3 class="no_margin">Jeffrey D Ohmen</h3><p><sup>6</sup>Cell Biology and Genetics Division, House Research Institute, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Ohmen%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Jeffrey D Ohmen</a></div></div><div id="_co_idm140160368470064"><h3 class="no_margin">Paul Van de Heyning</h3><p><sup>7</sup>Department of Otolaryngology, University Hospital of Antwerp, Edegem, Belgium</p><div>Find articles by <a href="/pubmed/?term=Van%20de%20Heyning%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Paul Van de Heyning</a></div></div><div id="_co_idm140160368468080"><h3 class="no_margin">Umberto Ambrosetti</h3><p><sup>4</sup>Audiology Unit, Department of Clinical Sciences and Community Health, University of Milan, Milan, Italy</p><p><sup>5</sup>Fondazione IRCCS Ca' Granda Ospedale Maggiore Policlinico, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Ambrosetti%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Umberto Ambrosetti</a></div></div><div id="_co_idm140160375603808"><h3 class="no_margin">Matthew J Huentelman</h3><p><sup>3</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, USA</p><div>Find articles by <a href="/pubmed/?term=Huentelman%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Matthew J Huentelman</a></div></div><div id="_co_idm140160375601824"><h3 class="no_margin">Guy Van Camp</h3><p><sup>1</sup>Center for Medical Genetics, University of Antwerp, Antwerp, Belgium</p><div>Find articles by <a href="/pubmed/?term=Van%20Camp%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Guy Van Camp</a></div></div><div id="_co_idm140160397254240"><h3 class="no_margin">Rick A Friedman</h3><p><sup>6</sup>Cell Biology and Genetics Division, House Research Institute, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Friedman%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24939585">Rick A Friedman</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140160434209952_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140160434209952_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140160434209952_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140160434209952_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Center for Medical Genetics, University of Antwerp, Antwerp, Belgium</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>StatUa Center for Statistics, University of Antwerp, Antwerp, Belgium</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Neurogenomics Division, The Translational Genomics Research Institute, Phoenix, AZ, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Audiology Unit, Department of Clinical Sciences and Community Health, University of Milan, Milan, Italy</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Fondazione IRCCS Ca' Granda Ospedale Maggiore Policlinico, Milan, Italy</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Cell Biology and Genetics Division, House Research Institute, Los Angeles, CA, USA</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Department of Otolaryngology, University Hospital of Antwerp, Edegem, Belgium</div><div id="caf1"><sup>*</sup>Center for Medical Genetics, University of Antwerp, Universiteitsplein 1, 2610 Antwerp, Belgium. Tel: +32 3 275 9762; Fax: +32 3 275 9723; E-mail: <a href="mailto:dev@null" data-email="eb.neprewtnau@pmacnav.yug" class="oemail">eb.neprewtnau@pmacnav.yug</a></div><div id="caf2"><sup>*</sup>Keck School of Medicine, University of Southern California, 1520 San Pablo Street, Suite 4600, Los Angeles, CA 90033-9401, USA. Tel: +1 323 442 4827; E-mail: <a href="mailto:dev@null" data-email="ude.csu.dem@namdeirf.kcir" class="oemail">ude.csu.dem@namdeirf.kcir</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140160434209952_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2013 Jun 27; Revised 2014 Jan 17; Accepted 2014 Mar 5.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140160434209952_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2015 Macmillan Publishers Limited</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4266741/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="10" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure 1.</strong></div><div><a href="/pmc/articles/PMC4266741/bin/ejhg201456x1.pdf" data-ga-action="click_feat_suppl">ejhg201456x1.pdf</a><span style="color:gray"> (764K)</span></div><div class="small guid">GUID:&nbsp;85520690-0DFC-4CEF-A4D7-D448FC08D4D9</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure 2.</strong></div><div><a href="/pmc/articles/PMC4266741/bin/ejhg201456x2.tif" data-ga-action="click_feat_suppl">ejhg201456x2.tif</a><span style="color:gray"> (971K)</span></div><div class="small guid">GUID:&nbsp;8FB1C8C4-42AF-4CA6-BE88-251800EF54F5</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 1.</strong></div><div><a href="/pmc/articles/PMC4266741/bin/ejhg201456x3.pdf" data-ga-action="click_feat_suppl">ejhg201456x3.pdf</a><span style="color:gray"> (618K)</span></div><div class="small guid">GUID:&nbsp;3CC6BD5A-FC25-4C3B-8776-758BAF091445</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 2.</strong></div><div><a href="/pmc/articles/PMC4266741/bin/ejhg201456x4.pdf" data-ga-action="click_feat_suppl">ejhg201456x4.pdf</a><span style="color:gray"> (88K)</span></div><div class="small guid">GUID:&nbsp;6A326A69-BFF1-4B03-BA49-701387337AEE</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 3.</strong></div><div><a href="/pmc/articles/PMC4266741/bin/ejhg201456x5.pdf" data-ga-action="click_feat_suppl">ejhg201456x5.pdf</a><span style="color:gray"> (50K)</span></div><div class="small guid">GUID:&nbsp;F03A5D54-9B96-4DCC-BD24-57F4FB158600</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 4.</strong></div><div><a href="/pmc/articles/PMC4266741/bin/ejhg201456x6.pdf" data-ga-action="click_feat_suppl">ejhg201456x6.pdf</a><span style="color:gray"> (50K)</span></div><div class="small guid">GUID:&nbsp;BBE6E1C5-6E6D-421E-BA5A-4AED766719B4</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 5.</strong></div><div><a href="/pmc/articles/PMC4266741/bin/ejhg201456x7.pdf" data-ga-action="click_feat_suppl">ejhg201456x7.pdf</a><span style="color:gray"> (96K)</span></div><div class="small guid">GUID:&nbsp;9C20F60E-192A-455E-8303-E80631E91978</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure Legends.</strong></div><div><a href="/pmc/articles/PMC4266741/bin/ejhg201456x8.doc" data-ga-action="click_feat_suppl">ejhg201456x8.doc</a><span style="color:gray"> (35K)</span></div><div class="small guid">GUID:&nbsp;544F98D1-1213-4DF0-BEFE-ECD4A74DE9DB</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table Legends.</strong></div><div><a href="/pmc/articles/PMC4266741/bin/ejhg201456x9.doc" data-ga-action="click_feat_suppl">ejhg201456x9.doc</a><span style="color:gray"> (23K)</span></div><div class="small guid">GUID:&nbsp;2C862435-A6C9-4E10-B07D-9191D3A38D57</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Methods.</strong></div><div><a href="/pmc/articles/PMC4266741/bin/ejhg201456x10.doc" data-ga-action="click_feat_suppl">ejhg201456x10.doc</a><span style="color:gray"> (39K)</span></div><div class="small guid">GUID:&nbsp;AB132082-E1E9-4873-B1EC-B0B0C428AC75</div></div></dd></dl></div><div id="idm140160369841104" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140160369841104title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">We performed a genome-wide association study (GWAS) to identify the genes responsible for age-related hearing impairment (ARHI), the most common form of hearing impairment in the elderly. Analysis of common variants, with and without adjustment for stratification and environmental covariates, rare variants and interactions, as well as gene-set enrichment analysis, showed no variants with genome-wide significance. No evidence for replication of any previously reported genes was found. A study of the genetic architecture indicates for the first time that ARHI is highly polygenic in nature, with probably no major genes involved. The phenotype depends on the aggregated effect of a large number of SNPs, of which the individual effects are undetectable in a modestly powered GWAS. We estimated that 22% of the variance in our data set can be explained by the collective effect of all genotyped SNPs. A score analysis showed a modest enrichment in causative SNPs among the SNPs with a <em>P</em>-value below 0.01.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Age-related hearing loss is characterized by a symmetric sensorineural hearing loss that is most pronounced in the high frequencies. Age of onset, progression, and severity of age-related hearing impairment (ARHI) show great variation in the population, but with a demonstrable increased prevalence in males.<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723314">1</a></sup> Heritability studies have shown that the sources of this variance are both genetic and environmental, with approximately half of the variance attributable to hereditary factors.<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723302">2</a></sup></p><p id="__p3">Only a limited number of genome-wide association studies (GWASs) have been performed up to now. The first GWAS into ARHI reported an association between ARHI and SNPs within the <em>GRM7</em> gene. This gene encodes the metabotropic glutamate receptor type7, which is activated through L-glutamate, the primary excitatory neurotransmitter in the cochlear hair cell.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723324">3</a></sup> A GWAS in 352 samples from the Saami, an isolated population originating from northern Finland, revealed no genome-wide significant associations for ARHI, although the authors noted one SNP immediately downstream of the <em>GRM7</em> gene among the most significant association signals.<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723304">4</a></sup> In a study on normal hearing function in 3417 persons over age 18, Girotto <em>et al</em><sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723310">5</a></sup> performed a GWAS in 6 isolated European populations using a meta-analysis approach. No SNPs were identified with genome-wide significance, but one of the most significant SNPs was found in the <em>GRM8</em> gene, a close homolog of the <em>GRM7</em> gene that contained several significant SNPs in previous association studies.</p><p id="__p4">Large-scale studies on the genetic architecture of complex diseases and traits have indicated that these traits are usually polygenic in nature: their phenotypic variance is influenced by many genetic variants, each which of only contributes a very small fraction of the variance.</p><p id="__p5">Studies on the genetic architecture of adult height, BMI, and psychiatric diseases indicate that even SNPs that fail to reach genome-wide significance substantially contribute to the phenotypic variance. Although it is not possible to pinpoint these SNPs individually, their collective effect can be estimated, and they may be one of the factors involved in the paradox of the missing heritability.<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723325">6</a>, <a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723307">7</a>, <a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723330">8</a></sup></p><p id="__p6" class="p p-last">As part of our continuing effort to elucidate the genetic and environmental causes of ARHI, we have enlarged our sample set to 2161 individuals all originating from Antwerp. Here, we present the results of a GWAS on ARHI, with and without adjustment for environmental risk factors, compare our results with previous studies, and look for potential pathways using pathway analysis. Apart from the classic association tests on common variants, we also look for the effect of rare variants and the interactions between SNPs on ARHI. Furthermore, we discuss the implications for the genetic architecture of ARHI and compare this with the findings for other complex traits.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">MATERIALS AND METHODS</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Sample collection and phenotyping</h3><p id="__p7" class="p p-first-last">Subjects were collected at the audiological center from the Antwerp University Hospital using the selection criteria and questionnaire we described previously.<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723305">9</a></sup> Phenotypes were described using Principal component (PC) scores, as described in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary data</a>. In total, PC scores were obtained for 2161 individuals.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Sample selection for the three PC phenotypes</h3><p id="__p8" class="p p-first">Each PC score has an approximately normal distribution. Power calculations (not shown) showed that most power for the subsequent association analyses resides within the samples with a relatively large or small PC score. Specifically, the most informative samples consist of those within the 20% highest and 20% lowest PC scores. Selecting only these samples reduced the genotyping effort, with little effect on the total power.</p><p id="__p9" class="p p-last">To make a sample selection based upon the three PCs simultaneously, maintaining high power for associations tests on all three PCs, sample selection was carried out in two steps, as shown in <a href="/pmc/articles/PMC4266741/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>: First, we flagged the samples that were within the 20% highest or lowest percentile of the distribution for each of the three PCs separately. The final selection for genotyping was obtained by including the samples that were flagged for at least one of the three PCs. Hence, samples that were never flagged in any of the three PCs, representing samples that had low information content for any of the three PCs, were not selected.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC4266741/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140160366998464" class="figure"><a href="/pmc/articles/PMC4266741/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4266741_ejhg201456f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg201456f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4266741/bin/ejhg201456f1.jpg"></a></div><div id="largeobj_idm140160366998464" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4266741/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC4266741/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p55">Schematic representation of the sample selection. Starting from a total sample size of 2161, the most informative samples were selected for genotyping based upon principle component analysis. Samples with an extreme value for either one of the first three PCs were included for genotyping. After various quality control steps, 1489 samples were included in the association analyses.</p></div></div></div></div><div id="__sec5" class="sec"><h3 id="__sec5title">Genotyping</h3><p id="__p10" class="p p-first">DNA was extracted from blood using standard procedures.</p><p id="__p11" class="p p-last">Genomic DNA concentrations were determined with PicoGreen (Invitrogen, Carlsbad, CA, USA). Additionally, the quality of the genomic DNA was assessed for each sample by gel electrophoresis. Five hundred DNA samples were genotyped with the Illumina CNV370 quad chip, 1060 DNA samples using the Illumina HumanOmniExpress BeadChip (Illumina, Inc., San Diego, CA, USA).</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Imputation and filtering</h3><p id="__p12" class="p p-first">Pre-imputation filtering was carried out to exclude SNPs with a minor allele frequency (MAF) below 1%, a <em>P</em>-value for Hardy–Weinberg equilibrium below 10e−6, a call rate below 95% across all samples and across all SNPs. Duplicate samples were identified using pi-hat analysis (pi-hat value&gt;0.99). For each of these duplicates, we removed the sample with the lowest genotyping call rate.</p><p id="__p13" class="p p-last">Imputation was performed using the program impute2, version 2.1.2,<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723318">10</a></sup> with the 1000 Genomes Phase 1 Interim panel (June 2011) as a reference panel. Imputation resulted in a total of 11 626 570 SNPs. An additional post-imputation filtering was carried out using the same exclusion criteria as in the pre-imputation stage, and removing imputed SNPs with an Impute2 info metric below 0.5.</p></div><div id="__sec7" class="sec"><h3 id="__sec7title">Association analysis</h3><p id="__p14" class="p p-first">The association between the common variant genotypes (MAF&gt;0.01) and the scores of the first three PCs was analyzed using the linear regression option in the software package PLINK v1.07 (<a href="http://pngu.mgh.harvard.edu/purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/purcell/plink/</a>). An additive model was fitted taking the PC scores as a quantitative outcome variable. Significance of the association was tested through the Wald test. This association test was repeated using the Efficient Mixed-Model Association eXpedited (EMMAX) software (<a href="http://genetics.cs.ucla.edu/emmax/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genetics.cs.ucla.edu/emmax/</a>), to account for possible cryptic relatedness or population stratification. A cutoff for cryptic relatedness of 0.05 was used.</p><p id="__p15">The association test accounting for environmental risk factors was performed using a linear regression model as implemented in PLINK, with the following covariates: exposure to occupational noise (binary variable), smoking (in pack years), solvents (binary variable) and alcohol (binary variables), and the BMI. We refer to our previous paper on environmental risk factors for ARHI for a detailed description on how these risk factors were ascertained and coded.<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723306">11</a></sup></p><p id="__p16" class="p p-last">Details on the analysis of rare variants, gene–gene interactions and pathway analysis are given in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Methods</a>. In all analyses except the gene–gene interaction analysis, the hearing phenotypes were treated as a quantitative trait. Although the sample selection was performed based upon informativity for the three separate PC phenotypes (<a href="/pmc/articles/PMC4266741/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>), subsequent association testing, rare variant analyses, genome-wide complex trait analysis (GCTA), and pathway analyses were performed on all included samples (<em>N</em>=1489), regardless what PC phenotype they were selected for. This means that an individual that was highly informative for PC1 was also included in the analyses on PC2 and PC3 and vice versa.</p></div><div id="__sec8" class="sec"><h3 id="__sec8title">Genome-wide complex trait analysis</h3><p id="__p17" class="p p-first">The estimate of the genetic variance was performed using the Genome-wide Complex Trait Analysis tool.<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723322">12</a></sup> Details on the variance components analysis are given in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Methods</a>.</p><p id="__p18" class="p p-last">The score analysis to test for polygenic inheritance of ARHI was carried out as follows: first, we randomly partitioned the data set into 10 equal subsets. Nine of the subsets were combined to build a discovery set and one subset was retained as a validation set (Discovery set: 1326 individuals, validation set: 163 individuals). In the discovery set, the <em>P</em>-values for association with PC1 were recalculated using PLINK in the same way as the original analysis. We then estimated the random effects of the SNPs using the Best Linear Unbiased Prediction (BLUP) method, as implemented in GCTA. This method estimates the effects of the individual SNPs on the phenotype. Using these effects, a phenotypic score was obtained for the individuals in the validation set using the score function in PLINK. This step was carried out using various subsets of SNPs, using different <em>P</em>-value cutoffs. The correlation between this phenotypic score and the observed PC1 score in the validation set was calculated. As a negative control, the correlation between the phenotypic score and an independent phenotype (the PC2 score) was calculated. This procedure was repeated 10 times, whereby each of the 10 subsets served exactly once as a validation set. For each <em>P</em>-value cutoff, the mean and standard error of the 10 correlation coefficients was calculated.</p></div><div id="__sec9" class="sec sec-last"><h3 id="__sec9title">Public data access</h3><p id="__p19" class="p p-first-last">Data from this study are publicly available through the following link <a href="https://tgen.org/research/research-divisions/neurogenomics/supplementary-data/gwas_polygenic_arhi_fransen_et_al.aspx" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://tgen.org/research/research-divisions/neurogenomics/supplementary-data/gwas_polygenic_arhi_fransen_et_al.aspx</a>.</p></div></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">RESULTS</h2><div id="__sec11" class="sec sec-first"><h3 id="__sec11title">Sample selection and genotyping</h3><p id="__p20" class="p p-first">To study the genetic factors influencing the normal variation in ARHI, we collected a population-based sample of 2161 unrelated individuals from a residential area near Antwerp, Belgium. The sample was obtained through population registries made available by the local city councils, and not specifically enriched for hearing-impaired people. To make the population ethnically homogenous, we requested that at least three out of the four grandparents originated from the same region as the study subject. No inbreeding was present in this population, and if full sibs were present, then only one of them was included in the study.</p><p id="__p21" class="p p-last">PC analysis was performed on the pure-tone audiometric data, adjusting for age and gender. In line with previous studies,<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723332">4</a></sup> we found that the first three PCs contain most of the information in the audiogram, with the PC1 score giving an overall measure of hearing capabilities across all frequencies, PC2 indicating whether the audiogram is flat or sloping, and PC3 being an indicator of the convexity/concavity of the audiogram. As shown in <a href="/pmc/articles/PMC4266741/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>, the 1560 most informative samples were selected for genotyping, based upon the PC scores for the first 3 PCs. Genotyping was performed using either the Illumina CNV370 quad chip or the Illumina HumanOmniExpress BeadChip, and the results from these two platforms were integrated using the program Impute2. After various quality control steps, a total of 4 167 292 markers remained for the association test in 1489 samples.</p></div><div id="__sec12" class="sec"><h3 id="__sec12title">Common variant analysis</h3><p id="__p22" class="p p-first">All SNPs that passed quality control were tested for association with the three first PCs. A list with all SNPs with a <em>P</em>-value below 1.0e−3 is given in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 1</a>. <a href="/pmc/articles/PMC4266741/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a> shows a Manhattan plot, with the most significant genes highlighted in red. None of the markers included in our analysis reached the threshold for genome-wide significance. The most significant <em>P</em>-values were found for the association between the PC2 scores and several SNPs within the <em>ACVR1B</em> gene (minimum <em>P</em>-value=4.6e−7), and between the PC3 scores and an SNP in the <em>CCBE1</em> gene (<em>P</em>=3.5e−7). For PC1, the most significant SNPs were an order of magnitude less significant, with a minimal <em>P</em>-value of 7.4e−6 for an SNP within the <em>ZNF536</em> gene. None of the top-ranked SNPs in our study were statistically significant in any of the previous GWASs on ARHI.<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723320">4</a>, <a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723312">5</a></sup></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC4266741/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140160373655968" class="figure"><a href="/pmc/articles/PMC4266741/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4266741_ejhg201456f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg201456f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4266741/bin/ejhg201456f2.jpg"></a></div><div id="largeobj_idm140160373655968" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4266741/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC4266741/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p56">Manhattan plot for the common variant analysis. The figure shows the logarithm of the <em>P</em>-values for the association of the genotypes <em>versus</em> each of the three PC phenotypes. <em>P</em>-values above 0.01 are not shown. Red dots indicate the SNPs in the genes that show the strongest association in the current study. Blue dots highlight the genes with a previously reported association with ARHI by Girotto <em>et al</em>,<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723326">5</a></sup> plus the <em>GRM7</em> gene reported by Friedman <em>et al</em><sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723328">3</a></sup> and the <em>IQGAP2</em> gene reported by Van Laer <em>et al.</em><sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723319">4</a></sup></p></div></div></div><p id="__p23">To look for evidence of replication of previous studies, we first looked up the <em>P</em>-values for all individual SNPs that were reported to be significant in the previous GWASs and checked their significance in the current study. For the GWAS by Girotto <em>et al</em>,<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723309">5</a></sup> this included the SNPs having a <em>P</em>-value below 10e−5. For the GWAS on the Saami, we checked all SNPs with a <em>P</em>-value below 1.0e−4. For the study by Friedman <em>et al</em>,<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723323">3</a></sup> we selected the SNPs that were significant in the <em>GRM7</em> gene and compared these with our PC1 results. The results are shown in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 2</a>. In PC2, the current study found marginally significant associations for SNPs in the <em>GPATCH3</em> and <em>OTX2</em> genes. No significant <em>P</em>-values were found among the SNPs previously found significant in PC1 and PC3.</p><p id="__p24" class="p p-last">In <a href="/pmc/articles/PMC4266741/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a>, blue dots highlight the SNPs in genes previously associated with ARHI, and reaching a <em>P</em>-value below 0.01 in the current study. The most remarkable gene is <em>GRM8</em>, with several SNPs reaching a <em>P</em>-value near 1e−4 for association with the PC1 score. None of the SNPs in the previously reported <em>GRM7</em> gene showed a <em>P</em>-value below 0.001.</p></div><div id="__sec13" class="sec"><h3 id="__sec13title">Common variants accounting for environmental risk factors</h3><p id="__p25" class="p p-first-last">We and others previously showed that ARHI is significantly associated with several environmental risk factors, including occupational noise, smoking, high BMI, solvents, and alcohol.<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723317">11</a></sup> A detailed description on how these risk factors were ascertained can be found in this latter paper. We repeated the association test on the common variants taking into account exposure to the most significant environmental risk factors. The most significant associations for this analysis are graphically shown in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 1</a>. The ranking of the SNPs and genes differed compared with the analysis without covariate adjustment, but genome-wide significance was not reached (data not shown).</p></div><div id="__sec14" class="sec"><h3 id="__sec14title">Common variants accounting for population stratification</h3><p id="__p26" class="p p-first-last">To exclude an effect of cryptic relatedness between individuals or hidden population stratification on the test statistics, we compared the results from the common variant analysis with an association analysis using the software package EMMAX. This program implements a variance-component approach that accounts for empirically estimated relationships between the individuals when testing for association. As shown in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 1</a>, the differences between the uncorrected association tests using PLINK and the corrected test in EMMAX are very small. The same genes and SNPs are found and show the strongest association, albeit with a slightly lower <em>P</em>-value in the adjusted analysis. The correlation between the <em>P</em>-values from the two methods equals 0.967, 0.999, and 0.994 for PC1, PC2, and PC3, respectively. <em>λ</em><sub>GC</sub> values for the uncorrected association analysis were 1.000, 0.995, and 1.002 for the first three PCs, whereas for the EMMAX analysis these values were 1.014, 0.997, and 1.001, respectively, confirming there was little effect of population stratification on the association results.</p></div><div id="__sec15" class="sec"><h3 id="__sec15title">Rare variant analysis</h3><p id="__p27" class="p p-first-last">Classic statistical tests offer low power to detect the effect of SNPs with low allele frequencies (MAF&lt;0.05). To search for the effect of the rare variants on the phenotype, we used the Sequence Kernel Association Tests (SKAT).<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723329">13</a></sup> This test searches for the joint effects of multiple variants in a gene or a pre-defined region of the genome on a phenotype, without assuming that all rare variants have the same effect on the phenotype. We partitioned the genome into 13 000 regions according to the gene annotation, and performed the SKAT for every gene separately. <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 3</a> lists the <em>P</em>-values obtained for each gene separately, for each of the three PCs. None of the genes reached the threshold for genome-wide significance (<em>P</em>=3.8e−6). The lowest observed <em>P</em>-value was 4.7e−5 for PC2.</p></div><div id="__sec16" class="sec"><h3 id="__sec16title">Gene–gene interaction analysis</h3><p id="__p28" class="p p-first-last">To test for the interaction between SNPs in different genes, we carried out the Gametic Phase Disequilibrium test as implemented in the software package SIXPAC.<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723331">14</a></sup> This analysis first filters candidate SNP pairs, by searching for strong LD between distant, physically unlinked SNPs in the cases alone. The shortlist of SNP pairs showing the strongest LD in cases is then followed up in the second step, which verifies whether the strength of the LD differs between cases and controls. This strategy identifies pairs of alleles from distant SNPs co-occurring more often in cases compared with controls, which could be indicative of gene–gene interactions. Since the search for high LD is not exhaustive, and the algorithm is non-deterministic, the program was run multiple times. <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 4</a> shows the most significant across all runs. As 629 437 SNPs were included into this analysis, the number of tests equals (629 437<sup>2</sup>–629 437)/2, times four genetic models per SNP pair. This leads to a Bonferroni-corrected significance threshold of 6.3e−14. None of the SNP pairs reached this threshold in our data set. The most significant interaction, that shows up in several of the SIXPAC runs, reaches a <em>P</em>-value of 4.2e−11 for an interaction between SNP rs877674 and SNP rs6952893. This latter gene lies within an intron of the <em>DYNC1I1</em> gene on Chr7, while the former is situated between the <em>BAI2</em> and <em>SPOCD1</em> genes on Chr1.</p></div><div id="__sec17" class="sec"><h3 id="__sec17title">Genome-wide complex trait analysis</h3><p id="__p29" class="p p-first">To study the genetic architecture of ARHI, we have analyzed the GWAS results using the Genome-wide Complex Analysis Tool (GCTA).<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723303">12</a></sup> Variance component analysis estimated the percentage of variance explained by the SNPs, obtained as the ratio of the genetic variance to the total variance at 22%. This analysis adjusts for incomplete LD between the SNPs in the GWAS and the causative SNPs, but assumes that the MAF between causative SNPs and GWAS SNPs is the same. We repeated the analysis with an adjustment that the causal SNPs could have a lower MAF compared with the GWAS SNPs, but the estimate of the variance explained by the SNPs remained almost the same (21%). Although this result does not formally prove that the causal SNPs have the same MAF spectrum as the GWAS SNPs, there is no evidence for the causal SNPs to be enriched in SNPs with lower allele frequency. Due to the relatively small sample size, the standard error of the estimate was large, and a more accurate estimate can be obtained using a larger sample size.</p><p id="__p30">The association results do not seem to indicate the presence of a major <em>ARHI</em> gene. To analyze whether the model of polygenic inheritance, which was reported for several other complex traits, also applies to ARHI, we performed a score analysis. First, we randomly split the individuals from our data set into 10 equal parts. Nine of the ten parts formed the discovery set and one part was retained as a validation set. Second, we estimated the effects of all individual SNPs in the discovery set. Third, using these estimated SNP effects, we predicted a phenotype score for the individuals from the validation set. This latter step was carried out using several subsets of SNPs, based upon different <em>P</em>-value cutoffs. Finally, the Pearson's correlation between this phenotypic score and the observed phenotype in the validation set is calculated. Ten-fold cross-validation was performed to obtain standard errors of the estimated correlation coefficients.</p><p id="__p31" class="p p-last"><a href="/pmc/articles/PMC4266741/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure 3</span></a> shows the mean correlation coefficients and their standard errors for various <em>P</em>-value cutoffs. When only the most significant fraction of the SNPs is used to calculate the phenotypic score (<em>P</em>&lt;0.001), the correlation between the phenotypic score and the observed phenotype only reached a value of 0.07. However, when SNPs are included using more liberal <em>P</em>-value cutoffs, correlation coefficients increase to ∼0.10 (<em>P</em>&lt;0.01). Cutoff values beyond 0.01 do not lead to a further increase in correlation, suggesting that the group of SNPs with <em>P</em>&lt;0.01 harbors the majority of the causal SNPs. To test whether the effect of the SNPs is specific, we have also calculated the correlation between the phenotypic score and an independent phenotype (PC2). Across all <em>P</em>-value cutoffs, the correlation coefficients from the score analysis were significantly higher than the correlation coefficients with the independent phenotype (not shown).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC4266741/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140160368627248" class="figure"><a href="/pmc/articles/PMC4266741/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4266741_ejhg201456f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg201456f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4266741/bin/ejhg201456f3.jpg"></a></div><div id="largeobj_idm140160368627248" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4266741/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC4266741/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p57">Score analysis. Pearson correlation coefficients between the observed phenotype (PC1) and the predicted phenotype from the individuals in the validation set are shown in black. Correlation coefficients between these predicted phenotypes and an independent phenotype (PC2), which served as a negative control, are shown in red. Error bars denote the standard errors obtained by 10-fold cross-validation.</p></div></div></div></div><div id="__sec18" class="sec sec-last"><h3 id="__sec18title">Pathway analysis</h3><p id="__p32" class="p p-first-last">To investigate whether genes in specific biological processes or pathways were enriched in SNPs with low <em>P</em>-values, gene-set enrichment analysis was carried out using the MAGENTA package. <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 5</a> shows the result of the gene-set enrichment analysis for the three PC scores, as well as for the analysis adjusting for environmental risk factors. Upon FDR correction, only two gene sets show a significant enrichment: The arachidonic acid secretion pathway (PC1, 95th percentile cutoff) and the transforming growth factor beta (TGF<em>β</em> signaling pathway (PC3, 75th percentile cutoff). The Janus Kinase/Signal Transducer and Activator of Transcription (JAK/STAT) signaling pathway is significantly enriched in low <em>P</em>-values in the analysis of PC2 adjusted for environmental risk factors.</p></div></div><div id="__sec19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec19title">DISCUSSION</h2><p id="__p33" class="p p-first">Traditionally, GWASs concentrate on the significance of common variants in a particular disease. Power calculations using the program QUANTO<sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup> showed that the current study design holds 80% power to reach genome-wide significance for SNPs accounting for 2% of the phenotypic variance of the trait. Doubling the sample size would allow the detection of SNPs accounting for 1% of the variance. The inability of this and previous studies to find consistently replicated genes associated with ARHI reflects the fact that there are no major genes involved in ARHI. The most significant SNPs we obtained in our GWAS are probably enriched with causative SNPs, but at the moment these SNPs cannot be individually identified.</p><p id="__p34">An estimate of the collective effect of all SNPs in our data set has shown that 22% of the variance can be explained by genetic differences. This is still much less than the heritability estimates based upon epidemiological studies, which typically reach a heritability of 40–50%. Our estimate covers the combined effects of the common SNPs, including the non-significant ones (see further), gene–gene interactions and rare variants, but does not cover the influence of CNVs, epigenetics or a possible overestimation of the heritability in the epidemiological studies. These latter three reasons are possible explanations for the observed heritability gap. Alternatively, the 22% may be an underestimate if the MAF spectrum of the causative SNPs is strongly shifted toward SNPs with a low MAF.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723327">7</a></sup> We have repeated the estimate of the variance explained accounting for this possibility, but obtained an estimate very close to the initial analysis. Our estimates have a large standard error, and larger sample sizes will lead to a more accurate estimate.</p><p id="__p35">This is the fourth genome-wide association analysis into ARHI. Apart from the analysis of common variants, we have studied the effect of rare variants, gene–gene interactions, and gene-set enrichment analysis. All previous studies reported significant or near-significant associations between the phenotype and SNPs in genes encoding GRM7 and GRM8. A recent study by Newman <em>et al</em><sup>16</sup> showed association of several audiometric traits with <em>GRM7</em> haplotypes, although the initial findings could not be replicated. Others and we have suggested that these genes be the prime candidates for association with ARHI. The current study also finds several SNPs within these two genes with suggestive <em>P</em>-values, but lacking genome-wide significance. To find out whether our current results support a role for GRM7 and GRM8 in ARHI, we studied the relationship between the previously reported SNPs and the most significant <em>GRM7</em> and <em>GRM8</em> SNPs in the current study. For both genes, the SNPs with low <em>P</em>-values in the current study are located in a different region of the gene compared with the previously reported SNPs, and linkage disequilibrium with these latter SNPs is very weak (not shown). Conversely, none of the previously reported associated SNPs shows any trend toward association in the current study. Hence, for both genes the current and the previously reported association signals appear to be completely independent, and no region of the <em>GRM7</em> or <em>GRM8</em> gene was ever found to be associated in more than one study.</p><p id="__p36">The genes encoding GRM7 and GRM8 contained 1285 and 1509 SNPs, respectively, and both genes span over 800 kB. Even though the SNPs within these two genes are not completely independent due to linkage disequilibrium, association testing of so many SNPs leads to an inflated type 1 error: due to testing multiple hypotheses, it becomes very likely to observe highly significant <em>P</em>-values even in the absence of any true association. As long as no adjustment is made for the number of SNPs tested within a gene, <em>GRM7</em> and <em>GRM8</em> are very likely to contain significant SNPs in any GWAS due to the sheer size of these genes. In <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 2</a>, we have plotted the observed distribution of <em>P</em>-values for the <em>GRM7</em> and <em>GRM8</em> SNPs <em>versus</em> the expected distribution in the absence of any association. There seems to be no evidence that the SNPs in these genes are enriched in low <em>P</em>-values. The current study shows one <em>GRM7</em> SNP and several <em>GRM8</em> SNPs with a significant <em>P</em>-value, and it cannot be excluded that one or more of the signals in these genes represent genuine associations. Conversely, our results do not mean that all previous associations in <em>GRM7</em> or <em>GRM8</em> represent false positives. All gene regions and SNPs that were reported previously can still be confirmed by replication, but the current result does not add support to any of the previously reported associations in <em>GRM7</em> or <em>GRM8</em>.</p><p id="__p37">The current data set is powered to detect large effects and no SNPs reached genome-wide significance. Unlike Crohn's disease or AMD, there is no indication for the presence of any major genes, not even among the genes involved in monogenic hearing loss. Among the post-GWAS analyses, only the pathway analysis showed three hits that remained significant after correction for multiple testing. We found a significant involvement of the arachidonic acid secretion pathway, the JAK/STAT and TGF<em>β</em> signaling pathways. Arachidonic acid metabolites are found in virtually all cells and tissues, and are involved in the modulation of cell function. In the inflammation cascade, arachidonic acid is the precursor of prostaglandins (PGs) and leukotrienes (LTs). One possible link with hearing loss is the suggestion that salicylate-induced ototoxicity can be mediated through altered levels of PGs and LTs in the perilymph.<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723311">17</a>, <a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723315">18</a></sup> The TGF<em>β</em> signaling pathway is involved in a wide range of cellular processes including cell growth, differentiation, apoptosis, and cellular homeostasis, in both developing and adult organisms. In the developing mouse embryo, TGF<em>β</em> proteins are expressed in the cochlear epithelium of the inner ear.<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723321">19</a></sup> In the adult inner ear, TGF<em>β</em> is activated during an immune response.<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723308">20</a></sup> The JAK/STAT signaling pathway is a major signal transduction pathway. Disruptions of this pathway have been implicated in immune deficiencies and cancers.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723316">18</a></sup> No link to inner ear pathology or hearing loss was found.</p><p id="__p38">At the moment, it is too early to draw firm conclusions on these latter findings. This study should be regarded as a first, exploratory step into the discovery of the responsible genes and pathways. Combination of these data with other GWAS, and the integration with results from expression studies or resequencing efforts can eventually lead to better insights into the pathophysiology of ARHI.</p><p id="__p39">Several recent papers have evaluated the genetic architecture of complex traits. One of their main findings is that many complex traits are highly polygenic in nature, sometimes involving one major gene but often without any variants with a substantial effect size. The genetic variance is not solely attributable to the SNPs reaching genome-wide significance, but is also due to the small effects of many causal alleles that never reach genome-wide significance. The collective effect of these undetectable SNPs can be evaluated using the SNPs with weakly significant <em>P</em>-values as predictors for the phenotype in an independent data set, and estimating how strongly the predicted phenotype correlates with the observed phenotype. Here, we have performed such a score analysis on the ARHI data, and show for the first time that the genetic architecture of ARHI is similar to other polygenic complex traits: including less-significant SNPs considerably improves the prediction of the phenotype in an independent data set. The number of causative SNPs was enriched when considering the set of SNPs with a <em>P</em>-value below 0.01.</p><p id="__p40">These findings are important in view of the discussion on the missing heritability. One of the explanations for the discrepancy between the heritability observed in epidemiological studies, and the combined variance explained by the associated SNPs, is that part of the missing heritability is contained within the SNPs that do not reach genome-wide significance. For human height, Yang <em>et al</em><sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_417723313">7</a></sup> showed that these SNPs might completely explain the missing heritability. Our study indicates that for ARHI, this is at least partly the case. The signal contained in the weakly significant SNPs (<em>P</em>&lt;0.01) is weak but significant, and larger than the information contained in the SNPs with a more stringent <em>P</em>-value (<em>P</em>&lt;0.001). More accurate estimates will be obtained if larger sample sizes become available. As the sample size increases, the subset of SNPs with low <em>P</em>-values gradually becomes more enriched with truly causative SNPs, and the collective effect of these SNPs on the phenotype will therefore increase.</p><p id="__p41" class="p p-last">An estimate of the collective effect of all SNPs in our data set showed that 22% of the variance explained in the phenotype are due to genetic factors. The most significant SNPs we obtained in our GWAS are probably enriched with causative SNPs, but at the moment these SNPs cannot be individually identified. Increasing the sample size will likely result in several replicated SNPs, but the majority of causative SNPs are probably not detectable using GWAS.</p></div><div id="idm140160361972944" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140160361972944title">Acknowledgments</h2><div class="sec"><p id="__p42">SB holds a doctoral fellowship of the FWO Vlaanderen. IS is a postdoctoral fellow of the FWO Vlaanderen. We wish to thank Jian Yang and Peter M Visscher (Queensland Institute of Medical Research) for help with the analysis using the GCTA software. This research was supported by funding from the Belgian Science Policy Office Interuniversity Attraction Poles (BELSPO-IAP) programme through the project IAP P7/43-BeMGI, through R01 grant DC010215, the Seaver Foundation, the Schwartz Foundation, and the State of Arizona.</p></div></div><div id="idm140160361970176" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140160361970176title">Notes</h2><p id="__p44">The authors declare no conflict of interest.</p></div><div id="idm140160361971776" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140160361971776title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140160361971648"></p><p id="__p43" class="p p-first-last"><a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> accompanies this paper on European Journal of Human Genetics website (http://www.nature.com/ejhg)</p><p></p></div></div><div id="sup1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sup1title">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="xob1"><h4>Supplementary Figure 1</h4><div class="sup-box half_rhythm" id="idm140160375875248"><a href="/pmc/articles/PMC4266741/bin/ejhg201456x1.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(764K, pdf)</sup></div></div><div class="sec suppmat" id="xob2"><h4>Supplementary Figure 2</h4><div class="sup-box half_rhythm" id="idm140160377612208"><a href="/pmc/articles/PMC4266741/bin/ejhg201456x2.tif" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(971K, tif)</sup></div></div><div class="sec suppmat" id="xob3"><h4>Supplementary Table 1</h4><div class="sup-box half_rhythm" id="idm140160362999168"><a href="/pmc/articles/PMC4266741/bin/ejhg201456x3.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(618K, pdf)</sup></div></div><div class="sec suppmat" id="xob4"><h4>Supplementary Table 2</h4><div class="sup-box half_rhythm" id="idm140160362996352"><a href="/pmc/articles/PMC4266741/bin/ejhg201456x4.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(88K, pdf)</sup></div></div><div class="sec suppmat" id="xob5"><h4>Supplementary Table 3</h4><div class="sup-box half_rhythm" id="idm140160369039360"><a href="/pmc/articles/PMC4266741/bin/ejhg201456x5.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(50K, pdf)</sup></div></div><div class="sec suppmat" id="xob6"><h4>Supplementary Table 4</h4><div class="sup-box half_rhythm" id="idm140160368924416"><a href="/pmc/articles/PMC4266741/bin/ejhg201456x6.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(50K, pdf)</sup></div></div><div class="sec suppmat" id="xob7"><h4>Supplementary Table 5</h4><div class="sup-box half_rhythm" id="idm140160364799600"><a href="/pmc/articles/PMC4266741/bin/ejhg201456x7.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(96K, pdf)</sup></div></div><div class="sec suppmat" id="xob8"><h4>Supplementary Figure Legends</h4><div class="sup-box half_rhythm" id="idm140160364796784"><a href="/pmc/articles/PMC4266741/bin/ejhg201456x8.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(35K, doc)</sup></div></div><div class="sec suppmat" id="xob9"><h4>Supplementary Table Legends</h4><div class="sup-box half_rhythm" id="idm140160360990224"><a href="/pmc/articles/PMC4266741/bin/ejhg201456x9.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(23K, doc)</sup></div></div><div class="sec suppmat" id="xob10"><h4>Supplementary Methods</h4><div class="sup-box half_rhythm" id="idm140160372641472"><a href="/pmc/articles/PMC4266741/bin/ejhg201456x10.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(39K, doc)</sup></div></div></div><div id="idm140160373450544" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140160373450544title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="bib1"><span class="mixed-citation">Gates GA, Mills JH. Presbycusis. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">366</span>:1111–1120.</span> [<a href="/pubmed/16182900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Presbycusis&amp;author=GA+Gates&amp;author=JH+Mills&amp;volume=366&amp;publication_year=2005&amp;pages=1111-1120&amp;pmid=16182900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib2"><span class="mixed-citation">Huang Q, Tang J. Age-related hearing loss or presbycusis. <span><span class="ref-journal">Eur Arch Otorhinolaryngol. </span>2010;<span class="ref-vol">267</span>:1179–1191.</span> [<a href="/pubmed/20464410" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+Arch+Otorhinolaryngol&amp;title=Age-related+hearing+loss+or+presbycusis&amp;author=Q+Huang&amp;author=J+Tang&amp;volume=267&amp;publication_year=2010&amp;pages=1179-1191&amp;pmid=20464410&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib3"><span class="mixed-citation">Friedman RA, Van Laer L, Huentelman MJ, et al.  GRM7 variants confer susceptibility to age-related hearing impairment. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:785–796.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2638831/">PMC free article</a>]</span> [<a href="/pubmed/19047183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=GRM7+variants+confer+susceptibility+to+age-related+hearing+impairment&amp;author=RA+Friedman&amp;author=L+Van+Laer&amp;author=MJ+Huentelman&amp;volume=18&amp;publication_year=2009&amp;pages=785-796&amp;pmid=19047183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib4"><span class="mixed-citation">Van Laer L, Huyghe JR, Hannula S, et al.  A genome-wide association study for age-related hearing impairment in the Saami. <span><span class="ref-journal">Eur J Hum Genet. </span>2010;<span class="ref-vol">18</span>:685–693.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2987344/">PMC free article</a>]</span> [<a href="/pubmed/20068591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=A+genome-wide+association+study+for+age-related+hearing+impairment+in+the+Saami&amp;author=L+Van+Laer&amp;author=JR+Huyghe&amp;author=S+Hannula&amp;volume=18&amp;publication_year=2010&amp;pages=685-693&amp;pmid=20068591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib5"><span class="mixed-citation">Girotto G, Pirastu N, Sorice R, et al.  Hearing function and thresholds: a genome-wide association study in European isolated populations identifies new loci and pathways. <span><span class="ref-journal">J Med Genet. </span>2011;<span class="ref-vol">48</span>:369–374.</span> [<a href="/pubmed/21493956" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Hearing+function+and+thresholds:+a+genome-wide+association+study+in+European+isolated+populations+identifies+new+loci+and+pathways&amp;author=G+Girotto&amp;author=N+Pirastu&amp;author=R+Sorice&amp;volume=48&amp;publication_year=2011&amp;pages=369-374&amp;pmid=21493956&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib6"><span class="mixed-citation">Purcell SM, Wray NR, Stone JL, et al.  Common polygenic variation contributes to risk of schizophrenia and bipolar disorder. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:748–752.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912837/">PMC free article</a>]</span> [<a href="/pubmed/19571811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+polygenic+variation+contributes+to+risk+of+schizophrenia+and+bipolar+disorder&amp;author=SM+Purcell&amp;author=NR+Wray&amp;author=JL+Stone&amp;volume=460&amp;publication_year=2009&amp;pages=748-752&amp;pmid=19571811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib7"><span class="mixed-citation">Yang J, Benyamin B, McEvoy BP, et al.  Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:565–U131.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J+Yang&amp;author=B+Benyamin&amp;author=BP+McEvoy&amp;volume=42&amp;publication_year=2010&amp;pages=565-U131&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib8"><span class="mixed-citation">Lango Allen H, Estrada K, Lettre G, et al.  Hundreds of variants clustered in genomic loci and biological pathways affect human height. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:832–838.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955183/">PMC free article</a>]</span> [<a href="/pubmed/20881960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Hundreds+of+variants+clustered+in+genomic+loci+and+biological+pathways+affect+human+height&amp;author=H+Lango+Allen&amp;author=K+Estrada&amp;author=G+Lettre&amp;volume=467&amp;publication_year=2010&amp;pages=832-838&amp;pmid=20881960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib9"><span class="mixed-citation">Van Laer L, Van Eyken E, Fransen E, et al.  The grainyhead like 2 gene (GRHL2), alias TFCP2L3, is associated with age-related hearing impairment. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:159–169.</span> [<a href="/pubmed/17921507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=The+grainyhead+like+2+gene+(GRHL2),+alias+TFCP2L3,+is+associated+with+age-related+hearing+impairment&amp;author=L+Van+Laer&amp;author=E+Van+Eyken&amp;author=E+Fransen&amp;volume=17&amp;publication_year=2008&amp;pages=159-169&amp;pmid=17921507&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib10"><span class="mixed-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib11"><span class="mixed-citation">Fransen E, Topsakal V, Hendrickx J-J, et al.  Occupational noise, smoking, and a high body mass index are risk factors for age-related hearing impairment and moderate alcohol consumption is protective: a European population-based multicenter study. <span><span class="ref-journal">J Assoc Res Otolaryngol. </span>2008;<span class="ref-vol">9</span>:264–276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2492985/">PMC free article</a>]</span> [<a href="/pubmed/18543032" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Assoc+Res+Otolaryngol&amp;title=Occupational+noise,+smoking,+and+a+high+body+mass+index+are+risk+factors+for+age-related+hearing+impairment+and+moderate+alcohol+consumption+is+protective:+a+European+population-based+multicenter+study&amp;author=E+Fransen&amp;author=V+Topsakal&amp;author=J-J+Hendrickx&amp;volume=9&amp;publication_year=2008&amp;pages=264-276&amp;pmid=18543032&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib12"><span class="mixed-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib13"><span class="mixed-citation">Wu MC, Lee S, Cai T, Li Y, Boehnke M, Lin X. Rare-variant association testing for sequencing data with the Sequence Kernel Association Test. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">89</span>:82–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3135811/">PMC free article</a>]</span> [<a href="/pubmed/21737059" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Rare-variant+association+testing+for+sequencing+data+with+the+Sequence+Kernel+Association+Test&amp;author=MC+Wu&amp;author=S+Lee&amp;author=T+Cai&amp;author=Y+Li&amp;author=M+Boehnke&amp;volume=89&amp;publication_year=2011&amp;pages=82-93&amp;pmid=21737059&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib14"><span class="mixed-citation">Prabhu S, Pe'er I. Ultrafast genome-wide scan for SNP-SNP interactions in common complex disease. <span><span class="ref-journal">Genome Res. </span>2012;<span class="ref-vol">22</span>:2230–2240.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3483552/">PMC free article</a>]</span> [<a href="/pubmed/22767386" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Ultrafast+genome-wide+scan+for+SNP-SNP+interactions+in+common+complex+disease&amp;author=S+Prabhu&amp;author=I+Pe'er&amp;volume=22&amp;publication_year=2012&amp;pages=2230-2240&amp;pmid=22767386&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib15"><span class="mixed-citation">Gauderman WJ, Morrison JM.QUANTO 1.1: A computer program for power and sample size calculations for genetic-epidemiology studies
<a href="http://hydrauscedu/gxe" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://hydrauscedu/gxe</a>
,
2006</span></li><li id="bib16"><span class="mixed-citation">Newman DL, Fisher LM, Ohmen J, et al.  GRM7 variants associated with age-related hearing loss based on auditory perception. <span><span class="ref-journal">Hear Res. </span>2012;<span class="ref-vol">294</span>:125–132.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3705704/">PMC free article</a>]</span> [<a href="/pubmed/23102807" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hear+Res&amp;title=GRM7+variants+associated+with+age-related+hearing+loss+based+on+auditory+perception&amp;author=DL+Newman&amp;author=LM+Fisher&amp;author=J+Ohmen&amp;volume=294&amp;publication_year=2012&amp;pages=125-132&amp;pmid=23102807&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib17"><span class="mixed-citation">Jung TT, Park YM, Miller SK, Rozehnal S, Woo HY, Baer W. Effect of exogenous arachidonic acid metabolites applied on round window membrane on hearing and their levels in the perilymph. <span><span class="ref-journal">Acta Otolaryngol Suppl. </span>1992;<span class="ref-vol">493</span>:171–176.</span> [<a href="/pubmed/1636419" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Otolaryngol+Suppl&amp;title=Effect+of+exogenous+arachidonic+acid+metabolites+applied+on+round+window+membrane+on+hearing+and+their+levels+in+the+perilymph&amp;author=TT+Jung&amp;author=YM+Park&amp;author=SK+Miller&amp;author=S+Rozehnal&amp;author=HY+Woo&amp;volume=493&amp;publication_year=1992&amp;pages=171-176&amp;pmid=1636419&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib18"><span class="mixed-citation">Aaronson DS, Horvath CM. A road map for those who don't know JAK-STAT. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">296</span>:1653–1655.</span> [<a href="/pubmed/12040185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+road+map+for+those+who+don't+know+JAK-STAT&amp;author=DS+Aaronson&amp;author=CM+Horvath&amp;volume=296&amp;publication_year=2002&amp;pages=1653-1655&amp;pmid=12040185&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib19"><span class="mixed-citation">Paradies NE, Sanford LP, Doetschman T, Friedman RA. Developmental expression of the TGF beta s in the mouse cochlea. <span><span class="ref-journal">Mech Dev. </span>1998;<span class="ref-vol">79</span>:165–168.</span> [<a href="/pubmed/10349630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mech+Dev&amp;title=Developmental+expression+of+the+TGF+beta+s+in+the+mouse+cochlea&amp;author=NE+Paradies&amp;author=LP+Sanford&amp;author=T+Doetschman&amp;author=RA+Friedman&amp;volume=79&amp;publication_year=1998&amp;pages=165-168&amp;pmid=10349630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib20"><span class="mixed-citation">Satoh H, Billings P, Firestein GS, Harris JP, Keithley EM. Transforming growth factor beta expression during an inner ear immune response. <span><span class="ref-journal">Ann Otol Rhinol Laryngol. </span>2006;<span class="ref-vol">115</span>:81–88.</span> [<a href="/pubmed/16466104" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Otol+Rhinol+Laryngol&amp;title=Transforming+growth+factor+beta+expression+during+an+inner+ear+immune+response&amp;author=H+Satoh&amp;author=P+Billings&amp;author=GS+Firestein&amp;author=JP+Harris&amp;author=EM+Keithley&amp;volume=115&amp;publication_year=2006&amp;pages=81-88&amp;pmid=16466104&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4266741&amp;issue-id=246534&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140160369841104title">Abstract</a></li><li><a href="#__sec1title">INTRODUCTION</a></li><li><a href="#__sec2title">MATERIALS AND METHODS</a></li><li><a href="#__sec10title">RESULTS</a></li><li><a href="#__sec19title">DISCUSSION</a></li><li><a href="#idm140160361972944title">Acknowledgments</a></li><li><a href="#idm140160361970176title">Notes</a></li><li><a href="#idm140160361971776title">Footnotes</a></li><li><a href="#sup1title">Supplementary Material</a></li><li><a href="#idm140160373450544title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4266741/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4266741/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4266741/pdf/ejhg201456a.pdf">PDF (823K)</a></li> | <li><a href="#" data-citationid="PMC4266741" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266741%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266741%2F&amp;text=Genome-wide%20association%20analysis%20demonstrates%20the%20highly%20polygenic%20character%20of%20age-related%20hearing%20impairment" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266741%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24939585" data-aiid="4266741" data-aid="4266741" data-iid="246534" data-domainid="1300" data-domain="ejhg" data-accid="PMC4266741" data-md5="4925840c963e3547bfd9caddf30ca63f"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:20:28-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B7681DD56D5210000000005F2038F&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266741%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4266741/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>