<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="pnas"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4664299/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Polymorphisms of large effect explain the majority of the host genetic contribution to variation of HIV-1 virus load</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4664299/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America"><meta name="citation_title" content="Polymorphisms of large effect explain the majority of the host genetic contribution to variation of HIV-1 virus load"><meta name="citation_authors" content="Paul J. McLaren, Cedric Coulonges, István Bartha, Tobias L. Lenz, Aaron J. Deutsch, Arman Bashirova, Susan Buchbinder, Mary N. Carrington, Andrea Cossarizza, Judith Dalmau, Andrea De Luca, James J. Goedert, Deepti Gurdasani, David W. Haas, Joshua T. Herbeck, Eric O. Johnson, Gregory D. Kirk, Olivier Lambotte, Ma Luo, Simon Mallal, Daniëlle van Manen, Javier Martinez-Picado, Laurence Meyer, José M. Miro, James I. Mullins, Niels Obel, Guido Poli, Manjinder S. Sandhu, Hanneke Schuitemaker, Patrick R. Shea, Ioannis Theodorou, Bruce D. Walker, Amy C. Weintrob, Cheryl A. Winkler, Steven M. Wolinsky, Soumya Raychaudhuri, David B. Goldstein, Amalio Telenti, Paul I. W. de Bakker, Jean-François Zagury, Jacques Fellay"><meta name="citation_date" content="24 November 2015"><meta name="citation_issue" content="47"><meta name="citation_volume" content="112"><meta name="citation_firstpage" content="14658"><meta name="citation_doi" content="10.1073/pnas.1514867112"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4664299/?report=abstract"><meta name="citation_pmid" content="26553974"><meta name="DC.Title" content="Polymorphisms of large effect explain the majority of the host genetic contribution to variation of HIV-1 virus load"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="National Academy of Sciences"><meta name="DC.Contributor" content="Paul J. McLaren"><meta name="DC.Contributor" content="Cedric Coulonges"><meta name="DC.Contributor" content="István Bartha"><meta name="DC.Contributor" content="Tobias L. Lenz"><meta name="DC.Contributor" content="Aaron J. Deutsch"><meta name="DC.Contributor" content="Arman Bashirova"><meta name="DC.Contributor" content="Susan Buchbinder"><meta name="DC.Contributor" content="Mary N. Carrington"><meta name="DC.Contributor" content="Andrea Cossarizza"><meta name="DC.Contributor" content="Judith Dalmau"><meta name="DC.Contributor" content="Andrea De Luca"><meta name="DC.Contributor" content="James J. Goedert"><meta name="DC.Contributor" content="Deepti Gurdasani"><meta name="DC.Contributor" content="David W. Haas"><meta name="DC.Contributor" content="Joshua T. Herbeck"><meta name="DC.Contributor" content="Eric O. Johnson"><meta name="DC.Contributor" content="Gregory D. Kirk"><meta name="DC.Contributor" content="Olivier Lambotte"><meta name="DC.Contributor" content="Ma Luo"><meta name="DC.Contributor" content="Simon Mallal"><meta name="DC.Contributor" content="Daniëlle van Manen"><meta name="DC.Contributor" content="Javier Martinez-Picado"><meta name="DC.Contributor" content="Laurence Meyer"><meta name="DC.Contributor" content="José M. Miro"><meta name="DC.Contributor" content="James I. Mullins"><meta name="DC.Contributor" content="Niels Obel"><meta name="DC.Contributor" content="Guido Poli"><meta name="DC.Contributor" content="Manjinder S. Sandhu"><meta name="DC.Contributor" content="Hanneke Schuitemaker"><meta name="DC.Contributor" content="Patrick R. Shea"><meta name="DC.Contributor" content="Ioannis Theodorou"><meta name="DC.Contributor" content="Bruce D. Walker"><meta name="DC.Contributor" content="Amy C. Weintrob"><meta name="DC.Contributor" content="Cheryl A. Winkler"><meta name="DC.Contributor" content="Steven M. Wolinsky"><meta name="DC.Contributor" content="Soumya Raychaudhuri"><meta name="DC.Contributor" content="David B. Goldstein"><meta name="DC.Contributor" content="Amalio Telenti"><meta name="DC.Contributor" content="Paul I. W. de Bakker"><meta name="DC.Contributor" content="Jean-François Zagury"><meta name="DC.Contributor" content="Jacques Fellay"><meta name="DC.Date" content="2015 Nov 24"><meta name="DC.Identifier" content="10.1073/pnas.1514867112"><meta name="DC.Language" content="en"><meta property="og:title" content="Polymorphisms of large effect explain the majority of the host genetic contribution to variation of HIV-1 virus load"><meta property="og:type" content="article"><meta property="og:description" content="A proportion of the variation in HIV-1 viral load in the infected population is influenced by host genetics. Using a large sample of infected individuals (n = 6,315) with genome-wide genotype data, we sought to map genomic regions that influence HIV viral ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4664299/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-pnas.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4664299/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4664299/pdf/pnas.201514867.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B12E1DD580C71000000000A3605DB.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4664299%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/261407/">v.112(47); 2015 Nov 24</a></li><li class="accid">PMC4664299</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-pnas.gif" alt="Logo of pnas" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="306,2,479,64" alt="PNAS" title="PNAS" href="http://www.pnas.org" ref="reftype=publisher&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="59,34,142,50" alt="Info for Authors" title="Info for Authors" href="http://www.pnas.org/site/misc/iforc.shtml" ref="reftype=publisher&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="143,34,200,50" alt="Subscriptions" title="Subscriptions" href="http://www.pnas.org/site/subscriptions/" ref="reftype=publisher&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="201,34,239,50" alt="About" title="About" href="http://www.pnas.org/site/misc/about.shtml" ref="reftype=publisher&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,34,57,50" alt="This Article" title="This Article" href="//dx.doi.org/10.1073/pnas.1514867112" ref="reftype=publisher&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Proc Natl Acad Sci U S A</a></span>. 2015 Nov 24; 112(47): 14658–14663. </span></div><div><span class="fm-vol-iss-date">Published online 2015 Nov 9. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1073%2Fpnas.1514867112" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.1514867112</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4664299</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26553974">26553974</a></div></div></div><div class="fm-doctopic no_margin">Genetics</div></div><h1 class="content-title">Polymorphisms of large effect explain the majority of the host genetic contribution to variation of HIV-1 virus load</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=McLaren%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323972348560" co-class="co-affbox">Paul J. McLaren</a>,<sup>a,</sup><sup>b,</sup><sup>1</sup> <a href="/pubmed/?term=Coulonges%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323968912112" co-class="co-affbox">Cedric Coulonges</a>,<sup>c,</sup><sup>d,</sup><sup>1</sup> <a href="/pubmed/?term=Bartha%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140324006025696" co-class="co-affbox">István Bartha</a>,<sup>a,</sup><sup>b,</sup><sup>1</sup> <a href="/pubmed/?term=Lenz%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323972765888" co-class="co-affbox">Tobias L. Lenz</a>,<sup>e</sup> <a href="/pubmed/?term=Deutsch%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323972763776" co-class="co-affbox">Aaron J. Deutsch</a>,<sup>f,</sup><sup>g,</sup><sup>h</sup> <a href="/pubmed/?term=Bashirova%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323969374592" co-class="co-affbox">Arman Bashirova</a>,<sup>i</sup> <a href="/pubmed/?term=Buchbinder%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323964027312" co-class="co-affbox">Susan Buchbinder</a>,<sup>j</sup> <a href="/pubmed/?term=Carrington%20MN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323964025200" co-class="co-affbox">Mary N. Carrington</a>,<sup>i,</sup><sup>k</sup> <a href="/pubmed/?term=Cossarizza%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323962251424" co-class="co-affbox">Andrea Cossarizza</a>,<sup>l</sup> <a href="/pubmed/?term=Dalmau%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323964443200" co-class="co-affbox">Judith Dalmau</a>,<sup>m</sup> <a href="/pubmed/?term=De%20Luca%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323964441088" co-class="co-affbox">Andrea De Luca</a>,<sup>n,</sup><sup>o</sup> <a href="/pubmed/?term=Goedert%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323970987088" co-class="co-affbox">James J. Goedert</a>,<sup>p</sup> <a href="/pubmed/?term=Gurdasani%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323969786112" co-class="co-affbox">Deepti Gurdasani</a>,<sup>q,</sup><sup>r</sup> <a href="/pubmed/?term=Haas%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323969783136" co-class="co-affbox">David W. Haas</a>,<sup>s</sup> <a href="/pubmed/?term=Herbeck%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323968657360" co-class="co-affbox">Joshua T. Herbeck</a>,<sup>t</sup> <a href="/pubmed/?term=Johnson%20EO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323968655248" co-class="co-affbox">Eric O. Johnson</a>,<sup>u</sup> <a href="/pubmed/?term=Kirk%20GD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323967724448" co-class="co-affbox">Gregory D. Kirk</a>,<sup>v</sup> <a href="/pubmed/?term=Lambotte%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323967722336" co-class="co-affbox">Olivier Lambotte</a>,<sup>w,</sup><sup>x,</sup><sup>y</sup> <a href="/pubmed/?term=Luo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323969596736" co-class="co-affbox">Ma Luo</a>,<sup>z,</sup><sup>aa</sup> <a href="/pubmed/?term=Mallal%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323967122464" co-class="co-affbox">Simon Mallal</a>,<sup>bb</sup> <a href="/pubmed/?term=van%20Manen%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323970663792" co-class="co-affbox">Daniëlle van Manen</a>,<sup>cc,</sup><sup>2</sup> <a href="/pubmed/?term=Martinez-Picado%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323970660816" co-class="co-affbox">Javier Martinez-Picado</a>,<sup>m,</sup><sup>dd</sup> <a href="/pubmed/?term=Meyer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323959860912" co-class="co-affbox">Laurence Meyer</a>,<sup>d,</sup><sup>ee,</sup><sup>ff,</sup><sup>gg</sup> <a href="/pubmed/?term=Miro%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140324009060880" co-class="co-affbox">José M. Miro</a>,<sup>hh</sup> <a href="/pubmed/?term=Mullins%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323967461232" co-class="co-affbox">James I. Mullins</a>,<sup>ii</sup> <a href="/pubmed/?term=Obel%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323967459120" co-class="co-affbox">Niels Obel</a>,<sup>jj</sup> <a href="/pubmed/?term=Poli%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323962368944" co-class="co-affbox">Guido Poli</a>,<sup>kk,</sup><sup>ll</sup> <a href="/pubmed/?term=Sandhu%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140324008168832" co-class="co-affbox">Manjinder S. Sandhu</a>,<sup>q,</sup><sup>r</sup> <a href="/pubmed/?term=Schuitemaker%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140324008165856" co-class="co-affbox">Hanneke Schuitemaker</a>,<sup>cc,</sup><sup>2</sup> <a href="/pubmed/?term=Shea%20PR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323966166880" co-class="co-affbox">Patrick R. Shea</a>,<sup>mm</sup> <a href="/pubmed/?term=Theodorou%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140324009073888" co-class="co-affbox">Ioannis Theodorou</a>,<sup>d,</sup><sup>nn</sup> <a href="/pubmed/?term=Walker%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140324009070912" co-class="co-affbox">Bruce D. Walker</a>,<sup>i,</sup><sup>oo</sup> <a href="/pubmed/?term=Weintrob%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323965927552" co-class="co-affbox">Amy C. Weintrob</a>,<sup>pp</sup> <a href="/pubmed/?term=Winkler%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323973348256" co-class="co-affbox">Cheryl A. Winkler</a>,<sup>qq</sup> <a href="/pubmed/?term=Wolinsky%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323973346144" co-class="co-affbox">Steven M. Wolinsky</a>,<sup>rr</sup> <a href="/pubmed/?term=Raychaudhuri%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140324007128496" co-class="co-affbox">Soumya Raychaudhuri</a>,<sup>g,</sup><sup>h,</sup><sup>ss</sup> <a href="/pubmed/?term=Goldstein%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323964572544" co-class="co-affbox">David B. Goldstein</a>,<sup>mm</sup> <a href="/pubmed/?term=Telenti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323964570432" co-class="co-affbox">Amalio Telenti</a>,<sup>tt</sup> <a href="/pubmed/?term=de%20Bakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323971373712" co-class="co-affbox">Paul I. W. de Bakker</a>,<sup>uu,</sup><sup>vv</sup> <a href="/pubmed/?term=Zagury%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323964849520" co-class="co-affbox">Jean-François Zagury</a>,<sup>c,</sup><sup>d</sup> and  <a href="/pubmed/?term=Fellay%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974" class="affpopup" co-rid="_co_idm140323964846544" co-class="co-affbox">Jacques Fellay</a><sup>a,</sup><sup>b,</sup><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140323972348560"><h3 class="no_margin">Paul J. McLaren</h3><p><sup>a</sup>Global Health Institute, School of Life Sciences, École Polytechnique Fédérale de Lausanne, 1015 Lausanne, Switzerland;</p><p><sup>b</sup>Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland;</p><div>Find articles by <a href="/pubmed/?term=McLaren%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Paul J. McLaren</a></div></div><div id="_co_idm140323968912112"><h3 class="no_margin">Cedric Coulonges</h3><p><sup>c</sup>Laboratoire Génomique, Bioinformatique, et Applications, EA4627, Chaire de Bioinformatique, Conservatoire National des Arts et Métiers, 75003 Paris, France;</p><p><sup>d</sup>ANRS Genomic Group (French Agency for Research on AIDS and Hepatitis), 75013 Paris, France;</p><div>Find articles by <a href="/pubmed/?term=Coulonges%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Cedric Coulonges</a></div></div><div id="_co_idm140324006025696"><h3 class="no_margin">István Bartha</h3><p><sup>a</sup>Global Health Institute, School of Life Sciences, École Polytechnique Fédérale de Lausanne, 1015 Lausanne, Switzerland;</p><p><sup>b</sup>Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland;</p><div>Find articles by <a href="/pubmed/?term=Bartha%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">István Bartha</a></div></div><div id="_co_idm140323972765888"><h3 class="no_margin">Tobias L. Lenz</h3><p><sup>e</sup>Evolutionary Immunogenomics, Department of Evolutionary Ecology, Max Planck Institute for Evolutionary Biology, 24306 Ploen, Germany;</p><div>Find articles by <a href="/pubmed/?term=Lenz%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Tobias L. Lenz</a></div></div><div id="_co_idm140323972763776"><h3 class="no_margin">Aaron J. Deutsch</h3><p><sup>f</sup>Harvard-MIT Division of Health Sciences and Technology and Harvard Medical School, Harvard University, Boston, MA,  02115;</p><p><sup>g</sup>Division of Genetics and Rheumatology, Department of Medicine, Brigham and Women's Hospital, Boston, MA,  02115;</p><p><sup>h</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA,  02142;</p><div>Find articles by <a href="/pubmed/?term=Deutsch%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Aaron J. Deutsch</a></div></div><div id="_co_idm140323969374592"><h3 class="no_margin">Arman Bashirova</h3><p><sup>i</sup>Ragon Institute of Massachusetts General Hospital, Massachusetts Institute of Technology, and Harvard, Boston, MA,  02129;</p><div>Find articles by <a href="/pubmed/?term=Bashirova%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Arman Bashirova</a></div></div><div id="_co_idm140323964027312"><h3 class="no_margin">Susan Buchbinder</h3><p><sup>j</sup>Bridge HIV-1, San Francisco Department of Public Health, San Francisco, CA,  94102;</p><div>Find articles by <a href="/pubmed/?term=Buchbinder%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Susan Buchbinder</a></div></div><div id="_co_idm140323964025200"><h3 class="no_margin">Mary N. Carrington</h3><p><sup>i</sup>Ragon Institute of Massachusetts General Hospital, Massachusetts Institute of Technology, and Harvard, Boston, MA,  02129;</p><p><sup>k</sup>Cancer and Inflammation Program, Laboratory of Experimental Immunology, Leidos Biomedical Research, Inc., Frederick National Laboratory for Cancer Research, Frederick, MD,  21702;</p><div>Find articles by <a href="/pubmed/?term=Carrington%20MN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Mary N. Carrington</a></div></div><div id="_co_idm140323962251424"><h3 class="no_margin">Andrea Cossarizza</h3><p><sup>l</sup>Department of Surgery, Medicine, Dentistry and Morphological Sciences, University of Modena and Reggio Emilia School of Medicine, 41121 Modena, Italy;</p><div>Find articles by <a href="/pubmed/?term=Cossarizza%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Andrea Cossarizza</a></div></div><div id="_co_idm140323964443200"><h3 class="no_margin">Judith Dalmau</h3><p><sup>m</sup>AIDS Research Institute IrsiCaixa, Institut d'Investigació en Ciències de la Salut Germans Trias i Pujol, Universitat Autònoma de Barcelona, 08916 Badalona, Spain;</p><div>Find articles by <a href="/pubmed/?term=Dalmau%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Judith Dalmau</a></div></div><div id="_co_idm140323964441088"><h3 class="no_margin">Andrea De Luca</h3><p><sup>n</sup>University Division of Infectious Diseases, Siena University Hospital, 53100 Siena, Italy;</p><p><sup>o</sup>Department of Medical Biotechnologies, University of Siena, 53100 Siena, Italy;</p><div>Find articles by <a href="/pubmed/?term=De%20Luca%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Andrea De Luca</a></div></div><div id="_co_idm140323970987088"><h3 class="no_margin">James J. Goedert</h3><p><sup>p</sup>Infections and Immunoepidemiology Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute, Rockville, MD,  20850;</p><div>Find articles by <a href="/pubmed/?term=Goedert%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">James J. Goedert</a></div></div><div id="_co_idm140323969786112"><h3 class="no_margin">Deepti Gurdasani</h3><p><sup>q</sup>Human Genetics, Wellcome Trust Sanger Institute, CB10 1SA Hinxton, United Kingdom;</p><p><sup>r</sup>Department of Medicine, University of Cambridge, CB2 2QQ Cambridge, United Kingdom;</p><div>Find articles by <a href="/pubmed/?term=Gurdasani%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Deepti Gurdasani</a></div></div><div id="_co_idm140323969783136"><h3 class="no_margin">David W. Haas</h3><p><sup>s</sup>Vanderbilt University School of Medicine, Nashville, TN,  37212;</p><div>Find articles by <a href="/pubmed/?term=Haas%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">David W. Haas</a></div></div><div id="_co_idm140323968657360"><h3 class="no_margin">Joshua T. Herbeck</h3><p><sup>t</sup>Department of Global Health, University of Washington, Seattle, WA,  98195;</p><div>Find articles by <a href="/pubmed/?term=Herbeck%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Joshua T. Herbeck</a></div></div><div id="_co_idm140323968655248"><h3 class="no_margin">Eric O. Johnson</h3><p><sup>u</sup>Behavioral Health Epidemiology, RTI International, Research Triangle Park, NC,  27709;</p><div>Find articles by <a href="/pubmed/?term=Johnson%20EO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Eric O. Johnson</a></div></div><div id="_co_idm140323967724448"><h3 class="no_margin">Gregory D. Kirk</h3><p><sup>v</sup>Department of Epidemiology, Johns Hopkins University, Baltimore, MD,  21205;</p><div>Find articles by <a href="/pubmed/?term=Kirk%20GD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Gregory D. Kirk</a></div></div><div id="_co_idm140323967722336"><h3 class="no_margin">Olivier Lambotte</h3><p><sup>w</sup>INSERM U1012, 94270 Bicêtre, France;</p><p><sup>x</sup>University Paris-Sud, 94270 Bicêtre, France;</p><p><sup>y</sup>Assistance Publique-Hôpitaux de Paris, Department of Internal Medicine and Infectious Diseases, Bicêtre Hospital, 94270 Bicêtre, France;</p><div>Find articles by <a href="/pubmed/?term=Lambotte%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Olivier Lambotte</a></div></div><div id="_co_idm140323969596736"><h3 class="no_margin">Ma Luo</h3><p><sup>z</sup>Department of Medical Microbiology, University of Manitoba, R3E 0J6 Winnipeg, MB, Canada;</p><p><sup>aa</sup>National Microbiology Laboratory, R3E 3P6 Winnipeg, MB, Canada;</p><div>Find articles by <a href="/pubmed/?term=Luo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Ma Luo</a></div></div><div id="_co_idm140323967122464"><h3 class="no_margin">Simon Mallal</h3><p><sup>bb</sup>Institute for Immunology &amp; Infectious Diseases, Murdoch University and Pathwest, 6150 Perth, Australia;</p><div>Find articles by <a href="/pubmed/?term=Mallal%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Simon Mallal</a></div></div><div id="_co_idm140323970663792"><h3 class="no_margin">Daniëlle van Manen</h3><p><sup>cc</sup>Center for Infectious Diseases and Immunity Amsterdam, Academic Medical Center of the University of Amsterdam, 1105 AZ Amsterdam, The Netherlands;</p><div>Find articles by <a href="/pubmed/?term=van%20Manen%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Daniëlle van Manen</a></div></div><div id="_co_idm140323970660816"><h3 class="no_margin">Javier Martinez-Picado</h3><p><sup>m</sup>AIDS Research Institute IrsiCaixa, Institut d'Investigació en Ciències de la Salut Germans Trias i Pujol, Universitat Autònoma de Barcelona, 08916 Badalona, Spain;</p><p><sup>dd</sup>Institució Catalana de Recerca i Estudis Avançats, 08916 Barcelona, Spain;</p><div>Find articles by <a href="/pubmed/?term=Martinez-Picado%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Javier Martinez-Picado</a></div></div><div id="_co_idm140323959860912"><h3 class="no_margin">Laurence Meyer</h3><p><sup>d</sup>ANRS Genomic Group (French Agency for Research on AIDS and Hepatitis), 75013 Paris, France;</p><p><sup>ee</sup>Inserm, Centre de Recherche en Épidémiologie et Santé des Populations, U1018, Le Kremlin 94270 Bicêtre, France;</p><p><sup>ff</sup>Faculté de Médecine Paris-Sud, Université Paris-Sud, UMRS 1018, Le Kremlin 94270 Bicêtre, France;</p><p><sup>gg</sup>Epidemiology and Public Health Service, Assistance Publique-Hôpitaux de Paris, Hôpital Bicêtre, Le Kremlin 94270 Bicêtre, France;</p><div>Find articles by <a href="/pubmed/?term=Meyer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Laurence Meyer</a></div></div><div id="_co_idm140324009060880"><h3 class="no_margin">José M. Miro</h3><p><sup>hh</sup>Infectious Diseases Service, Hospital Clinic-Institut d'Investigacions Biomèdiques August Pi i Sunyer, University of Barcelona, 08036 Barcelona, Spain;</p><div>Find articles by <a href="/pubmed/?term=Miro%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">José M. Miro</a></div></div><div id="_co_idm140323967461232"><h3 class="no_margin">James I. Mullins</h3><p><sup>ii</sup>Department of Microbiology, University of Washington, Seattle, WA,  98195;</p><div>Find articles by <a href="/pubmed/?term=Mullins%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">James I. Mullins</a></div></div><div id="_co_idm140323967459120"><h3 class="no_margin">Niels Obel</h3><p><sup>jj</sup>Department of Infectious Diseases, Rigshospitalet, Copenhagen University Hospital, 2100 Copenhagen, Denmark;</p><div>Find articles by <a href="/pubmed/?term=Obel%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Niels Obel</a></div></div><div id="_co_idm140323962368944"><h3 class="no_margin">Guido Poli</h3><p><sup>kk</sup>Division of Immunology, Transplantation and Infectious Diseases, San Raffaele Scientific Institute, 20132 Milan, Italy;</p><p><sup>ll</sup>Vita-Salute San Raffaele University School of Medicine, 20132 Milan, Italy;</p><div>Find articles by <a href="/pubmed/?term=Poli%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Guido Poli</a></div></div><div id="_co_idm140324008168832"><h3 class="no_margin">Manjinder S. Sandhu</h3><p><sup>q</sup>Human Genetics, Wellcome Trust Sanger Institute, CB10 1SA Hinxton, United Kingdom;</p><p><sup>r</sup>Department of Medicine, University of Cambridge, CB2 2QQ Cambridge, United Kingdom;</p><div>Find articles by <a href="/pubmed/?term=Sandhu%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Manjinder S. Sandhu</a></div></div><div id="_co_idm140324008165856"><h3 class="no_margin">Hanneke Schuitemaker</h3><p><sup>cc</sup>Center for Infectious Diseases and Immunity Amsterdam, Academic Medical Center of the University of Amsterdam, 1105 AZ Amsterdam, The Netherlands;</p><div>Find articles by <a href="/pubmed/?term=Schuitemaker%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Hanneke Schuitemaker</a></div></div><div id="_co_idm140323966166880"><h3 class="no_margin">Patrick R. Shea</h3><p><sup>mm</sup>Institute for Genomic Medicine, Columbia University, New York, NY,  10032;</p><div>Find articles by <a href="/pubmed/?term=Shea%20PR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Patrick R. Shea</a></div></div><div id="_co_idm140324009073888"><h3 class="no_margin">Ioannis Theodorou</h3><p><sup>d</sup>ANRS Genomic Group (French Agency for Research on AIDS and Hepatitis), 75013 Paris, France;</p><p><sup>nn</sup>INSERM UMRS 945, 75014 Paris, France;</p><div>Find articles by <a href="/pubmed/?term=Theodorou%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Ioannis Theodorou</a></div></div><div id="_co_idm140324009070912"><h3 class="no_margin">Bruce D. Walker</h3><p><sup>i</sup>Ragon Institute of Massachusetts General Hospital, Massachusetts Institute of Technology, and Harvard, Boston, MA,  02129;</p><p><sup>oo</sup>Howard Hughes Medical Institute, Chevy Chase, MD,  20815;</p><div>Find articles by <a href="/pubmed/?term=Walker%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Bruce D. Walker</a></div></div><div id="_co_idm140323965927552"><h3 class="no_margin">Amy C. Weintrob</h3><p><sup>pp</sup>Infectious Disease Clinical Research Program, Uniformed Services University of the Health Sciences, Bethesda, MD,  20814;</p><div>Find articles by <a href="/pubmed/?term=Weintrob%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Amy C. Weintrob</a></div></div><div id="_co_idm140323973348256"><h3 class="no_margin">Cheryl A. Winkler</h3><p><sup>qq</sup>Basic Research Laboratory, Molecular Genetic Epidemiology Section, Center for Cancer Research, National Cancer Institute, Leidos Biomedical Research, Inc., Frederick National Laboratory, Frederick, MD,  21702;</p><div>Find articles by <a href="/pubmed/?term=Winkler%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Cheryl A. Winkler</a></div></div><div id="_co_idm140323973346144"><h3 class="no_margin">Steven M. Wolinsky</h3><p><sup>rr</sup>Division of Infectious Diseases, The Feinberg School of Medicine, Northwestern University, Chicago, IL,  60611;</p><div>Find articles by <a href="/pubmed/?term=Wolinsky%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Steven M. Wolinsky</a></div></div><div id="_co_idm140324007128496"><h3 class="no_margin">Soumya Raychaudhuri</h3><p><sup>g</sup>Division of Genetics and Rheumatology, Department of Medicine, Brigham and Women's Hospital, Boston, MA,  02115;</p><p><sup>h</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA,  02142;</p><p><sup>ss</sup>Faculty of Medical and Human Sciences, University of Manchester, M13 9PL Manchester, United Kingdom;</p><div>Find articles by <a href="/pubmed/?term=Raychaudhuri%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Soumya Raychaudhuri</a></div></div><div id="_co_idm140323964572544"><h3 class="no_margin">David B. Goldstein</h3><p><sup>mm</sup>Institute for Genomic Medicine, Columbia University, New York, NY,  10032;</p><div>Find articles by <a href="/pubmed/?term=Goldstein%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">David B. Goldstein</a></div></div><div id="_co_idm140323964570432"><h3 class="no_margin">Amalio Telenti</h3><p><sup>tt</sup>The J. Craig Venter Institute, La Jolla, CA,  92037;</p><div>Find articles by <a href="/pubmed/?term=Telenti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Amalio Telenti</a></div></div><div id="_co_idm140323971373712"><h3 class="no_margin">Paul I. W. de Bakker</h3><p><sup>uu</sup>Department of Medical Genetics, University Medical Center Utrecht, 3584 CX Utrecht, The Netherlands;</p><p><sup>vv</sup>Department of Epidemiology, University Medical Center Utrecht, 3584 CX Utrecht, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=de%20Bakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Paul I. W. de Bakker</a></div></div><div id="_co_idm140323964849520"><h3 class="no_margin">Jean-François Zagury</h3><p><sup>c</sup>Laboratoire Génomique, Bioinformatique, et Applications, EA4627, Chaire de Bioinformatique, Conservatoire National des Arts et Métiers, 75003 Paris, France;</p><p><sup>d</sup>ANRS Genomic Group (French Agency for Research on AIDS and Hepatitis), 75013 Paris, France;</p><div>Find articles by <a href="/pubmed/?term=Zagury%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Jean-François Zagury</a></div></div><div id="_co_idm140323964846544"><h3 class="no_margin">Jacques Fellay</h3><p><sup>a</sup>Global Health Institute, School of Life Sciences, École Polytechnique Fédérale de Lausanne, 1015 Lausanne, Switzerland;</p><p><sup>b</sup>Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland;</p><div>Find articles by <a href="/pubmed/?term=Fellay%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26553974">Jacques Fellay</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140324042851168_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140324042851168_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140324042851168_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>a</sup>Global Health Institute, School of Life Sciences, École Polytechnique Fédérale de Lausanne, 1015 Lausanne, Switzerland;</div><div class="fm-affl" id="aff2" lang="en"><sup>b</sup>Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland;</div><div class="fm-affl" id="aff3" lang="en"><sup>c</sup>Laboratoire Génomique, Bioinformatique, et Applications, EA4627, Chaire de Bioinformatique, Conservatoire National des Arts et Métiers, 75003 Paris, France;</div><div class="fm-affl" id="aff4" lang="en"><sup>d</sup>ANRS Genomic Group (French Agency for Research on AIDS and Hepatitis), 75013 Paris, France;</div><div class="fm-affl" id="aff5" lang="en"><sup>e</sup>Evolutionary Immunogenomics, Department of Evolutionary Ecology, Max Planck Institute for Evolutionary Biology, 24306 Ploen, Germany;</div><div class="fm-affl" id="aff6" lang="en"><sup>f</sup>Harvard-MIT Division of Health Sciences and Technology and Harvard Medical School, Harvard University, Boston, MA,  02115;</div><div class="fm-affl" id="aff7" lang="en"><sup>g</sup>Division of Genetics and Rheumatology, Department of Medicine, Brigham and Women's Hospital, Boston, MA,  02115;</div><div class="fm-affl" id="aff8" lang="en"><sup>h</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA,  02142;</div><div class="fm-affl" id="aff9" lang="en"><sup>i</sup>Ragon Institute of Massachusetts General Hospital, Massachusetts Institute of Technology, and Harvard, Boston, MA,  02129;</div><div class="fm-affl" id="aff10" lang="en"><sup>j</sup>Bridge HIV-1, San Francisco Department of Public Health, San Francisco, CA,  94102;</div><div class="fm-affl" id="aff11" lang="en"><sup>k</sup>Cancer and Inflammation Program, Laboratory of Experimental Immunology, Leidos Biomedical Research, Inc., Frederick National Laboratory for Cancer Research, Frederick, MD,  21702;</div><div class="fm-affl" id="aff12" lang="en"><sup>l</sup>Department of Surgery, Medicine, Dentistry and Morphological Sciences, University of Modena and Reggio Emilia School of Medicine, 41121 Modena, Italy;</div><div class="fm-affl" id="aff13" lang="en"><sup>m</sup>AIDS Research Institute IrsiCaixa, Institut d'Investigació en Ciències de la Salut Germans Trias i Pujol, Universitat Autònoma de Barcelona, 08916 Badalona, Spain;</div><div class="fm-affl" id="aff14" lang="en"><sup>n</sup>University Division of Infectious Diseases, Siena University Hospital, 53100 Siena, Italy;</div><div class="fm-affl" id="aff15" lang="en"><sup>o</sup>Department of Medical Biotechnologies, University of Siena, 53100 Siena, Italy;</div><div class="fm-affl" id="aff16" lang="en"><sup>p</sup>Infections and Immunoepidemiology Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute, Rockville, MD,  20850;</div><div class="fm-affl" id="aff17" lang="en"><sup>q</sup>Human Genetics, Wellcome Trust Sanger Institute, CB10 1SA Hinxton, United Kingdom;</div><div class="fm-affl" id="aff18" lang="en"><sup>r</sup>Department of Medicine, University of Cambridge, CB2 2QQ Cambridge, United Kingdom;</div><div class="fm-affl" id="aff19" lang="en"><sup>s</sup>Vanderbilt University School of Medicine, Nashville, TN,  37212;</div><div class="fm-affl" id="aff20" lang="en"><sup>t</sup>Department of Global Health, University of Washington, Seattle, WA,  98195;</div><div class="fm-affl" id="aff21" lang="en"><sup>u</sup>Behavioral Health Epidemiology, RTI International, Research Triangle Park, NC,  27709;</div><div class="fm-affl" id="aff22" lang="en"><sup>v</sup>Department of Epidemiology, Johns Hopkins University, Baltimore, MD,  21205;</div><div class="fm-affl" id="aff23" lang="en"><sup>w</sup>INSERM U1012, 94270 Bicêtre, France;</div><div class="fm-affl" id="aff24" lang="en"><sup>x</sup>University Paris-Sud, 94270 Bicêtre, France;</div><div class="fm-affl" id="aff25" lang="en"><sup>y</sup>Assistance Publique-Hôpitaux de Paris, Department of Internal Medicine and Infectious Diseases, Bicêtre Hospital, 94270 Bicêtre, France;</div><div class="fm-affl" id="aff26" lang="en"><sup>z</sup>Department of Medical Microbiology, University of Manitoba, R3E 0J6 Winnipeg, MB, Canada;</div><div class="fm-affl" id="aff27" lang="en"><sup>aa</sup>National Microbiology Laboratory, R3E 3P6 Winnipeg, MB, Canada;</div><div class="fm-affl" id="aff28" lang="en"><sup>bb</sup>Institute for Immunology &amp; Infectious Diseases, Murdoch University and Pathwest, 6150 Perth, Australia;</div><div class="fm-affl" id="aff29" lang="en"><sup>cc</sup>Center for Infectious Diseases and Immunity Amsterdam, Academic Medical Center of the University of Amsterdam, 1105 AZ Amsterdam, The Netherlands;</div><div class="fm-affl" id="aff30" lang="en"><sup>dd</sup>Institució Catalana de Recerca i Estudis Avançats, 08916 Barcelona, Spain;</div><div class="fm-affl" id="aff31" lang="en"><sup>ee</sup>Inserm, Centre de Recherche en Épidémiologie et Santé des Populations, U1018, Le Kremlin 94270 Bicêtre, France;</div><div class="fm-affl" id="aff32" lang="en"><sup>ff</sup>Faculté de Médecine Paris-Sud, Université Paris-Sud, UMRS 1018, Le Kremlin 94270 Bicêtre, France;</div><div class="fm-affl" id="aff33" lang="en"><sup>gg</sup>Epidemiology and Public Health Service, Assistance Publique-Hôpitaux de Paris, Hôpital Bicêtre, Le Kremlin 94270 Bicêtre, France;</div><div class="fm-affl" id="aff34" lang="en"><sup>hh</sup>Infectious Diseases Service, Hospital Clinic-Institut d'Investigacions Biomèdiques August Pi i Sunyer, University of Barcelona, 08036 Barcelona, Spain;</div><div class="fm-affl" id="aff35" lang="en"><sup>ii</sup>Department of Microbiology, University of Washington, Seattle, WA,  98195;</div><div class="fm-affl" id="aff36" lang="en"><sup>jj</sup>Department of Infectious Diseases, Rigshospitalet, Copenhagen University Hospital, 2100 Copenhagen, Denmark;</div><div class="fm-affl" id="aff37" lang="en"><sup>kk</sup>Division of Immunology, Transplantation and Infectious Diseases, San Raffaele Scientific Institute, 20132 Milan, Italy;</div><div class="fm-affl" id="aff38" lang="en"><sup>ll</sup>Vita-Salute San Raffaele University School of Medicine, 20132 Milan, Italy;</div><div class="fm-affl" id="aff39" lang="en"><sup>mm</sup>Institute for Genomic Medicine, Columbia University, New York, NY,  10032;</div><div class="fm-affl" id="aff40" lang="en"><sup>nn</sup>INSERM UMRS 945, 75014 Paris, France;</div><div class="fm-affl" id="aff41" lang="en"><sup>oo</sup>Howard Hughes Medical Institute, Chevy Chase, MD,  20815;</div><div class="fm-affl" id="aff42" lang="en"><sup>pp</sup>Infectious Disease Clinical Research Program, Uniformed Services University of the Health Sciences, Bethesda, MD,  20814;</div><div class="fm-affl" id="aff43" lang="en"><sup>qq</sup>Basic Research Laboratory, Molecular Genetic Epidemiology Section, Center for Cancer Research, National Cancer Institute, Leidos Biomedical Research, Inc., Frederick National Laboratory, Frederick, MD,  21702;</div><div class="fm-affl" id="aff44" lang="en"><sup>rr</sup>Division of Infectious Diseases, The Feinberg School of Medicine, Northwestern University, Chicago, IL,  60611;</div><div class="fm-affl" id="aff45" lang="en"><sup>ss</sup>Faculty of Medical and Human Sciences, University of Manchester, M13 9PL Manchester, United Kingdom;</div><div class="fm-affl" id="aff46" lang="en"><sup>tt</sup>The J. Craig Venter Institute, La Jolla, CA,  92037;</div><div class="fm-affl" id="aff47" lang="en"><sup>uu</sup>Department of Medical Genetics, University Medical Center Utrecht, 3584 CX Utrecht, The Netherlands;</div><div class="fm-affl" id="aff48" lang="en"><sup>vv</sup>Department of Epidemiology, University Medical Center Utrecht, 3584 CX Utrecht, The Netherlands</div><div id="cor1"><sup>3</sup>To whom correspondence should be addressed. Email: <a href="mailto:dev@null" data-email="hc.lfpe@yalleF.seuqcaJ" class="oemail">hc.lfpe@yalleF.seuqcaJ</a>.</div><div id="idm140323966179040">Edited by John M. Coffin, Tufts University School of Medicine, Boston, MA, and approved October 13, 2015 (received for review July 28, 2015)</div><div><p id="__p2" class="p p-first-last">Author contributions: P.J.M., I.B., S.R., D.B.G., A.T., P.I.d.B., J.-F.Z., and J.F. designed research; P.J.M., C.C., and I.B. performed research; A.B., S.B., M.N.C., A.C., J.D., A.D.L., J.J.G., D.G., D.W.H., J.T.H., E.O.J., G.D.K., O.L., M.L., S.M., D.v.M., J.M.-P., L.M., J.M.M., J.I.M., N.O., G.P., M.S.S., H.S., P.R.S., I.T., B.D.W., A.C.W., C.A.W., S.M.W., and S.R. contributed new reagents/analytic tools; P.J.M., C.C., I.B., T.L.L., and A.J.D. analyzed data; and P.J.M., C.C., I.B., J.-F.Z., and J.F. wrote the paper.</p></div><div id="fn1"><sup>1</sup>P.J.M., C.C., and I.B. contributed equally to this work.</div><div id="fn2"><sup>2</sup>Present address: Janssen Pharmaceuticals, 2333 Leiden, The Netherlands.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140324042851168_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div class="license half_rhythm">Freely available online through the PNAS open access option.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4664299/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary File.</strong></div><div><a href="/pmc/articles/PMC4664299/bin/pnas.1514867112.sapp.pdf" data-ga-action="click_feat_suppl">pnas.1514867112.sapp.pdf</a><span style="color:gray"> (5.1M)</span></div><div class="small guid">GUID:&nbsp;54074A8A-FA4F-427E-90C7-BE96B00FA4D3</div></div></dd></dl></div><div id="idm140323968968272" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140323968968272title" style="text-transform: uppercase;">Significance</h2><!--article-meta--><div><p id="__p5" class="p p-first-last">A proportion of the variation in HIV-1 viral load in the infected population is influenced by host genetics. Using a large sample of infected individuals (<em>n</em> = 6,315) with genome-wide genotype data, we sought to map genomic regions that influence HIV viral load and quantify their impact. We identified amino acid positions located in the binding groove of class I HLA proteins (HLA-A and -B) and SNPs in the chemokine (C-C motif) receptor 5 gene region that together explain 14.5% of the observed variation in HIV viral load. Controlling for these signals, we estimated that an additional 5.5% can be explained by common, additive genetic variation. Thus, we demonstrate that common variants of large effect explain the majority of the host genetic component of HIV viral load.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">HIV-1 control, GWAS, heritability, infectious disease, genomics</span></div></div><div id="idm140323970816864" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140323970816864title" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p id="__p6" class="p p-first-last">Previous genome-wide association studies (GWAS) of HIV-1–infected populations have been underpowered to detect common variants with moderate impact on disease outcome and have not assessed the phenotypic variance explained by genome-wide additive effects. By combining the majority of available genome-wide genotyping data in HIV-infected populations, we tested for association between ∼8 million variants and viral load (HIV RNA copies per milliliter of plasma) in 6,315 individuals of European ancestry. The strongest signal of association was observed in the HLA class I region that was fully explained by independent effects mapping to five variable amino acid positions in the peptide binding grooves of the HLA-B and HLA-A proteins. We observed a second genome-wide significant association signal in the chemokine (C-C motif) receptor (CCR) gene cluster on chromosome 3. Conditional analysis showed that this signal could not be fully attributed to the known protective <em>CCR5Δ32</em> allele and the risk P1 haplotype, suggesting further causal variants in this region. Heritability analysis demonstrated that common human genetic variation—mostly in the HLA and CCR5 regions—explains 25% of the variability in viral load. This study suggests that analyses in non-European populations and of variant classes not assessed by GWAS should be priorities for the field going forward.</p></div></div><div id="idm140323968713200" class="tsec sec headless whole_rhythm"><p id="__p7" class="p p-first">Upon infection with human immunodeficiency virus type 1 (HIV-1), there is substantial variability in viral control and rate of disease progression. After primary infection, characterized by high levels of viremia (HIV-1 RNA copies per milliliter of plasma) and transient loss of CD4<sup>+</sup> T cells, most patients enter an asymptomatic period and maintain a relatively stable viral load off therapy. It has been well-established that this set point viral load (spVL) varies in the infected population and positively correlates with rate of disease progression (<a href="#r1" rid="r1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132750">1</a>). Thus, spVL is an easily measured and informative marker of clinical outcome.</p><p id="__p8">Variability in spVL is influenced by host, viral, and environmental factors, including human genetic variation. Genome-wide association studies (GWAS) have consistently identified variation in the major histocompatibility complex (MHC) region on chromosome 6 as the major host determinant of HIV-1 viral load and disease progression (usually rate of CD4<sup>+</sup> T-cell decline) (<a href="#r2" rid="r2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132782">2</a><a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132756"></a><a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132743"></a><a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132752"></a>–<a href="#r6" rid="r6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132767">6</a>). Similarly, studies of extreme phenotypes of HIV-1 progression [i.e., elite controllers (<a href="#r7" rid="r7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132759">7</a>, <a href="#r8" rid="r8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132798">8</a>), long-term nonprogressors (<a href="#r9" rid="r9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132755">9</a>), and rapid progressors (<a href="#r10" rid="r10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132773">10</a>)] have underscored the primary role of the MHC in determining HIV-1 outcome. However, the GWAS of HIV-1—related phenotypes performed to date have been underpowered to identify the types of variants with modest effect sizes that have been observed to influence other complex human traits. To what extent additional host genetic factors contribute to HIV-1 control and the total variability in spVL explained by host genetics remain open questions.</p><p id="__p9" class="p">Here, we report the results from the second phase of the International Collaboration for the Genomics of HIV-1 (for a complete list of contributors see <a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, <em>Note S1</em></a>) (<a href="#r11" rid="r11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132742">11</a>), which has collected the majority of available genome-wide genotype data from HIV-1–infected patients with clinical follow-up. We tested ∼8 million variants, including single nucleotide polymorphisms (SNPs), short insertions and deletions (indels), classical human leukocyte antigen (<em>HLA</em>) alleles, and variable amino acids in HLA proteins for association with spVL in 6,315 HIV-1–infected individuals of European ancestry. We demonstrate that multiple independent signals exist at two genomic loci and implicate novel, potentially causal variants within these regions. Through heritability analysis, we estimate that the additive genetic contribution to spVL measurable through GWAS is 24.6%, the majority of which maps to variants in these two associated regions.</p></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title" style="text-transform: uppercase;">Results</h2><div id="s2" class="sec sec-first"><p></p><h3 id="s2title" class="inline">Genome-Wide Association Analysis. </h3><p id="__p10" class="p p-first">High-quality genotype data were obtained for 7,468 individuals of European ancestry from eight independent GWAS forming 10 genotype groups (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S1</a>). The phenotypic endpoint most commonly shared between contributing centers was spVL, available for 6,315 individuals. After genome-wide genotype imputation, we tested ∼8 million common variants for association with spVL per group by linear regression and combined results using inverse-variance weighted metaanalysis. We observed significant associations on chromosomes 6 and 3, with several SNPs passing the threshold of genome-wide significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>) (<a href="/pmc/articles/PMC4664299/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1</span></a>). The strongest associated SNP on chromosome 6, rs59440261 (<em>P</em> = 2.0 × 10<sup>−83</sup>), lies in the MHC regions and is in strong linkage disequilibrium (LD) with the previously reported SNP rs2395029 (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132797">3</a>) [<em>r</em><sup>2</sup> = 0.78, D′ = 1, minor allele frequency (MAF) rs59440261 = 0.06, MAF rs2395029 = 0.05]. The top chromosome 3 SNP, rs1015164 (<em>P</em> = 1.5 × 10<sup>−19</sup>), lies downstream of <em>CCRL2</em>, near an antisense transcribed sequence that overlaps chemokine (C-C motif) receptor 5 (<em>CCR5</em>) and is only weakly correlated to the <em>CCR5Δ32</em> polymorphism known to impact HIV-1 disease progression (<em>r</em><sup>2</sup> = 0.03, D′ = 0.89, MAF rs1015164 = 0.30, MAF <em>CCR5Δ32</em> = 0.10). Per-group analyses using the primary phenotypic endpoint (i.e., not necessarily spVL) (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S1</a>) did not reveal any additional associated regions, and metaanalysis of these results was consistent with analysis of spVL (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Fig. S1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig01" co-legend-rid="lgnd_fig01"><a href="/pmc/articles/PMC4664299/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140323961244352" class="figure"><a href="/pmc/articles/PMC4664299/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4664299_pnas.1514867112fig01.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pnas.1514867112fig01.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4664299/bin/pnas.1514867112fig01.jpg"></a></div><div id="largeobj_idm140323961244352" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4664299/figure/fig01/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig01"><div><a class="figpopup" href="/pmc/articles/PMC4664299/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01">Fig. 1.</a></div><!--caption a7--><div class="caption"><p id="__p11">Manhattan plot of genome-wide association results. After genotype imputation, ∼8 million common variants were tested for association with spVL in 6,315 individuals of European ancestry using linear regression. Per SNP -log<sub>10</sub>(<em>P</em> value) (<em>y</em> axis) are plotted by physical position (<em>x</em> axis). Genome-wide signals of association (<em>P</em> &lt; 5 × 10<sup>−8</sup>, dotted line) were observed on chromosomes 6 and 3. The strongest associated SNPs per region were rs59440261 on chromosome 6 (<em>P</em> = 2.0 × 10<sup>−83</sup>) and rs1015164 on chromosome 3 (<em>P</em> = 1.5 × 10<sup>−19</sup>).</p></div></div></div><p id="__p12" class="p p-last">Additionally, we performed association analyses restricting the sample to extreme phenotypes of elite control (<em>n</em> = 887 HIV-1 controllers; <em>n</em> = 2,745 noncontrollers) or disease progression (<em>n</em> = 517 rapid progressors; <em>n</em> = 467 long-term nonprogressors). Association results were comparable with those obtained in the spVL analysis, with regions on chromosomes 6 and 3 being strongly associated (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Fig. S2</a>). Thus, all further analyses were performed using the spVL phenotype.</p></div><div id="s3" class="sec"><p></p><h3 id="s3title" class="inline">Effect of Classical HLA Alleles on spVL. </h3><p id="__p13" class="p p-first">The SNP association signal on chromosome 6 centers on the class I <em>HLA</em> gene <em>HLA-B</em>, which is known to impact spVL (<a href="#r2" rid="r2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132785">2</a>, <a href="#r8" rid="r8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132784">8</a>, <a href="#r12" rid="r12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132788">12</a>). To gain a better understanding of functional variants in this region, we imputed classical class I and II <em>HLA</em> alleles, variable amino acid positions in HLA proteins, and additional single nucleotide variants. Association testing at these variants showed an increase in signal, with several variants having lower <em>P</em> values than those observed after genome-wide SNP imputation (<a href="/pmc/articles/PMC4664299/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig02" co-legend-rid="lgnd_fig02"><a href="/pmc/articles/PMC4664299/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140324007084016" class="figure"><a href="/pmc/articles/PMC4664299/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4664299_pnas.1514867112fig02.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pnas.1514867112fig02.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4664299/bin/pnas.1514867112fig02.jpg"></a></div><div id="largeobj_idm140324007084016" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4664299/figure/fig02/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig02"><div><a class="figpopup" href="/pmc/articles/PMC4664299/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02">Fig. 2.</a></div><!--caption a7--><div class="caption"><p id="__p14">Regional association plot of the chromosome 6 association peak. Association results, −log<sub>10</sub>(<em>P</em> value), for SNPs (gray circles), classical <em>HLA</em> alleles (blue boxes), and amino acids within HLA proteins (red diamonds). For biallelic markers, results were calculated by linear regression, including covariates. Association at amino acid positions with more than two alleles was calculated using a multi–degree-of-freedom omnibus test. The dashed line indicates genome-wide significance (<em>P</em> = 5 × 10<sup>−8</sup>). Amino acid position 97 (<em>P</em> = 4.6 × 10<sup>−143</sup>) in HLA-B showed the strongest association signal of any variant tested genome-wide.</p></div></div></div><p id="__p15" class="p p-last">Several classical <em>HLA-A</em>, <em>HLA-B</em>, and <em>HLA-C</em> alleles were associated with spVL, ranging in effect from strongly decreasing (notably <em>HLA-B*57:01</em>, effect size = −0.84) to strongly increasing (notably <em>HLA-B*35:02</em>, effect size = 0.36) (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S2</a>). Given the presumed benefit of recognizing multiple viral epitopes through increased diversity at <em>HLA</em> alleles, we next tested for evidence of nonadditive effects at the <em>HLA-B</em> locus. Controlling for the additive effect at each allele, we observed evidence for a general heterozygote advantage across all <em>HLA-B</em> alleles that decreased spVL (<em>P</em> = 0.016, df = 1, effect size = −0.14) (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Fig. S3</a>). Modeling per-allele nonadditive effects did not improve the fit over the general heterozygosity effect (<em>P</em> = 0.14, df = 13), and no single allele showed significant departure from additivity after accounting for multiple comparisons (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S3</a>). Additionally, testing for a multiplicative effect between all pairs of <em>HLA-B</em> alleles did not uncover any significant interactions. These data confirm a protective role for general heterozygosity at <em>HLA-B</em> beyond the individual allelic additive effects.</p></div><div id="s4" class="sec"><p></p><h3 id="s4title" class="inline">Fine Mapping of MHC Association Signals. </h3><p id="__p16" class="p p-first">Variable amino acid positions within the HLA class I proteins showed the strongest signal for association (<a href="/pmc/articles/PMC4664299/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a>). Notably, HLA-B position 97 (<em>P</em> = 4.6 × 10<sup>−143</sup>) was the strongest observed association study-wide, consistent with previous reports (<a href="#r7" rid="r7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132775">7</a>, <a href="#r8" rid="r8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132801">8</a>). To determine which amino acid positions associated independently with spVL, we performed a forward conditional regression analysis. We identified (in order) positions 97, 67, and 45 in HLA-B and positions 77 and 95 in HLA-A as independently associated with spVL (<a href="/pmc/articles/PMC4664299/table/t01/" target="table" class="fig-table-link figpopup" rid-figpopup="t01" rid-ob="ob-t01" co-legend-rid=""><span>Table 1</span></a>). These positions fall within the peptide-binding groove of the respective protein (<a href="/pmc/articles/PMC4664299/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3 <em>A</em> and <em>B</em></span></a>), and alleles at these positions had varying impact on spVL, ranging in effect from strongly decreasing to strongly increasing (<a href="/pmc/articles/PMC4664299/figure/fig03/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig03" rid-ob="ob-fig03" co-legend-rid="lgnd_fig03"><span>Fig. 3<em>C</em></span></a> and <a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S4</a>). Combining all alleles at these five positions explained 12.3% of the variance in spVL and accounted for the majority of the association signal at this locus (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Fig. S4</a>). The relationship between these amino acid positions and classical HLA alleles is listed in <a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S5</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t01"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p17">Independently associated amino acid positions in HLA proteins identified by stepwise forward conditional analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140323971712576" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1">Step</td><td rowspan="1" colspan="1" align="center">Position</td><td rowspan="1" colspan="1" align="center">Alleles<sup>*</sup></td><td rowspan="1" colspan="1" align="center">Position <em>P</em><sup>†</sup></td><td rowspan="1" colspan="1" align="center">Model <em>P</em><sup>‡</sup></td><td rowspan="1" colspan="1" align="center">Cumulative variance explained<sup>§</sup></td></tr></thead><tbody><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1" align="center">HLA-B 97</td><td rowspan="1" colspan="1" align="center">V/N/W/T/R/S</td><td rowspan="1" colspan="1" align="center">4.6 x 10<sup>−143</sup></td><td rowspan="1" colspan="1" align="center">na</td><td rowspan="1" colspan="1" align="center">0.102</td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1" align="center">HLA-B 67</td><td rowspan="1" colspan="1" align="center">Y/F/S/C/M</td><td rowspan="1" colspan="1" align="center">3.7 x 10<sup>−112</sup></td><td rowspan="1" colspan="1" align="center">3.2 x 10<sup>−15</sup></td><td rowspan="1" colspan="1" align="center">0.112</td></tr><tr><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1" align="center">HLA-B 45</td><td rowspan="1" colspan="1" align="center">E/T/K/M</td><td rowspan="1" colspan="1" align="center">8.2 x 10<sup>−49</sup></td><td rowspan="1" colspan="1" align="center">1.8 x 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">0.114</td></tr><tr><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1" align="center">HLA-A 77</td><td rowspan="1" colspan="1" align="center">N/S/D</td><td rowspan="1" colspan="1" align="center">1.8 x 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">9.4 x 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">0.122</td></tr><tr><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1" align="center">HLA-A 95</td><td rowspan="1" colspan="1" align="center">L/I/V</td><td rowspan="1" colspan="1" align="center">3.6 x 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">3.2 x 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">0.123</td></tr></tbody></table></div><div id="largeobj_idm140323971712576" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4664299/table/t01/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140323963752976"><p id="__p18" class="p p-first-last">na, not applicable.</p></div><div id="tfn1"><sup>*</sup>Per allele association statistics and frequencies are listed in <a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S4</a>.</div><div id="tfn2"><sup>†</sup>Position <em>P</em> values were calculated by a multi–degree-of-freedom omnibus test, including covariates and all alleles at that position.</div><div id="tfn3"><sup>‡</sup>Model <em>P</em> values were calculated by the likelihood ratio test comparing the model from the previous step to a model including the next position.</div><div id="tfn4"><sup>§</sup>Cumulative variance explained was calculated by linear regression and represents the variance explained by including the positions identified at each step to the model from the previous step.</div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig03" co-legend-rid="lgnd_fig03"><a href="/pmc/articles/PMC4664299/figure/fig03/" target="figure" rid-figpopup="fig03" rid-ob="ob-fig03"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140323964951888" class="figure"><a href="/pmc/articles/PMC4664299/figure/fig03/" target="figure" rid-figpopup="fig03" rid-ob="ob-fig03"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4664299_pnas.1514867112fig03.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pnas.1514867112fig03.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4664299/bin/pnas.1514867112fig03.jpg"></a></div><div id="largeobj_idm140323964951888" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4664299/figure/fig03/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig03"><div><a class="figpopup" href="/pmc/articles/PMC4664299/figure/fig03/" target="figure" rid-figpopup="fig03" rid-ob="ob-fig03">Fig. 3.</a></div><!--caption a7--><div class="caption"><p id="__p23">Location and effect of independently associated amino acids. Three-dimensional structures of (<em>A</em>) HLA-B (PDB ID code 2bvp) and (<em>B</em>) HLA-A (PDB ID code 4hwz) proteins. Conditional analysis identified five independent amino acid positions [positions 97, 67, and 45 in HLA-B and positions 77 and 95 in HLA-A (orange residues)] that line the peptide-binding groove and explain the majority of the association signal in the MHC. (<em>C</em>) Effect on spVL (i.e., change in log<sub>10</sub> HIV-1 spVL per allele copy) of individual amino acid residues at each position. Results were calculated per allele using linear regression models, including allele dosage and principal components. Gray bars indicate the estimated change in spVL per amino acid allele at each position with standard error (whiskers). All identified positions accommodate &gt;2 amino acid alleles, with allelic effects ranging from strongly protective (i.e., viral load decreasing) to deleterious (viral load increasing). Full association statistics and amino acid allele frequencies are listed in <a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S4</a>.</p></div></div></div></div><div id="s5" class="sec"><p></p><h3 id="s5title" class="inline">Fine Mapping of CCR5 Region Association Signals. </h3><p id="__p24" class="p p-first">The second highly significant signal of association centered over the <em>CCR</em> gene cluster on chromosome 3. Variation in the <em>CCR5</em> gene is known to impact HIV-1 pathogenesis (<a href="#r13" rid="r13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132746">13</a><a href="#r14" rid="r14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132794"></a><a href="#r15" rid="r15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132754"></a>–<a href="#r16" rid="r16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132779">16</a>). The strongest known causal variant in this region is <em>CCR5Δ32</em>, which is known to reduce HIV-1 susceptibility and slow disease progression (<a href="#r13" rid="r13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132762">13</a>). Additionally, the <em>CCR5</em> promoter haplotype P1 (Hap-P1) has been shown to associate with AIDS progression (<a href="#r15" rid="r15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132747">15</a>, <a href="#r17" rid="r17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132786">17</a>). To account for these effects, we restricted the conditional analysis to 5,559 individuals for whom the <em>CCR5Δ32</em> genotype was available and Hap-P1 carriage could be determined (<a href="/pmc/articles/PMC4664299/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4</span></a>). The top SNP association in this subset, rs4317138 (<em>P</em> = 7.7 × 10<sup>−22</sup>) (<a href="/pmc/articles/PMC4664299/figure/fig04/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig04" rid-ob="ob-fig04" co-legend-rid="lgnd_fig04"><span>Fig. 4</span></a>), is highly correlated to the top SNP identified in the analysis of the full sample (rs1015164, <em>r</em><sup>2</sup> = 0.97, D′ = 1, MAF rs4317138 = 0.31). Consistent with expectation, we observed a strong association between <em>CCR5Δ32</em> and reduced spVL (<em>P</em> = 1.6 × 10<sup>−16</sup>, effect size = −0.28) and between <em>CCR5</em> Hap-P1 haplotype and increased spVL (<em>P</em> = 1.8 × 10<sup>−19</sup>, effect size = 0.18).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig04" co-legend-rid="lgnd_fig04"><a href="/pmc/articles/PMC4664299/figure/fig04/" target="figure" rid-figpopup="fig04" rid-ob="ob-fig04"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140323966199392" class="figure"><a href="/pmc/articles/PMC4664299/figure/fig04/" target="figure" rid-figpopup="fig04" rid-ob="ob-fig04"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4664299_pnas.1514867112fig04.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pnas.1514867112fig04.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4664299/bin/pnas.1514867112fig04.jpg"></a></div><div id="largeobj_idm140323966199392" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4664299/figure/fig04/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig04"><div><a class="figpopup" href="/pmc/articles/PMC4664299/figure/fig04/" target="figure" rid-figpopup="fig04" rid-ob="ob-fig04">Fig. 4.</a></div><!--caption a7--><div class="caption"><p id="__p25">Regional association plot of the chromosome 3 association peak. Association results for Mb 45.5–47 (Hg19) of chromosome 3 in a subset of individuals genotyped for <em>CCR5Δ32</em> (<em>n</em> = 5,559). <em>P</em> values were calculated by linear regression, including covariates. The blue diamond, red square, and red diamond indicate the association strength of the top SNP (rs4317138, <em>P</em> = 7.7 × 10<sup>−22</sup>), Hap-P1 (<em>P</em> = 1.8 × 10<sup>−19</sup>), and <em>CCR5Δ32</em> (<em>P</em> = 1.6 × 10<sup>−16</sup>), respectively. The dashed line indicates genome-wide significance (<em>P</em> = 5 × 10<sup>−8</sup>).</p></div></div></div><p id="__p26" class="p">Conditioning on <em>CCR5Δ32</em>, 122 SNPs remained genome-wide significant (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Fig. S5</a>). The top seven SNPs are in strong LD and fall within/near an antisense transcribed sequence <em>RP11-24F11.2</em> (<em>LOC102724297</em>) that overlaps <em>CCR5</em> (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Fig. S6</a>). Conditioning on Hap-P1, these SNPs remained associated, with the strongest signal being rs1015164 (conditional <em>P</em> = 1.6 × 10<sup>−4</sup>). This SNP remained associated when conditioning on both Hap-P1 and <em>CCR5Δ32</em> (<em>P</em> = 5.2 × 10<sup>−4</sup>) (<a href="/pmc/articles/PMC4664299/table/t02/" target="table" class="fig-table-link figpopup" rid-figpopup="t02" rid-ob="ob-t02" co-legend-rid=""><span>Table 2</span></a>). Interestingly, conditioning on rs1015164 explained the observed effect of Hap-P1 (conditional <em>P</em> = 0.09) but not <em>CCR5Δ32</em> (<em>P</em> = 1.4 × 10<sup>−10</sup>), suggesting that this SNP tags additional, undescribed causal variants in this region. Taken together, these three variants explained 2.2% of the variance in spVL.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t02"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p27">Conditional association results for variants in the CCR5 region</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140323971617968" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="3" colspan="1">Variant</td><td colspan="10" rowspan="1" align="center">Condition</td></tr><tr><td colspan="2" rowspan="1" align="center">None</td><td colspan="2" rowspan="1" align="center"><em>CCR5Δ32</em></td><td colspan="2" rowspan="1" align="center">Hap-P1</td><td colspan="2" rowspan="1" align="center">rs1015164</td><td colspan="2" rowspan="1" align="center"><em>CCR5Δ32</em> and Hap-P1</td></tr><tr><td rowspan="1" colspan="1" align="center">Effect size</td><td rowspan="1" colspan="1" align="center"><em>P</em> value</td><td rowspan="1" colspan="1" align="center">Effect size</td><td rowspan="1" colspan="1" align="center"><em>P</em> value</td><td rowspan="1" colspan="1" align="center">Effect size</td><td rowspan="1" colspan="1" align="center"><em>P</em> value</td><td rowspan="1" colspan="1" align="center">Effect size</td><td rowspan="1" colspan="1" align="center"><em>P</em> value</td><td rowspan="1" colspan="1" align="center">Effect size</td><td rowspan="1" colspan="1" align="center"><em>P</em> value</td></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>CCR5Δ32</em></td><td rowspan="1" colspan="1" align="center">−0.28</td><td rowspan="1" colspan="1" align="center">1.6 x 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">na</td><td rowspan="1" colspan="1" align="center">na</td><td rowspan="1" colspan="1" align="center">−0.22</td><td rowspan="1" colspan="1" align="center">1.4 x 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">−0.22</td><td rowspan="1" colspan="1" align="center">1.4 x 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">na</td><td rowspan="1" colspan="1" align="center">na</td></tr><tr><td rowspan="1" colspan="1">Hap-P1</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">1.8 x 10<sup>−19</sup></td><td rowspan="1" colspan="1" align="center">0.15</td><td rowspan="1" colspan="1" align="center">1.4 x 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">na</td><td rowspan="1" colspan="1" align="center">na</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1" align="center">0.09</td><td rowspan="1" colspan="1" align="center">na</td><td rowspan="1" colspan="1" align="center">na</td></tr><tr><td rowspan="1" colspan="1">rs1015164(A)</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">1.5 x 10<sup>−21</sup></td><td rowspan="1" colspan="1" align="center">0.20</td><td rowspan="1" colspan="1" align="center">1.2 x 10<sup>−15</sup></td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">1.6 x 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">na</td><td rowspan="1" colspan="1" align="center">na</td><td rowspan="1" colspan="1" align="center">0.15</td><td rowspan="1" colspan="1" align="center">5.2 x 10<sup>−4</sup></td></tr></tbody></table></div><div id="largeobj_idm140323971617968" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4664299/table/t02/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140323959687392"><p id="__p28" class="p p-first-last">Effect size and <em>P</em> values were calculated using linear regression, including covariates to adjust for population structure and, where applicable, the variant/haplotype dosage (condition).</p></div></div></div></div><div id="s6" class="sec sec-last"><p></p><h3 id="s6title" class="inline">Assessing Narrow-Sense Heritability of HIV-1 spVL. </h3><p id="__p29" class="p p-first-last">Combining the effects of the independently associated common variants in the HLA and <em>CCR5</em> region explained 14.5% of the variability in spVL. We used genome-wide complex trait analysis (GCTA) (<a href="#r18" rid="r18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132790">18</a>) to address the extent to which additional, additive genetic factors may influence spVL and observed that genome-wide variation explains 24.6% [standard deviation (SD) = 3%] of the narrow-sense heritability (i.e., additive effects). We assessed the sensitivity of this estimate to potential overfitting by verifying that a randomly permuted phenotype vector (30 permutations) showed zero heritability. This genome-wide estimate decreased to 5.5% (SD = 3%) after controlling for the effects in the MHC/CCR5 regions. A series of analyses where we randomly selected two-thirds of all available samples supported this estimate (median 5% heritability, 6.9% interquartile range). Additionally, a complementary analysis using a polygenic score test demonstrated a similar lack of contribution from variants outside of the MHC and CCR5 regions (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Fig. S7</a>). These results suggest that the identified common variants of large effect explain the majority of the host genetic component of HIV-1 spVL.</p></div></div><div id="s7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s7title" style="text-transform: uppercase;">Discussion</h2><p id="__p30" class="p p-first">Previous GWAS of HIV-1 control and disease progression lacked power to detect variants with modest effect sizes. By combining available genome-wide genotypes and clinical data from 6,315 HIV-1–infected individuals, we sought to get a more complete picture of the impact of common human genetic variation on HIV-1 disease across a range of effect sizes.</p><p id="__p31">The MHC region demonstrated the strongest signal of association with spVL, with multiple, independent common variants of large effect mapping to this region. The long-range LD structure and high gene density (including many immunologically relevant genes) of the MHC make it impossible to definitively assign causality to any particular variant through purely statistical methods. However, the abundance of functional evidence and the centrality of the association signal in this study point to the class I HLA genes and, in particular, to <em>HLA-B</em> as being causal. Here, we observed strong associations between spVL and multiple alleles at <em>HLA-A, -B</em>, and <em>-C</em> over a broad range of effect sizes. Consistent with previous results (<a href="#r19" rid="r19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132745">19</a>), we observed evidence for a heterozygote advantage at the <em>HLA-B</em> locus. The comparatively weak statistical strength we report here may be due to methodological differences because (<em>i</em>) we control for additive effects at each allele and (<em>ii</em>) the larger sample size allows the consideration of an increased number of homozygous genotypes, reducing bias due to the low frequency (and thus increased proportion of heterozygosity) of strongly protective alleles. Thus, our results may more accurately reflect the true heterozygous effect.</p><p id="__p32">By testing variant amino acid positions in classical HLA proteins, we confirmed the strong associations at positions 97 and 67 in HLA-B and observed additional signals at position 45 in HLA-B and positions 77 and 95 in HLA-A. The location of these amino acids in the peptide-binding groove of the respective proteins supports the hypothesis that the presentation of specific viral epitopes, directly dependent on the shape of the HLA peptide binding groove, is critical in determining the efficiency of the cytotoxic T-cell response. In addition to peptide presentation, HLA-C expression levels (<a href="#r20" rid="r20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132757">20</a>) and variation in non-<em>HLA</em> genes in the MHC region (<a href="#r21" rid="r21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132761">21</a>) have been proposed as impacting HIV-1 control. Detailed functional analyses of these effects will be required to fully understand the extent of the influence of MHC variation on the natural history of HIV-1 disease.</p><p id="__p33">Although the impact of <em>CCR5Δ32</em> on HIV-1 acquisition and disease progression has been well-described, this association has not been previously identified through GWAS. This lack of detection is likely due to the relatively limited LD between common SNPs and the <em>CCR5Δ32</em> allele. Indeed, the top SNP identified on chromosome 3 in the full sample, rs1015164, is only weakly correlated to <em>CCR5Δ32</em> (<em>r</em><sup>2</sup> = 0.03). Conditional analysis showed that several SNPs in this region were independently associated after controlling for the known effects of <em>CCR5Δ32</em> and Hap-P1. These SNPs are located within/near an antisense transcribed sequence that overlaps <em>CCR5</em> and thus may play a role in regulating its expression. Demonstration of causality of these variants and/or a silencing effect of the antisense transcribed sequence will require functional studies.</p><p id="__p34">Measurable narrow-sense heritability attributable to non–genome-wide significant loci has been demonstrated for multiple complex traits (<a href="#r22" rid="r22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132792">22</a>, <a href="#r23" rid="r23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132776">23</a>). Using genome-wide variants, we estimated that additive host genetic effects explain approximately one-quarter of the variance in HIV-1 spVL. However, after controlling for the genome-wide significant signals, the remainder of the genome explained only ∼5%. This limited residual heritability underscores the primary role of common variants of large effects in the MHC and <em>CCR5</em> in HIV-1 control. Interestingly, analyses aimed at estimating the viral genetic component of heritability have been generally higher, ∼30–50%, than our estimated host component (<a href="#r24" rid="r24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568138385">24</a>). However, it is difficult to disentangle these two values because host genetic variation, in particular the class I <em>HLA</em> region, exerts substantial pressure on the viral genetic sequence (<a href="#r25" rid="r25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132744">25</a>). Indeed, if the influence of host and viral genetics highly overlaps, up to an additional 70% of variability in spVL may remain unaccounted for. In addition to known nongenetic factors that impact spVL, such as age and sex, host genetic factors not measured by this study design (e.g., somatic recombination of T- and B-cell receptors, copy number variation, and rare variation) may also explain a substantial proportion of the variation. Comprehensive, joint analysis of the host and viral genetic components of spVL variation in large samples will also be of great interest due to the high sensitivity of HIV to reflect variation in the host environment.</p><p id="__p35">For single variant analysis, this study had ∼80% power to detect common variants (at 10% frequency) that explain &gt;0.5% of the variability in spVL. This level of sensitivity suggests that previous candidate gene studies that have claimed associations with spVL (of variants with relatively large effect size) are unlikely to be valid, given their lack of replication in the present study. This observation is consistent with previous GWAS that have directly examined (and failed to replicate) a number of these associations (<a href="#r2" rid="r2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132777">2</a>, <a href="#r8" rid="r8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132760">8</a>).</p><p id="__p36" class="p p-last">The results presented herein combine the majority of genetic data available on untreated HIV-1–infected individuals of European ancestry. Because a substantial increase in sample size is unrealistic, because of current antiretroviral treatment guidelines (<a href="#r26" rid="r26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132800">26</a>), additional GWAS in this population are unlikely to provide further insight into the genetic architecture of HIV-1 control. Thus, studies in non-European populations, which heretofore have been underrepresented in GWAS, as well as investigations of other classes of genetic variation and genome-wide nonadditive and/or epistatic effects, should now be clear priorities in the field.</p></div><div id="s8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s8title" style="text-transform: uppercase;">Methods</h2><div id="s9" class="sec sec-first"><p></p><h3 id="s9title" class="inline">Ethics Statement. </h3><p id="__p37" class="p p-first-last">All participants were HIV-1–infected adults, and written informed consent for genetic testing was obtained from all individuals as part of the original study in which they were enrolled (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, <em>Note S1</em></a>). Ethical approval was obtained from institutional review boards for each of the respective contributing centers.</p></div><div id="s10" class="sec"><p></p><h3 id="s10title" class="inline">Samples and Contributing Centers. </h3><p id="__p38" class="p p-first-last">DNA samples obtained from 21 individual cohorts or centers were genotyped as part of eight independent GWAS using various genotyping platforms (<a href="#r2" rid="r2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132753">2</a>, <a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132799">5</a>, <a href="#r6" rid="r6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132749">6</a>, <a href="#r8" rid="r8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132766">8</a><a href="#r9" rid="r9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132765"></a>–<a href="#r10" rid="r10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132796">10</a>, <a href="#r27" rid="r27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132793">27</a>, <a href="#r28" rid="r28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132774">28</a>) and combined as part of the International Collaboration for the Genomics of HIV (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S1 and <em>Note S1</em></a>). All individuals were infected with HIV-1 and had phenotypic data relevant to viral control or disease progression. Primary phenotypes included spVL, long-term nonprogression, and elite control of HIV-1 viremia. The phenotype most commonly available was spVL (<em>n</em> = 6,315), which was used for the primary analysis. Additional analyses were performed on extreme phenotypes (elite control, long-term nonprogression, and rapid progression) and are presented in <a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em></a>.</p></div><div id="s11" class="sec"><p></p><h3 id="s11title" class="inline">Genotype Quality Control and Imputation. </h3><p id="__p39" class="p p-first">All quality control steps were performed per study using PLINK version 1.07 (<a href="#r29" rid="r29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132772">29</a>). Genotype data were combined based on geographic origin of the samples and/or genotyping platform, resulting in 10 genotype groups (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S1</a>). Ancestry was inferred by principal components analysis using EIGENSTRAT (<a href="#r30" rid="r30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132770">30</a>), taking the HapMap 3 (<a href="#r31" rid="r31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132771">31</a>) sample as a reference. Only samples clustering with the HapMap Europeans were included. Study participants were excluded based on the following criteria: identity-by-descent of &gt;0.125 (one individual per pair was removed), missingness of &gt;2%, and inbreeding coefficients of &lt;−0.1 or &gt;0.1. SNPs were removed based on missingness of &gt;5%, MAF of &lt;1%, or Hardy–Weinberg equilibrium of <em>P</em> &lt; 1 × 10<sup>−7</sup>.</p><p id="__p40" class="p p-last">Per group, genotypes for additional polymorphisms not directly assessed by the original genotyping platform were inferred using haplotype information (i.e., imputation of missing genotypes) (<a href="#r32" rid="r32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132783">32</a>) from the 1,000 Genomes Project Phase 1 v3 reference panel. Genotypes were prephased with mach v1 (<a href="#r33" rid="r33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132780">33</a>) and imputed using minimac (<a href="#r34" rid="r34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132768">34</a>). An additional imputation protocol using shapeit v2 (<a href="#r35" rid="r35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132763">35</a>, <a href="#r36" rid="r36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132748">36</a>) and impute2 (<a href="#r37" rid="r37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132781">37</a>) was also implemented with highly concordant results. Imputed SNPs having a reported <em>r</em><sup>2</sup> score of &lt;0.3 and minor allele frequency of &lt;0.5% were excluded from downstream analysis.</p></div><div id="s12" class="sec"><p></p><h3 id="s12title" class="inline">Association Testing and Metaanalysis. </h3><p id="__p41" class="p p-first-last">Single marker association tests were performed per genotype group regressing spVL on variant dosage using linear regression including principal components (PCs) to correct for population structure (<a href="#r30" rid="r30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132789">30</a>). In all cases, inclusion of PCs was sufficient to control for genomic inflation (lambda of ∼1) (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S1</a>). Results were combined across genotype groups using inverse-variance weighted metaanalysis (<a href="#r38" rid="r38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132769">38</a>). In some cases, the primary endpoint for the original study was a binary trait (<a href="#d35e1085" rid="d35e1085" class=" supplementary-material"><em>SI Appendix</em>, Table S1</a>). For these cohorts, we also tested the binary phenotype for association using logistic regression, including covariates as above and metaanalyzed across binary and quantitative endpoints using z-scores weighted by the group sample size. Power for detection of single variants was estimated using the genetic power calculator for quantitative traits (<a href="#r39" rid="r39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132764">39</a>).</p></div><div id="s13" class="sec"><p></p><h3 id="s13title" class="inline">Imputation and Association Testing in the MHC Region. </h3><p id="__p42" class="p p-first-last">Classical <em>HLA</em> alleles, variant amino acids within HLA proteins, and additional SNPs in the MHC were imputed using the SNP2HLA pipeline, with a reference panel consisting of 5,225 individuals of European ancestry from the Type 1 Diabetes Genetics Consortium (<a href="#r40" rid="r40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132787">40</a>). Classical alleles and binary amino acid positions were individually tested for association using linear regression corrected for PCs and study-specific effects. Association was tested at multiallelic amino acid positions (i.e., three or more possible states) using a multi–degree-of-freedom omnibus test including covariates as above.</p></div><div id="s14" class="sec"><p></p><h3 id="s14title" class="inline">Testing for Nonadditive Effects of HLA-B Alleles. </h3><p id="__p43" class="p p-first-last">Evidence of nonadditive effects at the <em>HLA-B</em> locus was assessed in a subset of individuals (<em>n</em> = 3,882) that carried two common alleles (minimum of five homozygous observations, <em>n</em> = 14 alleles). We first compared a model that included covariates (PCs and genotype group) and additive effects for each classical allele to a model that additionally included a heterozygosity effect; this approach is equivalent to having a general dominance term across all alleles. We similarly assessed the nonadditive effect of each allele individually. To estimate effect sizes of homozygote and heterozygote genotypes on spVL, we constructed additive models after excluding all homozygous individuals (for heterozygous effects) or excluding all heterozygous individuals (for homozygous effects). Interactions between specific alleles were assessed using models that contained additive terms for each allele and interaction terms between each pair of alleles.</p></div><div id="s15" class="sec"><p></p><h3 id="s15title" class="inline">Fine Mapping of Associated Regions. </h3><p id="__p44" class="p p-first-last">To identify independent variants in associated regions, we used step-wise forward conditional testing, including covariates as above. In the MHC, due to the presence of multiallelic variants (i.e., &gt;2 states), we used the likelihood ratio test (LRT). A position was considered independently associated if its addition to the model improved the fit after correcting for the total number of amino acids considered (LRT of <em>P</em> &gt; 2 × 10<sup>−4</sup>). In the <em>CCR5</em> region, conditional analysis was restricted to a subset of 5,559 participants genotyped for <em>CCR5Δ32</em> and for whom the <em>CCR5</em> Hap-P1 haplotype could be inferred (<a href="#r15" rid="r15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132778">15</a>, <a href="#r17" rid="r17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132751">17</a>, <a href="#r41" rid="r41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132758">41</a>). Variance explained by independently associated variants was calculated by comparing the adjusted <em>r</em><sup>2</sup> values from linear regression models, including covariates alone to one containing covariates and the selected variants.</p></div><div id="s16" class="sec sec-last"><p></p><h3 id="s16title" class="inline">Assessment of Narrow-Sense Heritability of spVL. </h3><p id="__p45" class="p p-first-last">Heritability analysis was conducted with the GCTA software package (<a href="#r18" rid="r18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_494132791">18</a>) using common variants (MAF of &gt;1%), which were accurately imputed in at least 99% of samples, pruned based on LD (<em>r</em><sup>2</sup> &lt; 0.1). To avoid deflation of the total heritability estimate, the independently associated variants from the conditional analysis were also included. To reduce bias due to nonnormally distributed spVL measurements, cohorts enriched for HIV-1 controllers were removed. To empirically assess the error of the estimated variance component, we performed the analyses on 30 bootstrap replicates, by resampling the included individuals with replacement. To check for potential overfitting, we performed heritability analyses on 30 random assignments of the phenotypes to the genotypes. We assessed the effect of sample size by repeating the analysis over a grid of different sample sizes.</p></div></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="d35e1085"><h4>Supplementary File</h4><div class="sup-box half_rhythm" id="d35e1085a1051501"><a href="/pmc/articles/PMC4664299/bin/pnas.1514867112.sapp.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(5.1M, pdf)</sup></div></div></div><div id="idm140323959701472" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140323959701472title" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p id="__p46">We thank Stuart Z. Shapiro (Program Officer, Division of AIDS, National Institute of Allergy and Infectious Diseases) and Stacy Carrington-Lawrence (Chair of Etiology and Pathogenesis, NIH Office of AIDS Research) for continued support. A portion of the computations were performed at the Vital-IT (<a href="http://www.vital-it.ch/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.vital-it.ch</a>) Center for high-performance computing of the SIB Swiss Institute of Bioinformatics.</p></div></div><div id="idm140323959699424" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140323959699424title" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140323965356720"> </p><p id="__p47" class="p p-first-last">The authors declare no conflict of interest.</p><p></p><p class="fn sec" id="idm140323965355856"></p><p id="__p48" class="p p-first-last">This article is a PNAS Direct Submission.</p><p></p><p class="fn sec" id="idm140323965354992"></p><p id="__p49" class="p p-first-last">This article contains supporting information online at <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1514867112/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.pnas.org/lookup/suppl/doi:10.1073/pnas.1514867112/-/DCSupplemental</a>.</p><p></p></div></div><div id="idm140323965353024" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140323965353024title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="r1">1. <span class="element-citation">Mellors JW, et al.  Quantitation of HIV-1 RNA in plasma predicts outcome after seroconversion. <span><span class="ref-journal">Ann Intern Med. </span>1995;<span class="ref-vol">122</span>(8):573–579.</span> [<a href="/pubmed/7887550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Intern+Med&amp;title=Quantitation+of+HIV-1+RNA+in+plasma+predicts+outcome+after+seroconversion&amp;author=JW+Mellors&amp;volume=122&amp;issue=8&amp;publication_year=1995&amp;pages=573-579&amp;pmid=7887550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r2">2. <span class="element-citation">Fellay J, et al.  NIAID Center for HIV/AIDS Vaccine Immunology (CHAVI)  Common genetic variation and the control of HIV-1 in humans. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>(12):e1000791.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2791220/">PMC free article</a>]</span> [<a href="/pubmed/20041166" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Common+genetic+variation+and+the+control+of+HIV-1+in+humans&amp;author=J+Fellay&amp;volume=5&amp;issue=12&amp;publication_year=2009&amp;pages=e1000791&amp;pmid=20041166&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r3">3. <span class="element-citation">Fellay J, et al.  A whole-genome association study of major determinants for host control of HIV-1. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">317</span>(5840):944–947.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1991296/">PMC free article</a>]</span> [<a href="/pubmed/17641165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+whole-genome+association+study+of+major+determinants+for+host+control+of+HIV-1&amp;author=J+Fellay&amp;volume=317&amp;issue=5840&amp;publication_year=2007&amp;pages=944-947&amp;pmid=17641165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r4">4. <span class="element-citation">Pelak K, et al.  Infectious Disease Clinical Research Program HIV Working Group  National Institute of Allergy and Infectious Diseases Center for HIV/AIDS Vaccine Immunology (CHAVI)  Host determinants of HIV-1 control in African Americans. <span><span class="ref-journal">J Infect Dis. </span>2010;<span class="ref-vol">201</span>(8):1141–1149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2838940/">PMC free article</a>]</span> [<a href="/pubmed/20205591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Host+determinants+of+HIV-1+control+in+African+Americans&amp;author=K+Pelak&amp;volume=201&amp;issue=8&amp;publication_year=2010&amp;pages=1141-1149&amp;pmid=20205591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r5">5. <span class="element-citation">Dalmasso C, et al.  ANRS Genome Wide Association 01  Distinct genetic loci control plasma HIV-RNA and cellular HIV-DNA levels in HIV-1 infection: The ANRS Genome Wide Association 01 study. <span><span class="ref-journal">PLoS One. </span>2008;<span class="ref-vol">3</span>(12):e3907.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2603319/">PMC free article</a>]</span> [<a href="/pubmed/19107206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Distinct+genetic+loci+control+plasma+HIV-RNA+and+cellular+HIV-DNA+levels+in+HIV-1+infection:+The+ANRS+Genome+Wide+Association+01+study&amp;author=C+Dalmasso&amp;volume=3&amp;issue=12&amp;publication_year=2008&amp;pages=e3907&amp;pmid=19107206&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r6">6. <span class="element-citation">van Manen D, et al.  Genome-wide association scan in HIV-1-infected individuals identifying variants influencing disease course. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>(7):e22208.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3141012/">PMC free article</a>]</span> [<a href="/pubmed/21811574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Genome-wide+association+scan+in+HIV-1-infected+individuals+identifying+variants+influencing+disease+course&amp;author=D+van+Manen&amp;volume=6&amp;issue=7&amp;publication_year=2011&amp;pages=e22208&amp;pmid=21811574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r7">7. <span class="element-citation">McLaren PJ, et al.  International HIV Controllers Study  Fine-mapping classical HLA variation associated with durable host control of HIV-1 infection in African Americans. <span><span class="ref-journal">Hum Mol Genet. </span>2012;<span class="ref-vol">21</span>(19):4334–4347.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3441117/">PMC free article</a>]</span> [<a href="/pubmed/22718199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Fine-mapping+classical+HLA+variation+associated+with+durable+host+control+of+HIV-1+infection+in+African+Americans&amp;author=PJ+McLaren&amp;volume=21&amp;issue=19&amp;publication_year=2012&amp;pages=4334-4347&amp;pmid=22718199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r8">8. <span class="element-citation">Pereyra F, et al.  International HIV Controllers Study  The major genetic determinants of HIV-1 control affect HLA class I peptide presentation. <span><span class="ref-journal">Science. </span>2010;<span class="ref-vol">330</span>(6010):1551–1557.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3235490/">PMC free article</a>]</span> [<a href="/pubmed/21051598" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+major+genetic+determinants+of+HIV-1+control+affect+HLA+class+I+peptide+presentation&amp;author=F+Pereyra&amp;volume=330&amp;issue=6010&amp;publication_year=2010&amp;pages=1551-1557&amp;pmid=21051598&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r9">9. <span class="element-citation">Limou S, et al.  ANRS Genomic Group  Genomewide association study of an AIDS-nonprogression cohort emphasizes the role played by HLA genes (ANRS Genomewide Association Study 02) <span><span class="ref-journal">J Infect Dis. </span>2009;<span class="ref-vol">199</span>(3):419–426.</span> [<a href="/pubmed/19115949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Genomewide+association+study+of+an+AIDS-nonprogression+cohort+emphasizes+the+role+played+by+HLA+genes+(ANRS+Genomewide+Association+Study+02)&amp;author=S+Limou&amp;volume=199&amp;issue=3&amp;publication_year=2009&amp;pages=419-426&amp;pmid=19115949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r10">10. <span class="element-citation">Le Clerc S, et al.  ANRS Genomic Group  Genomewide association study of a rapid progression cohort identifies new susceptibility alleles for AIDS (ANRS Genomewide Association Study 03) <span><span class="ref-journal">J Infect Dis. </span>2009;<span class="ref-vol">200</span>(8):1194–1201.</span> [<a href="/pubmed/19754311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Genomewide+association+study+of+a+rapid+progression+cohort+identifies+new+susceptibility+alleles+for+AIDS+(ANRS+Genomewide+Association+Study+03)&amp;author=S+Le+Clerc&amp;volume=200&amp;issue=8&amp;publication_year=2009&amp;pages=1194-1201&amp;pmid=19754311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r11">11. <span class="element-citation">McLaren PJ, et al.  Association study of common genetic variants and HIV-1 acquisition in 6,300 infected cases and 7,200 controls. <span><span class="ref-journal">PLoS Pathog. </span>2013;<span class="ref-vol">9</span>(7):e1003515.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3723635/">PMC free article</a>]</span> [<a href="/pubmed/23935489" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Association+study+of+common+genetic+variants+and+HIV-1+acquisition+in+6,300+infected+cases+and+7,200+controls&amp;author=PJ+McLaren&amp;volume=9&amp;issue=7&amp;publication_year=2013&amp;pages=e1003515&amp;pmid=23935489&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r12">12. <span class="element-citation">Migueles SA, et al.  HLA B*5701 is highly associated with restriction of virus replication in a subgroup of HIV-infected long term nonprogressors. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2000;<span class="ref-vol">97</span>(6):2709–2714.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC15994/">PMC free article</a>]</span> [<a href="/pubmed/10694578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=HLA+B*5701+is+highly+associated+with+restriction+of+virus+replication+in+a+subgroup+of+HIV-infected+long+term+nonprogressors&amp;author=SA+Migueles&amp;volume=97&amp;issue=6&amp;publication_year=2000&amp;pages=2709-2714&amp;pmid=10694578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r13">13. <span class="element-citation">Dean M, et al.  Genetic restriction of HIV-1 infection and progression to AIDS by a deletion allele of the CKR5 structural gene: Hemophilia Growth and Development Study, Multicenter AIDS Cohort Study, Multicenter Hemophilia Cohort Study, San Francisco City Cohort, ALIVE Study. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">273</span>(5283):1856–1862.</span> [<a href="/pubmed/8791590" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic+restriction+of+HIV-1+infection+and+progression+to+AIDS+by+a+deletion+allele+of+the+CKR5+structural+gene:+Hemophilia+Growth+and+Development+Study,+Multicenter+AIDS+Cohort+Study,+Multicenter+Hemophilia+Cohort+Study,+San+Francisco+City+Cohort,+ALIVE+Study&amp;author=M+Dean&amp;volume=273&amp;issue=5283&amp;publication_year=1996&amp;pages=1856-1862&amp;pmid=8791590&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r14">14. <span class="element-citation">Gonzalez E, et al.  Race-specific HIV-1 disease-modifying effects associated with CCR5 haplotypes. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1999;<span class="ref-vol">96</span>(21):12004–12009.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC18402/">PMC free article</a>]</span> [<a href="/pubmed/10518566" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Race-specific+HIV-1+disease-modifying+effects+associated+with+CCR5+haplotypes&amp;author=E+Gonzalez&amp;volume=96&amp;issue=21&amp;publication_year=1999&amp;pages=12004-12009&amp;pmid=10518566&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r15">15. <span class="element-citation">Martin MP, et al.  Genetic acceleration of AIDS progression by a promoter variant of CCR5. <span><span class="ref-journal">Science. </span>1998;<span class="ref-vol">282</span>(5395):1907–1911.</span> [<a href="/pubmed/9836644" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic+acceleration+of+AIDS+progression+by+a+promoter+variant+of+CCR5&amp;author=MP+Martin&amp;volume=282&amp;issue=5395&amp;publication_year=1998&amp;pages=1907-1911&amp;pmid=9836644&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r16">16. <span class="element-citation">Smith MW, et al.  Contrasting genetic influence of CCR2 and CCR5 variants on HIV-1 infection and disease progression: Hemophilia Growth and Development Study (HGDS), Multicenter AIDS Cohort Study (MACS), Multicenter Hemophilia Cohort Study (MHCS), San Francisco City Cohort (SFCC), ALIVE Study. <span><span class="ref-journal">Science. </span>1997;<span class="ref-vol">277</span>(5328):959–965.</span> [<a href="/pubmed/9252328" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Contrasting+genetic+influence+of+CCR2+and+CCR5+variants+on+HIV-1+infection+and+disease+progression:+Hemophilia+Growth+and+Development+Study+(HGDS),+Multicenter+AIDS+Cohort+Study+(MACS),+Multicenter+Hemophilia+Cohort+Study+(MHCS),+San+Francisco+City+Cohort+(SFCC),+ALIVE+Study&amp;author=MW+Smith&amp;volume=277&amp;issue=5328&amp;publication_year=1997&amp;pages=959-965&amp;pmid=9252328&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r17">17. <span class="element-citation">McDermott DH, et al.  Multicenter AIDS Cohort Study (MACS)  CCR5 promoter polymorphism and HIV-1 disease progression. <span><span class="ref-journal">Lancet. </span>1998;<span class="ref-vol">352</span>(9131):866–870.</span> [<a href="/pubmed/9742978" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=CCR5+promoter+polymorphism+and+HIV-1+disease+progression&amp;author=DH+McDermott&amp;volume=352&amp;issue=9131&amp;publication_year=1998&amp;pages=866-870&amp;pmid=9742978&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r18">18. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: A tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>(1):76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+A+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;issue=1&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r19">19. <span class="element-citation">Carrington M, et al.  HLA and HIV-1: Heterozygote advantage and B*35-Cw*04 disadvantage. <span><span class="ref-journal">Science. </span>1999;<span class="ref-vol">283</span>(5408):1748–1752.</span> [<a href="/pubmed/10073943" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=HLA+and+HIV-1:+Heterozygote+advantage+and+B*35-Cw*04+disadvantage&amp;author=M+Carrington&amp;volume=283&amp;issue=5408&amp;publication_year=1999&amp;pages=1748-1752&amp;pmid=10073943&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r20">20. <span class="element-citation">Apps R, et al.  Influence of HLA-C expression level on HIV control. <span><span class="ref-journal">Science. </span>2013;<span class="ref-vol">340</span>(6128):87–91.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3784322/">PMC free article</a>]</span> [<a href="/pubmed/23559252" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Influence+of+HLA-C+expression+level+on+HIV+control&amp;author=R+Apps&amp;volume=340&amp;issue=6128&amp;publication_year=2013&amp;pages=87-91&amp;pmid=23559252&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r21">21. <span class="element-citation">Le Clerc S, et al.  Evidence after imputation for a role of MICA variants in nonprogression and elite control of HIV type 1 Infection. <span><span class="ref-journal">J Infect Dis. </span>2014;<span class="ref-vol">210</span>(12):1946–1950.</span> [<a href="/pubmed/24939907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Evidence+after+imputation+for+a+role+of+MICA+variants+in+nonprogression+and+elite+control+of+HIV+type+1+Infection&amp;author=S+Le+Clerc&amp;volume=210&amp;issue=12&amp;publication_year=2014&amp;pages=1946-1950&amp;pmid=24939907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r22">22. <span class="element-citation">Gusev A, et al.  Schizophrenia Working Group of the Psychiatric Genomics Consortium; SWE-SCZ Consortium; Schizophrenia Working Group of the Psychiatric Genomics Consortium; SWE-SCZ Consortium  Partitioning heritability of regulatory and cell-type-specific variants across 11 common diseases. <span><span class="ref-journal">Am J Hum Genet. </span>2014;<span class="ref-vol">95</span>(5):535–552.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4225595/">PMC free article</a>]</span> [<a href="/pubmed/25439723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Partitioning+heritability+of+regulatory+and+cell-type-specific+variants+across+11+common+diseases&amp;author=A+Gusev&amp;volume=95&amp;issue=5&amp;publication_year=2014&amp;pages=535-552&amp;pmid=25439723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r23">23. <span class="element-citation">Yang J, et al.  Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>(7):565–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J+Yang&amp;volume=42&amp;issue=7&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r24">24. <span class="element-citation">Fraser C, et al.  Virulence and pathogenesis of HIV-1 infection: An evolutionary perspective. <span><span class="ref-journal">Science. </span>2014;<span class="ref-vol">343</span>(6177):1243727.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5034889/">PMC free article</a>]</span> [<a href="/pubmed/24653038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Virulence+and+pathogenesis+of+HIV-1+infection:+An+evolutionary+perspective&amp;author=C+Fraser&amp;volume=343&amp;issue=6177&amp;publication_year=2014&amp;pages=1243727&amp;pmid=24653038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r25">25. <span class="element-citation">Bartha I, et al.  A genome-to-genome analysis of associations between human genetic variation, HIV-1 sequence diversity, and viral control. <span><span class="ref-journal">eLife. </span>2013;<span class="ref-vol">2</span>:e01123.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3807812/">PMC free article</a>]</span> [<a href="/pubmed/24171102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=A+genome-to-genome+analysis+of+associations+between+human+genetic+variation,+HIV-1+sequence+diversity,+and+viral+control&amp;author=I+Bartha&amp;volume=2&amp;publication_year=2013&amp;pages=e01123&amp;pmid=24171102&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r26">26. <span class="element-citation">Günthard HF, et al.  International Antiviral Society-USA Panel  Antiretroviral treatment of adult HIV infection: 2014 recommendations of the International Antiviral Society-USA Panel. <span><span class="ref-journal">JAMA. </span>2014;<span class="ref-vol">312</span>(4):410–425.</span> [<a href="/pubmed/25038359" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Antiretroviral+treatment+of+adult+HIV+infection:+2014+recommendations+of+the+International+Antiviral+Society-USA+Panel&amp;author=HF+Günthard&amp;volume=312&amp;issue=4&amp;publication_year=2014&amp;pages=410-425&amp;pmid=25038359&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r27">27. <span class="element-citation">Herbeck JT, et al.  Multistage genomewide association study identifies a locus at 1q41 associated with rate of HIV-1 disease progression to clinical AIDS. <span><span class="ref-journal">J Infect Dis. </span>2010;<span class="ref-vol">201</span>(4):618–626.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928718/">PMC free article</a>]</span> [<a href="/pubmed/20064070" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Multistage+genomewide+association+study+identifies+a+locus+at+1q41+associated+with+rate+of+HIV-1+disease+progression+to+clinical+AIDS&amp;author=JT+Herbeck&amp;volume=201&amp;issue=4&amp;publication_year=2010&amp;pages=618-626&amp;pmid=20064070&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r28">28. <span class="element-citation">Troyer JL, et al.  Genome-wide association study implicates PARD3B-based AIDS restriction. <span><span class="ref-journal">J Infect Dis. </span>2011;<span class="ref-vol">203</span>(10):1491–1502.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080910/">PMC free article</a>]</span> [<a href="/pubmed/21502085" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Genome-wide+association+study+implicates+PARD3B-based+AIDS+restriction&amp;author=JL+Troyer&amp;volume=203&amp;issue=10&amp;publication_year=2011&amp;pages=1491-1502&amp;pmid=21502085&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r29">29. <span class="element-citation">Purcell S, et al.  PLINK: A tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>(3):559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+A+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r30">30. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>(8):904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;issue=8&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r31">31. <span class="element-citation">Altshuler DM, et al.  International HapMap 3 Consortium  Integrating common and rare genetic variation in diverse human populations. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>(7311):52–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173859/">PMC free article</a>]</span> [<a href="/pubmed/20811451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrating+common+and+rare+genetic+variation+in+diverse+human+populations&amp;author=DM+Altshuler&amp;volume=467&amp;issue=7311&amp;publication_year=2010&amp;pages=52-58&amp;pmid=20811451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r32">32. <span class="element-citation">Marchini J, Howie B. Genotype imputation for genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>(7):499–511.</span> [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J+Marchini&amp;author=B+Howie&amp;volume=11&amp;issue=7&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r33">33. <span class="element-citation">Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. MaCH: Using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2010;<span class="ref-vol">34</span>(8):816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=MaCH:+Using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y+Li&amp;author=CJ+Willer&amp;author=J+Ding&amp;author=P+Scheet&amp;author=GR+Abecasis&amp;volume=34&amp;issue=8&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r34">34. <span class="element-citation">Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>(8):955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B+Howie&amp;author=C+Fuchsberger&amp;author=M+Stephens&amp;author=J+Marchini&amp;author=GR+Abecasis&amp;volume=44&amp;issue=8&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r35">35. <span class="element-citation">Delaneau O, Marchini J, Zagury JF. A linear complexity phasing method for thousands of genomes. <span><span class="ref-journal">Nat Methods. </span>2012;<span class="ref-vol">9</span>(2):179–181.</span> [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes&amp;author=O+Delaneau&amp;author=J+Marchini&amp;author=JF+Zagury&amp;volume=9&amp;issue=2&amp;publication_year=2012&amp;pages=179-181&amp;pmid=22138821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r36">36. <span class="element-citation">Delaneau O, Zagury JF, Marchini J. Improved whole-chromosome phasing for disease and population genetic studies. <span><span class="ref-journal">Nat Methods. </span>2013;<span class="ref-vol">10</span>(1):5–6.</span> [<a href="/pubmed/23269371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Improved+whole-chromosome+phasing+for+disease+and+population+genetic+studies&amp;author=O+Delaneau&amp;author=JF+Zagury&amp;author=J+Marchini&amp;volume=10&amp;issue=1&amp;publication_year=2013&amp;pages=5-6&amp;pmid=23269371&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r37">37. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>(6):e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;issue=6&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r38">38. <span class="element-citation">de Bakker PI, et al.  Practical aspects of imputation-driven meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>(R2):R122–R128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782358/">PMC free article</a>]</span> [<a href="/pubmed/18852200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Practical+aspects+of+imputation-driven+meta-analysis+of+genome-wide+association+studies&amp;author=PI+de+Bakker&amp;volume=17&amp;issue=R2&amp;publication_year=2008&amp;pages=R122-R128&amp;pmid=18852200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r39">39. <span class="element-citation">Purcell S, Cherny SS, Sham PC. Genetic Power Calculator: Design of linkage and association genetic mapping studies of complex traits. <span><span class="ref-journal">Bioinformatics. </span>2003;<span class="ref-vol">19</span>(1):149–150.</span> [<a href="/pubmed/12499305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Genetic+Power+Calculator:+Design+of+linkage+and+association+genetic+mapping+studies+of+complex+traits&amp;author=S+Purcell&amp;author=SS+Cherny&amp;author=PC+Sham&amp;volume=19&amp;issue=1&amp;publication_year=2003&amp;pages=149-150&amp;pmid=12499305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r40">40. <span class="element-citation">Jia X, et al.  Imputing amino acid polymorphisms in human leukocyte antigens. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>(6):e64683.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3675122/">PMC free article</a>]</span> [<a href="/pubmed/23762245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Imputing+amino+acid+polymorphisms+in+human+leukocyte+antigens&amp;author=X+Jia&amp;volume=8&amp;issue=6&amp;publication_year=2013&amp;pages=e64683&amp;pmid=23762245&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r41">41. <span class="element-citation">Winkler CA, et al.  Dominant effects of CCR2-CCR5 haplotypes in HIV-1 disease progression. <span><span class="ref-journal">J Acquir Immune Defic Syndr. </span>2004;<span class="ref-vol">37</span>(4):1534–1538.</span> [<a href="/pubmed/15602133" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Acquir+Immune+Defic+Syndr&amp;title=Dominant+effects+of+CCR2-CCR5+haplotypes+in+HIV-1+disease+progression&amp;author=CA+Winkler&amp;volume=37&amp;issue=4&amp;publication_year=2004&amp;pages=1534-1538&amp;pmid=15602133&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4664299&amp;issue-id=261407&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140323968968272title">Significance</a></li><li><a href="#idm140323970816864title">Abstract</a></li><li><a href="#s1title">Results</a></li><li><a href="#s7title">Discussion</a></li><li><a href="#s8title">Methods</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140323959701472title">Acknowledgments</a></li><li><a href="#idm140323959699424title">Footnotes</a></li><li><a href="#idm140323965353024title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4664299/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4664299/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4664299/pdf/pnas.201514867.pdf">PDF (1.2M)</a></li> | <li><a href="#" data-citationid="PMC4664299" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4664299%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4664299%2F&amp;text=Polymorphisms%20of%20large%20effect%20explain%20the%20majority%20of%20the%20host%20genetic%20contribution%20to%20variation%20of%20HIV-1%20virus%20load" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4664299%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26553974" data-aiid="4664299" data-aid="4664299" data-iid="261407" data-domainid="2" data-domain="pnas" data-accid="PMC4664299" data-md5="c6fab61028f08b7a82dba1f8d0572015"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:14:36-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B12E1DD580C71000000000A3605DB&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4664299%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=pnas&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4664299/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>