<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4981330/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of a systemic lupus erythematosus risk locus spanning ATG16L2, FCHSD2, and P2RY2 in Koreans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4981330/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Arthritis &amp; rheumatology (Hoboken, N.J.)"><meta name="citation_title" content="Identification of a systemic lupus erythematosus risk locus spanning ATG16L2, FCHSD2, and P2RY2 in Koreans"><meta name="citation_authors" content="Christopher J. Lessard, Satria Sajuthi, Jian Zhao, Kwangwoo Kim, John A. Ice, He Li, Hannah Ainsworth, Astrid Rasmussen, Jennifer A. Kelly, Mindy Marion, So-Young Bang, Young Bin Joo, Jeongim Choi, Hye-Soon Lee, Young Mo Kang, Chang-Hee Suh, Won Tae Chung, Soo-Kon Lee, Jung-Yoon Choe, Seung Cheol Shim, Ji Hee Oh, Young Jin Kim, Bok-Ghee Han, Nan Shen, Hwee Siew Howe, Edward K. Wakeland, Quan-Zhen Li, Yeong Wook Song, Patrick M. Gaffney, Marta E. Alarcón-Riquelme, Lindsey A. Criswell, Chaim O. Jacob, Robert P. Kimberly, Timothy J. Vyse, John B. Harley, Kathy L. Sivils, Sang-Cheol Bae, Carl D. Langefeld, Betty P. Tsao"><meta name="citation_date" content="May 2016"><meta name="citation_issue" content="5"><meta name="citation_volume" content="68"><meta name="citation_firstpage" content="1197"><meta name="citation_doi" content="10.1002/art.39548"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4981330/?report=abstract"><meta name="citation_pmid" content="26663301"><meta name="DC.Title" content="Identification of a systemic lupus erythematosus risk locus spanning ATG16L2, FCHSD2, and P2RY2 in Koreans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Christopher J. Lessard"><meta name="DC.Contributor" content="Satria Sajuthi"><meta name="DC.Contributor" content="Jian Zhao"><meta name="DC.Contributor" content="Kwangwoo Kim"><meta name="DC.Contributor" content="John A. Ice"><meta name="DC.Contributor" content="He Li"><meta name="DC.Contributor" content="Hannah Ainsworth"><meta name="DC.Contributor" content="Astrid Rasmussen"><meta name="DC.Contributor" content="Jennifer A. Kelly"><meta name="DC.Contributor" content="Mindy Marion"><meta name="DC.Contributor" content="So-Young Bang"><meta name="DC.Contributor" content="Young Bin Joo"><meta name="DC.Contributor" content="Jeongim Choi"><meta name="DC.Contributor" content="Hye-Soon Lee"><meta name="DC.Contributor" content="Young Mo Kang"><meta name="DC.Contributor" content="Chang-Hee Suh"><meta name="DC.Contributor" content="Won Tae Chung"><meta name="DC.Contributor" content="Soo-Kon Lee"><meta name="DC.Contributor" content="Jung-Yoon Choe"><meta name="DC.Contributor" content="Seung Cheol Shim"><meta name="DC.Contributor" content="Ji Hee Oh"><meta name="DC.Contributor" content="Young Jin Kim"><meta name="DC.Contributor" content="Bok-Ghee Han"><meta name="DC.Contributor" content="Nan Shen"><meta name="DC.Contributor" content="Hwee Siew Howe"><meta name="DC.Contributor" content="Edward K. Wakeland"><meta name="DC.Contributor" content="Quan-Zhen Li"><meta name="DC.Contributor" content="Yeong Wook Song"><meta name="DC.Contributor" content="Patrick M. Gaffney"><meta name="DC.Contributor" content="Marta E. Alarcón-Riquelme"><meta name="DC.Contributor" content="Lindsey A. Criswell"><meta name="DC.Contributor" content="Chaim O. Jacob"><meta name="DC.Contributor" content="Robert P. Kimberly"><meta name="DC.Contributor" content="Timothy J. Vyse"><meta name="DC.Contributor" content="John B. Harley"><meta name="DC.Contributor" content="Kathy L. Sivils"><meta name="DC.Contributor" content="Sang-Cheol Bae"><meta name="DC.Contributor" content="Carl D. Langefeld"><meta name="DC.Contributor" content="Betty P. Tsao"><meta name="DC.Date" content="2016 May"><meta name="DC.Identifier" content="10.1002/art.39548"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of a systemic lupus erythematosus risk locus spanning ATG16L2, FCHSD2, and P2RY2 in Koreans"><meta property="og:type" content="article"><meta property="og:description" content="Systemic lupus erythematosus (SLE) is a chronic autoimmune disorder whose etiology is incompletely understood, but likely involves environmental triggers in genetically susceptible individuals. We sought to identify the genetic loci associated with SLE ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4981330/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4981330/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4981330/pdf/nihms-804385.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D9B57DD5916E1000000000052002F.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4981330%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4981330</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Arthritis Rheumatol</a></span>. Author manuscript; available in PMC 2016 Aug 11.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26663301" target="pmc_ext" ref="reftype=publisher&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Arthritis Rheumatol. 2016 May; 68(5): 1197–1209. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fart.39548" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/art.39548</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4981330</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS804385</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26663301">26663301</a></div></div></div></div><h1 class="content-title">Identification of a systemic lupus erythematosus risk locus spanning <em>ATG16L2, FCHSD2</em>, and <em>P2RY2</em> in Koreans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Lessard%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248613686704" co-class="co-affbox">Christopher J. Lessard</a>,<sup>#</sup><sup>1</sup> <a href="/pubmed/?term=Sajuthi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248624138144" co-class="co-affbox">Satria Sajuthi</a>,<sup>#</sup><sup>2</sup> <a href="/pubmed/?term=Zhao%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248611297536" co-class="co-affbox">Jian Zhao</a>,<sup>#</sup><sup>3</sup> <a href="/pubmed/?term=Kim%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248614896976" co-class="co-affbox">Kwangwoo Kim</a>,<sup>#</sup><sup>4</sup> <a href="/pubmed/?term=Ice%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248611763600" co-class="co-affbox">John A. Ice</a>,<sup>1</sup> <a href="/pubmed/?term=Li%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248622825232" co-class="co-affbox">He Li</a>,<sup>1,</sup><sup>5</sup> <a href="/pubmed/?term=Ainsworth%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248612957472" co-class="co-affbox">Hannah Ainsworth</a>,<sup>2</sup> <a href="/pubmed/?term=Rasmussen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248614221760" co-class="co-affbox">Astrid Rasmussen</a>,<sup>1</sup> <a href="/pubmed/?term=Kelly%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248623612304" co-class="co-affbox">Jennifer A. Kelly</a>,<sup>1</sup> <a href="/pubmed/?term=Marion%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248623554496" co-class="co-affbox">Mindy Marion</a>,<sup>2</sup> <a href="/pubmed/?term=Bang%20SY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248615207120" co-class="co-affbox">So-Young Bang</a>,<sup>4</sup> <a href="/pubmed/?term=Joo%20YB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248695455840" co-class="co-affbox">Young Bin Joo</a>,<sup>4</sup> <a href="/pubmed/?term=Choi%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248611067536" co-class="co-affbox">Jeongim Choi</a>,<sup>4</sup> <a href="/pubmed/?term=Lee%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248610754016" co-class="co-affbox">Hye-Soon Lee</a>,<sup>4</sup> <a href="/pubmed/?term=Kang%20YM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248610622400" co-class="co-affbox">Young Mo Kang</a>,<sup>6</sup> <a href="/pubmed/?term=Suh%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248616299536" co-class="co-affbox">Chang-Hee Suh</a>,<sup>7</sup> <a href="/pubmed/?term=Chung%20WT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248698332608" co-class="co-affbox">Won Tae Chung</a>,<sup>8</sup> <a href="/pubmed/?term=Lee%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248619555376" co-class="co-affbox">Soo-Kon Lee</a>,<sup>9</sup> <a href="/pubmed/?term=Choe%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248623567024" co-class="co-affbox">Jung-Yoon Choe</a>,<sup>10</sup> <a href="/pubmed/?term=Shim%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248614121472" co-class="co-affbox">Seung Cheol Shim</a>,<sup>11</sup> <a href="/pubmed/?term=Oh%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248621186832" co-class="co-affbox">Ji Hee Oh</a>,<sup>12</sup> <a href="/pubmed/?term=Kim%20YJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248659961952" co-class="co-affbox">Young Jin Kim</a>,<sup>12</sup> <a href="/pubmed/?term=Han%20BG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248610698304" co-class="co-affbox">Bok-Ghee Han</a>,<sup>12</sup> <a href="/pubmed/?term=Shen%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248622706352" co-class="co-affbox">Nan Shen</a>,<sup>13,</sup><sup>14</sup> <a href="/pubmed/?term=Howe%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248619969568" co-class="co-affbox">Hwee Siew Howe</a>,<sup>15</sup> <a href="/pubmed/?term=Wakeland%20EK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248614465888" co-class="co-affbox">Edward K. Wakeland</a>,<sup>16</sup> <a href="/pubmed/?term=Li%20QZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248618843584" co-class="co-affbox">Quan-Zhen Li</a>,<sup>16</sup> <a href="/pubmed/?term=Song%20YW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248621148528" co-class="co-affbox">Yeong Wook Song</a>,<sup>17</sup> <a href="/pubmed/?term=Gaffney%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248694782160" co-class="co-affbox">Patrick M. Gaffney</a>,<sup>1</sup> <a href="/pubmed/?term=Alarc%26%23x000f3%3Bn-Riquelme%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248611427008" co-class="co-affbox">Marta E. Alarcón-Riquelme</a>,<sup>1,</sup><sup>18</sup> <a href="/pubmed/?term=Criswell%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248620298560" co-class="co-affbox">Lindsey A. Criswell</a>,<sup>19</sup> <a href="/pubmed/?term=Jacob%20CO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248660717536" co-class="co-affbox">Chaim O. Jacob</a>,<sup>20</sup> <a href="/pubmed/?term=Kimberly%20RP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248616233392" co-class="co-affbox">Robert P. Kimberly</a>,<sup>21</sup> <a href="/pubmed/?term=Vyse%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248660981648" co-class="co-affbox">Timothy J. Vyse</a>,<sup>22</sup> <a href="/pubmed/?term=Harley%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248611250304" co-class="co-affbox">John B. Harley</a>,<sup>23,</sup><sup>24</sup> <a href="/pubmed/?term=Sivils%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248611232800" co-class="co-affbox">Kathy L. Sivils</a>,<sup>1,</sup><sup>5,</sup><sup>26</sup> <a href="/pubmed/?term=Bae%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248621910224" co-class="co-affbox">Sang-Cheol Bae</a>,<sup>4,</sup><sup>26,</sup><sup>27</sup> <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248609798528" co-class="co-affbox">Carl D. Langefeld</a>,<sup>2,</sup><sup>26</sup> and  <a href="/pubmed/?term=Tsao%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301" class="affpopup" co-rid="_co_idm140248618266400" co-class="co-affbox">Betty P. Tsao</a><sup>3,</sup><sup>26,</sup><sup>27</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140248613686704"><h3 class="no_margin">Christopher J. Lessard</h3><p><sup>1</sup> Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><div>Find articles by <a href="/pubmed/?term=Lessard%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Christopher J. Lessard</a></div></div><div id="_co_idm140248624138144"><h3 class="no_margin">Satria Sajuthi</h3><p><sup>2</sup> Center for Public Health Genomics and Department of Biostatistical Sciences, Wake Forest University Health Sciences, Winston-Salem, NC 27157-106, USA</p><div>Find articles by <a href="/pubmed/?term=Sajuthi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Satria Sajuthi</a></div></div><div id="_co_idm140248611297536"><h3 class="no_margin">Jian Zhao</h3><p><sup>3</sup> Division of Rheumatology, Department of Medicine, University of California Los Angeles, Los Angeles, CA 90095, USA</p><div>Find articles by <a href="/pubmed/?term=Zhao%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Jian Zhao</a></div></div><div id="_co_idm140248614896976"><h3 class="no_margin">Kwangwoo Kim</h3><p><sup>4</sup> Department of Rheumatology, Hanyang University Hospital for Rheumatic Diseases, Seoul 133-792, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Kim%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Kwangwoo Kim</a></div></div><div id="_co_idm140248611763600"><h3 class="no_margin">John A. Ice</h3><p><sup>1</sup> Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><div>Find articles by <a href="/pubmed/?term=Ice%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">John A. Ice</a></div></div><div id="_co_idm140248622825232"><h3 class="no_margin">He Li</h3><p><sup>1</sup> Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><p><sup>5</sup> Department of Pathology, University of Oklahoma Health Sciences Center, Oklahoma City, OK 73014, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">He Li</a></div></div><div id="_co_idm140248612957472"><h3 class="no_margin">Hannah Ainsworth</h3><p><sup>2</sup> Center for Public Health Genomics and Department of Biostatistical Sciences, Wake Forest University Health Sciences, Winston-Salem, NC 27157-106, USA</p><div>Find articles by <a href="/pubmed/?term=Ainsworth%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Hannah Ainsworth</a></div></div><div id="_co_idm140248614221760"><h3 class="no_margin">Astrid Rasmussen</h3><p><sup>1</sup> Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><div>Find articles by <a href="/pubmed/?term=Rasmussen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Astrid Rasmussen</a></div></div><div id="_co_idm140248623612304"><h3 class="no_margin">Jennifer A. Kelly</h3><p><sup>1</sup> Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><div>Find articles by <a href="/pubmed/?term=Kelly%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Jennifer A. Kelly</a></div></div><div id="_co_idm140248623554496"><h3 class="no_margin">Mindy Marion</h3><p><sup>2</sup> Center for Public Health Genomics and Department of Biostatistical Sciences, Wake Forest University Health Sciences, Winston-Salem, NC 27157-106, USA</p><div>Find articles by <a href="/pubmed/?term=Marion%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Mindy Marion</a></div></div><div id="_co_idm140248615207120"><h3 class="no_margin">So-Young Bang</h3><p><sup>4</sup> Department of Rheumatology, Hanyang University Hospital for Rheumatic Diseases, Seoul 133-792, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Bang%20SY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">So-Young Bang</a></div></div><div id="_co_idm140248695455840"><h3 class="no_margin">Young Bin Joo</h3><p><sup>4</sup> Department of Rheumatology, Hanyang University Hospital for Rheumatic Diseases, Seoul 133-792, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Joo%20YB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Young Bin Joo</a></div></div><div id="_co_idm140248611067536"><h3 class="no_margin">Jeongim Choi</h3><p><sup>4</sup> Department of Rheumatology, Hanyang University Hospital for Rheumatic Diseases, Seoul 133-792, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Choi%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Jeongim Choi</a></div></div><div id="_co_idm140248610754016"><h3 class="no_margin">Hye-Soon Lee</h3><p><sup>4</sup> Department of Rheumatology, Hanyang University Hospital for Rheumatic Diseases, Seoul 133-792, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Lee%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Hye-Soon Lee</a></div></div><div id="_co_idm140248610622400"><h3 class="no_margin">Young Mo Kang</h3><p><sup>6</sup> Kyungpook National University Hospital, Daegu 700-721, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Kang%20YM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Young Mo Kang</a></div></div><div id="_co_idm140248616299536"><h3 class="no_margin">Chang-Hee Suh</h3><p><sup>7</sup> Ajou University Hospital, Suwon 443-380, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Suh%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Chang-Hee Suh</a></div></div><div id="_co_idm140248698332608"><h3 class="no_margin">Won Tae Chung</h3><p><sup>8</sup> Dong-A University Hospital, Busan 602-715, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Chung%20WT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Won Tae Chung</a></div></div><div id="_co_idm140248619555376"><h3 class="no_margin">Soo-Kon Lee</h3><p><sup>9</sup> Department of Internal Medicine, Yonsei University College of Medicine, Seoul 120-749, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Lee%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Soo-Kon Lee</a></div></div><div id="_co_idm140248623567024"><h3 class="no_margin">Jung-Yoon Choe</h3><p><sup>10</sup> Department of Internal Medicine, Catholic University of Daegu School of Medicine, Daegu 705-718, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Choe%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Jung-Yoon Choe</a></div></div><div id="_co_idm140248614121472"><h3 class="no_margin">Seung Cheol Shim</h3><p><sup>11</sup> Daejeon Rheumatoid &amp; Degenerative Arthritis Center, Chungnam National University Hospital, Daejeon 305-764, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Shim%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Seung Cheol Shim</a></div></div><div id="_co_idm140248621186832"><h3 class="no_margin">Ji Hee Oh</h3><p><sup>12</sup> Korea National Institute of Health, Osong 361-709, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Oh%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Ji Hee Oh</a></div></div><div id="_co_idm140248659961952"><h3 class="no_margin">Young Jin Kim</h3><p><sup>12</sup> Korea National Institute of Health, Osong 361-709, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Kim%20YJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Young Jin Kim</a></div></div><div id="_co_idm140248610698304"><h3 class="no_margin">Bok-Ghee Han</h3><p><sup>12</sup> Korea National Institute of Health, Osong 361-709, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Han%20BG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Bok-Ghee Han</a></div></div><div id="_co_idm140248622706352"><h3 class="no_margin">Nan Shen</h3><p><sup>13</sup> Shanghai Institute of Rheumatology, Renji Hospital, Shanghai, China 200001</p><p><sup>14</sup> Shanghai JiaoTong University School of Medicine, Shanghai, China 200025</p><div>Find articles by <a href="/pubmed/?term=Shen%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Nan Shen</a></div></div><div id="_co_idm140248619969568"><h3 class="no_margin">Hwee Siew Howe</h3><p><sup>15</sup> Department of Rheumatology, Allergy and Immunology, Tan Tock Seng Hospital, Singapore 308433</p><div>Find articles by <a href="/pubmed/?term=Howe%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Hwee Siew Howe</a></div></div><div id="_co_idm140248614465888"><h3 class="no_margin">Edward K. Wakeland</h3><p><sup>16</sup> University of Texas Southwestern Medical Center, Dallas, TX 75390, USA</p><div>Find articles by <a href="/pubmed/?term=Wakeland%20EK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Edward K. Wakeland</a></div></div><div id="_co_idm140248618843584"><h3 class="no_margin">Quan-Zhen Li</h3><p><sup>16</sup> University of Texas Southwestern Medical Center, Dallas, TX 75390, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20QZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Quan-Zhen Li</a></div></div><div id="_co_idm140248621148528"><h3 class="no_margin">Yeong Wook Song</h3><p><sup>17</sup> Department of Internal Medicine, Seoul National University Hospital, 101, Daehak-ro, Jongno-gu, Seoul 110-744, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Song%20YW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Yeong Wook Song</a></div></div><div id="_co_idm140248694782160"><h3 class="no_margin">Patrick M. Gaffney</h3><p><sup>1</sup> Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><div>Find articles by <a href="/pubmed/?term=Gaffney%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Patrick M. Gaffney</a></div></div><div id="_co_idm140248611427008"><h3 class="no_margin">Marta E. Alarcón-Riquelme</h3><p><sup>1</sup> Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><p><sup>18</sup> Centro de Genómica e Investigaciones Oncológicas, Pfizer-Universidad de Granada-Junta de Andalućıa, Granada 18100, Spain</p><div>Find articles by <a href="/pubmed/?term=Alarc%26%23x000f3%3Bn-Riquelme%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Marta E. Alarcón-Riquelme</a></div></div><div id="_co_idm140248620298560"><h3 class="no_margin">Lindsey A. Criswell</h3><p><sup>19</sup> Rosalind Russell / Ephraim P. Engleman Rheumatology Research Center, University of California San Francisco, San Francisco, CA, 94117, USA</p><div>Find articles by <a href="/pubmed/?term=Criswell%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Lindsey A. Criswell</a></div></div><div id="_co_idm140248660717536"><h3 class="no_margin">Chaim O. Jacob</h3><p><sup>20</sup> Department of Medicine, University of Southern California, Los Angeles, CA 90095</p><div>Find articles by <a href="/pubmed/?term=Jacob%20CO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Chaim O. Jacob</a></div></div><div id="_co_idm140248616233392"><h3 class="no_margin">Robert P. Kimberly</h3><p><sup>21</sup> Department of Medicine, Division of Clinical Immunology and Rheumatology, University of Alabama at Birmingham, Birmingham, AL 35294, USA</p><div>Find articles by <a href="/pubmed/?term=Kimberly%20RP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Robert P. Kimberly</a></div></div><div id="_co_idm140248660981648"><h3 class="no_margin">Timothy J. Vyse</h3><p><sup>22</sup> Divisions of Genetics and Molecular Medicine and Immunology, Infection and Inflammatory Disease, King's College London, London, UK WC2R 2LS</p><div>Find articles by <a href="/pubmed/?term=Vyse%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Timothy J. Vyse</a></div></div><div id="_co_idm140248611250304"><h3 class="no_margin">John B. Harley</h3><p><sup>23</sup> Division of Rheumatology, Cincinnati Children's Hospital Medical Center, Cincinnati, OH 45229, USA</p><p><sup>24</sup> US Department of Veterans Affairs Medical Center, Cincinnati, OH 45220, USA</p><div>Find articles by <a href="/pubmed/?term=Harley%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">John B. Harley</a></div></div><div id="_co_idm140248611232800"><h3 class="no_margin">Kathy L. Sivils</h3><p><sup>1</sup> Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><p><sup>5</sup> Department of Pathology, University of Oklahoma Health Sciences Center, Oklahoma City, OK 73014, USA</p><div>Find articles by <a href="/pubmed/?term=Sivils%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Kathy L. Sivils</a></div></div><div id="_co_idm140248621910224"><h3 class="no_margin">Sang-Cheol Bae</h3><p><sup>4</sup> Department of Rheumatology, Hanyang University Hospital for Rheumatic Diseases, Seoul 133-792, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Bae%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Sang-Cheol Bae</a></div></div><div id="_co_idm140248609798528"><h3 class="no_margin">Carl D. Langefeld</h3><p><sup>2</sup> Center for Public Health Genomics and Department of Biostatistical Sciences, Wake Forest University Health Sciences, Winston-Salem, NC 27157-106, USA</p><div>Find articles by <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Carl D. Langefeld</a></div></div><div id="_co_idm140248618266400"><h3 class="no_margin">Betty P. Tsao</h3><p><sup>3</sup> Division of Rheumatology, Department of Medicine, University of California Los Angeles, Los Angeles, CA 90095, USA</p><div>Find articles by <a href="/pubmed/?term=Tsao%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26663301">Betty P. Tsao</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140248617429216_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140248617429216_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140248617429216_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup> Arthritis and Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup> Center for Public Health Genomics and Department of Biostatistical Sciences, Wake Forest University Health Sciences, Winston-Salem, NC 27157-106, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup> Division of Rheumatology, Department of Medicine, University of California Los Angeles, Los Angeles, CA 90095, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup> Department of Rheumatology, Hanyang University Hospital for Rheumatic Diseases, Seoul 133-792, Republic of Korea</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup> Department of Pathology, University of Oklahoma Health Sciences Center, Oklahoma City, OK 73014, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup> Kyungpook National University Hospital, Daegu 700-721, Republic of Korea</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup> Ajou University Hospital, Suwon 443-380, Republic of Korea</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup> Dong-A University Hospital, Busan 602-715, Republic of Korea</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup> Department of Internal Medicine, Yonsei University College of Medicine, Seoul 120-749, Republic of Korea</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup> Department of Internal Medicine, Catholic University of Daegu School of Medicine, Daegu 705-718, Republic of Korea</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup> Daejeon Rheumatoid &amp; Degenerative Arthritis Center, Chungnam National University Hospital, Daejeon 305-764, Republic of Korea</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup> Korea National Institute of Health, Osong 361-709, Republic of Korea</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup> Shanghai Institute of Rheumatology, Renji Hospital, Shanghai, China 200001</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup> Shanghai JiaoTong University School of Medicine, Shanghai, China 200025</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup> Department of Rheumatology, Allergy and Immunology, Tan Tock Seng Hospital, Singapore 308433</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup> University of Texas Southwestern Medical Center, Dallas, TX 75390, USA</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup> Department of Internal Medicine, Seoul National University Hospital, 101, Daehak-ro, Jongno-gu, Seoul 110-744, Republic of Korea</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup> Centro de Genómica e Investigaciones Oncológicas, Pfizer-Universidad de Granada-Junta de Andalućıa, Granada 18100, Spain</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup> Rosalind Russell / Ephraim P. Engleman Rheumatology Research Center, University of California San Francisco, San Francisco, CA, 94117, USA</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup> Department of Medicine, University of Southern California, Los Angeles, CA 90095</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup> Department of Medicine, Division of Clinical Immunology and Rheumatology, University of Alabama at Birmingham, Birmingham, AL 35294, USA</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup> Divisions of Genetics and Molecular Medicine and Immunology, Infection and Inflammatory Disease, King's College London, London, UK WC2R 2LS</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup> Division of Rheumatology, Cincinnati Children's Hospital Medical Center, Cincinnati, OH 45229, USA</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup> US Department of Veterans Affairs Medical Center, Cincinnati, OH 45220, USA</div><div><sup>#</sup>Contributed equally.</div><div id="FN1"><sup>26</sup>These authors co-directed this manuscript</div><div id="CR1"><sup>27</sup> Corresponding Authors: Betty P. Tsao, PhD, 900 Veteran Avenue, Warren Hall Rm 14-224, Division of Rheumatology, Department of Medicine, University of California, Los Angeles, CA 90095, USA, Tel: +1 (310) 825-8906, <a href="mailto:dev@null" data-email="ude.alcu.tendem@oastb" class="oemail">ude.alcu.tendem@oastb</a>, Sang-Cheol Bae, MD, PhD, MPH, 222 Wangsimni-ro, Seondong-gu, Department of Rheumatology, Hanyang University Hospital for Rheumatic Diseases, Seoul 133-792, Republic of Korea, Tel: +82 2-2290-9237, <a href="mailto:dev@null" data-email="rk.ca.gnaynah@eabcs" class="oemail">rk.ca.gnaynah@eabcs</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140248617429216_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26663301" target="pmc_ext" ref="reftype=publisher&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Arthritis Rheumatol</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4981330/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemental 1.</strong></div><div><a href="/pmc/articles/PMC4981330/bin/NIHMS804385-supplement-Supplemental_1.pdf" data-ga-action="click_feat_suppl">NIHMS804385-supplement-Supplemental_1.pdf</a><span style="color:gray"> (3.3M)</span></div><div class="small guid">GUID:&nbsp;6FEEA9B2-D8C9-4BB8-96C3-B852EC6739CC</div></div></dd></dl></div><div id="idm140248599380720" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140248599380720title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Objective</h3><p id="P2" class="p p-first-last">Systemic lupus erythematosus (SLE) is a chronic autoimmune disorder whose etiology is incompletely understood, but likely involves environmental triggers in genetically susceptible individuals. We sought to identify the genetic loci associated with SLE in a Korean population by performing an unbiased genome-wide association scan.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P3" class="p p-first-last">A total of 1,174 Korean SLE cases and 4,248 population controls were genotyped with strict quality control measures and analyzed for association. For select variants, replication was tested in an independent set of 1,412 SLE cases and 1,163 population controls of Korean and Chinese ancestries.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P4" class="p p-first-last">Eleven regions outside the HLA exceeded genome-wide significance (P&lt;5×10<sup>−8</sup>). A novel SNP-SLE association was identified between <em>FCHSD2</em> and <em>P2RY2</em> peaking at rs11235667 (P = 1.0×10<sup>−8</sup>, odds ratio (OR) = 0.59) on a 33kb haplotype upstream to <em>ATG16L2</em>. Replication for rs11235667 resulted in P<sub>meta-rep</sub>=0.001 and P<sub>meta-overall</sub>=6.67×10<sup>−11</sup> (OR=0.63). Within the HLA region, association peaked in the Class II region at rs116727542 with multiple independent effects. Classical HLA allele imputation identified HLA-DRB1*1501 and HLA-DQB1*0602, both highly correlated, as most strongly associated with SLE. We replicated ten previously established SLE risk loci: STAT1-STAT4, TNFSF4, TNFAIP3, IKZF1, HIP1, IRF5, BLK, WDFY4, ETS1 and <em>IRAK1-MECP2</em>. Of these loci, we identified previously unreported independent second effects in <em>TNFAIP3</em> and <em>TNFSF4</em> as well as differences in the association for a putative causal variant in the <em>WDFY4</em> region.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions</h3><p id="P5" class="p p-first-last">Further studies are needed to identify true SLE risk effects in other suggestive loci and to identify the causal variant(s) in the regions of <em>ATG16L2, FCHSD2,</em> and <em>P2RY2.</em></p></div></div></div><div id="idm140248617681872" class="tsec sec headless whole_rhythm"><p id="P6" class="p p-first">Systemic lupus erythematosus (SLE; [MIM152700]) is a chronic, heterogeneous autoimmune disease characterized by the loss of tolerance to self-antigens, dysregulated type I interferon responses, and inflammation, often resulting in systemic end-organ damage(<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186814">1</a>). Immune dysfunction of SLE involves both B and T lymphocytes of the adaptive immune system, together with elements of the innate immune system, including dendritic cells and the complement system(<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186776">1</a>). The clinical manifestations of SLE can be quite variable and can involve virtually any organ system. Although the precise etiology of SLE is largely unknown, the pathogenic mechanism likely involves environmental triggers in a genetically susceptible host(<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186811">2</a>). Few effective treatment options exist, largely due to an incomplete understanding of the pathophysiological basis of the disease.</p><p id="P7" class="p">Genetic predisposition leading to increased risk of SLE is supported by high heritability (&gt;66%), increased risk among siblings of affected patients (λ<sub>s</sub>≈30), and an ~25% monozygotic twin concordance(<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186799">3</a>). Today, associations of more than 50 loci with SLE susceptibility have been identified and confirmed(<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186819">4</a>). Many of these genes fall into known pathways that are key to innate and adaptive immune responses, lymphocyte activation and/or function, and immune complex clearance(<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186821">4</a>). However, a significant proportion of heritable risk to SLE has yet to be explained(<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186770">5</a>). The identification of SLE-associated genes and their pathogenic mechanisms will greatly enhance our understanding of lupus pathophysiology and facilitate the development of effective diagnostic, prognostic, and therapeutic tools. To date, large-scale genome-wide genetic studies of Asian SLE populations have focused on Han Chinese(<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186820">6</a>-<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186807">8</a>) and Japanese(<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186769">9</a>). Moreover, several reports have shown that transracial mapping of SLE loci can aid in the dissection of risk effects(<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186809">4</a>). In this study, we performed a genome-wide association (GWA) scan to identify genes associated with SLE in an East Asian population from Korea.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Methods</h2><div id="S6" class="sec sec-first"><h3 id="S6title">Subjects</h3><p id="P8" class="p p-first">A total of 1,174 patients with SLE were recruited from the Hanyang University Hospital for Rheumatic Diseases (HUHRD) and six other university hospitals in Korea(<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_577950653">10</a>). In addition, 552 ethnically matched healthy controls were recruited from HUHRD. The 3,700 ethnically matched out-of-study population controls were recruited from the Korean National Institutes of Health(<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_577950654">10</a>). In addition, an independent cohort of 1,412 SLE cases and 1,163 population controls were used for the replication studies(<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186795">11</a>, <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186781">12</a>). This sample set consisted of 739 Korean SLE cases and 436 Korean controls as well as 677 Chinese SLE cases and 709 Chinese controls (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>).</p><p id="P9" class="p p-last">Written, informed consent from each participant was obtained by each participant following protocols approved by the Institutional Review Boards of participating institutes. All cases used in this study fulfilled at least 4 of the 11 American College of Rheumatology criteria for SLE(<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186803">13</a>), while healthy, population-based controls were without family history of SLE or any other autoimmune disease.</p></div><div id="S7" class="sec"><h3 id="S7title">GWA scan Genotyping, Sample Quality Control, and Ascertainment of Populations Stratification</h3><p id="P10" class="p p-first">Samples were genotyped using the Illumina HumanOmni1-Quad or HumanOmniExpress arrays using Infinium chemistry at Oklahoma Medical Research Foundation (OMRF) following the manufacturer's protocol (Illumina, Inc., San Diego, CA). The out-of-study GWA controls were genotyped on the HumanOmni1-Quad arrays by the Korea National Institutes of Health. Strict quality control standards were implemented for SNPs retained in the association analysis, including requirements for well-defined cluster scatter plots. Samples were excluded if they had a SNP call rate &lt;90%. SNPs were considered high quality SNPs if they had call rates &gt;95%, no evidence of differential missingness between cases and controls (P &lt; 0.05) and no evidence of a departure from expected Hardy-Weinberg proportions (controls P &lt; 0.01, cases P &lt; 0.000001). Inference is primarily based on those SNPs with minor allele frequency (MAF) greater than 1%.</p><p id="P11" class="p p-last">Based on the SNPs that passed the above quality control thresholds, samples were removed if there were inconsistencies between recorded and genotype-inferred gender or excess heterozygosity on the autosomes. Duplicates and first- or second-degree relatives were removed based on identity-by-descent statistics computed by the program KING(<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186786">14</a>). Principal components (PCs) were computed with the samples and merged with HapMap phase 3 individuals (CEU, YRI, and CHB) as reference populations(<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186787">15</a>) using EIGENSOFT(<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186818">16</a>). Principal component analysis (PCA) was performed on a subset of autosomal SNPs that were selected by removing regions of known high linkage disequilibrium (LD), removing variants with MAF &lt; 0.05, and pruning markers to reduce extended pairwise LD. The PCs were used to remove genetic outliers (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>). The dataset that passed laboratory and statistical quality control was composed of 1174 SLE cases (1096 females and 78 males) and 548 within-study controls (547 females and 1 male). In addition, 3698 out-of-study controls (2330 females and 1368 males) were merged into the within-study genotype data.</p></div><div id="S8" class="sec"><h3 id="S8title">Statistical Analysis</h3><p id="P12" class="p p-first">To test for an association between a SNP and SLE status, a logistic regression analysis was computed including PC 3 as a covariate since no additional PC significantly changed the inflation factor (λ). Primary inference was based on the additive genetic model unless there was significant lack-of-fit (P &lt; 0.05). If there was evidence of a departure from an additive model, then inference was based on the most significant value from the dominant, additive or recessive genetic models. The additive and recessive models were computed only if there were at least 10 and 30 individuals homozygous for the minor allele, respectively. The analyses were completed using the program SNPGWA version 4.0. For the analysis of chromosome X SNPs, the samples were stratified by gender and then meta-analyzed across gender using the program METAL(<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186804">17</a>).</p><p id="P13" class="p p-last">To determine the number of independent associations within each SLE-risk locus exceeding the genome-wide significance threshold, a manual stepwise model or conditional analysis was computed. The stepwise modeling or conditional analysis was implemented using forward selection with backward elimination using the entry and exit criteria of P &lt; 0.0001, which accounted for approximately 500 independent variants within a given genomic region. Specifically, for each region of interest, the top SNP was included as a covariate and the association statistics were re-calculated. SNPs were allowed to enter and exit models in this stepwise fashion until no additional SNPs met a significance threshold of P &lt; 0.0001.</p></div><div id="S9" class="sec"><h3 id="S9title">Replication Genotyping, Sample Quality Control, and Ascertainment of Populations Stratification</h3><p id="P14" class="p p-first-last">Genotypes were obtained using TaqMan assays (Life Technologies, Grand Island, NY) for four SNPs: rs2267828, rs10901656, rs11235667, and rs1048257. Analysis was conducted for these cohorts independently to allow for PC analysis using previously collected data. Ancestry adjustments for the Koreans were described previously in Lessard et al., (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186805">11</a>). For the Chinese subjects, the PCA was done with slight modification from what was reported in Kaiser et al. (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186825">12</a>). In this study, 7,918 randomly selected autosomal ImmunoChip SNPs with MAF&gt;1%, low pairwise LD (<em>r<sup>2</sup></em> &lt; 0.1), and no evidence of association with SLE (P &gt; 0.01) were used to perform PC analysis using EIGENSOFT. PC analysis plots of the CHB and CHS subjects in the 1000 Genomes Project along with our subjects were used to select and remove genetic outliers. The first PC (Chinese cohort) and PCs 1, 2, and 3 (OMRF and UCLA Korean datasets) were included as covariates in the logistic regression models based on the variance explained in each dataset. These dataset were than meta-analyzed using the program METAL(<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186784">17</a>). To test for heterogeneity among the individual association results in the meta-analysis, we utilized both the Cochran's <em>Q</em> test statistic(<a href="#R18" rid="R18" class=" bibr popnode">18</a>) and <em>I<sup>2</sup></em> index(<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186773">19</a>).</p></div><div id="S10" class="sec sec-last"><h3 id="S10title">Imputation</h3><p id="P15" class="p p-first">To help localize the associations in the genome-wide significant regions, ungenotyped variants were imputed based on the reference panel from the 1000 Genomes Project(<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186810">20</a>). Specifically, the program SHAPEIT was used to pre-phase the genotype data(<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186802">21</a>). After phasing the data, IMPUTE2 was used for the imputation with the 1000 Genomes Phase I integrated reference panel(<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186783">22</a>). The imputed data was filtered using standard post-imputation quality control based on IMPUTE2 information scores &gt;0.5 and confidence scores &gt;0.9 for subsequent association tests. Post-association analysis required genotyped SNPs in LD with imputed variants to support the inferred alleles as true signals. The program SNPTESTv2 was used to test for association of the imputed variants(<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186771">23</a>).</p><p id="P16" class="p p-last">Imputation of the HLA classical alleles in the genes HLA-A, -B, -C, -DPB1, -DQA1, -DQB1, and -DRB1 was done using the program HiBAG(<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186823">24</a>) and the Asian reference panel. In this sample, ~21% of the reference SNPs used by HiBAG were missing genotype data. To address this issue, HLA imputation was repeated after filling in the missing genotype data with the “best guess” imputed SNP data from the 1000 Genomes imputation described above. By using the “best guess” genotype data with a posterior probability &gt;0.90, the percent of missing variants in the reference set was reduced to 0.36%.</p></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Results</h2><div id="S12" class="sec sec-first"><h3 id="S12title">Summary of the genome-wide association phase</h3><p id="P17" class="p p-first-last">We observed modest inflation in the test statistic (λ=1.09) with only slight deviation from expected once the HLA and other known SLE loci were removed (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a>). A total of eleven regions surpassed the genome-wide significance threshold of P &lt; 5×10<sup>−8</sup> with <em>STAT4</em> (MIM600558) yielding the most significant genotyped association with SLE at rs11889341 (P = 8.02×10<sup>−19</sup>; <a href="/pmc/articles/PMC4981330/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a> and <a href="/pmc/articles/PMC4981330/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Of the non-HLA regions, 10 risk loci had been previously identified and confirmed as risk loci for SLE, including <em>STAT1</em> (MIM600555)-<em>STAT4</em>, <em>IKZF1</em> (MIM603023), <em>TNFAIP3</em> (MIM191163), <em>TNFSF4</em> (MIM603594), <em>HIP1</em> (MIM601767), <em>IRF5</em> (MIM607218), <em>ETS1</em> (MIM164740), <em>BLK</em> (MIM191305), <em>WDFY4</em> (MIM613316), and <em>IRAK1</em> (MIM300283)<em>-MECP2</em> (MIM300005). In addition, association not previously described for SLE risk was observed at 11q14 (<a href="/pmc/articles/PMC4981330/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4981330/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140248616936960" class="figure"><a href="/pmc/articles/PMC4981330/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4981330_nihms-804385-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-804385-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4981330/bin/nihms-804385-f0001.jpg"></a></div><div id="largeobj_idm140248616936960" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4981330/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4981330/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Summary of the genome-wide association results for 1174 SLE cases and 3698 controls of Korean ancestry and zoomed plot of the region associated with SLE at 11q14</strong><p id="__p2">(A) The -log<sub>10</sub>(P-value) for each genotyped variant is plotted along the Y-axis with the chromosome and chromosomal position along the X-axis. The gray line indicates the genome-wide significance threshold of P = 5×10<sup>−8</sup>. (B) The −log<sub>10</sub>(P-value) is plotted for each genotyped (shown as circles) and imputed (shown as triangles) variants, and with the peak association, rs11235667, is plotted as a diamond. The linkage disequilibrium with rs11235667 is given by the scale on the figure. The genome-wide significance threshold is displayed as a dashed line at P = 5 ×10<sup>−8</sup>. Association exceeding this threshold was found extending from <em>ATG16L2</em> through <em>FCHSD2</em> to the shared promoter region with <em>P2RY2</em>.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Single locus analysis of previous SLE associations outside of HLA region</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140248620439808" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Marker</th><th rowspan="1" colspan="1" valign="top" align="center">Chr<sup><sup>A</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">Position</th><th rowspan="1" colspan="1" valign="top" align="center">Upstream Gene</th><th rowspan="1" colspan="1" valign="top" align="center">Downstream Gene</th><th rowspan="1" colspan="1" valign="top" align="center">Within Gene</th><th rowspan="1" colspan="1" valign="top" align="center">MAF<sup><sup>B</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">P<sub>GWAS</sub></th><th rowspan="1" colspan="1" valign="top" align="center">Model</th><th rowspan="1" colspan="1" valign="top" align="center">Obs/Imp<sup><sup>C</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI<sup><sup>D</sup></sup>)</th><th rowspan="1" colspan="1" valign="top" align="center">Maj/Min<sup><sup>E</sup></sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1234314</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">173177392</td><td rowspan="1" colspan="1" valign="top" align="center">1kb from <em>TNFSF4</em></td><td rowspan="1" colspan="1" valign="top" align="center">269kb from <em>PRDX6</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">9.25×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">1.37 (1.25-1.51)</td><td rowspan="1" colspan="1" valign="top" align="center">C/G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2205960</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">173191475</td><td rowspan="1" colspan="1" valign="top" align="center">15kb from <em>TNFSF4</em></td><td rowspan="1" colspan="1" valign="top" align="center">255kb from <em>PRDX6</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">1.03×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">1.44 (1.30-1.60)</td><td rowspan="1" colspan="1" valign="top" align="center">G/T</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs76413021</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">173206297</td><td rowspan="1" colspan="1" valign="top" align="center">30kb from <em>TNFSF4</em></td><td rowspan="1" colspan="1" valign="top" align="center">240kb from <em>PRDX6</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">3.26×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">1.52 (1.36-1.71)</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs844644</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">173209495</td><td rowspan="1" colspan="1" valign="top" align="center">33kb from <em>TNFSF4</em></td><td rowspan="1" colspan="1" valign="top" align="center">237kb from <em>PRDX6</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.4</td><td rowspan="1" colspan="1" valign="top" align="center">4.47×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.37 (1.25-1.50)</td><td rowspan="1" colspan="1" valign="top" align="center">A/C</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10489265</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">173236065</td><td rowspan="1" colspan="1" valign="top" align="center">60kb from <em>TNFSF4</em></td><td rowspan="1" colspan="1" valign="top" align="center">211kb from <em>PRDX6</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">8.19×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.43 (1.29-1.58)</td><td rowspan="1" colspan="1" valign="top" align="center">A/C</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4916342</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">173347837</td><td rowspan="1" colspan="1" valign="top" align="center">172kb from <em>TNFSF4</em></td><td rowspan="1" colspan="1" valign="top" align="center">99kb from <em>PRDX6</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.3</td><td rowspan="1" colspan="1" valign="top" align="center">8.22×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">0.75 (0.67-0.82)</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs16833239</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">191940260</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>STAT4</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">9.69×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">0.67 (0.59-0.76)</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11889341</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">191943742</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>STAT4</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">8.02×10<sup>−19</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.53 (1.40-1.69)</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12612769</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">191953998</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>STAT4</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">2.37×10<sup>−19</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">1.59 (1.43-1.75)</td><td rowspan="1" colspan="1" valign="top" align="center">A/C</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs13192841</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">137967214</td><td rowspan="1" colspan="1" valign="top" align="center">152kb from <em>OLIG3</em></td><td rowspan="1" colspan="1" valign="top" align="center">221kb from <em>TNFAIP3</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">7.50×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.23 (1.06-1.43)</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs5029937</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">138195151</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>TNFAIP3</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">3.98×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">2.11 (1.74-2.55)</td><td rowspan="1" colspan="1" valign="top" align="center">G/T</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs5029939</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">138195723</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>TNFAIP3</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">4.09×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">2.10 (1.74-2.55)</td><td rowspan="1" colspan="1" valign="top" align="center">C/G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2230926</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">138196066</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>TNFAIP3</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">2.34×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.93 (1.63-2.28)</td><td rowspan="1" colspan="1" valign="top" align="center">T/G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs9373203</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">138289848</td><td rowspan="1" colspan="1" valign="top" align="center">86kb from <em>TNFAIP3</em></td><td rowspan="1" colspan="1" valign="top" align="center">120kb from <em>PERP</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">4.89×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">1.25 (1.14-1.37)</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs6922466</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">138444930</td><td rowspan="1" colspan="1" valign="top" align="center">16kb from <em>PERP</em></td><td rowspan="1" colspan="1" valign="top" align="center">38kb from <em>KIAA1244</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.297</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.08 (0.94-1.24)</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11185602</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">50299077</td><td rowspan="1" colspan="1" valign="top" align="center">100kb from <em>C7orf72</em></td><td rowspan="1" colspan="1" valign="top" align="center">45kb from <em>IKZF1</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">1.53×10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">0.66 (0.60-0.73)</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs17552904</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">50318308</td><td rowspan="1" colspan="1" valign="top" align="center">120kb from <em>C7of72</em></td><td rowspan="1" colspan="1" valign="top" align="center">26kb from IKZF1</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">3.51×10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">0.65 (0.59-0.72)</td><td rowspan="1" colspan="1" valign="top" align="center">G/T</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs6964720</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">75180344</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>HIP1</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">2.00×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.40 (1.26-1.56)</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs139110493</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">75209951</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>HIP1</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">1.21×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">2.48 (1.93-3.19)</td><td rowspan="1" colspan="1" valign="top" align="center">G/C</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4728142</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">128573967</td><td rowspan="1" colspan="1" valign="top" align="center">11kb from <em>LOC392787</em></td><td rowspan="1" colspan="1" valign="top" align="center">4kb from <em>IRF5</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">1.38×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.53 (1.35-1.73)</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs113478424</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">128575797</td><td rowspan="1" colspan="1" valign="top" align="center">13kb from <em>LOC392787</em></td><td rowspan="1" colspan="1" valign="top" align="center">2kb from <em>IRF5</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">3.97×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">1.59 (1.39-1.81)</td><td rowspan="1" colspan="1" valign="top" align="center">15-mer<sup><sup>*</sup></sup>/T</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs922483</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">11351912</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>BLK</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">2.00×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">0.71 (0.64-0.79)</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2736345</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">11352485</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>BLK</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">7.88×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">0.70 (0.63-0.78)</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10857631</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">49955821</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>WDFY4</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">1.67×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">1.30 (1.13-1.48)</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs7097397</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">50025396</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>WDFY4</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">2.10×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.33 (1.21-1.46)</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs877819</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">50042951</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>WDFY4</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.0558</td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">1.13 (1.00-1.28)</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10776651</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">50084526</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>WDFY4</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">1.54×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">1.29 (1.18-1.43)</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1913517</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">50119054</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>WDFY4</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">2.54×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.24 (1.12-1.37)</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12576753</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">128304141</td><td rowspan="1" colspan="1" valign="top" align="center">None within 500kb</td><td rowspan="1" colspan="1" valign="top" align="center">25kb from <em>ETS1</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">1.74×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">1.37 (1.25-1.56)</td><td rowspan="1" colspan="1" valign="top" align="center">C/A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1128334</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">128328959</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>ETS1</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">7.17×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">1.39 (1.26-1.52)</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs5986948</td><td rowspan="1" colspan="1" valign="top" align="center">X</td><td rowspan="1" colspan="1" valign="top" align="center">153266172</td><td rowspan="1" colspan="1" valign="top" align="center">17.3kb from <em>TMEM187</em></td><td rowspan="1" colspan="1" valign="top" align="center">9.8kb from <em>IRAK1</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">4.36 ×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Rec</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">0.64 (0.56-0.74)</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1059702</td><td rowspan="1" colspan="1" valign="top" align="center">X</td><td rowspan="1" colspan="1" valign="top" align="center">153284192</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>IRAK1</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">5.14×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Rec</td><td rowspan="1" colspan="1" valign="top" align="center">Imp</td><td rowspan="1" colspan="1" valign="top" align="center">0.65 (0.57-0.74)</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2734647</td><td rowspan="1" colspan="1" valign="top" align="center">X</td><td rowspan="1" colspan="1" valign="top" align="center">153292180</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>MECP2</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">7.54×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">Obs</td><td rowspan="1" colspan="1" valign="top" align="center">0.62 (0.51-0.75)</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td></tr></tbody></table></div><div id="largeobj_idm140248620439808" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4981330/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>A</sup>Chr = Chromosome</div><div id="TFN2"><sup>B</sup>MAF = Minor allele frequency</div><div id="TFN3"><sup>C</sup>Obs/Imp = observed/imputed</div><div id="TFN4"><sup>D</sup>CI = Confidence interval</div><div id="TFN5"><sup>E</sup>Maj/Min = Major/Minor allele</div><div id="TFN6"><sup>*</sup>15-mer= CTTAGCTATTGCTC</div></div></div></div><div id="S13" class="sec"><h3 id="S13title">Association at 11q14 with SLE</h3><p id="P18" class="p p-first">This SNP-SLE association was observed with a single variant located between <em>FCHSD2</em> (MIM not available) and <em>P2RY2</em> (MIM600041) (rs11235667; P=1.03×10<sup>−8</sup>; odds ratio (OR) = 0.59; 95% confidence interval (CI) = 0.50-0.71; <a href="/pmc/articles/PMC4981330/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>, <a href="/pmc/articles/PMC4981330/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Moreover, additional support was observed with genotyped variants in the region (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). After imputation of the 11q14 region showing association with SLE, rs11235667 remained the most significant association (<a href="/pmc/articles/PMC4981330/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). However, a haplotype was identified with 8 variants exceeding the genome-wide significance threshold that spanned from <em>ATG16L2</em> (MIM not available) through <em>FCHSD2</em> to the shared promoter region with <em>P2RY2</em>. Stepwise logistic regression analysis adjusting for rs11235667 indicated the presence of only a single effect (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 3</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">Single locus analysis results for regions genotyped in the replication study.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140248614139808" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Marker</th><th rowspan="1" colspan="1" valign="top" align="center">Region Name</th><th rowspan="1" colspan="1" valign="top" align="center">Maj/Min<sup><sup>A</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">MAF<sup><sup>B</sup></sup> Case / Ctrl<sup><sup>C</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">P<sub>GWAS</sub></th><th rowspan="1" colspan="1" valign="top" align="center">Model</th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI<sup><sup>D</sup></sup>)</th><th rowspan="1" colspan="1" valign="top" align="center">P Meta Rep<sup><sup>E</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">Q / I<sup>2</sup> Meta Rep<sup><sup>E</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI<sup><sup>D</sup></sup>) Meta Rep<sup><sup>E</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">P Meta Overall<sup>F</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Q / I<sup>2</sup> Meta Overall<sup>F</sup></th><th rowspan="1" colspan="1" valign="top" align="center">OR (95%CI<sup><sup>D</sup></sup>) Meta Overall</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2267828</td><td rowspan="1" colspan="1" valign="top" align="center">GTF2IRD1</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.40 / 0.45</td><td rowspan="1" colspan="1" valign="top" align="center">7.02×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">0.79 (0.72-0.87)</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">0.56 / 0</td><td rowspan="1" colspan="1" valign="top" align="center">0.87 (0.77-0.98)</td><td rowspan="1" colspan="1" valign="top" align="center">6.46×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.41 / 0</td><td rowspan="1" colspan="1" valign="top" align="center">0.81 (0.76-0.88)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10901656</td><td rowspan="1" colspan="1" valign="top" align="center">DOCK1</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.27 / 0.23</td><td rowspan="1" colspan="1" valign="top" align="center">6.91×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">1.39 (1.22-1.58)</td><td rowspan="1" colspan="1" valign="top" align="center">0.095</td><td rowspan="1" colspan="1" valign="top" align="center">0.6 / 0</td><td rowspan="1" colspan="1" valign="top" align="center">1.14 (0.98-1.31)</td><td rowspan="1" colspan="1" valign="top" align="center">9.56×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.23 / 28.72</td><td rowspan="1" colspan="1" valign="top" align="center">1.21 (1.12-1.32)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11235667</td><td rowspan="1" colspan="1" valign="top" align="center">FCHSD2-P2RY2</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.07 / 0.11</td><td rowspan="1" colspan="1" valign="top" align="center">1.03×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">0.59 (0.50-0.71)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0014</td><td rowspan="1" colspan="1" valign="top" align="center">0.29 / 18.43</td><td rowspan="1" colspan="1" valign="top" align="center">0.71 (0.57-0.87)</td><td rowspan="1" colspan="1" valign="top" align="center">6.67×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.14 / 44.37</td><td rowspan="1" colspan="1" valign="top" align="center">0.63 (0.55-0.72)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1048257</td><td rowspan="1" colspan="1" valign="top" align="center">AHNAK2</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.34 / 0.39</td><td rowspan="1" colspan="1" valign="top" align="center">1.67×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Add</td><td rowspan="1" colspan="1" valign="top" align="center">0.79 (0.72-0.87)</td><td rowspan="1" colspan="1" valign="top" align="center">0.086</td><td rowspan="1" colspan="1" valign="top" align="center">0.29 / 17.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.90 (0.80-1.01)</td><td rowspan="1" colspan="1" valign="top" align="center">8.66×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.12 / 47.82</td><td rowspan="1" colspan="1" valign="top" align="center">0.82 (0.76-0.89)</td></tr></tbody></table></div><div id="largeobj_idm140248614139808" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4981330/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN7"><sup>A</sup>Maj/Min = Major/Minor allele</div><div id="TFN8"><sup>B</sup>MAF = Minor allele frequency</div><div id="TFN9"><sup>C</sup>Case/Ctrl = Case/Control</div><div id="TFN10"><sup>D</sup>CI = Confidence interval</div><div id="TFN11"><sup>E</sup>Meta Rep = Meta-analysis for the replication</div></div></div><p id="P19">Replication analysis for the primary signal in the region of <em>FCHSD2-P2RY2</em> was done using independent cohorts from Korea and China (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). The SNP rs11235667 between <em>FCHSD2</em> and <em>P2RY2</em> continued to show significant SLE association and similar effect size (P<sub>meta-rep</sub> = 0.001; OR = 0.71, 95% CI = 0.57-0.87). The overall meta-analysis between GWA and replication studies yielded a P<sub>meta-overall</sub> = 6.67×10<sup>−11</sup> (OR = 0.63, 95% CI = 0.55-0.72; <a href="/pmc/articles/PMC4981330/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). No evidence of heterogeneity was observed in the meta-analysis (<a href="/pmc/articles/PMC4981330/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P20">Bioinformatics analysis using Haploreg v2(<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186788">25</a>) revealed that the region around rs11235667 was hypersensitive to DNase 1 in B cells by the ENCODE project(<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186832">26</a>). This variant has been shown to be located within an enhancer element in multiple immunological cell types based on the Epigenetic Roadmap data (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>)(<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186828">27</a>). Chromatin immunoprecipitation followed by sequencing (ChIP-seq) experiments carried out by the ENCODE project found POL2 and YY1 proteins cross-linked to this region. Moreover, sequence prediction methods indicate that rs11235667 can alter the binding motif for the FOXa family of transcription factors using sequence prediction methods according to Haploreg v2(<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186798">25</a>). These data suggest the likely functional mechanism involves regulation of expression of <em>ATG16L2, FCHSD2</em> and/or <em>P2RY2</em>. However, current eQTL databases do not suggest that rs11235667 influences the expression of these loci(<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186817">25</a>, <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186815">28</a>). This could be due to the lack of data in the correct cell and/or tissue type and/or that some databases do not interrogate this SNP in their studies.</p><p id="P21" class="p p-last">Of the other 8 variants that exceeded genome-wide significance on the haplotype, several findings make rs11235604 an intriguing potential causal variant (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 3</a>). This variant is a missense allele (R220W) that resides in the coding region of <em>ATG16L2</em>. Although it is predicted to be benign by PolyPhen-2(<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186833">29</a>), it is possible this variant my still impact SLE. Haploreg v2(<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186793">25</a>) does report that rs11235604 alters 8 predicted regulatory motifs, and this variant is thought to be an active enhancer in several immunologically relevant cell types (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). Further work is needed to conclusively identify the polymorphism(s) responsible for this association signal. These studies would include evaluating the potential impact of rs11235667 on the expression of <em>ATG16L2, FCHSD2,</em> and/or <em>P2RY2.</em> In addition, experiments are needed toassess the impact on <em>ATG16L2</em> of the missense allele arising from rs11235604 and/or any other variant(s) within this haplotype.</p></div><div id="S14" class="sec"><h3 id="S14title">Association in the HLA region in Koreans with SLE</h3><p id="P22" class="p p-first">One of the most consistent associations with SLE has been with the human leukocyte antigen (HLA) region. Although the HLA was not the most statistically significant genotyped region, the SNP rs116727542 (P = 6.15×10<sup>−24</sup>; <a href="/pmc/articles/PMC4981330/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>), which is located in a broad peak of association that was observed spanning HLA-DR (MIM142860) through -DQ (MIM146880), showed the strongest SNP-SLE association after imputation (<a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>, <a href="/pmc/articles/PMC4981330/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). The interval between HLA-DR and -DQ has previously been implicated Koreans(<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186789">30</a>). In an attempt to identify the number of independent effects in this complex region, we used the stepwise approach described above and found ten independent effects (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4A</a> for results of the stepwise regression analysis). The first four variants identified in the stepwise regression analysis, rs116727542, rs9273371, rs114653103, and rs115253455, are all located in the HLA Class II region (<a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4A, and Supplementary Table 4</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140248618044784" class="figure"><a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4981330_nihms-804385-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-804385-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4981330/bin/nihms-804385-f0002.jpg"></a></div><div id="largeobj_idm140248618044784" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4981330/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4981330/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Expanded view of the association between SLE and the HLA, <em>TNFAIP3, TNFSF4,</em> and <em>WDFY4</em> regions</strong><p id="__p3">The –log<sub>10</sub>(P-value) is plotted for each observed (shown as circles) and imputed (shown as triangles) variant in the MHC region according to base-pair position from 26Mb to 34Mb on chromosome 6 (A). Linkage disequilibrium with the first four variants included in the stepwise logistic regression analysis is shown with rs116727542 (blue diamond), rs114653103 (red diamond), rs9273371 (green diamond), and rs115253455 (gold diamond) all located within the HLA Class II region. The insert on the left shows the –log<sub>10</sub>(P-value) of the imputed classical alleles plotted according to base-pair position from 31Mb to 33.5Mb. The additional plots show results for Chromosome 6 in the region of <em>TNFAIP3</em> (B), Chromosome 1 for the <em>TNFSF4</em> (C) effects, and Chromosome 10 for <em>WDFY4</em> (D). For each independent effect, the peak associations are represented by a diamond (blue for the first effect and red for the second, if applicable), and the correlation of variants accounted for by each effect is given in their respective color according the legends present in each plot. The genome-wide significance threshold is displayed as a dashed line on each plot at P = 5 ×10<sup>−8</sup>.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p6">Single locus and stepwise results for top ten independent HLA associations</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140248621224192" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Marker<sup><sup>A</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">Position</th><th rowspan="1" colspan="1" valign="top" align="center">Upstream Gene</th><th rowspan="1" colspan="1" valign="top" align="center">Downstream Gene</th><th rowspan="1" colspan="1" valign="top" align="center">Within Gene</th><th rowspan="1" colspan="1" valign="top" align="center">MAF<sup><sup>B</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">P-value</th><th rowspan="1" colspan="1" valign="top" align="center">OR (95%CI<sup><sup>C</sup></sup>)</th><th rowspan="1" colspan="1" valign="top" align="center">Maj/Min<sup><sup>D</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">Stepwise P-value</th><th rowspan="1" colspan="1" valign="top" align="center">Stepwise OR (95%CI<sup><sup>C</sup></sup>)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs116727542</td><td rowspan="1" colspan="1" valign="top" align="center">32421227</td><td rowspan="1" colspan="1" valign="top" align="center">8.4kb from <em>HLA-DRA</em></td><td rowspan="1" colspan="1" valign="top" align="center">64kb from <em>HLA-DRB5</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.1700</td><td rowspan="1" colspan="1" valign="top" align="center">6.15×10<sup>−24</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.53 (0.47-0.60)</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">1.96×10<sup>−18</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.74 (1.54-1.97)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs9273371</td><td rowspan="1" colspan="1" valign="top" align="center">32626565</td><td rowspan="1" colspan="1" valign="top" align="center">14kb from <em>HLA-DQA1</em></td><td rowspan="1" colspan="1" valign="top" align="center">675bp from <em>HLA-DQB1</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.1000</td><td rowspan="1" colspan="1" valign="top" align="center">1.18×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.61 (1.38-1.87)</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">8.43×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.38 (1.18-1.63)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs114653103</td><td rowspan="1" colspan="1" valign="top" align="center">32668846</td><td rowspan="1" colspan="1" valign="top" align="center">34kb from <em>HLA-DQB1</em></td><td rowspan="1" colspan="1" valign="top" align="center">40kb from <em>HLA-DQA2</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.1200</td><td rowspan="1" colspan="1" valign="top" align="center">7.31×10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.57 (0.49-0.66)</td><td rowspan="1" colspan="1" valign="top" align="center">G/T</td><td rowspan="1" colspan="1" valign="top" align="center">1.81×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.50 (0.41-0.60)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs115253455</td><td rowspan="1" colspan="1" valign="top" align="center">33100021</td><td rowspan="1" colspan="1" valign="top" align="center">43kb from <em>HLA-DPB1</em></td><td rowspan="1" colspan="1" valign="top" align="center">30kb from <em>COL11A2</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.1300</td><td rowspan="1" colspan="1" valign="top" align="center">4.68×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.70 (0.61-0.80)</td><td rowspan="1" colspan="1" valign="top" align="center">T/A</td><td rowspan="1" colspan="1" valign="top" align="center">4.51×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.66 (0.56-0.77)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">chr6:31996524</td><td rowspan="1" colspan="1" valign="top" align="center">31996524</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>C4B</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.1700</td><td rowspan="1" colspan="1" valign="top" align="center">4.95×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.71 (0.63-0.80)</td><td rowspan="1" colspan="1" valign="top" align="center">C/A</td><td rowspan="1" colspan="1" valign="top" align="center">4.87×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.64 (0.55-0.73)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs113833333</td><td rowspan="1" colspan="1" valign="top" align="center">32594898</td><td rowspan="1" colspan="1" valign="top" align="center">37kb from <em>HLA-DRB1</em></td><td rowspan="1" colspan="1" valign="top" align="center">10kb from <em>HLA-DQA1</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.4100</td><td rowspan="1" colspan="1" valign="top" align="center">2.30×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.82 (0.74-0.90)</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">3.09×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.74 (0.67-0.82)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs116427960</td><td rowspan="1" colspan="1" valign="top" align="center">31319226</td><td rowspan="1" colspan="1" valign="top" align="center">79kb from <em>HLA-C</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.4kb from <em>HLA-B</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.0081</td><td rowspan="1" colspan="1" valign="top" align="center">8.96×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.57 (2.49-8.38)</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">1.69×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.17 (1.87-5.35)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs114904515</td><td rowspan="1" colspan="1" valign="top" align="center">29362756</td><td rowspan="1" colspan="1" valign="top" align="center">20kb from <em>OR12D3</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.7kb from <em>OR12D2</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.1000</td><td rowspan="1" colspan="1" valign="top" align="center">1.55×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.69 (0.60-0.80)</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">5.70×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.67 (0.56-0.79)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs118044183</td><td rowspan="1" colspan="1" valign="top" align="center">30954150</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>MUC21</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.1200</td><td rowspan="1" colspan="1" valign="top" align="center">3.61×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.50 (1.27-1.79)</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">1.39×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.39 (1.17-1.64)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2736191</td><td rowspan="1" colspan="1" valign="top" align="center">31560910</td><td rowspan="1" colspan="1" valign="top" align="center">~150bp from <em>NCR3</em></td><td rowspan="1" colspan="1" valign="top" align="center">22kb from <em>AIF1</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0.3800</td><td rowspan="1" colspan="1" valign="top" align="center">7.53×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.79 (0.71-0.86)</td><td rowspan="1" colspan="1" valign="top" align="center">C/G</td><td rowspan="1" colspan="1" valign="top" align="center">1.01×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.81 (0.73-0.90)</td></tr></tbody></table></div><div id="largeobj_idm140248621224192" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4981330/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN12"><sup>A</sup>All variants within this table have been imputed.</div><div id="TFN13"><sup>B</sup>CI = Confidence interval</div><div id="TFN14"><sup>C</sup>MAF = Minor allele frequency</div><div id="TFN15"><sup>D</sup>Maj/Min = Major allele/Minor allele</div><p id="__p7">Note: Tables are in the order they were identified in the stepwise model. The stepwise results presented in this table are for adjusting for all other variants in the table. For complete results in the HLA region, please refer to <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>.</p></div></div><p id="P23" class="p p-last">To better understand the relationship between the variants reported in this study and the classical HLA alleles, we imputed alleles at HLA A, B, C, DPB1, DQA1, DQB1 and DRB1. The peak statistical significance was observed at P = 5.55×10<sup>−16</sup> for the two tightly linked alleles, HLA-DRB1*1501 (OR = 1.85; 95% CI = 1.59-2.14) and HLA-DQB1*0602 (OR = 1.90; 95% CI = 1.62-2.21; <a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>, <a href="/pmc/articles/PMC4981330/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>, and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). Stepwise logistic regression modeling of the HiBAG-imputed HLA alleles identified 13 independent effects (<a href="/pmc/articles/PMC4981330/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). To better relate the classical alleles to the variants identified in this GWA scan, stepwise modeling was done with both SNPs and classical HLA alleles. The peak effect after 1000 Genomes imputation was rs116727542, which accounted for by HLA-DQB1*0602 and HLA-DRB1*0803 (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4B</a> for results of the additional 8 rounds of the stepwise regression analysis).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p8">Multi-locus model of HIBAG-imputed HLA dosages, single locus and stepwise results</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140248612471664" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="2" rowspan="1" valign="top" align="center">
<span class="underline">Dosage Frequency</span>
</th><th colspan="2" rowspan="1" valign="top" align="center">
<span class="underline">Best Guess Count</span>
</th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">HLA Allele</th><th rowspan="1" colspan="1" valign="top" align="center">Cases</th><th rowspan="1" colspan="1" valign="top" align="center">Controls</th><th rowspan="1" colspan="1" valign="top" align="center">Cases</th><th rowspan="1" colspan="1" valign="top" align="center">Controls</th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI<sup><sup>A</sup></sup>)</th><th rowspan="1" colspan="1" valign="top" align="center">Single Locus P-value</th><th rowspan="1" colspan="1" valign="top" align="center">Stepwise P-value</th><th rowspan="1" colspan="1" valign="top" align="center">Stepwise OR (95%CI<sup><sup>A</sup></sup>)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">DQB1*0602<sup><sup>B</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">301</td><td rowspan="1" colspan="1" valign="top" align="center">636</td><td rowspan="1" colspan="1" valign="top" align="center">1.90 (1.62 - 2.21)</td><td rowspan="1" colspan="1" valign="top" align="center">5.55×10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.93×10<sup>−23</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.35 (1.99 – 2.78)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DRB1*0803</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">251</td><td rowspan="1" colspan="1" valign="top" align="center">613</td><td rowspan="1" colspan="1" valign="top" align="center">1.59 (1.34 - 1.88)</td><td rowspan="1" colspan="1" valign="top" align="center">7.37×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.63×10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.14 (1.78 - 2.58)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DQB1*0202</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">212</td><td rowspan="1" colspan="1" valign="top" align="center">502</td><td rowspan="1" colspan="1" valign="top" align="center">1.60 (1.35 - 1.90)</td><td rowspan="1" colspan="1" valign="top" align="center">7.57×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.19×10<sup>−18</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.50 (2.04 - 3.07)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DQA1*0302</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">286</td><td rowspan="1" colspan="1" valign="top" align="center">847</td><td rowspan="1" colspan="1" valign="top" align="center">1.41 (1.15 - 1.75)</td><td rowspan="1" colspan="1" valign="top" align="center">1.27×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.92×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.98 (1.58 - 2.50)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">B*0801</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">5.43 (2.66 - 11.08)</td><td rowspan="1" colspan="1" valign="top" align="center">3.42×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.15×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.71 (2.70 -12.07)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DQA1*0401</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">54</td><td rowspan="1" colspan="1" valign="top" align="center">133</td><td rowspan="1" colspan="1" valign="top" align="center">1.73 (1.17 - 2.57)</td><td rowspan="1" colspan="1" valign="top" align="center">5.94×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.11×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.36 (1.58 - 3.54)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">C*0702</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">261</td><td rowspan="1" colspan="1" valign="top" align="center">711</td><td rowspan="1" colspan="1" valign="top" align="center">1.37 (1.17 - 1.59)</td><td rowspan="1" colspan="1" valign="top" align="center">5.45×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.06×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.30 (1.10 - 1.54)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DRB1*0406</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">41</td><td rowspan="1" colspan="1" valign="top" align="center">354</td><td rowspan="1" colspan="1" valign="top" align="center">0.15 (0.09 - 0.26)</td><td rowspan="1" colspan="1" valign="top" align="center">2.63×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.11×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.32 (0.18 - 0.57)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DPB1*0501</td><td rowspan="1" colspan="1" valign="top" align="center">0.79</td><td rowspan="1" colspan="1" valign="top" align="center">0.72</td><td rowspan="1" colspan="1" valign="top" align="center">930</td><td rowspan="1" colspan="1" valign="top" align="center">3092</td><td rowspan="1" colspan="1" valign="top" align="center">1.16 (1.05 - 1.28)</td><td rowspan="1" colspan="1" valign="top" align="center">2.87×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.06×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.23 (1.11 - 1.36)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DRB1*1602</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">38</td><td rowspan="1" colspan="1" valign="top" align="center">87</td><td rowspan="1" colspan="1" valign="top" align="center">1.78 (1.16 - 2.72)</td><td rowspan="1" colspan="1" valign="top" align="center">7.73×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.53×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.05 (1.32 - 3.20)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DPB1*1701</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">41</td><td rowspan="1" colspan="1" valign="top" align="center">160</td><td rowspan="1" colspan="1" valign="top" align="center">0.90 (0.62 - 1.30)</td><td rowspan="1" colspan="1" valign="top" align="center">5.66×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.07×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.5 (0.33 - 0.76)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">C*0102</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">359</td><td rowspan="1" colspan="1" valign="top" align="center">1429</td><td rowspan="1" colspan="1" valign="top" align="center">0.88 (0.77 - 1.01)</td><td rowspan="1" colspan="1" valign="top" align="center">6.51×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.82×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.82 (0.71 - 0.94)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DRB1*1202</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">39</td><td rowspan="1" colspan="1" valign="top" align="center">266</td><td rowspan="1" colspan="1" valign="top" align="center">0.52 (0.37 - 0.74)</td><td rowspan="1" colspan="1" valign="top" align="center">2.98×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.07×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.61 (0.42 - 0.87)</td></tr></tbody></table></div><div id="largeobj_idm140248612471664" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4981330/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN16"><sup>A</sup>CI = Confidence interval</div><div id="TFN17"><sup>B</sup>The allele HLA-DRB1*1501 had the same P-value as HLA-DQB1*0602, but the later was selected by the stepwise modeling procedure. For complete results, please refer to <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>.</div><p id="__p9">Note: Tables are in the order they were identified in the stepwise model. The stepwise results presented in this table are for adjusting for all other variants in the table</p></div></div></div><div id="S15" class="sec"><h3 id="S15title">Non-HLA SLE associations previously reported and identification of novel independent effects</h3><p id="P24" class="p p-first">Several previously identified non-HLA SLE loci were also replicated in this study, including: STAT1-STAT4, TNFSF4, TNFAIP3, IKZF1, HIP1, IRF5, BLK, WDFY4, ETS1, and <em>IRAK1-MECP2</em> (<a href="/pmc/articles/PMC4981330/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B-D</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 5 to 11, and Supplementary Tables 6 to 15</a>. Of these loci, the associations in the region of <em>TNFAIP3, TNFSF4</em>, and <em>WDFY4</em> have notable differences from previous studies.</p><p id="P25">After imputation of the <em>TNFAIP3</em> region, the primary independent effect in the stepwise model was observed at rs5029937 located within the second intron of <em>TNFAIP3</em> (<a href="/pmc/articles/PMC4981330/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a>). The second independent effect was identified at rs9373203 3’ of the <em>TNFAIP3</em> coding region. A previous SLE study in Han Chinese(<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186796">6</a>) reported rs2230926 as associated with disease, and another SLE transracial mapping study in Koreans (with partial overlap of subjects with the current study) and Europeans(<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186767">31</a>) identified risk of SLE with rs7749323. Both variants (rs2230926 and rs7749323) are highly correlated with rs5029937, with D’ = 1.0 and <em>r<sup>2</sup></em> &gt; 0.98, indicating consistency between our results and these previous reports (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 12</a>). In addition, the second effect tagged by rs9373203 was not identified in either the Han et al or Adrianto et al. studies (<a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a>). Musone et al. (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186775">32</a>) identified multiple effects, some of which spanned even further 3’ of <em>TNFAIP3</em> than rs9373203. After their stepwise analysis, they identified rs6922466 as the tagging variant accounting for this association; however, this variant is not associated with SLE in the current study of Koreans. Moreover, the LD between these variants is very weak in Koreans (<em>r<sup>2</sup></em> = 0.00; D’ = 0.43), giving additional evidence that rs9373203 may be an independent effect warranting further study (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 12</a>).</p><p id="P26">In the region of <em>TNFSF4</em> after imputation, two independent effects were observed in the stepwise model. The first effect, peaking at rs76413021, is located in the first intron of the <em>TNFSF4</em> coding region (<a href="/pmc/articles/PMC4981330/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2C</span></a>). This variant is in LD with rs2205960 (D’=0.98, <em>r<sup>2</sup></em>=0.94) and rs1234315 (D’ = 0.97, <em>r<sup>2</sup></em> = 0.48) and was previously identified from the Han Chinese GWA scan(<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186768">6</a>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 13</a>). Moreover, this effect is consistent with the results reported in Europeans(<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186779">33</a>). The second independent effect, which is distinct from previous studies, peaks at rs4916342, which is located in an intron LOC100506023 just 5’ of <em>TNFSF4</em> (<a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2C</span></a>). Neither Han et al. (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186830">6</a>) nor Cunninghame Graham et al. (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186772">33</a>) reported association signals as far 5’ of <em>TNFSF4</em> as our observation of the second independent effect tagged by rs4916342. In a transracial mapping study of this region by Manku et al. (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186812">34</a>) that included subjects from East Asia, a second independent effect (tagged by rs1234314) was identified that is. In our current study of Koreans, we found that the first effect, tagged by rs76413021, accounted for rs1234314 in the stepwise model. Although rs1234314 and rs4916342 are located in the same general genomic location, the LD structure further supports the observation that they are not the same genetic effect; however, all the risk variants are located on a single risk haplotype (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 13</a>). This suggests that risk alleles for both rs76413021 and rs4916342 are needed to confer susceptibility to disease.</p><p id="P27" class="p p-last">In the region of <em>WDFY4</em>, the current study did not replicate rs877819, which has been previously reported to result in a downregulation of <em>WDFY4</em> through modification of a YY1 binding site (<a href="/pmc/articles/PMC4981330/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>)(<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186822">35</a>). However, the results for <em>WDFY4</em> in Koreans are consistent with two previous studies. First, the most statistically significant association within this region, the coding variant rs7097397 leading to an amino acid substitution R1816Q (P = 2.10×10<sup>−9</sup>), was previously reported by Yang et al. (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186808">8</a>). Second, we also demonstrate association for rs1913517, which was identified previously by Han et al. (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186816">6</a>) (P = 2.54×10<sup>−5</sup>; <a href="/pmc/articles/PMC4981330/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2D</span></a>). Our haplotype and stepwise regression analysis indicated that there were two independent effects in the region, with rs7097397 accounting for the association observed at rs1913517, and rs10857631 tagging the second independent effect (<a href="/pmc/articles/PMC4981330/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2D</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 14 and 15</a>).</p></div><div id="S16" class="sec sec-last"><h3 id="S16title">Suggestive association with SLE identified in the genome-wide phase</h3><p id="P28" class="p p-first-last">In total, 15 genotyped variants surpassed the suggestive threshold of P &lt; 2×10<sup>−6</sup> and were considered for further replication (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 16</a>). Replication was attempted for three additional variants located within <em>GTF2IRD1, DOCK1</em>, and <em>AHNAK2,</em> all of which have multiple genotyped variants showing suggestive significance and/or have been previously implicated in other related phenotypes (<a href="/pmc/articles/PMC4981330/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Only rs2267828 near <em>GTF2IRD1</em> yielded a P<sub>meta-rep</sub> &lt; 0.05, but this variant did not surpass genome-wide significance after meta-analysis with the GWA scan (<a href="/pmc/articles/PMC4981330/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The variant in the region of <em>AHNAK2,</em> rs1048257, was trending towards significance, while rs10901656 near <em>DOCK1</em> showed association in one replication cohort with the opposite allele (<a href="/pmc/articles/PMC4981330/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Outside of the 10 regions previously reported SLE loci described above, we observed only eight additional loci with 5×10<sup>−8</sup> &lt; P &lt; 5×10<sup>−5</sup> on the list of ~50 that have been described previously (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 17</a>). This is likely due to the limited power of this study and/or results from population-specific differences from the studies in which these discoveries were originally identified.</p></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Discussion</h2><p id="P29" class="p p-first">The association in this region peaks between three candidate genes, <em>ATG16L2</em>, <em>FCHSD2</em>, and <em>P2RY2</em>, all of which have the biological potential to impact SLE pathophysiology. While this locus has not been reported in other systemic autoimmune diseases, variants in this region are associated with Crohn's disease (MIM266600) in Korean subjects(<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186790">36</a>). Moreover, the peak variant identified in Crohn's disease, rs11235667, was also the variant discovered in this current study with SLE. The missense variant, rs11235604, was also reported to be associated in Crohn's disease(<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186797">36</a>). GWA studies conducted in SLE in Europeans did not identify this locus since it is monomorphic in that population. Moreover, GWA scans in Han Chinese both used the Illumina Human 610-Quad bead chip, which did not contain rs11235667(<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186800">6</a>-<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186824">8</a>).</p><p id="P30"><em>ATG16L2</em> (autophagy related 16-like 2) is a ubiquitously expressed homologue of the gene <em>ATG16L1</em> (MIM610767) that has been implicated as a risk locus for Crohn's disease in patients of European descent(<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186774">37</a>, <a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186780">38</a>). Both loci are involved in autophagy; however, little is known about the role <em>ATG16L2</em> plays in the process. Interestingly, this pathway has been previously implicated in SLE. The gene <em>ATG5</em> (MIM604261) has also been implicated as a risk locus for lupus(<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186801">6</a>, <a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186827">39</a>). Studies in the mouse have shown that Apg16l (the mouse equivalent of human ATG16L) interacts with Apg5 (the mouse equivalent of human ATG5) suggesting that it is possible that ATG16L2 and ATG5 may also interact humans(<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186792">40</a>). More studies are needed to understand the function <em>ATG16L2</em> and if it is involved in the association with SLE.</p><p id="P31"><em>FCHSD2</em> (FCH and double SH3 domains 2) has been described as regulator of F-actin assembly through interactions with WAS (also known as WASP) and WASL (also known as N-WASP)(<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186782">41</a>). <em>FCHSD2</em> is primarily expressed in CD19+ B cells, dendritic cells, myeloid cells, CD4+ T cells, and CD8+ T cells(<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186831">38</a>). Previous studies have shown that WAS plays an important role in the migration of T cells through reorganization of the actin cytoskeleton subsequent to interactions with dendritic or B cells(<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186777">42</a>).</p><p id="P32"><em>P2RY2</em> (purinergic receptor P2Y, G-protein coupled, 2) is known to be involved in many cellular functions and is expressed in myeloid cells including monocytes(<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186785">38</a>). P2RY2 is a receptor for ATP and UTP that acts as a sensor for the release of nucleotides by apoptotic cells(<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186806">43</a>). Mice null for P2RY2 showed a decreased ability to recruit monocytes and macrophages upon activation of nucleotides from apoptotic cells(<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186778">43</a>). P2RY2 is also known to induce CCL2 secretion in macrophages, and coding variants in the receptor have been shown to influence secretion of this proinflammatory chemokine(<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186794">44</a>).</p><p id="P33">Although the HLA region has been implicated in SLE susceptibility since the 1970s, the precise loci responsible for risk have not been fully characterized. A further cross comparison of populations will be beneficial to take advantage of differences in linkage disequilibrium and will likely help further refine association signals seen in the GWA studies. For the classical alleles, previous studies have identified associations with alleles in the HLA-DR locus in Europeans, Chinese, Japanese, and Koreans, but HLA-DQB1*0602 has not been implicated in Koreans before this current study(<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186813">45</a>-<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186829">48</a>). Two prominent Classical HLA alleles identified in Europeans with SLE showed differences in association in Koreans. While HLA-DRB1*1501 was among the most significantly associated with SLE, HLA-DRB1*0301 was found to be at low frequency in this population and is not associated with SLE (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). These results are consistent with a recent study in Koreans that evaluated the role of HLA in this population(<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_577950652">49</a>). Moreover, this study report that amino acid changes to HLA-DRB1 at positions 11, 13, and 26 account for the HLA association in SLE(<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_577950655">49</a>). However, it is possible that there are other amino acid changing variants, non-coding RNAs, and/or transcriptional changes for other HLA loci that are co-inherited with HLA-DRB1 on these haplotypes may also contributing to SLE risk.</p><p id="P34">This GWA scan replicated several loci that have been identified by prior studies, including STAT1-STAT4, TNFSF4, TNFAIP3, IKZF1, HIP1, IRF5, BLK, WDFY4, ETS1, and <em>IRAK1-MECP2</em>. It is important to note that a previous GWA scan of Korean women with SLE has also reported replication of <em>STAT4</em> and <em>BLK</em> at a genome-wide significant level(<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186791">30</a>). Of these replicated regions, <em>TNFSF4, TNFAIP3, IKZF1, HIP1, IRF5, BLK,</em> and <em>ETS1</em> have functional effects that have been previously described (see details in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 18</a>). Although most of the signals in these loci are identical, we did describe notable differences with independent effects in <em>TNFSF4</em> and <em>TNFAIP3</em>. Moreover, we did not observe association with rs877819, which had been proposed as putative causal variant leading to expression differences of <em>WDFY4(<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_551186826">35</a>)</em>.</p><p id="P35" class="p p-last">In conclusion, we performed a GWA scan of Korean SLE cases and population controls in which we identified 12 regions that surpassed genome-wide significance. The region from <em>ATG16L2</em> through <em>FCHSD2</em> to the promoter region of the <em>P2RY2</em> locus was identified and confirmed as an SLE-associated region. Here, we also observed strong associations in the HLA region and showed the relationship between the classical HLA alleles and the variants reported within this GWA study in Koreans. The ten additional regions, STAT1-STAT4, TNFSF4, TNFAIP3, IKZF1, HIP1, IRF5, BLK, WDFY4, ETS1, and <em>IRAK1-MECP2</em> have previously been implicated in SLE. Additional replication is needed for the suggestive loci identified in this study to determine their relationship with SLE. Although GWA approaches have been very successful in the identification of risk loci, continued efforts are need to narrow association signals to the causal variant(s) and to determine the functional causal mechanism(s) contributing to SLE pathogenesis. 
</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplemental 1</h4><div class="sup-box half_rhythm" id="d37e1373"><a href="/pmc/articles/PMC4981330/bin/NIHMS804385-supplement-Supplemental_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.3M, pdf)</sup></div></div></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Acknowledgments</h2><div class="sec"><p id="__p1">We are grateful to all the individuals with SLE and those serving as healthy controls who participated in this study. Sample collection and phenotyping of the subjects utilized in this study was made possible by funds from the Korea Healthcare Technology R&amp;D Project of the Ministry for Health &amp; Welfare (HI13C2124, S.C.B.). This publication was made possible by grants 5P01 <a id="__tag_801884250" class="tag_hotlink" href="/nuccore/AI083194" ref="/nuccore/AI083194"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AI083194","term_id":"3421617","term_text":"AI083194"}}</span>AI083194</a> (C.J.L., A.R., P.M.G., M.E.A.R., L.A.C., C.O.J., R.P.K., T.J.V., J.B.H., C.D.L., K.L.S, and B.P.T.), 5P01 <a id="__tag_801884251" class="tag_hotlink" href="/nuccore/AR049084" ref="/nuccore/AR049084"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AR049084","term_id":"6005123","term_text":"AR049084"}}</span>AR049084</a> (R.P.K. and C.D.L.), R01 AR043814 (B.P.T.), R21 AR065626 (B.P.T.) by the NIH. The contents are the sole responsibility of the authors and do not necessarily represent the official views of the NIH. Additional support was obtained from Wake Forest School of Medicine Center for Public Health Genomics (C.D.L.), Oklahoma Medical Research Foundation (C.J.L., P.M.G., and K.L.S.), the Alliance for Lupus Research (L.A.C. and B.P.T.), and Kirkland Scholar Award (L.A.C.). The out-of-study Korean control data was provided from Korean Biobank Project supported by the Korea Center for Disease Control and Prevention at the Korea National Institute of Health.</p></div></div><div id="idm140248623159984" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140248623159984title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P36" class="p p-first-last"><strong>Author Contributions:</strong> C.J.L., J.A.K., A.R., M.E.A.R., L.A.C., C.O.J., R.P.K., T.J.V., J.B.H., K.L.S., S.-C.B., C.D.L., and B.P.T were responsible for the study design. K.K, S.-Y.B., Y.B.J., J.C., H.-S.L., Y.M.K., C.-H.S., W.T.C., S.-K.L., J.-Y.C., S.C.S., J.H.O., Y.J.K., B.-G.H., N.S., H.S.H., Y.W.S., S.-C.B., and B.P.T. assisted in the collection and characterization of the SS cases and healthy controls. J.Z., J.A.I., E.K.W., Q.-Z.L., and P.M.G. performed the genotyping. C.J.L., S.S, H.A., performed all analyses under the guidance of K.L.S., C.D.L. and B.P.T. C.J.L., S.S., J.Z., K.K., H.L., J.A.I., K.L.S., S.-C.B., C.D.L., and B.P.T. prepared the manuscript, and all authors approved the final draft.</p><p></p><p class="fn sec" id="FN3"></p><p id="P37" class="p p-first">Competing Financial Interests:</p><p id="P38" class="p p-last">The authors declare no competing financial interests.</p><p></p><p class="fn sec" id="FN4"></p><p id="P39" class="p p-first">Supplemental Data Summary</p><p id="P40" class="p p-last">Supplemental data includes 16 tables and 14 figures.</p><p></p><p class="fn sec" id="FN5"></p><p id="P41" class="p p-first">Web-based Resources:</p><p id="P42" class="p">OMIM, <a href="http://www.omim.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.omim.org/</a></p><p id="P43" class="p">SNPGWA version 4.0, <a href="http://www.phs.wfubmc.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.phs.wfubmc.edu</a></p><p id="P44" class="p p-last">SHAPEIT, <a href="http://www.shapeit.fr/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.shapeit.fr/</a></p><p></p></div></div><div id="idm140248697826944" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140248697826944title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Tsokos GC. Systemic lupus erythematosus. <span><span class="ref-journal">N Engl J Med. </span>2011;<span class="ref-vol">365</span>(22):2110–21.</span> [<a href="/pubmed/22129255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Systemic+lupus+erythematosus.&amp;author=GC+Tsokos&amp;volume=365&amp;issue=22&amp;publication_year=2011&amp;pages=2110-21&amp;pmid=22129255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Arbuckle MR, McClain MT, Rubertone MV, Scofield RH, Dennis GJ, James JA, et al.  Development of autoantibodies before the clinical onset of systemic lupus erythematosus. <span><span class="ref-journal">N Engl J Med. </span>2003;<span class="ref-vol">349</span>(16):1526–33.</span> [<a href="/pubmed/14561795" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Development+of+autoantibodies+before+the+clinical+onset+of+systemic+lupus+erythematosus.&amp;author=MR+Arbuckle&amp;author=MT+McClain&amp;author=MV+Rubertone&amp;author=RH+Scofield&amp;author=GJ+Dennis&amp;volume=349&amp;issue=16&amp;publication_year=2003&amp;pages=1526-33&amp;pmid=14561795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Moser KL, Kelly JA, Lessard CJ, Harley JB. Recent insights into the genetic basis of systemic lupus erythematosus. <span><span class="ref-journal">Genes and immunity. </span>2009;<span class="ref-vol">10</span>(5):373–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3144759/">PMC free article</a>]</span> [<a href="/pubmed/19440199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+and+immunity&amp;title=Recent+insights+into+the+genetic+basis+of+systemic+lupus+erythematosus.&amp;author=KL+Moser&amp;author=JA+Kelly&amp;author=CJ+Lessard&amp;author=JB+Harley&amp;volume=10&amp;issue=5&amp;publication_year=2009&amp;pages=373-9&amp;pmid=19440199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Deng Y, Tsao BP. Advances in lupus genetics and epigenetics. <span><span class="ref-journal">Curr Opin Rheumatol. </span>2014;<span class="ref-vol">26</span>(5):482–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4222581/">PMC free article</a>]</span> [<a href="/pubmed/25010439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Rheumatol&amp;title=Advances+in+lupus+genetics+and+epigenetics.&amp;author=Y+Deng&amp;author=BP+Tsao&amp;volume=26&amp;issue=5&amp;publication_year=2014&amp;pages=482-92&amp;pmid=25010439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Dai C, Deng Y, Quinlan A, Gaskin F, Tsao BP, Fu SM. Genetics of systemic lupus erythematosus: immune responses and end organ resistance to damage. <span><span class="ref-journal">Curr Opin Immunol. </span>2014;<span class="ref-vol">31C</span>:87–96.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4274270/">PMC free article</a>]</span> [<a href="/pubmed/25458999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Immunol&amp;title=Genetics+of+systemic+lupus+erythematosus:+immune+responses+and+end+organ+resistance+to+damage.&amp;author=C+Dai&amp;author=Y+Deng&amp;author=A+Quinlan&amp;author=F+Gaskin&amp;author=BP+Tsao&amp;volume=31C&amp;publication_year=2014&amp;pages=87-96&amp;pmid=25458999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Han JW, Zheng HF, Cui Y, Sun LD, Ye DQ, Hu Z, et al.  Genome-wide association study in a Chinese Han population identifies nine new susceptibility loci for systemic lupus erythematosus. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>(11):1234–7.</span> [<a href="/pubmed/19838193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+in+a+Chinese+Han+population+identifies+nine+new+susceptibility+loci+for+systemic+lupus+erythematosus.&amp;author=JW+Han&amp;author=HF+Zheng&amp;author=Y+Cui&amp;author=LD+Sun&amp;author=DQ+Ye&amp;volume=41&amp;issue=11&amp;publication_year=2009&amp;pages=1234-7&amp;pmid=19838193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Yang W, Tang H, Zhang Y, Tang X, Zhang J, Sun L, et al.  Meta-analysis followed by replication identifies loci in or near CDKN1B, TET3, CD80, DRAM1, and ARID5B as associated with systemic lupus erythematosus in Asians. <span><span class="ref-journal">American journal of human genetics. </span>2013;<span class="ref-vol">92</span>(1):41–51.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3542470/">PMC free article</a>]</span> [<a href="/pubmed/23273568" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+journal+of+human+genetics&amp;title=Meta-analysis+followed+by+replication+identifies+loci+in+or+near+CDKN1B,+TET3,+CD80,+DRAM1,+and+ARID5B+as+associated+with+systemic+lupus+erythematosus+in+Asians.&amp;author=W+Yang&amp;author=H+Tang&amp;author=Y+Zhang&amp;author=X+Tang&amp;author=J+Zhang&amp;volume=92&amp;issue=1&amp;publication_year=2013&amp;pages=41-51&amp;pmid=23273568&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Yang W, Shen N, Ye DQ, Liu Q, Zhang Y, Qian XX, et al.  Genome-wide association study in Asian populations identifies variants in ETS1 and WDFY4 associated with systemic lupus erythematosus. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>(2):e1000841.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820522/">PMC free article</a>]</span> [<a href="/pubmed/20169177" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+in+Asian+populations+identifies+variants+in+ETS1+and+WDFY4+associated+with+systemic+lupus+erythematosus.&amp;author=W+Yang&amp;author=N+Shen&amp;author=DQ+Ye&amp;author=Q+Liu&amp;author=Y+Zhang&amp;volume=6&amp;issue=2&amp;publication_year=2010&amp;pages=e1000841&amp;pmid=20169177&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Okada Y, Shimane K, Kochi Y, Tahira T, Suzuki A, Higasa K, et al.  A genome-wide association study identified AFF1 as a susceptibility locus for systemic lupus eyrthematosus in Japanese. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>(1):e1002455.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3266877/">PMC free article</a>]</span> [<a href="/pubmed/22291604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+association+study+identified+AFF1+as+a+susceptibility+locus+for+systemic+lupus+eyrthematosus+in+Japanese.&amp;author=Y+Okada&amp;author=K+Shimane&amp;author=Y+Kochi&amp;author=T+Tahira&amp;author=A+Suzuki&amp;volume=8&amp;issue=1&amp;publication_year=2012&amp;pages=e1002455&amp;pmid=22291604&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Kim K, Bang SY, Lee HS, Cho SK, Choi CB, Sung YK, et al.  High-density genotyping of immune loci in Koreans and Europeans identifies eight new rheumatoid arthritis risk loci. <span><span class="ref-journal">Annals of the rheumatic diseases. </span>2014</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4467986/">PMC free article</a>]</span> [<a href="/pubmed/24532676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+the+rheumatic+diseases&amp;title=High-density+genotyping+of+immune+loci+in+Koreans+and+Europeans+identifies+eight+new+rheumatoid+arthritis+risk+loci.&amp;author=K+Kim&amp;author=SY+Bang&amp;author=HS+Lee&amp;author=SK+Cho&amp;author=CB+Choi&amp;publication_year=2014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Lessard CJ, Adrianto I, Ice JA, Wiley GB, Kelly JA, Glenn SB, et al.  Identification of IRF8, TMEM39A, and IKZF3-ZPBP2 as susceptibility loci for systemic lupus erythematosus in a large-scale multiracial replication study. <span><span class="ref-journal">American journal of human genetics. </span>2012;<span class="ref-vol">90</span>(4):648–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3322228/">PMC free article</a>]</span> [<a href="/pubmed/22464253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+journal+of+human+genetics&amp;title=Identification+of+IRF8,+TMEM39A,+and+IKZF3-ZPBP2+as+susceptibility+loci+for+systemic+lupus+erythematosus+in+a+large-scale+multiracial+replication+study.&amp;author=CJ+Lessard&amp;author=I+Adrianto&amp;author=JA+Ice&amp;author=GB+Wiley&amp;author=JA+Kelly&amp;volume=90&amp;issue=4&amp;publication_year=2012&amp;pages=648-60&amp;pmid=22464253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Kaiser R, Taylor KE, Deng Y, Zhao J, Li Y, Nititham J, et al.  Brief Report: Single-nucleotide polymorphisms in VKORC1 are risk factors for systemic lupus erythematosus in Asians. <span><span class="ref-journal">Arthritis and rheumatism. </span>2013;<span class="ref-vol">65</span>(1):211–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3670944/">PMC free article</a>]</span> [<a href="/pubmed/23124848" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+rheumatism&amp;title=Brief+Report:+Single-nucleotide+polymorphisms+in+VKORC1+are+risk+factors+for+systemic+lupus+erythematosus+in+Asians.&amp;author=R+Kaiser&amp;author=KE+Taylor&amp;author=Y+Deng&amp;author=J+Zhao&amp;author=Y+Li&amp;volume=65&amp;issue=1&amp;publication_year=2013&amp;pages=211-5&amp;pmid=23124848&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Hochberg MC. Updating the American College of Rheumatology revised criteria for the classification of systemic lupus erythematosus. <span><span class="ref-journal">Arthritis and rheumatism. </span>1997;<span class="ref-vol">40</span>(9):1725.</span> [<a href="/pubmed/9324032" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+rheumatism&amp;title=Updating+the+American+College+of+Rheumatology+revised+criteria+for+the+classification+of+systemic+lupus+erythematosus.&amp;author=MC+Hochberg&amp;volume=40&amp;issue=9&amp;publication_year=1997&amp;pages=1725&amp;pmid=9324032&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Manichaikul A, Mychaleckyj JC, Rich SS, Daly K, Sale M, Chen WM. Robust relationship inference in genome-wide association studies. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>(22):2867–73.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025716/">PMC free article</a>]</span> [<a href="/pubmed/20926424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Robust+relationship+inference+in+genome-wide+association+studies.&amp;author=A+Manichaikul&amp;author=JC+Mychaleckyj&amp;author=SS+Rich&amp;author=K+Daly&amp;author=M+Sale&amp;volume=26&amp;issue=22&amp;publication_year=2010&amp;pages=2867-73&amp;pmid=20926424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Frazer KA, Ballinger DG, Cox DR, Hinds DA, Stuve LL, Gibbs RA, et al.  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>(7164):851–61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs.&amp;author=KA+Frazer&amp;author=DG+Ballinger&amp;author=DR+Cox&amp;author=DA+Hinds&amp;author=LL+Stuve&amp;volume=449&amp;issue=7164&amp;publication_year=2007&amp;pages=851-61&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>(8):904–9.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies.&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;issue=8&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>(17):2190–1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans.&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;issue=17&amp;publication_year=2010&amp;pages=2190-1&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Cochran WG. The Combination of Estimates from Different Experiments. <span><span class="ref-journal">Biometrics. </span>1954;<span class="ref-vol">10</span>:101–29.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=The+Combination+of+Estimates+from+Different+Experiments.&amp;author=WG+Cochran&amp;volume=10&amp;publication_year=1954&amp;pages=101-29&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Higgins JP, Thompson SG, Deeks JJ, Altman DG. Measuring inconsistency in meta-analyses. <span><span class="ref-journal">Bmj. </span>2003;<span class="ref-vol">327</span>(7414):557–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC192859/">PMC free article</a>]</span> [<a href="/pubmed/12958120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bmj&amp;title=Measuring+inconsistency+in+meta-analyses.&amp;author=JP+Higgins&amp;author=SG+Thompson&amp;author=JJ+Deeks&amp;author=DG+Altman&amp;volume=327&amp;issue=7414&amp;publication_year=2003&amp;pages=557-60&amp;pmid=12958120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Durbin RM, Abecasis GR, Altshuler DL, Auton A, Brooks LD, Gibbs RA, et al.  A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>(7319):1061–73.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing.&amp;author=RM+Durbin&amp;author=GR+Abecasis&amp;author=DL+Altshuler&amp;author=A+Auton&amp;author=LD+Brooks&amp;volume=467&amp;issue=7319&amp;publication_year=2010&amp;pages=1061-73&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Delaneau O, Marchini J, Zagury JF. A linear complexity phasing method for thousands of genomes. <span><span class="ref-journal">Nat Methods. </span>2012;<span class="ref-vol">9</span>(2):179–81.</span> [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes.&amp;author=O+Delaneau&amp;author=J+Marchini&amp;author=JF+Zagury&amp;volume=9&amp;issue=2&amp;publication_year=2012&amp;pages=179-81&amp;pmid=22138821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>(6):e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies.&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;issue=6&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Marchini J, Howie B. Genotype imputation for genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>(7):499–511.</span> [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genotype+imputation+for+genome-wide+association+studies.&amp;author=J+Marchini&amp;author=B+Howie&amp;volume=11&amp;issue=7&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Zheng X, Shen J, Cox C, Wakefield JC, Ehm MG, Nelson MR, et al.  HIBAG--HLA genotype imputation with attribute bagging. <span><span class="ref-journal">The pharmacogenomics journal. </span>2014;<span class="ref-vol">14</span>(2):192–200.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772955/">PMC free article</a>]</span> [<a href="/pubmed/23712092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+pharmacogenomics+journal&amp;title=HIBAG--HLA+genotype+imputation+with+attribute+bagging.&amp;author=X+Zheng&amp;author=J+Shen&amp;author=C+Cox&amp;author=JC+Wakefield&amp;author=MG+Ehm&amp;volume=14&amp;issue=2&amp;publication_year=2014&amp;pages=192-200&amp;pmid=23712092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Ward LD, Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. <span><span class="ref-journal">Nucleic Acids Res. </span>2012;<span class="ref-vol">40</span>(Database issue):D930–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245002/">PMC free article</a>]</span> [<a href="/pubmed/22064851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=HaploReg:+a+resource+for+exploring+chromatin+states,+conservation,+and+regulatory+motif+alterations+within+sets+of+genetically+linked+variants.&amp;author=LD+Ward&amp;author=M+Kellis&amp;volume=40&amp;issue=Database+issue&amp;publication_year=2012&amp;pages=D930-4&amp;pmid=22064851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Dunham I, Kundaje A, Aldred SF, Collins PJ, Davis CA, Doyle F, et al.  An integrated encyclopedia of DNA elements in the human genome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>(7414):57–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3439153/">PMC free article</a>]</span> [<a href="/pubmed/22955616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+encyclopedia+of+DNA+elements+in+the+human+genome.&amp;author=I+Dunham&amp;author=A+Kundaje&amp;author=SF+Aldred&amp;author=PJ+Collins&amp;author=CA+Davis&amp;volume=489&amp;issue=7414&amp;publication_year=2012&amp;pages=57-74&amp;pmid=22955616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Lister R, Pelizzola M, Dowen RH, Hawkins RD, Hon G, Tonti-Filippini J, et al.  Human DNA methylomes at base resolution show widespread epigenomic differences. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">462</span>(7271):315–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2857523/">PMC free article</a>]</span> [<a href="/pubmed/19829295" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Human+DNA+methylomes+at+base+resolution+show+widespread+epigenomic+differences.&amp;author=R+Lister&amp;author=M+Pelizzola&amp;author=RH+Dowen&amp;author=RD+Hawkins&amp;author=G+Hon&amp;volume=462&amp;issue=7271&amp;publication_year=2009&amp;pages=315-22&amp;pmid=19829295&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Raj T, Rothamel K, Mostafavi S, Ye C, Lee MN, Replogle JM, et al.  Polarization of the effects of autoimmune and neurodegenerative risk alleles in leukocytes. <span><span class="ref-journal">Science. </span>2014;<span class="ref-vol">344</span>(6183):519–23.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4910825/">PMC free article</a>]</span> [<a href="/pubmed/24786080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Polarization+of+the+effects+of+autoimmune+and+neurodegenerative+risk+alleles+in+leukocytes.&amp;author=T+Raj&amp;author=K+Rothamel&amp;author=S+Mostafavi&amp;author=C+Ye&amp;author=MN+Lee&amp;volume=344&amp;issue=6183&amp;publication_year=2014&amp;pages=519-23&amp;pmid=24786080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Adzhubei IA, Schmidt S, Peshkin L, Ramensky VE, Gerasimova A, Bork P, et al.  A method and server for predicting damaging missense mutations. <span><span class="ref-journal">Nat Methods. </span>2010;<span class="ref-vol">7</span>(4):248–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2855889/">PMC free article</a>]</span> [<a href="/pubmed/20354512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=A+method+and+server+for+predicting+damaging+missense+mutations.&amp;author=IA+Adzhubei&amp;author=S+Schmidt&amp;author=L+Peshkin&amp;author=VE+Ramensky&amp;author=A+Gerasimova&amp;volume=7&amp;issue=4&amp;publication_year=2010&amp;pages=248-9&amp;pmid=20354512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Lee HS, Kim T, Bang SY, Na YJ, Kim I, Kim K, et al.  Ethnic specificity of lupus-associated loci identified in a genome-wide association study in Korean women. <span><span class="ref-journal">Annals of the rheumatic diseases. </span>2014;<span class="ref-vol">73</span>(6):1240–5.</span> [<a href="/pubmed/23740238" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+the+rheumatic+diseases&amp;title=Ethnic+specificity+of+lupus-associated+loci+identified+in+a+genome-wide+association+study+in+Korean+women.&amp;author=HS+Lee&amp;author=T+Kim&amp;author=SY+Bang&amp;author=YJ+Na&amp;author=I+Kim&amp;volume=73&amp;issue=6&amp;publication_year=2014&amp;pages=1240-5&amp;pmid=23740238&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Adrianto I, Wen F, Templeton A, Wiley G, King JB, Lessard CJ, et al.  Association of a functional variant downstream of TNFAIP3 with systemic lupus erythematosus. <span><span class="ref-journal">Nature genetics. </span>2011;<span class="ref-vol">43</span>(3):253–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3103780/">PMC free article</a>]</span> [<a href="/pubmed/21336280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Association+of+a+functional+variant+downstream+of+TNFAIP3+with+systemic+lupus+erythematosus.&amp;author=I+Adrianto&amp;author=F+Wen&amp;author=A+Templeton&amp;author=G+Wiley&amp;author=JB+King&amp;volume=43&amp;issue=3&amp;publication_year=2011&amp;pages=253-8&amp;pmid=21336280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Musone SL, Taylor KE, Lu TT, Nititham J, Ferreira RC, Ortmann W, et al.  Multiple polymorphisms in the TNFAIP3 region are independently associated with systemic lupus erythematosus. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>(9):1062–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3897246/">PMC free article</a>]</span> [<a href="/pubmed/19165919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+polymorphisms+in+the+TNFAIP3+region+are+independently+associated+with+systemic+lupus+erythematosus.&amp;author=SL+Musone&amp;author=KE+Taylor&amp;author=TT+Lu&amp;author=J+Nititham&amp;author=RC+Ferreira&amp;volume=40&amp;issue=9&amp;publication_year=2008&amp;pages=1062-4&amp;pmid=19165919&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Cunninghame Graham DS, Graham RR, Manku H, Wong AK, Whittaker JC, Gaffney PM, et al.  Polymorphism at the TNF superfamily gene TNFSF4 confers susceptibility to systemic lupus erythematosus. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>(1):83–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3705866/">PMC free article</a>]</span> [<a href="/pubmed/18059267" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Polymorphism+at+the+TNF+superfamily+gene+TNFSF4+confers+susceptibility+to+systemic+lupus+erythematosus.&amp;author=DS+Cunninghame+Graham&amp;author=RR+Graham&amp;author=H+Manku&amp;author=AK+Wong&amp;author=JC+Whittaker&amp;volume=40&amp;issue=1&amp;publication_year=2008&amp;pages=83-9&amp;pmid=18059267&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Manku H, Langefeld CD, Guerra SG, Malik TH, Alarcon-Riquelme M, Anaya JM, et al.  Trans-ancestral studies fine map the SLE-susceptibility locus TNFSF4. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>(7):e1003554.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3715547/">PMC free article</a>]</span> [<a href="/pubmed/23874208" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Trans-ancestral+studies+fine+map+the+SLE-susceptibility+locus+TNFSF4.&amp;author=H+Manku&amp;author=CD+Langefeld&amp;author=SG+Guerra&amp;author=TH+Malik&amp;author=M+Alarcon-Riquelme&amp;volume=9&amp;issue=7&amp;publication_year=2013&amp;pages=e1003554&amp;pmid=23874208&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Zhao H, Yang W, Qiu R, Li J, Xin Q, Wang X, et al.  An intronic variant associated with systemic lupus erythematosus changes the binding affinity of Yinyang1 to downregulate WDFY4. <span><span class="ref-journal">Genes and immunity. </span>2012;<span class="ref-vol">13</span>(7):536–42.</span> [<a href="/pubmed/22972472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+and+immunity&amp;title=An+intronic+variant+associated+with+systemic+lupus+erythematosus+changes+the+binding+affinity+of+Yinyang1+to+downregulate+WDFY4.&amp;author=H+Zhao&amp;author=W+Yang&amp;author=R+Qiu&amp;author=J+Li&amp;author=Q+Xin&amp;volume=13&amp;issue=7&amp;publication_year=2012&amp;pages=536-42&amp;pmid=22972472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Yang SK, Hong M, Zhao W, Jung Y, Baek J, Tayebi N, et al.  Genome-wide association study of Crohn's disease in Koreans revealed three new susceptibility loci and common attributes of genetic susceptibility across ethnic populations. <span><span class="ref-journal">Gut. </span>2014;<span class="ref-vol">63</span>(1):80–7.</span> [<a href="/pubmed/23850713" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Genome-wide+association+study+of+Crohn's+disease+in+Koreans+revealed+three+new+susceptibility+loci+and+common+attributes+of+genetic+susceptibility+across+ethnic+populations.&amp;author=SK+Yang&amp;author=M+Hong&amp;author=W+Zhao&amp;author=Y+Jung&amp;author=J+Baek&amp;volume=63&amp;issue=1&amp;publication_year=2014&amp;pages=80-7&amp;pmid=23850713&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Hampe J, Franke A, Rosenstiel P, Till A, Teuber M, Huse K, et al.  A genome-wide association scan of nonsynonymous SNPs identifies a susceptibility variant for Crohn disease in ATG16L1. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>(2):207–11.</span> [<a href="/pubmed/17200669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+scan+of+nonsynonymous+SNPs+identifies+a+susceptibility+variant+for+Crohn+disease+in+ATG16L1.&amp;author=J+Hampe&amp;author=A+Franke&amp;author=P+Rosenstiel&amp;author=A+Till&amp;author=M+Teuber&amp;volume=39&amp;issue=2&amp;publication_year=2007&amp;pages=207-11&amp;pmid=17200669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Wu C, Orozco C, Boyer J, Leglise M, Goodale J, Batalov S, et al.  BioGPS: an extensible and customizable portal for querying and organizing gene annotation resources. <span><span class="ref-journal">Genome biology. </span>2009;<span class="ref-vol">10</span>(11):R130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3091323/">PMC free article</a>]</span> [<a href="/pubmed/19919682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+biology&amp;title=BioGPS:+an+extensible+and+customizable+portal+for+querying+and+organizing+gene+annotation+resources.&amp;author=C+Wu&amp;author=C+Orozco&amp;author=J+Boyer&amp;author=M+Leglise&amp;author=J+Goodale&amp;volume=10&amp;issue=11&amp;publication_year=2009&amp;pages=R130&amp;pmid=19919682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Harley JB, Alarcon-Riquelme ME, Criswell LA, Jacob CO, Kimberly RP, Moser KL, et al.  Genome-wide association scan in women with systemic lupus erythematosus identifies susceptibility variants in ITGAM, PXK, KIAA1542 and other loci. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>(2):204–10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3712260/">PMC free article</a>]</span> [<a href="/pubmed/18204446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+scan+in+women+with+systemic+lupus+erythematosus+identifies+susceptibility+variants+in+ITGAM,+PXK,+KIAA1542+and+other+loci.&amp;author=JB+Harley&amp;author=ME+Alarcon-Riquelme&amp;author=LA+Criswell&amp;author=CO+Jacob&amp;author=RP+Kimberly&amp;volume=40&amp;issue=2&amp;publication_year=2008&amp;pages=204-10&amp;pmid=18204446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Mizushima N, Kuma A, Kobayashi Y, Yamamoto A, Matsubae M, Takao T, et al.  Mouse Apg16L, a novel WD-repeat protein, targets to the autophagic isolation membrane with the Apg12-Apg5 conjugate. <span><span class="ref-journal">Journal of cell science. </span>2003;<span class="ref-vol">116</span>(Pt 9):1679–88.</span> [<a href="/pubmed/12665549" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+cell+science&amp;title=Mouse+Apg16L,+a+novel+WD-repeat+protein,+targets+to+the+autophagic+isolation+membrane+with+the+Apg12-Apg5+conjugate.&amp;author=N+Mizushima&amp;author=A+Kuma&amp;author=Y+Kobayashi&amp;author=A+Yamamoto&amp;author=M+Matsubae&amp;volume=116&amp;issue=Pt+9&amp;publication_year=2003&amp;pages=1679-88&amp;pmid=12665549&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Cao H, Yin X, Cao Y, Jin Y, Wang S, Kong Y, et al.  FCHSD1 and FCHSD2 are expressed in hair cell stereocilia and cuticular plate and regulate actin polymerization in vitro. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>(2):e56516.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3577914/">PMC free article</a>]</span> [<a href="/pubmed/23437151" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=FCHSD1+and+FCHSD2+are+expressed+in+hair+cell+stereocilia+and+cuticular+plate+and+regulate+actin+polymerization+in+vitro.&amp;author=H+Cao&amp;author=X+Yin&amp;author=Y+Cao&amp;author=Y+Jin&amp;author=S+Wang&amp;volume=8&amp;issue=2&amp;publication_year=2013&amp;pages=e56516&amp;pmid=23437151&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Lafouresse F, Cotta-de-Almeida V, Malet-Engra G, Galy A, Valitutti S, Dupre L. Wiskott-Aldrich syndrome protein controls antigen-presenting cell-driven CD4+ T-cell motility by regulating adhesion to intercellular adhesion molecule-1. <span><span class="ref-journal">Immunology. </span>2012;<span class="ref-vol">137</span>(2):183–96.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3461399/">PMC free article</a>]</span> [<a href="/pubmed/22804504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunology&amp;title=Wiskott-Aldrich+syndrome+protein+controls+antigen-presenting+cell-driven+CD4++T-cell+motility+by+regulating+adhesion+to+intercellular+adhesion+molecule-1.&amp;author=F+Lafouresse&amp;author=V+Cotta-de-Almeida&amp;author=G+Malet-Engra&amp;author=A+Galy&amp;author=S+Valitutti&amp;volume=137&amp;issue=2&amp;publication_year=2012&amp;pages=183-96&amp;pmid=22804504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Elliott MR, Chekeni FB, Trampont PC, Lazarowski ER, Kadl A, Walk SF, et al.  Nucleotides released by apoptotic cells act as a find-me signal to promote phagocytic clearance. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">461</span>(7261):282–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2851546/">PMC free article</a>]</span> [<a href="/pubmed/19741708" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Nucleotides+released+by+apoptotic+cells+act+as+a+find-me+signal+to+promote+phagocytic+clearance.&amp;author=MR+Elliott&amp;author=FB+Chekeni&amp;author=PC+Trampont&amp;author=ER+Lazarowski&amp;author=A+Kadl&amp;volume=461&amp;issue=7261&amp;publication_year=2009&amp;pages=282-6&amp;pmid=19741708&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Higgins KR, Kovacevic W, Stokes L. Nucleotides regulate secretion of the inflammatory chemokine CCL2 from human macrophages and monocytes. <span><span class="ref-journal">Mediators of inflammation. </span>2014;<span class="ref-vol">2014</span>:293925.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4170906/">PMC free article</a>]</span> [<a href="/pubmed/25276053" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mediators+of+inflammation&amp;title=Nucleotides+regulate+secretion+of+the+inflammatory+chemokine+CCL2+from+human+macrophages+and+monocytes.&amp;author=KR+Higgins&amp;author=W+Kovacevic&amp;author=L+Stokes&amp;volume=2014&amp;publication_year=2014&amp;pages=293925&amp;pmid=25276053&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Lee HS, Chung YH, Kim TG, Kim TH, Jun JB, Jung S, et al.  Independent association of HLA-DR and FCgamma receptor polymorphisms in Korean patients with systemic lupus erythematosus. <span><span class="ref-journal">Rheumatology (Oxford) </span>2003;<span class="ref-vol">42</span>(12):1501–7.</span> [<a href="/pubmed/12867584" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Rheumatology+(Oxford)&amp;title=Independent+association+of+HLA-DR+and+FCgamma+receptor+polymorphisms+in+Korean+patients+with+systemic+lupus+erythematosus.&amp;author=HS+Lee&amp;author=YH+Chung&amp;author=TG+Kim&amp;author=TH+Kim&amp;author=JB+Jun&amp;volume=42&amp;issue=12&amp;publication_year=2003&amp;pages=1501-7&amp;pmid=12867584&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Graham RR, Ortmann W, Rodine P, Espe K, Langefeld C, Lange E, et al.  Specific combinations of HLA-DR2 and DR3 class II haplotypes contribute graded risk for disease susceptibility and autoantibodies in human SLE. <span><span class="ref-journal">Eur J Hum Genet. </span>2007;<span class="ref-vol">15</span>(8):823–30.</span> [<a href="/pubmed/17406641" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Specific+combinations+of+HLA-DR2+and+DR3+class+II+haplotypes+contribute+graded+risk+for+disease+susceptibility+and+autoantibodies+in+human+SLE.&amp;author=RR+Graham&amp;author=W+Ortmann&amp;author=P+Rodine&amp;author=K+Espe&amp;author=C+Langefeld&amp;volume=15&amp;issue=8&amp;publication_year=2007&amp;pages=823-30&amp;pmid=17406641&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Furukawa H, Oka S, Shimada K, Sugii S, Hashimoto A, Komiya A, et al.  Association of increased frequencies of HLA-DPB1*05:01 with the presence of anti-Ro/SS-A and anti-La/SS-B antibodies in Japanese rheumatoid arthritis and systemic lupus erythematosus patients. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>(1):e53910.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3540046/">PMC free article</a>]</span> [<a href="/pubmed/23320107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Association+of+increased+frequencies+of+HLA-DPB1*05:01+with+the+presence+of+anti-Ro/SS-A+and+anti-La/SS-B+antibodies+in+Japanese+rheumatoid+arthritis+and+systemic+lupus+erythematosus+patients.&amp;author=H+Furukawa&amp;author=S+Oka&amp;author=K+Shimada&amp;author=S+Sugii&amp;author=A+Hashimoto&amp;volume=8&amp;issue=1&amp;publication_year=2013&amp;pages=e53910&amp;pmid=23320107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Furukawa H, Kawasaki A, Oka S, Ito I, Shimada K, Sugii S, et al.  Human leukocyte antigens and systemic lupus erythematosus: a protective role for the HLA-DR6 alleles DRB1*13:02 and *14:03. <span><span class="ref-journal">PLoS One. </span>2014;<span class="ref-vol">9</span>(2):e87792.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912000/">PMC free article</a>]</span> [<a href="/pubmed/24498373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Human+leukocyte+antigens+and+systemic+lupus+erythematosus:+a+protective+role+for+the+HLA-DR6+alleles+DRB1*13:02+and+*14:03.&amp;author=H+Furukawa&amp;author=A+Kawasaki&amp;author=S+Oka&amp;author=I+Ito&amp;author=K+Shimada&amp;volume=9&amp;issue=2&amp;publication_year=2014&amp;pages=e87792&amp;pmid=24498373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Kim K, Bang SY, Lee HS, Okada Y, Han B, Saw WY, et al.  The HLADRbeta1 amino acid positions 11-13-26 explain the majority of SLE-MHC associations. <span><span class="ref-journal">Nature communications. </span>2014;<span class="ref-vol">5</span>:5902.</span> [<a href="/pubmed/25533202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+communications&amp;title=The+HLADRbeta1+amino+acid+positions+11-13-26+explain+the+majority+of+SLE-MHC+associations.&amp;author=K+Kim&amp;author=SY+Bang&amp;author=HS+Lee&amp;author=Y+Okada&amp;author=B+Han&amp;volume=5&amp;publication_year=2014&amp;pages=5902&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4981330&amp;issue-id=273963&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140248599380720title">Abstract</a></li><li><a href="#S5title">Methods</a></li><li><a href="#S11title">Results</a></li><li><a href="#S17title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S18title">Acknowledgments</a></li><li><a href="#idm140248623159984title">Footnotes</a></li><li><a href="#idm140248697826944title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4981330/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4981330/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4981330/pdf/nihms-804385.pdf">PDF (865K)</a></li> | <li><a href="#" data-citationid="PMC4981330" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4981330%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4981330%2F&amp;text=Identification%20of%20a%20systemic%20lupus%20erythematosus%20risk%20locus%20spanning%20ATG16L2%2C%20FCHSD2%2C%20and%20P2RY2%20in%20Koreans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4981330%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26663301" data-aiid="4981330" data-aid="4981330" data-iid="273963" data-domainid="319" data-domain="nihpa" data-accid="PMC4981330" data-md5="9da66277dcddeea1d388357c14bab54e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:20:13-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D9B57DD5916E1000000000052002F&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4981330%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4981330/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>