<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4909543/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of Susceptibility Loci and Genes for Colorectal Cancer Risk</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4909543/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Gastroenterology"><meta name="citation_title" content="Identification of Susceptibility Loci and Genes for Colorectal Cancer Risk"><meta name="citation_authors" content="Chenjie Zeng, Koichi Matsuda, Wei-Hua Jia, Jiang Chang, Sun-Seog Kweon, Yong-Bing Xiang, Aesun Shin, Sun Ha Jee, Dong-Hyun Kim, Ben Zhang, Qiuyin Cai, Xingyi Guo, Jirong Long, Nan Wang, Regina Courtney, Zhi-Zhong Pan, Chen Wu, Atsushi Takahashi, Min-Ho Shin, Keitaro Matsuo, Fumihiko Matsuda, Yu-Tang Gao, Jae Hwan Oh, Soriul Kim, Keum Ji Jung, Yoon-Ok Ahn, Zefang Ren, Hong-Lan Li, Jie Wu, Jiajun Shi, Wanqing Wen, Gong Yang, Bingshan Li, Bu-Tian Ji, Genetics and Epidemiology of Colorectal Cancer Consortium (GECCO), Hermann Brenner, Robert E. Schoen, Sébastien Küry, Colorectal Transdisciplinary (CORECT) Study, Stephen B. Gruber, Fredrick R. Schumacher, Stephanie L. Stenzel, Colon Cancer Family Registry (CCFR), Graham Casey, John L. Hopper, Mark A. Jenkins, Hyeong-Rok Kim, Jin-Young Jeong, Ji Won Park, Kazuo Tajima, Sang-Hee Cho, Michiaki Kubo, Xiao-Ou Shu, Dongxin Lin, Yi-Xin Zeng, Wei Zheng"><meta name="citation_date" content="June 2016"><meta name="citation_issue" content="7"><meta name="citation_volume" content="150"><meta name="citation_firstpage" content="1633"><meta name="citation_doi" content="10.1053/j.gastro.2016.02.076"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4909543/?report=abstract"><meta name="citation_pmid" content="26965516"><meta name="DC.Title" content="Identification of Susceptibility Loci and Genes for Colorectal Cancer Risk"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Chenjie Zeng"><meta name="DC.Contributor" content="Koichi Matsuda"><meta name="DC.Contributor" content="Wei-Hua Jia"><meta name="DC.Contributor" content="Jiang Chang"><meta name="DC.Contributor" content="Sun-Seog Kweon"><meta name="DC.Contributor" content="Yong-Bing Xiang"><meta name="DC.Contributor" content="Aesun Shin"><meta name="DC.Contributor" content="Sun Ha Jee"><meta name="DC.Contributor" content="Dong-Hyun Kim"><meta name="DC.Contributor" content="Ben Zhang"><meta name="DC.Contributor" content="Qiuyin Cai"><meta name="DC.Contributor" content="Xingyi Guo"><meta name="DC.Contributor" content="Jirong Long"><meta name="DC.Contributor" content="Nan Wang"><meta name="DC.Contributor" content="Regina Courtney"><meta name="DC.Contributor" content="Zhi-Zhong Pan"><meta name="DC.Contributor" content="Chen Wu"><meta name="DC.Contributor" content="Atsushi Takahashi"><meta name="DC.Contributor" content="Min-Ho Shin"><meta name="DC.Contributor" content="Keitaro Matsuo"><meta name="DC.Contributor" content="Fumihiko Matsuda"><meta name="DC.Contributor" content="Yu-Tang Gao"><meta name="DC.Contributor" content="Jae Hwan Oh"><meta name="DC.Contributor" content="Soriul Kim"><meta name="DC.Contributor" content="Keum Ji Jung"><meta name="DC.Contributor" content="Yoon-Ok Ahn"><meta name="DC.Contributor" content="Zefang Ren"><meta name="DC.Contributor" content="Hong-Lan Li"><meta name="DC.Contributor" content="Jie Wu"><meta name="DC.Contributor" content="Jiajun Shi"><meta name="DC.Contributor" content="Wanqing Wen"><meta name="DC.Contributor" content="Gong Yang"><meta name="DC.Contributor" content="Bingshan Li"><meta name="DC.Contributor" content="Bu-Tian Ji"><meta name="DC.Contributor" content="Genetics and Epidemiology of Colorectal Cancer Consortium (GECCO)"><meta name="DC.Contributor" content="Hermann Brenner"><meta name="DC.Contributor" content="Robert E. Schoen"><meta name="DC.Contributor" content="Sébastien Küry"><meta name="DC.Contributor" content="Colorectal Transdisciplinary (CORECT) Study"><meta name="DC.Contributor" content="Stephen B. Gruber"><meta name="DC.Contributor" content="Fredrick R. Schumacher"><meta name="DC.Contributor" content="Stephanie L. Stenzel"><meta name="DC.Contributor" content="Colon Cancer Family Registry (CCFR)"><meta name="DC.Contributor" content="Graham Casey"><meta name="DC.Contributor" content="John L. Hopper"><meta name="DC.Contributor" content="Mark A. Jenkins"><meta name="DC.Contributor" content="Hyeong-Rok Kim"><meta name="DC.Contributor" content="Jin-Young Jeong"><meta name="DC.Contributor" content="Ji Won Park"><meta name="DC.Contributor" content="Kazuo Tajima"><meta name="DC.Contributor" content="Sang-Hee Cho"><meta name="DC.Contributor" content="Michiaki Kubo"><meta name="DC.Contributor" content="Xiao-Ou Shu"><meta name="DC.Contributor" content="Dongxin Lin"><meta name="DC.Contributor" content="Yi-Xin Zeng"><meta name="DC.Contributor" content="Wei Zheng"><meta name="DC.Date" content="2016 Jun"><meta name="DC.Identifier" content="10.1053/j.gastro.2016.02.076"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of Susceptibility Loci and Genes for Colorectal Cancer Risk"><meta property="og:type" content="article"><meta property="og:description" content="Known Genetic factors explain only a small fraction of genetic variation in colorectal cancer (CRC). We conducted a genome-wide association study (GWAS) to identify risk loci for CRC.This discovery stage included 8027 cases and 22577 controls of East-Asian ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4909543/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4909543/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4909543/pdf/nihms766698.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8DA629DD59210100000000015600DC.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4909543%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4909543</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Gastroenterology</a></span>. Author manuscript; available in PMC 2017 Jun 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26965516" target="pmc_ext" ref="reftype=publisher&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Gastroenterology. 2016 Jun; 150(7): 1633–1645. </span></a></div><span class="fm-vol-iss-date">Published online 2016 Mar 8. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1053%2Fj.gastro.2016.02.076" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1053/j.gastro.2016.02.076</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4909543</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS766698</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26965516">26965516</a></div></div></div></div><h1 class="content-title">Identification of Susceptibility Loci and Genes for Colorectal Cancer Risk</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Zeng%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959289153440" co-class="co-affbox">Chenjie Zeng</a>,<sup>1</sup> <a href="/pubmed/?term=Matsuda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959292816208" co-class="co-affbox">Koichi Matsuda</a>,<sup>2</sup> <a href="/pubmed/?term=Jia%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959289151072" co-class="co-affbox">Wei-Hua Jia</a>,<sup>3</sup> <a href="/pubmed/?term=Chang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959289149200" co-class="co-affbox">Jiang Chang</a>,<sup>4</sup> <a href="/pubmed/?term=Kweon%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959289147216" co-class="co-affbox">Sun-Seog Kweon</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Xiang%20YB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959289144496" co-class="co-affbox">Yong-Bing Xiang</a>,<sup>7</sup> <a href="/pubmed/?term=Shin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959256431904" co-class="co-affbox">Aesun Shin</a>,<sup>8,</sup><sup>9</sup> <a href="/pubmed/?term=Jee%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959256429184" co-class="co-affbox">Sun Ha Jee</a>,<sup>10</sup> <a href="/pubmed/?term=Kim%20DH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959256427200" co-class="co-affbox">Dong-Hyun Kim</a>,<sup>11</sup> <a href="/pubmed/?term=Zhang%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959256425216" co-class="co-affbox">Ben Zhang</a>,<sup>1</sup> <a href="/pubmed/?term=Cai%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959256423232" co-class="co-affbox">Qiuyin Cai</a>,<sup>1</sup> <a href="/pubmed/?term=Guo%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959256421248" co-class="co-affbox">Xingyi Guo</a>,<sup>1</sup> <a href="/pubmed/?term=Long%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257674832" co-class="co-affbox">Jirong Long</a>,<sup>1</sup> <a href="/pubmed/?term=Wang%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257672848" co-class="co-affbox">Nan Wang</a>,<sup>1,</sup><sup>12</sup> <a href="/pubmed/?term=Courtney%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257670128" co-class="co-affbox">Regina Courtney</a>,<sup>1</sup> <a href="/pubmed/?term=Pan%20ZZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257668144" co-class="co-affbox">Zhi-Zhong Pan</a>,<sup>3</sup> <a href="/pubmed/?term=Wu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257666160" co-class="co-affbox">Chen Wu</a>,<sup>4</sup> <a href="/pubmed/?term=Takahashi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257664176" co-class="co-affbox">Atsushi Takahashi</a>,<sup>13</sup> <a href="/pubmed/?term=Shin%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257662192" co-class="co-affbox">Min-Ho Shin</a>,<sup>5</sup> <a href="/pubmed/?term=Matsuo%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257660208" co-class="co-affbox">Keitaro Matsuo</a>,<sup>14</sup> <a href="/pubmed/?term=Matsuda%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257658224" co-class="co-affbox">Fumihiko Matsuda</a>,<sup>15</sup> <a href="/pubmed/?term=Gao%20YT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257656240" co-class="co-affbox">Yu-Tang Gao</a>,<sup>7</sup> <a href="/pubmed/?term=Oh%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257654256" co-class="co-affbox">Jae Hwan Oh</a>,<sup>16</sup> <a href="/pubmed/?term=Kim%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257652272" co-class="co-affbox">Soriul Kim</a>,<sup>10</sup> <a href="/pubmed/?term=Jung%20KJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959257650288" co-class="co-affbox">Keum Ji Jung</a>,<sup>10</sup> <a href="/pubmed/?term=Ahn%20YO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254299024" co-class="co-affbox">Yoon-Ok Ahn</a>,<sup>9</sup> <a href="/pubmed/?term=Ren%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254297040" co-class="co-affbox">Zefang Ren</a>,<sup>17</sup> <a href="/pubmed/?term=Li%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254295056" co-class="co-affbox">Hong-Lan Li</a>,<sup>7</sup> <a href="/pubmed/?term=Wu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254293072" co-class="co-affbox">Jie Wu</a>,<sup>1</sup> <a href="/pubmed/?term=Shi%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254291088" co-class="co-affbox">Jiajun Shi</a>,<sup>1</sup> <a href="/pubmed/?term=Wen%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254289104" co-class="co-affbox">Wanqing Wen</a>,<sup>1</sup> <a href="/pubmed/?term=Yang%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254287120" co-class="co-affbox">Gong Yang</a>,<sup>1</sup> <a href="/pubmed/?term=Li%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254285136" co-class="co-affbox">Bingshan Li</a>,<sup>18</sup> <a href="/pubmed/?term=Ji%20BT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254283152" co-class="co-affbox">Bu-Tian Ji</a>,<sup>19</sup> Genetics and Epidemiology of Colorectal Cancer Consortium (GECCO),<sup>20</sup> <a href="/pubmed/?term=Brenner%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254279808" co-class="co-affbox">Hermann Brenner</a>,<sup>21,</sup><sup>22,</sup><sup>23</sup> <a href="/pubmed/?term=Schoen%20RE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254276352" co-class="co-affbox">Robert E. Schoen</a>,<sup>24</sup> <a href="/pubmed/?term=K%26%23x000fc%3Bry%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959254274368" co-class="co-affbox">Sébastien Küry</a>,<sup>25</sup> Colorectal Transdisciplinary (CORECT) Study,<sup>20</sup> <a href="/pubmed/?term=Gruber%20SB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251179936" co-class="co-affbox">Stephen B. Gruber</a>,<sup>26</sup> <a href="/pubmed/?term=Schumacher%20FR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251177952" co-class="co-affbox">Fredrick R. Schumacher</a>,<sup>26</sup> <a href="/pubmed/?term=Stenzel%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251175968" co-class="co-affbox">Stephanie L. Stenzel</a>,<sup>26</sup> Colon Cancer Family Registry (CCFR),<sup>20</sup> <a href="/pubmed/?term=Casey%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251172624" co-class="co-affbox">Graham Casey</a>,<sup>26</sup> <a href="/pubmed/?term=Hopper%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251170640" co-class="co-affbox">John L. Hopper</a>,<sup>27</sup> <a href="/pubmed/?term=Jenkins%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251168656" co-class="co-affbox">Mark A. Jenkins</a>,<sup>28</sup> <a href="/pubmed/?term=Kim%20HR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251166672" co-class="co-affbox">Hyeong-Rok Kim</a>,<sup>29</sup> <a href="/pubmed/?term=Jeong%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251164688" co-class="co-affbox">Jin-Young Jeong</a>,<sup>11</sup> <a href="/pubmed/?term=Park%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251162704" co-class="co-affbox">Ji Won Park</a>,<sup>16,</sup><sup>30</sup> <a href="/pubmed/?term=Tajima%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251159984" co-class="co-affbox">Kazuo Tajima</a>,<sup>31</sup> <a href="/pubmed/?term=Cho%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251158000" co-class="co-affbox">Sang-Hee Cho</a>,<sup>32</sup> <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959251156016" co-class="co-affbox">Michiaki Kubo</a>,<sup>13</sup> <a href="/pubmed/?term=Shu%20XO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959289273120" co-class="co-affbox">Xiao-Ou Shu</a>,<sup>1</sup> <a href="/pubmed/?term=Lin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959289271136" co-class="co-affbox">Dongxin Lin</a>,<sup>4</sup> <a href="/pubmed/?term=Zeng%20YX%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959289269152" co-class="co-affbox">Yi-Xin Zeng</a>,<sup>3</sup> and  <a href="/pubmed/?term=Zheng%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516" class="affpopup" co-rid="_co_idm139959289267168" co-class="co-affbox">Wei Zheng</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139959289153440"><h3 class="no_margin">Chenjie Zeng</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Zeng%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Chenjie Zeng</a></div></div><div id="_co_idm139959292816208"><h3 class="no_margin">Koichi Matsuda</h3><p><sup>2</sup>Laboratory of Genome Technology, Human Genome Center, Institute of Medical Science, The University of Tokyo, Tokyo, Japan</p><div>Find articles by <a href="/pubmed/?term=Matsuda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Koichi Matsuda</a></div></div><div id="_co_idm139959289151072"><h3 class="no_margin">Wei-Hua Jia</h3><p><sup>3</sup>State Key Laboratory of Oncology in South China, Cancer Center, Sun Yat-sen University, Guangzhou, China</p><div>Find articles by <a href="/pubmed/?term=Jia%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Wei-Hua Jia</a></div></div><div id="_co_idm139959289149200"><h3 class="no_margin">Jiang Chang</h3><p><sup>4</sup>State Key Laboratory of Molecular Oncology, Cancer Institute and Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China</p><div>Find articles by <a href="/pubmed/?term=Chang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Jiang Chang</a></div></div><div id="_co_idm139959289147216"><h3 class="no_margin">Sun-Seog Kweon</h3><p><sup>5</sup>Department of Preventive Medicine, Chonnam National University Medical School, Gwangju, South Korea</p><p><sup>6</sup>Jeonnam Regional Cancer Center, Chonnam National University Hwasun Hospital, Hwasun, South Korea</p><div>Find articles by <a href="/pubmed/?term=Kweon%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Sun-Seog Kweon</a></div></div><div id="_co_idm139959289144496"><h3 class="no_margin">Yong-Bing Xiang</h3><p><sup>7</sup>Department of Epidemiology, Shanghai Cancer Institute, Shanghai, China</p><div>Find articles by <a href="/pubmed/?term=Xiang%20YB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Yong-Bing Xiang</a></div></div><div id="_co_idm139959256431904"><h3 class="no_margin">Aesun Shin</h3><p><sup>8</sup>Molecular Epidemiology Branch, National Cancer Center, Goyang-si, South Korea</p><p><sup>9</sup>Department of Preventive Medicine, Seoul National University College of Medicine, Seoul, South Korea</p><div>Find articles by <a href="/pubmed/?term=Shin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Aesun Shin</a></div></div><div id="_co_idm139959256429184"><h3 class="no_margin">Sun Ha Jee</h3><p><sup>10</sup>Institute for Health Promotion, Department of Epidemiology and Health Promotion, Graduate School of Public Health, Yonsei University, Seoul, South Korea</p><div>Find articles by <a href="/pubmed/?term=Jee%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Sun Ha Jee</a></div></div><div id="_co_idm139959256427200"><h3 class="no_margin">Dong-Hyun Kim</h3><p><sup>11</sup>Department of Social and Preventive Medicine, Hallym University College of Medicine, Okcheon-dong, South Korea</p><div>Find articles by <a href="/pubmed/?term=Kim%20DH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Dong-Hyun Kim</a></div></div><div id="_co_idm139959256425216"><h3 class="no_margin">Ben Zhang</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Zhang%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Ben Zhang</a></div></div><div id="_co_idm139959256423232"><h3 class="no_margin">Qiuyin Cai</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Cai%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Qiuyin Cai</a></div></div><div id="_co_idm139959256421248"><h3 class="no_margin">Xingyi Guo</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Guo%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Xingyi Guo</a></div></div><div id="_co_idm139959257674832"><h3 class="no_margin">Jirong Long</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Long%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Jirong Long</a></div></div><div id="_co_idm139959257672848"><h3 class="no_margin">Nan Wang</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><p><sup>12</sup>General Department, Tangdu Hospital, Fourth Military Medical University, Xi’an, Shaanxi, China</p><div>Find articles by <a href="/pubmed/?term=Wang%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Nan Wang</a></div></div><div id="_co_idm139959257670128"><h3 class="no_margin">Regina Courtney</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Courtney%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Regina Courtney</a></div></div><div id="_co_idm139959257668144"><h3 class="no_margin">Zhi-Zhong Pan</h3><p><sup>3</sup>State Key Laboratory of Oncology in South China, Cancer Center, Sun Yat-sen University, Guangzhou, China</p><div>Find articles by <a href="/pubmed/?term=Pan%20ZZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Zhi-Zhong Pan</a></div></div><div id="_co_idm139959257666160"><h3 class="no_margin">Chen Wu</h3><p><sup>4</sup>State Key Laboratory of Molecular Oncology, Cancer Institute and Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China</p><div>Find articles by <a href="/pubmed/?term=Wu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Chen Wu</a></div></div><div id="_co_idm139959257664176"><h3 class="no_margin">Atsushi Takahashi</h3><p><sup>13</sup>RIKEN Center for Integrative Medical Sciences, Kanagawa, Japan</p><div>Find articles by <a href="/pubmed/?term=Takahashi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Atsushi Takahashi</a></div></div><div id="_co_idm139959257662192"><h3 class="no_margin">Min-Ho Shin</h3><p><sup>5</sup>Department of Preventive Medicine, Chonnam National University Medical School, Gwangju, South Korea</p><div>Find articles by <a href="/pubmed/?term=Shin%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Min-Ho Shin</a></div></div><div id="_co_idm139959257660208"><h3 class="no_margin">Keitaro Matsuo</h3><p><sup>14</sup>Division of Molecular Medicine, Aichi Cancer Center Research Institute, Nagoya, Japan</p><div>Find articles by <a href="/pubmed/?term=Matsuo%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Keitaro Matsuo</a></div></div><div id="_co_idm139959257658224"><h3 class="no_margin">Fumihiko Matsuda</h3><p><sup>15</sup>Center for Genomic Medicine, Graduate School of Medicine, Kyoto University, Kyoto, Japan</p><div>Find articles by <a href="/pubmed/?term=Matsuda%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Fumihiko Matsuda</a></div></div><div id="_co_idm139959257656240"><h3 class="no_margin">Yu-Tang Gao</h3><p><sup>7</sup>Department of Epidemiology, Shanghai Cancer Institute, Shanghai, China</p><div>Find articles by <a href="/pubmed/?term=Gao%20YT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Yu-Tang Gao</a></div></div><div id="_co_idm139959257654256"><h3 class="no_margin">Jae Hwan Oh</h3><p><sup>16</sup>Center for Colorectal Cancer, National Cancer Center, Goyang-si, South Korea</p><div>Find articles by <a href="/pubmed/?term=Oh%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Jae Hwan Oh</a></div></div><div id="_co_idm139959257652272"><h3 class="no_margin">Soriul Kim</h3><p><sup>10</sup>Institute for Health Promotion, Department of Epidemiology and Health Promotion, Graduate School of Public Health, Yonsei University, Seoul, South Korea</p><div>Find articles by <a href="/pubmed/?term=Kim%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Soriul Kim</a></div></div><div id="_co_idm139959257650288"><h3 class="no_margin">Keum Ji Jung</h3><p><sup>10</sup>Institute for Health Promotion, Department of Epidemiology and Health Promotion, Graduate School of Public Health, Yonsei University, Seoul, South Korea</p><div>Find articles by <a href="/pubmed/?term=Jung%20KJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Keum Ji Jung</a></div></div><div id="_co_idm139959254299024"><h3 class="no_margin">Yoon-Ok Ahn</h3><p><sup>9</sup>Department of Preventive Medicine, Seoul National University College of Medicine, Seoul, South Korea</p><div>Find articles by <a href="/pubmed/?term=Ahn%20YO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Yoon-Ok Ahn</a></div></div><div id="_co_idm139959254297040"><h3 class="no_margin">Zefang Ren</h3><p><sup>17</sup>School of Public Health, Sun Yat-sen University, Guangzhou, China</p><div>Find articles by <a href="/pubmed/?term=Ren%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Zefang Ren</a></div></div><div id="_co_idm139959254295056"><h3 class="no_margin">Hong-Lan Li</h3><p><sup>7</sup>Department of Epidemiology, Shanghai Cancer Institute, Shanghai, China</p><div>Find articles by <a href="/pubmed/?term=Li%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Hong-Lan Li</a></div></div><div id="_co_idm139959254293072"><h3 class="no_margin">Jie Wu</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Wu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Jie Wu</a></div></div><div id="_co_idm139959254291088"><h3 class="no_margin">Jiajun Shi</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Shi%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Jiajun Shi</a></div></div><div id="_co_idm139959254289104"><h3 class="no_margin">Wanqing Wen</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Wen%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Wanqing Wen</a></div></div><div id="_co_idm139959254287120"><h3 class="no_margin">Gong Yang</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Yang%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Gong Yang</a></div></div><div id="_co_idm139959254285136"><h3 class="no_margin">Bingshan Li</h3><p><sup>18</sup>Department of Molecular Physiology and Biophysics, Vanderbilt University School of Medicine, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Li%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Bingshan Li</a></div></div><div id="_co_idm139959254283152"><h3 class="no_margin">Bu-Tian Ji</h3><p><sup>19</sup>Division of Cancer Epidemiology &amp; Genetics, National Cancer Institute, Bethesda, Maryland, the United States</p><div>Find articles by <a href="/pubmed/?term=Ji%20BT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Bu-Tian Ji</a></div></div><div id="_co_idm139959254279808"><h3 class="no_margin">Hermann Brenner</h3><p><sup>21</sup>Division of Clinical Epidemiology and Aging Research, German Cancer Research Center (DKFZ), Heidelberg, Germany</p><p><sup>22</sup>Division of Preventive Oncology, National Center for Tumor Diseases (NCT) and German Cancer Research Center (DKFZ), Heidelberg, Germany</p><p><sup>23</sup>German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Brenner%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Hermann Brenner</a></div></div><div id="_co_idm139959254276352"><h3 class="no_margin">Robert E. Schoen</h3><p><sup>24</sup>Department of Medicine and Epidemiology, University of Pittsburgh Medical Center, Pittsburgh, Pennsylvania, the United States</p><div>Find articles by <a href="/pubmed/?term=Schoen%20RE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Robert E. Schoen</a></div></div><div id="_co_idm139959254274368"><h3 class="no_margin">Sébastien Küry</h3><p><sup>25</sup>CHU Nantes, Service de Génétique Médicale, Nantes, République française</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000fc%3Bry%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Sébastien Küry</a></div></div><div id="_co_idm139959251179936"><h3 class="no_margin">Stephen B. Gruber</h3><p><sup>26</sup>USC Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, the United States</p><div>Find articles by <a href="/pubmed/?term=Gruber%20SB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Stephen B. Gruber</a></div></div><div id="_co_idm139959251177952"><h3 class="no_margin">Fredrick R. Schumacher</h3><p><sup>26</sup>USC Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, the United States</p><div>Find articles by <a href="/pubmed/?term=Schumacher%20FR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Fredrick R. Schumacher</a></div></div><div id="_co_idm139959251175968"><h3 class="no_margin">Stephanie L. Stenzel</h3><p><sup>26</sup>USC Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, the United States</p><div>Find articles by <a href="/pubmed/?term=Stenzel%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Stephanie L. Stenzel</a></div></div><div id="_co_idm139959251172624"><h3 class="no_margin">Graham Casey</h3><p><sup>26</sup>USC Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, the United States</p><div>Find articles by <a href="/pubmed/?term=Casey%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Graham Casey</a></div></div><div id="_co_idm139959251170640"><h3 class="no_margin">John L. Hopper</h3><p><sup>27</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, Melbourne School of Population and Global Health, University of Melbourne, Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Hopper%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">John L. Hopper</a></div></div><div id="_co_idm139959251168656"><h3 class="no_margin">Mark A. Jenkins</h3><p><sup>28</sup>Centre for Epidemiology &amp; Biostatistics, Melbourne School of Population and Global Health, University of Melbourne, Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Jenkins%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Mark A. Jenkins</a></div></div><div id="_co_idm139959251166672"><h3 class="no_margin">Hyeong-Rok Kim</h3><p><sup>29</sup>Department of Surgery, Chonnam National University Medical School, Gwangju, South Korea</p><div>Find articles by <a href="/pubmed/?term=Kim%20HR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Hyeong-Rok Kim</a></div></div><div id="_co_idm139959251164688"><h3 class="no_margin">Jin-Young Jeong</h3><p><sup>11</sup>Department of Social and Preventive Medicine, Hallym University College of Medicine, Okcheon-dong, South Korea</p><div>Find articles by <a href="/pubmed/?term=Jeong%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Jin-Young Jeong</a></div></div><div id="_co_idm139959251162704"><h3 class="no_margin">Ji Won Park</h3><p><sup>16</sup>Center for Colorectal Cancer, National Cancer Center, Goyang-si, South Korea</p><p><sup>30</sup>Department of Surgery, Seoul National University Hospital, Seoul, South Korea</p><div>Find articles by <a href="/pubmed/?term=Park%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Ji Won Park</a></div></div><div id="_co_idm139959251159984"><h3 class="no_margin">Kazuo Tajima</h3><p><sup>31</sup>Department of Public Health and Occupational Medicine, Mie University Graduate School of Medicine, Mie, Japan</p><div>Find articles by <a href="/pubmed/?term=Tajima%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Kazuo Tajima</a></div></div><div id="_co_idm139959251158000"><h3 class="no_margin">Sang-Hee Cho</h3><p><sup>32</sup>Department of Hemato-oncology, Chonnam National University Medical School, Gwangju, South Korea</p><div>Find articles by <a href="/pubmed/?term=Cho%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Sang-Hee Cho</a></div></div><div id="_co_idm139959251156016"><h3 class="no_margin">Michiaki Kubo</h3><p><sup>13</sup>RIKEN Center for Integrative Medical Sciences, Kanagawa, Japan</p><div>Find articles by <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Michiaki Kubo</a></div></div><div id="_co_idm139959289273120"><h3 class="no_margin">Xiao-Ou Shu</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Shu%20XO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Xiao-Ou Shu</a></div></div><div id="_co_idm139959289271136"><h3 class="no_margin">Dongxin Lin</h3><p><sup>4</sup>State Key Laboratory of Molecular Oncology, Cancer Institute and Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China</p><div>Find articles by <a href="/pubmed/?term=Lin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Dongxin Lin</a></div></div><div id="_co_idm139959289269152"><h3 class="no_margin">Yi-Xin Zeng</h3><p><sup>3</sup>State Key Laboratory of Oncology in South China, Cancer Center, Sun Yat-sen University, Guangzhou, China</p><div>Find articles by <a href="/pubmed/?term=Zeng%20YX%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Yi-Xin Zeng</a></div></div><div id="_co_idm139959289267168"><h3 class="no_margin">Wei Zheng</h3><p><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</p><div>Find articles by <a href="/pubmed/?term=Zheng%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26965516">Wei Zheng</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139959292793472_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139959292793472_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139959292793472_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Division of Epidemiology, Department of Medicine, Vanderbilt-Ingram Cancer Center, Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, Nashville, Tennessee, the United States</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Laboratory of Genome Technology, Human Genome Center, Institute of Medical Science, The University of Tokyo, Tokyo, Japan</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>State Key Laboratory of Oncology in South China, Cancer Center, Sun Yat-sen University, Guangzhou, China</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>State Key Laboratory of Molecular Oncology, Cancer Institute and Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Preventive Medicine, Chonnam National University Medical School, Gwangju, South Korea</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Jeonnam Regional Cancer Center, Chonnam National University Hwasun Hospital, Hwasun, South Korea</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Epidemiology, Shanghai Cancer Institute, Shanghai, China</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Molecular Epidemiology Branch, National Cancer Center, Goyang-si, South Korea</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Preventive Medicine, Seoul National University College of Medicine, Seoul, South Korea</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Institute for Health Promotion, Department of Epidemiology and Health Promotion, Graduate School of Public Health, Yonsei University, Seoul, South Korea</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Social and Preventive Medicine, Hallym University College of Medicine, Okcheon-dong, South Korea</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>General Department, Tangdu Hospital, Fourth Military Medical University, Xi’an, Shaanxi, China</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>RIKEN Center for Integrative Medical Sciences, Kanagawa, Japan</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Division of Molecular Medicine, Aichi Cancer Center Research Institute, Nagoya, Japan</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Center for Genomic Medicine, Graduate School of Medicine, Kyoto University, Kyoto, Japan</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Center for Colorectal Cancer, National Cancer Center, Goyang-si, South Korea</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>School of Public Health, Sun Yat-sen University, Guangzhou, China</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Department of Molecular Physiology and Biophysics, Vanderbilt University School of Medicine, Tennessee, the United States</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Division of Cancer Epidemiology &amp; Genetics, National Cancer Institute, Bethesda, Maryland, the United States</div><div class="fm-affl" id="A20" lang="en"><sup>21</sup>Division of Clinical Epidemiology and Aging Research, German Cancer Research Center (DKFZ), Heidelberg, Germany</div><div class="fm-affl" id="A21" lang="en"><sup>22</sup>Division of Preventive Oncology, National Center for Tumor Diseases (NCT) and German Cancer Research Center (DKFZ), Heidelberg, Germany</div><div class="fm-affl" id="A22" lang="en"><sup>23</sup>German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), Heidelberg, Germany</div><div class="fm-affl" id="A23" lang="en"><sup>24</sup>Department of Medicine and Epidemiology, University of Pittsburgh Medical Center, Pittsburgh, Pennsylvania, the United States</div><div class="fm-affl" id="A24" lang="en"><sup>25</sup>CHU Nantes, Service de Génétique Médicale, Nantes, République française</div><div class="fm-affl" id="A25" lang="en"><sup>26</sup>USC Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, the United States</div><div class="fm-affl" id="A26" lang="en"><sup>27</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, Melbourne School of Population and Global Health, University of Melbourne, Melbourne, Victoria, Australia</div><div class="fm-affl" id="A27" lang="en"><sup>28</sup>Centre for Epidemiology &amp; Biostatistics, Melbourne School of Population and Global Health, University of Melbourne, Melbourne, Victoria, Australia</div><div class="fm-affl" id="A28" lang="en"><sup>29</sup>Department of Surgery, Chonnam National University Medical School, Gwangju, South Korea</div><div class="fm-affl" id="A29" lang="en"><sup>30</sup>Department of Surgery, Seoul National University Hospital, Seoul, South Korea</div><div class="fm-affl" id="A30" lang="en"><sup>31</sup>Department of Public Health and Occupational Medicine, Mie University Graduate School of Medicine, Mie, Japan</div><div class="fm-affl" id="A31" lang="en"><sup>32</sup>Department of Hemato-oncology, Chonnam National University Medical School, Gwangju, South Korea</div><div id="FN1">Corresponding author contact information: Wei Zheng, M.D., Ph.D., Vanderbilt Epidemiology Center, Vanderbilt University School of Medicine, 2525 West End Avenue, 8<sup>th</sup> Floor, Nashville, TN 37203-1738, Phone: (615) 936-0682; Fax: (615) 936-8241, <a href="mailto:dev@null" data-email="ude.tlibrednav@gnehz.iew" class="oemail">ude.tlibrednav@gnehz.iew</a></div><div id="FN2"><sup>20</sup>A complete list of members is provided in the Acknowledgements.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139959292793472_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26965516" target="pmc_ext" ref="reftype=publisher&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Gastroenterology</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4909543/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC4909543/bin/NIHMS766698-supplement-1.docx" data-ga-action="click_feat_suppl">NIHMS766698-supplement-1.docx</a><span style="color:gray"> (238K)</span></div><div class="small guid">GUID:&nbsp;FA538D45-7929-47D0-A3AC-D7AE2CA47B04</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC4909543/bin/NIHMS766698-supplement-2.tiff" data-ga-action="click_feat_suppl">NIHMS766698-supplement-2.tiff</a><span style="color:gray"> (5.9M)</span></div><div class="small guid">GUID:&nbsp;D111E015-2222-4950-A88D-A5F0195D94E1</div></div></dd></dl></div><div id="idm139959251479824" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139959251479824title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background &amp; Aims</h3><p id="P1" class="p p-first-last">Known Genetic factors explain only a small fraction of genetic variation in colorectal cancer (CRC). We conducted a genome-wide association study (GWAS) to identify risk loci for CRC.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">This discovery stage included 8027 cases and 22577 controls of East-Asian ancestry. Promising variants were evaluated in studies including as many as 11044 cases and 12047 controls. Tumor-adjacent normal tissues from 188 patients were analyzed to evaluate correlations of risk variants with expression levels of nearby genes. Potential functionality of risk variants were evaluated using public genomic and epigenomic databases.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">We identified 4 loci associated with CRC risk; <em>P</em> values for the most significant variant in each locus ranged from 3.92×10<sup>−8</sup> to 1.24×10<sup>−12</sup>: 6p21.1 (rs4711689), 8q23.3 (rs2450115, rs6469656), 10q24.3 (rs4919687), and 12p13.3 (rs11064437). We also identified 2 risk variants at loci previously associated with CRC: 10q25.2 (rs10506868) and 20q13.3 (rs6061231). These risk variants, conferring an approximate 10%–18% increase in risk per allele, are located either inside or near protein-coding genes that include <em>TFEB</em> (lysosome biogenesis and autophagy), <em>EIF3H</em> (initiation of translation), <em>CYP17A1</em> (steroidogenesis), <em>SPSB2</em> (proteasome degradation), and RPS21 (ribosome biogenesis). Gene expression analyses showed a significant association (<em>P</em> &lt;.05) for rs4711689 with <em>TFEB</em>, rs6469656 with <em>EIF3H</em>, rs11064437 with <em>SPSB2</em>, and rs6061231 with <em>RPS21</em>.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions</h3><p id="P4" class="p p-first-last">We identified susceptibility loci and genes associated with CRC risk, linking CRC predisposition to steroid hormone, protein synthesis and degradation, and autophagy pathways and providing added insight into the mechanism of CRC pathogenesis.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Epidemiology, single nucleotide polymorphisms, colon cancer, eQTL</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">INTRODUCTION</h2><p id="P5" class="p p-first">Colorectal cancer (CRC) is one of the most frequently diagnosed malignancies in the world, including most Asian countries.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517205">1</a></sup> Genetic factors play an important role in the etiology of both familial and sporadic CRC.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517212">2</a></sup> Through family-based studies, multiple CRC susceptibility genes have been identified, including <em>APC, MLH1, MSH2, MSH6, PMS2, BMPR1A, SMAD4</em>, <em>POLE</em>, <em>POLD1</em>, <em>GREM1</em>, <em>NTHL1</em>, <em>LKB1/STK11</em> and <em>MUTYH</em>.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517181">3</a></sup> Deleterious germline mutations in these genes, however, account for less than 5% of CRC cases in the population. It is believed that many common, low-penetrance genetic risk variants exist for CRC and, collectively, these variants explain a substantial proportion of genetic variation for CRC. Genome-wide association studies (GWAS) have become a powerful tool to uncover genetic susceptibility factors for complex diseases. To date, more than 40 CRC GWAS risk loci have been identified,<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_565504392">4</a>–<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517207">21</a></sup> which have expanded our understanding of the etiology of CRC. However, these common risk variants, along with known CRC susceptibility genes, explain only a small fraction of genetic variation for CRC.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_565504391">22</a></sup></p><p id="P6" class="p p-last">It has been demonstrated that GWAS conducted in non-European populations are valuable in identifying genetic variants for complex traits.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517237">8</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517187">11</a></sup> Because of different genetic architectures and environmental exposures, studies in non-European populations can often discover important genetic risk variants that are otherwise difficult to identify.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517191">8</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517227">11</a></sup> Furthermore, these studies can uncover risk variants specific to the study population. To search for new CRC susceptibility loci, we conducted a large, multi-staged genetic study including 19,071 cases and 34,624 controls of East Asian ancestry. To investigate the generalizability of our findings to other populations, we evaluated the newly-identified risk variants in16,984 cases and 18,262 controls of European ancestry.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Materials and Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Study population</h3><p id="P7" class="p p-first">This study was conducted as part of the Asia Colorectal Cancer Consortium (ACCC), including a total of 19,071 cases and 34,624 controls of East Asian ancestry from 14 studies conducted in China, South Korea and Japan (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). The details of these studies are described in the <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary materials</a>. Compared to our previous GWAS studies,<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517218">8</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517204">11</a></sup> we expanded the discovery effort by including the Japan Biobank (BBJ) study (2,814 cases/11,358 controls) and three additional studies (3,519 cases/5,989 controls) genotyped with a customized exome array, the details of which are described in the genotyping section. Specifically, the discovery stage consisted of 8,027 cases and 22,577 controls, including 4,888 cases and 10,243 controls from six studies conducted among Chinese in Shanghai and Guangzhou, 2,814 cases and 11,358 controls from a study conducted among Japanese in Japan and 325 cases and 976 controls from a study conducted among Koreans in South Korea. Replication stage 1 consisted of 6,532 cases and 8,140 controls, including 677 cases and 1,114 controls conducted among Chinese in Guangzhou, 236 cases and 472 controls from a study conducted among Japanese in Japan and 5,619 cases and 6,554 controls from three studies conducted among Koreans in South Korea. Replication stage 2 included 4,512 cases and 3,907 controls from a study conducted among Chinese in Beijing. All study protocols were approved by the relevant institutional review boards and all study participants provided written informed consent.</p><p id="P8" class="p p-last">The studies in European descendants comprised 16,984 cases and 18,262 controls recruited in North America, Europe and Australia, which are included in three consortia; the Genetics and Epidemiology of Colorectal Cancer Consortium (GECCO), the Colorectal Transdisciplinary (CORECT) Study, and the Colon Cancer Family Registry (CCFR). The details of these studies have been described previously.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517179">10</a></sup></p></div><div id="S8" class="sec"><h3 id="S8title">Genotyping and imputation in the discovery stage</h3><p id="P9" class="p p-first-last">Genotyping and quality control (QC) for studies in the discovery stage have been described previously.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517229">8</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517219">11</a></sup> Briefly, genotyping for five of the eight studies included in the discovery stage were completed using Affymetrix or Illumina SNP arrays (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). In previous studies conducted in the ACCC, imputation was conducted using the CHB and JPT HapMap 2 panel as the reference.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517210">8</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517189">11</a></sup> To increase the coverage of genotypes in this study, we imputed genotypes for each of these five studies with data from the 1000 Genomes Project Phase I Release (Version 3) as the reference. Only SNPs with minor allele frequency (MAF) &gt; 5% and a high imputation quality (R<sup>2</sup> &gt; 0.70) were included. After QC filtering, approximately 4 million genotyped or imputed SNPs on 22 autosomes remained for analysis. Genotyping for the remaining three studies included in the discovery stage was completed using Illumina Infinium assays with 59,317 variants as the customer add-on contents onto the Illumina HumanExome Beadchip. Of the 302,218 variants included in this array, 44,428 have an MAF of 5% or higher. We did not impute genotypes in these three studies given the low-density coverage of the genome by these common variants. After QC filtering, 40,647 SNPs with an MAF of 5% or higher remained for the analysis. Little evidence of substantial population stratification was found in any of the eight studies included in this stage (the inflation factor for each study was &lt; 1.05).</p></div><div id="S9" class="sec"><h3 id="S9title">SNP selection and genotyping in the replication stages</h3><p id="P10" class="p p-first">No apparent heterogeneity across the eight studies included in the discovery stage was noted, and thus a fixed-effects meta-analysis was performed to obtain the overall summary statistics for the association of genetic variants with CRC risk. We selected SNPs for further evaluation according to the following criteria: (i) <em>P</em> ≤ 0.0001 in the discovery stage; (ii) consistent direction of association across all eight studies included in the discovery stage (<em>P</em> for heterogeneity &gt; 0.05), (iii) in no or moderate linkage disequilibrium (LD) (r<sup>2</sup> &lt; 0.30 in East Asians) with any known CRC risk variant or SNPs that had been evaluated in our previous replication stages.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517235">8</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517196">11</a></sup> A total of 38 independent association signals were identified. For each signal, we selected the SNP with the most significant <em>P</em>-value for replication. Of these 38 SNPs, 36 were successfully genotyped and evaluated in an independent sample of 6,532 cases and 8,140 controls (replication stage 1). SNPs that showed evidence of association (one-sided <em>P</em> &lt; 0.05) in replication stage 1 and in the same direction of the association as seen in the discovery stage were further evaluated in another independent sample of 4,512 cases and 3,907 controls (replication stage 2) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>).</p><p id="P11">Using the iPLEX Sequenom MassARRAY platform, genotyping for replication stage 1 and 2 studies was conducted at the Vanderbilt Molecular Epidemiology Laboratory and the Fudan-VARI Center for Genetic Epidemiology Core lab, respectively. Four negative controls (water) and eight positive quality controls (HapMap or duplicate samples) were included in each 384-well plate. We filtered out SNPs with (i) genotype call rate &lt; 95%, (ii) genotyping concordance rate &lt; 95% in positive control samples, (iii) an unclear genotype call or (iv) <em>P</em> for Hardy-Weinberg equilibrium &lt; 10<sup>−5</sup> in controls. We calculated the mean concordance rate using data from positive quality control samples. The mean concordant rate was 99% with a median value of 100% in each of the five participating studies included in replication stage 1 and was 98% with a median value of 98% in the replication stage 2 study.</p><p id="P12" class="p p-last">Genotyping of samples in GECCO, CORECT, and CCFR was conducted using Illumina and Affymetrix Arrays. The details of genotyping, QC and imputation have been reported previously.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_565504389">5</a>, <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517199">10</a></sup></p></div><div id="S10" class="sec"><h3 id="S10title">Gene expression and protein abundance evaluation of candidate genes</h3><p id="P13" class="p p-first-last">The gene closest to each of the risk variants identified in this GWAS was selected for evaluation of its expression level in correlation with the risk variant in adjacent normal tissues from 188 CRC patients of East-Asian descent. The details of quantitative real-time polymerase chain reaction analysis and quantitative western blotting are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>.</p></div><div id="S11" class="sec"><h3 id="S11title">Statistical analysis</h3><p id="P14" class="p p-first">Associations between SNPs and CRC risk were evaluated on the basis of the log-additive model using mach2dat,<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517216">23</a></sup> PLINK version 1.0.7,<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517228">24</a></sup> R version 3.0.2 (See URLs) and SAS version 9.4. Per-allele odds ratios (ORs) and 95% confidence intervals (CIs) were derived from logistic regression models, adjusting for age, sex and the first ten principal components when appropriate. Association analysis was conducted separately for each study. Because there was no apparent heterogeneity across participating studies, fixed-effects meta-analyses were used to obtain summary results for each stage and for all stages combined with the inverse-variance method using METAL.<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517215">25</a></sup> SNPs associated with CRC risk at <em>P</em> &lt; 5×10<sup>−8</sup> in the combined analysis were considered genome-wide significant. Stratified analyses were performed to assess whether the associations differ by tumor sites (colon or rectum), ethnicity (Chinese, Korean or Japanese) and sex (men or women). We estimated heterogeneity across studies and subgroups with Cochran’s <em>Q</em> test. We estimated haplotype frequencies with the expectation maximum algorithms in the Haplo. stat package (see URLs) in R (version 3.0.2) and conducted haplotype analyses when two or more SNPs were identified at the same locus.</p><p id="P15" class="p p-last">Details of the eQTL analysis using data from East-Asian patients and the analysis of the SPSB2 isoform using data from The Cancer Genome Atlas project are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">Bioinformatics analyses</h3><p id="P16" class="p p-first-last">To identify putative functional variants for newly identified loci, all variants in LD (<em>r<sup>2</sup></em> &gt; 0.20) with the risk variant in each locus were identified using data from the 1000 Genomes Project (phase1 release v3.0). Non-coding variants were mapped to the 15 chromatin states across multiple normal colorectal mucosal tissues derived by the Roadmap Epigenomics (REMC) project <sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517223">26</a></sup> and to the epigenomic maps of histone markers (H3K4Me1, H3K4Me3, and H3K27Ac) across CRC cell lines derived by the Encyclopedia of DNA elements (ENCODE) project.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517230">27</a></sup> We focused on variants in promoters or enhancers. We identified the corresponding genes as the target genes for variants in promoters. We identified the interacting genes for variants in enhancers using data of enhancer- transcription start site (TSS) association from the Functional ANnoTation Of the Mammalian Genome 5 (FANTOM5) project, which evaluated correlations of the transcriptional activity of the enhancer and putative target genes. <sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_565504388">28</a></sup> Coding variants were evaluated using the SIFT<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517220">29</a></sup> and PolyPhen-2 algorithms.<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517231">30</a></sup> We proposed the most likely candidate gene at each locus according to the following criteria: 1) functional evidence; 2) distance to the risk variant; 3) results from eQTL analysis; 4) biologic functions and potential roles in cancer according to previous literature.</p></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Results</h2><p id="P17" class="p p-first">Combined analyses of data from the discovery and replication stages revealed seven SNPs associated with CRC risk at the genome-wide significance level (<em>P</em> &lt; 5×10<sup>−8</sup>), including rs4711689 at 6p21.1 (<em>TFEB</em>), rs4919687 at 10q24.3 (<em>CYP17A1</em>), rs11064437 at 12p13.3 (<em>SPSB2</em>), rs2450115 and rs6469656 at 8q23.3 (<em>EIF3H</em>), rs10506868 at 10q25.2 (<em>VTI1A</em>), and rs6061231 at 20q13.3 (<em>RPS21</em>) (<a href="/pmc/articles/PMC4909543/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). SNP rs10506868 at 10q25.2 was correlated with a risk variant (rs12241008, r<sup>2</sup> = 0.60, D′ &lt;1 in East Asians) identified in a recent GWAS study,<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517185">7</a></sup> and thus this locus was not further evaluated in the subsequent analysis of this study. Interestingly, rs10506868 is located in intron 6 of the <em>VTI1A</em> gene, a fusion partner of a CRC susceptibility candidate gene, <em>TCF7L2</em>, identified in our previous GWAS study.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517195">8</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p34">Summary results for newly identified genetic risk variants associated with colorectal cancer risk at P &lt; 5.0 ×10<sup>−8</sup> East-Asian descendants</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139959257133776" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="4" colspan="1" valign="middle" align="left">SNP</th><th rowspan="4" colspan="1" valign="middle" align="left">Locus</th><th rowspan="4" colspan="1" valign="middle" align="center">Gene<sup>a</sup></th><th rowspan="4" colspan="1" valign="middle" align="center">Position<sup>b</sup></th><th rowspan="4" colspan="1" valign="middle" align="center">Alleles<sup>c</sup></th><th rowspan="4" colspan="1" valign="middle" align="center">EAF<sup>d</sup></th><th colspan="2" rowspan="1" valign="top" align="center">Discovery<sup>e</sup></th><th colspan="2" rowspan="1" valign="top" align="center">Replication<sup>e</sup></th><th colspan="2" rowspan="1" valign="top" align="center">Combined<sup>e</sup></th><th colspan="2" rowspan="3" valign="top" align="center">Heterogeneity</th></tr><tr><th colspan="2" rowspan="1" valign="top" align="center">8,027 cases /22,577 controls</th><th colspan="2" rowspan="1" valign="top" align="center">11,044 cases/12,047controls</th><th colspan="2" rowspan="1" valign="top" align="center">19,071 cases /34,624 controls</th></tr><tr><th colspan="6" rowspan="1" valign="bottom" align="center">
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">OR (95% CI)<sup>e</sup></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em><sup>e</sup></th><th rowspan="1" colspan="1" valign="top" align="left">OR (95% CI)<sup>e</sup></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em><sup>e</sup></th><th rowspan="1" colspan="1" valign="top" align="left">OR (95% CI)<sup>e</sup></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em><sup>e</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>I<sup>2</sup></em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em><sup>f</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4711689</td><td rowspan="1" colspan="1" valign="top" align="left">6p21.1</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TFEB</em></td><td rowspan="1" colspan="1" valign="top" align="center">41692812</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>A</strong>/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.85</td><td rowspan="1" colspan="1" valign="top" align="left">1.12 (1.06–1.18)</td><td rowspan="1" colspan="1" valign="top" align="left">3.41×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.10 (1.05–1.16)</td><td rowspan="1" colspan="1" valign="top" align="left">2.89×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.11 (1.07–1.15)</td><td rowspan="1" colspan="1" valign="top" align="left">3.92×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">31</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2450115</td><td rowspan="1" colspan="1" valign="top" align="left">8q23.3</td><td rowspan="1" colspan="1" valign="top" align="center"><em>EIF3H</em></td><td rowspan="1" colspan="1" valign="top" align="center">117624093</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>T/</strong>C</td><td rowspan="1" colspan="1" valign="top" align="center">0.58</td><td rowspan="1" colspan="1" valign="top" align="left">1.11 (1.06–1.17)</td><td rowspan="1" colspan="1" valign="top" align="left">9.19×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.11 (1.07–1.16)</td><td rowspan="1" colspan="1" valign="top" align="left">2.94×10–<sup>8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.12 (1.09–1.15)</td><td rowspan="1" colspan="1" valign="top" align="left">1.24×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.51</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6469656</td><td rowspan="1" colspan="1" valign="top" align="left">8q23.3</td><td rowspan="1" colspan="1" valign="top" align="center"><em>EIF3H</em></td><td rowspan="1" colspan="1" valign="top" align="center">117647788</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>A/</strong>G</td><td rowspan="1" colspan="1" valign="top" align="center">0.67</td><td rowspan="1" colspan="1" valign="top" align="left">1.12 (1.07–1.16)</td><td rowspan="1" colspan="1" valign="top" align="left">2.59×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.10 (1.06–1.15)</td><td rowspan="1" colspan="1" valign="top" align="left">1.55×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.11 (1.08–1.14)</td><td rowspan="1" colspan="1" valign="top" align="left">2.03×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">38</td><td rowspan="1" colspan="1" valign="top" align="left">0.07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4919687</td><td rowspan="1" colspan="1" valign="top" align="left">10q24.3</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CYP17A1</em></td><td rowspan="1" colspan="1" valign="top" align="center">104595248</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>G</strong>/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.80</td><td rowspan="1" colspan="1" valign="top" align="left">1.16 (1.09–1.24)</td><td rowspan="1" colspan="1" valign="top" align="left">7.52×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.13 (1.08–1.19)</td><td rowspan="1" colspan="1" valign="top" align="left">1.91×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.14 (1.10–1.19)</td><td rowspan="1" colspan="1" valign="top" align="left">7.82×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.48</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10506868</td><td rowspan="1" colspan="1" valign="top" align="left">10q25.2</td><td rowspan="1" colspan="1" valign="top" align="center"><em>VTI1A</em></td><td rowspan="1" colspan="1" valign="top" align="center">114319380</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>T/C</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="left">1.09 (1.04–1.14)</td><td rowspan="1" colspan="1" valign="top" align="left">1.00×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.10 (1.05–1.14)</td><td rowspan="1" colspan="1" valign="top" align="left">9.25×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.10 (1.06–1.13)</td><td rowspan="1" colspan="1" valign="top" align="left">3.98×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.84</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11064437</td><td rowspan="1" colspan="1" valign="top" align="left">12p13.3</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SPSB2</em></td><td rowspan="1" colspan="1" valign="top" align="center">6982162</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>C</strong>/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.75</td><td rowspan="1" colspan="1" valign="top" align="left">1.14 (1.09–1.20)</td><td rowspan="1" colspan="1" valign="top" align="left">2.80×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.10 (1.05–1.15)</td><td rowspan="1" colspan="1" valign="top" align="left">2.11×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.12 (1.08–1.16)</td><td rowspan="1" colspan="1" valign="top" align="left">4.48×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">52.5</td><td rowspan="1" colspan="1" valign="top" align="left">0.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6061231</td><td rowspan="1" colspan="1" valign="top" align="left">20q13.3</td><td rowspan="1" colspan="1" valign="top" align="center"><em>RPS21</em></td><td rowspan="1" colspan="1" valign="top" align="center">60956917</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>C</strong>/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="left">1.19 (1.12–1.27)</td><td rowspan="1" colspan="1" valign="top" align="left">8.56×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.18 (1.07–1.28)</td><td rowspan="1" colspan="1" valign="top" align="left">2.20×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.18 (1.13–1.25)</td><td rowspan="1" colspan="1" valign="top" align="left">8.04×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.47</td></tr></tbody></table></div><div id="largeobj_idm139959257133776" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4909543/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p35" class="p p-first-last">EAF, effect allele frequency; OR, odds ratio; CI, confidence interval.</p></div><div id="TFN2"><sup>a</sup>The closest gene.</div><div id="TFN3"><sup>b</sup>The chromosome position (bp) is based on the National Center for Biotechnology Information (NCBI) database, build 37.</div><div id="TFN4"><sup>c</sup>Effect/reference alleles are based on forward allele coding in NCBI, build 37. OR and 95% CI were estimated based on the effect allele (bold).</div><div id="TFN5"><sup>d</sup>EAF in controls from all studies combined.</div><div id="TFN6"><sup>e</sup>Summary OR (95% CI) and <em>P</em> were obtained from a fixed-effects meta-analysis.</div><div id="TFN7"><sup>f</sup><em>P</em> for heterogeneity across all studies was calculated using Cochran’s <em>Q</em> test.</div></div></div><p id="P18">Associations with CRC risk in each of the six risk variants were, in general, consistent across all studies with little evidence of heterogeneity (<a href="/pmc/articles/PMC4909543/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). Stratified analyses by sex and tumor site showed little evidence of heterogeneity (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). Except for rs11064437 at 12p13.3 (<em>P</em> = 0.0003), no apparent heterogeneity was found for the association of variants with CRC risk by ethnicity. The heterogeneity with rs11064437 was primarily driven by its null association in studies in South Korea (OR = 1.01, <em>P</em> = 0.73).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4909543/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139959257149008" class="figure"><a href="/pmc/articles/PMC4909543/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4909543_nihms766698f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms766698f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4909543/bin/nihms766698f1.jpg"></a></div><div id="largeobj_idm139959257149008" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4909543/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4909543/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Forest plots for newly identified CRC risk variants (a) rs4711689, (b) rs2450115, (c) rs6469656, (d) rs4919687, (e) rs11064437, (f) rs6061231</strong><p id="__p31">Per-allele OR estimates are presented as boxes with the area proportional to the inverse variance of the estimates. Horizontal lines represent the coverage of 95% confidence intervals (CIs).</p></div></div></div><p id="P19">We next evaluated the association of the six newly identified CRC risk variants in populations of European ancestry using data from 16,984 cases and 18,262 controls. With the exception of rs4711689 and rs11064437, all other SNPs were associated with CRC risk at <em>P</em> &lt; 0.05 and in the same direction as observed in East Asian studies (<a href="/pmc/articles/PMC4909543/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). There was evidence of heterogeneity in the association of CRC risk with rs4919687, rs4711689 and rs6061231 between populations of European ancestry and East Asian ancestry (<em>P</em> &lt; 0.01). The effect allele frequency (EAF) for all six risk variants differs substantially between European descendants and East Asians (<a href="/pmc/articles/PMC4909543/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>–<a href="/pmc/articles/PMC4909543/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>2</span></a>). For example, for rs11064437, the EAF is 0.75 for East Asians while it is 0.996 for European descendants, which may explain the non-significant finding for this SNP in populations of European ancestry.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p42">Associations of newly identified risk variants with CRC risk in populations of European ancestry (16,984 cases and 18,262 controls)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139959251342608" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Locus</th><th rowspan="1" colspan="1" valign="top" align="left">SNP (alleles)<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">Gene<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">Position<sup>c</sup></th><th rowspan="1" colspan="1" valign="top" align="left">EAF<sup>d</sup></th><th rowspan="1" colspan="1" valign="top" align="left">OR (95% CI)<sup>e</sup></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em><sup>e</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em>. het<sup>f</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">6p21.1</td><td rowspan="1" colspan="1" valign="top" align="left">rs4711689 (<strong>A</strong>/G)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TFEB</em></td><td rowspan="1" colspan="1" valign="top" align="left">41692812</td><td rowspan="1" colspan="1" valign="top" align="left">0.54</td><td rowspan="1" colspan="1" valign="top" align="left">1.01 (0.97–1.04)</td><td rowspan="1" colspan="1" valign="top" align="left">0.69</td><td rowspan="1" colspan="1" valign="top" align="center">8.94×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8q23.3</td><td rowspan="1" colspan="1" valign="top" align="left">rs2450115 (<strong>T</strong>/C)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>EIF3H</em></td><td rowspan="1" colspan="1" valign="top" align="left">117624093</td><td rowspan="1" colspan="1" valign="top" align="left">0.81</td><td rowspan="1" colspan="1" valign="top" align="left">1.08 (1.03–1.12)</td><td rowspan="1" colspan="1" valign="top" align="left">0.0003</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8q23.3</td><td rowspan="1" colspan="1" valign="top" align="left">rs6469656 (<strong>A</strong>/G)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>EIF3H</em></td><td rowspan="1" colspan="1" valign="top" align="left">117647788</td><td rowspan="1" colspan="1" valign="top" align="left">0.89</td><td rowspan="1" colspan="1" valign="top" align="left">1.06 (1.01–1.11)</td><td rowspan="1" colspan="1" valign="top" align="left">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10q24.3</td><td rowspan="1" colspan="1" valign="top" align="left">rs4919687 (<strong>G</strong>/A)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CYP17A1</em></td><td rowspan="1" colspan="1" valign="top" align="left">104595248</td><td rowspan="1" colspan="1" valign="top" align="left">0.70</td><td rowspan="1" colspan="1" valign="top" align="left">1.05 (1.02–1.09)</td><td rowspan="1" colspan="1" valign="top" align="left">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">12p13.3</td><td rowspan="1" colspan="1" valign="top" align="left">rs11064437 (<strong>C</strong>/T)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SPSB2</em></td><td rowspan="1" colspan="1" valign="top" align="left">6982162</td><td rowspan="1" colspan="1" valign="top" align="left">0.996</td><td rowspan="1" colspan="1" valign="top" align="left">1.17 (0.85–1.62)</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.77</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">20q13.3</td><td rowspan="1" colspan="1" valign="top" align="left">rs6061231(<strong>C</strong>/A)</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RPS21</em></td><td rowspan="1" colspan="1" valign="top" align="left">60956917</td><td rowspan="1" colspan="1" valign="top" align="left">0.71</td><td rowspan="1" colspan="1" valign="top" align="left">1.07 (1.04–1.11)</td><td rowspan="1" colspan="1" valign="top" align="left">8.90×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td></tr></tbody></table></div><div id="largeobj_idm139959251342608" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4909543/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN8"><p id="__p43" class="p p-first-last">EAF, effect allele frequency; OR, odds ratio; CI, confidence interval.</p></div><div id="TFN9"><sup>a</sup>Effect/reference alleles are based on forward allele coding in NCBI, build 37.</div><div id="TFN10"><sup>b</sup>the closest gene.</div><div id="TFN11"><sup>c</sup>Chromosome position (bp) is based on NCBI Build 37.</div><div id="TFN12"><sup>d</sup>EAF from 1000 Genomes Project October 2013 release (CEU+FIN+GBR+IBS+TSI).</div><div id="TFN13"><sup>e</sup>The summary OR (95% CI) and <em>P</em> value were obtained from a fixed-effects meta-analysis.</div><div id="TFN14"><sup>f</sup><em>P</em> for heterogeneity between East Asian and European populations was calculated using Cochran’s <em>Q</em> test.</div></div></div><p id="P20">SNPs rs2450115 and rs6469656 lie approximately 24kb apart at 8q23.3, and were correlated only moderately (r<sup>2</sup>= 0.20, D′ &lt;1 in East Asians). The association with either of these SNPs remained statistically significant (<em>P</em> = 9.60×10<sup>−6</sup> for rs2450115 and <em>P</em> = 8.30×10<sup>−4</sup> for rs6469656) in the conditional analysis including both SNPs in the same model (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>), suggesting that there may be two independent association signals. Furthermore, the haplotype (TA) comprising the risk allele of both SNPs was significantly associated with a 1.20-fold increased risk of CRC (<em>P</em> = 1.34 ×10<sup>−15</sup>) compared to the haplotype (CG) comprising the low-risk allele of both SNPs. These two SNPs lie close to rs16892766 (6.6kb centromeric for rs2450115 and 17 kb telomeric for rs6469656), a CRC-associated variant identified in a previous GWAS in European descendants.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517232">17</a></sup> However, rs16892766 and any of its highly correlated SNPs are monomorphic in East Asian populations. Thus rs16892766 was not in LD with either of the two risk variants identified in our study. Even in European-ancestry populations, these two SNPs also are poor surrogates for rs16892766 (r<sup>2</sup> = 0.02, D′=1 for rs2450115 and r<sup>2</sup> = 0.01, D′&lt;1 for rs6469656). Interestingly, neither rs2450115 nor rs6469656 was reported to be associated with CRC risk in two previous fine-mapping studies of 8q23.3 for CRC risk in European descendants.<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517192">31</a>, <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_565504390">32</a></sup> In our study with a larger sample size, we showed that rs2450115 and rs6469656 were associated with CRC risk in European descendants (<em>P</em> = 0.0003 and 0.02, respectively, <a href="/pmc/articles/PMC4909543/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). These two SNPs are in LD with each other in European descendants (r<sup>2</sup> = 0.40, D′=1), and only SNP rs2450115 remained statistically significant in the conditional analysis including both SNPs in the model (<em>P</em> = 0.007 for rs2450115 and <em>P</em> = 0.78 for rs6469656).</p><p id="P21">We analyzed the expression levels of the nearest genes in adjacent normal tissues obtained from 188 CRC patients of East Asian ancestry and performed expression quantitative trait locus (eQTL) analysis. <em>CYP17A1</em> expression is largely confined to adrenal and gonadal tissues in adults, and thus was not detected in our samples. The remaining four genes evaluated were all highly expressed in normal colorectal tissues. Of the two variants evaluated at 8q23.3, only rs6469656 was significantly correlated with <em>EIF3H</em> expression at <em>P</em> &lt; 0.05 (<a href="/pmc/articles/PMC4909543/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). For the remaining three loci, each of the risk variants was statistically significantly correlated with its nearest gene at <em>P</em> &lt; 0.05 (<a href="/pmc/articles/PMC4909543/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>). At 8q23.3, we also evaluated the expression of the <em>UTP23</em> gene, a candidate gene suggested by a previous fine-mapping study at this locus.<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517184">31</a></sup> However, we did not find any statistically significant correlation of risk variants at 8q23.3 with <em>UTP23</em> expression (<em>P</em> = 0.90 and 0.74, respectively, for rs2450115 and rs6469656). To evaluate the correlation of mRNA and protein levels for the four genes whose expression levels were significantly associated with the risk variants, we analyzed the protein level for each of these genes in adjacent normal tissues obtained from 16 patients (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 1</a>). The null hypothesis was that there would be no correlation between mRNA and protein levels. The correlation (r) between mRNA expression and protein level was −0.76, 0.64, −0.42 and 0.58 for TFEB, EIF3H, SPSB2 and RPS21, respectively (<em>P</em> = 0.05, 0.12, 0.07 and 0.24, respectively).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4909543/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139959257143504" class="figure"><a href="/pmc/articles/PMC4909543/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4909543_nihms766698f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms766698f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4909543/bin/nihms766698f2.jpg"></a></div><div id="largeobj_idm139959257143504" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4909543/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4909543/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Expression quantitative trait locus (eQTL) analyses in samples of normal colorectal mucosa adjacent to the tumor obtained from 188 CRC patients of East Asian ancestry</strong><p id="__p32">a. <em>TFEB</em> relative expression and genotypes of rs4711689; b. <em>EIF3H</em> relative expression and genotypes of rs6469656; c. <em>SPSB2</em> relative expression and genotypes of rs11064437; d. <em>RPS21</em> relative expression and genotypes of rs6061231.</p></div></div></div><p id="P22">To evaluate whether the expressions of these genes were deregulated in tumor tissues, we evaluated the differences between tumor and normal tissues in the expression level of the <em>EIF3H</em>, <em>UTP23</em>, <em>SPSB2</em>, <em>TFEB</em> and <em>RPS21</em> genes using data from the 188 patients mentioned above. With the exception of <em>TFEB</em>, expression levels of the other four genes were significantly higher in tumors (<em>P</em> &lt; 0.005, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>). For <em>TFEB,</em> the expression level in tumors was significantly lower than normal tissues (<em>P</em> = 9.26 ×10<sup>−11</sup>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>).</p><p id="P23">We used publically available functional genomic data to further annotate the risk loci identified in our study. We observed an overlap of promoter and enhancer sequences at each locus and identified multiple potential target genes. Combining results from eQTL analyses, functional annotation, and literature review, as provided in detail in the discussion section, we proposed the likely candidate gene: <em>TFEB</em> at 6p21.1, <em>EIH3H</em> at 8q23.3, <em>CYP17A1</em> at 10q24.3, <em>SPSB2</em> at 12p13.3 and <em>RPS21</em> at 20q13.3. The results of <em>in silico</em> functional annotation of the newly identified loci and literature review on potential roles of candidate genes are summarized in <a href="/pmc/articles/PMC4909543/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>. Regional association plots and functional genomic landscapes of these loci are presented in <a href="/pmc/articles/PMC4909543/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 2</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4909543/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139959257136880" class="figure"><a href="/pmc/articles/PMC4909543/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4909543_nihms766698f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms766698f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4909543/bin/nihms766698f3.jpg"></a></div><div id="largeobj_idm139959257136880" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4909543/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4909543/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Regional association plots of the risk loci (a) rs4711689 at 6p21.1<sup>i</sup>;(b) rs2450115 at 8q23.3;(c) rs6469656 at 8q23.3; (d) rs4919687 at 10q24.3; (e) rs11064437 at 12p13.3; (f) rs6061231 at 20q13.3</strong><p id="__p33">In each plot, the log<sub>10</sub> (<em>P</em>-values) (y axis) for the association of SNPs with CRC risk are shown according to their chromosomal positions (x axis) in NCBI Build 37. Blue lines represent the estimated recombination rates from the 1000 Genomes Project (Phase 3). Arrows indicate genomic locations of genes within the 1Mb regions centered on the newly-identified risk variants. The color of the SNP represents its LD (r<sup>2</sup>) with the index SNP at each locus. Only results from the GWAS studies (4,508 cases /16,588 controls in total) are shown. The plots were generated using the online tool LocusZoom.<sup><a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517224">59</a></sup></p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p50">Summary of risk variants, potential regulatory roles and the likely candidate gene in each new CRC risk locus identified in this study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139959252579680" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">SNP (Locus)</th><th rowspan="1" colspan="1" valign="middle" align="left">Genomic annotation</th><th rowspan="1" colspan="1" valign="middle" align="left">No. of genes in 1-M window</th><th rowspan="1" colspan="1" valign="middle" align="left">No. of correlated SNPs (r<sup>2</sup> &gt; 0.20)</th><th rowspan="1" colspan="1" valign="middle" align="left">Potential regulatory roles</th><th rowspan="1" colspan="1" valign="middle" align="left">The likely candidate gene</th><th rowspan="1" colspan="1" valign="middle" align="left">Expression difference in tumor vs. adjacent normal tissues<sup>*</sup></th><th rowspan="1" colspan="1" valign="middle" align="left">Biological functions</th><th rowspan="1" colspan="1" valign="middle" align="left">Known roles in cancer</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs4711689 (6p21.1)</td><td rowspan="1" colspan="1" valign="middle" align="left">Intron 6 of <em>TFEB</em></td><td rowspan="1" colspan="1" valign="middle" align="left">43</td><td rowspan="1" colspan="1" valign="middle" align="left">17</td><td rowspan="1" colspan="1" valign="middle" align="left">Promoter, enhancer</td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>TFEB</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left">Down-regulated</td><td rowspan="1" colspan="1" valign="middle" align="left">Lysosomal biogenesis and autophagy</td><td rowspan="1" colspan="1" valign="middle" align="left">Oncogenic transcriptional factor for some cancers</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2450115 (8q23.3)</td><td rowspan="1" colspan="1" valign="middle" align="left">3′ of <em>EIF3H</em></td><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="left">65</td><td rowspan="1" colspan="1" valign="middle" align="left">Unknown</td><td rowspan="1" colspan="1" valign="middle" align="left">-</td><td rowspan="1" colspan="1" valign="middle" align="left">-</td><td rowspan="1" colspan="1" valign="middle" align="left">-</td><td rowspan="1" colspan="1" valign="middle" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs6469656 (8q23.3)</td><td rowspan="1" colspan="1" valign="middle" align="left">3′ of <em>EIF3H</em></td><td rowspan="1" colspan="1" valign="middle" align="left">13</td><td rowspan="1" colspan="1" valign="middle" align="left">125</td><td rowspan="1" colspan="1" valign="middle" align="left">Promoter, enhancer</td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>EIF3H</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left">Up-regulated</td><td rowspan="1" colspan="1" valign="middle" align="left">Translational initiation</td><td rowspan="1" colspan="1" valign="middle" align="left">Promotion of protein synthesis and cancer cell proliferation</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs4919687 (10q24.3)</td><td rowspan="1" colspan="1" valign="middle" align="left">Intron 1 of <em>CYP17A1</em></td><td rowspan="1" colspan="1" valign="middle" align="left">40</td><td rowspan="1" colspan="1" valign="middle" align="left">412</td><td rowspan="1" colspan="1" valign="middle" align="left">Promoter, enhancer</td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>CYP17A1</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left">N/A</td><td rowspan="1" colspan="1" valign="middle" align="left">Steroid biogenesis</td><td rowspan="1" colspan="1" valign="middle" align="left">Modification of hormone level</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs11064437 (12p13.3)</td><td rowspan="1" colspan="1" valign="middle" align="left">Splice receptor at intron 1 of <em>SPSB2</em></td><td rowspan="1" colspan="1" valign="middle" align="left">79</td><td rowspan="1" colspan="1" valign="middle" align="left">108</td><td rowspan="1" colspan="1" valign="middle" align="left">Promoter, enhancer</td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>SPSB2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left">Up-regulated</td><td rowspan="1" colspan="1" valign="middle" align="left">Proteasomal degradation</td><td rowspan="1" colspan="1" valign="middle" align="left">Less known</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs6061231 (20q13.3)</td><td rowspan="1" colspan="1" valign="middle" align="left">5′ flanking of <em>RPS21</em></td><td rowspan="1" colspan="1" valign="middle" align="left">52</td><td rowspan="1" colspan="1" valign="middle" align="left">126</td><td rowspan="1" colspan="1" valign="middle" align="left">Promoter, enhancer</td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>RPS21</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left">Up-regulated</td><td rowspan="1" colspan="1" valign="middle" align="left">Ribosome biogenesis</td><td rowspan="1" colspan="1" valign="middle" align="left">Potential cancer driver for CRC</td></tr></tbody></table></div><div id="largeobj_idm139959252579680" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4909543/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN15"><sup>*</sup>The expression of each gene was evaluated in paired tumor and adjacent normal colorectal mucosal samples from 188 East Asian patients.</div></div></div><p id="P24" class="p p-last">We evaluated the association of all GWAS-identified CRC genetic susceptibility loci with data available in our study (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>). We replicated all 10 risk loci initially identified in GWAS conducted in East Asians at <em>P</em> &lt; 5 × 10<sup>−8</sup>. Of the 27 risk variants initially identified in European descendants, 21 were associated with CRC risk at <em>P</em> &lt; 0.05 in East Asians in the same direction as reported previously. In particular, variants at 1q41, 8q24.21, 10p14 and 18q21.1 were associated with CRC risk at <em>P</em> &lt; 5 × 10<sup>−8</sup>. SNPs rs6691170, rs16892766, rs35360328, rs3184504, rs73208120, rs72647484 and rs17094983 are monomorphic in East Asian descendants, and thus we were not able to evaluate their associations in our study. We also did not evaluate the X-chromosome SNP rs5934683, for which the genotype data were not available in our study.</p></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Discussion</h2><p id="P25" class="p p-first">In this multiple-staged GWAS with a total of 19,071 cases and 34,624 controls, we identified four novel risk loci for CRC (6p21.1, 8q23.3, 10q24.3 and 12p13.3) and new variants in two previously GWAS-identified loci (10q25.2 and 20q13.3). Using gene expression data from 188 CRC patients of East Asian ancestry, we identified potential candidate genes including <em>TFEB</em>, <em>EIF3H</em>, <em>SPSB2</em> and <em>RPS21.</em></p><p id="P26">At 12p13.3, rs11064437 lies at a splice receptor site within intron 1 of <em>SPSB2</em>. The T allele of rs11064437 changes the splice site from TC (canonical site AG<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517202">33</a></sup> on the reverse strand) to TT, and is predicted to disrupt the splicing and introduce a transcriptional isoform with a shortened untranslated region at exon 2 (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 3</a>). Using data in colon tumor tissues from TCGA, we found a suggestive association of the T allele of rs11064437 with the relative abundance of this isoform (<em>P</em> = 0.11, <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 4</a>). In addition, some of the highly correlated variants were mapped to an active promoter region of <em>SPSB2</em> in normal colorectal cells (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 2</a>). These findings, together with results from the eQTL analysis in CRC patients, suggested that variants at 12p13.3 might contribute to CRC risk by regulating the expression of <em>SPSB</em>2. Previous studies have shown that alternative splicing of untranslated regions of exons can affect the stability<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517222">34</a></sup> and translation efficiency of pre-mRNAs.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517180">35</a></sup> Similarly, it is possible that rs11064437 may affect post-transcriptional regulation of <em>SPSB2</em>. Misregulated pre-mRNA splicing, which produces aberrant isoforms, has been found to contribute to tumorigenesis.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517234">36</a></sup> Therefore, rs11064437 may be one of the causal variants for the association observed at this locus. Future experiments evaluating <em>SPSB2</em> splicing diversity and its relevance for the observed association are needed to elucidate the role of this variant. <em>SPSB2</em> encodes a protein with a suppressor of cytokine signaling domain that functions as an E3 ubiquitin ligase and a SPRY domain that recognizes substrate for ubiquitination and degradation.<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517217">37</a></sup> This protein is predicted to regulate the expression of interacting proteins by targeting them for ubiquitination and subsequent proteasomal degradation.<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517211">38</a></sup> For example, <em>SPSB2</em> specifically interacts with the inducible nitric oxide synthase (iNOS), promotes its degradation and decreases the NO production.<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517198">38</a></sup> Selective inhibitors of iNOS have been proposed for chemoprevention of CRC.<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517221">39</a></sup> Nonetheless, there were other potential target genes at this locus, such as <em>ENO2</em>, <em>CDCA3 and USP5</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>).</p><p id="P27">At 6p21.1, rs4711689 lies in intron 6 of <em>TFEB</em>. This SNP and its highly correlated SNPs were mapped to a region with enhancer activities in normal colorectal tissues (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 2</a>). Using enhancer-promoter interaction data derived by the FANTOM5 project, we identified <em>TFEB</em> as a potential target gene (enhancer near rs4711689 (chr6:41672431-41703298) and TFEB expression pairwise correlation r =0.33, FDR &lt; 2.2 ×10<sup>−16</sup>). Together with the eQTL analysis at this locus, we proposed that <em>TFEB</em> was a likely candidate gene at this locus. <em>TFEB</em> encodes a basic helix-loop helix leucine zipper transcription factor, which has been found to be oncogenic for melanoma,<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517226">40</a></sup> renal cell carcinoma<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517203">41</a></sup> and some sarcomas.<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517183">42</a></sup> However, the role of <em>TFEB</em> in CRC is unknown. Recent studies have shown that this gene is a master regulator of the autophagy-lysosome pathway and of energy metabolism.<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517201">43</a></sup> Deregulated autophagy has been linked to cancer initiation and progression.<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517190">44</a></sup> These pieces of evidence further support our hypothesis that <em>TFEB</em> is the likely candidate gene at this locus. Notably, <em>TFEB</em> activation is regulated by mTORC1 signaling<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517233">45</a></sup>, which is thought to be one of the fundamental mechanisms for sustaining tumor growth in CRC.<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517214">46</a></sup> It is possible that <em>TFEB</em> may contribute to CRC risk, at least in part, by mediating downstream effects of the mTORC1 pathways. Our results, along with those from previous studies, support a new role for <em>TFEB</em> and provide clues for future studies on the contribution of autophagy deregulation to colorectal etiology.</p><p id="P28">At 10q24.3, rs4919687 lies in intron 1 of the gene <em>CYP17A1</em>. This gene encodes a membrane-bound dual-function monooxygenase that catalyzes both 17α-hydroxylase and 17, 20-lyase activities to produce androgenic and estrogenic sex steroids, which is the key branch point in steroidogenesis. Epidemiological studies suggest that steroid hormones may be associated with risk of CRC and other cancers.<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517188">47</a></sup> Thought to affect steroid hormone levels, <em>CYP17A1</em> variants<em>,</em> particularly promoter variant rs743572, have been extensively studied in association with risk of hormone-related cancers; <sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517178">48</a>, <a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517236">49</a></sup> however, the results were very inconsistent. Interestingly, this promoter variant (rs743572) is correlated with the risk variant rs4919687 identified in our study (r<sup>2</sup> = 0.24, D′&lt;1 in East Asians and r<sup>2</sup> = 0.67, D′=1 in European descendants). We herein provide, for the first time, convincing evidence for an association of <em>CYP17A1</em> gene variants with CRC risk, suggesting that steroid hormones may indeed play a role in the etiology of CRC.</p><p id="P29">At 8q23.3, two risk variants were identified in our study. Consistent with results from our eQTL analyses, rs2450115 and its correlated variants were mapped to a region of a quiescent state in both normal and cancerous colorectal cells (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 2</a>), suggesting that these DNAs may be largely inactive, while some variants that are highly correlated with rs6469656 were mapped to active promoter regions of the gene <em>EIF3H</em> (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 2</a>). Thus, it is likely that some of these variants might be involved in the transcriptional regulation of <em>EIF3H</em>. Together with the evidence from the eQTL analysis, we identified <em>EIF3H</em> as a likely target gene for variants at this locus. <em>EIF3H</em> encodes a subunit of the eukaryotic translation initiation factor 3. An oncogenic role of this gene was demonstrated by its ability to transform NIH-3T3 cells into their malignant forms.<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517193">50</a></sup> Recent studies have shown that <em>EIF3H</em> promoted cancer cell growth by stimulating protein synthesis.<sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517209">51</a></sup> Another possible candidate gene at this locus is <em>RAD21</em> (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 2</a>), which encodes a subunit of cohesin that involves chromosome segregation and DNA repair. A recent study showed that <em>RAD21</em> was a pivotal mediator of <em>APC</em> heterozygous loss, the event initiating colorectal tumorigenesis.<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517213">52</a></sup></p><p id="P30">At 20q13.3, rs6061231 lies 5.2kb upstream of the gene <em>RPS21</em>, and 36kb telomeric to rs4925386 (r<sup>2</sup> = 0.15, D′ &lt;1 in East Asians and r<sup>2</sup> = 0.44, D′ &lt;1 European descendants), a risk variant identified in a previous GWAS conducted in populations of European ancestry.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517182">27</a></sup> However, rs4925386 showed a much weaker association with CRC risk (OR = 1.05, <em>P</em> = 0.06) than rs6061231 (OR = 1.20, <em>P</em> = 2.06 ×10<sup>−6</sup>) in our analysis of 2,098 cases and 6,172 controls of East-Asian ancestry for which these two SNPs were genotyped (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>). The association with rs6061231 remained statistically significant, while the association with rs4925386 diminished in the conditional analysis including both SNPs in the model (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>), indicating that the association signal at 20q13.3 was better captured by rs6061231 in East Asians. SNP rs6061231 and some of its highly correlated SNPs were mapped to a 5′ flanking region of <em>RPS21</em> (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 2</a>), suggesting that these variants might affect the transcriptional activities of <em>RPS21</em>. In line with evidence from eQTL analysis, these epigenomic data supported the hypothesis that <em>RPS21</em> was the likely target gene. <em>RPS21</em> encodes a ribosomal protein that is a component of the 40S subunit of ribosomes. However, its role in tumorigenesis remains unclear. A recent study that characterized differences in eQTLs between tumor and matched normal colon tissues showed that <em>RPS21</em> had a significantly higher allelic-specific expression somatic event rate and suggested that this gene was a potential driver in CRC.<sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517200">53</a></sup> This finding further supported that <em>RPS21</em> played a role in CRC etiology.</p><p id="P31">In the analysis comparing mRNA and protein levels in adjacent normal tissues, we observed a positive correlation of the mRNA level and protein abundance for the <em>EIF3H</em> and <em>RPS21</em> genes. For the <em>TFEB</em> and <em>SPSB2</em> genes, however, their mRNA and protein levels were inversely correlated, although the correlation for the <em>SPSB2</em> gene was not statistically significant. Although we could not rule out the possibility of chance finding for the inverse correlation, TFEB and SPSB2 are regulatory proteins that are likely to be unstable but have stable mRNAs in steady-state cells. As suggested by previous studies,<sup><a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517206">54</a></sup> such proteins are predisposed to rapid translational regulation, and thus may sometimes show an inverse correlation of their mRNAs. On the other hand, EIF3H and RPS21 are structural proteins with stable mRNAs and proteins, for which mRNA levels may explain a majority of the variation in protein levels, according to previous studies of proteomics in colon tumors cells and other mammalian cells. <sup><a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517186">55</a></sup></p><p id="P32">By comparing the expression level of candidate genes in tumors with adjacent normal tissues, we showed that these genes were deregulated in tumor tissues, suggesting a possible role of these genes in the development of CRC. For example, the down regulation of <em>TFEB</em>, the master regulator of autophagy, in tumor tissues compared to adjacent normal tissues suggests a decreased level of autophagy in the tumors, which was consistent with previous studies showing that autophagy was suppressed in certain cancer cells.<sup><a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517225">56</a></sup> Furthermore, it has been proposed that <em>TFEB</em> is a druggable target for autophagy induction therapy<sup><a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517208">57</a></sup> that has been suggested as a treatment for some cancers. <sup><a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539517197">58</a></sup></p><p id="P33">We found that more than three quarters of the risk variants initially identified in GWAS conducted in European descendants were directly replicated in East Asians at <em>P</em> &lt; 0.05, indicating that European and East Asian descendants share most of the genetic risk variants for CRC. However, the strength of the association for some of these risk variants was weaker in East Asians than European descendants. Furthermore, approximately 25% of the risk variants identified initially in studies of European descendants were not replicated in our study, including 7 SNPs that are monomorphic in Asians. Not replicating some of the known risk variants is expected given the difference in LD patterns between Asian and European-ancestry populations. It is likely that other variants in these loci may be associated with CRC risk, and fine-mapping studies are needed to identify these variants.</p><p id="P34">Our study has several limitations. Multiple participating studies did not collect information for family history of CRC, preventing us from evaluating the possible interaction between family history and these risk variants with adequate power, particularly since the prevalence of family history of CRC is low in the study populations. We evaluated eQTLs only for the genes closest to the risk variants identified in this study. However, it is possible that these risk variants might be correlated with other genes. Future studies are needed to fully uncover the biologic mechanism for the associations observed at these novel loci.</p><p id="P35" class="p p-last">In conclusion, to our knowledge, this is the largest GWAS conducted to date to search for novel genetic susceptibility loci for CRC. Our study provides strong evidence for possible roles of <em>SPSB2, TFEB, EIF3H, CYP17A1</em> and <em>RPS21</em> in the etiology of CRC. These genes are involved in different aspects of cellular homeostasis, from translational initiation, protein synthesis, to proteasomal degradation and lysosomal biogenesis and autophagy, highlighting the connection of homeostatic regulation and CRC etiology. Aberrant regulation of intestinal epithelial homeostasis plays an important role in the initiation and progression of CRC. Our study has provided additional insights into the genetics and biology of CRC. Results from our study will be helpful for future studies to uncover molecular mechanisms of CRC tumorigenesis.</p></div><div id="S15" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d37e1609"><a href="/pmc/articles/PMC4909543/bin/NIHMS766698-supplement-1.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(238K, docx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><div class="sup-box half_rhythm" id="d37e1613"><a href="/pmc/articles/PMC4909543/bin/NIHMS766698-supplement-2.tiff" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(5.9M, tiff)</sup></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Acknowledgments</h2><div class="sec"><p id="__p2"><strong>Grant Support</strong></p><p id="__p3">The work at the Vanderbilt University School of Medicine was supported by U.S. National Institutes of Health grants R01CA188214, R37CA070867, R01CA124558, and R01CA148667 as well as Ingram Professorship and Anne Potter Wilson Chair funds from the Vanderbilt University School of Medicine. Sample preparation and genotyping assays at Vanderbilt University were conducted at the Survey and Biospecimen Shared Resources and Vanderbilt Microarray Shared Resource, which are supported in part by the Vanderbilt-Ingram Cancer Center (P30CA068485). Imputation and statistical analyses were performed at the servers maintained by the Advanced Computing Center for Research and Education at Vanderbilt University. Studies (grant support) participating in the Asia Colorectal Cancer Consortium include the Shanghai Women’s Health Study (US NIH, R37CA070867, UM1CA182910), the Shanghai Men’s Health Study (US NIH, R01CA082729, UM1CA173640), the Shanghai Breast and Endometrial Cancer Studies (US NIH, R01CA064277 and R01CA092585; contributing only controls), Shanghai Colorectal Cancer Study 3 (US NIH, R37CA070867, R01CA188214 and Ingram Professorship funds), the Guangzhou Colorectal Cancer Study (National Key Scientific and Technological Project, 2011ZX09307-001-04; the National Basic Research Program, 2011CB504303, contributing only controls; the Natural Science Foundation of China, 81072383, contributing only controls), the Japan BioBank Colorectal Cancer Study (grant from the Ministry of Education, Culture, Sports, Science and Technology of the Japanese government), the Hwasun Cancer Epidemiology Study–Colon and Rectum Cancer (HCES-CRC; grants from Chonnam National University Hwasun Hospital, HCRI15011-1). The Aichi Colorectal Cancer Study (Grant-in-Aid for Cancer Research, grant for the Third Term Comprehensive Control Research for Cancer and Grants-in-Aid for Scientific Research from the Japanese Ministry of Education, Culture, Sports, Science and Technology, 17015018 and 221S0001), the Korea-NCC (National Cancer Center) Colorectal Cancer Study (Basic Science Research Program through the National Research Foundation of Korea, 2010-0010276 and 2013R1A1A2A10008260; National Cancer Center Korea, 0910220), and the KCPS-II Colorectal Cancer Study (National R&amp;D Program for Cancer Control, 1220180; Seoul R&amp;D Program, 10526).</p><p id="__p4">Participating studies (grant support) in the GECCO, CORECT and CCFR GWAS meta-analysis are GECCO (US NIH, U01CA137088 and R01CA059045), DALS (US NIH, R01CA048998), DACHS (German Federal Ministry of Education and Research, BR 1704/6-1, BR 1704/6-3, BR 1704/6-4, CH 117/1-1, 01KH0404 and 01ER0814), HPFS (P01 CA 055075, UM1 <a id="__tag_805013097" class="tag_hotlink" href="/nuccore/CA167552" ref="/nuccore/CA167552"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA167552","term_id":"35088086","term_text":"CA167552"}}</span>CA167552</a>, R01 CA137178, R01 CA151993 and P50 CA127003), NHS (UM1 <a id="__tag_805013095" class="tag_hotlink" href="/nuccore/CA186107" ref="/nuccore/CA186107"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA186107","term_id":"35125552","term_text":"CA186107"}}</span>CA186107</a>, R01 CA137178, P01 CA87969, R01 CA151993 and P50 CA127003), OFCCR (US NIH, U01CA074783), PMH (US NIH, R01CA076366), PHS (US NIH, R01CA042182), VITAL (US NIH, K05CA154337), WHI (US NIH, HHSN268201100046C, HHSN268201100001C, HHSN268201100002C, HHSN268201100003C, HHSN268201100004C, and HHSN271201100004C) and PLCO (US NIH, Z01CP 010200, U01HG004446 and U01HG 004438). CORECT is supported by the National Cancer Institute as part of the GAME-ON consortium (US NIH, U19CA148107) with additional support from National Cancer Institute grants (R01CA81488 and P30CA014089), the National Human Genome Research Institute at the US NIH (T32HG000040) and the National Institute of Environmental Health Sciences at the US NIH (T32ES013678). CCFR is supported by the National Cancer Institute, US NIH under RFA CA-95-011 and through cooperative agreements with members of the Colon Cancer Family Registry and principal investigators of the Australasian Colorectal Cancer Family Registry (US NIH, U01CA097735), the Familial Colorectal Neoplasia Collaborative Group (US NIH, U01CA074799) (University of Southern California), the Mayo Clinic Cooperative Family Registry for Colon Cancer Studies (US NIH, U01CA074800), the Ontario Registry for Studies of Familial Colorectal Cancer (US NIH, U01CA074783), the Seattle Colorectal Cancer Family Registry (US NIH, U01CA074794) and the University of Hawaii Colorectal Cancer Family Registry (US NIH, U01CA074806). The GWAS work was supported by a National Cancer Institute grant (US NIH, U01CA122839). OFCCR was supported by a GL2 grant from the Ontario Research Fund, Canadian Institutes of Health Research and a Cancer Risk Evaluation (CaRE) Program grant from the Canadian Cancer Society Research Institute. T.J. Hudson and B.W. Zanke are recipients of Senior Investigator Awards from the Ontario Institute for Cancer Research, through support from the Ontario Ministry of Economic Development and Innovation. ASTERISK was funded by a Regional Hospital Clinical Research Program (PHRC) and supported by the Regional Council of Pays de la Loire, the Groupement des Entreprises Françaises dans la Lutte contre le Cancer (GEFLUC), the Association Anne de Bretagne Génétique and the Ligue Régionale Contre le Cancer (LRCC).</p><p id="__p5">The authors are solely responsible for the scientific content of this paper. The sponsors of this study had no role in study design, data collection, analysis, interpretation, writing of the report or the decision for submission. We thank all study participants and research staff of all parent studies for their contributions and commitment to this project. We thank Vanderbilt staff members Jing He for data processing and analyses and Kimberly Kreth for editing and preparing the manuscript. We thank Dr. Bing Zhang, Vanderbilt University for his suggestions on protein validation. Sample preparation and genotyping assays at Vanderbilt University were conducted at the Survey and Biospecimen Shared Resources and Vanderbilt Microarray Shared Resource, which are supported in part by the Vanderbilt-Ingram Cancer Center (P30CA068485). Imputation and statistical analyses were performed at the servers maintained by the Advanced Computing Center for Research and Education at Vanderbilt University. Studies (grant support) participating in the Asia Colorectal Cancer Consortium include the Shanghai Women’s Health Study (US NIH, R37CA070867), the Shanghai Men’s Health Study (US NIH, R01CA082729), the Shanghai Breast and Endometrial Cancer Studies (US NIH, R01CA064277 and R01CA092585; contributing only controls), Shanghai Colorectal Cancer Study 3 (US NIH, R37CA070867, R01CA188214 and Ingram Professorship funds), the Guangzhou Colorectal Cancer Study (National Key Scientific and Technological Project, 2011ZX09307-001-04; the National Basic Research Program, 2011CB504303, contributing only controls; the Natural Science Foundation of China, 81072383, contributing only controls), the Japan BioBank Colorectal Cancer Study (grant from the Ministry of Education, Culture, Sports, Science and Technology of the Japanese government), the Hwasun Cancer Epidemiology Study–Colon and Rectum Cancer (HCES-CRC; grants from Chonnam National University Hwasun Hospital, HCRI15011-1). The Aichi Colorectal Cancer Study (Grant-in-Aid for Cancer Research, grant for the Third Term Comprehensive Control Research for Cancer and Grants-in-Aid for Scientific Research from the Japanese Ministry of Education, Culture, Sports, Science and Technology, 17015018 and 221S0001), the Korea-NCC (National Cancer Center) Colorectal Cancer Study (Basic Science Research Program through the National Research Foundation of Korea, 2010-0010276 and 2013R1A1A2A10008260; National Cancer Center Korea, 0910220), and the KCPS-II Colorectal Cancer Study (National R&amp;D Program for Cancer Control, 1220180; Seoul R&amp;D Program, 10526).</p><p id="__p6">The paired colorectal tumor and adjacent normal tissues and data used for the gene expression analysis were provide by the Biobank of Chonnam National University Hwasun Hospital, a member of the Korea Biobank Network and General Department, Tangdu Hospital, Fourth Military Medical University, Xi’an, Shaanxi, China.</p><p id="__p7">We also thank all participants, staff and investigators from the GECCO, CORECT and CCFR consortia for making it possible to present results from populations of European ancestry for the new CRC-associated loci identified among East Asians. GECCO, CORECT and CCFR are directed by U. Peters, S. Gruber and G. Casey, respectively. Complete lists of investigators from the GECCO, CORECT and CCFR consortia are provided below.</p><p id="__p8">Investigators (institution and location) in the GECCO consortium include (in alphabetical order): John A. Baron (Division of Gastroenterology and Hepatology, University of North Carolina School of Medicine, Chapel Hill, North Carolina, USA), Sonja I. Berndt (Division of Cancer Epidemiology and Genetics, National Cancer Institute, US NIH, Bethesda, Maryland, USA), Stéphane Bezieau (Service de Génétique Médicale, Centre Hospitalier Universitaire (CHU) Nantes, Nantes, France), Hermann Brenner (Division of Clinical Epidemiology and Aging Research, German Cancer Research Center, Heidelberg, Germany), Bette J. Caan (Division of Research, Kaiser Permanente Medical Care Program, Oakland, California, USA), Christopher S. Carlson (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, School of Public Health, University of Washington, Seattle, Washington, USA), Graham Casey (Department of Preventive Medicine, University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA), Andrew T. Chan (Division of Gastroenterology, Massachusetts General Hospital and Harvard Medical School, Boston, Massachusetts, USA and Channing Division of Network Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA), Jenny Chang-Claude (Division of Cancer Epidemiology, German Cancer Research Center, Heidelberg, Germany), Stephen J. Chanock (Division of Cancer Epidemiology and Genetics, National Cancer Institute, US NIH, Bethesda, Maryland, USA), David V. Conti (Department of Preventive Medicine, University of Southern California, Los Angeles, California, USA), Keith Curtis (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA), David Duggan (Translational Genomics Research Institute, Phoenix, Arizona, USA), Charles S. Fuchs (Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts, USA and Channing Division of Network Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA), Steven Gallinger (Department of Surgery, Mount Sinai Hospital, Toronto, Ontario, Canada and Samuel Lunenfeld Research Institute, Toronto, Ontario, Canada), Edward L. Giovannucci (Channing Division of Network Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts, USA and Department of Nutrition, Harvard School of Public Health, Boston, Massachusetts, USA), Stephen B. Gruber (University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA), Robert W. Haile (Stanford School of Medicine, Stanford, CA ), Tabitha A. Harrison (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA), Richard B. Hayes (Division of Epidemiology, Department of Environmental Medicine, New York University School of Medicine, New York, New York, USA), Michael Hoffmeister (Division of Clinical Epidemiology and Aging Research, German Cancer Research Center, Heidelberg, Germany), John L. Hopper (Melbourne School of Population Health, The University of Melbourne, Melbourne, Victoria, Australia), Li Hsu (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA and Department of Biostatistics, University of Washington, Seattle, Washington, USA), Thomas J. Hudson (Department of Medical Biophysics, University of Toronto, Toronto, Ontario, Canada and Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada), David J. Hunter (Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts, USA), Carolyn M. Hutter (Division of Cancer Control and Population Sciences, National Cancer Institute, US NIH, Bethesda, Maryland, USA), Rebecca D. Jackson (Division of Endocrinology, Diabetes and Metabolism, Ohio State University, Columbus, Ohio, USA), Mark A. Jenkins (Melbourne School of Population Health, The University of Melbourne, Melbourne, Victoria, Australia), Shuo Jiao (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA), Sébastien Küry (Service de Génétique Médicale, CHU Nantes, Nantes, France), Loic Le Marchand (Epidemiology Program, University of Hawaii Cancer Center, Honolulu, Hawaii, USA), Mathieu Lemire (Ontario Institute for Cancer Research, Toronto, Ontario, Canada), Noralane M. Lindor (Department of Health Sciences Research, Mayo Clinic, Scottsdale, Arizona, USA), Jing Ma (Channing Division of Network Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA), Polly A. Newcomb (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA and Department of Epidemiology, University of Washington School of Public Health, Seattle, Washington, USA), Ulrike Peters (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA and Department of Epidemiology, University of Washington School of Public Health, Seattle, Washington, USA), John D. Potter (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA, Department of Epidemiology, University of Washington School of Public Health, Seattle, Washington, USA and Centre for Public Health Research, Massey University, Palmerston North, New Zealand), Conghui Qu (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA), Robert E. Schoen (Department of Medicine and Epidemiology, University of Pittsburgh Medical Center, Pittsburgh, Pennsylvania, USA), Fredrick R. Schumacher (Department of Preventive Medicine, University of Southern California, Los Angeles, California, USA), Daniela Seminara (Division of Cancer Control and Population Sciences, National Cancer Institute, US NIH, Bethesda, Maryland, USA), Martha L. Slattery (Department of Internal Medicine, University of Utah Health Sciences Center, Salt Lake City, Utah, USA), Stephen N. Thibodeau (Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, Minnesota, USA and Department of Laboratory Genetics, Mayo Clinic, Rochester, Minnesota, USA), Emily White (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA and Department of Epidemiology, University of Washington School of Public Health, Seattle, Washington, USA) and Brent W. Zanke (Clinical Epidemiology Program, Ottawa Hospital Research Institute, Ottawa, Ontario, Canada).</p><p id="__p9">Investigators (institution and location) from the CORECT consortium include (in alphabetical order): Kendra Blalock (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA), Peter T. Campbell (Epidemiology Research Program, American Cancer Society, Atlanta, Georgia, USA), Graham Casey (Department of Preventive Medicine, University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA), David V. Conti (Department of Preventive Medicine, University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA), Christopher K. Edlund (Department of Preventive Medicine, University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA), Jane Figueiredo (Department of Preventive Medicine, University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA), W. James Gauderman (Department of Preventive Medicine, University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA), Jian Gong (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA), Roger C. Green (Faculty of Medicine, Memorial University of Newfoundland, St. John’s, Newfoundland, Canada), Stephen B. Gruber (University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA), John F. Harju (University of Michigan Comprehensive Cancer Center, University of Michigan, Ann Arbor, Michigan, USA), Tabitha A. Harrison (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA), Eric J. Jacobs (Epidemiology Research Program, American Cancer Society, Atlanta, Georgia, USA), Mark A. Jenkins (Melbourne School of Population Health, The University of Melbourne, Melbourne, Victoria, Australia), Shuo Jiao (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA), Li Li (Case Comprehensive Cancer Center, Case Western Reserve University, Cleveland, Ohio, USA), Yi Lin (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA), Frank J. Manion (University of Michigan Comprehensive Cancer Center, University of Michigan, Ann Arbor, Michigan, USA), Victor Moreno (Institut d’Investigació Biomèdica de Bellvitge, Institut Catala d’Oncologia, Hospitalet, Barcelona, Spain), Bhramar Mukherjee (University of Michigan Comprehensive Cancer Center, University of Michigan, Ann Arbor, Michigan, USA), Ulrike Peters (Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, USA), Leon Raskin (University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA), Fredrick R. Schumacher (Department of Preventive Medicine, University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA), Daniela Seminara (Division of Cancer Control and Population Sciences, National Cancer Institute, US NIH, Bethesda, Maryland, USA), Gianluca Severi (Melbourne School of Population Health, The University of Melbourne, Melbourne, Victoria, Australia), Stephanie L. Stenzel (Department of Preventive Medicine, University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA) and Duncan C. Thomas (Department of Preventive Medicine, University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA).</p><p id="__p10">The CCFR consortium is represented by Graham Casey (Department of Preventive Medicine, University of Southern California Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA).</p><p id="__p11">We also thank B. Buecher of ASTERISK; U. Handte-Daub, M. Celik, R. Hettler-Jensen, U. Benscheid and U. Eilber of DACHS; and P. Soule, H. Ranu, I. Devivo, D.J. Hunter, Q. Guo, L. Zhu and H. Zhang of HPFS, NHS and PHS, as well as the following state cancer registries for their help: Alabama, Arizona, Arkansas, California, Colorado, Connecticut, Delaware, Florida, Georgia, Idaho, Illinois, Indiana, Iowa, Kentucky, Louisiana, Maine, Maryland, Massachusetts, Michigan, Nebraska, New Hampshire, New Jersey, New York, North Carolina, North Dakota, Ohio, Oklahoma, Oregon, Pennsylvania, Rhode Island, South Carolina, Tennessee, Texas, Virginia, Washington and Wyoming. We thank C. Berg and P. Prorok of PLCO; T. Riley of Information Management Services, Inc.; B. O’Brien of Westat, Inc.; B. Kopp and W. Shao of SAIC-Frederick; the WHI investigators (see <a href="http://www.whi.org/researchers/Documents%20%20Write%20a%20Paper/WHI%20Investigator" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.whi.org/researchers/Documents%20%20Write%20a%20Paper/WHI%20Investigator%20Short%20List.pdf</a>) and the GECCO Coordinating Center. The authors acknowledge Dave Duggan and team members at TGEN (Translational Genomics Research Institute), the Broad Institute, and the Génome Québec Innovation Center for genotyping DNA samples of cases and controls in GECCO, and for scientific input for GECCO. Participating studies (grant support) in the GECCO, CORECT and CCFR GWAS meta-analysis are GECCO (US NIH, U01CA137088 and R01CA059045), DALS (US NIH, R01CA048998), DACHS (German Federal Ministry of Education and Research, BR 1704/6-1, BR 1704/6-3, BR 1704/6-4, CH 117/1-1, 01KH0404 and 01ER0814), HPFS (P01 CA 055075, UM1 <a id="__tag_805013096" class="tag_hotlink" href="/nuccore/CA167552" ref="/nuccore/CA167552"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA167552","term_id":"35088086","term_text":"CA167552"}}</span>CA167552</a>, R01 137178, R01 CA151993 and P50 CA127003), NHS (UM1 <a id="__tag_805013094" class="tag_hotlink" href="/nuccore/CA186107" ref="/nuccore/CA186107"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA186107","term_id":"35125552","term_text":"CA186107"}}</span>CA186107</a>, R01 CA137178, P01 CA87969, R01 CA151993 and P50 CA127003), OFCCR (US NIH, U01CA074783), PMH (US NIH, R01CA076366), PHS (US NIH, R01CA042182), VITAL (US NIH, K05CA154337), WHI (US NIH, HHSN268201100046C, HHSN268201100001C, HHSN268201100002C, HHSN268201100003C, HHSN268201100004C, and HHSN271201100004C) and PLCO (US NIH, Z01CP 010200, U01HG004446 and U01HG 004438). CORECT is supported by the National Cancer Institute as part of the GAME-ON consortium (US NIH, U19CA148107) with additional support from National Cancer Institute grants (R01CA81488 and P30CA014089), the National Human Genome Research Institute at the US NIH (T32HG000040) and the National Institute of Environmental Health Sciences at the US NIH (T32ES013678). CCFR is supported by the National Cancer Institute, US NIH under RFA CA-95-011 and through cooperative agreements with members of the Colon Cancer Family Registry and principal investigators of the Australasian Colorectal Cancer Family Registry (US NIH, U01CA097735), the Familial Colorectal Neoplasia Collaborative Group (US NIH, U01CA074799) (University of Southern California), the Mayo Clinic Cooperative Family Registry for Colon Cancer Studies (US NIH, U01CA074800), the Ontario Registry for Studies of Familial Colorectal Cancer (US NIH, U01CA074783), the Seattle Colorectal Cancer Family Registry (US NIH, U01CA074794) and the University of Hawaii Colorectal Cancer Family Registry (US NIH, U01CA074806). The GWAS work was supported by a National Cancer Institute grant (US NIH, U01CA122839). OFCCR was supported by a GL2 grant from the Ontario Research Fund, Canadian Institutes of Health Research and a Cancer Risk Evaluation (CaRE) Program grant from the Canadian Cancer Society Research Institute. T.J. Hudson and B.W. Zanke are recipients of Senior Investigator Awards from the Ontario Institute for Cancer Research, through support from the Ontario Ministry of Economic Development and Innovation. ASTERISK was funded by a Regional Hospital Clinical Research Program (PHRC) and supported by the Regional Council of Pays de la Loire, the Groupement des Entreprises Françaises dans la Lutte contre le Cancer (GEFLUC), the Association Anne de Bretagne Génétique and the Ligue Régionale Contre le Cancer (LRCC). PLCO data sets were accessed with approval through dbGaP (CGEMS prostate cancer scan, phs000207.v1.p1 (Yeager, M et al. Genome-wide association study of prostate cancer identifies a second risk locus at 8q24. <em>Nat Genet</em> 2007 May;39(5):645-9); CGEMS pancreatic cancer scan, phs000206.v4.p3 (Amundadottir, L et al. Genome-wide association study identifies variants in the ABO locus associated with susceptibility to pancreatic cancer. <em>Nat Genet</em>. 2009 Sep;41(9):986-90, and Petersen, GM et al. A genome-wide association study identifies pancreatic cancer susceptibility loci on chromosomes 13q22.1, 1q32.1 and 5p15.33. <em>Nat Genet</em>. 2010 Mar;42(3):224-8); and GWAS of Lung Cancer and Smoking, phs000093.v2.p2 (Landi MT, et al. A genome-wide association study of lung cancer identifies a region of chromosome 5p15 associated with risk for adenocarcinoma. <em>Am J Hum Genet.</em> 2009 Nov;85(5):679-91), which was funded by Z01CP 010200, U01HG004446 and U01HG 004438 from the US NIH).</p></div></div><div id="GL" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="GLtitle">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">ACCC</td><td>Asia Colorectal Cancer Consortium</td></tr><tr><td id="G2">CRC</td><td>colorectal cancer</td></tr><tr><td id="G3">CYP17A1</td><td>cytochrome P450, family 17, subfamily A, polypeptide 1</td></tr><tr><td id="G4">EIF3H</td><td>Eukaryotic Translation Initiation Factor 3, Subunit H</td></tr><tr><td id="G5">GWAS</td><td>genome-wide association study</td></tr><tr><td id="G6">RPS21</td><td>ribosomal protein S21</td></tr><tr><td id="G7">SNP</td><td>single nucleotide polymorphism</td></tr><tr><td id="G8">SPSB2</td><td>splA/ryanodine receptor domain and SOCS box containing 2</td></tr><tr><td id="G9">TFEB</td><td>transcription factor EB</td></tr><tr><td id="G10">VTI1A</td><td>vesicle transport through interaction with t-SNAREs 1A</td></tr></tbody></table></div></div></div><div id="idm139959251106320" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139959251106320title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="__p12" class="p p-first"><strong>Disclosures</strong></p><p id="__p13" class="p p-last">The authors declare no competing financial, professional, or personal interests.</p><p></p><div><p id="__p14" class="p p-first"><strong>Author Contributions</strong></p>W.Z. conceived and directed the Asia Colorectal Cancer Consortium and the Shanghai-Vanderbilt Colorectal Cancer Genetics Project. W.H.J. and Y.X.Z., D.L., K. Matsuda, S.S.K., K. Matsuo, X.O.S., Y.B.X. and Y.T.G., A.S., S.H.J., D.H.K., U.P., S.B.G, and G.C. directed CRC projects in Guangzhou, Beijing, BBJ, HCES-CRC, Aichi, Shanghai, Korea-NCC, KCPS-II, Korea-Seoul, GECCO, CORECT, and CCFR, respectively. C.Z. and W.Z. wrote the paper with significant contributions from X.O.S., Q.C., J.L., W.W., B.L. and X.G. Q.C. coordinated the project and directed the lab operations. N.W. and J.W. performed the gene expression experiments. Q.C. and R.C. performed the Western blot analysis. J.W. and J.S. performed the genotyping experiments. C.Z., B.Z. and W.W. performed the statistical analyses. C.Z. and X.G. performed the bioinformatics analyses. A.T. conducted the statistical analyses and imputation for BBJ. C.Z. and J.L. managed the data. All authors contributed to data and biological sample collection in the original studies included in this project and/or manuscript revision. J.C. and C.W. contributed to data and biological sample collection for the study in Beijing. H.B., J.A.B., K.B., S.B., S.I.B., A.T.C., B.J.C., C.S.C., D.V.C., G.C., K.C., P.T.C., S.J.C., J.C.-C., D.D., C.K.E., C.S.F., J.F., E.L.G., J.G., R.C.G., S.G., S.B.G., W.J.G., C.M.H., D.J.H., J.D.H., J.F.H., J.L.H., L.H., M.H., R.B.H., R.W.H., T.A.H., T.J.H., E.J.J., M.A.J., R.D.J., S.J., S.K., L.L., M.L., N.M.L., Y.L., L.Le Marchand, B.M., F.J.M., J.M., V.M., P.A.N, J.D.P., U.P., C.Q., L.R., T.R., D.S., F.R.S., G.S., M.L.S., R.E.S., S.L.S., D.C.T., S.N.T., E.W., E.W. and B.W.Z. contributed to data and biological sample collection for the studies included in GECCO, CORECT and CCFR. All authors have reviewed and approved the content of the paper.</div><p class="fn sec" id="FN6"></p><p id="__p16" class="p p-first"><strong>URLs</strong></p><p id="__p17" class="p">Illumina HumanEXome-12v1_A Beadchip: <a href="http://www.illumina.com/products/infinium_humanexome_beadchip_kit.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.illumina.com/products/infinium_humanexome_beadchip_kit.html</a></p><p id="__p18" class="p">R: <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a></p><p id="__p19" class="p p-last">Haplo.stats package: <a href="http://www.mayo.edu/research/labs/statistical-genetics-genetic-epidemiology/software" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.mayo.edu/research/labs/statistical-genetics-genetic-epidemiology/software</a></p><p></p></div></div><div id="idm139959292762448" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139959292762448title">References</h2><div class="ref-list-sec sec" id="reference-list"><p id="__p30">Author names in bold designate shared co-first authorship.</p><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Torre LA, Bray F, Siegel RL, et al.  Global cancer statistics, 2012. <span><span class="ref-journal">CA Cancer J Clin. </span>2015;<span class="ref-vol">65</span>:87–108.</span> [<a href="/pubmed/25651787" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CA+Cancer+J+Clin&amp;title=Global+cancer+statistics,+2012&amp;author=LA+Torre&amp;author=F+Bray&amp;author=RL+Siegel&amp;volume=65&amp;publication_year=2015&amp;pages=87-108&amp;pmid=25651787&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Lichtenstein P, Holm NV, Verkasalo PK, et al.  Environmental and heritable factors in the causation of cancer - Analyses of cohorts of twins from Sweden, Denmark, and Finland. <span><span class="ref-journal">New England Journal of Medicine. </span>2000;<span class="ref-vol">343</span>:78–85.</span> [<a href="/pubmed/10891514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=New+England+Journal+of+Medicine&amp;title=Environmental+and+heritable+factors+in+the+causation+of+cancer+-+Analyses+of+cohorts+of+twins+from+Sweden,+Denmark,+and+Finland&amp;author=P+Lichtenstein&amp;author=NV+Holm&amp;author=PK+Verkasalo&amp;volume=343&amp;publication_year=2000&amp;pages=78-85&amp;pmid=10891514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Ma X, Zhang B, Zheng W. Genetic variants associated with colorectal cancer risk: comprehensive research synopsis, meta-analysis, and epidemiological evidence. <span><span class="ref-journal">Gut. </span>2014;<span class="ref-vol">63</span>:326–36.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4020522/">PMC free article</a>]</span> [<a href="/pubmed/23946381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Genetic+variants+associated+with+colorectal+cancer+risk:+comprehensive+research+synopsis,+meta-analysis,+and+epidemiological+evidence&amp;author=X+Ma&amp;author=B+Zhang&amp;author=W+Zheng&amp;volume=63&amp;publication_year=2014&amp;pages=326-36&amp;pmid=23946381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Lemire M, Qu C, Loo LW, et al.  A genome-wide association study for colorectal cancer identifies a risk locus in 14q23.1. <span><span class="ref-journal">Hum Genet. </span>2015</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4687971/">PMC free article</a>]</span> [<a href="/pubmed/26404086" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=A+genome-wide+association+study+for+colorectal+cancer+identifies+a+risk+locus+in+14q23.1&amp;author=M+Lemire&amp;author=C+Qu&amp;author=LW+Loo&amp;publication_year=2015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Schumacher FR, Schmit SL, Jiao S, et al.  Genome-wide association study of colorectal cancer identifies six new susceptibility loci. <span><span class="ref-journal">Nat Commun. </span>2015;<span class="ref-vol">6</span>:7138.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4967357/">PMC free article</a>]</span> [<a href="/pubmed/26151821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Genome-wide+association+study+of+colorectal+cancer+identifies+six+new+susceptibility+loci&amp;author=FR+Schumacher&amp;author=SL+Schmit&amp;author=S+Jiao&amp;volume=6&amp;publication_year=2015&amp;pages=7138&amp;pmid=26151821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Al-Tassan NA, Whiffin N, Hosking FJ, et al.  A new GWAS and meta-analysis with 1000Genomes imputation identifies novel risk variants for colorectal cancer. <span><span class="ref-journal">Sci Rep. </span>2015;<span class="ref-vol">5</span>:10442.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4438486/">PMC free article</a>]</span> [<a href="/pubmed/25990418" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=A+new+GWAS+and+meta-analysis+with+1000Genomes+imputation+identifies+novel+risk+variants+for+colorectal+cancer&amp;author=NA+Al-Tassan&amp;author=N+Whiffin&amp;author=FJ+Hosking&amp;volume=5&amp;publication_year=2015&amp;pages=10442&amp;pmid=25990418&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Wang H, Burnett T, Kono S, et al.  Trans-ethnic genome-wide association study of colorectal cancer identifies a new susceptibility locus in VTI1A. <span><span class="ref-journal">Nat Commun. </span>2014;<span class="ref-vol">5</span>:4613.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4180879/">PMC free article</a>]</span> [<a href="/pubmed/25105248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Trans-ethnic+genome-wide+association+study+of+colorectal+cancer+identifies+a+new+susceptibility+locus+in+VTI1A&amp;author=H+Wang&amp;author=T+Burnett&amp;author=S+Kono&amp;volume=5&amp;publication_year=2014&amp;pages=4613&amp;pmid=25105248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Zhang B, Jia WH, Matsuda K, et al.  Large-scale genetic study in East Asians identifies six new loci associated with colorectal cancer risk. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:533–42.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4068797/">PMC free article</a>]</span> [<a href="/pubmed/24836286" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Large-scale+genetic+study+in+East+Asians+identifies+six+new+loci+associated+with+colorectal+cancer+risk&amp;author=B+Zhang&amp;author=WH+Jia&amp;author=K+Matsuda&amp;volume=46&amp;publication_year=2014&amp;pages=533-42&amp;pmid=24836286&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Whiffin N, Hosking FJ, Farrington SM, et al.  Identification of susceptibility loci for colorectal cancer in a genome-wide meta-analysis. <span><span class="ref-journal">Hum Mol Genet. </span>2014;<span class="ref-vol">23</span>:4729–37.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4133584/">PMC free article</a>]</span> [<a href="/pubmed/24737748" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Identification+of+susceptibility+loci+for+colorectal+cancer+in+a+genome-wide+meta-analysis&amp;author=N+Whiffin&amp;author=FJ+Hosking&amp;author=SM+Farrington&amp;volume=23&amp;publication_year=2014&amp;pages=4729-37&amp;pmid=24737748&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Peters U, Jiao S, Schumacher FR, Hutter CM, et al.  Identification of Genetic Susceptibility Loci for Colorectal Tumors in a Genome-Wide Meta-analysis. <span><span class="ref-journal">Gastroenterology. </span>2013;<span class="ref-vol">144</span>:799–807. e24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3636812/">PMC free article</a>]</span> [<a href="/pubmed/23266556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Identification+of+Genetic+Susceptibility+Loci+for+Colorectal+Tumors+in+a+Genome-Wide+Meta-analysis&amp;author=U+Peters&amp;author=S+Jiao&amp;author=FR+Schumacher&amp;author=CM+Hutter&amp;volume=144&amp;publication_year=2013&amp;pages=799-807&amp;pmid=23266556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Jia WH, Zhang B, Matsuo K, et al.  Genome-wide association analyses in East Asians identify new susceptibility loci for colorectal cancer. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:191–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3679924/">PMC free article</a>]</span> [<a href="/pubmed/23263487" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analyses+in+East+Asians+identify+new+susceptibility+loci+for+colorectal+cancer&amp;author=WH+Jia&amp;author=B+Zhang&amp;author=K+Matsuo&amp;volume=45&amp;publication_year=2013&amp;pages=191-6&amp;pmid=23263487&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Dunlop MG, Dobbins SE, Farrington SM, et al.  Common variation near CDKN1A, POLD3 and SHROOM2 influences colorectal cancer risk. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:770–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4747430/">PMC free article</a>]</span> [<a href="/pubmed/22634755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variation+near+CDKN1A,+POLD3+and+SHROOM2+influences+colorectal+cancer+risk&amp;author=MG+Dunlop&amp;author=SE+Dobbins&amp;author=SM+Farrington&amp;volume=44&amp;publication_year=2012&amp;pages=770-6&amp;pmid=22634755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Houlston RS, Cheadle J, Dobbins SE, et al.  Meta-analysis of three genome-wide association studies identifies susceptibility loci for colorectal cancer at 1q41, 3q26.2, 12q13.13 and 20q13. 33. <span><span class="ref-journal">Nature Genetics. </span>2010;<span class="ref-vol">42</span>:973–U89.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5098601/">PMC free article</a>]</span> [<a href="/pubmed/20972440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Meta-analysis+of+three+genome-wide+association+studies+identifies+susceptibility+loci+for+colorectal+cancer+at+1q41,+3q26.2,+12q13.13+and+20q13.+33&amp;author=RS+Houlston&amp;author=J+Cheadle&amp;author=SE+Dobbins&amp;volume=42&amp;publication_year=2010&amp;pages=973-U89&amp;pmid=20972440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Tomlinson IPM, Webb E, Carvajal-Carmona L, et al.  A genome-wide association study identifies colorectal cancer susceptibility loci on chromosomes 10p14 and 8q23. 3. <span><span class="ref-journal">Nature Genetics. </span>2008;<span class="ref-vol">40</span>:623–630.</span> [<a href="/pubmed/18372905" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=A+genome-wide+association+study+identifies+colorectal+cancer+susceptibility+loci+on+chromosomes+10p14+and+8q23.+3&amp;author=IPM+Tomlinson&amp;author=E+Webb&amp;author=L+Carvajal-Carmona&amp;volume=40&amp;publication_year=2008&amp;pages=623-630&amp;pmid=18372905&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Jaeger E, Webb E, Howarth K, et al.  Common genetic variants at the CRAC1 (HMPS) locus on chromosome 15q13. 3 influence colorectal cancer risk. <span><span class="ref-journal">Nature Genetics. </span>2008;<span class="ref-vol">40</span>:26–28.</span> [<a href="/pubmed/18084292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Common+genetic+variants+at+the+CRAC1+(HMPS)+locus+on+chromosome+15q13.+3+influence+colorectal+cancer+risk&amp;author=E+Jaeger&amp;author=E+Webb&amp;author=K+Howarth&amp;volume=40&amp;publication_year=2008&amp;pages=26-28&amp;pmid=18084292&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Houlston RS, Webb E, Broderick P, et al.  Meta-analysis of genome-wide association data identifies four new susceptibility loci for colorectal cancer. <span><span class="ref-journal">Nature Genetics. </span>2008;<span class="ref-vol">40</span>:1426–1435.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2836775/">PMC free article</a>]</span> [<a href="/pubmed/19011631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Meta-analysis+of+genome-wide+association+data+identifies+four+new+susceptibility+loci+for+colorectal+cancer&amp;author=RS+Houlston&amp;author=E+Webb&amp;author=P+Broderick&amp;volume=40&amp;publication_year=2008&amp;pages=1426-1435&amp;pmid=19011631&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Tenesa A, Farrington SM, Prendergast JGD, et al.  Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21. <span><span class="ref-journal">Nature Genetics. </span>2008;<span class="ref-vol">40</span>:631–637.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2778004/">PMC free article</a>]</span> [<a href="/pubmed/18372901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+11q23+and+replicates+risk+loci+at+8q24+and+18q21&amp;author=A+Tenesa&amp;author=SM+Farrington&amp;author=JGD+Prendergast&amp;volume=40&amp;publication_year=2008&amp;pages=631-637&amp;pmid=18372901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Tomlinson I, Webb E, Carvajal-Carmona L, et al.  A genome-wide association scan of tag SNPs identifies a susceptibility variant for colorectal cancer at 8q24. 21. <span><span class="ref-journal">Nature Genetics. </span>2007;<span class="ref-vol">39</span>:984– 988.</span> [<a href="/pubmed/17618284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=A+genome-wide+association+scan+of+tag+SNPs+identifies+a+susceptibility+variant+for+colorectal+cancer+at+8q24.+21&amp;author=I+Tomlinson&amp;author=E+Webb&amp;author=L+Carvajal-Carmona&amp;volume=39&amp;publication_year=2007&amp;pages=984-+988&amp;pmid=17618284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Broderick P, Carvajal-Carmona L, Pittman AM, Webb E, et al.  A genome-wide association study shows that common alleles of SMAD7 influence colorectal cancer risk. <span><span class="ref-journal">Nature Genetics. </span>2007;<span class="ref-vol">39</span>:1315–1317.</span> [<a href="/pubmed/17934461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=A+genome-wide+association+study+shows+that+common+alleles+of+SMAD7+influence+colorectal+cancer+risk&amp;author=P+Broderick&amp;author=L+Carvajal-Carmona&amp;author=AM+Pittman&amp;author=E+Webb&amp;volume=39&amp;publication_year=2007&amp;pages=1315-1317&amp;pmid=17934461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Zanke BW, Greenwood CMT, Rangrej J, et al.  Genome-wide association scan identifies a colorectal cancer susceptibility locus on chromosome 8q24. <span><span class="ref-journal">Nature Genetics. </span>2007;<span class="ref-vol">39</span>:989–994.</span> [<a href="/pubmed/17618283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+chromosome+8q24&amp;author=BW+Zanke&amp;author=CMT+Greenwood&amp;author=J+Rangrej&amp;volume=39&amp;publication_year=2007&amp;pages=989-994&amp;pmid=17618283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Tomlinson IP, Carvajal-Carmona LG, Dobbins SE, et al.  Multiple common susceptibility variants near BMP pathway loci GREM1, BMP4, and BMP2 explain part of the missing heritability of colorectal cancer. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002105.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3107194/">PMC free article</a>]</span> [<a href="/pubmed/21655089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Multiple+common+susceptibility+variants+near+BMP+pathway+loci+GREM1,+BMP4,+and+BMP2+explain+part+of+the+missing+heritability+of+colorectal+cancer&amp;author=IP+Tomlinson&amp;author=LG+Carvajal-Carmona&amp;author=SE+Dobbins&amp;volume=7&amp;publication_year=2011&amp;pages=e1002105&amp;pmid=21655089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Peters U, Bien S, Zubair N. Genetic architecture of colorectal cancer. <span><span class="ref-journal">Gut. </span>2015</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4567512/">PMC free article</a>]</span> [<a href="/pubmed/26187503" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Genetic+architecture+of+colorectal+cancer&amp;author=U+Peters&amp;author=S+Bien&amp;author=N+Zubair&amp;publication_year=2015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Li Y, Willer CJ, Ding J, et al.  MaCH: Using Sequence and Genotype Data to Estimate Haplotypes and Unobserved Genotypes. <span><span class="ref-journal">Genetic Epidemiology. </span>2010;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetic+Epidemiology&amp;title=MaCH:+Using+Sequence+and+Genotype+Data+to+Estimate+Haplotypes+and+Unobserved+Genotypes&amp;author=Y+Li&amp;author=CJ+Willer&amp;author=J+Ding&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: A tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">American Journal of Human Genetics. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=PLINK:+A+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Romanoski CE, Glass CK, Stunnenberg HG, et al.  Epigenomics: Roadmap for regulation. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">518</span>:314–6.</span> [<a href="/pubmed/25693562" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Epigenomics:+Roadmap+for+regulation&amp;author=CE+Romanoski&amp;author=CK+Glass&amp;author=HG+Stunnenberg&amp;volume=518&amp;publication_year=2015&amp;pages=314-6&amp;pmid=25693562&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Consortium EP. A user’s guide to the encyclopedia of DNA elements (ENCODE) <span><span class="ref-journal">PLoS Biol. </span>2011;<span class="ref-vol">9</span>:e1001046.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3079585/">PMC free article</a>]</span> [<a href="/pubmed/21526222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=A+user’s+guide+to+the+encyclopedia+of+DNA+elements+(ENCODE)&amp;volume=9&amp;publication_year=2011&amp;pages=e1001046&amp;pmid=21526222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Lizio M, Harshbarger J, Shimoji H, et al.  Gateways to the FANTOM5 promoter level mammalian expression atlas. <span><span class="ref-journal">Genome Biology. </span>2015;<span class="ref-vol">16</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4310165/">PMC free article</a>]</span> [<a href="/pubmed/25723102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biology&amp;title=Gateways+to+the+FANTOM5+promoter+level+mammalian+expression+atlas&amp;author=M+Lizio&amp;author=J+Harshbarger&amp;author=H+Shimoji&amp;volume=16&amp;publication_year=2015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Kumar P, Henikoff S, Ng PC. Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm. <span><span class="ref-journal">Nat Protoc. </span>2009;<span class="ref-vol">4</span>:1073–81.</span> [<a href="/pubmed/19561590" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Predicting+the+effects+of+coding+non-synonymous+variants+on+protein+function+using+the+SIFT+algorithm&amp;author=P+Kumar&amp;author=S+Henikoff&amp;author=PC+Ng&amp;volume=4&amp;publication_year=2009&amp;pages=1073-81&amp;pmid=19561590&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Adzhubei I, Jordan DM, Sunyaev SR. Predicting functional effect of human missense mutations using PolyPhen-2. <span><span class="ref-journal">Curr Protoc Hum Genet. </span>2013;<span class="ref-vol">Chapter 7</span>(Unit 7):20.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4480630/">PMC free article</a>]</span> [<a href="/pubmed/23315928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Hum+Genet&amp;title=Predicting+functional+effect+of+human+missense+mutations+using+PolyPhen-2&amp;author=I+Adzhubei&amp;author=DM+Jordan&amp;author=SR+Sunyaev&amp;volume=Chapter+7&amp;issue=Unit+7&amp;publication_year=2013&amp;pages=20&amp;pmid=23315928&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Carvajal-Carmona LG, Cazier JB, Jones AM, et al.  Fine-mapping of colorectal cancer susceptibility loci at 8q23.3, 16q22.1 and 19q13. 11: refinement of association signals and use of in silico analysis to suggest functional variation and unexpected candidate target genes. <span><span class="ref-journal">Human Molecular Genetics. </span>2011;<span class="ref-vol">20</span>:2879–2888.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3118761/">PMC free article</a>]</span> [<a href="/pubmed/21531788" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Molecular+Genetics&amp;title=Fine-mapping+of+colorectal+cancer+susceptibility+loci+at+8q23.3,+16q22.1+and+19q13.+11:+refinement+of+association+signals+and+use+of+in+silico+analysis+to+suggest+functional+variation+and+unexpected+candidate+target+genes&amp;author=LG+Carvajal-Carmona&amp;author=JB+Cazier&amp;author=AM+Jones&amp;volume=20&amp;publication_year=2011&amp;pages=2879-2888&amp;pmid=21531788&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Pittman AM, Naranjo S, Jalava SE, et al.  Allelic Variation at the 8q23.3 Colorectal Cancer Risk Locus Functions as a Cis-Acting Regulator of EIF3H. <span><span class="ref-journal">Plos Genetics. </span>2010;<span class="ref-vol">6</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2940760/">PMC free article</a>]</span> [<a href="/pubmed/20862326" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plos+Genetics&amp;title=Allelic+Variation+at+the+8q23.3+Colorectal+Cancer+Risk+Locus+Functions+as+a+Cis-Acting+Regulator+of+EIF3H&amp;author=AM+Pittman&amp;author=S+Naranjo&amp;author=SE+Jalava&amp;volume=6&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Burset M, Seledtsov IA, Solovyev VV. Analysis of canonical and non-canonical splice sites in mammalian genomes. <span><span class="ref-journal">Nucleic Acids Res. </span>2000;<span class="ref-vol">28</span>:4364–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC113136/">PMC free article</a>]</span> [<a href="/pubmed/11058137" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Analysis+of+canonical+and+non-canonical+splice+sites+in+mammalian+genomes&amp;author=M+Burset&amp;author=IA+Seledtsov&amp;author=VV+Solovyev&amp;volume=28&amp;publication_year=2000&amp;pages=4364-75&amp;pmid=11058137&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Andolfo I, De Falco L, Asci R, et al.  Regulation of divalent metal transporter 1 (DMT1) non-IRE isoform by the microRNA Let-7d in erythroid cells. <span><span class="ref-journal">Haematologica. </span>2010;<span class="ref-vol">95</span>:1244–52.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2913071/">PMC free article</a>]</span> [<a href="/pubmed/20410187" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Haematologica&amp;title=Regulation+of+divalent+metal+transporter+1+(DMT1)+non-IRE+isoform+by+the+microRNA+Let-7d+in+erythroid+cells&amp;author=I+Andolfo&amp;author=L+De+Falco&amp;author=R+Asci&amp;volume=95&amp;publication_year=2010&amp;pages=1244-52&amp;pmid=20410187&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Lin YS, Yasuda K, Assem M, et al.  The major human pregnane X receptor (PXR) splice variant, PXR. 2, exhibits significantly diminished ligand-activated transcriptional regulation. <span><span class="ref-journal">Drug Metab Dispos. </span>2009;<span class="ref-vol">37</span>:1295–304.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683691/">PMC free article</a>]</span> [<a href="/pubmed/19251824" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Metab+Dispos&amp;title=The+major+human+pregnane+X+receptor+(PXR)+splice+variant,+PXR.+2,+exhibits+significantly+diminished+ligand-activated+transcriptional+regulation&amp;author=YS+Lin&amp;author=K+Yasuda&amp;author=M+Assem&amp;volume=37&amp;publication_year=2009&amp;pages=1295-304&amp;pmid=19251824&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Zhang J, Manley JL. Misregulation of pre-mRNA alternative splicing in cancer. <span><span class="ref-journal">Cancer Discov. </span>2013;<span class="ref-vol">3</span>:1228–37.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3823817/">PMC free article</a>]</span> [<a href="/pubmed/24145039" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Discov&amp;title=Misregulation+of+pre-mRNA+alternative+splicing+in+cancer&amp;author=J+Zhang&amp;author=JL+Manley&amp;volume=3&amp;publication_year=2013&amp;pages=1228-37&amp;pmid=24145039&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Kile BT, Schulman BA, Alexander WS, et al.  The SOCS box: a tale of destruction and degradation. <span><span class="ref-journal">Trends Biochem Sci. </span>2002;<span class="ref-vol">27</span>:235–41.</span> [<a href="/pubmed/12076535" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Biochem+Sci&amp;title=The+SOCS+box:+a+tale+of+destruction+and+degradation&amp;author=BT+Kile&amp;author=BA+Schulman&amp;author=WS+Alexander&amp;volume=27&amp;publication_year=2002&amp;pages=235-41&amp;pmid=12076535&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Kuang Z, Lewis RS, Curtis JM, et al.  The SPRY domain-containing SOCS box protein SPSB2 targets iNOS for proteasomal degradation. <span><span class="ref-journal">J Cell Biol. </span>2010;<span class="ref-vol">190</span>:129–41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2911665/">PMC free article</a>]</span> [<a href="/pubmed/20603330" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol&amp;title=The+SPRY+domain-containing+SOCS+box+protein+SPSB2+targets+iNOS+for+proteasomal+degradation&amp;author=Z+Kuang&amp;author=RS+Lewis&amp;author=JM+Curtis&amp;volume=190&amp;publication_year=2010&amp;pages=129-41&amp;pmid=20603330&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Janakiram NB, Rao CV. iNOS-selective inhibitors for cancer prevention: promise and progress. <span><span class="ref-journal">Future Med Chem. </span>2012;<span class="ref-vol">4</span>:2193–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3588580/">PMC free article</a>]</span> [<a href="/pubmed/23190107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Future+Med+Chem&amp;title=iNOS-selective+inhibitors+for+cancer+prevention:+promise+and+progress&amp;author=NB+Janakiram&amp;author=CV+Rao&amp;volume=4&amp;publication_year=2012&amp;pages=2193-204&amp;pmid=23190107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Levy C, Khaled M, Fisher DE. MITF: master regulator of melanocyte development and melanoma oncogene. <span><span class="ref-journal">Trends Mol Med. </span>2006;<span class="ref-vol">12</span>:406–14.</span> [<a href="/pubmed/16899407" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Mol+Med&amp;title=MITF:+master+regulator+of+melanocyte+development+and+melanoma+oncogene&amp;author=C+Levy&amp;author=M+Khaled&amp;author=DE+Fisher&amp;volume=12&amp;publication_year=2006&amp;pages=406-14&amp;pmid=16899407&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Kuiper RP, Schepens M, Thijssen J, et al.  Upregulation of the transcription factor TFEB in t(6;11)(p21;q13)-positive renal cell carcinomas due to promoter substitution. <span><span class="ref-journal">Hum Mol Genet. </span>2003;<span class="ref-vol">12</span>:1661–9.</span> [<a href="/pubmed/12837690" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Upregulation+of+the+transcription+factor+TFEB+in+t(6;11)(p21;q13)-positive+renal+cell+carcinomas+due+to+promoter+substitution&amp;author=RP+Kuiper&amp;author=M+Schepens&amp;author=J+Thijssen&amp;volume=12&amp;publication_year=2003&amp;pages=1661-9&amp;pmid=12837690&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Davis IJ, Kim JJ, Ozsolak F, et al.  Oncogenic MITF dysregulation in clear cell sarcoma: defining the MiT family of human cancers. <span><span class="ref-journal">Cancer Cell. </span>2006;<span class="ref-vol">9</span>:473–84.</span> [<a href="/pubmed/16766266" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=Oncogenic+MITF+dysregulation+in+clear+cell+sarcoma:+defining+the+MiT+family+of+human+cancers&amp;author=IJ+Davis&amp;author=JJ+Kim&amp;author=F+Ozsolak&amp;volume=9&amp;publication_year=2006&amp;pages=473-84&amp;pmid=16766266&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Settembre C, De Cegli R, Mansueto G, et al.  TFEB controls cellular lipid metabolism through a starvation-induced autoregulatory loop. <span><span class="ref-journal">Nat Cell Biol. </span>2013;<span class="ref-vol">15</span>:647–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3699877/">PMC free article</a>]</span> [<a href="/pubmed/23604321" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Cell+Biol&amp;title=TFEB+controls+cellular+lipid+metabolism+through+a+starvation-induced+autoregulatory+loop&amp;author=C+Settembre&amp;author=R+De+Cegli&amp;author=G+Mansueto&amp;volume=15&amp;publication_year=2013&amp;pages=647-58&amp;pmid=23604321&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">White E. Deconvoluting the context-dependent role for autophagy in cancer. <span><span class="ref-journal">Nat Rev Cancer. </span>2012;<span class="ref-vol">12</span>:401–10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3664381/">PMC free article</a>]</span> [<a href="/pubmed/22534666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Deconvoluting+the+context-dependent+role+for+autophagy+in+cancer&amp;author=E+White&amp;volume=12&amp;publication_year=2012&amp;pages=401-10&amp;pmid=22534666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Pena-Llopis S, Vega-Rubin-de-Celis S, Schwartz JC, et al.  Regulation of TFEB and V-ATPases by mTORC1. <span><span class="ref-journal">Embo Journal. </span>2011;<span class="ref-vol">30</span>:3242–3258.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3160667/">PMC free article</a>]</span> [<a href="/pubmed/21804531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Embo+Journal&amp;title=Regulation+of+TFEB+and+V-ATPases+by+mTORC1&amp;author=S+Pena-Llopis&amp;author=S+Vega-Rubin-de-Celis&amp;author=JC+Schwartz&amp;volume=30&amp;publication_year=2011&amp;pages=3242-3258&amp;pmid=21804531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Faller WJ, Jackson TJ, Knight JR, et al.  mTORC1-mediated translational elongation limits intestinal tumour initiation and growth. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">517</span>:497–500.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4304784/">PMC free article</a>]</span> [<a href="/pubmed/25383520" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=mTORC1-mediated+translational+elongation+limits+intestinal+tumour+initiation+and+growth&amp;author=WJ+Faller&amp;author=TJ+Jackson&amp;author=JR+Knight&amp;volume=517&amp;publication_year=2015&amp;pages=497-500&amp;pmid=25383520&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Lin JH, Giovannucci E. Sex hormones and colorectal cancer: what have we learned so far? <span><span class="ref-journal">J Natl Cancer Inst. </span>2010;<span class="ref-vol">102</span>:1746–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2994865/">PMC free article</a>]</span> [<a href="/pubmed/21068431" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Sex+hormones+and+colorectal+cancer:+what+have+we+learned+so+far?&amp;author=JH+Lin&amp;author=E+Giovannucci&amp;volume=102&amp;publication_year=2010&amp;pages=1746-7&amp;pmid=21068431&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Xu J, Lin X, Zhu H, et al.  Genetic variation of the CYP17 and susceptibility to endometrial cancer: a meta-analysis. <span><span class="ref-journal">Mol Biol Rep. </span>2013;<span class="ref-vol">40</span>:5085–91.</span> [<a href="/pubmed/23649771" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Rep&amp;title=Genetic+variation+of+the+CYP17+and+susceptibility+to+endometrial+cancer:+a+meta-analysis&amp;author=J+Xu&amp;author=X+Lin&amp;author=H+Zhu&amp;volume=40&amp;publication_year=2013&amp;pages=5085-91&amp;pmid=23649771&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Yao L, Fang F, Wu Q, et al.  No association between CYP17 T-34C polymorphism and breast cancer risk: a meta-analysis involving 58,814 subjects. <span><span class="ref-journal">Breast Cancer Res Treat. </span>2010;<span class="ref-vol">122</span>:221–7.</span> [<a href="/pubmed/20013047" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Breast+Cancer+Res+Treat&amp;title=No+association+between+CYP17+T-34C+polymorphism+and+breast+cancer+risk:+a+meta-analysis+involving+58,814+subjects&amp;author=L+Yao&amp;author=F+Fang&amp;author=Q+Wu&amp;volume=122&amp;publication_year=2010&amp;pages=221-7&amp;pmid=20013047&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Zhang L, Pan X, Hershey JW. Individual overexpression of five subunits of human translation initiation factor eIF3 promotes malignant transformation of immortal fibroblast cells. <span><span class="ref-journal">J Biol Chem. </span>2007;<span class="ref-vol">282</span>:5790–800.</span> [<a href="/pubmed/17170115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Individual+overexpression+of+five+subunits+of+human+translation+initiation+factor+eIF3+promotes+malignant+transformation+of+immortal+fibroblast+cells&amp;author=L+Zhang&amp;author=X+Pan&amp;author=JW+Hershey&amp;volume=282&amp;publication_year=2007&amp;pages=5790-800&amp;pmid=17170115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Mahmood SF, Gruel N, Chapeaublanc E, et al.  A siRNA screen identifies RAD21, EIF3H, CHRAC1 and TANC2 as driver genes within the 8q23, 8q24. 3 and 17q23 amplicons in breast cancer with effects on cell growth, survival and transformation. <span><span class="ref-journal">Carcinogenesis. </span>2014;<span class="ref-vol">35</span>:670–82.</span> [<a href="/pubmed/24148822" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=A+siRNA+screen+identifies+RAD21,+EIF3H,+CHRAC1+and+TANC2+as+driver+genes+within+the+8q23,+8q24.+3+and+17q23+amplicons+in+breast+cancer+with+effects+on+cell+growth,+survival+and+transformation&amp;author=SF+Mahmood&amp;author=N+Gruel&amp;author=E+Chapeaublanc&amp;volume=35&amp;publication_year=2014&amp;pages=670-82&amp;pmid=24148822&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Xu H, Yan Y, Deb S, et al.  Cohesin Rad21 mediates loss of heterozygosity and is upregulated via Wnt promoting transcriptional dysregulation in gastrointestinal tumors. <span><span class="ref-journal">Cell Rep. </span>2014;<span class="ref-vol">9</span>:1781– 97.</span> [<a href="/pubmed/25464844" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep&amp;title=Cohesin+Rad21+mediates+loss+of+heterozygosity+and+is+upregulated+via+Wnt+promoting+transcriptional+dysregulation+in+gastrointestinal+tumors&amp;author=H+Xu&amp;author=Y+Yan&amp;author=S+Deb&amp;volume=9&amp;publication_year=2014&amp;pages=1781-+97&amp;pmid=25464844&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Ongen H, Andersen CL, Bramsen JB, et al.  Putative cis-regulatory drivers in colorectal cancer. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">512</span>:87–90.</span> [<a href="/pubmed/25079323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Putative+cis-regulatory+drivers+in+colorectal+cancer&amp;author=H+Ongen&amp;author=CL+Andersen&amp;author=JB+Bramsen&amp;volume=512&amp;publication_year=2014&amp;pages=87-90&amp;pmid=25079323&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Schwanhausser B, Busse D, Li N, et al.  Global quantification of mammalian gene expression control. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">473</span>:337–42.</span> [<a href="/pubmed/21593866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Global+quantification+of+mammalian+gene+expression+control&amp;author=B+Schwanhausser&amp;author=D+Busse&amp;author=N+Li&amp;volume=473&amp;publication_year=2011&amp;pages=337-42&amp;pmid=21593866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Jovanovic M, Rooney MS, Mertins P, et al.  Immunogenetics Dynamic profiling of the protein life cycle in response to pathogens. <span><span class="ref-journal">Science. </span>2015;<span class="ref-vol">347</span>:1259038.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4506746/">PMC free article</a>]</span> [<a href="/pubmed/25745177" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Immunogenetics+Dynamic+profiling+of+the+protein+life+cycle+in+response+to+pathogens&amp;author=M+Jovanovic&amp;author=MS+Rooney&amp;author=P+Mertins&amp;volume=347&amp;publication_year=2015&amp;pages=1259038&amp;pmid=25745177&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">White E. The role for autophagy in cancer. <span><span class="ref-journal">Journal of Clinical Investigation. </span>2015;<span class="ref-vol">125</span>:42–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4382247/">PMC free article</a>]</span> [<a href="/pubmed/25654549" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Clinical+Investigation&amp;title=The+role+for+autophagy+in+cancer&amp;author=E+White&amp;volume=125&amp;publication_year=2015&amp;pages=42-46&amp;pmid=25654549&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Levine B, Packer M, Codogno P. Development of autophagy inducers in clinical medicine. <span><span class="ref-journal">J Clin Invest. </span>2015;<span class="ref-vol">125</span>:14–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4382267/">PMC free article</a>]</span> [<a href="/pubmed/25654546" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Development+of+autophagy+inducers+in+clinical+medicine&amp;author=B+Levine&amp;author=M+Packer&amp;author=P+Codogno&amp;volume=125&amp;publication_year=2015&amp;pages=14-24&amp;pmid=25654546&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Kroemer G. Autophagy: a druggable process that is deregulated in aging and human disease. <span><span class="ref-journal">Journal of Clinical Investigation. </span>2015;<span class="ref-vol">125</span>:1–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4382251/">PMC free article</a>]</span> [<a href="/pubmed/25654544" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Clinical+Investigation&amp;title=Autophagy:+a+druggable+process+that+is+deregulated+in+aging+and+human+disease&amp;author=G+Kroemer&amp;volume=125&amp;publication_year=2015&amp;pages=1-4&amp;pmid=25654544&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Pruim RJ, Welch RP, Sanna S, et al.  LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;volume=26&amp;publication_year=2010&amp;pages=2336-7&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4909543&amp;issue-id=271013&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139959251479824title">Abstract</a></li><li><a href="#S5title">INTRODUCTION</a></li><li><a href="#S6title">Materials and Methods</a></li><li><a href="#S13title">Results</a></li><li><a href="#S14title">Discussion</a></li><li><a href="#S15title">Supplementary Material</a></li><li><a href="#S16title">Acknowledgments</a></li><li><a href="#GLtitle">Abbreviations</a></li><li><a href="#idm139959251106320title">Footnotes</a></li><li><a href="#idm139959292762448title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4909543/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4909543/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4909543/pdf/nihms766698.pdf">PDF (754K)</a></li> | <li><a href="#" data-citationid="PMC4909543" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4909543%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4909543%2F&amp;text=Identification%20of%20Susceptibility%20Loci%20and%20Genes%20for%20Colorectal%20Cancer%20Risk" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4909543%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26965516" data-aiid="4909543" data-aid="4909543" data-iid="271013" data-domainid="319" data-domain="nihpa" data-accid="PMC4909543" data-md5="f6bc73df9cc8f0f36c01c39ddf27a0d0"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:30:06-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8DA629DD59210100000000015600DC&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4909543%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4909543/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>