<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4656791/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies variants at 16p13 associated with survival in multiple myeloma patients</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4656791/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature communications"><meta name="citation_title" content="Genome-wide association study identifies variants at 16p13 associated with survival in multiple myeloma patients"><meta name="citation_authors" content="Elad Ziv, Eric Dean, Donglei Hu, Alessandro Martino, Daniel Serie, Karen Curtin, Daniele Campa, Blake Aftab, Paige Bracci, Gabriele Buda, Yi Zhao, Jennifer Caswell-Jin, Robert Diasio, Charles Dumontet, Marek Dudziński, Laura Fejerman, Alexandra Greenberg, Scott Huntsman, Krzysztof Jamroziak, Artur Jurczyszyn, Shaji Kumar, Djordje Atanackovic, Martha Glenn, Lisa A. Cannon-Albright, Brandt Jones, Adam Lee, Herlander Marques, Thomas Martin, Joaquin Martinez-Lopez, Vincent Rajkumar, Juan Sainz, Annette Juul Vangsted, Marzena Wątek, Jeffrey Wolf, Susan Slager, Nicola J Camp, Federico Canzian, Celine Vachon"><meta name="citation_date" content="2015"><meta name="citation_volume" content="6"><meta name="citation_firstpage" content="7539"><meta name="citation_doi" content="10.1038/ncomms8539"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4656791/?report=abstract"><meta name="citation_pmid" content="26198393"><meta name="DC.Title" content="Genome-wide association study identifies variants at 16p13 associated with survival in multiple myeloma patients"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Elad Ziv"><meta name="DC.Contributor" content="Eric Dean"><meta name="DC.Contributor" content="Donglei Hu"><meta name="DC.Contributor" content="Alessandro Martino"><meta name="DC.Contributor" content="Daniel Serie"><meta name="DC.Contributor" content="Karen Curtin"><meta name="DC.Contributor" content="Daniele Campa"><meta name="DC.Contributor" content="Blake Aftab"><meta name="DC.Contributor" content="Paige Bracci"><meta name="DC.Contributor" content="Gabriele Buda"><meta name="DC.Contributor" content="Yi Zhao"><meta name="DC.Contributor" content="Jennifer Caswell-Jin"><meta name="DC.Contributor" content="Robert Diasio"><meta name="DC.Contributor" content="Charles Dumontet"><meta name="DC.Contributor" content="Marek Dudziński"><meta name="DC.Contributor" content="Laura Fejerman"><meta name="DC.Contributor" content="Alexandra Greenberg"><meta name="DC.Contributor" content="Scott Huntsman"><meta name="DC.Contributor" content="Krzysztof Jamroziak"><meta name="DC.Contributor" content="Artur Jurczyszyn"><meta name="DC.Contributor" content="Shaji Kumar"><meta name="DC.Contributor" content="Djordje Atanackovic"><meta name="DC.Contributor" content="Martha Glenn"><meta name="DC.Contributor" content="Lisa A. Cannon-Albright"><meta name="DC.Contributor" content="Brandt Jones"><meta name="DC.Contributor" content="Adam Lee"><meta name="DC.Contributor" content="Herlander Marques"><meta name="DC.Contributor" content="Thomas Martin"><meta name="DC.Contributor" content="Joaquin Martinez-Lopez"><meta name="DC.Contributor" content="Vincent Rajkumar"><meta name="DC.Contributor" content="Juan Sainz"><meta name="DC.Contributor" content="Annette Juul Vangsted"><meta name="DC.Contributor" content="Marzena Wątek"><meta name="DC.Contributor" content="Jeffrey Wolf"><meta name="DC.Contributor" content="Susan Slager"><meta name="DC.Contributor" content="Nicola J Camp"><meta name="DC.Contributor" content="Federico Canzian"><meta name="DC.Contributor" content="Celine Vachon"><meta name="DC.Date" content="2015"><meta name="DC.Identifier" content="10.1038/ncomms8539"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies variants at 16p13 associated with survival in multiple myeloma patients"><meta property="og:type" content="article"><meta property="og:description" content="Here we perform the first genome wide association study (GWAS) of multiple myeloma (MM) survival. In a meta-analysis of 306 MM patients treated at UCSF and 239 patients treated at the Mayo clinic, we find a significant association between SNPs near the ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4656791/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4656791/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4656791/pdf/nihms-730494.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CA341DD5866A100000000034401D4.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4656791%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4656791</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Commun</a></span>. Author manuscript; available in PMC 2016 Jan 22.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26198393" target="pmc_ext" ref="reftype=publisher&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Commun. 2015; 6: 7539. </span></a></div><span class="fm-vol-iss-date">Published online 2015 Jul 22. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fncomms8539" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ncomms8539</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4656791</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS730494</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26198393">26198393</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies variants at 16p13 associated with survival in multiple myeloma patients</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Ziv%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292072048592" co-class="co-affbox">Elad Ziv</a>,<sup>#</sup><sup>1,</sup><sup>#</sup> <a href="/pubmed/?term=Dean%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292067927712" co-class="co-affbox">Eric Dean</a>,<sup>#</sup><sup>2</sup> <a href="/pubmed/?term=Hu%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292067925488" co-class="co-affbox">Donglei Hu</a>,<sup>1</sup> <a href="/pubmed/?term=Martino%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292072045696" co-class="co-affbox">Alessandro Martino</a>,<sup>3</sup> <a href="/pubmed/?term=Serie%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292072043712" co-class="co-affbox">Daniel Serie</a>,<sup>4</sup> <a href="/pubmed/?term=Curtin%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292106498720" co-class="co-affbox">Karen Curtin</a>,<sup>22</sup> <a href="/pubmed/?term=Campa%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292106496736" co-class="co-affbox">Daniele Campa</a>,<sup>3</sup> <a href="/pubmed/?term=Aftab%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068265072" co-class="co-affbox">Blake Aftab</a>,<sup>5</sup> <a href="/pubmed/?term=Bracci%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068263088" co-class="co-affbox">Paige Bracci</a>,<sup>6</sup> <a href="/pubmed/?term=Buda%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068261104" co-class="co-affbox">Gabriele Buda</a>,<sup>7</sup> <a href="/pubmed/?term=Zhao%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292066927344" co-class="co-affbox">Yi Zhao</a>,<sup>22</sup> <a href="/pubmed/?term=Caswell-Jin%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292066925360" co-class="co-affbox">Jennifer Caswell-Jin</a>,<sup>1</sup> <a href="/pubmed/?term=Diasio%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292073928480" co-class="co-affbox">Robert Diasio</a>,<sup>8</sup> <a href="/pubmed/?term=Dumontet%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292073926496" co-class="co-affbox">Charles Dumontet</a>,<sup>9</sup> <a href="/pubmed/?term=Dudzi%26%23x00144%3Bski%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292070417136" co-class="co-affbox">Marek Dudziński</a>,<sup>10</sup> <a href="/pubmed/?term=Fejerman%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292070415152" co-class="co-affbox">Laura Fejerman</a>,<sup>1</sup> <a href="/pubmed/?term=Greenberg%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292072144976" co-class="co-affbox">Alexandra Greenberg</a>,<sup>11</sup> <a href="/pubmed/?term=Huntsman%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292072142992" co-class="co-affbox">Scott Huntsman</a>,<sup>1</sup> <a href="/pubmed/?term=Jamroziak%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292065151504" co-class="co-affbox">Krzysztof Jamroziak</a>,<sup>12</sup> <a href="/pubmed/?term=Jurczyszyn%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292065149520" co-class="co-affbox">Artur Jurczyszyn</a>,<sup>13</sup> <a href="/pubmed/?term=Kumar%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068950544" co-class="co-affbox">Shaji Kumar</a>,<sup>14</sup> <a href="/pubmed/?term=Atanackovic%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068948560" co-class="co-affbox">Djordje Atanackovic</a>,<sup>22</sup> <a href="/pubmed/?term=Glenn%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292065825648" co-class="co-affbox">Martha Glenn</a>,<sup>22</sup> <a href="/pubmed/?term=Cannon-Albright%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292065823664" co-class="co-affbox">Lisa A. Cannon-Albright</a>,<sup>22</sup> <a href="/pubmed/?term=Jones%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292064199552" co-class="co-affbox">Brandt Jones</a>,<sup>22</sup> <a href="/pubmed/?term=Lee%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292064197568" co-class="co-affbox">Adam Lee</a>,<sup>8</sup> <a href="/pubmed/?term=Marques%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292064195584" co-class="co-affbox">Herlander Marques</a>,<sup>15</sup> <a href="/pubmed/?term=Martin%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068056192" co-class="co-affbox">Thomas Martin</a>,<sup>5</sup> <a href="/pubmed/?term=Martinez-Lopez%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068054208" co-class="co-affbox">Joaquin Martinez-Lopez</a>,<sup>16</sup> <a href="/pubmed/?term=Rajkumar%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068602432" co-class="co-affbox">Vincent Rajkumar</a>,<sup>14</sup> <a href="/pubmed/?term=Sainz%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068600448" co-class="co-affbox">Juan Sainz</a>,<sup>17</sup> <a href="/pubmed/?term=Vangsted%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292072639616" co-class="co-affbox">Annette Juul Vangsted</a>,<sup>18</sup> <a href="/pubmed/?term=W%26%23x00105%3Btek%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292072637632" co-class="co-affbox">Marzena Wątek</a>,<sup>19</sup> <a href="/pubmed/?term=Wolf%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292070421808" co-class="co-affbox">Jeffrey Wolf</a>,<sup>5</sup> <a href="/pubmed/?term=Slager%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292070419824" co-class="co-affbox">Susan Slager</a>,<sup>20</sup> <a href="/pubmed/?term=Camp%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068272768" co-class="co-affbox">Nicola J Camp</a>,<sup>22,</sup><sup>&amp;</sup> <a href="/pubmed/?term=Canzian%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292068270048" co-class="co-affbox">Federico Canzian</a>,<sup>3,</sup><sup>&amp;</sup> and  <a href="/pubmed/?term=Vachon%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393" class="affpopup" co-rid="_co_idm140292070346672" co-class="co-affbox">Celine Vachon</a><sup>21,</sup><sup>&amp;</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140292072048592"><h3 class="no_margin">Elad Ziv</h3><p><sup>1</sup>Department of Medicine, Division of General Internal Medicine, Institute for Human Genetics, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</p><div>Find articles by <a href="/pubmed/?term=Ziv%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Elad Ziv</a></div></div><div id="_co_idm140292067927712"><h3 class="no_margin">Eric Dean</h3><p><sup>2</sup>Sutter Medical Center of Santa Rosa, Santa Rosa, California</p><div>Find articles by <a href="/pubmed/?term=Dean%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Eric Dean</a></div></div><div id="_co_idm140292067925488"><h3 class="no_margin">Donglei Hu</h3><p><sup>1</sup>Department of Medicine, Division of General Internal Medicine, Institute for Human Genetics, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</p><div>Find articles by <a href="/pubmed/?term=Hu%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Donglei Hu</a></div></div><div id="_co_idm140292072045696"><h3 class="no_margin">Alessandro Martino</h3><p><sup>3</sup>Genomic Epidemiology Group, German Cancer Research Center (DKFZ), Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Martino%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Alessandro Martino</a></div></div><div id="_co_idm140292072043712"><h3 class="no_margin">Daniel Serie</h3><p><sup>4</sup>Division of Biomedical Statistics and Informatics. Department of Health Sciences Research, Mayo Clinic, Jacksonville, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Serie%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Daniel Serie</a></div></div><div id="_co_idm140292106498720"><h3 class="no_margin">Karen Curtin</h3><p><sup>22</sup>Department of Internal Medicine, University of Utah School of Medicine, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Curtin%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Karen Curtin</a></div></div><div id="_co_idm140292106496736"><h3 class="no_margin">Daniele Campa</h3><p><sup>3</sup>Genomic Epidemiology Group, German Cancer Research Center (DKFZ), Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Campa%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Daniele Campa</a></div></div><div id="_co_idm140292068265072"><h3 class="no_margin">Blake Aftab</h3><p><sup>5</sup>Department of Medicine, Division of Hematology, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</p><div>Find articles by <a href="/pubmed/?term=Aftab%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Blake Aftab</a></div></div><div id="_co_idm140292068263088"><h3 class="no_margin">Paige Bracci</h3><p><sup>6</sup>Department of Epidemiology and Biostatistics, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</p><div>Find articles by <a href="/pubmed/?term=Bracci%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Paige Bracci</a></div></div><div id="_co_idm140292068261104"><h3 class="no_margin">Gabriele Buda</h3><p><sup>7</sup>Department of Oncology, Transplants and Advanced Technologies, Section of Hematology, Pisa University Hospital, Pisa, Italy</p><div>Find articles by <a href="/pubmed/?term=Buda%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Gabriele Buda</a></div></div><div id="_co_idm140292066927344"><h3 class="no_margin">Yi Zhao</h3><p><sup>22</sup>Department of Internal Medicine, University of Utah School of Medicine, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Zhao%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Yi Zhao</a></div></div><div id="_co_idm140292066925360"><h3 class="no_margin">Jennifer Caswell-Jin</h3><p><sup>1</sup>Department of Medicine, Division of General Internal Medicine, Institute for Human Genetics, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</p><div>Find articles by <a href="/pubmed/?term=Caswell-Jin%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Jennifer Caswell-Jin</a></div></div><div id="_co_idm140292073928480"><h3 class="no_margin">Robert Diasio</h3><p><sup>8</sup>Department of Molecular Pharmacology and Experimental Therapeutics, College of Medicine, Mayo Clinic, Rochester, MN</p><div>Find articles by <a href="/pubmed/?term=Diasio%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Robert Diasio</a></div></div><div id="_co_idm140292073926496"><h3 class="no_margin">Charles Dumontet</h3><p><sup>9</sup>INSERM UMR 1052 / CNRS 5286, Laboratoire de Cytologie Analytique, Faculté de Médecine Rockefeller, Université Claude Bernard Lyon I, Lyon, France</p><div>Find articles by <a href="/pubmed/?term=Dumontet%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Charles Dumontet</a></div></div><div id="_co_idm140292070417136"><h3 class="no_margin">Marek Dudziński</h3><p><sup>10</sup>Department of Hematology, Rzeszow Regional Hospital, Rzeszow, Poland</p><div>Find articles by <a href="/pubmed/?term=Dudzi%26%23x00144%3Bski%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Marek Dudziński</a></div></div><div id="_co_idm140292070415152"><h3 class="no_margin">Laura Fejerman</h3><p><sup>1</sup>Department of Medicine, Division of General Internal Medicine, Institute for Human Genetics, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</p><div>Find articles by <a href="/pubmed/?term=Fejerman%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Laura Fejerman</a></div></div><div id="_co_idm140292072144976"><h3 class="no_margin">Alexandra Greenberg</h3><p><sup>11</sup>Center for Translational Science Activities, Mayo Clinic, Rochester, MN, USA Division of Epidemiology, Department of Health Sciences Research, Mayo Clinic, Rochester, USA</p><div>Find articles by <a href="/pubmed/?term=Greenberg%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Alexandra Greenberg</a></div></div><div id="_co_idm140292072142992"><h3 class="no_margin">Scott Huntsman</h3><p><sup>1</sup>Department of Medicine, Division of General Internal Medicine, Institute for Human Genetics, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</p><div>Find articles by <a href="/pubmed/?term=Huntsman%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Scott Huntsman</a></div></div><div id="_co_idm140292065151504"><h3 class="no_margin">Krzysztof Jamroziak</h3><p><sup>12</sup>Department of Hematology, Medical University of Lodz, Lodz, Poland</p><div>Find articles by <a href="/pubmed/?term=Jamroziak%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Krzysztof Jamroziak</a></div></div><div id="_co_idm140292065149520"><h3 class="no_margin">Artur Jurczyszyn</h3><p><sup>13</sup>Department of Hematology, Cracow University Hospital, Cracow, Poland</p><div>Find articles by <a href="/pubmed/?term=Jurczyszyn%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Artur Jurczyszyn</a></div></div><div id="_co_idm140292068950544"><h3 class="no_margin">Shaji Kumar</h3><p><sup>14</sup>Division of Hematology, Department of Internal Medicine, Mayo Clinic, Rochester, MN, USA</p><div>Find articles by <a href="/pubmed/?term=Kumar%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Shaji Kumar</a></div></div><div id="_co_idm140292068948560"><h3 class="no_margin">Djordje Atanackovic</h3><p><sup>22</sup>Department of Internal Medicine, University of Utah School of Medicine, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Atanackovic%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Djordje Atanackovic</a></div></div><div id="_co_idm140292065825648"><h3 class="no_margin">Martha Glenn</h3><p><sup>22</sup>Department of Internal Medicine, University of Utah School of Medicine, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Glenn%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Martha Glenn</a></div></div><div id="_co_idm140292065823664"><h3 class="no_margin">Lisa A. Cannon-Albright</h3><p><sup>22</sup>Department of Internal Medicine, University of Utah School of Medicine, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Cannon-Albright%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Lisa A. Cannon-Albright</a></div></div><div id="_co_idm140292064199552"><h3 class="no_margin">Brandt Jones</h3><p><sup>22</sup>Department of Internal Medicine, University of Utah School of Medicine, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Jones%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Brandt Jones</a></div></div><div id="_co_idm140292064197568"><h3 class="no_margin">Adam Lee</h3><p><sup>8</sup>Department of Molecular Pharmacology and Experimental Therapeutics, College of Medicine, Mayo Clinic, Rochester, MN</p><div>Find articles by <a href="/pubmed/?term=Lee%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Adam Lee</a></div></div><div id="_co_idm140292064195584"><h3 class="no_margin">Herlander Marques</h3><p><sup>15</sup>Life and Health Sciences Research Institute (ICVS), School of Health Sciences, University of Minho, Braga, Portugal</p><div>Find articles by <a href="/pubmed/?term=Marques%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Herlander Marques</a></div></div><div id="_co_idm140292068056192"><h3 class="no_margin">Thomas Martin</h3><p><sup>5</sup>Department of Medicine, Division of Hematology, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</p><div>Find articles by <a href="/pubmed/?term=Martin%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Thomas Martin</a></div></div><div id="_co_idm140292068054208"><h3 class="no_margin">Joaquin Martinez-Lopez</h3><p><sup>16</sup>Hematology Service, CRIS facility for Hematological research, Hospital Universitario 12 de Octubre, Universidad Complutense, Madrid, Spain</p><div>Find articles by <a href="/pubmed/?term=Martinez-Lopez%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Joaquin Martinez-Lopez</a></div></div><div id="_co_idm140292068602432"><h3 class="no_margin">Vincent Rajkumar</h3><p><sup>14</sup>Division of Hematology, Department of Internal Medicine, Mayo Clinic, Rochester, MN, USA</p><div>Find articles by <a href="/pubmed/?term=Rajkumar%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Vincent Rajkumar</a></div></div><div id="_co_idm140292068600448"><h3 class="no_margin">Juan Sainz</h3><p><sup>17</sup>Genomic Oncology Area, GENYO, Centre for Genomics and Oncological Research: Pfizer / University of Granada / Andalusian Regional Government, Granada, Spain; Hematology department, Virgen de las Nieves University Hospital, Granada, Spain</p><div>Find articles by <a href="/pubmed/?term=Sainz%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Juan Sainz</a></div></div><div id="_co_idm140292072639616"><h3 class="no_margin">Annette Juul Vangsted</h3><p><sup>18</sup>Department of Hematology, Righospitalet and Roskilde Hospital, Copenhagen University, Denmark</p><div>Find articles by <a href="/pubmed/?term=Vangsted%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Annette Juul Vangsted</a></div></div><div id="_co_idm140292072637632"><h3 class="no_margin">Marzena Wątek</h3><p><sup>19</sup>Department of Hematology, Holycross Cancer Center, Kielce, Poland</p><div>Find articles by <a href="/pubmed/?term=W%26%23x00105%3Btek%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Marzena Wątek</a></div></div><div id="_co_idm140292070421808"><h3 class="no_margin">Jeffrey Wolf</h3><p><sup>5</sup>Department of Medicine, Division of Hematology, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</p><div>Find articles by <a href="/pubmed/?term=Wolf%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Jeffrey Wolf</a></div></div><div id="_co_idm140292070419824"><h3 class="no_margin">Susan Slager</h3><p><sup>20</sup>Division of Biomedical Statistics and Informatics. Department of Health Sciences Research, Mayo Clinic College of Medicine, Rochester, MN, USA</p><div>Find articles by <a href="/pubmed/?term=Slager%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Susan Slager</a></div></div><div id="_co_idm140292068272768"><h3 class="no_margin">Nicola J Camp</h3><p><sup>22</sup>Department of Internal Medicine, University of Utah School of Medicine, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Camp%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Nicola J Camp</a></div></div><div id="_co_idm140292068270048"><h3 class="no_margin">Federico Canzian</h3><p><sup>3</sup>Genomic Epidemiology Group, German Cancer Research Center (DKFZ), Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Canzian%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Federico Canzian</a></div></div><div id="_co_idm140292070346672"><h3 class="no_margin">Celine Vachon</h3><p><sup>21</sup>Division of Epidemiology, Department of Health Sciences Research, Mayo Clinic College of Medicine Rochester, MN, USA</p><div>Find articles by <a href="/pubmed/?term=Vachon%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26198393">Celine Vachon</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140292066286128_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140292066286128_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140292066286128_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Medicine, Division of General Internal Medicine, Institute for Human Genetics, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Sutter Medical Center of Santa Rosa, Santa Rosa, California</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Genomic Epidemiology Group, German Cancer Research Center (DKFZ), Heidelberg, Germany</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Division of Biomedical Statistics and Informatics. Department of Health Sciences Research, Mayo Clinic, Jacksonville, FL, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Medicine, Division of Hematology, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Epidemiology and Biostatistics, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, California</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Oncology, Transplants and Advanced Technologies, Section of Hematology, Pisa University Hospital, Pisa, Italy</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Molecular Pharmacology and Experimental Therapeutics, College of Medicine, Mayo Clinic, Rochester, MN</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>INSERM UMR 1052 / CNRS 5286, Laboratoire de Cytologie Analytique, Faculté de Médecine Rockefeller, Université Claude Bernard Lyon I, Lyon, France</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Hematology, Rzeszow Regional Hospital, Rzeszow, Poland</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Center for Translational Science Activities, Mayo Clinic, Rochester, MN, USA Division of Epidemiology, Department of Health Sciences Research, Mayo Clinic, Rochester, USA</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Hematology, Medical University of Lodz, Lodz, Poland</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Hematology, Cracow University Hospital, Cracow, Poland</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Division of Hematology, Department of Internal Medicine, Mayo Clinic, Rochester, MN, USA</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Life and Health Sciences Research Institute (ICVS), School of Health Sciences, University of Minho, Braga, Portugal</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Hematology Service, CRIS facility for Hematological research, Hospital Universitario 12 de Octubre, Universidad Complutense, Madrid, Spain</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Genomic Oncology Area, GENYO, Centre for Genomics and Oncological Research: Pfizer / University of Granada / Andalusian Regional Government, Granada, Spain; Hematology department, Virgen de las Nieves University Hospital, Granada, Spain</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Department of Hematology, Righospitalet and Roskilde Hospital, Copenhagen University, Denmark</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Hematology, Holycross Cancer Center, Kielce, Poland</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Division of Biomedical Statistics and Informatics. Department of Health Sciences Research, Mayo Clinic College of Medicine, Rochester, MN, USA</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Division of Epidemiology, Department of Health Sciences Research, Mayo Clinic College of Medicine Rochester, MN, USA</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Internal Medicine, University of Utah School of Medicine, Salt Lake City, UT, USA</div><div><sup>#</sup>Contributed equally.</div><div id="FN1"><sup>&amp;</sup>Authors jointly supervised this work.</div><div id="CR1"><sup>#</sup>Corresponding author: <a href="mailto:dev@null" data-email="ude.fscu@viz.dale" class="oemail">ude.fscu@viz.dale</a>; Helen Diller Family Comprehensive Cancer Center, 1450 3rd St. HD286, San Francisco, CA 94143</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140292066286128_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26198393" target="pmc_ext" ref="reftype=publisher&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Commun</a></div><div><strong>This article has been corrected. </strong>See <a href="/pmc/articles/PMC5729160/">Nat Commun. 2015 December 09; 6: 10203</a>.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4656791/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Fig 1-2 and Table 1-11.</strong></div><div><a href="/pmc/articles/PMC4656791/bin/NIHMS730494-supplement-Supplementary_Fig_1-2_and_Table_1-11.docx" data-ga-action="click_feat_suppl">NIHMS730494-supplement-Supplementary_Fig_1-2_and_Table_1-11.docx</a><span style="color:gray"> (891K)</span></div><div class="small guid">GUID:&nbsp;89439A49-BA51-4ADF-9A69-E431AACE6BFB</div></div></dd></dl></div><div id="idm140292074307024" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140292074307024title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">Here we perform the first genome wide association study (GWAS) of multiple myeloma (MM) survival. In a meta-analysis of 306 MM patients treated at UCSF and 239 patients treated at the Mayo clinic, we find a significant association between SNPs near the gene <em>FOPNL</em> on chromosome 16p13 and survival (rs72773978; p=6 × 10<sup>−10</sup>). Patients with the minor allele are at increased risk for mortality (HR 2.65; 95% CI: 1.94 – 3.58) relative to patients homozygous for the major allele. We replicate the association in the IMMEnSE cohort including 772 patients, and a University of Utah cohort including 318 patients (rs72773978 p=0.044). Using publically available data, we find that the minor allele was associated with increased expression of <em>FOPNL</em> and increased expression of <em>FOPNL</em> was associated with higher expression of centrosomal genes and with shorter survival.. Polymorphisms at the <em>FOPNL</em> locus are associated with survival among MM patients.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">INTRODUCTION</h2><p id="P3" class="p p-first">Multiple myeloma (MM) is an incurable hematological malignancy of plasma cells. Approximately 22,000 new cases are diagnosed each year in the United States and over 10,000 deaths occur annually<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup>. Family history is a strong risk factor for MM<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512907">2</a></sup>. Recent genome-wide association studies (GWAS) reported 8 loci associated with susceptibility to MM<sup><a href="#R3" rid="R3" class=" bibr popnode">3</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604630463">5</a></sup>.</p><p id="P4">A variety of clinical features and biomarkers are associated with MM prognosis <sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512914">6</a></sup>. Chromosomal abnormalities are also associated with prognosis; deletions at 17p, 13q, amplifications at 1q and translocations t(4;14) and t(14;16) have been associated with a poor prognosis, while hyperdiploidy is associated with a favorable prognosis<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512918">7</a>–<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512926">9</a></sup>. Gene expression signatures of the myeloma cells also predict survival<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512905">10</a>–<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512902">14</a></sup>.</p><p id="P5" class="p p-last">Germline genetic variants are associated with survival among patients with esophageal<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512937">15</a></sup>, breast<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512906">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512925">17</a></sup>,pancreatic<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512933">18</a>,<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512894">19</a></sup> and small cell lung cancer<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512903">20</a></sup>. We performed the first GWAS of MM survival, by conducting a meta-analysis of two studies from the University of California San Francisco (UCSF) and the Mayo Clinic. We found a locus on chromosome 16 associated with survival. We replicated the findings in the IMMEnSE consortium and the University of Utah cohort. The top SNPs were at the <em>FOPNL</em> locus. Using publicly available data of gene expression from peripheral blood of normal individuals, we found that the risk alleles at the top SNPs were associated with increased expression of <em>FOPNL</em> We also found that increased expression of FOPNL was associated with higher “centrosome index”, a gene expression correlate of centrosome amplification in multiple myeloma cells which has been associated with poor survival.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">RESULTS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Identifying a Locus for Survival</h3><p id="P6" class="p p-first">We performed a GWAS of overall survival among MM patients in cohorts (<a href="/pmc/articles/PMC4656791/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) from UCSF (n=306) and Mayo Clinic (n=239) separately. One <em>locus</em> mapping to chromosome 16p13.11 (Hg19) showed a suggestive association in both the UCSF (p=8.4 × 10<sup>−7</sup>; proportional hazards model) and the Mayo Clinic studies (p= 1.1 × 10<sup>−4</sup>; proportional hazards model). In a meta-analysis of these GWAS, the locus was genome-wide significant (<a href="/pmc/articles/PMC4656791/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1a</span></a>) with the strongest evidence at 2 SNPs in perfect linkage disequilibrium rs72773978 and rs117863986 (p=6.0 × 10<sup>−10</sup> for both; meta-analysis p value is calculated using inverse variance based weighting). We found no significant deviation from the proportional hazards assumption for the top SNP in either the UCSF (p=0.74; p values calculated by testing whether scaled Schoenfeld residuals vary with time) or Mayo clinic studies (p=0.95). We identified 131 SNPs at this locus associated with survival at p&lt;5×10<sup>−8</sup> (Supplementary Data 1); these SNPs had approximately 5–7% minor allele frequency and were in tight linkage disequilibrium (r<sup>2</sup>&gt;0.8) with the top SNPs (<a href="/pmc/articles/PMC4656791/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>figure 1b</span></a>). Of the 131 top SNPs, 17 were genotyped in the UCSF dataset and 1 was genotyped in the Mayo Clinic dataset (SupplementaryData 1). The remaining SNPs were imputed, but had very high imputation quality scores (Information&gt;0.9 or r<sup>2</sup>&gt;0.9). We directly genotyped 8 additional SNPs in the Mayo clinic dataset, including one of the top 2 SNPs, rs117863986, and found consistently strong levels of association with the genotyped SNPs (rs117863986 HR: 2.26; 95% CI: 1.46 – 3.40; p=0.00021; proportional hazards model) and other SNPs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4656791/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140292066753472" class="figure"><a href="/pmc/articles/PMC4656791/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4656791_nihms-730494-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-730494-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4656791/bin/nihms-730494-f0001.jpg"></a></div><div id="largeobj_idm140292066753472" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4656791/figure/F1/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140292066752624" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4656791_nihms-730494-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-730494-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4656791/bin/nihms-730494-f0002.jpg"></a></div><div id="largeobj_idm140292066752624" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4656791/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4656791/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">GWAS results for survival among MM patients (a) Manhattan plot of results of genome-wide association analysis for survival in MM patients. Each point represents the negative log p value of the meta-analysis for association with survival using the UCSF and Mayo Clinic data (b) Locuszoom plot for association statistics at the 16p13 region in the meta-analysis of UCSF and Mayo. Each dot represents the negative log p value for the association statistics from the meta-analysis. The top associated SNP (rs72773978) is colored in purple and the remaining SNPs are colored according to linkage disequilibrium values (r<sup>2</sup>) with the top SNP.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Clinical characteristics of patients in discovery and replication studies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140292074088896" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="right"></th><th rowspan="1" colspan="1" valign="top" align="center">UCSF (N=306)</th><th rowspan="1" colspan="1" valign="top" align="center">Mayo Clinic (N=239)</th><th rowspan="1" colspan="1" valign="top" align="center">IMMENSE Cohorts &amp; University of Utah (N=1090)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Age at diagnosis Mean (SD)</td><td rowspan="1" colspan="1" valign="top" align="center">55.9 ± 9.0</td><td rowspan="1" colspan="1" valign="top" align="center">62.2 ± 11.4</td><td rowspan="1" colspan="1" valign="top" align="center">54.4 ± 10.1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Average follow up time Mean years (SD)</td><td rowspan="1" colspan="1" valign="top" align="center">3.8 ± 2.5</td><td rowspan="1" colspan="1" valign="top" align="center">5.3 ± 3.5</td><td rowspan="1" colspan="1" valign="top" align="center">4.8 ± 3.3</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mortality N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">103 (33.7)</td><td rowspan="1" colspan="1" valign="top" align="center">174 (72.8)</td><td rowspan="1" colspan="1" valign="top" align="center">423 (38.5%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Type of disease (N with data)</td><td rowspan="1" colspan="1" valign="top" align="center">292</td><td rowspan="1" colspan="1" valign="top" align="center">239</td><td rowspan="1" colspan="1" valign="top" align="center">649</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">IgG Kappa N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">127 (43.5)</td><td rowspan="1" colspan="1" valign="top" align="center">86 (36.0)</td><td rowspan="1" colspan="1" valign="top" align="center">145 (22.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">IgG Lambda N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">44 (15.1)</td><td rowspan="1" colspan="1" valign="top" align="center">49 (20.5)</td><td rowspan="1" colspan="1" valign="top" align="center">64 (9.9)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">IgG light chain not specified N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">11 (3.8)</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">173 (26.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">IgA Kappa N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">31 (10.6)</td><td rowspan="1" colspan="1" valign="top" align="center">35 (15.0)</td><td rowspan="1" colspan="1" valign="top" align="center">35 (5.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">IgA Lambda N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">21 (7.2)</td><td rowspan="1" colspan="1" valign="top" align="center">22 (9.1)</td><td rowspan="1" colspan="1" valign="top" align="center">33 (5.1)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">IgA light chain not specified N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">75 (11.6)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">Light chain only N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">51 (17.5)</td><td rowspan="1" colspan="1" valign="top" align="center">31 (13.0)</td><td rowspan="1" colspan="1" valign="top" align="center">86 (13.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">Other/non-secretory N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">7 (2.4)</td><td rowspan="1" colspan="1" valign="top" align="center">16 (6.7)</td><td rowspan="1" colspan="1" valign="top" align="center">38 (5.9)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ISS Stage (N with data)</td><td rowspan="1" colspan="1" valign="top" align="center">140</td><td rowspan="1" colspan="1" valign="top" align="center">221</td><td rowspan="1" colspan="1" valign="top" align="center">513</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">1 N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">49 (35.0)</td><td rowspan="1" colspan="1" valign="top" align="center">86 (38.9)</td><td rowspan="1" colspan="1" valign="top" align="center">162 (31.6)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">2 N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">55 (39.3)</td><td rowspan="1" colspan="1" valign="top" align="center">74 (33.5)</td><td rowspan="1" colspan="1" valign="top" align="center">165 (32.2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">3 N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">36 (25.7)</td><td rowspan="1" colspan="1" valign="top" align="center">61 (27.6)</td><td rowspan="1" colspan="1" valign="top" align="center">186 (36.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Durie-Salmon Stage (N with data)</td><td rowspan="1" colspan="1" valign="top" align="center">256</td><td rowspan="1" colspan="1" valign="top" align="center">187</td><td rowspan="1" colspan="1" valign="top" align="center">633</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">IA or IB N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">39 (15.2)</td><td rowspan="1" colspan="1" valign="top" align="center">14 (7.5)</td><td rowspan="1" colspan="1" valign="top" align="center">69 (10.9)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">IIA or IIB N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">40 (15.6)</td><td rowspan="1" colspan="1" valign="top" align="center">30 (16.4)</td><td rowspan="1" colspan="1" valign="top" align="center">149 (23.5)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">IIIA or IIIIB N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">177 (69.2)</td><td rowspan="1" colspan="1" valign="top" align="center">143 (76.5)</td><td rowspan="1" colspan="1" valign="top" align="center">415 (65.6)</td></tr></tbody></table></div><div id="largeobj_idm140292074088896" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4656791/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P7">Analysis of the genome wide distribution of association statistics (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1a</a>) revealed minimal deviation from the expectation under the null (lambda=1.002). After removing SNPs from a 200KB region around the top locus on chromosome 16, we found no evidence for additional signal genome wide (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1b</a>) although some other loci had some suggestive signals with p values 5×10<sup>−7</sup> – 1×10<sup>−7</sup> (Supplementary Data 1).</p><p id="P8">The UCSF study had a median time of 7.6 months (interquartile range 5.7–8.9 months) between date of diagnosis and date of ascertainment. Therefore, we considered whether this delay affected our results. First, we adjusted for the time difference between date of diagnosis and ascertainment in the proportional hazards models and found no attenuation (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). We also considered models stratified by the delay between date of diagnosis and ascertainment (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). We found that the effect was consistent among patients enrolled between 0 to 5.9 months after diagnosis (HR 3.23; 95% CI: 1.28 – 8.17; p=0.013; proportional hazards model) those enrolled between 6 and 11.9 months after diagnosis (HR 3.04; 95% CI: 1.67 –5.55; p=0.00028; proportional hazards model) and those enrolled from 12 to 23.9 months (HR 5.06; 95%CI: 1.18 – 21.79; p=0.029; proportional hazards model). Thus, we concluded that the delay between diagnosis and ascertainment within the first two years was unlikely to affect the association between the SNP and overall survival. We also considered models that adjusted for the difference between date of diagnosis and ascertainment in the Mayo study and found no change in the association as expected (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>), since nearly all of the participants were ascertained within 1 month of diagnosis.</p><p id="P9">We searched for additional SNPs that were associated independently of the top SNP by performing conditional analyses including rs72773978 and other SNPs within 100 KB of that SNP. We performed survival analyses for all SNPs that were either in no LD (R<sup>2</sup>&lt;0.1) or in modest LD (R<sup>2</sup> 0.1–0.5) with RS72773978, adjusting for the effect of PC's and for the effect of RS72773978. We found no other significant associations in the locus after adjusting for multiple hypothesis testing.</p><p id="P10">Median survival was decreased by approximately 2.7 years among patients who were either heterozygous or homozygous for the rare variant (T allele) of rs72773978 compared to patients homozygous for the common variant in both the UCSF and Mayo Clinic cohorts (<a href="/pmc/articles/PMC4656791/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). In models that adjusted for age, gender and genetic ancestry, approximately 10–14% of patients had an increased risk of death (hazard ratio ~2.6; <a href="/pmc/articles/PMC4656791/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>) in the meta-analysis of these 2 datasets.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4656791/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140292072100816" class="figure"><a href="/pmc/articles/PMC4656791/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4656791_nihms-730494-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-730494-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4656791/bin/nihms-730494-f0003.jpg"></a></div><div id="largeobj_idm140292072100816" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4656791/figure/F2/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140292072099968" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4656791_nihms-730494-f0004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-730494-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4656791/bin/nihms-730494-f0004.jpg"></a></div><div id="largeobj_idm140292072099968" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4656791/figure/F2/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140292072099120" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4656791_nihms-730494-f0005.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-730494-f0005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4656791/bin/nihms-730494-f0005.jpg"></a></div><div id="largeobj_idm140292072099120" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4656791/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4656791/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Kaplan-Meier Survivorship plot by genotype for rs72773978 in the UCSF (N=306) (A) Mayo (N=239) (B) and IMMENSE/Utah (N=1080) (C) studies. We plotted the results by genotype. To generate the plots for the discovery datasets, we rounded the imputation results to the nearest whole numbers to infer genotypes of individuals.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">Association of rs72773978 and MM survival</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140292070138720" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="5" rowspan="1" valign="bottom" align="left">Age, Sex adjusted<sup>*</sup></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Study</th><th rowspan="1" colspan="1" valign="bottom" align="center">Allele frequency</th><th rowspan="1" colspan="1" valign="bottom" align="center">HR</th><th rowspan="1" colspan="1" valign="bottom" align="center">95% CI</th><th rowspan="1" colspan="1" valign="bottom" align="center">P- value<sup>***</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="bottom" align="left">UCSF (n=306)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.075</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.03</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.95 – 4.73</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.0 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Mayo (n=239)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.054</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.31</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.51 – 3.53</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.1 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">
<strong>Discovery Meta-analysis (UCSF &amp; Mayo)</strong>
</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">
<strong>2.65</strong>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<strong>1.94 – 3.58</strong>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<strong>6.0 × 10<sup>−10</sup></strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Replication<sup>**</sup> (1,090): IMMENSE n=772, Utah n=315</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.051</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.34</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.01 – 1.78</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.044</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">
<strong>Discovery &amp; Replication Meta-analysis</strong>
</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.93</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.54 – 2.41</td><td rowspan="1" colspan="1" valign="bottom" align="center">6.7 × 10<sup>−9</sup></td></tr></tbody></table></div><div id="largeobj_idm140292070138720" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4656791/table/T2/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140292075264112" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="4" rowspan="1" valign="bottom" align="left">Age, Sex, Stage adjusted <sup>*</sup></th></tr></thead><tbody><tr><td colspan="4" rowspan="1" valign="bottom" align="left">ISS adjusted</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">UCSF (n=140)</td><td rowspan="1" colspan="1" valign="bottom" align="left">4.06</td><td rowspan="1" colspan="1" valign="bottom" align="left">2.04 – 8.05</td><td rowspan="1" colspan="1" valign="bottom" align="left">6.3 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Mayo (n=221)</td><td rowspan="1" colspan="1" valign="bottom" align="left">2.41</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.51 – 3.83</td><td rowspan="1" colspan="1" valign="bottom" align="left">2.2 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Replication <sup>**</sup> (n=513)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.70</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.19 – 2.44</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.004</td></tr><tr><td colspan="4" rowspan="1" valign="bottom" align="left">Salmon-Durie adjusted</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">UCSF (n=256)</td><td rowspan="1" colspan="1" valign="bottom" align="left">2.81</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.73 – 4.57</td><td rowspan="1" colspan="1" valign="bottom" align="left">3.2 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Mayo (n=184)</td><td rowspan="1" colspan="1" valign="bottom" align="left">2.75</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.68 – 4.49</td><td rowspan="1" colspan="1" valign="bottom" align="left">5.3 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Replication <sup>**</sup> (n=625)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.46</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.04– 2.05</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.029</td></tr></tbody></table></div><div id="largeobj_idm140292075264112" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4656791/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>The UCSF and Mayo Clinic study results are also adjusted by PCA. The IMMENSE data is not due to the lack of availability of GWAS data.</div><div id="TFN2"><sup>**</sup>The IMMENSE results are from a meta-analysis of the individual regions (see supplementary table 3).</div><div id="TFN3"><sup>***</sup>P values are calculated from proportional hazards models.</div></div></div><p id="P11">We also performed a separate analysis of individuals who genetically clustered with Caucasians in the UCSF dataset (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a>). We repeated the meta-analysis of the Mayo Clinic data with the UCSF Caucasian only sample (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>), and found the association with rs72773978 remained significant (p=2.4×10<sup>−9</sup>; proportional hazards model). The UCSF cohort included African American patients (N=25 and patients of Latino (N=27) or other patients who clustered with those of mixed ancestry (N=24). In an analysis of these patients, we also found a nominally significant association between shorter survival and the minor allele (HR 2.43; 95% CI: 1.09 – 5.39; p=0.029 proportional hazards model).</p><p id="P12" class="p p-last">We also examined whether the top SNPs that we identified were associated with susceptibility to MM in our two studies. We found no significant difference in genotype frequencies between cases and controls (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>).</p></div><div id="S4" class="sec"><h3 id="S4title">Relationship to Stage at Diagnosis and Treatment</h3><p id="P13" class="p p-first">We evaluated the effect of the genotype on survival after adjustment for clinical stage. In analyses that adjusted for stage using either the ISS definition or the Durie-Salmon staging system, rs72773978 genotype remained a strong predictor of survival (<a href="/pmc/articles/PMC4656791/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Among the participants in the Mayo Clinic on whom LDH levels were available (N=154), we saw a consistent level of association (HR: 2.25; 95% CI: 1.26 – 4.03; p=0.006; proportional hazards model).</p><p id="P14">Since MM treatment has improved significantly in the last decade, we used data available from both cohorts to determine whether the SNP effect varied by initial treatment (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). We found a consistent effect of the SNP regardless of the type of treatment initiated (<a href="/pmc/articles/PMC4656791/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>) and no evidence of interaction between treatment and the SNP in either the UCSF (p=0.9; p for interaction derived using proportional hazards model) or Mayo Clinic (p=0.52; p for interaction derived using proportional hazards model) cohorts.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p6">Effect of SNP by initial treatment among patients in the UCSF and Mayo Clinic cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140292073097520" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="right"></th><th rowspan="1" colspan="1" valign="bottom" align="center">HR<sup>*</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">95% CI</th><th rowspan="1" colspan="1" valign="bottom" align="center">P value<sup>***</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Mayo Clinic</td><td colspan="3" rowspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">Old treatments<sup>**</sup></td><td colspan="3" rowspan="1" valign="top" align="center">N=136, 102 deaths</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">RS72773978</td><td rowspan="1" colspan="1" valign="top" align="center">1.90</td><td rowspan="1" colspan="1" valign="top" align="center">0.98 – 3.83</td><td rowspan="1" colspan="1" valign="top" align="center">0.057</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">New treatments<sup>***</sup></td><td colspan="3" rowspan="1" valign="top" align="center">N=93, 64 deaths</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">RS72773978</td><td rowspan="1" colspan="1" valign="top" align="center">2.71</td><td rowspan="1" colspan="1" valign="top" align="center">1.56 – 4.70</td><td rowspan="1" colspan="1" valign="top" align="center">0.00045</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">Entire sample adjusted for treatment</td><td colspan="3" rowspan="1" valign="top" align="center">N=229, 166 deaths</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">RS72773978</td><td rowspan="1" colspan="1" valign="top" align="center">2.18</td><td rowspan="1" colspan="1" valign="top" align="center">1.43 – 3.32</td><td rowspan="1" colspan="1" valign="top" align="center">0.00028</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">UCSF</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">Old treatments<sup>**</sup></td><td colspan="3" rowspan="1" valign="top" align="center">N= 109 60 deaths</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">RS72773978</td><td rowspan="1" colspan="1" valign="top" align="center">3.35</td><td rowspan="1" colspan="1" valign="top" align="center">1.74 – 6.44</td><td rowspan="1" colspan="1" valign="top" align="center">0.00028</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">New treatments<sup>***</sup></td><td colspan="3" rowspan="1" valign="top" align="center">N=187, 30 deaths</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">RS72773978</td><td rowspan="1" colspan="1" valign="top" align="center">3.57</td><td rowspan="1" colspan="1" valign="top" align="center">1.71 – 7.43</td><td rowspan="1" colspan="1" valign="top" align="center">0.0007</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">Entire Sample Adjusted for treatment</td><td colspan="3" rowspan="1" valign="top" align="center">N=296, 90 deaths</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="right">RS72773978</td><td rowspan="1" colspan="1" valign="top" align="center">3.35</td><td rowspan="1" colspan="1" valign="top" align="center">2.07 – 5.41</td><td rowspan="1" colspan="1" valign="top" align="center">8.2 × 10<sup>−7</sup></td></tr></tbody></table></div><div id="largeobj_idm140292073097520" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4656791/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><sup>*</sup>All models are adjusted for age, gender and principal components 1–10.</div><div id="TFN5"><sup>**</sup>Regimens including vincristine/Adriamycin/Dexamethasone or melphalan/prednisone</div><div id="TFN6"><sup>***</sup>Treatments containing at least on of the following agents: thalidomide, botezomib or lenalidomide</div><div id="TFN7"><sup>****</sup>P values are calculated from proportional hazards models.</div></div></div><p id="P15" class="p p-last">Among patients in the Mayo Clinic study, 134 (56%) were treated by both high dose chemotherapy (HDC) followed by autologous stem-cell rescue. We adjusted for HDC in the proportional hazards models and found that although HDC was a strong predictor of longer survival in the cohort, there was only mild attenuation of the SNP association with survival (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>). Since nearly all (97%) of the UCSF patients received HDC, the same analysis could not be done within the UCSF study.</p></div><div id="S5" class="sec"><h3 id="S5title">Replication</h3><p id="P16" class="p p-first">We replicated the association of top SNPs from the UCSF and Mayo Clinic meta-analysis in a replication meta-analysis of 1,090 MM cases, including 772 European MM patients from the IMMEnSE consortium and 318 from the Utah cohort (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>). We selected 2 SNPs for replication including rs72773978, one of the top associated SNPs from the meta-analysis and rs12598966, a SNP resulting in an amino acid substitution in the <em>FOPNL</em> gene. We performed a meta-analysis of all of the replication studies and found a significant association between rs72773978 and survival in the replication cohorts adjusted by age and sex (HR 1.34; 95% CI: 1.01 – 1.74; p=0.044; proportional hazards model) with survival shorter by approximately 1.2 years among carriers of the minor allele (<a href="/pmc/articles/PMC4656791/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2c</span></a>). There was no evidence of heterogeneity of effect within the replication studies (p=0.14; chi-squared test for heterogeneity). The other SNP, rs12598966, was not significantly associated with survival in the replication.</p><p id="P17" class="p p-last">We noted a slightly stronger effect size in analyses that adjusted for stage. However, the change in effect size was not due to negative confounding between stage and the SNP, but rather to the fact that the cohorts with missing data on stage were the ones with an inconsistent point estimate (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). In a meta-analysis that did not adjust for ISS stage but just included participants without missing data on ISS stage, we saw approximately the same effect size (HR 1.71; 95% CI: 1.18 – 2.47; p=0.005; proportional hazards model) as in the meta-analysis that adjusted for ISS stage (<a href="#SD1" rid="SD1" class=" supplementary-material">supplementary table 7</a>.)</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Analysis of Function</h3><p id="P18" class="p p-first">The top SNPs were in a region that overlapped the entire <em>FOPNL</em> gene and a portion of the <em>MYH11</em> gene (<a href="/pmc/articles/PMC4656791/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>figure 1b</span></a>). In addition, known drug transporters, <em>ABCC1</em> and <em>ABCC6</em>, are located about 50kb and 300kb away, respectively. One of the top SNPs, rs12598966, is located in the coding sequence of <em>FOPNL</em>, and leads to a non-synonymous amino acid substitution: E-&gt;K at amino acid 156; however, this SNP was not significantly associated with survival in the replication and was not predicted to have a deleterious effect on protein function (SIFT score =0.89 and Polyphen 2 score=0.275). Next, we investigated the top 145 SNPs (all in tight LD (r<sup>2</sup>&gt;0.8) with rs72773978) for an effect on gene expression using GENEVAR<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512910">21</a></sup>. The top 2 SNPs, rs72773978 and rs117863986, were not included in the database, but six other SNPs in strong LD (r<sup>2</sup>=1) with rs72773978 are present in the database and are associated with expression of <em>FOPNL</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>). The minor allele of these SNPs predicted higher expression of <em>FOPNL</em>. There was no significant association between these SNPs and expression of other genes within 1 MB of the locus.</p><p id="P19">We identified 13 SNPs in LD with rs72773978 as being potentially functional (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>). Six of these SNPs are in the 3' UTR of the <em>FOPNL</em> gene and, therefore, may be involved in transcript stability. Seven SNPs were identified as being in sites of open chromatin and thus may be involved in transcriptional regulation.</p><p id="P20" class="p p-last">Since the top SNPs were associated with gene expression, we hypothesized that expression of <em>FOPNL</em> may be associated with survival among MM patients. In particular, higher expression of <em>FOPNL</em> is associated with the minor allele of the top SNPs from the GWAS and should also be associated with shorter survival. We used publically available data on gene expression (<a id="__tag_492512940" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE2658" ref="/geo/query/acc.cgi?acc=GSE2658"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE2658","term_id":"2658"}}</span>GSE2658</a>) and survival from 414 MM cases to test this hypothesis<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512927">13</a></sup>. As predicted, we found a significant association between higher expression of <em>FOPNL</em> and worse survival (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 10</a>). FOPNL is known to localize to the centrosome and the pericentriolar satellites. Since centrosome amplification is known to be a predictor of poor prognosis, we evaluated the association between <em>FOPNL</em> expression and the centrosome index (CI), a previously validated gene expression signature of centrosome amplification. We found a very strong correlation (<a href="#SD1" rid="SD1" class=" supplementary-material">supplementary table 11</a>) between higher <em>FOPNL</em> expression and increased CI in the study we analyzed for survival (<a id="__tag_492512945" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE2658" ref="/geo/query/acc.cgi?acc=GSE2658"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE2658","term_id":"2658"}}</span>GSE2658</a>) and in two additional studies (<a id="__tag_492512939" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE19784" ref="/geo/query/acc.cgi?acc=GSE19784"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE19784","term_id":"19784"}}</span>GSE19784</a> and <a id="__tag_492512944" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE26760" ref="/geo/query/acc.cgi?acc=GSE26760"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE26760","term_id":"26760"}}</span>GSE26760</a>).</p></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">DISCUSSION</h2><p id="P21" class="p p-first">We performed a GWAS for survival among MM patients and identified SNPs at chromosome 16p13 that were strongly associated with mortality. The SNPs were in the region of the <em>FOPNL</em> gene and a subset of the SNPs were associated with <em>FOPNL</em> expression levels, with the minor allele predicting higher expression<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512892">22</a></sup>. We also found that <em>FOPNL</em> expression was associated with poorer survival using data from a previous study<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512935">13</a></sup>. Thus, our results strongly suggest that <em>FOPNL</em> is a gene involved in myeloma progression.</p><p id="P22">FOPNL is known to be associated with centrosome function<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512909">23</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512900">24</a></sup>. Centrosome amplification is common in MM and is associated with poor prognosis<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512896">25</a>,<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512908">26</a></sup>. Furthermore, inhibition of centrosomal clustering may be effective in treatment of MM<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512911">27</a>,<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512916">28</a></sup>. We found that germline variation that affects a gene involved in centrosomal function may also contribute to disease progression. Furthermore, we found an association between <em>FOPNL</em> expression and centrosome index in 3 datasets of gene expression from myeloma samples. However, our results implicating <em>FOPNL</em> as the causal gene at this locus rely on the synthesis of several datasets. We were not able to directly correlate the SNPs with gene expression, centrosome index and survival in the same dataset. It is possible that another gene/s at this locus may be responsible for the effect we observe, or that the effect is mediated by FOPNL, but that it is not through a mechanism related to centrosome amplification. Additional studies of this gene and centrosomal function will help to further define the mechanism underlying the association that we identified.</p><p id="P23">Our results imply that germline genetic variation is associated with survival among patients with MM. Other GWAS have identified loci that affect survival in other cancer types<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512904">15</a>–<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512899">18</a>,<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512895">20</a></sup>. At least one of the known loci for MM susceptibility is associated with the risk for a particular subtype of MM<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_570590281">4</a></sup> which may also have an effect on prognosis.</p><p id="P24">MM is a heterogeneous disease with substantial variation in prognosis among different patients. Identifying patients who are at higher risk of progression may be of importance in treating these patients more aggressively earlier in their disease. Our results identify <em>FOPNL</em> genotype as a predictor of survival, and we found that the association remains significant after adjustment for stage at diagnosis. However, the effect size we observed in the replication cohorts was substantially attenuated compared to the effect size in the discovery cohorts. This difference is most likely due to “winner's curse” – a tendency for the initial study to over-estimate the true effect size<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512913">29</a></sup>. Thus, the replication cohorts in our study are more likely to represent the true effect size in future studies.</p><p id="P25">Our study has several important limitations. We could not examine the association between SNPs and MM survival by cytogenetic abnormalities since the majority of our patients were diagnosed prior to the common use of cytogenetic and fluorescent in-situ hybridization (FISH) analysis in clinical practice. Therefore, it will be important to examine the utility of this genotype in the setting of cytogenetic and FISH analysis and gene expression signatures. Furthermore, we could not adjust for gene expression patterns which are also known to be associated with survival.</p><p id="P26">We found no difference between patients who were initially treated with modern vs. older therapies. However, our ability to analyze the SNP by different therapies was limited to the UCSF dataset and we had inadequate power to detect interactions between the SNPs and particular drugs. Furthermore, we only adjusted for the association between initial treatment and survival, and it is likely that many of the patients who were initially treated with older regimens received newer regimens if they survived to the era when these became available. It is possible that the effect that we saw is modified by one regimen or by one class of medications. Additional studies should be done in the context of clinical trials or other cohorts where treatment regimens are more uniform to investigate whether the effect of the locus we identified is modified by particular treatments.</p><p id="P27" class="p p-last">In summary, we found a strong association between a locus on chromosome 16p and MM survival that is likely due to an effect on expression of the <em>FOPNL</em> gene. The SNPs we identified may become important clinical predictors of outcome among MM patients.</p></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">METHODS</h2><div id="S9" class="sec sec-first"><h3 id="S9title">UCSF Study</h3><p id="P28" class="p p-first">The UCSF Institutional Review Board (IRB) approved ascertainment of cases and use of existing biospecimens for genetic analysis. All participants gave informed consent. The study included 370 patients treated for MM at UCSF between 1989 and 2010. We excluded 10 samples due to insufficient clinical data. We also excluded 42 participants whose blood was collected ≥2 years after diagnosis from the survival analysis due to the potential bias towards long-term survivors among these participants. The median delay between diagnosis and ascertainment among the 42 participants excluded was 2.9 years (interquartile range 2.4 to 4.7 years). The median delay among the participants included was 7.6 months (interquartile range 5.7–8.9 months).</p><p id="P29">We used white blood cells harvested after mobilization of stem cells with granulocyte colony stimulating factor (GCSF) as a source of DNA. Mobilization of stem cells is performed prior to high dose chemotherapy followed by autologous stem cell transplantation. The patient receives GCSF and then undergoes harvesting of peripheral white blood cells via apheresis several days later<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512920">30</a>,<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512919">31</a></sup>. Bone marrow stem cell fraction is monitored via CD34 antibody and apheresis is continued until an adequate number of CD34+ cells have been collected for stem cell rescue. The cells are then stored in liquid nitrogen.</p><p id="P30" class="p p-last">We ascertained date of death using chart reviews and death registry data. Survival time was determined as the date of diagnosis until date of death or last clinic visit for patients who were not known to have died. Clinical stage and initial chemotherapy regimen was determined by chart review. For analysis of treatment, we dichotomized treatments into either newer regimens (including an Imid and/or proteasome Inhibitor) or older regimens (including neither an Imid nor proteasome Inhibitor). Nearly all of the participants in the UCSF study (&gt;97%) received high dose chemotherapy with stem cell rescue (autologous bone marrow transplant).</p></div><div id="S10" class="sec"><h3 id="S10title">Mayo Clinic Study</h3><p id="P31" class="p p-first-last">Ascertainment of MM cases and genotyping was approved by the Mayo Clinic Institutional Review Board. The study included incident MM cases seen in the regional practice between 1998 and 2007 and recruited within 6 months of initial diagnosis. Nearly all participants (96%) were recruited within 1 month of diagnosis and the remainder were recruited between 1 and 5.5 months after diagnosis. Eligible cases provided consent and a blood sample for research studies of MM. A total of 243 MM cases were used for analyses. DNA was extracted from stored peripheral blood samples. We ascertained date of death and calculated survival time as described for the UCSF study. Clinical stage was determined by chart review. Clinical data on survival could be ascertained on 239 patients. Initial chemotherapy regimen and use of high dose chemotherapy was determined by chart review. For analysis of treatment, we dichotomized treatments into either newer regimens (including an Imid and/or proteasome Inhibitor) or older regimens (including neither an Imid nor proteasome Inhibitor). Approximately 56% of participants were treated with autologous stem cell transplant.</p></div><div id="S11" class="sec"><h3 id="S11title">IMMENSE Study</h3><p id="P32" class="p p-first-last">The International Multiple Myeloma rESEarch (IMMEnSE) consortium is a case-control study recruited from 7 different European and North American countries<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512934">32</a></sup>. MM cases are defined by a confirmed diagnosis of MM, according to the International Myeloma Working Group (IMWG) criteria<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512938">33</a></sup>. For each patient, demographic and clinical parameters at diagnosis and survival were collected by the responsible clinicians in each of the IMMEnSE centers. The data collected are standardized in a central database kept at the German Cancer Research Center (DKFZ, Heidelberg, Germany). For each subject, a sample of peripheral blood or extracted DNA has been collected and sent to DKFZ. A total of 772 MM cases with survival information available in the IMMEnSE consortium were included in this study.</p></div><div id="S12" class="sec"><h3 id="S12title">Utah study</h3><p id="P33" class="p p-first">Sampling and genetic analysis was approved by the University of Utah Institutional Review Board. The study included prevalent MM cases in the state of Utah, ascertained up to 2012. Eligible cases provided consent and a blood or saliva sample from which DNA was extracted. Date of diagnosis was confirmed from chart review and Utah Cancer Registry data. Date of death was confirmed from chart review and death registry data. Survival time was determined as the date of diagnosis until date of death, last contact with the study, or last known event in Utah (determined from statewide vital records, driver's license renewals, and voter registrations in the Utah Population Database) for patients who were not known to have died. A total of 318 MM cases with DNA and survival information were available for this study.</p><p id="P34" class="p p-last">All SNP positions were annotated using the Genome Reference Consortium GRCh37 (Hg19) version of the human genome.</p></div><div id="S13" class="sec"><h3 id="S13title">UCSF</h3><p id="P35" class="p p-first-last">A pilot study of 81 MM samples used an Illumina 660 array genotyped at the UCSF Genomics Core Facility. In a second phase, we genotyped 289 MM samples using an Illumina Omni5 array at Expression Analysis (Durham, NC). Of the 370 participants in the GWAS, 52 participants were excluded from the survival analysis as noted above due to either insufficient clinical data (N=10) or due to &gt;2 year time difference between diagnosis and ascertainment (N=42). Of the remaining 318 samples we dropped 12 since they did not pass quality control for genotyping. Eleven were dropped due to high missing genotype values (&gt;5% missing genotypes per sample) and 1 sample was dropped due to potential contamination, leaving 306 patients. We dropped SNPs that had &gt;5% missing values, or were monomorphic. Imputation was performed using IMPUTE2<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_492512932">34</a></sup> (<a href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html#home" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/impute/impute_v2.html#home</a>) with all samples from 1000 Genomes dataset (Version 2, May 2011 release<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938526">35</a></sup>) as a reference. Imputed SNPs with Information&lt;0.5 or minor allele frequency (MAF)&lt;0.025 were excluded, leaving 8,036,255 SNPs for analysis.</p></div><div id="S14" class="sec sec-last"><h3 id="S14title">Mayo Clinic</h3><p id="P36" class="p p-first">Cases were genotyped using the Affymetrix 6.0 array. Monomorphic SNPs and those with a call rate &lt; 95% were excluded, leaving 786,950 observed SNPs. Four samples with call rates &lt; 95% and one sample with non-European ancestry according to principal components analysis were excluded, leaving 243 MM cases and 239 with follow-up past date of diagnosis. Imputation was performed with BEAGLE<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938527">36</a></sup> (<a href="http://faculty.washington.edu/browning/beagle/beagle.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://faculty.washington.edu/browning/beagle/beagle.html</a>), using all samples from version 2 of the 1000 Genomes data (May 2011 release) as reference. Imputed SNPs with an r<sup>2</sup> &lt; 0.3 or MAF &lt; 0.025 were excluded, leaving 7,276,170 SNPs for further analyses.</p><p id="P37">We selected 2 SNPs for replication in IMMEnSE and Utah samples including one of the top two SNPs from the meta-analysis (rs72773978) and a SNP in high linkage disequilibrium with the top SNP, encoding a non-synonymous amino acid substitution in <em>FOPNL</em> (rs12598966). These SNPs were typed using 5' exonuclease (TaqMan) assays (ABI) at the German Cancer Research Center (DKFZ) in Heidelberg (IMMEnSE samples) and at the Genomics Core at the University of Utah (Utah samples). Duplicates of 12% of the samples were interspersed throughout the plates and concordance rate among duplicates was &gt;99.9%.</p><p id="P38">We performed genome-wide analyses for association with survival using proportional hazards models in the UCSF and Mayo datasets separately. We inferred genetic ancestry using principal components analysis (PCA) in each cohort using SmartPCA<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938539">37</a></sup>. Each SNP was entered into the model under an assumption of log-additive increased risk, and adjusting for PC's 1–3, age and gender. Imputed SNPs were modeled using the probability of genotypes. We tested the proportional hazards assumption for the top SNPs by calculating the scaled Schoenfeld residuals and testing whether they are significantly associated with time<sup><a href="#R38" rid="R38" class=" bibr popnode">38</a></sup>.</p><p id="P39">We also performed a subset analysis of Caucasians only in the UCSF dataset. We identified Caucasians based on genetic ancestry (see <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary figure 2</a>). Individuals who clustered with self-described Caucasians (PC1&gt;0, PC2&lt;0) were included in this subset analysis (N=229).</p><p id="P40">All analyses were performed in R. For graphing survival results, we used the Kaplan-Meier estimates of the survival function and graphed the results using Stata (Version 10). We graphed the association statistics for all SNPs near the top locus using LocusZoom<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938532">39</a></sup>.</p><p id="P41">We used Cox regression models adjusted for age and gender to test the association between SNPs and survival in the IMMEnSE consortium and the Utah cohort.</p><p id="P42">We performed a meta-analysis of the UCSF and Mayo Clinic results on a total of 6,026,834 SNPs in common from both GWAS that met the allele frequency and imputation quality thresholds. We also conducted a meta-analysis of data on two top SNPs from 7 centers within the IMMEnSE consortium and the Utah study in our replication study. We calculated a fixed effects model for each SNP using METAL <sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938535">40</a></sup>. We used Cochran's Q statistic to test for heterogeneity.</p><p id="P43">To examine the association of SNPs and risk of MM, we compared the genotype frequencies of cases vs. ethnically matched controls from the UCSF (N=298) and the Mayo Clinic (N=295) sites respectively. We used logistic regression models, adjusting for PC1-3 age and gender.</p><p id="P44">We used the dataset from Grundberg <em>et al</em><sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938530">22</a></sup> for eQTL analyses, which consists of 856 Caucasian individuals including 154 monozygotic twin pairs, 232 dizygotic twin pairs and 84 singletons. We focused on expression in lymphocytes in this dataset. We used GENEVAR<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938531">21</a></sup> to query the top 145 SNPs from the GWAS and identified 6 SNPs, that were also in the Grundberg et al dataset. We queried GENEVAR for beta coefficients and p values for associations between 6 SNPs and the genes within a 1 Mb windowincluding: FOPNL, MYH11, ABCC1, NDE1, KIAA0430, ABCC6, RRN3, NTAN1 KIAA0250, KIAA0251.</p><p id="P45">We downloaded gene expression data from Zhan <em>et al</em><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938538">13</a></sup>, Broyl <em>et al</em><sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938537">41</a></sup>, and Chapman <em>et al</em><sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938529">42</a></sup> from the National Institutes of Health Gene Expression Omnibus (accession number: <a id="__tag_543938541" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE2658" ref="/geo/query/acc.cgi?acc=GSE2658"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE2658","term_id":"2658"}}</span>GSE2658</a>, <a id="__tag_543938540" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE19784" ref="/geo/query/acc.cgi?acc=GSE19784"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE19784","term_id":"19784"}}</span>GSE19784</a>, and <a id="__tag_543938542" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE26760" ref="/geo/query/acc.cgi?acc=GSE26760"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE26760","term_id":"26760"}}</span>GSE26760</a>, respectively). Zhan <em>et al</em> consisted of gene expression data from 559 MM samples assayed on Affymetrix U133 arrays; Broyl <em>et al</em> consisted of gene expression data from purified CD138+ plasma cells of 320 newly diagnosed myeloma patients using Affymetrix GeneChip U133 plus 2.0 arrays; Chapman <em>et al</em> consisted of 304 CD138-purified bone marrow samples from patients with multiple myeloma were analyzed on Affymetrix U133 Plus 2.0 microarrays.</p><p id="P46">Gene expression and MM outcome Of the samples in the Zhan <em>et al</em> dataset, 414 also had available clinical data and were included in the original publication<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938534">13</a></sup>, and therefore, we used the data from these 414 samples in our analyses. We used log-transformed probe intensity values as predictors of survival, entering these as continuous variables into a proportional hazards model. We analyzed each of the two probes for <em>FOPNL</em> on the Affymetrix U133 array separately and also considered the average of the two probes as a predictor of overall survival in the proportional hazards model.</p><p id="P47" class="p p-last">Analysis of potential SNP function: We used SIFT<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938533">43</a></sup> and Polyphen2<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938536">44</a></sup> to determine the likelihood that a non-synonymous amino acid substitution has a deleterious effect on protein function. We used FunciSNP<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_543938528">45</a></sup> to determine whether any of the SNPs may affect gene expression, including any SNPs with r<sup>2</sup>&gt;0.7 with rs72773928. r<sup>2</sup> values for linkage disequilibrium were calculated in European ancestry samples from 1000 genomes.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Fig 1-2 and Table 1-11</h4><div class="sup-box half_rhythm" id="d36e1068"><a href="/pmc/articles/PMC4656791/bin/NIHMS730494-supplement-Supplementary_Fig_1-2_and_Table_1-11.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(891K, docx)</sup></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Acknowledgements</h2><div class="sec"><p id="__p1">This work was supported by the Steve and Nancy Grand Multiple Myeloma Translational Initiative and a grant from Expression Analysis and a grant from the National Cancer Institute (R21CA191896). E Ziv is supported in part by a mid-career award in patient research from the National Cancer Institute (K24169004). Collection of blood samples from Polish patients and controls from Łodz area and DNA extraction was supported by a grant from Polish Ministry of Science and Higher Education (No. NN402178334). DNA extraction from Danish healthy controls was supported by The Research Fund at Region Sjælland, Denmark. The Utah study was supported by LLS 6067-09, <a id="__tag_792547828" class="tag_hotlink" href="/nuccore/CA152336" ref="/nuccore/CA152336"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA152336","term_id":"35057305","term_text":"CA152336"}}</span>CA152336</a> and <a id="__tag_792547827" class="tag_hotlink" href="/nuccore/CA134674" ref="/nuccore/CA134674"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA134674","term_id":"35021920","term_text":"CA134674"}}</span>CA134674</a>, with data collection made possible, in part, by the Utah Population Database (UPDB) and the Utah Cancer Registry (UCR). Partial support for all datasets within the UPDB is provided by the Huntsman Cancer Institute (HCI) and the HCI Cancer Center Support grant, P30 CA42014. The UCR is supported in part by NIH contract HHSN261201000026C from the NCI SEER Program with additional support from the Utah State Department of Health and the University of Utah. In Utah, Jathine Wong is thanked for support in formatting files and advice on R programming. Several authors (EZ, NC, FC, CV) are members of the International Multiple Myeloma Consortium. The authors are grateful to the leadership and other members of the International Multiple Myeloma Consortium for facilitating this collaborative study.</p></div></div><div id="idm140292069600704" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140292069600704title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P48" class="p p-first-last">Author Contributions: Study conception: EZ, ED, CV; collection of the data: EZ, ED, KC, DC, PB, JC, CD, MD, AG, KJ, AJ, SK, DA, HM, TM, JML, VR, JS, AJV, MW, JW, NC, FC, CV; performed genotyping:, AM, RD, AL BJ; analysis of the data:, DH, DS, DC, BA, YZ; drafting of the manuscript: EZ, ED; critical revision of the manuscript: EZ, ED, DH, AM, DS, KC, DC, BA, PB, GB, YZ, JC, RD, CD, MD, LF, AG, SH, KJ, AJ, SK, DA, MG, LCA, BJ, AL, HM, TM, JML, VR, JS, AJV, MW, JW, SS, NC, FC, CV</p><p></p><p class="fn sec" id="FN3"></p><p id="P49" class="p p-first-last">Competing Financial Interests The authors declare no conflicts of interest.</p><p></p></div></div><div id="idm140292069598848" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140292069598848title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Howlader N, et al.   <span class="ref-journal">SEER Cancer Statistics Review, 1975–2010.</span> National Cancer Institute; Bethesda, MD: 2010.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=SEER+Cancer+Statistics+Review,+1975–2010&amp;author=N+Howlader&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Greenberg AJ, Rajkumar SV, Vachon CM. Familial monoclonal gammopathy of undetermined significance and multiple myeloma: epidemiology, risk factors, and biological characteristics. <span><span class="ref-journal">Blood. </span>2012;<span class="ref-vol">119</span>:5359–66.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3369675/">PMC free article</a>]</span> [<a href="/pubmed/22354002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Familial+monoclonal+gammopathy+of+undetermined+significance+and+multiple+myeloma:+epidemiology,+risk+factors,+and+biological+characteristics&amp;author=AJ+Greenberg&amp;author=SV+Rajkumar&amp;author=CM+Vachon&amp;volume=119&amp;publication_year=2012&amp;pages=5359-66&amp;pmid=22354002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Chubb D, et al.  Common variation at 3q26.2, 6p21.33, 17p11.2 and 22q13.1 influences multiple myeloma risk. <span><span class="ref-journal">Nature genetics. </span>2013</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Common+variation+at+3q26.2,+6p21.33,+17p11.2+and+22q13.1+influences+multiple+myeloma+risk&amp;author=D+Chubb&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Weinhold N, et al.  The CCND1 c.870G&gt;A polymorphism is a risk factor for t(11;14)(q13;q32) multiple myeloma. <span><span class="ref-journal">Nature genetics. </span>2013;<span class="ref-vol">45</span>:522–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5056630/">PMC free article</a>]</span> [<a href="/pubmed/23502783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=The+CCND1+c.870G>A+polymorphism+is+a+risk+factor+for+t(11;14)(q13;q32)+multiple+myeloma&amp;author=N+Weinhold&amp;volume=45&amp;publication_year=2013&amp;pages=522-5&amp;pmid=23502783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Broderick P, et al.  Common variation at 3p22.1 and 7p15.3 influences multiple myeloma risk. <span><span class="ref-journal">Nature genetics. </span>2012;<span class="ref-vol">44</span>:58–61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5108406/">PMC free article</a>]</span> [<a href="/pubmed/22120009" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Common+variation+at+3p22.1+and+7p15.3+influences+multiple+myeloma+risk&amp;author=P+Broderick&amp;volume=44&amp;publication_year=2012&amp;pages=58-61&amp;pmid=22120009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Rajkumar SV, Greipp PR. Prognostic factors in multiple myeloma. <span><span class="ref-journal">Hematology/oncology clinics of North America. </span>1999;<span class="ref-vol">13</span>:1295–314. xi.</span> [<a href="/pubmed/10626152" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hematology/oncology+clinics+of+North+America&amp;title=Prognostic+factors+in+multiple+myeloma&amp;author=SV+Rajkumar&amp;author=PR+Greipp&amp;volume=13&amp;publication_year=1999&amp;pages=1295-314&amp;pmid=10626152&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Kumar S, et al.  Trisomies in multiple myeloma: impact on survival in patients with high-risk cytogenetics. <span><span class="ref-journal">Blood. </span>2012;<span class="ref-vol">119</span>:2100–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3311247/">PMC free article</a>]</span> [<a href="/pubmed/22234687" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Trisomies+in+multiple+myeloma:+impact+on+survival+in+patients+with+high-risk+cytogenetics&amp;author=S+Kumar&amp;volume=119&amp;publication_year=2012&amp;pages=2100-5&amp;pmid=22234687&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Fonseca R, et al.  Prognostic value of chromosome 1q21 gain by fluorescent in situ hybridization and increase CKS1B expression in myeloma. <span><span class="ref-journal">Leukemia. </span>2006;<span class="ref-vol">20</span>:2034–40.</span> [<a href="/pubmed/17024118" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Prognostic+value+of+chromosome+1q21+gain+by+fluorescent+in+situ+hybridization+and+increase+CKS1B+expression+in+myeloma&amp;author=R+Fonseca&amp;volume=20&amp;publication_year=2006&amp;pages=2034-40&amp;pmid=17024118&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Fonseca R, et al.  Clinical and biologic implications of recurrent genomic aberrations in myeloma. <span><span class="ref-journal">Blood. </span>2003;<span class="ref-vol">101</span>:4569–75.</span> [<a href="/pubmed/12576322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Clinical+and+biologic+implications+of+recurrent+genomic+aberrations+in+myeloma&amp;author=R+Fonseca&amp;volume=101&amp;publication_year=2003&amp;pages=4569-75&amp;pmid=12576322&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Zhan F, et al.  Gene-expression signature of benign monoclonal gammopathy evident in multiple myeloma is linked to good prognosis. <span><span class="ref-journal">Blood. </span>2007;<span class="ref-vol">109</span>:1692–700.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1794073/">PMC free article</a>]</span> [<a href="/pubmed/17023574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Gene-expression+signature+of+benign+monoclonal+gammopathy+evident+in+multiple+myeloma+is+linked+to+good+prognosis&amp;author=F+Zhan&amp;volume=109&amp;publication_year=2007&amp;pages=1692-700&amp;pmid=17023574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Zhan F, Barlogie B, Mulligan G, Shaughnessy JD, Jr., Bryant B. High-risk myeloma: a gene expression based risk-stratification model for newly diagnosed multiple myeloma treated with high-dose therapy is predictive of outcome in relapsed disease treated with single-agent bortezomib or high-dose dexamethasone. <span><span class="ref-journal">Blood. </span>2008;<span class="ref-vol">111</span>:968–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2200846/">PMC free article</a>]</span> [<a href="/pubmed/18182586" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=High-risk+myeloma:+a+gene+expression+based+risk-stratification+model+for+newly+diagnosed+multiple+myeloma+treated+with+high-dose+therapy+is+predictive+of+outcome+in+relapsed+disease+treated+with+single-agent+bortezomib+or+high-dose+dexamethasone&amp;author=F+Zhan&amp;author=B+Barlogie&amp;author=G+Mulligan&amp;author=JD+Shaughnessy&amp;author=B+Bryant&amp;volume=111&amp;publication_year=2008&amp;pages=968-9&amp;pmid=18182586&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Zhan F, et al.  Global gene expression profiling of multiple myeloma, monoclonal gammopathy of undetermined significance, and normal bone marrow plasma cells. <span><span class="ref-journal">Blood. </span>2002;<span class="ref-vol">99</span>:1745–57.</span> [<a href="/pubmed/11861292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Global+gene+expression+profiling+of+multiple+myeloma,+monoclonal+gammopathy+of+undetermined+significance,+and+normal+bone+marrow+plasma+cells&amp;author=F+Zhan&amp;volume=99&amp;publication_year=2002&amp;pages=1745-57&amp;pmid=11861292&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Zhan F, et al.  The molecular classification of multiple myeloma. <span><span class="ref-journal">Blood. </span>2006;<span class="ref-vol">108</span>:2020–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1895543/">PMC free article</a>]</span> [<a href="/pubmed/16728703" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=The+molecular+classification+of+multiple+myeloma&amp;author=F+Zhan&amp;volume=108&amp;publication_year=2006&amp;pages=2020-8&amp;pmid=16728703&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Zhan F, et al.  Gene expression profiling of human plasma cell differentiation and classification of multiple myeloma based on similarities to distinct stages of late-stage B-cell development. <span><span class="ref-journal">Blood. </span>2003;<span class="ref-vol">101</span>:1128–40.</span> [<a href="/pubmed/12393520" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Gene+expression+profiling+of+human+plasma+cell+differentiation+and+classification+of+multiple+myeloma+based+on+similarities+to+distinct+stages+of+late-stage+B-cell+development&amp;author=F+Zhan&amp;volume=101&amp;publication_year=2003&amp;pages=1128-40&amp;pmid=12393520&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Wu C, et al.  Genome-wide association study identifies common variants in SLC39A6 associated with length of survival in esophageal squamous-cell carcinoma. <span><span class="ref-journal">Nature genetics. </span>2013;<span class="ref-vol">45</span>:632–8.</span> [<a href="/pubmed/23644492" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Genome-wide+association+study+identifies+common+variants+in+SLC39A6+associated+with+length+of+survival+in+esophageal+squamous-cell+carcinoma&amp;author=C+Wu&amp;volume=45&amp;publication_year=2013&amp;pages=632-8&amp;pmid=23644492&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Kiyotani K, et al.  A genome-wide association study identifies locus at 10q22 associated with clinical outcomes of adjuvant tamoxifen therapy for breast cancer patients in Japanese. <span><span class="ref-journal">Human molecular genetics. </span>2012;<span class="ref-vol">21</span>:1665–72.</span> [<a href="/pubmed/22180457" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+molecular+genetics&amp;title=A+genome-wide+association+study+identifies+locus+at+10q22+associated+with+clinical+outcomes+of+adjuvant+tamoxifen+therapy+for+breast+cancer+patients+in+Japanese&amp;author=K+Kiyotani&amp;volume=21&amp;publication_year=2012&amp;pages=1665-72&amp;pmid=22180457&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Shu XO, et al.  Novel genetic markers of breast cancer survival identified by a genome-wide association study. <span><span class="ref-journal">Cancer research. </span>2012;<span class="ref-vol">72</span>:1182–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3294129/">PMC free article</a>]</span> [<a href="/pubmed/22232737" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+research&amp;title=Novel+genetic+markers+of+breast+cancer+survival+identified+by+a+genome-wide+association+study&amp;author=XO+Shu&amp;volume=72&amp;publication_year=2012&amp;pages=1182-9&amp;pmid=22232737&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Innocenti F, et al.  A genome-wide association study of overall survival in pancreatic cancer patients treated with gemcitabine in CALGB 80303. <span><span class="ref-journal">Clinical cancer research : an official journal of the American Association for Cancer Research. </span>2012;<span class="ref-vol">18</span>:577–84.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3412624/">PMC free article</a>]</span> [<a href="/pubmed/22142827" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clinical+cancer+research+:+an+official+journal+of+the+American+Association+for+Cancer+Research&amp;title=A+genome-wide+association+study+of+overall+survival+in+pancreatic+cancer+patients+treated+with+gemcitabine+in+CALGB+80303&amp;author=F+Innocenti&amp;volume=18&amp;publication_year=2012&amp;pages=577-84&amp;pmid=22142827&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Wu C, et al.  Genome-wide association study of survival in patients with pancreatic adenocarcinoma. <span><span class="ref-journal">Gut. </span>2012</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3816124/">PMC free article</a>]</span> [<a href="/pubmed/23180869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Genome-wide+association+study+of+survival+in+patients+with+pancreatic+adenocarcinoma&amp;author=C+Wu&amp;publication_year=2012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Wu C, et al.  Genome-wide interrogation identifies YAP1 variants associated with survival of small-cell lung cancer patients. <span><span class="ref-journal">Cancer research. </span>2010;<span class="ref-vol">70</span>:9721–9.</span> [<a href="/pubmed/21118971" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+research&amp;title=Genome-wide+interrogation+identifies+YAP1+variants+associated+with+survival+of+small-cell+lung+cancer+patients&amp;author=C+Wu&amp;volume=70&amp;publication_year=2010&amp;pages=9721-9&amp;pmid=21118971&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Yang TP, et al.  Genevar: a database and Java application for the analysis and visualization of SNP-gene associations in eQTL studies. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2474–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2944204/">PMC free article</a>]</span> [<a href="/pubmed/20702402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Genevar:+a+database+and+Java+application+for+the+analysis+and+visualization+of+SNP-gene+associations+in+eQTL+studies&amp;author=TP+Yang&amp;volume=26&amp;publication_year=2010&amp;pages=2474-6&amp;pmid=20702402&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Grundberg E, et al.  Mapping cis- and trans-regulatory effects across multiple tissues in twins. <span><span class="ref-journal">Nature genetics. </span>2012;<span class="ref-vol">44</span>:1084–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3784328/">PMC free article</a>]</span> [<a href="/pubmed/22941192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Mapping+cis-+and+trans-regulatory+effects+across+multiple+tissues+in+twins&amp;author=E+Grundberg&amp;volume=44&amp;publication_year=2012&amp;pages=1084-9&amp;pmid=22941192&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Aubusson-Fleury A, et al.  The conserved centrosomal protein FOR20 is required for assembly of the transition zone and basal body docking at the cell surface. <span><span class="ref-journal">Journal of cell science. </span>2012;<span class="ref-vol">125</span>:4395–404.</span> [<a href="/pubmed/22718349" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+cell+science&amp;title=The+conserved+centrosomal+protein+FOR20+is+required+for+assembly+of+the+transition+zone+and+basal+body+docking+at+the+cell+surface&amp;author=A+Aubusson-Fleury&amp;volume=125&amp;publication_year=2012&amp;pages=4395-404&amp;pmid=22718349&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Sedjai F, et al.  Control of ciliogenesis by FOR20, a novel centrosome and pericentriolar satellite protein. <span><span class="ref-journal">Journal of cell science. </span>2010;<span class="ref-vol">123</span>:2391–401.</span> [<a href="/pubmed/20551181" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+cell+science&amp;title=Control+of+ciliogenesis+by+FOR20,+a+novel+centrosome+and+pericentriolar+satellite+protein&amp;author=F+Sedjai&amp;volume=123&amp;publication_year=2010&amp;pages=2391-401&amp;pmid=20551181&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Chng WJ, et al.  The centrosome index is a powerful prognostic marker in myeloma and identifies a cohort of patients that might benefit from aurora kinase inhibition. <span><span class="ref-journal">Blood. </span>2008;<span class="ref-vol">111</span>:1603–9.</span> [<a href="/pubmed/18006703" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=The+centrosome+index+is+a+powerful+prognostic+marker+in+myeloma+and+identifies+a+cohort+of+patients+that+might+benefit+from+aurora+kinase+inhibition&amp;author=WJ+Chng&amp;volume=111&amp;publication_year=2008&amp;pages=1603-9&amp;pmid=18006703&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Chng WJ, et al.  Clinical implication of centrosome amplification in plasma cell neoplasm. <span><span class="ref-journal">Blood. </span>2006;<span class="ref-vol">107</span>:3669–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1895774/">PMC free article</a>]</span> [<a href="/pubmed/16373658" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Clinical+implication+of+centrosome+amplification+in+plasma+cell+neoplasm&amp;author=WJ+Chng&amp;volume=107&amp;publication_year=2006&amp;pages=3669-75&amp;pmid=16373658&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Raab MS, et al.  GF-15, a novel inhibitor of centrosomal clustering, suppresses tumor cell growth in vitro and in vivo. <span><span class="ref-journal">Cancer research. </span>2012;<span class="ref-vol">72</span>:5374–85.</span> [<a href="/pubmed/22942257" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+research&amp;title=GF-15,+a+novel+inhibitor+of+centrosomal+clustering,+suppresses+tumor+cell+growth+in+vitro+and+in+vivo&amp;author=MS+Raab&amp;volume=72&amp;publication_year=2012&amp;pages=5374-85&amp;pmid=22942257&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Shiheido H, et al.  A phthalimide derivative that inhibits centrosomal clustering is effective on multiple myeloma. <span><span class="ref-journal">PloS one. </span>2012;<span class="ref-vol">7</span>:e38878.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3382596/">PMC free article</a>]</span> [<a href="/pubmed/22761710" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+one&amp;title=A+phthalimide+derivative+that+inhibits+centrosomal+clustering+is+effective+on+multiple+myeloma&amp;author=H+Shiheido&amp;volume=7&amp;publication_year=2012&amp;pages=e38878&amp;pmid=22761710&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Zollner S, Pritchard JK. Overcoming the winner's curse: estimating penetrance parameters from case-control data. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>:605–15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1852705/">PMC free article</a>]</span> [<a href="/pubmed/17357068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Overcoming+the+winner's+curse:+estimating+penetrance+parameters+from+case-control+data&amp;author=S+Zollner&amp;author=JK+Pritchard&amp;volume=80&amp;publication_year=2007&amp;pages=605-15&amp;pmid=17357068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Stewart DA, et al.  Factors predicting engraftment of autologous blood stem cells: CD34+ subsets inferior to the total CD34+ cell dose. <span><span class="ref-journal">Bone Marrow Transplant. </span>1999;<span class="ref-vol">23</span>:1237–43.</span> [<a href="/pubmed/10414909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=Factors+predicting+engraftment+of+autologous+blood+stem+cells:+CD34++subsets+inferior+to+the+total+CD34++cell+dose&amp;author=DA+Stewart&amp;volume=23&amp;publication_year=1999&amp;pages=1237-43&amp;pmid=10414909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Stewart DA, et al.  The CD34+90+ cell dose does not predict early engraftment of autologous blood stem cells as well as the total CD34+ cell dose. <span><span class="ref-journal">Bone Marrow Transplant. </span>2000;<span class="ref-vol">25</span>:435–40.</span> [<a href="/pubmed/10723588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone+Marrow+Transplant&amp;title=The+CD34+90++cell+dose+does+not+predict+early+engraftment+of+autologous+blood+stem+cells+as+well+as+the+total+CD34++cell+dose&amp;author=DA+Stewart&amp;volume=25&amp;publication_year=2000&amp;pages=435-40&amp;pmid=10723588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Martino A, et al.  Genetics and molecular epidemiology of multiple myeloma: the rationale for the IMMEnSE consortium (review) <span><span class="ref-journal">International journal of oncology. </span>2012;<span class="ref-vol">40</span>:625–38.</span> [<a href="/pubmed/22159523" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=International+journal+of+oncology&amp;title=Genetics+and+molecular+epidemiology+of+multiple+myeloma:+the+rationale+for+the+IMMEnSE+consortium+(review)&amp;author=A+Martino&amp;volume=40&amp;publication_year=2012&amp;pages=625-38&amp;pmid=22159523&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Criteria for the classification of monoclonal gammopathies, multiple myeloma and related disorders: a report of the International Myeloma Working Group  <span><span class="ref-journal">British journal of haematology. </span>2003;<span class="ref-vol">121</span>:749–57.</span> [<a href="/pubmed/12780789" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=British+journal+of+haematology&amp;volume=121&amp;publication_year=2003&amp;pages=749-57&amp;pmid=12780789&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Genomes Project, C. et al.  An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Browning BL, Browning SR. A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>:210–23.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668004/">PMC free article</a>]</span> [<a href="/pubmed/19200528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+unified+approach+to+genotype+imputation+and+haplotype-phase+inference+for+large+data+sets+of+trios+and+unrelated+individuals&amp;author=BL+Browning&amp;author=SR+Browning&amp;volume=84&amp;publication_year=2009&amp;pages=210-23&amp;pmid=19200528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Patterson N, Price AL, Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N+Patterson&amp;author=AL+Price&amp;author=D+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Grambsch PA, Therneau T. Proportional hazards tests and diagnostics based on weighted residuals. <span><span class="ref-journal">Biometrika. </span>1994;<span class="ref-vol">81</span>:515–526.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrika&amp;title=Proportional+hazards+tests+and+diagnostics+based+on+weighted+residuals&amp;author=PA+Grambsch&amp;author=T+Therneau&amp;volume=81&amp;publication_year=1994&amp;pages=515-526&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Pruim RJ, et al.  LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;volume=26&amp;publication_year=2010&amp;pages=2336-7&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-1&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Broyl A, et al.  Gene expression profiling for molecular classification of multiple myeloma in newly diagnosed patients. <span><span class="ref-journal">Blood. </span>2010;<span class="ref-vol">116</span>:2543–53.</span> [<a href="/pubmed/20574050" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Gene+expression+profiling+for+molecular+classification+of+multiple+myeloma+in+newly+diagnosed+patients&amp;author=A+Broyl&amp;volume=116&amp;publication_year=2010&amp;pages=2543-53&amp;pmid=20574050&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Chapman MA, et al.  Initial genome sequencing and analysis of multiple myeloma. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">471</span>:467–72.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3560292/">PMC free article</a>]</span> [<a href="/pubmed/21430775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Initial+genome+sequencing+and+analysis+of+multiple+myeloma&amp;author=MA+Chapman&amp;volume=471&amp;publication_year=2011&amp;pages=467-72&amp;pmid=21430775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Sim NL, et al.  SIFT web server: predicting effects of amino acid substitutions on proteins. <span><span class="ref-journal">Nucleic acids research. </span>2012;<span class="ref-vol">40</span>:W452–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3394338/">PMC free article</a>]</span> [<a href="/pubmed/22689647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+acids+research&amp;title=SIFT+web+server:+predicting+effects+of+amino+acid+substitutions+on+proteins&amp;author=NL+Sim&amp;volume=40&amp;publication_year=2012&amp;pages=W452-7&amp;pmid=22689647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Adzhubei I, Jordan DM, Sunyaev SR. Predicting functional effect of human missense mutations using PolyPhen-2. <span><span class="ref-journal">Current protocols in human genetics / editorial board, Jonathan L. Haines … [et al.] </span>2013;<span class="ref-vol">Chapter 7</span>(Unit7):20.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4480630/">PMC free article</a>]</span> [<a href="/pubmed/23315928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Current+protocols+in+human+genetics+/+editorial+board,+Jonathan+L.+Haines+…+[et+al.]&amp;title=Predicting+functional+effect+of+human+missense+mutations+using+PolyPhen-2&amp;author=I+Adzhubei&amp;author=DM+Jordan&amp;author=SR+Sunyaev&amp;volume=Chapter+7&amp;issue=Unit7&amp;publication_year=2013&amp;pages=20&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Coetzee SG, Rhie SK, Berman BP, Coetzee GA, Noushmehr H. FunciSNP: an R/bioconductor tool integrating functional non-coding data sets with genetic association studies to identify candidate regulatory SNPs. <span><span class="ref-journal">Nucleic acids research. </span>2012;<span class="ref-vol">40</span>:e139.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3467035/">PMC free article</a>]</span> [<a href="/pubmed/22684628" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+acids+research&amp;title=FunciSNP:+an+R/bioconductor+tool+integrating+functional+non-coding+data+sets+with+genetic+association+studies+to+identify+candidate+regulatory+SNPs&amp;author=SG+Coetzee&amp;author=SK+Rhie&amp;author=BP+Berman&amp;author=GA+Coetzee&amp;author=H+Noushmehr&amp;volume=40&amp;publication_year=2012&amp;pages=e139&amp;pmid=22684628&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4656791&amp;issue-id=261169&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140292074307024title">Abstract</a></li><li><a href="#S1title">INTRODUCTION</a></li><li><a href="#S2title">RESULTS</a></li><li><a href="#S7title">DISCUSSION</a></li><li><a href="#S8title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S15title">Acknowledgements</a></li><li><a href="#idm140292069600704title">Footnotes</a></li><li><a href="#idm140292069598848title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4656791/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4656791/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4656791/pdf/nihms-730494.pdf">PDF (688K)</a></li> | <li><a href="#" data-citationid="PMC4656791" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4656791%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4656791%2F&amp;text=Genome-wide%20association%20study%20identifies%20variants%20at%2016p13%20associated%20with%20survival%20in%20multiple%20myeloma%20patients" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4656791%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26198393" data-aiid="4656791" data-aid="4656791" data-iid="261169" data-domainid="319" data-domain="nihpa" data-accid="PMC4656791" data-md5="46b7f28e70dc6b8d81e27e79eed7ad7a"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:52:39-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CA341DD5866A100000000034401D4&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4656791%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4656791/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>