<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4004964/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A whole genome association study of neuroticism using DNA pooling</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4004964/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Molecular psychiatry"><meta name="citation_title" content="A whole genome association study of neuroticism using DNA pooling"><meta name="citation_authors" content="S Shifman, A Bhomra, S Smiley, NR Wray, MR James, NG Martin, JM Hettema, SS An, MC Neale, EJCG van den Oord, KS Kendler, X Chen, DI Boomsma, CM Middeldorp, JJ Hottenga, PE Slagboom, J Flint"><meta name="citation_date" content="March 2008"><meta name="citation_issue" content="3"><meta name="citation_volume" content="13"><meta name="citation_firstpage" content="302"><meta name="citation_doi" content="10.1038/sj.mp.4002048"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4004964/?report=abstract"><meta name="citation_pmid" content="17667963"><meta name="DC.Title" content="A whole genome association study of neuroticism using DNA pooling"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="S Shifman"><meta name="DC.Contributor" content="A Bhomra"><meta name="DC.Contributor" content="S Smiley"><meta name="DC.Contributor" content="NR Wray"><meta name="DC.Contributor" content="MR James"><meta name="DC.Contributor" content="NG Martin"><meta name="DC.Contributor" content="JM Hettema"><meta name="DC.Contributor" content="SS An"><meta name="DC.Contributor" content="MC Neale"><meta name="DC.Contributor" content="EJCG van den Oord"><meta name="DC.Contributor" content="KS Kendler"><meta name="DC.Contributor" content="X Chen"><meta name="DC.Contributor" content="DI Boomsma"><meta name="DC.Contributor" content="CM Middeldorp"><meta name="DC.Contributor" content="JJ Hottenga"><meta name="DC.Contributor" content="PE Slagboom"><meta name="DC.Contributor" content="J Flint"><meta name="DC.Date" content="2008 Mar"><meta name="DC.Identifier" content="10.1038/sj.mp.4002048"><meta name="DC.Language" content="en"><meta property="og:title" content="A whole genome association study of neuroticism using DNA pooling"><meta property="og:type" content="article"><meta property="og:description" content="We describe a multistage approach to identify single nucleotide polymorphisms (SNPs) associated with neuroticism, a personality trait that shares genetic determinants with major depression and anxiety disorders. Whole genome association with 452 574 SNPs ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4004964/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4004964/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4004964/pdf/nihms572905.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE890FBFDD3EFFA10000000002230151.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4004964%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4004964</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Mol Psychiatry</a></span>. Author manuscript; available in PMC 2014 Apr 30.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17667963" target="pmc_ext" ref="reftype=publisher&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Mol Psychiatry. 2008 Mar; 13(3): 302–312. </span></a></div><span class="fm-vol-iss-date">Published online 2007 Jul 31. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fsj.mp.4002048" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/sj.mp.4002048</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4004964</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS572905</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/17667963">17667963</a></div></div></div></div><h1 class="content-title">A whole genome association study of neuroticism using DNA pooling</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Shifman%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963717720928" co-class="co-affbox">S Shifman</a>,<sup>1</sup> <a href="/pubmed/?term=Bhomra%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963719357776" co-class="co-affbox">A Bhomra</a>,<sup>1</sup> <a href="/pubmed/?term=Smiley%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963717706768" co-class="co-affbox">S Smiley</a>,<sup>1</sup> <a href="/pubmed/?term=Wray%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963717704896" co-class="co-affbox">NR Wray</a>,<sup>2</sup> <a href="/pubmed/?term=James%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963717702912" co-class="co-affbox">MR James</a>,<sup>2</sup> <a href="/pubmed/?term=Martin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963717700928" co-class="co-affbox">NG Martin</a>,<sup>2</sup> <a href="/pubmed/?term=Hettema%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963717698944" co-class="co-affbox">JM Hettema</a>,<sup>3</sup> <a href="/pubmed/?term=An%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963717696960" co-class="co-affbox">SS An</a>,<sup>3</sup> <a href="/pubmed/?term=Neale%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963717694976" co-class="co-affbox">MC Neale</a>,<sup>3</sup> <a href="/pubmed/?term=van%20den%20Oord%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963718379488" co-class="co-affbox">EJCG van den Oord</a>,<sup>3</sup> <a href="/pubmed/?term=Kendler%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963718377504" co-class="co-affbox">KS Kendler</a>,<sup>3</sup> <a href="/pubmed/?term=Chen%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963718375520" co-class="co-affbox">X Chen</a>,<sup>3</sup> <a href="/pubmed/?term=Boomsma%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963718373536" co-class="co-affbox">DI Boomsma</a>,<sup>4</sup> <a href="/pubmed/?term=Middeldorp%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963718371552" co-class="co-affbox">CM Middeldorp</a>,<sup>4</sup> <a href="/pubmed/?term=Hottenga%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963718369568" co-class="co-affbox">JJ Hottenga</a>,<sup>4</sup> <a href="/pubmed/?term=Slagboom%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963718367584" co-class="co-affbox">PE Slagboom</a>,<sup>5</sup> and  <a href="/pubmed/?term=Flint%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963" class="affpopup" co-rid="_co_idm139963718365600" co-class="co-affbox">J Flint</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139963717720928"><h3 class="no_margin">S Shifman</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Shifman%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">S Shifman</a></div></div><div id="_co_idm139963719357776"><h3 class="no_margin">A Bhomra</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Bhomra%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">A Bhomra</a></div></div><div id="_co_idm139963717706768"><h3 class="no_margin">S Smiley</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Smiley%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">S Smiley</a></div></div><div id="_co_idm139963717704896"><h3 class="no_margin">NR Wray</h3><p><sup>2</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Wray%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">NR Wray</a></div></div><div id="_co_idm139963717702912"><h3 class="no_margin">MR James</h3><p><sup>2</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=James%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">MR James</a></div></div><div id="_co_idm139963717700928"><h3 class="no_margin">NG Martin</h3><p><sup>2</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Martin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">NG Martin</a></div></div><div id="_co_idm139963717698944"><h3 class="no_margin">JM Hettema</h3><p><sup>3</sup>Department of Psychiatry, Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Hettema%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">JM Hettema</a></div></div><div id="_co_idm139963717696960"><h3 class="no_margin">SS An</h3><p><sup>3</sup>Department of Psychiatry, Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, VA, USA</p><div>Find articles by <a href="/pubmed/?term=An%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">SS An</a></div></div><div id="_co_idm139963717694976"><h3 class="no_margin">MC Neale</h3><p><sup>3</sup>Department of Psychiatry, Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Neale%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">MC Neale</a></div></div><div id="_co_idm139963718379488"><h3 class="no_margin">EJCG van den Oord</h3><p><sup>3</sup>Department of Psychiatry, Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, VA, USA</p><div>Find articles by <a href="/pubmed/?term=van%20den%20Oord%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">EJCG van den Oord</a></div></div><div id="_co_idm139963718377504"><h3 class="no_margin">KS Kendler</h3><p><sup>3</sup>Department of Psychiatry, Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Kendler%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">KS Kendler</a></div></div><div id="_co_idm139963718375520"><h3 class="no_margin">X Chen</h3><p><sup>3</sup>Department of Psychiatry, Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">X Chen</a></div></div><div id="_co_idm139963718373536"><h3 class="no_margin">DI Boomsma</h3><p><sup>4</sup>Biological Psychology, Vrije Universiteit, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Boomsma%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">DI Boomsma</a></div></div><div id="_co_idm139963718371552"><h3 class="no_margin">CM Middeldorp</h3><p><sup>4</sup>Biological Psychology, Vrije Universiteit, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Middeldorp%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">CM Middeldorp</a></div></div><div id="_co_idm139963718369568"><h3 class="no_margin">JJ Hottenga</h3><p><sup>4</sup>Biological Psychology, Vrije Universiteit, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Hottenga%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">JJ Hottenga</a></div></div><div id="_co_idm139963718367584"><h3 class="no_margin">PE Slagboom</h3><p><sup>5</sup>Molecular Epidemiology, Leiden University Medical Centre, Leiden, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Slagboom%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">PE Slagboom</a></div></div><div id="_co_idm139963718365600"><h3 class="no_margin">J Flint</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Flint%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17667963">J Flint</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139963721818304_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139963721818304_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139963721818304_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Psychiatry, Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, VA, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Biological Psychology, Vrije Universiteit, Amsterdam, The Netherlands</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Molecular Epidemiology, Leiden University Medical Centre, Leiden, The Netherlands</div><div id="cor1">Correspondence: Dr J Flint, Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK. <a href="mailto:dev@null" data-email="ku.ca.xo.llew@fj" class="oemail">ku.ca.xo.llew@fj</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139963721818304_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17667963" target="pmc_ext" ref="reftype=publisher&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Mol Psychiatry</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4004964/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplemental material.</strong></div><div><a href="/pmc/articles/PMC4004964/bin/NIHMS572905-supplement-supplemental_material.pdf" data-ga-action="click_feat_suppl">NIHMS572905-supplement-supplemental_material.pdf</a><span style="color:gray"> (11K)</span></div><div class="small guid">GUID:&nbsp;7F3C3599-745B-4718-8D3B-756C2BA8F472</div></div></dd></dl></div><div id="idm139963725389408" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139963725389408title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">We describe a multistage approach to identify single nucleotide polymorphisms (SNPs) associated with neuroticism, a personality trait that shares genetic determinants with major depression and anxiety disorders. Whole genome association with 452 574 SNPs was performed on DNA pools from ~2000 individuals selected on extremes of neuroticism scores from a cohort of 88 142 people from southwest England. The most significant SNPs were then genotyped on independent samples to replicate findings. We were able to replicate association of one SNP within the PDE4D gene in a second sample collected by our laboratory and in a family-based test in an independent sample; however, the SNP was not significantly associated with neuroticism in two other independent samples. We also observed an enrichment of low <em>P</em>-values in known regions of copy number variations. Simulation indicates that our study had ~80% power to identify neuroticism loci in the genome with odds ratio (OR) &gt; 2, and ~50% power to identify small effects (OR = 1.5). Since we failed to find any loci accounting for more than 1% of the variance, the heritability of neuroticism probably arises from many loci each explaining much less than 1%. Our findings argue the need for much larger samples than anticipated in genetic association studies and that the biological basis of emotional disorders is extremely complex.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">neuroticism, whole genome association, DNA pooling, personality, quantitative trait, PDE4D</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">Ever since neuroticism was first proposed as personality trait that reflects a tendency toward states of negative affect,<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup> it has been included in nearly every theory of personality. Its importance as a psychological construct is further enhanced by its well documented correlation with common psychiatric disorders, that is, high levels of neuroticism predict the onset and subsequent episodes of major depression<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593178">2</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593177">5</a></sup> and are associated with symptoms of depression and anxiety in the general population.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593184">6</a>–<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593196">9</a></sup> The correlation between anxiety, major depression and neuroticism is, in part, due to the presence of shared genetic factors,<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593181">10</a>–<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593187">13</a></sup> an observation that has spurred attempts to map the genetic basis of neuroticism.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593172">14</a>–<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593160">16</a></sup> It is much easier to acquire the large samples necessary for genetic studies of neuroticism, which is assessed using a self-administered questionnaire, than it is to acquire large patient samples that require diagnostic interviews for accurate assessment of major depression and anxiety.</p><p id="P3">Genetic linkage analysis has had mixed success in mapping susceptibility loci for neuroticism. One study of extremely discordant and concordant siblings identified five loci that exceeded a 5% genome-wide significance threshold,<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593173">14</a></sup> but a second study, also using a selected sample, failed to identify any genome-wide significant signals.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593170">15</a></sup> One explanation for the difficulties encountered is that the proportion of genetic variance attributable to an individual locus is extremely small, so that the linkage studies are underpowered. A recent survey of some 50 meta-analyses and 752 individual studies concluded that the typical effect sizes of individual genetic variants for complex disease ranged from 1.2 to 1.6.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593195">17</a></sup></p><p id="P4">Whole genome association provides an alternative approach to detect small genetic effects. This is still an expensive option, but costs can be reduced by pooling equal amounts of DNA from each individual, separately for cases and controls, and estimating allele frequencies from the DNA pools (allelotyping). Even though allelotyping is replicated several times to increase accuracy, there is still a 10- to 100-fold decrease in cost compared to individual genotyping. DNA pooling can also be applied to quantitative traits in a method known as selective DNA pooling.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593169">18</a></sup> Selective DNA pooling combines DNA pooling and selective genotyping, which is the genotyping of individuals only from the extreme of the distribution, with the most informative trait values. While DNA pooling offers a substantial reduction in genotyping cost, it possesses some disadvantages, primarily in the loss of information (genotype and haplotype distributions) and by introducing estimation errors. Recent studies have shown the feasibility and accuracy of estimating single-nucleotide polymorphism (SNP) allele frequencies in DNA pools using microarrays containing thousands of SNPs,<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593192">19</a>–<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593190">23</a></sup> which makes it possible to perform whole genome association studies in a cost-effective way.</p><p id="P5" class="p p-last">We have collected a sample suitable for association studies of small genetic effects on neuroticism. We selected ~3600 individuals from a very large community cohort (<em>n</em> = 88 142) from southwest England, on the basis of the extremity of their neuroticism scores. Using a genomic control approach we have previously shown that stratification is unlikely to be a major confusion in the sample.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593162">24</a>–<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593214">26</a></sup> Here we describe a multistage approach to identify SNPs associated with neuroticism in this as well as in replication samples. Although our initial sample was phenotyped for other personality dimensions, the sample used in this study was selected for having extreme neuroticism score and thus it is not suitable for the study of other dimensions.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials and methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">The sample</h3><p id="P6" class="p p-first">Individuals were selected from the patient registers of general practices in four counties in southwest England, that is, Oxfordshire, Gloucestershire, Somerset and Berkshire (as described by Fullerton <em>et al.</em><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593202">14</a></sup> and Martin <em>et al.</em><sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593208">27</a></sup>). We chose patients in rural practices, so that the sample is likely to be relatively genetically homogeneous. A total of more than 88 141 individuals completed the revised Eysenck Personality Questionnaire (EPQ). The neuroticism (N) scale of the EPQ consists of 23 questions scored on a two-point scale (0 or 1).<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> N scores of the EPQ are correlated with similar measures on other personality measures such as the NEO PI-R N scale and the TPQ.<sup><a href="#R29" rid="R29" class=" bibr popnode">29</a>,<a href="#R30" rid="R30" class=" bibr popnode">30</a></sup> The total scores were regressed on age and sex and the residuals were standardized so that each individual’s score is expressed in s.d. units.</p><p id="P7">From this population, two independent samples were selected for genotyping according to the extremeness of EPQ–N scores. The first population, originally selected for neuroticism linkage mapping, consists of one member of extremely concordant and discordant sibling pairs (as described by Fullerton <em>et al.</em><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593191">14</a></sup> and Martin <em>et al.</em><sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593183">27</a></sup>), while the second population, collected for genetic association, consists of singletons (as described by Willis-Owen <em>et al.</em><sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593205">24</a></sup>). For the singletons, we received around 20%replies to our request of DNA samples. There was no difference in the distribution of N scores between subjects that replied and the ones that did not reply. We also received an equivalent rate of reply from high and low N extreme groups.</p><p id="P8" class="p p-last">We screened a total of 3589 individuals; the first group of 2054 individuals was used in the whole genome scan with DNA pools and the second group of 1534 individuals was used as a replication sample. Out of the 2054 individuals (1038 high N and 1016 low N) that were included in the pools, 1301 were women and 753 were men. The replication sample consisted of 932 women and 601 men (831 high N and 702 low N). This study was approved by a research ethics committee in the UK, and appropriate informed consent was obtained.</p></div><div id="S4" class="sec"><h3 id="S4title">DNA pools</h3><p id="P9" class="p p-first">DNA samples were collected and extracted from mouth swabs, quantified four times using the Pico-Green assay (Molecular Probes, Eugene, OR, USA) and tested by PCR. The samples were divided into eight groups according to N score (high and low) and gender (men and women). Equal amounts of DNA (150 ng) from each subject were mixed together to create 48 small pools, which on average were made from 43 individuals. Eight different pools were then created once by combining DNA from the small pools in proportion to the number of individuals in each pool. The eight pools are as follows: (1) men with high N score (<em>n</em> = 112), (2) men with low N score (<em>n</em> = 158), (3) men with very high N score (<em>n</em> = 245), (4) men with very low N score (<em>n</em> = 238), (5) women with high N score (<em>n</em> = 320), (6) women with low N score (<em>n</em> = 205), (7) women with very N high score (<em>n</em> = 340) and (8) women with very N low score (<em>n</em> = 436) (very high or low N scores are more than 1.5 s.d. from the mean score adjusted to age and sex (on average 2 s.d.); high and low N scores are between 1 and 1.5 s.d. from the mean score (on average 1.3 s.d.)).</p><p id="P10" class="p p-last">We tested the accuracy of the pools before starting the experiment with the Affymetrix arrays by individually genotyping five SNPs and allelotyping the pools seven times using the Sequenom platform. Frequencies were averaged and corrected for unequal detection of the two alleles based on the ratios in individual heterozygotes.<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593203">31</a></sup> The pools were found to be very accurate, with an s.d. from the expected frequency of 0.016 for each of the eight large pools. The s.d. attributed to the pools (construction error) without the measurement error is estimated to be 0.012 for each pool and 0.0059 for frequency estimate for the high and low N groups with four pools in each. The observed frequencies were highly correlated with the expected frequencies for the small pools and, as expected, higher for the eight large pools (small pools: <em>R</em><sup>2</sup> = 98.8%; large pools: <em>R</em><sup>2</sup> = 99.7%).</p></div><div id="S5" class="sec"><h3 id="S5title">Allelotyping with Affymetrix chips</h3><p id="P11" class="p p-first">We used the Affymetrix 100 and 500K mapping arrays sets. The arrays and the number of SNPs on each array are as follows: Nsp array with 255 931 SNPs, Sty with 238 304 SNPs, Hind with 57 244 SNPs and Xba with 58 960 SNPs. The total number of unique SNPs with dbSNP rs ID numbers is 577 889 (31 462 SNPs are present on more than one array). Out of those, 452 574 SNPs have a minor allele frequency above 5%.</p><p id="P12" class="p p-last">According to the manufacturer’s protocol, 250 ng from each pool was digested with a restriction enzyme, ligated to an adaptor, amplified, fragmented, labeled and hybridized to the chip. Washing and staining was performed using the Fluidics Station 450 and scanned using the GeneChip Scanner 3000 7G. Each DNA pool was amplified and hybridized five times to different chips of the same type. The detection rate (MDR) for the 100K arrays was above 99% for all arrays. For the early access to 500K arrays the detection rate (MDR) was not available. Water, instead of DNA, was used as a negative control to test for contamination and the manufacturer’s individual reference DNA as a positive control to test for assay performance. Accuracy of genotyping call of the reference DNA was 99.8%.</p></div><div id="S6" class="sec"><h3 id="S6title">Data analysis of DNA pools</h3><p id="P13" class="p p-first-last">An allele frequency estimate was calculated based on the relative intensities of the two alleles (know as the relative allele score; RAS) for each SNP and for each of the eight pools.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593186">32</a></sup> The frequency estimate in each pool is the average across the five measurements. Frequency estimates for high and low N were obtained from the average frequency in the four pools belonging to each category. An SNP-specific variance was estimated by calculating the variance of the five measurements for each DNA pool across the eight pools <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" overflow="scroll"><mo stretchy="false">(</mo><msubsup><mi mathvariant="normal">σ</mi><mtext>pool</mtext><mn>2</mn></msubsup><mo stretchy="false">)</mo></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-1-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" overflow="scroll"><mo stretchy="false">(</mo><msubsup><mi mathvariant="normal">σ</mi><mtext>pool</mtext><mn>2</mn></msubsup><mo stretchy="false">)</mo></math></script>. The allele frequencies from HapMap Centre d’Etude du Polymorphisme Humain (CEPH) trios were used as an estimate for the allele frequencies in the population (<em><span class="MathJax_Preview"><math><mover accent="true"><mi>P</mi><mo>̂</mo></mover></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-2-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-2"><math><mover accent="true"><mi>P</mi><mo>̂</mo></mover></math></script></em>). We modified the standard chi-squared statistic to include the error variance introduced by the DNA pooling procedure. The test statistic is expressed as:
</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="block" overflow="scroll"><mi>Z</mi><mo>=</mo><mfrac><mrow><msub><mi>P</mi><mi mathvariant="normal">H</mi></msub><mo>−</mo><msub><mi>P</mi><mi mathvariant="normal">L</mi></msub></mrow><mrow><msqrt><mi>P̂</mi><mo>×</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mi>P̂</mi><mo stretchy="false">)</mo><mo>×</mo><mo stretchy="true">[</mo><mfrac><mn>1</mn><mrow><mn>2</mn><msub><mi>N</mi><mi mathvariant="normal">H</mi></msub></mrow></mfrac><mo>+</mo><mfrac><mn>1</mn><mrow><mn>2</mn><msub><mi>N</mi><mi mathvariant="normal">L</mi></msub></mrow></mfrac><mo stretchy="true">]</mo><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mtext>pool</mtext><mn>2</mn></msubsup><mo>×</mo><mo stretchy="true">[</mo><mfrac><mn>1</mn><mrow><mi>n</mi><msub><mi>p</mi><mi mathvariant="normal">H</mi></msub></mrow></mfrac><mo>+</mo><mfrac><mn>1</mn><mrow><mi>n</mi><msub><mi>p</mi><mi mathvariant="normal">L</mi></msub></mrow></mfrac><mo stretchy="true">]</mo></msqrt></mrow></mfrac></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="block" overflow="scroll"><mi>Z</mi><mo>=</mo><mfrac><mrow><msub><mi>P</mi><mi mathvariant="normal">H</mi></msub><mo>−</mo><msub><mi>P</mi><mi mathvariant="normal">L</mi></msub></mrow><mrow><msqrt><mi>P̂</mi><mo>×</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mi>P̂</mi><mo stretchy="false">)</mo><mo>×</mo><mo stretchy="true">[</mo><mfrac><mn>1</mn><mrow><mn>2</mn><msub><mi>N</mi><mi mathvariant="normal">H</mi></msub></mrow></mfrac><mo>+</mo><mfrac><mn>1</mn><mrow><mn>2</mn><msub><mi>N</mi><mi mathvariant="normal">L</mi></msub></mrow></mfrac><mo stretchy="true">]</mo><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mtext>pool</mtext><mn>2</mn></msubsup><mo>×</mo><mo stretchy="true">[</mo><mfrac><mn>1</mn><mrow><mi>n</mi><msub><mi>p</mi><mi mathvariant="normal">H</mi></msub></mrow></mfrac><mo>+</mo><mfrac><mn>1</mn><mrow><mi>n</mi><msub><mi>p</mi><mi mathvariant="normal">L</mi></msub></mrow></mfrac><mo stretchy="true">]</mo></msqrt></mrow></mfrac></math></script></div></div><p>
where <em>P</em><sub>H</sub> is the frequency in high N pools and <em>P</em><sub>L</sub> in low N pools, <em>N</em><sub>H</sub> is the number of individuals in high N pools and <em>N</em><sub>L</sub> is the number of individuals in low N pools and <em>np</em><sup>H</sup> and <em>np</em><sup>L</sup> are the number of measurements for high and low N pools. <em>P</em>-values were calculated using a chi-squared distribution for all SNPs with a minor allele frequency above 0.05. We did not attempt to produce a test statistic with desirable statistical properties. The log <em>P</em> values calculated based on this statistic do not correspond to the expected type I error and were only used to rank and prioritize the SNPs for individual genotyping. The empirical distribution of the pools’ log <em>P</em> values was assessed using a simulation with a null effect (see below).</p></div><div id="S7" class="sec"><h3 id="S7title">Simulation</h3><p id="P14" class="p p-first">To estimate a genome-wide significance threshold, we generated 1000 panels of 4000 chromosomes by random sampling with the replacement from the phased HapMap data (<em>n</em> = 120 from unrelated individuals in CEU).<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593193">33</a></sup> For each panel we calculated the significance of allele frequency differences between two groups of 2000 chromosomes for all SNPs with a minor allele frequency greater than 0.05, included in the 500 and 100K Affymetrix arrays. The most significant value in each panel was recorded.</p><p id="P15">For each SNP in the DNA pooling simulation the allele frequency in the HapMap CEPH sample was used. To simulate the errors introduced by the DNA pooling procedure we first generated a random normal deviate from the true HapMap frequency using the estimated construction error variance. We then assigned a number of measurements to each simulated pool with an error based on the SNP-specific measurement error and assumed a normal distribution.</p><p id="P16" class="p p-last">To evaluate the power of individual genotyping and DNA pooling we designated 2556 random SNPs from the HapMap (whether they were included in the arrays or not) to be the causal variant with an effect size, measured in terms of odds ratios (ORs), of 1.2, 1.3, 1.5, 1.7, 2 and 3. All the SNPs in HapMap phase II with minor allele frequency (MAF) &gt; 0.05, in the CEPH population,<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593200">33</a></sup> had an equal chance (0.1%) of being the causal variant. Based on the frequency of the causal SNP, we assigned the allele frequencies to two groups with a frequency difference that corresponded to the effect sizes. We sampled phased chromosomes with different probabilities based on the allele frequencies. We calculated the significance of the allelic χ<sup>2</sup> test at the causal SNP and for SNPs in a 100 kb window surrounding the causal SNP that are included in the Affymetrix arrays. For the DNA pooling we introduced two sources of error, that is, construction error and measurement error, as described above. Similar to our study design, we simulated 20 measurements for each of the two groups for all SNPs in the 100 kb window. We calculated the significance of each SNP in the same way used for the real allelotyping data. For the individual genotyping simulation we recorded the minimum <em>P</em>-value across all SNPs in the window. For the DNA pooling we simulated a design where all SNPs with log <em>P</em> &gt; 4 (the negative logarithm, base 10, of the <em>P</em>-value) were individually genotyped. To simulate this design, we recorded the minimum <em>P</em>-value of the individual genotypes across all SNPs in the window that exceeded a log <em>P</em> of 4 in the DNA pooling simulation.</p></div><div id="S8" class="sec sec-last"><h3 id="S8title">Individual genotyping</h3><p id="P17" class="p p-first-last">Individual genotyping was performed using the Sequenom MassArray platform according to the manufacturer’s instructions (SEQUENOM, San Diego, CA, USA). The DNA samples in the pools were genotyped using the iPLEX assay with up to 29 SNPs multiplexed together. The replication sample was genotyped using the homogenous MassEXTEND (hME) assay. Chips were read using the Bruker Autoflex Mass Spectrometer system (Bruker-Sequenom, San Diego, CA, USA). As a quality control for the iPLEX assay we genotyped 69 HapMap CEPH DNA samples with known genotypes. The concordance level between the genotypes was on average 99.4%. Hardy–Weinberg (HW) equilibrium was tested on all SNPs, but was not used automatically to exclude SNPs because all samples used are selected from the extreme of the population and might show deviation from HW in SNPs associated with the trait. To check the genotyping quality of SNP rs702543, we compared 634 genotypes obtained using the HME and iPLEX assays. The concordance level was 100%. Haplotypes were analyzed based on a score statistic using ‘haplo.stats’ package for R.<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593210">34</a></sup></p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Results</h2><div id="S10" class="sec sec-first"><h3 id="S10title">Sensitivity and specificity of DNA pooling</h3><p id="P18" class="p p-first">The design and interpretation of the DNA pooling experiment require an assessment of the method’s likely power, that is, its sensitivity and specificity. If we were to genotype all 2000 individuals individually, we would be able to detect an effect explaining 1, 0.5 and 0.1% (ORs of 1.72, 1.47 and 1.19) of the additive genetic variance with power of 0.94, 0.34 and 0.001, respectively (assuming an alpha of 1.66 × 10<sup>−7</sup> (300 000 independent tests, a complete linkage disequilibrium (LD) between the marker and the causative SNP and a trait increasing allele frequency of 0.2)). If we test only 20 markers, for example, in a replication sample, our power is 1, 0.96 and 0.19 for 1, 0.5 and 0.1% effects, respectively.</p><p id="P19">To investigate the pooling strategy we simulated the design of the study in which we used eight different pools, each allelotyped five times (that is, 20 measurements for each group). Using samples of CEPH individuals genotyped as part of the HapMap project,<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593165">33</a></sup> we established that we could accurately simulate the pooling experiment (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary material</a> for a full description). We first simulated a situation with a null effect and found that the genome-wide 5% threshold for individual genotyping is 1.7 × 10<sup>−7</sup> (log <em>P</em> = 6.77), which is equivalent to a 5% threshold after applying a Bonferroni correction for 294 071 independent tests. Then, by simulation, we determined the power to detect an association by genotyping directly the simulated causal SNP, by individual genotyping of SNPs on the Affymetrix arrays (500 and 100K with MAF&gt; 0.05) and using a DNA pooling strategy. We designed a random SNP from the HapMap to be a causal SNP and we looked at SNPs in the array within a 100 kb window. By simulating an SNP from the HapMap as a causal SNP we include in our simulation the effect of the coverage of the arrays (the LD between the SNPs on the array and SNPs in the genome, represented by HapMap data) and the possibility that more than one SNP on the array is in LD with the causal SNP.</p><p id="P20" class="p p-last">We looked at two study designs. In the first design, we assessed our ability to detect an SNP that exceeds the 5% genome-wide threshold (log <em>P</em> = 6.77). In the second design, we assessed what would happen when we test in a second sample all SNPs that exceed a log <em>P</em> of 3 in the analysis of the first sample. We calculated the power using different effect sizes of OR between 1.2 and 3 (<a href="/pmc/articles/PMC4004964/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). For small effect sizes, that is, ORs of 1.2, 1.3 and 1.5, the power to exceed the 5% genome-wide threshold using the DNA pooling strategy is 0, 3 and 31%, respectively. This is around 75% of the power achieved by individual genotyping, but incurs only 2% of the cost (40 × 4 arrays relative to 2000 × 4). For the study design that includes testing all log <em>P</em> &gt; 3 SNPs in a second sample, power is the probability of inclusion in the list for replication without taking into account the power of the second sample to replicate the findings. The power of DNA pooling with the second design is 4, 16 and 47% for ORs of 1.2, 1.3 and 1.5, respectively. For this design and with these ORs the reduction in power ranges between 41 and 79% compared to individual genotyping.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P46">Power of DNA pooling strategy compared to individual genotyping</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139963725276576" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">OR</th><th rowspan="1" colspan="1" align="right">Functional<br>SNP (%)</th><th rowspan="1" colspan="1" align="right">Individual<br>genotyping (%)</th><th rowspan="1" colspan="1" align="right">DNA<br>pooling (%)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"><em>5% threshold</em></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;1.2</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">0</td><td rowspan="1" colspan="1" align="right">0</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;1.3</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">3</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;1.5</td><td rowspan="1" colspan="1" align="right">51</td><td rowspan="1" colspan="1" align="right">41</td><td rowspan="1" colspan="1" align="right">31</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;1.7</td><td rowspan="1" colspan="1" align="right">80</td><td rowspan="1" colspan="1" align="right">68</td><td rowspan="1" colspan="1" align="right">56</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;2</td><td rowspan="1" colspan="1" align="right">93</td><td rowspan="1" colspan="1" align="right">82</td><td rowspan="1" colspan="1" align="right">72</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;3</td><td rowspan="1" colspan="1" align="right">99</td><td rowspan="1" colspan="1" align="right">90</td><td rowspan="1" colspan="1" align="right">85</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>Log</em> P &gt;<em>3</em></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;1.2</td><td rowspan="1" colspan="1" align="right">19</td><td rowspan="1" colspan="1" align="right">19</td><td rowspan="1" colspan="1" align="right">4</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;1.3</td><td rowspan="1" colspan="1" align="right">53</td><td rowspan="1" colspan="1" align="right">49</td><td rowspan="1" colspan="1" align="right">16</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;1.5</td><td rowspan="1" colspan="1" align="right">88</td><td rowspan="1" colspan="1" align="right">80</td><td rowspan="1" colspan="1" align="right">47</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;1.7</td><td rowspan="1" colspan="1" align="right">97</td><td rowspan="1" colspan="1" align="right">90</td><td rowspan="1" colspan="1" align="right">66</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;2</td><td rowspan="1" colspan="1" align="right">99</td><td rowspan="1" colspan="1" align="right">92</td><td rowspan="1" colspan="1" align="right">76</td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;3</td><td rowspan="1" colspan="1" align="right">100</td><td rowspan="1" colspan="1" align="right">95</td><td rowspan="1" colspan="1" align="right">87</td></tr></tbody></table></div><div id="largeobj_idm139963725276576" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4004964/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P47" class="p p-first-last">Abbreviations: OR, odds ratio; SNP, single-nucleotide polymorphism; 5% threshold, power to detect the association with significance above the 5% genome-wide threshold; log <em>P</em> &gt; 3, percentage of cases with log <em>P</em> &gt; 3, in a design that includes testing the log <em>P</em> &gt; 3 SNPs in a replication sample.</p></div></div></div></div><div id="S11" class="sec"><h3 id="S11title">Pooling and individual genotyping</h3><p id="P21" class="p p-first-last">We screened 2054 extreme scoring individuals in eight DNA pools for an association with neuroticism (<a href="/pmc/articles/PMC4004964/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). We calculated an association log <em>P</em> value for 452 574 SNPs (with MAF &gt; 5%) based on the frequency differences between high and low N pools, the SNP allele frequency and the SNP-specific measurement error. SNPs with MAF below 0.05 were excluded because of the limited power to detect a significant association and the decrease in the accuracy of DNA pooling with rare variants. We ranked the SNPs based on the log <em>P</em> values and used a log <em>P</em>&gt;4 as the cutoff for individual genotyping. We excluded 78 SNPs because the allele frequency estimates had been biased by an outlier in one of the pools. Four hundred and forty SNPs were considered for individual genotyping. Out of 440 SNPs, the assays of 405 SNPs were successfully designed and were used to genotype a subset of the sample (759 individuals). We calculated the significance for allele frequency differences between high and low N for sex-averaged and sexspecific frequencies for each SNP. Forty-two SNPs (10%) had a <em>P</em>-value between 0.05 and 0.01, and 28 SNPs (7%) had a <em>P</em>-value &lt; 0.01. We genotyped the whole sample with 110 SNPs that showed sex-specific or sex-averaged <em>P</em>-values &lt; 0.1. No SNP had a <em>P</em>-value that remained significant after correcting for multiple testing (300 000 tests). Eighteen SNPs had a <em>P</em>-value &lt;10<sup>−3</sup>, three SNPs had a <em>P</em>-value &lt; 10<sup>−4</sup> and two had a <em>P</em>-value &lt; 10<sup>−5</sup>. As expected, our results using DNA pooling (with a log <em>P</em> &gt; 4 as cutoff) produced a significant deficiency in <em>P</em>-values &lt; 10<sup>−3</sup> compared to what would be expected by chance if all SNPs were individually genotyped (~300 expected by chance). However, we do not see a significant deficiency of <em>P</em>-values &lt; 10<sup>−5</sup> (~3 expected by chance), which indicates that our method would have potentially picked SNPs with genome-wide significant values after Bonferroni correction for multiple tests (<em>P</em>-values &lt; 0.05/300 000 = 1.67 × 10<sup>−7</sup>). The highest scoring SNPs and their associated genes are shown in <a href="/pmc/articles/PMC4004964/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4004964/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139963725283776" class="figure"><a href="/pmc/articles/PMC4004964/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4004964_nihms572905f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms572905f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4004964/bin/nihms572905f1.jpg"></a></div><div id="largeobj_idm139963725283776" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4004964/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4004964/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="P44">Distribution of N scores as standardized residuals of the sum scores after regression on age and sex (88 141 individuals). The selection of individuals with extreme score is presented below the histogram. The black ticks mark the sample used in the DNA pools, and the gray represents the sample used in the replication sample.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P48">Results of individual genotyping</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139963717654160" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="4" rowspan="1" valign="top" align="center">SNP</th><th rowspan="5" colspan="1" valign="top" align="center">DNA<br>pooling<br>rank<sup>a</sup></th><th colspan="6" rowspan="1" valign="top" align="center"><em>P</em>-values for allele frequencies differences</th></tr><tr><th colspan="4" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="6" rowspan="1" valign="bottom" align="left"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP<br>identifier</th><th rowspan="1" colspan="1" valign="top" align="center">Gene<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="right">Chr<sup>c</sup></th><th rowspan="1" colspan="1" valign="top" align="right">Position</th><th colspan="3" rowspan="1" valign="top" align="center">Genotyping<sup>d</sup></th><th colspan="3" rowspan="1" valign="top" align="center">Replication<sup>e</sup></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="right"></th><th rowspan="1" colspan="1" align="right"></th><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="right"></th><th rowspan="1" colspan="1" align="right"></th><th rowspan="1" colspan="1" valign="top" align="center">All</th><th rowspan="1" colspan="1" valign="top" align="center">M</th><th rowspan="1" colspan="1" valign="top" align="center">F</th><th rowspan="1" colspan="1" valign="top" align="center">All</th><th rowspan="1" colspan="1" valign="top" align="center">M</th><th rowspan="1" colspan="1" valign="top" align="center">F</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs322239</td><td rowspan="1" colspan="1" align="center">PTN</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">136413862</td><td rowspan="1" colspan="1" align="right">12</td><td rowspan="1" colspan="1" align="center">4.8E–06</td><td rowspan="1" colspan="1" align="center">1.1E–04</td><td rowspan="1" colspan="1" align="center">5.7E–03</td><td rowspan="1" colspan="1" align="center">8.8E–01</td><td rowspan="1" colspan="1" align="center">7.4E–01</td><td rowspan="1" colspan="1" align="center">6.5E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4841017</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">8614503</td><td rowspan="1" colspan="1" align="right">53</td><td rowspan="1" colspan="1" align="center">6.5E–06</td><td rowspan="1" colspan="1" align="center">2.3E–05</td><td rowspan="1" colspan="1" align="center">1.5E–02</td><td rowspan="1" colspan="1" align="center">2.3E–01</td><td rowspan="1" colspan="1" align="center">3.7E–01</td><td rowspan="1" colspan="1" align="center">4.1E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">rs17385253</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">34564102</td><td rowspan="1" colspan="1" align="right">21</td><td rowspan="1" colspan="1" align="center">3.3E–05</td><td rowspan="1" colspan="1" align="center">7.7E–03</td><td rowspan="1" colspan="1" align="center">1.5E–03</td><td rowspan="1" colspan="1" align="center">2.8E–01</td><td rowspan="1" colspan="1" align="center">1.3E–01</td><td rowspan="1" colspan="1" align="center">8.5E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9329165<sup>f</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">8614052</td><td rowspan="1" colspan="1" align="right">160</td><td rowspan="1" colspan="1" align="center">3.8E–05</td><td rowspan="1" colspan="1" align="center">3.0E–05</td><td rowspan="1" colspan="1" align="center">4.9E–02</td><td rowspan="1" colspan="1" align="center">2.1E–01</td><td rowspan="1" colspan="1" align="center">9.4E–01</td><td rowspan="1" colspan="1" align="center">9.5E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">rs10483573</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="right">14</td><td rowspan="1" colspan="1" align="right">46932227</td><td rowspan="1" colspan="1" align="right">258</td><td rowspan="1" colspan="1" align="center">5.6E–05</td><td rowspan="1" colspan="1" align="center">3.6E–03</td><td rowspan="1" colspan="1" align="center">3.2E–03</td><td rowspan="1" colspan="1" align="center">6.1E–01</td><td rowspan="1" colspan="1" align="center">7.8E–01</td><td rowspan="1" colspan="1" align="center">3.9E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">rs10504830</td><td rowspan="1" colspan="1" align="center">CNBD1</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">88448321</td><td rowspan="1" colspan="1" align="right">115</td><td rowspan="1" colspan="1" align="center">1.1E–04</td><td rowspan="1" colspan="1" align="center">6.8E–04</td><td rowspan="1" colspan="1" align="center">2.2E–02</td><td rowspan="1" colspan="1" align="center">5.7E–01</td><td rowspan="1" colspan="1" align="center">1.4E–01</td><td rowspan="1" colspan="1" align="center">6.8E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">rs13290746<sup>g</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="right">9</td><td rowspan="1" colspan="1" align="right">34858377</td><td rowspan="1" colspan="1" align="right">146</td><td rowspan="1" colspan="1" align="center">1.5E–04</td><td rowspan="1" colspan="1" align="center">3.6E–02</td><td rowspan="1" colspan="1" align="center">1.6E–03</td><td rowspan="1" colspan="1" align="center">3.8E–01</td><td rowspan="1" colspan="1" align="center">2.8E–01</td><td rowspan="1" colspan="1" align="center">7.8E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9959800</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="right">18</td><td rowspan="1" colspan="1" align="right">52115138</td><td rowspan="1" colspan="1" align="right">48</td><td rowspan="1" colspan="1" align="center">1.7E–04</td><td rowspan="1" colspan="1" align="center">1.1E–02</td><td rowspan="1" colspan="1" align="center">6.0E–03</td><td rowspan="1" colspan="1" align="center">1.4E–01</td><td rowspan="1" colspan="1" align="center">8.0E–01</td><td rowspan="1" colspan="1" align="center">3.8E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">rs10797812</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">179716254</td><td rowspan="1" colspan="1" align="right">159</td><td rowspan="1" colspan="1" align="center">2.4E–04</td><td rowspan="1" colspan="1" align="center">2.2E–02</td><td rowspan="1" colspan="1" align="center">4.1E–03</td><td rowspan="1" colspan="1" align="center">5.1E–01</td><td rowspan="1" colspan="1" align="center">2.6E–01</td><td rowspan="1" colspan="1" align="center">7.2E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4875610</td><td rowspan="1" colspan="1" align="center">CSMD1</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">3165318</td><td rowspan="1" colspan="1" align="right">430</td><td rowspan="1" colspan="1" align="center">4.1E–04</td><td rowspan="1" colspan="1" align="center">2.6E–02</td><td rowspan="1" colspan="1" align="center">6.3E–03</td><td rowspan="1" colspan="1" align="center">6.5E–01</td><td rowspan="1" colspan="1" align="center">4.7E–01</td><td rowspan="1" colspan="1" align="center">9.9E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2123315<sup>f</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">18642838</td><td rowspan="1" colspan="1" align="right">136</td><td rowspan="1" colspan="1" align="center">4.3E–04</td><td rowspan="1" colspan="1" align="center">2.0E–01</td><td rowspan="1" colspan="1" align="center">3.8E–03</td><td rowspan="1" colspan="1" align="center">9.6E–02</td><td rowspan="1" colspan="1" align="center">5.1E–01</td><td rowspan="1" colspan="1" align="center">1.3E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">rs702543</td><td rowspan="1" colspan="1" align="center">PDE4D</td><td rowspan="1" colspan="1" align="right">5</td><td rowspan="1" colspan="1" align="right">58878531</td><td rowspan="1" colspan="1" align="right">184</td><td rowspan="1" colspan="1" align="center">8.2E–04</td><td rowspan="1" colspan="1" align="center">9.8E–04</td><td rowspan="1" colspan="1" align="center">9.5E–02</td><td rowspan="1" colspan="1" align="center">7.7E–04</td><td rowspan="1" colspan="1" align="center">2.7E–01</td><td rowspan="1" colspan="1" align="center">5.9E–04</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>rs7666238</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">142453368</td><td rowspan="1" colspan="1" align="right">315</td><td rowspan="1" colspan="1" align="center">9.0E–04</td><td rowspan="1" colspan="1" align="center">2.1E–03</td><td rowspan="1" colspan="1" align="center">6.9E–02</td><td rowspan="1" colspan="1" align="center">6.1E–01</td><td rowspan="1" colspan="1" align="center">1.8E–01</td><td rowspan="1" colspan="1" align="center">7.9E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9431663</td><td rowspan="1" colspan="1" align="center">TRIM67</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">227620956</td><td rowspan="1" colspan="1" align="right">211</td><td rowspan="1" colspan="1" align="center">1.2E–03</td><td rowspan="1" colspan="1" align="center">8.2E–01</td><td rowspan="1" colspan="1" align="center">1.2E–04</td><td rowspan="1" colspan="1" align="center">9.7E–01</td><td rowspan="1" colspan="1" align="center">8.6E–02</td><td rowspan="1" colspan="1" align="center">1.6E–01</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>rs7594674</strong><sup>g</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">67191611</td><td rowspan="1" colspan="1" align="right">432</td><td rowspan="1" colspan="1" align="center">2.6E–03</td><td rowspan="1" colspan="1" align="center">4.3E–01</td><td rowspan="1" colspan="1" align="center">6.5E–04</td><td rowspan="1" colspan="1" align="center">3.9E–01</td><td rowspan="1" colspan="1" align="center">4.0E–01</td><td rowspan="1" colspan="1" align="center">7.9E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">rs873989</td><td rowspan="1" colspan="1" align="center">DAB1</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">58268619</td><td rowspan="1" colspan="1" align="right">148</td><td rowspan="1" colspan="1" align="center">3.5E–03</td><td rowspan="1" colspan="1" align="center">8.2E–01</td><td rowspan="1" colspan="1" align="center">1.3E–04</td><td rowspan="1" colspan="1" align="center">9.3E–01</td><td rowspan="1" colspan="1" align="center">4.9E–01</td><td rowspan="1" colspan="1" align="center">5.2E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">rs401897</td><td rowspan="1" colspan="1" align="center">C20orf32</td><td rowspan="1" colspan="1" align="right">20</td><td rowspan="1" colspan="1" align="right">54463165</td><td rowspan="1" colspan="1" align="right">394</td><td rowspan="1" colspan="1" align="center">5.6E–03</td><td rowspan="1" colspan="1" align="center">6.5E–04</td><td rowspan="1" colspan="1" align="center">3.4E–01</td><td rowspan="1" colspan="1" align="center">5.2E–01</td><td rowspan="1" colspan="1" align="center">8.3E–01</td><td rowspan="1" colspan="1" align="center">4.4E–01</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>rs201997</strong></td><td rowspan="1" colspan="1" align="center">ADAM18</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">39703325</td><td rowspan="1" colspan="1" align="right">186</td><td rowspan="1" colspan="1" align="center">6.4E–03</td><td rowspan="1" colspan="1" align="center">2.7E–04</td><td rowspan="1" colspan="1" align="center">5.3E–01</td><td rowspan="1" colspan="1" align="center">1.2E–01</td><td rowspan="1" colspan="1" align="center">6.0E–02</td><td rowspan="1" colspan="1" align="center">6.4E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1452788</td><td rowspan="1" colspan="1" align="center">TCF4</td><td rowspan="1" colspan="1" align="right">18</td><td rowspan="1" colspan="1" align="right">51268302</td><td rowspan="1" colspan="1" align="right">384</td><td rowspan="1" colspan="1" align="center">6.5E–03</td><td rowspan="1" colspan="1" align="center">6.7E–01</td><td rowspan="1" colspan="1" align="center">9.3E–04</td><td rowspan="1" colspan="1" align="center">5.7E–01</td><td rowspan="1" colspan="1" align="center">3.9E–01</td><td rowspan="1" colspan="1" align="center">9.7E–01</td></tr></tbody></table></div><div id="largeobj_idm139963717654160" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4004964/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="P49" class="p p-first-last">Abbreviation: SNPs, single-nucleotide polymorphisms.</p></div><div id="TFN3"><p id="P50" class="p p-first-last">SNPs within a copy number variation region are in bold.</p></div><div id="TFN4"><sup>a</sup>Rank of SNPs based on log P results from the DNA pooling experiment.</div><div id="TFN5"><sup>b</sup>SNP associate gene based on NCBI dbSNP.</div><div id="TFN6"><sup>c</sup>Chromosome.</div><div id="TFN7"><sup>d</sup>Results from individual genotyping the sample in the pools.</div><div id="TFN8"><sup>e</sup>Results from genotyping the replication sample.</div><div id="TFN9"><sup>f</sup>Genotyping distribution deviates from Hardy–Weinberg equilibrium in sample included in the pools (<em>P</em>-value &lt; 0.0026).</div><div id="TFN10"><sup>g</sup>Genotyping distribution deviates from Hardy–Weinberg equilibrium in both samples using two different SNP genotyping platforms (iPLEX and HME).</div></div></div></div><div id="S12" class="sec"><h3 id="S12title">Haplotype analysis</h3><p id="P22" class="p p-first-last">To examine the possibility that multiple SNPs, each of small effect, might lie within the same gene, we searched for genes or intervals of 50 kb or less containing one SNP with a log <em>P</em> greater than 4 (based on the pooling analysis) and at least one other SNP with a log <em>P</em> greater than 2. Since LD would explain the presence of clusters of SNPs with high log <em>P</em> values, we excluded the pairs of SNPs whose <em>r</em><sup>2</sup> values (from the HapMap project) were greater than 0.6. We identified 81 SNPs from 34 regions for individual genotyping. Nineteen clustered SNPs showed a <em>P</em>-value &lt; 0.05 and eight SNPs a <em>P</em>-value &lt; 0.01 (<a href="/pmc/articles/PMC4004964/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). From each region we selected three SNPs (or two when three could not be found) with the most significant <em>P</em>-values for haplotype analysis since, in some situations, haplotype analysis is statistically more powerful.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593211">35</a></sup> We compared the minimum <em>P</em>-value obtained from the single SNP analysis and the minimum <em>P</em>-value obtained from the haplotype analysis (global and haplotype-specific analyses). The minimum <em>P</em>-value from the haplotype analysis was substantially lower (&lt; 0.5) in only two cases than that obtained from single marker analysis. The analysis of three SNP haplotypes (rs702543, rs17782374 and rs35277) of the PDE4D gene gave a simulated <em>P</em>-value for the maximum haplotype-specific score statistics of 0.00022 relative to a minimum <em>P</em>-value of 0.00082 for the single SNP analysis. The analysis of two SNP haplotypes (rs17225638 and rs11177914) at the RAB3IP gene gave a simulated <em>P</em>-value for the global statistics of 0.0069 relative to a minimum <em>P</em>-value of 0.020 for the single SNP analysis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="P58">Haplotype analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139963726643888" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Gene<sup>a</sup></th><th rowspan="1" colspan="1" align="right">Chr<sup>b</sup></th><th rowspan="1" colspan="1" align="right">Position</th><th rowspan="1" colspan="1" align="center">SNP1</th><th rowspan="1" colspan="1" align="center">SNP2</th><th rowspan="1" colspan="1" align="center">SNP3</th><th rowspan="1" colspan="1" align="center">SNP min <em>P</em><sup>c</sup></th><th rowspan="1" colspan="1" align="center">Global sim <em>P</em><sup>d</sup></th><th rowspan="1" colspan="1" align="center">Max-stat sim <em>P</em><sup>e</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">LOC284661</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">4370062</td><td rowspan="1" colspan="1" align="center">rs6679220</td><td rowspan="1" colspan="1" align="center">rs349410</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">4.2E–02</td><td rowspan="1" colspan="1" align="center">2.1E–01</td><td rowspan="1" colspan="1" align="center">1.9E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">DAB1</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="right">58268619</td><td rowspan="1" colspan="1" align="center">rs873989</td><td rowspan="1" colspan="1" align="center">rs1213659</td><td rowspan="1" colspan="1" align="center">rs1202835</td><td rowspan="1" colspan="1" align="center">1.3E–04</td><td rowspan="1" colspan="1" align="center">1.5E–02</td><td rowspan="1" colspan="1" align="center">3.9E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">NRXN1</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">50761056</td><td rowspan="1" colspan="1" align="center"><strong>rs7576283</strong></td><td rowspan="1" colspan="1" align="center">rs1037426</td><td rowspan="1" colspan="1" align="center">rs930295</td><td rowspan="1" colspan="1" align="center">1.0E–01</td><td rowspan="1" colspan="1" align="center">7.2E–01</td><td rowspan="1" colspan="1" align="center">7.3E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">CTNNA2</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">80715942</td><td rowspan="1" colspan="1" align="center"><strong>rs216617</strong></td><td rowspan="1" colspan="1" align="center">rs3770369</td><td rowspan="1" colspan="1" align="center"><strong>rs7609261</strong></td><td rowspan="1" colspan="1" align="center">2.9E–03</td><td rowspan="1" colspan="1" align="center">3.6E–02</td><td rowspan="1" colspan="1" align="center">1.2E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">DPP10</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">116252992</td><td rowspan="1" colspan="1" align="center"><strong>rs10496506</strong></td><td rowspan="1" colspan="1" align="center">rs11123306</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">2.1E–01</td><td rowspan="1" colspan="1" align="center">5.4E–01</td><td rowspan="1" colspan="1" align="center">8.2E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">LRP1B</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="right">140847651</td><td rowspan="1" colspan="1" align="center">rs13021003</td><td rowspan="1" colspan="1" align="center">rs2046563</td><td rowspan="1" colspan="1" align="center">rs17551974</td><td rowspan="1" colspan="1" align="center">1.5E–02</td><td rowspan="1" colspan="1" align="center">8.6E–02</td><td rowspan="1" colspan="1" align="center">2.9E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">FHIT</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">60694463</td><td rowspan="1" colspan="1" align="center">rs17361653</td><td rowspan="1" colspan="1" align="center">rs1882899</td><td rowspan="1" colspan="1" align="center">rs1716741</td><td rowspan="1" colspan="1" align="center">1.3E–03</td><td rowspan="1" colspan="1" align="center">8.6E–04</td><td rowspan="1" colspan="1" align="center">1.9E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">153382923</td><td rowspan="1" colspan="1" align="center">rs641025</td><td rowspan="1" colspan="1" align="center">rs989920</td><td rowspan="1" colspan="1" align="center">rs325738</td><td rowspan="1" colspan="1" align="center">2.6E–03</td><td rowspan="1" colspan="1" align="center">2.6E–03</td><td rowspan="1" colspan="1" align="center">8.7E–03</td></tr><tr><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">162481199</td><td rowspan="1" colspan="1" align="center">rs7652267</td><td rowspan="1" colspan="1" align="center">rs1478565</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">8.3E–02</td><td rowspan="1" colspan="1" align="center">1.3E–01</td><td rowspan="1" colspan="1" align="center">1.4E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">NLGN1</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="right">175393516</td><td rowspan="1" colspan="1" align="center">rs6806890</td><td rowspan="1" colspan="1" align="center">rs6779246</td><td rowspan="1" colspan="1" align="center">rs7623402</td><td rowspan="1" colspan="1" align="center">1.6E–02</td><td rowspan="1" colspan="1" align="center">7.2E–02</td><td rowspan="1" colspan="1" align="center">1.1E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">116523708</td><td rowspan="1" colspan="1" align="center"><strong>rs1112531</strong></td><td rowspan="1" colspan="1" align="center"><strong>rs963251</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">5.1E–02</td><td rowspan="1" colspan="1" align="center">3.6E–01</td><td rowspan="1" colspan="1" align="center">6.4E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">PDE4D</td><td rowspan="1" colspan="1" align="right">5</td><td rowspan="1" colspan="1" align="right">58878531</td><td rowspan="1" colspan="1" align="center">rs702543</td><td rowspan="1" colspan="1" align="center">rs17782374</td><td rowspan="1" colspan="1" align="center">rs35277</td><td rowspan="1" colspan="1" align="center">8.2E–04</td><td rowspan="1" colspan="1" align="center">4.7E–04</td><td rowspan="1" colspan="1" align="center">2.2E–04</td></tr><tr><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">33080668</td><td rowspan="1" colspan="1" align="center"><strong>rs592625</strong></td><td rowspan="1" colspan="1" align="center">rs7743563</td><td rowspan="1" colspan="1" align="center"><strong>rs2523666</strong></td><td rowspan="1" colspan="1" align="center">6.9E–02</td><td rowspan="1" colspan="1" align="center">4.2E–01</td><td rowspan="1" colspan="1" align="center">1.9E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">120286484</td><td rowspan="1" colspan="1" align="center">rs10484965</td><td rowspan="1" colspan="1" align="center">rs10499094</td><td rowspan="1" colspan="1" align="center">rs9320695</td><td rowspan="1" colspan="1" align="center">7.3E–03</td><td rowspan="1" colspan="1" align="center">1.3E–01</td><td rowspan="1" colspan="1" align="center">3.7E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">LAMA2</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">129281463</td><td rowspan="1" colspan="1" align="center">rs7776116</td><td rowspan="1" colspan="1" align="center"><strong>rs9321142</strong></td><td rowspan="1" colspan="1" align="center">rs6928495</td><td rowspan="1" colspan="1" align="center">1.1E–03</td><td rowspan="1" colspan="1" align="center">3.4E–02</td><td rowspan="1" colspan="1" align="center">5.9E–03</td></tr><tr><td rowspan="1" colspan="1" align="left">PARK2</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">162539612</td><td rowspan="1" colspan="1" align="center">rs6904579</td><td rowspan="1" colspan="1" align="center">rs3019449</td><td rowspan="1" colspan="1" align="center">rs17649761</td><td rowspan="1" colspan="1" align="center">1.6E–03</td><td rowspan="1" colspan="1" align="center">4.1E–02</td><td rowspan="1" colspan="1" align="center">1.9E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">LOC442275</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="right">164461465</td><td rowspan="1" colspan="1" align="center">rs206694</td><td rowspan="1" colspan="1" align="center">rs2764649</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">4.3E–02</td><td rowspan="1" colspan="1" align="center">8.1E–02</td><td rowspan="1" colspan="1" align="center">5.0E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">CREB5</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">28296787</td><td rowspan="1" colspan="1" align="center">rs4722801</td><td rowspan="1" colspan="1" align="center">rs177582</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3.7E–02</td><td rowspan="1" colspan="1" align="center">9.0E–02</td><td rowspan="1" colspan="1" align="center">1.3E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">95721578</td><td rowspan="1" colspan="1" align="center">rs2724041</td><td rowspan="1" colspan="1" align="center">rs2524976</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">6.9E–03</td><td rowspan="1" colspan="1" align="center">6.5E–02</td><td rowspan="1" colspan="1" align="center">1.0E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">RELN</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">102871243</td><td rowspan="1" colspan="1" align="center">rs362698</td><td rowspan="1" colspan="1" align="center">rs17321820</td><td rowspan="1" colspan="1" align="center">rs1541329</td><td rowspan="1" colspan="1" align="center">2.4E–02</td><td rowspan="1" colspan="1" align="center">3.0E–01</td><td rowspan="1" colspan="1" align="center">2.6E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">PTN</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">136413862</td><td rowspan="1" colspan="1" align="center">rs322239</td><td rowspan="1" colspan="1" align="center">rs322323</td><td rowspan="1" colspan="1" align="center">rs322236</td><td rowspan="1" colspan="1" align="center">4.8E–06</td><td rowspan="1" colspan="1" align="center">8.0E–05</td><td rowspan="1" colspan="1" align="center">1.0E–05</td></tr><tr><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">8614503</td><td rowspan="1" colspan="1" align="center">rs4841017</td><td rowspan="1" colspan="1" align="center">rs9329165</td><td rowspan="1" colspan="1" align="center">rs6601729</td><td rowspan="1" colspan="1" align="center">6.5E–06</td><td rowspan="1" colspan="1" align="center">1.9E–04</td><td rowspan="1" colspan="1" align="center">1.7E–04</td></tr><tr><td rowspan="1" colspan="1" align="left">CSMD1</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">3178277</td><td rowspan="1" colspan="1" align="center">rs4395910</td><td rowspan="1" colspan="1" align="center"><strong>rs10503232</strong></td><td rowspan="1" colspan="1" align="center">rs4875610</td><td rowspan="1" colspan="1" align="center">4.1E–04</td><td rowspan="1" colspan="1" align="center">8.5E–04</td><td rowspan="1" colspan="1" align="center">6.2E–04</td></tr><tr><td rowspan="1" colspan="1" align="left">NRG1</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">32336767</td><td rowspan="1" colspan="1" align="center">rs1487152</td><td rowspan="1" colspan="1" align="center">rs7002732</td><td rowspan="1" colspan="1" align="center">rs12114401</td><td rowspan="1" colspan="1" align="center">4.6E–04</td><td rowspan="1" colspan="1" align="center">4.1E–04</td><td rowspan="1" colspan="1" align="center">3.0E–03</td></tr><tr><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">108959180</td><td rowspan="1" colspan="1" align="center">rs1389976</td><td rowspan="1" colspan="1" align="center">rs10505119</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">1.9E–02</td><td rowspan="1" colspan="1" align="center">7.9E–02</td><td rowspan="1" colspan="1" align="center">3.2E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">CSMD3</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="right">114031134</td><td rowspan="1" colspan="1" align="center">rs10505196</td><td rowspan="1" colspan="1" align="center">rs4876501</td><td rowspan="1" colspan="1" align="center">rs10505174</td><td rowspan="1" colspan="1" align="center">2.2E–02</td><td rowspan="1" colspan="1" align="center">2.0E–01</td><td rowspan="1" colspan="1" align="center">4.4E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">PTPRD</td><td rowspan="1" colspan="1" align="right">9</td><td rowspan="1" colspan="1" align="right">8481878</td><td rowspan="1" colspan="1" align="center">rs6477311</td><td rowspan="1" colspan="1" align="center">rs1359114</td><td rowspan="1" colspan="1" align="center">rs10511494</td><td rowspan="1" colspan="1" align="center">1.2E–02</td><td rowspan="1" colspan="1" align="center">7.5E–02</td><td rowspan="1" colspan="1" align="center">3.1E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">ASTN2</td><td rowspan="1" colspan="1" align="right">9</td><td rowspan="1" colspan="1" align="right">116859364</td><td rowspan="1" colspan="1" align="center">rs2418446</td><td rowspan="1" colspan="1" align="center">rs10513281</td><td rowspan="1" colspan="1" align="center">rs10983238</td><td rowspan="1" colspan="1" align="center">3.4E–02</td><td rowspan="1" colspan="1" align="center">2.2E–01</td><td rowspan="1" colspan="1" align="center">4.1E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">TMEM16C</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="right">26402767</td><td rowspan="1" colspan="1" align="center">rs10834976</td><td rowspan="1" colspan="1" align="center">rs1381181</td><td rowspan="1" colspan="1" align="center">rs10834971</td><td rowspan="1" colspan="1" align="center">4.4E–03</td><td rowspan="1" colspan="1" align="center">1.4E–02</td><td rowspan="1" colspan="1" align="center">2.7E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">BDNF</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="right">27715568</td><td rowspan="1" colspan="1" align="center">rs985205</td><td rowspan="1" colspan="1" align="center">rs2049045</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">1.7E–01</td><td rowspan="1" colspan="1" align="center">4.3E–01</td><td rowspan="1" colspan="1" align="center">3.6E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">DLG2</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="right">84283401</td><td rowspan="1" colspan="1" align="center">rs17147761</td><td rowspan="1" colspan="1" align="center">rs891773</td><td rowspan="1" colspan="1" align="center">rs10501548</td><td rowspan="1" colspan="1" align="center">4.4E–02</td><td rowspan="1" colspan="1" align="center">3.7E–01</td><td rowspan="1" colspan="1" align="center">4.9E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">CNTN5</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="right">98413115</td><td rowspan="1" colspan="1" align="center">rs10501893</td><td rowspan="1" colspan="1" align="center">rs10501898</td><td rowspan="1" colspan="1" align="center">rs10501903</td><td rowspan="1" colspan="1" align="center">4.1E–02</td><td rowspan="1" colspan="1" align="center">4.9E–02</td><td rowspan="1" colspan="1" align="center">6.2E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">GRIP1</td><td rowspan="1" colspan="1" align="right">12</td><td rowspan="1" colspan="1" align="right">65150558</td><td rowspan="1" colspan="1" align="center">rs4913307</td><td rowspan="1" colspan="1" align="center">rs12314237</td><td rowspan="1" colspan="1" align="center"><strong>rs12581890</strong></td><td rowspan="1" colspan="1" align="center">1.2E–02</td><td rowspan="1" colspan="1" align="center">1.1E–02</td><td rowspan="1" colspan="1" align="center">2.0E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">RAB3IP</td><td rowspan="1" colspan="1" align="right">12</td><td rowspan="1" colspan="1" align="right">68480702</td><td rowspan="1" colspan="1" align="center">rs17225638</td><td rowspan="1" colspan="1" align="center">rs11177914</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">2.0E–02</td><td rowspan="1" colspan="1" align="center">6.9E–03</td><td rowspan="1" colspan="1" align="center">1.5E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">LOC284058</td><td rowspan="1" colspan="1" align="right">17</td><td rowspan="1" colspan="1" align="right">41550514</td><td rowspan="1" colspan="1" align="center"><strong>rs10514901</strong></td><td rowspan="1" colspan="1" align="center"><strong>rs17631303</strong></td><td rowspan="1" colspan="1" align="center"><strong>rs2532276</strong></td><td rowspan="1" colspan="1" align="center">9.4E–03</td><td rowspan="1" colspan="1" align="center">6.7E–02</td><td rowspan="1" colspan="1" align="center">2.2E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">TCF4</td><td rowspan="1" colspan="1" align="right">18</td><td rowspan="1" colspan="1" align="right">51268302</td><td rowspan="1" colspan="1" align="center">rs1452788</td><td rowspan="1" colspan="1" align="center">rs12326693</td><td rowspan="1" colspan="1" align="center">rs9957668</td><td rowspan="1" colspan="1" align="center">9.3E–04</td><td rowspan="1" colspan="1" align="center">2.8E–01</td><td rowspan="1" colspan="1" align="center">3.2E–01</td></tr><tr><td rowspan="1" colspan="1" align="left">NA</td><td rowspan="1" colspan="1" align="right">18</td><td rowspan="1" colspan="1" align="right">52115138</td><td rowspan="1" colspan="1" align="center">rs9959800</td><td rowspan="1" colspan="1" align="center">rs1229587</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">1.7E–04</td><td rowspan="1" colspan="1" align="center">1.5E–03</td><td rowspan="1" colspan="1" align="center">1.4E–03</td></tr><tr><td rowspan="1" colspan="1" align="left">GRIK1</td><td rowspan="1" colspan="1" align="right">21</td><td rowspan="1" colspan="1" align="right">30306168</td><td rowspan="1" colspan="1" align="center">rs2832533</td><td rowspan="1" colspan="1" align="center">rs2832438</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">5.4E–03</td><td rowspan="1" colspan="1" align="center">1.0E–02</td><td rowspan="1" colspan="1" align="center">1.0E–02</td></tr><tr><td rowspan="1" colspan="1" align="left">ERG</td><td rowspan="1" colspan="1" align="right">21</td><td rowspan="1" colspan="1" align="right">38888946</td><td rowspan="1" colspan="1" align="center">rs17194186</td><td rowspan="1" colspan="1" align="center">rs2836502</td><td rowspan="1" colspan="1" align="center">rs2836365</td><td rowspan="1" colspan="1" align="center">2.2E–02</td><td rowspan="1" colspan="1" align="center">2.4E–02</td><td rowspan="1" colspan="1" align="center">1.8E–02</td></tr></tbody></table></div><div id="largeobj_idm139963726643888" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4004964/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN11"><p id="P59" class="p p-first-last">Abbreviation: NA, not available.</p></div><div id="TFN12"><p id="P60" class="p p-first-last">SNPs within a copy number variation region are in bold.</p></div><div id="TFN13"><sup>a</sup>SNP associate gene based on NCBI dbSNP.</div><div id="TFN14"><sup>b</sup>Chromosome.</div><div id="TFN15"><sup>c</sup>Minimum <em>P</em>-value from the single SNP analysis.</div><div id="TFN16"><sup>d</sup>Simulation <em>P</em>-value for the global score statistic.</div><div id="TFN17"><sup>e</sup>Simulation <em>P</em>-value for the maximum score statistic.</div></div></div></div><div id="S13" class="sec"><h3 id="S13title">Distribution of <em>P</em>-values obtained in the DNA pooling stage in linkage regions</h3><p id="P23" class="p p-first">To enhance the power to detect association in regions where we had prior information for a genetic effect, we took into account evidence from a previous linkage scan that used the same population analyzed here by association.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593171">27</a></sup> Analysis of allele sharing in discordant and concordant sib pairs previously identified five significant quantitative trait loci (QTLs) for neuroticism, on 1q, 4q, 7p, 12q and 13q. We tested whether the distribution of <em>P</em>-values obtained in the DNA pooling stage was higher than expected in the QTL regions.</p><p id="P24" class="p p-last">We calculated the total proportion of log <em>P</em> values above 2, 3, 4 and 5 in the 1-LOD score intervals for the five significant peaks. The proportion of log <em>P</em> values greater than 2, 3, 4 and 5 was also calculated in a set of five regions of the same size chosen at random from across the genome, and this process was repeated 1000 times. We found no significant enrichment in the 1-LOD score intervals of the QTL regions, when analyzed together. However, when we tested individual regions (comparing the proportion of log <em>P</em> values above a certain value in each 1-LOD score interval compared to random selected region), we found a nominally significant (<em>P</em> = 0.022) enrichment in log <em>P</em> &gt; 2 values in the 1-LOD interval on chromosome 1q. This result becomes non-significant when corrected for multiple testing.</p></div><div id="S14" class="sec"><h3 id="S14title">Copy number variants</h3><p id="P25" class="p p-first">We noticed that some SNPs with high log <em>P</em> values in the DNA pooling experiment are clustered in regions of known copy number variation (CNV). In the DNA pooling experiment we calculated the association score using the relative intensities of the probes, which might be influenced by CNVs. Hence, a cluster of high log <em>P</em> values could be a result of different frequencies of CNVs in cases and controls rather than of SNP allele frequency differences. We tested the distribution of log <em>P</em> values in regions of known CNVs and random regions of the same size (in the same way as we investigated the distribution of log <em>P</em> values in the linkage regions). We observed a positive enrichment of log <em>P</em> values in CNVs obtained from The Database of Genomic Variants, <a href="http://projects.tcag.ca/variation/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://projects.tcag.ca/variation/</a> (not including the data from Redon <em>et al.</em><sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593189">36</a></sup>) (<em>P</em> = 0.0085, 0.017, 0.024 and 0.031 for log <em>P</em> of 2, 3, 4 and 5). We also tested the distribution of log <em>P</em> values in CNVs reported recently.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593180">36</a></sup> We examined CNVs identified using the 500K EA arrays with a frequency of greater than 5%. The positive enrichment of log <em>P</em> values using this dataset was even more significant (<em>P</em> = 0.017, 0.003, 0.002 and <em>P</em> &lt; 0.001 for log <em>P</em> of 2, 3, 4 and 5).</p><p id="P26">We considered whether this result could be artifactual. Within CNVs, the amount of DNA added to each pool might be more variable than in other regions, so we expect first that there will be more variation in allele frequency estimates in these regions and second that the differences between groups (as measured by the log <em>P</em> values) will be less when frequencies are obtained by individual genotyping.</p><p id="P27">Neither expectation was realized. Variation between DNA pools at SNPs within CNVs was not significantly different from that seen at SNPs else-where in the genome, and the enrichment of high log <em>P</em> values at CNVs was also seen in some cases in individually genotyped SNPs. These results indicate that the explanation for the enrichment may be related to the biology of neuroticism.</p><p id="P28" class="p p-last">We examined each CNV to identify those contributing most to the observed enrichment of log <em>P</em> values. A region on chromosome 17q21.31 is the CNV region with the highest enrichment of high log <em>P</em> values (<a href="/pmc/articles/PMC4004964/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). The enrichment of log <em>P</em> values in other CNVs is no longer significant when chromosome 17 is excluded from the analysis (minimum <em>P</em> = 0.087 for log <em>P</em> = 5). Individual genotyping of the sample used in the pools with an SNP (rs10514901) within the CNV region showed a modest association with neuroticism (<em>P</em>=0.0094).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4004964/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139963725279152" class="figure"><a href="/pmc/articles/PMC4004964/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4004964_nihms572905f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms572905f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4004964/bin/nihms572905f2.jpg"></a></div><div id="largeobj_idm139963725279152" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4004964/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4004964/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="P45">DNA pooling association results in a region of chromosome 17q21.31. Results of the DNA pooling are presented in log <em>P</em> values (vertical axis). The gray bars show the positions in megabases (Mb) of copy number variations (CNVs) on 17q21.3.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593206">36</a></sup></p></div></div></div></div><div id="S15" class="sec sec-last"><h3 id="S15title">Replication</h3><p id="P29" class="p p-first">We attempted to replicate the results of the most significant SNPs in an independent sample. Nineteen SNPs with a <em>P</em>-value &lt; 0.001 (sex-specific or sex-averaged) were genotyped on a sample that consisted of 933 women and 601 men (all in the 10% low or high tail of the N distribution) (<a href="/pmc/articles/PMC4004964/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). Only one SNP showed a statistically significant association in the replication. The significant SNP, rs702543, had a <em>P</em>-value of 0.00081 in the original sample, a <em>P</em>-value of 0.00078 (Bonferroni corrected <em>P</em>-value with 19 tests = 0.015) in the replication sample and a combined <em>P</em>-value = 2 × 10<sup>−6</sup>. In both the samples the frequency of the A allele was increased in the high Ns (frequency in the combined sample = 0.61) relative to the low Ns (frequency = 0.55) with an OR of 1.27 (95% confidence interval for the OR (95% CI) is 1.15– 1.4) and proportion of variance explained of 0.64% in an ANOVA model. Using the age and sex regressed N score to analyze the replication sample we obtain a <em>P</em>-value of 0.00065 for the additive model (note that because the scores are so extreme in the selected samples, this result is almost identical to the value obtained by applying a χ<sup>2</sup> test: <em>P</em> = 0.00078). The ORs for men (OR = 1.30 in the combined sample) and women (OR = 1.25) were not statistically different in the pooling sample, the replication sample or the combined sample.</p><p id="P30">We attempted to replicate rs702543 in other populations. We investigated three populations: (1) a sample selected from 15 027 Australian twins and 11 389 of their family members. Unrelated individuals with neuroticism, anxiety and depression scores in the upper or lower deciles were genotyped (<em>n</em>=761)<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593198">37</a></sup>; (2) unrelated individuals from the Virginia Adult Twin Study of Psychiatric and Substance Use Disorders.<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593174">38</a></sup> We used a sample of 1022 individuals selected from 9270 twins for a study of candidate genes on anxiety, depression and neuroticism; (3) a sample of 417 unrelated individuals from the Netherlands twin family for a study of anxious depression.<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593159">39</a></sup> Individuals were selected from a total of 13 717 individuals from 3344 families on the basis of extreme factor scores. The factor score included a neuroticism assessment, using the Amsterdamse Biografische Vragenlijst (ABV), a self-report personality instrument similar in content to the EPQ.<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593212">40</a>,<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593167">41</a></sup></p><p id="P31">We divided the samples from all populations into two groups based on N scores. We analyzed the allele and genotype frequencies in high and low N in each population and also in the combined sample (<a href="/pmc/articles/PMC4004964/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). The combined sample from the three populations showed a non-significant trend for an excess of the A allele in high N-score group (OR = 1.05, 95% CI = 0.92–1.19, <em>P</em> = 0.46). The allelic effect of SNP rs702543 estimated from the combination of the three populations and our replication sample in terms of OR is 1.13 (95% CI = 1.03–1.24, <em>P</em>=0.012).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="P66">Genotypes in replication samples</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139963718333296" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Sample</th><th colspan="5" rowspan="1" align="center">High N</th><th colspan="5" rowspan="1" align="center">Low N</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left"><em>P</em> allele</th><th rowspan="1" colspan="1" align="left"><em>P</em> genotype</th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="5" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="5" rowspan="1" valign="bottom" align="left"><hr></th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left"></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="right">AA</th><th rowspan="1" colspan="1" align="right">AG</th><th rowspan="1" colspan="1" align="right">GG</th><th rowspan="1" colspan="1" align="center">A%<sup>a</sup></th><th rowspan="1" colspan="1" align="right">N</th><th rowspan="1" colspan="1" align="right">AA</th><th rowspan="1" colspan="1" align="center">AG</th><th rowspan="1" colspan="1" align="right">GG</th><th rowspan="1" colspan="1" align="center">A%</th><th rowspan="1" colspan="1" align="right">N</th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Pool sample</td><td rowspan="1" colspan="1" align="right">348</td><td rowspan="1" colspan="1" align="right">428</td><td rowspan="1" colspan="1" align="right">156</td><td rowspan="1" colspan="1" align="center">60.3</td><td rowspan="1" colspan="1" align="right">932</td><td rowspan="1" colspan="1" align="right">291</td><td rowspan="1" colspan="1" align="center">468</td><td rowspan="1" colspan="1" align="right">197</td><td rowspan="1" colspan="1" align="center">54.9</td><td rowspan="1" colspan="1" align="right">956</td><td rowspan="1" colspan="1" align="left">1.25</td><td rowspan="1" colspan="1" align="left">0.00082</td><td rowspan="1" colspan="1" align="left">0.0035</td></tr><tr><td rowspan="1" colspan="1" align="left">Our replication</td><td rowspan="1" colspan="1" align="right">289</td><td rowspan="1" colspan="1" align="right">316</td><td rowspan="1" colspan="1" align="right">123</td><td rowspan="1" colspan="1" align="center">61.4</td><td rowspan="1" colspan="1" align="right">728</td><td rowspan="1" colspan="1" align="right">197</td><td rowspan="1" colspan="1" align="center">305</td><td rowspan="1" colspan="1" align="right">133</td><td rowspan="1" colspan="1" align="center">55.0</td><td rowspan="1" colspan="1" align="right">635</td><td rowspan="1" colspan="1" align="left">1.3</td><td rowspan="1" colspan="1" align="left">0.00077</td><td rowspan="1" colspan="1" align="left">0.0029</td></tr><tr><td rowspan="1" colspan="1" align="left">Australian</td><td rowspan="1" colspan="1" align="right">129</td><td rowspan="1" colspan="1" align="right">163</td><td rowspan="1" colspan="1" align="right">57</td><td rowspan="1" colspan="1" align="center">60.3</td><td rowspan="1" colspan="1" align="right">349</td><td rowspan="1" colspan="1" align="right">144</td><td rowspan="1" colspan="1" align="center">196</td><td rowspan="1" colspan="1" align="right">71</td><td rowspan="1" colspan="1" align="center">58.9</td><td rowspan="1" colspan="1" align="right">411</td><td rowspan="1" colspan="1" align="left">1.06</td><td rowspan="1" colspan="1" align="left">0.57</td><td rowspan="1" colspan="1" align="left">0.85</td></tr><tr><td rowspan="1" colspan="1" align="left">Dutch</td><td rowspan="1" colspan="1" align="right">75</td><td rowspan="1" colspan="1" align="right">84</td><td rowspan="1" colspan="1" align="right">31</td><td rowspan="1" colspan="1" align="center">62.0</td><td rowspan="1" colspan="1" align="right">190</td><td rowspan="1" colspan="1" align="right">85</td><td rowspan="1" colspan="1" align="center">106</td><td rowspan="1" colspan="1" align="right">36</td><td rowspan="1" colspan="1" align="center">61.0</td><td rowspan="1" colspan="1" align="right">227</td><td rowspan="1" colspan="1" align="left">1.03</td><td rowspan="1" colspan="1" align="left">0.82</td><td rowspan="1" colspan="1" align="left">0.88</td></tr><tr><td rowspan="1" colspan="1" align="left">Virginia</td><td rowspan="1" colspan="1" align="right">83</td><td rowspan="1" colspan="1" align="right">132</td><td rowspan="1" colspan="1" align="right">38</td><td rowspan="1" colspan="1" align="center">58.9</td><td rowspan="1" colspan="1" align="right">253</td><td rowspan="1" colspan="1" align="right">261</td><td rowspan="1" colspan="1" align="center">379</td><td rowspan="1" colspan="1" align="right">129</td><td rowspan="1" colspan="1" align="center">58.6</td><td rowspan="1" colspan="1" align="right">769</td><td rowspan="1" colspan="1" align="left">1.01</td><td rowspan="1" colspan="1" align="left">0.90</td><td rowspan="1" colspan="1" align="left">0.69</td></tr><tr><td rowspan="1" colspan="1" align="left">Three groups</td><td rowspan="1" colspan="1" align="right">287</td><td rowspan="1" colspan="1" align="right">379</td><td rowspan="1" colspan="1" align="right">126</td><td rowspan="1" colspan="1" align="center">60.2</td><td rowspan="1" colspan="1" align="right">792</td><td rowspan="1" colspan="1" align="right">490</td><td rowspan="1" colspan="1" align="center">681</td><td rowspan="1" colspan="1" align="right">236</td><td rowspan="1" colspan="1" align="center">59.0</td><td rowspan="1" colspan="1" align="right">1407</td><td rowspan="1" colspan="1" align="left">1.05</td><td rowspan="1" colspan="1" align="left">0.46</td><td rowspan="1" colspan="1" align="left">0.76</td></tr><tr><td rowspan="1" colspan="1" align="left">All replications</td><td rowspan="1" colspan="1" align="right">576</td><td rowspan="1" colspan="1" align="right">695</td><td rowspan="1" colspan="1" align="right">249</td><td rowspan="1" colspan="1" align="center">60.8</td><td rowspan="1" colspan="1" align="right">1520</td><td rowspan="1" colspan="1" align="right">687</td><td rowspan="1" colspan="1" align="center">986</td><td rowspan="1" colspan="1" align="right">369</td><td rowspan="1" colspan="1" align="center">57.8</td><td rowspan="1" colspan="1" align="right">2042</td><td rowspan="1" colspan="1" align="left">1.13</td><td rowspan="1" colspan="1" align="left">0.012</td><td rowspan="1" colspan="1" align="left">0.029</td></tr></tbody></table></div><div id="largeobj_idm139963718333296" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4004964/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN18"><p id="P67" class="p p-first-last">Abbreviation: OR, odds ratio.</p></div><div id="TFN19"><sup>a</sup>Frequency of the A allele.</div></div></div><p id="P32" class="p p-last">We also used family based analysis to test for the association of rs702543 with neuroticism in the Netherlands and Australian full family samples. There was no evidence for association in the Netherlands family sample (<em>N</em>= 1148, wave 6, analyzed using quantitative transmission-disequilibrium test (QTDT), <em>P</em> = 0.88). The Australian family sample was analyzed using the age and sex transformed scores<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593185">42</a></sup> based on the full 23-item EPQ-R N scale and the 10-item Kessler Psychological Distress scale<sup><a href="#R43" rid="R43" class=" bibr popnode">43</a></sup> (K10), a measure of current non-specific psychological distress in the anxiety–depression spectrum. Individuals were considered ‘affected’ for EPQ-R and K10 if their residual scores were in the top 5, 10, 15 and 20 percentile of available residuals. Consistent with the findings in the original sample, an overtransmission of the A allele to affected individuals was observed for all percentiles. The most significant transmission distortion was observed when the individuals in the 20 percentile were treated as affected (analyzed using UNPHASED v3.03, EPQ <em>P</em>-value = 0.040, K10 <em>P</em>-value = 0.0059).</p></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Discussion</h2><p id="P33" class="p p-first">We have performed the first whole genome analysis of neuroticism, using pooled DNA. We show that, by using eight pools with five replicates of each, we retain between 21 and 94% (depending on the effect size and threshold used) of the power from genotyping all 452 574 SNPs individually, equivalent to a 50-fold saving in the costs. We identified one SNP rs702543 from an analysis of ~2000 individuals selected from the neuroticism extremes of 88 142 people, which we were able to replicate in a separate sample from the same cohort. The SNP was genotyped in three other laboratories, on related, but not identical, phenotypes. In each case the direction of allelic effects was the same as in our sample, but the test reached statistical significance in only one sample, that is, only in one method of analysis (family based analysis). Our study raises a number of issues for whole genome association studies in general and the genetic basis of neuroticism in particular.</p><p id="P34">We deal first with the performance of the pooling strategy. Our estimates of the power of our DNA pooling strategy were derived by simulation using the measurement error across all SNPs from both 500 and 100K arrays. However, there is a 3.2-fold difference between the variances of the repeated measurements for the two types of arrays (mean variance for 100K= 0.0017, and for 500K= 0.0055). This difference in measurement accuracy is probably due to the smaller number of features per SNP and the reduction in feature size in the 500K array relative to the 100 K.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593179">21</a></sup> It means that the measurements in the 500K arrays should be at least triplicated to achieve the genotype accuracy obtained with the 100K arrays.</p><p id="P35">DNA pooling allows substantial savings in genotyping costs (see, for example, references<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593158">44</a>–<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593204">46</a></sup>) and there have been successes using DNA pooling in the analysis of a relatively small number of markers, for example, in schizophrenia<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593157">47</a>,<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593166">48</a></sup> multiple sclerosis,<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593213">49</a></sup> and serum high-density lipoprotein cholesterol levels.<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593207">50</a>,<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593199">51</a></sup> However, in the two cases where whole genome association has been attempted (that is, in memory<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593197">52</a></sup> and mild mental impairment<sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593176">53</a></sup>), the yields have been low, that is, in both cases only one locus was considered significant.</p><p id="P36">Our results are comparable, in that we have found just one locus of small effect. Failure to replicate the finding in all samples is not unexpected, given the differences between the cohorts, in both phenotype and recruitment. No sample used exactly the same phenotype. We used the 23-item Eysenck N scale, while the US and Australian sample used the short (12-item) scale and the Dutch sample used the Amsterdamse Biografische Vragenlijst, which is based on the Eysenck scale but is not identical. However, differences in the measures are likely to be outweighed by differences in the recruitment strategy. Our sample represents the extremes of about 61 367 unrelated people, the Dutch from 3444 families, the Australian from around 7500 families and the US from 9270 twins.</p><p id="P37">Second, we consider the lessons for the design of whole genome association studies. Assuming that the genetic basis of neuroticism is typical of behavior, and also typical of other complex traits, our results reinforce the findings that the ORs of most complex trait loci are less than 1.6.<sup><a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593188">54</a></sup> Our linkage study on the same cohort failed to find any large effect loci (explaining more than 5% of the variance), and given that we had approximately 50% power to detect an OR of 1.5 (which is equivalent to a locus explaining 0.5–1% of the variance) and failed to find any loci accounting for more than 1% of the variance, it seems likely that the 40% additive genetic variance of neuroticism arises from many loci explaining much less than 1%. This means that to obtain adequate power in whole genome association studies, much larger samples will be needed than anticipated.</p><p id="P38">Using association analysis, we failed to identify any genes under the five significant linkage peaks that were obtained using the sib pairs from the same population-based study of personality (although it should be noted that there was a nominally significant enrichment in log <em>P</em> &gt; 2 values in the 1-LOD interval on chromosome 1q). Our significant SNP is located on chromosome 5, a region that was not indicated by the linkage analysis. Since linkage analysis is robust to allelic heterogeneity at a locus, this may indicate that rare variants are a major contributor to the heritability of neuroticism. The linkage signal could also be due to the co-localization under the linkage peaks of variants in different genes, each of too insufficient individual effect to be detected by association. Other possible factors that could contribute to the lower power of our study to identify the genes under the linkage peaks include the effect of incomplete genome coverage (low LD between the SNPs typed and the causal variants) and a high false-negative rate of the association analysis.</p><p id="P39">Unlike the linkage intervals that did not show a significant enrichment of high log <em>P</em> values, we did see the enrichment in known CNVs. Chromosomal abnormalities have been reported in patients with schizophrenia, bipolar disorder and major depression,<sup><a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593164">55</a>,<a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593175">56</a></sup> but it is not clear to what extent CNVs contribute to variation in behavior. It is noteworthy that CNVs on chromosome 17 (enriched for high log <em>P</em> values) contain two genes involved in behavior, that is, the Tau gene (MAPT) and the corticotropin-releasing hormone receptor-1 (CRHR1). CRHR1 has been shown to mediate anxiety-related behavior and hormonal adaptation to stress.<sup><a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593201">57</a></sup></p><p id="P40">Finally, our findings are important for the understanding of the genetic basis of neuroticism and the associated psychiatric disorders of anxiety and depression. The rs702543 SNP is located in the phosphodiesterase 4D, cAMP-specific (PDE4D) gene, in an intron between exon D3 and D8. The HapMap database (CEU, Release #21) shows one SNP (rs702542) in complete correlation with rs702543, and two other SNPs that are partially correlated, that is, rs296410 (<em>r</em><sup>2</sup> = 0.5) and rs40216 (<em>r</em><sup>2</sup> = 0.56). All three SNPs lie within a 65.3-kb haplotype block on chromosome 5 (58 872 707–58 937 990 bp). The G allele of rs702543 creates a putative cAMP response element (CRE) for CREB2/c-jun heterodimer (TGACGTTA), while the A allele destroys it.<sup><a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593209">58</a>,<a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593182">59</a></sup> The transcription of different isoforms of PDE4D was shown to be regulated by cAMP levels. For example, a CRE (TGACGTT) in the promoter of an isoform of PDE4D (PDE4D5) was shown to be involved in the cAMP responsiveness of the PDE4D5 promoter.<sup><a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593161">60</a></sup></p><p id="P41" class="p p-last">Several other lines of evidence support the involvement of the PDE4D gene in susceptibility to neuroticism and depression. A PDE4-specific inhibitor rolipram has antidepressant effects on animals and patients with major depression.<sup><a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593194">61</a>–<a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593168">63</a></sup> PDE4D knockout mice show antidepressant-like behavior, which is further increased by the antidepressants desipramine and fluoxetine but not by rolipram. This suggests that the PDE4D subtype is an essential mediator of the antidepressant effects of rolipram. PDE4D expression is increased in mouse cerebral cortex by the repeated treatment with desipramine, fluoxetine and rolipram and in the hippocampus by fluoxetine and rolipram. Recently, variants in two genes encoding PDEs were found to be associated with the diagnosis of major depression, and one of them (PDE11A) was also found to be associated with remission in response to antidepressants.<sup><a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371593163">64</a></sup> Together, these observations indicate that the PDE4D gene is likely to be involved in susceptibility to neuroticism and the associated psychiatric disorders of major depression and anxiety.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>supplemental material</h4><div class="sup-box half_rhythm" id="d36e1157"><a href="/pmc/articles/PMC4004964/bin/NIHMS572905-supplement-supplemental_material.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(11K, pdf)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="P42">Funding for this work was provided by the Wellcome Trust. SS is supported by the European Molecular Biology Organization.</p></div></div><div id="idm139963717925120" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139963717925120title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P43" class="p p-first-last">Supplementary Information accompanies the paper on the Molecular Psychiatry website (<a href="http://www.nature.com/mp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/mp</a>)</p><p></p></div></div><div id="idm139963717923168" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139963717923168title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Eysenck HJ.  <span class="ref-journal">The Biological Basis of Personality.</span> Springfield, IL: Thomas; 1967.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Biological+Basis+of+Personality&amp;author=HJ+Eysenck&amp;publication_year=1967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Angst J, Clayton P. Premorbid personality of depressive, bipolar and schizophrenic patients with special reference to suicidal issues. <span><span class="ref-journal">Compr Psychiatry. </span>1986;<span class="ref-vol">27</span>:511–532.</span> [<a href="/pubmed/3780193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Compr+Psychiatry&amp;title=Premorbid+personality+of+depressive,+bipolar+and+schizophrenic+patients+with+special+reference+to+suicidal+issues&amp;author=J+Angst&amp;author=P+Clayton&amp;volume=27&amp;publication_year=1986&amp;pages=511-532&amp;pmid=3780193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Boyce P, Parker G, Barnett B, Cooney M, Smith F. Personality as a vulnerability factor to depression. <span><span class="ref-journal">Br J Psychiatry. </span>1991;<span class="ref-vol">159</span>:106–114.</span> [<a href="/pubmed/1888956" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Psychiatry&amp;title=Personality+as+a+vulnerability+factor+to+depression&amp;author=P+Boyce&amp;author=G+Parker&amp;author=B+Barnett&amp;author=M+Cooney&amp;author=F+Smith&amp;volume=159&amp;publication_year=1991&amp;pages=106-114&amp;pmid=1888956&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Hirschfeld RM, Klerman GL, Lavori P, Keller MB, Griffith P, Coryell W. Premorbid personality assessments of first onset of major depression. <span><span class="ref-journal">Arch Gen Psychiatry. </span>1989;<span class="ref-vol">46</span>:345–350.</span> [<a href="/pubmed/2649038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Premorbid+personality+assessments+of+first+onset+of+major+depression&amp;author=RM+Hirschfeld&amp;author=GL+Klerman&amp;author=P+Lavori&amp;author=MB+Keller&amp;author=P+Griffith&amp;volume=46&amp;publication_year=1989&amp;pages=345-350&amp;pmid=2649038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Kendler KS, Neale MC, Kessler RC, Heath AC, Eaves LJ. A longitudinal twin study of personality and major depression in women. <span><span class="ref-journal">Arch Gen Psychiatry. </span>1993;<span class="ref-vol">50</span>:853–862.</span> [<a href="/pubmed/8215811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=A+longitudinal+twin+study+of+personality+and+major+depression+in+women&amp;author=KS+Kendler&amp;author=MC+Neale&amp;author=RC+Kessler&amp;author=AC+Heath&amp;author=LJ+Eaves&amp;volume=50&amp;publication_year=1993&amp;pages=853-862&amp;pmid=8215811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Bienvenu OJ, Samuels JF, Costa PT, Reti IM, Eaton WW, Nestadt G. Anxiety and depressive disorders and the five-factor model of personality: a higher- and lower-order personality trait investigation in a community sample. <span><span class="ref-journal">Depress Anxiety. </span>2004;<span class="ref-vol">20</span>:92–97.</span> [<a href="/pubmed/15390211" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Depress+Anxiety&amp;title=Anxiety+and+depressive+disorders+and+the+five-factor+model+of+personality:+a+higher-+and+lower-order+personality+trait+investigation+in+a+community+sample&amp;author=OJ+Bienvenu&amp;author=JF+Samuels&amp;author=PT+Costa&amp;author=IM+Reti&amp;author=WW+Eaton&amp;volume=20&amp;publication_year=2004&amp;pages=92-97&amp;pmid=15390211&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Cox BJ, MacPherson PS, Enns MW, McWilliams LA. Neuroticism and self-criticism associated with posttraumatic stress disorder in a nationally representative sample. <span><span class="ref-journal">Behav Res Ther. </span>2004;<span class="ref-vol">42</span>:105–114.</span> [<a href="/pubmed/14744527" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behav+Res+Ther&amp;title=Neuroticism+and+self-criticism+associated+with+posttraumatic+stress+disorder+in+a+nationally+representative+sample&amp;author=BJ+Cox&amp;author=PS+MacPherson&amp;author=MW+Enns&amp;author=LA+McWilliams&amp;volume=42&amp;publication_year=2004&amp;pages=105-114&amp;pmid=14744527&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Jylha P, Isometsa E. The relationship of neuroticism and extraversion to symptoms of anxiety and depression in the general population. <span><span class="ref-journal">Depress Anxiety. </span>2006;<span class="ref-vol">23</span>:281–289.</span> [<a href="/pubmed/16688731" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Depress+Anxiety&amp;title=The+relationship+of+neuroticism+and+extraversion+to+symptoms+of+anxiety+and+depression+in+the+general+population&amp;author=P+Jylha&amp;author=E+Isometsa&amp;volume=23&amp;publication_year=2006&amp;pages=281-289&amp;pmid=16688731&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Jylha P, Isometsa E. Temperament, character and symptoms of anxiety and depression in the general population. <span><span class="ref-journal">Eur Psychiatry. </span>2006;<span class="ref-vol">21</span>:389–395.</span> [<a href="/pubmed/16360306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+Psychiatry&amp;title=Temperament,+character+and+symptoms+of+anxiety+and+depression+in+the+general+population&amp;author=P+Jylha&amp;author=E+Isometsa&amp;volume=21&amp;publication_year=2006&amp;pages=389-395&amp;pmid=16360306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Hettema JM, Neale MC, Myers JM, Prescott CA, Kendler KS. A population-based twin study of the relationship between neuroticism and internalizing disorders. <span><span class="ref-journal">Am J Psychiatry. </span>2006;<span class="ref-vol">163</span>:857–864.</span> [<a href="/pubmed/16648327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=A+population-based+twin+study+of+the+relationship+between+neuroticism+and+internalizing+disorders&amp;author=JM+Hettema&amp;author=MC+Neale&amp;author=JM+Myers&amp;author=CA+Prescott&amp;author=KS+Kendler&amp;volume=163&amp;publication_year=2006&amp;pages=857-864&amp;pmid=16648327&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Kendler KS, Gardner CO, Prescott CA. Toward a comprehensive developmental model for major depression in women. <span><span class="ref-journal">Am J Psychiatry. </span>2002;<span class="ref-vol">159</span>:1133–1145.</span> [<a href="/pubmed/12091191" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=Toward+a+comprehensive+developmental+model+for+major+depression+in+women&amp;author=KS+Kendler&amp;author=CO+Gardner&amp;author=CA+Prescott&amp;volume=159&amp;publication_year=2002&amp;pages=1133-1145&amp;pmid=12091191&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Kendler KS, Gardner CO, Prescott CA. Toward a comprehensive developmental model for major depression in men. <span><span class="ref-journal">Am J Psychiatry. </span>2006;<span class="ref-vol">163</span>:115–124.</span> [<a href="/pubmed/16390898" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=Toward+a+comprehensive+developmental+model+for+major+depression+in+men&amp;author=KS+Kendler&amp;author=CO+Gardner&amp;author=CA+Prescott&amp;volume=163&amp;publication_year=2006&amp;pages=115-124&amp;pmid=16390898&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Kendler KS, Gatz M, Gardner CO, Pedersen NL. Personality and major depression: a Swedish longitudinal, population-based twin study. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2006;<span class="ref-vol">63</span>:1113–1120.</span> [<a href="/pubmed/17015813" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Personality+and+major+depression:+a+Swedish+longitudinal,+population-based+twin+study&amp;author=KS+Kendler&amp;author=M+Gatz&amp;author=CO+Gardner&amp;author=NL+Pedersen&amp;volume=63&amp;publication_year=2006&amp;pages=1113-1120&amp;pmid=17015813&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Fullerton J, Cubin M, Tiwari H, Wang C, Bomhra A, Davidson S, et al.  Linkage analysis of extremely discordant and concordant sibling pairs identifies quantitative-trait loci that influence variation in the human personality trait neuroticism. <span><span class="ref-journal">Am J Hum Genet. </span>2003;<span class="ref-vol">72</span>:879–890.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180351/">PMC free article</a>]</span> [<a href="/pubmed/12612864" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Linkage+analysis+of+extremely+discordant+and+concordant+sibling+pairs+identifies+quantitative-trait+loci+that+influence+variation+in+the+human+personality+trait+neuroticism&amp;author=J+Fullerton&amp;author=M+Cubin&amp;author=H+Tiwari&amp;author=C+Wang&amp;author=A+Bomhra&amp;volume=72&amp;publication_year=2003&amp;pages=879-890&amp;pmid=12612864&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Nash MW, Huezo-Diaz P, Williamson RJ, Sterne A, Purcell S, Hoda F, et al.  Genome-wide linkage analysis of a composite index of neuroticism and mood-related scales in extreme selected sibships. <span><span class="ref-journal">Hum Mol Genet. </span>2004;<span class="ref-vol">13</span>:2173–2182.</span> [<a href="/pubmed/15351774" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+linkage+analysis+of+a+composite+index+of+neuroticism+and+mood-related+scales+in+extreme+selected+sibships&amp;author=MW+Nash&amp;author=P+Huezo-Diaz&amp;author=RJ+Williamson&amp;author=A+Sterne&amp;author=S+Purcell&amp;volume=13&amp;publication_year=2004&amp;pages=2173-2182&amp;pmid=15351774&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Neale BM, Sullivan PF, Kendler KS. A genome scan of neuroticism in nicotine dependent smokers. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2005;<span class="ref-vol">132</span>:65–69.</span> [<a href="/pubmed/15389754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=A+genome+scan+of+neuroticism+in+nicotine+dependent+smokers&amp;author=BM+Neale&amp;author=PF+Sullivan&amp;author=KS+Kendler&amp;volume=132&amp;publication_year=2005&amp;pages=65-69&amp;pmid=15389754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Ioannidis JP, Trikalinos TA, Khoury MJ. Implications of small effect sizes of individual genetic variants on the design and interpretation of genetic association studies of complex diseases. <span><span class="ref-journal">Am J Epidemiol. </span>2006;<span class="ref-vol">164</span>:609–614.</span> [<a href="/pubmed/16893921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Implications+of+small+effect+sizes+of+individual+genetic+variants+on+the+design+and+interpretation+of+genetic+association+studies+of+complex+diseases&amp;author=JP+Ioannidis&amp;author=TA+Trikalinos&amp;author=MJ+Khoury&amp;volume=164&amp;publication_year=2006&amp;pages=609-614&amp;pmid=16893921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Darvasi A, Soller M. Selective DNA pooling for determination of linkage between a molecular marker and a quantitative trait locus. <span><span class="ref-journal">Genetics. </span>1994;<span class="ref-vol">138</span>:1365–1373.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1206272/">PMC free article</a>]</span> [<a href="/pubmed/7896115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Selective+DNA+pooling+for+determination+of+linkage+between+a+molecular+marker+and+a+quantitative+trait+locus&amp;author=A+Darvasi&amp;author=M+Soller&amp;volume=138&amp;publication_year=1994&amp;pages=1365-1373&amp;pmid=7896115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Butcher LM, Meaburn E, Liu L, Fernandes C, Hill L, Al-Chalabi A, et al.  Genotyping pooled DNA on microarrays: a systematic genome screen of thousands of SNPs in large samples to detect QTLs for complex traits. <span><span class="ref-journal">Behav Genet. </span>2004;<span class="ref-vol">34</span>:549–555.</span> [<a href="/pubmed/15319578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behav+Genet&amp;title=Genotyping+pooled+DNA+on+microarrays:+a+systematic+genome+screen+of+thousands+of+SNPs+in+large+samples+to+detect+QTLs+for+complex+traits&amp;author=LM+Butcher&amp;author=E+Meaburn&amp;author=L+Liu&amp;author=C+Fernandes&amp;author=L+Hill&amp;volume=34&amp;publication_year=2004&amp;pages=549-555&amp;pmid=15319578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Craig I, Meaburn E, Butcher L, Hill L, Plomin R. Single-nucleotide polymorphism genotyping in DNA pools. <span><span class="ref-journal">Methods Mol Biol. </span>2005;<span class="ref-vol">311</span>:147–164.</span> [<a href="/pubmed/16100406" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=Single-nucleotide+polymorphism+genotyping+in+DNA+pools&amp;author=I+Craig&amp;author=E+Meaburn&amp;author=L+Butcher&amp;author=L+Hill&amp;author=R+Plomin&amp;volume=311&amp;publication_year=2005&amp;pages=147-164&amp;pmid=16100406&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Kirov G, Nikolov I, Georgieva L, Moskvina V, Owen MJ, O’Donovan MC. Pooled DNA genotyping on Affymetrix SNP genotyping arrays. <span><span class="ref-journal">BMC Genomics. </span>2006;<span class="ref-vol">7</span>:27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1382214/">PMC free article</a>]</span> [<a href="/pubmed/16480507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Pooled+DNA+genotyping+on+Affymetrix+SNP+genotyping+arrays&amp;author=G+Kirov&amp;author=I+Nikolov&amp;author=L+Georgieva&amp;author=V+Moskvina&amp;author=MJ+Owen&amp;volume=7&amp;publication_year=2006&amp;pages=27&amp;pmid=16480507&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Macgregor S, Visscher PM, Montgomery G. Analysis of pooled DNA samples on high density arrays without prior knowledge of differential hybridization rates. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:e55.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1440945/">PMC free article</a>]</span> [<a href="/pubmed/16627870" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Analysis+of+pooled+DNA+samples+on+high+density+arrays+without+prior+knowledge+of+differential+hybridization+rates&amp;author=S+Macgregor&amp;author=PM+Visscher&amp;author=G+Montgomery&amp;volume=34&amp;publication_year=2006&amp;pages=e55&amp;pmid=16627870&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Meaburn E, Butcher LM, Schalkwyk LC, Plomin R. Genotyping pooled DNA using 100K SNP microarrays: a step towards genomewide association scans. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:e27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1368655/">PMC free article</a>]</span> [<a href="/pubmed/16478714" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Genotyping+pooled+DNA+using+100K+SNP+microarrays:+a+step+towards+genomewide+association+scans&amp;author=E+Meaburn&amp;author=LM+Butcher&amp;author=LC+Schalkwyk&amp;author=R+Plomin&amp;volume=34&amp;publication_year=2006&amp;pages=e27&amp;pmid=16478714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Willis-Owen SA, Turri MG, Munafo MR, Surtees PG, Wainwright NW, Brixey RD, et al.  The serotonin transporter length polymorphism, neuroticism, and depression: a comprehensive assessment of association. <span><span class="ref-journal">Biol Psychiatry. </span>2005;<span class="ref-vol">58</span>:451–456.</span> [<a href="/pubmed/16023085" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=The+serotonin+transporter+length+polymorphism,+neuroticism,+and+depression:+a+comprehensive+assessment+of+association&amp;author=SA+Willis-Owen&amp;author=MG+Turri&amp;author=MR+Munafo&amp;author=PG+Surtees&amp;author=NW+Wainwright&amp;volume=58&amp;publication_year=2005&amp;pages=451-456&amp;pmid=16023085&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Willis-Owen SA, Shifman S, Copley RR, Flint J. DCNP1: a novel candidate gene for major depression. <span><span class="ref-journal">Mol Psychiatry. </span>2006;<span class="ref-vol">11</span>:121–122.</span> [<a href="/pubmed/16189510" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=DCNP1:+a+novel+candidate+gene+for+major+depression&amp;author=SA+Willis-Owen&amp;author=S+Shifman&amp;author=RR+Copley&amp;author=J+Flint&amp;volume=11&amp;publication_year=2006&amp;pages=121-122&amp;pmid=16189510&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Willis-Owen SA, Fullerton J, Surtees PG, Wainwright NW, Miller S, Flint J. The Val66Met coding variant of the brain-derived neurotrophic factor (BDNF) gene does not contribute toward variation in the personality trait neuroticism. <span><span class="ref-journal">Biol Psychiatry. </span>2005;<span class="ref-vol">58</span>:738–742.</span> [<a href="/pubmed/16043130" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=The+Val66Met+coding+variant+of+the+brain-derived+neurotrophic+factor+(BDNF)+gene+does+not+contribute+toward+variation+in+the+personality+trait+neuroticism&amp;author=SA+Willis-Owen&amp;author=J+Fullerton&amp;author=PG+Surtees&amp;author=NW+Wainwright&amp;author=S+Miller&amp;volume=58&amp;publication_year=2005&amp;pages=738-742&amp;pmid=16043130&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Martin N, Goodwin G, Fairburn C, Wilson R, Allison D, Cardon LR, et al.  A population based study of personality in 34 000 sib-pairs. <span><span class="ref-journal">Twin Res. </span>2000;<span class="ref-vol">3</span>:310–315.</span> [<a href="/pubmed/11463152" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res&amp;title=A+population+based+study+of+personality+in+34+000+sib-pairs&amp;author=N+Martin&amp;author=G+Goodwin&amp;author=C+Fairburn&amp;author=R+Wilson&amp;author=D+Allison&amp;volume=3&amp;publication_year=2000&amp;pages=310-315&amp;pmid=11463152&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Eysenck HJ, Eysenck SBG, editors. <span class="ref-journal">Manual of the Eysenck Personality Questionnaire.</span> San Diego, CA: Educational and Industrial Testing Service; 1975.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Manual+of+the+Eysenck+Personality+Questionnaire&amp;publication_year=1975&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Aluja A, García O, Garcia LF. A comparative study of Zuckerman’s three structural models for personality through the NEO-PI-R: ZKPQ-III-R, EPQ-RS and Goldberg’s 50-bipolar adjectives. <span><span class="ref-journal">Pers Individ Dif. </span>2002;<span class="ref-vol">33</span>:713–725.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pers+Individ+Dif&amp;title=A+comparative+study+of+Zuckerman’s+three+structural+models+for+personality+through+the+NEO-PI-R:+ZKPQ-III-R,+EPQ-RS+and+Goldberg’s+50-bipolar+adjectives&amp;author=A+Aluja&amp;author=O+García&amp;author=LF+Garcia&amp;volume=33&amp;publication_year=2002&amp;pages=713-725&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">De Fruyt F, Van De Wiele L, Van Heeringen C. Cloninger’s psychobiological model of temperament and character and the five-factor model of personality. <span><span class="ref-journal">Pers Individ Dif. </span>2000;<span class="ref-vol">29</span>:441–452.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pers+Individ+Dif&amp;title=Cloninger’s+psychobiological+model+of+temperament+and+character+and+the+five-factor+model+of+personality&amp;author=F+De+Fruyt&amp;author=L+Van+De+Wiele&amp;author=C+Van+Heeringen&amp;volume=29&amp;publication_year=2000&amp;pages=441-452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Shifman S, Pisante-Shalom A, Yakir B, Darvasi A. Quantitative technologies for allele frequency estimation of SNPs in DNA pools. <span><span class="ref-journal">Mol Cell Probes. </span>2002;<span class="ref-vol">16</span>:429–434.</span> [<a href="/pubmed/12490144" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Probes&amp;title=Quantitative+technologies+for+allele+frequency+estimation+of+SNPs+in+DNA+pools&amp;author=S+Shifman&amp;author=A+Pisante-Shalom&amp;author=B+Yakir&amp;author=A+Darvasi&amp;volume=16&amp;publication_year=2002&amp;pages=429-434&amp;pmid=12490144&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Meaburn E, Butcher LM, Liu L, Fernandes C, Hansen V, Al-Chalabi A, et al.  Genotyping DNA pools on microarrays: tackling the QTL problem of large samples and large numbers of SNPs. <span><span class="ref-journal">BMC Genomics. </span>2005;<span class="ref-vol">6</span>:52.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1079828/">PMC free article</a>]</span> [<a href="/pubmed/15811185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Genotyping+DNA+pools+on+microarrays:+tackling+the+QTL+problem+of+large+samples+and+large+numbers+of+SNPs&amp;author=E+Meaburn&amp;author=LM+Butcher&amp;author=L+Liu&amp;author=C+Fernandes&amp;author=V+Hansen&amp;volume=6&amp;publication_year=2005&amp;pages=52&amp;pmid=15811185&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Altshuler D, Brooks LD, Chakravarti A, Collins FS, Daly MJ, Donnelly P. A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;author=D+Altshuler&amp;author=LD+Brooks&amp;author=A+Chakravarti&amp;author=FS+Collins&amp;author=MJ+Daly&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Schaid DJ, Rowland CM, Tines DE, Jacobson RM, Poland GA. Score tests for association between traits and haplotypes when linkage phase is ambiguous. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">70</span>:425–434.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384917/">PMC free article</a>]</span> [<a href="/pubmed/11791212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Score+tests+for+association+between+traits+and+haplotypes+when+linkage+phase+is+ambiguous&amp;author=DJ+Schaid&amp;author=CM+Rowland&amp;author=DE+Tines&amp;author=RM+Jacobson&amp;author=GA+Poland&amp;volume=70&amp;publication_year=2002&amp;pages=425-434&amp;pmid=11791212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Schaid DJ. Evaluating associations of haplotypes with traits. <span><span class="ref-journal">Genet Epidemiol. </span>2004;<span class="ref-vol">27</span>:348–364.</span> [<a href="/pubmed/15543638" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Evaluating+associations+of+haplotypes+with+traits&amp;author=DJ+Schaid&amp;volume=27&amp;publication_year=2004&amp;pages=348-364&amp;pmid=15543638&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Redon R, Ishikawa S, Fitch KR, Feuk L, Perry GH, Andrews TD, et al.  Global variation in copy number in the human genome. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">444</span>:444–454.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2669898/">PMC free article</a>]</span> [<a href="/pubmed/17122850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Global+variation+in+copy+number+in+the+human+genome&amp;author=R+Redon&amp;author=S+Ishikawa&amp;author=KR+Fitch&amp;author=L+Feuk&amp;author=GH+Perry&amp;volume=444&amp;publication_year=2006&amp;pages=444-454&amp;pmid=17122850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Kirk KM, Birley AJ, Statham DJ, Haddon B, Lake RI, Andrews JG, et al.  Anxiety and depression in twin and sib pairs extremely discordant and concordant for neuroticism: prodromus to a linkage study. <span><span class="ref-journal">Twin Res. </span>2000;<span class="ref-vol">3</span>:299–309.</span> [<a href="/pubmed/11463151" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res&amp;title=Anxiety+and+depression+in+twin+and+sib+pairs+extremely+discordant+and+concordant+for+neuroticism:+prodromus+to+a+linkage+study&amp;author=KM+Kirk&amp;author=AJ+Birley&amp;author=DJ+Statham&amp;author=B+Haddon&amp;author=RI+Lake&amp;volume=3&amp;publication_year=2000&amp;pages=299-309&amp;pmid=11463151&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Hettema JM, An SS, Neale MC, Bukszar J, van den Oord EJ, Kendler KS, et al.  Association between glutamic acid decarboxylase genes and anxiety disorders, major depression, and neuroticism. <span><span class="ref-journal">Mol Psychiatry. </span>2006;<span class="ref-vol">11</span>:752–762.</span> [<a href="/pubmed/16718280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Association+between+glutamic+acid+decarboxylase+genes+and+anxiety+disorders,+major+depression,+and+neuroticism&amp;author=JM+Hettema&amp;author=SS+An&amp;author=MC+Neale&amp;author=J+Bukszar&amp;author=EJ+van+den+Oord&amp;volume=11&amp;publication_year=2006&amp;pages=752-762&amp;pmid=16718280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Boomsma DI, Beem AL, van den Berg M, Dolan CV, Koopmans JR, Vink JM, et al.  Netherlands twin family study of anxious depression (NETSAD) <span><span class="ref-journal">Twin Res. </span>2000;<span class="ref-vol">3</span>:323–334.</span> [<a href="/pubmed/11463154" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res&amp;title=Netherlands+twin+family+study+of+anxious+depression+(NETSAD)&amp;author=DI+Boomsma&amp;author=AL+Beem&amp;author=M+van+den+Berg&amp;author=CV+Dolan&amp;author=JR+Koopmans&amp;volume=3&amp;publication_year=2000&amp;pages=323-334&amp;pmid=11463154&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Middeldorp CM, Birley AJ, Cath DC, Gillespie NA, Willemsen G, Statham DJ, et al.  Familial clustering of major depression and anxiety disorders in Australian and Dutch twins and siblings. <span><span class="ref-journal">Twin Res Hum Genet. </span>2005;<span class="ref-vol">8</span>:609–615.</span> [<a href="/pubmed/16354503" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res+Hum+Genet&amp;title=Familial+clustering+of+major+depression+and+anxiety+disorders+in+Australian+and+Dutch+twins+and+siblings&amp;author=CM+Middeldorp&amp;author=AJ+Birley&amp;author=DC+Cath&amp;author=NA+Gillespie&amp;author=G+Willemsen&amp;volume=8&amp;publication_year=2005&amp;pages=609-615&amp;pmid=16354503&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Rettew DC, Vink JM, Willemsen G, Doyle A, Hudziak JJ, Boomsma DI. The genetic architecture of neuroticism in 3301 Dutch adolescent twins as a function of age and sex: a study from the Dutch twin register. <span><span class="ref-journal">Twin Res Hum Genet. </span>2006;<span class="ref-vol">9</span>:24–29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3319038/">PMC free article</a>]</span> [<a href="/pubmed/16611464" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res+Hum+Genet&amp;title=The+genetic+architecture+of+neuroticism+in+3301+Dutch+adolescent+twins+as+a+function+of+age+and+sex:+a+study+from+the+Dutch+twin+register&amp;author=DC+Rettew&amp;author=JM+Vink&amp;author=G+Willemsen&amp;author=A+Doyle&amp;author=JJ+Hudziak&amp;volume=9&amp;publication_year=2006&amp;pages=24-29&amp;pmid=16611464&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Jardine R, Martin NG, Henderson AS. Genetic covariation between neuroticism and the symptoms of anxiety and depression. <span><span class="ref-journal">Genet Epidemiol. </span>1984;<span class="ref-vol">1</span>:89–107.</span> [<a href="/pubmed/6544237" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Genetic+covariation+between+neuroticism+and+the+symptoms+of+anxiety+and+depression&amp;author=R+Jardine&amp;author=NG+Martin&amp;author=AS+Henderson&amp;volume=1&amp;publication_year=1984&amp;pages=89-107&amp;pmid=6544237&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Kessler R, Mroczek D.  <span class="ref-journal">Final Version of the Psychological Distress Scale. Technical Note.</span> Ann Arbor, MI: Institute for Social Research, University of Michigan; 1994.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Final+Version+of+the+Psychological+Distress+Scale.+Technical+Note&amp;author=R+Kessler&amp;author=D+Mroczek&amp;publication_year=1994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Sham P, Bader JS, Craig I, O’Donovan M, Owen M. DNA pooling: a tool for large-scale association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2002;<span class="ref-vol">3</span>:862–871.</span> [<a href="/pubmed/12415316" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=DNA+pooling:+a+tool+for+large-scale+association+studies&amp;author=P+Sham&amp;author=JS+Bader&amp;author=I+Craig&amp;author=M+O’Donovan&amp;author=M+Owen&amp;volume=3&amp;publication_year=2002&amp;pages=862-871&amp;pmid=12415316&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Mohlke KL, Erdos MR, Scott LJ, Fingerlin TE, Jackson AU, Silander K, et al.  High-throughput screening for evidence of association by using mass spectrometry genotyping on DNA pools. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2002;<span class="ref-vol">99</span>:16928–16933.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC139246/">PMC free article</a>]</span> [<a href="/pubmed/12482934" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=High-throughput+screening+for+evidence+of+association+by+using+mass+spectrometry+genotyping+on+DNA+pools&amp;author=KL+Mohlke&amp;author=MR+Erdos&amp;author=LJ+Scott&amp;author=TE+Fingerlin&amp;author=AU+Jackson&amp;volume=99&amp;publication_year=2002&amp;pages=16928-16933&amp;pmid=12482934&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Barratt BJ, Payne F, Rance HE, Nutland S, Todd JA, Clayton DG. Identification of the sources of error in allele frequency estimations from pooled DNA indicates an optimal experimental design. <span><span class="ref-journal">Ann Hum Genet. </span>2002;<span class="ref-vol">66</span>:393–405.</span> [<a href="/pubmed/12485472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Identification+of+the+sources+of+error+in+allele+frequency+estimations+from+pooled+DNA+indicates+an+optimal+experimental+design&amp;author=BJ+Barratt&amp;author=F+Payne&amp;author=HE+Rance&amp;author=S+Nutland&amp;author=JA+Todd&amp;volume=66&amp;publication_year=2002&amp;pages=393-405&amp;pmid=12485472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Norton N, Williams HJ, Williams NM, Spurlock G, Zammit S, Jones G, et al.  Mutation screening of the Homer gene family and association analysis in schizophrenia. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2003;<span class="ref-vol">120</span>:18–21.</span> [<a href="/pubmed/12815733" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Mutation+screening+of+the+Homer+gene+family+and+association+analysis+in+schizophrenia&amp;author=N+Norton&amp;author=HJ+Williams&amp;author=NM+Williams&amp;author=G+Spurlock&amp;author=S+Zammit&amp;volume=120&amp;publication_year=2003&amp;pages=18-21&amp;pmid=12815733&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Shifman S, Bronstein M, Sternfeld M, Pisante-Shalom A, Lev-Lehman E, Weizman A, et al.  A highly significant association between a COMT haplotype and schizophrenia. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">71</span>:1296–1302.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC378567/">PMC free article</a>]</span> [<a href="/pubmed/12402217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+highly+significant+association+between+a+COMT+haplotype+and+schizophrenia&amp;author=S+Shifman&amp;author=M+Bronstein&amp;author=M+Sternfeld&amp;author=A+Pisante-Shalom&amp;author=E+Lev-Lehman&amp;volume=71&amp;publication_year=2002&amp;pages=1296-1302&amp;pmid=12402217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Sawcer S, Maranian M, Setakis E, Curwen V, Akesson E, Hensiek A, et al.  A whole genome screen for linkage disequilibrium in multiple sclerosis confirms disease associations with regions previously linked to susceptibility. <span><span class="ref-journal">Brain. </span>2002;<span class="ref-vol">125</span>:1337–1347.</span> [<a href="/pubmed/12023322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=A+whole+genome+screen+for+linkage+disequilibrium+in+multiple+sclerosis+confirms+disease+associations+with+regions+previously+linked+to+susceptibility&amp;author=S+Sawcer&amp;author=M+Maranian&amp;author=E+Setakis&amp;author=V+Curwen&amp;author=E+Akesson&amp;volume=125&amp;publication_year=2002&amp;pages=1337-1347&amp;pmid=12023322&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Bansal A, van den Boom D, Kammerer S, Honisch C, Adam G, Cantor CR, et al.  Association testing by DNA pooling: an effective initial screen. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2002;<span class="ref-vol">99</span>:16871–16874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC139236/">PMC free article</a>]</span> [<a href="/pubmed/12475937" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Association+testing+by+DNA+pooling:+an+effective+initial+screen&amp;author=A+Bansal&amp;author=D+van+den+Boom&amp;author=S+Kammerer&amp;author=C+Honisch&amp;author=G+Adam&amp;volume=99&amp;publication_year=2002&amp;pages=16871-16874&amp;pmid=12475937&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Hinds DA, Seymour AB, Durham LK, Banerjee P, Ballinger DG, Milos PM, et al.  Application of pooled genotyping to scan candidate regions for association with HDL cholesterol levels. <span><span class="ref-journal">Hum Genomics. </span>2004;<span class="ref-vol">1</span>:421–434.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3500196/">PMC free article</a>]</span> [<a href="/pubmed/15606997" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genomics&amp;title=Application+of+pooled+genotyping+to+scan+candidate+regions+for+association+with+HDL+cholesterol+levels&amp;author=DA+Hinds&amp;author=AB+Seymour&amp;author=LK+Durham&amp;author=P+Banerjee&amp;author=DG+Ballinger&amp;volume=1&amp;publication_year=2004&amp;pages=421-434&amp;pmid=15606997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Papassotiropoulos A, Stephan DA, Huentelman MJ, Hoerndli FJ, Craig DW, Pearson JV, et al.  Common Kibra alleles are associated with human memory performance. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">314</span>:475–478.</span> [<a href="/pubmed/17053149" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Common+Kibra+alleles+are+associated+with+human+memory+performance&amp;author=A+Papassotiropoulos&amp;author=DA+Stephan&amp;author=MJ+Huentelman&amp;author=FJ+Hoerndli&amp;author=DW+Craig&amp;volume=314&amp;publication_year=2006&amp;pages=475-478&amp;pmid=17053149&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Butcher LM, Meaburn E, Knight J, Sham PC, Schalkwyk LC, Craig IW, et al.  SNPs, microarrays and pooled DNA: identification of four loci associated with mild mental impairment in a sample of 6000 children. <span><span class="ref-journal">Hum Mol Genet. </span>2005;<span class="ref-vol">14</span>:1315–1325.</span> [<a href="/pubmed/15800012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=SNPs,+microarrays+and+pooled+DNA:+identification+of+four+loci+associated+with+mild+mental+impairment+in+a+sample+of+6000+children&amp;author=LM+Butcher&amp;author=E+Meaburn&amp;author=J+Knight&amp;author=PC+Sham&amp;author=LC+Schalkwyk&amp;volume=14&amp;publication_year=2005&amp;pages=1315-1325&amp;pmid=15800012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Ioannidis JP, Trikalinos TA, Khoury MJ. Implications of small effect sizes of individual genetic variants on the design and interpretation of genetic association studies of complex diseases. <span><span class="ref-journal">Am J Epidemiol. </span>2006;<span class="ref-vol">164</span>:609–614.</span> [<a href="/pubmed/16893921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Implications+of+small+effect+sizes+of+individual+genetic+variants+on+the+design+and+interpretation+of+genetic+association+studies+of+complex+diseases&amp;author=JP+Ioannidis&amp;author=TA+Trikalinos&amp;author=MJ+Khoury&amp;volume=164&amp;publication_year=2006&amp;pages=609-614&amp;pmid=16893921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Lee JA, Lupski JR. Genomic rearrangements and gene copynumber alterations as a cause of nervous system disorders. <span><span class="ref-journal">Neuron. </span>2006;<span class="ref-vol">52</span>:103–121.</span> [<a href="/pubmed/17015230" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Genomic+rearrangements+and+gene+copynumber+alterations+as+a+cause+of+nervous+system+disorders&amp;author=JA+Lee&amp;author=JR+Lupski&amp;volume=52&amp;publication_year=2006&amp;pages=103-121&amp;pmid=17015230&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Wilson GM, Flibotte S, Chopra V, Melnyk BL, Honer WG, Holt RA. DNA copy-number analysis in bipolar disorder and schizophrenia reveals aberrations in genes involved in glutamate signaling. <span><span class="ref-journal">Hum Mol Genet. </span>2006;<span class="ref-vol">15</span>:743–749.</span> [<a href="/pubmed/16434481" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=DNA+copy-number+analysis+in+bipolar+disorder+and+schizophrenia+reveals+aberrations+in+genes+involved+in+glutamate+signaling&amp;author=GM+Wilson&amp;author=S+Flibotte&amp;author=V+Chopra&amp;author=BL+Melnyk&amp;author=WG+Honer&amp;volume=15&amp;publication_year=2006&amp;pages=743-749&amp;pmid=16434481&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Smith GW, Aubry JM, Dellu F, Contarino A, Bilezikjian LM, Gold LH, et al.  Corticotropin releasing factor receptor 1-deficient mice display decreased anxiety, impaired stress response, and aberrant neuroendocrine development. <span><span class="ref-journal">Neuron. </span>1998;<span class="ref-vol">20</span>:1093–1102.</span> [<a href="/pubmed/9655498" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Corticotropin+releasing+factor+receptor+1-deficient+mice+display+decreased+anxiety,+impaired+stress+response,+and+aberrant+neuroendocrine+development&amp;author=GW+Smith&amp;author=JM+Aubry&amp;author=F+Dellu&amp;author=A+Contarino&amp;author=LM+Bilezikjian&amp;volume=20&amp;publication_year=1998&amp;pages=1093-1102&amp;pmid=9655498&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Benbrook DM, Jones NC. Heterodimer formation between CREB and JUN proteins. <span><span class="ref-journal">Oncogene. </span>1990;<span class="ref-vol">5</span>:295–302.</span> [<a href="/pubmed/2138276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Heterodimer+formation+between+CREB+and+JUN+proteins&amp;author=DM+Benbrook&amp;author=NC+Jones&amp;volume=5&amp;publication_year=1990&amp;pages=295-302&amp;pmid=2138276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Benbrook DM, Jones NC. Different binding specificities and transactivation of variant CRE’s by CREB complexes. <span><span class="ref-journal">Nucleic Acids Res. </span>1994;<span class="ref-vol">22</span>:1463–1469.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC308006/">PMC free article</a>]</span> [<a href="/pubmed/8190638" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Different+binding+specificities+and+transactivation+of+variant+CRE’s+by+CREB+complexes&amp;author=DM+Benbrook&amp;author=NC+Jones&amp;volume=22&amp;publication_year=1994&amp;pages=1463-1469&amp;pmid=8190638&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="element-citation">Le Jeune IR, Shepherd M, Van Heeke G, Houslay MD, Hall IP. Cyclic AMP-dependent transcriptional up-regulation of phosphodiesterase 4D5 in human airway smooth muscle cells. Identification and characterization of a novel PDE4D5 promoter. <span><span class="ref-journal">J Biol Chem. </span>2002;<span class="ref-vol">277</span>:35980–35989.</span> [<a href="/pubmed/12121997" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Cyclic+AMP-dependent+transcriptional+up-regulation+of+phosphodiesterase+4D5+in+human+airway+smooth+muscle+cells.+Identification+and+characterization+of+a+novel+PDE4D5+promoter&amp;author=IR+Le+Jeune&amp;author=M+Shepherd&amp;author=G+Van+Heeke&amp;author=MD+Houslay&amp;author=IP+Hall&amp;volume=277&amp;publication_year=2002&amp;pages=35980-35989&amp;pmid=12121997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="element-citation">Fleischhacker WW, Hinterhuber H, Bauer H, Pflug B, Berner P, Simhandl C, et al.  A multicenter double-blind study of three different doses of the new cAMP-phosphodiesterase inhibitor rolipram in patients with major depressive disorder. <span><span class="ref-journal">Neuropsychobiology. </span>1992;<span class="ref-vol">26</span>:59–64.</span> [<a href="/pubmed/1475038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychobiology&amp;title=A+multicenter+double-blind+study+of+three+different+doses+of+the+new+cAMP-phosphodiesterase+inhibitor+rolipram+in+patients+with+major+depressive+disorder&amp;author=WW+Fleischhacker&amp;author=H+Hinterhuber&amp;author=H+Bauer&amp;author=B+Pflug&amp;author=P+Berner&amp;volume=26&amp;publication_year=1992&amp;pages=59-64&amp;pmid=1475038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="element-citation">Scott AI, Perini AF, Shering PA, Whalley LJ. In-patient major depression: is rolipram as effective as amitriptyline? <span><span class="ref-journal">Eur J Clin Pharmacol. </span>1991;<span class="ref-vol">40</span>:127–129.</span> [<a href="/pubmed/2065693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Clin+Pharmacol&amp;title=In-patient+major+depression:+is+rolipram+as+effective+as+amitriptyline?&amp;author=AI+Scott&amp;author=AF+Perini&amp;author=PA+Shering&amp;author=LJ+Whalley&amp;volume=40&amp;publication_year=1991&amp;pages=127-129&amp;pmid=2065693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="element-citation">Zhang HT, Huang Y, Jin SL, Frith SA, Suvarna N, Conti M, et al.  Antidepressant-like profile and reduced sensitivity to rolipram in mice deficient in the PDE4D phosphodiesterase enzyme. <span><span class="ref-journal">Neuropsychopharmacology. </span>2002;<span class="ref-vol">27</span>:587–595.</span> [<a href="/pubmed/12377395" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&amp;title=Antidepressant-like+profile+and+reduced+sensitivity+to+rolipram+in+mice+deficient+in+the+PDE4D+phosphodiesterase+enzyme&amp;author=HT+Zhang&amp;author=Y+Huang&amp;author=SL+Jin&amp;author=SA+Frith&amp;author=N+Suvarna&amp;volume=27&amp;publication_year=2002&amp;pages=587-595&amp;pmid=12377395&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="element-citation">Wong ML, Whelan F, Deloukas P, Whittaker P, Delgado M, Cantor RM, et al.  Phosphodiesterase genes are associated with susceptibility to major depression and antidepressant treatment response. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2006;<span class="ref-vol">103</span>:15124–15129.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1581426/">PMC free article</a>]</span> [<a href="/pubmed/17008408" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Phosphodiesterase+genes+are+associated+with+susceptibility+to+major+depression+and+antidepressant+treatment+response&amp;author=ML+Wong&amp;author=F+Whelan&amp;author=P+Deloukas&amp;author=P+Whittaker&amp;author=M+Delgado&amp;volume=103&amp;publication_year=2006&amp;pages=15124-15129&amp;pmid=17008408&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4004964&amp;issue-id=237027&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139963725389408title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials and methods</a></li><li><a href="#S9title">Results</a></li><li><a href="#S16title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#idm139963717925120title">Footnotes</a></li><li><a href="#idm139963717923168title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4004964/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4004964/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4004964/pdf/nihms572905.pdf">PDF (881K)</a></li> | <li><a href="#" data-citationid="PMC4004964" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4004964%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4004964%2F&amp;text=A%20whole%20genome%20association%20study%20of%20neuroticism%20using%20DNA%20pooling" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4004964%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="17667963" data-aiid="4004964" data-aid="4004964" data-iid="237027" data-domainid="319" data-domain="nihpa" data-accid="PMC4004964" data-md5="ede866c6657a80348b3c72e184666ab9"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T08:48:43-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE890FBFDD3EFFA10000000002230151&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4004964%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4004964/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size1, monospace;"></div></div></body></html>