<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="blood"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4492197/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide analysis links NFATC2 with asparaginase hypersensitivity</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4492197/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Blood"><meta name="citation_title" content="Genome-wide analysis links NFATC2 with asparaginase hypersensitivity"><meta name="citation_authors" content="Christian A. Fernandez, Colton Smith, Wenjian Yang, Charles G. Mullighan, Chunxu Qu, Eric Larsen, W. Paul Bowman, Chengcheng Liu, Laura B. Ramsey, Tamara Chang, Seth E. Karol, Mignon L. Loh, Elizabeth A. Raetz, Naomi J. Winick, Stephen P. Hunger, William L. Carroll, Sima Jeha, Ching-Hon Pui, William E. Evans, Meenakshi Devidas, Mary V. Relling"><meta name="citation_date" content="2 July 2015"><meta name="citation_issue" content="1"><meta name="citation_volume" content="126"><meta name="citation_firstpage" content="69"><meta name="citation_doi" content="10.1182/blood-2015-02-628800"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4492197/?report=abstract"><meta name="citation_pmid" content="25987655"><meta name="DC.Title" content="Genome-wide analysis links NFATC2 with asparaginase hypersensitivity"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="The American Society of Hematology"><meta name="DC.Contributor" content="Christian A. Fernandez"><meta name="DC.Contributor" content="Colton Smith"><meta name="DC.Contributor" content="Wenjian Yang"><meta name="DC.Contributor" content="Charles G. Mullighan"><meta name="DC.Contributor" content="Chunxu Qu"><meta name="DC.Contributor" content="Eric Larsen"><meta name="DC.Contributor" content="W. Paul Bowman"><meta name="DC.Contributor" content="Chengcheng Liu"><meta name="DC.Contributor" content="Laura B. Ramsey"><meta name="DC.Contributor" content="Tamara Chang"><meta name="DC.Contributor" content="Seth E. Karol"><meta name="DC.Contributor" content="Mignon L. Loh"><meta name="DC.Contributor" content="Elizabeth A. Raetz"><meta name="DC.Contributor" content="Naomi J. Winick"><meta name="DC.Contributor" content="Stephen P. Hunger"><meta name="DC.Contributor" content="William L. Carroll"><meta name="DC.Contributor" content="Sima Jeha"><meta name="DC.Contributor" content="Ching-Hon Pui"><meta name="DC.Contributor" content="William E. Evans"><meta name="DC.Contributor" content="Meenakshi Devidas"><meta name="DC.Contributor" content="Mary V. Relling"><meta name="DC.Date" content="2015 Jul 2"><meta name="DC.Identifier" content="10.1182/blood-2015-02-628800"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide analysis links NFATC2 with asparaginase hypersensitivity"><meta property="og:type" content="article"><meta property="og:description" content="Asparaginase is used to treat acute lymphoblastic leukemia (ALL); however, hypersensitivity reactions can lead to suboptimal asparaginase exposure. Our objective was to use a genome-wide approach to identify loci associated with asparaginase hypersensitivity ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4492197/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-blood.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4492197/epub/">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A9D5EDD57FB210000000005BF034A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4492197%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Blood%22[journal]" class="navlink">Blood</a></li><li class="accid">PMC4492197</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-blood.gif" alt="Logo of blood" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="139,56,200,76" alt="Original Article" title="Original Article" href="http://www.bloodjournal.org/content/126/1/69" ref="reftype=publisher&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7CArticle%7CFullText" target="pmc_ext"><area shape="rect" coords="0,0,157,57" alt="Blood Journal" title="Blood Journal" href="http://www.bloodjournal.org/" ref="reftype=publisher&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="200,56,258,76" alt="Current Issue" title="Current Issue" href="http://www.bloodjournal.org/content/current" ref="reftype=publisher&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="257,56,311,76" alt="About Blood" title="About Blood" href="http://www.bloodjournal.org/page/about-blood" ref="reftype=publisher&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="309,56,364,77" alt="Submissions" title="Submissions" href="https://submit.bloodjournal.org/" ref="reftype=publisher&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="363,56,420,78" alt="Subscriptions" title="Subscriptions" href="http://www.bloodjournal.org/page/subscriptions" ref="reftype=publisher&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="418,56,476,76" alt="Contact Us" title="Contact Us" href="http://bloodjournal.org" ref="reftype=publisher&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="poly" coords="450,2,431,9,425,27,429,43,437,54,459,55,471,52,483,42,486,29,479,11,466,2,449,2" alt="ASH Homepage" title="ASH Homepage" href="http://www.hematology.org" ref="reftype=publisher&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Blood</a></span>. 2015 Jul 2; 126(1): 69–75. </span></div><div><span class="fm-vol-iss-date">Prepublished online 2015 May 18. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1182%2Fblood-2015-02-628800" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1182/blood-2015-02-628800</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4492197</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25987655">25987655</a></div></div></div></div><h1 class="content-title">Genome-wide analysis links <em>NFATC2</em> with asparaginase hypersensitivity</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Fernandez%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563554288544" co-class="co-affbox">Christian A. Fernandez</a>,<sup>1</sup> <a href="/pubmed/?term=Smith%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563471865824" co-class="co-affbox">Colton Smith</a>,<sup>1</sup> <a href="/pubmed/?term=Yang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563484685456" co-class="co-affbox">Wenjian Yang</a>,<sup>1</sup> <a href="/pubmed/?term=Mullighan%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563484683472" co-class="co-affbox">Charles G. Mullighan</a>,<sup>2</sup> <a href="/pubmed/?term=Qu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563485727072" co-class="co-affbox">Chunxu Qu</a>,<sup>2</sup> <a href="/pubmed/?term=Larsen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563482944016" co-class="co-affbox">Eric Larsen</a>,<sup>3</sup> <a href="/pubmed/?term=Bowman%20WP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563481345312" co-class="co-affbox">W. Paul Bowman</a>,<sup>4</sup> <a href="/pubmed/?term=Liu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563481276944" co-class="co-affbox">Chengcheng Liu</a>,<sup>1</sup> <a href="/pubmed/?term=Ramsey%20LB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563480493216" co-class="co-affbox">Laura B. Ramsey</a>,<sup>1</sup> <a href="/pubmed/?term=Chang%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563480491344" co-class="co-affbox">Tamara Chang</a>,<sup>5</sup> <a href="/pubmed/?term=Karol%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563483313536" co-class="co-affbox">Seth E. Karol</a>,<sup>5</sup> <a href="/pubmed/?term=Loh%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563480845680" co-class="co-affbox">Mignon L. Loh</a>,<sup>6</sup> <a href="/pubmed/?term=Raetz%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563472506816" co-class="co-affbox">Elizabeth A. Raetz</a>,<sup>7</sup> <a href="/pubmed/?term=Winick%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563472504832" co-class="co-affbox">Naomi J. Winick</a>,<sup>8</sup> <a href="/pubmed/?term=Hunger%20SP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563486416128" co-class="co-affbox">Stephen P. Hunger</a>,<sup>9</sup> <a href="/pubmed/?term=Carroll%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563480906032" co-class="co-affbox">William L. Carroll</a>,<sup>10</sup> <a href="/pubmed/?term=Jeha%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563485343344" co-class="co-affbox">Sima Jeha</a>,<sup>5</sup> <a href="/pubmed/?term=Pui%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563481466368" co-class="co-affbox">Ching-Hon Pui</a>,<sup>5</sup> <a href="/pubmed/?term=Evans%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563481464384" co-class="co-affbox">William E. Evans</a>,<sup>1</sup> <a href="/pubmed/?term=Devidas%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563480913856" co-class="co-affbox">Meenakshi Devidas</a>,<sup>11</sup> and  <a href="/pubmed/?term=Relling%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655" class="affpopup" co-rid="_co_idm140563478204880" co-class="co-affbox">Mary V. Relling</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140563554288544"><h3 class="no_margin">Christian A. Fernandez</h3><p><sup>1</sup>Department of Pharmaceutical Sciences, and </p><div>Find articles by <a href="/pubmed/?term=Fernandez%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Christian A. Fernandez</a></div></div><div id="_co_idm140563471865824"><h3 class="no_margin">Colton Smith</h3><p><sup>1</sup>Department of Pharmaceutical Sciences, and </p><div>Find articles by <a href="/pubmed/?term=Smith%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Colton Smith</a></div></div><div id="_co_idm140563484685456"><h3 class="no_margin">Wenjian Yang</h3><p><sup>1</sup>Department of Pharmaceutical Sciences, and </p><div>Find articles by <a href="/pubmed/?term=Yang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Wenjian Yang</a></div></div><div id="_co_idm140563484683472"><h3 class="no_margin">Charles G. Mullighan</h3><p><sup>2</sup>Department of Pathology, St. Jude Children’s Research Hospital, Memphis, TN; </p><div>Find articles by <a href="/pubmed/?term=Mullighan%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Charles G. Mullighan</a></div></div><div id="_co_idm140563485727072"><h3 class="no_margin">Chunxu Qu</h3><p><sup>2</sup>Department of Pathology, St. Jude Children’s Research Hospital, Memphis, TN; </p><div>Find articles by <a href="/pubmed/?term=Qu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Chunxu Qu</a></div></div><div id="_co_idm140563482944016"><h3 class="no_margin">Eric Larsen</h3><p><sup>3</sup>Maine Children’s Cancer Program, Scarborough, ME; </p><div>Find articles by <a href="/pubmed/?term=Larsen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Eric Larsen</a></div></div><div id="_co_idm140563481345312"><h3 class="no_margin">W. Paul Bowman</h3><p><sup>4</sup>Department of Hematology and Oncology, Cook Children’s Medical Center, Fort Worth, TX; </p><div>Find articles by <a href="/pubmed/?term=Bowman%20WP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">W. Paul Bowman</a></div></div><div id="_co_idm140563481276944"><h3 class="no_margin">Chengcheng Liu</h3><p><sup>1</sup>Department of Pharmaceutical Sciences, and </p><div>Find articles by <a href="/pubmed/?term=Liu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Chengcheng Liu</a></div></div><div id="_co_idm140563480493216"><h3 class="no_margin">Laura B. Ramsey</h3><p><sup>1</sup>Department of Pharmaceutical Sciences, and </p><div>Find articles by <a href="/pubmed/?term=Ramsey%20LB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Laura B. Ramsey</a></div></div><div id="_co_idm140563480491344"><h3 class="no_margin">Tamara Chang</h3><p><sup>5</sup>Department of Oncology, St. Jude Children's Research Hospital, Memphis, TN; </p><div>Find articles by <a href="/pubmed/?term=Chang%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Tamara Chang</a></div></div><div id="_co_idm140563483313536"><h3 class="no_margin">Seth E. Karol</h3><p><sup>5</sup>Department of Oncology, St. Jude Children's Research Hospital, Memphis, TN; </p><div>Find articles by <a href="/pubmed/?term=Karol%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Seth E. Karol</a></div></div><div id="_co_idm140563480845680"><h3 class="no_margin">Mignon L. Loh</h3><p><sup>6</sup>Department of Pediatrics, University of California School of Medicine, San Francisco, CA; </p><div>Find articles by <a href="/pubmed/?term=Loh%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Mignon L. Loh</a></div></div><div id="_co_idm140563472506816"><h3 class="no_margin">Elizabeth A. Raetz</h3><p><sup>7</sup>Department of Pediatrics, University of Utah, Salt Lake City, UT; </p><div>Find articles by <a href="/pubmed/?term=Raetz%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Elizabeth A. Raetz</a></div></div><div id="_co_idm140563472504832"><h3 class="no_margin">Naomi J. Winick</h3><p><sup>8</sup>University of Texas Southwestern Medical Center, Dallas, TX; </p><div>Find articles by <a href="/pubmed/?term=Winick%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Naomi J. Winick</a></div></div><div id="_co_idm140563486416128"><h3 class="no_margin">Stephen P. Hunger</h3><p><sup>9</sup>Division of Pediatric Oncology and Center for Childhood Cancer Research, Children's Hospital of Philadelphia, Philadelphia, PA; </p><div>Find articles by <a href="/pubmed/?term=Hunger%20SP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Stephen P. Hunger</a></div></div><div id="_co_idm140563480906032"><h3 class="no_margin">William L. Carroll</h3><p><sup>10</sup>Department of Pediatrics, New York University Medical Center, New York, NY; and </p><div>Find articles by <a href="/pubmed/?term=Carroll%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">William L. Carroll</a></div></div><div id="_co_idm140563485343344"><h3 class="no_margin">Sima Jeha</h3><p><sup>5</sup>Department of Oncology, St. Jude Children's Research Hospital, Memphis, TN; </p><div>Find articles by <a href="/pubmed/?term=Jeha%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Sima Jeha</a></div></div><div id="_co_idm140563481466368"><h3 class="no_margin">Ching-Hon Pui</h3><p><sup>5</sup>Department of Oncology, St. Jude Children's Research Hospital, Memphis, TN; </p><div>Find articles by <a href="/pubmed/?term=Pui%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Ching-Hon Pui</a></div></div><div id="_co_idm140563481464384"><h3 class="no_margin">William E. Evans</h3><p><sup>1</sup>Department of Pharmaceutical Sciences, and </p><div>Find articles by <a href="/pubmed/?term=Evans%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">William E. Evans</a></div></div><div id="_co_idm140563480913856"><h3 class="no_margin">Meenakshi Devidas</h3><p><sup>11</sup>Department of Biostatistics, Colleges of Medicine, Public Health &amp; Health Professions, University of Florida, Gainesville, FL</p><div>Find articles by <a href="/pubmed/?term=Devidas%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Meenakshi Devidas</a></div></div><div id="_co_idm140563478204880"><h3 class="no_margin">Mary V. Relling</h3><p><sup>1</sup>Department of Pharmaceutical Sciences, and </p><div>Find articles by <a href="/pubmed/?term=Relling%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25987655">Mary V. Relling</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140563483684064_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140563483684064_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140563483684064_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140563483684064_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Pharmaceutical Sciences, and </div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Pathology, St. Jude Children’s Research Hospital, Memphis, TN; </div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Maine Children’s Cancer Program, Scarborough, ME; </div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Department of Hematology and Oncology, Cook Children’s Medical Center, Fort Worth, TX; </div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Department of Oncology, St. Jude Children's Research Hospital, Memphis, TN; </div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Department of Pediatrics, University of California School of Medicine, San Francisco, CA; </div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Department of Pediatrics, University of Utah, Salt Lake City, UT; </div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>University of Texas Southwestern Medical Center, Dallas, TX; </div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Division of Pediatric Oncology and Center for Childhood Cancer Research, Children's Hospital of Philadelphia, Philadelphia, PA; </div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>Department of Pediatrics, New York University Medical Center, New York, NY; and </div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>Department of Biostatistics, Colleges of Medicine, Public Health &amp; Health Professions, University of Florida, Gainesville, FL</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140563483684064_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2015 Feb 12; Accepted 2015 May 1.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140563483684064_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2015 by The American Society of Hematology</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4492197/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140563483780272" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140563483780272title">Key Points</h2><!--article-meta--><div><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="__p2">The rs6021191 variant in <em>NFATC2</em> is associated with an increased risk of asparaginase hypersensitivity and is an expression quantitative trait locus associated with expression of <em>NFATC2.</em></p></li><li><p id="__p3">Exome interrogation confirms the importance of the <em>HLA-DRB1*07:01</em> allele in asparaginase hypersensitivity.</p></li></ul><p></p></div></div><div id="idm140563481119776" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140563481119776title">Abstract</h2><!--article-meta--><div><p id="__p4" class="p p-first-last">Asparaginase is used to treat acute lymphoblastic leukemia (ALL); however, hypersensitivity reactions can lead to suboptimal asparaginase exposure. Our objective was to use a genome-wide approach to identify loci associated with asparaginase hypersensitivity in children with ALL enrolled on St. Jude Children’s Research Hospital (SJCRH) protocols Total XIIIA (n = 154), Total XV (n = 498), and Total XVI (n = 271), or Children’s Oncology Group protocols POG 9906 (n = 222) and AALL0232 (n = 2163). Germline DNA was genotyped using the Affymetrix 500K, Affymetrix 6.0, or the Illumina Exome BeadChip array. In multivariate logistic regression, the intronic rs6021191 variant in nuclear factor of activated T cells 2 (<em>NFATC2</em>) had the strongest association with hypersensitivity (<em>P</em> = 4.1 × 10<sup>−8</sup>; odds ratio [OR] = 3.11). RNA-seq data available from 65 SJCRH ALL tumor samples and 52 Yoruba HapMap samples showed that samples carrying the rs6021191 variant had higher <em>NFATC2</em> expression compared with noncarriers (<em>P</em> = 1.1 × 10<sup>−3</sup> and 0.03, respectively). The top ranked nonsynonymous polymorphism was rs17885382 in <em>HLA-DRB1</em> (<em>P</em> = 3.2 × 10<sup>−6</sup>; OR = 1.63), which is in near complete linkage disequilibrium with the <em>HLA-DRB1</em>*07:01 allele we previously observed in a candidate gene study. The strongest risk factors for asparaginase allergy are variants within genes regulating the immune response.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p5" class="p p-first">Current acute lymphoblastic leukemia (ALL) and lymphoma therapies depend heavily upon the use of glucocorticoids and asparaginase, among other drugs. Asparaginase is a heterologous enzyme that can influence the exposure of other drugs, including glucocorticoids.<sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931623">1</a></sup> Moreover, allergy to asparaginase can compromise the effectiveness of both asparaginase and dexamethasone by decreasing plasma exposure to both agents, and can result in a higher risk of relapse.<sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931607">2</a></sup> Hypersensitivity reactions to asparaginase during treatment are common, and have been associated with anti-asparaginase immunoglobulin G (IgG) antibodies rather than IgE antibodies.<sup><a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931593">3</a>,<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931602">4</a></sup> However, recent reports indicate that patients receiving PEGylated <em>Escherichia coli</em> asparaginase can develop hypersensitivity reactions to the drug without any evidence of detectable antibodies; because IgE may be bound to mast cells and may be present for only a limited time period, it is possible that IgE plays a role in asparaginase-induced reactions but eludes detection.<sup><a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931600">5</a></sup> Therefore, multiple pathways of asparaginase hypersensitivity are possible and currently they are not well understood.</p><p id="__p6">Few studies have investigated genetic risk factors for asparaginase hypersensitivity. In a study of 485 pediatric ALL patients on St. Jude Children’s Research Hospital (SJCRH) Total XV, the rs4958351 variant in the glutamate receptor gene <em>GRIA1</em> was associated with asparaginase hypersensitivity.<sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931619">6</a></sup> Our recent study investigating the role of <em>HLA-DRB1</em> genes on asparaginase hypersensitivity in 1870 patients of European ancestry identified an association with the <em>HLA-DRB1</em>*07:01 allele.<sup><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931595">7</a></sup> However, additional agnostic genome-wide studies that encompass the racial diversity of patients with ALL are required to better understand the risk and the mechanisms of asparaginase hypersensitivity.</p><p id="__p7" class="p p-last">The present study is the largest investigation of genetic predispositions to asparaginase hypersensitivity and includes 3308 patients of diverse ancestry. We used a genome-wide association approach, including a focus on exonic variants in addition to noncoding variants, to identify genetic loci associated with asparaginase hypersensitivity.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Materials and methods</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Patients</h3><p id="__p8" class="p p-first-last">Asparaginase hypersensitivity was assessed in children with ALL enrolled on SJCRH protocols Total XIIIA (n = 154), Total XV (n = 498, <a href="http://ClinicalTrials.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">ClinicalTrials.gov</a> #<a id="__tag_560383993" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT00137111" ref="reftype=extlink-clinical-trial&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT00137111","term_id":"NCT00137111"}}</span>NCT00137111</a>), and Total XVI (n = 271, <a href="http://ClinicalTrials.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">ClinicalTrials.gov</a> #<a id="__tag_560383992" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT00549848" ref="reftype=extlink-clinical-trial&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT00549848","term_id":"NCT00549848"}}</span>NCT00549848</a>) or Children’s Oncology Group (COG) protocols POG 9906 (n = 222, <a href="http://ClinicalTrials.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">ClinicalTrials.gov</a> #<a id="__tag_560383994" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT00005603" ref="reftype=extlink-clinical-trial&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT00005603","term_id":"NCT00005603"}}</span>NCT00005603</a>) and AALL0232 (n = 2163, <a href="http://ClinicalTrials.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">ClinicalTrials.gov</a> #<a id="__tag_560383991" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT00075725" ref="reftype=extlink-clinical-trial&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT00075725","term_id":"NCT00075725"}}</span>NCT00075725</a>). Informed consent from the parents or guardians, and consent from the patients as appropriate were obtained according to Institutional Review Board guidelines for genomic research and for treatment. The schedules and doses of asparaginase used during treatment varied by protocol and have been described elsewhere.<sup><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931579">7</a></sup> The association between HLA variants and asparaginase allergy among patients of European ancestry enrolled in these studies has been reported previously<sup><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931589">7</a></sup> (see supplemental Table 1, available on the <em>Blood</em> Web site), and we have previously reported an analysis of standard noncoding “GWAS” array single nucleotide polymorphisms (SNPs) for allergy for a subset of the Total XV patients<sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931578">6</a></sup> (supplemental Table 1). Patients enrolled on protocols Total XIIIA, Total XV, and POG 9906 received native <em>E coli</em> asparaginase (26%), whereas patients on Total XVI and AALL0232 received PEGylated <em>E coli</em> asparaginase (74%). Hypersensitivities were graded using the scales described in the National Cancer Institute’s common toxicity criteria version 1.0 for Total XIIIA,<sup><a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931613">8</a></sup> version 2.0 for POG 9906, version 3.0 for Total XV<sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931582">6</a></sup> and Total XVI, and version 4.0 for COG AALL0232. Hypersensitivity reactions of grade 2 and above were considered cases. The clinical symptoms of reactions grade 2 and above can include rash, flushing, urticaria, fever ≥38°C, symptomatic bronchospasm, and anaphylaxis.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Genotyping</h3><p id="__p9" class="p p-first-last">Germline DNA was genotyped using the Affymetrix Human Mapping 500K Array Set, the Affymetrix Genome-Wide Human SNP Array 6.0, or the Illumina Exome BeadChip array.<sup><a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931618">9</a></sup> Hardy-Weinberg equilibrium tests were performed using PLINK in patients of European ancestry. SNPs with genotyping call rates &lt;95% and SNPs that were not in Hardy-Weinberg equilibrium (<em>P</em> &lt; .001 for SNPs with a minor allele frequency [MAF] ≥1%) were excluded from the association analysis. The genetic ancestry of patients was estimated using STRUCTURE as previously described,<sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931583">10</a></sup> with the European-ancestry group defined as having &gt;90% Northern European ancestry (CEU), the Asian-ancestry group defined as having &gt;90% East Asian ancestry (CHB/JBT), the African-ancestry group defined as having &gt;70% West African ancestry (YRI), the Hispanics defined as having Native American ancestry &gt;10% and whose Native American ancestry was greater than their % of African ancestry, and finally, others whose ancestry was outside the above boundaries. Affymetrix genotyping was available for 94% of patients with asparaginase hypersensitivity data (supplemental Table 1). Illumina exome array genotyping was available for 95% of the patients in the combined patient cohort (supplemental Table 1). Overall, 90% of the ALL patients with asparaginase hypersensitivity data had genotyping available from both platforms (supplemental Table 1).</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Nuclear factor of activated T cells 2 (<em>NFATC2</em>) gene expression analysis</h3><p id="__p10" class="p p-first-last"><em>NFATC2</em> gene expression was assessed in ALL tumor samples from patients enrolled on SJCRH protocols Total XV (n = 57) and Total XVI (n = 8). RNA-seq was performed as previously described.<sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931586">11</a>,<a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931608">12</a></sup> Expression levels of <em>NFATC2</em> were estimated as fragment per kilobase of transcript per million mapped reads (FPKM), and gene FPKM values were computed by summing the transcript FPKM values for each gene as described previously.<sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931599">13</a>,<a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931604">14</a></sup>
<em>NFATC2</em> gene expression and genotyping was also available for 52 unrelated Yoruba (YRI) HapMap samples (<a id="__tag_460931625" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE9703" ref="/geo/query/acc.cgi?acc=GSE9703"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE9703","term_id":"9703"}}</span>GSE9703</a>).<sup><a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931609">15</a>,<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931590">16</a></sup>
<em>NFATC2</em> expression levels in HapMap YRI samples were estimated by adding the hybridization signals of probes within spliced messenger RNA regions of the gene.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Data analysis</h3><p id="__p11" class="p p-first-last">Univariate and multivariate logistic regression was used to identify clinical risk factors associated with asparaginase hypersensitivity. The association analysis between SNP genotypes and asparaginase hypersensitivity reactions was performed using logistic regression, adjusting for significant covariates and assuming an additive genetic model.<sup><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931585">17</a></sup> Genotype and phenotype association analysis was performed in PLINK.<sup><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931614">17</a></sup> The association analysis between <em>NFATC2</em> gene expression and rs6021191 genotyping were performed using a general linear model with R statistical software (version 2.13.2).</p></div><div id="__sec7" class="sec sec-last"><h3 id="__sec7title">Pathway analysis using ingenuity</h3><p id="__p12" class="p p-first-last">Ingenuity Pathway Analysis (IPA) was used to prioritize biological pathways related to asparaginase hypersensitivity reactions within the genome-wide analysis. Genes containing interrogated SNPs with the strongest associations (lowest 0.1% <em>P</em> values, with <em>P</em> ≤ 1.1 × 10<sup>−3</sup>) to asparaginase hypersensitivity were included in the pathway analysis (334 genes representing 499 SNPs; supplemental Table 2). Fisher’s exact test was used to determine if the genes most associated with asparaginase hypersensitivity were over-represented in each canonical pathway.</p></div></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">Results</h2><p id="__p13" class="p p-first">Hypersensitivity reaction data to native <em>E coli</em> asparaginase or PEGylated <em>E coli</em> asparaginase were available from a total of 3308 pediatric ALL patients enrolled on 5 different protocols (<a href="/pmc/articles/PMC4492197/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Their median age was 9.3 years (range, 0.17 to 30), there were slightly more males than females (56.2% vs 43.8%; <a href="/pmc/articles/PMC4492197/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), and 59.5% of the genotyped patients were of European genetic ancestry (<a href="/pmc/articles/PMC4492197/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p14">Demographics of ALL patients with asparaginase hypersensitivity data</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563485239344" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1">Characteristics</th><th colspan="6" rowspan="1" align="center">Treatment protocol</th></tr><tr><th rowspan="1" colspan="1" align="center">SJCRH total XIIIA (%)</th><th rowspan="1" colspan="1" align="center">SJCRH total XV (%)</th><th rowspan="1" colspan="1" align="center">SJCRH total XVI (%)</th><th rowspan="1" colspan="1" align="center">POG 9906 (%)</th><th rowspan="1" colspan="1" align="center">COG AALL0232 (%)</th><th rowspan="1" colspan="1" align="center">Combined</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1"><strong>Sample size</strong><br> (n)</td><td rowspan="1" colspan="1" align="center">154 (4.7%)<sup>*</sup></td><td rowspan="1" colspan="1" align="center">498 (15.1%)<sup>*</sup></td><td rowspan="1" colspan="1" align="center">271 (8.2%)<sup>*</sup></td><td rowspan="1" colspan="1" align="center">222 (6.7%)<sup>*</sup></td><td rowspan="1" colspan="1" align="center">2163 (65.4%)<sup>*</sup></td><td rowspan="1" colspan="1" align="center">3308<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1"><strong>Age</strong></td><td rowspan="2" colspan="1" align="center">43 (28.3%),<sup>†</sup><br>7.1</td><td rowspan="2" colspan="1" align="center">126 (25.3%),<sup>†</sup><br>7.0</td><td rowspan="2" colspan="1" align="center">73 (26.9%),<sup>†</sup><br>7.2</td><td rowspan="2" colspan="1" align="center">146 (65.8%),<sup>†</sup><br>11.2</td><td rowspan="2" colspan="1" align="center">1372 (63.4%),<sup>†</sup><br>10.1</td><td rowspan="2" colspan="1" align="center">1760 (53.2%),<sup>†</sup><br>9.3</td></tr><tr><td rowspan="1" colspan="1"> (n ≥ 10, mean age)</td></tr><tr><td rowspan="1" colspan="1"><strong>Gender</strong></td><td rowspan="2" colspan="1" align="center">90 (58.4%)<sup>†</sup></td><td rowspan="2" colspan="1" align="center">279 (56.0%)<sup>†</sup></td><td rowspan="2" colspan="1" align="center">156 (57.6%)<sup>†</sup></td><td rowspan="2" colspan="1" align="center">150 (67.6%)<sup>†</sup></td><td rowspan="2" colspan="1" align="center">1183 (54.7%)<sup>†</sup></td><td rowspan="2" colspan="1" align="center">1858 (56.2%)<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1"> (n, Male)</td></tr><tr><td rowspan="1" colspan="1"><strong>Ancestry</strong><sup>‡</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> European</td><td rowspan="1" colspan="1" align="center">99 (75.6%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">322 (66.1%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">187 (69.5%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">122 (55.0%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">1,215 (56.2%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">1959 (59.5%)<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1"> Hispanic</td><td rowspan="1" colspan="1" align="center">11 (8.4%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">66 (13.6%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">21 (7.8%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">64 (28.8%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">560 (25.9%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">725 (22.0%)<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1"> African</td><td rowspan="1" colspan="1" align="center">19 (17.6%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">72 (14.8%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">11 (4.1%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">14 (6.3%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">134 (6.2%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">254 (7.7%)<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1"> Asian</td><td rowspan="1" colspan="1" align="center">0 (0.0%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">6 (1.2%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">1 (0.4%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">4 (1.8%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">42 (1.9%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">53 (1.6%)<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1"> Other</td><td rowspan="1" colspan="1" align="center">2 (1.5%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">21 (4.3%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">49 (18.2%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">18 (8.1%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">212 (9.8%)<sup>†</sup></td><td rowspan="1" colspan="1" align="center">302 (9.2%)<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1"><strong>ALL lineage</strong><sup>§</sup></td><td rowspan="2" colspan="1" align="center">130 (85.5%)<sup>†</sup></td><td rowspan="2" colspan="1" align="center">423 (84.9%)<sup>†</sup></td><td rowspan="2" colspan="1" align="center">225 (83.0%)<sup>†</sup></td><td rowspan="2" colspan="1" align="center">222 (100.0%)<sup>†</sup></td><td rowspan="2" colspan="1" align="center">2153 (100.0%)<sup>†</sup></td><td rowspan="2" colspan="1" align="center">3153 (95.6%)<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1"> (n, B-ALL)</td></tr></tbody></table></div><div id="largeobj_idm140563485239344" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4492197/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TF1-1"><sup>*</sup>The percentage of patients enrolled within each protocol relative to the total number of patients enrolled across all protocols.</div><div id="TF1-2"><sup>†</sup>The percentage of patients with the characteristic described in the first column.</div><div id="TF1-3"><sup>‡</sup>The genetic ancestry of patients with available genotyping was estimated using STRUCTURE, with numbers of patients indicated for those in the European-ancestry group defined as &gt;90% Northern European ancestry (CEU), Asian-ancestry group as &gt;90% East Asian ancestry (CHB/JBT), African-ancestry group as &gt;70% West African ancestry (YRI), Hispanics as &gt;10% Native American ancestry and greater than % of African ancestry, and others as those outside the above boundaries.</div><div id="TF1-4"><sup>§</sup>ALL lineage refers to the ALL immunophenotype; patients without B-lineage ALL had T-cell ALL and were only enrolled on SJCRH treatment protocols.</div></div></div><p id="__p19">A total of 589 patients had asparaginase hypersensitivity reactions (17.8%; <a href="/pmc/articles/PMC4492197/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). By univariate analysis both treatment protocol (<em>P</em> = 2.1 × 10<sup>−51</sup>; <a href="/pmc/articles/PMC4492197/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) and asparaginase formulation (<em>P</em> = 2.1 × 10<sup>−50</sup>; <a href="/pmc/articles/PMC4492197/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) were associated with hypersensitivity. Protocols including native <em>E coli</em> asparaginase had a higher frequency of hypersensitivity than those using PEGylated <em>E coli</em> asparaginase (35.1% vs 11.6%; <a href="/pmc/articles/PMC4492197/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Age &lt;10 years, ancestry, and B-cell ALL were also associated with the risk of hypersensitivity (<a href="/pmc/articles/PMC4492197/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). In multivariate analysis including treatment protocol, gender, age group, ancestry, and ALL lineage (<a href="/pmc/articles/PMC4492197/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), only protocol (<em>P</em> = 2.3 × 10<sup>−46</sup>), gender (<em>P</em> = .04), and ALL lineage (<em>P</em> = .003) were associated with asparaginase hypersensitivity. Patients with T-cell ALL were only enrolled on SJCRH protocols; within the standard/high-risk arms of these studies, there were more asparaginase hypersensitivity reactions among patients with B-lineage compared with those with T-cell ALL receiving identical treatment (supplemental Figure 1; <em>P</em> = .001). Asparaginase preparation was confounded with protocol, but it also associated with hypersensitivity in multivariate analysis (<em>P</em> = 7.2 × 10<sup>−45</sup>) when removing protocol as a covariate from the regression model.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p20">Analysis of covariates associated with asparaginase hypersensitivity</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563484554800" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1">Covariate</th><th rowspan="1" colspan="1" align="center">% Asparaginase allergy<sup>*</sup></th><th rowspan="1" colspan="1" align="center">Univariate <em>P</em> value</th><th rowspan="1" colspan="1" align="center">Multivariate <em>P</em> value<sup>†</sup></th></tr></thead><tbody valign="top"><tr><td rowspan="6" colspan="1"><strong>Protocol</strong><br> (% of cases)</td><td rowspan="1" colspan="1">Total XV: 204 (41.0%)</td><td rowspan="6" colspan="1" align="center">2.1 × 10<sup>−51</sup></td><td rowspan="6" colspan="1" align="center">2.3 × 10<sup>−46</sup></td></tr><tr><td rowspan="1" colspan="1">Total XIIIA: 50 (32.5%)</td></tr><tr><td rowspan="1" colspan="1">POG 9906: 53 (23.9%)</td></tr><tr><td rowspan="1" colspan="1">Total XVI: 38 (14.0%)</td></tr><tr><td rowspan="1" colspan="1">COG AALL0232: 244 (11.3%)</td></tr><tr><td rowspan="1" colspan="1">Combined: 589 (17.8%)</td></tr><tr><td rowspan="2" colspan="1"><strong>Asparaginase preparation</strong><sup>‡</sup><br> (% of cases)</td><td rowspan="1" colspan="1"><em>E coli</em> asparaginase: 307 (35.1%)</td><td rowspan="2" colspan="1" align="center">2.1 × 10<sup>−50</sup></td><td rowspan="2" colspan="1" align="center">7.2 × 10<sup>−45</sup></td></tr><tr><td rowspan="1" colspan="1">PEGylated <em>E coli</em> asparaginase: 282 (11.6%)</td></tr><tr><td rowspan="2" colspan="1"><strong>Gender</strong><br> (% of cases)</td><td rowspan="1" colspan="1">Male: 351 (18.9%)</td><td rowspan="2" colspan="1" align="center">.06</td><td rowspan="2" colspan="1" align="center">.04</td></tr><tr><td rowspan="1" colspan="1">Female: 238 (16.4%)</td></tr><tr><td rowspan="1" colspan="1"><strong>Age (&lt;10, ≥10)</strong></td><td rowspan="1" colspan="1">&lt;10: 314 (20.3%)</td><td rowspan="2" colspan="1" align="center">3.3 × 10<sup>−4</sup></td><td rowspan="2" colspan="1" align="center">.11</td></tr><tr><td rowspan="1" colspan="1"> (% of cases)</td><td rowspan="1" colspan="1">≥10: 273 (15.5%)</td></tr><tr><td rowspan="5" colspan="1"><strong>Ancestry</strong><sup>§</sup> (% of cases)</td><td rowspan="1" colspan="1">African: 51 (20.1%)</td><td rowspan="5" colspan="1" align="center">2.1 × 10<sup>−4</sup></td><td rowspan="5" colspan="1" align="center">.07</td></tr><tr><td rowspan="1" colspan="1">European: 385 (19.7%)</td></tr><tr><td rowspan="1" colspan="1">Hispanic: 102 (14.1%)</td></tr><tr><td rowspan="1" colspan="1">Other: 40 (13.2%)</td></tr><tr><td rowspan="1" colspan="1">Asian: 4 (7.5%)</td></tr><tr><td rowspan="2" colspan="1"><strong>ALL lineage</strong><sup>||</sup> (% of cases)</td><td rowspan="1" colspan="1">B-lineage<sup>¶</sup>: 263 (33.8%)</td><td rowspan="2" colspan="1" align="center">5.3 × 10<sup>−4</sup></td><td rowspan="2" colspan="1" align="center">.003</td></tr><tr><td rowspan="1" colspan="1">T-lineage<sup>¶</sup>: 27 (18.8%)</td></tr></tbody></table></div><div id="largeobj_idm140563484554800" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4492197/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TF2-1"><sup>*</sup>Percent asparaginase allergy refers to the percentage of cases within each category that had asparaginase hypersensitivity reactions.</div><div id="TF2-2"><sup>†</sup>Multivariate <em>P</em> values refer to the association of each covariate with treatment arm, gender, age, ancestry, and ALL lineage included in the model as categorical covariates.</div><div id="TF2-3"><sup>‡</sup>Asparaginase preparation was confounded with protocol, therefore the multivariate <em>P</em> value for asparaginase was estimated by removing protocol from the model.</div><div id="TF2-4"><sup>§</sup>Ancestry was estimated using genotyped SNPs.</div><div id="TF2-5"><sup>||</sup>ALL lineage refers to the ALL immunophenotype.</div><div id="TF2-6"><sup>¶</sup>Univariate <em>P</em> value for ALL lineage was calculated only for the SJCRH cohort because T-ALL patients were only enrolled within those protocols; all COG protocols were B-lineage ALL only; see also supplemental Figure 1.</div></div></div><p id="__p27">To identify genomic variants associated with asparaginase hypersensitivity, a multivariate logistic regression model adjusted for treatment arm, ancestry (as a categorical covariate), gender, age, and ALL lineage was used. The minor allele of an intronic polymorphism in <em>NFATC2</em> (rs6021191) was associated with a higher risk of hypersensitivity at the genome-wide significance threshold (<em>P</em> = 4.1 × 10<sup>−8</sup>, odds ratio [OR] = 3.11; <a href="/pmc/articles/PMC4492197/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A-B</span></a>). Focusing on nonsynonymous SNPs, the minor allele at rs17885382 in the HLA-DRB1 gene (<em>P</em> = 3.23 × 10<sup>−26</sup>, OR = 1.63; <a href="/pmc/articles/PMC4492197/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A,C</span></a>) had the strongest association with asparaginase hypersensitivity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4492197/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563483764944" class="figure"><a href="/pmc/articles/PMC4492197/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4492197_69f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 69f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4492197/bin/69f1.jpg"></a></div><div id="largeobj_idm140563483764944" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4492197/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4492197/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p28"><strong>Manhattan plot of <em>P</em> values for genome-wide SNP association with asparaginase hypersensitivity.</strong> (A) The Manhattan plot shows the negative log10 of the <em>P</em> values for association with asparaginase hypersensitivity vs each chromosome on the x-axis. The open circles indicate SNPs genotyped using the Affymetrix 6.0 array, whereas the open squares indicate SNPs genotyped by the Illumina Exome Beadchip array. The dashed line denotes the genome-wide threshold (<em>P</em> = 5 × 10<sup>−8</sup>). (B) SNPs genotyped using the Affymetrix SNP 6.0 array identified the <em>NFATC2</em> rs6021191 variant associated with asparaginase hypersensitivity (<em>P</em> = 4.1 × 10<sup>−8</sup>, OR = 3.11, CI = 2.07-4.66). AA, subjects homozygous for the reference allele (A); TT, subjects homozygous for the variant allele (T); AT, heterozygotes. (C) Illumina Exome array genotyping identified the strongest association with the <em>HLA-DRB1</em> rs17885382 variant (<em>P</em> = 3.2 × 10<sup>−6</sup>, OR = 1.63, CI = 1.33-2.00). CC, subjects homozygous for the reference allele (C); TT, subjects homozygous for the variant allele (T); CT, heterozygotes. Associations were determined using a general linear model adjusted for treatment, ALL immunophenotype, gender, age group, and ancestry.</p></div></div></div><p id="__p29">The MAF for the <em>NFATC2</em> rs6021191 variant was highest in patients of non-European ancestry (0.062; supplemental Table 3), and only 0.001 among patients of European ancestry, MAFs that are consistent with those reported in the general population (supplemental Table 3). The rs6021191 variant was associated with hypersensitivity across all non-European ancestries (supplemental Table 3), with the largest effect size seen in those with primarily Asian ancestry (OR = 8.71; supplemental Figure 2A), followed by those with African ancestry (OR = 4.45; supplemental Figure 2A). The association was also present across all protocols (supplemental Figure 2B) and with both asparaginase preparations (supplemental Figure 2C).</p><p id="__p30">The <em>NFATC2</em> rs6021191 variant is located in an intron of the gene and it is in a region with weak enhancer activity in HUVEC and K562 cell lines,<sup><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931620">18</a></sup> suggesting that the SNP may influence the expression of <em>NFATC2</em>. We found higher <em>NFATC2</em> gene expression in both ALL tumor samples and in YRI HapMap lymphoblastoid cell line samples carrying the rs6021191 variant allele (<em>P</em> = 1.1 × 10<sup>−3</sup> and 0.03, respectively; <a href="/pmc/articles/PMC4492197/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A-B</span></a>), indicating that the SNP is an expression quantitative trait locus (eQTL).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4492197/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563482645520" class="figure"><a href="/pmc/articles/PMC4492197/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4492197_69f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 69f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4492197/bin/69f2.jpg"></a></div><div id="largeobj_idm140563482645520" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4492197/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4492197/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p31"><strong>The <em>NFATC2</em> rs6021191 variant is an eQTL.</strong> RNA-seq data were available from the ALL tumor samples of 65 SJCRH patients and from 52 unrelated YRI HapMap samples to investigate the association between rs6021191 and <em>NFATC2</em> gene expression. We found higher <em>NFATC2</em> gene expression in samples with the rs6021191 variant compared with samples without the variant in both (A) ALL tumor samples (<em>P</em> = 1.1 × 10<sup>−3</sup>) and (B) YRI HapMap samples (<em>P</em> = .03). Allele abbreviations are explained in Figure 1.</p></div></div></div><p id="__p32">The MAF for the <em>HLA-DRB1</em> rs17885382 variant was highest in patients of European ancestry (MAF = 0.125; supplemental Table 4) and present in all non-European ancestries at a MAF ≥0.05 (supplemental Table 4). The effect size was most pronounced in patients of European ancestry (<em>P</em> = 5.9 × 10<sup>−5</sup>, OR = 1.68; supplemental Figure 3A), but elevated in other ancestries, for all treatment protocols, and for both asparaginase preparations (supplemental Figure 3A-C).</p><p id="__p33">The rs17885382 variant is located in exon 2 of the <em>HLA-DRB1</em> gene (Arg &gt; Gln, 54). Our previous candidate gene analysis of HLA alleles associated with asparaginase allergies in patients of European ancestry identified an association with 10 amino acids in HLA-DRB1, and position 54 (amino acid position 25 when excluding the leader signal sequence) was among those implicated.<sup><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931588">7</a></sup> From a total of 1540 <em>HLA-DRB1</em> alleles with amino acid sequences included in the International ImMunoGene Tics/HLA database,<sup><a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931581">19</a></sup> only <em>HLA-DRB1</em>*01:13, <em>HLA-DRB1</em>*03:54, <em>HLA-DRB1</em>*13:92, <em>HLA-DRB1</em>*14:26, <em>HLA-DRB1</em>*15:24, and <em>HLA-DRB1</em>*07 encode for a Gln in position 54. Our prior HLA-focused analysis discovered that <em>HLA-DRB1</em>*07:01 was associated with hypersensitivity,<sup><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931624">7</a></sup> and it is the only allele among those encoding for Gln at position 54 with a MAF &gt;1%. Consistent with our previous analysis, the <em>HLA-DRB1</em> rs17885382 variant was in linkage disequilibrium with the <em>HLA-DRB1</em>*07:01 allele (R<sup>2</sup> = 0.94); therefore, it is likely that the haplotype is driving the association between rs17885382 and asparaginase hypersensitivity.</p><p id="__p34">In a multivariate analysis, including both rs6021191 (<em>NFATC2</em>) and rs17885382 (<em>HLA-DRB1</em>) genotype, both variants remained associated with asparaginase hypersensitivity (P<sub>rs6021191</sub> = 1.8 × 10<sup>−8</sup>, P<sub>rs17885382</sub> = 3.2 × 10<sup>−6</sup>), and modest increases in effect size were seen after adjusting for the genotypes (OR<sub>rs6021191</sub> = 3.35, OR<sub>rs17885382</sub> = 1.66). The frequency of asparaginase hypersensitivity in patients with neither of these variants was 14.8% (321 of 2173). In comparison, 168 of the 738 (22.7%) patients with one variant (at rs6021191 or rs17885382) developed hypersensitivity (<a href="/pmc/articles/PMC4492197/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>) (OR<sub>rs6021191</sub> = 3.07, P<sub>rs6021191</sub> = 5.4 × 10<sup>−6</sup>; OR<sub>rs17885382</sub> = 1.66, P<sub>rs17885382</sub> = 3.5 × 10<sup>−5</sup>). The risk of hypersensitivity was higher still in patients carrying both alleles (9 out of 19 or 47.4%) as compared with patients with no variant alleles (<a href="/pmc/articles/PMC4492197/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>) (OR = 8.58, <em>P</em> = 1.7 × 10<sup>−5</sup>) and to patients carrying a single variant allele (OR<sub>rs6021191</sub> = 2.79, P<sub>rs6021191</sub> = 0.05; OR<sub>rs17885382</sub> = 5.18, P<sub>rs17885382</sub> = 1.2 × 10<sup>−3</sup>). Similar results were found taking the preparation of asparaginase into consideration. Patients receiving native <em>E coli</em> asparaginase carrying both risk alleles had an incidence of allergy of 50% compared with 32% among noncarriers of risk alleles, whereas patients receiving PEGylated <em>E coli</em> asparaginase had an incidence of allergy of 47% compared with 10% among noncarriers. Furthermore, after conditioning on the association for the <em>NFATC2</em> rs6021191 and <em>HLA-DRB1</em> rs17885382 variants, no other SNP reached the genome-wide significance threshold.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4492197/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563476722400" class="figure"><a href="/pmc/articles/PMC4492197/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4492197_69f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 69f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4492197/bin/69f3.jpg"></a></div><div id="largeobj_idm140563476722400" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4492197/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4492197/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p35"><strong>Patients carrying both the <em>HLA-DRB1</em> rs17885382 and <em>NFATC2</em> rs6021191 variant have a higher risk of developing asparaginase hypersensitivity compared with patients with no risk variant.</strong> The risk of developing hypersensitivity was determined for patients carrying a single-risk variant (<em>HLA-DRB1</em> rs17885382 or <em>NFATC2</em> rs6021191) or for patients carrying both risk variants (<em>NFATC2</em> rs6021191 and <em>HLA-DRB1</em> rs17885382). The risk of hypersensitivity was higher in patients carrying a single variant (OR<sub>rs6021191</sub> = 3.07, CI<sub>rs6021191</sub> = 1.87-4.94, P<sub>rs6021191</sub> = 5.4 × 10<sup>−6</sup>; OR<sub>rs17885382</sub> = 1.66, CI<sub>rs17885382</sub> = 1.3-2.1, P<sub>rs17885382</sub> = 3.5 × 10<sup>−5</sup>) or both variants (OR = 8.58, CI = 3.14-22.92, <em>P</em> = 1.7 × 10<sup>−5</sup>) compared with patients with no risk alleles. The associations were determined using a general linear model adjusted for treatment, ALL immunophenotype, gender, age group, and ancestry.</p></div></div></div><p id="__p36" class="p">Among the top 20 pathways with <em>P</em> values &lt;3 × 10<sup>−3</sup> invoked by the 334 genes having top ranked SNPs associated with asparaginase hypersensitivity, 13 pathways involved T-cell apoptosis, T-cell signaling, T-cell activation, or T-cell disorders (<a href="/pmc/articles/PMC4492197/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4A</span></a>). Genes within the top 20 pathways included <em>PPP3CA</em>, <em>PRKCQ</em>, and <em>PRKCE</em>, which are involved in T-cell function (<a href="/pmc/articles/PMC4492197/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4B</span></a>). Therefore, the pathway analysis suggests that genetic variants within genes involved in T-cell function may play a role in the development of asparaginase hypersensitivity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC4492197/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140563483481936" class="figure"><a href="/pmc/articles/PMC4492197/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4492197_69f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 69f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4492197/bin/69f4.jpg"></a></div><div id="largeobj_idm140563483481936" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4492197/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC4492197/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p37"><strong>Genes involved in T-cell function may contribute to the risk of developing asparaginase hypersensitivity reactions.</strong> IPA was used to determine if genes most associated with asparaginase hypersensitivity were overrepresented in specific biological pathways. A total of 334 genes invoked by 499 SNPs associated with asparaginase hypersensitivity annotated to 234 distinct IPA pathways. (A) The top 20 pathways identified, all with <em>P</em> &lt; 3 × 10<sup>−3</sup>, were enriched for genes involved in T-cell apoptosis, T-cell signaling, T-cell activation, or T-cell disorders. (B) A total of 33 genes containing SNPs associated with hypersensitivity were included within the top 20 canonical pathways (shown on x-axis). Many of these genes were included in multiple pathways. The number of pathways (out of 20) in which each of these 33 genes is involved is plotted on the y-axis. For example, <em>HLA-DRB1</em> is involved in 14 out of the 20 top canonical pathways. GPCR, G-protein coupled receptor; nNOS, neuronal NOS; IL, interleukin; iCOS, inducible T-cell costimulator and its ligand, iCOSL.</p></div></div></div></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title">Discussion</h2><p id="__p38" class="p p-first">Asparaginase is a mainstay in the treatment of ALL, but immune responses to asparaginase can decrease the systemic exposure to asparaginase and result in a higher risk of relapse.<sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931597">2</a></sup> In this study, we used a genome-wide association approach to identify genetic loci associated with asparaginase hypersensitivity within a racially diverse set of pediatric patients with ALL. We found that the rs6021191 variant is associated with a higher risk of asparaginase hypersensitivity at the genome-wide significance level (<a href="/pmc/articles/PMC4492197/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A-B</span></a>), and the variant is associated with higher NFATC2 messenger RNA expression (<a href="/pmc/articles/PMC4492197/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A-B</span></a>). The <em>NFATC2</em> variant was rare in patients of European ancestry (supplemental Table 3); nevertheless, we were able to identify the association between the <em>NFATC2</em> variant and asparaginase hypersensitivity because a multiethnic cohort was available for our investigation. Other variants in <em>NFATC2</em> have also been related to immune-mediated phenotypes, such as narcolepsy,<sup><a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931616">20</a></sup> self-reported allergies (cat, pollen, and dust-mite allergy),<sup><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931622">21</a></sup> and thiazolidinedione-induced edema.<sup><a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931592">22</a></sup> Interestingly, both self-reported allergies and narcolepsy also have associations with HLA genetic variants.<sup><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931580">21</a>,<a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931611">23</a></sup></p><p id="__p39">The <em>NFATC2</em> gene encodes a cytoplasmic component of the NFAT transcription factor family. Upon T-cell receptor stimulation, cytoplasmic NFATC2 is dephosphorylated and translocated to the nucleus where it participates in gene regulation.<sup><a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931617">24</a></sup> The role of <em>NFATC2</em> on the risk of drug-induced allergy is unknown, yet studies have shown that <em>NFATC2</em> can influence the development and function of regulatory T cells and can have either a negative<sup><a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931584">25</a></sup> or positive<sup><a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931601">26</a></sup> regulatory influence on the immune response, depending on the antigen. Nevertheless, several studies using NFAT inhibitors for different immune-related disease models support that inhibition of the NFAT pathway can attenuate an immune response.<sup><a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931612">27</a><a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931621"></a><a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931596"></a><a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931603"></a><a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931598"></a>-<a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931615">32</a></sup> Moreover, among individuals with Down syndrome, the inhibition of the NFAT pathway (partly via overexpression of <em>DYRK1A</em> and <em>DSCR1</em> due to trisomy 21)<sup><a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931591">33</a>,<a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931606">34</a></sup> is hypothesized to be related to the cardiac, neurologic, and immunologic defects associated with Down syndrome,<sup><a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931594">34</a></sup> and individuals with Down syndrome may have attenuated antibody responses to several vaccine antigens.<sup><a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931587">35</a></sup> Based on this role of the NFAT pathway in Down syndrome, and on our finding of a higher incidence of asparaginase hypersensitivity among those carrying the <em>NFATC2</em> variant, we hypothesized that Down syndrome patients might have a lower incidence of asparaginase hypersensitivity. Indeed, we observed a lower incidence of asparaginase hypersensitivity among ALL patients with Down syndrome than in those without Down syndrome (6% vs 17%, <em>P</em> multivariate = 8.4 × 10<sup>−3</sup>; supplemental Figure 4), and this effect was found among protocols using either preparation of asparaginase, with 8% (4 of 48) of patients with Down syndrome having allergic reactions to PEGylated <em>E coli</em> asparaginase compared to 12% of patients without Down syndrome, and 0% (0 of 22) of patients with Down syndrome having allergic reactions to native <em>E coli</em> asparaginase compared with 37% among patients without Down syndrome. These results further support the notion that variability in NFATC2 plays a role in the risk of allergy to asparaginase, and that inhibition of NFATC2 might mitigate sensitization to asparaginase.</p><p id="__p40">Our study included not only standard “GWAS” arrays (mostly noncoding SNPs), but also the exome array for interrogation of coding variants. The top hit for a coding variant was in the <em>HLA-DRB1</em> gene and the SNP was in linkage disequilibrium with the <em>HLA-DRB1*07:01</em> allele.<sup><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931605">7</a></sup> Therefore, the top coding variant that we identified using the exome array was the same variant identified by our candidate gene study. However, the patients included in our candidate gene study were solely of European ancestry (supplemental Table 1), and the current study extends our previous report identifying <em>HLA-DBRB1</em>*07:01 as a risk allele for asparaginase hypersensitivity by showing that <em>HLA-DRB1</em>*07:01 is also a risk allele in non-European patients (supplemental Figure 3). Furthermore, we show that the association between the <em>HLA-DRB1</em> rs17885382 variant is independent from the <em>NFATC2</em> rs6021191 variant and that patients carrying both alleles have a higher risk of developing asparaginase hypersensitivity compared with patients with no risk alleles or with a single risk allele (<a href="/pmc/articles/PMC4492197/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). Interestingly, the <em>GRIA1</em> rs4958351 variant that we previously found to associate with asparaginase hypersensitivity<sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_460931610">6</a></sup> was also associated with the risk of asparaginase hypersensitivity in our combined patient cohort (<em>P</em> = .03; OR = 1.2). However, the risk of allergy was stronger in patients receiving native <em>E coli</em> asparaginase (OR = 1.55) than in patients receiving PEGylated <em>E coli</em> asparaginase (OR = 1.02).</p><p id="__p41" class="p p-last">In conclusion, we have identified that the <em>NFATC2</em> rs6021191 variant is an eQTL that is associated with the risk of developing asparaginase hypersensitivity. This is the first study to identify an association between <em>NFATC2</em> and a drug-induced hypersensitivity. Moreover, the results from our single SNP and pathway analyses suggest that T cells play an important role in the development of asparaginase hypersensitivity, likely due to their role in B-cell activation and differentiation.</p></div><div id="idm140563484473072" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140563484473072title">Acknowledgments</h2><div class="sec"><p id="__p44">This study was supported by grants GM 92666, CA 21765, CA 142665, CA 36401, RCA 156449, and U10 CA98543, COG Statistical Center (U10 CA98413), and COG Specimen Banking (U24 <a id="__tag_782375295" class="tag_hotlink" href="/nuccore/CA114766" ref="/nuccore/CA114766"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA114766","term_id":"34968073","term_text":"CA114766"}}</span>CA114766</a>) from the National Insitutes of Health and by the American Lebanese Syrian Associated Charities.</p></div></div><div id="idm140563476595744" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140563476595744title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140563476595616"></p><p id="__p42" class="p p-first-last">The online version of this article contains a data supplement.</p><p></p><p class="fn sec" id="idm140563484474176"></p><p id="__p43" class="p p-first-last">The publication costs of this article were defrayed in part by page charge payment. Therefore, and solely to indicate this fact, this article is hereby marked “advertisement” in accordance with 18 USC section 1734.</p><p></p></div></div><div id="__sec10" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">Authorship</h2><!--/article/back/sec/--><p id="__p45" class="p p-first">Contribution: C.A.F. and M.V.R. designed the project, analyzed and interpreted the data, and drafted the manuscript; C.A.F., C.S., and W.Y. performed statistical analysis; and C.S., W.Y., C.G.M., C.Q., E.L.,W.P.B., C.L., L.B.R., T.C., S.E.K., M.L.L., E.A.R., N.J.W. S.P.H., W.L.C., S.J., C.-H.P., W.E.E., and M.D. contributed to the writing of the manuscript.</p><p id="__p46">Conflict-of-interest disclosure: The authors declare no competing financial interests.</p><p id="__p47" class="p p-last">Correspondence: Mary V. Relling, St. Jude Children’s Research Hospital, 262 Danny Thomas Place, Mail Stop 313, Memphis, TN 38105; e-mail: <a href="mailto:dev@null" data-email="gro.edujts@gniller.yram" class="oemail">gro.edujts@gniller.yram</a>.</p></div><div id="idm140563463015680" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140563463015680title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="element-citation">Yang L, Boyd K, Kaste SC, Kamdem Kamdem L, Rahija RJ, Relling MV. A mouse model for glucocorticoid-induced osteonecrosis: effect of a steroid holiday. <span><span class="ref-journal">J Orthop Res. </span>2009;<span class="ref-vol">27</span>(2):169–175.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2718787/">PMC free article</a>]</span> [<a href="/pubmed/18683891" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Orthop+Res&amp;title=A+mouse+model+for+glucocorticoid-induced+osteonecrosis:+effect+of+a+steroid+holiday.&amp;author=L+Yang&amp;author=K+Boyd&amp;author=SC+Kaste&amp;author=L+Kamdem+Kamdem&amp;author=RJ+Rahija&amp;volume=27&amp;issue=2&amp;publication_year=2009&amp;pages=169-175&amp;pmid=18683891&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="element-citation">Kawedia JD, Liu C, Pei D, et al.  Dexamethasone exposure and asparaginase antibodies affect relapse risk in acute lymphoblastic leukemia. <span><span class="ref-journal">Blood. </span>2012;<span class="ref-vol">119</span>(7):1658–1664.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3286344/">PMC free article</a>]</span> [<a href="/pubmed/22117041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Dexamethasone+exposure+and+asparaginase+antibodies+affect+relapse+risk+in+acute+lymphoblastic+leukemia.&amp;author=JD+Kawedia&amp;author=C+Liu&amp;author=D+Pei&amp;volume=119&amp;issue=7&amp;publication_year=2012&amp;pages=1658-1664&amp;pmid=22117041&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="element-citation">Killander D, Dohlwitz A, Engstedt L, et al.  Hypersensitive reactions and antibody formation during L-asparaginase treatment of children and adults with acute leukemia. <span><span class="ref-journal">Cancer. </span>1976;<span class="ref-vol">37</span>(1):220–228.</span> [<a href="/pubmed/1061636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=Hypersensitive+reactions+and+antibody+formation+during+L-asparaginase+treatment+of+children+and+adults+with+acute+leukemia.&amp;author=D+Killander&amp;author=A+Dohlwitz&amp;author=L+Engstedt&amp;volume=37&amp;issue=1&amp;publication_year=1976&amp;pages=220-228&amp;pmid=1061636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="element-citation">Liu C, Kawedia JD, Cheng C, et al.  Clinical utility and implications of asparaginase antibodies in acute lymphoblastic leukemia. <span><span class="ref-journal">Leukemia. </span>2012;<span class="ref-vol">26</span>(11):2303–2309.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3516853/">PMC free article</a>]</span> [<a href="/pubmed/22484422" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Clinical+utility+and+implications+of+asparaginase+antibodies+in+acute+lymphoblastic+leukemia.&amp;author=C+Liu&amp;author=JD+Kawedia&amp;author=C+Cheng&amp;volume=26&amp;issue=11&amp;publication_year=2012&amp;pages=2303-2309&amp;pmid=22484422&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="element-citation">Fernandez CA, Stewart E, Panetta JC, et al.  Successful challenges using native <em>E. coli</em> asparaginase after hypersensitivity reactions to PEGylated E. coli asparaginase. <span><span class="ref-journal">Cancer Chemother Pharmacol. </span>2014;<span class="ref-vol">73</span>(6):1307–1313.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4137479/">PMC free article</a>]</span> [<a href="/pubmed/24771103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Chemother+Pharmacol&amp;title=Successful+challenges+using+native+E.+coli+asparaginase+after+hypersensitivity+reactions+to+PEGylated+E.+coli+asparaginase.&amp;author=CA+Fernandez&amp;author=E+Stewart&amp;author=JC+Panetta&amp;volume=73&amp;issue=6&amp;publication_year=2014&amp;pages=1307-1313&amp;pmid=24771103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="element-citation">Chen SH, Pei D, Yang W, et al.  Genetic variations in GRIA1 on chromosome 5q33 related to asparaginase hypersensitivity. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2010;<span class="ref-vol">88</span>(2):191–196.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3000799/">PMC free article</a>]</span> [<a href="/pubmed/20592726" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Genetic+variations+in+GRIA1+on+chromosome+5q33+related+to+asparaginase+hypersensitivity.&amp;author=SH+Chen&amp;author=D+Pei&amp;author=W+Yang&amp;volume=88&amp;issue=2&amp;publication_year=2010&amp;pages=191-196&amp;pmid=20592726&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="element-citation">Fernandez CA, Smith C, Yang W, et al.  HLA-DRB1*07:01 is associated with a higher risk of asparaginase allergies. <span><span class="ref-journal">Blood. </span>2014;<span class="ref-vol">124</span>(8):1266–1276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4141516/">PMC free article</a>]</span> [<a href="/pubmed/24970932" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=HLA-DRB1*07:01+is+associated+with+a+higher+risk+of+asparaginase+allergies.&amp;author=CA+Fernandez&amp;author=C+Smith&amp;author=W+Yang&amp;volume=124&amp;issue=8&amp;publication_year=2014&amp;pages=1266-1276&amp;pmid=24970932&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="element-citation">Woo MH, Hak LJ, Storm MC, et al.  Hypersensitivity or development of antibodies to asparaginase does not impact treatment outcome of childhood acute lymphoblastic leukemia. <span><span class="ref-journal">J Clin Oncol. </span>2000;<span class="ref-vol">18</span>(7):1525–1532.</span> [<a href="/pubmed/10735901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Oncol&amp;title=Hypersensitivity+or+development+of+antibodies+to+asparaginase+does+not+impact+treatment+outcome+of+childhood+acute+lymphoblastic+leukemia.&amp;author=MH+Woo&amp;author=LJ+Hak&amp;author=MC+Storm&amp;volume=18&amp;issue=7&amp;publication_year=2000&amp;pages=1525-1532&amp;pmid=10735901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="element-citation">Treviño LR, Yang W, French D, et al.  Germline genomic variants associated with childhood acute lymphoblastic leukemia. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>(9):1001–1005.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2762391/">PMC free article</a>]</span> [<a href="/pubmed/19684603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Germline+genomic+variants+associated+with+childhood+acute+lymphoblastic+leukemia.&amp;author=LR+Treviño&amp;author=W+Yang&amp;author=D+French&amp;volume=41&amp;issue=9&amp;publication_year=2009&amp;pages=1001-1005&amp;pmid=19684603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="element-citation">Yang JJ, Cheng C, Devidas M, et al.  Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>(3):237–241.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3104508/">PMC free article</a>]</span> [<a href="/pubmed/21297632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Ancestry+and+pharmacogenomics+of+relapse+in+acute+lymphoblastic+leukemia.&amp;author=JJ+Yang&amp;author=C+Cheng&amp;author=M+Devidas&amp;volume=43&amp;issue=3&amp;publication_year=2011&amp;pages=237-241&amp;pmid=21297632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="element-citation">Goya R, Sun MG, Morin RD, et al.  SNVMix: predicting single nucleotide variants from next-generation sequencing of tumors. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>(6):730–736.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2832826/">PMC free article</a>]</span> [<a href="/pubmed/20130035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNVMix:+predicting+single+nucleotide+variants+from+next-generation+sequencing+of+tumors.&amp;author=R+Goya&amp;author=MG+Sun&amp;author=RD+Morin&amp;volume=26&amp;issue=6&amp;publication_year=2010&amp;pages=730-736&amp;pmid=20130035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="element-citation">Roberts KG, Morin RD, Zhang J, et al.  Genetic alterations activating kinase and cytokine receptor signaling in high-risk acute lymphoblastic leukemia. <span><span class="ref-journal">Cancer Cell. </span>2012;<span class="ref-vol">22</span>(2):153–166.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3422513/">PMC free article</a>]</span> [<a href="/pubmed/22897847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=Genetic+alterations+activating+kinase+and+cytokine+receptor+signaling+in+high-risk+acute+lymphoblastic+leukemia.&amp;author=KG+Roberts&amp;author=RD+Morin&amp;author=J+Zhang&amp;volume=22&amp;issue=2&amp;publication_year=2012&amp;pages=153-166&amp;pmid=22897847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="element-citation">Shah S, Schrader KA, Waanders E, et al.  A recurrent germline PAX5 mutation confers susceptibility to pre-B cell acute lymphoblastic leukemia. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>(10):1226–1231.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3919799/">PMC free article</a>]</span> [<a href="/pubmed/24013638" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+recurrent+germline+PAX5+mutation+confers+susceptibility+to+pre-B+cell+acute+lymphoblastic+leukemia.&amp;author=S+Shah&amp;author=KA+Schrader&amp;author=E+Waanders&amp;volume=45&amp;issue=10&amp;publication_year=2013&amp;pages=1226-1231&amp;pmid=24013638&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="element-citation">Trapnell C, Williams BA, Pertea G, et al.  Transcript assembly and quantification by RNA-seq reveals unannotated transcripts and isoform switching during cell differentiation. <span><span class="ref-journal">Nat Biotechnol. </span>2010;<span class="ref-vol">28</span>(5):511–515.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3146043/">PMC free article</a>]</span> [<a href="/pubmed/20436464" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Biotechnol&amp;title=Transcript+assembly+and+quantification+by+RNA-seq+reveals+unannotated+transcripts+and+isoform+switching+during+cell+differentiation.&amp;author=C+Trapnell&amp;author=BA+Williams&amp;author=G+Pertea&amp;volume=28&amp;issue=5&amp;publication_year=2010&amp;pages=511-515&amp;pmid=20436464&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="element-citation">Zhang W, Dolan ME. Use of cell lines in the investigation of pharmacogenetic loci. <span><span class="ref-journal">Curr Pharm Des. </span>2009;<span class="ref-vol">15</span>(32):3782–3795.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782819/">PMC free article</a>]</span> [<a href="/pubmed/19925429" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Pharm+Des&amp;title=Use+of+cell+lines+in+the+investigation+of+pharmacogenetic+loci.&amp;author=W+Zhang&amp;author=ME+Dolan&amp;volume=15&amp;issue=32&amp;publication_year=2009&amp;pages=3782-3795&amp;pmid=19925429&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="element-citation">Zhang W, Duan S, Bleibel WK, et al.  Identification of common genetic variants that account for transcript isoform variation between human populations. <span><span class="ref-journal">Hum Genet. </span>2009;<span class="ref-vol">125</span>(1):81–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2665168/">PMC free article</a>]</span> [<a href="/pubmed/19052777" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Identification+of+common+genetic+variants+that+account+for+transcript+isoform+variation+between+human+populations.&amp;author=W+Zhang&amp;author=S+Duan&amp;author=WK+Bleibel&amp;volume=125&amp;issue=1&amp;publication_year=2009&amp;pages=81-93&amp;pmid=19052777&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>(3):559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses.&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="element-citation">Hoffman MM, Buske OJ, Wang J, Weng Z, Bilmes JA, Noble WS. Unsupervised pattern discovery in human chromatin structure through genomic segmentation. <span><span class="ref-journal">Nat Methods. </span>2012;<span class="ref-vol">9</span>(5):473–476.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3340533/">PMC free article</a>]</span> [<a href="/pubmed/22426492" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Unsupervised+pattern+discovery+in+human+chromatin+structure+through+genomic+segmentation.&amp;author=MM+Hoffman&amp;author=OJ+Buske&amp;author=J+Wang&amp;author=Z+Weng&amp;author=JA+Bilmes&amp;volume=9&amp;issue=5&amp;publication_year=2012&amp;pages=473-476&amp;pmid=22426492&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="element-citation">Robinson J, Mistry K, McWilliam H, Lopez R, Parham P, Marsh SG. The IMGT/HLA database. <span><span class="ref-journal">Nucleic Acids Res. </span>2011;<span class="ref-vol">39</span>(database issue):D1171–D1176.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3013815/">PMC free article</a>]</span> [<a href="/pubmed/21071412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+IMGT/HLA+database.&amp;author=J+Robinson&amp;author=K+Mistry&amp;author=H+McWilliam&amp;author=R+Lopez&amp;author=P+Parham&amp;volume=39&amp;issue=database+issue&amp;publication_year=2011&amp;pages=D1171-D1176&amp;pmid=21071412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="element-citation">Shimada M, Miyagawa T, Kawashima M, et al.  An approach based on a genome-wide association study reveals candidate loci for narcolepsy. <span><span class="ref-journal">Hum Genet. </span>2010;<span class="ref-vol">128</span>(4):433–441.</span> [<a href="/pubmed/20677014" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=An+approach+based+on+a+genome-wide+association+study+reveals+candidate+loci+for+narcolepsy.&amp;author=M+Shimada&amp;author=T+Miyagawa&amp;author=M+Kawashima&amp;volume=128&amp;issue=4&amp;publication_year=2010&amp;pages=433-441&amp;pmid=20677014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="element-citation">Hinds DA, McMahon G, Kiefer AK, et al.  A genome-wide association meta-analysis of self-reported allergy identifies shared and allergy-specific susceptibility loci. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>(8):907–911.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3753407/">PMC free article</a>]</span> [<a href="/pubmed/23817569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+meta-analysis+of+self-reported+allergy+identifies+shared+and+allergy-specific+susceptibility+loci.&amp;author=DA+Hinds&amp;author=G+McMahon&amp;author=AK+Kiefer&amp;volume=45&amp;issue=8&amp;publication_year=2013&amp;pages=907-911&amp;pmid=23817569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="element-citation">Bailey SD, Xie C, Do R, et al.  DREAM investigators. Variation at the NFATC2 locus increases the risk of thiazolidinedione-induced edema in the Diabetes REduction Assessment with ramipril and rosiglitazone Medication (DREAM) study. <span><span class="ref-journal">Diabetes Care. </span>2010;<span class="ref-vol">33</span>(10):2250–2253.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2945168/">PMC free article</a>]</span> [<a href="/pubmed/20628086" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes+Care&amp;title=Variation+at+the+NFATC2+locus+increases+the+risk+of+thiazolidinedione-induced+edema+in+the+Diabetes+REduction+Assessment+with+ramipril+and+rosiglitazone+Medication+(DREAM)+study.&amp;author=SD+Bailey&amp;author=C+Xie&amp;author=R+Do&amp;volume=33&amp;issue=10&amp;publication_year=2010&amp;pages=2250-2253&amp;pmid=20628086&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="element-citation">Miyagawa T, Toyoda H, Hirataka A, et al.  New susceptibility variants to narcolepsy identified in HLA class II region. <span><span class="ref-journal">Hum Mol Genet. </span>2015;<span class="ref-vol">24</span>(3):891–898.</span> [<a href="/pubmed/25256355" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=New+susceptibility+variants+to+narcolepsy+identified+in+HLA+class+II+region.&amp;author=T+Miyagawa&amp;author=H+Toyoda&amp;author=A+Hirataka&amp;volume=24&amp;issue=3&amp;publication_year=2015&amp;pages=891-898&amp;pmid=25256355&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="element-citation">Rao A, Luo C, Hogan PG. Transcription factors of the NFAT family: regulation and function. <span><span class="ref-journal">Annu Rev Immunol. </span>1997;<span class="ref-vol">15</span>:707–747.</span> [<a href="/pubmed/9143705" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Immunol&amp;title=Transcription+factors+of+the+NFAT+family:+regulation+and+function.&amp;author=A+Rao&amp;author=C+Luo&amp;author=PG+Hogan&amp;volume=15&amp;publication_year=1997&amp;pages=707-747&amp;pmid=9143705&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="element-citation">Xanthoudakis S, Viola JP, Shaw KT, et al.  An enhanced immune response in mice lacking the transcription factor NFAT1. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">272</span>(5263):892–895.</span> [<a href="/pubmed/8629027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=An+enhanced+immune+response+in+mice+lacking+the+transcription+factor+NFAT1.&amp;author=S+Xanthoudakis&amp;author=JP+Viola&amp;author=KT+Shaw&amp;volume=272&amp;issue=5263&amp;publication_year=1996&amp;pages=892-895&amp;pmid=8629027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="element-citation">Karwot R, Übel C, Bopp T, Schmitt E, Finotto S. Increased immunosuppressive function of CD4(+)CD25(+)Foxp3(+)GITR+ T regulatory cells from NFATc2((-/-)) mice controls allergen-induced experimental asthma. <span><span class="ref-journal">Immunobiology. </span>2012;<span class="ref-vol">217</span>(9):905–911.</span> [<a href="/pubmed/22317750" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunobiology&amp;title=Increased+immunosuppressive+function+of+CD4(+)CD25(+)Foxp3(+)GITR++T+regulatory+cells+from+NFATc2((-/-))+mice+controls+allergen-induced+experimental+asthma.&amp;author=R+Karwot&amp;author=C+Übel&amp;author=T+Bopp&amp;author=E+Schmitt&amp;author=S+Finotto&amp;volume=217&amp;issue=9&amp;publication_year=2012&amp;pages=905-911&amp;pmid=22317750&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="element-citation">Nagamoto-Combs K, Combs CK. Microglial phenotype is regulated by activity of the transcription factor, NFAT (nuclear factor of activated T cells). <span><span class="ref-journal">J Neurosci. </span>2010;<span class="ref-vol">30</span>(28):9641–9646.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2914496/">PMC free article</a>]</span> [<a href="/pubmed/20631193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Microglial+phenotype+is+regulated+by+activity+of+the+transcription+factor,+NFAT+(nuclear+factor+of+activated+T+cells).&amp;author=K+Nagamoto-Combs&amp;author=CK+Combs&amp;volume=30&amp;issue=28&amp;publication_year=2010&amp;pages=9641-9646&amp;pmid=20631193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="element-citation">Choi JM, Sohn JH, Park TY, Park JW, Lee SK. Cell permeable NFAT inhibitory peptide Sim-2-VIVIT inhibits T-cell activation and alleviates allergic airway inflammation and hyper-responsiveness. <span><span class="ref-journal">Immunol Lett. </span>2012;<span class="ref-vol">143</span>(2):170–176.</span> [<a href="/pubmed/22342853" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunol+Lett&amp;title=Cell+permeable+NFAT+inhibitory+peptide+Sim-2-VIVIT+inhibits+T-cell+activation+and+alleviates+allergic+airway+inflammation+and+hyper-responsiveness.&amp;author=JM+Choi&amp;author=JH+Sohn&amp;author=TY+Park&amp;author=JW+Park&amp;author=SK+Lee&amp;volume=143&amp;issue=2&amp;publication_year=2012&amp;pages=170-176&amp;pmid=22342853&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="element-citation">Elloumi HZ, Maharshak N, Rao KN, et al.  A cell permeable peptide inhibitor of NFAT inhibits macrophage cytokine expression and ameliorates experimental colitis. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>(3):e34172.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3313977/">PMC free article</a>]</span> [<a href="/pubmed/22479554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=A+cell+permeable+peptide+inhibitor+of+NFAT+inhibits+macrophage+cytokine+expression+and+ameliorates+experimental+colitis.&amp;author=HZ+Elloumi&amp;author=N+Maharshak&amp;author=KN+Rao&amp;volume=7&amp;issue=3&amp;publication_year=2012&amp;pages=e34172&amp;pmid=22479554&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="element-citation">Noguchi H, Matsushita M, Okitsu T, et al.  A new cell-permeable peptide allows successful allogeneic islet transplantation in mice. <span><span class="ref-journal">Nat Med. </span>2004;<span class="ref-vol">10</span>(3):305–309.</span> [<a href="/pubmed/14770176" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Med&amp;title=A+new+cell-permeable+peptide+allows+successful+allogeneic+islet+transplantation+in+mice.&amp;author=H+Noguchi&amp;author=M+Matsushita&amp;author=T+Okitsu&amp;volume=10&amp;issue=3&amp;publication_year=2004&amp;pages=305-309&amp;pmid=14770176&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="element-citation">Kuriyama M, Matsushita M, Tateishi A, et al.  A cell-permeable NFAT inhibitor peptide prevents pressure-overload cardiac hypertrophy. <span><span class="ref-journal">Chem Biol Drug Des. </span>2006;<span class="ref-vol">67</span>(3):238–243.</span> [<a href="/pubmed/16611217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Biol+Drug+Des&amp;title=A+cell-permeable+NFAT+inhibitor+peptide+prevents+pressure-overload+cardiac+hypertrophy.&amp;author=M+Kuriyama&amp;author=M+Matsushita&amp;author=A+Tateishi&amp;volume=67&amp;issue=3&amp;publication_year=2006&amp;pages=238-243&amp;pmid=16611217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="element-citation">Zhang S, Luo L, Wang Y, Gomez MF, Thorlacius H. Nuclear factor of activated T cells regulates neutrophil recruitment, systemic inflammation, and T-cell dysfunction in abdominal sepsis. <span><span class="ref-journal">Infect Immun. </span>2014;<span class="ref-vol">82</span>(8):3275–3288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4136225/">PMC free article</a>]</span> [<a href="/pubmed/24866796" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Immun&amp;title=Nuclear+factor+of+activated+T+cells+regulates+neutrophil+recruitment,+systemic+inflammation,+and+T-cell+dysfunction+in+abdominal+sepsis.&amp;author=S+Zhang&amp;author=L+Luo&amp;author=Y+Wang&amp;author=MF+Gomez&amp;author=H+Thorlacius&amp;volume=82&amp;issue=8&amp;publication_year=2014&amp;pages=3275-3288&amp;pmid=24866796&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="element-citation">Roberts I, Izraeli S. Haematopoietic development and leukaemia in Down syndrome. <span><span class="ref-journal">Br J Haematol. </span>2014;<span class="ref-vol">167</span>(5):587–599.</span> [<a href="/pubmed/25155832" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=Haematopoietic+development+and+leukaemia+in+Down+syndrome.&amp;author=I+Roberts&amp;author=S+Izraeli&amp;volume=167&amp;issue=5&amp;publication_year=2014&amp;pages=587-599&amp;pmid=25155832&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="element-citation">Arron JR, Winslow MM, Polleri A, et al.  NFAT dysregulation by increased dosage of DSCR1 and DYRK1A on chromosome 21. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">441</span>(7093):595–600.</span> [<a href="/pubmed/16554754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=NFAT+dysregulation+by+increased+dosage+of+DSCR1+and+DYRK1A+on+chromosome+21.&amp;author=JR+Arron&amp;author=MM+Winslow&amp;author=A+Polleri&amp;volume=441&amp;issue=7093&amp;publication_year=2006&amp;pages=595-600&amp;pmid=16554754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="element-citation">Kusters MA, Verstegen RH, Gemen EF, de Vries E. Intrinsic defect of the immune system in children with Down syndrome: a review. <span><span class="ref-journal">Clin Exp Immunol. </span>2009;<span class="ref-vol">156</span>(2):189–193.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2759463/">PMC free article</a>]</span> [<a href="/pubmed/19250275" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Exp+Immunol&amp;title=Intrinsic+defect+of+the+immune+system+in+children+with+Down+syndrome:+a+review.&amp;author=MA+Kusters&amp;author=RH+Verstegen&amp;author=EF+Gemen&amp;author=E+de+Vries&amp;volume=156&amp;issue=2&amp;publication_year=2009&amp;pages=189-193&amp;pmid=19250275&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4492197&amp;issue-id=254995&amp;journal-id=446&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140563483780272title">Key Points</a></li><li><a href="#idm140563481119776title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Materials and methods</a></li><li><a href="#__sec8title">Results</a></li><li><a href="#__sec9title">Discussion</a></li><li><a href="#idm140563484473072title">Acknowledgments</a></li><li><a href="#idm140563476595744title">Footnotes</a></li><li><a href="#__sec10title">Authorship</a></li><li><a href="#idm140563463015680title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Blood</span> are provided here courtesy of <strong>The American Society of Hematology</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4492197/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4492197/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4492197/?report=printable">Printer Friendly</a></li> | <li><a href="#" data-citationid="PMC4492197" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4492197%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4492197%2F&amp;text=Genome-wide%20analysis%20links%20NFATC2%20with%20asparaginase%20hypersensitivity" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4492197%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25987655" data-aiid="4492197" data-aid="4492197" data-iid="254995" data-domainid="446" data-domain="blood" data-accid="PMC4492197" data-md5="2c5de52e0593f13b525f26ca53c13591"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:39:31-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A9D5EDD57FB210000000005BF034A&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4492197%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=blood&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4492197/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>