<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4051826/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies a new SMAD7 risk variant associated with colorectal cancer risk in East Asians</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4051826/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="International journal of cancer. Journal international du cancer"><meta name="citation_title" content="Genome-wide association study identifies a new SMAD7 risk variant associated with colorectal cancer risk in East Asians"><meta name="citation_authors" content="Ben Zhang, Wei-Hua Jia, Keitaro Matsuo, Aesun Shin, Yong-Bing Xiang, Koichi Matsuda, Sun Ha Jee, Dong-Hyun Kim, Peh Yean Cheah, Zefang Ren, Qiuyin Cai, Jirong Long, Jiajun Shi, Wanqing Wen, Gong Yang, Bu-Tian Ji, Zhi-Zhong Pan, Fumihiko Matsuda, Yu-Tang Gao, Jae Hwan Oh, Yoon-Ok Ahn, Michiaki Kubo, Lai Fun Thean, Eun Jung Park, Hong-Lan Li, Ji Won Park, Jaeseong Jo, Jin-Young Jeong, Satoyo Hosono, Yusuke Nakamura, Xiao-Ou Shu, Yi-Xin Zeng, Wei Zheng"><meta name="citation_date" content="15 August 2014"><meta name="citation_issue" content="4"><meta name="citation_volume" content="135"><meta name="citation_firstpage" content="948"><meta name="citation_doi" content="10.1002/ijc.28733"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4051826/?report=abstract"><meta name="citation_pmid" content="24448986"><meta name="DC.Title" content="Genome-wide association study identifies a new SMAD7 risk variant associated with colorectal cancer risk in East Asians"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Ben Zhang"><meta name="DC.Contributor" content="Wei-Hua Jia"><meta name="DC.Contributor" content="Keitaro Matsuo"><meta name="DC.Contributor" content="Aesun Shin"><meta name="DC.Contributor" content="Yong-Bing Xiang"><meta name="DC.Contributor" content="Koichi Matsuda"><meta name="DC.Contributor" content="Sun Ha Jee"><meta name="DC.Contributor" content="Dong-Hyun Kim"><meta name="DC.Contributor" content="Peh Yean Cheah"><meta name="DC.Contributor" content="Zefang Ren"><meta name="DC.Contributor" content="Qiuyin Cai"><meta name="DC.Contributor" content="Jirong Long"><meta name="DC.Contributor" content="Jiajun Shi"><meta name="DC.Contributor" content="Wanqing Wen"><meta name="DC.Contributor" content="Gong Yang"><meta name="DC.Contributor" content="Bu-Tian Ji"><meta name="DC.Contributor" content="Zhi-Zhong Pan"><meta name="DC.Contributor" content="Fumihiko Matsuda"><meta name="DC.Contributor" content="Yu-Tang Gao"><meta name="DC.Contributor" content="Jae Hwan Oh"><meta name="DC.Contributor" content="Yoon-Ok Ahn"><meta name="DC.Contributor" content="Michiaki Kubo"><meta name="DC.Contributor" content="Lai Fun Thean"><meta name="DC.Contributor" content="Eun Jung Park"><meta name="DC.Contributor" content="Hong-Lan Li"><meta name="DC.Contributor" content="Ji Won Park"><meta name="DC.Contributor" content="Jaeseong Jo"><meta name="DC.Contributor" content="Jin-Young Jeong"><meta name="DC.Contributor" content="Satoyo Hosono"><meta name="DC.Contributor" content="Yusuke Nakamura"><meta name="DC.Contributor" content="Xiao-Ou Shu"><meta name="DC.Contributor" content="Yi-Xin Zeng"><meta name="DC.Contributor" content="Wei Zheng"><meta name="DC.Date" content="2014 Aug 15"><meta name="DC.Identifier" content="10.1002/ijc.28733"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies a new SMAD7 risk variant associated with colorectal cancer risk in East Asians"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association studies (GWAS) of colorectal cancer (CRC) have been conducted primarily in European descendants. In a GWAS conducted in East Asians, we first analyzed approximately 1.7 million single-nucleotide polymorphisms (SNPs) in four studies ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4051826/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4051826/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4051826/pdf/nihms-560464.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C5FD1DD5684A10000000004520289.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4051826%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books">Books</option><option value="mesh">MeSH</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4051826</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Int J Cancer</a></span>. Author manuscript; available in PMC 2015 Aug 15.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=24448986" target="pmc_ext" ref="reftype=publisher&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Int J Cancer. 2014 Aug 15; 135(4): 948–955. </span></a></div><span class="fm-vol-iss-date">Published online 2014 Jan 29. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fijc.28733" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/ijc.28733</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4051826</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS560464</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24448986">24448986</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies a new <em>SMAD7</em> risk variant associated with colorectal cancer risk in East Asians</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Zhang%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439241176096" co-class="co-affbox">Ben Zhang</a>,<sup>1</sup> <a href="/pubmed/?term=Jia%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439241468704" co-class="co-affbox">Wei-Hua Jia</a>,<sup>2</sup> <a href="/pubmed/?term=Matsuo%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439239863952" co-class="co-affbox">Keitaro Matsuo</a>,<sup>3</sup> <a href="/pubmed/?term=Shin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439237325728" co-class="co-affbox">Aesun Shin</a>,<sup>4</sup> <a href="/pubmed/?term=Xiang%20YB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439241750144" co-class="co-affbox">Yong-Bing Xiang</a>,<sup>5</sup> <a href="/pubmed/?term=Matsuda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439241748160" co-class="co-affbox">Koichi Matsuda</a>,<sup>6</sup> <a href="/pubmed/?term=Jee%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439238867600" co-class="co-affbox">Sun Ha Jee</a>,<sup>7</sup> <a href="/pubmed/?term=Kim%20DH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439230892336" co-class="co-affbox">Dong-Hyun Kim</a>,<sup>8</sup> <a href="/pubmed/?term=Cheah%20PY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439238258976" co-class="co-affbox">Peh Yean Cheah</a>,<sup>9,</sup><sup>10,</sup><sup>11</sup> <a href="/pubmed/?term=Ren%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439239620480" co-class="co-affbox">Zefang Ren</a>,<sup>2</sup> <a href="/pubmed/?term=Cai%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439276432224" co-class="co-affbox">Qiuyin Cai</a>,<sup>1</sup> <a href="/pubmed/?term=Long%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439276430240" co-class="co-affbox">Jirong Long</a>,<sup>1</sup> <a href="/pubmed/?term=Shi%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439230886224" co-class="co-affbox">Jiajun Shi</a>,<sup>1</sup> <a href="/pubmed/?term=Wen%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439239439168" co-class="co-affbox">Wanqing Wen</a>,<sup>1</sup> <a href="/pubmed/?term=Yang%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439310667536" co-class="co-affbox">Gong Yang</a>,<sup>1</sup> <a href="/pubmed/?term=Ji%20BT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439310665552" co-class="co-affbox">Bu-Tian Ji</a>,<sup>12</sup> <a href="/pubmed/?term=Pan%20ZZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439231896464" co-class="co-affbox">Zhi-Zhong Pan</a>,<sup>2</sup> <a href="/pubmed/?term=Matsuda%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439240048528" co-class="co-affbox">Fumihiko Matsuda</a>,<sup>13</sup> <a href="/pubmed/?term=Gao%20YT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439240046544" co-class="co-affbox">Yu-Tang Gao</a>,<sup>5</sup> <a href="/pubmed/?term=Oh%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439236859984" co-class="co-affbox">Jae Hwan Oh</a>,<sup>14</sup> <a href="/pubmed/?term=Ahn%20YO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439238635616" co-class="co-affbox">Yoon-Ok Ahn</a>,<sup>15</sup> <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439233332864" co-class="co-affbox">Michiaki Kubo</a>,<sup>16</sup> <a href="/pubmed/?term=Thean%20LF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439233330880" co-class="co-affbox">Lai Fun Thean</a>,<sup>9</sup> <a href="/pubmed/?term=Park%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439232301680" co-class="co-affbox">Eun Jung Park</a>,<sup>7</sup> <a href="/pubmed/?term=Li%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439309958384" co-class="co-affbox">Hong-Lan Li</a>,<sup>5</sup> <a href="/pubmed/?term=Park%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439232169376" co-class="co-affbox">Ji Won Park</a>,<sup>14</sup> <a href="/pubmed/?term=Jo%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439232167392" co-class="co-affbox">Jaeseong Jo</a>,<sup>7</sup> <a href="/pubmed/?term=Jeong%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439231587776" co-class="co-affbox">Jin-Young Jeong</a>,<sup>8</sup> <a href="/pubmed/?term=Hosono%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439240934048" co-class="co-affbox">Satoyo Hosono</a>,<sup>3</sup> <a href="/pubmed/?term=Nakamura%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439240932064" co-class="co-affbox">Yusuke Nakamura</a>,<sup>6</sup> <a href="/pubmed/?term=Shu%20XO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439237612704" co-class="co-affbox">Xiao-Ou Shu</a>,<sup>1</sup> <a href="/pubmed/?term=Zeng%20YX%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439276809616" co-class="co-affbox">Yi-Xin Zeng</a>,<sup>2</sup> and  <a href="/pubmed/?term=Zheng%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986" class="affpopup" co-rid="_co_idm140439312524240" co-class="co-affbox">Wei Zheng</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140439241176096"><h3 class="no_margin">Ben Zhang</h3><p><sup>1</sup> Division of Epidemiology, Department of Medicine, Vanderbilt University School of Medicine, Nashville, TN, USA.</p><div>Find articles by <a href="/pubmed/?term=Zhang%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Ben Zhang</a></div></div><div id="_co_idm140439241468704"><h3 class="no_margin">Wei-Hua Jia</h3><p><sup>2</sup> State Key Laboratory of Oncology in South China, Cancer Center, Sun Yat-sen University, Guangzhou, China.</p><div>Find articles by <a href="/pubmed/?term=Jia%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Wei-Hua Jia</a></div></div><div id="_co_idm140439239863952"><h3 class="no_margin">Keitaro Matsuo</h3><p><sup>3</sup> Division of Epidemiology and Prevention, Aichi Cancer Center Research Institute, Nagoya, Japan.</p><div>Find articles by <a href="/pubmed/?term=Matsuo%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Keitaro Matsuo</a></div></div><div id="_co_idm140439237325728"><h3 class="no_margin">Aesun Shin</h3><p><sup>4</sup> Molecular Epidemiology Branch, National Cancer Center, Goyang-si, Korea.</p><div>Find articles by <a href="/pubmed/?term=Shin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Aesun Shin</a></div></div><div id="_co_idm140439241750144"><h3 class="no_margin">Yong-Bing Xiang</h3><p><sup>5</sup> Department of Epidemiology, Shanghai Cancer Institute, Shanghai, P.R. China.</p><div>Find articles by <a href="/pubmed/?term=Xiang%20YB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Yong-Bing Xiang</a></div></div><div id="_co_idm140439241748160"><h3 class="no_margin">Koichi Matsuda</h3><p><sup>6</sup> Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, The University of Tokyo, 4-6-1, Shirokanedai, Minato-ku, Tokyo 108-8639, Japan.</p><div>Find articles by <a href="/pubmed/?term=Matsuda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Koichi Matsuda</a></div></div><div id="_co_idm140439238867600"><h3 class="no_margin">Sun Ha Jee</h3><p><sup>7</sup> Institute for Health Promotion, Department of Epidemiology and Health Promotion, Graduate School of Public Health, Yonsei University, Seoul, Korea.</p><div>Find articles by <a href="/pubmed/?term=Jee%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Sun Ha Jee</a></div></div><div id="_co_idm140439230892336"><h3 class="no_margin">Dong-Hyun Kim</h3><p><sup>8</sup> Department of Social and Preventive Medicine, Hallym University College of Medicine, Okcheon-dong, Republic of Korea.</p><div>Find articles by <a href="/pubmed/?term=Kim%20DH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Dong-Hyun Kim</a></div></div><div id="_co_idm140439238258976"><h3 class="no_margin">Peh Yean Cheah</h3><p><sup>9</sup> Department of Colorectal Surgery, Singapore General Hospital, Singapore.</p><p><sup>10</sup> Saw Swee Hock School of Public Health, National University of Singapore, Singapore.</p><p><sup>11</sup> Duke-NUS Graduate Medical School, National University of Singapore.</p><div>Find articles by <a href="/pubmed/?term=Cheah%20PY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Peh Yean Cheah</a></div></div><div id="_co_idm140439239620480"><h3 class="no_margin">Zefang Ren</h3><p><sup>2</sup> State Key Laboratory of Oncology in South China, Cancer Center, Sun Yat-sen University, Guangzhou, China.</p><div>Find articles by <a href="/pubmed/?term=Ren%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Zefang Ren</a></div></div><div id="_co_idm140439276432224"><h3 class="no_margin">Qiuyin Cai</h3><p><sup>1</sup> Division of Epidemiology, Department of Medicine, Vanderbilt University School of Medicine, Nashville, TN, USA.</p><div>Find articles by <a href="/pubmed/?term=Cai%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Qiuyin Cai</a></div></div><div id="_co_idm140439276430240"><h3 class="no_margin">Jirong Long</h3><p><sup>1</sup> Division of Epidemiology, Department of Medicine, Vanderbilt University School of Medicine, Nashville, TN, USA.</p><div>Find articles by <a href="/pubmed/?term=Long%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Jirong Long</a></div></div><div id="_co_idm140439230886224"><h3 class="no_margin">Jiajun Shi</h3><p><sup>1</sup> Division of Epidemiology, Department of Medicine, Vanderbilt University School of Medicine, Nashville, TN, USA.</p><div>Find articles by <a href="/pubmed/?term=Shi%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Jiajun Shi</a></div></div><div id="_co_idm140439239439168"><h3 class="no_margin">Wanqing Wen</h3><p><sup>1</sup> Division of Epidemiology, Department of Medicine, Vanderbilt University School of Medicine, Nashville, TN, USA.</p><div>Find articles by <a href="/pubmed/?term=Wen%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Wanqing Wen</a></div></div><div id="_co_idm140439310667536"><h3 class="no_margin">Gong Yang</h3><p><sup>1</sup> Division of Epidemiology, Department of Medicine, Vanderbilt University School of Medicine, Nashville, TN, USA.</p><div>Find articles by <a href="/pubmed/?term=Yang%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Gong Yang</a></div></div><div id="_co_idm140439310665552"><h3 class="no_margin">Bu-Tian Ji</h3><p><sup>12</sup> Division of Cancer Epidemiology &amp; Genetics, National Cancer Institute, Bethesda, MD, USA.</p><div>Find articles by <a href="/pubmed/?term=Ji%20BT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Bu-Tian Ji</a></div></div><div id="_co_idm140439231896464"><h3 class="no_margin">Zhi-Zhong Pan</h3><p><sup>2</sup> State Key Laboratory of Oncology in South China, Cancer Center, Sun Yat-sen University, Guangzhou, China.</p><div>Find articles by <a href="/pubmed/?term=Pan%20ZZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Zhi-Zhong Pan</a></div></div><div id="_co_idm140439240048528"><h3 class="no_margin">Fumihiko Matsuda</h3><p><sup>13</sup> Center for Genomic Medicine, Graduate School of Medicine, Kyoto University, Kyoto, Japan</p><div>Find articles by <a href="/pubmed/?term=Matsuda%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Fumihiko Matsuda</a></div></div><div id="_co_idm140439240046544"><h3 class="no_margin">Yu-Tang Gao</h3><p><sup>5</sup> Department of Epidemiology, Shanghai Cancer Institute, Shanghai, P.R. China.</p><div>Find articles by <a href="/pubmed/?term=Gao%20YT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Yu-Tang Gao</a></div></div><div id="_co_idm140439236859984"><h3 class="no_margin">Jae Hwan Oh</h3><p><sup>14</sup> Center for Colorectal Cancer, National Cancer Center, Goyang-si, Korea.</p><div>Find articles by <a href="/pubmed/?term=Oh%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Jae Hwan Oh</a></div></div><div id="_co_idm140439238635616"><h3 class="no_margin">Yoon-Ok Ahn</h3><p><sup>15</sup> Department of Preventive Medicine, Seoul National University College of Medicine, Seoul, Korea.</p><div>Find articles by <a href="/pubmed/?term=Ahn%20YO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Yoon-Ok Ahn</a></div></div><div id="_co_idm140439233332864"><h3 class="no_margin">Michiaki Kubo</h3><p><sup>16</sup> Center for Genomic Medicine, The Institute of Physical and Chemical Research (RIKEN), Kanagawa, Japan.</p><div>Find articles by <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Michiaki Kubo</a></div></div><div id="_co_idm140439233330880"><h3 class="no_margin">Lai Fun Thean</h3><p><sup>9</sup> Department of Colorectal Surgery, Singapore General Hospital, Singapore.</p><div>Find articles by <a href="/pubmed/?term=Thean%20LF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Lai Fun Thean</a></div></div><div id="_co_idm140439232301680"><h3 class="no_margin">Eun Jung Park</h3><p><sup>7</sup> Institute for Health Promotion, Department of Epidemiology and Health Promotion, Graduate School of Public Health, Yonsei University, Seoul, Korea.</p><div>Find articles by <a href="/pubmed/?term=Park%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Eun Jung Park</a></div></div><div id="_co_idm140439309958384"><h3 class="no_margin">Hong-Lan Li</h3><p><sup>5</sup> Department of Epidemiology, Shanghai Cancer Institute, Shanghai, P.R. China.</p><div>Find articles by <a href="/pubmed/?term=Li%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Hong-Lan Li</a></div></div><div id="_co_idm140439232169376"><h3 class="no_margin">Ji Won Park</h3><p><sup>14</sup> Center for Colorectal Cancer, National Cancer Center, Goyang-si, Korea.</p><div>Find articles by <a href="/pubmed/?term=Park%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Ji Won Park</a></div></div><div id="_co_idm140439232167392"><h3 class="no_margin">Jaeseong Jo</h3><p><sup>7</sup> Institute for Health Promotion, Department of Epidemiology and Health Promotion, Graduate School of Public Health, Yonsei University, Seoul, Korea.</p><div>Find articles by <a href="/pubmed/?term=Jo%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Jaeseong Jo</a></div></div><div id="_co_idm140439231587776"><h3 class="no_margin">Jin-Young Jeong</h3><p><sup>8</sup> Department of Social and Preventive Medicine, Hallym University College of Medicine, Okcheon-dong, Republic of Korea.</p><div>Find articles by <a href="/pubmed/?term=Jeong%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Jin-Young Jeong</a></div></div><div id="_co_idm140439240934048"><h3 class="no_margin">Satoyo Hosono</h3><p><sup>3</sup> Division of Epidemiology and Prevention, Aichi Cancer Center Research Institute, Nagoya, Japan.</p><div>Find articles by <a href="/pubmed/?term=Hosono%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Satoyo Hosono</a></div></div><div id="_co_idm140439240932064"><h3 class="no_margin">Yusuke Nakamura</h3><p><sup>6</sup> Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, The University of Tokyo, 4-6-1, Shirokanedai, Minato-ku, Tokyo 108-8639, Japan.</p><div>Find articles by <a href="/pubmed/?term=Nakamura%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Yusuke Nakamura</a></div></div><div id="_co_idm140439237612704"><h3 class="no_margin">Xiao-Ou Shu</h3><p><sup>1</sup> Division of Epidemiology, Department of Medicine, Vanderbilt University School of Medicine, Nashville, TN, USA.</p><div>Find articles by <a href="/pubmed/?term=Shu%20XO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Xiao-Ou Shu</a></div></div><div id="_co_idm140439276809616"><h3 class="no_margin">Yi-Xin Zeng</h3><p><sup>2</sup> State Key Laboratory of Oncology in South China, Cancer Center, Sun Yat-sen University, Guangzhou, China.</p><div>Find articles by <a href="/pubmed/?term=Zeng%20YX%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Yi-Xin Zeng</a></div></div><div id="_co_idm140439312524240"><h3 class="no_margin">Wei Zheng</h3><p><sup>1</sup> Division of Epidemiology, Department of Medicine, Vanderbilt University School of Medicine, Nashville, TN, USA.</p><div>Find articles by <a href="/pubmed/?term=Zheng%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24448986">Wei Zheng</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140439232137104_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140439232137104_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140439232137104_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup> Division of Epidemiology, Department of Medicine, Vanderbilt University School of Medicine, Nashville, TN, USA.</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup> State Key Laboratory of Oncology in South China, Cancer Center, Sun Yat-sen University, Guangzhou, China.</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup> Division of Epidemiology and Prevention, Aichi Cancer Center Research Institute, Nagoya, Japan.</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup> Molecular Epidemiology Branch, National Cancer Center, Goyang-si, Korea.</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup> Department of Epidemiology, Shanghai Cancer Institute, Shanghai, P.R. China.</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup> Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, The University of Tokyo, 4-6-1, Shirokanedai, Minato-ku, Tokyo 108-8639, Japan.</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup> Institute for Health Promotion, Department of Epidemiology and Health Promotion, Graduate School of Public Health, Yonsei University, Seoul, Korea.</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup> Department of Social and Preventive Medicine, Hallym University College of Medicine, Okcheon-dong, Republic of Korea.</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup> Department of Colorectal Surgery, Singapore General Hospital, Singapore.</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup> Saw Swee Hock School of Public Health, National University of Singapore, Singapore.</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup> Duke-NUS Graduate Medical School, National University of Singapore.</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup> Division of Cancer Epidemiology &amp; Genetics, National Cancer Institute, Bethesda, MD, USA.</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup> Center for Genomic Medicine, Graduate School of Medicine, Kyoto University, Kyoto, Japan</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup> Center for Colorectal Cancer, National Cancer Center, Goyang-si, Korea.</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup> Department of Preventive Medicine, Seoul National University College of Medicine, Seoul, Korea.</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup> Center for Genomic Medicine, The Institute of Physical and Chemical Research (RIKEN), Kanagawa, Japan.</div><div id="CR1"><strong>Corresponding author contact information</strong>: Wei Zheng, M.D., Ph.D. Vanderbilt Epidemiology Center and Vanderbilt-Ingram Cancer Center Vanderbilt University School of Medicine 2525 West End Avenue, 8<sup>th</sup> Floor, Nashville, TN 37203-1738 Phone: (615) 936-0682 Fax: (615) 936-8241 <a href="mailto:dev@null" data-email="ude.tlibrednav@gnehz.iew" class="oemail">ude.tlibrednav@gnehz.iew</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140439232137104_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=24448986" target="pmc_ext" ref="reftype=publisher&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Int J Cancer</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4051826/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Material.</strong></div><div><a href="/pmc/articles/PMC4051826/bin/NIHMS560464-supplement-Supplementary_Material.pdf" data-ga-action="click_feat_suppl">NIHMS560464-supplement-Supplementary_Material.pdf</a><span style="color:gray"> (785K)</span></div><div class="small guid">GUID:&nbsp;4E2254E1-95AB-486B-ADE8-9832C715767D</div></div></dd></dl></div><div id="idm140439242626064" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140439242626064title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Genome-wide association studies (GWAS) of colorectal cancer (CRC) have been conducted primarily in European descendants. In a GWAS conducted in East Asians, we first analyzed approximately 1.7 million single-nucleotide polymorphisms (SNPs) in four studies with 1,773 CRC cases and 2,642 controls. We then selected 66 promising SNPs for replication and genotyped them in three independent studies with 3,612 cases and 3,523 controls. Five SNPs were further evaluated using data from four additional studies including up to 3,290 cases and 4,339 controls. SNP rs7229639 in the <em>SMAD7</em> gene was found to be associated with CRC risk with an odds ratio (95% confidence interval) associated with the minor allele (A) of 1.22 (1.15-1.29) in the combined analysis of all 11 studies (<em>P</em> = 2.93×10<sup>−11</sup>). SNP rs7229639 is 2,487 bp upstream from rs4939827, a risk variant identified previously in a European-ancestry GWAS in relation to CRC risk. However, these two SNPs are not correlated in East Asians (<em>r</em><sup>2</sup> = 0.008) nor in Europeans (<em>r</em><sup>2</sup> = 0.146). The CRC association with rs7229639 remained statistically significant after adjusting for rs4939827 as well as three additional CRC risk variants (rs58920878, rs12953717, and rs4464148) reported previously in this region. SNPs rs7229639 and rs4939827 explained approximately 1% of the familial relative risk of CRC in East Asians. This study identifies a new CRC risk variant in the <em>SMAD7</em> gene, further highlighting the significant role of this gene in the etiology of CRC.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Genome-wide association study, GWAS, colorectal cancer, <em>SMAD7</em>, genetic susceptibility, single-nucleotide polymorphisms, epidemiology</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">Colorectal cancer (CRC) is the third-most common cancer and second-leading cause of cancer death worldwide (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138268">1</a>). While environmental factors are believed to play an important role in the etiology of CRC, it is estimated that approximately 35% of CRC risk may be attributable to inherited factors (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138247">2</a>). To date, approximately 19 common genetic susceptibility loci for CRC have been identified in genome-wide association studies (GWAS) (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138252">3</a>-<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138261">13</a>). However, these genetic variants, along with high-penetrance germline mutations in known CRC susceptibility genes, including <em>APC</em>, the DNA mismatch repair genes, <em>SMAD4, AXIN2, BMPR1A, TGFBR2, POLD1, STK11</em>, and <em>MUTYH</em> (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138253">14</a>;<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462760797">15</a>), explain less than 15% of excess familial risk of CRC (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138274">3</a>-<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462760798">15</a>).</p><p id="P3" class="p p-last">Most previous GWAS for CRC were conducted in European-ancestry populations. Given the potential difference in genetic architectures between East Asians and European ancestry populations, it is possible that some genetic risk variants for CRC identified in European descendants may not be generalizable to East Asians. Also, GWAS conducted in East Asians could possibly identify genetic risk variants unique to this population. In 2009, we initiated a GWAS in East Asians, the Asia Colorectal Cancer Consortium, and identified three novel genetic susceptibility loci for CRC (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138273">16</a>). In this paper, we reported additional findings from this consortium regarding the identification of a new risk variant for CRC in the <em>SMAD7</em> gene.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials and Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Study populations</h3><p id="P4" class="p p-first-last">This study, conducted as part of the Asia Colorectal Cancer Consortium, included 8,891 CRC cases and 10,547 cancer-free controls of East Asian ancestry recruited in eight centers located in China, Korea, Japan, and Singapore (<strong><a href="/pmc/articles/PMC4051826/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a></strong>). Specifically, Stage 1 consisted of four studies: Shanghai Study 1 (Shanghai-1, <em>n</em> = 982), Shanghai Study 2 (Shanghai-2, <em>n</em> = 553), Guangzhou Study 1 (Guangzhou-1, <em>n</em> = 1,666), Aichi Study 1 (Aichi-1, <em>n</em> = 1,439). Stage 2 consisted of seven studies: Guangzhou Study 2 (Guangzhou-2, <em>n</em> = 2,892), Korean-National Cancer Center Study (Korea-NCC, <em>n</em> = 2,721), Seoul Study (Korea-Seoul, <em>n</em> = 1,522), Korean Cancer Prevention Study-II (KCPS-II, <em>n</em> = 1,302), the Japan-BioBank Study (Japan-BioBank, <em>n</em> = 3,498), Singapore Chinese Study (SCH, <em>n</em> = 2,000), and Aichi Study 2 (Aichi-2, <em>n</em> = 863). Summary descriptions of these 11 participating studies from eight centers are provided in <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong>. Study protocols were approved by relevant institutional review boards for all study sites.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p24">Descriptions of participating studies and subjects included in this analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140439238992208" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="2" rowspan="1" valign="top" align="center">Sample size<sup><sup>1</sup></sup><hr></th><th colspan="2" rowspan="1" valign="top" align="center">Mean age (years)<sup><sup>2</sup></sup><hr></th><th colspan="2" rowspan="1" valign="top" align="center">Female (%)<sup><sup>2</sup></sup><hr></th><th rowspan="1" colspan="1" valign="top" align="left"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Study</th><th rowspan="1" colspan="1" valign="top" align="left">Population</th><th rowspan="1" colspan="1" valign="top" align="left">Genotyped</th><th rowspan="1" colspan="1" valign="top" align="left">After QC</th><th rowspan="1" colspan="1" valign="top" align="left">Cases</th><th rowspan="1" colspan="1" valign="top" align="left">Controls</th><th rowspan="1" colspan="1" valign="top" align="left">Cases</th><th rowspan="1" colspan="1" valign="top" align="left">Controls</th><th rowspan="1" colspan="1" valign="top" align="left">Genotyping platform</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Stage 1</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Shanghai-1</td><td rowspan="1" colspan="1" valign="top" align="left">Chinese</td><td rowspan="1" colspan="1" valign="top" align="left">481/501</td><td rowspan="1" colspan="1" valign="top" align="left">474/497</td><td rowspan="1" colspan="1" valign="top" align="left">60.02</td><td rowspan="1" colspan="1" valign="top" align="left">60.20</td><td rowspan="1" colspan="1" valign="top" align="left">73.84</td><td rowspan="1" colspan="1" valign="top" align="left">72.64</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Shanghai-2</td><td rowspan="1" colspan="1" valign="top" align="left">Chinese</td><td rowspan="1" colspan="1" valign="top" align="left">296/257</td><td rowspan="1" colspan="1" valign="top" align="left">254/231</td><td rowspan="1" colspan="1" valign="top" align="left">61.16</td><td rowspan="1" colspan="1" valign="top" align="left">60.75</td><td rowspan="1" colspan="1" valign="top" align="left">54.72</td><td rowspan="1" colspan="1" valign="top" align="left">56.71</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina OmniExpress</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Guangzhou-1</td><td rowspan="1" colspan="1" valign="top" align="left">Chinese</td><td rowspan="1" colspan="1" valign="top" align="left">694/972</td><td rowspan="1" colspan="1" valign="top" align="left">641/972</td><td rowspan="1" colspan="1" valign="top" align="left">54.86</td><td rowspan="1" colspan="1" valign="top" align="left">47.40</td><td rowspan="1" colspan="1" valign="top" align="left">36.51</td><td rowspan="1" colspan="1" valign="top" align="left">27.06</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina OmniExpress/Human610-Quad</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Aichi-1</td><td rowspan="1" colspan="1" valign="top" align="left">Japanese</td><td rowspan="1" colspan="1" valign="top" align="left">497/942</td><td rowspan="1" colspan="1" valign="top" align="left">404/942</td><td rowspan="1" colspan="1" valign="top" align="left">59.43</td><td rowspan="1" colspan="1" valign="top" align="left">47.88</td><td rowspan="1" colspan="1" valign="top" align="left">37.38</td><td rowspan="1" colspan="1" valign="top" align="left">47.77</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina OmniExpress/HumanHap610</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Subtotal</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">1,968/2,672</td><td rowspan="1" colspan="1" valign="top" align="left">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Stage 2</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Guangzhou-2<sup><sup>3</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Chinese</td><td rowspan="1" colspan="1" valign="top" align="left">1,371/1,521</td><td rowspan="1" colspan="1" valign="top" align="left">1,371/1,521</td><td rowspan="1" colspan="1" valign="top" align="left">58.22</td><td rowspan="1" colspan="1" valign="top" align="left">54.64</td><td rowspan="1" colspan="1" valign="top" align="left">37.96</td><td rowspan="1" colspan="1" valign="top" align="left">39.25</td><td rowspan="1" colspan="1" valign="top" align="left">Sequenom</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Korea-NCC<sup><sup>3</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Korean</td><td rowspan="1" colspan="1" valign="top" align="left">1,392/1,329</td><td rowspan="1" colspan="1" valign="top" align="left">1,392/1,329</td><td rowspan="1" colspan="1" valign="top" align="left">58.19</td><td rowspan="1" colspan="1" valign="top" align="left">55.59</td><td rowspan="1" colspan="1" valign="top" align="left">37.64</td><td rowspan="1" colspan="1" valign="top" align="left">38.60</td><td rowspan="1" colspan="1" valign="top" align="left">Sequenom</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Korea-Seoul<sup><sup>3</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Korean</td><td rowspan="1" colspan="1" valign="top" align="left">849/673</td><td rowspan="1" colspan="1" valign="top" align="left">849/673</td><td rowspan="1" colspan="1" valign="top" align="left">59.05</td><td rowspan="1" colspan="1" valign="top" align="left">57.19</td><td rowspan="1" colspan="1" valign="top" align="left">40.99</td><td rowspan="1" colspan="1" valign="top" align="left">47.85</td><td rowspan="1" colspan="1" valign="top" align="left">Sequenom</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;KCPS-II<sup><sup>4</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Korean</td><td rowspan="1" colspan="1" valign="top" align="left">325/977</td><td rowspan="1" colspan="1" valign="top" align="left">325/976</td><td rowspan="1" colspan="1" valign="top" align="left">51.38</td><td rowspan="1" colspan="1" valign="top" align="left">41.27</td><td rowspan="1" colspan="1" valign="top" align="left">27.08</td><td rowspan="1" colspan="1" valign="top" align="left">43.34</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 5.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Japan-BioBank<sup><sup>4</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Japanese</td><td rowspan="1" colspan="1" valign="top" align="left">1,595/1,903</td><td rowspan="1" colspan="1" valign="top" align="left">1,583/1,898</td><td rowspan="1" colspan="1" valign="top" align="left">58.37</td><td rowspan="1" colspan="1" valign="top" align="left">52.49</td><td rowspan="1" colspan="1" valign="top" align="left">39.17</td><td rowspan="1" colspan="1" valign="top" align="left">36.22</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina HumanHap 610K/550K</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;SCH<sup><sup>4</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Chinese</td><td rowspan="1" colspan="1" valign="top" align="left">1,000/1,000</td><td rowspan="1" colspan="1" valign="top" align="left">991/993</td><td rowspan="1" colspan="1" valign="top" align="left">69.99</td><td rowspan="1" colspan="1" valign="top" align="left">69.40</td><td rowspan="1" colspan="1" valign="top" align="left">42.99</td><td rowspan="1" colspan="1" valign="top" align="left">43.00</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 6.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Aichi-2<sup><sup>3</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Japanese</td><td rowspan="1" colspan="1" valign="top" align="left">391/472</td><td rowspan="1" colspan="1" valign="top" align="left">391/472</td><td rowspan="1" colspan="1" valign="top" align="left">59.87</td><td rowspan="1" colspan="1" valign="top" align="left">60.08</td><td rowspan="1" colspan="1" valign="top" align="left">36.06</td><td rowspan="1" colspan="1" valign="top" align="left">35.17</td><td rowspan="1" colspan="1" valign="top" align="left">Sequenom</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;Subtotal</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">6,923/7,875</td><td rowspan="1" colspan="1" valign="top" align="left">6,902/7,862</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Total</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">8,891/10,547</td><td rowspan="1" colspan="1" valign="top" align="left">8,675/10,504</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr></tbody></table></div><div id="largeobj_idm140439238992208" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4051826/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>1</sup>Number of cases/controls. QC, quality control.</div><div id="TFN2"><sup>2</sup>Among samples remained after QC exclusion.</div><div id="TFN3"><sup>3</sup>Studies for direct genotyping.</div><div id="TFN4"><sup>4</sup>Studies for in silico replication.</div></div></div></div><div id="S4" class="sec"><h3 id="S4title">Laboratory procedures</h3><p id="P5" class="p p-first">Genomic DNA was extracted from either blood or saliva samples according to standard protocols. In Stage 1, genotyping was performed using Affymetrix Genome-Wide Human SNP Array 6.0 (Affy 6.0, 906,602 SNPs) for Shanghai-1 cases and controls; Illumina HumanOmniExpress BeadChip (Illumina OmniExpress, 729,462 SNPs) for Shanghai-2 cases and controls, Guangzhou-1 cases and Aichi-1 cases; Illumina Human610-Quad BeadChip (620,901 SNPs) for Guangzhou-1 controls; and Illumina Infinium HumanHap610 BeadChip (592,044 SNPs) for Aichi-1 controls. Genotype calling was performed using Birdseed algorithm for Affymetrix 6.0 or GenomeStudio software for Illumina GWAS platforms based on manufacturer's protocols.</p><p id="P6">Quality control (QC) protocols were applied to exclude samples and SNPs from all four studies in Stage 1 as described previously (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138246">17</a>-<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138258">19</a>), including: 1) genotype call rate per sample &lt; 95%, 2) genetically identical (PI_HAT &gt; 0.9) or duplicated samples, 3) genetic sex inconsistent with survey/clinical data, 4) samples with close relative (PI_HAT &gt; 0.25), 5) population structure inconsistent with HapMap Asians (see <strong>Statistical analysis</strong>), 6) genotype call rate per SNP &lt; 95%, 7) minor allele frequency (MAF) &lt; 0.05, 8) genotyping concordance &lt; 95% in QC samples, 9) Hardy-Weinberg equilibrium (HWE) <em>P</em> &lt; 1×10<sup>−5</sup> in controls, or 10) SNPs not in autosomes. After these QC procedures, 580,086 SNPs for 971 individuals (474 cases and 497 controls) remained in the Shanghai-1 dataset; 515,701 SNPs for 485 individuals (254 cases and 231 controls) remained in the Shanghai-2 dataset; 522,096 SNPs for 641 cases and 435,925 SNPs for 972 controls remained in the Guangzhou-1 dataset; and 478,246 SNPs for cases and 443,065 SNPs for controls remained in the Aichi-1 dataset.</p><p id="P7" class="p p-last">Stage 2 genotyping was performed using the Sequenom MassARRAY platform (Sequenom, San Diego, CA, USA) for the promising 66 SNPs selected from Stage 1. These SNPs were genotyped in Guangzhou-2, Korea-NCC, and Korea-Seoul studies. Again in Stage 2, standard QC protocols were applied to exclude SNPs, including: 1) genotype call rate per SNP &lt; 95%, 2) unclear genotyping cluster, 3) genotyping concordance &lt; 95% in QC samples, 4) HWE <em>P</em> &lt; 7.7×10<sup>−4</sup> (0.05/65) in controls. After these QC procedures, the number of eligible SNPs was 65 in Guangzhou-2, 64 in Korea-NCC, and 60 in Korea-Seoul studies. Five of these SNPs were taken forward for <em>in silico</em> replication in KCPS-II (n = 5), Japan-BioBank (n = 5), and SCH (n = 2; rs7229639 and rs2143619). Genotyping was conducted using either Affymetrix Genome-Wide Human SNP Array 5.0 in KCPS-II or Illumina HumanHap 610K and 550K in Japan-BioBank or Affymetrix Genome-Wide Human SNP Array 6.0 for SCH. Details of the QC procedures and data processing for samples included in these three studies have been previously reported in elsewhere (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138271">11</a>;<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138275">20</a>;<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138249">21</a>). Finally, SNP rs7229639 was further genotyped in Aichi-2 using Sequenom along with SNPs for other projects.</p></div><div id="S5" class="sec sec-last"><h3 id="S5title">Statistical analysis</h3><p id="P8" class="p p-first">Genome-wide imputation for samples in four Stage 1 studies was performed using program MACH 1.0 (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138255">22</a>) based on data from the 90 CHB (Han Chinese in Beijing, China) and JPT (Japanese in Tokyo, Japan) samples included in the HapMap project (release 22). After exclusion of imputed SNPs with MAF &lt; 0.05 and RSQ &lt; 0.50, 1,695,815 genotyped or imputed SNPs remained for meta-analyses.</p><p id="P9">To evaluate the population structure and identify potential genetic outliers, we performed principal components analysis (PCA) using EIGENSTRAT, version 2.5 (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138281">23</a>). We selected a set of ~6,000 uncorrelated SNPs (closest distance between two SNPs &gt; 200kb, MAF &gt; 0.2, <em>r</em><sup>2</sup> &lt; 0.1, and call rate &gt; 99%) shared among all 4,415 samples included in Stage 1 and the HapMap Project using PLINK version 1.07 (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138260">24</a>). Genotype data of these SNPs from the four Stage 1 studies were pooled together with HapMap data (release 23a) to generate the first ten principal components. Samples were removed from the final analysis if they were more than 6σ away from the means of PC1 and PC2.</p><p id="P10">Associations of SNPs with CRC risk in each of the four studies included in Stage 1 were assessed by assuming a log-additive effect of the allelic dosage of the SNPs. Odds ratios (ORs) and 95% confidence intervals (CIs) were generated from logistic regression models, adjusted for age, sex, and the first ten principal components. We coded 0, 1, or 2 copies of the effect alleles as dosage for genotyped SNPs and used the expected number of copies of the effect alleles as dosage score for imputed SNPs to account for imputation uncertainty (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138251">25</a>). The meta-analysis was performed using an inverse-variance method assuming fixed-effects, with a Cochran's <em>Q</em> statistic to test for heterogeneity (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138265">26</a>) and <em>I</em><sup>2</sup> statistic to quantify heterogeneity (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138272">27</a>) across studies. Summary statistics of genome-wide meta-analyses were generated using the METAL program (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138266">28</a>). Similar to Stage 1, we evaluated associations of CRC risk with SNPs in each of the studies included in Stage 2 using logistic regression models with adjustment for age and sex. Summary estimates in Stage 2, all studies combined, and subgroups by populations (Chinese, Korean, and Japanese) and sex (male, female) were also obtained using a fixed-effects meta-analysis with METAL. SNPs showing an association at <em>P</em> &lt;5×10<sup>−8</sup> in the combined analysis of all studies were considered genome-wide significant. We conducted haplotype association analysis for two SNPs in 18q21.1 using SAS Genetics v9.3 with logistic regression models. The familial relative risk (<em>λ</em>) to offspring of an affected individual due to a single locus is estimated using formula: λ = (<em>pr</em><sup>2</sup> + <em>q</em>)/(<em>pr</em> + <em>q</em>)<sup>2</sup>, where <em>p</em> is the frequency of the risk allele, <em>q</em> = 1-<em>p</em> is the frequency of the reference allele, and <em>r</em> is the per-allele relative risk (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138257">29</a>). The proportion of the familial relative risk explained by a locus, assuming a multiplicative interaction between markers in the locus and other loci, is calculated as ln(∏<sub><em>i</em></sub><em>λ</em><sub><em>i</em></sub>) ∕ ln(<em>λ</em><sub>0</sub>), where <em>λ</em><sub>o</sub> is the overall familial relative risk which is assigned to be 2.2 for CRC estimated from a previous meta-analysis (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138248">30</a>). Assuming that the risks associated with each locus combine multiplicatively, the combined contribution of the familial relative risks from multiple loci is equal to: (∏<sub><em>i</em></sub><em>λ</em><sub><em>i</em></sub>) ∕ ln(<em>λ</em><sub>0</sub>).</p><p id="P11" class="p p-last">To visualize population substructure, we drew a PCA plot using data from the 4,415 Stage 1 samples and 270 subjects from HapMap based on the first two principal components using R version 2.13.0 (<a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>). We also used R package to generate a forest plot to display the association of rs7229639 with CRC risk across studies. We generated regional association plots using the website-based software LocusZoom, version 1.1 (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138254">31</a>). Haploview version 4.2 (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138283">32</a>) was used to infer linkage disequilibrium (LD) structure.</p></div></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Results</h2><p id="P12" class="p p-first">A total of 19,179 samples are included in the current analysis (<strong><a href="/pmc/articles/PMC4051826/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a></strong>). Cases and controls were reasonably well matched by age and sex in most of the participating studies. All samples in Stage 1 showed a clear East Asian origin and none of them were more than 6σ away from the means of PC1 and PC2 (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a></strong>). Cases and controls in each of the four studies were in the same cluster compared with East Asians in HapMap. After standard QC filter, a total of 1,695,815 SNPs were finally included in the association analyses. Using Stage 1 data, we evaluated association of CRC risk with the 26 previously reported SNPs. Of the 22 SNPs initially identified from GWAS conducted in European-ancestry populations, rs6691170 and rs16892766 are monomorphic in East Asians. One SNP, rs5934683 in Chromosome X, was excluded from analyses in this study. All other 19 SNPs showed an association with CRC risk in the same direction as reported initially (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a></strong>). Nine of 19 SNPs showed a statistically significant association with CRC risk at <em>P</em> &lt; 0.05. Of the four SNPs initially identified in East Asians, three (rs647161, rs10774214 and rs2423279) were also significantly associated with CRC in our Stage 1, and one (rs7758229) was not associated with CRC risk in these data. To identify new genetic risk variants for CRC, we selected the 66 most promising SNPs for replication in Stage 2 using the following criteria: 1) MAF &gt; 0.05 in each of the four Stage 1 studies, 2) RSQ &gt; 0.70 in all four studies, 3) <em>P</em><sub>meta</sub> &lt; 5.5 × 10<sup>−5</sup>, 4) no heterogeneity (<em>P</em><sub>heterogeneity</sub> &gt; 0.05 and I<sup>2</sup> &lt; 25%), 5) uncorrelated with SNPs in known CRC loci or with each other (<em>r</em><sup>2</sup> &lt; 0.20), and 6) data available in all four Stage 1 studies. These 66 SNPs were not evaluated in our previously published study (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138259">16</a>).</p><p id="P13">Of the 66 SNPs selected for Stage 2 replication, 59 SNPs were successfully genotyped in 3,612 cases and 3,523 controls from three studies (Guangzhou-2, Korea-NCC, and Korea-Seoul) included in Stage 2 (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a></strong>). Five SNPs (rs7923556, rs1539213, rs7229639, rs7247381, and rs2143619) from five different regions (10q21.2, 14q21.3, 18q21.1, 19q12, and 20p12.2) showed an association with CRC risk at <em>P</em>-value &lt; 0.05 in the same direction as observed in Stage 1 (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a></strong>). For these five SNPs, we conducted <em>in silico</em> replication using data from three additional studies (KCPS-II, Japan-BioBank, and SCH) with 2,899 cases and 3,867 controls. SNP rs7229639 was further genotyped in Aichi-2 including 391 cases and 472 controls. Of all the 59 SNPs evaluated in Stage 2, only rs7229639 was consistently associated with CRC risk across all seven studies, showing strong evidence of replication, with <em>P</em>-value 3.39 × 10<sup>−8</sup>. Joint analysis of samples in Stages 1 and 2 yielded per-allele OR (95% CI) 1.22 (1.15-1.29) and <em>P</em>-value 2.93 × 10<sup>−11</sup> (<strong><a href="/pmc/articles/PMC4051826/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a></strong>), which is substantially lower than the genome-wide significance level of 5 × 10<sup>−8</sup>. This association was consistent across all eleven studies in Stages 1 and 2 (<strong><a href="/pmc/articles/PMC4051826/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a></strong>), with little evidence of between-study heterogeneity (<em>P</em> for heterogeneity = 0.726, <em>I</em><sup>2</sup> = 0%). Stratification analysis did not reveal any apparent heterogeneity across Chinese (OR = 1.20), Korean (OR = 1.21), or Japanese (OR = 1.23) subjects (<em>P</em> for heterogeneity = 0.958), or between men (OR = 1.28) and women (OR = 1.19) (<em>P</em> for heterogeneity = 0.268) for this SNP (<strong><a href="/pmc/articles/PMC4051826/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a></strong>). Finally, using genotype data from four studies included in Stage 2, we found that the risk of CRC was increased in a dose-response manner with the number of minor allele (A) of rs7229639 (<em>P</em> for trend = 4.78 × 10<sup>−7</sup>), with ORs of 1.21 (95% CI: 1.10-1.34) and 1.62 (95% CI: 1.26-2.08) for heterozygotes and homozygotes, respectively. These data support an additive model rather than dominant (OR = 1.25, 95% CI: 1.14-1.38; <em>P</em> = 4.39 × 10<sup>−6</sup>) or recessive model (OR= 1.52, 95% CI: 1.19-1.96; <em>P</em> = 9.95 × 10<sup>−4</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4051826/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140439241267872" class="figure"><a href="/pmc/articles/PMC4051826/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4051826_nihms-560464-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-560464-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4051826/bin/nihms-560464-f0001.jpg"></a></div><div id="largeobj_idm140439241267872" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4051826/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4051826/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Association of rs7229639 with colorectal cancer risk by studies</strong><p id="__p23">Per-allele ORs are presented with the area of the box proportion to the inverse variance weight of the estimate. Horizontal lines show 95% CIs.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p25">Association of rs7229639 in the <em>SMAD7</em> gene with colorectal cancer risk in East Asians</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140439242387360" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="2" rowspan="1" valign="top" align="center">Sample size<sup><sup>1</sup></sup><hr></th><th colspan="2" rowspan="1" valign="top" align="center">EAF<sup><sup>2</sup></sup><hr></th><th colspan="2" rowspan="1" valign="top" align="center">Per-allele association<hr></th><th colspan="2" rowspan="1" valign="top" align="center">Heterogeneity<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Study</th><th rowspan="1" colspan="1" valign="top" align="center">Cases</th><th rowspan="1" colspan="1" valign="top" align="center">Controls</th><th rowspan="1" colspan="1" valign="top" align="center">Cases</th><th rowspan="1" colspan="1" valign="top" align="center">Controls</th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI)<sup><sup>3</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center">
<em>P</em>
<sub>heterogeneity</sub>
<sup><sup>4</sup></sup>
</th><th rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sup><sup>2</sup></sup> (%)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Stage 1</td><td rowspan="1" colspan="1" valign="top" align="center">1,773</td><td rowspan="1" colspan="1" valign="top" align="center">2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.186</td><td rowspan="1" colspan="1" valign="top" align="center">0.152</td><td rowspan="1" colspan="1" valign="top" align="center">1.32 (1.17-1.50)</td><td rowspan="1" colspan="1" valign="top" align="center">1.22×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.269</td><td rowspan="1" colspan="1" valign="top" align="center">24</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Stage 2</td><td rowspan="1" colspan="1" valign="top" align="center">6,800</td><td rowspan="1" colspan="1" valign="top" align="center">7,761</td><td rowspan="1" colspan="1" valign="top" align="center">0.176</td><td rowspan="1" colspan="1" valign="top" align="center">0.153</td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.13-1.28)</td><td rowspan="1" colspan="1" valign="top" align="center">3.39×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.897</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Overall</td><td rowspan="1" colspan="1" valign="top" align="center">8,573</td><td rowspan="1" colspan="1" valign="top" align="center">10,403</td><td rowspan="1" colspan="1" valign="top" align="center">0.178</td><td rowspan="1" colspan="1" valign="top" align="center">0.152</td><td rowspan="1" colspan="1" valign="top" align="center">1.22 (1.15-1.29)</td><td rowspan="1" colspan="1" valign="top" align="center">2.93×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.726</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chinese</td><td rowspan="1" colspan="1" valign="top" align="center">3,670</td><td rowspan="1" colspan="1" valign="top" align="center">4,204</td><td rowspan="1" colspan="1" valign="top" align="center">0.170</td><td rowspan="1" colspan="1" valign="top" align="center">0.145</td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.10-1.32)</td><td rowspan="1" colspan="1" valign="top" align="center">6.65×10<sup>−5</sup></td><td rowspan="3" colspan="1" valign="top" align="center">0.958</td><td rowspan="3" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Korean</td><td rowspan="1" colspan="1" valign="top" align="center">2,525</td><td rowspan="1" colspan="1" valign="top" align="center">2,889</td><td rowspan="1" colspan="1" valign="top" align="center">0.215</td><td rowspan="1" colspan="1" valign="top" align="center">0.185</td><td rowspan="1" colspan="1" valign="top" align="center">1.21 (1.09-1.33)</td><td rowspan="1" colspan="1" valign="top" align="center">1.85×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Japanese</td><td rowspan="1" colspan="1" valign="top" align="center">2,378</td><td rowspan="1" colspan="1" valign="top" align="center">3,310</td><td rowspan="1" colspan="1" valign="top" align="center">0.151</td><td rowspan="1" colspan="1" valign="top" align="center">0.133</td><td rowspan="1" colspan="1" valign="top" align="center">1.23 (1.10-1.38)</td><td rowspan="1" colspan="1" valign="top" align="center">3.21×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Male</td><td rowspan="1" colspan="1" valign="top" align="center">3,542</td><td rowspan="1" colspan="1" valign="top" align="center">4,338</td><td rowspan="1" colspan="1" valign="top" align="center">0.190</td><td rowspan="1" colspan="1" valign="top" align="center">0.156</td><td rowspan="1" colspan="1" valign="top" align="center">1.28 (1.17-1.40)</td><td rowspan="1" colspan="1" valign="top" align="center">2.88×10<sup>−8</sup></td><td rowspan="2" colspan="1" valign="top" align="center">0.268</td><td rowspan="2" colspan="1" valign="top" align="center">19</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Female</td><td rowspan="1" colspan="1" valign="top" align="center">2,456</td><td rowspan="1" colspan="1" valign="top" align="center">3,175</td><td rowspan="1" colspan="1" valign="top" align="center">0.200</td><td rowspan="1" colspan="1" valign="top" align="center">0.174</td><td rowspan="1" colspan="1" valign="top" align="center">1.19 (1.07-1.32)</td><td rowspan="1" colspan="1" valign="top" align="center">8.13×10<sup>−4</sup></td></tr></tbody></table></div><div id="largeobj_idm140439242387360" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4051826/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><sup>1</sup>Data from Japan-BioBank and SCH studies were not included in the stratified analysis by sex.</div><div id="TFN6"><sup>2</sup>Effect allele (A) frequency of cases and controls.</div><div id="TFN7"><sup>3</sup>OR was estimated based on the effect allele (A).</div><div id="TFN8"><sup>4</sup><em>P</em> for heterogeneity across studies was calculated using a Cochran's <em>Q</em> test</div></div></div><p id="P14">SNP rs7229639 is located in intron 3 of the <em>SMAD7</em> gene at 18q21.1 (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a></strong>), where three other SNPs (rs4939827, rs12953717, and rs4464148) have been reported in previous GWAS conducted in European descendants to be associated with CRC risk (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138256">5</a>). In a fine-mapping analysis, rs58920878 was identified as a potential causal variant in this region (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138279">33</a>). SNP rs7229639 is not correlated with any of four previously reported risk variants in this region in East Asians with <em>r</em><sup>2</sup> all under 0.05 (data from the 1000 Genomes Project). In European descendants, rs7229639 was weakly correlated with r4939827 (<em>r</em><sup>2</sup> = 0.146) and not correlated with any of the four other SNPs (<em>r</em><sup>2</sup> &lt; 0.07). Data for these four previously reported SNPs were available in Stage 1, and three of the SNPs showed statistically significant (rs4939827) or marginally significant (rs58920878 and rs12953717) association with CRC risk in our Stage 1 (<strong><a href="/pmc/articles/PMC4051826/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a></strong>, <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a></strong>). Of these previously reported risk variants, rs4939827 showed the strongest association with CRC risk (OR =0.89, 95% CI, 0.80-0.98, <em>P</em> = 0.022) in Stage 1 of our study. Because rs4939827 is in LD with both rs58920878 and rs12953717 in Asians (<em>r</em><sup>2</sup> &gt; 0.8), we selected rs4939827 for additional genotyping in Stage 2 (<em>P</em> = 0.008) (data not shown). A combined analysis of samples from both Stages 1 and 2 yielded a per-allele OR (95% CI) of 0.90 (0.85-0.96) (<em>P</em> = 0.001) (data not shown).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p26">Evaluation of independent association of rs7229639 and other four risk variants in the <em>SMAD7</em> gene in relation to colorectal cancer risk</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140439238483104" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Test SNP (allele)</th><th rowspan="1" colspan="1" valign="top" align="left">Adjusted SNP(s)</th><th rowspan="1" colspan="1" valign="top" align="center">Cases/controls</th><th rowspan="1" colspan="1" valign="top" align="center">EAF<sup><sup>1</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="right"><em>P</em> value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs58920878 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">None</td><td rowspan="1" colspan="1" valign="top" align="center">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.734</td><td rowspan="1" colspan="1" valign="top" align="center">0.92 (0.82-1.02)</td><td rowspan="1" colspan="1" valign="top" align="right">0.097</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7229639 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">None</td><td rowspan="1" colspan="1" valign="top" align="center">4,840/5,925</td><td rowspan="1" colspan="1" valign="top" align="center">0.156</td><td rowspan="1" colspan="1" valign="top" align="center">1.25 (1.16-1.34)</td><td rowspan="1" colspan="1" valign="top" align="right">5.25×10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4939827 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">None</td><td rowspan="1" colspan="1" valign="top" align="center">4,840/5,925</td><td rowspan="1" colspan="1" valign="top" align="center">0.727</td><td rowspan="1" colspan="1" valign="top" align="center">0.90 (0.85-0.96)</td><td rowspan="1" colspan="1" valign="top" align="right">8.87×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12953717 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">None</td><td rowspan="1" colspan="1" valign="top" align="center">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.733</td><td rowspan="1" colspan="1" valign="top" align="center">0.92 (0.83-1.01)</td><td rowspan="1" colspan="1" valign="top" align="right">0.085</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4464148 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">None</td><td rowspan="1" colspan="1" valign="top" align="center">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.059</td><td rowspan="1" colspan="1" valign="top" align="center">0.97 (0.80-1.18)</td><td rowspan="1" colspan="1" valign="top" align="right">0.770</td></tr><tr><td colspan="6" rowspan="1" valign="top" align="right"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7229639 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">rs58920878</td><td rowspan="1" colspan="1" valign="top" align="center">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.152</td><td rowspan="1" colspan="1" valign="top" align="center">1.40 (1.23-1.60)</td><td rowspan="1" colspan="1" valign="top" align="right">4.19×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7229639 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">rs4939827</td><td rowspan="1" colspan="1" valign="top" align="center">4,840/5,925</td><td rowspan="1" colspan="1" valign="top" align="center">0.156</td><td rowspan="1" colspan="1" valign="top" align="center">1.30 (1.20-1.40)</td><td rowspan="1" colspan="1" valign="top" align="right">1.87×10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7229639 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">rs12953717</td><td rowspan="1" colspan="1" valign="top" align="center">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.152</td><td rowspan="1" colspan="1" valign="top" align="center">1.40 (1.23-1.60)</td><td rowspan="1" colspan="1" valign="top" align="right">3.97×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7229639 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">rs4464148</td><td rowspan="1" colspan="1" valign="top" align="center">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.152</td><td rowspan="1" colspan="1" valign="top" align="center">1.32 (1.17-1.50)</td><td rowspan="1" colspan="1" valign="top" align="right">1.25×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7229639 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">All four SNPs<sup><sup>2</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="center">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.152</td><td rowspan="1" colspan="1" valign="top" align="center">1.41 (1.23-1.61)</td><td rowspan="1" colspan="1" valign="top" align="right">8.96×10<sup>−7</sup></td></tr><tr><td colspan="6" rowspan="1" valign="top" align="right"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs58920878 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">rs7229639</td><td rowspan="1" colspan="1" valign="top" align="center">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.734</td><td rowspan="1" colspan="1" valign="top" align="center">0.84 (0.76-0.94)</td><td rowspan="1" colspan="1" valign="top" align="right">0.002</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4939827 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">rs7229639</td><td rowspan="1" colspan="1" valign="top" align="center">4,840/5,925</td><td rowspan="1" colspan="1" valign="top" align="center">0.727</td><td rowspan="1" colspan="1" valign="top" align="center">0.86 (0.81-0.92)</td><td rowspan="1" colspan="1" valign="top" align="right">2.57×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12953717 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">rs7229639</td><td rowspan="1" colspan="1" valign="top" align="center">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.733</td><td rowspan="1" colspan="1" valign="top" align="center">0.85 (0.76-0.94)</td><td rowspan="1" colspan="1" valign="top" align="right">0.002</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4464148 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">rs7229639</td><td rowspan="1" colspan="1" valign="top" align="center">1,773/2,642</td><td rowspan="1" colspan="1" valign="top" align="center">0.059</td><td rowspan="1" colspan="1" valign="top" align="center">1.02 (0.84-1.24)</td><td rowspan="1" colspan="1" valign="top" align="right">0.854</td></tr></tbody></table></div><div id="largeobj_idm140439238483104" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4051826/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p27">Abbreviations: EAF, effect allele frequency; OR, odds ratio; and CI, confidence interval.</p><div id="TFN9"><sup>1</sup>Effect allele frequency of the tested SNP in controls.</div><div id="TFN10"><sup>2</sup>rs58920878, rs4939827, rs12953717 and rs4464148.</div></div></div><p id="P15" class="p p-last">Conditional analyses were performed to determine whether the observed association with rs7229639 was independent of the other GWAS-identified SNPs in this region (<strong><a href="/pmc/articles/PMC4051826/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a></strong>). The association with rs7229639 remained statistically significant after adjusting for any of these previously GWAS-identified SNPs individually or in combination. Interestingly, adjusting for rs7229639 strengthened the associations of CRC risk with three (rs4939827, rs58920878, and rs12953717) of the four SNPs reported from previous GWAS. Because these three previously reported SNPs were strongly correlated, we selected rs4939827 for further evaluation with rs7229639 in haplotype analysis. Haplotype analysis of rs7229639 and rs4939827 revealed two common haplotypes, A-C and G-T, to be statistically significantly associated with CRC risk (<strong><a href="/pmc/articles/PMC4051826/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a></strong>). For haplotype A-C, the strength of association with CRC (OR = 1.68, 95% CI, 1.37-2.07; <em>P</em> = 9.92 × 10<sup>−7</sup>) was greater than that of either rs7229639 (OR = 1.22, 95% CI, 1.11-1.33; <em>P</em> = 3.43 × 10<sup>−5</sup>) or rs4939827 (OR = 0.90, 95% CI, 0.83-0.97; <em>P</em> = 0.007) alone. When dominant model was applied, three minor haplotypes were associated with a 1.49-fold (95% CI, 1.29-1.72) increased risk of CRC (<em>P</em> = 5.01 × 10<sup>−8</sup>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p28">Association of colorectal cancer risk with the haplotypes comprising rs7229639 and rs4939827</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140439236761280" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="2" rowspan="1" valign="top" align="center">Frequency<sup><sup>2</sup></sup><hr></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="right"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Haplotype<sup><sup>1</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="center">Cases</th><th rowspan="1" colspan="1" valign="top" align="center">Controls</th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI)<sup><sup>3</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="right"><em>P</em> value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">G-C</td><td rowspan="1" colspan="1" valign="top" align="center">0.537</td><td rowspan="1" colspan="1" valign="top" align="center">0.582</td><td rowspan="1" colspan="1" valign="top" align="center">1.00 (Reference)</td><td rowspan="1" colspan="1" valign="top" align="right"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">G-T</td><td rowspan="1" colspan="1" valign="top" align="center">0.273</td><td rowspan="1" colspan="1" valign="top" align="center">0.258</td><td rowspan="1" colspan="1" valign="top" align="center">1.41 (1.18-1.67)</td><td rowspan="1" colspan="1" valign="top" align="right">1.01×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">A-C</td><td rowspan="1" colspan="1" valign="top" align="center">0.176</td><td rowspan="1" colspan="1" valign="top" align="center">0.149</td><td rowspan="1" colspan="1" valign="top" align="center">1.68 (1.37-2.07)</td><td rowspan="1" colspan="1" valign="top" align="right">9.92×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">A-T</td><td rowspan="1" colspan="1" valign="top" align="center">0.013</td><td rowspan="1" colspan="1" valign="top" align="center">0.010</td><td rowspan="1" colspan="1" valign="top" align="center">1.33 (0.54-3.26)</td><td rowspan="1" colspan="1" valign="top" align="right">0.532</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">G-T/A-C/A-T</td><td rowspan="1" colspan="1" valign="top" align="center">0.463</td><td rowspan="1" colspan="1" valign="top" align="center">0.418</td><td rowspan="1" colspan="1" valign="top" align="center">1.49 (1.29-1.72)</td><td rowspan="1" colspan="1" valign="top" align="right">5.01×10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm140439236761280" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4051826/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN11"><sup>1</sup>Haplotypes of rs7229639 and rs4939827.</div><div id="TFN12"><sup>2</sup>Analyses were based on 3,067 cases and 3,283 controls included in Stage 2.</div><div id="TFN13"><sup>3</sup>Adjusted for age, sex and study site.</div></div></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Discussion</h2><p id="P16" class="p p-first">In this two-stage GWAS of CRC including 8,675 cases and 10,504 controls from China, Korea, Japan, and Singapore, we identified a new genetic variant (rs7229639) in the <em>SMAD7</em> gene to be associated with CRC risk at genome-wide significance level. The association of this SNP with CRC risk remained highly statistically significant after adjusting for all four other risk variants reported previously in this region (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138264">5</a>;<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138250">33</a>), providing evidence for the presence of multiple genetic risk variants in the <em>SMAD7</em> gene for CRC. SNP rs7229639 explained approximately 0.75% of the familial relative risk of CRC in East Asians, while rs4939827, the risk variant identified originally in GWAS, explained about 0.3% of familial relative risk. When haplotypes of these two variants are considered, these variants would explain 1.9% of familial relative risk, approximately six-fold the 0.3% explained by the original risk variant (rs4939827).</p><p id="P17">SNP rs7229639 (46,450,976 bp, NCBI Human Genome Build 37.3) is located in intron 3 of the <em>SMAD7</em> gene at chromosome 18q21.1. In this region, three other variants including rs4939827 (46,453,463 bp), rs12953717 (46,453,929 bp), and rs4464148 (46,459,032 bp) have been found to be associated with CRC risk through GWAS conducted in European-ancestry populations (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138282">5</a>). A subsequent fine-mapping through resequencing 17 kb region of 18q21.1 in a study conducted in European descendants identified a new variant, rs58920878 (46,449,565 bp) as a potential causal variant at this locus (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138284">33</a>). SNP rs58920878 showed the strongest association in that study and is correlated with rs4939827 (<em>r</em><sup>2</sup>=0.533), rs12953717 (<em>r</em><sup>2</sup>=0.927), and rs4464148 (<em>r</em><sup>2</sup>=0.327) in European descendants. Although rs58920878 is also in high LD with both rs4939827 (<em>r</em><sup>2</sup>=0.830) and rs12953717 (<em>r</em><sup>2</sup>=0.785) in East Asians, it was only weakly associated with CRC risk in our study. The SNP (rs7229639) we identified in this study showed the strongest association with CRC risk and is not correlated with any of the four previously reported SNPs in both East Asian and European populations. Therefore, as expected, adjustment for any or all of these four SNPs did not attenuate the association of rs7229639 with CRC risk in the current study. Haplotype analyses further confirmed the independent association of rs7229639 and rs4939827 with CRC risk in this locus. It is possible that multiple causal variants may be present in this locus among East Asians.</p><p id="P18">The <em>SMAD</em>7 gene is a key member in TGF-β family signaling pathway which has been shown to play a dual role in carcinogenesis, including tumor suppressor in early stage and oncogene in advanced stage of cancers (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138276">34</a>). Multiple genes in this pathway are known to be involved in colorectal pathogenesis (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138269">35</a>). <em>SMAD7</em> gene encodes Smad family member 7 (Smad7), an inhibitory protein that functions as an antagonist of TGF-β signaling by blocking phosphorylation of receptor-activated Smads or by competitive inhibition of complex formation of receptor-activated Smads with the common-mediator Smad4 in the cytoplasm and nucleus (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138280">36</a>). Smad7 also serves as an important cross-talk mediator of the TGF-β signaling pathway with other signaling pathways including Wnt signaling (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138263">36</a>). Expression of Smad7 was found in both normal colon mucosa and tumor cells (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138277">37</a>), and aberrant Smad7 expression may influence CRC progression (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138267">38</a>). Recently, Smad7 was shown to induce colorectal tumorigenicity through blocking TGF-beta-induced growth inhibition and inhibiting apoptosis, and a certain proportion of human colorectal tumors may become refractory to tumor suppressive actions of TGF-β that might lead to increased tumorigenicity (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138270">39</a>).</p><p id="P19">A recent study has showed that the previously-identified CRC risk variant rs4939827 may be associated with CRC survival (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138278">40</a>). This SNP was associated with certain characteristics of CRC, including invasiveness of the cancer and <em>RUNX3</em> methylation status (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382138262">41</a>). No survival data, however, were collected in our study, and thus we could not evaluate these <em>SMAD7</em> variants, including the newly-identified risk variant rs7229639, with CRC survival in our study.</p><p id="P20" class="p p-last">In summary, this study identified a new CRC risk variant in the <em>SMAD7</em> gene among East Asians, which further highlights the significant role of this gene in the etiology of CRC. To date, multiple CRC susceptibility loci have been identified by GWAS in TGFβ pathway genes, including <em>SMAD7</em>, supporting an important role of this pathway in the pathogenesis of CRC. Future studies are warranted to investigate these CRC susceptibility loci to identify causal variants underlying the associations discovered in GWAS.</p></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><p id="__p21">What's new?</p><p id="__p22">GWAS, conducted primarily in European ancestry populations, have identified common genetic variants in approximately 20 loci associated with CRC risk. These variants account for approximately 10% of familial risk of this common cancer. In a large GWAS of CRC including approximately 19,000 individuals of East Asian ancestry, SNP rs7229639 in the <em>SMAD7</em> gene was found to be associated with CRC risk at genome-wide significance level (<em>P</em>&lt;5×10<sup>−8</sup>), independent of rs4939827 reported previously in this region by an European GWAS. Findings from this study highlight the significant role of this gene in the etiology of CRC and suggest that genetic risk variants may differ by study populations.</p></div></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Material</h4><div class="sup-box half_rhythm" id="d36e1094"><a href="/pmc/articles/PMC4051826/bin/NIHMS560464-supplement-Supplementary_Material.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(785K, pdf)</sup></div></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Acknowledgements</h2><div class="sec"><p id="__p1">The content of this paper is solely the responsibility of the authors and does not necessarily represent the official views of the funding agencies. The authors wish to thank the study participants and research staff for their contributions and commitment to this project, Regina Courtney for DNA preparation, Jing He for data processing and analyses, and Mary Jo Daly for clerical support in manuscript preparation. This research was supported in part by U.S. National Institutes of Health grants R37CA070867, R01CA082729, R01CA124558, R01CA148667, and R01CA122364, as well as Ingram Professorship and Research Reward funds from the Vanderbilt University School of Medicine. Participating studies (grant support) in the consortium are as follows: Shanghai Women's Health Study (R37CA070867), Shanghai Men's Health Study (R01CA082729), Shanghai Breast and Endometrial Cancer Studies (R01CA064277 and R01CA092585, contributing only controls), Guangzhou Colorectal Cancer Study (National Key Scientific and Technological Project – 2011ZX09307-001-04; the National Basic Research Program – 2011CB504303, contributing only controls; the Natural Science Foundation of China – 81072383, contributing only controls), Aichi Colorectal Cancer Study (Grant-in-aid for Cancer Research, the Grant for the Third Term Comprehensive Control Research for Cancer and Grants-in-Aid for Scientific Research from the Japanese Ministry of Education, Culture, Sports, Science and Technology, Nos. 17015018 and 221S0001), Korea-NCC Colorectal Cancer Study (Basic Science Research Program through the National Research Foundation of Korea, 2010-0010276 and National Cancer Center Korea, 0910220), Japan-BioBank Colorectal Cancer Study (Grant from the Ministry of Education, Culture, Sports, Science and Technology of the Japanese government), KCPS-II colorectal cancer study (National R&amp;D Program for cancer control, 0920330; Seoul R&amp;D Program, 10526), Korea-Seoul Colorectal Cancer Study (None reported), and Singapore Chinese Study (NMRC/1193/2008).</p></div></div><div id="idm140439240079856" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140439240079856title">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td>CHB</td><td>Han Chinese in Beijing, China</td></tr><tr><td>CI</td><td>confidence intervals</td></tr><tr><td>CRC</td><td>colorectal cancer</td></tr><tr><td>EAF</td><td>effect allele frequency</td></tr><tr><td>GWAS</td><td>genome-wide association study</td></tr><tr><td>HERPACC-II</td><td>Hospital-based Epidemiologic Research Program at Aichi Cancer Center</td></tr><tr><td>HWE</td><td>Hardy-Weinberg equilibrium</td></tr><tr><td>JPT</td><td>Japanese in Tokyo, Japan</td></tr><tr><td>KCPS-II</td><td>Korean Cancer Prevention Study-II</td></tr><tr><td>LD</td><td>linkage disequilibrium</td></tr><tr><td>MAF</td><td>minor allele frequency</td></tr><tr><td>NCC</td><td>National Cancer Center</td></tr><tr><td>OR</td><td>odds ratio</td></tr><tr><td>PCA</td><td>principal components analysis</td></tr><tr><td>QC</td><td>quality control</td></tr><tr><td>SCH</td><td>Singapore Chinese Study</td></tr><tr><td>SMHS</td><td>Shanghai Men's Health Study</td></tr><tr><td>SNP</td><td>single-nucleotide polymorphism</td></tr><tr><td>SWHS</td><td>Shanghai Women's Health Study</td></tr></tbody></table></div></div></div><div id="idm140439242838592" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140439242838592title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P21" class="p p-first">Conflict of interest</p><p id="P22" class="p p-last">We declare that we have no conflicts of interest.</p><p></p></div></div><div id="idm140439239949184" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140439239949184title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Jemal A, Bray F, Center MM, Ferlay J, Ward E, Forman D. Global cancer statistics. <span><span class="ref-journal">CA Cancer J Clin. </span>2011;<span class="ref-vol">61</span>:69–90.</span> [<a href="/pubmed/21296855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CA+Cancer+J+Clin&amp;title=Global+cancer+statistics.&amp;author=A+Jemal&amp;author=F+Bray&amp;author=MM+Center&amp;author=J+Ferlay&amp;author=E+Ward&amp;volume=61&amp;publication_year=2011&amp;pages=69-90&amp;pmid=21296855&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Lichtenstein P, Holm NV, Verkasalo PK, Iliadou A, Kaprio J, Koskenvuo M, Pukkala E, Skytthe A, Hemminki K. Environmental and heritable factors in the causation of cancer--analyses of cohorts of twins from Sweden, Denmark, and Finland. <span><span class="ref-journal">N Engl J Med. </span>2000;<span class="ref-vol">343</span>:78–85.</span> 13. [<a href="/pubmed/10891514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Environmental+and+heritable+factors+in+the+causation+of+cancer--analyses+of+cohorts+of+twins+from+Sweden,+Denmark,+and+Finland.&amp;author=P+Lichtenstein&amp;author=NV+Holm&amp;author=PK+Verkasalo&amp;author=A+Iliadou&amp;author=J+Kaprio&amp;volume=343&amp;publication_year=2000&amp;pages=78-85&amp;pmid=10891514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Zanke BW, Greenwood CM, Rangrej J, Kustra R, Tenesa A, Farrington SM, Prendergast J, Olschwang S, Chiang T, Crowdy E, Ferretti V, Laflamme P, et al.  Genome-wide association scan identifies a colorectal cancer susceptibility locus on chromosome 8q24. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:989–94.</span> [<a href="/pubmed/17618283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+chromosome+8q24.&amp;author=BW+Zanke&amp;author=CM+Greenwood&amp;author=J+Rangrej&amp;author=R+Kustra&amp;author=A+Tenesa&amp;volume=39&amp;publication_year=2007&amp;pages=989-94&amp;pmid=17618283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Tomlinson I, Webb E, Carvajal-Carmona L, Broderick P, Kemp Z, Spain S, Penegar S, Chandler I, Gorman M, Wood W, Barclay E, Lubbe S, et al.  A genome-wide association scan of tag SNPs identifies a susceptibility variant for colorectal cancer at 8q24.21. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:984–8.</span> [<a href="/pubmed/17618284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+scan+of+tag+SNPs+identifies+a+susceptibility+variant+for+colorectal+cancer+at+8q24.21.&amp;author=I+Tomlinson&amp;author=E+Webb&amp;author=L+Carvajal-Carmona&amp;author=P+Broderick&amp;author=Z+Kemp&amp;volume=39&amp;publication_year=2007&amp;pages=984-8&amp;pmid=17618284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Broderick P, Carvajal-Carmona L, Pittman AM, Webb E, Howarth K, Rowan A, Lubbe S, Spain S, Sullivan K, Fielding S, Jaeger E, Vijayakrishnan J, et al.  A genome-wide association study shows that common alleles of SMAD7 influence colorectal cancer risk. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1315–7.</span> [<a href="/pubmed/17934461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+shows+that+common+alleles+of+SMAD7+influence+colorectal+cancer+risk.&amp;author=P+Broderick&amp;author=L+Carvajal-Carmona&amp;author=AM+Pittman&amp;author=E+Webb&amp;author=K+Howarth&amp;volume=39&amp;publication_year=2007&amp;pages=1315-7&amp;pmid=17934461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Jaeger E, Webb E, Howarth K, Carvajal-Carmona L, Rowan A, Broderick P, Walther A, Spain S, Pittman A, Kemp Z, Sullivan K, Heinimann K, et al.  Common genetic variants at the CRAC1 (HMPS) locus on chromosome 15q13.3 influence colorectal cancer risk. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:26–8.</span> [<a href="/pubmed/18084292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+genetic+variants+at+the+CRAC1+(HMPS)+locus+on+chromosome+15q13.3+influence+colorectal+cancer+risk.&amp;author=E+Jaeger&amp;author=E+Webb&amp;author=K+Howarth&amp;author=L+Carvajal-Carmona&amp;author=A+Rowan&amp;volume=40&amp;publication_year=2008&amp;pages=26-8&amp;pmid=18084292&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Tenesa A, Farrington SM, Prendergast JG, Porteous ME, Walker M, Haq N, Barnetson RA, Theodoratou E, Cetnarskyj R, Cartwright N, Semple C, Clark AJ, et al.  Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:631–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2778004/">PMC free article</a>]</span> [<a href="/pubmed/18372901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+11q23+and+replicates+risk+loci+at+8q24+and+18q21.&amp;author=A+Tenesa&amp;author=SM+Farrington&amp;author=JG+Prendergast&amp;author=ME+Porteous&amp;author=M+Walker&amp;volume=40&amp;publication_year=2008&amp;pages=631-7&amp;pmid=18372901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Tomlinson IP, Webb E, Carvajal-Carmona L, Broderick P, Howarth K, Pittman AM, Spain S, Lubbe S, Walther A, Sullivan K, Jaeger E, Fielding S, et al.  A genome-wide association study identifies colorectal cancer susceptibility loci on chromosomes 10p14 and 8q23.3. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:623–30.</span> [<a href="/pubmed/18372905" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+identifies+colorectal+cancer+susceptibility+loci+on+chromosomes+10p14+and+8q23.3.&amp;author=IP+Tomlinson&amp;author=E+Webb&amp;author=L+Carvajal-Carmona&amp;author=P+Broderick&amp;author=K+Howarth&amp;volume=40&amp;publication_year=2008&amp;pages=623-30&amp;pmid=18372905&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Houlston RS, Webb E, Broderick P, Pittman AM, Di Bernardo MC, Lubbe S, Chandler I, Vijayakrishnan J, Sullivan K, Penegar S, Carvajal-Carmona L, Howarth K, et al.  Meta-analysis of genome-wide association data identifies four new susceptibility loci for colorectal cancer. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1426–35.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2836775/">PMC free article</a>]</span> [<a href="/pubmed/19011631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+data+identifies+four+new+susceptibility+loci+for+colorectal+cancer.&amp;author=RS+Houlston&amp;author=E+Webb&amp;author=P+Broderick&amp;author=AM+Pittman&amp;author=MC+Di+Bernardo&amp;volume=40&amp;publication_year=2008&amp;pages=1426-35&amp;pmid=19011631&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Houlston RS, Cheadle J, Dobbins SE, Tenesa A, Jones AM, Howarth K, Spain SL, Broderick P, Domingo E, Farrington S, Prendergast JG, Pittman AM, et al.  Meta-analysis of three genome-wide association studies identifies susceptibility loci for colorectal cancer at 1q41, 3q26.2, 12q13.13 and 20q13.33. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:973–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5098601/">PMC free article</a>]</span> [<a href="/pubmed/20972440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+three+genome-wide+association+studies+identifies+susceptibility+loci+for+colorectal+cancer+at+1q41,+3q26.2,+12q13.13+and+20q13.33.&amp;author=RS+Houlston&amp;author=J+Cheadle&amp;author=SE+Dobbins&amp;author=A+Tenesa&amp;author=AM+Jones&amp;volume=42&amp;publication_year=2010&amp;pages=973-7&amp;pmid=20972440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Cui R, Okada Y, Jang SG, Ku JL, Park JG, Kamatani Y, Hosono N, Tsunoda T, Kumar V, Tanikawa C, Kamatani N, Yamada R, et al.  Common variant in 6q26-q27 is associated with distal colon cancer in an Asian population. <span><span class="ref-journal">Gut. </span>2011;<span class="ref-vol">60</span>:799–805.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3095478/">PMC free article</a>]</span> [<a href="/pubmed/21242260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Common+variant+in+6q26-q27+is+associated+with+distal+colon+cancer+in+an+Asian+population.&amp;author=R+Cui&amp;author=Y+Okada&amp;author=SG+Jang&amp;author=JL+Ku&amp;author=JG+Park&amp;volume=60&amp;publication_year=2011&amp;pages=799-805&amp;pmid=21242260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Dunlop MG, Dobbins SE, Farrington SM, Jones AM, Palles C, Whiffin N, Tenesa A, Spain S, Broderick P, Ooi LY, Domingo E, Smillie C, et al.  Common variation near CDKN1A, POLD3 and SHROOM2 influences colorectal cancer risk. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:770–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4747430/">PMC free article</a>]</span> [<a href="/pubmed/22634755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variation+near+CDKN1A,+POLD3+and+SHROOM2+influences+colorectal+cancer+risk.&amp;author=MG+Dunlop&amp;author=SE+Dobbins&amp;author=SM+Farrington&amp;author=AM+Jones&amp;author=C+Palles&amp;volume=44&amp;publication_year=2012&amp;pages=770-6&amp;pmid=22634755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Peters U, Jiao S, Schumacher FR, Hutter CM, Aragaki AK, Baron JA, Berndt SI, Bezieau S, Brenner H, Butterbach K, Caan BJ, Campbell PT, et al.  Identification of Genetic Susceptibility Loci for Colorectal Tumors in a Genome-Wide Meta-analysis. <span><span class="ref-journal">Gastroenterology. </span>2013;<span class="ref-vol">144</span>:799–807.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3636812/">PMC free article</a>]</span> [<a href="/pubmed/23266556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Identification+of+Genetic+Susceptibility+Loci+for+Colorectal+Tumors+in+a+Genome-Wide+Meta-analysis.&amp;author=U+Peters&amp;author=S+Jiao&amp;author=FR+Schumacher&amp;author=CM+Hutter&amp;author=AK+Aragaki&amp;volume=144&amp;publication_year=2013&amp;pages=799-807&amp;pmid=23266556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">de la Chapelle A. Genetic predisposition to colorectal cancer. <span><span class="ref-journal">Nat Rev Cancer. </span>2004;<span class="ref-vol">4</span>:769–80.</span> [<a href="/pubmed/15510158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Genetic+predisposition+to+colorectal+cancer.&amp;author=A+de+la+Chapelle&amp;volume=4&amp;publication_year=2004&amp;pages=769-80&amp;pmid=15510158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Ma X, Zhang B, Zheng W. Genetic variants associated with colorectal cancer risk: comprehensive research synopsis, meta-analysis, and epidemiological evidence. <span><span class="ref-journal">Gut. </span>2013</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4020522/">PMC free article</a>]</span> [<a href="/pubmed/23946381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Genetic+variants+associated+with+colorectal+cancer+risk:+comprehensive+research+synopsis,+meta-analysis,+and+epidemiological+evidence.&amp;author=X+Ma&amp;author=B+Zhang&amp;author=W+Zheng&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Jia WH, Zhang B, Matsuo K, Shin A, Xiang YB, Jee SH, Kim DH, Ren Z, Cai Q, Long J, Shi J, Wen W, et al.  Genome-wide association analyses in East Asians identify new susceptibility loci for colorectal cancer. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:191–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3679924/">PMC free article</a>]</span> [<a href="/pubmed/23263487" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analyses+in+East+Asians+identify+new+susceptibility+loci+for+colorectal+cancer.&amp;author=WH+Jia&amp;author=B+Zhang&amp;author=K+Matsuo&amp;author=A+Shin&amp;author=YB+Xiang&amp;volume=45&amp;publication_year=2013&amp;pages=191-6&amp;pmid=23263487&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Zheng W, Long J, Gao YT, Li C, Zheng Y, Xiang YB, Wen W, Levy S, Deming SL, Haines JL, Gu K, Fair AM, et al.  Genome-wide association study identifies a new breast cancer susceptibility locus at 6q25.1. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:324–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2754845/">PMC free article</a>]</span> [<a href="/pubmed/19219042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+a+new+breast+cancer+susceptibility+locus+at+6q25.1.&amp;author=W+Zheng&amp;author=J+Long&amp;author=YT+Gao&amp;author=C+Li&amp;author=Y+Zheng&amp;volume=41&amp;publication_year=2009&amp;pages=324-8&amp;pmid=19219042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Bei JX, Li Y, Jia WH, Feng BJ, Zhou G, Chen LZ, Feng QS, Low HQ, Zhang H, He F, Tai ES, Kang T, et al.  A genome-wide association study of nasopharyngeal carcinoma identifies three new susceptibility loci. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:599–603.</span> [<a href="/pubmed/20512145" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+of+nasopharyngeal+carcinoma+identifies+three+new+susceptibility+loci.&amp;author=JX+Bei&amp;author=Y+Li&amp;author=WH+Jia&amp;author=BJ+Feng&amp;author=G+Zhou&amp;volume=42&amp;publication_year=2010&amp;pages=599-603&amp;pmid=20512145&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Nakata I, Yamashiro K, Yamada R, Gotoh N, Nakanishi H, Hayashi H, Tsujikawa A, Otani A, Saito M, Iida T, Oishi A, Matsuo K, et al.  Association between the SERPING1 gene and age-related macular degeneration and polypoidal choroidal vasculopathy in Japanese. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>:e19108.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3079757/">PMC free article</a>]</span> [<a href="/pubmed/21526158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Association+between+the+SERPING1+gene+and+age-related+macular+degeneration+and+polypoidal+choroidal+vasculopathy+in+Japanese.&amp;author=I+Nakata&amp;author=K+Yamashiro&amp;author=R+Yamada&amp;author=N+Gotoh&amp;author=H+Nakanishi&amp;volume=6&amp;publication_year=2011&amp;pages=e19108&amp;pmid=21526158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Jee SH, Sull JW, Lee JE, Shin C, Park J, Kimm H, Cho EY, Shin ES, Yun JE, Park JW, Kim SY, Lee SJ, et al.  Adiponectin concentrations: a genome-wide association study. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>:545–52.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2948810/">PMC free article</a>]</span> [<a href="/pubmed/20887962" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Adiponectin+concentrations:+a+genome-wide+association+study.&amp;author=SH+Jee&amp;author=JW+Sull&amp;author=JE+Lee&amp;author=C+Shin&amp;author=J+Park&amp;volume=87&amp;publication_year=2010&amp;pages=545-52&amp;pmid=20887962&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Thean LF, Li HH, Teo YY, Koh WP, Yuan JM, Teoh ML, Koh PK, Tang CL, Cheah PY. Association of caucasian-identified variants with colorectal cancer risk in singapore chinese. <span><span class="ref-journal">PLoS One. </span>2012;<span class="ref-vol">7</span>:e42407.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3411754/">PMC free article</a>]</span> [<a href="/pubmed/22879968" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Association+of+caucasian-identified+variants+with+colorectal+cancer+risk+in+singapore+chinese.&amp;author=LF+Thean&amp;author=HH+Li&amp;author=YY+Teo&amp;author=WP+Koh&amp;author=JM+Yuan&amp;volume=7&amp;publication_year=2012&amp;pages=e42407&amp;pmid=22879968&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2010;<span class="ref-vol">34</span>:816–34.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes.&amp;author=Y+Li&amp;author=CJ+Willer&amp;author=J+Ding&amp;author=P+Scheet&amp;author=GR+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-34&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–9.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies.&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses.&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Zheng J, Li Y, Abecasis GR, Scheet P. A comparison of approaches to account for uncertainty in analysis of imputed genotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>:102–10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3143715/">PMC free article</a>]</span> [<a href="/pubmed/21254217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=A+comparison+of+approaches+to+account+for+uncertainty+in+analysis+of+imputed+genotypes.&amp;author=J+Zheng&amp;author=Y+Li&amp;author=GR+Abecasis&amp;author=P+Scheet&amp;volume=35&amp;publication_year=2011&amp;pages=102-10&amp;pmid=21254217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Lau J, Ioannidis JP, Schmid CH. Quantitative synthesis in systematic reviews. <span><span class="ref-journal">Ann Intern Med. </span>1997;<span class="ref-vol">127</span>:820–6.</span> [<a href="/pubmed/9382404" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Intern+Med&amp;title=Quantitative+synthesis+in+systematic+reviews.&amp;author=J+Lau&amp;author=JP+Ioannidis&amp;author=CH+Schmid&amp;volume=127&amp;publication_year=1997&amp;pages=820-6&amp;pmid=9382404&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Higgins JP, Thompson SG. Quantifying heterogeneity in a meta-analysis. <span><span class="ref-journal">Stat Med. </span>2002;<span class="ref-vol">21</span>:1539–58.</span> [<a href="/pubmed/12111919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=Quantifying+heterogeneity+in+a+meta-analysis.&amp;author=JP+Higgins&amp;author=SG+Thompson&amp;volume=21&amp;publication_year=2002&amp;pages=1539-58&amp;pmid=12111919&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans.&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-1&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Zheng W, Zhang B, Cai Q, Sung H, Michailidou K, Shi J, Choi JY, Long J, Dennis J, Humphreys MK, Wang Q, Lu W, et al.  Common genetic determinants of breast-cancer risk in East Asian women: a collaborative study of 23 637 breast cancer cases and 25 579 controls. <span><span class="ref-journal">Hum Mol Genet. </span>2013;<span class="ref-vol">22</span>:2539–50.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3658167/">PMC free article</a>]</span> [<a href="/pubmed/23535825" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Common+genetic+determinants+of+breast-cancer+risk+in+East+Asian+women:+a+collaborative+study+of+23+637+breast+cancer+cases+and+25+579+controls.&amp;author=W+Zheng&amp;author=B+Zhang&amp;author=Q+Cai&amp;author=H+Sung&amp;author=K+Michailidou&amp;volume=22&amp;publication_year=2013&amp;pages=2539-50&amp;pmid=23535825&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Johns LE, Houlston RS. A systematic review and meta-analysis of familial colorectal cancer risk. <span><span class="ref-journal">Am J Gastroenterol. </span><span class="ref-vol">200196</span>:2992–3003.</span> [<a href="/pubmed/11693338" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Gastroenterol&amp;title=A+systematic+review+and+meta-analysis+of+familial+colorectal+cancer+risk.&amp;author=LE+Johns&amp;author=RS+Houlston&amp;volume=200196&amp;pages=2992-3003&amp;pmid=11693338&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, Boehnke M, Abecasis GR, Willer CJ. LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results.&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-7&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–5.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps.&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-5&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Pittman AM, Naranjo S, Webb E, Broderick P, Lips EH, van WT, Morreau H, Sullivan K, Fielding S, Twiss P, Vijayakrishnan J, Casares F, et al.  The colorectal cancer risk at 18q21 is caused by a novel variant altering SMAD7 expression. <span><span class="ref-journal">Genome Res. </span>2009;<span class="ref-vol">19</span>:987–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2694486/">PMC free article</a>]</span> [<a href="/pubmed/19395656" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=The+colorectal+cancer+risk+at+18q21+is+caused+by+a+novel+variant+altering+SMAD7+expression.&amp;author=AM+Pittman&amp;author=S+Naranjo&amp;author=E+Webb&amp;author=P+Broderick&amp;author=EH+Lips&amp;volume=19&amp;publication_year=2009&amp;pages=987-93&amp;pmid=19395656&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Blobe GC, Schiemann WP, Lodish HF. Role of transforming growth factor beta in human disease. <span><span class="ref-journal">N Engl J Med. </span>2000;<span class="ref-vol">342</span>:1350–8.</span> [<a href="/pubmed/10793168" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Role+of+transforming+growth+factor+beta+in+human+disease.&amp;author=GC+Blobe&amp;author=WP+Schiemann&amp;author=HF+Lodish&amp;volume=342&amp;publication_year=2000&amp;pages=1350-8&amp;pmid=10793168&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Markowitz SD, Bertagnolli MM. Molecular origins of cancer: Molecular basis of colorectal cancer. <span><span class="ref-journal">N Engl J Med. </span>2009;<span class="ref-vol">361</span>:2449–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2843693/">PMC free article</a>]</span> [<a href="/pubmed/20018966" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Molecular+origins+of+cancer:+Molecular+basis+of+colorectal+cancer.&amp;author=SD+Markowitz&amp;author=MM+Bertagnolli&amp;volume=361&amp;publication_year=2009&amp;pages=2449-60&amp;pmid=20018966&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Yan X, Chen YG. Smad7: not only a regulator, but also a cross-talk mediator of TGF-beta signalling. <span><span class="ref-journal">Biochem J. </span>2011;<span class="ref-vol">434</span>:1–10.</span> [<a href="/pubmed/21269274" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=Smad7:+not+only+a+regulator,+but+also+a+cross-talk+mediator+of+TGF-beta+signalling.&amp;author=X+Yan&amp;author=YG+Chen&amp;volume=434&amp;publication_year=2011&amp;pages=1-10&amp;pmid=21269274&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Korchynskyi O, Landstrom M, Stoika R, Funa K, Heldin CH, ten DP, Souchelnytskyi S. Expression of Smad proteins in human colorectal cancer. <span><span class="ref-journal">Int J Cancer. </span>1999;<span class="ref-vol">82</span>:197–202.</span> [<a href="/pubmed/10389752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Cancer&amp;title=Expression+of+Smad+proteins+in+human+colorectal+cancer.&amp;author=O+Korchynskyi&amp;author=M+Landstrom&amp;author=R+Stoika&amp;author=K+Funa&amp;author=CH+Heldin&amp;volume=82&amp;publication_year=1999&amp;pages=197-202&amp;pmid=10389752&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Levy L, Hill CS. Alterations in components of the TGF-beta superfamily signaling pathways in human cancer. <span><span class="ref-journal">Cytokine Growth Factor Rev. </span>2006;<span class="ref-vol">17</span>:41–58.</span> [<a href="/pubmed/16310402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine+Growth+Factor+Rev&amp;title=Alterations+in+components+of+the+TGF-beta+superfamily+signaling+pathways+in+human+cancer.&amp;author=L+Levy&amp;author=CS+Hill&amp;volume=17&amp;publication_year=2006&amp;pages=41-58&amp;pmid=16310402&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Halder SK, Beauchamp RD, Datta PK. Smad7 induces tumorigenicity by blocking TGF-beta-induced growth inhibition and apoptosis. <span><span class="ref-journal">Exp Cell Res. </span>2005;<span class="ref-vol">307</span>:231–46.</span> [<a href="/pubmed/15922743" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp+Cell+Res&amp;title=Smad7+induces+tumorigenicity+by+blocking+TGF-beta-induced+growth+inhibition+and+apoptosis.&amp;author=SK+Halder&amp;author=RD+Beauchamp&amp;author=PK+Datta&amp;volume=307&amp;publication_year=2005&amp;pages=231-46&amp;pmid=15922743&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Phipps AI, Newcomb PA, Garcia-Albeniz X, Hutter CM, White E, Fuchs CS, Hazra A, Ogino S, Nan H, Ma J, Campbell PT, Figueiredo JC, et al.  Association between colorectal cancer susceptibility Loci and survival time after diagnosis with colorectal cancer. <span><span class="ref-journal">Gastroenterology. </span>2012;<span class="ref-vol">143</span>:51–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3579620/">PMC free article</a>]</span> [<a href="/pubmed/22580541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Association+between+colorectal+cancer+susceptibility+Loci+and+survival+time+after+diagnosis+with+colorectal+cancer.&amp;author=AI+Phipps&amp;author=PA+Newcomb&amp;author=X+Garcia-Albeniz&amp;author=CM+Hutter&amp;author=E+White&amp;volume=143&amp;publication_year=2012&amp;pages=51-4&amp;pmid=22580541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Garcia-Albeniz X, Nan H, Valeri L, Morikawa T, Kuchiba A, Phipps AI, Hutter CM, Peters U, Newcomb PA, Fuchs CS, Giovannucci EL, Ogino S, et al.  Phenotypic and tumor molecular characterization of colorectal cancer in relation to a susceptibility SMAD7 variant associated with survival. <span><span class="ref-journal">Carcinogenesis. </span>2013;<span class="ref-vol">34</span>:292–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3564438/">PMC free article</a>]</span> [<a href="/pubmed/23104301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Phenotypic+and+tumor+molecular+characterization+of+colorectal+cancer+in+relation+to+a+susceptibility+SMAD7+variant+associated+with+survival.&amp;author=X+Garcia-Albeniz&amp;author=H+Nan&amp;author=L+Valeri&amp;author=T+Morikawa&amp;author=A+Kuchiba&amp;volume=34&amp;publication_year=2013&amp;pages=292-8&amp;pmid=23104301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4051826&amp;issue-id=238798&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140439242626064title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials and Methods</a></li><li><a href="#S6title">Results</a></li><li><a href="#S7title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S8title">Acknowledgements</a></li><li><a href="#idm140439240079856title">Abbreviations</a></li><li><a href="#idm140439242838592title">Footnotes</a></li><li><a href="#idm140439239949184title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4051826/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4051826/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4051826/pdf/nihms-560464.pdf">PDF (750K)</a></li> | <li><a href="#" data-citationid="PMC4051826" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4051826%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4051826%2F&amp;text=Genome-wide%20association%20study%20identifies%20a%20new%20SMAD7%20risk%20variant%20associated%20with%20colorectal%20cancer%20risk%20in%20East%20Asians" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4051826%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24448986" data-aiid="4051826" data-aid="4051826" data-iid="238798" data-domainid="319" data-domain="nihpa" data-accid="PMC4051826" data-md5="c49dea8d34cd5e987d41532f255bf20c"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T11:46:33-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C5FD1DD5684A10000000004520289&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4051826%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4051826/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>