<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="pediatrics"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4811310/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-Wide Association and Exome Sequencing Study of Language Disorder in an Isolated Population</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4811310/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Pediatrics"><meta name="citation_title" content="Genome-Wide Association and Exome Sequencing Study of Language Disorder in an Isolated Population"><meta name="citation_authors" content="Sergey A. Kornilov, Natalia Rakhlin, Roman Koposov, Maria Lee, Carolyn Yrigollen, Ahmet Okay Caglayan, James S. Magnuson, Shrikant Mane, Joseph T. Chang, Elena L. Grigorenko"><meta name="citation_date" content="April 2016"><meta name="citation_issue" content="4"><meta name="citation_volume" content="137"><meta name="citation_doi" content="10.1542/peds.2015-2469"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4811310/?report=abstract"><meta name="citation_pmid" content="27016271"><meta name="DC.Title" content="Genome-Wide Association and Exome Sequencing Study of Language Disorder in an Isolated Population"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Academy of Pediatrics"><meta name="DC.Contributor" content="Sergey A. Kornilov"><meta name="DC.Contributor" content="Natalia Rakhlin"><meta name="DC.Contributor" content="Roman Koposov"><meta name="DC.Contributor" content="Maria Lee"><meta name="DC.Contributor" content="Carolyn Yrigollen"><meta name="DC.Contributor" content="Ahmet Okay Caglayan"><meta name="DC.Contributor" content="James S. Magnuson"><meta name="DC.Contributor" content="Shrikant Mane"><meta name="DC.Contributor" content="Joseph T. Chang"><meta name="DC.Contributor" content="Elena L. Grigorenko"><meta name="DC.Date" content="2016 Apr"><meta name="DC.Identifier" content="10.1542/peds.2015-2469"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-Wide Association and Exome Sequencing Study of Language Disorder in an Isolated Population"><meta property="og:type" content="article"><meta property="og:description" content="Developmental language disorder (DLD) is a highly prevalent neurodevelopmental disorder associated with negative outcomes in different domains; the etiology of DLD is unknown. To investigate the genetic underpinnings of DLD, we performed genome-wide association ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4811310/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-pediatrics.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4811310/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4811310/pdf/PEDS_20152469.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D8159DD58FE010000000003690241.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4811310%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Pediatrics%22[journal]" class="navlink">Pediatrics</a></li><li class="accid">PMC4811310</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-pediatrics.gif" alt="Logo of pediatrics" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.pediatrics.org" target="pmc_ext" ref="reftype=publisher&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Pediatrics</a></span>. 2016 Apr; 137(4): e20152469. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1542%2Fpeds.2015-2469" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1542/peds.2015-2469</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4811310</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27016271">27016271</a></div></div></div></div><h1 class="content-title">Genome-Wide Association and Exome Sequencing Study of Language Disorder in an Isolated Population</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Kornilov%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271" class="affpopup" co-rid="_co_idm140598282461088" co-class="co-affbox">Sergey A. Kornilov</a>, PhD,<sup>a,</sup><sup>b,</sup><sup>c,</sup><sup>d,</sup><sup>e</sup> <a href="/pubmed/?term=Rakhlin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271" class="affpopup" co-rid="_co_idm140598323467664" co-class="co-affbox">Natalia Rakhlin</a>, PhD,<sup>a,</sup><sup>f</sup> <a href="/pubmed/?term=Koposov%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271" class="affpopup" co-rid="_co_idm140598281238224" co-class="co-affbox">Roman Koposov</a>, MD, PhD,<sup>g</sup> <a href="/pubmed/?term=Lee%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271" class="affpopup" co-rid="_co_idm140598357719872" co-class="co-affbox">Maria Lee</a>, BSc,<sup>a</sup> <a href="/pubmed/?term=Yrigollen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271" class="affpopup" co-rid="_co_idm140598357717632" co-class="co-affbox">Carolyn Yrigollen</a>, PhD,<sup>h</sup> <a href="/pubmed/?term=Caglayan%20AO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271" class="affpopup" co-rid="_co_idm140598286453312" co-class="co-affbox">Ahmet Okay Caglayan</a>, MD,<sup>a,</sup><sup>i</sup> <a href="/pubmed/?term=Magnuson%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271" class="affpopup" co-rid="_co_idm140598278137328" co-class="co-affbox">James S. Magnuson</a>, PhD,<sup>b,</sup><sup>c</sup> <a href="/pubmed/?term=Mane%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271" class="affpopup" co-rid="_co_idm140598278134352" co-class="co-affbox">Shrikant Mane</a>, PhD,<sup>a</sup> <a href="/pubmed/?term=Chang%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271" class="affpopup" co-rid="_co_idm140598277792992" co-class="co-affbox">Joseph T. Chang</a>, PhD,<sup>a</sup> and  <a href="/pubmed/?term=Grigorenko%20EL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271" class="affpopup" co-rid="_co_idm140598284483664" co-class="co-affbox">Elena L. Grigorenko</a>, PhD<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>a,</sup><sup>c,</sup><sup>e,</sup><sup>j</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140598282461088"><h3 class="no_margin">Sergey A. Kornilov</h3><p><sup>a</sup>Child Study Center, School of Medicine, Yale University, New Haven, Connecticut;</p><p><sup>b</sup>Department of Psychology, University of Connecticut, Storrs, Connecticut;</p><p><sup>c</sup>Haskins Laboratories, New Haven, Connecticut;</p><p><sup>d</sup>Department of Psychology, Moscow State University, Moscow, Russia;</p><p><sup>e</sup>Department of Psychology, Saint Petersburg State University, Saint Petersburg, Russia;</p><div>Find articles by <a href="/pubmed/?term=Kornilov%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271">Sergey A. Kornilov</a></div></div><div id="_co_idm140598323467664"><h3 class="no_margin">Natalia Rakhlin</h3><p><sup>a</sup>Child Study Center, School of Medicine, Yale University, New Haven, Connecticut;</p><p><sup>f</sup>Department of Communication Sciences and Disorders, Wayne State University, Detroit, Michigan;</p><div>Find articles by <a href="/pubmed/?term=Rakhlin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271">Natalia Rakhlin</a></div></div><div id="_co_idm140598281238224"><h3 class="no_margin">Roman Koposov</h3><p><sup>g</sup>Regional Centre for Child and Youth Mental Health and Child Welfare, UiT The Arctic University of Norway, Tromsø, Norway;</p><div>Find articles by <a href="/pubmed/?term=Koposov%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271">Roman Koposov</a></div></div><div id="_co_idm140598357719872"><h3 class="no_margin">Maria Lee</h3><p><sup>a</sup>Child Study Center, School of Medicine, Yale University, New Haven, Connecticut;</p><div>Find articles by <a href="/pubmed/?term=Lee%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271">Maria Lee</a></div></div><div id="_co_idm140598357717632"><h3 class="no_margin">Carolyn Yrigollen</h3><p><sup>h</sup>The Children's Hospital of Philadelphia, University of Pennsylvania, Philadelphia, Pennsylvania;</p><div>Find articles by <a href="/pubmed/?term=Yrigollen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271">Carolyn Yrigollen</a></div></div><div id="_co_idm140598286453312"><h3 class="no_margin">Ahmet Okay Caglayan</h3><p><sup>a</sup>Child Study Center, School of Medicine, Yale University, New Haven, Connecticut;</p><p><sup>i</sup>Department of Medical Genetics, Istanbul Bilim University, Istanbul, Turkey; and</p><div>Find articles by <a href="/pubmed/?term=Caglayan%20AO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271">Ahmet Okay Caglayan</a></div></div><div id="_co_idm140598278137328"><h3 class="no_margin">James S. Magnuson</h3><p><sup>b</sup>Department of Psychology, University of Connecticut, Storrs, Connecticut;</p><p><sup>c</sup>Haskins Laboratories, New Haven, Connecticut;</p><div>Find articles by <a href="/pubmed/?term=Magnuson%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271">James S. Magnuson</a></div></div><div id="_co_idm140598278134352"><h3 class="no_margin">Shrikant Mane</h3><p><sup>a</sup>Child Study Center, School of Medicine, Yale University, New Haven, Connecticut;</p><div>Find articles by <a href="/pubmed/?term=Mane%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271">Shrikant Mane</a></div></div><div id="_co_idm140598277792992"><h3 class="no_margin">Joseph T. Chang</h3><p><sup>a</sup>Child Study Center, School of Medicine, Yale University, New Haven, Connecticut;</p><div>Find articles by <a href="/pubmed/?term=Chang%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271">Joseph T. Chang</a></div></div><div id="_co_idm140598284483664"><h3 class="no_margin">Elena L. Grigorenko</h3><p><sup>a</sup>Child Study Center, School of Medicine, Yale University, New Haven, Connecticut;</p><p><sup>c</sup>Haskins Laboratories, New Haven, Connecticut;</p><p><sup>e</sup>Department of Psychology, Saint Petersburg State University, Saint Petersburg, Russia;</p><p><sup>j</sup>Moscow State University for Psychology and Education, Moscow, Russia</p><div>Find articles by <a href="/pubmed/?term=Grigorenko%20EL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27016271">Elena L. Grigorenko</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140598321447680_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140598321447680_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140598321447680_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140598321447680_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>a</sup>Child Study Center, School of Medicine, Yale University, New Haven, Connecticut;</div><div class="fm-affl" id="aff2" lang="en"><sup>b</sup>Department of Psychology, University of Connecticut, Storrs, Connecticut;</div><div class="fm-affl" id="aff3" lang="en"><sup>c</sup>Haskins Laboratories, New Haven, Connecticut;</div><div class="fm-affl" id="aff4" lang="en"><sup>d</sup>Department of Psychology, Moscow State University, Moscow, Russia;</div><div class="fm-affl" id="aff5" lang="en"><sup>e</sup>Department of Psychology, Saint Petersburg State University, Saint Petersburg, Russia;</div><div class="fm-affl" id="aff6" lang="en"><sup>f</sup>Department of Communication Sciences and Disorders, Wayne State University, Detroit, Michigan;</div><div class="fm-affl" id="aff7" lang="en"><sup>g</sup>Regional Centre for Child and Youth Mental Health and Child Welfare, UiT The Arctic University of Norway, Tromsø, Norway;</div><div class="fm-affl" id="aff8" lang="en"><sup>h</sup>The Children's Hospital of Philadelphia, University of Pennsylvania, Philadelphia, Pennsylvania;</div><div class="fm-affl" id="aff9" lang="en"><sup>i</sup>Department of Medical Genetics, Istanbul Bilim University, Istanbul, Turkey; and</div><div class="fm-affl" id="aff10" lang="en"><sup>j</sup>Moscow State University for Psychology and Education, Moscow, Russia</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="cor11">Address correspondence to Elena L. Grigorenko, PhD, Child Study Center, Yale School of Medicine, 230 South Frontage Rd, New Haven, CT 06519. E-mail: <a href="mailto:dev@null" data-email="ude.elay@oknerogirg.anele" class="oemail">ude.elay@oknerogirg.anele</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140598321447680_an" style="display:none"><div class="fm-pubdate half_rhythm">Accepted 2016 Jan 26.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140598321447680_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2016 by the American Academy of Pediatrics</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4811310/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140598280115248" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140598280115248title">Abstract</h2><!--article-meta--><div><div id="s01" class="sec sec-first"><h3 id="s01title">BACKGROUND AND OBJECTIVE:</h3><p id="__p1" class="p p-first-last">Developmental language disorder (DLD) is a highly prevalent neurodevelopmental disorder associated with negative outcomes in different domains; the etiology of DLD is unknown. To investigate the genetic underpinnings of DLD, we performed genome-wide association and whole exome sequencing studies in a geographically isolated population with a substantially elevated prevalence of the disorder (ie, the AZ sample).</p></div><div id="s02" class="sec"><h3 id="s02title">METHODS:</h3><p id="__p2" class="p p-first-last">DNA samples were collected from 359 individuals for the genome-wide association study and from 12 severely affected individuals for whole exome sequencing. Multifaceted phenotypes, representing major domains of expressive language functioning, were derived from collected speech samples.</p></div><div id="s03" class="sec"><h3 id="s03title">RESULTS:</h3><p id="__p3" class="p p-first-last">Gene-based analyses revealed a significant association between <em>SETBP1</em> and complexity of linguistic output (<em>P</em> = 5.47 × 10<sup>−7</sup>). The analysis of exome variants revealed coding sequence variants in 14 genes, most of which play a role in neural development. Targeted enrichment analysis implicated myocyte enhancer factor–2 (MEF2)-regulated genes in DLD in the AZ population. The main findings were successfully replicated in an independent cohort of children at risk for related disorders (<em>n</em> = 372).</p></div><div id="s04" class="sec sec-last"><h3 id="s04title">CONCLUSIONS:</h3><p id="__p4" class="p p-first-last">MEF2-regulated pathways were identified as potential candidate pathways in the etiology of DLD. Several genes (including the candidate <em>SETBP1</em> and other MEF2-related genes) seem to jointly influence certain, but not all, facets of the DLD phenotype. Even when genetic and environmental diversity is reduced, DLD is best conceptualized as etiologically complex. Future research should establish whether the signals detected in the AZ population can be replicated in other samples and languages and provide further characterization of the identified pathway.</p></div></div></div><div id="idm140598288453168" class="tsec sec headless whole_rhythm"><div class="boxed-text-box whole_rhythm hide-overflow" id="b1"><div class="sec" id="s05"><h3 id="s05title">What’s Known on This Subject:</h3><p id="__p5" class="p p-first-last">Genetic underpinnings of common forms of pediatric disorders of language are heavily understudied. Recent association studies identified several tentative candidate genes. However, thus far, none of these candidates has received strong support in replication or confirmation analyses.</p></div></div><div class="boxed-text-box whole_rhythm hide-overflow" id="b2"><div class="sec" id="s06"><h3 id="s06title">What This Study Adds:</h3><p id="__p6" class="p p-first-last">We established a statistically significant association between <em>SETBP1</em> and language disorders in a geographically isolated population. Whole exome sequencing convergently implicated the myocyte enhancer factor–2–regulated pathways (of which <em>SETBP1</em> is part) in language disorders in this special population.</p></div></div><p id="__p7" class="p p-first">Developmental language disorder (DLD) is a prevalent neurodevelopmental disorder, with 7-10% of children<sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561715">1</a></sup><sup>,</sup><sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_588514858">2</a></sup> exhibiting atypical patterns of language development despite not having apparent sensorimotor/cognitive impairments or other medical conditions.<sup><a href="#B3" rid="B3" class=" bibr popnode">3</a></sup> DLD is lifelong,<sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561696">4</a></sup> comorbid with other neurodevelopmental<sup><a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196909">5</a></sup> and psychiatric<sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196906">6</a></sup> disorders, and associated with adverse academic<sup><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561697">7</a></sup> and socio-emotional<sup><a href="#B8" rid="B8" class=" bibr popnode">8</a></sup> outcomes. It is phenotypically complex and genetically heterogeneous; although highly heritable,<sup><a href="#B9" rid="B9" class=" bibr popnode">9</a></sup> the etiology and pathogenesis of DLD are poorly understood.</p><p id="__p8">A rare Mendelian type of DLD has been attributed to deleterious variants in the <em>FOXP2</em> gene<sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561695">10</a></sup><sup>–</sup><sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561700">12</a></sup> (7q31); however, it is not associated with the disorder’s common forms.<sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561687">13</a></sup> For the latter, linkage studies have identified 3 susceptibility regions: 16q24, 19q13,<sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561709">14</a></sup> and 13q2.<sup><a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561712">15</a></sup> Targeted association studies implicated <em>CNTNAP2</em><sup><a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561711">16</a></sup> (7q35; downregulated by <em>FOXP2</em>) and <em>CMIP</em> and <em>ATP2C2</em><sup><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561706">17</a></sup> (16q) genes in phonological memory deficits. Four genome-wide association studies (GWAS) divulged no genome-wide significant signals,<sup><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561704">18</a></sup><sup>–</sup><sup><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561714">21</a></sup> with the exception of gene-based associations for <em>CDC2L1, CDC2L2, LOC728661</em>, and <em>RCAN3</em>.<sup><a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561684">19</a></sup> A whole exome sequencing (WES) study of DLD in an admixed Chilean founder population suggested the involvement of a nonsynonymous single nucleotide variant (SNV) in <em>NFXL1</em><sup><a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196912">22</a></sup>; however, its location is not in the previously identified linkage regions in this population.<sup><a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196915">23</a></sup></p><p id="__p9">This pattern of findings highlights the complexity of DLD’s etiology, driven by the exclusionary nature of the diagnosis, the multicomponential nature of the phenotype, and the heterogeneity of the samples studied. The main objective of the present study was to identify genetic bases of DLD in a unique population (small, geographically secluded, and with an elevated prevalence of DLD [hereafter, the AZ population]) in which genetic and environmental variability is constrained. Genetic profiles of isolated populations are characterized by restricted genetic and allelic heterogeneity, thus rendering them ideal for studying the genetic bases of complex disorders.<sup><a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561688">24</a></sup></p><p id="__p10" class="p">The study population resides in a remote cluster of villages in Russia’s rural north; it was founded in the 15th century by several nuclear families. Currently, the AZ population comprises ∼860 individuals (∼120 children aged 3–18 years). Of these, 74.6% are represented by a set of multigenerational family structures (6391 individuals), of whom 82% are interconnected through a single 11-generational pedigree. The environmental conditions in the population are relatively uniform: all children go to the same kindergarten and school, and the socioeconomic indicators such as parental education and income show little variation. The AZ population is relatively geographically isolated and is characterized by an atypically high prevalence of DLD<sup><a href="#B25" rid="B25" class=" bibr popnode">25</a></sup> (ie, ∼30% compared with 9% in the control rural population). This finding suggests the presence of a shared genetic component, potentially attributable to the founder effect(s).</p></div><div id="s07" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s07title">Methods</h2><div id="s08" class="sec sec-first"><h3 id="s08title">Population and Sample</h3><p id="__p11" class="p p-first-last">Altogether, 474 AZ individuals donated DNA. When considered in combination with first-degree relatives, 405 of these donors represented 79 nuclear and extended pedigrees (<em>N</em> = 1152; range, 3–474; median, 6). Of these, 359 underwent phenotyping and constituted the GWAS sample: 124 children (62 male subjects; age, 5.33–17.92 years) and 235 adults (102 male subjects; age, 18.83–83.42 years). A total of 149 were classified as affected (DLD) and 210 as typically developing individuals (<a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Information</a>).</p></div><div id="s09" class="sec"><h3 id="s09title">Phenotyping</h3><p id="__p12" class="p p-first-last">Phenotyping was performed by clinical linguists using elicited semi-structured speech samples. These samples were scored by using previously described phenotyping procedures<sup><a href="#B25" rid="B25" class=" bibr popnode">25</a></sup> to produce 5 quantitative phenotypes representing the major facets of DLD: phonetic/prosodic characteristics (eg, phonological omissions, misarticulations); well-formedness (rate of grammatical/lexical errors); complex structures (frequency of complex syntactic structures); mean length of utterance in words; and semantic/pragmatic errors (rate of errors in sentence meaning). Age-adjusted <em>z</em> scores were computed by using data from healthy control subjects from the comparison population to determine impairment status (ie, a <em>z</em> score below –1). Individuals were classified as overall DLD if they met the impairment criterion for ≥2 facets. Principal component analysis revealed that the 5 phenotypes formed 2 independent components: linguistic errors (phonetic/prosodic characteristics, well-formedness, and semantic/pragmatic errors) and syntactic complexity (complex structures and mean length of utterance in words).</p></div><div id="s10" class="sec"><h3 id="s10title">Single Nucleotide Polymorphism Genotyping</h3><p id="__p13" class="p p-first">The DNA extracted from peripheral blood (<em>n</em> = 384) or saliva/buccal swabs (<em>n</em> = 21) underwent quality control (QC) assessment for purity and degradation after standard collection, storage, and extraction procedures recommended by the manufacturers (Qiagen N.V. [Hilden, Germany] and DNA Genotek, Inc [Ottawa, ON, Canada]), and prepared at a concentration of 50 ng/µL.</p><p id="__p14">Samples were genotyped at the Yale Center for Genome Analysis using HumanCNV 370k-Duo (<em>n</em> = 315) or 610k-Quad (<em>n</em> = 90) BeadChips (Illumina, Inc, San Diego, CA). Language status and gender distributions across the plates were not statistically different from random. Allele calling was performed by using the GenCall algorithm in GenomeStudio version 2011.1.</p><p id="__p15" class="p p-last">Samples and markers underwent QC review with GenomeStudio and SNP &amp; Variation Suite (SVS) version 7.7.8 (GoldenHelix, Inc, Bozeman, MT). Samples with call rates &gt;95% and verified gender were retained. A total of 223 580 autosomal single nucleotide polymorphisms (SNPs) common to 2 genotyping platforms were retained after QC so that the GenCall score was &gt;0.30, the call rate was &gt;95%, and minor allele frequency was &gt;1%.</p></div><div id="s11" class="sec"><h3 id="s11title">Whole Exome DNA Sequencing</h3><p id="__p16" class="p p-first">Four subpedigrees were chosen for WES based on the results of complex segregation analysis<sup><a href="#B26" rid="B26" class=" bibr popnode">26</a></sup> that suggested possible Mendelian transmission. From these subpedigrees, 12 severely affected individuals were selected. Nine control non-AZ individuals without DLD from the same geographical region also underwent sequencing.</p><p id="__p17" class="p p-last">Exome capture was completed by using NimbleGen EZ Exome SeqCap v2 (Roche NimbleGen, Madison, WI). One microgram of fragmented genomic DNA was used to prepare the library using the manufacturer’s protocol (<a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Information</a>). The bar-coded libraries were sequenced by using Illumina’s HiSeq 2500 platform, producing 75-bp paired-end reads that were aligned to the hg19 human genome build using NovoAlign (<a href="http://www.novocraft.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.novocraft.com</a>; Novocraft Technologies Sn Bhd, Selangor, Malaysia). Variant calling was performed jointly for all samples by using the HaplotypeCaller algorithm in GATK.</p></div><div id="s12" class="sec sec-last"><h3 id="s12title">Genetic Association Analysis</h3><p id="__p18" class="p p-first">All of the quantitative trait loci association analyses were performed within the AZ sample for a set of 5 quantitative phenotypes. SNP-based association analysis of age- and gender-adjusted quantile-normalized phenotypes was performed by using mixed linear modeling (MLM) as implemented in GEMMA<sup><a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561710">27</a></sup> version 0.94. MLM tests for genetic association of SNPs with quantitative traits were performed under the additive model while controlling for sample structure estimated directly from data as a genetic relatedness matrix. MLM can be considered an example of the de-correlation approach to family-based data, and we chose it as our analytical framework for several reasons. First, although a number of transmission-based approaches (eg, family-based association testing, FBAT) have been developed, their use in a large complex multigenerational pedigree is problematic and computationally intensive in the presence of missing genotypic or phenotypic data, requiring splitting the larger pedigree into smaller units; this approach, coupled with conditioning on the founders’ genotypes, can lead to a loss of power. Second and most importantly, comparative studies suggest that decorrelation approaches (and MLM among them) tend to have higher (or at least comparable) statistical power than transmission approaches even in large and complex pedigrees.<sup><a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196905">28</a></sup></p><p id="__p19">All 5 phenotypes were first used in a multivariate MLM analysis. Two multivariate MLMs were then fitted: 1 that modeled the genetic effects on the indicators of linguistic errors and the second that used syntactic complexity. We performed gene-based association analyses as implemented in KGG3 software<sup><a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561701">29</a></sup> version 3.0 by using the hybrid set–based test.</p><p id="__p20">Copy number variant (CNV) association analysis was performed in the FBAT<sup><a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561690">30</a></sup> framework. Samples underwent additional CNV-specific QC (<a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Information</a>). CNVs were identified by using a univariate Copy Number Analysis Method algorithm as implemented in SVS with a minimum of 5 markers per segment. Permutation testing was used to identify cut-points, and average segment intensity was used in the analyses.</p><p id="__p21">Homozygosity mapping was completed by using the runs of homozygosity (ROH) detection algorithm in SVS. The minimum size was set to 250 kb and 25 SNPs, allowing for up to 1 heterozygote and 5 missing genotypes. The maximum gap between SNPs was 100 kb, and the minimum density was 18 kb. The total length of ROHs in 5 different length brackets was log-transformed to ensure normality before analysis. ROH association and burden analyses were performed by using univariate linear and logistic regression in <em>R</em> (R Foundation for Statistical Computing, Vienna, Austria).</p><p id="__p22">WES data were analyzed by using a set of annotation and filtering tools. This analysis assumed that the most severely affected individuals in the AZ population from familial substructures with suggestive evidence for Mendelian transmission could provide additional information about the genetic architecture of DLD in the sample by focusing on: (1) the coding variants in candidate genes highlighted in the larger GWAS sample; or (2) the disruptive coding variants in other genes that could be conferring additional DLD risk in a subsample of the AZ population. Thus, we focused on coding sequence variants that were frequent among severely affected AZ probands (present in at least 4 of the 12 affected AZ individuals) but were not present in the control sample of 9 exomes. We then excluded variants observed in &gt;5% of the National Heart, Lung, and Blood Institute Go Exome Sequencing Project (<a href="https://esp.gs.washington.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://esp.gs.washington.edu</a>) and the 1000 Genomes project (<a href="http://www.1000genomes.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.1000genomes.org/</a>) Phase 1 exomes. We then retained only those variants that were located within the genes associated with any of the phenotypes in the GWAS (<em>P</em> &lt; .05 for gene-level tests), disruptive frameshift variants, and variants prioritized by eXtasy<sup><a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196908">31</a></sup> based on the fusion of the information about their pathogenicity, haploinsufficiency predictions, and similarity to other genes linked to related phenotypes (<a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Information</a>).</p><p id="__p23" class="p p-last"><em>P</em> values were corrected by using either standard or adjusted Bonferroni procedures (<a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Information</a>). The study protocol was approved by the Yale University (New Haven, CT) and Northern State Medical University (Arkhangelsk, Russia) internal review boards.</p></div></div><div id="s13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s13title">Results</h2><div id="s14" class="sec sec-first"><h3 id="s14title">Genome-wide SNP Associations</h3><p id="__p24" class="p p-first">No single SNP reached genome-wide statistical significance (<a href="/pmc/articles/PMC4811310/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig 1</span></a>). <a href="/pmc/articles/PMC4811310/table/t1/" target="table" class="fig-table-link figpopup" rid-figpopup="t1" rid-ob="ob-t1" co-legend-rid=""><span>Table 1</span></a> lists the top 10 nominally significant SNPs for each analysis. For linguistic errors, the strongest association (<em>P</em> = 5.35 × 10<sup>−7</sup>) was for rs3787751 (21q22), located in the noncoding region of the <em>HLCS</em> (holocarboxylase synthetase) gene, involved in the biotinylation of apocarboxylases. Holocarboxylase synthetase deficiency syndrome (MIM#253270) is characterized by neurologic, developmental, and metabolic abnormalities in infancy.<sup><a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561682">32</a></sup> For syntactic complexity, the top 10 SNPs included 4 SNPs (rs378968, rs3789867, rs2480933, and rs2482078) located in intronic regions of the <em>TNC</em> gene on chromosome 9q33 (<a href="/pmc/articles/PMC4811310/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig 2</span></a>). <em>TNC</em> codes for an extracellular matrix protein implicated in cochlear development<sup><a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561689">33</a></sup> and autosomal dominant deafness (MIM#615629). The univariate GWAS analyses produced similar results (<a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Information</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC4811310/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140598284694512" class="figure"><a href="/pmc/articles/PMC4811310/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4811310_PEDS_20152469_fig1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is PEDS_20152469_fig1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4811310/bin/PEDS_20152469_fig1.jpg"></a></div><div id="largeobj_idm140598284694512" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4811310/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC4811310/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">FIGURE 1</a></div><!--caption a7--><div class="caption"><p id="__p25">Manhattan plots of P values for three multivariate GWAS analyses. Top row - MLM analysis of all five phenotypes; Middle row - MLM analysis of linguistic errors; Bottom row - MLM analysis of syntactic complexity. </p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t1"><h3>TABLE 1</h3><!--caption a7--><div class="caption"><p id="__p26">Top 10 Nominally Significant SNP Associations for Each of the 3 Multivariate GWAS Analyses</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140598281399888" class="xtable"><table frame="above" rules="groups" class="rendered small default_table"><thead><tr><th scope="col" rowspan="1" colspan="1" valign="top" align="left">SNP</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Phenotype</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Chr</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Min/Maj</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">MAF</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">PPC</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">WF</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">CS</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">MLU</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">SPE</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center"><em>P</em>
</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Gene</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Variant Type</th></tr></thead><tbody><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs722208</td><td rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="center">6q25</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">−0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">3.09 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>ESR1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs1559831</td><td rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="center">15q14</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">−0.17</td><td rowspan="1" colspan="1" valign="top" align="center">−0.64</td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">−0.34</td><td rowspan="1" colspan="1" valign="top" align="center">3.81 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs6496012</td><td rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="center">15q14</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">−0.17</td><td rowspan="1" colspan="1" valign="top" align="center">−0.64</td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">−0.34</td><td rowspan="1" colspan="1" valign="top" align="center">3.81 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs3787751</td><td rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="center">21q22</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">−0.16</td><td rowspan="1" colspan="1" valign="top" align="center">−0.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">−0.07</td><td rowspan="1" colspan="1" valign="top" align="center">4.88 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>HLCS</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs639373</td><td rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">−0.10</td><td rowspan="1" colspan="1" valign="top" align="center">1.37 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>C2CD2L</em></td><td rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs2180386</td><td rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="center">14q32</td><td rowspan="1" colspan="1" valign="top" align="center">A/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">−0.35</td><td rowspan="1" colspan="1" valign="top" align="center">−0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">−0.11</td><td rowspan="1" colspan="1" valign="top" align="center">−0.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.60 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>MEG8</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs780382</td><td rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="center">11p15</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">−0.26</td><td rowspan="1" colspan="1" valign="top" align="center">−0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">−0.04</td><td rowspan="1" colspan="1" valign="top" align="center">1.89 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>SBF2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs585149</td><td rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">−0.02</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">2.83 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>SIK2, PPP2R1B</em></td><td rowspan="1" colspan="1" valign="top" align="left">UTR′-3</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs8066993</td><td rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="center">17q25</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">−0.06</td><td rowspan="1" colspan="1" valign="top" align="center">−0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">−0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">2.90 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>RNF213</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs2183850</td><td rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="center">13q34</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="left">2.98 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs3789868</td><td rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="center">9q33</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">−0.34</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">3.81 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>TNC</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs3789867</td><td rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="center">9q33</td><td rowspan="1" colspan="1" valign="top" align="center">T/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">−0.35</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">4.02 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>TNC</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs2480933</td><td rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="center">9q33</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">−0.34</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">5.51 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>TNC</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs2482078</td><td rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="center">9q33</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">−0.33</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">6.57 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>TNC</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs2011604</td><td rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="center">14q32</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">−0.30</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">1.25 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs585149</td><td rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">−0.01</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">1.70 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>SIK2, PPP2R1B</em></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs2893567</td><td rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="center">7p14-p13</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">−0.28</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">2.04 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>VPS41</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs1537722</td><td rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="center">9q33</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">−0.03</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">2.62 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs4796604</td><td rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="center">17q21</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">2.96 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>HAP1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Missense</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs2699376</td><td rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="center">2q14</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">−0.39</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="left">3.15 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>CNTNAP5</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs3787751</td><td rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="center">21q22</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">−0.16</td><td rowspan="1" colspan="1" valign="top" align="center">−0.39</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">−0.07</td><td rowspan="1" colspan="1" valign="top" align="left">5.35 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>HLCS</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs10504229</td><td rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="center">8q12</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="left">5.59 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>LINC00588</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs2180386</td><td rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="center">14q32</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">−0.33</td><td rowspan="1" colspan="1" valign="top" align="center">−0.27</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">−0.21</td><td rowspan="1" colspan="1" valign="top" align="left">7.37 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>MEG8</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs12121864</td><td rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="center">1p36</td><td rowspan="1" colspan="1" valign="top" align="center">C/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.64</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="left">8.00 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs10486031</td><td rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="center">7p21</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">−0.05</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">−0.32</td><td rowspan="1" colspan="1" valign="top" align="left">1.06 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>DGKB</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs639373</td><td rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">−0.09</td><td rowspan="1" colspan="1" valign="top" align="left">1.21 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>C2CD2L</em></td><td rowspan="1" colspan="1" valign="top" align="left">Upstream</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs643788</td><td rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">−0.06</td><td rowspan="1" colspan="1" valign="top" align="left">1.22 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>DPAGT1, H2AFX</em></td><td rowspan="1" colspan="1" valign="top" align="left">Missense</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs17445063</td><td rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="center">20q13</td><td rowspan="1" colspan="1" valign="top" align="center">G/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">2.16 ×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs722208</td><td rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="center">6q25</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="left">2.48 ×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>ESR1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">rs4895970</td><td rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="center">6q23</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">—</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">2.75 ×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr></tbody></table></div><div id="largeobj_idm140598281399888" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4811310/table/t1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140598288214080"><p id="__p27" class="p p-first-last">Coefficients for phonetic/prosodic characteristics (PPC), well-formedness (WF), complex structures (CS), mean length of utterance (in words) (MLU), and semantic/pragmatic errors (SPE) represent effect size estimates (regression coefficients in SD units) for the minor allele. Chr, chromosomal location (cytoband); Ling. err, linguistic errors; MAF, minor allele frequency; Min/Maj, minor/major allele; Syn. comp., syntactic complexity; — coefficient was not estimated in the analysis and is not available.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC4811310/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140598283973008" class="figure"><a href="/pmc/articles/PMC4811310/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4811310_PEDS_20152469_fig2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is PEDS_20152469_fig2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4811310/bin/PEDS_20152469_fig2.jpg"></a></div><div id="largeobj_idm140598283973008" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4811310/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC4811310/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">FIGURE 2</a></div><!--caption a7--><div class="caption"><p id="__p28">Regional association plots for the <em>TNC</em> (left) and <em>SETBP1</em> (right) genes and syntactic complexity phenotype. The purple diamond represents the SNP with the lowest P value in the plotted region.</p></div></div></div></div><div id="s15" class="sec"><h3 id="s15title">Gene-based Associations</h3><p id="__p29" class="p p-first">We found no genome-wide significant gene-based associations for the 5-phenotype multivariate analysis or linguistic errors. Importantly, such an association was established between <em>SETBP1</em> (SET binding protein 1; 18q21) and the multivariate syntactic complexity phenotype (<em>P</em> = 5.47 × 10<sup>−7</sup>) (<a href="/pmc/articles/PMC4811310/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig 2</span></a>). The nuclear protein encoded by <em>SETBP1</em> binds the SET nuclear oncogene protein involved in DNA replication, apoptosis, transcription, and nucleosome assembly. Rare variants in <em>SETBP1</em> are associated with Schinzel-Giedion syndrome (MIM#269150) characterized by severe developmental delays.</p><p id="__p30"><a href="/pmc/articles/PMC4811310/table/t2/" target="table" class="fig-table-link figpopup" rid-figpopup="t2" rid-ob="ob-t2" co-legend-rid=""><span>Table 2</span></a> presents the top 10 genes for the gene-based analyses. After <em>SETBP1</em>, the 2 next strongest associations with syntactic complexity were found for 2 genes on chromosome 11q23: <em>PPP2R1B</em> (<em>P</em> = 4.77 × 10<sup>−5</sup>), encoding a constant regulatory subunit of protein phosphatase 2A, and <em>SIK2</em> (<em>P</em> = 5.00 × 10<sup>−5</sup>), a gene hypothesized to play a role in neuronal protection. These findings are likely driven by the top hit SNP rs585149 (<em>P</em> = 1.70 × 10<sup>−5</sup>), assigned to both genes and located in the 3′-UTR region of <em>SIK2</em>. We also found a nominally significant association of syntactic complexity with <em>TNC</em> (<em>P</em> = .0068).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t2"><h3>TABLE 2</h3><!--caption a7--><div class="caption"><p id="__p31">Top 10 Gene-Based Associations for Each of the 3 Multivariate GWAS Analyses</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140598283235392" class="xtable"><table frame="above" rules="groups" class="rendered small default_table"><thead><tr><th scope="col" rowspan="1" colspan="1" valign="top" align="left">Phenotype</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Gene</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center"><em>P</em>
</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Chr</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Length, bp</th></tr></thead><tbody><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ESR1</em></td><td rowspan="1" colspan="1" valign="top" align="center">4.76 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6q25</td><td rowspan="1" colspan="1" valign="top" align="left">297 602</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ABCG4</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.53 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">13 626</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SBF2</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.58 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11p15</td><td rowspan="1" colspan="1" valign="top" align="left">515 542</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PPP2R1B</em></td><td rowspan="1" colspan="1" valign="top" align="center">7.90 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">28 566</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SIK2</em></td><td rowspan="1" colspan="1" valign="top" align="center">8.30 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">124 464</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HYOU1</em></td><td rowspan="1" colspan="1" valign="top" align="center">8.44 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">13 022</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HINFP</em></td><td rowspan="1" colspan="1" valign="top" align="center">8.77 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">13 534</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SNORD113-9,-7,-8</em></td><td rowspan="1" colspan="1" valign="top" align="center">9.05 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">14q32</td><td rowspan="1" colspan="1" valign="top" align="left">72–74</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SNORD113-4,-5</em></td><td rowspan="1" colspan="1" valign="top" align="center">9.06 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">14q32</td><td rowspan="1" colspan="1" valign="top" align="left">75–78</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">All</td><td rowspan="1" colspan="1" valign="top" align="left"><em>H2AFX</em></td><td rowspan="1" colspan="1" valign="top" align="center">9.10 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">1594</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SETBP1</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.47 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">18q21</td><td rowspan="1" colspan="1" valign="top" align="left">388 337</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PPP2R1B</em></td><td rowspan="1" colspan="1" valign="top" align="center">4.77 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">28 566</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SIK2</em></td><td rowspan="1" colspan="1" valign="top" align="center">5.00 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">124 464</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SERPINA1</em></td><td rowspan="1" colspan="1" valign="top" align="center">7.83 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">14q32</td><td rowspan="1" colspan="1" valign="top" align="left">13 947</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SPATA2</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.05 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">20q13</td><td rowspan="1" colspan="1" valign="top" align="left">12 153</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>EIF1</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.14 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">17q21</td><td rowspan="1" colspan="1" valign="top" align="left">2773</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ST7-OT3</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.33 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7q31</td><td rowspan="1" colspan="1" valign="top" align="left">27 258</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RNF114</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.40 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">20q13</td><td rowspan="1" colspan="1" valign="top" align="left">17 510</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HAP1</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.52 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">17q21</td><td rowspan="1" colspan="1" valign="top" align="left">12 009</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Syn. comp.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GAST</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.54 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">17q21</td><td rowspan="1" colspan="1" valign="top" align="left">3645</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LINC00588</em></td><td rowspan="1" colspan="1" valign="top" align="center">3.44 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8q12</td><td rowspan="1" colspan="1" valign="top" align="left">5190</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RP11-513O17.3</em></td><td rowspan="1" colspan="1" valign="top" align="center">3.60 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8q12</td><td rowspan="1" colspan="1" valign="top" align="left">5387</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SNORD113-9,-7,-8,-4,-5</em></td><td rowspan="1" colspan="1" valign="top" align="center">4.17 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">14q32</td><td rowspan="1" colspan="1" valign="top" align="left">72–78</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HLCS</em></td><td rowspan="1" colspan="1" valign="top" align="center">4.40 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">21q22</td><td rowspan="1" colspan="1" valign="top" align="left">239 358</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SNORD113-3</em></td><td rowspan="1" colspan="1" valign="top" align="center">4.76 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">14q32</td><td rowspan="1" colspan="1" valign="top" align="left">72</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SNORD113-6</em></td><td rowspan="1" colspan="1" valign="top" align="center">4.90 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">14q32</td><td rowspan="1" colspan="1" valign="top" align="left">75</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SNORD113-2,-1</em></td><td rowspan="1" colspan="1" valign="top" align="center">5.46 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">14q32</td><td rowspan="1" colspan="1" valign="top" align="left">71–72</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ABCG4</em></td><td rowspan="1" colspan="1" valign="top" align="center">5.75 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">13 626</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HYOU1</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.34 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">13 022</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ling. err.</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HINFP</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.59 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left">13 534</td></tr></tbody></table></div><div id="largeobj_idm140598283235392" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4811310/table/t2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140598274187760"><p id="__p32" class="p p-first-last">Chr, chromosomal location (cytoband); Ling. err., linguistic errors; Syn. comp., syntactic complexity. </p></div><div id="tt2n2"><sup>a</sup>Statistically significant after Bonferroni corrections for multiple testing.</div></div></div><p id="__p34" class="p p-last">Nominally significant associations were also established between linguistic errors and several genes (<em>ABCG4, HYOU1</em>, and <em>HINFP</em>) 7 Mb away from <em>PPP2R1B</em> (11q23); these genes and <em>DPAGT1</em> and <em>H2AFX</em> were associated with the combined multivariate DLD phenotype, with the top hits being rs639373 (<em>P</em> = 1.21 × 10<sup>−5</sup>) and rs643788 (<em>P</em> = 1.22 × 10<sup>−5</sup>). The functional significance of the <em>ABCG4</em> product is unknown; the <em>DPAGT1</em> product is crucial for glycoprotein biosynthesis; and <em>H2AFX</em> encodes a histone involved in the maintenance of chromatin structure. The transcription factor encoded by <em>HINFP</em> plays an important role in DNA methylation. We found an association between the combined multivariate phenotype and estrogen-receptor 1 (<em>ESR1; P</em> = 4.76 × 10<sup>−5</sup>), with rs722208 (<em>P</em> = 3.09 × 10<sup>−6</sup>) as a top hit. There was a nominally significant association between linguistic errors and the <em>HLCS</em> gene (<em>P</em> = 4.40 × 10<sup>−5</sup>). Neither linguistic errors nor syntactic complexity was associated with previously identified candidate DLD genes.</p></div><div id="s16" class="sec"><h3 id="s16title">CNV Analysis and Homozygosity Mapping</h3><p id="__p35" class="p p-first">The multivariate FBAT CNV analysis revealed several nominally statistically significant and 1 highly statistically significant CNV. However, follow-up confirmation using real-time polymerase chain reaction (PCR) failed to substantiate the presence of these CNVs. An alternative pipeline that integrated 3 CNV detection algorithms yielded no genome-wide significant associations (<a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Information</a>).</p><p id="__p36" class="p p-last">Overall, AZ-affected individuals, compared with unaffected individuals, had longer cumulative lengths of ROHs that were 250 to 500 kb long (<em>P</em> = .006) and 1000 to 1750 kb long (<em>P</em> = .004), corresponding to ∼10% and ∼1% increases in estimated autosomal homozygosity, respectively (<a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Fig 3</a>). The association analysis did not reveal any ROHs that were genome-wide significantly enriched in affected individuals. None of the top 20 regions overlapped with the regions identified in the SNP analyses. Several potentially relevant identified regions are discussed in <a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Information</a>.</p></div><div id="s17" class="sec"><h3 id="s17title">Whole Exome DNA Sequencing</h3><p id="__p37" class="p p-first">We identified 14 coding sequence variants, frequent in affected AZ individuals: 4 frameshift indels, 1 inframe insertion, 2 stop gain/loss, and 7 missense variants (<a href="/pmc/articles/PMC4811310/table/t3/" target="table" class="fig-table-link figpopup" rid-figpopup="t3" rid-ob="ob-t3" co-legend-rid=""><span>Table 3</span></a>). SNVs were predicted by polymorphism phenotyping (PolyPhen) to be possibly or probably damaging. Although any or all of these 14 variants could be implicated in the etiology of DLD in AZ, 2 sets of findings deserve special attention.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t3"><h3>TABLE 3</h3><!--caption a7--><div class="caption"><p id="__p38">Prioritized Coding Variants Identified in WES Data</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140598279256176" class="xtable"><table frame="above" rules="groups" class="rendered small default_table"><thead><tr><th scope="col" rowspan="1" colspan="1" valign="top" align="center">ID</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Type</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Variant</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center"><em>N</em></th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Gene (GWAS <em>P</em><sup>a</sup>)</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Brain Exp.</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">NHLBI Freq.</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">rsid</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Protein Change</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">PolyPhen</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">GERP</th></tr></thead><tbody><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW1</td><td rowspan="1" colspan="1" valign="top" align="left">Del (FS)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294394" class="tag_hotlink" href="/nuccore/NC_000001.10" ref="/nuccore/NC_000001.10"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000001.10","term_id":"224589800","term_text":"NC_000001.10"}}</span>NC_000001.10</a>:g.6727803_6727804delTC</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DNAJC1</em> (<em>P</em> = .0509)</td><td rowspan="1" colspan="1" valign="top" align="left">Low</td><td rowspan="1" colspan="1" valign="top" align="center">0.0000</td><td rowspan="1" colspan="1" valign="top" align="left">rs374290353</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294380" class="tag_hotlink" href="/protein/NP_060668.2" ref="/protein/NP_060668.2"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_060668.2","term_id":"217035105","term_text":"NP_060668.2"}}</span>NP_060668.2</a>:p.Glu115fs</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW2</td><td rowspan="1" colspan="1" valign="top" align="left">Ins (FS)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294393" class="tag_hotlink" href="/nuccore/NC_000003.11" ref="/nuccore/NC_000003.11"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000003.11","term_id":"224589815","term_text":"NC_000003.11"}}</span>NC_000003.11</a>:g.52559298_52559299insCAAA</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NT5DC2</em> (<em>P</em> = .3050)</td><td rowspan="1" colspan="1" valign="top" align="left">High (pn)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0002</td><td rowspan="1" colspan="1" valign="top" align="left">rs745485897</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294389" class="tag_hotlink" href="/protein/NP_001127703.1" ref="/protein/NP_001127703.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_001127703.1","term_id":"197116386","term_text":"NP_001127703.1"}}</span>NP_001127703.1</a>:p.Asp378fs</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">5.1</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW3</td><td rowspan="1" colspan="1" valign="top" align="left">Stop gain (SNV)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294392" class="tag_hotlink" href="/nuccore/NC_000006.11" ref="/nuccore/NC_000006.11"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000006.11","term_id":"224589818","term_text":"NC_000006.11"}}</span>NC_000006.11</a>:g.167591956C&gt;T</td><td rowspan="1" colspan="1" valign="top" align="center">7/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TCP10L2</em> (<em>P</em> = .0198)</td><td rowspan="1" colspan="1" valign="top" align="left">Medium</td><td rowspan="1" colspan="1" valign="top" align="center">0.0000</td><td rowspan="1" colspan="1" valign="top" align="left">rs2297462</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294385" class="tag_hotlink" href="/protein/NP_001138593.1" ref="/protein/NP_001138593.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_001138593.1","term_id":"223468602","term_text":"NP_001138593.1"}}</span>NP_001138593.1</a>:p.Gln195Ter</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW4</td><td rowspan="1" colspan="1" valign="top" align="left">Stop loss (SNV)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294404" class="tag_hotlink" href="/nuccore/NC_000009.11" ref="/nuccore/NC_000009.11"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000009.11","term_id":"224589821","term_text":"NC_000009.11"}}</span>NC_000009.11</a>:g.71152177A&gt;G</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TMEM252</em> (<em>P</em> = .0921)</td><td rowspan="1" colspan="1" valign="top" align="left">Low/ medium</td><td rowspan="1" colspan="1" valign="top" align="center">0.0030</td><td rowspan="1" colspan="1" valign="top" align="left">rs147022534</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294379" class="tag_hotlink" href="/protein/NP_694969.1" ref="/protein/NP_694969.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_694969.1","term_id":"23397518","term_text":"NP_694969.1"}}</span>NP_694969.1</a>:p.Ter171fs</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">0.4</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW5</td><td rowspan="1" colspan="1" valign="top" align="left">Ins</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294398" class="tag_hotlink" href="/nuccore/NC_000011.9" ref="/nuccore/NC_000011.9"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000011.9","term_id":"224589802","term_text":"NC_000011.9"}}</span>NC_000011.9</a>:g.1651095insGGCTGTGGCTCCGGCTGTGGAGGCCTTGGCTCCGGCTGTGGG</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>KRTAP5-5</em> (<em>P</em> = .0163)</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="center">0.0000</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294382" class="tag_hotlink" href="/protein/NP_001001480.2" ref="/protein/NP_001001480.2"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_001001480.2","term_id":"122891874","term_text":"NP_001001480.2"}}</span>NP_001001480.2</a>:p.Gly9_Cys10insLeuTrpLeuArgLeuTrpArgProTrpLeuArgLeuTrpGly</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW6</td><td rowspan="1" colspan="1" valign="top" align="left">Del (FS)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294400" class="tag_hotlink" href="/nuccore/NC_000019.9" ref="/nuccore/NC_000019.9"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000019.9","term_id":"224589810","term_text":"NC_000019.9"}}</span>NC_000019.9</a>:g.13255238delG</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>STX10</em> (<em>P</em> = .3610)</td><td rowspan="1" colspan="1" valign="top" align="left">Medium (us)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0018</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294387" class="tag_hotlink" href="/protein/NP_003756.1" ref="/protein/NP_003756.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_003756.1","term_id":"4507285","term_text":"NP_003756.1"}}</span>NP_003756.1</a>:p.Leu246fs</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">1.1</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW7</td><td rowspan="1" colspan="1" valign="top" align="left">Del (FS)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294401" class="tag_hotlink" href="/nuccore/NC_000022.10" ref="/nuccore/NC_000022.10"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000022.10","term_id":"224589814","term_text":"NC_000022.10"}}</span>NC_000022.10</a>:g.40161342_40161343delGA</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ENTHD1</em> (<em>P</em> = .4250)</td><td rowspan="1" colspan="1" valign="top" align="left">High (us)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0075</td><td rowspan="1" colspan="1" valign="top" align="left">rs200712517</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294386" class="tag_hotlink" href="/protein/NP_689725.2" ref="/protein/NP_689725.2"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_689725.2","term_id":"31377575","term_text":"NP_689725.2"}}</span>NP_689725.2</a>:p.Pro369fs</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">5.9</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW8</td><td rowspan="1" colspan="1" valign="top" align="left">Missense (SNV)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294395" class="tag_hotlink" href="/nuccore/NC_000007.13" ref="/nuccore/NC_000007.13"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000007.13","term_id":"224589819","term_text":"NC_000007.13"}}</span>NC_000007.13</a>:g.100465824G&gt;A</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TRIP6</em> (<em>P</em> = .4260)</td><td rowspan="1" colspan="1" valign="top" align="left">Medium (us)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0205</td><td rowspan="1" colspan="1" valign="top" align="left">rs2437100</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294377" class="tag_hotlink" href="/protein/NP_003293.2" ref="/protein/NP_003293.2"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_003293.2","term_id":"91208423","term_text":"NP_003293.2"}}</span>NP_003293.2</a>:p.Arg111Gln</td><td rowspan="1" colspan="1" valign="top" align="center">PrD</td><td rowspan="1" colspan="1" valign="top" align="center">4.5</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW9</td><td rowspan="1" colspan="1" valign="top" align="left">Missense (SNV)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294403" class="tag_hotlink" href="/nuccore/NC_000008.10" ref="/nuccore/NC_000008.10"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000008.10","term_id":"224589820","term_text":"NC_000008.10"}}</span>NC_000008.10</a>:g.91953077G&gt;T</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NECAB1</em> (<em>P</em> = .0246)</td><td rowspan="1" colspan="1" valign="top" align="left">High (esp. temporal)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0001</td><td rowspan="1" colspan="1" valign="top" align="left">rs115555424</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294384" class="tag_hotlink" href="/protein/NP_071746.1" ref="/protein/NP_071746.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_071746.1","term_id":"24308279","term_text":"NP_071746.1"}}</span>NP_071746.1</a>:p.Ala271Thr</td><td rowspan="1" colspan="1" valign="top" align="center">PrD</td><td rowspan="1" colspan="1" valign="top" align="center">4.4</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW10</td><td rowspan="1" colspan="1" valign="top" align="left">Missense (SNV)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294397" class="tag_hotlink" href="/nuccore/NC_000009.11" ref="/nuccore/NC_000009.11"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000009.11","term_id":"224589821","term_text":"NC_000009.11"}}</span>NC_000009.11</a>:g.124072992G&gt;A</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GSN</em> (<em>P</em> = .8910)</td><td rowspan="1" colspan="1" valign="top" align="left">Medium (us)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0047</td><td rowspan="1" colspan="1" valign="top" align="left">rs41305623</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294378" class="tag_hotlink" href="/protein/NP_000168.1" ref="/protein/NP_000168.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_000168.1","term_id":"4504165","term_text":"NP_000168.1"}}</span>NP_000168.1</a>:p.Val179Met</td><td rowspan="1" colspan="1" valign="top" align="center">PrD</td><td rowspan="1" colspan="1" valign="top" align="center">5.3</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW11</td><td rowspan="1" colspan="1" valign="top" align="left">Missense (SNV)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294399" class="tag_hotlink" href="/nuccore/NC_000011.9" ref="/nuccore/NC_000011.9"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000011.9","term_id":"224589802","term_text":"NC_000011.9"}}</span>NC_000011.9</a>:g.3700876C&gt;T</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NUP98</em> (<em>P</em> = .9480)</td><td rowspan="1" colspan="1" valign="top" align="left">Low/medium</td><td rowspan="1" colspan="1" valign="top" align="center">0.0164</td><td rowspan="1" colspan="1" valign="top" align="left">rs34446357</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294381" class="tag_hotlink" href="/protein/NP_057404.2" ref="/protein/NP_057404.2"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_057404.2","term_id":"21264365","term_text":"NP_057404.2"}}</span>NP_057404.2</a>:p.Glu1661Lys</td><td rowspan="1" colspan="1" valign="top" align="center">PrD</td><td rowspan="1" colspan="1" valign="top" align="center">5.6</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW12</td><td rowspan="1" colspan="1" valign="top" align="left">Missense (SNV)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294405" class="tag_hotlink" href="/nuccore/NC_000011.9" ref="/nuccore/NC_000011.9"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000011.9","term_id":"224589802","term_text":"NC_000011.9"}}</span>NC_000011.9</a>:g.6629343T&gt;A</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ILK</em> (<em>P</em> = .1540)</td><td rowspan="1" colspan="1" valign="top" align="left">Low</td><td rowspan="1" colspan="1" valign="top" align="center">0.0001</td><td rowspan="1" colspan="1" valign="top" align="left">rs200336608</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294383" class="tag_hotlink" href="/protein/NP_001014794.1" ref="/protein/NP_001014794.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_001014794.1","term_id":"62420873","term_text":"NP_001014794.1"}}</span>NP_001014794.1</a>:p.Leu53Met</td><td rowspan="1" colspan="1" valign="top" align="center">PrD</td><td rowspan="1" colspan="1" valign="top" align="center">3.9</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW13</td><td rowspan="1" colspan="1" valign="top" align="left">Missense (SNV)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294402" class="tag_hotlink" href="/nuccore/NC_000017.10" ref="/nuccore/NC_000017.10"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000017.10","term_id":"224589808","term_text":"NC_000017.10"}}</span>NC_000017.10</a>:g.56355397G&gt;A</td><td rowspan="1" colspan="1" valign="top" align="center">4/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MPO</em> (<em>P</em> = .1420)</td><td rowspan="1" colspan="1" valign="top" align="left">No</td><td rowspan="1" colspan="1" valign="top" align="center">0.0135</td><td rowspan="1" colspan="1" valign="top" align="left">rs28730837</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294390" class="tag_hotlink" href="/protein/NP_000241.1" ref="/protein/NP_000241.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_000241.1","term_id":"4557759","term_text":"NP_000241.1"}}</span>NP_000241.1</a>:p.Ala332Val</td><td rowspan="1" colspan="1" valign="top" align="center">PoD</td><td rowspan="1" colspan="1" valign="top" align="center">4.3</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">AZW14</td><td rowspan="1" colspan="1" valign="top" align="left">Missense (SNV)</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294396" class="tag_hotlink" href="/nuccore/NC_000018.9" ref="/nuccore/NC_000018.9"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000018.9","term_id":"224589809","term_text":"NC_000018.9"}}</span>NC_000018.9</a>:g.25532304T&gt;C</td><td rowspan="1" colspan="1" valign="top" align="center">6/12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CDH2</em> (<em>P</em> = .0041)</td><td rowspan="1" colspan="1" valign="top" align="left">High (pn)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0266</td><td rowspan="1" colspan="1" valign="top" align="left">rs2289664</td><td rowspan="1" colspan="1" valign="top" align="left"><a id="__tag_717294388" class="tag_hotlink" href="/protein/NP_001783.2" ref="/protein/NP_001783.2"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_001783.2","term_id":"14589889","term_text":"NP_001783.2"}}</span>NP_001783.2</a>:p.Asn845Se</td><td rowspan="1" colspan="1" valign="top" align="center">PoD</td><td rowspan="1" colspan="1" valign="top" align="center">5.5</td></tr></tbody></table></div><div id="largeobj_idm140598279256176" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4811310/table/t3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140598284461104"><p id="__p39" class="p p-first-last">Brain expression (Brain Exp.) information is based on the combination of the data from the BrainSpan Atlas of the Developing Human Brain (<a href="http://www.brainspan.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.brainspan.org</a>), the Human Protein Atlas (<a href="http://www.proteinatlas.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.proteinatlas.org</a>), and the Expression Atlas (<a href="http://www.ebi.ac.uk/gxa/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ebi.ac.uk/gxa/</a>). Del, deletion; FS, frameshift; GERP, Genomic Evolutionary Rate Profiling sequence conservation score; Ins, insertion; <em>N</em>, number of carriers (of 12 individuals); NA, not available; NHLBI Freq., variant frequency in National Heart, Lung, and Blood Institute European American exomes; pn, prenatally; PoD, possibly damaging; PrD, probably damaging; us, ubiquitous; rsid, Single Nucleotide Polymorphism Database submission number. </p></div><div id="tt3n2"><sup>a</sup><em>P</em> value for the HYST gene-based analysis (multivariate, all phenotypes).</div></div></div><p id="__p41">First, multiple individuals in the AZ population carried coding sequence variants in genes that regulate neural development or are highly expressed in the brain; that is, a frameshift insertion in <em>NT5DC2</em> (3p21.1) and missense SNVs in <em>NECAB1</em> (8q21.3) and <em>ILK</em> (11p15.4). <em>NT5DC2</em> has been implicated in schizophrenia<sup><a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196904">34</a></sup> and borderline personality disorder.<sup><a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196917">35</a></sup>
<em>NECAB1</em> is a member of the neuronal calcium-binding family of proteins essential to Ca<sup>2+</sup>-mediated signaling and is highly expressed in the temporal lobe.<sup><a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561691">36</a></sup> The protein encoded by <em>ILK</em> is 1 of the key regulators of neural stem cell astrocytic differentiation<sup><a href="#B37" rid="B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561685">37</a></sup> and neurite outgrowth.<sup><a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196916">38</a></sup> We also found that 7 (58%) of 12 individuals in the AZ population carried a known missense variant in <em>CDH2</em> (18q12) that was found only at a 2% frequency in the 1000 Genomes data set. <em>CDH2</em> codes for a major cadherin that is widely expressed prenatally in neural stem cells and supports their differentiation and migration,<sup><a href="#B39" rid="B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196902">39</a></sup> regulating the laminar organization of the cortex.<sup><a href="#B40" rid="B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196919">40</a></sup> Moreover, 7 of 12 AZ individuals carried a stop-gain variant in <em>TCP10L2</em> (6q27). It is unknown whether <em>TCP10L2</em> codes for a functional protein; it is highly similar to <em>TCP10L</em>, a primate-specific transcription factor thought to evolve via segmental duplication<sup><a href="#B41" rid="B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196913">41</a></sup> from <em>TCP10L2</em> or <em>TCP10.</em></p><p id="__p42">Second, a missense SNV in <em>TRIP6</em> (7q22.1) and a frameshift deletion in <em>ENTHD1</em> (22q13) indicate commonalities between the genetic pathways identified through GWAS and WES. <em>TRIP6</em> is a transcription factor that has been identified as a regulator of postnatal neural stem cell maintenance in the subventricular zone.<sup><a href="#B42" rid="B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561686">42</a></sup>
<em>ENTHD1</em> codes for ENTH domain-containing protein 1. ENTH domain-containing proteins are involved in synaptic vesicle endocytosis at nerve terminals at the crucial stages that precede synapse formation.<sup><a href="#B43" rid="B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196903">43</a></sup> Importantly, <em>TRIP6</em> interacts with and <em>ENTHD1</em> is upregulated by the same family of genes, myocyte enhancer factor–2 (MEF2), labeled <em>MEF2A-D</em>. MEF2 are transcription factors implicated in muscle and central nervous system differentiation. In addition to <em>ENTHD1</em>, MEF2 targets in human neural stem cells include <em>SETBP1, TNC,</em> and <em>DKGB</em> (3 genes highlighted by our GWAS), as well as individual genes (<em>BDNF</em>, <em>DMD</em>, and <em>NCAM2</em>) and gene families (cadherins, contactins, semaphorins, and serpins) implicated in (a)typical central nervous system development. A targeted formal analysis of gene list enrichment using the Enrichr tool<sup><a href="#B44" rid="B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561708">44</a></sup> suggested that, combined, GWAS and WES hits in this population are indeed enriched for MEF2 targets (for <em>MEF2A</em>, <em>P</em> = 1.28 × 10<sup>−6</sup>) (<a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Information</a>), providing support to this hypothesis.</p><p id="__p43" class="p p-last">Our WES analysis also revealed the presence of 2 heterozygous missense mutations in <em>SETBP1</em>, carried by 2 (rs3744825) and 1 (rs1064204) sequenced AZ individual, respectively. Both were common (for European ancestry, minor allele frequency &gt;10% in National Heart, Lung, and Blood Institute exome database) known SNPs, projected to be tolerated according to 5 different functional prediction algorithms.</p></div><div id="s18" class="sec sec-last"><h3 id="s18title">Replication</h3><p id="__p44" class="p p-first">We interrogated the main loci highlighted in the GWAS or WES analyses of DLD in the AZ population in an independent sample (<em>n</em> = 372) of children at risk for developmental disorders of language (spoken and written) by using teachers’ ratings of student’s spoken and written language skills as the main phenotype (details are given in the <a href="http://pediatrics.aappublications.org/lookup/suppl/doi:10.1542/peds.2015-2469/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Information</a>). Association analysis controlled for age and gender and was performed by using EMMAX,<sup><a href="#B45" rid="B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561692">45</a></sup> a MLM algorithm implemented in SVS.</p><p id="__p45">Both main findings were replicated. First, a significant gene-based association was found between language scores and <em>SETBP1</em> (<em>P</em> = .009360). The top signal originated at exm1383999/rs11082414 (<em>P</em> = .000359), a missense SNP located within exon 4 of <em>SETBP1</em> that explained 3.41% of the variance in children’s language skills. Predicted to be tolerated according to sorting intolerant from tolerant/PolyPhen, this SNP may play a role in the regulation of expression of <em>SETBP1</em>. The analysis of the Braineac<sup><a href="#B46" rid="B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561705">46</a></sup> brain expression quantitative trait loci database suggested that it differentiates levels of <em>SETBP1</em> expression in the brain, including the cerebellar cortex, hippocampus, and temporal cortex.</p><p id="__p46" class="p p-last">Second, genes nominally associated (at <em>P</em> &lt; .05) with teacher ratings of students’ spoken and written language skills were enriched for <em>MEF2A</em> targets (<em>P</em> = .0007024), replicating the finding from the discovery cohort.</p></div></div><div id="s19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s19title">Discussion</h2><p id="__p47" class="p p-first">We established a genome-wide association between syntactic complexity and the <em>SETBP1</em> gene in the AZ sample and then replicated it in an independent sample. <em>SETBP1</em> is relatively large (388 337 bp), has 2 isoforms, and is expressed widely. Although little is known about its function, it is implicated in several neurodevelopmental conditions: <em>SETBP1</em> haploinsufficiency is documented in expressive DLD<sup><a href="#B47" rid="B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561699">47</a></sup><sup>–</sup><sup><a href="#B49" rid="B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196911">49</a></sup> and intellectual disability.<sup><a href="#B50" rid="B50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561683">50</a></sup> Moreover, several tentative SNP associations were found between syntactic complexity and <em>TNC</em> that encodes tenascin, an extracellular matrix glycoprotein involved in neural development; TNC-deficient mice exhibit structural and functional cortical abnormalities, including atypical neuronal density and abnormal dendrite morphology.<sup><a href="#B51" rid="B51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561703">51</a></sup> However, the combined multivariate phenotype was also nominally associated with <em>ESR1</em>, a nuclear hormone receptor involved in regulation of gene expression, cell proliferation, and differentiation. Estrogen is involved in synaptogenesis, regulates neurotransmission, and modulates the activity of all types of neural cells.<sup><a href="#B52" rid="B52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561693">52</a></sup> This finding is intriguing given the male bias in incidence of DLD and the recent report of associations between early postnatal gender hormone concentrations and later language development.<sup><a href="#B53" rid="B53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561694">53</a></sup></p><p id="__p48">Our WES highlighted 14 coding variants in a set of genes implicated in neural development and/or differentiation. Intriguingly, 2 of the WES-identified genes (<em>ENTHD1</em> and <em>TRIP6</em>) and 3 of the GWAS-identified genes (<em>SETBP1, TNC</em>, and <em>DKGB</em>) interact with or are regulated by the MEF2 transcription factors. MEF2 isoforms are widely expressed in neural cells,<sup><a href="#B54" rid="B54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_788446216">54</a></sup> and their activity is regulated by extracellular factors (eg, in neurons via neurotrophin stimulation or Ca<sup>2+</sup> influx after the release of neurotransmitters). MEF2 targets show enriched expression in the central nervous system and implicate multiple signaling pathways, rendering MEF2 as a key regulator of activity-dependent synapse development.<sup><a href="#B55" rid="B55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561707">55</a></sup> The complex transcriptional program of MEF2 results in the restriction of excitatory synaptic transmission via the reduction of the number of excitatory neurons, elimination of glutamatergic synapses,<sup><a href="#B56" rid="B56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196918">56</a></sup> and postsynaptic differentiation of neurons (dendrite morphogenesis).<sup><a href="#B57" rid="B57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196910">57</a></sup></p><p id="__p49">The cascade of events regulated by the transcriptional activity of MEF2 is critical for learning and memory.<sup><a href="#B58" rid="B58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561698">58</a></sup><sup>,</sup><sup><a href="#B59" rid="B59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196907">59</a></sup> A recent electrophysiological study partially attributed the DLD phenotype in the AZ population to atypicalities in the functioning of neural circuits that support attention and memory<sup><a href="#B60" rid="B60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530561713">60</a></sup> that were linked to syntactic complexity. It is plausible they at least partially stem from the dysregulation of common genetic pathways that orchestrate neural development.</p><p id="__p50">This dysregulation can take multiple forms. Given the partial convergence of the results from the GWAS and WES, we hypothesized that the DLD phenotype in the AZ population emerged as the result of the interaction between common genetic variants that conferred background DLD susceptibility and rare variants that altered the development of language and memory circuits against that background. This extension of the threshold-dependent response model suggests that common variants in several genes (eg, <em>SETBP1</em>, <em>TNC</em>) formed the probabilistic landscape(s) of DLD vulnerability, and that coding variants in multiple different genes (eg, regulated by MEF2 such as such <em>ENTHD1</em> and <em>TRIP6</em> or other genes important for neural development such as <em>CDH2</em> or <em>NECAB1</em>) conferred the critical amount of vulnerability and pushed this landscape into a critical state.</p><p id="__p51">Finally, we established a higher rate of autosomal ROH burden among the affected AZ individuals compared with unaffected AZ individuals; this finding is not surprising given the isolated nature of the population and the role of ROHs in several developmental disorders.<sup><a href="#B61" rid="B61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559196914">61</a></sup> However, no single specific ROH was strongly associated with DLD. In addition, there was little overlap between the genetic loci identified in the GWAS analyses of the 2 multivariate phenotypes; this outcome raises an interesting hypothesis that the 2 global facets of DLD may be relatively independent at the level of their molecular neurobiology.</p><p id="__p52" class="p p-last">Our study has several limitations. First, it has a small sample size. Although it was modest for a GWAS study, however, the sample size was almost one-half of the total AZ population. Second, the unique nature of the population poses a complex issue for future research seeking to replicate these signals in other samples. Although we replicated the association finding for <em>SETBP1</em> and the enrichment findings for GWAS-highlighted DLD genes for MEF2 targets in an independent sample of children at risk for a related disorder, further molecular and analytical studies in larger samples are necessary to better characterize the joint contribution of common and rare variants in the identified genes to DLD susceptibility and decipher the molecular pathways they affect.</p></div><div id="s20" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s20title">Conclusions</h2><p id="__p53" class="p p-first-last">This study presented a set of novel candidate genes and coding DNA sequence variants contributing to DLD phenotypes in the AZ population; the chief findings from this population have been replicated in an independent sample. Overall, the findings suggest that multiple genes (including a novel genome-wide significant candidate <em>SETBP1</em>) and genetic pathways (including the suggested MEF2-regulated pathway) are involved in DLD. This study underlines the complexity of the genetic architecture of DLDs and illustrates that even in populations with reduced genetic and environmental diversity, DLD is best conceptualized as a polygenic and etiologically complex disorder.</p></div><div id="idm140598276328144" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140598276328144title">Acknowledgments</h2><div class="sec"><p id="__p59">The authors thank the families who participated in the study for their cooperation and patience, and the local medical, kindergarten, and school officials of the AZ community for their help with data collection. They also thank Igor Pushkin, Anastasia Strelina, Liudmila Kniazeva, and various students, trainees, and employees from Northern State Medical University for their help with the logistics of the study; Dr Lesley Hart for her contributions to the early stages of the project; Drs Seongmin Han and Dean Palejev for their involvement at various stages of the project; Ms Mei Tan for her editorial assistance; and the late Dr Maria Babyonyshev for her contribution to the linguistic component of the study.</p></div></div><div id="idm140598282918496" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140598282918496title">Glossary</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">ACES</td><td>Academic Competence Evaluation Scales</td></tr><tr><td id="G2">CNV</td><td>copy number variant</td></tr><tr><td id="G3">DLD</td><td>developmental language disorder</td></tr><tr><td id="G4">GWAS</td><td>genome-wide association study</td></tr><tr><td id="G5">HWE</td><td>Hardy-Weinberg equilibrium</td></tr><tr><td id="G6">MEF2</td><td>myocyte enhancer factor–2</td></tr><tr><td id="G7">MLM</td><td>mixed linear modeling</td></tr><tr><td id="G8">PCR</td><td>polymerase chain reaction</td></tr><tr><td id="G9">PolyPhen</td><td>polymorphism phenotyping</td></tr><tr><td id="G10">QC</td><td>quality control</td></tr><tr><td id="G11">ROH</td><td>runs of homozygosity</td></tr><tr><td id="G12">SNP</td><td>single nucleotide polymorphism</td></tr><tr><td id="G13">SNV</td><td>single nucleotide variant</td></tr><tr><td id="G14">SVS</td><td>SNP &amp; Variation Suite</td></tr><tr><td id="G15">WES</td><td>whole exome sequencing</td></tr></tbody></table></div></div></div><div id="idm140598280990656" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140598280990656title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><div>Contributed by <p id="__p54" class="p p-first-last">Dr Kornilov collected the data, performed data analyses, interpreted the data, drafted the initial manuscript, and critically revised the manuscript; Dr Rakhlin collected the data, designed the scoring rubrics for phenotyping, scored the data, interpreted the data, and revised the manuscript; Dr Koposov supervised data collection on-site, collected the data, and revised the manuscript; Ms Lee and Dr Yrigollen managed DNA specimens, processed and interpreted the molecular genetic data, and drafted the initial manuscript; Drs Caglayan, Magnuson, and Mane contributed to data analyses, interpreted the data, and critically revised the manuscript; Dr Chang designed the study, developed the sampling strategy, performed data analyses, interpreted the data, drafted the initial manuscript, and critically revised the manuscript; Dr Grigorenko conceptualized and designed the study, supervised data collection, performed data analyses, interpreted the data, drafted the initial manuscript, and critically revised the manuscript; and all authors approved the final manuscript as submitted.</p></div><div>Contributed by <p id="__p55" class="p p-first-last">Grantees undertaking such projects are encouraged to express freely their professional judgment. This article, therefore, does not necessarily reflect the position or policies of the funding agencies, and no official endorsement should be inferred. The National Institutes of Health and the National Science Foundation were not involved in the study design, data collection and analysis, interpretation of findings, writing of this report, or the decision to submit the manuscript for publication.</p></div><p class="fn sec" id="fn14"></p><p id="__p56" class="p p-first-last"><strong>FINANCIAL DISCLOSURE:</strong> The authors have indicated they have no financial relationships relevant to this article to disclose. </p><p></p><p class="fn sec" id="fn15"></p><p id="__p57" class="p p-first-last"><strong>FUNDING:</strong> Supported by National Institute of Health grants R01 DC007665 (Dr Grigorenko, Principal Investigator) and P50 HD052120 (Richard Wagner, Principal Investigator), NIH Centers for Mendelian Genomics (5U54HG006504), National Science Foundation Integrative Graduate Education and Research Traineeship grant 114399 (Dr Magnuson, Principal Investigator), and grant 14.Z50.31.0027 from the Government of the Russian Federation (Dr Grigorenko, Principal Investigator). Funded by the National Institutes of Health (NIH).</p><p></p><p class="fn sec" id="fn16"> </p><p id="__p58" class="p p-first-last"><strong>POTENTIAL CONFLICT OF INTEREST:</strong> The authors have indicated they have no potential conflicts of interest to disclose. </p><p></p></div></div><div id="idm140598289362128" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140598289362128title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="mixed-citation">Law J, Boyle J, Harris F, Harkness A, Nye C.. <span class="ref-title">Prevalence and natural history of primary speech and language delay: findings from a systematic review of the literature.</span>
<span class="ref-journal">Int J Lang Commun Disord</span>. 2000;<span class="ref-vol">35</span>(<span class="ref-iss">2</span>):165–188 [<a href="/pubmed/10912250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Lang+Commun+Disord&amp;title=Prevalence+and+natural+history+of+primary+speech+and+language+delay:+findings+from+a+systematic+review+of+the+literature.&amp;author=J+Law&amp;author=J+Boyle&amp;author=F+Harris&amp;author=A+Harkness&amp;author=C+Nye&amp;volume=35&amp;issue=2&amp;publication_year=2000&amp;pages=165-188&amp;pmid=10912250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="mixed-citation">Tomblin JB, Records NL, Buckwalter P, Zhang X, Smith E, O’Brien M.. <span class="ref-title">Prevalence of specific language impairment in kindergarten children.</span>
<span class="ref-journal">J Speech Lang Hear Res</span>. 1997;<span class="ref-vol">40</span>(<span class="ref-iss">6</span>):1245–1260 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5075245/">PMC free article</a>]</span> [<a href="/pubmed/9430746" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Speech+Lang+Hear+Res&amp;title=Prevalence+of+specific+language+impairment+in+kindergarten+children.&amp;author=JB+Tomblin&amp;author=NL+Records&amp;author=P+Buckwalter&amp;author=X+Zhang&amp;author=E+Smith&amp;volume=40&amp;issue=6&amp;publication_year=1997&amp;pages=1245-1260&amp;pmid=9430746&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="mixed-citation">American Psychiatric Association 
<span class="ref-journal">Diagnostic and Statistical Manual of Mental Disorders</span>. 5th ed.
Arlington, VA: American Psychiatric Publishing; 2013 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Diagnostic+and+Statistical+Manual+of+Mental+Disorders&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="mixed-citation">Clegg J, Hollis C, Mawhood L, Rutter M.. <span class="ref-title">Developmental language disorders—a follow-up in later adult life. Cognitive, language and psychosocial outcomes.</span>
<span class="ref-journal">J Child Psychol Psychiatry</span>. 2005;<span class="ref-vol">46</span>(<span class="ref-iss">2</span>):128–149 [<a href="/pubmed/15679523" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Child+Psychol+Psychiatry&amp;title=Developmental+language+disorders—a+follow-up+in+later+adult+life.+Cognitive,+language+and+psychosocial+outcomes.&amp;author=J+Clegg&amp;author=C+Hollis&amp;author=L+Mawhood&amp;author=M+Rutter&amp;volume=46&amp;issue=2&amp;publication_year=2005&amp;pages=128-149&amp;pmid=15679523&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="mixed-citation">McGrath LM, Hutaff-Lee C, Scott A, Boada R, Shriberg LD, Pennington BF. Children with comorbid speech sound disorder and specific language impairment are at increased risk for attention-deficit/hyperactivity disorder. <em>J Abnorm Child Psychol</em>
2008;36(2):151–163 [<a href="/pubmed/17882543" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="mixed-citation">Im-Bolter N, Cohen NJ. Language impairment and psychiatric comorbidities. <em>Pediatr Clin North Am</em>
2007;54(3):525–542 [<a href="/pubmed/17543908" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="mixed-citation">Snowling MJ, Adams JW, Bishop DV, Stothard SE.. <span class="ref-title">Educational attainments of school leavers with a preschool history of speech-language impairments.</span>
<span class="ref-journal">Int J Lang Commun Disord</span>. 2001;<span class="ref-vol">36</span>(<span class="ref-iss">2</span>):173–183 [<a href="/pubmed/11344593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Lang+Commun+Disord&amp;title=Educational+attainments+of+school+leavers+with+a+preschool+history+of+speech-language+impairments.&amp;author=MJ+Snowling&amp;author=JW+Adams&amp;author=DV+Bishop&amp;author=SE+Stothard&amp;volume=36&amp;issue=2&amp;publication_year=2001&amp;pages=173-183&amp;pmid=11344593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="mixed-citation">Durkin K, Conti-Ramsden G.
<span class="ref-title">Young people with specific language impairment: a review of social and emotional functioning in adolescence.</span>
<span class="ref-journal">Child Lang Teach Ther</span>. 2010;<span class="ref-vol">26</span>(<span class="ref-iss">2</span>):105–121 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Child+Lang+Teach+Ther&amp;title=Young+people+with+specific+language+impairment:+a+review+of+social+and+emotional+functioning+in+adolescence.&amp;author=K+Durkin&amp;author=G+Conti-Ramsden&amp;volume=26&amp;issue=2&amp;publication_year=2010&amp;pages=105-121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="mixed-citation">Stromswold K.
<span class="ref-title">The heritability of language: a review and metaanalysis of twin, adoption, and linkage studies.</span>
<span class="ref-journal">Language</span>. 2001;<span class="ref-vol">77</span>(<span class="ref-iss">4</span>):647–723 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Language&amp;title=The+heritability+of+language:+a+review+and+metaanalysis+of+twin,+adoption,+and+linkage+studies.&amp;author=K+Stromswold&amp;volume=77&amp;issue=4&amp;publication_year=2001&amp;pages=647-723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="mixed-citation">Lai CS, Fisher SE, Hurst JA, Vargha-Khadem F, Monaco AP.. <span class="ref-title">A forkhead-domain gene is mutated in a severe speech and language disorder.</span>
<span class="ref-journal">Nature</span>. 2001;<span class="ref-vol">413</span>(<span class="ref-iss">6855</span>):519–523 [<a href="/pubmed/11586359" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+forkhead-domain+gene+is+mutated+in+a+severe+speech+and+language+disorder.&amp;author=CS+Lai&amp;author=SE+Fisher&amp;author=JA+Hurst&amp;author=F+Vargha-Khadem&amp;author=AP+Monaco&amp;volume=413&amp;issue=6855&amp;publication_year=2001&amp;pages=519-523&amp;pmid=11586359&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="mixed-citation">Palka C, Alfonsi M, Mohn A, et al.  Mosaic 7q31 deletion involving FOXP2 gene associated with language impairment. <em>Pediatrics</em>
2012;129(1). Available at: <a href="http://www.pediatrics.org/cgi/content/full/129/1/e183" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">www.pediatrics.org/cgi/content/full/129/1/e183</a> [<a href="/pubmed/22144704" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="mixed-citation">MacDermot KD, Bonora E, Sykes N, et al. . <span class="ref-title">Identification of FOXP2 truncation as a novel cause of developmental speech and language deficits.</span>
<span class="ref-journal">Am J Hum Genet</span>. 2005;<span class="ref-vol">76</span>(<span class="ref-iss">6</span>):1074–1080 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1196445/">PMC free article</a>]</span> [<a href="/pubmed/15877281" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Identification+of+FOXP2+truncation+as+a+novel+cause+of+developmental+speech+and+language+deficits.&amp;author=KD+MacDermot&amp;author=E+Bonora&amp;author=N+Sykes&amp;volume=76&amp;issue=6&amp;publication_year=2005&amp;pages=1074-1080&amp;pmid=15877281&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="mixed-citation">Newbury DF, Bonora E, Lamb JA, et al. ; International Molecular Genetic Study of Autism Consortium .. <span class="ref-title">FOXP2 is not a major susceptibility gene for autism or specific language impairment.</span>
<span class="ref-journal">Am J Hum Genet</span>. 2002;<span class="ref-vol">70</span>(<span class="ref-iss">5</span>):1318–1327 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC447606/">PMC free article</a>]</span> [<a href="/pubmed/11894222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=FOXP2+is+not+a+major+susceptibility+gene+for+autism+or+specific+language+impairment.&amp;author=DF+Newbury&amp;author=E+Bonora&amp;author=JA+Lamb&amp;volume=70&amp;issue=5&amp;publication_year=2002&amp;pages=1318-1327&amp;pmid=11894222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="mixed-citation">SLI Consortium . <span class="ref-title">A genomewide scan identifies two novel loci involved in specific language impairment.</span>
<span class="ref-journal">Am J Hum Genet</span>. 2002;<span class="ref-vol">70</span>(<span class="ref-iss">2</span>):384–398 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384915/">PMC free article</a>]</span> [<a href="/pubmed/11791209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genomewide+scan+identifies+two+novel+loci+involved+in+specific+language+impairment.&amp;volume=70&amp;issue=2&amp;publication_year=2002&amp;pages=384-398&amp;pmid=11791209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="mixed-citation">Bartlett CW, Flax JF, Logue MW, et al. . <span class="ref-title">A major susceptibility locus for specific language impairment is located on 13q21.</span>
<span class="ref-journal">Am J Hum Genet</span>. 2002;<span class="ref-vol">71</span>(<span class="ref-iss">1</span>):45–55 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384992/">PMC free article</a>]</span> [<a href="/pubmed/12048648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+major+susceptibility+locus+for+specific+language+impairment+is+located+on+13q21.&amp;author=CW+Bartlett&amp;author=JF+Flax&amp;author=MW+Logue&amp;volume=71&amp;issue=1&amp;publication_year=2002&amp;pages=45-55&amp;pmid=12048648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="mixed-citation">Vernes SC, Newbury DF, Abrahams BS, et al. . <span class="ref-title">A functional genetic link between distinct developmental language disorders.</span>
<span class="ref-journal">N Engl J Med</span>. 2008;<span class="ref-vol">359</span>(<span class="ref-iss">22</span>):2337–2345 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756409/">PMC free article</a>]</span> [<a href="/pubmed/18987363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=A+functional+genetic+link+between+distinct+developmental+language+disorders.&amp;author=SC+Vernes&amp;author=DF+Newbury&amp;author=BS+Abrahams&amp;volume=359&amp;issue=22&amp;publication_year=2008&amp;pages=2337-2345&amp;pmid=18987363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="mixed-citation">Newbury DF, Winchester L, Addis L, et al. . <span class="ref-title">CMIP and ATP2C2 modulate phonological short-term memory in language impairment.</span>
<span class="ref-journal">Am J Hum Genet</span>. 2009;<span class="ref-vol">85</span>(<span class="ref-iss">2</span>):264–272 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2725236/">PMC free article</a>]</span> [<a href="/pubmed/19646677" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=CMIP+and+ATP2C2+modulate+phonological+short-term+memory+in+language+impairment.&amp;author=DF+Newbury&amp;author=L+Winchester&amp;author=L+Addis&amp;volume=85&amp;issue=2&amp;publication_year=2009&amp;pages=264-272&amp;pmid=19646677&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="mixed-citation">Eicher JD, Powers NR, Miller LL, et al. ; Pediatric Imaging, Neurocognition, and Genetics Study .. <span class="ref-title">Genome-wide association study of shared components of reading disability and language impairment.</span>
<span class="ref-journal">Genes Brain Behav</span>. 2013;<span class="ref-vol">12</span>(<span class="ref-iss">8</span>):792–801 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3904347/">PMC free article</a>]</span> [<a href="/pubmed/24024963" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Brain+Behav&amp;title=Genome-wide+association+study+of+shared+components+of+reading+disability+and+language+impairment.&amp;author=JD+Eicher&amp;author=NR+Powers&amp;author=LL+Miller&amp;volume=12&amp;issue=8&amp;publication_year=2013&amp;pages=792-801&amp;pmid=24024963&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="mixed-citation">Luciano M, Evans DM, Hansell NK, et al. . <span class="ref-title">A genome-wide association study for reading and language abilities in two population cohorts.</span>
<span class="ref-journal">Genes Brain Behav</span>. 2013;<span class="ref-vol">12</span>(<span class="ref-iss">6</span>):645–652 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3908370/">PMC free article</a>]</span> [<a href="/pubmed/23738518" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Brain+Behav&amp;title=A+genome-wide+association+study+for+reading+and+language+abilities+in+two+population+cohorts.&amp;author=M+Luciano&amp;author=DM+Evans&amp;author=NK+Hansell&amp;volume=12&amp;issue=6&amp;publication_year=2013&amp;pages=645-652&amp;pmid=23738518&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="mixed-citation">Nudel R, Simpson NH, Baird G, et al. ; SLI Consortium .. <span class="ref-title">Genome-wide association analyses of child genotype effects and parent-of-origin effects in specific language impairment.</span>
<span class="ref-journal">Genes Brain Behav</span>. 2014;<span class="ref-vol">13</span>(<span class="ref-iss">4</span>):418–429 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4114547/">PMC free article</a>]</span> [<a href="/pubmed/24571439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Brain+Behav&amp;title=Genome-wide+association+analyses+of+child+genotype+effects+and+parent-of-origin+effects+in+specific+language+impairment.&amp;author=R+Nudel&amp;author=NH+Simpson&amp;author=G+Baird&amp;volume=13&amp;issue=4&amp;publication_year=2014&amp;pages=418-429&amp;pmid=24571439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="mixed-citation">Gialluisi A, Newbury DF, Wilcutt EG, et al. ; SLI Consortium .. <span class="ref-title">Genome-wide screening for DNA variants associated with reading and language traits.</span>
<span class="ref-journal">Genes Brain Behav</span>. 2014;<span class="ref-vol">13</span>(<span class="ref-iss">7</span>):686–701 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4165772/">PMC free article</a>]</span> [<a href="/pubmed/25065397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Brain+Behav&amp;title=Genome-wide+screening+for+DNA+variants+associated+with+reading+and+language+traits.&amp;author=A+Gialluisi&amp;author=DF+Newbury&amp;author=EG+Wilcutt&amp;volume=13&amp;issue=7&amp;publication_year=2014&amp;pages=686-701&amp;pmid=25065397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="mixed-citation">Villanueva P, Nudel R, Hoischen A, et al. ; SLI Consortium .. <span class="ref-title">Exome sequencing in an admixed isolated population indicates NFXL1 variants confer a risk for specific language impairment [published correction appears in <em>PLoS Genet</em></span>. 2015;11(6):e1005336]<span class="ref-title">.</span>
<span class="ref-journal">PLoS Genet</span>. 2015;<span class="ref-vol">11</span>(<span class="ref-iss">3</span>):e1004925. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4363375/">PMC free article</a>]</span> [<a href="/pubmed/25781923" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Exome+sequencing+in+an+admixed+isolated+population+indicates+NFXL1+variants+confer+a+risk+for+specific+language+impairment+[published+correction+appears+in+PLoS+Genet&amp;title=.&amp;author=P+Villanueva&amp;author=R+Nudel&amp;author=A+Hoischen&amp;volume=11&amp;issue=3&amp;publication_year=2015&amp;pages=e1004925&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="mixed-citation">Villanueva P, Newbury DF, Jara L, et al.  Genome-wide analysis of genetic susceptibility to language impairment in an isolated Chilean population. <em>Eur J Hum Genet</em>
2011;19(6):687–695 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3110042/">PMC free article</a>]</span> [<a href="/pubmed/21248734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="mixed-citation">Shifman S, Darvasi A.. <span class="ref-title">The value of isolated populations.</span>
<span class="ref-journal">Nat Genet</span>. 2001;<span class="ref-vol">28</span>(<span class="ref-iss">4</span>):309–310 [<a href="/pubmed/11479587" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+value+of+isolated+populations.&amp;author=S+Shifman&amp;author=A+Darvasi&amp;volume=28&amp;issue=4&amp;publication_year=2001&amp;pages=309-310&amp;pmid=11479587&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="mixed-citation">Rakhlin N, Kornilov SA, Palejev D, Koposov RA, Chang JT, Grigorenko EL.
<span class="ref-title">The language phenotype of a small geographically isolated Russian-speaking population: implications for genetic and clinical studies of developmental language disorder.</span>
<span class="ref-journal">Appl Psycholinguist</span>. 2013;<span class="ref-vol">34</span>(<span class="ref-iss">5</span>):971–1003
 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl+Psycholinguist&amp;title=The+language+phenotype+of+a+small+geographically+isolated+Russian-speaking+population:+implications+for+genetic+and+clinical+studies+of+developmental+language+disorder.&amp;author=N+Rakhlin&amp;author=SA+Kornilov&amp;author=D+Palejev&amp;author=RA+Koposov&amp;author=JT+Chang&amp;volume=34&amp;issue=5&amp;publication_year=2013&amp;pages=971-1003&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="mixed-citation">S.A.G.E., Statistical Analysis for Genetic Epidemiology [computer program]. Version Release 6.3; 2012. Available at: <a href="http://darwin.cwru.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://darwin.cwru.edu</a>. Accessed February 1, 2014</span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="mixed-citation">Zhou X, Stephens M.. <span class="ref-title">Genome-wide efficient mixed-model analysis for association studies.</span>
<span class="ref-journal">Nat Genet</span>. 2012;<span class="ref-vol">44</span>(<span class="ref-iss">7</span>):821–824 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3386377/">PMC free article</a>]</span> [<a href="/pubmed/22706312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+efficient+mixed-model+analysis+for+association+studies.&amp;author=X+Zhou&amp;author=M+Stephens&amp;volume=44&amp;issue=7&amp;publication_year=2012&amp;pages=821-824&amp;pmid=22706312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="mixed-citation">Li D, Zhou J, Thomas DC, Fardo DW. Complex pedigrees in the sequencing era: to track transmission or decorrelate? <em>Genet Epidemiol</em>
2014;38(suppl 1):S29–S36 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4272198/">PMC free article</a>]</span> [<a href="/pubmed/25112185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="mixed-citation">Li MX, Kwan JS, Sham PC.. <span class="ref-title">HYST: a hybrid set-based test for genome-wide association studies, with application to protein-protein interaction-based association analysis.</span>
<span class="ref-journal">Am J Hum Genet</span>. 2012;<span class="ref-vol">91</span>(<span class="ref-iss">3</span>):478–488 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3511992/">PMC free article</a>]</span> [<a href="/pubmed/22958900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=HYST:+a+hybrid+set-based+test+for+genome-wide+association+studies,+with+application+to+protein-protein+interaction-based+association+analysis.&amp;author=MX+Li&amp;author=JS+Kwan&amp;author=PC+Sham&amp;volume=91&amp;issue=3&amp;publication_year=2012&amp;pages=478-488&amp;pmid=22958900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="mixed-citation">Ionita-Laza I, Perry GH, Raby BA, et al. . <span class="ref-title">On the analysis of copy-number variations in genome-wide association studies: a translation of the family-based association test.</span>
<span class="ref-journal">Genet Epidemiol</span>. 2008;<span class="ref-vol">32</span>(<span class="ref-iss">3</span>):273–284 [<a href="/pubmed/18228561" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=On+the+analysis+of+copy-number+variations+in+genome-wide+association+studies:+a+translation+of+the+family-based+association+test.&amp;author=I+Ionita-Laza&amp;author=GH+Perry&amp;author=BA+Raby&amp;volume=32&amp;issue=3&amp;publication_year=2008&amp;pages=273-284&amp;pmid=18228561&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="mixed-citation">Sifrim A, Popovic D, Tranchevent LC, et al.  eXtasy: variant prioritization by genomic data fusion. <em>Nat Meth</em>
2013;10(11):1083–1084 [<a href="/pubmed/24076761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="mixed-citation">Suzuki Y, Yang X, Aoki Y, Kure S, Matsubara Y.. <span class="ref-title">Mutations in the holocarboxylase synthetase gene HLCS.</span>
<span class="ref-journal">Hum Mutat</span>. 2005;<span class="ref-vol">26</span>(<span class="ref-iss">4</span>):285–290 [<a href="/pubmed/16134170" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mutat&amp;title=Mutations+in+the+holocarboxylase+synthetase+gene+HLCS.&amp;author=Y+Suzuki&amp;author=X+Yang&amp;author=Y+Aoki&amp;author=S+Kure&amp;author=Y+Matsubara&amp;volume=26&amp;issue=4&amp;publication_year=2005&amp;pages=285-290&amp;pmid=16134170&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="mixed-citation">Zhao Y, Zhao F, Zong L, et al. . <span class="ref-title">Exome sequencing and linkage analysis identified tenascin-C (TNC) as a novel causative gene in nonsyndromic hearing loss.</span>
<span class="ref-journal">PLoS One</span>. 2013;<span class="ref-vol">8</span>(<span class="ref-iss">7</span>):e69549. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3728356/">PMC free article</a>]</span> [<a href="/pubmed/23936043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Exome+sequencing+and+linkage+analysis+identified+tenascin-C+(TNC)+as+a+novel+causative+gene+in+nonsyndromic+hearing+loss.&amp;author=Y+Zhao&amp;author=F+Zhao&amp;author=L+Zong&amp;volume=8&amp;issue=7&amp;publication_year=2013&amp;pages=e69549&amp;pmid=23936043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="mixed-citation">Ripke S, O'Dushlaine C, Chambert K, et al.  Genome-wide association analysis identifies 13 new risk loci for schizophrenia. <em>Nat Genet</em>
2013;45(10):1150–1159 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3827979/">PMC free article</a>]</span> [<a href="/pubmed/23974872" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="mixed-citation">Prados J, Stenz L, Courtet P, et al.  Borderline personality disorder and childhood maltreatment: a genome-wide methylation analysis. <em>Genes Brain Behav</em>
2015;<span class="ref-vol">14</span>(<span class="ref-iss">2</span>):177–188 [<a href="/pubmed/25612291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Prados+J+,++Stenz+L+,++Courtet+P+,+++Borderline+personality+disorder+and+childhood+maltreatment:+a+genome-wide+methylation+analysis.++Genes+Brain+Behav+
+2015+;+14+(+2+):+177+–+188+" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="mixed-citation">Wu H, Li D, Shan Y, et al. . <span class="ref-title">EFCBP1/NECAB1, a brain-specifically expressed gene with highest abundance in temporal lobe, encodes a protein containing EF-hand and antibiotic biosynthesis monooxygenase domains.</span>
<span class="ref-journal">DNA Seq</span>. 2007;<span class="ref-vol">18</span>(<span class="ref-iss">1</span>):73–79 [<a href="/pubmed/17364817" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Seq&amp;title=EFCBP1/NECAB1,+a+brain-specifically+expressed+gene+with+highest+abundance+in+temporal+lobe,+encodes+a+protein+containing+EF-hand+and+antibiotic+biosynthesis+monooxygenase+domains.&amp;author=H+Wu&amp;author=D+Li&amp;author=Y+Shan&amp;volume=18&amp;issue=1&amp;publication_year=2007&amp;pages=73-79&amp;pmid=17364817&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="mixed-citation">Pan L, North HA, Sahni V, et al. . <span class="ref-title">β1-Integrin and integrin linked kinase regulate astrocytic differentiation of neural stem cells.</span>
<span class="ref-journal">PLoS One</span>. 2014;<span class="ref-vol">9</span>(<span class="ref-iss">8</span>):e104335. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4123915/">PMC free article</a>]</span> [<a href="/pubmed/25098415" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=β1-Integrin+and+integrin+linked+kinase+regulate+astrocytic+differentiation+of+neural+stem+cells.&amp;author=L+Pan&amp;author=HA+North&amp;author=V+Sahni&amp;volume=9&amp;issue=8&amp;publication_year=2014&amp;pages=e104335&amp;pmid=25098415&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="mixed-citation">Mills J, Digicaylioglu M, Legg AT, et al.  Role of integrin-linked kinase in nerve growth factor-stimulated neurite outgrowth. <em>J Neurosci</em>
2003;23(5):1638–1648 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6741947/">PMC free article</a>]</span> [<a href="/pubmed/12629168" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="mixed-citation">Zhang J, Woodhead GJ, Swaminathan SK, et al.  Cortical neural precursors inhibit their own differentiation via N-cadherin maintenance of β-catenin signaling. <em>Dev Cell</em>
2010;18(3):472–479 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2865854/">PMC free article</a>]</span> [<a href="/pubmed/20230753" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="mixed-citation">Kadowaki M, Nakamura S, Machon O, Krauss S, Radice GL, Takeichi M. N-cadherin mediates cortical organization in the mouse brain. <em>Dev Biol</em>
2007;304(1):22–33 [<a href="/pubmed/17222817" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="mixed-citation">Zhong Z, Qiu J, Chen X, et al.  Identification of TCP10L as primate-specific gene derived via segmental duplication and homodimerization of TCP10L through the leucine zipper motif. <em>Mol Biol Rep</em>
2008;35(2):171–178 [<a href="/pubmed/17377852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="mixed-citation">Lai YJ, Li MY, Yang CY, Huang KH, Tsai JC, Wang TW.. <span class="ref-title">TRIP6 regulates neural stem cell maintenance in the postnatal mammalian subventricular zone.</span>
<span class="ref-journal">Dev Dyn</span>. 2014;<span class="ref-vol">243</span>(<span class="ref-iss">9</span>):1130–1142 [<a href="/pubmed/25044744" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Dyn&amp;title=TRIP6+regulates+neural+stem+cell+maintenance+in+the+postnatal+mammalian+subventricular+zone.&amp;author=YJ+Lai&amp;author=MY+Li&amp;author=CY+Yang&amp;author=KH+Huang&amp;author=JC+Tsai&amp;volume=243&amp;issue=9&amp;publication_year=2014&amp;pages=1130-1142&amp;pmid=25044744&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="mixed-citation">Ryan TA. A pre-synaptic to-do list for coupling exocytosis to endocytosis. <em>Curr Opin Cell Biol</em>
2006;18(4):416–421 [<a href="/pubmed/16806881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="mixed-citation">Chen EY, Tan CM, Kou Y, et al. . <span class="ref-title">Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool.</span>
<span class="ref-journal">BMC Bioinformatics</span>. 2013;<span class="ref-vol">14</span>:128. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3637064/">PMC free article</a>]</span> [<a href="/pubmed/23586463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=Enrichr:+interactive+and+collaborative+HTML5+gene+list+enrichment+analysis+tool.&amp;author=EY+Chen&amp;author=CM+Tan&amp;author=Y+Kou&amp;volume=14&amp;publication_year=2013&amp;pages=128&amp;pmid=23586463&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45">45. <span class="mixed-citation">Kang HM, Sul JH, Service SK, et al. . <span class="ref-title">Variance component model to account for sample structure in genome-wide association studies.</span>
<span class="ref-journal">Nat Genet</span>. 2010;<span class="ref-vol">42</span>(<span class="ref-iss">4</span>):348–354 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3092069/">PMC free article</a>]</span> [<a href="/pubmed/20208533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variance+component+model+to+account+for+sample+structure+in+genome-wide+association+studies.&amp;author=HM+Kang&amp;author=JH+Sul&amp;author=SK+Service&amp;volume=42&amp;issue=4&amp;publication_year=2010&amp;pages=348-354&amp;pmid=20208533&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46">46. <span class="mixed-citation">Ramasamy A, Trabzuni D, Guelfi S, et al. ; UK Brain Expression Consortium; North American Brain Expression Consortium .. <span class="ref-title">Genetic variability in the regulation of gene expression in ten regions of the human brain.</span>
<span class="ref-journal">Nat Neurosci</span>. 2014;<span class="ref-vol">17</span>(<span class="ref-iss">10</span>):1418–1428 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4208299/">PMC free article</a>]</span> [<a href="/pubmed/25174004" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=Genetic+variability+in+the+regulation+of+gene+expression+in+ten+regions+of+the+human+brain.&amp;author=A+Ramasamy&amp;author=D+Trabzuni&amp;author=S+Guelfi&amp;volume=17&amp;issue=10&amp;publication_year=2014&amp;pages=1418-1428&amp;pmid=25174004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47">47. <span class="mixed-citation">Filges I, Shimojima K, Okamoto N, et al. . <span class="ref-title">Reduced expression by SETBP1 haploinsufficiency causes developmental and expressive language delay indicating a phenotype distinct from Schinzel-Giedion syndrome.</span>
<span class="ref-journal">J Med Genet</span>. 2011;<span class="ref-vol">48</span>(<span class="ref-iss">2</span>):117–122 [<a href="/pubmed/21037274" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Reduced+expression+by+SETBP1+haploinsufficiency+causes+developmental+and+expressive+language+delay+indicating+a+phenotype+distinct+from+Schinzel-Giedion+syndrome.&amp;author=I+Filges&amp;author=K+Shimojima&amp;author=N+Okamoto&amp;volume=48&amp;issue=2&amp;publication_year=2011&amp;pages=117-122&amp;pmid=21037274&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B48">48. <span class="mixed-citation">Marseglia G, Scordo MR, Pescucci C, et al.  372 Kb microdeletion in 18q12.3 causing SETBP1 haploinsufficiency associated with mild mental retardation and expressive speech impairment. <em>Eur J Med Genet</em>
2012;55(3):216–221 [<a href="/pubmed/22333924" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B49">49. <span class="mixed-citation">Bouquillon S, Andrieux J, Landais E, et al.  A 5.3 Mb deletion in chromosome 18q12.3 as the smallest region of overlap in two patients with expressive speech delay. <em>Eur J Med Genet</em>
2011;54(2):194–197 [<a href="/pubmed/21145994" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B50">50. <span class="mixed-citation">Coe BP, Witherspoon K, Rosenfeld JA, et al. . <span class="ref-title">Refining analyses of copy number variation identifies specific genes associated with developmental delay.</span>
<span class="ref-journal">Nat Genet</span>. 2014;<span class="ref-vol">46</span>(<span class="ref-iss">10</span>):1063–1071 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4177294/">PMC free article</a>]</span> [<a href="/pubmed/25217958" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Refining+analyses+of+copy+number+variation+identifies+specific+genes+associated+with+developmental+delay.&amp;author=BP+Coe&amp;author=K+Witherspoon&amp;author=JA+Rosenfeld&amp;volume=46&amp;issue=10&amp;publication_year=2014&amp;pages=1063-1071&amp;pmid=25217958&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B51">51. <span class="mixed-citation">Irintchev A, Rollenhagen A, Troncoso E, Kiss JZ, Schachner M.. <span class="ref-title">Structural and functional aberrations in the cerebral cortex of tenascin-C deficient mice.</span>
<span class="ref-journal">Cereb Cortex</span>. 2005;<span class="ref-vol">15</span>(<span class="ref-iss">7</span>):950–962 [<a href="/pubmed/15537675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cereb+Cortex&amp;title=Structural+and+functional+aberrations+in+the+cerebral+cortex+of+tenascin-C+deficient+mice.&amp;author=A+Irintchev&amp;author=A+Rollenhagen&amp;author=E+Troncoso&amp;author=JZ+Kiss&amp;author=M+Schachner&amp;volume=15&amp;issue=7&amp;publication_year=2005&amp;pages=950-962&amp;pmid=15537675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B52">52. <span class="mixed-citation">Maggi A, Ciana P, Belcredito S, Vegeto E.. <span class="ref-title">Estrogens in the nervous system: mechanisms and nonreproductive functions.</span>
<span class="ref-journal">Annu Rev Physiol</span>. 2004;<span class="ref-vol">66</span>:291–313 [<a href="/pubmed/14977405" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Physiol&amp;title=Estrogens+in+the+nervous+system:+mechanisms+and+nonreproductive+functions.&amp;author=A+Maggi&amp;author=P+Ciana&amp;author=S+Belcredito&amp;author=E+Vegeto&amp;volume=66&amp;publication_year=2004&amp;pages=291-313&amp;pmid=14977405&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B53">53. <span class="mixed-citation">Schaadt G, Hesse V, Friederici AD.. <span class="ref-title">Sex hormones in early infancy seem to predict aspects of later language development.</span>
<span class="ref-journal">Brain Lang</span>. 2015;<span class="ref-vol">141</span>:70–76 [<a href="/pubmed/25540858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Lang&amp;title=Sex+hormones+in+early+infancy+seem+to+predict+aspects+of+later+language+development.&amp;author=G+Schaadt&amp;author=V+Hesse&amp;author=AD+Friederici&amp;volume=141&amp;publication_year=2015&amp;pages=70-76&amp;pmid=25540858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B54">54. <span class="mixed-citation">Lyons GE, Micales BK, Schwarz J, Martin JF, Olson EN.. <span class="ref-title">Expression of mef2 genes in the mouse central nervous system suggests a role in neuronal maturation.</span>
<span class="ref-journal">J Neurosci</span>. 1995;<span class="ref-vol">15</span>(<span class="ref-iss">8</span>):5727–5738 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6577647/">PMC free article</a>]</span> [<a href="/pubmed/7643214" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Expression+of+mef2+genes+in+the+mouse+central+nervous+system+suggests+a+role+in+neuronal+maturation.&amp;author=GE+Lyons&amp;author=BK+Micales&amp;author=J+Schwarz&amp;author=JF+Martin&amp;author=EN+Olson&amp;volume=15&amp;issue=8&amp;publication_year=1995&amp;pages=5727-5738&amp;pmid=7643214&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B55">55. <span class="mixed-citation">Flavell SW, Kim TK, Gray JM, et al. . <span class="ref-title">Genome-wide analysis of MEF2 transcriptional program reveals synaptic target genes and neuronal activity-dependent polyadenylation site selection.</span>
<span class="ref-journal">Neuron</span>. 2008;<span class="ref-vol">60</span>(<span class="ref-iss">6</span>):1022–1038 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2630178/">PMC free article</a>]</span> [<a href="/pubmed/19109909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Genome-wide+analysis+of+MEF2+transcriptional+program+reveals+synaptic+target+genes+and+neuronal+activity-dependent+polyadenylation+site+selection.&amp;author=SW+Flavell&amp;author=TK+Kim&amp;author=JM+Gray&amp;volume=60&amp;issue=6&amp;publication_year=2008&amp;pages=1022-1038&amp;pmid=19109909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B56">56. <span class="mixed-citation">Flavell SW, Cowan CW, Kim TK, et al.  Activity-dependent regulation of MEF2 transcription factors suppresses excitatory synapse number. <em>Science</em>
2006;311(5763):1008-1012 [<a href="/pubmed/16484497" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B57">57. <span class="mixed-citation">Shalizi A, Gaudillière B, Yuan Z, et al.  A calcium-regulated MEF2 sumoylation switch controls postsynaptic differentiation. <em>Science</em>
2006;311(5763):1012–1017 [<a href="/pubmed/16484498" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B58">58. <span class="mixed-citation">Barbosa AC, Kim MS, Ertunc M, et al. 
<span class="ref-title">MEF2C, a transcription factor that facilitates learning and memory by negative regulation of synapse numbers and function.</span>
<span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2008;105(27):9391–9396 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2453723/">PMC free article</a>]</span> [<a href="/pubmed/18599438" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=MEF2C,+a+transcription+factor+that+facilitates+learning+and+memory+by+negative+regulation+of+synapse+numbers+and+function.&amp;author=AC+Barbosa&amp;author=MS+Kim&amp;author=M+Ertunc&amp;publication_year=2008&amp;pages=9391-9396&amp;pmid=18599438&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B59">59. <span class="mixed-citation">Cole CJ, Mercaldo V, Restivo L, et al.  MEF2 negatively regulates learning-induced structural plasticity and memory formation. <em>Nature Neurosci</em>
2012;15(9):1255–1264 [<a href="/pubmed/22885849" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B60">60. <span class="mixed-citation">Kornilov SA, Landi N, Rakhlin N, Fang SY, Grigorenko EL, Magnuson JS.. <span class="ref-title">Attentional but not pre-attentive neural measures of auditory discrimination are atypical in children with developmental language disorder.</span>
<span class="ref-journal">Dev Neuropsychol</span>. 2014;<span class="ref-vol">39</span>(<span class="ref-iss">7</span>):543–567 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4399717/">PMC free article</a>]</span> [<a href="/pubmed/25350759" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Neuropsychol&amp;title=Attentional+but+not+pre-attentive+neural+measures+of+auditory+discrimination+are+atypical+in+children+with+developmental+language+disorder.&amp;author=SA+Kornilov&amp;author=N+Landi&amp;author=N+Rakhlin&amp;author=SY+Fang&amp;author=EL+Grigorenko&amp;volume=39&amp;issue=7&amp;publication_year=2014&amp;pages=543-567&amp;pmid=25350759&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B61">61. <span class="mixed-citation">Gamsiz E D, Viscidi E W, Frederick A M, et al.  Intellectual disability is associated with increased runs of homozygosity in simplex autism. <em>Am J Hum Genet</em>
2013;93(1):103–109 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3710760/">PMC free article</a>]</span> [<a href="/pubmed/23830515" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4811310&amp;issue-id=267367&amp;journal-id=1407&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140598280115248title">Abstract</a></li><li><a href="#s07title">Methods</a></li><li><a href="#s13title">Results</a></li><li><a href="#s19title">Discussion</a></li><li><a href="#s20title">Conclusions</a></li><li><a href="#idm140598276328144title">Acknowledgments</a></li><li><a href="#idm140598282918496title">Glossary</a></li><li><a href="#idm140598280990656title">Footnotes</a></li><li><a href="#idm140598289362128title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Pediatrics</span> are provided here courtesy of <strong>American Academy of Pediatrics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4811310/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4811310/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4811310/pdf/PEDS_20152469.pdf">PDF (1.9M)</a></li> | <li><a href="#" data-citationid="PMC4811310" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4811310%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4811310%2F&amp;text=Genome-Wide%20Association%20and%20Exome%20Sequencing%20Study%20of%20Language%20Disorder%20in%20an%20Isolated%20Population" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4811310%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27016271" data-aiid="4811310" data-aid="4811310" data-iid="267367" data-domainid="1407" data-domain="pediatrics" data-accid="PMC4811310" data-md5="8f7e41a12deecf147e3c4da543e6e885"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:35:30-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D8159DD58FE010000000003690241&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4811310%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=pediatrics&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4811310/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>