<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4321789/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Polygenic Transmission and Complex Neuro developmental Network for Attention Deficit Hyperactivity Disorder: Genome-Wide Association Study of Both Common and Rare Variants</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4321789/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American journal of medical genetics. Part B, Neuropsychiatric genetics : the official publication of the International Society of Psychiatric Genetics"><meta name="citation_title" content="Polygenic Transmission and Complex Neuro developmental Network for Attention Deficit Hyperactivity Disorder: Genome-Wide Association Study of Both Common and Rare Variants"><meta name="citation_authors" content="Li Yang, Benjamin M. Neale, Lu Liu, S. Hong Lee, Naomi R. Wray, Ning Ji, Haimei Li, Qiujin Qian, Dongliang Wang, Jun Li, Stephen V. Faraone, Yufeng Wang, Psychiatric GWAS Consortium: ADHD Subgroup"><meta name="citation_date" content="July 2013"><meta name="citation_issue" content="5"><meta name="citation_volume" content="0"><meta name="citation_firstpage" content="419"><meta name="citation_doi" content="10.1002/ajmg.b.32169"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4321789/?report=abstract"><meta name="citation_pmid" content="23728934"><meta name="DC.Title" content="Polygenic Transmission and Complex Neuro developmental Network for Attention Deficit Hyperactivity Disorder: Genome-Wide Association Study of Both Common and Rare Variants"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Li Yang"><meta name="DC.Contributor" content="Benjamin M. Neale"><meta name="DC.Contributor" content="Lu Liu"><meta name="DC.Contributor" content="S. Hong Lee"><meta name="DC.Contributor" content="Naomi R. Wray"><meta name="DC.Contributor" content="Ning Ji"><meta name="DC.Contributor" content="Haimei Li"><meta name="DC.Contributor" content="Qiujin Qian"><meta name="DC.Contributor" content="Dongliang Wang"><meta name="DC.Contributor" content="Jun Li"><meta name="DC.Contributor" content="Stephen V. Faraone"><meta name="DC.Contributor" content="Yufeng Wang"><meta name="DC.Contributor" content="Psychiatric GWAS Consortium: ADHD Subgroup"><meta name="DC.Date" content="2013 Jul"><meta name="DC.Identifier" content="10.1002/ajmg.b.32169"><meta name="DC.Language" content="en"><meta property="og:title" content="Polygenic Transmission and Complex Neuro developmental Network for Attention Deficit Hyperactivity Disorder: Genome-Wide Association Study of Both Common and Rare Variants"><meta property="og:type" content="article"><meta property="og:description" content="Attention-deficit hyperactivity disorder (ADHD) is a complex polygenic disorder. This study aimed to discover common and rare DNA variants associated with ADHD in a large homogeneous Han Chinese ADHD case–control sample. The sample comprised 1,040 ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4321789/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4321789/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4321789/pdf/nihms-656151.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B5102DD55DB9100000000044002B1.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4321789%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="clinvar" class="last">ClinVar</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4321789</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Med Genet B Neuropsychiatr Genet</a></span>. Author manuscript; available in PMC 2015 Feb 9.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23728934" target="pmc_ext" ref="reftype=publisher&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Am J Med Genet B Neuropsychiatr Genet. 2013 Jul; 0(5): 419–430. </span></a></div><span class="fm-vol-iss-date">Published online 2013 May 31. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fajmg.b.32169" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/ajmg.b.32169</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4321789</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS656151</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23728934">23728934</a></div></div></div></div><h1 class="content-title">Polygenic Transmission and Complex Neuro developmental Network for Attention Deficit Hyperactivity Disorder: Genome-Wide Association Study of Both Common and Rare Variants</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Yang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711988023984" co-class="co-affbox">Li Yang</a>,<sup>#</sup><sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Neale%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711979426304" co-class="co-affbox">Benjamin M. Neale</a>,<sup>#</sup><sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Liu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711977568768" co-class="co-affbox">Lu Liu</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Lee%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711986922368" co-class="co-affbox">S. Hong Lee</a>,<sup>5</sup> <a href="/pubmed/?term=Wray%20NR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711986920384" co-class="co-affbox">Naomi R. Wray</a>,<sup>5</sup> <a href="/pubmed/?term=Ji%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711984333712" co-class="co-affbox">Ning Ji</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Li%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711986509824" co-class="co-affbox">Haimei Li</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Qian%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140712020351040" co-class="co-affbox">Qiujin Qian</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Wang%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711980396496" co-class="co-affbox">Dongliang Wang</a>,<sup>6</sup> <a href="/pubmed/?term=Li%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711982875728" co-class="co-affbox">Jun Li</a>,<sup>7</sup> <a href="/pubmed/?term=Faraone%20SV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711986593120" co-class="co-affbox">Stephen V. Faraone</a>,<sup>6,</sup><sup>**</sup> <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934" class="affpopup" co-rid="_co_idm140711975376048" co-class="co-affbox">Yufeng Wang</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> and  Psychiatric GWAS Consortium: ADHD Subgroup<sup>†</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140711988023984"><h3 class="no_margin">Li Yang</h3><p><sup>1</sup>Peking University Sixth Hospital/Institute of Mental Health, Beijing, China</p><p><sup>2</sup>Key Laboratory of Mental Health, Ministry of Health, China</p><div>Find articles by <a href="/pubmed/?term=Yang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Li Yang</a></div></div><div id="_co_idm140711979426304"><h3 class="no_margin">Benjamin M. Neale</h3><p><sup>3</sup>Broad Institute of Harvard and MIT, Cambridge, Massachusetts</p><p><sup>4</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts</p><div>Find articles by <a href="/pubmed/?term=Neale%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Benjamin M. Neale</a></div></div><div id="_co_idm140711977568768"><h3 class="no_margin">Lu Liu</h3><p><sup>1</sup>Peking University Sixth Hospital/Institute of Mental Health, Beijing, China</p><p><sup>2</sup>Key Laboratory of Mental Health, Ministry of Health, China</p><div>Find articles by <a href="/pubmed/?term=Liu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Lu Liu</a></div></div><div id="_co_idm140711986922368"><h3 class="no_margin">S. Hong Lee</h3><p><sup>5</sup>The University of Queensland, Queensland Brain Institute, Brisbane, Queensland, Australia</p><div>Find articles by <a href="/pubmed/?term=Lee%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">S. Hong Lee</a></div></div><div id="_co_idm140711986920384"><h3 class="no_margin">Naomi R. Wray</h3><p><sup>5</sup>The University of Queensland, Queensland Brain Institute, Brisbane, Queensland, Australia</p><div>Find articles by <a href="/pubmed/?term=Wray%20NR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Naomi R. Wray</a></div></div><div id="_co_idm140711984333712"><h3 class="no_margin">Ning Ji</h3><p><sup>1</sup>Peking University Sixth Hospital/Institute of Mental Health, Beijing, China</p><p><sup>2</sup>Key Laboratory of Mental Health, Ministry of Health, China</p><div>Find articles by <a href="/pubmed/?term=Ji%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Ning Ji</a></div></div><div id="_co_idm140711986509824"><h3 class="no_margin">Haimei Li</h3><p><sup>1</sup>Peking University Sixth Hospital/Institute of Mental Health, Beijing, China</p><p><sup>2</sup>Key Laboratory of Mental Health, Ministry of Health, China</p><div>Find articles by <a href="/pubmed/?term=Li%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Haimei Li</a></div></div><div id="_co_idm140712020351040"><h3 class="no_margin">Qiujin Qian</h3><p><sup>1</sup>Peking University Sixth Hospital/Institute of Mental Health, Beijing, China</p><p><sup>2</sup>Key Laboratory of Mental Health, Ministry of Health, China</p><div>Find articles by <a href="/pubmed/?term=Qian%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Qiujin Qian</a></div></div><div id="_co_idm140711980396496"><h3 class="no_margin">Dongliang Wang</h3><p><sup>6</sup>Department of Psychiatry and Behavioral Sciences, SUNY Upstate Medical University, Syracuse, New York</p><div>Find articles by <a href="/pubmed/?term=Wang%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Dongliang Wang</a></div></div><div id="_co_idm140711982875728"><h3 class="no_margin">Jun Li</h3><p><sup>7</sup>State Key Laboratory of Cognitive Neuroscience and Learning, Beijing Normal University, Beijing, China</p><div>Find articles by <a href="/pubmed/?term=Li%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Jun Li</a></div></div><div id="_co_idm140711986593120"><h3 class="no_margin">Stephen V. Faraone</h3><p><sup>6</sup>Department of Psychiatry and Behavioral Sciences, SUNY Upstate Medical University, Syracuse, New York</p><div>Find articles by <a href="/pubmed/?term=Faraone%20SV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Stephen V. Faraone</a></div></div><div id="_co_idm140711975376048"><h3 class="no_margin">Yufeng Wang</h3><p><sup>1</sup>Peking University Sixth Hospital/Institute of Mental Health, Beijing, China</p><p><sup>2</sup>Key Laboratory of Mental Health, Ministry of Health, China</p><div>Find articles by <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23728934">Yufeng Wang</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140711978137472_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140711978137472_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140711978137472_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Peking University Sixth Hospital/Institute of Mental Health, Beijing, China</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Key Laboratory of Mental Health, Ministry of Health, China</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Broad Institute of Harvard and MIT, Cambridge, Massachusetts</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>The University of Queensland, Queensland Brain Institute, Brisbane, Queensland, Australia</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Psychiatry and Behavioral Sciences, SUNY Upstate Medical University, Syracuse, New York</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>State Key Laboratory of Cognitive Neuroscience and Learning, Beijing Normal University, Beijing, China</div><div><sup>#</sup>Contributed equally.</div><div id="FN1"><sup>†</sup>For Psychiatric GWAS Consortium-ADHD subgroup authors see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Materials</a>.</div><div id="CR1"><sup>*</sup>Correspondence to: Yufeng Wang, Peking University Sixth Hospital, 51, Huayuan Bei Road, Haidian District, Beijing 100191, China. <a href="mailto:dev@null" data-email="nc.ude.umjb@fygnaw" class="oemail">nc.ude.umjb@fygnaw</a></div><div id="CR2"><sup>**</sup>Correspondence to: Stephen V. Faraone, Ph.D., Department of Psychiatry and Behavioral Sciences, SUNY Upstate Medical University, Syracuse, NY 13210. <a href="mailto:dev@null" data-email="gro.hcraeserhcyspdlihc@enoarafs" class="oemail">gro.hcraeserhcyspdlihc@enoarafs</a> Article first published online in Wiley Online Library (<a href="http://wileyonlinelibrary.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">wileyonlinelibrary.com</a>): 31 May 2013 DOI 10.1002/ajmg.b.32169</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140711978137472_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23728934" target="pmc_ext" ref="reftype=publisher&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Am J Med Genet B Neuropsychiatr Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4321789/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="10" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>sup data doc.</strong></div><div><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_data_doc.doc" data-ga-action="click_feat_suppl">NIHMS656151-supplement-sup_data_doc.doc</a><span style="color:gray"> (271K)</span></div><div class="small guid">GUID:&nbsp;E3697518-650E-4181-81AD-FADEAB67A38B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>sup fig s8.</strong></div><div><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s8.tif" data-ga-action="click_feat_suppl">NIHMS656151-supplement-sup_fig_s8.tif</a><span style="color:gray"> (291K)</span></div><div class="small guid">GUID:&nbsp;B4C98E51-E88C-478F-889C-62E7D78D3BB5</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>sup data xls.</strong></div><div><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_data_xls.xlsx" data-ga-action="click_feat_suppl">NIHMS656151-supplement-sup_data_xls.xlsx</a><span style="color:gray"> (18K)</span></div><div class="small guid">GUID:&nbsp;BA47FC30-63FC-4C19-9F3D-FE060BA99918</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>sup fig s1.</strong></div><div><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s1.tif" data-ga-action="click_feat_suppl">NIHMS656151-supplement-sup_fig_s1.tif</a><span style="color:gray"> (675K)</span></div><div class="small guid">GUID:&nbsp;1E03F836-4B88-4D6D-B9A7-3ABC963114F8</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>sup fig s2.</strong></div><div><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s2.tif" data-ga-action="click_feat_suppl">NIHMS656151-supplement-sup_fig_s2.tif</a><span style="color:gray"> (675K)</span></div><div class="small guid">GUID:&nbsp;A6AFE39B-C22B-4305-93D4-98169F388848</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>sup fig s3.</strong></div><div><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s3.tif" data-ga-action="click_feat_suppl">NIHMS656151-supplement-sup_fig_s3.tif</a><span style="color:gray"> (675K)</span></div><div class="small guid">GUID:&nbsp;A5AFEA5B-2AEA-41E3-8B5A-7680FDFE6365</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>sup fig s4.</strong></div><div><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s4.tif" data-ga-action="click_feat_suppl">NIHMS656151-supplement-sup_fig_s4.tif</a><span style="color:gray"> (675K)</span></div><div class="small guid">GUID:&nbsp;F6298C90-15CB-4399-A62C-B61BACFC377C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>sup fig s5.</strong></div><div><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s5.tif" data-ga-action="click_feat_suppl">NIHMS656151-supplement-sup_fig_s5.tif</a><span style="color:gray"> (675K)</span></div><div class="small guid">GUID:&nbsp;73023D00-F8FC-4320-B609-3F732CE4F855</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>sup fig s6.</strong></div><div><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s6.tif" data-ga-action="click_feat_suppl">NIHMS656151-supplement-sup_fig_s6.tif</a><span style="color:gray"> (675K)</span></div><div class="small guid">GUID:&nbsp;225C9608-D8F0-48F4-AF54-83D12DD339D1</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>sup fig s7.</strong></div><div><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s7.tif" data-ga-action="click_feat_suppl">NIHMS656151-supplement-sup_fig_s7.tif</a><span style="color:gray"> (675K)</span></div><div class="small guid">GUID:&nbsp;9AC476B3-9A5D-4F12-B286-38E13555E739</div></div></dd></dl></div><div id="idm140711977889632" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140711977889632title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">Attention-deficit hyperactivity disorder (ADHD) is a complex polygenic disorder. This study aimed to discover common and rare DNA variants associated with ADHD in a large homogeneous Han Chinese ADHD case–control sample. The sample comprised 1,040 cases and 963 controls. All cases met DSM-IV ADHD diagnostic criteria. We used the Affymetrix6.0 array to assay both single nucleotide polymorphisms (SNPs) and copy number variants (CNVs). Genome-wide association analyses were performed using PLINK. SNP-heritability and SNP-genetic correlations with ADHD in Caucasians were estimated with genome-wide complex trait analysis (GCTA). Pathway analyses were performed using the Interval enRICHment Test (INRICH), the Disease Association Protein–Protein Link Evaluator (DAPPLE), and the Genomic Regions Enrichment of Annotations Tool (GREAT). We did not find genome-wide significance for single SNPs but did find an increased burden of large, rare CNVs in the ADHD sample (<em>P</em> = 0.038). SNP-heritability was estimated to be 0.42 (standard error, 0.13, <em>P</em> = 0.0017) and the SNP-genetic correlation with European Ancestry ADHD samples was 0.39 (SE 0.15, <em>P</em> = 0.0072). The INRICH, DAPPLE, and GREAT analyses implicated several gene ontology cellular components, including neuron projections and synaptic components, which are consistent with a neurodevelopmental pathophysiology for ADHD. This study suggested the genetic architecture of ADHD comprises both common and rare variants. Some common causal variants are likely to be shared between Han Chinese and Caucasians. Complex neurodevelopmental networks may underlie ADHD's etiology.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">ADHD, GWAS, pathway, neurodevelopment</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">INTRODUCTION</h2><p id="P3" class="p p-first">Attention-deficit hyperactivity disorder (ADHD) is a common behavioral disorder of childhood, affecting 3–6% of school-age children around the world [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654085">Faraone et al., 2003</a>]. It has been viewed as a polygenic, multifactorial disorder. Both common and rare DNA variants contribute to its complex etiology [<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654082">Poelmans et al., 2011</a>; <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654100">Stergiakouli et al., 2012</a>; <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654105">Williams et al., 2012</a>].</p><p id="P4">Genome-wide association studies (GWAS) are hypothesis-free, interrogate all genes and regulatory regions of the genome and have the potential to discover novel risk genes. The first GWAS of ADHD performed by <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654090">Neale et al. [2008]</a> analyzed 438,784 SNPs in 909 Caucasian ADHD trios. Although none of the SNP association tests achieved genome-wide significance, the top-25 SNPs (based on <em>P</em>-value) implicated some interesting candidate genes, including cytoskeleton-organizer <em>DCLK1</em>, extracellular matrix component <em>SPOCK3</em>, cell-cell adhesion protein <em>CDH13</em>, as well as two potassium-channel regulators <em>KCNIP1</em> and <em>KCNIP4</em>. Using the same sample set, <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654084">Lasky-Su et al. [2008]</a> performed a quantitative genome-wide association analysis of ADHD symptoms. A high percentage (30/32, 94%) genes hit by the 58 SNPs with <em>P</em> values less than 10<sup>−5</sup> were brain-expressed, including five related to transcription factors.</p><p id="P5">Meanwhile, <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654072">Lesch et al. [2008]</a> used independent DNA pools from343 ADHD-affected adults and 304 controls for association analyses of the ADHD diagnostic phenotype. Of the 30 top-hit genes, seven were involved in cell adhesion/migration/neurogenesis (e.g., <sup>CDH13</sup>, <sup>ASTN2</sup>, <sup>CSMD2</sup>, <sup>ITGAE</sup>, <sup>ITGA11</sup>, <sup>CDH23</sup>, <sup>SDK2</sup>), two regulated synaptic plasticity (e.g., CTNNA2, KALRN), three were transcription factors (MYT1L, TFEB, SUPT3H), and one coded for a potassium channel (<sup>KCNC1</sup>) [<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654098">Lesch et al., 2008</a>].</p><p id="P6"><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654065">Neale et al. [2010a]</a> performed case-control analyses in896 cases with DSM-IV ADHD and 2,455 controls. A consensus dataset of 1,033,244 SNPs was imputed (using the HapMap Phase III European CEU and TSI samples as the reference). No genome-wide significant associations were found. The most significant results implicated <em>PRKG1</em>, <em>FLNC</em>, <em>TCERG1L</em>, <em>PPM1H</em>, <em>NXPH1</em>, <sup>CDH13</sup>, <em>HK1</em>, and <em>HKDC1</em>. Combining data from four ADHD GWAS projects, <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654091">Neale et al. [2010b]</a> performed a meta-analysis in a sample of 2,064 trios, 896 cases, and 2,455 controls. Even with this much larger sample size, no genome-wide significant associations were found. One reason for this isthat the samples were underpowered to estimate effect sizes of common variants of small effect. This has been confirmed by analyses that estimate the variance contributed by common variants all together. Genome-wide complex trait analysis (GCTA) [<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654101">Yang et al., 2010</a>, <a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654088">2011</a>; <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654075">Lee et al., 2011</a>] applied to ADHD samples (4,163 cases and 12, 040 controls) from the Psychiatric Genomics Consortium, estimated SNP chip heritability to be 0.28 (SE 0.02; Psychiatric GWAS Consortium ADHD Group. Paper submitted for publication).</p><p id="P7">Copy number variations (CNVs) have also been implicated in the etiology of ADHD. <a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654097">Elia et al. [2010]</a> found that inherited rare CNVs in an ADHD sample were significantly enriched for genes known to be important for psychological and neurological functions, including learning, behavior, synaptic transmission, and central nervous system development. Williams et al. [<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654071">Williams et al., 2010</a>, <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654076">2012</a>] found an increased burden of large, rare CNVs and reported excess of chromosome 16p13.11 and 15q13.3duplications and an overlap between CNVs reported for ADHD and autism spectrum disorders. <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654069">Elia et al. [2012]</a> further showed that CNVs affecting the metabotropic glutamate receptor genes GRM5, GRM7, GRM8, and GRM1were enriched across several independent samples.</p><p id="P8" class="p p-last">In summary, although ADHD is acknowledged to be a genetic disorder, GWAS has not revealed any common SNP variants with genome-wide significance. This study used both common and rare variants, using polygenic and pathway analyses, to evaluate the genetic etiology of ADHD in a large homogenous Han Chinese case–control sample.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">MATERIALS AND METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Participants</h3><p id="P9" class="p p-first">One thousand and forty ADHD cases (876 boys, 84.2%) aged between 6 and 16 years [average (9.7 ± 2.4) years] were recruited from the Child and Adolescent Psychiatric Outpatient Department of the Sixth Hospital, Peking University. All cases met DSM-IV ADHD diagnostic criteria. A clinical diagnosis was first made by a senior child and adolescent psychiatrist based on the parent and teacher completed ADHD Rating Scale-IV (ADHD-RS-IV), and then confirmed by semi-structured interview with the parents and child using the Chinese version of the Clinical Diagnostic Interview Scale [<a href="#R1" rid="R1" class=" bibr popnode">Barkley, 1998</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654102">Yang et al., 2004</a>]. Those with major neurological disorders (e.g., epilepsy), schizophrenia, pervasive development disorder, and mental retardation (IQ &lt; 70) were excluded. The sample consists of 680 (65.4%) ADHD combined type and 360 (34.6%) inattentive type. The comorbidities included oppositional defiant disorder (ODD) in 380 patients (36.5%), conduct disorder in 58 (5.6%), and tic disorder in 167 (16.1%).</p><p id="P10">Nine hundred sixty three controls were students from local elementary schools, healthy blood donors from the Blood Center of the First Hospital, Peking University, and healthy volunteers from our institute. Six hundred and eight were males (63.1%). The average age was (15.4 ± 8.8) years. Parents or adults themselves completed the ADHD Rating Scale-IV (ADHD RS-IV) to exclude ADHD. Major psychiatric disorders, family history of psychosis, severe physical diseases, and substance abuse were also excluded according to a medical history report form. All the cases and controls were of Han Chinese decent.</p><p id="P11" class="p p-last">The study was approved by the Institutional Review Board of the Peking University Health Science Center. After complete description of the study to the subjects, written informed consent was obtained from parents of the ADHD probands.</p></div><div id="S4" class="sec"><h3 id="S4title">Genotyping</h3><p id="P12" class="p p-first-last">Both cases and controls were genotyped using the Affymetrix6.0 array at CapitalBio Ltd. (Beijing) using the standard Affymetrix protocol. Samples of cases and controls were added in equal proportion to each chip to avoid batch effects. The Affymetrix 6.0 array included 906,600 SNP probes and 946,000 CNV probes. The SNP genotypes were called with BIRDSEED v2, while CNVs were called with Genotyping Console (GTC) 4.0 using default parameters. A total of 2003 cases and controls passed the first stage sample control with call rates &gt;98%, no first or second-degree relative relationships, and genders consistent with site reports.</p></div><div id="S5" class="sec"><h3 id="S5title">Data Quality Control and Statistical Analysis</h3><p id="P13" class="p p-first">Data quality control and association analysis were performed using PLINK 1.07 [<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654074">Purcell et al., 2007</a>, <a href="http://pngu.mgh.harvard.edu/purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/purcell/plink/</a>]. For inclusion of SNPs we required: call rate &gt;95%, MAF &gt;1%, and HWE <em>P</em>-value &gt;10<sup>−6</sup>. After data cleaning, there were 656,051 SNPs for the association analyses. To examine population stratification, we performed multi-dimensional scaling (MDS). In the pair-wise MDS plot for 10 dimensions, the majority of subjects were tightly clustered, suggesting no substantial population stratification (SF1). We then conducted logistic regression to adjust the association <em>P</em>-value, using the 10 principal components from the MDS procedure as covariates.</p><p id="P14">CNV calling only included segments larger than 100 kb, spanning at least 10 consecutive, informative SNPs. Quality control for samples excluded 136 individuals (71 cases, 65 controls) who carried more than 40 apparent CNVs. Analysis focused on rare CNVs with frequency &lt;1%. We used the human reference sequence of NCBI Build 36.1 - hg18 to filter known segmental duplications.</p><p id="P15" class="p p-last">Known common CNVs defined by the Genome Structural Variation Consortium (<a href="http://projects.tcag.ca/variation/ng42m_cnv.php" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://projects.tcag.ca/variation/ng42m_cnv.php</a>) and known gaps of at least 200 kb in the SNP array were also filtered. Burden analysis counted the number of total CNVs, deletions and duplications in cases and controls, calculated the CNV rate, as well as percent of cases and controls that carried rare CNVs. The significance of CNV differences between cases and controls was assessed by permutation test with 50,000replicates.</p></div><div id="S6" class="sec"><h3 id="S6title">Polygenic analyses</h3><p id="P16" class="p p-first-last">To investigate the contribution of common SNPs to variation in liability to ADHD, we estimated the SNP-heritability using GCTA [<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654080">Yang et al., 2011</a>]. A non-zero heritability is estimated if cases are genetically more similar to other cases than they are to controls [<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654066">Lee et al., 2011</a>]. We removed individuals such that no pair had genetic similarity relationship &gt;0.05 (as this may inflate estimates unfairly), so that 1,010 cases and 917 controls remained. We used Caucasian samples from the Psychiatric Genomics Consortium for ADHD (4,163 ADHD cases and 12,040 controls) and a bivariate model of analysis [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654077">Lee et al., 2012b</a>] to estimate the SNP-genetic correlation between Han Chinese and Caucasians for liability to ADHD. Since the SNP frequencies differ between ethnic groups the additive genetic similarities between individuals i and j were estimated as </p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" overflow="scroll"><mrow><msup><mi mathvariant="normal">A</mi><mi>ij</mi></msup><mfrac><mn>1</mn><mi mathvariant="normal">L</mi></mfrac><munderover><mo>∑</mo><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mi mathvariant="normal">L</mi></munderover><mrow><mo>(</mo><msub><mi mathvariant="normal">x</mi><mi>il</mi></msub><mo>−</mo><mn>2</mn><msub><mi mathvariant="normal">p</mi><mrow><mi>is</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub><mo>)</mo></mrow><mfrac><mrow><msub><mi mathvariant="normal">x</mi><mi>jl</mi></msub><mo>−</mo><mn>2</mn><msub><mi mathvariant="normal">p</mi><mrow><mi>is</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub></mrow><msqrt><mrow><mn>2</mn><msub><mi mathvariant="normal">p</mi><mrow><mi>is</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub><msub><mi mathvariant="normal">q</mi><mrow><mi>is</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub><mn>2</mn><msub><mi mathvariant="normal">p</mi><mrow><mi>js</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub><msub><mi>q</mi><mrow><mi>js</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub></mrow></msqrt></mfrac><mrow><mo>(</mo><mi mathvariant="normal">i</mi><mo>≠</mo><mi mathvariant="normal">j</mi><mo>)</mo></mrow></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" overflow="scroll"><mrow><msup><mi mathvariant="normal">A</mi><mi>ij</mi></msup><mfrac><mn>1</mn><mi mathvariant="normal">L</mi></mfrac><munderover><mo>∑</mo><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mi mathvariant="normal">L</mi></munderover><mrow><mo>(</mo><msub><mi mathvariant="normal">x</mi><mi>il</mi></msub><mo>−</mo><mn>2</mn><msub><mi mathvariant="normal">p</mi><mrow><mi>is</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub><mo>)</mo></mrow><mfrac><mrow><msub><mi mathvariant="normal">x</mi><mi>jl</mi></msub><mo>−</mo><mn>2</mn><msub><mi mathvariant="normal">p</mi><mrow><mi>is</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub></mrow><msqrt><mrow><mn>2</mn><msub><mi mathvariant="normal">p</mi><mrow><mi>is</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub><msub><mi mathvariant="normal">q</mi><mrow><mi>is</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub><mn>2</mn><msub><mi mathvariant="normal">p</mi><mrow><mi>js</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub><msub><mi>q</mi><mrow><mi>js</mi><mo>,</mo><mi mathvariant="normal">l</mi></mrow></msub></mrow></msqrt></mfrac><mrow><mo>(</mo><mi mathvariant="normal">i</mi><mo>≠</mo><mi mathvariant="normal">j</mi><mo>)</mo></mrow></mrow></math></script></div></div><p> for the L SNPs with minor allele frequency &gt;0.01and imputation R<sup>2</sup> &gt; 0.6 (L = 917,066), where i s represent a population that individual i belong to and p and q = 1 − p are allele frequencies of the first and other allele and x<em>il</em> is the number of first alleles for the lth SNP in individual i. The analysis model include sex, cohort and 20 ancestry principal components are covariates.</p></div><div id="S7" class="sec sec-last"><h3 id="S7title">Pathway analysis</h3><p id="P17" class="p p-first">To determine if any neurobiological pathways were implicated by our association signals, we input our top hit intervals from the SNP and CNV association analyses into Interval enRICHment Test (INRICH [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654094">Lee et al., 2012a</a>]). Associated intervals for SNPs included those with <em>P</em>-values &lt;10<sup>−4</sup> after correcting for the MDS components. The SNP tagging function in PLINK was used to generate LD independent genomic intervals (tag r2: 0.2, tag kb: 1,000). We included CNV intervals that were more prevalent in cases than in controls with at least a trend difference of statistical significance (<em>P</em> &lt; 0.15). We used the Gene Ontology (GO) nodes as our target gene sets. After size filtering, 5,237 target gene sets (nodes) each comprising at least three genes were examined. Interval overlap was limited to 20 kb up/downstream of a gene. The number of overlapping genes was recorded as Real<sub>i</sub>. Ten thousand replicates generated random interval sets each matching to the number of associated intervals. The empirical gene-set P-value equals the percent of replicates with at least Real<sub>i</sub> number of random intervals overlapping with genes in a target gene set. Bootstrapping-based re-sampling was used for multiple testing to correct the empirical gene-set <em>P</em>-value over all gene sets.</p><p id="P18">To explore potential physical interactions among proteins encoded in associated intervals, we used a second method for pathway analysis, that is, Disease Association Protein–Protein Link Evaluator (DAPPLE) [<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654103">Rossin et al., 2011</a>]. In consideration of the contributions of both common and rare variants to the etiology of ADHD, and that both might separately capture nodes in the ADHD pathogenesis network, we used the same genomic intervals for both SNPs and CNV that we used for INRICH. DAPPLE uses experimentally validated, protein–protein interaction databases to identify direct and indirect networks from associated proteins and scores network and protein connectivity. We built 10,000 random networks and compared these with the ADHD associated networks to determine if the connectivity of the ADHD networks and each seed protein was greater than expected by chance.</p><p id="P19" class="p p-last">The third pathway analyses we used the Genomic Regions Enrichment of Annotations Tool (GREAT, [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_529162570">McLean et al., 2010</a>] to assess for enrichment of cis-regulatory regions. GREAT examines not only proximal but also distal regulatory regions up to 1 Mb upstream or downstream of transcription start sites. In addition to typical calculation of gene-based <em>P</em>-values for enrichment, GREAT computes a binomial test over genomic regions, which uses the fraction of the genome associated with each ontology term as the probability of selecting the term. This method explicitly accounts for the variability in length of gene regulatory domains, eliminating the bias that leads to false positive enrichments for distal regulatory regions.</p></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">RESULTS</h2><div id="S9" class="sec sec-first"><h3 id="S9title">Single Variant Analyses</h3><p id="P20" class="p p-first">The quantile-quantile (QQ) plot (SF2) for SNPs’ association was almost completely diagonal. The lambda statistic (λ) was 1.02.The distribution of observed <em>P</em>-values did not deviate from the distribution expected under the null hypothesis of no association. The corrected Manhattan plot is shown inSF3. The lowest <em>P</em> values were about 10<sup>−5</sup> to 10<sup>−6</sup>.The SNPs associated with <em>P</em> values of 10<sup>−5</sup> or lower are listed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table SI</a>. All hit genes were expressed in brain. Most of them were known to be involved in neurodevelopment (including cell adhesion, neuron migration, neurite outgrowth, neuronal morphogenesis, and synaptic plasticity: <em>ITGA1</em>, <em>NYAP2</em>, <em>ADAM28</em>, <em>CNTN2</em>, <em>LRFN2</em>, <em>NTM</em>, <em>GJA1</em>, <em>FLRT2</em>, <em>PRKG1</em>, <em>PICK1</em>, <em>CAMK2G</em>; glutamate receptor and transporter: <em>GRIK4</em>, <em>GRM7</em>, <em>SLC38A1</em>; and related transcription factors: PAPOLA, MED27, TAF2, ZNF516).</p><p id="P21" class="p p-last">We included 3,460 rare CNVs (1,817 in cases and 1,643 in controls) in the analyses, with all segments intersecting with one or more genes (hg18). Burden analyses showed a significantly higher rate of rare CNVs (1.875% vs. 1.830%, ratio: 1.02, <em>P</em> = 0.038) and proportion of individuals carrying rare CNVs (55.8% vs. 51.2%, ratio: 1.09, <em>P</em> = 0.026) for the ADHD group than for controls. Association analyses found six regions nominally associated with ADHD (<em>P</em> &lt; 0.05, with 50,000 permutation tests), though none of them survived genome-wide correction (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table SII</a>).</p></div><div id="S10" class="sec"><h3 id="S10title">Polygenic Analyses</h3><p id="P22" class="p p-first-last">The estimate of the SNP-heritability calculated in the bivariate analysis was 0.42 (SE 0.13)for the Han Chinese sample <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M2" overflow="scroll"><mrow><mo>(</mo><msubsup><mi mathvariant="normal">h</mi><mrow><mi>SNP</mi><mo>−</mo><mi>HC</mi></mrow><mn>2</mn></msubsup><mo>)</mo></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-2-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M2" overflow="scroll"><mrow><mo>(</mo><msubsup><mi mathvariant="normal">h</mi><mrow><mi>SNP</mi><mo>−</mo><mi>HC</mi></mrow><mn>2</mn></msubsup><mo>)</mo></mrow></math></script>.A maximum likelihood ratio test of H0: <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M3" overflow="scroll"><mrow><msubsup><mi mathvariant="normal">h</mi><mi>SNP</mi><mn>2</mn></msubsup><mo>=</mo><mn>0</mn></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-3-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M3" overflow="scroll"><mrow><msubsup><mi mathvariant="normal">h</mi><mi>SNP</mi><mn>2</mn></msubsup><mo>=</mo><mn>0</mn></mrow></math></script>. In ancestry the bivariate analysis the SNP-heritability for the European sample <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M4" overflow="scroll"><mrow><mo>(</mo><msubsup><mi mathvariant="normal">h</mi><mi>SNPEA</mi><mn>2</mn></msubsup><mo>)</mo></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-4-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M4" overflow="scroll"><mrow><mo>(</mo><msubsup><mi mathvariant="normal">h</mi><mi>SNPEA</mi><mn>2</mn></msubsup><mo>)</mo></mrow></math></script> was 0.28 (SE 0.02, <em>P</em> = 0), in close agreement (as expected) with the univariate estimate PGC Cross Disorder Group, paper in submission. The estimate of the SNP-genetic correlation between Chinese and European samples (r<sub>g-SNP</sub>) was 0.39 (SE 0.15, <em>P</em> = 0.0072).</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Pathway Analyses</h3><p id="P23" class="p p-first">Interval enrichment tests of the most significantly associated SNPs found 23 pathways enriched for associated signals (<a href="/pmc/articles/PMC4321789/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>). Although none of these achieved significance after correcting for multiple comparisons, many implicated neurobiological functions potentially relevant to ADHD, e.g. neuron projection morphogenesis (<em>ITGA1</em>, <em>GJA1</em>), neuron migration (<em>PRKG1</em>, <em>GJA1</em>), endocytic vesicle membrane (<em>PICK1</em>, <em>CAMK2G</em>), synaptic transmission (<em>PICK1</em>, <em>CAMK2G</em>, <em>SLC38A1</em>, <em>GRM7</em>). Pathways related to transcription were observed, that is, transcription initiation from RNA polymerase II promoter (<em>MED27</em>, <em>TAF2</em>). Interval enrichment tests of CNVs found 9 pathway nominal significant at <em>P</em> &lt; 0.05. None achieved significance after correcting for multiple comparisons (<a href="/pmc/articles/PMC4321789/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>). Most were related to transmembrane transport, including water, sodium and potassium ion transport.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>TABLE I</h3><!--caption a7--><div class="caption"><p id="__p5">Pathways Enriched for Associated SNPs and CNVs by INRICH Test</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711980540000" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Target</th><th rowspan="1" colspan="1" valign="top" align="right">Target size</th><th rowspan="1" colspan="1" valign="top" align="right">Interval no.</th><th rowspan="1" colspan="1" valign="top" align="left">Emp. P</th><th rowspan="1" colspan="1" valign="top" align="left">Cor. P</th><th rowspan="1" colspan="1" valign="top" align="center">Gene list</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Pathways enriched for associated SNPs<sup><sup>a</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0009268Response to pH</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.00009999</td><td rowspan="1" colspan="1" valign="top" align="left">0.14917</td><td rowspan="1" colspan="1" valign="top" align="center">ARSB, GJA1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0043403Skeletal muscle tissue regeneration</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.00019998</td><td rowspan="1" colspan="1" valign="top" align="left">0.192162</td><td rowspan="1" colspan="1" valign="top" align="center">PLAU, GJA1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0048812Neuron projection morphogenesis</td><td rowspan="1" colspan="1" valign="top" align="right">18</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.00029997</td><td rowspan="1" colspan="1" valign="top" align="left">0.231354</td><td rowspan="1" colspan="1" valign="top" align="center">ITGA1, GJA1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0007160Cell-matrix adhesion</td><td rowspan="1" colspan="1" valign="top" align="right">72</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="left">0.00049995</td><td rowspan="1" colspan="1" valign="top" align="left">0.295941</td><td rowspan="1" colspan="1" valign="top" align="center">VCL, ITGA1, BCL2L11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0005916Fascia adherens</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.00079992</td><td rowspan="1" colspan="1" valign="top" align="left">0.378724</td><td rowspan="1" colspan="1" valign="top" align="center">VCL, GJA1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0006936Muscle contraction</td><td rowspan="1" colspan="1" valign="top" align="right">93</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="left">0.00109989</td><td rowspan="1" colspan="1" valign="top" align="left">0.447111</td><td rowspan="1" colspan="1" valign="top" align="center">VCL, ITGA1, GJA1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0030666Endocyticvesicle membrane</td><td rowspan="1" colspan="1" valign="top" align="right">24</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.00269973</td><td rowspan="1" colspan="1" valign="top" align="left">0.661668</td><td rowspan="1" colspan="1" valign="top" align="center">PICK1, CAMK2G</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0006367Transcription initiation from RNA polymerase II promoter</td><td rowspan="1" colspan="1" valign="top" align="right">67</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.00379962</td><td rowspan="1" colspan="1" valign="top" align="left">0.727854</td><td rowspan="1" colspan="1" valign="top" align="center">MED27, TAF2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0005741Mitochondrial outer membrane</td><td rowspan="1" colspan="1" valign="top" align="right">85</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.00679932</td><td rowspan="1" colspan="1" valign="top" align="left">0.847231</td><td rowspan="1" colspan="1" valign="top" align="center">GJA1, BCL2L11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0007229Integrin-mediated signaling pathway</td><td rowspan="1" colspan="1" valign="top" align="right">58</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.010399</td><td rowspan="1" colspan="1" valign="top" align="left">0.894421</td><td rowspan="1" colspan="1" valign="top" align="center">ITGA1, ADAMDEC1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0005178Integrin binding</td><td rowspan="1" colspan="1" valign="top" align="right">64</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0114989</td><td rowspan="1" colspan="1" valign="top" align="left">0.907419</td><td rowspan="1" colspan="1" valign="top" align="center">ITGA1, ADAMDEC1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0007268Synaptic transmission</td><td rowspan="1" colspan="1" valign="top" align="right">266</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">0.0121988</td><td rowspan="1" colspan="1" valign="top" align="left">0.915217</td><td rowspan="1" colspan="1" valign="top" align="center">PICK1, CAMK2G, SLC38A1, GRM7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0005764Lysosome</td><td rowspan="1" colspan="1" valign="top" align="right">154</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0157984</td><td rowspan="1" colspan="1" valign="top" align="left">0.946011</td><td rowspan="1" colspan="1" valign="top" align="center">ARSB, GJA1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0030165PDZ domain binding</td><td rowspan="1" colspan="1" valign="top" align="right">54</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.019898</td><td rowspan="1" colspan="1" valign="top" align="left">0.965807</td><td rowspan="1" colspan="1" valign="top" align="center">GRM7, GJA1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0015293Symporter activity</td><td rowspan="1" colspan="1" valign="top" align="right">112</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.019898</td><td rowspan="1" colspan="1" valign="top" align="left">0.965807</td><td rowspan="1" colspan="1" valign="top" align="center">SLC38A1, SLC16A8</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0005624Membrane fraction</td><td rowspan="1" colspan="1" valign="top" align="right">467</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">0.0243976</td><td rowspan="1" colspan="1" valign="top" align="left">0.982004</td><td rowspan="1" colspan="1" valign="top" align="center">ITGA1, SLC16A8, PSD3, BCL2L11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0001764Neuron migration</td><td rowspan="1" colspan="1" valign="top" align="right">59</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0255974</td><td rowspan="1" colspan="1" valign="top" align="left">0.983003</td><td rowspan="1" colspan="1" valign="top" align="center">PRKG1, GJA1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0045121Membrane raft</td><td rowspan="1" colspan="1" valign="top" align="right">110</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0256974</td><td rowspan="1" colspan="1" valign="top" align="left">0.983203</td><td rowspan="1" colspan="1" valign="top" align="center">ITGA1, GJA1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0006814Sodiumjon transport</td><td rowspan="1" colspan="1" valign="top" align="right">118</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0265973</td><td rowspan="1" colspan="1" valign="top" align="left">0.984003</td><td rowspan="1" colspan="1" valign="top" align="center">SLC38A1, SCN9A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0005654Nucleoplasm</td><td rowspan="1" colspan="1" valign="top" align="right">732</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">0.0214979</td><td rowspan="1" colspan="1" valign="top" align="left">0.968806</td><td rowspan="1" colspan="1" valign="top" align="center">MED27, CAMK2G, DSCC1, PAPOLA</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0017124SH3 domain binding</td><td rowspan="1" colspan="1" valign="top" align="right">101</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0357964</td><td rowspan="1" colspan="1" valign="top" align="left">0.992402</td><td rowspan="1" colspan="1" valign="top" align="center">BAIAP2L2, GJA1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0043234Protein complex</td><td rowspan="1" colspan="1" valign="top" align="right">152</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0437956</td><td rowspan="1" colspan="1" valign="top" align="left">0.995601</td><td rowspan="1" colspan="1" valign="top" align="center">PICK1, VCL</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0001701In utero embryonic development</td><td rowspan="1" colspan="1" valign="top" align="right">142</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0459954</td><td rowspan="1" colspan="1" valign="top" align="left">0.996601</td><td rowspan="1" colspan="1" valign="top" align="center">GJA1, BCL2L11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Pathways enriched for associated CNVs<sup><sup>b</sup></sup></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0006833Water transport</td><td rowspan="1" colspan="1" valign="top" align="right">36</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.00549945</td><td rowspan="1" colspan="1" valign="top" align="left">0.508498</td><td rowspan="1" colspan="1" valign="top" align="center">AQP9, ADCY8</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO:0005244 Voltage-gated ion channel activity</td><td rowspan="1" colspan="1" valign="top" align="right">149</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">0.0115988</td><td rowspan="1" colspan="1" valign="top" align="left">0.734653</td><td rowspan="1" colspan="1" valign="top" align="center">KCNH7, KCNQ1, SCN10A, SCN11A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0055085Transmembrane transport</td><td rowspan="1" colspan="1" valign="top" align="right">619</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">0.0125975</td><td rowspan="1" colspan="1" valign="top" align="left">0.752849</td><td rowspan="1" colspan="1" valign="top" align="center">KCNH7, SCN10A, KCNQ1, AQP9, ADCY8, SCN11A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0042493Response to drug</td><td rowspan="1" colspan="1" valign="top" align="right">241</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="left">0.0164984</td><td rowspan="1" colspan="1" valign="top" align="left">0.834233</td><td rowspan="1" colspan="1" valign="top" align="center">USP47, CPS1, SCN11A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO:0005248 Voltage-gated sodium channel activity</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0172983</td><td rowspan="1" colspan="1" valign="top" align="left">0.84883</td><td rowspan="1" colspan="1" valign="top" align="center">SCN10A, SCN11A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO:0001518 Voltage-gated sodium channel complex</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0172983</td><td rowspan="1" colspan="1" valign="top" align="left">0.84883</td><td rowspan="1" colspan="1" valign="top" align="center">SCN10A, SCN11A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO:0006814Sodiumjon transport</td><td rowspan="1" colspan="1" valign="top" align="right">118</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">0.0258974</td><td rowspan="1" colspan="1" valign="top" align="left">0.909018</td><td rowspan="1" colspan="1" valign="top" align="center">SCN10A, SCN11A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO:0006811 Iontransport</td><td rowspan="1" colspan="1" valign="top" align="right">565</td><td rowspan="1" colspan="1" valign="top" align="right">5</td><td rowspan="1" colspan="1" valign="top" align="left">0.030397</td><td rowspan="1" colspan="1" valign="top" align="left">0.928814</td><td rowspan="1" colspan="1" valign="top" align="center">KCNH7, KCNQ1, KCNT2, SCN10A, SCN11A</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">&nbsp;&nbsp;&nbsp;&nbsp;GO: 0006813Potassiumiontransport</td><td rowspan="1" colspan="1" valign="top" align="right">153</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="left">0.0392961</td><td rowspan="1" colspan="1" valign="top" align="left">0.956809</td><td rowspan="1" colspan="1" valign="top" align="center">KCNH7, KCNQ1, KCNT2</td></tr></tbody></table></div><div id="largeobj_idm140711980540000" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4321789/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>With corrected <em>P</em>-value &lt;10e–4.</div><div id="TFN2"><sup>b</sup>Including CNVs more in cases than in controls with <em>P</em> &lt; 0.15.</div></div></div><p id="P24">DAPPLE identified 16 direct connections among proteins in 152 associated regions (<a href="/pmc/articles/PMC4321789/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table II</span></a>). Compared to 10,000 random networks, the associated network (SF4) is significantly enriched for direct connectivity (16 vs. 9.7, <em>P</em> = 0.030). The connected proteins formed six groups. Their functions involved cell adhesion/synaptic formation/plasticity, especially for glutamatergic synaptic plasticity, as well as related transcription factors. For each seed protein, taking the best of the direct and indirect scores and correcting for the number of tests as well as for the number of genes in one locus, we identified seven genes significant for connectivity to be candidate genes for future research: <em>NCL</em> (<em>P</em> = 2 × 10<sup>−4</sup>), <em>KCNH7</em> (<em>P</em> = 8 × 10<sup>−4</sup>), <em>NXPH1</em> (1 × 10<sup>−3</sup>), <em>LANCL1</em> (6 × 10<sup>−3</sup>), <em>CNTNAP2</em> (9 × 10<sup>−3</sup>), <em>SV2C</em> (1.2 × 10<sup>−2</sup>), and <em>PICK1</em> (4 × 10<sup>−2</sup>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>TABLE II</h3><!--caption a7--><div class="caption"><p id="__p6">Direct Connections Between Proteins Encoded by Genes Overlapped With Potential Associated SNPs and CNVs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711981311472" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Direct connection</th><th rowspan="1" colspan="1" valign="top" align="left">Associated SNP/interval</th><th rowspan="1" colspan="1" valign="top" align="left">Gene function<sup><sup>a</sup></sup></th><th rowspan="1" colspan="1" valign="top" align="left">Gene and pathway implicated in other studies of ADHD or psychiatric disorders</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">
<div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711977479408" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0001.jpg" src="/pmc/articles/PMC4321789/bin/nihms-656151-t0001.jpg"></div><div id="largeobj_idm140711977479408" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4321789/table/T2/?report=objectonly">Open in a separate window</a></div>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>NXPH1: chr7: 8471930-8567358</em>
<br>
<em>NRXN1: chr2: 50226778-50323795</em>
</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>NXPH1:</em></strong> The encoded protein forms a very tight complex with alpha neurexins, a group of proteins that promote adhesion between dendrites and axons<br><strong><em>NRXN1:</em></strong> The encoded protein functions as cell adhesion molecules and receptors. May play a role in formation or maintenance of synaptic junctions</td><td rowspan="1" colspan="1" valign="top" align="left">Both <strong><em>NXPH1</em></strong> and <strong><em>NRXN1</em></strong> were among the top hit genes [10<sup>–</sup><sup>5</sup>] in ADHD GWAS by <a href="#R22" rid="R22" class=" bibr popnode">Neale et al. [2010a]</a>. CNVs of <strong><em>NXPH1</em></strong> were found in ASD families [<a href="#R29" rid="R29" class=" bibr popnode">Salyakina et al., 2011</a>]. CNVs of <strong><em>NRXN1</em></strong> were also observed in schizophrenia and autism [<a href="#R5" rid="R5" class=" bibr popnode">Doherty et al., 2012</a>]<br><strong><em>CNTNAP2</em></strong> has been implicated in multiple neurodevelopmental disorders, including Gilles de la Tourette syndrome, schizophrenia, epilepsy, autism, ADHD and mental retardation. <strong><em>CNTNAPS</em></strong> was among the top hit genes in ADHD GWAS by <a href="#R22" rid="R22" class=" bibr popnode">Neale et al. [2010a]</a>. <strong><em>ZMIZ1</em></strong> was among the top hit genes in <a href="#R21" rid="R21" class=" bibr popnode">Neale et al. [2008</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<div data-largeobj="" data-largeobj-link-rid="largeobj_idm140712053331328" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0002.jpg" src="/pmc/articles/PMC4321789/bin/nihms-656151-t0002.jpg"></div><div id="largeobj_idm140712053331328" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4321789/table/T2/?report=objectonly">Open in a separate window</a></div>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CNTN2: rs2802837</em>
<br>
<em>CNTNAP2: chr7: 145653708-145657817</em>
<br>
<em>Chr7: 145755346-145813647</em>
<br>
<em>ZMIZ1: chr10: 80350172-80476638</em>
</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>CNTN2:</em></strong> It is a neuronal membrane protein that functions as a cell adhesion molecule. It may play a role in the formation of axon connections in the developing nervous system<br><strong><em>CNTNAP2:</em></strong> This gene encodes a member of the neurexin family which functions as cell adhesion molecules and receptors.This protein is localized at the juxtaparanodes of myelinated axons, and mediates interactions between neurons and glia during nervous system development and is also involved in localization of potassium channels within differentiating axons<br><strong><em>ZMIZ1:</em></strong> The encoded protein regulates the activity of various transcription factors</td><td rowspan="1" colspan="1" valign="top" align="left"><strong>Cell adhesion</strong> molecule had been reported in previous ADHD GWAS studies [<a href="#R15" rid="R15" class=" bibr popnode">Lasky-Su et al., 2008</a>; <a href="#R40" rid="R40" class=" bibr popnode">Zhou et al., 2008</a>; <a href="#R22" rid="R22" class=" bibr popnode">Neale et al., 2010a</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711973367760" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0003.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0003.jpg" src="/pmc/articles/PMC4321789/bin/nihms-656151-t0003.jpg"></div><div id="largeobj_idm140711973367760" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4321789/table/T2/?report=objectonly">Open in a separate window</a></div>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>SCN10A: chr3: 38787799-38889198</em>
<br>
<em>GJA1: rs7753979</em>
<br>
<em>TNNT2: chr1:199488134-199580414</em>
</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>SCN10A:</em></strong> Voltage-gated sodium channels are integral membrane glycoproteins that are responsible for the initial rising phase of action in most excitable cells<br><strong><em>GJA1:</em></strong> This gene is a member of the connexin gene family. The encoded protein is a component of gap junctions. Gap channels allow electrical and biochemical coupling between cells and in excitable tissues, such as neurons and heart.<br><strong><em>TNNT2</em></strong>: The protein encoded by this gene is the tropomyosin-binding subunit of the troponin complex, which is located on the thin filament of striated muscles and regulates muscle contraction in response to alterations in intracellular calcium ion concentration. This gene expresses highest in the heart, but also expresses in the brain</td><td rowspan="1" colspan="1" valign="top" align="left">4 SNPs of <strong><em>SCN10A</em></strong> showed nominal association with ADHD in the meta-analysis by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b]</a>, with the lowest <em>P-</em>value = 0.022for rs7430438<br>rs7740467, which is approximately 3 kb upstream of <strong><em>GJA1,</em></strong> was found nominal significant (<em>P =</em> 0.0204) in the meta-analysis of ADHD GWAS by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b]</a>.<br>8 SNPs of <strong><em>TNNT2</em></strong> showed nominal association with ADHD in the meta-analysis by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b]</a>, with the lowest <em>P</em>-value = 0.015for rs10800775</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<div data-largeobj="" data-largeobj-link-rid="largeobj_idm140712021432688" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0004.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0004.jpg" src="/pmc/articles/PMC4321789/bin/nihms-656151-t0004.jpg"></div><div id="largeobj_idm140712021432688" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4321789/table/T2/?report=objectonly">Open in a separate window</a></div>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>GRM7: rs13317247</em>
<br>
<em>GRM3: chr7: 86132321-86244019</em>
<br>
<em>PICK1: rs8142185</em>
<br>
<em>EPHA7: chr6: 94194161-94216651</em>
</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>GRM7</em></strong>
<em>and <strong>GRM3:</strong> L</em>-glutamate is the major excitatory neurotransmitter in the central nervous system, and it activates both ionotropic and metabotropic glutamate receptors. Glutamatergic neurotransmission is involved in most aspects of normal brainfunction and can be perturbed in many neuropathologic conditions. GRM3 belongs to the metabotropic glutamate receptors Group II, while GRM7 belongs to Group III<br><strong><em>PICK1:</em></strong> The protein encoded by this gene has been shown to interact with multiple glutamate receptor subtypes, Probable adapter protein that bind to and organize the subcellular localization of a variety of membrane proteins containing some PDZ recognition sequence. Involved in the clustering of various receptors, possibly by acting at the receptor internalization level. Plays a role in synaptic plasticity by regulating the trafficking and internalization of AMPA receptors<br><strong><em>EPHA7:</em></strong> This gene belongs to the ephrin receptor subfamily of the protein-tyrosine kinase family. EPH and EPH-related receptors have been implicated in mediating developmental events, particularly in the nervous system</td><td rowspan="1" colspan="1" valign="top" align="left">More than 20 SNPs of <strong><em>GRM7</em></strong> showed association with ADHD in the meta-analysis by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b]</a>, with the lowest <em>P</em> value = 2.96E-3 for rsl532544. CNVs of <strong><em>GRM7</em></strong> were found in a Genome-wide copy number variation study of ADHD [<a href="#R7" rid="R7" class=" bibr popnode">Elia et al., 2012</a>]. rs17031835, rsl2491G20, rsl450099, and rs3749380 of <strong><em>GRM7</em></strong> were associated with SCZ [<a href="#R13" rid="R13" class=" bibr popnode">Ganda et al., 2009</a>; <a href="#R24" rid="R24" class=" bibr popnode">Ohtsuki et al., 2008</a>; <a href="#R31" rid="R31" class=" bibr popnode">Shibata et al., 2009</a>]. <strong><em>GRM3</em></strong>
<em>and <strong>EPHA3</strong></em> were identified to be candidate genes for ASD [<a href="#R2" rid="R2" class=" bibr popnode">Casey et al., 2012</a>]. <strong><em>GRM3</em></strong>
<em>was associated with SCZ in some candidate gene studies</em> [<a href="#R3" rid="R3" class=" bibr popnode">Cherlyn et al., 2010</a>]. <em>It was also related to psychosis and relapse in bipolar disorder</em> [<a href="#R4" rid="R4" class=" bibr popnode">Dalvie et al., 2010</a>]. <strong><em>PICK1</em></strong> located in the linkage and association region of SCZ [<a href="#R26" rid="R26" class=" bibr popnode">Pulver et al., 1994</a>; <a href="#R14" rid="R14" class=" bibr popnode">Hong et al., 2004</a>; <a href="#R12" rid="R12" class=" bibr popnode">Fujii et al., 2006</a>] rsl2GG1215 of <strong><em>εPHA7</em></strong> was found nominal significant (<em>P=</em> 0.0107) in the meta-analysis of ADHD GWAS by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b]</a> rs2GG4283 in <strong><em>CAMK2G</em></strong> was found nominal significant (<em>P = 7</em>.5E–3) in the meta-analysis of ADHD GWAS by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b]</a><br><strong>Metabotropic glutamate receptor genes family and their interacting genes</strong> were previously found to be enriched with CNVs in ADHD samples [<a href="#R7" rid="R7" class=" bibr popnode">Elia et al., 2012</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711972756352" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0005.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0005.jpg" src="/pmc/articles/PMC4321789/bin/nihms-656151-t0005.jpg"></div><div id="largeobj_idm140711972756352" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4321789/table/T2/?report=objectonly">Open in a separate window</a></div>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CAMK2G: rs10824051, rs11000831</em>
<br>
<em>ADAM28: rs7012077</em>
</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>CAMK2G:</em></strong> The product of this gene is one of the four subunits of an enzyme which belongs to the Ca(2+)/calmodulin-dependent protein kinase subfamily. Calcium signaling is crucial for several aspects of plasticity at glutamatergic synapses<br><strong><em>ADAM28:</em></strong> This gene encodes a member of the ADAM (a disintegrin and metalloprotease domain) family. Members of this family have been implicated in a variety of biological processes involving cell-cell and cell-matrix interactions, including neurogenesis</td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711986052864" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0006.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-t0006.jpg" src="/pmc/articles/PMC4321789/bin/nihms-656151-t0006.jpg"></div><div id="largeobj_idm140711986052864" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4321789/table/T2/?report=objectonly">Open in a separate window</a></div>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>MED20: chr6: 41904795, 42015370</em>
<br>
<em>MED27: rs10512416, rs6597539</em>
<br>
<em>TAF2: rs3812463, rs6469849, rs6469852, rs6989791, rs7012857</em>
<br>
<em>PAPOLA: rs7149784, rs71B0B41</em>
<br>
<em>P0LR2F: rs8142185</em>
<br>
<em>NCL: rs16828074</em>
<br>
<em>RSL101: chr16: 11823863-11943065</em>
<br>
<em>BYSL: chr6: 41904795-42015370</em>
</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>MεD20:</em></strong> Component of the Mediator complex, a coactivator involved in the regulated transcription of nearly all RNApolymerase II-dependent genes<br><strong><em>MED27:</em></strong> mediator complex subunit <em>27.</em> These factors work with co-activators to direct transcriptional initiation by the RNA polymerase II apparatus<br><strong><em>TAF2:</em></strong> stabilizes TFIID binding to core promoter. Transcription factor TFIID is one of the general factors required for accurate and regulated initiation by RNA polymerase II<br><strong><em>PAPOLA:</em></strong> Polymerase that creates the 3'-poly (A) tail of mRNA's<br><strong><em>POLR2F:</em></strong> This gene encodes the sixth largest subunit of RNA polymerase II, the polymerase responsible for synthesizing messenger RNA in eukaryotes, that is also shared by the other two DNA-directed RNA polymerases<br><strong><em>NCL:</em></strong> Nucleolin play a role in pre-rRNA transcription and ribosome assembly. May play a role in the process of transcriptional elongation<br><strong><em>RSL1D1:</em></strong> ribosomal L1 domain containing<br><strong><em>BYSL:</em></strong> Required for processing of 20S pre-rRNA precursor and biogenesis of 40S ribosomal subunits. May be required for regulating cell adhesion during implantation of human embryos</td><td rowspan="1" colspan="1" valign="top" align="left">rs3218100, 15 kb downstream of <strong><em>MεD20,</em></strong> was nominal significant (<em>P=</em> 7.06E–3) in the meta-analysis of ADHD GWAS by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b</a>]<br>rsl0901091, which is approximately 90 kb upstream of <strong><em>MεD2F,</em></strong> was found nominal significant (<em>P</em> = 1.13E–3) in the meta-analysis of ADHD GWAS by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b]</a>. 3 SNPs, approximately 135 kb downstream of <strong><em>PAPOLA,</em></strong> showed association with ADHD in the meta-analysis by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b]</a>, with the lowest <em>P-</em>alue = 7.02E–3 for rsl9G9795. <strong><em>POLR2F</em></strong> Located in the linkage and association region of SCZ [<a href="#R26" rid="R26" class=" bibr popnode">Pulver et al., 1994</a>; <a href="#R14" rid="R14" class=" bibr popnode">Hong et al., 2004</a>; <a href="#R12" rid="R12" class=" bibr popnode">Fujii et al., 2006</a>]. <strong><em>RSL1D1</em></strong> located in 16pl3 of the linkage region for ADHD [<a href="#R10" rid="R10" class=" bibr popnode">Fisher et al., 2002</a>; <a href="#R32" rid="R32" class=" bibr popnode">Smalley et al., 2002</a>]. A SNP, rs4G4017, 65 kb downstream of RSL1D1, was nominal significant (<em>P</em> = 0.0387) in the meta-analysis of ADHD GWAS by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b]</a>. A SNP, rs3218100, about 4 kb downstream of <strong><em>BYSL,</em></strong> was nominal significant (<em>P =</em> 7.06E–3) in the meta-analysis of ADHD GWAS by <a href="#R23" rid="R23" class=" bibr popnode">Neale et al. [2010b]</a><br><strong>Transcription regulating proteins,</strong> such as members of <em>ZNF</em> family, <em>DMRT2, FHIT, FOXP1,</em> and <em>MEIS2,</em> had been implicated in previous GWAS [<a href="#R15" rid="R15" class=" bibr popnode">Lasky-Su et al., 2008</a>; <a href="#R23" rid="R23" class=" bibr popnode">Neale et al., 2010b</a>]</td></tr></tbody></table></div><div id="largeobj_idm140711981311472" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4321789/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>a</sup>From the UCSC Browser, UniProtKB, NCBI's OMIM, and GeneCards.</div></div></div><p id="P25" class="p p-last">Using the same set of associated SNPs and CNVs for GREAT analyses, we found significant enrichment for 6 GO Cellular Component terms after correcting for multiple comparisons (<a href="/pmc/articles/PMC4321789/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table III</span></a>). The six terms were from two GO branches and their child nodes (<a href="/pmc/articles/PMC4321789/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>): synapse (15 genes hit, FDR Q-val: 0.0055; three child nodes were also significant: synapse part, synaptic membrane, and presynaptic membrane) and neuron projection (16 genes hit, <em>FDR Q-val</em>: 0.013; one child node was also significant: axon).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4321789/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711974550880" class="figure"><a href="/pmc/articles/PMC4321789/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4321789_nihms-656151-f0007.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-656151-f0007.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4321789/bin/nihms-656151-f0007.jpg"></a></div><div id="largeobj_idm140711974550880" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4321789/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4321789/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">FIG. 1</a></div><!--caption a7--><div class="caption"><p id="__p4">Neurodevelopmental network predicted by proximal and distal regulatory region among the top hit of genome-wide SNPs and CNVs association.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>TABLE III</h3><!--caption a7--><div class="caption"><p id="__p7">Significant Enriched Gene Ontology Terms by Genes Associated With Regulatory Regions</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711973076064" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Term name</th><th rowspan="1" colspan="1" valign="top" align="right">Binom<sup><sup>a</sup></sup> rank</th><th rowspan="1" colspan="1" valign="top" align="left">Binom raw <em>P</em>-value</th><th rowspan="1" colspan="1" valign="top" align="right">
<em>Binom FDR Q-Val</em>
</th><th rowspan="1" colspan="1" valign="top" align="right">Binom fold enrichment</th><th rowspan="1" colspan="1" valign="top" align="right">Binom observed region hits</th><th rowspan="1" colspan="1" valign="top" align="right">Binom region set coverage (%)</th><th rowspan="1" colspan="1" valign="top" align="right">Hyper<sup><sup>a</sup></sup> rank</th><th rowspan="1" colspan="1" valign="top" align="right">
<em>Hyper FDR Q-Val</em>
</th><th rowspan="1" colspan="1" valign="top" align="right">Hyper fold enrichment</th><th rowspan="1" colspan="1" valign="top" align="right">Hyper observed gene hits</th><th rowspan="1" colspan="1" valign="top" align="right">Hyper total genes</th><th rowspan="1" colspan="1" valign="top" align="right">Hyper gene set coverage (%)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Presynaptic<br>membrane</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="left">2.4669E–11</td><td rowspan="1" colspan="1" valign="top" align="right">2.7654E–8</td><td rowspan="1" colspan="1" valign="top" align="right">9.4056</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="right">10.53</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">5.6208E–3</td><td rowspan="1" colspan="1" valign="top" align="right">13.5654</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="right">50</td><td rowspan="1" colspan="1" valign="top" align="right">3.85</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Synaptic<br>membrane</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">4.2028E–11</td><td rowspan="1" colspan="1" valign="top" align="right">2.3557E–8</td><td rowspan="1" colspan="1" valign="top" align="right">4.7787</td><td rowspan="1" colspan="1" valign="top" align="right">26</td><td rowspan="1" colspan="1" valign="top" align="right">17.11</td><td rowspan="1" colspan="1" valign="top" align="right">5</td><td rowspan="1" colspan="1" valign="top" align="right">1.3568E–2</td><td rowspan="1" colspan="1" valign="top" align="right">5.2175</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="right">195</td><td rowspan="1" colspan="1" valign="top" align="right">5.77</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Synapse part</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">1.2576E–9</td><td rowspan="1" colspan="1" valign="top" align="right">3.5243E–7</td><td rowspan="1" colspan="1" valign="top" align="right">3.5619</td><td rowspan="1" colspan="1" valign="top" align="right">30</td><td rowspan="1" colspan="1" valign="top" align="right">19.74</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">6.9311E–3</td><td rowspan="1" colspan="1" valign="top" align="right">4.2597</td><td rowspan="1" colspan="1" valign="top" align="right">13</td><td rowspan="1" colspan="1" valign="top" align="right">345</td><td rowspan="1" colspan="1" valign="top" align="right">8.33</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Synapse</td><td rowspan="1" colspan="1" valign="top" align="right">5</td><td rowspan="1" colspan="1" valign="top" align="left">6.4704E–8</td><td rowspan="1" colspan="1" valign="top" align="right">1.4507E–5</td><td rowspan="1" colspan="1" valign="top" align="right">2.8497</td><td rowspan="1" colspan="1" valign="top" align="right">32</td><td rowspan="1" colspan="1" valign="top" align="right">21.05</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">5.4656E–3</td><td rowspan="1" colspan="1" valign="top" align="right">3.6862</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="right">460</td><td rowspan="1" colspan="1" valign="top" align="right">9.62</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Axon</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="left">4.2006E–5</td><td rowspan="1" colspan="1" valign="top" align="right">4.2808E–3</td><td rowspan="1" colspan="1" valign="top" align="right">2.9590</td><td rowspan="1" colspan="1" valign="top" align="right">18</td><td rowspan="1" colspan="1" valign="top" align="right">11.84</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="right">1.4342E–2</td><td rowspan="1" colspan="1" valign="top" align="right">4.4506</td><td rowspan="1" colspan="1" valign="top" align="right">10</td><td rowspan="1" colspan="1" valign="top" align="right">254</td><td rowspan="1" colspan="1" valign="top" align="right">6.41</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Neuron projection</td><td rowspan="1" colspan="1" valign="top" align="right">13</td><td rowspan="1" colspan="1" valign="top" align="left">1.5147E–4</td><td rowspan="1" colspan="1" valign="top" align="right">1.3062E–2</td><td rowspan="1" colspan="1" valign="top" align="right">2.0886</td><td rowspan="1" colspan="1" valign="top" align="right">28</td><td rowspan="1" colspan="1" valign="top" align="right">18.42</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="right">1.2807E–2</td><td rowspan="1" colspan="1" valign="top" align="right">3.1788</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="right">569</td><td rowspan="1" colspan="1" valign="top" align="right">10.26</td></tr></tbody></table></div><div id="largeobj_idm140711973076064" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4321789/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><sup>a</sup>“Binom” represented binomial test over genomic regions; “Hyper” represented hypergeometic test over genes.</div></div></div></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">DISCUSSION</h2><p id="P26" class="p p-first">This GWAS of ADHD, comprising 1,040 cases and 963 controls, is the first performed in a homogeneous Han Chinese population. Although we did not find any genome-wide significant SNP or CNV variants, we did find significant evidence for a polygenic SNP component and an increased burden of rare CNVs.</p><p id="P27">The significant SNP-heritability implies that common variants are associated with ADHD, but that our sample is underpowered to detect them at the stringent significance level imposed by the genome-wide burden of multiple testing. The SNP-heritability in Han Chinese was 0.42 (SE 0.15). Although the point estimate is higher than for the larger European ancestry sample from the PGC-ADHD, 0.28 (SE 0.02), its high standard error shows that the estimates are not significantly different. The estimate of the SNP-genetic correlation (r<sub>g-SNP</sub>) was 0.39 (SE 0.15, <em>P</em> = 0.0072), which indicates that common SNP risk variants are shared by the Han Chinese and European Ancestry samples. To our knowledge, this is the first such correlation reported for any disease or disorder. The significant correlation indicates that ancient common variants associated with ADHD are shared between the ethnic groups.</p><p id="P28">However, the point estimate of the SNP correlation between Han Chinese and European Ancestry samples is lower than between sub-samples of the European Ancestry cohort. Specifically, when the PGC-ADHD data was split into two sub-samples, the <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M5" overflow="scroll"><mrow><msubsup><mi mathvariant="normal">h</mi><mrow><mi>SNP</mi><mo>−</mo><mi>EA</mi></mrow><mn>2</mn></msubsup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-5-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-5"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M5" overflow="scroll"><mrow><msubsup><mi mathvariant="normal">h</mi><mrow><mi>SNP</mi><mo>−</mo><mi>EA</mi></mrow><mn>2</mn></msubsup></mrow></math></script> estimates were 0.21 (SE 0.07) for the first sample and 0.41 (SE 0.03) for the other sample with a genetic correlation of 0.71 (SE 0.17) implying, as expected, more sharing of associated variants and/or higher linkage disequilibrium between causal variants and SNPs within than between ethnic populations.</p><p id="P29">Despite the fact that no individual SNPs reached association at genome-wide significance, our most significant findings implicated genes participating in neurodevelopmental processes such as cell adhesion, neuron migration, neurite outgrowth, neuronal morphogenesis, and synaptic plasticity. Similar sets of genes were also suggested by previous ADHD GWAS and a meta-analysis (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table SI</a>). For example, PRKG1 was implicated by <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654104">Neale et al. [2010a]</a>, ITGA1, CAMK2G, CAMK1D were implicated in the meta-analysis by <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654086">Neale et al. [2010b]</a>, and ITGAE and ITGA11 were implicated by <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654095">Lesch et al. [2008]</a>. Some of our top genes code for glutamate receptors and transporters. The same genes and gene family members (GRM7, GRIK1) were reported in the quantitative GWAS by <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654078">Lasky-Su et al. [2008]</a>, the meta-analysis by <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654073">Neale et al. [2010b]</a>, and the genome-wide CNV study by <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654081">Elia et al. [2012]</a>. Some genes related to transcription (ZNF544, ZNF385D, ZNF423, ZNF516, ZNF75A, DMRT2, FHIT, FOXP1, and MEIS2) were also implicated by <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654067">Lasky-Su et al. [2008]</a> and by <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654079">Neale et al. [2010b]</a>'s meta-analysis.</p><p id="P30">Although not significant after correcting for multiple comparisons, the pathways revealed by the INRICH analyses of associated SNPs involved neurobiological functions consistent with the prior findings discussed above. For example, neuron projection morphogenesis and neuron migration pathways were implicated by genes encoding adhesion molecules (e.g., GJA1, ITGA1, PRKG1). Neuron migration and axon guidance toward the target in the development of the nervous system involve interactions between molecules on the surface of the axon and those in the extra-cellular matrix [<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654087">Tsiotra et al., 1993</a>]. The endocytic vesicle membrane and synaptic transmission pathways involve glutamatergic synaptic function. The transcription related pathway is a ubiquitous biological process, if, as our findings suggest, it is implicated in ADHD's pathophysiology, any defects in the implicated transcription network must require other etiological factors to lead to a pathophysiologic state.</p><p id="P31">Because the GO “pathways” used by INRICH are based on bibliometric gene annotations rather than experimental data, we also used DAPPLE, which is based only on experimentally documented physical interactions among proteins. Considering the complexity of the genetic basis of ADHD, we hypothesized that both common and rare variants contribute to the disorder and act on similar functional classes of genes [<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654093">Poelmans et al., 2011</a>; <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654070">Stergiakouli et al., 2012</a>]. The DAPPLE analyses showed that the proteins implicated by our GWAS were significantly more likely to be interconnected with one another than expected by chance, suggesting that risk variants might exist in suites of genes involved in the underlying biological process of protein-protein interaction networks. The DAPPLE results are consistent with the INRICH results implicating three pathways: cell adhesion (<em>NXPH1–NRXN1</em>, <em>CNTN2–CNTNAP2–ZMIZ1</em>), glutamate synaptic development (<em>GRM7–PICK1–GRM3</em>, <em>PICK1–EPHA7</em>), and the transcription pathway (<em>TAF2–PAPOLA–POLR2F–MED27–MED20</em>, <em>POLR2F– NCL–RSL1D1–BYSL</em>).</p><p id="P32">Using the regulatory annotation of associated signals, GREAT depicted a clearer outline of associated genes, which encoded proteins comprising neuronal cellular components from the Neuron Projection and Synapse branches of the GO tree. Most of the genes from these pathways were consistent with the INRICH and DAPPLE findings; they encode adhesion molecules, glutamate receptors and proteins involved in axon and synapse development (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table SIII</a>).</p><p id="P33">All the above pathways are consistent with the hypothesis that mis-wiring of the brain during neurodevelopment might cause ADHD. Similar conclusions were drawn by <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654089">Lesch et al. [2008]</a> and <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654096">Franke et al. [2009]</a> based on the findings from existing GWAS, which suggested that neuronal spine formation and plasticity might underlie the pathophysiology of ADHD. Consistent with these ideas, a recent integration of ADHD GWAS findings found significant evidence for a neurodevelopmental network of directed neurite outgrowth [<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654083">Poelmans et al., 2011</a>]. Although our findings are consistent with prior work, they also provide evidence for a more comprehensive network, involving neuron migration, neurite outgrowth, neuronal morphogenesis, and synaptic plasticity, especially glutamatergic synaptic development. The glutamate system is a reasonable candidate for ADHD's pathophysiology as glutamate is the major excitatory neurotransmitter in the central nervous system, and regulates the catecholaminergic activity which has been implicated in ADHD by neurobiological [<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654068">Scassellati et al., 2012</a>] and treatment [<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_425654099">Faraone and Glatt, 2010</a>] studies.</p><p id="P34" class="p p-last">Although our findings are intriguing, we have only captured fragments of the puzzle of ADHD's etiology in this study. We could not paint the full picture. Our work must be considered in the context of its limitations. We had no genome-wide significant findings for any single variant, which might be due to the sample size. However, our bioinformatic and pathway analyses found some interesting genes and neurobiological pathways which implicate complex neurodevelopmental network underlying ADHD. Our finding of a significant polygenic component suggests that there are many common SNP variants with small effect sizes that increase the risk for ADHD. Individually, these SNPs will be difficult to detect with currently available sample sizes. </p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>sup data doc</h4><div class="sup-box half_rhythm" id="d36e1038"><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_data_doc.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(271K, doc)</sup></div></div><div class="sec suppmat" id="SD10"><h4>sup fig s8</h4><div class="sup-box half_rhythm" id="d36e1042"><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s8.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(291K, tif)</sup></div></div><div class="sec suppmat" id="SD2"><h4>sup data xls</h4><div class="sup-box half_rhythm" id="d36e1046"><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_data_xls.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(18K, xlsx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>sup fig s1</h4><div class="sup-box half_rhythm" id="d36e1050"><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s1.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(675K, tif)</sup></div></div><div class="sec suppmat" id="SD4"><h4>sup fig s2</h4><div class="sup-box half_rhythm" id="d36e1054"><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s2.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(675K, tif)</sup></div></div><div class="sec suppmat" id="SD5"><h4>sup fig s3</h4><div class="sup-box half_rhythm" id="d36e1058"><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s3.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(675K, tif)</sup></div></div><div class="sec suppmat" id="SD6"><h4>sup fig s4</h4><div class="sup-box half_rhythm" id="d36e1062"><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s4.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(675K, tif)</sup></div></div><div class="sec suppmat" id="SD7"><h4>sup fig s5</h4><div class="sup-box half_rhythm" id="d36e1066"><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s5.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(675K, tif)</sup></div></div><div class="sec suppmat" id="SD8"><h4>sup fig s6</h4><div class="sup-box half_rhythm" id="d36e1070"><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s6.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(675K, tif)</sup></div></div><div class="sec suppmat" id="SD9"><h4>sup fig s7</h4><div class="sup-box half_rhythm" id="d36e1074"><a href="/pmc/articles/PMC4321789/bin/NIHMS656151-supplement-sup_fig_s7.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(675K, tif)</sup></div></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p1">We thank all colleagues who helped collect and manage the data. We thank the patients and the family members who provided data for this project, which was supported by the following grants: National Natural Science Foundation of China (30800302 to Li Yang, 81071109 to Q.Q.), Chinese Ministry of Health grants (200802073 to Q.Q.), Ministry of Education Program for New Century Excellent Talents in University (NCET-11-0013 to Q.Q.), Ministry of Science and Technology grants (2007BAI17B03 to B.X., Y.W.). US National of Institute of Health Grants R13MH059126, R01MH62873, U01MH085518 and R01MH081803 to S.V.F., U01MH085515 to M.D., and K23MH066275-01 to J. Elia; and R01MH58277 to S.S.; Funding from the Australian Research council (FT0991360) and the Australian National Health and Medical Research Council to N.R.W. (1011506, 1047956). Institutional Development Award to the Center for Applied Genomics from the Children's Hospital of Philadelphia to H.H. Affymetrix Power Award, 2007 to B.F; NHMRC (Australia) and Sidney Sax Public Health Fellowship (443036) to SEM; Wellcome Trust, UK for sample collection to L.K. UMC Utrecht Genvlag Grant and Internal Grant of Radboud University, Nijmegen Medical Centre to J.B., the Deutsche Forschungsgemeinschaft (KFO 125, SFB 581, GRK 1156 to K.P.L., ME 1923/5-1, ME 1923/5-3, GRK 1389 to C.F., and J. Meyer, SCHA 542/10-3 to H. Schaäfer) and the BundesministeriumfürBildung und Forschung (BMBF 01GV0605 to K.P.L.). PND0080/2011, PI040524, PI080519, PI1100571, PI1101629, PI1201139, CP09/00119, 092330/31, 2009SGR1554, SAF2012-33484, and 2009SGR0971 (to M. Casas, J.A. Ramos-Quiroga, MònicaBayés, Bru Cormand, Cristina Sánchez-Mora, Marta Ribasés). Canadian Institutes of Health Research MOP 64277, 44070, 74699.</p><p id="__p2">Dr. Li Yang received research grant from Janssen Science Council of China. She had been a speaker for Janssen and received travel support from Eli Lilly. In the past year, Dr. Faraone received consulting income and/or research support from Shire, Akili Interactive Labs and Alcobra and research support from theNational Institutes of Health (NIH). He is also on the Clinical Advisory Board for Akili Interactive Labs. In previous years, he received consulting fees or was on Advisory Boards or participated in continuing medical education programs sponsored by: Shire, Alcobra, Otsuka, McNeil, Janssen, Novartis, Pfizer and Eli Lilly. Dr. Faraone receives royalties from books published by Guilford Press: Straight Talk about Your Child's Mental Health and Oxford University Press: Schizophrenia: The Facts. Dr. Yufeng Wang has served on an advisory board and received funding for research, lectures, and travel from Xi’an Janssen Pharmaceutical and the advisor of Eli lilly and Company.</p><p id="__p3">Grant sponsor: National Natural Science Foundation of China; Grant numbers: 30800302, 81071109; Grant sponsor: Chinese Ministry of Health; Grant number: 200802073; Grant sponsor: Ministry of Education Program for New Century Excellent Talents in University; Grant number: NCET-11-0013; Grant sponsor: Ministry of Science and Technology; Grant number: 2007BAI17B03; Grant sponsor: National Institute of Health Grants; Grant numbers: R13MH059126, R01MH62873, U01MH085518, R01MH081803, U01MH085515, K23MH066275-01, R01MH58277; Grant sponsor: Funding from the Australian Research council; Grant number: FT0991360; Grant sponsor: Australian National Health and Medical Research Council; Grant numbers: 1011506, 1047956; Grant sponsor: Children's Hospital of Philadelphia; Grant sponsor: Affymetrix Power Award; Grant sponsor: Sidney Sax Public Health Fellowship; Grant number: 443036; Grant sponsor: Wellcome Trust, UK; Grant sponsor: UMC Utrecht Genvlag Grant and Internal Grant of Radboud University, Nijmegen Medical Centre; Grant sponsor: Deutsche Forschungsgemeinschaft; Grant numbers: KFO 125, SFB 581, GRK 1156, ME 1923/5-1, ME 1923/5-3, GRK 1389; Grant sponsor: J. Meyer; Grant number: SCHA 542/10-3; Grant sponsor: BundesministeriumfürBildung und Forschung; Grant number: BMBF 01GV0605; Grant sponsor: Canadian Institutes of Health Research; Grant numbers: MOP 64277, 44070, 74699.</p></div></div><div id="idm140711980289712" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140711980289712title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P39" class="p p-first">SUPPORTING INFORMATION</p><p id="P40" class="p p-last">Additional supporting information may be found in the online version of this article at the publisher's web-site.</p><p></p><p class="fn sec" id="FN3"></p><p id="P41" class="p p-first">How to Cite this Article:</p><p id="P42" class="p">Yang L, Neale BM, Liu L, Lee SH, Wray NR, Ji N, Li H, Qian Q, Wang D, Li J, Faraone SV, Wang Y, Psychiatric GWAS Consortium: ADHD Subgroup. 2013. Polygenic Transmission and Complex Neuro Developmental Network for Attention Deficit Hyperactivity Disorder: Genome-Wide Association Study of Both Common and Rare Variants.</p><p id="P43" class="p p-last">Am J Med Genet Part B 162B:419–430.</p><p></p><p class="fn sec" id="FN4"></p><p id="P44" class="p p-first-last">The other authors reported no financial relationships with commercial interests.</p><p></p></div></div><div id="idm140711980554144" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140711980554144title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Barkley RA.  <span class="ref-journal">Attention-deficit hyperactivity disorder: A handbook for diagnosis and treatment.</span> xii. Guilford Press; New York: 1998. p. 628. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Attention-deficit+hyperactivity+disorder:+A+handbook+for+diagnosis+and+treatment&amp;author=RA+Barkley&amp;publication_year=1998&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Casey JP, Magalhaes T, Conroy JM, Regan R, Shah N, Anney R, Shields DC, Abrahams BS, Almeida J, Bacchelli E, et al.  A novel approach of homozygous haplotype sharing identifies candidate genes in autism spectrum disorder. <span><span class="ref-journal">Hum Genet. </span>2012;<span class="ref-vol">131</span>(4):565–579.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3303079/">PMC free article</a>]</span> [<a href="/pubmed/21996756" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=A+novel+approach+of+homozygous+haplotype+sharing+identifies+candidate+genes+in+autism+spectrum+disorder.&amp;author=JP+Casey&amp;author=T+Magalhaes&amp;author=JM+Conroy&amp;author=R+Regan&amp;author=N+Shah&amp;volume=131&amp;issue=4&amp;publication_year=2012&amp;pages=565-579&amp;pmid=21996756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Cherlyn SY, Woon PS, Liu JJ, Ong WY, Tsai GC, Sim K. Genetic association studies of glutamate, GABA and related genes in schizophrenia and bipolar disorder: A decade of advance. <span><span class="ref-journal">Neurosci Biobehav Rev. </span>2010;<span class="ref-vol">34</span>(6):958–977.</span> [<a href="/pubmed/20060416" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurosci+Biobehav+Rev&amp;title=Genetic+association+studies+of+glutamate,+GABA+and+related+genes+in+schizophrenia+and+bipolar+disorder:+A+decade+of+advance.&amp;author=SY+Cherlyn&amp;author=PS+Woon&amp;author=JJ+Liu&amp;author=WY+Ong&amp;author=GC+Tsai&amp;volume=34&amp;issue=6&amp;publication_year=2010&amp;pages=958-977&amp;pmid=20060416&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Dalvie S, Horn N, Nossek C, van der Merwe L, Stein DJ, Ramesar R. Psychosis and relapse in bipolar disorder are related to GRM3, DAOA, and GRIN2B genotype. <span><span class="ref-journal">Afr J Psychiatry (Johannesbg) </span>2010;<span class="ref-vol">13</span>(4):297–301.</span> [<a href="/pubmed/20957330" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Afr+J+Psychiatry+(Johannesbg)&amp;title=Psychosis+and+relapse+in+bipolar+disorder+are+related+to+GRM3,+DAOA,+and+GRIN2B+genotype.&amp;author=S+Dalvie&amp;author=N+Horn&amp;author=C+Nossek&amp;author=L+van+der+Merwe&amp;author=DJ+Stein&amp;volume=13&amp;issue=4&amp;publication_year=2010&amp;pages=297-301&amp;pmid=20957330&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Doherty JL, O'Donovan MC, Owen MJ. Recent genomic advances in schizophrenia. <span><span class="ref-journal">Clin Genet. </span>2012;<span class="ref-vol">81</span>(2):103–109.</span> [<a href="/pubmed/21895634" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Genet&amp;title=Recent+genomic+advances+in+schizophrenia.&amp;author=JL+Doherty&amp;author=MC+O'Donovan&amp;author=MJ+Owen&amp;volume=81&amp;issue=2&amp;publication_year=2012&amp;pages=103-109&amp;pmid=21895634&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Elia J, Gai X, Xie HM, Perin JC, Geiger E, Glessner JT, D'Arcy M, deBerardinis R, Frackelton E, Kim C, et al.  Rare structural variants found in attention-deficit hyperactivity disorder are preferentially associated with neurodevelopmental genes. <span><span class="ref-journal">Mol Psychiatry. </span>2010;<span class="ref-vol">15</span>(6):637–646.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2877197/">PMC free article</a>]</span> [<a href="/pubmed/19546859" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Rare+structural+variants+found+in+attention-deficit+hyperactivity+disorder+are+preferentially+associated+with+neurodevelopmental+genes.&amp;author=J+Elia&amp;author=X+Gai&amp;author=HM+Xie&amp;author=JC+Perin&amp;author=E+Geiger&amp;volume=15&amp;issue=6&amp;publication_year=2010&amp;pages=637-646&amp;pmid=19546859&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Elia J, Glessner JT, Wang K, Takahashi N, Shtir CJ, Hadley D, Sleiman PM, Zhang H, Kim CE, Robison R, et al.  Genome-wide copy number variation study associates metabotropic glutamate receptor gene networks with attention deficit hyperactivity disorder. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>(1):78–84.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4310555/">PMC free article</a>]</span> [<a href="/pubmed/22138692" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+copy+number+variation+study+associates+metabotropic+glutamate+receptor+gene+networks+with+attention+deficit+hyperactivity+disorder.&amp;author=J+Elia&amp;author=JT+Glessner&amp;author=K+Wang&amp;author=N+Takahashi&amp;author=CJ+Shtir&amp;volume=44&amp;issue=1&amp;publication_year=2012&amp;pages=78-84&amp;pmid=22138692&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Faraone SV, Glatt SJ. A comparison of the efficacy of medications for adult attention-deficit/hyperactivity disorder using meta-analysis of effect sizes. <span><span class="ref-journal">J Clin Psychiatry. </span>2010;<span class="ref-vol">71</span>(6):754–763.</span> [<a href="/pubmed/20051220" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Psychiatry&amp;title=A+comparison+of+the+efficacy+of+medications+for+adult+attention-deficit/hyperactivity+disorder+using+meta-analysis+of+effect+sizes.&amp;author=SV+Faraone&amp;author=SJ+Glatt&amp;volume=71&amp;issue=6&amp;publication_year=2010&amp;pages=754-763&amp;pmid=20051220&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Faraone SV, Sergeant J, Gillberg C, Biederman J. The worldwide prevalence of ADHD: Is it an American condition? <span><span class="ref-journal">World Psychiatry. </span>2003;<span class="ref-vol">2</span>(2):104–113.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1525089/">PMC free article</a>]</span> [<a href="/pubmed/16946911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=World+Psychiatry&amp;title=The+worldwide+prevalence+of+ADHD:+Is+it+an+American+condition?&amp;author=SV+Faraone&amp;author=J+Sergeant&amp;author=C+Gillberg&amp;author=J+Biederman&amp;volume=2&amp;issue=2&amp;publication_year=2003&amp;pages=104-113&amp;pmid=16946911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Fisher SE, Francks C, McCracken JT, McGough JJ, Marlow AJ, MacPhie IL, Newbury DF, Crawford LR, Palmer CG, Woodward JA, et al.  A genomewide scan for loci involved in attention-deficit/hyperactivity disorder. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">70</span>(5):1183–1196.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC447594/">PMC free article</a>]</span> [<a href="/pubmed/11923911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genomewide+scan+for+loci+involved+in+attention-deficit/hyperactivity+disorder.&amp;author=SE+Fisher&amp;author=C+Francks&amp;author=JT+McCracken&amp;author=JJ+McGough&amp;author=AJ+Marlow&amp;volume=70&amp;issue=5&amp;publication_year=2002&amp;pages=1183-1196&amp;pmid=11923911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Franke B, Neale BM, Faraone SV. Genome-wide association studies in ADHD. <span><span class="ref-journal">Hum Genet. </span>2009;<span class="ref-vol">126</span>(1):13–50.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3774416/">PMC free article</a>]</span> [<a href="/pubmed/19384554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Genome-wide+association+studies+in+ADHD.&amp;author=B+Franke&amp;author=BM+Neale&amp;author=SV+Faraone&amp;volume=126&amp;issue=1&amp;publication_year=2009&amp;pages=13-50&amp;pmid=19384554&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Fujii K, Maeda K, Hikida T, Mustafa AK, Balkissoon R, Xia J, Yamada T, Ozeki Y, Kawahara R, Okawa M, et al.  Serine racemase binds to PICK1: Potential relevance to schizophrenia. <span><span class="ref-journal">Mol Psychiatry. </span>2006;<span class="ref-vol">11</span>(2):150–157.</span> [<a href="/pubmed/16314870" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Serine+racemase+binds+to+PICK1:+Potential+relevance+to+schizophrenia.&amp;author=K+Fujii&amp;author=K+Maeda&amp;author=T+Hikida&amp;author=AK+Mustafa&amp;author=R+Balkissoon&amp;volume=11&amp;issue=2&amp;publication_year=2006&amp;pages=150-157&amp;pmid=16314870&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Ganda C, Schwab SG, Amir N, Heriani H, Irmansyah I, Kusumawardhani A, Nasrun M, Widyawati I, Maier W, Wildenauer DB. A family-based association study of DNA sequence variants in GRM7 with schizophrenia in an Indonesian population. <span><span class="ref-journal">Int J Neuropsychopharmacol. </span>2009;<span class="ref-vol">12</span>(9):1283–1289.</span> [<a href="/pubmed/19638256" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Neuropsychopharmacol&amp;title=A+family-based+association+study+of+DNA+sequence+variants+in+GRM7+with+schizophrenia+in+an+Indonesian+population.&amp;author=C+Ganda&amp;author=SG+Schwab&amp;author=N+Amir&amp;author=H+Heriani&amp;author=I+Irmansyah&amp;volume=12&amp;issue=9&amp;publication_year=2009&amp;pages=1283-1289&amp;pmid=19638256&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Hong CJ, Liao DL, Shih HL, Tsai SJ. Association study of PICK1 rs3952 polymorphism and schizophrenia. <span><span class="ref-journal">Neuroreport. </span>2004;<span class="ref-vol">15</span>(12):1965–1967.</span> [<a href="/pubmed/15305146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroreport&amp;title=Association+study+of+PICK1+rs3952+polymorphism+and+schizophrenia.&amp;author=CJ+Hong&amp;author=DL+Liao&amp;author=HL+Shih&amp;author=SJ+Tsai&amp;volume=15&amp;issue=12&amp;publication_year=2004&amp;pages=1965-1967&amp;pmid=15305146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Lasky-Su J, Neale BM, Franke B, Anney RJ, Zhou K, Maller JB, Vasquez AA, Chen W, Asherson P, Buitelaar J, et al.  Genome-wide association scan of quantitative traits for attention deficit hyperactivity disorder identifies novel associations and confirms candidate gene associations. <span><span class="ref-journal">Am J Med Genet Part B. </span>2008;<span class="ref-vol">147B</span>(8):1345–1354.</span> [<a href="/pubmed/18821565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+Part+B&amp;title=Genome-wide+association+scan+of+quantitative+traits+for+attention+deficit+hyperactivity+disorder+identifies+novel+associations+and+confirms+candidate+gene+associations.&amp;author=J+Lasky-Su&amp;author=BM+Neale&amp;author=B+Franke&amp;author=RJ+Anney&amp;author=K+Zhou&amp;volume=147B&amp;issue=8&amp;publication_year=2008&amp;pages=1345-1354&amp;pmid=18821565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Lee SH, Wray NR, Goddard ME, Visscher PM. Estimating missing heritability for disease from genome-wide association studies. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>(3):294–305.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3059431/">PMC free article</a>]</span> [<a href="/pubmed/21376301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Estimating+missing+heritability+for+disease+from+genome-wide+association+studies.&amp;author=SH+Lee&amp;author=NR+Wray&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;issue=3&amp;publication_year=2011&amp;pages=294-305&amp;pmid=21376301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Lee PH, O'Dushlaine C, Thomas B, Purcell SM. INRICH: Interval-based enrichment analysis for genome wide association studies. <span><span class="ref-journal">Bioinformatics. </span>2012a;<span class="ref-vol">28</span>(13):1797–1799.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3381960/">PMC free article</a>]</span> [<a href="/pubmed/22513993" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=INRICH:+Interval-based+enrichment+analysis+for+genome+wide+association+studies.&amp;author=PH+Lee&amp;author=C+O'Dushlaine&amp;author=B+Thomas&amp;author=SM+Purcell&amp;volume=28&amp;issue=13&amp;publication_year=2012a&amp;pages=1797-1799&amp;pmid=22513993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Lee SH, Yang J, Goddard ME, Visscher PM, Wray NR. Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood. <span><span class="ref-journal">Bioinformatics. </span>2012b;<span class="ref-vol">28</span>(19):2540–2542.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3463125/">PMC free article</a>]</span> [<a href="/pubmed/22843982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Estimation+of+pleiotropy+between+complex+diseases+using+single-nucleotide+polymorphism-derived+genomic+relationships+and+restricted+maximum+likelihood.&amp;author=SH+Lee&amp;author=J+Yang&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;author=NR+Wray&amp;volume=28&amp;issue=19&amp;publication_year=2012b&amp;pages=2540-2542&amp;pmid=22843982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Lesch KP, Timmesfeld N, Renner TJ, Halperin R, Roser C, Nguyen TT, Craig DW, Romanos J, Heine M, Meyer J, et al.  Molecular genetics of adult ADHD: Converging evidence from genome-wide association and extended pedigree linkage studies. <span><span class="ref-journal">J Neural Transmembr. </span>2008;<span class="ref-vol">115</span>(11):1573–1585.</span> [<a href="/pubmed/18839057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neural+Transmembr&amp;title=Molecular+genetics+of+adult+ADHD:+Converging+evidence+from+genome-wide+association+and+extended+pedigree+linkage+studies.&amp;author=KP+Lesch&amp;author=N+Timmesfeld&amp;author=TJ+Renner&amp;author=R+Halperin&amp;author=C+Roser&amp;volume=115&amp;issue=11&amp;publication_year=2008&amp;pages=1573-1585&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">McLean CY, Bristor D, Hiller M, Clarke SL, Schaar BT, Lowe CB, Wenger AM, Bejerano G. GREAT improves functional interpretation of cis-regulatory regions. <span><span class="ref-journal">Nat Biotechnol. </span>2010;<span class="ref-vol">28</span>(5):495–501.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4840234/">PMC free article</a>]</span> [<a href="/pubmed/20436461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Biotechnol&amp;title=GREAT+improves+functional+interpretation+of+cis-regulatory+regions.&amp;author=CY+McLean&amp;author=D+Bristor&amp;author=M+Hiller&amp;author=SL+Clarke&amp;author=BT+Schaar&amp;volume=28&amp;issue=5&amp;publication_year=2010&amp;pages=495-501&amp;pmid=20436461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Neale BM, Lasky-Su J, Anney R, Franke B, Zhou K, Maller JB, Vasquez AA, Asherson P, Chen W, Banaschewski T, et al.  Genome-wide association scan of attention deficit hyperactivity disorder. <span><span class="ref-journal">Am J Med Genet Part B. </span>2008;<span class="ref-vol">147B</span>(8):1337–1344.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2831205/">PMC free article</a>]</span> [<a href="/pubmed/18980221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+Part+B&amp;title=Genome-wide+association+scan+of+attention+deficit+hyperactivity+disorder.&amp;author=BM+Neale&amp;author=J+Lasky-Su&amp;author=R+Anney&amp;author=B+Franke&amp;author=K+Zhou&amp;volume=147B&amp;issue=8&amp;publication_year=2008&amp;pages=1337-1344&amp;pmid=18980221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Neale BM, Medland S, Ripke S, Anney RJ, Asherson P, Buitelaar J, Franke B, Gill M, Kent L, Holmans P, et al.  Case-control genome-wide association study of attention-deficit/hyperactivity disorder. <span><span class="ref-journal">J Am Acad Child Adolesc Psychiatry. </span>2010a;<span class="ref-vol">49</span>(9):906–920.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928577/">PMC free article</a>]</span> [<a href="/pubmed/20732627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Child+Adolesc+Psychiatry&amp;title=Case-control+genome-wide+association+study+of+attention-deficit/hyperactivity+disorder.&amp;author=BM+Neale&amp;author=S+Medland&amp;author=S+Ripke&amp;author=RJ+Anney&amp;author=P+Asherson&amp;volume=49&amp;issue=9&amp;publication_year=2010a&amp;pages=906-920&amp;pmid=20732627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Neale BM, Medland SE, Ripke S, Asherson P, Franke B, Lesch KP, Faraone SV, Nguyen TT, Schafer H, Holmans P, et al.  Meta-analysis of genome-wide association studies of attention-deficit/hyperactivity disorder. <span><span class="ref-journal">J Am Acad Child Adolesc Psychiatry. </span>2010b;<span class="ref-vol">49</span>(9):884–897.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928252/">PMC free article</a>]</span> [<a href="/pubmed/20732625" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Child+Adolesc+Psychiatry&amp;title=Meta-analysis+of+genome-wide+association+studies+of+attention-deficit/hyperactivity+disorder.&amp;author=BM+Neale&amp;author=SE+Medland&amp;author=S+Ripke&amp;author=P+Asherson&amp;author=B+Franke&amp;volume=49&amp;issue=9&amp;publication_year=2010b&amp;pages=884-897&amp;pmid=20732625&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Ohtsuki T, Koga M, Ishiguro H, Horiuchi Y, Arai M, Niizato K, Itokawa M, Inada T, Iwata N, Iritani S, et al.  A polymorphism of the metabotropic glutamate receptor mGluR7 (GRM7) gene is associated with schizophrenia. <span><span class="ref-journal">Schizophr Res. </span>2008;<span class="ref-vol">101</span>(1–3):9–16.</span> [<a href="/pubmed/18329248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Schizophr+Res&amp;title=A+polymorphism+of+the+metabotropic+glutamate+receptor+mGluR7+(GRM7)+gene+is+associated+with+schizophrenia.&amp;author=T+Ohtsuki&amp;author=M+Koga&amp;author=H+Ishiguro&amp;author=Y+Horiuchi&amp;author=M+Arai&amp;volume=101&amp;issue=1–3&amp;publication_year=2008&amp;pages=9-16&amp;pmid=18329248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Poelmans G, Pauls DL, Buitelaar JK, Franke B. Integrated genome-wide association study findings: identification of a neurodevelopmental network for attention deficit hyperactivity disorder. <span><span class="ref-journal">Am J Psychiatry. </span>2011;<span class="ref-vol">168</span>(4):365–377.</span> [<a href="/pubmed/21324949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=Integrated+genome-wide+association+study+findings:+identification+of+a+neurodevelopmental+network+for+attention+deficit+hyperactivity+disorder.&amp;author=G+Poelmans&amp;author=DL+Pauls&amp;author=JK+Buitelaar&amp;author=B+Franke&amp;volume=168&amp;issue=4&amp;publication_year=2011&amp;pages=365-377&amp;pmid=21324949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Pulver AE, Karayiorgou M, Wolyniec PS, Lasseter VK, Kasch L, Nestadt G, Antonarakis S, Housman D, Kazazian HH, Meyers D, et al.  Sequential strategy to identify a susceptibility gene for schizophrenia: report of potential linkage on chromosome 22q12-q13.1: Part 1. <span><span class="ref-journal">Am J Med Genet. </span>1994;<span class="ref-vol">54</span>(1):36–43.</span> [<a href="/pubmed/8178837" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Sequential+strategy+to+identify+a+susceptibility+gene+for+schizophrenia:+report+of+potential+linkage+on+chromosome+22q12-q13.1:+Part+1.&amp;author=AE+Pulver&amp;author=M+Karayiorgou&amp;author=PS+Wolyniec&amp;author=VK+Lasseter&amp;author=L+Kasch&amp;volume=54&amp;issue=1&amp;publication_year=1994&amp;pages=36-43&amp;pmid=8178837&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, et al.  PLINK: A tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>(3):559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+A+tool+set+for+whole-genome+association+and+population-based+linkage+analyses.&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Rossin EJ, Lage K, Raychaudhuri S, Xavier RJ, Tatar D, Benita Y, International Inflammatory Bowel Disease Genetics C. Cotsapas C, Daly MJ. Proteins encoded in genomic regions associated with immune-mediated disease physically interact and suggest underlying biology. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>(1):e1001273.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3020935/">PMC free article</a>]</span> [<a href="/pubmed/21249183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Proteins+encoded+in+genomic+regions+associated+with+immune-mediated+disease+physically+interact+and+suggest+underlying+biology.&amp;author=EJ+Rossin&amp;author=K+Lage&amp;author=S+Raychaudhuri&amp;author=RJ+Xavier&amp;author=D+Tatar&amp;volume=7&amp;issue=1&amp;publication_year=2011&amp;pages=e1001273&amp;pmid=21249183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Salyakina D, Cukier HN, Lee JM, Sacharow S, Nations LD, Ma D, Jaworski JM, Konidari I, Whitehead PL, Wright HH, et al.  Copy number variants in extended autism spectrum disorder families reveal candidates potentially involved in autism risk. <span><span class="ref-journal">PLoS ONE. </span>2011;<span class="ref-vol">6</span>(10):e26049.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3189231/">PMC free article</a>]</span> [<a href="/pubmed/22016809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Copy+number+variants+in+extended+autism+spectrum+disorder+families+reveal+candidates+potentially+involved+in+autism+risk.&amp;author=D+Salyakina&amp;author=HN+Cukier&amp;author=JM+Lee&amp;author=S+Sacharow&amp;author=LD+Nations&amp;volume=6&amp;issue=10&amp;publication_year=2011&amp;pages=e26049&amp;pmid=22016809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">Scassellati C, Bonvicini C, Faraone SV, Gennarelli M. Biomarkers and attention-deficit/hyperactivity disorder: A systematic review and meta-analyses. <span><span class="ref-journal">J Am Acad Child Adolesc Psychiatry. </span>2012;<span class="ref-vol">51</span>(10):1003–1019. e20.</span> [<a href="/pubmed/23021477" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Child+Adolesc+Psychiatry&amp;title=Biomarkers+and+attention-deficit/hyperactivity+disorder:+A+systematic+review+and+meta-analyses.&amp;author=C+Scassellati&amp;author=C+Bonvicini&amp;author=SV+Faraone&amp;author=M+Gennarelli&amp;volume=51&amp;issue=10&amp;publication_year=2012&amp;pages=1003-1019&amp;pmid=23021477&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="element-citation">Shibata H, Tani A, Chikuhara T, Kikuta R, Sakai M, Ninomiya H, Tashiro N, Iwata N, Ozaki N, Fukumaki Y. Association study of polymorphisms in the group III metabotropic glutamate receptor genes, GRM4 and GRM7, with schizophrenia. <span><span class="ref-journal">Psychiatry Res. </span>2009;<span class="ref-vol">167</span>(1–2):88–96.</span> [<a href="/pubmed/19351574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychiatry+Res&amp;title=Association+study+of+polymorphisms+in+the+group+III+metabotropic+glutamate+receptor+genes,+GRM4+and+GRM7,+with+schizophrenia.&amp;author=H+Shibata&amp;author=A+Tani&amp;author=T+Chikuhara&amp;author=R+Kikuta&amp;author=M+Sakai&amp;volume=167&amp;issue=1–2&amp;publication_year=2009&amp;pages=88-96&amp;pmid=19351574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="element-citation">Smalley SL, Kustanovich V, Minassian SL, Stone JL, Ogdie MN, McGough JJ, McCracken JT, MacPhie IL, Francks C, Fisher SE, et al.  Genetic linkage of attention-deficit/hyperactivity disorder on chromosome 16p13, in a region implicated in autism. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">71</span>(4):959–963.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC378550/">PMC free article</a>]</span> [<a href="/pubmed/12187510" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genetic+linkage+of+attention-deficit/hyperactivity+disorder+on+chromosome+16p13,+in+a+region+implicated+in+autism.&amp;author=SL+Smalley&amp;author=V+Kustanovich&amp;author=SL+Minassian&amp;author=JL+Stone&amp;author=MN+Ogdie&amp;volume=71&amp;issue=4&amp;publication_year=2002&amp;pages=959-963&amp;pmid=12187510&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="element-citation">Stergiakouli E, Hamshere M, Holmans P, Langley K, Zaharieva I, Hawi Z, Kent L, Gill M, Williams N, Owen MJ, et al.  Investigating the contribution of common genetic variants to the risk and pathogenesis of ADHD. <span><span class="ref-journal">Am J Psychiatry. </span>2012;<span class="ref-vol">169</span>(2):186–194.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3601404/">PMC free article</a>]</span> [<a href="/pubmed/22420046" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=Investigating+the+contribution+of+common+genetic+variants+to+the+risk+and+pathogenesis+of+ADHD.&amp;author=E+Stergiakouli&amp;author=M+Hamshere&amp;author=P+Holmans&amp;author=K+Langley&amp;author=I+Zaharieva&amp;volume=169&amp;issue=2&amp;publication_year=2012&amp;pages=186-194&amp;pmid=22420046&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="element-citation">Tsiotra PC, Karagogeos D, Theodorakis K, Michaelidis TM, Modi WS, Furley AJ, Jessell TM, Papamatheakis J. Isolation of the cDNA and chromosomal localization of the gene (TAX1) encoding the human axonal glycoprotein TAG-1. <span><span class="ref-journal">Genomics. </span>1993;<span class="ref-vol">18</span>(3):562–567.</span> [<a href="/pubmed/8307567" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Isolation+of+the+cDNA+and+chromosomal+localization+of+the+gene+(TAX1)+encoding+the+human+axonal+glycoprotein+TAG-1.&amp;author=PC+Tsiotra&amp;author=D+Karagogeos&amp;author=K+Theodorakis&amp;author=TM+Michaelidis&amp;author=WS+Modi&amp;volume=18&amp;issue=3&amp;publication_year=1993&amp;pages=562-567&amp;pmid=8307567&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="element-citation">Williams NM, Zaharieva I, Martin A, Langley K, Mantripragada K, Fossdal R, Stefansson H, Stefansson K, Magnusson P, Gudmundsson OO, et al.  Rare chromosomal deletions and duplications in attention-deficit hyperactivity disorder: A genome-wide analysis. <span><span class="ref-journal">Lancet. </span>2010;<span class="ref-vol">376</span>(9750):1401–1408.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2965350/">PMC free article</a>]</span> [<a href="/pubmed/20888040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Rare+chromosomal+deletions+and+duplications+in+attention-deficit+hyperactivity+disorder:+A+genome-wide+analysis.&amp;author=NM+Williams&amp;author=I+Zaharieva&amp;author=A+Martin&amp;author=K+Langley&amp;author=K+Mantripragada&amp;volume=376&amp;issue=9750&amp;publication_year=2010&amp;pages=1401-1408&amp;pmid=20888040&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="element-citation">Williams NM, Franke B, Mick E, Anney RJ, Freitag CM, Gill M, Thapar A, O'Donovan MC, Owen MJ, Holmans P, et al.  Genome-wide analysis of copy number variants in attention deficit hyperactivity disorder: the role of rare variants and duplications at 15q13.3. <span><span class="ref-journal">Am J Psychiatry. </span>2012;<span class="ref-vol">169</span>(2):195–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3601405/">PMC free article</a>]</span> [<a href="/pubmed/22420048" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=Genome-wide+analysis+of+copy+number+variants+in+attention+deficit+hyperactivity+disorder:+the+role+of+rare+variants+and+duplications+at+15q13.3.&amp;author=NM+Williams&amp;author=B+Franke&amp;author=E+Mick&amp;author=RJ+Anney&amp;author=CM+Freitag&amp;volume=169&amp;issue=2&amp;publication_year=2012&amp;pages=195-204&amp;pmid=22420048&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="element-citation">Yang L, Wang YF, Qian QJ, Biederman J, Faraone SV. DSM-IV subtypes of ADHD in a Chinese outpatient sample. <span><span class="ref-journal">J Am Acad Child Adolesc Psychiatry. </span>2004;<span class="ref-vol">43</span>(3):248–250.</span> [<a href="/pubmed/15076256" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Child+Adolesc+Psychiatry&amp;title=DSM-IV+subtypes+of+ADHD+in+a+Chinese+outpatient+sample.&amp;author=L+Yang&amp;author=YF+Wang&amp;author=QJ+Qian&amp;author=J+Biederman&amp;author=SV+Faraone&amp;volume=43&amp;issue=3&amp;publication_year=2004&amp;pages=248-250&amp;pmid=15076256&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="element-citation">Yang J, Benyamin B, McEvoy BP, Gordon S, Henders AK, Nyholt DR, Madden PA, Heath AC, Martin NG, Montgomery GW, et al.  Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:565–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height.&amp;author=J+Yang&amp;author=B+Benyamin&amp;author=BP+McEvoy&amp;author=S+Gordon&amp;author=AK+Henders&amp;volume=42&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R39"><span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: A tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>(1):76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+A+tool+for+genome-wide+complex+trait+analysis.&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;issue=1&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="element-citation">Zhou K, Dempfle A, Arcos-Burgos M, Bakker SC, Banaschewski T, Biederman J, Buitelaar J, Castellanos FX, Doyle A, Ebstein RP, et al.  Meta-analysis of genome-wide linkage scans of attention deficit hyperactivity disorder. <span><span class="ref-journal">Am J Med Genet Part B. </span>2008;<span class="ref-vol">147B</span>(8):1392–1398.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2890047/">PMC free article</a>]</span> [<a href="/pubmed/18988193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+Part+B&amp;title=Meta-analysis+of+genome-wide+linkage+scans+of+attention+deficit+hyperactivity+disorder.&amp;author=K+Zhou&amp;author=A+Dempfle&amp;author=M+Arcos-Burgos&amp;author=SC+Bakker&amp;author=T+Banaschewski&amp;volume=147B&amp;issue=8&amp;publication_year=2008&amp;pages=1392-1398&amp;pmid=18988193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4321789&amp;issue-id=249072&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140711977889632title">Abstract</a></li><li><a href="#S1title">INTRODUCTION</a></li><li><a href="#S2title">MATERIALS AND METHODS</a></li><li><a href="#S8title">RESULTS</a></li><li><a href="#S12title">DISCUSSION</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S13title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140711980289712title">Footnotes</a></li><li><a href="#idm140711980554144title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4321789/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4321789/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4321789/pdf/nihms-656151.pdf">PDF (468K)</a></li> | <li><a href="#" data-citationid="PMC4321789" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4321789%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4321789%2F&amp;text=Polygenic%20Transmission%20and%20Complex%20Neuro%20developmental%20Network%20for%20Attention%20Deficit%20Hyperactivity%20Disorder%3A%20Genome-Wide%20Association%20Study%20of%20Both%20Common%20and%20Rare%20Variants" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4321789%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23728934" data-aiid="4321789" data-aid="4321789" data-iid="249072" data-domainid="319" data-domain="nihpa" data-accid="PMC4321789" data-md5="631ffe29346df8c1b689164470606347"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T11:00:22-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B5102DD55DB9100000000044002B1&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4321789%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4321789/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size2, monospace;"></div></div></body></html>