<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4141024/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genetic Determinants of P Wave Duration and PR Segment</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4141024/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Circulation. Cardiovascular genetics"><meta name="citation_title" content="Genetic Determinants of P Wave Duration and PR Segment"><meta name="citation_authors" content="Niek Verweij, Irene Mateo Leach, Malou van den Boogaard, Dirk J. van Veldhuisen, Vincent M. Christoffels, LifeLines Cohort Study, Hans L. Hillege, Wiek H. van Gilst, Phil Barnett, Rudolf A. de Boer, Pim van der Harst"><meta name="citation_date" content="August 2014"><meta name="citation_issue" content="4"><meta name="citation_volume" content="7"><meta name="citation_firstpage" content="475"><meta name="citation_doi" content="10.1161/CIRCGENETICS.113.000373"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4141024/?report=abstract"><meta name="citation_pmid" content="24850809"><meta name="DC.Title" content="Genetic Determinants of P Wave Duration and PR Segment"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Niek Verweij"><meta name="DC.Contributor" content="Irene Mateo Leach"><meta name="DC.Contributor" content="Malou van den Boogaard"><meta name="DC.Contributor" content="Dirk J. van Veldhuisen"><meta name="DC.Contributor" content="Vincent M. Christoffels"><meta name="DC.Contributor" content="LifeLines Cohort Study"><meta name="DC.Contributor" content="Hans L. Hillege"><meta name="DC.Contributor" content="Wiek H. van Gilst"><meta name="DC.Contributor" content="Phil Barnett"><meta name="DC.Contributor" content="Rudolf A. de Boer"><meta name="DC.Contributor" content="Pim van der Harst"><meta name="DC.Date" content="2014 Aug"><meta name="DC.Identifier" content="10.1161/CIRCGENETICS.113.000373"><meta name="DC.Language" content="en"><meta property="og:title" content="Genetic Determinants of P Wave Duration and PR Segment"><meta property="og:type" content="article"><meta property="og:description" content="The PR interval on the electrocardiogram reflects atrial depolarization and AV nodal delay which can be partially differentiated by P wave duration and PR segment, respectively. GWAS have identified a number of genetic loci for PR interval but it remains ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4141024/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4141024/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4141024/pdf/nihms-598853.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B1394DD5691F100000000081E04CF.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4141024%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="nuccore" class="last">Nucleotide</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4141024</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Circ Cardiovasc Genet</a></span>. Author manuscript; available in PMC 2015 Aug 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=24850809" target="pmc_ext" ref="reftype=publisher&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Circ Cardiovasc Genet. 2014 Aug; 7(4): 475–481. </span></a></div><span class="fm-vol-iss-date">Published online 2014 May 21. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1161%2FCIRCGENETICS.113.000373" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1161/CIRCGENETICS.113.000373</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4141024</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS598853</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24850809">24850809</a></div></div></div></div><h1 class="content-title">Genetic Determinants of P Wave Duration and PR Segment</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Verweij%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809" class="affpopup" co-rid="_co_idm139808681386560" co-class="co-affbox">Niek Verweij</a>, Msc,<sup>1</sup> <a href="/pubmed/?term=Leach%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809" class="affpopup" co-rid="_co_idm139808667686656" co-class="co-affbox">Irene Mateo Leach</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=van%20den%20Boogaard%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809" class="affpopup" co-rid="_co_idm139808680411888" co-class="co-affbox">Malou van den Boogaard</a>, Msc,<sup>2</sup> <a href="/pubmed/?term=van%20Veldhuisen%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809" class="affpopup" co-rid="_co_idm139808663629936" co-class="co-affbox">Dirk J. van Veldhuisen</a>, MD, PhD,<sup>1</sup> <a href="/pubmed/?term=Christoffels%20VM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809" class="affpopup" co-rid="_co_idm139808674007536" co-class="co-affbox">Vincent M. Christoffels</a>, PhD,<sup>2</sup> LifeLines Cohort Study,<sup>3</sup> <a href="/pubmed/?term=Hillege%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809" class="affpopup" co-rid="_co_idm139808678116352" co-class="co-affbox">Hans L. Hillege</a>, MD, PhD,<sup>1</sup> <a href="/pubmed/?term=van%20Gilst%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809" class="affpopup" co-rid="_co_idm139808664457056" co-class="co-affbox">Wiek H. van Gilst</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Barnett%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809" class="affpopup" co-rid="_co_idm139808662957600" co-class="co-affbox">Phil Barnett</a>, PhD,<sup>2</sup> <a href="/pubmed/?term=de%20Boer%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809" class="affpopup" co-rid="_co_idm139808662955360" co-class="co-affbox">Rudolf A. de Boer</a>, MD, PhD,<sup>1</sup> and  <a href="/pubmed/?term=van%20der%20Harst%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809" class="affpopup" co-rid="_co_idm139808673814384" co-class="co-affbox">Pim van der Harst</a>, MD, PhD<sup>1,</sup><sup>4,</sup><sup>5</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139808681386560"><h3 class="no_margin">Niek Verweij</h3><p><sup>1</sup>Department of Cardiology, University Medical Center Groningen, Groningen </p><div>Find articles by <a href="/pubmed/?term=Verweij%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">Niek Verweij</a></div></div><div id="_co_idm139808667686656"><h3 class="no_margin">Irene Mateo Leach</h3><p><sup>1</sup>Department of Cardiology, University Medical Center Groningen, Groningen </p><div>Find articles by <a href="/pubmed/?term=Leach%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">Irene Mateo Leach</a></div></div><div id="_co_idm139808680411888"><h3 class="no_margin">Malou van den Boogaard</h3><p><sup>2</sup>Department of Anatomy, Embryology and Physiology, University of Amsterdam, Academic Medical Center, Amsterdam </p><div>Find articles by <a href="/pubmed/?term=van%20den%20Boogaard%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">Malou van den Boogaard</a></div></div><div id="_co_idm139808663629936"><h3 class="no_margin">Dirk J. van Veldhuisen</h3><p><sup>1</sup>Department of Cardiology, University Medical Center Groningen, Groningen </p><div>Find articles by <a href="/pubmed/?term=van%20Veldhuisen%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">Dirk J. van Veldhuisen</a></div></div><div id="_co_idm139808674007536"><h3 class="no_margin">Vincent M. Christoffels</h3><p><sup>2</sup>Department of Anatomy, Embryology and Physiology, University of Amsterdam, Academic Medical Center, Amsterdam </p><div>Find articles by <a href="/pubmed/?term=Christoffels%20VM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">Vincent M. Christoffels</a></div></div><div id="_co_idm139808678117712"><h3 class="no_margin">LifeLines Cohort Study</h3><p><sup>3</sup>LifeLines Cohort Study, Groningen </p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">LifeLines Cohort Study</a></div></div><div id="_co_idm139808678116352"><h3 class="no_margin">Hans L. Hillege</h3><p><sup>1</sup>Department of Cardiology, University Medical Center Groningen, Groningen </p><div>Find articles by <a href="/pubmed/?term=Hillege%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">Hans L. Hillege</a></div></div><div id="_co_idm139808664457056"><h3 class="no_margin">Wiek H. van Gilst</h3><p><sup>1</sup>Department of Cardiology, University Medical Center Groningen, Groningen </p><div>Find articles by <a href="/pubmed/?term=van%20Gilst%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">Wiek H. van Gilst</a></div></div><div id="_co_idm139808662957600"><h3 class="no_margin">Phil Barnett</h3><p><sup>2</sup>Department of Anatomy, Embryology and Physiology, University of Amsterdam, Academic Medical Center, Amsterdam </p><div>Find articles by <a href="/pubmed/?term=Barnett%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">Phil Barnett</a></div></div><div id="_co_idm139808662955360"><h3 class="no_margin">Rudolf A. de Boer</h3><p><sup>1</sup>Department of Cardiology, University Medical Center Groningen, Groningen </p><div>Find articles by <a href="/pubmed/?term=de%20Boer%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">Rudolf A. de Boer</a></div></div><div id="_co_idm139808673814384"><h3 class="no_margin">Pim van der Harst</h3><p><sup>1</sup>Department of Cardiology, University Medical Center Groningen, Groningen </p><p><sup>4</sup>Department of Genetics, University of Groningen, University Medical Center Groningen, Groningen </p><p><sup>5</sup>Durrer Center for Cardiogenetic Research, ICIN-Netherlands Heart Institute, Utrecht, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=van%20der%20Harst%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24850809">Pim van der Harst</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139808672055376_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139808672055376_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139808672055376_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Cardiology, University Medical Center Groningen, Groningen </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Anatomy, Embryology and Physiology, University of Amsterdam, Academic Medical Center, Amsterdam </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>LifeLines Cohort Study, Groningen </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Genetics, University of Groningen, University Medical Center Groningen, Groningen </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Durrer Center for Cardiogenetic Research, ICIN-Netherlands Heart Institute, Utrecht, The Netherlands </div><div id="CR1"><strong>Correspondence:</strong> Pim van der Harst, MD, PhD, Department of Cardiology, University Medical Center Groningen, Hanzeplein 1, 9700RB Groningen, The Netherlands, Tel: +31 50 36112234, Fax: +31 50 3615525, <a href="mailto:dev@null" data-email="ln.gcmu@tsrah.red.nav.p" class="oemail">ln.gcmu@tsrah.red.nav.p</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139808672055376_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=24850809" target="pmc_ext" ref="reftype=publisher&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Circ Cardiovasc Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4141024/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="5" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>000373 - PAP.</strong></div><div><a href="/pmc/articles/PMC4141024/bin/NIHMS598853-supplement-000373_-_PAP.pdf" data-ga-action="click_feat_suppl">NIHMS598853-supplement-000373_-_PAP.pdf</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID:&nbsp;9FF53993-55A9-4BC0-91E3-543C08FECEF8</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>000373 - Supplemental Material.</strong></div><div><a href="/pmc/articles/PMC4141024/bin/NIHMS598853-supplement-000373_-_Supplemental_Material.pdf" data-ga-action="click_feat_suppl">NIHMS598853-supplement-000373_-_Supplemental_Material.pdf</a><span style="color:gray"> (863K)</span></div><div class="small guid">GUID:&nbsp;6250425D-5C7F-4D5A-8305-31814AE46622</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>000373 - Supplemental Material Table.</strong></div><div><a href="/pmc/articles/PMC4141024/bin/NIHMS598853-supplement-000373_-_Supplemental_Material_Table.txt" data-ga-action="click_feat_suppl">NIHMS598853-supplement-000373_-_Supplemental_Material_Table.txt</a><span style="color:gray"> (119M)</span></div><div class="small guid">GUID:&nbsp;5BBF087E-042E-4A49-924A-09962A6CFB6E</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>CircGenetics_CIRCCVG-2013-000373.xml.</strong></div><div><a href="/pmc/articles/PMC4141024/bin/NIHMS598853-supplement-CircGenetics_CIRCCVG-2013-000373_xml.xml" data-ga-action="click_feat_suppl">NIHMS598853-supplement-CircGenetics_CIRCCVG-2013-000373_xml.xml</a><span style="color:gray"> (16K)</span></div><div class="small guid">GUID:&nbsp;90BBEA29-F977-40F3-9C07-9FFA83C7BB7F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Clinical Perspective.</strong></div><div><a href="/pmc/articles/PMC4141024/bin/NIHMS598853-supplement-Clinical_Perspective.docx" data-ga-action="click_feat_suppl">NIHMS598853-supplement-Clinical_Perspective.docx</a><span style="color:gray"> (14K)</span></div><div class="small guid">GUID:&nbsp;6F27E0D4-C829-4E1D-8126-645AB8BB3B1E</div></div></dd></dl></div><div id="idm139808679379584" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139808679379584title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P1" class="p p-first-last">The PR interval on the electrocardiogram reflects atrial depolarization and AV nodal delay which can be partially differentiated by P wave duration and PR segment, respectively. GWAS have identified a number of genetic loci for PR interval but it remains to be determined whether this is driven by P wave duration, PR segment or both.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods and Results</h3><p id="P2" class="p p-first-last">We replicated 7 of the 9 known PR interval loci in 16,468 individuals of European ancestry. Four loci were unambiguously associated with PR segment while the others were shared for P wave duration and PR segment. Next, we performed a genome-wide analysis on P wave duration and PR segment separately and identified five novel loci. SNPs in <em>KCND3</em> (<em>P</em>=8.3×10<sup>−11</sup>) and <em>FADS2</em> (<em>P</em>=2.7×10<sup>−8</sup>) were associated with P wave duration, whereas SNPs near <em>IL17D (P</em>=2.3×10<sup>−8</sup>), in <em>EFHA1</em> (<em>P</em>=3.3×10<sup>−10</sup>) and <em>LRCH1</em> (<em>P</em>=2.1×10<sup>−8</sup>) were associated with PR segment. Analysis on DNA elements indicated that genome-wide significant SNPs were enriched at genomic regions suggesting active gene transcription in the human right atrium. Quantitative-PCR showed that genes were significantly higher expressed in the right atrium and AV-node compared to left ventricle (<em>P</em>=5.6×10<sup>−6</sup>).</p></div><div id="S3" class="sec sec-last"><h3 id="S3title">Conclusions</h3><p id="P3" class="p p-first-last">Genetic associations of PR interval appear to be mainly driven by genetic determinants of the PR segment. Some of the PR interval associations are strengthened by a directional consistent effect of genetic determinants of P wave duration. Through genome-wide association we also identified genetic variants specifically associated with P wave duration which might be relevant for cardiac biology.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genetics, electrocardiography, epigenetics, atrium, aging, PR interval</span></div></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">Introduction</h2><p id="P4" class="p p-first-last">Recent genome-wide association (GWA) studies on PR interval, derived from the surface electrocardiogram (ECG), have been instrumental for gaining novel insight into the biology of underlying conduction traits. These analyses have led to the discovery of 9 genetic loci in Europeans<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046522">1</a>, <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046500">2</a></sup>, Asians<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046494">3</a></sup> and African Americans<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046523">4</a>, <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046512">5</a></sup>, some of which also confer an increased risk of atrial fibrillation. However, PR interval is the summation of P wave duration and PR segment, which are considered to reflect different aspects of the cardiac cycle. P wave duration reflects the electrical signals that propagate through the atria and can indicate atrial enlargement<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046503">6</a></sup>. The PR segment corresponds to the period that electrical signals are delayed at the atrioventricular (AV) node, before it travels through the ventricular branches to induce cardiac depolarization and may be prolonged during AV nodal dysfunction. It remains to be determined whether the previously identified PR-interval variants are driven by P wave duration, PR segment, or by both. Understanding the origin of these genetic associations can further refine our understanding of atrial and AV-node function. The aims of current study are to (1.) determine the association of the genetic variants identified by previous PR interval studies with P wave duration and PR segment (2.) determine whether genome-wide analyses of the sub-traits (PR segment and P wave duration) can lead to the identification of novel SNP-associations.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Methods</h2><div id="S6" class="sec sec-first"><h3 id="S6title">Study samples</h3><p id="P5" class="p p-first">Participants of the Prevention of Renal and Vascular End-Stage Disease (PREVEND) study<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046520">7</a></sup> and the Lifelines study cohort <sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046515">8</a></sup> were included. The primary objective of the PREVEND program was to prospectively investigate the natural course of increased levels of urinary albumin excretion (UAE) as an indicator of increased cardiovascular and renal risk in the general population.</p><p id="P6" class="p p-last">LifeLines is a multi-disciplinary prospective population-based cohort study examining in a unique three-generation design the health and health-related behaviors of 165,000 persons living in the North East region of The Netherlands. It employs a broad range of investigative procedures in assessing the biomedical, socio-demographic, behavioral, physical and psychological factors which contribute to the health and disease of the general population, with a special focus on multimorbidity and complex genetics. Both are community based cohort studies from the northern part of the Netherlands and have been approved by the review board of the University Medical Center Groningen. This study adheres to the principles expressed in the Declaration of Helsinki. All subjects provided written informed consent. Standard 12 lead ECGs in PREVEND and Lifelines were recorded using Cardio Perfect equipment (Welch Allyn Cardio Control, Delft, The Netherlands). Individuals were excluded from this analysis for the following reasons: atrial fibrillation, atrial flutter, electronic pacemaker rhythm, complete AV-block, or Wolff-Parkinson-White syndrome. Also participants with extreme measurements (more than ±4SD from the mean) were excluded on a per phenotype basis. Myocardial infarction and medication were not, or minimally associated to durations of the P wave or PR segment in PREVEND, for this reason we did not adjust for these variables as previous analysis did<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046492">1</a></sup>. After exclusions, there were 3,496 PREVEND and 12,972 Lifelines individuals with phenotype and genotype data contributing to the genotype-phenotype association analyses. Characteristics of participants are summarized in <a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>.</p></div><div id="S7" class="sec"><h3 id="S7title">Genotyping</h3><p id="P7" class="p p-first-last">Genotyping for PREVEND and Lifelines was performed on the Illumina CytoSNP12 v2 chip. Samples were excluded based on call rates below 0.95, gender mismatch, duplicate discordance and genetic similarity (identity by state &gt; 0.2). Population stratification was assessed by principal component analysis (PCA) over the sample correlation matrix, based on 16,842 independent (LD-pruned) SNPs. Samples were excluded when they diverged from the mean with at least 3 standard deviations (Z-score &gt; 3) for the first 5 principal components. SNPs were excluded with a minor allele frequency of &lt;0.01, call rate &lt;0.95, or deviation from Hardy Weinberg equilibrium (P &lt; 1 × 10<sup>−5</sup>). Genome-wide genotype imputation was performed using Beagle v. 3.1.0, using the NCBI build 36 of Phase II HapMap CEU data (release 22) as reference panel.</p></div><div id="S8" class="sec"><h3 id="S8title">Statistical Analyses</h3><p id="P8" class="p p-first">Residuals of P wave duration and PR segment were calculated using general linear regression models to adjust for age, gender and body mass index (BMI). GWAS analyses of imputed SNPs were performed on the standardized residuals using an additive genetic model in PLINK (v.1.07). Test statistics from each cohort were then corrected for their respective genomic control inflation factor to adjust for residual population sub-structure. Meta-analysis was performed using the inverse-variance method. SNPs with MAF &lt;1% (weighted average across cohorts) were removed.</p><p id="P9">The initial analysis considered variants to be independent if the pair-wise LD (r<sup>2</sup>) was less than 0.1 and if they were separated by at least 1 MB; this was defined a ‘locus’. To test if there were multiple SNPs in the same loci independently associated with P wave duration and/or PR segment we repeated the primary association analysis for each trait whilst adjusting on the trait-specific genome-wide significant sentinel SNPs by adding the SNP dosages as covariates in the regression analyses. We then combined the association results again for each study by inverse variance weighting.</p><p id="P10" class="p p-last">Similarly, to assess the association of selected SNPs (Genome-wide significant SNPs and literature SNPs) with PR interval, we performed association analyses in PREVEND and Lifelines and combined the results using the inverse-variance method.</p></div><div id="S9" class="sec"><h3 id="S9title">Regulatory DNA</h3><p id="P11" class="p p-first">We used data available on DNAse I hypersensitivity sites (DHSs) from 349 human tissues and cell lines (GEO accession numbers <a id="__tag_396046524" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE29692" ref="/geo/query/acc.cgi?acc=GSE29692"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE29692","term_id":"29692"}}</span>GSE29692</a> and <a id="__tag_396046527" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE18927" ref="/geo/query/acc.cgi?acc=GSE18927"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE18927","term_id":"18927"}}</span>GSE18927</a>) of the ENCODE project<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046519">9</a></sup> and Roadmap Epigenomics Program<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046504">10</a></sup>. Hotspots were identified using the hotspot algorithm and peaks were called at 5% false discovery rate in a uniform manner, as previously described<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046489">11</a></sup>.</p><p id="P12">Aligned sequence reads (bed files) from 7 distinct histone modification assays in various human tissues were obtained from the Roadmap Epigenomics Project release 8. Only samples with matching input DNA samples were included. If replicate experiments were available we aggregated the sequence reads. MACS (v1.4) software was used to identify significant peaks (1 × 10<sup>−3</sup>) using a fixed DNA fragment size of 146 <sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046511">12</a></sup>. Three samples could not be called with MACS due to inconsistencies in the original data. As a result, this data set included aligned sequence reads of 323 samples (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S2</a>).</p><p id="P13" class="p p-last">For annotation of functional elements we included cardiac transcription factor data on Tbx3, Gata4 and Nkx2-5 from mouse heart<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046514">13</a></sup>; p300 marks in human adult and fetal heart and RNAP2 from human fetal heart <sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046518">14</a></sup>; and Gata4, Mef2, Nkx2-5, Srf and Tbx5 from the atrial HL-1 cell line <sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046506">15</a></sup>. We used the called peak data from GEO (<a id="__tag_396046526" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE35151" ref="/geo/query/acc.cgi?acc=GSE35151"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE35151","term_id":"35151"}}</span>GSE35151</a>, <a id="__tag_396046525" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE32587" ref="/geo/query/acc.cgi?acc=GSE32587"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE32587","term_id":"32587"}}</span>GSE32587</a> and <a id="__tag_396046528" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE21529" ref="/geo/query/acc.cgi?acc=GSE21529"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE21529","term_id":"21529"}}</span>GSE21529</a>). Peaks from mice were lift-over to human using the UCSC Genome Browser liftOver tool with the options “-minMatch=0.1 -multiple” after extending the regions by 1kb <sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046513">16</a></sup>. Conversion of hg18 to hg19 was also performed using the UCSC LiftOver tool when appropriate.</p></div><div id="S10" class="sec sec-last"><h3 id="S10title">Quantitative expression analysis of genes associated with P wave duration and/or PR segment</h3><p id="P14" class="p p-first-last">Wild-type FVB animals were sacrificed at the age of 6-8 weeks. Left ventricles, right atria and atrioventricular node / junction tissue were dissected in sterile PBS and pooled per tissue type in 3 groups of 7 samples. Total RNA isolation was performed using the MagNA Lyser instrument (product no. 03358976001; Roche Applied Science) with MagNA Lyser Green beads (product no. 03358941001, Roche Applied Science) and TRIzol Reagent (catalog no. 15596-026; Invitrogen) according to manufacturer’s protocol. After quantification, 400ng of total RNA was treated with DNase I (18068-015, Invitrogen) to remove genomic DNA from the RNA sample. cDNA synthesis was performed using the SuperScript® II Reverse Transcriptase kit (catalog no. 18064-014, Invitrogen). Expression of different genes was assayed with quantitative real-time PCR using the Roche LightCycler 480 system. Primer sequences are listed in <a href="#SD2" rid="SD2" class=" supplementary-material">Table S3</a>. Relative start concentration (N(0)) was calculated as previously described <sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046517">17</a></sup>. Values were normalized to Gapdh expression levels, differences in expression values were tested by 2-tailed t-tests. All animal work was approved by the Animal Experimental Committee of the Academic Medical Center, University of Amsterdam, and was carried out in compliance with Dutch government guidelines.</p></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Results</h2><p id="P15" class="p p-first">We performed GWA studies with 2.3 Million imputed autosomal SNPs (HapMap 2 build 36 CEU panel) in the same individuals on P wave duration and PR segment. There was no evidence for inflation of test statistics at the final meta-analysis results, λ<sub>P-duration</sub> = 1.002, λ<sub>PR-segment</sub> = 1.019 (<a href="#SD2" rid="SD2" class=" supplementary-material">Figure S1</a>). Although PR segment is a smaller interval (~44 ms) than P duration (~113 ms) on the ECG, PR segment explains more of the PR interval (r2=0.66, P &lt; 10<sup>−99</sup>) than P wave duration (r2=0.30, P &lt; 10<sup>−99</sup>), due to greater variation in PR segment (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>).</p><p id="P16">First, we looked up the association of the known PR interval SNPs in the P wave duration and PR segment association results (<a href="/pmc/articles/PMC4141024/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Table S4</a>). All previously identified PR interval loci that were significant in this study also showed an association signal with PR segment (<em>P</em>&lt;0.004). Loci containing genes <em>SCN5A/SCN10A, CAV1</em> and <em>SOX5</em> were also found to be associated with P wave duration; however none of the known PR interval loci were specific for P wave duration. The association of the <em>ARHGAP24</em> and <em>TBX3</em> SNPs were higher for PR segment (<em>P</em>=1.1×10<sup>−14</sup>, <em>P</em>=1.2×10<sup>−9</sup>, respectively) compared to the association with PR interval (<em>P</em>=9.9×10<sup>−12</sup> and <em>P</em>=5.0×10<sup>−6</sup>, respectively). All other PR interval associations decreased by separating PR interval into PR segment and P wave duration.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p5">Seven of the nine known genome-wide PR interval loci <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046493">1</a>-<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046496">5</a></sup> reached the threshold of significance for replication (P &lt; 0.05/9) and were directionally concordant with previous reports, as indicated by bold P values and effect estimates. All loci that were significant for PR interval in this analysis were significantly associated with PR segment. However, SCN5A/SCN10A, CAV1 and SOX5 also showed an association with P wave duration. PR interval statistics were obtained in PREVEND and Lifelines and combined using the inverse-variance method. Frequencies were calculated from the individuals of Lifelines and PREVEND that were included in the meta-analyses. SNPs were pruned on LD (r<sup>2</sup>&gt;0.6, HapMap). Beta values estimate the difference in duration in millisecond per copy of the coding allele, adjusted for the covariates in the model.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139808676434480" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="2" rowspan="1" valign="middle" align="center">P wave duration GWAS</th><th colspan="2" rowspan="1" valign="middle" align="center">PR segment GWAS</th><th colspan="4" rowspan="1" valign="middle" align="left">PR interval</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Region</th><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="left">Coding(FRQ)<br>/ non-coding<br>allele</th><th rowspan="1" colspan="1" valign="middle" align="left">P-value</th><th rowspan="1" colspan="1" valign="middle" align="left">Beta(s.e.)</th><th rowspan="1" colspan="1" valign="middle" align="left">P-value</th><th rowspan="1" colspan="1" valign="middle" align="left">Beta(s.e.)</th><th rowspan="1" colspan="1" valign="middle" align="left">P-value</th><th rowspan="1" colspan="1" valign="middle" align="left">Beta(s.e.)</th><th rowspan="1" colspan="1" valign="middle" align="left">Ref</th><th rowspan="1" colspan="1" valign="middle" align="left">Gene</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">2p14</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3891585</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.41)/G</td><td rowspan="1" colspan="1" valign="middle" align="left">0.024</td><td rowspan="1" colspan="1" valign="middle" align="left">0.29(0.13)</td><td rowspan="1" colspan="1" valign="middle" align="left">2.30E-08</td><td rowspan="1" colspan="1" valign="middle" align="left">1.16(0.21)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.75E-09</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.44(0.24)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left"><a href="#R1" rid="R1" class=" bibr popnode">1</a>, <a href="#R4" rid="R4" class=" bibr popnode">4</a>, <a href="#R5" rid="R5" class=" bibr popnode">5</a></td><td rowspan="1" colspan="1" valign="middle" align="left">MEIS1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">3p22.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs267567</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.47)/G</td><td rowspan="1" colspan="1" valign="middle" align="left">0.781</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.04(0.14)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.320</td><td rowspan="1" colspan="1" valign="middle" align="left">0.22(0.23)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.836</td><td rowspan="1" colspan="1" valign="middle" align="left">0.05(0.26)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<a href="#R5" rid="R5" class=" bibr popnode">5</a>
</td><td rowspan="1" colspan="1" valign="middle" align="left">ITGA9</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">3p22.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3922844</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.33)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">1.78E-08</td><td rowspan="1" colspan="1" valign="middle" align="left">−1.41(0.25)</td><td rowspan="1" colspan="1" valign="middle" align="left">3.79E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">−1.46(0.41)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>4.12E-10</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>−2.96(0.47)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left"><a href="#R4" rid="R4" class=" bibr popnode">4</a>, <a href="#R5" rid="R5" class=" bibr popnode">5</a></td><td rowspan="1" colspan="1" valign="middle" align="left">SCN5A</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">3p22.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs6599222</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.84)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">6.80E-08</td><td rowspan="1" colspan="1" valign="middle" align="left">−1.98(0.37)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.008</td><td rowspan="1" colspan="1" valign="middle" align="left">−1.61(0.60)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>6.28E-08</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>−3.74(0.69)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<a href="#R4" rid="R4" class=" bibr popnode">4</a>
</td><td rowspan="1" colspan="1" valign="middle" align="left">SCN5A</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">3p22.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs6763048</td><td rowspan="1" colspan="1" valign="middle" align="left">NA</td><td rowspan="1" colspan="1" valign="middle" align="left">NA</td><td rowspan="1" colspan="1" valign="middle" align="left">NA</td><td rowspan="1" colspan="1" valign="middle" align="left">NA</td><td rowspan="1" colspan="1" valign="middle" align="left">NA</td><td rowspan="1" colspan="1" valign="middle" align="left">NA</td><td rowspan="1" colspan="1" valign="middle" align="left">NA</td><td rowspan="1" colspan="1" valign="middle" align="left">
<a href="#R5" rid="R5" class=" bibr popnode">5</a>
</td><td rowspan="1" colspan="1" valign="middle" align="left">SCN5A</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">3p22.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs6801957</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.43)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">8.44E-27</td><td rowspan="1" colspan="1" valign="middle" align="left">1.44(0.13)</td><td rowspan="1" colspan="1" valign="middle" align="left">6.94E-41</td><td rowspan="1" colspan="1" valign="middle" align="left">2.93(0.22)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>9.99E-69</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>4.41(0.25)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left"><a href="#R1" rid="R1" class=" bibr popnode">1</a>-<sup><a href="#R5" rid="R5" class=" bibr popnode">5</a></sup></td><td rowspan="1" colspan="1" valign="middle" align="left">SCN10A</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">4q21.23</td><td rowspan="1" colspan="1" valign="middle" align="left">rs7660702</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.70)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">0.761</td><td rowspan="1" colspan="1" valign="middle" align="left">0.04(0.14)</td><td rowspan="1" colspan="1" valign="middle" align="left">1.14E-14</td><td rowspan="1" colspan="1" valign="middle" align="left">1.79(0.23)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>9.87E-12</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.82(0.27)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left"><a href="#R1" rid="R1" class=" bibr popnode">1</a>, <a href="#R2" rid="R2" class=" bibr popnode">2</a>, <a href="#R4" rid="R4" class=" bibr popnode">4</a>, <a href="#R5" rid="R5" class=" bibr popnode">5</a></td><td rowspan="1" colspan="1" valign="middle" align="left">ARHGAP2<br>4</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">5q35.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs251253</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.59)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">0.078</td><td rowspan="1" colspan="1" valign="middle" align="left">0.22(0.12)</td><td rowspan="1" colspan="1" valign="middle" align="left">3.56E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.84(0.20)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.06E-05</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.04(0.24)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<a href="#R1" rid="R1" class=" bibr popnode">1</a>
</td><td rowspan="1" colspan="1" valign="middle" align="left">ATP6V0E1,<br>C5orf41</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">7q31.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3807989</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.43)/G</td><td rowspan="1" colspan="1" valign="middle" align="left">3.50E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.51(0.12)</td><td rowspan="1" colspan="1" valign="middle" align="left">4.99E-12</td><td rowspan="1" colspan="1" valign="middle" align="left">1.40(0.20)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>5.78E-16</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.89(0.23)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left"><a href="#R2" rid="R2" class=" bibr popnode">2</a>, <a href="#R5" rid="R5" class=" bibr popnode">5</a></td><td rowspan="1" colspan="1" valign="middle" align="left">CAV1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11q13.5</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4944092</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.76)/G</td><td rowspan="1" colspan="1" valign="middle" align="left">0.927</td><td rowspan="1" colspan="1" valign="middle" align="left">0.03(0.33)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.956</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.03(0.53)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.796</td><td rowspan="1" colspan="1" valign="middle" align="left">0.16(0.61)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<a href="#R1" rid="R1" class=" bibr popnode">1</a>
</td><td rowspan="1" colspan="1" valign="middle" align="left">WNT11</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">12p12.1</td><td rowspan="1" colspan="1" valign="middle" align="left">rs11047543</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.14)/G</td><td rowspan="1" colspan="1" valign="middle" align="left">4.93E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.65(0.19)</td><td rowspan="1" colspan="1" valign="middle" align="left">1.57E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">−1.32(0.31)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>3.44E-08</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>−1.94(0.35)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<a href="#R1" rid="R1" class=" bibr popnode">1</a>
</td><td rowspan="1" colspan="1" valign="middle" align="left">SOX5</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">12q24.21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3825214</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.78)/G</td><td rowspan="1" colspan="1" valign="middle" align="left">0.031</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.32(0.15)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.287</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.26(0.25)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.029</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.62(0.28)</td><td rowspan="1" colspan="1" valign="middle" align="left"><a href="#R2" rid="R2" class=" bibr popnode">2</a>, <a href="#R4" rid="R4" class=" bibr popnode">4</a>, <a href="#R5" rid="R5" class=" bibr popnode">5</a></td><td rowspan="1" colspan="1" valign="middle" align="left">TBX5</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">12q24.21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1896312</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.65)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">0.231</td><td rowspan="1" colspan="1" valign="middle" align="left">0.19(0.16)</td><td rowspan="1" colspan="1" valign="middle" align="left">1.17E-09</td><td rowspan="1" colspan="1" valign="middle" align="left">−1.60(0.26)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>5.01E-06</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>−1.39(0.30)</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<a href="#R1" rid="R1" class=" bibr popnode">1</a>
</td><td rowspan="1" colspan="1" valign="middle" align="left">TBX3</td></tr></tbody></table></div><div id="largeobj_idm139808676434480" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4141024/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P17">Secondly, the genome-wide meta-analysis identified a total of 93 SNPs in 3 loci to be significantly genome-wide associated (<em>P</em>&lt;5.0×10<sup>−8</sup>) with P wave duration and 184 SNPs in 7 loci to be genome-wide associated with PR segment (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S5</a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Table S6</a>). Summary statistics of all 2.3M SNPs are provided as supplemental material. We identified a secondary signal (rs2253017) on chromosome 13 to remain genome-wide associated with PR segment while adjusting for rs2798269 (<a href="/pmc/articles/PMC4141024/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="/pmc/articles/PMC4141024/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Figure S2</a>). One locus was shared among both phenotypes (the <em>SCN10A</em> locus). In total we identified 10 independent genetic variants to be genome-wide associated with P wave duration and/or PR segment. The variants explained on average 1.2% (P wave duration) and 3.1% (PR segment) of the phenotypic variation in Lifelines and PREVEND. All genome-wide top SNPs (sentinel SNPs) per independent locus were well imputed as shown in <a href="#SD2" rid="SD2" class=" supplementary-material">Table S4</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4141024/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139808679383520" class="figure"><a href="/pmc/articles/PMC4141024/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4141024_nihms-598853-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-598853-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4141024/bin/nihms-598853-f0001.jpg"></a></div><div id="largeobj_idm139808679383520" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4141024/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4141024/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Manhattan plots of the association of SNPs with a) PR segment and b) P wave duration. The <em>x</em>-axis represents the chromosomal position for each SNP; the y-axis represents the −log10 of the P-value for association. Three loci were significant for P wave duration and 7 loci were significant for PR segment.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p6">Summary statistics for the sentinel SNPs that were identified in the genome-wide association study on P wave duration and/or PR segment. P-values in bold indicate genome-wide associations. Beta values estimate the difference in duration in millisecond per copy of the coding allele, adjusted for the covariates in the model.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139808675073264" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="2" rowspan="1" valign="middle" align="left">P wave duration</th><th colspan="2" rowspan="1" valign="middle" align="left">PR segment</th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Region</th><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="left">Coding(FRQ)/<br>non-coding</th><th rowspan="1" colspan="1" valign="middle" align="left">P-value</th><th rowspan="1" colspan="1" valign="middle" align="left">Beta(s.e.)</th><th rowspan="1" colspan="1" valign="middle" align="left">P value</th><th rowspan="1" colspan="1" valign="middle" align="left">Beta(s.e.)</th><th rowspan="1" colspan="1" valign="middle" align="left">Ref</th><th rowspan="1" colspan="1" valign="middle" align="left">Gene</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">1p13.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2798334</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.29)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>7.92E-11</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.92(0.14)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.002</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.72(0.23)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left">KCND3</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2p14</td><td rowspan="1" colspan="1" valign="middle" align="left">rs11678354</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.36)/T</td><td rowspan="1" colspan="1" valign="middle" align="left">0.010</td><td rowspan="1" colspan="1" valign="middle" align="left">0.34(0.13)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>4.71E-09</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.26(0.22)</td><td rowspan="1" colspan="1" valign="middle" align="left"><a href="#R1" rid="R1" class=" bibr popnode">1</a>, <a href="#R4" rid="R4" class=" bibr popnode">4</a>, <a href="#R5" rid="R5" class=" bibr popnode">5</a></td><td rowspan="1" colspan="1" valign="middle" align="left">MEIS1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">3p22.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs6801957</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.43)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>8.44E-27</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.44(0.13)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>6.94E-41</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">2.93(0.22)</td><td rowspan="1" colspan="1" valign="middle" align="left"><a href="#R1" rid="R1" class=" bibr popnode">1</a>-<sup><a href="#R5" rid="R5" class=" bibr popnode">5</a></sup></td><td rowspan="1" colspan="1" valign="middle" align="left">SCN10A</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">4q21.23</td><td rowspan="1" colspan="1" valign="middle" align="left">rs13137008</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.70)/G</td><td rowspan="1" colspan="1" valign="middle" align="left">0.638</td><td rowspan="1" colspan="1" valign="middle" align="left">0.07(0.14)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.12E-14</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.76(0.23)</td><td rowspan="1" colspan="1" valign="middle" align="left"><a href="#R1" rid="R1" class=" bibr popnode">1</a>, <a href="#R2" rid="R2" class=" bibr popnode">2</a>, <a href="#R4" rid="R4" class=" bibr popnode">4</a>, <a href="#R5" rid="R5" class=" bibr popnode">5</a></td><td rowspan="1" colspan="1" valign="middle" align="left">ARHGAP24</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">7q31.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3807989</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.43)/G</td><td rowspan="1" colspan="1" valign="middle" align="left">3.50E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.51(0.12)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>4.99E-12</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.40(0.20)</td><td rowspan="1" colspan="1" valign="middle" align="left"><a href="#R2" rid="R2" class=" bibr popnode">2</a>, <a href="#R5" rid="R5" class=" bibr popnode">5</a></td><td rowspan="1" colspan="1" valign="middle" align="left">CAV1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11q12.2</td><td rowspan="1" colspan="1" valign="middle" align="left">rs174577</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.33)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>2.59E-08</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.73(0.13)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.203</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.28(0.22)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left">FADS2</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">12q24.21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10850409</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.27)/G</td><td rowspan="1" colspan="1" valign="middle" align="left">0.097</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.23(0.14)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>7.62E-13</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.65(0.23)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<a href="#R1" rid="R1" class=" bibr popnode">1</a>
</td><td rowspan="1" colspan="1" valign="middle" align="left">TBX3</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">13q12.11</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2253017</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.15)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">0.963</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.01(0.17)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>2.20E-08</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.58(0.28)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left">IL17D, N6AMT2</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<em>rs2253017 adjusted for rs2798269</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">-</td><td rowspan="1" colspan="1" valign="middle" align="left">-</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.27E-08</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.59(0.28)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">13q12.11</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2798269</td><td rowspan="1" colspan="1" valign="middle" align="left">T(0.40)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">0.923</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.01(0.13)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>3.22E-10</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">−1.38(0.22)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left">EFHA1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">13q14.13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs9590974</td><td rowspan="1" colspan="1" valign="middle" align="left">A(0.65)/C</td><td rowspan="1" colspan="1" valign="middle" align="left">0.243</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.15(0.13)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>2.00E-08</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">−1.19(0.21)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left">LRCH1</td></tr></tbody></table></div><div id="largeobj_idm139808675073264" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4141024/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P18" class="p">The Phenotype-Genotype Integrator at the National Center for Biotechnology Information (<a href="http://www.ncbi.nlm.nih.gov/gap/phegeni" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gap/phegeni</a>) was queried for literature on the significant SNPs and SNPs in LD (r<sup>2</sup>&gt;0.6, Hapmap). The <em>FADS2</em> locus stands out, rs174577 (11q12), a SNP that has previously been associated with metabolic traits (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S7</a>).</p><div id="S12" class="sec"><h3 id="S12title">SNPs in regulatory DNA</h3><p id="P19" class="p p-first">To provide insight into tissue specific regulatory DNA mechanisms influencing P wave duration and PR segment, we explored DNAse I hypersensitivity sites and histone marks <sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046495">11</a></sup>. We assigned the lowest <em>P</em> value of P wave duration and PR duration to each of the 2.3 M SNPs. We then compared the ratio of all SNPs with a P value below 5 × 10<sup>−8</sup> in peaks of DNA elements to the ratio of all 2.3 M SNPs in DNA elements. We considered all available tissues/experiments. Enrichment was defined if tissues passed the threshold of Q3+1.5*IQR. (<strong><a href="/pmc/articles/PMC4141024/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a></strong>). SNPs in all available human heart tissues of interest (right atrium left ventricle, fetal heart), atria, and a few other non-heart tissues were over-enriched for various DNA elements.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4141024/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139808673458080" class="figure"><a href="/pmc/articles/PMC4141024/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4141024_nihms-598853-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-598853-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4141024/bin/nihms-598853-f0002.jpg"></a></div><div id="largeobj_idm139808673458080" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4141024/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4141024/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2 A</a></div><!--caption a7--><div class="caption"><p id="__p3">Enrichment of SNPs in human tissues was tested in marks of H3K27ac (n=34), H3K27me3 (n=55), H3K36me3 (n=55), H3K4me1 (n=50), H3K4me3 (n=50, H3K9ac (n=26), H3K9me3(n=53) and DHS (n=349). Genome-wide significant SNPs were over-enriched in the active histone marks H3K27ac and H3K4me1 of the right atrium. In contrast, genome-wide significant loci were under-enriched in H3k9me3 of the right atrium, a mark that is associated with low levels of gene transcription. These results indicate that genome-wide significant loci are more active in the right atrium compared to other tissues, as expected. Furthermore, all available heart tissues were at least ≥1 time considered to be over-enriched, as well as a few other tissues such as aorta, skeletal muscle and fibroblasts. <strong>B)</strong> To test the hypothesis that we identified regions actively transcribed in the right atrium and AV-node, we performed qPCR’s of the nearest genes identified in the genome-wide analysis of this study in the right atrium, AV-node and left ventricle. Three groups of 7 pooled tissue samples were measured for the expression of each gene and were normalized to the expression of Gapdh. Error bars represent the standard deviation. Significance of difference (* and #) between the left ventricle and the right atria or AV-node was tested using a t-test. Six of the 11 tested candidate genes were significantly higher expressed in both the node and right atrium when compared to the left ventricle. This is a significantly higher number than would be expected by chance (P<sub>binomial</sub> =5.56*10<sup>−6</sup> at the 0.05 level of significance).</p><p id="__p4">LV=Left Ventricle, RA=Right atrium, FH=Fetal Heart, AVN=atrio-ventricular-node 1) Penis foreskin fibroblast primary cells, 2) Stomach, smooth muscle, 3) Aorta, 4) Brain anterior caudate, 5) Human skeletal muscle myoblasts, 6) Human mammary fibroblasts</p></div></div></div><p id="P20" class="p p-last">The genome-wide significant SNPs were annotated with various DNA elements that had been measured in the human or mouse heart. We found that all sentinel SNPs or SNPs in LD (r<sup>2</sup>&gt;0.8, 1000genomes) overlap with at least one DNase 1 hypersensitivity site of human fetal heart. In <a href="#SD2" rid="SD2" class=" supplementary-material">Figure S3</a> we provide an overview of the sentinel SNPs in DNA elements to prioritize loci for experimental follow-up.</p></div><div id="S13" class="sec sec-last"><h3 id="S13title">Candidate genes and gene expression</h3><p id="P21" class="p p-first">We prioritized candidate genes by searching for (1) protein coding gene nearest to the sentinel SNP, and any other protein coding gene within 10kb (11 genes); (2) we also considered genes containing a coding SNP in high LD (1000G EUR, r<sup>2</sup>&gt;0.8) with the sentinel SNP (1 gene, SCN10A). This analysis identified 11 candidate genes (<a href="/pmc/articles/PMC4141024/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) for the 10 independent, genome-wide significant, SNPs.</p><p id="P22" class="p p-last">To test the hypothesis that we identified regions actively transcribed in the right atrium and AV-node, we performed qPCR’s of the nearest gene to the sentinel SNP or any gene within 10kb. Using carefully dissected tissue samples from adult mouse hearts, we analyzed the expression of the candidate genes in atrial, ventricular and AV-nodal components. From this analysis it is first noteworthy that <em>Tbx3</em> expression was most highly expressed in the AV-node tissue samples, thus acting as a validation for the dissection procedure. Using qPCR we observed that 10 of the 11 candidate genes were expressed in left ventricle, right atrium and AV-node in mice. Notably, <em>SCN10A</em> transcripts were not reproducibly detectable in mice, also in line with a recent study of atrial gene expression using RNAseq <sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046510">18</a></sup>. All genes except <em>N6AMT2</em> (lower expression) and <em>ARHGAP24</em> showed significantly higher expression in atria or AV-node compared to the left ventricle (P&lt;0.05, <a href="/pmc/articles/PMC4141024/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). This number of genes (6) was also a significantly higher number than would be expected by chance (P<sub>binomial</sub>=5.56*10<sup>−6</sup> at the 0.05 level of significance). <em>N6AMT2</em> is annotated to rs2253017 (P=2.3×10<sup>−8</sup>, PR segment), which is located between <em>IL17D</em> (−3.7kb) and <em>N6AMT2</em> (+2.2kb). In contrast to <em>N6AMT2, IL17D</em> is expressed higher in the AV-node and right atrium compared to the left ventricle, making it a more likely candidate gene.</p></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Discussion</h2><p id="P23" class="p p-first">The P wave and PR interval on the ECG are important traits that have proven relationship with and predictive value for normal and abnormal heart rhythm, supraventricular arrhythmias, and conduction disturbances. We replicated 7 out of 9 loci that were previously associated with PR interval, and report that these are mainly driven by genetic variants affecting the PR segment, and to a lesser extent P wave duration. We also observed that PR segment explains more of PR interval than P wave duration, in support of a genetic investigation of P wave duration. By studying the specific sub-traits, we identified 5 novel independent genome-wide significant associations with PR segment or P wave duration. Variant near <em>IL17D/N6AMT2, EFHA1</em> and <em>LRCH1</em> were specifically associated with PR segment, whereas variants near <em>KCND3</em> and <em>FADS2</em> were specifically associated with PR duration.</p><p id="P24">We observed that genome-wide significant SNPs were specifically localized in chromatin marks and DNase I hypersensitive sites of the human fetal heart, compared to many other human tissues. This indicates that functionality of regulatory DNA elements may underlie some of the associations. The SNP in <em>SCN10A</em> (rs6801957) has already been studied and shown to affect a functional enhancer in the heart <sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046507">14</a></sup>. Genome-wide SNPs were also over-enriched in cell type specific histone marks associated with active enhancers and promoters, H3K27ac and H3K4me1, of the human right atrium<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046499">19</a></sup>. The repressive-state associated mark H3K9ac in the right atrium contained no genome-wide SNPs, and showed depletion compared to the other tissues. This finding suggests the regulatory mechanism to be mediated by gene activation and not repression.</p><p id="P25">Our expression analyses showed specific AV-nodal expression of Tbx3, which is in line with the association of rs10850409 near <em>TBX3</em> with PR-segment, reflecting the AV-node-dependent duration on the ECG. Tbx3 is a well-characterized transcription factor that is required for the development of many tissues. In the heart it allows the cells to acquire pacemaker properties and is required for the specification of the AV conduction system<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046490">20</a></sup>.</p><p id="P26" class="p">We identified novel associations with P wave duration and PR segment in the genomic regions 1p13.2, 11q12.2, 13q12.11 (containing 2 independent signals) and 13q14.13. In locus 13q12.11, <em>IL17D</em> was a strong candidate based on the expression analysis. To date, there has only been one study on <em>IL17D</em> which described the protein to be preferentially expressed in skeletal muscle, brain, adipose tissue, heart, lung, and pancreas, and to stimulate IL-6 and NF-κB dependent IL-8 production<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046502">21</a></sup> relevant for heart function<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046497">22</a>, <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046501">23</a></sup>. Future studies should identify its potential role in the atria. In the same locus we identified another independent signal containing <em>EFHA1</em>, the exact function of the protein remains to be determined but a recent publication suggests a role in mitochondrial calcium handling <sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046508">24</a></sup>. In locus 11q12.2, <em>FADS2,</em> a member of the fatty acid desaturases has been previously associated with a multitude of lipids<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046516">25</a>, <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046498">26</a></sup>, and a prominent role of lipid metabolism in the heart has been extensively studied <sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046505">27</a></sup>. In locus 1p13.2, <em>KCND3</em> encodes the Kv4.3 α-subunit that conducts the cardiac fast transient outward K+ current (Ito). This current is prominent in the repolarization phase of cardiac action potential. Several gain-of-function mutations have been recently been associated with Brugada syndrome and early-onset lone atrial fibrillation <sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046521">28</a></sup>. Finally, 13q14.13 contains <em>LRCH1;</em> no function has been assigned to the protein product of this gene.</p><div id="S15" class="sec sec-last"><h3 id="S15title">Strengths and Limitations</h3><p id="P27" class="p p-first-last">This study is the first to investigate the differences in SNP associations between P wave duration and PR segment. The participants in this study are from the northern parts of the Netherlands and measurements have been performed in the same center, which ensures a homogenous population and reduces variation in assays. The novel genotype-phenotype associations were highly statistically significant but future efforts are required to understand the origin of these associations. Additionally, we note that the identified variants explained little of the phenotypic variation in the population, but anticipate that the distinct genotype-phenotype associations with PR-interval’s sub-traits are more relevant to provide new biological insight of the heart’s function <sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046509">29</a></sup>. In this study we were the first to combine as many as 672 experiments on histone modifications and DNase I hypersensitivity sites in human tissues for insight into the results of genome-wide analyses and thereby validating them. However, the analyses on DNA elements and gene expression in the right atrium might differ from the left atrial tissue which is thought to be the responsible tissue for AF vulnerability<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396046491">18</a></sup>. It is unknown how the identified polymorphisms or which polymorphisms in LD have an effect on genes function causing the association. In-depth experimental studies are necessary to address the loci for their causal genes and the mechanisms on how genomic regions can affect the different aspects of P wave durations.</p></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Conclusion</h2><p id="P28" class="p p-first-last">PR interval appears to be mainly genetically driven by PR segment but some of the associations are further strengthened by a directional consistent effect of genetic determinants on P wave duration. By GWA of the sub-phenotypes, we also identified two novel SNPs specifically associated with P wave duration which may be relevant for atrial biology and three novel SNP associations for PR segment. Analysis on DNA elements indicated that we identified regions actively transcribed in the right atrium, which was further validated by expression profiling of the candidate genes. In conclusion, analyzing sub-phenotypes of the electrocardiogram can lead to the identification of novel loci and genes and therefore might provide new insights into cardiac biology.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>000373 - PAP</h4><div class="sup-box half_rhythm" id="d36e723"><a href="/pmc/articles/PMC4141024/bin/NIHMS598853-supplement-000373_-_PAP.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.3M, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>000373 - Supplemental Material</h4><div class="sup-box half_rhythm" id="d36e727"><a href="/pmc/articles/PMC4141024/bin/NIHMS598853-supplement-000373_-_Supplemental_Material.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(863K, pdf)</sup></div></div><div class="sec suppmat" id="SD3"><h4>000373 - Supplemental Material Table</h4><div class="sup-box half_rhythm" id="d36e731"><a href="/pmc/articles/PMC4141024/bin/NIHMS598853-supplement-000373_-_Supplemental_Material_Table.txt" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(119M, txt)</sup></div></div><div class="sec suppmat" id="SD4"><h4>CircGenetics_CIRCCVG-2013-000373.xml</h4><div class="sup-box half_rhythm" id="d36e735"><a href="/pmc/articles/PMC4141024/bin/NIHMS598853-supplement-CircGenetics_CIRCCVG-2013-000373_xml.xml" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(16K, xml)</sup></div></div><div class="sec suppmat" id="SD5"><h4>Clinical Perspective</h4><div class="sup-box half_rhythm" id="d36e739"><a href="/pmc/articles/PMC4141024/bin/NIHMS598853-supplement-Clinical_Perspective.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(14K, docx)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="__p1"><strong>Funding Sources:</strong> This work was supported by the following grants: PREVEND genetics is supported by the Dutch Kidney Foundation (Grant E033), the National Institutes of Health (grant LM010098), The Netherlands Organization for Scientific Research (NWO-Groot 175.010.2007.006, NWO VENI grant 916.761.70 and NWO VIDI grant 917.13.350, ZonMW 90.700.441), and the Dutch Inter University Cardiology Institute Netherlands. The LifeLines Cohort Study, and generation and management of GWAS genotype data for the LifeLines Cohort Study is supported by the Netherlands Organization of Scientific Research NWO (grant 175.010.2007.006), the Economic Structure Enhancing Fund (FES) of the Dutch government, the Ministry of Economic Affairs, the Ministry of Education, Culture and Science, the Ministry for Health, Welfare and Sports, the Northern Netherlands Collaboration of Provinces (SNN), the Province of Groningen, University Medical Center Groningen, the University of Groningen, Dutch Kidney Foundation and Dutch Diabetes Research Foundation. N. Verweij is supported by the Netherlands Heart Foundation (grant NHS2010B280).</p></div></div><div id="idm139808680369840" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139808680369840title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P29" class="p p-first-last"><strong>Journal Subject Codes:</strong> [8] Epidemiology, [171] Electrocardiology, [142] Gene expression</p><p></p><p class="fn sec" id="FN2"></p><p id="P30" class="p p-first-last"><strong>Conflict of Interest Disclosures:</strong> None.</p><p></p></div></div><div id="idm139808672535840" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139808672535840title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Pfeufer A, van Noord C, Marciante KD, Arking DE, Larson MG, Smith AV, et al.  Genome-wide association study of pr interval. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:153–159.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850197/">PMC free article</a>]</span> [<a href="/pubmed/20062060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+of+pr+interval&amp;author=A+Pfeufer&amp;author=C+van+Noord&amp;author=KD+Marciante&amp;author=DE+Arking&amp;author=MG+Larson&amp;volume=42&amp;publication_year=2010&amp;pages=153-159&amp;pmid=20062060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, et al.  Several common variants modulate heart rate, pr interval and qrs duration. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:117–122.</span> [<a href="/pubmed/20062063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Several+common+variants+modulate+heart+rate,+pr+interval+and+qrs+duration&amp;author=H+Holm&amp;author=DF+Gudbjartsson&amp;author=DO+Arnar&amp;author=G+Thorleifsson&amp;author=G+Thorgeirsson&amp;volume=42&amp;publication_year=2010&amp;pages=117-122&amp;pmid=20062063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Chambers JC, Zhao J, Terracciano CM, Bezzina CR, Zhang W, Kaba R, et al.  Genetic variation in scn10a influences cardiac conduction. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:149–152.</span> [<a href="/pubmed/20062061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genetic+variation+in+scn10a+influences+cardiac+conduction&amp;author=JC+Chambers&amp;author=J+Zhao&amp;author=CM+Terracciano&amp;author=CR+Bezzina&amp;author=W+Zhang&amp;volume=42&amp;publication_year=2010&amp;pages=149-152&amp;pmid=20062061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Smith JG, Magnani JW, Palmer C, Meng YA, Soliman EZ, Musani SK, et al.  Genome-wide association studies of the pr interval in african americans. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1001304.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037415/">PMC free article</a>]</span> [<a href="/pubmed/21347284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+studies+of+the+pr+interval+in+african+americans&amp;author=JG+Smith&amp;author=JW+Magnani&amp;author=C+Palmer&amp;author=YA+Meng&amp;author=EZ+Soliman&amp;volume=7&amp;publication_year=2011&amp;pages=e1001304&amp;pmid=21347284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Butler AM, Yin X, Evans DS, Nalls MA, Smith EN, Tanaka T, et al.  Novel loci associated with pr interval in a genome-wide association study of 10 african american cohorts. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2012;<span class="ref-vol">5</span>:639–646.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3560365/">PMC free article</a>]</span> [<a href="/pubmed/23139255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Novel+loci+associated+with+pr+interval+in+a+genome-wide+association+study+of+10+african+american+cohorts&amp;author=AM+Butler&amp;author=X+Yin&amp;author=DS+Evans&amp;author=MA+Nalls&amp;author=EN+Smith&amp;volume=5&amp;publication_year=2012&amp;pages=639-646&amp;pmid=23139255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Ariyarajah V, Prajapat L, Kumar KK, Barac I, Apiyasawat S, Spodick DH. Quantitative estimation of left atrial linear dimension on a transthoracic echocardiogram using an electrocardiographic formulaic assessment. <span><span class="ref-journal">Am. J. Cardiol. </span>2007;<span class="ref-vol">100</span>:894–898.</span> [<a href="/pubmed/17719341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol&amp;title=Quantitative+estimation+of+left+atrial+linear+dimension+on+a+transthoracic+echocardiogram+using+an+electrocardiographic+formulaic+assessment&amp;author=V+Ariyarajah&amp;author=L+Prajapat&amp;author=KK+Kumar&amp;author=I+Barac&amp;author=S+Apiyasawat&amp;volume=100&amp;publication_year=2007&amp;pages=894-898&amp;pmid=17719341&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Hillege HL, Fidler V, Diercks GF, van Gilst WH, de Zeeuw D, van Veldhuisen DJ, et al.  Urinary albumin excretion predicts cardiovascular and noncardiovascular mortality in general population. <span><span class="ref-journal">Circulation. </span>2002;<span class="ref-vol">106</span>:1777–1782.</span> [<a href="/pubmed/12356629" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Urinary+albumin+excretion+predicts+cardiovascular+and+noncardiovascular+mortality+in+general+population&amp;author=HL+Hillege&amp;author=V+Fidler&amp;author=GF+Diercks&amp;author=WH+van+Gilst&amp;author=D+de+Zeeuw&amp;volume=106&amp;publication_year=2002&amp;pages=1777-1782&amp;pmid=12356629&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Stolk RP, Rosmalen JG, Postma DS, de Boer RA, Navis G, Slaets JP, et al.  Universal risk factors for multifactorial diseases: Lifelines: A three-generation population-based study. <span><span class="ref-journal">Eur. J. Epidemiol. </span>2008;<span class="ref-vol">23</span>:67–74.</span> [<a href="/pubmed/18075776" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Epidemiol&amp;title=Universal+risk+factors+for+multifactorial+diseases:+Lifelines:+A+three-generation+population-based+study&amp;author=RP+Stolk&amp;author=JG+Rosmalen&amp;author=DS+Postma&amp;author=RA+de+Boer&amp;author=G+Navis&amp;volume=23&amp;publication_year=2008&amp;pages=67-74&amp;pmid=18075776&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Thurman RE, Rynes E, Humbert R, Vierstra J, Maurano MT, Haugen E, et al.  The accessible chromatin landscape of the human genome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:75–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3721348/">PMC free article</a>]</span> [<a href="/pubmed/22955617" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+accessible+chromatin+landscape+of+the+human+genome&amp;author=RE+Thurman&amp;author=E+Rynes&amp;author=R+Humbert&amp;author=J+Vierstra&amp;author=MT+Maurano&amp;volume=489&amp;publication_year=2012&amp;pages=75-82&amp;pmid=22955617&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Bernstein BE, Stamatoyannopoulos JA, Costello JF, Ren B, Milosavljevic A, Meissner A, et al.  The nih roadmap epigenomics mapping consortium. <span><span class="ref-journal">Nat. Biotechnol. </span>2010;<span class="ref-vol">28</span>:1045–1048.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3607281/">PMC free article</a>]</span> [<a href="/pubmed/20944595" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol&amp;title=The+nih+roadmap+epigenomics+mapping+consortium&amp;author=BE+Bernstein&amp;author=JA+Stamatoyannopoulos&amp;author=JF+Costello&amp;author=B+Ren&amp;author=A+Milosavljevic&amp;volume=28&amp;publication_year=2010&amp;pages=1045-1048&amp;pmid=20944595&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Maurano MT, Humbert R, Rynes E, Thurman RE, Haugen E, Wang H, et al.  Systematic localization of common disease-associated variation in regulatory DNA. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">337</span>:1190–1195.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3771521/">PMC free article</a>]</span> [<a href="/pubmed/22955828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Systematic+localization+of+common+disease-associated+variation+in+regulatory+DNA&amp;author=MT+Maurano&amp;author=R+Humbert&amp;author=E+Rynes&amp;author=RE+Thurman&amp;author=E+Haugen&amp;volume=337&amp;publication_year=2012&amp;pages=1190-1195&amp;pmid=22955828&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Feng J, Liu T, Qin B, Zhang Y, Liu XS. Identifying chip-seq enrichment using macs. <span><span class="ref-journal">Nat Protoc. </span>2012;<span class="ref-vol">7</span>:1728–1740.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3868217/">PMC free article</a>]</span> [<a href="/pubmed/22936215" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Identifying+chip-seq+enrichment+using+macs&amp;author=J+Feng&amp;author=T+Liu&amp;author=B+Qin&amp;author=Y+Zhang&amp;author=XS+Liu&amp;volume=7&amp;publication_year=2012&amp;pages=1728-1740&amp;pmid=22936215&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">van den Boogaard M, Wong LY, Tessadori F, Bakker ML, Dreizehnter LK, Wakker V, et al.  Genetic variation in t-box binding element functionally affects scn5a/scn10a enhancer. <span><span class="ref-journal">J. Clin. Invest. </span>2012;<span class="ref-vol">122</span>:2519–2530.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3386824/">PMC free article</a>]</span> [<a href="/pubmed/22706305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Genetic+variation+in+t-box+binding+element+functionally+affects+scn5a/scn10a+enhancer&amp;author=M+van+den+Boogaard&amp;author=LY+Wong&amp;author=F+Tessadori&amp;author=ML+Bakker&amp;author=LK+Dreizehnter&amp;volume=122&amp;publication_year=2012&amp;pages=2519-2530&amp;pmid=22706305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">May D, Blow MJ, Kaplan T, McCulley DJ, Jensen BC, Akiyama JA, et al.  Large-scale discovery of enhancers from human heart tissue. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:89–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3246570/">PMC free article</a>]</span> [<a href="/pubmed/22138689" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Large-scale+discovery+of+enhancers+from+human+heart+tissue&amp;author=D+May&amp;author=MJ+Blow&amp;author=T+Kaplan&amp;author=DJ+McCulley&amp;author=BC+Jensen&amp;volume=44&amp;publication_year=2012&amp;pages=89-93&amp;pmid=22138689&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">He A, Kong SW, Ma Q, Pu WT. Co-occupancy by multiple cardiac transcription factors identifies transcriptional enhancers active in heart. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A. </span>2011;<span class="ref-vol">108</span>:5632–5637.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3078411/">PMC free article</a>]</span> [<a href="/pubmed/21415370" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A&amp;title=Co-occupancy+by+multiple+cardiac+transcription+factors+identifies+transcriptional+enhancers+active+in+heart&amp;author=A+He&amp;author=SW+Kong&amp;author=Q+Ma&amp;author=WT+Pu&amp;volume=108&amp;publication_year=2011&amp;pages=5632-5637&amp;pmid=21415370&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Blow MJ, McCulley DJ, Li Z, Zhang T, Akiyama JA, Holt A, et al.  Chip-seq identification of weakly conserved heart enhancers. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:806–810.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3138496/">PMC free article</a>]</span> [<a href="/pubmed/20729851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Chip-seq+identification+of+weakly+conserved+heart+enhancers&amp;author=MJ+Blow&amp;author=DJ+McCulley&amp;author=Z+Li&amp;author=T+Zhang&amp;author=JA+Akiyama&amp;volume=42&amp;publication_year=2010&amp;pages=806-810&amp;pmid=20729851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Ruijter JM, Ramakers C, Hoogaars WM, Karlen Y, Bakker O, van den Hoff MJ, et al.  Amplification efficiency: Linking baseline and bias in the analysis of quantitative pcr data. <span><span class="ref-journal">Nucleic Acids Res. </span>2009;<span class="ref-vol">37</span>:e45.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2665230/">PMC free article</a>]</span> [<a href="/pubmed/19237396" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Amplification+efficiency:+Linking+baseline+and+bias+in+the+analysis+of+quantitative+pcr+data&amp;author=JM+Ruijter&amp;author=C+Ramakers&amp;author=WM+Hoogaars&amp;author=Y+Karlen&amp;author=O+Bakker&amp;volume=37&amp;publication_year=2009&amp;pages=e45&amp;pmid=19237396&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Hsu J, Hanna P, Van Wagoner DR, Barnard J, Serre D, Chung MK, et al.  Whole genome expression differences in human left and right atria ascertained by rna sequencing. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2012;<span class="ref-vol">5</span>:327–335.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3380143/">PMC free article</a>]</span> [<a href="/pubmed/22474228" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Whole+genome+expression+differences+in+human+left+and+right+atria+ascertained+by+rna+sequencing&amp;author=J+Hsu&amp;author=P+Hanna&amp;author=DR+Van+Wagoner&amp;author=J+Barnard&amp;author=D+Serre&amp;volume=5&amp;publication_year=2012&amp;pages=327-335&amp;pmid=22474228&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Trynka G, Sandor C, Han B, Xu H, Stranger BE, Liu XS, et al.  Chromatin marks identify critical cell types for fine mapping complex trait variants. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:124–130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3826950/">PMC free article</a>]</span> [<a href="/pubmed/23263488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Chromatin+marks+identify+critical+cell+types+for+fine+mapping+complex+trait+variants&amp;author=G+Trynka&amp;author=C+Sandor&amp;author=B+Han&amp;author=H+Xu&amp;author=BE+Stranger&amp;volume=45&amp;publication_year=2013&amp;pages=124-130&amp;pmid=23263488&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Sizarov A, Devalla HD, Anderson RH, Passier R, Christoffels VM, Moorman AF. Molecular analysis of patterning of conduction tissues in the developing human heart. <span><span class="ref-journal">Circ Arrhythm Electrophysiol. </span>2011;<span class="ref-vol">4</span>:532–542.</span> [<a href="/pubmed/21576278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Arrhythm+Electrophysiol&amp;title=Molecular+analysis+of+patterning+of+conduction+tissues+in+the+developing+human+heart&amp;author=A+Sizarov&amp;author=HD+Devalla&amp;author=RH+Anderson&amp;author=R+Passier&amp;author=VM+Christoffels&amp;volume=4&amp;publication_year=2011&amp;pages=532-542&amp;pmid=21576278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Starnes T, Broxmeyer HE, Robertson MJ, Hromas R. Cutting edge: Il-17d, a novel member of the il-17 family, stimulates cytokine production and inhibits hemopoiesis. <span><span class="ref-journal">J. Immunol. </span>2002;<span class="ref-vol">169</span>:642–646.</span> [<a href="/pubmed/12097364" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol&amp;title=Cutting+edge:+Il-17d,+a+novel+member+of+the+il-17+family,+stimulates+cytokine+production+and+inhibits+hemopoiesis&amp;author=T+Starnes&amp;author=HE+Broxmeyer&amp;author=MJ+Robertson&amp;author=R+Hromas&amp;volume=169&amp;publication_year=2002&amp;pages=642-646&amp;pmid=12097364&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Apostolakis S, Vogiatzi K, Amanatidou V, Spandidos DA. Interleukin 8 and cardiovascular disease. <span><span class="ref-journal">Cardiovasc. Res. </span>2009;<span class="ref-vol">84</span>:353–360.</span> [<a href="/pubmed/19617600" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cardiovasc.+Res&amp;title=Interleukin+8+and+cardiovascular+disease&amp;author=S+Apostolakis&amp;author=K+Vogiatzi&amp;author=V+Amanatidou&amp;author=DA+Spandidos&amp;volume=84&amp;publication_year=2009&amp;pages=353-360&amp;pmid=19617600&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Sarwar N, Butterworth AS, Freitag DF, Gregson J, Willeit P, Gorman DN, et al.  Interleukin-6 receptor pathways in coronary heart disease: A collaborative meta-analysis of 82 studies. <span><span class="ref-journal">Lancet. </span>2012;<span class="ref-vol">379</span>:1205–1213.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3316940/">PMC free article</a>]</span> [<a href="/pubmed/22421339" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Interleukin-6+receptor+pathways+in+coronary+heart+disease:+A+collaborative+meta-analysis+of+82+studies&amp;author=N+Sarwar&amp;author=AS+Butterworth&amp;author=DF+Freitag&amp;author=J+Gregson&amp;author=P+Willeit&amp;volume=379&amp;publication_year=2012&amp;pages=1205-1213&amp;pmid=22421339&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Plovanich M, Bogorad RL, Sancak Y, Kamer KJ, Strittmatter L, Li AA, et al.  Micu2, a paralog of micu1, resides within the mitochondrial uniporter complex to regulate calcium handling. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>:e55785.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3567112/">PMC free article</a>]</span> [<a href="/pubmed/23409044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Micu2,+a+paralog+of+micu1,+resides+within+the+mitochondrial+uniporter+complex+to+regulate+calcium+handling&amp;author=M+Plovanich&amp;author=RL+Bogorad&amp;author=Y+Sancak&amp;author=KJ+Kamer&amp;author=L+Strittmatter&amp;volume=8&amp;publication_year=2013&amp;pages=e55785&amp;pmid=23409044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Demirkan A, van Duijn CM, Ugocsai P, Isaacs A, Pramstaller PP, Liebisch G, et al.  Genome-wide association study identifies novel loci associated with circulating phospho- and sphingolipid concentrations. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002490.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3280968/">PMC free article</a>]</span> [<a href="/pubmed/22359512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+identifies+novel+loci+associated+with+circulating+phospho-+and+sphingolipid+concentrations&amp;author=A+Demirkan&amp;author=CM+van+Duijn&amp;author=P+Ugocsai&amp;author=A+Isaacs&amp;author=PP+Pramstaller&amp;volume=8&amp;publication_year=2012&amp;pages=e1002490&amp;pmid=22359512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Kettunen J, Tukiainen T, Sarin AP, Ortega-Alonso A, Tikkanen E, Lyytikainen LP, et al.  Genome-wide association study identifies multiple loci influencing human serum metabolite levels. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:269–276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3605033/">PMC free article</a>]</span> [<a href="/pubmed/22286219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+identifies+multiple+loci+influencing+human+serum+metabolite+levels&amp;author=J+Kettunen&amp;author=T+Tukiainen&amp;author=AP+Sarin&amp;author=A+Ortega-Alonso&amp;author=E+Tikkanen&amp;volume=44&amp;publication_year=2012&amp;pages=269-276&amp;pmid=22286219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Goldberg IJ, Trent CM, Schulze PC. Lipid metabolism and toxicity in the heart. <span><span class="ref-journal">Cell Metab. </span>2012;<span class="ref-vol">15</span>:805–812.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3387529/">PMC free article</a>]</span> [<a href="/pubmed/22682221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Metab&amp;title=Lipid+metabolism+and+toxicity+in+the+heart&amp;author=IJ+Goldberg&amp;author=CM+Trent&amp;author=PC+Schulze&amp;volume=15&amp;publication_year=2012&amp;pages=805-812&amp;pmid=22682221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Olesen MS, Refsgaard L, Holst AG, Larsen AP, Grubb S, Haunso S, et al.  A novel kcnd3 gain-of-function mutation associated with early-onset of persistent lone atrial fibrillation. <span><span class="ref-journal">Cardiovasc. Res. </span>2013;<span class="ref-vol">98</span>:488–495.</span> [<a href="/pubmed/23400760" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cardiovasc.+Res&amp;title=A+novel+kcnd3+gain-of-function+mutation+associated+with+early-onset+of+persistent+lone+atrial+fibrillation&amp;author=MS+Olesen&amp;author=L+Refsgaard&amp;author=AG+Holst&amp;author=AP+Larsen&amp;author=S+Grubb&amp;volume=98&amp;publication_year=2013&amp;pages=488-495&amp;pmid=23400760&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Hirschhorn JN. Genomewide association studies--illuminating biologic pathways. <span><span class="ref-journal">N. Engl. J. Med. </span>2009;<span class="ref-vol">360</span>:1699–1701.</span> [<a href="/pubmed/19369661" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Genomewide+association+studies--illuminating+biologic+pathways&amp;author=JN+Hirschhorn&amp;volume=360&amp;publication_year=2009&amp;pages=1699-1701&amp;pmid=19369661&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4141024&amp;issue-id=241802&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139808679379584title">Abstract</a></li><li><a href="#S4title">Introduction</a></li><li><a href="#S5title">Methods</a></li><li><a href="#S11title">Results</a></li><li><a href="#S14title">Discussion</a></li><li><a href="#S16title">Conclusion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#idm139808680369840title">Footnotes</a></li><li><a href="#idm139808672535840title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4141024/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4141024/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4141024/pdf/nihms-598853.pdf">PDF (410K)</a></li> | <li><a href="#" data-citationid="PMC4141024" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4141024%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4141024%2F&amp;text=Genetic%20Determinants%20of%20P%20Wave%20Duration%20and%20PR%20Segment" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4141024%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24850809" data-aiid="4141024" data-aid="4141024" data-iid="241802" data-domainid="319" data-domain="nihpa" data-accid="PMC4141024" data-md5="3aa823c68cc235f8de706993f75c6caf"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:14:34-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B1394DD5691F100000000081E04CF&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4141024%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4141024/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>