<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5968830/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Multiancestry genome-wide association study of 520,000 subjects identifies 32 loci associated with stroke and stroke subtypes</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5968830/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Multiancestry genome-wide association study of 520,000 subjects identifies 32 loci associated with stroke and stroke subtypes"><meta name="citation_authors" content="Rainer Malik, Ganesh Chauhan, Matthew Traylor, Muralidharan Sargurupremraj, Yukinori Okada, Aniket Mishra, Loes Rutten-Jacobs, Anne-Katrin Giese, Sander W. van der Laan, Solveig Gretarsdottir, Christopher D. Anderson, Michael Chong, Hieab H. H. Adams, Tetsuro Ago, Peter Almgren, Philippe Amouyel, Hakan Ay, Traci M. Bartz, Oscar R. Benavente, Steve Bevan, Giorgio B. Boncoraglio, Robert D. Brown, Jr, Adam S. Butterworth, Caty Carrera, Cara L. Carty, Daniel I. Chasman, Wei-Min Chen, John W. Cole, Adolfo Correa, Ioana Cotlarciuc, Carlos Cruchaga, John Danesh, Paul I. W. de Bakker, Anita L. DeStefano, Marcel den Hoed, Qing Duan, Stefan T. Engelter, Guido J. Falcone, Rebecca F. Gottesman, Raji P. Grewal, Vilmundur Gudnason, Stefan Gustafsson, Jeffrey Haessler, Tamara B. Harris, Ahamad Hassan, Aki S. Havulinna, Susan R. Heckbert, Elizabeth G. Holliday, George Howard, Fang-Chi Hsu, Hyacinth I. Hyacinth, M. Arfan Ikram, Erik ingelsson, Marguerite R. Irvin, Xueqiu Jian, Jordi Jimenez-Conde, Julie A. Johnson, J. Wouter Jukema, Masahiro Kanai, Keith L. Keene, Brett M. Kissela, Dawn O. Kleindorfer, Charles Kooperberg, Michiaki Kubo, Leslie A. Lange, Carl D. Langefeld, Claudia Langenberg, Lenore J. Launer, Jin-Moo Lee, Robin Lemmens, Didier Leys, Cathryn M. Lewis, Wei-Yu Lin, Arne G. Lindgren, Erik Lorentzen, Patrik K. Magnusson, Jane Maguire, Ani Manichaikul, Patrick F. McArdle, James F. Meschia, Braxton D. Mitchell, Thomas H. Mosley, Michael A. Nalls, Toshiharu Ninomiya, Martin J. O’Donnell, Bruce M. Psaty, Sara L. Pulit, Kristiina Rannikmäe, Alexander P. Reiner, Kathryn M. Rexrode, Kenneth Rice, Stephen S. Rich, Paul M. Ridker, Natalia S. Rost, Peter M. Rothwell, Jerome I. Rotter, Tatjana Rundek, Ralph L. Sacco, Saori Sakaue, Michele M. Sale, Veikko Salomaa, Bishwa R. Sapkota, Reinhold Schmidt, Carsten O. Schmidt, Ulf Schminke, Pankaj Sharma, Agnieszka Slowik, Cathie L. M. Sudlow, Christian Tanislav, Turgut Tatlisumak, Kent D. Taylor, Vincent N. S. Thijs, Gudmar Thorleifsson, Unnur Thorsteinsdottir, Steffen Tiedt, Stella Trompet, Christophe Tzourio, Cornelia M. van Duijn, Matthew Walters, Nicholas J. Wareham, Sylvia Wassertheil-Smoller, James G. Wilson, Kerri L. Wiggins, Qiong Yang, Salim Yusuf, AFGen Consortium, Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium, International Genomics of Blood Pressure (iGEN-BP) Consortium, INVENT Consortium, STARNET, Joshua C. Bis, Tomi Pastinen, Arno Ruusalepp, Eric E. Schadt, Simon Koplev, Johan L. M. Björkegren, Veronica Codoni, Mete Civelek, Nicholas L. Smith, David A. Tregouet, Ingrid E. Christophersen, Carolina Roselli, Steven A. Lubitz, Patrick T. Ellinor, E. Shyong Tai, Jaspal S. Kooner, Norihiro Kato, Jiang He, Pim van der Harst, Paul Elliott, John C. Chambers, Fumihiko Takeuchi, Andrew D. Johnson, BioBank Japan Cooperative Hospital Group, COMPASS Consortium, EPiC-CVD Consortium, EPiC-interAct Consortium, International Stroke Genetics Consortium (ISGC), METASTROKE Consortium, Neurology Working Group of the CHARGE Consortium, NiNDS Stroke Genetics Network (SiGN), UK Young Lacunar DNA Study, MEGASTROKE Consortium, Dharambir K. Sanghera, Olle Melander, Christina Jern, Daniel Strbian, Israel Fernandez-Cadenas, W. T. Longstreth, Jr, Arndt Rolfs, Jun Hata, Daniel Woo, Jonathan Rosand, Guillaume Pare, Jemma C. Hopewell, Danish Saleheen, Kari Stefansson, Bradford B. Worrall, Steven J. Kittner, Sudha Seshadri, Myriam Fornage, Hugh S. Markus, Joanna M. M. Howson, Yoichiro Kamatani, Stephanie Debette, Martin Dichgans"><meta name="citation_date" content="April 2018"><meta name="citation_issue" content="4"><meta name="citation_volume" content="50"><meta name="citation_firstpage" content="524"><meta name="citation_doi" content="10.1038/s41588-018-0058-3"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5968830/?report=abstract"><meta name="citation_pmid" content="29531354"><meta name="DC.Title" content="Multiancestry genome-wide association study of 520,000 subjects identifies 32 loci associated with stroke and stroke subtypes"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Rainer Malik"><meta name="DC.Contributor" content="Ganesh Chauhan"><meta name="DC.Contributor" content="Matthew Traylor"><meta name="DC.Contributor" content="Muralidharan Sargurupremraj"><meta name="DC.Contributor" content="Yukinori Okada"><meta name="DC.Contributor" content="Aniket Mishra"><meta name="DC.Contributor" content="Loes Rutten-Jacobs"><meta name="DC.Contributor" content="Anne-Katrin Giese"><meta name="DC.Contributor" content="Sander W. van der Laan"><meta name="DC.Contributor" content="Solveig Gretarsdottir"><meta name="DC.Contributor" content="Christopher D. Anderson"><meta name="DC.Contributor" content="Michael Chong"><meta name="DC.Contributor" content="Hieab H. H. Adams"><meta name="DC.Contributor" content="Tetsuro Ago"><meta name="DC.Contributor" content="Peter Almgren"><meta name="DC.Contributor" content="Philippe Amouyel"><meta name="DC.Contributor" content="Hakan Ay"><meta name="DC.Contributor" content="Traci M. Bartz"><meta name="DC.Contributor" content="Oscar R. Benavente"><meta name="DC.Contributor" content="Steve Bevan"><meta name="DC.Contributor" content="Giorgio B. Boncoraglio"><meta name="DC.Contributor" content="Robert D. Brown, Jr"><meta name="DC.Contributor" content="Adam S. Butterworth"><meta name="DC.Contributor" content="Caty Carrera"><meta name="DC.Contributor" content="Cara L. Carty"><meta name="DC.Contributor" content="Daniel I. Chasman"><meta name="DC.Contributor" content="Wei-Min Chen"><meta name="DC.Contributor" content="John W. Cole"><meta name="DC.Contributor" content="Adolfo Correa"><meta name="DC.Contributor" content="Ioana Cotlarciuc"><meta name="DC.Contributor" content="Carlos Cruchaga"><meta name="DC.Contributor" content="John Danesh"><meta name="DC.Contributor" content="Paul I. W. de Bakker"><meta name="DC.Contributor" content="Anita L. DeStefano"><meta name="DC.Contributor" content="Marcel den Hoed"><meta name="DC.Contributor" content="Qing Duan"><meta name="DC.Contributor" content="Stefan T. Engelter"><meta name="DC.Contributor" content="Guido J. Falcone"><meta name="DC.Contributor" content="Rebecca F. Gottesman"><meta name="DC.Contributor" content="Raji P. Grewal"><meta name="DC.Contributor" content="Vilmundur Gudnason"><meta name="DC.Contributor" content="Stefan Gustafsson"><meta name="DC.Contributor" content="Jeffrey Haessler"><meta name="DC.Contributor" content="Tamara B. Harris"><meta name="DC.Contributor" content="Ahamad Hassan"><meta name="DC.Contributor" content="Aki S. Havulinna"><meta name="DC.Contributor" content="Susan R. Heckbert"><meta name="DC.Contributor" content="Elizabeth G. Holliday"><meta name="DC.Contributor" content="George Howard"><meta name="DC.Contributor" content="Fang-Chi Hsu"><meta name="DC.Contributor" content="Hyacinth I. Hyacinth"><meta name="DC.Contributor" content="M. Arfan Ikram"><meta name="DC.Contributor" content="Erik ingelsson"><meta name="DC.Contributor" content="Marguerite R. Irvin"><meta name="DC.Contributor" content="Xueqiu Jian"><meta name="DC.Contributor" content="Jordi Jimenez-Conde"><meta name="DC.Contributor" content="Julie A. Johnson"><meta name="DC.Contributor" content="J. Wouter Jukema"><meta name="DC.Contributor" content="Masahiro Kanai"><meta name="DC.Contributor" content="Keith L. Keene"><meta name="DC.Contributor" content="Brett M. Kissela"><meta name="DC.Contributor" content="Dawn O. Kleindorfer"><meta name="DC.Contributor" content="Charles Kooperberg"><meta name="DC.Contributor" content="Michiaki Kubo"><meta name="DC.Contributor" content="Leslie A. Lange"><meta name="DC.Contributor" content="Carl D. Langefeld"><meta name="DC.Contributor" content="Claudia Langenberg"><meta name="DC.Contributor" content="Lenore J. Launer"><meta name="DC.Contributor" content="Jin-Moo Lee"><meta name="DC.Contributor" content="Robin Lemmens"><meta name="DC.Contributor" content="Didier Leys"><meta name="DC.Contributor" content="Cathryn M. Lewis"><meta name="DC.Contributor" content="Wei-Yu Lin"><meta name="DC.Contributor" content="Arne G. Lindgren"><meta name="DC.Contributor" content="Erik Lorentzen"><meta name="DC.Contributor" content="Patrik K. Magnusson"><meta name="DC.Contributor" content="Jane Maguire"><meta name="DC.Contributor" content="Ani Manichaikul"><meta name="DC.Contributor" content="Patrick F. McArdle"><meta name="DC.Contributor" content="James F. Meschia"><meta name="DC.Contributor" content="Braxton D. Mitchell"><meta name="DC.Contributor" content="Thomas H. Mosley"><meta name="DC.Contributor" content="Michael A. Nalls"><meta name="DC.Contributor" content="Toshiharu Ninomiya"><meta name="DC.Contributor" content="Martin J. O’Donnell"><meta name="DC.Contributor" content="Bruce M. Psaty"><meta name="DC.Contributor" content="Sara L. Pulit"><meta name="DC.Contributor" content="Kristiina Rannikmäe"><meta name="DC.Contributor" content="Alexander P. Reiner"><meta name="DC.Contributor" content="Kathryn M. Rexrode"><meta name="DC.Contributor" content="Kenneth Rice"><meta name="DC.Contributor" content="Stephen S. Rich"><meta name="DC.Contributor" content="Paul M. Ridker"><meta name="DC.Contributor" content="Natalia S. Rost"><meta name="DC.Contributor" content="Peter M. Rothwell"><meta name="DC.Contributor" content="Jerome I. Rotter"><meta name="DC.Contributor" content="Tatjana Rundek"><meta name="DC.Contributor" content="Ralph L. Sacco"><meta name="DC.Contributor" content="Saori Sakaue"><meta name="DC.Contributor" content="Michele M. Sale"><meta name="DC.Contributor" content="Veikko Salomaa"><meta name="DC.Contributor" content="Bishwa R. Sapkota"><meta name="DC.Contributor" content="Reinhold Schmidt"><meta name="DC.Contributor" content="Carsten O. Schmidt"><meta name="DC.Contributor" content="Ulf Schminke"><meta name="DC.Contributor" content="Pankaj Sharma"><meta name="DC.Contributor" content="Agnieszka Slowik"><meta name="DC.Contributor" content="Cathie L. M. Sudlow"><meta name="DC.Contributor" content="Christian Tanislav"><meta name="DC.Contributor" content="Turgut Tatlisumak"><meta name="DC.Contributor" content="Kent D. Taylor"><meta name="DC.Contributor" content="Vincent N. S. Thijs"><meta name="DC.Contributor" content="Gudmar Thorleifsson"><meta name="DC.Contributor" content="Unnur Thorsteinsdottir"><meta name="DC.Contributor" content="Steffen Tiedt"><meta name="DC.Contributor" content="Stella Trompet"><meta name="DC.Contributor" content="Christophe Tzourio"><meta name="DC.Contributor" content="Cornelia M. van Duijn"><meta name="DC.Contributor" content="Matthew Walters"><meta name="DC.Contributor" content="Nicholas J. Wareham"><meta name="DC.Contributor" content="Sylvia Wassertheil-Smoller"><meta name="DC.Contributor" content="James G. Wilson"><meta name="DC.Contributor" content="Kerri L. Wiggins"><meta name="DC.Contributor" content="Qiong Yang"><meta name="DC.Contributor" content="Salim Yusuf"><meta name="DC.Contributor" content="AFGen Consortium"><meta name="DC.Contributor" content="Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium"><meta name="DC.Contributor" content="International Genomics of Blood Pressure (iGEN-BP) Consortium"><meta name="DC.Contributor" content="INVENT Consortium"><meta name="DC.Contributor" content="STARNET"><meta name="DC.Contributor" content="Joshua C. Bis"><meta name="DC.Contributor" content="Tomi Pastinen"><meta name="DC.Contributor" content="Arno Ruusalepp"><meta name="DC.Contributor" content="Eric E. Schadt"><meta name="DC.Contributor" content="Simon Koplev"><meta name="DC.Contributor" content="Johan L. M. Björkegren"><meta name="DC.Contributor" content="Veronica Codoni"><meta name="DC.Contributor" content="Mete Civelek"><meta name="DC.Contributor" content="Nicholas L. Smith"><meta name="DC.Contributor" content="David A. Tregouet"><meta name="DC.Contributor" content="Ingrid E. Christophersen"><meta name="DC.Contributor" content="Carolina Roselli"><meta name="DC.Contributor" content="Steven A. Lubitz"><meta name="DC.Contributor" content="Patrick T. Ellinor"><meta name="DC.Contributor" content="E. Shyong Tai"><meta name="DC.Contributor" content="Jaspal S. Kooner"><meta name="DC.Contributor" content="Norihiro Kato"><meta name="DC.Contributor" content="Jiang He"><meta name="DC.Contributor" content="Pim van der Harst"><meta name="DC.Contributor" content="Paul Elliott"><meta name="DC.Contributor" content="John C. Chambers"><meta name="DC.Contributor" content="Fumihiko Takeuchi"><meta name="DC.Contributor" content="Andrew D. Johnson"><meta name="DC.Contributor" content="BioBank Japan Cooperative Hospital Group"><meta name="DC.Contributor" content="COMPASS Consortium"><meta name="DC.Contributor" content="EPiC-CVD Consortium"><meta name="DC.Contributor" content="EPiC-interAct Consortium"><meta name="DC.Contributor" content="International Stroke Genetics Consortium (ISGC)"><meta name="DC.Contributor" content="METASTROKE Consortium"><meta name="DC.Contributor" content="Neurology Working Group of the CHARGE Consortium"><meta name="DC.Contributor" content="NiNDS Stroke Genetics Network (SiGN)"><meta name="DC.Contributor" content="UK Young Lacunar DNA Study"><meta name="DC.Contributor" content="MEGASTROKE Consortium"><meta name="DC.Contributor" content="Dharambir K. Sanghera"><meta name="DC.Contributor" content="Olle Melander"><meta name="DC.Contributor" content="Christina Jern"><meta name="DC.Contributor" content="Daniel Strbian"><meta name="DC.Contributor" content="Israel Fernandez-Cadenas"><meta name="DC.Contributor" content="W. T. Longstreth, Jr"><meta name="DC.Contributor" content="Arndt Rolfs"><meta name="DC.Contributor" content="Jun Hata"><meta name="DC.Contributor" content="Daniel Woo"><meta name="DC.Contributor" content="Jonathan Rosand"><meta name="DC.Contributor" content="Guillaume Pare"><meta name="DC.Contributor" content="Jemma C. Hopewell"><meta name="DC.Contributor" content="Danish Saleheen"><meta name="DC.Contributor" content="Kari Stefansson"><meta name="DC.Contributor" content="Bradford B. Worrall"><meta name="DC.Contributor" content="Steven J. Kittner"><meta name="DC.Contributor" content="Sudha Seshadri"><meta name="DC.Contributor" content="Myriam Fornage"><meta name="DC.Contributor" content="Hugh S. Markus"><meta name="DC.Contributor" content="Joanna M. M. Howson"><meta name="DC.Contributor" content="Yoichiro Kamatani"><meta name="DC.Contributor" content="Stephanie Debette"><meta name="DC.Contributor" content="Martin Dichgans"><meta name="DC.Date" content="2018 Apr"><meta name="DC.Identifier" content="10.1038/s41588-018-0058-3"><meta name="DC.Language" content="en"><meta property="og:title" content="Multiancestry genome-wide association study of 520,000 subjects identifies 32 loci associated with stroke and stroke subtypes"><meta property="og:type" content="article"><meta property="og:description" content="Stroke has multiple etiologies, but the underlying genes and pathways are largely unknown. We conducted a multiancestry genome-wide-association meta-analysis in 521,612 individuals (67,162 cases and 454,450 controls) and discovered 22 new stroke risk ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5968830/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5968830/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5968830/pdf/nihms959658.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BDAA3DD5A876100000000095E0584.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5968830%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5968830</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2018 May 25.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=29531354" target="pmc_ext" ref="reftype=publisher&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2018 Apr; 50(4): 524–537. </span></a></div><span class="fm-vol-iss-date">Published online 2018 Mar 12. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fs41588-018-0058-3" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/s41588-018-0058-3</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5968830</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS959658</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29531354">29531354</a></div></div></div></div><h1 class="content-title">Multiancestry genome-wide association study of 520,000 subjects identifies 32 loci associated with stroke and stroke subtypes</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Malik%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458738455280" co-class="co-affbox">Rainer Malik</a>,<sup>1,</sup><sup>185</sup> <a href="/pubmed/?term=Chauhan%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458735264112" co-class="co-affbox">Ganesh Chauhan</a>,<sup>2,</sup><sup>3,</sup><sup>185</sup> <a href="/pubmed/?term=Traylor%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741814512" co-class="co-affbox">Matthew Traylor</a>,<sup>4,</sup><sup>185</sup> <a href="/pubmed/?term=Sargurupremraj%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458749389984" co-class="co-affbox">Muralidharan Sargurupremraj</a>,<sup>3,</sup><sup>5,</sup><sup>185</sup> <a href="/pubmed/?term=Okada%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742646208" co-class="co-affbox">Yukinori Okada</a>,<sup>67,</sup><sup>8,</sup><sup>185</sup> <a href="/pubmed/?term=Mishra%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741523568" co-class="co-affbox">Aniket Mishra</a>,<sup>3,</sup><sup>5</sup> <a href="/pubmed/?term=Rutten-Jacobs%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458739856064" co-class="co-affbox">Loes Rutten-Jacobs</a>,<sup>4</sup> <a href="/pubmed/?term=Giese%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740055648" co-class="co-affbox">Anne-Katrin Giese</a>,<sup>9</sup> <a href="/pubmed/?term=van%20der%20Laan%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458749339392" co-class="co-affbox">Sander W. van der Laan</a>,<sup>10</sup> <a href="/pubmed/?term=Gretarsdottir%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458737294800" co-class="co-affbox">Solveig Gretarsdottir</a>,<sup>11</sup> <a href="/pubmed/?term=Anderson%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458737292816" co-class="co-affbox">Christopher D. Anderson</a>,<sup>12,</sup><sup>13,</sup><sup>14</sup> <a href="/pubmed/?term=Chong%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458746352096" co-class="co-affbox">Michael Chong</a>,<sup>15</sup> <a href="/pubmed/?term=Adams%20HH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458739123760" co-class="co-affbox">Hieab H. H. Adams</a>,<sup>16,</sup><sup>17</sup> <a href="/pubmed/?term=Ago%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458728776128" co-class="co-affbox">Tetsuro Ago</a>,<sup>18</sup> <a href="/pubmed/?term=Almgren%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747300096" co-class="co-affbox">Peter Almgren</a>,<sup>19</sup> <a href="/pubmed/?term=Amouyel%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747298112" co-class="co-affbox">Philippe Amouyel</a>,<sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=Ay%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742723744" co-class="co-affbox">Hakan Ay</a>,<sup>13,</sup><sup>22</sup> <a href="/pubmed/?term=Bartz%20TM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458779776048" co-class="co-affbox">Traci M. Bartz</a>,<sup>23</sup> <a href="/pubmed/?term=Benavente%20OR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458749408400" co-class="co-affbox">Oscar R. Benavente</a>,<sup>24</sup> <a href="/pubmed/?term=Bevan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458749406416" co-class="co-affbox">Steve Bevan</a>,<sup>25</sup> <a href="/pubmed/?term=Boncoraglio%20GB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458783618000" co-class="co-affbox">Giorgio B. Boncoraglio</a>,<sup>26</sup> <a href="/pubmed/?term=Brown%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458744696928" co-class="co-affbox">Robert D. Brown, Jr</a>,<sup>27</sup> <a href="/pubmed/?term=Butterworth%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458749452256" co-class="co-affbox">Adam S. Butterworth</a>,<sup>28,</sup><sup>29</sup> <a href="/pubmed/?term=Carrera%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458739181792" co-class="co-affbox">Caty Carrera</a>,<sup>30,</sup><sup>31</sup> <a href="/pubmed/?term=Carty%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747313008" co-class="co-affbox">Cara L. Carty</a>,<sup>32,</sup><sup>33</sup> <a href="/pubmed/?term=Chasman%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747656864" co-class="co-affbox">Daniel I. Chasman</a>,<sup>34,</sup><sup>35</sup> <a href="/pubmed/?term=Chen%20WM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742297824" co-class="co-affbox">Wei-Min Chen</a>,<sup>36</sup> <a href="/pubmed/?term=Cole%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458744360768" co-class="co-affbox">John W. Cole</a>,<sup>37</sup> <a href="/pubmed/?term=Correa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741359664" co-class="co-affbox">Adolfo Correa</a>,<sup>38</sup> <a href="/pubmed/?term=Cotlarciuc%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741745952" co-class="co-affbox">Ioana Cotlarciuc</a>,<sup>39</sup> <a href="/pubmed/?term=Cruchaga%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458737573712" co-class="co-affbox">Carlos Cruchaga</a>,<sup>40,</sup><sup>41</sup> <a href="/pubmed/?term=Danesh%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748154336" co-class="co-affbox">John Danesh</a>,<sup>28,</sup><sup>42,</sup><sup>43,</sup><sup>44</sup> <a href="/pubmed/?term=de%20Bakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747641488" co-class="co-affbox">Paul I. W. de Bakker</a>,<sup>45,</sup><sup>46</sup> <a href="/pubmed/?term=DeStefano%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458738108512" co-class="co-affbox">Anita L. DeStefano</a>,<sup>47,</sup><sup>48</sup> <a href="/pubmed/?term=den%20Hoed%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747614528" co-class="co-affbox">Marcel den Hoed</a>,<sup>49</sup> <a href="/pubmed/?term=Duan%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458701464256" co-class="co-affbox">Qing Duan</a>,<sup>50</sup> <a href="/pubmed/?term=Engelter%20ST%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741568720" co-class="co-affbox">Stefan T. Engelter</a>,<sup>51,</sup><sup>52</sup> <a href="/pubmed/?term=Falcone%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741156496" co-class="co-affbox">Guido J. Falcone</a>,<sup>53,</sup><sup>54</sup> <a href="/pubmed/?term=Gottesman%20RF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742623360" co-class="co-affbox">Rebecca F. Gottesman</a>,<sup>55</sup> <a href="/pubmed/?term=Grewal%20RP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747957936" co-class="co-affbox">Raji P. Grewal</a>,<sup>56</sup> <a href="/pubmed/?term=Gudnason%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747955696" co-class="co-affbox">Vilmundur Gudnason</a>,<sup>57,</sup><sup>58</sup> <a href="/pubmed/?term=Gustafsson%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743452688" co-class="co-affbox">Stefan Gustafsson</a>,<sup>59</sup> <a href="/pubmed/?term=Haessler%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748216112" co-class="co-affbox">Jeffrey Haessler</a>,<sup>60</sup> <a href="/pubmed/?term=Harris%20TB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748213872" co-class="co-affbox">Tamara B. Harris</a>,<sup>61</sup> <a href="/pubmed/?term=Hassan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458779913744" co-class="co-affbox">Ahamad Hassan</a>,<sup>62</sup> <a href="/pubmed/?term=Havulinna%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458732010000" co-class="co-affbox">Aki S. Havulinna</a>,<sup>63,</sup><sup>64</sup> <a href="/pubmed/?term=Heckbert%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458732113376" co-class="co-affbox">Susan R. Heckbert</a>,<sup>65</sup> <a href="/pubmed/?term=Holliday%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458732111136" co-class="co-affbox">Elizabeth G. Holliday</a>,<sup>66,</sup><sup>67</sup> <a href="/pubmed/?term=Howard%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458737728272" co-class="co-affbox">George Howard</a>,<sup>68</sup> <a href="/pubmed/?term=Hsu%20FC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458731811296" co-class="co-affbox">Fang-Chi Hsu</a>,<sup>69</sup> <a href="/pubmed/?term=Hyacinth%20HI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458731809200" co-class="co-affbox">Hyacinth I. Hyacinth</a>,<sup>70</sup> <a href="/pubmed/?term=Ikram%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740278336" co-class="co-affbox">M. Arfan Ikram</a>,<sup>16</sup> <a href="/pubmed/?term=ingelsson%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741252512" co-class="co-affbox">Erik ingelsson</a>,<sup>71,</sup><sup>72</sup> <a href="/pubmed/?term=Irvin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458729539456" co-class="co-affbox">Marguerite R. Irvin</a>,<sup>73</sup> <a href="/pubmed/?term=Jian%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747771264" co-class="co-affbox">Xueqiu Jian</a>,<sup>74</sup> <a href="/pubmed/?term=Jimenez-Conde%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458745197616" co-class="co-affbox">Jordi Jimenez-Conde</a>,<sup>75</sup> <a href="/pubmed/?term=Johnson%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740471088" co-class="co-affbox">Julie A. Johnson</a>,<sup>76,</sup><sup>77</sup> <a href="/pubmed/?term=Jukema%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741501552" co-class="co-affbox">J. Wouter Jukema</a>,<sup>78</sup> <a href="/pubmed/?term=Kanai%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741803072" co-class="co-affbox">Masahiro Kanai</a>,<sup>67,</sup><sup>79</sup> <a href="/pubmed/?term=Keene%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740584880" co-class="co-affbox">Keith L. Keene</a>,<sup>80,</sup><sup>81</sup> <a href="/pubmed/?term=Kissela%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743486128" co-class="co-affbox">Brett M. Kissela</a>,<sup>82</sup> <a href="/pubmed/?term=Kleindorfer%20DO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742670912" co-class="co-affbox">Dawn O. Kleindorfer</a>,<sup>82</sup> <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742668736" co-class="co-affbox">Charles Kooperberg</a>,<sup>60</sup> <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747643008" co-class="co-affbox">Michiaki Kubo</a>,<sup>83</sup> <a href="/pubmed/?term=Lange%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458780215728" co-class="co-affbox">Leslie A. Lange</a>,<sup>84</sup> <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748931728" co-class="co-affbox">Carl D. Langefeld</a>,<sup>85</sup> <a href="/pubmed/?term=Langenberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740414592" co-class="co-affbox">Claudia Langenberg</a>,<sup>86</sup> <a href="/pubmed/?term=Launer%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747358320" co-class="co-affbox">Lenore J. Launer</a>,<sup>87</sup> <a href="/pubmed/?term=Lee%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458745388928" co-class="co-affbox">Jin-Moo Lee</a>,<sup>88</sup> <a href="/pubmed/?term=Lemmens%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743113296" co-class="co-affbox">Robin Lemmens</a>,<sup>89,</sup><sup>90</sup> <a href="/pubmed/?term=Leys%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741497216" co-class="co-affbox">Didier Leys</a>,<sup>91</sup> <a href="/pubmed/?term=Lewis%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458736956640" co-class="co-affbox">Cathryn M. Lewis</a>,<sup>92,</sup><sup>93</sup> <a href="/pubmed/?term=Lin%20WY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458745897280" co-class="co-affbox">Wei-Yu Lin</a>,<sup>28,</sup><sup>94</sup> <a href="/pubmed/?term=Lindgren%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458729510352" co-class="co-affbox">Arne G. Lindgren</a>,<sup>95,</sup><sup>96</sup> <a href="/pubmed/?term=Lorentzen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458737800112" co-class="co-affbox">Erik Lorentzen</a>,<sup>97</sup> <a href="/pubmed/?term=Magnusson%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740169680" co-class="co-affbox">Patrik K. Magnusson</a>,<sup>98</sup> <a href="/pubmed/?term=Maguire%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458739882864" co-class="co-affbox">Jane Maguire</a>,<sup>99</sup> <a href="/pubmed/?term=Manichaikul%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458739902112" co-class="co-affbox">Ani Manichaikul</a>,<sup>36</sup> <a href="/pubmed/?term=McArdle%20PF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747662720" co-class="co-affbox">Patrick F. McArdle</a>,<sup>100</sup> <a href="/pubmed/?term=Meschia%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458738207744" co-class="co-affbox">James F. Meschia</a>,<sup>101</sup> <a href="/pubmed/?term=Mitchell%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458732003632" co-class="co-affbox">Braxton D. Mitchell</a>,<sup>100,</sup><sup>102</sup> <a href="/pubmed/?term=Mosley%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458732610496" co-class="co-affbox">Thomas H. Mosley</a>,<sup>103,</sup><sup>104</sup> <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741475456" co-class="co-affbox">Michael A. Nalls</a>,<sup>105,</sup><sup>106</sup> <a href="/pubmed/?term=Ninomiya%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742315168" co-class="co-affbox">Toshiharu Ninomiya</a>,<sup>107</sup> <a href="/pubmed/?term=O%26%23x02019%3BDonnell%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740230304" co-class="co-affbox">Martin J. O’Donnell</a>,<sup>15,</sup><sup>108</sup> <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458732133120" co-class="co-affbox">Bruce M. Psaty</a>,<sup>109,</sup><sup>110,</sup><sup>111,</sup><sup>112</sup> <a href="/pubmed/?term=Pulit%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742690368" co-class="co-affbox">Sara L. Pulit</a>,<sup>45,</sup><sup>113</sup> <a href="/pubmed/?term=Rannikm%26%23x000e4%3Be%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748538304" co-class="co-affbox">Kristiina Rannikmäe</a>,<sup>114,</sup><sup>115</sup> <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748880000" co-class="co-affbox">Alexander P. Reiner</a>,<sup>65,</sup><sup>116</sup> <a href="/pubmed/?term=Rexrode%20KM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458735916864" co-class="co-affbox">Kathryn M. Rexrode</a>,<sup>117</sup> <a href="/pubmed/?term=Rice%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458728947920" co-class="co-affbox">Kenneth Rice</a>,<sup>118</sup> <a href="/pubmed/?term=Rich%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458738279344" co-class="co-affbox">Stephen S. Rich</a>,<sup>36</sup> <a href="/pubmed/?term=Ridker%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741672800" co-class="co-affbox">Paul M. Ridker</a>,<sup>34,</sup><sup>35</sup> <a href="/pubmed/?term=Rost%20NS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458820577376" co-class="co-affbox">Natalia S. Rost</a>,<sup>9,</sup><sup>13</sup> <a href="/pubmed/?term=Rothwell%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743739632" co-class="co-affbox">Peter M. Rothwell</a>,<sup>119</sup> <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458739157680" co-class="co-affbox">Jerome I. Rotter</a>,<sup>120,</sup><sup>121</sup> <a href="/pubmed/?term=Rundek%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748884512" co-class="co-affbox">Tatjana Rundek</a>,<sup>122</sup> <a href="/pubmed/?term=Sacco%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458737212944" co-class="co-affbox">Ralph L. Sacco</a>,<sup>122</sup> <a href="/pubmed/?term=Sakaue%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748867360" co-class="co-affbox">Saori Sakaue</a>,<sup>7,</sup><sup>123</sup> <a href="/pubmed/?term=Sale%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747365904" co-class="co-affbox">Michele M. Sale</a>,<sup>124</sup> <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458738886944" co-class="co-affbox">Veikko Salomaa</a>,<sup>63</sup> <a href="/pubmed/?term=Sapkota%20BR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458738884736" co-class="co-affbox">Bishwa R. Sapkota</a>,<sup>125</sup> <a href="/pubmed/?term=Schmidt%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458729190896" co-class="co-affbox">Reinhold Schmidt</a>,<sup>126</sup> <a href="/pubmed/?term=Schmidt%20CO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458729589200" co-class="co-affbox">Carsten O. Schmidt</a>,<sup>127</sup> <a href="/pubmed/?term=Schminke%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458735244912" co-class="co-affbox">Ulf Schminke</a>,<sup>128</sup> <a href="/pubmed/?term=Sharma%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458784252304" co-class="co-affbox">Pankaj Sharma</a>,<sup>39</sup> <a href="/pubmed/?term=Slowik%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742238944" co-class="co-affbox">Agnieszka Slowik</a>,<sup>129</sup> <a href="/pubmed/?term=Sudlow%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458731287200" co-class="co-affbox">Cathie L. M. Sudlow</a>,<sup>114,</sup><sup>115</sup> <a href="/pubmed/?term=Tanislav%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747775520" co-class="co-affbox">Christian Tanislav</a>,<sup>130</sup> <a href="/pubmed/?term=Tatlisumak%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458745846560" co-class="co-affbox">Turgut Tatlisumak</a>,<sup>131,</sup><sup>132</sup> <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748647200" co-class="co-affbox">Kent D. Taylor</a>,<sup>120,</sup><sup>121</sup> <a href="/pubmed/?term=Thijs%20VN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741464416" co-class="co-affbox">Vincent N. S. Thijs</a>,<sup>133,</sup><sup>134</sup> <a href="/pubmed/?term=Thorleifsson%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743044384" co-class="co-affbox">Gudmar Thorleifsson</a>,<sup>11</sup> <a href="/pubmed/?term=Thorsteinsdottir%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458780974512" co-class="co-affbox">Unnur Thorsteinsdottir</a>,<sup>11</sup> <a href="/pubmed/?term=Tiedt%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458737576416" co-class="co-affbox">Steffen Tiedt</a>,<sup>1</sup> <a href="/pubmed/?term=Trompet%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458820574848" co-class="co-affbox">Stella Trompet</a>,<sup>135</sup> <a href="/pubmed/?term=Tzourio%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458820572544" co-class="co-affbox">Christophe Tzourio</a>,<sup>3,</sup><sup>136,</sup><sup>137</sup> <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458738214176" co-class="co-affbox">Cornelia M. van Duijn</a>,<sup>138,</sup><sup>139</sup> <a href="/pubmed/?term=Walters%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748208224" co-class="co-affbox">Matthew Walters</a>,<sup>140</sup> <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740693424" co-class="co-affbox">Nicholas J. Wareham</a>,<sup>86</sup> <a href="/pubmed/?term=Wassertheil-Smoller%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741327248" co-class="co-affbox">Sylvia Wassertheil-Smoller</a>,<sup>141</sup> <a href="/pubmed/?term=Wilson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743882416" co-class="co-affbox">James G. Wilson</a>,<sup>142</sup> <a href="/pubmed/?term=Wiggins%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743880112" co-class="co-affbox">Kerri L. Wiggins</a>,<sup>109</sup> <a href="/pubmed/?term=Yang%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742082464" co-class="co-affbox">Qiong Yang</a>,<sup>47</sup> <a href="/pubmed/?term=Yusuf%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458744176480" co-class="co-affbox">Salim Yusuf</a>,<sup>15</sup> AFGen Consortium,<sup>143</sup> Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium,<sup>143</sup> International Genomics of Blood Pressure (iGEN-BP) Consortium,<sup>143</sup> INVENT Consortium,<sup>143</sup> STARNET,<sup>143</sup> <a href="/pubmed/?term=Bis%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458745349664" co-class="co-affbox">Joshua C. Bis</a>,<sup>109</sup> <a href="/pubmed/?term=Pastinen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747110384" co-class="co-affbox">Tomi Pastinen</a>,<sup>144</sup> <a href="/pubmed/?term=Ruusalepp%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747108144" co-class="co-affbox">Arno Ruusalepp</a>,<sup>145,</sup><sup>146,</sup><sup>147</sup> <a href="/pubmed/?term=Schadt%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741506256" co-class="co-affbox">Eric E. Schadt</a>,<sup>148</sup> <a href="/pubmed/?term=Koplev%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743248400" co-class="co-affbox">Simon Koplev</a>,<sup>148</sup> <a href="/pubmed/?term=Bj%26%23x000f6%3Brkegren%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458735657376" co-class="co-affbox">Johan L. M. Björkegren</a>,<sup>148,</sup><sup>149,</sup><sup>150,</sup><sup>151</sup> <a href="/pubmed/?term=Codoni%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741137072" co-class="co-affbox">Veronica Codoni</a>,<sup>152,</sup><sup>153</sup> <a href="/pubmed/?term=Civelek%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458749683872" co-class="co-affbox">Mete Civelek</a>,<sup>124,</sup><sup>154</sup> <a href="/pubmed/?term=Smith%20NL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740612208" co-class="co-affbox">Nicholas L. Smith</a>,<sup>65,</sup><sup>155,</sup><sup>156</sup> <a href="/pubmed/?term=Tregouet%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458744378192" co-class="co-affbox">David A. Tregouet</a>,<sup>152,</sup><sup>153</sup> <a href="/pubmed/?term=Christophersen%20IE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458746446368" co-class="co-affbox">Ingrid E. Christophersen</a>,<sup>54,</sup><sup>157,</sup><sup>158</sup> <a href="/pubmed/?term=Roselli%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743150096" co-class="co-affbox">Carolina Roselli</a>,<sup>54</sup> <a href="/pubmed/?term=Lubitz%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458729070960" co-class="co-affbox">Steven A. Lubitz</a>,<sup>54,</sup><sup>157</sup> <a href="/pubmed/?term=Ellinor%20PT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458746004752" co-class="co-affbox">Patrick T. Ellinor</a>,<sup>54,</sup><sup>157</sup> <a href="/pubmed/?term=Tai%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741677792" co-class="co-affbox">E. Shyong Tai</a>,<sup>159</sup> <a href="/pubmed/?term=Kooner%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742731376" co-class="co-affbox">Jaspal S. Kooner</a>,<sup>160</sup> <a href="/pubmed/?term=Kato%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742729104" co-class="co-affbox">Norihiro Kato</a>,<sup>161</sup> <a href="/pubmed/?term=He%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743334208" co-class="co-affbox">Jiang He</a>,<sup>162</sup> <a href="/pubmed/?term=van%20der%20Harst%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458739386448" co-class="co-affbox">Pim van der Harst</a>,<sup>163</sup> <a href="/pubmed/?term=Elliott%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458743425424" co-class="co-affbox">Paul Elliott</a>,<sup>164</sup> <a href="/pubmed/?term=Chambers%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747826064" co-class="co-affbox">John C. Chambers</a>,<sup>165,</sup><sup>166</sup> <a href="/pubmed/?term=Takeuchi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458780946864" co-class="co-affbox">Fumihiko Takeuchi</a>,<sup>161</sup> <a href="/pubmed/?term=Johnson%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458732090240" co-class="co-affbox">Andrew D. Johnson</a>,<sup>48,</sup><sup>167</sup> BioBank Japan Cooperative Hospital Group,<sup>143</sup> COMPASS Consortium,<sup>143</sup> EPiC-CVD Consortium,<sup>143</sup> EPiC-interAct Consortium,<sup>143</sup> International Stroke Genetics Consortium (ISGC),<sup>143</sup> METASTROKE Consortium,<sup>143</sup> Neurology Working Group of the CHARGE Consortium,<sup>143</sup> NiNDS Stroke Genetics Network (SiGN),<sup>143</sup> UK Young Lacunar DNA Study,<sup>143</sup> MEGASTROKE Consortium,<sup>168</sup> <a href="/pubmed/?term=Sanghera%20DK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741209984" co-class="co-affbox">Dharambir K. Sanghera</a>,<sup>125,</sup><sup>169,</sup><sup>170</sup> <a href="/pubmed/?term=Melander%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458748247760" co-class="co-affbox">Olle Melander</a>,<sup>19</sup> <a href="/pubmed/?term=Jern%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458749241200" co-class="co-affbox">Christina Jern</a>,<sup>171</sup> <a href="/pubmed/?term=Strbian%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458737139008" co-class="co-affbox">Daniel Strbian</a>,<sup>172,</sup><sup>173</sup> <a href="/pubmed/?term=Fernandez-Cadenas%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740479696" co-class="co-affbox">Israel Fernandez-Cadenas</a>,<sup>30,</sup><sup>31</sup> <a href="/pubmed/?term=Longstreth%20WT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458729685440" co-class="co-affbox">W. T. Longstreth, Jr</a>,<sup>65,</sup><sup>174</sup> <a href="/pubmed/?term=Rolfs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458741875344" co-class="co-affbox">Arndt Rolfs</a>,<sup>175</sup> <a href="/pubmed/?term=Hata%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458749727856" co-class="co-affbox">Jun Hata</a>,<sup>107</sup> <a href="/pubmed/?term=Woo%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458737593136" co-class="co-affbox">Daniel Woo</a>,<sup>82</sup> <a href="/pubmed/?term=Rosand%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747679312" co-class="co-affbox">Jonathan Rosand</a>,<sup>12,</sup><sup>13,</sup><sup>14</sup> <a href="/pubmed/?term=Pare%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458744143504" co-class="co-affbox">Guillaume Pare</a>,<sup>15</sup> <a href="/pubmed/?term=Hopewell%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458750072240" co-class="co-affbox">Jemma C. Hopewell</a>,<sup>176</sup> <a href="/pubmed/?term=Saleheen%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458731439264" co-class="co-affbox">Danish Saleheen</a>,<sup>177</sup> <a href="/pubmed/?term=Stefansson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458747554144" co-class="co-affbox">Kari Stefansson</a>,<sup>11,</sup><sup>178,</sup><sup>186</sup> <a href="/pubmed/?term=Worrall%20BB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458740973984" co-class="co-affbox">Bradford B. Worrall</a>,<sup>179,</sup><sup>186</sup> <a href="/pubmed/?term=Kittner%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458744708672" co-class="co-affbox">Steven J. Kittner</a>,<sup>37,</sup><sup>186</sup> <a href="/pubmed/?term=Seshadri%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458738181776" co-class="co-affbox">Sudha Seshadri</a>,<sup>48,</sup><sup>180,</sup><sup>186</sup> <a href="/pubmed/?term=Fornage%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458745382640" co-class="co-affbox">Myriam Fornage</a>,<sup>74,</sup><sup>181,</sup><sup>186</sup> <a href="/pubmed/?term=Markus%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458742663376" co-class="co-affbox">Hugh S. Markus</a>,<sup>4,</sup><sup>186</sup> <a href="/pubmed/?term=Howson%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458746934784" co-class="co-affbox">Joanna M. M. Howson</a>,<sup>28,</sup><sup>186</sup> <a href="/pubmed/?term=Kamatani%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458738771168" co-class="co-affbox">Yoichiro Kamatani</a>,<sup>6,</sup><sup>182,</sup><sup>186</sup> <a href="/pubmed/?term=Debette%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458744499600" co-class="co-affbox">Stephanie Debette</a>,<sup>3,</sup><sup>5,</sup><sup>186,</sup><sup>*</sup> and  <a href="/pubmed/?term=Dichgans%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354" class="affpopup" co-rid="_co_idm140458749779504" co-class="co-affbox">Martin Dichgans</a><sup>1,</sup><sup>183,</sup><sup>184,</sup><sup>186,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140458738455280"><h3 class="no_margin">Rainer Malik</h3><p>  <sup>1</sup>Institute for Stroke and Dementia Research (ISD), University Hospital, LMU Munich, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Malik%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Rainer Malik</a></div></div><div id="_co_idm140458735264112"><h3 class="no_margin">Ganesh Chauhan</h3><p>  <sup>2</sup>Centre for Brain Research, Indian Institute of Science, Bangalore, India</p><p>  <sup>3</sup>INSERM U1219 Bordeaux Population Health Research Center, University of Bordeaux, France</p><div>Find articles by <a href="/pubmed/?term=Chauhan%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Ganesh Chauhan</a></div></div><div id="_co_idm140458741814512"><h3 class="no_margin">Matthew Traylor</h3><p>  <sup>4</sup>Stroke Research Group, Division of Clinical Neurosciences, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Traylor%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Matthew Traylor</a></div></div><div id="_co_idm140458749389984"><h3 class="no_margin">Muralidharan Sargurupremraj</h3><p>  <sup>3</sup>INSERM U1219 Bordeaux Population Health Research Center, University of Bordeaux, France</p><p>  <sup>5</sup>Department of Neurology, Institute for Neurodegenerative Disease, Bordeaux University Hospital, Bordeaux, France</p><div>Find articles by <a href="/pubmed/?term=Sargurupremraj%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Muralidharan Sargurupremraj</a></div></div><div id="_co_idm140458742646208"><h3 class="no_margin">Yukinori Okada</h3><p>  <sup>8</sup>Laboratory of Statistical Immunology, Immunology Frontier Research Center (WPI-IFReC), Osaka University, Suita, Japan</p><p>  <sup>67</sup>Faculty of Health and Medicine, University of Newcastle, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Okada%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Yukinori Okada</a></div></div><div id="_co_idm140458741523568"><h3 class="no_margin">Aniket Mishra</h3><p>  <sup>3</sup>INSERM U1219 Bordeaux Population Health Research Center, University of Bordeaux, France</p><p>  <sup>5</sup>Department of Neurology, Institute for Neurodegenerative Disease, Bordeaux University Hospital, Bordeaux, France</p><div>Find articles by <a href="/pubmed/?term=Mishra%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Aniket Mishra</a></div></div><div id="_co_idm140458739856064"><h3 class="no_margin">Loes Rutten-Jacobs</h3><p>  <sup>4</sup>Stroke Research Group, Division of Clinical Neurosciences, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Rutten-Jacobs%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Loes Rutten-Jacobs</a></div></div><div id="_co_idm140458740055648"><h3 class="no_margin">Anne-Katrin Giese</h3><p>  <sup>9</sup>Department of Neurology, Massachusetts General Hospital (MGH), Harvard Medical School, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Giese%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Anne-Katrin Giese</a></div></div><div id="_co_idm140458749339392"><h3 class="no_margin">Sander W. van der Laan</h3><p>  <sup>10</sup>Laboratory of Experimental Cardiology, Department of Cardiology, Division of Heart and Lungs, University Medical Center Utrecht, University of Utrecht, Utrecht, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20der%20Laan%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Sander W. van der Laan</a></div></div><div id="_co_idm140458737294800"><h3 class="no_margin">Solveig Gretarsdottir</h3><p>  <sup>11</sup>deCODE genetics/AMGEN Inc., Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Gretarsdottir%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Solveig Gretarsdottir</a></div></div><div id="_co_idm140458737292816"><h3 class="no_margin">Christopher D. Anderson</h3><p>  <sup>12</sup>Center for Genomic Medicine, MGH, Boston, MA, USA</p><p>  <sup>13</sup>J. Philip Kistler Stroke Research Center, Department of Neurology, MGH, Boston, MA, USA</p><p>  <sup>14</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Anderson%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Christopher D. Anderson</a></div></div><div id="_co_idm140458746352096"><h3 class="no_margin">Michael Chong</h3><p>  <sup>15</sup>Population Health Research Institute, McMaster University, Hamilton, Ontario, Canada</p><div>Find articles by <a href="/pubmed/?term=Chong%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Michael Chong</a></div></div><div id="_co_idm140458739123760"><h3 class="no_margin">Hieab H. H. Adams</h3><p>  <sup>16</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands</p><p>  <sup>17</sup>Department of Radiology and Nuclear Medicine, Erasmus University Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Adams%20HH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Hieab H. H. Adams</a></div></div><div id="_co_idm140458728776128"><h3 class="no_margin">Tetsuro Ago</h3><p>  <sup>18</sup>Department of Medicine and Clinical Science, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Japan</p><div>Find articles by <a href="/pubmed/?term=Ago%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Tetsuro Ago</a></div></div><div id="_co_idm140458747300096"><h3 class="no_margin">Peter Almgren</h3><p>  <sup>19</sup>Department of Clinical Sciences, Lund University, Malmö, Sweden</p><div>Find articles by <a href="/pubmed/?term=Almgren%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Peter Almgren</a></div></div><div id="_co_idm140458747298112"><h3 class="no_margin">Philippe Amouyel</h3><p>  <sup>20</sup>INSERM, Institut Pasteur de Lille, LabEx DISTALZ-UMR1167, Risk Factors and Molecular Determinants of Aging-Related Diseases, Université Lille, Lille, France</p><p>  <sup>21</sup>Centre Hospitalier Universite Lille, Epidemiology and Public Health Department, Lille, France</p><div>Find articles by <a href="/pubmed/?term=Amouyel%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Philippe Amouyel</a></div></div><div id="_co_idm140458742723744"><h3 class="no_margin">Hakan Ay</h3><p>  <sup>13</sup>J. Philip Kistler Stroke Research Center, Department of Neurology, MGH, Boston, MA, USA</p><p>  <sup>22</sup>AA Martinos Center for Biomedical Imaging, Department of Radiology, MGH, Harvard Medical School, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Ay%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Hakan Ay</a></div></div><div id="_co_idm140458779776048"><h3 class="no_margin">Traci M. Bartz</h3><p>  <sup>23</sup>Cardiovascular Health Research Unit, Departments of Biostatistics and Medicine, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Bartz%20TM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Traci M. Bartz</a></div></div><div id="_co_idm140458749408400"><h3 class="no_margin">Oscar R. Benavente</h3><p>  <sup>24</sup>Division of Neurology, Faculty of Medicine, Brain Research Center, University of British Columbia, Vancouver, British Columbia, Canada</p><div>Find articles by <a href="/pubmed/?term=Benavente%20OR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Oscar R. Benavente</a></div></div><div id="_co_idm140458749406416"><h3 class="no_margin">Steve Bevan</h3><p>  <sup>25</sup>School of Life Science, University of Lincoln, Lincoln, UK</p><div>Find articles by <a href="/pubmed/?term=Bevan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Steve Bevan</a></div></div><div id="_co_idm140458783618000"><h3 class="no_margin">Giorgio B. Boncoraglio</h3><p>  <sup>26</sup>Department of Cerebrovascular Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, Milan, Italy</p><div>Find articles by <a href="/pubmed/?term=Boncoraglio%20GB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Giorgio B. Boncoraglio</a></div></div><div id="_co_idm140458744696928"><h3 class="no_margin">Robert D. Brown, Jr</h3><p>  <sup>27</sup>Department of Neurology, Mayo Clinic Rochester, Rochester, MN, USA</p><div>Find articles by <a href="/pubmed/?term=Brown%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Robert D. Brown, Jr</a></div></div><div id="_co_idm140458749452256"><h3 class="no_margin">Adam S. Butterworth</h3><p>  <sup>28</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>29</sup>National Institute for Health Research Blood and Transplant Research Unit in Donor Health and Genomics, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Butterworth%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Adam S. Butterworth</a></div></div><div id="_co_idm140458739181792"><h3 class="no_margin">Caty Carrera</h3><p>  <sup>30</sup>Neurovascular Research Laboratory, Vall d’Hebron Institut of Research, Neurology and Medicine Departments-Universitat Autònoma de Barcelona, Vall d’Hebrón Hospital, Barcelona, Spain</p><p>  <sup>31</sup>Stroke Pharmacogenomics and Genetics, Fundacio Docència i Recerca MutuaTerrassa, Terrassa, Spain</p><div>Find articles by <a href="/pubmed/?term=Carrera%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Caty Carrera</a></div></div><div id="_co_idm140458747313008"><h3 class="no_margin">Cara L. Carty</h3><p>  <sup>32</sup>Children’s Research Institute, Children’s National Medical Center, Washington, DC, USA</p><p>  <sup>33</sup>Center for Translational Science, George Washington University, Washington, DC, USA</p><div>Find articles by <a href="/pubmed/?term=Carty%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Cara L. Carty</a></div></div><div id="_co_idm140458747656864"><h3 class="no_margin">Daniel I. Chasman</h3><p>  <sup>34</sup>Division of Preventive Medicine, Brigham and Women’s Hospital, Boston, MA, USA</p><p>  <sup>35</sup>Harvard Medical School, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Chasman%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Daniel I. Chasman</a></div></div><div id="_co_idm140458742297824"><h3 class="no_margin">Wei-Min Chen</h3><p>  <sup>36</sup>Center for Public Health Genomics, Department of Public Health Sciences, University of Virginia, Charlottesville, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20WM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Wei-Min Chen</a></div></div><div id="_co_idm140458744360768"><h3 class="no_margin">John W. Cole</h3><p>  <sup>37</sup>Department of Neurology, University of Maryland School of Medicine and Baltimore VAMC, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Cole%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">John W. Cole</a></div></div><div id="_co_idm140458741359664"><h3 class="no_margin">Adolfo Correa</h3><p>  <sup>38</sup>Departments of Medicine, Pediatrics and Population Health Science, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Correa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Adolfo Correa</a></div></div><div id="_co_idm140458741745952"><h3 class="no_margin">Ioana Cotlarciuc</h3><p>  <sup>39</sup>Institute of Cardiovascular Research, Royal Holloway University of London, London, UK, and Ashford and St Peters Hospital, Surrey, UK</p><div>Find articles by <a href="/pubmed/?term=Cotlarciuc%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Ioana Cotlarciuc</a></div></div><div id="_co_idm140458737573712"><h3 class="no_margin">Carlos Cruchaga</h3><p>  <sup>40</sup>Department of Psychiatry, Hope Center Program on Protein Aggregation and Neurodegeneration (HPAN), Washington University School of Medicine, St. Louis, MO, USA</p><p>  <sup>41</sup>Department of Developmental Biology, Washington University School of Medicine, St. Louis, MO, USA</p><div>Find articles by <a href="/pubmed/?term=Cruchaga%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Carlos Cruchaga</a></div></div><div id="_co_idm140458748154336"><h3 class="no_margin">John Danesh</h3><p>  <sup>28</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>42</sup>NIHR Blood and Transplant Research Unit in Donor Health and Genomics, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>43</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK</p><p>  <sup>44</sup>British Heart Foundation, Cambridge Centre of Excellence, Department of Medicine, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Danesh%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">John Danesh</a></div></div><div id="_co_idm140458747641488"><h3 class="no_margin">Paul I. W. de Bakker</h3><p>  <sup>45</sup>Department of Medical Genetics, University Medical Center Utrecht, Utrecht, the Netherlands</p><p>  <sup>46</sup>Department of Epidemiology, Julius Center for Health Sciences and Primary Care, University Medical Center Utrecht, Utrecht, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=de%20Bakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Paul I. W. de Bakker</a></div></div><div id="_co_idm140458738108512"><h3 class="no_margin">Anita L. DeStefano</h3><p>  <sup>47</sup>Boston University School of Public Health, Boston, MA, USA</p><p>  <sup>48</sup>Framingham Heart Study, Framingham, MA, USA</p><div>Find articles by <a href="/pubmed/?term=DeStefano%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Anita L. DeStefano</a></div></div><div id="_co_idm140458747614528"><h3 class="no_margin">Marcel den Hoed</h3><p>  <sup>49</sup>Department of Immunology, Genetics and Pathology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden</p><div>Find articles by <a href="/pubmed/?term=den%20Hoed%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Marcel den Hoed</a></div></div><div id="_co_idm140458701464256"><h3 class="no_margin">Qing Duan</h3><p>  <sup>50</sup>Department of Genetics, University of North Carolina, Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Duan%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Qing Duan</a></div></div><div id="_co_idm140458741568720"><h3 class="no_margin">Stefan T. Engelter</h3><p>  <sup>51</sup>Department of Neurology and Stroke Center, Basel University Hospital, Basel, Switzerland</p><p>  <sup>52</sup>Neurorehabilitation Unit, University of Basel and University Center for Medicine of Aging and Rehabilitation Basel, Felix Platter Hospital, Basel, Switzerland</p><div>Find articles by <a href="/pubmed/?term=Engelter%20ST%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Stefan T. Engelter</a></div></div><div id="_co_idm140458741156496"><h3 class="no_margin">Guido J. Falcone</h3><p>  <sup>53</sup>Department of Neurology, Yale University School of Medicine, New Haven, CT, USA</p><p>  <sup>54</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Falcone%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Guido J. Falcone</a></div></div><div id="_co_idm140458742623360"><h3 class="no_margin">Rebecca F. Gottesman</h3><p>  <sup>55</sup>Department of Neurology, Johns Hopkins University School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Gottesman%20RF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Rebecca F. Gottesman</a></div></div><div id="_co_idm140458747957936"><h3 class="no_margin">Raji P. Grewal</h3><p>  <sup>56</sup>Neuroscience Institute, SF Medical Center, Trenton, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Grewal%20RP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Raji P. Grewal</a></div></div><div id="_co_idm140458747955696"><h3 class="no_margin">Vilmundur Gudnason</h3><p>  <sup>57</sup>Icelandic Heart Association Research Institute, Kopavogur, Iceland</p><p>  <sup>58</sup>Faculty of Medicine, University of Iceland, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Gudnason%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Vilmundur Gudnason</a></div></div><div id="_co_idm140458743452688"><h3 class="no_margin">Stefan Gustafsson</h3><p>  <sup>59</sup>Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden</p><div>Find articles by <a href="/pubmed/?term=Gustafsson%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Stefan Gustafsson</a></div></div><div id="_co_idm140458748216112"><h3 class="no_margin">Jeffrey Haessler</h3><p>  <sup>60</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Haessler%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Jeffrey Haessler</a></div></div><div id="_co_idm140458748213872"><h3 class="no_margin">Tamara B. Harris</h3><p>  <sup>61</sup>Laboratory of Epidemiology and Population Science, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Harris%20TB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Tamara B. Harris</a></div></div><div id="_co_idm140458779913744"><h3 class="no_margin">Ahamad Hassan</h3><p>  <sup>62</sup>Department of Neurology, Leeds General Infirmary, Leeds Teaching Hospitals NHS Trust, Leeds, UK</p><div>Find articles by <a href="/pubmed/?term=Hassan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Ahamad Hassan</a></div></div><div id="_co_idm140458732010000"><h3 class="no_margin">Aki S. Havulinna</h3><p>  <sup>63</sup>National Institute for Health and Welfare, Helsinki, Finland</p><p>  <sup>64</sup>FIMM-Institute for Molecular Medicine Finland, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Havulinna%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Aki S. Havulinna</a></div></div><div id="_co_idm140458732113376"><h3 class="no_margin">Susan R. Heckbert</h3><p>  <sup>65</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Heckbert%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Susan R. Heckbert</a></div></div><div id="_co_idm140458732111136"><h3 class="no_margin">Elizabeth G. Holliday</h3><p>  <sup>66</sup>Public Health Stream, Hunter Medical Research Institute, New Lambton, New South Wales, Australia</p><p>  <sup>67</sup>Faculty of Health and Medicine, University of Newcastle, Newcastle, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Holliday%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Elizabeth G. Holliday</a></div></div><div id="_co_idm140458737728272"><h3 class="no_margin">George Howard</h3><p>  <sup>68</sup>School of Public Health, University of Alabama at Birmingham, Birmingham, AL, USA</p><div>Find articles by <a href="/pubmed/?term=Howard%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">George Howard</a></div></div><div id="_co_idm140458731811296"><h3 class="no_margin">Fang-Chi Hsu</h3><p>  <sup>69</sup>Department of Biostatistical Sciences, Wake Forest School of Medicine, Winston-Salem, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Hsu%20FC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Fang-Chi Hsu</a></div></div><div id="_co_idm140458731809200"><h3 class="no_margin">Hyacinth I. Hyacinth</h3><p>  <sup>70</sup>Aflac Cancer and Blood Disorder Center, Department of Pediatrics, Emory University School of Medicine, Atlanta, GA, USA</p><div>Find articles by <a href="/pubmed/?term=Hyacinth%20HI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Hyacinth I. Hyacinth</a></div></div><div id="_co_idm140458740278336"><h3 class="no_margin">M. Arfan Ikram</h3><p>  <sup>16</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Ikram%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">M. Arfan Ikram</a></div></div><div id="_co_idm140458741252512"><h3 class="no_margin">Erik ingelsson</h3><p>  <sup>71</sup>Department of Medicine, Division of Cardiovascular Medicine, Stanford University School of Medicine, Stanford, CA, USA</p><p>  <sup>72</sup>Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden</p><div>Find articles by <a href="/pubmed/?term=ingelsson%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Erik ingelsson</a></div></div><div id="_co_idm140458729539456"><h3 class="no_margin">Marguerite R. Irvin</h3><p>  <sup>73</sup>Epidemiology, School of Public Health, University of Alabama at Birmingham, Birmingham, AL, USA</p><div>Find articles by <a href="/pubmed/?term=Irvin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Marguerite R. Irvin</a></div></div><div id="_co_idm140458747771264"><h3 class="no_margin">Xueqiu Jian</h3><p>  <sup>74</sup>Brown Foundation Institute of Molecular Medicine, University of Texas Health Science Center at Houston, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Jian%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Xueqiu Jian</a></div></div><div id="_co_idm140458745197616"><h3 class="no_margin">Jordi Jimenez-Conde</h3><p>  <sup>75</sup>Neurovascular Research Group (NEUVAS), Neurology Department, Institut Hospital del Mar d’Investigació Mèdica, Universitat Autònoma de Barcelona, Barcelona, Spain</p><div>Find articles by <a href="/pubmed/?term=Jimenez-Conde%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Jordi Jimenez-Conde</a></div></div><div id="_co_idm140458740471088"><h3 class="no_margin">Julie A. Johnson</h3><p>  <sup>76</sup>Department of Pharmacotherapy and Translational Research and Center for Pharmacogenomics, University of Florida, College of Pharmacy, Gainesville, FL, USA</p><p>  <sup>77</sup>Division of Cardiovascular Medicine, College of Medicine, University of Florida, Gainesville, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Johnson%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Julie A. Johnson</a></div></div><div id="_co_idm140458741501552"><h3 class="no_margin">J. Wouter Jukema</h3><p>  <sup>78</sup>Department of Cardiology, Leiden University Medical Center, Leiden, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Jukema%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">J. Wouter Jukema</a></div></div><div id="_co_idm140458741803072"><h3 class="no_margin">Masahiro Kanai</h3><p>  <sup>67</sup>Faculty of Health and Medicine, University of Newcastle, Newcastle, New South Wales, Australia</p><p>  <sup>79</sup>Program in Bioinformatics and Integrative Genomics, Harvard Medical School, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Kanai%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Masahiro Kanai</a></div></div><div id="_co_idm140458740584880"><h3 class="no_margin">Keith L. Keene</h3><p>  <sup>80</sup>Department of Biology, East Carolina University, Greenville, NC, USA</p><p>  <sup>81</sup>Center for Health Disparities, East Carolina University, Greenville, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Keene%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Keith L. Keene</a></div></div><div id="_co_idm140458743486128"><h3 class="no_margin">Brett M. Kissela</h3><p>  <sup>82</sup>University of Cincinnati College of Medicine, Cincinnati, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Kissela%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Brett M. Kissela</a></div></div><div id="_co_idm140458742670912"><h3 class="no_margin">Dawn O. Kleindorfer</h3><p>  <sup>82</sup>University of Cincinnati College of Medicine, Cincinnati, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Kleindorfer%20DO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Dawn O. Kleindorfer</a></div></div><div id="_co_idm140458742668736"><h3 class="no_margin">Charles Kooperberg</h3><p>  <sup>60</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Charles Kooperberg</a></div></div><div id="_co_idm140458747643008"><h3 class="no_margin">Michiaki Kubo</h3><p>  <sup>83</sup>RIKEN Center for Integrative Medical Sciences, Yokohama, Japan</p><div>Find articles by <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Michiaki Kubo</a></div></div><div id="_co_idm140458780215728"><h3 class="no_margin">Leslie A. Lange</h3><p>  <sup>84</sup>Department of Medicine, University of Colorado Denver, Anschutz Medical Campus, Aurora, CO, USA</p><div>Find articles by <a href="/pubmed/?term=Lange%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Leslie A. Lange</a></div></div><div id="_co_idm140458748931728"><h3 class="no_margin">Carl D. Langefeld</h3><p>  <sup>85</sup>Center for Public Health Genomics and Department of Biostatistical Sciences, Wake Forest School of Medicine, Winston-Salem, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Carl D. Langefeld</a></div></div><div id="_co_idm140458740414592"><h3 class="no_margin">Claudia Langenberg</h3><p>  <sup>86</sup>MRC Epidemiology Unit, University of Cambridge School of Clinical Medicine, Institute of Metabolic Science, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Langenberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Claudia Langenberg</a></div></div><div id="_co_idm140458747358320"><h3 class="no_margin">Lenore J. Launer</h3><p>  <sup>87</sup>Intramural Research Program, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Launer%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Lenore J. Launer</a></div></div><div id="_co_idm140458745388928"><h3 class="no_margin">Jin-Moo Lee</h3><p>  <sup>88</sup>Department of Neurology, Radiology, and Biomedical Engineering, Washington University School of Medicine, St. Louis, MO, USA</p><div>Find articles by <a href="/pubmed/?term=Lee%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Jin-Moo Lee</a></div></div><div id="_co_idm140458743113296"><h3 class="no_margin">Robin Lemmens</h3><p>  <sup>89</sup>Department of Neurosciences, Experimental Neurology, KU Leuven-University of Leuven, Leuven, Belgium</p><p>  <sup>90</sup>VIB Center for Brain &amp; Disease Research, University Hospitals Leuven, Department of Neurology, Leuven, Belgium</p><div>Find articles by <a href="/pubmed/?term=Lemmens%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Robin Lemmens</a></div></div><div id="_co_idm140458741497216"><h3 class="no_margin">Didier Leys</h3><p>  <sup>91</sup>INSERM U 1171, CHU Lille, Universite Lille, Lille, France</p><div>Find articles by <a href="/pubmed/?term=Leys%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Didier Leys</a></div></div><div id="_co_idm140458736956640"><h3 class="no_margin">Cathryn M. Lewis</h3><p>  <sup>92</sup>Department of Medical and Molecular Genetics, King’s College London, London, UK</p><p>  <sup>93</sup>SGDP Centre, Institute of Psychiatry, Psychology &amp; Neuroscience, King’s College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Lewis%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Cathryn M. Lewis</a></div></div><div id="_co_idm140458745897280"><h3 class="no_margin">Wei-Yu Lin</h3><p>  <sup>28</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>94</sup>Northern Institute for Cancer Research, Newcastle University, Newcastle, UK</p><div>Find articles by <a href="/pubmed/?term=Lin%20WY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Wei-Yu Lin</a></div></div><div id="_co_idm140458729510352"><h3 class="no_margin">Arne G. Lindgren</h3><p>  <sup>95</sup>Department of Clinical Sciences Lund, Neurology, Lund University, Lund, Sweden</p><p>  <sup>96</sup>Department of Neurology and Rehabilitation Medicine, Skåne University Hospital, Lund, Sweden</p><div>Find articles by <a href="/pubmed/?term=Lindgren%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Arne G. Lindgren</a></div></div><div id="_co_idm140458737800112"><h3 class="no_margin">Erik Lorentzen</h3><p>  <sup>97</sup>Bioinformatics Core Facility, University of Gothenburg, Gothenburg, Sweden</p><div>Find articles by <a href="/pubmed/?term=Lorentzen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Erik Lorentzen</a></div></div><div id="_co_idm140458740169680"><h3 class="no_margin">Patrik K. Magnusson</h3><p>  <sup>98</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Magnusson%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Patrik K. Magnusson</a></div></div><div id="_co_idm140458739882864"><h3 class="no_margin">Jane Maguire</h3><p>  <sup>99</sup>University of Technology Sydney, Faculty of Health, Ultimo, New South Wales, Australia</p><div>Find articles by <a href="/pubmed/?term=Maguire%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Jane Maguire</a></div></div><div id="_co_idm140458739902112"><h3 class="no_margin">Ani Manichaikul</h3><p>  <sup>36</sup>Center for Public Health Genomics, Department of Public Health Sciences, University of Virginia, Charlottesville, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Manichaikul%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Ani Manichaikul</a></div></div><div id="_co_idm140458747662720"><h3 class="no_margin">Patrick F. McArdle</h3><p>  <sup>100</sup>Department of Medicine, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=McArdle%20PF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Patrick F. McArdle</a></div></div><div id="_co_idm140458738207744"><h3 class="no_margin">James F. Meschia</h3><p>  <sup>101</sup>Department of Neurology, Mayo Clinic, Jacksonville, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Meschia%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">James F. Meschia</a></div></div><div id="_co_idm140458732003632"><h3 class="no_margin">Braxton D. Mitchell</h3><p>  <sup>100</sup>Department of Medicine, University of Maryland School of Medicine, Baltimore, MD, USA</p><p>  <sup>102</sup>Geriatrics Research and Education Clinical Center, Baltimore Veterans Administration Medical Center, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Mitchell%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Braxton D. Mitchell</a></div></div><div id="_co_idm140458732610496"><h3 class="no_margin">Thomas H. Mosley</h3><p>  <sup>103</sup>Division of Geriatrics, School of Medicine, University of Mississippi Medical Center, Jackson, MS, USA</p><p>  <sup>104</sup>Memory Impairment and Neurodegenerative Dementia Center, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Mosley%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Thomas H. Mosley</a></div></div><div id="_co_idm140458741475456"><h3 class="no_margin">Michael A. Nalls</h3><p>  <sup>105</sup>Laboratory of Neurogenetics, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</p><p>  <sup>106</sup>Data Tecnica International, Glen Echo, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Michael A. Nalls</a></div></div><div id="_co_idm140458742315168"><h3 class="no_margin">Toshiharu Ninomiya</h3><p>  <sup>107</sup>Department of Epidemiology and Public Health, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Japan</p><div>Find articles by <a href="/pubmed/?term=Ninomiya%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Toshiharu Ninomiya</a></div></div><div id="_co_idm140458740230304"><h3 class="no_margin">Martin J. O’Donnell</h3><p>  <sup>15</sup>Population Health Research Institute, McMaster University, Hamilton, Ontario, Canada</p><p>  <sup>108</sup>Clinical Research Facility, Department of Medicine, NUI Galway, Galway, Ireland</p><div>Find articles by <a href="/pubmed/?term=O%26%23x02019%3BDonnell%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Martin J. O’Donnell</a></div></div><div id="_co_idm140458732133120"><h3 class="no_margin">Bruce M. Psaty</h3><p>  <sup>109</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</p><p>  <sup>110</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</p><p>  <sup>111</sup>Department of Health Services, University of Washington, Seattle, WA, USA</p><p>  <sup>112</sup>Kaiser Permanente Washington Health Research Institute, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Bruce M. Psaty</a></div></div><div id="_co_idm140458742690368"><h3 class="no_margin">Sara L. Pulit</h3><p>  <sup>45</sup>Department of Medical Genetics, University Medical Center Utrecht, Utrecht, the Netherlands</p><p>  <sup>113</sup>Brain Center Rudolf Magnus, Department of Neurology, University Medical Center Utrecht, Utrecht, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Pulit%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Sara L. Pulit</a></div></div><div id="_co_idm140458748538304"><h3 class="no_margin">Kristiina Rannikmäe</h3><p>  <sup>114</sup>Usher Institute of Population Health Sciences and Informatics, University of Edinburgh, Edinburgh, UK</p><p>  <sup>115</sup>Centre for Clinical Brain Sciences, University of Edinburgh, Edinburgh, UK</p><div>Find articles by <a href="/pubmed/?term=Rannikm%26%23x000e4%3Be%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Kristiina Rannikmäe</a></div></div><div id="_co_idm140458748880000"><h3 class="no_margin">Alexander P. Reiner</h3><p>  <sup>65</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</p><p>  <sup>116</sup>Fred Hutchinson Cancer Research Center, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Alexander P. Reiner</a></div></div><div id="_co_idm140458735916864"><h3 class="no_margin">Kathryn M. Rexrode</h3><p>  <sup>117</sup>Department of Medicine, Brigham and Women’s Hospital, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Rexrode%20KM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Kathryn M. Rexrode</a></div></div><div id="_co_idm140458728947920"><h3 class="no_margin">Kenneth Rice</h3><p>  <sup>118</sup>Department of Biostatistics, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Rice%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Kenneth Rice</a></div></div><div id="_co_idm140458738279344"><h3 class="no_margin">Stephen S. Rich</h3><p>  <sup>36</sup>Center for Public Health Genomics, Department of Public Health Sciences, University of Virginia, Charlottesville, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Rich%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Stephen S. Rich</a></div></div><div id="_co_idm140458741672800"><h3 class="no_margin">Paul M. Ridker</h3><p>  <sup>34</sup>Division of Preventive Medicine, Brigham and Women’s Hospital, Boston, MA, USA</p><p>  <sup>35</sup>Harvard Medical School, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Ridker%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Paul M. Ridker</a></div></div><div id="_co_idm140458820577376"><h3 class="no_margin">Natalia S. Rost</h3><p>  <sup>9</sup>Department of Neurology, Massachusetts General Hospital (MGH), Harvard Medical School, Boston, MA, USA</p><p>  <sup>13</sup>J. Philip Kistler Stroke Research Center, Department of Neurology, MGH, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Rost%20NS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Natalia S. Rost</a></div></div><div id="_co_idm140458743739632"><h3 class="no_margin">Peter M. Rothwell</h3><p>  <sup>119</sup>Nuffield Department of Clinical Neurosciences, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Rothwell%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Peter M. Rothwell</a></div></div><div id="_co_idm140458739157680"><h3 class="no_margin">Jerome I. Rotter</h3><p>  <sup>120</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute at Harbor-UCLA Medical Center, Torrance, CA, USA</p><p>  <sup>121</sup>Division of Genomic Outcomes, Department of Pediatrics, Harbor-UCLA Medical Center, Torrance, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Jerome I. Rotter</a></div></div><div id="_co_idm140458748884512"><h3 class="no_margin">Tatjana Rundek</h3><p>  <sup>122</sup>Department of Neurology, Miller School of Medicine, University of Miami, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Rundek%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Tatjana Rundek</a></div></div><div id="_co_idm140458737212944"><h3 class="no_margin">Ralph L. Sacco</h3><p>  <sup>122</sup>Department of Neurology, Miller School of Medicine, University of Miami, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Sacco%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Ralph L. Sacco</a></div></div><div id="_co_idm140458748867360"><h3 class="no_margin">Saori Sakaue</h3><p>  <sup>7</sup>Department of Statistical Genetics, Osaka University Graduate School of Medicine, Osaka, Japan</p><p>  <sup>123</sup>Department of Allergy and Rheumatology, Graduate School of Medicine, University of Tokyo, Tokyo, Japan</p><div>Find articles by <a href="/pubmed/?term=Sakaue%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Saori Sakaue</a></div></div><div id="_co_idm140458747365904"><h3 class="no_margin">Michele M. Sale</h3><p>  <sup>124</sup>Center for Public Health Genomics, University of Virginia, Charlottesville, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Sale%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Michele M. Sale</a></div></div><div id="_co_idm140458738886944"><h3 class="no_margin">Veikko Salomaa</h3><p>  <sup>63</sup>National Institute for Health and Welfare, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Veikko Salomaa</a></div></div><div id="_co_idm140458738884736"><h3 class="no_margin">Bishwa R. Sapkota</h3><p>  <sup>125</sup>Department of Pediatrics, College of Medicine, University of Oklahoma Health Sciences Center, Oklahoma City, OK, USA</p><div>Find articles by <a href="/pubmed/?term=Sapkota%20BR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Bishwa R. Sapkota</a></div></div><div id="_co_idm140458729190896"><h3 class="no_margin">Reinhold Schmidt</h3><p>  <sup>126</sup>Department of Neurology, Medical University of Graz, Graz, Austria</p><div>Find articles by <a href="/pubmed/?term=Schmidt%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Reinhold Schmidt</a></div></div><div id="_co_idm140458729589200"><h3 class="no_margin">Carsten O. Schmidt</h3><p>  <sup>127</sup>Institute for Community Medicine, SHIP-KEF, University Medicine Greifswald, Greifswald, Germany</p><div>Find articles by <a href="/pubmed/?term=Schmidt%20CO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Carsten O. Schmidt</a></div></div><div id="_co_idm140458735244912"><h3 class="no_margin">Ulf Schminke</h3><p>  <sup>128</sup>Department of Neurology, University Medicine Greifswald, Greifswald, Germany</p><div>Find articles by <a href="/pubmed/?term=Schminke%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Ulf Schminke</a></div></div><div id="_co_idm140458784252304"><h3 class="no_margin">Pankaj Sharma</h3><p>  <sup>39</sup>Institute of Cardiovascular Research, Royal Holloway University of London, London, UK, and Ashford and St Peters Hospital, Surrey, UK</p><div>Find articles by <a href="/pubmed/?term=Sharma%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Pankaj Sharma</a></div></div><div id="_co_idm140458742238944"><h3 class="no_margin">Agnieszka Slowik</h3><p>  <sup>129</sup>Department of Neurology, Jagiellonian University, Krakow, Poland</p><div>Find articles by <a href="/pubmed/?term=Slowik%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Agnieszka Slowik</a></div></div><div id="_co_idm140458731287200"><h3 class="no_margin">Cathie L. M. Sudlow</h3><p>  <sup>114</sup>Usher Institute of Population Health Sciences and Informatics, University of Edinburgh, Edinburgh, UK</p><p>  <sup>115</sup>Centre for Clinical Brain Sciences, University of Edinburgh, Edinburgh, UK</p><div>Find articles by <a href="/pubmed/?term=Sudlow%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Cathie L. M. Sudlow</a></div></div><div id="_co_idm140458747775520"><h3 class="no_margin">Christian Tanislav</h3><p>  <sup>130</sup>Department of Neurology, Justus Liebig University, Giessen, Germany</p><div>Find articles by <a href="/pubmed/?term=Tanislav%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Christian Tanislav</a></div></div><div id="_co_idm140458745846560"><h3 class="no_margin">Turgut Tatlisumak</h3><p>  <sup>131</sup>Department of Clinical Neurosciences/Neurology, Institute of Neuroscience and Physiology, Sahlgrenska Academy at University of Gothenburg, Gothenburg, Sweden</p><p>  <sup>132</sup>Sahlgrenska University Hospital, Gothenburg, Sweden</p><div>Find articles by <a href="/pubmed/?term=Tatlisumak%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Turgut Tatlisumak</a></div></div><div id="_co_idm140458748647200"><h3 class="no_margin">Kent D. Taylor</h3><p>  <sup>120</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute at Harbor-UCLA Medical Center, Torrance, CA, USA</p><p>  <sup>121</sup>Division of Genomic Outcomes, Department of Pediatrics, Harbor-UCLA Medical Center, Torrance, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Kent D. Taylor</a></div></div><div id="_co_idm140458741464416"><h3 class="no_margin">Vincent N. S. Thijs</h3><p>  <sup>133</sup>Stroke Division, Florey Institute of Neuroscience and Mental Health, University of Melbourne, Heidelberg, Victoria, Australia</p><p>  <sup>134</sup>Austin Health, Department of Neurology, Heidelberg, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Thijs%20VN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Vincent N. S. Thijs</a></div></div><div id="_co_idm140458743044384"><h3 class="no_margin">Gudmar Thorleifsson</h3><p>  <sup>11</sup>deCODE genetics/AMGEN Inc., Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Thorleifsson%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Gudmar Thorleifsson</a></div></div><div id="_co_idm140458780974512"><h3 class="no_margin">Unnur Thorsteinsdottir</h3><p>  <sup>11</sup>deCODE genetics/AMGEN Inc., Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Thorsteinsdottir%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Unnur Thorsteinsdottir</a></div></div><div id="_co_idm140458737576416"><h3 class="no_margin">Steffen Tiedt</h3><p>  <sup>1</sup>Institute for Stroke and Dementia Research (ISD), University Hospital, LMU Munich, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Tiedt%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Steffen Tiedt</a></div></div><div id="_co_idm140458820574848"><h3 class="no_margin">Stella Trompet</h3><p>  <sup>135</sup>Department of Internal Medicine, Section Gerontology and Geriatrics, Leiden University Medical Center, Leiden, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Trompet%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Stella Trompet</a></div></div><div id="_co_idm140458820572544"><h3 class="no_margin">Christophe Tzourio</h3><p>  <sup>3</sup>INSERM U1219 Bordeaux Population Health Research Center, University of Bordeaux, France</p><p>  <sup>136</sup>INSERM, U1219 Bordeaux, France</p><p>  <sup>137</sup>Department of Public Health, Bordeaux University Hospital, Bordeaux, France</p><div>Find articles by <a href="/pubmed/?term=Tzourio%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Christophe Tzourio</a></div></div><div id="_co_idm140458738214176"><h3 class="no_margin">Cornelia M. van Duijn</h3><p>  <sup>138</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands</p><p>  <sup>139</sup>Center for Medical Systems Biology, Leiden, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Cornelia M. van Duijn</a></div></div><div id="_co_idm140458748208224"><h3 class="no_margin">Matthew Walters</h3><p>  <sup>140</sup>School of Medicine, Dentistry and Nursing at the University of Glasgow, Glasgow, UK</p><div>Find articles by <a href="/pubmed/?term=Walters%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Matthew Walters</a></div></div><div id="_co_idm140458740693424"><h3 class="no_margin">Nicholas J. Wareham</h3><p>  <sup>86</sup>MRC Epidemiology Unit, University of Cambridge School of Clinical Medicine, Institute of Metabolic Science, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Nicholas J. Wareham</a></div></div><div id="_co_idm140458741327248"><h3 class="no_margin">Sylvia Wassertheil-Smoller</h3><p>  <sup>141</sup>Department of Epidemiology and Population Health, Albert Einstein College of Medicine, New York, NY, USA</p><div>Find articles by <a href="/pubmed/?term=Wassertheil-Smoller%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Sylvia Wassertheil-Smoller</a></div></div><div id="_co_idm140458743882416"><h3 class="no_margin">James G. Wilson</h3><p>  <sup>142</sup>Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Wilson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">James G. Wilson</a></div></div><div id="_co_idm140458743880112"><h3 class="no_margin">Kerri L. Wiggins</h3><p>  <sup>109</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Wiggins%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Kerri L. Wiggins</a></div></div><div id="_co_idm140458742082464"><h3 class="no_margin">Qiong Yang</h3><p>  <sup>47</sup>Boston University School of Public Health, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Yang%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Qiong Yang</a></div></div><div id="_co_idm140458744176480"><h3 class="no_margin">Salim Yusuf</h3><p>  <sup>15</sup>Population Health Research Institute, McMaster University, Hamilton, Ontario, Canada</p><div>Find articles by <a href="/pubmed/?term=Yusuf%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Salim Yusuf</a></div></div><div id="_co_idm140458781371024"><h3 class="no_margin">AFGen Consortium</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">AFGen Consortium</a></div></div><div id="_co_idm140458743266864"><h3 class="no_margin">Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium</a></div></div><div id="_co_idm140458743265488"><h3 class="no_margin">International Genomics of Blood Pressure (iGEN-BP) Consortium</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">International Genomics of Blood Pressure (iGEN-BP) Consortium</a></div></div><div id="_co_idm140458732208464"><h3 class="no_margin">INVENT Consortium</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">INVENT Consortium</a></div></div><div id="_co_idm140458732206912"><h3 class="no_margin">STARNET</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">STARNET</a></div></div><div id="_co_idm140458745349664"><h3 class="no_margin">Joshua C. Bis</h3><p>  <sup>109</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Bis%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Joshua C. Bis</a></div></div><div id="_co_idm140458747110384"><h3 class="no_margin">Tomi Pastinen</h3><p>  <sup>144</sup>Department of Human Genetics, McGill University, Montreal, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Pastinen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Tomi Pastinen</a></div></div><div id="_co_idm140458747108144"><h3 class="no_margin">Arno Ruusalepp</h3><p>  <sup>145</sup>Department of Pathophysiology, Institute of Biomedicine and Translation Medicine, University of Tartu, Tartu, Estonia</p><p>  <sup>146</sup>Department of Cardiac Surgery, Tartu University Hospital, Tartu, Estonia</p><p>  <sup>147</sup>Clinical Gene Networks AB, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Ruusalepp%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Arno Ruusalepp</a></div></div><div id="_co_idm140458741506256"><h3 class="no_margin">Eric E. Schadt</h3><p>  <sup>148</sup>Department of Genetics and Genomic Sciences, Icahn Institute for Genomics and Multiscale Biology Icahn School of Medicine at Mount Sinai, New York, NY, USA</p><div>Find articles by <a href="/pubmed/?term=Schadt%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Eric E. Schadt</a></div></div><div id="_co_idm140458743248400"><h3 class="no_margin">Simon Koplev</h3><p>  <sup>148</sup>Department of Genetics and Genomic Sciences, Icahn Institute for Genomics and Multiscale Biology Icahn School of Medicine at Mount Sinai, New York, NY, USA</p><div>Find articles by <a href="/pubmed/?term=Koplev%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Simon Koplev</a></div></div><div id="_co_idm140458735657376"><h3 class="no_margin">Johan L. M. Björkegren</h3><p>  <sup>148</sup>Department of Genetics and Genomic Sciences, Icahn Institute for Genomics and Multiscale Biology Icahn School of Medicine at Mount Sinai, New York, NY, USA</p><p>  <sup>149</sup>Department of Pathophysiology, Institute of Biomedicine and Translation Medicine, Biomeedikum, University of Tartu, Tartu, Estonia</p><p>  <sup>150</sup>Integrated Cardio Metabolic Centre, Department of Medicine, Karolinska Institutet, Karolinska Universitetssjukhuset, Huddinge, Sweden</p><p>  <sup>151</sup>Clinical Gene Networks AB, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Bj%26%23x000f6%3Brkegren%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Johan L. M. Björkegren</a></div></div><div id="_co_idm140458741137072"><h3 class="no_margin">Veronica Codoni</h3><p>  <sup>152</sup>UPMC Univ. Paris 06, INSERM, UMR_S 1166, Team Genomics &amp; Pathophysiology of Cardiovascular Diseases, Sorbonne Universités, Paris, France</p><p>  <sup>153</sup>I CAN Institute for Cardiometabolism and Nutrition, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Codoni%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Veronica Codoni</a></div></div><div id="_co_idm140458749683872"><h3 class="no_margin">Mete Civelek</h3><p>  <sup>124</sup>Center for Public Health Genomics, University of Virginia, Charlottesville, VA, USA</p><p>  <sup>154</sup>Department of Biomedical Engineering, University of Virginia, Charlottesville, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Civelek%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Mete Civelek</a></div></div><div id="_co_idm140458740612208"><h3 class="no_margin">Nicholas L. Smith</h3><p>  <sup>65</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</p><p>  <sup>155</sup>Group Health Research Institute, Group Health Cooperative, Seattle, WA, USA</p><p>  <sup>156</sup>Seattle Epidemiologic Research and Information Center, VA Office of Research and Development, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Smith%20NL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Nicholas L. Smith</a></div></div><div id="_co_idm140458744378192"><h3 class="no_margin">David A. Tregouet</h3><p>  <sup>152</sup>UPMC Univ. Paris 06, INSERM, UMR_S 1166, Team Genomics &amp; Pathophysiology of Cardiovascular Diseases, Sorbonne Universités, Paris, France</p><p>  <sup>153</sup>I CAN Institute for Cardiometabolism and Nutrition, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Tregouet%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">David A. Tregouet</a></div></div><div id="_co_idm140458746446368"><h3 class="no_margin">Ingrid E. Christophersen</h3><p>  <sup>54</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA, USA</p><p>  <sup>157</sup>Cardiovascular Research Center, MGH, Boston, MA, USA</p><p>  <sup>158</sup>Department of Medical Research, Bærum Hospital, Vestre Viken Hospital Trust, Gjettum, Norway</p><div>Find articles by <a href="/pubmed/?term=Christophersen%20IE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Ingrid E. Christophersen</a></div></div><div id="_co_idm140458743150096"><h3 class="no_margin">Carolina Roselli</h3><p>  <sup>54</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Roselli%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Carolina Roselli</a></div></div><div id="_co_idm140458729070960"><h3 class="no_margin">Steven A. Lubitz</h3><p>  <sup>54</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA, USA</p><p>  <sup>157</sup>Cardiovascular Research Center, MGH, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Lubitz%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Steven A. Lubitz</a></div></div><div id="_co_idm140458746004752"><h3 class="no_margin">Patrick T. Ellinor</h3><p>  <sup>54</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA, USA</p><p>  <sup>157</sup>Cardiovascular Research Center, MGH, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Ellinor%20PT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Patrick T. Ellinor</a></div></div><div id="_co_idm140458741677792"><h3 class="no_margin">E. Shyong Tai</h3><p>  <sup>159</sup>Saw Swee Hock School of Public Health, National University of Singapore and National University Health System, Singapore, Singapore</p><div>Find articles by <a href="/pubmed/?term=Tai%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">E. Shyong Tai</a></div></div><div id="_co_idm140458742731376"><h3 class="no_margin">Jaspal S. Kooner</h3><p>  <sup>160</sup>National Heart and Lung Institute, Imperial College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Kooner%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Jaspal S. Kooner</a></div></div><div id="_co_idm140458742729104"><h3 class="no_margin">Norihiro Kato</h3><p>  <sup>161</sup>Department of Gene Diagnostics and Therapeutics, Research Institute, National Center for Global Health and Medicine, Tokyo, Japan</p><div>Find articles by <a href="/pubmed/?term=Kato%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Norihiro Kato</a></div></div><div id="_co_idm140458743334208"><h3 class="no_margin">Jiang He</h3><p>  <sup>162</sup>Department of Epidemiology, Tulane University School of Public Health and Tropical Medicine, New Orleans, LA, USA</p><div>Find articles by <a href="/pubmed/?term=He%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Jiang He</a></div></div><div id="_co_idm140458739386448"><h3 class="no_margin">Pim van der Harst</h3><p>  <sup>163</sup>Department of Cardiology, University Medical Center Groningen, University of Groningen, Groningen, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20der%20Harst%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Pim van der Harst</a></div></div><div id="_co_idm140458743425424"><h3 class="no_margin">Paul Elliott</h3><p>  <sup>164</sup>MRC-PHE Centre for Environment and Health, School of Public Health, Department of Epidemiology and Biostatistics and the NIHR Imperial Biomedical Research Centre, Imperial College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Elliott%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Paul Elliott</a></div></div><div id="_co_idm140458747826064"><h3 class="no_margin">John C. Chambers</h3><p>  <sup>165</sup>Department of Epidemiology and Biostatistics, Imperial College London, London, UK</p><p>  <sup>166</sup>Department of Cardiology, Ealing Hospital NHS Trust, Southall, UK</p><div>Find articles by <a href="/pubmed/?term=Chambers%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">John C. Chambers</a></div></div><div id="_co_idm140458780946864"><h3 class="no_margin">Fumihiko Takeuchi</h3><p>  <sup>161</sup>Department of Gene Diagnostics and Therapeutics, Research Institute, National Center for Global Health and Medicine, Tokyo, Japan</p><div>Find articles by <a href="/pubmed/?term=Takeuchi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Fumihiko Takeuchi</a></div></div><div id="_co_idm140458732090240"><h3 class="no_margin">Andrew D. Johnson</h3><p>  <sup>48</sup>Framingham Heart Study, Framingham, MA, USA</p><p>  <sup>167</sup>National Heart, Lung and Blood Research Institute, Division of Intramural Research, Population Sciences Branch, Framingham, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Johnson%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Andrew D. Johnson</a></div></div><div id="_co_idm140458731790224"><h3 class="no_margin">BioBank Japan Cooperative Hospital Group</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">BioBank Japan Cooperative Hospital Group</a></div></div><div id="_co_idm140458731788736"><h3 class="no_margin">COMPASS Consortium</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">COMPASS Consortium</a></div></div><div id="_co_idm140458742275760"><h3 class="no_margin">EPiC-CVD Consortium</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">EPiC-CVD Consortium</a></div></div><div id="_co_idm140458742274240"><h3 class="no_margin">EPiC-interAct Consortium</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">EPiC-interAct Consortium</a></div></div><div id="_co_idm140458732057936"><h3 class="no_margin">International Stroke Genetics Consortium (ISGC)</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">International Stroke Genetics Consortium (ISGC)</a></div></div><div id="_co_idm140458732056448"><h3 class="no_margin">METASTROKE Consortium</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">METASTROKE Consortium</a></div></div><div id="_co_idm140458748181536"><h3 class="no_margin">Neurology Working Group of the CHARGE Consortium</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Neurology Working Group of the CHARGE Consortium</a></div></div><div id="_co_idm140458748180048"><h3 class="no_margin">NiNDS Stroke Genetics Network (SiGN)</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">NiNDS Stroke Genetics Network (SiGN)</a></div></div><div id="_co_idm140458731926176"><h3 class="no_margin">UK Young Lacunar DNA Study</h3><p>  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">UK Young Lacunar DNA Study</a></div></div><div id="_co_idm140458741209984"><h3 class="no_margin">Dharambir K. Sanghera</h3><p>  <sup>125</sup>Department of Pediatrics, College of Medicine, University of Oklahoma Health Sciences Center, Oklahoma City, OK, USA</p><p>  <sup>169</sup>Department of Pharmaceutical Sciences, College of Pharmacy, University of Oklahoma Health Sciences Center, Oklahoma City, OK, USA</p><p>  <sup>170</sup>Oklahoma Center for Neuroscience, Oklahoma City, OK, USA</p><div>Find articles by <a href="/pubmed/?term=Sanghera%20DK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Dharambir K. Sanghera</a></div></div><div id="_co_idm140458748247760"><h3 class="no_margin">Olle Melander</h3><p>  <sup>19</sup>Department of Clinical Sciences, Lund University, Malmö, Sweden</p><div>Find articles by <a href="/pubmed/?term=Melander%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Olle Melander</a></div></div><div id="_co_idm140458749241200"><h3 class="no_margin">Christina Jern</h3><p>  <sup>171</sup>Department of Pathology and Genetics, Institute of Biomedicine, Sahlgrenska Academy at University of Gothenburg, Gothenburg, Sweden</p><div>Find articles by <a href="/pubmed/?term=Jern%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Christina Jern</a></div></div><div id="_co_idm140458737139008"><h3 class="no_margin">Daniel Strbian</h3><p>  <sup>172</sup>Department of Neurology, Helsinki University Hospital, Helsinki, Finland</p><p>  <sup>173</sup>Clinical Neurosciences, Neurology, University of Helsinki, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Strbian%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Daniel Strbian</a></div></div><div id="_co_idm140458740479696"><h3 class="no_margin">Israel Fernandez-Cadenas</h3><p>  <sup>30</sup>Neurovascular Research Laboratory, Vall d’Hebron Institut of Research, Neurology and Medicine Departments-Universitat Autònoma de Barcelona, Vall d’Hebrón Hospital, Barcelona, Spain</p><p>  <sup>31</sup>Stroke Pharmacogenomics and Genetics, Fundacio Docència i Recerca MutuaTerrassa, Terrassa, Spain</p><div>Find articles by <a href="/pubmed/?term=Fernandez-Cadenas%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Israel Fernandez-Cadenas</a></div></div><div id="_co_idm140458729685440"><h3 class="no_margin">W. T. Longstreth, Jr</h3><p>  <sup>65</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</p><p>  <sup>174</sup>Department of Neurology, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Longstreth%20WT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">W. T. Longstreth, Jr</a></div></div><div id="_co_idm140458741875344"><h3 class="no_margin">Arndt Rolfs</h3><p>  <sup>175</sup>Albrecht Kossel Institute, University Clinic of Rostock, Rostock, Germany</p><div>Find articles by <a href="/pubmed/?term=Rolfs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Arndt Rolfs</a></div></div><div id="_co_idm140458749727856"><h3 class="no_margin">Jun Hata</h3><p>  <sup>107</sup>Department of Epidemiology and Public Health, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Japan</p><div>Find articles by <a href="/pubmed/?term=Hata%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Jun Hata</a></div></div><div id="_co_idm140458737593136"><h3 class="no_margin">Daniel Woo</h3><p>  <sup>82</sup>University of Cincinnati College of Medicine, Cincinnati, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Woo%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Daniel Woo</a></div></div><div id="_co_idm140458747679312"><h3 class="no_margin">Jonathan Rosand</h3><p>  <sup>12</sup>Center for Genomic Medicine, MGH, Boston, MA, USA</p><p>  <sup>13</sup>J. Philip Kistler Stroke Research Center, Department of Neurology, MGH, Boston, MA, USA</p><p>  <sup>14</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Rosand%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Jonathan Rosand</a></div></div><div id="_co_idm140458744143504"><h3 class="no_margin">Guillaume Pare</h3><p>  <sup>15</sup>Population Health Research Institute, McMaster University, Hamilton, Ontario, Canada</p><div>Find articles by <a href="/pubmed/?term=Pare%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Guillaume Pare</a></div></div><div id="_co_idm140458750072240"><h3 class="no_margin">Jemma C. Hopewell</h3><p>  <sup>176</sup>Clinical Trial Service Unit and Epidemiological Studies Unit, Nuffield Department of Population Health, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Hopewell%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Jemma C. Hopewell</a></div></div><div id="_co_idm140458731439264"><h3 class="no_margin">Danish Saleheen</h3><p>  <sup>177</sup>Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Saleheen%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Danish Saleheen</a></div></div><div id="_co_idm140458747554144"><h3 class="no_margin">Kari Stefansson</h3><p>  <sup>11</sup>deCODE genetics/AMGEN Inc., Reykjavik, Iceland</p><p>  <sup>178</sup>Faculty of Medicine, University of Iceland, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Stefansson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Kari Stefansson</a></div></div><div id="_co_idm140458740973984"><h3 class="no_margin">Bradford B. Worrall</h3><p>  <sup>179</sup>Departments of Neurology and Public Health Sciences, University of Virginia School of Medicine, Charlottesville, VA, USA</p><div>Find articles by <a href="/pubmed/?term=Worrall%20BB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Bradford B. Worrall</a></div></div><div id="_co_idm140458744708672"><h3 class="no_margin">Steven J. Kittner</h3><p>  <sup>37</sup>Department of Neurology, University of Maryland School of Medicine and Baltimore VAMC, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Kittner%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Steven J. Kittner</a></div></div><div id="_co_idm140458738181776"><h3 class="no_margin">Sudha Seshadri</h3><p>  <sup>48</sup>Framingham Heart Study, Framingham, MA, USA</p><p>  <sup>180</sup>Glenn Biggs Institute for Alzheimer’s and Neurodegenerative Diseases, University of Texas Health Sciences Center, San Antonio, San Antonio, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Seshadri%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Sudha Seshadri</a></div></div><div id="_co_idm140458745382640"><h3 class="no_margin">Myriam Fornage</h3><p>  <sup>74</sup>Brown Foundation Institute of Molecular Medicine, University of Texas Health Science Center at Houston, Houston, TX, USA</p><p>  <sup>181</sup>Human Genetics Center, University of Texas Health Science Center at Houston, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Fornage%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Myriam Fornage</a></div></div><div id="_co_idm140458742663376"><h3 class="no_margin">Hugh S. Markus</h3><p>  <sup>4</sup>Stroke Research Group, Division of Clinical Neurosciences, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Markus%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Hugh S. Markus</a></div></div><div id="_co_idm140458746934784"><h3 class="no_margin">Joanna M. M. Howson</h3><p>  <sup>28</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Howson%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Joanna M. M. Howson</a></div></div><div id="_co_idm140458738771168"><h3 class="no_margin">Yoichiro Kamatani</h3><p>  <sup>6</sup>Laboratory for Statistical Analysis, RIKEN Center for Integrative Medical Sciences, Yokohama, Japan</p><p>  <sup>182</sup>Center for Genomic Medicine, Kyoto University Graduate School of Medicine, Kyoto, Japan</p><div>Find articles by <a href="/pubmed/?term=Kamatani%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Yoichiro Kamatani</a></div></div><div id="_co_idm140458744499600"><h3 class="no_margin">Stephanie Debette</h3><p>  <sup>3</sup>INSERM U1219 Bordeaux Population Health Research Center, University of Bordeaux, France</p><p>  <sup>5</sup>Department of Neurology, Institute for Neurodegenerative Disease, Bordeaux University Hospital, Bordeaux, France</p><div>Find articles by <a href="/pubmed/?term=Debette%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Stephanie Debette</a></div></div><div id="_co_idm140458749779504"><h3 class="no_margin">Martin Dichgans</h3><p>  <sup>1</sup>Institute for Stroke and Dementia Research (ISD), University Hospital, LMU Munich, Munich, Germany</p><p>  <sup>183</sup>Munich Cluster for Systems Neurology (SyNergy), Munich, Germany</p><p>  <sup>184</sup>German Center for Neurodegenerative Diseases (DZNE), Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Dichgans%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29531354">Martin Dichgans</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140458748872832_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140458748872832_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140458748872832_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">  <sup>1</sup>Institute for Stroke and Dementia Research (ISD), University Hospital, LMU Munich, Munich, Germany</div><div class="fm-affl" id="A2" lang="en">  <sup>2</sup>Centre for Brain Research, Indian Institute of Science, Bangalore, India</div><div class="fm-affl" id="A3" lang="en">  <sup>3</sup>INSERM U1219 Bordeaux Population Health Research Center, University of Bordeaux, France</div><div class="fm-affl" id="A4" lang="en">  <sup>4</sup>Stroke Research Group, Division of Clinical Neurosciences, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A5" lang="en">  <sup>5</sup>Department of Neurology, Institute for Neurodegenerative Disease, Bordeaux University Hospital, Bordeaux, France</div><div class="fm-affl" id="A6" lang="en">  <sup>6</sup>Laboratory for Statistical Analysis, RIKEN Center for Integrative Medical Sciences, Yokohama, Japan</div><div class="fm-affl" id="A7" lang="en">  <sup>7</sup>Department of Statistical Genetics, Osaka University Graduate School of Medicine, Osaka, Japan</div><div class="fm-affl" id="A8" lang="en">  <sup>8</sup>Laboratory of Statistical Immunology, Immunology Frontier Research Center (WPI-IFReC), Osaka University, Suita, Japan</div><div class="fm-affl" id="A9" lang="en">  <sup>9</sup>Department of Neurology, Massachusetts General Hospital (MGH), Harvard Medical School, Boston, MA, USA</div><div class="fm-affl" id="A10" lang="en">  <sup>10</sup>Laboratory of Experimental Cardiology, Department of Cardiology, Division of Heart and Lungs, University Medical Center Utrecht, University of Utrecht, Utrecht, the Netherlands</div><div class="fm-affl" id="A11" lang="en">  <sup>11</sup>deCODE genetics/AMGEN Inc., Reykjavik, Iceland</div><div class="fm-affl" id="A12" lang="en">  <sup>12</sup>Center for Genomic Medicine, MGH, Boston, MA, USA</div><div class="fm-affl" id="A13" lang="en">  <sup>13</sup>J. Philip Kistler Stroke Research Center, Department of Neurology, MGH, Boston, MA, USA</div><div class="fm-affl" id="A14" lang="en">  <sup>14</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, MA, USA</div><div class="fm-affl" id="A15" lang="en">  <sup>15</sup>Population Health Research Institute, McMaster University, Hamilton, Ontario, Canada</div><div class="fm-affl" id="A16" lang="en">  <sup>16</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands</div><div class="fm-affl" id="A17" lang="en">  <sup>17</sup>Department of Radiology and Nuclear Medicine, Erasmus University Medical Center, Rotterdam, the Netherlands</div><div class="fm-affl" id="A18" lang="en">  <sup>18</sup>Department of Medicine and Clinical Science, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Japan</div><div class="fm-affl" id="A19" lang="en">  <sup>19</sup>Department of Clinical Sciences, Lund University, Malmö, Sweden</div><div class="fm-affl" id="A20" lang="en">  <sup>20</sup>INSERM, Institut Pasteur de Lille, LabEx DISTALZ-UMR1167, Risk Factors and Molecular Determinants of Aging-Related Diseases, Université Lille, Lille, France</div><div class="fm-affl" id="A21" lang="en">  <sup>21</sup>Centre Hospitalier Universite Lille, Epidemiology and Public Health Department, Lille, France</div><div class="fm-affl" id="A22" lang="en">  <sup>22</sup>AA Martinos Center for Biomedical Imaging, Department of Radiology, MGH, Harvard Medical School, Boston, MA, USA</div><div class="fm-affl" id="A23" lang="en">  <sup>23</sup>Cardiovascular Health Research Unit, Departments of Biostatistics and Medicine, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="A24" lang="en">  <sup>24</sup>Division of Neurology, Faculty of Medicine, Brain Research Center, University of British Columbia, Vancouver, British Columbia, Canada</div><div class="fm-affl" id="A25" lang="en">  <sup>25</sup>School of Life Science, University of Lincoln, Lincoln, UK</div><div class="fm-affl" id="A26" lang="en">  <sup>26</sup>Department of Cerebrovascular Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, Milan, Italy</div><div class="fm-affl" id="A27" lang="en">  <sup>27</sup>Department of Neurology, Mayo Clinic Rochester, Rochester, MN, USA</div><div class="fm-affl" id="A28" lang="en">  <sup>28</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A29" lang="en">  <sup>29</sup>National Institute for Health Research Blood and Transplant Research Unit in Donor Health and Genomics, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A30" lang="en">  <sup>30</sup>Neurovascular Research Laboratory, Vall d’Hebron Institut of Research, Neurology and Medicine Departments-Universitat Autònoma de Barcelona, Vall d’Hebrón Hospital, Barcelona, Spain</div><div class="fm-affl" id="A31" lang="en">  <sup>31</sup>Stroke Pharmacogenomics and Genetics, Fundacio Docència i Recerca MutuaTerrassa, Terrassa, Spain</div><div class="fm-affl" id="A32" lang="en">  <sup>32</sup>Children’s Research Institute, Children’s National Medical Center, Washington, DC, USA</div><div class="fm-affl" id="A33" lang="en">  <sup>33</sup>Center for Translational Science, George Washington University, Washington, DC, USA</div><div class="fm-affl" id="A34" lang="en">  <sup>34</sup>Division of Preventive Medicine, Brigham and Women’s Hospital, Boston, MA, USA</div><div class="fm-affl" id="A35" lang="en">  <sup>35</sup>Harvard Medical School, Boston, MA, USA</div><div class="fm-affl" id="A36" lang="en">  <sup>36</sup>Center for Public Health Genomics, Department of Public Health Sciences, University of Virginia, Charlottesville, VA, USA</div><div class="fm-affl" id="A37" lang="en">  <sup>37</sup>Department of Neurology, University of Maryland School of Medicine and Baltimore VAMC, Baltimore, MD, USA</div><div class="fm-affl" id="A38" lang="en">  <sup>38</sup>Departments of Medicine, Pediatrics and Population Health Science, University of Mississippi Medical Center, Jackson, MS, USA</div><div class="fm-affl" id="A39" lang="en">  <sup>39</sup>Institute of Cardiovascular Research, Royal Holloway University of London, London, UK, and Ashford and St Peters Hospital, Surrey, UK</div><div class="fm-affl" id="A40" lang="en">  <sup>40</sup>Department of Psychiatry, Hope Center Program on Protein Aggregation and Neurodegeneration (HPAN), Washington University School of Medicine, St. Louis, MO, USA</div><div class="fm-affl" id="A41" lang="en">  <sup>41</sup>Department of Developmental Biology, Washington University School of Medicine, St. Louis, MO, USA</div><div class="fm-affl" id="A42" lang="en">  <sup>42</sup>NIHR Blood and Transplant Research Unit in Donor Health and Genomics, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A43" lang="en">  <sup>43</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK</div><div class="fm-affl" id="A44" lang="en">  <sup>44</sup>British Heart Foundation, Cambridge Centre of Excellence, Department of Medicine, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A45" lang="en">  <sup>45</sup>Department of Medical Genetics, University Medical Center Utrecht, Utrecht, the Netherlands</div><div class="fm-affl" id="A46" lang="en">  <sup>46</sup>Department of Epidemiology, Julius Center for Health Sciences and Primary Care, University Medical Center Utrecht, Utrecht, the Netherlands</div><div class="fm-affl" id="A47" lang="en">  <sup>47</sup>Boston University School of Public Health, Boston, MA, USA</div><div class="fm-affl" id="A48" lang="en">  <sup>48</sup>Framingham Heart Study, Framingham, MA, USA</div><div class="fm-affl" id="A49" lang="en">  <sup>49</sup>Department of Immunology, Genetics and Pathology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden</div><div class="fm-affl" id="A50" lang="en">  <sup>50</sup>Department of Genetics, University of North Carolina, Chapel Hill, NC, USA</div><div class="fm-affl" id="A51" lang="en">  <sup>51</sup>Department of Neurology and Stroke Center, Basel University Hospital, Basel, Switzerland</div><div class="fm-affl" id="A52" lang="en">  <sup>52</sup>Neurorehabilitation Unit, University of Basel and University Center for Medicine of Aging and Rehabilitation Basel, Felix Platter Hospital, Basel, Switzerland</div><div class="fm-affl" id="A53" lang="en">  <sup>53</sup>Department of Neurology, Yale University School of Medicine, New Haven, CT, USA</div><div class="fm-affl" id="A54" lang="en">  <sup>54</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA, USA</div><div class="fm-affl" id="A55" lang="en">  <sup>55</sup>Department of Neurology, Johns Hopkins University School of Medicine, Baltimore, MD, USA</div><div class="fm-affl" id="A56" lang="en">  <sup>56</sup>Neuroscience Institute, SF Medical Center, Trenton, NJ, USA</div><div class="fm-affl" id="A57" lang="en">  <sup>57</sup>Icelandic Heart Association Research Institute, Kopavogur, Iceland</div><div class="fm-affl" id="A58" lang="en">  <sup>58</sup>Faculty of Medicine, University of Iceland, Reykjavik, Iceland</div><div class="fm-affl" id="A59" lang="en">  <sup>59</sup>Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden</div><div class="fm-affl" id="A60" lang="en">  <sup>60</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</div><div class="fm-affl" id="A61" lang="en">  <sup>61</sup>Laboratory of Epidemiology and Population Science, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</div><div class="fm-affl" id="A62" lang="en">  <sup>62</sup>Department of Neurology, Leeds General Infirmary, Leeds Teaching Hospitals NHS Trust, Leeds, UK</div><div class="fm-affl" id="A63" lang="en">  <sup>63</sup>National Institute for Health and Welfare, Helsinki, Finland</div><div class="fm-affl" id="A64" lang="en">  <sup>64</sup>FIMM-Institute for Molecular Medicine Finland, Helsinki, Finland</div><div class="fm-affl" id="A65" lang="en">  <sup>65</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="A66" lang="en">  <sup>66</sup>Public Health Stream, Hunter Medical Research Institute, New Lambton, New South Wales, Australia</div><div class="fm-affl" id="A67" lang="en">  <sup>67</sup>Faculty of Health and Medicine, University of Newcastle, Newcastle, New South Wales, Australia</div><div class="fm-affl" id="A68" lang="en">  <sup>68</sup>School of Public Health, University of Alabama at Birmingham, Birmingham, AL, USA</div><div class="fm-affl" id="A69" lang="en">  <sup>69</sup>Department of Biostatistical Sciences, Wake Forest School of Medicine, Winston-Salem, NC, USA</div><div class="fm-affl" id="A70" lang="en">  <sup>70</sup>Aflac Cancer and Blood Disorder Center, Department of Pediatrics, Emory University School of Medicine, Atlanta, GA, USA</div><div class="fm-affl" id="A71" lang="en">  <sup>71</sup>Department of Medicine, Division of Cardiovascular Medicine, Stanford University School of Medicine, Stanford, CA, USA</div><div class="fm-affl" id="A72" lang="en">  <sup>72</sup>Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden</div><div class="fm-affl" id="A73" lang="en">  <sup>73</sup>Epidemiology, School of Public Health, University of Alabama at Birmingham, Birmingham, AL, USA</div><div class="fm-affl" id="A74" lang="en">  <sup>74</sup>Brown Foundation Institute of Molecular Medicine, University of Texas Health Science Center at Houston, Houston, TX, USA</div><div class="fm-affl" id="A75" lang="en">  <sup>75</sup>Neurovascular Research Group (NEUVAS), Neurology Department, Institut Hospital del Mar d’Investigació Mèdica, Universitat Autònoma de Barcelona, Barcelona, Spain</div><div class="fm-affl" id="A76" lang="en">  <sup>76</sup>Department of Pharmacotherapy and Translational Research and Center for Pharmacogenomics, University of Florida, College of Pharmacy, Gainesville, FL, USA</div><div class="fm-affl" id="A77" lang="en">  <sup>77</sup>Division of Cardiovascular Medicine, College of Medicine, University of Florida, Gainesville, FL, USA</div><div class="fm-affl" id="A78" lang="en">  <sup>78</sup>Department of Cardiology, Leiden University Medical Center, Leiden, the Netherlands</div><div class="fm-affl" id="A79" lang="en">  <sup>79</sup>Program in Bioinformatics and Integrative Genomics, Harvard Medical School, Boston, MA, USA</div><div class="fm-affl" id="A80" lang="en">  <sup>80</sup>Department of Biology, East Carolina University, Greenville, NC, USA</div><div class="fm-affl" id="A81" lang="en">  <sup>81</sup>Center for Health Disparities, East Carolina University, Greenville, NC, USA</div><div class="fm-affl" id="A82" lang="en">  <sup>82</sup>University of Cincinnati College of Medicine, Cincinnati, OH, USA</div><div class="fm-affl" id="A83" lang="en">  <sup>83</sup>RIKEN Center for Integrative Medical Sciences, Yokohama, Japan</div><div class="fm-affl" id="A84" lang="en">  <sup>84</sup>Department of Medicine, University of Colorado Denver, Anschutz Medical Campus, Aurora, CO, USA</div><div class="fm-affl" id="A85" lang="en">  <sup>85</sup>Center for Public Health Genomics and Department of Biostatistical Sciences, Wake Forest School of Medicine, Winston-Salem, NC, USA</div><div class="fm-affl" id="A86" lang="en">  <sup>86</sup>MRC Epidemiology Unit, University of Cambridge School of Clinical Medicine, Institute of Metabolic Science, Cambridge, UK</div><div class="fm-affl" id="A87" lang="en">  <sup>87</sup>Intramural Research Program, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</div><div class="fm-affl" id="A88" lang="en">  <sup>88</sup>Department of Neurology, Radiology, and Biomedical Engineering, Washington University School of Medicine, St. Louis, MO, USA</div><div class="fm-affl" id="A89" lang="en">  <sup>89</sup>Department of Neurosciences, Experimental Neurology, KU Leuven-University of Leuven, Leuven, Belgium</div><div class="fm-affl" id="A90" lang="en">  <sup>90</sup>VIB Center for Brain &amp; Disease Research, University Hospitals Leuven, Department of Neurology, Leuven, Belgium</div><div class="fm-affl" id="A91" lang="en">  <sup>91</sup>INSERM U 1171, CHU Lille, Universite Lille, Lille, France</div><div class="fm-affl" id="A92" lang="en">  <sup>92</sup>Department of Medical and Molecular Genetics, King’s College London, London, UK</div><div class="fm-affl" id="A93" lang="en">  <sup>93</sup>SGDP Centre, Institute of Psychiatry, Psychology &amp; Neuroscience, King’s College London, London, UK</div><div class="fm-affl" id="A94" lang="en">  <sup>94</sup>Northern Institute for Cancer Research, Newcastle University, Newcastle, UK</div><div class="fm-affl" id="A95" lang="en">  <sup>95</sup>Department of Clinical Sciences Lund, Neurology, Lund University, Lund, Sweden</div><div class="fm-affl" id="A96" lang="en">  <sup>96</sup>Department of Neurology and Rehabilitation Medicine, Skåne University Hospital, Lund, Sweden</div><div class="fm-affl" id="A97" lang="en">  <sup>97</sup>Bioinformatics Core Facility, University of Gothenburg, Gothenburg, Sweden</div><div class="fm-affl" id="A98" lang="en">  <sup>98</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden</div><div class="fm-affl" id="A99" lang="en">  <sup>99</sup>University of Technology Sydney, Faculty of Health, Ultimo, New South Wales, Australia</div><div class="fm-affl" id="A100" lang="en">  <sup>100</sup>Department of Medicine, University of Maryland School of Medicine, Baltimore, MD, USA</div><div class="fm-affl" id="A101" lang="en">  <sup>101</sup>Department of Neurology, Mayo Clinic, Jacksonville, FL, USA</div><div class="fm-affl" id="A102" lang="en">  <sup>102</sup>Geriatrics Research and Education Clinical Center, Baltimore Veterans Administration Medical Center, Baltimore, MD, USA</div><div class="fm-affl" id="A103" lang="en">  <sup>103</sup>Division of Geriatrics, School of Medicine, University of Mississippi Medical Center, Jackson, MS, USA</div><div class="fm-affl" id="A104" lang="en">  <sup>104</sup>Memory Impairment and Neurodegenerative Dementia Center, University of Mississippi Medical Center, Jackson, MS, USA</div><div class="fm-affl" id="A105" lang="en">  <sup>105</sup>Laboratory of Neurogenetics, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</div><div class="fm-affl" id="A106" lang="en">  <sup>106</sup>Data Tecnica International, Glen Echo, MD, USA</div><div class="fm-affl" id="A107" lang="en">  <sup>107</sup>Department of Epidemiology and Public Health, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Japan</div><div class="fm-affl" id="A108" lang="en">  <sup>108</sup>Clinical Research Facility, Department of Medicine, NUI Galway, Galway, Ireland</div><div class="fm-affl" id="A109" lang="en">  <sup>109</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="A110" lang="en">  <sup>110</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="A111" lang="en">  <sup>111</sup>Department of Health Services, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="A112" lang="en">  <sup>112</sup>Kaiser Permanente Washington Health Research Institute, Seattle, WA, USA</div><div class="fm-affl" id="A113" lang="en">  <sup>113</sup>Brain Center Rudolf Magnus, Department of Neurology, University Medical Center Utrecht, Utrecht, the Netherlands</div><div class="fm-affl" id="A114" lang="en">  <sup>114</sup>Usher Institute of Population Health Sciences and Informatics, University of Edinburgh, Edinburgh, UK</div><div class="fm-affl" id="A115" lang="en">  <sup>115</sup>Centre for Clinical Brain Sciences, University of Edinburgh, Edinburgh, UK</div><div class="fm-affl" id="A116" lang="en">  <sup>116</sup>Fred Hutchinson Cancer Research Center, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="A117" lang="en">  <sup>117</sup>Department of Medicine, Brigham and Women’s Hospital, Boston, MA, USA</div><div class="fm-affl" id="A118" lang="en">  <sup>118</sup>Department of Biostatistics, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="A119" lang="en">  <sup>119</sup>Nuffield Department of Clinical Neurosciences, University of Oxford, Oxford, UK</div><div class="fm-affl" id="A120" lang="en">  <sup>120</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute at Harbor-UCLA Medical Center, Torrance, CA, USA</div><div class="fm-affl" id="A121" lang="en">  <sup>121</sup>Division of Genomic Outcomes, Department of Pediatrics, Harbor-UCLA Medical Center, Torrance, CA, USA</div><div class="fm-affl" id="A122" lang="en">  <sup>122</sup>Department of Neurology, Miller School of Medicine, University of Miami, Miami, FL, USA</div><div class="fm-affl" id="A123" lang="en">  <sup>123</sup>Department of Allergy and Rheumatology, Graduate School of Medicine, University of Tokyo, Tokyo, Japan</div><div class="fm-affl" id="A124" lang="en">  <sup>124</sup>Center for Public Health Genomics, University of Virginia, Charlottesville, VA, USA</div><div class="fm-affl" id="A125" lang="en">  <sup>125</sup>Department of Pediatrics, College of Medicine, University of Oklahoma Health Sciences Center, Oklahoma City, OK, USA</div><div class="fm-affl" id="A126" lang="en">  <sup>126</sup>Department of Neurology, Medical University of Graz, Graz, Austria</div><div class="fm-affl" id="A127" lang="en">  <sup>127</sup>Institute for Community Medicine, SHIP-KEF, University Medicine Greifswald, Greifswald, Germany</div><div class="fm-affl" id="A128" lang="en">  <sup>128</sup>Department of Neurology, University Medicine Greifswald, Greifswald, Germany</div><div class="fm-affl" id="A129" lang="en">  <sup>129</sup>Department of Neurology, Jagiellonian University, Krakow, Poland</div><div class="fm-affl" id="A130" lang="en">  <sup>130</sup>Department of Neurology, Justus Liebig University, Giessen, Germany</div><div class="fm-affl" id="A131" lang="en">  <sup>131</sup>Department of Clinical Neurosciences/Neurology, Institute of Neuroscience and Physiology, Sahlgrenska Academy at University of Gothenburg, Gothenburg, Sweden</div><div class="fm-affl" id="A132" lang="en">  <sup>132</sup>Sahlgrenska University Hospital, Gothenburg, Sweden</div><div class="fm-affl" id="A133" lang="en">  <sup>133</sup>Stroke Division, Florey Institute of Neuroscience and Mental Health, University of Melbourne, Heidelberg, Victoria, Australia</div><div class="fm-affl" id="A134" lang="en">  <sup>134</sup>Austin Health, Department of Neurology, Heidelberg, Victoria, Australia</div><div class="fm-affl" id="A135" lang="en">  <sup>135</sup>Department of Internal Medicine, Section Gerontology and Geriatrics, Leiden University Medical Center, Leiden, the Netherlands</div><div class="fm-affl" id="A136" lang="en">  <sup>136</sup>INSERM, U1219 Bordeaux, France</div><div class="fm-affl" id="A137" lang="en">  <sup>137</sup>Department of Public Health, Bordeaux University Hospital, Bordeaux, France</div><div class="fm-affl" id="A138" lang="en">  <sup>138</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands</div><div class="fm-affl" id="A139" lang="en">  <sup>139</sup>Center for Medical Systems Biology, Leiden, the Netherlands</div><div class="fm-affl" id="A140" lang="en">  <sup>140</sup>School of Medicine, Dentistry and Nursing at the University of Glasgow, Glasgow, UK</div><div class="fm-affl" id="A141" lang="en">  <sup>141</sup>Department of Epidemiology and Population Health, Albert Einstein College of Medicine, New York, NY, USA</div><div class="fm-affl" id="A142" lang="en">  <sup>142</sup>Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS, USA</div><div class="fm-affl" id="A143" lang="en">  <sup>143</sup>A list of members and affiliations appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></div><div class="fm-affl" id="A144" lang="en">  <sup>144</sup>Department of Human Genetics, McGill University, Montreal, Quebec, Canada</div><div class="fm-affl" id="A145" lang="en">  <sup>145</sup>Department of Pathophysiology, Institute of Biomedicine and Translation Medicine, University of Tartu, Tartu, Estonia</div><div class="fm-affl" id="A146" lang="en">  <sup>146</sup>Department of Cardiac Surgery, Tartu University Hospital, Tartu, Estonia</div><div class="fm-affl" id="A147" lang="en">  <sup>147</sup>Clinical Gene Networks AB, Stockholm, Sweden</div><div class="fm-affl" id="A148" lang="en">  <sup>148</sup>Department of Genetics and Genomic Sciences, Icahn Institute for Genomics and Multiscale Biology Icahn School of Medicine at Mount Sinai, New York, NY, USA</div><div class="fm-affl" id="A149" lang="en">  <sup>149</sup>Department of Pathophysiology, Institute of Biomedicine and Translation Medicine, Biomeedikum, University of Tartu, Tartu, Estonia</div><div class="fm-affl" id="A150" lang="en">  <sup>150</sup>Integrated Cardio Metabolic Centre, Department of Medicine, Karolinska Institutet, Karolinska Universitetssjukhuset, Huddinge, Sweden</div><div class="fm-affl" id="A151" lang="en">  <sup>151</sup>Clinical Gene Networks AB, Stockholm, Sweden</div><div class="fm-affl" id="A152" lang="en">  <sup>152</sup>UPMC Univ. Paris 06, INSERM, UMR_S 1166, Team Genomics &amp; Pathophysiology of Cardiovascular Diseases, Sorbonne Universités, Paris, France</div><div class="fm-affl" id="A153" lang="en">  <sup>153</sup>I CAN Institute for Cardiometabolism and Nutrition, Paris, France</div><div class="fm-affl" id="A154" lang="en">  <sup>154</sup>Department of Biomedical Engineering, University of Virginia, Charlottesville, VA, USA</div><div class="fm-affl" id="A155" lang="en">  <sup>155</sup>Group Health Research Institute, Group Health Cooperative, Seattle, WA, USA</div><div class="fm-affl" id="A156" lang="en">  <sup>156</sup>Seattle Epidemiologic Research and Information Center, VA Office of Research and Development, Seattle, WA, USA</div><div class="fm-affl" id="A157" lang="en">  <sup>157</sup>Cardiovascular Research Center, MGH, Boston, MA, USA</div><div class="fm-affl" id="A158" lang="en">  <sup>158</sup>Department of Medical Research, Bærum Hospital, Vestre Viken Hospital Trust, Gjettum, Norway</div><div class="fm-affl" id="A159" lang="en">  <sup>159</sup>Saw Swee Hock School of Public Health, National University of Singapore and National University Health System, Singapore, Singapore</div><div class="fm-affl" id="A160" lang="en">  <sup>160</sup>National Heart and Lung Institute, Imperial College London, London, UK</div><div class="fm-affl" id="A161" lang="en">  <sup>161</sup>Department of Gene Diagnostics and Therapeutics, Research Institute, National Center for Global Health and Medicine, Tokyo, Japan</div><div class="fm-affl" id="A162" lang="en">  <sup>162</sup>Department of Epidemiology, Tulane University School of Public Health and Tropical Medicine, New Orleans, LA, USA</div><div class="fm-affl" id="A163" lang="en">  <sup>163</sup>Department of Cardiology, University Medical Center Groningen, University of Groningen, Groningen, the Netherlands</div><div class="fm-affl" id="A164" lang="en">  <sup>164</sup>MRC-PHE Centre for Environment and Health, School of Public Health, Department of Epidemiology and Biostatistics and the NIHR Imperial Biomedical Research Centre, Imperial College London, London, UK</div><div class="fm-affl" id="A165" lang="en">  <sup>165</sup>Department of Epidemiology and Biostatistics, Imperial College London, London, UK</div><div class="fm-affl" id="A166" lang="en">  <sup>166</sup>Department of Cardiology, Ealing Hospital NHS Trust, Southall, UK</div><div class="fm-affl" id="A167" lang="en">  <sup>167</sup>National Heart, Lung and Blood Research Institute, Division of Intramural Research, Population Sciences Branch, Framingham, MA, USA</div><div class="fm-affl" id="A168" lang="en">  <sup>169</sup>Department of Pharmaceutical Sciences, College of Pharmacy, University of Oklahoma Health Sciences Center, Oklahoma City, OK, USA</div><div class="fm-affl" id="A169" lang="en">  <sup>170</sup>Oklahoma Center for Neuroscience, Oklahoma City, OK, USA</div><div class="fm-affl" id="A170" lang="en">  <sup>171</sup>Department of Pathology and Genetics, Institute of Biomedicine, Sahlgrenska Academy at University of Gothenburg, Gothenburg, Sweden</div><div class="fm-affl" id="A171" lang="en">  <sup>172</sup>Department of Neurology, Helsinki University Hospital, Helsinki, Finland</div><div class="fm-affl" id="A172" lang="en">  <sup>173</sup>Clinical Neurosciences, Neurology, University of Helsinki, Helsinki, Finland</div><div class="fm-affl" id="A173" lang="en">  <sup>174</sup>Department of Neurology, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="A174" lang="en">  <sup>175</sup>Albrecht Kossel Institute, University Clinic of Rostock, Rostock, Germany</div><div class="fm-affl" id="A175" lang="en">  <sup>176</sup>Clinical Trial Service Unit and Epidemiological Studies Unit, Nuffield Department of Population Health, University of Oxford, Oxford, UK</div><div class="fm-affl" id="A176" lang="en">  <sup>177</sup>Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA</div><div class="fm-affl" id="A177" lang="en">  <sup>178</sup>Faculty of Medicine, University of Iceland, Reykjavik, Iceland</div><div class="fm-affl" id="A178" lang="en">  <sup>179</sup>Departments of Neurology and Public Health Sciences, University of Virginia School of Medicine, Charlottesville, VA, USA</div><div class="fm-affl" id="A179" lang="en">  <sup>180</sup>Glenn Biggs Institute for Alzheimer’s and Neurodegenerative Diseases, University of Texas Health Sciences Center, San Antonio, San Antonio, TX, USA</div><div class="fm-affl" id="A180" lang="en">  <sup>181</sup>Human Genetics Center, University of Texas Health Science Center at Houston, Houston, TX, USA</div><div class="fm-affl" id="A181" lang="en">  <sup>182</sup>Center for Genomic Medicine, Kyoto University Graduate School of Medicine, Kyoto, Japan</div><div class="fm-affl" id="A182" lang="en">  <sup>183</sup>Munich Cluster for Systems Neurology (SyNergy), Munich, Germany</div><div class="fm-affl" id="A183" lang="en">  <sup>184</sup>German Center for Neurodegenerative Diseases (DZNE), Munich, Germany</div><div id="FN1"><sup>*</sup><strong>Correspondence and requests for materials</strong> should be addressed to S.D. or M.D. <a href="mailto:dev@null" data-email="rf.xuaedrob-u@ettebed.einahpets" class="oemail">rf.xuaedrob-u@ettebed.einahpets</a>; <a href="mailto:dev@null" data-email="ed.nehcneum-inu.dem@snaghcid.nitram" class="oemail">ed.nehcneum-inu.dem@snaghcid.nitram</a></div><div id="FN2"><sup>185</sup>These authors contributed equally: Rainer Malik, Ganesh Chauhan, Matthew Traylor, Muralidharan Sargurupremraj and Yukinori Okada.</div><div id="FN3"><sup>186</sup>These authors jointly supervised this work: Kari Stefansson, Bradford B. Worrall, Steven J. Kittner, Sudha Seshadri, Myriam Fornage, Hugh S. Markus, Joanna M. M. Howson, Yoichiro Kamatani, Stephanie Debette and Martin Dichgans.</div><div id="FN4"><sup>168</sup>A list of members and affiliations appears at the end of the paper</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140458748872832_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div class="license half_rhythm"><strong>Reprints and permissions information</strong> is available at <a href="http://www.nature.com/reprints" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">www.nature.com/reprints</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=29531354" target="pmc_ext" ref="reftype=publisher&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div><strong>This article has been corrected. </strong>See <a href="/pubmed/31160810" class="int-reflink">Nat Genet. 2019 June 03; : </a>.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5968830/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="17" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Information.pdf" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Information.pdf</a><span style="color:gray"> (6.8M)</span></div><div class="small guid">GUID:&nbsp;CD870FAE-D1DA-48B9-9160-3CFF59838B9B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 11.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_11.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_11.xlsx</a><span style="color:gray"> (3.0M)</span></div><div class="small guid">GUID:&nbsp;68F4F1FD-1D7E-4F0C-B6C5-48273A715425</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 13.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_13.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_13.xlsx</a><span style="color:gray"> (43K)</span></div><div class="small guid">GUID:&nbsp;95B43C83-8362-4C02-B54C-E1FF69993F88</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 15.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_15.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_15.xlsx</a><span style="color:gray"> (20K)</span></div><div class="small guid">GUID:&nbsp;2D425973-8AC7-4F6B-962F-CBA359D665E6</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 17.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_17.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_17.xlsx</a><span style="color:gray"> (26K)</span></div><div class="small guid">GUID:&nbsp;9175BCCD-5E6C-4205-86DB-ACCF7CACE4DE</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 18.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_18.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_18.xlsx</a><span style="color:gray"> (58K)</span></div><div class="small guid">GUID:&nbsp;67E78E13-6A4D-4C12-8079-CD3290AFCB71</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 19.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_19.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_19.xlsx</a><span style="color:gray"> (6.7M)</span></div><div class="small guid">GUID:&nbsp;703F5DBF-8E85-49A2-807D-C85A748503F2</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 2.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_2.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_2.xlsx</a><span style="color:gray"> (67K)</span></div><div class="small guid">GUID:&nbsp;650C1430-4787-49DA-AF0F-F6E6B80FACD4</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 20.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_20.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_20.xlsx</a><span style="color:gray"> (3.9M)</span></div><div class="small guid">GUID:&nbsp;F8DC8A41-1EF3-404A-9C52-C456CA7BC90D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 21.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_21.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_21.xlsx</a><span style="color:gray"> (19K)</span></div><div class="small guid">GUID:&nbsp;9A06BD32-9191-4421-B763-52E14F42CF55</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 22.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_22.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_22.xlsx</a><span style="color:gray"> (73K)</span></div><div class="small guid">GUID:&nbsp;A56BC4A9-4EF8-48FF-A40F-500568C0A6AA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 23.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_23.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_23.xlsx</a><span style="color:gray"> (18K)</span></div><div class="small guid">GUID:&nbsp;4AE81712-B1FD-4BBE-814A-FD5D41A9E59C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 24.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_24.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_24.xlsx</a><span style="color:gray"> (25K)</span></div><div class="small guid">GUID:&nbsp;C371C1C3-34A8-4EAE-942F-07A823ED19AB</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 25.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_25.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_25.xlsx</a><span style="color:gray"> (1.5M)</span></div><div class="small guid">GUID:&nbsp;E4BBCCC1-2327-46B8-88E5-CC02DB9D3C6A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 27.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_27.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_27.xlsx</a><span style="color:gray"> (25K)</span></div><div class="small guid">GUID:&nbsp;F88A252F-CEF2-4E4B-9936-3357F2D5F808</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 6.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_6.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_6.xlsx</a><span style="color:gray"> (26K)</span></div><div class="small guid">GUID:&nbsp;988E7751-D7F4-467C-A397-3CC673D1A422</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 7.</strong></div><div><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_7.xlsx" data-ga-action="click_feat_suppl">NIHMS959658-supplement-Supplementary_Table_7.xlsx</a><span style="color:gray"> (20K)</span></div><div class="small guid">GUID:&nbsp;4BBF2FB6-7E3F-4610-BF0B-57BD86175BBB</div></div></dd></dl><dl data-length="133" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">The datasets generated and/or analyzed during the current study are available from the corresponding authors upon reasonable request.</p></dd></dl></div><div id="idm140458741176288" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140458741176288title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Stroke has multiple etiologies, but the underlying genes and pathways are largely unknown. We conducted a multiancestry genome-wide-association meta-analysis in 521,612 individuals (67,162 cases and 454,450 controls) and discovered 22 new stroke risk loci, bringing the total to 32. We further found shared genetic variation with related vascular traits, including blood pressure, cardiac traits, and venous thromboembolism, at individual loci (<em>n</em> = 18), and using genetic risk scores and linkage-disequilibrium-score regression. Several loci exhibited distinct association and pleiotropy patterns for etiological stroke subtypes. Eleven new susceptibility loci indicate mechanisms not previously implicated in stroke pathophysiology, with prioritization of risk variants and genes accomplished through bioinformatics analyses using extensive functional datasets. Stroke risk loci were significantly enriched in drug targets for antithrombotic therapy.</p></div></div><div id="idm140458738281344" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Stroke is the second leading cause of death and disability-adjusted life years worldwide<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943481">1</a>,<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943419">2</a></sup>. Characterized by a neurological deficit of sudden onset, stroke is primarily caused by brain infarction (ischemic stroke) and, less often, by intracerebral hemorrhage (ICH). Common etiological subtypes of ischemic stroke include large-artery atherosclerotic stroke (LAS), cardioembolic stroke (CES), and stroke caused by small-vessel disease (small-vessel stroke (SVS)), which is also the leading cause of ICH. Previous genome-wide association studies (GWAS) in predominantly European-ancestry groups have identified ten loci robustly associated with stroke<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943487">3</a>–<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943416">12</a></sup>. In most instances, the associations with stroke were attributed to individual subtypes of ischemic stroke, such as LAS<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943463">5</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943428">8</a>,<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943417">9</a></sup>, CES<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943507">3</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943458">4</a></sup>, and SVS<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943439">10</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943543">12</a></sup>, or of ICH<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943482">6</a></sup>, although some loci were associated with two or more stroke subtypes<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943501">7</a>,<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943522">9</a>,<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943465">11</a>,<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943437">13</a></sup> or with any stroke<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943420">10</a></sup>. We hypothesized that combining a substantially larger sample size with a transancestral analytic approach would identify additional risk loci and improve fine mapping of causal variants. Hence, we combined all available stroke samples with published or unpublished GWAS data, including samples of non-European ancestry that were underrepresented in previous GWAS. We further hypothesized that stroke shares genetic influences with vascular risk factors, intermediate phenotypes for stroke (for example, carotid artery plaque (cPL)), and related phenotypes (for example, coronary artery disease (CAD)) and that a systematic approach to identify genetic influences shared among these traits would provide insights into stroke pathophysiology.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Results</h2><p id="P3" class="p p-first">We tested ~8 million SNPs and indels with minor-allele frequency (MAF) ≥ 0.01 in up to 67,162 stroke cases and 454,450 controls for association with stroke. One analysis involved European participants only (40,585 cases; 406,111 controls), and a second involved participants of European, East Asian (17,369; 28,195), African (5,541; 15,154), South Asian (2,437; 6,707), mixed Asian (365; 333), and Latin American (865; 692) ancestry (<a href="/pmc/articles/PMC5968830/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Participants were drawn from 29 studies with genome-wide genotypes imputed to 1000 Genomes Project (1000G) phase 1v3 or similar<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943506">14</a></sup> (MEGASTROKE consortium; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note and Supplementary Tables 1 and 2</a>). Ancestry-specific meta-analyses and subsequent fixed-effects transancestral meta-analyses and MANTRA transancestral meta-analyses were conducted<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943429">15</a></sup>. Analyses were performed for any stroke (AS), comprising ischemic stroke, ICH, and stroke of unknown or undetermined type (<em>n</em> = 67,162); any ischemic stroke (AIS) regardless of subtype <em>(n</em> = 60,341); and ischemic stroke subtypes (LAS, <em>n</em> = 6,688; CES, <em>n</em> = 9,006; SVS, <em>n</em> = 11,710).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5968830/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140458739119024" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms959658f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms959658f1.jpg" src="/pmc/articles/PMC5968830/bin/nihms959658f1.jpg"></div></a><div id="largeobj_idm140458739119024" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5968830/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC5968830/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5968830/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>MEGASTROKE study design</strong><p id="__p9">Variants were retained that passed central quality control (QC) criteria (Methods). The numbers of cases and controls are listed for each ancestry group. HRC, Haplotype Reference Consortium; imp, measure of imputation quality (Methods); FE, fixed effects; EUR, European ancestry; AFR, African ancestry; EAS, East Asian ancestry; SAS, South Asian ancestry; ASN, mixed Asian ancestry; LAT, Latin American ancestry; <em>P</em><sub>het</sub>, heterogeneity <em>P</em> value; <em>PP</em><sub>het</sub>, posterior probability of heterogeneity. *The ASN and LAT ancestries were composed of a single study and hence did not require ancestry-specific meta-analysis.</p></div></div></div><div id="S2" class="sec"><h3 id="S2title">New genome-wide-significant stroke loci</h3><p id="P4" class="p p-first">We identified 32 genome-wide significant loci, 22 of which were novel (<a href="/pmc/articles/PMC5968830/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="/pmc/articles/PMC5968830/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 3 and 4, and Supplementary Figs. 1–7</a>). Of the 22 novel loci, 18 were identified by transancestral meta-analyses (fixed-effects &lt;5.0 ×10<sup>−8</sup> or MANTRA log<sub>10</sub>(Bayes factor (BF)) &gt;6) (<a href="/pmc/articles/PMC5968830/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 1–5</a>), and the remaining four loci were identified by the ancestry-specific meta-analysis in European samples (fixed-effects <em>P</em> &lt;5.0 ×10<sup>−8</sup>) (<a href="/pmc/articles/PMC5968830/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 1–5</a>). Apart from two novel loci with a MAF between 0.01 and 0.05 and large effect-size estimates (odds ratios (ORs) of 2.33 and 1.95), the remaining 20 novel loci contained common variants (MAF 0.16–0.48) with observed ORs between 1.05 and 1.20 (<a href="/pmc/articles/PMC5968830/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Comparison of the 32 loci across Europeans and East Asians, the two largest ancestral subgroups, demonstrated significant correlations of risk-allele frequencies and ORs between populations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 8</a>), although six loci exhibited population-specific association (defined as <em>P</em> &lt;5.0×10<sup>−8</sup> in Europeans and <em>P</em> &gt;0.05 in East Asians or MAF in East Asians &lt;0.01) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). Estimates for the phenotypic variance explained by the 32 lead variants ranged between 0.6% and 1.8% (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5968830/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140458739115072" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms959658f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms959658f2.jpg" src="/pmc/articles/PMC5968830/bin/nihms959658f2.jpg"></div></a><div id="largeobj_idm140458739115072" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5968830/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC5968830/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5968830/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Association results of the transancestral GWAS meta-analysis and the prespecified ancestry-specific meta-analysis in European samples</strong><p id="__p10">Shown are novel (red) and known (black) genetic loci associated with any stroke or stroke subtypes. Top, Manhattan plot from the MANTRA transancestral GWAS meta-analysis for any stroke. The dotted line marks the threshold of statistical significance (log<sub>10</sub>(BF) &gt;6.0).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p14">Results from the MANTRA (transancestral) and METAL fixed-effects (transancestral and Europeans-only) GWAS meta-analyses</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140458738686176" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">rsID</th><th rowspan="1" colspan="1" valign="top" align="left">Chromosome</th><th rowspan="1" colspan="1" valign="top" align="left">Gene(s)</th><th rowspan="1" colspan="1" valign="top" align="left">Location relative to gene</th><th rowspan="1" colspan="1" valign="top" align="left">Risk allele/reference allele</th><th rowspan="1" colspan="1" valign="top" align="left">Risk-allele frequency (%)</th><th rowspan="1" colspan="1" valign="top" align="left">Phenotype</th><th rowspan="1" colspan="1" valign="top" align="left">Analysis</th><th rowspan="1" colspan="1" valign="top" align="left">OR</th><th rowspan="1" colspan="1" valign="top" align="left">95% CI</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="left">log<sub>10</sub> (BF)</th></tr></thead><tbody><tr><td colspan="2" rowspan="1" valign="top" align="left"><strong>Novel associations</strong></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs880315</td><td rowspan="1" colspan="1" valign="top" align="left">1p36</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>CASZ1</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">40</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.04–1.07</td><td rowspan="1" colspan="1" valign="top" align="left">3.62 ×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8.09</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12037987</td><td rowspan="1" colspan="1" valign="top" align="left">1p13</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>WNT2B</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.07</td><td rowspan="1" colspan="1" valign="top" align="left">1.05–1.10</td><td rowspan="1" colspan="1" valign="top" align="left">2.73 ×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.33</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs146390073</td><td rowspan="1" colspan="1" valign="top" align="left">1q43</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>RGS7</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">CES</td><td rowspan="1" colspan="1" valign="top" align="left">EUR</td><td rowspan="1" colspan="1" valign="top" align="left">1.95</td><td rowspan="1" colspan="1" valign="top" align="left">1.54–2.47</td><td rowspan="1" colspan="1" valign="top" align="left">2.20×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">NA<sup>a</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12476527</td><td rowspan="1" colspan="1" valign="top" align="left">2p23</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>KCNK3</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">5′-UTR</td><td rowspan="1" colspan="1" valign="top" align="left">G/T</td><td rowspan="1" colspan="1" valign="top" align="left">48</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.03–1.07</td><td rowspan="1" colspan="1" valign="top" align="left">6.44×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.47</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7610618</td><td rowspan="1" colspan="1" valign="top" align="left">3q25</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TM4SF4–TM4Sn</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">LAS</td><td rowspan="1" colspan="1" valign="top" align="left">EUR</td><td rowspan="1" colspan="1" valign="top" align="left">2.33</td><td rowspan="1" colspan="1" valign="top" align="left">1.74–3.12</td><td rowspan="1" colspan="1" valign="top" align="left">1.44×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">NA<sup>b</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs34311906</td><td rowspan="1" colspan="1" valign="top" align="left">4q25</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ANK2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">41</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">EUR</td><td rowspan="1" colspan="1" valign="top" align="left">1.07</td><td rowspan="1" colspan="1" valign="top" align="left">1.04–1.09</td><td rowspan="1" colspan="1" valign="top" align="left">1.07×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.67</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17612742</td><td rowspan="1" colspan="1" valign="top" align="left">4q31</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>EDNRA</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">21</td><td rowspan="1" colspan="1" valign="top" align="left">LAS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.19</td><td rowspan="1" colspan="1" valign="top" align="left">1.13–1.26</td><td rowspan="1" colspan="1" valign="top" align="left">1.46 ×10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9.47</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6825454</td><td rowspan="1" colspan="1" valign="top" align="left">4q31</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FGA</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">31</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.06</td><td rowspan="1" colspan="1" valign="top" align="left">1.04–1.08</td><td rowspan="1" colspan="1" valign="top" align="left">7.43 ×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7.53</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11957829</td><td rowspan="1" colspan="1" valign="top" align="left">5q23</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>LOC100505841</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">82</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.07</td><td rowspan="1" colspan="1" valign="top" align="left">1.05–1.10</td><td rowspan="1" colspan="1" valign="top" align="left">7.51 ×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.67</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6891174</td><td rowspan="1" colspan="1" valign="top" align="left">5q35</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NKX2-5</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">35</td><td rowspan="1" colspan="1" valign="top" align="left">CES</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.07–1.16</td><td rowspan="1" colspan="1" valign="top" align="left">5.82 ×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.96</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16896398</td><td rowspan="1" colspan="1" valign="top" align="left">6p21</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC22A7–ZNF318</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">T/A</td><td rowspan="1" colspan="1" valign="top" align="left">34</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.03–1.07</td><td rowspan="1" colspan="1" valign="top" align="left">1.30×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.60</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs42039</td><td rowspan="1" colspan="1" valign="top" align="left">7q21</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>CDK6</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">3′-UTR</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">77</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.07</td><td rowspan="1" colspan="1" valign="top" align="left">1.04–1.09</td><td rowspan="1" colspan="1" valign="top" align="left">6.55×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.84</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7859727</td><td rowspan="1" colspan="1" valign="top" align="left">9p21</td><td rowspan="1" colspan="1" valign="top" align="left"><strong>chr9p21</strong></td><td rowspan="1" colspan="1" valign="top" align="left">ncRNA intronic</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">53</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.03–1.07</td><td rowspan="1" colspan="1" valign="top" align="left">4.22 ×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8.01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10820405</td><td rowspan="1" colspan="1" valign="top" align="left">9q31</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>LINC01492</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">ncRNA intronic</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">82</td><td rowspan="1" colspan="1" valign="top" align="left">LAS</td><td rowspan="1" colspan="1" valign="top" align="left">EUR</td><td rowspan="1" colspan="1" valign="top" align="left">1.20</td><td rowspan="1" colspan="1" valign="top" align="left">1.12–1.28</td><td rowspan="1" colspan="1" valign="top" align="left">4.51 ×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.74</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2295786</td><td rowspan="1" colspan="1" valign="top" align="left">10q24</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SH3PXD2A</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">A/T</td><td rowspan="1" colspan="1" valign="top" align="left">60</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.04–1.07</td><td rowspan="1" colspan="1" valign="top" align="left">1.80×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8.34</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7304841</td><td rowspan="1" colspan="1" valign="top" align="left">12p12</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>PDE3A</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">A/C</td><td rowspan="1" colspan="1" valign="top" align="left">59</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.03–1.07</td><td rowspan="1" colspan="1" valign="top" align="left">4.93 ×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.87</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs35436</td><td rowspan="1" colspan="1" valign="top" align="left">12q24</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TBX3</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">62</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.03–1.06</td><td rowspan="1" colspan="1" valign="top" align="left">2.87×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.29</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9526212</td><td rowspan="1" colspan="1" valign="top" align="left">13q14</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>LRCH1</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">76</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.06</td><td rowspan="1" colspan="1" valign="top" align="left">1.04–1.08</td><td rowspan="1" colspan="1" valign="top" align="left">5.03×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7.97</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4932370</td><td rowspan="1" colspan="1" valign="top" align="left">15q26</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FURIN-FES</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">33</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.03–1.07</td><td rowspan="1" colspan="1" valign="top" align="left">2.88×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11867415</td><td rowspan="1" colspan="1" valign="top" align="left">17p13</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>PRPF8</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">18</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.09</td><td rowspan="1" colspan="1" valign="top" align="left">1.06–1.13</td><td rowspan="1" colspan="1" valign="top" align="left">4.81 ×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2229383</td><td rowspan="1" colspan="1" valign="top" align="left">19p13</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>ILF3</em></strong><em>-SLC44A2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic; synonymous</td><td rowspan="1" colspan="1" valign="top" align="left">T/G</td><td rowspan="1" colspan="1" valign="top" align="left">65</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.03–1.07</td><td rowspan="1" colspan="1" valign="top" align="left">4.72 ×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.02</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs8103309</td><td rowspan="1" colspan="1" valign="top" align="left">19p13</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SMARCA4-LDLR</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">65</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.03–1.07</td><td rowspan="1" colspan="1" valign="top" align="left">3.40×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.85</td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left"><strong>Previously known associations</strong></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12124533</td><td rowspan="1" colspan="1" valign="top" align="left">1p13</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TSPAN2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">24</td><td rowspan="1" colspan="1" valign="top" align="left">LAS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.17</td><td rowspan="1" colspan="1" valign="top" align="left">1.11–1.23</td><td rowspan="1" colspan="1" valign="top" align="left">1.22 ×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.60</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1052053</td><td rowspan="1" colspan="1" valign="top" align="left">1q22</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>PMF1</em></strong>–<em>SEMA4A</em></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic; nonsynonymous</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">40</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.06</td><td rowspan="1" colspan="1" valign="top" align="left">1.05–1.08</td><td rowspan="1" colspan="1" valign="top" align="left">2.70×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">11.92</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs13143308</td><td rowspan="1" colspan="1" valign="top" align="left">4q25</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PITX2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">T/G</td><td rowspan="1" colspan="1" valign="top" align="left">28</td><td rowspan="1" colspan="1" valign="top" align="left">CES</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.32</td><td rowspan="1" colspan="1" valign="top" align="left">1.27–1.37</td><td rowspan="1" colspan="1" valign="top" align="left">1.86×10<sup>−47</sup></td><td rowspan="1" colspan="1" valign="top" align="left">45.10</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4959130</td><td rowspan="1" colspan="1" valign="top" align="left">6p25</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FOXF2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">14</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.08</td><td rowspan="1" colspan="1" valign="top" align="left">1.05–1.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.42 ×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7.52</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2107595</td><td rowspan="1" colspan="1" valign="top" align="left">7p21</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HDAC9–TWIST1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">24</td><td rowspan="1" colspan="1" valign="top" align="left">LAS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.21</td><td rowspan="1" colspan="1" valign="top" align="left">1.15–1.26</td><td rowspan="1" colspan="1" valign="top" align="left">3.65×10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="left">12.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs635634</td><td rowspan="1" colspan="1" valign="top" align="left">9q34</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ABO</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">19</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">EUR</td><td rowspan="1" colspan="1" valign="top" align="left">1.08</td><td rowspan="1" colspan="1" valign="top" align="left">1.05–1.11</td><td rowspan="1" colspan="1" valign="top" align="left">9.18×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2005108</td><td rowspan="1" colspan="1" valign="top" align="left">11q22</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MMP12</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.08</td><td rowspan="1" colspan="1" valign="top" align="left">1.05–1.11</td><td rowspan="1" colspan="1" valign="top" align="left">3.33×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.12</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3184504</td><td rowspan="1" colspan="1" valign="top" align="left">12q24</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>SH2B3</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">Exonic; nonsynonymous</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">45</td><td rowspan="1" colspan="1" valign="top" align="left">AIS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.08</td><td rowspan="1" colspan="1" valign="top" align="left">1.06–1.10</td><td rowspan="1" colspan="1" valign="top" align="left">2.17×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">12.04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12932445</td><td rowspan="1" colspan="1" valign="top" align="left">16q22</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>ZFHX3</em></strong></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">21</td><td rowspan="1" colspan="1" valign="top" align="left">CES</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.20</td><td rowspan="1" colspan="1" valign="top" align="left">1.15–1.25</td><td rowspan="1" colspan="1" valign="top" align="left">6.86×10<sup>−18</sup></td><td rowspan="1" colspan="1" valign="top" align="left">15.49</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12445022</td><td rowspan="1" colspan="1" valign="top" align="left">16q24</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZCCHC14</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">31</td><td rowspan="1" colspan="1" valign="top" align="left">AS</td><td rowspan="1" colspan="1" valign="top" align="left">TRANS</td><td rowspan="1" colspan="1" valign="top" align="left">1.06</td><td rowspan="1" colspan="1" valign="top" align="left">1.04–1.08</td><td rowspan="1" colspan="1" valign="top" align="left">1.05×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8.57</td></tr></tbody></table></div><div id="largeobj_idm140458738686176" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5968830/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p15" class="p p-first-last">For each locus, the variant reaching the highest BF in the MANTRA or the lowest <em>P</em> value in the fixed-effects transancestral meta-analysis or the fixed-effects Europeans-only meta-analysis, respectively, is shown, and the respective stroke phenotype showing the strongest association is specified. Gene names in bold indicate that the variant is located within the gene; in other cases, the first gene corresponds to the closest gene, whereas additional gene names indicate eQTL signals from multiple studies, or from both eQTLs and meQTLs, or genes previously suspected to be causal (LDLR), with a maximum of two genes reported. The lead SNPs in <em>ILF3-SLC44A2</em> and <em>SMARCA–LDLR</em> are in low LD (<em>r</em><sup>2</sup> = 0.082). TRANS, MANTRA transancestral meta-analysis; EUR, Europeans-only fixed-effects meta-analysis; OR, odds ratio; CI, confidence interval; NA, not assessed; UTR, untranslated region.</p></div><div id="TFN2"><sup>a</sup>rs146390073 did not meet the MAF threshold of 0.01 in samples other than those of European ancestry.</div><div id="TFN3"><sup>b</sup>rs7610618: The transancestral meta-analysis results showed high heterogeneity (<em>PP</em><sub>het</sub> = 0.96) and were thus excluded.</div></div></div><p id="P5">Gene-based tests using VEGAS2 (ref. <sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943453">16</a></sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 9</a>) confirmed the loci identified by the GWAS analyses above and yielded a novel significant (P &lt;2.02 ×10<sup>−6</sup>, Bonferroni corrected for the number of genes) association of the neighboring genes <em>ICA1L</em> and <em>WDR12</em> with SVS (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 9 and 10</a>). Prior studies have demonstrated that variants in this region are associated with white-matter hyperintensity (WMH) burden<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943505">17</a></sup>, a brain magnetic resonance imaging marker of small-vessel disease (SVD).</p><p id="P6" class="p p-last">Twenty-one additional loci met a less stringent threshold for suggestive evidence of association (log<sub>10</sub>(BF) &gt;5.0 or <em>P</em> &lt;1.0 × 10<sup>−6</sup> in the transancestral fixed-effects analysis) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>), including three loci previously implicated in Mendelian stroke (<em>HTRA1</em>, <em>COL4A1</em>, and <em>COL4A2</em>)<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943521">18</a>–<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943517">21</a></sup>.</p></div><div id="S3" class="sec"><h3 id="S3title">Associations with etiological stroke subtypes</h3><p id="P7" class="p p-first-last">Genome-wide significance was reached for 18 loci (12 novel) for AS, 20 (12 novel) for AIS, 6 (3 novel) for LAS, 4 (2 novel) for CES, and 2 (<em>ICA1L–WDR12</em> novel, discovered in gene-based tests) for SVS (<a href="/pmc/articles/PMC5968830/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>, <a href="/pmc/articles/PMC5968830/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 1–5 and 10</a>). Several loci reaching genome-wide significance for one of the ischemic stroke subtypes were also genome-wide significant for AIS or AS, whereas none reached genome-wide significance for multiple ischemic stroke subtypes (<a href="/pmc/articles/PMC5968830/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>). For some novel loci, the association was strictly confined to a single subtype <em>(P</em> &gt;0.5 for other stroke subtypes): <em>EDNRA</em> and <em>LINC01492</em> showed association with LAS only, thus suggesting mechanisms limited to atherosclerosis, and <em>NKX2-5</em> showed association with CES only, thus suggesting that the association may be primarily mediated by cardioembolism. We also found subtype specificity for previously described loci <em>(TSPAN2</em> for LAS and <em>PITX2</em> for CES). We further investigated shared genetic influences of individual loci on different stroke subtypes by using gwas-pw analyses<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943535">22</a></sup>, which estimate the posterior probability that a specified genomic region influences two different traits. By applying a posterior-probability cutoff of 90% for shared contribution at a given locus (model 3), we found shared genetic influence between LAS and SVS at <em>SH2B3</em>, and between LAS and CES at <em>ABO</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 10</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 11</a>).</p></div><div id="S4" class="sec"><h3 id="S4title">Conditional analysis to identify independent signals within loci</h3><p id="P8" class="p p-first-last">When conditioning all SNPs in a ±0.5-Mb window on the lead SNPs in the Europeans-only analysis, we found two additional independent genome-wide signals at the <em>PITX2</em> locus for CES, in agreement with known multiple independent loci at <em>PITX2</em> for atrial fibrillation (AF)<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943547">23</a></sup>, thus suggesting that a similar genetic architecture at this locus influences both conditions (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 12</a>). We further found suggestive independent signals at <em>MMP12, SH2B3</em>, and <em>HDAC9-TWIST1</em> that did not reach genome-wide significance (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 11</a>).</p></div><div id="S5" class="sec"><h3 id="S5title">Association of individual stroke risk variants with related vascular traits</h3><p id="P9" class="p p-first-last">Several of our loci are in the genomic vicinity of established risk loci for vascular risk factors (for example, blood pressure (BP)), and related vascular phenotypes affecting the heart (for example, CAD), vasculature (for example, carotid intima media thickness (cIMT)), or the brain (WMH). To systematically explore the genetic overlap between stroke and these traits, we surveyed published GWAS for BP, blood lipids, type 2 diabetes (T2D), cIMT, cPL, AF, venous thromboembolism (VTE), CAD, and WMH, assembled through the IGEN-BP<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943424">24</a></sup>, ENGAGE<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943449">25</a></sup>, DIAGRAM<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943480">26</a></sup>, CHARGE<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943414">27</a>,<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943526">28</a></sup>, AFGen<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943484">29</a></sup>, INVENT<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943444">30</a></sup>, and CARDIoGRAMplusC4D<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943442">31</a></sup> consortia (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 12</a>). When constructing sets of index SNPs of the nonstroke phenotypes (Bonferroni-adjusted <em>P</em> &lt;1.3 ×10<sup>−4</sup> = 0.05/32 loci/12 related vascular traits) and SNPS in high linkage disequilibrium (LD) (<em>r</em><sup>2</sup> &gt;0.9 in the 1000G European-ancestry dataset (EUR)) with those index variants, 17 of the 32 stroke lead variants showed overlap with these sets (<a href="/pmc/articles/PMC5968830/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 13</a>). Fourteen loci reached genome-wide significance (P &lt;5.0 ×10<sup>−8</sup>) for association with one or more of the following phenotypes: BP (five loci), CAD (five loci), AF (two loci), VTE (two loci), low-density liproprotein (LDL) cholesterol (two loci), cPL (one locus), and WMH (one locus). Among the 21 additional subthreshold loci for stroke (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>), six loci have previously been associated with related vascular traits, including AF (<em>PRRX</em> and CAV1–CAV2)<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943520">32</a></sup>, VTE (F11)<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943436">30</a></sup>, CAD <em>(SWAP70</em> and LPA)<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943532">31</a></sup>, blood lipids <em>(LPA</em>)<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943536">31</a></sup>, and WMH <em>(ICA1L–WDR12)</em><sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943539">28</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC5968830/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140458737692016" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms959658f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms959658f3.jpg" src="/pmc/articles/PMC5968830/bin/nihms959658f3.jpg"></div></a><div id="largeobj_idm140458737692016" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5968830/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC5968830/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC5968830/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic overlap between stroke and related vascular traits at the 32 genome-wide-significant loci for stroke</strong><p id="__p11"><strong>a</strong>, Association results from the look-ups in published GWAS data for related vascular traits. Symbol sizes reflect <em>P</em> values for association with the related trait. <strong>b</strong>, Venn diagram. Loci reaching genome-wide significance for association with stroke subtypes are marked with a dagger symbol (for CES), underlined (for LAS), or marked with an asterisk (for SVS). Novel loci are in bold. <em>SH3PXD2A, WNT2B, PDE3A</em>, and <em>OBFC1</em> have previously been associated with AF <em>(SH3PXD2A)</em><sup><a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943540">65</a></sup>, or diastolic <em>(WNT2B</em> and PDE3A)<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943461">24</a>,<a href="#R72" rid="R72" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943529">72</a></sup> or systolic <em>(OBFC1)</em><sup><a href="#R73" rid="R73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943471">73</a></sup> BP, but the respective lead SNPs were in low LD (<em>r</em><sup>2</sup> &lt;0.1 in the 1000G cosmopolitan panel) with variants associated with stroke in the current GWAS. MRI, magnetic resonance imaging; IMT, intima-media thickness; LDL, low-density lipoprotein; HDL, high-density lipoprotein. The lead variant for <em>TBX3</em> is not included in the original datasets for BP traits (SBP and DBP). Results are based on a perfect proxy SNP (rs35432, <em>r</em><sup>2</sup> = 1 in the European 1000G phase 3 reference).</p></div></div></div></div><div id="S6" class="sec"><h3 id="S6title">Association of genetic risk scores of related vascular traits</h3><p id="P10" class="p p-first">Second, we generated weighted genetic risk scores (wGRS) for VTE, BP-related traits, blood lipids, T2D, and CAD by using the lead SNPs from published GWAS and tested these wGRS for association with each stroke phenotype, implementing the inverse-variance weighting approach (Methods and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 14</a>). We found significant associations (P &lt;5.6 ×10<sup>−3</sup>, correcting for nine independent phenotypes; Methods) with wGRS for all traits examined, except for triglyceride and LDL-cholesterol levels, and observed clear differences between stroke subtypes (<a href="/pmc/articles/PMC5968830/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). The strongest association was between the wGRS for CAD and LAS, in agreement with shared pathophysiology through atherosclerosis. We further found associations of all stroke subtypes with wGRS for BP traits. The wGRS for VTE was significantly associated with both LAS and CES (all <em>P</em> &lt;1.0 ×10<sup>−4</sup>) but not SVS. The wGRS for high-density lipoprotein (HDL) cholesterol showed a significant inverse association with SVS.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC5968830/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140458738693168" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms959658f4.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms959658f4.jpg" src="/pmc/articles/PMC5968830/bin/nihms959658f4.jpg"></div></a><div id="largeobj_idm140458738693168" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5968830/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a target="object" href="/pmc/articles/PMC5968830/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC5968830/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Fig. 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Shared genetic contribution between stroke and related vascular traits</strong><p id="__p12">Contributions determined by weighted genetic risk scores (wGRS, top) and LD-score regression analysis (bottom). Effect sizes and significance levels are represented by color and symbol size. <em>β</em>, wGRS effect size; <em>R</em>(g), genetic correlation. DBP, diastolic blood pressure; SBP, systolic blood pressure; MAP, mean arterial pressure; PP, pulse pressure; HTN, hypertension, TGL, triglyceride level. Sample sizes for related vascular traits are displayed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 12</a>. NS, nonsignificant.</p></div></div></div><p id="P11" class="p p-last">In the present setting, the wGRS analysis was used primarily to explore the genetic overlap with related vascular traits rather than as a tool for establishing causal inference. In sensitivity analyses, we conducted an MR–Egger regression to explore whether any of the significant associations between vascular wGRS and stroke might be partly driven by directional pleiotropy. There was no indication of directional pleiotropy except for the association between the SBP wGRS and AS (MR–Egger intercept estimate <em>P</em> = 0.015), which was no longer significant after removal of 6 of 37 SNPs appearing as outliers from the leave-one-out analysis (Methods), thus leading to causal estimates in broad agreement across regression techniques (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 15</a>).</p></div><div id="S7" class="sec"><h3 id="S7title">Shared genetic contribution to stroke and related vascular traits genome wide</h3><p id="P12" class="p p-first-last">Third, we applied LD-score regression to quantify the extent of shared genetic contributions between traits at a genome-wide level<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943443">33</a>,<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943513">34</a></sup>. Using available GWAS results from individuals of European ancestry, we found significant positive correlations (<em>r</em><sub>g</sub> &gt;0; <em>P</em> &lt;5.6 × 10<sup>−3</sup>, correcting for nine independent phenotypes), mostly corroborating the wGRS results (<a href="/pmc/articles/PMC5968830/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 16</a>). In addition, we found significant genetic overlap between triglyceride levels and AIS, and similar results were obtained in available GWAS datasets from individuals of East Asian ancestry (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 16</a>). The results did not substantially change after removal of genome-wide signals for stroke and related vascular traits and their proxies (<em>r</em><sup>2</sup> &gt;0.8 in 1000G EUR).</p></div><div id="S8" class="sec"><h3 id="S8title">Global epigenetic patterns at the 32 stroke risk loci</h3><p id="P13" class="p p-first-last">To test for cell-specific enrichment in chromatin marks that were previously shown to be phenotypically cell-type specific in the Encyclopedia of DNA Elements (ENCODE)/RoadMap (histone H3 modifications H3K4me1, H3K4me3, and H3K9ac)<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943491">35</a></sup>, we implemented the epigwas tool<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943427">35</a></sup> and the narrow peak information from the latest RoadMap dataset (127 tissues)<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943489">36</a></sup>. Epigwas estimates the enrichment score (ratio of the height of the nearest narrow peak to the distance to the peak) for the lead variant and proxies (<em>r</em><sup>2</sup> ≥0.8 in the 1000G cosmopolitan panel) and calculates statistical significance by examining the relative proximity and specificity of the test SNP set with 10,000 sets of matched background. The analysis showed significant enrichment of enhancer and promoter sites (marked by H3K4me1 and H3K4me3) in mesenchymal stem cells, embryonic stem cells, epithelial cells, and blood and T cells, and of active promoters (marked by H3K9ac) in embryonic stem cells and digestive tissue (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 17</a>).</p></div><div id="S9" class="sec"><h3 id="S9title">Pathway analyses</h3><p id="P14" class="p p-first-last">To identify pathways overrepresented in the stroke association results, we used the DEPICT gene-set enrichment tool<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943545">37</a></sup>, using all SNPs with log<sub>10</sub>(BF) &gt;5 for the respective stroke subtype. We found three gene sets to be significantly (false discovery rate (FDR) &lt;5%) associated with AS: enlarged heart, decreased cardiac muscle contractility, and oxaloacetate metabolic process (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 18</a>). Next, we used Ingenuity Pathway Analysis (IPA; URLs), examining genes within the 53 stroke loci with log<sub>10</sub>(BF) &gt;5. The extended gene list (<em>r</em><sup>2</sup> &gt;0.5 in 1000G Europeans or East Asians, or located within 50 kb of the lead SNP) consisted of 214 genes. We found the coagulation system to be the most significant canonical pathway, followed by cardiomyocyte differentiation via bone-morphogenetic-protein receptors (FDR of 5%) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 19</a>). Finally, we tested enrichment of VEGAS2-derived gene-based <em>P</em> values in expert-curated and computationally predicted Biosystem gene sets<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943510">38</a></sup>, adapting VEGAS2Pathway<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943500">39</a></sup>, and identified significant association with 18 pathways, including various cardiac pathways, muscle-cell fate commitment, and nitric oxide metabolic process with CES (FDR of 5%) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 20</a>).</p></div><div id="S10" class="sec"><h3 id="S10title">Fine mapping derived from credible SNP-set analyses</h3><p id="P15" class="p p-first-last">To decrease the number of candidate variants per locus to the most noteworthy associations, we constructed 95% credible SNP sets for each of the 32 loci (lead SNP and proxy SNPs <em>r</em><sup>2</sup> &gt;0.1 in 1000G panels), assuming one causal SNP per locus and uniform priors<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943441">40</a></sup>. Credible SNP sets were generated in all stroke phenotypes and for European, East Asian, and African ancestries separately. We found a marked decrease in credible SNP sets for most loci, a result expectedly most pronounced for the phenotype showing the strongest association signal (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 21</a>). The greatest refinement was observed at <em>RGS7, HDAC9–TWIST1</em>, and <em>SH2B3</em>, where the lead SNP was the only SNP contained in the 95% credible set for the stroke phenotype showing the strongest association.</p></div><div id="S11" class="sec"><h3 id="S11title">Stroke loci with nonsynonymous or predicted deleterious variants</h3><p id="P16" class="p p-first-last">To determine SNPs with protein-altering effects, we annotated all SNPs by using ANNOVAR<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943548">41</a></sup>. Of the 32 lead SNPs, three were exonic, of which two were nonsynonymous: rs3184504 (p.Arg262Trp) in <em>SH2B3</em> and rs1052053 (p.Gln75Arg) in <em>PMF1.</em> SH2B3 p.Arg262Trp is a loss-of function variant that leads to expansion of hematopoietic stem cells and enhanced megakaryopoiesis in humans<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943485">42</a></sup>. Both variants are predicted to be benign or tolerated by PolyPhen<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943426">43</a></sup> and SIFT<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943538">44</a></sup>. In addition, we identified a proxy SNP (<em>r</em><sup>2</sup> = 0.99 in 1000G EUR) for another lead SNP that was nonsynonymous, rs6050 (p.Thr331Ala) in <em>FGA</em>, also predicted to be benign or tolerated.</p></div><div id="S12" class="sec"><h3 id="S12title">Investigation of eQTLs, meQTLs, and pQTLs in different tissues</h3><p id="P17" class="p p-first">To determine whether stroke risk SNPs influenced the cis regulation of nearby genes, we interrogated genome-wide quantitative information (expression quantitative trait loci (eQTLs), methylation quantitative trait loci (meQTLs), and proteinexpression quantitative trait loci (pQTLs)) in extensive publicly and nonpublicly available datasets. These datasets encompass numerous tissues and cell types, including cardiac, vascular, and brain tissue; circulating cells; and vascular endothelial cells (Methods). These comprised the following: for eQTLs, GTEx V6 (ref. <sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943504">45</a></sup>), an expanded version of GRASP2 (refs <sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943478">46</a>,<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943531">47</a></sup>), HGVD<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943490">48</a></sup>, BIOS<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943515">49</a></sup>, Blueprint epigenome project (subset)<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943434">50</a></sup>, STARNET<sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943474">51</a></sup>, and the human aortic endothelial cell study<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943498">52</a></sup>; for meQTLs, the Blueprint epigenome project (subset)<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943451">50</a></sup> and the ARIC cohort<sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943472">53</a></sup>; and for pQTLs, the KORA cohort<sup><a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943446">54</a></sup>. Only cis eQTLs, meQTLs, and pQTLs were considered.</p><p id="P18">We found that in 18 of the 32 stroke risk loci, the lead stroke risk variant either overlapped or was in moderate to high LD (<em>r</em><sup>2</sup> &gt;0.8) with the most significant QTL variant for a nearby gene in at least one tissue or cell type (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 22 and 23</a>). For seven loci, we observed association of the lead SNP and proxies with expression of a single gene (or methylation or protein level), sometimes the nearest gene (<em>LRCH1, CDK6, CDKN2B, PRPF8</em>, and <em>MMP12)</em>, and sometimes a more distant nearby gene <em>(ZCCHC14</em> for the <em>ZCCHC14</em> locus, and <em>TWIST1</em> for the <em>HDAC9–TWIST1</em> locus), within the datasets explored. Associations were found primarily in stroke-relevant tissues and cell types, including vascular tissues, aortic endothelial cells, brain, blood, and immune cells. In most instances (11 loci, 61.1%), the risk SNP affected expression of multiple genes, thus suggesting that at individual loci, pleiotropic mechanisms, which might differ according to tissue/cell type, may in some instances influence stroke susceptibility<sup><a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943495">55</a>,<a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943415">56</a></sup>. For several of these loci, there was a clear predominance of eQTL associations with one gene in stroke-relevant tissues, such as <em>ZNF318</em> (6p21), <em><a id="__tag_702943601" class="tag_hotlink" href="/nuccore/AL049919" ref="/nuccore/AL049919"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AL049919","term_id":"4884168","term_text":"AL049919"}}</span>AL049919</a></em> (12q24), and <em>FES</em> (15q26) in brain tissues (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 22 and 23</a>).</p><p id="P19">At some loci, meQTLs and eQTLs provided complementary information on the regulatory pattern. For instance, for the <em>SH3PXD2A</em> locus, SNPs in high LD with the lead stroke risk variant were found to be eQTLs for multiple genes (<em>SH3PXD2A, SLK, GSTO1, GSTO2</em>, and <em>LOC729081)</em>, whereas several high-LD proxies (<em>r</em><sup>2</sup> &gt;0.96) functioned as the most significant meQTL for CpG probes located in the promoter region of <em>SH3PXD2A</em> and not any of the other genes.</p><p id="P20" class="p p-last">For the 149 genes located in the 32 genome-wide-significant loci (<em>r</em><sup>2</sup> &gt;0.5 in Europeans or East Asians, or located ±50 kb from the lead SNP; Methods), we assigned an empirical functional score based on the presence and number of eQTLs, meQTLs, pQTLs, and other biological criteria<sup><a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943460">57</a>,<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943533">58</a></sup> (Methods and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 24</a>), reasoning that genes with a higher functional score would be more likely to be causal, although this score requires validation by experimental data.</p></div><div id="S13" class="sec"><h3 id="S13title">Joint modeling of epigenetic marks and association statistics</h3><p id="P21" class="p p-first-last">In an additional approach to identify the most plausible causal variants and genes, we used RiVIERA<sup><a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943534">59</a></sup>, which jointly models summary association statistics and corresponding epigenetic regulatory information in a Bayesian framework to estimate the posterior probability of association (PPA). RiVIERA uses the RoadMap epigenome data of 127 tissue types and information on chromatin (H3K4me1, H3K4me3, H3K36me3, H3K27me3, H3K9me3, H3K27ac, and H3K9ac) and DNA-accessibility (DNase I) marks. Three of the stroke risk loci (<em>PMF1–SEMA4A, SH3PXD2A</em>, and <em>EDNRA)</em> displayed a pattern in which the association statistics and epigenetic regulatory information jointly contributed to the modeling of the RiVIERA credible SNP set (the minimum number of SNPs whose PPA, accounting for both association statistics and epigenetic regulatory information, sum to ≥ 95%) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 13</a>). The variants identified by RiVIERA as having the highest PPA were in moderate to high LD in the 1000G cosmopolitan panel with the respective lead SNP (rs7534434 for <em>PMF1-SEMA4A, r</em><sup>2</sup> = 0.79 with lead SNP; rs11191829 for <em>SH3PXD2A, r</em><sup>2</sup> = 0.99 with lead SNP; rs4835084 for <em>EDNRA, r<sup>2</sup></em> = 0.35 with lead SNP). Two of these (at <em>PMF1–SEMA4A</em> and <em>SH3PXD2A</em>) were significantly enriched in RNA polymerase II binding in ENCODE cell types<sup><a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943470">60</a></sup>, including H1 human embryonic stem cells (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 13</a>).</p></div><div id="S14" class="sec sec-last"><h3 id="S14title">Enrichment in drug-target genes</h3><p id="P22" class="p p-first-last">Given the previous evidence of the utility of GWAS in drug discovery and drug repositioning<sup><a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943445">57</a>,<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943492">61</a>,<a href="#R62" rid="R62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943422">62</a></sup>, we evaluated the overlap between stroke-associated genes and known drug targets. Among the 149 genes located within the 32 stroke risk loci, 16 (11%) were registered as targets of currently approved drugs in the DrugBank database and the Therapeutic Target Database (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 25</a>). Of these, two genes (<em>FGA</em> and <em>PDE3A)</em> were targets of approved drugs for antithrombotic therapy (ATC B01), i.e., alteplase, tenecteplase, reteplase, and anistreplase for <em>FGA</em>, and cilostazol for <em>PDE3A</em> (enrichment OR = 5.46, <em>P</em>=0.0369; <a href="/pmc/articles/PMC5968830/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>). This enrichment was strengthened after removal of the locus with the largest number of genes <em>(SH2B3</em>, 73 genes) (OR = 8.89, <em>P</em>=0.0166) and after addition of 65 genes in 21 suggestive stroke risk loci (OR = 7.83, <em>P</em> = 0.00606).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC5968830/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140458738689296" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms959658f5.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms959658f5.jpg" src="/pmc/articles/PMC5968830/bin/nihms959658f5.jpg"></div></a><div id="largeobj_idm140458738689296" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5968830/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a target="object" href="/pmc/articles/PMC5968830/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC5968830/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Fig. 5</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Connection between stroke risk genes and approved drugs for antithrombotic therapy</strong><p id="__p13">Shown are the connections among lead SNPs at stroke risk loci, biological stroke risk genes, and individual targeted drugs. Lead SNPs reaching suggestive evidence for association (MANTRA transancestral meta-analysis log<sub>10</sub>(BF) &gt;5) are shown in gray.</p></div></div></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Discussion</h2><p id="P23" class="p p-first">The current transancestral meta-analysis more than triples the number of stroke risk loci and identifies novel loci for AS, AIS, and all major subtypes of ischemic stroke. Our results highlight several major features of stroke genomics: (i) Approximately half of the identified stroke loci showed shared genetic association with other vascular traits, and the largest genetic correlation was found for blood pressure. We also identified shared genetic association with VTE, and distinct patterns of individual stroke subtypes provided further mechanistic insight. (ii) Eleven of the novel stroke risk loci (<em>ANK2, CDK6, KCNK3, LINC01492, LRCH1, NKX2-5, PDE3A, PRPF8, RGS7, TM4SF4–TM4SF1</em>, and <em>WNT2B)</em> suggest mechanisms not previously implicated in stroke pathophysiology; some of these suggest a strong link with cardiac mechanisms beyond those expected from established sources of cardioembolism. (iii) The 32 stroke risk loci were significantly enriched in drug targets for antithrombotic therapy—one for an approved thrombolytic drug (alteplase) and the other for an antiplatelet agent (cilostazol) approved for stroke prevention in Asia. (iv) Through incorporation of extensive functional datasets and bioinformatics analyses, we provide detailed information on prioritization of stroke risk variants and genes as a resource for further experimental follow-up.</p><p id="P24">Most of the genome-wide associations were identified with both AS and AIS. Although this result relates in part to a greater statistical power compared with that in subtype analysis, we also found shared genetic influences between stroke subtypes, as exemplified by the gwas-pw analyses (<em>SH2B3</em> and <em>ABO).</em> A notable finding was the identification of <em>PMF1-SEMA4A</em> as a risk locus for AIS. <em>PMF1–SEMA4A</em> is an established risk locus for nonlobar ICH<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943546">6</a></sup> and thus is, to our knowledge, the first reported locus reaching genome-wide significance for ischemic as well as hemorrhagic stroke. <em>PMF1–SEMA4A</em> further reached genome-wide association for WMH burden<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943496">28</a></sup> (<a href="/pmc/articles/PMC5968830/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>), an established marker for SVD, and showed a strong signal in the SVS subtype, thus suggesting that the association with stroke is at least in part mediated by SVD. The underlying biological pathways do not seem to involve known vascular risk factors and may thus identify new targets for stroke prevention.</p><p id="P25">Among the novel loci showing associations restricted to specific stroke subtypes, <em>EDNRA</em> is consistent with atherosclerotic mechanisms, given its association with LAS, cPL<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943438">27</a></sup>, and CAD<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943537">31</a></sup> (<a href="/pmc/articles/PMC5968830/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). <em>LINC01492</em> and the previously reported <em>TSPAN2</em> locus likewise displayed associations restricted to LAS but showed no association with related phenotypes in our look-ups and in prior literature, thus evidencing mechanisms more specific for LAS. <em>NKX2-5</em>, showing association restricted to CES, has previously been reported as a genome-wide risk locus for heart rate and PR interval<sup><a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943477">63</a>,<a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943530">64</a></sup> but not consistently for AF<sup><a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943450">63</a>,<a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943469">65</a></sup>, thus implicating cardiac mechanisms other than AF.</p><p id="P26">Although the number of loci reaching genome-wide significance for association with SVS remained low, our results suggest an important role of common genetic variation in SVS. First, several of the associations with AS or AIS, including those at novel loci (<em>CASZ1, LOC100505841, SH3PXD2A</em>, and <em>ICA1L–WDR12)</em>, showed predominant association with the SVS subtype (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 7 and 9</a>). Second, three of the top loci <em>(PMF1–SEMA4A, LOC100505841</em>, and <em>SH3PXD2A</em>) showed genetic overlap with loci for WMH. Third, several suggestive loci (log<sub>10</sub>(BF) ≥5) for AS and SVS contained genes implicated in monogenic SVD <em>(HTRA1, COL4A1</em>, and <em>COL4A2)</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>).</p><p id="P27">Our extensive exploration of shared genetic variation between stroke and related vascular traits found the most widespread correlations with BP phenotypes, in agreement with epidemiological data showing that high BP is the leading risk factor for stroke. A quarter of the 32 genome-wide-significant stroke loci were BP loci, most of which were novel with respect to stroke risk and showed association with risk of AS or AIS. Aside from the expected genetic overlap between LAS and CAD, we identified significant overlap between a wGRS for VTE and both LAS, and CES, but not SVS (<a href="/pmc/articles/PMC5968830/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 14</a>) despite a greater statistical power for this subtype, thus potentially suggesting that thrombotic processes play a less important role in SVS.</p><p id="P28">Three of our novel loci (<em>NKX2-5, ANK2</em>, and <em>LRCH1)</em> have previously been associated with cardiac pacing<sup><a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943544">63</a>,<a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943464">64</a>,<a href="#R66" rid="R66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943456">66</a></sup>. <em>NKX2-5</em> and <em>ANK2</em> have been further implicated in familial forms of cardiac disease<sup><a href="#R67" rid="R67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943413">67</a>–<a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943418">70</a></sup>, but none of the three loci were associated with AF or CAD in the latest published GWAS<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943528">31</a>,<a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943499">65</a></sup>. Apart from <em>NKX2-5</em>, these loci were not specifically associated with CES, thus possibly indicating an involvement of the underlying genes in roles beyond cardiac development and function. rs9526212, the lead variant in <em>LRCH1</em>, was an eQTL for <em>LRCH1</em> in multiple tissues, including the left ventricle, atherosclerotic aorta, atherosclerotic-lesion-free arteries, and blood (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 22</a>). Pathway analyses further supported a strong link with cardiac mechanisms.</p><p id="P29">The extensive in silico functional annotation of identified stroke risk loci provides informative elements for future prioritization and follow-up of the most compelling biological candidates. In some instances, the eQTL, meQTL, and pQTL information strongly supports involvement of one gene over others in the region, for example, for <em>SH3PXD2A</em>, encoding SH3 and PX-domain-containing protein 2A, an adaptor protein involved in formation of invadopodia and podosomes as well as extracellular-matrix degradation. For some loci, joint analysis of epigenetic regulatory effects and association statistics enabled prioritization of credible SNPs. When exploring the overall epigenetic patterns of identified stroke risk loci, we observed some enrichment in enhancer and promoter sites in developmental tissues, thus suggesting that some associations may be driven by developmental effects, as has recently been proposed for the <em>FOXF2</em> locus<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943524">10</a></sup>.</p><p id="P30"><em>RGS7</em> and <em>TM4SF4–TM4SF1</em> showed low MAFs, high heterogeneity, poor imputation quality in non-Europeans, and large effect-size estimates, and they must therefore be interpreted with caution. Moreover, although our extensive functional exploration provides guidance on gene prioritization for further exploration, additional experiments are required to identify the causal genes and variants. Several studies have provided limited information on stroke subtypes. Hence, the sample sizes for ischemic stroke subtypes were still relatively small. In addition, the proportion of the phenotypic variance explained by the 32 lead SNPs was relatively small but comparable to that in other complex diseases<sup><a href="#R71" rid="R71" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943508">71</a></sup>. Collectively, these aspects highlight the potential for gene discovery in the future.</p><p id="P31" class="p">In conclusion, we identified 22 novel stroke risk loci and demonstrated shared genetic variation with multiple related vascular traits. We further identified new loci offering mechanisms not previously implicated in stroke pathophysiology and provided a framework for prioritization of stroke risk variants and genes for further functional and experimental follow-up. Stroke risk loci were significantly enriched in drug targets for antithrombotic therapy, thus highlighting the potential of stroke genetics for drug discovery. Collectively, these findings represent a major advance in understanding the genetic underpinnings of stroke.</p><div id="S16" class="sec sec-last"><h3 id="S16title">URLs</h3><p id="P32" class="p p-first-last">Ingenuity Pathway Analysis, <a href="https://www.qiagenbioinformatics.com/products/ingenuity-pathway-analysis/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.qiagenbioinformatics.com/products/ingenuity-pathway-analysis/</a>.</p></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Methods</h2><p id="P33" class="p p-first-last">Methods, including statements of data availability and any associated accession codes and references, are available at <a href="https://doi.org/10.1038/s41588-018-0058-3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://doi.org/10.1038/s41588-018-0058-3</a>.</p></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Methods</h2><div id="S19" class="sec sec-first"><h3 id="S19title">Study design and phenotyping</h3><p id="P34" class="p p-first-last">A detailed description of the study design, participating studies, and phenotype definitions for stroke and stroke subtypes is provided in the Supplementary Note. Characteristics of study participants are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> for each study. All participants provided written informed consent, and local research ethics committees and institutional review boards approved the individual studies.</p></div><div id="S20" class="sec"><h3 id="S20title">Genotyping, imputation, and quality control</h3><p id="P35" class="p p-first">Genotyping platforms and imputation methods for each participating study are described in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>. All studies used imputed genotypes based on at least the 1000G phase 1 multiancestral reference panel and conducted logistic regression analyses (or Cox regression for longitudinal population-based cohort studies) for five stroke traits (AS, AIS, LAS, CES, and SVS) with all measured and imputed genetic variants in dosage format by using appropriate software under an additive genetic model with a minimum of sex and age as covariates. Information on additional covariates is given in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>.</p><p id="P36">Before ancestry-specific meta-analysis, QC was performed on each study by two independent researchers following a standardized protocol based on the suggestions of Winkler et al.<sup><a href="#R74" rid="R74" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943435">74</a></sup>. Marker names and alleles were harmonized across studies. Meta-analyses were restricted to autosomal biallelic markers from the 1000G phase 1 v3. Duplicate markers were removed from each study. <em>P–Z</em> plots, <em>QQ</em> plots and allele-frequency-plots were constructed for each study. After visual inspection, analysis and QC were repeated if deemed necessary. QC was conducted independently for all participating studies in at least two sites.</p><p id="P37" class="p p-last">Individual study-level filters were set to remove extreme effect values (<em>β</em> &gt; 5 or <em>β</em> &lt;−5), rare SNPs (MAF &lt;0.01) and variants with low imputation accuracy (oevar_imp or info score &lt;0.5). The effective allele count was defined as twice the product of the MAF, imputation accuracy (<em>r</em><sup>2</sup>, info score or oevar_imp), and number of cases. Variants with an effective allele count &lt;10 were excluded<sup><a href="#R74" rid="R74" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943509">74</a></sup>. The number of SNPs passing QC for each study is given in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 26</a>.</p></div><div id="S21" class="sec"><h3 id="S21title">Genome-wide-association meta-analyses</h3><p id="P38" class="p p-first">The overall analytical strategy is shown in <a href="/pmc/articles/PMC5968830/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>. We first conducted fixed-effects inverse-variance-weighted meta-analysis with METAL<sup><a href="#R75" rid="R75" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943511">75</a></sup> in each ancestral group (EUR, EAS, AFR, SAS, LAT, and other ASN), then performed meta-analysis of the ancestry-specific meta-analysis results. We constructed two versions of each meta-analysis: one with single genomic control applied and one without genomic control (for LD-score regression analysis).</p><p id="P39" class="p p-last">The EUR-specific and transancestral fixed-effects meta-analyses were further filtered for heterogeneity (<em>P</em><sub>het</sub> &lt;5.0 × 10<sup>−8</sup>) and for the number of cases included for a specific marker (&lt;50% of stroke cases were excluded). In addition, we ran a transancestral GWAS meta-analysis, using MANTRA<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943488">15</a></sup>, which was based on ancestry-specific meta-analysis results. The final MANTRA results were filtered for a MANTRA posterior-probability heterogeneity <em>P</em> &lt;0.95. SNPs with log<sub>10</sub> (BF) &gt;6 were considered to be genome-wide significant, whereas SNPs with 6 &gt; log<sub>10</sub>(BF) &gt;5 were considered to show suggestive association. We used a method based on summary statistics<sup><a href="#R76" rid="R76" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943483">76</a></sup> to estimate the variance in liability explained by each lead variant. Disease prevalence was set to 5.5% for AS, to 4.4% for AIS, and to 0.11% for IS subtype in Europeans<sup><a href="#R77" rid="R77" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943455">77</a></sup>. Disease prevalence was set to 2.97% for AIS, to 0.91% for LAS, to 0.24% for CES, and to 1.76% for SVS in East Asians (Hisayama study, J. Hata unpublished data and ref. <sup><a href="#R90" rid="R90" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943486">90</a></sup>). We used summary statistics from the Europeans-only fixed-effects meta-analysis and the East Asian–only fixed-effects meta-analysis. Genomic inflation was calculated as lambda in the GenABEL package (available through CRAN repositories). In addition, we calculated the LD-score-regression intercepts for the Europeans-only fixed-effects meta-analysis, using European LD scores.</p></div><div id="S22" class="sec"><h3 id="S22title">Shared genetic influences of individual loci on mechanistically defined stroke subtypes</h3><p id="P40" class="p p-first-last">We used gwas-pw<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943433">22</a></sup> to detect shared genetic influences of LAS, CES, and SVS, aiming to identify genetic variants that influence respective pairs of these traits. Gwas-pw estimates the PPA for four models. Model 3 is the model in which a given genomic region contains a genetic variant that influences both traits. We used the fixed-effects transancestral meta-analysis results as input, transforming results into signed <em>Z</em> scores based on the <em>P</em> value and sign of the log(OR). The chunk size (number of SNPs included in each chunk analyzed) was set automatically by using an approximately independent block file (ld-select), as provided by the software. Correlation was set to reflect the overlap in controls. We deemed the results of model 3 with a PPA &gt;0.9 significant<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943542">22</a></sup>.</p></div><div id="S23" class="sec"><h3 id="S23title">Conditional analysis</h3><p id="P41" class="p p-first-last">We used GCTA-COJO<sup><a href="#R78" rid="R78" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943467">78</a></sup> to perform conditional association analysis in each of the stroke loci in Europeans. We first fit a stepwise joint regression model including all SNPs with joint <em>P</em> &lt;5.0× 10<sup>−8</sup>. In instances in which regions included only one SNP, we fit a model including the top two SNPs from each region. The models made use of (i) summary statistics from the Europeans-only meta-analysis presented herein and (ii) genotype data for 3,291 stroke cases and 11,820 controls of North European ancestry from NINDS-SiGN as an LD reference for each region.</p></div><div id="S24" class="sec"><h3 id="S24title">Gene-based analysis</h3><p id="P42" class="p p-first-last">We performed gene-based tests by using the VEGAS approach<sup><a href="#R79" rid="R79" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943516">79</a></sup> implemented in VEGAS2 software<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943527">16</a></sup>. We used 24,769 autosomal refseq genes to perform gene-based association studies. To perform gene-based association tests, we used the 1000G phase 3 super populations African (AFR), East Asian (EAS), European (EUR), American (AMR) and South Asian (SAS) as a reference to compute the pairwise LD between variants residing within a gene. We performed gene-based tests, using the ‘-top 10’ parameter in VEGAS2, which tests enrichment of the top 10% of association <em>P</em> values within a gene. To maintain specificity while including cis-regulatory variants, we included variants located within 10 kb of a gene’s 3′ and 5′ UTRs. We performed 1 × 10<sup>6</sup> simulations to compute empirical <em>P</em> values for association with each gene. For genes with <em>P</em> &lt;1× 10<sup>−5</sup>, we increased the number of simulations to 1 × 10<sup>8</sup> to increase the accuracy of the association <em>P</em> values. For individual stroke subtypes, we performed ancestry-specific gene-based association followed by meta-analysis of gene association <em>P</em> values by using Stouffer’s method, based on sample size.</p></div><div id="S25" class="sec"><h3 id="S25title">Association of individual stroke risk variants with related vascular traits</h3><p id="P43" class="p p-first-last">We systematically explored genetic overlap with AF, CAD, cIMT, cPL, diastolic BP, systolic BP, HDL-cholesterol levels, LDL-cholesterol levels, triglyceride levels, T2D, VTE, and WMH. First, we acquired summary statistics from the appropriate consortia (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 12</a>). For each of the nonstroke phenotypes, we constructed a SNP set including the index variant of the nonstroke phenotype with <em>P</em> &lt;1.3 × 10<sup>−4</sup> plus all variants in high LD (<em>r</em><sup>2</sup> in 1000G EUR &gt;0.9 with this index variant). If the MEGASTROKE lead SNP was included in this set of SNPs, we deemed the overlap with the nonstroke phenotype to be significant. We show two different tiers: (i) variants that showed genome-wide significance in the related vascular trait (P &lt;5.0 × 10<sup>−8</sup>) and (ii) variants that were not genome-wide significant but passed Bonferroni correction (P = 1.3 × 10<sup>−4</sup>).</p></div><div id="S26" class="sec"><h3 id="S26title">Association of genetic risk scores of related vascular traits with stroke and stroke subtypes</h3><p id="P44" class="p p-first">Genetic risk scores generated from variants shown to have genome-wide association with various vascular risk factors (VTE, DBP, SBP, MAP, PP, HTN, HDL cholesterol, LDL cholesterol, triglycerides, T2D, and CAD) were used to estimate the overlap between vascular traits and stroke and its subtypes. The effect allele for each risk-factor variant was defined as the allele associated with increased risk-factor levels. The corresponding allele information, <em>β</em> coefficients and standard errors from different stroke subtypes were extracted and used as input. Association was tested with the inverse-variance weighting (IVW) method implemented as an R package gtx V 0.0.8 (available through CRAN repositories).</p><p id="P45">We further conducted sensitivity analyses, using the MR-Egger method implemented as an R package (TwoSampleMR, available through CRAN repositories)<sup><a href="#R80" rid="R80" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943454">80</a></sup>, which, unlike the IVW method, estimates the intercept term as part of the analysis. An intercept term significantly differing from zero suggests the presence of directional pleiotropy. We used a conservative significance threshold of <em>P</em> &lt;0.05 for the intercept. In the presence of directional pleiotropy, leave-one-out analysis was carried out by retesting the association of the vascular GRS with the outcome (stroke), leaving out each SNP in turn to determine whether a single SNP drives the association. We manually identified outlier SNPs that might drive the observed directional pleiotropy and then repeated the analyses (IVW and MR-Egger) after excluding the variants exhibiting directional pleiotropy.</p><p id="P46" class="p p-last">The selection of SNPs for the vascular GRS was based on literature (PubMed) searches and the GWAS catalog (<a href="http://www.ebi.ac.uk/gwas/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ebi.ac.uk/gwas/</a>), and was used to identify studies that performed GWAS of the various risk factors. The most recent and largest GWAS of each risk factor was selected, and the associated variant details were retrieved. For the GRS analysis, only independent variants (<em>r</em><sup>2</sup> &lt;0.01, based on the 1000G EUR panel) were used for the analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 27</a>). Risk-variant selection for BP traits (SBP, DBP, MAP, and PP) was further extended to studies with gene-centric chips. We used <em>β</em> coefficients extracted from the summary statistics of the International Consortium of BP GWAS<sup><a href="#R81" rid="R81" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943432">81</a>,<a href="#R82" rid="R82" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943514">82</a></sup> as weights for this GRS analysis. A P-value &lt;5.6 × 10<sup>−3</sup> correcting for nine independent phenotypes was considered significant. The number of independent vascular phenotypes, taking into account the correlation between the phenotypes considered, was estimated on the basis of individual-level data from the 3C study by using the online tool matSpDlite (<a href="http://neurogenetics.qimrberghofer.edu.au/matSpDlite/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://neurogenetics.qimrberghofer.edu.au/matSpDlite/</a>).</p></div><div id="S27" class="sec"><h3 id="S27title">Shared genetic contribution to stroke and related vascular traits at the genomewide level</h3><p id="P47" class="p p-first-last">We used LD-score regression to estimate the genetic correlation between stroke and related vascular traits<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943421">33</a>,<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943502">34</a></sup>. We conducted analyses on the European and East Asian stroke GWAS summary statistics only. Summary statistics from the GWAS meta-analyses for vascular risk factors and intermediate or related vascular phenotypes (BP, blood lipids, T2D, cIMT, cPL, AF, VTE, CAD, and WMH) were acquired from the respective consortia, as detailed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 12</a>. For LD-score regression in East Asians, we further received prepublication access to summary statistics of GWAS for blood lipids conducted in BioBank Japan <sup><a href="#R91" rid="R91" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_703109333">91</a></sup>, as described in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>. For each trait, we filtered the summary statistics to the subset of HapMap 3 SNPs to decrease the potential for bias due to poor imputation quality. Analyses were performed separately by using summary statistics from the European- and East Asian–specific meta-analysis. We used the European or East Asian LD-score files calculated from the 1000G reference panel and provided by the developers. A <em>P</em> value &lt;5.6 × 10<sup>−3</sup> correcting for nine independent phenotypes was considered significant. All analyses were performed with the ldsc package (<a href="https://github.com/bulik/ldsc/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/bulik/ldsc/</a>).</p></div><div id="S28" class="sec"><h3 id="S28title">Global epigenetic patterns at the 32 stroke risk loci</h3><p id="P48" class="p p-first-last">We used the epigwas tool<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943518">35</a></sup> to test for cell-specific enrichment in chromatin marks that have previously been shown to be phenotypically cell-type specific in ENCODE and/or RoadMap epigenome data (H3K4me1, H3K4me3, and H3K9ac)<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943512">35</a></sup>, leveraging the recent release of ENCODE/RoadMap epigenome data from 127 tissue types<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943423">36</a></sup>. Histone ChIP–seq data for narrow contiguous regions of enrichment were used to calculate the enrichment score (height of the nearest tall peak/distance to the peak) for the lead variant and proxies (<em>r</em><sup>2</sup> &gt;0.8 in the 1000G cosmopolitan panel). Significance was estimated by examining the relative proximity and specificity of the test SNP set with 10,000 sets (permutation) of matched background. In addition, Bonferroni correction for the number of chromatin marks tested was applied.</p></div><div id="S29" class="sec"><h3 id="S29title">Pathway analyses</h3><p id="P49" class="p p-first">To identify pathways overrepresented in the stroke association results, we used data-driven expression-prioritized integration for complex traits (DEPICT<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943466">37</a></sup>), IPA (<a href="https://www.qiagenbioinformatics.com/products/ingenuity-pathway-analysis/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.qiagenbioinformatics.com/products/ingenuity-pathway-analysis/</a>), and VEGAS2Pathway<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943493">39</a></sup>. DEPICT version 1 release 194 was used to identify biological pathways, tissues, and cell types enriched among suggestive associations (log<sub>10</sub>(BF) &gt;5) for any stroke and stroke subtypes in the MANTRA transancestral GWAS. Results are presented for the MANTRA transancestral analysis. We deemed DEPICT pathways with an FDR &lt;0.05 statistically significant.</p><p id="P50">IPA was conducted by using an extended list comprising 214 genes located in the boundaries defined by <em>r</em><sup>2</sup> &gt;0.5 with the lead SNP in Europeans or East Asians, or located +50 kb from the lead SNP, for all suggestive loci reaching <em>P</em> &lt;1.0 × 10<sup>−5</sup> or log<sub>10</sub>(BF) &gt;5 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 25</a>). This gene list was taken as an input for IPA using only findings from human and experimentally verified results. Otherwise, standard parameters were used for the analysis. We corrected canonical pathway <em>P</em>-value analysis with the Benjamini–Hochberg method and deemed an FDR &lt;0.05 significant.</p><p id="P51">We performed gene-wide gene-set enrichment analysis, using the VEGAS2Pathway approach<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943425">39</a></sup> to test which Biosystem terms<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943503">38</a></sup> were enriched with VEGAS2-derived gene association <em>P</em> values for stroke subtypes. VEGAS2Pathway performs a competitive gene-set enrichment test while accounting for gene density in LD blocks (or correlated association <em>P</em> values of neighboring genes), SNP density, and pathway size by using a resampling strategy.</p><p id="P52" class="p p-last">For individual stroke subtypes, we performed separate ancestry-specific geneset enrichment analysis. Next, we combined the gene-set-enrichment association <em>P</em> values across ancestries by using Stouffer’s method for sample-size-weighted combinations of <em>P</em> values. For each stroke subtype, we tested the association of 9,981 Biosystem gene-set terms.</p></div><div id="S30" class="sec"><h3 id="S30title">Fine mapping derived from credible SNP-set analyses</h3><p id="P53" class="p p-first-last">We implemented the method of Maller et al.<sup><a href="#R83" rid="R83" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943494">83</a></sup>, converting our ancestry-specific meta-analysis <em>P</em> values to Bayes factors through Wakefield’s approximation<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943476">40</a></sup> in all stroke phenotypes in the EU-only, EAS-only, and AFR-only analysis. We used all SNPs in LD with the lead SNP (<em>r</em><sup>2</sup> &gt;0.1, ancestry specific). The Bayes factors were then used to calculate posterior probabilities on the basis of the assumption of a single causal SNP in each region. For all regions, we constructed 95% credible sets of potentially causal SNPs.</p></div><div id="S31" class="sec"><h3 id="S31title">Investigation of eQTLs, pQTLs, meQTLs, and regulatory marks in different tissues</h3><p id="P54" class="p p-first">The following datasets, covering a large variety of tissue and cell types, were interrogated for eQTLs, pQTLs, and meQTLs: 
</p><ol class="enumerated" style="list-style-type:decimal" id="L1"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><p id="P55">The Genotype-Tissue Expression (GTEx-V6) project data, providing significant eQTL information from 44 postmortem tissues (449 individuals) (<a href="http://biorxiv.org/content/early/2016/09/09/074450/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://biorxiv.org/content/early/2016/09/09/074450/</a>), with significance based on a gene-specific <em>P</em>-value threshold that is permutation-adjusted for multiple SNPs per gene</p></li><li><p id="P56">The Genome-wide Repository of Associations between SNPs and Phenotypes, build 2.0 (GRASP2)<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943525">46</a>,<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943468">47</a></sup>, as well as a collected expression and epigenetic QTL database of &gt; 100 sources covering a wide range of cell and tissue types (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>), using <em>P</em> &lt;5× 10<sup>−6</sup> as a significance threshold for association with expression of a transcript in the original study</p></li><li><p id="P57">The Human Genetic Variation Database (HGVD)<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943430">48</a></sup>, providing eQTL information from peripheral-blood cells in a Japanese population (<em>n</em> = 1,208), with significance defined by FDR &lt;5%</p></li><li><p id="P58">The Biobank-based Integrative Omics Studies (BIOS), providing eQTLs from peripheral-blood RNA-seq data in 2,116 unrelated individuals<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943452">49</a></sup>, with significance defined by FDR &lt;5%</p></li><li><p id="P59">A subset of the Blueprint epigenome project<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943497">50</a></sup> with eQTL, meQTL, and histone-modification data (H3K4me1 and H3K27ac) in CD14<sup>+</sup> monocytes, CD16<sup>+</sup> neutrophils, and CD4<sup>+</sup> naive T cells from 197 individuals; these were mapped through the classical QTL association test, allele-specific-expression test, and combined haplotype test, with significance defined by FDR &lt;5%</p></li><li><p id="P60">The Stockholm-Tartu Atherosclerosis Reverse Networks Engineering Task study (STARNET)<sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943473">51</a></sup>, providing eQTL data from vascular and metabolic tissues in 600 patients with CAD, with Benjamini-Hochberg-corrected association <em>P</em> values (<em>P</em> &lt;0.05)</p></li><li><p id="P61">The aortic endothelial cell study<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943475">52</a></sup>, providing eQTL data from human aortic endothelial cells in 147 individuals, with Bonferroni multiple testing correction for the number of independent SNPs (P &lt;1.0 × 10<sup>−4</sup>)</p></li><li><p id="P62">The ARIC cohort<sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943541">53</a></sup>, providing meQTL information from peripheral blood in 794 individuals of European ancestry and 784 individuals of African American ancestry, with multiple testing correction for the number of unique CpG probes in the look-up</p></li><li><p id="P63">The Cooperative Health Research in the Region of Augsburg (KORA) cohort, with pQTL information from the human blood plasma proteome<sup><a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943440">54</a></sup>, measuring 1,124 proteins on the SomaSCAN platform in 1,000 participants; significance for each association was set at <em>P</em> &lt;5.0 × 10<sup>−8</sup></p></li></ol><p></p><p id="P64">In each of these datasets, we report the most significant cis-QTL, meQTL, or pQTL surpassing a study-specific predefined significance level or FDR, considering only QTLs in LD with the lead stroke SNP at an <em>r</em><sup>2</sup> &gt;0.8 (in 1000G, as well as queries of multiple builds of SNAP<sup><a href="#R84" rid="R84" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943431">84</a></sup> and SNiPA<sup><a href="#R85" rid="R85" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943523">85</a></sup>), thus suggesting high concordance. The results are presented grouped per tissue or cell type (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 23</a>), or per stroke risk locus (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 22</a>). In addition, we also systematically report the association of the top QTL with stroke risk and of the lead stroke risk variant with the corresponding transcript expression, methylation level, or protein level (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 23</a>).</p><p id="P65" class="p p-last">In addition, we used a subset of the Blueprint epigenome project in CD14<sup>+</sup> monocytes, CD16<sup>+</sup> neutrophils, and CD4<sup>+</sup> naive T cells from 197 individuals<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943448">50</a></sup> and Haploreg V4 (ref. <sup><a href="#R86" rid="R86" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_703109332">86</a></sup>) to annotate the lead variants and proxies for enrichment in specific histone-modification marks for the chromatin state, on the basis of ChIP-seq data from multiple cell/tissue types from ENCODE<sup><a href="#R87" rid="R87" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943462">87</a></sup> and NIH RoadMap epigenome<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943519">36</a></sup>. The results for each of the lead SNPs and its proxies are displayed in detail in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 22</a>.</p></div><div id="S32" class="sec"><h3 id="S32title">Integration of association statistics and in silico functional information in RiVIERA-beta</h3><p id="P66" class="p p-first-last">To identify the most plausible causal variants and genes, we used RiVIERA software<sup><a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943457">59</a></sup>, which jointly models the summary association statistics and the corresponding epigenetic regulatory information in a Bayesian framework to estimate the PPA. The empirical prior of a variant to be associated with the respective trait through regulatory features was generated by using the 848 tissue-specific epigenomic data in seven chromatin (H3K4me1, H3K4me3, H3K36me3, H3K27me3, H3K9me3, H3K27ac, and H3K9ac) and DNA-accessibility (DNase I) marks from the ENCODE/RoadMap epigenome data. Binary epigenomic annotation matrices of a variant overlapping the narrow peaks were generated. For inferring the causal region, RiVIERA-beta performs a repeated (<em>n</em> = 1,000) random-sampling step per locus, with the step size set to 1.0 × 10<sup>−4</sup>. Iteration is performed until convergence (acceptance rate &gt;60%) is achieved, which is critical for the accurate estimation of PPA. We generated 95% credible sets in each region on the basis of the PPA. Regional plots were generated by using the association statistics and the PPA. Epigenetic enrichment over a fixed window size (50 bp) per tissue group was generated by taking the cumulative sum of empirical prior weighted global epigenetic enrichment. Tissues were divided into 19 groups, as defined in the NIH RoadMap epigenome project.</p></div><div id="S33" class="sec"><h3 id="S33title">Scoring method</h3><p id="P67" class="p p-first-last">To prioritize the most likely biological-candidate genes, we integrated functional and biological information into an empirical score for each of the genes residing in the 32 genome-wide-significant loci. These comprised 149 genes within the region defined by an <em>r</em><sup>2</sup> &gt;0.5 in any of the 1000G European or East Asian populations or physical distances of ±50 kb from the lead SNP of the respective locus (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 25</a>). A score of 1 was assigned for being the nearest gene to the lead SNP, for containing a missense variant, for containing histone-mark H3K4me3, H3K9ac, and H3K4me1 peaks in cell types that showed significant enrichment in epigwas analysis, and for functioning as an eGene for an eQTL, meQTL, or pQTL (one point for each) in at least one study and one cell/tissue type. In addition, a score of 1 was assigned for each stroke phenotype showing evidence of being a drug-target gene in the DrugBank database (ATC-C and ATC-B01) and the Therapeutic Target Database (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 25</a>), and for overlap with biological pathways in DEPICT, IPA, or VEGAS2 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 18–20</a>).</p></div><div id="S34" class="sec"><h3 id="S34title">Drug-target gene-enrichment analysis</h3><p id="P68" class="p p-first-last">For each locus containing a variant with log<sub>10</sub>(BF) &gt;5 in the MANTRA analysis, we annotated the genes by considering LD structures (<em>r</em><sup>2</sup> &gt;0.5 in any of 1000G EUR or ASN populations) or physical distances (±50 kb) from the lead SNP of the respective locus. Drug-target genes were extracted from the DrugBank database<sup><a href="#R88" rid="R88" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943447">88</a></sup> (considering those registered as pharmacological active targets; <a href="https://www.drugbank.ca/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.drugbank.ca/</a>) and Therapeutic Target Database<sup><a href="#R89" rid="R89" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702943459">89</a></sup> (TTD; <a href="http://bidd.nus.edu.sg/group/cjttd/TTD_HOME.asp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://bidd.nus.edu.sg/group/cjttd/TTD_HOME.asp</a>), thus resulting in a list of 1,123 genes (and corresponding proteins) annotated to currently approved drugs indicated for any diseases (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 25</a>). Drugs indicated for antithrombotic therapy (<em>n</em> = 69) and cardiovascular diseases (<em>n</em> = 324) were curated from Anatomical Therapeutic Chemical (ATC) codes (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 25</a>). Enrichment of overlap between stroke-associated genes with drug targets for antithrombotic therapy and cardiovascular diseases was assessed with Fisher’s exact test.</p></div><div id="S35" class="sec"><h3 id="S35title">Life Sciences Reporting Summary</h3><p id="P69" class="p p-first-last">Further information on experimental design is available in the Life Sciences Reporting Summary.</p></div><div id="S36" class="sec sec-last"><h3 id="S36title">Data availability</h3><p id="P70" class="p p-first-last">The datasets generated and/or analyzed during the current study are available from the corresponding authors upon reasonable request.</p></div></div><div id="S37" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S37title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="d36e4379"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Information.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(6.8M, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supplementary Table 11</h4><div class="sup-box half_rhythm" id="d36e4383"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_11.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.0M, xlsx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>Supplementary Table 13</h4><div class="sup-box half_rhythm" id="d36e4387"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_13.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(43K, xlsx)</sup></div></div><div class="sec suppmat" id="SD4"><h4>Supplementary Table 15</h4><div class="sup-box half_rhythm" id="d36e4391"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_15.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(20K, xlsx)</sup></div></div><div class="sec suppmat" id="SD5"><h4>Supplementary Table 17</h4><div class="sup-box half_rhythm" id="d36e4395"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_17.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(26K, xlsx)</sup></div></div><div class="sec suppmat" id="SD6"><h4>Supplementary Table 18</h4><div class="sup-box half_rhythm" id="d36e4399"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_18.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(58K, xlsx)</sup></div></div><div class="sec suppmat" id="SD7"><h4>Supplementary Table 19</h4><div class="sup-box half_rhythm" id="d36e4403"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_19.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(6.7M, xlsx)</sup></div></div><div class="sec suppmat" id="SD8"><h4>Supplementary Table 2</h4><div class="sup-box half_rhythm" id="d36e4407"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(67K, xlsx)</sup></div></div><div class="sec suppmat" id="SD9"><h4>Supplementary Table 20</h4><div class="sup-box half_rhythm" id="d36e4411"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_20.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.9M, xlsx)</sup></div></div><div class="sec suppmat" id="SD10"><h4>Supplementary Table 21</h4><div class="sup-box half_rhythm" id="d36e4415"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_21.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(19K, xlsx)</sup></div></div><div class="sec suppmat" id="SD11"><h4>Supplementary Table 22</h4><div class="sup-box half_rhythm" id="d36e4420"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_22.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(73K, xlsx)</sup></div></div><div class="sec suppmat" id="SD12"><h4>Supplementary Table 23</h4><div class="sup-box half_rhythm" id="d36e4424"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_23.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(18K, xlsx)</sup></div></div><div class="sec suppmat" id="SD13"><h4>Supplementary Table 24</h4><div class="sup-box half_rhythm" id="d36e4428"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_24.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(25K, xlsx)</sup></div></div><div class="sec suppmat" id="SD14"><h4>Supplementary Table 25</h4><div class="sup-box half_rhythm" id="d36e4432"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_25.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.5M, xlsx)</sup></div></div><div class="sec suppmat" id="SD15"><h4>Supplementary Table 27</h4><div class="sup-box half_rhythm" id="d36e4436"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_27.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(25K, xlsx)</sup></div></div><div class="sec suppmat" id="SD16"><h4>Supplementary Table 6</h4><div class="sup-box half_rhythm" id="d36e4440"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_6.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(26K, xlsx)</sup></div></div><div class="sec suppmat" id="SD17"><h4>Supplementary Table 7</h4><div class="sup-box half_rhythm" id="d36e4444"><a href="/pmc/articles/PMC5968830/bin/NIHMS959658-supplement-Supplementary_Table_7.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(20K, xlsx)</sup></div></div></div><div id="S38" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S38title">Acknowledgments</h2><div class="sec"><p id="__p4">A full list of Acknowledgements appears in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>.</p></div></div><div id="APP1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP1title">MEGASTROKE Consortium</h2><!--/article/back/app-group/app/--><p id="P71"><strong>Rainer Malik<sup>1</sup>, Ganesh Chauhan<sup>2</sup>, Matthew Traylor<sup>3</sup>, Muralidharan Sargurupremraj<sup>4, 5</sup>, Yukinori Okada<sup>6, 7, 8</sup>, Aniket Mishra<sup>4, 5</sup>, Loes Rutten-Jacobs<sup>3</sup>, Anne-Katrin Giese<sup>9</sup>, sander W. van der Laan<sup>10</sup>, Solveig Gretarsdottir<sup>11</sup>, Christopher D. Anderson<sup>12, 13, 14</sup>, Michael Chong<sup>15</sup>, Hieab H. H. Adams<sup>16, 17</sup>, Tetsuro Ago<sup>18</sup>, Peter Almgren<sup>19</sup>, Philippe Amouyel<sup>20, 21</sup>, Hakan Ay<sup>13, 22</sup>, Traci M. Bartz<sup>23</sup>, Oscar R. Benavente<sup>24</sup>, Steve Bevan<sup>25</sup>, Giorgio B. Boncoraglio<sup>26</sup>, Robert D. Brown Jr<sup>27</sup>, Adam S. Butterworth<sup>28, 29</sup>, Caty Carrera<sup>30, 31</sup>, Cara L. Carty<sup>32, 33</sup>, Daniel i. Chasman<sup>34, 35</sup>, Wei-Min Chen<sup>36</sup>, John W. Cole<sup>37</sup>, Adolfo Correa<sup>38</sup>, ioana Cotlarciuc<sup>39</sup>, Carlos Cruchaga<sup>40, 41</sup>, John Danesh<sup>28, 42, 43, 44</sup>, Paul i. W. de Bakker<sup>45, 46</sup>, Anita L. DeStefano<sup>47, 48</sup>, Marcel den Hoed<sup>49</sup>, Qing Duan<sup>50</sup>, Stefan T. Engelter<sup>51, 52</sup>, Guido J. Falcone<sup>53, 54</sup>, Rebecca F. Gottesman<sup>55</sup>, Raji P. Grewal<sup>56</sup>, Vilmundur Gudnason<sup>57, 58</sup>, Stefan Gustafsson<sup>59</sup>, Jeffrey Haessler<sup>60</sup>, Tamara B. Harris<sup>61</sup>, Ahamad Hassan<sup>62</sup>, Aki S. Havulinna<sup>63, 64</sup>, Susan R. Heckbert<sup>65</sup>, Elizabeth G. Holliday<sup>66, 67</sup>, George Howard<sup>68</sup>, Fang-Chi Hsu<sup>69</sup>, Hyacinth i. Hyacinth<sup>70</sup>, M. Arfan ikram<sup>16</sup>, Erik ingelsson<sup>71, 72</sup>, Marguerite R. irvin<sup>73</sup>, Xueqiu Jian<sup>74</sup>, Jordi Jimenez-Conde<sup>75</sup>, Julie A. Johnson<sup>76, 77</sup>, J. Wouter Jukema<sup>78</sup>, Masahiro Kanai<sup>6, 7, 79</sup>, Keith L. Keene<sup>80, 81</sup>, Brett M. Kissela<sup>82</sup>, Dawn O. Kleindorfer<sup>82</sup>, Charles Kooperberg<sup>60</sup>, Michiaki Kubo<sup>83</sup>, Leslie A. Lange<sup>84</sup>, Carl D. Langefeld<sup>85</sup>, Claudia Langenberg<sup>86</sup>, Lenore J. Launer<sup>87</sup>, Jin-Moo Lee<sup>88</sup>, Robin Lemmens<sup>89, 90</sup>, Didier Leys<sup>91</sup>, Cathryn M. Lewis<sup>92, 93</sup>, Wei-Yu Lin<sup>28, 94</sup>, Arne G. Lindgren<sup>95, 96</sup>, Erik Lorentzen<sup>97</sup>, Patrik K. Magnusson<sup>98</sup>, Jane Maguire<sup>99</sup>, Ani Manichaikul<sup>36</sup>, Patrick F. McArdle<sup>100</sup>, James F. Meschia<sup>101</sup>, Braxton D. Mitchell<sup>100, 102</sup>, Thomas H. Mosley<sup>103, 104</sup>, Michael A. Nalls<sup>105, 106</sup>, Toshiharu Ninomiya<sup>107</sup>, Martin J. O’Donnell<sup>15, 108</sup>, Bruce M. Psaty<sup>109, 110, 111, 112</sup>, Sara L. Pulit <sup>45, 113</sup>, Kristiina Rannikmäe<sup>114, 115</sup>, Alexander P. Reiner<sup>65, 116</sup>, Kathryn M. Rexrode<sup>117</sup>, Kenneth Rice<sup>118</sup>, Stephen S. Rich<sup>36</sup>, Paul M. Ridker<sup>34, 35</sup>, Natalia S. Rost<sup>9, 13</sup>, Peter M. Rothwell<sup>119</sup>, Jerome i. Rotter<sup>120, 121</sup>, Tatjana Rundek<sup>122</sup>, Ralph L. Sacco<sup>122</sup>, Saori Sakaue<sup>7, 123</sup>, Michele M. Sale<sup>124</sup>, Veikko Salomaa<sup>63</sup>, Bishwa R. Sapkota<sup>125</sup>, Reinhold Schmidt<sup>126</sup>, Carsten O. Schmidt<sup>127</sup>, Ulf Schminke<sup>128</sup>, Pankaj Sharma<sup>39</sup>, Agnieszka Slowik<sup>129</sup>, Cathie L. M. Sudlow<sup>114, 115</sup>, Christian Tanislav<sup>130</sup>, Turgut Tatlisumak<sup>131, 132</sup>, Kent D. Taylor<sup>120, 121</sup>, Vincent N. S. Thijs<sup>133, 134</sup>, Gudmar Thorleifsson<sup>11</sup>, Unnur Thorsteinsdottir<sup>11</sup>, Steffen Tiedt<sup>1</sup>, Stella Trompet<sup>135</sup>, Christophe Tzourio<sup>5, 136, 137</sup>, Cornelia M. van Duijn<sup>138, 139</sup>, Matthew Walters<sup>140</sup>, Nicholas J. Wareham<sup>86</sup>, Sylvia Wassertheil-Smoller<sup>141</sup>, James G. Wilson<sup>142</sup>, Kerri L. Wiggins<sup>109</sup>, Qiong Yang<sup>47</sup>, Salim Yusuf<sup>15</sup>, Najaf Amin<sup>16</sup>, Hugo S. Aparicio<sup>48, 187</sup>, Donna K. Arnett<sup>188</sup>, John Attia<sup>189</sup>, Alexa S. Beiser<sup>47, 48</sup>, Claudine Berr<sup>190</sup>, Julie E. Buring<sup>34, 35</sup>, Mariana Bustamante<sup>191</sup>, Valeria Caso<sup>192</sup>, Yu-Ching Cheng<sup>193</sup>, Seung Hoan Choi<sup>48, 194</sup>, Ayesha Chowhan<sup>48, 187</sup>, Natalia Cullell<sup>31</sup>, Jean-Franęois Dartigues<sup>195, 196</sup>, Hossein Delavaran<sup>95, 96</sup>, Pilar Delgado<sup>197</sup>, Marcus Dörr<sup>198, 199</sup>, Gunnar Engström<sup>19</sup>, ian Ford<sup>200</sup>, Wander S. Gurpreet<sup>201</sup>, Anders Hamsten</strong><sup>202,</sup><strong><sup>203</sup>, Laura Heitsch<sup>204</sup>, Atsushi Hozawa<sup>205</sup>, Laura ibanez<sup>206</sup>, Andreea ilinca<sup>95, 96</sup>, Martin ingelsson<sup>207</sup>, Motoki iwasaki<sup>208</sup>, Rebecca D. Jackson<sup>209</sup>, Katarina Jood<sup>210</sup>, Pekka Jousilahti<sup>63</sup>, Sara Kaffashian<sup>3</sup>, Lalit Kalra<sup>211</sup>, Masahiro Kamouchi<sup>212</sup>, Takanari Kitazono<sup>213</sup>, Olafur Kjartansson<sup>214</sup>, Manja Kloss<sup>215</sup>, Peter J. Koudstaal<sup>216</sup>, Jerzy Krupinski<sup>217</sup>, Daniel L. Labovitz<sup>218</sup>, Cathy C. Laurie<sup>118</sup>, Christopher R. Levi<sup>219</sup>, Linxin Li<sup>220</sup>, Lars Lind<sup>221</sup>, Cecilia M. Lindgren<sup>222, 223</sup>, Vasileios Lioutas</strong><sup>48,</sup><strong><sup>224</sup>, Yong Mei Liu<sup>225</sup>, Oscar L. Lopez<sup>226</sup>, Hirata Makoto<sup>227</sup>, Nicolas Martinez-Majander<sup>172</sup>, Koichi Matsuda<sup>227</sup>, Naoko Minegishi<sup>205</sup>, Joan Montaner<sup>228</sup>, Andrew P. Morris<sup>229, 230</sup>, Elena Muiño<sup>31</sup>, Martina Müller-Nurasyid</strong><sup>231, 232,</sup><strong><sup>233</sup>, Bo Norrving<sup>95, 96</sup>, Soichi Ogishima<sup>205</sup>, Eugenio A. Parati<sup>234</sup>, Leema Reddy Peddareddygari<sup>56</sup>, Nancy L. Pedersen<sup>98, 235</sup>, Joanna Pera<sup>129</sup>, Markus Perola</strong><sup>63,</sup><strong><sup>236</sup>, Alessandro Pezzini<sup>237</sup>, Silvana Pileggi<sup>238</sup>, Raquel Rabionet<sup>239</sup>, iolanda Riba-Llena<sup>30</sup>, Marta Ribases<sup>240</sup>, Jose R. Romero<sup>48, 187</sup>, Jaume Roquer</strong><sup>241,</sup><strong><sup>242</sup>, Anthony G. Rudd<sup>243, 244</sup>, Antti-Pekka Sarin<sup>245, 246</sup>, Ralhan Sarju<sup>201</sup>, Chloe Sarnowski<sup>47, 48</sup>, Makoto Sasaki<sup>247</sup>, Claudia L. Satizabal<sup>48, 187</sup>, Mamoru Satoh<sup>247</sup>, Naveed Sattar<sup>248</sup>, Norie Sawada<sup>208</sup>, Gerli Sibolt<sup>172</sup>, Asgeir Sigurdsson<sup>249</sup>, Albert Smith<sup>250</sup>, Kenji Sobue<sup>247</sup>, Carolina Soriano-Tarraga<sup>242</sup>, Tara Stanne<sup>251</sup>, O. Colin Stine<sup>252</sup>, David J. Stott<sup>253</sup>, Konstantin Strauch<sup>231, 254</sup>, Takako Takai<sup>205</sup>, Hideo Tanaka<sup>255, 256</sup>, Kozo Tanno<sup>247</sup>, Alexander Teumer<sup>257</sup>, Liisa Tomppo<sup>172</sup>, Nuria P. Torres-Aguila<sup>31</sup>, Emmanuel Touze</strong><sup>258,</sup><strong><sup>259</sup>, Shoichiro Tsugane<sup>208</sup>, Andre G. Uitterlinden<sup>260</sup>, Einar M. Valdimarsson<sup>261</sup>, Sven J. van der Lee<sup>16</sup>, Henry Völzke<sup>257</sup>, Kenji Wakai<sup>255</sup>, David Weir<sup>262</sup>, Stephen R. Williams<sup>263</sup>, Charles D. A. Wolfe<sup>243, 244</sup>, Quenna Wong<sup>118</sup>, Huichun Xu<sup>193</sup>, Taiki Yamaji<sup>208</sup>, Dharambir K. Sanghera<sup>125, 169, 170</sup>, Olle Melander<sup>19</sup>, Christina Jern<sup>171</sup>, Daniel Strbian<sup>172, 173</sup>, israel Fernandez-Cadenas<sup>30, 31</sup>, W. T. Longstreth Jr<sup>65, 174</sup>, Arndt Rolfs<sup>175</sup>, Jun Hata<sup>107</sup>, Daniel Woo<sup>82</sup>, Jonathan Rosand<sup>12, 13, 14</sup>, Guillaume Pare<sup>15</sup>, Jemma C. Hopewell<sup>176</sup>, Danish Saleheen<sup>177</sup>, Kari Stefansson<sup>11, 178</sup>, Bradford B. Worrall<sup>179</sup>, Steven J. Kittner<sup>37</sup>, Sudha Seshadri<sup>48, 180</sup>, Myriam Fornage<sup>74, 181</sup>, Hugh S. Markus<sup>3</sup>, Joanna M. M. Howson<sup>28</sup>, Yoichiro Kamatani<sup>6, 182</sup>, Stephanie Debette</strong><sup>4, 5</sup><strong>and Martin Dichgans</strong><sup>1, 183, 184</sup></p><p id="P72"><sup>187</sup>Boston University School of Medicine, Boston, MA, USA. <sup>188</sup>University of Kentucky College of Public Health, Lexington, KY, USA. <sup>189</sup>University of Newcastle and Hunter Medical Research Institute, New Lambton, New South Wales, Australia. <sup>190</sup>INSERM, U1061, Universite Montpellier, Montpellier, France. <sup>191</sup>Centre for Research in Environmental Epidemiology, Barcelona, Spain. <sup>192</sup>Department of Neurology, Universita degli Studi di Perugia, Umbria, Italy. <sup>193</sup>Department of Medicine, University of Maryland School of Medicine, Baltimore, MD, USA. <sup>194</sup>Broad Institute, Cambridge, MA, USA. <sup>195</sup>Bordeaux Population Health Research Center, INSERM, UMR 1219, Université Bordeaux, Bordeaux, France. <sup>196</sup>Department of Neurology, Memory Clinic, Bordeaux University Hospital, Bordeaux, France. <sup>197</sup>Neurovascular Research Laboratory. Vall d’Hebron Institut of Research, Neurology and Medicine Departments-Universitat Autònoma de Barcelona. Vall d’Hebrón Hospital, Barcelona, Spain. <sup>198</sup>Department of Internal Medicine B, University Medicine Greifswald, Greifswald, Germany. <sup>199</sup>DZHK, Greifswald, Germany. <sup>200</sup>Robertson Center for Biostatistics, University of Glasgow, Glasgow, UK. <sup>201</sup>Hero DMC Heart Institute, Dayanand Medical College &amp; Hospital, Ludhiana, India. <sup>202</sup>Atherosclerosis Research Unit, Department of Medicine Solna, Karolinska Institutet, Stockholm, Sweden. <sup>203</sup>Karolinska Institutet, Stockholm, Sweden. <sup>204</sup>Division of Emergency Medicine, and Department of Neurology, Washington University School of Medicine, St. Louis, MO, USA. <sup>205</sup>Tohoku Medical Megabank Organization, Sendai, Japan. <sup>206</sup>Department of Psychiatry, Washington University School of Medicine, St. Louis, MO, USA. <sup>207</sup>Department of Public Health and Caring Sciences/Geriatrics, Uppsala University, Uppsala, Sweden. <sup>208</sup>Epidemiology and Prevention Group, Center for Public Health Sciences, National Cancer Center, Tokyo, Japan. <sup>209</sup>Department of Internal Medicine and the Center for Clinical and Translational Science, Ohio State University, Columbus, OH, USA. <sup>210</sup>Institute of Neuroscience and Physiology, Sahlgrenska Academy at University of Gothenburg, Goteborg, Sweden. <sup>211</sup>Department of Basic and Clinical Neurosciences, King’s College London, London, UK. <sup>212</sup>Department of Health Care Administration and Management, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Japan. <sup>213</sup>Department of Medicine and Clinical Science, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Japan. <sup>214</sup>Departments of Neurology &amp; Radiology, Landspitali National University Hospital, Reykjavik, Iceland. <sup>215</sup>Department of Neurology, Heidelberg University Hospital, Heidelberg, Germany. <sup>216</sup>Department of Neurology, Erasmus University Medical Center, Rotterdam, the Netherlands. <sup>217</sup>Hospital Universitari Mutua Terrassa, Terrassa (Barcelona), Spain. <sup>218</sup>Montefiore Medical Center, Albert Einstein College of Medicine, New York, NY, USA. <sup>219</sup>John Hunter Hospital, Hunter Medical Research Institute and University of Newcastle, Newcastle, New South Wales, Australia. <sup>220</sup>Centre for Prevention of Stroke and Dementia, Nuffield Department of Clinical Neurosciences, University of Oxford, Oxford, UK. <sup>221</sup>Department of Medical Sciences, Uppsala University, Uppsala, Sweden. <sup>222</sup>Genetic and Genomic Epidemiology Unit, Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. <sup>223</sup>Wellcome Trust Centre for Human Genetics, Oxford, UK. <sup>224</sup>Beth Israel Deaconess Medical Center, Boston, MA, USA. <sup>225</sup>Wake Forest School of Medicine, Wake Forest, NC, USA. <sup>226</sup>Department of Neurology, University of Pittsburgh, Pittsburgh, PA, USA. <sup>227</sup>BioBank Japan, Laboratory of Clinical Sequencing, Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, University of Tokyo, Tokyo, Japan. <sup>228</sup>Neurovascular Research Laboratory, Vall d’Hebron Institut of Research, Neurology and Medicine Departments-Universitat Autònoma de Barcelona, Vall d’Hebrón Hospital, Barcelona, Spain. <sup>229</sup>Department of Biostatistics, University of Liverpool, Liverpool, UK. <sup>230</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK. <sup>231</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München-German Research Center for Environmental Health, Neuherberg, Germany. <sup>232</sup>Department of Medicine I, Ludwig-Maximilians-Universität, Munich, Germany. <sup>233</sup>DZHK (German Centre for Cardiovascular Research), partner site Munich Heart Alliance, Munich, Germany. <sup>234</sup>Department of Cerebrovascular Diseases, Fondazione IRCCS Istituto Neurologico ‘Carlo Besta’, Milan, Italy. <sup>235</sup>MEB, Karolinska Institutet, Stockholm, Sweden. <sup>236</sup>Estonian Genome Center, University of Tartu, Tartu, Estonia. <sup>237</sup>Department of Clinical and Experimental Sciences, Neurology Clinic, University of Brescia, Brescia, Italy. <sup>238</sup>Translational Genomics Unit, Department of Oncology, IRCCS Istituto di Ricerche Farmacologiche Mario Negri, Milan, Italy. <sup>239</sup>Department of Genetics, Microbiology and Statistics, University of Barcelona, Barcelona, Spain. <sup>240</sup>Psychiatric Genetics Unit, Group of Psychiatry, Mental Health and Addictions, Vall d’Hebron Research Institute (VHIR), Universitat Autonoma de Barcelona, Biomedical Network Research Centre on Mental Health (CIBERSAM), Barcelona, Spain. <sup>241</sup>Department of Neurology, IMIM–Hospital del Mar, and Universitat Autonoma de Barcelona, Barceloina, Spain. <sup>242</sup>IMIM (Hospital del Mar Medical Research Institute), Barcelona, Spain. <sup>243</sup>National Institute for Health Research Comprehensive Biomedical Research Centre, Guy’s &amp; St. Thomas’ NHS Foundation Trust and King’s College London, London, UK. <sup>244</sup>Division of Health and Social Care Research, King’s College London, London, UK. <sup>245</sup>FIMM-Institute for Molecular Medicine Finland, Helsinki, Finland. <sup>246</sup>THL-National Institute for Health and Welfare, Helsinki, Finland. <sup>247</sup>Iwate Tohoku Medical Megabank Organization, Iwate Medical University, Iwate, Japan. <sup>248</sup>BHF Glasgow Cardiovascular Research Centre, Faculty of Medicine, Glasgow, UK. <sup>249</sup>deCODE Genetics/Amgen, Inc., Reykjavik, Iceland. <sup>250</sup>Icelandic Heart Association, Reykjavik, Iceland. <sup>251</sup>Institute of Biomedicine, Sahlgrenska Academy at University of Gothenburg, Goteborg, Sweden. <sup>252</sup>Department of Epidemiology, University of Maryland School of Medicine, Baltimore, MD, USA. <sup>253</sup>Institute of Cardiovascular and Medical Sciences, Faculty of Medicine, University of Glasgow, Glasgow, UK. <sup>254</sup>IBE, Faculty of Medicine, LMU Munich, Munich, Germany. <sup>255</sup>Division of Epidemiology and Prevention, Aichi Cancer Center Research Institute, Nagoya, Japan. <sup>256</sup>Department of Epidemiology, Nagoya University Graduate School of Medicine, Nagoya, Japan. <sup>257</sup>Institute for Community Medicine, SHIP-KEF, University Medicine Greifswald, Greifswald, Germany. <sup>258</sup>Department of Neurology, Caen University Hospital, Caen, France. <sup>259</sup>University of Caen Normandy, Caen, France. <sup>260</sup>Department of Internal Medicine, Erasmus University Medical Center, Rotterdam, the Netherlands. <sup>261</sup>Landspitali University Hospital, Reykjavik, Iceland. <sup>262</sup>Survey Research Center, University of Michigan, Ann Arbor, MI, USA. <sup>263</sup>Department of Neurology, University of Virginia, Charlottesville, VA, USA.</p></div><div id="APP2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP2title">Author contributions</h2><!--/article/back/app-group/app/--><p id="P73">Writing and editing the manuscript: R.M., G.C., M.T., M.S., Y.O., S.D., and M.D. Study design/conception: R.M., M.D., S.D., B.M.P., G.J.F., J.W.J., J.I.R., J.G.W, M.F., H.I.Y., C.J., S. Seshadri, WT.L., B.B.W, B.D.M., S.J.K., H.S.M., J.D., J.R., K.S., and O.M. Statistical analysis: A.-K.G., G.J.F., M.F., C.D.L., Y.O., E.L., B.R.S., R.M., M.S., M.T., A. Mishra, E.G.H., C.D.A., T.M.B., C. Carrera, I.C., W-Y.L., S.L.P., K. Rannikmäe, K. Rice, S. Tiedt, J.C.C., A.D.J., P.I.W.d.B., S.W.v.d.L., P. Almgren, S. Gretarsdottir, and F.T. Sample/phenotype contribution: M.D., S.D., C.D.A., C. Cruchaga, I.C., H.I.H., J.W.J., N. S.R., A.S.B., A.C., A.S., A.S.H., A.P.R., A.L.D., A. Rolfs, A. Ruusalepp, A.G.L., A. Manichaikul, B.M.K., C.L.C., C.R., C.K., C. Tanislav, C. Tzourio, C.M.v.D., D.I.C., D.W, D.A.T., D.O.K., D.K.S., D.L., E.S.T., E.E.S., E.I., F.-C.H., G.P., H.A., H.H.H.A., H.S.M., I.E.C., J. Haessler, J. He, J. Hata, J.F.M., J.S.K., J.-M.L., J.D., J.W.C., J.R., J.J-C., J.A.J., K.S., K.M.R., K.L.K., K.L.W., L.J.L., L.A.L., M.A.N., M.A.I., M.d.H., M.R.I., M.J.O., M. Kanai, M. Kubo, M.W, M.M.S., N.J.W, N.K., O.R.B., P.F.M., P.T.E., P.K.M., P.E., P. Amouyel, P.v.d.H., Q.D., Q.Y., R.P.G., R.L.S., R.F.G., R.S., S.Y., S.K., S.T.E., S.B., S.A.L., S.J.K., S.R.H., S.W-S., T.B.H., T.R., T.H.M., T.P., T.T., U.S., U.T., V.C., V.G., W-M.C., V.N.S.T., X.J., B.M.P., J.I.R., J.G.W, O.M., C.J., J.C.H., S. Seshadri, T.A., G.B.B., R. D.B., A.H., N.L.S., R.L., C.M.L., T.N., P. M. Ridker, P. M. Rothwell, V.S., C.O.S., P.S., C.L.M.S., K.D.T., M. Civelek, D. Saleheen, D. Strbian, S. Sakaue, S. Gustafsson, S. Tiedt, S. Trompet, and I.F.-C. Critical revision of article: R.M., M.D., S.D., B.M.P., C.J., J.I.R., O. M., S. Seshadri, G.J.F., J.W.J., W.T.L., C.D.A., D. Strbian, E.G.H., I.F.-C., S. Tiedt, C.L.M.S., C.O.S., C. Cruchaga, G.B.B., I.C., J.C.B., J. Hata, K. Rice, S.L.P., N.S.R., S.S.R., T.A., T.N., J.M.M.H., T.M.B., and V.S. Supervision: M.D., S.D., C.D.A., J.M.M.H, J.I.R., S. Seshadri, C.M.L., C.L.M.S., J.W.J., V.S., and J.C.B. GWAS analyses: R.M., G.C., M.T., S. Gretarsdottir, G.T., J. Hata, A.K.G., M. Chong, J.L.M.B., C. Carrera, A.H., G.J.F., and Y.K. Functional annotation: M.S., A. Mishra, R.M., G.C., M.T., L.R.-J., and A.K.G. Gene-based analysis: A. Mishra. Pathway analyses: A. Mishra, R.M., M. Chong, and K. Rice. Drug-target analysis: Y.O. Scoring method: M.S., R.M., S.D., and M.D. wGRS analysis: M.S. and R.M. LD-score regression analysis: R.M., M.S., and Y.K. Credible-SNP-set analysis: R.M., G.C., and M.S. Data for GWAS analysis, cross-phenotype analysis or QTL analysis: AFGen Consortium, Cohorts for Heart and AgingResearch in Genomic Epidemiology (CHARGE) Consortium, iGEN-BP Consortium, INVENT Consortium, STARNET, and Biobank Japan Cooperative Hospital Group. Consortia providing stroke data: COMPASS Consortium, EPIC-CVD Consortium, EPIC-InterAct Consortium, ISGC, METASTROKE Consortium, Neurology Working Group of the CHARGE Consortium, NINDS-SiGN, UK Young Lacunar DNA Study, and MEGASTROKE Consortium. The views expressed in this manuscript are those of the authors and do not necessarily represent the views of the National Heart, Lung, and Blood Institute or the National Institute of Neurological Disorders and Stroke.</p></div><div id="idm140458731278816" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140458731278816title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN6"></p><p id="__p5" class="p p-first"><strong>Competing interests</strong></p><p id="__p6" class="p p-last">S. Gretarsdottir, G.T., U.T., and K.S. are all employees of deCODE Genetics/Amgen, Inc. M.A.N. is an employee of Data Tecnica International. P.T.E. is the PI on a grant from Bayer HealthCare to the Broad Institute, focused on the genetics and therapeutics of atrial fibrillation. S.A.L. receives sponsored research support from Bayer HealthCare, Biotronik, and Boehringer Ingelheim, and has consulted for St. Jude Medical and Quest Diagnostics. E.I. is a scientific advisor for Precision Wellness, Cellink and Olink Proteomics for work unrelated to the present project. B.M.P. serves on the DSMB of a clinical trial funded by Zoll LifeCor and on the Steering Committee of the Yale Open Data Access Project funded by Johnson &amp; Johnson. The remaining authors have no disclosures.</p><p></p><p class="fn sec" id="FN7"></p><p id="__p7" class="p p-first-last"><strong>Supplementary information</strong> is available for this paper at <a href="https://doi.org/10.1038/s41588-018-0058-3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://doi.org/10.1038/s41588-018-0058-3</a>.</p><p></p><p class="fn sec" id="FN8"></p><p id="__p8" class="p p-first-last"><strong>Publisher’s note:</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p><p></p></div></div><div id="idm140458737761904" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140458737761904title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">GBD 2015 DALYs and HALE Collaborators. Global regional and national disability-adjusted life-years (DALYs) for 315 diseases injuries and healthy life expectancy (HALE) 1990–2015: a systematic analysis for the Global Burden of Disease Study 2015. <span><span class="ref-journal">Lancet. </span>2016;<span class="ref-vol">388</span>:1603–1658.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5388857/">PMC free article</a>]</span> [<a href="/pubmed/27733283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Global+regional+and+national+disability-adjusted+life-years+(DALYs)+for+315+diseases+injuries+and+healthy+life+expectancy+(HALE)+1990–2015:+a+systematic+analysis+for+the+Global+Burden+of+Disease+Study+2015&amp;volume=388&amp;publication_year=2016&amp;pages=1603-1658&amp;pmid=27733283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">GBD 2015 Mortality and Causes of Death Collaborators. Global, regional, and national life expectancy, all-cause mortality cause-specific mortality for 249 causes of death 1980–2015: a systematic analysis for the Global Burden of Disease Study 2015. <span><span class="ref-journal">Lancet. </span>2016;<span class="ref-vol">388</span>:1459–1544.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5388903/">PMC free article</a>]</span> [<a href="/pubmed/27733281" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Global,+regional,+and+national+life+expectancy,+all-cause+mortality+cause-specific+mortality+for+249+causes+of+death+1980–2015:+a+systematic+analysis+for+the+Global+Burden+of+Disease+Study+2015&amp;volume=388&amp;publication_year=2016&amp;pages=1459-1544&amp;pmid=27733281&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Gudbjartsson DF, et al.  Variants conferring risk of atrial fibrillation on chromosome 4q25. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">448</span>:353–357.</span> [<a href="/pubmed/17603472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Variants+conferring+risk+of+atrial+fibrillation+on+chromosome+4q25&amp;author=DF+Gudbjartsson&amp;volume=448&amp;publication_year=2007&amp;pages=353-357&amp;pmid=17603472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Gudbjartsson DF, et al.  A sequence variant in <em>ZFHX3</em> on 16q22 associates with atrial fibrillation and ischemic stroke. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:876–878.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2740741/">PMC free article</a>]</span> [<a href="/pubmed/19597491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+sequence+variant+in+ZFHX3+on+16q22+associates+with+atrial+fibrillation+and+ischemic+stroke&amp;author=DF+Gudbjartsson&amp;volume=41&amp;publication_year=2009&amp;pages=876-878&amp;pmid=19597491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">International Stroke Genetics Consortium (ISGC) et al.  Genome-wide association study identifies a variant in HDAC9 associated with large vessel ischemic stroke. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:328–333.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3303115/">PMC free article</a>]</span> [<a href="/pubmed/22306652" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+a+variant+in+HDAC9+associated+with+large+vessel+ischemic+stroke&amp;volume=44&amp;publication_year=2012&amp;pages=328-333&amp;pmid=22306652&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Woo D, et al.  Meta-analysis of genome-wide association studies identifies 1q22 as a susceptibility locus for intracerebral hemorrhage. <span><span class="ref-journal">Am J Hum Genet. </span>2014;<span class="ref-vol">94</span>:511–521.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3980413/">PMC free article</a>]</span> [<a href="/pubmed/24656865" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Meta-analysis+of+genome-wide+association+studies+identifies+1q22+as+a+susceptibility+locus+for+intracerebral+hemorrhage&amp;author=D+Woo&amp;volume=94&amp;publication_year=2014&amp;pages=511-521&amp;pmid=24656865&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Kilarski LL, et al.  Meta-analysis in more than 17,900 cases of ischemic stroke reveals a novel association at 12q24.12. <span><span class="ref-journal">Neurology. </span>2014;<span class="ref-vol">83</span>:678–685.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4150131/">PMC free article</a>]</span> [<a href="/pubmed/25031287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Meta-analysis+in+more+than+17,900+cases+of+ischemic+stroke+reveals+a+novel+association+at+12q24.12&amp;author=LL+Kilarski&amp;volume=83&amp;publication_year=2014&amp;pages=678-685&amp;pmid=25031287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Traylor M, et al.  A novel <em>MMP12</em> locus is associated with large artery atherosclerotic stroke using a genome-wide age-at-onset informed approach. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004469.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4117446/">PMC free article</a>]</span> [<a href="/pubmed/25078452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+novel+MMP12+locus+is+associated+with+large+artery+atherosclerotic+stroke+using+a+genome-wide+age-at-onset+informed+approach&amp;author=M+Traylor&amp;volume=10&amp;publication_year=2014&amp;pages=e1004469&amp;pmid=25078452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">NINDS, Stroke Genetics Network (SiGN) &amp; International Stroke Genetics Consortium (ISGC) Loci associated with ischaemic stroke and its subtypes (SiGN): a genome-wide association study. <span><span class="ref-journal">Lancet Neurol. </span>2016;<span class="ref-vol">15</span>:174–184.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4912948/">PMC free article</a>]</span> [<a href="/pubmed/26708676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Loci+associated+with+ischaemic+stroke+and+its+subtypes+(SiGN):+a+genome-wide+association+study&amp;volume=15&amp;publication_year=2016&amp;pages=174-184&amp;pmid=26708676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Neurology Working Group of the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium, the Stroke Genetics Network (SiGN) &amp; the International Stroke Genetics Consortium (ISGC) Identification of additional risk loci for stroke small vessel disease: a meta-analysis of genome-wide association studies. <span><span class="ref-journal">Lancet Neurol. </span>2016;<span class="ref-vol">15</span>:695–707.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4943223/">PMC free article</a>]</span> [<a href="/pubmed/27068588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Identification+of+additional+risk+loci+for+stroke+small+vessel+disease:+a+meta-analysis+of+genome-wide+association+studies&amp;volume=15&amp;publication_year=2016&amp;pages=695-707&amp;pmid=27068588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Malik R, et al.  Low-frequency and common genetic variation in ischemic stroke: the METASTROKE collaboration. <span><span class="ref-journal">Neurology. </span>2016;<span class="ref-vol">86</span>:1217–1226.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4818561/">PMC free article</a>]</span> [<a href="/pubmed/26935894" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Low-frequency+and+common+genetic+variation+in+ischemic+stroke:+the+METASTROKE+collaboration&amp;author=R+Malik&amp;volume=86&amp;publication_year=2016&amp;pages=1217-1226&amp;pmid=26935894&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Traylor M, et al.  Genetic variation at 16q24.2 is associated with small vessel stroke. <span><span class="ref-journal">Ann Neurol. </span>2017;<span class="ref-vol">81</span>:383–394.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5366092/">PMC free article</a>]</span> [<a href="/pubmed/27997041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Genetic+variation+at+16q24.2+is+associated+with+small+vessel+stroke&amp;author=M+Traylor&amp;volume=81&amp;publication_year=2017&amp;pages=383-394&amp;pmid=27997041&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Williams FM, et al.  Ischemic stroke is associated with the <em>ABO</em> locus: the EuroCLOT study. <span><span class="ref-journal">Ann Neurol. </span>2013;<span class="ref-vol">73</span>:16–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3582024/">PMC free article</a>]</span> [<a href="/pubmed/23381943" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Ischemic+stroke+is+associated+with+the+ABO+locus:+the+EuroCLOT+study&amp;author=FM+Williams&amp;volume=73&amp;publication_year=2013&amp;pages=16-31&amp;pmid=23381943&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">1000 Genomes Project Consortium et al.  A global reference for human genetic variation. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">526</span>:68–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4750478/">PMC free article</a>]</span> [<a href="/pubmed/26432245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+global+reference+for+human+genetic+variation&amp;volume=526&amp;publication_year=2015&amp;pages=68-74&amp;pmid=26432245&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Morris AP. Transethnic meta-analysis of genomewide association studies. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>:809–822.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3460225/">PMC free article</a>]</span> [<a href="/pubmed/22125221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Transethnic+meta-analysis+of+genomewide+association+studies&amp;author=AP+Morris&amp;volume=35&amp;publication_year=2011&amp;pages=809-822&amp;pmid=22125221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Mishra A, Macgregor S. VEGAS2: software for more flexible gene-based testing. <span><span class="ref-journal">Twin Res Hum Genet. </span>2015;<span class="ref-vol">18</span>:86–91.</span> [<a href="/pubmed/25518859" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res+Hum+Genet&amp;title=VEGAS2:+software+for+more+flexible+gene-based+testing&amp;author=A+Mishra&amp;author=S+Macgregor&amp;volume=18&amp;publication_year=2015&amp;pages=86-91&amp;pmid=25518859&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Traylor M, et al.  Genome-wide meta-analysis of cerebral white matter hyperintensities in patients with stroke. <span><span class="ref-journal">Neurology. </span>2016;<span class="ref-vol">86</span>:146–153.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4731688/">PMC free article</a>]</span> [<a href="/pubmed/26674333" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Genome-wide+meta-analysis+of+cerebral+white+matter+hyperintensities+in+patients+with+stroke&amp;author=M+Traylor&amp;volume=86&amp;publication_year=2016&amp;pages=146-153&amp;pmid=26674333&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Hara K, et al.  Association of <em>HTRA1</em> mutations and familial ischemic cerebral small-vessel disease. <span><span class="ref-journal">N Engl J Med. </span>2009;<span class="ref-vol">360</span>:1729–1739.</span> [<a href="/pubmed/19387015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Association+of+HTRA1+mutations+and+familial+ischemic+cerebral+small-vessel+disease&amp;author=K+Hara&amp;volume=360&amp;publication_year=2009&amp;pages=1729-1739&amp;pmid=19387015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Verdura E, et al.  Heterozygous <em>HTRA1</em> mutations are associated with autosomal dominant cerebral small vessel disease. <span><span class="ref-journal">Brain. </span>2015;<span class="ref-vol">138</span>:2347–2358.</span> [<a href="/pubmed/26063658" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Heterozygous+HTRA1+mutations+are+associated+with+autosomal+dominant+cerebral+small+vessel+disease&amp;author=E+Verdura&amp;volume=138&amp;publication_year=2015&amp;pages=2347-2358&amp;pmid=26063658&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Gould DB, et al.  Role of <em>COL4A1</em> in small-vessel disease and hemorrhagic stroke. <span><span class="ref-journal">N Engl J Med. </span>2006;<span class="ref-vol">354</span>:1489–1496.</span> [<a href="/pubmed/16598045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Role+of+COL4A1+in+small-vessel+disease+and+hemorrhagic+stroke&amp;author=DB+Gould&amp;volume=354&amp;publication_year=2006&amp;pages=1489-1496&amp;pmid=16598045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Jeanne M, et al.  <em>COL4A2</em> mutations impair COL4A1 and COL4A2 secretion and cause hemorrhagic stroke. <span><span class="ref-journal">Am J Hum Genet. </span>2012;<span class="ref-vol">90</span>:91–101.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3257894/">PMC free article</a>]</span> [<a href="/pubmed/22209247" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=COL4A2+mutations+impair+COL4A1+and+COL4A2+secretion+and+cause+hemorrhagic+stroke&amp;author=M+Jeanne&amp;volume=90&amp;publication_year=2012&amp;pages=91-101&amp;pmid=22209247&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Pickrell JK, et al.  Detection and interpretation of shared genetic influences on 42 human traits. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:709–717.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5207801/">PMC free article</a>]</span> [<a href="/pubmed/27182965" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Detection+and+interpretation+of+shared+genetic+influences+on+42+human+traits&amp;author=JK+Pickrell&amp;volume=48&amp;publication_year=2016&amp;pages=709-717&amp;pmid=27182965&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Lubitz SA, et al.  Independent susceptibility markers for atrial fibrillation on chromosome 4q25. <span><span class="ref-journal">Circulation. </span>2010;<span class="ref-vol">122</span>:976–984.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2978508/">PMC free article</a>]</span> [<a href="/pubmed/20733104" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Independent+susceptibility+markers+for+atrial+fibrillation+on+chromosome+4q25&amp;author=SA+Lubitz&amp;volume=122&amp;publication_year=2010&amp;pages=976-984&amp;pmid=20733104&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Kato N, et al.  Trans-ancestry genome-wide association study identifies 12 genetic loci influencing blood pressure and implicates a role for DNA methylation. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:1282–1293.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4719169/">PMC free article</a>]</span> [<a href="/pubmed/26390057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Trans-ancestry+genome-wide+association+study+identifies+12+genetic+loci+influencing+blood+pressure+and+implicates+a+role+for+DNA+methylation&amp;author=N+Kato&amp;volume=47&amp;publication_year=2015&amp;pages=1282-1293&amp;pmid=26390057&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Surakka I, et al.  The impact of low-frequency and rare variants on lipid levels. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:589–597.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4757735/">PMC free article</a>]</span> [<a href="/pubmed/25961943" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+impact+of+low-frequency+and+rare+variants+on+lipid+levels&amp;author=I+Surakka&amp;volume=47&amp;publication_year=2015&amp;pages=589-597&amp;pmid=25961943&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Morris AP, et al.  Large-scale association analysis provides insights into the genetic architecture and pathophysiology of type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:981–990.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3442244/">PMC free article</a>]</span> [<a href="/pubmed/22885922" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Large-scale+association+analysis+provides+insights+into+the+genetic+architecture+and+pathophysiology+of+type+2+diabetes&amp;author=AP+Morris&amp;volume=44&amp;publication_year=2012&amp;pages=981-990&amp;pmid=22885922&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Bis JC, et al.  Meta-analysis of genome-wide association studies from the CHARGE consortium identifies common variants associated with carotid intima media thickness and plaque. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:940–947.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3257519/">PMC free article</a>]</span> [<a href="/pubmed/21909108" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+studies+from+the+CHARGE+consortium+identifies+common+variants+associated+with+carotid+intima+media+thickness+and+plaque&amp;author=JC+Bis&amp;volume=43&amp;publication_year=2011&amp;pages=940-947&amp;pmid=21909108&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Verhaaren BF, et al.  Multiethnic genome-wide association study of cerebral white matter hyperintensities on MRI. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2015;<span class="ref-vol">8</span>:398–409.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4427240/">PMC free article</a>]</span> [<a href="/pubmed/25663218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Multiethnic+genome-wide+association+study+of+cerebral+white+matter+hyperintensities+on+MRI&amp;author=BF+Verhaaren&amp;volume=8&amp;publication_year=2015&amp;pages=398-409&amp;pmid=25663218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Sinner MF, et al.  Integrating genetic, transcriptional, and functional analyses to identify 5 novel genes for atrial fibrillation. <span><span class="ref-journal">Circulation. </span>2014;<span class="ref-vol">130</span>:1225–1235.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4190011/">PMC free article</a>]</span> [<a href="/pubmed/25124494" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Integrating+genetic,+transcriptional,+and+functional+analyses+to+identify+5+novel+genes+for+atrial+fibrillation&amp;author=MF+Sinner&amp;volume=130&amp;publication_year=2014&amp;pages=1225-1235&amp;pmid=25124494&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Germain M, et al.  Meta-analysis of 65,734 individuals identifies <em>TSPAN15</em> and <em>SLC44A2</em> as two susceptibility loci for venous thromboembolism. <span><span class="ref-journal">Am J Hum Genet. </span>2015;<span class="ref-vol">96</span>:532–542.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4385184/">PMC free article</a>]</span> [<a href="/pubmed/25772935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Meta-analysis+of+65,734+individuals+identifies+TSPAN15+and+SLC44A2+as+two+susceptibility+loci+for+venous+thromboembolism&amp;author=M+Germain&amp;volume=96&amp;publication_year=2015&amp;pages=532-542&amp;pmid=25772935&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Nikpay M, et al.  A comprehensive 1,000 Genomes-based genome-wide association meta-analysis of coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:1121–1130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4589895/">PMC free article</a>]</span> [<a href="/pubmed/26343387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+comprehensive+1,000+Genomes-based+genome-wide+association+meta-analysis+of+coronary+artery+disease&amp;author=M+Nikpay&amp;volume=47&amp;publication_year=2015&amp;pages=1121-1130&amp;pmid=26343387&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Ellinor PT, et al.  Meta-analysis identifies six new susceptibility loci for atrial fibrillation. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:670–675.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3366038/">PMC free article</a>]</span> [<a href="/pubmed/22544366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+identifies+six+new+susceptibility+loci+for+atrial+fibrillation&amp;author=PT+Ellinor&amp;volume=44&amp;publication_year=2012&amp;pages=670-675&amp;pmid=22544366&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Bulik-Sullivan B, et al.  An atlas of genetic correlations across human diseases and traits. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:1236–1241.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4797329/">PMC free article</a>]</span> [<a href="/pubmed/26414676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=An+atlas+of+genetic+correlations+across+human+diseases+and+traits&amp;author=B+Bulik-Sullivan&amp;volume=47&amp;publication_year=2015&amp;pages=1236-1241&amp;pmid=26414676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Bulik-Sullivan BK, et al.  LD score regression distinguishes confounding from polygenicity in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:291–295.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4495769/">PMC free article</a>]</span> [<a href="/pubmed/25642630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=LD+score+regression+distinguishes+confounding+from+polygenicity+in+genome-wide+association+studies&amp;author=BK+Bulik-Sullivan&amp;volume=47&amp;publication_year=2015&amp;pages=291-295&amp;pmid=25642630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Trynka G, et al.  Chromatin marks identify critical cell types for fine mapping complex trait variants. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:124–130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3826950/">PMC free article</a>]</span> [<a href="/pubmed/23263488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Chromatin+marks+identify+critical+cell+types+for+fine+mapping+complex+trait+variants&amp;author=G+Trynka&amp;volume=45&amp;publication_year=2013&amp;pages=124-130&amp;pmid=23263488&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Roadmap Epigenomics Consortium et al.  Integrative analysis of 111 reference human epigenomes. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">518</span>:317–330.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4530010/">PMC free article</a>]</span> [<a href="/pubmed/25693563" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrative+analysis+of+111+reference+human+epigenomes&amp;volume=518&amp;publication_year=2015&amp;pages=317-330&amp;pmid=25693563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Pers TH, et al.  Biological interpretation of genome-wide association studies using predicted gene functions. <span><span class="ref-journal">Nat Commun. </span>2015;<span class="ref-vol">6</span>:5890.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4420238/">PMC free article</a>]</span> [<a href="/pubmed/25597830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Biological+interpretation+of+genome-wide+association+studies+using+predicted+gene+functions&amp;author=TH+Pers&amp;volume=6&amp;publication_year=2015&amp;pages=5890&amp;pmid=25597830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Geer LY, et al.  The NCBI BioSystems database. <span><span class="ref-journal">Nucleic Acids Res. </span>2010;<span class="ref-vol">38</span>:D492–D496.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2808896/">PMC free article</a>]</span> [<a href="/pubmed/19854944" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+NCBI+BioSystems+database&amp;author=LY+Geer&amp;volume=38&amp;publication_year=2010&amp;pages=D492-D496&amp;pmid=19854944&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Mishra A, MacGregor S. A novel approach for pathway analysis of GWAS data highlights role of BMP signaling and muscle cell differentiation in colorectal cancer susceptibility. <span><span class="ref-journal">Twin Res Hum Genet. </span>2017;<span class="ref-vol">20</span>:1–9.</span> [<a href="/pubmed/28105966" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res+Hum+Genet&amp;title=A+novel+approach+for+pathway+analysis+of+GWAS+data+highlights+role+of+BMP+signaling+and+muscle+cell+differentiation+in+colorectal+cancer+susceptibility&amp;author=A+Mishra&amp;author=S+MacGregor&amp;volume=20&amp;publication_year=2017&amp;pages=1-9&amp;pmid=28105966&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Wakefield J. A Bayesian measure of the probability of false discovery in genetic epidemiology studies. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:208–227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950810/">PMC free article</a>]</span> [<a href="/pubmed/17668372" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+Bayesian+measure+of+the+probability+of+false+discovery+in+genetic+epidemiology+studies&amp;author=J+Wakefield&amp;volume=81&amp;publication_year=2007&amp;pages=208-227&amp;pmid=17668372&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Yang H, Wang K. Genomic variant annotation and prioritization with ANNOVAR and wANNOVAR. <span><span class="ref-journal">Nat Protoc. </span>2015;<span class="ref-vol">10</span>:1556–1566.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4718734/">PMC free article</a>]</span> [<a href="/pubmed/26379229" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Genomic+variant+annotation+and+prioritization+with+ANNOVAR+and+wANNOVAR&amp;author=H+Yang&amp;author=K+Wang&amp;volume=10&amp;publication_year=2015&amp;pages=1556-1566&amp;pmid=26379229&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Wang W, et al.  LNK/SH2B3 loss of function promotes atherosclerosis and thrombosis. <span><span class="ref-journal">Circ Res. </span>2016;<span class="ref-vol">119</span>:e91–e103.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5016083/">PMC free article</a>]</span> [<a href="/pubmed/27430239" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=LNK/SH2B3+loss+of+function+promotes+atherosclerosis+and+thrombosis&amp;author=W+Wang&amp;volume=119&amp;publication_year=2016&amp;pages=e91-e103&amp;pmid=27430239&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Ramensky V, Bork P, Sunyaev S. Human non-synonymous SNPs: server and survey. <span><span class="ref-journal">Nucleic Acids Res. </span>2002;<span class="ref-vol">30</span>:3894–3900.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC137415/">PMC free article</a>]</span> [<a href="/pubmed/12202775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Human+non-synonymous+SNPs:+server+and+survey&amp;author=V+Ramensky&amp;author=P+Bork&amp;author=S+Sunyaev&amp;volume=30&amp;publication_year=2002&amp;pages=3894-3900&amp;pmid=12202775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Kumar P, Henikoff S, Ng PC. Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm. <span><span class="ref-journal">Nat Protoc. </span>2009;<span class="ref-vol">4</span>:1073–1081.</span> [<a href="/pubmed/19561590" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Predicting+the+effects+of+coding+non-synonymous+variants+on+protein+function+using+the+SIFT+algorithm&amp;author=P+Kumar&amp;author=S+Henikoff&amp;author=PC+Ng&amp;volume=4&amp;publication_year=2009&amp;pages=1073-1081&amp;pmid=19561590&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">GTEx Consortium. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. <span><span class="ref-journal">Science. </span>2015;<span class="ref-vol">348</span>:648–660.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4547484/">PMC free article</a>]</span> [<a href="/pubmed/25954001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+Genotype-Tissue+Expression+(GTEx)+pilot+analysis:+multitissue+gene+regulation+in+humans&amp;volume=348&amp;publication_year=2015&amp;pages=648-660&amp;pmid=25954001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Eicher JD, et al.  GRASP v2.0: an update on the Genome-Wide Repository of Associations between SNPs and phenotypes. <span><span class="ref-journal">Nucleic Acids Res. </span>2015;<span class="ref-vol">43</span>:D799–D804.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4383982/">PMC free article</a>]</span> [<a href="/pubmed/25428361" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=GRASP+v2.0:+an+update+on+the+Genome-Wide+Repository+of+Associations+between+SNPs+and+phenotypes&amp;author=JD+Eicher&amp;volume=43&amp;publication_year=2015&amp;pages=D799-D804&amp;pmid=25428361&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Leslie R, O’Donnell CJ, Johnson AD. GRASP: analysis of genotype-phenotype results from 1390 genome-wide association studies and corresponding open access database. <span><span class="ref-journal">Bioinformatics. </span>2014;<span class="ref-vol">30</span>:i185–i194.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4072913/">PMC free article</a>]</span> [<a href="/pubmed/24931982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GRASP:+analysis+of+genotype-phenotype+results+from+1390+genome-wide+association+studies+and+corresponding+open+access+database&amp;author=R+Leslie&amp;author=CJ+O’Donnell&amp;author=AD+Johnson&amp;volume=30&amp;publication_year=2014&amp;pages=i185-i194&amp;pmid=24931982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Higasa K, et al.  Human genetic variation database, a reference database of genetic variations in the Japanese population. <span><span class="ref-journal">J Hum Genet. </span>2016;<span class="ref-vol">61</span>:547–553.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4931044/">PMC free article</a>]</span> [<a href="/pubmed/26911352" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hum+Genet&amp;title=Human+genetic+variation+database,+a+reference+database+of+genetic+variations+in+the+Japanese+population&amp;author=K+Higasa&amp;volume=61&amp;publication_year=2016&amp;pages=547-553&amp;pmid=26911352&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Bonder MJ, et al.  Disease variants alter transcription factor levels and methylation of their binding sites. <span><span class="ref-journal">Nat Genet. </span>2017;<span class="ref-vol">49</span>:131–138.</span> [<a href="/pubmed/27918535" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Disease+variants+alter+transcription+factor+levels+and+methylation+of+their+binding+sites&amp;author=MJ+Bonder&amp;volume=49&amp;publication_year=2017&amp;pages=131-138&amp;pmid=27918535&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Adams D, et al.  BLUEPRINT to decode the epigenetic signature written in blood. <span><span class="ref-journal">Nat Biotechnol. </span>2012;<span class="ref-vol">30</span>:224–226.</span> [<a href="/pubmed/22398613" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Biotechnol&amp;title=BLUEPRINT+to+decode+the+epigenetic+signature+written+in+blood&amp;author=D+Adams&amp;volume=30&amp;publication_year=2012&amp;pages=224-226&amp;pmid=22398613&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Franzen O, et al.  Cardiometabolic risk loci share downstream cis- and trans-gene regulation across tissues and diseases. <span><span class="ref-journal">Science. </span>2016;<span class="ref-vol">353</span>:827–830.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5534139/">PMC free article</a>]</span> [<a href="/pubmed/27540175" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Cardiometabolic+risk+loci+share+downstream+cis-+and+trans-gene+regulation+across+tissues+and+diseases&amp;author=O+Franzen&amp;volume=353&amp;publication_year=2016&amp;pages=827-830&amp;pmid=27540175&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Erbilgin A, et al.  Identification of CAD candidate genes in GWAS loci and their expression in vascular cells. <span><span class="ref-journal">J Lipid Res. </span>2013;<span class="ref-vol">54</span>:1894–1905.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3679391/">PMC free article</a>]</span> [<a href="/pubmed/23667179" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Lipid+Res&amp;title=Identification+of+CAD+candidate+genes+in+GWAS+loci+and+their+expression+in+vascular+cells&amp;author=A+Erbilgin&amp;volume=54&amp;publication_year=2013&amp;pages=1894-1905&amp;pmid=23667179&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">The ARIC investigators. The Atherosclerosis Risk in Communities (ARIC) Study: design and objectives. <span><span class="ref-journal">Am J Epidemiol. </span>1989;<span class="ref-vol">129</span>:687–702.</span> [<a href="/pubmed/2646917" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=The+Atherosclerosis+Risk+in+Communities+(ARIC)+Study:+design+and+objectives&amp;volume=129&amp;publication_year=1989&amp;pages=687-702&amp;pmid=2646917&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Suhre K, et al.  Connecting genetic risk to disease end points through the human blood plasma proteome. <span><span class="ref-journal">Nat Commun. </span>2017;<span class="ref-vol">8</span>:14357.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5333359/">PMC free article</a>]</span> [<a href="/pubmed/28240269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Connecting+genetic+risk+to+disease+end+points+through+the+human+blood+plasma+proteome&amp;author=K+Suhre&amp;volume=8&amp;publication_year=2017&amp;pages=14357&amp;pmid=28240269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Brænne I, et al.  Prediction of causal candidate genes in coronary artery disease loci. <span><span class="ref-journal">Arterioscler Thromb Vasc Biol. </span>2015;<span class="ref-vol">35</span>:2207–2217.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4583353/">PMC free article</a>]</span> [<a href="/pubmed/26293461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler+Thromb+Vasc+Biol&amp;title=Prediction+of+causal+candidate+genes+in+coronary+artery+disease+loci&amp;author=I+Brænne&amp;volume=35&amp;publication_year=2015&amp;pages=2207-2217&amp;pmid=26293461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Flister MJ, et al.  Identifying multiple causative genes at a single GWAS locus. <span><span class="ref-journal">Genome Res. </span>2013;<span class="ref-vol">23</span>:1996–2002.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3847770/">PMC free article</a>]</span> [<a href="/pubmed/24006081" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Identifying+multiple+causative+genes+at+a+single+GWAS+locus&amp;author=MJ+Flister&amp;volume=23&amp;publication_year=2013&amp;pages=1996-2002&amp;pmid=24006081&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Okada Y, et al.  Genetics of rheumatoid arthritis contributes to biology and drug discovery. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">506</span>:376–381.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3944098/">PMC free article</a>]</span> [<a href="/pubmed/24390342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetics+of+rheumatoid+arthritis+contributes+to+biology+and+drug+discovery&amp;author=Y+Okada&amp;volume=506&amp;publication_year=2014&amp;pages=376-381&amp;pmid=24390342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Kemp JP, et al.  Identification of 153 new loci associated with heel bone mineral density and functional involvement of GPC6 in osteoporosis. <span><span class="ref-journal">Nat Genet. </span>2017;<span class="ref-vol">49</span>:1468–1475.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5621629/">PMC free article</a>]</span> [<a href="/pubmed/28869591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Identification+of+153+new+loci+associated+with+heel+bone+mineral+density+and+functional+involvement+of+GPC6+in+osteoporosis&amp;author=JP+Kemp&amp;volume=49&amp;publication_year=2017&amp;pages=1468-1475&amp;pmid=28869591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Li Y, Kellis M. Joint Bayesian inference of risk variants and tissue-specific epigenomic enrichments across multiple complex human diseases. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>:e144.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5062982/">PMC free article</a>]</span> [<a href="/pubmed/27407109" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Joint+Bayesian+inference+of+risk+variants+and+tissue-specific+epigenomic+enrichments+across+multiple+complex+human+diseases&amp;author=Y+Li&amp;author=M+Kellis&amp;volume=44&amp;publication_year=2016&amp;pages=e144&amp;pmid=27407109&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="element-citation">Lee BK, et al.  Cell-type specific and combinatorial usage of diverse transcription factors revealed by genome-wide binding studies in multiple human cells. <span><span class="ref-journal">Genome Res. </span>2012;<span class="ref-vol">22</span>:9–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3246210/">PMC free article</a>]</span> [<a href="/pubmed/22090374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Cell-type+specific+and+combinatorial+usage+of+diverse+transcription+factors+revealed+by+genome-wide+binding+studies+in+multiple+human+cells&amp;author=BK+Lee&amp;volume=22&amp;publication_year=2012&amp;pages=9-24&amp;pmid=22090374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="element-citation">Sanseau P, et al.  Use of genome-wide association studies for drug repositioning. <span><span class="ref-journal">Nat Biotechnol. </span>2012;<span class="ref-vol">30</span>:317–320.</span> [<a href="/pubmed/22491277" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Biotechnol&amp;title=Use+of+genome-wide+association+studies+for+drug+repositioning&amp;author=P+Sanseau&amp;volume=30&amp;publication_year=2012&amp;pages=317-320&amp;pmid=22491277&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="element-citation">Nelson MR, et al.  The support of human genetic evidence for approved drug indications. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:856–860.</span> [<a href="/pubmed/26121088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+support+of+human+genetic+evidence+for+approved+drug+indications&amp;author=MR+Nelson&amp;volume=47&amp;publication_year=2015&amp;pages=856-860&amp;pmid=26121088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="element-citation">den Hoed M, et al.  Identification of heart rate-associated loci and their effects on cardiac conduction and rhythm disorders. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:621–631.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696959/">PMC free article</a>]</span> [<a href="/pubmed/23583979" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Identification+of+heart+rate-associated+loci+and+their+effects+on+cardiac+conduction+and+rhythm+disorders&amp;author=M+den+Hoed&amp;volume=45&amp;publication_year=2013&amp;pages=621-631&amp;pmid=23583979&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="element-citation">Pfeufer A, et al.  Genome-wide association study of PR interval. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:153–159.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850197/">PMC free article</a>]</span> [<a href="/pubmed/20062060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+PR+interval&amp;author=A+Pfeufer&amp;volume=42&amp;publication_year=2010&amp;pages=153-159&amp;pmid=20062060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="element-citation">Christophersen IE, et al.  Large-scale analyses of common and rare variants identify 12 new loci associated with atrial fibrillation. <span><span class="ref-journal">Nat Genet. </span>2017;<span class="ref-vol">49</span>:946–952.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5585859/">PMC free article</a>]</span> [<a href="/pubmed/28416818" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Large-scale+analyses+of+common+and+rare+variants+identify+12+new+loci+associated+with+atrial+fibrillation&amp;author=IE+Christophersen&amp;volume=49&amp;publication_year=2017&amp;pages=946-952&amp;pmid=28416818&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R66">66. <span class="element-citation">Verweij N, et al.  Genetic determinants of P wave duration and PR segment. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2014;<span class="ref-vol">7</span>:475–481.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4141024/">PMC free article</a>]</span> [<a href="/pubmed/24850809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Genetic+determinants+of+P+wave+duration+and+PR+segment&amp;author=N+Verweij&amp;volume=7&amp;publication_year=2014&amp;pages=475-481&amp;pmid=24850809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R67">67. <span class="element-citation">Le Scouarnec S, et al.  Dysfunction in ankyrin-B-dependent ion channel and transporter targeting causes human sinus node disease. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2008;<span class="ref-vol">105</span>:15617–15622.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2563133/">PMC free article</a>]</span> [<a href="/pubmed/18832177" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Dysfunction+in+ankyrin-B-dependent+ion+channel+and+transporter+targeting+causes+human+sinus+node+disease&amp;author=S+Le+Scouarnec&amp;volume=105&amp;publication_year=2008&amp;pages=15617-15622&amp;pmid=18832177&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R68">68. <span class="element-citation">Schott JJ, et al.  Congenital heart disease caused by mutations in the transcription factor NKX2-5. <span><span class="ref-journal">Science. </span>1998;<span class="ref-vol">281</span>:108–111.</span> [<a href="/pubmed/9651244" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Congenital+heart+disease+caused+by+mutations+in+the+transcription+factor+NKX2-5&amp;author=JJ+Schott&amp;volume=281&amp;publication_year=1998&amp;pages=108-111&amp;pmid=9651244&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R69">69. <span class="element-citation">Ellesoe SG, et al.  Familial atrial septal defect and sudden cardiac death: identification of a novel <em>NKX2-5</em> mutation and a review of the literature. <span><span class="ref-journal">Congenit Heart Dis. </span>2016;<span class="ref-vol">11</span>:283–290.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5019245/">PMC free article</a>]</span> [<a href="/pubmed/26679770" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Congenit+Heart+Dis&amp;title=Familial+atrial+septal+defect+and+sudden+cardiac+death:+identification+of+a+novel+NKX2-5+mutation+and+a+review+of+the+literature&amp;author=SG+Ellesoe&amp;volume=11&amp;publication_year=2016&amp;pages=283-290&amp;pmid=26679770&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R70">70. <span class="element-citation">Mohler PJ, et al.  Ankyrin-B mutation causes type 4 long-QT cardiac arrhythmia and sudden cardiac death. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">421</span>:634–639.</span> [<a href="/pubmed/12571597" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Ankyrin-B+mutation+causes+type+4+long-QT+cardiac+arrhythmia+and+sudden+cardiac+death&amp;author=PJ+Mohler&amp;volume=421&amp;publication_year=2003&amp;pages=634-639&amp;pmid=12571597&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R71">71. <span class="element-citation">Shi H, Kichaev G, Pasaniuc B. Contrasting the genetic architecture of 30 complex traits from summary association data. <span><span class="ref-journal">Am J Hum Genet. </span>2016;<span class="ref-vol">99</span>:139–153.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5005444/">PMC free article</a>]</span> [<a href="/pubmed/27346688" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Contrasting+the+genetic+architecture+of+30+complex+traits+from+summary+association+data&amp;author=H+Shi&amp;author=G+Kichaev&amp;author=B+Pasaniuc&amp;volume=99&amp;publication_year=2016&amp;pages=139-153&amp;pmid=27346688&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R72">72. <span class="element-citation">Kato N, et al.  Meta-analysis of genome-wide association studies identifies common variants associated with blood pressure variation in east Asians. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:531–538.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3158568/">PMC free article</a>]</span> [<a href="/pubmed/21572416" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+studies+identifies+common+variants+associated+with+blood+pressure+variation+in+east+Asians&amp;author=N+Kato&amp;volume=43&amp;publication_year=2011&amp;pages=531-538&amp;pmid=21572416&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R73">73. <span class="element-citation">Surendran P, et al.  Trans-ancestry meta-analyses identify rare and common variants associated with blood pressure and hypertension. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:1151–1161.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5056636/">PMC free article</a>]</span> [<a href="/pubmed/27618447" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Trans-ancestry+meta-analyses+identify+rare+and+common+variants+associated+with+blood+pressure+and+hypertension&amp;author=P+Surendran&amp;volume=48&amp;publication_year=2016&amp;pages=1151-1161&amp;pmid=27618447&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R74">74. <span class="element-citation">Winkler TW, et al.  Quality control and conduct of genome-wide association meta-analyses. <span><span class="ref-journal">Nat Protoc. </span>2014;<span class="ref-vol">9</span>:1192–1212.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4083217/">PMC free article</a>]</span> [<a href="/pubmed/24762786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Quality+control+and+conduct+of+genome-wide+association+meta-analyses&amp;author=TW+Winkler&amp;volume=9&amp;publication_year=2014&amp;pages=1192-1212&amp;pmid=24762786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R75">75. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R76">76. <span class="element-citation">So HC, Gui AH, Cherny SS, Sham PC. Evaluating the heritability explained by known susceptibility variants: a survey of ten complex diseases. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>:310–317.</span> [<a href="/pubmed/21374718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Evaluating+the+heritability+explained+by+known+susceptibility+variants:+a+survey+of+ten+complex+diseases&amp;author=HC+So&amp;author=AH+Gui&amp;author=SS+Cherny&amp;author=PC+Sham&amp;volume=35&amp;publication_year=2011&amp;pages=310-317&amp;pmid=21374718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R77">77. <span class="element-citation">Feigin VL, Lawes CM, Bennett DA, Anderson CS. Stroke epidemiology: a review of population-based studies of incidence, prevalence, and case-fatality in the late 20th century. <span><span class="ref-journal">Lancet Neurol. </span>2003;<span class="ref-vol">2</span>:43–53.</span> [<a href="/pubmed/12849300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Stroke+epidemiology:+a+review+of+population-based+studies+of+incidence,+prevalence,+and+case-fatality+in+the+late+20th+century&amp;author=VL+Feigin&amp;author=CM+Lawes&amp;author=DA+Bennett&amp;author=CS+Anderson&amp;volume=2&amp;publication_year=2003&amp;pages=43-53&amp;pmid=12849300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R78">78. <span class="element-citation">Yang J, et al.  Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:369–375. S1–S3.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3593158/">PMC free article</a>]</span> [<a href="/pubmed/22426310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Conditional+and+joint+multiple-SNP+analysis+of+GWAS+summary+statistics+identifies+additional+variants+influencing+complex+traits&amp;author=J+Yang&amp;volume=44&amp;publication_year=2012&amp;pages=369-375&amp;pmid=22426310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R79">79. <span class="element-citation">Liu JZ, et al.  A versatile gene-based test for genome-wide association studies. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>:139–145.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896770/">PMC free article</a>]</span> [<a href="/pubmed/20598278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+versatile+gene-based+test+for+genome-wide+association+studies&amp;author=JZ+Liu&amp;volume=87&amp;publication_year=2010&amp;pages=139-145&amp;pmid=20598278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R80">80. <span class="element-citation">Bowden J, Davey Smith G, Burgess S. Mendelian randomization with invalid instruments: effect estimation and bias detection through Egger regression. <span><span class="ref-journal">Int J Epidemiol. </span>2015;<span class="ref-vol">44</span>:512–525.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4469799/">PMC free article</a>]</span> [<a href="/pubmed/26050253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=Mendelian+randomization+with+invalid+instruments:+effect+estimation+and+bias+detection+through+Egger+regression&amp;author=J+Bowden&amp;author=G+Davey+Smith&amp;author=S+Burgess&amp;volume=44&amp;publication_year=2015&amp;pages=512-525&amp;pmid=26050253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R81">81. <span class="element-citation">International Consortium for Blood Pressure Genome-Wide Association Studies et al.  Genetic variants in novel pathways influence blood pressure and cardiovascular disease risk. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">478</span>:103–109.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3340926/">PMC free article</a>]</span> [<a href="/pubmed/21909115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+variants+in+novel+pathways+influence+blood+pressure+and+cardiovascular+disease+risk&amp;volume=478&amp;publication_year=2011&amp;pages=103-109&amp;pmid=21909115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R82">82. <span class="element-citation">Wain LV, et al.  Genome-wide association study identifies six new loci influencing pulse pressure and mean arterial pressure. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:1005–1011.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3445021/">PMC free article</a>]</span> [<a href="/pubmed/21909110" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+six+new+loci+influencing+pulse+pressure+and+mean+arterial+pressure&amp;author=LV+Wain&amp;volume=43&amp;publication_year=2011&amp;pages=1005-1011&amp;pmid=21909110&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R83">83. <span class="element-citation">Wellcome Trust Case Control Consortium et al.  Bayesian refinement of association signals for 14 loci in 3 common diseases. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:1294–1301.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3791416/">PMC free article</a>]</span> [<a href="/pubmed/23104008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Bayesian+refinement+of+association+signals+for+14+loci+in+3+common+diseases&amp;volume=44&amp;publication_year=2012&amp;pages=1294-1301&amp;pmid=23104008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R84">84. <span class="element-citation">Johnson AD, et al.  SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. <span><span class="ref-journal">Bioinformatics. </span>2008;<span class="ref-vol">24</span>:2938–2939.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720775/">PMC free article</a>]</span> [<a href="/pubmed/18974171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNAP:+a+web-based+tool+for+identification+and+annotation+of+proxy+SNPs+using+HapMap&amp;author=AD+Johnson&amp;volume=24&amp;publication_year=2008&amp;pages=2938-2939&amp;pmid=18974171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R85">85. <span class="element-citation">Arnold M, Raffler J, Pfeufer A, Suhre K, Kastenmüller G. SNiPA: an interactive, genetic variant-centered annotation browser. <span><span class="ref-journal">Bioinformatics. </span>2015;<span class="ref-vol">31</span>:1334–1336.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4393511/">PMC free article</a>]</span> [<a href="/pubmed/25431330" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNiPA:+an+interactive,+genetic+variant-centered+annotation+browser&amp;author=M+Arnold&amp;author=J+Raffler&amp;author=A+Pfeufer&amp;author=K+Suhre&amp;author=G+Kastenmüller&amp;volume=31&amp;publication_year=2015&amp;pages=1334-1336&amp;pmid=25431330&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R86">86. <span class="element-citation">Ward LD, Kellis M. HaploReg v4: systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>:D1, D877–D881.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4702929/">PMC free article</a>]</span> [<a href="/pubmed/26657631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=HaploReg+v4:+systematic+mining+of+putative+causal+variants,+cell+types,+regulators+and+target+genes+for+human+complex+traits+and+disease&amp;author=LD+Ward&amp;author=M+Kellis&amp;volume=44&amp;publication_year=2016&amp;pages=D1-D881&amp;pmid=26740669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R87">87. <span class="element-citation">ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:57–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3439153/">PMC free article</a>]</span> [<a href="/pubmed/22955616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+encyclopedia+of+DNA+elements+in+the+human+genome&amp;volume=489&amp;publication_year=2012&amp;pages=57-74&amp;pmid=22955616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R88">88. <span class="element-citation">Wishart DS, et al.  DrugBank: a comprehensive resource for in silico drug discovery and exploration. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:D668–D672.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1347430/">PMC free article</a>]</span> [<a href="/pubmed/16381955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=DrugBank:+a+comprehensive+resource+for+in+silico+drug+discovery+and+exploration&amp;author=DS+Wishart&amp;volume=34&amp;publication_year=2006&amp;pages=D668-D672&amp;pmid=16381955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R89">89. <span class="element-citation">Yang H, et al.  Therapeutic target database update 2016: enriched resource for bench to clinical drug target and targeted pathway information. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>:D1069–D1074.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4702870/">PMC free article</a>]</span> [<a href="/pubmed/26578601" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Therapeutic+target+database+update+2016:+enriched+resource+for+bench+to+clinical+drug+target+and+targeted+pathway+information&amp;author=H+Yang&amp;volume=44&amp;publication_year=2016&amp;pages=D1069-D1074&amp;pmid=26578601&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R90">90. <span class="element-citation">Hachiya T, et al.  Genetic predisposition to ischemic stroke: a polygenic risk score. <span><span class="ref-journal">Stroke. </span>2017;<span class="ref-vol">48</span>:253–258.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5266416/">PMC free article</a>]</span> [<a href="/pubmed/28034966" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Genetic+predisposition+to+ischemic+stroke:+a+polygenic+risk+score&amp;author=T+Hachiya&amp;volume=48&amp;publication_year=2017&amp;pages=253-258&amp;pmid=28034966&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R91">91. <span class="element-citation">Kanai M, et al.  Genetic analysis of quantitative traits in the Japanese population links cell types to complex human diseases. <span><span class="ref-journal">Nat Genet. </span>2018</span>  <a href="https://doi.org/10.1038/s41588-018-0047-6" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://doi.org/10.1038/s41588-018-0047-6</a>. [<a href="/pubmed/29403010" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5968830&amp;issue-id=313542&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140458741176288title">Abstract</a></li><li><a href="#S1title">Results</a></li><li><a href="#S15title">Discussion</a></li><li><a href="#S17title">Methods</a></li><li><a href="#S18title">Methods</a></li><li><a href="#S37title">Supplementary Material</a></li><li><a href="#S38title">Acknowledgments</a></li><li><a href="#APP1title">MEGASTROKE Consortium</a></li><li><a href="#APP2title">Author contributions</a></li><li><a href="#idm140458731278816title">Footnotes</a></li><li><a href="#idm140458737761904title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5968830/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5968830/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5968830/pdf/nihms959658.pdf">PDF (866K)</a></li> | <li><a href="#" data-citationid="PMC5968830" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5968830%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5968830%2F&amp;text=Multiancestry%20genome-wide%20association%20study%20of%20520%2C000%20subjects%20identifies%2032%20loci%20associated%20with%20stroke%20and%20stroke%20subtypes" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5968830%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29531354" data-aiid="5968830" data-aid="5968830" data-iid="313542" data-domainid="319" data-domain="nihpa" data-accid="PMC5968830" data-md5="343869191d3820111e53d95f6ebc15b3"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T17:03:18-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BDAA3DD5A876100000000095E0584&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5968830%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5968830/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>