<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5807179/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of a nicotine metabolism biomarker in African American smokers: impact of chromosome 19 genetic influences</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5807179/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Addiction (Abingdon, England)"><meta name="citation_title" content="Genome-wide association study of a nicotine metabolism biomarker in African American smokers: impact of chromosome 19 genetic influences"><meta name="citation_authors" content="Meghan J. Chenoweth, Jennifer J. Ware, Andy Z.X. Zhu, Christopher B. Cole, Lisa Sanderson Cox, Nikki Nollen, Jasjit S. Ahluwalia, Neal L. Benowitz, Robert A. Schnoll, Larry W. Hawk, Jr., Paul M. Cinciripini, Tony P. George, Caryn Lerman, Joanne Knight, Rachel F. Tyndale, on behalf of the PGRN-PNAT Research Group"><meta name="citation_date" content="March 2018"><meta name="citation_issue" content="3"><meta name="citation_volume" content="113"><meta name="citation_firstpage" content="509"><meta name="citation_doi" content="10.1111/add.14032"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5807179/?report=abstract"><meta name="citation_pmid" content="28921760"><meta name="DC.Title" content="Genome-wide association study of a nicotine metabolism biomarker in African American smokers: impact of chromosome 19 genetic influences"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Meghan J. Chenoweth"><meta name="DC.Contributor" content="Jennifer J. Ware"><meta name="DC.Contributor" content="Andy Z.X. Zhu"><meta name="DC.Contributor" content="Christopher B. Cole"><meta name="DC.Contributor" content="Lisa Sanderson Cox"><meta name="DC.Contributor" content="Nikki Nollen"><meta name="DC.Contributor" content="Jasjit S. Ahluwalia"><meta name="DC.Contributor" content="Neal L. Benowitz"><meta name="DC.Contributor" content="Robert A. Schnoll"><meta name="DC.Contributor" content="Larry W. Hawk, Jr."><meta name="DC.Contributor" content="Paul M. Cinciripini"><meta name="DC.Contributor" content="Tony P. George"><meta name="DC.Contributor" content="Caryn Lerman"><meta name="DC.Contributor" content="Joanne Knight"><meta name="DC.Contributor" content="Rachel F. Tyndale"><meta name="DC.Contributor" content="on behalf of the PGRN-PNAT Research Group"><meta name="DC.Date" content="2018 Mar"><meta name="DC.Identifier" content="10.1111/add.14032"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of a nicotine metabolism biomarker in African American smokers: impact of chromosome 19 genetic influences"><meta property="og:type" content="article"><meta property="og:description" content="The activity of CYP2A6, the major nicotine-inactivating enzyme, is measurable in smokers using the nicotine metabolite ratio (NMR; 3’hydroxycotinine/cotinine). Due to its role in nicotine clearance, the NMR is associated with smoking behaviours ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5807179/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5807179/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5807179/pdf/nihms907043.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D843CDD5AE4E10000000001B400F7.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5807179%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="nuccore" class="last">Nucleotide</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5807179</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Addiction</a></span>. Author manuscript; available in PMC 2019 Mar 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28921760" target="pmc_ext" ref="reftype=publisher&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Addiction. 2018 Mar; 113(3): 509–523. </span></a></div><span class="fm-vol-iss-date">Published online 2017 Nov 2. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1111%2Fadd.14032" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1111/add.14032</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5807179</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS907043</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28921760">28921760</a></div></div></div></div><h1 class="content-title">Genome-wide association study of a nicotine metabolism biomarker in African American smokers: impact of chromosome 19 genetic influences</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Chenoweth%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773454578544" co-class="co-affbox">Meghan J. Chenoweth</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Ware%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773379492640" co-class="co-affbox">Jennifer J. Ware</a>,<sup>3</sup> <a href="/pubmed/?term=Zhu%20AZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773373194112" co-class="co-affbox">Andy Z.X. Zhu</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Cole%20CB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773371861552" co-class="co-affbox">Christopher B. Cole</a>,<sup>4</sup> <a href="/pubmed/?term=Sanderson%20Cox%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773376553264" co-class="co-affbox">Lisa Sanderson Cox</a>,<sup>5</sup> <a href="/pubmed/?term=Nollen%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773378906160" co-class="co-affbox">Nikki Nollen</a>,<sup>5</sup> <a href="/pubmed/?term=Ahluwalia%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773378904176" co-class="co-affbox">Jasjit S. Ahluwalia</a>,<sup>6</sup> <a href="/pubmed/?term=Benowitz%20NL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773368502368" co-class="co-affbox">Neal L. Benowitz</a>,<sup>7</sup> <a href="/pubmed/?term=Schnoll%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773372612032" co-class="co-affbox">Robert A. Schnoll</a>,<sup>8</sup> <a href="/pubmed/?term=Hawk%20LW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773372610160" co-class="co-affbox">Larry W. Hawk, Jr.</a>,<sup>9</sup> <a href="/pubmed/?term=Cinciripini%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773377692176" co-class="co-affbox">Paul M. Cinciripini</a>,<sup>10</sup> <a href="/pubmed/?term=George%20TP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773418997392" co-class="co-affbox">Tony P. George</a>,<sup>11</sup> <a href="/pubmed/?term=Lerman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773418995408" co-class="co-affbox">Caryn Lerman</a>,<sup>12</sup> <a href="/pubmed/?term=Knight%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773374404464" co-class="co-affbox">Joanne Knight</a>,<sup>4</sup> <a href="/pubmed/?term=Tyndale%20RF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760" class="affpopup" co-rid="_co_idm139773417644432" co-class="co-affbox">Rachel F. Tyndale</a>,<sup>*,</sup><sup>1,</sup><sup>2,</sup><sup>13</sup> and  on behalf of the PGRN-PNAT Research Group<sup>ˆ</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139773454578544"><h3 class="no_margin">Meghan J. Chenoweth</h3><p>  <sup>1</sup>Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario, M5S 1A8, Canada</p><p>  <sup>2</sup>Campbell Family Mental Health Research Institute, Centre for Addiction and Mental Health, Toronto, Ontario, M5T 1R8, Canada</p><div>Find articles by <a href="/pubmed/?term=Chenoweth%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Meghan J. Chenoweth</a></div></div><div id="_co_idm139773379492640"><h3 class="no_margin">Jennifer J. Ware</h3><p>  <sup>3</sup>MRC Integrative Epidemiology Unit (IEU) and School of Social and Community Medicine at the University of Bristol, Bristol, BS8 2BN, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Ware%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Jennifer J. Ware</a></div></div><div id="_co_idm139773373194112"><h3 class="no_margin">Andy Z.X. Zhu</h3><p>  <sup>1</sup>Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario, M5S 1A8, Canada</p><p>  <sup>2</sup>Campbell Family Mental Health Research Institute, Centre for Addiction and Mental Health, Toronto, Ontario, M5T 1R8, Canada</p><div>Find articles by <a href="/pubmed/?term=Zhu%20AZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Andy Z.X. Zhu</a></div></div><div id="_co_idm139773371861552"><h3 class="no_margin">Christopher B. Cole</h3><p>  <sup>4</sup>Data Science Institute and Lancaster University Medical School, Lancaster, LA1 4YW, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Cole%20CB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Christopher B. Cole</a></div></div><div id="_co_idm139773376553264"><h3 class="no_margin">Lisa Sanderson Cox</h3><p>  <sup>5</sup>Department of Preventive Medicine and Public Health, University of Kansas School of Medicine, Kansas City, Kansas, 66160, USA</p><div>Find articles by <a href="/pubmed/?term=Sanderson%20Cox%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Lisa Sanderson Cox</a></div></div><div id="_co_idm139773378906160"><h3 class="no_margin">Nikki Nollen</h3><p>  <sup>5</sup>Department of Preventive Medicine and Public Health, University of Kansas School of Medicine, Kansas City, Kansas, 66160, USA</p><div>Find articles by <a href="/pubmed/?term=Nollen%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Nikki Nollen</a></div></div><div id="_co_idm139773378904176"><h3 class="no_margin">Jasjit S. Ahluwalia</h3><p>  <sup>6</sup>Department of Behavioral and Social Sciences, Brown University School of Public Health, Providence, Rhode Island 02912, USA</p><div>Find articles by <a href="/pubmed/?term=Ahluwalia%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Jasjit S. Ahluwalia</a></div></div><div id="_co_idm139773368502368"><h3 class="no_margin">Neal L. Benowitz</h3><p>  <sup>7</sup>Departments of Medicine and Bioengineering and Therapeutic Sciences, Division of Clinical Pharmacology and Experimental Therapeutics, University of California, San Francisco, San Francisco, California, 94110, USA</p><div>Find articles by <a href="/pubmed/?term=Benowitz%20NL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Neal L. Benowitz</a></div></div><div id="_co_idm139773372612032"><h3 class="no_margin">Robert A. Schnoll</h3><p>  <sup>8</sup>Department of Psychiatry, Perelman School of Medicine, and Abramson Cancer Center, University of Pennsylvania, Philadelphia, Pennsylvania, 19104-6376, USA</p><div>Find articles by <a href="/pubmed/?term=Schnoll%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Robert A. Schnoll</a></div></div><div id="_co_idm139773372610160"><h3 class="no_margin">Larry W. Hawk, Jr.</h3><p>  <sup>9</sup>Department of Psychology, University at Buffalo, SUNY, Buffalo, New York, 14260-4110, USA</p><div>Find articles by <a href="/pubmed/?term=Hawk%20LW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Larry W. Hawk, Jr.</a></div></div><div id="_co_idm139773377692176"><h3 class="no_margin">Paul M. Cinciripini</h3><p>  <sup>10</sup>Department of Behavioral Science, University of Texas, MD Anderson Cancer Center, Houston, Texas, 77030, USA</p><div>Find articles by <a href="/pubmed/?term=Cinciripini%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Paul M. Cinciripini</a></div></div><div id="_co_idm139773418997392"><h3 class="no_margin">Tony P. George</h3><p>  <sup>11</sup>Division of Schizophrenia, Centre for Addiction and Mental Health, Toronto, Ontario, Canada and Division of Brain and Therapeutics, Department of Psychiatry, University of Toronto, Toronto, Ontario, M5T 1R8, Canada</p><div>Find articles by <a href="/pubmed/?term=George%20TP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Tony P. George</a></div></div><div id="_co_idm139773418995408"><h3 class="no_margin">Caryn Lerman</h3><p>  <sup>12</sup>Department of Psychiatry, Annenberg School for Communication, and Abramson Cancer Center, University of Pennsylvania, Philadelphia, Pennsylvania, 19104-6376, USA</p><div>Find articles by <a href="/pubmed/?term=Lerman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Caryn Lerman</a></div></div><div id="_co_idm139773374404464"><h3 class="no_margin">Joanne Knight</h3><p>  <sup>4</sup>Data Science Institute and Lancaster University Medical School, Lancaster, LA1 4YW, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Knight%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Joanne Knight</a></div></div><div id="_co_idm139773417644432"><h3 class="no_margin">Rachel F. Tyndale</h3><p>  <sup>1</sup>Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario, M5S 1A8, Canada</p><p>  <sup>2</sup>Campbell Family Mental Health Research Institute, Centre for Addiction and Mental Health, Toronto, Ontario, M5T 1R8, Canada</p><p>  <sup>13</sup>Department of Psychiatry, University of Toronto, Toronto, Ontario, M5T 1R8, Canada</p><div>Find articles by <a href="/pubmed/?term=Tyndale%20RF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">Rachel F. Tyndale</a></div></div><div id="_co_idm139773416262928"><h3 class="no_margin">on behalf of the PGRN-PNAT Research Group</h3><p>  <sup>ˆ</sup>Membership of the PGRN-PNAT Research Group is provided in the Acknowledgments</p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28921760">on behalf of the PGRN-PNAT Research Group</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139773383673024_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139773383673024_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139773383673024_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">  <sup>1</sup>Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario, M5S 1A8, Canada</div><div class="fm-affl" id="A2" lang="en">  <sup>2</sup>Campbell Family Mental Health Research Institute, Centre for Addiction and Mental Health, Toronto, Ontario, M5T 1R8, Canada</div><div class="fm-affl" id="A3" lang="en">  <sup>3</sup>MRC Integrative Epidemiology Unit (IEU) and School of Social and Community Medicine at the University of Bristol, Bristol, BS8 2BN, United Kingdom</div><div class="fm-affl" id="A4" lang="en">  <sup>4</sup>Data Science Institute and Lancaster University Medical School, Lancaster, LA1 4YW, United Kingdom</div><div class="fm-affl" id="A5" lang="en">  <sup>5</sup>Department of Preventive Medicine and Public Health, University of Kansas School of Medicine, Kansas City, Kansas, 66160, USA</div><div class="fm-affl" id="A6" lang="en">  <sup>6</sup>Department of Behavioral and Social Sciences, Brown University School of Public Health, Providence, Rhode Island 02912, USA</div><div class="fm-affl" id="A7" lang="en">  <sup>7</sup>Departments of Medicine and Bioengineering and Therapeutic Sciences, Division of Clinical Pharmacology and Experimental Therapeutics, University of California, San Francisco, San Francisco, California, 94110, USA</div><div class="fm-affl" id="A8" lang="en">  <sup>8</sup>Department of Psychiatry, Perelman School of Medicine, and Abramson Cancer Center, University of Pennsylvania, Philadelphia, Pennsylvania, 19104-6376, USA</div><div class="fm-affl" id="A9" lang="en">  <sup>9</sup>Department of Psychology, University at Buffalo, SUNY, Buffalo, New York, 14260-4110, USA</div><div class="fm-affl" id="A10" lang="en">  <sup>10</sup>Department of Behavioral Science, University of Texas, MD Anderson Cancer Center, Houston, Texas, 77030, USA</div><div class="fm-affl" id="A11" lang="en">  <sup>11</sup>Division of Schizophrenia, Centre for Addiction and Mental Health, Toronto, Ontario, Canada and Division of Brain and Therapeutics, Department of Psychiatry, University of Toronto, Toronto, Ontario, M5T 1R8, Canada</div><div class="fm-affl" id="A12" lang="en">  <sup>12</sup>Department of Psychiatry, Annenberg School for Communication, and Abramson Cancer Center, University of Pennsylvania, Philadelphia, Pennsylvania, 19104-6376, USA</div><div class="fm-affl" id="A13" lang="en">  <sup>13</sup>Department of Psychiatry, University of Toronto, Toronto, Ontario, M5T 1R8, Canada</div><div class="fm-affl" id="A14" lang="en">  <sup>ˆ</sup>Membership of the PGRN-PNAT Research Group is provided in the Acknowledgments</div><div id="FN1"><sup>*</sup>Corresponding author <a href="mailto:dev@null" data-email="ac.otnorotu@eladnyt.r" class="oemail">ac.otnorotu@eladnyt.r</a> (RFT)</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139773383673024_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28921760" target="pmc_ext" ref="reftype=publisher&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Addiction</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5807179/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="8" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Figure 01: </strong>Figure S1. Quality control pipeline utilized in PNAT2 and KIS3 smokers prior to GWAS analysis The quality control (QC) steps for both samples and markers (i.e., variants) carried out prior to analysis are depicted in chronological order. The original sample sizes were: n=1,687 PNAT2 smokers (self-reported Caucasian, African or African American, Asian, Native American, Hawaiian/Polynesian, multi-racial, or other), and n=502 KIS3 smokers (self-reported African American). Sample QC on PNAT2 and KIS3 was performed separately for steps 1 and 2. The PNAT2 and KIS3 samples remaining following sample QC step 2 were analyzed together for relatedness (step 3) and ancestry determination (step 4). Remaining sample QC (step 5; heterozygosity determination) and all marker QC (steps 6-8) were performed separately for PNAT2 AA and KIS3 AA. The final numbers of samples and markers remaining after sample and marker QC are indicated in the grey boxes. The final number of PNAT2 AA and KIS3 AA samples remaining after sample, marker, and biomarker QC (step 9) are indicated in the black boxes. There were n=406 PNAT2 AA individuals available for final analyses after excluding those with missing menthol data (n=98), and n=449 KIS3 AA individuals were available for final analyses after excluding one participant with missing BMI data.<p>Abbreviations: QC, quality control; IBS, identity by state; PCA, principal components analysis; PC, principal component; SD, standard deviation; AA, African American</p></div><div><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Figure_01.jpg" data-ga-action="click_feat_suppl">NIHMS907043-supplement-Figure_01.jpg</a><span style="color:gray"> (248K)</span></div><div class="small guid">GUID:&nbsp;CC9DE78C-CAC6-4568-BBB7-1E71233B558C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Figure 02: </strong>Figure S2. Ancestry clustering in PNAT2 and KIS3 smokers using HapMap 3 as a reference population Principal Components 1 and 2, generated from Sample QC Step 4 (see <a href="#SD2" rid="SD2" class=" supplementary-material">Figure S1</a>), are plotted for the whole PNAT2 clinical sample (turquoise open circles), KIS3 (black open circles), and HapMap reference populations: CEU (Utah residents with Northern and Western European ancestry; red open circles), ASW (African ancestry in Southwest USA; purple open circles), YRI (Yoruba in Ibadan, Nigeria; green open circles), and CHB (Han Chinese in Beijing, China; blue open circles). African American ancestry for PNAT2 and KIS3 was determined using the following cut-points: Principal Component 1 ≤-0.025, and Principal Component 2 ≤0.01, indicated with black dotted lines. These conservative thresholds were selected to ensure homogeneity of the population and were chosen based on visual inspection of the plot.</div><div><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Figure_02.jpg" data-ga-action="click_feat_suppl">NIHMS907043-supplement-Figure_02.jpg</a><span style="color:gray"> (110K)</span></div><div class="small guid">GUID:&nbsp;369E79D3-8018-457E-A022-0ECFC18AA9DC</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Figure 03: </strong>Figure S3. Histograms depicting the distribution of the NMR and square-root NMR in PNAT2, KIS3, and KIS2 African American smokers The NMR distribution is shown in PNAT2 (A), KIS3 (B), and KIS2 (C) African American smokers. The square-root NMR distribution is shown in PNAT2 (D), KIS3 (E), and KIS2 (F) African American smokers. NMR was square-root transformed to help correct for positive skew.<p>Abbreviation: SD, standard deviation</p></div><div><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Figure_03.jpg" data-ga-action="click_feat_suppl">NIHMS907043-supplement-Figure_03.jpg</a><span style="color:gray"> (169K)</span></div><div class="small guid">GUID:&nbsp;DF2FF20C-68C6-4CCE-957E-014AE055B6B2</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Figure 04: </strong>Figure S4. Quantile-quantile (QQ) plots of the square-root NMR GWAS results in PNAT2 and KIS3 African American smokers QQ plots of the full results of the square-root NMR GWAS are shown in PNAT2 (A) and KIS3 (B), and after excluding genome-wide significant hits in PNAT2 (C) and KIS3 (D). Expected P-values are those expected under the null hypothesis. The shaded area around the red line indicates the 95% confidence interval under the null.</div><div><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Figure_04.jpg" data-ga-action="click_feat_suppl">NIHMS907043-supplement-Figure_04.jpg</a><span style="color:gray"> (151K)</span></div><div class="small guid">GUID:&nbsp;4135006A-153D-4065-9093-32AB9E907FC4</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 01: </strong>Table S1. 2,688 variants included as a custom iSelect® add-on to the Illumina HumanOmniExpressExome-8 v1.2 array.</div><div><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Table_01.xlsx" data-ga-action="click_feat_suppl">NIHMS907043-supplement-Table_01.xlsx</a><span style="color:gray"> (58K)</span></div><div class="small guid">GUID:&nbsp;17C7E278-9F29-46E6-B27D-2B9403FE8E00</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 02: </strong>Table S2. Complete list of chromosome 19 variants significantly (P&lt;5e-8) associated with the NMR in the meta-analysis of PNAT2 and KIS3 African American smokers.</div><div><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Table_02.xlsx" data-ga-action="click_feat_suppl">NIHMS907043-supplement-Table_02.xlsx</a><span style="color:gray"> (20K)</span></div><div class="small guid">GUID:&nbsp;DAD32180-1C5A-4648-95EE-5F756ABFCDE7</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 03: </strong>Table S3. Complete list of chromosome 19 variants significantly (P&lt;5e-8) associated with the NMR in PNAT2 African American smokers.</div><div><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Table_03.xlsx" data-ga-action="click_feat_suppl">NIHMS907043-supplement-Table_03.xlsx</a><span style="color:gray"> (15K)</span></div><div class="small guid">GUID:&nbsp;C92512D9-4699-49AE-93D0-391BF56CB015</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 04: </strong>Table S4. Complete list of chromosome 19 variants significantly (P&lt;5e-8) associated with the NMR in KIS3 African American smokers.</div><div><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Table_04.xlsx" data-ga-action="click_feat_suppl">NIHMS907043-supplement-Table_04.xlsx</a><span style="color:gray"> (14K)</span></div><div class="small guid">GUID:&nbsp;591CB20E-D6AC-41D9-B45B-771B917750A3</div></div></dd></dl></div><div id="idm139773418089120" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139773418089120title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background and aims</h3><p id="P1" class="p p-first-last">The activity of CYP2A6, the major nicotine-inactivating enzyme, is measurable in smokers using the nicotine metabolite ratio (NMR; 3’hydroxycotinine/cotinine). Due to its role in nicotine clearance, the NMR is associated with smoking behaviours and response to pharmacotherapies. The NMR is highly heritable (~80%), and on average lower in African Americans (AA) versus Whites. We previously identified several reduce and loss-of-function <em>CYP2A6</em> variants common in individuals of African descent. Our current aim was to identify novel genetic influences on the NMR in AA smokers using genome-wide approaches.</p></div><div id="S2" class="sec"><h3 id="S2title">Design</h3><p id="P2" class="p p-first-last">Genome-wide association study (GWAS).</p></div><div id="S3" class="sec"><h3 id="S3title">Setting</h3><p id="P3" class="p p-first-last">Multiple sites within Canada and the United States.</p></div><div id="S4" class="sec"><h3 id="S4title">Participants</h3><p id="P4" class="p p-first-last">AA smokers from two clinical trials: Pharmacogenetics of Nicotine Addiction Treatment (PNAT)-2 (<a id="__tag_664731984" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT01314001" ref="reftype=extlink-clinical-trial&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT01314001","term_id":"NCT01314001"}}</span>NCT01314001</a>; n=504) and Kick-it-at-Swope (KIS)-3 (<a id="__tag_664731986" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT00666978" ref="reftype=extlink-clinical-trial&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT00666978","term_id":"NCT00666978"}}</span>NCT00666978</a>; n=450).</p></div><div id="S5" class="sec"><h3 id="S5title">Measurements</h3><p id="P5" class="p p-first-last">Genome-wide SNP genotyping, the NMR (phenotype), and population substructure and NMR covariates.</p></div><div id="S6" class="sec"><h3 id="S6title">Findings</h3><p id="P6" class="p p-first-last">Meta-analysis revealed three independent chromosome 19 signals (rs12459249, rs111645190, and rs185430475) associated with the NMR. The top overall hit, rs12459249 (P=1.47e-39; beta=0.59 per C (versus T) allele, SE=0.045), located ~9.5kb 3’ of <em>CYP2A6</em>, remained genome-wide significant after controlling for the common (~10% in AA) non-functional <em>CYP2A6*17</em> allele. In contrast, rs111645190 and rs185430475 were not genome-wide significant when controlling for <em>CYP2A6*17.</em> In total, 96 signals associated with the NMR were identified; many were not found in prior NMR GWASs in European descent individuals. The top hits were also associated with the NMR in a third cohort of AA (KIS2; n=480). None of the hits were in <em>UGT</em> or <em>OCT2</em> genes.</p></div><div id="S7" class="sec sec-last"><h3 id="S7title">Conclusions</h3><p id="P7" class="p p-first-last">Three independent chromosome 19 signals account for ~20% of the variability in the nicotine metabolite ratio in African-American smokers. The hits identified may contribute to inter-ethnic variability in nicotine metabolism, smoking behaviours, and tobacco-related disease risk.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">CYP2A6, genome-wide association study, nicotine metabolism biomarker, cigarette smoking, African Americans, treatment-seeking smokers</span></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Introduction</h2><p id="P8" class="p p-first">The prevalence of cigarette smoking remains high despite widespread tobacco control efforts; recent estimates suggest 15% of Americans are current smokers (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731851">1</a>). A growing segment of American smokers are light smokers (smoke ≤10 cigarettes/day) who, like heavy smokers, experience elevated risks of disease and mortality compared to never smokers (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665310227">2</a>). Smoking behaviour and smoking-related morbidity differ by ethnicity. For instance, although African American (AA) smokers on average smoke fewer cigarettes/day than European American (EA) smokers, the level of total nicotine equivalents, a biomarker of total nicotine intake, is similar in AA and EA smokers, suggesting more intensive smoking (e.g., greater puff volume) among AAs (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731819">3</a>). At an equivalent number of cigarettes per day, the risk for lung cancer is higher in AA compared to EA smokers, perhaps due in part to more intensive smoking and, therefore, greater exposure to tobacco-specific nitrosamines and other harmful chemicals (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731843">4</a>). Of note, AA (vs. EA) smokers are also more likely to make quit attempts and are less likely to achieve cessation (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665310224">5</a>). Understanding the factors that contribute to this increased risk for lung cancer and reduced likelihood of cessation among AA smokers will help guide treatment interventions for this population.</p><p id="P9">Nicotine is the predominant psychoactive compound in cigarettes (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731842">6</a>). Nicotine undergoes metabolic inactivation by the hepatic CYP2A6 enzyme to cotinine, which is further metabolized to 3’hydroxycotinine exclusively by CYP2A6 (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731797">7</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731803">8</a>). The <em>CYP2A6</em> gene, located on chromosome 19q13.2, contains several functional polymorphisms, leading to inter-individual variation in the rate of nicotine clearance (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731833">9</a>). The nicotine metabolite ratio (NMR; 3’hydroxycotinine/cotinine) is an established and validated phenotypic marker of CYP2A6 activity in smokers; it is associated with <em>CYP2A6</em> genotype and correlates with nicotine clearance (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731859">10</a>-<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731807">15</a>). The NMR is 80% heritable (estimated in Finnish European twins) (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731863">16</a>); in addition to genetic influences, the NMR captures relatively minor environmental (e.g., mentholated cigarette use and BMI) (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731793">17</a>, <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731838">18</a>) sources of variation in CYP2A6 activity.</p><p id="P10">The NMR has been evaluated as a clinical marker for personalizing smoking cessation treatment. Compared to higher NMR, lower NMR (i.e., slower nicotine metabolism) is associated with higher cessation rates with behavioural counseling (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731779">19</a>) and among nicotine patch treated smokers (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731820">20</a>-<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731855">22</a>). In a placebo-controlled bupropion trial, bupropion increased quit rates over placebo in those with higher but not lower NMR (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731813">19</a>). In smokers prospectively randomized to treatment based on the NMR (PNAT2 Trial; <a id="__tag_664731989" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT01314001" ref="reftype=extlink-clinical-trial&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT01314001","term_id":"NCT01314001"}}</span>NCT01314001</a>), those with higher, but not lower, NMR had higher quit rates on varenicline (versus nicotine patch) (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731812">23</a>). Number-needed-to-treat analyses in smokers with higher NMR indicated 5 and 26 smokers would need to be treated with varenicline (versus placebo) and nicotine patch (versus placebo), respectively, for one smoker to quit, again indicating the superiority of varenicline for those with higher NMR (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731796">23</a>); in those with lower NMR these values were 8 and 10, respectively. In addition, those with lower (versus higher) NMR experienced greater negative side effects on varenicline (versus placebo) (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731808">23</a>). Thus, the evidence indicates that smokers with higher NMR show greater benefit from varenicline or bupropion compared to behavioural counseling or nicotine patch, while smokers with lower NMR are treated more effectively and safely with nicotine patch and/or behavioural counseling.</p><p id="P11">Variability in <em>CYP2A6</em> genetics and/or the NMR also influences the level of tobacco consumption (from cigarettes and smokeless tobacco), dependence, and risk for tobacco-related disease; smokers with slower metabolism (i.e., slow <em>CYP2A6</em> metabolism groups or lower NMR values) generally show lower consumption, dependence, and disease risk (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731850">17</a>, <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731792">24</a>-<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665310231">26</a>).</p><p id="P12">The NMR varies by ethnicity, with AA smokers having on average lower NMR (and nicotine clearance) versus smokers of EA descent (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731784">17</a>, <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731862">27</a>), due in part to the higher frequency of known <em>CYP2A6</em> reduced or loss-of-function variants in AAs (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731814">28</a>); many of these variants, including <em>*23</em> , <em>*24</em>, <em>*25</em>, <em>*28</em>, <em>*35</em>, and <em>*39-*45</em> (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731861">29</a>-<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731798">32</a>), were identified and functionally characterized by our group using <em>in vitro</em> (e.g., CYP2A6 cDNA expression system), <em>ex vivo</em> (e.g., human liver bank), and <em>in vivo</em> (e.g., human smokers) nicotine metabolism rate assessments. These variants are common (&gt;1% frequency) in AA, but exceedingly rare in EA populations. <em>CYP2A6*17</em>, with an allele frequency of 10% in AA (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731839">33</a>), explains ~8% of the variability in the NMR (unpublished observations in the KIS3 trial (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731815">34</a>)). Although &gt;40 <em>CYP2A6</em> variants have been identified and functionally characterized, estimates in Finnish Europeans indicate only ~30% of NMR variation is currently explained by detected <em>CYP2A6</em> variants (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731806">16</a>). To date, three NMR GWASs have been performed, predominantly or exclusively in Whites who were non-treatment-seeking smokers (only 413 AA total among &gt;4000 total participants from three studies) (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731847">16</a>, <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665310225">35</a>, <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731832">36</a>); no GWAS has examined the NMR in smokers seeking treatment, for whom personalized medicine approaches based on <em>CYP2A6</em>/the NMR would be targeted.</p><p id="P13" class="p p-last">Here we performed a GWAS of the NMR, assessed at baseline when participants were smoking <em>ad libitum</em> (i.e., when NMR is stable (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731818">13</a>)), in AA smokers from two smoking cessation clinical trials and genotyped top hits in a third trial to confirm associations with the NMR. The first involved heavy smokers (≥10 cigarettes/day) screened for the PNAT2 trial (<a id="__tag_664731983" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT01314001" ref="reftype=extlink-clinical-trial&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT01314001","term_id":"NCT01314001"}}</span>NCT01314001</a>), where smokers were randomized to placebo, nicotine patch, or varenicline (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731811">23</a>). The second involved light-smokers (≤10 cigarettes/day) that participated in a placebo-controlled bupropion trial for smoking cessation (KIS3 trial; <a id="__tag_664731985" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT00666978" ref="reftype=extlink-clinical-trial&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT00666978","term_id":"NCT00666978"}}</span>NCT00666978</a>) (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731785">34</a>, <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665310228">37</a>). To further investigate associations between selected GWAS hits and the NMR, we utilized a third sample of AA light-smokers from a placebo-controlled nicotine gum trial (KIS2 trial) (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731830">38</a>). Our goals were to better understand the genetic underpinnings of the NMR in AA smokers, and to compare genetic signals with those previously found in European populations to identify potential common and unique genetic influences on the NMR in AA smokers.</p></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Methods</h2><p id="P14" class="p p-first">The original trial protocols were approved by institutional review boards at all participating sites and at the University of Toronto. Individuals providing written informed consent for DNA sample collection and release of de-identified information to investigators underwent genotyping.</p><div id="S10" class="sec"><h3 id="S10title">PNAT2 Clinical Trial (<a id="__tag_664731988" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT01314001" ref="reftype=extlink-clinical-trial&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT01314001","term_id":"NCT01314001"}}</span>NCT01314001</a>) (<a href="#R23" rid="R23" class=" bibr popnode">23</a>)</h3><p id="P15" class="p p-first-last">Participant characteristics and trial procedures are described in detail elsewhere (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731791">17</a>, <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731788">23</a>). Briefly, eligible adult (aged 18-65 years) smokers (≥10 cigarettes/day) from four clinical sites (University of Pennsylvania, University of Toronto/Centre for Addiction and Mental Health, MD Anderson, and the State University of New York at Buffalo) were randomized prospectively based on their pre-treatment NMR to receive placebo, nicotine patch, or varenicline treatment for smoking cessation.</p></div><div id="S11" class="sec"><h3 id="S11title">KIS3 Clinical Trial (<a id="__tag_664731987" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT00666978" ref="reftype=extlink-clinical-trial&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT00666978","term_id":"NCT00666978"}}</span>NCT00666978</a>) (<a href="#R34" rid="R34" class=" bibr popnode">34</a>)</h3><p id="P16" class="p p-first-last">Participant characteristics and clinical trial procedures are described in detail elsewhere (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731840">34</a>, <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665310229">37</a>). Briefly, eligible adult (aged ≥18 years) light-smokers (≤10 cigarettes/day) from Kansas City, Missouri, were randomized to bupropion plus health education or placebo plus health education for smoking cessation.</p></div><div id="S12" class="sec"><h3 id="S12title">KIS2 Clinical Trial (<a href="#R38" rid="R38" class=" bibr popnode">38</a>)</h3><p id="P17" class="p p-first-last">Participant characteristics and clinical trial procedures are described in detail elsewhere (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731860">38</a>). Briefly, eligible adult (aged ≥18 years) light-smokers (≤10 cigarettes/day) from Kansas City, Missouri, were randomized to nicotine gum or placebo and health education or motivational interviewing for smoking cessation.</p></div><div id="S13" class="sec"><h3 id="S13title">Genome-Wide SNP Genotyping</h3><p id="P18" class="p p-first-last">Genome-wide SNP genotyping (PNAT2 and KIS3) was conducted using the Illumina HumanOmniExpressExome-8 v1.2 array (Illumina, San Diego, CA, USA) at the Centre for Applied Genomics at the Hospital for Sick Children (Toronto, ON, Canada). A custom iSelect® add-on comprising 2,688 variants (<a href="#SD6" rid="SD6" class=" supplementary-material">Table S1</a>) was included based on previous associations with nicotine metabolism and/or smoking behaviours including cessation; these variants cover the <em>CYP2ABFGST</em> cluster (chromosome 19), the <em>CHRNA5-A3-B4</em> nicotinic receptor cluster (chromosome 15), <em>OCT2</em> (chromosome 6), and the <em>UGT2B</em> cluster (chromosome 4).</p></div><div id="S14" class="sec"><h3 id="S14title">TaqMan SNP Genotyping</h3><p id="P19" class="p p-first-last">Candidate chromosome 19 polymorphisms (rs12459249, rs111645190, rs2644890, and rs111825958) were genotyped in KIS2 using an ABI ViiA<sup>TM</sup> 7 Real-Time PCR System and TaqMan® SNP genotyping assays (Thermo Fisher Scientific, Waltham, Massachusetts, USA) according to the manufacturer’s protocol. The resulting genotype frequencies for all four SNPs were in Hardy-Weinberg Equilibrium (each P&gt;0.05).</p></div><div id="S15" class="sec"><h3 id="S15title">Quality Control (QC) Procedures for Genome-Wide SNP Genotyping and Imputation</h3><p id="P20" class="p p-first">QC procedures for sample and variant were carried out, and genotypes were imputed, prior to analysis as outlined in <a href="#SD2" rid="SD2" class=" supplementary-material">Figures S1</a> and <a href="#SD3" rid="SD3" class=" supplementary-material">S2</a>. After identifying and removing samples with discordant sex information and excessive missingness of genetic data, the PNAT2 and KIS3 samples were combined to assess relatedness and ancestry to ensure a) an appropriate level of independence of individuals, and b) use of an equivalent threshold for determining ancestry. Individuals of AA ancestry, determined using principal components analysis in combination with data from HapMap 3 (<a href="#SD3" rid="SD3" class=" supplementary-material">Figure S2</a>), were selected for further analyses. In PNAT2 and KIS3, 98.5% and 96.6% of African descent smokers, respectively, had genetic ancestries concordant with self-reported ancestry. Following QC, the final number of individuals and markers available was: n=506 PNAT2 AAs (251 males, 255 females), 733,629 variants; and n=458 KIS3 AAs (154 males, 304 females), 742,493 variants (<a href="#SD2" rid="SD2" class=" supplementary-material">Figure S1</a>). QC procedures were performed using PLINK (version 1.07) (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731778">39</a>) and R software.</p><p id="P21" class="p p-last">The PNAT2 AA and KIS3 AA genotypes were then phased using SHAPEIT (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731841">40</a>) and imputed using IMPUTE2 (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731858">41</a>). The genomic data were divided into individual chromosomes (chromosomes 1-22) and aligned against the reference panel (Phase I release of 1000 Genomes). Following the elimination of duplicated SNPs, a second alignment step was performed, followed by pre-phasing and imputation, according to previously established protocols (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731795">41</a>-<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731856">44</a>). Variants with INFO (i.e., quality) scores &gt; 0.4 (threshold of 0.3 or higher is recommended (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731844">45</a>)) and a minor allele frequency &gt; 1% were selected for further analyses. Overall, 17,970,591 and 17,919,969 variants in PNAT2 and KIS3, respectively, were available for analysis.</p></div><div id="S16" class="sec"><h3 id="S16title">Assessment of Imputation Quality for CYP2A6 Relative to Other Chromosome 19 Genes</h3><p id="P22" class="p p-first-last"><em>CYP2A6</em> shares high homology with <em>CYP2A7</em> and <em>CYP2A13</em>, which can confound the accuracy of <em>CYP2A6</em> calls (<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731823">46</a>). IMPUTE2 info (i.e., imputation quality) scores for <em>CYP2A6</em> were compared to those of <em>EIF3K</em> and <em>TGFβ1</em>, located outside of this region of high homology (~2,222kb 5’ and ~480kb 3’ of <em>CYP2A6</em>, respectively).</p></div><div id="S17" class="sec"><h3 id="S17title">Phenotype: Nicotine Metabolite Ratio</h3><p id="P23" class="p p-first-last">The levels of cotinine and 3’hydroxycotinine were determined from blood samples collected at intake when participants were smoking <em>ad libitum</em> using identical liquid chromatography-tandem mass spectrometry according to previously established protocols (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731810">10</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731821">11</a>, <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731805">47</a>). The NMR (3’hydroxycotinine/cotinine) was square-root-transformed to correct for positive skew (<a href="#SD4" rid="SD4" class=" supplementary-material">Figure S3</a>). Individuals with cotinine values below 10 ng/ml, suggestive of non-daily smoking (<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731790">48</a>), were excluded from analyses.</p></div><div id="S18" class="sec"><h3 id="S18title">Covariates</h3><p id="P24" class="p p-first-last">Analyses in PNAT2 and KIS3 included principal components 1 and 2 as covariates to control for possible effects of population stratification (<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731787">49</a>). To identify additional covariates, we performed separate linear regression analyses to identify whether factors previously significantly associated with the NMR in the whole PNAT2 sample (sex, age, estrogen-containing therapy use, BMI, alcohol use (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731783">17</a>)) were associated with square-root NMR (with P&lt;0.10) in PNAT2 AA and KIS3 AA. In PNAT2, the following were included as covariates: sex (P=0.038), age (P=0.006), BMI (P=0.038), and use of menthol cigarettes (P=0.063). In KIS3, the following were included as covariates: sex (P=0.056), age (P&lt;0.001), and BMI (P&lt;0.001), but not mentholated cigarette use (P=0.60).</p></div><div id="S19" class="sec"><h3 id="S19title">Statistical Analyses</h3><div id="S20" class="sec sec-first"><p></p><h4 id="S20title" class="inline">GWAS of the NMR </h4><p id="P25" class="p p-first">SNPTEST (version 2.5.2) was used to identify genetic associations with the NMR separately in PNAT2 and KIS3; chromosomes 1-22 were analyzed separately. Frequentist additive models were specified, and genotype uncertainty was controlled for by using the “-method expected” option (uses expected genotype counts or genotype dosages). We also performed a separate set of analyses specifying frequentist dominant models, and the “-method score” option, and acquired similar results. Variants with P&lt;5e-8 were considered to be significant at the genome-wide level (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731822">50</a>).</p><p id="P26" class="p p-last">A meta-analysis of chromosome 19 results, adjusting for population sub-structure and NMR covariates, was then performed using META (version 1.7) (<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731837">51</a>). The genomic control inflation factor (λ) (calculated using PLINK) for the full GWAS analysis (chromosomes 1-22) was 1 and the QQ plots showed no deviation from the null (<a href="#SD5" rid="SD5" class=" supplementary-material">Figure S4</a>). Because the same phenotype (square-root NMR; <a href="#SD4" rid="SD4" class=" supplementary-material">Figure S3</a>) measured on the same scale was specified in both cohorts, the inverse-variance method based on a fixed-effects model was implemented (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731834">16</a>). Variants with INFO scores ≥0.50 were included in the meta-analysis; a total of 367,834 markers were in the union list.</p></div><div id="S21" class="sec"><p></p><h4 id="S21title" class="inline">Conditional Analysis of Chromosome 19 NMR Results in PNAT2 and KIS3 </h4><p id="P27" class="p p-first-last">To identify putatively independent chromosome 19 signals associated with the NMR, conditional analyses were performed (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731825">16</a>); the variant with the smallest P-value in the meta-analysis (i.e., rs12459249) was considered the first independent signal, and then ‘conditioned on’ (i.e., entered as a covariate) in subsequent frequentist additive models performed separately in PNAT2 and KIS3. These results were meta-analyzed, with the variant with the smallest P-value (i.e., the second independent signal) entered as a covariate along with the first independent signal in the second round of conditional analyses. The procedure was repeated until no additional significant (i.e., P&lt;5e-8) signals emerged.</p></div><div id="S22" class="sec sec-last"><p></p><h4 id="S22title" class="inline">Proportion of Variation in the NMR Accounted for by rs12459249, rs111645190, rs2644890, and rs11879604 </h4><p id="P28" class="p p-first-last">Separate linear regression models were used to determine the proportion of NMR variability attributable to selected variants (three-genotype coding), using SPSS version 23 (IBM, Armonk, New York, USA). The outcome measure was square-root NMR. Models in PNAT2 controlled for sex, age, BMI, and the use of mentholated cigarettes, while models in KIS3 and KIS2 controlled for sex, age, and BMI. The proportion of NMR variability accounted for by each variant was calculated by squaring the variant’s part correlation coefficient and multiplying by 100.</p></div></div><div id="S23" class="sec sec-last"><h3 id="S23title">Final Sample Sizes</h3><p id="P29" class="p p-first-last">Two of the n=506 PNAT2 AA participants were excluded from further analyses due to missing and outlying (&gt;4 SD from the mean) square-root NMR values (<a href="#SD2" rid="SD2" class=" supplementary-material">Figure S1</a>). After additionally excluding individuals with missing menthol covariate data (n=98), n=406 PNAT2 participants were available for GWAS analysis. Eight KIS3 AA participants were excluded due to having cotinine levels &lt;10 ng/ml (<a href="#SD2" rid="SD2" class=" supplementary-material">Figure S1</a>) which suggests non-daily smoking, and one participant was missing BMI data. Thus, n=449 KIS3 participants were available for GWAS analysis. Of the n=495 KIS2 individuals with pre-treatment NMR that provided a blood sample and consented to genetic testing, n=15 were excluded from further analyses due to insufficient quantity of blood remaining (n=7), cotinine level &lt;10 ng/ml (n=6), and outlying (&gt;4 SD from the mean) square-root NMR values (n=2). Thus, the final KIS2 AA sample comprised n=480 individuals.</p></div></div><div id="S24" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S24title">Results</h2><p id="P30" class="p p-first">Characteristics of the final analyzed sample are provided in <a href="/pmc/articles/PMC5807179/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. These values are similar to those reported in the full trial samples (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731835">17</a>, <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731854">34</a>, <a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731781">38</a>). In PNAT2 and KIS3, the median info score (out of 1, with higher scores indicating higher imputation quality) for <em>CYP2A6</em> was 0.9, compared to 0.6 and 0.9 for <em>EIF3K</em> and <em>TGFβ1</em>, respectively, suggesting adequate imputation quality for <em>CYP2A6</em>. In each of PNAT2 and KIS3, 98% of the variants significantly associated with the NMR at the genome-wide level (P&lt;5e-8) were located on chromosome 19, within or near to (several kilobases) the <em>CYP2A6</em> gene. Of note, no genetic variants in UGT enzymes (involved in the glucuronidation of nicotine, cotinine, and 3’hydroxycotinine (<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731799">52</a>)) or the OCT2 transporter (involved in nicotine transport (<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731789">52</a>)) reached genome-wide significance in either PNAT2 or KIS3.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p8">Characteristics of African American smokers from the three clinical trial samples</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139773372933632" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Characteristic</th><th rowspan="1" colspan="1" valign="top" align="center">PNAT2 (N=504)</th><th rowspan="1" colspan="1" valign="top" align="center">KIS3 (N=450)</th><th rowspan="1" colspan="1" valign="top" align="center">KIS2 (N=480)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center">% Female (N)</td><td rowspan="1" colspan="1" align="center">50.4 (254)</td><td rowspan="1" colspan="1" align="center">66.4 (299)</td><td rowspan="1" colspan="1" align="center">69.6 (334)</td></tr><tr><td rowspan="1" colspan="1" align="center">Age, mean (SD); range</td><td rowspan="1" colspan="1" align="center">47.3 (9.8); 20-65</td><td rowspan="1" colspan="1" align="center">46.8 (11.6); 19-80</td><td rowspan="1" colspan="1" align="center">45.0 (11.2); 19-81</td></tr><tr><td rowspan="1" colspan="1" align="center">BMI, mean (SD); range</td><td rowspan="1" colspan="1" align="center">30.5 (7.1); 17.6-58.3</td><td rowspan="1" colspan="1" align="center">31.2 (7.8); 14.8-68.4</td><td rowspan="1" colspan="1" align="center">30.5 (8.0); 14.0-73.5</td></tr><tr><td rowspan="1" colspan="1" align="center">Cigarettes/day, mean (SD); range</td><td rowspan="1" colspan="1" align="center">16.3 (6.3); 5-40<sup>b</sup></td><td rowspan="1" colspan="1" align="center">7.8 (2.6); 1-17</td><td rowspan="1" colspan="1" align="center">7.6 (3.3); 0-30</td></tr><tr><td rowspan="1" colspan="1" align="center">Cotinine in ng/ml, mean (SD), median; range</td><td rowspan="1" colspan="1" align="center">274.2 (130.4), 252.7; 32.2-837.3</td><td rowspan="1" colspan="1" align="center">243.7 (122.4), 233.2; 13.7-680.7</td><td rowspan="1" colspan="1" align="center">248.9 (144.9), 235.9; 10.1-927.3</td></tr><tr><td rowspan="1" colspan="1" align="center">NMR, mean (SD), median; range</td><td rowspan="1" colspan="1" align="center">0.33 (0.20), 0.28; 0.0090-1.17</td><td rowspan="1" colspan="1" align="center">0.38 (0.26), 0.33; 0.02-1.79</td><td rowspan="1" colspan="1" align="center">0.33 (0.23), 0.27; 0.02-1.70</td></tr></tbody></table></div><div id="largeobj_idm139773372933632" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5807179/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p9" class="p p-first-last">Abbreviations: PNAT, Pharmacogenetics of Nicotine Addiction Treatment; KIS, Kick-It-At-Swope; SD, standard deviation; BMI, body mass index; NMR, nicotine metabolite ratio</p></div></div></div><div id="S25" class="sec"><h3 id="S25title">Meta-analysis of Chromosome 19 NMR Results in PNAT2 and KIS3 AA Smokers</h3><p id="P31" class="p p-first-last">Ninety-six genome-wide significant chromosome 19 variants were identified after adjusting for cohort-specific principal components 1 and 2 and NMR covariates (top 10 variants in <a href="/pmc/articles/PMC5807179/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, full list in <a href="#SD7" rid="SD7" class=" supplementary-material">Table S2</a>; all top variants had info (quality) scores &gt;0.9). Of note, the top 10 variants did not differ (similar betas and P-values) when four principal components were adjusted for. The top (smallest P-value) overall variant in the meta-analysis was rs12459249 (I<sup>2</sup>=0; Heterogeneity P=0.80), with a combined P-value of 1.47e-39 (<a href="/pmc/articles/PMC5807179/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>); this was the top variant in PNAT2 (<a href="/pmc/articles/PMC5807179/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a>), and the second-most significant variant in KIS3 (<a href="/pmc/articles/PMC5807179/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>). Overall, 58 (60.4%) of the 96 significant hits were not genome-wide significant in the GWAS of the NMR performed in ~1,500 Finnish European smokers (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731782">16</a>); the most significant of these 58 AA hits in the meta-analysis was rs111825958 (I<sup>2</sup>=0.66; Heterogeneity P=0.32), with a combined P-value of 5.93e-26 (<a href="/pmc/articles/PMC5807179/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Effect sizes and P-values for the top variants in each population (PNAT2 and KIS3) are also provided in <a href="/pmc/articles/PMC5807179/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. In a separate meta-analysis that additionally controlled for cigarettes/day and menthol use in KIS3, the top hit was rs11878604 with a beta of -0.68 (SE=0.069; P=5.65e-23 per C vs. T allele), while rs12459249 was the second top hit with a beta of 0.59 (SE=0.063; P=5.73e-21 per C vs. T allele); these effect sizes did not substantially differ from those in the primary analysis (<a href="/pmc/articles/PMC5807179/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5807179/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139773450964064" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms907043f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms907043f1.jpg" src="/pmc/articles/PMC5807179/bin/nihms907043f1.jpg"></div></a><div id="largeobj_idm139773450964064" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5807179/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC5807179/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5807179/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>rs12459249 and rs111645190 tagged the top two independent signals significantly associated with the NMR from the meta-analysis of conditional analyses in PNAT2 and KIS3 African American smokers</strong><p id="__p4">The top (i.e., smallest P-value) overall variant associated with the NMR was rs12459249. LocusZoom plots depicting rs12459249 (indicated with a purple diamond) in PNAT2 and KIS3 are shown in (A) and (B), respectively. P-values are adjusted for principal components 1 and 2. The second independent signal associated with the NMR was tagged by rs111645190. LocusZoom plots depicting rs111645190 (indicated with a purple diamond) in PNAT2 and KIS3 are shown in (C) and (D), respectively. P-values are adjusted for principal components 1 and 2. LD patterns are based upon the hg19/1000 Genomes November 2014 release AFR reference population.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p10">Top 10 overall genetic variants significantly associated with the NMR in the meta-analyzed GWAS results from PNAT2 and KIS3 African American smokers</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139773364465120" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Variant</th><th rowspan="1" colspan="1" valign="top" align="center">Genotyped or Imputed (Imputation Quality Score)</th><th rowspan="1" colspan="1" valign="top" align="center">Gene/location</th><th rowspan="1" colspan="1" valign="top" align="center">Base-Pair Location (GRCh37)</th><th rowspan="1" colspan="1" valign="top" align="center">Ref. Allele</th><th rowspan="1" colspan="1" valign="top" align="center">Test Allele</th><th rowspan="1" colspan="1" valign="top" align="center">MAF (%) in PNAT2</th><th rowspan="1" colspan="1" valign="top" align="center">MAF (%) in KIS3</th><th rowspan="1" colspan="1" valign="top" align="center">Beta (SE); P-Value<sup>a</sup>,<sup>b</sup> in Meta-Analysis</th><th rowspan="1" colspan="1" valign="top" align="center">Beta (SE); P-Value<sup>a</sup> in PNAT2</th><th rowspan="1" colspan="1" valign="top" align="center">Beta (SE); P-Value<sup>b</sup> in KIS3</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center">rs12459249</td><td rowspan="1" colspan="1" align="center">Imputed in PNAT2 (0.99) and KIS3 (0.99)</td><td rowspan="1" colspan="1" align="center">~9.5kb 3’ of <em>CYP2A6</em></td><td rowspan="1" colspan="1" align="center">41339896</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">31.2</td><td rowspan="1" colspan="1" align="center">35.9</td><td rowspan="1" colspan="1" align="center">0.59 (0.045); 1.47e-39</td><td rowspan="1" colspan="1" align="center">0.61 (0.066); 1.59e-18</td><td rowspan="1" colspan="1" align="center">0.58 (0.062); 3.41e-19</td></tr><tr><td rowspan="1" colspan="1" align="center">rs10853742</td><td rowspan="1" colspan="1" align="center">Imputed in PNAT2 (0.99) and KIS3 (0.99)</td><td rowspan="1" colspan="1" align="center">~8.9kb 3’ of <em>CYP2A6</em></td><td rowspan="1" colspan="1" align="center">41340573</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">31.0</td><td rowspan="1" colspan="1" align="center">36.0</td><td rowspan="1" colspan="1" align="center">0.59 (0.045); 2.10e-39</td><td rowspan="1" colspan="1" align="center">0.60 (0.066); 1.92e-18</td><td rowspan="1" colspan="1" align="center">0.58 (0.062); 3.79e-19</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11667314</td><td rowspan="1" colspan="1" align="center">Imputed in PNAT2 (0.98) and KIS3 (0.98)</td><td rowspan="1" colspan="1" align="center">~8.5kb 3’ of <em>CYP2A6</em></td><td rowspan="1" colspan="1" align="center">41340983</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">30.8</td><td rowspan="1" colspan="1" align="center">35.8</td><td rowspan="1" colspan="1" align="center">0.59 (0.045); 5.00e-39</td><td rowspan="1" colspan="1" align="center">0.60 (0.066); 2.90e-18</td><td rowspan="1" colspan="1" align="center">0.58 (0.062); 5.17e-19</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11878604</td><td rowspan="1" colspan="1" align="center">Imputed in PNAT2 (0.97) and KIS3 (0.95)</td><td rowspan="1" colspan="1" align="center">~16kb 3’ of <em>CYP2A6</em></td><td rowspan="1" colspan="1" align="center">41333284</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">22.7</td><td rowspan="1" colspan="1" align="center">22.8</td><td rowspan="1" colspan="1" align="center">-0.65 (0.050); 7.36e-39</td><td rowspan="1" colspan="1" align="center">-0.64 (0.074); 9.60e-17</td><td rowspan="1" colspan="1" align="center">-0.67 (0.069); 2.19e-20</td></tr><tr><td rowspan="1" colspan="1" align="center">rs11083569</td><td rowspan="1" colspan="1" align="center">Imputed in PNAT2 (0.95) and KIS3 (0.95)</td><td rowspan="1" colspan="1" align="center">~9.1kb 3’ of <em>CYP2A6</em></td><td rowspan="1" colspan="1" align="center">41340321</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">37.5</td><td rowspan="1" colspan="1" align="center">41.1</td><td rowspan="1" colspan="1" align="center">0.53 (0.045); 3.97e-32</td><td rowspan="1" colspan="1" align="center">0.50 (0.066); 2.09e-13</td><td rowspan="1" colspan="1" align="center">0.56 (0.061); 4.88e-18</td></tr><tr><td rowspan="1" colspan="1" align="center">rs56267346</td><td rowspan="1" colspan="1" align="center">Imputed in PNAT2 (0.97) and KIS3 (0.96)</td><td rowspan="1" colspan="1" align="center"><em>CYP2A6</em> (intronic)</td><td rowspan="1" colspan="1" align="center">41353338</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">38.8</td><td rowspan="1" colspan="1" align="center">39.6</td><td rowspan="1" colspan="1" align="center">-0.50 (0.047); 5.49e-27</td><td rowspan="1" colspan="1" align="center">-0.56 (0.068); 4.02e-15</td><td rowspan="1" colspan="1" align="center">-0.46 (0.064); 5.83e-12</td></tr><tr><td rowspan="1" colspan="1" align="center">rs111825958<sup>c</sup></td><td rowspan="1" colspan="1" align="center">Genotyped in PNAT2 (N/A); imputed in KIS3 (0.92)</td><td rowspan="1" colspan="1" align="center">~17kb 3’ of <em>EGLN2</em></td><td rowspan="1" colspan="1" align="center">41331209</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">12.1</td><td rowspan="1" colspan="1" align="center">12.4</td><td rowspan="1" colspan="1" align="center">-0.71 (0.067); 5.93e-26</td><td rowspan="1" colspan="1" align="center">-0.64 (0.099); 4.11e-10</td><td rowspan="1" colspan="1" align="center">-0.77 (0.092); 5.28e-16</td></tr><tr><td rowspan="1" colspan="1" align="center">rs12986371</td><td rowspan="1" colspan="1" align="center">Genotyped in PNAT2 (N/A) and KIS3 (N/A)</td><td rowspan="1" colspan="1" align="center">~5.7kb 3’ of <em>CYP2A6</em></td><td rowspan="1" colspan="1" align="center">41343698</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">21.3</td><td rowspan="1" colspan="1" align="center">26.7</td><td rowspan="1" colspan="1" align="center">0.52 (0.051); 6.79e-24</td><td rowspan="1" colspan="1" align="center">0.53 (0.078); 3.70e-11</td><td rowspan="1" colspan="1" align="center">0.51 (0.069); 5.40e-13</td></tr><tr><td rowspan="1" colspan="1" align="center">rs111645190 <sup>c</sup></td><td rowspan="1" colspan="1" align="center">Imputed in PNAT2 (1.0) and KIS3 (0.99)</td><td rowspan="1" colspan="1" align="center">~5.5kb 5’ of <em>CYP2A6</em></td><td rowspan="1" colspan="1" align="center">41361808</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">13.7</td><td rowspan="1" colspan="1" align="center">14.2</td><td rowspan="1" colspan="1" align="center">-0.62 (0.062); 1.06e-23</td><td rowspan="1" colspan="1" align="center">-0.59 (0.092); 4.10e-10</td><td rowspan="1" colspan="1" align="center">-0.65 (0.085); 6.88e-14</td></tr><tr><td rowspan="1" colspan="1" align="center">rs145638254<sup>c</sup></td><td rowspan="1" colspan="1" align="center">Imputed in PNAT2 (1.0) and KIS3 (0.99)</td><td rowspan="1" colspan="1" align="center">~4.7kb 5’ of <em>CYP2A6</em></td><td rowspan="1" colspan="1" align="center">41361027</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">13.7</td><td rowspan="1" colspan="1" align="center">14.2</td><td rowspan="1" colspan="1" align="center">-0.62 (0.062); 1.07e-23</td><td rowspan="1" colspan="1" align="center">-0.59 (0.092); 4.10e-10</td><td rowspan="1" colspan="1" align="center">-0.65 (0.085); 6.88e-14</td></tr></tbody></table></div><div id="largeobj_idm139773364465120" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5807179/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="__p11" class="p p-first-last">MAF, minor allele frequency; N/A, not applicable</p></div><div id="TFN3"><sup>a</sup>In PNAT2, GWAS results were adjusted for principal components 1 and 2, sex, age, BMI, and the use of mentholated cigarettes.</div><div id="TFN4"><sup>b</sup>In KIS3, GWAS results were adjusted for principal components 1 and 2, sex, age, and BMI. The genomic inflation factor score (λ) in each population was 1 and therefore not adjusted for in the meta-analysis.</div><div id="TFN5"><sup>c</sup>These variants were not genome-wide significant in a study of ~1500 Finnish European smokers (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731817">16</a>)</div></div></div></div><div id="S26" class="sec"><h3 id="S26title">Conditional Analysis of Chromosome 19 NMR Results in African American Smokers</h3><p id="P32" class="p p-first-last">Conditional analyses of the chromosome 19 NMR results in PNAT2 and KIS3 AA smokers revealed a total of three independent signals associated with the NMR; the first two were tagged by rs12459249 and rs111645190 (<a href="/pmc/articles/PMC5807179/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). In PNAT2 and KIS3, rs12459249, located ~9.5kb 3’ of <em>CYP2A6</em>, substantially altered NMR (<a href="/pmc/articles/PMC5807179/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A and 2B</span></a>), explaining 17.1% and 15.3% of the variability in the NMR, respectively. The association between the rs12459249 variant and the NMR also replicated in KIS2 (P=1.30e-17; <a href="/pmc/articles/PMC5807179/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2C</span></a>). After conditioning on rs12459249, rs111645190 (located ~5.5kb 5’ of <em>CYP2A6</em>) had a P-value of 1.19e-11 (beta=-0.42 per A versus G allele; SE=0.062; <a href="/pmc/articles/PMC5807179/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and <a href="/pmc/articles/PMC5807179/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1C and 1D</span></a>). In PNAT2 and KIS3, the influence of rs111645190 on the NMR was also pronounced (<a href="/pmc/articles/PMC5807179/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2D and 2E</span></a>), explaining an additional 2.9% and 5.2% of the variation in the NMR, respectively, after controlling for rs12459249. Of note, in a separate meta-analysis that additionally controlled for cigarettes/day and menthol use in KIS3, the effect size for rs111645190 was similar to the primary analysis (<a href="/pmc/articles/PMC5807179/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) (beta=-0.67, SE=0.085; P=4.10e-15 per A vs. G allele). The association for rs111645190 also replicated in KIS2 (P=1.77e-7; <a href="/pmc/articles/PMC5807179/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2F</span></a>). After conditioning on both rs12459249 and rs111645190, a third independent signal emerged, tagged by rs185430475 (MAF = 2%; located &gt;10MB 3’ of <em>CYP2A6</em>), with a P-value of 1.94e-8 (beta=1.27 per G versus C allele; SE=0.23). Of note, the rs185430475 variant was not significantly associated with the NMR in the meta-analysis (beta = 1.25 per G versus C allele; SE=0.26; P=9.26e-7), nor in PNAT2 (beta = 1.02 per G versus C allele; SE=0.33; P=0.0023) or KIS3 (beta = 1.47 per G versus C allele; SE=0.34; P=2.39e-5).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5807179/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139773373136736" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms907043f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms907043f2.jpg" src="/pmc/articles/PMC5807179/bin/nihms907043f2.jpg"></div></a><div id="largeobj_idm139773373136736" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5807179/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC5807179/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5807179/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Influence of the top two independent signals, tagged by rs12459249 and rs111645190, identified in the meta-analysis of conditional analyses on the NMR in PNAT2, KIS3, and KIS2 smokers</strong><p id="__p5">Associations between rs12459249 and the NMR (not transformed) are shown in PNAT2 (P=1.59e-18) (A), KIS3 (P=3.41e-19) (B), and KIS2 (P=1.30e-17) (C) African American smokers using boxplots. Associations between rs111645190 and the NMR (not transformed) are shown in PNAT2 (P=4.10e-10) (D), KIS3 (P=6.88e-14) (E), and KIS2 (P=1.77e-7) (F) African American smokers using boxplots. The box represents the interquartile (IQ) range. The line across the box indicates the median NMR value. Open circles represent NMR values that are between 1.5X and 3X the IQ range, while asterisks represent NMR values that are greater than 3X the IQ range. The P-values for PNAT2 and KIS3 are derived from the square-root NMR GWAS conducted separately in each sample, and are adjusted for cohort-specific principal components 1 and 2 and NMR covariates. The P-values for KIS2 are derived from additive linear regression models of square-root NMR adjusting for sex, age, and BMI. In KIS2, n=5 individuals with NMR values of 1.70, 1.52, 1.45, 1.37, and 1.36 are omitted from the graph but were included in the analysis. In KIS3, n=5 individuals with NMR values of 1.79, 1.78, 1.56, 1.52, and 1.48 are omitted from the graph but were included in the analysis.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p15">The variants tagging the top two independent signals from the meta-analysis of conditional analyses in PNAT2 and KIS3 African American smokers</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139773373676704" class="xtable"><table frame="hsides" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Variant</th><th rowspan="1" colspan="1" valign="top" align="center">Gene/location</th><th rowspan="1" colspan="1" valign="top" align="center">Ref. Allele</th><th rowspan="1" colspan="1" valign="top" align="center">Test Allele</th><th rowspan="1" colspan="1" valign="top" align="center">MAF (%) in PNAT2</th><th rowspan="1" colspan="1" valign="top" align="center">MAF (%) in KIS3</th><th rowspan="1" colspan="1" valign="top" align="center">Beta (SE); P-Value in PNAT2<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Beta (SE); P-Value in KIS3<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Beta (SE); P-Value in Meta-Analysis<sup>b</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center">rs12459249</td><td rowspan="1" colspan="1" align="center">~9.5kb 3’ of <em>CYP2A6</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">31.2</td><td rowspan="1" colspan="1" align="center">35.9</td><td rowspan="1" colspan="1" align="center">0.60 (0.063); 1.16e-19</td><td rowspan="1" colspan="1" align="center">0.61 (0.064); 1.21e-19</td><td rowspan="1" colspan="1" align="center">0.59 (0.045); 1.47e-39</td></tr><tr><td rowspan="1" colspan="1" align="center">rs111645190</td><td rowspan="1" colspan="1" align="center">~5.5kb 5’ of <em>CYP2A6</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">13.7</td><td rowspan="1" colspan="1" align="center">14.2</td><td rowspan="1" colspan="1" align="center">-0.63 (0.088); 3.29e-12</td><td rowspan="1" colspan="1" align="center">-0.65 (0.089); 1.57e-12</td><td rowspan="1" colspan="1" align="center">-0.62 (0.06); 1.06e-23</td></tr></tbody></table></div><div id="largeobj_idm139773373676704" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5807179/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><p id="__p16" class="p p-first-last">MAF, minor allele frequency</p></div><div id="TFN7"><sup>a</sup>Adjusted for cohort-specific principal components 1 and 2</div><div id="TFN8"><sup>b</sup>In PNAT2, GWAS results were adjusted for principal components 1 and 2, sex, age, BMI, and the use of mentholated cigarettes. In KIS3, GWAS results were adjusted for principal components 1 and 2, sex, age, and BMI. The genomic inflation factor score (λ) in each population was 1 and therefore not adjusted for in the meta-analysis.</div></div></div></div><div id="S27" class="sec sec-last"><h3 id="S27title">Genetic Variants Associated with the NMR in African American Smokers (PNAT2 and KIS3 Analyzed Separately)</h3><p id="P33" class="p p-first">In PNAT2, 56 chromosome 19 variants significantly (P&lt;5e-8) associated with the NMR were identified after adjusting for population sub-structure; 53 remained significant after additionally controlling for NMR covariates (<a href="#SD8" rid="SD8" class=" supplementary-material">Table S3</a>). Controlling for clinical site did not substantially alter the findings (53 hits were still observed; rs12459249 remained the top hit with a beta (SE) per C vs. T allele of 0.61 (0.066); P=1.39e-18). A variant within chromosome 2 was also significantly associated with the NMR (rs16984355; P=2.1e-9). The top overall variant identified in PNAT2 was rs12459249 (<a href="/pmc/articles/PMC5807179/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a> and <a href="/pmc/articles/PMC5807179/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2A),</span></span><span>2A</span></a>), explaining 17.1% of NMR variation. Thirty-five of the 56 significant hits were not genome-wide significant in the ~1500 Finnish Europeans (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731800">16</a>); the most significant of these hits was rs2644890 (<a href="/pmc/articles/PMC5807179/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC5807179/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3A</span></a> and <a href="/pmc/articles/PMC5807179/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and4A),</span></span><span>4A</span></a>), explaining 2.3% of NMR variation after controlling for rs12459249. Per 1000 Genomes, rs12459249 and rs2644890 are not in appreciable linkage disequilibrium (LD) in individuals of African descent (r<sup>2</sup>&lt;0.20). The rs2644890 variant was also significantly associated with the NMR in KIS3 (P=1.24e-7; <a href="/pmc/articles/PMC5807179/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3B</span></a> and <a href="/pmc/articles/PMC5807179/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and4B)</span></span><span>4B</span></a>) and KIS2 (P=5.60e-5; <a href="/pmc/articles/PMC5807179/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4C</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC5807179/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139773371698688" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms907043f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms907043f3.jpg" src="/pmc/articles/PMC5807179/bin/nihms907043f3.jpg"></div></a><div id="largeobj_idm139773371698688" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5807179/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC5807179/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC5807179/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>rs2644890 and rs111825958 were the top unique variants significantly associated with the NMR in PNAT2 and KIS3 African American smokers, respectively</strong><p id="__p6">The top (i.e., smallest P-value) unique variant associated with the NMR in PNAT2 was rs2644890. LocusZoom plots depicting rs2644890 (indicated with a purple diamond) in PNAT2 and KIS3 are shown in (A) and (B), respectively. P-values are adjusted for principal components 1 and 2. The top (i.e., smallest P-value) unique variant associated with the NMR in KIS3 was rs111825958. LocusZoom plots depicting rs111825958 (indicated with a purple diamond) in PNAT2 and KIS3 are shown in (C) and (D), respectively. P-values are adjusted for principal components 1 and 2. LD patterns are based upon the hg19/1000 Genomes November 2014 release AFR reference population.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC5807179/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139773373820368" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms907043f4.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms907043f4.jpg" src="/pmc/articles/PMC5807179/bin/nihms907043f4.jpg"></div></a><div id="largeobj_idm139773373820368" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5807179/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a target="object" href="/pmc/articles/PMC5807179/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC5807179/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Influence of the top unique variants, rs2644890 and rs111825958, on the NMR in PNAT2, KIS3, and KIS2 smokers</strong><p id="__p7">Associations between rs2644890 and the NMR (not transformed) are shown in PNAT2 (P=2.04e-12) (A), KIS3 (P=5.95e-7) (B), and KIS2 (P=5.60e-5) (C) African American smokers using boxplots. Associations between rs111825958 and the NMR (not transformed) are shown in PNAT2 (P=4.11e-10) (D), KIS3 (P=5.28e-16) (E), and KIS2 (P=4.25e-11) (F) African American smokers using boxplots. The box represents the interquartile (IQ) range. The line across the box indicates the median NMR value. Open circles represent NMR values that are between 1.5X and 3X the IQ range, while asterisks represent NMR values that are greater than 3X the IQ range. The P-values for PNAT2 and KIS3 are derived from the square-root NMR GWAS conducted separately in each sample, and are adjusted for cohort-specific principal components 1 and 2 and NMR covariates. The P-values for KIS2 are derived from additive linear regression models of square-root NMR adjusting for sex, age, and BMI. In KIS2, n=5 individuals with NMR values of 1.70, 1.52, 1.45, 1.37, and 1.36 are omitted from the graph but were included in the analysis. In KIS3, n=5 individuals with NMR values of 1.79, 1.78, 1.56, 1.52, and 1.48 are omitted from the graph but were included in the analysis.</p></div></div></div><p id="P34">In KIS3, 46 chromosome 19 variants significantly (P&lt;5e-8) associated with the NMR were identified after adjusting for population sub-structure; 38 (&gt;80%) of these variants were also genome-wide significant in PNAT2. After additionally controlling for NMR covariates, 44 chromosome 19 variants remained significant (<a href="#SD9" rid="SD9" class=" supplementary-material">Table S4</a>). A variant within chromosome 2 was also significantly associated with the NMR (rs139278877; P=5.2e-9). The top overall variant in KIS3 was rs11878604 (<a href="/pmc/articles/PMC5807179/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), accounting for 17.1% of NMR variation; rs11878604 was also significant in PNAT2 (P=9.60e-17; <a href="/pmc/articles/PMC5807179/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Twenty-eight of the 46 significant hits in KIS3 were not genome-wide significant in the ~1500 Finnish Europeans (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731845">16</a>); the most significant of these hits was rs111825958 (<a href="/pmc/articles/PMC5807179/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC5807179/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3D</span></a> and <a href="/pmc/articles/PMC5807179/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and4E),</span></span><span>4E</span></a>), which explained 0.8% of NMR variation after controlling for rs11878604. Per 1000 Genomes, rs11878604 and rs111825958 are in moderate LD in individuals of African descent (r<sup>2</sup>=0.39). The rs111825958 variant was also significantly associated with the NMR in PNAT2 (P=4.11e-10; <a href="/pmc/articles/PMC5807179/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3C</span></a> and <a href="/pmc/articles/PMC5807179/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and4D)</span></span><span>4D</span></a>) and KIS2 (P=4.25e-11; <a href="/pmc/articles/PMC5807179/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4F</span></a>).</p><p id="P35" class="p p-last">Of note, the previously characterized nonsynonymous rs28399454 (C&gt;T) variant in exon 7 of <em>CYP2A6</em>, which defines the non-functional <em>CYP2A6*17</em> allele present at high frequency in AAs (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731853">33</a>), was significantly associated with the NMR in both PNAT2 (P=4.56e-11; beta=-0.68 per T versus C allele; SE=0.10, allele frequency=10.5%) and KIS3 (P=5.90e-11; beta=-0.68 per T versus C allele; SE=0.10 allele frequency=11.0%). In a model that controlled for population sub-structure, cohort-specific NMR covariates, and additionally for rs28399454, the P-values for rs12459249 in PNAT2 and KIS3 increased somewhat (from 1.59e-18 to 1.03e-11, and from 3.41e-19 to 1.13e-11, respectively). However, the P-value for rs111645190 was not genome- wide significant in each of PNAT2 and KIS3 after controlling for rs28399454: P-values increased from 4.10e-10 to 0.023 in PNAT2, and from 6.88e-14 to 0.011 in KIS3.</p></div></div><div id="S28" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S28title">Discussion</h2><p id="P36" class="p p-first">This is the first NMR GWAS conducted exclusively in African Americans. We identified three independent signals tagged by rs12459249, rs111645190, and rs185430475. These three signals were not in LD (r<sup>2</sup>&lt;0.20) with (in the 1000 Genomes Project AFR population) the four independent signals (rs56113850, rs113288603, esv2663194, and rs12461964) identified in the first NMR GWAS, which we conducted in ~1500 Finnish European smokers (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731836">16</a>). Together these findings extend our prior work (e.g., for <em>*23</em> , <em>*24</em>, <em>*25</em>, <em>*28</em>, <em>*35</em>, and <em>*39-*45</em> (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731849">29</a>-<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731786">32</a>)) showing the existence of unique genetic influences on CYP2A6 function and the NMR in AA. The top independent signal, rs12459249, located ~9.5kb 3’ of <em>CYP2A6</em>, was also genome-wide significant in the Finnish sample (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731802">16</a>), suggesting a common ancestral origin; however, it is possible that rs12459249 tags different functional variants in different populations. After controlling for rs28399454, the defining variant in the <em>CYP2A6*17</em> allele present at high frequencies in AA (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731831">33</a>), the P-values for rs12459249 in PNAT2 and KIS3 increased only somewhat (from ~10<sup>-18</sup> to 10<sup>-11</sup>), suggesting at least a portion of the influence of rs12459249 on the NMR is independent of <em>CYP2A6*17</em>. A recent study examined the NMR following oral or i.v. administration of labeled nicotine and cotinine in n=212, n=51, and n=49 individuals of EA, Asian American, and AA ancestry, respectively, and identified rs12459249 as the top-ranked SNP overall; rs12459249 was also non-significantly associated with the NMR (P=5.76e-6) in the small sample of AA (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665310223">35</a>).</p><p id="P37">The second independent signal was tagged by rs111645190, located ~5.5kb 5’ of <em>CYP2A6</em>. The top two independent variants (rs12459249 and rs111645190) explained ~20% of NMR variation, comparable to the amount of variability captured in the ~1500 Finnish European smokers (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731828">16</a>), where the independent signals explained ~18-31% of NMR variation. However, the influence of rs111645190 on the NMR was no longer significant in either PNAT2 or KIS3 after controlling for rs28399454 (<em>CYP2A6*17</em> allele), suggesting this second independent signal is largely driven by rs28399454 (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731852">33</a>).</p><p id="P38">Of note, over half (~60%) of the 96 hits found in the meta-analysis were not genome-wide significant in the ~1500 Finnish Europeans (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731826">16</a>), in part reflecting unique population LD structure. The top unique variant in the meta-analysis, rs111825958, was also associated with the NMR in KIS2 (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731846">38</a>). After controlling for rs28399454 (<em>CYP2A6*17</em>), the P-values in PNAT2 and KIS3 increased from 4.11e-10 to 0.018, and from 5.28e-16 to 1.18e-5, respectively, suggesting, as for rs111645190, that rs28399454 explains a large portion of the influence of rs111825958 on the NMR.</p><p id="P39">Our previous NMR GWAS in ~1500 Finnish European smokers (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731780">16</a>) identified &gt;700 hits, all found on chromosome 19q13 in or near to the <em>CYP2A6</em> locus. The top hit, rs56113850, located in intron 4 of <em>CYP2A6</em>, also replicated in the EA participants from the smaller GWAS of laboratory-based NMR (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665310226">35</a>). A subsequent GWAS of urinary NMR, conducted in ~2,200 smokers (including n=364 AA) from a prospective multi-ethnic cohort study, identified 248 variants (~99% of which were within or near <em>CYP2A6</em>) significantly associated with the NMR, and replicated this top hit (rs56113850) (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731801">36</a>). The rs56113850 variant was also significantly associated with the NMR in PNAT2 (P=1.30e-10, beta = 0.46 per C versus T allele, SE= 0.069) and KIS3 (P=8.02e-12, beta = 0.45 per C versus T allele, SE = 0.064) AA smokers, suggesting, as for rs12459249, a common ancestral origin. The demonstrated influence of rs12459249 and rs56113850 on the NMR in a variety of ethnic groups combined with their high variant allele frequencies (~30-60% in individuals of European and African descent) and only moderate LD (r<sup>2</sup>=0.46 and &lt;0.20 in European and African descent individuals, respectively; 1000 Genomes data), suggest that these SNPs should be routinely included in genotyping platforms for genomic investigations of nicotine metabolism and smoking cessation. GTEx expression quantitative trait loci analyses suggest rs12459249 is associated with CYP2A6 protein expression in the lung, and possibly liver (effect size=0.12 for C vs. T), while rs56113850 has a greater relative (vs. rs12459249) influence on liver CYP2A6 mRNA expression (effect size=0.26 for C vs. T).</p><p id="P40">Because the NMR is 80% heritable (estimated in Finnish twins) (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731777">16</a>), largely mediated by a single enzyme (i.e., CYP2A6), and not appreciably altered by environmental factors (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731824">17</a>), the usefulness of <em>CYP2A6</em> genetics for personalizing therapy and understanding tobacco-related disease risk shows great promise. However, the NMR can only be reliably used to assess CYP2A6 activity in current, regular (i.e., daily) smokers (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731829">12</a>-<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731776">14</a>), while <em>CYP2A6</em> genetics could be used to predict activity phenotype in current, former, and non-smokers in epidemiological investigations of cancer risk, for example. Thus, it is likely that a <em>CYP2A6</em> genetics-based approach could have greater utility and wider applicability compared to the NMR. In addition, because CYP2A6 also metabolizes therapeutic drugs including letrozole (<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731857">53</a>) and tegafur (<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731809">54</a>), two chemotherapeutics, as well as other drugs (e.g., efavirenz, metronidazole, artemisinin, valproic acid) (<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731827">55</a>), the usefulness of <em>CYP2A6</em> genetics in personalized medicine approaches extends beyond tobacco dependence.</p><p id="P41">Several limitations of our work warrant mention. By virtue of the genome-wide genotyping chip, we were unable to adequately examine structural variation in <em>CYP2A6</em>. Copy number variation, such as the <em>CYP2A6*1XA</em> duplication and <em>CYP2A6*4</em> deletion variants (<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731804">46</a>), is known to alter CYP2A6 activity; it is possible that known and/or novel copy number variants in <em>CYP2A6</em> are in LD with the variants identified in our study. In addition, the lack of overlap in signals observed between AA and European descent smokers may be due, in part, to differences in the genotyping platforms used, reference panels for imputation, quality control/imputation/MAF filtering pipelines, as well as potential inter-ethnic variation in environmental confounding factors and/or additional potential differences between smokers seeking treatment versus those that are not. However, head-to-head comparisons of NMR GWAS signals in PNAT2 AA and PNAT2 European descent smokers, analyzed using an identical genotyping platform and phase I release of 1000 Genomes, also indicate a substantial lack overlap (unpublished observations). Finally, analyzing treatment-seeking smokers may limit generalizability to general smoking populations, however personalized medicine approaches based on <em>CYP2A6</em> or the NMR would be targeted to treatment-seeking smokers and future GWAS in treatment-seeking smokers from other ethnic backgrounds should be considered. Future larger studies may identify important signals outside of <em>CYP2A6</em> that influence the NMR.</p><p id="P42" class="p p-last">In summary, we identified three independent signals in the largest NMR GWAS of AA smokers performed to date, accounting for ~20% of the total variability in the NMR. Over half (~60%) of the 96 total hits were not found in the largest NMR GWAS of European descent smokers (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731816">16</a>), and might contribute to unique regulation of <em>CYP2A6</em> in AA. Further investigation of these hits, including haplotype characterization and functional assessments will help identify which variants are causally influencing the NMR beyond known functional variants (e.g., <em>CYP2A6*17</em> (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731794">33</a>)). There may also be rare <em>CYP2A6</em> variants (<a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665310230">56</a>, <a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664731848">57</a>) with substantial impacts on the NMR; future sequencing-based studies will complement GWAS approaches and may further improve our understanding of the genetic influences on the NMR. Determining whether these genetic variants influence other phenotypes including smoking cessation will set the stage for genomics-based personalization of tobacco dependence treatment. Functional characterization studies may also provide insight into inter-ethnic variability in nicotine metabolism/CYP2A6 activity and resulting smoking behaviours and tobacco-related disease risk.</p></div><div id="S29" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S29title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD2"><h4>Figure 01</h4><!--caption a9--><strong>Figure S1. Quality control pipeline utilized in PNAT2 and KIS3 smokers prior to GWAS analysis: </strong><p id="P43">The quality control (QC) steps for both samples and markers (i.e., variants) carried out prior to analysis are depicted in chronological order. The original sample sizes were: n=1,687 PNAT2 smokers (self-reported Caucasian, African or African American, Asian, Native American, Hawaiian/Polynesian, multi-racial, or other), and n=502 KIS3 smokers (self-reported African American). Sample QC on PNAT2 and KIS3 was performed separately for steps 1 and 2. The PNAT2 and KIS3 samples remaining following sample QC step 2 were analyzed together for relatedness (step 3) and ancestry determination (step 4). Remaining sample QC (step 5; heterozygosity determination) and all marker QC (steps 6-8) were performed separately for PNAT2 AA and KIS3 AA. The final numbers of samples and markers remaining after sample and marker QC are indicated in the grey boxes. The final number of PNAT2 AA and KIS3 AA samples remaining after sample, marker, and biomarker QC (step 9) are indicated in the black boxes. There were n=406 PNAT2 AA individuals available for final analyses after excluding those with missing menthol data (n=98), and n=449 KIS3 AA individuals were available for final analyses after excluding one participant with missing BMI data.</p><p id="P44">Abbreviations: QC, quality control; IBS, identity by state; PCA, principal components analysis; PC, principal component; SD, standard deviation; AA, African American</p><div class="sup-box half_rhythm" id="d36e1182"><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Figure_01.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(248K, jpg)</sup></div></div><div class="sec suppmat" id="SD3"><h4>Figure 02</h4><!--caption a9--><strong>Figure S2. Ancestry clustering in PNAT2 and KIS3 smokers using HapMap 3 as a reference population: </strong><p id="P45">Principal Components 1 and 2, generated from Sample QC Step 4 (see <a href="#SD2" rid="SD2" class=" supplementary-material">Figure S1</a>), are plotted for the whole PNAT2 clinical sample (turquoise open circles), KIS3 (black open circles), and HapMap reference populations: CEU (Utah residents with Northern and Western European ancestry; red open circles), ASW (African ancestry in Southwest USA; purple open circles), YRI (Yoruba in Ibadan, Nigeria; green open circles), and CHB (Han Chinese in Beijing, China; blue open circles). African American ancestry for PNAT2 and KIS3 was determined using the following cut-points: Principal Component 1 ≤-0.025, and Principal Component 2 ≤0.01, indicated with black dotted lines. These conservative thresholds were selected to ensure homogeneity of the population and were chosen based on visual inspection of the plot.</p><div class="sup-box half_rhythm" id="d36e1194"><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Figure_02.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(110K, jpg)</sup></div></div><div class="sec suppmat" id="SD4"><h4>Figure 03</h4><!--caption a9--><strong>Figure S3. Histograms depicting the distribution of the NMR and square-root NMR in PNAT2, KIS3, and KIS2 African American smokers: </strong><p id="P46">The NMR distribution is shown in PNAT2 (A), KIS3 (B), and KIS2 (C) African American smokers. The square-root NMR distribution is shown in PNAT2 (D), KIS3 (E), and KIS2 (F) African American smokers. NMR was square-root transformed to help correct for positive skew.</p><p id="P47">Abbreviation: SD, standard deviation</p><div class="sup-box half_rhythm" id="d36e1205"><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Figure_03.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(169K, jpg)</sup></div></div><div class="sec suppmat" id="SD5"><h4>Figure 04</h4><!--caption a9--><strong>Figure S4. Quantile-quantile (QQ) plots of the square-root NMR GWAS results in PNAT2 and KIS3 African American smokers: </strong><p id="P48">QQ plots of the full results of the square-root NMR GWAS are shown in PNAT2 (A) and KIS3 (B), and after excluding genome-wide significant hits in PNAT2 (C) and KIS3 (D). Expected P-values are those expected under the null hypothesis. The shaded area around the red line indicates the 95% confidence interval under the null.</p><div class="sup-box half_rhythm" id="d36e1214"><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Figure_04.jpg" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(151K, jpg)</sup></div></div><div class="sec suppmat" id="SD6"><h4>Table 01</h4><p id="P49">Table S1. 2,688 variants included as a custom iSelect® add-on to the Illumina HumanOmniExpressExome-8 v1.2 array.</p><div class="sup-box half_rhythm" id="d36e1221"><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Table_01.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(58K, xlsx)</sup></div></div><div class="sec suppmat" id="SD7"><h4>Table 02</h4><p id="P50">Table S2. Complete list of chromosome 19 variants significantly (P&lt;5e-8) associated with the NMR in the meta-analysis of PNAT2 and KIS3 African American smokers.</p><div class="sup-box half_rhythm" id="d36e1228"><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Table_02.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(20K, xlsx)</sup></div></div><div class="sec suppmat" id="SD8"><h4>Table 03</h4><p id="P51">Table S3. Complete list of chromosome 19 variants significantly (P&lt;5e-8) associated with the NMR in PNAT2 African American smokers.</p><div class="sup-box half_rhythm" id="d36e1235"><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Table_03.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(15K, xlsx)</sup></div></div><div class="sec suppmat" id="SD9"><h4>Table 04</h4><p id="P52">Table S4. Complete list of chromosome 19 variants significantly (P&lt;5e-8) associated with the NMR in KIS3 African American smokers.</p><div class="sup-box half_rhythm" id="d36e1242"><a href="/pmc/articles/PMC5807179/bin/NIHMS907043-supplement-Table_04.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(14K, xlsx)</sup></div></div></div><div id="S30" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S30title">Acknowledgments</h2><div class="sec"><p id="__p1">Computations were performed on the Centre for Addiction and Mental Health (CAMH) Specialized Computing Cluster (SCC), funded by the Canada Foundation for Innovation Research Hospital Fund. J. J. Ware contributed to this work while supported by a Post-Doctoral Research Fellowship from the Oak Foundation (<a href="http://www.oakfnd.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.oakfnd.org/</a>). J. J. Ware was also a member of the Medical Research Council (MRC) Integrative Epidemiology Unit at this time; support from the MRC is gratefully acknowledged (MC_UU_12013/6). We also acknowledge these sources of funding: Canada Research Chair in Pharmacogenomics (R. F. Tyndale); National Institutes of Health (NIH) grants PGRN DA020830 (R. F. Tyndale and C. Lerman) and CA091912 (L. Sanderson Cox); Canadian Institutes of Health Research (CIHR) grant TMH-109787 (R. F. Tyndale); the Campbell Family Mental Health Research Institute of the Centre for Addiction and Mental Health (CAMH); the CAMH Foundation; the Canada Foundation for Innovation (#20289 and #16014); and the Ontario Ministry of Research and Innovation. Members of the Pharmacogenomics Research Network: Pharmacogenetics of Nicotine Addiction Treatment (PGRN-PNAT) Research Group include Frank Leone, Henry Glick, Angela Pinto, Paul Sanborn, Peter Gariti, Richard Landis (University of Pennsylvania); Maria Novalen, Bin Zhao, Ewa Hoffmann, Qian Zhou, Adel Aziziyeh (CAMH/University of Toronto); Martin Mahoney (Roswell Cancer Center, University of Buffalo); Maher Karam-Hage (The University of Texas M.D. Anderson Cancer Center); David Conti (University of Southern California); Andrew Bergen (SRI International).</p></div></div><div id="idm139773376956896" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139773376956896title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p2" class="p p-first"><strong>Declaration of Interests</strong></p><p id="__p3" class="p p-last">R. F. Tyndale has consulted in the past for Apotex on unrelated topics. N. L. Benowitz has consulted with pharmaceutical companies that market smoking cessation medications and has been a paid expert witness in litigation against tobacco companies. P. M. Cinciripini served on the scientific advisory board of Pfizer, conducted educational talks sponsored by Pfizer on smoking cessation (2006-2008), and has received grant support from Pfizer. R. A. Schnoll has provided consultation to Pfizer and GlaxoSmithKline. Pfizer Inc. provided varenicline and placebo pills at no cost for the PNAT2 clinical trial. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p></p></div></div><div id="idm139773378710400" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139773378710400title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Jamal A, King BA, Neff LJ, Whitmill J, Babb SD, Graffunder CM. Current Cigarette Smoking Among Adults - United States, 2005-2015. <span><span class="ref-journal">Mmwr-Morbidity and Mortality Weekly Report. </span>2016 Nov 11;<span class="ref-vol">65</span>(44):1205–11.</span> [<a href="/pubmed/27832052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mmwr-Morbidity+and+Mortality+Weekly+Report&amp;title=Current+Cigarette+Smoking+Among+Adults+-+United+States,+2005-2015&amp;author=A+Jamal&amp;author=BA+King&amp;author=LJ+Neff&amp;author=J+Whitmill&amp;author=SD+Babb&amp;volume=65&amp;issue=44&amp;publication_year=2016&amp;pages=1205-11&amp;pmid=27832052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Inoue-Choi M, Liao LM, Reyes-Guzman C, Hartge P, Caporaso N, Freedman ND. Association of Long-term, Low-Intensity Smoking With All-Cause and Cause-Specific Mortality in the National Institutes of Health-AARP Diet and Health Study. <span><span class="ref-journal">JAMA Intern Med. </span>2016 Dec 05;</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5555224/">PMC free article</a>]</span> [<a href="/pubmed/27918784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA+Intern+Med&amp;title=Association+of+Long-term,+Low-Intensity+Smoking+With+All-Cause+and+Cause-Specific+Mortality+in+the+National+Institutes+of+Health-AARP+Diet+and+Health+Study&amp;author=M+Inoue-Choi&amp;author=LM+Liao&amp;author=C+Reyes-Guzman&amp;author=P+Hartge&amp;author=N+Caporaso&amp;publication_year=2016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">St Helen G, Dempsey D, Wilson M, Jacob P, Benowitz NL. Racial differences in the relationship between tobacco dependence and nicotine and carcinogen exposure. <span><span class="ref-journal">Addiction. </span>2013 Mar;<span class="ref-vol">108</span>(3):607–17.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3553231/">PMC free article</a>]</span> [<a href="/pubmed/22971134" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Addiction&amp;title=Racial+differences+in+the+relationship+between+tobacco+dependence+and+nicotine+and+carcinogen+exposure&amp;author=G+St+Helen&amp;author=D+Dempsey&amp;author=M+Wilson&amp;author=P+Jacob&amp;author=NL+Benowitz&amp;volume=108&amp;issue=3&amp;publication_year=2013&amp;pages=607-17&amp;pmid=22971134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Haiman CA, Stram DO, Wilkens LR, Pike MC, Kolonel LN, Henderson BE, et al.  Ethnic and racial differences in the smoking-related risk of lung cancer. <span><span class="ref-journal">N Engl J Med. </span>2006 Jan 26;<span class="ref-vol">354</span>(4):333–42.</span> [<a href="/pubmed/16436765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Ethnic+and+racial+differences+in+the+smoking-related+risk+of+lung+cancer&amp;author=CA+Haiman&amp;author=DO+Stram&amp;author=LR+Wilkens&amp;author=MC+Pike&amp;author=LN+Kolonel&amp;volume=354&amp;issue=4&amp;publication_year=2006&amp;pages=333-42&amp;pmid=16436765&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Goren A, Annunziata K, Schnoll RA, Suaya JA. Smoking Cessation and Attempted Cessation among Adults in the United States. <span><span class="ref-journal">Plos One. </span>2014 Mar 27;<span class="ref-vol">9</span>(3)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3968049/">PMC free article</a>]</span> [<a href="/pubmed/24676348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plos+One&amp;title=Smoking+Cessation+and+Attempted+Cessation+among+Adults+in+the+United+States&amp;author=A+Goren&amp;author=K+Annunziata&amp;author=RA+Schnoll&amp;author=JA+Suaya&amp;volume=9&amp;issue=3&amp;publication_year=2014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Benowitz NL. Nicotine addiction. <span><span class="ref-journal">N Engl J Med. </span>2010 Jun 17;<span class="ref-vol">362</span>(24):2295–303.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928221/">PMC free article</a>]</span> [<a href="/pubmed/20554984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Nicotine+addiction&amp;author=NL+Benowitz&amp;volume=362&amp;issue=24&amp;publication_year=2010&amp;pages=2295-303&amp;pmid=20554984&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Nakajima M, Yamamoto T, Nunoya K, Yokoi T, Nagashima K, Inoue K, et al.  Role of human cytochrome P4502A6 in C-oxidation of nicotine. <span><span class="ref-journal">Drug Metab Dispos. </span>1996 Nov;<span class="ref-vol">24</span>(11):1212–7.</span> [<a href="/pubmed/8937855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Metab+Dispos&amp;title=Role+of+human+cytochrome+P4502A6+in+C-oxidation+of+nicotine&amp;author=M+Nakajima&amp;author=T+Yamamoto&amp;author=K+Nunoya&amp;author=T+Yokoi&amp;author=K+Nagashima&amp;volume=24&amp;issue=11&amp;publication_year=1996&amp;pages=1212-7&amp;pmid=8937855&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Nakajima M, Yamamoto T, Nunoya K, Yokoi T, Nagashima K, Inoue K, et al.  Characterization of CYP2A6 involved in 3’-hydroxylation of cotinine in human liver microsomes. <span><span class="ref-journal">J Pharmacol Exp Ther. </span>1996 May;<span class="ref-vol">277</span>(2):1010–5.</span> [<a href="/pubmed/8627511" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pharmacol+Exp+Ther&amp;title=Characterization+of+CYP2A6+involved+in+3’-hydroxylation+of+cotinine+in+human+liver+microsomes&amp;author=M+Nakajima&amp;author=T+Yamamoto&amp;author=K+Nunoya&amp;author=T+Yokoi&amp;author=K+Nagashima&amp;volume=277&amp;issue=2&amp;publication_year=1996&amp;pages=1010-5&amp;pmid=8627511&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Benowitz NL, Swan GE, Jacob P, 3rd, Lessov-Schlaggar CN, Tyndale RF. CYP2A6 genotype and the metabolism and disposition kinetics of nicotine. <span><span class="ref-journal">Clinical Pharmacology &amp; Therapeutics. </span>2006 Nov;<span class="ref-vol">80</span>(5):457–67.</span> [<a href="/pubmed/17112802" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clinical+Pharmacology+&amp;+Therapeutics&amp;title=CYP2A6+genotype+and+the+metabolism+and+disposition+kinetics+of+nicotine&amp;author=NL+Benowitz&amp;author=GE+Swan&amp;author=P+Jacob&amp;author=CN+Lessov-Schlaggar&amp;author=RF+Tyndale&amp;volume=80&amp;issue=5&amp;publication_year=2006&amp;pages=457-67&amp;pmid=17112802&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Dempsey D, Tutka P, Jacob P, Allen F, Schoedel K, Tyndale RF, et al.  Nicotine metabolite ratio as an index of cytochrome P450 2A6 metabolic activity. <span><span class="ref-journal">Clinical Pharmacology &amp; Therapeutics. </span>2004 Jul;<span class="ref-vol">76</span>(1):64–72.</span> [<a href="/pubmed/15229465" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clinical+Pharmacology+&amp;+Therapeutics&amp;title=Nicotine+metabolite+ratio+as+an+index+of+cytochrome+P450+2A6+metabolic+activity&amp;author=D+Dempsey&amp;author=P+Tutka&amp;author=P+Jacob&amp;author=F+Allen&amp;author=K+Schoedel&amp;volume=76&amp;issue=1&amp;publication_year=2004&amp;pages=64-72&amp;pmid=15229465&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Tanner JA, Novalen M, Jatlow P, Huestis MA, Murphy SE, Kaprio J, et al.  Nicotine metabolite ratio (3-hydroxycotinine/cotinine) in plasma and urine by different analytical methods and laboratories: implications for clinical implementation. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2015 Aug;<span class="ref-vol">24</span>(8):1239–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4526326/">PMC free article</a>]</span> [<a href="/pubmed/26014804" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Nicotine+metabolite+ratio+(3-hydroxycotinine/cotinine)+in+plasma+and+urine+by+different+analytical+methods+and+laboratories:+implications+for+clinical+implementation&amp;author=JA+Tanner&amp;author=M+Novalen&amp;author=P+Jatlow&amp;author=MA+Huestis&amp;author=SE+Murphy&amp;volume=24&amp;issue=8&amp;publication_year=2015&amp;pages=1239-46&amp;pmid=26014804&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Lea RA, Dickson S, Benowitz NL. Within-subject variation of the salivary 3HC/COT ratio in regular daily smokers: prospects for estimating CYP2A6 enzyme activity in large-scale surveys of nicotine metabolic rate. <span><span class="ref-journal">J Anal Toxicol. </span>2006 Jul-Aug;<span class="ref-vol">30</span>(6):386–9.</span> [<a href="/pubmed/16872570" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Anal+Toxicol&amp;title=Within-subject+variation+of+the+salivary+3HC/COT+ratio+in+regular+daily+smokers:+prospects+for+estimating+CYP2A6+enzyme+activity+in+large-scale+surveys+of+nicotine+metabolic+rate&amp;author=RA+Lea&amp;author=S+Dickson&amp;author=NL+Benowitz&amp;volume=30&amp;issue=6&amp;publication_year=2006&amp;pages=386-9&amp;pmid=16872570&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Mooney ME, Li ZZ, Murphy SE, Pentel PR, Le C, Hatsukami DK. Stability of the nicotine metabolite ratio in ad libitum and reducing smokers. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2008 Jun;<span class="ref-vol">17</span>(6):1396–400.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2765478/">PMC free article</a>]</span> [<a href="/pubmed/18559554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Stability+of+the+nicotine+metabolite+ratio+in+ad+libitum+and+reducing+smokers&amp;author=ME+Mooney&amp;author=ZZ+Li&amp;author=SE+Murphy&amp;author=PR+Pentel&amp;author=C+Le&amp;volume=17&amp;issue=6&amp;publication_year=2008&amp;pages=1396-400&amp;pmid=18559554&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">St Helen G, Jacob P, 3rd, Benowitz NL. Stability of the nicotine metabolite ratio in smokers of progressively reduced nicotine content cigarettes. <span><span class="ref-journal">Nicotine Tob Res. </span>2013 Nov;<span class="ref-vol">15</span>(11):1939–42.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3790625/">PMC free article</a>]</span> [<a href="/pubmed/23674838" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nicotine+Tob+Res&amp;title=Stability+of+the+nicotine+metabolite+ratio+in+smokers+of+progressively+reduced+nicotine+content+cigarettes&amp;author=G+St+Helen&amp;author=P+Jacob&amp;author=NL+Benowitz&amp;volume=15&amp;issue=11&amp;publication_year=2013&amp;pages=1939-42&amp;pmid=23674838&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Benowitz NL, St Helen G, Dempsey DA, Jacob P, 3rd, Tyndale RF. Disposition kinetics and metabolism of nicotine and cotinine in African American smokers: impact of CYP2A6 genetic variation and enzymatic activity. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2016 Jul;<span class="ref-vol">26</span>(7):340–50.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4892970/">PMC free article</a>]</span> [<a href="/pubmed/27035242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=Disposition+kinetics+and+metabolism+of+nicotine+and+cotinine+in+African+American+smokers:+impact+of+CYP2A6+genetic+variation+and+enzymatic+activity&amp;author=NL+Benowitz&amp;author=G+St+Helen&amp;author=DA+Dempsey&amp;author=P+Jacob&amp;author=RF+Tyndale&amp;volume=26&amp;issue=7&amp;publication_year=2016&amp;pages=340-50&amp;pmid=27035242&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Loukola A, Buchwald J, Gupta R, Palviainen T, Hallfors J, Tikkanen E, et al.  A Genome-Wide Association Study of a Biomarker of Nicotine Metabolism. <span><span class="ref-journal">PLoS Genet. </span>2015;<span class="ref-vol">11</span>(9):e1005498.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4583245/">PMC free article</a>]</span> [<a href="/pubmed/26407342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+Genome-Wide+Association+Study+of+a+Biomarker+of+Nicotine+Metabolism&amp;author=A+Loukola&amp;author=J+Buchwald&amp;author=R+Gupta&amp;author=T+Palviainen&amp;author=J+Hallfors&amp;volume=11&amp;issue=9&amp;publication_year=2015&amp;pages=e1005498&amp;pmid=26407342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Chenoweth MJ, Novalen M, Hawk LW, Jr, Schnoll RA, George TP, Cinciripini PM, et al.  Known and novel sources of variability in the nicotine metabolite ratio in a large sample of treatment-seeking smokers. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2014 Sep;<span class="ref-vol">23</span>(9):1773–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4154993/">PMC free article</a>]</span> [<a href="/pubmed/25012994" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Known+and+novel+sources+of+variability+in+the+nicotine+metabolite+ratio+in+a+large+sample+of+treatment-seeking+smokers&amp;author=MJ+Chenoweth&amp;author=M+Novalen&amp;author=LW+Hawk&amp;author=RA+Schnoll&amp;author=TP+George&amp;volume=23&amp;issue=9&amp;publication_year=2014&amp;pages=1773-82&amp;pmid=25012994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Swan GE, Lessov-Schlaggar CN, Bergen AW, He Y, Tyndale RF, Benowitz NL. Genetic and environmental influences on the ratio of 3’hydroxycotinine to cotinine in plasma and urine. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2009 May;<span class="ref-vol">19</span>(5):388–98.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2849278/">PMC free article</a>]</span> [<a href="/pubmed/19300303" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=Genetic+and+environmental+influences+on+the+ratio+of+3’hydroxycotinine+to+cotinine+in+plasma+and+urine&amp;author=GE+Swan&amp;author=CN+Lessov-Schlaggar&amp;author=AW+Bergen&amp;author=Y+He&amp;author=RF+Tyndale&amp;volume=19&amp;issue=5&amp;publication_year=2009&amp;pages=388-98&amp;pmid=19300303&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Patterson F, Schnoll RA, Wileyto EP, Pinto A, Epstein LH, Shields PG, et al.  Toward personalized therapy for smoking cessation: a randomized placebo-controlled trial of bupropion. <span><span class="ref-journal">Clinical Pharmacology &amp; Therapeutics. </span>2008 Sep;<span class="ref-vol">84</span>(3):320–5.</span> [<a href="/pubmed/18388868" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clinical+Pharmacology+&amp;+Therapeutics&amp;title=Toward+personalized+therapy+for+smoking+cessation:+a+randomized+placebo-controlled+trial+of+bupropion&amp;author=F+Patterson&amp;author=RA+Schnoll&amp;author=EP+Wileyto&amp;author=A+Pinto&amp;author=LH+Epstein&amp;volume=84&amp;issue=3&amp;publication_year=2008&amp;pages=320-5&amp;pmid=18388868&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Lerman C, Tyndale R, Patterson F, Wileyto EP, Shields PG, Pinto A, et al.  Nicotine metabolite ratio predicts efficacy of transdermal nicotine for smoking cessation. <span><span class="ref-journal">Clinical Pharmacology &amp; Therapeutics. </span>2006 Jun;<span class="ref-vol">79</span>(6):600–8.</span> [<a href="/pubmed/16765148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clinical+Pharmacology+&amp;+Therapeutics&amp;title=Nicotine+metabolite+ratio+predicts+efficacy+of+transdermal+nicotine+for+smoking+cessation&amp;author=C+Lerman&amp;author=R+Tyndale&amp;author=F+Patterson&amp;author=EP+Wileyto&amp;author=PG+Shields&amp;volume=79&amp;issue=6&amp;publication_year=2006&amp;pages=600-8&amp;pmid=16765148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Schnoll RA, Patterson F, Wileyto EP, Tyndale RF, Benowitz N, Lerman C. Nicotine metabolic rate predicts successful smoking cessation with transdermal nicotine: a validation study. <span><span class="ref-journal">Pharmacol Biochem Behav. </span>2009 Mar;<span class="ref-vol">92</span>(1):6–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2657225/">PMC free article</a>]</span> [<a href="/pubmed/19000709" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol+Biochem+Behav&amp;title=Nicotine+metabolic+rate+predicts+successful+smoking+cessation+with+transdermal+nicotine:+a+validation+study&amp;author=RA+Schnoll&amp;author=F+Patterson&amp;author=EP+Wileyto&amp;author=RF+Tyndale&amp;author=N+Benowitz&amp;volume=92&amp;issue=1&amp;publication_year=2009&amp;pages=6-11&amp;pmid=19000709&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Lerman C, Jepson C, Wileyto EP, Patterson F, Schnoll R, Mroziewicz M, et al.  Genetic variation in nicotine metabolism predicts the efficacy of extended-duration transdermal nicotine therapy. <span><span class="ref-journal">Clinical Pharmacology &amp; Therapeutics. </span>2010 May;<span class="ref-vol">87</span>(5):553–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3896974/">PMC free article</a>]</span> [<a href="/pubmed/20336063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clinical+Pharmacology+&amp;+Therapeutics&amp;title=Genetic+variation+in+nicotine+metabolism+predicts+the+efficacy+of+extended-duration+transdermal+nicotine+therapy&amp;author=C+Lerman&amp;author=C+Jepson&amp;author=EP+Wileyto&amp;author=F+Patterson&amp;author=R+Schnoll&amp;volume=87&amp;issue=5&amp;publication_year=2010&amp;pages=553-7&amp;pmid=20336063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Lerman C, Schnoll RA, Hawk LW, Cinciripini P, George TP, Wileyto EP, et al.  Use of the nicotine metabolite ratio as a genetically informed biomarker of response to nicotine patch or varenicline for smoking cessation: a randomised, double-blind placebo-controlled trial. <span><span class="ref-journal">Lancet Respiratory Medicine. </span>2015 Feb;<span class="ref-vol">3</span>(2):131–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4480925/">PMC free article</a>]</span> [<a href="/pubmed/25588294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Respiratory+Medicine&amp;title=Use+of+the+nicotine+metabolite+ratio+as+a+genetically+informed+biomarker+of+response+to+nicotine+patch+or+varenicline+for+smoking+cessation:+a+randomised,+double-blind+placebo-controlled+trial&amp;author=C+Lerman&amp;author=RA+Schnoll&amp;author=LW+Hawk&amp;author=P+Cinciripini&amp;author=TP+George&amp;volume=3&amp;issue=2&amp;publication_year=2015&amp;pages=131-8&amp;pmid=25588294&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Wassenaar CA, Dong Q, Wei Q, Amos CI, Spitz MR, Tyndale RF. Relationship between CYP2A6 and CHRNA5-CHRNA3-CHRNB4 variation and smoking behaviors and lung cancer risk. <span><span class="ref-journal">J Natl Cancer Inst. </span>2011 Sep 7;<span class="ref-vol">103</span>(17):1342–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3168937/">PMC free article</a>]</span> [<a href="/pubmed/21747048" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Relationship+between+CYP2A6+and+CHRNA5-CHRNA3-CHRNB4+variation+and+smoking+behaviors+and+lung+cancer+risk&amp;author=CA+Wassenaar&amp;author=Q+Dong&amp;author=Q+Wei&amp;author=CI+Amos&amp;author=MR+Spitz&amp;volume=103&amp;issue=17&amp;publication_year=2011&amp;pages=1342-6&amp;pmid=21747048&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Sofuoglu M, Herman AI, Nadim H, Jatlow P. Rapid nicotine clearance is associated with greater reward and heart rate increases from intravenous nicotine. <span><span class="ref-journal">Neuropsychopharmacology. </span>2012 May;<span class="ref-vol">37</span>(6):1509–16.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3327855/">PMC free article</a>]</span> [<a href="/pubmed/22334123" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&amp;title=Rapid+nicotine+clearance+is+associated+with+greater+reward+and+heart+rate+increases+from+intravenous+nicotine&amp;author=M+Sofuoglu&amp;author=AI+Herman&amp;author=H+Nadim&amp;author=P+Jatlow&amp;volume=37&amp;issue=6&amp;publication_year=2012&amp;pages=1509-16&amp;pmid=22334123&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Chenoweth MJ, Tyndale RF. Pharmacogenetic Optimization of Smoking Cessation Treatment. <span><span class="ref-journal">Trends Pharmacol Sci. </span>2016 Oct 3;</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5195866/">PMC free article</a>]</span> [<a href="/pubmed/27712845" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Pharmacol+Sci&amp;title=Pharmacogenetic+Optimization+of+Smoking+Cessation+Treatment&amp;author=MJ+Chenoweth&amp;author=RF+Tyndale&amp;publication_year=2016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Kandel DB, Hu MC, Schaffran C, Udry JR, Benowitz NL. Urine nicotine metabolites and smoking behavior in a multiracial/multiethnic national sample of young adults. <span><span class="ref-journal">Am J Epidemiol. </span>2007 Apr 15;<span class="ref-vol">165</span>(8):901–10.</span> [<a href="/pubmed/17322544" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Urine+nicotine+metabolites+and+smoking+behavior+in+a+multiracial/multiethnic+national+sample+of+young+adults&amp;author=DB+Kandel&amp;author=MC+Hu&amp;author=C+Schaffran&amp;author=JR+Udry&amp;author=NL+Benowitz&amp;volume=165&amp;issue=8&amp;publication_year=2007&amp;pages=901-10&amp;pmid=17322544&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Nakajima M, Fukami T, Yamanaka H, Higashi E, Sakai H, Yoshida R, et al.  Comprehensive evaluation of variability in nicotine metabolism and CYP2A6 polymorphic alleles in four ethnic populations. <span><span class="ref-journal">Clinical Pharmacology &amp; Therapeutics. </span>2006 Sep;<span class="ref-vol">80</span>(3):282–97.</span> [<a href="/pubmed/16952495" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clinical+Pharmacology+&amp;+Therapeutics&amp;title=Comprehensive+evaluation+of+variability+in+nicotine+metabolism+and+CYP2A6+polymorphic+alleles+in+four+ethnic+populations&amp;author=M+Nakajima&amp;author=T+Fukami&amp;author=H+Yamanaka&amp;author=E+Higashi&amp;author=H+Sakai&amp;volume=80&amp;issue=3&amp;publication_year=2006&amp;pages=282-97&amp;pmid=16952495&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Ho MK, Mwenifumbo JC, Zhao B, Gillam EM, Tyndale RF. A novel CYP2A6 allele, CYP2A6*23, impairs enzyme function in vitro and in vivo and decreases smoking in a population of Black-African descent. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2008 Jan;<span class="ref-vol">18</span>(1):67–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2910083/">PMC free article</a>]</span> [<a href="/pubmed/18216723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=A+novel+CYP2A6+allele,+CYP2A6*23,+impairs+enzyme+function+in+vitro+and+in+vivo+and+decreases+smoking+in+a+population+of+Black-African+descent&amp;author=MK+Ho&amp;author=JC+Mwenifumbo&amp;author=B+Zhao&amp;author=EM+Gillam&amp;author=RF+Tyndale&amp;volume=18&amp;issue=1&amp;publication_year=2008&amp;pages=67-75&amp;pmid=18216723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Mwenifumbo JC, AlKoudsi N, Ho MK, Zhou Q, Hoffmann EB, Sellers EM, et al.  Novel and established CYP2A6 alleles impair in vivo nicotine metabolism in a population of Black African descent. <span><span class="ref-journal">Hum Mutat. </span>2008 May;<span class="ref-vol">29</span>(5):679–88.</span> [<a href="/pubmed/18360915" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mutat&amp;title=Novel+and+established+CYP2A6+alleles+impair+in+vivo+nicotine+metabolism+in+a+population+of+Black+African+descent&amp;author=JC+Mwenifumbo&amp;author=N+AlKoudsi&amp;author=MK+Ho&amp;author=Q+Zhou&amp;author=EB+Hoffmann&amp;volume=29&amp;issue=5&amp;publication_year=2008&amp;pages=679-88&amp;pmid=18360915&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Al Koudsi N, Ahluwalia JS, Lin SK, Sellers EM, Tyndale RF. A novel CYP2A6 allele (CYP2A6*35) resulting in an amino-acid substitution (Asn438Tyr) is associated with lower CYP2A6 activity in vivo. <span><span class="ref-journal">Pharmacogenomics J. </span>2009 Aug;<span class="ref-vol">9</span>(4):274–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922203/">PMC free article</a>]</span> [<a href="/pubmed/19365400" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics+J&amp;title=A+novel+CYP2A6+allele+(CYP2A6*35)+resulting+in+an+amino-acid+substitution+(Asn438Tyr)+is+associated+with+lower+CYP2A6+activity+in+vivo&amp;author=N+Al+Koudsi&amp;author=JS+Ahluwalia&amp;author=SK+Lin&amp;author=EM+Sellers&amp;author=RF+Tyndale&amp;volume=9&amp;issue=4&amp;publication_year=2009&amp;pages=274-82&amp;pmid=19365400&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Piliguian M, Zhu AZ, Zhou Q, Benowitz NL, Ahluwalia JS, Sanderson Cox L, et al.  Novel CYP2A6 variants identified in African Americans are associated with slow nicotine metabolism in vitro and in vivo. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2014 Feb;<span class="ref-vol">24</span>(2):118–28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3931261/">PMC free article</a>]</span> [<a href="/pubmed/24305170" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=Novel+CYP2A6+variants+identified+in+African+Americans+are+associated+with+slow+nicotine+metabolism+in+vitro+and+in+vivo&amp;author=M+Piliguian&amp;author=AZ+Zhu&amp;author=Q+Zhou&amp;author=NL+Benowitz&amp;author=JS+Ahluwalia&amp;volume=24&amp;issue=2&amp;publication_year=2014&amp;pages=118-28&amp;pmid=24305170&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Fukami T, Nakajima M, Yoshida R, Tsuchiya Y, Fujiki Y, Katoh M, et al.  A novel polymorphism of human CYP2A6 gene CTP2A6*17 has an amino acid substitution (V365M) that decreases enzymatic activity in vitro and in vivo. <span><span class="ref-journal">Clinical Pharmacology &amp; Therapeutics. </span>2004 Dec;<span class="ref-vol">76</span>(6):519–27.</span> [<a href="/pubmed/15592323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clinical+Pharmacology+&amp;+Therapeutics&amp;title=A+novel+polymorphism+of+human+CYP2A6+gene+CTP2A6*17+has+an+amino+acid+substitution+(V365M)+that+decreases+enzymatic+activity+in+vitro+and+in+vivo&amp;author=T+Fukami&amp;author=M+Nakajima&amp;author=R+Yoshida&amp;author=Y+Tsuchiya&amp;author=Y+Fujiki&amp;volume=76&amp;issue=6&amp;publication_year=2004&amp;pages=519-27&amp;pmid=15592323&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Cox LS, Nollen NL, Mayo MS, Choi WS, Faseru B, Benowitz NL, et al.  Bupropion for smoking cessation in African American light smokers: a randomized controlled trial. <span><span class="ref-journal">J Natl Cancer Inst. </span>2012 Feb 22;<span class="ref-vol">104</span>(4):290–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3283533/">PMC free article</a>]</span> [<a href="/pubmed/22282543" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Bupropion+for+smoking+cessation+in+African+American+light+smokers:+a+randomized+controlled+trial&amp;author=LS+Cox&amp;author=NL+Nollen&amp;author=MS+Mayo&amp;author=WS+Choi&amp;author=B+Faseru&amp;volume=104&amp;issue=4&amp;publication_year=2012&amp;pages=290-8&amp;pmid=22282543&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Baurley JW, Edlund CK, Pardamean CI, Conti DV, Krasnow R, Javitz HS, et al.  Genome-Wide Association of the Laboratory-Based Nicotine Metabolite Ratio in Three Ancestries. <span><span class="ref-journal">Nicotine Tob Res. </span>2016 Apr 25;</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4978985/">PMC free article</a>]</span> [<a href="/pubmed/27113016" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nicotine+Tob+Res&amp;title=Genome-Wide+Association+of+the+Laboratory-Based+Nicotine+Metabolite+Ratio+in+Three+Ancestries&amp;author=JW+Baurley&amp;author=CK+Edlund&amp;author=CI+Pardamean&amp;author=DV+Conti&amp;author=R+Krasnow&amp;publication_year=2016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Patel YM, Park SL, Han Y, Wilkens LR, Bickeboller H, Rosenberger A, et al.  Novel Association of Genetic Markers Affecting CYP2A6 Activity and Lung Cancer Risk. <span><span class="ref-journal">Cancer Res. </span>2016 Oct 1;<span class="ref-vol">76</span>(19):5768–76.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5050097/">PMC free article</a>]</span> [<a href="/pubmed/27488534" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Novel+Association+of+Genetic+Markers+Affecting+CYP2A6+Activity+and+Lung+Cancer+Risk&amp;author=YM+Patel&amp;author=SL+Park&amp;author=Y+Han&amp;author=LR+Wilkens&amp;author=H+Bickeboller&amp;volume=76&amp;issue=19&amp;publication_year=2016&amp;pages=5768-76&amp;pmid=27488534&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Cox LS, Faseru B, Mayo MS, Krebill R, Snow TS, Bronars CA, et al.  Design, baseline characteristics, and retention of African American light smokers into a randomized trial involving biological data. <span><span class="ref-journal">Trials. </span>2011 Jan 25;<span class="ref-vol">12</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3038942/">PMC free article</a>]</span> [<a href="/pubmed/21266057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trials&amp;title=Design,+baseline+characteristics,+and+retention+of+African+American+light+smokers+into+a+randomized+trial+involving+biological+data&amp;author=LS+Cox&amp;author=B+Faseru&amp;author=MS+Mayo&amp;author=R+Krebill&amp;author=TS+Snow&amp;volume=12&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Ahluwalia JS, Okuyemi K, Nollen N, Choi WS, Kaur H, Pulvers K, et al.  The effects of nicotine gum and counseling among African American light smokers: a 2 × 2 factorial design. <span><span class="ref-journal">Addiction. </span>2006 Jun;<span class="ref-vol">101</span>(6):883–91.</span> [<a href="/pubmed/16696632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Addiction&amp;title=The+effects+of+nicotine+gum+and+counseling+among+African+American+light+smokers:+a+2+×+2+factorial+design&amp;author=JS+Ahluwalia&amp;author=K+Okuyemi&amp;author=N+Nollen&amp;author=WS+Choi&amp;author=H+Kaur&amp;volume=101&amp;issue=6&amp;publication_year=2006&amp;pages=883-91&amp;pmid=16696632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007 Sep;<span class="ref-vol">81</span>(3):559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Delaneau O, Marchini J. Integrating sequence and array data to create an improved 1000 Genomes Project haplotype reference panel. <span><span class="ref-journal">Nat Commun. </span>2014;<span class="ref-vol">5</span>:3934.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4338501/">PMC free article</a>]</span> [<a href="/pubmed/25653097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Integrating+sequence+and+array+data+to+create+an+improved+1000+Genomes+Project+haplotype+reference+panel&amp;author=O+Delaneau&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2014&amp;pages=3934&amp;pmid=25653097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009 Jun;<span class="ref-vol">5</span>(6):e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;issue=6&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Delaneau O, Howie B, Cox AJ, Zagury JF, Marchini J. Haplotype Estimation Using Sequencing Reads. <span><span class="ref-journal">American Journal of Human Genetics. </span>2013 Oct 3;<span class="ref-vol">93</span>(4):687–96.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3791270/">PMC free article</a>]</span> [<a href="/pubmed/24094745" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Haplotype+Estimation+Using+Sequencing+Reads&amp;author=O+Delaneau&amp;author=B+Howie&amp;author=AJ+Cox&amp;author=JF+Zagury&amp;author=J+Marchini&amp;volume=93&amp;issue=4&amp;publication_year=2013&amp;pages=687-96&amp;pmid=24094745&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Howie B, Marchini J, Stephens M. Genotype Imputation with Thousands of Genomes. <span><span class="ref-journal">G3-Genes Genomes Genetics. </span>2011 Nov 1;<span class="ref-vol">1</span>(6):457–69.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3276165/">PMC free article</a>]</span> [<a href="/pubmed/22384356" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=G3-Genes+Genomes+Genetics&amp;title=Genotype+Imputation+with+Thousands+of+Genomes&amp;author=B+Howie&amp;author=J+Marchini&amp;author=M+Stephens&amp;volume=1&amp;issue=6&amp;publication_year=2011&amp;pages=457-69&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">O’Connell J, Gurdasani D, Delaneau O, Pirastu N, Ulivi S, Cocca M, et al.  A general approach for haplotype phasing across the full spectrum of relatedness. <span><span class="ref-journal">PLoS Genet. </span>2014 Apr;<span class="ref-vol">10</span>(4):e1004234.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3990520/">PMC free article</a>]</span> [<a href="/pubmed/24743097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+general+approach+for+haplotype+phasing+across+the+full+spectrum+of+relatedness&amp;author=J+O’Connell&amp;author=D+Gurdasani&amp;author=O+Delaneau&amp;author=N+Pirastu&amp;author=S+Ulivi&amp;volume=10&amp;issue=4&amp;publication_year=2014&amp;pages=e1004234&amp;pmid=24743097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Roshyara NR, Kirsten H, Horn K, Ahnert P, Scholz M. Impact of pre-imputation SNP-filtering on genotype imputation results. <span><span class="ref-journal">BMC Genet. </span>2014 Aug 12;<span class="ref-vol">15</span>:88.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4236550/">PMC free article</a>]</span> [<a href="/pubmed/25112433" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet&amp;title=Impact+of+pre-imputation+SNP-filtering+on+genotype+imputation+results&amp;author=NR+Roshyara&amp;author=H+Kirsten&amp;author=K+Horn&amp;author=P+Ahnert&amp;author=M+Scholz&amp;volume=15&amp;publication_year=2014&amp;pages=88&amp;pmid=25112433&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Wassenaar CA, Zhou Q, Tyndale RF. CYP2A6 genotyping methods and strategies using real-time and end point PCR platforms. <span><span class="ref-journal">Pharmacogenomics. </span>2016;<span class="ref-vol">17</span>(2):147–62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4712027/">PMC free article</a>]</span> [<a href="/pubmed/26670214" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&amp;title=CYP2A6+genotyping+methods+and+strategies+using+real-time+and+end+point+PCR+platforms&amp;author=CA+Wassenaar&amp;author=Q+Zhou&amp;author=RF+Tyndale&amp;volume=17&amp;issue=2&amp;publication_year=2016&amp;pages=147-62&amp;pmid=26670214&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Schnoll RA, George TP, Hawk L, Cinciripini P, Wileyto P, Tyndale RF. The relationship between the nicotine metabolite ratio and three self-report measures of nicotine dependence across sex and race. <span><span class="ref-journal">Psychopharmacology. </span>2014 Jun;<span class="ref-vol">231</span>(12):2515–23.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4040302/">PMC free article</a>]</span> [<a href="/pubmed/24402139" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychopharmacology&amp;title=The+relationship+between+the+nicotine+metabolite+ratio+and+three+self-report+measures+of+nicotine+dependence+across+sex+and+race&amp;author=RA+Schnoll&amp;author=TP+George&amp;author=L+Hawk&amp;author=P+Cinciripini&amp;author=P+Wileyto&amp;volume=231&amp;issue=12&amp;publication_year=2014&amp;pages=2515-23&amp;pmid=24402139&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Vartiainen E, Seppala T, Lillsunde P, Puska P. Validation of self reported smoking by serum cotinine measurement in a community-based study. <span><span class="ref-journal">J Epidemiol Community Health. </span>2002 Mar;<span class="ref-vol">56</span>(3):167–70.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1732104/">PMC free article</a>]</span> [<a href="/pubmed/11854334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Epidemiol+Community+Health&amp;title=Validation+of+self+reported+smoking+by+serum+cotinine+measurement+in+a+community-based+study&amp;author=E+Vartiainen&amp;author=T+Seppala&amp;author=P+Lillsunde&amp;author=P+Puska&amp;volume=56&amp;issue=3&amp;publication_year=2002&amp;pages=167-70&amp;pmid=11854334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006 Aug;<span class="ref-vol">38</span>(8):904–9.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;issue=8&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Risch N, Merikangas K. The future of genetic studies of complex human diseases. <span><span class="ref-journal">Science. </span>1996 Sep 13;<span class="ref-vol">273</span>(5281):1516–7.</span> [<a href="/pubmed/8801636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+future+of+genetic+studies+of+complex+human+diseases&amp;author=N+Risch&amp;author=K+Merikangas&amp;volume=273&amp;issue=5281&amp;publication_year=1996&amp;pages=1516-7&amp;pmid=8801636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Liu JZ, Tozzi F, Waterworth DM, Pillai SG, Muglia P, Middleton L, et al.  Meta-analysis and imputation refines the association of 15q25 with smoking quantity. <span><span class="ref-journal">Nat Genet. </span>2010 May;<span class="ref-vol">42</span>(5):436–40.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3612983/">PMC free article</a>]</span> [<a href="/pubmed/20418889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+and+imputation+refines+the+association+of+15q25+with+smoking+quantity&amp;author=JZ+Liu&amp;author=F+Tozzi&amp;author=DM+Waterworth&amp;author=SG+Pillai&amp;author=P+Muglia&amp;volume=42&amp;issue=5&amp;publication_year=2010&amp;pages=436-40&amp;pmid=20418889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Benowitz NL, Hukkanen J, Jacob P., 3rd Nicotine chemistry, metabolism, kinetics and biomarkers. <span><span class="ref-journal">Handb Exp Pharmacol. </span>2009;(192):29–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2953858/">PMC free article</a>]</span> [<a href="/pubmed/19184645" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Handb+Exp+Pharmacol&amp;title=Nicotine+chemistry,+metabolism,+kinetics+and+biomarkers&amp;author=NL+Benowitz&amp;author=J+Hukkanen&amp;author=P+Jacob&amp;issue=192&amp;publication_year=2009&amp;pages=29-60&amp;pmid=19184645&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Murai K, Yamazaki H, Nakagawa K, Kawai R, Kamataki T. Deactivation of anti-cancer drug letrozole to a carbinol metabolite by polymorphic cytochrome P450 2A6 in human liver microsomes. <span><span class="ref-journal">Xenobiotica. </span>2009 Nov;<span class="ref-vol">39</span>(11):795–802.</span> [<a href="/pubmed/19845430" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Xenobiotica&amp;title=Deactivation+of+anti-cancer+drug+letrozole+to+a+carbinol+metabolite+by+polymorphic+cytochrome+P450+2A6+in+human+liver+microsomes&amp;author=K+Murai&amp;author=H+Yamazaki&amp;author=K+Nakagawa&amp;author=R+Kawai&amp;author=T+Kamataki&amp;volume=39&amp;issue=11&amp;publication_year=2009&amp;pages=795-802&amp;pmid=19845430&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Ikeda K, Yoshisue K, Matsushima E, Nagayama S, Kobayashi K, Tyson CA, et al.  Bioactivation of tegafur to 5-fluorouracil is catalyzed by cytochrome P-450 2A6 in human liver microsomes in vitro. <span><span class="ref-journal">Clin Cancer Res. </span>2000 Nov;<span class="ref-vol">6</span>(11):4409–15.</span> [<a href="/pubmed/11106261" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Cancer+Res&amp;title=Bioactivation+of+tegafur+to+5-fluorouracil+is+catalyzed+by+cytochrome+P-450+2A6+in+human+liver+microsomes+in+vitro&amp;author=K+Ikeda&amp;author=K+Yoshisue&amp;author=E+Matsushima&amp;author=S+Nagayama&amp;author=K+Kobayashi&amp;volume=6&amp;issue=11&amp;publication_year=2000&amp;pages=4409-15&amp;pmid=11106261&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">McDonagh EM, Wassenaar C, David SP, Tyndale RF, Altman RB, Whirl-Carrillo M, et al.  PharmGKB summary: very important pharmacogene information for cytochrome P-450, family 2, subfamily A, polypeptide 6. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2012 Sep;<span class="ref-vol">22</span>(9):695–708.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3413746/">PMC free article</a>]</span> [<a href="/pubmed/22547082" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=PharmGKB+summary:+very+important+pharmacogene+information+for+cytochrome+P-450,+family+2,+subfamily+A,+polypeptide+6&amp;author=EM+McDonagh&amp;author=C+Wassenaar&amp;author=SP+David&amp;author=RF+Tyndale&amp;author=RB+Altman&amp;volume=22&amp;issue=9&amp;publication_year=2012&amp;pages=695-708&amp;pmid=22547082&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Kozyra M, Ingelman-Sundberg M, Lauschke VM. Rare genetic variants in cellular transporters, metabolic enzymes, and nuclear receptors can be important determinants of interindividual differences in drug response. <span><span class="ref-journal">Genet Med. </span>2016 Apr 21;</span> [<a href="/pubmed/27101133" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Med&amp;title=Rare+genetic+variants+in+cellular+transporters,+metabolic+enzymes,+and+nuclear+receptors+can+be+important+determinants+of+interindividual+differences+in+drug+response&amp;author=M+Kozyra&amp;author=M+Ingelman-Sundberg&amp;author=VM+Lauschke&amp;publication_year=2016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Fujikura K, Ingelman-Sundberg M, Lauschke VM. Genetic variation in the human cytochrome P450 supergene family. <span><span class="ref-journal">Pharmacogenetics and Genomics. </span>2015 Dec;<span class="ref-vol">25</span>(12):584–94.</span> [<a href="/pubmed/26340336" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics+and+Genomics&amp;title=Genetic+variation+in+the+human+cytochrome+P450+supergene+family&amp;author=K+Fujikura&amp;author=M+Ingelman-Sundberg&amp;author=VM+Lauschke&amp;volume=25&amp;issue=12&amp;publication_year=2015&amp;pages=584-94&amp;pmid=26340336&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5807179&amp;issue-id=306441&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139773418089120title">Abstract</a></li><li><a href="#S8title">Introduction</a></li><li><a href="#S9title">Methods</a></li><li><a href="#S24title">Results</a></li><li><a href="#S28title">Discussion</a></li><li><a href="#S29title">Supplementary Material</a></li><li><a href="#S30title">Acknowledgments</a></li><li><a href="#idm139773376956896title">Footnotes</a></li><li><a href="#idm139773378710400title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5807179/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5807179/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5807179/pdf/nihms907043.pdf">PDF (1.0M)</a></li> | <li><a href="#" data-citationid="PMC5807179" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5807179%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5807179%2F&amp;text=Genome-wide%20association%20study%20of%20a%20nicotine%20metabolism%20biomarker%20in%20African%20American%20smokers%3A%20impact%20of%20chromosome%2019%20genetic%20influences" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5807179%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28921760" data-aiid="5807179" data-aid="5807179" data-iid="306441" data-domainid="319" data-domain="nihpa" data-accid="PMC5807179" data-md5="87c7838bbdb894508d249c52121c062c"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:34:00-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D843CDD5AE4E10000000001B400F7&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5807179%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5807179/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>