<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5974016/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A novel variant associated with HDL-C levels by modifying DAGLB expression levels: An annotation-based genome-wide association study</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5974016/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="A novel variant associated with HDL-C levels by modifying DAGLB expression levels: An annotation-based genome-wide association study"><meta name="citation_authors" content="Dan  Zhou, Dandan Zhang, Xiaohui Sun, Zhiqiang Li, Yaqin Ni, Zhongyan Shan, Hong Li, Chengguo Liu, Shuai Zhang, Yi Liu, Ruizhi Zheng, Feixia Pan, Yimin Zhu, Yongyong Shi, Maode Lai"><meta name="citation_date" content="June 2018"><meta name="citation_issue" content="6"><meta name="citation_volume" content="26"><meta name="citation_firstpage" content="838"><meta name="citation_doi" content="10.1038/s41431-018-0108-4"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5974016/?report=abstract"><meta name="citation_pmid" content="29476167"><meta name="DC.Title" content="A novel variant associated with HDL-C levels by modifying DAGLB expression levels: An annotation-based genome-wide association study"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Dan  Zhou"><meta name="DC.Contributor" content="Dandan Zhang"><meta name="DC.Contributor" content="Xiaohui Sun"><meta name="DC.Contributor" content="Zhiqiang Li"><meta name="DC.Contributor" content="Yaqin Ni"><meta name="DC.Contributor" content="Zhongyan Shan"><meta name="DC.Contributor" content="Hong Li"><meta name="DC.Contributor" content="Chengguo Liu"><meta name="DC.Contributor" content="Shuai Zhang"><meta name="DC.Contributor" content="Yi Liu"><meta name="DC.Contributor" content="Ruizhi Zheng"><meta name="DC.Contributor" content="Feixia Pan"><meta name="DC.Contributor" content="Yimin Zhu"><meta name="DC.Contributor" content="Yongyong Shi"><meta name="DC.Contributor" content="Maode Lai"><meta name="DC.Date" content="2018 Jun"><meta name="DC.Identifier" content="10.1038/s41431-018-0108-4"><meta name="DC.Language" content="en"><meta property="og:title" content="A novel variant associated with HDL-C levels by modifying DAGLB expression levels: An annotation-based genome-wide association study"><meta property="og:type" content="article"><meta property="og:description" content="Although numbers of genome-wide association studies (GWAS) have been performed for serum lipid levels, limited heritability has been explained. Studies showed that combining data from GWAS and expression quantitative trait loci (eQTLs) signals can both ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5974016/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5974016/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5974016/pdf/41431_2018_Article_108.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A5E7CDD5B44D10000000001B30100.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5974016%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/314102/">v.26(6); 2018 Jun</a></li><li class="accid">PMC5974016</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2018 Jun; 26(6): 838–847. </span></div><div><span class="fm-vol-iss-date">Published online 2018 Feb 23. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fs41431-018-0108-4" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/s41431-018-0108-4</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5974016</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29476167">29476167</a></div></div></div></div><h1 class="content-title">A novel variant associated with HDL-C levels by modifying <em>DAGLB</em> expression levels: An annotation-based genome-wide association study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=%20Zhou%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669434265856" co-class="co-affbox">Dan  Zhou</a>,<sup>#</sup><sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Zhang%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669399815152" co-class="co-affbox">Dandan Zhang</a>,<sup>#</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Sun%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669382820496" co-class="co-affbox">Xiaohui Sun</a>,<sup>#</sup><sup>1</sup> <a href="/pubmed/?term=Li%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669403472432" co-class="co-affbox">Zhiqiang Li</a>,<sup>#</sup><sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Ni%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669391486304" co-class="co-affbox">Yaqin Ni</a>,<sup>1</sup> <a href="/pubmed/?term=Shan%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669390954224" co-class="co-affbox">Zhongyan Shan</a>,<sup>6</sup> <a href="/pubmed/?term=Li%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669390952240" co-class="co-affbox">Hong Li</a>,<sup>7</sup> <a href="/pubmed/?term=Liu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669391248112" co-class="co-affbox">Chengguo Liu</a>,<sup>8</sup> <a href="/pubmed/?term=Zhang%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669391246128" co-class="co-affbox">Shuai Zhang</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669402626368" co-class="co-affbox">Yi Liu</a>,<sup>1</sup> <a href="/pubmed/?term=Zheng%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669402624384" co-class="co-affbox">Ruizhi Zheng</a>,<sup>1</sup> <a href="/pubmed/?term=Pan%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669390498192" co-class="co-affbox">Feixia Pan</a>,<sup>1</sup> <a href="/pubmed/?term=Zhu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669394139776" co-class="co-affbox">Yimin Zhu</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>#</sup><sup>1</sup> <a href="/pubmed/?term=Shi%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669394136928" co-class="co-affbox">Yongyong Shi</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>#</sup><sup>4,</sup><sup>5,</sup><sup>9</sup> and  <a href="/pubmed/?term=Lai%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167" class="affpopup" co-rid="_co_idm140669434262416" co-class="co-affbox">Maode Lai</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>#</sup><sup>2,</sup><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140669434265856"><h3 class="no_margin">Dan  Zhou</h3><p><sup>1</sup>Department of Epidemiology &amp; Biostatistics, Zhejiang University School of Public Health, Hangzhou, Zhejiang 310058 China </p><p><sup>2</sup>Department of Pathology, Zhejiang University School of Medicine, Hangzhou, Zhejiang 310058 China </p><p><sup>3</sup>Key Laboratory of Disease Proteomics of Zhejiang Province, Hangzhou, Zhejiang 310058 China </p><div>Find articles by <a href="/pubmed/?term=%20Zhou%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Dan  Zhou</a></div></div><div id="_co_idm140669399815152"><h3 class="no_margin">Dandan Zhang</h3><p><sup>2</sup>Department of Pathology, Zhejiang University School of Medicine, Hangzhou, Zhejiang 310058 China </p><p><sup>3</sup>Key Laboratory of Disease Proteomics of Zhejiang Province, Hangzhou, Zhejiang 310058 China </p><div>Find articles by <a href="/pubmed/?term=Zhang%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Dandan Zhang</a></div></div><div id="_co_idm140669382820496"><h3 class="no_margin">Xiaohui Sun</h3><p><sup>1</sup>Department of Epidemiology &amp; Biostatistics, Zhejiang University School of Public Health, Hangzhou, Zhejiang 310058 China </p><div>Find articles by <a href="/pubmed/?term=Sun%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Xiaohui Sun</a></div></div><div id="_co_idm140669403472432"><h3 class="no_margin">Zhiqiang Li</h3><p><sup>4</sup>The Affiliated Hospital of Qingdao University &amp; The Biomedical Sciences Institute of Qingdao University (Qingdao Branch of SJTU Bio-X Institutes), Qingdao University, Qingdao, 266000 China </p><p><sup>5</sup>Bio-X Institutes, Key Laboratory for the Genetics of Developmental and Neuropsychiatric Disorders (Ministry of Education) Collaborative Innovation Center for Brain Science, Shanghai Jiao Tong University, Shanghai, 200030 China </p><div>Find articles by <a href="/pubmed/?term=Li%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Zhiqiang Li</a></div></div><div id="_co_idm140669391486304"><h3 class="no_margin">Yaqin Ni</h3><p><sup>1</sup>Department of Epidemiology &amp; Biostatistics, Zhejiang University School of Public Health, Hangzhou, Zhejiang 310058 China </p><div>Find articles by <a href="/pubmed/?term=Ni%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Yaqin Ni</a></div></div><div id="_co_idm140669390954224"><h3 class="no_margin">Zhongyan Shan</h3><p><sup>6</sup>The Endocrine Institute and Liaoning Provincial Key Laboratory of Endocrine Diseases, Department of Endocrinology and Metabolism, The First Hospital of China Medical University, Shenyang, Liaoning 110001 China </p><div>Find articles by <a href="/pubmed/?term=Shan%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Zhongyan Shan</a></div></div><div id="_co_idm140669390952240"><h3 class="no_margin">Hong Li</h3><p><sup>7</sup>Department of Endocrinology, Sir Run Run Shaw Hospital Affiliated to School of Medicine, Zhejiang University, Hangzhou, Zhejiang 310020 China </p><div>Find articles by <a href="/pubmed/?term=Li%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Hong Li</a></div></div><div id="_co_idm140669391248112"><h3 class="no_margin">Chengguo Liu</h3><p><sup>8</sup>Putuo District People’s Hospital, Zhoushan, Zhejiang 316100 China </p><div>Find articles by <a href="/pubmed/?term=Liu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Chengguo Liu</a></div></div><div id="_co_idm140669391246128"><h3 class="no_margin">Shuai Zhang</h3><p><sup>2</sup>Department of Pathology, Zhejiang University School of Medicine, Hangzhou, Zhejiang 310058 China </p><p><sup>3</sup>Key Laboratory of Disease Proteomics of Zhejiang Province, Hangzhou, Zhejiang 310058 China </p><div>Find articles by <a href="/pubmed/?term=Zhang%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Shuai Zhang</a></div></div><div id="_co_idm140669402626368"><h3 class="no_margin">Yi Liu</h3><p><sup>1</sup>Department of Epidemiology &amp; Biostatistics, Zhejiang University School of Public Health, Hangzhou, Zhejiang 310058 China </p><div>Find articles by <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Yi Liu</a></div></div><div id="_co_idm140669402624384"><h3 class="no_margin">Ruizhi Zheng</h3><p><sup>1</sup>Department of Epidemiology &amp; Biostatistics, Zhejiang University School of Public Health, Hangzhou, Zhejiang 310058 China </p><div>Find articles by <a href="/pubmed/?term=Zheng%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Ruizhi Zheng</a></div></div><div id="_co_idm140669390498192"><h3 class="no_margin">Feixia Pan</h3><p><sup>1</sup>Department of Epidemiology &amp; Biostatistics, Zhejiang University School of Public Health, Hangzhou, Zhejiang 310058 China </p><div>Find articles by <a href="/pubmed/?term=Pan%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Feixia Pan</a></div></div><div id="_co_idm140669394139776"><h3 class="no_margin">Yimin Zhu</h3><p><sup>1</sup>Department of Epidemiology &amp; Biostatistics, Zhejiang University School of Public Health, Hangzhou, Zhejiang 310058 China </p><div>Find articles by <a href="/pubmed/?term=Zhu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Yimin Zhu</a></div></div><div id="_co_idm140669394136928"><h3 class="no_margin">Yongyong Shi</h3><p><sup>4</sup>The Affiliated Hospital of Qingdao University &amp; The Biomedical Sciences Institute of Qingdao University (Qingdao Branch of SJTU Bio-X Institutes), Qingdao University, Qingdao, 266000 China </p><p><sup>5</sup>Bio-X Institutes, Key Laboratory for the Genetics of Developmental and Neuropsychiatric Disorders (Ministry of Education) Collaborative Innovation Center for Brain Science, Shanghai Jiao Tong University, Shanghai, 200030 China </p><p><sup>9</sup>Department of Psychiatry, The First Teaching Hospital of Xinjiang Medical University, Urumqi, 830000 China </p><div>Find articles by <a href="/pubmed/?term=Shi%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Yongyong Shi</a></div></div><div id="_co_idm140669434262416"><h3 class="no_margin">Maode Lai</h3><p><sup>2</sup>Department of Pathology, Zhejiang University School of Medicine, Hangzhou, Zhejiang 310058 China </p><p><sup>3</sup>Key Laboratory of Disease Proteomics of Zhejiang Province, Hangzhou, Zhejiang 310058 China </p><div>Find articles by <a href="/pubmed/?term=Lai%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29476167">Maode Lai</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140669390239792_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140669390239792_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140669390239792_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140669390239792_ai" style="display:none"><div class="fm-affl" id="Aff1" lang="en"><sup>1</sup>Department of Epidemiology &amp; Biostatistics, Zhejiang University School of Public Health, Hangzhou, Zhejiang 310058 China </div><div class="fm-affl" id="Aff2" lang="en"><sup>2</sup>Department of Pathology, Zhejiang University School of Medicine, Hangzhou, Zhejiang 310058 China </div><div class="fm-affl" id="Aff3" lang="en"><sup>3</sup>Key Laboratory of Disease Proteomics of Zhejiang Province, Hangzhou, Zhejiang 310058 China </div><div class="fm-affl" id="Aff4" lang="en"><sup>4</sup>The Affiliated Hospital of Qingdao University &amp; The Biomedical Sciences Institute of Qingdao University (Qingdao Branch of SJTU Bio-X Institutes), Qingdao University, Qingdao, 266000 China </div><div class="fm-affl" id="Aff5" lang="en"><sup>5</sup>Bio-X Institutes, Key Laboratory for the Genetics of Developmental and Neuropsychiatric Disorders (Ministry of Education) Collaborative Innovation Center for Brain Science, Shanghai Jiao Tong University, Shanghai, 200030 China </div><div class="fm-affl" id="Aff6" lang="en"><sup>6</sup>The Endocrine Institute and Liaoning Provincial Key Laboratory of Endocrine Diseases, Department of Endocrinology and Metabolism, The First Hospital of China Medical University, Shenyang, Liaoning 110001 China </div><div class="fm-affl" id="Aff7" lang="en"><sup>7</sup>Department of Endocrinology, Sir Run Run Shaw Hospital Affiliated to School of Medicine, Zhejiang University, Hangzhou, Zhejiang 310020 China </div><div class="fm-affl" id="Aff8" lang="en"><sup>8</sup>Putuo District People’s Hospital, Zhoushan, Zhejiang 316100 China </div><div class="fm-affl" id="Aff9" lang="en"><sup>9</sup>Department of Psychiatry, The First Teaching Hospital of Xinjiang Medical University, Urumqi, 830000 China </div><div><span class="fm-affl">Yimin Zhu, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="nc.ude.ujz@myuhz" class="oemail">nc.ude.ujz@myuhz</a></span>.</div><div><a href="#idm140669390239792aff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div><sup>#</sup>Contributed equally.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140669390239792_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2017 Mar 6; Revised 2017 Dec 10; Accepted 2018 Jan 23.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140669390239792_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © European Society of Human Genetics 2018</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5974016/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplementary file</div><div><a href="/pmc/articles/PMC5974016/bin/41431_2018_108_MOESM1_ESM.docx" data-ga-action="click_feat_suppl">41431_2018_108_MOESM1_ESM.docx</a><span style="color:gray"> (449K)</span></div><div class="small guid">GUID:&nbsp;5DBF2952-D24E-4676-9ADE-77B619F0B90F</div></div></dd></dl></div><div id="Abs1" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Abs1title">Abstract</h2><!--article-meta--><div><p id="Par1" class="p p-first-last">Although numbers of genome-wide association studies (GWAS) have been performed for serum lipid levels, limited heritability has been explained. Studies showed that combining data from GWAS and expression quantitative trait loci (eQTLs) signals can both enhance the discovery of trait-associated SNPs and gain a better understanding of the mechanism. We performed an annotation-based, multistage genome-wide screening for serum-lipid-level-associated loci in totally 6863 Han Chinese. A serum high-density lipoprotein cholesterol (HDL-C) associated variant rs1880118 (hg19 chr7:g. 6435220G&gt;C) was replicated (<em>P</em><sub>combined</sub> = 1.4E-10). rs1880118 was associated with <em>DAGLB</em> (diacylglycerol lipase, beta) expression levels in subcutaneous adipose tissue (<em>P</em> = 5.9E-42) and explained 47.7% of the expression variance. After the replication, an active segment covering variants tagged by rs1880118 near 5′ of <em>DAGLB</em> was annotated using histone modification and transcription factor binding signals. The luciferase report assay revealed that the segment containing the minor alleles showed increased transcriptional activity compared with segment contains the major alleles, which was consistent with the eQTL analyses. The expression-trait association tests indicated the association between the <em>DAGLB</em> and serum HDL-C levels using gene-based approaches called “TWAS” (<em>P</em> <em>=</em> 3.0E-8), “SMR” (<em>P</em> = 1.1E-4), and “Sherlock” (<em>P</em> = 1.6E-6). To summarize, we identified a novel HDL-C-associated variant which explained nearly half of the expression variance of <em>DAGLB</em>. Integrated analyses established a genotype-gene-phenotype three-way association and expanded our knowledge of <em>DAGLB</em> in lipid metabolism.</p></div></div><div id="Sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Sec1title">Introduction</h2><p id="Par2" class="p p-first">Dyslipidemia, involving abnormal levels of serum triglyceride (TG), total cholesterol (TC), high-density lipoprotein cholesterol (HDL-C), and low-density lipoprotein cholesterol (LDL-C), is a well-known risk factor for cardiovascular disease (CVDs) [<a href="#CR1" rid="CR1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849105">1</a>]. Genetic factors contribute a considerable part (54–66%) to lipid metabolism [<a href="#CR2" rid="CR2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849119">2</a>]. A number of loci have been mapped by genome-wide association studies (GWAS). Although vastly expanded sample sizes have been used in combined analyses in European studies, only a limited proportion of heritability has been explained by these loci [<a href="#CR3" rid="CR3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849129">3</a>, <a href="#CR4" rid="CR4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849081">4</a>]. Further explorations are needed, especially in East Asians. Similar to other GWAS, only a few of the reported lipid associated tag single nucleotide polymorphisms (SNPs) have been fine-mapped for causal SNP localization [<a href="#CR5" rid="CR5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849100">5</a>, <a href="#CR6" rid="CR6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849111">6</a>]. Meanwhile, a limited number of genes mediating the associations between the tagSNPs and lipid phenotypes have been uncovered. The translation and application of the reported variants discovered by classical GWAS strategy are the major challenges currently [<a href="#CR7" rid="CR7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849088">7</a>].</p><p id="Par3">According to the results of genotype-phenotype association studies for complex diseases, ~80–90% of these GWAS tag SNPs lie outside of the protein-coding regions, which means that these loci likely manifest their effects via regulating the gene expression or splicing [<a href="#CR6" rid="CR6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849106">6</a>, <a href="#CR8" rid="CR8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849115">8</a>–<a href="#CR10" rid="CR10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849123">10</a>]. A study performed by Nicolae and his colleagues found that expression quantitative trait loci (eQTLs) are significantly enriched in trait-associated SNPs reported by GWAS for complex traits [<a href="#CR11" rid="CR11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849082">11</a>]. Annotation via eQTL can improve the ability to discover true associations and identify the major steps of the regulatory cascade behind the statistical genotype-phenotype associations. A number of projects and databases focused on genotype-expression associations (e.g., MuTHER, SCAN, etc.) launched the eQTL annotation and SNP function prediction after GWAS [<a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849122">12</a>–<a href="#CR17" rid="CR17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849104">17</a>]. Recently, the Genotype-Tissue Expression (GTEx) project showed a landscape of eQTL annotation in multiple tissues and accelerated the driver gene prediction in the post-GWAS era [<a href="#CR18" rid="CR18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849109">18</a>, <a href="#CR19" rid="CR19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849108">19</a>].</p><p id="Par4">Integrated analyses for GWAS and eQTL in specific tissues have been promising for type 2 diabetes, bipolar disorder, immune-related diseases, etc. [<a href="#CR20" rid="CR20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849091">20</a>–<a href="#CR24" rid="CR24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849092">24</a>]. We wonder whether such an annotation (eQTL signal in relevant tissues) could be considered as a prior for screening lipid-associated loci in Han Chinese with limited sample size in the discovery stage.</p><p id="Par5">Besides eQTL annotation, chromatin activity regions can be annotated through chromatin immunoprecipitation sequencing (ChIP-seq) or DNase-seq for transcription factors, DNase sensitivity, and histone modification markers such as histone H3 lysine 4 trimethylation (H3K4me3), histone H3 lysine 9 acetylation (H3K9ac), and histone H3 lysine 27 acetylation (H3K27ac) [<a href="#CR6" rid="CR6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849094">6</a>, <a href="#CR25" rid="CR25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849085">25</a>–<a href="#CR28" rid="CR28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849128">28</a>]. The NIH Roadmap Epigenomics Consortium has published the largest collection of human epigenomes which facilitated the identification of gene regulation in multiple tissues [<a href="#CR29" rid="CR29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849117">29</a>].</p><p id="Par6" class="p p-last">In the current study, we performed a multistage GWAS for serum lipid levels. First, SNPs associated with gene expression levels in liver and adipose tissue were annotated as candidates for replications. Second, epigenetic modification signals were used for annotating putative regulatory regions. Then, luciferase reporter assay was conducted to verify the segment involving the causal variants for the regulation of the transcription activity. Finally, gene-based expression-trait association analyses were conducted.</p></div><div id="Sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Sec2title">Methods</h2><div id="Sec3" class="sec sec-first"><h3 id="Sec3title">Study subjects</h3><p id="Par7" class="p p-first-last">In the discovery stage, 998 participants with metabolic syndrome (according to the criteria of the Metabolic Syndrome Study Cooperation Group of the Chinese Diabetes Society) and 996 healthy controls were recruited from a community-based survey in Hangzhou, Zhejiang Province, China, as described in our previous study [<a href="#CR30" rid="CR30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849118">30</a>]. For the replication stage I, six independent community-based cohorts with a total of 3027 subjects were recruited from east China (Dongyang, Liuheng, Zhujiajian, Tongxiang, and Hangzhou) and Northeast China (Shenyang). For replication stage II, 2130 subjects from Shanghai were included [<a href="#CR31" rid="CR31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849103">31</a>] (S Table&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">1</a>). All participants were Han Chinese. Individuals were excluded if they had cancer or serious chronic liver, lung, or kidney disorders. The study protocol was approved by the Research Ethics Committee of the School of Medicine, Zhejiang University. All participants gave informed consent.</p></div><div id="Sec4" class="sec"><h3 id="Sec4title">Genotyping, imputation, and quality controls</h3><p id="Par8" class="p p-first-last">Genomic DNA was isolated from whole blood using a TACO automatic nucleic acid extraction apparatus. The Illumina Omni-Express 760k chip was used for the genome-wide assay of samples in the discovery stage. SNPs with call rates ≥95% and Hardy–Weinberg equilibrium <em>P</em>-values ≥1.0E-6 were included in the discovery stage. Samples were excluded from analyses if they (1) had genotyping call rates ≤95%, (2) were population outliers identified by running EIGENSOFT [<a href="#CR32" rid="CR32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849090">32</a>], or (3) had probable relatives (PI_hat&gt;0.25). Then we imputed ungenotyped SNPs via IMPUTE2 [<a href="#CR33" rid="CR33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849080">33</a>]. In the replication stage I, genotyping was done using SNPscan<sup>TM</sup> (Genesky Biotechnologies Inc., Shanghai, China). In the replication stage II, genotyping data were extracted from our previous study using the Affymetrix Genome-Wide Human SNP Array 6.0, the quality control filtering of the genotyping data can be found elsewhere [<a href="#CR31" rid="CR31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849116">31</a>]. Genotyping call-rate control (≥95%) and Hardy–Weinberg equilibrium control (<em>P</em> &gt; 1.0E-3) were implemented in the replication stages. After the quality control, 1742, 3027, and 2130 subjects were included in analyses in the discovery, the replication stage I and II, respectively.</p></div><div id="Sec5" class="sec"><h3 id="Sec5title">Online data acquisition</h3><p id="Par9" class="p p-first">Tissue-specific eQTL signals were acquired from the Genotype-Tissue Expression (GTEx) project [<a href="#CR18" rid="CR18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849112">18</a>], the Multiple Tissue Human Expression Resource (MuTHER) project, <a id="__tag_707849133" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE26106" ref="/geo/query/acc.cgi?acc=GSE26106"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE26106","term_id":"26106"}}</span>GSE26106</a> (206 and 60 subjects recruited by the University of Chicago and University of Washington) [<a href="#CR14" rid="CR14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849096">14</a>], <a id="__tag_707849131" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE9588" ref="/geo/query/acc.cgi?acc=GSE9588"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE9588","term_id":"9588"}}</span>GSE9588</a> (266 subjects recruited by Merck) [<a href="#CR14" rid="CR14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849126">14</a>], and a published meta-analysis in whole-blood samples from 5311 subjects [<a href="#CR34" rid="CR34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849093">34</a>]. Subcutaneous adipose-tissue-specific (GTEx and MuTHER) and liver-tissue-specific (<a id="__tag_707849132" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE26106" ref="/geo/query/acc.cgi?acc=GSE26106"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE26106","term_id":"26106"}}</span>GSE26106</a> and <a id="__tag_707849130" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE9588" ref="/geo/query/acc.cgi?acc=GSE9588"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE9588","term_id":"9588"}}</span>GSE9588</a>) eQTL signals were used for candidate SNP screening. eQTL signals in whole blood, skin tissue, and transformed fibroblasts were used for additional annotations.</p><p id="Par10">Summarized genome-wide lipid-associated signals were downloaded from the Global Lipids Genetics Consortium (GLGC) [<a href="#CR3" rid="CR3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849095">3</a>]. For combined analyses, 94,595 European ancestry individuals from 23 studies genotyped with SNP arrays were included.</p><p id="Par11" class="p p-last">A cluster of chromatin status was annotated using the data from ENCODE project via the UCSC genome browser. The H3K27ac modification signals were combined from seven cell lines. The DNase I hypersensitivity loci were integrated from 125 cell types. A cluster of 161 transcription factor ChIP-seq data were annotated. Another set of histone modification marks including H3K4me3, H3K9ac, and H3K27ac were used for mapping active regions via ChIP-seq. The ChIP-seq data in primary mononuclear cells (E062) and adipose nuclei (E063) and was acquired from the Roadmap project [<a href="#CR29" rid="CR29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849098">29</a>].</p></div><div id="Sec6" class="sec"><h3 id="Sec6title">Luciferase reporter assay</h3><p id="Par12" class="p p-first">To evaluate the transcriptional activity of the variant with a potential function, a segment covering the peaks of the transcription activity signals and the variants was amplified by polymerase chain reaction (PCR) from a heterozygote of rs1880118 genomic DNA sample. The recovered fragments were cloned into pGL-3 basic vectors.</p><p id="Par13" class="p p-last">Then transient transfections were conducted in the HEK 293T cell line. The cells (1.0E6) were plated into six-well plates and transfected using Lipofectamine 2000. After 48 h of transfection, the fluorescence intensities were evaluated following the protocol of the kit (Promega Dual-Luciferase Reporter Assay System). The pRL-TK vector encoding for the Renilla luciferase was transcribed, which was used as internal controls to normalize firefly luciferase expression. Transfections were done in triplicate and replicated in independent experiments.</p></div><div id="Sec7" class="sec"><h3 id="Sec7title">Strategies and statistical analyses</h3><p id="Par14" class="p p-first">First, as shown in Supplementary Fig.&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">1</a>, linear regressions were performed for the associations between lipid components (including TG, TC, HDL-C, and LDL-C) via PLINK [<a href="#CR35" rid="CR35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849114">35</a>]. Second, SNPs associated with at least two lipid components in the discovery stage (<em>P</em> &lt; 0.05 for the association tests) were annotated using eQTL signals either in subcutaneous adipose tissue or in liver tissue. Identical subcutaneous adipose-tissue-specific eQTL signals (cis SNP-gene pairs) from the GTEx (V4) and MuTHER datasets were identified. The liver-tissue-specific SNP-gene pairs (<em>P</em> &lt; 0.05) were combined from three studies. Third, conditional analyses were performed with any two candidate SNPs within ~1 Mb for linkage disequilibrium (LD) pruning. We kept only one of the statistically significant SNPs if the other SNP had a <em>P</em>-value &gt; 0.05 in conditional analyses. Fourth, trait-associated loci with eQTL annotations were considered for replications. Replications I and II with seven independent Han Chinese cohorts were performed. Finally, we combined the effects of replicable SNPs in Han Chinese samples and 94595 individuals of European ancestry reported by GLGC.</p><p id="Par15" class="p p-last">For comparison of the transcriptional activities of the vectors, <em>P</em>-value was determined by <em>t</em>-test for independent samples.</p></div><div id="Sec8" class="sec sec-last"><h3 id="Sec8title">Gene-based analyses</h3><p id="Par16" class="p p-first">After replication for the trait-associated tagSNP, the mediator gene through which these tagSNPs exert their effects on the traits were estimated using gene-based analyses including “TWAS” (transcriptome-wide association study) [<a href="#CR36" rid="CR36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849127">36</a>], “SMR”(summary data–based Mendelian randomization) [<a href="#CR37" rid="CR37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849087">37</a>], and “Sherlock” [<a href="#CR38" rid="CR38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849102">38</a>].</p><p id="Par17">The basic idea of “TWAS” and “SMR” is based on Mendelian Randomization. SNPs are used as instrumental variables for the test between the gene expression and the trait. The method “TWAS” builds a regression model with a cluster of gene expression of associated SNPs through a sample set with genomic and transcriptomic data as a reference. Then the expression levels of the gene were predicted using this model in a much larger cohort with genotype and phenotype data. The expression-trait association was evaluated using the imputed expression data. Similarly, the “SMR” integrates summary-level data from GWAS with data from eQTL studies to identify genes whose expression levels are associated with a complex trait.</p><p id="Par18">The method “Sherlock” identifies potential gene-disease association by matching the genetic signature (GWAS and eQTLs signals) using a Bayesian statistical framework. If a gene’s genetic signature, provided by eQTL data, matches the gene’s signature of GWAS, it suggested that the expression level of the gene was associated with the phenotype of GWAS.</p><p id="Par19" class="p p-last">The data used for these analyses were described in Supplementary Methods.</p></div></div><div id="Sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Sec9title">Results</h2><div id="Sec10" class="sec sec-first"><h3 id="Sec10title">eQTL-based genome wide screening for lipid-associated loci</h3><p id="Par20" class="p p-first">In the discovery stage, SNPs associated with ≥2 lipid components (including TG, TC, HDL-C, and LDL-C) were annotated with eQTL signals either in subcutaneous adipose tissue or in liver tissue (S Fig.&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">1</a>). After LD pruning, fourteen SNPs with independent effects were selected for replications (Table <a href="/pmc/articles/PMC5974016/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">(Table1,&nbsp;S</span></span><span>1</span></a>,&nbsp;S Table&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">2</a>). In these loci, a reported locus rs1077834 (at <em>LIPC</em>) was removed from the candidate list. Finally, nine SNPs were successfully genotyped in the replication stage I. After the replication stage I, a significant association was found between rs1880118 and HDL-C levels (<em>P</em> = 1.6E-3). The minor allele of rs1880118 (C allele) was associated with increased levels of HDL-C (beta = 0.032, se = 0.010). The effect was consistent with that at the discovery stage (effect allele = C, beta = 0.066, se = 0.019, <em>P</em> = 4.0E-4). Consistent result was also observed in the replication stage II (effect allele = C, beta = 0.038, se = 0.012, <em>P</em> = 1.6E-3). In addition, we checked the effect of rs1880118 in 94,595 individuals of European ancestry and found a significant association between rs1880118 and HDL-C (effect allele = C, beta = 0.022, se = 0.006, <em>P</em> = 1.6E-4). Combining the results of these association studies, the C allele of rs1880118 was associated with increased levels of HDL-C (Han Chinese combined: beta = 0.039, se = 0.007, <em>P</em> = 4.4E-8; all ancestries combined: beta = 0.029, se = 0.005, <em>P</em> = 1.4E-10). The associations between rs1880118 and TC were also shown in Table&nbsp;<a href="/pmc/articles/PMC5974016/table/Tab2/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab2" rid-ob="ob-Tab2" co-legend-rid=""><span>2</span></a> and S Fig.&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">2</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="Tab1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p1">Serum lipid-associated loci with eQTL annotations</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140669397858112" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Chr</th><th rowspan="1" colspan="1">rsid</th><th rowspan="1" colspan="1">Major allele and minor allele (minor allele frequency)</th><th rowspan="1" colspan="1">Position (hg19)</th><th rowspan="1" colspan="1">Associated traits</th><th rowspan="1" colspan="1">eGene</th><th colspan="2" rowspan="1">eQTL signal (Subcutaneous adipose)</th><th colspan="3" rowspan="1">eQTL signal (Liver)</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"><em>P</em>_MuTHER</th><th rowspan="1" colspan="1"><em>P</em>_GTEx (V4)</th><th rowspan="1" colspan="1"><em>P</em>_U_of _C</th><th rowspan="1" colspan="1"><em>P</em>_U_of _W</th><th rowspan="1" colspan="1"><em>P</em>_Merck</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs12622106</td><td rowspan="1" colspan="1">C&gt;T (0.44)</td><td rowspan="1" colspan="1">20883561</td><td rowspan="1" colspan="1">HDL-C, LDL-C</td><td rowspan="1" colspan="1">
<em>C2orf43</em>
</td><td rowspan="1" colspan="1">5.10E-36</td><td rowspan="1" colspan="1">9.90E-10</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs2834578</td><td rowspan="1" colspan="1">T&gt;C (0.36)</td><td rowspan="1" colspan="1">36059609</td><td rowspan="1" colspan="1">TC, LDL-C</td><td rowspan="1" colspan="1">
<em>CLIC6</em>
</td><td rowspan="1" colspan="1">2.50E-47</td><td rowspan="1" colspan="1">8.50E-07</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs2076673</td><td rowspan="1" colspan="1">G&gt;C (0.41)</td><td rowspan="1" colspan="1">36124860</td><td rowspan="1" colspan="1">TC, LDL-C</td><td rowspan="1" colspan="1">
<em>APOL5</em>
</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">2.60E-06</td><td rowspan="1" colspan="1">2.40E-03</td><td rowspan="1" colspan="1">4.10E-03</td></tr><tr><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">rs1680074</td><td rowspan="1" colspan="1">G&gt;A (0.48)</td><td rowspan="1" colspan="1">1279621</td><td rowspan="1" colspan="1">TG, HDL-C</td><td rowspan="1" colspan="1">
<em>CTBP1</em>
</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">8.60E-06</td><td rowspan="1" colspan="1">3.00E-02</td><td rowspan="1" colspan="1">3.50E-02</td></tr><tr><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">rs17772583</td><td rowspan="1" colspan="1">A&gt;G (0.43)</td><td rowspan="1" colspan="1">131953510</td><td rowspan="1" colspan="1">TG, TC</td><td rowspan="1" colspan="1">
<em>RAD50</em>
</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">&lt;1.0E-16</td><td rowspan="1" colspan="1">6.50E-06</td><td rowspan="1" colspan="1">4.60E-03</td></tr><tr><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">rs1880118</td><td rowspan="1" colspan="1">G&gt;C (0.19)</td><td rowspan="1" colspan="1">6435220</td><td rowspan="1" colspan="1">TC, HDL-C</td><td rowspan="1" colspan="1">
<em>DAGLB</em>
</td><td rowspan="1" colspan="1">1.00E-04</td><td rowspan="1" colspan="1">2.20E-12</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs532473</td><td rowspan="1" colspan="1">G&gt;A (0.34)</td><td rowspan="1" colspan="1">27669655</td><td rowspan="1" colspan="1">TC, LDL-C</td><td rowspan="1" colspan="1">
<em>PTCHD3</em>
</td><td rowspan="1" colspan="1">8.40E-41</td><td rowspan="1" colspan="1">2.50E-09</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs224045</td><td rowspan="1" colspan="1">T&gt;C (0.42)</td><td rowspan="1" colspan="1">64525547</td><td rowspan="1" colspan="1">TG, TC</td><td rowspan="1" colspan="1">
<em>ADO</em>
</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">3.30E-03</td><td rowspan="1" colspan="1">1.70E-03</td><td rowspan="1" colspan="1">3.10E-06</td></tr><tr><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">rs7296841</td><td rowspan="1" colspan="1">A&gt;C (0.23)</td><td rowspan="1" colspan="1">112444109</td><td rowspan="1" colspan="1">TG, TC</td><td rowspan="1" colspan="1">
<em>TMEM116</em>
</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">8.90E-16</td><td rowspan="1" colspan="1">1.30E-05</td><td rowspan="1" colspan="1">3.70E-05</td></tr><tr><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">rs11831480</td><td rowspan="1" colspan="1">G&gt;T (0.12)</td><td rowspan="1" colspan="1">132554702</td><td rowspan="1" colspan="1">TG, LDL-C</td><td rowspan="1" colspan="1">
<em>DDX51</em>
</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">&lt;1.0E-16</td><td rowspan="1" colspan="1">2.60E-02</td><td rowspan="1" colspan="1">5.50E-03</td></tr><tr><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">rs2296347</td><td rowspan="1" colspan="1">T&gt;G (0.43)</td><td rowspan="1" colspan="1">52728423</td><td rowspan="1" colspan="1">TG, TC</td><td rowspan="1" colspan="1">
<em>NEK3</em>
</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">6.20E-10</td><td rowspan="1" colspan="1">3.30E-02</td><td rowspan="1" colspan="1">1.40E-05</td></tr><tr><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">rs1077834</td><td rowspan="1" colspan="1">T&gt;C (0.40)</td><td rowspan="1" colspan="1">58723479</td><td rowspan="1" colspan="1">TG, TC, HDL-C</td><td rowspan="1" colspan="1">
<em>LIPC</em>
</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">7.60E-08</td><td rowspan="1" colspan="1">7.10E-03</td><td rowspan="1" colspan="1">2.50E-04</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs2286590</td><td rowspan="1" colspan="1">G&gt;A (0.21)</td><td rowspan="1" colspan="1">74683666</td><td rowspan="1" colspan="1">TC, HDL-C</td><td rowspan="1" colspan="1">
<em>MXRA7</em>
</td><td rowspan="1" colspan="1">1.60E-14</td><td rowspan="1" colspan="1">2.30E-11</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs3786505</td><td rowspan="1" colspan="1">G&gt;A (0.49)</td><td rowspan="1" colspan="1">45490570</td><td rowspan="1" colspan="1">TC, LDL-C</td><td rowspan="1" colspan="1">
<em>APOC4</em>
</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">&lt;1.0E-16</td><td rowspan="1" colspan="1">9.10E-04</td><td rowspan="1" colspan="1">1.50E-03</td></tr></tbody></table></div><div id="largeobj_idm140669397858112" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5974016/table/Tab1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p2">In the discovery stage, genotype–phenotype associations were performed for SNPs with eQTL annotations either in subcutaneous adipose tissue or in liver tissues. Fourteen eQTLs associated with at least two lipid components are shown in Table&nbsp;<a href="/pmc/articles/PMC5974016/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span>1</span></a></p><p id="__p3">The minor allele is considered as the effect allele in analysis</p><p id="__p4">U of C an eQTL study from the University of Chicago, U of W an eQTL study from the University of Washington, Merck an eQTL study from Merck, Chr chromosome, hg19: Human Genome version 19, also known as GRCh37, eGene the gene that has a significant eQTL acting upon it, <em>TC</em> total cholesterol, <em>TG</em> triglyceride, <em>HDL-C</em> high-density lipoprotein cholesterol, <em>LDL-C</em> low-density lipoprotein cholesterol</p></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="Tab2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">Associations between genotypes of rs1880118 and levels of serum HDL-C and TC in multistage and combined analyses</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140669398521024" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Data source</th><th rowspan="1" colspan="1">
<em>N</em>
</th><th rowspan="1" colspan="1">Race</th><th rowspan="1" colspan="1">BETA</th><th rowspan="1" colspan="1">SE</th><th rowspan="1" colspan="1">
<em>P</em>
</th></tr></thead><tbody><tr><td colspan="6" rowspan="1">A (trait=HDL-C)</td></tr><tr><td rowspan="1" colspan="1">Discovery</td><td rowspan="1" colspan="1">1666</td><td rowspan="1" colspan="1">Han Chinese</td><td rowspan="1" colspan="1">0.066</td><td rowspan="1" colspan="1">0.019</td><td rowspan="1" colspan="1">4.0E-04</td></tr><tr><td rowspan="1" colspan="1">Replication I</td><td rowspan="1" colspan="1">3027</td><td rowspan="1" colspan="1">Han Chinese</td><td rowspan="1" colspan="1">0.032</td><td rowspan="1" colspan="1">0.010</td><td rowspan="1" colspan="1">1.6E-03</td></tr><tr><td rowspan="1" colspan="1">Replication II</td><td rowspan="1" colspan="1">2094</td><td rowspan="1" colspan="1">Han Chinese</td><td rowspan="1" colspan="1">0.038</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">1.6E-03</td></tr><tr><td rowspan="1" colspan="1">Combined (Han Chinese)</td><td rowspan="1" colspan="1">6787</td><td rowspan="1" colspan="1">Han Chinese</td><td rowspan="1" colspan="1">0.039</td><td rowspan="1" colspan="1">0.007</td><td rowspan="1" colspan="1">4.4E-08</td></tr><tr><td rowspan="1" colspan="1">GLGC</td><td rowspan="1" colspan="1">91033</td><td rowspan="1" colspan="1">European ancestry</td><td rowspan="1" colspan="1">0.022</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">1.6E-04</td></tr><tr><td rowspan="1" colspan="1">Combined (All ancestries)</td><td rowspan="1" colspan="1">97820</td><td rowspan="1" colspan="1">All ancestries</td><td rowspan="1" colspan="1">0.029</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">1.4E-10</td></tr><tr><td colspan="6" rowspan="1">B (trait=TC)</td></tr><tr><td rowspan="1" colspan="1">Discovery</td><td rowspan="1" colspan="1">1665</td><td rowspan="1" colspan="1">Han Chinese</td><td rowspan="1" colspan="1">0.087</td><td rowspan="1" colspan="1">0.040</td><td rowspan="1" colspan="1">3.2E-02</td></tr><tr><td rowspan="1" colspan="1">Replication I</td><td rowspan="1" colspan="1">3025</td><td rowspan="1" colspan="1">Han Chinese</td><td rowspan="1" colspan="1">0.017</td><td rowspan="1" colspan="1">0.029</td><td rowspan="1" colspan="1">5.5E-01</td></tr><tr><td rowspan="1" colspan="1">Replication II</td><td rowspan="1" colspan="1">2094</td><td rowspan="1" colspan="1">Han Chinese</td><td rowspan="1" colspan="1">0.061</td><td rowspan="1" colspan="1">0.035</td><td rowspan="1" colspan="1">8.0E-02</td></tr><tr><td rowspan="1" colspan="1">Combined (Han Chinese)</td><td rowspan="1" colspan="1">6784</td><td rowspan="1" colspan="1">Han Chinese</td><td rowspan="1" colspan="1">0.047</td><td rowspan="1" colspan="1">0.019</td><td rowspan="1" colspan="1">1.6E-02</td></tr><tr><td rowspan="1" colspan="1">GLGC</td><td rowspan="1" colspan="1">91171</td><td rowspan="1" colspan="1">European ancestry</td><td rowspan="1" colspan="1">0.019</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">1.9E-03</td></tr><tr><td rowspan="1" colspan="1">Combined (All ancestries)</td><td rowspan="1" colspan="1">97955</td><td rowspan="1" colspan="1">All ancestries</td><td rowspan="1" colspan="1">0.022</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">2.1E-04</td></tr></tbody></table></div><div id="largeobj_idm140669398521024" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5974016/table/Tab2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p6">The minor allele (C allele) was considered as the effective allele in analyses. Beta and SE were calculated using a linear regression model adjusted for age and gender. The effects of rs1880118 were combined using fixed-effect model. No heterogeneities were observed</p><p id="__p7"><em>TC</em> total cholesterol, <em>HDL-C</em> high-density lipoprotein cholesterol, <em>N</em> number of subjects, <em>GLGC</em> Global Lipids Genetics Consortium</p></div></div><p id="Par21" class="p p-last">As Fig.&nbsp;<a href="/pmc/articles/PMC5974016/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1a</span></a> shows, the LD block of rs1880118 contains two genes, <em>RAC1</em> and <em>DAGLB</em>. A variant rs702485, which is located at 3′UTR of <em>DAGLB</em>, has been reported to be associated with serum HDL-C levels in population of European ancestry, so we evaluated the LD between the reported locus and the novel locus of rs1880118. We found that the <em>r</em><sup><em>2</em></sup> of LD was only 0.03 in Han Chinese. The conditional analysis was performed by adjusting for rs702485, an independent effect of rs1880118 was observed (S Fig.&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">3</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="Fig1" co-legend-rid="lgnd_Fig1"><a href="/pmc/articles/PMC5974016/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140669401416640" class="figure"><a href="/pmc/articles/PMC5974016/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5974016_41431_2018_108_Fig1_HTML.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 41431_2018_108_Fig1_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5974016/bin/41431_2018_108_Fig1_HTML.jpg"></a></div><div id="largeobj_idm140669401416640" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5974016/figure/Fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_Fig1"><div><a class="figpopup" href="/pmc/articles/PMC5974016/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p8">Regional plot of the replicated SNP and annotations including histone modification, transcription factor binding site, and DNase signals in this region. <strong>a</strong> Regional plot of the tagSNP rs1880118 plotted using Locuszoom. The tagSNP and another three SNPs located in a potential regulatory region are labeled. <strong>b</strong> Annotations including DNase and transcription factor binding site ChIP-seq (combined with multiple cell lines) from ENCODE project. <strong>c</strong> Histone modification signals using primary mononuclear cells from peripheral blood (E062, Roadmap project). <strong>d</strong> Histone modification signals using adipose nuclei (E063, Roadmap project)</p></div></div></div></div><div id="Sec11" class="sec"><h3 id="Sec11title">Annotations for the replicated tagSNP rs1880118</h3><p id="Par22" class="p p-first">The minor allele of rs1880118 (C allele) was associated with increased mRNA expression levels of <em>DAGLB</em> in subcutaneous adipose tissue. Consistent associations were found in eQTL analyses using data from GTEx (V4, <em>P</em> = 2.2E-12), GTEx (V6, <em>P</em> = 5.9E-42), and MuTHER project (<em>P</em> = 1.0E-4). The associations between rs1880118 and the expression level of <em>DAGLB</em> were also observed in whole blood samples from multiple eQTL data sources (S Table&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">3</a>). The 47.7% and 36.4% of the <em>DAGLB</em> expression variance could be explained by the variation of rs1880118 in subcutaneous adipose tissue and whole blood samples according to GTEx V6 data (Fig.&nbsp;<a href="/pmc/articles/PMC5974016/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>). In addition, the C allele of rs1880118 was associated with the decreased expression level of <em>RAC1</em> in Transformed fibroblast (GTEx V6, <em>P</em> = 1.9E-14, S Table&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">3</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="Fig2" co-legend-rid="lgnd_Fig2"><a href="/pmc/articles/PMC5974016/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140669402712784" class="figure"><a href="/pmc/articles/PMC5974016/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5974016_41431_2018_108_Fig2_HTML.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 41431_2018_108_Fig2_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5974016/bin/41431_2018_108_Fig2_HTML.jpg"></a></div><div id="largeobj_idm140669402712784" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5974016/figure/Fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_Fig2"><div><a class="figpopup" href="/pmc/articles/PMC5974016/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p9">Boxplots of the <em>DAGLB</em> expression levels classified by the genotypes of rs1880118 in subcutaneous adipose tissue (<strong>a</strong>) and whole blood (<strong>b</strong>) samples from GTEx project. The proportions explained (<em>r</em><sup>2</sup>, square of the coefficient of determination) by SNP were calculated using summarized data from GTEx</p></div></div></div><p id="Par23" class="p p-last">As shown in Fig.&nbsp;<a href="/pmc/articles/PMC5974016/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1a</span></a>, the LD block of rs1880118 captured a region containing genes of <em>DAGLB</em> and <em>RAC1</em>. To identify the putative regulatory region, chromatin state markers including histone modifications and transcription factor binding site were annotated. An active transcription region (hg19, chr7: 6,486,750-6,488,250) near 5′ of <em>DAGLB</em> was uncovered by H3K27ac modification, transcription factor binding site, and DNase I sensitivity signals (Fig.&nbsp;<a href="/pmc/articles/PMC5974016/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1b</span></a>). Histone modification signals using adipose nuclei and primary mononuclear cells from peripheral blood (corresponding to the eQTL analyses) were annotated using data from the Roadmap project (Fig.&nbsp;<a href="/pmc/articles/PMC5974016/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1c, d</span></a>). We considered variants in high LD (<em>r</em><sup><em>2</em></sup> <sup>&gt;</sup> 0.6 in ASN, 1000 genome project phase I) with the proxy SNP rs1880118. Of the tagged variants meeting the LD threshold, three SNPs (rs4724806, rs3828944, and rs7807755, the LD correlation among these variants and the tagSNP rs1880118 is showed in S Fig.&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">3</a>) were found to be located in this active region. The narrow peaks of the histone modification signals and their relative positions to the three variants are shown in S Table&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">4</a> and S Fig.&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">4</a>. Two loci (rs4724806 and rs3828944) are located in the first intron of <em>DAGLB</em>, and another variant rs7807755 is located in the promoter region of <em>DAGLB</em>.</p></div><div id="Sec12" class="sec"><h3 id="Sec12title">Allele-specific activities of the active fragment</h3><p id="Par24" class="p p-first">To evaluate the transcriptional activity of the three candidate loci in the putative regulatory region, 1807-bp segments covering the major alleles (wild-type) and minor alleles (mutant) of three variants (rs4724806, rs3828944, and rs7807755, labeled as “A”, “B”, and “C”, respectively) were cloned into XholI and HindIII sites of pGL-3 basic vectors. Notably, the in silico PCR amplified fragment involving a microsatellite sequence of a poly-T structure including 11 bp repeats of thymine, while the reference sequence contains 40 bp repeats of thymine. In order to make the vectors comparable, we kept the numbers of thymines (11 bp repeats of thymine) in vectors and validated the numbers of thymines using Sanger sequencing.</p><p id="Par25" class="p p-last">The activities of the wild-type and the mutant vectors were measured using luciferase reporter assay. As depicted in Fig.&nbsp;<a href="/pmc/articles/PMC5974016/figure/Fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig3" rid-ob="ob-Fig3" co-legend-rid="lgnd_Fig3"><span>3</span></a>, we observed that, compared to the wild-type (with major alleles), the mutant (with minor alleles) had significantly higher transcriptional activities (<em>P</em> &lt; 0.001). These findings show that this segment contains three complete LD variants exhibited allele difference in transcription activity and suggested that it acted as a <em>cis</em>-regulatory element of <em>DAGLB</em>. This result is consistent with the eQTL signal in subcutaneous adipose tissue (<em>P</em> = 5.4E-42).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="Fig3" co-legend-rid="lgnd_Fig3"><a href="/pmc/articles/PMC5974016/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140669403121200" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is 41431_2018_108_Fig3_HTML.jpg" title="An external file that holds a picture, illustration, etc.
Object name is 41431_2018_108_Fig3_HTML.jpg" src="/pmc/articles/PMC5974016/bin/41431_2018_108_Fig3_HTML.jpg"></div></a><div id="largeobj_idm140669403121200" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5974016/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3"></a><a target="object" href="/pmc/articles/PMC5974016/figure/Fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_Fig3"><div><a class="figpopup" href="/pmc/articles/PMC5974016/figure/Fig3/" target="figure" rid-figpopup="Fig3" rid-ob="ob-Fig3">Fig. 3</a></div><!--caption a7--><div class="caption"><p id="__p10">Segment contain the minor alleles of the tagged SNPs increased the transcriptional activity using luciferase reporter assay. The segments of the putative regulatory region containing the three candidate loci were inserted into the PGL-3 basic vector. Then the two vectors were transfected into the 293T cell line. The pRL-TK vector encoding the Renilla luciferase was transcribed, which was used as internal controls to normalize firefly luciferase expression. The error bars denote the SD of the relative activities. Comparisons were performed, and the <em>P</em>-values are labeled in the plot</p></div></div></div></div><div id="Sec13" class="sec sec-last"><h3 id="Sec13title">Expression-trait association analysis</h3><p id="Par26" class="p p-first">As shown in Fig.&nbsp;<a href="/pmc/articles/PMC5974016/figure/Fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig4" rid-ob="ob-Fig4" co-legend-rid="lgnd_Fig4"><span>4</span></a>, replicable evidence has been established between the genotype (rs1880118) to the phenotype (HDL-C) and the genotype (rs1880118) to the expression (<em>DAGLB</em>). The relationship between the expression <em>DAGLB</em> and HDL-C levels remained unknown. We performed the analyses using three novel integrated approaches including TWAS, SMR, and Sherlock for the expression-trait association tests.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="Fig4" co-legend-rid="lgnd_Fig4"><a href="/pmc/articles/PMC5974016/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140669397804704" class="figure"><a href="/pmc/articles/PMC5974016/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5974016_41431_2018_108_Fig4_HTML.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 41431_2018_108_Fig4_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5974016/bin/41431_2018_108_Fig4_HTML.jpg"></a></div><div id="largeobj_idm140669397804704" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5974016/figure/Fig4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_Fig4"><div><a class="figpopup" href="/pmc/articles/PMC5974016/figure/Fig4/" target="figure" rid-figpopup="Fig4" rid-ob="ob-Fig4">Fig. 4</a></div><!--caption a7--><div class="caption"><p id="__p11">Integrated plot of three-way association between the tagSNP rs1880118, the mRNA expression levels of <em>DAGLB</em>, and the serum HDL-C levels. Evidence for each line is summarized and marked on the plot. <em>P</em>-values were determined using TagSNP and phenotype: genotype–phenotype linear regression and combined analyses using fixed-effect model; TagSNP and gene expression: eQTL analyses and luciferase reporter assay; gene expression and phenotype: two powerful approaches called “TWAS” and “Sherlock” based on eQTL and GWAS data</p></div></div></div><p id="Par27" class="p p-last">Using TWAS, we found that <em>DAGLB</em> levels were significantly associated with serum HDL-C levels (<em>P</em> = 3.0E-8) and TC levels (<em>P</em> = 2.0E-6). Based on SMR, we also observed a significant association between <em>DAGLB</em> and HDL-C (<em>P</em><sub><em>SMR</em></sub> = 1.1E-4, <em>P</em><sub><em>HEIDI</em></sub> = 0.13) and TC (<em>P</em><sub><em>SMR</em></sub> = 2.5E-3, <em>P</em><sub><em>HEIDI</em></sub> = 0.38). The method “Sherlock” showed that the expression level of <em>DAGLB</em> was associated with serum HDL-C level (logarithm of Bayesian factor = 5.62, <em>P</em> = 1.58E-6).</p></div></div><div id="Sec14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Sec14title">Discussion</h2><p id="Par28" class="p p-first">In this study, we identified a novel locus at rs1880118 associated with serum HDL-C levels in Han Chinese. The genotype of rs1880118 explained 47.7% of the variance of the mRNA expression level of <em>DAGLB</em> in subcutaneous adipose tissue. Luciferase reporter assay indicated a causal segment contains three variants of <em>DAGLB</em> transcription regulatory.</p><p id="Par29">As enrichment analysis has shown that trait-associated loci are more likely to be eQTLs [<a href="#CR11" rid="CR11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849089">11</a>]. That means transcriptional annotations can enhance discovery of trait-associated SNPs which will raise the possibility that some of the missing heritability would be found. Compared with classical GWAS strategy, eQTL-based screening avoids excessively rigid threshold and reliance on <em>P</em>-value of genotype-phenotype association in discovery stage. Using such strategy, a significant association between the eQTL at rs1880118 and serum HDL-C levels was observed and replicated in Han Chinese. After fine-mapping using epigenetic annotations, transcriptional activity was observed in a segment which contains three candidate causal variants. Reporter assay, Electrophoretic Mobility Shift Assay (EMSA) and ChIP could be performed to verify the regulatory mechanism in the further.</p><p id="Par30">Although we replicated the genotype-phenotype association and found the causal segment, we still could not conclude whether this variant influences the HDL-C level by modulating the expression level of <em>DAGLB</em> because this variant may affect <em>DAGLB</em> and HDL-C independently. Powerful strategies were applied to identify eGene (eQTL gene, the gene that has a significant eQTL acting upon it) whose <em>cis</em>-regulated expression is associated with complex traits. TWAS and SMR have conceptual similarities with the Mendelian randomization (MR) test which aims to identify causal relations using genetic variations as instrument variables [<a href="#CR36" rid="CR36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849084">36</a>, <a href="#CR37" rid="CR37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849121">37</a>]. Such strategies could benefit from the MR which will never result in reverse causality [<a href="#CR39" rid="CR39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849124">39</a>–<a href="#CR41" rid="CR41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849125">41</a>]. Also, only a small number of transcriptome data are required as reference for the analyses [<a href="#CR41" rid="CR41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849101">41</a>]. Then we verified the finding using a computational algorithm named Sherlock [<a href="#CR38" rid="CR38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849110">38</a>]. Consistent result supporting the association between <em>DAGLB</em> and Serum HDL-C levels was observed.</p><p id="Par31" class="p p-last">Similar to DAGLα, a sequence-related enzyme DAGLβ (coded by <em>DAGLB</em>) has a role in the biosynthesis of the endocannabinoid 2-arachidnoyl glycerol (2-AG) [<a href="#CR42" rid="CR42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849083">42</a>]. <em>DAGLA</em> is mainly expressed in the nervous system and seems to be the principal regulator of 2-AG [<a href="#CR43" rid="CR43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849086">43</a>]. <em>DAGLB</em> is widely expressed in multiple tissues, including adipose tissue, white blood cells, and brain (data from GTEx) [<a href="#CR18" rid="CR18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849099">18</a>]. <em>DAGLB</em> has been linked to hypotrichosis and the influence of early life stress [<a href="#CR44" rid="CR44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849097">44</a>, <a href="#CR45" rid="CR45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849120">45</a>]. However, the evidence of <em>DAGLB</em> in lipid metabolism is limited. A recent study showed that DAGLβ inactivation lowers 2-AG, arachidonic acid, and eicosanoids in mouse peritoneal macrophages using <em>DAGLB</em> knock-out mice [<a href="#CR46" rid="CR46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849107">46</a>]. A corresponding reduction in lipopolysaccharide-induced TNF-α release was observed as well. These findings indicate that DAGLβ plays a role within the lipid network that regulates proinflammatory responses in macrophages [<a href="#CR46" rid="CR46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_707849113">46</a>]. However, it remains unknown whether the expression levels of <em>DAGLB</em> in multiple tissues is responsible for serum HDL-C and TC levels. Gene function studies using cell lines or mouse model with <em>DAGLB</em> knock in or knock down are needed for clarifying the biological role of the gene. In conclusion, we identified a novel serum HDL-C-associated variant rs1880118. The genotype of rs1880118 explained nearly half of the expression variance of <em>DAGLB</em>. After fine-mapping for the tagSNP, a regulatory segment which contributes to the variance of <em>DAGLB</em> was identified. The expression level of <em>DAGLB</em> was associated with serum HDL-C by means of integrated approaches using eQTL and GWAS data. The current study indicates the role of <em>DAGLB</em> and provides novel insight into the lipid mechanism in Han Chinese.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Electronic supplementary material</h2><!--/article/body/sec/--><div id="Sec15" class="sec sec-first-last"><p id="__p12" class="p p-first-last">
</p><div class="sec suppmat" id="MOESM1"><div class="sup-box half_rhythm" id="idm140669394072192"><a href="/pmc/articles/PMC5974016/bin/41431_2018_108_MOESM1_ESM.docx" data-ga-action="click_feat_suppl">Supplementary file</a><sup>(449K, docx)</sup></div></div><p>
</p></div></div><div id="idm140669400619056" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140669400619056title">Acknowledgements</h2><div class="sec"><p id="__p18">This work was supported by the National Key Technology R &amp; D Program of China (2009BAI80B02, 2012BAI02B03), the 973 Program (2015CB559100), the National Natural Science Foundation of China (81172755, 31325014, 81421061,&nbsp;81701321), the 111 Project (B13026), the Program for Zhejiang Leading Team of Science and Technology Innovation (2010R50050), Zhejiang Provincial Program for the Cultivation of High-level Innovative Health Talents, the Program of Shanghai Academic Research Leader (15XD1502200), the National Program for Support of Top-Notch Young Professionals. We would like to acknowledge all the staff and participants in this study for their important contribution.</p></div></div><div id="idm140669400617472" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140669400617472title">Compilance with ethical standards</h2><div id="FPar1" class="sec sec-first"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="FPar1title" class="ui-helper-clearfix">Conflict of interest</h2><p id="Par32" class="p p-first-last">The authors declare that they have no conflict of interest.</p></div></div><div id="idm140669394070464" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140669394070464title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140669394070336"></p><p id="__p14" class="p p-first-last">These authors contributed equally: Dan Zhou, Dandan Zhang, Xiaohui Sun, Zhiqiang Li.</p><p></p><p class="fn sec" id="idm140669394069712"></p><p id="__p15" class="p p-first-last">These authors jointly supervised this work: Yimin Zhu, Yongyong Shi, Maode Lai.</p><p></p><p class="fn sec" id="idm140669394069088"></p><p id="__p16" class="p p-first"><strong>Electronic supplementary material</strong></p><p id="__p17" class="p p-last">The online version of this article (10.1038/s41431-018-0108-4) contains supplementary material, which is available to authorized users.</p><p></p></div></div><div id="idm140669390239792aff-info" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140669390239792aff-infotitle">Contributor Information</h2><p><span class="fm-affl">Yimin Zhu, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="nc.ude.ujz@myuhz" class="oemail">nc.ude.ujz@myuhz</a></span>.</p><p><span class="fm-affl">Yongyong Shi, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="moc.liamg@gnoygnoyihs" class="oemail">moc.liamg@gnoygnoyihs</a></span>.</p><p><span class="fm-affl">Maode Lai, </span><span class="fm-affl"><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="nc.ude.ujz@pml" class="oemail">nc.ude.ujz@pml</a></span>.</p></div><div id="Bib1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Bib1title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CR1">1. <span class="element-citation">Castelli W. Cholesterol and lipids in the risk of coronary artery disease–the Framingham Heart Study. <span><span class="ref-journal">Can J Cardiol. </span>1988;<span class="ref-vol">4</span>:5A–10A.</span> [<a href="/pubmed/3179802" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Can+J+Cardiol.&amp;title=Cholesterol+and+lipids+in+the+risk+of+coronary+artery+disease–the+Framingham+Heart+Study&amp;author=W+Castelli&amp;volume=4&amp;publication_year=1988&amp;pages=5A-10A&amp;pmid=3282627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR2">2. <span class="element-citation">Rahman I, Bennet AM, Pedersen NL, De Faire U, Svensson P, Magnusson PK. Genetic dominance influences blood biomarker levels in a sample of 12,000 Swedish elderly twins. <span><span class="ref-journal">Twin Res Human Genet. </span>2009;<span class="ref-vol">12</span>:286–94. doi: 10.1375/twin.12.3.286.</span> [<a href="/pubmed/19456221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1375%2Ftwin.12.3.286" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res+Human+Genet.&amp;title=Genetic+dominance+influences+blood+biomarker+levels+in+a+sample+of+12,000+Swedish+elderly+twins&amp;author=I+Rahman&amp;author=AM+Bennet&amp;author=NL+Pedersen&amp;author=U+De+Faire&amp;author=P+Svensson&amp;volume=12&amp;publication_year=2009&amp;pages=286-94&amp;pmid=19456221&amp;doi=10.1375/twin.12.3.286&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR3">3. <span class="element-citation">Willer CJ, Schmidt EM, Sengupta S, et al.  Discovery and refinement of loci associated with lipid levels. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:1274–83. doi: 10.1038/ng.2797.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3838666/">PMC free article</a>]</span> [<a href="/pubmed/24097068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.2797" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet.&amp;title=Discovery+and+refinement+of+loci+associated+with+lipid+levels&amp;author=CJ+Willer&amp;author=EM+Schmidt&amp;author=S+Sengupta&amp;volume=45&amp;publication_year=2013&amp;pages=1274-83&amp;pmid=24097068&amp;doi=10.1038/ng.2797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR4">4. <span class="element-citation">Lu W, Cheng YC, Chen K, et al.  Evidence for several independent genetic variants affecting lipoprotein (a) cholesterol levels. <span><span class="ref-journal">Hum Mol Genet. </span>2015;<span class="ref-vol">24</span>:2390–2400. doi: 10.1093/hmg/ddu731.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4380064/">PMC free article</a>]</span> [<a href="/pubmed/25575512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fhmg%2Fddu731" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet.&amp;title=Evidence+for+several+independent+genetic+variants+affecting+lipoprotein+(a)+cholesterol+levels&amp;author=W+Lu&amp;author=YC+Cheng&amp;author=K+Chen&amp;volume=24&amp;publication_year=2015&amp;pages=2390-2400&amp;pmid=25575512&amp;doi=10.1093/hmg/ddu731&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR5">5. <span class="element-citation">Albert FW, Kruglyak L. The role of regulatory variation in complex traits and disease. <span><span class="ref-journal">Nat Rev Genet. </span>2015;<span class="ref-vol">16</span>:197–212. doi: 10.1038/nrg3891.</span> [<a href="/pubmed/25707927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnrg3891" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet.&amp;title=The+role+of+regulatory+variation+in+complex+traits+and+disease&amp;author=FW+Albert&amp;author=L+Kruglyak&amp;volume=16&amp;publication_year=2015&amp;pages=197-212&amp;pmid=25707927&amp;doi=10.1038/nrg3891&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR6">6. <span class="element-citation">Edwards S, Beesley J, French J, Dunning A. Beyond GWASs: illuminating the dark road from association to function. <span><span class="ref-journal">Am J Hum Genet. </span>2013;<span class="ref-vol">93</span>:779–97. doi: 10.1016/j.ajhg.2013.10.012.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3824120/">PMC free article</a>]</span> [<a href="/pubmed/24210251" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ajhg.2013.10.012" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet.&amp;title=Beyond+GWASs:+illuminating+the+dark+road+from+association+to+function&amp;author=S+Edwards&amp;author=J+Beesley&amp;author=J+French&amp;author=A+Dunning&amp;volume=93&amp;publication_year=2013&amp;pages=779-97&amp;pmid=24210251&amp;doi=10.1016/j.ajhg.2013.10.012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR7">7. <span class="element-citation">Manolio TA, Fowler DM, Starita LM, et al.  Bedside back to bench: building bridges between basic and clinical genomic research. <span><span class="ref-journal">Cell. </span>2017;<span class="ref-vol">169</span>:6–12. doi: 10.1016/j.cell.2017.03.005.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5511379/">PMC free article</a>]</span> [<a href="/pubmed/28340351" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.cell.2017.03.005" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell.&amp;title=Bedside+back+to+bench:+building+bridges+between+basic+and+clinical+genomic+research&amp;author=TA+Manolio&amp;author=DM+Fowler&amp;author=LM+Starita&amp;volume=169&amp;publication_year=2017&amp;pages=6-12&amp;pmid=28340351&amp;doi=10.1016/j.cell.2017.03.005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR8">8. <span class="element-citation">Zhang X, Bailey SD, Lupien M. Laying a solid foundation for Manhattan–setting the functional basis for the post-GWAS era. <span><span class="ref-journal">Trends Genet. </span>2014;<span class="ref-vol">30</span>:140–9. doi: 10.1016/j.tig.2014.02.006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4026049/">PMC free article</a>]</span> [<a href="/pubmed/24661571" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.tig.2014.02.006" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet.&amp;title=Laying+a+solid+foundation+for+Manhattan–setting+the+functional+basis+for+the+post-GWAS+era&amp;author=X+Zhang&amp;author=SD+Bailey&amp;author=M+Lupien&amp;volume=30&amp;publication_year=2014&amp;pages=140-9&amp;pmid=24661571&amp;doi=10.1016/j.tig.2014.02.006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR9">9. <span class="element-citation">Li YI, Van dGB, Raj A, et al.  RNA splicing is a primary link between genetic variation and disease. <span><span class="ref-journal">Science. </span>2016;<span class="ref-vol">352</span>:600–4. doi: 10.1126/science.aad9417.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5182069/">PMC free article</a>]</span> [<a href="/pubmed/27126046" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1126%2Fscience.aad9417" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science.&amp;title=RNA+splicing+is+a+primary+link+between+genetic+variation+and+disease&amp;author=YI+Li&amp;author=dGB+Van&amp;author=A+Raj&amp;volume=352&amp;publication_year=2016&amp;pages=600-4&amp;pmid=27126046&amp;doi=10.1126/science.aad9417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR10">10. <span class="element-citation">Feng Z, Lupski JR. Non-coding genetic variants in human disease. <span><span class="ref-journal">Hum Mol Genet. </span>2015;<span class="ref-vol">24</span>:R102–10. doi: 10.1093/hmg/ddv259.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4572001/">PMC free article</a>]</span> [<a href="/pubmed/26152199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fhmg%2Fddv259" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet.&amp;title=Non-coding+genetic+variants+in+human+disease&amp;author=Z+Feng&amp;author=JR+Lupski&amp;volume=24&amp;publication_year=2015&amp;pages=R102-10&amp;pmid=26152199&amp;doi=10.1093/hmg/ddv259&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR11">11. <span class="element-citation">Nicolae DL, Gamazon E, Zhang W, Duan S, Dolan ME, Cox NJ. Trait-associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000888. doi: 10.1371/journal.pgen.1000888.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2848547/">PMC free article</a>]</span> [<a href="/pubmed/20369019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pgen.1000888" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Trait-associated+SNPs+are+more+likely+to+be+eQTLs:+annotation+to+enhance+discovery+from+GWAS&amp;author=DL+Nicolae&amp;author=E+Gamazon&amp;author=W+Zhang&amp;author=S+Duan&amp;author=ME+Dolan&amp;volume=6&amp;publication_year=2010&amp;pages=e1000888&amp;pmid=20369019&amp;doi=10.1371/journal.pgen.1000888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR12">12. <span class="element-citation">Gamazon ER, Zhang W, Konkashbaev A, et al.  SCAN: SNP and copy number annotation. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:259–62. doi: 10.1093/bioinformatics/btp644.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852202/">PMC free article</a>]</span> [<a href="/pubmed/19933162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtp644" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics.&amp;title=SCAN:+SNP+and+copy+number+annotation&amp;author=ER+Gamazon&amp;author=W+Zhang&amp;author=A+Konkashbaev&amp;volume=26&amp;publication_year=2010&amp;pages=259-62&amp;pmid=19933162&amp;doi=10.1093/bioinformatics/btp644&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR13">13. <span class="element-citation">Grundberg E, Small KS, Hedman AringK, et al.  Mapping cis-and trans-regulatory effects across multiple tissues in twins. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:1084–9. doi: 10.1038/ng.2394.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3784328/">PMC free article</a>]</span> [<a href="/pubmed/22941192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.2394" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet.&amp;title=Mapping+cis-and+trans-regulatory+effects+across+multiple+tissues+in+twins&amp;author=E+Grundberg&amp;author=KS+Small&amp;author=AringK+Hedman&amp;volume=44&amp;publication_year=2012&amp;pages=1084-9&amp;pmid=22941192&amp;doi=10.1038/ng.2394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR14">14. <span class="element-citation">Innocenti F, Cooper GM, Stanaway IB, et al.  Identification, replication, and functional fine-mapping of expression quantitative trait loci in primary human liver tissue. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:165–71. doi: 10.1371/journal.pgen.1002078.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3102751/">PMC free article</a>]</span> [<a href="/pubmed/21637794" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pgen.1002078" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Identification,+replication,+and+functional+fine-mapping+of+expression+quantitative+trait+loci+in+primary+human+liver+tissue&amp;author=F+Innocenti&amp;author=GM+Cooper&amp;author=IB+Stanaway&amp;volume=7&amp;publication_year=2011&amp;pages=165-71&amp;doi=10.1371/journal.pgen.1002078&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR15">15. <span class="element-citation">Schadt EE, Molony C, Chudin E, et al.  Mapping the genetic architecture of gene expression in human liver. <span><span class="ref-journal">PLoS Biol. </span>2008;<span class="ref-vol">6</span>:1020–32. doi: 10.1371/journal.pbio.0060107.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2365981/">PMC free article</a>]</span> [<a href="/pubmed/18462017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pbio.0060107" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol.&amp;title=Mapping+the+genetic+architecture+of+gene+expression+in+human+liver&amp;author=EE+Schadt&amp;author=C+Molony&amp;author=E+Chudin&amp;volume=6&amp;publication_year=2008&amp;pages=1020-32&amp;doi=10.1371/journal.pbio.0060107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR16">16. <span class="element-citation">Liang L, Morar N, Dixon AL, et al.  A cross-platform analysis of 14,177 expression quantitative trait loci derived from lymphoblastoid cell lines. <span><span class="ref-journal">Genome Res. </span>2013;<span class="ref-vol">23</span>:716–26. doi: 10.1101/gr.142521.112.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3613588/">PMC free article</a>]</span> [<a href="/pubmed/23345460" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1101%2Fgr.142521.112" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=A+cross-platform+analysis+of+14,177+expression+quantitative+trait+loci+derived+from+lymphoblastoid+cell+lines&amp;author=L+Liang&amp;author=N+Morar&amp;author=AL+Dixon&amp;volume=23&amp;publication_year=2013&amp;pages=716-26&amp;pmid=23345460&amp;doi=10.1101/gr.142521.112&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR17">17. <span class="element-citation">Xia K, Shabalin AA, Huang S, et al.  seeQTL: a searchable database for human eQTLs. <span><span class="ref-journal">Bioinformatics. </span>2012;<span class="ref-vol">28</span>:451–2. doi: 10.1093/bioinformatics/btr678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3268245/">PMC free article</a>]</span> [<a href="/pubmed/22171328" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtr678" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics.&amp;title=seeQTL:+a+searchable+database+for+human+eQTLs&amp;author=K+Xia&amp;author=AA+Shabalin&amp;author=S+Huang&amp;volume=28&amp;publication_year=2012&amp;pages=451-2&amp;pmid=22171328&amp;doi=10.1093/bioinformatics/btr678&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR18">18. <span class="element-citation">Lonsdale J, Thomas J, Salvatore M, et al.  The Genotype-Tissue Expression (GTEx) project. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:580–5. doi: 10.1038/ng.2653.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4010069/">PMC free article</a>]</span> [<a href="/pubmed/23715323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.2653" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet.&amp;title=The+Genotype-Tissue+Expression+(GTEx)+project&amp;author=J+Lonsdale&amp;author=J+Thomas&amp;author=M+Salvatore&amp;volume=45&amp;publication_year=2013&amp;pages=580-5&amp;pmid=23715323&amp;doi=10.1038/ng.2653&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR19">19. <span class="element-citation">Consortium G. Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. <span><span class="ref-journal">Science. </span>2015;<span class="ref-vol">348</span>:648–60. doi: 10.1126/science.1262110.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4547484/">PMC free article</a>]</span> [<a href="/pubmed/25954001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1126%2Fscience.1262110" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science.&amp;title=Human+genomics.+The+Genotype-Tissue+Expression+(GTEx)+pilot+analysis:+multitissue+gene+regulation+in+humans&amp;author=G+Consortium&amp;volume=348&amp;publication_year=2015&amp;pages=648-60&amp;pmid=25954001&amp;doi=10.1126/science.1262110&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR20">20. <span class="element-citation">Gaulton KJ, Ferreira T, Lee Y, et al.  Genetic fine mapping and genomic annotation defines causal mechanisms at type 2 diabetes susceptibility loci. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:1415–25. doi: 10.1038/ng.3437.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4666734/">PMC free article</a>]</span> [<a href="/pubmed/26551672" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3437" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet.&amp;title=Genetic+fine+mapping+and+genomic+annotation+defines+causal+mechanisms+at+type+2+diabetes+susceptibility+loci&amp;author=KJ+Gaulton&amp;author=T+Ferreira&amp;author=Y+Lee&amp;volume=47&amp;publication_year=2015&amp;pages=1415-25&amp;pmid=26551672&amp;doi=10.1038/ng.3437&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR21">21. <span class="element-citation">Peters JE, Lyons PA, Lee JC, et al.  Insight into genotype-phenotype associations through eQTL mapping in multiple cell types in health and immune-mediated disease. <span><span class="ref-journal">PLoS Genet. </span>2016;<span class="ref-vol">12</span>:e1005908. doi: 10.1371/journal.pgen.1005908.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4807835/">PMC free article</a>]</span> [<a href="/pubmed/27015630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pgen.1005908" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Insight+into+genotype-phenotype+associations+through+eQTL+mapping+in+multiple+cell+types+in+health+and+immune-mediated+disease&amp;author=JE+Peters&amp;author=PA+Lyons&amp;author=JC+Lee&amp;volume=12&amp;publication_year=2016&amp;pages=e1005908&amp;pmid=27015630&amp;doi=10.1371/journal.pgen.1005908&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR22">22. <span class="element-citation">Li M, Luo XJ, Landén M, et al.  Impact of a cis-associated gene expression SNP on chromosome 20q11.22 on bipolar disorder susceptibility, hippocampal structure and cognitive performance. <span><span class="ref-journal">Br J Psychiatry. </span>2015;<span class="ref-vol">208</span>:128–37. doi: 10.1192/bjp.bp.114.156976.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4829352/">PMC free article</a>]</span> [<a href="/pubmed/26338991" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1192%2Fbjp.bp.114.156976" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Psychiatry.&amp;title=Impact+of+a+cis-associated+gene+expression+SNP+on+chromosome+20q11.22+on+bipolar+disorder+susceptibility,+hippocampal+structure+and+cognitive+performance&amp;author=M+Li&amp;author=XJ+Luo&amp;author=M+Landén&amp;volume=208&amp;publication_year=2015&amp;pages=128-37&amp;pmid=26338991&amp;doi=10.1192/bjp.bp.114.156976&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR23">23. <span class="element-citation">Giambartolomei C, Vukcevic D, Schadt EE, et al.  Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:568–71. doi: 10.1371/journal.pgen.1004383.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4022491/">PMC free article</a>]</span> [<a href="/pubmed/24830394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pgen.1004383" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Bayesian+test+for+colocalisation+between+pairs+of+genetic+association+studies+using+summary+statistics&amp;author=C+Giambartolomei&amp;author=D+Vukcevic&amp;author=EE+Schadt&amp;volume=10&amp;publication_year=2014&amp;pages=568-71&amp;doi=10.1371/journal.pgen.1004383&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR24">24. <span class="element-citation">Yao L, Tak YG, Berman BP, Farnham PJ. Functional annotation of colon cancer risk SNPs. <span><span class="ref-journal">Nat Commun. </span>2014;<span class="ref-vol">5</span>:5114–5114. doi: 10.1038/ncomms6114.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4200523/">PMC free article</a>]</span> [<a href="/pubmed/25268989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fncomms6114" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun.&amp;title=Functional+annotation+of+colon+cancer+risk+SNPs&amp;author=L+Yao&amp;author=YG+Tak&amp;author=BP+Berman&amp;author=PJ+Farnham&amp;volume=5&amp;publication_year=2014&amp;pages=5114-5114&amp;pmid=25268989&amp;doi=10.1038/ncomms6114&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR25">25. <span class="element-citation">Birney E, Stamatoyannopoulos JA, Dutta A, et al.  Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:799–816. doi: 10.1038/nature05874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2212820/">PMC free article</a>]</span> [<a href="/pubmed/17571346" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnature05874" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature.&amp;title=Identification+and+analysis+of+functional+elements+in+1%+of+the+human+genome+by+the+ENCODE+pilot+project&amp;author=E+Birney&amp;author=JA+Stamatoyannopoulos&amp;author=A+Dutta&amp;volume=447&amp;publication_year=2007&amp;pages=799-816&amp;pmid=17571346&amp;doi=10.1038/nature05874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR26">26. <span class="element-citation">Degner JF, Pai AA, Piqueregi R, et al.  DNaseI sensitivity QTLs are a major determinant of human expression variation. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">482</span>:390–4. doi: 10.1038/nature10808.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3501342/">PMC free article</a>]</span> [<a href="/pubmed/22307276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnature10808" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature.&amp;title=DNaseI+sensitivity+QTLs+are+a+major+determinant+of+human+expression+variation&amp;author=JF+Degner&amp;author=AA+Pai&amp;author=R+Piqueregi&amp;volume=482&amp;publication_year=2012&amp;pages=390-4&amp;pmid=22307276&amp;doi=10.1038/nature10808&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR27">27. <span class="element-citation">Grubert F, Zaugg J, Kasowski M, et al.  Genetic control of chromatin states in humans involves local and distal chromosomal interactions. <span><span class="ref-journal">Cell. </span>2015;<span class="ref-vol">162</span>:1051–65. doi: 10.1016/j.cell.2015.07.048.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4556133/">PMC free article</a>]</span> [<a href="/pubmed/26300125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.cell.2015.07.048" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell.&amp;title=Genetic+control+of+chromatin+states+in+humans+involves+local+and+distal+chromosomal+interactions&amp;author=F+Grubert&amp;author=J+Zaugg&amp;author=M+Kasowski&amp;volume=162&amp;publication_year=2015&amp;pages=1051-65&amp;pmid=26300125&amp;doi=10.1016/j.cell.2015.07.048&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR28">28. <span class="element-citation">Spain SL, Barrett JC. Strategies for fine-mapping complex traits. <span><span class="ref-journal">Hum Mol Genet. </span>2015;<span class="ref-vol">24</span>:R111–R119. doi: 10.1093/hmg/ddv260.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4572002/">PMC free article</a>]</span> [<a href="/pubmed/26157023" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fhmg%2Fddv260" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet.&amp;title=Strategies+for+fine-mapping+complex+traits&amp;author=SL+Spain&amp;author=JC+Barrett&amp;volume=24&amp;publication_year=2015&amp;pages=R111-R119&amp;pmid=26157023&amp;doi=10.1093/hmg/ddv260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR29">29. <span class="element-citation">Kundaje A, Meuleman W, Ernst J, et al.  Integrative analysis of 111 reference human epigenomes. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">518</span>:317–30. doi: 10.1038/nature14248.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4530010/">PMC free article</a>]</span> [<a href="/pubmed/25693563" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnature14248" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature.&amp;title=Integrative+analysis+of+111+reference+human+epigenomes&amp;author=A+Kundaje&amp;author=W+Meuleman&amp;author=J+Ernst&amp;volume=518&amp;publication_year=2015&amp;pages=317-30&amp;pmid=25693563&amp;doi=10.1038/nature14248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR30">30. <span class="element-citation">Zhu Y, Zhang D, Zhou D, et al.  Susceptibility loci for metabolic syndrome and metabolic components identified in Han Chinese: a multi‐stage genome‐wide association study. <span><span class="ref-journal">J Cell Mol Med. </span>2017;<span class="ref-vol">21</span>:1106–16. doi: 10.1111/jcmm.13042.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5431133/">PMC free article</a>]</span> [<a href="/pubmed/28371326" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1111%2Fjcmm.13042" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Mol+Med.&amp;title=Susceptibility+loci+for+metabolic+syndrome+and+metabolic+components+identified+in+Han+Chinese:+a+multi‐stage+genome‐wide+association+study&amp;author=Y+Zhu&amp;author=D+Zhang&amp;author=D+Zhou&amp;volume=21&amp;publication_year=2017&amp;pages=1106-16&amp;pmid=28371326&amp;doi=10.1111/jcmm.13042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR31">31. <span class="element-citation">Liu Y, Cao L, Li Z, et al.  A genome-wide association study identifies a locus on TERT for mean telomere length in Han Chinese. <span><span class="ref-journal">PLoS ONE. </span>2014;<span class="ref-vol">9</span>:e85043. doi: 10.1371/journal.pone.0085043.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3897378/">PMC free article</a>]</span> [<a href="/pubmed/24465473" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pone.0085043" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE.&amp;title=A+genome-wide+association+study+identifies+a+locus+on+TERT+for+mean+telomere+length+in+Han+Chinese&amp;author=Y+Liu&amp;author=L+Cao&amp;author=Z+Li&amp;volume=9&amp;publication_year=2014&amp;pages=e85043&amp;pmid=24465473&amp;doi=10.1371/journal.pone.0085043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR32">32. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–9. doi: 10.1038/ng1847.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng1847" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;doi=10.1038/ng1847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR33">33. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529. doi: 10.1371/journal.pgen.1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pgen.1000529" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;doi=10.1371/journal.pgen.1000529&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR34">34. <span class="element-citation">Westra HJ, Peters MJ, Esko T, et al.  Systematic identification of trans eQTLs as putative drivers of known disease associations. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:1238–43. doi: 10.1038/ng.2756.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3991562/">PMC free article</a>]</span> [<a href="/pubmed/24013639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.2756" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet.&amp;title=Systematic+identification+of+trans+eQTLs+as+putative+drivers+of+known+disease+associations&amp;author=HJ+Westra&amp;author=MJ+Peters&amp;author=T+Esko&amp;volume=45&amp;publication_year=2013&amp;pages=1238-43&amp;pmid=24013639&amp;doi=10.1038/ng.2756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR35">35. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Human Genet. </span>2007;<span class="ref-vol">81</span>:559–75. doi: 10.1086/519795.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1086%2F519795" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Human+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;doi=10.1086/519795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR36">36. <span class="element-citation">Gusev A, Ko A, Shi H, et al.  Integrative approaches for large-scale transcriptome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:245–52. doi: 10.1038/ng.3506.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4767558/">PMC free article</a>]</span> [<a href="/pubmed/26854917" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3506" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet.&amp;title=Integrative+approaches+for+large-scale+transcriptome-wide+association+studies&amp;author=A+Gusev&amp;author=A+Ko&amp;author=H+Shi&amp;volume=48&amp;publication_year=2016&amp;pages=245-52&amp;pmid=26854917&amp;doi=10.1038/ng.3506&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR37">37. <span class="element-citation">Zhu Z, Zhang F, Hu H, et al.  Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:481–7. doi: 10.1038/ng.3538.</span> [<a href="/pubmed/27019110" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3538" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet.&amp;title=Integration+of+summary+data+from+GWAS+and+eQTL+studies+predicts+complex+trait+gene+targets&amp;author=Z+Zhu&amp;author=F+Zhang&amp;author=H+Hu&amp;volume=48&amp;publication_year=2016&amp;pages=481-7&amp;pmid=27019110&amp;doi=10.1038/ng.3538&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR38">38. <span class="element-citation">He X, Fuller C, Song Y, et al.  Sherlock: detecting gene-disease associations by matching patterns of expression QTL and GWAS. <span><span class="ref-journal">Am J Hum Genet. </span>2013;<span class="ref-vol">92</span>:667–80. doi: 10.1016/j.ajhg.2013.03.022.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3644637/">PMC free article</a>]</span> [<a href="/pubmed/23643380" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ajhg.2013.03.022" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet.&amp;title=Sherlock:+detecting+gene-disease+associations+by+matching+patterns+of+expression+QTL+and+GWAS&amp;author=X+He&amp;author=C+Fuller&amp;author=Y+Song&amp;volume=92&amp;publication_year=2013&amp;pages=667-80&amp;pmid=23643380&amp;doi=10.1016/j.ajhg.2013.03.022&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR39">39. <span class="element-citation">Davey Smith G, Ebrahim S. ‘Mendelian randomization’: can genetic epidemiology contribute to understanding environmental determinants of disease? <span><span class="ref-journal">Int J Epidemiol. </span>2003;<span class="ref-vol">32</span>:1–22. doi: 10.1093/ije/dyg070.</span> [<a href="/pubmed/12689998" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fije%2Fdyg070" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol.&amp;title=‘Mendelian+randomization’:+can+genetic+epidemiology+contribute+to+understanding+environmental+determinants+of+disease?&amp;author=G+Davey+Smith&amp;author=S+Ebrahim&amp;volume=32&amp;publication_year=2003&amp;pages=1-22&amp;pmid=12689998&amp;doi=10.1093/ije/dyg070&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR40">40. <span class="element-citation">Smith GD, Ebrahim S. Mendelian randomization: prospects, potentials, and limitations. <span><span class="ref-journal">Int J Epidemiol. </span>2004;<span class="ref-vol">33</span>:30–42. doi: 10.1093/ije/dyh132.</span> [<a href="/pubmed/15075143" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fije%2Fdyh132" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol.&amp;title=Mendelian+randomization:+prospects,+potentials,+and+limitations&amp;author=GD+Smith&amp;author=S+Ebrahim&amp;volume=33&amp;publication_year=2004&amp;pages=30-42&amp;pmid=15075143&amp;doi=10.1093/ije/dyh132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR41">41. <span class="element-citation">Gamazon ER, Wheeler HE, Shah KP, et al.  A gene-based association method for mapping traits using reference transcriptome data. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:1091–8. doi: 10.1038/ng.3367.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4552594/">PMC free article</a>]</span> [<a href="/pubmed/26258848" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3367" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet.&amp;title=A+gene-based+association+method+for+mapping+traits+using+reference+transcriptome+data&amp;author=ER+Gamazon&amp;author=HE+Wheeler&amp;author=KP+Shah&amp;volume=47&amp;publication_year=2015&amp;pages=1091-8&amp;pmid=26258848&amp;doi=10.1038/ng.3367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR42">42. <span class="element-citation">Wattanamongkhol N, Kumhom P, Chamnongthai K. Cloning of the first sn1-DAG lipases points to the spatial and temporal regulation of endocannabinoid signaling in the brain. <span><span class="ref-journal">J Cell Biol. </span>2003;<span class="ref-vol">163</span>:463–8. doi: 10.1083/jcb.200305129.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2173631/">PMC free article</a>]</span> [<a href="/pubmed/14610053" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1083%2Fjcb.200305129" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol.&amp;title=Cloning+of+the+first+sn1-DAG+lipases+points+to+the+spatial+and+temporal+regulation+of+endocannabinoid+signaling+in+the+brain&amp;author=N+Wattanamongkhol&amp;author=P+Kumhom&amp;author=K+Chamnongthai&amp;volume=163&amp;publication_year=2003&amp;pages=463-8&amp;pmid=14610053&amp;doi=10.1083/jcb.200305129&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR43">43. <span class="element-citation">T J, J WM, K M, A S. Diacylglycerol lipaseα (DAGLα) and DAGLβ cooperatively regulate the production of 2-arachidonoyl glycerol in autaptic hippocampal neurons. <span><span class="ref-journal">Mol Pharmacol. </span>2013;<span class="ref-vol">84</span>:296–302. doi: 10.1124/mol.113.085217.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3716324/">PMC free article</a>]</span> [<a href="/pubmed/23748223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1124%2Fmol.113.085217" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Pharmacol&amp;title=Diacylglycerol+lipaseα+(DAGLα)+and+DAGLβ+cooperatively+regulate+the+production+of+2-arachidonoyl+glycerol+in+autaptic+hippocampal+neurons&amp;author=J+T&amp;author=WM+J&amp;author=M+K&amp;author=S+A&amp;volume=84&amp;publication_year=2013&amp;pages=296-302&amp;pmid=23748223&amp;doi=10.1124/mol.113.085217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR44">44. <span class="element-citation">Marco EM, Echeverryalzate V, Lópezmoreno JA, Giné E, Peñasco S, Viveros MP. Consequences of early life stress on the expression of endocannabinoid-related genes in the rat brain. <span><span class="ref-journal">Behav Pharmacol. </span>2014;<span class="ref-vol">25</span>:547–56.</span> [<a href="/pubmed/25083571" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behav+Pharmacol.&amp;title=Consequences+of+early+life+stress+on+the+expression+of+endocannabinoid-related+genes+in+the+rat+brain&amp;author=EM+Marco&amp;author=V+Echeverryalzate&amp;author=JA+Lópezmoreno&amp;author=E+Giné&amp;author=S+Peñasco&amp;volume=25&amp;publication_year=2014&amp;pages=547-56&amp;pmid=25083571&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR45">45. <span class="element-citation">Basit S, Ali G, Wasif N, Ansar M, Ahmad W. Genetic mapping of a novel hypotrichosis locus to chromosome 7p21.3–p22.3 in a Pakistani family and screening of the candidate genes. <span><span class="ref-journal">Hum Genet. </span>2010;<span class="ref-vol">128</span>:213–20. doi: 10.1007/s00439-010-0847-y.</span> [<a href="/pubmed/20544222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1007%2Fs00439-010-0847-y" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet.&amp;title=Genetic+mapping+of+a+novel+hypotrichosis+locus+to+chromosome+7p21.3–p22.3+in+a+Pakistani+family+and+screening+of+the+candidate+genes&amp;author=S+Basit&amp;author=G+Ali&amp;author=N+Wasif&amp;author=M+Ansar&amp;author=W+Ahmad&amp;volume=128&amp;publication_year=2010&amp;pages=213-20&amp;pmid=20544222&amp;doi=10.1007/s00439-010-0847-y&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR46">46. <span class="element-citation">Hsu KL, Tsuboi K, Adibekian A, Pugh H, Masuda K, Cravatt BF. DAGLβ inhibition perturbs a lipid network involved in macrophage inflammatory responses. <span><span class="ref-journal">Nat Chem Biol. </span>2012;<span class="ref-vol">8</span>:999–1007. doi: 10.1038/nchembio.1105.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3513945/">PMC free article</a>]</span> [<a href="/pubmed/23103940" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnchembio.1105" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Chem+Biol.&amp;title=DAGLβ+inhibition+perturbs+a+lipid+network+involved+in+macrophage+inflammatory+responses&amp;author=KL+Hsu&amp;author=K+Tsuboi&amp;author=A+Adibekian&amp;author=H+Pugh&amp;author=K+Masuda&amp;volume=8&amp;publication_year=2012&amp;pages=999-1007&amp;pmid=23103940&amp;doi=10.1038/nchembio.1105&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5974016&amp;issue-id=314102&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#Abs1title">Abstract</a></li><li><a href="#Sec1title">Introduction</a></li><li><a href="#Sec2title">Methods</a></li><li><a href="#Sec9title">Results</a></li><li><a href="#Sec14title">Discussion</a></li><li><a href="#__sec1title">Electronic supplementary material</a></li><li><a href="#idm140669400619056title">Acknowledgements</a></li><li><a href="#idm140669400617472title">Compilance with ethical standards</a></li><li><a href="#FPar1title">Conflict of interest</a></li><li><a href="#idm140669394070464title">Footnotes</a></li><li><a href="#idm140669390239792aff-infotitle">Contributor Information</a></li><li><a href="#Bib1title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5974016/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5974016/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5974016/pdf/41431_2018_Article_108.pdf">PDF (1.4M)</a></li> | <li><a href="#" data-citationid="PMC5974016" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5974016%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5974016%2F&amp;text=A%20novel%20variant%20associated%20with%20HDL-C%20levels%20by%20modifying%20DAGLB%20expression%20levels%3A%20An%20annotation-based%20genome-wide%20association%20study" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5974016%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29476167" data-aiid="5974016" data-aid="5974016" data-iid="314102" data-domainid="1300" data-domain="ejhg" data-accid="PMC5974016" data-md5="20dc95ade36aed210375607a9924babf"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:59:43-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A5E7CDD5B44D10000000001B30100&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5974016%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5974016/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>