<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5135610/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genetic polymorphisms in the long noncoding RNA MIR2052HG offer a pharmacogenomic basis for the response of breast cancer patients to aromatase inhibitor therapy</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5135610/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Cancer research"><meta name="citation_title" content="Genetic polymorphisms in the long noncoding RNA MIR2052HG offer a pharmacogenomic basis for the response of breast cancer patients to aromatase inhibitor therapy"><meta name="citation_authors" content="James N. Ingle, Fang Xie, Matthew J. Ellis, Paul E. Goss, Lois E. Shepherd, Judith-Anne W. Chapman, Bingshu E. Chen, Michiaki Kubo, Yoichi Furukawa, Yukihide Momozawa, Vered Stearns, Kathleen I. Pritchard, Poulami Barman, Erin E. Carlson, Matthew P. Goetz, Richard M. Weinshilboum, Krishna R. Kalari, Liewei Wang"><meta name="citation_date" content="1 December 2016"><meta name="citation_issue" content="23"><meta name="citation_volume" content="76"><meta name="citation_firstpage" content="7012"><meta name="citation_doi" content="10.1158/0008-5472.CAN-16-1371"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5135610/?report=abstract"><meta name="citation_pmid" content="27758888"><meta name="DC.Title" content="Genetic polymorphisms in the long noncoding RNA MIR2052HG offer a pharmacogenomic basis for the response of breast cancer patients to aromatase inhibitor therapy"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="James N. Ingle"><meta name="DC.Contributor" content="Fang Xie"><meta name="DC.Contributor" content="Matthew J. Ellis"><meta name="DC.Contributor" content="Paul E. Goss"><meta name="DC.Contributor" content="Lois E. Shepherd"><meta name="DC.Contributor" content="Judith-Anne W. Chapman"><meta name="DC.Contributor" content="Bingshu E. Chen"><meta name="DC.Contributor" content="Michiaki Kubo"><meta name="DC.Contributor" content="Yoichi Furukawa"><meta name="DC.Contributor" content="Yukihide Momozawa"><meta name="DC.Contributor" content="Vered Stearns"><meta name="DC.Contributor" content="Kathleen I. Pritchard"><meta name="DC.Contributor" content="Poulami Barman"><meta name="DC.Contributor" content="Erin E. Carlson"><meta name="DC.Contributor" content="Matthew P. Goetz"><meta name="DC.Contributor" content="Richard M. Weinshilboum"><meta name="DC.Contributor" content="Krishna R. Kalari"><meta name="DC.Contributor" content="Liewei Wang"><meta name="DC.Date" content="2016 Dec 1"><meta name="DC.Identifier" content="10.1158/0008-5472.CAN-16-1371"><meta name="DC.Language" content="en"><meta property="og:title" content="Genetic polymorphisms in the long noncoding RNA MIR2052HG offer a pharmacogenomic basis for the response of breast cancer patients to aromatase inhibitor therapy"><meta property="og:type" content="article"><meta property="og:description" content="Genetic risks in breast cancer remain only partly understood. Here we report the results of a genome-wide association study of germline DNA from 4,658 women, including 252 women experiencing a breast cancer recurrence, who were entered on the MA.27 adjuvant ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5135610/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5135610/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5135610/pdf/nihms827024.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8817BADD59A1A100000000039A021A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5135610%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5135610</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Cancer Res</a></span>. Author manuscript; available in PMC 2017 Dec 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=27758888" target="pmc_ext" ref="reftype=publisher&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Cancer Res. 2016 Dec 1; 76(23): 7012–7023. </span></a></div><span class="fm-vol-iss-date">Published online 2016 Oct 10. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1158%2F0008-5472.CAN-16-1371" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1158/0008-5472.CAN-16-1371</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5135610</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS827024</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27758888">27758888</a></div></div></div></div><h1 class="content-title">Genetic polymorphisms in the long noncoding RNA MIR2052HG offer a pharmacogenomic basis for the response of breast cancer patients to aromatase inhibitor therapy</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Ingle%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605744055072" co-class="co-affbox">James N. Ingle</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Xie%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605705592928" co-class="co-affbox">Fang Xie</a>,<sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Ellis%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605708334400" co-class="co-affbox">Matthew J. Ellis</a>,<sup>3,</sup><sup>*</sup> <a href="/pubmed/?term=Goss%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605708331680" co-class="co-affbox">Paul E. Goss</a>,<sup>4</sup> <a href="/pubmed/?term=Shepherd%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605708329696" co-class="co-affbox">Lois E. Shepherd</a>,<sup>5</sup> <a href="/pubmed/?term=Chapman%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605705241120" co-class="co-affbox">Judith-Anne W. Chapman</a>,<sup>5</sup> <a href="/pubmed/?term=Chen%20BE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605705239136" co-class="co-affbox">Bingshu E. Chen</a>,<sup>5</sup> <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605705237152" co-class="co-affbox">Michiaki Kubo</a>,<sup>6</sup> <a href="/pubmed/?term=Furukawa%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605705235168" co-class="co-affbox">Yoichi Furukawa</a>,<sup>7</sup> <a href="/pubmed/?term=Momozawa%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605671005760" co-class="co-affbox">Yukihide Momozawa</a>,<sup>6</sup> <a href="/pubmed/?term=Stearns%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605671003776" co-class="co-affbox">Vered Stearns</a>,<sup>8</sup> <a href="/pubmed/?term=Pritchard%20KI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605671001792" co-class="co-affbox">Kathleen I. Pritchard</a>,<sup>9</sup> <a href="/pubmed/?term=Barman%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605662394592" co-class="co-affbox">Poulami Barman</a>,<sup>10</sup> <a href="/pubmed/?term=Carlson%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605662392608" co-class="co-affbox">Erin E. Carlson</a>,<sup>10</sup> <a href="/pubmed/?term=Goetz%20MP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605662390624" co-class="co-affbox">Matthew P. Goetz</a>,<sup>1</sup> <a href="/pubmed/?term=Weinshilboum%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605662388640" co-class="co-affbox">Richard M. Weinshilboum</a>,<sup>2</sup> <a href="/pubmed/?term=Kalari%20KR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605669709536" co-class="co-affbox">Krishna R. Kalari</a>,<sup>10</sup> and  <a href="/pubmed/?term=Wang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888" class="affpopup" co-rid="_co_idm140605669707552" co-class="co-affbox">Liewei Wang</a><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140605744055072"><h3 class="no_margin">James N. Ingle</h3><p><sup>1</sup>Division of Medical Oncology, Mayo Clinic, Rochester, MN</p><div>Find articles by <a href="/pubmed/?term=Ingle%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">James N. Ingle</a></div></div><div id="_co_idm140605705592928"><h3 class="no_margin">Fang Xie</h3><p><sup>2</sup>Division of Clinical Pharmacology, Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, MN</p><div>Find articles by <a href="/pubmed/?term=Xie%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Fang Xie</a></div></div><div id="_co_idm140605708334400"><h3 class="no_margin">Matthew J. Ellis</h3><p><sup>3</sup>Baylor Breast Center, Houston, TX</p><div>Find articles by <a href="/pubmed/?term=Ellis%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Matthew J. Ellis</a></div></div><div id="_co_idm140605708331680"><h3 class="no_margin">Paul E. Goss</h3><p><sup>4</sup>Massachusetts General Hospital Cancer Center, Harvard University, Boston, MA</p><div>Find articles by <a href="/pubmed/?term=Goss%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Paul E. Goss</a></div></div><div id="_co_idm140605708329696"><h3 class="no_margin">Lois E. Shepherd</h3><p><sup>5</sup>Canadian Cancer Trials Group, Kingston, Ontario, Canada</p><div>Find articles by <a href="/pubmed/?term=Shepherd%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Lois E. Shepherd</a></div></div><div id="_co_idm140605705241120"><h3 class="no_margin">Judith-Anne W. Chapman</h3><p><sup>5</sup>Canadian Cancer Trials Group, Kingston, Ontario, Canada</p><div>Find articles by <a href="/pubmed/?term=Chapman%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Judith-Anne W. Chapman</a></div></div><div id="_co_idm140605705239136"><h3 class="no_margin">Bingshu E. Chen</h3><p><sup>5</sup>Canadian Cancer Trials Group, Kingston, Ontario, Canada</p><div>Find articles by <a href="/pubmed/?term=Chen%20BE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Bingshu E. Chen</a></div></div><div id="_co_idm140605705237152"><h3 class="no_margin">Michiaki Kubo</h3><p><sup>6</sup>RIKEN Center for Integrative Medical Sciences, Kanagawa, Japan</p><div>Find articles by <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Michiaki Kubo</a></div></div><div id="_co_idm140605705235168"><h3 class="no_margin">Yoichi Furukawa</h3><p><sup>7</sup>University of Tokyo, Tokyo, Japan</p><div>Find articles by <a href="/pubmed/?term=Furukawa%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Yoichi Furukawa</a></div></div><div id="_co_idm140605671005760"><h3 class="no_margin">Yukihide Momozawa</h3><p><sup>6</sup>RIKEN Center for Integrative Medical Sciences, Kanagawa, Japan</p><div>Find articles by <a href="/pubmed/?term=Momozawa%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Yukihide Momozawa</a></div></div><div id="_co_idm140605671003776"><h3 class="no_margin">Vered Stearns</h3><p><sup>8</sup>Johns Hopkins School of Medicine, Baltimore, MD</p><div>Find articles by <a href="/pubmed/?term=Stearns%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Vered Stearns</a></div></div><div id="_co_idm140605671001792"><h3 class="no_margin">Kathleen I. Pritchard</h3><p><sup>9</sup>Sunnybrook Odette Regional Cancer Centre, University of Toronto, Toronto, Ontario, Canada</p><div>Find articles by <a href="/pubmed/?term=Pritchard%20KI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Kathleen I. Pritchard</a></div></div><div id="_co_idm140605662394592"><h3 class="no_margin">Poulami Barman</h3><p><sup>10</sup>Division of Biomedical Statistics and Informatics, Department of Health Sciences Research, Mayo Clinic, Rochester, MN</p><div>Find articles by <a href="/pubmed/?term=Barman%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Poulami Barman</a></div></div><div id="_co_idm140605662392608"><h3 class="no_margin">Erin E. Carlson</h3><p><sup>10</sup>Division of Biomedical Statistics and Informatics, Department of Health Sciences Research, Mayo Clinic, Rochester, MN</p><div>Find articles by <a href="/pubmed/?term=Carlson%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Erin E. Carlson</a></div></div><div id="_co_idm140605662390624"><h3 class="no_margin">Matthew P. Goetz</h3><p><sup>1</sup>Division of Medical Oncology, Mayo Clinic, Rochester, MN</p><div>Find articles by <a href="/pubmed/?term=Goetz%20MP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Matthew P. Goetz</a></div></div><div id="_co_idm140605662388640"><h3 class="no_margin">Richard M. Weinshilboum</h3><p><sup>2</sup>Division of Clinical Pharmacology, Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, MN</p><div>Find articles by <a href="/pubmed/?term=Weinshilboum%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Richard M. Weinshilboum</a></div></div><div id="_co_idm140605669709536"><h3 class="no_margin">Krishna R. Kalari</h3><p><sup>10</sup>Division of Biomedical Statistics and Informatics, Department of Health Sciences Research, Mayo Clinic, Rochester, MN</p><div>Find articles by <a href="/pubmed/?term=Kalari%20KR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Krishna R. Kalari</a></div></div><div id="_co_idm140605669707552"><h3 class="no_margin">Liewei Wang</h3><p><sup>2</sup>Division of Clinical Pharmacology, Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, MN</p><div>Find articles by <a href="/pubmed/?term=Wang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27758888">Liewei Wang</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140605709826544_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140605709826544_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140605709826544_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Division of Medical Oncology, Mayo Clinic, Rochester, MN</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Division of Clinical Pharmacology, Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, MN</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Baylor Breast Center, Houston, TX</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Massachusetts General Hospital Cancer Center, Harvard University, Boston, MA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Canadian Cancer Trials Group, Kingston, Ontario, Canada</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>RIKEN Center for Integrative Medical Sciences, Kanagawa, Japan</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>University of Tokyo, Tokyo, Japan</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Johns Hopkins School of Medicine, Baltimore, MD</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Sunnybrook Odette Regional Cancer Centre, University of Toronto, Toronto, Ontario, Canada</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Division of Biomedical Statistics and Informatics, Department of Health Sciences Research, Mayo Clinic, Rochester, MN</div><div id="FN1">Correspondence and reprint requests: James N. Ingle, M.D., Mayo Clinic, 200 First Street S.W., Rochester, MN 55905, <a href="mailto:dev@null" data-email="ude.oyam@semaj.elgni" class="oemail">ude.oyam@semaj.elgni</a>, phone: 507-284-4857, fax 507-284-1803</div><div id="FN2"><sup>*</sup>Contributed equally</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140605709826544_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=27758888" target="pmc_ext" ref="reftype=publisher&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Cancer Res</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5135610/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="7" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-1.docx" data-ga-action="click_feat_suppl">NIHMS827024-supplement-1.docx</a><span style="color:gray"> (60K)</span></div><div class="small guid">GUID:&nbsp;622D5532-C63B-4581-ACC8-DE262F1280FB</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-2.tif" data-ga-action="click_feat_suppl">NIHMS827024-supplement-2.tif</a><span style="color:gray"> (24M)</span></div><div class="small guid">GUID:&nbsp;A87F482C-C6E6-45EC-BD35-6D13E4E465EC</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>3.</strong></div><div><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-3.tif" data-ga-action="click_feat_suppl">NIHMS827024-supplement-3.tif</a><span style="color:gray"> (5.7M)</span></div><div class="small guid">GUID:&nbsp;0FC7F24D-A9EE-4E68-89C5-06EFC1863802</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>4.</strong></div><div><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-4.tif" data-ga-action="click_feat_suppl">NIHMS827024-supplement-4.tif</a><span style="color:gray"> (18M)</span></div><div class="small guid">GUID:&nbsp;1764EE28-CA21-47A3-A30B-1339B5913C06</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>5.</strong></div><div><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-5.tif" data-ga-action="click_feat_suppl">NIHMS827024-supplement-5.tif</a><span style="color:gray"> (3.5M)</span></div><div class="small guid">GUID:&nbsp;14E192C1-5AE0-4940-8143-BDED815614E4</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>6.</strong></div><div><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-6.tif" data-ga-action="click_feat_suppl">NIHMS827024-supplement-6.tif</a><span style="color:gray"> (3.2M)</span></div><div class="small guid">GUID:&nbsp;4CB6D8A0-3AC0-4195-AAE2-257F64783A9A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>7.</strong></div><div><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-7.tif" data-ga-action="click_feat_suppl">NIHMS827024-supplement-7.tif</a><span style="color:gray"> (3.2M)</span></div><div class="small guid">GUID:&nbsp;C01BED0D-1C41-4263-9401-9CDB552056C3</div></div></dd></dl></div><div id="idm140605670293696" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140605670293696title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Genetic risks in breast cancer remain only partly understood. Here we report the results of a genome-wide association study of germline DNA from 4,658 women, including 252 women experiencing a breast cancer recurrence, who were entered on the MA.27 adjuvant trial comparing the aromatase inhibitors (AI) anastrozole and exemestane. Single nucleotide polymorphisms (SNP) of top significance were identified in the gene encoding MIR2052HG, a long noncoding RNA of unknown function. Heterozygous or homozygous individuals for variant alleles exhibited a ~40% or ~63% decrease, respectively, in the hazard of breast cancer recurrence relative to homozygous wild-type individuals. Functional genomic studies in lymphoblastoid cell lines and ERα-positive breast cancer cell lines showed that expression from MIR2052HG and the ESR1 gene encoding estrogen receptor-α (ERα) was induced by estrogen and AI in a SNP-dependent manner. Variant SNP genotypes exhibited increased ERα binding to estrogen response elements, relative to wild-type genotypes, a pattern that was reversed by AI treatment. Further, variant SNPs were associated with lower expression of MIR2052HG and ERα. RNAi-mediated silencing of MIR2052HG in breast cancer cell lines decreased ERα expression, cell proliferation and anchorage-independent colony formation. Mechanistic investigations revealed that MIR2052HG sustained ERα levels both by promoting AKT/FOXO3-mediated ESR1 transcription and by limiting ubiquitin-mediated, proteasome-dependent degradation of ERα. Taken together, our results define MIR2052HS as a functionally polymorphic gene that affects risks of breast cancer recurrence in women treated with AI. More broadly, our results offer a pharmacogenomic basis to understand differences in the response of breast cancer patients to AI therapy.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">GWAS, breast cancer, aromatase inhibitors, functional genomics, <em>MIR2052HG</em></span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">Breast cancer is the most common form of cancer in women both in the United States (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193033">1</a>) and worldwide (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193056">2</a>). Endocrine therapy is the most important treatment modality in the majority of women who have estrogen receptor α (ERα) positive breast cancer. Whereas tamoxifen, a selective ER modulator (SERM), has substantial value in reducing the risk of disease recurrence in women with ERα-positive early stage breast cancer (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193023">3</a>), a recent meta-analysis demonstrated that, when compared directly as monotherapy, aromatase inhibitors (AIs) were superior to tamoxifen in terms of local recurrence, distant recurrence, contralateral recurrence, breast cancer mortality, and all-cause mortality (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193021">4</a>). However, despite the clear efficacy of AIs as adjuvant therapy, this meta-analysis revealed that 19.1% of women treated with an AI, anastrozole or letrozole, experienced a recurrence of their breast cancer at 10 years and there was no indication of a plateau in the time to recurrence curve (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193010">4</a>).</p><p id="P3" class="p p-last">Canadian Cancer Trials Group MA.27 (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193037">5</a>) is the largest adjuvant endocrine therapy trial that exclusively studied AIs. Postmenopausal women with hormone receptor-positive early stage breast cancer were randomized to the steroidal AI exemestane or the non-steroidal AI anastrozole, and no difference in efficacy was identified (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193014">5</a>). We performed a genome-wide association study (GWAS), which indicated that germline genetic variability in SNP genotypes related to a gene encoding a lncRNA, may alter ERα expression and impact outcomes after treatment with AIs.. Functional genomic studies of this lncRNA and the SNPs related to it provided novel mechanisms by which the lncRNA might affect the level of AI benefit.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Source of Patients</h3><p id="P4" class="p p-first-last">Patients were obtained from the MA.27 trial (ClinicalTrials.gov number <a id="__tag_575193094" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT00066573" ref="reftype=extlink-clinical-trial&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT00066573","term_id":"NCT00066573"}}</span>NCT00066573</a>). MA.27 included postmenopausal women with histologically confirmed and completely resected stage I–III breast cancer (AJCC Version 6) that was ERα and/or PgR positive. Patients were randomized to five years of anastrozole or exemestane. Only North American patients were offered participation in collection of blood specimens and 5221 of 6827 (76.5%) of the North American patients contributed blood and gave consent for genetic testing. This research was performed after approval by local institutional review boards in accordance with assurances filed with, and approved by, the Department of Health and Human Services.</p></div><div id="S4" class="sec"><h3 id="S4title">Primary Outcome: Breast Cancer-Free Interval</h3><p id="P5" class="p p-first-last">The primary outcome was the STEEP endpoint of BCFI (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193032">6</a>), defined as the time from randomization to the first local-regional breast cancer recurrence (including ipsilateral DCIS), distant breast cancer recurrence, contralateral breast cancer (invasive or DCIS) or death with or from breast cancer without a prior recurrence date. Follow-up was censored at non-breast cancer death, or longest follow-up without recurrence.</p></div><div id="S5" class="sec"><h3 id="S5title">Genotyping, Imputation, and Quality Control</h3><p id="P6" class="p p-first">Three cohorts of patients from MA.27 were genotyped by the RIKEN Center for Integrative Medical Sciences. Cohort 1 involved patients genotyped as part of a GWAS with musculoskeletal adverse events as the phenotype utilizing the Illumina Human610 Quad Beadchip (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193016">7</a>). Cohort 2 involved patients genotyped as part of a GWAS with fragility bone fractures as the phenotype utilizing the Illumina Human OmniExpress platform (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193027">8</a>). Cohort 3 involved the remainder of the patients from MA.27 with DNA and consent. The quality control measures for cohorts 1 and 2 have been published (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193038">7</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193024">8</a>). For cohort 3, the following measures were taken for quality control purposes. One case and three controls were randomly chosen as duplicates for quality control of genotype concordance. A Caucasian parent-child Centre d’Etude du Polymorphisme Human trio from the HapMap was included to check for Mendelian transmission of alleles. Genotypes were determined utilizing the Illumina Human OmniExpressExome platform. This platform provided genotype data for 964,193 SNPs of which 2,923 were removed because they were from chromosome Y, mitochondria, or unplaced chromosomes. Additionally, 40,631 SNPs failed genotyping and 250,843 were rare SNPs with MAF&lt;0.01. Imputation was performed using EZimputer (<a href="#R9" rid="R9" class=" bibr popnode">9</a>) across the three cohorts separately. We then combined the genotyping data from all three cohorts to perform the current GWAS. Since each of the three cohorts were genotyped with a different platform, there were some SNPs that were genotyped in some patients but not in others. EZimputer (<a href="#R9" rid="R9" class=" bibr popnode">9</a>) (<a href="http://www.mayo.edu/research/departments-divisions/department-health-sciences-research/division-biomedical-statistics-informatics/software/bioinformatics-software-packages" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.mayo.edu/research/departments-divisions/department-health-sciences-research/division-biomedical-statistics-informatics/software/bioinformatics-software-packages</a>) imputes both un-genotyped SNPs and missing SNPs on a given platform. SNPs selected from imputation by EZimputer (<a href="#R9" rid="R9" class=" bibr popnode">9</a>) had a dosage r squared &gt;0.8 [r squared here is defined as the estimated squared correlation between the estimated allele dosage (0*P(Hom Ref/first) + 1*P(AB) + 2*P(Hom Alt/second)) and the true allele dosage].</p><p id="P7" class="p p-last">The data from this GWAS have been deposited in the Data Base of Genotypes and Phenotypes (dbGaP). The dbGaP Study Accession Number is phs001043 and the URL is <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001043.v1.p1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001043.v1.p1</a>.</p></div><div id="S6" class="sec"><h3 id="S6title">Deep Sequencing Methodology</h3><p id="P8" class="p p-first-last">We performed deep sequencing of the region containing the top imputed SNPs to determine the quality of the imputation as detailed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Materials</a>.</p></div><div id="S7" class="sec"><h3 id="S7title">Statistical Analyses</h3><p id="P9" class="p p-first">Exact Fisher tests were used to examine whether there were imbalances between North American patients who were included in this GWAS and those who were not. We also evaluated if any additional clinical variables were significantly associated with breast recurrence event using Stepwise selection method (<a href="#R10" rid="R10" class=" bibr popnode">10</a>). Our analysis employed a stratified genome-wide Cox-proportional hazards model using significant stratification factors and the model was further controlled for additional covariates including treatment arm, cohort, race, ER/PgR status, T-stage, ECOG performance score and bisphosphonate use. To avoid biases that might arise from differences in genetic ancestry (i.e., population stratification), the EigenStrat software was used to determine eigenvalues for the SNP correlation matrix that statistically differed from zero based on Tracy-Widom p-values (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193062">11</a>) (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193057">12</a>). All the analyses were run using the R statistical computing package (<a href="#R13" rid="R13" class=" bibr popnode">13</a>), PLINK (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193009">14</a>), and SAS (SAS Institute, Cary, NC).</p><p id="P10" class="p p-last">Haplotype analysis was performed using the top six SNPs. Haplotype probabilities for individual samples were estimated using haplo.stats package v1.7.7 in R v3.2.0 (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193059">15</a>) (details in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Materials</a>)</p></div><div id="S8" class="sec"><h3 id="S8title">Deep Sequencing Data Analysis</h3><p id="P11" class="p p-first-last">In order to determine the quality of imputation, we performed targeted deep sequencing of the 300kb region (chr8: 75,400,000–75,700,000) surrounding the chromosome 8 GWAS signal in a total of 997 patients (249 with and 748 without a breast event). We removed both 5′ and 3′-end primers with cutadapt-1.7.1 with minimum-length option = 20. The trimmed reads were aligned to the hg19 reference genome using BWA-MEM. SNVs and INDELs were called using HaplotypeCaller and genotyped across all samples with GenotypeGVCF from the Genome Analysis Tool Kit (GATK) (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193060">16</a>–<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_585664999">18</a>). Variants were annotated with functional features, impact prediction, and clinical significance using SnpEff (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193050">19</a>), ClinVar, HGMD, and ExAC population frequencies with the BioR annotation tool (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193013">20</a>). To demonstrate the imputation quality of our GWAS SNPs, we correlated the variant calls from deep sequencing with a set of imputed SNPs from the GWAS signal.</p></div><div id="S9" class="sec"><h3 id="S9title">Cell lines</h3><p id="P12" class="p p-first-last">Human breast cancer cell lines CAMA-1, HCC1428, BT474, AU565, BT549 and human embryonic kidney cell line 293T were obtained from American Type Culture Collection (ATCC, Manassus, VA) in 2014 and the identities of all cell lines were confirmed by the medical genome facility at Mayo Clinic Center (Rochester MN) using short tandem repeat profiling upon receipt. The breast cancer cell line MCF7/AC1, stably overexpressing aromatase (stably transfected <em>CYP19A1</em> gene), was generously gifted from Angela H. Brodie, Ph. D. (University of Maryland, Baltimore, MD). The cells were authenticated in 2015 by Genetica DNA Laboratories (Cincinnati, OH) using a StemElite ID system that uses short tandem repeat genotyping. All the cells used in our studies were within the initial five passages.</p></div><div id="S10" class="sec sec-last"><h3 id="S10title">Functional Genomics Studies</h3><p id="P13" class="p p-first-last">Details with respect to materials used, cell culture and lymphoblastoid cell line (LCLs) techniques, quantitative real-time PCR assay (qRT-PCR), chromatin Immunoprecipitation (ChIP) assays, cell proliferation assays, colony forming assays, western blotting are given in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Materials</a>.</p></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Results</h2><div id="S12" class="sec sec-first"><h3 id="S12title">Population of Patients Studied</h3><p id="P14" class="p p-first">The Participant Flow Diagram (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Fig. S1</a>) shows the patients included and excluded from the GWAS, and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> provides the patient characteristics and analyses revealed good comparability (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Materials</a>).</p><p id="P15">A total of 4,784 patients had sufficient DNA for genotyping but 4,658 patients were utilized for the GWAS following quality control procedures as specified in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Materials</a> and in the Participant Flow Diagram (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193030">21</a>) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Fig. S1</a>). The analysis included 252 women with a breast event and 4406 women who had not experienced a breast event (<a href="/pmc/articles/PMC5135610/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). We did not find any eigenvectors that were significantly associated with BCFI.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p10">Patient Characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140605708630064" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center">Breast Event=No (n=4406)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Breast Event=Yes (n=252)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Total (n=4658)</th></tr></thead><tbody><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>Genotyped Cohort</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Cohort 1</td><td rowspan="1" colspan="1" valign="top" align="center">826 (18.7%)</td><td rowspan="1" colspan="1" valign="top" align="center">43 (17.1%)</td><td rowspan="1" colspan="1" valign="top" align="center">869 (18.7%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Cohort 2</td><td rowspan="1" colspan="1" valign="top" align="center">834 (18.9%)</td><td rowspan="1" colspan="1" valign="top" align="center">47 (18.7%)</td><td rowspan="1" colspan="1" valign="top" align="center">881 (18.9%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Cohort 3</td><td rowspan="1" colspan="1" valign="top" align="center">2746 (62.3%)</td><td rowspan="1" colspan="1" valign="top" align="center">162 (64.3%)</td><td rowspan="1" colspan="1" valign="top" align="center">2908 (62.4%)</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>Age at Randomization</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Median</td><td rowspan="1" colspan="1" valign="top" align="center">64.4</td><td rowspan="1" colspan="1" valign="top" align="center">64</td><td rowspan="1" colspan="1" valign="top" align="center">64.4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Q1, Q3</td><td rowspan="1" colspan="1" valign="top" align="center">58.3, 71.4</td><td rowspan="1" colspan="1" valign="top" align="center">58.5, 71.3</td><td rowspan="1" colspan="1" valign="top" align="center">58.4, 71.4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Range</td><td rowspan="1" colspan="1" valign="top" align="center">36.1 – 95.1</td><td rowspan="1" colspan="1" valign="top" align="center">35.9 – 93.6</td><td rowspan="1" colspan="1" valign="top" align="center">35.9 – 95.1</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>Genotypic Race</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> African American</td><td rowspan="1" colspan="1" valign="top" align="center">134 (3.0%)</td><td rowspan="1" colspan="1" valign="top" align="center">6 (2.4%)</td><td rowspan="1" colspan="1" valign="top" align="center">140 (3.0%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Caucasian</td><td rowspan="1" colspan="1" valign="top" align="center">4216 (95.7%)</td><td rowspan="1" colspan="1" valign="top" align="center">243 (96.4%)</td><td rowspan="1" colspan="1" valign="top" align="center">4459 (95.7%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Asian</td><td rowspan="1" colspan="1" valign="top" align="center">56 (1.3%)</td><td rowspan="1" colspan="1" valign="top" align="center">3 (1.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">59 (1.3%)</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>Treatment Arm</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Anastrozole</td><td rowspan="1" colspan="1" valign="top" align="center">2210 (50.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">127 (50.4%)</td><td rowspan="1" colspan="1" valign="top" align="center">2337 (50.2%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Exemestane</td><td rowspan="1" colspan="1" valign="top" align="center">2196 (49.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">125 (49.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">2321 (49.8%)</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>Adjuvant Chemotherapy</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> No</td><td rowspan="1" colspan="1" valign="top" align="center">3143 (71.3%)</td><td rowspan="1" colspan="1" valign="top" align="center">124 (49.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">3267 (70.1%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Yes</td><td rowspan="1" colspan="1" valign="top" align="center">1263 (28.7%)</td><td rowspan="1" colspan="1" valign="top" align="center">128 (50.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">1391 (29.9%)</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>ECOG Performance Score</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> 0</td><td rowspan="1" colspan="1" valign="top" align="center">3694 (83.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">190 (75.4%)</td><td rowspan="1" colspan="1" valign="top" align="center">3884 (83.4%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> 1+</td><td rowspan="1" colspan="1" valign="top" align="center">712 (16.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">62 (24.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">774 (16.6%)</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>T-Stage</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> T1/TX</td><td rowspan="1" colspan="1" valign="top" align="center">3317 (75.3%)</td><td rowspan="1" colspan="1" valign="top" align="center">120 (47.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">3437 (73.8%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> T2</td><td rowspan="1" colspan="1" valign="top" align="center">997 (22.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">115 (45.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">1112 (23.9%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> T3</td><td rowspan="1" colspan="1" valign="top" align="center">92 (2.1%)</td><td rowspan="1" colspan="1" valign="top" align="center">17 (6.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">109 (2.3%)</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>Nodal Status</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Negative (N0)</td><td rowspan="1" colspan="1" valign="top" align="center">3290 (74.7%)</td><td rowspan="1" colspan="1" valign="top" align="center">121 (48.0%)</td><td rowspan="1" colspan="1" valign="top" align="center">3411 (73.2%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Positive (N1-3)</td><td rowspan="1" colspan="1" valign="top" align="center">1037 (23.5%)</td><td rowspan="1" colspan="1" valign="top" align="center">129 (51.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">1166 (25%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Unknown (NX)</td><td rowspan="1" colspan="1" valign="top" align="center">79 (1.79%)</td><td rowspan="1" colspan="1" valign="top" align="center">2 (0.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">81 (1.7%)</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>BMI at baseline</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Median</td><td rowspan="1" colspan="1" valign="top" align="center">28.5</td><td rowspan="1" colspan="1" valign="top" align="center">28.4</td><td rowspan="1" colspan="1" valign="top" align="center">28.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Q1, Q3</td><td rowspan="1" colspan="1" valign="top" align="center">25.0, 32.8</td><td rowspan="1" colspan="1" valign="top" align="center">24.9, 32.4</td><td rowspan="1" colspan="1" valign="top" align="center">25.0, 32.8</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Range</td><td rowspan="1" colspan="1" valign="top" align="center">15.3 – 68.7</td><td rowspan="1" colspan="1" valign="top" align="center">16.0 – 56.8</td><td rowspan="1" colspan="1" valign="top" align="center">15.3 – 68.7</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>Estrogen receptor, progesterone receptor</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Positive, Positive</td><td rowspan="1" colspan="1" valign="top" align="center">3553 (80.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">181 (71.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">3734 (80.2%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Positive, Negative</td><td rowspan="1" colspan="1" valign="top" align="center">732 (16.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">61 (24.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">793 (17%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Positive, Missing</td><td rowspan="1" colspan="1" valign="top" align="center">99 (2.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">6 (2.4%)</td><td rowspan="1" colspan="1" valign="top" align="center">105 (2.2%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Negative, Positive</td><td rowspan="1" colspan="1" valign="top" align="center">22 (0.5%)</td><td rowspan="1" colspan="1" valign="top" align="center">4 (1.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">26 (0.6%)</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>Bisphosphonate Use</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> No</td><td rowspan="1" colspan="1" valign="top" align="center">2949 (66.9%)</td><td rowspan="1" colspan="1" valign="top" align="center">196 (77.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">3145 (67.5%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Yes</td><td rowspan="1" colspan="1" valign="top" align="center">1457 (33.1%)</td><td rowspan="1" colspan="1" valign="top" align="center">56 (22.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">1513 (32.5%)</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>Trastuzumab use</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> No</td><td rowspan="1" colspan="1" valign="top" align="center">859 (19.5%)</td><td rowspan="1" colspan="1" valign="top" align="center">27 (10.7%)</td><td rowspan="1" colspan="1" valign="top" align="center">886 (19%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Unknown</td><td rowspan="1" colspan="1" valign="top" align="center">3507 (79.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">223 (88.5%)</td><td rowspan="1" colspan="1" valign="top" align="center">3730 (80.1%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Yes</td><td rowspan="1" colspan="1" valign="top" align="center">40 (0.9%)</td><td rowspan="1" colspan="1" valign="top" align="center">2 (0.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">42 (0.9%)</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"><strong>Type of Recurrence</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Any Distant</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">170 (67.5%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Local/Regional +/− Contra (no distant)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">42 (16.7%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Contralateral Only</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">40 (15.9%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140605708630064" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5135610/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P16" class="p p-last"><a href="/pmc/articles/PMC5135610/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> shows that the percentage of patients with a breast event was similar within each of the three genotyped cohorts (Cohort 1: 4.9%, Cohort 2: 5.3%, Cohort 3: 5.6%). Patients with and without a breast event were well balanced for age, treatment arm (anastrozole versus exemestane), and baseline body mass index. There were significant imbalances for adjuvant chemotherapy (P=3.68e-12), T stage (P=&lt;2e-16), lymph node status (p=&lt;2e-16) Eastern Cooperative Oncology Group (ECOG) performance score (P=0.0003), bisphosphonate use (P=0.0004), and ER/progesterone receptor (PgR) (p=0.008) with patients with a breast event having a higher percentage of ER-positive/PgR-negative and a lower percentage of ER-positive/PgR-positive tumors.</p></div><div id="S13" class="sec"><h3 id="S13title">Genotyping and Imputation</h3><p id="P17" class="p p-first-last">Genotyping for Cohorts 1 and 2 has been described previously (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193019">7</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193047">8</a>) utilizing the Illumina Human610 Quad BeadChip and the Illumina HumanOmniExpress platforms, respectively. For Cohort 3, after quality control measures, 669,796 genotyped SNPs were available for combining with genotype data from Cohorts 1 and 2. Imputation was performed using an in-house method, EZimputer (<a href="#R9" rid="R9" class=" bibr popnode">9</a>), which returned a total of 9.57M SNPs (observed plus imputed) with r squared &gt;0.8, of which 7.4M (observed + imputed) SNPs had a MAF&gt;0.01 and were used for the analyses.</p></div><div id="S14" class="sec"><h3 id="S14title">Genome-wide Association Study Analyses</h3><p id="P18" class="p p-first">We performed a stratified Cox-proportional hazards analysis utilizing stratification factors and other covariates as detailed in the Methods section. The Manhattan plot (<a href="/pmc/articles/PMC5135610/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1A</span></a>) shows that the SNPs with the lowest p-values mapped to chromosome 8 and the quantile-quantile plot (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Fig. S2</a>) revealed a lambda of 0.999. <a href="/pmc/articles/PMC5135610/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a> shows the locus zoom for the region with the top SNPs. Characteristics of the top six SNPs on chromosome 8 are listed in <a href="/pmc/articles/PMC5135610/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and these SNPs are all “favorable” in that they were associated with longer BCFI. All six SNPs were in strong linkage disequilibrium with R<sup>2</sup> values ranging from 0.95 to 0.99. The hazard ratios provided in <a href="/pmc/articles/PMC5135610/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> are for the presence of a single variant SNP genotype, which would be a heterozygous state. The presence of two variant SNP genotypes would be a homozygous variant state and the hazard ratio would be multiplicative. For example, considering rs4476990, the hazard ratio was 0.61, indicating ~39% reduction in the hazard of a breast cancer event for the heterozygous state relative to the homozygous wild type state, but the hazard ratio for women with the homozygous variant state would be 0.61×0.61=0.37, indicating ~63% reduction in the hazard of a breast cancer event, again relative to those with a homozygous wild type state. The p-values for the top six SNPs ranged from 2.15E-07 to 6.24E-07). Importantly, these variant SNP genotypes were common with MAFs ranging from 0.32 to 0.42.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5135610/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140605744250160" class="figure"><a href="/pmc/articles/PMC5135610/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5135610_nihms827024f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms827024f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5135610/bin/nihms827024f1.jpg"></a></div><div id="largeobj_idm140605744250160" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5135610/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5135610/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p5"><strong>A</strong>, GWAS Manhattan Plot. <strong>B</strong>, Locus zoom of the chromosome 8 region surrounding the <em>MIR2052HG</em> gene.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p11">SNPs with Lowest P-Values on Chromosome 8</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140605745289440" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center">Reference sequence</th><th rowspan="1" colspan="1" valign="bottom" align="center">Base Position</th><th rowspan="1" colspan="1" valign="middle" align="center">Hazard Ratio<sup>*</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">P-Value</th><th rowspan="1" colspan="1" valign="middle" align="center">Minor Allele Frequency</th><th rowspan="1" colspan="1" valign="middle" align="center">Genotyped or Imputed</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs13260300</td><td rowspan="1" colspan="1" valign="top" align="center">75476924</td><td rowspan="1" colspan="1" valign="top" align="center">0.61 (0.50–0.74)</td><td rowspan="1" colspan="1" valign="top" align="center">2.15E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">O<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4476990<sup>§</sup></td><td rowspan="1" colspan="1" valign="top" align="center">75478412</td><td rowspan="1" colspan="1" valign="top" align="center">0.61 (0.51–0.74)</td><td rowspan="1" colspan="1" valign="top" align="center">2.51E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">I<sup>‡</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2891356</td><td rowspan="1" colspan="1" valign="top" align="center">75473223</td><td rowspan="1" colspan="1" valign="top" align="center">0.61 (0.51–0.74)</td><td rowspan="1" colspan="1" valign="top" align="center">2.76E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">I</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs746460</td><td rowspan="1" colspan="1" valign="top" align="center">75474529</td><td rowspan="1" colspan="1" valign="top" align="center">0.62 (0.51–0.75)</td><td rowspan="1" colspan="1" valign="top" align="center">3.05E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">O</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4735715<sup>¶</sup></td><td rowspan="1" colspan="1" valign="top" align="center">75514554</td><td rowspan="1" colspan="1" valign="top" align="center">0.59 (0.48–0.73)</td><td rowspan="1" colspan="1" valign="top" align="center">4.47E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">I</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3802201<sup>§</sup>,<sup>¶</sup></td><td rowspan="1" colspan="1" valign="top" align="center">75516199</td><td rowspan="1" colspan="1" valign="top" align="center">0.60 (0.48–0.74)</td><td rowspan="1" colspan="1" valign="top" align="center">6.24E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">I</td></tr></tbody></table></div><div id="largeobj_idm140605745289440" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5135610/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>For event in patients carrying the variant SNP genotype;</div><div id="TFN2"><sup>†</sup>O: genotyped in at least one cohort;</div><div id="TFN3"><sup>‡</sup>I: imputed in all three cohorts;</div><div id="TFN4"><sup>§</sup>SNP in or near (within 500 bp) an estrogen response element motif;</div><div id="TFN5"><sup>¶</sup>SNP in <em>MIR2052HG</em>.</div></div></div><p id="P19">To validate our imputation quality, we used deep sequencing techniques to call variants surrounding the chromosome 8 peak. We compared the deep sequencing variants calls of these SNPs with those obtained from MA-27 data. The correlations between the variants (from deep sequencing) and the MA.27 SNPs were at least 0.9 in all the top SNPs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>), and thus of high quality.</p><p id="P20">The top SNP (rs13260300) was located 32.4kb 5′ of <em>MIR2052HG</em> (other names: <em>FLJ39080</em>, LOC441355), a gene located on chromosome 8q21.11, and two of the top SNPs were located in <em>MIR2052HG. MIR2052HG</em> encodes a lncRNA whose function is not known and with very few publications referring to this gene (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193048">22</a>, <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193042">23</a>).</p><p id="P21" class="p p-last">Because this study involved AIs as therapy for ER-positive breast cancer, we interrogated the top SNPs to determine which were located in or within 500 bp of a putative estrogen response element (ERE), similar to what we have done in previous studies (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193035">24</a>, <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193011">25</a>). The distance of 500 bp was chosen as chromatin immunoprecipitation (ChIP) assays become less reliable with greater distances between a SNP and an ERE. Two of the top SNPs fulfilled one of these two criteria with rs4476990 being located 33kb 5′ of <em>MIR2052HG</em> and in an ERE, while rs3802201 mapped to intron 1 of <em>MIR2052HG</em> and located 16bp from an ERE. We focused on these two SNPs, which are in moderate linkage disequilibrium (R<sup>2</sup>=0.6), in our functional studies. The significant relationships between the genotypes of the two SNPs, rs4476990 and rs3802201, and BCFI are shown in <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Figs. 3A and 3B</a>, respectively. After adjustment for covariates (treatment arm, cohort, race, ER/PR status, T-Stage, ECOG performance status and bisphosphonate use), a stratified Cox model using stratification factors (adjuvant chemotherapy, lymph node status, trastuzumab use) determined p=2.51E-07 for rs4476990 and p=6.24E-07 for rs3802201.</p></div><div id="S15" class="sec"><h3 id="S15title">Haplotype Analysis of the Top Six SNPs</h3><p id="P22" class="p p-first-last">A haplotype analysis of the top six SNPs was performed to determine the degree of association. The results of this analysis are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a> and reveal that the association for the most significant haplotype (HR 0.58, p=1.23E-06) was not superior to the top SNPs considered individually (<a href="/pmc/articles/PMC5135610/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p></div><div id="S16" class="sec"><h3 id="S16title">Functional Genomic Studies of Chromosome 8 SNPs</h3><p id="P23" class="p p-first-last">Because of the importance of ERα in endocrine therapy, we interrogated the Cancer Genome Atlas (TCGA) breast cancer data (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193054">26</a>) for a possible relationship between the expression of MIR2052HG and ESR1, which encodes ERα, in 485 ER-positive breast cancers. There was a positive correlation (Spearman correlation coefficient 0.370) between the expressions of these two genes. This provided an indication that there might be an important relationship between MIR2052HG and ESR1, which was supported by the functional genomic studies described in subsequent paragraphs. MIR2052HG is expressed in multiple breast cancer cell lines including ERα-positive cell lines, according to the Cancer Genomics Hub (<a href="https://cghub.ucsc.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://cghub.ucsc.edu</a>).</p></div><div id="S17" class="sec"><h3 id="S17title"><em>MIR2052HG</em> SNPs determine estradiol-dependent MIR2052HG expression and ERα binding to EREs</h3><p id="P24" class="p p-first-last">As mentioned earlier, the rs4476990 SNP was located in a putative ERE and the rs3802201 SNP in intron 1 of <em>MIR2052HG</em> was near another ERE (<a href="/pmc/articles/PMC5135610/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>). To test the possible functional impact of these two SNPs, we utilized a model system consisting of 300 individual human lymphoblastoid cell lines (LCLs) for which we have extensive genomic and transcriptomic data. This LCL model system has repeatedly shown its value in both generating and testing pharmacogenomics hypotheses (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193031">8</a>, <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193058">25</a>, <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193008">27</a>). Specifically, we selected 5 LCLs with homozygous wild type (WT) genotypes for both SNPs and 5 LCLs homozygous for variant genotypes for both SNPs to perform estradiol (E2) treatment and ChIP assays. In the presence of E2, cells homozygous for the variant SNP genotypes showed a dose-dependent increase in MIR2052HG expression (<a href="/pmc/articles/PMC5135610/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>) as well as increased binding of ERα to the EREs shown in <a href="/pmc/articles/PMC5135610/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a> for variant genotypes for both SNPs relative to WT in ChIP assays using ERα antibody (<a href="/pmc/articles/PMC5135610/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figs. 2C and 2D</span></a>). 4-Hydroxy-tamoxifen (4-OH-Tam), a SERM that competes with E2 for ERα binding, could reverse this effect (<a href="/pmc/articles/PMC5135610/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figs. 2C and 2D</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5135610/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140605671379232" class="figure"><a href="/pmc/articles/PMC5135610/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5135610_nihms827024f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms827024f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5135610/bin/nihms827024f2.jpg"></a></div><div id="largeobj_idm140605671379232" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5135610/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5135610/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p6"><strong>A</strong>. Schematic of estrogen response elements (EREs) around rs4476990 and rs3802201. The EREs are indicated as boxes and the SNPs are indicated as red circles. <strong>B</strong>. MIR2052HG mRNA expression in lymphoblastoid cell lines (LCLs) with wild type SNP (W) and variant SNP (V) genotypes for both rs4476990 and rs3802201 after exposure to increasing concentrations of E2. Error bars represent SEM. *P&lt;0.05. <strong>C.</strong> and <strong>D.</strong>, ChIP assay using six LCLs with known genotypes for rs4476990 and rs3802201 SNPs. Error bars represent SEM of three independent experiments. Representative PCR products are visualized on agarose DNA gels.</p></div></div></div></div><div id="S18" class="sec"><h3 id="S18title">Aromatase inhibitors reverse estradiol-dependent and SNP-dependent MIR2052HG and ESR1 expression</h3><p id="P25" class="p p-first">The major function of AIs is to reduce estrogen levels by the inhibition of aromatase, the rate-limiting step in estrogen biosynthesis. The reduction of estrogens could have an effect on ERα-mediated function. As we have shown in our previous studies, estrogens and SERMs can alter gene expression in a SNP-dependent fashion (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193039">25</a>). Thus, we proceeded to examine the effect of the AIs anastrozole and exemestane on MIR2052HG and ESR1 expression in the presence of androstenedione, which is aromatized to estrone by aromatase, the target for the AIs under study.</p><p id="P26" class="p p-last">In the presence of androstenedione, LCLs with variant genotypes for both SNPs showed dose-dependent increases in MIR2052HG expression (<a href="/pmc/articles/PMC5135610/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figs. 3A and B</span></a>) that was similar to that for E2 (<a href="/pmc/articles/PMC5135610/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>). However, the addition an AI, either exemestane (<a href="/pmc/articles/PMC5135610/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3A</span></a>) or anastrozole (<a href="/pmc/articles/PMC5135610/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3B</span></a>), to the androstenedione-treated LCLs caused a “reversal” of the expression pattern with increased MIR2052HG expression in LCLs homozygous for the WT genotypes but a marked decrease in LCLs homozygous for variant genotypes. Of particular interest was the observation of a direct correlation between this striking pattern of expression for MIR2052HG and that of ESR1 in the same cell lines (<a href="/pmc/articles/PMC5135610/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figs. 3C and 3D</span></a>), bringing us back to the correlation that we had observed between the expression of MIR2052HG and ESR1 in the TCGA data.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC5135610/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140605671374880" class="figure"><a href="/pmc/articles/PMC5135610/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5135610_nihms827024f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms827024f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5135610/bin/nihms827024f3.jpg"></a></div><div id="largeobj_idm140605671374880" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5135610/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC5135610/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p7"><strong>A</strong> and <strong>B</strong>. MIR2052HG mRNA expression in lymphoblastoid cell lines (LCLs) withWT and V genotypes for both rs4476990 and rs3802201 after exposure to androstenedione alone and with increasing concentrations of exemestane or anastrozole. <strong>C</strong> and <strong>D</strong>. mRNA expression for ESR1 in LCLs with the same conditions as A and B. *P&lt;0.05, **P&lt;0.01.</p></div></div></div></div><div id="S19" class="sec"><h3 id="S19title">MIR2052HG affects ESR1 and ERα expression and proliferation, colony formation, and response to AIs in ERα-positive breast cancer cell lines</h3><p id="P27" class="p p-first-last">Having determined that, in a SNP and AI-dependent fashion, the expression of MIR2052HG was correlated with that of ESR1 (<a href="/pmc/articles/PMC5135610/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>), we set out to study the possible functional impact of the MIR2052HG lncRNA on AI response and on cell proliferation. When we began our studies, the function of MIR2052HG was not known, but we hypothesized, based on the results shown in <a href="/pmc/articles/PMC5135610/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>, that this lncRNA might influence AI response through its effect on the downstream expression of ESR1. To determine the effect of MIR2052HG knock down on ERα levels, we chose two ERα-positive breast cancer cell lines with relatively high endogenous MIR2052HG expression, CAMA1 and the aromatase expressing MCF7/AC1 (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193026">28</a>) cell lines. Knock down of MIR2052HG resulted in striking decreases of ERα expression, both at the mRNA and the protein levels (<a href="/pmc/articles/PMC5135610/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figs. 4A and 4B</span></a>), consistent with the TCGA data that showed a positive correlation between the two genes. Furthermore, in MCF7/AC1 cells, knock down of MIR2052HG decreased cell proliferation and colony formation (<a href="/pmc/articles/PMC5135610/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figs. 4C and 4D</span></a>), while overexpression of MIR2052HG increased cell proliferation, colony formation, and ERα expression (<a href="/pmc/articles/PMC5135610/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4E</span></a>), functionally confirming the positive relationship between ERα and MIR2052HG. We observed the same results in two additional ERα-positive breast cancer cell lines, HCC1428 and BT474, with regard to the effect of MIR2052HG on ERα protein levels and cell proliferation (<a href="#SD5" rid="SD5" class=" supplementary-material">Supplementary Fig. S4</a>). Also, it is well known that androstenedione increases MCF7/AC1 proliferation and that this increase in proliferation can be abrogated with AIs (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193046">29</a>). However, overexpression of MIR2052HG significantly increased cell proliferation even in the presence of AI treatment (<a href="/pmc/articles/PMC5135610/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4F</span></a>). Conversely, down-regulation of MIR2052HG inhibited MCF7/AC1 cell proliferation induced by either E2 or androstenedione (<a href="#SD6" rid="SD6" class=" supplementary-material">Supplementary Fig. S5</a>). No significant change was observed in the proliferation of ERα-negative cells (AU565 and BT549) after the down-regulation of MIR2052HG (<a href="#SD7" rid="SD7" class=" supplementary-material">Supplementary Fig. S6</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC5135610/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140605672519088" class="figure"><a href="/pmc/articles/PMC5135610/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5135610_nihms827024f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms827024f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5135610/bin/nihms827024f4.jpg"></a></div><div id="largeobj_idm140605672519088" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5135610/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC5135610/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p8"><strong>A</strong>, Knock down of MIR2052HG by antisense oligonucleotides (ASO1 and ASO2) down-regulated ERα protein. The histogram shows knock down efficiency in CAMA1 and MCF7/AC1 cells. <strong>B</strong>, Knock down of MIR2052HG decreased ESR1 mRNA expression levels in CAMA1 and MCF7/AC1 cell lines. <strong>C</strong> and <strong>D</strong>. Knock down of MIR2052HG decreased proliferation and colony formation in CAMA1 and MCF7/AC1 cells. The representative colony formation pictures from triplicate experiments are shown. The colony formation rates are quantified as percentages. Error bars represent SEM; ** P&lt; 0.01 compared to baseline (negative control). <strong>E</strong>, Overexpression of MIR2052HG increased ERα protein levels. Overexpression efficiency was determined by qRT-PCR. Overexpression of MIR2052HG increased the proliferation and colony formation and ERα protein levels in MCF7/AC1 cells. <strong>F.</strong> Overexpression of MIR2052HG conferred resistance to AIs (exemestane and anastrozole) and 4-hydroxy-tamoxifen treatments in MCF7/AC1 compared to negative controls. The assay was performed as described in C and D. Error bars represent SEM. The concentrations for androstenedione (A), exemestane (EXE, anastrozole (ANA), estradiol (E2) and 4-hydroxy-tamoxifen (TAM) are indicated.</p></div></div></div></div><div id="S20" class="sec sec-last"><h3 id="S20title">MIR2052HG regulates ERα expression through transcription and protein degradation pathways</h3><p id="P28" class="p p-first">MIR2052HG expression is associated with both ERα mRNA and protein levels (<a href="/pmc/articles/PMC5135610/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4A</span></a>). To pursue these observations, we began by determining whether MIR2052HG might affect ERα protein stability mediated by protein degradation. Treatment with cycloheximide, resulted in a decrease in the half-life of ERα protein in cells in which MIR2052HG had been knocked down (<a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5A and 5B</span></a>). We also treated MIR2052HG knock down cells with MG132, a proteasome inhibitor, and found that it reversed ERα degradation in these cells (<a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5C</span></a>, upper panel). The same phenomenon was observed using bortezomib, another specific proteasome inhibitor (<a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5C</span></a>, lower panel). These results indicated that MIR2052HG regulates ERα protein stability through a proteasome-mediated degradation pathway. Furthermore, we observed that ERα ubiquitination increased after knocking down MIR2052HG in 293T cells (<a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5D</span></a>), confirming the involvement of ubiquitin-dependent and proteasome-mediated degradation.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140605672512256" class="figure"><a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5135610_nihms827024f5.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms827024f5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5135610/bin/nihms827024f5.jpg"></a></div><div id="largeobj_idm140605672512256" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5135610/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC5135610/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p id="__p9"><strong>A</strong>, Knock down of MIR2052HG shortened ERα protein half-life. CAMA1 cells were transfected with <em>MIR2052HG</em> specific antisense oligonucleotides (ASOs) or a negative control ASO and then treated with cycloheximide (CHX). The representative western blotting results from three independent experiments are shown. The knock down efficiency was determined by qRT-PCR. <strong>B</strong>. Quantitative intensities of ERα are mean values with SEM (error bars) from three independent experiments. <strong>C</strong>. MIR2052HG regulated ERα stability in a proteasome-dependent manner. MIR2052HG was knocked down with ASOs in CAMA1 cells that were treated with MG132 or bortezomib. <strong>D</strong>. Knock down of MIR2052HG promoted the ubiquitination of ERα. 293T cells were transfected with HA-Ub plasmid and FLAG-ERα plasmid, and then transfected with either the MIR2052HG specific ASOs or the negative control ASO followed by MG132. ERα proteins were immunoprecipitated and analyzed by western blotting. Knock down efficiency in 293T cells was determined by qRT-PCR. <strong>E</strong>. MIR2052HG regulated ERα transcription through the AKT-FOXO3 pathway. Knock down of MIR2052HG increased AKT phosphorylation and decreased FOXO3 phosphorylation and FOXO3 total level in MCF7/AC1 and CAMA1 cells. <strong>F</strong>. Overexpression of FOXO3 in MIR2052HG knocked-down MCF7/AC1 cells reversed ERα protein and mRNA levels. Overexpression of FOXO3 was determined by western blotting. HA: expression tag.</p></div></div></div><p id="P29" class="p p-last">We next examined possible mechanisms by which MIR2052HG was involved in the control of ERα transcription. During these experiments, we observed in both MCF7/AC1 and CAMA1 cells that down-regulation of MIR2052HG resulted in increased phospho-AKT (pAKT) levels at both the Ser473 and Thr308 sites, but total AKT levels did not change (<a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig 5E</span></a>). Because FOXO3 is downstream of AKT and activated AKT phosphorylates FOXO3 resulting in degradation of FOXO3 through a proteasome-dependent process (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193017">30</a>), we determined the effects of MIR2052HG knock down on total FOXO3 and phospho-FOXO3 levels (on S318/S321) and observed both of them to be reduced (<a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5E</span></a>), consistent with the known effect of pAKT (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193044">30</a>). It is also well known that FOXO3 regulates the expression of ERα (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193022">31</a>–<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193029">33</a>) and that the expression of FOXO3 is directly correlated with the expression of ERα. To further confirm that the regulation of <em>ESR1</em> mRNA levels by MIR2052HG is mediated through the regulation of FOXO3, we overexpressed FOXO3 in MCF/AC1 cells, in which MIR2052HG had been knocked down, and observed that FOXO3 overexpression could reverse the down-regulation of <em>ESR1</em> mRNA caused by knocking down MIR2052HG (<a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5F</span></a>). In summary, these results indicate that the down-regulation of MIR2052HG can reduce <em>ESR1</em> mRNA levels by promoting AKT-mediated down-regulation of FOXO3, which regulates ESR1 transcription. Thus, it appears that tumor expression of MIR2052HG plays a role in the regulation of ERα transcription in addition to ERα protein degradation.</p></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Discussion</h2><p id="P30" class="p p-first">Recurrence of breast cancer in women with early stage disease treated with adjuvant endocrine therapy implies endocrine resistance. Multiple potential mechanisms, mainly focusing on ERα function, have been proposed for this resistance but, by and large, these mechanisms have been related to factors present in the cancers (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193064">34</a>–<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193025">36</a>). Much less attention has been paid to host-related factors for endocrine resistance or, more appropriately, lack of efficacy of endocrine therapy, such as CYP2D6 poor metabolizer genotype in the case of tamoxifen (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193018">37</a>). In the current study, our goal was to interrogate the germline genome for SNPs related to breast cancer events in women treated with adjuvant AI therapy, relate those SNPs to genes, and perform functional studies to identify potential mechanisms for the observed associations. We controlled for significant imbalances in three stratification factors by performing stratified Cox analyses and for the effects of baseline factors that impacted BCFI with their forced inclusion in the Cox model. The p-values for the top SNPs were approximately 2E-07, which approaches, but does not reach genome-wide significance. However, because of the importance of the phenotype and the fact that we were studying the largest study that had evaluated AIs and had DNA available, we chose to pursue these signals with functional genomic experiments, with strikingly positive findings. We acknowledge that a replication dataset would have been of value but we considered it important to report our compelling data despite the lack of an available dataset.</p><p id="P31">Our GWAS identified variant SNPs on chromosome 8 that were protective and in or near a gene (<em>MIR2052HG</em>) that encodes a lncRNA. There is increasing appreciation of the role of lncRNAs in regulation of the genome (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193020">38</a>). For example, lncRNAs can form extensive networks of ribonucleoprotein complexes with chromatin regulators and modulate them (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193036">39</a>, <a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193055">40</a>). There is also increasing evidence suggesting lncRNAs play important roles in cancers (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193063">41</a>, <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193053">42</a>). For example, HOTAIR (Hox antisense intergenic RNA) is highly induced in about one-quarter of patients with breast cancer (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193043">43</a>). The long intergenic non-coding RNA-ROR has been shown to induce epithelial-to-mesenchymal transition and contribute to breast cancer metastasis (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193049">44</a>) and to enhance ERα signaling, conferring resistance to tamoxifen (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193041">45</a>). A recent study also indicated that a cluster of lncRNAs, termed Eleanors (ESR1 locus enhancing and activating noncoding RNAs) located within the genomic region containing the ESR1 gene can regulate ERα levels through an enhancer function (<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193015">46</a>). Of note is the fact that the lncRNA SRA1 (steroid receptor RNA activator 1) acted as a coactivator of ERα, and this action depended on the phosphorylation of ERα at Ser118 (<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193034">47</a>).</p><p id="P32">The role of lncRNAs in resistance to endocrine therapy is an area of emerging interest (<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193040">48</a>, <a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193051">49</a>). However, our study is, to our knowledge, the first to focus on the impact of a lncRNA on outcomes in a large prospective trial of AI therapy in women with early-stage breast cancer. Our finding of the relationship between SNPs related to the <em>MIR2052HG</em> lncRNA and recurrence of breast cancer in women treated with adjuvant AIs led us to perform a series of functional studies to discover potential mechanisms for this association. Using an LCL model system, we showed that, in the presence of E2, these variant SNPs increased both MIR2052HG and ESR1 expression with increased ERα binding to ERE motifs for variant SNP genotypes as shown by ChIP assays (<a href="/pmc/articles/PMC5135610/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figs. 2C, 2D</span></a>). However, when an AI (exemestane or anastrozole) was added, LCLs with the WT SNP genotype displayed a marked up-regulation of MIR2052HG expression and, in parallel, ESR1 expression whereas cells with the variant SNP genotypes showed a clear decrease in both MIR2052HG and ESR1 expression (<a href="/pmc/articles/PMC5135610/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). That is, the presence of the AI brought about a “reversal” of the SNP-dependent MIR2052HG expression pattern. SNP- and drug-dependent regulation of gene expression has been previously reported by our group in the case of selective estrogen receptor modulators (SERMs), tamoxifen and raloxifene, when given as preventive therapy, which led to the identification of novel mechanisms by which SNPs can regulate gene expression (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193052">25</a>). Thus, our previous observations with SERMs (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193045">25</a>) provided an impetus to investigate the role of this lncRNA in the efficacy of AI therapy as well as in the regulation of hormone-dependent breast cancer.</p><p id="P33"><em>MIR2052HG</em> is located on chromosome 8 and not in the <em>ESR1</em> genomic region on chromosome 6, but it has a significant effect on ERα regulation as noted above. ERα plays an essential role in cell proliferation and survival in estrogen-dependent breast cancers and in AI-treated patients, <em>ESR1</em> amplification, resulting in increased ERα expression, is associated with endocrine resistance (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193028">36</a>). We showed that overexpression of MIR2052HG increased ERα expression and accelerated cell proliferation of MCF/AC1 cells in the presence of AI treatments (<a href="/pmc/articles/PMC5135610/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figs. 4E, 4F</span></a>). Conversely, down-regulation of MIR2052HG reduced cell proliferation and colony formation even after E2 or androstenedione treatment (<a href="/pmc/articles/PMC5135610/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figs. 4C, 4D</span></a> and <a href="#SD6" rid="SD6" class=" supplementary-material">Supplementary Fig. 5</a>). These phenomena are specific for ERα-positive cells, since in ER-negative cell lines, we did not observe an effect of MIR2052HG on cell proliferation (<a href="#SD7" rid="SD7" class=" supplementary-material">Supplementary Fig. 6</a>). Based on our observations of the effects of MIR2052HG on both ERα protein and mRNA levels, we hypothesized that MIR2052HG might regulate ERα through a proteasome-mediated pathway, which we experimentally confirmed by treatment with proteasome inhibitors and with ubiquitin assays (<a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>). Furthermore, regulation of the transcription of ERα by MIR2052HG was found to be through AKT-dependent FOXO3 regulation; FOXO3 is a known transcription factor for ERα (<a href="/pmc/articles/PMC5135610/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figs. 5E, 5F</span></a>).</p><p id="P34">Through our GWAS using germline DNA samples from the largest AI clinical trial (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193061">5</a>), we have identified a novel lncRNA that potentially plays an important role in the regulation of ERα levels, one of the mechanisms involved in AI resistance. Our GWAS indicated that two variant SNPs (rs4476990 and rs3802201) were common variants with MAF values of 42% and 32%, respectively, and that both were protective, i.e. patients with the variant SNP genotypes had a longer BCFI. The results of our mechanistic studies supported the association in that both variant SNPs down-regulated MIR2052HG expression in the presence of AIs, which was associated with the down-regulation of ERα at both the mRNA level and the protein levels. The presence of markedly increased ESR1 expression in the presence of either anastrozole or exemestane in LCLs with the WT SNP genotype is a potential mechanism for the adverse outcomes in patients carrying the WT SNP genotype. Conversely, down-regulation of ERα in the presence of the variant SNP genotypes after exposure to anastrozole or exemestane might be a factor contributing to their more favorable BCFI.</p><p id="P35">At the mechanistic level, how MIR2052HG regulates the AKT pathway remains to be further investigated. It could have a direct impact on AKT phosphorylation or more likely, through the regulation of upstream proteins that affect AKT activity. Additionally, the mechanism by which MIR2052HG regulates ERα ubiquitin- and proteasome-mediated degradation also remains unresolved, but previous studies have suggested ER phosphorylation can influence its ubiquitination (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_575193012">50</a>). Therefore, one possibility is that the MIR2052HG could affect ERα levels by regulating various proteins that might affect ERα phosphorylation. Our current findings suggest that MIR2052HG could affect both ERα mRNA and protein levels through different mechanisms.</p><p id="P36" class="p p-last">In summary, we have identified SNP genotypes on chromosome 8 that were associated with breast cancer outcomes in women treated with the AIs anastrozole or exemestane as adjuvant therapy for their early-stage breast cancer, and we related these SNPs to a lncRNA, which, in a SNP-dependent and AI-dependent fashion, regulated ERα expression. The variant SNP genotypes, which are favorable in women treated with anastrozole and exemestane, are potential markers that could identify women for whom either AI would be appropriate therapy, i.e., for those women whose germline carries the variant SNP genotypes. This would require further corroboration with additional clinical testing. However, our GWAS and functional studies have provided initial evidence that germline genetic variability in SNP genotypes related to a gene encoding a lncRNA, <em>MIR2052HG</em>, may impact outcomes after treatment with the AIs anastrozole or exemestane.</p></div><div id="S22" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d36e846"><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-1.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(60K, docx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><div class="sup-box half_rhythm" id="d36e850"><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-2.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(24M, tif)</sup></div></div><div class="sec suppmat" id="SD3"><h4>3</h4><div class="sup-box half_rhythm" id="d36e854"><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-3.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(5.7M, tif)</sup></div></div><div class="sec suppmat" id="SD4"><h4>4</h4><div class="sup-box half_rhythm" id="d36e858"><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-4.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(18M, tif)</sup></div></div><div class="sec suppmat" id="SD5"><h4>5</h4><div class="sup-box half_rhythm" id="d36e862"><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-5.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.5M, tif)</sup></div></div><div class="sec suppmat" id="SD6"><h4>6</h4><div class="sup-box half_rhythm" id="d36e866"><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-6.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.2M, tif)</sup></div></div><div class="sec suppmat" id="SD7"><h4>7</h4><div class="sup-box half_rhythm" id="d36e870"><a href="/pmc/articles/PMC5135610/bin/NIHMS827024-supplement-7.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.2M, tif)</sup></div></div></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">Acknowledgments</h2><div class="sec"><p id="__p2">Financial support: These studies were supported in part by NIH grants U19 GM61388 (The Pharmacogenomics Research Network), P50CA116201 (Mayo Clinic Breast Cancer Specialized Program of Research Excellence), U10CA77202, R01CA196648, CCS 015469 from the Canadian Cancer Society, the Breast Cancer Research Foundation, and the RIKEN Center for Integrative Medical Sciences and the Biobank Japan Project funded by the Ministry of Education, Culture, Sports, Science and Technology, Japan. The MA.27 trial was supported, in part, by Pfizer, Inc.</p><p id="__p3">The authors acknowledge the women who participated in the MA.27 clinical trial and provided DNA and consent for its use in genetic studies.</p></div></div><div id="idm140605671253696" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140605671253696title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="__p4" class="p p-first-last">Conflict of interest: No potential conflicts of interest were disclosed by the authors.</p><p></p></div></div><div id="idm140605671252464" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140605671252464title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Siegel RL, Miller KD, Jemal A. Cancer statistics, 2015. <span><span class="ref-journal">CA Cancer J Clin. </span>2015;<span class="ref-vol">65</span>:5–29.</span> [<a href="/pubmed/25559415" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CA+Cancer+J+Clin&amp;title=Cancer+statistics,+2015&amp;author=RL+Siegel&amp;author=KD+Miller&amp;author=A+Jemal&amp;volume=65&amp;publication_year=2015&amp;pages=5-29&amp;pmid=25559415&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Ferlay J, Soerjomataram I, Dikshit R, Eser S, Mathers C, Rebelo M, et al.  Cancer incidence and mortality worldwide: sources, methods and major patterns in GLOBOCAN 2012. <span><span class="ref-journal">Int J Cancer. </span>2015;<span class="ref-vol">136</span>:E359–86.</span> [<a href="/pubmed/25220842" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Cancer&amp;title=Cancer+incidence+and+mortality+worldwide:+sources,+methods+and+major+patterns+in+GLOBOCAN+2012&amp;author=J+Ferlay&amp;author=I+Soerjomataram&amp;author=R+Dikshit&amp;author=S+Eser&amp;author=C+Mathers&amp;volume=136&amp;publication_year=2015&amp;pages=E359-86&amp;pmid=25220842&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Davies C, Godwin J, Gray R, Clarke M, Cutter D, Darby S, et al.  Relevance of breast cancer hormone receptors and other factors to the efficacy of adjuvant tamoxifen: patient-level meta-analysis of randomised trials. <span><span class="ref-journal">Lancet. </span>2011;<span class="ref-vol">378</span>:771–84.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3163848/">PMC free article</a>]</span> [<a href="/pubmed/21802721" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Relevance+of+breast+cancer+hormone+receptors+and+other+factors+to+the+efficacy+of+adjuvant+tamoxifen:+patient-level+meta-analysis+of+randomised+trials&amp;author=C+Davies&amp;author=J+Godwin&amp;author=R+Gray&amp;author=M+Clarke&amp;author=D+Cutter&amp;volume=378&amp;publication_year=2011&amp;pages=771-84&amp;pmid=21802721&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Dowsett M, Forbes JF, Bradley R, Ingle J, Aihara T, Bliss J, et al.  Aromatase inhibitors versus tamoxifen in early breast cancer: patient-level meta-analysis of the randomised trials. <span><span class="ref-journal">Lancet. </span>2015;<span class="ref-vol">386</span>:1341–52.</span> [<a href="/pubmed/26211827" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Aromatase+inhibitors+versus+tamoxifen+in+early+breast+cancer:+patient-level+meta-analysis+of+the+randomised+trials&amp;author=M+Dowsett&amp;author=JF+Forbes&amp;author=R+Bradley&amp;author=J+Ingle&amp;author=T+Aihara&amp;volume=386&amp;publication_year=2015&amp;pages=1341-52&amp;pmid=26211827&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Goss PE, Ingle JN, Pritchard KI, Ellis MJ, Sledge GW, Budd GT, et al.  Exemestane versus anastrozole in postmenopausal women with early breast cancer: NCIC CTG MA.27--a randomized controlled phase III trial. <span><span class="ref-journal">J Clin Oncol. </span>2013;<span class="ref-vol">31</span>:1398–404.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3612593/">PMC free article</a>]</span> [<a href="/pubmed/23358971" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Oncol&amp;title=Exemestane+versus+anastrozole+in+postmenopausal+women+with+early+breast+cancer:+NCIC+CTG+MA.27--a+randomized+controlled+phase+III+trial&amp;author=PE+Goss&amp;author=JN+Ingle&amp;author=KI+Pritchard&amp;author=MJ+Ellis&amp;author=GW+Sledge&amp;volume=31&amp;publication_year=2013&amp;pages=1398-404&amp;pmid=23358971&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Hudis CA, Barlow WE, Costantino JP, Gray RJ, Pritchard KI, Chapman JA, et al.  Proposal for standardized definitions for efficacy end points in adjuvant breast cancer trials: the STEEP system. <span><span class="ref-journal">J Clin Oncol. </span>2007;<span class="ref-vol">25</span>:2127–32.</span> [<a href="/pubmed/17513820" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Oncol&amp;title=Proposal+for+standardized+definitions+for+efficacy+end+points+in+adjuvant+breast+cancer+trials:+the+STEEP+system&amp;author=CA+Hudis&amp;author=WE+Barlow&amp;author=JP+Costantino&amp;author=RJ+Gray&amp;author=KI+Pritchard&amp;volume=25&amp;publication_year=2007&amp;pages=2127-32&amp;pmid=17513820&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Ingle JN, Schaid DJ, Goss PE, Liu M, Mushiroda T, Chapman JA, et al.  Genome-wide associations and functional genomic studies of musculoskeletal adverse events in women receiving aromatase inhibitors. <span><span class="ref-journal">J Clin Oncol. </span>2010;<span class="ref-vol">28</span>:4674–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3020700/">PMC free article</a>]</span> [<a href="/pubmed/20876420" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Oncol&amp;title=Genome-wide+associations+and+functional+genomic+studies+of+musculoskeletal+adverse+events+in+women+receiving+aromatase+inhibitors&amp;author=JN+Ingle&amp;author=DJ+Schaid&amp;author=PE+Goss&amp;author=M+Liu&amp;author=T+Mushiroda&amp;volume=28&amp;publication_year=2010&amp;pages=4674-82&amp;pmid=20876420&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Liu M, Goss PE, Ingle JN, Kubo M, Furukawa Y, Batzler A, et al.  Aromatase inhibitor-associated bone fractures: a case-cohort GWAS and functional genomics. <span><span class="ref-journal">Mol Endocrinol. </span>2014;<span class="ref-vol">28</span>:1740–51.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4179631/">PMC free article</a>]</span> [<a href="/pubmed/25148458" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Endocrinol&amp;title=Aromatase+inhibitor-associated+bone+fractures:+a+case-cohort+GWAS+and+functional+genomics&amp;author=M+Liu&amp;author=PE+Goss&amp;author=JN+Ingle&amp;author=M+Kubo&amp;author=Y+Furukawa&amp;volume=28&amp;publication_year=2014&amp;pages=1740-51&amp;pmid=25148458&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Sicotte HNPN, Johnson JA, Lin Y, Decker PA, Eckel-Passow JE, Jenkins RB, Goetz MPR, Weinshilboum R, Boughey JC, Wang L, Kalari KR, McDonnell SK, de Andrade M, Kocher JP. Single Sample Imputation from Next Generation Sequencing (NGS) Exome data can improve genotypes in low-coverage regions. 65th Annual Meeting of The American Society of Human Genetics; Baltimore, MD. 2015. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Sicotte+HNPN+Johnson+JA+Lin+Y+Decker+PA+Eckel-Passow+JE+Jenkins+RB+Goetz+MPR+Weinshilboum+R+Boughey+JC+Wang+L+Kalari+KR+McDonnell+SK+de+Andrade+M+Kocher+JP+Single+Sample+Imputation+from+Next+Generation+Sequencing+(NGS)+Exome+data+can+improve+genotypes+in+low-coverage+regions+65th+Annual+Meeting+of+The+American+Society+of+Human+Genetics+Baltimore,+MD+2015+" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">PROC PHREG : : SAS Institute Inc. <span class="ref-journal">SAS/STAT® 9.2 User’s Guide.</span> Cary, NC: SAS Institute Inc; 2008.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=SAS/STAT®+9.2+User’s+Guide&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Patterson N, Price AL, Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N+Patterson&amp;author=AL+Price&amp;author=D+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–9.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Therneau TM. <span><span class="ref-journal">A Package for Survival Analysis in S. version 2.38. </span>2015</span> Retrieved from  <a href="http://CRAN.R-project.org/package=survival" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://CRAN.R-project.org/package=survival</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Schaid DJ, Rowland CM, Tines DE, Jacobson RM, Poland GA. Score tests for association between traits and haplotypes when linkage phase is ambiguous. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">70</span>:425–34.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384917/">PMC free article</a>]</span> [<a href="/pubmed/11791212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Score+tests+for+association+between+traits+and+haplotypes+when+linkage+phase+is+ambiguous&amp;author=DJ+Schaid&amp;author=CM+Rowland&amp;author=DE+Tines&amp;author=RM+Jacobson&amp;author=GA+Poland&amp;volume=70&amp;publication_year=2002&amp;pages=425-34&amp;pmid=11791212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">DePristo MA, Banks E, Poplin R, Garimella KV, Maguire JR, Hartl C, et al.  A framework for variation discovery and genotyping using next-generation DNA sequencing data. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:491–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3083463/">PMC free article</a>]</span> [<a href="/pubmed/21478889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+framework+for+variation+discovery+and+genotyping+using+next-generation+DNA+sequencing+data&amp;author=MA+DePristo&amp;author=E+Banks&amp;author=R+Poplin&amp;author=KV+Garimella&amp;author=JR+Maguire&amp;volume=43&amp;publication_year=2011&amp;pages=491-8&amp;pmid=21478889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, et al.  The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. <span><span class="ref-journal">Genome Res. </span>2010;<span class="ref-vol">20</span>:1297–303.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928508/">PMC free article</a>]</span> [<a href="/pubmed/20644199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=The+Genome+Analysis+Toolkit:+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data&amp;author=A+McKenna&amp;author=M+Hanna&amp;author=E+Banks&amp;author=A+Sivachenko&amp;author=K+Cibulskis&amp;volume=20&amp;publication_year=2010&amp;pages=1297-303&amp;pmid=20644199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Van der Auwera GA, Carneiro MO, Hartl C, Poplin R, Del Angel G, Levy-Moonshine A, et al.  From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline. <span><span class="ref-journal">Curr Protoc Bioinformatics. </span>2013;<span class="ref-vol">43</span>:1101–33.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4243306/">PMC free article</a>]</span> [<a href="/pubmed/25431634" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Bioinformatics&amp;title=From+FastQ+data+to+high+confidence+variant+calls:+the+Genome+Analysis+Toolkit+best+practices+pipeline&amp;author=GA+Van+der+Auwera&amp;author=MO+Carneiro&amp;author=C+Hartl&amp;author=R+Poplin&amp;author=G+Del+Angel&amp;volume=43&amp;publication_year=2013&amp;pages=1101-33&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Cingolani P, Platts A, Wang le L, Coon M, Nguyen T, Wang L, et al.  A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. <span><span class="ref-journal">Fly (Austin) </span>2012;<span class="ref-vol">6</span>:80–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3679285/">PMC free article</a>]</span> [<a href="/pubmed/22728672" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fly+(Austin)&amp;title=A+program+for+annotating+and+predicting+the+effects+of+single+nucleotide+polymorphisms,+SnpEff:+SNPs+in+the+genome+of+Drosophila+melanogaster+strain+w1118;+iso-2;+iso-3&amp;author=P+Cingolani&amp;author=A+Platts&amp;author=L+Wang+le&amp;author=M+Coon&amp;author=T+Nguyen&amp;volume=6&amp;publication_year=2012&amp;pages=80-92&amp;pmid=22728672&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Kocher JP, Quest DJ, Duffy P, Meiners MA, Moore RM, Rider D, et al.  The Biological Reference Repository (BioR): a rapid and flexible system for genomics annotation. <span><span class="ref-journal">Bioinformatics. </span>2014;<span class="ref-vol">30</span>:1920–2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4071205/">PMC free article</a>]</span> [<a href="/pubmed/24618464" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The+Biological+Reference+Repository+(BioR):+a+rapid+and+flexible+system+for+genomics+annotation&amp;author=JP+Kocher&amp;author=DJ+Quest&amp;author=P+Duffy&amp;author=MA+Meiners&amp;author=RM+Moore&amp;volume=30&amp;publication_year=2014&amp;pages=1920-2&amp;pmid=24618464&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Altman DG, McShane LM, Sauerbrei W, Taube SE. Reporting Recommendations for Tumor Marker Prognostic Studies (REMARK): explanation and elaboration. <span><span class="ref-journal">PLoS Med. </span>2012;<span class="ref-vol">9</span>:e1001216.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3362085/">PMC free article</a>]</span> [<a href="/pubmed/22675273" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med&amp;title=Reporting+Recommendations+for+Tumor+Marker+Prognostic+Studies+(REMARK):+explanation+and+elaboration&amp;author=DG+Altman&amp;author=LM+McShane&amp;author=W+Sauerbrei&amp;author=SE+Taube&amp;volume=9&amp;publication_year=2012&amp;pages=e1001216&amp;pmid=22675273&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Melzer D, Perry JR, Hernandez D, Corsi AM, Stevens K, Rafferty I, et al.  A genome-wide association study identifies protein quantitative trait loci (pQTLs) <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000072.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2362067/">PMC free article</a>]</span> [<a href="/pubmed/18464913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+association+study+identifies+protein+quantitative+trait+loci+(pQTLs)&amp;author=D+Melzer&amp;author=JR+Perry&amp;author=D+Hernandez&amp;author=AM+Corsi&amp;author=K+Stevens&amp;volume=4&amp;publication_year=2008&amp;pages=e1000072&amp;pmid=18464913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Ota T, Suzuki Y, Nishikawa T, Otsuki T, Sugiyama T, Irie R, et al.  Complete sequencing and characterization of 21,243 full-length human cDNAs. <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>:40–5.</span> [<a href="/pubmed/14702039" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Complete+sequencing+and+characterization+of+21,243+full-length+human+cDNAs&amp;author=T+Ota&amp;author=Y+Suzuki&amp;author=T+Nishikawa&amp;author=T+Otsuki&amp;author=T+Sugiyama&amp;volume=36&amp;publication_year=2004&amp;pages=40-5&amp;pmid=14702039&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Ho MF, Bongartz T, Liu M, Kalari KR, Goss PE, Shepherd LE, et al.  Estrogen, SNP-Dependent Chemokine Expression and Selective Estrogen Receptor Modulator Regulation. <span><span class="ref-journal">Mol Endocrinol. </span>2016;<span class="ref-vol">30</span>:382–98.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4771694/">PMC free article</a>]</span> [<a href="/pubmed/26866883" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Endocrinol&amp;title=Estrogen,+SNP-Dependent+Chemokine+Expression+and+Selective+Estrogen+Receptor+Modulator+Regulation&amp;author=MF+Ho&amp;author=T+Bongartz&amp;author=M+Liu&amp;author=KR+Kalari&amp;author=PE+Goss&amp;volume=30&amp;publication_year=2016&amp;pages=382-98&amp;pmid=26866883&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Ingle JN, Liu M, Wickerham DL, Schaid DJ, Wang L, Mushiroda T, et al.  Selective estrogen receptor modulators and pharmacogenomic variation in ZNF423 regulation of BRCA1 expression: individualized breast cancer prevention. <span><span class="ref-journal">Cancer Discov. </span>2013;<span class="ref-vol">3</span>:812–25.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3710533/">PMC free article</a>]</span> [<a href="/pubmed/23764426" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Discov&amp;title=Selective+estrogen+receptor+modulators+and+pharmacogenomic+variation+in+ZNF423+regulation+of+BRCA1+expression:+individualized+breast+cancer+prevention&amp;author=JN+Ingle&amp;author=M+Liu&amp;author=DL+Wickerham&amp;author=DJ+Schaid&amp;author=L+Wang&amp;volume=3&amp;publication_year=2013&amp;pages=812-25&amp;pmid=23764426&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Comprehensive molecular portraits of human breast tumours. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">490</span>:61–70.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3465532/">PMC free article</a>]</span> [<a href="/pubmed/23000897" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive+molecular+portraits+of+human+breast+tumours&amp;volume=490&amp;publication_year=2012&amp;pages=61-70&amp;pmid=23000897&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Li L, Fridley B, Kalari K, Jenkins G, Batzler A, Safgren S, et al.  Gemcitabine and cytosine arabinoside cytotoxicity: association with lymphoblastoid cell expression. <span><span class="ref-journal">Cancer Res. </span>2008;<span class="ref-vol">68</span>:7050–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2562356/">PMC free article</a>]</span> [<a href="/pubmed/18757419" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Gemcitabine+and+cytosine+arabinoside+cytotoxicity:+association+with+lymphoblastoid+cell+expression&amp;author=L+Li&amp;author=B+Fridley&amp;author=K+Kalari&amp;author=G+Jenkins&amp;author=A+Batzler&amp;volume=68&amp;publication_year=2008&amp;pages=7050-8&amp;pmid=18757419&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Macedo LF, Guo Z, Tilghman SL, Sabnis GJ, Qiu Y, Brodie A. Role of androgens on MCF-7 breast cancer cell growth and on the inhibitory effect of letrozole. <span><span class="ref-journal">Cancer Res. </span>2006;<span class="ref-vol">66</span>:7775–82.</span> [<a href="/pubmed/16885381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Role+of+androgens+on+MCF-7+breast+cancer+cell+growth+and+on+the+inhibitory+effect+of+letrozole&amp;author=LF+Macedo&amp;author=Z+Guo&amp;author=SL+Tilghman&amp;author=GJ+Sabnis&amp;author=Y+Qiu&amp;volume=66&amp;publication_year=2006&amp;pages=7775-82&amp;pmid=16885381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Yue W, Zhou D, Chen S, Brodie A. A new nude mouse model for postmenopausal breast cancer using MCF-7 cells transfected with the human aromatase gene. <span><span class="ref-journal">Cancer Res. </span>1994;<span class="ref-vol">54</span>:5092–5.</span> [<a href="/pubmed/7923123" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=A+new+nude+mouse+model+for+postmenopausal+breast+cancer+using+MCF-7+cells+transfected+with+the+human+aromatase+gene&amp;author=W+Yue&amp;author=D+Zhou&amp;author=S+Chen&amp;author=A+Brodie&amp;volume=54&amp;publication_year=1994&amp;pages=5092-5&amp;pmid=7923123&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Giamas G, Filipovic A, Jacob J, Messier W, Zhang H, Yang D, et al.  Kinome screening for regulators of the estrogen receptor identifies LMTK3 as a new therapeutic target in breast cancer. <span><span class="ref-journal">Nat Med. </span>2011;<span class="ref-vol">17</span>:715–9.</span> [<a href="/pubmed/21602804" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Med&amp;title=Kinome+screening+for+regulators+of+the+estrogen+receptor+identifies+LMTK3+as+a+new+therapeutic+target+in+breast+cancer&amp;author=G+Giamas&amp;author=A+Filipovic&amp;author=J+Jacob&amp;author=W+Messier&amp;author=H+Zhang&amp;volume=17&amp;publication_year=2011&amp;pages=715-9&amp;pmid=21602804&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Guo S, Sonenshein GE. Forkhead box transcription factor FOXO3a regulates estrogen receptor alpha expression and is repressed by the Her-2/neu/phosphatidylinositol 3-kinase/Akt signaling pathway. <span><span class="ref-journal">Mol Cell Biol. </span>2004;<span class="ref-vol">24</span>:8681–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC516736/">PMC free article</a>]</span> [<a href="/pubmed/15367686" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Forkhead+box+transcription+factor+FOXO3a+regulates+estrogen+receptor+alpha+expression+and+is+repressed+by+the+Her-2/neu/phosphatidylinositol+3-kinase/Akt+signaling+pathway&amp;author=S+Guo&amp;author=GE+Sonenshein&amp;volume=24&amp;publication_year=2004&amp;pages=8681-90&amp;pmid=15367686&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Zou Y, Tsai WB, Cheng CJ, Hsu C, Chung YM, Li PC, et al.  Forkhead box transcription factor FOXO3a suppresses estrogen-dependent breast cancer cell proliferation and tumorigenesis. <span><span class="ref-journal">Breast Cancer Res. </span>2008;<span class="ref-vol">10</span>:R21.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2374977/">PMC free article</a>]</span> [<a href="/pubmed/18312651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Breast+Cancer+Res&amp;title=Forkhead+box+transcription+factor+FOXO3a+suppresses+estrogen-dependent+breast+cancer+cell+proliferation+and+tumorigenesis&amp;author=Y+Zou&amp;author=WB+Tsai&amp;author=CJ+Cheng&amp;author=C+Hsu&amp;author=YM+Chung&amp;volume=10&amp;publication_year=2008&amp;pages=R21&amp;pmid=18312651&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Morelli C, Lanzino M, Garofalo C, Maris P, Brunelli E, Casaburi I, et al.  Akt2 inhibition enables the forkhead transcription factor FoxO3a to have a repressive role in estrogen receptor alpha transcriptional activity in breast cancer cells. <span><span class="ref-journal">Mol Cell Biol. </span>2010;<span class="ref-vol">30</span>:857–70.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2812247/">PMC free article</a>]</span> [<a href="/pubmed/19933843" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Akt2+inhibition+enables+the+forkhead+transcription+factor+FoxO3a+to+have+a+repressive+role+in+estrogen+receptor+alpha+transcriptional+activity+in+breast+cancer+cells&amp;author=C+Morelli&amp;author=M+Lanzino&amp;author=C+Garofalo&amp;author=P+Maris&amp;author=E+Brunelli&amp;volume=30&amp;publication_year=2010&amp;pages=857-70&amp;pmid=19933843&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Musgrove EA, Sutherland RL. Biological determinants of endocrine resistance in breast cancer. <span><span class="ref-journal">Nat Rev Cancer. </span>2009;<span class="ref-vol">9</span>:631–43.</span> [<a href="/pubmed/19701242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Biological+determinants+of+endocrine+resistance+in+breast+cancer&amp;author=EA+Musgrove&amp;author=RL+Sutherland&amp;volume=9&amp;publication_year=2009&amp;pages=631-43&amp;pmid=19701242&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Osborne CK, Schiff R. Mechanisms of endocrine resistance in breast cancer. <span><span class="ref-journal">Annu Rev Med. </span>2011;<span class="ref-vol">62</span>:233–47.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3656649/">PMC free article</a>]</span> [<a href="/pubmed/20887199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Med&amp;title=Mechanisms+of+endocrine+resistance+in+breast+cancer&amp;author=CK+Osborne&amp;author=R+Schiff&amp;volume=62&amp;publication_year=2011&amp;pages=233-47&amp;pmid=20887199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Ma CX, Reinert T, Chmielewska I, Ellis MJ. Mechanisms of aromatase inhibitor resistance. <span><span class="ref-journal">Nat Rev Cancer. </span>2015;<span class="ref-vol">15</span>:261–75.</span> [<a href="/pubmed/25907219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Mechanisms+of+aromatase+inhibitor+resistance&amp;author=CX+Ma&amp;author=T+Reinert&amp;author=I+Chmielewska&amp;author=MJ+Ellis&amp;volume=15&amp;publication_year=2015&amp;pages=261-75&amp;pmid=25907219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Schroth W, Goetz MP, Hamann U, Fasching PA, Schmidt M, Winter S, et al.  Association between CYP2D6 polymorphisms and outcomes among women with early stage breast cancer treated with tamoxifen. <span><span class="ref-journal">Jama. </span>2009;<span class="ref-vol">302</span>:1429–36.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3909953/">PMC free article</a>]</span> [<a href="/pubmed/19809024" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Jama&amp;title=Association+between+CYP2D6+polymorphisms+and+outcomes+among+women+with+early+stage+breast+cancer+treated+with+tamoxifen&amp;author=W+Schroth&amp;author=MP+Goetz&amp;author=U+Hamann&amp;author=PA+Fasching&amp;author=M+Schmidt&amp;volume=302&amp;publication_year=2009&amp;pages=1429-36&amp;pmid=19809024&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Iyer MK, Niknafs YS, Malik R, Singhal U, Sahu A, Hosono Y, et al.  The landscape of long noncoding RNAs in the human transcriptome. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:199–208.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4417758/">PMC free article</a>]</span> [<a href="/pubmed/25599403" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+landscape+of+long+noncoding+RNAs+in+the+human+transcriptome&amp;author=MK+Iyer&amp;author=YS+Niknafs&amp;author=R+Malik&amp;author=U+Singhal&amp;author=A+Sahu&amp;volume=47&amp;publication_year=2015&amp;pages=199-208&amp;pmid=25599403&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Rinn JL, Chang HY. Genome regulation by long noncoding RNAs. <span><span class="ref-journal">Annu Rev Biochem. </span>2012;<span class="ref-vol">81</span>:145–66.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3858397/">PMC free article</a>]</span> [<a href="/pubmed/22663078" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Biochem&amp;title=Genome+regulation+by+long+noncoding+RNAs&amp;author=JL+Rinn&amp;author=HY+Chang&amp;volume=81&amp;publication_year=2012&amp;pages=145-66&amp;pmid=22663078&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Paci P, Colombo T, Farina L. Computational analysis identifies a sponge interaction network between long non-coding RNAs and messenger RNAs in human breast cancer. <span><span class="ref-journal">BMC Syst Biol. </span>2014;<span class="ref-vol">8</span>:83.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4113672/">PMC free article</a>]</span> [<a href="/pubmed/25033876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Syst+Biol&amp;title=Computational+analysis+identifies+a+sponge+interaction+network+between+long+non-coding+RNAs+and+messenger+RNAs+in+human+breast+cancer&amp;author=P+Paci&amp;author=T+Colombo&amp;author=L+Farina&amp;volume=8&amp;publication_year=2014&amp;pages=83&amp;pmid=25033876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Tsai MC, Spitale RC, Chang HY. Long intergenic noncoding RNAs: new links in cancer progression. <span><span class="ref-journal">Cancer Res. </span>2011;<span class="ref-vol">71</span>:3–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3057914/">PMC free article</a>]</span> [<a href="/pubmed/21199792" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Long+intergenic+noncoding+RNAs:+new+links+in+cancer+progression&amp;author=MC+Tsai&amp;author=RC+Spitale&amp;author=HY+Chang&amp;volume=71&amp;publication_year=2011&amp;pages=3-7&amp;pmid=21199792&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Cheetham SW, Gruhl F, Mattick JS, Dinger ME. Long noncoding RNAs and the genetics of cancer. <span><span class="ref-journal">Br J Cancer. </span>2013;<span class="ref-vol">108</span>:2419–25.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3694235/">PMC free article</a>]</span> [<a href="/pubmed/23660942" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Long+noncoding+RNAs+and+the+genetics+of+cancer&amp;author=SW+Cheetham&amp;author=F+Gruhl&amp;author=JS+Mattick&amp;author=ME+Dinger&amp;volume=108&amp;publication_year=2013&amp;pages=2419-25&amp;pmid=23660942&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Gupta RA, Shah N, Wang KC, Kim J, Horlings HM, Wong DJ, et al.  Long non-coding RNA HOTAIR reprograms chromatin state to promote cancer metastasis. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">464</span>:1071–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3049919/">PMC free article</a>]</span> [<a href="/pubmed/20393566" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Long+non-coding+RNA+HOTAIR+reprograms+chromatin+state+to+promote+cancer+metastasis&amp;author=RA+Gupta&amp;author=N+Shah&amp;author=KC+Wang&amp;author=J+Kim&amp;author=HM+Horlings&amp;volume=464&amp;publication_year=2010&amp;pages=1071-6&amp;pmid=20393566&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Hou P, Zhao Y, Li Z, Yao R, Ma M, Gao Y, et al.  LincRNA-ROR induces epithelial-to-mesenchymal transition and contributes to breast cancer tumorigenesis and metastasis. <span><span class="ref-journal">Cell Death Dis. </span>2014;<span class="ref-vol">5</span>:e1287.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4611722/">PMC free article</a>]</span> [<a href="/pubmed/24922071" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Death+Dis&amp;title=LincRNA-ROR+induces+epithelial-to-mesenchymal+transition+and+contributes+to+breast+cancer+tumorigenesis+and+metastasis&amp;author=P+Hou&amp;author=Y+Zhao&amp;author=Z+Li&amp;author=R+Yao&amp;author=M+Ma&amp;volume=5&amp;publication_year=2014&amp;pages=e1287&amp;pmid=24922071&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Xue X, Yang YA, Zhang A, Fong KW, Kim J, Song B, et al.  LncRNA HOTAIR enhances ER signaling and confers tamoxifen resistance in breast cancer. <span><span class="ref-journal">Oncogene. </span>2016;<span class="ref-vol">35</span>:2746–55.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4791209/">PMC free article</a>]</span> [<a href="/pubmed/26364613" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=LncRNA+HOTAIR+enhances+ER+signaling+and+confers+tamoxifen+resistance+in+breast+cancer&amp;author=X+Xue&amp;author=YA+Yang&amp;author=A+Zhang&amp;author=KW+Fong&amp;author=J+Kim&amp;volume=35&amp;publication_year=2016&amp;pages=2746-55&amp;pmid=26364613&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Tomita S, Abdalla MO, Fujiwara S, Matsumori H, Maehara K, Ohkawa Y, et al.  A cluster of noncoding RNAs activates the ESR1 locus during breast cancer adaptation. <span><span class="ref-journal">Nat Commun. </span>2015;<span class="ref-vol">6</span>:6966.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4421845/">PMC free article</a>]</span> [<a href="/pubmed/25923108" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=A+cluster+of+noncoding+RNAs+activates+the+ESR1+locus+during+breast+cancer+adaptation&amp;author=S+Tomita&amp;author=MO+Abdalla&amp;author=S+Fujiwara&amp;author=H+Matsumori&amp;author=K+Maehara&amp;volume=6&amp;publication_year=2015&amp;pages=6966&amp;pmid=25923108&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Deblois G, Giguere V. Ligand-independent coactivation of ERalpha AF-1 by steroid receptor RNA activator (SRA) via MAPK activation. <span><span class="ref-journal">J Steroid Biochem Mol Biol. </span>2003;<span class="ref-vol">85</span>:123–31.</span> [<a href="/pubmed/12943696" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Steroid+Biochem+Mol+Biol&amp;title=Ligand-independent+coactivation+of+ERalpha+AF-1+by+steroid+receptor+RNA+activator+(SRA)+via+MAPK+activation&amp;author=G+Deblois&amp;author=V+Giguere&amp;volume=85&amp;publication_year=2003&amp;pages=123-31&amp;pmid=12943696&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Hayes EL, Lewis-Wambi JS. Mechanisms of endocrine resistance in breast cancer: an overview of the proposed roles of noncoding RNA. <span><span class="ref-journal">Breast Cancer Res. </span>2015;<span class="ref-vol">17</span>:40.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4362832/">PMC free article</a>]</span> [<a href="/pubmed/25849966" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Breast+Cancer+Res&amp;title=Mechanisms+of+endocrine+resistance+in+breast+cancer:+an+overview+of+the+proposed+roles+of+noncoding+RNA&amp;author=EL+Hayes&amp;author=JS+Lewis-Wambi&amp;volume=17&amp;publication_year=2015&amp;pages=40&amp;pmid=25849966&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Adams BD, Anastasiadou E, Esteller M, He L, Slack FJ. The Inescapable Influence of Noncoding RNAs in Cancer. <span><span class="ref-journal">Cancer Res. </span>2015;<span class="ref-vol">75</span>:5206–10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4823153/">PMC free article</a>]</span> [<a href="/pubmed/26567137" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=The+Inescapable+Influence+of+Noncoding+RNAs+in+Cancer&amp;author=BD+Adams&amp;author=E+Anastasiadou&amp;author=M+Esteller&amp;author=L+He&amp;author=FJ+Slack&amp;volume=75&amp;publication_year=2015&amp;pages=5206-10&amp;pmid=26567137&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Reid G, Denger S, Kos M, Gannon F. Human estrogen receptor-alpha: regulation by synthesis, modification and degradation. <span><span class="ref-journal">Cell Mol Life Sci. </span>2002;<span class="ref-vol">59</span>:821–31.</span> [<a href="/pubmed/12088282" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Mol+Life+Sci&amp;title=Human+estrogen+receptor-alpha:+regulation+by+synthesis,+modification+and+degradation&amp;author=G+Reid&amp;author=S+Denger&amp;author=M+Kos&amp;author=F+Gannon&amp;volume=59&amp;publication_year=2002&amp;pages=821-31&amp;pmid=12088282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5135610&amp;issue-id=280022&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140605670293696title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Methods</a></li><li><a href="#S11title">Results</a></li><li><a href="#S21title">Discussion</a></li><li><a href="#S22title">Supplementary Material</a></li><li><a href="#S23title">Acknowledgments</a></li><li><a href="#idm140605671253696title">Footnotes</a></li><li><a href="#idm140605671252464title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5135610/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5135610/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5135610/pdf/nihms827024.pdf">PDF (1.2M)</a></li> | <li><a href="#" data-citationid="PMC5135610" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5135610%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5135610%2F&amp;text=Genetic%20polymorphisms%20in%20the%20long%20noncoding%20RNA%20MIR2052HG%20offer%20a%20pharmacogenomic%20basis%20for%20the%20response%20of%20breast%20cancer%20patients%20to%20aromatase%20inhibitor%20therapy" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5135610%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27758888" data-aiid="5135610" data-aid="5135610" data-iid="280022" data-domainid="319" data-domain="nihpa" data-accid="PMC5135610" data-md5="6f780197b5344172e9a521875f33be16"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:17:15-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE8817BADD59A1A100000000039A021A&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5135610%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5135610/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>