<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5980656/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association with MRI atrophy measures as a quantitative trait locus for Alzheimer's disease</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5980656/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Molecular psychiatry"><meta name="citation_title" content="Genome-wide association with MRI atrophy measures as a quantitative trait locus for Alzheimer's disease"><meta name="citation_authors" content="SJ Furney, A Simmons, G Breen, I Pedroso, K Lunnon, P Proitsi, A Hodges, J Powell, L-O Wahlund, I Kloszewska, P Mecocci, H Soininen, M Tsolaki, B Vellas, C Spenger, M Lathrop, L Shen, S Kim, AJ Saykin, MW Weiner, S Lovestone"><meta name="citation_date" content="November 2011"><meta name="citation_issue" content="11"><meta name="citation_volume" content="16"><meta name="citation_firstpage" content="1130"><meta name="citation_doi" content="10.1038/mp.2010.123"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5980656/?report=abstract"><meta name="citation_pmid" content="21116278"><meta name="DC.Title" content="Genome-wide association with MRI atrophy measures as a quantitative trait locus for Alzheimer's disease"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="SJ Furney"><meta name="DC.Contributor" content="A Simmons"><meta name="DC.Contributor" content="G Breen"><meta name="DC.Contributor" content="I Pedroso"><meta name="DC.Contributor" content="K Lunnon"><meta name="DC.Contributor" content="P Proitsi"><meta name="DC.Contributor" content="A Hodges"><meta name="DC.Contributor" content="J Powell"><meta name="DC.Contributor" content="L-O Wahlund"><meta name="DC.Contributor" content="I Kloszewska"><meta name="DC.Contributor" content="P Mecocci"><meta name="DC.Contributor" content="H Soininen"><meta name="DC.Contributor" content="M Tsolaki"><meta name="DC.Contributor" content="B Vellas"><meta name="DC.Contributor" content="C Spenger"><meta name="DC.Contributor" content="M Lathrop"><meta name="DC.Contributor" content="L Shen"><meta name="DC.Contributor" content="S Kim"><meta name="DC.Contributor" content="AJ Saykin"><meta name="DC.Contributor" content="MW Weiner"><meta name="DC.Contributor" content="S Lovestone"><meta name="DC.Date" content="2011 Nov"><meta name="DC.Identifier" content="10.1038/mp.2010.123"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association with MRI atrophy measures as a quantitative trait locus for Alzheimer's disease"><meta property="og:type" content="article"><meta property="og:description" content="Alzheimer's disease (AD) is a progressive neurodegenerative disorder with considerable evidence suggesting an initiation of disease in the entorhinal cortex and hippocampus and spreading thereafter to the rest of the brain. In this study, we combine genetics ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5980656/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5980656/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5980656/pdf/nihms968492.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE882381DD51E0710000000001F5015D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5980656%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5980656</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Mol Psychiatry</a></span>. Author manuscript; available in PMC 2018 May 31.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21116278" target="pmc_ext" ref="reftype=publisher&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Mol Psychiatry. 2011 Nov; 16(11): 1130–1138. </span></a></div><span class="fm-vol-iss-date">Published online 2010 Nov 30. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fmp.2010.123" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/mp.2010.123</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5980656</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS968492</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21116278">21116278</a></div></div></div></div><h1 class="content-title">Genome-wide association with MRI atrophy measures as a quantitative trait locus for Alzheimer's disease</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Furney%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621392486048" co-class="co-affbox">SJ Furney</a>,<sup>1</sup> <a href="/pubmed/?term=Simmons%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621396680272" co-class="co-affbox">A Simmons</a>,<sup>1</sup> <a href="/pubmed/?term=Breen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621432759776" co-class="co-affbox">G Breen</a>,<sup>1</sup> <a href="/pubmed/?term=Pedroso%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621432758240" co-class="co-affbox">I Pedroso</a>,<sup>1</sup> <a href="/pubmed/?term=Lunnon%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621392757504" co-class="co-affbox">K Lunnon</a>,<sup>1</sup> <a href="/pubmed/?term=Proitsi%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621392519792" co-class="co-affbox">P Proitsi</a>,<sup>1</sup> <a href="/pubmed/?term=Hodges%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621392517808" co-class="co-affbox">A Hodges</a>,<sup>1</sup> <a href="/pubmed/?term=Powell%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621389931760" co-class="co-affbox">J Powell</a>,<sup>1</sup> <a href="/pubmed/?term=Wahlund%20LO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621397061344" co-class="co-affbox">L-O Wahlund</a>,<sup>2</sup> <a href="/pubmed/?term=Kloszewska%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621391376544" co-class="co-affbox">I Kloszewska</a>,<sup>3</sup> <a href="/pubmed/?term=Mecocci%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621397058480" co-class="co-affbox">P Mecocci</a>,<sup>4</sup> <a href="/pubmed/?term=Soininen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621433526352" co-class="co-affbox">H Soininen</a>,<sup>5</sup> <a href="/pubmed/?term=Tsolaki%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621391881360" co-class="co-affbox">M Tsolaki</a>,<sup>6</sup> <a href="/pubmed/?term=Vellas%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621391879376" co-class="co-affbox">B Vellas</a>,<sup>7</sup> <a href="/pubmed/?term=Spenger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621399329344" co-class="co-affbox">C Spenger</a>,<sup>8</sup> <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621399327360" co-class="co-affbox">M Lathrop</a>,<sup>9</sup> <a href="/pubmed/?term=Shen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621399408704" co-class="co-affbox">L Shen</a>,<sup>10,</sup><sup>11</sup> <a href="/pubmed/?term=Kim%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621395270912" co-class="co-affbox">S Kim</a>,<sup>10,</sup><sup>11</sup> <a href="/pubmed/?term=Saykin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621395379168" co-class="co-affbox">AJ Saykin</a>,<sup>10,</sup><sup>12</sup> <a href="/pubmed/?term=Weiner%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621399307952" co-class="co-affbox">MW Weiner</a>,<sup>13</sup> and  <a href="/pubmed/?term=Lovestone%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278" class="affpopup" co-rid="_co_idm139621399305968" co-class="co-affbox">S Lovestone</a><sup>1</sup>, on behalf of the Alzheimer's Disease Neuroimaging Initiative and the AddNeuroMed Consortium</div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139621392486048"><h3 class="no_margin">SJ Furney</h3><p>  <sup>1</sup>National Institute for Health Research (NIHR) Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Furney%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">SJ Furney</a></div></div><div id="_co_idm139621396680272"><h3 class="no_margin">A Simmons</h3><p>  <sup>1</sup>National Institute for Health Research (NIHR) Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Simmons%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">A Simmons</a></div></div><div id="_co_idm139621432759776"><h3 class="no_margin">G Breen</h3><p>  <sup>1</sup>National Institute for Health Research (NIHR) Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Breen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">G Breen</a></div></div><div id="_co_idm139621432758240"><h3 class="no_margin">I Pedroso</h3><p>  <sup>1</sup>National Institute for Health Research (NIHR) Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Pedroso%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">I Pedroso</a></div></div><div id="_co_idm139621392757504"><h3 class="no_margin">K Lunnon</h3><p>  <sup>1</sup>National Institute for Health Research (NIHR) Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Lunnon%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">K Lunnon</a></div></div><div id="_co_idm139621392519792"><h3 class="no_margin">P Proitsi</h3><p>  <sup>1</sup>National Institute for Health Research (NIHR) Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Proitsi%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">P Proitsi</a></div></div><div id="_co_idm139621392517808"><h3 class="no_margin">A Hodges</h3><p>  <sup>1</sup>National Institute for Health Research (NIHR) Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Hodges%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">A Hodges</a></div></div><div id="_co_idm139621389931760"><h3 class="no_margin">J Powell</h3><p>  <sup>1</sup>National Institute for Health Research (NIHR) Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Powell%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">J Powell</a></div></div><div id="_co_idm139621397061344"><h3 class="no_margin">L-O Wahlund</h3><p>  <sup>2</sup>Department of Neurobiology, Care Sciences and Society, Section of Clinical Geriatrics, Karolinska Institutet, Karolinska University Hospital, Huddinge, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Wahlund%20LO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">L-O Wahlund</a></div></div><div id="_co_idm139621391376544"><h3 class="no_margin">I Kloszewska</h3><p>  <sup>3</sup>Department of Old Age Psychiatry and Psychotic Disorders, Medical University of Lodz, Lodz, Poland</p><div>Find articles by <a href="/pubmed/?term=Kloszewska%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">I Kloszewska</a></div></div><div id="_co_idm139621397058480"><h3 class="no_margin">P Mecocci</h3><p>  <sup>4</sup>Department of Clinical and Experimental Medicine, Section of Gerontology and Geriatrics, University of Perugia, Perugia, Ital</p><div>Find articles by <a href="/pubmed/?term=Mecocci%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">P Mecocci</a></div></div><div id="_co_idm139621433526352"><h3 class="no_margin">H Soininen</h3><p>  <sup>5</sup>Department of Neurology, Kuopio University and University Hospital, Kuopio, Finland</p><div>Find articles by <a href="/pubmed/?term=Soininen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">H Soininen</a></div></div><div id="_co_idm139621391881360"><h3 class="no_margin">M Tsolaki</h3><p>  <sup>6</sup>Third Department of Neurology, Aristotle University of Thessaloniki, Thessaloniki, Greece</p><div>Find articles by <a href="/pubmed/?term=Tsolaki%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">M Tsolaki</a></div></div><div id="_co_idm139621391879376"><h3 class="no_margin">B Vellas</h3><p>  <sup>7</sup>Department of Internal and Geriatrics Medicine, Hôpitaux de Toulouse, Toulouse, France</p><div>Find articles by <a href="/pubmed/?term=Vellas%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">B Vellas</a></div></div><div id="_co_idm139621399329344"><h3 class="no_margin">C Spenger</h3><p>  <sup>8</sup>Department of Clinical Science, Intervention and Technology, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Spenger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">C Spenger</a></div></div><div id="_co_idm139621399327360"><h3 class="no_margin">M Lathrop</h3><p>  <sup>9</sup>Centre National de Genotypage, Institut Genomique, Commissariat à l'Énergie Atomique, Evry, France</p><div>Find articles by <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">M Lathrop</a></div></div><div id="_co_idm139621399408704"><h3 class="no_margin">L Shen</h3><p>  <sup>10</sup>Department of Radiology and Imaging Sciences, Center for Neuroimaging, Indiana University School of Medicine, Indianapolis, IN, USA</p><p>  <sup>11</sup>Center for Computational Biology and Bioinformatics, Indiana University School of Medicine, Indianapolis, IN, USA</p><div>Find articles by <a href="/pubmed/?term=Shen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">L Shen</a></div></div><div id="_co_idm139621395270912"><h3 class="no_margin">S Kim</h3><p>  <sup>10</sup>Department of Radiology and Imaging Sciences, Center for Neuroimaging, Indiana University School of Medicine, Indianapolis, IN, USA</p><p>  <sup>11</sup>Center for Computational Biology and Bioinformatics, Indiana University School of Medicine, Indianapolis, IN, USA</p><div>Find articles by <a href="/pubmed/?term=Kim%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">S Kim</a></div></div><div id="_co_idm139621395379168"><h3 class="no_margin">AJ Saykin</h3><p>  <sup>10</sup>Department of Radiology and Imaging Sciences, Center for Neuroimaging, Indiana University School of Medicine, Indianapolis, IN, USA</p><p>  <sup>12</sup>Department of Medical and Molecular Genetics, Indiana University School of Medicine, Indianapolis, IN, USA</p><div>Find articles by <a href="/pubmed/?term=Saykin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">AJ Saykin</a></div></div><div id="_co_idm139621399307952"><h3 class="no_margin">MW Weiner</h3><p>  <sup>13</sup>Departments of Radiology, Medicine and Psychiatry, University of California San Francisco, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Weiner%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">MW Weiner</a></div></div><div id="_co_idm139621399305968"><h3 class="no_margin">S Lovestone</h3><p>  <sup>1</sup>National Institute for Health Research (NIHR) Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Lovestone%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21116278">S Lovestone</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139621470022304_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139621470022304_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139621470022304_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">  <sup>1</sup>National Institute for Health Research (NIHR) Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London, UK</div><div class="fm-affl" id="A2" lang="en">  <sup>2</sup>Department of Neurobiology, Care Sciences and Society, Section of Clinical Geriatrics, Karolinska Institutet, Karolinska University Hospital, Huddinge, Stockholm, Sweden</div><div class="fm-affl" id="A3" lang="en">  <sup>3</sup>Department of Old Age Psychiatry and Psychotic Disorders, Medical University of Lodz, Lodz, Poland</div><div class="fm-affl" id="A4" lang="en">  <sup>4</sup>Department of Clinical and Experimental Medicine, Section of Gerontology and Geriatrics, University of Perugia, Perugia, Ital</div><div class="fm-affl" id="A5" lang="en">  <sup>5</sup>Department of Neurology, Kuopio University and University Hospital, Kuopio, Finland</div><div class="fm-affl" id="A6" lang="en">  <sup>6</sup>Third Department of Neurology, Aristotle University of Thessaloniki, Thessaloniki, Greece</div><div class="fm-affl" id="A7" lang="en">  <sup>7</sup>Department of Internal and Geriatrics Medicine, Hôpitaux de Toulouse, Toulouse, France</div><div class="fm-affl" id="A8" lang="en">  <sup>8</sup>Department of Clinical Science, Intervention and Technology, Karolinska Institutet, Stockholm, Sweden</div><div class="fm-affl" id="A9" lang="en">  <sup>9</sup>Centre National de Genotypage, Institut Genomique, Commissariat à l'Énergie Atomique, Evry, France</div><div class="fm-affl" id="A10" lang="en">  <sup>10</sup>Department of Radiology and Imaging Sciences, Center for Neuroimaging, Indiana University School of Medicine, Indianapolis, IN, USA</div><div class="fm-affl" id="A11" lang="en">  <sup>11</sup>Center for Computational Biology and Bioinformatics, Indiana University School of Medicine, Indianapolis, IN, USA</div><div class="fm-affl" id="A12" lang="en">  <sup>12</sup>Department of Medical and Molecular Genetics, Indiana University School of Medicine, Indianapolis, IN, USA</div><div class="fm-affl" id="A13" lang="en">  <sup>13</sup>Departments of Radiology, Medicine and Psychiatry, University of California San Francisco, San Francisco, CA, USA</div><div id="FN1">Correspondence: Dr S Lovestone, National Institute for Health Research (NIHR), Biomedical Research Centre for Mental Health, Institute of Psychiatry, King's College London, London SE5 8AF, UK. <a href="mailto:dev@null" data-email="ku.ca.lck.poi@enotsevoL.nomiS" class="oemail">ku.ca.lck.poi@enotsevoL.nomiS</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139621470022304_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21116278" target="pmc_ext" ref="reftype=publisher&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Mol Psychiatry</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5980656/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemental Materials.</strong></div><div><a href="/pmc/articles/PMC5980656/bin/NIHMS968492-supplement-Supplemental_Materials.pdf" data-ga-action="click_feat_suppl">NIHMS968492-supplement-Supplemental_Materials.pdf</a><span style="color:gray"> (736K)</span></div><div class="small guid">GUID:&nbsp;DF2DD0C5-F921-47EC-8498-9D012B31AAAD</div></div></dd></dl></div><div id="idm139621435093280" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139621435093280title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Alzheimer's disease (AD) is a progressive neurodegenerative disorder with considerable evidence suggesting an initiation of disease in the entorhinal cortex and hippocampus and spreading thereafter to the rest of the brain. In this study, we combine genetics and imaging data obtained from the Alzheimer's Disease Neuroimaging Initiative and the AddNeuroMed study. To identify genetic susceptibility loci for AD, we conducted a genome-wide study of atrophy in regions associated with neurodegeneration in this condition. We identified one single-nucleotide polymorphism (SNP) with a disease-specific effect associated with entorhinal cortical volume in an intron of the <em>ZNF292</em> gene (rs1925690; P-value = 2.6× 10<sup>−8</sup>; corrected P-value for equivalent number of independent quantitative traits = 7.7 × 10<sup>−8</sup>) and an intergenic SNP, flanking the <em>ARPP-21</em> gene, with an overall effect on entorhinal cortical thickness (rs11129640; P-value = 5.6 × 10<sup>−8</sup>; corrected P-value = 1.7 × 10<sup>−7</sup>). Gene-wide scoring also highlighted <em>PICALM</em> as the most significant gene associated with entorhinal cortical thickness (P-value = 6.7 × 10<sup>−6</sup>).</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Alzheimer's disease, imaging-genetics, quantitative trait</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">The US NIH Alzheimer's Disease Neuroimaging Initiative (ADNI)<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264482">1</a></sup> and the European Union FP6-funded AddNeuroMed study<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264435">2</a></sup> have complementary goals. ADNI is a 5-year public–private partnership investigating the effectiveness of integrating neuroimaging, genetic/biological markers, as well as clinical and neuropsychological assessments in measuring the progression of mild cognitive impairment (MCI) and early Alzheimer's disease (AD). The main aim of the AddNeuroMed study, also a private–public partnership and a pilot for the European FP6 Innovative Medicines Initiative, is to find biomarkers for AD, in particular for early diagnosis of the disease. Both projects have acquired similar data, and in this study, we integrate neuroimaging and genetic data obtained from the two projects to identify novel susceptibility loci for AD.</p><p id="P3">Research to identify genetic factors that predispose to AD recently resulted in the publication of two large-scale genome-wide association studies (GWASs). Harold et al.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264471">3</a></sup> analyzed &gt;16000 individuals in a two-stage case–control analysis, identifying, in addition to <em>APOE</em>, two loci that were significantly associated with AD (namely <em>CLU</em> and <em>PICALM</em>). Another large-scale study by Lambert <em>et al</em>.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264459">4</a></sup> also found association with the <em>CLU</em> locus and additionally with the <em>CR1</em> locus. These findings have elucidated some of the genes that are causally implicated in predisposition to AD. However, the diseased case versus normal control design of these GWASs and the studies that preceded them have an inherent confound in that a proportion of the apparently normal control population will already harbor disease because of the long clinically silent prodromal phase. One way to address this confound is the use of endophenotypes of disease—that is, markers of pathology <em>in vivo</em> that can be used to separate diseased from normal individuals more accurately than can be achieved by clinical assessment alone, and hence have the potential for greater power than studies using the standard AD/normal case–control design. One approach to endophenotypes as a marker of disease in genetic studies is the use of imaging measures in a quantitative trait (QT) analysis.</p><p id="P4">This imaging-genetics approach has been undertaken by a number of previous studies in AD. Potkin <em>et al</em>.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264468">5</a></sup> identified a number of loci potentially associated with hippocampal atrophy in a subset of the ADNI cohort (AD and control samples). Shen <em>et al</em>.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264467">6</a></sup> combined whole-genome genotyping data with 142 measures of gray matter density, volume and cortical thickness in the full ADNI cohort. However, they focused on the main effects of each single-nucleotide polymorphism (SNP) and did not investigate disease-specific effects.</p><p id="P5" class="p p-last">In this study, we integrate magnetic resonance imaging (MRI) measures and genome-wide common variation data in a combined data set of 939 subjects (236 AD, 424 MCI and 279 controls) from the AddNeuroMed<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264480">2</a></sup> study and the ADNI;<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264473">1</a></sup> both these public–private partnerships having common objectives and, critically for this analysis, common genetics and imaging protocols.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704919850">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264456">8</a></sup> We had two aims: (1) to carry out a GWAS of AD-related brain regions and (2) to investigate the influence of AD-related genes and loci reported by recent large-scale GWASs for AD,<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264474">3</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264487">4</a></sup> specifically, the <em>CLU, PICALM</em> and <em>CR1</em> loci. We focus on brain regions that are unequivocally affected by AD-related neurodegeneration, namely the hippocampus and entorhinal cortex, and also on the related measures whole-brain volume (WBV) and total ventricular volume (VV). We use these MRI-derived measures as QTs to determine disease-specific common variants associated with neural atrophy in patients with AD.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials and methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Samples</h3><p id="P6" class="p p-first">A total of 281 subjects were selected from the AddNeuroMed study, a prospective, longitudinal multicenter study, to discover biomarkers for AD.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264478">2</a></sup> Data were collected from six medical centers across Europe: University of Kuopio (Finland), University of Perugia (Italy), Aristotle University of Thessaloniki (Greece), King's College London (United Kingdom), Medical University of Lodz (Poland) and University of Toulouse (France). Informed consent was obtained for all subjects, and protocols and procedures were approved by the relevant institutional review board at each data acquisition site. Neuropsychological tests were conducted at baseline. None of the MCI and AD subjects had other neurological or psychiatric disease, significant unstable systemic illness or organ failure and alcohol or substance misuse. AD diagnosis was made using the NINCDS-ADRDA (National Institute of Neurological and Communicative Disease and Stroke and Alzheimer's disease) criteria for probable AD.<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264484">9</a></sup> At baseline, all MCI subjects fulfilled the diagnostic criteria for amnestic MCI.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264448">10</a>,<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264481">11</a></sup> A total of 281 patients were genotyped (94 AD, 96 MCI and 91 controls) and underwent MRI.</p><p id="P7" class="p p-last">Data for 818 ADNI subjects were downloaded from the ADNI database (<a href="http://www.loni.ucla.edu/ADNI" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.loni.ucla.edu/ADNI</a>), including baseline 1.5-T MRI scans, Illumina genotyping data (Illumina, San Diego, CA, USA), demographic and neuropsychological information, <em>APOE</em> genotype and baseline diagnosis information (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a> for details).</p></div><div id="S4" class="sec"><h3 id="S4title">Genotyping</h3><p id="P8" class="p p-first">Genotyping for all samples was performed using the Illumina HumanHap610-Quad BeadChips, which contain 620 901 markers. ADNI genotyping was conducted as described previously. <sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264455">6</a></sup> AddNeuroMed genotyping was performed by the CNG (Centre National de Génotypage CNG, France) (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a> for details). The <em>APOE</em> genotype was obtained separately using standard methods.</p><p id="P9" class="p p-last">AddNeuroMed and ADNI genotyping data were merged, and any ambiguous or uncalled SNPs were removed using PLINK.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264439">12</a></sup> SNPs with call rates &lt; 98% or minor allele frequencies &lt;5% were excluded. The Hardy–Weinberg equilibrium was assessed in control samples and SNPs with Hardy–Weinberg equilibrium &lt;10<sup>−5</sup> were excluded. A total of 488 911 SNPs passed quality control protocols. Individual samples with SNP call rates &lt;98% were excluded. Gender and relatedness tests were conducted in PLINK. Individuals with incongruous or ambiguous SNP genders compared with clinical genders were excluded. In cases in which pairs of samples were related (PI-HAT &gt; 0.1875), one of the samples was removed at random. Ethnicity information was available for all participants. To restrict the analysis data set to Caucasian samples, so as to avoid confounding effects due to differing allele frequencies in other populations, the samples were merged with the Hapmap phase 3 populations,<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264440">13</a></sup> and a population stratification analysis was conducted using Eigenstrat.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264434">14</a></sup> Samples classified as non-European were excluded based on ethnicity information and Eigenstrat analysis (AddNeuroMed <em>n</em> =0, ADNI <em>n</em> =64). Overall, 1004 samples passed these quality control filters (AddNeuroMed <em>n</em> = 278, ADNI <em>n</em> =726). The genotyping rate in these samples was 0.998.</p></div><div id="S5" class="sec"><h3 id="S5title">Imaging</h3><div id="S6" class="sec sec-first"><p></p><h4 id="S6title" class="inline">Data acquisition </h4><p id="P10" class="p p-first-last">Data acquisition for AddNeuroMed<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704919851">15</a></sup> and ADNI<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264461">8</a></sup> took place using 1.5-T MR systems using a common protocol, which included a high-resolution sagittal three-dimensional T1-weighted MPRAGE volume (voxel size 1.1 × 1.1 × 1.2 mm<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264475">3</a></sup>) and axial proton density/T2-weighted fast spin echo images. Full brain and skull coverage was required for both of the latter data sets, and a detailed quality control was carried out on all MR images using the AddNeuroMed QC framework.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704919848">15</a></sup> All MR images received a clinical read by a radiologist to exclude any subjects with non-AD-related pathologies.</p></div><div id="S7" class="sec sec-last"><p></p><h4 id="S7title" class="inline">AddNeuroMed and ADNI image analysis </h4><p id="P11" class="p p-first">A highly automated structural MRI image processing pipeline developed by Fischl and Dale<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264445">16</a></sup> and Fischl <em>et al.</em><sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264460">17</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264485">18</a></sup> and producing both regional cortical thickness measures and regional volume measures was used for data analysis. Cortical reconstruction and volumetric segmentation included removal of the nonbrain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (including the hippocampus, amygdala, caudate, putamen, ventricles) intensity normalization, tessellation of the gray matter–white matter boundary, automated topology correction and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class. Surface inflation was followed by registration to a spherical atlas, which used individual cortical folding patterns to match cortical geometry across subjects and parcellation of the cerebral cortex into units based on gyral and sulcal structures. All volumes were normalized by the subject's intracranial volume (ICV).</p><p id="P12" class="p p-last">Regional cortical thickness was measured from 34 areas, and regional cortical volume was measured bilaterally from 24 areas. In total, images obtained from 1097 AddNeuroMed and ADNI samples passed imaging quality control and produced normalized WBVs. Measures for total hippocampal volume (HPV), VV, total entorhinal cortical volume (ERV) and mean (of left and right) entorhinal cortical thickness were extracted for 1070 samples.</p></div></div><div id="S8" class="sec"><h3 id="S8title">QT analysis</h3><p id="P13" class="p p-first">The four normalized (by ICV) volumetric measures WBV, HPV, VV, ERV and the cortical thickness measure mean (of left and right) entorhinal cortical thickness (ERT) were used as QTs in separate regression analyses. ERT was not normalized by ICV. Including ICV as a covariate in the ERT analysis produces very similar results (r of regression coefficients = 0.999), and ICV is not significant in any of the regressions. The numbers of samples that passed both imaging and genotyping quality control protocols and for which there were complete relevant phenotypic and demographic data were 963 (WBV) and 939 (HPV, VV, ERV and ERT).</p><p id="P14">QT analyses were implemented using the linear model function (--linear) in PLINK, which allows covariates to be included in the linear model.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264469">12</a></sup> Each QT was regressed on the number of minor alleles of the 478 081 autosomal SNPs passing quality control criteria, assuming an additive allele effect. A number of different linear models were analyzed:</p><ol class="enumerated" style="list-style-type:decimal" id="L1"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><p id="P15">The additive allele effect was ascertained in an initial model, including age, gender, <em>APOE</em> ε4 allele dosage and disease status as covariates. Disease status was coded as a continuous variable in the model (controls=0, MCI=1, AD=2) to reflect the nonindependent nature of the three diagnosis states. The genomic inflation factor (based on median χ<sup>2</sup> of the additive term in the model) was calculated to investigate potential population stratification effects in the analysis. The genomic inflation factors for WBV, HPV and VV for this model were &gt; 1.05, indicating potential population effects. The ERV and ERT analyses had genomic inflation factors of 1.01.</p></li><li><p id="P16">To allow for population effects, Eigenstrat<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264465">14</a></sup> was run using the 1004 samples that passed quality control. Seven significant eigenvectors were detected using Tracy–Widom statistics. Model 1 was re-run for WBV, HPV and VV, including the seven significant eigenvalues for each sample. The genomic inflation factors for WBV, HPV and VV for this model were 1.00, 1.00 and 1.01, respectively.</p></li><li><p id="P17">To detect SNP effects that may differ according to disease status, an interaction term for disease status and the additive allele effect was included in the third model. For ERV and ERT, the covariates age, gender, <em>APOE</em> ε4 allele dosage and disease status were included in the model. For WBV, HPV and VV, significant eigenvalues were further added.</p></li></ol><p id="P18" class="p p-last">To allow for multiple testing of five QTs (namely WBV, HPV, VV, ERV and ERT) that are correlated (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>), we applied Matrix Spectral Decomposition (see <a href="http://gump.qimr.edu.au/general/daleN/matSpD/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://gump.qimr.edu.au/general/daleN/matSpD/</a>), based on the study by Nyholt,<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264458">19</a></sup> to QTs, which results in an effective number of independent variables of 3, using VeffLi.</p></div><div id="S9" class="sec"><h3 id="S9title">Gene analysis</h3><p id="P19" class="p p-first">We performed linkage disequilibrium (LD)-based SNP grouping in PLINK<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264488">12</a></sup> on the results of our entorhinal cortical thickness analysis using the --clump command with default settings. We also performed a set-based analysis in PLINK, using SNPs ±100 kb of <em>PICALM</em> and ±750 kb of <em>ARPP</em>-21 (because of the intergenic location of the significant associated SNP) as sets of genes and permuting 10000 times. This analysis was implemented by PLINK as follows (from <a href="http://pngu.mgh.harvard.edu/~purcell/plink/anal.shtml#set" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/∼purcell/plink/anal.shtml#set</a>): For the gene set, for each SNP, determine which other SNPs are in LD, above a certain threshold <em>R</em> (0.5). Perform standard single SNP QT analysis. For the set, select up to <em>N</em> ‘independent’ SNPs (as defined before) with <em>P</em>-values below <em>P</em> (0.05). The best SNP is selected first; subsequent SNPs are selected in order of decreasing statistical significance, after removing SNPs in LD with previously selected SNPs. From these subsets of SNPs, the statistic for each set is calculated as the mean of these single SNP statistics. Permute the data set 10 000 times, keeping LD between SNPs constant (that is, permute trait). The empirical <em>P</em>-value for set (EMP1) is the number of times the permuted set statistic exceeds the original one for the set.</p><p id="P20" class="p p-last">Finally, we used a novel method for calculating gene-based <em>P</em>-values from GWA results (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a> for details). We restricted our analysis to ∼20 000 protein-coding genes annotated in Ensembl v56 and mapped them to SNPs if the SNP was within 20 kb of the annotated coordinates.</p></div><div id="S10" class="sec sec-last"><h3 id="S10title">Imputation</h3><p id="P21" class="p p-first-last">To further investigate the evidence of association at the <em>PICALM, ARRP-21</em> and <em>ZNF292</em> loci, we imputed SNPs in the region ±500 kb of <em>PICALM</em> and <em>ZNF292</em> and in ±750 kb of the <em>ARRP-21</em> locus with minor allele frequency &gt;0.01 and imputation quality scores ≥0.99 in the CEU Hapmap phase 2 population<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264477">20</a></sup> and the 1000 Genomes project<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> using Beagle<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264457">22</a></sup> and IMPUTE,<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264443">23</a></sup> respectively, and applied the QT analysis.</p></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Results</h2><div id="S12" class="sec sec-first-last"><h3 id="S12title">QT analysis</h3><p id="P22" class="p p-first">Previous work has shown brain atrophy in patients with AD to be most pronounced in hippocampal and entorhinal cortical regions, as well as in WBV in MRI studies.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704919846">7</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264453">24</a>–<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264451">26</a></sup> The decreases in these areas and the corresponding increase in ventricular volume in the participants in this study are congruent with previous findings (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>). Although there are significant correlations between the neuroimaging traits that we measured (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>), we aimed to identify structure-specific variants associated with disease and markers of general neurodegeneration (<a href="/pmc/articles/PMC5980656/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Mean age, percentage female, mean <em>APOE</em> ε4 allele dosage and total <em>n</em> of control, MCI and AD subjects who passed the genetic and imaging quality control protocols for whole-brain volume (WBV) and all other (Other) quantitative trait analyses</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139621395746336" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="right">Control</th><th rowspan="1" colspan="1" valign="top" align="right">MCI</th><th rowspan="1" colspan="1" valign="top" align="right">AD</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>WBV (N=963)</em></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Age</td><td rowspan="1" colspan="1" valign="top" align="right">74.99</td><td rowspan="1" colspan="1" valign="top" align="right">74.86</td><td rowspan="1" colspan="1" valign="top" align="right">75.38</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Sex (% female)</td><td rowspan="1" colspan="1" valign="top" align="right">49.3</td><td rowspan="1" colspan="1" valign="top" align="right">39.2</td><td rowspan="1" colspan="1" valign="top" align="right">51.8</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> <em>APOE</em> ε4</td><td rowspan="1" colspan="1" valign="top" align="right">0.32</td><td rowspan="1" colspan="1" valign="top" align="right">0.66</td><td rowspan="1" colspan="1" valign="top" align="right">0.82</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> <em>n</em></td><td rowspan="1" colspan="1" valign="top" align="right">284</td><td rowspan="1" colspan="1" valign="top" align="right">434</td><td rowspan="1" colspan="1" valign="top" align="right">245</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>Other (N=939)</em></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Age</td><td rowspan="1" colspan="1" valign="top" align="right">75.04</td><td rowspan="1" colspan="1" valign="top" align="right">74.72</td><td rowspan="1" colspan="1" valign="top" align="right">75.19</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Sex (% female)</td><td rowspan="1" colspan="1" valign="top" align="right">49.5</td><td rowspan="1" colspan="1" valign="top" align="right">39.6</td><td rowspan="1" colspan="1" valign="top" align="right">52.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> <em>APOE</em> ε4</td><td rowspan="1" colspan="1" valign="top" align="right">0.30</td><td rowspan="1" colspan="1" valign="top" align="right">0.66</td><td rowspan="1" colspan="1" valign="top" align="right">0.83</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> <em>n</em></td><td rowspan="1" colspan="1" valign="top" align="right">279</td><td rowspan="1" colspan="1" valign="top" align="right">424</td><td rowspan="1" colspan="1" valign="top" align="right">236</td></tr></tbody></table></div><div id="largeobj_idm139621395746336" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5980656/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p7" class="p p-first-last">Abbreviations: AD, Alzheimer's disease; MCI, mild cognitive impairment.</p></div></div></div><p id="P23">In brief, 1118 subjects were genotyped using the Illumina 610K Beadchip, and standard sample and SNP quality control measures were implemented (see the ‘Materials and methods’ section). Samples classified as non-European were excluded based on ethnicity information and principal components analysis.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264454">14</a></sup> Measures of entorhinal cortical thickness and volume, hippocampal volume, together with total brain and ventricular volumes were derived using automated regional analysis as described previously<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704919847">7</a></sup> (see the ‘Materials and methods’ section). We implemented models with and without the SNP by disease interaction, the latter model allowing us to investigate the main SNP effect in addition to disease-associated SNP effects identified by the interaction term in the former model. Quantile–quantile plots and Manhattan plots of the results are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 2–5</a>.</p><p id="P24">This analysis identified one SNP (<a href="/pmc/articles/PMC5980656/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2a</span></a>) with a disease-specific effect associated with entorhinal cortical volume at the genome-wide significance threshold of <em>P</em>&lt;5 × 10<sup>−8</sup> in an intron of the <em>ZNF292</em> gene (rs1925690; <em>P</em>-value = 2.6 × 10<sup>−8</sup>; corrected <em>P</em>-value for equivalent number of independent QTs = <em>7.7</em> × 10<sup>−8</sup>; <a href="/pmc/articles/PMC5980656/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1a</span></a>) and an intergenic SNP, flanking the <em>ARPP-21</em> gene, with a main effect in entorhinal cortical thickness close to this threshold (rs11129640; <em>P</em>-value = 5.6 × 10<sup>−8</sup>; corrected <em>P</em>-value = 1.7 × 10<sup>−7</sup>; <a href="/pmc/articles/PMC5980656/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1b</span></a>). Neither of these variants has been previously associated with neurodegeneration, nor is either significantly associated with AD in the recent GWASs.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264433">3</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264436">4</a></sup>
<em>ZNF292</em> encodes a putative zinc-finger protein of unknown function. The disease-specific effect of this SNP is shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 6</a>. The <em>ARPP-21</em> gene encodes a neuronal phosphoprotein enriched in regions of the brain receiving dopaminergic innervation.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264462">27</a></sup> In all, 29 SNPs representing 21 different loci have main effects with <em>P</em>-values &lt; 10<sup>−5</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>), and 24 SNPs representing 17 loci have disease-associated effects with <em>P</em>-values &lt; 10<sup>−5</sup> (<a href="/pmc/articles/PMC5980656/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Supplementary Table 3</span></a>). Imputation of variants proximal to the <em>ARPP-21</em> and <em>ZNF292</em> loci (see the ‘Materials and methods’ section) identified a number of additional variants at <em>P</em>-values &lt;10<sup>−4</sup> proximal to rs11129640 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 7</a>), but none in the case of the ZNF292 rs1925690 variant, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 8</a>). LD-based SNP grouping was carried out for the ARPP-21 and ZNF292 loci; however, this analysis did not reveal any further SNPs. A set-based analysis of the <em>ARPP-21</em> locus using SNPs ± 750 kb of <em>ARPP-21</em> as a set of genes and permuting 10 000 times, resulted in an empirical <em>P</em>-value &lt; 10<sup>−4</sup> and revealed 5 SNPs at <em>P</em>&lt;0.05 passing the LD threshold (see the ‘Materials and methods’ section).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5980656/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139621398458656" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms968492f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms968492f1.jpg" src="/pmc/articles/PMC5980656/bin/nihms968492f1.jpg"></div></a><div id="largeobj_idm139621398458656" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5980656/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC5980656/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5980656/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p5">(<strong>a</strong>) Manhattan plot of SNP effect P-values from the disease-specific entorhinal cortical volume analysis. (<strong>b</strong>) Manhattan plot of SNP effect P-values from the main entorhinal cortical thickness analysis. SNP, single-nucleotide polymorphism.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p8">(a) Uncorrected <em>P</em>-values for SNPs that have marginal genome-wide significance; (b) <em>PICALM</em> locus SNPs (<em>P</em>-value &lt; 10<sup>−5</sup>) from Harold <em>et al.</em><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264446">3</a></sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139621435250496" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="8" rowspan="1" valign="top" align="left">(a)</th></tr><tr><th colspan="8" rowspan="1" valign="bottom">
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Chromosome</th><th rowspan="1" colspan="1" valign="top" align="left">Position</th><th rowspan="1" colspan="1" valign="top" align="left">Nearest gene</th><th rowspan="1" colspan="1" valign="top" align="left">Relation to gene</th><th rowspan="1" colspan="1" valign="top" align="center">Beta</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em>-value</th><th rowspan="1" colspan="1" valign="top" align="center">Trait</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11129640</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="left">35117515</td><td rowspan="1" colspan="1" valign="top" align="left">ARPP-21</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">−0.124</td><td rowspan="1" colspan="1" valign="top" align="center">5.57 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">ERT main</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1925690</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">87923782</td><td rowspan="1" colspan="1" valign="top" align="left">ZNF292</td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td><td rowspan="1" colspan="1" valign="top" align="center">2.99 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.56 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">ERV disease</td></tr><tr><td colspan="8" rowspan="1" valign="top" align="left">(b)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="8" rowspan="1" valign="bottom">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SNP</td><td rowspan="1" colspan="1" valign="top" align="center">Chromosome</td><td rowspan="1" colspan="1" valign="top" align="left">Position</td><td rowspan="1" colspan="1" valign="top" align="left">Nearest gene</td><td rowspan="1" colspan="1" valign="top" align="left">Relation to gene</td><td rowspan="1" colspan="1" valign="top" align="center">Beta</td><td rowspan="1" colspan="1" valign="top" align="center">ERT <em>P</em>-value</td><td rowspan="1" colspan="1" valign="top" align="center">GWAS <em>P</em>-value</td></tr><tr><td colspan="8" rowspan="1" valign="bottom">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3851179</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">85546288</td><td rowspan="1" colspan="1" valign="top" align="left">PICALM</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">0.043</td><td rowspan="1" colspan="1" valign="top" align="center">0.034</td><td rowspan="1" colspan="1" valign="top" align="center">1.9 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7941541</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">85536186</td><td rowspan="1" colspan="1" valign="top" align="left">PICALM</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">0.049</td><td rowspan="1" colspan="1" valign="top" align="center">0.019</td><td rowspan="1" colspan="1" valign="top" align="center">2.1 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs543293</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">85497725</td><td rowspan="1" colspan="1" valign="top" align="left">PICALM</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">0.054</td><td rowspan="1" colspan="1" valign="top" align="center">0.009</td><td rowspan="1" colspan="1" valign="top" align="center">6.9 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1237999</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">85492678</td><td rowspan="1" colspan="1" valign="top" align="left">PICALM</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">0.053</td><td rowspan="1" colspan="1" valign="top" align="center">0.009</td><td rowspan="1" colspan="1" valign="top" align="center">1.1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs541458</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">85465999</td><td rowspan="1" colspan="1" valign="top" align="left">PICALM</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">0.052</td><td rowspan="1" colspan="1" valign="top" align="center">0.013</td><td rowspan="1" colspan="1" valign="top" align="center">2.3 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs536841</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">85465472</td><td rowspan="1" colspan="1" valign="top" align="left">PICALM</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">0.053</td><td rowspan="1" colspan="1" valign="top" align="center">0.011</td><td rowspan="1" colspan="1" valign="top" align="center">3.5 × 10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm139621435250496" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5980656/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="__p9" class="p p-first-last">Abbreviations: AD, Alzheimer's disease; ERT, entorhinal cortical thickness; ERV, entorhinal cortical volume; GWAS, genome-wide association study; SNP, single-nucleotide polymorphism.</p></div><div id="TFN3"><p id="__p10" class="p p-first-last">AD GWAS (GWAS <em>P</em>-value) and the uncorrected <em>P</em>-values in our main entorhinal cortex thickness (ERT <em>P</em>-value) quantitative trait analysis.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Main entorhinal cortical thickness analysis gene-based results for genes with local FDR &lt;0.1 (see the ‘Materials and methods’ section for details)</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139621434569552" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Ensembl ID</th><th rowspan="1" colspan="1" valign="top" align="left">Gene symbol</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em>-value</th><th rowspan="1" colspan="1" valign="top" align="center">FDR</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">ENSG00000073921</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PICALM</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.66 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.039</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ENSG00000100266</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PACSIN2</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.81 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.039</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ENSG00000175911</td><td rowspan="1" colspan="1" valign="top" align="left"><em><a id="__tag_704264511" class="tag_hotlink" href="/nuccore/AC127496.2" ref="/nuccore/AC127496.2"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC127496.2","term_id":"22297519","term_text":"AC127496.2"}}</span>AC127496.2</a></em></td><td rowspan="1" colspan="1" valign="top" align="center">1.38 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.050</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ENSG00000196600</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC22A25</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.10 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.059</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ENSG00000146067</td><td rowspan="1" colspan="1" valign="top" align="left"><em><a id="__tag_704264512" class="tag_hotlink" href="/nuccore/AC145098.2" ref="/nuccore/AC145098.2"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC145098.2","term_id":"37999245","term_text":"AC145098.2"}}</span>AC145098.2</a></em></td><td rowspan="1" colspan="1" valign="top" align="center">2.16 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.060</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ENSG00000203724</td><td rowspan="1" colspan="1" valign="top" align="left"><em>C1orf53</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.97 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.068</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ENSG00000113811</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RP11-884K10.1</em></td><td rowspan="1" colspan="1" valign="top" align="center">5.95 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.088</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ENSG00000124588</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NQO2</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.01 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.088</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ENSG00000182747</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC35D3</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.28 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.090</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ENSG00000176108</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CHMP6</em></td><td rowspan="1" colspan="1" valign="top" align="center">7.36 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.095</td></tr></tbody></table></div><div id="largeobj_idm139621434569552" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5980656/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><p id="__p11" class="p p-first-last">Abbreviation: FDR, false discovery rate.</p></div></div></div><p id="P25">In light of recent large-scale GWASs for AD,<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264486">3</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264470">4</a></sup> we then sought to identify any corroboratory evidence in our analyses for the genome-wide significant variants at the <em>CLU, PICALM</em> and <em>CR1</em> loci from these studies. The rs11136000 SNP in clusterin identified by Harold <em>et al</em>.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264483">3</a></sup> and Lambert <em>et al</em>.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264476">4</a></sup> is not significantly associated with any of the imaging QTs, nor is the rs6701713 variant in CR1 identified by Lambert <em>et al</em>.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264444">4</a></sup> The <em>PICALM</em>-associated SNP rs3851179 identified by Harold <em>et al</em>.,<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264442">3</a></sup> the minor allele of which they found to have an odds ratio of 0.86, is associated with an increase in entorhinal cortical thickness (uncorrected <em>P</em>&lt; 0.05; <a href="/pmc/articles/PMC5980656/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2b</span></a>) in our main effect analysis. We then performed LD-based SNP grouping in PLINK<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264431">12</a></sup> on the results of our entorhinal cortical thickness analysis. This highlighted the intronic <em>PICALM</em> SNP rs642949 (uncorrected <em>P</em>-value = 9.6 × 10<sup>−5</sup>) as being in LD with two further SNPs with <em>P</em>-values &lt; 0.001 (<a href="/pmc/articles/PMC5980656/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2a</span></a>). We also performed a set-based analysis in PLINK, using SNPs ± 100 kb of <em>PICALM</em> as a gene set and permuting 10 000 times. Of the 11 SNPs significant at <em>P</em>&lt;0.05, 5 variants were independently significant (based on an <em>r<sup>2</sup></em> threshold of 0.5), resulting in an empirical <em>P</em>-value = 0.0034. To investigate the evidence of association at the <em>PICALM</em> locus further, we imputed SNPs with minor allele frequency &gt;0.01 and imputation quality scores ≥0.99 in the CEU Hapmap phase 2 population<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264432">20</a></sup> and the 1000 Genomes project<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> using Beagle<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264452">22</a></sup> and IMPUTE,<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264447">23</a></sup> respectively, and applied the QT analysis. Although none of the imputed SNPs reaches suggestive significance, there are a further eleven Hapmap phase 2 SNPs and five 1000 genome SNPs at <em>P</em>-values &lt;10<sup>−3</sup> (<a href="/pmc/articles/PMC5980656/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2b</span></a> and c).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5980656/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139621391071536" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms968492f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms968492f2.jpg" src="/pmc/articles/PMC5980656/bin/nihms968492f2.jpg"></div></a><div id="largeobj_idm139621391071536" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC5980656/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC5980656/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5980656/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p6">SNP effect <em>P</em>-values from the main entorhinal cortical thickness analysis in the region surrounding PICALM for (<strong>a</strong>) original results (<strong>b</strong>) HapMap phase 2 SNPs and (<strong>c</strong>) 1000 Genomes Project SNPs. rs3851179 (in blue) is the SNP identified in the Harold et al.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264464">3</a></sup> AD GWAS. <a href="/pmc/articles/PMC5980656/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> was adapted from SNAP<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704919845">34</a></sup> output. AD, Alzheimer's disease; GWAS, genome-wide association study; SNP, single-nucleotide polymorphism. The color reproduction of this figure is available on the html full text version of the manuscript.</p></div></div></div><p id="P26" class="p p-last">As there seemed to be multiple nominally positive SNPs in the region, we used a novel method for calculating gene-based P-values from GWA results (see the ‘Materials and methods’ section). We restricted our analysis to ∼20 000 protein-coding genes annotated in Ensembl v56 and mapped them to genotyped SNPs within 20 kb of the annotated coordinates. We calculated gene-wide P-values using a modified Fisher's method to combine <em>P</em>-values from correlated tests and applied a local false discovery rate. This method identified <em>PICALM</em> as the most significant gene in the main entorhinal cortical thickness analysis (<em>P</em>-value = 6.7 × 10<sup>−6</sup>; false discovery rate = 0.039; <a href="/pmc/articles/PMC5980656/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The <em>ARPP-21</em> gene was not significant in this analysis (false discovery rate = 1), most likely because of the most significant SNP being located outside our threshold of 20 kb. The method was also applied to the disease-specific entorhinal cortical volume analysis, revealing <em>ZNF292</em> as the thirty-fifth most significant gene (false discovery rate = 0.25).</p></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Discussion</h2><p id="P27" class="p p-first">We present a genome-wide QT study for brain atrophy as an endophenotype for AD pathology of &gt;900 participants from the ADNI and AddNeuroMed projects. To our knowledge, this is the largest study integrating imaging measures and genetics in AD. We selected brain regions known to be affected by AD (such as the hippocampus, entorhinal cortex and whole brain) and in which atrophy has been highlighted by previous MRI studies.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704919849">7</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264441">24</a>–<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264479">26</a>,<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264438">28</a></sup> MRI scans were processed to produce volumetric measures of these regions of interest and also entorhinal cortical thicknesses. Genotyping data (∼478K SNPs) were integrated with the imaging traits in QT analyses.</p><p id="P28">We used this imaging QTL approach first for all subjects and then with disease status as an interaction effect because we reasoned that this study design would have greater power to detect disease susceptibility loci than a case versus control design. In the latter, more usual approach, the presence of clinically silent cases in the control group and susceptible but unaffected people presents a substantial confound. Using a measure of active disease as an endophenotype might help to overcome this problem.</p><p id="P29">Our design also includes a large number of MCI subjects, adding to the power of the study. However, many MCI subjects will not progress to AD and a recent meta-analysis by Mitchell <em>et al.</em><sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264437">29</a></sup> showed an annual conversion rate of 8.1% and a cumulative proportion of 33.6% for MCI conversion to dementia, although it is noted that some studies show higher proportions of conversion, and in all studies, some MCI subjects fail to progress only because of death. Although the findings we report in this study will require replication, we note that despite the relatively small sample size compared with GWASs (1000 versus &gt; 10 000) it is striking that we observe main SNP and disease-specific effects that reach nominal genome-wide significance (<a href="/pmc/articles/PMC5980656/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2a</span></a>).</p><p id="P30">The most significant SNP (<em>P</em> = 6.54 × 10<sup>−8</sup>; main ERT analysis) we identify is proximal to the <em>ARPP-21</em> gene, a regulator of calmodulin signaling, that is involved in the cellular cAMP signaling pathway, particularly in striatal medium spiny neurons.<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704919844">30</a></sup> Calmodulin signaling has been widely implicated in AD pathogenesis (see the study by O'Day and Myre<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264450">31</a></sup> for a review). In addition to identifying common variants that are associated with disease susceptibility, we included an interaction term between disease status and the additive SNP effect in the analysis. Implementing this model with our QTs, we identified 24 SNPs correlated with the brain regions at <em>P</em>-values &lt;10<sup>−5</sup> and one SNP &lt;10<sup>−6</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). The most significant SNP (ERV; <em>P</em>-value = 3.45 × 10∼<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264463">8</a></sup>) is in an intron in the <em>ZNF292</em> gene, which codes for a zinc-finger protein expressed in the brain.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264449">32</a></sup> Our disease-specific analysis highlights the association between the minor allele of this SNP (rs1925690) and increased entorhinal cortical volume (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 6</a>) in AD subjects, whereas the converse is evident in control individuals.</p><p id="P31">Our analysis suggests further evidence for the <em>PICALM</em> locus in neurodegeneration. Using a genecentric approach, we identify a number of SNPs contributing to a locus association with entorhinal cortical thickness (<a href="/pmc/articles/PMC5980656/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). The <em>PICALM</em> variant identified by Harold <em>et al.</em><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264430">3</a></sup> and other more significant variants in our study are associated with less atrophy in the entorhinal cortex, suggesting a protective effect. The PICALM protein is involved in clathrin-mediated endocytosis, inhibition of which has recently been shown to decrease levels of amyloid-β in brain interstitial fluid in mice.<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264472">33</a></sup></p><p id="P32">We have focused on a small number of regions known to be affected in AD and have conducted main and disease-specific effects for all SNPs in both the ADNI and the AddNeuroMed cohorts. Our study design was different from the previously published study of the ADNI cohort by Shen <em>et al.</em><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_704264466">6</a></sup> in that we restricted our analysis to regions previously strongly implicated in the etiopathology of AD and reprocessed all neuroimages. This meant that we tested different phenotypes to those of Shen <em>et al</em>. in our combined ADNI and AddNeuroMed cohort. These differences and the design of the study by Shen <em>et al.</em> do not allow a direct comparison of the results, and we would consider that we have conducted an independent analysis of the data.</p><p id="P33" class="p p-last">Our data provide further evidence of a causal role for <em>PICALM</em> in AD, demonstrate the power of endophenotype-genotype association studies and further identify two genes, namely <em>ARPP-21</em> and <em>ZNF292</em>, as putative risk factors for this devastating and common neurodegenerative disease.</p></div><div id="S14" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplemental Materials</h4><div class="sup-box half_rhythm" id="d36e1136"><a href="/pmc/articles/PMC5980656/bin/NIHMS968492-supplement-Supplemental_Materials.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(736K, pdf)</sup></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Acknowledgments</h2><div class="sec"><p id="__p1">AddNeuroMed is funded through the EU FP6 program as part of InnoMed. In addition, we are grateful for additional support from the NIHR Specialist Biomedical Research Centre for Mental Health at the South London and Maudsley NHS Foundation Trust and King's College London, Institute of Psychiatry, London, United Kingdom. Data collection and sharing for this project were funded by the Alzheimer's Disease Neuroimaging Initiative (ADNI) (National Institutes of Health Grant U01 AG024904). ADNI is funded by the National Institute on Aging, the National Institute of Biomedical Imaging and Bioengineering, and through generous contributions from the following: Abbott, AstraZeneca AB, Bayer Schering Pharma AG, Bristol-Myers Squibb, Eisai Global Clinical Development, Elan Corporation, Genentech, GE Healthcare, GlaxoSmithKline, Innogenetics, Johnson and Johnson, Eli Lilly and Co., Medpace Inc., Merck and Co. Inc., Novartis AG, Pfizer Inc., F Hoffman-La Roche, Schering-Plough, Synarc Inc., and Wyeth, as well as nonprofit partners, the Alzheimer's Association and Alzheimer's Drug Discovery Foundation, with participation from the US Food and Drug Administration. Private sector contributions to ADNI are facilitated by the Foundation for the National Institutes of Health (<a href="http://www.fnih.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.fnih.org</a>). The grantee organization is the Northern California Institute for Research and Education, and the study is coordinated by the Alzheimer's Disease Cooperative Study at the University of California, San Diego. ADNI data are disseminated by the Laboratory for NeuroImaging at the University of California, Los Angeles. This research was also supported by NIH grants P30 AG010129, K01 AG030514 and the Dana Foundation. We thank Mike Weale for assistance with the implementation of population stratification and are grateful to Cathryn Lewis, Amanda Myers and Peter Holmans for advice on the analysis. Data used in the preparation of this article were obtained from the Alzheimer's Disease Neuroimaging Initiative (ADNI) database (<a href="http://www.loni.ucla.edu/ADNI" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.loni.ucla.edu/ADNI</a>). As such, the investigators within the ADNI contributed to the design and implementation of ADNI and/or provided data, but did not participate in analysis or writing of this report. ADNI investigators include (complete listing available at <a href="http://www.loni.ucla.edu/ADNI/Collaboration/ADNI_Manuscript_Citations.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.loni.ucla.edu/ADNI/Collaboration/ADNI_Manuscript_Citations.pdf</a>).</p></div></div><div id="idm139621390954912" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139621390954912title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p2" class="p p-first-last"><strong>Conflict of interest</strong>: The authors declare no conflict of interest.</p><p></p><p class="fn sec" id="FN3"></p><p id="__p3" class="p p-first-last"><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a> accompanies the paper on the Molecular Psychiatry website (<a href="http://www.nature.com/mp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/mp</a>)</p><p></p></div></div><div id="idm139621393947552" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139621393947552title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Mueller SG, Weiner MW, Thal LJ, Petersen RC, Jack CR, Jagust W, et al.  Ways toward an early diagnosis in Alzheimer's disease: the Alzheimer's Disease Neuroimaging Initiative (ADNI) <span><span class="ref-journal">Alzheimers Dement. </span>2005;<span class="ref-vol">1</span>:55–66.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1864941/">PMC free article</a>]</span> [<a href="/pubmed/17476317" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alzheimers+Dement&amp;title=Ways+toward+an+early+diagnosis+in+Alzheimer's+disease:+the+Alzheimer's+Disease+Neuroimaging+Initiative+(ADNI)&amp;author=SG+Mueller&amp;author=MW+Weiner&amp;author=LJ+Thal&amp;author=RC+Petersen&amp;author=CR+Jack&amp;volume=1&amp;publication_year=2005&amp;pages=55-66&amp;pmid=17476317&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Lovestone S, Francis P, Strandgaard K. Biomarkers for disease modification trials—the innovative medicines initiative and AddNeuroMed. <span><span class="ref-journal">J Nutr Health Aging. </span>2007;<span class="ref-vol">11</span>:359–361.</span> [<a href="/pubmed/17653500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Nutr+Health+Aging&amp;title=Biomarkers+for+disease+modification+trials—the+innovative+medicines+initiative+and+AddNeuroMed&amp;author=S+Lovestone&amp;author=P+Francis&amp;author=K+Strandgaard&amp;volume=11&amp;publication_year=2007&amp;pages=359-361&amp;pmid=17653500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Harold D, Abraham R, Hollingworth P, Sims R, Gerrish A, Hamshere ML, et al.  Genome-wide association study identifies variants at CLU and PICALM associated with Alzheimer's disease. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1088–1093.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2845877/">PMC free article</a>]</span> [<a href="/pubmed/19734902" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+variants+at+CLU+and+PICALM+associated+with+Alzheimer's+disease&amp;author=D+Harold&amp;author=R+Abraham&amp;author=P+Hollingworth&amp;author=R+Sims&amp;author=A+Gerrish&amp;volume=41&amp;publication_year=2009&amp;pages=1088-1093&amp;pmid=19734902&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Lambert JC, Heath S, Even G, Campion D, Sleegers K, Hiltunen M, et al.  Genome-wide association study identifies variants at CLU and CR1 associated with Alzheimer's disease. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1094–1099.</span> [<a href="/pubmed/19734903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+variants+at+CLU+and+CR1+associated+with+Alzheimer's+disease&amp;author=JC+Lambert&amp;author=S+Heath&amp;author=G+Even&amp;author=D+Campion&amp;author=K+Sleegers&amp;volume=41&amp;publication_year=2009&amp;pages=1094-1099&amp;pmid=19734903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Potkin SG, Guffanti G, Lakatos A, Turner JA, Kruggel F, Fallon JH, et al.  Hippocampal atrophy as a quantitative trait in a genome-wide association study identifying novel susceptibility genes for Alzheimer's disease. <span><span class="ref-journal">PloS One. </span>2009;<span class="ref-vol">4</span>:e6501.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719581/">PMC free article</a>]</span> [<a href="/pubmed/19668339" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+One&amp;title=Hippocampal+atrophy+as+a+quantitative+trait+in+a+genome-wide+association+study+identifying+novel+susceptibility+genes+for+Alzheimer's+disease&amp;author=SG+Potkin&amp;author=G+Guffanti&amp;author=A+Lakatos&amp;author=JA+Turner&amp;author=F+Kruggel&amp;volume=4&amp;publication_year=2009&amp;pages=e6501&amp;pmid=19668339&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Shen L, Kim S, Risacher SL, Nho K, Swaminathan S, West JD, et al.  Whole Genome Association Study of Brain-Wide Imaging Phenotypes for Identifying Quantitative Trait Loci in MCI and AD: a Study of the ADNI Cohort. <span><span class="ref-journal">NeuroImage. </span>2010;<span class="ref-vol">53</span>:1051–1063.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2892122/">PMC free article</a>]</span> [<a href="/pubmed/20100581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=NeuroImage&amp;title=Whole+Genome+Association+Study+of+Brain-Wide+Imaging+Phenotypes+for+Identifying+Quantitative+Trait+Loci+in+MCI+and+AD:+a+Study+of+the+ADNI+Cohort&amp;author=L+Shen&amp;author=S+Kim&amp;author=SL+Risacher&amp;author=K+Nho&amp;author=S+Swaminathan&amp;volume=53&amp;publication_year=2010&amp;pages=1051-1063&amp;pmid=20100581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Simmons A, Westman E, Muehlboeck S, Mecocci P, Vellas B, Tsolaki M, et al.  MRI measures of Alzheimer's disease and the AddNeuroMed study. <span><span class="ref-journal">Ann NYAcad Sci. </span>2009;<span class="ref-vol">1180</span>:47–55.</span> [<a href="/pubmed/19906260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+NYAcad+Sci&amp;title=MRI+measures+of+Alzheimer's+disease+and+the+AddNeuroMed+study&amp;author=A+Simmons&amp;author=E+Westman&amp;author=S+Muehlboeck&amp;author=P+Mecocci&amp;author=B+Vellas&amp;volume=1180&amp;publication_year=2009&amp;pages=47-55&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Jack CR, Jr, Bernstein MA, Fox NC, Thompson P, Alexander G, Harvey D, et al.  The Alzheimer's Disease Neuroimaging Initiative (ADNI): MRI methods. <span><span class="ref-journal">J Magn Reson Imaging. </span>2008;<span class="ref-vol">27</span>:685–691.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2544629/">PMC free article</a>]</span> [<a href="/pubmed/18302232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Magn+Reson+Imaging&amp;title=The+Alzheimer's+Disease+Neuroimaging+Initiative+(ADNI):+MRI+methods&amp;author=CR+Jack&amp;author=MA+Bernstein&amp;author=NC+Fox&amp;author=P+Thompson&amp;author=G+Alexander&amp;volume=27&amp;publication_year=2008&amp;pages=685-691&amp;pmid=18302232&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">McKhann G, Drachman D, Folstein M, Katzman R, Price D, Stadlan EM. Clinical diagnosis of Alzheimer's disease: Report of the NINCDS-ADRDA Work Group under the Auspices of Department of Health and Human Services Task Force on Alzheimer's Disease. <span><span class="ref-journal">Neurology. </span>1984;<span class="ref-vol">34</span>:939–944.</span> [<a href="/pubmed/6610841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Clinical+diagnosis+of+Alzheimer's+disease:+Report+of+the+NINCDS-ADRDA+Work+Group+under+the+Auspices+of+Department+of+Health+and+Human+Services+Task+Force+on+Alzheimer's+Disease&amp;author=G+McKhann&amp;author=D+Drachman&amp;author=M+Folstein&amp;author=R+Katzman&amp;author=D+Price&amp;volume=34&amp;publication_year=1984&amp;pages=939-944&amp;pmid=6610841&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Petersen RC, Smith GE, Waring SC, Ivnik RJ, Tangalos EG, Kokmen E. Mild cognitive impairment: clinical characterization and outcome. <span><span class="ref-journal">Arch Neurol. </span>1999;<span class="ref-vol">56</span>:303–308.</span> [<a href="/pubmed/10190820" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Neurol&amp;title=Mild+cognitive+impairment:+clinical+characterization+and+outcome&amp;author=RC+Petersen&amp;author=GE+Smith&amp;author=SC+Waring&amp;author=RJ+Ivnik&amp;author=EG+Tangalos&amp;volume=56&amp;publication_year=1999&amp;pages=303-308&amp;pmid=10190820&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Petersen RC, Doody R, Kurz A, Mohs RC, Morris JC, Rabins PV, et al.  Current concepts in mild cognitive impairment. <span><span class="ref-journal">Arch Neurol. </span>2001;<span class="ref-vol">58</span>:1985–1992.</span> [<a href="/pubmed/11735772" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Neurol&amp;title=Current+concepts+in+mild+cognitive+impairment&amp;author=RC+Petersen&amp;author=R+Doody&amp;author=A+Kurz&amp;author=RC+Mohs&amp;author=JC+Morris&amp;volume=58&amp;publication_year=2001&amp;pages=1985-1992&amp;pmid=11735772&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Human Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Human+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">International HapMap Consortium. The International HapMap Project. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">426</span>:789–796.</span> [<a href="/pubmed/14685227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+International+HapMap+Project&amp;volume=426&amp;publication_year=2003&amp;pages=789-796&amp;pmid=14685227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Simmons A, Westman E, Muehlboeck S, Mecocci P, Vellas B, Tsolaki M, et al.  The AddNeuroMed framework for multi-centre MRI assessment of longitudinal changes in Alzheimer's disease: experience from the first 24 months. <span><span class="ref-journal">Int J Ger Psych. </span></span> in press. [<a href="/pubmed/21157852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Ger+Psych&amp;title=The+AddNeuroMed+framework+for+multi-centre+MRI+assessment+of+longitudinal+changes+in+Alzheimer's+disease:+experience+from+the+first+24+months&amp;author=A+Simmons&amp;author=E+Westman&amp;author=S+Muehlboeck&amp;author=P+Mecocci&amp;author=B+Vellas&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Fischl B, Dale AM. Measuring the thickness of the human cerebral cortex from magnetic resonance images. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2000;<span class="ref-vol">97</span>:11050–11055.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC27146/">PMC free article</a>]</span> [<a href="/pubmed/10984517" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Measuring+the+thickness+of+the+human+cerebral+cortex+from+magnetic+resonance+images&amp;author=B+Fischl&amp;author=AM+Dale&amp;volume=97&amp;publication_year=2000&amp;pages=11050-11055&amp;pmid=10984517&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Fischl B, Salat DH, Busa E, Albert M, Dieterich M, Haselgrove C, et al.  Whole brain segmentation: automated labeling of neuroanatomical structures in the human brain. <span><span class="ref-journal">Neuron. </span>2002;<span class="ref-vol">33</span>:341–355.</span> [<a href="/pubmed/11832223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Whole+brain+segmentation:+automated+labeling+of+neuroanatomical+structures+in+the+human+brain&amp;author=B+Fischl&amp;author=DH+Salat&amp;author=E+Busa&amp;author=M+Albert&amp;author=M+Dieterich&amp;volume=33&amp;publication_year=2002&amp;pages=341-355&amp;pmid=11832223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Fischl B, Salat DH, van der Kouwe AJ, Makris N, Segonne F, Quinn BT, et al.  Sequence-independent segmentation of magnetic resonance images. <span><span class="ref-journal">NeuroImage. </span>2004;<span class="ref-vol">23</span>(Suppl 1):S69–S84.</span> [<a href="/pubmed/15501102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=NeuroImage&amp;title=Sequence-independent+segmentation+of+magnetic+resonance+images&amp;author=B+Fischl&amp;author=DH+Salat&amp;author=AJ+van+der+Kouwe&amp;author=N+Makris&amp;author=F+Segonne&amp;volume=23&amp;issue=Suppl+1&amp;publication_year=2004&amp;pages=S69-S84&amp;pmid=15501102&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Nyholt DR. A simple correction for multiple testing for single-nucleotide polymorphisms in linkage disequilibrium with each other. <span><span class="ref-journal">Am J Human Genet. </span>2004;<span class="ref-vol">74</span>:765–769.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1181954/">PMC free article</a>]</span> [<a href="/pubmed/14997420" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Human+Genet&amp;title=A+simple+correction+for+multiple+testing+for+single-nucleotide+polymorphisms+in+linkage+disequilibrium+with+each+other&amp;author=DR+Nyholt&amp;volume=74&amp;publication_year=2004&amp;pages=765-769&amp;pmid=14997420&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Frazer KA, Ballinger DG, Cox DR, Hinds DA, Stuve LL, Gibbs RA, et al.  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=KA+Frazer&amp;author=DG+Ballinger&amp;author=DR+Cox&amp;author=DA+Hinds&amp;author=LL+Stuve&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span>The 1000 Genomes Project.  <a href="http://www1000genomesorg/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www1000genomesorg/</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Browning SR, Browning BL. Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering. <span><span class="ref-journal">Am J Human Genet. </span>2007;<span class="ref-vol">81</span>:1084–1097.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265661/">PMC free article</a>]</span> [<a href="/pubmed/17924348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Human+Genet&amp;title=Rapid+and+accurate+haplotype+phasing+and+missing-data+inference+for+whole-genome+association+studies+by+use+of+localized+haplotype+clustering&amp;author=SR+Browning&amp;author=BL+Browning&amp;volume=81&amp;publication_year=2007&amp;pages=1084-1097&amp;pmid=17924348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Kesslak JP, Nalcioglu O, Cotman CW. Quantification of magnetic resonance scans for hippocampal and parahippocampal atrophy in Alzheimer's disease. <span><span class="ref-journal">Neurology. </span>1991;<span class="ref-vol">41</span>:51–54.</span> [<a href="/pubmed/1985296" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Quantification+of+magnetic+resonance+scans+for+hippocampal+and+parahippocampal+atrophy+in+Alzheimer's+disease&amp;author=JP+Kesslak&amp;author=O+Nalcioglu&amp;author=CW+Cotman&amp;volume=41&amp;publication_year=1991&amp;pages=51-54&amp;pmid=1985296&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Convit A, De Leon MJ, Tarshish C, De Santi S, Tsui W, Rusinek H, et al.  Specific hippocampal volume reductions in individuals at risk for Alzheimer's disease. <span><span class="ref-journal">Neurobiol Aging. </span>1997;<span class="ref-vol">18</span>:131–138.</span> [<a href="/pubmed/9258889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol+Aging&amp;title=Specific+hippocampal+volume+reductions+in+individuals+at+risk+for+Alzheimer's+disease&amp;author=A+Convit&amp;author=MJ+De+Leon&amp;author=C+Tarshish&amp;author=S+De+Santi&amp;author=W+Tsui&amp;volume=18&amp;publication_year=1997&amp;pages=131-138&amp;pmid=9258889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Jack CR, Jr, Petersen RC, Xu Y, O'Brien PC, Smith GE, Ivnik RJ, et al.  Rate of medial temporal lobe atrophy in typical aging and Alzheimer's disease. <span><span class="ref-journal">Neurology. </span>1998;<span class="ref-vol">51</span>:993–999.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2768817/">PMC free article</a>]</span> [<a href="/pubmed/9781519" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Rate+of+medial+temporal+lobe+atrophy+in+typical+aging+and+Alzheimer's+disease&amp;author=CR+Jack&amp;author=RC+Petersen&amp;author=Y+Xu&amp;author=PC+O'Brien&amp;author=GE+Smith&amp;volume=51&amp;publication_year=1998&amp;pages=993-999&amp;pmid=9781519&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Hemmings HC, Jr, Girault JA, Williams KR, LoPresti MB, Greengard P. ARPP- 21 a cyclic AMP-regulated phosphoprotein (Mr = 21 000) enriched in dopamine-innervated brain regions. <span><span class="ref-journal">Amino acid sequence of the site phosphorylated by cyclic AMP in intact cells and kinetic studies of its phosphorylation in vitro J Biol Chem. </span>1989;<span class="ref-vol">264</span>:7726–7733.</span> [<a href="/pubmed/2540203" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Amino+acid+sequence+of+the+site+phosphorylated+by+cyclic+AMP+in+intact+cells+and+kinetic+studies+of+its+phosphorylation+in+vitro+J+Biol+Chem&amp;title=ARPP-+21+a+cyclic+AMP-regulated+phosphoprotein+(Mr+=+21+000)+enriched+in+dopamine-innervated+brain+regions&amp;author=HC+Hemmings&amp;author=JA+Girault&amp;author=KR+Williams&amp;author=MB+LoPresti&amp;author=P+Greengard&amp;volume=264&amp;publication_year=1989&amp;pages=7726-7733&amp;pmid=2540203&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Risacher SL, Saykin AJ, West JD, Shen L, Firpi HA, McDonald BC. Baseline MRI predictors of conversion from MCI to probable AD in the ADNI cohort. <span><span class="ref-journal">Curr Alzheimer Res. </span>2009;<span class="ref-vol">6</span>:347–361.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2764863/">PMC free article</a>]</span> [<a href="/pubmed/19689234" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Alzheimer+Res&amp;title=Baseline+MRI+predictors+of+conversion+from+MCI+to+probable+AD+in+the+ADNI+cohort&amp;author=SL+Risacher&amp;author=AJ+Saykin&amp;author=JD+West&amp;author=L+Shen&amp;author=HA+Firpi&amp;volume=6&amp;publication_year=2009&amp;pages=347-361&amp;pmid=19689234&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Mitchell AJ, Shiri-Feshki M. Rate of progression of mild cognitive impairment to dementia—meta-analysis of 41 robust inception cohort studies. <span><span class="ref-journal">Acta Psychiatr Scand. </span>2009;<span class="ref-vol">119</span>:252–265.</span> [<a href="/pubmed/19236314" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Psychiatr+Scand&amp;title=Rate+of+progression+of+mild+cognitive+impairment+to+dementia—meta-analysis+of+41+robust+inception+cohort+studies&amp;author=AJ+Mitchell&amp;author=M+Shiri-Feshki&amp;volume=119&amp;publication_year=2009&amp;pages=252-265&amp;pmid=19236314&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Rakhilin SV, Olson PA, Nishi A, Starkova NN, Fienberg AA, Nairn AC, et al.  A network of control mediated by regulator of calcium/calmodulin-dependent signaling. <span><span class="ref-journal">Science (New York, NY) </span>2004;<span class="ref-vol">306</span>:698–701.</span> [<a href="/pubmed/15499021" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science+(New+York,+NY)&amp;title=A+network+of+control+mediated+by+regulator+of+calcium/calmodulin-dependent+signaling&amp;author=SV+Rakhilin&amp;author=PA+Olson&amp;author=A+Nishi&amp;author=NN+Starkova&amp;author=AA+Fienberg&amp;volume=306&amp;publication_year=2004&amp;pages=698-701&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">O'Day DH, Myre MA. Calmodulin-binding domains in Alzheimer's disease proteins: extending the calcium hypothesis. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>2004;<span class="ref-vol">320</span>:1051–1054.</span> [<a href="/pubmed/15249195" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=Calmodulin-binding+domains+in+Alzheimer's+disease+proteins:+extending+the+calcium+hypothesis&amp;author=DH+O'Day&amp;author=MA+Myre&amp;volume=320&amp;publication_year=2004&amp;pages=1051-1054&amp;pmid=15249195&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Nagase T, Ishikawa K, Miyajima N, Tanaka A, Kotani H, Nomura N, et al.  Prediction of the coding sequences of unidentified human genes. IX. The complete sequences of 100 new cDNA clones from brain which can code for large proteins <em>in vitro</em>. <span><span class="ref-journal">DNA Res. </span>1998;<span class="ref-vol">5</span>:31–39.</span> [<a href="/pubmed/9628581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Res&amp;title=Prediction+of+the+coding+sequences+of+unidentified+human+genes.+IX.+The+complete+sequences+of+100+new+cDNA+clones+from+brain+which+can+code+for+large+proteins+in+vitro&amp;author=T+Nagase&amp;author=K+Ishikawa&amp;author=N+Miyajima&amp;author=A+Tanaka&amp;author=H+Kotani&amp;volume=5&amp;publication_year=1998&amp;pages=31-39&amp;pmid=9628581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Cirrito JR, Kang JE, Lee J, Stewart FR, Verges DK, Silverio LM, et al.  Endocytosis is required for synaptic activity-dependent release of amyloid-beta <em>in vivo</em>. <span><span class="ref-journal">Neuron. </span>2008;<span class="ref-vol">58</span>:42–51.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2390913/">PMC free article</a>]</span> [<a href="/pubmed/18400162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Endocytosis+is+required+for+synaptic+activity-dependent+release+of+amyloid-beta+in+vivo&amp;author=JR+Cirrito&amp;author=JE+Kang&amp;author=J+Lee&amp;author=FR+Stewart&amp;author=DK+Verges&amp;volume=58&amp;publication_year=2008&amp;pages=42-51&amp;pmid=18400162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Johnson AD, Handsaker RE, Pulit SL, Nizzari MM, O'Donnell CJ, de Bakker PI. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. <span><span class="ref-journal">Bioinformatics (Oxford, England) </span>2008;<span class="ref-vol">24</span>:2938–2939.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720775/">PMC free article</a>]</span> [<a href="/pubmed/18974171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics+(Oxford,+England)&amp;title=SNAP:+a+web-based+tool+for+identification+and+annotation+of+proxy+SNPs+using+HapMap&amp;author=AD+Johnson&amp;author=RE+Handsaker&amp;author=SL+Pulit&amp;author=MM+Nizzari&amp;author=CJ+O'Donnell&amp;volume=24&amp;publication_year=2008&amp;pages=2938-2939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5980656&amp;issue-id=314205&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139621435093280title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials and methods</a></li><li><a href="#S11title">Results</a></li><li><a href="#S13title">Discussion</a></li><li><a href="#S14title">Supplementary Material</a></li><li><a href="#S15title">Acknowledgments</a></li><li><a href="#idm139621390954912title">Footnotes</a></li><li><a href="#idm139621393947552title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5980656/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5980656/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5980656/pdf/nihms968492.pdf">PDF (439K)</a></li> | <li><a href="#" data-citationid="PMC5980656" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5980656%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5980656%2F&amp;text=Genome-wide%20association%20with%20MRI%20atrophy%20measures%20as%20a%20quantitative%20trait%20locus%20for%20Alzheimer's%20disease" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5980656%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21116278" data-aiid="5980656" data-aid="5980656" data-iid="314205" data-domainid="319" data-domain="nihpa" data-accid="PMC5980656" data-md5="729e53d2cf9e71028621376d2e2c6f74"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:19:15-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE882381DD51E0710000000001F5015D&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5980656%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5980656/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>