<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="bloodadv"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5728318/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genetic variants in ADAMTS13 as well as smoking are major determinants of plasma ADAMTS13 levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728318/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Blood Advances"><meta name="citation_title" content="Genetic variants in ADAMTS13 as well as smoking are major determinants of plasma ADAMTS13 levels"><meta name="citation_authors" content="Qianyi Ma, Paula M. Jacobi, Brian T. Emmer, Colin A. Kretz, Ayse Bilge Ozel, Beth McGee, Chava Kimchi-Sarfaty, David Ginsburg, Jun Z. Li, Karl C. Desch"><meta name="citation_date" content="27 June 2017"><meta name="citation_issue" content="15"><meta name="citation_volume" content="1"><meta name="citation_firstpage" content="1037"><meta name="citation_doi" content="10.1182/bloodadvances.2017005629"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5728318/?report=abstract"><meta name="citation_pmid" content="29296746"><meta name="DC.Title" content="Genetic variants in ADAMTS13 as well as smoking are major determinants of plasma ADAMTS13 levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="The American Society of Hematology"><meta name="DC.Contributor" content="Qianyi Ma"><meta name="DC.Contributor" content="Paula M. Jacobi"><meta name="DC.Contributor" content="Brian T. Emmer"><meta name="DC.Contributor" content="Colin A. Kretz"><meta name="DC.Contributor" content="Ayse Bilge Ozel"><meta name="DC.Contributor" content="Beth McGee"><meta name="DC.Contributor" content="Chava Kimchi-Sarfaty"><meta name="DC.Contributor" content="David Ginsburg"><meta name="DC.Contributor" content="Jun Z. Li"><meta name="DC.Contributor" content="Karl C. Desch"><meta name="DC.Date" content="2017 Jun 27"><meta name="DC.Identifier" content="10.1182/bloodadvances.2017005629"><meta name="DC.Language" content="en"><meta property="og:title" content="Genetic variants in ADAMTS13 as well as smoking are major determinants of plasma ADAMTS13 levels"><meta property="og:type" content="article"><meta property="og:description" content="The metalloprotease ADAMTS13 cleaves von Willebrand factor (VWF) in circulating blood, limiting the size of VWF multimers and regulating VWF activity. Abnormal regulation of VWF contributes to bleeding and to thrombotic disorders. ADAMTS13 levels in plasma ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728318/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-bloodadv.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5728318/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5728318/pdf/advances005629.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE884819DD5A93B10000000007950449.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5728318%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/3318/">Blood Adv</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/302983/">v.1(15); 2017 Jun 27</a></li><li class="accid">PMC5728318</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-bloodadv.png" alt="Logo of bloodadv" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,0,500,75" alt="Blood Advances" title="Blood Advances" href="http://www.bloodadvances.org/" ref="reftype=publisher&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Blood Adv</a></span>. 2017 Jun 27; 1(15): 1037–1046. </span></div><div><span class="fm-vol-iss-date">Published online 2017 Jun 19. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1182%2Fbloodadvances.2017005629" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1182/bloodadvances.2017005629</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5728318</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29296746">29296746</a></div></div></div></div><h1 class="content-title">Genetic variants in <em>ADAMTS13</em> as well as smoking are major determinants of plasma ADAMTS13 levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Ma%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746" class="affpopup" co-rid="_co_idm140517551486480" co-class="co-affbox">Qianyi Ma</a>,<sup>1</sup> <a href="/pubmed/?term=Jacobi%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746" class="affpopup" co-rid="_co_idm140517542577600" co-class="co-affbox">Paula M. Jacobi</a>,<sup>2</sup> <a href="/pubmed/?term=Emmer%20BT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746" class="affpopup" co-rid="_co_idm140517552215392" co-class="co-affbox">Brian T. Emmer</a>,<sup>3</sup> <a href="/pubmed/?term=Kretz%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746" class="affpopup" co-rid="_co_idm140517540621600" co-class="co-affbox">Colin A. Kretz</a>,<sup>4</sup> <a href="/pubmed/?term=Ozel%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746" class="affpopup" co-rid="_co_idm140517544382512" co-class="co-affbox">Ayse Bilge Ozel</a>,<sup>1</sup> <a href="/pubmed/?term=McGee%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746" class="affpopup" co-rid="_co_idm140517549612464" co-class="co-affbox">Beth McGee</a>,<sup>5</sup> <a href="/pubmed/?term=Kimchi-Sarfaty%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746" class="affpopup" co-rid="_co_idm140517549610480" co-class="co-affbox">Chava Kimchi-Sarfaty</a>,<sup>6</sup> <a href="/pubmed/?term=Ginsburg%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746" class="affpopup" co-rid="_co_idm140517551404528" co-class="co-affbox">David Ginsburg</a>,<sup>1</sup><sup></sup><sup></sup><sup>4</sup> <a href="/pubmed/?term=Li%20JZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746" class="affpopup" co-rid="_co_idm140517549671536" co-class="co-affbox">Jun Z. Li</a>,<sup>1,</sup><sup>7</sup> and  <a href="/pubmed/?term=Desch%20KC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746" class="affpopup" co-rid="_co_idm140517619361504" co-class="co-affbox">Karl C. Desch</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140517551486480"><h3 class="no_margin">Qianyi Ma</h3><p><sup>1</sup>Department of Human Genetics,</p><div>Find articles by <a href="/pubmed/?term=Ma%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746">Qianyi Ma</a></div></div><div id="_co_idm140517542577600"><h3 class="no_margin">Paula M. Jacobi</h3><p><sup>2</sup>Department of Pediatrics and Communicable Disease,</p><div>Find articles by <a href="/pubmed/?term=Jacobi%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746">Paula M. Jacobi</a></div></div><div id="_co_idm140517552215392"><h3 class="no_margin">Brian T. Emmer</h3><p><sup>3</sup>Department of Internal Medicine,</p><div>Find articles by <a href="/pubmed/?term=Emmer%20BT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746">Brian T. Emmer</a></div></div><div id="_co_idm140517540621600"><h3 class="no_margin">Colin A. Kretz</h3><p><sup>4</sup>Howard Hughes Medical Institute, and</p><div>Find articles by <a href="/pubmed/?term=Kretz%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746">Colin A. Kretz</a></div></div><div id="_co_idm140517544382512"><h3 class="no_margin">Ayse Bilge Ozel</h3><p><sup>1</sup>Department of Human Genetics,</p><div>Find articles by <a href="/pubmed/?term=Ozel%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746">Ayse Bilge Ozel</a></div></div><div id="_co_idm140517549612464"><h3 class="no_margin">Beth McGee</h3><p><sup>5</sup>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI;</p><div>Find articles by <a href="/pubmed/?term=McGee%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746">Beth McGee</a></div></div><div id="_co_idm140517549610480"><h3 class="no_margin">Chava Kimchi-Sarfaty</h3><p><sup>6</sup>Hemostasis Branch, Division of Plasma Protein Therapeutics, Office of Tissues and Advanced Therapies, Center for Biologics Evaluation and Research, US Food and Drug Administration, Silver Spring, MD; and</p><div>Find articles by <a href="/pubmed/?term=Kimchi-Sarfaty%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746">Chava Kimchi-Sarfaty</a></div></div><div id="_co_idm140517551404528"><h3 class="no_margin">David Ginsburg</h3><p><sup>1</sup>Department of Human Genetics,</p><p><sup>2</sup>Department of Pediatrics and Communicable Disease,</p><p><sup>3</sup>Department of Internal Medicine,</p><p><sup>4</sup>Howard Hughes Medical Institute, and</p><div>Find articles by <a href="/pubmed/?term=Ginsburg%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746">David Ginsburg</a></div></div><div id="_co_idm140517549671536"><h3 class="no_margin">Jun Z. Li</h3><p><sup>1</sup>Department of Human Genetics,</p><p><sup>7</sup>Thrombosis and Atherosclerosis Research Institute, Department of Medicine, McMaster University, Hamilton, ON, Canada</p><div>Find articles by <a href="/pubmed/?term=Li%20JZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746">Jun Z. Li</a></div></div><div id="_co_idm140517619361504"><h3 class="no_margin">Karl C. Desch</h3><p><sup>2</sup>Department of Pediatrics and Communicable Disease,</p><div>Find articles by <a href="/pubmed/?term=Desch%20KC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29296746">Karl C. Desch</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140517537835424_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140517537835424_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140517537835424_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140517537835424_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Human Genetics,</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Pediatrics and Communicable Disease,</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Department of Internal Medicine,</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Howard Hughes Medical Institute, and</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI;</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Hemostasis Branch, Division of Plasma Protein Therapeutics, Office of Tissues and Advanced Therapies, Center for Biologics Evaluation and Research, US Food and Drug Administration, Silver Spring, MD; and</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Thrombosis and Atherosclerosis Research Institute, Department of Medicine, McMaster University, Hamilton, ON, Canada</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140517537835424_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2017 Feb 8; Accepted 2017 Apr 22.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140517537835424_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5728318/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC5728318/bin/ba005629-suppl1.pdf" data-ga-action="click_feat_suppl">ba005629-suppl1.pdf</a><span style="color:gray"> (6.1M)</span></div><div class="small guid">GUID:&nbsp;E7AFF400-60EF-4F16-A801-929547CEF706</div></div></dd></dl></div><div id="idm140517546355920" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140517546355920title">Key Points</h2><!--article-meta--><div><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="__p2">Three independent association signals at <em>ADAMTS13</em> and smoking were identified as major predictors of plasma ADAMTS13 levels.</p></li><li><p id="__p3">Evidence was presented that 2 nonsynonymous <em>ADAMTS13</em> variants were driving the variation of plasma ADAMTS13 concentrations.</p></li></ul><p></p></div></div><div id="idm140517536551632" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140517536551632title">Abstract</h2><!--article-meta--><div><p id="__p4" class="p p-first-last">The metalloprotease ADAMTS13 cleaves von Willebrand factor (VWF) in circulating blood, limiting the size of VWF multimers and regulating VWF activity. Abnormal regulation of VWF contributes to bleeding and to thrombotic disorders. ADAMTS13 levels in plasma are highly variable among healthy individuals, although the heritability and the genetic determinants of this variation are unclear. We performed genome-wide association studies of plasma ADAMTS13 concentrations in 3244 individuals from 2 independent cohorts of healthy individuals. The heritability of ADAMTS13 levels was between 59.1% (all individuals) and 83.5% (siblings only), whereas tobacco smoking was associated with a decrease in plasma ADAMTS13 levels. Meta-analysis identified common variants near the <em>ADAMTS13</em> locus on chromosome 9q34.2 that were significantly associated with ADAMTS13 levels and collectively explained 20.0% of the variance. The top single nucleotide polymorphism (SNP), rs28673647, resides in an intron of <em>ADAMTS13</em> (β, 6.7%; <em>P</em> = 1.3E-52). Conditional analysis revealed 3 additional independent signals represented by rs3739893 (β, −22.3%; <em>P</em> = 1.2E-30) and rs3124762 (β, 3.5%; <em>P</em> = 8.9E-9) close to <em>ADAMTS13</em> and rs4075970 (β, 2.4%; <em>P</em> = 6.8E-9) on 21q22.3. Linkage analysis also identified the region around <em>ADAMTS13</em> (9q34.2) as the top signal (LOD 3.5), consistent with our SNP association analyses. Two nonsynonymous <em>ADAMTS13</em> variants in the top 2 independent linkage disequilibrium blocks (Q448E and A732V) were identified and characterized in vitro. This study uncovered specific common genetic polymorphisms that are key genetic determinants of the variation in plasma ADAMTS13 levels in healthy individuals.</p></div></div><div id="idm140517538828064" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140517538828064title">Visual Abstract</h2><!--article-meta--><div><p id="__p5" class="p p-first-last"><!--fig ft0--><!--fig mode=article f1--></p><div class="fig iconblock whole_rhythm clearfix" id="absf1" co-legend-rid="lgnd_absf1"><a href="/pmc/articles/PMC5728318/figure/absf1/" target="figure" rid-figpopup="absf1" rid-ob="ob-absf1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140517540906208" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is advances005629absf1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is advances005629absf1.jpg" src="/pmc/articles/PMC5728318/bin/advances005629absf1.jpg"></div></a><div id="largeobj_idm140517540906208" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5728318/figure/absf1/" target="figure" rid-figpopup="absf1" rid-ob="ob-absf1"></a><a target="object" href="/pmc/articles/PMC5728318/figure/absf1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_absf1"></div></div><p></p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p6" class="p p-first">von Willebrand factor (VWF) plays a central role in hemostasis by tethering platelets to areas of endothelial injury and by serving as a carrier for coagulation factor VIII. As a large multimeric glycoprotein, VWF requires specialized assembly, posttranslational modification, and pH-dependent storage conformations to facilitate the regulated secretion of VWF into flowing blood.<sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844836">1</a></sup> The mature VWF complex varies from dimers to concatamers of more than 250 monomers (ultralarge VWF), and the size of VWF is correlated with its prothrombotic activity in circulation.<sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844842">2</a></sup> The homeostatic distribution of VWF multimer size is modulated by ADAMTS13, a metalloprotease that cleaves circulating VWF into smaller concatamers.<sup><a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844848">3</a></sup> Previous studies have reported loss of higher-molecular-weight multimers in the bleeding disorder von Willebrand disease type 2A, and conversely, an accumulation of ultralarge VWF monomers<sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844868">4</a></sup> in thrombotic thrombocytopenic purpura,<sup><a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844823">5</a></sup> demonstrating the importance of the regulation of VWF activity by ADAMTS13. Recent reports have also linked lower ADAMTS13 plasma concentrations with an increased risk of ischemic stroke,<sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844859">6</a>,<a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844864">7</a></sup> coronary artery disease, and all-cause mortality.<sup><a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844865">8</a>,<a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844861">9</a></sup> Unlike many proteases, ADAMTS13 is secreted in an active form and, to date, no specific inhibitor of ADAMTS13 has been identified. Recent reports have suggested that the regulation of ADAMTS13 activity is primarily allosteric and is dependent on specific exosite-substrate interactions.<sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844862">10</a></sup> Therefore, ADAMTS13 activity is primarily dependent on its concentration in circulation and its ability to interact with VWF. In a healthy population, VWF levels vary up to fivefold and are determined by multiple environmental and genetic factors.<sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844820">11</a></sup> ADAMTS13 activity levels seem to be less variable than VWF concentrations in healthy populations and decrease with age after 30 years.<sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844835">12</a></sup> ADAMTS13 antigen levels exhibit less variability than VWF when measured serially.<sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844832">13</a></sup> To date, the amount of variation in ADAMTS13 antigen levels determined by genetic factors is unknown. Recently, de Vries et al<sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844863">14</a></sup> reported a genome-wide association study (GWAS) of ADAMTS13 activity toward a synthetic VWF substrate in an adult Dutch population, that uncovered an association of single nucleotide variants at the <em>ADAMTS13</em> locus that explained ∼6% of the variance.</p><p id="__p7" class="p p-last">In this study, we measured ADAMTS13 antigen levels in 2 independent young healthy cohorts (n = 3244) and reported heritability, genetic associations, and the impact of smoking status. Consistent with the previously published GWAS of ADAMTS13 activity, our analyses identified a strong influence of variants at <em>ADAMTS13,</em> our study identified a new top association signal for ADAMTS13 concentrations, and stepwise conditional analyses demonstrated the presence of a previously unreported association signal on chromosome 21q2.3. In vitro analyses of 2 common nonsynonymous variants of <em>ADAMTS13</em> support their causal role in plasma ADAMTS13 variation.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Materials and methods</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Genes and Blood Clotting (GABC) study</h3><p id="__p8" class="p p-first-last">A cohort of 1189 healthy siblings representing 507 sibships between 14 and 35 years old was collected between 2006 and 2009 at the University of Michigan (Ann Arbor, MI). Participants who reported that they were pregnant or had a known bleeding disorder or chronic illness requiring regular medical care were excluded. All participants gave informed consent before participating.<sup><a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844827">15</a></sup></p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Trinity Student Study (TSS)</h3><p id="__p9" class="p p-first-last">Healthy Irish individuals age 18 to 28 years who attended Trinity College of the University of Dublin (Dublin, Ireland) were recruited between 2003 and 2004 for genetic analysis of nutrition and diet-related traits. Ethical approval was obtained from the Dublin Federated Hospitals Research Ethics Committee and was reviewed by the Office of Human Subjects Research at the US National Institutes of Health. Participants provided written informed consent before recruitment.<sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844852">11</a></sup></p></div><div id="__sec5" class="sec"><h3 id="__sec5title">ADAMTS13 phenotyping and data processing</h3><p id="__p10" class="p p-first">ADAMTS13 levels were determined by AlphaLISA (Perkin-Elmer, Waltham, MA) from platelet-poor plasma for the 2 cohorts and cell culture media or lysates for the in vitro characterization. A polyclonal anti-ADAMTS13 antibody, raised in rabbit against a recombinant human ADAMTS13 variant with C1r-C1s urinary epidermal growth factor bone morphogenetic protein (CUB) domain deleted and recombinant murine full-length ADAMTS13 (9:1 ratio) (Open Biosystems, Huntsville, AL) was obtained from the laboratory of X. Long Zheng (University of Alabama at Birmingham, Birmingham, AL). Monoclonal anti-ADAMTS13 antibody (GMA-360) was purchased from Green Mountain Antibodies (Burlington, VT). This antibody pair was used in an ADAMTS13-specific AlphaLISA assay. ADAMTS13 antigen levels were calculated by using a standard dilution curve of standard plasma (FACT; George King Biomedical, Overland Park, KS). Each plasma sample was independently assayed at least 4 times. The mean replicate sample coefficients of variation were 2.15% (GABC) and 2.87% (TSS).</p><p id="__p11">We noted a difference in the mean and median ADAMTS13 concentrations in the GABC and TSS samples. To detect a difference in assay performance between samples collected in acid citrate dextrose solution (GABC) and samples collected in EDTA (TSS), we measured ADAMTS13 levels from a single blood donor collected in citrate or EDTA with and without additions of 0.8 mM CaCl<sub>2</sub> to the assay buffer. Consistent with the differences between the GABC and TSS samples, we noted a 20% reduction in the AlphaLISA signal from plasma collected in an EDTA-coated tube compared with the same plasma collected in a 10% acid citrate dextrose solution (Sigma, St. Louis, MO) that was not corrected by CaCl<sub>2</sub> additions. Therefore, we mean-centered the distributions to allow for comparisons between the 2 cohorts.</p><p id="__p12" class="p p-last">Three GABC and 8 TSS participants did not have ADAMTS13 measurements and were excluded. The raw ADAMTS13 levels were log2-transformed and mean-centered within each cohort. Potential covariates, including age, sex, height, weight, body mass index, smoking status, and principal components (PCs) of genotype data were evaluated for their influence on ADAMTS13 levels by stepwise linear regression with forward selection. As a result, log-transformed ADAMTS13 levels were adjusted by smoking and PC1 in the GABC cohort, and by sex, smoking status, body mass index, and PC2 in the TSS cohort. After covariate adjustment, 9 outlier samples (defined as 3 times the standard deviation from mean) were removed in GABC, and 34 outliers were removed in TSS. We also separately analyzed the effect of smoking.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Genotyping and imputation</h3><p id="__p13" class="p p-first-last">GABC and TSS samples were genotyped by using the Illumina HumanOmni1-Quad_v1-0_B array. Details of genotyping and the data-cleaning process have been previously published.<sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844840">11</a>,<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844826">16</a></sup> Nine hundred forty of the 1152 GABC participants and all 2304 TSS participants had verified European ancestry.<sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844853">11</a>,<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844828">16</a></sup> The final cleaned genotype data set contained 795 544 single nucleotide polymorphisms (SNPs) for 1152 GABC participants and 755 451 SNPs for 2304 TSS participants. After combining imputed genotypes from both cohorts,<sup><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844855">17</a></sup> the final data set contained ∼5.82 million SNPs common to both cohorts. All the genotyping data from SNP arrays has been uploaded to dbGaP (<a href="https://www.ncbi.nlm.nih.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/gap</a> accession number phs000304.v2.p1).</p></div><div id="__sec7" class="sec"><h3 id="__sec7title">Heritability estimation using SNP genotypes and pedigree data</h3><p id="__p14" class="p p-first-last">The proportions of variance of adjusted ADAMTS13 levels explained by all genotyped SNPs or the top associated SNPs were estimated by using a restricted maximum likelihood method with Genome-Wide Complex Trait Analysis, v1.25 (GCTA).<sup><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844849">18</a></sup> We used 557 sibships (1139 GABC and 138 TSS sibs) and also used 2 pedigree-based methods to estimate the narrow-sense heritability of ADAMTS13 levels: (1) intraclass correlation of sibpairs<sup><a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844847">16</a></sup> using the <em>irr</em> package<sup><a href="#B19" rid="B19" class=" bibr popnode">19</a></sup> in R<sup><a href="#B20" rid="B20" class=" bibr popnode">20</a></sup> and (2) pedigree-wide regression analysis using MERLIN-REGRESS (v1.1.2).<sup><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844844">21</a></sup></p></div><div id="__sec8" class="sec"><h3 id="__sec8title">Association analyses</h3><p id="__p15" class="p p-first-last">SNP quantitative trait association analysis for the adjusted ADAMTS13 level was performed in the European subset of GABC (n = 940) and in TSS (n = 2304) using a variance component model implemented in EMMAX (Efficient Mixed-Model Association Expedited).<sup><a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844833">22</a></sup> The genome-wide significance level was set at <em>P</em> = 5 × 10<sup>−8</sup> based on a Bonferroni correction for 1 million independent tests. To identify independent signals that could be masked by the top-associated SNPs, we performed stepwise conditional analysis using GCTA-COJO<sup><a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844829">23</a></sup> by adding the SNP with the strongest evidence of association as a new covariate iteratively until there was no significantly associated SNP.</p></div><div id="__sec9" class="sec"><h3 id="__sec9title">Meta-analysis of TSS and GABC</h3><p id="__p16" class="p p-first-last">Meta-analysis was carried out by using a fixed effect, sample size–weighted approach implemented in METAL<sup><a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844850">24</a></sup> using EMMAX association results for European GABC and TSS from a common set of ∼5.82 million SNPs. The genomic control factor<sup><a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844843">25</a></sup> was 0.991 for GABC and 0.995 for TSS for single-marker test results in EMMAX. Regional plots of top-associated SNPs were generated by LocusZoom.<sup><a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844825">26</a></sup> SNP annotation was obtained from SNPnexus.<sup><a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844837">27</a></sup> Damaging status of nonsynonymous coding variants was predicted by PolyPhen-2 (Polymorphism Phenotyping v2).<sup><a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844822">28</a></sup></p></div><div id="__sec10" class="sec"><h3 id="__sec10title">Linkage analysis using GABC and TSS sibling data sets</h3><p id="__p17" class="p p-first-last">Linkage analyses were performed by MERLIN-REGRESS, v1.1.2<sup><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844866">21</a></sup> for the 557 sibships using 35 356 linkage disequilibrium (LD) clusters as previously described.<sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844846">11</a></sup></p></div><div id="__sec11" class="sec sec-last"><h3 id="__sec11title">Recombinant ADAMTS13</h3><p id="__p18" class="p p-first-last">ADAMTS13 variants Q448E and A732V were synthesized by site-directed mutagenesis of a stable mammalian expression vector containing wild-type <em>ADAMTS13</em> complementary DNA (cDNA). Vector sequence was generated by amplification of pDEST-pcDNA5-BirA-FLAG C-term<sup><a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844821">29</a></sup> in which BirA and FLAG tags were excluded in the final Gibson assembly. Plasmids were fully sequenced to ensure that no polymerase chain reaction–generated mutations were present. Plasmids were transfected into flip-in T-REx-293 cells by using Lipofectamine-2000 (Life Technologies, Carlsbad CA), and drug-resistant cells were selected with hygromycin and maintained with 5 μg/mL blasticidin. To compare synthesis and secretion rates, cells were plated into Dulbecco’s modified Eagle medium media with 10% fetal bovine serum at a cell density of 5 × 10<sup>5</sup> cells per well and incubated at 37°C in 5% CO<sub>2</sub>. Conditioned media was collected after 48-hour induction with 1 μg/mL tetracycline and clarified by centrifugation, and cells were washed in phosphate-buffered saline and harvested to obtain cell lysates. Western blots using mouse anti-ADAMTS13 antibodies (Green Mountain Antibodies) confirmed synthesis of all wild-type and ADAMTS13 variants at expected molecular weights. Antigen levels were determined by AlphaLISA as described above, substituting conditioned media and cell lysates for plasma. As an estimate of cell numbers per well, levels of glyceraldehyde-3-phosphate dehydrogenase were determined according to the AlphaLISA kit protocol (Perkin-Elmer). Activity of the recombinant ADAMTS13 variants was determined by FRETS-VWF73 assays according to the manufacturer’s specifications (Peptides International, Louisville, KY) with the following modifications: 25 µL recombinant wild-type or variant ADAMTS13 in serum-free conditioned OptiMEM media (Life Technologies, Carlsbad, CA) was used as the enzyme source at reaction concentrations of 0.1 to 3.2 nM and a volume of 25 µL. FRETS-VWF73 substrate was used at final concentrations of 0.125 to 4.0 µM for a total reaction volume of 50 µL. We measured signal generation over 60 minutes at varying enzyme and substrate concentrations to calculate specific activity by using a Molecular Devices Spectramax M2 plate reader. Differences in synthesis and expression among the ADAMTS13 variants were tested by using Mann-Whitney <em>U</em> tests (Prism 7; GraphPad Software, San Diego, CA).</p></div></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Results</h2><div id="__sec13" class="sec sec-first"><h3 id="__sec13title">Study cohorts and ADAMTS13 antigen levels</h3><p id="__p19" class="p p-first">The demographic characteristics and raw ADAMTS13 antigen levels of the GABC and TSS cohorts are summarized in <a href="/pmc/articles/PMC5728318/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. The 5th and 95th percentiles of raw ADAMTS13 levels spanned a 2.0-fold range for GABC (81.8-165.6 IU/dL) and a 1.8-fold range for TSS (56.0-99.6 IU/dL). There was no significant difference in the distribution of mean-centered log-transformed ADAMTS13 levels between GABC and TSS (Kolmogorov-Smirnov test, <em>P</em> = .06; Mann-Whitney <em>U</em> test, <em>P</em> = .61; supplemental Figure 1). Mean-centered levels were adjusted for covariates as described in “Materials and methods.”</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p20">Characteristics of study cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140517547780192" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1">Cohort</th><th colspan="5" rowspan="1" align="center">GABC</th><th colspan="5" rowspan="1" align="center">TSS</th></tr><tr><th rowspan="1" colspan="1" align="center">Sibship size (no. of sibships)</th><th rowspan="1" colspan="1" align="center">n/N</th><th rowspan="1" colspan="1" align="center">%</th><th rowspan="1" colspan="1" align="center">Q1</th><th rowspan="1" colspan="1" align="center">Q3</th><th rowspan="1" colspan="1" align="center">Sibship size (no. of sibships)</th><th rowspan="1" colspan="1" align="center">n/N</th><th rowspan="1" colspan="1" align="center">%</th><th rowspan="1" colspan="1" align="center">Q1</th><th rowspan="1" colspan="1" align="center">Q3</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1">No. of participants</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1152</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">2304</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Age, y</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">23</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="center">24</td></tr><tr><td rowspan="1" colspan="1">Females</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">721</td><td rowspan="1" colspan="1" align="center">62.6</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1353</td><td rowspan="1" colspan="1" align="center">58.7</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Weight, lbs (kg)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">145 (65.8)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">125 (56.7)</td><td rowspan="1" colspan="1" align="center">165 (74.8)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">148 (67.1)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">133 (60.3)</td><td rowspan="1" colspan="1" align="center">167 (75.7)</td></tr><tr><td rowspan="1" colspan="1">Height, inches (cm)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">67 (170)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">64 (163)</td><td rowspan="1" colspan="1" align="center">70 (178)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">68 (173)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">65 (165)</td><td rowspan="1" colspan="1" align="center">71 (180)</td></tr><tr><td rowspan="1" colspan="1">BMI, kg/m<sup>2</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">22.5</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">20.7</td><td rowspan="1" colspan="1" align="center">25.0</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">22.6</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">21.0</td><td rowspan="1" colspan="1" align="center">24.5</td></tr><tr><td rowspan="1" colspan="1">Current smoker</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">53/1151<sup>*</sup></td><td rowspan="1" colspan="1" align="center">4.6</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">723/2299</td><td rowspan="1" colspan="1" align="center">31.4</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Sibships</td><td rowspan="1" colspan="1" align="center">2 (366)<br>3 (94)<br>4 (22)<br>5 (5)<br>6 (2)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">2 (66)<br>3 (2)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">ADAMTS13, IU/dL</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">115.4<sup>*</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">101.8</td><td rowspan="1" colspan="1" align="center">131.6</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">76.3<sup>*</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">68.0</td><td rowspan="1" colspan="1" align="center">85.2</td></tr></tbody></table></div><div id="largeobj_idm140517547780192" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5728318/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140517552124848"><p id="__p21" class="p p-first-last">Values are medians for age, height, weight, body mass index (BMI), and raw ADAMTS13 levels.</p></div><div id="idm140517552123872"><p id="__p22" class="p p-first-last">N or n, number of individuals in the indicated study cohort(s); Q1, first quartile; Q3, third quartile.</p></div><div id="TF1-1"><sup>*</sup>Previously unreported data. All other tabulated data are adapted from Ma et al.<sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844869">14</a></sup></div></div></div></div><div id="__sec14" class="sec"><h3 id="__sec14title">ADAMTS13 levels are highly heritable</h3><p id="__p24" class="p p-first-last">The narrow-sense heritability (h<sup>2</sup>) of the ADAMTS13 antigen levels derived from all SNP genotyping data from GABC and TSS combined was 59.1% using GCTA. The upper bound of h<sup>2</sup> in the sibling subset of GABC and TSS combined was 83.5% estimated by interclass correlation, which was consistent with the pedigree-based estimate of 79.5% by MERLIN-REGRESS (supplemental Table 1).</p></div><div id="__sec15" class="sec"><h3 id="__sec15title">Linkage analysis for ADAMTS13 in 1139 GABC and 138 TSS sibs</h3><p id="__p25" class="p p-first-last">Linkage analysis for ADAMTS13 in 489 sibships (1139 individuals) from GABC and 68 sibships (138 individuals) from TSS identified 1 independent region of linkage with LOD score greater than 3. The highest signal was at ∼136 Mb onchromosome 9q34.2, close to the <em>ADAMTS13</em> gene (LOD, 3.5; MERLIN-REGRESS <em>P</em> = 2.9E-5; simulation-based empirical <em>P</em> = .10) (supplemental Figure 2). No other independent regions of linkage were identified.</p></div><div id="__sec16" class="sec"><h3 id="__sec16title">Variants at <em>ADAMTS13</em> are associated with ADAMTS13 levels</h3><p id="__p26" class="p p-first">GWAS of the European subset of GABC (n = 940) revealed 46 significantly associated SNPs for ADAMTS13 levels (<em>P</em> &lt; 5.0E-8) in an additive model (supplemental Figure 3A). All 46 SNPs resided in a 200-kb region on 9q34.2 surrounding the <em>ADAMTS13</em> gene (supplemental Table 2). The Q-Q plot of the observed vs expected –log<sub>10</sub>(<em>P</em>) demonstrated a deviation from expectation, entirely because of the significant signals on 9q34.2 (supplemental Figure 3B). The minor allele G of the top SNP, rs28673647, was associated with ADAMTS13 with a β coefficient of 6.9% ± 1.0% (minor allele frequency [MAF], 0.36; <em>P</em> = 8.1E-13), equivalent to a 6.9% increase in ADAMTS13 level per allele (supplemental Figure 3C).</p><p id="__p27" class="p p-last">GWAS of TSS revealed 162 significantly associated SNPs for ADAMTS13 residing in the same 200-kb region on 9q34.2 containing <em>ADAMTS13</em> (supplemental Table 3; supplemental Figure 4A-B). The top SNP in GABC, rs28673647, was associated with a 6.6% increase in ADAMTS13 level in TSS (MAF, 0.39; <em>P</em> = 9.3E-42). The top SNP in TSS, rs28680325, had a β of 6.7% (MAF, 0.38; <em>P</em> = 1.3E-42) (supplemental Figure 4C). The top SNPs discovered in TSS showed similar allelic effect sizes and directions (<em>r</em>, 0.98; <em>P</em> &lt; 2.2E-16) as well as <em>P</em> values (<em>r</em>, 0.92; <em>P</em> &lt; 2.2E-16) as in GABC and vice versa (supplemental Figure 5).</p></div><div id="__sec17" class="sec"><h3 id="__sec17title">Meta-analysis in GABC and TSS confirms <em>ADAMTS13</em> signals</h3><p id="__p28" class="p p-first">To increase statistical power, a meta-analysis of the GABC and TSS cohorts using a common set of ∼5.82 million SNPs was performed, revealing 153 SNPs significantly associated with ADAMTS13 levels (<em>P</em> &lt; 5.0E-8) (<a href="/pmc/articles/PMC5728318/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A-B</span></a>; supplemental Table 4). These SNPs collectively explained 20.0% ± 5.9% of ADAMTS13 level variation in the combined GABC and TSS data. All significant SNPs were close to the <em>ADAMTS13</em> gene on 9q34.2.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5728318/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140517541730912" class="figure"><a href="/pmc/articles/PMC5728318/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5728318_advances005629f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is advances005629f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5728318/bin/advances005629f1.jpg"></a></div><div id="largeobj_idm140517541730912" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5728318/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5728318/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p29"><strong>Meta-analysis of ADAMTS13 levels in GABC and TSS.</strong> (A) Genome-wide plot of –log10(<em>P</em>) for ∼5.82 million SNPs. The red line marks the 5.0E-8 threshold of genome-wide significance. (B) Quantile-quantile plot of observed vs expected –log10(<em>P</em>) for ADAMTS13 meta-analysis. The observed <em>P</em> &lt; 5.0E-8 are shown in red. (C) Regional plot for the associated region near <em>ADAMTS13</em> on Chr9. (D) Regional plot for <em>ADAMTS13</em> and <em>STKLD1</em> (<em>C9orf96</em>) genes on chromosome 9 (chr9) conditioned by the top SNP rs28673647 from meta-analysis. (E) Regional plot for <em>ADAMTS13</em> conditioned by the top 2 independent SNPs, rs28673647 and rs3739893. (F) Regional plot for <em>PRDM15</em> on chr21 when using the top 2 independent SNPs, rs28673647 and rs3739893, as covariates.</p></div></div></div><p id="__p30" class="p">Stepwise conditional analysis in GABC and TSS combined was performed to identify other association signals masked by the strong primary signal on 9q34.2. In conditional analysis, 3 independent signals on 9q34.2 represented by top SNPs rs28673647 (MAF, 0.38; β, 7.1% ± 0.5%; <em>P</em> = 1.2E-57), rs3739893 (β, −22.3% ± 1.8%; <em>P</em> = 1.2E-30), and a borderline-significant SNP rs3124762 (β, 3.5% ± 0.6%; <em>P</em> = 8.9E-9) were uncovered (<a href="/pmc/articles/PMC5728318/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; <a href="/pmc/articles/PMC5728318/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1C-E</span></a>). The second independent SNP on 9q34.2, rs3739893 (MAF, 1.6%) was located in an intron of the <em>STKLD1</em> (<em>C9orf96</em>) gene and had a larger effect size (β, −15.6% in GABC and −23.3% in TSS) than that of the top SNP, rs28673647. An additional signal in an intron of <em>PRDM15</em> on 21q22.3 represented by the SNP rs4075970 (MAF, 0.45; β, 2.4% ± 0.4%; <em>P</em> = 6.8E-9) was masked by the first 2 independent 9q34.2 signals but revealed by conditional analysis (<a href="/pmc/articles/PMC5728318/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; <a href="/pmc/articles/PMC5728318/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1F</span></a>). The individual genotypes of these 4 SNPs decreased ADAMTS13 levels independently (supplemental Figure 6), which supports the findings in the conditional analyses.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p31">Significant SNPs in conditional meta-analysis in individual cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140517551323888" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1">CHR</th><th colspan="5" rowspan="1" align="center">GABC + TSS (N = 3238)</th><th colspan="4" rowspan="1" align="center">GABC (n = 934)</th><th colspan="4" rowspan="1" align="center">TSS (n = 2304)</th></tr><tr><th rowspan="1" colspan="1" align="center">SNP</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">βJ (%)</th><th rowspan="1" colspan="1" align="center">SE (%)</th><th rowspan="1" colspan="1" align="center"><em>P</em>J</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">β (%)</th><th rowspan="1" colspan="1" align="center">SE (%)</th><th rowspan="1" colspan="1" align="center"><em>P</em></th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">β (%)</th><th rowspan="1" colspan="1" align="center">SE (%)</th><th rowspan="1" colspan="1" align="center"><em>P</em></th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1">9q34.2</td><td rowspan="1" colspan="1" align="center">rs28673647</td><td rowspan="1" colspan="1" align="center">0.38</td><td rowspan="1" colspan="1" align="center">7.1</td><td rowspan="1" colspan="1" align="center">0.5</td><td rowspan="1" colspan="1" align="center">1.2E-57</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">6.9</td><td rowspan="1" colspan="1" align="center">1.0</td><td rowspan="1" colspan="1" align="center">8.1E-13</td><td rowspan="1" colspan="1" align="center">0.39</td><td rowspan="1" colspan="1" align="center">6.6</td><td rowspan="1" colspan="1" align="center">0.5</td><td rowspan="1" colspan="1" align="center">9.3E-42</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">rs3739893</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">−22.3</td><td rowspan="1" colspan="1" align="center">1.8</td><td rowspan="1" colspan="1" align="center">1.2E-30</td><td rowspan="1" colspan="1" align="center">0.01</td><td rowspan="1" colspan="1" align="center">−15.6</td><td rowspan="1" colspan="1" align="center">4.7</td><td rowspan="1" colspan="1" align="center">1.7E-3</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">−23.3</td><td rowspan="1" colspan="1" align="center">1.9</td><td rowspan="1" colspan="1" align="center">2.0E-28</td></tr><tr><td rowspan="1" colspan="1">21q22.3</td><td rowspan="1" colspan="1" align="center">rs4075970</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center">2.4</td><td rowspan="1" colspan="1" align="center">0.4</td><td rowspan="1" colspan="1" align="center">6.8E-9</td><td rowspan="1" colspan="1" align="center">0.48</td><td rowspan="1" colspan="1" align="center">5.0</td><td rowspan="1" colspan="1" align="center">1.0</td><td rowspan="1" colspan="1" align="center">1.4E-7</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">1.4</td><td rowspan="1" colspan="1" align="center">0.5</td><td rowspan="1" colspan="1" align="center">4.6E-03</td></tr><tr><td rowspan="1" colspan="1">9q34.2</td><td rowspan="1" colspan="1" align="center">rs3124762</td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center">3.5</td><td rowspan="1" colspan="1" align="center">0.6</td><td rowspan="1" colspan="1" align="center">8.9E-9</td><td rowspan="1" colspan="1" align="center">0.15</td><td rowspan="1" colspan="1" align="center">−1.2</td><td rowspan="1" colspan="1" align="center">1.4</td><td rowspan="1" colspan="1" align="center">.38</td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">0.9</td><td rowspan="1" colspan="1" align="center">0.7</td><td rowspan="1" colspan="1" align="center">.18</td></tr></tbody></table></div><div id="largeobj_idm140517551323888" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5728318/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140517542538048"><p id="__p32" class="p p-first-last">β, percentage change per allele; CHR, chromosome; J, joint effects of independently associated SNPs; N or n, number of individuals in the indicated study cohort(s); SE, standard error.</p></div></div></div></div><div id="__sec18" class="sec"><h3 id="__sec18title">Functional annotation of associated variants in 9q24.3</h3><p id="__p33" class="p p-first">There were 153 SNPs in the 9q24.3 region associated with ADAMTS13 antigen levels identified through meta-analysis. To identify causal variants, we looked for nonsynonymous variants in <em>ADAMTS13</em> or in SNPs at cis expression quantitative loci (eQTLs) that were in LD with the top independent association signals. Of the 153 significant SNPs, we did not identify eQTL SNPs in our analysis, which used 2 different eQTL browsers,<sup><a href="#B30" rid="B30" class=" bibr popnode">30</a>,<a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844831">31</a></sup> but we did detect 2 missense SNPs in <em>ADAMTS13</em> that were associated with ADAMTS13 levels. The top missense variant, rs2301612, encoding Q448E, was in high LD with the top meta-analysis SNP rs28673647 (<em>r</em><sup>2</sup>, 0.74) and was associated with a 5.4% increase (<em>P</em> = 4.9E-36) in ADAMTS13 levels. The second strongest missense SNP (rs41314453 encoding A732V) was in high LD with the top conditional analysis SNP (rs3739893; <em>r</em><sup>2</sup>, 0.84) and was associated with a 26.5% decrease (<em>P</em> = 2.3E-34) in ADAMTS13 levels. Variants encoding Q448E and A732V were not in LD with each other (<em>r</em><sup>2</sup>, 0.02).</p><p id="__p34" class="p">The Q448E amino acid substitution occurs in the cysteine-rich domain of ADAMTS13, whereas A732V alters the second thrombospondin-1 repeat domain (<a href="/pmc/articles/PMC5728318/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>). We hypothesized that these missense variants were driving the direction of associations in the top 2 LD blocks, so we compared the synthesis and secretion of wild-type ADAMTS13 to Q448E and A732V mutants in vitro. We generated stable HEK293 Flp-In T-REx cell lines expressing these 3 forms of ADAMTS13 and measured the concentration of ADAMTS13 (μg/μL) in conditioned media and cell lysates, adjusting concentrations for the estimated number of cells per well with glyceraldehyde-3-phosphate dehydrogenase (μg/μL) levels from cell lysates. This cell culture system, in which a transposase is used to insert cDNA into a specific genomic locus, facilitated equally controlled transgene expression under a tet promotor and allowed for a more precise quantification of the differences in synthesis and secretion due only to the missense variants. Mean media concentrations of adjusted ADAMTS13 were highest in wells plated with wild-type ADAMTS13 and Q448E-producing cells (1.14 and 0.92, respectively; <em>P</em> = .180). Cell lysates from Q448E variants had higher concentrations compared with wild-type cells (0.03 and 0.02 respectively, <em>P</em> = .009) (<a href="/pmc/articles/PMC5728318/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B-C</span></a>). Conditioned media from cells expressing A732V had lower adjusted ADAMTS13 concentrations compared with wild-type media (0.55 and 1.14, respectively; <em>P</em> = .002) and higher concentrations in cell lysates compared with wild-type (0.04 and 0.02, respectively; <em>P</em> = .009) (<a href="/pmc/articles/PMC5728318/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B-C</span></a>). Specific activity was 13.0 ± 0.44 nM/min/nM for wild-type, 6.492 ± 0.23 nM/min/nM for Q448E, and 8.849 ± 0.30 nM/min/nM for A732V (<a href="/pmc/articles/PMC5728318/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2D</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5728318/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140517540323984" class="figure"><a href="/pmc/articles/PMC5728318/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5728318_advances005629f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is advances005629f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5728318/bin/advances005629f2.jpg"></a></div><div id="largeobj_idm140517540323984" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5728318/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5728318/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p35"><strong>Functional tests of ADAMTS13 variant expression secretion and activity.</strong> (A) Cell lines expressing wild-type, Q448E, and A732V ADAMTS13 were generated by using an FLP-recombinase system in 293 cells. Q448E is rs2301612 and A732V is rs41314453. (B-C) Concentrations of ADAMTS13 in (B) conditioned media and (C) cell lysate were measured by using AlphaLISA and adjusted by cell lysate glyceraldehyde-3-phosphate dehydrogenase levels to account for variations in cell number. Mean and standard deviations are plotted as well as individual values. (D) Recombinant enzyme-specific activity was measured with FRETS-VWF73 substrate. Differences in group levels were determined by the Mann Whitney <em>U</em> test. 1-8, thrombospondin type 1 repeats; CUB, C1r-C1s urinary epidermal growth factor bone morphogenetic protein domains; Cys-rich, cysteine-rich domain; P, propeptide; S, signal peptide.</p></div></div></div></div><div id="__sec19" class="sec"><h3 id="__sec19title">Focused analysis with ABO blood group serotypes</h3><p id="__p36" class="p p-first-last">The <em>ABO</em> gene is located ∼200 kb 5′ to the <em>ADAMTS13</em> locus. The ABO blood group serotypes (A1, A2, B, and O) were estimated by using haplotypes tagged by 3 ABO SNPs: rs8176749, rs8176704, and rs687289.<sup><a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844860">32</a></sup> Meta-analysis identified 15 SNPs 5′-upstream of <em>ABO</em> and 0 SNPs within <em>ABO</em> significantly associated with ADAMTS13 levels. None of the 3 ABO-serotype SNPs were in high LD with the 15 <em>ABO</em> 5′-upstream SNPs (<em>r</em><sup>2</sup>, &lt;0.2 in both GABC and TSS). Although several SNP haploblocks in <em>ABO</em> were associated with ADAMTS13 levels, ABO-serotype tagging alleles were not significantly associated with ADAMTS13 levels (minimum <em>P</em> = 4.14E-5 for A1 haplotype tagging SNP) (supplemental Table 5), suggesting that the associated SNPs upstream of <em>ABO</em> are in LD with functional SNPs in <em>ADAMTS13</em> and not independently associated with ADAMTS13 levels. This is consistent with the minimal correlation between VWF levels, which are highly influenced by ABO serotypes, and ADAMTS13 levels (<em>r</em>, 0.04; <em>P</em> = .02) in the GABC and TSS cohorts (supplemental Figure 7). In addition, previous reports demonstrated no association between ADAMTS13 antigen levels<sup><a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844830">33</a></sup> or activity<sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844857">12</a></sup> and ABO blood groups.</p></div><div id="__sec20" class="sec sec-last"><h3 id="__sec20title">Smoking is associated with decreased ADAMTS13 levels</h3><p id="__p37" class="p p-first-last">As a complex genetic trait, plasma ADAMTS13 concentrations are affected by both genetic and environmental factors such as tobacco smoking. ADAMTS13 levels were significantly lower for smokers compared with nonsmokers in both GABC (β, 8.5% ± 2.8%; <em>P</em> = 2.6E-3) and TSS (β, 3.3% ± 0.74%; <em>P</em> = 8.3E-6), despite the much smaller fraction of smokers in GABC (4.6%) than in TSS (31.4%) (supplemental Figure 8A). To explore gene-smoking interactions, we examined ADAMTS13 levels in current smokers and nonsmokers stratified by using the top 2 independent genetic signals at the <em>ADAMTS13</em> locus (rs28673647 and rs3739893). ADAMTS13 levels in GABC and TSS combined were decreased by the GA and AA genotypes of the top SNP (rs28673647) and the TC genotype of rs3739893, as well as by smoking status (supplemental Figure 8B). The influence of smoking was observed in each stratum of rs28673647 and rs3739893 genotypes, demonstrating that the effect of smoking on ADAMTS13 antigen level is additive to and independent of the genetic effects.</p></div></div><div id="__sec21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec21title">Discussion</h2><p id="__p38" class="p p-first">Plasma ADAMTS13 levels vary twofold in healthy populations<sup><a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844824">33</a></sup> but are tightly regulated within narrow ranges at the individual level. In this study, the estimated heritability of ADAMTS13 antigen levels was between 59.1% and 83.5%, suggesting that the majority of the population variance of plasma ADAMTS13 concentrations is the result of genetic factors. Even our lower-bound heritability estimates were higher than the lower-bound estimate of 35.2% reported by de Vries et al.<sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844845">14</a></sup> This difference may have been a result of the younger age and the lack of age-related vascular diseases in our cohorts or the difference between the measurements of antigen levels in our study vs activity in de Vries et al. The reduction in the influence of environmental factors to ADAMTS13 variability may have also explained why our study accounted for a higher percentage of the variability of plasma ADAMTS13 by common genetic variants (∼20% in our study vs ∼6.0% in de Vries et al).</p><p id="__p39">Tobacco smoking is known to be procoagulant and has a pleiotropic effect on multiple coagulation factors.<sup><a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844834">34</a>,<a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844839">35</a></sup> Our results in European-ancestry cohorts demonstrated that smoking exposure was associated with a 3.6% decrease in ADAMTS13 levels. This finding is consistent with a previous study in Arab males that observed lower ADAMTS13 antigen levels in smokers.<sup><a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844838">36</a></sup> The effect of smoking on ADAMTS13 levels was independent of the genetic effects of significant signals at the <em>ADAMTS13</em> locus. To date the correlation between ADAMTS13 and smoking has not been examined except in extreme deficiencies of ADAMTS13, such as those observed in patients with acute acquired or congenital thrombotic thrombocytopenic purpura.<sup><a href="#B37" rid="B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844856">37</a></sup></p><p id="__p40">A genome-wide study has been conducted for plasma ADAMTS13 activity<sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844841">14</a></sup> but not for ADAMTS13 protein concentrations. Although we expected these 2 traits to be highly correlated and identified similar associations, our top signal was not detected in the GWAS for ADAMTS13 activity. As a comparison, de Vries et al<sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844867">14</a></sup> reported 4 common SNPs and 3 rare SNPs associated with ADAMTS13 activity at the <em>ADAMTS13</em> locus. The top common SNP in their analyses, rs41314453, was in strong LD with the second strongest signal in our study. The second strongest SNP in the de Vries study was rs3118667, and it was also significantly associated with ADAMTS13 levels in our meta-analysis (β, 5.3%; <em>P</em> = 2.64E-36). Interestingly, our top SNP association, rs28673647, which is in high LD with the variant Q448E, was not reported in the de Vries et al study. The high minor allele frequency of this SNP (38%) indicated that the de Vries et al study was well powered to detect an association with rs28673647 and ADAMTS13 activity if it was present. The strong association with antigen levels and lack of association with activity suggests that the increased antigen levels must be balanced by decreased specific activity in vivo. Our experimental results are consistent with this finding with equivocal media levels of Q448E and decreased median activity of Q448E compared with wild-type.</p><p id="__p41">When considering rare variants of 3 significant rare <em>ADAMTS13</em> SNPs reported by de Vries et al, 2 SNPs had an MAF of 0% in GABC and the other, rs142572218, had an MAF of 0.05% in GABC with an association <em>P</em> = .72 compared with their MAF of 0.06% with <em>P</em> = 1.8E-5. The lack of replication in our study with these rare SNPs may have been secondary to reduced power in our study or to a more predominant effect of these rare variants on ADAMTS13 activity compared with plasma concentration. Outside the <em>ADAMTS13</em> locus, 2 common SNPs in the <em>SUPT3H</em> and <em>OBP2B</em> genes, reported as significant in the de Vries study, were not significantly associated with ADAMTS13 antigen levels in our study (<em>P</em> = .44 and <em>P</em> = 0.75, respectively). In fact, only 1 SNP outside the <em>ADAMTS13</em> locus was associated with a change in ADAMTS13 levels in our study: rs4075970 in the <em>PRDM15</em> gene on chromosome 21q22. This signal has not been previously reported. <em>PRDM15</em> encodes a transcription factor with zinc fingers that may play a role in the regulated expression of ADAMTS13. Experimental chromatin immunoprecipitation sequencing data from the Gene Transcription Regulation Database<sup><a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844858">38</a></sup> reveals that multiple transcription factors, including those with zinc fingers, bind to cis regulatory elements at <em>ADAMTS13</em>. However, this database had no direct evidence that the PRDM family transcription factor binds to <em>ADAMTS13</em>.</p><p id="__p42">To more deeply understand the mechanisms of genetic control of complex phenotypes, GWASs would ideally lead to the identification of causal genetic variants. Our study used imputation to increase the number of variants tested and identified 2 missense variants in LD with the top 2 independent association signals at the <em>ADAMTS13</em> locus. Although other unidentified regulatory elements near <em>ADAMTS13</em> likely play additional roles in the genetic control of plasma ADAMTS13 levels, these 2 nonsynonymous mutations are strong candidate causal variants. Indeed, when expressed in cell culture, the A732V form of ADAMTS13 had decreased levels in the conditioned media and increased levels in cell lysate consistent with decreased secretion compared with wild-type. The Q448E variant of ADAMTS13 had equivalent concentrations in conditioned media compared with wild-type despite increased levels in the cell lysate, suggesting that the association with a modest increase in plasma ADAMTS13 may be the result of a combination of higher synthesis rates and/or decreased clearance rates in vivo (not tested). These 2 common ADAMTS13 variants have been examined in 2 previous studies. One study examined expression and secretion of Q448E in cell culture and found results similar to those of our study but an increased specific activity of this variant compared with wild-type,<sup><a href="#B39" rid="B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844854">39</a></sup> and another article examined both the Q448E and A732V variants expressed in cell culture and found similar decreases in secretion into culture media and ∼25% reduction in enzyme activity as measured by a collagen binding assay.<sup><a href="#B40" rid="B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_654844851">40</a></sup> However, both previous studies used transient transfection methods which may have altered patterns of synthesis and secretion compared with our experiments in cell lines with stable expression from the same locus.</p><p id="__p43" class="p p-last">In summary, this article described the results from genome-wide association and linkage studies of ADAMTS13 levels in 2 healthy young cohorts. We identified 153 SNPs at the <em>ADAMTS13</em> locus that collectively explained 20.0% of ADAMTS13-level variation. We also identified and tested 2 common nonsynonymous ADAMTS13 variants with altered synthesis and/or secretion rates compared with reference ADAMTS13. Taken together, these findings suggest that common variants in <em>ADAMTS13</em> are the major common genetic determinants of plasma ADAMTS13 levels and may be primarily driven by differences in secretion by 2 nonsynonymous variants on independent haplotypes.</p></div><div id="__sec22" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec22title">Supplementary Material</h2><!--/article/body/sec/--><p id="__p44" class="p p-first">The full-text version of this article contains a data supplement.</p><div class="sec suppmat" id="SD1"><div class="sup-box half_rhythm" id="idm140517535492496"><a href="/pmc/articles/PMC5728318/bin/ba005629-suppl1.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(6.1M, pdf)</sup></div></div></div><div id="idm140517536530176" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140517536530176title">Acknowledgments</h2><div class="sec"><p id="__p46">The authors thank the participants of the Genes and Blood Clotting study and the Trinity Student Study (TSS) for their contributions to this work; Lawrence Brody, James Mills, and Anne Molloy for their collaboration with the TSS; and X. Long Zheng for providing the polyclonal anti-ADAMTS13 antibodies.</p><p id="__p47">This work was supported by the Intramural Research Programs of the National Human Genome Research Institute and the Eunice Kennedy Shriver National Institute of Child Health and Human Development (Trinity Student Study); by grants R35HL135793 (K.C.D., A.B.O., and D.G.) and R01HL112642 (D.G., J.Z.L., A.B.O., and K.C.D.) from the National Institutes of Health, National Heart, Lung, and Blood Institute; by the Charles Woodson Accelerator grant at the University of Michigan (K.C.D. and P.M.J.); and by funds from the Center for Biologics Evaluation and Research, US Food and Drug Administration. D.G. is an investigator at the Howard Hughes Medical Institute.</p></div></div><div id="__sec23" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec23title">Authorship</h2><!--/article/back/sec/--><p id="__p48" class="p p-first">Contribution: Q.M. designed the research, analyzed the data, created tables and figures, and wrote the article; P.M.J., B.T.E., and B.M. performed the experiments; C.A.K. designed the activity experiments and analyzed the results; A.B.O. analyzed the data; C.K.-S. designed the research and performed the experiments; D.G. and J.Z.L. designed the research and wrote the article; and K.C.D. designed the research, performed the experiments, analyzed the data, created figures and tables, and wrote the paper.</p><p id="__p49">Conflict-of-interest disclosure: The authors declare no competing financial interests.</p><p id="__p50" class="p p-last">Correspondence: Karl C. Desch, Department of Pediatrics and Communicable Diseases, University of Michigan, 8-621 C&amp;W Mott Hospital, SPC 4254, 1540 Hospital Dr, Ann Arbor, MI 48109; e-mail: <a href="mailto:dev@null" data-email="ude.hcimu.dem@hcsedk" class="oemail">ude.hcimu.dem@hcsedk</a>.</p></div><div id="idm140517584997184" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140517584997184title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="mixed-citation">Sadler JE.
<span class="ref-title">von Willebrand factor assembly and secretion</span>. <span class="ref-journal">J Thromb Haemost</span>. 2009;<span class="ref-vol">7</span>(<span class="ref-iss">Suppl 1</span>):24-27. [<a href="/pubmed/19630761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=von+Willebrand+factor+assembly+and+secretion&amp;author=JE+Sadler&amp;volume=7&amp;issue=Suppl+1&amp;publication_year=2009&amp;pages=24-27&amp;pmid=19630761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="mixed-citation">Springer TA.
<span class="ref-title">Biology and physics of von Willebrand factor concatamers</span>. <span class="ref-journal">J Thromb Haemost</span>. 2011;<span class="ref-vol">9</span>(<span class="ref-iss">Suppl 1</span>):130-143. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4117350/">PMC free article</a>]</span> [<a href="/pubmed/21781248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=Biology+and+physics+of+von+Willebrand+factor+concatamers&amp;author=TA+Springer&amp;volume=9&amp;issue=Suppl+1&amp;publication_year=2011&amp;pages=130-143&amp;pmid=21781248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="mixed-citation">Dong JF, Moake JL, Nolasco L, et al. 
<span class="ref-title">ADAMTS-13 rapidly cleaves newly secreted ultralarge von Willebrand factor multimers on the endothelial surface under flowing conditions</span>. <span class="ref-journal">Blood</span>. 2002;<span class="ref-vol">100</span>(<span class="ref-iss">12</span>):4033-4039. [<a href="/pubmed/12393397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=ADAMTS-13+rapidly+cleaves+newly+secreted+ultralarge+von+Willebrand+factor+multimers+on+the+endothelial+surface+under+flowing+conditions&amp;author=JF+Dong&amp;author=JL+Moake&amp;author=L+Nolasco&amp;volume=100&amp;issue=12&amp;publication_year=2002&amp;pages=4033-4039&amp;pmid=12393397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="mixed-citation">Moake JL, Rudy CK, Troll JH, et al. 
<span class="ref-title">Unusually large plasma factor VIII:von Willebrand factor multimers in chronic relapsing thrombotic thrombocytopenic purpura</span>. <span class="ref-journal">N Engl J Med</span>. 1982;<span class="ref-vol">307</span>(<span class="ref-iss">23</span>):1432-1435. [<a href="/pubmed/6813740" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Unusually+large+plasma+factor+VIII:von+Willebrand+factor+multimers+in+chronic+relapsing+thrombotic+thrombocytopenic+purpura&amp;author=JL+Moake&amp;author=CK+Rudy&amp;author=JH+Troll&amp;volume=307&amp;issue=23&amp;publication_year=1982&amp;pages=1432-1435&amp;pmid=6813740&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="mixed-citation">Budde U, Schneppenheim R.
<span class="ref-title">Interactions of von Willebrand factor and ADAMTS13 in von Willebrand disease and thrombotic thrombocytopenic purpura</span>. <span class="ref-journal">Hamostaseologie</span>. 2014;<span class="ref-vol">34</span>(<span class="ref-iss">3</span>):215-225. [<a href="/pubmed/25010251" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hamostaseologie&amp;title=Interactions+of+von+Willebrand+factor+and+ADAMTS13+in+von+Willebrand+disease+and+thrombotic+thrombocytopenic+purpura&amp;author=U+Budde&amp;author=R+Schneppenheim&amp;volume=34&amp;issue=3&amp;publication_year=2014&amp;pages=215-225&amp;pmid=25010251&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="mixed-citation">Sonneveld MA, de Maat MP, Portegies ML, et al. 
<span class="ref-title">Low ADAMTS13 activity is associated with an increased risk of ischemic stroke</span>. <span class="ref-journal">Blood</span>. 2015;<span class="ref-vol">126</span>(<span class="ref-iss">25</span>):2739-2746. [<a href="/pubmed/26511134" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Low+ADAMTS13+activity+is+associated+with+an+increased+risk+of+ischemic+stroke&amp;author=MA+Sonneveld&amp;author=MP+de+Maat&amp;author=ML+Portegies&amp;volume=126&amp;issue=25&amp;publication_year=2015&amp;pages=2739-2746&amp;pmid=26511134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="mixed-citation">Stoll M, Rühle F, Witten A, et al. 
<span class="ref-title">Rare Variants in the ADAMTS13 Von Willebrand Factor-Binding Domain Contribute to Pediatric Stroke</span>. <span class="ref-journal">Circ Cardiovasc Genet</span>. 2016;<span class="ref-vol">9</span>(<span class="ref-iss">4</span>):357-367. [<a href="/pubmed/27412500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Rare+Variants+in+the+ADAMTS13+Von+Willebrand+Factor-Binding+Domain+Contribute+to+Pediatric+Stroke&amp;author=M+Stoll&amp;author=F+Rühle&amp;author=A+Witten&amp;volume=9&amp;issue=4&amp;publication_year=2016&amp;pages=357-367&amp;pmid=27412500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="mixed-citation">Bongers TN, de Bruijne EL, Dippel DW, et al. 
<span class="ref-title">Lower levels of ADAMTS13 are associated with cardiovascular disease in young patients</span>. <span class="ref-journal">Atherosclerosis</span>. 2009;<span class="ref-vol">207</span>(<span class="ref-iss">1</span>):250-254. [<a href="/pubmed/19439298" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Lower+levels+of+ADAMTS13+are+associated+with+cardiovascular+disease+in+young+patients&amp;author=TN+Bongers&amp;author=EL+de+Bruijne&amp;author=DW+Dippel&amp;volume=207&amp;issue=1&amp;publication_year=2009&amp;pages=250-254&amp;pmid=19439298&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="mixed-citation">Sonneveld MA, Franco OH, Ikram MA, et al. 
<span class="ref-title">Von Willebrand Factor, ADAMTS13, and the Risk of Mortality: The Rotterdam Study</span>. <span class="ref-journal">Arterioscler Thromb Vasc Biol</span>. 2016;<span class="ref-vol">36</span>(<span class="ref-iss">12</span>):2446-2451. [<a href="/pubmed/27737864" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler+Thromb+Vasc+Biol&amp;title=Von+Willebrand+Factor,+ADAMTS13,+and+the+Risk+of+Mortality:+The+Rotterdam+Study&amp;author=MA+Sonneveld&amp;author=OH+Franco&amp;author=MA+Ikram&amp;volume=36&amp;issue=12&amp;publication_year=2016&amp;pages=2446-2451&amp;pmid=27737864&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="mixed-citation">Muia J, Zhu J, Gupta G, et al. 
<span class="ref-title">Allosteric activation of ADAMTS13 by von Willebrand factor</span>. <span class="ref-journal">Proc Natl Acad Sci USA</span>. 2014;<span class="ref-vol">111</span>(<span class="ref-iss">52</span>):18584-18589. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4284596/">PMC free article</a>]</span> [<a href="/pubmed/25512528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Allosteric+activation+of+ADAMTS13+by+von+Willebrand+factor&amp;author=J+Muia&amp;author=J+Zhu&amp;author=G+Gupta&amp;volume=111&amp;issue=52&amp;publication_year=2014&amp;pages=18584-18589&amp;pmid=25512528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="mixed-citation">Desch KC, Ozel AB, Siemieniak D, et al. 
<span class="ref-title">Linkage analysis identifies a locus for plasma von Willebrand factor undetected by genome-wide association</span>. <span class="ref-journal">Proc Natl Acad Sci USA</span>. 2013;<span class="ref-vol">110</span>(<span class="ref-iss">2</span>):588-593. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3545809/">PMC free article</a>]</span> [<a href="/pubmed/23267103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Linkage+analysis+identifies+a+locus+for+plasma+von+Willebrand+factor+undetected+by+genome-wide+association&amp;author=KC+Desch&amp;author=AB+Ozel&amp;author=D+Siemieniak&amp;volume=110&amp;issue=2&amp;publication_year=2013&amp;pages=588-593&amp;pmid=23267103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="mixed-citation">Kokame K, Sakata T, Kokubo Y, Miyata T.
<span class="ref-title">von Willebrand factor-to-ADAMTS13 ratio increases with age in a Japanese population</span>. <span class="ref-journal">J Thromb Haemost</span>. 2011;<span class="ref-vol">9</span>(<span class="ref-iss">7</span>):1426-1428. [<a href="/pubmed/21535397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=von+Willebrand+factor-to-ADAMTS13+ratio+increases+with+age+in+a+Japanese+population&amp;author=K+Kokame&amp;author=T+Sakata&amp;author=Y+Kokubo&amp;author=T+Miyata&amp;volume=9&amp;issue=7&amp;publication_year=2011&amp;pages=1426-1428&amp;pmid=21535397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="mixed-citation">Kilercik M, Coskun A, Serteser M, Inan D, Unsal I.
<span class="ref-title">Biological variations of ADAMTS13 and von Willebrand factor in human adults</span>. <span class="ref-journal">Biochem Med (Zagreb)</span>. 2014;<span class="ref-vol">24</span>(<span class="ref-iss">1</span>):138-145. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3936977/">PMC free article</a>]</span> [<a href="/pubmed/24627722" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Med+(Zagreb)&amp;title=Biological+variations+of+ADAMTS13+and+von+Willebrand+factor+in+human+adults&amp;author=M+Kilercik&amp;author=A+Coskun&amp;author=M+Serteser&amp;author=D+Inan&amp;author=I+Unsal&amp;volume=24&amp;issue=1&amp;publication_year=2014&amp;pages=138-145&amp;pmid=24627722&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="mixed-citation">de Vries PS, Boender J, Sonneveld MA, et al. 
<span class="ref-title">Genetic variants in the ADAMTS13 and SUPT3H genes are associated with ADAMTS13 activity</span>. <span class="ref-journal">Blood</span>. 2015;<span class="ref-vol">125</span>(<span class="ref-iss">25</span>):3949-3955. [<a href="/pubmed/25934476" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Genetic+variants+in+the+ADAMTS13+and+SUPT3H+genes+are+associated+with+ADAMTS13+activity&amp;author=PS+de+Vries&amp;author=J+Boender&amp;author=MA+Sonneveld&amp;volume=125&amp;issue=25&amp;publication_year=2015&amp;pages=3949-3955&amp;pmid=25934476&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="mixed-citation">Desch K, Li J, Kim S, et al. 
<span class="ref-title">Analysis of informed consent document utilization in a minimal-risk genetic study</span>. <span class="ref-journal">Ann Intern Med</span>. 2011;<span class="ref-vol">155</span>(<span class="ref-iss">5</span>):316-322. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3540806/">PMC free article</a>]</span> [<a href="/pubmed/21893624" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Intern+Med&amp;title=Analysis+of+informed+consent+document+utilization+in+a+minimal-risk+genetic+study&amp;author=K+Desch&amp;author=J+Li&amp;author=S+Kim&amp;volume=155&amp;issue=5&amp;publication_year=2011&amp;pages=316-322&amp;pmid=21893624&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="mixed-citation">Ma Q, Ozel AB, Ramdas S, et al. 
<span class="ref-title">Genetic variants in PLG, LPA, and SIGLEC 14 as well as smoking contribute to plasma plasminogen levels</span>. <span class="ref-journal">Blood</span>. 2014;<span class="ref-vol">124</span>(<span class="ref-iss">20</span>):3155-3164. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4231423/">PMC free article</a>]</span> [<a href="/pubmed/25208887" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Genetic+variants+in+PLG,+LPA,+and+SIGLEC+14+as+well+as+smoking+contribute+to+plasma+plasminogen+levels&amp;author=Q+Ma&amp;author=AB+Ozel&amp;author=S+Ramdas&amp;volume=124&amp;issue=20&amp;publication_year=2014&amp;pages=3155-3164&amp;pmid=25208887&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="mixed-citation">Ozel AB, McGee B, Siemieniak D, et al. 
<span class="ref-title">Genome-wide studies of von Willebrand factor propeptide identify loci contributing to variation in propeptide levels and von Willebrand factor clearance</span>. <span class="ref-journal">J Thromb Haemost</span>. 2016;<span class="ref-vol">14</span>(<span class="ref-iss">9</span>):1888-1898. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5035595/">PMC free article</a>]</span> [<a href="/pubmed/27359253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=Genome-wide+studies+of+von+Willebrand+factor+propeptide+identify+loci+contributing+to+variation+in+propeptide+levels+and+von+Willebrand+factor+clearance&amp;author=AB+Ozel&amp;author=B+McGee&amp;author=D+Siemieniak&amp;volume=14&amp;issue=9&amp;publication_year=2016&amp;pages=1888-1898&amp;pmid=27359253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="mixed-citation">Yang J, Lee SH, Goddard ME, Visscher PM.
<span class="ref-title">GCTA: a tool for genome-wide complex trait analysis</span>. <span class="ref-journal">Am J Hum Genet</span>. 2011;<span class="ref-vol">88</span>(<span class="ref-iss">1</span>):76-82. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;issue=1&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="mixed-citation">Gamer M, Lemon J, Fellows I, Singh P.
<span class="ref-title">irr: Various Coefficients of Interrater Reliability and Agreement</span>. 2012. 
<a href="https://cran.r-project.org/web/packages/irr/irr.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://cran.r-project.org/web/packages/irr/irr.pdf</a>.</span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="mixed-citation">R Core Team. <span class="ref-title">R: A Language and Environment for Statistical Computing</span>. Vienna, Austria: R Foundation for Statistical Computing; 2015 <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=R+Core+Team+.++R:+A+Language+and+Environment+for+Statistical+Computing+.++Vienna,+Austria+:++R+Foundation+for+Statistical+Computing+;++2015+" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="mixed-citation">Abecasis GR, Cherny SS, Cookson WO, Cardon LR.
<span class="ref-title">Merlin--rapid analysis of dense genetic maps using sparse gene flow trees</span>. <span class="ref-journal">Nat Genet</span>. 2002;<span class="ref-vol">30</span>(<span class="ref-iss">1</span>):97-101. [<a href="/pubmed/11731797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Merlin--rapid+analysis+of+dense+genetic+maps+using+sparse+gene+flow+trees&amp;author=GR+Abecasis&amp;author=SS+Cherny&amp;author=WO+Cookson&amp;author=LR+Cardon&amp;volume=30&amp;issue=1&amp;publication_year=2002&amp;pages=97-101&amp;pmid=11731797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="mixed-citation">Kang HM, Sul JH, Service SK, et al. 
<span class="ref-title">Variance component model to account for sample structure in genome-wide association studies</span>. <span class="ref-journal">Nat Genet</span>. 2010;<span class="ref-vol">42</span>(<span class="ref-iss">4</span>):348-354. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3092069/">PMC free article</a>]</span> [<a href="/pubmed/20208533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variance+component+model+to+account+for+sample+structure+in+genome-wide+association+studies&amp;author=HM+Kang&amp;author=JH+Sul&amp;author=SK+Service&amp;volume=42&amp;issue=4&amp;publication_year=2010&amp;pages=348-354&amp;pmid=20208533&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="mixed-citation">Yang J, Ferreira T, Morris AP, et al. 
<span class="ref-title">Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits</span>. <span class="ref-journal">Nat Genet</span>. 2012;<span class="ref-vol">44</span>(<span class="ref-iss">4</span>):369-375. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3593158/">PMC free article</a>]</span> [<a href="/pubmed/22426310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Conditional+and+joint+multiple-SNP+analysis+of+GWAS+summary+statistics+identifies+additional+variants+influencing+complex+traits&amp;author=J+Yang&amp;author=T+Ferreira&amp;author=AP+Morris&amp;volume=44&amp;issue=4&amp;publication_year=2012&amp;pages=369-375&amp;pmid=22426310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="mixed-citation">Willer CJ, Li Y, Abecasis GR.
<span class="ref-title">METAL: fast and efficient meta-analysis of genomewide association scans</span>. <span class="ref-journal">Bioinformatics</span>. 2010;<span class="ref-vol">26</span>(<span class="ref-iss">17</span>):2190-2191. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;issue=17&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="mixed-citation">Devlin B, Roeder K, Wasserman L.
<span class="ref-title">Genomic control, a new approach to genetic-based association studies</span>. <span class="ref-journal">Theor Popul Biol</span>. 2001;<span class="ref-vol">60</span>(<span class="ref-iss">3</span>):155-166. [<a href="/pubmed/11855950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Theor+Popul+Biol&amp;title=Genomic+control,+a+new+approach+to+genetic-based+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;author=L+Wasserman&amp;volume=60&amp;issue=3&amp;publication_year=2001&amp;pages=155-166&amp;pmid=11855950&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="mixed-citation">Pruim RJ, Welch RP, Sanna S, et al. 
<span class="ref-title">LocusZoom: regional visualization of genome-wide association scan results</span>. <span class="ref-journal">Bioinformatics</span>. 2010;<span class="ref-vol">26</span>(<span class="ref-iss">18</span>):2336-2337. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;volume=26&amp;issue=18&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="mixed-citation">Chelala C, Khan A, Lemoine NR.
<span class="ref-title">SNPnexus: a web database for functional annotation of newly discovered and public domain single nucleotide polymorphisms</span>. <span class="ref-journal">Bioinformatics</span>. 2009;<span class="ref-vol">25</span>(<span class="ref-iss">5</span>):655-661. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2647830/">PMC free article</a>]</span> [<a href="/pubmed/19098027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNPnexus:+a+web+database+for+functional+annotation+of+newly+discovered+and+public+domain+single+nucleotide+polymorphisms&amp;author=C+Chelala&amp;author=A+Khan&amp;author=NR+Lemoine&amp;volume=25&amp;issue=5&amp;publication_year=2009&amp;pages=655-661&amp;pmid=19098027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="mixed-citation">Adzhubei IA, Schmidt S, Peshkin L, et al. 
<span class="ref-title">A method and server for predicting damaging missense mutations</span>. <span class="ref-journal">Nat Methods</span>. 2010;<span class="ref-vol">7</span>(<span class="ref-iss">4</span>):248-249. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2855889/">PMC free article</a>]</span> [<a href="/pubmed/20354512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=A+method+and+server+for+predicting+damaging+missense+mutations&amp;author=IA+Adzhubei&amp;author=S+Schmidt&amp;author=L+Peshkin&amp;volume=7&amp;issue=4&amp;publication_year=2010&amp;pages=248-249&amp;pmid=20354512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="mixed-citation">Couzens AL, Knight JD, Kean MJ, et al. 
<span class="ref-title">Protein interaction network of the mammalian Hippo pathway reveals mechanisms of kinase-phosphatase interactions</span>. <span class="ref-journal">Sci Signal</span>. 2013;<span class="ref-vol">6</span>(<span class="ref-iss">302</span>):rs15. [<a href="/pubmed/24255178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Signal&amp;title=Protein+interaction+network+of+the+mammalian+Hippo+pathway+reveals+mechanisms+of+kinase-phosphatase+interactions&amp;author=AL+Couzens&amp;author=JD+Knight&amp;author=MJ+Kean&amp;volume=6&amp;issue=302&amp;publication_year=2013&amp;pages=rs15&amp;pmid=24255178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="mixed-citation">Pitchard JK.
<span class="ref-title">eQTL browser</span>. 2013. 
<a href="http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/</a>. Accessed 6 June 2017. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Pitchard+JK+
+eQTL+browser+.++2013+
+http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/+.+Accessed+6+June+2017.+" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="mixed-citation">Westra HJ, Peters MJ, Esko T, et al. 
<span class="ref-title">Systematic identification of trans eQTLs as putative drivers of known disease associations</span>. <span class="ref-journal">Nat Genet</span>. 2013;<span class="ref-vol">45</span>(<span class="ref-iss">10</span>):1238-1243. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3991562/">PMC free article</a>]</span> [<a href="/pubmed/24013639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Systematic+identification+of+trans+eQTLs+as+putative+drivers+of+known+disease+associations&amp;author=HJ+Westra&amp;author=MJ+Peters&amp;author=T+Esko&amp;volume=45&amp;issue=10&amp;publication_year=2013&amp;pages=1238-1243&amp;pmid=24013639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="mixed-citation">Barbalic M, Dupuis J, Dehghan A, et al. 
<span class="ref-title">Large-scale genomic studies reveal central role of ABO in sP-selectin and sICAM-1 levels</span>. <span class="ref-journal">Hum Mol Genet</span>. 2010;<span class="ref-vol">19</span>(<span class="ref-iss">9</span>):1863-1872. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850624/">PMC free article</a>]</span> [<a href="/pubmed/20167578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Large-scale+genomic+studies+reveal+central+role+of+ABO+in+sP-selectin+and+sICAM-1+levels&amp;author=M+Barbalic&amp;author=J+Dupuis&amp;author=A+Dehghan&amp;volume=19&amp;issue=9&amp;publication_year=2010&amp;pages=1863-1872&amp;pmid=20167578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="mixed-citation">Chion CK, Doggen CJ, Crawley JT, Lane DA, Rosendaal FR.
<span class="ref-title">ADAMTS13 and von Willebrand factor and the risk of myocardial infarction in men</span>. <span class="ref-journal">Blood</span>. 2007;<span class="ref-vol">109</span>(<span class="ref-iss">5</span>):1998-2000. [<a href="/pubmed/17053057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=ADAMTS13+and+von+Willebrand+factor+and+the+risk+of+myocardial+infarction+in+men&amp;author=CK+Chion&amp;author=CJ+Doggen&amp;author=JT+Crawley&amp;author=DA+Lane&amp;author=FR+Rosendaal&amp;volume=109&amp;issue=5&amp;publication_year=2007&amp;pages=1998-2000&amp;pmid=17053057&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="mixed-citation">Semlali A, Witoled C, Alanazi M, Rouabhia M.
<span class="ref-title">Whole cigarette smoke increased the expression of TLRs, HBDs, and proinflammory cytokines by human gingival epithelial cells through different signaling pathways</span>. <span class="ref-journal">PLoS One</span>. 2012;<span class="ref-vol">7</span>(<span class="ref-iss">12</span>):e52614. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3532503/">PMC free article</a>]</span> [<a href="/pubmed/23300722" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Whole+cigarette+smoke+increased+the+expression+of+TLRs,+HBDs,+and+proinflammory+cytokines+by+human+gingival+epithelial+cells+through+different+signaling+pathways&amp;author=A+Semlali&amp;author=C+Witoled&amp;author=M+Alanazi&amp;author=M+Rouabhia&amp;volume=7&amp;issue=12&amp;publication_year=2012&amp;pages=e52614&amp;pmid=23300722&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="mixed-citation">Barua RS, Ambrose JA.
<span class="ref-title">Mechanisms of coronary thrombosis in cigarette smoke exposure</span>. <span class="ref-journal">Arterioscler Thromb Vasc Biol</span>. 2013;<span class="ref-vol">33</span>(<span class="ref-iss">7</span>):1460-1467. [<a href="/pubmed/23685556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler+Thromb+Vasc+Biol&amp;title=Mechanisms+of+coronary+thrombosis+in+cigarette+smoke+exposure&amp;author=RS+Barua&amp;author=JA+Ambrose&amp;volume=33&amp;issue=7&amp;publication_year=2013&amp;pages=1460-1467&amp;pmid=23685556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="mixed-citation">Al-Awadhi AM, Jadaon MM, Alsayegh FA, Al-Sharrah SK.
<span class="ref-title">Smoking, von Willebrand factor and ADAMTS-13 in healthy males</span>. <span class="ref-journal">Scand J Clin Lab Invest</span>. 2012;<span class="ref-vol">72</span>(<span class="ref-iss">8</span>):614-618. [<a href="/pubmed/23020229" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Scand+J+Clin+Lab+Invest&amp;title=Smoking,+von+Willebrand+factor+and+ADAMTS-13+in+healthy+males&amp;author=AM+Al-Awadhi&amp;author=MM+Jadaon&amp;author=FA+Alsayegh&amp;author=SK+Al-Sharrah&amp;volume=72&amp;issue=8&amp;publication_year=2012&amp;pages=614-618&amp;pmid=23020229&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="mixed-citation">Balasubramaniyam N, Kolte D, Palaniswamy C, et al. 
<span class="ref-title">Predictors of in-hospital mortality and acute myocardial infarction in thrombotic thrombocytopenic purpura</span>. <span class="ref-journal">Am J Med</span>. 2013;<span class="ref-vol">126</span>(<span class="ref-iss">11</span>):1016.e1-1016.e7.  [<a href="/pubmed/23993262" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med&amp;title=Predictors+of+in-hospital+mortality+and+acute+myocardial+infarction+in+thrombotic+thrombocytopenic+purpura&amp;author=N+Balasubramaniyam&amp;author=D+Kolte&amp;author=C+Palaniswamy&amp;volume=126&amp;issue=11&amp;publication_year=2013&amp;pages=1016.e1-1016.e7&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="mixed-citation">Yevshin I, Sharipov R, Valeev T, Kel A, Kolpakov F.
<span class="ref-title">GTRD: a database of transcription factor binding sites identified by ChIP-seq experiments</span>. <span class="ref-journal">Nucleic Acids Res</span>. 2016;<span class="ref-vol">45</span>(<span class="ref-iss">D1</span>):D61-D67. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5210645/">PMC free article</a>]</span> [<a href="/pubmed/27924024" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=GTRD:+a+database+of+transcription+factor+binding+sites+identified+by+ChIP-seq+experiments&amp;author=I+Yevshin&amp;author=R+Sharipov&amp;author=T+Valeev&amp;author=A+Kel&amp;author=F+Kolpakov&amp;volume=45&amp;issue=D1&amp;publication_year=2016&amp;pages=D61-D67&amp;pmid=27924024&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="mixed-citation">Edwards NC, Hing ZA, Perry A, et al. 
<span class="ref-title">Characterization of coding synonymous and non-synonymous variants in ADAMTS13 using ex vivo and in silico approaches</span>. <span class="ref-journal">PLoS One</span>. 2012;<span class="ref-vol">7</span>(<span class="ref-iss">6</span>):e38864. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3387200/">PMC free article</a>]</span> [<a href="/pubmed/22768050" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Characterization+of+coding+synonymous+and+non-synonymous+variants+in+ADAMTS13+using+ex+vivo+and+in+silico+approaches&amp;author=NC+Edwards&amp;author=ZA+Hing&amp;author=A+Perry&amp;volume=7&amp;issue=6&amp;publication_year=2012&amp;pages=e38864&amp;pmid=22768050&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="mixed-citation">Plaimauer B, Fuhrmann J, Mohr G, et al. 
<span class="ref-title">Modulation of ADAMTS13 secretion and specific activity by a combination of common amino acid polymorphisms and a missense mutation</span>. <span class="ref-journal">Blood</span>. 2006;<span class="ref-vol">107</span>(<span class="ref-iss">1</span>):118-125. [<a href="/pubmed/16160007" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Modulation+of+ADAMTS13+secretion+and+specific+activity+by+a+combination+of+common+amino+acid+polymorphisms+and+a+missense+mutation&amp;author=B+Plaimauer&amp;author=J+Fuhrmann&amp;author=G+Mohr&amp;volume=107&amp;issue=1&amp;publication_year=2006&amp;pages=118-125&amp;pmid=16160007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5728318&amp;issue-id=302983&amp;journal-id=3318&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140517546355920title">Key Points</a></li><li><a href="#idm140517536551632title">Abstract</a></li><li><a href="#idm140517538828064title">Visual Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Materials and methods</a></li><li><a href="#__sec12title">Results</a></li><li><a href="#__sec21title">Discussion</a></li><li><a href="#__sec22title">Supplementary Material</a></li><li><a href="#idm140517536530176title">Acknowledgments</a></li><li><a href="#__sec23title">Authorship</a></li><li><a href="#idm140517584997184title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Blood Advances</span> are provided here courtesy of <strong>The American Society of Hematology</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5728318/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5728318/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5728318/pdf/advances005629.pdf">PDF (1.4M)</a></li> | <li><a href="#" data-citationid="PMC5728318" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5728318%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5728318%2F&amp;text=Genetic%20variants%20in%20ADAMTS13%20as%20well%20as%20smoking%20are%20major%20determinants%20of%20plasma%20ADAMTS13%20levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5728318%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29296746" data-aiid="5728318" data-aid="5728318" data-iid="302983" data-domainid="3318" data-domain="bloodadv" data-accid="PMC5728318" data-md5="f67c952622bbed35206858cbf0106d4a"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:52:40-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE884819DD5A93B10000000007950449&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5728318%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=bloodadv&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5728318/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>