<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="jnephrol"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5967769/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-Wide Association Studies of Metabolites in Patients with CKD Identify Multiple Loci and Illuminate Tubular Transport Mechanisms</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5967769/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of the American Society of Nephrology : JASN"><meta name="citation_title" content="Genome-Wide Association Studies of Metabolites in Patients with CKD Identify Multiple Loci and Illuminate Tubular Transport Mechanisms"><meta name="citation_authors" content="Yong Li, Peggy Sekula, Matthias Wuttke, Judith Wahrheit, Birgit Hausknecht, Ulla T. Schultheiss, Wolfram Gronwald, Pascal Schlosser, Sara Tucci, Arif B. Ekici, Ute Spiekerkoetter, Florian Kronenberg, Kai-Uwe Eckardt, Peter J. Oefner, Anna Köttgen, the GCKD Investigators"><meta name="citation_date" content="May 2018"><meta name="citation_issue" content="5"><meta name="citation_volume" content="29"><meta name="citation_firstpage" content="1513"><meta name="citation_doi" content="10.1681/ASN.2017101099"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5967769/?report=abstract"><meta name="citation_pmid" content="29545352"><meta name="DC.Title" content="Genome-Wide Association Studies of Metabolites in Patients with CKD Identify Multiple Loci and Illuminate Tubular Transport Mechanisms"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Society of Nephrology"><meta name="DC.Contributor" content="Yong Li"><meta name="DC.Contributor" content="Peggy Sekula"><meta name="DC.Contributor" content="Matthias Wuttke"><meta name="DC.Contributor" content="Judith Wahrheit"><meta name="DC.Contributor" content="Birgit Hausknecht"><meta name="DC.Contributor" content="Ulla T. Schultheiss"><meta name="DC.Contributor" content="Wolfram Gronwald"><meta name="DC.Contributor" content="Pascal Schlosser"><meta name="DC.Contributor" content="Sara Tucci"><meta name="DC.Contributor" content="Arif B. Ekici"><meta name="DC.Contributor" content="Ute Spiekerkoetter"><meta name="DC.Contributor" content="Florian Kronenberg"><meta name="DC.Contributor" content="Kai-Uwe Eckardt"><meta name="DC.Contributor" content="Peter J. Oefner"><meta name="DC.Contributor" content="Anna Köttgen"><meta name="DC.Contributor" content="the GCKD Investigators"><meta name="DC.Date" content="2018 May"><meta name="DC.Identifier" content="10.1681/ASN.2017101099"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-Wide Association Studies of Metabolites in Patients with CKD Identify Multiple Loci and Illuminate Tubular Transport Mechanisms"><meta property="og:type" content="article"><meta property="og:description" content="Background The kidneys have a central role in the generation, turnover, transport, and excretion of metabolites, and these functions can be altered in CKD. Genetic studies of metabolite concentrations can identify proteins performing these functions.Methods ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5967769/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-jnephrol.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5967769/epub/">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A53CCDD5B3CA100000000029E0183.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5967769%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/612/">J Am Soc Nephrol</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/313532/">v.29(5); 2018 May</a></li><li class="accid">PMC5967769</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-jnephrol.gif" alt="Logo of jnephrol" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://jasn.asnjournals.org/" target="pmc_ext" ref="reftype=publisher&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Am Soc Nephrol</a></span>. 2018 May; 29(5): 1513–1524. </span></div><div><span class="fm-vol-iss-date">Published online 2018 Mar 15. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1681%2FASN.2017101099" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1681/ASN.2017101099</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5967769</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29545352">29545352</a></div></div></div></div><h1 class="content-title">Genome-Wide Association Studies of Metabolites in Patients with CKD Identify Multiple Loci and Illuminate Tubular Transport Mechanisms</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086555600816" co-class="co-affbox">Yong Li</a>,<sup>1</sup> <a href="/pubmed/?term=Sekula%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086605472880" co-class="co-affbox">Peggy Sekula</a>,<sup>1</sup> <a href="/pubmed/?term=Wuttke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086605071216" co-class="co-affbox">Matthias Wuttke</a>,<sup>1</sup> <a href="/pubmed/?term=Wahrheit%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086564356688" co-class="co-affbox">Judith Wahrheit</a>,<sup>2</sup> <a href="/pubmed/?term=Hausknecht%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086563588432" co-class="co-affbox">Birgit Hausknecht</a>,<sup>3</sup> <a href="/pubmed/?term=Schultheiss%20UT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086564042768" co-class="co-affbox">Ulla T. Schultheiss</a>,<sup>1</sup> <a href="/pubmed/?term=Gronwald%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086551459840" co-class="co-affbox">Wolfram Gronwald</a>,<sup>4</sup> <a href="/pubmed/?term=Schlosser%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086564580128" co-class="co-affbox">Pascal Schlosser</a>,<sup>1</sup> <a href="/pubmed/?term=Tucci%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086564367488" co-class="co-affbox">Sara Tucci</a>,<sup>5</sup> <a href="/pubmed/?term=Ekici%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086559944400" co-class="co-affbox">Arif B. Ekici</a>,<sup>6</sup> <a href="/pubmed/?term=Spiekerkoetter%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086561615264" co-class="co-affbox">Ute Spiekerkoetter</a>,<sup>5</sup> <a href="/pubmed/?term=Kronenberg%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086559499584" co-class="co-affbox">Florian Kronenberg</a>,<sup>7</sup> <a href="/pubmed/?term=Eckardt%20KU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086564981616" co-class="co-affbox">Kai-Uwe Eckardt</a>,<sup>3</sup> <a href="/pubmed/?term=Oefner%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086554862176" co-class="co-affbox">Peter J. Oefner</a>,<sup>4</sup> <a href="/pubmed/?term=K%26%23x000f6%3Bttgen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352" class="affpopup" co-rid="_co_idm140086555954352" co-class="co-affbox">Anna Köttgen</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1</sup> and  the GCKD Investigators</div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140086555600816"><h3 class="no_margin">Yong Li</h3><p><sup>1</sup>Institute of Genetic Epidemiology, Department of Biometry, Epidemiology, and Medical Bioinformatics, and</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Yong Li</a></div></div><div id="_co_idm140086605472880"><h3 class="no_margin">Peggy Sekula</h3><p><sup>1</sup>Institute of Genetic Epidemiology, Department of Biometry, Epidemiology, and Medical Bioinformatics, and</p><div>Find articles by <a href="/pubmed/?term=Sekula%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Peggy Sekula</a></div></div><div id="_co_idm140086605071216"><h3 class="no_margin">Matthias Wuttke</h3><p><sup>1</sup>Institute of Genetic Epidemiology, Department of Biometry, Epidemiology, and Medical Bioinformatics, and</p><div>Find articles by <a href="/pubmed/?term=Wuttke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Matthias Wuttke</a></div></div><div id="_co_idm140086564356688"><h3 class="no_margin">Judith Wahrheit</h3><p><sup>2</sup>BIOCRATES Life Sciences Aktiengesellschaft, Innsbruck, Austria;</p><div>Find articles by <a href="/pubmed/?term=Wahrheit%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Judith Wahrheit</a></div></div><div id="_co_idm140086563588432"><h3 class="no_margin">Birgit Hausknecht</h3><p><sup>3</sup>Department of Nephrology and Hypertension and</p><div>Find articles by <a href="/pubmed/?term=Hausknecht%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Birgit Hausknecht</a></div></div><div id="_co_idm140086564042768"><h3 class="no_margin">Ulla T. Schultheiss</h3><p><sup>1</sup>Institute of Genetic Epidemiology, Department of Biometry, Epidemiology, and Medical Bioinformatics, and</p><div>Find articles by <a href="/pubmed/?term=Schultheiss%20UT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Ulla T. Schultheiss</a></div></div><div id="_co_idm140086551459840"><h3 class="no_margin">Wolfram Gronwald</h3><p><sup>4</sup>Institute of Functional Genomics, University of Regensburg, Regensburg, Germany; and</p><div>Find articles by <a href="/pubmed/?term=Gronwald%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Wolfram Gronwald</a></div></div><div id="_co_idm140086564580128"><h3 class="no_margin">Pascal Schlosser</h3><p><sup>1</sup>Institute of Genetic Epidemiology, Department of Biometry, Epidemiology, and Medical Bioinformatics, and</p><div>Find articles by <a href="/pubmed/?term=Schlosser%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Pascal Schlosser</a></div></div><div id="_co_idm140086564367488"><h3 class="no_margin">Sara Tucci</h3><p><sup>5</sup>Department of General Pediatrics, Center for Pediatrics and Adolescent Medicine, Faculty of Medicine and Medical Center, University of Freiburg, Freiburg, Germany;</p><div>Find articles by <a href="/pubmed/?term=Tucci%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Sara Tucci</a></div></div><div id="_co_idm140086559944400"><h3 class="no_margin">Arif B. Ekici</h3><p><sup>6</sup>Insitute of Human Genetics, University of Erlangen-Nürnberg, Erlangen, Germany;</p><div>Find articles by <a href="/pubmed/?term=Ekici%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Arif B. Ekici</a></div></div><div id="_co_idm140086561615264"><h3 class="no_margin">Ute Spiekerkoetter</h3><p><sup>5</sup>Department of General Pediatrics, Center for Pediatrics and Adolescent Medicine, Faculty of Medicine and Medical Center, University of Freiburg, Freiburg, Germany;</p><div>Find articles by <a href="/pubmed/?term=Spiekerkoetter%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Ute Spiekerkoetter</a></div></div><div id="_co_idm140086559499584"><h3 class="no_margin">Florian Kronenberg</h3><p><sup>7</sup>Division of Genetic Epidemiology, Department of Medical Genetics, Molecular and Clinical Pharmacology, Medical University of Innsbruck, Innsbruck, Austria</p><div>Find articles by <a href="/pubmed/?term=Kronenberg%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Florian Kronenberg</a></div></div><div id="_co_idm140086564981616"><h3 class="no_margin">Kai-Uwe Eckardt</h3><p><sup>3</sup>Department of Nephrology and Hypertension and</p><div>Find articles by <a href="/pubmed/?term=Eckardt%20KU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Kai-Uwe Eckardt</a></div></div><div id="_co_idm140086554862176"><h3 class="no_margin">Peter J. Oefner</h3><p><sup>4</sup>Institute of Functional Genomics, University of Regensburg, Regensburg, Germany; and</p><div>Find articles by <a href="/pubmed/?term=Oefner%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Peter J. Oefner</a></div></div><div id="_co_idm140086555954352"><h3 class="no_margin">Anna Köttgen</h3><p><sup>1</sup>Institute of Genetic Epidemiology, Department of Biometry, Epidemiology, and Medical Bioinformatics, and</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000f6%3Bttgen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29545352">Anna Köttgen</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140086557490736_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140086557490736_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140086557490736_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140086557490736_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Institute of Genetic Epidemiology, Department of Biometry, Epidemiology, and Medical Bioinformatics, and</div><div class="fm-affl" id="aff2" lang="en"><sup>5</sup>Department of General Pediatrics, Center for Pediatrics and Adolescent Medicine, Faculty of Medicine and Medical Center, University of Freiburg, Freiburg, Germany;</div><div class="fm-affl" id="aff3" lang="en"><sup>2</sup>BIOCRATES Life Sciences Aktiengesellschaft, Innsbruck, Austria;</div><div class="fm-affl" id="aff4" lang="en"><sup>3</sup>Department of Nephrology and Hypertension and</div><div class="fm-affl" id="aff5" lang="en"><sup>6</sup>Insitute of Human Genetics, University of Erlangen-Nürnberg, Erlangen, Germany;</div><div class="fm-affl" id="aff6" lang="en"><sup>4</sup>Institute of Functional Genomics, University of Regensburg, Regensburg, Germany; and</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Division of Genetic Epidemiology, Department of Medical Genetics, Molecular and Clinical Pharmacology, Medical University of Innsbruck, Innsbruck, Austria</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="afn1">Y.L. and P.S. contributed equally to this work.</div><div id="cor1"><strong>Correspondence:</strong> Dr. Anna Köttgen, Institute of Genetic Epidemiology, Medical Center – University of Freiburg, Hugstetter Straße 49, 79106 Freiburg, Germany. Email: <a href="mailto:dev@null" data-email="ed.grubierf-kinilkinu@negtteok.anna" class="oemail">ed.grubierf-kinilkinu@negtteok.anna</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140086557490736_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2017 Oct 16; Accepted 2018 Feb 9.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140086557490736_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2018 by the American Society of Nephrology</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>See editorial "<a href="/pmc/articles/PMC5967764/">Insights into CKD from Metabolite GWAS</a>" in volume 29 on&nbsp;page&nbsp;1349.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>This article has been <a href="/pmc/articles/PMC5967769/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplemental Data
</div>

<div><a href="/pmc/articles/PMC5967769/bin/supp_29_5_1513__index.html" data-ga-action="click_feat_suppl">supp_29_5_1513__index.html</a><span style="color:gray"> (1017 bytes)</span></div><div class="small guid">GUID:&nbsp;CC4612AD-72F6-4C38-9F0C-73F132A0C9BE</div>
<div><a href="/pmc/articles/PMC5967769/bin/ASN.2017101099SupplementaryData1.xlsx" data-ga-action="click_feat_suppl">ASN.2017101099SupplementaryData1.xlsx</a><span style="color:gray"> (111K)</span></div><div class="small guid">GUID:&nbsp;5F69E171-E0C6-45CA-BE0D-A9CCCF4DC047</div>
<div><a href="/pmc/articles/PMC5967769/bin/ASN.2017101099SupplementaryData2.pdf" data-ga-action="click_feat_suppl">ASN.2017101099SupplementaryData2.pdf</a><span style="color:gray"> (262K)</span></div><div class="small guid">GUID:&nbsp;5D920C84-C159-4B39-90E4-C66AA16B2821</div>
<div><a href="/pmc/articles/PMC5967769/bin/ASN.2017101099SupplementaryData3.pdf" data-ga-action="click_feat_suppl">ASN.2017101099SupplementaryData3.pdf</a><span style="color:gray"> (16K)</span></div><div class="small guid">GUID:&nbsp;3D04834F-3324-4E8B-A243-10A1B2A4250A</div>
</div></dd></dl></div><div id="idm140086555559040" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140086555559040title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first"><strong>Background</strong> The kidneys have a central role in the generation, turnover, transport, and excretion of metabolites, and these functions can be altered in CKD. Genetic studies of metabolite concentrations can identify proteins performing these functions.</p><p id="__p3"><strong>Methods</strong> We conducted genome-wide association studies and aggregate rare variant tests of the concentrations of 139 serum metabolites and 41 urine metabolites, as well as their pairwise ratios and fractional excretions in up to 1168 patients with CKD.</p><p id="__p4"><strong>Results</strong> After correction for multiple testing, genome-wide significant associations were detected for 25 serum metabolites, two urine metabolites, and 259 serum and 14 urinary metabolite ratios. These included associations already known from population-based studies. Additional findings included an association for the uremic toxin putrescine and variants upstream of an enzyme catalyzing the oxidative deamination of polyamines (<em>AOC1</em>, <em>P</em>-min=2.4×10<sup>−12</sup>), a relatively high carrier frequency (2%) for rare deleterious missense variants in <em>ACADM</em> that are collectively associated with serum ratios of medium-chain acylcarnitines (<em>P</em>-burden=6.6×10<sup>−16</sup>), and associations of a common variant in <em>SLC7A9</em> with several ratios of lysine to neutral amino acids in urine, including the lysine/glutamine ratio (<em>P</em>=2.2×10<sup>−23</sup>). The associations of this <em>SLC7A9</em> variant with ratios of lysine to specific neutral amino acids were much stronger than the association with lysine concentration alone. This finding is consistent with <em>SLC7A9</em> functioning as an exchanger of urinary cationic amino acids against specific intracellular neutral amino acids at the apical membrane of proximal tubular cells.</p><p id="__p5" class="p p-last"><strong>Conclusions</strong> Metabolomic indices of specific kidney functions in genetic studies may provide insight into human renal physiology.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">chronic kidney disease, genome-wide association studies, metabolites, renal tubular epithelial cells</span></div></div><div id="idm140086554395808" class="tsec sec headless whole_rhythm"><p id="__p6" class="p p-first">Metabolites are small molecules that represent intermediates or products of metabolic processes. They play important roles in energy generation, signaling, and the regulation of enzymatic reactions. The concentrations of these small molecules in cells and body fluids result from a balance of their intake and generation, their transport across compartments, and their breakdown and excretion. The kidneys play a central role in all of these processes, providing a rationale for metabolomics research in nephrology.<sup><sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851686">1</a></sup></sup> Previous studies have linked blood metabolite concentrations to common genetic markers across the genome and found a strong genetic component to the measured concentrations of many metabolites.<sup><sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851715">2</a></sup>–<sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851655">10</a></sup></sup> The implicated genes are often involved in balancing blood metabolite concentrations through metabolite generation (<em>e.g.</em>, encoding the rate-limiting enzyme), turnover, or excretion (<em>e.g.</em>, encoding metabolite transporters). In CKD, reduced GFR leads to elevated concentrations of many metabolites in blood. We therefore hypothesized that the presence of CKD represents a “challenge” model for metabolite handling. Under such a model, metabolites may be quantifiable that are usually below the limit of detection, and renal mechanisms that facilitate active metabolite reabsorption or excretion may be altered and could inform about tubular functions. Studying metabolite concentrations in patients with CKD may therefore allow for the detection of genetic loci influencing such processes.</p><p id="__p7">Only a few previous studies have investigated genetic influences on metabolite concentrations in urine,<sup><sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851706">11</a></sup>–<sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851672">13</a></sup></sup> which are of particular interest for the field of nephrology. Not only does the urine contain metabolites that are exclusively or predominantly generated in the kidneys and secreted into urine, but urinary metabolite concentrations also allow for the modeling of specific renal functions that may be affected in the presence of CKD. For example, genetic screens for the metabolite’s fractional excretion (FE) could identify tubular transport proteins for this metabolite, or genetic variants associated with the ratio of metabolites in the urine could identify substrates that are counter-transported across the apical tubular cell membrane.</p><p id="__p8" class="p">We therefore set out to test the following hypotheses: first, genetic investigations of metabolite concentrations in serum and urine of patients with CKD can replicate findings from previous population-based studies and identify additional genetic loci that may reflect the metabolic challenge posed by CKD. Second, modeling of kidney-specific functions on the basis of metabolite concentrations can be used to gain insights into tubular transport mechanisms and metabolic reactions of importance in CKD or detectable in its presence. To test these hypotheses in a proof-of-principle study, we carried out genome-wide association studies (GWAS) of metabolite concentrations in serum and urine as well as their FEs and pairwise ratios in up to 1168 patients with CKD participating in the German Chronic Kidney Disease (GCKD) study.<sup><sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851661">14</a></sup></sup></p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">Methods</h2><div id="s3" class="sec sec-first"><h3 id="s3title">Study Design and Participants</h3><p id="__p9" class="p p-first">The GCKD study is a prospective cohort study of patients with CKD treated by nephrologists. It was approved by the local ethics committees and registered in the national registry for clinical studies (DRKS 00003971). Between 2010 and 2012, 5217 eligible adult patients provided written consent and were enrolled into the study.<sup><sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851659">14</a></sup></sup> Patients were included if they had an eGFR between 30 and 60 ml/min per 1.73 m<sup>2</sup> or an eGFR&gt;60 ml/min per 1.73 m<sup>2</sup> and a urinary albumin-to-creatinine ratio (UACR) &gt;300 mg/g, albuminuria &gt;300 mg/d, a urinary protein-to-creatinine ratio &gt;500 mg/g, or proteinuria &gt;500 mg/d.<sup><sup><a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851705">15</a></sup></sup> Trained personnel obtained information on clinical data, socio-demographic factors, medical and family history, medications, and health-related quality of life. The leading cause of CKD was ascertained from the treating nephrologist. Moreover, biospecimens (plasma, serum, whole blood, spot urine) were collected in a standardized way at the enrollment visit, processed, and shipped frozen to a central laboratory for routine clinical chemistry and to a central biobank for future analyses following standard operating procedures.<sup><sup><a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851682">16</a></sup></sup> A complete description of study design and the recruited study population can be found elsewhere.<sup><sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851704">14</a></sup>,<sup><a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851689">15</a></sup></sup></p><p id="__p10" class="p p-last">For the current analysis, serum and urine specimens collected at baseline were selected for metabolite measurements from a subset of GCKD participants: all participants recruited from the Freiburg study center and additionally all GCKD patients from other study centers with autosomal-dominant polycystic kidney disease, focal segmental glomerulosclerosis, membranous nephropathy, membranoproliferative GN, rapid progressive GN, pauci-immune, and anti–glomerular basement membrane GN as the leading cause of CKD (<a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Material</a>).</p></div><div id="s4" class="sec"><h3 id="s4title">Genotyping and Imputation</h3><p id="__p11" class="p p-first">Genomic DNA was extracted from whole blood using an automated magnetic bead–based technology, quantified and normalized on a pipetting robot platform, and available for 5123 GCKD participants. Genotyping was conducted for 2,612,357 markers at the Helmholtz Center Munich using Omni2.5Exome BeadChip arrays (Illumina, GenomeStudio, Genotyping Module Version 1.9.4). Data cleaning was carried out separately for the Omni2.5 content and the Exome Chip content of the array. For the Omni2.5 content, data QC and cleaning followed the protocol of Anderson <em>et al</em>.<sup><sup><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851670">17</a></sup></sup> Per-individual QC steps included evaluation of call rate, sex check, heterozygosity, cryptic relatedness, and genetic ancestry. Altogether, 89 individuals were removed during the per-individual QC. In the per–single nucleotide polymorphism (SNP) QC steps, SNPs with &lt;96% call rate, deviating from Hardy–Weinberg equilibrium (<em>P</em> value &lt;1E−5), and those with duplicate positions were removed. Detailed steps of data cleaning are available in the <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Material</a>. The final dataset for the Omni2.5 genotypes contained 5034 individuals and 2,337,794 SNPs. The scripts from Anderson <em>et al</em>.<sup><sup><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851676">17</a></sup></sup> which are on the basis of Plink v1.90 and R programming language were used in the genotyping data cleaning steps.</p><p id="__p12">Cleaned Omni2.5 genotype data were then imputed using IMPUTE2 (v2.3.1)<sup><sup><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851684">18</a></sup></sup> following the Best Practices for Imputation on the IMPUTE2 website (<a href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html#best_practices" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/impute/impute_v2.html#best_practices</a>). SHAPEIT v2<sup><sup><a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851696">19</a></sup></sup> was used for strand alignment as well as for prephasing. The 1000 Genomes Project ALL haplotypes – Phase 3 integrated variant set was used as the reference. Imputation was performed chunk-wise with 5-Mbp sequences in each chunk. Initial imputation yielded genotypes for 81,698,995 variants. Filtering for imputation quality (info) value of &gt;0.8 and minor allele frequency (MAF) ≥1% led to 9,281,895 high-quality imputed markers that were subsequently used for GWAS.</p><p id="__p13" class="p p-last">Similarly, per-individual QC and per-SNP QC steps were performed for the Exome Chip content, with additional checks specific to exome chip data as in Guo <em>et al</em>.<sup><sup><a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851711">20</a></sup></sup> In the per-individual QC steps, 96 individuals were removed, and in the per-SNP QC, 3818 SNPs were removed on the basis of filtering for call rate &gt;95% and Hardy–Weinberg equilibrium <em>P</em> value ≥1E−5. More detailed information of the Exome Chip QC is found in the <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Material</a>. The cleaned Exome Chip dataset contained genotypes of 226,233 variants from 5027 individuals. These cleaned data were postprocessed by zCall<sup><sup><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851718">21</a></sup></sup> with a z-score threshold of six and then used in the Exome Chip association analyses.</p></div><div id="s5" class="sec"><h3 id="s5title">Metabolite Measurements</h3><p id="__p14" class="p p-first">Samples were shipped frozen to BIOCRATES Life Sciences AG in Innsbruck (Austria) and stored at −80°C until the time of analysis in 2014/2015 when aliquots were thawed, centrifuged, and the supernatant was used for analysis. Following the manufacturer’s protocol of the commercially available Absolute<em>IDQ</em> p180 Kit, serum and urine samples were handled in a uniform and standard way that includes a precipitation/extraction step. Using that kit, &gt;180 metabolites of different metabolite classes (amino acids, biogenic amines, acylcarnitines, [lyso-]phosphatidylcholines, sphingomyelins, and hexoses) were quantified from samples (input: 10 <em>µ</em>l).</p><p id="__p15">The quantitative assay was on the basis of PITC (phenylisothiocyanate)-derivatization in the presence of stable isotope-labeled internal standards, followed by flow injection analysis of acylcarnitines, lipids, and hexose and liquid chromatography of amino acids and biogenic amines, with detection accomplished by multiple reaction monitoring on a SCIEX 4000 QTrap mass spectrometer (SCIEX, Darmstadt, Germany). The experimental metabolomics measurement technique is described in detail in patents EP 1 897 014 B1 and EP 1 875 401 B1.<sup><sup><a href="#B22" rid="B22" class=" bibr popnode">22</a></sup>,<sup><a href="#B23" rid="B23" class=" bibr popnode">23</a></sup></sup></p><p id="__p16">Analytic performance was assessed by measuring replicates of human plasma samples spiked at three different concentrations as quality controls on each measurement plate along with the GCKD samples. Reference samples were within the predefined tolerances of the method. Accuracy of the measurements, determined by the accuracy of external calibrators, was in the normal range of the method (deviations from target ≤20%). Concentrations were normalized to the median of spiked reference samples to correct for possible batch effects.</p><p id="__p17" class="p p-last">Overall, serum measurements from 1200 individuals and urinary measurements from 1187 individuals could be obtained. All concentration values given are in micromolar (micromoles per liter). A detailed overview of the measured metabolites is provided in <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 1</a>. Measurements were further subjected to detailed quality control and data cleaning procedures (<a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Material</a>).</p></div><div id="s6" class="sec"><h3 id="s6title">Additional Variables</h3><p id="__p18" class="p p-first-last">Serum creatinine was measured in the GCKD study in a central laboratory using an IDMS traceable enzymatic assay (Creatinine plus, Roche). eGFR values were calculated using the creatinine-based Chronic Kidney Disease Epidemiology Collaboration formula.<sup><sup><a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851662">24</a></sup></sup> The UACR was calculated from urinary albumin and creatinine measurements, where creatinine was measured using the same assay as in serum and albumin with the ALBU-XS assay (Roche/Hitachi Diagnostics GmbH, Mannheim, Germany).</p></div><div id="s7" class="sec"><h3 id="s7title">GWAS and Exome Chip Analyses</h3><p id="__p19" class="p p-first">Cleaned metabolite data were merged with genotype data yielding a dataset for 139 serum metabolites from 1168 individuals and 41 urinary metabolites from 1155 individuals for subsequent genetic analyses (<a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Material</a>). The concentrations of the urinary metabolites were normalized using the probabilistic quotient<sup><sup><a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851712">25</a></sup></sup> to account for dilution. Within each matrix, pairwise ratios were calculated: Met A/Met B. For 37 of the metabolites, both serum and urinary concentrations were available, and the FE for each of these metabolites was calculated as (100×Met<sub>u</sub>×Crea<sub>s</sub>)/(Met<sub>s</sub>×Crea<sub>u</sub>) where Met<sub>u</sub> and Crea<sub>u</sub> are the urinary concentrations of the metabolite and of creatinine, and Met<sub>s</sub> and Crea<sub>s</sub> their serum concentrations. FEs and ratios were subjected to analyte level QC on the basis of evaluations of completeness, variability, and outliers, resulting in 34 FEs, 9591 serum metabolite ratios, and 820 urinary metabolite ratios for analyses. Before GWAS, all values (single metabolites, FEs, ratios) were transformed using rank-based inverse normal transformation.</p><p id="__p20">An in-house GWAS analysis pipeline was set up using SNPTEST (v2.5.2; <a href="https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html</a>) for association testing and GWAtoolbox (V2.2.4)<sup><sup><a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851675">26</a></sup></sup> for QC of the GWAS results. This pipeline was used for the GWAS of single serum and urinary metabolite concentrations and the FEs, using filtered imputed data of genotype dosages and assuming an additive genetic model. As in previous GWAS of metabolite concentrations,<sup><sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851697">4</a></sup>,<sup><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851680">7</a></sup></sup> age and sex were included as covariates in all analyses; analyses of serum metabolite concentrations additionally included adjustment for log(eGFR). After correcting for multiple testing using a Bonferroni procedure, statistical significance was defined as <em>P</em>&lt;3.6E−10 for 139 serum metabolites, <em>P</em>&lt;1.2E−09 for 41 urinary metabolites, and <em>P</em>&lt;1.5E−09 for 34 FEs; suggestive significance was defined as <em>P</em> value &lt;5.0E−08. Regional association plots of loci with significant associations were generated by LocusZoom (v1.3).<sup><sup><a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851660">27</a></sup></sup></p><p id="__p21">The proportion of variance in inverse-normal transformed metabolite trait (y) explained by an index SNP was calculated as <em>R</em><sup>2</sup>=<em>β</em><sup>2</sup>×var(SNP)/var(y), where <em>β</em> is the estimated effect of the SNP on y, var(SNP)=2×MAF<sup>SNP</sup>×(1−MAF<sup>SNP</sup>) according to the binomial distribution of the alleles.<sup><sup><a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851683">28</a></sup></sup></p><p id="__p22">Because of the large number of evaluated ratios, a two-step analytic approach was used, with the first step consisting of GWAS of the 9591 serum and 820 urinary metabolite ratios using the software OmicABEL<sup><sup><a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851692">29</a></sup></sup> on the basis of cleaned, genotyped markers. Adjustment variables are the same as in the single variant analysis by SNPTEST. The OmicABEL results were filtered for associations with chi square–association statistics &gt;30, which is equivalent to a <em>P</em> value &lt;4.3E−8. Serum and urinary metabolite ratios with at least one SNP that passed this filter were then subjected in a second step to the SNPTEST-based GWAS pipeline using imputed genotype data, as for single metabolites and FEs. The Bonferroni-corrected significance threshold was set at 5E−8/9591=5.2E−12 for serum metabolite ratios to account for the conduct of 9591 GWAS, and 5E−8/820=6.1E−11 for urinary metabolite ratios. Additionally, the GWAS results of ratios were filtered on the “<em>P</em>-gain,” defined as the minimum of the two <em>P</em> values of the single metabolites composing the ratio divided by the <em>P</em> value of the ratio.<sup><sup><a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851669">30</a></sup></sup> Only associations with a <em>P</em>-gain&gt;9591 in serum and <em>P</em>-gain&gt;820 in urine were further considered.</p><p id="__p23" class="p p-last">For the analysis of the exome chip data, two kinds of gene-based tests of single metabolites, FEs, and metabolite ratios were carried out using the R package seqMeta (v1.6.5) and using the same phenotypes and adjustment variables as for the single variant associations: a burden test and the sequence kernel association test. For both tests, only variants with MAF&lt;1% and likely to be functional (splicing, nonsynonymous, stopgain, and stoploss variants on the basis of the SNPInfo annotation file used in the CHARGE Consortium) were included.<sup><sup><a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851714">31</a></sup></sup> The test results were filtered and only genes with cumulative minor allele count ≥10 and with ≥2 variants per gene were retained. In total, 6924 genes met these filter criteria. To account for multiple testing, the overall significance threshold (0.05) was adjusted for the number of respective outcomes (per matrix), the number of genes (<em>n</em>=6924), and the number of performed tests (<em>n</em>=2). The significance thresholds were thus <em>P</em>&lt;2.6E−08 for 139 serum metabolites, <em>P</em>&lt;8.8E−08 for 41 urinary metabolites, <em>P</em>&lt;3.8E−10 for 9591 serum metabolite ratios, <em>P</em>&lt;4.4E−09 for 820 urinary metabolite ratios, and <em>P</em>&lt;1.1E−07 for 34 FEs. Suggestive statistical significance was defined as <em>P</em>&lt;1E−06.</p></div><div id="s8" class="sec sec-last"><h3 id="s8title">Power Considerations</h3><p id="__p24" class="p p-first-last">The power to detect associations between metabolites and genetic variants in a study of 1150 unrelated individuals such as ours is provided across a range of MAFs and effect sizes in <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 10</a>. For example, the power to detect an association at the genome-wide significance level of 5.0E−08 for genetic markers with MAF&gt;0.1 and an absolute effect size &gt;0.5 is &gt;80%.</p></div></div><div id="s9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s9title">Results</h2><p id="__p25" class="p p-first">Among the 1143 patients with CKD for whom both genetic information as well as serum and urinary metabolite measurements were available, the mean eGFR and the median UACR were 51.6 ml/min per 1.73 m<sup>2</sup> and 160.8 mg/g, respectively (<a href="/pmc/articles/PMC5967769/table/t1/" target="table" class="fig-table-link figpopup" rid-figpopup="t1" rid-ob="ob-t1" co-legend-rid=""><span>Table 1</span></a>). The mean age was 56 years, and 63% of the patients were male. An overview of all metabolites studied, including their abbreviations, limits of detection, and coefficients of variation, is provided in <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 1</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p26">Characteristics of the study sample</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140086565827104" class="xtable"><table frame="above" rules="groups" class="rendered small default_table"><thead><tr><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Characteristic</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">All Patients in the GCKD Study (<em>n</em>=5217)</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">GCKD Patients with Genetic and Metabolite Information (<em>n</em>=1143)</th></tr></thead><tbody><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Male sex, % (<em>n</em>)</td><td rowspan="1" colspan="1" valign="top" align="center">60 (3132)</td><td rowspan="1" colspan="1" valign="top" align="center">62.9 (719)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Mean age (SD), yr</td><td rowspan="1" colspan="1" valign="top" align="center">60.1 (11.96)</td><td rowspan="1" colspan="1" valign="top" align="center">56 (13.1)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Mean BMI (SD), kg/m<sup>2</sup><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">29.8 (5.97)</td><td rowspan="1" colspan="1" valign="top" align="center">28.6 (5.64)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Mean systolic BP (SD), mm Hg<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">139.5 (20.36)</td><td rowspan="1" colspan="1" valign="top" align="center">138.5 (19.11)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Diabetes, % (<em>n</em>)</td><td rowspan="1" colspan="1" valign="top" align="center">35.5 (1853)</td><td rowspan="1" colspan="1" valign="top" align="center">23 (263)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Mean serum creatinine (SD), mg/dl<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.5 (0.48)</td><td rowspan="1" colspan="1" valign="top" align="center">1.5 (0.52)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Mean eGFR (SD), ml/min per 1.73 m<sup>2</sup><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">49.5 (18.22)</td><td rowspan="1" colspan="1" valign="top" align="center">51.6 (21.12)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Mean urinary creatinine (SD), mg/dl<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">82.1 (55.58)</td><td rowspan="1" colspan="1" valign="top" align="center">79.4 (53.72)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Median urinary albumin-to-creatinine ratio (IQR), mg/g<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">50.9 (9.66–391.79)</td><td rowspan="1" colspan="1" valign="top" align="center">160.8 (24.35–872.27)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">CKD cause: primary glomerular disease, % (<em>n</em>)</td><td rowspan="1" colspan="1" valign="top" align="center">18.7 (978)</td><td rowspan="1" colspan="1" valign="top" align="center">52.4 (599)</td></tr></tbody></table></div><div id="largeobj_idm140086565827104" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5967769/table/t1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140086561000800"><p id="__p27" class="p p-first-last">The number of GCKD patients with genetic data and either serum measurements or urinary measurements is slightly higher: <em>n</em>=1168 and <em>n</em>=1155, respectively. BMI, body mass index; IQR, interquartile range.</p></div><div id="t1n1"><sup>a</sup>These variables were not available for some patients (proportion&lt;2%).</div></div></div><div id="s10" class="sec"><h3 id="s10title">Genetic Association Studies of Metabolites in Patients with CKD: Model Selection</h3><p id="__p29" class="p p-first">As a first step, we tested whether known genetic associations with metabolites could be replicated in a CKD population, and evaluated the potential influence of covariate selection and, specifically for urinary metabolites, the method used to account for dilution. We systematically selected variants with large genetic effects on metabolites from early publications of metabolite GWAS<sup><sup><a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851687">3</a></sup>,<sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851679">12</a></sup></sup> where both variant and metabolite were available in our data. These known genetic associations also achieved genome-wide significance among patients with CKD, as illustrated by associations between serum nonanoylcarnitine concentrations and an SNP in <em>ACADL</em> or serum butyrylcarnitine concentrations and an SNP in <em>ACADS</em>, as well as urinary lysine concentrations and an SNP in <em>SLC7A9</em> (<a href="/pmc/articles/PMC5967769/table/t2/" target="table" class="fig-table-link figpopup" rid-figpopup="t2" rid-ob="ob-t2" co-legend-rid=""><span>Table 2</span></a>). Adjustment of serum metabolite associations for eGFR led to increased statistical significance, mainly due to smaller SEM. Thus, genetic screens with metabolite concentrations in serum were adjusted for eGFR in subsequent analyses. In urine, the association between rs7247977 in <em>SLC7A9</em> and lysine concentrations was best detected when probabilistic quotient–normalization<sup><sup><a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851678">25</a></sup></sup> was used to account for urine dilution, rather than normalization by urinary creatinine concentrations. The association changed little upon further adjustment for UACR. Hence, subsequent GWAS for urinary metabolite concentrations were adjusted for age and sex.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p30">Evaluation of known genetic associations in a CKD population, exemplified for nonanoylcarnitine (serum) and lysine (urine)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140086555377712" class="xtable"><table frame="above" rules="groups" class="rendered small default_table"><thead><tr><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Metabolite<sup>a</sup></th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Normalization<sup>b</sup></th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Model</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">SNP</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Chr</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Position</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Ref/Effect Allele</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Freq Effect Allele</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Effect</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">SEM</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center"><em>P</em> Value</th></tr></thead><tbody><tr><td scope="col" rowspan="1" colspan="1" valign="top" align="left">Serum</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="2" scope="row" colspan="1" valign="top" align="left"> Nonanoylcarnitine, C9</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="left">adjustment for age, sex</td><td rowspan="2" colspan="1" valign="top" align="center">rs2286963</td><td rowspan="2" colspan="1" valign="top" align="center">2</td><td rowspan="2" colspan="1" valign="top" align="center">211060050</td><td rowspan="2" colspan="1" valign="top" align="center">T/G</td><td rowspan="2" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">0.041</td><td rowspan="1" colspan="1" valign="top" align="center">7.0E−29</td></tr><tr><td colspan="1" scope="row" rowspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="left">adjustment for age, sex, eGFR</td><td colspan="1" rowspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">0.036</td><td rowspan="1" colspan="1" valign="top" align="center">3.1E−35</td></tr><tr><td rowspan="2" scope="row" colspan="1" valign="top" align="left"> Butyrylcarnitine, C4</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="left">adjustment for age, sex</td><td rowspan="2" colspan="1" valign="top" align="center">rs2014355</td><td rowspan="2" colspan="1" valign="top" align="center">12</td><td rowspan="2" colspan="1" valign="top" align="center">121175524</td><td rowspan="2" colspan="1" valign="top" align="center">T/C</td><td rowspan="2" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">0.043</td><td rowspan="1" colspan="1" valign="top" align="center">1.6E−22</td></tr><tr><td colspan="1" scope="row" rowspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="left">adjustment for age, sex, eGFR</td><td colspan="1" rowspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.038</td><td rowspan="1" colspan="1" valign="top" align="center">1.4E−25</td></tr><tr><td scope="col" rowspan="1" colspan="1" valign="top" align="left">Urine</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="4" scope="row" colspan="1" valign="top" align="left"> Lysine</td><td rowspan="1" colspan="1" valign="top" align="left">creatinine-normalized</td><td rowspan="1" colspan="1" valign="top" align="left">adjustment for age, sex</td><td rowspan="4" colspan="1" valign="top" align="center">rs7247977</td><td rowspan="4" colspan="1" valign="top" align="center">19</td><td rowspan="4" colspan="1" valign="top" align="center">33358355</td><td rowspan="4" colspan="1" valign="top" align="center">T/C</td><td rowspan="4" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.042</td><td rowspan="1" colspan="1" valign="top" align="center">2.6E−07</td></tr><tr><td colspan="1" scope="row" rowspan="1" valign="top" align="left">creatinine-normalized</td><td rowspan="1" colspan="1" valign="top" align="left">adjustment for age, sex, lnUACR</td><td colspan="1" rowspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.041</td><td rowspan="1" colspan="1" valign="top" align="center">1.5E−08</td></tr><tr><td colspan="1" scope="row" rowspan="1" valign="top" align="left">pq-normalized</td><td rowspan="1" colspan="1" valign="top" align="left">adjustment for age, sex</td><td colspan="1" rowspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.042</td><td rowspan="1" colspan="1" valign="top" align="center">7.3E−12</td></tr><tr><td colspan="1" scope="row" rowspan="1" valign="top" align="left">pq-normalized</td><td rowspan="1" colspan="1" valign="top" align="left">adjustment for age, sex, lnUACR</td><td colspan="1" rowspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.042</td><td rowspan="1" colspan="1" valign="top" align="center">9.7E−12</td></tr></tbody></table></div><div id="largeobj_idm140086555377712" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5967769/table/t2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140086604582256"><p id="__p31" class="p p-first-last">Chr, chromosome; Ref, reference; Freq, frequency; NA, not available; pq, probabilistic quotient.</p></div><div id="t2n1"><sup>a</sup>Metabolite measurements were transformed before analysis using rank-based inverse normal transformation. Small differences in comparison to results from subsequent GWAS originate from differences in software and covariate adjustment.</div><div id="t2n2"><sup>b</sup>We compared the state-of-the-art normalization with urinary creatinine and the probabilistic quotient normalization<sup><sup><a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851694">25</a></sup></sup> to evaluate correction of urinary metabolite concentrations for urine dilution.</div></div></div></div><div id="s11" class="sec"><h3 id="s11title">GWAS and Burden Tests Identify Significant Associations with 25 Serum Metabolites</h3><p id="__p34" class="p p-first">We carried out GWAS of common SNPs (MAF&gt;0.01) with concentrations of 139 metabolites in serum. The genomic inflation factors ranged from 0.99 to 1.03, consistent with the absence of inflation from systematic errors. Multiple previously reported associations were identified at genome-wide significance (<em>P</em>&lt;3.6E−10, see Methods), such as genetic variants in <em>NAT8</em> and N-acetyl-ornithine (<em>P</em>=1.0E−62), <em>CPS1</em> and glycine (<em>P</em>=2.4E−20), <em>PRODH</em> and proline (<em>P</em>=1.8E−17), <em>ACADL</em> and nonanoylcarnitine (<em>P</em>=1.3E−35), and <em>FADS1</em> and phosphatidylcholines such as PC aa C36:5 (<em>P</em>=4.7E−14; <a href="/pmc/articles/PMC5967769/table/t3/" target="table" class="fig-table-link figpopup" rid-figpopup="t3" rid-ob="ob-t3" co-legend-rid=""><span>Table 3</span></a>). The 25 metabolites for which significant associations were found mapped into 13 loci.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p35">Common variants (MAF≥0.05) associated with serum metabolites at genome-wide significance (<em>P</em>&lt;3.6E−10)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140086562897696" class="xtable"><table frame="above" rules="groups" class="rendered small default_table"><thead><tr><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Metabolite</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">SNP</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Chr</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Position</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">NEA</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">EA</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Effect</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">SEM</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center"><em>P</em> Value</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">AF_EA</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Proportion of Variance Explained</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Genes</th></tr></thead><tbody><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">C10</td><td rowspan="1" colspan="1" valign="top" align="left">rs12134854</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">76137606</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="center">−0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">1.43E−12</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC44A5</em>(dist=60807),<em>ACADM</em>(dist=52437)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Ac-Orn</td><td rowspan="1" colspan="1" valign="top" align="left">rs375811360</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">73870324</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">−0.84</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">1.04E−62</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NAT8</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">C9</td><td rowspan="1" colspan="1" valign="top" align="left">rs3764913</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">211074909</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">1.34E−35</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ACADL</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Gly</td><td rowspan="1" colspan="1" valign="top" align="left">rs1047891</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">211540507</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">2.43E−20</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CPS1</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">C18:2</td><td rowspan="1" colspan="1" valign="top" align="left">rs12194000</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">110776012</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">−0.31</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">3.44E−11</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC22A16</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">C4</td><td rowspan="1" colspan="1" valign="top" align="left">rs146203232</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">160521885</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="center">−0.54</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">1.57E−18</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IGF2R</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Putrescine</td><td rowspan="1" colspan="1" valign="top" align="left">rs4725366</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">150522054</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">−0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">2.42E−12</td><td rowspan="1" colspan="1" valign="top" align="center">0.81</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TMEM176A</em>(dist=19846),<em>AOC1</em>(dist=27511)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">C0</td><td rowspan="1" colspan="1" valign="top" align="left">rs1171616</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">61468589</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">2.84E−15</td><td rowspan="1" colspan="1" valign="top" align="center">0.77</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC16A9</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC ae C38:4</td><td rowspan="1" colspan="1" valign="top" align="left">rs11320420</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61542006</td><td rowspan="1" colspan="1" valign="top" align="left">GA</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">−0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">2.35E−10</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MYRF</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC ae C38:5</td><td rowspan="1" colspan="1" valign="top" align="left">rs11320420</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61542006</td><td rowspan="1" colspan="1" valign="top" align="left">GA</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">−0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">2.79E−13</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MYRF</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC ae C36:5</td><td rowspan="1" colspan="1" valign="top" align="left">rs11320420</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61542006</td><td rowspan="1" colspan="1" valign="top" align="left">GA</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">−0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">1.47E−14</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MYRF</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC ae C38:5</td><td rowspan="1" colspan="1" valign="top" align="left">rs174537</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61552680</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="center">−0.32</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">9.45E−13</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MYRF</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC ae C36:5</td><td rowspan="1" colspan="1" valign="top" align="left">rs174537</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61552680</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="center">−0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">2.47E−14</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MYRF</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC aa C38:5</td><td rowspan="1" colspan="1" valign="top" align="left">rs174537</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61552680</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="center">−0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">1.37E−16</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MYRF</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC aa C38:4</td><td rowspan="1" colspan="1" valign="top" align="left">rs102275</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61557803</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="center">−0.40</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">9.76E−21</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TMEM258</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">lysoPC a C20:4</td><td rowspan="1" colspan="1" valign="top" align="left">rs102275</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61557803</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="center">−0.50</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">2.63E−32</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TMEM258</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC aa C36:4</td><td rowspan="1" colspan="1" valign="top" align="left">rs102274</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61557826</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="center">−0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">8.07E−21</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TMEM258</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC aa C36:3</td><td rowspan="1" colspan="1" valign="top" align="left">rs174560</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61581764</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">2.52E−10</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FADS1</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC aa C36:5</td><td rowspan="1" colspan="1" valign="top" align="left">rs174566</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61592362</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">−0.32</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">4.66E−14</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FADS1</em>(dist=7833)<em>,FADS2</em>(dist=3351)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">C4</td><td rowspan="1" colspan="1" valign="top" align="left">rs2014355</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">121175524</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">3.90E−27</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ACADS</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC aa C28:1</td><td rowspan="1" colspan="1" valign="top" align="left">rs7160525</td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">64232220</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">6.57E−15</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SGPP1</em>(dist=37464),<em>SYNE2</em>(dist=87463)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">SM (OH) C14:1</td><td rowspan="1" colspan="1" valign="top" align="left">rs112329286</td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">64239877</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">TTTTA</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">5.44E−12</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SGPP1</em>(dist=45121),<em>SYNE2</em>(dist=79806)</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">PC ae C32:1</td><td rowspan="1" colspan="1" valign="top" align="left">rs1077989</td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">67975822</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="center">−0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">1.97E−12</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TMEM229B</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Asn</td><td rowspan="1" colspan="1" valign="top" align="left">rs2282377</td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">104571820</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">1.23E−11</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ASPG</em></td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">Pro</td><td rowspan="1" colspan="1" valign="top" align="left">rs117935223</td><td rowspan="1" colspan="1" valign="top" align="center">22</td><td rowspan="1" colspan="1" valign="top" align="center">18911333</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.62</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">1.79E−17</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PRODH</em></td></tr></tbody></table></div><div id="largeobj_idm140086562897696" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5967769/table/t3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140086552636992"><p id="__p36" class="p p-first-last">Sample size: <em>n</em>=1168. Chr, chromosome; NEA, noneffect allele; EA, effect allele; AF_EA, allele frequency of effect allele; Genes, gene the SNP maps into or genes neighboring the SNP and the corresponding distances in base pairs.</p></div></div></div><p id="__p37">One locus, intergenic to <em>TMEM176A</em> and <em>AOC1</em>, showed a novel genome-wide significant association with concentration of serum putrescine (<em>P</em>=2.4E−12), a known uremic toxin (<a href="/pmc/articles/PMC5967769/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>).<sup><sup><a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851690">32</a></sup></sup> The effect of the G allele at the index SNP in this locus, rs4725366, on serum putrescine concentration was twice as large in individuals with eGFR&lt;30 ml/min per 1.73 m<sup>2</sup> compared with those with eGFR≥60 (−0.28 versus −0.60, <a href="/pmc/articles/PMC5967769/table/t4/" target="table" class="fig-table-link figpopup" rid-figpopup="t4" rid-ob="ob-t4" co-legend-rid=""><span>Table 4</span></a>). The relationship between the genetic effect size and eGFR, however, was not linear, and a statistical interaction term between SNP and eGFR category was not statistically significant (<em>P</em>=0.3). SNP rs4725366 is significantly associated with the expression of the neighboring <em>AOC1</em> gene (<em>P</em>=1.5e−22 in testis in the GTEx resource<sup><sup><a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851695">33</a></sup></sup> that does not contain eQTL information from kidney tissue). <em>AOC1</em> encodes amine oxidase copper-containing 1, an enzyme catalyzing the oxidative deamination of polyamines such as histamine, putrescine, and cadaverine.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC5967769/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140086564702832" class="figure"><a href="/pmc/articles/PMC5967769/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5967769_ASN.2017101099f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ASN.2017101099f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5967769/bin/ASN.2017101099f1.jpg"></a></div><div id="largeobj_idm140086564702832" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5967769/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC5967769/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p38">Regional association plot illustrates the presence of serum putrescine-associated SNPs upstream of AOC1 and downstream of TMEM176A on chromosome 7. <em>y</em> axis, negative log<sub>10</sub>(<em>P</em> value) from GWAS analysis; <em>x</em> axis, genomic position. Linkage disequilibrium with the index SNP rs4725949 (purple) in the 1000 Genomes Project EUR reference population is color-coded as detailed in the legend.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t4"><h3>Table 4.</h3><!--caption a7--><div class="caption"><p id="__p39">Effect of G allele of rs4725366 (<em>AOC1</em> locus) on serum putrescine concentrations stratified by eGFR</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140086555463024" class="xtable"><table frame="above" rules="groups" class="rendered small default_table"><thead><tr><th scope="col" rowspan="1" colspan="1" valign="top" align="center">eGFR Category (ml/min per 1.73 m<sup>2</sup>)</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center"><em>n</em></th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">Effect</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center">SEM</th><th scope="col" rowspan="1" colspan="1" valign="top" align="center"><em>P</em> Value</th></tr></thead><tbody><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">≥60</td><td rowspan="1" colspan="1" valign="top" align="center">293</td><td rowspan="1" colspan="1" valign="top" align="center">−0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">6.37E−03</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">45–59</td><td rowspan="1" colspan="1" valign="top" align="center">343</td><td rowspan="1" colspan="1" valign="top" align="center">−0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">4.16E−06</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">30–44</td><td rowspan="1" colspan="1" valign="top" align="center">391</td><td rowspan="1" colspan="1" valign="top" align="center">−0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">1.34E−03</td></tr><tr><td scope="row" rowspan="1" colspan="1" valign="top" align="left">&lt;30</td><td rowspan="1" colspan="1" valign="top" align="center">117</td><td rowspan="1" colspan="1" valign="top" align="center">−0.60</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">1.12E−04</td></tr></tbody></table></div><div id="largeobj_idm140086555463024" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5967769/table/t4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140086554830256"><p id="__p40" class="p p-first-last"><em>n</em>, sample size.</p></div></div></div><p id="__p41">A full table listing the index variant for metabolites with at least one suggestive association signal (<em>P</em>&lt;5E−08, see Methods) is shown in <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 2</a>. These include loci not significantly associated with blood metabolite concentrations in previous studies, such as the <em>PTPRN2</em> locus and concentrations of lysophosphatidylcholine with acyl residue C18:1 (<em>P</em>=1.7E−08). Because this locus also showed suggestive evidence of association with the blood concentrations of another lipid in a previous metabolite GWAS,<sup><sup><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851691">7</a></sup></sup> such associations are excellent candidates to achieve genome-wide significance in future, larger association studies.</p><p id="__p42" class="p p-last">We additionally examined the aggregate effect of rare deleterious variants within the same gene on serum metabolite concentrations. No gene passed the significance threshold for multiple testing (<em>P</em>&lt;2.6E−08, see Methods), but the burden of rare deleterious variants in four genes showed suggestive evidence for association (<em>P</em>&lt;1E−06) with at least one metabolite (<a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 3</a>). Among these was an association between serum concentrations of histidine and rare variants in the <em>HAL</em> gene (<em>P</em>-burden=7.1E−08), encoding histidine ammonia-lyase.<sup><sup><a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851674">34</a></sup></sup> Rare mutations in <em>HAL</em> can cause autosomal-recessive histidinemia (MIM #235800), and associations between rare genetic variants in <em>HAL</em> and blood histidine concentrations have previously been reported in a population-based study.<sup><sup><a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851666">9</a></sup></sup> In that study, 14 rare <em>HAL</em> variants were underlying the association signal with blood histidine concentrations, whereas in our study 17 such variants were identified (nine of which overlapped). These findings indicate that rare deleterious variants also influence metabolite concentrations in patients with CKD. Because additional and unique potentially deleterious variants are identified with each new study population examined, results in the suggestive significance range identified in our study likely contain other true positive findings that await discovery in future experimental or larger genetic association studies.</p></div><div id="s12" class="sec"><h3 id="s12title">GWAS and Burden Tests of Urinary Metabolites Confirm Population-Based Findings in CKD</h3><p id="__p43" class="p p-first-last">Next, we carried out GWAS of common SNPs and metabolite concentrations in urine. Among the 41 metabolites that passed quality control, significant associations were observed for two previously reported loci, <em>SLC7A9</em> and urinary lysine concentrations (<em>P</em>=4.7E−12) and <em>NAT8</em> and urinary N-acetylornithine concentrations (<em>P</em>=9.6E−12, <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 4</a>). These results were stable to additional adjustment for UACR. Suggestive associations were detected for several loci that were previously described as associated with the metabolite’s blood concentrations, such as <em>SLC22A1</em> and butyrylcarnitine.<sup><sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851713">10</a></sup></sup> The rare variant analyses did not lead to the identification of an aggregate effect of deleterious variants in any gene with <em>P</em>&lt;1E−06. The number of loci associated with urinary metabolite concentrations among patients with CKD in this study needs to be interpreted in light of the assay used to quantify metabolites: the BIOCRATES Absolute<em>IDQ</em> p180 Kit is a targeted assay originally developed to quantify blood metabolite concentrations, only a subset of which are detected in urine (<a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 1</a>).</p></div><div id="s13" class="sec"><h3 id="s13title">GWAS and Burden Tests of Serum Metabolite Ratios Highlight Variants Central to Lipid and Acylcarnitine Metabolism</h3><p id="__p44" class="p p-first">Metabolite concentrations can be used to model physiologic functions of the kidney. Specifically, ratios of metabolites can reflect not only the activity of enzymatic reactions, but also co- or counter-transport at the apical or basolateral membrane of tubular cells, resulting in synchronized changes of these metabolites in urine and blood, respectively. We therefore first evaluated the association between common genetic variants and metabolite ratios in serum. Genome-wide significant associations were identified between genetic variants and 259 metabolite ratios mapping into 11 loci (<a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 5</a>), many of which implicated known genes central to lipid biosynthesis. These analyses confirmed <em>FADS1</em> as a major player in the metabolism of phosphatidylcholines, <em>ACADM</em> and <em>ACADS</em> in metabolism of medium-chain acylcarnitines, <em>SPTLC3</em> in the synthesis of sphingomyelins, as well as <em>PDXDC1</em> in phospholipid metabolism.<sup><sup><a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851673">3</a></sup>,<sup><a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851699">35</a></sup></sup></p><p id="__p45" class="p">In the gene-based tests of rare variants on serum metabolite ratios, we identified an exome-wide significant association (<em>P</em>&lt;2.4E−10) for one gene, <em>ACADM</em>. Significant associations were observed for several metabolite ratios, with the lowest <em>P</em> value observed for the decanoylcarnitine/dodecanoylcarnitine ratio from the burden test (<em>P</em>=6.6E−16, <a href="/pmc/articles/PMC5967769/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a>). The low burden test <em>P</em> value, despite the modest sample size and low frequency of the evaluated deleterious variants (MAF&lt;0.01), suggested large and concordant effects of the contributing variants affecting the metabolite concentrations. Indeed, the four variants in <em>ACADM</em> that contributed to the test were all associated with increases in the decanoylcarnitine/dodecanoylcarnitine ratio (<a href="/pmc/articles/PMC5967769/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a>). <em>ACADM</em> encodes medium-chain acyl-CoA dehydrogenase, which catalyzes the initial reaction in the <em>β</em>-oxidation of C4 to C12 straight-chain acyl-CoAs. Mutations in the <em>ACADM</em> gene cause MCAD deficiency (MIM #201450), a condition that is part of newborn screening on the basis of acylcarnitine analysis in blood. Three of the four variants detected in our screen have been reported as cause of classic MCAD deficiency if present in the homozygous state (p.E43K, p.Y67H, p.K329E), confirming the functional and clinical relevance of these mutations. The mutation p.K329E gives rise to a severe disease phenotype, whereas the clinical picture is milder for the mutations p.E43K and p.Y67H, suggesting residual enzyme function. These observations from monogenic manifestations of the disease are consistent with the effect sizes observed for heterozygous carriers in our study, with p.K329E showing the largest effect on the decanoylcarnitine/dodecanoylcarnitine ratio. The last variant identified in our study, p.I364T, is yet of uncertain clinical significance (<a href="/pmc/articles/PMC5967769/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a>). Our findings reveal a relatively high carrier frequency of potentially deleterious <em>ACADM</em> mutations with large effects on metabolite concentrations in an adult population of European ancestry (23 of 1157=2.0%; 1.9% without p.I364T). This relatively high frequency is also observed among European ancestry individuals in the gnomAD database, suggesting that this finding is more widely applicable. Full results of all serum metabolite ratios associated at suggestive significance with rare variants in one or more genes are listed in <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 6</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC5967769/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140086561551536" class="figure"><a href="/pmc/articles/PMC5967769/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5967769_ASN.2017101099f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ASN.2017101099f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5967769/bin/ASN.2017101099f2.jpg"></a></div><div id="largeobj_idm140086561551536" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5967769/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC5967769/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p46">Rare coding variants in <em>ACADM</em> are cumulatively associated with the serum concentrations of acylcarnitine ratios. The upper panel contains a schematic representation of the <em>ACADM</em> gene and the mapping of the identified deleterious variants to exons. The table contains <em>P</em> values for the cumulative effect of the variants on the concentrations of the listed acylcarnitine ratios, as well as the association <em>P</em> values for the single variants that contribute to the gene-based tests (burden test and sequence kernel association test, see Methods). The allele frequencies of these variants in European individuals in gnomAD database are: 0.0032 for rs147559466, 0.00099 for rs121434280, 0.0061 for rs77931234, and 0.00049 for rs150710061. C10, decanoylcarnitine; C12, dodecanoylcarnitine; C14:1, Tetradecenoylcarnitine; cMAF, cumulative MAF; MAC, minor allele count; SKAT, Sequence Kernel Association Test.</p></div></div></div></div><div id="s14" class="sec"><h3 id="s14title">GWAS of Urine Metabolite Ratios Highlight Amino Acids Counter-Transported by SLC7A9</h3><p id="__p47" class="p p-first">There were several genetic loci with significant SNP associations with urinary metabolite ratios (<a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 7</a>). Interestingly, these associations delivered valuable insights into renal physiology: genetic variants in <em>SLC7A9</em> showed significant associations with lysine-containing ratios that were many orders of magnitude stronger than the association of the respective SNP with urinary lysine concentrations alone (<a href="/pmc/articles/PMC5967769/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure 3</span></a>). This is exemplified by the lysine/glutamine ratio (<em>P</em>=2.2E−23, <em>P</em>-gain=2.1E+11; see Methods), the lysine/tyrosine ratio (<em>P</em>=3.1E−22, <em>P</em>-gain=1.5E+10), and the lysine/threonine ratio (<em>P</em>=5.8E−22, <em>P</em>-gain=1.5E+10). Other significant lysine-containing ratios contained alanine, glycine, serine, asparagine, phenylalanine, and tryptophan. Of note, all of these amino acids carry no net electric charge at physiologic pH, consistent with the known role of SCL7A9 as an exchanger of cationic amino acids (lysine, arginine, and ornithine) and cystine in the renal tubular lumen against intracellular neutral amino acids (AA<sup>0</sup>) described in <em>in vitro</em> studies.<sup><sup><a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851657">36</a></sup>,<sup><a href="#B37" rid="B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851703">37</a></sup></sup> This finding illustrates the potential of modeling metabolite concentrations to uncover specific physiologic functions of the kidney. The concept is illustrated in <a href="/pmc/articles/PMC5967769/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure 3A</span></a>: individuals homozygous for the minor C allele at the intronic index SNP rs12460876 show higher urinary lysine/AA<sup>0</sup> ratios than those homozygous for the major T allele, indicating less efficient function of the encoded SLC7A9 transport protein. Modeling of the association between rs12460876 and all pairwise amino acid ratios in urine revealed specific lysine/AA<sup>0</sup> ratios with strong associations (<a href="/pmc/articles/PMC5967769/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure 3B</span></a>), representing a readout of human SLC7A9 function <em>in vivo</em>. Other amino acid ratios showed little (mostly ornithine-containing ratios) or no association, such as lysine/AA<sup>−</sup> ratios (aspartate, glutamate).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC5967769/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140086554222816" class="figure"><a href="/pmc/articles/PMC5967769/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5967769_ASN.2017101099f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ASN.2017101099f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5967769/bin/ASN.2017101099f3.jpg"></a></div><div id="largeobj_idm140086554222816" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5967769/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC5967769/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure 3.</a></div><!--caption a7--><div class="caption"><p id="__p48">Counter-transport of amino acids by SLC7A9 in the renal proximal tubule cell is revealed by the analysis of urinary metabolite ratios. The upper part (A) contains a schematic representation of SLC7A9 function in the proximal tubule, reabsorbing dibasic urinary amino acids such as lysine in exchange for intracellular neutral amino acids. Individuals homozygous for the minor C allele at the index SNP rs12460876 show higher lysine/neutral amino acid (AA<sup>0</sup>) ratios in the urine, indicating less efficient lysine reuptake at the brush border. Listed amino acids are restricted to the ones measured in this study. The lower panel (B) represents a heatmap of the strength of the association between genotype at intronic rs12460876 (scaled chi square statistics) and the evaluated pairwise metabolite ratios in urine. It highlights the AA<sup>0</sup> counter-transported against lysine <em>in vivo</em>. For comparison, lysine by itself has a scaled chi square statistic of 0.39, indicated by light orange color. ala, alanine; asn, asparagine; asp, aspartate; gln, glutamine; glu, glutamate; gly, glycine; his, histidine; ile, isoleucine; lys, lysine; orn, ornithine; phe, phenylalanine; pro, proline; ser, serine; thr, threonine; trp, tryptophan; tyr, tyrosine; val, valine.</p></div></div></div><p id="__p49" class="p p-last">The evaluation of the aggregate effect of rare variants on urinary metabolite ratios did not yield findings that achieved exome-wide significance (<em>P</em>&lt;4.4E−09). There were, however, biologically plausible findings of suggestive significance (<em>P</em>&lt;1E−06, <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 8</a>). For example, rare variants in <em>SLC36A2</em> were associated with the urinary glycine-to-serine ratio (<em>P</em>=4.0E−07). This matches the known transport capacity of the encoded transporter for glycine and its localization to the S1 segment of the proximal tubule.<sup><sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851665">13</a></sup>,<sup><a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851708">38</a></sup></sup> Transport capacity for serine has only been described for murine SLC36A2 so far<sup><sup><a href="#B39" rid="B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851701">39</a></sup></sup> and could now be tested experimentally for human SLC36A2. Another example is the association between rare variants in <em>PAH</em> and the phenylalanine/tyrosine ratio (<em>P</em>=5.9E−07), which matches the function of the encoded phenylalanine hydroxylase that catalyzes the rate-limiting step in phenylalanine catabolism, the hydroxylation of phenylalanine to tyrosine. Although an earlier study had reported the association between rare variants in <em>PAH</em> and blood phenylalanine concentrations,<sup><sup><a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851656">9</a></sup></sup> our findings highlight that modeling of enzyme function as the ratio of the enzyme’s substrate and product adds additional information. In addition, the examination of urine adds information, because the gene is exclusively expressed in liver and kidney, and urinary concentrations integrate the information of reactions occurring in the kidney. More generally, these findings suggest that the study of metabolite ratios in urine of patients with CKD in a larger study sample is a promising avenue for future research.</p></div><div id="s15" class="sec sec-last"><h3 id="s15title">GWAS and Burden Tests of Metabolite FEs</h3><p id="__p50" class="p p-first-last">Lastly, GWAS of common SNPs with the FEs of 34 metabolites yielded one finding of genome-wide significance, the association between variants in the <em>MLEC</em> gene at the <em>ACADS</em> locus and the FE of butyrylcarnitine (C4, <em>P</em>=2.8E−14; <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplemental Table 9</a>). However, these associations were not as significant as the ones with serum butyrylcarnitine concentration alone (<em>P</em>=3.9E−27, <a href="/pmc/articles/PMC5967769/table/t3/" target="table" class="fig-table-link figpopup" rid-figpopup="t3" rid-ob="ob-t3" co-legend-rid=""><span>Table 3</span></a>). No significant or suggestive associations (<em>P</em>&lt;1E−06) were identified for the aggregate effect of rare deleterious variants in any gene and the FEs of the 34 evaluated metabolites. Thus, investigation of the FEs of metabolites did not provide additional insights.</p></div></div><div id="s16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s16title">Discussion</h2><p id="__p51" class="p p-first">Our study has several main findings: first, genetic influences on metabolite concentrations in serum and urine from previous population-based studies<sup><sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851671">2</a></sup>–<sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851717">4</a></sup>,<sup><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851681">7</a></sup>–<sup><a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851667">9</a></sup>,<sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851700">11</a></sup>,<sup><a href="#B40" rid="B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851707">40</a></sup>,<sup><a href="#B41" rid="B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851677">41</a></sup></sup> translate to patients with CKD. Second, the study of patients with CKD revealed a novel association between concentrations of a uremic toxin, putrescine,<sup><sup><a href="#B42" rid="B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851668">42</a></sup></sup> and variation near the gene encoding amine oxidase copper-containing 1. Third, metabolite ratios are useful composite indices that reflect underlying physiologic mechanisms such as enzymatic reactions and counter-transport mechanisms of importance in nephrology, as exemplified by variants in <em>SLC7A9</em> and the urinary ratios of lysine and several neutral amino acids. Our study generates multiple new hypotheses about the role of identified genes in human metabolism that can now be tested experimentally.</p><p id="__p52">The great majority of genomic loci identified in our study have previously been detected in equally sized or larger GWAS of metabolite concentrations in population-based studies. Our study reveals that these genetic determinants of metabolite concentrations are also important in the setting of CKD. Thus, patients with both CKD and metabolite-increasing genotypes may represent a particularly high-risk group for adverse events related to such metabolites. The different techniques and assays used for metabolite quantification as well as the different transformations and covariate adjustments across these studies unfortunately preclude a systematic comparison of genetic effect sizes between population-based individuals and patients with CKD to answer the question of whether CKD as a metabolic challenge is reflected in different genetic effect sizes. A systematic comparison of genetic effect sizes between population-based individuals and patients with CKD should therefore be the focus of future joint studies that harmonize the methods for metabolite quantification and data analysis across studies <em>a priori</em>.</p><p id="__p53">There was one novel association between serum putrescine concentrations and genetic variants at the <em>AOC1</em> locus, encoding an enzyme catalyzing the oxidative deamination of polyamines such as putrescine.<sup><sup><a href="#B43" rid="B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851658">43</a></sup></sup> The doubling in genetic effect size in patients with CKD with eGFR&lt;30 compared with those with eGFR≥60 ml/min per 1.73 m<sup>2</sup> may hint at the presence of dose-response effects with worsening kidney function and illustrate the value of studying “challenged” populations (such as disease cohorts) in general. Again, however, differences across studies complicate comparisons: the lack of reported associations with putrescine in previous population-based studies may be because it was not included on targeted assays or removed during quality control, because it was concentrated too lowly for quantification, because it was measured but not identified as putrescine, or because it showed no genetic associations and was therefore not reported. Although transcriptional activation of <em>AOC1</em> has recently been linked to kidney morphogenesis in mice by affecting polyamine breakdown,<sup><sup><a href="#B44" rid="B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851698">44</a></sup></sup> it cannot be inferred from our study whether elevated putrescine concentrations occur as a consequence of CKD or contribute causally to CKD etiology and/or progression.</p><p id="__p54">In terms of biologic insights, our study offers snapshots of renal physiology in humans and generates novel hypotheses for experimental confirmation. This former is exemplified by the identification of the neutral amino acids transported in exchange for urinary lysine in humans <em>in vivo</em>, and the latter by the potential transport capacity of human SLC36A2 for serine. Several previous studies reported associations between the same index or a proxy variant at the <em>SLC7A9</em> locus with urinary lysine concentrations<sup><sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851702">6</a></sup>,<sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851663">11</a></sup>–<sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851716">13</a></sup></sup> or some lysine-containing ratios,<sup><sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851688">11</a></sup></sup> but without biologic interpretations in light of tubular counter-transport mechanisms. Interestingly, GWAS of eGFR in the general population could show that the rs12460876 C allele is associated not only with higher urinary lysine concentrations but also with higher eGFR and lower CKD prevalence and incidence,<sup><sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851709">6</a></sup>,<sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851710">13</a></sup>,<sup><a href="#B45" rid="B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851685">45</a></sup></sup> consistent with the association between higher urinary lysine concentrations and decreased risk for CKD.<sup><sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851693">13</a></sup></sup> The molecular mechanisms connecting reduced SLC7A9-mediated transport, resulting in higher urinary lysine and cystine and higher intracellular AA<sup>0</sup> concentrations, to better eGFR/lower CKD risk are presently unclear.</p><p id="__p55">Studies of genetic influences on metabolite concentrations in patients with CKD have the potential to ultimately deliver insights of clinical importance; for example, by identifying genetic predispositions to unfavorable profiles of CKD risk factors or to CKD comorbidities. This may be especially relevant for rare variants with large effects on CKD risk factors, the presence and prevalence of which is illustrated by the discovery of rare coding <em>ACADM</em> variants in our study. Illuminating pathways that underlie processes such as hyperlipidemia or stone formation may eventually help to better treat or prevent these common CKD-related conditions. In addition, genetic discoveries related to metabolites that are considered uremic toxins such as putrescine can be used to test their causal involvement in disease progression once large-scale genetic studies of CKD progression become available.</p><p id="__p56">Some limitations of our study warrant mention: the use of the BIOCRATES p180 assay for metabolite quantification results in a limited number of metabolites to study. Although nontargeted approaches to quantify metabolites result in a much greater number of metabolites for study and cover classic uremic toxins that are not part of the BIOCRATES assay, the latter was chosen in this study because it includes isotope-labeled standards for many metabolites that allow for comparison and integration of their concentrations across urine and blood. We could not comprehensively compare genetic effect sizes on metabolite concentrations in patients with CKD to those from previous population-based studies because of differences in methods to quantify metabolites and statistical data analysis. Our study was limited to individuals of European ancestry, and can therefore not necessarily be generalized. Twenty-four-hour urine collections were not available in the GCKD study, which may have led to increased variation of urinary metabolite concentrations because of differences in circadian rhythms resulting in reduced power to detect associations. Urate is one of few metabolites whose FE has been assessed from different urine collection methods in studies of healthy individuals. In these studies, the FEs for urate quantified from spot urine were similar to those from 24-hour urines,<sup><sup><a href="#B46" rid="B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851664">46</a></sup>–<sup><a href="#B48" rid="B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_702851654">48</a></sup></sup> but this may not pertain to all metabolites.</p><p id="__p57" class="p p-last">Our study represents the first study of genetic influences on metabolite profiles in a large population of patients with CKD. We were able to examine up to 1168 patients with CKD and benefitted from the standardized data collection, sample preprocessing and storage procedures, DNA extraction, and genotyping in the GCKD study. Our study shows that genetic influences on metabolite profiles in the general population translate to patients with CKD and that novel genetic associations can be identified in this study population. Modeling of renal metabolite handling such as counter-transport mechanisms delivered novel insights into human renal (patho-)physiology and generated new hypotheses for experimental confirmation. Future extensions of sample size and metabolite coverage hold great promise to generate additional insights into human renal physiology <em>in vivo</em>.</p></div><div id="s17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s17title">Disclosures</h2><p id="__p58" class="p p-first-last">J.W. is employed by BIOCRATES Life Sciences Aktiengesellschaft (AG) (Innsbruck, Austria).</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplemental Data:</strong>

<div class="sup-box half_rhythm" id="idm140086564831344"><a href="/pmc/articles/PMC5967769/bin/supp_29_5_1513__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>



</div></div><div id="idm140086552071504" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140086552071504title">Acknowledgments</h2><div class="sec"><p id="__p62">We are grateful for the willingness of the patients to participate in the German Chronic Kidney Disease (GCKD) study. The enormous effort of the study personnel of the various regional centers is highly appreciated. We thank the large number of nephrologists who provide routine care for the patients and collaborate with the GCKD study. The GCKD Investigators are listed in the Supplementary Information.</p><p id="__p63">The work of P.S., Y.L., and A.K. and metabolite quantification was supported by grants KO 3598/2-1 and KO 3598/4-1 of the German Research Foundation (DFG). The work of M.W., U.T.S., and A.K. was supported by DFG CRC 1140. P. Schlosser was supported by DFG CRC 992. A.K. was additionally supported by a DFG Heisenberg Professorship (KO 3598/3-1); U.T.S. was additionally supported by the Else Kroener Fresenius Kolleg Nierenfunktionsstörungen als Komplikation von Systemerkrankungen (NAKSYS). Genotyping was supported by Bayer Pharma Aktiengesellschaft (AG). Metabolite quantification was additionally supported by the Else Kroener Fresenius Foundation and the German Ministry of Education and Research (BMBF) Gerontosys II NephAge Project, 031 5896A. The GCKD study was/is funded by grants from the BMBF (grant number 01ER0804) and the Kuratorium fuer Heimdialyse (KfH) Foundation for Preventive Medicine.</p></div></div><div id="idm140086554755936" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140086554755936title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn1"></p><p id="__p59" class="p p-first-last">Published online ahead of print. Publication date available at <a href="http://www.jasn.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.jasn.org</a>.</p><p></p><p class="fn sec" id="fn3"></p><p id="__p60" class="p p-first-last">See related article, “Insights into CKD from Metabolite GWAS,” on pages <a href="/pmc/articles/PMC5967764/">1349–1351</a>.</p><p></p><p class="fn sec" id="fn2"></p><p id="__p61" class="p p-first-last">This article contains supplemental material online at <a href="http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://jasn.asnjournals.org/lookup/suppl/doi:10.1681/ASN.2017101099/-/DCSupplemental</a>.</p><p></p></div></div><div id="idm140086557277424" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140086557277424title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="mixed-citation">Kalim S, Rhee EP.: <span class="ref-title">An overview of renal metabolomics.</span>
<span class="ref-journal">Kidney Int</span>
<span class="ref-vol">91</span>: 61–69, 2017 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5380230/">PMC free article</a>]</span> [<a href="/pubmed/27692817" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=An+overview+of+renal+metabolomics.&amp;author=S+Kalim&amp;author=EP+Rhee&amp;volume=91&amp;publication_year=2017&amp;pages=61-69&amp;pmid=27692817&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="mixed-citation">Gieger C, Geistlinger L, Altmaier E, Hrabé de Angelis M, Kronenberg F, Meitinger T, et al. .: <span class="ref-title">Genetics meets metabolomics: A genome-wide association study of metabolite profiles in human serum.</span>
<span class="ref-journal">PLoS Genet</span>
<span class="ref-vol">4</span>: e1000282, 2008 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2581785/">PMC free article</a>]</span> [<a href="/pubmed/19043545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetics+meets+metabolomics:+A+genome-wide+association+study+of+metabolite+profiles+in+human+serum.&amp;author=C+Gieger&amp;author=L+Geistlinger&amp;author=E+Altmaier&amp;author=M+Hrabé+de+Angelis&amp;author=F+Kronenberg&amp;volume=4&amp;publication_year=2008&amp;pages=e1000282&amp;pmid=19043545&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="mixed-citation">Illig T, Gieger C, Zhai G, Römisch-Margl W, Wang-Sattler R, Prehn C, et al. .: <span class="ref-title">A genome-wide perspective of genetic variation in human metabolism.</span>
<span class="ref-journal">Nat Genet</span>
<span class="ref-vol">42</span>: 137–141, 2010 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3773904/">PMC free article</a>]</span> [<a href="/pubmed/20037589" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+perspective+of+genetic+variation+in+human+metabolism.&amp;author=T+Illig&amp;author=C+Gieger&amp;author=G+Zhai&amp;author=W+Römisch-Margl&amp;author=R+Wang-Sattler&amp;volume=42&amp;publication_year=2010&amp;pages=137-141&amp;pmid=20037589&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="mixed-citation">Suhre K, Shin SY, Petersen AK, Mohney RP, Meredith D, Wägele B, et al. ; CARDIoGRAM .: <span class="ref-title">Human metabolic individuality in biomedical and pharmaceutical research.</span>
<span class="ref-journal">Nature</span>
<span class="ref-vol">477</span>: 54–60, 2011 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3832838/">PMC free article</a>]</span> [<a href="/pubmed/21886157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Human+metabolic+individuality+in+biomedical+and+pharmaceutical+research.&amp;author=K+Suhre&amp;author=SY+Shin&amp;author=AK+Petersen&amp;author=RP+Mohney&amp;author=D+Meredith&amp;volume=477&amp;publication_year=2011&amp;pages=54-60&amp;pmid=21886157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="mixed-citation">Kettunen J, Tukiainen T, Sarin AP, Ortega-Alonso A, Tikkanen E, Lyytikäinen LP, et al. .: <span class="ref-title">Genome-wide association study identifies multiple loci influencing human serum metabolite levels.</span>
<span class="ref-journal">Nat Genet</span>
<span class="ref-vol">44</span>: 269–276, 2012 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3605033/">PMC free article</a>]</span> [<a href="/pubmed/22286219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+multiple+loci+influencing+human+serum+metabolite+levels.&amp;author=J+Kettunen&amp;author=T+Tukiainen&amp;author=AP+Sarin&amp;author=A+Ortega-Alonso&amp;author=E+Tikkanen&amp;volume=44&amp;publication_year=2012&amp;pages=269-276&amp;pmid=22286219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="mixed-citation">Rueedi R, Ledda M, Nicholls AW, Salek RM, Marques-Vidal P, Morya E, et al. .: <span class="ref-title">Genome-wide association study of metabolic traits reveals novel gene-metabolite-disease links.</span>
<span class="ref-journal">PLoS Genet</span>
<span class="ref-vol">10</span>: e1004132, 2014 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3930510/">PMC free article</a>]</span> [<a href="/pubmed/24586186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+of+metabolic+traits+reveals+novel+gene-metabolite-disease+links.&amp;author=R+Rueedi&amp;author=M+Ledda&amp;author=AW+Nicholls&amp;author=RM+Salek&amp;author=P+Marques-Vidal&amp;volume=10&amp;publication_year=2014&amp;pages=e1004132&amp;pmid=24586186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="mixed-citation">Shin SY, Fauman EB, Petersen AK, Krumsiek J, Santos R, Huang J, et al. .; Multiple Tissue Human Expression Resource (MuTHER) Consortium .: <span class="ref-title">An atlas of genetic influences on human blood metabolites.</span>
<span class="ref-journal">Nat Genet</span>
<span class="ref-vol">46</span>: 543–550, 2014 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4064254/">PMC free article</a>]</span> [<a href="/pubmed/24816252" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=An+atlas+of+genetic+influences+on+human+blood+metabolites.&amp;author=SY+Shin&amp;author=EB+Fauman&amp;author=AK+Petersen&amp;author=J+Krumsiek&amp;author=R+Santos&amp;volume=46&amp;publication_year=2014&amp;pages=543-550&amp;pmid=24816252&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="mixed-citation">Draisma HHM, Pool R, Kobl M, Jansen R, Petersen AK, Vaarhorst AAM, et al. .: <span class="ref-title">Genome-wide association study identifies novel genetic variants contributing to variation in blood metabolite levels.</span>
<span class="ref-journal">Nat Commun</span>
<span class="ref-vol">6</span>: 7208, 2015 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4745136/">PMC free article</a>]</span> [<a href="/pubmed/26068415" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Genome-wide+association+study+identifies+novel+genetic+variants+contributing+to+variation+in+blood+metabolite+levels.&amp;author=HHM+Draisma&amp;author=R+Pool&amp;author=M+Kobl&amp;author=R+Jansen&amp;author=AK+Petersen&amp;volume=6&amp;publication_year=2015&amp;pages=7208&amp;pmid=26068415&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="mixed-citation">Rhee EP, Yang Q, Yu B, Liu X, Cheng S, Deik A, et al. .: <span class="ref-title">An exome array study of the plasma metabolome.</span>
<span class="ref-journal">Nat Commun</span>
<span class="ref-vol">7</span>: 12360, 2016 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4962516/">PMC free article</a>]</span> [<a href="/pubmed/27453504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=An+exome+array+study+of+the+plasma+metabolome.&amp;author=EP+Rhee&amp;author=Q+Yang&amp;author=B+Yu&amp;author=X+Liu&amp;author=S+Cheng&amp;volume=7&amp;publication_year=2016&amp;pages=12360&amp;pmid=27453504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="mixed-citation">Long T, Hicks M, Yu HC, Biggs WH, Kirkness EF, Menni C, et al. .: <span class="ref-title">Whole-genome sequencing identifies common-to-rare variants associated with human blood metabolites.</span>
<span class="ref-journal">Nat Genet</span>
<span class="ref-vol">49</span>: 568–578, 2017 [<a href="/pubmed/28263315" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Whole-genome+sequencing+identifies+common-to-rare+variants+associated+with+human+blood+metabolites.&amp;author=T+Long&amp;author=M+Hicks&amp;author=HC+Yu&amp;author=WH+Biggs&amp;author=EF+Kirkness&amp;volume=49&amp;publication_year=2017&amp;pages=568-578&amp;pmid=28263315&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="mixed-citation">Suhre K, Wallaschofski H, Raffler J, Friedrich N, Haring R, Michael K, et al. .: <span class="ref-title">A genome-wide association study of metabolic traits in human urine.</span>
<span class="ref-journal">Nat Genet</span>
<span class="ref-vol">43</span>: 565–569, 2011 [<a href="/pubmed/21572414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+of+metabolic+traits+in+human+urine.&amp;author=K+Suhre&amp;author=H+Wallaschofski&amp;author=J+Raffler&amp;author=N+Friedrich&amp;author=R+Haring&amp;volume=43&amp;publication_year=2011&amp;pages=565-569&amp;pmid=21572414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="mixed-citation">Raffler J, Friedrich N, Arnold M, Kacprowski T, Rueedi R, Altmaier E, et al. .: <span class="ref-title">Genome-wide association Study with targeted and non-targeted NMR metabolomics identifies 15 novel loci of urinary human metabolic individuality.</span>
<span class="ref-journal">PLoS Genet</span>
<span class="ref-vol">11</span>: e1005487, 2015 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4564198/">PMC free article</a>]</span> [<a href="/pubmed/26352407" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+Study+with+targeted+and+non-targeted+NMR+metabolomics+identifies+15+novel+loci+of+urinary+human+metabolic+individuality.&amp;author=J+Raffler&amp;author=N+Friedrich&amp;author=M+Arnold&amp;author=T+Kacprowski&amp;author=R+Rueedi&amp;volume=11&amp;publication_year=2015&amp;pages=e1005487&amp;pmid=26352407&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="mixed-citation">McMahon GM, Hwang SJ, Clish CB, Tin A, Yang Q, Larson MG, et al. .; CKDGen Consortium .: <span class="ref-title">Urinary metabolites along with common and rare&nbsp;genetic variations are associated with incident&nbsp;chronic kidney disease.</span>
<span class="ref-journal">Kidney Int</span>
<span class="ref-vol">91</span>: 1426–1435, 2017 [<a href="/pubmed/28302371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Urinary+metabolites+along+with+common+and+rare&nbsp;genetic+variations+are+associated+with+incident&nbsp;chronic+kidney+disease.&amp;author=GM+McMahon&amp;author=SJ+Hwang&amp;author=CB+Clish&amp;author=A+Tin&amp;author=Q+Yang&amp;volume=91&amp;publication_year=2017&amp;pages=1426-1435&amp;pmid=28302371&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="mixed-citation">Eckardt KU, Bärthlein B, Baid-Agrawal S, Beck A, Busch M, Eitner F, et al. .: <span class="ref-title">The German Chronic Kidney Disease (GCKD) study: Design and methods.</span>
<span class="ref-journal">Nephrol Dial Transplant</span>
<span class="ref-vol">27</span>: 1454–1460, 2012 [<a href="/pubmed/21862458" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephrol+Dial+Transplant&amp;title=The+German+Chronic+Kidney+Disease+(GCKD)+study:+Design+and+methods.&amp;author=KU+Eckardt&amp;author=B+Bärthlein&amp;author=S+Baid-Agrawal&amp;author=A+Beck&amp;author=M+Busch&amp;volume=27&amp;publication_year=2012&amp;pages=1454-1460&amp;pmid=21862458&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="mixed-citation">Titze S, Schmid M, Köttgen A, Busch M, Floege J, Wanner C, et al. .; GCKD study investigators .: <span class="ref-title">Disease burden and risk profile in referred patients with moderate chronic kidney disease: Composition of the German Chronic Kidney Disease (GCKD) cohort.</span>
<span class="ref-journal">Nephrol Dial Transplant</span>
<span class="ref-vol">30</span>: 441–451, 2015 [<a href="/pubmed/25271006" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephrol+Dial+Transplant&amp;title=Disease+burden+and+risk+profile+in+referred+patients+with+moderate+chronic+kidney+disease:+Composition+of+the+German+Chronic+Kidney+Disease+(GCKD)+cohort.&amp;author=S+Titze&amp;author=M+Schmid&amp;author=A+Köttgen&amp;author=M+Busch&amp;author=J+Floege&amp;volume=30&amp;publication_year=2015&amp;pages=441-451&amp;pmid=25271006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="mixed-citation">Prokosch HU, Mate S, Christoph J, Beck A, Köpcke F, Stephan S, et al. .: <span class="ref-title">Designing and implementing a biobanking IT framework for multiple research scenarios.</span>
<span class="ref-journal">Stud Health Technol Inform</span>
<span class="ref-vol">180</span>: 559–563, 2012 [<a href="/pubmed/22874253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stud+Health+Technol+Inform&amp;title=Designing+and+implementing+a+biobanking+IT+framework+for+multiple+research+scenarios.&amp;author=HU+Prokosch&amp;author=S+Mate&amp;author=J+Christoph&amp;author=A+Beck&amp;author=F+Köpcke&amp;volume=180&amp;publication_year=2012&amp;pages=559-563&amp;pmid=22874253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="mixed-citation">Anderson CA, Pettersson FH, Clarke GM, Cardon LR, Morris AP, Zondervan KT.: <span class="ref-title">Data quality control in genetic case-control association studies.</span>
<span class="ref-journal">Nat Protoc</span>
<span class="ref-vol">5</span>: 1564–1573, 2010 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025522/">PMC free article</a>]</span> [<a href="/pubmed/21085122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Data+quality+control+in+genetic+case-control+association+studies.&amp;author=CA+Anderson&amp;author=FH+Pettersson&amp;author=GM+Clarke&amp;author=LR+Cardon&amp;author=AP+Morris&amp;volume=5&amp;publication_year=2010&amp;pages=1564-1573&amp;pmid=21085122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="mixed-citation">Howie BN, Donnelly P, Marchini J.: <span class="ref-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.</span>
<span class="ref-journal">PLoS Genet</span>
<span class="ref-vol">5</span>: e1000529, 2009 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies.&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="mixed-citation">Delaneau O, Zagury JF, Marchini J.: <span class="ref-title">Improved whole-chromosome phasing for disease and population genetic studies.</span>
<span class="ref-journal">Nat Methods</span>
<span class="ref-vol">10</span>: 5–6, 2013 [<a href="/pubmed/23269371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Improved+whole-chromosome+phasing+for+disease+and+population+genetic+studies.&amp;author=O+Delaneau&amp;author=JF+Zagury&amp;author=J+Marchini&amp;volume=10&amp;publication_year=2013&amp;pages=5-6&amp;pmid=23269371&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="mixed-citation">Guo Y, He J, Zhao S, Wu H, Zhong X, Sheng Q, et al. .: <span class="ref-title">Illumina human exome genotyping array clustering and quality control.</span>
<span class="ref-journal">Nat Protoc</span>
<span class="ref-vol">9</span>: 2643–2662, 2014 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4441213/">PMC free article</a>]</span> [<a href="/pubmed/25321409" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Illumina+human+exome+genotyping+array+clustering+and+quality+control.&amp;author=Y+Guo&amp;author=J+He&amp;author=S+Zhao&amp;author=H+Wu&amp;author=X+Zhong&amp;volume=9&amp;publication_year=2014&amp;pages=2643-2662&amp;pmid=25321409&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="mixed-citation">Goldstein JI, Crenshaw A, Carey J, Grant GB, Maguire J, Fromer M, et al. .; Swedish Schizophrenia Consortium; ARRA Autism Sequencing Consortium .: <span class="ref-title">zCall: A rare variant caller for array-based genotyping: Genetics and population analysis.</span>
<span class="ref-journal">Bioinformatics</span>
<span class="ref-vol">28</span>: 2543–2545, 2012 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3463112/">PMC free article</a>]</span> [<a href="/pubmed/22843986" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=zCall:+A+rare+variant+caller+for+array-based+genotyping:+Genetics+and+population+analysis.&amp;author=JI+Goldstein&amp;author=A+Crenshaw&amp;author=J+Carey&amp;author=GB+Grant&amp;author=J+Maguire&amp;volume=28&amp;publication_year=2012&amp;pages=2543-2545&amp;pmid=22843986&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="mixed-citation">Ramsay, S.L.B.D. (Heidelberg 3084, AU), Guggenbichler, W. (Austrasse 26A, A-6063 Rum, AT), Weinberger, K.M. (Weidach 82, A-6414 Mieming, AT), Graber, A. (Fishnalerstrasse 14/Top3, A-6020 Innsbruck, AT), Stöggl, W.M. (Hoher Weg 9/7, A-6020 Innsbruck, AT); Device for quantitative analysis of a metabolite profile. (BIOCRATES Life Sciences AG (Innrain 66, 6020 Innsbruck, AT), 2014)</span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="mixed-citation">Ramsay, S.L.B.D. (Heidelberg 3084, AT), Stöggl, W.M. (Hoher Weg 9/7, A-6020 Innsbruck, AT), Weinberger, K.M. (Weidach 82, A-6414 Mieming, AT), Graber, A. (Fischnalerstrasse Innsbruck 14/Top 3, A-6020 Innsbruck, AT), Guggenbichler, W. (Austrasse 26A, A-6063 Rum, AT); Apparatus for analyzing a metabolite profile. (BIOCRATES Life Sciences AG (Innrain 66, 6020 Innsbruck, AT), 2014)</span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="mixed-citation">Levey AS, Stevens LA, Schmid CH, Zhang YL, Castro AF III, Feldman HI, et al. .; CKD-EPI (Chronic Kidney Disease Epidemiology Collaboration) : <span class="ref-title">A new equation to estimate glomerular filtration rate.</span>
<span class="ref-journal">Ann Intern Med</span>
<span class="ref-vol">150</span>: 604–612, 2009 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2763564/">PMC free article</a>]</span> [<a href="/pubmed/19414839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Intern+Med&amp;title=A+new+equation+to+estimate+glomerular+filtration+rate.&amp;author=AS+Levey&amp;author=LA+Stevens&amp;author=CH+Schmid&amp;author=YL+Zhang&amp;author=AF+Castro&amp;volume=150&amp;publication_year=2009&amp;pages=604-612&amp;pmid=19414839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="mixed-citation">Dieterle F, Ross A, Schlotterbeck G, Senn H.: <span class="ref-title">Probabilistic quotient normalization as robust method to account for dilution of complex biological mixtures. Application in <sup>1</sup>H NMR metabonomics.</span>
<span class="ref-journal">Anal Chem</span>
<span class="ref-vol">78</span>: 4281–4290, 2006 [<a href="/pubmed/16808434" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anal+Chem&amp;title=Probabilistic+quotient+normalization+as+robust+method+to+account+for+dilution+of+complex+biological+mixtures.+Application+in+1H+NMR+metabonomics.&amp;author=F+Dieterle&amp;author=A+Ross&amp;author=G+Schlotterbeck&amp;author=H+Senn&amp;volume=78&amp;publication_year=2006&amp;pages=4281-4290&amp;pmid=16808434&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="mixed-citation">Fuchsberger C, Taliun D, Pramstaller PP, Pattaro C; CKDGen consortium .: <span class="ref-title">GWAtoolbox: An R package for fast quality control and handling of genome-wide association studies meta-analysis data.</span>
<span class="ref-journal">Bioinformatics</span>
<span class="ref-vol">28</span>: 444–445, 2012 [<a href="/pubmed/22155946" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GWAtoolbox:+An+R+package+for+fast+quality+control+and+handling+of+genome-wide+association+studies+meta-analysis+data.&amp;author=C+Fuchsberger&amp;author=D+Taliun&amp;author=PP+Pramstaller&amp;author=C+Pattaro&amp;volume=28&amp;publication_year=2012&amp;pages=444-445&amp;pmid=22155946&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="mixed-citation">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, et al. .: <span class="ref-title">LocusZoom: Regional visualization of genome-wide association scan results.</span>
<span class="ref-journal">Bioinformatics</span>
<span class="ref-vol">26</span>: 2336–2337, 2010 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+Regional+visualization+of+genome-wide+association+scan+results.&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="mixed-citation">Pattaro C, Teumer A, Gorski M, Chu AY, Li M, Mijatovic V, et al. .; ICBP Consortium; AGEN Consortium; CARDIOGRAM; CHARGe-Heart Failure Group; ECHOGen Consortium .: <span class="ref-title">Genetic associations at 53 loci highlight cell types and biological pathways relevant for kidney function.</span>
<span class="ref-journal">Nat Commun</span>
<span class="ref-vol">7</span>: 10023, 2016 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4735748/">PMC free article</a>]</span> [<a href="/pubmed/26831199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Genetic+associations+at+53+loci+highlight+cell+types+and+biological+pathways+relevant+for+kidney+function.&amp;author=C+Pattaro&amp;author=A+Teumer&amp;author=M+Gorski&amp;author=AY+Chu&amp;author=M+Li&amp;volume=7&amp;publication_year=2016&amp;pages=10023&amp;pmid=26831199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="mixed-citation">Fabregat-Traver D, Sharapov SZ, Hayward C, Rudan I, Campbell H, Aulchenko Y, et al. .: <span class="ref-title">High-performance mixed models based genome-wide association analysis with omicABEL software.</span>
<span class="ref-journal">F1000 Res</span>
<span class="ref-vol">3</span>: 200, 2014 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4329600/">PMC free article</a>]</span> [<a href="/pubmed/25717363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=F1000+Res&amp;title=High-performance+mixed+models+based+genome-wide+association+analysis+with+omicABEL+software.&amp;author=D+Fabregat-Traver&amp;author=SZ+Sharapov&amp;author=C+Hayward&amp;author=I+Rudan&amp;author=H+Campbell&amp;volume=3&amp;publication_year=2014&amp;pages=200&amp;pmid=25717363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="mixed-citation">Petersen AK, Krumsiek J, Wägele B, Theis FJ, Wichmann HE, Gieger C, et al. .: <span class="ref-title">On the hypothesis-free testing of metabolite ratios in genome-wide and metabolome-wide association studies.</span>
<span class="ref-journal">BMC Bioinformatics</span>
<span class="ref-vol">13</span>: 120, 2012 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3537592/">PMC free article</a>]</span> [<a href="/pubmed/22672667" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=On+the+hypothesis-free+testing+of+metabolite+ratios+in+genome-wide+and+metabolome-wide+association+studies.&amp;author=AK+Petersen&amp;author=J+Krumsiek&amp;author=B+Wägele&amp;author=FJ+Theis&amp;author=HE+Wichmann&amp;volume=13&amp;publication_year=2012&amp;pages=120&amp;pmid=22672667&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="mixed-citation">Grove ML, Yu B, Cochran BJ, Haritunians T, Bis JC, Taylor KD, et al. .: <span class="ref-title">Best practices and joint calling of the HumanExome BeadChip: The CHARGE consortium.</span>
<span class="ref-journal">PLoS One</span>
<span class="ref-vol">8</span>: e68095, 2013 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3709915/">PMC free article</a>]</span> [<a href="/pubmed/23874508" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Best+practices+and+joint+calling+of+the+HumanExome+BeadChip:+The+CHARGE+consortium.&amp;author=ML+Grove&amp;author=B+Yu&amp;author=BJ+Cochran&amp;author=T+Haritunians&amp;author=JC+Bis&amp;volume=8&amp;publication_year=2013&amp;pages=e68095&amp;pmid=23874508&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="mixed-citation">Duranton F, Cohen G, De Smet R, Rodriguez M, Jankowski J, Vanholder R, et al. .; European Uremic Toxin Work Group .: <span class="ref-title">Normal and pathologic concentrations of uremic toxins.</span>
<span class="ref-journal">J Am Soc Nephrol</span>
<span class="ref-vol">23</span>: 1258–1270, 2012 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3380651/">PMC free article</a>]</span> [<a href="/pubmed/22626821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=Normal+and+pathologic+concentrations+of+uremic+toxins.&amp;author=F+Duranton&amp;author=G+Cohen&amp;author=R+De+Smet&amp;author=M+Rodriguez&amp;author=J+Jankowski&amp;volume=23&amp;publication_year=2012&amp;pages=1258-1270&amp;pmid=22626821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="mixed-citation">Consortium GT; GTEx Consortium .: <span class="ref-title">Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans.</span>
<span class="ref-journal">Science</span>
<span class="ref-vol">348</span>: 648–660, 2015 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4547484/">PMC free article</a>]</span> [<a href="/pubmed/25954001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Human+genomics.+The+Genotype-Tissue+Expression+(GTEx)+pilot+analysis:+Multitissue+gene+regulation+in+humans.&amp;author=GT+Consortium&amp;volume=348&amp;publication_year=2015&amp;pages=648-660&amp;pmid=25954001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="mixed-citation">Suchi M, Harada N, Wada Y, Takagi Y.: <span class="ref-title">Molecular cloning of a cDNA encoding human histidase.</span>
<span class="ref-journal">Biochim Biophys Acta</span>
<span class="ref-vol">1216</span>: 293–295, 1993 [<a href="/pubmed/7916645" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=Molecular+cloning+of+a+cDNA+encoding+human+histidase.&amp;author=M+Suchi&amp;author=N+Harada&amp;author=Y+Wada&amp;author=Y+Takagi&amp;volume=1216&amp;publication_year=1993&amp;pages=293-295&amp;pmid=7916645&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="mixed-citation">Demirkan A, van Duijn CM, Ugocsai P, Isaacs A, Pramstaller PP, Liebisch G, et al. .; DIAGRAM Consortium; CARDIoGRAM Consortium; CHARGE Consortium; EUROSPAN consortium .: <span class="ref-title">Genome-wide association study identifies novel loci associated with circulating phospho- and sphingolipid concentrations.</span>
<span class="ref-journal">PLoS Genet</span>
<span class="ref-vol">8</span>: e1002490, 2012 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3280968/">PMC free article</a>]</span> [<a href="/pubmed/22359512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+identifies+novel+loci+associated+with+circulating+phospho-+and+sphingolipid+concentrations.&amp;author=A+Demirkan&amp;author=CM+van+Duijn&amp;author=P+Ugocsai&amp;author=A+Isaacs&amp;author=PP+Pramstaller&amp;volume=8&amp;publication_year=2012&amp;pages=e1002490&amp;pmid=22359512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="mixed-citation">Munck BG: <span class="ref-title">Lysine transport across the small intestine. Stimulating and inhibitory effects of neutral amino acids.</span>
<span class="ref-journal">J Membr Biol</span>
<span class="ref-vol">53</span>: 45–53, 1980 [<a href="/pubmed/6768890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Membr+Biol&amp;title=Lysine+transport+across+the+small+intestine.+Stimulating+and+inhibitory+effects+of+neutral+amino+acids.&amp;author=BG+Munck&amp;volume=53&amp;publication_year=1980&amp;pages=45-53&amp;pmid=6768890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="mixed-citation">Munck BG: <span class="ref-title">Transport of neutral and cationic amino acids across the brush-border membrane of the rabbit ileum.</span>
<span class="ref-journal">J Membr Biol</span>
<span class="ref-vol">83</span>: 1–13, 1985 [<a href="/pubmed/3923195" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Membr+Biol&amp;title=Transport+of+neutral+and+cationic+amino+acids+across+the+brush-border+membrane+of+the+rabbit+ileum.&amp;author=BG+Munck&amp;volume=83&amp;publication_year=1985&amp;pages=1-13&amp;pmid=3923195&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="mixed-citation">Bröer S, Bailey CG, Kowalczuk S, Ng C, Vanslambrouck JM, Rodgers H, et al. .: <span class="ref-title">Iminoglycinuria and hyperglycinuria are discrete human phenotypes resulting from complex mutations in proline and glycine transporters.</span>
<span class="ref-journal">J Clin Invest</span>
<span class="ref-vol">118</span>: 3881–3892, 2008 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2579706/">PMC free article</a>]</span> [<a href="/pubmed/19033659" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Iminoglycinuria+and+hyperglycinuria+are+discrete+human+phenotypes+resulting+from+complex+mutations+in+proline+and+glycine+transporters.&amp;author=S+Bröer&amp;author=CG+Bailey&amp;author=S+Kowalczuk&amp;author=C+Ng&amp;author=JM+Vanslambrouck&amp;volume=118&amp;publication_year=2008&amp;pages=3881-3892&amp;pmid=19033659&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="mixed-citation">Kennedy DJ, Gatfield KM, Winpenny JP, Ganapathy V, Thwaites DT.: <span class="ref-title">Substrate specificity and functional characterisation of the H+/amino acid transporter rat PAT2 (Slc36a2).</span>
<span class="ref-journal">Br J Pharmacol</span>
<span class="ref-vol">144</span>: 28–41, 2005 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1575977/">PMC free article</a>]</span> [<a href="/pubmed/15644866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Pharmacol&amp;title=Substrate+specificity+and+functional+characterisation+of+the+H+/amino+acid+transporter+rat+PAT2+(Slc36a2).&amp;author=DJ+Kennedy&amp;author=KM+Gatfield&amp;author=JP+Winpenny&amp;author=V+Ganapathy&amp;author=DT+Thwaites&amp;volume=144&amp;publication_year=2005&amp;pages=28-41&amp;pmid=15644866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="mixed-citation">Rhee EP, Ho JE, Chen MH, Shen D, Cheng S, Larson MG, et al. .: <span class="ref-title">A genome-wide association study of the human metabolome in a community-based cohort.</span>
<span class="ref-journal">Cell Metab</span>
<span class="ref-vol">18</span>: 130–143, 2013 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3973158/">PMC free article</a>]</span> [<a href="/pubmed/23823483" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Metab&amp;title=A+genome-wide+association+study+of+the+human+metabolome+in+a+community-based+cohort.&amp;author=EP+Rhee&amp;author=JE+Ho&amp;author=MH+Chen&amp;author=D+Shen&amp;author=S+Cheng&amp;volume=18&amp;publication_year=2013&amp;pages=130-143&amp;pmid=23823483&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="mixed-citation">Burkhardt R, Kirsten H, Beutner F, Holdt LM, Gross A, Teren A, et al. .: <span class="ref-title">Integration of genome-wide SNP data and gene-expression profiles reveals six novel loci and regulatory mechanisms for amino acids and acylcarnitines in whole blood.</span>
<span class="ref-journal">PLoS Genet</span>
<span class="ref-vol">11</span>: e1005510, 2015 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4581711/">PMC free article</a>]</span> [<a href="/pubmed/26401656" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Integration+of+genome-wide+SNP+data+and+gene-expression+profiles+reveals+six+novel+loci+and+regulatory+mechanisms+for+amino+acids+and+acylcarnitines+in+whole+blood.&amp;author=R+Burkhardt&amp;author=H+Kirsten&amp;author=F+Beutner&amp;author=LM+Holdt&amp;author=A+Gross&amp;volume=11&amp;publication_year=2015&amp;pages=e1005510&amp;pmid=26401656&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="mixed-citation">Igarashi K, Ueda S, Yoshida K, Kashiwagi K.: <span class="ref-title">Polyamines in renal failure.</span>
<span class="ref-journal">Amino Acids</span>
<span class="ref-vol">31</span>: 477–483, 2006 [<a href="/pubmed/16554974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Amino+Acids&amp;title=Polyamines+in+renal+failure.&amp;author=K+Igarashi&amp;author=S+Ueda&amp;author=K+Yoshida&amp;author=K+Kashiwagi&amp;volume=31&amp;publication_year=2006&amp;pages=477-483&amp;pmid=16554974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="mixed-citation">Chassande O, Renard S, Barbry P, Lazdunski M.: <span class="ref-title">The human gene for diamine oxidase, an amiloride binding protein. Molecular cloning, sequencing, and characterization of the promoter.</span>
<span class="ref-journal">J Biol Chem</span>
<span class="ref-vol">269</span>: 14484–14489, 1994 [<a href="/pubmed/8182053" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+human+gene+for+diamine+oxidase,+an+amiloride+binding+protein.+Molecular+cloning,+sequencing,+and+characterization+of+the+promoter.&amp;author=O+Chassande&amp;author=S+Renard&amp;author=P+Barbry&amp;author=M+Lazdunski&amp;volume=269&amp;publication_year=1994&amp;pages=14484-14489&amp;pmid=8182053&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="mixed-citation">Kirschner KM, Braun JF, Jacobi CL, Rudigier LJ, Persson AB, Scholz H.: <span class="ref-title">Amine oxidase copper-containing 1 (AOC1) is a downstream target gene of the Wilms tumor protein, WT1, during kidney development.</span>
<span class="ref-journal">J Biol Chem</span>
<span class="ref-vol">289</span>: 24452–24462, 2014 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4148871/">PMC free article</a>]</span> [<a href="/pubmed/25037221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Amine+oxidase+copper-containing+1+(AOC1)+is+a+downstream+target+gene+of+the+Wilms+tumor+protein,+WT1,+during+kidney+development.&amp;author=KM+Kirschner&amp;author=JF+Braun&amp;author=CL+Jacobi&amp;author=LJ+Rudigier&amp;author=AB+Persson&amp;volume=289&amp;publication_year=2014&amp;pages=24452-24462&amp;pmid=25037221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45">45. <span class="mixed-citation">Köttgen A, Pattaro C, Böger CA, Fuchsberger C, Olden M, Glazer NL, et al. .: <span class="ref-title">New loci associated with kidney function and chronic kidney disease.</span>
<span class="ref-journal">Nat Genet</span>
<span class="ref-vol">42</span>: 376–384, 2010 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2997674/">PMC free article</a>]</span> [<a href="/pubmed/20383146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+loci+associated+with+kidney+function+and+chronic+kidney+disease.&amp;author=A+Köttgen&amp;author=C+Pattaro&amp;author=CA+Böger&amp;author=C+Fuchsberger&amp;author=M+Olden&amp;volume=42&amp;publication_year=2010&amp;pages=376-384&amp;pmid=20383146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46">46. <span class="mixed-citation">Quaratino CP, Di Sciascio N, Rucci C, Ciaglia P, Giacomello A.: <span class="ref-title">The normal range of serum urate levels and of fractional urate excretion.</span>
<span class="ref-journal">Adv Exp Med Biol</span>
<span class="ref-vol">370</span>: 91–93, 1994 [<a href="/pubmed/7661034" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv+Exp+Med+Biol&amp;title=The+normal+range+of+serum+urate+levels+and+of+fractional+urate+excretion.&amp;author=CP+Quaratino&amp;author=N+Di+Sciascio&amp;author=C+Rucci&amp;author=P+Ciaglia&amp;author=A+Giacomello&amp;volume=370&amp;publication_year=1994&amp;pages=91-93&amp;pmid=7661034&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47">47. <span class="mixed-citation">Giacomello A, Di Sciascio N, Quaratino CP.: <span class="ref-title">Relation between serum triglyceride level, serum urate concentration, and fractional urate excretion.</span>
<span class="ref-journal">Metabolism</span>
<span class="ref-vol">46</span>: 1085–1089, 1997 [<a href="/pubmed/9284901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metabolism&amp;title=Relation+between+serum+triglyceride+level,+serum+urate+concentration,+and+fractional+urate+excretion.&amp;author=A+Giacomello&amp;author=N+Di+Sciascio&amp;author=CP+Quaratino&amp;volume=46&amp;publication_year=1997&amp;pages=1085-1089&amp;pmid=9284901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B48">48. <span class="mixed-citation">Perez-Ruiz F, Calabozo M, Erauskin GG, Ruibal A, Herrero-Beites AM.: <span class="ref-title">Renal underexcretion of uric acid is present in patients with apparent high urinary uric acid output.</span>
<span class="ref-journal">Arthritis Rheum</span>
<span class="ref-vol">47</span>: 610–613, 2002 [<a href="/pubmed/12522834" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Renal+underexcretion+of+uric+acid+is+present+in+patients+with+apparent+high+urinary+uric+acid+output.&amp;author=F+Perez-Ruiz&amp;author=M+Calabozo&amp;author=GG+Erauskin&amp;author=A+Ruibal&amp;author=AM+Herrero-Beites&amp;volume=47&amp;publication_year=2002&amp;pages=610-613&amp;pmid=12522834&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5967769&amp;issue-id=313532&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140086555559040title">Abstract</a></li><li><a href="#s2title">Methods</a></li><li><a href="#s9title">Results</a></li><li><a href="#s16title">Discussion</a></li><li><a href="#s17title">Disclosures</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140086552071504title">Acknowledgments</a></li><li><a href="#idm140086554755936title">Footnotes</a></li><li><a href="#idm140086557277424title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Journal of the American Society of Nephrology : JASN</span> are provided here courtesy of <strong>American Society of Nephrology</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5967769/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5967769/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5967769/?report=printable">Printer Friendly</a></li> | <li><a href="#" data-citationid="PMC5967769" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5967769%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5967769%2F&amp;text=Genome-Wide%20Association%20Studies%20of%20Metabolites%20in%20Patients%20with%20CKD%20Identify%20Multiple%20Loci%20and%20Illuminate%20Tubular%20Transport%20Mechanisms" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5967769%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29545352" data-aiid="5967769" data-aid="5967769" data-iid="313532" data-domainid="612" data-domain="jnephrol" data-accid="PMC5967769" data-md5="94a76ceea378ab5f37f48cf4ed9618ff"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T17:04:07-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A53CCDD5B3CA100000000029E0183&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5967769%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=jnephrol&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5967769/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>