<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5812477/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A meta-analysis of genome-wide association studies identifies 17 new Parkinson’s disease risk loci</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5812477/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="A meta-analysis of genome-wide association studies identifies 17 new Parkinson’s disease risk loci"><meta name="citation_authors" content="Diana Chang, Mike A Nalls, Ingileif B Hallgrímsdóttir, Julie Hunkapiller, Marcel van der Brug, Fang Cai, International Parkinson’s Disease Genomics Consortium, 23andMe ResearchTeam, Geoffrey A Kerchner, Gai Ayalon, Baris Bingol, Morgan Sheng, David Hinds, Timothy W Behrens, Andrew B Singleton, Tushar R Bhangale, Robert R Graham"><meta name="citation_date" content="October 2017"><meta name="citation_issue" content="10"><meta name="citation_volume" content="49"><meta name="citation_firstpage" content="1511"><meta name="citation_doi" content="10.1038/ng.3955"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5812477/?report=abstract"><meta name="citation_pmid" content="28892059"><meta name="DC.Title" content="A meta-analysis of genome-wide association studies identifies 17 new Parkinson’s disease risk loci"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Diana Chang"><meta name="DC.Contributor" content="Mike A Nalls"><meta name="DC.Contributor" content="Ingileif B Hallgrímsdóttir"><meta name="DC.Contributor" content="Julie Hunkapiller"><meta name="DC.Contributor" content="Marcel van der Brug"><meta name="DC.Contributor" content="Fang Cai"><meta name="DC.Contributor" content="International Parkinson’s Disease Genomics Consortium"><meta name="DC.Contributor" content="23andMe ResearchTeam"><meta name="DC.Contributor" content="Geoffrey A Kerchner"><meta name="DC.Contributor" content="Gai Ayalon"><meta name="DC.Contributor" content="Baris Bingol"><meta name="DC.Contributor" content="Morgan Sheng"><meta name="DC.Contributor" content="David Hinds"><meta name="DC.Contributor" content="Timothy W Behrens"><meta name="DC.Contributor" content="Andrew B Singleton"><meta name="DC.Contributor" content="Tushar R Bhangale"><meta name="DC.Contributor" content="Robert R Graham"><meta name="DC.Date" content="2017 Oct"><meta name="DC.Identifier" content="10.1038/ng.3955"><meta name="DC.Language" content="en"><meta property="og:title" content="A meta-analysis of genome-wide association studies identifies 17 new Parkinson’s disease risk loci"><meta property="og:type" content="article"><meta property="og:description" content="Common variant genome-wide association studies (GWASs) have, to date, identified >24 risk loci for Parkinson’s disease (PD). To discover additional loci, we carried out a GWAS comparing 6,476 PD cases with 302,042 controls, followed by ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5812477/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5812477/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5812477/pdf/nihms940616.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A170FDD5A322100000000086504B9.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5812477%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books">Books</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5812477</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2018 Feb 14.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28892059" target="pmc_ext" ref="reftype=publisher&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2017 Oct; 49(10): 1511–1516. </span></a></div><span class="fm-vol-iss-date">Published online 2017 Sep 11. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.3955" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.3955</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5812477</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS940616</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28892059">28892059</a></div></div></div></div><h1 class="content-title">A meta-analysis of genome-wide association studies identifies 17 new Parkinson’s disease risk loci</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Chang%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739639840" co-class="co-affbox">Diana Chang</a>,<sup>1</sup> <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628743217776" co-class="co-affbox">Mike A Nalls</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Hallgr%26%23x000ed%3Bmsd%26%23x000f3%3Bttir%20IB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628742453792" co-class="co-affbox">Ingileif B Hallgrímsdóttir</a>,<sup>4,</sup><sup>6</sup> <a href="/pubmed/?term=Hunkapiller%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628742450832" co-class="co-affbox">Julie Hunkapiller</a>,<sup>1</sup> <a href="/pubmed/?term=van%20der%20Brug%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739636464" co-class="co-affbox">Marcel van der Brug</a>,<sup>1,</sup><sup>6</sup> <a href="/pubmed/?term=Cai%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739633648" co-class="co-affbox">Fang Cai</a>,<sup>1</sup> International Parkinson’s Disease Genomics Consortium,<sup>5</sup> 23andMe ResearchTeam,<sup>5</sup> <a href="/pubmed/?term=Kerchner%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739584480" co-class="co-affbox">Geoffrey A Kerchner</a>,<sup>1</sup> <a href="/pubmed/?term=Ayalon%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739582448" co-class="co-affbox">Gai Ayalon</a>,<sup>1</sup> <a href="/pubmed/?term=Bingol%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739580368" co-class="co-affbox">Baris Bingol</a>,<sup>1</sup> <a href="/pubmed/?term=Sheng%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739578336" co-class="co-affbox">Morgan Sheng</a>,<sup>1</sup> <a href="/pubmed/?term=Hinds%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739576304" co-class="co-affbox">David Hinds</a>,<sup>4</sup> <a href="/pubmed/?term=Behrens%20TW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739574272" co-class="co-affbox">Timothy W Behrens</a>,<sup>1</sup> <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739572240" co-class="co-affbox">Andrew B Singleton</a>,<sup>2</sup> <a href="/pubmed/?term=Bhangale%20TR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739570208" co-class="co-affbox">Tushar R Bhangale</a>,<sup>1,</sup><sup>7</sup> and  <a href="/pubmed/?term=Graham%20RR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059" class="affpopup" co-rid="_co_idm140628739567296" co-class="co-affbox">Robert R Graham</a><sup>1,</sup><sup>7,</sup><sup>iD</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140628739639840"><h3 class="no_margin">Diana Chang</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Chang%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Diana Chang</a></div></div><div id="_co_idm140628743217776"><h3 class="no_margin">Mike A Nalls</h3><p>  <sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, US National Institutes of Health, Bethesda, Maryland, USA</p><p>  <sup>3</sup>Data Tecnica International, Glen Echo, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Mike A Nalls</a></div></div><div id="_co_idm140628742453792"><h3 class="no_margin">Ingileif B Hallgrímsdóttir</h3><p>  <sup>4</sup>23andMe Inc., Mountain View, California, USA</p><div>Find articles by <a href="/pubmed/?term=Hallgr%26%23x000ed%3Bmsd%26%23x000f3%3Bttir%20IB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Ingileif B Hallgrímsdóttir</a></div></div><div id="_co_idm140628742450832"><h3 class="no_margin">Julie Hunkapiller</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Hunkapiller%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Julie Hunkapiller</a></div></div><div id="_co_idm140628739636464"><h3 class="no_margin">Marcel van der Brug</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=van%20der%20Brug%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Marcel van der Brug</a></div></div><div id="_co_idm140628739633648"><h3 class="no_margin">Fang Cai</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Cai%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Fang Cai</a></div></div><div id="_co_idm140628739584480"><h3 class="no_margin">Geoffrey A Kerchner</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Kerchner%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Geoffrey A Kerchner</a></div></div><div id="_co_idm140628739582448"><h3 class="no_margin">Gai Ayalon</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Ayalon%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Gai Ayalon</a></div></div><div id="_co_idm140628739580368"><h3 class="no_margin">Baris Bingol</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Bingol%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Baris Bingol</a></div></div><div id="_co_idm140628739578336"><h3 class="no_margin">Morgan Sheng</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Sheng%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Morgan Sheng</a></div></div><div id="_co_idm140628739576304"><h3 class="no_margin">David Hinds</h3><p>  <sup>4</sup>23andMe Inc., Mountain View, California, USA</p><div>Find articles by <a href="/pubmed/?term=Hinds%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">David Hinds</a></div></div><div id="_co_idm140628739574272"><h3 class="no_margin">Timothy W Behrens</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Behrens%20TW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Timothy W Behrens</a></div></div><div id="_co_idm140628739572240"><h3 class="no_margin">Andrew B Singleton</h3><p>  <sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, US National Institutes of Health, Bethesda, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Andrew B Singleton</a></div></div><div id="_co_idm140628739570208"><h3 class="no_margin">Tushar R Bhangale</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Bhangale%20TR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Tushar R Bhangale</a></div></div><div id="_co_idm140628739567296"><h3 class="no_margin">Robert R Graham</h3><p>  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Graham%20RR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28892059">Robert R Graham</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140628741786000_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140628741786000_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140628741786000_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">  <sup>1</sup>Genentech, Inc., South San Francisco, California, USA</div><div class="fm-affl" id="A2" lang="en">  <sup>2</sup>Laboratory of Neurogenetics, National Institute on Aging, US National Institutes of Health, Bethesda, Maryland, USA</div><div class="fm-affl" id="A3" lang="en">  <sup>3</sup>Data Tecnica International, Glen Echo, Maryland, USA</div><div class="fm-affl" id="A4" lang="en">  <sup>4</sup>23andMe Inc., Mountain View, California, USA</div><div id="FN1">Correspondence should be addressed to R.R.G. (<a href="mailto:dev@null" data-email="moc.eneg@trebor.maharg" class="oemail">moc.eneg@trebor.maharg</a>)</div><div id="FN2"><sup>5</sup>A list of members appears in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note 1</a></div><div id="FN3"><sup>6</sup>Present addresses: Amgen, South San Francisco, California, USA (I.B.H.); E-Scape Bio, South San Francisco, California, USA (M.v.d.B.).</div><div id="FN4"><sup>7</sup>These authors contributed equally to this work</div><div id="FN5">Robert R Graham 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms940616ig1.gif" src="/pmc/articles/PMC5812477/bin/nihms940616ig1.gif">
<a href="http://orcid.org/0000-0001-7151-4277" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://orcid.org/0000-0001-7151-4277</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140628741786000_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div class="license half_rhythm">Reprints and permissions information is available online at <a href="http://www.nature.com/reprints/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/reprints/index.html</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28892059" target="pmc_ext" ref="reftype=publisher&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5812477/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplements.</strong></div><div><a href="/pmc/articles/PMC5812477/bin/NIHMS940616-supplement-supplements.pdf" data-ga-action="click_feat_suppl">NIHMS940616-supplement-supplements.pdf</a><span style="color:gray"> (4.6M)</span></div><div class="small guid">GUID:&nbsp;28850797-1470-4978-A48B-86C45AB991FE</div></div></dd></dl><dl data-length="566" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">A <strong>Life Sciences Reporting Summary</strong> for this paper is available. Summary statistics for the 9,830 variants presented in the discovery phase meta-analysis are available at <a href="http://research-pub.gene.com/chang_et_al_2017" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://research-pub.gene.com/chang_et_al_2017</a>. The full GWAS summary statistics for PDWBS will be made available through 23andMe and Genentech to qualified researchers under an agreement with 23andMe that protects the privacy of the 23andMe participants and an agreement with Genentech for data sharing. Please contact D.H. (<a href="http://dhinds23andme.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">dhinds23andme.com</a>) for more information and to apply to access the data.</p></dd></dl></div><div id="idm140628774051152" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140628774051152title">Abstract</h2><!--article-meta--><div><p id="P5" class="p p-first-last">Common variant genome-wide association studies (GWASs) have, to date, identified &gt;24 risk loci for Parkinson’s disease (PD). To discover additional loci, we carried out a GWAS comparing 6,476 PD cases with 302,042 controls, followed by a meta-analysis with a recent study of over 13,000 PD cases and 95,000 controls at 9,830 overlapping variants. We then tested 35 loci (<em>P</em> &lt; 1 × 10<sup>−6</sup>) in a replication cohort of 5,851 cases and 5,866 controls. We identified 17 novel risk loci (<em>P</em> &lt; 5 × 10<sup>−8</sup>) in a joint analysis of 26,035 cases and 403,190 controls. We used a neurocentric strategy to assign candidate risk genes to the loci. We identified protein-altering or <em>cis</em>–expression quantitative trait locus (<em>cis</em>-eQTL) variants in linkage disequilibrium with the index variant in 29 of the 41 PD loci. These results indicate a key role for autophagy and lysosomal biology in PD risk, and suggest potential new drug targets for PD.</p></div></div><div id="idm140628740501808" class="tsec sec headless whole_rhythm"><p id="P6" class="p p-first">PD is the second most common neurodegenerative disorder<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812292">1</a>,<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812358">2</a></sup>, with a prevalence of 3–4% in individuals over 80 years of age<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812310">3</a></sup>. PD is characterized by the loss of dopaminergic neurons in the substantia nigra and the presence of Lewy bodies<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812353">1</a>,<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812339">2</a></sup>. These neuropathologies manifest in affected individuals primarily as motor-related symptoms, but the involvement of other brain regions can lead to nonmotor symptoms<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812323">4</a></sup>.</p><p id="P7">Early-onset, familial PD (onset at &lt;60 years of age) accounts for a small fraction of cases<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812314">5</a></sup>, but the identified associated genes, including <em>LRRK2</em>, <em>GBA</em>, and <em>SNCA</em>, provide insight into disease pathogenesis<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812286">6</a>,<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812336">7</a></sup>. For the later-onset, common form of PD, at least 24 loci have been associated at a genome-wide significant level with disease risk in individuals of European ancestry<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812369">8</a></sup>. The narrow-sense heritability (<em>h</em><sup>2</sup>) explained by the confirmed PD risk loci is low (0.033)<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812293">9</a></sup>; however, the heritability explained by common variants is estimated at 0.227 (s.d.: 0.08)<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812355">9</a></sup>, which suggests that additional loci with smaller effect sizes remain to be discovered.</p><p id="P8">We carried out a GWAS of 6,476 subjects from a 23andMe PD cohort (PDWBS (Web-Based Study of Parkinson’s Disease)) and 302,042 controls genotyped on custom Illumina arrays (<a href="/pmc/articles/PMC5812477/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). The 6,476 PD cases of European ancestry were independent from those previously reported<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812366">8</a></sup> but met the same inclusion criteria, except that carriers of the <em>LRRK2</em> G2019S mutation were not removed<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812304">8</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812300">10</a></sup>. The 302,042 controls did not report having PD and were of similar ancestry as the cases. The data were imputed with Minimac2 using 1000 Genomes phase 1 haplotypes<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812373">11</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812357">12</a></sup>. Single-nucleotide polymorphisms (SNPs) with low imputation quality or that failed general quality control metrics were removed (Online Methods). After correcting for age, sex, and the top principal components (Online Methods), we observed minimal inflation for <em>P</em> values genome-wide (λ<sub>gc</sub> = 1.057; λ<sub>1000</sub> = 1.004; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5812477/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628742880224" class="figure"><a href="/pmc/articles/PMC5812477/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5812477_nihms940616f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms940616f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5812477/bin/nihms940616f1.jpg"></a></div><div id="largeobj_idm140628742880224" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5812477/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5812477/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="P54">A flow chart of the two-stage meta-analysis design. In stage 1, we carried out a meta-analysis of 9,830 SNPs between the PDWBS and PDGene studies. Thirty-five loci with <em>P</em> &lt; 1 × 10<sup>−6</sup> were carried forward into the replication-phase meta-analysis. In stage 2, we carried out a meta-analysis between the two discovery-phase studies and the NeuroX study for these 35 loci. Of these loci, 16 of the 29 available in NeuroX and 1 locus without replication data were carried forward for downstream analyses (see the main text for further details).</p></div></div></div><p id="P9">A total of 12 loci had <em>P</em> &lt; 5 × 10<sup>−8</sup> in the PDWBS analysis, including 11 of the loci that were reported in a previous GWAS in individuals of European ancestry<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812305">8</a></sup> (<a href="/pmc/articles/PMC5812477/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). For the remaining 13 previously reported loci, we observed <em>P</em> &lt; 0.05 for 11 loci, with no significant evidence for association observed in the PDWBS sample for <em>CHMP2B</em> (rs115185635) or <em>TMEM229B</em> (rs155399). The remaining novel locus in the PDWBS analysis, rs9468199 (<em>P</em> = 1.77 × 10<sup>−9</sup>), is more than 4 Mb from the nearest PD association in the HLA class II region and is independent of rs9275326 (<em>P</em><sub>conditional</sub> = 2.64 × 10<sup>−9</sup>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P57">Parkinson’s disease risk loci previously reported at genome-wide significance levels</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628741783920" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">CHR:BP<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="left">Candidate gene<sup>b</sup></th><th rowspan="1" colspan="1" align="center">Effect allele/<br>alternate<br>allele</th><th rowspan="1" colspan="1" align="center">EAF in<br>1000<br>Genomes</th><th rowspan="1" colspan="1" valign="bottom" align="left">EAF<sub>cases/controls</sub><sup>c</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><sub>PDGene</sub><sup>d</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sub>PDGene</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><sub>PDWBS</sub><sup>e</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sub>PDWBS</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><sub>discovery</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sub>discovery</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sub>discovery</sub><br>95% CI</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">1:155135036</td><td rowspan="1" colspan="1" align="left">rs35749011</td><td rowspan="1" colspan="1" align="left"><em>GBA</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.976</td><td rowspan="1" colspan="1" align="left">0.979/0.988</td><td rowspan="1" colspan="1" align="center">6.10 × 10<sup>−23</sup></td><td rowspan="1" colspan="1" align="center">0.57</td><td rowspan="1" colspan="1" align="center">5.33 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">0.59</td><td rowspan="1" colspan="1" align="center">2.59 × 10<sup>−35</sup></td><td rowspan="1" colspan="1" align="center">0.58</td><td rowspan="1" colspan="1" align="center">0.53–0.63</td></tr><tr><td rowspan="1" colspan="1" align="left">1:205723572</td><td rowspan="1" colspan="1" align="left">rs823118</td><td rowspan="1" colspan="1" align="left"><em>NUCKS1</em>, <em>SLC41A1</em></td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.467</td><td rowspan="1" colspan="1" align="left">0.419/0.443</td><td rowspan="1" colspan="1" align="center">1.96 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">0.89</td><td rowspan="1" colspan="1" align="center">8.78 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1" align="center">1.12 × 10<sup>−23</sup></td><td rowspan="1" colspan="1" align="center">0.89</td><td rowspan="1" colspan="1" align="center">0.87–0.91</td></tr><tr><td rowspan="1" colspan="1" align="left">1:232664611</td><td rowspan="1" colspan="1" align="left">rs10797576</td><td rowspan="1" colspan="1" align="left"><em>SIPA1L2</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.137</td><td rowspan="1" colspan="1" align="left">0.145/0.135</td><td rowspan="1" colspan="1" align="center">1.76 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="center">7.4 × 810<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">8.41 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="center">1.09–1.15</td></tr><tr><td rowspan="1" colspan="1" align="left">2:135539967</td><td rowspan="1" colspan="1" align="left">rs6430538</td><td rowspan="1" colspan="1" align="left"><em>TMEM163</em>,<em>CCNT2</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.488</td><td rowspan="1" colspan="1" align="left">0.426/0.450</td><td rowspan="1" colspan="1" align="center">3.35 × 10<sup>−19</sup></td><td rowspan="1" colspan="1" align="center">0.88</td><td rowspan="1" colspan="1" align="center">1.5 × 410<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">8.24 × 10<sup>−24</sup></td><td rowspan="1" colspan="1" align="center">0.89</td><td rowspan="1" colspan="1" align="center">0.87–0.91</td></tr><tr><td rowspan="1" colspan="1" align="left">2:169110394</td><td rowspan="1" colspan="1" align="left">rs1474055</td><td rowspan="1" colspan="1" align="left"><em>STK39</em></td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.881</td><td rowspan="1" colspan="1" align="left">0.855/0.874</td><td rowspan="1" colspan="1" align="center">7.11 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">0.82</td><td rowspan="1" colspan="1" align="center">1.11 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.83</td><td rowspan="1" colspan="1" align="center">5.68 × 10<sup>−26</sup></td><td rowspan="1" colspan="1" align="center">0.83</td><td rowspan="1" colspan="1" align="center">0.80–0.86</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>3:87520857</strong><sup>f</sup></td><td rowspan="1" colspan="1" align="left"><strong>rs115185635</strong></td><td rowspan="1" colspan="1" align="left"><em>CHMP2B</em></td><td rowspan="1" colspan="1" align="center"><strong>C/G</strong></td><td rowspan="1" colspan="1" align="center"><strong>0.036</strong></td><td rowspan="1" colspan="1" align="left"><strong>0.040/0.039</strong></td><td rowspan="1" colspan="1" align="center"><strong>2.2 × 10<sup>−8</sup></strong></td><td rowspan="1" colspan="1" align="center"><strong>1.79</strong></td><td rowspan="1" colspan="1" align="center"><strong>0.182</strong></td><td rowspan="1" colspan="1" align="center"><strong>1.08</strong></td><td rowspan="1" colspan="1" align="center"><strong>1.22 × 10<sup>−4</sup></strong></td><td rowspan="1" colspan="1" align="center"><strong>1.21</strong></td><td rowspan="1" colspan="1" align="center"><strong>1.10–1.33</strong></td></tr><tr><td rowspan="1" colspan="1" align="left">3:182762437</td><td rowspan="1" colspan="1" align="left">rs12637471</td><td rowspan="1" colspan="1" align="left"><em>MCCC1</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.219</td><td rowspan="1" colspan="1" align="left">0.175/0.198</td><td rowspan="1" colspan="1" align="center">5.38 × 10<sup>−22</sup></td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="center">4.27 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.86</td><td rowspan="1" colspan="1" align="center">2.11 × 10<sup>−30</sup></td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">0.82–0.87</td></tr><tr><td rowspan="1" colspan="1" align="left">4:951947</td><td rowspan="1" colspan="1" align="left">rs34311866</td><td rowspan="1" colspan="1" align="left"><em>TMEM175</em>,<em>DGKQ</em></td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.199</td><td rowspan="1" colspan="1" align="left">0.212/0.184</td><td rowspan="1" colspan="1" align="center">6.00 × 10<sup>−41</sup></td><td rowspan="1" colspan="1" align="center">1.26</td><td rowspan="1" colspan="1" align="center">2.48 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">1.18</td><td rowspan="1" colspan="1" align="center">1.47 × 10<sup>−50</sup></td><td rowspan="1" colspan="1" align="center">1.23</td><td rowspan="1" colspan="1" align="center">1.20–1.27</td></tr><tr><td rowspan="1" colspan="1" align="left">4:15737101</td><td rowspan="1" colspan="1" align="left">rs11724635</td><td rowspan="1" colspan="1" align="left"><em>FAM200B</em>, <em>CD38</em></td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.437</td><td rowspan="1" colspan="1" align="left">0.437/0.452</td><td rowspan="1" colspan="1" align="center">4.26 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" align="center">0.89</td><td rowspan="1" colspan="1" align="center">1.0 × 410<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">1.22 × 10<sup>−19</sup></td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1" align="center">0.88–0.92</td></tr><tr><td rowspan="1" colspan="1" align="left">4:77198986</td><td rowspan="1" colspan="1" align="left">rs6812193<sup>g</sup></td><td rowspan="1" colspan="1" align="left"><em>FAM47E</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.398</td><td rowspan="1" colspan="1" align="left">0.351/0.370</td><td rowspan="1" colspan="1" align="center">1.85 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">1.24 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">1.43 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">0.90–0.94</td></tr><tr><td rowspan="1" colspan="1" align="left">4:90626111</td><td rowspan="1" colspan="1" align="left">rs356182</td><td rowspan="1" colspan="1" align="left"><em>SNCA</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.375</td><td rowspan="1" colspan="1" align="left">0.406/0.349</td><td rowspan="1" colspan="1" align="center">1.85 × 10<sup>−82</sup></td><td rowspan="1" colspan="1" align="center">1.34</td><td rowspan="1" colspan="1" align="center">1.44 × 10<sup>−42</sup></td><td rowspan="1" colspan="1" align="center">1.31</td><td rowspan="1" colspan="1" align="center">5.21 × 10<sup>−123</sup></td><td rowspan="1" colspan="1" align="center">1.33</td><td rowspan="1" colspan="1" align="center">1.30–1.36</td></tr><tr><td rowspan="1" colspan="1" align="left">6:32666660</td><td rowspan="1" colspan="1" align="left">rs9275326</td><td rowspan="1" colspan="1" align="left"><em>HLA-DRB6</em>, <em>HLA-DQA1</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.114</td><td rowspan="1" colspan="1" align="left">0.099/0.105</td><td rowspan="1" colspan="1" align="center">5.81 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">0.80</td><td rowspan="1" colspan="1" align="center">1.04 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1" align="center">1.26 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">0.82–0.89</td></tr><tr><td rowspan="1" colspan="1" align="left">7:23293746</td><td rowspan="1" colspan="1" align="left">rs199347</td><td rowspan="1" colspan="1" align="left"><em>KLHL7</em>, <em>NUPL2</em>, <em>GPNMB</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.368</td><td rowspan="1" colspan="1" align="left">0.389/0.412</td><td rowspan="1" colspan="1" align="center">5.62 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1" align="center">8.66 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">3.51 × 10<sup>−18</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.89–0.93</td></tr><tr><td rowspan="1" colspan="1" align="left">8:16697091</td><td rowspan="1" colspan="1" align="left">rs591323</td><td rowspan="1" colspan="1" align="left"><em>MICU3</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.293</td><td rowspan="1" colspan="1" align="left">0.258/0.274</td><td rowspan="1" colspan="1" align="center">3.17 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">1.61 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">2.38 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.89–0.94</td></tr><tr><td rowspan="1" colspan="1" align="left">10:121536327</td><td rowspan="1" colspan="1" align="left">rs117896735</td><td rowspan="1" colspan="1" align="left"><em>BAG3</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.012</td><td rowspan="1" colspan="1" align="left">0.021/0.015</td><td rowspan="1" colspan="1" align="center">1.21 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.77</td><td rowspan="1" colspan="1" align="center">1.75 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">1.57</td><td rowspan="1" colspan="1" align="center">2.23 × 10<sup>−19</sup></td><td rowspan="1" colspan="1" align="center">1.65</td><td rowspan="1" colspan="1" align="center">1.48–1.85</td></tr><tr><td rowspan="1" colspan="1" align="left">11:83544472</td><td rowspan="1" colspan="1" align="left">rs3793947</td><td rowspan="1" colspan="1" align="left"><em>DLG2</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.463</td><td rowspan="1" colspan="1" align="left">0.431/0.442</td><td rowspan="1" colspan="1" align="center">2.59 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">8.92 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.95</td><td rowspan="1" colspan="1" align="center">3.72 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">0.91–0.95</td></tr><tr><td rowspan="1" colspan="1" align="left">11:133765367</td><td rowspan="1" colspan="1" align="left">rs329648</td><td rowspan="1" colspan="1" align="left"><em>MIR4697</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.327</td><td rowspan="1" colspan="1" align="left">0.369/0.351</td><td rowspan="1" colspan="1" align="center">8.05 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">1.11</td><td rowspan="1" colspan="1" align="center">9.16 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.07</td><td rowspan="1" colspan="1" align="center">1.11 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">1.09</td><td rowspan="1" colspan="1" align="center">1.07–1.12</td></tr><tr><td rowspan="1" colspan="1" align="left">12:40614434</td><td rowspan="1" colspan="1" align="left">rs76904798<sup>h</sup></td><td rowspan="1" colspan="1" align="left"><em>LRRK2</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.132</td><td rowspan="1" colspan="1" align="left">0.152/0.137</td><td rowspan="1" colspan="1" align="center">4.86 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1" align="center">4.10 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.14</td><td rowspan="1" colspan="1" align="center">1.21 × 10<sup>−19</sup></td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="center">1.12–1.19</td></tr><tr><td rowspan="1" colspan="1" align="left">12:123303586</td><td rowspan="1" colspan="1" align="left">rs11060180</td><td rowspan="1" colspan="1" align="left"><em>OGFOD2</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="left">0.423/0.449</td><td rowspan="1" colspan="1" align="center">3.08 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">4.95 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.88</td><td rowspan="1" colspan="1" align="center">2.05 × 10<sup>−20</sup></td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1" align="center">0.88–0.92</td></tr><tr><td rowspan="1" colspan="1" align="left">14:55348869</td><td rowspan="1" colspan="1" align="left">rs11158026</td><td rowspan="1" colspan="1" align="left"><em>GCH1</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.307</td><td rowspan="1" colspan="1" align="left">0.309/0.331</td><td rowspan="1" colspan="1" align="center">2.88 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">2.65 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1" align="center">4.30 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.89–0.93</td></tr><tr><td rowspan="1" colspan="1" align="left">14:67984370</td><td rowspan="1" colspan="1" align="left">rs1555399</td><td rowspan="1" colspan="1" align="left"><em>TMEM229B</em></td><td rowspan="1" colspan="1" align="center">T/A</td><td rowspan="1" colspan="1" align="center">0.544</td><td rowspan="1" colspan="1" align="left">0.518/0.514</td><td rowspan="1" colspan="1" align="center">5.70 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="center">0.453</td><td rowspan="1" colspan="1" align="center">1.01</td><td rowspan="1" colspan="1" align="center">9.61 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.09</td><td rowspan="1" colspan="1" align="center">1.06–1.11</td></tr><tr><td rowspan="1" colspan="1" align="left">15:61994134</td><td rowspan="1" colspan="1" align="left">rs2414739</td><td rowspan="1" colspan="1" align="left"><em>VPS13C</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.292</td><td rowspan="1" colspan="1" align="left">0.250/0.266</td><td rowspan="1" colspan="1" align="center">3.59 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1" align="center">1.1 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">3.94 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.89–0.93</td></tr><tr><td rowspan="1" colspan="1" align="left">16:31121793</td><td rowspan="1" colspan="1" align="left">rs14235</td><td rowspan="1" colspan="1" align="left"><em>ZNF646</em>, <em>KAT8</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.397</td><td rowspan="1" colspan="1" align="left">0.388/0.378</td><td rowspan="1" colspan="1" align="center">3.63 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">0.0339</td><td rowspan="1" colspan="1" align="center">1.04</td><td rowspan="1" colspan="1" align="center">5.44 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">1.06–1.10</td></tr><tr><td rowspan="1" colspan="1" align="left">17:43994648</td><td rowspan="1" colspan="1" align="left">rs17649553</td><td rowspan="1" colspan="1" align="left"><em>ARHGAP27</em>, <em>CRHR1</em>, <em>SPPL2C</em>, <em>MAPT</em>, <em>STH</em>, <em>KANSL1</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.232</td><td rowspan="1" colspan="1" align="left">0.187/0.221</td><td rowspan="1" colspan="1" align="center">6.11 × 10<sup>−49</sup></td><td rowspan="1" colspan="1" align="center">0.77</td><td rowspan="1" colspan="1" align="center">9.24 × 10<sup>−22</sup></td><td rowspan="1" colspan="1" align="center">0.80</td><td rowspan="1" colspan="1" align="center">1.26 × 10<sup>−68</sup></td><td rowspan="1" colspan="1" align="center">0.78</td><td rowspan="1" colspan="1" align="center">0.76–0.80</td></tr><tr><td rowspan="1" colspan="1" align="left">18:40673380</td><td rowspan="1" colspan="1" align="left">rs12456492</td><td rowspan="1" colspan="1" align="left"><em>SYT4</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.332</td><td rowspan="1" colspan="1" align="left">0.336/0.315</td><td rowspan="1" colspan="1" align="center">2.15 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">5.13 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">5.56 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">1.07–1.12</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>19:2363319</strong><sup>f</sup></td><td rowspan="1" colspan="1" align="left"><strong>rs62120679</strong></td><td rowspan="1" colspan="1" align="left"><strong><em>LSM7</em></strong></td><td rowspan="1" colspan="1" align="center"><strong>t/C</strong></td><td rowspan="1" colspan="1" align="center"><strong>0.324</strong></td><td rowspan="1" colspan="1" align="left"><strong>0.314/0.310</strong></td><td rowspan="1" colspan="1" align="center"><strong>2.52 × 10<sup>−9</sup></strong></td><td rowspan="1" colspan="1" align="center"><strong>1.14</strong></td><td rowspan="1" colspan="1" align="center"><strong>0.24O</strong></td><td rowspan="1" colspan="1" align="center"><strong>1.03</strong></td><td rowspan="1" colspan="1" align="center"><strong>6.64 × 10<sup>−7</sup></strong></td><td rowspan="1" colspan="1" align="center"><strong>1.08</strong></td><td rowspan="1" colspan="1" align="center"><strong>1.05–1.11</strong></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>20:3168166</strong><sup>f</sup></td><td rowspan="1" colspan="1" align="left"><strong>rs8118008</strong></td><td rowspan="1" colspan="1" align="left"><strong><em>DDRGK1</em></strong></td><td rowspan="1" colspan="1" align="center"><strong>A/G</strong></td><td rowspan="1" colspan="1" align="center"><strong>0.596</strong></td><td rowspan="1" colspan="1" align="left"><strong>0.615/0.609</strong></td><td rowspan="1" colspan="1" align="center"><strong>2.32 × 10<sup>−8</sup></strong></td><td rowspan="1" colspan="1" align="center"><strong>1.11</strong></td><td rowspan="1" colspan="1" align="center"><strong>0.283</strong></td><td rowspan="1" colspan="1" align="center"><strong>1.02</strong></td><td rowspan="1" colspan="1" align="center"><strong>1.99 × 10<sup>−6</sup></strong></td><td rowspan="1" colspan="1" align="center"><strong>1.07</strong></td><td rowspan="1" colspan="1" align="center"><strong>1.04–1.09</strong></td></tr></tbody></table></div><div id="largeobj_idm140628741783920" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5812477/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P58" class="p p-first-last">Rows in bold text refer to loci that did not pass the genome-wide significance threshold (5 × 10<sup>−8</sup>) in the discovery-phase meta-analysis.</p></div><div id="TFN2"><sup>a</sup>Chromosome and physical position according to Hg19.</div><div id="TFN3"><sup>b</sup>Details regarding the assignment of candidate genes are provided in the Online Methods.</div><div id="TFN4"><sup>c</sup>Effect allele frequency (EAF) measured in PDWBS controls or cases.</div><div id="TFN5"><sup>d</sup><em>P</em> value for SNP in the publicly available PDGene data (13,708 cases, 95,282 controls). Publicly available data for the following SNPs include an additional 5,450 cases and 5,798 controls genotyped on NeuroX: rs115185635, rs35749011, rs117896735, rs62120679, rs9275326, rs3793947, rs1555399, rs1474055, and rs8118008.</div><div id="TFN6"><sup>e</sup><em>P</em> value for SNP in PDWBS (6,476 cases, 302,042 controls).</div><div id="TFN7"><sup>f</sup>The alternate SNP is genome-wide significant (rs12651582; <em>P</em> = 3.51 × 10<sup>−8</sup>).</div><div id="TFN8"><sup>g</sup>The alternate SNP is genome-wide significant (rs76904798; <em>P</em> = 4.45 × 10<sup>−75</sup>).</div></div></div><p id="P10">Using genome-wide summary statistics from the PDWBS analysis, we estimated the <em>h</em><sup>2</sup> value for PD explained by common variants as 0.209 (95% confidence interval (CI): 0.148–0.271, assuming a prevalence of 0.01), which is similar to the <em>h</em><sup>2</sup> value reported previously<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812285">9</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812284">10</a></sup>. Regions contributing to PD heritability were significantly enriched for acetylation of histone H3 at lysine 27 (<em>P</em> = 0.001; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>), a mark of active regulatory regions. PD heritability was also enriched for histone marks in central nervous system, adrenal, and pancreatic cell types (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>), in agreement with a previous study<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812345">13</a></sup>.</p><p id="P11">We next carried out a meta-analysis between the PDWBS GWAS and results for the top 10,000 variants available from a large-scale meta-analysis for PD with over 13,000 cases and 95,000 controls<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812368">8</a></sup> (PDGene) (<a href="/pmc/articles/PMC5812477/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). For the 9,830 overlapping SNPs between the PDWBS and PDGene studies, we used an inverse-variance weighted method to combine association statistics for meta-analysis<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812307">14</a></sup>. The odds ratios and <em>P</em> values for the 9,830 overlapping SNPs in the PDWBS and PDGene studies were correlated (ρ<sub>−log10(<em>P</em> value)</sub> = 0.85, ρ<sub>OR</sub> = 0.58). Furthermore, quantile–quantile (Q-Q) plots indicated an increase in the number of variants with low <em>P</em> values (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>), even after the exclusion of variants in regions previously reported as associated with PD risk at a genome-wide significant level (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>).</p><p id="P12">The meta-analysis identified 35 loci associated at <em>P</em> &lt; 1 × 10<sup>−6</sup>, including 15 loci with <em>P</em> &lt; 5 × 10<sup>−8</sup> (<a href="/pmc/articles/PMC5812477/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 2 and 3, Supplementary Table 4</a>). Only two of the previously reported loci (<em>BCKDK:</em>rs14235 and <em>MAPT:</em>rs17649553) and 2 of the 20 suggestive loci (<em>FCGR2A:</em>rs4657041, <em>ITGA2B:</em>rs5910) were in linkage disequilibrium (LD) (<em>r</em><sup>2</sup> &gt; 0.8) with variants associated (at <em>P</em> &lt; 5 × 10<sup>−8</sup>) with any phenotype in the NHGRI GWAS catalog<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812356">15</a></sup>. Significant pleiotropy of PD risk loci with other complex diseases has not been identified<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812322">16</a></sup>, but this pleiotropy landscape may change as more modest effects are uncovered.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5812477/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628742872656" class="figure"><a href="/pmc/articles/PMC5812477/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5812477_nihms940616f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms940616f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5812477/bin/nihms940616f2.jpg"></a></div><div id="largeobj_idm140628742872656" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5812477/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5812477/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="P55">Results of the Parkinson’s disease discovery-phase meta-analysis. The top SNPs in associated regions are indicated by pink symbols. Candidate genes for previously associated loci are labeled in black (<em>P</em> &lt; 5 × 10<sup>−8</sup> in the discovery phase) or gray text (<em>P</em> &gt; 5 × 10<sup>−8</sup> in the discovery phase); candidate genes for newly identified loci are labeled in red. The <em>y</em>-axis shows the two-sided unadjusted −log<sub>10</sub>(<em>P</em>) values for association with PD. SNPs with <em>P</em> &lt; 1 × 10<sup>−25</sup> are indicated by triangles.</p></div></div></div><p id="P13">We next sought validation of these 35 candidate loci in an independent cohort of 5,851 cases and 5,866 controls of European ancestry genotyped with the semi-customized NeuroX Illumina array<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812362">8</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_666529105">17</a></sup> (<a href="/pmc/articles/PMC5812477/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Twenty-nine of the 35 loci either were directly genotyped on the NeuroX array or had suitable proxies (<em>r</em><sup>2</sup> &gt; 0.9 with the original SNP; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). Weaker proxies at four additional SNPs (<em>r</em><sup>2</sup> &gt; 0.5) were available but were not used for validation in this study (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). In a replication-phase joint analysis of these 29 loci (meta-analysis of PDGene, PDWBS, and NeuroX), 16 had <em>P</em> &lt; 5 × 10<sup>−8</sup> (<a href="/pmc/articles/PMC5812477/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Of these 16, all but 3 (rs4073221, rs10906923, and rs9468199) were also nominally associated in the NeuroX study (one-sided <em>P</em> &lt; 0.05). A genetic risk score<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812352">8</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812338">18</a>,<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812375">19</a></sup> defined by these 16 loci, in addition to the previously reported loci, had a non-negligible ability to predict PD case status (area under the curve, 0.6518; 95% CI, 0.6419–0.6616). This represents a significant improvement over the predictive power of risk scores defined by previously reported loci alone (<em>P</em> = 6 × 10<sup>−8</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note 1</a>). In sum, we identified 16 independent PD risk loci with a joint <em>P</em> &lt; 5 × 10<sup>−8</sup> and 1 locus (rs601999) with <em>P</em> &lt; 5 × 10<sup>−8</sup> in the discovery cohort with no suitable proxy for replication in the NeuroX cohort (<a href="/pmc/articles/PMC5812477/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P66">Seventeen novel regions associated with Parkinson’s disease at genome-wide significance levels</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628741195104" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">CHR:BP<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Candidate<br>gene<sup>b</sup></th><th rowspan="1" colspan="1" align="center">Effect allele/<br>alternate<br>allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">EAF in 1000<br>Genomes</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><sub>discovery</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sub>discovery</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><sub>NeuroX</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sub>NeuroX</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><sub>joint</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sub>Joint</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sub>Joint</sub> (95%<br>CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">1:226916078</td><td rowspan="1" colspan="1" align="center">rs4653767</td><td rowspan="1" colspan="1" align="center"><em>ITPKB</em></td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.315</td><td rowspan="1" colspan="1" align="center">2.40 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">0.017</td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">1.63 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">0.90–0.94</td></tr><tr><td rowspan="1" colspan="1" align="left">2:102413116</td><td rowspan="1" colspan="1" align="center">rs34043159</td><td rowspan="1" colspan="1" align="center"><em>IL1R2</em></td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.352</td><td rowspan="1" colspan="1" align="center">3.83 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1.07</td><td rowspan="1" colspan="1" align="center">1.91 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.11</td><td rowspan="1" colspan="1" align="center">5.48 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">1.06–1.10</td></tr><tr><td rowspan="1" colspan="1" align="left">2:166133632</td><td rowspan="1" colspan="1" align="center">rs353116</td><td rowspan="1" colspan="1" align="center"><em>SCN3A</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.385</td><td rowspan="1" colspan="1" align="center">9.73 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">0.94</td><td rowspan="1" colspan="1" align="center">8.98 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">2.98 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.94</td><td rowspan="1" colspan="1" align="center">0.92–0.96</td></tr><tr><td rowspan="1" colspan="1" align="left">3:18277488</td><td rowspan="1" colspan="1" align="center">rs4073221</td><td rowspan="1" colspan="1" align="center"><em>SATB1</em></td><td rowspan="1" colspan="1" align="center">G/T</td><td rowspan="1" colspan="1" align="center">0.132</td><td rowspan="1" colspan="1" align="center">3.02 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">1.11</td><td rowspan="1" colspan="1" align="center">0.583</td><td rowspan="1" colspan="1" align="center">1.02</td><td rowspan="1" colspan="1" align="center">1.57 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">1.06–1.13</td></tr><tr><td rowspan="1" colspan="1" align="left">3:48748989</td><td rowspan="1" colspan="1" align="center">rs12497850</td><td rowspan="1" colspan="1" align="center"><em>NCKIPSD</em>, <em>CDC71</em></td><td rowspan="1" colspan="1" align="center">G/T</td><td rowspan="1" colspan="1" align="center">0.347</td><td rowspan="1" colspan="1" align="center">6.80 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">0.040</td><td rowspan="1" colspan="1" align="center">0.94</td><td rowspan="1" colspan="1" align="center">9.16 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">0.91–0.96</td></tr><tr><td rowspan="1" colspan="1" align="left">3:52816840</td><td rowspan="1" colspan="1" align="center">rs143918452</td><td rowspan="1" colspan="1" align="center"><em>ALAS1</em>, <em>TLR9</em>, <em>DNAH1</em>, <em>BAP1</em>, <em>PHF7</em>, <em>NISCH</em>, <em>STAB1</em>, <em>ITIH3</em>, <em>ITIH4</em></td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.996</td><td rowspan="1" colspan="1" align="center">2.25 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">0.68</td><td rowspan="1" colspan="1" align="center">0.095</td><td rowspan="1" colspan="1" align="center">0.73</td><td rowspan="1" colspan="1" align="center">3.20 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.68</td><td rowspan="1" colspan="1" align="center">0.60–0.78</td></tr><tr><td rowspan="1" colspan="1" align="left">4:114360372</td><td rowspan="1" colspan="1" align="center">rs78738012</td><td rowspan="1" colspan="1" align="center"><em>ANK2</em>, <em>CAMK2D</em></td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.106</td><td rowspan="1" colspan="1" align="center">2.11 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">1.14</td><td rowspan="1" colspan="1" align="center">7.5 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="center">4.78 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="center">1.09–1.17</td></tr><tr><td rowspan="1" colspan="1" align="left">5:60273923</td><td rowspan="1" colspan="1" align="center">rs2694528</td><td rowspan="1" colspan="1" align="center"><em>ELOVL7</em></td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.115</td><td rowspan="1" colspan="1" align="center">1.69 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="center">6.25 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.19</td><td rowspan="1" colspan="1" align="center">4.84 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="center">1.11–1.20</td></tr><tr><td rowspan="1" colspan="1" align="left">6:27681215</td><td rowspan="1" colspan="1" align="center">rs9468199</td><td rowspan="1" colspan="1" align="center"><em>ZNF184</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.172</td><td rowspan="1" colspan="1" align="center">3.44 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="center">0.302</td><td rowspan="1" colspan="1" align="center">1.04</td><td rowspan="1" colspan="1" align="center">1.46 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">1.11</td><td rowspan="1" colspan="1" align="center">1.08–1.14</td></tr><tr><td rowspan="1" colspan="1" align="left">8:11707174</td><td rowspan="1" colspan="1" align="center">rs2740594<sup>c</sup></td><td rowspan="1" colspan="1" align="center"><em>CTSB</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.753</td><td rowspan="1" colspan="1" align="center">9.54 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">7.95 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">5.91 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">1.09</td><td rowspan="1" colspan="1" align="center">1.07–1.12</td></tr><tr><td rowspan="1" colspan="1" align="left">8:22525980</td><td rowspan="1" colspan="1" align="center">rs2280104</td><td rowspan="1" colspan="1" align="center"><em>SORBS3</em>, <em>PDLIM2</em>, <em>C8orf58</em>, <em>BIN3</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.367</td><td rowspan="1" colspan="1" align="center">9.06 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.06</td><td rowspan="1" colspan="1" align="center">7.87 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">2.53 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1.07</td><td rowspan="1" colspan="1" align="center">1.04–1.09</td></tr><tr><td rowspan="1" colspan="1" align="left">9:17579690</td><td rowspan="1" colspan="1" align="center">rs13294100</td><td rowspan="1" colspan="1" align="center"><em>SH3GL2</em></td><td rowspan="1" colspan="1" align="center">T/G</td><td rowspan="1" colspan="1" align="center">0.371</td><td rowspan="1" colspan="1" align="center">1.99 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.037</td><td rowspan="1" colspan="1" align="center">0.94</td><td rowspan="1" colspan="1" align="center">4.84 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">0.89–0.94</td></tr><tr><td rowspan="1" colspan="1" align="left">10:15569598</td><td rowspan="1" colspan="1" align="center">rs10906923</td><td rowspan="1" colspan="1" align="center"><em>FAM171A1</em></td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.306</td><td rowspan="1" colspan="1" align="center">2.37 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">0.133</td><td rowspan="1" colspan="1" align="center">0.96</td><td rowspan="1" colspan="1" align="center">1.35 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">0.91–0.96</td></tr><tr><td rowspan="1" colspan="1" align="left">14:88472612</td><td rowspan="1" colspan="1" align="center">rs8005172</td><td rowspan="1" colspan="1" align="center"><em>GALC</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.424</td><td rowspan="1" colspan="1" align="center">1.20 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">1.06</td><td rowspan="1" colspan="1" align="center">8.77 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">1.05–1.10</td></tr><tr><td rowspan="1" colspan="1" align="left">16:19279464</td><td rowspan="1" colspan="1" align="center">rs11343</td><td rowspan="1" colspan="1" align="center"><em>COQ7</em></td><td rowspan="1" colspan="1" align="center">T/G</td><td rowspan="1" colspan="1" align="center">0.454</td><td rowspan="1" colspan="1" align="center">1.46 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">1.07</td><td rowspan="1" colspan="1" align="center">0.019</td><td rowspan="1" colspan="1" align="center">1.06</td><td rowspan="1" colspan="1" align="center">9.13 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.07</td><td rowspan="1" colspan="1" align="center">1.05–1.10</td></tr><tr><td rowspan="1" colspan="1" align="left">16:52599188</td><td rowspan="1" colspan="1" align="center">rs4784227</td><td rowspan="1" colspan="1" align="center"><em>TOX3</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.265</td><td rowspan="1" colspan="1" align="center">8.29 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">1.47 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="center">9.75 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.09</td><td rowspan="1" colspan="1" align="center">1.06–1.12</td></tr><tr><td rowspan="1" colspan="1" align="left">17:40698158</td><td rowspan="1" colspan="1" align="center">rs601999</td><td rowspan="1" colspan="1" align="center"><em>ATP6V0A1</em>, <em>PSMC3IP</em>, <em>TUBG2</em></td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.699</td><td rowspan="1" colspan="1" align="center">8.03 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td></tr></tbody></table></div><div id="largeobj_idm140628741195104" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5812477/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN9"><p id="P67" class="p p-first">Summary statistics are shown for the discovery cohort (PDWBS and PDGene), NeuroX (5,851 cases, 5,866 controls), and the joint meta-analysis of the discovery and NeuroX data.</p><p id="P68" class="p p-last">Additional summary statistics for NeuroX and the joint meta-analysis are available in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>. EAF, effect allele frequency.</p></div><div id="TFN10"><sup>a</sup>Chromosome and physical position according to Hg19.</div><div id="TFN11"><sup>b</sup>Details regarding the assignment of candidate genes are provided in the Online Methods.</div><div id="TFN12"><sup>c</sup>NeuroX and joint statistics are shown for proxy SNP rs1293298.</div></div></div><p id="P14">Overall, 11 of 17 novel loci were in high LD (<em>r</em><sup>2</sup> &gt; 0.8) with at least one variant predicted to affect transcription factor binding (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>). Of the 17 novel loci and 24 previously reported loci, 10 contained residual associations with <em>P</em> &lt; 1 × 10<sup>−3</sup> after conditioning on each region’s most significant SNP in the PDWBS data (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>). These regions included three of the four independent secondary signals reported by Nalls <em>et al.</em><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812308">8</a></sup>, as well as one variant previously reported at a non-genome-wide significant level (<em>P</em> = 5.15 × 10<sup>−7</sup>)<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812330">20</a></sup>.</p><p id="P15">We note that the HLA region association with PD is particularly complex. Two candidate genes from the HLA region were nominated on the basis of support from either a protein-coding variant or an eQTL (<a href="/pmc/articles/PMC5812477/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). This is in line with a previous study that suggested that the PD association in the HLA region may point to multiple HLA factors, including independent regulatory factors<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812374">21</a></sup>. The association pattern observed at this locus may be reminiscent of the HLA association observed in schizophrenia and linked to C4 copy number<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812346">22</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC5812477/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140628742868160" class="figure"><a href="/pmc/articles/PMC5812477/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5812477_nihms940616f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms940616f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5812477/bin/nihms940616f3.jpg"></a></div><div id="largeobj_idm140628742868160" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5812477/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC5812477/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="P56">The candidate genes for regions associated with Parkinson’s disease. The most likely candidate gene is annotated for each region that was significantly associated with PD in the final joint analysis. Black or gray text indicates previously reported loci that had <em>P</em> values less than or greater than 5 × 10<sup>−8</sup> in the discovery phase, respectively. Red text indicates newly identified loci that were significantly associated with PD in the final joint analysis. Gray lines at the outer edge spanning multiple genes indicate candidate genes within a single locus. Chromosome numbers are shown in the gray shaded ring, and support for candidate genes is indicated by color-coding in the inner rings. The innermost ring indicates expression of the gene in brain cell types (in a mouse expression data set) or in human brain regions (in GTEx), or differential expression between PD brains and healthy control brains.</p></div></div></div><p id="P16">The identification of the causal variants and genes underlying regions associated with common, complex disease is a major challenge<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812341">23</a></sup>. Several statistical methods have been proposed for the fine-mapping of causal variants<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812347">23</a>–<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812335">25</a></sup>. Alternatively, some studies have narrowed down lists of candidate genes by combining multiple levels of evidence with scoring-based strategies<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812316">26</a>,<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812313">27</a></sup>. Here we implemented a neurocentric strategy to nominate candidate genes for PD-associated loci.</p><p id="P17">We incorporated seven sources of data to annotate the index variant and linked variants from PD-associated loci (including eQTLs and expression data from GTEx<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812342">28</a></sup>, as well as expression data from brain cell types in mice<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812319">29</a></sup>; a full list is provided in the Online Methods). We used a two-stage approach to assign candidate genes to each locus (see the Online Methods for further details, and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a> for a graphical visualization). In the first stage, we assigned a gene to a locus if (i) the index SNP or linked variants (<em>r</em><sup>2</sup> &gt; 0.6) altered the protein sequence or (ii) the index variant was a <em>cis</em>-eQTL for the gene. When no candidate genes were identified by the first stage, we ranked neighboring gene(s) on the basis of neurologically related phenotypes and expression and assigned the gene with the highest score to the locus (Online Methods).</p><p id="P18">With this strategy we identified a single candidate gene for 28 loci, and multiple candidate genes with similar levels of supporting evidence for 13 loci (<a href="/pmc/articles/PMC5812477/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 5 and 6</a>). The candidate-gene nomination strategy confirmed several known PD risk genes, including <em>GBA</em>, <em>LRRK2</em>, <em>SNCA</em>, and <em>MAPT</em>. Among the 41 PD risk loci, a total of 29 loci (71%) had either a protein-altering or a <em>cis</em>-eQTL variant linked to the index SNP (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 8 and 9</a>). In addition, we carried out a colocalization analysis to determine whether the GWAS signal and the eQTL signal pointed to the same causal variant<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812333">30</a></sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note 1</a>). Seven candidate genes also had evidence for protein–protein interaction (Online Methods, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 10</a>). Further studies are needed to experimentally determine the causal genes in the PD risk loci; however, the identification of candidate genes provides testable hypotheses for functional studies.</p><p id="P19">To gain insight into the biology, we tested the identified candidate genes in the 41 PD risk loci for association with any pathways or gene sets compared with a background gene list (Online Methods). We investigated whether candidate genes were enriched for pathways previously implicated in PD: autophagy, lysosomal, and mitochondrial biology<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812302">1</a></sup>. PD-associated signals were enriched (at a threshold of <em>P</em> &lt; 0.05/3 = 0.017) for lysosomal and autophagy genes (<em>P</em> = 3.35 × 10<sup>−6</sup> and <em>P</em> = 5.71 × 10<sup>−3</sup>, respectively). The addition of candidate genes more than doubled the number of lysosomal genes observed in PD loci and improved the enrichment significance (<em>P</em><sub>all_loci</sub> = 3.35 × 10<sup>−6</sup>, <em>P</em><sub>novel_loci</sub> = 3.64 × 10<sup>−5</sup>). We also observed that one previously identified gene (<em>MCCC1</em>) and two novel candidate genes (<em>COQ7</em> and <em>ALAS1</em>) mapped to the mitochondrial gene set (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 11</a>).</p><p id="P20">Lysosomal biology and its role in the degradation of protein aggregates emerged as a highly significant pathway in PD risk. Among the five candidate genes linked to lysosomal biology, two were previously identified candidate genes (<em>GBA</em> (glucocerebrosidase) and <em>TMEM175</em> (transmembrane protein 175)), and three were newly identified candidate genes (<em>CTSB</em> (cathepsin B), <em>ATP6V0A1</em> (ATPase H<sup>+</sup> transporting V0 subunit a1), and <em>GALC</em> (galactosylceramidase)). Glucocerebrosidase is required for normal lysosomal activity and α-synuclein degradation. In addition, <em>GBA</em> loss-of-function alleles are a common PD risk factor<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812354">31</a></sup>. <em>TMEM175</em> was recently shown to encode a potassium channel that can regulate lysosomal function<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812328">32</a></sup>, and the missense variant <em>TMEM175</em> M393T is strongly linked to the index variant in the region (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>). <em>CTSB</em> is a lysosomal cysteine protease. A PD risk allele is linked to a <em>cis</em>-eQTL for <em>CTSB</em> in multiple tissues (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>), where the risk allele is associated with reduced levels of <em>CTSB</em> mRNA. Double-knockout mice for <em>Ctsb</em> and <em>Ctsl</em> (cathepsin L) show a tremor phenotype with cerebral and cerebellar atrophy<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812290">33</a></sup>. CTSB is also capable of degrading membrane-bound and soluble α-synuclein in mice<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812364">34</a></sup>.</p><p id="P21">Autophagy is the catabolic process that targets long-lived proteins and dysfunctional organelles for lysosomal degradation. Autophagy and lysosomal degradation have been implicated in PD by rare familial and common GWAS-associated <em>GBA</em> variants. We note that a strong <em>cis</em>-eQTL for lysine acetyltransferase 8 (<em>KAT8</em>) is associated with PD risk, with lower levels of <em>KAT8</em> mRNA linked to increased PD risk. Inhibition of KAT8 was recently shown to decrease autophagic flux<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812344">35</a></sup>.</p><p id="P22">Next, we used INRICH<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812361">36</a></sup> to investigate whether PD-associated regions were enriched for gene sets in an unbiased fashion. Once again, we found significant enrichment of the lysosomal pathway (<em>P</em><sub>adjusted</sub> = 0.02) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 12</a>). We further examined the expression of the PD candidate genes in a brain-specific cell-type expression data set in mice<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812324">29</a></sup>; however, we observed broad expression across the major brain cell types, and no clear cell-type-specific pattern was evident (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 7</a>).</p><p id="P23">Among the candidate genes newly identified in this study is <em>SH3GL2</em> (SH3 domain-containing GRB2-like 2, endophilin A1), a gene recently demonstrated to be phosphorylated by LRRK2 and which may have a role in clathrin-mediated endocytosis of synaptic vesicles<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812367">37</a></sup>. Dysregulation of <em>Elovl7</em> (elongation of very long chain fatty acids protein 7) in mice results in several neurological phenotypes, including inflammatory astrocytosis and microgliosis in the brain, and neuronal degeneration<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812363">38</a></sup>. Upregulation of the candidate gene <em>SCN3A</em> (sodium voltage-gated channel α-subunit 3) enhances neuronal excitability and is associated with epilepsy in both humans and animal models<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812343">39</a></sup>.</p><p id="P24">The new loci also encode three transcription factors: SATB1, ZNF184, and TOX3. TOX3 has been implicated in neuronal survival<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812288">40</a></sup>, and SATB1 has been associated with T cell function, particularly the development of regulatory T cells<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812327">41</a></sup>.</p><p id="P25">Several of the PD candidate genes are within the ‘druggable’ genome<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812299">42</a></sup>, including the previously identified serine/threonine kinase 39 (<em>STK39</em>) and the novel candidate gene inositol 1,4,5-trisphosphate kinase B (<em>ITPKB</em>). An in-frame deletion of <em>ITPKB</em> (rs147889095) is linked to a PD-associated variant, and complete loss of ITPKB was reported in a patient with common-variable immunodeficiency<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812298">43</a></sup>. STK39 is a kinase linked to hypertension<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812309">44</a></sup>, regulation of K<sup>+</sup> levels, and the cellular stress response.</p><p id="P26" class="p">In summary, this study presents what to our knowledge is the largest meta-analysis of PD so far, involving a total of 26,035 cases and 403,190 controls. We identified 17 novel PD loci and, using a neurocentric candidate-gene nomination pipeline, found that several of the newly identified PD risk genes have a role in lysosomal biology and autophagy. The identification of these candidate genes allows for the prioritization of functional studies to determine causal genes for PD and possible therapeutic targets.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">ONLINE METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">PDWBS GWAS</h3><p id="P29" class="p p-first">The PDWBS is a genome-wide analysis of 6,476 PD cases and 302,042 control subjects, all of whom were customers of 23andMe Inc. and consented to participate in research. The study protocol was approved by the external AAHRPP-accredited institutional review board, Ethical and Independent Review Services (E&amp;I Review). Cases and controls were designated on the basis of surveys<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812365">10</a></sup>. Controls were selected from 23andMe Inc. research participants who did not self-report as having been diagnosed with PD. Although the use of self-reported controls can result in a reduction of power, the effect of this on the current study was probably minimal (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note 1</a>). Any samples present in the PDGene study<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812294">8</a></sup> were removed from the PDWBS analysis. The average age of cases and controls was 67.6 and 50.8 years, respectively. The study also included 147 cases (2.3%) and 554 controls (0.18%) that were <em>LRRK2</em> G2019S carriers. Removing <em>LRRK2</em> G2019S carriers from the analysis removed genome-wide significant associations at the <em>LRRK2</em> locus.</p><p id="P30">DNA extraction and genotyping were performed on saliva samples by CLIA-certified CAP-accredited clinical laboratories of the Laboratory Corporation of America. Samples were genotyped on one of the following four platforms: V1 and V2, two variants of the Illumina HumanHap550+ BeadChip, with ~25,000 custom SNPs and ~950,000 total SNPs; V3, Illumina OmniExpress+BeadChip with custom SNPs to increase overlap with the V2 chip, with a total of ~950,000 SNPs; and V4, a custom chip that included SNPs overlapping V2 and V3 chips, low-frequency coding variants and ~570,000 SNPs. Samples with a call rate lower than 98.5% were reanalyzed, and research participants with samples that failed repeatedly were re-contacted and asked to provide additional samples.</p><p id="P31">Research participants were restricted to those of mainly (&gt;97%) European ancestry<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812321">10</a>,<a href="#R45" rid="R45" class=" bibr popnode">45</a></sup>. All research participants in the study were also required to share &lt;700 cM identity by descent (IBD) (estimated by a segmental IBD estimation algorithm<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812287">46</a></sup>), corresponding approximately to the sharing expected between first cousins. We additionally excluded individuals who shared &gt;700 cM IBD with any 23andMe research participant whose data was used in the PDGene GWAS. Data were imputed on 1000 Genomes phase 1 haplotypes (September 2013 release) with Minimac2 on default settings<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812377">11</a>,<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812297">47</a></sup>. Imputation was run separately on data from each genotyping platform.</p><p id="P32">For genotyped SNPs, SNPs were removed if they were genotyped on only the V1 and/or the V2 chip, if they failed a parent–offspring transmission test on trio data, if they were not in Hardy–Weinberg equilibrium (<em>P</em> &lt; 10<sup>−20</sup>), or if they had a call rate &lt; 0.90. For imputed SNPs, SNPs were removed if they had an average <em>r</em><sup>2</sup> &lt; 0.5 or minimum <em>r</em><sup>2</sup> &lt; 0.3 in any imputation batch, or failed a test for imputation batch effect (testing imputation dosage with imputation batch; <em>P</em> &lt; 10<sup>−50</sup>).</p><p id="P33" class="p p-last">We applied logistic regression assuming an additive model to test for association between case/control status and either genotypes or imputed dosages (for imputed SNPs). Only SNPs with minor allele frequency (MAF) &gt; 0.1% were analyzed. Covariates were added to adjust for age, sex, the first five principal components, and genotyping platform version. A total of 12,896,220 variants (11,933,700 SNPs) were analyzed. The genomic inflation factor was calculated from the median <em>P</em> value of analyzed variants. Scaling of the genomic inflation factor by sample size was carried out as described previously for 1,000 cases and 1,000 controls<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812306">48</a></sup>.</p></div><div id="S4" class="sec"><h3 id="S4title">Meta-analysis of PD GWASs</h3><p id="P34" class="p p-first">Summary odds ratios, 95% CIs, and <em>P</em> values of the 10,000 most significant GWAS meta-analysis results were obtained from PDGene (“URLs”). Cohort descriptions, quality control, and meta-analysis for this study have been described previously<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812372">8</a></sup>. SNP s.e. was derived from the reported <em>P</em> values and odds ratios. More specifically, the <em>z</em>-statistic was calculated as the square root of the inverse χ-square transformation of the <em>P</em> value, and the s.e. was calculated as follows: s.e.m. = ln(odds ratio)/absolute(<em>z</em>-score).</p><p id="P35" class="p p-last">There were 9,830 SNPs in common between the PDGene and the PDWBS data sets. A fixed-effects model based on inverse-variance weighting, as implemented in METAL, was used to combine summary statistics from the two studies<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812348">14</a></sup>. Heterogeneity values (<em>I</em><sup>2</sup> and <em>Q</em>) were obtained with PLINK<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812291">49</a></sup>. Novel signals of association were defined as genome-wide significant associations in the meta-analysis that did not overlap loci associated with PD at genome-wide significant thresholds in the PDGene data (35 loci with <em>P</em> &lt; 1 × 10<sup>−6</sup>).</p></div><div id="S5" class="sec"><h3 id="S5title">Joint analysis with NeuroX</h3><p id="P36" class="p p-first-last">The NeuroX cohort was previously described<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812331">8</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_666529106">17</a></sup>. Briefly, 5,851 cases and 5,866 controls of European ancestry were genotyped on a semi-custom NeuroX array. A logistic regression was carried out to test for association, with covariates to adjust for age, sex, and population ancestry (the first five principal components). Twenty-five of the 35 novel loci were directly genotyped on the chip, and four additional SNPs had suitable proxies (<em>r</em><sup>2</sup> &gt; 0.9). At these 29 SNPs, we carried out a fixed-effects inverse-variance weighted meta-analysis14 for all three studies (PDGene, PDWBS, and NeuroX) as described above.</p></div><div id="S6" class="sec"><h3 id="S6title">Conditional analysis</h3><p id="P37" class="p p-first-last">Conditional analysis was run on all 17 loci that were significantly associated with PD in the joint meta-analysis (<em>P</em> &lt; 5 × 10<sup>−8</sup>) and the 24 previously reported PD loci using the PDWBS study. For each locus, SNPs within 500 kb of the index SNP (the SNP with the most significant <em>P</em> value) were tested for association by the same methods as described above for the PDWBS GWAS with the index SNP added as an additional covariate.</p></div><div id="S7" class="sec"><h3 id="S7title">Heritability estimates</h3><p id="P38" class="p p-first-last">We used LD score regression (LDSC)<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812332">50</a>,<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812296">51</a></sup> to compute the narrow-sense heritability (<em>h</em><sup>2</sup>) estimates of PD in the PDWBS GWAS data (described above). Several methods exist for estimating <em>h</em><sup>2</sup> with GWAS data<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812334">50</a>–<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812329">52</a></sup>. We used LDSC to estimate <em>h</em><sup>2</sup> in this study because it requires only summary-level data and is more computationally efficient for larger data sets. Reference LD scores were computed with the European ancestry subset of the 1000 Genomes data for SNPs within 500 kb of the SNP to be scored. Strict filtering was applied to ensure the robustness of heritability estimates as recommended<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812326">50</a>,<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812371">51</a></sup>. After filtering, <em>Z</em>-scores for 7,629,099 SNPs from the 23andMe study were used as input to LDSC. We further used the stratified LD-score regression approach to partition heritability into 24 different cell-type-agnostic annotation categories including conserved regions, histone marks, DNase I hypersensitivity sites, ENCODE chromatin states, and enhancers<sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812350">51</a></sup>, as well as 10 different cell-type-specific histone annotations. Significant enrichment was assessed at a strict Bonferroni threshold of 0.0021 (0.05/24) for the 24 general categories, and 0.005 for the cell-type-specific enrichment.</p></div><div id="S8" class="sec"><h3 id="S8title">Pleiotropy analysis: overlap with EBI-NHGRI GWAS catalog</h3><p id="P39" class="p p-first-last">Data were downloaded from the EBI-NHGRI catalog15 (version available on 17 April 2016). If a variant in the meta-analysis was within 500 kb and in LD (<em>r</em><sup>2</sup> &gt; 0.8) with an association (<em>P</em> &lt; 5 × 10<sup>−8</sup>) in the catalog, the meta-analysis signal was considered to be overlapping the reported signal.</p></div><div id="S9" class="sec"><h3 id="S9title">A neurocentric strategy to identify candidate causal variants and genes</h3><p id="P40" class="p p-first">Associated index SNPs were paired to candidate genes on the basis of two broad levels of evidence: variant-level support and gene-level support (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a> for a graphic representation). In the former category, index SNPs were paired with candidate genes if there was evidence that the index SNP or an SNP in LD (<em>r</em><sup>2</sup> &gt; 0.6) with the index SNP was annotated with a putative high-impact variant (chromosome number variation, exon loss variant, frame-shift variant, rare amino acid variant, splice donor or acceptor variant, start-lost, stop-gained or stop-lost, and transcript ablation) or moderate-impact variant (3′ or 5′ UTR truncation and exon loss, coding sequence variant, disruptive in-frame deletion or insertion, in-frame deletion or insertion, missense variant, regulatory region ablation, splice region variant, and transcription factor binding-site ablation). We obtained variant annotations by running SnpEff<sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812315">53</a></sup> on dbSNP build 142. A second source of variant-level support consisted of <em>cis</em>-eQTL evidence. <em>Cis</em>-eQTLs as pre-computed by GTEx (v6)<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812318">28</a></sup> were downloaded directly from the GTEx portal (“URLs”). Although eQTL results were available for 46 tissues, including ten regions from the brain, our search for eQTLs was limited by the sampled tissues and cell types, and therefore we might have missed any eQTLs that are cell-type or tissue specific, in addition to eQTLs that are present only under certain stimuli (for example, ‘response’ eQTLs). The index SNP was tested for significant association with any gene where the TSS was within 250 kb of the index SNP. As roughly 90% of eQTLs are within 250 kb of a gene<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812295">28</a></sup>, it is likely that we captured the majority of eQTLs while missing rarer, more distal events. For brain eQTLs, a strict Bonferroni correction was applied to the raw eQTL <em>P</em> values to adjust for multiple testing of genes within 250 kb of the index SNP. For other tissues, only eQTLs with a false discovery rate of &lt;0.05 as determined by GTEx<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812289">28</a></sup> were considered. We weighted brain and non-brain eQTLs equally.</p><p id="P41" class="p p-last">Gene-level support was used when an index SNP had no candidate genes supported by variant-level data as described above. A list of genes within 250 kb of the index SNP was obtained (gene models used by GTEx were downloaded from the GTEx portal), and each gene was scored for neurological relevant features or annotations. Genes were first weighted for neurological relevant phenotypic annotations. Genes were (i) scored for being differentially expressed between PD patients and healthy controls (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note 1</a> for further details) (311 genes total genome-wide); (ii) annotated with ‘neuro’-associated phenotypes in FlyBase<sup><a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812337">54</a></sup> (1,521 genes); (iii) scored for behavioral, neurological, and olfactory phenotypes annotated in MGI55 (3,890 genes); and (iv) annotated with any phenotypes related to neurological disorders or the brain in OMIM<sup><a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812325">56</a></sup> (521 genes). Lastly, genes were scored for being expressed (median expression across samples &gt; 2 reads per kilobase per million mapped reads) in any cohort of GTEx brain region samples (15,197 genes) or in at least one brain cell type in the mouse expression data set<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812351">29</a></sup> (astrocyte, microglia, neuron, or oligodendrocyte) (12,092 genes). For the gene-level support, we used a tiered scoring scheme to weight phenotypic annotations more heavily than expression in the brain (scores demarked in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>) to enrich for genes with demonstrated neurological related roles. At each locus, the gene (or tied genes) with the highest score was nominated as the candidate gene for the region.</p></div><div id="S10" class="sec"><h3 id="S10title">Protein–protein and coexpression analysis</h3><p id="P42" class="p p-first-last">All protein-coding genes within 250 kb of PD-associated loci (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 13</a>) were used as input to STRING<sup><a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812301">57</a></sup>. Gene pairs that were either coexpressed or involved in experimentally validated protein–protein interactions with a medium score or higher (score ≥ 0.4) are reported in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 10</a>.</p></div><div id="S11" class="sec"><h3 id="S11title">Pathway enrichment analysis</h3><p id="P43" class="p p-first">Previously reported PD loci and novel PD associations were tested for enrichment in particular pathways or gene sets. First, the nominated candidate genes for these PD-associated loci were tested for enrichment in several targeted gene sets by a hypergeometric test. The background list of genes for comparison was matched to the neurological-centric candidate-gene nomination pipeline. The background list thus consisted of genes that had mouse knockout phenotypes, had fly mutant phenotypes, had OMIM-related phenotype annotations, had nominally significant <em>cis</em>-eQTLs in GTEx, were differentially expressed in PD patients versus controls, and were expressed in GTEx brain tissue or mouse brain cell types in the Barres data set.</p><p id="P44">We obtained mitochondrial genes from MitoMiner<sup><a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812359">58</a></sup> using the MitoCarta<sup><a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812317">59</a>,<a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812360">60</a></sup> reference set after excluding genes that mapped to the mitochondria (genes that map to the mitochondria were not included in this metaanalysis). Lysosomal genes were obtained from the hlGDB<sup><a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812311">61</a></sup> using only the proteomics and literature resources. Finally, we obtained autophagy genes from the Human Autophagy Database<sup><a href="#R62" rid="R62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812376">62</a></sup>, as well as ten additional genes reported in a recent siRNA screen of autophagic flux modulators<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812312">35</a></sup>. A list of all genes in each pathway is provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note 1</a>. The minimum <em>P</em> value per gene (for genes that an SNP within the 9,830 variants assayed in this meta-analysis mapped to) is provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 14–16</a>.</p><p id="P45">Second, we applied a non-targeted gene-set enrichment approach using INRICH<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812349">36</a></sup> to assess whether regions associated with PD were enriched for genes in KEGG<sup><a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812303">63</a></sup> and Gene Ontology (GO)<sup><a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812340">64</a></sup> gene sets. The 24 previously reported PD index variants and the 17 novel PD-associated variants reported in this study were used as input into PLINK’s<sup><a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812370">65</a></sup> “show-tags” function. The European 1000 Genomes<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_665812320">12</a></sup> samples were used for reference LD patterns. An interval for each PD-associated variant was defined as the region from the leftmost tag variant to the rightmost tag variant in the 1000 Genomes data.</p><p id="P46" class="p p-last">We ran INRICH on these 41 intervals with the default settings, with the exception of increasing the number of replicates and bootstraps to 5,000 (-r 5000 --q 5000) and setting the pre-compute feature to false for software stability (-c). Enrichment for KEGG and GO gene sets was assessed separately.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">Data availability</h3><p id="P47" class="p p-first-last">A <strong>Life Sciences Reporting Summary</strong> for this paper is available. Summary statistics for the 9,830 variants presented in the discovery phase meta-analysis are available at <a href="http://research-pub.gene.com/chang_et_al_2017" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://research-pub.gene.com/chang_et_al_2017</a>. The full GWAS summary statistics for PDWBS will be made available through 23andMe and Genentech to qualified researchers under an agreement with 23andMe that protects the privacy of the 23andMe participants and an agreement with Genentech for data sharing. Please contact D.H. (<a href="http://dhinds23andme.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">dhinds23andme.com</a>) for more information and to apply to access the data.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>supplements</h4><div class="sup-box half_rhythm" id="d36e1374"><a href="/pmc/articles/PMC5812477/bin/NIHMS940616-supplement-supplements.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(4.6M, pdf)</sup></div></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Acknowledgments</h2><div class="sec"><p id="P53">We thank all of the subjects who donated their time and biological samples to be a part of this study. Funding details and additional acknowledgments are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note 1</a>.</p></div></div><div id="idm140628734280144" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140628734280144title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN9"></p><p id="P27" class="p p-first-last"><strong>URLs.</strong> PDGene, <a href="http://pdgene.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pdgene.org/</a>; LDScore, <a href="https://github.com/bulik/ldsc" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/bulik/ldsc</a>; INRICH, <a href="https://atgu.mgh.harvard.edu/inrich/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://atgu.mgh.harvard.edu/inrich/</a>; GWAS catalog, <a href="https://www.ebi.ac.uk/gwas/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ebi.ac.uk/gwas/</a>; GTEx portal, <a href="http://gtexportal.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://gtexportal.org/</a>; STRING, <a href="http://string-db.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://string-db.org/</a>.</p><p></p><p class="fn sec" id="FN6"></p><p id="P48" class="p p-first-last">Note: Any Supplementary Information and Source Data files are available in the <a href="http://dx.doi.org/10.1038/ng.3955" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">online version of the paper</a></p><p></p><div><p id="P49" class="p p-first"><strong>AUTHOR CONTRIBUTIONS</strong></p>D.C., M.A.N., I.B.H., the 23andMe Research Team, G.A.K., B.B., M.S., D.H., T.W.B., A.B.S., T.R.B., and R.R.G. contributed to the study design. D.C., M.A.N., I.B.H., T.R.B., and D.H. contributed to analysis and methods. D.C., M.A.N., T.W.B., A.B.S., T.R.B., and R.R.G. wrote the manuscript. D.C., M.A.N., I.B.H., J.H., M.v.d.B., F.C., the International Parkinson’s Disease Genomics Consortium (IPDGC), the 23andMe Research Team, G.A.K., G.A., B.B., M.S., D.H., T.W.B., A.B.S., T.R.B., and R.R.G. reviewed the manuscript. M.v.d.B., F.C., IPDGC and the 23andMe Research Team provided samples or data.</div><p class="fn sec" id="FN8"></p><p id="P51" class="p p-first"><strong>COMPETING FINANCIAL INTERESTS</strong></p><p id="P52" class="p p-last">The authors declare competing financial interests: details are available in the <a href="http://dx.doi.org/10.1038/ng.3955" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">online version of the paper</a>.</p><p></p></div></div><div id="idm140628734265072" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140628734265072title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Corti O, Lesage S, Brice A. What genetics tells us about the causes and mechanisms of Parkinson’s disease. <span><span class="ref-journal">Physiol. Rev. </span>2011;<span class="ref-vol">91</span>:1161–1218.</span> [<a href="/pubmed/22013209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol.+Rev&amp;title=What+genetics+tells+us+about+the+causes+and+mechanisms+of+Parkinson’s+disease&amp;author=O+Corti&amp;author=S+Lesage&amp;author=A+Brice&amp;volume=91&amp;publication_year=2011&amp;pages=1161-1218&amp;pmid=22013209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Verstraeten A, Theuns J, Van Broeckhoven C. Progress in unraveling the genetic etiology of Parkinson disease in a genomic era. <span><span class="ref-journal">Trends Genet. </span>2015;<span class="ref-vol">31</span>:140–149.</span> [<a href="/pubmed/25703649" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=Progress+in+unraveling+the+genetic+etiology+of+Parkinson+disease+in+a+genomic+era&amp;author=A+Verstraeten&amp;author=J+Theuns&amp;author=C+Van+Broeckhoven&amp;volume=31&amp;publication_year=2015&amp;pages=140-149&amp;pmid=25703649&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Nussbaum RL, Ellis CE. Alzheimer’s disease and Parkinson’s disease. <span><span class="ref-journal">N. Engl. J. Med. </span>2003;<span class="ref-vol">348</span>:1356–1364.</span> [<a href="/pubmed/12672864" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Alzheimer’s+disease+and+Parkinson’s+disease&amp;author=RL+Nussbaum&amp;author=CE+Ellis&amp;volume=348&amp;publication_year=2003&amp;pages=1356-1364&amp;pmid=12672864&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Shulman JM, De Jager PL, Feany MB. Parkinson’s disease: genetics and pathogenesis. <span><span class="ref-journal">Annu. Rev. Pathol. </span>2011;<span class="ref-vol">6</span>:193–222.</span> [<a href="/pubmed/21034221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Pathol&amp;title=Parkinson’s+disease:+genetics+and+pathogenesis&amp;author=JM+Shulman&amp;author=PL+De+Jager&amp;author=MB+Feany&amp;volume=6&amp;publication_year=2011&amp;pages=193-222&amp;pmid=21034221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Klein C, Westenberger A. Genetics of Parkinson’s disease. <span><span class="ref-journal">Cold Spring Harb. Perspect. Med. </span>2012;<span class="ref-vol">2</span>:a008888.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3253033/">PMC free article</a>]</span> [<a href="/pubmed/22315721" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cold+Spring+Harb.+Perspect.+Med&amp;title=Genetics+of+Parkinson’s+disease&amp;author=C+Klein&amp;author=A+Westenberger&amp;volume=2&amp;publication_year=2012&amp;pages=a008888&amp;pmid=22315721&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Hardy J. Genetic analysis of pathways to Parkinson disease. <span><span class="ref-journal">Neuron. </span>2010;<span class="ref-vol">68</span>:201–206.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2997424/">PMC free article</a>]</span> [<a href="/pubmed/20955928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Genetic+analysis+of+pathways+to+Parkinson+disease&amp;author=J+Hardy&amp;volume=68&amp;publication_year=2010&amp;pages=201-206&amp;pmid=20955928&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Singleton AB, Farrer MJ, Bonifati V. The genetics of Parkinson’s disease: progress and therapeutic implications. <span><span class="ref-journal">Mov. Disord. </span>2013;<span class="ref-vol">28</span>:14–23.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3578399/">PMC free article</a>]</span> [<a href="/pubmed/23389780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mov.+Disord&amp;title=The+genetics+of+Parkinson’s+disease:+progress+and+therapeutic+implications&amp;author=AB+Singleton&amp;author=MJ+Farrer&amp;author=V+Bonifati&amp;volume=28&amp;publication_year=2013&amp;pages=14-23&amp;pmid=23389780&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Nalls MA, et al.  Large-scale meta-analysis of genome-wide association data identifies six new risk loci for Parkinson’s disease. <span><span class="ref-journal">Nat. Genet. </span>2014;<span class="ref-vol">46</span>:989–993.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4146673/">PMC free article</a>]</span> [<a href="/pubmed/25064009" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Large-scale+meta-analysis+of+genome-wide+association+data+identifies+six+new+risk+loci+for+Parkinson’s+disease&amp;author=MA+Nalls&amp;volume=46&amp;publication_year=2014&amp;pages=989-993&amp;pmid=25064009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Keller MF, et al.  Using genome-wide complex trait analysis to quantify ‘missing heritability’ in Parkinson’s disease. <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">21</span>:4996–5009.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3576713/">PMC free article</a>]</span> [<a href="/pubmed/22892372" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Using+genome-wide+complex+trait+analysis+to+quantify+‘missing+heritability’+in+Parkinson’s+disease&amp;author=MF+Keller&amp;volume=21&amp;publication_year=2012&amp;pages=4996-5009&amp;pmid=22892372&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Do CB, et al.  Web-based genome-wide association study identifies two novel loci and a substantial genetic component for Parkinson’s disease. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3121750/">PMC free article</a>]</span> [<a href="/pubmed/21738487" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Web-based+genome-wide+association+study+identifies+two+novel+loci+and+a+substantial+genetic+component+for+Parkinson’s+disease&amp;author=CB+Do&amp;volume=7&amp;publication_year=2011&amp;pages=e1002141&amp;pmid=21738487&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Fuchsberger C, Abecasis GR, Hinds DA. minimac2: faster genotype imputation. <span><span class="ref-journal">Bioinformatics. </span>2015;<span class="ref-vol">31</span>:782–784.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4341061/">PMC free article</a>]</span> [<a href="/pubmed/25338720" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=minimac2:+faster+genotype+imputation&amp;author=C+Fuchsberger&amp;author=GR+Abecasis&amp;author=DA+Hinds&amp;volume=31&amp;publication_year=2015&amp;pages=782-784&amp;pmid=25338720&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Abecasis GR, et al.  An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=GR+Abecasis&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Gagliano SA, et al.  Genomics implicates adaptive and innate immunity in Alzheimer’s and Parkinson’s diseases. <span><span class="ref-journal">Ann. Clin. Transl. Neurol. </span>2016;<span class="ref-vol">3</span>:924–933.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5224821/">PMC free article</a>]</span> [<a href="/pubmed/28097204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Clin.+Transl.+Neurol&amp;title=Genomics+implicates+adaptive+and+innate+immunity+in+Alzheimer’s+and+Parkinson’s+diseases&amp;author=SA+Gagliano&amp;volume=3&amp;publication_year=2016&amp;pages=924-933&amp;pmid=28097204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Welter D, et al.  The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. <span><span class="ref-journal">Nucleic Acids Res. </span>2014;<span class="ref-vol">42</span>:D1001–D1006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3965119/">PMC free article</a>]</span> [<a href="/pubmed/24316577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+NHGRI+GWAS+Catalog,+a+curated+resource+of+SNP-trait+associations&amp;author=D+Welter&amp;volume=42&amp;publication_year=2014&amp;pages=D1001-D1006&amp;pmid=24316577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Pickrell JK, et al.  Detection and interpretation of shared genetic influences on 42 human traits. <span><span class="ref-journal">Nat. Genet. </span>2016;<span class="ref-vol">48</span>:709–717.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5207801/">PMC free article</a>]</span> [<a href="/pubmed/27182965" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Detection+and+interpretation+of+shared+genetic+influences+on+42+human+traits&amp;author=JK+Pickrell&amp;volume=48&amp;publication_year=2016&amp;pages=709-717&amp;pmid=27182965&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Nalls MA, et al.  NeuroX, a fast and efficient genotyping platform for investigation of neurodegenerative diseases. <span><span class="ref-journal">Neurobiol. Aging. </span>2015;<span class="ref-vol">36</span>:1605.e7–1605.e12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4317375/">PMC free article</a>]</span> [<a href="/pubmed/25444595" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol.+Aging&amp;title=NeuroX,+a+fast+and+efficient+genotyping+platform+for+investigation+of+neurodegenerative+diseases&amp;author=MA+Nalls&amp;volume=36&amp;publication_year=2015&amp;pages=1605.e7-1605.e12&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Nalls MA, et al.  Imputation of sequence variants for identification of genetic risks for Parkinson’s disease: a meta-analysis of genome-wide association studies. <span><span class="ref-journal">Lancet. </span>2011;<span class="ref-vol">377</span>:641–649.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696507/">PMC free article</a>]</span> [<a href="/pubmed/21292315" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Imputation+of+sequence+variants+for+identification+of+genetic+risks+for+Parkinson’s+disease:+a+meta-analysis+of+genome-wide+association+studies&amp;author=MA+Nalls&amp;volume=377&amp;publication_year=2011&amp;pages=641-649&amp;pmid=21292315&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">International Parkinson’s Disease Genomics Consortium &amp; Wellcome Trust Case Control Consortium 2. A two-stage meta-analysis identifies several new loci for Parkinson’s disease. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002142.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128098/">PMC free article</a>]</span> [<a href="/pubmed/21738488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=International+Parkinson’s+Disease+Genomics+Consortium+&amp;+Wellcome+Trust+Case+Control+Consortium+2.+A+two-stage+meta-analysis+identifies+several+new+loci+for+Parkinson’s+disease&amp;volume=7&amp;publication_year=2011&amp;pages=e1002142&amp;pmid=21738488&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Pankratz N, et al.  Meta-analysis of Parkinson’s disease: identification of a novel locus. <span><span class="ref-journal">RIT2. Ann. Neurol. </span>2012;<span class="ref-vol">71</span>:370–384.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3354734/">PMC free article</a>]</span> [<a href="/pubmed/22451204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=RIT2.+Ann.+Neurol&amp;title=Meta-analysis+of+Parkinson’s+disease:+identification+of+a+novel+locus&amp;author=N+Pankratz&amp;volume=71&amp;publication_year=2012&amp;pages=370-384&amp;pmid=22451204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Wissemann WT, et al.  Association of Parkinson disease with structural and regulatory variants in the HLA region. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2013;<span class="ref-vol">93</span>:984–993.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3824116/">PMC free article</a>]</span> [<a href="/pubmed/24183452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Association+of+Parkinson+disease+with+structural+and+regulatory+variants+in+the+HLA+region&amp;author=WT+Wissemann&amp;volume=93&amp;publication_year=2013&amp;pages=984-993&amp;pmid=24183452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Sekar A, et al.  Schizophrenia risk from complex variation of complement component 4. <span><span class="ref-journal">Nature. </span>2016;<span class="ref-vol">530</span>:177–183.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4752392/">PMC free article</a>]</span> [<a href="/pubmed/26814963" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Schizophrenia+risk+from+complex+variation+of+complement+component+4&amp;author=A+Sekar&amp;volume=530&amp;publication_year=2016&amp;pages=177-183&amp;pmid=26814963&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Kichaev G, et al.  Integrating functional data to prioritize causal variants in statistical fine-mapping studies. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004722.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4214605/">PMC free article</a>]</span> [<a href="/pubmed/25357204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Integrating+functional+data+to+prioritize+causal+variants+in+statistical+fine-mapping+studies&amp;author=G+Kichaev&amp;volume=10&amp;publication_year=2014&amp;pages=e1004722&amp;pmid=25357204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Maller JB, et al.  Bayesian refinement of association signals for 14 loci in 3 common diseases. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:1294–1301.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3791416/">PMC free article</a>]</span> [<a href="/pubmed/23104008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Bayesian+refinement+of+association+signals+for+14+loci+in+3+common+diseases&amp;author=JB+Maller&amp;volume=44&amp;publication_year=2012&amp;pages=1294-1301&amp;pmid=23104008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Chen W, et al.  Fine mapping causal variants with an approximate Bayesian method using marginal test statistics. <span><span class="ref-journal">Genetics. </span>2015;<span class="ref-vol">200</span>:719–736.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4512539/">PMC free article</a>]</span> [<a href="/pubmed/25948564" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Fine+mapping+causal+variants+with+an+approximate+Bayesian+method+using+marginal+test+statistics&amp;author=W+Chen&amp;volume=200&amp;publication_year=2015&amp;pages=719-736&amp;pmid=25948564&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Okada Y, et al.  Genetics of rheumatoid arthritis contributes to biology and drug discovery. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">506</span>:376–381.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3944098/">PMC free article</a>]</span> [<a href="/pubmed/24390342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetics+of+rheumatoid+arthritis+contributes+to+biology+and+drug+discovery&amp;author=Y+Okada&amp;volume=506&amp;publication_year=2014&amp;pages=376-381&amp;pmid=24390342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Bentham J, et al.  Genetic association analyses implicate aberrant regulation of innate and adaptive immunity genes in the pathogenesis of systemic lupus erythematosus. <span><span class="ref-journal">Nat. Genet. </span>2015;<span class="ref-vol">47</span>:1457–1464.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4668589/">PMC free article</a>]</span> [<a href="/pubmed/26502338" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genetic+association+analyses+implicate+aberrant+regulation+of+innate+and+adaptive+immunity+genes+in+the+pathogenesis+of+systemic+lupus+erythematosus&amp;author=J+Bentham&amp;volume=47&amp;publication_year=2015&amp;pages=1457-1464&amp;pmid=26502338&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">GTEx Consortium. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. <span><span class="ref-journal">Science. </span>2015;<span class="ref-vol">348</span>:648–660.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4547484/">PMC free article</a>]</span> [<a href="/pubmed/25954001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+Genotype-Tissue+Expression+(GTEx)+pilot+analysis:+multitissue+gene+regulation+in+humans&amp;volume=348&amp;publication_year=2015&amp;pages=648-660&amp;pmid=25954001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Zhang Y, et al.  An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex. <span><span class="ref-journal">J. Neurosci. </span>2014;<span class="ref-vol">34</span>:11929–11947.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4152602/">PMC free article</a>]</span> [<a href="/pubmed/25186741" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosci&amp;title=An+RNA-sequencing+transcriptome+and+splicing+database+of+glia,+neurons,+and+vascular+cells+of+the+cerebral+cortex&amp;author=Y+Zhang&amp;volume=34&amp;publication_year=2014&amp;pages=11929-11947&amp;pmid=25186741&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Giambartolomei C, et al.  Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004383.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4022491/">PMC free article</a>]</span> [<a href="/pubmed/24830394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Bayesian+test+for+colocalisation+between+pairs+of+genetic+association+studies+using+summary+statistics&amp;author=C+Giambartolomei&amp;volume=10&amp;publication_year=2014&amp;pages=e1004383&amp;pmid=24830394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Sidransky E, et al.  Multicenter analysis of glucocerebrosidase mutations in Parkinson’s disease. <span><span class="ref-journal">N. Engl. J. Med. </span>2009;<span class="ref-vol">361</span>:1651–1661.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2856322/">PMC free article</a>]</span> [<a href="/pubmed/19846850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Multicenter+analysis+of+glucocerebrosidase+mutations+in+Parkinson’s+disease&amp;author=E+Sidransky&amp;volume=361&amp;publication_year=2009&amp;pages=1651-1661&amp;pmid=19846850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Cang C, Aranda K, Seo YJ, Gasnier B, Ren D. TMEM175 is an organelle K<sup>+</sup> channel regulating lysosomal function. <span><span class="ref-journal">Cell. </span>2015;<span class="ref-vol">162</span>:1101–1112.</span> [<a href="/pubmed/26317472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=TMEM175+is+an+organelle+K++channel+regulating+lysosomal+function&amp;author=C+Cang&amp;author=K+Aranda&amp;author=YJ+Seo&amp;author=B+Gasnier&amp;author=D+Ren&amp;volume=162&amp;publication_year=2015&amp;pages=1101-1112&amp;pmid=26317472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Felbor U, et al.  Neuronal loss and brain atrophy in mice lacking cathepsins B and L. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2002;<span class="ref-vol">99</span>:7883–7888.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC122989/">PMC free article</a>]</span> [<a href="/pubmed/12048238" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Neuronal+loss+and+brain+atrophy+in+mice+lacking+cathepsins+B+and+L&amp;author=U+Felbor&amp;volume=99&amp;publication_year=2002&amp;pages=7883-7888&amp;pmid=12048238&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">McGlinchey RP, Lee JC. Cysteine cathepsins are essential in lysosomal degradation of α-synuclein. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2015;<span class="ref-vol">112</span>:9322–9327.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4522768/">PMC free article</a>]</span> [<a href="/pubmed/26170293" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Cysteine+cathepsins+are+essential+in+lysosomal+degradation+of+α-synuclein&amp;author=RP+McGlinchey&amp;author=JC+Lee&amp;volume=112&amp;publication_year=2015&amp;pages=9322-9327&amp;pmid=26170293&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Hale CM, et al.  Identification of modulators of autophagic flux in an image-based high content siRNA screen. <span><span class="ref-journal">Autophagy. </span>2016;<span class="ref-vol">12</span>:713–726.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4836002/">PMC free article</a>]</span> [<a href="/pubmed/27050463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=Identification+of+modulators+of+autophagic+flux+in+an+image-based+high+content+siRNA+screen&amp;author=CM+Hale&amp;volume=12&amp;publication_year=2016&amp;pages=713-726&amp;pmid=27050463&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Lee PH, O’Dushlaine C, Thomas B, Purcell SM. INRICH: interval-based enrichment analysis for genome-wide association studies. <span><span class="ref-journal">Bioinformatics. </span>2012;<span class="ref-vol">28</span>:1797–1799.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3381960/">PMC free article</a>]</span> [<a href="/pubmed/22513993" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=INRICH:+interval-based+enrichment+analysis+for+genome-wide+association+studies&amp;author=PH+Lee&amp;author=C+O’Dushlaine&amp;author=B+Thomas&amp;author=SM+Purcell&amp;volume=28&amp;publication_year=2012&amp;pages=1797-1799&amp;pmid=22513993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Arranz AM, et al.  LRRK2 functions in synaptic vesicle endocytosis through a kinase-dependent mechanism. <span><span class="ref-journal">J. Cell Sci. </span>2015;<span class="ref-vol">128</span>:541–552.</span> [<a href="/pubmed/25501810" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Sci&amp;title=LRRK2+functions+in+synaptic+vesicle+endocytosis+through+a+kinase-dependent+mechanism&amp;author=AM+Arranz&amp;volume=128&amp;publication_year=2015&amp;pages=541-552&amp;pmid=25501810&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Shin D, Shin JY, McManus MT, Ptácek LJ, Fu YH. Dicer ablation in oligodendrocytes provokes neuronal impairment in mice. <span><span class="ref-journal">Ann. Neurol. </span>2009;<span class="ref-vol">66</span>:843–857.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2885004/">PMC free article</a>]</span> [<a href="/pubmed/20035504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Neurol&amp;title=Dicer+ablation+in+oligodendrocytes+provokes+neuronal+impairment+in+mice&amp;author=D+Shin&amp;author=JY+Shin&amp;author=MT+McManus&amp;author=LJ+Ptácek&amp;author=YH+Fu&amp;volume=66&amp;publication_year=2009&amp;pages=843-857&amp;pmid=20035504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Tan NN, et al.  Epigenetic downregulation of <em>Scn3a</em> expression by valproate: a possible role in its anticonvulsant activity. <span><span class="ref-journal">Mol. Neurobiol. </span>2016;<span class="ref-vol">54</span>:2831–2842.</span> [<a href="/pubmed/27013471" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Neurobiol&amp;title=Epigenetic+downregulation+of+Scn3a+expression+by+valproate:+a+possible+role+in+its+anticonvulsant+activity&amp;author=NN+Tan&amp;volume=54&amp;publication_year=2016&amp;pages=2831-2842&amp;pmid=27013471&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Dittmer S, et al.  TOX3 is a neuronal survival factor that induces transcription depending on the presence of CITED1 or phosphorylated CREB in the transcriptionally active complex. <span><span class="ref-journal">J. Cell Sci. </span>2011;<span class="ref-vol">124</span>:252–260.</span> [<a href="/pubmed/21172805" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Sci&amp;title=TOX3+is+a+neuronal+survival+factor+that+induces+transcription+depending+on+the+presence+of+CITED1+or+phosphorylated+CREB+in+the+transcriptionally+active+complex&amp;author=S+Dittmer&amp;volume=124&amp;publication_year=2011&amp;pages=252-260&amp;pmid=21172805&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Kondo M, et al.  SATB1 plays a critical role in establishment of immune tolerance. <span><span class="ref-journal">J. Immunol. </span>2016;<span class="ref-vol">196</span>:563–572.</span> [<a href="/pubmed/26667169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol&amp;title=SATB1+plays+a+critical+role+in+establishment+of+immune+tolerance&amp;author=M+Kondo&amp;volume=196&amp;publication_year=2016&amp;pages=563-572&amp;pmid=26667169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Hopkins AL, Groom CR. The druggable genome. <span><span class="ref-journal">Nat. Rev. Drug Discov. </span>2002;<span class="ref-vol">1</span>:727–730.</span> [<a href="/pubmed/12209152" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Drug+Discov&amp;title=The+druggable+genome&amp;author=AL+Hopkins&amp;author=CR+Groom&amp;volume=1&amp;publication_year=2002&amp;pages=727-730&amp;pmid=12209152&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Louis AG, Yel L, Cao JN, Agrawal S, Gupta S. Common variable immunodeficiency associated with microdeletion of chromosome 1q42.1–q42.3 and inositol 1,4,5-trisphosphate kinase B (ITPKB) deficiency. <span><span class="ref-journal">Clin. Transl. Immunology. </span>2016;<span class="ref-vol">5</span>:e59.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4735063/">PMC free article</a>]</span> [<a href="/pubmed/26900472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Transl.+Immunology&amp;title=Common+variable+immunodeficiency+associated+with+microdeletion+of+chromosome+1q42.1–q42.3+and+inositol+1,4,5-trisphosphate+kinase+B+(ITPKB)+deficiency&amp;author=AG+Louis&amp;author=L+Yel&amp;author=JN+Cao&amp;author=S+Agrawal&amp;author=S+Gupta&amp;volume=5&amp;publication_year=2016&amp;pages=e59&amp;pmid=26900472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Wang Y, et al.  Whole-genome association study identifies <em>STK39</em> as a hypertension susceptibility gene. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2009;<span class="ref-vol">106</span>:226–231.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2629209/">PMC free article</a>]</span> [<a href="/pubmed/19114657" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Whole-genome+association+study+identifies+STK39+as+a+hypertension+susceptibility+gene&amp;author=Y+Wang&amp;volume=106&amp;publication_year=2009&amp;pages=226-231&amp;pmid=19114657&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Durand EY, Do CB, Mountain JL, Macpherson JM. Ancestry composition: a novel, efficient pipeline for ancestry deconvolution. <span><span class="ref-journal">bioRxiv. </span>2014</span> Preprint at  <a href="http://www.biorxiv.org/content/early/2014/10/18/010512" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.biorxiv.org/content/early/2014/10/18/010512</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Henn BM, et al.  Cryptic distant relatives are common in both isolated and cosmopolitan genetic samples. <span><span class="ref-journal">PLoS One. </span>2012;<span class="ref-vol">7</span>:e34267.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3317976/">PMC free article</a>]</span> [<a href="/pubmed/22509285" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Cryptic+distant+relatives+are+common+in+both+isolated+and+cosmopolitan+genetic+samples&amp;author=BM+Henn&amp;volume=7&amp;publication_year=2012&amp;pages=e34267&amp;pmid=22509285&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through prephasing. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+prephasing&amp;author=B+Howie&amp;author=C+Fuchsberger&amp;author=M+Stephens&amp;author=J+Marchini&amp;author=GR+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">de Bakker PI, et al.  Practical aspects of imputation-driven meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:R122–R128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782358/">PMC free article</a>]</span> [<a href="/pubmed/18852200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Practical+aspects+of+imputation-driven+meta-analysis+of+genome-wide+association+studies&amp;author=PI+de+Bakker&amp;volume=17&amp;publication_year=2008&amp;pages=R122-R128&amp;pmid=18852200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Bulik-Sullivan BK, et al.  LD score regression distinguishes confounding from polygenicity in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2015;<span class="ref-vol">47</span>:291–295.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4495769/">PMC free article</a>]</span> [<a href="/pubmed/25642630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=LD+score+regression+distinguishes+confounding+from+polygenicity+in+genome-wide+association+studies&amp;author=BK+Bulik-Sullivan&amp;volume=47&amp;publication_year=2015&amp;pages=291-295&amp;pmid=25642630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Finucane HK, et al.  Partitioning heritability by functional annotation using genome-wide association summary statistics. <span><span class="ref-journal">Nat. Genet. </span>2015;<span class="ref-vol">47</span>:1228–1235.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4626285/">PMC free article</a>]</span> [<a href="/pubmed/26414678" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Partitioning+heritability+by+functional+annotation+using+genome-wide+association+summary+statistics&amp;author=HK+Finucane&amp;volume=47&amp;publication_year=2015&amp;pages=1228-1235&amp;pmid=26414678&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Lee SH, Yang J, Goddard ME, Visscher PM, Wray NR. Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood. <span><span class="ref-journal">Bioinformatics. </span>2012;<span class="ref-vol">28</span>:2540–2542.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3463125/">PMC free article</a>]</span> [<a href="/pubmed/22843982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Estimation+of+pleiotropy+between+complex+diseases+using+single-nucleotide+polymorphism-derived+genomic+relationships+and+restricted+maximum+likelihood&amp;author=SH+Lee&amp;author=J+Yang&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;author=NR+Wray&amp;volume=28&amp;publication_year=2012&amp;pages=2540-2542&amp;pmid=22843982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Cingolani P, et al.  A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso-2; iso-3. <span><span class="ref-journal">Fly (Austin) </span>2012;<span class="ref-vol">6</span>:80–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3679285/">PMC free article</a>]</span> [<a href="/pubmed/22728672" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fly+(Austin)&amp;title=A+program+for+annotating+and+predicting+the+effects+of+single+nucleotide+polymorphisms,+SnpEff:+SNPs+in+the+genome+of+Drosophila+melanogaster+strain+w1118;+iso-2;+iso-3&amp;author=P+Cingolani&amp;volume=6&amp;publication_year=2012&amp;pages=80-92&amp;pmid=22728672&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">dos Santos G, et al.  FlyBase: introduction of the <em>Drosophila melanogaster</em> Release 6 reference genome assembly and large-scale migration of genome annotations. <span><span class="ref-journal">Nucleic Acids Res. </span>2015;<span class="ref-vol">43</span>:D690–D697.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4383921/">PMC free article</a>]</span> [<a href="/pubmed/25398896" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=FlyBase:+introduction+of+the+Drosophila+melanogaster+Release+6+reference+genome+assembly+and+large-scale+migration+of+genome+annotations&amp;author=G+dos+Santos&amp;volume=43&amp;publication_year=2015&amp;pages=D690-D697&amp;pmid=25398896&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Eppig JT, Blake JA, Bult CJ, Kadin JA, Richardson JE. The Mouse Genome Database (MGD): facilitating mouse as a model for human biology and disease. <span><span class="ref-journal">Nucleic Acids Res. </span>2015;<span class="ref-vol">43</span>:D726–D736.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4384027/">PMC free article</a>]</span> [<a href="/pubmed/25348401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+Mouse+Genome+Database+(MGD):+facilitating+mouse+as+a+model+for+human+biology+and+disease&amp;author=JT+Eppig&amp;author=JA+Blake&amp;author=CJ+Bult&amp;author=JA+Kadin&amp;author=JE+Richardson&amp;volume=43&amp;publication_year=2015&amp;pages=D726-D736&amp;pmid=25348401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">McKusick VA. MENDELIAN Inheritance in Man and its online version, OMIM. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:588–604.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1852721/">PMC free article</a>]</span> [<a href="/pubmed/17357067" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=MENDELIAN+Inheritance+in+Man+and+its+online+version,+OMIM&amp;author=VA+McKusick&amp;volume=80&amp;publication_year=2007&amp;pages=588-604&amp;pmid=17357067&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Szklarczyk D, et al.  The STRING database in 2017: quality-controlled protein-protein association networks, made broadly accessible. <span><span class="ref-journal">Nucleic Acids Res. </span>2017;<span class="ref-vol">45</span>:D362–D368.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5210637/">PMC free article</a>]</span> [<a href="/pubmed/27924014" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+STRING+database+in+2017:+quality-controlled+protein-protein+association+networks,+made+broadly+accessible&amp;author=D+Szklarczyk&amp;volume=45&amp;publication_year=2017&amp;pages=D362-D368&amp;pmid=27924014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Smith AC, Robinson AJ. MitoMiner v3.1, an update on the mitochondrial proteomics database. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>:D1258–D1261.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4702766/">PMC free article</a>]</span> [<a href="/pubmed/26432830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=MitoMiner+v3.1,+an+update+on+the+mitochondrial+proteomics+database&amp;author=AC+Smith&amp;author=AJ+Robinson&amp;volume=44&amp;publication_year=2016&amp;pages=D1258-D1261&amp;pmid=26432830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Calvo SE, Clauser KR, Mootha VK. MitoCarta2.0: an updated inventory of mammalian mitochondrial proteins. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>:D1251–D1257.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4702768/">PMC free article</a>]</span> [<a href="/pubmed/26450961" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=MitoCarta2.0:+an+updated+inventory+of+mammalian+mitochondrial+proteins&amp;author=SE+Calvo&amp;author=KR+Clauser&amp;author=VK+Mootha&amp;volume=44&amp;publication_year=2016&amp;pages=D1251-D1257&amp;pmid=26450961&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="element-citation">Pagliarini DJ, et al.  A mitochondrial protein compendium elucidates complex I disease biology. <span><span class="ref-journal">Cell. </span>2008;<span class="ref-vol">134</span>:112–123.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2778844/">PMC free article</a>]</span> [<a href="/pubmed/18614015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+mitochondrial+protein+compendium+elucidates+complex+I+disease+biology&amp;author=DJ+Pagliarini&amp;volume=134&amp;publication_year=2008&amp;pages=112-123&amp;pmid=18614015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="element-citation">Brozzi A, Urbanelli L, Germain PL, Magini A, Emiliani C. hLGDB: a database of human lysosomal genes and their regulation. <span><span class="ref-journal">Database (Oxford) </span>2013;<span class="ref-vol">2013</span>:bat024.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3625959/">PMC free article</a>]</span> [<a href="/pubmed/23584836" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Database+(Oxford)&amp;title=hLGDB:+a+database+of+human+lysosomal+genes+and+their+regulation&amp;author=A+Brozzi&amp;author=L+Urbanelli&amp;author=PL+Germain&amp;author=A+Magini&amp;author=C+Emiliani&amp;volume=2013&amp;publication_year=2013&amp;pages=bat024&amp;pmid=23584836&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="element-citation">Moussay E, et al.  The acquisition of resistance to TNFα in breast cancer cells is associated with constitutive activation of autophagy as revealed by a transcriptome analysis using a custom microarray. <span><span class="ref-journal">Autophagy. </span>2011;<span class="ref-vol">7</span>:760–770.</span> [<a href="/pubmed/21490427" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=The+acquisition+of+resistance+to+TNFα+in+breast+cancer+cells+is+associated+with+constitutive+activation+of+autophagy+as+revealed+by+a+transcriptome+analysis+using+a+custom+microarray&amp;author=E+Moussay&amp;volume=7&amp;publication_year=2011&amp;pages=760-770&amp;pmid=21490427&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="element-citation">Kanehisa M, Goto S. KEGG: Kyoto Encyclopedia of Genes and Genomes. <span><span class="ref-journal">Nucleic Acids Res. </span>2000;<span class="ref-vol">28</span>:27–30.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC102409/">PMC free article</a>]</span> [<a href="/pubmed/10592173" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=KEGG:+Kyoto+Encyclopedia+of+Genes+and+Genomes&amp;author=M+Kanehisa&amp;author=S+Goto&amp;volume=28&amp;publication_year=2000&amp;pages=27-30&amp;pmid=10592173&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="element-citation">Gene Ontology Consortium. Gene Ontology Consortium: going forward. <span><span class="ref-journal">Nucleic Acids Res. </span>2015;<span class="ref-vol">43</span>:D1049–D1056.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4383973/">PMC free article</a>]</span> [<a href="/pubmed/25428369" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Gene+Ontology+Consortium:+going+forward&amp;volume=43&amp;publication_year=2015&amp;pages=D1049-D1056&amp;pmid=25428369&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="element-citation">Chang CC, et al.  Second-generation PLINK: rising to the challenge of larger and richer datasets. <span><span class="ref-journal">Gigascience. </span>2015;<span class="ref-vol">4</span>:7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4342193/">PMC free article</a>]</span> [<a href="/pubmed/25722852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Second-generation+PLINK:+rising+to+the+challenge+of+larger+and+richer+datasets&amp;author=CC+Chang&amp;volume=4&amp;publication_year=2015&amp;pages=7&amp;pmid=25722852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5812477&amp;issue-id=306613&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140628774051152title">Abstract</a></li><li><a href="#S2title">ONLINE METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S13title">Acknowledgments</a></li><li><a href="#idm140628734280144title">Footnotes</a></li><li><a href="#idm140628734265072title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5812477/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5812477/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5812477/pdf/nihms940616.pdf">PDF (1.4M)</a></li> | <li><a href="#" data-citationid="PMC5812477" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5812477%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5812477%2F&amp;text=A%20meta-analysis%20of%20genome-wide%20association%20studies%20identifies%2017%20new%20Parkinson%E2%80%99s%20disease%20risk%20loci" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5812477%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28892059" data-aiid="5812477" data-aid="5812477" data-iid="306613" data-domainid="319" data-domain="nihpa" data-accid="PMC5812477" data-md5="b4225a4c3427b700c4e70b71f91f9b8d"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:31:36-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A170FDD5A322100000000086504B9&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5812477%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5812477/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>