<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5291202/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Fine-mapping, novel loci identification, and SNP association transferability in a genome-wide association study of QRS duration in African Americans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5291202/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Fine-mapping, novel loci identification, and SNP association transferability in a genome-wide association study of QRS duration in African Americans"><meta name="citation_authors" content="Daniel S. Evans, Christy L. Avery, Mike A. Nalls, Guo Li, John Barnard, Erin N. Smith, Toshiko Tanaka, Anne M. Butler, Sarah G. Buxbaum, Alvaro Alonso, Dan E. Arking, Gerald S. Berenson, Joshua C. Bis, Steven Buyske, Cara L. Carty, Wei Chen, Mina K. Chung, Steven R. Cummings, Rajat Deo, Charles B. Eaton, Ervin R. Fox, Susan R. Heckbert, Gerardo Heiss, Lucia A. Hindorff, Wen-Chi Hsueh, Aaron Isaacs, Yalda Jamshidi, Kathleen F. Kerr, Felix Liu, Yongmei Liu, Kurt K. Lohman, Jared W. Magnani, Joseph F. Maher, Reena Mehra, Yan A. Meng, Solomon K. Musani, Christopher Newton-Cheh, Kari E. North, Bruce M. Psaty, Susan Redline, Jerome I. Rotter, Renate B. Schnabel, Nicholas J. Schork, Ralph V. Shohet, Andrew B. Singleton, Jonathan D. Smith, Elsayed Z. Soliman, Sathanur R. Srinivasan, Herman A. Taylor, Jr., David R. Van Wagoner, James G. Wilson, Taylor Young, Zhu-Ming Zhang, Alan B. Zonderman, Michele K. Evans, Luigi Ferrucci, Sarah S. Murray, Gregory J. Tranah, Eric A. Whitsel, Alex P. Reiner, CHARGE QRS Consortium, Nona Sotoodehnia"><meta name="citation_date" content="1 October 2016"><meta name="citation_issue" content="19"><meta name="citation_volume" content="25"><meta name="citation_firstpage" content="4350"><meta name="citation_doi" content="10.1093/hmg/ddw284"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5291202/?report=abstract"><meta name="citation_pmid" content="27577874"><meta name="DC.Title" content="Fine-mapping, novel loci identification, and SNP association transferability in a genome-wide association study of QRS duration in African Americans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Daniel S. Evans"><meta name="DC.Contributor" content="Christy L. Avery"><meta name="DC.Contributor" content="Mike A. Nalls"><meta name="DC.Contributor" content="Guo Li"><meta name="DC.Contributor" content="John Barnard"><meta name="DC.Contributor" content="Erin N. Smith"><meta name="DC.Contributor" content="Toshiko Tanaka"><meta name="DC.Contributor" content="Anne M. Butler"><meta name="DC.Contributor" content="Sarah G. Buxbaum"><meta name="DC.Contributor" content="Alvaro Alonso"><meta name="DC.Contributor" content="Dan E. Arking"><meta name="DC.Contributor" content="Gerald S. Berenson"><meta name="DC.Contributor" content="Joshua C. Bis"><meta name="DC.Contributor" content="Steven Buyske"><meta name="DC.Contributor" content="Cara L. Carty"><meta name="DC.Contributor" content="Wei Chen"><meta name="DC.Contributor" content="Mina K. Chung"><meta name="DC.Contributor" content="Steven R. Cummings"><meta name="DC.Contributor" content="Rajat Deo"><meta name="DC.Contributor" content="Charles B. Eaton"><meta name="DC.Contributor" content="Ervin R. Fox"><meta name="DC.Contributor" content="Susan R. Heckbert"><meta name="DC.Contributor" content="Gerardo Heiss"><meta name="DC.Contributor" content="Lucia A. Hindorff"><meta name="DC.Contributor" content="Wen-Chi Hsueh"><meta name="DC.Contributor" content="Aaron Isaacs"><meta name="DC.Contributor" content="Yalda Jamshidi"><meta name="DC.Contributor" content="Kathleen F. Kerr"><meta name="DC.Contributor" content="Felix Liu"><meta name="DC.Contributor" content="Yongmei Liu"><meta name="DC.Contributor" content="Kurt K. Lohman"><meta name="DC.Contributor" content="Jared W. Magnani"><meta name="DC.Contributor" content="Joseph F. Maher"><meta name="DC.Contributor" content="Reena Mehra"><meta name="DC.Contributor" content="Yan A. Meng"><meta name="DC.Contributor" content="Solomon K. Musani"><meta name="DC.Contributor" content="Christopher Newton-Cheh"><meta name="DC.Contributor" content="Kari E. North"><meta name="DC.Contributor" content="Bruce M. Psaty"><meta name="DC.Contributor" content="Susan Redline"><meta name="DC.Contributor" content="Jerome I. Rotter"><meta name="DC.Contributor" content="Renate B. Schnabel"><meta name="DC.Contributor" content="Nicholas J. Schork"><meta name="DC.Contributor" content="Ralph V. Shohet"><meta name="DC.Contributor" content="Andrew B. Singleton"><meta name="DC.Contributor" content="Jonathan D. Smith"><meta name="DC.Contributor" content="Elsayed Z. Soliman"><meta name="DC.Contributor" content="Sathanur R. Srinivasan"><meta name="DC.Contributor" content="Herman A. Taylor, Jr."><meta name="DC.Contributor" content="David R. Van Wagoner"><meta name="DC.Contributor" content="James G. Wilson"><meta name="DC.Contributor" content="Taylor Young"><meta name="DC.Contributor" content="Zhu-Ming Zhang"><meta name="DC.Contributor" content="Alan B. Zonderman"><meta name="DC.Contributor" content="Michele K. Evans"><meta name="DC.Contributor" content="Luigi Ferrucci"><meta name="DC.Contributor" content="Sarah S. Murray"><meta name="DC.Contributor" content="Gregory J. Tranah"><meta name="DC.Contributor" content="Eric A. Whitsel"><meta name="DC.Contributor" content="Alex P. Reiner"><meta name="DC.Contributor" content="CHARGE QRS Consortium"><meta name="DC.Contributor" content="Nona Sotoodehnia"><meta name="DC.Date" content="2016 Oct 1"><meta name="DC.Identifier" content="10.1093/hmg/ddw284"><meta name="DC.Language" content="en"><meta property="og:title" content="Fine-mapping, novel loci identification, and SNP association transferability in a genome-wide association study of QRS duration in African Americans"><meta property="og:type" content="article"><meta property="og:description" content="The electrocardiographic QRS duration, a measure of ventricular depolarization and conduction, is associated with cardiovascular mortality. While single nucleotide polymorphisms (SNPs) associated with QRS duration have been identified at 22 loci in populations ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5291202/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5291202/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5291202/pdf/ddw284.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CB404DD594991000000000471029B.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5291202%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC5291202</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2016 Oct 1; 25(19): 4350–4368. </span></div><div><span class="fm-vol-iss-date">Published online 2016 Aug 29. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddw284" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddw284</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5291202</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27577874">27577874</a></div></div></div></div><h1 class="content-title">Fine-mapping, novel loci identification, and SNP association transferability in a genome-wide association study of QRS duration in African Americans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Evans%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953987867616" co-class="co-affbox">Daniel S. Evans</a>,<sup>1,</sup><sup>†</sup><sup>*</sup> <a href="/pubmed/?term=Avery%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953987851616" co-class="co-affbox">Christy L. Avery</a>,<sup>2,</sup><sup>†</sup> <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953982197456" co-class="co-affbox">Mike A. Nalls</a>,<sup>3,</sup><sup>†</sup> <a href="/pubmed/?term=Li%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953982194032" co-class="co-affbox">Guo Li</a>,<sup>4</sup> <a href="/pubmed/?term=Barnard%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953982191664" co-class="co-affbox">John Barnard</a>,<sup>5</sup> <a href="/pubmed/?term=Smith%20EN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953982189264" co-class="co-affbox">Erin N. Smith</a>,<sup>6</sup> <a href="/pubmed/?term=Tanaka%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953982186864" co-class="co-affbox">Toshiko Tanaka</a>,<sup>7</sup> <a href="/pubmed/?term=Butler%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953982184464" co-class="co-affbox">Anne M. Butler</a>,<sup>2</sup> <a href="/pubmed/?term=Buxbaum%20SG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953982182128" co-class="co-affbox">Sarah G. Buxbaum</a>,<sup>8,</sup><sup>9</sup> <a href="/pubmed/?term=Alonso%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953982178704" co-class="co-affbox">Alvaro Alonso</a>,<sup>10</sup> <a href="/pubmed/?term=Arking%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986572464" co-class="co-affbox">Dan E. Arking</a>,<sup>11</sup> <a href="/pubmed/?term=Berenson%20GS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986570064" co-class="co-affbox">Gerald S. Berenson</a>,<sup>12</sup> <a href="/pubmed/?term=Bis%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986567664" co-class="co-affbox">Joshua C. Bis</a>,<sup>4</sup> <a href="/pubmed/?term=Buyske%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986565360" co-class="co-affbox">Steven Buyske</a>,<sup>13</sup> <a href="/pubmed/?term=Carty%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986562960" co-class="co-affbox">Cara L. Carty</a>,<sup>14</sup> <a href="/pubmed/?term=Chen%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986560560" co-class="co-affbox">Wei Chen</a>,<sup>15</sup> <a href="/pubmed/?term=Chung%20MK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986558192" co-class="co-affbox">Mina K. Chung</a>,<sup>16,</sup><sup>17,</sup><sup>18</sup> <a href="/pubmed/?term=Cummings%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986553744" co-class="co-affbox">Steven R. Cummings</a>,<sup>1</sup> <a href="/pubmed/?term=Deo%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985513760" co-class="co-affbox">Rajat Deo</a>,<sup>19</sup> <a href="/pubmed/?term=Eaton%20CB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985511392" co-class="co-affbox">Charles B. Eaton</a>,<sup>20</sup> <a href="/pubmed/?term=Fox%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985508992" co-class="co-affbox">Ervin R. Fox</a>,<sup>21</sup> <a href="/pubmed/?term=Heckbert%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985506624" co-class="co-affbox">Susan R. Heckbert</a>,<sup>4,</sup><sup>22,</sup><sup>23</sup> <a href="/pubmed/?term=Heiss%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985502240" co-class="co-affbox">Gerardo Heiss</a>,<sup>2</sup> <a href="/pubmed/?term=Hindorff%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985499904" co-class="co-affbox">Lucia A. Hindorff</a>,<sup>24</sup> <a href="/pubmed/?term=Hsueh%20WC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985497504" co-class="co-affbox">Wen-Chi Hsueh</a>,<sup>25</sup> <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985495104" co-class="co-affbox">Aaron Isaacs</a>,<sup>26,</sup><sup>27</sup> <a href="/pubmed/?term=Jamshidi%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985861744" co-class="co-affbox">Yalda Jamshidi</a>,<sup>28</sup> <a href="/pubmed/?term=Kerr%20KF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985859344" co-class="co-affbox">Kathleen F. Kerr</a>,<sup>29</sup> <a href="/pubmed/?term=Liu%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985856944" co-class="co-affbox">Felix Liu</a>,<sup>30</sup> <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985854576" co-class="co-affbox">Yongmei Liu</a>,<sup>31</sup> <a href="/pubmed/?term=Lohman%20KK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985852208" co-class="co-affbox">Kurt K. Lohman</a>,<sup>31</sup> <a href="/pubmed/?term=Magnani%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985849872" co-class="co-affbox">Jared W. Magnani</a>,<sup>32</sup> <a href="/pubmed/?term=Maher%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985847472" co-class="co-affbox">Joseph F. Maher</a>,<sup>33</sup> <a href="/pubmed/?term=Mehra%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985845072" co-class="co-affbox">Reena Mehra</a>,<sup>16,17,18</sup> <a href="/pubmed/?term=Meng%20YA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985842672" co-class="co-affbox">Yan A. Meng</a>,<sup>34</sup> <a href="/pubmed/?term=Musani%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953985840240" co-class="co-affbox">Solomon K. Musani</a>,<sup>35</sup> <a href="/pubmed/?term=Newton-Cheh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953988153120" co-class="co-affbox">Christopher Newton-Cheh</a>,<sup>34,</sup><sup>36</sup> <a href="/pubmed/?term=North%20KE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953988149760" co-class="co-affbox">Kari E. North</a>,<sup>2,</sup><sup>37</sup> <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953988146400" co-class="co-affbox">Bruce M. Psaty</a>,<sup>4,</sup><sup>22,</sup><sup>23,</sup><sup>38</sup> <a href="/pubmed/?term=Redline%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953988141120" co-class="co-affbox">Susan Redline</a>,<sup>39</sup> <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953988138720" co-class="co-affbox">Jerome I. Rotter</a>,<sup>40</sup> <a href="/pubmed/?term=Schnabel%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953988136320" co-class="co-affbox">Renate B. Schnabel</a>,<sup>41</sup> <a href="/pubmed/?term=Schork%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953988133920" co-class="co-affbox">Nicholas J. Schork</a>,<sup>42</sup> <a href="/pubmed/?term=Shohet%20RV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986086176" co-class="co-affbox">Ralph V. Shohet</a>,<sup>43</sup> <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986083776" co-class="co-affbox">Andrew B. Singleton</a>,<sup>3</sup> <a href="/pubmed/?term=Smith%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986081440" co-class="co-affbox">Jonathan D. Smith</a>,<sup>44</sup> <a href="/pubmed/?term=Soliman%20EZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986079040" co-class="co-affbox">Elsayed Z. Soliman</a>,<sup>45</sup> <a href="/pubmed/?term=Srinivasan%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986076640" co-class="co-affbox">Sathanur R. Srinivasan</a>,<sup>15</sup> <a href="/pubmed/?term=Taylor%20HA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986074304" co-class="co-affbox">Herman A. Taylor, Jr.</a>,<sup>33</sup> <a href="/pubmed/?term=Van%20Wagoner%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986071712" co-class="co-affbox">David R. Van Wagoner</a>,<sup>17</sup> <a href="/pubmed/?term=Wilson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986069376" co-class="co-affbox">James G. Wilson</a>,<sup>46</sup> <a href="/pubmed/?term=Young%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953986066976" co-class="co-affbox">Taylor Young</a>,<sup>34</sup> <a href="/pubmed/?term=Zhang%20ZM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139954053304256" co-class="co-affbox">Zhu-Ming Zhang</a>,<sup>45</sup> <a href="/pubmed/?term=Zonderman%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139954053301920" co-class="co-affbox">Alan B. Zonderman</a>,<sup>47</sup> <a href="/pubmed/?term=Evans%20MK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139954053299520" co-class="co-affbox">Michele K. Evans</a>,<sup>47</sup> <a href="/pubmed/?term=Ferrucci%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139954053297184" co-class="co-affbox">Luigi Ferrucci</a>,<sup>7</sup> <a href="/pubmed/?term=Murray%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139954053294848" co-class="co-affbox">Sarah S. Murray</a>,<sup>48</sup> <a href="/pubmed/?term=Tranah%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139954053292448" co-class="co-affbox">Gregory J. Tranah</a>,<sup>1</sup> <a href="/pubmed/?term=Whitsel%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139954053290112" co-class="co-affbox">Eric A. Whitsel</a>,<sup>2,</sup><sup>49</sup> <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139954053286752" co-class="co-affbox">Alex P. Reiner</a>,<sup>14,</sup><sup>22</sup> CHARGE QRS Consortium,<sup>50</sup> and  <a href="/pubmed/?term=Sotoodehnia%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874" class="affpopup" co-rid="_co_idm139953987130368" co-class="co-affbox">Nona Sotoodehnia</a><sup>4,</sup><sup>51,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139953987867616"><h3 class="no_margin">Daniel S. Evans</h3><p><sup>1</sup>California Pacific Medical Center Research Institute, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Evans%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Daniel S. Evans</a></div></div><div id="_co_idm139953987851616"><h3 class="no_margin">Christy L. Avery</h3><p><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina, Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Avery%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Christy L. Avery</a></div></div><div id="_co_idm139953982197456"><h3 class="no_margin">Mike A. Nalls</h3><p><sup>3</sup>Laboratory of Neurogenetics, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Mike A. Nalls</a></div></div><div id="_co_idm139953982194032"><h3 class="no_margin">Guo Li</h3><p><sup>4</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Guo Li</a></div></div><div id="_co_idm139953982191664"><h3 class="no_margin">John Barnard</h3><p><sup>5</sup>Department of Quantitative Health Sciences, Lerner Research Institute, Cleveland Clinic, Cleveland, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Barnard%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">John Barnard</a></div></div><div id="_co_idm139953982189264"><h3 class="no_margin">Erin N. Smith</h3><p><sup>6</sup>Department of Pediatrics and Rady Children's Hospital, University of California at San Diego, School of Medicine, La Jolla, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Smith%20EN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Erin N. Smith</a></div></div><div id="_co_idm139953982186864"><h3 class="no_margin">Toshiko Tanaka</h3><p><sup>7</sup>Translational Gerontology Branch, National Institute on Aging, National Institutes of Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Tanaka%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Toshiko Tanaka</a></div></div><div id="_co_idm139953982184464"><h3 class="no_margin">Anne M. Butler</h3><p><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina, Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Butler%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Anne M. Butler</a></div></div><div id="_co_idm139953982182128"><h3 class="no_margin">Sarah G. Buxbaum</h3><p><sup>8</sup>Center of Excellence in Minority Health and Health Disparities, Jackson State University, Jackson, MS, USA</p><p><sup>9</sup>Department of Epidemiology and Biostatistics, Jackson State University School of Public Health (Initiative), Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Buxbaum%20SG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Sarah G. Buxbaum</a></div></div><div id="_co_idm139953982178704"><h3 class="no_margin">Alvaro Alonso</h3><p><sup>10</sup>Department of Epidemiology, Rollins School of Public Health, Emory University, Atlanta, GA, USA</p><div>Find articles by <a href="/pubmed/?term=Alonso%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Alvaro Alonso</a></div></div><div id="_co_idm139953986572464"><h3 class="no_margin">Dan E. Arking</h3><p><sup>11</sup>McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Arking%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Dan E. Arking</a></div></div><div id="_co_idm139953986570064"><h3 class="no_margin">Gerald S. Berenson</h3><p><sup>12</sup>Department of Medicine and Cardiology, Tulane University, New Orleans, LA, USA</p><div>Find articles by <a href="/pubmed/?term=Berenson%20GS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Gerald S. Berenson</a></div></div><div id="_co_idm139953986567664"><h3 class="no_margin">Joshua C. Bis</h3><p><sup>4</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Bis%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Joshua C. Bis</a></div></div><div id="_co_idm139953986565360"><h3 class="no_margin">Steven Buyske</h3><p><sup>13</sup>Department of Statistics and Biostatistics and Department of Genetics, Rutgers University, Piscataway, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Buyske%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Steven Buyske</a></div></div><div id="_co_idm139953986562960"><h3 class="no_margin">Cara L. Carty</h3><p><sup>14</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Carty%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Cara L. Carty</a></div></div><div id="_co_idm139953986560560"><h3 class="no_margin">Wei Chen</h3><p><sup>15</sup>Department of Epidemiology, Tulane University, New Orleans, LA, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Wei Chen</a></div></div><div id="_co_idm139953986558192"><h3 class="no_margin">Mina K. Chung</h3><p><sup>16</sup>Department of Cardiovascular Medicine, Heart and Vascular Institute, Cleveland Clinic, Cleveland, OH, USA</p><p><sup>17</sup>Department of Molecular Cardiology, Lerner Research Institute, Cleveland Clinic, Cleveland, OH, USA</p><p><sup>18</sup>Cleveland Clinic Lerner College of Medicine of Case Western Reserve University, Cleveland, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Chung%20MK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Mina K. Chung</a></div></div><div id="_co_idm139953986553744"><h3 class="no_margin">Steven R. Cummings</h3><p><sup>1</sup>California Pacific Medical Center Research Institute, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Cummings%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Steven R. Cummings</a></div></div><div id="_co_idm139953985513760"><h3 class="no_margin">Rajat Deo</h3><p><sup>19</sup>Division of Cardiovascular Medicine, University of Pennsylvania, Philadelphia, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Deo%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Rajat Deo</a></div></div><div id="_co_idm139953985511392"><h3 class="no_margin">Charles B. Eaton</h3><p><sup>20</sup>Departments of Family Medicine and Epidemiology, Alpert Medical School, Brown University, Providence, RI, USA</p><div>Find articles by <a href="/pubmed/?term=Eaton%20CB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Charles B. Eaton</a></div></div><div id="_co_idm139953985508992"><h3 class="no_margin">Ervin R. Fox</h3><p><sup>21</sup>Department of Medicine, Division of Cardiovascular Disease, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Fox%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Ervin R. Fox</a></div></div><div id="_co_idm139953985506624"><h3 class="no_margin">Susan R. Heckbert</h3><p><sup>4</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</p><p><sup>22</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</p><p><sup>23</sup>Group Health Research Institute, Group Health Cooperative, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Heckbert%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Susan R. Heckbert</a></div></div><div id="_co_idm139953985502240"><h3 class="no_margin">Gerardo Heiss</h3><p><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina, Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Heiss%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Gerardo Heiss</a></div></div><div id="_co_idm139953985499904"><h3 class="no_margin">Lucia A. Hindorff</h3><p><sup>24</sup>National Institutes of Health, National Human Genome Research Institute, Office of Population Genomics, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Hindorff%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Lucia A. Hindorff</a></div></div><div id="_co_idm139953985497504"><h3 class="no_margin">Wen-Chi Hsueh</h3><p><sup>25</sup>Phoenix Epidemiology and Clinical Research Branch, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Phoenix, AZ, USA</p><div>Find articles by <a href="/pubmed/?term=Hsueh%20WC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Wen-Chi Hsueh</a></div></div><div id="_co_idm139953985495104"><h3 class="no_margin">Aaron Isaacs</h3><p><sup>26</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands</p><p><sup>27</sup>CARIM School for Cardiovascular Diseases, Maastricht Centre for Systems Biology (MaCSBio), Dept. of Biochemistry, Maastricht University, Maastricht, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Aaron Isaacs</a></div></div><div id="_co_idm139953985861744"><h3 class="no_margin">Yalda Jamshidi</h3><p><sup>28</sup>Cardiogenetics Lab, Institute of Cardiovascular and Cell Sciences, St George's University of London, UK</p><div>Find articles by <a href="/pubmed/?term=Jamshidi%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Yalda Jamshidi</a></div></div><div id="_co_idm139953985859344"><h3 class="no_margin">Kathleen F. Kerr</h3><p><sup>29</sup>Department of Biostatistics, School of Public Health, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Kerr%20KF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Kathleen F. Kerr</a></div></div><div id="_co_idm139953985856944"><h3 class="no_margin">Felix Liu</h3><p><sup>30</sup>Department of Epidemiology and Biostatistics, University of California, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Felix Liu</a></div></div><div id="_co_idm139953985854576"><h3 class="no_margin">Yongmei Liu</h3><p><sup>31</sup>Department of Epidemiology and Prevention, Division of Public Health Sciences, Wake Forest University, Winston-Salem, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Yongmei Liu</a></div></div><div id="_co_idm139953985852208"><h3 class="no_margin">Kurt K. Lohman</h3><p><sup>31</sup>Department of Epidemiology and Prevention, Division of Public Health Sciences, Wake Forest University, Winston-Salem, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Lohman%20KK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Kurt K. Lohman</a></div></div><div id="_co_idm139953985849872"><h3 class="no_margin">Jared W. Magnani</h3><p><sup>32</sup>Department of Medicine, Division of Cardiology, University of Pittsburgh Medical Center Heart and Vascular Institute, University of Pittsburgh, Pittsburgh, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Magnani%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Jared W. Magnani</a></div></div><div id="_co_idm139953985847472"><h3 class="no_margin">Joseph F. Maher</h3><p><sup>33</sup>Department of Medicine, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Maher%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Joseph F. Maher</a></div></div><div id="_co_idm139953985845072"><h3 class="no_margin">Reena Mehra</h3><p><sup>34</sup>Program for Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Mehra%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Reena Mehra</a></div></div><div id="_co_idm139953985842672"><h3 class="no_margin">Yan A. Meng</h3><p><sup>35</sup>Jackson Heart Study, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Meng%20YA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Yan A. Meng</a></div></div><div id="_co_idm139953985840240"><h3 class="no_margin">Solomon K. Musani</h3><p><sup>36</sup>Cardiovascular Research Center and Center for Human Genetic Research, Massachusetts General Hospital and Harvard Medical School, Boston,MA, USA</p><div>Find articles by <a href="/pubmed/?term=Musani%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Solomon K. Musani</a></div></div><div id="_co_idm139953988153120"><h3 class="no_margin">Christopher Newton-Cheh</h3><p><sup>35</sup>Jackson Heart Study, University of Mississippi Medical Center, Jackson, MS, USA</p><p><sup>37</sup>Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Newton-Cheh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Christopher Newton-Cheh</a></div></div><div id="_co_idm139953988149760"><h3 class="no_margin">Kari E. North</h3><p><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina, Chapel Hill, NC, USA</p><p><sup>38</sup>Department of Health Services, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=North%20KE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Kari E. North</a></div></div><div id="_co_idm139953988146400"><h3 class="no_margin">Bruce M. Psaty</h3><p><sup>4</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</p><p><sup>22</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</p><p><sup>23</sup>Group Health Research Institute, Group Health Cooperative, Seattle, WA, USA</p><p><sup>39</sup>Department of Medicine, Division of Sleep Medicine, Brigham and Women's Hospital, Harvard Medical School, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Bruce M. Psaty</a></div></div><div id="_co_idm139953988141120"><h3 class="no_margin">Susan Redline</h3><p><sup>40</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute and Departments of Medicine and Pediatrics, Harbor-UCLA Medical Center, Torrance, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Redline%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Susan Redline</a></div></div><div id="_co_idm139953988138720"><h3 class="no_margin">Jerome I. Rotter</h3><p><sup>41</sup>University Heart Center Hamburg and German Center for Cardiovascular Research, Hamburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Jerome I. Rotter</a></div></div><div id="_co_idm139953988136320"><h3 class="no_margin">Renate B. Schnabel</h3><p><sup>42</sup>J. Craig Venter Institute, San Diego, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Schnabel%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Renate B. Schnabel</a></div></div><div id="_co_idm139953988133920"><h3 class="no_margin">Nicholas J. Schork</h3><p><sup>43</sup>Center for Cardiovascular Research, John A. Burns School of Medicine, University of Hawaii, Honolulu, HI, USA</p><div>Find articles by <a href="/pubmed/?term=Schork%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Nicholas J. Schork</a></div></div><div id="_co_idm139953986086176"><h3 class="no_margin">Ralph V. Shohet</h3><p><sup>44</sup>Department of Cellular and Molecular Medicine, Cleveland Clinic, Cleveland, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Shohet%20RV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Ralph V. Shohet</a></div></div><div id="_co_idm139953986083776"><h3 class="no_margin">Andrew B. Singleton</h3><p><sup>3</sup>Laboratory of Neurogenetics, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Andrew B. Singleton</a></div></div><div id="_co_idm139953986081440"><h3 class="no_margin">Jonathan D. Smith</h3><p><sup>45</sup>Epidemiological Cardiology Research Center (EPICARE), Department of Epidemiology and Prevention, Division of Public Health Sciences, Wake Forest School of Medicine, Winston-Salem, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Smith%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Jonathan D. Smith</a></div></div><div id="_co_idm139953986079040"><h3 class="no_margin">Elsayed Z. Soliman</h3><p><sup>46</sup>Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Soliman%20EZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Elsayed Z. Soliman</a></div></div><div id="_co_idm139953986076640"><h3 class="no_margin">Sathanur R. Srinivasan</h3><p><sup>15</sup>Department of Epidemiology, Tulane University, New Orleans, LA, USA</p><div>Find articles by <a href="/pubmed/?term=Srinivasan%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Sathanur R. Srinivasan</a></div></div><div id="_co_idm139953986074304"><h3 class="no_margin">Herman A. Taylor, Jr.</h3><p><sup>33</sup>Department of Medicine, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Taylor%20HA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Herman A. Taylor, Jr.</a></div></div><div id="_co_idm139953986071712"><h3 class="no_margin">David R. Van Wagoner</h3><p><sup>17</sup>Department of Molecular Cardiology, Lerner Research Institute, Cleveland Clinic, Cleveland, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Van%20Wagoner%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">David R. Van Wagoner</a></div></div><div id="_co_idm139953986069376"><h3 class="no_margin">James G. Wilson</h3><p><sup>47</sup>Laboratory of Epidemiology and Population Science, National Institute on Aging, National Institutes of Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Wilson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">James G. Wilson</a></div></div><div id="_co_idm139953986066976"><h3 class="no_margin">Taylor Young</h3><p><sup>35</sup>Jackson Heart Study, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Young%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Taylor Young</a></div></div><div id="_co_idm139954053304256"><h3 class="no_margin">Zhu-Ming Zhang</h3><p><sup>46</sup>Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Zhang%20ZM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Zhu-Ming Zhang</a></div></div><div id="_co_idm139954053301920"><h3 class="no_margin">Alan B. Zonderman</h3><p><sup>48</sup>Department of Pathology, University of California San Diego, La Jolla, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Zonderman%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Alan B. Zonderman</a></div></div><div id="_co_idm139954053299520"><h3 class="no_margin">Michele K. Evans</h3><p><sup>48</sup>Department of Pathology, University of California San Diego, La Jolla, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Evans%20MK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Michele K. Evans</a></div></div><div id="_co_idm139954053297184"><h3 class="no_margin">Luigi Ferrucci</h3><p><sup>7</sup>Translational Gerontology Branch, National Institute on Aging, National Institutes of Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Ferrucci%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Luigi Ferrucci</a></div></div><div id="_co_idm139954053294848"><h3 class="no_margin">Sarah S. Murray</h3><p><sup>49</sup>Department of Medicine, School of Medicine, University of North Carolina, Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Murray%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Sarah S. Murray</a></div></div><div id="_co_idm139954053292448"><h3 class="no_margin">Gregory J. Tranah</h3><p><sup>1</sup>California Pacific Medical Center Research Institute, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Tranah%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Gregory J. Tranah</a></div></div><div id="_co_idm139954053290112"><h3 class="no_margin">Eric A. Whitsel</h3><p><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina, Chapel Hill, NC, USA</p><p><sup>50</sup>Membership of the CHARGE QRS Consortium is provided in the acknowledgements and</p><div>Find articles by <a href="/pubmed/?term=Whitsel%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Eric A. Whitsel</a></div></div><div id="_co_idm139954053286752"><h3 class="no_margin">Alex P. Reiner</h3><p><sup>4</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</p><p><sup>14</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Alex P. Reiner</a></div></div><div id="_co_idm139954053283456"><h3 class="no_margin">CHARGE QRS Consortium</h3><p><sup>51</sup>Division of Cardiology, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">CHARGE QRS Consortium</a></div></div><div id="_co_idm139953987130368"><h3 class="no_margin">Nona Sotoodehnia</h3><p><sup>22</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</p><p><sup>51</sup>Division of Cardiology, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Sotoodehnia%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27577874">Nona Sotoodehnia</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139953984739280_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139953984739280_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139953984739280_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139953984739280_ai" style="display:none"><div class="fm-affl" id="ddw284-aff1" lang="en"><sup>1</sup>California Pacific Medical Center Research Institute, San Francisco, CA, USA</div><div class="fm-affl" id="ddw284-aff2" lang="en"><sup>2</sup>Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina, Chapel Hill, NC, USA</div><div class="fm-affl" id="ddw284-aff3" lang="en"><sup>3</sup>Laboratory of Neurogenetics, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</div><div class="fm-affl" id="ddw284-aff4" lang="en"><sup>4</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="ddw284-aff5" lang="en"><sup>5</sup>Department of Quantitative Health Sciences, Lerner Research Institute, Cleveland Clinic, Cleveland, OH, USA</div><div class="fm-affl" id="ddw284-aff6" lang="en"><sup>6</sup>Department of Pediatrics and Rady Children's Hospital, University of California at San Diego, School of Medicine, La Jolla, CA, USA</div><div class="fm-affl" id="ddw284-aff7" lang="en"><sup>7</sup>Translational Gerontology Branch, National Institute on Aging, National Institutes of Health, Baltimore, MD, USA</div><div class="fm-affl" id="ddw284-aff8" lang="en"><sup>8</sup>Center of Excellence in Minority Health and Health Disparities, Jackson State University, Jackson, MS, USA</div><div class="fm-affl" id="ddw284-aff9" lang="en"><sup>9</sup>Department of Epidemiology and Biostatistics, Jackson State University School of Public Health (Initiative), Jackson, MS, USA</div><div class="fm-affl" id="ddw284-aff10" lang="en"><sup>10</sup>Department of Epidemiology, Rollins School of Public Health, Emory University, Atlanta, GA, USA</div><div class="fm-affl" id="ddw284-aff11" lang="en"><sup>11</sup>McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD, USA</div><div class="fm-affl" id="ddw284-aff12" lang="en"><sup>12</sup>Department of Medicine and Cardiology, Tulane University, New Orleans, LA, USA</div><div class="fm-affl" id="ddw284-aff14" lang="en"><sup>13</sup>Department of Statistics and Biostatistics and Department of Genetics, Rutgers University, Piscataway, NJ, USA</div><div class="fm-affl" id="ddw284-aff15" lang="en"><sup>14</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</div><div class="fm-affl" id="ddw284-aff16" lang="en"><sup>15</sup>Department of Epidemiology, Tulane University, New Orleans, LA, USA</div><div class="fm-affl" id="ddw284-aff17" lang="en"><sup>16</sup>Department of Cardiovascular Medicine, Heart and Vascular Institute, Cleveland Clinic, Cleveland, OH, USA</div><div class="fm-affl" id="ddw284-aff18" lang="en"><sup>17</sup>Department of Molecular Cardiology, Lerner Research Institute, Cleveland Clinic, Cleveland, OH, USA</div><div class="fm-affl" id="ddw284-aff19" lang="en"><sup>18</sup>Cleveland Clinic Lerner College of Medicine of Case Western Reserve University, Cleveland, OH, USA</div><div class="fm-affl" id="ddw284-aff20" lang="en"><sup>19</sup>Division of Cardiovascular Medicine, University of Pennsylvania, Philadelphia, PA, USA</div><div class="fm-affl" id="ddw284-aff21" lang="en"><sup>20</sup>Departments of Family Medicine and Epidemiology, Alpert Medical School, Brown University, Providence, RI, USA</div><div class="fm-affl" id="ddw284-aff22" lang="en"><sup>21</sup>Department of Medicine, Division of Cardiovascular Disease, University of Mississippi Medical Center, Jackson, MS, USA</div><div class="fm-affl" id="ddw284-aff13" lang="en"><sup>22</sup>Department of Epidemiology, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="ddw284-aff23" lang="en"><sup>23</sup>Group Health Research Institute, Group Health Cooperative, Seattle, WA, USA</div><div class="fm-affl" id="ddw284-aff24" lang="en"><sup>24</sup>National Institutes of Health, National Human Genome Research Institute, Office of Population Genomics, Bethesda, MD, USA</div><div class="fm-affl" id="ddw284-aff25" lang="en"><sup>25</sup>Phoenix Epidemiology and Clinical Research Branch, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Phoenix, AZ, USA</div><div class="fm-affl" id="ddw284-aff26" lang="en"><sup>26</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands</div><div class="fm-affl" id="ddw284-aff27" lang="en"><sup>27</sup>CARIM School for Cardiovascular Diseases, Maastricht Centre for Systems Biology (MaCSBio), Dept. of Biochemistry, Maastricht University, Maastricht, the Netherlands</div><div class="fm-affl" id="ddw284-aff28" lang="en"><sup>28</sup>Cardiogenetics Lab, Institute of Cardiovascular and Cell Sciences, St George's University of London, UK</div><div class="fm-affl" id="ddw284-aff29" lang="en"><sup>29</sup>Department of Biostatistics, School of Public Health, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="ddw284-aff30" lang="en"><sup>30</sup>Department of Epidemiology and Biostatistics, University of California, San Francisco, CA, USA</div><div class="fm-affl" id="ddw284-aff31" lang="en"><sup>31</sup>Department of Epidemiology and Prevention, Division of Public Health Sciences, Wake Forest University, Winston-Salem, NC, USA</div><div class="fm-affl" id="ddw284-aff32" lang="en"><sup>32</sup>Department of Medicine, Division of Cardiology, University of Pittsburgh Medical Center Heart and Vascular Institute, University of Pittsburgh, Pittsburgh, PA, USA</div><div class="fm-affl" id="ddw284-aff33" lang="en"><sup>33</sup>Department of Medicine, University of Mississippi Medical Center, Jackson, MS, USA</div><div class="fm-affl" id="ddw284-aff34" lang="en"><sup>34</sup>Program for Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, MA, USA</div><div class="fm-affl" id="ddw284-aff35" lang="en"><sup>35</sup>Jackson Heart Study, University of Mississippi Medical Center, Jackson, MS, USA</div><div class="fm-affl" id="ddw284-aff36" lang="en"><sup>36</sup>Cardiovascular Research Center and Center for Human Genetic Research, Massachusetts General Hospital and Harvard Medical School, Boston,MA, USA</div><div class="fm-affl" id="ddw284-aff37" lang="en"><sup>37</sup>Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, NC, USA</div><div class="fm-affl" id="ddw284-aff38" lang="en"><sup>38</sup>Department of Health Services, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="ddw284-aff39" lang="en"><sup>39</sup>Department of Medicine, Division of Sleep Medicine, Brigham and Women's Hospital, Harvard Medical School, Boston, MA, USA</div><div class="fm-affl" id="ddw284-aff40" lang="en"><sup>40</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute and Departments of Medicine and Pediatrics, Harbor-UCLA Medical Center, Torrance, CA, USA</div><div class="fm-affl" id="ddw284-aff41" lang="en"><sup>41</sup>University Heart Center Hamburg and German Center for Cardiovascular Research, Hamburg, Germany</div><div class="fm-affl" id="ddw284-aff42" lang="en"><sup>42</sup>J. Craig Venter Institute, San Diego, CA, USA</div><div class="fm-affl" id="ddw284-aff43" lang="en"><sup>43</sup>Center for Cardiovascular Research, John A. Burns School of Medicine, University of Hawaii, Honolulu, HI, USA</div><div class="fm-affl" id="ddw284-aff44" lang="en"><sup>44</sup>Department of Cellular and Molecular Medicine, Cleveland Clinic, Cleveland, OH, USA</div><div class="fm-affl" id="ddw284-aff45" lang="en"><sup>45</sup>Epidemiological Cardiology Research Center (EPICARE), Department of Epidemiology and Prevention, Division of Public Health Sciences, Wake Forest School of Medicine, Winston-Salem, NC, USA</div><div class="fm-affl" id="ddw284-aff46" lang="en"><sup>46</sup>Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS, USA</div><div class="fm-affl" id="ddw284-aff47" lang="en"><sup>47</sup>Laboratory of Epidemiology and Population Science, National Institute on Aging, National Institutes of Health, Baltimore, MD, USA</div><div class="fm-affl" id="ddw284-aff48" lang="en"><sup>48</sup>Department of Pathology, University of California San Diego, La Jolla, CA, USA</div><div class="fm-affl" id="ddw284-aff49" lang="en"><sup>49</sup>Department of Medicine, School of Medicine, University of North Carolina, Chapel Hill, NC, USA</div><div class="fm-affl" id="ddw284-aff50" lang="en"><sup>50</sup>Membership of the CHARGE QRS Consortium is provided in the acknowledgements and</div><div class="fm-affl" id="ddw284-aff51" lang="en"><sup>51</sup>Division of Cardiology, University of Washington, Seattle, WA, USA</div><div id="ddw284-FM1"><sup>†</sup>The authors wish it to be known that, in their opinion, the first three authors should be regarded as joint First Authors.</div><div id="ddw284-cor1"><sup>*</sup>To whom correspondence should be addressed at: Daniel S. Evans, Mission Hall: Global Health &amp; Clinical Sciences Building, 550 16<sup>th</sup> Street, 2<sup>nd</sup> floor, Box #0560, San Francisco, CA 94158-2549, USA. Tel: 415-476-6090; Fax: 415-514-8150; Email: <a href="mailto:dev@null" data-email=".ude.fscu.gsp@snaved" class="oemail">.ude.fscu.gsp@snaved</a> Nona Sotoodehnia, Metropolitan Park East Tower, 1730 Minor Ave, Suite 1360, Seattle, WA 98101, USA. Tel: 206-221-7775; Fax: 206-221-2662; Email: <a href="mailto:dev@null" data-email="ude.notgnihsaw.u@ootosn" class="oemail">ude.notgnihsaw.u@ootosn</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139953984739280_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 Apr 11; Revised 2016 Aug 3; Accepted 2016 Aug 19.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139953984739280_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2016. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5291202/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm139953982941104" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139953982941104title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">The electrocardiographic QRS duration, a measure of ventricular depolarization and conduction, is associated with cardiovascular mortality. While single nucleotide polymorphisms (SNPs) associated with QRS duration have been identified at 22 loci in populations of European descent, the genetic architecture of QRS duration in non-European populations is largely unknown. We therefore performed a genome-wide association study (GWAS) meta-analysis of QRS duration in 13,031 African Americans from ten cohorts and a transethnic GWAS meta-analysis with additional results from populations of European descent. In the African American GWAS, a single genome-wide significant SNP association was identified (rs3922844, <em>P =</em> 4 × 10<sup>−14</sup>) in intron 16 of <em>SCN5A</em>, a voltage-gated cardiac sodium channel gene. The QRS-prolonging rs3922844 C allele was also associated with decreased <em>SCN5A</em> RNA expression in human atrial tissue (<em>P =</em> 1.1 × 10<sup>−4</sup>). High density genotyping revealed that the <em>SCN5A</em> association region in African Americans was confined to intron 16. Transethnic GWAS meta-analysis identified novel SNP associations on chromosome 18 in <em>MYL12A</em> (rs1662342, <em>P =</em> 4.9 × 10<sup>−8</sup>) and chromosome 1 near <em>CD1E</em> and <em>SPTA1</em> (rs7547997, <em>P =</em> 7.9 × 10<sup>−9</sup>). The 22 QRS loci previously identified in populations of European descent were enriched for significant SNP associations with QRS duration in African Americans (<em>P =</em> 9.9 × 10<sup>−7</sup>), and index SNP associations in or near <em>SCN5A</em>, <em>SCN10A</em>, <em>CDKN1A</em>, <em>NFIA</em>, <em>HAND1</em>, <em>TBX5</em> and <em>SETBP1</em> replicated in African Americans. In summary, rs3922844 was associated with QRS duration and <em>SCN5A</em> expression, two novel QRS loci were identified using transethnic meta-analysis, and a significant proportion of QRS–SNP associations discovered in populations of European descent were transferable to African Americans when adequate power was achieved.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p3" class="p p-first">The QRS duration, measured by the surface electrocardiogram (ECG), reflects depolarization and conduction of the electrical signal throughout the ventricular myocardium. Longer QRS duration is associated with an increased risk of cardiovascular (CV) mortality in Caucasians and all-cause mortality in African Americans from study populations unselected for specific CV disorders (<a href="#ddw284-B1" rid="ddw284-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871083">1</a>,<a href="#ddw284-B2" rid="ddw284-B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871082">2</a>). Among patients hospitalized for heart failure with reduced left ventricular ejection fraction, prolonged QRS duration (≥ 120 milliseconds) is associated with post-discharge all-cause mortality (<a href="#ddw284-B3" rid="ddw284-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871060">3</a>). The QRS duration has been reported to be shorter in African Americans compared to Caucasians, even after taking into account cardiovascular risk factors and coronary heart disease (<a href="#ddw284-B4" rid="ddw284-B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871063">4–6</a>).</p><p id="__p4" class="p p-last">Prior genome-wide association (GWA) studies of QRS duration performed in populations of European descent have identified 22 independent QRS loci (<a href="#ddw284-B7" rid="ddw284-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871080">7</a>,<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871077">8</a>). The genetic architecture of QRS duration among African Americans, by contrast, is largely unknown. Expanding GWA studies to populations of African descent holds the potential to refine association regions due to shorter-range linkage disequilibrium (LD) and to discover novel genetic associations given the presence of population-specific allele frequencies (<a href="#ddw284-B9" rid="ddw284-B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871183">9</a>,<a href="#ddw284-B10" rid="ddw284-B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871112">10</a>). We therefore performed a GWAS meta-analysis of QRS duration in a total of 13,031 African Americans from 10 cohort studies in order to: (1) fine-map the QRS association regions previously identified among those of European descent; (2) discover novel QRS loci through a meta-analysis of results from African American cohorts as well as a transethnic meta-analysis; (3) determine whether SNP associations with QRS duration discovered in populations of European descent are transferable to African Americans.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Results</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">GWAS meta-analysis of QRS duration in African Americans</h3><p id="__p5" class="p p-first">We conducted a meta-analysis of 2,955,816 autosomal SNPs in 13,031 African Americans from 10 GWA studies of QRS duration (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Materials, Tables S1 and S2), </a>with little evidence of genomic inflation in the individual cohorts (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Materials, Figs S1–S10 and Table S1</a>) or the meta-analysis (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S11</a>, λ = 1.027). The percentage of European genetic ancestry was similar across the cohorts (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>) and was not significantly associated with QRS duration (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>).</p><p id="__p6">A single SNP, rs3922844 (MAF = 0.42), in intron 16 of the cardiac sodium channel gene <em>SCN5A</em>, was associated with QRS duration among African Americans at the genome-wide significance threshold (β ± SE = 0.94 ms ± 0.12 ms, <em>P =</em> 4×10<sup>−14</sup>) (<a href="/pmc/articles/PMC5291202/table/ddw284-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T1" rid-ob="ob-ddw284-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="/pmc/articles/PMC5291202/figure/ddw284-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw284-F1" rid-ob="ob-ddw284-F1" co-legend-rid="lgnd_ddw284-F1"><span>Fig. 1</span></a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S12</a>). Adjustment for local ancestry minimally altered the association (<a href="/pmc/articles/PMC5291202/table/ddw284-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T1" rid-ob="ob-ddw284-T1" co-legend-rid=""><span>Table 1</span></a>). We fine-mapped the <em>SCN5A</em>-<em>SCN10A</em> region surrounding this signal by examining SNPs genotyped using the MetaboChip array in ARIC and WHI PAGE participants and imputed in the WHI SHARE participants (<a href="#ddw284-B11" rid="ddw284-B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871129">11</a>). In the meta-analysis of the three cohorts, a second SNP in intron 16 of <em>SCN5A</em>, rs12635898, was in high LD with rs3922844 (HapMap YRI: <em>r</em><sup>2 </sup>=<sup> </sup>0.93, <a href="/pmc/articles/PMC5291202/table/ddw284-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T2" rid-ob="ob-ddw284-T2" co-legend-rid=""><span>Table 2</span></a>) and was similarly associated with QRS duration (<a href="/pmc/articles/PMC5291202/table/ddw284-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T2" rid-ob="ob-ddw284-T2" co-legend-rid=""><span>Table 2</span></a>). Furthermore, LD was low (1000 Genomes AFR population <em>r</em><sup>2 </sup>=<sup> </sup>0.03) between rs3922844 and rs7626962, an <em>SCN5A</em> missense mutation (S1103Y) associated with cardiac conduction and arrhythmias that is common in African Americans but rare in populations of European descent (<a href="#ddw284-B12" rid="ddw284-B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871087">12–15</a>).


</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw284-F1" co-legend-rid="lgnd_ddw284-F1"><a href="/pmc/articles/PMC5291202/figure/ddw284-F1/" target="figure" rid-figpopup="ddw284-F1" rid-ob="ob-ddw284-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139953983893408" class="figure"><a href="/pmc/articles/PMC5291202/figure/ddw284-F1/" target="figure" rid-figpopup="ddw284-F1" rid-ob="ob-ddw284-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5291202_ddw284f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddw284f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5291202/bin/ddw284f1.jpg"></a></div><div id="largeobj_idm139953983893408" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5291202/figure/ddw284-F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw284-F1"><div><a class="figpopup" href="/pmc/articles/PMC5291202/figure/ddw284-F1/" target="figure" rid-figpopup="ddw284-F1" rid-ob="ob-ddw284-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p7">Regional association plot at <em>SCN5A/SCN10A</em> locus. African American SNP association meta-analysis results are plotted in the top panel, and meta-analysis results from cohorts of European ancestry are plotted in the bottom panel. The AA index SNP (rs3922844) is designated by a red diamond in both panels. The LD (<em>r</em><sup>2</sup>) shown is relative to the AA index SNP and is based on HapMap YRI in the top panel and HapMap CEU in the bottom panel. Gray circles are SNPs without HapMap LD data. The X-axis marks the chromosomal position. Recombination rates estimated from African Americans and HapMap CEU individuals are shown in the top and bottom panels, respectively. The dashed horizontal lines in the top and bottom panels mark the GWAS significance level in African Americans (2.5 × 10 <sup>−</sup> <sup>8</sup>) and populations of European descent (5.0 × 10 <sup>−</sup> <sup>8</sup>), respectively.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw284-T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p8">Association results for rs3922844 with and without adjustment for local ancestry estimates</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139953983890240" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1">Assay</th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1" align="center">Adjusted for global ancestry<sup>d</sup><hr></th><th colspan="2" rowspan="1" align="center">Adjusted for local ancestry<sup>d</sup><sup>,e</sup><hr></th></tr><tr><th rowspan="1" colspan="1">Study</th><th rowspan="1" colspan="1">(G/I)<sup>a</sup></th><th rowspan="1" colspan="1">CAF<sup>b</sup></th><th rowspan="1" colspan="1" align="center">n</th><th rowspan="1" colspan="1" align="center">β ± SE<sub>GC</sub></th><th rowspan="1" colspan="1" align="center"><em>P</em><sub>GC</sub></th><th rowspan="1" colspan="1" align="center">β ± SE<sub>GC</sub></th><th rowspan="1" colspan="1" align="center"><em>P</em><sub>GC</sub></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">WHI</td><td rowspan="1" colspan="1" align="center">I</td><td rowspan="1" colspan="1" align="center">0.42</td><td rowspan="1" colspan="1" align="center">4012</td><td rowspan="1" colspan="1" align="center">0.86 ± 0.21</td><td rowspan="1" colspan="1" align="center">6x10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">0.90 ± 0.23</td><td rowspan="1" colspan="1" align="center">3x10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">ARIC</td><td rowspan="1" colspan="1" align="center">I</td><td rowspan="1" colspan="1" align="center">0.41</td><td rowspan="1" colspan="1" align="center">2372</td><td rowspan="1" colspan="1" align="center">0.61 ± 0.28</td><td rowspan="1" colspan="1" align="center">0.03</td><td rowspan="1" colspan="1" align="center">0.68 ± 0.29</td><td rowspan="1" colspan="1" align="center">0.02</td></tr><tr><td rowspan="1" colspan="1">JHS</td><td rowspan="1" colspan="1" align="center">I</td><td rowspan="1" colspan="1" align="center">0.40</td><td rowspan="1" colspan="1" align="center">1918</td><td rowspan="1" colspan="1" align="center">1.41 ± 0.32</td><td rowspan="1" colspan="1" align="center">1x10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.40 ± 0.33</td><td rowspan="1" colspan="1" align="center">2x10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">MESA</td><td rowspan="1" colspan="1" align="center">I</td><td rowspan="1" colspan="1" align="center">0.43</td><td rowspan="1" colspan="1" align="center">1554</td><td rowspan="1" colspan="1" align="center">1.02 ± 0.36</td><td rowspan="1" colspan="1" align="center">0.005</td><td rowspan="1" colspan="1" align="center">1.17 ± 0.38</td><td rowspan="1" colspan="1" align="center">0.002</td></tr><tr><td rowspan="1" colspan="1">Health ABC</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.43</td><td rowspan="1" colspan="1" align="center">993</td><td rowspan="1" colspan="1" align="center">0.98 ± 0.47</td><td rowspan="1" colspan="1" align="center">0.04</td><td rowspan="1" colspan="1" align="center">0.99 ± 0.50</td><td rowspan="1" colspan="1" align="center">0.05</td></tr><tr><td rowspan="1" colspan="1">HANDLS</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.41</td><td rowspan="1" colspan="1" align="center">945</td><td rowspan="1" colspan="1" align="center">0.96 ± 0.43</td><td rowspan="1" colspan="1" align="center">0.03</td><td rowspan="1" colspan="1" align="center">0.85 ± 0.45</td><td rowspan="1" colspan="1" align="center">0.06</td></tr><tr><td rowspan="1" colspan="1">CHS</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">621</td><td rowspan="1" colspan="1" align="center">0.84 ± 0.56</td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center">ND</td><td rowspan="1" colspan="1" align="center">ND</td></tr><tr><td rowspan="1" colspan="1">CFS</td><td rowspan="1" colspan="1" align="center">I</td><td rowspan="1" colspan="1" align="center">0.39</td><td rowspan="1" colspan="1" align="center">315</td><td rowspan="1" colspan="1" align="center">0.16 ± 0.80</td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="center">0.21 ± 0.82</td><td rowspan="1" colspan="1" align="center">0.80</td></tr><tr><td rowspan="1" colspan="1">BLSA</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">153</td><td rowspan="1" colspan="1" align="center">3.27 ± 0.99</td><td rowspan="1" colspan="1" align="center">9x10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">ND</td><td rowspan="1" colspan="1" align="center">ND</td></tr><tr><td rowspan="1" colspan="1">BHS</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.42</td><td rowspan="1" colspan="1" align="center">147</td><td rowspan="1" colspan="1" align="center">−1.14 ± 1.27</td><td rowspan="1" colspan="1" align="center">0.37</td><td rowspan="1" colspan="1" align="center">ND</td><td rowspan="1" colspan="1" align="center">ND</td></tr><tr><td rowspan="1" colspan="1">Meta-analysis</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">All studies</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.42</td><td rowspan="1" colspan="1" align="center">12,877</td><td rowspan="1" colspan="1" align="center">0.94 ± 0.12</td><td rowspan="1" colspan="1" align="center">4x10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td></tr><tr><td rowspan="1" colspan="1">LA studies<sup>c</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.42</td><td rowspan="1" colspan="1" align="center">12,109</td><td rowspan="1" colspan="1" align="center">0.91 ± 0.13</td><td rowspan="1" colspan="1" align="center">3x10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">0.95 ± 0.13</td><td rowspan="1" colspan="1" align="center">7x10<sup>−13</sup></td></tr></tbody></table></div><div id="largeobj_idm139953983890240" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5291202/table/ddw284-T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw284-TF1"><p id="__p9" class="p p-first-last"><sup>a</sup>G = directly genotyped SNP, I = imputed SNP.</p></div><div id="ddw284-TF2"><p id="__p10" class="p p-first-last"><sup>b</sup>Coded allele frequency. C and T are the coded and non-coded alleles for rs3922844, respectively. CAF for the meta-analysis is the weighted average across the 10 cohorts.</p></div><div id="ddw284-TF3"><p id="__p11" class="p p-first-last"><sup>c</sup>Studies with local ancestry (LA) estimates: WHI, ARIC, JHS, MESA, Health ABC, HANDLS, and CFS.</p></div><div id="ddw284-TF4"><p id="__p12" class="p p-first-last"><sup>d</sup>β and SE expressed in units of milliseconds.</p></div><div id="ddw284-TF5"><p id="__p13" class="p p-first-last"><sup>e</sup>ND = not determined.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw284-T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p14">Association results from dense SNP genotyping study at the <em>SCN5A</em> locus</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139953983487744" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="2" colspan="1">Coded/Reference allele (CAF)</th><th colspan="2" rowspan="1" align="center">ARIC<hr></th><th colspan="2" rowspan="1" align="center">WHI PAGE<hr></th><th colspan="2" rowspan="1" align="center">WHI SHARE<hr></th><th rowspan="1" colspan="1">Meta-analysis</th><th rowspan="2" colspan="1">rs3922844 LD (<em>r</em><sup>2</sup>)<sup>c</sup></th></tr><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Position<sup>a</sup></th><th rowspan="1" colspan="1">n</th><th rowspan="1" colspan="1">β ± SE<sup>b</sup> (<em>P</em>)</th><th rowspan="1" colspan="1">n</th><th rowspan="1" colspan="1">β ± SE<sup>b</sup> (<em>P</em>)</th><th rowspan="1" colspan="1">n</th><th rowspan="1" colspan="1">β ± SE<sup>b</sup> (<em>P</em>)</th><th rowspan="1" colspan="1">β ± SE<sup>b</sup> (<em>P</em>)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs12635898</td><td rowspan="1" colspan="1">38,601,069</td><td rowspan="1" colspan="1">A/C (0.30)</td><td rowspan="1" colspan="1">2911</td><td rowspan="1" colspan="1">1.03 ± 0.34 (0.002)</td><td rowspan="1" colspan="1">725</td><td rowspan="1" colspan="1">1.18 ± 0.52 (0.02)</td><td rowspan="1" colspan="1">3283</td><td rowspan="1" colspan="1">1.04 ± 0.24 (2x10<sup>−5</sup>)</td><td rowspan="1" colspan="1">1.05 ± 0.19 (1x10<sup>−8</sup>)</td><td rowspan="1" colspan="1">0.93</td></tr></tbody></table></div><div id="largeobj_idm139953983487744" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5291202/table/ddw284-T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw284-TF6"><p id="__p15" class="p p-first-last"><sup>a</sup>Position based on NCBI reference sequence build 36.</p></div><div id="ddw284-TF7"><p id="__p16" class="p p-first-last"><sup>b</sup>β and SE expressed in units of milliseconds.</p></div><div id="ddw284-TF8"><p id="__p17" class="p p-first-last"><sup>c</sup>LD estimates between rs12635898 and rs3922844 based on rs12635898 genotype data from unrelated YRI samples on MetaboChip arrays used in the PAGE study and rs3922844 genotype data from unrelated YRI samples downloaded from HapMap phase 2 data.</p></div></div></div><p id="__p18" class="p p-last">While the most significant SNP associations reported from previous GWA studies of QRS interval among other ethnic groups are similarly located on chromosome 3 at the <em>SCN5A</em>/<em>SCN10A</em> locus, the SNP association region is broad, spanning approximately 300 kb, and multiple independent signals have been identified (<a href="/pmc/articles/PMC5291202/figure/ddw284-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw284-F1" rid-ob="ob-ddw284-F1" co-legend-rid="lgnd_ddw284-F1"><span>Fig. 1</span></a>) (<a href="#ddw284-B7" rid="ddw284-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871173">7</a>,<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871108">8</a>,<a href="#ddw284-B16" rid="ddw284-B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871070">16</a>). By contrast, the genome-wide association among African Americans points to a single SNP (<a href="/pmc/articles/PMC5291202/figure/ddw284-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw284-F1" rid-ob="ob-ddw284-F1" co-legend-rid="lgnd_ddw284-F1"><span>Fig. 1</span></a>). The rs3922844-QRS association discovered in African American cohorts replicated in cohorts of European ancestry (<em>P =</em> 2×10<sup>−13</sup>) (<a href="/pmc/articles/PMC5291202/table/ddw284-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T3" rid-ob="ob-ddw284-T3" co-legend-rid=""><span>Table 3</span></a>), but this variant was not in high LD with other <em>SCN5A</em> or <em>SCN10A</em> index SNPs among European ancestry individuals (<a href="/pmc/articles/PMC5291202/table/ddw284-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T3" rid-ob="ob-ddw284-T3" co-legend-rid=""><span>Table 3</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw284-T3"><!--caption a7--><div class="caption"><p id="__p19">Table <strong>3.</strong> SNP association transferability at the <em>SCN5A/SCN10A</em> locus</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139953982889264" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="4" rowspan="1" align="center">European ancestry<hr></th><th colspan="4" rowspan="1" align="center">African ancestry<hr></th><th colspan="3" rowspan="1" align="center">rs3922844 LD (r<sup>2</sup>)<sup>d</sup><hr></th></tr><tr><th rowspan="1" colspan="1">Index SNP/ annotation</th><th rowspan="1" colspan="1">Nearest gene</th><th rowspan="1" colspan="1">Chr (Position)<sup>h</sup></th><th rowspan="1" colspan="1">Discovery population (Ref)</th><th rowspan="1" colspan="1">CA (Freq)<sup>a</sup></th><th rowspan="1" colspan="1">β±SE<sub>GC</sub> (<em>P</em><sub>GC</sub>)<sup>e</sup></th><th rowspan="1" colspan="1">I<sup>2 f</sup></th><th rowspan="1" colspan="1">Power<sup>b</sup></th><th rowspan="1" colspan="1">CA (Freq)<sup>a</sup></th><th rowspan="1" colspan="1">β±SE<sub>GC</sub> (<em>P</em><sub>GC</sub>)<sup>g</sup></th><th rowspan="1" colspan="1">I<sup>2 f</sup></th><th rowspan="1" colspan="1">Power<sup>c</sup></th><th rowspan="1" colspan="1">CEU</th><th rowspan="1" colspan="1">YRI</th><th rowspan="1" colspan="1">ASW</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs6801957/</td><td rowspan="1" colspan="1"><em>SCN10A</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">EA</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1">0.77 ± 0.07</td><td rowspan="1" colspan="1"><strong>45.3</strong></td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1">0.54 ± 0.16</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1">0.95</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">ND</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(38,742,319)</td><td rowspan="1" colspan="1">(<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871106">8</a>)</td><td rowspan="1" colspan="1" align="center">(0.41)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−28</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.17)</td><td rowspan="1" colspan="1" align="center">(<strong>8x10<sup>−4</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs6795970/</td><td rowspan="1" colspan="1"><em>SCN10A</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">EA/Asian</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1">0.75 ± 0.07</td><td rowspan="1" colspan="1">44.8</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1">0.44 ± 0.20</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1">0.73</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.03</td></tr><tr><td rowspan="1" colspan="1">missense</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(38,741,679)</td><td rowspan="1" colspan="1">(<a href="#ddw284-B7" rid="ddw284-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871144">7</a>,<a href="#ddw284-B16" rid="ddw284-B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871085">16</a>)</td><td rowspan="1" colspan="1" align="center">(0.40)</td><td rowspan="1" colspan="1" align="center">(<strong>5x10<sup>−27</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.10)</td><td rowspan="1" colspan="1" align="center">(0.03)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs9851724/</td><td rowspan="1" colspan="1"><em>SCN10A</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">EA</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">−0.66 ± 0.07</td><td rowspan="1" colspan="1"><strong>57.1</strong></td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">−0.30 ± 0.16</td><td rowspan="1" colspan="1">7.2</td><td rowspan="1" colspan="1">0.82</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.01</td></tr><tr><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(38,694,939)</td><td rowspan="1" colspan="1">(<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871158">8</a>)</td><td rowspan="1" colspan="1" align="center">(0.33)</td><td rowspan="1" colspan="1" align="center">(<strong>2x10<sup>−20</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.16)</td><td rowspan="1" colspan="1" align="center">(0.06)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs11710077/</td><td rowspan="1" colspan="1"><em>SCN5A</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">EA</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1">−0.84 ± 0.09</td><td rowspan="1" colspan="1">23.8</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1">−0.87 ± 0.19</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1">0.90</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.09</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(38,632,903)</td><td rowspan="1" colspan="1">(<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871147">8</a>)</td><td rowspan="1" colspan="1" align="center">(0.21)</td><td rowspan="1" colspan="1" align="center">(<strong>6x10<sup>−22</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.11)</td><td rowspan="1" colspan="1" align="center">(<strong>7x10<sup>−6</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs11708996/</td><td rowspan="1" colspan="1"><em>SCN5A</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">EA</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">0.79 ± 0.10</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">0.86 ± 0.80</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1">ND</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(38,608,927)</td><td rowspan="1" colspan="1">(<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871143">8</a>)</td><td rowspan="1" colspan="1" align="center">(0.16)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−16</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.04)</td><td rowspan="1" colspan="1" align="center">(0.28)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs10865879/</td><td rowspan="1" colspan="1"><em>SCN5A</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">EA</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">0.78 ± 0.08</td><td rowspan="1" colspan="1"><strong>53.6</strong></td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">0.46 ± 0.15</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">ND</td></tr><tr><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(38,552,366)</td><td rowspan="1" colspan="1">(<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871180">8</a>)</td><td rowspan="1" colspan="1" align="center">(0.26)</td><td rowspan="1" colspan="1" align="center">(<strong>3x10<sup>−23</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.18)</td><td rowspan="1" colspan="1" align="center">(3x10<sup>−3</sup>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs2051211/</td><td rowspan="1" colspan="1"><em>EXOG</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">EA</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1">−0.44 ± 0.08</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1">−0.41 ± 0.15</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1">0.38</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.05</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(38,534,753)</td><td rowspan="1" colspan="1">(<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871119">8</a>)</td><td rowspan="1" colspan="1" align="center">(0.26)</td><td rowspan="1" colspan="1" align="center">(<strong>2x10<sup>−8</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.18)</td><td rowspan="1" colspan="1" align="center">(7x10<sup>−3</sup>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs3922844/</td><td rowspan="1" colspan="1"><em>SCN5A</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">AA</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">0.56 ± 0.08</td><td rowspan="1" colspan="1">23.7</td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">0.94 ± 0.12</td><td rowspan="1" colspan="1">10.6</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(38,599,257)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.69)</td><td rowspan="1" colspan="1" align="center">(<strong>2x10<sup>−13</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.42)</td><td rowspan="1" colspan="1" align="center">(<strong>4x10<sup>−14</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm139953982889264" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5291202/table/ddw284-T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw284-TF9"><p id="__p20" class="p p-first-last"><sup>a</sup>CA = coded effect allele. Freq = frequency.</p></div><div id="ddw284-TF10"><p id="__p21" class="p p-first-last"><sup>b</sup>Power to detect effect size reported in COGENT/CARe results using the coded effect allele frequency and trait variance as observed in European samples and keeping α = 0.002 adjusting for 22 SNPs.</p></div><div id="ddw284-TF11"><p id="__p22" class="p p-first-last"><sup>c</sup>Power to detect effect size reported in European discovery population using the effect allele frequency and trait variance as observed in African-American samples and keeping α = 0.002 adjusting for 30 SNPs.</p></div><div id="ddw284-TF12"><p id="__p23" class="p p-first-last"><sup>d</sup>LD between index SNP and rs3922844. ND indicates the SNP was not genotyped in the HapMap population or was monomorphic.</p></div><div id="ddw284-TF13"><p id="__p24" class="p p-first-last"><sup>e</sup>In non-African ancestry discovery rows, significance was based on the genome-wide discovery threshold and <em>P</em>-values ≤ 5.0x10 <sup>−</sup> <sup>8</sup> were bolded. In the African ancestry discovery row, significance was based on the replication threshold and <em>P</em>-values ≤ 0.002 were bolded. β and SE expressed in units of milliseconds.</p></div><div id="ddw284-TF14"><p id="__p25" class="p p-first-last"><sup>f</sup>Heterogeneity I<sup>2</sup> statistics bolded if the Cochran’s χ<sup>2</sup>
<em>P</em>-value ≤ 0.05.</p></div><div id="ddw284-TF15"><p id="__p26" class="p p-first-last"><sup>g</sup>In non-African ancestry discovery rows, significance was based on replication of the 30 previously identified SNPs and <em>P</em>-values ≤ 0.002 were bolded. In the African ancestry discovery row, significance was based on genome-wide discovery and <em>P</em>-values ≤ 2.5x10 <sup>−</sup> <sup>8</sup> were bolded. β and SE expressed in units of milliseconds.</p></div><div id="ddw284-TF16"><p id="__p27" class="p p-first-last"><sup>h</sup>Position based on NCBI reference sequence build 36.</p></div></div></div></div><div id="__sec4" class="sec"><h3 id="__sec4title">SNP functional annotation and association with SCN5A expression in human cardiac tissue</h3><p id="__p28" class="p p-first">Functional annotation indicated that rs3922844 and variants in LD (1000 Genomes AFR population <em>r</em><sup>2 </sup>≥<sup> </sup>0.2) overlapped and were near regulatory genomic features. These variants altered transcription factor binding motifs and overlapped with DNAse I hypersensitive sites (DHS) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>). In multiple cell types, rs3922844 was located near a cluster of transcription factor binding events and a peak of Histone H3 Lysine 4 mono-methylation, a marker of putative enhancers (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S13</a>). Rs3922844 and variants in LD overlapped with DHS and enhancer histone marks in fetal heart tissue based on data from the Roadmap Epigenomics Mapping Consortium (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Materials, Table S4 and Fig. S14</a>).</p><p id="__p29" class="p p-last">We next assessed the functional relevance of rs3922844 by examining its association with <em>SCN5A</em> RNA expression levels in human cardiac tissue. The rs3922844 C allele associated with longer QRS duration was also associated with lower <em>SCN5A</em> RNA expression in left atrial appendage samples from 289 individuals of European ancestry (β ± SE (units are RNA levels on log<sub>2</sub> scale) = -0.11 ± 0.03, <em>P =</em> 1.6×10<sup>−3</sup>) and 40 individuals of African ancestry (β ± SE = -0.29 ± 0.11, <em>P =</em> 0.013), as well as in the meta-analysis across the two ethnic groups (β ± SE = -0.13 ± 0.03, <em>P =</em> 1.1×10<sup>−4</sup>, <a href="/pmc/articles/PMC5291202/figure/ddw284-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw284-F2" rid-ob="ob-ddw284-F2" co-legend-rid="lgnd_ddw284-F2"><span>Fig. 2</span></a>).

</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw284-F2" co-legend-rid="lgnd_ddw284-F2"><a href="/pmc/articles/PMC5291202/figure/ddw284-F2/" target="figure" rid-figpopup="ddw284-F2" rid-ob="ob-ddw284-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139953981954528" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddw284f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddw284f2.jpg" src="/pmc/articles/PMC5291202/bin/ddw284f2.jpg"></div></a><div id="largeobj_idm139953981954528" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC5291202/figure/ddw284-F2/" target="figure" rid-figpopup="ddw284-F2" rid-ob="ob-ddw284-F2"></a><a target="object" href="/pmc/articles/PMC5291202/figure/ddw284-F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw284-F2"><div><a class="figpopup" href="/pmc/articles/PMC5291202/figure/ddw284-F2/" target="figure" rid-figpopup="ddw284-F2" rid-ob="ob-ddw284-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p30">Association between rs3922844 genotype and <em>SCN5A</em> expression in human atrial tissue. Box plots display data from individuals of African ancestry (<strong>A</strong>) and European ancestry (<strong>B</strong>). Plotted along the Y-axis are RNA levels adjusted for covariates. The bottom and top of each box indicates the 25<sup>th</sup> and 75<sup>th</sup> percentiles, and the band within the box is the median. Whiskers extend to the most extreme value or the most extreme value within 1.5*interquartile range (IQR), whichever value is closer to the median. The fitted regression line is shown in blue.</p></div></div></div></div><div id="__sec5" class="sec"><h3 id="__sec5title">Transethnic meta-analysis of QRS duration</h3><p id="__p31" class="p p-first">QRS duration GWAS results from 13,031 African Americans (reported above) and from 40,407 European-ancestry individuals in the previously reported CHARGE analysis (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871094">8</a>) were meta-analysed, with little evidence of genomic inflation (λ = 1.018, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S15</a>). In addition to the previously identified QRS loci (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Materials, Fig. S16 and Table S5</a>), the transethnic meta-analysis identified SNPs at two novel loci associated with cardiac ventricular conduction: an intronic SNP (rs1662342, <em>P =</em> 4.9×10<sup>−8</sup>) in a myosin light chain regulatory gene, <em>MYL12A</em>, on chromosome 18 and an intergenic SNP (rs7547997, <em>P =</em> 7.9×10<sup>−9</sup>) near <em>CD1E</em> on chromosome 1 (<a href="/pmc/articles/PMC5291202/table/ddw284-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T4" rid-ob="ob-ddw284-T4" co-legend-rid=""><span>Table 4</span></a>, <a href="/pmc/articles/PMC5291202/figure/ddw284-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw284-F3" rid-ob="ob-ddw284-F3" co-legend-rid="lgnd_ddw284-F3"><span>Figs 3 and 4</span></a>). For both SNPs, the coded allele frequency was higher in African Americans than individuals of European ancestry (<a href="/pmc/articles/PMC5291202/table/ddw284-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T4" rid-ob="ob-ddw284-T4" co-legend-rid=""><span>Table 4</span></a>), indicating that the addition of African American participants may have provided greater gains in power for these two SNPs than the addition of an equivalent number of individuals of European ancestry.


</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw284-F3" co-legend-rid="lgnd_ddw284-F3"><a href="/pmc/articles/PMC5291202/figure/ddw284-F3/" target="figure" rid-figpopup="ddw284-F3" rid-ob="ob-ddw284-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139953981116560" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddw284f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddw284f3.jpg" src="/pmc/articles/PMC5291202/bin/ddw284f3.jpg"></div></a><div id="largeobj_idm139953981116560" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC5291202/figure/ddw284-F3/" target="figure" rid-figpopup="ddw284-F3" rid-ob="ob-ddw284-F3"></a><a target="object" href="/pmc/articles/PMC5291202/figure/ddw284-F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw284-F3"><div><a class="figpopup" href="/pmc/articles/PMC5291202/figure/ddw284-F3/" target="figure" rid-figpopup="ddw284-F3" rid-ob="ob-ddw284-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><p id="__p32">Regional association plot of transethnic meta-analysis results at <em>MYL12A</em> locus. Transethnic meta-analysis SNP association results are plotted in the top panel, meta-analysis results from cohorts of European ancestry are plotted in the middle panel, and African American SNP association meta-analysis results are plotted in the bottom panel. The transethnic index SNP (rs1662342) is designated by a red diamond in all panels. The LD (<em>r</em><sup>2</sup>) shown is relative to the index SNP and is based on HapMap CEU in the top and middle panels and is based on HapMap YRI in the bottom panel. Gray circles are SNPs without HapMap LD data. The X-axis marks the chromosomal position. Recombination rates averaged across HapMap populations, in HapMap CEU individuals, and in African Americans are shown in the top panel, middle panel, and bottom panel, respectively. The dashed horizontal line in the top and middle panels marks the GWAS significance level for European ancestry (5.0 × 10 <sup>−</sup> <sup>8</sup>) and in the bottom panel marks the GWAS significance level for African Americans (2.5 × 10 <sup>−</sup> <sup>8</sup>).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw284-F4" co-legend-rid="lgnd_ddw284-F4"><a href="/pmc/articles/PMC5291202/figure/ddw284-F4/" target="figure" rid-figpopup="ddw284-F4" rid-ob="ob-ddw284-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139953981110640" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddw284f4.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddw284f4.jpg" src="/pmc/articles/PMC5291202/bin/ddw284f4.jpg"></div></a><div id="largeobj_idm139953981110640" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC5291202/figure/ddw284-F4/" target="figure" rid-figpopup="ddw284-F4" rid-ob="ob-ddw284-F4"></a><a target="object" href="/pmc/articles/PMC5291202/figure/ddw284-F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw284-F4"><div><a class="figpopup" href="/pmc/articles/PMC5291202/figure/ddw284-F4/" target="figure" rid-figpopup="ddw284-F4" rid-ob="ob-ddw284-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><p id="__p33">Regional association plot of transethnic meta-analysis results at chromosome 1q23.1. Transethnic meta-analysis SNP association results are plotted in the top panel, meta-analysis results from cohorts of European ancestry are plotted in the middle panel, and African American SNP association meta-analysis results are plotted in the bottom panel. The transethnic index SNP (rs7547997) is designated by a red diamond in all panels. The LD (<em>r</em><sup>2</sup>) shown is relative to the index SNP and is based on HapMap CEU in the top and middle panels and is based on HapMap YRI in the bottom panel. Gray circles are SNPs without HapMap LD data. The X-axis marks the chromosomal position. Recombination rates averaged across HapMap populations, in HapMap CEU individuals, and in African Americans are shown in the top panel, middle panel, and bottom panel, respectively. The dashed horizontal line in the top and middle panels marks the GWAS significance level for European ancestry (5.0 × 10<sup>−8</sup>) and in the bottom panel marks the GWAS significance level for African Americans (2.5 × 10<sup>−8</sup>).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw284-T4"><h3>Table 4.</h3><!--caption a7--><div class="caption"><p id="__p34">Novel genome-wide significant SNP associations from transethnic meta-analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139953981107488" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="5" rowspan="1"></th><th colspan="3" rowspan="1" align="center">European ancestry<hr></th><th colspan="3" rowspan="1" align="center">African ancestry<hr></th><th colspan="3" rowspan="1" align="center">Meta-analysis<hr></th></tr><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Chr</th><th rowspan="1" colspan="1">Position<sup>a</sup></th><th rowspan="1" colspan="1">Nearby Genes</th><th rowspan="1" colspan="1">SNP Annotation</th><th rowspan="1" colspan="1">CA (Freq)<sup>b</sup></th><th rowspan="1" colspan="1">β ± SE<sub>GC</sub> (<em>P</em><sub>GC</sub>)<sup>c</sup></th><th rowspan="1" colspan="1">I<sup>2 d</sup></th><th rowspan="1" colspan="1">CA (Freq)<sup>b</sup></th><th rowspan="1" colspan="1">β ± SE<sub>GC</sub> (<em>P</em><sub>GC</sub>)<sup>c</sup></th><th rowspan="1" colspan="1">I<sup>2 d</sup></th><th rowspan="1" colspan="1">CA (Freq)<sup>b</sup></th><th rowspan="1" colspan="1">β ± SE<sub>GC</sub> (<em>P</em><sub>GC</sub>)<sup>c</sup></th><th rowspan="1" colspan="1">I<sup>2 d</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs1662342</td><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">3,245,301</td><td rowspan="1" colspan="1"><em>MYL12A</em></td><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1" align="center">A (0.14)</td><td rowspan="1" colspan="1">0.53 ± 0.10 (3.1x10<sup>−7</sup>)</td><td rowspan="1" colspan="1">34.7</td><td rowspan="1" colspan="1" align="center">A (0.21)</td><td rowspan="1" colspan="1">0.35 ± 0.16 (0.03)</td><td rowspan="1" colspan="1">16.6</td><td rowspan="1" colspan="1" align="center">A (0.16)</td><td rowspan="1" colspan="1">0.47 ± 0.09 (4.9x10<sup>−8</sup>)</td><td rowspan="1" colspan="1">0.0</td></tr><tr><td rowspan="1" colspan="1">rs7547997</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">156,607,897</td><td rowspan="1" colspan="1"><em>CD1E</em>, <em>OR10T2</em>, <em>SPTA1</em></td><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1" align="center">A (0.16)</td><td rowspan="1" colspan="1">0.48 ± 0.10 (1.0x10<sup>−6</sup>)</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1" align="center">A (0.59)</td><td rowspan="1" colspan="1">0.38 ± 0.12 (1.5x10<sup>−3</sup>)</td><td rowspan="1" colspan="1">18.8</td><td rowspan="1" colspan="1" align="center">A (0.33)</td><td rowspan="1" colspan="1">0.44 ± 0.08 (7.9x10<sup>−9</sup>)</td><td rowspan="1" colspan="1">0.0</td></tr></tbody></table></div><div id="largeobj_idm139953981107488" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5291202/table/ddw284-T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw284-TF17"><p id="__p35" class="p p-first-last"><sup>a</sup>Position based on NCBI reference sequence build 36.</p></div><div id="ddw284-TF18"><p id="__p36" class="p p-first-last"><sup>b</sup>CA = coded effect allele. Freq = frequency.</p></div><div id="ddw284-TF19"><p id="__p37" class="p p-first-last"><sup>c</sup>β and SE expressed in units of milliseconds.</p></div><div id="ddw284-TF20"><p id="__p38" class="p p-first-last"><sup>d</sup>Heterogeneity I<sup>2</sup> statistics bolded if the Cochran’s Χ<sup>2</sup>
<em>P</em>-value ≤ 0.05.</p></div></div></div><p id="__p39" class="p p-last">We examined whether rs1662342 or rs7547997 were eQTLs in human left atrial appendage tissue. While the rs1662342 A allele was nominally associated with higher <em>MYL12A</em> RNA levels in the meta-analysis of results from 289 individuals of European ancestry and 40 individuals of African ancestry (β ± SE = 0.08 ± 0.04, <em>P =</em> 0.03), the association was not significant after correction for multiple testing. Gene expression for only two genes (<em>CD1C</em> and <em>CD1E</em>) was detected within 250 kb upstream and downstream of rs7547997, and this SNP was not significantly associated with expression of either of these genes (<em>P &gt;</em> 0.05).</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Gene set enrichment and transferability of QRS-associated loci and SNPs</h3><p id="__p40" class="p p-first-last">A gene set enrichment analysis (GSEA) revealed that genes identified from the 22 European-descent QRS loci were enriched for significant SNP associations in the African American QRS meta-analysis results, suggesting the transferability of QRS associations at the gene-set level between the two population groups. Gene-based <em>P</em>-values for 9 of the 22 QRS loci were in the top 95<sup>th</sup> percentile of all gene scores genome-wide in African Americans, whereas only one would be expected by chance, and the 22 QRS loci were significantly enriched for significant QRS-SNP associations in African Americans compared with randomly sampled gene sets (GSEA empirical <em>P =</em> 9.9×10<sup>−7</sup>). Importantly, the direction of the association was the same for all index SNPs at the 22 previously identified QRS loci in both ethnic groups, further supporting the transferability of associations between those of European and African ancestry (<a href="/pmc/articles/PMC5291202/table/ddw284-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T3" rid-ob="ob-ddw284-T3" co-legend-rid=""><span>Tables 3</span></a> and <a href="/pmc/articles/PMC5291202/table/ddw284-T5/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T5" rid-ob="ob-ddw284-T5" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and5,</span></span><span>5</span></a>, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S6</a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw284-T5"><h3>Table 5.</h3><!--caption a7--><div class="caption"><p id="__p41">SNP replication results with adequate power from GWAS in African Americans</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139953983057024" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="4" rowspan="1" align="center">European ancestry<hr></th><th colspan="4" rowspan="1" align="center">African ancestry<hr></th><th colspan="3" rowspan="1" align="center">LD (r<sup>2</sup>)<sup>d</sup><hr></th></tr><tr><th rowspan="1" colspan="1">Index SNP/ annotation</th><th rowspan="1" colspan="1">Nearby gene</th><th rowspan="1" colspan="1">Chr (Position)<sup>h</sup></th><th rowspan="1" colspan="1">Discovery population (Ref)</th><th rowspan="1" colspan="1">CA (Freq)<sup>a</sup></th><th rowspan="1" colspan="1">β±SE<sub>GC</sub> (<em>P</em><sub>GC</sub>)<sup>e</sup></th><th rowspan="1" colspan="1">I<sup>2 f</sup></th><th rowspan="1" colspan="1">Power<sup>b</sup></th><th rowspan="1" colspan="1">CA (Freq)<sup>a</sup></th><th rowspan="1" colspan="1">β±SE<sub>GC</sub> (<em>P</em><sub>GC</sub>)<sup>g</sup></th><th rowspan="1" colspan="1">I<sup>2 f</sup></th><th rowspan="1" colspan="1">Power<sup>c</sup></th><th rowspan="1" colspan="1">CEU</th><th rowspan="1" colspan="1">YRI</th><th rowspan="1" colspan="1">ASW</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs9470361/</td><td rowspan="1" colspan="1"><em>CDKN1A</em></td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">EA (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871117">8</a>)</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.87 ± 0.08</td><td rowspan="1" colspan="1" align="center">14.6</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.43 ± 0.13</td><td rowspan="1" colspan="1" align="center">17.1</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">0.95</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">ND</td></tr><tr><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(36,731,357)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.25)</td><td rowspan="1" colspan="1" align="center">(<strong>3x10<sup>−27</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.31)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−3</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs1321311/</td><td rowspan="1" colspan="1"><em>CDKN1A</em></td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">EA (<a href="#ddw284-B7" rid="ddw284-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871121">7</a>)</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.94 ± 0.15</td><td rowspan="1" colspan="1" align="center">ND</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.40 ± 0.12</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">0.95</td><td rowspan="1" colspan="1" align="center">0.70</td><td rowspan="1" colspan="1" align="center">ND</td></tr><tr><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(36,730,878)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.21)</td><td rowspan="1" colspan="1" align="center">(<strong>3x10<sup>−10</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.39)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−3</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs7756236/</td><td rowspan="1" colspan="1"><em>CDKN1A</em></td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">AA</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.83 ± 0.08</td><td rowspan="1" colspan="1" align="center">34.3</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.53 ± 0.13</td><td rowspan="1" colspan="1" align="center">3.0</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td></tr><tr><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(36,735,031)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.26)</td><td rowspan="1" colspan="1" align="center">(<strong>2x10<sup>−26</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.30)</td><td rowspan="1" colspan="1" align="center">(4x10<sup>−5</sup>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs11153730/</td><td rowspan="1" colspan="1"><em>PLN</em></td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">EA (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871101">8</a>)</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0.59 ± 0.07</td><td rowspan="1" colspan="1" align="center">5.3</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0.32 ± 0.13</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">0.27</td><td rowspan="1" colspan="1" align="center">ND</td></tr><tr><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(118,774,215)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.49)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−18</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.29)</td><td rowspan="1" colspan="1" align="center">(0.02)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4307206/</td><td rowspan="1" colspan="1"><em>PLN</em></td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">AA</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.54 ± 0.07</td><td rowspan="1" colspan="1" align="center">14.4</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.52 ± 0.15</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(118,920,013)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.45)</td><td rowspan="1" colspan="1" align="center">(<strong>5x10<sup>−16</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.21)</td><td rowspan="1" colspan="1" align="center">(4x10<sup>−4</sup>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs9436640/</td><td rowspan="1" colspan="1"><em>NFIA</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">EA (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871131">8</a>)</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">−0.59 ± 0.07</td><td rowspan="1" colspan="1" align="center"><strong>51.2</strong></td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">−0.51 ± 0.12</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.95</td><td rowspan="1" colspan="1" align="center">0.54</td><td rowspan="1" colspan="1" align="center">0.53</td><td rowspan="1" colspan="1" align="center">0.69</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(61,646,265)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.46)</td><td rowspan="1" colspan="1" align="center">(<strong>5x10<sup>−18</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.37)</td><td rowspan="1" colspan="1" align="center">(<strong>2x10<sup>−5</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs2207791/</td><td rowspan="1" colspan="1"><em>NFIA</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1">AA</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">−0.59 ± 0.07</td><td rowspan="1" colspan="1" align="center">44.5</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">−0.62 ± 0.13</td><td rowspan="1" colspan="1" align="center">6.7</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(61,667,490)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.48)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−17</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.31)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−6</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs13165478/</td><td rowspan="1" colspan="1"><em>HAND1</em></td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">EA (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871169">8</a>)</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">−0.55 ± 0.07</td><td rowspan="1" colspan="1" align="center"><strong>64.6</strong></td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">−0.45 ± 0.14</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">1.00</td><td rowspan="1" colspan="1" align="center">1.00</td><td rowspan="1" colspan="1" align="center">ND</td></tr><tr><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(153,849,233)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.36)</td><td rowspan="1" colspan="1" align="center">(<strong>7x10<sup>−14</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.53)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−3</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs13185595/</td><td rowspan="1" colspan="1"><em>HAND1</em></td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1">AA</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.56 ± 0.07</td><td rowspan="1" colspan="1" align="center">64.8</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.44 ± 0.13</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td></tr><tr><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(153,852,363)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.63)</td><td rowspan="1" colspan="1" align="center">(<strong>9x10<sup>−14</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.47)</td><td rowspan="1" colspan="1" align="center">(5x10<sup>−4</sup>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs883079/</td><td rowspan="1" colspan="1"><em>TBX5</em></td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">EA (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871154">8</a>)</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.49 ± 0.08</td><td rowspan="1" colspan="1" align="center">8.3</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.60 ± 0.13</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.76</td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">0.75</td></tr><tr><td rowspan="1" colspan="1">3’ UTR</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(113,277,623)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.25)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−10</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.33)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−6</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs3825214/</td><td rowspan="1" colspan="1"><em>TBX5</em></td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">EA (<a href="#ddw284-B7" rid="ddw284-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871137">7</a>)</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">1.06 ± 0.15</td><td rowspan="1" colspan="1" align="center">ND</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.49 ± 0.14</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">0.71</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">0.65</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(113,279,826)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.22)</td><td rowspan="1" colspan="1" align="center">(<strong>3x10<sup>−13</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.25)</td><td rowspan="1" colspan="1" align="center">(<strong>3x10<sup>−4</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs7312625/</td><td rowspan="1" colspan="1"><em>TBX5</em></td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1">AA</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.46 ± 0.08</td><td rowspan="1" colspan="1" align="center">15.9</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.66 ± 0.13</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(113,284,357)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.28)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−9</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.29)</td><td rowspan="1" colspan="1" align="center">(<strong>7x10<sup>−7</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs7342028/</td><td rowspan="1" colspan="1"><em>VTI1A</em></td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">EA (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871105">8</a>)</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">0.48 ± 0.08</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">0.18 ± 0.12</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.81</td><td rowspan="1" colspan="1" align="center">1.0</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">0.46</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(114,469,252)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.27)</td><td rowspan="1" colspan="1" align="center">(<strong>5x10<sup>−10</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.54)</td><td rowspan="1" colspan="1" align="center">(0.12)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs7907361/</td><td rowspan="1" colspan="1"><em>VTI1A</em></td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1">AA</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">−0.47 ± 0.08</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.96</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">−0.39 ± 0.12</td><td rowspan="1" colspan="1" align="center">23.8</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(114,458,127)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.74)</td><td rowspan="1" colspan="1" align="center">(<strong>1x10<sup>−9</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.47)</td><td rowspan="1" colspan="1" align="center">(9x10<sup>−4</sup>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4687718/</td><td rowspan="1" colspan="1"><em>TKT</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">EA (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871148">8</a>)</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">−0.63 ± 0.11</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">−0.06 ± 0.12</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.98</td><td rowspan="1" colspan="1" align="center">ND</td><td rowspan="1" colspan="1" align="center">0.05</td><td rowspan="1" colspan="1" align="center">0.02</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(53,257,343)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.14)</td><td rowspan="1" colspan="1" align="center">(<strong>6x10<sup>−9</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.53)</td><td rowspan="1" colspan="1" align="center">(0.64)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs9311496/</td><td rowspan="1" colspan="1"><em>TKT</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">AA</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.46 ± 4.49</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.00</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">−0.48 ± 0.26</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td></tr><tr><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(53,347,118)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.001)</td><td rowspan="1" colspan="1" align="center">(0.92)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.06)</td><td rowspan="1" colspan="1" align="center">(0.06)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs17391905/</td><td rowspan="1" colspan="1"><em>CDKN2C</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">EA (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871118">8</a>)</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">−1.35 ± 0.23</td><td rowspan="1" colspan="1" align="center">4.0</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">−0.12 ± 0.26</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">ND</td></tr><tr><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(51,318,728)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.05)</td><td rowspan="1" colspan="1" align="center">(<strong>9x10<sup>−9</sup></strong>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.07)</td><td rowspan="1" colspan="1" align="center">(0.65)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs11205809/</td><td rowspan="1" colspan="1"><em>CDKN2C</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1">AA</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.11 ± 0.10</td><td rowspan="1" colspan="1" align="center">6.6</td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.42 ± 0.12</td><td rowspan="1" colspan="1" align="center">0.0</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td></tr><tr><td rowspan="1" colspan="1">intergenic</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(51,273,357)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.84)</td><td rowspan="1" colspan="1" align="center">(0.25)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">(0.48)</td><td rowspan="1" colspan="1" align="center">(8x10<sup>−4</sup>)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm139953983057024" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5291202/table/ddw284-T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw284-TF21"><p id="__p42" class="p p-first-last"><sup>a</sup>CA = coded effect allele. Freq = frequency.</p></div><div id="ddw284-TF22"><p id="__p43" class="p p-first-last"><sup>b</sup>Power to detect effect size reported in COGENT/CARe results using the effect allele frequency and trait variance as observed in European samples and keeping α = 0.002 adjusting for 22 SNPs.</p></div><div id="ddw284-TF23"><p id="__p44" class="p p-first-last"><sup>c</sup>Power to detect effect size reported in European discovery population using the effect allele frequency and trait variance as observed in African-American samples and keeping α = 0.002 adjusting for 30 SNPs.</p></div><div id="ddw284-TF24"><p id="__p45" class="p p-first-last"><sup>d</sup>LD between AA and EA index SNPs. ND indicates the SNP was not genotyped in the HapMap population or was monomorphic.</p></div><div id="ddw284-TF25"><p id="__p46" class="p p-first-last"><sup>e</sup>In non-African ancestry discovery rows, significance was based on the genome-wide discovery threshold and <em>P</em>-values ≤ 5.0x10 <sup>−</sup> <sup>8</sup> were bolded. In African ancestry discovery rows, significance was based on the replication threshold and <em>P</em>-values ≤ 0.002 were bolded. β and SE expressed in units of milliseconds.</p></div><div id="ddw284-TF26"><p id="__p47" class="p p-first-last"><sup>f</sup>Heterogeneity I<sup>2</sup> statistics bolded if the Cochran’s Χ<sup>2</sup>
<em>P</em>-value ≤ 0.05.</p></div><div id="ddw284-TF27"><p id="__p48" class="p p-first-last"><sup>g</sup>In non-African ancestry discovery rows, significance was based on replication of the 30 previously identified SNPs and <em>P</em>-values ≤ 0.002 were bolded. In African ancestry discovery rows, significance was based on discovery within the 22 loci and <em>P</em>-values ≤ 1.4×10 <sup>−</sup> <sup>5</sup> were bolded. β and SE expressed in units of milliseconds.</p></div><div id="ddw284-TF28"><p id="__p49" class="p p-first-last"><sup>h</sup>Position based on NCBI reference sequence build 36.</p></div></div></div></div><div id="__sec7" class="sec"><h3 id="__sec7title">SNP association transferability at <em>SCN5A/SCN10A</em> locus</h3><p id="__p50" class="p p-first">The most significant SNP associations reported from previous GWA studies of QRS interval among other ethnic groups are similarly located on chromosome 3 at the <em>SCN5A</em>/<em>SCN10A</em> locus, and multiple independent signals have been identified (<a href="/pmc/articles/PMC5291202/figure/ddw284-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw284-F1" rid-ob="ob-ddw284-F1" co-legend-rid="lgnd_ddw284-F1"><span>Fig. 1</span></a>) (<a href="#ddw284-B7" rid="ddw284-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871149">7</a>,<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871163">8</a>,<a href="#ddw284-B16" rid="ddw284-B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871109">16</a>). The coded allele frequencies for all <em>SCN5A/SCN10A</em> European-ancestry index SNPs were lower among African Americans than individuals of European ancestry, which along with the smaller sample size among the African Americans we examined, reduces power to replicate (<a href="/pmc/articles/PMC5291202/table/ddw284-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T3" rid-ob="ob-ddw284-T3" co-legend-rid=""><span>Table 3</span></a>). Of the 4 independent European descent QRS index SNPs with adequate power (≥80%) to replicate among African Americans, 2 were significant, including the top SNP-QRS association among individuals of European ancestry, rs6801957 (<em>P =</em> 8×10<sup>−4</sup> among African Americans, <a href="/pmc/articles/PMC5291202/table/ddw284-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T3" rid-ob="ob-ddw284-T3" co-legend-rid=""><span>Table 3</span></a>).</p><p id="__p51" class="p p-last">The absence of genome-wide significant QRS associations among SNPs within <em>SCN10A</em> in African Americans does not appear to be due to a difference in the MAF distribution of <em>SCN10A</em> SNPs between populations of European and African descent (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a>). Compared to populations of European descent (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S17</a>), LD was reduced and haplotypes were shorter in the <em>SCN10A</em> region in populations of African descent (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Materials, Figs S18 and S19</a>), which could impact the ability for assayed SNPs to tag potential non-genotyped causal variants.</p></div><div id="__sec8" class="sec sec-last"><h3 id="__sec8title">SNP association transferability at the remaining known QRS loci</h3><p id="__p52" class="p p-first">At the remaining 21 QRS loci (<a href="#ddw284-B7" rid="ddw284-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871161">7</a>,<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871160">8</a>), there was adequate power (≥80%) to replicate European-ancestry index SNPs at 8 loci, and index SNPs at 4 of these 8 loci replicated in African Americans at the multiple-testing corrected significance threshold of <em>P =</em> 0.002 (<em>CDKN1A</em>, <em>NFIA</em>, <em>HAND1</em> and <em>TBX5</em>, <a href="/pmc/articles/PMC5291202/table/ddw284-T5/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T5" rid-ob="ob-ddw284-T5" co-legend-rid=""><span>Table 5</span></a>, binomial test <em>P =</em> 4×10<sup>−4</sup>). Of the 13 loci where there was not adequate power to replicate, a European-ancestry index SNP at one locus replicated in African Americans (<em>SETBP1</em>, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S6</a>). At the more liberal replication significance threshold of 0.05, European-ancestry index SNPs at 5 of the 8 loci with adequate power replicated (binomial test <em>P =</em> 2×10<sup>−5</sup>), and index SNPs at 7 of the 13 loci without adequate power replicated (binomial test <em>P =</em> 1×10<sup>−6</sup>) (<a href="/pmc/articles/PMC5291202/table/ddw284-T5/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T5" rid-ob="ob-ddw284-T5" co-legend-rid=""><span>Table 5</span></a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S6</a>).</p><p id="__p53">We next expanded our characterization of each of the 22 European-ancestry QRS loci to identify the most significant SNP association with QRS duration among African Americans (African-American index SNPs) and to determine the LD between European and African-ancestry index SNPs. Other than <em>SCN5A</em>, two other loci (<em>TBX5</em> and <em>NFIA</em>) contained African-ancestry index SNPs that passed the significance threshold for discovery (<em>P</em> ≤ 1.4×10 <sup>−</sup> <sup>5</sup>) within the 22 QRS loci (<a href="/pmc/articles/PMC5291202/table/ddw284-T5/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T5" rid-ob="ob-ddw284-T5" co-legend-rid=""><span>Table 5</span></a>). The African-ancestry index SNP associations in <em>TBX5</em> (rs7312625) and <em>NFIA</em> (rs2207791) replicated at the genome-wide significance level in populations of European descent (<a href="/pmc/articles/PMC5291202/table/ddw284-T5/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T5" rid-ob="ob-ddw284-T5" co-legend-rid=""><span>Table 5</span></a>). For both loci, our results did not provide evidence for allelic heterogeneity, as the African-ancestry index SNPs were in moderate LD with the European-ancestry index SNPs (HapMap ASW <em>r</em><sup>2 </sup>≥<sup> </sup>0.65, HapMap CEU <em>r</em><sup>2 </sup>≥<sup> </sup>0.5) (<a href="/pmc/articles/PMC5291202/table/ddw284-T5/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T5" rid-ob="ob-ddw284-T5" co-legend-rid=""><span>Table 5</span></a>, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Materials, Fig. S20 and S21</a>).</p><p id="__p54" class="p p-last">Fine-mapping intervals based on Bayes factors and the resulting posterior probabilities generated from a transethnic meta-analysis with MANTRA (<a href="#ddw284-B17" rid="ddw284-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871074">17</a>) were constructed at the 22 QRS loci previously identified in populations of European descent and the two new QRS loci reported here. At each locus, the 95% credible set (CS) defines the genomic boundary that contains the smallest set of SNPs accounting for 95% of the posterior probability (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>). Other than the loci at which the 95% CS contained a single SNP, the largest percentage decrease from the genome-wide significant interval discovered in European populations to the 95% CS in the transethnic meta-analysis was at the <em>TBX5</em> locus (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S20</a>).</p></div></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title">Discussion</h2><p id="__p55" class="p p-first">Our genetic association study of cardiac ventricular conduction among African Americans identified variants in a putative transcriptional regulatory region within intron 16 of the cardiac sodium channel <em>SCN5A</em> that were associated with QRS duration and <em>SCN5A</em> RNA levels. Furthermore, two novel loci associated with cardiac ventricular depolarization and conduction were identified through a transethnic GWAS meta-analysis. Finally, our study demonstrated the transferability of QRS-SNP associations between populations of European and African-descent by both gene-set enrichment analysis as well as direct evaluation of top QRS-SNP associations.</p><p id="__p56">Our GWAS meta-analysis with additional fine-mapping identified two SNPs in high LD (rs3922844 and rs12635898, HapMap YRI: <em>r</em><sup>2 </sup>=<sup> </sup>0.93) in intron 16 of the cardiac sodium channel <em>SCN5A</em> gene associated with QRS duration among African Americans. <em>SCN5A</em> encodes the pore-forming α subunit of the cardiac voltage-gated sodium channel Na<sub>v</sub>1.5, and opening of the Na<sub>v</sub>1.5 channel drives rapid membrane depolarization during the cardiac action potential (<a href="#ddw284-B18" rid="ddw284-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871125">18</a>). Common and rare <em>SCN5A</em> genetic variants have been associated with cardiac depolarization, conduction, and repolarization (<a href="#ddw284-B7" rid="ddw284-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871157">7</a>,<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871132">8</a>,<a href="#ddw284-B19" rid="ddw284-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871067">19</a>). The most significantly associated SNP identified in this study (rs3922844) has been previously shown to associate with atrioventricular conduction (PR interval) among African Americans that included a subset of the same cohorts as this study (<a href="#ddw284-B20" rid="ddw284-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871115">20</a>,<a href="#ddw284-B21" rid="ddw284-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871152">21</a>), similar to other SNPs at this locus where variants that prolong PR interval also prolong QRS duration.</p><p id="__p57">Functional annotation indicated that rs3922844 and other intronic variants in LD overlapped with a putative intronic transcriptional regulatory region. While functional intronic enhancers have not been identified in <em>SCN5A</em>, transcriptional enhancers are commonly found within intronic regions (<a href="#ddw284-B22" rid="ddw284-B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871081">22</a>). Consistent with rs3922844’s overlap with putative transcriptional regulatory features, we found that rs3922844 was associated with <em>SCN5A</em> expression in human atrial tissue. The rs3922844 C allele associated with longer QRS duration was also associated with lower <em>SCN5A</em> RNA expression in human cardiac tissue, supporting the hypothesis that fewer available Na<sub>v</sub>1.5 channels would lead to subtly slower depolarization and conduction in cardiac tissue. It is intriguing that both genome-wide significant SNPs were located within intron 16 of <em>SCN5A</em>, which is immediately adjacent to an alternative splicing event that skips exons 17 and 18, resulting in the production of the non-functional Na<sub>v</sub>1.5b isoform that contains a partial deletion of the sixth transmembrane spanning segment of the DII domain and a deletion of a large segment of the intracellular DII-DIII linker (<a href="#ddw284-B18" rid="ddw284-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871170">18</a>). While the array-based expression data in our eQTL study did not enable our examination of specific transcript isoforms, future studies could investigate this potential molecular consequence that could lead to an increased production of non-functional transcripts that would further reduce the number of functional Na<sub>v</sub>1.5 voltage-gated sodium channels.</p><p id="__p58">Using a transethnic meta-analysis approach, two novel QRS loci were identified: one in the gene <em>MYL12A</em> on chromosome 18 (rs1662342) and one in an intergenic region on chromosome one (rs7547997) near a cluster of five <em>CD1</em> genes. For both SNPs, the higher allele frequency in African Americans combined with the 13,031 additional sample size increased power to find associations in the transethnic meta-analysis.</p><p id="__p59">While the association between intronic SNP rs1662342 and <em>MYL12A</em> gene expression did not pass multiple test correction, the nominal significance of the association provided suggestive evidence that rs1662342 could be associated with <em>MYL12A</em> gene expression. <em>MYL12A</em> encodes the myosin regulatory light chain that binds to a variety of myosin heavy chain IIs (MHC IIs) in multiple cell types (<a href="#ddw284-B23" rid="ddw284-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871062">23</a>). <em>MYL12A</em> is expressed in the heart and cardiac myocytes in humans (<a href="#ddw284-B24" rid="ddw284-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871185">24</a>). Knockdown of <em>MYL12A</em> in mouse fibroblasts resulted in a significant reduction in cellular contractility, a disruption of cellular structure and morphology, and a decrease in non-muscle MHC II expression (<a href="#ddw284-B23" rid="ddw284-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871096">23</a>). The most significant SNP associations with RR interval in populations of European descent were located in <em>MYH6</em>, the α-heavy chain subunit of cardiac myosin (<a href="#ddw284-B7" rid="ddw284-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871146">7</a>,<a href="#ddw284-B25" rid="ddw284-B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871130">25</a>), and the associations replicated in African Americans (<a href="#ddw284-B26" rid="ddw284-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871103">26</a>). Our results suggest that genetic variation in a different component of myosin, myosin regulatory light chain, may play a role in QRS duration.</p><p id="__p60">The candidate gene for the second novel QRS locus is not as obvious, as rs7547997 was not associated with gene expression of nearby genes in our eQTL study. SNP rs7547997 is located within an intergenic region on chromosome one near a cluster of five <em>CD1</em> genes, which mediate lipid and glycolipid antigen presentation to T cells (<a href="#ddw284-B27" rid="ddw284-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871116">27</a>) and near a cluster of 15 olfactory receptor genes (<a href="#ddw284-B28" rid="ddw284-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871110">28</a>). <em>SPTA1</em>, which encodes α-spectrin, is located 240 kb from rs7547997, but measures of <em>SPTA1</em> RNA did not pass quality filters in our eQTL study and rs7547997’s association with <em>SPTA1</em> gene expression could not be determined. Spectrin, a tetramer composed of α-β dimers, acts as an actin crosslinking and molecular scaffold protein that regulates cell shape and membrane protein localization (<a href="#ddw284-B29" rid="ddw284-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871153">29</a>). Spectrin binds ankyrin-G, and ankyrin-G is required for Na<sub>v</sub>1.5 membrane targeting and excitability (<a href="#ddw284-B29" rid="ddw284-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871102">29</a>). The E1053K <em>SCN5A</em> variant disrupts the ankyrin-G/Na<sub>v</sub>1.5 binding interaction and results in Brugada Syndrome (<a href="#ddw284-B30" rid="ddw284-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871097">30</a>). In mouse cardiomyocytes, β-spectrin co-localizes with ankyrin-G and Na<sub>v</sub>1.5 (31). β-spectrin also targets CaMKII to Na<sub>v</sub>1.5 where CaMKII regulates Na<sub>v</sub>1.5 by phosphorylation (<a href="#ddw284-B31" rid="ddw284-B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871159">31</a>). While <em>SPTA1</em> might be an attractive candidate gene, further studies are needed to identify causal gene(s) and variant(s) in the novel QRS association region identified on chromosome one.</p><p id="__p61">Our transethnic meta-analysis results provide evidence that a large proportion of SNP and loci associations with QRS duration are shared between populations of European and African descent. Gene-set enrichment analysis revealed that the 22 previously reported European-descent QRS-associated loci were enriched for significant QRS-SNP associations among African Americans compared with randomly selected gene sets matched for gene set characteristics such as gene size and LD properties, which provided evidence for replication at the gene-set level. Where the power for replication was adequate, half of the European-descent index SNPs were also associated with QRS duration among African Americans. Even among the SNPs where power was inadequate, the majority were at least nominally associated with QRS duration among African Americans. Furthermore, the direction of the association for all European-ancestry index SNPs was the same in both ethnic groups. While these results taken together provide evidence for the transferability of QRS genetic associations from one ethnic group to another, there are some exceptions. There are SNPs associated with QRS duration among those of European descent, for instance rs9851724 near <em>SCN10A</em> (<a href="/pmc/articles/PMC5291202/table/ddw284-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T3" rid-ob="ob-ddw284-T3" co-legend-rid=""><span>Table 3</span></a>) and rs4687718 in <em>TKT</em> (<a href="/pmc/articles/PMC5291202/table/ddw284-T5/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw284-T5" rid-ob="ob-ddw284-T5" co-legend-rid=""><span>Table 5</span></a>), where despite adequate power, no evidence for association was identified among African Americans.</p><p id="__p62">In addition to replication sample size and population-specific allele frequencies, SNP associations could fail to replicate across continental ancestry groups for several reasons, including: (1) population-specific causal variants, (2) population-specific LD between assayed SNPs and causal variants, and (3) interactions between SNPs and population-specific non-genetic factors (<a href="#ddw284-B9" rid="ddw284-B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871140">9</a>,<a href="#ddw284-B32" rid="ddw284-B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871098">32</a>). In a systematic survey of SNPs identified through GWA studies (GWAS SNPs), the allele frequency and LD with nearby SNPs differed significantly between population groups for a number of GWAS SNPs, suggesting that at least some GWAS SNPs identified in European populations might not generalize to other populations (<a href="#ddw284-B10" rid="ddw284-B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871056">10</a>). In addition, it has been posited that rare variants, which are more likely to be population-specific (<a href="#ddw284-B33" rid="ddw284-B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871090">33–36</a>), can create synthetic associations with common variants identified in GWA studies, which would result in the lack of transferability of findings across populations (<a href="#ddw284-B37" rid="ddw284-B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871126">37</a>). Others have argued that synthetic associations might exist, but they are unlikely to account for most GWAS results (<a href="#ddw284-B38" rid="ddw284-B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871071">38</a>,<a href="#ddw284-B39" rid="ddw284-B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871079">39</a>). While a direct test of the synthetic association hypothesis requires a comprehensive collection of rare and common variants, our GWAS in African Americans provides an opportunity to test a prediction from the synthetic association hypothesis that SNP associations would not generalize across populations.</p><p id="__p63">Well-powered studies for a limited number of traits and diseases have provided evidence that a majority of GWAS SNPs discovered in populations of European descent generalize to multiple populations (<a href="#ddw284-B40" rid="ddw284-B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871078">40–43</a>). For instance, a high proportion of SNP associations with blood lipids discovered in populations of European descent generalized to populations of non-European descent (<a href="#ddw284-B44" rid="ddw284-B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871088">44</a>,<a href="#ddw284-B45" rid="ddw284-B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871100">45</a>), but allelic heterogeneity was observed at some loci (<a href="#ddw284-B45" rid="ddw284-B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871164">45</a>,<a href="#ddw284-B46" rid="ddw284-B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871177">46</a>). A recent systematic examination of GWAS SNP replication across populations found that 45.8% of GWAS SNPs initially identified in populations of European descent replicated in East Asian populations, and the percentage increased to 76.5% when replication attempts were limited to those that achieved sufficient power (<a href="#ddw284-B47" rid="ddw284-B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871150">47</a>). The same study found that only 7 of 73 (9.6%) SNP associations replicated in populations of African descent, and the replication percentage only increased to 20% among the 25 replication attempts that achieved sufficient power (<a href="#ddw284-B47" rid="ddw284-B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871184">47</a>). In an analysis of five traits and diseases in the Population Architecture Using Genomics and Epidemiology (PAGE) study, a consortium of multi-ancestry population-based studies, a significant proportion of GWAS SNPs discovered in European populations replicated in populations of non-European descent (<a href="#ddw284-B43" rid="ddw284-B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871138">43</a>). However, the effect estimates in non-European populations tended to be closer to the null, especially in African Americans (<a href="#ddw284-B43" rid="ddw284-B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871058">43</a>). Our analysis of SNP associations with QRS duration is consistent with findings from other traits; namely, SNP associations generalize at most loci but not all. GWA studies of ECG traits other than QRS duration have been conducted in African Americans, and SNP association transferability has been found (<a href="#ddw284-B13" rid="ddw284-B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871073">13</a>,<a href="#ddw284-B20" rid="ddw284-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871175">20</a>,<a href="#ddw284-B21" rid="ddw284-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871061">21</a>,<a href="#ddw284-B26" rid="ddw284-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871054">26</a>,<a href="#ddw284-B48" rid="ddw284-B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871128">48</a>,<a href="#ddw284-B49" rid="ddw284-B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871176">49</a>). In two large meta-analyses, the proportion of SNP associations that replicated in African Americans was 7 of 13 for RR interval and 10 of 22 for QT interval at α = 0.05 (<a href="#ddw284-B49" rid="ddw284-B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871086">26</a>,<a href="#ddw284-B49" rid="ddw284-B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871084">48</a>). While Dickson <em>et al</em>. predicted that synthetic associations would be inconsistent across populations (<a href="#ddw284-B37" rid="ddw284-B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871093">37</a>), results from our study and others indicate that a majority of SNP associations do generalize across populations. Continuing to perform well-powered GWA studies in African Americans should reveal whether SNP associations for a variety of traits and conditions are transferable.</p><p id="__p64" class="p p-last">In conclusion, by conducting a GWAS meta-analysis of QRS duration in African Americans, we refined the <em>SCN5A</em> association region to a single intron, and the associated SNP rs3922844 was also associated with reduced <em>SCN5A</em> expression in human atrial tissue in individuals of African and European ancestry. Two novel genome-wide significant SNP associations in or near intriguing candidate genes (<em>MYL12A</em> and <em>SPTA1</em>) were identified using transethnic meta-analysis. In addition, the high proportion of QRS associations that were transferable between populations of European and African-descent indicated that, at many of the associated loci, common genetic variation shared across populations contributes to QRS duration.</p></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">Materials and Methods</h2><div id="__sec11" class="sec sec-first"><h3 id="__sec11title">Study samples and ECG recordings</h3><p id="__p65" class="p p-first-last">The following ten cohorts with African American participants contributed to this study (in order of decreasing sample size): the Women’s Health Initiative (WHI), the Atherosclerosis Risk in Communities (ARIC) study, the Jackson Heart Study (JHS), the Multi-Ethnic Study of Atherosclerosis (MESA), the Health, Aging, and Body Composition Study (Health ABC), the Healthy Aging in Neighborhoods of Diversity across the Life Span study (HANDLS), the Cardiovascular Health Study (CHS), the Cleveland Family Study (CFS), the Baltimore Longitudinal Study on Aging (BLSA) study, and the Bogalusa Heart Study (BHS). Detailed descriptions of the study samples and ECG recording methods are provided in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Text S1</a>. The European-descent participants and cohorts that contributed to this analysis have been previously described (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871162">8</a>). The study was approved by the Institutional Review Board at all participating institutions. All individuals included in this analysis provided written informed consent.</p></div><div id="__sec12" class="sec"><h3 id="__sec12title">Genotyping and genotype imputation</h3><p id="__p66" class="p p-first-last">Cohorts used Affymetrix or Illumina SNP genotyping arrays and applied quality control filters to samples and SNPs (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>). Participants unlikely to be of African descent based on principal component analysis were excluded from the analysis (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Text S1</a>). Genotype imputation was performed using MACH or Beagle software. Individual studies performed genotype imputation using reference haplotypes that consisted of either a mixture of phased haplotype data from HapMap 2 YRI and CEU in a 1:1 ratio or a combination of HapMap 2 YRI and CEU in a 1:1 ratio and HapMap 3 YRI, CEU, and ASW in a 1:1:1 ratio (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>). Detailed descriptions of genotyping methods, quality control steps, and imputation methods can be found in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Materials, Text S1 and Table S1</a>.</p></div><div id="__sec13" class="sec"><h3 id="__sec13title">Genome-wide association analysis among African Americans</h3><p id="__p67" class="p p-first">Study participants were excluded from analysis based on the following criteria: missing covariates, younger than 18 years of age, atrial fibrillation on the ECG, history of heart failure or myocardial infarction, QRS duration ≥ 120 ms, Wolff-Parkinson-White pattern, pacemaker or defibrillator implant, or use of class I and III antiarrhythmic medications. Genetic association analysis was performed in each cohort using linear regression models with the following covariates: age, sex, study site (if multiple sites were present), BMI, height, and principal components derived from principal component analysis (PCA) of genotype data (<a href="#ddw284-B50" rid="ddw284-B50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871122">50</a>) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Text S1</a>). The exclusion criteria and covariate adjustment we applied were the same as those applied in a previous GWAS of QRS duration in populations of European descent (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871127">8</a>), which facilitated comparisons of results between the two studies. In addition to adjustment for global genetic ancestry estimates, the genome-wide significant SNP association was additionally adjusted for local genetic ancestry estimates (see below for local ancestry estimation details). GWA studies in ARIC, JHS, MESA, and CFS were performed as part of the Candidate gene Association Resource (CARe) using QRS standardized residuals adjusted for the covariates mentioned above (<a href="#ddw284-B51" rid="ddw284-B51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871155">51</a>,<a href="#ddw284-B52" rid="ddw284-B52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871139">52</a>). All cohorts except for CFS performed a GWAS using PLINK, R, MACH2QTL, or Merlin (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Materials, Table S1 Table and Text S1</a>). The family-based CFS study performed a GWAS using linear mixed models to account for relatedness (<a href="#ddw284-B52" rid="ddw284-B52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871055">52</a>,<a href="#ddw284-B53" rid="ddw284-B53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871151">53</a>). A subcohort of JHS was family-based, but previous analyses determined that the use of methods accounting for family structure had minimal influence on effect estimates, so linear regression was used as in previous studies (<a href="#ddw284-B52" rid="ddw284-B52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871069">52</a>). Imputed allele dosages were modelled with an additive mode of inheritance. When available, results from directly genotyped SNP data were used in preference over those from imputed SNP data. Detailed descriptions of cohort-specific GWAS analytic methods can be found in S1 Text.</p><p id="__p68">Prior to meta-analysis, GWAS results from each cohort were filtered to remove SNPs with minor allele frequency (MAF) &lt; 0.01 or imputation quality scores &lt; 0.3. Effect estimates and their standard errors estimated from cohorts that used QRS standardized residuals as the phenotype (ARIC, JHS, MESA, and CFS) were transformed back to units of milliseconds by multiplying by the study-specific standard deviation of the residuals. Cohort-specific GWAS results were combined using fixed effect meta-analysis with inverse variance weights as implemented in METAL (<a href="#ddw284-B54" rid="ddw284-B54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871167">54</a>). SNPs that were non-autosomal or were only present in a single study were excluded from analysis. Genomic control was applied to the results from each cohort prior to meta-analysis and to the results of the meta-analysis (double GC-correction) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>) (<a href="#ddw284-B55" rid="ddw284-B55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871178">55</a>). To maintain an experiment-wide type I error rate of 0.05, a genome-wide significance threshold of 2.5×10<sup>−8</sup> was pre-specified based on a Bonferroni correction for the 2 million independent common variants estimated to exist in the genomes of individuals of African ancestry (<a href="#ddw284-B56" rid="ddw284-B56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871156">56</a>). Heterogeneity across samples was assessed using Cochran’s Χ<sup>2</sup> test of heterogeneity with 9 degrees of freedom (<a href="#ddw284-B57" rid="ddw284-B57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871166">57</a>) and the I<sup>2</sup> statistic (<a href="#ddw284-B58" rid="ddw284-B58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871072">58</a>). A transethnic GWAS meta-analysis of QRS duration was performed using fixed-effect inverse variance weighted meta-analysis to combine the double GC-corrected African American GWAS meta-analysis results with the double GC-corrected European ancestry CHARGE GWAS meta-analysis results (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871099">8</a>).</p><p id="__p69">Global and local genetic ancestry was estimated in ARIC, JHS, MESA, and CFS cohorts using ANCESTRYMAP (<a href="#ddw284-B59" rid="ddw284-B59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871092">59</a>) and HAPMIX (<a href="#ddw284-B60" rid="ddw284-B60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871057">60</a>) as previously described (<a href="#ddw284-B52" rid="ddw284-B52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871172">52</a>). For the WHI cohort, global ancestry was estimated using Frappe (<a href="#ddw284-B61" rid="ddw284-B61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871089">61</a>) and local ancestry was estimated using SABRE (<a href="#ddw284-B62" rid="ddw284-B62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871181">62</a>). For the HANDLS and Health ABC cohorts, STRUCTURE (<a href="#ddw284-B63" rid="ddw284-B63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871114">63</a>) was used to estimate global ancestry, as previously described for the Health ABC cohort (<a href="#ddw284-B64" rid="ddw284-B64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871142">64</a>), and LAMP (<a href="#ddw284-B65" rid="ddw284-B65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871133">65</a>) was used to estimate local ancestry. Additional details of global and local ancestry estimation can be found in S1 Text. Linear regression was used to estimate the association between global genetic ancestry estimates and QRS duration.</p><p id="__p70">Variant annotation was performed using HaploReg (<a href="#ddw284-B66" rid="ddw284-B66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871168">66</a>), which leveraged data from the 1000 Genomes Project (<a href="#ddw284-B36" rid="ddw284-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871065">36</a>), ENCODE (<a href="#ddw284-B67" rid="ddw284-B67" class=" bibr popnode">67</a>), and the Roadmap Epigenomics Mapping Consortium (<a href="#ddw284-B68" rid="ddw284-B68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871145">68</a>).</p><p id="__p71" class="p p-last">Genome-wide SNP results from the GWAS in African Americans and the transethnic GWAS will be made available through the dbGaP CHARGE summary results site under dbGaP accession phs000930.</p></div><div id="__sec14" class="sec"><h3 id="__sec14title">Gene set enrichment analysis</h3><p id="__p72" class="p p-first">A gene set enrichment analysis (GSEA) was performed using MAGENTA version 2.4 (<a href="#ddw284-B69" rid="ddw284-B69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871111">69</a>) on the QC-filtered double GC-corrected African American GWAS meta-analysis results. Briefly, MAGENTA assigns a gene score based on the most significant SNP association in the gene region, corrects for potential gene score confounders (e.g., gene size and number of independent SNPs), and determines the proportion of gene scores in the gene set above a specified percentile cut-off (i.e., the leading edge fraction). The construction of gene scores from the minimum QRS-SNP association <em>P</em>-value for each gene region enabled a gene score to be significant in the presence of allelic heterogeneity, i.e., different SNPs that are the most significantly associated with QRS duration at each locus in populations of European and African descent. The significance of the gene set’s leading edge fraction was determined using an empirical null distribution of 10,000 leading edge fractions from randomly sampled gene sets matched to the user-defined gene set by gene score confounder characteristics. To enable MAGENTA to calculate the number of SNPs per gene for populations of African descent, we generated a genome-wide list of SNP positions after SNPs in high LD (pairwise genotypic correlation <em>r</em><sup>2 </sup>≥<sup> </sup>0.8) were removed by applying PLINK’s LD pruning function to HapMap phase 2 YRI genotype data using sliding windows of 50 SNPs moving by 5 SNP increments (<a href="#ddw284-B70" rid="ddw284-B70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871104">70</a>).</p><p id="__p73" class="p p-last">The candidate gene set was created using the gene closest to the index SNP from the GWAS of QRS duration previously conducted by the CHARGE consortium (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871141">8</a>), the largest GWAS of QRS duration, and included: <em>SCN5A</em>, <em>SCN10A</em>, <em>CDKN1A</em>, <em>PLN</em>, <em>NFIA</em>, <em>HAND1</em>, <em>TBX20</em>, <em>SIPA1L1</em>, <em>TBX5</em>, <em>TBX3</em>, <em>VTI1A</em>, <em>SETBP1</em>, <em>STRN</em>, <em>TKT</em>, <em>CRIM1</em>, <em>CDKN2C</em>, <em>PRKCA</em>, <em>IGFBP3</em>, <em>CASQ2</em>, <em>KLF12</em>, <em>LRIG1</em>, <em>DKK1</em>, and <em>GOSR2</em>. <em>SCN5A</em> and <em>SCN10A</em> were individually included in our gene set because the previously published CHARGE GWAS of QRS duration determined that SNP associations in these adjacent genes were independent (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871135">8</a>).</p></div><div id="__sec15" class="sec"><h3 id="__sec15title">Locus-specific replication and regional association analysis</h3><p id="__p74" class="p p-first">The 30 European ancestry index SNPs (EA index SNPs) from the 22 previously reported EA QRS loci that were examined for replication in African Americans were 3 <em>SCN10A</em> SNPs, 3 <em>SCN5A</em> SNPs, 1 <em>EXOG</em> SNP, 2 <em>CDKN1A</em> SNPs, 2 <em>TBX5</em> SNPs, and a single SNP from each of the other 19 previously reported loci (<a href="#ddw284-B7" rid="ddw284-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871165">7</a>,<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871095">8</a>,<a href="#ddw284-B16" rid="ddw284-B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871123">16</a>). To account for the 30 examined SNPs, a significance threshold of 0.002 was adopted. The following parameters were used in power calculations for the 30 SNP replication analysis: α = 0.002, African American sample size = 13,031, African American QRS duration mean ± SD = 89.23 ± 9.70 ms (weighted average and pooled SD across the 10 COGENT/CARe cohorts), the previously reported SNP effect size, and the SNP coded allele frequency in populations of African descent (weighted average across the 10 COGENT/CARe cohorts).</p><p id="__p75">In the analysis of the genomic region surrounding each of the European-ancestry index SNPs, each region was defined as the genomic interval encompassing all SNPs with QRS association <em>P</em>-values ≤ 5×10<sup>−8</sup> from the largest GWAS meta-analysis conducted in individuals of European ancestry (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871179">8</a>), and then extending the genomic interval boundaries by 100 kb in both directions. To set the significance threshold, the number of independent SNPs in these 22 regions was determined from HapMap phase 2 YRI SNP genotype data using the same LD pruning procedure that was described for our GSEA method. For the 22 previously reported QRS loci, 3,526 independent SNPs were identified and a Bonferroni-based significance threshold of 1.4×10<sup>−5</sup> was adopted.</p><p id="__p76">At each of the 22 previously identified QRS loci, the most significant SNP association detected in African Americans (AA index SNP) was also examined in GWAS results from populations of European descent (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871124">8</a>). Power to replicate African-ancestry index SNP associations in populations of European descent was calculated using the following parameters: α = 0.002, sample size of individuals with European ancestry = 40,407, QRS duration mean ± SD from individuals of European ancestry = 88.32 ± 10.13 ms (estimated from 2,845 CHS subjects of European ancestry after application of exclusion factors described above), African-ancestry index SNP effect size, and the previously reported SNP coded allele frequency in populations of European descent (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871120">8</a>). All power calculations were performed using QUANTO (<a href="#ddw284-B71" rid="ddw284-B71" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871066">71</a>).</p><p id="__p77" class="p p-last">Regional association plots were created using LocusZoom (<a href="#ddw284-B72" rid="ddw284-B72" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871134">72</a>), customized to separately plot recombination rates estimated from African Americans (<a href="#ddw284-B73" rid="ddw284-B73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871076">73</a>) and HapMap CEU individuals. For <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Materials, Figs S17–S19</a>, LD plots were created using Haploview and LD blocks were estimated using 95% confidence bounds on D prime (<a href="#ddw284-B74" rid="ddw284-B74" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871091">74</a>,<a href="#ddw284-B75" rid="ddw284-B75" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871171">75</a>).</p></div><div id="__sec16" class="sec"><h3 id="__sec16title">95% Credible set construction</h3><p id="__p78" class="p p-first">Regions used to construct credible sets (CS) at the 22 QRS loci discovered in populations of European descent were defined as genomic intervals encompassing all SNPs with QRS association <em>P</em>-values ≤ 5×10 <sup>−</sup> <sup>8</sup> reported by Sotoodehnia <em>et al</em>. (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871059">8</a>), and then extending those genomic intervals by 100 kb in both directions. At the two new QRS loci reported here, intervals 100 kb upstream and downstream of the transethnic index SNP were used as regions for CS construction. Transethnic meta-analysis of fixed-effect meta-analysis results from populations of European descent (<a href="#ddw284-B8" rid="ddw284-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871113">8</a>) and African Americans was performed using MANTRA (<a href="#ddw284-B17" rid="ddw284-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871174">17</a>). As previously described (<a href="#ddw284-B76" rid="ddw284-B76" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871107">76</a>) and in the context of a transethnic meta-analysis (<a href="#ddw284-B77" rid="ddw284-B77" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871136">77</a>), for <em>n</em> SNPs in a region, a Bayes factor (BF<sub><em>i</em></sub>) was estimated for each SNP<sub><em>i</em></sub> using MANTRA, and the posterior probability for SNP<sub><em>i</em></sub> is equal to
</p><div class="disp-formula" id="E1"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="EQ1" overflow="scroll" role="math" style="width: 5.516em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.545em; height: 0px; font-size: 121%;"><span style="position: absolute; clip: rect(-0.533em, 1004.55em, 2.188em, -1000em); top: -0.981em; left: 0em;"><span class="mrow" id="MathJax-Span-2"><span class="mfrac" id="MathJax-Span-3"><span style="display: inline-block; position: relative; width: 4.287em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.139em, 1001.72em, 4.29em, -1000em); top: -4.654em; left: 50%; margin-left: -0.86em;"><span class="mrow" id="MathJax-Span-4"><span class="msub" id="MathJax-Span-5"><span style="display: inline-block; position: relative; width: 1.721em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1001.51em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mrow" id="MathJax-Span-6"><span class="mi" id="MathJax-Span-7" style="font-family: MathJax_Math; font-style: italic;">B</span><span class="mi" id="MathJax-Span-8" style="font-family: MathJax_Math; font-style: italic;">F<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.106em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 1.402em;"><span class="mrow" id="MathJax-Span-9"><span class="mi" id="MathJax-Span-10" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.033em, 1004.17em, 4.425em, -1000em); top: -3.219em; left: 50%; margin-left: -2.083em;"><span class="mrow" id="MathJax-Span-11"><span class="mrow" id="MathJax-Span-12"><span class="msubsup" id="MathJax-Span-13"><span style="display: inline-block; position: relative; width: 2.279em; height: 0px;"><span style="position: absolute; clip: rect(3.072em, 1001em, 4.382em, -1000em); top: -3.977em; left: 0em;"><span class="mo" id="MathJax-Span-14" style="font-family: MathJax_Size1; vertical-align: 0em;">∑</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.51em, 1000.5em, 4.14em, -1000em); top: -4.454em; left: 1.056em;"><span class="mrow" id="MathJax-Span-15"><span class="mi" id="MathJax-Span-16" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">n</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.351em, 1001.22em, 4.14em, -1000em); top: -3.692em; left: 1.056em;"><span class="mrow" id="MathJax-Span-17"><span class="mi" id="MathJax-Span-18" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-19" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-20" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mrow" id="MathJax-Span-21" style="padding-left: 0.167em;"><span class="msub" id="MathJax-Span-22"><span style="display: inline-block; position: relative; width: 1.721em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1001.51em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mrow" id="MathJax-Span-23"><span class="mi" id="MathJax-Span-24" style="font-family: MathJax_Math; font-style: italic;">B</span><span class="mi" id="MathJax-Span-25" style="font-family: MathJax_Math; font-style: italic;">F<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.106em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 1.402em;"><span class="mrow" id="MathJax-Span-26"><span class="mi" id="MathJax-Span-27" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.865em, 1004.29em, 1.24em, -1000em); top: -1.305em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 4.287em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.981em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.335em; border-left: 0px solid; width: 0px; height: 3.043em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="EQ1" overflow="scroll"><mfrac><mrow><msub><mrow><mi>B</mi><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mrow><msubsup><mo stretchy="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mrow><msub><mrow><mi>B</mi><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></mrow></mrow></mfrac></math></script></div></div><p>
</p><p id="__p79" class="p p-last">SNPs with high posterior probabilities are more likely to be associated with a trait than SNPs with low posterior probabilities. SNPs at each region were ranked by their posterior probabilities in decreasing order, and the 95% CS consisted of the smallest set of ranked SNPs for which the cumulative sum of their posterior probabilities reached 0.95. The genomic interval of a 95% CS was the range of the positions of the SNPs in the CS.</p></div><div id="__sec17" class="sec"><h3 id="__sec17title">Metabochip analysis of <em>SCN5A-SCN10A</em> gene region</h3><p id="__p80" class="p p-first-last">To more comprehensively evaluate genetic variation in the <em>SCN5A</em>-<em>SCN10A</em> genomic region, we examined SNP associations using the MetaboChip, a high-density custom Illumina iSelect array that includes SNPs from the 1000 Genomes Project (<a href="#ddw284-B11" rid="ddw284-B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871064">11</a>,<a href="#ddw284-B78" rid="ddw284-B78" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871182">78</a>). In the <em>SCN5A</em>-<em>SCN10A</em> region (NCBI build 36 positions 38,490,026 - 38,818,967), 654 MetaboChip SNPs were directly genotyped and passed QC filters (SNP and sample call rate &gt; 90%, concordance among blind duplicates &gt; 98%, HWE <em>P</em>-value &gt; 0.001, MAF &gt; 0.01) in ARIC and WHI-PAGE participants. MetaboChip SNPs were imputed in WHI-SHARE participants using genome-wide genotype data (Affymetrix 6.0 SNP array), as previously described (<a href="#ddw284-B79" rid="ddw284-B79" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871068">79</a>).</p></div><div id="__sec18" class="sec sec-last"><h3 id="__sec18title">Gene expression analysis in human atrial tissue</h3><p id="__p81" class="p p-first-last">Human left atrial appendage and pulmonary vein trimming tissues were obtained with written informed consent from 289 European-ancestry and 40 African-ancestry patients undergoing cardiac surgery. Use of discarded surgical tissue was approved by the Institutional Review Board of the Cleveland Clinic. Total RNA was extracted using TRIzol. Genome-wide RNA levels were measured using Illumina HT12 v.3 and v.4 expression arrays. RNA expression levels were background corrected, log<sub>2</sub>-transformed, quantile normalized, and batch adjusted. Genome-wide SNPs were genotyped in these subjects using Illumina Hap550 and Hap610 arrays, and multidimensional scaling (MDS) was performed. SNP association with RNA levels was determined separately for each racial group using linear regression with SNPs coded as dosages and additive adjustment for sex, tissue location, MDS dimensions, and surrogate variables, which were included to reduce expression heterogeneity and improve power to detect eQTLs (<a href="#ddw284-B80" rid="ddw284-B80" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587871075">80</a>). Effect estimates were expressed on the log<sub>2</sub>-transformed RNA scale. Analysis was performed using expression probes detected in at least 10% of samples within 250 kb of the query SNP. Nine eQTLs were examined: rs3922844 and <em>SCN5A</em>, rs1662342 and probes in <em>MYL12A</em>, <em>TGIF1</em> (2 probes), <em>LPIN2</em> (2 probes), and <em>MYOM1</em>, and rs7547997 and probes in <em>CD1C</em> and <em>CD1E</em>, and an eQTL <em>P</em>-value &lt; 0.006 (0.05/9) was deemed significant. <em>SCN5A</em> exonic locations are expressed relative to the longest isoform, <a id="__tag_587871278" class="tag_hotlink" href="/nuccore/NM_198056.2" ref="/nuccore/NM_198056.2"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_198056.2","term_id":"124518659","term_text":"NM_198056.2"}}</span>NM_198056.2</a>, which contains 28 exons.</p></div></div><div id="__sec19" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec19title">Supplementary Material</h2><!--/article/body/sec/--><p id="__p82" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddw284/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a> is available at <em>HMG</em> online.</p></div><div id="idm139953975919312" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139953975919312title">Acknowledgements</h2><div class="sec"><p id="__p83">The authors wish to acknowledge the contributions of the involved research institutions, study investigators, field staff, and study participants of WHI, ARIC, JHS, MESA, Health ABC, HANDLS, CHS, CFS, BLSA, and BHS.</p><p id="__p84">The authors would also like to acknowledge the generous sharing of QRS GWAS results from the CHARGE consortium. The following individuals are members of the CHARGE QRS Consortium: Nona Sotoodehnia, Aaron Isaacs, Paul I.W. de Bakker, Marcus Dörr, Christopher Newton-Cheh, Ilja M. Nolte, Pim van der Harst, Martina Müller, Mark Eijgelsheim, Alvaro Alonso, Andrew A. Hicks, Sandosh Padmanabhan, Caroline Hayward, Albert Vernon Smith, Ozren Polasek, Steven Giovannone, Jingyuan Fu, Jared W. Magnani, Kristin D. Marciante, Arne Pfeufer, Sina A. Gharib, Alexander Teumer, Man Li, Joshua C. Bis, Fernando Rivadeneira, Thor Aspelund, Anna Köttgen, Toby Johnson, Kenneth Rice, Mark P.S. Sie, Amanda Ying Wang, Norman Klopp, Christian Fuchsberger, Sarah H. Wild, Irene Mateo Leach, Karol Estrada, Uwe Völker, Alan F. Wright, Folkert W. Asselbergs, Jiaxiang Qu, Aravinda Chakravarti, Moritz F. Sinner, Jan A. Kors, Astrid Petersmann, Tamara B. Harris, Elsayed Z. Soliman, Patricia B. Munroe, Bruce M. Psaty, Ben A. Oostra, L. Adrienne Cupples, Siegfried Perz, Rudolf A. de Boer, André G. Uitterlinden, Henry Völzke, Timothy D. Spector, Fang-Yu Liu, Eric Boerwinkle, Anna F. Dominiczak, Jerome I. Rotter, Gé van Herpen, Daniel Levy, H.-Erich Wichmann, Wiek H. van Gilst, Jacqueline C.M. Witteman, Heyo K. Kroemer, W.H. Linda Kao, Susan R. Heckbert, Thomas Meitinger, Albert Hofman, Harry Campbell, Aaron R. Folsom, Dirk J. van Veldhuisen, Christine Schwienbacher, Christopher J. O’Donnell, Claudia Beu Volpato, Mark J. Caulfield, John M. Connell, Lenore Launer, Xiaowen Lu, Lude Franke, Rudolf S.N. Fehrmann, Gerard te Meerman, Harry J.M. Groen, Rinse K. Weersma, Leonard H. van den Berg, Cisca Wijmenga, Roel A. Ophoff, Gerjan Navis, Igor Rudan, Harold Snieder, James F. Wilson, Peter P. Pramstaller, David S. Siscovick, Thomas J. Wang, Vilmundur Gudnason, Cornelia M. van Duijn, Stephan B. Felix, Glenn I. Fishman, Yalda Jamshidi, Bruno H Ch Stricker, Nilesh J. Samani, Stefan Kääb, Dan E. Arking.</p><p id="__p85"><em>Conflict of Interest statement</em>. Bruce M. Psaty serves on the DSMB of a clinical trial funded by the manufacturer (Zoll LifeCor) and on the Steering Committee of the Yale Open Data Access Project funded by Johnson &amp; Johnson. Anne M. Butler has received investigator-initiated support from Amgen and AstraZeneca for unrelated work. </p><div id="__sec20" class="sec"><h3 id="__sec20title">Funding </h3><p id="__p86" class="p p-first">This work was supported by the National Institutes of Health [<span>R01 HL088456 to N.S., R01 HL116747 to N.S., R01 HL111089 to N.S., R01 HL091244 to N.S., K99 HL098458 to C.L.A., 5T32CA009330-30 to A.M.B., R01 ES017794 to E.A.W., P20MD006899 to S.G.B., and U24AG051129 to D.S.E.</span>], the Laughlin Family [to<span> N.S.</span>], and the German Research Foundation [<span>SCHNA 1149/3-1 to R.B.S.</span>].</p><p id="__p87">As part of the National Heart, Lung, and Blood Institute (NHLBI)-sponsored Candidate gene Association Resource (CARe) project, the ARIC, JHS, MESA, and CFS studies contributed parent study data, ancillary study data, and DNA samples through the Broad Institute of Harvard and MIT (<span>N01-HC-65226</span>) to create a genotype/phenotype database for wide dissemination to the biomedical research community.</p><p id="__p88">Atherosclerosis Risk in Communities (ARIC): The ARIC study is carried out as a collaborative study supported by National Heart, Lung, and Blood Institute contracts (<span>HHSN268201100005C, HHSN268201100006C, HHSN268201100007C, HHSN268201100008C, HHSN268201100009C, HHSN268201100010C, HHSN268201100011C, and HHSN268201100012C</span>;), <span>R01HL087641, R01HL59367 and R01HL086694;</span>
National Human Genome Research Institute contract
<span>U01HG004402</span>; and National Institutes of Health contract
<span>HHSN268200625226C</span>. The authors thank the staff and participants of the ARIC study for their important contributions. Infrastructure was partly supported by Grant Number <span>UL1RR025005</span>, a component of the National Institutes of Health and NIH Roadmap for Medical Research.</p><p id="__p89">Cleveland Family Study (CFS): Case Western Reserve University (<span>NIH HL 46380, M01RR00080</span>).</p><p id="__p90">Jackson Heart Study (JHS): The Jackson Heart Study is supported by contracts <span>HHSN268201300046C, HHSN268201300047C, HHSN268201300048C, HHSN268201300049C, HHSN268201300050C</span> from the National Heart, Lung, and Blood Institute and the National Institute on Minority Health and Health Disparities.</p><p id="__p91">Multi-Ethnic Study of Atherosclerosis (MESA): University of Washington (<span>N01-HC-95159</span>), Regents of the University of California (<span>N01-HC-95160</span>), Columbia University (<span>N01-HC-95161</span>), Johns Hopkins University (<span>N01-HC-95162, N01-HC-95168</span>), University of Minnesota (<span>N01-HC95163</span>), Northwestern University (<span>N01-HC-95164</span>), Wake Forest University (<span>N01-HC-95165</span>), University of Vermont (<span>N01-HC-95166</span>), New England Medical Center (<span>N01-HC-95167</span>), Harbor-UCLA Research and Education Institute (<span>N01-HC-95169</span>), Cedars-Sinai Medical Center (<span>R01-HL-071205</span>), and University of Virginia (<span>subcontract to R01-HL-071205</span>).</p><p id="__p92">Health, Aging, and Body Composition Study (Health ABC): The Health ABC study was supported by NIA contracts <span>N01AG62101, N01AG62103, and N01AG62106</span>. The genome-wide association study was funded by NIA grant <span>1R01AG032098-01A1</span> to Wake Forest University Health Sciences and genotyping services were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, contract number <span>HHSN268200782096C</span>. This research was supported in part by the Intramural Research Program of the NIH, National Institute on Aging.</p><p id="__p93">Healthy Aging in Neighbourhoods of Diversity across the Life Span Study (HANDLS): The HANDLS study was in part supported by the intramural research program of the National Institute on Aging and the National Center for Minority Health and Health Disparities, National Institutes of Health. This research was supported by the Intramural Research Program of the NIH, National Institute on Aging and the National Center on Minority Health and Health Disparities (contract <span># Z01-AG000513</span>; and human subjects protocol <span># 2009-149</span>). Data analyses for the HANDLS study utilized the high-performance computational capabilities of the Biowulf Linux cluster at the National Institutes of Health, Bethesda, Md. (<a href="http://biowulf.nih.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://biowulf.nih.gov</a>).</p><p id="__p94">Women’s Health Initiative (WHI): The WHI program is funded by the National Heart, Lung, and Blood Institute, National Institutes of Health, U.S. Department of Health and Human Services through contracts HHSN268201100046C, HHSN268201100001C, HHSN268201100002C, HHSN268201100003C, HHSN268201100004C, and HHSN271201100004C. This manuscript was prepared in collaboration with investigators of the WHI, and has been reviewed and/or approved by the Women’s Health Initiative (WHI). WHI investigators are listed at <a href="https://www.whi.org/researchers/Documents%20%20Write%20a%20Paper/WHI%20Investigator%20Short%20List.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.whi.org/researchers/Documents%20%20Write%20a%20Paper/WHI%20Investigator%20Short%20List.pdf</a>. Funding for WHI SHARe genotyping was provided by NHLBI Contract N02-HL-64278.</p><p id="__p95">Cardiovascular Health Study (CHS): This CHS research was supported by NHLBI contracts <span>HHSN268201200036C, HHSN268200800007C, N01HC55222, N01HC85079, N01HC85080, N01HC85081, N01HC85082, N01HC85083, N01HC85086, HHSN268200960009C</span>; and NHLBI grants <span>HL080295, HL087652, HL105756, HL103612, HL120393, and HL130114, HL085251</span> with additional contribution from the National Institute of Neurological Disorders and Stroke (NINDS). Additional support was provided through <span>AG023629</span> from the National Institute on Aging (NIA). A full list of principal CHS investigators and institutions can be found at CHS-NHLBI.org/. The provision of genotyping data was supported in part by the National Center for Advancing Translational Sciences, CTSI grant <span>UL1TR000124</span>, and the National Institute of Diabetes and Digestive and Kidney Disease Diabetes Research Center (DRC) grant <span>DK063491</span> to the Southern California Diabetes Endocrinology Research Center. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p><p id="__p96">Baltimore Longitudinal Study of Aging (BLSA): The BLSA was supported in part by the Intramural Research Program of the NIH, National Institute on Aging. A portion of that support was through a R&amp;D contract with MedStar Research Institute.</p><p id="__p97">Bogalusa Heart Study (BHS): BHS is supported by grants <span>R01ES021724</span> from National Institute of Environmental Health Science and <span>R01AG016592</span> from the National Institute on Aging. Analysis performed at STSI/TSRI was supported by <span>U54 NS056883</span> and Scripps Genomic Medicine.</p><p id="__p98">The Population Architecture Using Genomics and Epidemiology (PAGE) program is funded by the National Human Genome Research Institute (NHGRI), supported by <span>U01HG007416 (CALiCo), U01HG007417 (ISMMS), U01HG007397 (MEC), U01HG007376 (WHI), and U01HG007419 (Coordinating Center)</span>. The contents of this paper are solely the responsibility of the authors and do not necessarily represent the official views of the NIH. The complete list of PAGE members can be found at <a href="http://www.pagestudy.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.pagestudy.org</a>. Assistance with data management, data integration, data dissemination, genotype imputation, ancestry deconvolution, and general study coordination was provided by the PAGE Coordinating Center (<span>U01HG007419</span>). The PAGE consortium thanks the staff and participants of all PAGE studies for their important contributions.</p><p id="__p99">Funding for the atrial tissue eQTL study was provided by NIH
<span>5R01HL090620</span>, NIH
<span>5R01HL111314</span>, Fondation Leducq CVD-07-03, European North American Atrial Fibrillation Research Alliance.</p><p id="__p100" class="p p-last">The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div></div></div><div id="idm139953975863680" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139953975863680title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="ddw284-B1">1. <span class="mixed-citation">Badheka A.O., Singh V., Patel N.J., Deshmukh A., Shah N., Chothani A., Mehta K., Grover P., Savani G.T., Gupta S., et al. (2013) 
<span class="ref-title">QRS duration on electrocardiography and cardiovascular mortality (from the National Health and Nutrition Examination Survey-III)</span>. <span class="ref-journal">Am. J. Cardiol</span>, <span class="ref-vol">112</span>, 671–677. [<a href="/pubmed/23726176" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol&amp;title=QRS+duration+on+electrocardiography+and+cardiovascular+mortality+(from+the+National+Health+and+Nutrition+Examination+Survey-III)&amp;author=A.O.+Badheka&amp;author=V.+Singh&amp;author=N.J.+Patel&amp;author=A.+Deshmukh&amp;author=N.+Shah&amp;volume=112&amp;publication_year=2013&amp;pages=671-677&amp;pmid=23726176&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B2">2. <span class="mixed-citation">Mentz R.J., Greiner M.A., DeVore A.D., Dunlay S.M., Choudhary G., Ahmad T., Khazanie P., Randolph T.C., Griswold M.E., Eapen Z.J., et al. (2015) 
<span class="ref-title">Ventricular conduction and long-term heart failure outcomes and mortality in African Americans: insights from the Jackson Heart Study</span>. <span class="ref-journal">Circ. Heart Fail</span>., <span class="ref-vol">8</span>, 243–251. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4383661/">PMC free article</a>]</span> [<a href="/pubmed/25550439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Heart+Fail&amp;title=Ventricular+conduction+and+long-term+heart+failure+outcomes+and+mortality+in+African+Americans:+insights+from+the+Jackson+Heart+Study&amp;author=R.J.+Mentz&amp;author=M.A.+Greiner&amp;author=A.D.+DeVore&amp;author=S.M.+Dunlay&amp;author=G.+Choudhary&amp;volume=8&amp;publication_year=2015&amp;pages=243-251&amp;pmid=25550439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B3">3. <span class="mixed-citation">Wang N.C., Maggioni A.P., Konstam M.A., Zannad F., Krasa H.B., Burnett J.C., Jr., Grinfeld L., Swedberg K., Udelson J.E., Cook T., et al. (2008) 
<span class="ref-title">Clinical implications of QRS duration in patients hospitalized with worsening heart failure and reduced left ventricular ejection fraction</span>. <span class="ref-journal">JAMA</span>, <span class="ref-vol">299</span>, 2656–2666. [<a href="/pubmed/18544725" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Clinical+implications+of+QRS+duration+in+patients+hospitalized+with+worsening+heart+failure+and+reduced+left+ventricular+ejection+fraction&amp;author=N.C.+Wang&amp;author=A.P.+Maggioni&amp;author=M.A.+Konstam&amp;author=F.+Zannad&amp;author=H.B.+Krasa&amp;volume=299&amp;publication_year=2008&amp;pages=2656-2666&amp;pmid=18544725&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B4">4. <span class="mixed-citation">Vitelli L.L., Crow R.S., Shahar E., Hutchinson R.G., Rautaharju P.M., Folsom A.R. (1998) 
<span class="ref-title">Electrocardiographic findings in a healthy biracial population. Atherosclerosis Risk in Communities (ARIC) Study Investigators</span>. <span class="ref-journal">Am. J. Cardiol</span>., <span class="ref-vol">81</span>, 453–459. [<a href="/pubmed/9485136" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol&amp;title=Electrocardiographic+findings+in+a+healthy+biracial+population.+Atherosclerosis+Risk+in+Communities+(ARIC)+Study+Investigators&amp;author=L.L.+Vitelli&amp;author=R.S.+Crow&amp;author=E.+Shahar&amp;author=R.G.+Hutchinson&amp;author=P.M.+Rautaharju&amp;volume=81&amp;publication_year=1998&amp;pages=453-459&amp;pmid=9485136&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B5">5. <span class="mixed-citation">Rautaharju P.M., Prineas R.J., Kadish A., Larson J.C., Hsia J., Lund B. (2006) 
<span class="ref-title">Normal standards for QT and QT subintervals derived from a large ethnically diverse population of women aged 50 to 79 years (the Women's Health Initiative [WHI])</span>. <span class="ref-journal">Am. J. Cardiol</span>., <span class="ref-vol">97</span>, 730–737. [<a href="/pubmed/16490447" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol&amp;title=Normal+standards+for+QT+and+QT+subintervals+derived+from+a+large+ethnically+diverse+population+of+women+aged+50+to+79+years+(the+Women's+Health+Initiative+[WHI])&amp;author=P.M.+Rautaharju&amp;author=R.J.+Prineas&amp;author=A.+Kadish&amp;author=J.C.+Larson&amp;author=J.+Hsia&amp;volume=97&amp;publication_year=2006&amp;pages=730-737&amp;pmid=16490447&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B6">6. <span class="mixed-citation">Walsh J.A., 3rd, Prineas R., Daviglus M.L., Ning H., Liu K., Lewis C.E., Sidney S., Schreiner P.J., Iribarren C., Lloyd-Jones D.M. (2010) 
<span class="ref-title">Prevalence of electrocardiographic abnormalities in a middle-aged, biracial population: Coronary Artery Risk Development in Young Adults study</span>. <span class="ref-journal">J. Electrocardiol</span>., <span class="ref-vol">43</span>, 385 e381–389. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3569004/">PMC free article</a>]</span> [<a href="/pubmed/20374967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Electrocardiol&amp;title=Prevalence+of+electrocardiographic+abnormalities+in+a+middle-aged,+biracial+population:+Coronary+Artery+Risk+Development+in+Young+Adults+study&amp;author=J.A.+Walsh&amp;author=R.+Prineas&amp;author=M.L.+Daviglus&amp;author=H.+Ning&amp;author=K.+Liu&amp;volume=43&amp;publication_year=2010&amp;pages=385+e381-389&amp;pmid=20374967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B7">7. <span class="mixed-citation">Holm H., Gudbjartsson D.F., Arnar D.O., Thorleifsson G., Thorgeirsson G., Stefansdottir H., Gudjonsson S.A., Jonasdottir A., Mathiesen E.B., Njolstad I., et al. (2010) 
<span class="ref-title">Several common variants modulate heart rate, PR interval and QRS duration</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">42</span>, 117–122. [<a href="/pubmed/20062063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Several+common+variants+modulate+heart+rate,+PR+interval+and+QRS+duration&amp;author=H.+Holm&amp;author=D.F.+Gudbjartsson&amp;author=D.O.+Arnar&amp;author=G.+Thorleifsson&amp;author=G.+Thorgeirsson&amp;volume=42&amp;publication_year=2010&amp;pages=117-122&amp;pmid=20062063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B8">8. <span class="mixed-citation">Sotoodehnia N., Isaacs A., de Bakker P.I., Dorr M., Newton-Cheh C., Nolte I.M., van der Harst P., Muller M., Eijgelsheim M., Alonso A., et al. (2010) 
<span class="ref-title">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">42</span>, 1068–1076. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3338195/">PMC free article</a>]</span> [<a href="/pubmed/21076409" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+variants+in+22+loci+are+associated+with+QRS+duration+and+cardiac+ventricular+conduction&amp;author=N.+Sotoodehnia&amp;author=A.+Isaacs&amp;author=P.I.+de+Bakker&amp;author=M.+Dorr&amp;author=C.+Newton-Cheh&amp;volume=42&amp;publication_year=2010&amp;pages=1068-1076&amp;pmid=21076409&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B9">9. <span class="mixed-citation">Rosenberg N.A., Huang L., Jewett E.M., Szpiech Z.A., Jankovic I., Boehnke M. (2010) 
<span class="ref-title">Genome-wide association studies in diverse populations</span>. <span class="ref-journal">Nat. Rev. Genet</span>., <span class="ref-vol">11</span>, 356–366. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3079573/">PMC free article</a>]</span> [<a href="/pubmed/20395969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Genome-wide+association+studies+in+diverse+populations&amp;author=N.A.+Rosenberg&amp;author=L.+Huang&amp;author=E.M.+Jewett&amp;author=Z.A.+Szpiech&amp;author=I.+Jankovic&amp;volume=11&amp;publication_year=2010&amp;pages=356-366&amp;pmid=20395969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B10">10. <span class="mixed-citation">Casto A.M., Feldman M.W. (2011) 
<span class="ref-title">Genome-wide association study SNPs in the human genome diversity project populations: does selection affect unlinked SNPs with shared trait associations?</span>
<span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">7</span>, e1001266. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3017115/">PMC free article</a>]</span> [<a href="/pubmed/21253569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+SNPs+in+the+human+genome+diversity+project+populations:+does+selection+affect+unlinked+SNPs+with+shared+trait+associations?&amp;author=A.M.+Casto&amp;author=M.W.+Feldman&amp;volume=7&amp;publication_year=2011&amp;pages=e1001266&amp;pmid=21253569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B11">11. <span class="mixed-citation">Matise T.C., Ambite J.L., Buyske S., Carlson C.S., Cole S.A., Crawford D.C., Haiman C.A., Heiss G., Kooperberg C., Marchand L.L., et al. (2011) 
<span class="ref-title">The Next PAGE in understanding complex traits: design for the analysis of Population Architecture Using Genetics and Epidemiology (PAGE) Study</span>. <span class="ref-journal">Am. J. Epidemiol</span>., <span class="ref-vol">174</span>, 849–859. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3176830/">PMC free article</a>]</span> [<a href="/pubmed/21836165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol&amp;title=The+Next+PAGE+in+understanding+complex+traits:+design+for+the+analysis+of+Population+Architecture+Using+Genetics+and+Epidemiology+(PAGE)+Study&amp;author=T.C.+Matise&amp;author=J.L.+Ambite&amp;author=S.+Buyske&amp;author=C.S.+Carlson&amp;author=S.A.+Cole&amp;volume=174&amp;publication_year=2011&amp;pages=849-859&amp;pmid=21836165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B12">12. <span class="mixed-citation">Splawski I., Timothy K.W., Tateyama M., Clancy C.E., Malhotra A., Beggs A.H., Cappuccio F.P., Sagnella G.A., Kass R.S., Keating M.T. (2002) 
<span class="ref-title">Variant of SCN5A sodium channel implicated in risk of cardiac arrhythmia</span>. <span class="ref-journal">Science</span>, <span class="ref-vol">297</span>, 1333–1336. [<a href="/pubmed/12193783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Variant+of+SCN5A+sodium+channel+implicated+in+risk+of+cardiac+arrhythmia&amp;author=I.+Splawski&amp;author=K.W.+Timothy&amp;author=M.+Tateyama&amp;author=C.E.+Clancy&amp;author=A.+Malhotra&amp;volume=297&amp;publication_year=2002&amp;pages=1333-1336&amp;pmid=12193783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B13">13. <span class="mixed-citation">Jeff J.M., Brown-Gentry K., Buxbaum S.G., Sarpong D.F., Taylor H.A., George A.L., Jr., Roden D.M., Crawford D.C. (2011) 
<span class="ref-title">SCN5A variation is associated with electrocardiographic traits in the Jackson Heart Study</span>. <span class="ref-journal">Circ. Cardiovasc. Genet</span>., <span class="ref-vol">4</span>, 139–144. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080430/">PMC free article</a>]</span> [<a href="/pubmed/21325150" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet&amp;title=SCN5A+variation+is+associated+with+electrocardiographic+traits+in+the+Jackson+Heart+Study&amp;author=J.M.+Jeff&amp;author=K.+Brown-Gentry&amp;author=S.G.+Buxbaum&amp;author=D.F.+Sarpong&amp;author=H.A.+Taylor&amp;volume=4&amp;publication_year=2011&amp;pages=139-144&amp;pmid=21325150&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B14">14. <span class="mixed-citation">Magnani J.W., Brody J.A., Prins B.P., Arking D.E., Lin H., Yin X., Liu C.T., Morrison A.C., Zhang F., Spector T.D., et al. (2014) 
<span class="ref-title">Sequencing of SCN5A identifies rare and common variants associated with cardiac conduction: Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium</span>. <span class="ref-journal">Circ. Cardiovasc. Genet</span>., <span class="ref-vol">7</span>, 365–373. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4177904/">PMC free article</a>]</span> [<a href="/pubmed/24951663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet&amp;title=Sequencing+of+SCN5A+identifies+rare+and+common+variants+associated+with+cardiac+conduction:+Cohorts+for+Heart+and+Aging+Research+in+Genomic+Epidemiology+(CHARGE)+Consortium&amp;author=J.W.+Magnani&amp;author=J.A.+Brody&amp;author=B.P.+Prins&amp;author=D.E.+Arking&amp;author=H.+Lin&amp;volume=7&amp;publication_year=2014&amp;pages=365-373&amp;pmid=24951663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B15">15. <span class="mixed-citation">Ilkhanoff L., Arking D.E., Lemaitre R.N., Alonso A., Chen L.Y., Durda P., Hesselson S.E., Kerr K.F., Magnani J.W., Marcus G.M., et al. (2014) 
<span class="ref-title">A common SCN5A variant is associated with PR interval and atrial fibrillation among African Americans</span>. <span class="ref-journal">J. Cardiovasc. Electrophysiol</span>., <span class="ref-vol">25</span>, 1150–1157. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4454499/">PMC free article</a>]</span> [<a href="/pubmed/25065297" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cardiovasc.+Electrophysiol&amp;title=A+common+SCN5A+variant+is+associated+with+PR+interval+and+atrial+fibrillation+among+African+Americans&amp;author=L.+Ilkhanoff&amp;author=D.E.+Arking&amp;author=R.N.+Lemaitre&amp;author=A.+Alonso&amp;author=L.Y.+Chen&amp;volume=25&amp;publication_year=2014&amp;pages=1150-1157&amp;pmid=25065297&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B16">16. <span class="mixed-citation">Chambers J.C., Zhao J., Terracciano C.M., Bezzina C.R., Zhang W., Kaba R., Navaratnarajah M., Lotlikar A., Sehmi J.S., Kooner M.K., et al. (2010) 
<span class="ref-title">Genetic variation in SCN10A influences cardiac conduction</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">42</span>, 149–152. [<a href="/pubmed/20062061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genetic+variation+in+SCN10A+influences+cardiac+conduction&amp;author=J.C.+Chambers&amp;author=J.+Zhao&amp;author=C.M.+Terracciano&amp;author=C.R.+Bezzina&amp;author=W.+Zhang&amp;volume=42&amp;publication_year=2010&amp;pages=149-152&amp;pmid=20062061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B17">17. <span class="mixed-citation">Morris A.P. (2011) 
<span class="ref-title">Transethnic meta-analysis of genomewide association studies</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">35</span>, 809–822. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3460225/">PMC free article</a>]</span> [<a href="/pubmed/22125221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Transethnic+meta-analysis+of+genomewide+association+studies&amp;author=A.P.+Morris&amp;volume=35&amp;publication_year=2011&amp;pages=809-822&amp;pmid=22125221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B18">18. <span class="mixed-citation">Schroeter A., Walzik S., Blechschmidt S., Haufe V., Benndorf K., Zimmer T. (2010) 
<span class="ref-title">Structure and function of splice variants of the cardiac voltage-gated sodium channel Na(v)1.5</span>. <span class="ref-journal">J. Mol. Cell. Cardiol</span>., <span class="ref-vol">49</span>, 16–24. [<a href="/pubmed/20398673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Cell.+Cardiol&amp;title=Structure+and+function+of+splice+variants+of+the+cardiac+voltage-gated+sodium+channel+Na(v)1.5&amp;author=A.+Schroeter&amp;author=S.+Walzik&amp;author=S.+Blechschmidt&amp;author=V.+Haufe&amp;author=K.+Benndorf&amp;volume=49&amp;publication_year=2010&amp;pages=16-24&amp;pmid=20398673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B19">19. <span class="mixed-citation">Zimmer T., Surber R. (2008) 
<span class="ref-title">SCN5A channelopathies–an update on mutations and mechanisms</span>. <span class="ref-journal">Prog. Biophys. Mol. Biol</span>., <span class="ref-vol">98</span>, 120–136. [<a href="/pubmed/19027780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prog.+Biophys.+Mol.+Biol&amp;title=SCN5A+channelopathies–an+update+on+mutations+and+mechanisms&amp;author=T.+Zimmer&amp;author=R.+Surber&amp;volume=98&amp;publication_year=2008&amp;pages=120-136&amp;pmid=19027780&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B20">20. <span class="mixed-citation">Smith J.G., Magnani J.W., Palmer C., Meng Y.A., Soliman E.Z., Musani S.K., Kerr K.F., Schnabel R.B., Lubitz S.A., Sotoodehnia N., et al. (2011) 
<span class="ref-title">Genome-wide association studies of the PR interval in African Americans</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">7</span>, e1001304. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037415/">PMC free article</a>]</span> [<a href="/pubmed/21347284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+studies+of+the+PR+interval+in+African+Americans&amp;author=J.G.+Smith&amp;author=J.W.+Magnani&amp;author=C.+Palmer&amp;author=Y.A.+Meng&amp;author=E.Z.+Soliman&amp;volume=7&amp;publication_year=2011&amp;pages=e1001304&amp;pmid=21347284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B21">21. <span class="mixed-citation">Butler A.M., Yin X., Evans D.S., Nalls M.A., Smith E.N., Tanaka T., Li G., Buxbaum S.G., Whitsel E.A., Alonso A., et al. (2012) 
<span class="ref-title">Novel loci associated with PR interval in a genome-wide association study of 10 African American cohorts</span>. <span class="ref-journal">Circ. Cardiovasc. Genet</span>., <span class="ref-vol">5</span>, 639–646. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3560365/">PMC free article</a>]</span> [<a href="/pubmed/23139255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet&amp;title=Novel+loci+associated+with+PR+interval+in+a+genome-wide+association+study+of+10+African+American+cohorts&amp;author=A.M.+Butler&amp;author=X.+Yin&amp;author=D.S.+Evans&amp;author=M.A.+Nalls&amp;author=E.N.+Smith&amp;volume=5&amp;publication_year=2012&amp;pages=639-646&amp;pmid=23139255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B22">22. <span class="mixed-citation">Stadhouders R., van den Heuvel A., Kolovos P., Jorna R., Leslie K., Grosveld F., Soler E. (2012) 
<span class="ref-title">Transcription regulation by distal enhancers: who's in the loop?</span>
<span class="ref-journal">Transcription</span>, <span class="ref-vol">3</span>, 181–186. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3654767/">PMC free article</a>]</span> [<a href="/pubmed/22771987" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transcription&amp;title=Transcription+regulation+by+distal+enhancers:+who's+in+the+loop?&amp;author=R.+Stadhouders&amp;author=A.+van+den+Heuvel&amp;author=P.+Kolovos&amp;author=R.+Jorna&amp;author=K.+Leslie&amp;volume=3&amp;publication_year=2012&amp;pages=181-186&amp;pmid=22771987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B23">23. <span class="mixed-citation">Park I., Han C., Jin S., Lee B., Choi H., Kwon J.T., Kim D., Kim J., Lifirsu E., Park W.J., et al. (2011) 
<span class="ref-title">Myosin regulatory light chains are required to maintain the stability of myosin II and cellular integrity</span>. <span class="ref-journal">Biochem. J</span>., <span class="ref-vol">434</span>, 171–180. [<a href="/pubmed/21126233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+J&amp;title=Myosin+regulatory+light+chains+are+required+to+maintain+the+stability+of+myosin+II+and+cellular+integrity&amp;author=I.+Park&amp;author=C.+Han&amp;author=S.+Jin&amp;author=B.+Lee&amp;author=H.+Choi&amp;volume=434&amp;publication_year=2011&amp;pages=171-180&amp;pmid=21126233&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B24">24. <span class="mixed-citation">Su A.I., Wiltshire T., Batalov S., Lapp H., Ching K.A., Block D., Zhang J., Soden R., Hayakawa M., Kreiman G., et al. (2004) 
<span class="ref-title">A gene atlas of the mouse and human protein-encoding transcriptomes. <em>Proc.</em></span>
<span class="ref-journal">Natl. Acad. Sci. U S A</span>, <span class="ref-vol">101</span>, 6062–6067. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC395923/">PMC free article</a>]</span> [<a href="/pubmed/15075390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Natl.+Acad.+Sci.+U+S+A&amp;title=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes.+Proc.&amp;author=A.I.+Su&amp;author=T.+Wiltshire&amp;author=S.+Batalov&amp;author=H.+Lapp&amp;author=K.A.+Ching&amp;volume=101&amp;publication_year=2004&amp;pages=6062-6067&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B25">25. <span class="mixed-citation">Eijgelsheim M., Newton-Cheh C., Sotoodehnia N., de Bakker P.I., Muller M., Morrison A.C., Smith A.V., Isaacs A., Sanna S., Dorr M., et al. (2010) 
<span class="ref-title">Genome-wide association analysis identifies multiple loci related to resting heart rate</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">19</span>, 3885–3894. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3657480/">PMC free article</a>]</span> [<a href="/pubmed/20639392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Genome-wide+association+analysis+identifies+multiple+loci+related+to+resting+heart+rate&amp;author=M.+Eijgelsheim&amp;author=C.+Newton-Cheh&amp;author=N.+Sotoodehnia&amp;author=P.I.+de+Bakker&amp;author=M.+Muller&amp;volume=19&amp;publication_year=2010&amp;pages=3885-3894&amp;pmid=20639392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B26">26. <span class="mixed-citation">Deo R., Nalls M.A., Avery C.L., Smith J.G., Evans D.S., Keller M.F., Butler A.M., Buxbaum S.G., Li G., Miguel Quibrera P., et al. (2013) 
<span class="ref-title">Common genetic variation near the connexin-43 gene is associated with resting heart rate in African Americans: a genome-wide association study of 13,372 participants</span>. <span class="ref-journal">Heart Rhythm</span>, <span class="ref-vol">10</span>, 401–408. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3718037/">PMC free article</a>]</span> [<a href="/pubmed/23183192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Heart+Rhythm&amp;title=Common+genetic+variation+near+the+connexin-43+gene+is+associated+with+resting+heart+rate+in+African+Americans:+a+genome-wide+association+study+of+13,372+participants&amp;author=R.+Deo&amp;author=M.A.+Nalls&amp;author=C.L.+Avery&amp;author=J.G.+Smith&amp;author=D.S.+Evans&amp;volume=10&amp;publication_year=2013&amp;pages=401-408&amp;pmid=23183192&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B27">27. <span class="mixed-citation">Cohen N.R., Garg S., Brenner M.B. (2009) 
<span class="ref-title">Antigen Presentation by CD1 Lipids, T Cells, and NKT Cells in Microbial Immunity</span>. <span class="ref-journal">Adv. Immunol</span>., <span class="ref-vol">102</span>, 1–94. [<a href="/pubmed/19477319" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Immunol&amp;title=Antigen+Presentation+by+CD1+Lipids,+T+Cells,+and+NKT+Cells+in+Microbial+Immunity&amp;author=N.R.+Cohen&amp;author=S.+Garg&amp;author=M.B.+Brenner&amp;volume=102&amp;publication_year=2009&amp;pages=1-94&amp;pmid=19477319&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B28">28. <span class="mixed-citation">Malnic B., Godfrey P.A., Buck L.B. (2004) 
<span class="ref-title">The human olfactory receptor gene family. <em>Proc.</em></span>
<span class="ref-journal">Natl. Acad. Sci. U S Aa</span>, <span class="ref-vol">101</span>, 2584–2589. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC356993/">PMC free article</a>]</span> [<a href="/pubmed/14983052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Natl.+Acad.+Sci.+U+S+Aa&amp;title=The+human+olfactory+receptor+gene+family.+Proc.&amp;author=B.+Malnic&amp;author=P.A.+Godfrey&amp;author=L.B.+Buck&amp;volume=101&amp;publication_year=2004&amp;pages=2584-2589&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B29">29. <span class="mixed-citation">Smith S., Curran J., Hund T.J., Mohler P.J. (2012) 
<span class="ref-title">Defects in cytoskeletal signaling pathways, arrhythmia, and sudden cardiac death</span>. <span class="ref-journal">Front. Physiol</span>., <span class="ref-vol">3</span>, 122. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3343379/">PMC free article</a>]</span> [<a href="/pubmed/22586405" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Physiol&amp;title=Defects+in+cytoskeletal+signaling+pathways,+arrhythmia,+and+sudden+cardiac+death&amp;author=S.+Smith&amp;author=J.+Curran&amp;author=T.J.+Hund&amp;author=P.J.+Mohler&amp;volume=3&amp;publication_year=2012&amp;pages=122&amp;pmid=22586405&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B30">30. <span class="mixed-citation">Mohler P.J., Rivolta I., Napolitano C., LeMaillet G., Lambert S., Priori S.G., Bennett V. (2004) 
<span class="ref-title">Nav1.5 E1053K mutation causing Brugada syndrome blocks binding to ankyrin-G and expression of Nav1.5 on the surface of cardiomyocytes. <em>Proc.</em></span>
<span class="ref-journal">Natl. Acad. Sci. U S Aa</span>, <span class="ref-vol">101</span>, 17533–17538. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC536011/">PMC free article</a>]</span> [<a href="/pubmed/15579534" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Natl.+Acad.+Sci.+U+S+Aa&amp;title=Nav1.5+E1053K+mutation+causing+Brugada+syndrome+blocks+binding+to+ankyrin-G+and+expression+of+Nav1.5+on+the+surface+of+cardiomyocytes.+Proc.&amp;author=P.J.+Mohler&amp;author=I.+Rivolta&amp;author=C.+Napolitano&amp;author=G.+LeMaillet&amp;author=S.+Lambert&amp;volume=101&amp;publication_year=2004&amp;pages=17533-17538&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B31">31. <span class="mixed-citation">Hund T.J., Koval O.M., Li J., Wright P.J., Qian L., Snyder J.S., Gudmundsson H., Kline C.F., Davidson N.P., Cardona N., et al. (2010) 
<span class="ref-title">A beta(IV)-spectrin/CaMKII signaling complex is essential for membrane excitability in mice</span>. <span class="ref-journal">J. Clin. Invest</span>., <span class="ref-vol">120</span>, 3508–3519. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947241/">PMC free article</a>]</span> [<a href="/pubmed/20877009" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=A+beta(IV)-spectrin/CaMKII+signaling+complex+is+essential+for+membrane+excitability+in+mice&amp;author=T.J.+Hund&amp;author=O.M.+Koval&amp;author=J.+Li&amp;author=P.J.+Wright&amp;author=L.+Qian&amp;volume=120&amp;publication_year=2010&amp;pages=3508-3519&amp;pmid=20877009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B32">32. <span class="mixed-citation">Teo Y.Y., Small K.S., Kwiatkowski D.P. (2010) 
<span class="ref-title">Methodological challenges of genome-wide association analysis in Africa</span>. <span class="ref-journal">Nat. Rev. Genet</span>., <span class="ref-vol">11</span>, 149–160. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3769612/">PMC free article</a>]</span> [<a href="/pubmed/20084087" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Methodological+challenges+of+genome-wide+association+analysis+in+Africa&amp;author=Y.Y.+Teo&amp;author=K.S.+Small&amp;author=D.P.+Kwiatkowski&amp;volume=11&amp;publication_year=2010&amp;pages=149-160&amp;pmid=20084087&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B33">33. <span class="mixed-citation">Gravel S., Henn B.M., Gutenkunst R.N., Indap A.R., Marth G.T., Clark A.G., Yu F., Gibbs R.A., Bustamante C.D. (2011) 
<span class="ref-title">Demographic history and rare allele sharing among human populations</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U S Aa</span>, <span class="ref-vol">108</span>, 11983–11988. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3142009/">PMC free article</a>]</span> [<a href="/pubmed/21730125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U+S+Aa&amp;title=Demographic+history+and+rare+allele+sharing+among+human+populations&amp;author=S.+Gravel&amp;author=B.M.+Henn&amp;author=R.N.+Gutenkunst&amp;author=A.R.+Indap&amp;author=G.T.+Marth&amp;volume=108&amp;publication_year=2011&amp;pages=11983-11988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B34">34. <span class="mixed-citation">Tennessen J.A., Bigham A.W., O'Connor T.D., Fu W., Kenny E.E., Gravel S., McGee S., Do R., Liu X., Jun G., et al. (2012) 
<span class="ref-title">Evolution and functional impact of rare coding variation from deep sequencing of human exomes</span>. <span class="ref-journal">Science</span>, <span class="ref-vol">337</span>, 64–69. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3708544/">PMC free article</a>]</span> [<a href="/pubmed/22604720" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Evolution+and+functional+impact+of+rare+coding+variation+from+deep+sequencing+of+human+exomes&amp;author=J.A.+Tennessen&amp;author=A.W.+Bigham&amp;author=T.D.+O'Connor&amp;author=W.+Fu&amp;author=E.E.+Kenny&amp;volume=337&amp;publication_year=2012&amp;pages=64-69&amp;pmid=22604720&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B35">35. <span class="mixed-citation">Fu W., O'Connor T.D., Jun G., Kang H.M., Abecasis G., Leal S.M., Gabriel S., Rieder M.J., Altshuler D., Shendure J., et al. (2013) 
<span class="ref-title">Analysis of 6,515 exomes reveals the recent origin of most human protein-coding variants</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">493</span>, 216–220. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3676746/">PMC free article</a>]</span> [<a href="/pubmed/23201682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Analysis+of+6,515+exomes+reveals+the+recent+origin+of+most+human+protein-coding+variants&amp;author=W.+Fu&amp;author=T.D.+O'Connor&amp;author=G.+Jun&amp;author=H.M.+Kang&amp;author=G.+Abecasis&amp;volume=493&amp;publication_year=2013&amp;pages=216-220&amp;pmid=23201682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B36">36. <span class="mixed-citation">Abecasis G.R., Auton A., Brooks L.D., DePristo M.A., Durbin R.M., Handsaker R.E., Kang H.M., Marth G.T., McVean G.A. (2012) 
<span class="ref-title">An integrated map of genetic variation from 1,092 human genomes</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">491</span>, 56–65. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=G.R.+Abecasis&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=M.A.+DePristo&amp;author=R.M.+Durbin&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B37">37. <span class="mixed-citation">Dickson S.P., Wang K., Krantz I., Hakonarson H., Goldstein D.B. (2010) 
<span class="ref-title">Rare variants create synthetic genome-wide associations</span>. <span class="ref-journal">PLoS Biol</span>., <span class="ref-vol">8</span>, e1000294. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2811148/">PMC free article</a>]</span> [<a href="/pubmed/20126254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Rare+variants+create+synthetic+genome-wide+associations&amp;author=S.P.+Dickson&amp;author=K.+Wang&amp;author=I.+Krantz&amp;author=H.+Hakonarson&amp;author=D.B.+Goldstein&amp;volume=8&amp;publication_year=2010&amp;pages=e1000294&amp;pmid=20126254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B38">38. <span class="mixed-citation">Anderson C.A., Soranzo N., Zeggini E., Barrett J.C. (2011) 
<span class="ref-title">Synthetic associations are unlikely to account for many common disease genome-wide association signals</span>. <span class="ref-journal">PLoS Biol</span>., <span class="ref-vol">9</span>, e1000580. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3022527/">PMC free article</a>]</span> [<a href="/pubmed/21267062" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Synthetic+associations+are+unlikely+to+account+for+many+common+disease+genome-wide+association+signals&amp;author=C.A.+Anderson&amp;author=N.+Soranzo&amp;author=E.+Zeggini&amp;author=J.C.+Barrett&amp;volume=9&amp;publication_year=2011&amp;pages=e1000580&amp;pmid=21267062&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B39">39. <span class="mixed-citation">Wray N.R., Purcell S.M., Visscher P.M. (2011) 
<span class="ref-title">Synthetic associations created by rare variants do not explain most GWAS results</span>. <span class="ref-journal">PLoS Biol</span>., <span class="ref-vol">9</span>, e1000579. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3022526/">PMC free article</a>]</span> [<a href="/pubmed/21267061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Synthetic+associations+created+by+rare+variants+do+not+explain+most+GWAS+results&amp;author=N.R.+Wray&amp;author=S.M.+Purcell&amp;author=P.M.+Visscher&amp;volume=9&amp;publication_year=2011&amp;pages=e1000579&amp;pmid=21267061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B40">40. <span class="mixed-citation">Waters K.M., Stram D.O., Hassanein M.T., Le Marchand L., Wilkens L.R., Maskarinec G., Monroe K.R., Kolonel L.N., Altshuler D., Henderson B.E., et al. (2010) 
<span class="ref-title">Consistent association of type 2 diabetes risk variants found in europeans in diverse racial and ethnic groups</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">6</span>, pii: e1001078. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928808/">PMC free article</a>]</span> [<a href="/pubmed/20865176" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Consistent+association+of+type+2+diabetes+risk+variants+found+in+europeans+in+diverse+racial+and+ethnic+groups&amp;author=K.M.+Waters&amp;author=D.O.+Stram&amp;author=M.T.+Hassanein&amp;author=L.+Le+Marchand&amp;author=L.R.+Wilkens&amp;volume=6&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B41">41. <span class="mixed-citation">Haiman C.A., Chen G.K., Blot W.J., Strom S.S., Berndt S.I., Kittles R.A., Rybicki B.A., Isaacs W.B., Ingles S.A., Stanford J.L., et al. (2011) 
<span class="ref-title">Characterizing genetic risk at known prostate cancer susceptibility loci in African Americans</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">7</span>, e1001387. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3102736/">PMC free article</a>]</span> [<a href="/pubmed/21637779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Characterizing+genetic+risk+at+known+prostate+cancer+susceptibility+loci+in+African+Americans&amp;author=C.A.+Haiman&amp;author=G.K.+Chen&amp;author=W.J.+Blot&amp;author=S.S.+Strom&amp;author=S.I.+Berndt&amp;volume=7&amp;publication_year=2011&amp;pages=e1001387&amp;pmid=21637779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B42">42. <span class="mixed-citation">Chen F., Chen G.K., Millikan R.C., John E.M., Ambrosone C.B., Bernstein L., Zheng W., Hu J.J., Ziegler R.G., Deming S.L., et al. (2011) 
<span class="ref-title">Fine-mapping of breast cancer susceptibility loci characterizes genetic risk in African Americans</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">20</span>, 4491–4503. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3196889/">PMC free article</a>]</span> [<a href="/pubmed/21852243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Fine-mapping+of+breast+cancer+susceptibility+loci+characterizes+genetic+risk+in+African+Americans&amp;author=F.+Chen&amp;author=G.K.+Chen&amp;author=R.C.+Millikan&amp;author=E.M.+John&amp;author=C.B.+Ambrosone&amp;volume=20&amp;publication_year=2011&amp;pages=4491-4503&amp;pmid=21852243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B43">43. <span class="mixed-citation">Carlson C.S., Matise T.C., North K.E., Haiman C.A., Fesinmeyer M.D., Buyske S., Schumacher F.R., Peters U., Franceschini N., Ritchie M.D., et al. (2013) 
<span class="ref-title">Generalization and dilution of association results from European GWAS in populations of non-European ancestry: the PAGE study</span>. <span class="ref-journal">PLoS Biol</span>., <span class="ref-vol">11</span>, e1001661. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3775722/">PMC free article</a>]</span> [<a href="/pubmed/24068893" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Generalization+and+dilution+of+association+results+from+European+GWAS+in+populations+of+non-European+ancestry:+the+PAGE+study&amp;author=C.S.+Carlson&amp;author=T.C.+Matise&amp;author=K.E.+North&amp;author=C.A.+Haiman&amp;author=M.D.+Fesinmeyer&amp;volume=11&amp;publication_year=2013&amp;pages=e1001661&amp;pmid=24068893&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B44">44. <span class="mixed-citation">Dumitrescu L., Carty C.L., Taylor K., Schumacher F.R., Hindorff L.A., Ambite J.L., Anderson G., Best L.G., Brown-Gentry K., Buzkova P., et al. (2011) 
<span class="ref-title">Genetic determinants of lipid traits in diverse populations from the population architecture using genomics and epidemiology (PAGE) study</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">7</span>, e1002138. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128106/">PMC free article</a>]</span> [<a href="/pubmed/21738485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetic+determinants+of+lipid+traits+in+diverse+populations+from+the+population+architecture+using+genomics+and+epidemiology+(PAGE)+study&amp;author=L.+Dumitrescu&amp;author=C.L.+Carty&amp;author=K.+Taylor&amp;author=F.R.+Schumacher&amp;author=L.A.+Hindorff&amp;volume=7&amp;publication_year=2011&amp;pages=e1002138&amp;pmid=21738485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B45">45. <span class="mixed-citation">Coram M.A., Duan Q., Hoffmann T.J., Thornton T., Knowles J.W., Johnson N.A., Ochs-Balcom H.M., Donlon T.A., Martin L.W., Eaton C.B., et al. (2013) 
<span class="ref-title">Genome-wide characterization of shared and distinct genetic components that influence blood lipid levels in ethnically diverse human populations</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">92</span>, 904–916. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3675231/">PMC free article</a>]</span> [<a href="/pubmed/23726366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genome-wide+characterization+of+shared+and+distinct+genetic+components+that+influence+blood+lipid+levels+in+ethnically+diverse+human+populations&amp;author=M.A.+Coram&amp;author=Q.+Duan&amp;author=T.J.+Hoffmann&amp;author=T.+Thornton&amp;author=J.W.+Knowles&amp;volume=92&amp;publication_year=2013&amp;pages=904-916&amp;pmid=23726366&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B46">46. <span class="mixed-citation">Wu Y., Waite L.L., Jackson A.U., Sheu W.H., Buyske S., Absher D., Arnett D.K., Boerwinkle E., Bonnycastle L.L., Carty C.L., et al. (2013) 
<span class="ref-title">Trans-ethnic fine-mapping of lipid loci identifies population-specific signals and allelic heterogeneity that increases the trait variance explained</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">9</span>, e1003379. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3605054/">PMC free article</a>]</span> [<a href="/pubmed/23555291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Trans-ethnic+fine-mapping+of+lipid+loci+identifies+population-specific+signals+and+allelic+heterogeneity+that+increases+the+trait+variance+explained&amp;author=Y.+Wu&amp;author=L.L.+Waite&amp;author=A.U.+Jackson&amp;author=W.H.+Sheu&amp;author=S.+Buyske&amp;volume=9&amp;publication_year=2013&amp;pages=e1003379&amp;pmid=23555291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B47">47. <span class="mixed-citation">Marigorta U.M., Navarro A. (2013) 
<span class="ref-title">High trans-ethnic replicability of GWAS results implies common causal variants</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">9</span>, e1003566. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3681663/">PMC free article</a>]</span> [<a href="/pubmed/23785302" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=High+trans-ethnic+replicability+of+GWAS+results+implies+common+causal+variants&amp;author=U.M.+Marigorta&amp;author=A.+Navarro&amp;volume=9&amp;publication_year=2013&amp;pages=e1003566&amp;pmid=23785302&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B48">48. <span class="mixed-citation">Smith J.G., Avery C.L., Evans D.S., Nalls M.A., Meng Y.A., Smith E.N., Palmer C., Tanaka T., Mehra R., Butler A.M., et al. (2012) 
<span class="ref-title">Impact of ancestry and common genetic variants on QT interval in African Americans</span>. <span class="ref-journal">Circ. Cardiovasc. Genet</span>., <span class="ref-vol">5</span>, 647–655. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3568265/">PMC free article</a>]</span> [<a href="/pubmed/23166209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet&amp;title=Impact+of+ancestry+and+common+genetic+variants+on+QT+interval+in+African+Americans&amp;author=J.G.+Smith&amp;author=C.L.+Avery&amp;author=D.S.+Evans&amp;author=M.A.+Nalls&amp;author=Y.A.+Meng&amp;volume=5&amp;publication_year=2012&amp;pages=647-655&amp;pmid=23166209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B49">49. <span class="mixed-citation">Jeff J.M., Ritchie M.D., Denny J.C., Kho A.N., Ramirez A.H., Crosslin D., Armstrong L., Basford M.A., Wolf W.A., Pacheco J.A., et al. (2013) 
<span class="ref-title">Generalization of variants identified by genome-wide association studies for electrocardiographic traits in African Americans</span>. <span class="ref-journal">Ann. Hum. Genet</span>., <span class="ref-vol">77</span>, 321–332. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3743946/">PMC free article</a>]</span> [<a href="/pubmed/23534349" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Hum.+Genet&amp;title=Generalization+of+variants+identified+by+genome-wide+association+studies+for+electrocardiographic+traits+in+African+Americans&amp;author=J.M.+Jeff&amp;author=M.D.+Ritchie&amp;author=J.C.+Denny&amp;author=A.N.+Kho&amp;author=A.H.+Ramirez&amp;volume=77&amp;publication_year=2013&amp;pages=321-332&amp;pmid=23534349&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B50">50. <span class="mixed-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. (2006) 
<span class="ref-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">38</span>, 904–909. [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B51">51. <span class="mixed-citation">Musunuru K., Lettre G., Young T., Farlow D.N., Pirruccello J.P., Ejebe K.G., Keating B.J., Yang Q., Chen M.H., Lapchyk N., et al. (2010) 
<span class="ref-title">Candidate gene association resource (CARe): design, methods, and proof of concept</span>. <span class="ref-journal">Circ. Cardiovasc. Genet</span>., <span class="ref-vol">3</span>, 267–275. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3048024/">PMC free article</a>]</span> [<a href="/pubmed/20400780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet&amp;title=Candidate+gene+association+resource+(CARe):+design,+methods,+and+proof+of+concept&amp;author=K.+Musunuru&amp;author=G.+Lettre&amp;author=T.+Young&amp;author=D.N.+Farlow&amp;author=J.P.+Pirruccello&amp;volume=3&amp;publication_year=2010&amp;pages=267-275&amp;pmid=20400780&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B52">52. <span class="mixed-citation">Lettre G., Palmer C.D., Young T., Ejebe K.G., Allayee H., Benjamin E.J., Bennett F., Bowden D.W., Chakravarti A., Dreisbach A., et al. (2011) 
<span class="ref-title">Genome-wide association study of coronary heart disease and its risk factors in 8,090 African Americans: the NHLBI CARe Project</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">7</span>, e1001300. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037413/">PMC free article</a>]</span> [<a href="/pubmed/21347282" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+of+coronary+heart+disease+and+its+risk+factors+in+8,090+African+Americans:+the+NHLBI+CARe+Project&amp;author=G.+Lettre&amp;author=C.D.+Palmer&amp;author=T.+Young&amp;author=K.G.+Ejebe&amp;author=H.+Allayee&amp;volume=7&amp;publication_year=2011&amp;pages=e1001300&amp;pmid=21347282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B53">53. <span class="mixed-citation">Chen M.H., Yang Q. (2010) 
<span class="ref-title">GWAF: an R package for genome-wide association analyses with family data</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">26</span>, 580–581. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852219/">PMC free article</a>]</span> [<a href="/pubmed/20040588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GWAF:+an+R+package+for+genome-wide+association+analyses+with+family+data&amp;author=M.H.+Chen&amp;author=Q.+Yang&amp;volume=26&amp;publication_year=2010&amp;pages=580-581&amp;pmid=20040588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B54">54. <span class="mixed-citation">Willer C.J., Li Y., Abecasis G.R. (2010) 
<span class="ref-title">METAL: fast and efficient meta-analysis of genomewide association scans</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">26</span>, 2190–2191. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B55">55. <span class="mixed-citation">Devlin B., Roeder K. (1999) 
<span class="ref-title">Genomic control for association studies</span>. <span class="ref-journal">Biometrics</span>, <span class="ref-vol">55</span>, 997–1004. [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B56">56. <span class="mixed-citation">Pe'er I., Yelensky R., Altshuler D., Daly M.J. (2008) 
<span class="ref-title">Estimation of the multiple testing burden for genomewide association studies of nearly all common variants</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">32</span>, 381–385. [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I.+Pe'er&amp;author=R.+Yelensky&amp;author=D.+Altshuler&amp;author=M.J.+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B57">57. <span class="mixed-citation">Whitehead A., Whitehead J. (1991) 
<span class="ref-title">A general parametric approach to the meta-analysis of randomized clinical trials</span>. <span class="ref-journal">Stat. Med</span>., <span class="ref-vol">10</span>, 1665–1677. [<a href="/pubmed/1792461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Med&amp;title=A+general+parametric+approach+to+the+meta-analysis+of+randomized+clinical+trials&amp;author=A.+Whitehead&amp;author=J.+Whitehead&amp;volume=10&amp;publication_year=1991&amp;pages=1665-1677&amp;pmid=1792461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B58">58. <span class="mixed-citation">Higgins J.P., Thompson S.G. (2002) 
<span class="ref-title">Quantifying heterogeneity in a meta-analysis</span>. <span class="ref-journal">Stat. Med</span>., <span class="ref-vol">21</span>, 1539–1558. [<a href="/pubmed/12111919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Med&amp;title=Quantifying+heterogeneity+in+a+meta-analysis&amp;author=J.P.+Higgins&amp;author=S.G.+Thompson&amp;volume=21&amp;publication_year=2002&amp;pages=1539-1558&amp;pmid=12111919&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B59">59. <span class="mixed-citation">Patterson N., Hattangadi N., Lane B., Lohmueller K.E., Hafler D.A., Oksenberg J.R., Hauser S.L., Smith M.W., O'Brien S.J., Altshuler D., et al. (2004) 
<span class="ref-title">Methods for high-density admixture mapping of disease genes</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">74</span>, 979–1000. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1181990/">PMC free article</a>]</span> [<a href="/pubmed/15088269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Methods+for+high-density+admixture+mapping+of+disease+genes&amp;author=N.+Patterson&amp;author=N.+Hattangadi&amp;author=B.+Lane&amp;author=K.E.+Lohmueller&amp;author=D.A.+Hafler&amp;volume=74&amp;publication_year=2004&amp;pages=979-1000&amp;pmid=15088269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B60">60. <span class="mixed-citation">Price A.L., Tandon A., Patterson N., Barnes K.C., Rafaels N., Ruczinski I., Beaty T.H., Mathias R., Reich D., Myers S. (2009) 
<span class="ref-title">Sensitive detection of chromosomal segments of distinct ancestry in admixed populations</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">5</span>, e1000519. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689842/">PMC free article</a>]</span> [<a href="/pubmed/19543370" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Sensitive+detection+of+chromosomal+segments+of+distinct+ancestry+in+admixed+populations&amp;author=A.L.+Price&amp;author=A.+Tandon&amp;author=N.+Patterson&amp;author=K.C.+Barnes&amp;author=N.+Rafaels&amp;volume=5&amp;publication_year=2009&amp;pages=e1000519&amp;pmid=19543370&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B61">61. <span class="mixed-citation">Tang H., Peng J., Wang P., Risch N.J. (2005) 
<span class="ref-title">Estimation of individual admixture: analytical and study design considerations</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">28</span>, 289–301. [<a href="/pubmed/15712363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Estimation+of+individual+admixture:+analytical+and+study+design+considerations&amp;author=H.+Tang&amp;author=J.+Peng&amp;author=P.+Wang&amp;author=N.J.+Risch&amp;volume=28&amp;publication_year=2005&amp;pages=289-301&amp;pmid=15712363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B62">62. <span class="mixed-citation">Tang H., Coram M., Wang P., Zhu X., Risch N. (2006) 
<span class="ref-title">Reconstructing genetic ancestry blocks in admixed individuals</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">79</span>, 1–12. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1474129/">PMC free article</a>]</span> [<a href="/pubmed/16773560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Reconstructing+genetic+ancestry+blocks+in+admixed+individuals&amp;author=H.+Tang&amp;author=M.+Coram&amp;author=P.+Wang&amp;author=X.+Zhu&amp;author=N.+Risch&amp;volume=79&amp;publication_year=2006&amp;pages=1-12&amp;pmid=16773560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B63">63. <span class="mixed-citation">Pritchard J.K., Stephens M., Donnelly P. (2000) 
<span class="ref-title">Inference of population structure using multilocus genotype data</span>. <span class="ref-journal">Genetics</span>, <span class="ref-vol">155</span>, 945–959. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=J.K.+Pritchard&amp;author=M.+Stephens&amp;author=P.+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B64">64. <span class="mixed-citation">Nalls M.A., Wilson J.G., Patterson N.J., Tandon A., Zmuda J.M., Huntsman S., Garcia M., Hu D., Li R., Beamer B.A., et al. (2008) 
<span class="ref-title">Admixture mapping of white cell count: genetic locus responsible for lower white blood cell count in the Health ABC and Jackson Heart studies. <em>Am.</em></span>
<span class="ref-journal">J. Hum. Genet</span>., <span class="ref-vol">82</span>, 81–87. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253985/">PMC free article</a>]</span> [<a href="/pubmed/18179887" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hum.+Genet&amp;title=Admixture+mapping+of+white+cell+count:+genetic+locus+responsible+for+lower+white+blood+cell+count+in+the+Health+ABC+and+Jackson+Heart+studies.+Am.&amp;author=M.A.+Nalls&amp;author=J.G.+Wilson&amp;author=N.J.+Patterson&amp;author=A.+Tandon&amp;author=J.M.+Zmuda&amp;volume=82&amp;publication_year=2008&amp;pages=81-87&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B65">65. <span class="mixed-citation">Pasaniuc B., Sankararaman S., Kimmel G., Halperin E. (2009) 
<span class="ref-title">Inference of locus-specific ancestry in closely related populations</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">25</span>, i213–i221. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687951/">PMC free article</a>]</span> [<a href="/pubmed/19477991" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Inference+of+locus-specific+ancestry+in+closely+related+populations&amp;author=B.+Pasaniuc&amp;author=S.+Sankararaman&amp;author=G.+Kimmel&amp;author=E.+Halperin&amp;volume=25&amp;publication_year=2009&amp;pages=i213-i221&amp;pmid=19477991&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B66">66. <span class="mixed-citation">Ward L.D., Kellis M. (2012) 
<span class="ref-title">HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</span>. <span class="ref-journal">Nucleic Acids Res</span>., <span class="ref-vol">40</span>, D930–D934. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245002/">PMC free article</a>]</span> [<a href="/pubmed/22064851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=HaploReg:+a+resource+for+exploring+chromatin+states,+conservation,+and+regulatory+motif+alterations+within+sets+of+genetically+linked+variants&amp;author=L.D.+Ward&amp;author=M.+Kellis&amp;volume=40&amp;publication_year=2012&amp;pages=D930-D934&amp;pmid=22064851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B67">67. <span class="mixed-citation">Dunham I., Kundaje A., Aldred S.F., Collins P.J., Davis C.A., Doyle F., Epstein C.B., Frietze S., Harrow J., Kaul R., et al. (2012) 
<span class="ref-title">An integrated encyclopedia of DNA elements in the human genome</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">489</span>, 57–74. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+encyclopedia+of+DNA+elements+in+the+human+genome&amp;author=I.+Dunham&amp;author=A.+Kundaje&amp;author=S.F.+Aldred&amp;author=P.J.+Collins&amp;author=C.A.+Davis&amp;volume=489&amp;publication_year=2012&amp;pages=57-74&amp;pmid=22955616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B68">68. <span class="mixed-citation">Bernstein B.E., Stamatoyannopoulos J.A., Costello J.F., Ren B., Milosavljevic A., Meissner A., Kellis M., Marra M.A., Beaudet A.L., Ecker J.R., et al. (2010) 
<span class="ref-title">The NIH Roadmap Epigenomics Mapping Consortium</span>. <span class="ref-journal">Nat. Biotechnol</span>., <span class="ref-vol">28</span>, 1045–1048. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3607281/">PMC free article</a>]</span> [<a href="/pubmed/20944595" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol&amp;title=The+NIH+Roadmap+Epigenomics+Mapping+Consortium&amp;author=B.E.+Bernstein&amp;author=J.A.+Stamatoyannopoulos&amp;author=J.F.+Costello&amp;author=B.+Ren&amp;author=A.+Milosavljevic&amp;volume=28&amp;publication_year=2010&amp;pages=1045-1048&amp;pmid=20944595&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B69">69. <span class="mixed-citation">Segre A.V., Groop L., Mootha V.K., Daly M.J., Altshuler D. (2010) 
<span class="ref-title">Common inherited variation in mitochondrial genes is not enriched for associations with type 2 diabetes or related glycemic traits</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">6</span>, e1001058. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2920848/">PMC free article</a>]</span> [<a href="/pubmed/20714348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Common+inherited+variation+in+mitochondrial+genes+is+not+enriched+for+associations+with+type+2+diabetes+or+related+glycemic+traits&amp;author=A.V.+Segre&amp;author=L.+Groop&amp;author=V.K.+Mootha&amp;author=M.J.+Daly&amp;author=D.+Altshuler&amp;volume=6&amp;publication_year=2010&amp;pages=e1001058&amp;pmid=20714348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B70">70. <span class="mixed-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al. (2007) 
<span class="ref-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">81</span>, 559–575. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B71">71. <span class="mixed-citation">Gauderman W.J. (2002) 
<span class="ref-title">Sample size calculations for matched case-control studies of gene-environment interaction</span>. <span class="ref-journal">Stat. Med</span>., <span class="ref-vol">21</span>, 35–50. [<a href="/pubmed/11782049" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Med&amp;title=Sample+size+calculations+for+matched+case-control+studies+of+gene-environment+interaction&amp;author=W.J.+Gauderman&amp;volume=21&amp;publication_year=2002&amp;pages=35-50&amp;pmid=11782049&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B72">72. <span class="mixed-citation">Pruim R.J., Welch R.P., Sanna S., Teslovich T.M., Chines P.S., Gliedt T.P., Boehnke M., Abecasis G.R., Willer C.J. (2010) 
<span class="ref-title">LocusZoom: regional visualization of genome-wide association scan results</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">26</span>, 2336–2337. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=R.J.+Pruim&amp;author=R.P.+Welch&amp;author=S.+Sanna&amp;author=T.M.+Teslovich&amp;author=P.S.+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B73">73. <span class="mixed-citation">Hinch A.G., Tandon A., Patterson N., Song Y., Rohland N., Palmer C.D., Chen G.K., Wang K., Buxbaum S.G., Akylbekova E.L., et al. (2011) 
<span class="ref-title">The landscape of recombination in African Americans</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">476</span>, 170–175. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3154982/">PMC free article</a>]</span> [<a href="/pubmed/21775986" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+landscape+of+recombination+in+African+Americans&amp;author=A.G.+Hinch&amp;author=A.+Tandon&amp;author=N.+Patterson&amp;author=Y.+Song&amp;author=N.+Rohland&amp;volume=476&amp;publication_year=2011&amp;pages=170-175&amp;pmid=21775986&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B74">74. <span class="mixed-citation">Barrett J.C., Fry B., Maller J., Daly M.J. (2005) 
<span class="ref-title">Haploview: analysis and visualization of LD and haplotype maps</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">21</span>, 263–265. [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=J.C.+Barrett&amp;author=B.+Fry&amp;author=J.+Maller&amp;author=M.J.+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B75">75. <span class="mixed-citation">Gabriel S.B., Schaffner S.F., Nguyen H., Moore J.M., Roy J., Blumenstiel B., Higgins J., DeFelice M., Lochner A., Faggart M., et al. (2002) 
<span class="ref-title">The structure of haplotype blocks in the human genome</span>. <span class="ref-journal">Science</span>, <span class="ref-vol">296</span>, 2225–2229. [<a href="/pubmed/12029063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+structure+of+haplotype+blocks+in+the+human+genome&amp;author=S.B.+Gabriel&amp;author=S.F.+Schaffner&amp;author=H.+Nguyen&amp;author=J.M.+Moore&amp;author=J.+Roy&amp;volume=296&amp;publication_year=2002&amp;pages=2225-2229&amp;pmid=12029063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B76">76. <span class="mixed-citation">Wellcome Trust Case Control C., Maller J.B., McVean G., Byrnes J., Vukcevic D., Palin K., Su Z., Howson J.M., Auton A., Myers S., et al. (2012) 
<span class="ref-title">Bayesian refinement of association signals for 14 loci in 3 common diseases</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">44</span>, 1294–1301. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3791416/">PMC free article</a>]</span> [<a href="/pubmed/23104008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Bayesian+refinement+of+association+signals+for+14+loci+in+3+common+diseases&amp;author=C.+Wellcome+Trust+Case+Control&amp;author=J.B.+Maller&amp;author=G.+McVean&amp;author=J.+Byrnes&amp;author=D.+Vukcevic&amp;volume=44&amp;publication_year=2012&amp;pages=1294-1301&amp;pmid=23104008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B77">77. <span class="mixed-citation">Asimit J.L., Hatzikotoulas K., McCarthy M., Morris A.P., Zeggini E. (2016) 
<span class="ref-title">Trans-ethnic study design approaches for fine-mapping</span>. <span class="ref-journal">Eur. J. Hum. Genet</span>., <span class="ref-vol">24</span>(<span class="ref-iss">9</span>):1330–1336. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4856879/">PMC free article</a>]</span> [<a href="/pubmed/26839038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet&amp;title=Trans-ethnic+study+design+approaches+for+fine-mapping&amp;author=J.L.+Asimit&amp;author=K.+Hatzikotoulas&amp;author=M.+McCarthy&amp;author=A.P.+Morris&amp;author=E.+Zeggini&amp;volume=24&amp;issue=9&amp;publication_year=2016&amp;pages=1330-1336&amp;pmid=26839038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B78">78. <span class="mixed-citation">(2010) 
<span class="ref-title">A map of human genome variation from population-scale sequencing</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">467</span>, 1061–1073. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B79">79. <span class="mixed-citation">Liu E.Y., Buyske S., Aragaki A.K., Peters U., Boerwinkle E., Carlson C., Carty C., Crawford D.C., Haessler J., Hindorff L.A., et al. (2012) 
<span class="ref-title">Genotype imputation of Metabochip SNPs using a study-specific reference panel of ∼4,000 haplotypes in African Americans from the Women's Health Initiative</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">36</span>, 107–117. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3410659/">PMC free article</a>]</span> [<a href="/pubmed/22851474" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Genotype+imputation+of+Metabochip+SNPs+using+a+study-specific+reference+panel+of+∼4,000+haplotypes+in+African+Americans+from+the+Women's+Health+Initiative&amp;author=E.Y.+Liu&amp;author=S.+Buyske&amp;author=A.K.+Aragaki&amp;author=U.+Peters&amp;author=E.+Boerwinkle&amp;volume=36&amp;publication_year=2012&amp;pages=107-117&amp;pmid=22851474&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw284-B80">80. <span class="mixed-citation">Leek J.T., Storey J.D. (2007) 
<span class="ref-title">Capturing heterogeneity in gene expression studies by surrogate variable analysis</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">3</span>, 1724–1735. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1994707/">PMC free article</a>]</span> [<a href="/pubmed/17907809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Capturing+heterogeneity+in+gene+expression+studies+by+surrogate+variable+analysis&amp;author=J.T.+Leek&amp;author=J.D.+Storey&amp;volume=3&amp;publication_year=2007&amp;pages=1724-1735&amp;pmid=17907809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm139953982941104title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Results</a></li><li><a href="#__sec9title">Discussion</a></li><li><a href="#__sec10title">Materials and Methods</a></li><li><a href="#__sec19title">Supplementary Material</a></li><li><a href="#idm139953975919312title">Acknowledgements</a></li><li><a href="#idm139953975863680title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5291202/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5291202/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5291202/pdf/ddw284.pdf">PDF (818K)</a></li> | <li><a href="#" data-citationid="PMC5291202" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5291202%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5291202%2F&amp;text=Fine-mapping%2C%20novel%20loci%20identification%2C%20and%20SNP%20association%20transferability%20in%20a%20genome-wide%20association%20study%20of%20QRS%20duration%20in%20African%20Americans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5291202%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27577874" data-aiid="5291202" data-aid="5291202" data-iid="286901" data-domainid="740" data-domain="hmg" data-accid="PMC5291202" data-md5="4fba68c5168a7bd6e5b9e3a72a840742">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-4" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-4" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input name="absid" id="absid" value="5291202" type="hidden"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:'#favUL',starExists:true,createCollectionUrl:'#',manageCollectionsUrl:'/myncbi/collections/'"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input class="button_apply" name="addtocollection" value="1" type="submit"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/21347284" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association studies of the PR interval in African Americans.</a><span class="source">[PLoS Genet. 2011]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-6" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Smith JG, Magnani JW, Palmer C, Meng YA, Soliman EZ, Musani SK, Kerr KF, Schnabel RB, Lubitz SA, Sotoodehnia N, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">PLoS Genet. 2011 Feb 10; 7(2):e1001304. Epub 2011 Feb 10.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/23534349" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Generalization of variants identified by genome-wide association studies for electrocardiographic traits in African Americans.</a><span class="source">[Ann Hum Genet. 2013]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Jeff JM, Ritchie MD, Denny JC, Kho AN, Ramirez AH, Crosslin D, Armstrong L, Basford MA, Wolf WA, Pacheco JA, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Ann Hum Genet. 2013 Jul; 77(4):321-32. Epub 2013 Mar 28.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/31251759" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">GWAS of QRS duration identifies new loci specific to Hispanic/Latino populations.</a><span class="source">[PLoS One. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Swenson BR, Louie T, Lin HJ, Méndez-Giráldez R, Below JE, Laurie CC, Kerr KF, Highland H, Thornton TA, Ryckman KK, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">PLoS One. 2019; 14(6):e0217796. Epub 2019 Jun 28.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/21715604" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A systematic review of replication studies of prostate cancer susceptibility genetic variants in high-risk men originally identified from genome-wide association studies.</a><span class="source">[Cancer Epidemiol Biomarkers Pr...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Ishak MB, Giri VN. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Cancer Epidemiol Biomarkers Prev. 2011 Aug; 20(8):1599-610. Epub 2011 Jun 29.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/30235722" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Associations between common ion channel single nucleotide polymorphisms and sudden cardiac death in adults: A MOOSE-compliant meta-analysis.</a><span class="source">[Medicine (Baltimore). 2018]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Liu X, Shi J, Xiao P. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Medicine (Baltimore). 2018 Sep; 97(38):e12428. </em>
</div>
</li>
</ul>
<a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=27577874&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=27577874&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-11" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-11" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6858543/" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Allele-specific NKX2-5 binding underlies multiple genetic associations with human electrocardiographic traits</a><span class="source">[Nature genetics. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-12" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Benaglio P, D’Antonio-Chronowska A, Ma W, Yang F, Young Greenwald WW, Donovan MK, DeBoever C, Li H, Drees F, Singhal S, Matsui H, van Setten J, Sotoodehnia N, Gaulton KJ, Smith EN, D’Antonio M, Rosenfeld MG, Frazer KA. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nature genetics. 2019 Sep 30; 51(10): 1506-1517</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6805929/" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Sequence variants with large effects on cardiac electrophysiology and disease</a><span class="source">[Nature Communications. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Norland K, Sveinbjornsson G, Thorolfsdottir RB, Davidsson OB, Tragante V, Rajamani S, Helgadottir A, Gretarsdottir S, van Setten J, Asselbergs FW, Sverrisson JT, Stephensen SS, Oskarsson G, Sigurdsson EL, Andersen K, Danielsen R, Thorgeirsson G, Thorsteinsdottir U, Arnar DO, Sulem P, Holm H, Gudbjartsson DF, Stefansson K. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nature Communications. 2019 Oct 22; 10: 4803</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6715539/" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Variation in a Left Ventricle-Specific Hand1 Enhancer Impairs GATA Transcription Factor Binding and Disrupts Conduction System Development and Function</a><span class="source">[Circulation research. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Vincentz JW, Firulli BA, Toolan KP, Arking DE, Sotoodehnia N, Wan J, Chen PS, de Gier-de Vries C, Christoffels VM, Rubart-von der Lohe M, Firulli AB. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Circulation research. 2019 Aug 1; 125(6): 575-589</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6599128/" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">GWAS of QRS duration identifies new loci specific to Hispanic/Latino populations</a><span class="source">[PLoS ONE. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Swenson BR, Louie T, Lin HJ, Méndez-Giráldez R, Below JE, Laurie CC, Kerr KF, Highland H, Thornton TA, Ryckman KK, Kooperberg C, Soliman EZ, Seyerle AA, Guo X, Taylor KD, Yao J, Heckbert SR, Darbar D, Petty LE, McKnight B, Cheng S, Bello NA, Whitsel EA, Hanis CL, Nalls MA, Evans DS, Rotter JI, Sofer T, Avery CL, Sotoodehnia N. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">PLoS ONE. 2019 Jun 28; 14(6): e0217796</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6082306/" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genomic approaches for the elucidation of genes and gene networks underlying cardiovascular traits</a><span class="source">[Biophysical Reviews. 2018]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Adriaens ME, Bezzina CR. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Biophysical Reviews. 2018 Jun 22; 10(4): 1053-1060</em>
</div>
</li>
</ul>
<a class="seemore" href="citedby/" ref="reftype=relart&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-17" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-17" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gap&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gap&amp;IdsFromResult=5291202" ref="reftype=dbGaP%20Links&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">dbGaP Links</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-18" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">dbGaP Links<div class="brieflinkpopdesc">dbGaP Links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gene&amp;IdsFromResult=5291202" ref="reftype=Gene&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Gene&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Gene</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Gene<div class="brieflinkpopdesc">Gene links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gene_nucleotide&amp;IdsFromResult=5291202" ref="reftype=Gene%20(nucleotide)&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Gene (nucleotide)</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Gene (nucleotide)<div class="brieflinkpopdesc">Records in Gene identified from shared sequence links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=geoprofiles&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_geoprofiles&amp;IdsFromResult=5291202" ref="reftype=GEO%20Profiles&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">GEO Profiles</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-21" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">GEO Profiles<div class="brieflinkpopdesc">Related GEO records</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=5291202" ref="reftype=MedGen&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-22" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=nuccore&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_nuccore&amp;IdsFromResult=5291202" ref="reftype=Nucleotide&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Nucleotide&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Nucleotide</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-23" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Nucleotide<div class="brieflinkpopdesc">Published Nucleotide sequences</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pubmed/27577874/" ref="reftype=PubMed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-24" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=snp&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_snp&amp;IdsFromResult=5291202" ref="reftype=SNP&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|SNP&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">SNP</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-25" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">SNP<div class="brieflinkpopdesc">PMC to SNP links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=5291202" ref="reftype=Taxonomy&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-26" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-27" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-27" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5dd59ba2dd2142621d4863d0">Fine-mapping, novel loci identification, and SNP association transferability in ...</a><div class="ralinkpop offscreen_noflow">Fine-mapping, novel loci identification, and SNP association transferability in a genome-wide association study of QRS duration in African Americans<div class="brieflinkpopdesc">Human Molecular Genetics. 2016 Oct 1; 25(19)4350</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=5dd59b69dd2142621d3fb8eb">A Genome-Wide Association Study to Identify Single-Nucleotide Polymorphisms for ...</a><div class="ralinkpop offscreen_noflow">A Genome-Wide Association Study to Identify Single-Nucleotide Polymorphisms for Acute Kidney Injury<div class="brieflinkpopdesc">American Journal of Respiratory and Critical Care Medicine. 2017 Feb 15; 195(4)482</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=5dd59b40dd2142621d39372c">A Genome-wide Approach Validates that Thiopurine Methyltransferase Activity is a...</a><div class="ralinkpop offscreen_noflow">A Genome-wide Approach Validates that Thiopurine Methyltransferase Activity is a Monogenic Pharmacogenomic Trait<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2017 Mar; 101(3)373</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=5dd59b2175f66d018f41da9d">A genome-wide association meta-analysis of diarrhoeal disease in young children ...</a><div class="ralinkpop offscreen_noflow">A genome-wide association meta-analysis of diarrhoeal disease in young children identifies FUT2 locus and provides plausible biological pathways<div class="brieflinkpopdesc">Human Molecular Genetics. 2016 Sep 15; 25(18)4127</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=5dd59af275f66d018f3b7daf">Genome-Wide Gene-Environment Interaction Analysis of Pesticide Exposure and Risk...</a><div class="ralinkpop offscreen_noflow">Genome-Wide Gene-Environment Interaction Analysis of Pesticide Exposure and Risk of Parkinson’s Disease<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2016 Nov; 32()25</div>
</div>
<div class="tertiary"></div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p>
<a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p3" name="crb--__p3" rid="__p3" style="position: absolute; visibility: visible; width: 370px; top: 1275px; display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-28"><ul>
<li class="two_line" reference_id="ddw284-B1">
<a href="/pubmed/23726176/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">QRS duration on electrocardiography and cardiovascular mortality (from the National Health and Nutrition Examination Survey-III).</a><span class="one_line_source">[Am J Cardiol.  2013]</span><div class="alt-note">
<div class="authors">Badheka AO, Singh V, Patel NJ, Deshmukh A, Shah N, Chothani A, Mehta K, Grover P, Savani GT, Gupta S, Rathod A, Marzouka GR, Mitrani RD, Moscucci M, Cohen MG</div>
<div class="citation">Am J Cardiol. 2013 Sep 1; 112(5):671-7.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B2">
<a href="/pubmed/25550439/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Ventricular conduction and long-term heart failure outcomes and mortality in African Americans: insights from the Jackson Heart Study.</a><span class="one_line_source">[Circ Heart Fail.  2015]</span><div class="alt-note">
<div class="authors">Mentz RJ, Greiner MA, DeVore AD, Dunlay SM, Choudhary G, Ahmad T, Khazanie P, Randolph TC, Griswold ME, Eapen ZJ, O'Brien EC, Thomas KL, Curtis LH, Hernandez AF</div>
<div class="citation">Circ Heart Fail. 2015 Mar; 8(2):243-51.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B3">
<a href="/pubmed/18544725/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Clinical implications of QRS duration in patients hospitalized with worsening heart failure and reduced left ventricular ejection fraction.</a><span class="one_line_source">[JAMA.  2008]</span><div class="alt-note">
<div class="authors">Wang NC, Maggioni AP, Konstam MA, Zannad F, Krasa HB, Burnett JC Jr, Grinfeld L, Swedberg K, Udelson JE, Cook T, Traver B, Zimmer C, Orlandi C, Gheorghiade M, Efficacy of Vasopressin Antagonism in Heart Failure Outcome Study With Tolvaptan (EVEREST) Investigators.</div>
<div class="citation">JAMA. 2008 Jun 11; 299(22):2656-66.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B4">
<a href="/pubmed/9485136/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Electrocardiographic findings in a healthy biracial population. Atherosclerosis Risk in Communities (ARIC) Study Investigators.</a><span class="one_line_source">[Am J Cardiol.  1998]</span><div class="alt-note">
<div class="authors">Vitelli LL, Crow RS, Shahar E, Hutchinson RG, Rautaharju PM, Folsom AR</div>
<div class="citation">Am J Cardiol. 1998 Feb 15; 81(4):453-9.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p4" name="crb--__p4" rid="__p4" style="position: absolute; visibility: visible; width: 370px; top: 1462px; display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-29"><ul>
<li class="two_line" reference_id="ddw284-B7">
<a href="/pubmed/20062063/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Several common variants modulate heart rate, PR interval and QRS duration.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njølstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Løchen ML, Kong A, Thorsteinsdottir U, Stefansson K</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):117-22.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B9">
<a href="/pubmed/20395969/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Genome-wide association studies in diverse populations.</a><span class="one_line_source">[Nat Rev Genet.  2010]</span><div class="alt-note">
<div class="authors">Rosenberg NA, Huang L, Jewett EM, Szpiech ZA, Jankovic I, Boehnke M</div>
<div class="citation">Nat Rev Genet. 2010 May; 11(5):356-66.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B10">
<a href="/pubmed/21253569/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association study SNPs in the human genome diversity project populations: does selection affect unlinked SNPs with shared trait associations?</a><span class="one_line_source">[PLoS Genet.  2011]</span><div class="alt-note">
<div class="authors">Casto AM, Feldman MW</div>
<div class="citation">PLoS Genet. 2011 Jan 6; 7(1):e1001266.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p6" name="crb--__p6" rid="__p6" style="position: absolute; visibility: visible; width: 370px; top: 1918px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-30"><ul>
<li class="two_line" reference_id="ddw284-B11">
<a href="/pubmed/21836165/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The Next PAGE in understanding complex traits: design for the analysis of Population Architecture Using Genetics and Epidemiology (PAGE) Study.</a><span class="one_line_source">[Am J Epidemiol.  2011]</span><div class="alt-note">
<div class="authors">Matise TC, Ambite JL, Buyske S, Carlson CS, Cole SA, Crawford DC, Haiman CA, Heiss G, Kooperberg C, Marchand LL, Manolio TA, North KE, Peters U, Ritchie MD, Hindorff LA, Haines JL, PAGE Study.</div>
<div class="citation">Am J Epidemiol. 2011 Oct 1; 174(7):849-59.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B12">
<a href="/pubmed/12193783/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Variant of SCN5A sodium channel implicated in risk of cardiac arrhythmia.</a><span class="one_line_source">[Science.  2002]</span><div class="alt-note">
<div class="authors">Splawski I, Timothy KW, Tateyama M, Clancy CE, Malhotra A, Beggs AH, Cappuccio FP, Sagnella GA, Kass RS, Keating MT</div>
<div class="citation">Science. 2002 Aug 23; 297(5585):1333-6.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p18" name="crb--__p18" rid="__p18" style="position: absolute; visibility: visible; width: 370px; top: 4483px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-31"><ul>
<li class="two_line" reference_id="ddw284-B7">
<a href="/pubmed/20062063/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Several common variants modulate heart rate, PR interval and QRS duration.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njølstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Løchen ML, Kong A, Thorsteinsdottir U, Stefansson K</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):117-22.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B16">
<a href="/pubmed/20062061/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetic variation in SCN10A influences cardiac conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Chambers JC, Zhao J, Terracciano CM, Bezzina CR, Zhang W, Kaba R, Navaratnarajah M, Lotlikar A, Sehmi JS, Kooner MK, Deng G, Siedlecka U, Parasramka S, El-Hamamsy I, Wass MN, Dekker LR, de Jong JS, Sternberg MJ, McKenna W, Severs NJ, de Silva R, Wilde AA, Anand P, Yacoub M, Scott J, Elliott P, Wood JN, Kooner JS</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):149-52.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p31" name="crb--__p31" rid="__p31" style="position: absolute; visibility: visible; width: 370px; top: 6957px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-32"><ul><li class="two_line" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p40" name="crb--__p40" rid="__p40" style="position: absolute; visibility: visible; width: 370px; top: 9808px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-33"><ul>
<li class="two_line" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B7">
<a href="/pubmed/20062063/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Several common variants modulate heart rate, PR interval and QRS duration.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njølstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Løchen ML, Kong A, Thorsteinsdottir U, Stefansson K</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):117-22.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p50" name="crb--__p50" rid="__p50" style="position: absolute; visibility: visible; width: 370px; top: 11404px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-34"><ul>
<li class="two_line" reference_id="ddw284-B7">
<a href="/pubmed/20062063/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Several common variants modulate heart rate, PR interval and QRS duration.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njølstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Løchen ML, Kong A, Thorsteinsdottir U, Stefansson K</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):117-22.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B16">
<a href="/pubmed/20062061/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetic variation in SCN10A influences cardiac conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Chambers JC, Zhao J, Terracciano CM, Bezzina CR, Zhang W, Kaba R, Navaratnarajah M, Lotlikar A, Sehmi JS, Kooner MK, Deng G, Siedlecka U, Parasramka S, El-Hamamsy I, Wass MN, Dekker LR, de Jong JS, Sternberg MJ, McKenna W, Severs NJ, de Silva R, Wilde AA, Anand P, Yacoub M, Scott J, Elliott P, Wood JN, Kooner JS</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):149-52.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p52" name="crb--__p52" rid="__p52" style="position: absolute; visibility: visible; width: 370px; top: 11775px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-35"><ul>
<li class="two_line" reference_id="ddw284-B7">
<a href="/pubmed/20062063/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Several common variants modulate heart rate, PR interval and QRS duration.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njølstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Løchen ML, Kong A, Thorsteinsdottir U, Stefansson K</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):117-22.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p54" name="crb--__p54" rid="__p54" style="position: absolute; visibility: visible; width: 370px; top: 12174px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-36"><ul><li class="two_line" reference_id="ddw284-B17">
<a href="/pubmed/22125221/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Transethnic meta-analysis of genomewide association studies.</a><span class="one_line_source">[Genet Epidemiol.  2011]</span><div class="alt-note">
<div class="authors">Morris AP</div>
<div class="citation">Genet Epidemiol. 2011 Dec; 35(8):809-22.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p56" name="crb--__p56" rid="__p56" style="position: absolute; visibility: visible; width: 370px; top: 12568px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-37"><ul>
<li class="two_line" reference_id="ddw284-B18">
<a href="/pubmed/20398673/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Structure and function of splice variants of the cardiac voltage-gated sodium channel Na(v)1.5.</a><span class="one_line_source">[J Mol Cell Cardiol.  2010]</span><div class="alt-note">
<div class="authors">Schroeter A, Walzik S, Blechschmidt S, Haufe V, Benndorf K, Zimmer T</div>
<div class="citation">J Mol Cell Cardiol. 2010 Jul; 49(1):16-24.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B7">
<a href="/pubmed/20062063/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Several common variants modulate heart rate, PR interval and QRS duration.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njølstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Løchen ML, Kong A, Thorsteinsdottir U, Stefansson K</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):117-22.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B19">
<a href="/pubmed/19027780/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> SCN5A channelopathies--an update on mutations and mechanisms.</a><span class="one_line_source">[Prog Biophys Mol Biol.  2008]</span><div class="alt-note">
<div class="authors">Zimmer T, Surber R</div>
<div class="citation">Prog Biophys Mol Biol. 2008 Oct-Nov; 98(2-3):120-36.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B20">
<a href="/pubmed/21347284/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association studies of the PR interval in African Americans.</a><span class="one_line_source">[PLoS Genet.  2011]</span><div class="alt-note">
<div class="authors">Smith JG, Magnani JW, Palmer C, Meng YA, Soliman EZ, Musani SK, Kerr KF, Schnabel RB, Lubitz SA, Sotoodehnia N, Redline S, Pfeufer A, Müller M, Evans DS, Nalls MA, Liu Y, Newman AB, Zonderman AB, Evans MK, Deo R, Ellinor PT, Paltoo DN, Newton-Cheh C, Benjamin EJ, Mehra R, Alonso A, Heckbert SR, Fox ER, Candidate-gene Association Resource (CARe) Consortium.</div>
<div class="citation">PLoS Genet. 2011 Feb 10; 7(2):e1001304.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B21" style="display: none;">
<a href="/pubmed/23139255/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Novel loci associated with PR interval in a genome-wide association study of 10 African American cohorts.</a><span class="one_line_source">[Circ Cardiovasc Genet.  2012]</span><div class="alt-note">
<div class="authors">Butler AM, Yin X, Evans DS, Nalls MA, Smith EN, Tanaka T, Li G, Buxbaum SG, Whitsel EA, Alonso A, Arking DE, Benjamin EJ, Berenson GS, Bis JC, Chen W, Deo R, Ellinor PT, Heckbert SR, Heiss G, Hsueh WC, Keating BJ, Kerr KF, Li Y, Limacher MC, Liu Y, Lubitz SA, Marciante KD, Mehra R, Meng YA, Newman AB, Newton-Cheh C, North KE, Palmer CD, Psaty BM, Quibrera PM, Redline S, Reiner AP, Rotter JI, Schnabel RB, Schork NJ, Singleton AB, Smith JG, Soliman EZ, Srinivasan SR, Zhang ZM, Zonderman AB, Ferrucci L, Murray SS, Evans MK, Sotoodehnia N, Magnani JW, Avery CL</div>
<div class="citation">Circ Cardiovasc Genet. 2012 Dec; 5(6):639-46.</div>
</div>
</li>
</ul><a href="" class="seemore">See more ...</a></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p57" name="crb--__p57" rid="__p57" style="position: absolute; visibility: visible; width: 370px; top: 12800px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-38"><ul>
<li class="two_line" reference_id="ddw284-B22">
<a href="/pubmed/22771987/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Transcription regulation by distal enhancers: who's in the loop?</a><span class="one_line_source">[Transcription.  2012]</span><div class="alt-note">
<div class="authors">Stadhouders R, van den Heuvel A, Kolovos P, Jorna R, Leslie K, Grosveld F, Soler E</div>
<div class="citation">Transcription. 2012 Jul-Aug; 3(4):181-6.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B18">
<a href="/pubmed/20398673/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Structure and function of splice variants of the cardiac voltage-gated sodium channel Na(v)1.5.</a><span class="one_line_source">[J Mol Cell Cardiol.  2010]</span><div class="alt-note">
<div class="authors">Schroeter A, Walzik S, Blechschmidt S, Haufe V, Benndorf K, Zimmer T</div>
<div class="citation">J Mol Cell Cardiol. 2010 Jul; 49(1):16-24.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p59" name="crb--__p59" rid="__p59" style="position: absolute; visibility: visible; width: 370px; top: 13242px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-39"><ul>
<li class="two_line" reference_id="ddw284-B23">
<a href="/pubmed/21126233/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Myosin regulatory light chains are required to maintain the stability of myosin II and cellular integrity.</a><span class="one_line_source">[Biochem J.  2011]</span><div class="alt-note">
<div class="authors">Park I, Han C, Jin S, Lee B, Choi H, Kwon JT, Kim D, Kim J, Lifirsu E, Park WJ, Park ZY, Kim DH, Cho C</div>
<div class="citation">Biochem J. 2011 Feb 15; 434(1):171-80.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B24">
<a href="/pubmed/15075390/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A gene atlas of the mouse and human protein-encoding transcriptomes.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2004]</span><div class="alt-note">
<div class="authors">Su AI, Wiltshire T, Batalov S, Lapp H, Ching KA, Block D, Zhang J, Soden R, Hayakawa M, Kreiman G, Cooke MP, Walker JR, Hogenesch JB</div>
<div class="citation">Proc Natl Acad Sci U S A. 2004 Apr 20; 101(16):6062-7.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B7">
<a href="/pubmed/20062063/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Several common variants modulate heart rate, PR interval and QRS duration.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njølstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Løchen ML, Kong A, Thorsteinsdottir U, Stefansson K</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):117-22.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B25">
<a href="/pubmed/20639392/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association analysis identifies multiple loci related to resting heart rate.</a><span class="one_line_source">[Hum Mol Genet.  2010]</span><div class="alt-note">
<div class="authors">Eijgelsheim M, Newton-Cheh C, Sotoodehnia N, de Bakker PI, Müller M, Morrison AC, Smith AV, Isaacs A, Sanna S, Dörr M, Navarro P, Fuchsberger C, Nolte IM, de Geus EJ, Estrada K, Hwang SJ, Bis JC, Rückert IM, Alonso A, Launer LJ, Hottenga JJ, Rivadeneira F, Noseworthy PA, Rice KM, Perz S, Arking DE, Spector TD, Kors JA, Aulchenko YS, Tarasov KV, Homuth G, Wild SH, Marroni F, Gieger C, Licht CM, Prineas RJ, Hofman A, Rotter JI, Hicks AA, Ernst F, Najjar SS, Wright AF, Peters A, Fox ER, Oostra BA, Kroemer HK, Couper D, Völzke H, Campbell H, Meitinger T, Uda M, Witteman JC, Psaty BM, Wichmann HE, Harris TB, Kääb S, Siscovick DS, Jamshidi Y, Uitterlinden AG, Folsom AR, Larson MG, Wilson JF, Penninx BW, Snieder H, Pramstaller PP, van Duijn CM, Lakatta EG, Felix SB, Gudnason V, Pfeufer A, Heckbert SR, Stricker BH, Boerwinkle E, O'Donnell CJ</div>
<div class="citation">Hum Mol Genet. 2010 Oct 1; 19(19):3885-94.</div>
</div>
</li>
<li class="two_line" reference_id="ddw284-B26">
<a href="/pubmed/23183192/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common genetic variation near the connexin-43 gene is associated with resting heart rate in African Americans: a genome-wide association study of 13,372 participants.</a><span class="one_line_source">[Heart Rhythm.  2013]</span><div class="alt-note">
<div class="authors">Deo R, Nalls MA, Avery CL, Smith JG, Evans DS, Keller MF, Butler AM, Buxbaum SG, Li G, Miguel Quibrera P, Smith EN, Tanaka T, Akylbekova EL, Alonso A, Arking DE, Benjamin EJ, Berenson GS, Bis JC, Chen LY, Chen W, Cummings SR, Ellinor PT, Evans MK, Ferrucci L, Fox ER, Heckbert SR, Heiss G, Hsueh WC, Kerr KF, Limacher MC, Liu Y, Lubitz SA, Magnani JW, Mehra R, Marcus GM, Murray SS, Newman AB, Njajou O, North KE, Paltoo DN, Psaty BM, Redline SS, Reiner AP, Robinson JG, Rotter JI, Samdarshi TE, Schnabel RB, Schork NJ, Singleton AB, Siscovick D, Soliman EZ, Sotoodehnia N, Srinivasan SR, Taylor HA, Trevisan M, Zhang Z, Zonderman AB, Newton-Cheh C, Whitsel EA</div>
<div class="citation">Heart Rhythm. 2013 Mar; 10(3):401-8.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p60" name="crb--__p60" rid="__p60">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-40"><ul>
<li class="expanded" reference_id="ddw284-B27">
<a href="/pubmed/19477319/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Antigen Presentation by CD1 Lipids, T Cells, and NKT Cells in Microbial Immunity.</a><span class="one_line_source">[Adv Immunol.  2009]</span><div class="alt-note">
<div class="authors">Cohen NR, Garg S, Brenner MB</div>
<div class="citation">Adv Immunol. 2009; 102():1-94.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B28">
<a href="/pubmed/14983052/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The human olfactory receptor gene family.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2004]</span><div class="alt-note">
<div class="authors">Malnic B, Godfrey PA, Buck LB</div>
<div class="citation">Proc Natl Acad Sci U S A. 2004 Feb 24; 101(8):2584-9.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B29">
<a href="/pubmed/22586405/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Defects in cytoskeletal signaling pathways, arrhythmia, and sudden cardiac death.</a><span class="one_line_source">[Front Physiol.  2012]</span><div class="alt-note">
<div class="authors">Smith S, Curran J, Hund TJ, Mohler PJ</div>
<div class="citation">Front Physiol. 2012; 3():122.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B30">
<a href="/pubmed/15579534/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Nav1.5 E1053K mutation causing Brugada syndrome blocks binding to ankyrin-G and expression of Nav1.5 on the surface of cardiomyocytes.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2004]</span><div class="alt-note">
<div class="authors">Mohler PJ, Rivolta I, Napolitano C, LeMaillet G, Lambert S, Priori SG, Bennett V</div>
<div class="citation">Proc Natl Acad Sci U S A. 2004 Dec 14; 101(50):17533-8.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B31">
<a href="/pubmed/20877009/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A β(IV)-spectrin/CaMKII signaling complex is essential for membrane excitability in mice.</a><span class="one_line_source">[J Clin Invest.  2010]</span><div class="alt-note">
<div class="authors">Hund TJ, Koval OM, Li J, Wright PJ, Qian L, Snyder JS, Gudmundsson H, Kline CF, Davidson NP, Cardona N, Rasband MN, Anderson ME, Mohler PJ</div>
<div class="citation">J Clin Invest. 2010 Oct; 120(10):3508-19.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p62" name="crb--__p62" rid="__p62">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-41"><ul>
<li class="expanded" reference_id="ddw284-B9">
<a href="/pubmed/20395969/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Genome-wide association studies in diverse populations.</a><span class="one_line_source">[Nat Rev Genet.  2010]</span><div class="alt-note">
<div class="authors">Rosenberg NA, Huang L, Jewett EM, Szpiech ZA, Jankovic I, Boehnke M</div>
<div class="citation">Nat Rev Genet. 2010 May; 11(5):356-66.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B32">
<a href="/pubmed/20084087/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Methodological challenges of genome-wide association analysis in Africa.</a><span class="one_line_source">[Nat Rev Genet.  2010]</span><div class="alt-note">
<div class="authors">Teo YY, Small KS, Kwiatkowski DP</div>
<div class="citation">Nat Rev Genet. 2010 Feb; 11(2):149-60.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B10">
<a href="/pubmed/21253569/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association study SNPs in the human genome diversity project populations: does selection affect unlinked SNPs with shared trait associations?</a><span class="one_line_source">[PLoS Genet.  2011]</span><div class="alt-note">
<div class="authors">Casto AM, Feldman MW</div>
<div class="citation">PLoS Genet. 2011 Jan 6; 7(1):e1001266.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B33">
<a href="/pubmed/21730125/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Demographic history and rare allele sharing among human populations.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2011]</span><div class="alt-note">
<div class="authors">Gravel S, Henn BM, Gutenkunst RN, Indap AR, Marth GT, Clark AG, Yu F, Gibbs RA, 1000 Genomes Project., Bustamante CD</div>
<div class="citation">Proc Natl Acad Sci U S A. 2011 Jul 19; 108(29):11983-8.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B37">
<a href="/pubmed/20126254/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Rare variants create synthetic genome-wide associations.</a><span class="one_line_source">[PLoS Biol.  2010]</span><div class="alt-note">
<div class="authors">Dickson SP, Wang K, Krantz I, Hakonarson H, Goldstein DB</div>
<div class="citation">PLoS Biol. 2010 Jan 26; 8(1):e1000294.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B38">
<a href="/pubmed/21267062/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Synthetic associations are unlikely to account for many common disease genome-wide association signals.</a><span class="one_line_source">[PLoS Biol.  2011]</span><div class="alt-note">
<div class="authors">Anderson CA, Soranzo N, Zeggini E, Barrett JC</div>
<div class="citation">PLoS Biol. 2011 Jan 18; 9(1):e1000580.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B39">
<a href="/pubmed/21267061/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Synthetic associations created by rare variants do not explain most GWAS results.</a><span class="one_line_source">[PLoS Biol.  2011]</span><div class="alt-note">
<div class="authors">Wray NR, Purcell SM, Visscher PM</div>
<div class="citation">PLoS Biol. 2011 Jan 18; 9(1):e1000579.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p63" name="crb--__p63" rid="__p63">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-42"><ul>
<li class="expanded" reference_id="ddw284-B40">
<a href="/pubmed/20865176/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Consistent association of type 2 diabetes risk variants found in europeans in diverse racial and ethnic groups.</a><span class="one_line_source">[PLoS Genet.  2010]</span><div class="alt-note">
<div class="authors">Waters KM, Stram DO, Hassanein MT, Le Marchand L, Wilkens LR, Maskarinec G, Monroe KR, Kolonel LN, Altshuler D, Henderson BE, Haiman CA</div>
<div class="citation">PLoS Genet. 2010 Aug 26; 6(8):.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B44">
<a href="/pubmed/21738485/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetic determinants of lipid traits in diverse populations from the population architecture using genomics and epidemiology (PAGE) study.</a><span class="one_line_source">[PLoS Genet.  2011]</span><div class="alt-note">
<div class="authors">Dumitrescu L, Carty CL, Taylor K, Schumacher FR, Hindorff LA, Ambite JL, Anderson G, Best LG, Brown-Gentry K, Bůžková P, Carlson CS, Cochran B, Cole SA, Devereux RB, Duggan D, Eaton CB, Fornage M, Franceschini N, Haessler J, Howard BV, Johnson KC, Laston S, Kolonel LN, Lee ET, MacCluer JW, Manolio TA, Pendergrass SA, Quibrera M, Shohet RV, Wilkens LR, Haiman CA, Le Marchand L, Buyske S, Kooperberg C, North KE, Crawford DC</div>
<div class="citation">PLoS Genet. 2011 Jun; 7(6):e1002138.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B45">
<a href="/pubmed/23726366/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide characterization of shared and distinct genetic components that influence blood lipid levels in ethnically diverse human populations.</a><span class="one_line_source">[Am J Hum Genet.  2013]</span><div class="alt-note">
<div class="authors">Coram MA, Duan Q, Hoffmann TJ, Thornton T, Knowles JW, Johnson NA, Ochs-Balcom HM, Donlon TA, Martin LW, Eaton CB, Robinson JG, Risch NJ, Zhu X, Kooperberg C, Li Y, Reiner AP, Tang H</div>
<div class="citation">Am J Hum Genet. 2013 Jun 6; 92(6):904-16.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B46">
<a href="/pubmed/23555291/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Trans-ethnic fine-mapping of lipid loci identifies population-specific signals and allelic heterogeneity that increases the trait variance explained.</a><span class="one_line_source">[PLoS Genet.  2013]</span><div class="alt-note">
<div class="authors">Wu Y, Waite LL, Jackson AU, Sheu WH, Buyske S, Absher D, Arnett DK, Boerwinkle E, Bonnycastle LL, Carty CL, Cheng I, Cochran B, Croteau-Chonka DC, Dumitrescu L, Eaton CB, Franceschini N, Guo X, Henderson BE, Hindorff LA, Kim E, Kinnunen L, Komulainen P, Lee WJ, Le Marchand L, Lin Y, Lindström J, Lingaas-Holmen O, Mitchell SL, Narisu N, Robinson JG, Schumacher F, Stančáková A, Sundvall J, Sung YJ, Swift AJ, Wang WC, Wilkens L, Wilsgaard T, Young AM, Adair LS, Ballantyne CM, Bůžková P, Chakravarti A, Collins FS, Duggan D, Feranil AB, Ho LT, Hung YJ, Hunt SC, Hveem K, Juang JM, Kesäniemi AY, Kuusisto J, Laakso M, Lakka TA, Lee IT, Leppert MF, Matise TC, Moilanen L, Njølstad I, Peters U, Quertermous T, Rauramaa R, Rotter JI, Saramies J, Tuomilehto J, Uusitupa M, Wang TD, Boehnke M, Haiman CA, Chen YD, Kooperberg C, Assimes TL, Crawford DC, Hsiung CA, North KE, Mohlke KL</div>
<div class="citation">PLoS Genet. 2013 Mar; 9(3):e1003379.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B47">
<a href="/pubmed/23785302/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">High trans-ethnic replicability of GWAS results implies common causal variants.</a><span class="one_line_source">[PLoS Genet.  2013]</span><div class="alt-note">
<div class="authors">Marigorta UM, Navarro A</div>
<div class="citation">PLoS Genet. 2013 Jun; 9(6):e1003566.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B43">
<a href="/pubmed/24068893/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Generalization and dilution of association results from European GWAS in populations of non-European ancestry: the PAGE study.</a><span class="one_line_source">[PLoS Biol.  2013]</span><div class="alt-note">
<div class="authors">Carlson CS, Matise TC, North KE, Haiman CA, Fesinmeyer MD, Buyske S, Schumacher FR, Peters U, Franceschini N, Ritchie MD, Duggan DJ, Spencer KL, Dumitrescu L, Eaton CB, Thomas F, Young A, Carty C, Heiss G, Le Marchand L, Crawford DC, Hindorff LA, Kooperberg CL, PAGE Consortium.</div>
<div class="citation">PLoS Biol. 2013 Sep; 11(9):e1001661.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B13">
<a href="/pubmed/21325150/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">SCN5A variation is associated with electrocardiographic traits in the Jackson Heart Study.</a><span class="one_line_source">[Circ Cardiovasc Genet.  2011]</span><div class="alt-note">
<div class="authors">Jeff JM, Brown-Gentry K, Buxbaum SG, Sarpong DF, Taylor HA, George AL Jr, Roden DM, Crawford DC</div>
<div class="citation">Circ Cardiovasc Genet. 2011 Apr; 4(2):139-44.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B20">
<a href="/pubmed/21347284/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association studies of the PR interval in African Americans.</a><span class="one_line_source">[PLoS Genet.  2011]</span><div class="alt-note">
<div class="authors">Smith JG, Magnani JW, Palmer C, Meng YA, Soliman EZ, Musani SK, Kerr KF, Schnabel RB, Lubitz SA, Sotoodehnia N, Redline S, Pfeufer A, Müller M, Evans DS, Nalls MA, Liu Y, Newman AB, Zonderman AB, Evans MK, Deo R, Ellinor PT, Paltoo DN, Newton-Cheh C, Benjamin EJ, Mehra R, Alonso A, Heckbert SR, Fox ER, Candidate-gene Association Resource (CARe) Consortium.</div>
<div class="citation">PLoS Genet. 2011 Feb 10; 7(2):e1001304.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B21">
<a href="/pubmed/23139255/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Novel loci associated with PR interval in a genome-wide association study of 10 African American cohorts.</a><span class="one_line_source">[Circ Cardiovasc Genet.  2012]</span><div class="alt-note">
<div class="authors">Butler AM, Yin X, Evans DS, Nalls MA, Smith EN, Tanaka T, Li G, Buxbaum SG, Whitsel EA, Alonso A, Arking DE, Benjamin EJ, Berenson GS, Bis JC, Chen W, Deo R, Ellinor PT, Heckbert SR, Heiss G, Hsueh WC, Keating BJ, Kerr KF, Li Y, Limacher MC, Liu Y, Lubitz SA, Marciante KD, Mehra R, Meng YA, Newman AB, Newton-Cheh C, North KE, Palmer CD, Psaty BM, Quibrera PM, Redline S, Reiner AP, Rotter JI, Schnabel RB, Schork NJ, Singleton AB, Smith JG, Soliman EZ, Srinivasan SR, Zhang ZM, Zonderman AB, Ferrucci L, Murray SS, Evans MK, Sotoodehnia N, Magnani JW, Avery CL</div>
<div class="citation">Circ Cardiovasc Genet. 2012 Dec; 5(6):639-46.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B26">
<a href="/pubmed/23183192/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common genetic variation near the connexin-43 gene is associated with resting heart rate in African Americans: a genome-wide association study of 13,372 participants.</a><span class="one_line_source">[Heart Rhythm.  2013]</span><div class="alt-note">
<div class="authors">Deo R, Nalls MA, Avery CL, Smith JG, Evans DS, Keller MF, Butler AM, Buxbaum SG, Li G, Miguel Quibrera P, Smith EN, Tanaka T, Akylbekova EL, Alonso A, Arking DE, Benjamin EJ, Berenson GS, Bis JC, Chen LY, Chen W, Cummings SR, Ellinor PT, Evans MK, Ferrucci L, Fox ER, Heckbert SR, Heiss G, Hsueh WC, Kerr KF, Limacher MC, Liu Y, Lubitz SA, Magnani JW, Mehra R, Marcus GM, Murray SS, Newman AB, Njajou O, North KE, Paltoo DN, Psaty BM, Redline SS, Reiner AP, Robinson JG, Rotter JI, Samdarshi TE, Schnabel RB, Schork NJ, Singleton AB, Siscovick D, Soliman EZ, Sotoodehnia N, Srinivasan SR, Taylor HA, Trevisan M, Zhang Z, Zonderman AB, Newton-Cheh C, Whitsel EA</div>
<div class="citation">Heart Rhythm. 2013 Mar; 10(3):401-8.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B48">
<a href="/pubmed/23166209/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Impact of ancestry and common genetic variants on QT interval in African Americans.</a><span class="one_line_source">[Circ Cardiovasc Genet.  2012]</span><div class="alt-note">
<div class="authors">Smith JG, Avery CL, Evans DS, Nalls MA, Meng YA, Smith EN, Palmer C, Tanaka T, Mehra R, Butler AM, Young T, Buxbaum SG, Kerr KF, Berenson GS, Schnabel RB, Li G, Ellinor PT, Magnani JW, Chen W, Bis JC, Curb JD, Hsueh WC, Rotter JI, Liu Y, Newman AB, Limacher MC, North KE, Reiner AP, Quibrera PM, Schork NJ, Singleton AB, Psaty BM, Soliman EZ, Solomon AJ, Srinivasan SR, Alonso A, Wallace R, Redline S, Zhang ZM, Post WS, Zonderman AB, Taylor HA, Murray SS, Ferrucci L, Arking DE, Evans MK, Fox ER, Sotoodehnia N, Heckbert SR, Whitsel EA, Newton-Cheh C, CARe and COGENT consortia.</div>
<div class="citation">Circ Cardiovasc Genet. 2012 Dec; 5(6):647-55.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B49">
<a href="/pubmed/23534349/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Generalization of variants identified by genome-wide association studies for electrocardiographic traits in African Americans.</a><span class="one_line_source">[Ann Hum Genet.  2013]</span><div class="alt-note">
<div class="authors">Jeff JM, Ritchie MD, Denny JC, Kho AN, Ramirez AH, Crosslin D, Armstrong L, Basford MA, Wolf WA, Pacheco JA, Chisholm RL, Roden DM, Hayes MG, Crawford DC</div>
<div class="citation">Ann Hum Genet. 2013 Jul; 77(4):321-32.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B37">
<a href="/pubmed/20126254/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Rare variants create synthetic genome-wide associations.</a><span class="one_line_source">[PLoS Biol.  2010]</span><div class="alt-note">
<div class="authors">Dickson SP, Wang K, Krantz I, Hakonarson H, Goldstein DB</div>
<div class="citation">PLoS Biol. 2010 Jan 26; 8(1):e1000294.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p65" name="crb--__p65" rid="__p65">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-43"><ul><li class="expanded" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p67" name="crb--__p67" rid="__p67">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-44"><ul>
<li class="expanded" reference_id="ddw284-B50">
<a href="/pubmed/16862161/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Principal components analysis corrects for stratification in genome-wide association studies.</a><span class="one_line_source">[Nat Genet.  2006]</span><div class="alt-note">
<div class="authors">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D</div>
<div class="citation">Nat Genet. 2006 Aug; 38(8):904-9.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B51">
<a href="/pubmed/20400780/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Candidate gene association resource (CARe): design, methods, and proof of concept.</a><span class="one_line_source">[Circ Cardiovasc Genet.  2010]</span><div class="alt-note">
<div class="authors">Musunuru K, Lettre G, Young T, Farlow DN, Pirruccello JP, Ejebe KG, Keating BJ, Yang Q, Chen MH, Lapchyk N, Crenshaw A, Ziaugra L, Rachupka A, Benjamin EJ, Cupples LA, Fornage M, Fox ER, Heckbert SR, Hirschhorn JN, Newton-Cheh C, Nizzari MM, Paltoo DN, Papanicolaou GJ, Patel SR, Psaty BM, Rader DJ, Redline S, Rich SS, Rotter JI, Taylor HA Jr, Tracy RP, Vasan RS, Wilson JG, Kathiresan S, Fabsitz RR, Boerwinkle E, Gabriel SB, NHLBI Candidate Gene Association Resource.</div>
<div class="citation">Circ Cardiovasc Genet. 2010 Jun; 3(3):267-75.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B52">
<a href="/pubmed/21347282/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association study of coronary heart disease and its risk factors in 8,090 African Americans: the NHLBI CARe Project.</a><span class="one_line_source">[PLoS Genet.  2011]</span><div class="alt-note">
<div class="authors">Lettre G, Palmer CD, Young T, Ejebe KG, Allayee H, Benjamin EJ, Bennett F, Bowden DW, Chakravarti A, Dreisbach A, Farlow DN, Folsom AR, Fornage M, Forrester T, Fox E, Haiman CA, Hartiala J, Harris TB, Hazen SL, Heckbert SR, Henderson BE, Hirschhorn JN, Keating BJ, Kritchevsky SB, Larkin E, Li M, Rudock ME, McKenzie CA, Meigs JB, Meng YA, Mosley TH, Newman AB, Newton-Cheh CH, Paltoo DN, Papanicolaou GJ, Patterson N, Post WS, Psaty BM, Qasim AN, Qu L, Rader DJ, Redline S, Reilly MP, Reiner AP, Rich SS, Rotter JI, Liu Y, Shrader P, Siscovick DS, Tang WH, Taylor HA, Tracy RP, Vasan RS, Waters KM, Wilks R, Wilson JG, Fabsitz RR, Gabriel SB, Kathiresan S, Boerwinkle E</div>
<div class="citation">PLoS Genet. 2011 Feb 10; 7(2):e1001300.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B53">
<a href="/pubmed/20040588/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">GWAF: an R package for genome-wide association analyses with family data.</a><span class="one_line_source">[Bioinformatics.  2010]</span><div class="alt-note">
<div class="authors">Chen MH, Yang Q</div>
<div class="citation">Bioinformatics. 2010 Feb 15; 26(4):580-1.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p68" name="crb--__p68" rid="__p68">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-45"><ul>
<li class="expanded" reference_id="ddw284-B54">
<a href="/pubmed/20616382/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">METAL: fast and efficient meta-analysis of genomewide association scans.</a><span class="one_line_source">[Bioinformatics.  2010]</span><div class="alt-note">
<div class="authors">Willer CJ, Li Y, Abecasis GR</div>
<div class="citation">Bioinformatics. 2010 Sep 1; 26(17):2190-1.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B55">
<a href="/pubmed/11315092/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genomic control for association studies.</a><span class="one_line_source">[Biometrics.  1999]</span><div class="alt-note">
<div class="authors">Devlin B, Roeder K</div>
<div class="citation">Biometrics. 1999 Dec; 55(4):997-1004.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B56">
<a href="/pubmed/18348202/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Estimation of the multiple testing burden for genomewide association studies of nearly all common variants.</a><span class="one_line_source">[Genet Epidemiol.  2008]</span><div class="alt-note">
<div class="authors">Pe'er I, Yelensky R, Altshuler D, Daly MJ</div>
<div class="citation">Genet Epidemiol. 2008 May; 32(4):381-5.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B57">
<a href="/pubmed/1792461/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A general parametric approach to the meta-analysis of randomized clinical trials.</a><span class="one_line_source">[Stat Med.  1991]</span><div class="alt-note">
<div class="authors">Whitehead A, Whitehead J</div>
<div class="citation">Stat Med. 1991 Nov; 10(11):1665-77.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B58">
<a href="/pubmed/12111919/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Quantifying heterogeneity in a meta-analysis.</a><span class="one_line_source">[Stat Med.  2002]</span><div class="alt-note">
<div class="authors">Higgins JP, Thompson SG</div>
<div class="citation">Stat Med. 2002 Jun 15; 21(11):1539-58.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p69" name="crb--__p69" rid="__p69">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-46"><ul>
<li class="expanded" reference_id="ddw284-B59">
<a href="/pubmed/15088269/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Methods for high-density admixture mapping of disease genes.</a><span class="one_line_source">[Am J Hum Genet.  2004]</span><div class="alt-note">
<div class="authors">Patterson N, Hattangadi N, Lane B, Lohmueller KE, Hafler DA, Oksenberg JR, Hauser SL, Smith MW, O'Brien SJ, Altshuler D, Daly MJ, Reich D</div>
<div class="citation">Am J Hum Genet. 2004 May; 74(5):979-1000.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B60">
<a href="/pubmed/19543370/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Sensitive detection of chromosomal segments of distinct ancestry in admixed populations.</a><span class="one_line_source">[PLoS Genet.  2009]</span><div class="alt-note">
<div class="authors">Price AL, Tandon A, Patterson N, Barnes KC, Rafaels N, Ruczinski I, Beaty TH, Mathias R, Reich D, Myers S</div>
<div class="citation">PLoS Genet. 2009 Jun; 5(6):e1000519.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B52">
<a href="/pubmed/21347282/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association study of coronary heart disease and its risk factors in 8,090 African Americans: the NHLBI CARe Project.</a><span class="one_line_source">[PLoS Genet.  2011]</span><div class="alt-note">
<div class="authors">Lettre G, Palmer CD, Young T, Ejebe KG, Allayee H, Benjamin EJ, Bennett F, Bowden DW, Chakravarti A, Dreisbach A, Farlow DN, Folsom AR, Fornage M, Forrester T, Fox E, Haiman CA, Hartiala J, Harris TB, Hazen SL, Heckbert SR, Henderson BE, Hirschhorn JN, Keating BJ, Kritchevsky SB, Larkin E, Li M, Rudock ME, McKenzie CA, Meigs JB, Meng YA, Mosley TH, Newman AB, Newton-Cheh CH, Paltoo DN, Papanicolaou GJ, Patterson N, Post WS, Psaty BM, Qasim AN, Qu L, Rader DJ, Redline S, Reilly MP, Reiner AP, Rich SS, Rotter JI, Liu Y, Shrader P, Siscovick DS, Tang WH, Taylor HA, Tracy RP, Vasan RS, Waters KM, Wilks R, Wilson JG, Fabsitz RR, Gabriel SB, Kathiresan S, Boerwinkle E</div>
<div class="citation">PLoS Genet. 2011 Feb 10; 7(2):e1001300.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B61">
<a href="/pubmed/15712363/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Estimation of individual admixture: analytical and study design considerations.</a><span class="one_line_source">[Genet Epidemiol.  2005]</span><div class="alt-note">
<div class="authors">Tang H, Peng J, Wang P, Risch NJ</div>
<div class="citation">Genet Epidemiol. 2005 May; 28(4):289-301.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B62">
<a href="/pubmed/16773560/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Reconstructing genetic ancestry blocks in admixed individuals.</a><span class="one_line_source">[Am J Hum Genet.  2006]</span><div class="alt-note">
<div class="authors">Tang H, Coram M, Wang P, Zhu X, Risch N</div>
<div class="citation">Am J Hum Genet. 2006 Jul; 79(1):1-12.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B63">
<a href="/pubmed/10835412/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Inference of population structure using multilocus genotype data.</a><span class="one_line_source">[Genetics.  2000]</span><div class="alt-note">
<div class="authors">Pritchard JK, Stephens M, Donnelly P</div>
<div class="citation">Genetics. 2000 Jun; 155(2):945-59.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B64">
<a href="/pubmed/18179887/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Admixture mapping of white cell count: genetic locus responsible for lower white blood cell count in the Health ABC and Jackson Heart studies.</a><span class="one_line_source">[Am J Hum Genet.  2008]</span><div class="alt-note">
<div class="authors">Nalls MA, Wilson JG, Patterson NJ, Tandon A, Zmuda JM, Huntsman S, Garcia M, Hu D, Li R, Beamer BA, Patel KV, Akylbekova EL, Files JC, Hardy CL, Buxbaum SG, Taylor HA, Reich D, Harris TB, Ziv E</div>
<div class="citation">Am J Hum Genet. 2008 Jan; 82(1):81-7.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B65">
<a href="/pubmed/19477991/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Inference of locus-specific ancestry in closely related populations.</a><span class="one_line_source">[Bioinformatics.  2009]</span><div class="alt-note">
<div class="authors">Pasaniuc B, Sankararaman S, Kimmel G, Halperin E</div>
<div class="citation">Bioinformatics. 2009 Jun 15; 25(12):i213-21.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p70" name="crb--__p70" rid="__p70">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-47"><ul>
<li class="expanded" reference_id="ddw284-B66">
<a href="/pubmed/22064851/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants.</a><span class="one_line_source">[Nucleic Acids Res.  2012]</span><div class="alt-note">
<div class="authors">Ward LD, Kellis M</div>
<div class="citation">Nucleic Acids Res. 2012 Jan; 40(Database issue):D930-4.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B36">
<a href="/pubmed/23128226/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An integrated map of genetic variation from 1,092 human genomes.</a><span class="one_line_source">[Nature.  2012]</span><div class="alt-note">
<div class="authors">1000 Genomes Project Consortium., Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, Handsaker RE, Kang HM, Marth GT, McVean GA</div>
<div class="citation">Nature. 2012 Nov 1; 491(7422):56-65.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B68">
<a href="/pubmed/20944595/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The NIH Roadmap Epigenomics Mapping Consortium.</a><span class="one_line_source">[Nat Biotechnol.  2010]</span><div class="alt-note">
<div class="authors">Bernstein BE, Stamatoyannopoulos JA, Costello JF, Ren B, Milosavljevic A, Meissner A, Kellis M, Marra MA, Beaudet AL, Ecker JR, Farnham PJ, Hirst M, Lander ES, Mikkelsen TS, Thomson JA</div>
<div class="citation">Nat Biotechnol. 2010 Oct; 28(10):1045-8.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p72" name="crb--__p72" rid="__p72">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-48"><ul>
<li class="expanded" reference_id="ddw284-B69">
<a href="/pubmed/20714348/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common inherited variation in mitochondrial genes is not enriched for associations with type 2 diabetes or related glycemic traits.</a><span class="one_line_source">[PLoS Genet.  2010]</span><div class="alt-note">
<div class="authors">Segrè AV, DIAGRAM Consortium., MAGIC investigators., Groop L, Mootha VK, Daly MJ, Altshuler D</div>
<div class="citation">PLoS Genet. 2010 Aug 12; 6(8):.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B70">
<a href="/pubmed/17701901/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PLINK: a tool set for whole-genome association and population-based linkage analyses.</a><span class="one_line_source">[Am J Hum Genet.  2007]</span><div class="alt-note">
<div class="authors">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC</div>
<div class="citation">Am J Hum Genet. 2007 Sep; 81(3):559-75.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p73" name="crb--__p73" rid="__p73">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-49"><ul><li class="expanded" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p74" name="crb--__p74" rid="__p74">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-50"><ul>
<li class="expanded" reference_id="ddw284-B7">
<a href="/pubmed/20062063/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Several common variants modulate heart rate, PR interval and QRS duration.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njølstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Løchen ML, Kong A, Thorsteinsdottir U, Stefansson K</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):117-22.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B16">
<a href="/pubmed/20062061/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetic variation in SCN10A influences cardiac conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Chambers JC, Zhao J, Terracciano CM, Bezzina CR, Zhang W, Kaba R, Navaratnarajah M, Lotlikar A, Sehmi JS, Kooner MK, Deng G, Siedlecka U, Parasramka S, El-Hamamsy I, Wass MN, Dekker LR, de Jong JS, Sternberg MJ, McKenna W, Severs NJ, de Silva R, Wilde AA, Anand P, Yacoub M, Scott J, Elliott P, Wood JN, Kooner JS</div>
<div class="citation">Nat Genet. 2010 Feb; 42(2):149-52.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p75" name="crb--__p75" rid="__p75">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-51"><ul><li class="expanded" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p76" name="crb--__p76" rid="__p76">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-52"><ul>
<li class="expanded" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B71">
<a href="/pubmed/11782049/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Sample size requirements for matched case-control studies of gene-environment interaction.</a><span class="one_line_source">[Stat Med.  2002]</span><div class="alt-note">
<div class="authors">Gauderman WJ</div>
<div class="citation">Stat Med. 2002 Jan 15; 21(1):35-50.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p77" name="crb--__p77" rid="__p77">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-53"><ul>
<li class="expanded" reference_id="ddw284-B72">
<a href="/pubmed/20634204/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">LocusZoom: regional visualization of genome-wide association scan results.</a><span class="one_line_source">[Bioinformatics.  2010]</span><div class="alt-note">
<div class="authors">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, Boehnke M, Abecasis GR, Willer CJ</div>
<div class="citation">Bioinformatics. 2010 Sep 15; 26(18):2336-7.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B73">
<a href="/pubmed/21775986/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The landscape of recombination in African Americans.</a><span class="one_line_source">[Nature.  2011]</span><div class="alt-note">
<div class="authors">Hinch AG, Tandon A, Patterson N, Song Y, Rohland N, Palmer CD, Chen GK, Wang K, Buxbaum SG, Akylbekova EL, Aldrich MC, Ambrosone CB, Amos C, Bandera EV, Berndt SI, Bernstein L, Blot WJ, Bock CH, Boerwinkle E, Cai Q, Caporaso N, Casey G, Cupples LA, Deming SL, Diver WR, Divers J, Fornage M, Gillanders EM, Glessner J, Harris CC, Hu JJ, Ingles SA, Isaacs W, John EM, Kao WH, Keating B, Kittles RA, Kolonel LN, Larkin E, Le Marchand L, McNeill LH, Millikan RC, Murphy A, Musani S, Neslund-Dudas C, Nyante S, Papanicolaou GJ, Press MF, Psaty BM, Reiner AP, Rich SS, Rodriguez-Gil JL, Rotter JI, Rybicki BA, Schwartz AG, Signorello LB, Spitz M, Strom SS, Thun MJ, Tucker MA, Wang Z, Wiencke JK, Witte JS, Wrensch M, Wu X, Yamamura Y, Zanetti KA, Zheng W, Ziegler RG, Zhu X, Redline S, Hirschhorn JN, Henderson BE, Taylor HA Jr, Price AL, Hakonarson H, Chanock SJ, Haiman CA, Wilson JG, Reich D, Myers SR</div>
<div class="citation">Nature. 2011 Jul 20; 476(7359):170-5.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B74">
<a href="/pubmed/15297300/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Haploview: analysis and visualization of LD and haplotype maps.</a><span class="one_line_source">[Bioinformatics.  2005]</span><div class="alt-note">
<div class="authors">Barrett JC, Fry B, Maller J, Daly MJ</div>
<div class="citation">Bioinformatics. 2005 Jan 15; 21(2):263-5.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B75">
<a href="/pubmed/12029063/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The structure of haplotype blocks in the human genome.</a><span class="one_line_source">[Science.  2002]</span><div class="alt-note">
<div class="authors">Gabriel SB, Schaffner SF, Nguyen H, Moore JM, Roy J, Blumenstiel B, Higgins J, DeFelice M, Lochner A, Faggart M, Liu-Cordero SN, Rotimi C, Adeyemo A, Cooper R, Ward R, Lander ES, Daly MJ, Altshuler D</div>
<div class="citation">Science. 2002 Jun 21; 296(5576):2225-9.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p78" name="crb--__p78" rid="__p78">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-54"><ul>
<li class="expanded" reference_id="ddw284-B8">
<a href="/pubmed/21076409/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Sotoodehnia N, Isaacs A, de Bakker PI, Dörr M, Newton-Cheh C, Nolte IM, van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MP, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, de Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu FY, Boerwinkle E, Dominiczak AF, Rotter JI, van Herpen G, Levy D, Wichmann HE, van Gilst WH, Witteman JC, Kroemer HK, Kao WH, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, van Veldhuisen DJ, Schwienbacher C, O'Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RS, te Meerman G, Groen HJ, Weersma RK, van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, van Duijn CM, Felix SB, Fishman GI, Jamshidi Y, Stricker BH, Samani NJ, Kääb S, Arking DE</div>
<div class="citation">Nat Genet. 2010 Dec; 42(12):1068-76.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B17">
<a href="/pubmed/22125221/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Transethnic meta-analysis of genomewide association studies.</a><span class="one_line_source">[Genet Epidemiol.  2011]</span><div class="alt-note">
<div class="authors">Morris AP</div>
<div class="citation">Genet Epidemiol. 2011 Dec; 35(8):809-22.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B76">
<a href="/pubmed/23104008/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Bayesian refinement of association signals for 14 loci in 3 common diseases.</a><span class="one_line_source">[Nat Genet.  2012]</span><div class="alt-note">
<div class="authors">Wellcome Trust Case Control Consortium., Maller JB, McVean G, Byrnes J, Vukcevic D, Palin K, Su Z, Howson JM, Auton A, Myers S, Morris A, Pirinen M, Brown MA, Burton PR, Caulfield MJ, Compston A, Farrall M, Hall AS, Hattersley AT, Hill AV, Mathew CG, Pembrey M, Satsangi J, Stratton MR, Worthington J, Craddock N, Hurles M, Ouwehand W, Parkes M, Rahman N, Duncanson A, Todd JA, Kwiatkowski DP, Samani NJ, Gough SC, McCarthy MI, Deloukas P, Donnelly P</div>
<div class="citation">Nat Genet. 2012 Dec; 44(12):1294-301.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B77">
<a href="/pubmed/26839038/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Trans-ethnic study design approaches for fine-mapping.</a><span class="one_line_source">[Eur J Hum Genet.  2016]</span><div class="alt-note">
<div class="authors">Asimit JL, Hatzikotoulas K, McCarthy M, Morris AP, Zeggini E</div>
<div class="citation">Eur J Hum Genet. 2016 Aug; 24(9):1330-6.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p80" name="crb--__p80" rid="__p80">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-55"><ul>
<li class="expanded" reference_id="ddw284-B11">
<a href="/pubmed/21836165/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The Next PAGE in understanding complex traits: design for the analysis of Population Architecture Using Genetics and Epidemiology (PAGE) Study.</a><span class="one_line_source">[Am J Epidemiol.  2011]</span><div class="alt-note">
<div class="authors">Matise TC, Ambite JL, Buyske S, Carlson CS, Cole SA, Crawford DC, Haiman CA, Heiss G, Kooperberg C, Marchand LL, Manolio TA, North KE, Peters U, Ritchie MD, Hindorff LA, Haines JL, PAGE Study.</div>
<div class="citation">Am J Epidemiol. 2011 Oct 1; 174(7):849-59.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B78">
<a href="/pubmed/20981092/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A map of human genome variation from population-scale sequencing.</a><span class="one_line_source">[Nature.  2010]</span><div class="alt-note">
<div class="authors">1000 Genomes Project Consortium., Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, Gibbs RA, Hurles ME, McVean GA</div>
<div class="citation">Nature. 2010 Oct 28; 467(7319):1061-73.</div>
</div>
</li>
<li class="expanded" reference_id="ddw284-B79">
<a href="/pubmed/22851474/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genotype imputation of Metabochip SNPs using a study-specific reference panel of ~4,000 haplotypes in African Americans from the Women's Health Initiative.</a><span class="one_line_source">[Genet Epidemiol.  2012]</span><div class="alt-note">
<div class="authors">Liu EY, Buyske S, Aragaki AK, Peters U, Boerwinkle E, Carlson C, Carty C, Crawford DC, Haessler J, Hindorff LA, Marchand LL, Manolio TA, Matise T, Wang W, Kooperberg C, North KE, Li Y</div>
<div class="citation">Genet Epidemiol. 2012 Feb; 36(2):107-17.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p81" name="crb--__p81" rid="__p81">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-56"><ul><li class="expanded" reference_id="ddw284-B80">
<a href="/pubmed/17907809/" ref="reftype=pubmed&amp;article-id=5291202&amp;issue-id=286901&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Capturing heterogeneity in gene expression studies by surrogate variable analysis.</a><span class="one_line_source">[PLoS Genet.  2007]</span><div class="alt-note">
<div class="authors">Leek JT, Storey JD</div>
<div class="citation">PLoS Genet. 2007 Sep; 3(9):1724-35.</div>
</div>
</li></ul></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:01:38-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CB404DD594991000000000471029B&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5291202%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5291202/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Main, sans-serif;"></div></div></body></html>