<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5300750/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Dense Genotyping of Immune-Related Loci in the Idiopathic Inflammatory Myopathies Confirms HLA alleles as Strongest Genetic Risk Factor and Suggests Different Genetic Background for Major Clinical Subgroups</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5300750/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Annals of the rheumatic diseases"><meta name="citation_title" content="Dense Genotyping of Immune-Related Loci in the Idiopathic Inflammatory Myopathies Confirms HLA alleles as Strongest Genetic Risk Factor and Suggests Different Genetic Background for Major Clinical Subgroups"><meta name="citation_authors" content="Simon Rothwell, Robert G. Cooper, Ingrid E. Lundberg, Frederick W. Miller, Peter K. Gregersen, John Bowes, Jiri Vencovsky, Katalin Danko, Vidya Limaye, Albert Selva-O’Callaghan, Michael G. Hanna, Pedro M. Machado, Lauren M. Pachman, Ann M. Reed, Lisa G. Rider, Joanna Cobb, Hazel Platt, Øyvind Molberg, Olivier Benveniste, Pernille Mathiesen, Timothy Radstake, Andrea Doria, Jan De Bleecker, Boel De Paepe, Britta Maurer, William E. Ollier, Leonid Padyukov, Terrance P. O’Hanlon, Annette Lee, Christopher I. Amos, Christian Gieger, Thomas Meitinger, Juliane Winkelmann, Lucy R Wedderburn, Hector Chinoy, Janine A Lamb, Myositis Genetics Consortium"><meta name="citation_date" content="August 2016"><meta name="citation_issue" content="8"><meta name="citation_volume" content="75"><meta name="citation_firstpage" content="1558"><meta name="citation_doi" content="10.1136/annrheumdis-2015-208119"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5300750/?report=abstract"><meta name="citation_pmid" content="26362759"><meta name="DC.Title" content="Dense Genotyping of Immune-Related Loci in the Idiopathic Inflammatory Myopathies Confirms HLA alleles as Strongest Genetic Risk Factor and Suggests Different Genetic Background for Major Clinical Subgroups"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Simon Rothwell"><meta name="DC.Contributor" content="Robert G. Cooper"><meta name="DC.Contributor" content="Ingrid E. Lundberg"><meta name="DC.Contributor" content="Frederick W. Miller"><meta name="DC.Contributor" content="Peter K. Gregersen"><meta name="DC.Contributor" content="John Bowes"><meta name="DC.Contributor" content="Jiri Vencovsky"><meta name="DC.Contributor" content="Katalin Danko"><meta name="DC.Contributor" content="Vidya Limaye"><meta name="DC.Contributor" content="Albert Selva-O’Callaghan"><meta name="DC.Contributor" content="Michael G. Hanna"><meta name="DC.Contributor" content="Pedro M. Machado"><meta name="DC.Contributor" content="Lauren M. Pachman"><meta name="DC.Contributor" content="Ann M. Reed"><meta name="DC.Contributor" content="Lisa G. Rider"><meta name="DC.Contributor" content="Joanna Cobb"><meta name="DC.Contributor" content="Hazel Platt"><meta name="DC.Contributor" content="Øyvind Molberg"><meta name="DC.Contributor" content="Olivier Benveniste"><meta name="DC.Contributor" content="Pernille Mathiesen"><meta name="DC.Contributor" content="Timothy Radstake"><meta name="DC.Contributor" content="Andrea Doria"><meta name="DC.Contributor" content="Jan De Bleecker"><meta name="DC.Contributor" content="Boel De Paepe"><meta name="DC.Contributor" content="Britta Maurer"><meta name="DC.Contributor" content="William E. Ollier"><meta name="DC.Contributor" content="Leonid Padyukov"><meta name="DC.Contributor" content="Terrance P. O’Hanlon"><meta name="DC.Contributor" content="Annette Lee"><meta name="DC.Contributor" content="Christopher I. Amos"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Thomas Meitinger"><meta name="DC.Contributor" content="Juliane Winkelmann"><meta name="DC.Contributor" content="Lucy R Wedderburn"><meta name="DC.Contributor" content="Hector Chinoy"><meta name="DC.Contributor" content="Janine A Lamb"><meta name="DC.Contributor" content="Myositis Genetics Consortium"><meta name="DC.Date" content="2016 Aug"><meta name="DC.Identifier" content="10.1136/annrheumdis-2015-208119"><meta name="DC.Language" content="en"><meta property="og:title" content="Dense Genotyping of Immune-Related Loci in the Idiopathic Inflammatory Myopathies Confirms HLA alleles as Strongest Genetic Risk Factor and Suggests Different Genetic Background for Major Clinical Subgroups"><meta property="og:type" content="article"><meta property="og:description" content="The idiopathic inflammatory myopathies (IIM) are a heterogeneous group of rare autoimmune diseases characterized by muscle weakness and extramuscular manifestations such as skin rashes and interstitial lung disease. We genotyped 2,566 IIM cases of Caucasian ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5300750/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5300750/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5300750/pdf/nihms847779.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89FBF7DD588541000000000382021E.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5300750%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5300750</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Ann Rheum Dis</a></span>. Author manuscript; available in PMC 2017 Aug 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26362759" target="pmc_ext" ref="reftype=publisher&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Ann Rheum Dis. 2016 Aug; 75(8): 1558–1566. </span></a></div><span class="fm-vol-iss-date">Published online 2015 Sep 11. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1136%2Fannrheumdis-2015-208119" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1136/annrheumdis-2015-208119</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5300750</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS847779</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26362759">26362759</a></div></div></div></div><h1 class="content-title">Dense Genotyping of Immune-Related Loci in the Idiopathic Inflammatory Myopathies Confirms HLA alleles as Strongest Genetic Risk Factor and Suggests Different Genetic Background for Major Clinical Subgroups</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Rothwell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647818652400" co-class="co-affbox">Simon Rothwell</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Cooper%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811367312" co-class="co-affbox">Robert G. Cooper</a>,<sup>2</sup> <a href="/pubmed/?term=Lundberg%20IE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647818649392" co-class="co-affbox">Ingrid E. Lundberg</a>,<sup>3</sup> <a href="/pubmed/?term=Miller%20FW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647808403920" co-class="co-affbox">Frederick W. Miller</a>,<sup>4</sup> <a href="/pubmed/?term=Gregersen%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647808401936" co-class="co-affbox">Peter K. Gregersen</a>,<sup>5</sup> <a href="/pubmed/?term=Bowes%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647808399952" co-class="co-affbox">John Bowes</a>,<sup>1</sup> <a href="/pubmed/?term=Vencovsky%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647808397968" co-class="co-affbox">Jiri Vencovsky</a>,<sup>6</sup> <a href="/pubmed/?term=Danko%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647808395984" co-class="co-affbox">Katalin Danko</a>,<sup>7</sup> <a href="/pubmed/?term=Limaye%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647808394000" co-class="co-affbox">Vidya Limaye</a>,<sup>8</sup> <a href="/pubmed/?term=Selva-O%26%23x02019%3BCallaghan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811844352" co-class="co-affbox">Albert Selva-O’Callaghan</a>,<sup>9</sup> <a href="/pubmed/?term=Hanna%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811842368" co-class="co-affbox">Michael G. Hanna</a>,<sup>10</sup> <a href="/pubmed/?term=Machado%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811840384" co-class="co-affbox">Pedro M. Machado</a>,<sup>10</sup> <a href="/pubmed/?term=Pachman%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811838400" co-class="co-affbox">Lauren M. Pachman</a>,<sup>11</sup> <a href="/pubmed/?term=Reed%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811836416" co-class="co-affbox">Ann M. Reed</a>,<sup>12</sup> <a href="/pubmed/?term=Rider%20LG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811834432" co-class="co-affbox">Lisa G. Rider</a>,<sup>4</sup> <a href="/pubmed/?term=Cobb%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811151344" co-class="co-affbox">Joanna Cobb</a>,<sup>13</sup> <a href="/pubmed/?term=Platt%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811149360" co-class="co-affbox">Hazel Platt</a>,<sup>14</sup> <a href="/pubmed/?term=Molberg%20%26%23x000d8%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811147376" co-class="co-affbox">Øyvind Molberg</a>,<sup>15</sup> <a href="/pubmed/?term=Benveniste%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811145392" co-class="co-affbox">Olivier Benveniste</a>,<sup>16</sup> <a href="/pubmed/?term=Mathiesen%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811143408" co-class="co-affbox">Pernille Mathiesen</a>,<sup>17</sup> <a href="/pubmed/?term=Radstake%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647811141424" co-class="co-affbox">Timothy Radstake</a>,<sup>18</sup> <a href="/pubmed/?term=Doria%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647810489104" co-class="co-affbox">Andrea Doria</a>,<sup>19</sup> <a href="/pubmed/?term=De%20Bleecker%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647810487120" co-class="co-affbox">Jan De Bleecker</a>,<sup>20</sup> <a href="/pubmed/?term=De%20Paepe%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647810485136" co-class="co-affbox">Boel De Paepe</a>,<sup>20</sup> <a href="/pubmed/?term=Maurer%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647810483152" co-class="co-affbox">Britta Maurer</a>,<sup>21</sup> <a href="/pubmed/?term=Ollier%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647810481168" co-class="co-affbox">William E. Ollier</a>,<sup>14</sup> <a href="/pubmed/?term=Padyukov%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647810479184" co-class="co-affbox">Leonid Padyukov</a>,<sup>3</sup> <a href="/pubmed/?term=O%26%23x02019%3BHanlon%20TP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647816936912" co-class="co-affbox">Terrance P. O’Hanlon</a>,<sup>4</sup> <a href="/pubmed/?term=Lee%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647816934928" co-class="co-affbox">Annette Lee</a>,<sup>5</sup> <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647816932944" co-class="co-affbox">Christopher I. Amos</a>,<sup>22</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647816930960" co-class="co-affbox">Christian Gieger</a>,<sup>23</sup> <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647816928976" co-class="co-affbox">Thomas Meitinger</a>,<sup>24,</sup><sup>25</sup> <a href="/pubmed/?term=Winkelmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647816926256" co-class="co-affbox">Juliane Winkelmann</a>,<sup>26,</sup><sup>27</sup> <a href="/pubmed/?term=Wedderburn%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647818172304" co-class="co-affbox">Lucy R Wedderburn</a>,<sup>28</sup> <a href="/pubmed/?term=Chinoy%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647818170320" co-class="co-affbox">Hector Chinoy</a>,<sup>29</sup> <a href="/pubmed/?term=Lamb%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759" class="affpopup" co-rid="_co_idm140647818168336" co-class="co-affbox">Janine A Lamb</a>,<sup>14</sup> and  Myositis Genetics Consortium</div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140647818652400"><h3 class="no_margin">Simon Rothwell</h3><p><sup>1</sup>Centre for Genetics and Genomics, Arthritis Research UK, University of Manchester, Manchester, M13 9PT, UK</p><div>Find articles by <a href="/pubmed/?term=Rothwell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Simon Rothwell</a></div></div><div id="_co_idm140647811367312"><h3 class="no_margin">Robert G. Cooper</h3><p><sup>2</sup>Department of Musculoskeletal Biology, Institute of Ageing and Chronic Disease, University of Liverpool, Liverpool, L69 3GA, UK</p><div>Find articles by <a href="/pubmed/?term=Cooper%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Robert G. Cooper</a></div></div><div id="_co_idm140647818649392"><h3 class="no_margin">Ingrid E. Lundberg</h3><p><sup>3</sup>Rheumatology Unit, Department of Medicine, Karolinska University Hospital, Karolinska Institutet, Stockholm, 171 76, Sweden</p><div>Find articles by <a href="/pubmed/?term=Lundberg%20IE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Ingrid E. Lundberg</a></div></div><div id="_co_idm140647808403920"><h3 class="no_margin">Frederick W. Miller</h3><p><sup>4</sup>Environmental Autoimmunity Group, Clinical Research Branch, National Institute of Environmental Health Science, National Institutes of Health, Bethesda, MD 20892-1301, USA</p><div>Find articles by <a href="/pubmed/?term=Miller%20FW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Frederick W. Miller</a></div></div><div id="_co_idm140647808401936"><h3 class="no_margin">Peter K. Gregersen</h3><p><sup>5</sup>The Robert S Boas Center for Genomics and Human Genetics, Feinstein Institute for Medical Research, Manhasset, NY 11030, USA</p><div>Find articles by <a href="/pubmed/?term=Gregersen%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Peter K. Gregersen</a></div></div><div id="_co_idm140647808399952"><h3 class="no_margin">John Bowes</h3><p><sup>1</sup>Centre for Genetics and Genomics, Arthritis Research UK, University of Manchester, Manchester, M13 9PT, UK</p><div>Find articles by <a href="/pubmed/?term=Bowes%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">John Bowes</a></div></div><div id="_co_idm140647808397968"><h3 class="no_margin">Jiri Vencovsky</h3><p><sup>6</sup>Institute of Rheumatology and Department of Rheumatology, 1<sup>st</sup> Faculty of Medicine, Charles University, Prague, 12850, Czech Republic</p><div>Find articles by <a href="/pubmed/?term=Vencovsky%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Jiri Vencovsky</a></div></div><div id="_co_idm140647808395984"><h3 class="no_margin">Katalin Danko</h3><p><sup>7</sup>Division of Clinical Immunology, Department of Internal Medicine, University of Debrecen, Hungary</p><div>Find articles by <a href="/pubmed/?term=Danko%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Katalin Danko</a></div></div><div id="_co_idm140647808394000"><h3 class="no_margin">Vidya Limaye</h3><p><sup>8</sup>Royal Adelaide Hospital and University of Adelaide, North Tce, Adelaide, Australia</p><div>Find articles by <a href="/pubmed/?term=Limaye%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Vidya Limaye</a></div></div><div id="_co_idm140647811844352"><h3 class="no_margin">Albert Selva-O’Callaghan</h3><p><sup>9</sup>Department of Internal Medicine, Vall d’Hebron Hospital, Passeig Vall d’Hebron, Barcelona 08035, Spain</p><div>Find articles by <a href="/pubmed/?term=Selva-O%26%23x02019%3BCallaghan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Albert Selva-O’Callaghan</a></div></div><div id="_co_idm140647811842368"><h3 class="no_margin">Michael G. Hanna</h3><p><sup>10</sup>MRC Centre for Neuromuscular Diseases, UCL Institute of Neurology, London, WC1N 3BG, UK</p><div>Find articles by <a href="/pubmed/?term=Hanna%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Michael G. Hanna</a></div></div><div id="_co_idm140647811840384"><h3 class="no_margin">Pedro M. Machado</h3><p><sup>10</sup>MRC Centre for Neuromuscular Diseases, UCL Institute of Neurology, London, WC1N 3BG, UK</p><div>Find articles by <a href="/pubmed/?term=Machado%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Pedro M. Machado</a></div></div><div id="_co_idm140647811838400"><h3 class="no_margin">Lauren M. Pachman</h3><p><sup>11</sup>Northwestern University Feinberg School of Medicine and Ann &amp; Robert H. Lurie Children’s Hospital of Chicago, Chicago, IL 60611, USA</p><div>Find articles by <a href="/pubmed/?term=Pachman%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Lauren M. Pachman</a></div></div><div id="_co_idm140647811836416"><h3 class="no_margin">Ann M. Reed</h3><p><sup>12</sup>Department of Pediatrics, Duke University, Durham, NC 27710, USA</p><div>Find articles by <a href="/pubmed/?term=Reed%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Ann M. Reed</a></div></div><div id="_co_idm140647811834432"><h3 class="no_margin">Lisa G. Rider</h3><p><sup>4</sup>Environmental Autoimmunity Group, Clinical Research Branch, National Institute of Environmental Health Science, National Institutes of Health, Bethesda, MD 20892-1301, USA</p><div>Find articles by <a href="/pubmed/?term=Rider%20LG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Lisa G. Rider</a></div></div><div id="_co_idm140647811151344"><h3 class="no_margin">Joanna Cobb</h3><p><sup>13</sup>Arthritis Research UK, NIHR Manchester Musculoskeletal Biomedical Research Unit, Central Manchester NHS Foundation Trust, Manchester Academic Health Science Centre, University of Manchester, Manchester, M13 9PT, UK</p><div>Find articles by <a href="/pubmed/?term=Cobb%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Joanna Cobb</a></div></div><div id="_co_idm140647811149360"><h3 class="no_margin">Hazel Platt</h3><p><sup>14</sup>Centre for Integrated Genomic Medical Research, University of Manchester, Manchester, M13 9PT, UK</p><div>Find articles by <a href="/pubmed/?term=Platt%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Hazel Platt</a></div></div><div id="_co_idm140647811147376"><h3 class="no_margin">Øyvind Molberg</h3><p><sup>15</sup>Department of Rheumatology, Oslo University Hospital, Oslo 0424, Norway</p><div>Find articles by <a href="/pubmed/?term=Molberg%20%26%23x000d8%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Øyvind Molberg</a></div></div><div id="_co_idm140647811145392"><h3 class="no_margin">Olivier Benveniste</h3><p><sup>16</sup>Pitié-Salpêtrière Hospital, UPMC, APHP, Paris 75013, France</p><div>Find articles by <a href="/pubmed/?term=Benveniste%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Olivier Benveniste</a></div></div><div id="_co_idm140647811143408"><h3 class="no_margin">Pernille Mathiesen</h3><p><sup>17</sup>Paediatric Department, Naestved Hospital, Næstved 4700, Denmark</p><div>Find articles by <a href="/pubmed/?term=Mathiesen%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Pernille Mathiesen</a></div></div><div id="_co_idm140647811141424"><h3 class="no_margin">Timothy Radstake</h3><p><sup>18</sup>Department of Rheumatology and Clinical Immunology, University Medical Center Utrecht, 3584 CX Utrecht, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Radstake%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Timothy Radstake</a></div></div><div id="_co_idm140647810489104"><h3 class="no_margin">Andrea Doria</h3><p><sup>19</sup>Department of Medicine, University of Padova, 35128 Padova, Italy</p><div>Find articles by <a href="/pubmed/?term=Doria%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Andrea Doria</a></div></div><div id="_co_idm140647810487120"><h3 class="no_margin">Jan De Bleecker</h3><p><sup>20</sup>Department of Neurology and Neuromuscular Reference Centre, Ghent University Hospital, 9000 Ghent, Belgium</p><div>Find articles by <a href="/pubmed/?term=De%20Bleecker%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Jan De Bleecker</a></div></div><div id="_co_idm140647810485136"><h3 class="no_margin">Boel De Paepe</h3><p><sup>20</sup>Department of Neurology and Neuromuscular Reference Centre, Ghent University Hospital, 9000 Ghent, Belgium</p><div>Find articles by <a href="/pubmed/?term=De%20Paepe%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Boel De Paepe</a></div></div><div id="_co_idm140647810483152"><h3 class="no_margin">Britta Maurer</h3><p><sup>21</sup>Department of Rheumatology and Center of Experimental Rheumatology, University Hospital Zurich, Zurich CH-8091, Switzerland</p><div>Find articles by <a href="/pubmed/?term=Maurer%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Britta Maurer</a></div></div><div id="_co_idm140647810481168"><h3 class="no_margin">William E. Ollier</h3><p><sup>14</sup>Centre for Integrated Genomic Medical Research, University of Manchester, Manchester, M13 9PT, UK</p><div>Find articles by <a href="/pubmed/?term=Ollier%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">William E. Ollier</a></div></div><div id="_co_idm140647810479184"><h3 class="no_margin">Leonid Padyukov</h3><p><sup>3</sup>Rheumatology Unit, Department of Medicine, Karolinska University Hospital, Karolinska Institutet, Stockholm, 171 76, Sweden</p><div>Find articles by <a href="/pubmed/?term=Padyukov%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Leonid Padyukov</a></div></div><div id="_co_idm140647816936912"><h3 class="no_margin">Terrance P. O’Hanlon</h3><p><sup>4</sup>Environmental Autoimmunity Group, Clinical Research Branch, National Institute of Environmental Health Science, National Institutes of Health, Bethesda, MD 20892-1301, USA</p><div>Find articles by <a href="/pubmed/?term=O%26%23x02019%3BHanlon%20TP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Terrance P. O’Hanlon</a></div></div><div id="_co_idm140647816934928"><h3 class="no_margin">Annette Lee</h3><p><sup>5</sup>The Robert S Boas Center for Genomics and Human Genetics, Feinstein Institute for Medical Research, Manhasset, NY 11030, USA</p><div>Find articles by <a href="/pubmed/?term=Lee%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Annette Lee</a></div></div><div id="_co_idm140647816932944"><h3 class="no_margin">Christopher I. Amos</h3><p><sup>22</sup>Geisel School of Medicine, Dartmouth College, Hanover, NH 03750, USA</p><div>Find articles by <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Christopher I. Amos</a></div></div><div id="_co_idm140647816930960"><h3 class="no_margin">Christian Gieger</h3><p><sup>23</sup>Helmholtz Zentrum München, Deutsches Forschungszentrum für Gesundheit und Umwelt (GmbH), D-85764 Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Christian Gieger</a></div></div><div id="_co_idm140647816928976"><h3 class="no_margin">Thomas Meitinger</h3><p><sup>24</sup>Institute of Human Genetics, Technische Universität München, 81675, Munich, Germany</p><p><sup>25</sup>Institute of Human Genetics, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Thomas Meitinger</a></div></div><div id="_co_idm140647816926256"><h3 class="no_margin">Juliane Winkelmann</h3><p><sup>26</sup>Neurologische Klinik und Poliklinik, Klinikum rechts der Isar, Technische Universität München, 81675 Munich, Germany</p><p><sup>27</sup>Institute of Neurogenomics, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Winkelmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Juliane Winkelmann</a></div></div><div id="_co_idm140647818172304"><h3 class="no_margin">Lucy R Wedderburn</h3><p><sup>28</sup>Arthritis Research UK Centre for Adolescent Rheumatology, and Institute of Child Health, University College London, London, WC1N 1EH, UK</p><div>Find articles by <a href="/pubmed/?term=Wedderburn%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Lucy R Wedderburn</a></div></div><div id="_co_idm140647818170320"><h3 class="no_margin">Hector Chinoy</h3><p><sup>29</sup>National Institute of Health Research Manchester Musculoskeletal Biomedical Research Unit, Centre for Musculoskeletal Research, University of Manchester, Manchester, M13 9PT, UK</p><div>Find articles by <a href="/pubmed/?term=Chinoy%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Hector Chinoy</a></div></div><div id="_co_idm140647818168336"><h3 class="no_margin">Janine A Lamb</h3><p><sup>14</sup>Centre for Integrated Genomic Medical Research, University of Manchester, Manchester, M13 9PT, UK</p><div>Find articles by <a href="/pubmed/?term=Lamb%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26362759">Janine A Lamb</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140647809851856_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140647809851856_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140647809851856_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Centre for Genetics and Genomics, Arthritis Research UK, University of Manchester, Manchester, M13 9PT, UK</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Musculoskeletal Biology, Institute of Ageing and Chronic Disease, University of Liverpool, Liverpool, L69 3GA, UK</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Rheumatology Unit, Department of Medicine, Karolinska University Hospital, Karolinska Institutet, Stockholm, 171 76, Sweden</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Environmental Autoimmunity Group, Clinical Research Branch, National Institute of Environmental Health Science, National Institutes of Health, Bethesda, MD 20892-1301, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>The Robert S Boas Center for Genomics and Human Genetics, Feinstein Institute for Medical Research, Manhasset, NY 11030, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Institute of Rheumatology and Department of Rheumatology, 1<sup>st</sup> Faculty of Medicine, Charles University, Prague, 12850, Czech Republic</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Division of Clinical Immunology, Department of Internal Medicine, University of Debrecen, Hungary</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Royal Adelaide Hospital and University of Adelaide, North Tce, Adelaide, Australia</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Internal Medicine, Vall d’Hebron Hospital, Passeig Vall d’Hebron, Barcelona 08035, Spain</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>MRC Centre for Neuromuscular Diseases, UCL Institute of Neurology, London, WC1N 3BG, UK</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Northwestern University Feinberg School of Medicine and Ann &amp; Robert H. Lurie Children’s Hospital of Chicago, Chicago, IL 60611, USA</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Pediatrics, Duke University, Durham, NC 27710, USA</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Arthritis Research UK, NIHR Manchester Musculoskeletal Biomedical Research Unit, Central Manchester NHS Foundation Trust, Manchester Academic Health Science Centre, University of Manchester, Manchester, M13 9PT, UK</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Centre for Integrated Genomic Medical Research, University of Manchester, Manchester, M13 9PT, UK</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Rheumatology, Oslo University Hospital, Oslo 0424, Norway</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Pitié-Salpêtrière Hospital, UPMC, APHP, Paris 75013, France</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Paediatric Department, Naestved Hospital, Næstved 4700, Denmark</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Department of Rheumatology and Clinical Immunology, University Medical Center Utrecht, 3584 CX Utrecht, the Netherlands</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Medicine, University of Padova, 35128 Padova, Italy</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Department of Neurology and Neuromuscular Reference Centre, Ghent University Hospital, 9000 Ghent, Belgium</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Department of Rheumatology and Center of Experimental Rheumatology, University Hospital Zurich, Zurich CH-8091, Switzerland</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Geisel School of Medicine, Dartmouth College, Hanover, NH 03750, USA</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Helmholtz Zentrum München, Deutsches Forschungszentrum für Gesundheit und Umwelt (GmbH), D-85764 Neuherberg, Germany</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Institute of Human Genetics, Technische Universität München, 81675, Munich, Germany</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Institute of Human Genetics, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Neurologische Klinik und Poliklinik, Klinikum rechts der Isar, Technische Universität München, 81675 Munich, Germany</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Institute of Neurogenomics, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Arthritis Research UK Centre for Adolescent Rheumatology, and Institute of Child Health, University College London, London, WC1N 1EH, UK</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>National Institute of Health Research Manchester Musculoskeletal Biomedical Research Unit, Centre for Musculoskeletal Research, University of Manchester, Manchester, M13 9PT, UK</div><div id="FN1">Corresponding Author: Simon Rothwell, Centre for Genetics and Genomics, Arthritis Research UK, University of Manchester, Manchester, M13 9PT, UK, <a href="mailto:dev@null" data-email="ku.ca.retsehcnam@llewhtor.s" class="oemail">ku.ca.retsehcnam@llewhtor.s</a>, Phone: 07979501439</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140647809851856_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26362759" target="pmc_ext" ref="reftype=publisher&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Ann Rheum Dis</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5300750/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary.</strong></div><div><a href="/pmc/articles/PMC5300750/bin/NIHMS847779-supplement-Supplementary.pdf" data-ga-action="click_feat_suppl">NIHMS847779-supplement-Supplementary.pdf</a><span style="color:gray"> (748K)</span></div><div class="small guid">GUID:&nbsp;9AA1E855-C011-477B-81D8-E85F3B4A8A86</div></div></dd></dl></div><div id="idm140647812922144" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140647812922144title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">The idiopathic inflammatory myopathies (IIM) are a heterogeneous group of rare autoimmune diseases characterized by muscle weakness and extramuscular manifestations such as skin rashes and interstitial lung disease. We genotyped 2,566 IIM cases of Caucasian descent using the Immunochip; a custom array covering 186 established autoimmune susceptibility loci. The cohort was predominantly comprised of dermatomyositis (DM, n=879), juvenile dermatomyositis (JDM, n=481), polymyositis (PM, n=931) and inclusion body myositis (IBM, n=252) patients collected from 14 countries through the Myositis Genetics Consortium. The human leukocyte antigen (<em>HLA</em>) and <em>PTPN22</em> regions reached genome-wide significance (p&lt;5×10<sup>−8</sup>). Nine regions were associated at a significance level of p&lt;2.25×10<sup>−5</sup>, including <em>UBE2L3</em>, <em>CD28</em> and <em>TRAF6,</em> with evidence of independent effects within <em>STAT4</em>. Analysis of clinical subgroups revealed distinct differences between PM, and DM and JDM. <em>PTPN22</em> was associated at genome-wide significance with PM, but not DM and JDM, suggesting this effect is driven by PM. Additional suggestive associations including <em>IL18R1</em> and <em>RGS1</em> in PM and <em>GSDMB</em> in DM were identified. HLA imputation confirmed that alleles <em>HLA-DRB1*03:01</em> and <em>HLA-B*08:01</em> of the 8.1 ancestral haplotype (8.1AH) are most strongly associated with IIM, and provides evidence that amino acids within the HLA, such as <em>HLA-DQB1</em> position 57 in DM, may explain part of the risk in this locus. Associations with alleles outside the 8.1AH reveal differences between PM, DM, and JDM. This work represents the largest IIM genetic study to date, reveals new insights into the genetic architecture of these rare diseases and suggests different predominating pathophysiology in different clinical subgroups.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Genetics, idiopathic inflammatory myopathy, myositis, HLA, association</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">The idiopathic inflammatory myopathies (IIM) are a heterogeneous group of rare autoimmune diseases, the major phenotypes of which are dermatomyositis (DM), polymyositis (PM), inclusion body myositis (IBM) and DM/PM overlapping with other connective tissue diseases.[<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134060">1</a>] IIM are primarily characterised by the presence of proximal muscle weakness, elevated levels of skeletal muscle enzymes and inflammatory infiltrates in skeletal muscle, but may also present with extramuscular manifestations including skin rashes, interstitial lung disease and malignancy that often correlate with serum antibody status.[<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134081">2</a>]</p><p id="P3" class="p p-last">IIM are thought to be complex genetic diseases, initiated by immune activation following specific environmental events in genetically predisposed individuals. The strongest genetic association in the IIM has been consistently within the major histocompatibility complex (MHC),[<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134045">3</a>] specifically with the 8.1 ancestral haplotype (8.1 AH). A recent genome-wide association study (GWAS) in DM, and a candidate gene study, also indicate overlap of genes implicated in other autoimmune diseases.[<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134062">4</a>, <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134072">5</a>] The Immunochip is a custom-designed array containing coverage of 186 established autoimmune susceptibility loci and extended coverage across the MHC.[<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134056">6</a>] In this study, we report an Immunochip analysis of 2,566 global IIM cases and 15,651 controls, representing the largest genetic association study to date in IIM.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Samples</h3><p id="P4" class="p p-first">2,954 samples from 14 countries were collected through the Myositis Genetics Consortium (MYOGEN), and written informed consent was obtained from all cases with approval from research ethics boards at each participating centre. There is overlap between these samples and previous IIM genetic studies.[<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134052">3</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134054">5</a>] IIM cases were included if they fulfilled probable or definite Bohan and Peter classification criteria for PM, juvenile PM (JPM), DM or Juvenile DM (JDM),[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134048">7</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134079">8</a>] and Griggs or European Neuromuscular Centre (ENMC) or Medical Research Council (MRC) criteria for IBM.[<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134065">9</a>–<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134071">11</a>] Eleven samples met the criteria for anti-synthetase syndrome,[<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134059">12</a>] however available clinical data was not able to differentiate between PM or DM. These were included in the combined IIM analysis, but removed from the clinical subgroup analysis.</p><p id="P5" class="p p-last">Immunochip control data from 12 countries was provided by four disease consortia (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary methods</a>).</p></div><div id="S4" class="sec"><h3 id="S4title">Genotyping and quality control</h3><p id="P6" class="p p-first-last">Genotyping was performed in accordance with Illumina’s protocols in the UK (Centre for Genetics and Genomics Arthritis Research UK, University of Manchester, UK) and the US (Feinstein Institute, New York, USA). Standard sample and SNP QC was performed in PLINK v1.07 (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary methods</a>).</p></div><div id="S5" class="sec"><h3 id="S5title">Statistical Analysis</h3><p id="P7" class="p p-first-last">Statistical analysis was performed in PLINK v1.07 using a logistic regression applying an additive model, including the top ten principal components as covariates to control for population stratification. Evidence for additional independent effects was investigated using a stepwise logistic regression including the most associated variants as covariates in subsequent conditional analyses.</p></div><div id="S6" class="sec"><h3 id="S6title">Functional annotation</h3><p id="P8" class="p p-first-last">Associated loci were interrogated for potentially causative variants using eQTL databases, and the functional prediction tools PolyPhen-2,[<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589361910">13</a>] SIFT,[<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134080">14</a>] and phastCons17-way[<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589361911">15</a>] (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary methods</a>).</p></div><div id="S7" class="sec sec-last"><h3 id="S7title">MHC imputation and association analysis</h3><p id="P9" class="p p-first-last">Classical human leukocyte antigen (<em>HLA</em>) alleles and corresponding amino acid sequences were imputed using SNP2HLA. A logistic regression assuming an additive model was used to test for association, and forward stepwise logistic regression was used to test for independent effects (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary methods</a>). Classical 4-digit HLA alleles were preferentially reported, unless an amino acid association explained more risk than HLA alleles alone.</p></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Results</h2><div id="S9" class="sec sec-first"><h3 id="S9title">Genotyping quality control</h3><p id="P10" class="p p-first-last">After stringent SNP and sample quality control we analysed 90,536 genetic variants in 2,566 IIM cases and 15,651 controls of Caucasian descent (<a href="/pmc/articles/PMC5300750/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). A breakdown of this cohort by clinical subgroup is reported in <a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>. Australia, Denmark and Switzerland did not have an ethnically matched control group; however, these were adequately matched by existing cohorts (UK, Sweden, and Germany respectively). By including the top ten principal components as covariates and calculating the genomic inflation on a set of null SNPs (from a study investigating the genetic basis for reading and writing ability)[<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134074">16</a>] on the Immunochip gave a λGC<sub>1000</sub> = 1.05, indicating that cases and controls are well matched for ethnicity (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary Figure S1</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p23">Number of IIM cases included in the analysis.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140647808199776" class="xtable"><table frame="hsides" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Number of Cases</th><th colspan="3" rowspan="1" valign="top" align="center">Number of Controls</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">Australia</td><td rowspan="1" colspan="1" valign="top" align="center">120</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Belgium</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">Belgium</td><td rowspan="1" colspan="1" valign="top" align="center">351</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Czech Republic</td><td rowspan="1" colspan="1" valign="top" align="center">236</td><td rowspan="1" colspan="1" valign="top" align="center">Poland</td><td rowspan="1" colspan="1" valign="top" align="center">526</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Denmark</td><td rowspan="1" colspan="1" valign="top" align="center">53</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">France</td><td rowspan="1" colspan="1" valign="top" align="center">37</td><td rowspan="1" colspan="1" valign="top" align="center">France</td><td rowspan="1" colspan="1" valign="top" align="center">497</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Hungary</td><td rowspan="1" colspan="1" valign="top" align="center">209</td><td rowspan="1" colspan="1" valign="top" align="center">Hungary<br>Germany</td><td rowspan="1" colspan="1" valign="top" align="center">257<br>1029</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Italy</td><td rowspan="1" colspan="1" valign="top" align="center">37</td><td rowspan="1" colspan="1" valign="top" align="center">Italy<br>Italy (RAF cohort)</td><td rowspan="1" colspan="1" valign="top" align="center">969<br>813</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Netherlands</td><td rowspan="1" colspan="1" valign="top" align="center">38</td><td rowspan="1" colspan="1" valign="top" align="center">Netherlands</td><td rowspan="1" colspan="1" valign="top" align="center">2020</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Norway</td><td rowspan="1" colspan="1" valign="top" align="center">63</td><td rowspan="1" colspan="1" valign="top" align="center">Norway</td><td rowspan="1" colspan="1" valign="top" align="center">730</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Sweden</td><td rowspan="1" colspan="1" valign="top" align="center">269</td><td rowspan="1" colspan="1" valign="top" align="center">Sweden</td><td rowspan="1" colspan="1" valign="top" align="center">1938</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Spain</td><td rowspan="1" colspan="1" valign="top" align="center">73</td><td rowspan="1" colspan="1" valign="top" align="center">Spain</td><td rowspan="1" colspan="1" valign="top" align="center">409</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Switzerland</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">United Kingdom</td><td rowspan="1" colspan="1" valign="top" align="center">993</td><td rowspan="1" colspan="1" valign="top" align="center">United Kingdom</td><td rowspan="1" colspan="1" valign="top" align="center">4332</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">United States</td><td rowspan="1" colspan="1" valign="top" align="center">423</td><td rowspan="1" colspan="1" valign="top" align="center">United States</td><td rowspan="1" colspan="1" valign="top" align="center">1780</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>Total</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>2,566</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>Total</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>15,651</em></strong></td></tr></tbody></table></div><div id="largeobj_idm140647808199776" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5300750/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p24" class="p p-first-last">Number of IIM cases after QC, by country of origin. Control samples shared from Immunochip consortia matched by closest ethnicity.</p></div></div></div></div><div id="S10" class="sec"><h3 id="S10title"><em>HLA</em> and <em>PTPN22</em> are the most strongly associated regions in IIM</h3><p id="P11" class="p p-first">Two regions in this study reached genome-wide significance (p&lt;5×10<sup>−8</sup>) (<a href="/pmc/articles/PMC5300750/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a> and <a href="/pmc/articles/PMC5300750/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). As expected, the most strongly associated region was within the MHC (p=9×10<sup>−133</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary Figure S2</a>). HLA-imputation was performed separately on this locus.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5300750/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140647808204560" class="figure"><a href="/pmc/articles/PMC5300750/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5300750_nihms847779f1a.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms847779f1a.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5300750/bin/nihms847779f1a.jpg"></a></div><div id="largeobj_idm140647808204560" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5300750/figure/F1/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140647808203616" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5300750_nihms847779f1b.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms847779f1b.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5300750/bin/nihms847779f1b.jpg"></a></div><div id="largeobj_idm140647808203616" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5300750/figure/F1/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140647808202672" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5300750_nihms847779f1c.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms847779f1c.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5300750/bin/nihms847779f1c.jpg"></a></div><div id="largeobj_idm140647808202672" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5300750/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5300750/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Manhattan plots of the IIM, PM and DM+JDM analyses, with the MHC region removed</strong><p id="__p22">The red and blue lines represent genome-wide level of significance (p=5×10<sup>−8</sup>) and suggestive significance (p=2.25×10<sup>−5</sup>) respectively. A) Analysis of 2,566 IIM cases and 15,651 controls. B) Analysis of 931PM cases and 15,651 controls. C) Analysis of 1,360 DM+JDM cases and 15,651 controls.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p25">Loci associated with IIM, PM, DM and JDM cases</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140647810541184" class="xtable"><table frame="hsides" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Subgroup</th><th rowspan="1" colspan="1" valign="top" align="center">Gene Region</th><th rowspan="1" colspan="1" valign="top" align="center">Chr</th><th rowspan="1" colspan="1" valign="top" align="center">Position</th><th rowspan="1" colspan="1" valign="top" align="center">Most Significant SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Minor Allele</th><th rowspan="1" colspan="1" valign="top" align="center">MAF cases</th><th rowspan="1" colspan="1" valign="top" align="center">MAF controls</th><th rowspan="1" colspan="1" valign="top" align="center">P-value</th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center">Localisation of LD to nearest genes (r2 ≥ 0.9)</th><th rowspan="1" colspan="1" valign="top" align="center">Overlap with other autoimmune diseases</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>IIM</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>HLA</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>6</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>32395726</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>rs3129843</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>G</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>0.24</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>0.11</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>9.14×10<sup>−133</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>2.74 (2.53–2.97)</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>MHC</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>Multiple</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>IIM</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>PTPN22</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>1</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>114377568</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>rs2476601</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>A</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>0.12</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>0.09</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>7.22×10<sup>−9</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>1.32 (1.20–1.45)</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>Exon 12 of PTPN22</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>ATD,</strong>[<a href="#R17" rid="R17" class=" bibr popnode">17</a>] <strong>IBD,</strong>[<a href="#R16" rid="R16" class=" bibr popnode">16</a>] <strong>JIA,</strong>[<a href="#R18" rid="R18" class=" bibr popnode">18</a>] <strong>RA,</strong>[<a href="#R19" rid="R19" class=" bibr popnode">19</a>] <strong>SLE,</strong>[<a href="#R20" rid="R20" class=" bibr popnode">20</a>] <strong>SSC,</strong>[<a href="#R21" rid="R21" class=" bibr popnode">21</a>] <strong>T1D,</strong>[<a href="#R22" rid="R22" class=" bibr popnode">22</a>] <strong>VIT,</strong>[<a href="#R23" rid="R23" class=" bibr popnode">23</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">IIM</td><td rowspan="1" colspan="1" valign="top" align="center">YDJC</td><td rowspan="1" colspan="1" valign="top" align="center">22</td><td rowspan="1" colspan="1" valign="top" align="center">21985094</td><td rowspan="1" colspan="1" valign="top" align="center">rs5754467</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">4.67×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21 (1.12–1.30)</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 1 of UBE2L3 | complete YDJC</td><td rowspan="1" colspan="1" valign="top" align="center">CEL,[<a href="#R6" rid="R6" class=" bibr popnode">6</a>] IBD,[<a href="#R16" rid="R16" class=" bibr popnode">16</a>] JIA,[<a href="#R18" rid="R18" class=" bibr popnode">18</a>] PSO,[<a href="#R24" rid="R24" class=" bibr popnode">24</a>] RA,[<a href="#R19" rid="R19" class=" bibr popnode">19</a>] SLE,[<a href="#R20" rid="R20" class=" bibr popnode">20</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">IIM</td><td rowspan="1" colspan="1" valign="top" align="center">DGKQ</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">956047</td><td rowspan="1" colspan="1" valign="top" align="center">rs6599390</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">6.48×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.85 (0.79–0.90)</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 21 of DGKQ to intron 2 of SLC26A1 and intron 2 of IDUA</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">IIM</td><td rowspan="1" colspan="1" valign="top" align="center">STAT4</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">191917317</td><td rowspan="1" colspan="1" valign="top" align="center">rs4853540</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.57×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.83 (0.77–0.89)</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 3 to intron 14 of STAT4</td><td rowspan="1" colspan="1" valign="top" align="center">JIA,[<a href="#R18" rid="R18" class=" bibr popnode">18</a>] PBC,[<a href="#R25" rid="R25" class=" bibr popnode">25</a>] RA,[<a href="#R19" rid="R19" class=" bibr popnode">19</a>] SLE,[<a href="#R20" rid="R20" class=" bibr popnode">20</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">IIM</td><td rowspan="1" colspan="1" valign="top" align="center">MGAT4A</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">99389870</td><td rowspan="1" colspan="1" valign="top" align="center">rs10189330</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">2.68×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.16 (1.09–1.23)</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic of C2orf55 (KIAA1211L) and MGAT4A incorporating uncharacterized LOC101927070</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">IIM</td><td rowspan="1" colspan="1" valign="top" align="center">PRR5L | TRAF6</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">36492191</td><td rowspan="1" colspan="1" valign="top" align="center">rs570676</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">0.40</td><td rowspan="1" colspan="1" valign="top" align="center">9.42×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.87 (0.82–0.92)</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic PRR5L (also known as FLJ14213) | complete TRAF6</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">IIM</td><td rowspan="1" colspan="1" valign="top" align="center">CCL17</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">57445376</td><td rowspan="1" colspan="1" valign="top" align="center">rs223900</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">9.97×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.17 (1.09–1.25)</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 1 of CCL17</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">IIM</td><td rowspan="1" colspan="1" valign="top" align="center">EOMES</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">28076283</td><td rowspan="1" colspan="1" valign="top" align="center">rs376072</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">1.45×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.86 (0.80–0.92)</td><td rowspan="1" colspan="1" valign="top" align="center">310.68kb upstream of EOMES</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">IIM</td><td rowspan="1" colspan="1" valign="top" align="center">CD28</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">204592021</td><td rowspan="1" colspan="1" valign="top" align="center">rs3116494</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">1.54×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.16 (1.09–1.24)</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 1 to 8.5kb downstream of CD28</td><td rowspan="1" colspan="1" valign="top" align="center">PSC,[<a href="#R26" rid="R26" class=" bibr popnode">26</a>] RA,[<a href="#R19" rid="R19" class=" bibr popnode">19</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">IIM</td><td rowspan="1" colspan="1" valign="top" align="center">RPL31P10</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">6523249</td><td rowspan="1" colspan="1" valign="top" align="center">rs11064180</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">1.61×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.87 (0.82–0.93)</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic of LTBR and CD27</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>PM</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>HLA</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>6</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>31434366</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>rs3094013</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>T</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>0.25</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>0.12</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>6.36×10<sup>−76</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>2.97 (2.64–3.33)</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>MHC</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>Multiple</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>PM</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>PTPN22</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>1</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>114377568</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>rs2476601</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>A</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>0.12</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>0.09</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>7.90×10<sup>−11</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>1.58 (1.38–1.81)</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>Exon 12 of PTPN22</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>ATD,</strong>[<a href="#R17" rid="R17" class=" bibr popnode">17</a>] <strong>IBD,</strong>[<a href="#R16" rid="R16" class=" bibr popnode">16</a>] <strong>JIA</strong>[<a href="#R18" rid="R18" class=" bibr popnode">18</a>], <strong>RA,</strong>[<a href="#R19" rid="R19" class=" bibr popnode">19</a>] <strong>SLE,</strong>[<a href="#R20" rid="R20" class=" bibr popnode">20</a>] <strong>SSC,</strong>[<a href="#R21" rid="R21" class=" bibr popnode">21</a>] <strong>T1D,</strong>[<a href="#R22" rid="R22" class=" bibr popnode">22</a>] <strong>VIT,</strong>[<a href="#R23" rid="R23" class=" bibr popnode">23</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">PM</td><td rowspan="1" colspan="1" valign="top" align="center">LOC728073 | RPL38</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">71527243</td><td rowspan="1" colspan="1" valign="top" align="center">rs9905921</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">2.01×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.26 (1.14–1.39)</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic (provisional in refseq) of SDK2</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">PM</td><td rowspan="1" colspan="1" valign="top" align="center">UBE3B | MMAB</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">109980516</td><td rowspan="1" colspan="1" valign="top" align="center">rs7956536</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">3.66×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.80 (0.72–0.88)</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 19 of MYO1H, complete KCTD10; UBE3B; MMAB; MVK</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">PM</td><td rowspan="1" colspan="1" valign="top" align="center">NAB1</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">191535576</td><td rowspan="1" colspan="1" valign="top" align="center">rs2286896</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">3.76×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.35 (1.19–1.53)</td><td rowspan="1" colspan="1" valign="top" align="center">Complete NAB1</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">PM</td><td rowspan="1" colspan="1" valign="top" align="center">FAM167A | BLK</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">11333521</td><td rowspan="1" colspan="1" valign="top" align="center">rs17799348</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">4.13×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.80 (0.71–0.87)</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic FAM167A | BLK</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">PM</td><td rowspan="1" colspan="1" valign="top" align="center">IL18R1</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">103012902</td><td rowspan="1" colspan="1" valign="top" align="center">rs1420095</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">6.16×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.63 (0.52–0.78)</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 3 of IL18R1 : 1L18RAP</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">PM</td><td rowspan="1" colspan="1" valign="top" align="center">SLC26A1 | IDUA</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">980464</td><td rowspan="1" colspan="1" valign="top" align="center">rs4690220</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">7.47×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.25 (1.13–1.37)</td><td rowspan="1" colspan="1" valign="top" align="center">2.14 Kb 3’ to intron 2 of SLC26A1</td><td rowspan="1" colspan="1" valign="top" align="center">SJO,[<a href="#R27" rid="R27" class=" bibr popnode">27</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">PM</td><td rowspan="1" colspan="1" valign="top" align="center">RGS1</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">192545099</td><td rowspan="1" colspan="1" valign="top" align="center">rs7535818</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">1.37×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.74 (0.65–0.85)</td><td rowspan="1" colspan="1" valign="top" align="center">25kb upstream to intron 1 of RGS1</td><td rowspan="1" colspan="1" valign="top" align="center">CEL,[<a href="#R6" rid="R6" class=" bibr popnode">6</a>] MS,[<a href="#R28" rid="R28" class=" bibr popnode">28</a>] T1D,[<a href="#R22" rid="R22" class=" bibr popnode">22</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>DM+JDM</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>HLA</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>6</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>32395726</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>rs3129843</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>G</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>0.21</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>0.11</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>1.72×10<sup>−48</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>2.18 (1.97–2.42)</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>MHC</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>Multiple</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DM+JDM</td><td rowspan="1" colspan="1" valign="top" align="center">ROPN1L | ANKRD33B</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">10517908</td><td rowspan="1" colspan="1" valign="top" align="center">rs4702698</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">4.77×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.22 (1.12–1.33)</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic of ROPN1L and ANKRD33B</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DM+JDM</td><td rowspan="1" colspan="1" valign="top" align="center">PTTG1 | ATP10B</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">159928876</td><td rowspan="1" colspan="1" valign="top" align="center">rs4921293</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">8.27×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21 (1.11–1.31)</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic of PTTG1 and ATP10B</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DM+JDM</td><td rowspan="1" colspan="1" valign="top" align="center">GSDMB</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">38066267</td><td rowspan="1" colspan="1" valign="top" align="center">rs1008723</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">9.05×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.11–1.30)</td><td rowspan="1" colspan="1" valign="top" align="center">Complete IKZF3, ZPBP2 GSDMB</td><td rowspan="1" colspan="1" valign="top" align="center">IBD,[<a href="#R16" rid="R16" class=" bibr popnode">16</a>] MS,[<a href="#R28" rid="R28" class=" bibr popnode">28</a>] PBC,[<a href="#R25" rid="R25" class=" bibr popnode">25</a>] RA,[<a href="#R29" rid="R29" class=" bibr popnode">29</a>] T1D,[<a href="#R30" rid="R30" class=" bibr popnode">30</a>]</td></tr></tbody></table></div><div id="largeobj_idm140647810541184" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5300750/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="__p26" class="p p-first-last">Analysis of 2,566 IIM cases, 931 PM cases, and 1,360 DM and JDM cases, compared to 15,651 controls. Loci reported at genome wide significance p &lt;5×10<sup>−8</sup> (bold) or at suggestive significance p &lt;2.25×10<sup>−5</sup>. Coordinates based on GRCh37 assembly. Overlap defined as high LD (r<sup>2</sup> &gt;0.7) between the associations in IIM (including independent effects) and the lead association in another autoimmune disease (as reported on <a href="http://www.immunobase.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.immunobase.org</a>). ATD – Autoimmune Thyroid Disease, CEL – Celiac Disease, Chr – chromosome, IBD – Inflammatory Bowel Disease, JIA – Juvenile Idiopathic Arthritis, MAF – minor allele frequency, MS – Multiple Sclerosis, OR – Odds ratio, PBC – Primary Biliary Cirrhosis, PSC - Primary Sclerosing Cholangitis, PSO – Psoriasis, RA – Rheumatoid Arthritis, SJO - Sjogren Syndrome, SLE – Systemic Lupus Erythematosus, SSC - Systemic Scleroderma, T1D – Type 1 Diabetes, VIT – Vitiligo, 95% CI – 95% confidence interval</p></div></div></div><p id="P12" class="p p-last">The other region reaching genome-wide significance was within the <em>PTPN22</em> locus (rs2476601; p=7.22×10<sup>−9</sup>), an established autoimmune risk locus. This SNP/locus has been previously associated in an IIM candidate gene study,[<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134050">3</a>] but was not associated in a GWAS in DM.[<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134078">4</a>]</p></div><div id="S11" class="sec"><h3 id="S11title">A further nine regions were associated at a suggestive level of significance</h3><p id="P13" class="p p-first">We next investigated associations reaching our suggestive tier of association (p=2.25×10<sup>−5</sup>) calculated using the genetic Type 1 Error Calculator.[<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134044">31</a>] This estimates the effective number of independent tests based on the LD between SNPs contained on the genotyping array. Here, we found evidence of a further nine associated loci (<a href="/pmc/articles/PMC5300750/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P14">The third most strongly associated SNP in this analysis was in the <em>YDJC</em> gene (rs5754467) on chromosome 22 (4.67×10<sup>−7</sup>). This SNP tags a large haplotype block containing <em>UBE2L3</em>, an established autoimmune risk locus.</p><p id="P15"><em>STAT4</em> is a susceptibility locus for many autoimmune diseases. The lead SNP in this region was protective, which is a novel finding in IIM (rs4853540, p=1.57×10<sup>−6</sup>). Stepwise logistic regression analysis in this region suggested an independent risk effect of rs10174238 (p=1.08×10<sup>−5</sup>, OR=1.17, 95% CI=1.09–1.26) (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary Figure S3</a>) and a further potential independent effect was seen at rs932169 (p=2.88×10<sup>−5</sup>, OR=1.25, 95% CI=1.13–1.39).</p><p id="P16" class="p p-last">Further variants reaching our suggestive significance threshold reveal loci of interest that have been previously associated with autoimmune disease, including DGKQ, EOMES, CD28 and PRR5L/TRAF6. Associated SNPs that tag risk haplotypes (r2 &gt;0.7) in other autoimmune diseases are reported in <a href="/pmc/articles/PMC5300750/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, and the direction of effect is reported in <a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary Table S2</a>.</p></div><div id="S12" class="sec"><h3 id="S12title">Subgroup analysis reveals unique associations within PM and DM</h3><p id="P17" class="p p-first-last">We stratified our cohort by the two largest subgroups within IIM, consisting of 931 adult PM cases (<a href="/pmc/articles/PMC5300750/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>, <a href="/pmc/articles/PMC5300750/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) and 1,360 DM cases (<a href="/pmc/articles/PMC5300750/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1C</span></a>, <a href="/pmc/articles/PMC5300750/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). JDM cases were included in the DM analysis both to increase power, and on previous evidence that there is not extensive genetic heterogeneity between the subgroups.[<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134070">4</a>] The only non-HLA region to reach genome-wide significance in either subgroup was <em>PTPN22</em> in PM (rs2476601, p=7.9×10<sup>−11</sup>). Interestingly, with a smaller sample size, the association in PM with <em>PTPN22</em> was stronger than in the combined IIM analysis. There was no evidence of association in DM (p=0.19), therefore the stronger effect in PM appears to be driving the association in the combined IIM analysis. Other interesting regions reaching a suggestive level of significance were <em>SL26A1/IDUA</em> and <em>RGS1</em> in PM, and <em>GSDMB</em> in DM (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary Table S3</a>).</p></div><div id="S13" class="sec"><h3 id="S13title">Exonic and eQTL SNPs suggest potential causal variants</h3><p id="P18" class="p p-first-last">Potential functionally relevant variants were investigated for non-synonymous SNPs (<a href="/pmc/articles/PMC5300750/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>) or eQTLs (<a href="/pmc/articles/PMC5300750/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>) that are tagged by the lead SNP (r<sup>2</sup> &gt; 0.9). Two variants within the <em>GSDMB</em> gene, suggestively associated in DM, are ‘potentially damaging’ as predicted by PolyPhen-2. The <em>PTPN22</em> variant is confirmed to be conserved across vertebrates, as well as a SNP in <em>UBE3B</em>. Evidence for eQTLs in cells with immune function (lymphoblastoid cell lines and monocytes) was found in six loci and may help annotate our associations, for example, the association with <em>NAB1</em> in PM may be due to an eQTL affecting the expression of <em>STAT1,</em> 275Kb upstream.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p27">Potentially causal exonic SNPs.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140647812074928" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Subgroup</th><th rowspan="1" colspan="1" valign="top" align="center">Mapped Genes</th><th rowspan="1" colspan="1" valign="top" align="center">Most Significant SNP</th><th rowspan="1" colspan="1" valign="top" align="center">SNP in LD</th><th rowspan="1" colspan="1" valign="top" align="center">SNP Location</th><th rowspan="1" colspan="1" valign="top" align="center">Amino acid change</th><th rowspan="1" colspan="1" valign="top" align="center">PolyPhen-2/SIFT</th><th rowspan="1" colspan="1" valign="top" align="center">phastCons17-way</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>IIM + PM</strong></td><td rowspan="1" colspan="1" valign="top" align="center">PTPN22</td><td rowspan="1" colspan="1" valign="top" align="center">rs2476601</td><td rowspan="1" colspan="1" valign="top" align="center">rs2476601</td><td rowspan="1" colspan="1" valign="top" align="center">Non-synonymous coding in PTPN22</td><td rowspan="1" colspan="1" valign="top" align="center">R620W</td><td rowspan="1" colspan="1" valign="top" align="center">Benign/tolerated</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>IIM</strong></td><td rowspan="1" colspan="1" valign="top" align="center">DGKQ</td><td rowspan="1" colspan="1" valign="top" align="center">rs6599390</td><td rowspan="1" colspan="1" valign="top" align="center">rs3796622</td><td rowspan="1" colspan="1" valign="top" align="center">Non-synonymous coding SLC26A1</td><td rowspan="1" colspan="1" valign="top" align="center">Q556R</td><td rowspan="1" colspan="1" valign="top" align="center">Benign/tolerated</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>PM</strong></td><td rowspan="1" colspan="1" valign="top" align="center">UBE3B | MMAB</td><td rowspan="1" colspan="1" valign="top" align="center">rs7956536</td><td rowspan="1" colspan="1" valign="top" align="center">rs7298565</td><td rowspan="1" colspan="1" valign="top" align="center">Non-synonymous coding UBE3B</td><td rowspan="1" colspan="1" valign="top" align="center">R346Q</td><td rowspan="1" colspan="1" valign="top" align="center">Benign/tolerated</td><td rowspan="1" colspan="1" valign="top" align="center">1</td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">rs9593</td><td rowspan="1" colspan="1" valign="top" align="center">Non-synonymous coding in MMAB</td><td rowspan="1" colspan="1" valign="top" align="center">M239K</td><td rowspan="1" colspan="1" valign="top" align="center">Benign/tolerated</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>DM+JDM</strong></td><td rowspan="1" colspan="1" valign="top" align="center">GSDMB</td><td rowspan="1" colspan="1" valign="top" align="center">rs1008723</td><td rowspan="1" colspan="1" valign="top" align="center">rs11078928</td><td rowspan="1" colspan="1" valign="top" align="center">Splice acceptor variant in GSDMB</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">rs2305480</td><td rowspan="1" colspan="1" valign="top" align="center">Non-synonymous coding GSDMB</td><td rowspan="1" colspan="1" valign="top" align="center">P298S</td><td rowspan="1" colspan="1" valign="top" align="center">Possibly damaging/Tolerated</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">rs2305479</td><td rowspan="1" colspan="1" valign="top" align="center">Non-synonymous coding GSDMB</td><td rowspan="1" colspan="1" valign="top" align="center">G291R</td><td rowspan="1" colspan="1" valign="top" align="center">Probably damaging/Tolerated</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr></tbody></table></div><div id="largeobj_idm140647812074928" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5300750/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="__p28" class="p p-first-last">SNPs investigated for functionality may be the lead association or any SNP with r<sup>2</sup> ≥0.9. PolyPhen-2,[<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589361908">13</a>] and SIFT,[<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134076">14</a>] was used to predict the possible impact of an amino acid substitution. phastCons17-way,[<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589361909">15</a>] gives a score of evolutionary conservation in 17 vertebrates, with 1 being the most conserved at that base position.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p29">Evidence of expression quantitative trait loci (eQTL)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140647812151872" class="xtable"><table frame="hsides" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Subgroup</th><th rowspan="1" colspan="1" valign="top" align="center">Mapped Genes</th><th rowspan="1" colspan="1" valign="top" align="center">Immunochip lead SNP</th><th rowspan="1" colspan="1" valign="top" align="center">eQTL within r<sup>2</sup> &gt;0.9</th><th rowspan="1" colspan="1" valign="top" align="center">References</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>IIM</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><em>UBE2L3</em> | <em>YDJC</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs11089637</td><td rowspan="1" colspan="1" valign="top" align="center"><em>UBE2L3</em>, <em>RIMBP3</em></td><td rowspan="1" colspan="1" valign="top" align="center">[<a href="#R32" rid="R32" class=" bibr popnode">32</a>] [<a href="#R33" rid="R33" class=" bibr popnode">33</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>PM</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><em>UBE3B</em> | <em>MMAB</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs7956536</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KCTD10</em>, <em>MMAB</em></td><td rowspan="1" colspan="1" valign="top" align="center">[<a href="#R32" rid="R32" class=" bibr popnode">32</a>] [<a href="#R33" rid="R33" class=" bibr popnode">33</a>] [<a href="#R34" rid="R34" class=" bibr popnode">34</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>PM</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><em>RGS1</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs7535818</td><td rowspan="1" colspan="1" valign="top" align="center"><em>RGS1</em></td><td rowspan="1" colspan="1" valign="top" align="center">[<a href="#R32" rid="R32" class=" bibr popnode">32</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>PM</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><em>NAB1</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs2286896</td><td rowspan="1" colspan="1" valign="top" align="center"><em>STAT1</em></td><td rowspan="1" colspan="1" valign="top" align="center">[<a href="#R35" rid="R35" class=" bibr popnode">35</a>]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>DM+JDM</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><em>GSDMB</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs1008723</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ORMDL3, MED24, KRT222, NR1D1, GSDMB</em></td><td rowspan="1" colspan="1" valign="top" align="center">[<a href="#R34" rid="R34" class=" bibr popnode">34</a>] [<a href="#R35" rid="R35" class=" bibr popnode">35</a>] [<a href="#R36" rid="R36" class=" bibr popnode">36</a>] [<a href="#R33" rid="R33" class=" bibr popnode">33</a>]</td></tr></tbody></table></div><div id="largeobj_idm140647812151872" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5300750/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><p id="__p30" class="p p-first-last">eQTLs for the expression of a gene in cells with an immune function (Lymphoblastoid and Monocytes). RegulomeDB,[<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589361912">37</a>] Genevar,[<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134043">38</a>] and the Pritchard lab eQTL browser (<a href="http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/</a>) were interrogated. The most significant SNP or any SNP with r<sup>2</sup> ≥0.9 with most significant SNP was used. Genes are indicated along with studies in which the eQTLs were reported.</p></div></div></div></div><div id="S14" class="sec sec-last"><h3 id="S14title">HLA Imputation confirms alleles of the 8.1 ancestral haplotype as the strongest association in IIM</h3><p id="P19" class="p p-first-last">Due to the complex linkage disequilibrium/haplotype structure in the MHC, interpretation of causal associations and independent effects using SNPs may be inadequate. We used SNP2HLA to impute classical HLA alleles and amino acids from SNP genotyping information. For each analysis, all variants reaching statistical significance (p&lt;6.8×10<sup>−6</sup>) after each round of conditioning are included in <a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary Table S4–S15</a>. For many associations, amino acids unique to classical HLA risk alleles were associated at similar levels of significance to the HLA allele. For consistency, 4-digit HLA alleles are reported, unless an amino acid is significantly more associated than individual HLA alleles. In the combined IIM analysis (n=2,566), the most associated variants were classical HLA alleles, with <em>HLA-DRB1</em>*03:01 being the most significant 4-digit allele (p=2.58×10<sup>−135</sup>, OR=1.88, 95% CI=1.68–2.11). <em>HLA-DRB1</em>*03:01 forms part of the 8.1 AH which has been consistently associated with IIM. After conditioning on the effects of <em>HLA-DRB1</em>*03:01, a strong association was found with <em>HLA-B</em>*08:01 (p=3.23×10<sup>−14</sup>, OR=1.58, 95% CI=1.41–1.78) suggesting that there is an independent effect within this locus. Further residual associations highlight the heterogeneity within this cohort, so analysis was then conducted on clinical subgroups. Similar associations were found with PM (n=931), <em>HLA-DRB1</em>*03:01 being the most significant 4-digit allele (p=6.11×10<sup>−80</sup>, OR=1.99, 95% CI=1.67–2.36) and an independent effect with HLA-B*08:01 (p=4.17×10<sup>−9</sup>, OR=1.71, 95% CI=1.43–2.05). As the effect size of the HLA is strong in IIM, we hypothesised that we may be able to detect any potential differences between adult and juvenile DM, even with a reduced sample size. In adult DM (n=879), <em>HLA-B*</em>08:01 was the most significant allele (p=2.46×10<sup>−42</sup>, OR=1.90, 95% CI=1.66–2.17). Conditioning on <em>HLA-B*</em>08:01, there was evidence of multiple independent effects within the <em>HLA-DQB1</em> locus, therefore we analysed imputed amino acid residues. Amino acid position 57 of <em>HLA-DQB1</em> was more significantly associated with DM than individual <em>HLA-DQB1</em> alleles (p=8.95×10<sup>−14</sup>), with alanine (p=1.29×10<sup>−12</sup> OR=1.62, 95% CI=1.44–1.83) and serine (p=9.28×10<sup>−7</sup>, OR=2.15, 95% CI=1.60–2.84) conferring the greatest risk. Further association with <em>HLA-DQB1</em> remains after conditioning, notably an independent effect of <em>HLA-DQB1*</em>04:02 (p=2.01×10<sup>−6</sup>, OR=1.99, 95% CI=1.52–2.58). In the JDM subgroup (n=481), <em>HLA-DRB1</em>*03:01 was the most associated allele (p=7.91×10<sup>14</sup>, OR=1.90, 95% CI=1.61–2.22) and an independent association was observed with <em>HLA-C</em>*02:02 (p=3.28×10<sup>−7</sup>, OR=1.99, 95% CI=1.55–2.52) which is not a part of the 8.1 ancestral haplotype.</p></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Discussion</h2><p id="P20" class="p p-first">This is the largest genetic study to date in IIM, and has revealed several novel suggestive associations in adult and juvenile IIM emphasising the autoimmune architecture of these diseases. We have confirmed <em>HLA</em> and <em>PTPN22</em> as the most strongly associated regions in IIM, and identified nine additional associations at a suggestive level of significance. Subgroup analysis has identified distinct differences between PM and DM. Identification of exonic and eQTL SNPs has localised association signals to several potential causal variants.</p><p id="P21">It is reassuring that associations such as <em>PTPN22</em>, <em>STAT4</em> and <em>UBE2L3</em> follow a similar genetic profile as reported in other autoimmune diseases. The most significantly associated SNP in the <em>PTPN22</em> region is the rs2476601 variant, a C&gt;T polymorphism that results in a non-synonymous arginine (R) to tryptophan (W) amino acid change at position 620. Although this SNP has been extensively studied in the context of autoimmunity, there is no consensus regarding the functional consequences of this SNP. Some studies report a gain of function mutation by enhancing the inhibitory effect on TCR signalling,[<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134046">39</a>] while others report a loss of function by increased degradation of the protein and a diminished inhibitory effect on T-cell activation.[<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134053">40</a>]</p><p id="P22"><em>STAT4</em> is an important transcription factor for many genes involved in T-cell differentiation and has previously been associated with DM in the Japanese population.[<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134075">41</a>] Stepwise logistic regression analysis was conducted on all regions in this study; however <em>STAT4</em> is the only locus with evidence of independent associations. The three independent SNPs are in LD with associations in different diseases. The lead SNP in <em>STAT4</em> is protective, and in moderate LD with protective SNPs in <em>STAT4</em> reported in inflammatory bowel disease (IBD), Crohn’s and ulcerative colitis.[<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134058">16</a>] The independent risk effect of rs10174238 is the same SNP reported in juvenile idiopathic arthritis,[<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134049">18</a>] and is in strong LD with disease-associated SNPs in RA and SLE.[<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134057">19</a>, <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134047">42</a>] A SNP in high LD with rs932169 has been reported to be associated with primary biliary cirrhosis.[<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134066">25</a>]</p><p id="P23">The most significantly associated SNP in the <em>YDJC</em> gene tags an established autoimmune susceptibility locus where <em>UBE2L3</em> is thought to be the causal gene.[<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134073">43</a>] This risk haplotype is thought to increase <em>UBE2L3</em> expression in B cells and monocytes and amplify NF-κB activation.[<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134061">43</a>]</p><p id="P24">Stratification by clinical subgroup revealed further novel suggestive associations. These distinct differences between PM and DM suggest different autoimmune pathways in these subsets of IIM. For example, when splitting the total IIM cohort into PM and DM, we have shown that the association with <em>PTPN22</em> is predominantly driven by a strong association with PM. For all associations, we have stratified by clinical subgroup and reported the summary statistics in <a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary Table S3</a>. IBM patients were included in the combined IIM analysis on the basis of their diagnosis as an inflammatory autoimmune myopathy, however we did not analyse this subgroup separately due to a lack of power (n=252). Removing this group from our analyses did not make any substantial difference in associated regions, however the strength of the signals were attenuated in most instances. With eight non-HLA loci reaching our level of suggestive significance in PM, and only 3 in DM, it may be that the Immunochip is explaining less of the genetic risk to DM. This may be due to lack of power, the selected content of the Immunochip, heterogeneity of phenotypes within DM, or a weaker genetic influence compared to other autoimmune diseases. Some previous reported associations with DM failed to replicate in this study. We looked for evidence of association with loci that have previously been associated with IIM that did not reach our suggestive level of significance. For example, in the DM and JDM subgroup analysis, an association was found with rs2618476 (p=3.2×10<sup>−5</sup>, OR=1.2, 95% CI=1.1–1.32), a SNP in B Lymphoid Tyrosine Kinase (<em>BLK</em>). rs2618476 is a proxy for a SNP that was associated with DM in the Japanese population,[<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134077">44</a>] and is also highly correlated (r<sup>2</sup>&gt;0.8) with associations found in SLE and RA.[<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134067">20</a>, <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134042">29</a>] With this knowledge, this association becomes more convincing, whereas the single association in the <em>FAM167A-BLK</em> region in the PM subgroup (rs17799348) that is independent of the established risk haplotype, is less so.</p><p id="P25">It is important to note that this study was conducted on Caucasian IIM individuals. While there is evidence that risk loci may be shared across populations, such as <em>STAT4</em> and <em>BLK</em> in the Japanese population, the association between <em>PTPN22</em> and autoimmune disease is unique to Caucasians as the R620W variant is rarely seen in Asian populations. There is therefore a need to conduct further genetic studies on different IIM populations.</p><p id="P26">The Immunochip contains a dense set of SNPs covering 186 loci based on evidence of association with 12 different autoimmune and inflammatory diseases.[<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134064">18</a>] IIM was not one of these diseases, so this study can be seen as an exploratory investigation to assess genetic overlap with other autoimmune diseases, rather than the identification of genes novel to IIM. With 2,566 samples, Immunochip studies of similar size have revealed multiple non-HLA associations reaching genome-wide significance.[<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134063">18</a>, <a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134068">45</a>] The fact that only a single locus reached this threshold may due to low statistical power owing to phenotypic heterogeneity within IIM. A more conservative level of significance (p&lt;2.25×10<sup>−5</sup>) revealed suggestive associations of interest. SNPs that are the same, or in high LD with established autoimmune variants, along with biological knowledge and/or evidence of functionality may lead us to pursue these associations with more confidence. Indeed, a recent Immunochip study in T1D calculated a Bayesian posterior probability of disease association &gt;0.9 of SNPs reaching a suggestive level of significance (p&lt;1×10<sup>−5</sup>) when there is evidence of genome-wide significance in other Immunochip studies.[<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134055">30</a>]</p><p id="P27">Due to the extended haplotypes that are present in the HLA region, for many associations, alleles carried on the same haplotype reached an equivalent significance level. For consistency, the most associated allele was used in the stepwise conditional analysis; however, this is not to say that the allele is causative. Interestingly in PM, two alleles frequently inherited together on the 8.1 AH (<em>HLA-DRB1</em>*03:01 and <em>HLA-B</em>*08:01) show evidence of having independent effects. This may also be the case in DM, however, after conditioning on <em>HLA-B</em>*08:01, the association with other alleles of the 8.1 AH did not reach genome-wide significance. In DM, the independent association with amino acid position 57 in <em>HLA-DQB1</em> may explain part of the risk within this gene. Indeed, this position is an established risk factor for T1D.[<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134069">46</a>] In this study, classical 4-digit HLA alleles were preferentially reported, unless an amino acid association explained more risk than HLA alleles alone. However looking at amino acids may give insight into functionality. For example, the association with <em>HLA-DRB1*03:01</em> may be explained by the presence of amino acids that are unique to that allele. An asparagine at position 77, and an arginine at position 74 also were highly associated with IIM (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary Table S4</a>), and these residues are predominantly found on <em>DRB1*03</em> alleles. As there are multiple residues unique to this allele, it is hard to tease out which positions may be functionally important; however the location of these amino acids in the <em>HLA-DRB1</em> molecule may give insight (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary Figure S4</a>). Amino acid position 74 of <em>HLA-DRB1</em> lies within the peptide binding groove, and almost all of the risk at this position can be explained by the presence of an arginine (p=3.1×10<sup>−72</sup>, OR=2.83, 95% CI=2.53–3.17). The location of Arg-74 may change the structure of the peptide binding groove in such a way as to accommodate autoantigenic peptides. Indeed Arg-74 is an established risk factor for the development of autoimmune diseases.[<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_589134051">47</a>] A similar phenomenon is seen with <em>HLA-B*08:01</em> and the occurrence of Phe-67 and Asp-9 (<a href="#SD1" rid="SD1" class=" supplementary-material">online supplementary Figure S5</a>).</p><p id="P28">Alleles of the 8.1 AH have frequently been associated with the presence of myositis autoantibodies such as anti-Jo-1 and anti-PM-Scl. It may be that the association with the 8.1 AH and IIM is due to the prevalence of these antibodies, and weak associations with other <em>HLA</em>-alleles may be due to associations with autoantibodies that are less frequent in the disease subgroup. Further work is planned to stratify patients by serotype to clarify these differences.</p><p id="P29" class="p p-last">This study has revealed new suggestive associations with IIM in the Caucasian population, and independent associations with PM and DM. and has shown that subgrouping patients into clinical subgroups is important to expand our knowledge of IIM. This international collaboration has made it possible to perform the largest study to date in IIM and it has considerably expanded our knowledge about the genetic architecture of this rare disease.</p></div><div id="S16" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary</h4><div class="sup-box half_rhythm" id="d36e1130"><a href="/pmc/articles/PMC5300750/bin/NIHMS847779-supplement-Supplementary.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(748K, pdf)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="__p1">We thank Mrs. Fiona Marriage (Centre for Integrated Genomic Medical Research, University of Manchester) for technical support, Drs. Elaine Remmers, Douglas Bell (National Institutes of Health) and Jonathan Massey (University of Manchester) for critical review of the manuscript, and Mr. Paul New (Salford Royal Foundation Trust) for ethical and technical support. We thank all of the patients and their families who contributed to this study.</p><p id="__p2">This publication was supported by researchers at the National Institute for Health Research (NIHR) Biomedical Research Unit Funding Scheme and the University College London Hospitals Biomedical Research Centre. The views expressed are those of the authors and not necessarily those of the (UK) National Health Service (NHS), the NIHR or the (UK) Department of Health.</p><p id="__p3"><span class="underline">Further MYOGEN Investigators</span></p><p id="__p4">Study investigators of the Myositis Genetics Consortium, in addition to the authors of this article, are as follows: Drs. Christopher Denton (Royal Free Hospital, London, UK), Herman Mann (Institute of Rheumatology, Prague), David Hilton-Jones (John Radcliffe Hospital, Oxford, UK), Patrick Kiely (St. George’s Hospital, London, UK), Paul H. Plotz, Mark Gourley (National Institute of Arthritis and Musculoskeletal and Skin Diseases, National Institutes of Health, Bethesda, MD), Kelly Rouster-Stevens (Emory University School of Medicine, Atlanta, GA), Adam M Huber (Dalhousie University, Halifax, Nova Scotia, Canada), Galina Marder (North Shore Univeristy Hospital, Great Neck, NY) and Mazen Dimachkie (University of Kansas Medical Center, Kansas City, KS).</p><p id="__p5"><span class="underline">Funding</span></p><p id="__p6">This study was supported in part by: Association Francaise Contre Les Myopathies (AFM), The European Union Sixth Framework Programme (project AutoCure; LSH-018661), European Science Foundation (ESF) in the framework of the Research Networking Programme European Myositis Network (EUMYONET), The Swedish Research Council and The regional agreement on medical training and clinical research (ALF) between Stockholm County Council and Karolinska Institutet, the intramural research program of the National Institute of Environmental Health Sciences (NIEHS), the National Institutes of Health (NIH); European Community’s FP6, AutoCure LSHB CT-2006-018661; The UK Myositis Support Group; Arthritis Research UK (18474); The Cure JM Foundation; the European Science Foundation; the Wellcome Trust; the Henry Smith Charity UK; Action Medical UK; and the Swedish Research Council. The Czech cohort was supported by Project for Conceptual Development of Research Organization 00023728 from Ministry of Health in the Czech Republic.</p><p id="__p7"><span class="underline">Shared Immunochip Control Cohorts</span></p><p id="__p8">We thank the Rheumatoid Arthritis Consortium International (RACI) for Netherlands, Spanish, Swedish, UK and US Immunochip control genotypes. This study makes use of data generated by the Wellcome Trust Case-Control Consortium. A full list of the investigators who contributed to the generation of the data is available from <a href="http://www.wtccc.org.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.wtccc.org.uk</a>. Funding for the project was provided by the Wellcome Trust under award 076113 and 085475. Swedish control data was provided from EIRA study, Professor Lars Alfredsson, Department of Environmental Medicine, Karolinska Institutet, Stockholm, Sweden. Control data from the Netherlands was provided from the department of Rheumatology, Leiden University Medical Center, Leiden, The Netherlands.</p><p id="__p9">Polish control data was provided by the Celiac Disease Consortium and Hungarian control data collected with the help of the Hungarian Research Fund (OTKA) grant K101788 was provided by Prof Ilma. Korponay-Szabo, Celiac Disease Centre, Heim Pál Children’s Hospital, Budapest and University of Debrecen, Debrecen Hungary.</p><p id="__p10">We acknowledge the International MS Genetics Consortium for providing access to control sample data from Belgium, France, Norway, Italy and Germany. The collection and genotyping of these samples was made possible by: the Norwegian MS society and the Norwegian Bone Marrow Registry (Norwegian samples); the French Biological Resource Center for MS Genetics, Genethon and INSERM (French samples); and a FISM (Italian Foundation for Multiple Sclerosis) grant (“Progetto Speciale Immunochip”) for Italian samples.</p><p id="__p11">Italian samples were collected by Prof. Sandra D’Alfonso (Interdisciplinary Research Center of Autoimmune Diseases IRCAD, University of Eastern Piedmont, Novara, Italy; PROGEMUS Consortium) and Dr. Martinelli Boneschi (Laboratory of Genetics of Complex Neurological Disorders, Division of Neuroscience &amp; INSPE, San Raffaele Scientific Institute, Milan, Italy; PROGRESSO Consortium); funding was provided by a FISM (Italian Foundation for Multiple Sclerosis) grant (“Progetto Speciale Immunochip). The KORA study was initiated and financed by the Helmholtz Zentrum München – German Research Center for Environmental Health, which is funded by the German Federal Ministry of Education and Research (BMBF) and by the State of Bavaria. Furthermore, KORA research was supported within the Munich Center of Health Sciences (MC-Health), Ludwig-Maximilians-Universität, as part of LMUinnovativ.</p><p id="__p12"><span class="underline">The SweMyoNet</span></p><p id="__p13">Maria Lidén (Uppsala University), Awat Jalal (Örebro University), Helena Hellström (Rheumatology Clinic Falun), Thomas Skogh (Linköping University), Aladdin Mohammad (Lund).</p><p id="__p14"><span class="underline">UK Adult Onset Myositis Immunogenetic Collaboration (UKMYONET)</span></p><p id="__p15">Members of the UK Adult Onset Myositis Immunogenetic Collaboration who recruited and enrolled subjects are as follows: Drs. Yasmeen Ahmed (Llandudno General Hospital), Raymond Armstrong (Southampton General Hospital), Robert Bernstein (Manchester Royal Infirmary), Carol Black (Royal Free Hospital, London), Simon Bowman (University Hospital, Birmingham), Ian Bruce (Manchester Royal Infirmary), Robin Butler (Robert Jones &amp; Agnes Hunt Orthopaedic Hospital, Oswestry), John Carty (Lincoln County Hospital), Chandra Chattopadhyay (Wrightington Hospital), Easwaradhas Chelliah (Wrightington Hospital), Fiona Clarke (James Cook University Hospital, Middlesborough), Peter Dawes (Staffordshire Rheumatology Centre, Stoke on Trent), Joseph Devlin (Pinderfields General Hospital, Wakefield), Christopher Edwards (Southampton General Hospital), Paul Emery (Academic Unit of Musculoskeletal Disease, Leeds), John Fordham (South Cleveland Hospital, Middlesborough), Alexander Fraser (Academic Unit of Musculoskeletal Disease, Leeds), Hill Gaston (Addenbrooke’s Hospital, Cambridge), Patrick Gordon (King’s College Hospital, London), Bridget Griffiths (Freeman Hospital, Newcastle), Harsha Gunawardena (Frenchay Hospital, Bristol), Frances Hall (Addenbrooke’s Hospital, Cambridge), Beverley Harrison (North Manchester General Hospital), Elaine Hay (Staffordshire Rheumatology Centre, Stoke on Trent), Lesley Horden (Dewsbury District General Hospital), John Isaacs (Freeman Hospital, Newcastle), Adrian Jones (Nottingham University Hospital), Sanjeet Kamath (Staffordshire Rheumatology Centre, Stoke on Trent), Thomas Kennedy (Royal Liverpool Hospital), George Kitas (Dudley Group Hospitals Trust, Birmingham), Peter Klimiuk (Royal Oldham Hospital), Sally Knights (Yeovil District Hospital, Somerset), John Lambert (Doncaster Royal Infirmary), Peter Lanyon (Queen’s Medical Centre, Nottingham), Ramasharan Laxminarayan (Queen’s Hospital, Burton Upon Trent), Bryan Lecky (Walton Neuroscience Centre, Liverpool), Raashid Luqmani (Nuffield Orthopaedic Centre, Oxford), Jeffrey Marks (Steeping Hill Hospital, Stockport), Michael Martin (St. James University Hospital, Leeds), Dennis McGonagle (Academic Unit of Musculoskeletal Disease, Leeds), Neil McHugh (Royal National Hospital for Rheumatic Diseases, Bath), Francis McKenna (Trafford General Hospital, Manchester), John McLaren (Cameron Hospital, Fife), Michael McMahon (Dumfries &amp; Galloway Royal Infirmary, Dumfries), Euan McRorie (Western General Hospital, Edinburgh), Peter Merry (Norfolk &amp; Norwich University Hospital, Norwich), Sarah Miles (Dewsbury &amp; District General Hospital, Dewsbury), James Miller (Royal Victoria Hospital, Newcastle), Anne Nicholls (West Suffolk Hospital, Bury St. Edmunds), Jennifer Nixon (Countess of Chester Hospital, Chester), Voon Ong (Royal Free Hospital, London), Katherine Over (Countess of Chester Hospital, Chester), John Packham (Staffordshire Rheumatology Centre, Stoke on Trent), Nicolo Pipitone (King’s College Hospital, London), Michael Plant (South Cleveland Hospital, Middlesborough), Gillian Pountain (Hinchingbrooke Hospital, Huntington), Thomas Pullar (Ninewells Hospital, Dundee), Mark Roberts (Salford Royal Foundation Trust), Paul Sanders (Wythenshawe Hospital, Manchester), David Scott (King’s College Hospital, London), David Scott (Norfolk &amp; Norwich University Hospital, Norwich), Michael Shadforth (Staffordshire Rheumatology Centre, Stoke on Trent), Thomas Sheeran (Cannock Chase Hospital, Cannock, Staffordshire), Arul Srinivasan (Broomfield Hospital, Chelmsford), David Swinson (Wrightington Hospital), Lee-Suan Teh (Royal Blackburn Hospital, Blackburn), Michael Webley (Stoke Manderville Hospital, Aylesbury), Brian Williams (University Hospital of Wales, Cardiff), and Jonathan Winer (Queen Elizabeth Hospital, Birmingham).</p><p id="__p16"><span class="underline">UK Juvenile Dermatomyositis Research Group</span></p><p id="__p17">Dr. Kate Armon, Mr. Joe Ellis-Gage, Ms. Holly Roper, Ms. Vanja Briggs, and Ms. Joanna Watts (Norfolk and Norwich University Hospitals); Dr. Liza McCann, Mr. Ian Roberts, Dr. Eileen Baildam, Ms. Louise Hanna, Ms. Olivia Lloyd and Ms Susan Wadeson (The Royal Liverpool Children’s Hospital, Alder Hey, Liverpool); Dr. Phil Riley and Ms. Ann McGovern (Royal Manchester Children’s Hospital, Manchester); Dr. Clive Ryder, Mrs. Janis Scott, Mrs. Beverley Thomas, Professor Taunton Southwood, Dr. Eslam Al-Abadi and Lisa Charles (Birmingham Children’s Hospital, Birmingham); Dr. Sue Wyatt, Mrs. Gillian Jackson, Dr. Tania Amin, Dr. Mark Wood, Dr. Vanessa Van Rooyen, and Ms. Deborah Burton (Leeds General Infirmary, Leeds); Dr. Joyce Davidson, Dr. Janet Gardner-Medwin, Dr. Neil Martin, Ms. Sue Ferguson, Ms. Liz Waxman, and Mr. Michael Browne (The Royal Hospital for Sick Children, Yorkhill, Glasgow); Dr. Mark Friswell, Professor Helen Foster, Mrs. Alison Swift, Dr. Sharmila Jandial, Ms. Vicky Stevenson, Ms. Debbie Wade, Dr. Ethan Sen, Dr. Eve Smith, Ms. Lisa Qiao, Mr. Stuart Watson and Claire Duong (Great North Children’s Hospital, Newcastle); Dr. Helen Venning, Dr. Rangaraj Satyapal, Mrs. Elizabeth Stretton, Ms. Mary Jordan, Dr. Ellen Mosley, Ms. Anna Frost, Ms. Lindsay Crate, Dr. Kishore Warrier and Ms Stefanie Stafford (Queens Medical Centre, Nottingham); Professor Lucy Wedderburn, Dr. Clarissa Pilkington, Dr. N. Hasson, Mrs. Sue Maillard, Ms. Elizabeth Halkon, Ms. Virginia Brown, Ms. Audrey Juggins, Dr. Sally Smith, Mrs. Sian Lunt, Ms. Elli Enayat, Mrs. Hemlata Varsani, Miss Laura Kassoumeri, Miss Laura Beard, Miss Katie Arnold, Mrs. Yvonne Glackin, Miss Stephanie Simou, Dr. Beverley Almeida, Dr Raquel Marques, Dr Claire Deakin and Ms Stefanie Dowle (Great Ormond Street Hospital, London); Dr. Kevin Murray (Princess Margaret Hospital, Perth, Western Australia); Dr. John Ioannou and Ms. Linda Suffield (University College London Hospital); Dr. Muthana Al-Obaidi, Ms. Helen Lee, Ms. Sam Leach, Ms. Helen Smith, Dr Anne-Marie McMahon, Ms Heather Chisem and Ruth Kingshott (Sheffield’s Children’s Hospital); Dr. Nick Wilkinson, Ms. Emma Inness, Ms. Eunice Kendall, Mr. David Mayers, Ruth Etherton and Dr Kathryn Bailey (Oxford University Hospitals); Dr Jacqui Clinch, Ms Natalie Fineman and Ms Helen Pluess-Hall (Bristol Royal Hospital for Children, Bristol); Ms Lindsay Vallance (Royal Aberdeen Children’s Hospital); Ms Louise Akeroyd (Bradford Teaching Hospitals).</p><p id="__p18"><span class="underline">US Childhood Myositis Heterogeneity Study Group</span></p><p id="__p19">The following members of the US Childhood Myositis Heterogeneity Study Group contributed to this study: Drs. Barbara S. Adams (University of Michigan, Ann Arbor, MI), Catherine A. Bingham (Hershey Medical Center, Hershey, PA), Gail D. Cawkwell (All Children’s Hospital, St. Petersburg, FL), Terri H. Finkel (Children’s Hospital of Philadelphia, Philadelphia, PA), Steven W. George (Ellicott City, MD), Harry L. Gewanter (Richmond, VA), Ellen A. Goldmuntz (Children’s National Medical Center, Washington, DC), Donald P. Goldsmith (St. Christopher’s Hospital for Children, Philadelphia, PA), Michael Henrickson (Children’s Hospital, Madera, CA), Lisa Imundo (Columbia University, New York, NY), Ildy M. Katona (Uniformed Services University, Bethesda, MD), Carol B. Lindsley (University of Kansas, Kansas City), Chester P. Oddis (University of Pittsburgh, Pittsburgh, PA), Judyann C. Olson (Medical College of Wisconsin, Milwaukee), David Sherry (Children’s Hospital of Philadelphia, Philadelphia, PA), Scott A. Vogelgesang (Walter Reed Army Medical Center, Washington, DC), Carol A. Wallace (Children’s Medical Center, Seattle, WA), Patience H. White (George Washington University, Washington, DC), and Lawrence S. Zemel (Connecticut Children’s Hospital, Hartford).</p></div></div><div id="idm140647812825728" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140647812825728title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p20" class="p p-first"><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Materials</a></p><p id="__p21" class="p p-last"><a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental materials</a> include extended methods, five figures and 15 tables.</p><p></p></div></div><div id="idm140647812822896" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140647812822896title">Reference List</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Rider LG, Miller FW. Deciphering the clinical presentations, pathogenesis, and treatment of the idiopathic inflammatory myopathies. <span><span class="ref-journal">JAMA. </span>2011;<span class="ref-vol">305</span>:183–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4047218/">PMC free article</a>]</span> [<a href="/pubmed/21224460" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Deciphering+the+clinical+presentations,+pathogenesis,+and+treatment+of+the+idiopathic+inflammatory+myopathies&amp;author=LG+Rider&amp;author=FW+Miller&amp;volume=305&amp;publication_year=2011&amp;pages=183-90&amp;pmid=21224460&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Love LA, Leff RL, Fraser DD, Targoff IN, Dalakas M, Plotz PH, et al.  A new approach to the classification of idiopathic inflammatory myopathy: myositis-specific autoantibodies define useful homogeneous patient groups. <span><span class="ref-journal">Medicine (Baltimore) </span>1991;<span class="ref-vol">70</span>:360–74.</span> [<a href="/pubmed/1659647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Medicine+(Baltimore)&amp;title=A+new+approach+to+the+classification+of+idiopathic+inflammatory+myopathy:+myositis-specific+autoantibodies+define+useful+homogeneous+patient+groups&amp;author=LA+Love&amp;author=RL+Leff&amp;author=DD+Fraser&amp;author=IN+Targoff&amp;author=M+Dalakas&amp;volume=70&amp;publication_year=1991&amp;pages=360-74&amp;pmid=1659647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Chinoy H, Platt H, Lamb JA, Betteridge Z, Gunawardena H, Fertig N, et al.  The protein tyrosine phosphatase N22 gene is associated with juvenile and adult idiopathic inflammatory myopathy independent of the HLA 8.1 haplotype in British Caucasian patients. <span><span class="ref-journal">Arthritis Rheum. </span>2008;<span class="ref-vol">58</span>:3247–54.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4568569/">PMC free article</a>]</span> [<a href="/pubmed/18821667" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=The+protein+tyrosine+phosphatase+N22+gene+is+associated+with+juvenile+and+adult+idiopathic+inflammatory+myopathy+independent+of+the+HLA+8.1+haplotype+in+British+Caucasian+patients&amp;author=H+Chinoy&amp;author=H+Platt&amp;author=JA+Lamb&amp;author=Z+Betteridge&amp;author=H+Gunawardena&amp;volume=58&amp;publication_year=2008&amp;pages=3247-54&amp;pmid=18821667&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Miller FW, Cooper RG, Vencovsky J, Rider LG, Danko K, Wedderburn LR, et al.  Genome-wide association study of dermatomyositis reveals genetic overlap with other autoimmune disorders. <span><span class="ref-journal">Arthritis Rheum. </span>2013;<span class="ref-vol">65</span>:3239–47.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3934004/">PMC free article</a>]</span> [<a href="/pubmed/23983088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Genome-wide+association+study+of+dermatomyositis+reveals+genetic+overlap+with+other+autoimmune+disorders&amp;author=FW+Miller&amp;author=RG+Cooper&amp;author=J+Vencovsky&amp;author=LG+Rider&amp;author=K+Danko&amp;volume=65&amp;publication_year=2013&amp;pages=3239-47&amp;pmid=23983088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Jani M, Massey J, Wedderburn LR, Vencovsky J, Danko K, Lundberg IE, et al.  Genotyping of immune-related genetic variants identifies TYK2 as a novel associated locus for idiopathic inflammatory myopathies. <span><span class="ref-journal">Ann Rheum Dis. </span>2014;<span class="ref-vol">73</span>:1750–2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4471138/">PMC free article</a>]</span> [<a href="/pubmed/24812289" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Rheum+Dis&amp;title=Genotyping+of+immune-related+genetic+variants+identifies+TYK2+as+a+novel+associated+locus+for+idiopathic+inflammatory+myopathies&amp;author=M+Jani&amp;author=J+Massey&amp;author=LR+Wedderburn&amp;author=J+Vencovsky&amp;author=K+Danko&amp;volume=73&amp;publication_year=2014&amp;pages=1750-2&amp;pmid=24812289&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Trynka G, Hunt KA, Bockett NA, Romanos J, Mistry V, Szperl A, et al.  Dense genotyping identifies and localizes multiple common and rare variant association signals in celiac disease. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:1193–201.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3242065/">PMC free article</a>]</span> [<a href="/pubmed/22057235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Dense+genotyping+identifies+and+localizes+multiple+common+and+rare+variant+association+signals+in+celiac+disease&amp;author=G+Trynka&amp;author=KA+Hunt&amp;author=NA+Bockett&amp;author=J+Romanos&amp;author=V+Mistry&amp;volume=43&amp;publication_year=2011&amp;pages=1193-201&amp;pmid=22057235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Bohan A, Peter JB. Polymyositis and dermatomyositis (second of two parts) <span><span class="ref-journal">N Engl J Med. </span>1975;<span class="ref-vol">292</span>:403–7.</span> [<a href="/pubmed/1089199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Polymyositis+and+dermatomyositis+(second+of+two+parts)&amp;author=A+Bohan&amp;author=JB+Peter&amp;volume=292&amp;publication_year=1975&amp;pages=403-7&amp;pmid=1089199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Bohan A, Peter JB. Polymyositis and dermatomyositis (first of two parts) <span><span class="ref-journal">N Engl J Med. </span>1975;<span class="ref-vol">292</span>:344–7.</span> [<a href="/pubmed/1090839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Polymyositis+and+dermatomyositis+(first+of+two+parts)&amp;author=A+Bohan&amp;author=JB+Peter&amp;volume=292&amp;publication_year=1975&amp;pages=344-7&amp;pmid=1090839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Griggs RC, Askanas V, DiMauro S, Engel A, Karpati G, Mendell JR, et al.  Inclusion body myositis and myopathies. <span><span class="ref-journal">Ann Neurol. </span>1995;<span class="ref-vol">38</span>:705–13.</span> [<a href="/pubmed/7486861" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Inclusion+body+myositis+and+myopathies&amp;author=RC+Griggs&amp;author=V+Askanas&amp;author=S+DiMauro&amp;author=A+Engel&amp;author=G+Karpati&amp;volume=38&amp;publication_year=1995&amp;pages=705-13&amp;pmid=7486861&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Rose MR. 188th ENMC International Workshop: Inclusion Body Myositis, 2–4 December 2011, Naarden, The Netherlands. <span><span class="ref-journal">Neuromuscul Disord. </span>2013;<span class="ref-vol">23</span>:1044–55.</span> [<a href="/pubmed/24268584" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuromuscul+Disord&amp;title=188th+ENMC+International+Workshop:+Inclusion+Body+Myositis,+2–4+December+2011,+Naarden,+The+Netherlands&amp;author=MR+Rose&amp;volume=23&amp;publication_year=2013&amp;pages=1044-55&amp;pmid=24268584&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Hilton-Jones D, Miller A, Parton M, Holton J, Sewry C, Hanna MG. Inclusion body myositis: MRC Centre for Neuromuscular Diseases, IBM workshop, London, 13 June 2008. <span><span class="ref-journal">Neuromuscul Disord. </span>2010;<span class="ref-vol">20</span>:142–7.</span> [<a href="/pubmed/20074951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuromuscul+Disord&amp;title=Inclusion+body+myositis:+MRC+Centre+for+Neuromuscular+Diseases,+IBM+workshop,+London,+13+June+2008&amp;author=D+Hilton-Jones&amp;author=A+Miller&amp;author=M+Parton&amp;author=J+Holton&amp;author=C+Sewry&amp;volume=20&amp;publication_year=2010&amp;pages=142-7&amp;pmid=20074951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Connors GR, Christopher-Stine L, Oddis CV, Danoff SK. Interstitial lung disease associated with the idiopathic inflammatory myopathies: what progress has been made in the past 35 years? <span><span class="ref-journal">Chest. </span>2010;<span class="ref-vol">138</span>:1464–74.</span> [<a href="/pubmed/21138882" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chest&amp;title=Interstitial+lung+disease+associated+with+the+idiopathic+inflammatory+myopathies:+what+progress+has+been+made+in+the+past+35+years?&amp;author=GR+Connors&amp;author=L+Christopher-Stine&amp;author=CV+Oddis&amp;author=SK+Danoff&amp;volume=138&amp;publication_year=2010&amp;pages=1464-74&amp;pmid=21138882&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Adzhubei I, Jordan DM, Sunyaev SR. Predicting functional effect of human missense mutations using PolyPhen-2. <span><span class="ref-journal">Curr Protoc Hum Genet. </span>2013;<span class="ref-vol">Chapter 7</span>(Unit7)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4480630/">PMC free article</a>]</span> [<a href="/pubmed/23315928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Hum+Genet&amp;title=Predicting+functional+effect+of+human+missense+mutations+using+PolyPhen-2&amp;author=I+Adzhubei&amp;author=DM+Jordan&amp;author=SR+Sunyaev&amp;volume=Chapter+7&amp;issue=Unit7&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Kumar P, Henikoff S, Ng PC. Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm. <span><span class="ref-journal">Nat Protoc. </span>2009;<span class="ref-vol">4</span>:1073–81.</span> [<a href="/pubmed/19561590" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Predicting+the+effects+of+coding+non-synonymous+variants+on+protein+function+using+the+SIFT+algorithm&amp;author=P+Kumar&amp;author=S+Henikoff&amp;author=PC+Ng&amp;volume=4&amp;publication_year=2009&amp;pages=1073-81&amp;pmid=19561590&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, et al.  Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. <span>2005:1034–50.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182216/">PMC free article</a>]</span> [<a href="/pubmed/16024819" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Siepel+A+Bejerano+G+Pedersen+JS+Hinrichs+AS+Hou+M+Rosenbloom+K+Evolutionarily+conserved+elements+in+vertebrate,+insect,+worm,+and+yeast+genomes+2005+1034+50+" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Jostins L, Ripke S, Weersma RK, Duerr RH, McGovern DP, Hui KY, et al.  Host-microbe interactions have shaped the genetic architecture of inflammatory bowel disease. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:119–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3491803/">PMC free article</a>]</span> [<a href="/pubmed/23128233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Host-microbe+interactions+have+shaped+the+genetic+architecture+of+inflammatory+bowel+disease&amp;author=L+Jostins&amp;author=S+Ripke&amp;author=RK+Weersma&amp;author=RH+Duerr&amp;author=DP+McGovern&amp;volume=491&amp;publication_year=2012&amp;pages=119-24&amp;pmid=23128233&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Cooper JD, Simmonds MJ, Walker NM, Burren O, Brand OJ, Guo H, et al.  Seven newly identified loci for autoimmune thyroid disease. <span>2012:5202–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3490518/">PMC free article</a>]</span> [<a href="/pubmed/22922229" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Cooper+JD+Simmonds+MJ+Walker+NM+Burren+O+Brand+OJ+Guo+H+Seven+newly+identified+loci+for+autoimmune+thyroid+disease+2012+5202+8+" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Hinks A, Cobb J, Marion MC, Prahalad S, Sudman M, Bowes J, et al.  Dense genotyping of immune-related disease regions identifies 14 new susceptibility loci for juvenile idiopathic arthritis. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:664–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3673707/">PMC free article</a>]</span> [<a href="/pubmed/23603761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Dense+genotyping+of+immune-related+disease+regions+identifies+14+new+susceptibility+loci+for+juvenile+idiopathic+arthritis&amp;author=A+Hinks&amp;author=J+Cobb&amp;author=MC+Marion&amp;author=S+Prahalad&amp;author=M+Sudman&amp;volume=45&amp;publication_year=2013&amp;pages=664-9&amp;pmid=23603761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Okada Y, Wu D, Trynka G, Raj T, Terao C, Ikari K, et al.  Genetics of rheumatoid arthritis contributes to biology and drug discovery. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">506</span>:376–81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3944098/">PMC free article</a>]</span> [<a href="/pubmed/24390342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetics+of+rheumatoid+arthritis+contributes+to+biology+and+drug+discovery&amp;author=Y+Okada&amp;author=D+Wu&amp;author=G+Trynka&amp;author=T+Raj&amp;author=C+Terao&amp;volume=506&amp;publication_year=2014&amp;pages=376-81&amp;pmid=24390342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Gateva V, Sandling JK, Hom G, Taylor KE, Chung SA, Sun X, et al.  A large-scale replication study identifies TNIP1, PRDM1, JAZF1, UHRF1BP1 and IL10 as risk loci for systemic lupus erythematosus. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1228–33.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925843/">PMC free article</a>]</span> [<a href="/pubmed/19838195" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+large-scale+replication+study+identifies+TNIP1,+PRDM1,+JAZF1,+UHRF1BP1+and+IL10+as+risk+loci+for+systemic+lupus+erythematosus&amp;author=V+Gateva&amp;author=JK+Sandling&amp;author=G+Hom&amp;author=KE+Taylor&amp;author=SA+Chung&amp;volume=41&amp;publication_year=2009&amp;pages=1228-33&amp;pmid=19838195&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Mayes MD, Bossini-Castillo L, Gorlova O, Martin JE, Zhou X, Chen WV, et al.  Immunochip analysis identifies multiple susceptibility loci for systemic sclerosis. <span><span class="ref-journal">Am J Hum Genet. </span>2014;<span class="ref-vol">94</span>:47–61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3882906/">PMC free article</a>]</span> [<a href="/pubmed/24387989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Immunochip+analysis+identifies+multiple+susceptibility+loci+for+systemic+sclerosis&amp;author=MD+Mayes&amp;author=L+Bossini-Castillo&amp;author=O+Gorlova&amp;author=JE+Martin&amp;author=X+Zhou&amp;volume=94&amp;publication_year=2014&amp;pages=47-61&amp;pmid=24387989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Barrett JC, Clayton DG, Concannon P, Akolkar B, Cooper JD, Erlich HA, et al.  Genome-wide association study and meta-analysis find that over 40 loci affect risk of type 1 diabetes. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:703–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2889014/">PMC free article</a>]</span> [<a href="/pubmed/19430480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+and+meta-analysis+find+that+over+40+loci+affect+risk+of+type+1+diabetes&amp;author=JC+Barrett&amp;author=DG+Clayton&amp;author=P+Concannon&amp;author=B+Akolkar&amp;author=JD+Cooper&amp;volume=41&amp;publication_year=2009&amp;pages=703-7&amp;pmid=19430480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Jin Y, Birlea SA, Fain PR, Ferrara TM, Ben S, Riccardi SL, et al.  Genome-wide association analyses identify 13 new susceptibility loci for generalized vitiligo. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:676–80.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3366044/">PMC free article</a>]</span> [<a href="/pubmed/22561518" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analyses+identify+13+new+susceptibility+loci+for+generalized+vitiligo&amp;author=Y+Jin&amp;author=SA+Birlea&amp;author=PR+Fain&amp;author=TM+Ferrara&amp;author=S+Ben&amp;volume=44&amp;publication_year=2012&amp;pages=676-80&amp;pmid=22561518&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Tsoi LC, Spain SL, Knight J, Ellinghaus E, Stuart PE, Capon F, et al.  Identification of 15 new psoriasis susceptibility loci highlights the role of innate immunity. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:1341–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3510312/">PMC free article</a>]</span> [<a href="/pubmed/23143594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Identification+of+15+new+psoriasis+susceptibility+loci+highlights+the+role+of+innate+immunity&amp;author=LC+Tsoi&amp;author=SL+Spain&amp;author=J+Knight&amp;author=E+Ellinghaus&amp;author=PE+Stuart&amp;volume=44&amp;publication_year=2012&amp;pages=1341-8&amp;pmid=23143594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Liu JZ, Almarri MA, Gaffney DJ, Mells GF, Jostins L, Cordell HJ, et al.  Dense fine-mapping study identifies new susceptibility loci for primary biliary cirrhosis. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:1137–41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3459817/">PMC free article</a>]</span> [<a href="/pubmed/22961000" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Dense+fine-mapping+study+identifies+new+susceptibility+loci+for+primary+biliary+cirrhosis&amp;author=JZ+Liu&amp;author=MA+Almarri&amp;author=DJ+Gaffney&amp;author=GF+Mells&amp;author=L+Jostins&amp;volume=44&amp;publication_year=2012&amp;pages=1137-41&amp;pmid=22961000&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Liu JZ, Hov JR, Folseraas T, Ellinghaus E, Rushbrook SM, Doncheva NT, et al.  Dense genotyping of immune-related disease regions identifies nine new risk loci for primary sclerosing cholangitis. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:670–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3667736/">PMC free article</a>]</span> [<a href="/pubmed/23603763" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Dense+genotyping+of+immune-related+disease+regions+identifies+nine+new+risk+loci+for+primary+sclerosing+cholangitis&amp;author=JZ+Liu&amp;author=JR+Hov&amp;author=T+Folseraas&amp;author=E+Ellinghaus&amp;author=SM+Rushbrook&amp;volume=45&amp;publication_year=2013&amp;pages=670-5&amp;pmid=23603763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Lessard CJ, Li H, Adrianto I, Ice JA, Rasmussen A, Grundahl KM, et al.  Variants at multiple loci implicated in both innate and adaptive immune responses are associated with Sjogren’s syndrome. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:1284–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3867192/">PMC free article</a>]</span> [<a href="/pubmed/24097067" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variants+at+multiple+loci+implicated+in+both+innate+and+adaptive+immune+responses+are+associated+with+Sjogren’s+syndrome&amp;author=CJ+Lessard&amp;author=H+Li&amp;author=I+Adrianto&amp;author=JA+Ice&amp;author=A+Rasmussen&amp;volume=45&amp;publication_year=2013&amp;pages=1284-92&amp;pmid=24097067&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Beecham AH, Patsopoulos NA, Xifara DK, Davis MF, Kemppinen A, Cotsapas C, et al.  Analysis of immune-related loci identifies 48 new susceptibility variants for multiple sclerosis. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:1353–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3832895/">PMC free article</a>]</span> [<a href="/pubmed/24076602" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Analysis+of+immune-related+loci+identifies+48+new+susceptibility+variants+for+multiple+sclerosis&amp;author=AH+Beecham&amp;author=NA+Patsopoulos&amp;author=DK+Xifara&amp;author=MF+Davis&amp;author=A+Kemppinen&amp;volume=45&amp;publication_year=2013&amp;pages=1353-60&amp;pmid=24076602&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Eyre S, Bowes J, Diogo D, Lee A, Barton A, Martin P, et al.  High-density genetic mapping identifies new susceptibility loci for rheumatoid arthritis. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:1336–40.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3605761/">PMC free article</a>]</span> [<a href="/pubmed/23143596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=High-density+genetic+mapping+identifies+new+susceptibility+loci+for+rheumatoid+arthritis&amp;author=S+Eyre&amp;author=J+Bowes&amp;author=D+Diogo&amp;author=A+Lee&amp;author=A+Barton&amp;volume=44&amp;publication_year=2012&amp;pages=1336-40&amp;pmid=23143596&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Onengut-Gumuscu S, Chen WM, Burren O, Cooper NJ, Quinlan AR, Mychaleckyj JC, et al.  Fine mapping of type 1 diabetes susceptibility loci and evidence for colocalization of causal variants with lymphoid gene enhancers. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:381–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4380767/">PMC free article</a>]</span> [<a href="/pubmed/25751624" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Fine+mapping+of+type+1+diabetes+susceptibility+loci+and+evidence+for+colocalization+of+causal+variants+with+lymphoid+gene+enhancers&amp;author=S+Onengut-Gumuscu&amp;author=WM+Chen&amp;author=O+Burren&amp;author=NJ+Cooper&amp;author=AR+Quinlan&amp;volume=47&amp;publication_year=2015&amp;pages=381-6&amp;pmid=25751624&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Li MX, Yeung JM, Cherny SS, Sham PC. Evaluating the effective numbers of independent tests and significant p-value thresholds in commercial genotyping arrays and public imputation reference datasets. <span><span class="ref-journal">Hum Genet. </span>2012;<span class="ref-vol">131</span>:747–56.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3325408/">PMC free article</a>]</span> [<a href="/pubmed/22143225" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Evaluating+the+effective+numbers+of+independent+tests+and+significant+p-value+thresholds+in+commercial+genotyping+arrays+and+public+imputation+reference+datasets&amp;author=MX+Li&amp;author=JM+Yeung&amp;author=SS+Cherny&amp;author=PC+Sham&amp;volume=131&amp;publication_year=2012&amp;pages=747-56&amp;pmid=22143225&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Zeller T, Wild P, Szymczak S, Rotival M, Schillert A, Castagne R, et al.  Genetics and beyond--the transcriptome of human monocytes and disease susceptibility. <span><span class="ref-journal">PLoS One. </span>2010;<span class="ref-vol">5</span>:e10693.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2872668/">PMC free article</a>]</span> [<a href="/pubmed/20502693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Genetics+and+beyond--the+transcriptome+of+human+monocytes+and+disease+susceptibility&amp;author=T+Zeller&amp;author=P+Wild&amp;author=S+Szymczak&amp;author=M+Rotival&amp;author=A+Schillert&amp;volume=5&amp;publication_year=2010&amp;pages=e10693&amp;pmid=20502693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Stranger BE, Nica AC, Forrest MS, Dimas A, Bird CP, Beazley C, et al.  Population genomics of human gene expression. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1217–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683249/">PMC free article</a>]</span> [<a href="/pubmed/17873874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Population+genomics+of+human+gene+expression&amp;author=BE+Stranger&amp;author=AC+Nica&amp;author=MS+Forrest&amp;author=A+Dimas&amp;author=CP+Bird&amp;volume=39&amp;publication_year=2007&amp;pages=1217-24&amp;pmid=17873874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Veyrieras JB, Kudaravalli S, Kim SY, Dermitzakis ET, Gilad Y, Stephens M, et al.  High-resolution mapping of expression-QTLs yields insight into human gene regulation. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000214.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2556086/">PMC free article</a>]</span> [<a href="/pubmed/18846210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=High-resolution+mapping+of+expression-QTLs+yields+insight+into+human+gene+regulation&amp;author=JB+Veyrieras&amp;author=S+Kudaravalli&amp;author=SY+Kim&amp;author=ET+Dermitzakis&amp;author=Y+Gilad&amp;volume=4&amp;publication_year=2008&amp;pages=e1000214&amp;pmid=18846210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Montgomery SB, Sammeth M, Gutierrez-Arcelus M, Lach RP, Ingle C, Nisbett J, et al.  Transcriptome genetics using second generation sequencing in a Caucasian population. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">464</span>:773–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3836232/">PMC free article</a>]</span> [<a href="/pubmed/20220756" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Transcriptome+genetics+using+second+generation+sequencing+in+a+Caucasian+population&amp;author=SB+Montgomery&amp;author=M+Sammeth&amp;author=M+Gutierrez-Arcelus&amp;author=RP+Lach&amp;author=C+Ingle&amp;volume=464&amp;publication_year=2010&amp;pages=773-7&amp;pmid=20220756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Dimas AS, Deutsch S, Stranger BE, Montgomery SB, Borel C, ttar-Cohen H, et al.  Common regulatory variation impacts gene expression in a cell type-dependent manner. <span><span class="ref-journal">Science. </span>2009;<span class="ref-vol">325</span>:1246–50.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2867218/">PMC free article</a>]</span> [<a href="/pubmed/19644074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Common+regulatory+variation+impacts+gene+expression+in+a+cell+type-dependent+manner&amp;author=AS+Dimas&amp;author=S+Deutsch&amp;author=BE+Stranger&amp;author=SB+Montgomery&amp;author=C+Borel&amp;volume=325&amp;publication_year=2009&amp;pages=1246-50&amp;pmid=19644074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Boyle AP, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, et al.  Annotation of functional variation in personal genomes using RegulomeDB. <span>2012:1790–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3431494/">PMC free article</a>]</span> [<a href="/pubmed/22955989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Boyle+AP+Hong+EL+Hariharan+M+Cheng+Y+Schaub+MA+Kasowski+M+Annotation+of+functional+variation+in+personal+genomes+using+RegulomeDB+2012+1790+7+" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Yang TP, Beazley C, Montgomery SB, Dimas AS, Gutierrez-Arcelus M, Stranger BE, et al.  Genevar: a database and Java application for the analysis and visualization of SNP-gene associations in eQTL studies. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2474–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2944204/">PMC free article</a>]</span> [<a href="/pubmed/20702402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Genevar:+a+database+and+Java+application+for+the+analysis+and+visualization+of+SNP-gene+associations+in+eQTL+studies&amp;author=TP+Yang&amp;author=C+Beazley&amp;author=SB+Montgomery&amp;author=AS+Dimas&amp;author=M+Gutierrez-Arcelus&amp;volume=26&amp;publication_year=2010&amp;pages=2474-6&amp;pmid=20702402&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Vang T, Congia M, Macis MD, Musumeci L, Orru V, Zavattari P, et al.  Autoimmune-associated lymphoid tyrosine phosphatase is a gain-of-function variant. <span><span class="ref-journal">Nat Genet. </span>2005;<span class="ref-vol">37</span>:1317–9.</span> [<a href="/pubmed/16273109" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Autoimmune-associated+lymphoid+tyrosine+phosphatase+is+a+gain-of-function+variant&amp;author=T+Vang&amp;author=M+Congia&amp;author=MD+Macis&amp;author=L+Musumeci&amp;author=V+Orru&amp;volume=37&amp;publication_year=2005&amp;pages=1317-9&amp;pmid=16273109&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Zhang J, Zahir N, Jiang Q, Miliotis H, Heyraud S, Meng X, et al.  The autoimmune disease-associated PTPN22 variant promotes calpain-mediated Lyp/Pep degradation associated with lymphocyte and dendritic cell hyperresponsiveness. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:902–7.</span> [<a href="/pubmed/21841778" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+autoimmune+disease-associated+PTPN22+variant+promotes+calpain-mediated+Lyp/Pep+degradation+associated+with+lymphocyte+and+dendritic+cell+hyperresponsiveness&amp;author=J+Zhang&amp;author=N+Zahir&amp;author=Q+Jiang&amp;author=H+Miliotis&amp;author=S+Heyraud&amp;volume=43&amp;publication_year=2011&amp;pages=902-7&amp;pmid=21841778&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Sugiura T, Kawaguchi Y, Goto K, Hayashi Y, Tsuburaya R, Furuya T, et al.  Positive association between STAT4 polymorphisms and polymyositis/dermatomyositis in a Japanese population. <span><span class="ref-journal">Ann Rheum Dis. </span>2012;<span class="ref-vol">71</span>:1646–50.</span> [<a href="/pubmed/22402141" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Rheum+Dis&amp;title=Positive+association+between+STAT4+polymorphisms+and+polymyositis/dermatomyositis+in+a+Japanese+population&amp;author=T+Sugiura&amp;author=Y+Kawaguchi&amp;author=K+Goto&amp;author=Y+Hayashi&amp;author=R+Tsuburaya&amp;volume=71&amp;publication_year=2012&amp;pages=1646-50&amp;pmid=22402141&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Armstrong DL, Zidovetzki R, arcon-Riquelme ME, Tsao BP, Criswell LA, Kimberly RP, et al.  GWAS identifies novel SLE susceptibility genes and explains the association of the HLA region. <span><span class="ref-journal">Genes Immun. </span>2014;<span class="ref-vol">15</span>:347–54.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4156543/">PMC free article</a>]</span> [<a href="/pubmed/24871463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=GWAS+identifies+novel+SLE+susceptibility+genes+and+explains+the+association+of+the+HLA+region&amp;author=DL+Armstrong&amp;author=R+Zidovetzki&amp;author=ME+arcon-Riquelme&amp;author=BP+Tsao&amp;author=LA+Criswell&amp;volume=15&amp;publication_year=2014&amp;pages=347-54&amp;pmid=24871463&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Lewis MJ, Vyse S, Shields AM, Boeltz S, Gordon PA, Spector TD, et al.  UBE2L3 polymorphism amplifies NF-kappaB activation and promotes plasma cell development, linking linear ubiquitination to multiple autoimmune diseases. <span><span class="ref-journal">Am J Hum Genet. </span>2015;<span class="ref-vol">96</span>:221–34.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4320258/">PMC free article</a>]</span> [<a href="/pubmed/25640675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=UBE2L3+polymorphism+amplifies+NF-kappaB+activation+and+promotes+plasma+cell+development,+linking+linear+ubiquitination+to+multiple+autoimmune+diseases&amp;author=MJ+Lewis&amp;author=S+Vyse&amp;author=AM+Shields&amp;author=S+Boeltz&amp;author=PA+Gordon&amp;volume=96&amp;publication_year=2015&amp;pages=221-34&amp;pmid=25640675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Sugiura T, Kawaguchi Y, Goto K, Hayashi Y, Gono T, Furuya T, et al.  Association between a C8orf13-BLK polymorphism and polymyositis/dermatomyositis in the Japanese population: an additive effect with STAT4 on disease susceptibility. <span><span class="ref-journal">PLoS One. </span>2014;<span class="ref-vol">9</span>:e90019.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3954551/">PMC free article</a>]</span> [<a href="/pubmed/24632671" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Association+between+a+C8orf13-BLK+polymorphism+and+polymyositis/dermatomyositis+in+the+Japanese+population:+an+additive+effect+with+STAT4+on+disease+susceptibility&amp;author=T+Sugiura&amp;author=Y+Kawaguchi&amp;author=K+Goto&amp;author=Y+Hayashi&amp;author=T+Gono&amp;volume=9&amp;publication_year=2014&amp;pages=e90019&amp;pmid=24632671&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Faraco J, Lin L, Kornum BR, Kenny EE, Trynka G, Einen M, et al.  ImmunoChip study implicates antigen presentation to T cells in narcolepsy. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>:e1003270.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3573113/">PMC free article</a>]</span> [<a href="/pubmed/23459209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=ImmunoChip+study+implicates+antigen+presentation+to+T+cells+in+narcolepsy&amp;author=J+Faraco&amp;author=L+Lin&amp;author=BR+Kornum&amp;author=EE+Kenny&amp;author=G+Trynka&amp;volume=9&amp;publication_year=2013&amp;pages=e1003270&amp;pmid=23459209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Jia X, Han B, Onengut-Gumuscu S, Chen WM, Concannon PJ, Rich SS, et al.  Imputing amino acid polymorphisms in human leukocyte antigens. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>:e64683.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3675122/">PMC free article</a>]</span> [<a href="/pubmed/23762245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Imputing+amino+acid+polymorphisms+in+human+leukocyte+antigens&amp;author=X+Jia&amp;author=B+Han&amp;author=S+Onengut-Gumuscu&amp;author=WM+Chen&amp;author=PJ+Concannon&amp;volume=8&amp;publication_year=2013&amp;pages=e64683&amp;pmid=23762245&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Menconi F, Osman R, Monti MC, Greenberg DA, Concepcion ES, Tomer Y. Shared molecular amino acid signature in the HLA-DR peptide binding pocket predisposes to both autoimmune diabetes and thyroiditis. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2010;<span class="ref-vol">107</span>:16899–903.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947869/">PMC free article</a>]</span> [<a href="/pubmed/20837527" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Shared+molecular+amino+acid+signature+in+the+HLA-DR+peptide+binding+pocket+predisposes+to+both+autoimmune+diabetes+and+thyroiditis&amp;author=F+Menconi&amp;author=R+Osman&amp;author=MC+Monti&amp;author=DA+Greenberg&amp;author=ES+Concepcion&amp;volume=107&amp;publication_year=2010&amp;pages=16899-903&amp;pmid=20837527&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5300750&amp;issue-id=287208&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140647812922144title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Methods</a></li><li><a href="#S8title">Results</a></li><li><a href="#S15title">Discussion</a></li><li><a href="#S16title">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#idm140647812825728title">Footnotes</a></li><li><a href="#idm140647812822896title">Reference List</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5300750/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5300750/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5300750/pdf/nihms847779.pdf">PDF (686K)</a></li> | <li><a href="#" data-citationid="PMC5300750" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5300750%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5300750%2F&amp;text=Dense%20Genotyping%20of%20Immune-Related%20Loci%20in%20the%20Idiopathic%20Inflammatory%20Myopathies%20Confirms%20HLA%20alleles%20as%20Strongest%20Genetic%20Risk%20Factor%20and%20Suggests%20Different%20Genetic%20Background%20for%20Major%20Clinical%20Subgroups" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5300750%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26362759" data-aiid="5300750" data-aid="5300750" data-iid="287208" data-domainid="319" data-domain="nihpa" data-accid="PMC5300750" data-md5="7d572b09518f3d2cafa55f469ab5a5e9"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:02:31-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89FBF7DD588541000000000382021E&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5300750%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5300750/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>