<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5891499/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of Hirschsprung disease detects a novel low-frequency variant at the RET locus</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5891499/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="Genome-wide association study of Hirschsprung disease detects a novel low-frequency variant at the RET locus"><meta name="citation_authors" content="João Fadista, Marie Lund, Line Skotte, Frank Geller, Priyanka Nandakumar, Sumantra Chatterjee, Hans Matsson, Anna Löf Granström, Tomas Wester, Perttu Salo, Valtter Virtanen, Lisbeth Carstensen, Jonas Bybjerg-Grauholm, David Michael Hougaard, Mikko Pakarinen, Markus Perola, Agneta Nordenskjöld, Aravinda Chakravarti, Mads Melbye, Bjarke Feenstra"><meta name="citation_date" content="April 2018"><meta name="citation_issue" content="4"><meta name="citation_volume" content="26"><meta name="citation_firstpage" content="561"><meta name="citation_doi" content="10.1038/s41431-017-0053-7"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5891499/?report=abstract"><meta name="citation_pmid" content="29379196"><meta name="DC.Title" content="Genome-wide association study of Hirschsprung disease detects a novel low-frequency variant at the RET locus"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="João Fadista"><meta name="DC.Contributor" content="Marie Lund"><meta name="DC.Contributor" content="Line Skotte"><meta name="DC.Contributor" content="Frank Geller"><meta name="DC.Contributor" content="Priyanka Nandakumar"><meta name="DC.Contributor" content="Sumantra Chatterjee"><meta name="DC.Contributor" content="Hans Matsson"><meta name="DC.Contributor" content="Anna Löf Granström"><meta name="DC.Contributor" content="Tomas Wester"><meta name="DC.Contributor" content="Perttu Salo"><meta name="DC.Contributor" content="Valtter Virtanen"><meta name="DC.Contributor" content="Lisbeth Carstensen"><meta name="DC.Contributor" content="Jonas Bybjerg-Grauholm"><meta name="DC.Contributor" content="David Michael Hougaard"><meta name="DC.Contributor" content="Mikko Pakarinen"><meta name="DC.Contributor" content="Markus Perola"><meta name="DC.Contributor" content="Agneta Nordenskjöld"><meta name="DC.Contributor" content="Aravinda Chakravarti"><meta name="DC.Contributor" content="Mads Melbye"><meta name="DC.Contributor" content="Bjarke Feenstra"><meta name="DC.Date" content="2018 Apr"><meta name="DC.Identifier" content="10.1038/s41431-017-0053-7"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of Hirschsprung disease detects a novel low-frequency variant at the RET locus"><meta property="og:type" content="article"><meta property="og:description" content="Hirschsprung disease (HSCR) is a congenital disorder with a population incidence of ~1/5000 live births, defined by an absence of enteric ganglia along variable lengths of the colon. HSCR genome-wide association studies (GWAS) have found common associated ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5891499/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5891499/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5891499/pdf/41431_2017_Article_53.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B4CB7DD5AE6C10000000004C302C4.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5891499%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/310090/">v.26(4); 2018 Apr</a></li><li class="accid">PMC5891499</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2018 Apr; 26(4): 561–569. </span></div><div><span class="fm-vol-iss-date">Published online 2018 Jan 29. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fs41431-017-0053-7" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/s41431-017-0053-7</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5891499</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29379196">29379196</a></div></div></div></div><h1 class="content-title">Genome-wide association study of Hirschsprung disease detects a novel low-frequency variant at the <em>RET</em> locus</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Fadista%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452027267968" co-class="co-affbox">João Fadista</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1</sup> <a href="/pubmed/?term=Lund%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452017543104" co-class="co-affbox">Marie Lund</a>,<sup>1</sup> <a href="/pubmed/?term=Skotte%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452048169040" co-class="co-affbox">Line Skotte</a>,<sup>1</sup> <a href="/pubmed/?term=Geller%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452048167056" co-class="co-affbox">Frank Geller</a>,<sup>1</sup> <a href="/pubmed/?term=Nandakumar%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452045435040" co-class="co-affbox">Priyanka Nandakumar</a>,<sup>2</sup> <a href="/pubmed/?term=Chatterjee%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452015003280" co-class="co-affbox">Sumantra Chatterjee</a>,<sup>2</sup> <a href="/pubmed/?term=Matsson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452036397440" co-class="co-affbox">Hans Matsson</a>,<sup>3</sup> <a href="/pubmed/?term=Granstr%26%23x000f6%3Bm%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140451944983136" co-class="co-affbox">Anna Löf Granström</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Wester%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452048367104" co-class="co-affbox">Tomas Wester</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Salo%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452118011696" co-class="co-affbox">Perttu Salo</a>,<sup>5</sup> <a href="/pubmed/?term=Virtanen%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452080114960" co-class="co-affbox">Valtter Virtanen</a>,<sup>6</sup> <a href="/pubmed/?term=Carstensen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452014444112" co-class="co-affbox">Lisbeth Carstensen</a>,<sup>1</sup> <a href="/pubmed/?term=Bybjerg-Grauholm%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140451935958320" co-class="co-affbox">Jonas Bybjerg-Grauholm</a>,<sup>7</sup> <a href="/pubmed/?term=Hougaard%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140451941253792" co-class="co-affbox">David Michael Hougaard</a>,<sup>7</sup> <a href="/pubmed/?term=Pakarinen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452046051328" co-class="co-affbox">Mikko Pakarinen</a>,<sup>6</sup> <a href="/pubmed/?term=Perola%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452046049344" co-class="co-affbox">Markus Perola</a>,<sup>5</sup> <a href="/pubmed/?term=Nordenskj%26%23x000f6%3Bld%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452011828000" co-class="co-affbox">Agneta Nordenskjöld</a>,<sup>3,</sup><sup>4,</sup><sup>8</sup> <a href="/pubmed/?term=Chakravarti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452016781696" co-class="co-affbox">Aravinda Chakravarti</a>,<sup>2</sup> <a href="/pubmed/?term=Melbye%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452016779712" co-class="co-affbox">Mads Melbye</a>,<sup>1,</sup><sup>9,</sup><sup>10</sup> and  <a href="/pubmed/?term=Feenstra%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196" class="affpopup" co-rid="_co_idm140452045836112" co-class="co-affbox">Bjarke Feenstra</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140452027267968"><h3 class="no_margin">João Fadista</h3><p><sup>1</sup>Department of Epidemiology Research, Statens Serum Institut, Copenhagen, Denmark </p><div>Find articles by <a href="/pubmed/?term=Fadista%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">João Fadista</a></div></div><div id="_co_idm140452017543104"><h3 class="no_margin">Marie Lund</h3><p><sup>1</sup>Department of Epidemiology Research, Statens Serum Institut, Copenhagen, Denmark </p><div>Find articles by <a href="/pubmed/?term=Lund%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Marie Lund</a></div></div><div id="_co_idm140452048169040"><h3 class="no_margin">Line Skotte</h3><p><sup>1</sup>Department of Epidemiology Research, Statens Serum Institut, Copenhagen, Denmark </p><div>Find articles by <a href="/pubmed/?term=Skotte%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Line Skotte</a></div></div><div id="_co_idm140452048167056"><h3 class="no_margin">Frank Geller</h3><p><sup>1</sup>Department of Epidemiology Research, Statens Serum Institut, Copenhagen, Denmark </p><div>Find articles by <a href="/pubmed/?term=Geller%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Frank Geller</a></div></div><div id="_co_idm140452045435040"><h3 class="no_margin">Priyanka Nandakumar</h3><p><sup>2</sup>McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD USA </p><div>Find articles by <a href="/pubmed/?term=Nandakumar%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Priyanka Nandakumar</a></div></div><div id="_co_idm140452015003280"><h3 class="no_margin">Sumantra Chatterjee</h3><p><sup>2</sup>McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD USA </p><div>Find articles by <a href="/pubmed/?term=Chatterjee%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Sumantra Chatterjee</a></div></div><div id="_co_idm140452036397440"><h3 class="no_margin">Hans Matsson</h3><p><sup>3</sup>Department of Women’s and Children’s Health, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Matsson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Hans Matsson</a></div></div><div id="_co_idm140451944983136"><h3 class="no_margin">Anna Löf Granström</h3><p><sup>3</sup>Department of Women’s and Children’s Health, Karolinska Institutet, Stockholm, Sweden </p><p><sup>4</sup>Paediatric Surgery, Astrid Lindgren Children Hospital, Karolinska University Hospital, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Granstr%26%23x000f6%3Bm%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Anna Löf Granström</a></div></div><div id="_co_idm140452048367104"><h3 class="no_margin">Tomas Wester</h3><p><sup>3</sup>Department of Women’s and Children’s Health, Karolinska Institutet, Stockholm, Sweden </p><p><sup>4</sup>Paediatric Surgery, Astrid Lindgren Children Hospital, Karolinska University Hospital, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Wester%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Tomas Wester</a></div></div><div id="_co_idm140452118011696"><h3 class="no_margin">Perttu Salo</h3><p><sup>5</sup>Department of Health, National Institute for Health and Welfare, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Salo%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Perttu Salo</a></div></div><div id="_co_idm140452080114960"><h3 class="no_margin">Valtter Virtanen</h3><p><sup>6</sup>Pediatric Surgery, Children’s Hospital, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Virtanen%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Valtter Virtanen</a></div></div><div id="_co_idm140452014444112"><h3 class="no_margin">Lisbeth Carstensen</h3><p><sup>1</sup>Department of Epidemiology Research, Statens Serum Institut, Copenhagen, Denmark </p><div>Find articles by <a href="/pubmed/?term=Carstensen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Lisbeth Carstensen</a></div></div><div id="_co_idm140451935958320"><h3 class="no_margin">Jonas Bybjerg-Grauholm</h3><p><sup>7</sup>Department of Congenital Disorders, Danish Centre for Neonatal Screening, Statens Serum Institut, Copenhagen, Denmark </p><div>Find articles by <a href="/pubmed/?term=Bybjerg-Grauholm%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Jonas Bybjerg-Grauholm</a></div></div><div id="_co_idm140451941253792"><h3 class="no_margin">David Michael Hougaard</h3><p><sup>7</sup>Department of Congenital Disorders, Danish Centre for Neonatal Screening, Statens Serum Institut, Copenhagen, Denmark </p><div>Find articles by <a href="/pubmed/?term=Hougaard%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">David Michael Hougaard</a></div></div><div id="_co_idm140452046051328"><h3 class="no_margin">Mikko Pakarinen</h3><p><sup>6</sup>Pediatric Surgery, Children’s Hospital, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Pakarinen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Mikko Pakarinen</a></div></div><div id="_co_idm140452046049344"><h3 class="no_margin">Markus Perola</h3><p><sup>5</sup>Department of Health, National Institute for Health and Welfare, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Perola%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Markus Perola</a></div></div><div id="_co_idm140452011828000"><h3 class="no_margin">Agneta Nordenskjöld</h3><p><sup>3</sup>Department of Women’s and Children’s Health, Karolinska Institutet, Stockholm, Sweden </p><p><sup>4</sup>Paediatric Surgery, Astrid Lindgren Children Hospital, Karolinska University Hospital, Stockholm, Sweden </p><p><sup>8</sup>Center of Molecular Medicine, Karolinska institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Nordenskj%26%23x000f6%3Bld%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Agneta Nordenskjöld</a></div></div><div id="_co_idm140452016781696"><h3 class="no_margin">Aravinda Chakravarti</h3><p><sup>2</sup>McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD USA </p><div>Find articles by <a href="/pubmed/?term=Chakravarti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Aravinda Chakravarti</a></div></div><div id="_co_idm140452016779712"><h3 class="no_margin">Mads Melbye</h3><p><sup>1</sup>Department of Epidemiology Research, Statens Serum Institut, Copenhagen, Denmark </p><p><sup>9</sup>Department of Clinical Medicine, University of Copenhagen, Copenhagen, Denmark </p><p><sup>10</sup>Department of Medicine, Stanford University School of Medicine, Stanford, CA USA </p><div>Find articles by <a href="/pubmed/?term=Melbye%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Mads Melbye</a></div></div><div id="_co_idm140452045836112"><h3 class="no_margin">Bjarke Feenstra</h3><p><sup>1</sup>Department of Epidemiology Research, Statens Serum Institut, Copenhagen, Denmark </p><div>Find articles by <a href="/pubmed/?term=Feenstra%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29379196">Bjarke Feenstra</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140452026853424_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140452026853424_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140452026853424_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140452026853424_ai" style="display:none"><div class="fm-affl" id="Aff1" lang="en"><sup>1</sup>Department of Epidemiology Research, Statens Serum Institut, Copenhagen, Denmark </div><div class="fm-affl" id="Aff2" lang="en"><sup>2</sup>McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD USA </div><div class="fm-affl" id="Aff3" lang="en"><sup>3</sup>Department of Women’s and Children’s Health, Karolinska Institutet, Stockholm, Sweden </div><div class="fm-affl" id="Aff4" lang="en"><sup>4</sup>Paediatric Surgery, Astrid Lindgren Children Hospital, Karolinska University Hospital, Stockholm, Sweden </div><div class="fm-affl" id="Aff5" lang="en"><sup>5</sup>Department of Health, National Institute for Health and Welfare, Helsinki, Finland </div><div class="fm-affl" id="Aff6" lang="en"><sup>6</sup>Pediatric Surgery, Children’s Hospital, University of Helsinki, Helsinki, Finland </div><div class="fm-affl" id="Aff7" lang="en"><sup>7</sup>Department of Congenital Disorders, Danish Centre for Neonatal Screening, Statens Serum Institut, Copenhagen, Denmark </div><div class="fm-affl" id="Aff8" lang="en"><sup>8</sup>Center of Molecular Medicine, Karolinska institutet, Stockholm, Sweden </div><div class="fm-affl" id="Aff9" lang="en"><sup>9</sup>Department of Clinical Medicine, University of Copenhagen, Copenhagen, Denmark </div><div class="fm-affl" id="Aff10" lang="en"><sup>10</sup>Department of Medicine, Stanford University School of Medicine, Stanford, CA USA </div><div><span class="fm-affl">João Fadista, </span><span class="fm-affl">Phone: +45 32688153, <span class="email-label">Email: </span><a href="mailto:dev@null" data-email="kd.iss@fspj" class="oemail">kd.iss@fspj</a></span>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140452026853424_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2017 Jun 13; Revised 2017 Oct 3; Accepted 2017 Nov 7.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140452026853424_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © European Society of Human Genetics 2018</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5891499/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S1</div><div><a href="/pmc/articles/PMC5891499/bin/41431_2017_53_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl">41431_2017_53_MOESM1_ESM.pdf</a><span style="color:gray"> (205K)</span></div><div class="small guid">GUID:&nbsp;D924654C-D928-4B37-ACD4-65B59142DBD4</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supplemental Data</div><div><a href="/pmc/articles/PMC5891499/bin/41431_2017_53_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl">41431_2017_53_MOESM2_ESM.pdf</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID:&nbsp;6AEDE756-DF47-4266-945B-EE2332EC9707</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S1 Legend</div><div><a href="/pmc/articles/PMC5891499/bin/41431_2017_53_MOESM3_ESM.docx" data-ga-action="click_feat_suppl">41431_2017_53_MOESM3_ESM.docx</a><span style="color:gray"> (14K)</span></div><div class="small guid">GUID:&nbsp;56F79C98-C723-464A-8843-05835FFBB89D</div></div></dd></dl></div><div id="Abs1" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Abs1title">Abstract</h2><!--article-meta--><div><p id="Par1" class="p p-first-last">Hirschsprung disease (HSCR) is a congenital disorder with a population incidence of ~1/5000 live births, defined by an absence of enteric ganglia along variable lengths of the colon. HSCR genome-wide association studies (GWAS) have found common associated variants at <em>RET</em>, <em>SEMA3</em>, and <em>NRG1</em>, but they still fail to explain all of its heritability. To enhance gene discovery, we performed a GWAS of 170 cases identified from the Danish nationwide pathology registry with 4717 controls, based on 6.2 million variants imputed from the haplotype reference consortium panel. We found a novel low-frequency variant (rs144432435), which, when conditioning on the lead <em>RET</em> single-nucleotide polymorphism (SNP), was of genome-wide significance in the discovery analysis. This conditional association signal was replicated in a Swedish HSCR cohort with discovery plus replication meta-analysis conditional odds ratio of 6.6 (<em>P</em> = 7.7 × 10<sup>−10</sup>; 322 cases and 4893 controls). The conditional signal was, however, not replicated in two HSCR cohorts from USA and Finland, leading to the hypothesis that rs144432435 tags a rare haplotype present in Denmark and Sweden. Using the genome-wide complex trait analysis method, we estimated the SNP heritability of HSCR to be 88%, close to estimates based on classical family studies. Moreover, by using Lasso (least absolute shrinkage and selection operator) regression we were able to construct a genetic HSCR predictor with a area under the receiver operator characteristics curve of 76% in an independent validation set. In conclusion, we combined the largest collection of sporadic Hirschsprung cases to date (586 cases) to further elucidate HSCR’s genetic architecture.</p></div></div><div id="Sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Sec1title">Introduction</h2><p id="Par2" class="p p-first-last">Hirschsprung disease (HSCR: MIM 142623), or aganglionic megacolon, is a congenital disorder defined by an absence of enteric ganglia along variable lengths of the large intestine [<a href="#CR1" rid="CR1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494341">1</a>]. It has a population incidence of ~1/5000 live births, a sex ratio of 4:1 male:female, and a high heritability (&gt;80%) with sibling recurrence risk 250-fold greater than that in the general population [<a href="#CR1" rid="CR1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494318">1</a>, <a href="#CR2" rid="CR2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494322">2</a>]. HSCR is usually classified as short-segment HSCR (S-HSCR, 80% of HSCR cases), if aganglionosis goes no further than the upper sigmoid colon; as long-segment HSCR (L-HSCR, 15–20% of HSCR cases), if aganglionosis continues up to the splenic flexure and beyond; or as total colonic aganglionosis (TCA, ∼5% of HSCR cases), if aganglionosis affects the entire large intestine [<a href="#CR1" rid="CR1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494354">1</a>]. The condition results from failure in the migration, survival, proliferation, or differentiation of enteric neural crest-derived cells in the intestinal tract [<a href="#CR3" rid="CR3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494366">3</a>]. HSCR is one of the most common causes of intestinal obstruction in neonates [<a href="#CR1" rid="CR1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494338">1</a>, <a href="#CR4" rid="CR4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494325">4</a>], with typical presentation of the disease being a newborn without bowel movement or a young child with severe chronic constipation. Treatment is surgical resection of the aganglionic colon [<a href="#CR5" rid="CR5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691781370">5</a>]. HSCR is most frequently seen as isolated and non-syndromic cases (~80%), although familial aggregation and co-occurrence with chromosomal anomalies, such as Down syndrome, or other syndromes, have been documented [<a href="#CR6" rid="CR6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494312">6</a>]. HSCR is named after the Danish pediatrician Dr. Harald Hirschsprung, who first described it in 1888 [<a href="#CR7" rid="CR7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494307">7</a>], but despite having been known for more than a century, the etiology of the disease is not fully understood. The genetic architecture of HSCR is multifactorial and heterogeneous, influenced by both common and rare variants [<a href="#CR6" rid="CR6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494369">6</a>, <a href="#CR8" rid="CR8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494331">8</a>]. Rare high-penetrance mutations have been found in genes associated with the development of the enteric nervous system, but they cumulatively explain &lt;10% of cases [<a href="#CR9" rid="CR9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494349">9</a>]. Recent HSCR genome-wide association studies (GWAS) have found significantly greater phenotypic variation in common associated variants at <em>RET</em>, <em>SEMA3</em>, and <em>NRG1</em> loci, but they still fail to explain all of its heritability [<a href="#CR9" rid="CR9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494343">9</a>–<a href="#CR13" rid="CR13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494305">13</a>]. Therefore, it is likely that additional common or low-frequency genetic variants associated with isolated HSCR are yet to be found. Our aim was to do a GWAS on all isolated HSCR cases available in the Danish National Biobank in order to unravel possible novel genetic loci involved in the disease. We performed a GWAS on 170 cases with a pathology-confirmed diagnosis of HSCR, and 4717 controls from other projects, all genotyped on the Illumina HumanOmniExpressExome array.</p></div><div id="Sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Sec2title">Materials and methods</h2><div id="Sec3" class="sec sec-first"><h3 id="Sec3title">Study subjects</h3><p id="Par3" class="p p-first-last">Cases were defined as children with a pathology-confirmed diagnosis of aganglionosis as given by occurrence of the SNOMED-code <a id="__tag_726267287" class="tag_hotlink" href="/protein/S62590" ref="/protein/S62590"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"S62590","term_id":"2130307","term_text":"pir||S62590"}}</span>S62590</a> (Mb. Hirschsprung) in the Danish National Pathology Registry [<a href="#CR14" rid="CR14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494313">14</a>]. They also had to be singletons born in Denmark with parents and grandparents born in Northwestern Europe (birthplace taken from the Danish Civil Registration System [<a href="#CR15" rid="CR15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494332">15</a>]). The cases were also required to not have any chromosomal anomalies or congenital malformations affecting multiple systems. Controls were defined as samples genotyped on the same array as the cases from previous GWAS projects at Statens Serum Institut, namely studies on febrile seizures [<a href="#CR16" rid="CR16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494314">16</a>] (<em>n</em> = 2035), hypospadias [<a href="#CR17" rid="CR17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494321">17</a>] (<em>n</em> = 1029), hydrocephalus (<em>n</em> = 766), and post-partum depression (<em>n</em> = 1113). All samples from the discovery stage were drawn from the Danish Neonatal Screening Biobank (part of the Danish National Biobank), which includes dried blood spot samples from babies born in Denmark from 1982 and onwards [<a href="#CR18" rid="CR18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494329">18</a>]. In the discovery stage, we started with 179 cases and 4943 controls, while in the replication stage we had three cohorts: 189 cases and 426 controls from USA, 152 cases and 176 controls from Sweden, and 75 cases and 301 controls from Finland, all of European ancestry. The Danish Scientific Research Ethics Committee of the Capital Region (Copenhagen) and the Danish Data Protection Agency approved this study. The Danish Scientific Ethics Committee also granted exemption from obtaining informed consent from participants as this research project was based on genotyping samples from biobank material (H-4-2013-055). For the replication cohorts, the Ethics Committees at the Karolinska Institutet and University of Helsinki approved the use of the Swedish and Finnish samples, respectively, while the institutional Committee on Human Research at Johns Hopkins University School of Medicine approved the use of samples from the USA.</p></div><div id="Sec4" class="sec"><h3 id="Sec4title">Genotyping and quality control</h3><p id="Par4" class="p p-first-last">All samples from the discovery stage were drawn from the Danish Neonatal Screening Biobank, part of the Danish National Biobank, hosted at Statens Serum Institut, Copenhagen, Denmark. The DNA from these samples was extracted and whole genome amplified from punches of dried blood spots using a previously described protocol [<a href="#CR19" rid="CR19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691781368">19</a>]. The samples were then genotyped with the Illumina HumanOmniExpressExome array. Stringent quality control criteria were applied to the samples and variants in order to sequentially remove: (1) variant probes not aligned to the genome or that had &gt;1 high-quality match (&gt;90%) to the genome, (2) variants not present in dbSNP v.146 (<a href="https://www.ncbi.nlm.nih.gov/snp/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/snp/</a>), (3) variants with conflicting reference alleles (Illumina vs. dbSNP), (4) samples with discordant sex (registry records vs. genotypes), (5) variants with missingness &gt;5%, (6) samples with missingness &gt;2%, (7) variants that failed Hardy–Weinberg test at <em>P</em> &lt; 1 × 10<sup>−6</sup>, (8) variants with <em>P</em> &lt; 0.01 in tests of differences in missingness between cases and controls, (9) and variants with batch effect (HSCR and hydrocephalus samples were genotyped on the same batch, while the other controls were genotyped previously). These quality control steps were done with PLINK v1.90b3o [<a href="#CR20" rid="CR20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494337">20</a>]. We also removed related individuals using KING [<a href="#CR21" rid="CR21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494357">21</a>] as implemented in VCFtools [<a href="#CR22" rid="CR22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494345">22</a>], up until third-degree relatives, maintaining the samples with lower missingness. European ancestry outliers were removed using the R package SNPRelate v1.2.0 [<a href="#CR23" rid="CR23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494360">23</a>]. Outliers were defined as samples that had principal component analysis 1 (PCA1) and/or PCA2 absolute values larger than six times the interquartile range of the mean PCA1 and/or PCA2 values of all our cases and controls when computed together with the known European samples from the Human Genome Diversity Project [<a href="#CR24" rid="CR24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494319">24</a>] (Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S1A</a> and Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S1B</a>). Of note, the three clusters separated by eigenvector 2 in Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S1B</a> tag the common European chromosome 8 inversion [<a href="#CR25" rid="CR25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494317">25</a>]. The PCA outliers removal procedure was done until no more samples were removed (two rounds sufficed). After the quality control steps, we ended up with 170 cases and 4717 controls genotyped for 861,837 variants with a total genotyping success rate of 99.8%.</p></div><div id="Sec5" class="sec"><h3 id="Sec5title">Sensitivity analysis</h3><p id="Par5" class="p p-first-last">Since the controls in the discovery analysis were comprised of cases of other diseases with existing data from previous GWAS projects in our group, we performed a sensitivity analysis, where GWAS scans were performed leaving out one sub-group at a time from the controls. This had very little influence on estimates and <em>P</em> values and all results presented are therefore based on the full control set.</p></div><div id="Sec6" class="sec"><h3 id="Sec6title">GWAS replication</h3><p id="Par6" class="p p-first-last">We selected 15 variants for the GWAS replication stage using the criteria described in Supplementary Note&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">1</a>. Briefly, the replication <em>P</em> value criteria reflects the small size of the HSCR case sample, avoidance of batch effects, considerations on minor allele frequency (MAF) and imputation info score, and prior functional knowledge of HSCR disease. The replication cohort from the USA was genotyped on the Illumina HumanOmni2.5M array, while the Finnish cohort was genotyped on the Illumina HumanCoreChip. The USA cohort was subsequently imputed to the Haplotype Reference Consortium panel (version r1.1) [<a href="#CR26" rid="CR26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494344">26</a>], while the Finnish cohort was imputed to the 1000 Genomes reference panel [<a href="#CR27" rid="CR27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494339">27</a>]. Since the Swedish cohort did not have genome-wide data, genotyping of the 15 selected replication variants was done using PCR-based KASP genotyping chemistry (LGC Genomics). If any of the selected 15 variants could not get a functioning KASP assay, replacements were done with the following criteria: (1) choose variants with Linkage disequilibrium (LD) <em>r</em><sup>2</sup> &gt; 0.9 in 1000 Genomes European population (EUR 1000 Genomes Phase 1 data) as calculated in HaploReg v4.1 [<a href="#CR28" rid="CR28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494364">28</a>]; (2) if more than one appeared, choose the one where its MAF in African (AFR), Native American (AMR), and Asian (ASN) populations were the closest to the original SNP; (3) if there still were ties, choose the SNP with the lowest <em>P</em> value in our GWAS. The variants rs17653445, rs2505994, and rs12534941 were substituted with rs72789806, rs2505990, and rs6959065, respectively. Individuals in the Swedish cohort with &gt;10% missing genotypes were excluded; variants with &gt;2% missing data and with deviations from Hardy–Weinberg equilibrium (<em>P</em> &lt; 0.05) were also excluded. GWAS replicated loci were defined as having <em>P</em> &lt; 1 × 10<sup>−5</sup> in the discovery cohort, nominally significant associations in all three replication cohorts (<em>P</em> &lt; 0.05) with the same direction of effects, and a combined meta-analysis <em>P</em> &lt; 5 × 10<sup>−8</sup>.</p></div><div id="Sec7" class="sec"><h3 id="Sec7title">Replication of conditional analysis</h3><p id="Par7" class="p p-first-last">One variant (rs144432435) was selected for replication genotyping, as it was genome-wide significant in the discovery cohort, when conditioned on the lead replicated <em>RET</em> SNP (rs2505994).</p></div><div id="Sec8" class="sec"><h3 id="Sec8title">Genotype imputation</h3><p id="Par8" class="p p-first-last">We performed imputation of unobserved genotypes using phased haplotypes from the Haplotype Reference Consortium panel (version r1.0) [<a href="#CR26" rid="CR26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494330">26</a>] using the Positional Burrows–Wheeler Transform algorithm [<a href="#CR29" rid="CR29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494311">29</a>] available on the imputation server at the Wellcome Trust Sanger Institute (<a href="https://imputation.sanger.ac.uk/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://imputation.sanger.ac.uk/</a>). Phasing was also done at the server prior to imputation with SHAPEIT2 v2.r790 [<a href="#CR30" rid="CR30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494306">30</a>].</p></div><div id="Sec9" class="sec"><h3 id="Sec9title">Association analysis</h3><p id="Par9" class="p p-first-last">Variants were kept for association if: (1) imputation info score ≥0.9, (2) MAF &gt;1%, and (3) PLINK INFO score &lt;1.2 (INFO values much &gt;1 can indicate strong departure from Hardy–Weinberg equilibrium). GWAS analysis was done using logistic regression on 6,162,479 imputed SNP dosages under an additive genetic model using PLINK v1.90b3o [<a href="#CR20" rid="CR20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494320">20</a>]. We carried out an inverse variance-based fixed-effects meta-analysis of the discovery and replication data, as implemented in PLINK v1.90b3o [<a href="#CR20" rid="CR20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494347">20</a>]. Estimation of the inflation factor of the discovery stage <em>P</em> values was done with the function estlambda in GenABEL R package [<a href="#CR31" rid="CR31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494309">31</a>]. Regional association plots were created with LocusZoom [<a href="#CR32" rid="CR32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494351">32</a>].</p></div><div id="Sec10" class="sec"><h3 id="Sec10title">Genetic correlation</h3><p id="Par10" class="p p-first-last">Genetic correlation analysis of our GWAS of Hirschsprung disease vs. GWAS of other diseases/traits was done with the LD score regression method [<a href="#CR33" rid="CR33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494340">33</a>, <a href="#CR34" rid="CR34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494336">34</a>] available at LD Hub (<a href="http://ldsc.broadinstitute.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ldsc.broadinstitute.org/</a>).</p></div><div id="Sec11" class="sec"><h3 id="Sec11title">Linkage disequilibrium</h3><p id="Par11" class="p p-first-last">LD between pairs of variants was calculated in PLINK v1.90b3o [<a href="#CR20" rid="CR20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494308">20</a>]. The dosage genotypes were transformed to hard calls prior to LD calculations.</p></div><div id="Sec12" class="sec"><h3 id="Sec12title">Heritability</h3><p id="Par12" class="p p-first-last">Heritability estimation was done with genome-wide complex trait analysis (GCTA) v1.24.7 [<a href="#CR35" rid="CR35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494353">35</a>] on the genotyped QC variants. We first estimated the genetic relationship from all the autosomal SNPs, and then did the GCTA-GREML analysis for a case–control study with a specified disease prevalence of 0.000109 [<a href="#CR36" rid="CR36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691781369">36</a>].</p></div><div id="Sec13" class="sec sec-last"><h3 id="Sec13title">Case–control prediction</h3><p id="Par13" class="p p-first-last">To obtain a predictive genetic model, we used Lasso [<a href="#CR37" rid="CR37" class=" bibr popnode">37</a>] (least absolute shrinkage and selection operator) regression. To avoid overfitting, the optimal Lasso penalization parameter, <em>λ</em>*, was selected by 10-fold cross-validation (details below). Then, based on all variants with a marginal association <em>P</em> &lt; 0.01 (62,652 variants), a predictive genetic model was obtained using Lasso logistic regression with penalization parameter <em>λ</em>*. Finally, the performance of this predictive model was evaluated in an independent validation cohort (USA HSCR cohort). Cross-validation procedure for <em>λ</em>*: Since the variable selection by Lasso is preceded by a <em>P</em> value-based filtering step, it is important that the cross-validation procedure used to select the penalization parameter <em>λ</em>* appropriately reflects this filtering to avoid overfitting. Briefly, the discovery GWAS cohort was first randomly partitioned into 10 equally sized subsamples. A single subsample was labeled as the test data and reserved for quantifying the predictive performance, while a new marginal association analysis was carried out (as described in section “Association analysis” above) on the remaining training data. All variants with a marginal association <em>P</em> &gt; 0.01 were filtered out and a Lasso logistic regression model was fitted to the remaining variants in the training data for a decreasing sequence of penalization parameters; <em>λ</em>1, …, <em>λ</em>200. For each value of <em>λ</em>, the predictive performance of the model was evaluated on the test data in terms of the area under the receiver operator characteristics (ROC) curve (AUC). This procedure was repeated 10 times and subsequently the mean cross-validation AUC was calculated for each value of <em>λ</em>. The optimal Lasso penalization parameter, <em>λ</em>*, was selected as the value of <em>λ</em> achieving the highest mean cross-validation AUC.</p></div></div><div id="Sec14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Sec14title">Results</h2><div id="Sec15" class="sec sec-first"><h3 id="Sec15title">Study subjects</h3><p id="Par14" class="p p-first-last">In order to identify HSCR susceptibility loci in the Danish population, we did a GWAS with 170 cases, pathology confirmed with aganglionosis, and 4717 controls, all of Danish ancestry (see section “Materials and methods”). Male:female ratio was 3.1 in our cases and 1.1 in the controls. Our data did not permit distinction between S-HSCR and L-HSCR. Of note, information on the length of the aganglionosis was also not available in the replication cohorts.</p></div><div id="Sec16" class="sec"><h3 id="Sec16title">GWAS confirms known <em>RET</em> and <em>SEMA3</em> susceptibility variants</h3><p id="Par15" class="p p-first-last">All samples from the Danish discovery cohort were genotyped on the Illumina OmniExpressExome chip, with missing genotypes imputed from the Haplotype Reference Consortium panel. Association tests based on 1000G imputed chromosome X did not reveal any variants for follow-up and therefore the paper is focused on autosomal variation. Of note, GWAS analysis adjusting for sex was also done with minimal change in the results (data not shown). We tested for differences in allele dosages between cases and controls at 6,162,479 autosomal SNPs (MAF &gt; 1%, <em>r</em><sup>2</sup> imputation info score ≥0.9; see section “Materials and methods”). Our study subjects showed a genomic inflation factor of 1.03 (Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S2</a>), suggesting a low population stratification. Moreover, the genomic inflation factor was only 1.0005 when the known <em>RET</em> and <em>SEMA3</em> signals were removed and we therefore did not correct <em>P</em> values by genomic control. We selected 15 SNPs from different loci for replication (see section “Materials and methods”; Supplementary Note&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">1</a>), of which four replicated in three independent replication cohorts from USA, Sweden, and Finland (Table&nbsp;<a href="/pmc/articles/PMC5891499/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span>1</span></a>). Of the four replicated SNPs, three were at the <em>RET</em> locus and another at the <em>SEMA3</em> locus. We selected more than one SNP for replication genotyping at <em>RET</em> locus as this is the most significant locus associated with HSCR and therefore we did not want to miss any possible new hits. When conditioning on the top replicated <em>RET</em> SNP (rs2505994 hg19.chr10:g.43568887T&gt;C), the other two replicated <em>RET</em> SNPs ceased to be significantly associated with HSCR (rs1271546 hg19.chr10:g.43505098A&gt;G <em>P</em> = 0.04; rs10899836 hg19.chr10:g.44105044G&gt;A <em>P</em> = 0.06), despite their relative low LD (<em>r</em><sup>2</sup> of rs2505994−rs1271546 = 0.18; <em>r</em><sup>2</sup> of rs2505994−rs10899836 = 0.03). The three replicated <em>RET</em> locus SNPs are in LD with rs2505998 hg19.chr10:g.43570925A&gt;G, the previously reported lead <em>RET</em> SNP [<a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267295">12</a>] (<em>r</em><sup>2</sup> of rs2505994−rs2505998 = 0.85; <em>r</em><sup>2</sup> of rs1271546−rs2505998 = 0.18; <em>r</em><sup>2</sup> of rs10899836−rs2505998 = 0.06) (Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S4</a>). Similarly, our replicated <em>SEMA3</em> variant is also in LD with rs80227144 hg19.chr7:g.84349842C&gt;A, the previously reported lead <em>SEMA3</em> SNP [<a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267289">12</a>] (<em>r</em><sup>2</sup> of rs117617821−rs80227144 = 0.97) (Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S5</a>). In line with previous studies based on Caucasian individuals [<a href="#CR9" rid="CR9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494310">9</a>, <a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267290">12</a>], the <em>NRG1</em> locus only showed suggestive significance in the discovery cohort (lowest <em>P</em> = 9.83 × 10<sup>−6</sup> for rs73250444 hg19.chr8:g.32252694A&gt;C) (Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S3a</a>) and did not replicate (Table&nbsp;<a href="/pmc/articles/PMC5891499/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span>1</span></a>). The previously reported lead <em>NRG1</em> SNP in East Asians [<a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267293">12</a>] was even less significantly associated in our data (rs7005606 hg19.chr8:g.32401501T&gt;G; <em>P</em> = 5.22 × 10<sup>−5</sup>) (Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S3b</a>). While our top SNP in <em>NRG1</em> is in relatively low LD with the reported lead SNP in our discovery cohort (rs73250444–rs7005606; <em>r</em><sup>2</sup> = 0.13), conditional analysis of both SNPs revealed similar effects (rs73250444 <em>P</em> = 0.003 adjusting for rs7005606; rs7005606 <em>P</em> = 0.007 adjusting for rs73250444). Moreover, the previously reported lead NRG1 SNP in East Asians [<a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267294">12</a>] (rs7005606) was less significantly associated in our discovery cohort (<em>P</em> = 5.22 × 10<sup>−5</sup>) and Finnish replication cohort (<em>P</em> = 0.03), while below nominal significance in our US replication cohort (<em>P</em> = 0.29). After removal of the <em>RET</em> and <em>SEMA3</em> loci the quantile–quantile plot still indicated an excess of low <em>P</em> values (Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S6</a>), suggesting that additional loci is associated with the disease.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="Tab1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p1">Association results with Hirschsprung disease in discovery (Denmark), replication (USA, Sweden, Finland), and combined analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140452048288928" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">A1</th><th rowspan="1" colspan="1">A2</th><th colspan="5" rowspan="1">Denmark (170 cases; 4717 controls)</th><th rowspan="1" colspan="5">USA (189 cases; 426 controls)</th><th rowspan="1" colspan="5">Sweden (152 cases; 176 controls)</th><th rowspan="1" colspan="5">Finland (75 cases; 301 controls)</th><th colspan="3" rowspan="1">All (586 cases; 5678 controls)</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1">Cases</th><th rowspan="1" colspan="1">Ctrls</th><th rowspan="1" colspan="1">OR</th><th rowspan="1" colspan="1">SE</th><th rowspan="1" colspan="1">
<em>P</em>
</th><th rowspan="1" colspan="1">Cases</th><th rowspan="1" colspan="1">Ctrls</th><th rowspan="1" colspan="1">OR</th><th rowspan="1" colspan="1">SE</th><th rowspan="1" colspan="1">
<em>P</em>
</th><th rowspan="1" colspan="1">Cases</th><th rowspan="1" colspan="1">Ctrls</th><th rowspan="1" colspan="1">OR</th><th rowspan="1" colspan="1">SE</th><th rowspan="1" colspan="1">
<em>P</em>
</th><th rowspan="1" colspan="1">Cases</th><th rowspan="1" colspan="1">Ctrls</th><th rowspan="1" colspan="1">OR</th><th rowspan="1" colspan="1">SE</th><th rowspan="1" colspan="1">
<em>P</em>
</th><th rowspan="1" colspan="1">
<em>P</em>
</th><th rowspan="1" colspan="1">OR</th><th rowspan="1" colspan="1">
<em>Q</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs75026391</td><td rowspan="1" colspan="1">2:39442135</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">3.06</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">7.08E − 07</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.54</td><td rowspan="1" colspan="1">0.38</td><td rowspan="1" colspan="1">2.62E − 01</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">1.05</td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">9.25E − 01</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">1.29</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">6.01E − 01</td><td rowspan="1" colspan="1">1.00E − 05</td><td rowspan="1" colspan="1">2.11</td><td rowspan="1" colspan="1">8.49E − 02</td></tr><tr><td rowspan="1" colspan="1">rs349279</td><td rowspan="1" colspan="1">4:132884220</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">1.89</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">5.64E − 08</td><td rowspan="1" colspan="1">0.31</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">1.39</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">1.65E − 02</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">0.73</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">9.27E − 02</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.78</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">3.01E − 01</td><td rowspan="1" colspan="1">1.13E − 04</td><td rowspan="1" colspan="1">1.34</td><td rowspan="1" colspan="1">0.00E + 00</td></tr><tr><td rowspan="1" colspan="1">rs6452575</td><td rowspan="1" colspan="1">5:83981246</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">2.33</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">8.74E − 08</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">1.25</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">3.43E − 01</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.1</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">0.31</td><td rowspan="1" colspan="1">2.21E − 02</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.51</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">5.83E − 02</td><td rowspan="1" colspan="1">3.87E − 03</td><td rowspan="1" colspan="1">1.39</td><td rowspan="1" colspan="1">0.00E + 00</td></tr><tr><td rowspan="1" colspan="1">rs2935063</td><td rowspan="1" colspan="1">6:170451098</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.59</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">1.69</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">4.24E − 06</td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">6.45E − 02</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">0.95</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">7.25E − 01</td><td rowspan="1" colspan="1">0.5</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">1.06</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">7.70E − 01</td><td rowspan="1" colspan="1">8.76E − 02</td><td rowspan="1" colspan="1">1.12</td><td rowspan="1" colspan="1">1.00E − 04</td></tr><tr><td rowspan="1" colspan="1">rs12534941</td><td rowspan="1" colspan="1">7:16890982</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">2.98E − 06</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">1.21</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">1.92E − 01</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.2</td><td rowspan="1" colspan="1">1.16</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">4.35E − 01</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">1.18</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">4.93E − 01</td><td rowspan="1" colspan="1">2.99E − 01</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">0.00E + 00</td></tr><tr><td rowspan="1" colspan="1">rs62472985</td><td rowspan="1" colspan="1">7:84197311</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">2.15</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">9.87E − 10</td><td rowspan="1" colspan="1">0.2</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">1.2</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">2.46E − 01</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">1.28</td><td rowspan="1" colspan="1">0.2</td><td rowspan="1" colspan="1">2.30E − 01</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">2.9</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">1.14E − 05</td><td rowspan="1" colspan="1">2.59E−11</td><td rowspan="1" colspan="1">1.74</td><td rowspan="1" colspan="1">1.90E − 03</td></tr><tr><td rowspan="1" colspan="1">
<strong>rs117617821</strong>
</td><td rowspan="1" colspan="1">
<strong>7:84322189</strong>
</td><td rowspan="1" colspan="1">
<strong>C</strong>
</td><td rowspan="1" colspan="1">
<strong>T</strong>
</td><td rowspan="1" colspan="1">
<strong>0.12</strong>
</td><td rowspan="1" colspan="1">
<strong>0.04</strong>
</td><td rowspan="1" colspan="1">
<strong>3.3</strong>
</td><td rowspan="1" colspan="1">
<strong>0.18</strong>
</td><td rowspan="1" colspan="1"><strong>1.95E </strong>− <strong>11</strong></td><td rowspan="1" colspan="1">
<strong>0.09</strong>
</td><td rowspan="1" colspan="1">
<strong>0.05</strong>
</td><td rowspan="1" colspan="1">
<strong>1.88</strong>
</td><td rowspan="1" colspan="1">
<strong>0.24</strong>
</td><td rowspan="1" colspan="1"><strong>9.03E </strong>− <strong>03</strong></td><td rowspan="1" colspan="1">
<strong>0.08</strong>
</td><td rowspan="1" colspan="1">
<strong>0.05</strong>
</td><td rowspan="1" colspan="1">
<strong>1.99</strong>
</td><td rowspan="1" colspan="1">
<strong>0.34</strong>
</td><td rowspan="1" colspan="1"><strong>4.40E </strong>− <strong>02</strong></td><td rowspan="1" colspan="1">
<strong>0.12</strong>
</td><td rowspan="1" colspan="1">
<strong>0.03</strong>
</td><td rowspan="1" colspan="1">
<strong>3.82</strong>
</td><td rowspan="1" colspan="1">
<strong>0.35</strong>
</td><td rowspan="1" colspan="1"><strong>1.44E </strong>− <strong>04</strong></td><td rowspan="1" colspan="1"><strong>7.02E </strong>− <strong>16</strong></td><td rowspan="1" colspan="1">
<strong>2.71</strong>
</td><td rowspan="1" colspan="1"><strong>1.55E </strong>− <strong>01</strong></td></tr><tr><td rowspan="1" colspan="1">rs73250444</td><td rowspan="1" colspan="1">8:32252694</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">1.76</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">9.83E − 06</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">1.97E − 01</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.2</td><td rowspan="1" colspan="1">1.15</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">4.65E − 01</td><td rowspan="1" colspan="1">0.2</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">1.83</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">1.93E − 02</td><td rowspan="1" colspan="1">5.45E − 04</td><td rowspan="1" colspan="1">1.35</td><td rowspan="1" colspan="1">1.60E − 03</td></tr><tr><td rowspan="1" colspan="1">rs17653445</td><td rowspan="1" colspan="1">10:37835654</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">1.98</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">8.24E − 08</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">1.27</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">1.49E − 01</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">1.11</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">5.82E − 01</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">2.52</td><td rowspan="1" colspan="1">0.2</td><td rowspan="1" colspan="1">5.66E − 06</td><td rowspan="1" colspan="1">4.69E − 10</td><td rowspan="1" colspan="1">1.67</td><td rowspan="1" colspan="1">4.80E − 03</td></tr><tr><td rowspan="1" colspan="1">
<strong>rs1271546</strong>
</td><td rowspan="1" colspan="1">
<strong>10:43505098</strong>
</td><td rowspan="1" colspan="1">
<strong>A</strong>
</td><td rowspan="1" colspan="1">
<strong>G</strong>
</td><td rowspan="1" colspan="1">
<strong>0.24</strong>
</td><td rowspan="1" colspan="1">
<strong>0.09</strong>
</td><td rowspan="1" colspan="1">
<strong>3.4</strong>
</td><td rowspan="1" colspan="1">
<strong>0.14</strong>
</td><td rowspan="1" colspan="1"><strong>2.76E </strong>− <strong>19</strong></td><td rowspan="1" colspan="1">
<strong>0.21</strong>
</td><td rowspan="1" colspan="1">
<strong>0.09</strong>
</td><td rowspan="1" colspan="1">
<strong>2.79</strong>
</td><td rowspan="1" colspan="1">
<strong>0.18</strong>
</td><td rowspan="1" colspan="1"><strong>1.32E </strong>− <strong>08</strong></td><td rowspan="1" colspan="1">
<strong>0.25</strong>
</td><td rowspan="1" colspan="1">
<strong>0.11</strong>
</td><td rowspan="1" colspan="1">
<strong>2.5</strong>
</td><td rowspan="1" colspan="1">
<strong>0.22</strong>
</td><td rowspan="1" colspan="1"><strong>2.11E </strong>− <strong>05</strong></td><td rowspan="1" colspan="1">
<strong>0.13</strong>
</td><td rowspan="1" colspan="1">
<strong>0.07</strong>
</td><td rowspan="1" colspan="1">
<strong>1.9</strong>
</td><td rowspan="1" colspan="1">
<strong>0.28</strong>
</td><td rowspan="1" colspan="1"><strong>2.15E </strong>− <strong>02</strong></td><td rowspan="1" colspan="1"><strong>1.57E </strong>− <strong>30</strong></td><td rowspan="1" colspan="1">
<strong>2.88</strong>
</td><td rowspan="1" colspan="1"><strong>2.23E</strong>−<strong>01</strong></td></tr><tr><td rowspan="1" colspan="1">
<strong>rs2505994</strong>
</td><td rowspan="1" colspan="1">
<strong>10:43568887</strong>
</td><td rowspan="1" colspan="1">
<strong>T</strong>
</td><td rowspan="1" colspan="1">
<strong>C</strong>
</td><td rowspan="1" colspan="1">
<strong>0.63</strong>
</td><td rowspan="1" colspan="1">
<strong>0.23</strong>
</td><td rowspan="1" colspan="1">
<strong>5.9</strong>
</td><td rowspan="1" colspan="1">
<strong>0.12</strong>
</td><td rowspan="1" colspan="1"><strong>1.31E </strong>− <strong>48</strong></td><td rowspan="1" colspan="1">
<strong>0.56</strong>
</td><td rowspan="1" colspan="1">
<strong>0.21</strong>
</td><td rowspan="1" colspan="1">
<strong>4.13</strong>
</td><td rowspan="1" colspan="1">
<strong>0.14</strong>
</td><td rowspan="1" colspan="1"><strong>1.75E </strong>− <strong>23</strong></td><td rowspan="1" colspan="1">
<strong>0.68</strong>
</td><td rowspan="1" colspan="1">
<strong>0.23</strong>
</td><td rowspan="1" colspan="1">
<strong>5.24</strong>
</td><td rowspan="1" colspan="1">
<strong>0.19</strong>
</td><td rowspan="1" colspan="1"><strong>7.20E </strong>− <strong>19</strong></td><td rowspan="1" colspan="1">
<strong>0.78</strong>
</td><td rowspan="1" colspan="1">
<strong>0.29</strong>
</td><td rowspan="1" colspan="1">
<strong>7.49</strong>
</td><td rowspan="1" colspan="1">
<strong>0.24</strong>
</td><td rowspan="1" colspan="1"><strong>5.97E </strong>− <strong>17</strong></td><td rowspan="1" colspan="1"><strong>1.79E </strong>− <strong>101</strong></td><td rowspan="1" colspan="1">
<strong>5.3</strong>
</td><td rowspan="1" colspan="1"><strong>1.17E </strong>− <strong>01</strong></td></tr><tr><td rowspan="1" colspan="1">rs4519046</td><td rowspan="1" colspan="1">10:43874817</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">2.21</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">4.87E − 11</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">1.68</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">5.14E − 04</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">1.66</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">6.42E − 03</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">1.49</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">7.43E − 02</td><td rowspan="1" colspan="1">3.38E − 15</td><td rowspan="1" colspan="1">1.85</td><td rowspan="1" colspan="1">2.79E − 01</td></tr><tr><td rowspan="1" colspan="1">
<strong>rs10899836</strong>
</td><td rowspan="1" colspan="1">
<strong>10:44105044</strong>
</td><td rowspan="1" colspan="1">
<strong>A</strong>
</td><td rowspan="1" colspan="1">
<strong>G</strong>
</td><td rowspan="1" colspan="1">
<strong>0.26</strong>
</td><td rowspan="1" colspan="1">
<strong>0.16</strong>
</td><td rowspan="1" colspan="1">
<strong>1.9</strong>
</td><td rowspan="1" colspan="1">
<strong>0.13</strong>
</td><td rowspan="1" colspan="1"><strong>6.95E </strong>− <strong>07</strong></td><td rowspan="1" colspan="1">
<strong>0.2</strong>
</td><td rowspan="1" colspan="1">
<strong>0.14</strong>
</td><td rowspan="1" colspan="1">
<strong>1.6</strong>
</td><td rowspan="1" colspan="1">
<strong>0.16</strong>
</td><td rowspan="1" colspan="1"><strong>3.88E </strong>− <strong>03</strong></td><td rowspan="1" colspan="1">
<strong>0.24</strong>
</td><td rowspan="1" colspan="1">
<strong>0.16</strong>
</td><td rowspan="1" colspan="1">
<strong>1.59</strong>
</td><td rowspan="1" colspan="1">
<strong>0.2</strong>
</td><td rowspan="1" colspan="1"><strong>1.75E </strong>− <strong>02</strong></td><td rowspan="1" colspan="1">
<strong>0.37</strong>
</td><td rowspan="1" colspan="1">
<strong>0.17</strong>
</td><td rowspan="1" colspan="1">
<strong>2.59</strong>
</td><td rowspan="1" colspan="1">
<strong>0.2</strong>
</td><td rowspan="1" colspan="1"><strong>1.45E </strong>− <strong>06</strong></td><td rowspan="1" colspan="1"><strong>3.39E </strong>− <strong>14</strong></td><td rowspan="1" colspan="1">
<strong>1.85</strong>
</td><td rowspan="1" colspan="1"><strong>2.27E </strong>− <strong>01</strong></td></tr><tr><td rowspan="1" colspan="1">rs12428625</td><td rowspan="1" colspan="1">13:86708779</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">5.9</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">2.76E − 14</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.06</td><td rowspan="1" colspan="1">0.44</td><td rowspan="1" colspan="1">8.88E − 01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">1.41</td><td rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1">5.79E − 01</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">3.97E − 11</td><td rowspan="1" colspan="1">3.63</td><td rowspan="1" colspan="1">7.00E − 04</td></tr><tr><td rowspan="1" colspan="1">rs3861656</td><td rowspan="1" colspan="1">14:90596297</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">2.19</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">2.74E − 07</td><td rowspan="1" colspan="1">0.1</td><td rowspan="1" colspan="1">0.1</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">9.02E − 01</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.1</td><td rowspan="1" colspan="1">1.2</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">4.82E − 01</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.1</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">7.64E − 01</td><td rowspan="1" colspan="1">2.08E − 04</td><td rowspan="1" colspan="1">1.48</td><td rowspan="1" colspan="1">4.10E − 03</td></tr></tbody></table></div><div id="largeobj_idm140452048288928" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5891499/table/Tab1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p2">SNPs in bold are replicated (see section “Materials and methods”)</p><p id="__p3"><em>SNP</em> dbSNP v146 variant ID, <em>Position</em> hg19 human genome assembly, <em>A1</em> effect allele, <em>A2</em> alternative allele, <em>Cases</em> A1 frequency in cases, <em>Ctrls</em> A1 frequency in controls, <em>OR</em> odds ratio of the effect allele, <em>SE</em> standard error, <em>P</em>
<em>P</em> value for the logistic regression, <em>P</em> (<em>ALL</em>) fixed-effects meta-analysis <em>P</em> value, <em>Q</em>
<em>P</em> value for Cochrane’s <em>Q</em> statistic (measure of heterogeneity)</p></div></div></div><div id="Sec17" class="sec"><h3 id="Sec17title">Conditional analysis identifies a new low-frequency HSCR susceptibility variant</h3><p id="Par16" class="p p-first-last">We sought to detect multiple independent association signals at the <em>SEMA3</em> and <em>RET</em> loci by conditioning on their respective lead SNPs. A secondary association signal was detected at the <em>RET</em> locus (top SNP being rs144432435 hg19.chr10:g.43559390C&gt;T) when we adjusted for the effect of the <em>RET</em> lead SNP rs2505994 (Fig.&nbsp;<a href="/pmc/articles/PMC5891499/figure/Fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig1" rid-ob="ob-Fig1" co-legend-rid="lgnd_Fig1"><span>1</span></a>). Of note, this secondary association signal is still of genome-wide significance if we condition on all the three replicated <em>RET</em> SNPs (rs2505994, rs1271546, and rs10899836) (Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S7</a>). In the Danish discovery cohort, rs144432435 has a frequency of 1% in the controls and 3% in the cases, with a conditional odds ratio (OR) = 6.8, <em>P</em> = 2.7 × 10<sup>−8</sup>. Before conditional analysis, the effect of rs144432435 was masked, as its risk allele (T) is out of phase with the rs250994 risk allele (T), i.e., they are not on the same haplotype, resulting in an underestimation of the effect sizes of both SNPs if considered individually (Table&nbsp;<a href="/pmc/articles/PMC5891499/table/Tab2/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab2" rid-ob="ob-Tab2" co-legend-rid=""><span>2</span></a>). The conditional analysis was replicated in the Swedish cohort, with a conditional OR = 5.9, <em>P</em> = 8.3 × 10<sup>−3</sup>. Conditional meta-analysis of the Danish and Swedish cohorts yielded a conditional OR = 6.6, <em>P</em> = 7.7 × 10<sup>−10</sup> (Table&nbsp;<a href="/pmc/articles/PMC5891499/table/Tab2/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab2" rid-ob="ob-Tab2" co-legend-rid=""><span>2</span></a>). Since this low-frequency variant was not present in the Finnish cohort and not replicated in the US cohort (<em>P</em> = 0.58), we hypothesize that rs144432435 is tagging a rare haplotype present in Denmark and Sweden but not in the USA. Further studies are needed to determine the presence of the novel HSCR association signal in populations of different origin and experimental studies are also needed to uncover the functional mechanism underlying the association [<a href="#CR38" rid="CR38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494342">38</a>]. Similarly, a previous study [<a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267292">12</a>] has also detected an Asian-specific secondary HSCR association signal of low frequency at the <em>RET</em> locus. This provides further evidence that the <em>RET</em> locus may still contain low frequency or rare variants associated with HSCR that remain to be discovered by increasing sample size and/or performing full sequencing of the <em>RET</em> locus.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="Fig1" co-legend-rid="lgnd_Fig1"><a href="/pmc/articles/PMC5891499/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140451906638608" class="figure"><a href="/pmc/articles/PMC5891499/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5891499_41431_2017_53_Fig1_HTML.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 41431_2017_53_Fig1_HTML.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5891499/bin/41431_2017_53_Fig1_HTML.jpg"></a></div><div id="largeobj_idm140451906638608" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5891499/figure/Fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_Fig1"><div><a class="figpopup" href="/pmc/articles/PMC5891499/figure/Fig1/" target="figure" rid-figpopup="Fig1" rid-ob="ob-Fig1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p4">Regional conditional association plot of the new genome-wide significant HSCR variant. <strong>a</strong> Association results at <em>RET</em> locus before the conditional analysis. <strong>b</strong> Association results when conditioning on the <em>RET</em> lead SNP (rs2505994). Color-coded linkage disequilibrium is shown for the top conditional associated SNP rs144432435—in purple). The <em>x</em>-axis represents the chromosome 10 region (hg19 assembly) surrounding the new hit, while the <em>y</em>-axis represents the strength of the association in −log<sub>10</sub> (<em>P</em> value) (see color fig online)</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="Tab2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">Rs144432435 association analysis with Hirschsprung disease, conditioned on <em>RET</em> lead SNP rs2505994 in discovery (Denmark), replication (USA, Sweden, Finland), and combined analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140452014410768" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">A1</th><th rowspan="1" colspan="1">A2</th><th rowspan="1" colspan="1">Cohort</th><th rowspan="1" colspan="1">Cases</th><th rowspan="1" colspan="1">Controls</th><th colspan="3" rowspan="1">Univariate analysis</th><th colspan="3" rowspan="1">Conditional analysis</th><th colspan="3" rowspan="1">Conditional meta-analysis</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1">OR</th><th rowspan="1" colspan="1">SE</th><th rowspan="1" colspan="1">
<em>P</em>
</th><th rowspan="1" colspan="1">OR</th><th rowspan="1" colspan="1">SE</th><th rowspan="1" colspan="1">
<em>P</em>
</th><th rowspan="1" colspan="1">
<em>P</em>
</th><th rowspan="1" colspan="1">OR</th><th rowspan="1" colspan="1">
<em>Q</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs144432435</td><td rowspan="1" colspan="1">10:43559390</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">DK (170 cases; 4717 controls)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">3.28</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">2.39E − 04</td><td rowspan="1" colspan="1">6.76</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">2.72E − 08</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">SE (152 cases; 176 controls)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">2.68</td><td rowspan="1" colspan="1">0.61</td><td rowspan="1" colspan="1">0.1076</td><td rowspan="1" colspan="1">5.88</td><td rowspan="1" colspan="1">0.67</td><td rowspan="1" colspan="1">8.27E − 03</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">US (189 cases; 426 controls)</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.019</td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">0.7206</td><td rowspan="1" colspan="1">1.33</td><td rowspan="1" colspan="1">0.52</td><td rowspan="1" colspan="1">0.5793</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">FI (75 cases; 301 controls)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">DK + SE (322 cases; 4893)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">7.70E − 10</td><td rowspan="1" colspan="1">6.569</td><td rowspan="1" colspan="1">0.853</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">DK + SE + US (511 cases; 5319)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">5.55E − 07</td><td rowspan="1" colspan="1">3.734</td><td rowspan="1" colspan="1">0.001</td></tr></tbody></table></div><div id="largeobj_idm140452014410768" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5891499/table/Tab2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p6"><em>SNP</em> dbSNP v146 variant ID, <em>Position</em> hg19 human genome assembly, <em>A1</em> effect allele, <em>A2</em> alternative allele, <em>Cases</em> A1 frequency in cases, <em>Controls</em> A1 frequency in controls, <em>OR</em> odds ratio of the effect allele, <em>SE</em> standard error, <em>P</em>
<em>P</em> value for the logistic regression, <em>P_meta</em> fixed-effects conditional meta-analysis <em>P</em> value, <em>OR_meta</em> fixed-effects conditional meta-analysis odds ratio, <em>Q</em>
<em>P</em> value for Cochrane’s <em>Q</em> statistic (measure of heterogeneity), <em>DK</em> Denmark, <em>SE</em> Sweden, <em>US</em> USA, <em>FI</em> Finland</p></div></div></div><div id="Sec18" class="sec"><h3 id="Sec18title">Genetic correlation</h3><p id="Par17" class="p p-first-last">Since ulcerative colitis has been found to be associated with HSCR [<a href="#CR39" rid="CR39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494368">39</a>], we sought to calculate their genetic correlation by using the LD score regression method [<a href="#CR33" rid="CR33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494352">33</a>, <a href="#CR34" rid="CR34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494362">34</a>] on our HSCR GWAS vs. a publicly available ulcerative colitis GWAS [<a href="#CR39" rid="CR39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494324">39</a>]. We detected a positive genetic correlation of 0.23, although not statistically significant (<em>P</em> = 0.09). Future studies are needed to investigate possible shared genetic architecture of these diseases [<a href="#CR38" rid="CR38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494323">38</a>, <a href="#CR40" rid="CR40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494333">40</a>].</p></div><div id="Sec19" class="sec"><h3 id="Sec19title">Heritability estimation</h3><p id="Par18" class="p p-first-last">Using th GCTA tool [<a href="#CR35" rid="CR35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494371">35</a>] on the set of genotyped SNPs, we estimated the heritability to be 88% ± standard error (SE) 13% (<em>P</em> = 6.1 × 10<sup>−10</sup>). This heritability estimate is in line with previous family-based estimates of HSCR heritability [<a href="#CR1" rid="CR1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494316">1</a>], indicating that most of the genetic component of the disease is captured by the genotyped variants. This estimate contrasts with the recent heritability estimate of an Asian HSCR GWAS [<a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267291">12</a>] (also using GCTA) of 34% ± SE 11% (<em>P</em> = 3.0 × 10<sup>−3</sup>). Since the heritability estimate of the Asian GWAS [<a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267298">12</a>] was performed only for variants with MAF &gt;5%, we redid the heritability estimate of our discovery cohort with just MAF &gt;5%, which resulted in 79% ± S.E. 11% (<em>P</em> = 2.6 × 10<sup>−13</sup>) of variance explained. This heritability estimate is still much higher than what is reported in the Asian GWAS [<a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267296">12</a>], and suggests most of the HSCR heritability in our cohort is tagged by common variation. This discrepancy in heritability estimates between the two populations might indicate (1) different genetic architecture of HSCR in Europeans vs. Asians, (2) SNP chips tagging common variation better in populations of European ancestry [<a href="#CR41" rid="CR41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494348">41</a>], and/or (3) different case selection procedures (e.g., length of aganglionosis, male:female ratio, prevalence of HSCR), which all might play a role when calculating heritability estimates.</p></div><div id="Sec20" class="sec"><h3 id="Sec20title">Case–control prediction</h3><p id="Par19" class="p p-first-last">To investigate the possibility of predicting Hirschsprung disease based on a genetic prediction model, we first cross-validated a Lasso prediction approach in the discovery stage data (see section “Materials and methods”). The procedure obtained a mean cross-validation AUC of 81% for the optimal penalization parameter <em>λ</em>* = 0.0114. Based on this penalization, 10 variants were selected by the final Lasso regression on the full discovery stage data. Selected variants and loadings are provided in Table&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">S1</a>. We investigated how the predictive performance of this model generalize in the independent US cohort. Here we obtained a validation AUC of 76% (Fig.&nbsp;<a href="/pmc/articles/PMC5891499/figure/Fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="Fig2" rid-ob="ob-Fig2" co-legend-rid="lgnd_Fig2"><span>2</span></a>). The reduced case–control prediction accuracy in the US replication cohort might mean that the Lasso selected variants could be tagging underlying causal variants less well here than in the Danish ancestry discovery cohort. We note that five of the 10 variants are in the <em>RET</em> locus, one is the lead <em>SEMA3</em> variant, while the remaining four are in loci that also were selected for replication (Table&nbsp;<a href="#MOESM1" rid="MOESM1" class=" media">S1</a>). For most of these loci the association signal was attenuated in the US cohort (Table&nbsp;<a href="/pmc/articles/PMC5891499/table/Tab1/" target="table" class="fig-table-link figpopup" rid-figpopup="Tab1" rid-ob="ob-Tab1" co-legend-rid=""><span>1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="Fig2" co-legend-rid="lgnd_Fig2"><a href="/pmc/articles/PMC5891499/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140452016664032" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is 41431_2017_53_Fig2_HTML.jpg" title="An external file that holds a picture, illustration, etc.
Object name is 41431_2017_53_Fig2_HTML.jpg" src="/pmc/articles/PMC5891499/bin/41431_2017_53_Fig2_HTML.jpg"></div></a><div id="largeobj_idm140452016664032" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5891499/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2"></a><a target="object" href="/pmc/articles/PMC5891499/figure/Fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_Fig2"><div><a class="figpopup" href="/pmc/articles/PMC5891499/figure/Fig2/" target="figure" rid-figpopup="Fig2" rid-ob="ob-Fig2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p7">The ROC curve for our HSCR genetic risk prediction model using 10 Lasso selected SNPs tested in an independent cohort (see section “Materials and methods”) (see color fig online)</p></div></div></div></div><div id="Sec21" class="sec sec-last"><h3 id="Sec21title">Regulatory enrichment analysis</h3><p id="Par20" class="p p-first-last">We have also mapped the H3K4me1, H3K27ac ChIP-seq peak, and DNase I hypersensitive sites from the human fetal large intestine datasets obtained from the NIH Roadmap Epigenomics Project and the 225 kb topologically associated domain (TAD), which is common across nine cell lines as described previously [<a href="#CR42" rid="CR42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494363">42</a>]. Our lead <em>RET</em> SNP (rs2505994) and the newly identified variant (rs144432435) both fall within the TAD, although they do not seem to overlap any of the open chromatin regions or putative enhancer marks (Figure&nbsp;<a href="#MOESM2" rid="MOESM2" class=" media">S8</a>).</p></div></div><div id="Sec22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Sec22title">Discussion</h2><p id="Par21" class="p p-first">In this study, we confirmed the previously reported <em>RET</em> and <em>SEMA3</em> loci [<a href="#CR9" rid="CR9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494370">9</a>, <a href="#CR10" rid="CR10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494358">10</a>, <a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267297">12</a>] as being associated with HSCR in our Danish cohort. However, we could not replicate the <em>NRG1</em> locus previously reported to be associated with HSCR in East Asian populations [<a href="#CR11" rid="CR11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494315">11</a>, <a href="#CR12" rid="CR12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_726267288">12</a>]. Additionally, we detected and replicated a novel low-frequency variant, rs144432435, which was genome-wide significantly associated with HSCR, when conditioning on our <em>RET</em> lead SNP, rs2505994. Furthermore, we estimated the heritability of HSCR, based on the genotyped SNPs, to be 88%, close to classical family study-based estimates [<a href="#CR1" rid="CR1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494346">1</a>]. We were able to predict HSCR case–control status with good accuracy based on a subset of 10 SNPs (AUC = 0.76), which might prove useful as an add-on to a first tier HSCR diagnostic [<a href="#CR43" rid="CR43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_691494335">43</a>]. In conclusion, we combined the largest collection of sporadic Hirschsprung cases to date (586 cases) to further elucidate the genetic architecture of this rare disease.</p><p id="Par22" class="p p-last">The GWAS summary statistics from our study are available at <a href="http://danishnationalbiobank.com/gwas.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://danishnationalbiobank.com/gwas.html</a>.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Electronic supplementary material</h2><!--/article/body/sec/--><div id="Sec23" class="sec sec-first-last"><p id="__p8" class="p p-first-last">
</p><div class="sec suppmat" id="MOESM1"><div class="sup-box half_rhythm" id="idm140452014465376"><a href="/pmc/articles/PMC5891499/bin/41431_2017_53_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl">Table S1</a><sup>(205K, pdf)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM2"><div class="sup-box half_rhythm" id="idm140452081159280"><a href="/pmc/articles/PMC5891499/bin/41431_2017_53_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl">Supplemental Data</a><sup>(2.1M, pdf)</sup></div></div><p>
</p><div class="sec suppmat" id="MOESM3"><div class="sup-box half_rhythm" id="idm140452027315904"><a href="/pmc/articles/PMC5891499/bin/41431_2017_53_MOESM3_ESM.docx" data-ga-action="click_feat_suppl">Table S1 Legend</a><sup>(14K, docx)</sup></div></div><p>
</p></div></div><div id="idm140452081053568" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140452081053568title">Acknowledgements</h2><div class="sec"><p id="__p14">We wish to thank the International Hirschsprung Disease Consortium for sharing their data. This study was funded by the Danish Medical Research council (DFF 4004-00512) and has been conducted using the Danish National Biobank resource. The Danish National Biobank is supported by the Novo Nordisk Foundation. BF was supported by an Oak Foundation fellowship. AC was supported by NIH grant HD28088.</p></div></div><div id="idm140452014834496" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140452014834496title">Compliance with ethical standards</h2><div id="FPar1" class="sec sec-first"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="FPar1title" class="ui-helper-clearfix">Conflict of interest</h2><p id="Par23" class="p p-first-last">The authors declare they have no conflict of interest.</p></div></div><div id="idm140452081055248" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140452081055248title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140452081055120"></p><p id="__p12" class="p p-first"><strong>Electronic supplementary material</strong></p><p id="__p13" class="p p-last">The online version of this article (10.1038/s41431-017-0053-7) contains supplementary material, which is available to authorized users.</p><p></p></div></div><div id="Bib1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Bib1title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CR1">1. <span class="element-citation">Badner JA, Sieber WK, Garver KL, Chakravarti A. A genetic study of Hirschsprung disease. <span><span class="ref-journal">Am J Hum Genet. </span>1990;<span class="ref-vol">46</span>:568–80.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1683643/">PMC free article</a>]</span> [<a href="/pubmed/2309705" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genetic+study+of+Hirschsprung+disease&amp;author=JA+Badner&amp;author=WK+Sieber&amp;author=KL+Garver&amp;author=A+Chakravarti&amp;volume=46&amp;publication_year=1990&amp;pages=568-80&amp;pmid=2309705&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR2">2. <span class="element-citation">Löf Granström A, Svenningsson A, Hagel E, Oddsberg J, Nordenskjöld A, Wester T. Maternal risk factors and perinatal characteristics for Hirschsprung disease. <span><span class="ref-journal">Pediatrics. </span>2016;<span class="ref-vol">138</span>:e20154608. doi: 10.1542/peds.2015-4608.</span> [<a href="/pubmed/27307146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1542%2Fpeds.2015-4608" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Maternal+risk+factors+and+perinatal+characteristics+for+Hirschsprung+disease&amp;author=A+Löf+Granström&amp;author=A+Svenningsson&amp;author=E+Hagel&amp;author=J+Oddsberg&amp;author=A+Nordenskjöld&amp;volume=138&amp;publication_year=2016&amp;pages=e20154608&amp;pmid=27307146&amp;doi=10.1542/peds.2015-4608&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR3">3. <span class="element-citation">Heanue TA, Pachnis V. Enteric nervous system development and Hirschsprung’s disease: advances in genetic and stem cell studies. <span><span class="ref-journal">Nat Rev Neurosci. </span>2007;<span class="ref-vol">8</span>:466–79. doi: 10.1038/nrn2137.</span> [<a href="/pubmed/17514199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnrn2137" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Neurosci&amp;title=Enteric+nervous+system+development+and+Hirschsprung’s+disease:+advances+in+genetic+and+stem+cell+studies&amp;author=TA+Heanue&amp;author=V+Pachnis&amp;volume=8&amp;publication_year=2007&amp;pages=466-79&amp;pmid=17514199&amp;doi=10.1038/nrn2137&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR4">4. <span class="element-citation">Parisi MA, Kapur RP. Genetics of Hirschsprung disease. <span><span class="ref-journal">Curr Opin Pediatr. </span>2000;<span class="ref-vol">12</span>:610–7. doi: 10.1097/00008480-200012000-00017.</span> [<a href="/pubmed/11106284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1097%2F00008480-200012000-00017" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Pediatr&amp;title=Genetics+of+Hirschsprung+disease&amp;author=MA+Parisi&amp;author=RP+Kapur&amp;volume=12&amp;publication_year=2000&amp;pages=610-7&amp;pmid=11106284&amp;doi=10.1097/00008480-200012000-00017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR5">5. <span class="element-citation">Burkardt DD, Graham JM, Jr, Short SS, Frykman PK. Advances in Hirschsprung disease genetics and treatment strategies: an update for the primary care pediatrician. <span><span class="ref-journal">Clin Pediatr. </span>2014;<span class="ref-vol">53</span>:71–81. doi: 10.1177/0009922813500846.</span> [<a href="/pubmed/24002048" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1177%2F0009922813500846" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pediatr&amp;title=Advances+in+Hirschsprung+disease+genetics+and+treatment+strategies:+an+update+for+the+primary+care+pediatrician&amp;author=DD+Burkardt&amp;author=JM+Graham&amp;author=SS+Short&amp;author=PK+Frykman&amp;volume=53&amp;publication_year=2014&amp;pages=71-81&amp;doi=10.1177/0009922813500846&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR6">6. <span class="element-citation">Amiel J, Sproat-Emison E, Garcia-Barcelo M, et al.  Hirschsprung disease, associated syndromes and genetics: a review. <span><span class="ref-journal">J Med Genet. </span>2008;<span class="ref-vol">45</span>:1–14. doi: 10.1136/jmg.2007.053959.</span> [<a href="/pubmed/17965226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1136%2Fjmg.2007.053959" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Hirschsprung+disease,+associated+syndromes+and+genetics:+a+review&amp;author=J+Amiel&amp;author=E+Sproat-Emison&amp;author=M+Garcia-Barcelo&amp;volume=45&amp;publication_year=2008&amp;pages=1-14&amp;pmid=17965226&amp;doi=10.1136/jmg.2007.053959&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR7">7. <span class="element-citation">Sergi C. Hirschsprung’s disease: historical notes and pathological diagnosis on the occasion of the 100(th) anniversary of Dr. Harald Hirschsprung’s death. <span><span class="ref-journal">World J Clin Pediatr. </span>2015;<span class="ref-vol">4</span>:120–5. doi: 10.5409/wjcp.v4.i4.120.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4637802/">PMC free article</a>]</span> [<a href="/pubmed/26566484" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.5409%2Fwjcp.v4.i4.120" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=World+J+Clin+Pediatr&amp;title=Hirschsprung’s+disease:+historical+notes+and+pathological+diagnosis+on+the+occasion+of+the+100(th)+anniversary+of+Dr.+Harald+Hirschsprung’s+death&amp;author=C+Sergi&amp;volume=4&amp;publication_year=2015&amp;pages=120-5&amp;pmid=26566484&amp;doi=10.5409/wjcp.v4.i4.120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR8">8. <span class="element-citation">Alves MM, Sribudiani Y, Brouwer RW, et al.  Contribution of rare and common variants determine complex diseases-Hirschsprung disease as a model. <span><span class="ref-journal">Dev Biol. </span>2013;<span class="ref-vol">382</span>:320–9. doi: 10.1016/j.ydbio.2013.05.019.</span> [<a href="/pubmed/23707863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ydbio.2013.05.019" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Biol&amp;title=Contribution+of+rare+and+common+variants+determine+complex+diseases-Hirschsprung+disease+as+a+model&amp;author=MM+Alves&amp;author=Y+Sribudiani&amp;author=RW+Brouwer&amp;volume=382&amp;publication_year=2013&amp;pages=320-9&amp;pmid=23707863&amp;doi=10.1016/j.ydbio.2013.05.019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR9">9. <span class="element-citation">Jiang Q, Arnold S, Heanue T, et al.  Functional loss of semaphorin 3C and/or semaphorin 3D and their epistatic interaction with ret are critical to Hirschsprung disease liability. <span><span class="ref-journal">Am J Hum Genet. </span>2015;<span class="ref-vol">96</span>:581–96. doi: 10.1016/j.ajhg.2015.02.014.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4385176/">PMC free article</a>]</span> [<a href="/pubmed/25839327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ajhg.2015.02.014" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Functional+loss+of+semaphorin+3C+and/or+semaphorin+3D+and+their+epistatic+interaction+with+ret+are+critical+to+Hirschsprung+disease+liability&amp;author=Q+Jiang&amp;author=S+Arnold&amp;author=T+Heanue&amp;volume=96&amp;publication_year=2015&amp;pages=581-96&amp;pmid=25839327&amp;doi=10.1016/j.ajhg.2015.02.014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR10">10. <span class="element-citation">Emison ES, McCallion AS, Kashuk CS, et al.  A common sex-dependent mutation in a RET enhancer underlies Hirschsprung disease risk. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">434</span>:857–63. doi: 10.1038/nature03467.</span> [<a href="/pubmed/15829955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnature03467" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+common+sex-dependent+mutation+in+a+RET+enhancer+underlies+Hirschsprung+disease+risk&amp;author=ES+Emison&amp;author=AS+McCallion&amp;author=CS+Kashuk&amp;volume=434&amp;publication_year=2005&amp;pages=857-63&amp;pmid=15829955&amp;doi=10.1038/nature03467&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR11">11. <span class="element-citation">Garcia-Barcelo MM, Tang CS, Ngan ES, et al.  Genome-wide association study identifies NRG1 as a susceptibility locus for Hirschsprung’s disease. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2009;<span class="ref-vol">106</span>:2694–9. doi: 10.1073/pnas.0809630105.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2650328/">PMC free article</a>]</span> [<a href="/pubmed/19196962" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1073%2Fpnas.0809630105" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Genome-wide+association+study+identifies+NRG1+as+a+susceptibility+locus+for+Hirschsprung’s+disease&amp;author=MM+Garcia-Barcelo&amp;author=CS+Tang&amp;author=ES+Ngan&amp;volume=106&amp;publication_year=2009&amp;pages=2694-9&amp;pmid=19196962&amp;doi=10.1073/pnas.0809630105&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR12">12. <span class="element-citation">Tang CS, Gui H, Kapoor A, et al.  <em>Trans</em>-ethnic meta-analysis of genome-wide association studies for Hirschsprung disease. <span><span class="ref-journal">Hum Mol Genet. </span>2016;<span class="ref-vol">25</span>:5265–75. doi: 10.1093/hmg/ddw079.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6078638/">PMC free article</a>]</span> [<a href="/pubmed/27702942" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fhmg%2Fddw079" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Trans-ethnic+meta-analysis+of+genome-wide+association+studies+for+Hirschsprung+disease&amp;author=CS+Tang&amp;author=H+Gui&amp;author=A+Kapoor&amp;volume=25&amp;publication_year=2016&amp;pages=5265-75&amp;pmid=27702942&amp;doi=10.1093/hmg/ddw079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR13">13. <span class="element-citation">Manolio TA, Collins FS, Cox NJ, et al.  Finding the missing heritability of complex diseases. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">461</span>:747–53. doi: 10.1038/nature08494.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2831613/">PMC free article</a>]</span> [<a href="/pubmed/19812666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnature08494" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Finding+the+missing+heritability+of+complex+diseases&amp;author=TA+Manolio&amp;author=FS+Collins&amp;author=NJ+Cox&amp;volume=461&amp;publication_year=2009&amp;pages=747-53&amp;pmid=19812666&amp;doi=10.1038/nature08494&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR14">14. <span class="element-citation">Bjerregaard B, Larsen OB. The Danish Pathology Register. <span><span class="ref-journal">Scand J Public Health. </span>2011;<span class="ref-vol">39</span>:72–4. doi: 10.1177/1403494810393563.</span> [<a href="/pubmed/21775357" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1177%2F1403494810393563" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Scand+J+Public+Health&amp;title=The+Danish+Pathology+Register&amp;author=B+Bjerregaard&amp;author=OB+Larsen&amp;volume=39&amp;publication_year=2011&amp;pages=72-4&amp;pmid=21775357&amp;doi=10.1177/1403494810393563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR15">15. <span class="element-citation">Pedersen CB. The Danish Civil Registration System. <span><span class="ref-journal">Scand J Public Health. </span>2011;<span class="ref-vol">39</span>:22–5. doi: 10.1177/1403494810387965.</span> [<a href="/pubmed/21775345" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1177%2F1403494810387965" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Scand+J+Public+Health&amp;title=The+Danish+Civil+Registration+System&amp;author=CB+Pedersen&amp;volume=39&amp;publication_year=2011&amp;pages=22-5&amp;pmid=21775345&amp;doi=10.1177/1403494810387965&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR16">16. <span class="element-citation">Feenstra B, Pasternak B, Geller F, et al.  Common variants associated with general and MMR vaccine-related febrile seizures. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:1274–82. doi: 10.1038/ng.3129.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4244308/">PMC free article</a>]</span> [<a href="/pubmed/25344690" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3129" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+associated+with+general+and+MMR+vaccine-related+febrile+seizures&amp;author=B+Feenstra&amp;author=B+Pasternak&amp;author=F+Geller&amp;volume=46&amp;publication_year=2014&amp;pages=1274-82&amp;pmid=25344690&amp;doi=10.1038/ng.3129&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR17">17. <span class="element-citation">Geller F, Feenstra B, Carstensen L, et al.  Genome-wide association analyses identify variants in developmental genes associated with hypospadias. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:957–63. doi: 10.1038/ng.3063.</span> [<a href="/pubmed/25108383" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3063" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analyses+identify+variants+in+developmental+genes+associated+with+hypospadias&amp;author=F+Geller&amp;author=B+Feenstra&amp;author=L+Carstensen&amp;volume=46&amp;publication_year=2014&amp;pages=957-63&amp;pmid=25108383&amp;doi=10.1038/ng.3063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR18">18. <span class="element-citation">Nørgaard-Pedersen B, Hougaard DM. Storage policies and use of the Danish Newborn Screening Biobank. <span><span class="ref-journal">J Inherit Metab Dis. </span>2007;<span class="ref-vol">30</span>:530–6. doi: 10.1007/s10545-007-0631-x.</span> [<a href="/pubmed/17632694" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1007%2Fs10545-007-0631-x" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Inherit+Metab+Dis&amp;title=Storage+policies+and+use+of+the+Danish+Newborn+Screening+Biobank&amp;author=B+Nørgaard-Pedersen&amp;author=DM+Hougaard&amp;volume=30&amp;publication_year=2007&amp;pages=530-6&amp;pmid=17632694&amp;doi=10.1007/s10545-007-0631-x&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR19">19. <span class="element-citation">Hollegaard MV, Grauholm J, Børglum A, et al.  Genome-wide scans using archived neonatal dried blood spot samples. <span><span class="ref-journal">BMC Genom. </span>2009;<span class="ref-vol">10</span>:297. doi: 10.1186/1471-2164-10-297.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2713266/">PMC free article</a>]</span> [<a href="/pubmed/19575812" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1186%2F1471-2164-10-297" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genom&amp;title=Genome-wide+scans+using+archived+neonatal+dried+blood+spot+samples&amp;author=MV+Hollegaard&amp;author=J+Grauholm&amp;author=A+Børglum&amp;volume=10&amp;publication_year=2009&amp;pages=297&amp;doi=10.1186/1471-2164-10-297&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR20">20. <span class="element-citation">Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. <span><span class="ref-journal">Gigascience. </span>2015;<span class="ref-vol">4</span>:7. doi: 10.1186/s13742-015-0047-8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4342193/">PMC free article</a>]</span> [<a href="/pubmed/25722852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1186%2Fs13742-015-0047-8" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Second-generation+PLINK:+rising+to+the+challenge+of+larger+and+richer+datasets&amp;author=CC+Chang&amp;author=CC+Chow&amp;author=LC+Tellier&amp;author=S+Vattikuti&amp;author=SM+Purcell&amp;volume=4&amp;publication_year=2015&amp;pages=7&amp;pmid=25722852&amp;doi=10.1186/s13742-015-0047-8&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR21">21. <span class="element-citation">Manichaikul A, Mychaleckyj JC, Rich SS, Daly K, Sale M, Chen WM. Robust relationship inference in genome-wide association studies. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2867–73. doi: 10.1093/bioinformatics/btq559.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025716/">PMC free article</a>]</span> [<a href="/pubmed/20926424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtq559" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Robust+relationship+inference+in+genome-wide+association+studies&amp;author=A+Manichaikul&amp;author=JC+Mychaleckyj&amp;author=SS+Rich&amp;author=K+Daly&amp;author=M+Sale&amp;volume=26&amp;publication_year=2010&amp;pages=2867-73&amp;pmid=20926424&amp;doi=10.1093/bioinformatics/btq559&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR22">22. <span class="element-citation">Danecek P, Auton A, Abecasis G, et al.  The variant call format and VCFtools. <span><span class="ref-journal">Bioinformatics. </span>2011;<span class="ref-vol">27</span>:2156–8. doi: 10.1093/bioinformatics/btr330.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3137218/">PMC free article</a>]</span> [<a href="/pubmed/21653522" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtr330" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The+variant+call+format+and+VCFtools&amp;author=P+Danecek&amp;author=A+Auton&amp;author=G+Abecasis&amp;volume=27&amp;publication_year=2011&amp;pages=2156-8&amp;pmid=21653522&amp;doi=10.1093/bioinformatics/btr330&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR23">23. <span class="element-citation">Zheng X, Levine D, Shen J, Gogarten SM, Laurie C, Weir BS. A high-performance computing toolset for relatedness and principal component analysis of SNP data. <span><span class="ref-journal">Bioinformatics. </span>2012;<span class="ref-vol">28</span>:3326–8. doi: 10.1093/bioinformatics/bts606.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3519454/">PMC free article</a>]</span> [<a href="/pubmed/23060615" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbts606" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+high-performance+computing+toolset+for+relatedness+and+principal+component+analysis+of+SNP+data&amp;author=X+Zheng&amp;author=D+Levine&amp;author=J+Shen&amp;author=SM+Gogarten&amp;author=C+Laurie&amp;volume=28&amp;publication_year=2012&amp;pages=3326-8&amp;pmid=23060615&amp;doi=10.1093/bioinformatics/bts606&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR24">24. <span class="element-citation">Jakobsson M, Scholz SW, Scheet P, et al.  Genotype, haplotype and copy-number variation in worldwide human populations. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">451</span>:998–1003. doi: 10.1038/nature06742.</span> [<a href="/pubmed/18288195" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnature06742" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genotype,+haplotype+and+copy-number+variation+in+worldwide+human+populations&amp;author=M+Jakobsson&amp;author=SW+Scholz&amp;author=P+Scheet&amp;volume=451&amp;publication_year=2008&amp;pages=998-1003&amp;pmid=18288195&amp;doi=10.1038/nature06742&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR25">25. <span class="element-citation">Ma J, Amos CI. Investigation of inversion polymorphisms in the human genome using principal components analysis. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e40224. doi: 10.1371/journal.pone.0040224.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3392271/">PMC free article</a>]</span> [<a href="/pubmed/22808122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pone.0040224" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Investigation+of+inversion+polymorphisms+in+the+human+genome+using+principal+components+analysis&amp;author=J+Ma&amp;author=CI+Amos&amp;volume=7&amp;publication_year=2012&amp;pages=e40224&amp;pmid=22808122&amp;doi=10.1371/journal.pone.0040224&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR26">26. <span class="element-citation">McCarthy S, Das S, Kretzschmar W, et al.  A reference panel of 64,976 haplotypes for genotype imputation. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:1279–83. doi: 10.1038/ng.3643.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5388176/">PMC free article</a>]</span> [<a href="/pubmed/27548312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3643" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+reference+panel+of+64,976+haplotypes+for+genotype+imputation&amp;author=S+McCarthy&amp;author=S+Das&amp;author=W+Kretzschmar&amp;volume=48&amp;publication_year=2016&amp;pages=1279-83&amp;pmid=27548312&amp;doi=10.1038/ng.3643&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR27">27. <span class="element-citation">1000 Genomes Project Consortium.  A global reference for human genetic variation. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">526</span>:68–74. doi: 10.1038/nature15393.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4750478/">PMC free article</a>]</span> [<a href="/pubmed/26432245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnature15393" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+global+reference+for+human+genetic+variation&amp;volume=526&amp;publication_year=2015&amp;pages=68-74&amp;pmid=26432245&amp;doi=10.1038/nature15393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR28">28. <span class="element-citation">Ward LD, Kellis M. HaploRegv4: systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>:D877–81. doi: 10.1093/nar/gkv1340.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4702929/">PMC free article</a>]</span> [<a href="/pubmed/26657631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fnar%2Fgkv1340" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=HaploRegv4:+systematic+mining+of+putative+causal+variants,+cell+types,+regulators+and+target+genes+for+human+complex+traits+and+disease&amp;author=LD+Ward&amp;author=M+Kellis&amp;volume=44&amp;publication_year=2016&amp;pages=D877-81&amp;pmid=26657631&amp;doi=10.1093/nar/gkv1340&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR29">29. <span class="element-citation">Durbin R. Efficient haplotype matching and storage using the positional Burrows–Wheeler transform (PBWT) <span><span class="ref-journal">Bioinformatics. </span>2014;<span class="ref-vol">30</span>:1266–72. doi: 10.1093/bioinformatics/btu014.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3998136/">PMC free article</a>]</span> [<a href="/pubmed/24413527" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtu014" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Efficient+haplotype+matching+and+storage+using+the+positional+Burrows–Wheeler+transform+(PBWT)&amp;author=R+Durbin&amp;volume=30&amp;publication_year=2014&amp;pages=1266-72&amp;pmid=24413527&amp;doi=10.1093/bioinformatics/btu014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR30">30. <span class="element-citation">Delaneau O, Zagury JF, Marchini J. Improved whole-chromosome phasing for disease and population genetic studies. <span><span class="ref-journal">Nat Methods. </span>2013;<span class="ref-vol">10</span>:5–6. doi: 10.1038/nmeth.2307.</span> [<a href="/pubmed/23269371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnmeth.2307" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Improved+whole-chromosome+phasing+for+disease+and+population+genetic+studies&amp;author=O+Delaneau&amp;author=JF+Zagury&amp;author=J+Marchini&amp;volume=10&amp;publication_year=2013&amp;pages=5-6&amp;pmid=23269371&amp;doi=10.1038/nmeth.2307&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR31">31. <span class="element-citation">Aulchenko YS, Ripke S, Isaacs A, van Duijn CM. GenABEL: an R library for genome-wide association analysis. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:1294–6. doi: 10.1093/bioinformatics/btm108.</span> [<a href="/pubmed/17384015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtm108" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenABEL:+an+R+library+for+genome-wide+association+analysis&amp;author=YS+Aulchenko&amp;author=S+Ripke&amp;author=A+Isaacs&amp;author=CM+van+Duijn&amp;volume=23&amp;publication_year=2007&amp;pages=1294-6&amp;pmid=17384015&amp;doi=10.1093/bioinformatics/btm108&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR32">32. <span class="element-citation">Pruim RJ, Welch RP, Sanna S, et al.  LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–7. doi: 10.1093/bioinformatics/btq419.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtq419" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;volume=26&amp;publication_year=2010&amp;pages=2336-7&amp;pmid=20634204&amp;doi=10.1093/bioinformatics/btq419&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR33">33. <span class="element-citation">Bulik-Sullivan BK, Loh PR, Finucane HK, et al.  LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:291–5. doi: 10.1038/ng.3211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4495769/">PMC free article</a>]</span> [<a href="/pubmed/25642630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3211" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=LD+Score+regression+distinguishes+confounding+from+polygenicity+in+genome-wide+association+studies&amp;author=BK+Bulik-Sullivan&amp;author=PR+Loh&amp;author=HK+Finucane&amp;volume=47&amp;publication_year=2015&amp;pages=291-5&amp;pmid=25642630&amp;doi=10.1038/ng.3211&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR34">34. <span class="element-citation">Bulik-Sullivan B, Finucane HK, Anttila V, et al.  An atlas of genetic correlations across human diseases and traits. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:1236–41. doi: 10.1038/ng.3406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4797329/">PMC free article</a>]</span> [<a href="/pubmed/26414676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3406" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=An+atlas+of+genetic+correlations+across+human+diseases+and+traits&amp;author=B+Bulik-Sullivan&amp;author=HK+Finucane&amp;author=V+Anttila&amp;volume=47&amp;publication_year=2015&amp;pages=1236-41&amp;pmid=26414676&amp;doi=10.1038/ng.3406&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR35">35. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:76–82. doi: 10.1016/j.ajhg.2010.11.011.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ajhg.2010.11.011" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;doi=10.1016/j.ajhg.2010.11.011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR36">36. <span class="element-citation">Best KE, Addor MC, Arriola L, et al.  Hirschsprung’s disease prevalence in Europe: a register based study. <span><span class="ref-journal">Birth Defects Res A. </span>2014;<span class="ref-vol">100</span>:695–702. doi: 10.1002/bdra.23269.</span> [<a href="/pubmed/25066220" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1002%2Fbdra.23269" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res+A&amp;title=Hirschsprung’s+disease+prevalence+in+Europe:+a+register+based+study&amp;author=KE+Best&amp;author=MC+Addor&amp;author=L+Arriola&amp;volume=100&amp;publication_year=2014&amp;pages=695-702&amp;doi=10.1002/bdra.23269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR37">37. <span class="element-citation">Tibshirani R. Regression shrinkage and selection via the Lasso. <span><span class="ref-journal">J R Stat Soc B. </span>1996;<span class="ref-vol">58</span>:267–88.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+R+Stat+Soc+B&amp;title=Regression+shrinkage+and+selection+via+the+Lasso&amp;author=R+Tibshirani&amp;volume=58&amp;publication_year=1996&amp;pages=267-88&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR38">38. <span class="element-citation">Armstrong GR, Raafat F. Humoral reaction in the inflamed colon in Hirschsprung’s disease and ulcerative colitis. <span><span class="ref-journal">J Clin Pathol. </span>1988;<span class="ref-vol">41</span>:975–7. doi: 10.1136/jcp.41.9.975.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1141655/">PMC free article</a>]</span> [<a href="/pubmed/3263991" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1136%2Fjcp.41.9.975" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Pathol&amp;title=Humoral+reaction+in+the+inflamed+colon+in+Hirschsprung’s+disease+and+ulcerative+colitis&amp;author=GR+Armstrong&amp;author=F+Raafat&amp;volume=41&amp;publication_year=1988&amp;pages=975-7&amp;pmid=3263991&amp;doi=10.1136/jcp.41.9.975&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR39">39. <span class="element-citation">Liu JZ, van Sommeren S, Huang H, et al.  Association analyses identify 38 susceptibility loci for inflammatory bowel disease and highlight shared genetic risk across populations. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:979–86. doi: 10.1038/ng.3359.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4881818/">PMC free article</a>]</span> [<a href="/pubmed/26192919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3359" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Association+analyses+identify+38+susceptibility+loci+for+inflammatory+bowel+disease+and+highlight+shared+genetic+risk+across+populations&amp;author=JZ+Liu&amp;author=S+van+Sommeren&amp;author=H+Huang&amp;volume=47&amp;publication_year=2015&amp;pages=979-86&amp;pmid=26192919&amp;doi=10.1038/ng.3359&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR40">40. <span class="element-citation">Cucino C, Sonnenberg A. The comorbid occurrence of other diagnoses in patients with ulcerative colitis and Crohn’s disease. <span><span class="ref-journal">Am J Gastroenterol. </span>2001;<span class="ref-vol">96</span>:2107–12. doi: 10.1111/j.1572-0241.2001.03943.x.</span> [<a href="/pubmed/11467640" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1111%2Fj.1572-0241.2001.03943.x" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Gastroenterol&amp;title=The+comorbid+occurrence+of+other+diagnoses+in+patients+with+ulcerative+colitis+and+Crohn’s+disease&amp;author=C+Cucino&amp;author=A+Sonnenberg&amp;volume=96&amp;publication_year=2001&amp;pages=2107-12&amp;pmid=11467640&amp;doi=10.1111/j.1572-0241.2001.03943.x&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR41">41. <span class="element-citation">Ha NT, Freytag S, Bickeboeller H. Coverage and efficiency in current SNP chips. <span><span class="ref-journal">Eur J Hum Genet. </span>2014;<span class="ref-vol">22</span>:1124–30. doi: 10.1038/ejhg.2013.304.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4135415/">PMC free article</a>]</span> [<a href="/pubmed/24448550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fejhg.2013.304" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Coverage+and+efficiency+in+current+SNP+chips&amp;author=NT+Ha&amp;author=S+Freytag&amp;author=H+Bickeboeller&amp;volume=22&amp;publication_year=2014&amp;pages=1124-30&amp;pmid=24448550&amp;doi=10.1038/ejhg.2013.304&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR42">42. <span class="element-citation">Chatterjee S, Kapoor A, Akiyama JA, et al.  Enhancer variants synergistically drive dysfunction of a gene regulatory network in Hirschsprung disease. <span><span class="ref-journal">Cell. </span>2016;<span class="ref-vol">167</span>:355–68. doi: 10.1016/j.cell.2016.09.005.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5113733/">PMC free article</a>]</span> [<a href="/pubmed/27693352" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.cell.2016.09.005" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Enhancer+variants+synergistically+drive+dysfunction+of+a+gene+regulatory+network+in+Hirschsprung+disease&amp;author=S+Chatterjee&amp;author=A+Kapoor&amp;author=JA+Akiyama&amp;volume=167&amp;publication_year=2016&amp;pages=355-68&amp;pmid=27693352&amp;doi=10.1016/j.cell.2016.09.005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CR43">43. <span class="element-citation">Hackam DJ, Reblock KK, Redlinger RE, Barksdale EM., Jr Diagnosis and outcome of Hirschsprung’s disease: does age really matter? <span><span class="ref-journal">Pediatr Surg Int. </span>2004;<span class="ref-vol">20</span>:319–22. doi: 10.1007/s00383-004-1188-5.</span> [<a href="/pubmed/15185108" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1007%2Fs00383-004-1188-5" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr+Surg+Int&amp;title=Diagnosis+and+outcome+of+Hirschsprung’s+disease:+does+age+really+matter?&amp;author=DJ+Hackam&amp;author=KK+Reblock&amp;author=RE+Redlinger&amp;author=EM+Barksdale&amp;volume=20&amp;publication_year=2004&amp;pages=319-22&amp;pmid=15185108&amp;doi=10.1007/s00383-004-1188-5&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5891499&amp;issue-id=310090&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#Abs1title">Abstract</a></li><li><a href="#Sec1title">Introduction</a></li><li><a href="#Sec2title">Materials and methods</a></li><li><a href="#Sec14title">Results</a></li><li><a href="#Sec22title">Discussion</a></li><li><a href="#__sec1title">Electronic supplementary material</a></li><li><a href="#idm140452081053568title">Acknowledgements</a></li><li><a href="#idm140452014834496title">Compliance with ethical standards</a></li><li><a href="#FPar1title">Conflict of interest</a></li><li><a href="#idm140452081055248title">Footnotes</a></li><li><a href="#Bib1title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5891499/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5891499/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5891499/pdf/41431_2017_Article_53.pdf">PDF (893K)</a></li> | <li><a href="#" data-citationid="PMC5891499" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5891499%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5891499%2F&amp;text=Genome-wide%20association%20study%20of%20Hirschsprung%20disease%20detects%20a%20novel%20low-frequency%20variant%20at%20the%20RET%20locus" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5891499%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29379196" data-aiid="5891499" data-aid="5891499" data-iid="310090" data-domainid="1300" data-domain="ejhg" data-accid="PMC5891499" data-md5="fb9c6c2847a0b1dcd21709d7790eeaae"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:56:45-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B4CB7DD5AE6C10000000004C302C4&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5891499%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5891499/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>