<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5783787/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Novel Blood Pressure Locus and Gene Discovery Using Genome-Wide Association Study and Expression Data Sets From Blood and the Kidney</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5783787/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Hypertension (Dallas, Tex. : 1979)"><meta name="citation_title" content="Novel Blood Pressure Locus and Gene Discovery Using Genome-Wide Association Study and Expression Data Sets From Blood and the Kidney"><meta name="citation_authors" content="Louise V. Wain, Ahmad Vaez, Rick Jansen, Roby Joehanes, Peter J. van der Most, A. Mesut Erzurumluoglu, Paul F. O’Reilly, Claudia P. Cabrera, Helen R. Warren, Lynda M. Rose, Germaine C. Verwoert, Jouke-Jan Hottenga, Rona J. Strawbridge, Tonu Esko, Dan E. Arking, Shih-Jen Hwang, Xiuqing Guo, Zoltan Kutalik, Stella Trompet, Nick Shrine, Alexander Teumer, Janina S. Ried, Joshua C. Bis, Albert V. Smith, Najaf Amin, Ilja M. Nolte, Leo-Pekka Lyytikäinen, Anubha Mahajan, Nicholas J. Wareham, Edith Hofer, Peter K. Joshi, Kati Kristiansson, Michela Traglia, Aki S. Havulinna, Anuj Goel, Mike A. Nalls, Siim Sõber, Dragana Vuckovic, Jian’an Luan, M. Fabiola Del Greco, Kristin L. Ayers, Jaume Marrugat, Daniela Ruggiero, Lorna M. Lopez, Teemu Niiranen, Stefan Enroth, Anne U. Jackson, Christopher P. Nelson, Jennifer E. Huffman, Weihua Zhang, Jonathan Marten, Ilaria Gandin, Sarah E. Harris, Tatijana Zemunik, Yingchang Lu, Evangelos Evangelou, Nabi Shah, Martin H. de Borst, Massimo Mangino, Bram P. Prins, Archie Campbell, Ruifang Li-Gao, Ganesh Chauhan, Christopher Oldmeadow, Gonçalo Abecasis, Maryam Abedi, Caterina M. Barbieri, Michael R. Barnes, Chiara Batini, John Beilby, BIOS Consortium, Tineka Blake, Michael Boehnke, Erwin P. Bottinger, Peter S. Braund, Morris Brown, Marco Brumat, Harry Campbell, John C. Chambers, Massimiliano Cocca, Francis Collins, John Connell, Heather J. Cordell, Jeffrey J. Damman, Gail Davies, Eco J. de Geus, Renée de Mutsert, Joris Deelen, Yusuf Demirkale, Alex S.F. Doney, Marcus Dörr, Martin Farrall, Teresa Ferreira, Mattias Frånberg, He Gao, Vilmantas Giedraitis, Christian Gieger, Franco Giulianini, Alan J. Gow, Anders Hamsten, Tamara B. Harris, Albert Hofman, Elizabeth G. Holliday, Jennie Hui, Marjo-Riitta Jarvelin, Åsa Johansson, Andrew D. Johnson, Pekka Jousilahti, Antti Jula, Mika Kähönen, Sekar Kathiresan, Kay-Tee Khaw, Ivana Kolcic, Seppo Koskinen, Claudia Langenberg, Marty Larson, Lenore J. Launer, Benjamin Lehne, David C.M. Liewald, Lifelines Cohort Study, Li Lin, Lars Lind, François Mach, Chrysovalanto Mamasoula, Cristina Menni, Borbala Mifsud, Yuri Milaneschi, Anna Morgan, Andrew D. Morris, Alanna C. Morrison, Peter J. Munson, Priyanka Nandakumar, Quang Tri Nguyen, Teresa Nutile, Albertine J. Oldehinkel, Ben A. Oostra, Elin Org, Sandosh Padmanabhan, Aarno Palotie, Guillaume Paré, Alison Pattie, Brenda W.J.H. Penninx, Neil Poulter, Peter P. Pramstaller, Olli T. Raitakari, Meixia Ren, Kenneth Rice, Paul M. Ridker, Harriëtte Riese, Samuli Ripatti, Antonietta Robino, Jerome I. Rotter, Igor Rudan, Yasaman Saba, Aude Saint Pierre, Cinzia F. Sala, Antti-Pekka Sarin, Reinhold Schmidt, Rodney Scott, Marc A. Seelen, Denis C. Shields, David Siscovick, Rossella Sorice, Alice Stanton, David J. Stott, Johan Sundström, Morris Swertz, Kent D. Taylor, Simon Thom, Ioanna Tzoulaki, Christophe Tzourio, André G. Uitterlinden, Understanding Society Scientific Group, Uwe Völker, Peter Vollenweider, Sarah Wild, Gonneke Willemsen, Alan F. Wright, Jie Yao, Sébastien Thériault, David Conen, John Attia, Peter Sever, Stéphanie Debette, Dennis O. Mook-Kanamori, Eleftheria Zeggini, Tim D. Spector, Pim van der Harst, Colin N.A. Palmer, Anne-Claire Vergnaud, Ruth J.F. Loos, Ozren Polasek, John M. Starr, Giorgia Girotto, Caroline Hayward, Jaspal S. Kooner, Cecila M. Lindgren, Veronique Vitart, Nilesh J. Samani, Jaakko Tuomilehto, Ulf Gyllensten, Paul Knekt, Ian J. Deary, Marina Ciullo, Roberto Elosua, Bernard D. Keavney, Andrew A. Hicks, Robert A. Scott, Paolo Gasparini, Maris Laan, YongMei Liu, Hugh Watkins, Catharina A. Hartman, Veikko Salomaa, Daniela Toniolo, Markus Perola, James F. Wilson, Helena Schmidt, Jing Hua Zhao, Terho Lehtimäki, Cornelia M. van Duijn, Vilmundur Gudnason, Bruce M. Psaty, Annette Peters, Rainer Rettig, Alan James, J. Wouter Jukema, David P. Strachan, Walter Palmas, Andres Metspalu, Erik Ingelsson, Dorret I. Boomsma, Oscar H. Franco, Murielle Bochud, Christopher Newton-Cheh, Patricia B. Munroe, Paul Elliott, Daniel I. Chasman, Aravinda Chakravarti, Joanne Knight, Andrew P. Morris, Daniel Levy, Martin D. Tobin, Harold Snieder, Mark J. Caulfield, Georg B. Ehret"><meta name="citation_doi" content="10.1161/HYPERTENSIONAHA.117.09438"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5783787/?report=abstract"><meta name="citation_pmid" content="28739976"><meta name="DC.Title" content="Novel Blood Pressure Locus and Gene Discovery Using Genome-Wide Association Study and Expression Data Sets From Blood and the Kidney"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Louise V. Wain"><meta name="DC.Contributor" content="Ahmad Vaez"><meta name="DC.Contributor" content="Rick Jansen"><meta name="DC.Contributor" content="Roby Joehanes"><meta name="DC.Contributor" content="Peter J. van der Most"><meta name="DC.Contributor" content="A. Mesut Erzurumluoglu"><meta name="DC.Contributor" content="Paul F. O’Reilly"><meta name="DC.Contributor" content="Claudia P. Cabrera"><meta name="DC.Contributor" content="Helen R. Warren"><meta name="DC.Contributor" content="Lynda M. Rose"><meta name="DC.Contributor" content="Germaine C. Verwoert"><meta name="DC.Contributor" content="Jouke-Jan Hottenga"><meta name="DC.Contributor" content="Rona J. Strawbridge"><meta name="DC.Contributor" content="Tonu Esko"><meta name="DC.Contributor" content="Dan E. Arking"><meta name="DC.Contributor" content="Shih-Jen Hwang"><meta name="DC.Contributor" content="Xiuqing Guo"><meta name="DC.Contributor" content="Zoltan Kutalik"><meta name="DC.Contributor" content="Stella Trompet"><meta name="DC.Contributor" content="Nick Shrine"><meta name="DC.Contributor" content="Alexander Teumer"><meta name="DC.Contributor" content="Janina S. Ried"><meta name="DC.Contributor" content="Joshua C. Bis"><meta name="DC.Contributor" content="Albert V. Smith"><meta name="DC.Contributor" content="Najaf Amin"><meta name="DC.Contributor" content="Ilja M. Nolte"><meta name="DC.Contributor" content="Leo-Pekka Lyytikäinen"><meta name="DC.Contributor" content="Anubha Mahajan"><meta name="DC.Contributor" content="Nicholas J. Wareham"><meta name="DC.Contributor" content="Edith Hofer"><meta name="DC.Contributor" content="Peter K. Joshi"><meta name="DC.Contributor" content="Kati Kristiansson"><meta name="DC.Contributor" content="Michela Traglia"><meta name="DC.Contributor" content="Aki S. Havulinna"><meta name="DC.Contributor" content="Anuj Goel"><meta name="DC.Contributor" content="Mike A. Nalls"><meta name="DC.Contributor" content="Siim Sõber"><meta name="DC.Contributor" content="Dragana Vuckovic"><meta name="DC.Contributor" content="Jian’an Luan"><meta name="DC.Contributor" content="M. Fabiola Del Greco"><meta name="DC.Contributor" content="Kristin L. Ayers"><meta name="DC.Contributor" content="Jaume Marrugat"><meta name="DC.Contributor" content="Daniela Ruggiero"><meta name="DC.Contributor" content="Lorna M. Lopez"><meta name="DC.Contributor" content="Teemu Niiranen"><meta name="DC.Contributor" content="Stefan Enroth"><meta name="DC.Contributor" content="Anne U. Jackson"><meta name="DC.Contributor" content="Christopher P. Nelson"><meta name="DC.Contributor" content="Jennifer E. Huffman"><meta name="DC.Contributor" content="Weihua Zhang"><meta name="DC.Contributor" content="Jonathan Marten"><meta name="DC.Contributor" content="Ilaria Gandin"><meta name="DC.Contributor" content="Sarah E. Harris"><meta name="DC.Contributor" content="Tatijana Zemunik"><meta name="DC.Contributor" content="Yingchang Lu"><meta name="DC.Contributor" content="Evangelos Evangelou"><meta name="DC.Contributor" content="Nabi Shah"><meta name="DC.Contributor" content="Martin H. de Borst"><meta name="DC.Contributor" content="Massimo Mangino"><meta name="DC.Contributor" content="Bram P. Prins"><meta name="DC.Contributor" content="Archie Campbell"><meta name="DC.Contributor" content="Ruifang Li-Gao"><meta name="DC.Contributor" content="Ganesh Chauhan"><meta name="DC.Contributor" content="Christopher Oldmeadow"><meta name="DC.Contributor" content="Gonçalo Abecasis"><meta name="DC.Contributor" content="Maryam Abedi"><meta name="DC.Contributor" content="Caterina M. Barbieri"><meta name="DC.Contributor" content="Michael R. Barnes"><meta name="DC.Contributor" content="Chiara Batini"><meta name="DC.Contributor" content="John Beilby"><meta name="DC.Contributor" content="BIOS Consortium"><meta name="DC.Contributor" content="Tineka Blake"><meta name="DC.Contributor" content="Michael Boehnke"><meta name="DC.Contributor" content="Erwin P. Bottinger"><meta name="DC.Contributor" content="Peter S. Braund"><meta name="DC.Contributor" content="Morris Brown"><meta name="DC.Contributor" content="Marco Brumat"><meta name="DC.Contributor" content="Harry Campbell"><meta name="DC.Contributor" content="John C. Chambers"><meta name="DC.Contributor" content="Massimiliano Cocca"><meta name="DC.Contributor" content="Francis Collins"><meta name="DC.Contributor" content="John Connell"><meta name="DC.Contributor" content="Heather J. Cordell"><meta name="DC.Contributor" content="Jeffrey J. Damman"><meta name="DC.Contributor" content="Gail Davies"><meta name="DC.Contributor" content="Eco J. de Geus"><meta name="DC.Contributor" content="Renée de Mutsert"><meta name="DC.Contributor" content="Joris Deelen"><meta name="DC.Contributor" content="Yusuf Demirkale"><meta name="DC.Contributor" content="Alex S.F. Doney"><meta name="DC.Contributor" content="Marcus Dörr"><meta name="DC.Contributor" content="Martin Farrall"><meta name="DC.Contributor" content="Teresa Ferreira"><meta name="DC.Contributor" content="Mattias Frånberg"><meta name="DC.Contributor" content="He Gao"><meta name="DC.Contributor" content="Vilmantas Giedraitis"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Franco Giulianini"><meta name="DC.Contributor" content="Alan J. Gow"><meta name="DC.Contributor" content="Anders Hamsten"><meta name="DC.Contributor" content="Tamara B. Harris"><meta name="DC.Contributor" content="Albert Hofman"><meta name="DC.Contributor" content="Elizabeth G. Holliday"><meta name="DC.Contributor" content="Jennie Hui"><meta name="DC.Contributor" content="Marjo-Riitta Jarvelin"><meta name="DC.Contributor" content="Åsa Johansson"><meta name="DC.Contributor" content="Andrew D. Johnson"><meta name="DC.Contributor" content="Pekka Jousilahti"><meta name="DC.Contributor" content="Antti Jula"><meta name="DC.Contributor" content="Mika Kähönen"><meta name="DC.Contributor" content="Sekar Kathiresan"><meta name="DC.Contributor" content="Kay-Tee Khaw"><meta name="DC.Contributor" content="Ivana Kolcic"><meta name="DC.Contributor" content="Seppo Koskinen"><meta name="DC.Contributor" content="Claudia Langenberg"><meta name="DC.Contributor" content="Marty Larson"><meta name="DC.Contributor" content="Lenore J. Launer"><meta name="DC.Contributor" content="Benjamin Lehne"><meta name="DC.Contributor" content="David C.M. Liewald"><meta name="DC.Contributor" content="Lifelines Cohort Study"><meta name="DC.Contributor" content="Li Lin"><meta name="DC.Contributor" content="Lars Lind"><meta name="DC.Contributor" content="François Mach"><meta name="DC.Contributor" content="Chrysovalanto Mamasoula"><meta name="DC.Contributor" content="Cristina Menni"><meta name="DC.Contributor" content="Borbala Mifsud"><meta name="DC.Contributor" content="Yuri Milaneschi"><meta name="DC.Contributor" content="Anna Morgan"><meta name="DC.Contributor" content="Andrew D. Morris"><meta name="DC.Contributor" content="Alanna C. Morrison"><meta name="DC.Contributor" content="Peter J. Munson"><meta name="DC.Contributor" content="Priyanka Nandakumar"><meta name="DC.Contributor" content="Quang Tri Nguyen"><meta name="DC.Contributor" content="Teresa Nutile"><meta name="DC.Contributor" content="Albertine J. Oldehinkel"><meta name="DC.Contributor" content="Ben A. Oostra"><meta name="DC.Contributor" content="Elin Org"><meta name="DC.Contributor" content="Sandosh Padmanabhan"><meta name="DC.Contributor" content="Aarno Palotie"><meta name="DC.Contributor" content="Guillaume Paré"><meta name="DC.Contributor" content="Alison Pattie"><meta name="DC.Contributor" content="Brenda W.J.H. Penninx"><meta name="DC.Contributor" content="Neil Poulter"><meta name="DC.Contributor" content="Peter P. Pramstaller"><meta name="DC.Contributor" content="Olli T. Raitakari"><meta name="DC.Contributor" content="Meixia Ren"><meta name="DC.Contributor" content="Kenneth Rice"><meta name="DC.Contributor" content="Paul M. Ridker"><meta name="DC.Contributor" content="Harriëtte Riese"><meta name="DC.Contributor" content="Samuli Ripatti"><meta name="DC.Contributor" content="Antonietta Robino"><meta name="DC.Contributor" content="Jerome I. Rotter"><meta name="DC.Contributor" content="Igor Rudan"><meta name="DC.Contributor" content="Yasaman Saba"><meta name="DC.Contributor" content="Aude Saint Pierre"><meta name="DC.Contributor" content="Cinzia F. Sala"><meta name="DC.Contributor" content="Antti-Pekka Sarin"><meta name="DC.Contributor" content="Reinhold Schmidt"><meta name="DC.Contributor" content="Rodney Scott"><meta name="DC.Contributor" content="Marc A. Seelen"><meta name="DC.Contributor" content="Denis C. Shields"><meta name="DC.Contributor" content="David Siscovick"><meta name="DC.Contributor" content="Rossella Sorice"><meta name="DC.Contributor" content="Alice Stanton"><meta name="DC.Contributor" content="David J. Stott"><meta name="DC.Contributor" content="Johan Sundström"><meta name="DC.Contributor" content="Morris Swertz"><meta name="DC.Contributor" content="Kent D. Taylor"><meta name="DC.Contributor" content="Simon Thom"><meta name="DC.Contributor" content="Ioanna Tzoulaki"><meta name="DC.Contributor" content="Christophe Tzourio"><meta name="DC.Contributor" content="André G. Uitterlinden"><meta name="DC.Contributor" content="Understanding Society Scientific Group"><meta name="DC.Contributor" content="Uwe Völker"><meta name="DC.Contributor" content="Peter Vollenweider"><meta name="DC.Contributor" content="Sarah Wild"><meta name="DC.Contributor" content="Gonneke Willemsen"><meta name="DC.Contributor" content="Alan F. Wright"><meta name="DC.Contributor" content="Jie Yao"><meta name="DC.Contributor" content="Sébastien Thériault"><meta name="DC.Contributor" content="David Conen"><meta name="DC.Contributor" content="John Attia"><meta name="DC.Contributor" content="Peter Sever"><meta name="DC.Contributor" content="Stéphanie Debette"><meta name="DC.Contributor" content="Dennis O. Mook-Kanamori"><meta name="DC.Contributor" content="Eleftheria Zeggini"><meta name="DC.Contributor" content="Tim D. Spector"><meta name="DC.Contributor" content="Pim van der Harst"><meta name="DC.Contributor" content="Colin N.A. Palmer"><meta name="DC.Contributor" content="Anne-Claire Vergnaud"><meta name="DC.Contributor" content="Ruth J.F. Loos"><meta name="DC.Contributor" content="Ozren Polasek"><meta name="DC.Contributor" content="John M. Starr"><meta name="DC.Contributor" content="Giorgia Girotto"><meta name="DC.Contributor" content="Caroline Hayward"><meta name="DC.Contributor" content="Jaspal S. Kooner"><meta name="DC.Contributor" content="Cecila M. Lindgren"><meta name="DC.Contributor" content="Veronique Vitart"><meta name="DC.Contributor" content="Nilesh J. Samani"><meta name="DC.Contributor" content="Jaakko Tuomilehto"><meta name="DC.Contributor" content="Ulf Gyllensten"><meta name="DC.Contributor" content="Paul Knekt"><meta name="DC.Contributor" content="Ian J. Deary"><meta name="DC.Contributor" content="Marina Ciullo"><meta name="DC.Contributor" content="Roberto Elosua"><meta name="DC.Contributor" content="Bernard D. Keavney"><meta name="DC.Contributor" content="Andrew A. Hicks"><meta name="DC.Contributor" content="Robert A. Scott"><meta name="DC.Contributor" content="Paolo Gasparini"><meta name="DC.Contributor" content="Maris Laan"><meta name="DC.Contributor" content="YongMei Liu"><meta name="DC.Contributor" content="Hugh Watkins"><meta name="DC.Contributor" content="Catharina A. Hartman"><meta name="DC.Contributor" content="Veikko Salomaa"><meta name="DC.Contributor" content="Daniela Toniolo"><meta name="DC.Contributor" content="Markus Perola"><meta name="DC.Contributor" content="James F. Wilson"><meta name="DC.Contributor" content="Helena Schmidt"><meta name="DC.Contributor" content="Jing Hua Zhao"><meta name="DC.Contributor" content="Terho Lehtimäki"><meta name="DC.Contributor" content="Cornelia M. van Duijn"><meta name="DC.Contributor" content="Vilmundur Gudnason"><meta name="DC.Contributor" content="Bruce M. Psaty"><meta name="DC.Contributor" content="Annette Peters"><meta name="DC.Contributor" content="Rainer Rettig"><meta name="DC.Contributor" content="Alan James"><meta name="DC.Contributor" content="J. Wouter Jukema"><meta name="DC.Contributor" content="David P. Strachan"><meta name="DC.Contributor" content="Walter Palmas"><meta name="DC.Contributor" content="Andres Metspalu"><meta name="DC.Contributor" content="Erik Ingelsson"><meta name="DC.Contributor" content="Dorret I. Boomsma"><meta name="DC.Contributor" content="Oscar H. Franco"><meta name="DC.Contributor" content="Murielle Bochud"><meta name="DC.Contributor" content="Christopher Newton-Cheh"><meta name="DC.Contributor" content="Patricia B. Munroe"><meta name="DC.Contributor" content="Paul Elliott"><meta name="DC.Contributor" content="Daniel I. Chasman"><meta name="DC.Contributor" content="Aravinda Chakravarti"><meta name="DC.Contributor" content="Joanne Knight"><meta name="DC.Contributor" content="Andrew P. Morris"><meta name="DC.Contributor" content="Daniel Levy"><meta name="DC.Contributor" content="Martin D. Tobin"><meta name="DC.Contributor" content="Harold Snieder"><meta name="DC.Contributor" content="Mark J. Caulfield"><meta name="DC.Contributor" content="Georg B. Ehret"><meta name="DC.Identifier" content="10.1161/HYPERTENSIONAHA.117.09438"><meta name="DC.Language" content="en"><meta property="og:title" content="Novel Blood Pressure Locus and Gene Discovery Using Genome-Wide Association Study and Expression Data Sets From Blood and the Kidney"><meta property="og:type" content="article"><meta property="og:description" content="Elevated blood pressure is a major risk factor for cardiovascular disease and has a substantial genetic contribution. Genetic variation influencing blood pressure has the potential to identify new pharmacological targets for the treatment of hypertension. ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5783787/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5783787/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5783787/pdf/nihms902240.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A2BBCDD5A3E610000000007650454.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5783787%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5783787</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hypertension</a></span>. Author manuscript; available in PMC 2019 Jan 24.</div><div></div><div>Published before final editing as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28739976" target="pmc_ext" ref="reftype=publisher&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Hypertension. : HYPERTENSIONAHA.117.09438. </span></a></div><span class="fm-vol-iss-date">Published online 2017 Jul 24. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1161%2FHYPERTENSIONAHA.117.09438" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1161/HYPERTENSIONAHA.117.09438</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5783787</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS902240</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28739976">28739976</a></div></div></div></div><h1 class="content-title">Novel Blood Pressure Locus and Gene Discovery Using Genome-Wide Association Study and Expression Data Sets From Blood and the Kidney</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Wain%20LV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Louise V. Wain</a>, <a href="/pubmed/?term=Vaez%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ahmad Vaez</a>, <a href="/pubmed/?term=Jansen%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Rick Jansen</a>, <a href="/pubmed/?term=Joehanes%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Roby Joehanes</a>, <a href="/pubmed/?term=van%20der%20Most%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Peter J. van der Most</a>, <a href="/pubmed/?term=Erzurumluoglu%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">A. Mesut Erzurumluoglu</a>, <a href="/pubmed/?term=O%26%23x02019%3BReilly%20PF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Paul F. O’Reilly</a>, <a href="/pubmed/?term=Cabrera%20CP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Claudia P. Cabrera</a>, <a href="/pubmed/?term=Warren%20HR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Helen R. Warren</a>, <a href="/pubmed/?term=Rose%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Lynda M. Rose</a>, <a href="/pubmed/?term=Verwoert%20GC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Germaine C. Verwoert</a>, <a href="/pubmed/?term=Hottenga%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jouke-Jan Hottenga</a>, <a href="/pubmed/?term=Strawbridge%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Rona J. Strawbridge</a>, <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Tonu Esko</a>, <a href="/pubmed/?term=Arking%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Dan E. Arking</a>, <a href="/pubmed/?term=Hwang%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Shih-Jen Hwang</a>, <a href="/pubmed/?term=Guo%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Xiuqing Guo</a>, <a href="/pubmed/?term=Kutalik%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Zoltan Kutalik</a>, <a href="/pubmed/?term=Trompet%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Stella Trompet</a>, <a href="/pubmed/?term=Shrine%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Nick Shrine</a>, <a href="/pubmed/?term=Teumer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Alexander Teumer</a>, <a href="/pubmed/?term=Ried%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Janina S. Ried</a>, <a href="/pubmed/?term=Bis%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Joshua C. Bis</a>, <a href="/pubmed/?term=Smith%20AV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Albert V. Smith</a>, <a href="/pubmed/?term=Amin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Najaf Amin</a>, <a href="/pubmed/?term=Nolte%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ilja M. Nolte</a>, <a href="/pubmed/?term=Lyytik%26%23x000e4%3Binen%20LP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Leo-Pekka Lyytikäinen</a>, <a href="/pubmed/?term=Mahajan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Anubha Mahajan</a>, <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Nicholas J. Wareham</a>, <a href="/pubmed/?term=Hofer%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Edith Hofer</a>, <a href="/pubmed/?term=Joshi%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Peter K. Joshi</a>, <a href="/pubmed/?term=Kristiansson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Kati Kristiansson</a>, <a href="/pubmed/?term=Traglia%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Michela Traglia</a>, <a href="/pubmed/?term=Havulinna%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Aki S. Havulinna</a>, <a href="/pubmed/?term=Goel%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Anuj Goel</a>, <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Mike A. Nalls</a>, <a href="/pubmed/?term=S%26%23x000f5%3Bber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Siim Sõber</a>, <a href="/pubmed/?term=Vuckovic%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Dragana Vuckovic</a>, <a href="/pubmed/?term=Luan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jian’an Luan</a>, <a href="/pubmed/?term=Fabiola%20Del%20Greco%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">M. Fabiola Del Greco</a>, <a href="/pubmed/?term=Ayers%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Kristin L. Ayers</a>, <a href="/pubmed/?term=Marrugat%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jaume Marrugat</a>, <a href="/pubmed/?term=Ruggiero%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Daniela Ruggiero</a>, <a href="/pubmed/?term=Lopez%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Lorna M. Lopez</a>, <a href="/pubmed/?term=Niiranen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Teemu Niiranen</a>, <a href="/pubmed/?term=Enroth%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Stefan Enroth</a>, <a href="/pubmed/?term=Jackson%20AU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Anne U. Jackson</a>, <a href="/pubmed/?term=Nelson%20CP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Christopher P. Nelson</a>, <a href="/pubmed/?term=Huffman%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jennifer E. Huffman</a>, <a href="/pubmed/?term=Zhang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Weihua Zhang</a>, <a href="/pubmed/?term=Marten%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jonathan Marten</a>, <a href="/pubmed/?term=Gandin%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ilaria Gandin</a>, <a href="/pubmed/?term=Harris%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Sarah E. Harris</a>, <a href="/pubmed/?term=Zemunik%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Tatijana Zemunik</a>, <a href="/pubmed/?term=Lu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Yingchang Lu</a>, <a href="/pubmed/?term=Evangelou%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Evangelos Evangelou</a>, <a href="/pubmed/?term=Shah%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Nabi Shah</a>, <a href="/pubmed/?term=de%20Borst%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Martin H. de Borst</a>, <a href="/pubmed/?term=Mangino%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Massimo Mangino</a>, <a href="/pubmed/?term=Prins%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Bram P. Prins</a>, <a href="/pubmed/?term=Campbell%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Archie Campbell</a>, <a href="/pubmed/?term=Li-Gao%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ruifang Li-Gao</a>, <a href="/pubmed/?term=Chauhan%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ganesh Chauhan</a>, <a href="/pubmed/?term=Oldmeadow%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Christopher Oldmeadow</a>, <a href="/pubmed/?term=Abecasis%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Gonçalo Abecasis</a>, <a href="/pubmed/?term=Abedi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Maryam Abedi</a>, <a href="/pubmed/?term=Barbieri%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Caterina M. Barbieri</a>, <a href="/pubmed/?term=Barnes%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Michael R. Barnes</a>, <a href="/pubmed/?term=Batini%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Chiara Batini</a>, <a href="/pubmed/?term=Beilby%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">John Beilby</a>, BIOS Consortium,<sup>*</sup> <a href="/pubmed/?term=Blake%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Tineka Blake</a>, <a href="/pubmed/?term=Boehnke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Michael Boehnke</a>, <a href="/pubmed/?term=Bottinger%20EP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Erwin P. Bottinger</a>, <a href="/pubmed/?term=Braund%20PS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Peter S. Braund</a>, <a href="/pubmed/?term=Brown%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Morris Brown</a>, <a href="/pubmed/?term=Brumat%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Marco Brumat</a>, <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Harry Campbell</a>, <a href="/pubmed/?term=Chambers%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">John C. Chambers</a>, <a href="/pubmed/?term=Cocca%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Massimiliano Cocca</a>, <a href="/pubmed/?term=Collins%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Francis Collins</a>, <a href="/pubmed/?term=Connell%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">John Connell</a>, <a href="/pubmed/?term=Cordell%20HJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Heather J. Cordell</a>, <a href="/pubmed/?term=Damman%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jeffrey J. Damman</a>, <a href="/pubmed/?term=Davies%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Gail Davies</a>, <a href="/pubmed/?term=de%20Geus%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Eco J. de Geus</a>, <a href="/pubmed/?term=de%20Mutsert%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Renée de Mutsert</a>, <a href="/pubmed/?term=Deelen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Joris Deelen</a>, <a href="/pubmed/?term=Demirkale%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Yusuf Demirkale</a>, <a href="/pubmed/?term=Doney%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Alex S.F. Doney</a>, <a href="/pubmed/?term=D%26%23x000f6%3Brr%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Marcus Dörr</a>, <a href="/pubmed/?term=Farrall%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Martin Farrall</a>, <a href="/pubmed/?term=Ferreira%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Teresa Ferreira</a>, <a href="/pubmed/?term=Fr%26%23x000e5%3Bnberg%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Mattias Frånberg</a>, <a href="/pubmed/?term=Gao%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">He Gao</a>, <a href="/pubmed/?term=Giedraitis%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Vilmantas Giedraitis</a>, <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Christian Gieger</a>, <a href="/pubmed/?term=Giulianini%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Franco Giulianini</a>, <a href="/pubmed/?term=Gow%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Alan J. Gow</a>, <a href="/pubmed/?term=Hamsten%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Anders Hamsten</a>, <a href="/pubmed/?term=Harris%20TB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Tamara B. Harris</a>, <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Albert Hofman</a>, <a href="/pubmed/?term=Holliday%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Elizabeth G. Holliday</a>, <a href="/pubmed/?term=Hui%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jennie Hui</a>, <a href="/pubmed/?term=Jarvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Marjo-Riitta Jarvelin</a>, <a href="/pubmed/?term=Johansson%20%26%23x000c5%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Åsa Johansson</a>, <a href="/pubmed/?term=Johnson%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Andrew D. Johnson</a>, <a href="/pubmed/?term=Jousilahti%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Pekka Jousilahti</a>, <a href="/pubmed/?term=Jula%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Antti Jula</a>, <a href="/pubmed/?term=K%26%23x000e4%3Bh%26%23x000f6%3Bnen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Mika Kähönen</a>, <a href="/pubmed/?term=Kathiresan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Sekar Kathiresan</a>, <a href="/pubmed/?term=Khaw%20KT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Kay-Tee Khaw</a>, <a href="/pubmed/?term=Kolcic%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ivana Kolcic</a>, <a href="/pubmed/?term=Koskinen%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Seppo Koskinen</a>, <a href="/pubmed/?term=Langenberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Claudia Langenberg</a>, <a href="/pubmed/?term=Larson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Marty Larson</a>, <a href="/pubmed/?term=Launer%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Lenore J. Launer</a>, <a href="/pubmed/?term=Lehne%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Benjamin Lehne</a>, <a href="/pubmed/?term=Liewald%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">David C.M. Liewald</a>, Lifelines Cohort Study,<sup>*</sup> <a href="/pubmed/?term=Lin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Li Lin</a>, <a href="/pubmed/?term=Lind%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Lars Lind</a>, <a href="/pubmed/?term=Mach%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">François Mach</a>, <a href="/pubmed/?term=Mamasoula%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Chrysovalanto Mamasoula</a>, <a href="/pubmed/?term=Menni%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Cristina Menni</a>, <a href="/pubmed/?term=Mifsud%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Borbala Mifsud</a>, <a href="/pubmed/?term=Milaneschi%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Yuri Milaneschi</a>, <a href="/pubmed/?term=Morgan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Anna Morgan</a>, <a href="/pubmed/?term=Morris%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Andrew D. Morris</a>, <a href="/pubmed/?term=Morrison%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Alanna C. Morrison</a>, <a href="/pubmed/?term=Munson%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Peter J. Munson</a>, <a href="/pubmed/?term=Nandakumar%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Priyanka Nandakumar</a>, <a href="/pubmed/?term=Nguyen%20QT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Quang Tri Nguyen</a>, <a href="/pubmed/?term=Nutile%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Teresa Nutile</a>, <a href="/pubmed/?term=Oldehinkel%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Albertine J. Oldehinkel</a>, <a href="/pubmed/?term=Oostra%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ben A. Oostra</a>, <a href="/pubmed/?term=Org%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Elin Org</a>, <a href="/pubmed/?term=Padmanabhan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Sandosh Padmanabhan</a>, <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Aarno Palotie</a>, <a href="/pubmed/?term=Par%26%23x000e9%3B%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Guillaume Paré</a>, <a href="/pubmed/?term=Pattie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Alison Pattie</a>, <a href="/pubmed/?term=Penninx%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Brenda W.J.H. Penninx</a>, <a href="/pubmed/?term=Poulter%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Neil Poulter</a>, <a href="/pubmed/?term=Pramstaller%20PP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Peter P. Pramstaller</a>, <a href="/pubmed/?term=Raitakari%20OT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Olli T. Raitakari</a>, <a href="/pubmed/?term=Ren%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Meixia Ren</a>, <a href="/pubmed/?term=Rice%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Kenneth Rice</a>, <a href="/pubmed/?term=Ridker%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Paul M. Ridker</a>, <a href="/pubmed/?term=Riese%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Harriëtte Riese</a>, <a href="/pubmed/?term=Ripatti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Samuli Ripatti</a>, <a href="/pubmed/?term=Robino%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Antonietta Robino</a>, <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jerome I. Rotter</a>, <a href="/pubmed/?term=Rudan%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Igor Rudan</a>, <a href="/pubmed/?term=Saba%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Yasaman Saba</a>, <a href="/pubmed/?term=Pierre%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Aude Saint Pierre</a>, <a href="/pubmed/?term=Sala%20CF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Cinzia F. Sala</a>, <a href="/pubmed/?term=Sarin%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Antti-Pekka Sarin</a>, <a href="/pubmed/?term=Schmidt%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Reinhold Schmidt</a>, <a href="/pubmed/?term=Scott%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Rodney Scott</a>, <a href="/pubmed/?term=Seelen%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Marc A. Seelen</a>, <a href="/pubmed/?term=Shields%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Denis C. Shields</a>, <a href="/pubmed/?term=Siscovick%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">David Siscovick</a>, <a href="/pubmed/?term=Sorice%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Rossella Sorice</a>, <a href="/pubmed/?term=Stanton%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Alice Stanton</a>, <a href="/pubmed/?term=Stott%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">David J. Stott</a>, <a href="/pubmed/?term=Sundstr%26%23x000f6%3Bm%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Johan Sundström</a>, <a href="/pubmed/?term=Swertz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Morris Swertz</a>, <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Kent D. Taylor</a>, <a href="/pubmed/?term=Thom%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Simon Thom</a>, <a href="/pubmed/?term=Tzoulaki%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ioanna Tzoulaki</a>, <a href="/pubmed/?term=Tzourio%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Christophe Tzourio</a>, <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">André G. Uitterlinden</a>, Understanding Society Scientific Group,<sup>*</sup> <a href="/pubmed/?term=V%26%23x000f6%3Blker%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Uwe Völker</a>, <a href="/pubmed/?term=Vollenweider%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Peter Vollenweider</a>, <a href="/pubmed/?term=Wild%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Sarah Wild</a>, <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Gonneke Willemsen</a>, <a href="/pubmed/?term=Wright%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Alan F. Wright</a>, <a href="/pubmed/?term=Yao%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jie Yao</a>, <a href="/pubmed/?term=Th%26%23x000e9%3Briault%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Sébastien Thériault</a>, <a href="/pubmed/?term=Conen%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">David Conen</a>, <a href="/pubmed/?term=Attia%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">John Attia</a>, <a href="/pubmed/?term=Sever%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Peter Sever</a>, <a href="/pubmed/?term=Debette%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Stéphanie Debette</a>, <a href="/pubmed/?term=Mook-Kanamori%20DO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Dennis O. Mook-Kanamori</a>, <a href="/pubmed/?term=Zeggini%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Eleftheria Zeggini</a>, <a href="/pubmed/?term=Spector%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Tim D. Spector</a>, <a href="/pubmed/?term=van%20der%20Harst%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Pim van der Harst</a>, <a href="/pubmed/?term=Palmer%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Colin N.A. Palmer</a>, <a href="/pubmed/?term=Vergnaud%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Anne-Claire Vergnaud</a>, <a href="/pubmed/?term=Loos%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ruth J.F. Loos</a>, <a href="/pubmed/?term=Polasek%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ozren Polasek</a>, <a href="/pubmed/?term=Starr%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">John M. Starr</a>, <a href="/pubmed/?term=Girotto%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Giorgia Girotto</a>, <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Caroline Hayward</a>, <a href="/pubmed/?term=Kooner%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jaspal S. Kooner</a>, <a href="/pubmed/?term=Lindgren%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Cecila M. Lindgren</a>, <a href="/pubmed/?term=Vitart%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Veronique Vitart</a>, <a href="/pubmed/?term=Samani%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Nilesh J. Samani</a>, <a href="/pubmed/?term=Tuomilehto%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jaakko Tuomilehto</a>, <a href="/pubmed/?term=Gyllensten%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ulf Gyllensten</a>, <a href="/pubmed/?term=Knekt%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Paul Knekt</a>, <a href="/pubmed/?term=Deary%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Ian J. Deary</a>, <a href="/pubmed/?term=Ciullo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Marina Ciullo</a>, <a href="/pubmed/?term=Elosua%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Roberto Elosua</a>, <a href="/pubmed/?term=Keavney%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Bernard D. Keavney</a>, <a href="/pubmed/?term=Hicks%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Andrew A. Hicks</a>, <a href="/pubmed/?term=Scott%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Robert A. Scott</a>, <a href="/pubmed/?term=Gasparini%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Paolo Gasparini</a>, <a href="/pubmed/?term=Laan%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Maris Laan</a>, <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">YongMei Liu</a>, <a href="/pubmed/?term=Watkins%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Hugh Watkins</a>, <a href="/pubmed/?term=Hartman%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Catharina A. Hartman</a>, <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Veikko Salomaa</a>, <a href="/pubmed/?term=Toniolo%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Daniela Toniolo</a>, <a href="/pubmed/?term=Perola%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Markus Perola</a>, <a href="/pubmed/?term=Wilson%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">James F. Wilson</a>, <a href="/pubmed/?term=Schmidt%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Helena Schmidt</a>, <a href="/pubmed/?term=Zhao%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Jing Hua Zhao</a>, <a href="/pubmed/?term=Lehtim%26%23x000e4%3Bki%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Terho Lehtimäki</a>, <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Cornelia M. van Duijn</a>, <a href="/pubmed/?term=Gudnason%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Vilmundur Gudnason</a>, <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Bruce M. Psaty</a>, <a href="/pubmed/?term=Peters%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Annette Peters</a>, <a href="/pubmed/?term=Rettig%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Rainer Rettig</a>, <a href="/pubmed/?term=James%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Alan James</a>, <a href="/pubmed/?term=Jukema%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">J. Wouter Jukema</a>, <a href="/pubmed/?term=Strachan%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">David P. Strachan</a>, <a href="/pubmed/?term=Palmas%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Walter Palmas</a>, <a href="/pubmed/?term=Metspalu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Andres Metspalu</a>, <a href="/pubmed/?term=Ingelsson%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Erik Ingelsson</a>, <a href="/pubmed/?term=Boomsma%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Dorret I. Boomsma</a>, <a href="/pubmed/?term=Franco%20OH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Oscar H. Franco</a>, <a href="/pubmed/?term=Bochud%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Murielle Bochud</a>, <a href="/pubmed/?term=Newton-Cheh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Christopher Newton-Cheh</a>, <a href="/pubmed/?term=Munroe%20PB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Patricia B. Munroe</a>, <a href="/pubmed/?term=Elliott%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Paul Elliott</a>, <a href="/pubmed/?term=Chasman%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Daniel I. Chasman</a>, <a href="/pubmed/?term=Chakravarti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Aravinda Chakravarti</a>, <a href="/pubmed/?term=Knight%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Joanne Knight</a>, <a href="/pubmed/?term=Morris%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Andrew P. Morris</a>, <a href="/pubmed/?term=Levy%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Daniel Levy</a>, <a href="/pubmed/?term=Tobin%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Martin D. Tobin</a>, <a href="/pubmed/?term=Snieder%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Harold Snieder</a>,<sup>†</sup> <a href="/pubmed/?term=Caulfield%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Mark J. Caulfield</a>,<sup>†</sup> and  <a href="/pubmed/?term=Ehret%20GB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28739976">Georg B. Ehret</a><sup>†</sup></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140304064594544_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140304064594544_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140304064594544_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">Department of Health Sciences (L.V.W., A.M.E., N. Shrine, C.B., T.B., M.D.T.), and Department of Cardiovascular Sciences and NIHR Leicester Biomedical Research Centre (C.P.N., P.S.B., N.J.S.), University of Leicester, United Kingdom; Department of Epidemiology (A.V., P.J.v.d.M., I.M.N., H. Snieder), Division of Nephrology, Department of Internal Medicine (M.H.d.B., M.A.S.), Interdisciplinary Center Psychopathology and Emotion Regulation (IPCE) (A.J.O., H.R., C.A.H.), Department of Genetics, (M.S.), and Department of Cardiology (P.v.d.H.), University of Groningen, University Medical Center Groningen, The Netherlands; Research Institute for Primordial Prevention of Non-Communicable Disease, Isfahan University of Medical Sciences, Iran (A.V.); Department of Psychiatry, VU University Medical Center, Neuroscience Campus Amsterdam, The Netherlands (R. Jansen); Hebrew SeniorLife, Harvard Medical School, Boston, MA (R. Joehanes); National Heart, Lung and Blood Institute’s Framingham Heart Study, MA (R. Joehanes, A.D.J., M. Larson); Institute of Psychiatry, Psychology and Neuroscience (P.F.O.), and Department of Twin Research and Genetic Epidemiology (M.M., C. Menni, T.D.S.), King’s College London, United Kingdom; Clinical Pharmacology, William Harvey Research Institute (C.P.C., H.R.W., M.R.B., M. Brown, B.M., M.R., P.B.M., M.J.C.) and NIHR Barts Cardiovascular Biomedical Research Unit (C.P.C., H.R.W., M.R.B., M. Brown, P.B.M., M.J.C.), Barts and The London School of Medicine and Dentistry, Queen Mary University of London, United Kingdom; Division of Preventive Medicine, Brigham and Women’s Hospital, Boston, MA (L.M.R., F.G., P.M.R., D.I.C.); Department of Epidemiology (G.C.V., A. Hofman, A.G.U., O.H.F.), Genetic Epidemiology Unit, Department of Epidemiology (N.A., B.A.O., C.M.v.D.), and Department of Internal Medicine (A.G.U.), Erasmus MC, Rotterdam, The Netherlands; Department of Biological Psychology, Vrije Universiteit, Amsterdam, EMGO+ Institute, VU University Medical Center, The Netherlands (J.-J.H., E.J.d.G., G.W., D.I.B.); Cardiovascular Medicine Unit, Department of Medicine Solna, Karolinska Institutet, Stockholm, Sweden (R.J.S., M. Frånberg, A. Hamsten); Centre for Molecular Medicine, Karolinska Universitetsjukhuset, Solna, Sweden (R.J.S., M. Frånberg, A. Hamsten); Estonian Genome Center (T.E., E.O., A. Metspalu), Institute of Biomedicine and Translational Medicine (S.S., M. Laan), and Estonian Genome Center (M.P.), University of Tartu, Estonia; Divisions of Endocrinology/Children’s Hospital, Boston, MA (T.E.); Broad Institute of Harvard and MIT, Cambridge, MA (T.E., C.M.L., C.N.-C.); Center for Complex Disease Genomics, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD (D.E.A., P.N., A. Chakravarti, G.B.E.); The Population Science Branch, Division of Intramural Research, National Heart Lung and Blood Institute (S.-J.H., D.L.), Laboratory of Neurogenetics, National Institute on Aging (M.A.N.), Medical Genomics and Metabolic Genetics Branch, National Human Genome Research Institute (F.C.), and Center for Information Technology (Y.D., P.J.M., Q.T.N.), National Institutes of Health, Bethesda, MD; The Framingham Heart Study, Framingham, MA (S.-J.H., D.L.); The Institute for Translational Genomics and Population Sciences, Department of Pediatrics (X.G., J.Y.), and The Institute for Translational Genomics and Population Sciences, Departments of Pediatrics and Medicine (J.I.R.), LABioMed at Harbor-UCLA Medical Center, Torrance, CA; Institute of Social and Preventive Medicine, Lausanne University Hospital, Lausanne, Switzerland (Z.K., M. Bochud); Swiss Institute of Bioinformatics, Lausanne, Switzerland (Z.K.); Department of Cardiology (S. Trompet, J.W.J.) Department of Gerontology and Geriatrics (S. Trompet), Department of Clinical Epidemiology (R.L.-G., R.d.M., D.O.M.-K.), Department of Molecular Epidemiology (J.D.), and Department of Public Health and Primary Care (D.O.M.-K.), Leiden University Medical Center, The Netherlands; Institute for Community Medicine (A.T.), Department of Internal Medicine B (M.D.), and Interfaculty Institute for Genetics and Functional Genomics (U.V.), University Medicine Greifswald, Germany; DZHK (German Centre for Cardiovascular Research), partner site Greifswald, Germany (A.T., M.D., U.V.); Institute of Epidemiology II, Helmholtz Zentrum München, Neuherberg, Germany (J.S.R., A. Peters); Cardiovascular Health Research Unit, Department of Medicine (J.C.B., B.M.P.) and Departments of Biostatistics (K.R.), Epidemiology (B.M.P.), and Health Services (B.M.P.), University of Washington, Seattle; Icelandic Heart Association, Kopavogur, Iceland (A.V.S., V. Gudnason); Faculty of Medicine, University of Iceland, Reykjavik, Iceland (A.V.S., V. Gudnason); Department of Clinical Chemistry, Fimlab Laboratories, Tampere, Finland (L.-P.L., T.L.); Department of Clinical Chemistry, Faculty of Medicine and Life Sciences, University of Tampere, Finland (L.-P.L., T.L.); Wellcome Trust Centre for Human Genetics (A. Mahajan, A.G., M. Farrall, T.F., C.M.L., H.W., A.P.M.), and Division of Cardiovascular Medicine, Radcliffe Department of Medicine (A.G., M. Farrall, H.W.), University of Oxford, United Kingdom; MRC Epidemiology Unit, University of Cambridge School of Clinical Medicine, Institute of Metabolic Science, United Kingdom (N.J.W., J.L., C.L., R.J.F.L., R.A.S., J.H.Z.); Clinical Division of Neurogeriatrics, Department of Neurology (E.H., R. Schmidt), Institute of Medical Informatics, Statistics and Documentation (E.H.), and Department of Neurology (H. Schmidt), Medical University Graz, Austria; Centre for Global Health Research, Usher Institute of Population Health Sciences and Informatics (P.K.J., H.C., I.R., S.W., J.F.W.), Centre for Cognitive Ageing and Cognitive Epidemiology (L.M.L., S.E.H., G.D., A.J.G., D.C.M.L., J.M.S., I.J.D.), Medical Genetics Section, Centre for Genomic and Experimental Medicine, Institute of Genetics and Molecular Medicine (A. Campbell), Generation Scotland, Centre for Genomic and Experimental Medicine (A. Campbell, S.P., C.H.), Department of Psychology (G.D., D.C.M.L., A. Pattie, I.J.D.), Alzheimer Scotland Dementia Research Centre (J.M.S.), and Medical Research Council Human Genetics Unit, Institute of Genetics and Molecular Medicine (C.H.), University of Edinburgh, Scotland, United Kingdom; Department of Health (K.K., A.S.H., T. Niiranen, P.J., A.J., S. Koskinen, P.K., V.S., M.P.), and Chronic Disease Prevention Unit (J.T.), National Institute for Health and Welfare (THL), Helsinki, Finland; Division of Genetics and Cell Biology, San Raffaele Scientific Institute, Milano, Italy (M.T., C.M.B., C.F.S., D.T.); Data Tecnica International, Glen Echo, MD (M.A.N.); Medical Genetics, IRCCS-Burlo Garofolo Children Hospital, Trieste, Italy (D.V., G.G., P.G.); Department of Medical, Surgical and Health Sciences, University of Trieste, Italy (D.V., I.G., M. Brumat, M. Cocca, A. Morgan, G.G., P.G.); Institute for Biomedicine, Eurac Research, Affiliated Institute of the University of Lübeck, Bolzano, Italy (F.D.G.M., P.P.P., A.S.P., A.A.H.); Department of Genetics and Genomic Sciences (K.L.A.), The Charles Bronfman Institute for Personalized Medicine (Y.L., E.P.B., R.J.F.L.), and Mindich Child health Development Institute (R.J.F.L.), Icahn School of Medicine at Mount Sinai, New York; Cardiovascular Epidemiology and Genetics, IMIM, and CIBERCV, Barcelona, Spain (J. Marrugat, R.E.); Institute of Genetics and Biophysics A. Buzzati-Traverso, CNR, Napoli, Italy (D.R., T. Nutile, R. Sorice, M. Ciullo); Department of Psychiatry, Royal College of Surgeons in Ireland, Education and Research Centre, Beaumont Hospital, Dublin (L.M.L.); UCD Conway Institute, Centre for Proteome Research (L.M.L.), and School of Medicine, Conway Institute (D.C.S.), University College Dublin, Belfield, Ireland; Department of Immunology, Genetics and Pathology, Uppsala Universitet, Science for Life Laboratory, Sweden (S.E., Å. Johansson, U.G.); Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor (A.U.J., M. Boehnke); NIHR Leicester Cardiovascular Biomedical Research Unit, Glenfield Hospital, Leicester United Kingdom (C.P.N., P.S.B., N.J.S.); MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine (J.E.H., V.V., J. Marten, A.F.W., J.F.W.), and Medical Genetics Section, Centre for Genomic and Experimental Medicine and MRC Institute of Genetics and Molecular Medicine (S.E.H.), University of Edinburgh, Western General Hospital, Scotland, United Kingdom; Department of Epidemiology and Biostatistics, School of Public Health (W.Z., E.E., J.C.C., H.G., B.L., I.T., A.-C.V.), MRC-PHE Centre for Environment and Health, Department of Epidemiology and Biostatistics, School of Public Health (M.-R.J., P.E.), School of Public Health (N.P.), International Centre for Circulatory Health (S. Thom), and National Heart and Lung Institute (P.S.), Imperial College London, United Kingdom; Department of Cardiology, Ealing Hospital, London North West Healthcare NHS Trust, Southall, United Kingdom (W.Z., J.C.C., J.S.K.); Department of Medical Biology, Faculty of Medicine, University of Split, Croatia (T.Z.); Department of Hygiene and Epidemiology, University of Ioannina Medical School, Greece (E.E.); Medical Research Institute, University of Dundee, Ninewells Hospital and Medical School, Scotland, United Kingdom (N. Shah, A.S.F.D., C.N.A.P.); Department of Pharmacy, COMSATS Institute of Information Technology, Abbottabad, Pakistan (N. Shah); National Institute for Health Research Biomedical Research Centre, London, United Kingdom (M.M.); Department of Human Genetics, Wellcome Trust Sanger Institute, United Kingdom (B.P.P., E.Z.); INSERM U 1219, Bordeaux Population Health Center, France (G.C., C.T., S.D.); Bordeaux University, France (G.C., C.T., S.D.); Hunter Medical Research Institute, New Lambton, NSW, Australia (C.O., E.G.H., R. Scott, J.A.); Center for Statistical Genetics, Department of Biostatistics, Ann Arbor, MI (G.A.); Department of Genetics and Molecular Biology, Isfahan University of Medical Sciences, Iran (M.A.); Busselton Population Medical Research Institute, Western Australia (J.B., J.H.); PathWest Laboratory Medicine of Western Australia, Nedlands (J.B., J.H.); School of Pathology and Laboratory Medicine (J.B., J.H.), School of Population and Global Health (J.H.), and School of Medicine and Pharmacology (A. James), The University of Western Australia, Nedlands; Imperial College Healthcare NHS Trust, London, United Kingdom (J.C.C., J.S.K.); University of Dundee, Ninewells Hospital &amp; Medical School, United Kingdom (J.C.); Institute of Genetic Medicine (H.J.C.), and Institute of Health and Society (C. Mamasoula), Newcastle University, Newcastle upon Tyne, United Kingdom; Department of Pathology, Amsterdam Medical Center, The Netherlands (J.J.D.); Department of Numerical Analysis and Computer Science, Stockholm University, Sweden (M. Frånberg); Department of Public Health and Caring Sciences, Geriatrics, Uppsala, Sweden (V. Giedraitis); Helmholtz Zentrum Muenchen, Deutsches Forschungszentrum fuer Gesundheit und Umwelt (GmbH), Neuherberg, Germany (C.G.); Department of Psychology, School of Social Sciences, Heriot-Watt University, Edinburgh, United Kingdom (A.J.G.); Intramural Research Program, Laboratory of Epidemiology, Demography, and Biometry, National Institute on Aging (T.B.H., L.J.L.); Department of Epidemiology, Harvard T.H. Chan School of Public Health, Boston, MA (A. Hofman); Center For Life-Course Health Research (M.-R.J.), and Biocenter Oulu (M.-R.J.), University of Oulu, Finland; Unit of Primary Care, Oulu University Hospital, Finland (M.-R.J.); National Heart, Lung and Blood Institute, Cardiovascular Epidemiology and Human Genomics Branch, Bethesda, MD (A.D.J.); Department of Clinical Physiology, Tampere University Hospital, Finland (M.K.); Department of Clinical Physiology, Faculty of Medicine and Life Sciences, University of Tampere, Finland (M.K.); Cardiovascular Research Center (S. Kathiresan, C.N.-C.); Center for Human Genetics (S. Kathiresan), and Center for Human Genetic Research (C.N.-C.), Massachusetts General Hospital, Boston; Program in Medical and Population Genetics, Broad Institute, Cambridge, MA (S. Kathiresan, C.N.-C.); Department of Public Health and Primary Care, Institute of Public Health, University of Cambridge, United Kingdom (K.-T.K.); Department of Public Health, Faculty of Medicine, University of Split, Croatia (I.K., O.P.); Cardiology, Department of Specialties of Medicine, Geneva University Hospital, Switzerland (L. Lin, F.M., G.B.E.); Department of Medical Sciences, Cardiovascular Epidemiology (L. Lind, J.S.), and Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory (E.I.), Uppsala University, Sweden; Department of Psychiatry, EMGO Institute for Health and Care Research, VU University Medical Center, Amsterdam, The Netherlands (Y.M., B.W.J.H.P.); School of Molecular, Genetic and Population Health Sciences, University of Edinburgh, Medical School, Teviot Place, Scotland, United Kingdom (A.D.M.); Department of Epidemiology, Human Genetics and Environmental Sciences, School of Public Health, University of Texas Health Science Center at Houston (A.C.M.); British Heart Foundation Glasgow Cardiovascular Research Centre, Institute of Cardiovascular and Medical Sciences, College of Medical, Veterinary and Life Sciences (S.P.), and Institute of Cardiovascular and Medical Sciences, Faculty of Medicine (D.J.S.), University of Glasgow, United Kingdom; Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Finland (A. Palotie, S.R., A.-P.S., M.P.); Department of Pathology and Molecular Medicine, McMaster University, Hamilton, Canada (G.P., S. Thériault); Department of Neurology, General Central Hospital, Bolzano, Italy (P.P.P.); Department of Neurology, University of Lübeck, Germany (P.P.P.); Department of Clinical Physiology and Nuclear Medicine, Turku University Hospital, Finland (O.T.R.); Research Centre of Applied and Preventive Cardiovascular Medicine, University of Turku, Finland (O.T.R.); Department of Cardiology, Fujian Provincial Hospital, Fujian Medical University, Fuzhou, China (M.R.); Harvard Medical School, Boston, MA (P.M.R., D.I.C.); Institute for Maternal and Child Health IRCCS Burlo Garofolo, Trieste, Italy (A.R.); Institute of Molecular Biology and Biochemistry, Centre for Molecular Medicine, Medical University of Graz, Austria (Y.S., H. Schmidt); INSERM U1078, Etablissement Français du Sang, Brest Cedex, France (A.S.P.); Faculty of Health, University of Newcastle, Callaghan, NSW, Australia (R. Scott, J.A.); John Hunter Hospital, New Lambton, NSW, Australia (R. Scott, J.A.); The New York Academy of Medicine, New York (D.S.); IRCCS Neuromed, Pozzilli, Isernia, Italy (R. Sorice, M. Ciullo); Molecular and Cellular Therapeutics, Royal College of Surgeons in Ireland, Dublin 2, Ireland (A.S.); Institute for Translational Genomics and Population Sciences, Los Angeles BioMedical Research Institute at Harbor-UCLA Medical Center, Torrance, CA (K.D.T.); Division of Genetic Outcomes, Department of Pediatrics, Harbor-UCLA Medical Center, Torrance, CA (K.D.T.); Department of Public Health (C.T.), and Department of Neurology (S.D.), Bordeaux University Hospital, France; Department of Internal Medicine, Lausanne University Hospital, CHUV, Switzerland (P.V.); Population Health Research Institute, McMaster University, Hamilton Ontario, Canada (D.C.); National Heart and Lung Institute, Imperial College London, Hammersmith Hospital Campus, United Kingdom (J.S.K.); Dasman Diabetes Institute, Kuwait (J.T.); Diabetes Research Group, King Abdulaziz University, Jeddah, Saudi Arabia (J.T.); Department of Neurosciences and Preventive Medicine, Danube-University Krems, Austria (J.T.); Division of Cardiovascular Sciences, The University of Manchester and Central Manchester University Hospitals NHS Foundation Trust, United Kingdom (B.D.K.); Division of Public Health Sciences, Wake Forest School of Medicine, Winston-Salem (Y.M.L.); Kaiser Permanent Washington Health Research Institute, Seattle, WA (B.M.P.); Institute of Physiology, University Medicine Greifswald, Karlsburg, Germany (R.R); Department of Pulmonary Physiology and Sleep, Sir Charles Gairdner Hospital, Nedlands, Western Australia (A. James); Population Health Research Institute, St George’s, University of London, United Kingdom (D.P.S.); Department of Medicine, Columbia University Medical Center, New York (W.P.); Division of Cardiovascular Medicine, Department of Medicine, Stanford University School of Medicine, CA (E.I.); Data Science Institute and Lancaster Medical School, Lancaster University, United Kingdom (J.K.); and Department of Biostatistics, University of Liverpool, United Kingdom (A.P.M.).</div><div id="FN1">Correspondence to Georg B. Ehret, Cardiology, Department of Specialties of Medicine, Geneva University Hospital, 1205 Genève, Switzerland, <a href="mailto:dev@null" data-email="hc.enohr@groeg" class="oemail">hc.enohr@groeg</a> or Louise V. Wain, Department of Health Sciences, University of Leicester, Leicester LE1 7RH, United Kingdom, <a href="mailto:dev@null" data-email="ku.ca.el@niawesiuol" class="oemail">ku.ca.el@niawesiuol</a></div><div id="FN2"><sup>*</sup>A list of contributing authors is given in the <a href="#SD1" rid="SD1" class=" supplementary-material">online-only Data Supplement</a>.</div><div id="FN3"><sup>†</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140304064594544_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div class="license half_rhythm"><strong>Reprints:</strong> Information about reprints can be found online at: <a href="http://www.lww.com/reprints" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://www.lww.com/reprints</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's version of this article, before final editing, is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28739976" target="pmc_ext" ref="reftype=publisher&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Hypertension</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5783787/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC5783787/bin/NIHMS902240-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">NIHMS902240-supplement-supplement_1.pdf</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID:&nbsp;7AF31FA1-8888-4072-B3CE-8F27D220FFC8</div></div></dd></dl></div><div id="idm140304100444544" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140304100444544title">Abstract</h2><!--article-meta--><div><p id="P3" class="p p-first-last">Elevated blood pressure is a major risk factor for cardiovascular disease and has a substantial genetic contribution. Genetic variation influencing blood pressure has the potential to identify new pharmacological targets for the treatment of hypertension. To discover additional novel blood pressure loci, we used 1000 Genomes Project–based imputation in 150 134 European ancestry individuals and sought significant evidence for independent replication in a further 228 245 individuals. We report 6 new signals of association in or near <em>HSPB7, TNXB, LRP12, LOC283335, SEPT9</em>, and <em>AKT2</em>, and provide new replication evidence for a further 2 signals in <em>EBF2</em> and <em>NFKBIA</em>. Combining large whole-blood gene expression resources totaling 12 607 individuals, we investigated all novel and previously reported signals and identified 48 genes with evidence for involvement in blood pressure regulation that are significant in multiple resources. Three novel kidney-specific signals were also detected. These robustly implicated genes may provide new leads for therapeutic innovation.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">blood pressure, cardiovascular risk, complex traits, eSNP, GWAS, hypertension</span></div></div><div id="idm140304063842720" class="tsec sec headless whole_rhythm"><p id="P4" class="p p-first">Genetic support for a drug target increases the likelihood of success in drug development,<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012015">1</a></sup> and there is clear unmet need for novel therapeutic strategies to treat individuals with hypertension.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012018">2</a></sup> Several large studies have described blood pressure (BP) variant identification by genome-wide and targeted association approaches.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011991">3</a>–<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716510437">19</a></sup> Clinically, the most predictive BP traits for cardiovascular risk are systolic BP (SBP) and diastolic BP (DBP), reflecting roughly the peak and trough of the BP curve, and pulse pressure, the difference between SBP and DBP,<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011978">20</a></sup> reflecting arterial stiffness. Using these 3 traits, we undertook a meta-analysis of 150 134 individuals from 54 genome-wide association studies (GWAS) of European ancestry with imputation based on the 1000 Genomes Project Phase 1. To minimize reporting of false-positive associations, we sought stringent evidence for significant independent replication in a further 228 245 individuals. We further followed up novel and previously reported association signals in multiple large gene expression databases and the largest kidney tissue gene expression resource currently available. Finally, we searched for enrichment of associated genes in biological pathways and gene sets and identified whether any of the genes were known drug targets or had tool molecules.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Materials and Methods</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Studies Stage 1</h3><p id="P5" class="p p-first-last">Results from 54 independent European-ancestry studies, totaling 150 134 individuals, were included in the stage 1 meta-analysis: AGES (n=3215), ARIC (n=9402), ASPS (n=828), B58C (n=6458), BHS (n=4492), CHS (n=3254), Cilento study (n=999), COLAUS (n=5404), COROGENE-CTRL (n=1878), CROATIA-Vis (n=945), CROATIA-Split (n=494), CROATIA-Korcula (n=867), EGCUT (n=6395), EGCUT2 (n=1844), EPIC (n=2100), ERF (n=2617), Fenland (n=1357), FHS (n=8096), FINRISK-ctrl (n=861), FINRISK CASE (n=839), FUSION (n=1045), GRAPHIC (n=1010), H2000-CTRL (n=1078), HealthABC (n=1661), HTO (n=1000), INGI-CARL (n=456), INGI-FVG (n=746), INGI-VB (n=1775), IPM (n=300), KORAS3 (n=1590), KORAS4 (n=3748), LBC1921 (n=376), LBC1936 (n=800), LOLIPOP-EW610 (n=927), MESA (n=2678), MICROS (n=1148), MIGEN (n=1214), NESDA (n=2336), NSPHS (n=1005), NTR (n=1490), PHASE (n=4535), PIVUS (n=945), PROCARDIS (n=1652), SHIP (n=4068), ULSAM (n=1114), WGHS (n=23 049), YFS (n=1987), ORCADES (n=1908), RS1 (n=5645), RS2 (n=2152), RS3 (n=3018), TRAILS (n=1262), TRAILS-CC (n=282), and TWINGENE (n=9789). Full study names and general study information is given in Table S1 in the <a href="#SD1" rid="SD1" class=" supplementary-material">online-only Data Supplement</a>.</p></div><div id="S3" class="sec"><h3 id="S3title">Study-Level Genotyping and Association Testing</h3><p id="P6" class="p p-first-last">Three quantitative BP traits were analyzed: SBP, DBP, and pulse pressure (difference between SBP and DBP). Within each study, individuals known to be taking antihypertensive medication had 15 mm Hg added to their raw SBP value and 10 mm Hg added to their raw DBP values.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012003">21</a></sup> A summary of BP phenotypes in each study is given in Table S2. Association testing was undertaken according to a central analysis plan that specified the use of sex, age, age<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011988">2</a></sup>, and body mass index as covariates and optional inclusion of additional covariates to account for population stratification (Table S3). Trait residuals were calculated for each trait using a normal linear regression of the medication-adjusted trait values (mm Hg) onto all covariates. The genotyping array, preimputation quality control filters, imputation software, and association testing software used by each study are listed in Table S4. Each participating study imputed genotypes based on the 1000 Genomes Project Phase 1 integrated release version 3 (March 2012) all ancestry reference panel.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012016">22</a></sup> Imputed genotype dosages were used to take into account uncertainty in the imputation. Association testing was performed using linear regression of the trait residuals onto genotype dosages under an additive genetic model. Methods to account for relatedness within a study were used where appropriate (Table S3). Results for all variants (single nucleotide polymorphisms [SNPs] and insertion/deletion polymorphisms [INDELs]) were then returned to the central analysis group for further quality control checks and meta-analysis.</p></div><div id="S4" class="sec"><h3 id="S4title">Stage 1 Meta-Analysis</h3><p id="P7" class="p p-first-last">Central quality control checks were undertaken across all results sets. This included checks to ensure allele frequency consistency (across studies and with reference populations), checks of effect size and standard error distributions (ie, to highlight phenotype issues), and generation of quantile–quantile plots and genomic inflation factor lambdas to check for over- or underinflation of test statistics. Genomic control was applied (if lambda &gt;1) at study level. Variants with imputation quality &lt;0.3 were excluded prior to meta-analysis. Inverse variance–weighted meta-analysis was undertaken. After meta-analysis, variants with a weighted minor allele frequency of &lt;1% or N effective (product of study sample size and imputation quality summed across contributing studies) &lt;60% were then excluded and meta-analysis genomic control lambda calculated and used to adjust the meta-analysis results.</p></div><div id="S5" class="sec"><h3 id="S5title">Selection of Regions for Follow-Up</h3><p id="P8" class="p p-first-last">For each trait, regions of association were selected by ranking variants by <em>P</em> value, recording the variant with the lowest <em>P</em> value as a sentinel variant and then excluding all variants ±500 kb from the sentinel and reranking the remaining variants. This was undertaken iteratively until all sentinel variants representing 1 Mb regions containing associations with <em>P</em>&lt;10<sup>−6</sup> had been identified. To identify additional signals represented by secondary sentinel variants within 500 kb of each of the sentinel variants, GCTA (the Genome-wide Complex Trait Analysis software)<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012013">23</a></sup> was used to run conditional analyses (conditioned on the first sentinel variant) on each of the 1 Mb regions using GWAS summary statistics and linkage disequilibrium (LD) information from ARIC. This was done both for putatively novel regions and for regions that had previously been reported. A χ<sup>2</sup> test of heterogeneity of effect sizes across the 54 studies was run for each sentinel variant, and those with <em>P</em>&lt;0.05 for heterogeneity were excluded from further follow-up. Variants with <em>P</em>&lt;10<sup>−6</sup> after conditioning on the sentinel SNP (novel or known) in the region and for which any attenuation of the −log10 <em>P</em> value was &lt;1.5 fold were also taken forward for replication.</p></div><div id="S6" class="sec"><h3 id="S6title">Studies Stage 2</h3><p id="P9" class="p p-first-last">Data from 14 independent studies, totaling 87 360 individuals, and the first release of UK Biobank, totaling 140 886 individuals, were combined to replicate the findings from stage 1 (ie, totaling 228 245 individuals). Stage 2 study details, including full study names, are given in Table S6 and included 3C-Dijon (n=4061), Airwave (n=14 023), ASCOT-SC (n=2462), ASCOT-UK (n=3803), BRIGHT (n=1791), GAPP (n=1685), GoDARTs (n=7413), GS:SFHS (n=9749), HCS (n=2112), JUPITER (n=8718), LifeLines (n=13 376), NEO (n=5731), TwinsUK (n=4973), UK Biobank-CMC (n=140 886), and UKHLS (n=7462). Analysis was undertaken using the same methods as described for stage 1 studies. UK Biobank-CMC used a newer imputation reference panel than the other studies, and where a requested variant was not available, a proxy was used (next most significant <em>P</em> value with LD <em>r</em><sup>2</sup>&gt;0.6 with original top variant). Results from all stage 2 studies were meta-analyzed using inverse variance–weighted meta-analysis. Two of the variants, rs1048238 and chr1:243458005:INDEL, were not available in the largest study in stage 2 (UK Biobank-CMC), and so proxy variants were selected (based on <em>P</em> value and LD).</p></div><div id="S7" class="sec"><h3 id="S7title">Stage 1+Stage 2 Meta-Analysis</h3><p id="P10" class="p p-first-last">After meta-analysis of stage 1 and stage 2 results, signals with a <em>P</em>&gt;5×10<sup>−8</sup> were excluded. Of the signals with a final <em>P</em>&lt;5×10<sup>−8</sup>, support for independent replication within the stage 2 studies only was sought. Any signals that had <em>P</em>&lt;5×10<sup>−8</sup> and evidence for independent replication in stage 2 alone indicated by <em>P</em>&lt;8.2×10<sup>−4</sup> (Bonferroni correction for 61 tests) were reported as novel signals of association with BP. Any signals that were subsequently reported by other BP GWAS that were accepted for publication during the time this analysis was ongoing, or signals for which independence from another known signal could not be established, were removed from our list of novel signals at this stage (Table S5).</p></div><div id="S8" class="sec"><h3 id="S8title">Genotype and Gene Expression</h3><p id="P11" class="p p-first-last">We searched for signals of association of genotype with gene expression for the 22 signals (including 8 novel) described in this study (Table S7) and all signals reported prior to our study (Table S10)<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011972">3</a>–<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011971">16</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011974">18</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012012">24</a></sup> in 3 whole-blood data sets, 1 kidney data set, and the GTEx (Genotype-Tissue Expression) multiple tissue data resource, which included whole blood.<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012006">25</a></sup> We selected cis signals of association, which were significant after controlling for 5% false discovery rate. The 3 whole-blood expression quantitative trait loci (eQTL) data sets were the National Heart, Lung, and Blood Institute SABRe (Systems Approach to Biomarker Research in Cardiovascular Disease) initiative whole-blood eQTL resource (microarray, n=5257), NESDA-NTR (microarray, n=4896), BIOS (RNAseq, n=2116). The whole-blood data from GTEx was based on data from 338 samples. The kidney data set comprised 236 donor kidney samples from 134 donors.<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011987">26</a></sup> Full details of each data set can be found in the <a href="#SD1" rid="SD1" class=" supplementary-material">online-only Data Supplement</a>. The source transcriptomic renal data as described<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011989">26</a></sup> have been deposited in the GeneExpression Omnibus (NCBI) and are accessible online through GEO Series accession number <a id="__tag_661012021" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE43974" ref="/geo/query/acc.cgi?acc=GSE43974"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE43974","term_id":"43974"}}</span>GSE43974</a>.</p></div><div id="S9" class="sec"><h3 id="S9title">LD Lookup</h3><p id="P12" class="p p-first-last">The 1000 Genomes Project phase 3 release of variant calls was used (February 20, 2015) using 503 subjects of European ancestry.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011981">22</a></sup>
<em>r</em><sup>2</sup> between the sentinel SNPs and all other biallelic SNPs within the corresponding 2 Mb area were calculated using the Tabix and PLINK software package (v1.07).<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011979">27</a>,<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012009">28</a></sup> Annotation was performed using the ANNOVAR software package.<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012014">29</a></sup></p></div><div id="S10" class="sec"><h3 id="S10title">Gene-Based Pathway Analysis</h3><p id="P13" class="p p-first-last">All genes identified in 3 or 4 of the whole-blood eQTL resources above (<a href="/pmc/articles/PMC5783787/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) and genes containing a nonsynonymous variant with <em>r</em><sup>2</sup>&gt;0.5 with the sentinel variant (Table S14) were tested for enrichment of biological pathways and gene ontology (GO) terms using ConsensusPathDB<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011994">30</a></sup> using a false discovery rate &lt;5% cutoff. Enriched pathways and GO terms containing genes only implicated by a single BP-associated variant were not reported.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P173">BP-Associated SNPs Associated With Expression of the Same Gene Across 4 or 3 Independent Whole-Blood eQTL Resources and the Kidney Resource</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140304099141328" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Sentinel SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Chr</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="center">Blood Data Sets</th><th rowspan="1" colspan="1" valign="bottom" align="center">Top eQTL</th><th rowspan="1" colspan="1" align="center">Signal in Other<br>Tissue(s) in GTEx</th><th rowspan="1" colspan="1" align="center">Signal in<br>Kidney</th><th rowspan="1" colspan="1" align="center">eQTL Signal<br>Previously Reported</th></tr></thead><tbody><tr><td colspan="9" rowspan="1" align="left">Signal in 4 whole-blood eQTL resources</td></tr><tr><td rowspan="1" colspan="1" align="left">rs17367504</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">11862778</td><td rowspan="1" colspan="1" align="center"><em>CLCN6</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2169137</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">204497913</td><td rowspan="1" colspan="1" align="center"><em>MDM4</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs10926988</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">243483279</td><td rowspan="1" colspan="1" align="center"><em>SDCCAG8</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs319690</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">47927484</td><td rowspan="1" colspan="1" align="center"><em>MAP4</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs12521868</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">131784393</td><td rowspan="1" colspan="1" align="center"><em>SLC22A5</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs900145</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">13293905</td><td rowspan="1" colspan="1" align="center"><em>ARNTL</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1060105</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">123806219</td><td rowspan="1" colspan="1" align="center"><em>CDK2AP1</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1378942</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">75077367</td><td rowspan="1" colspan="1" align="center"><em>SCAMP2</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1126464</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">89704365</td><td rowspan="1" colspan="1" align="center"><em>CHMP1A</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1126464</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">89704365</td><td rowspan="1" colspan="1" align="center"><em>FANCA</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs12946454</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">43208121</td><td rowspan="1" colspan="1" align="center"><em>DCAKD</em></td><td rowspan="1" colspan="1" align="center">YYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td colspan="9" rowspan="1" align="left">Signal in 3 (out of 4) whole-blood eQTL resources</td></tr><tr><td rowspan="1" colspan="1" align="left">rs17367504</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">11862778</td><td rowspan="1" colspan="1" align="center"><em>MTHFR</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs871524</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">38411445</td><td rowspan="1" colspan="1" align="center"><em>FHL3</em></td><td rowspan="1" colspan="1" align="center">NYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs871524</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">38411445</td><td rowspan="1" colspan="1" align="center"><em>SF3A3</em></td><td rowspan="1" colspan="1" align="center">NYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs4660293</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">40028180</td><td rowspan="1" colspan="1" align="center"><em>PABPC4</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs6749447</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">169041386</td><td rowspan="1" colspan="1" align="center"><em>STK39</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs347591</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">11290122</td><td rowspan="1" colspan="1" align="center"><em>ATG7</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs319690</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">47927484</td><td rowspan="1" colspan="1" align="center"><em>ZNF589</em></td><td rowspan="1" colspan="1" align="center">YYNY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs12521868</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">131784393</td><td rowspan="1" colspan="1" align="center"><em>SLC22A4</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1563788</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">43308363</td><td rowspan="1" colspan="1" align="center"><em>CRIP3</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs10943605</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">79655477</td><td rowspan="1" colspan="1" align="center"><em>PHIP</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4728142</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">128573967</td><td rowspan="1" colspan="1" align="center"><em>IRF5</em></td><td rowspan="1" colspan="1" align="center">NYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4728142</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">128573967</td><td rowspan="1" colspan="1" align="center"><em>TNPO3</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs2898290</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center">11433909</td><td rowspan="1" colspan="1" align="center"><em>BLK</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs2898290</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center">11433909</td><td rowspan="1" colspan="1" align="center"><em>FAM167A</em></td><td rowspan="1" colspan="1" align="center">NYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs2898290</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center">11433909</td><td rowspan="1" colspan="1" align="center"><em>FDFT1</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs2071518</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center">120435812</td><td rowspan="1" colspan="1" align="center"><em>NOV</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs76452347</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">35906471</td><td rowspan="1" colspan="1" align="center"><em>TPM2</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs10760117</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">123586737</td><td rowspan="1" colspan="1" align="center"><em>MEGF9</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4494250</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">96563757</td><td rowspan="1" colspan="1" align="center"><em>HELLS</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs11191548</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">104846178</td><td rowspan="1" colspan="1" align="center"><em>NT5C2</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs661348</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">1905292</td><td rowspan="1" colspan="1" align="center"><em>TNNT3</em></td><td rowspan="1" colspan="1" align="center">NYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs2649044</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">9763969</td><td rowspan="1" colspan="1" align="center"><em>SBF2</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs2649044</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">9763969</td><td rowspan="1" colspan="1" align="center"><em>SWAP70</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">?</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7129220</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">10350538</td><td rowspan="1" colspan="1" align="center"><em>ADM</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7103648</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">47461783</td><td rowspan="1" colspan="1" align="center"><em>MYBPC3</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs3741378</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">65408937</td><td rowspan="1" colspan="1" align="center"><em>CTSW</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs7302981</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">50537815</td><td rowspan="1" colspan="1" align="center"><em>LIMA1</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7302981</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">50537815</td><td rowspan="1" colspan="1" align="center"><em>ATF1</em></td><td rowspan="1" colspan="1" align="center">YYNY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1036477</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">48914926</td><td rowspan="1" colspan="1" align="center"><em>FBN1</em></td><td rowspan="1" colspan="1" align="center">YNYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1378942</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">75077367</td><td rowspan="1" colspan="1" align="center"><em>CSK</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1378942</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">75077367</td><td rowspan="1" colspan="1" align="center"><em>MPI</em></td><td rowspan="1" colspan="1" align="center">NYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1378942</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">75077367</td><td rowspan="1" colspan="1" align="center"><em>ULK3</em></td><td rowspan="1" colspan="1" align="center">YNYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs12946454</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">43208121</td><td rowspan="1" colspan="1" align="center"><em>NMT1</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2304130</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">19789528</td><td rowspan="1" colspan="1" align="center"><em>GATAD2A</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs867186</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="center">33764554</td><td rowspan="1" colspan="1" align="center"><em>EIF6</em></td><td rowspan="1" colspan="1" align="center">NYYY</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs6095241</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="center">47308798</td><td rowspan="1" colspan="1" align="center"><em>PREX1</em></td><td rowspan="1" colspan="1" align="center">YYYN</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs9306160</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="center">45107562</td><td rowspan="1" colspan="1" align="center"><em>RRP1B</em></td><td rowspan="1" colspan="1" align="center">YNYY</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140304099141328" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5783787/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><p id="P174" class="p p-first-last">Signals of association of SNP genotype and gene expression in other nonblood tissues in GTEx and in kidney are also indicated. Blood data set order: (1) SABRe, (2) NESDA-NTR, (3) BIOS, and (4) GTEx (whole-blood). Top eQTL: top GWAS SNP is top eQTL SNP (or in high LD, <em>r</em><sup>2</sup>&gt;0.9, with top eQTL SNP) in at least 1 data set. eQTL signal previously reported: Genes for which eQTL signals have been previously reported for that sentinel SNP.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011986">15</a>,<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011968">16</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012001">18</a></sup> For full list, see Table S12 in the <a href="#SD1" rid="SD1" class=" supplementary-material">online-only Data Supplement</a>. eQTL indicates expression quantitative trait loci; GWAS, genome-wide association studies; GTEx, genotype-tissue expression; and LD, linkage disequilibrium; and SABRe, Systems Approach to Biomarker Research in Cardiovascular Disease.</p></div></div></div></div><div id="S11" class="sec"><h3 id="S11title">Network Analysis</h3><p id="P14" class="p p-first-last">To construct a functional association network, we combined 2 prioritized candidate gene sets into a single query gene set as (1) genes mapping to the nonsynonymous SNPs in high LD (<em>r</em><sup>2</sup>&gt;0.5) with the corresponding sentinel BP-associated SNP and (2) genes with eQTL evidence from 3 or 4 of the blood eQTL resources. Three sentinel SNPs (rs185819, rs926552, and rs805303) mapping to the HLA (human leukocyte antigen) region on chromosome 6 were excluded from downstream analyses. The single query gene set was then used as input for the functional network analysis.<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012004">31</a></sup> We used the Cytoscape<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011993">32</a></sup> software platform extended by the GeneMANIA<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_806140059">33</a></sup> plugin (Data Version: August 12, 2014).<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011963">34</a></sup> All the genes in the composite network, either from the query or the resulting gene sets, were then used for functional enrichment analysis against GO terms<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011984">35</a></sup> to identify the most relevant GO terms using the same plugin.<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012007">34</a></sup></p></div><div id="S12" class="sec"><h3 id="S12title">DNase1 Hypersensitivity Overlap Enrichment Across Tissue and Cell Types</h3><p id="P15" class="p p-first-last">The functional element overlap analysis of the results of GWAS experiments (Forge tool v1.1)<sup><a href="#R36" rid="R36" class=" bibr popnode">36</a></sup> was used to test for enrichment of overlap of BP SNPs in tissues and cell lines from the Roadmap and ENCODE (Encyclopedia of DNA Elements) projects. All 164 SNPs were entered and 143 were included in the analysis. SNPs from 9 commonly used GWAS arrays were used to select background sets of SNPs for comparison, and 10 000 background repetitions were run. A <em>Z</em> score threshold of ≥3.39 (estimated false-positive rate of 0.5%) was used to declare significance.</p></div><div id="S13" class="sec sec-last"><h3 id="S13title">Drug–Gene Interactions</h3><p id="P16" class="p p-first-last">Genes used for pathway and GO enrichment analyses were further investigated for potential druggable or drugged targets using DGIdb (drug gene interaction database).<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011997">37</a></sup> Known drug–gene interactions were interrogated across 15 source databases in DGIdb and include all types of interactions. The analysis performed for druggability prediction included all 9 databases exclusively inspecting expert curated data only. We also evaluate genes for known tool compounds using Chembl (<a href="http://www.ebi.ac.uk/chembl/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ebi.ac.uk/chembl/</a>; version 22.1).</p></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Results</h2><p id="P17" class="p p-first">The stage 1 discovery meta-analysis included 150 134 individuals (Tables S1 through S4 and <a href="#SD1" rid="SD1" class=" supplementary-material">Figures S1 and S2</a>) and 7 994 604 variants with minor allele frequency &gt;1% and an effective sample size of at least 60% of the total. We used the widely used 2-stage design<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011992">38</a></sup> and identified 61 signals in the discovery analysis that were candidates for novel BP signals (<em>P</em>&lt;10<sup>−6</sup> for any trait; Table S5). To ensure robustness of signals, we examined BP associations in an additional 228 245 individuals from 15 independent studies for replication, including 140 886 individuals from UK Biobank<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716510436">19</a></sup> (Table S6). We used the most significant (sentinel) SNP and trait for each locus in replication (61 tests). Twenty-two putatively novel association signals were initially confirmed, showing significant evidence of replication in the independent stage-2 studies (<em>P</em>&lt;8.2×10<sup>−4</sup>, Bonferroni correction for 61 tests) and genome-wide significance (<em>P</em>&lt;5×10<sup>−8</sup>) in a meta-analysis across all 378 376 individuals (<a href="/pmc/articles/PMC5783787/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and Table S7). Of these, 14 were subsequently published in 2 other studies<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011999">17</a>,<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716510435">19</a></sup> which presented genome-wide significant associations with evidence of replication. A further 2 were highlighted as putative novel signals in one of those studies<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012017">17</a></sup> but had not been confirmed by replication. In our study, we report the 6 remaining novel signals, and the 2 previously unconfirmed signals (in <em>EBF2</em> and in <em>NFKBIA</em>), as novel signals. The 8 novel signals included 7 signals at 7 independent loci (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S3</a>) and 1 novel independent signal near a previously reported hit near <em>TNXB</em> (Table S8 and <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S4</a>). The novel signals show both significant evidence of replication in the independent stage-2 studies (<em>P</em>&lt;8.2×10<sup>−4</sup>, Bonferroni correction for 61 tests) and genome-wide significance (<em>P</em>&lt;5×10<sup>−8</sup>) in a meta-analysis across all 378 376 individuals. The sentinel variants at all 8 signals were common (minor allele frequency &gt;5%), and the novel secondary signal at <em>TNXB</em> was in high linkage disequilibrium (<em>r</em><sup>2</sup>&gt;0.8) with a nonsynonymous SNP. With the exception of rs9710247, which was only significant for association with DBP, all signals were significantly associated (<em>P</em>&lt;0.006, Bonferroni corrected for 8 tests) with all 3 traits (<a href="/pmc/articles/PMC5783787/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and Table S9).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P168">Novel Genome-Wide Significant Signals of Association</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140304096823424" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="left">Variant ID<br>(Noncoded/Coded<br>Allele), Chr:Position,<br>Nearest Gene(s)<br>(Type<sup>*</sup>)</th><th rowspan="3" colspan="1" valign="bottom" align="center">CAF</th><th colspan="9" rowspan="1" valign="bottom" align="center">Results for Most Significant Trait</th><th colspan="3" rowspan="2" valign="bottom" align="center">Stage 1+Stage 2 Meta-Analysis <em>P</em><br>Values for All Traits</th></tr><tr><th colspan="2" rowspan="1" valign="bottom" align="center">Stage 1</th><th colspan="3" rowspan="1" valign="bottom" align="center">Stage 2</th><th colspan="4" rowspan="1" valign="bottom" align="center">Stage 1+Stage 2</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">Beta (SE)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> Value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Neff</th><th rowspan="1" colspan="1" valign="bottom" align="center">Beta (SE)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> Value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Neff</th><th rowspan="1" colspan="1" valign="bottom" align="center">Beta (SE)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> Value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Neff</th><th rowspan="1" colspan="1" valign="bottom" align="center">SBP</th><th rowspan="1" colspan="1" valign="bottom" align="center">DBP</th><th rowspan="1" colspan="1" valign="bottom" align="center">PP</th></tr></thead><tbody><tr><td colspan="14" rowspan="1" align="left">SBP</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1048238 (C/T), 1:16341649, <em>HSPB7</em> (3'UTR)</td><td rowspan="1" colspan="1" align="center">0.571</td><td rowspan="1" colspan="1" align="center">0.366 (0.074)</td><td rowspan="1" colspan="1" align="center">8.09E-07</td><td rowspan="1" colspan="1" align="center">140299</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1" align="left">rs848309 (proxy) (T/C), 1:16308447</td><td rowspan="1" colspan="1" align="center">0.567</td><td rowspan="1" colspan="1" align="center">0.347 (0.072)</td><td rowspan="1" colspan="1" align="center">1.70E-06</td><td rowspan="1" colspan="1" align="center">146755</td><td rowspan="1" colspan="1" align="center">0.347 (0.071)</td><td rowspan="1" colspan="1" align="center">9.10E-07</td><td rowspan="1" colspan="1" align="center">140462</td><td rowspan="1" colspan="1" align="center">0.347 (0.051)</td><td rowspan="1" colspan="1" align="center">7.07E-12<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">287217</td><td rowspan="1" colspan="1" align="center">7.07E-12<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">1.07E-10<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">5.48E-06</td></tr><tr><td rowspan="1" colspan="1" align="left">rs185819 (T/C),<sup>†</sup> 6:32,050,067, <em>TNXB</em> (ns)</td><td rowspan="1" colspan="1" align="center">0.513</td><td rowspan="1" colspan="1" align="center">0.534 (0.073)</td><td rowspan="1" colspan="1" align="center">1.93E-13<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">142397</td><td rowspan="1" colspan="1" align="center">0.277 (0.053)</td><td rowspan="1" colspan="1" align="center">1.49E-07</td><td rowspan="1" colspan="1" align="center">221748</td><td rowspan="1" colspan="1" align="center">0.365 (0.043)</td><td rowspan="1" colspan="1" align="center">1.04E-17<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">364144</td><td rowspan="1" colspan="1" align="center">1.04E-17<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">2.24E-11<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">8.50E-15<sup>‡</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs6557876 (C/T), 8:25,900,675, <em>EBF2</em></td><td rowspan="1" colspan="1" align="center">0.252</td><td rowspan="1" colspan="1" align="center">−0.411 (0.084)</td><td rowspan="1" colspan="1" align="center">8.50E-07</td><td rowspan="1" colspan="1" align="center">143653</td><td rowspan="1" colspan="1" align="center">−0.350 (0.060)</td><td rowspan="1" colspan="1" align="center">5.66E-09<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">225803</td><td rowspan="1" colspan="1" align="center">−0.371 (0.049)</td><td rowspan="1" colspan="1" align="center">2.85E-14<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">369457</td><td rowspan="1" colspan="1" align="center">2.85E-14<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">2.50E-10<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">1.51E-08<sup>‡</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs35783704 (G/A), 8:105,966,258, <em>LRP12/ZFPM2</em></td><td rowspan="1" colspan="1" align="center">0.109</td><td rowspan="1" colspan="1" align="center">−0.609 (0.121)</td><td rowspan="1" colspan="1" align="center">4.96E-07</td><td rowspan="1" colspan="1" align="center">133924</td><td rowspan="1" colspan="1" align="center">−0.310 (0.089)</td><td rowspan="1" colspan="1" align="center">4.78E-04</td><td rowspan="1" colspan="1" align="center">215528</td><td rowspan="1" colspan="1" align="center">−0.414 (0.072)</td><td rowspan="1" colspan="1" align="center">7.08E-09<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">349452</td><td rowspan="1" colspan="1" align="center">7.08E-09<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">1.60E-06</td><td rowspan="1" colspan="1" align="center">2.92E-07</td></tr><tr><td rowspan="1" colspan="1" align="left">rs73099903 (C/T), 12:53,440,779, <em>LOC283335</em></td><td rowspan="1" colspan="1" align="center">0.074</td><td rowspan="1" colspan="1" align="center">0.768 (0.143)</td><td rowspan="1" colspan="1" align="center">8.05E-08</td><td rowspan="1" colspan="1" align="center">136064</td><td rowspan="1" colspan="1" align="center">0.396 (0.098)</td><td rowspan="1" colspan="1" align="center">5.32E-05</td><td rowspan="1" colspan="1" align="center">207253</td><td rowspan="1" colspan="1" align="center">0.515 (0.081)</td><td rowspan="1" colspan="1" align="center">1.95E-10<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">343318</td><td rowspan="1" colspan="1" align="center">1.95E-10<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">4.53E-06</td><td rowspan="1" colspan="1" align="center">5.46E-08</td></tr><tr><td rowspan="1" colspan="1" align="left">rs8904 (G/A), 14:35,871,217, <em>NFKBIA</em> (3'UTR)</td><td rowspan="1" colspan="1" align="center">0.375</td><td rowspan="1" colspan="1" align="center">0.377 (0.076)</td><td rowspan="1" colspan="1" align="center">6.76E-07</td><td rowspan="1" colspan="1" align="center">140424</td><td rowspan="1" colspan="1" align="center">0.278 (0.054)</td><td rowspan="1" colspan="1" align="center">2.31E-07</td><td rowspan="1" colspan="1" align="center">224771</td><td rowspan="1" colspan="1" align="center">0.311 (0.044)</td><td rowspan="1" colspan="1" align="center">1.31E-12<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">365195</td><td rowspan="1" colspan="1" align="center">1.31E-12<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">1.13E-04</td><td rowspan="1" colspan="1" align="center">3.44E-12<sup>‡</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs57927100 (C/G), 17:75,317,300, <em>SEPT9</em></td><td rowspan="1" colspan="1" align="center">0.258</td><td rowspan="1" colspan="1" align="center">−0.489 (0.086)</td><td rowspan="1" colspan="1" align="center">1.10E-08<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">136624</td><td rowspan="1" colspan="1" align="center">−0.220 (0.061)</td><td rowspan="1" colspan="1" align="center">3.12E-04</td><td rowspan="1" colspan="1" align="center">210563</td><td rowspan="1" colspan="1" align="center">−0.310 (0.050)</td><td rowspan="1" colspan="1" align="center">4.04E-10<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">347188</td><td rowspan="1" colspan="1" align="center">4.04E-10<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">1.16E-10<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">1.81E-05</td></tr><tr><td colspan="14" rowspan="1" align="left">DBP</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9710247 (A/G), 19:40,760,449, <em>AKT2</em></td><td rowspan="1" colspan="1" align="center">0.447</td><td rowspan="1" colspan="1" align="center">0.252 (0.051)</td><td rowspan="1" colspan="1" align="center">8.11E-07</td><td rowspan="1" colspan="1" align="center">109695</td><td rowspan="1" colspan="1" align="center">0.129 (0.032)</td><td rowspan="1" colspan="1" align="center">5.76E-05</td><td rowspan="1" colspan="1" align="center">198332</td><td rowspan="1" colspan="1" align="center">0.164 (0.027)</td><td rowspan="1" colspan="1" align="center">1.61E-09<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">308028</td><td rowspan="1" colspan="1" align="center">3.82E-02</td><td rowspan="1" colspan="1" align="center">1.61E-09<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">5.03E-01</td></tr></tbody></table></div><div id="largeobj_idm140304096823424" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5783787/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P169" class="p p-first-last">Results from stage 1 and stage 2, and the meta-analysis of stage 1 and stage 2, for all novel genome-wide significant signals of association. <em>P</em> values of association for all 3 traits from a meta-analysis of stages 1 and 2 are also presented. Results from proxy SNPs are indicated by (proxy); rs848309 was a proxy SNP for rs1048238, and rs10926988 was a proxy SNP for chr1:243458005:INDEL. CAF indicates coded allele frequency; DBP, diastolic blood pressure; Neff, effective sample size; ns, nonsynonymous; PP, pulse pressure; s, synonymous; SBP, systolic blood pressure; and UTR, untranslated region.</p></div><div id="TFN2"><sup>*</sup>For intragenic variants, the nearest genes are listed; all other variants are intronic unless indicated otherwise.</div><div id="TFN3"><sup>†</sup>Novel signal at previously reported locus.</div><div id="TFN4"><sup>‡</sup>Genome-wide significant <em>P</em> values (<em>P</em>&lt;5×10<sup>−8</sup>).</div></div></div><p id="P18">We next sought to identify which genes might have expression levels that were associated with genotypes of the BP-associated variants reported in this study and others. Strong evidence of an association with expression of a specific gene may provide clues as to which gene(s) might be functionally relevant to that signal. We took the 139 BP association signals reported prior to these studies<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012000">17</a>,<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716510434">19</a></sup> and 22 novel signals of association identified and confirmed in this study and 2 contemporaneous studies<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011982">3</a>–<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716510433">19</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011973">24</a></sup> (Table S10) and searched for evidence of association with gene expression in whole blood (4 studies, total n=12 607; <a href="#SD1" rid="SD1" class=" supplementary-material">supporting information in the online-only Data Supplement</a>) and in kidney tissue (n=134, the largest kidney eQTL resource currently available). Although of unclear direct relevance to BP, whole blood was studied because of the availability of large data sets enabling a powerful assessment of expression patterns that are likely present across multiple cell and tissue types. Similarly, circulating blood cells have been used for ion transport experiments in the past, and altered ion transport levels in erythrocytes were linked to hypertension.<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011990">39</a></sup> Kidney was chosen because of the many renal pathways that regulate BP and outstanding questions about the relevance of kidney pathways to the genetic component of BP regulation in the general population.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012010">3</a>,<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012002">15</a></sup> eQTL signals were filtered by false discovery rate (&lt;5%), and we examined <em>cis</em> (within 1 Mb) associations only (<a href="#SD1" rid="SD1" class=" supplementary-material">supporting information in the online-only Data Supplement</a>).</p><p id="P19">The 4 blood eQTL data sets were NESDA-NTR,<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011977">40</a>,<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011964">41</a></sup> SABRe,<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012011">15</a></sup> the BIOS resource,<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011976">42</a></sup> and GTEx<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011980">25</a></sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">supporting information in the online-only Data Supplement</a>). The BIOS resource (n=2116) has not previously been used in the analysis of BP associations, and findings from NESDA-NTR and SABRe have been reported for a subset of the previously published signals.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011985">16</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012005">18</a></sup> For a total of 369 genes, gene expression was associated with the BP SNP in ≥1 of the 4 blood data sets at experiment-wide significance (Table S11). This included 14 genes for 6 of the 8 novel signals. For 110 genes, we found eQTL evidence in 2 out of 4 data sets (<a href="/pmc/articles/PMC5783787/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure</span></a>), including 4 genes for 2 of the novel signals: <em>EIF4B</em> and <em>TNS2</em> for rs73099903 and <em>MAP3K10</em> and <em>PLD3</em> for rs9710247. SNP rs73099903 was in strong LD (<em>r</em><sup>2</sup>&gt; 0.9), with the SNP most strongly associated with <em>TNS2</em> expression in the BIOS resource. <em>TNS2</em> encodes a tensin focal adhesion molecule and may have a role in renal function.<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011969">43</a></sup></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5783787/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140304096826000" class="figure"><a href="/pmc/articles/PMC5783787/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5783787_nihms902240f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms902240f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5783787/bin/nihms902240f1.jpg"></a></div><div id="largeobj_idm140304096826000" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5783787/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5783787/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure</a></div><!--caption a7--><div class="caption"><p id="P167">Overlap of expression quantitative trait loci (eQTL) evidence from 4 whole-blood and 1 kidney resource. The figure indicates overlap of evidence for eQTLs from 4 whole-blood studies (SABRe, NESDA-NTR, BIOS, and GTEx) and from 1 kidney resource (TransplantLines). Every colored line indicates that this gene was analysis-wide significant in a given resource. Only genes identified by at least 2 resources are shown. The genes are sorted by genomic position on the <em>y</em> axis.</p></div></div></div><p id="P20">For 48 genes, we found evidence in 3 out of the 4 resources (<a href="/pmc/articles/PMC5783787/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), suggesting robustness of the SNP–gene expression correlation signal and highlighting those genes as potential candidates in genetic BP regulation. Of the 48 genes, 28 have not previously been described in eQTL analyses using BP-associated SNPs, and all were correlated with previously reported BP association signals.</p><p id="P21">In the kidney data set (TransplantLines),<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011962">26</a></sup> there was association of gene expression and genotype for 9 SNPs and 13 genes (<a href="/pmc/articles/PMC5783787/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC5783787/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure</span></a>; Table S12). Nine of the SNP–gene expression associations were also observed in the whole-blood eQTL data sets, suggesting that those signals may not be unique to the kidney. We report 3 signals that were unique to the kidney and not previously reported (<em>C4orf34, HIP2</em>, and <em>ASIC1</em>) and confirm a previously reported kidney eQTL signal for an antisense RNA for <em>PSMD5</em>.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011995">15</a></sup> The same SNP was also an eQTL for <em>PSMD5</em> itself in both blood and kidney. <em>ASIC1</em> encodes the acid sensing ion channel subunit 1, which may interact (and be coexpressed) with ENaC subunits, which mediate transepithelial Na transport in the distal nephron of the kidney.<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011970">44</a></sup> The comparatively small number of signals using kidney tissue (<a href="/pmc/articles/PMC5783787/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC5783787/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure</span></a>) compared with whole blood could be because of the small sample size. Complete GTEx results are given in Table S13.</p><p id="P22">For genes implicated by eQTL information from whole blood, we tested for enrichment of biological pathways and GOs. We noted enrichment of the 48 genes implicated by 3 or 4 blood eQTL resources (<a href="/pmc/articles/PMC5783787/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) and a further 54 genes containing a nonsynonymous variant with <em>r</em><sup>2</sup>&gt;0.5 with the top SNP (Table S14) in pathways and ontology terms related to actin and striated muscle (Tables S15 and S16). Network analysis using the same genes highlighted further GO terms relating to muscle function, particularly cardiac muscle (Table S17). We tested the overlap of 161 non-HLA BP-associated variants with DNase hypersensitivity sites identified in the Roadmap and ENCODE cell lines and identified an overall enrichment in multiple cell and tissue types, including heart, kidney, and smooth muscle (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5</a>).</p><p id="P23" class="p p-last">We next investigated these genes for potential suitability as drug targets (druggability), known tool compounds, and clinically approved drugs using DGIdb<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011965">37</a></sup> (Table S18). Twelve genes had known drugs, including 4 genes with known antihypertensive drugs. We noted that drugs modulating all but 1 of the 12 drugged targets had a reported influence on BP, either as a primary antihypertensive indication or as a reported side effect of raised BP. Twenty additional genes were predicted druggable, among these 7 genes have known small molecule tool modulators, based on a query of the Chembl database (<a href="http://www.ebi.ac.uk/chembldb/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ebi.ac.uk/chembldb/</a>; version 22.1).</p></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Discussion</h2><p id="P24" class="p p-first">Enhanced discovery of BP loci increases the potential targets for therapeutic advances. After major advances in the number of BP loci known over the last years and months, we report 8 novel signals that implicate 5 regions of the genome not previously connected to BP regulation.</p><p id="P25">Six of the 8 novel signals we report had not previously been reported. Two signals (in <em>EBF2</em> and <em>NFKBIA</em>) have been suggested previously but without evidence for replication.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661011996">17</a></sup> For these 2 signals, we present, for the first time, stringent evidence of replication, confirming their relevance to BP genetics.</p><p id="P26">The path from signal to genes is the essential next step toward realizing the therapeutic potential of a genetic locus and understanding the mechanisms of BP regulation. We have used several large eQTL resources as a first step to realize this objective. As expected, we observed that even across eQTL studies of the same tissue, there is limited overlap in experiment-wide significant signals, suggesting either biological variability (differences in the characteristics of the samples or in the methods for extraction and processing of mRNA in each of the studies), technology-specific differences in coverage of genes (use of RNAseq data for the BIOS blood data set and microarray-based expression levels for the kidney and other blood data sets), or the possibility of false-positive results despite stringent within-experiment significance thresholds. We were unable to distinguish these scenarios using the data available to us, but by selecting genes that were significant in at least 3 resources, and therefore robust to these differences, we identified 48 genes as candidates for further study. These results are limited by the availability of large eQTL resources for whole blood only, which precludes well-powered comparisons across tissue types, particularly, as the origin of BP control is unlikely to be located in the blood. Enrichment and pathway analyses using these genes, and genes containing a correlated functional variant, highlight the potential relevance of muscular tissue and pathways, compatible with a vascular and cardiac origin of BP genetics, extending previous evidence.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012008">15</a></sup> We identify several drugged targets in the pathways identified, including 4 existing hypertension targets. Other drugs identified are not suitable candidates for repositioning to hypertension because most were reported in adverse events to raise BP; however, the targets would be valid for investigation using a reverse mechanism, for example, agonism in place of inhibition. We also identified 7 genes with small molecule tool modulators (mainly inhibitory or binding). These molecules and targets might be suitable candidates for further investigation to build a target validation case to support clinical investigation in hypertension.</p><p id="P27">Among the genes implicated in our eQTL, analyses were several for which there is already some evidence that they are relevant to BP regulation. The intronic SNP rs10926988 was independently associated with expression of <em>SDCCAG8</em> in all 4 whole-blood resources. Rare mutations in <em>SDCCAG8</em> cause Bardet–Biedl syndrome, which features hypertension. Expression levels of <em>MYBPC3</em> were correlated with rs7103648<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_661012019">15</a></sup> in the 3 largest blood eQTL resources (ie, SABRe, NESDA-NTR, and BIOS). <em>MYBPC3</em> encodes the cardiac isoform of myosin-binding protein C, which is expressed in heart muscle, and mutations in <em>MYBPC3</em> are known to cause familial hypertrophic cardiomyopathy.<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_806140060">45</a></sup></p><p id="P28" class="p p-last">This study has several limitations. Given the nature of statistical power for genome-wide association analyses, the sample size is limited, even though this is one of the largest efforts in BP GWAS undertaken to date. The study would clearly have benefited from the availability of larger eQTL resources on multiple tissues in sample sizes even larger than those available today. Our analyses were limited to <em>cis</em> signals, and future analyses, with larger sample sizes, might also consider <em>trans</em> signals.</p></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Perspectives</h2><p id="P29" class="p p-first">Our study reports robust novel BP association signals and reports new candidate BP genes, contributing to the transition from variants to genes to explain BP variation. These genes now require further functional validation to establish their potential as drug targets. Our study additionally highlights the challenges of combining and interpreting data from multiple eQTL studies and emphasizes the need for harmonization of data and development of new eQTL resources for multiple tissue types.</p><p id="P30" class="p p-last">In summary, our study reports novel BP association signals and reports new candidate BP genes, contributing to the transition from variants to genes to explain BP variation.</p></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="idm140304135526576title">Novelty and Significance</h3></div><div class="sec" id="S18"><h3 id="S18title">What Is New?</h3><ul class="unordered" style="list-style-type:disc" id="L1"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="P175">The root origin of hypertension and, hence, blood pressure (BP) variability in the population remains unclear.</p></li><li><p id="P176">This study adds data to explain the genetic basis of BP variability and identifies genes likely active in BP-regulating pathways.</p></li></ul></div><div class="sec" id="S19"><h3 id="S19title">What Is Relevant?</h3><ul class="unordered" style="list-style-type:disc" id="L2"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="P177">The results are of relevance for scientists, clinicians, and pharmacologists interested in hypertension.</p></li><li><p id="P178">The BP loci and the BP genes identified constitute new leads for the understanding of BP pathogenesis and possibly therapeutic innovation.</p></li></ul></div><div class="sec" id="S20"><h3 id="S20title">Summary</h3><p id="P179" class="p p-first-last">Using 1000 Genomes Project–based imputation in 150 134 European ancestry and independent replication in a further 228 245 individuals, we contribute 8 replicated BP loci to the collection of loci currently known. Using these and previous data, 48 BP genes are identified for priority follow-up.</p></div></div></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><div class="sup-box half_rhythm" id="d36e2225"><a href="/pmc/articles/PMC5783787/bin/NIHMS902240-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.6M, pdf)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="P110">We thank all the study participants of this study for their contributions. Detailed acknowledgment of funding sources is provided in the Sources of Funding section.</p><p id="P111"><strong>Sources of Funding</strong></p><p id="P112">This research used the ALICE and SPECTRE High Performance Computing Facilities at the University of Leicester. G.B. Ehret is supported by Geneva University Hospitals, Geneva University, de Reuter Foundation, the Swiss National Foundation project FN 33CM30-124087, and the Fondation pour Recherches Médicales, Geneva.</p><p id="P113">Airwave: We thank all participants of the Airwave Health Monitoring Study. The study is funded by the UK Home Office (Grant number 780-TETRA) with additional support from the National Institute for Health Research Imperial College Healthcare NHS Trust and Imperial College Biomedical Research Centre.</p><p id="P114">ARIC: The Atherosclerosis Risk in Communities Study is performed as a collaborative study supported by National Heart, Lung, and Blood Institute contracts (HHSN268201100005C, HHSN268201100006C, HHSN268201100007C, HHSN268201100008C, HHSN2682011000 09C, HHSN268201100010C, HHSN268201100011C, and HHSN2682 01100012C), R01HL087641, R01HL59367, and R01HL086694; National Human Genome Research Institute contract U01HG004402; and National Institutes of Health contract HHSN268200625226C. Funding support for the Genetic Epidemiology of Causal Variants Across the Life Course (CALiCo) program was provided through the NHGRI PAGE program (U01 HG007416). We thank the staff and participants of the ARIC study for their important contributions.</p><p id="P115">ASCOT: This work was supported by Pfizer, New York, NY, for the ASCOT study and the collection of the ASCOT DNA repository; by Servier Research Group, Paris, France; and by Leo Laboratories, Copenhagen, Denmark. We thank all ASCOT trial participants, physicians, nurses, and practices in the participating countries for their important contribution to the study. In particular, we thank Clare Muckian and David Toomey for their help in DNA extraction, storage, and handling. This work forms part of the research programme of the NIHR Cardiovascular Biomedical Research Unit at Barts.</p><p id="P116">ASPS: The research reported in this article was funded by the Austrian Science Fond (FWF) grant number P20545-P05 and <a id="__tag_806140058" class="tag_hotlink" href="/protein/P13180" ref="/protein/P13180"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P13180","term_id":"150417962","term_text":"P13180"}}</span>P13180</a>. The Medical University of Graz supports the databank of the ASPS. We thank the staff and the participants of the ASPS for their valuable contributions. We thank Birgit Reinhart for her long-term administrative commitment and Ing Johann Semmler for the technical assistance at creating the DNA bank.</p><p id="P117">BRIGHT: This work was supported by the Medical Research Council of Great Britain (grant number G9521010D) and by the British Heart Foundation (grant number PG/02/128). The BRIGHT study is extremely grateful to all the patients who participated in the study and the BRIGHT nursing team. This work forms part of the research programme of the NIHR Cardiovascular Biomedical Research Unit at Barts.</p><p id="P118">B58C: We acknowledge use of phenotype and genotype data from the British 1958 Birth Cohort DNA collection, funded by the Medical Research Council grant G0000934 and the Wellcome Trust grant 068545/Z/02. Genotyping for the B58C-WTCCC subset was funded by the Wellcome Trust grant 076113/B/04/Z. The B58C-T1DGC genotyping used resources provided by the Type 1 Diabetes Genetics Consortium, a collaborative clinical study sponsored by the National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK), National Institute of Allergy and Infectious Diseases (NIAID), National Human Genome Research Institute (NHGRI), National Institute of Child Health and Human Development (NICHD), and Juvenile Diabetes Research Foundation International (JDRF) and supported by U01 <a id="__tag_806140076" class="tag_hotlink" href="/nuccore/DK062418" ref="/nuccore/DK062418"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"DK062418","term_id":"187691508","term_text":"DK062418"}}</span>DK062418</a>. B58C-T1DGC GWAS data were deposited by the Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research (CIMR), University of Cambridge, which is funded by Juvenile Diabetes Research Foundation International, the Wellcome Trust, and the National Institute for Health Research Cambridge Biomedical Research Centre; the CIMR is in receipt of a Wellcome Trust Strategic Award (079895). The B58C-GABRIEL genotyping was supported by a contract from the European Commission Framework Programme 6 (018996) and grants from the French Ministry of Research.</p><p id="P119">CHS: This CHS research was supported by National Heart, Lung, and Blood Institute (NHLBI) contracts HHSN268201200036C, HHSN268200800007C, N01HC55222, N01HC85079, N01HC85080, N01HC85081, N01HC85082, N01HC 85083, N01HC85086, and HHSN268200960009C and NHLBI grants U01HL080295, R01HL087652, R01HL105756, R01HL103612, R01HL120393, and R01HL130114, with additional contribution from the National Institute of Neurological Disorders and Stroke (NINDS). Additional support was provided through R01AG023629 from the National Institute on Aging (NIA). A full list of principal CHS investigators and institutions can be found at <a href="http://CHS-NHLBI.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">CHS-NHLBI.org</a>. The provision of genotyping data was supported in part by the National Center for Advancing Translational Sciences, CTSI grant UL1TR000124, and the National Institute of Diabetes and Digestive and Kidney Disease Diabetes Research Center (DRC) grant DK063491 to the Southern California Diabetes Endocrinology Research Center. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p><p id="P120">Cilento study: The Cilento study was supported by the Italian Ministry of Education Universities and Research (Interomics Flagship Project, PON03PE_00060_7), FP6 (Vasoplus-037254), the Assessorato Ricerca Regione Campania, the Fondazione con il SUD (2011-PDR-13), and the Istituto Banco di Napoli-Fondazione to M. Ciullo. We address special thanks to the populations of Cilento for their participation in the study.</p><p id="P121">COLAUS: The CoLaus study was and is supported by research grants from GlaxoSmithKline (GSK), the Faculty of Biology and Medicine of Lausanne, and the Swiss National Science Foundation (grants 3200B0-105993, 3200B0- 118308, 33CSCO-122661, and 33CS30-139468). We thank all participants, involved physicians, and study nurses to the CoLaus cohort.</p><p id="P122">COROGENE_CTRL: This study has been funded by the Academy of Finland (grant numbers 139635, 129494, 118065, 129322, and 250207), the Orion-Farmos Research Foundation, the Finnish Foundation for Cardiovascular Research, and the Sigrid Jusélius Foundation. We are grateful for the THL DNA laboratory for its skillful work to produce the DNA samples used in this study. We thank the Sanger Institute genotyping facilities for genotyping the samples.</p><p id="P123">CROATIA Studies: The CROATIA-Vis, CROATIA-Korcula, and CROATIA-Split studies in the Croatian islands of Vis and Korcula and mainland city of Split were supported by grants from the Medical Research Council (UK); the Ministry of Science, Education, and Sport of the Republic of Croatia (grant number 216-1080315-0302); the European Union framework program 6 European Special Populations Research Network project (contract LSHG-CT-2006–018947); the European Union framework program 7 project BBMRI-LPC (FP7 313010); and the Croatian Science Foundation (grant 8875). The CROATIA studies acknowledge the invaluable contributions of the recruitment teams (including those from the Institute of Anthropological Research in Zagreb) in Vis, Korcula, and Split, the administrative teams in Croatia and Edinburgh, and the people of Vis, Korcula, and Split. SNP genotyping of the CROATIA-Vis samples was performed by the Genetics Core Laboratory at the Wellcome Trust Clinical Research Facility, WGH, Edinburgh, Scotland. SNP genotyping for CROATIA-Korcula was performed by Helmholtz ZentrumMünchen, GmbH, Neuherberg, Germany. The SNP genotyping for the CROATIA-Split cohort was performed by AROS Applied Biotechnology, Aarhus, Denmark.</p><p id="P124">ERF: The ERF study as a part of EUROSPAN (European Special Populations Research Network) was supported by European Commission FP6 STRP grant number 018947 (LSHG-CT-2006-01947) and also received funding from the European Community’s Seventh Framework Programme (FP7/2007–2013)/grant agreement HEALTH-F4-2007–201413 by the European Commission under the programme “Quality of Life and Management of the Living Resources” of 5th Framework Programme (no. QLG2-CT-2002-01254). High-throughput analysis of the ERF data was supported by a joint grant from the Netherlands Organization for Scientific Research and the Russian Foundation for Basic Research (NWO-RFBR 047.017.043). Exome sequencing analysis in ERF was supported by the ZonMw grant (project 91111025). Najaf Amin is supported by the Netherlands Brain Foundation (project number F2013(1)-28). We are grateful to all study participants and their relatives, general practitioners, and neurologists for their contributions and to P. Veraart for her help in genealogy, J. Vergeer for the supervision of the laboratory work, and P. Snijders for his help in data collection.</p><p id="P125">Fenland: J. Luan, C. Langenberg, R.A. Scott, and N.J. Wareham acknowledge support from the Medical Research Council (MC_ U106179471 and MC_UU_12015/1). The Fenland Study is funded by the Wellcome Trust and the Medical Research Council (MC_U106179471). We are grateful to all the volunteers for their time and help and to the General Practitioners and practice staff for assistance with recruitment. We thank the Fenland Study Investigators, Fenland Study Co-ordination team and the Epidemiology Field, Data and Laboratory teams. We further acknowledge support from the Medical Research Council (MC_UU_12015/1).</p><p id="P126">FHS: The National Heart, Lung and Blood Institute’s Framingham Heart Study is supported by contract N01-HC-25195.</p><p id="P127">FINRISK_PREDICT_CVD: This study has been funded by the Academy of Finland (grant numbers 139635, 129494, 118065, 129322, 250207, and 269517), the Orion-Farmos Research Foundation, the Finnish Foundation for Cardiovascular Research, and the Sigrid Jusélius Foundation. We are grateful for the THL DNA laboratory for its skillful work to produce the DNA samples used in this study. We thank the Sanger Institute genotyping facilities for genotyping the samples.</p><p id="P128">FUSION: Support for FUSION was provided by National Instituttes of Health (NIH) grants R01-DK062370 (to M. Boehnke) and intramural project number ZIA-<a id="__tag_806140064" class="tag_hotlink" href="/nuccore/HG000024" ref="/nuccore/HG000024"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG000024","term_id":"743402349","term_text":"HG000024"}}</span>HG000024</a> (to F. Collins). Genome-wide genotyping was conducted by the Johns Hopkins University Genetic Resources Core Facility SNP Center at the Center for Inherited Disease Research (CIDR), with support from CIDR NIH contract no. N01-HG-65403.</p><p id="P129">GAPP study: The GAPP study was supported by the Liechtenstein Government, the Swiss National Science Foundation, the Swiss Heart Foundation, the Swiss Society of Hypertension, the University of Basel, the University Hospital Basel, the Hanela Foundation, Schiller AG, and Novartis.</p><p id="P130">GS:SFHS: Generation Scotland received core support from the Chief Scientist Office of the Scottish Government Health Directorates (CZD/16/6) and the Scottish Funding Council (HR03006). Genotyping of the GS:SFHS samples was performed by the Genetics Core Laboratory at the Wellcome Trust Clinical Research Facility, Edinburgh, Scotland, and was funded by the Medical Research Council UK and the Wellcome Trust (Wellcome Trust Strategic Award “STratifying Resilience and Depression Longitudinally” (STRADL) Reference 104036/Z/14/Z). Ethics approval for the study was given by the NHS Tayside committee on research ethics (reference 05/S1401/89). We are grateful to all the families who took part, the general practitioners, and the Scottish School of Primary Care for their help in recruiting them and the whole Generation Scotland team, which includes interviewers, computer and laboratory technicians, clerical workers, research scientists, volunteers, managers, receptionists, healthcare assistants, and nurses.</p><p id="P131">GoDARTS: GoDARTS was funded by The Wellcome Trust (072960/Z/03/Z, 084726/Z/08/Z, 084727/Z/08/Z, 085475/Z/08/Z, and 085475/B/08/Z) and as part of the EU IMI-SUMMIT program. We acknowledge the support of the Health Informatics Centre, University of Dundee, for managing and supplying the anonymized data and NHS Tayside, the original data owner. We are grateful to all the participants who took part in the Go-DARTS study, to the general practitioners, to the Scottish School of Primary Care for their help in recruiting the participants, and to the whole team, which includes interviewers, computer and laboratory technicians, clerical workers, research scientists, volunteers, managers, receptionists, and nurses.</p><p id="P132">GRAPHIC: The GRAPHIC Study was funded by the British Heart Foundation (BHF/RG/2000004). C.P. Nelson and N.J. Samani are supported by the British Heart Foundation, and N.J. Samani is an NIHR Senior Investigator. This work falls under the portfolio of research supported by the NIHR Leicester Cardiovascular Biomedical Research Unit.</p><p id="P133">H2000: The Health 2000 Study was funded by the National Institute for Health and Welfare (THL), the Finnish Centre for Pensions (ETK), the Social Insurance Institution of Finland (KELA), the Local Government Pensions Institution (KEVA), and other organizations listed on the website of the survey (<a href="http://www.terveys2000.fi" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.terveys2000.fi</a>). We are grateful for the THL DNA laboratory for its skillful work to produce the DNA samples used in this study. We thank the Sanger Institute genotyping facilities for genotyping the GenMets subcohort.</p><p id="P134">HABC: The Health ABC Study was supported by NIA contracts N01AG62101, N01AG62103, and N01AG62106 and, in part, by the NIA Intramural Research Program. The genome-wide association study was funded by NIA grant 1R01AG032098-01A1 to Wake Forest University Health Sciences, and genotyping services were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, contract number HHSN268200782096C. This study used the high-performance computational capabilities of the Biowulf Linux cluster at the National Institutes of Health, Bethesda, MD (<a href="http://biowulf.nih.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://biowulf.nih.gov</a>).</p><p id="P135">HTO: The study was funded by the Wellcome Trust, Medical Research Council, and British Heart Foundation We thank all the families who participated in the study.</p><p id="P136">INGI-VB: The INGI-Val Borbera population is a collection of 1664 genotyped samples collected in the Val Borbera Valley, a geographically isolated valley located within the Appennine Mountains in Northwest Italy. The valley is inhabited by ≈3000 descendants from the original population, living in 7 villages along the valley and in the mountains. Participants were healthy people aged 18 to 102 years who had at least one grandfather living in the valley. The study plan and the informed consent form were reviewed and approved by the institutional review boards of San Raffaele Hospital in Milan. The research was supported by funds from Compagnia di San Paolo, Torino, Italy; Fondazione Cariplo, Italy and Ministry of Health, Ricerca Finalizzata 2008 and CCM 2010, PRIN 2009, and Telethon, Italy, to D. Toniolo. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the article. We thank the inhabitants of the VB who made this study possible, the local administrations, the MD of the San Raffaele Hospital, and Prof Clara Camaschella for clinical data collection. We also thank Fiammetta Viganò for technical help, Corrado Masciullo and Massimiliano Cocca for building and maintaining the analysis platform.</p><p id="P137">INGI-CARL: Italian Ministry of Health RF2010 to P. Gasparini and RC2008 to P. Gasparini.</p><p id="P138">INGI-FVG: Italian Ministry of Health RF2010 to P. Gasparini and RC2008 to P. Gasparini.</p><p id="P139">JUPITER: Genetic analysis in the JUPITER trial was funded by a grant from AstraZeneca (D.I. Chasman and P.M. Ridker, Co-Pis).</p><p id="P140">KORA S3: KORA S3 500K blood pressure project was supported by Estonian Research Council, grant IUT34-12 (for M. Laan). The KORA Augsburg studies have been financed by the Helmholtz Zentrum Munchen, German Research Center for Environmental Health, Neuherberg, Germany, and supported by grants from the German Federal Ministry of Education and Research (BMBF). The KORA study group consists of H.-E. Wichmann (speaker), A. Peters, C. Meisinger, T. Illig, R. Holle, J. John, and coworkers who are responsible for the design and conduct of the KORA studies. Part of this work was financed by the German National Genome Research Network (NGFN-2 and NGFNPlus:01GS0823) and supported within the Munich Center of Health Sciences (MC Health) as part of LMUinnovativ.</p><p id="P141">LBC1921: Phenotype collection in the Lothian Birth Cohort 1921 was supported by the UK’s Biotechnology and Biological Sciences Research Council (BBSRC), The Royal Society, and The Chief Scientist Office of the Scottish Government. Genotyping was funded by the BBSRC. The work was undertaken by The University of Edinburgh Centre for Cognitive Ageing and Cognitive Epidemiology, part of the cross council Lifelong Health and Wellbeing Initiative (MR/K026992/1). Funding from the BBSRC and Medical Research Council (MRC) is gratefully acknowledged. We thank the Lothian Birth Cohort 1921 participants and team members who contributed to these studies.</p><p id="P142">LBC1936: Phenotype collection in the Lothian Birth Cohort 1936 was supported by Age UK (The Disconnected Mind project). Genotyping was funded by the BBSRC. The work was undertaken by The University of Edinburgh Centre for Cognitive Ageing and Cognitive Epidemiology, part of the cross council Lifelong Health and Wellbeing Initiative (MR/K026992/1). Funding from the BBSRC and Medical Research Council (MRC) is gratefully acknowledged. We thank the Lothian Birth Cohort 1936 participants and team members who contributed to these studies.</p><p id="P143">Lifelines Cohort Study: The Lifelines Cohort Study and generation and management of GWAS genotype data for the Lifelines Cohort Study is supported by the Netherlands Organization of Scientific Research NWO (grant 175.010.2007.006), the Economic Structure Enhancing Fund (FES) of the Dutch government, the Ministry of Economic Affairs, the Ministry of Education, Culture and Science, the Ministry for Health, Welfare and Sports, the Northern Netherlands Collaboration of Provinces (SNN), the Province of Groningen, University Medical Center Groningen, the University of Groningen, Dutch Kidney Foundation, and Dutch Diabetes Research Foundation. We acknowledge the services of the Lifelines Cohort Study, the contributing research centers delivering data to Lifelines, and all the study participants.</p><p id="P144">LOLIPOP: The LOLIPOP study is funded by the British Heart Foundation (SP/04/002), the Medical Research Council (G0601966 and G0700931), the Wellcome Trust (084723/Z/08/Z), the NIHR (RP-PG-0407-10371), European Union FP7 (EpiMigrant, 279143), and Action on Hearing Loss (G51). The LOLIPOP study is supported by the National Institute for Health Research (NIHR) Comprehensive Biomedical Research Centre Imperial College Healthcare NHS Trust. The work was performed in part at the NIHR/Wellcome Trust Imperial Clinical Research Facility. We thank the participants and research staff who made the study possible.</p><p id="P145">MESA: This research was supported by the Multi-Ethnic Study of Atherosclerosis (MESA) contracts N01-HC-95159, N01-HC-95160, N01-HC-95161, N01-HC-95162, N01-HC-95163, N01-HC-95164, N01-HC-95165, N01-HC-95166, N01-HC-95167, N01-HC-95168, and N01-HC-95169 and by grants UL1-TR-000040 and UL1-RR-025005 from NCRR. Funding for MESA SHARe genotyping was provided by NHLBI Contract N02-HL-6-4278. The provision of genotyping data was supported in part by the National Center for Advancing Translational Sciences, CTSI grant UL1TR000124, and the National Institute of Diabetes and Digestive and Kidney Disease Diabetes Research Center (DRC) grant DK063491 to the Southern California Diabetes Endocrinology Research Center.</p><p id="P146">MICROS: The MICROS study was supported by the Ministry of Health and Department of Innovation, Research and University of the Autonomous Province of Bolzano, the South Tyrolean Sparkasse Foundation, and the European Union framework program 6 EUROSPAN project (contract no. LSHG-CT-2006–018947). For the MICROS study, we thank the primary care practitioners Raffaela Stocker, Stefan Waldner, Toni Pizzecco, Josef Plangger, Ugo Marcadent, and the personnel of the Hospital of Silandro (Department of Laboratory Medicine) for their participation and collaboration in the research project.</p><p id="P147">MIGEN: The MIGEN Consortium was funded by grant R01 HL087676 (NIH, USA), CIBERCV (Instituto Carlos III, Spain), and AGAUR (Generalitat de Catalunya, Spain).</p><p id="P148">NEO: The NEO study is supported by the participating Departments, the Division, and the Board of Directors of the Leiden University Medical Center and by the Leiden University, Research Profile Area Vascular and Regenerative Medicine. D.O. Mook-Kanamori is supported by Dutch Science Organization (ZonMW-VENI Grant 916.14.023). The authors of the NEO study thank all individuals who participated in the Netherlands Epidemiology in Obesity study, all participating general practitioners for inviting eligible participants, and all research nurses for collection of the data. We thank the NEO study group, Pat van Beelen, Petra Noordijk, and Ingeborg de Jonge for the coordination, laboratory, and data management of the NEO study. The genotyping in the NEO study was supported by the Centre National de Génotypage (Paris, France), headed by Jean-Francois Deleuze.</p><p id="P149">NESDA: Funding was obtained from the Netherlands Organization for Scientific Research (Geestkracht program grant 10-000-1002); the Center for Medical Systems Biology (CSMB, NOW Genomics), Biobanking and Biomolecular Resources Research Infrastructure (BBMRI-NL), VU University’s Institutes for Health and Care Research (EMGO+) and Neuroscience Campus Amsterdam, University Medical Center Groningen, Leiden University Medical Center, and National Institutes of Health (NIH, R01D0042157-01A, <a id="__tag_806140069" class="tag_hotlink" href="/nuccore/MH081802" ref="/nuccore/MH081802"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH081802","term_id":"1365466074","term_text":"MH081802"}}</span>MH081802</a>, Grand Opportunity grants 1RC2 <a id="__tag_806140074" class="tag_hotlink" href="/nuccore/MH089951" ref="/nuccore/MH089951"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH089951","term_id":"1391846823","term_text":"MH089951"}}</span>MH089951</a> and 1RC2 <a id="__tag_806140065" class="tag_hotlink" href="/nuccore/MH089995" ref="/nuccore/MH089995"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH089995","term_id":"1391846911","term_text":"MH089995"}}</span>MH089995</a>). Part of the genotyping and analyses were funded by the Genetic Association Information Network (GAIN) of the Foundation for the National Institutes of Health. Computing was supported by BiG Grid, the Dutch e-Science Grid, which is financially supported by NWO.</p><p id="P150">NSPHS: The Northern Swedish Population Health Study (NSPHS) was funded by the Swedish Medical Research Council (Project Number K2007-66X-20270-01-3, 2011–5252, 2012–2884, and 2011–2354), the Foundation for Strategic Research (SSF). NSPHS as part of EUROSPAN (European Special Populations Research Network) was also supported by the European Commission FP6 STRP grant number 01947 (LSHG-CT-2006-01947). This work has also been supported by the Swedish Society for Medical Research (SSMF) and the Swedish Medical Research Council (No. 2015–03327) We are grateful for the contribution of district nurse Svea Hennix for data collection and Inger Jonasson for logistics and coordination of the health survey. We also thank all the participants from the community for their interest and willingness to contribute to this study.</p><p id="P151">NTR: Funding was obtained from the Netherlands Organization for Scientific Research (NWO) and The Netherlands Organisation for Health Research and Development (ZonMW) grants 904-61-090, 985-10-002, 904-61-193,480-04-004, 400-05-717, Addiction-31160008, Middelgroot-911-09-032, Spinozapremie 56-464-14192, Biobanking, and Biomolecular Resources Research Infrastructure (BBMRI –NL, 184.021.007); the Netherlands Heart Foundation grants 86.083 and 88.042 and 90.313; the VU Institute for Health and Care Research (EMGO+); the European Community’s Seventh Framework Program (FP7/2007– 2013), ENGAGE (HEALTH-F4-2007–201413); the European Research Council (ERC Advanced, 230374), the Rutgers University Cell and DNA Repository (NIMH U24 MH068457-06), the Avera Institute, Sioux Falls, South Dakota (USA), and the National Institutes of Health (NIH, R01D0042157-01A, <a id="__tag_806140070" class="tag_hotlink" href="/nuccore/MH081802" ref="/nuccore/MH081802"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH081802","term_id":"1365466074","term_text":"MH081802"}}</span>MH081802</a>; Grand Opportunity grant 1RC2 <a id="__tag_806140072" class="tag_hotlink" href="/nuccore/MH089951" ref="/nuccore/MH089951"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH089951","term_id":"1391846823","term_text":"MH089951"}}</span>MH089951</a>). Part of the genotyping and analyses were funded by the Genetic Association Information Network (GAIN) of the Foundation for the National Institutes of Health. Computing was supported by BiG Grid, the Dutch e-Science Grid, which is financially supported by NWO.</p><p id="P152">ORCADES: ORCADES was supported by the Chief Scientist Office of the Scottish Government, the Royal Society, the MRC Human Genetics Unit, Arthritis Research UK, and the European Union framework program 6 EUROSPAN project (contract no. LSHG-CT-2006–018947). DNA extractions were performed at the Wellcome Trust Clinical Research Facility in Edinburgh. We acknowledge the invaluable contributions of Lorraine Anderson and the research nurses in Orkney, the administrative team in Edinburgh, and the people of Orkney.</p><p id="P153">PIVUS: This project was supported by Knut and Alice Wallenberg Foundation (Wallenberg Academy Fellow), European Research Council (ERC Starting Grant), Swedish Diabetes Foundation (grant no. 2013–024), Swedish Research Council (grant no. 2012-1397), and Swedish Heart-Lung Foundation (20120197). The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project b2011036. Genetic data analysis was funded by the Wellcome Trust under awards WT098017 and WT090532. We thank the SNP&amp;SEQ Technology Platform in Uppsala (<a href="http://www.genotyping.se" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.genotyping.se</a>) for excellent genotyping.</p><p id="P154">PROCARDIS: PROCARDIS was supported by the European Community Sixth Framework Program (LSHM-CT-2007–037273), AstraZeneca, the British Heart Foundation, the Swedish Research Council, the Knut and Alice Wallenberg Foundation, the Swedish Heart-Lung Foundation, the Torsten and Ragnar Söderberg Foundation, the Strategic Cardiovascular Program of Karolinska Institutet and Stockholm County Council, the Foundation for Strategic Research, and the Stockholm County Council (560283). M. Farrall and H. Watkins acknowledge the support of the Wellcome Trust core award (090532/Z/09/Z) and the BHF Centre of Research Excellence (RE/13/1/30181). A. Goel and H. Watkins acknowledge European Union Seventh Framework Programme FP7/2007–2013 under grant agreement no. HEALTH-F2-2013–601456 (CVGenes@Target) and A. Goel acknowledge the Wellcome Trust Institutional strategic support fund.</p><p id="P155">PROSPER: The PROSPER study was supported by an investigator-initiated grant obtained from Bristol-Myers Squibb. Dr J. W. Jukema is an Established Clinical Investigator of the Netherlands Heart Foundation (grant 2001 D 032). Support for genotyping was provided by the seventh framework program of the European commission (grant 223004) and by the Netherlands Genomics Initiative (Netherlands Consortium for Healthy Aging grant 050-060-810).</p><p id="P156">RS: The generation and management of GWAS genotype data for the Rotterdam Study (RS I, RS II, RS III) was executed by the Human Genotyping Facility of the Genetic Laboratory of the Department of Internal Medicine, Erasmus MC, Rotterdam, The Netherlands. The GWAS data sets are supported by the Netherlands Organisation of Scientific Research NWO Investments (no. 175.010.2005.011, 911-03-012), the Genetic Laboratory of the Department of Internal Medicine, Erasmus MC, the Research Institute for Diseases in the Elderly (014-93-015; RIDE2), the Netherlands Genomics Initiative (NGI)/Netherlands Organisation for Scientific Research (NWO) Netherlands Consortium for Healthy Aging (NCHA), project no. 050-060-810. The Rotterdam Study is funded by Erasmus Medical Center and Erasmus University, Rotterdam, Netherlands, Organization for the Health Research and Development (ZonMw), the Research Institute for Diseases in the Elderly (RIDE), the Ministry of Education, Culture and Science, the Ministry for Health, Welfare and Sports, the European Commission (DG XII), and the Municipality of Rotterdam. We thank Pascal Arp, Mila Jhamai, Marijn Verkerk, Lizbeth Herrera, and Marjolein Peters, MSc, and Carolina Medina-Gomez, MSc, for their help in creating the GWAS database, and Karol Estrada, PhD, Yurii Aulchenko, PhD, and Carolina Medina-Gomez, MSc, for the creation and analysis of imputed data. We are grateful to the study participants, the staff from the Rotterdam Study, and the participating general practitioners and pharmacists.</p><p id="P157">SHIP: SHIP is part of the Community Medicine Research net of the University of Greifswald, Germany, which is funded by the Federal Ministry of Education and Research (grants no. 01ZZ9603, 01ZZ0103, and 01ZZ0403), the Ministry of Cultural Affairs, as well as the Social Ministry of the Federal State of Mecklenburg-West Pomerania, and the network ‘Greifswald Approach to Individualized Medicine (GANI_MED)’ funded by the Federal Ministry of Education and Research (grant 03IS2061A). Genome-wide data have been supported by the Federal Ministry of Education and Research (grant no. 03ZIK012) and a joint grant from Siemens Healthcare, Erlangen, Germany, and the Federal State of Mecklenburg-West Pomerania. The University of Greifswald is a member of the Caché Campus program of the InterSystems GmbH.</p><p id="P158">Three City-Dijon: The 3-City Study is conducted under a partnership agreement among the Institut National de la Santé et de la Recherche Médicale (INSERM), the University of Bordeaux, and Sanofi-Aventis. The Fondation pour la Recherche Médicale funded the preparation and initiation of the study. The 3C Study is also supported by the Caisse Nationale Maladie des Travailleurs Salariés, Direction Générale de la Santé, Mutuelle Générale de l’Education Nationale (MGEN), Institut de la Longévité, Conseils Régionaux of Aquitaine and Bourgogne, Fondation de France, and Ministry of Research–INSERM Programme “Cohortes et collections de données biologiques.” This work was supported by the National Foundation for Alzheimer’s Disease and Related Disorders, the Institut Pasteur de Lille, the Centre National de Génotypage, and the LABEX (Laboratory of Excellence program investment for the future) DISTALZ—Development of Innovative Strategies for a Transdisciplinary approach to Alzheimer’s disease. G. Chauhan, C. Tzourio, and S. Debette are supported by a grant from the Fondation Leducq. We thank Philippe Amouyel and the UMR1167 Inserm Univ Lille Institut Pasteur de Lille for providing the 3C Dijon cohort SNP replication data funded by a grant from the French National Foundation on Alzheimer’s disease and related disorders.</p><p id="P159">UKHLS: These data are from Understanding Society: The UK Household Longitudinal Study, which is led by the Institute for Social and Economic Research at the University of Essex and funded by the Economic and Social Research Council. The data were collected by NatCen, and the genome-wide scan data were analyzed by the Wellcome Trust Sanger Institute. The Understanding Society DAC have an application system for genetics data, and all use of the data should be approved by them. The application form is available at <a href="https://www.understandingsociety.ac.uk/about/health/data" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.understandingsociety.ac.uk/about/health/data</a>.</p><p id="P160">TRAILS: This research is part of the Tracking Adolescents’ Individual Lives Survey (TRAILS). Participating centers of TRAILS include the University Medical Center and University of Groningen, the Erasmus University Medical Center Rotterdam, the University of Utrecht, the Radboud Medical Center Nijmegen, and the Parnassia Bavo group, all in The Netherlands. TRAILS has been financially supported by various grants from the Netherlands Organization for Scientific Research NWO (Medical Research Council program grant GB-MW 940-38-011; ZonMW Brainpower grant 100-001-004; ZonMw Risk Behavior and Dependence grants 60-60600-97-118; ZonMw Culture and Health grant 261-98-710; Social Sciences Council medium-sized investment grants GB-MaGW 480-01-006 and GB-MaGW 480-07-001; Social Sciences Council project grants GB-MaGW 452-04-314 and GB-MaGW 452-06-004; NWO large-sized investment grant 175.010.2003.005; NWO Longitudinal Survey and Panel Funding 481-08-013 and 481-11-001), the Dutch Ministry of Justice (WODC), the European Science Foundation (EuroSTRESS project FP-006), Biobanking and Biomolecular Resources Research Infrastructure BBMRI-NL (CP 32), and the participating universities. Statistical analyses were performed on the Genetic Cluster Computer (<a href="http://www.geneticcluster.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.geneticcluster.org</a>) hosted by SURFsara and financially supported by the Netherlands Scientific Organization (NWO 480-05-003 PI: Posthuma) along with a supplement from the Dutch Brain Foundation and the VU University Amsterdam.</p><p id="P161">TwinGene: This project was supported by grants from the Ministry for Higher Education, the Swedish Research Council (M-2005-1112 and 2009–2298), GenomEUtwin (EU/QLRT-2001-01254; QLG2-CT-2002-01254), NIH grant DK U01-066134, Knut and Alice Wallenberg Foundation (Wallenberg Academy Fellow), European Research Council (ERC Starting Grant), Swedish Diabetes Foundation (grant no. 2013–024), Swedish Research Council (grant no. 2012-1397), and Swedish Heart-Lung Foundation (20120197). We thank the SNP&amp;SEQ Technology Platform in Uppsala (<a href="http://www.genotyping.se" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.genotyping.se</a>) for excellent genotyping. The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project b2011036.</p><p id="P162">TwinsUK: The study was funded by the Wellcome Trust; European Community’s Seventh Framework Programme (FP7/2007–2013). The study also receives support from the National Institute for Health Research (NIHR) BioResource Clinical Research Facility and Biomedical Research Centre based at Guy’s and St Thomas’ NHS Foundation Trust and King’s College London (guysbrc-2012-1). We thank the staff from the Genotyping Facilities at the Wellcome Trust Sanger Institute for sample preparation, quality control, and genotyping; Le Centre National de Génotypage, France, for genotyping; Duke University, NC, for genotyping; and the Finnish Institute of Molecular Medicine, Finnish Genome Center, University of Helsinki. Genotyping was also done by CIDR as part of an NEI/NIH project grant.</p><p id="P163">UK Biobank_Cardiometabolic Consortium: The UKB-CMC received support from the British Heart Foundation (grant SP/SP/13/2/30111). This research has been conducted using the UK Biobank Resource under application number 236. H.R. Warren, C.P. Cabrera, and M.R. Barnes were funded by the National Institutes for Health Research (NIHR) as part of the portfolio of translational research of the NIHR Biomedical Research Unit at Barts. M. Ren was funded by the National Institute for Health Research (NIHR) Biomedical Research Unit in Cardiovascular Disease at Barts. M. Ren is recipient from China Scholarship Council (No. 2011632047). B. Mifsud holds an MRC eMedLab Medical Bioinformatics Career Development Fellowship, funded from award MR/L016311/1. P. Elliott was funded by the National Institutes for Health Research (NIHR) Imperial College Health Care NHS Trust and Imperial College London Biomedical Research Centre, the UK Medical Research Council and Public Health England as Director of the MRC-PHE Centre for Environment and Health, and the NIHR Health Protection Research Unit on the Health Effects of Environmental Hazards. Some of this work used computing resources provided by the Medical Research Council–funded UK MEDical Bioinformatics partnership programme (UK MED-BIO) (MR/L01632X/1)</p><p id="P164">ULSAM: This project was supported by Knut and Alice Wallenberg Foundation (Wallenberg Academy Fellow), European Research Council (ERC Starting Grant), Swedish Diabetes Foundation (grant no. 2013–024), Swedish Research Council (grant no. 2012-1397), and Swedish Heart-Lung Foundation (20120197). The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project b2011036. Genotyping was funded by the Wellcome Trust under award WT064890. Analysis of genetic data was funded by the Wellcome Trust under awards WT098017 and WT090532. A.P. Morris is a Wellcome Trust Senior Research Fellow in Basic Biomedical Science (WT098017). We thank the SNP&amp;SEQ Technology Platform in Uppsala (<a href="http://www.genotyping.se" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.genotyping.se</a>) for excellent genotyping.</p><p id="P165">WGHS: The WGHS is supported by the National Heart, Lung, and Blood Institute (<a id="__tag_806140067" class="tag_hotlink" href="/nuccore/HL043851" ref="/nuccore/HL043851"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL043851","term_id":"1051491008","term_text":"HL043851"}}</span>HL043851</a>, <a id="__tag_806140068" class="tag_hotlink" href="/nuccore/HL080467" ref="/nuccore/HL080467"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL080467","term_id":"1051650875","term_text":"HL080467"}}</span>HL080467</a>, HL09935) and the National Cancer Institute (<a id="__tag_806140066" class="tag_hotlink" href="/nuccore/CA047988" ref="/nuccore/CA047988"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA047988","term_id":"24354158","term_text":"CA047988"}}</span>CA047988</a> and UM1CA182913) with collaborative scientific support and funding for genotyping provided by Amgen.</p><p id="P166">YFS: The Young Finns Study has been financially supported by the Academy of Finland: grants 286284, 134309 (Eye), 126925, 121584, 124282, 129378 (Salve), 117787 (Gendi), and 41071 (Skidi); the Social Insurance Institution of Finland; Kuopio, Tampere, and Turku University Hospital Medical Funds (grant X51001); Juho Vainio Foundation; Paavo Nurmi Foundation; Finnish Foundation for Cardiovascular Research; Finnish Cultural Foundation; Tampere Tuberculosis Foundation; Emil Aaltonen Foundation; Yrjö Jahnsson Foundation; Signe and Ane Gyllenberg Foundation; and Diabetes Research Foundation of Finnish Diabetes Association. The expert technical assistance in the statistical analyses by Irina Lisinen is gratefully acknowledged.</p></div></div><div id="idm140304098893760" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140304098893760title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="P31" class="p p-first-last">The <a href="#SD1" rid="SD1" class=" supplementary-material">online-only Data Supplement</a> is available with this article at <a href="http://hyper.ahajournals.org/lookup/suppl/doi:10.1161/HYPERTENSIONAHA.117.09438/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hyper.ahajournals.org/lookup/suppl/doi:10.1161/HYPERTENSIONAHA.117.09438/-/DC1</a>.</p><p></p><div><p id="P32" class="p p-first"><strong>Author Contributions</strong></p><strong>Secondary Analyses</strong><p id="P34" class="p">Design of secondary analyses: L.V. Wain, G.B. Ehret, M.J. Caulfield, H. Snieder, M.D. Tobin, R. Joehanes, A. Vaez, R. Jansen, A.V. Smith, J. Knight, P.F. O’Reilly, A.P. Morris, and C.P. Cabrera. Computation of secondary analysis: L.V. Wain, G.B. Ehret, A.P. Morris, A.M. Erzurumluoglu, T. Blake, L. Lin, R. Joehanes, A. Vaez, P.J. van der Most, R. Jansen, and C.P. Cabrera.</p><p id="P35" class="p"><strong>Discovery</strong></p><p id="P36" class="p">WGHS: Study phenotyping, P.M. Ridker; Genotyping or analysis, D.I. Chasman and L.M. Rose; Study PI, D.I. Chasman and P.M. Ridker.</p><p id="P37" class="p">RS: Study phenotyping, G.C. Verwoert; Genotyping or analysis, G.C. Verwoert and A.G. Uitterlinden; Study PI, O.H. Franco, A. Hofman, and A.G. Uitterlinden.</p><p id="P38" class="p">NTR: Study phenotyping, E.J. de Geus and G. Willemsen; Genotyping or analysis: J.-J. Hottenga, E.J. de Geus, and G. Willemsen; Study PI, D.I. Boomsma and E.J. de Geus.</p><p id="P39" class="p">STR: Study phenotyping, E. Ingelsson; Genotyping or analysis, R.J. Strawbridge and M. Frånberg; Study PI, E. Ingelsson and A. Hamsten.</p><p id="P40" class="p">EGCUT: Genotyping or analysis, T. Esko; Study PI, A. Metspalu.</p><p id="P41" class="p">ARIC: Genotyping or analysis, D.E. Arking, A.C. Morrison, and P. Nandakumar; Study PI, A. Chakravarti.</p><p id="P42" class="p">FHS: Study phenotyping, D. Levy; Genotyping or analysis, S.-J. Hwang; Study PI: D. Levy.</p><p id="P43" class="p">MESA: Study phenotyping, J.I. Rotter; Genotyping or analysis, W. Palmas, X. Guo, J.I. Rotter, J. Yao; Study PI, W. Palmas.</p><p id="P44" class="p">B58C: Study phenotyping, D.P. Strachan; Genotyping or analysis, D.P. Strachan; Study PI, D.P. Strachan.</p><p id="P45" class="p">COLAUS: Study phenotyping, P. Vollenweider; Genotyping or analysis, M. Bochud and Z. Kutalik; Study PI, P. Vollenweider.</p><p id="P46" class="p">PROSPER: Study phenotyping, J.W. Jukema and D.J. Stott; Genotyping or analysis, S. Trompet and J. Deelen; Study PI, J.W. Jukema.</p><p id="P47" class="p">BHS: Study phenotyping, A. James; Genotyping or analysis, N. Shrine, J. Hui, and J. Beilby.</p><p id="P48" class="p">SHIP: Study phenotyping, M. Dörr; Genotyping or analysis, A. Teumer, M. Dörr, and U. Völker; Study PI, R. Rettig.</p><p id="P49" class="p">KORA S4: Genotyping or analysis, J.S. Ried; Study PI, A. Peters.</p><p id="P50" class="p">CHS: Study phenotyping, B.M. Psaty; Genotyping or analysis, J.C. Bis, K. Rice, and K.D. Taylor; Study PI, B.M. Psaty.</p><p id="P51" class="p">AGES-Reykjavik: Genotyping or analysis, A.V. Smith; Study PI, V. Gudnason, T.B. Harris, and L.J. Launer.</p><p id="P52" class="p">ERF: Study phenotyping, C.M. van Duijn and B.A. Oostra; Genotyping or analysis, N. Amin; Study PI, C.M. van Duijn and B.A. Oostra.</p><p id="P53" class="p">NESDA: Study phenotyping, B.W.J.H. Penninx; Genotyping or analysis, I.M. Nolte and Y. Milaneschi; Study PI, H. Snieder and B.W.J.H. Penninx.</p><p id="P54" class="p">YFS: Study phenotyping, T. Lehtimäki, M. Kähönen, and O.T. Raitakari; Genotyping or analysis, T. Lehtimäki, L.-P. Lyytikäinen, M. Kähönen, and O.T. Raitakari; Study PI, T. Lehtimäki, M. Kähönen, and O.T. Raitakari.</p><p id="P55" class="p">EPIC: Genotyping or analysis, N.J. Wareham; Study PI, J.H. Zhao.</p><p id="P56" class="p">ASPS: Study phenotyping, R. Schmidt; Genotyping or analysis, H. Schmidt, E. Hofer, Y. Saba, and R. Schmidt; Study PI, H. Schmidt and R. Schmidt.</p><p id="P57" class="p">ORCADES: Study phenotyping, J.F. Wilson, H. Campbell, and S. Wild; Genotyping or analysis, J.F. Wilson, P.K. Joshi, and S. Wild; Study PI, J.F. Wilson.</p><p id="P58" class="p">FINRISK (COROGENE_CTRL): Study phenotyping, P. Jousilahti; Genotyping or analysis, K. Kristiansson and A.P. Sarin; Study PI, M. Perola and P. Jousilahti.</p><p id="P59" class="p">INGI-VB: Study phenotyping, C.F. Sala; Genotyping or analysis, M. Traglia, C.M. Barbieri, and C.F. Sala; Study PI, D. Toniolo.</p><p id="P60" class="p">FINRISK_PREDICT_CVD: Study phenotyping, V. Salomaa and A.S. Havulinna; Study PI, V. Salomaa, A. Palotie, and S. Ripatti.</p><p id="P61" class="p">TRAILS: Study phenotyping, H. Riese; Genotyping or analysis, P.J. van der Most; Study PI, C.A. Hartman and A.J. Oldehinkel.</p><p id="P62" class="p">PROCARDIS: Study phenotyping, A. Goel; Genotyping or analysis, A. Goel; Study PI, H. Watkins, and M. Farrall.</p><p id="P63" class="p">HABC: Study phenotyping, Y. Liu and T.B. Harris; Genotyping or analysis, M.A. Nalls; Study PI, Y. Liu and T.B. Harris.</p><p id="P64" class="p">KORA S3: Study phenotyping, C. Gieger; Genotyping or analysis, S. Sõber, C. Gieger, and E. Org. Study PI, M. Laan.</p><p id="P65" class="p">INGI-FVG: Genotyping or analysis, D. Vuckovic, M. Brumat, and M. Cocca; Study PI, P. Gasparini.</p><p id="P66" class="p">Fenland: Study phenotyping, R.A. Scott, J. Luan, C. Langenberg, and N.J. Wareham; Genotyping or analysis, R.A. Scott, J. Luan, C. Langenberg, and N.J. Wareham; Study PI, R.A. Scott, C. Langenberg, and N.J. Wareham.</p><p id="P67" class="p">MICROS: Genotyping or analysis, A.A. Hicks, F. Del Greco M., and A. Saint Pierre; Study PI, F. Del Greco M. and P.P. Pramstaller.</p><p id="P68" class="p">HTO: Study phenotyping, B.D. Keavney; Genotyping or analysis, B.D. Keavney, K.L. Ayers, and C. Mamasoula; Study PI, B.D. Keavney and H.J. Cordell.</p><p id="P69" class="p">MIGEN: Study phenotyping, R. Elosua, J. Marrugat, S. Kathiresan, and D. Siscovick; Genotyping or analysis, R. Elosua, S. Kathiresan, and D. Siscovick; Study PI, S. Kathiresan.</p><p id="P70" class="p">ULSAM: Study phenotyping, V. Giedraitis and E. Ingelsson; Genotyping or analysis, A.P. Morris and A. Mahajan; Study PI, A.P. Morris, V. Giedraitis, and E. Ingelsson.</p><p id="P71" class="p">Cilento study: Study phenotyping, R. Sorice; Genotyping or analysis, D. Ruggiero, and T. Nutile; Study PI, M. Ciullo.</p><p id="P72" class="p">LBC1936: Study phenotyping, I.J. Deary and A.J. Gow; Genotyping or analysis, L.M. Lopez, G. Davies, and A.J. Gow; Study PI, I.J. Deary.</p><p id="P73" class="p">H2000_CTRL: Study phenotyping, T. Niiranen; Study PI, P. Knekt, A. Jula, and S. Koskinen.</p><p id="P74" class="p">NSPHS: Genotyping or analysis, S. Enroth and Å. Johansson; Study PI, U. Gyllensten.</p><p id="P75" class="p">FUSION: Genotyping or analysis, A.U. Jackson; Study PI, J. Tuomilehto, M. Boehnke, and F. Collins.</p><p id="P76" class="p">GRAPHIC: Study phenotyping, N.J. Samani, P.S. Braund, and M.D. Tobin. Genotyping or analysis: C.P. Nelson, P.S. Braund, and M.D. Tobin; Study PI, N.J. Samani.</p><p id="P77" class="p">CROATIA_Vis: Study phenotyping, I. Rudan; Genotyping or analysis, V. Vitart and J.E. Huffman; Study PI, V. Vitart and I. Rudan.</p><p id="P78" class="p">PIVUS: Study phenotyping, L. Lind and J. Sundström; Genotyping or analysis, C.M. Lindgren and A. Mahajan; Study PI, C.M. Lindgren, L. Lind, and J. Sundström.</p><p id="P79" class="p">LOLIPOP: Study phenotyping, J.S. Kooner and J.C. Chambers; Genotyping or analysis, J.S. Kooner, W. Zhang, J.C. Chambers, and B. Lehne; Study PI, J.S. Kooner and J.C. Chambers.</p><p id="P80" class="p">CROATIA_Korcula: Genotyping or analysis, C. Hayward and J. Marten; Study PI, C. Hayward and A.F. Wright.</p><p id="P81" class="p">INGI-CARL: Study phenotyping, G. Girotto; Genotyping or analysis, I. Gandin, A. Morgan, and A. Robino.</p><p id="P82" class="p">LBC1921: Study phenotyping, J.M. Starr and A. Pattie; Genotyping or analysis, J.M. Starr, S.E. Harris, D.C.M. Liewald, and A. Pattie; Study PI, J.M. Starr.</p><p id="P83" class="p">CROATIA_SPLIT: Study phenotyping, O. Polasek and I. Kolcic; Genotyping or analysis, O. Polasek and T. Zemunik; Study PI, O. Polasek.</p><p id="P84" class="p">BioMe (formerly IPM): Genotyping or analysis, Y. Lu; Study PI, R.J.F. Loos and E.P. Bottinger.</p><p id="P85" class="p"><strong>Replication</strong></p><p id="P86" class="p">UKB-BP: Genotyping or analysis, H.R. Warren, M.R. Barnes, C.P. Cabrera, E. Evangelou, H. Gao, B. Mifsud, M. Ren, and I. Tzoulaki; Study PI, P. Elliott and M.J. Caulfield.</p><p id="P87" class="p">GoDARTS: Study phenotyping, C.N.A. Palmer and A.S.F. Doney; Genotyping or analysis, C.N.A. Palmer and N. Shah; Study PI, C.N.A. Palmer and A.D. Morris.</p><p id="P88" class="p">Lifelines: Study phenotyping: M.H. de Borst; Genotyping or analysis, M. Swertz; Study PI, P. van der Harst.</p><p id="P89" class="p">TwinsUK: Study phenotyping, C. Menni; Genotyping or analysis, M. Mangino and C. Menni; Study PI, T.D. Spector.</p><p id="P90" class="p">Airwave Health Monitoring Study: Genotyping or analysis, A.C. Vergnaud, E. Evangelou, H. Gao, and I. Tzoulaki; Study PI, E. Evangelou.</p><p id="P91" class="p">The UK Household Longitudinal Study (UKHLS): Genotyping or analysis, B.P. Prins; Study PI, E. Zeggini.</p><p id="P92" class="p">Generation Scotland (GS:SFHS): Study phenotyping, S. Padmanabhan; Genotyping or analysis, C. Hayward and A. Campbell.</p><p id="P93" class="p">JUPITER: Study phenotyping, P.M. Ridker; Genotyping or analysis, D.I. Chasman, L.M. Rose, F. Giulianini, and P.M. Ridker; Study PI, D.I. Chasman and P.M. Ridker.</p><p id="P94" class="p">NEO: Study phenotyping, R. de Mutsert; Genotyping or analysis, D.O. Mook-Kanamori and R. Li-Gao; Study PI, R. de Mutsert.</p><p id="P95" class="p">Three City-Dijon: Study phenotyping, S. Debette and C. Tzourio; Genotyping or analysis, G. Chauhan; Study PI, S. Debette and C. Tzourio.</p><p id="P96" class="p">ASCOT-UK: Study phenotyping: P. Sever and N. Poulter; Genotyping or analysis, P.B. Munroe and H.R. Warren; Study PI, P.B. Munroe, P. Sever, N. Poulter, and M.J. Caulfield.</p><p id="P97" class="p">ASCOT-SC: Study phenotyping, S. Thom and M.J. Caulfield; Genotyping or analysis, D.C. Shields, A. Stanton, H.R. Warren, and P.B. Munroe; Study PI, S. Thom, M.J. Caulfield, and P.B. Munroe.</p><p id="P98" class="p">Hunter Community Study: Study phenotyping, R. Scott; Genotyping or analysis, C. Oldmeadow and E.G. Holliday; Study PI, J. Attia.</p><p id="P99" class="p">GAPP: Study phenotyping, D. Conen; Genotyping or analysis, D. Conen, S. Thériault, and G. Paré; Study PI, D. Conen.</p><p id="P100" class="p">BRIGHT: Study phenotyping, M. Brown and J. Connell; Genotyping or analysis, M. Farrall, P.B. Munroe, and H.R. Warren; Study PI, M. Brown, J. Connell, M. Farrall, P.B. Munroe, and M.J. Caulfield.</p><p id="P101" class="p"><strong>Resources for Secondary Analyses</strong></p><p id="P102" class="p">eQTL NESDA NTR: Design of secondary analysis, R. Jansen; Computation of secondary analysis, D.I. Boomsma, R. Jansen, and B.W.J.H. Penninx; Study PI, D.I. Boomsma and B.W.J.H. Penninx.</p><p id="P103" class="p">eQTL kidney: Study phenotyping, J.J. Damman and M.A. Seelen; Genotyping or analysis, P.J. van der Most; Study PI, H. Snieder.</p><p id="P104" class="p">eQTL BIOS: Design of secondary analysis, R. Jansen; Computation of secondary analysis, R. Jansen; Study PI, R. Jansen.</p><p id="P105" class="p">SABRe: Study phenotyping, Y. Demirkale, P.J. Munson, and Q.T. Nguyen; Genotyping or analysis, R. Joehanes; Design of secondary analysis, D. Levy; Study PI, D. Levy.</p><p id="P106" class="p"><strong>ICBP-Steering Committee</strong></p><p id="P107" class="p p-last">G. Abecasis, M.J. Caulfield, A. Chakravarti, D.I. Chasman, G.B. Ehret, P. Elliott, T. Ferreira, M.-R. Jarvelin, A.D. Johnson, M. Larson, D. Levy, A.P. Morris, P.B. Munroe, C. Newton-Cheh, P.F. O’Reilly, W. Palmas, B.M. Psaty, K. Rice, A.V. Smith, H. Snieder, M.D. Tobin, C.M. van Duijn, L.V. Wain, H.R. Warren.</p></div><p class="fn sec" id="FN6"></p><p id="P108" class="p p-first"><strong>Disclosures</strong></p><p id="P109" class="p p-last">We declare competing financial interests (see corresponding section in the <a href="#SD1" rid="SD1" class=" supplementary-material">online-only Data Supplement</a>).</p><p></p></div></div><div id="idm140304095893488" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140304095893488title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Nelson MR, Tipney H, Painter JL, Shen J, Nicoletti P, Shen Y, Floratos A, Sham PC, Li MJ, Wang J, Cardon LR, Whittaker JC, Sanseau P. The support of human genetic evidence for approved drug indications. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:856–860. doi: 10.1038/ng.3314.</span> [<a href="/pubmed/26121088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3314" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+support+of+human+genetic+evidence+for+approved+drug+indications&amp;author=MR+Nelson&amp;author=H+Tipney&amp;author=JL+Painter&amp;author=J+Shen&amp;author=P+Nicoletti&amp;volume=47&amp;publication_year=2015&amp;pages=856-860&amp;pmid=26121088&amp;doi=10.1038/ng.3314&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Mancia G, Fagard R, Narkiewicz K, et al.  2013 ESH/ESC guidelines for the management of arterial hypertension: the Task Force for the Management of Arterial Hypertension of the European Society of Hypertension (ESH) and of the European Society of Cardiology (ESC) <span><span class="ref-journal">Eur Heart J. </span>2013;<span class="ref-vol">34</span>:2159–2219. doi: 10.1093/eurheartj/eht151.</span> [<a href="/pubmed/23771844" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Feurheartj%2Feht151" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+Heart+J&amp;title=2013+ESH/ESC+guidelines+for+the+management+of+arterial+hypertension:+the+Task+Force+for+the+Management+of+Arterial+Hypertension+of+the+European+Society+of+Hypertension+(ESH)+and+of+the+European+Society+of+Cardiology+(ESC)&amp;author=G+Mancia&amp;author=R+Fagard&amp;author=K+Narkiewicz&amp;volume=34&amp;publication_year=2013&amp;pages=2159-2219&amp;pmid=23771844&amp;doi=10.1093/eurheartj/eht151&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Ehret GB, Munroe PB, Rice KM, et al.  Genetic variants in novel pathways influence blood pressure and cardiovascular disease risk. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">478</span>:103–109.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3340926/">PMC free article</a>]</span> [<a href="/pubmed/21909115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+variants+in+novel+pathways+influence+blood+pressure+and+cardiovascular+disease+risk&amp;author=GB+Ehret&amp;author=PB+Munroe&amp;author=KM+Rice&amp;volume=478&amp;publication_year=2011&amp;pages=103-109&amp;pmid=21909115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Ganesh SK, Chasman DI, Larson MG, et al.  Global Blood Pressure Genetics Consortium. Effects of long-term averaging of quantitative blood pressure traits on the detection of genetic associations. <span><span class="ref-journal">Am J Hum Genet. </span>2014;<span class="ref-vol">95</span>:49–65. doi: 10.1016/j.ajhg.2014.06.002.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4085637/">PMC free article</a>]</span> [<a href="/pubmed/24975945" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ajhg.2014.06.002" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Global+Blood+Pressure+Genetics+Consortium.+Effects+of+long-term+averaging+of+quantitative+blood+pressure+traits+on+the+detection+of+genetic+associations&amp;author=SK+Ganesh&amp;author=DI+Chasman&amp;author=MG+Larson&amp;volume=95&amp;publication_year=2014&amp;pages=49-65&amp;pmid=24975945&amp;doi=10.1016/j.ajhg.2014.06.002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Johnson AD, Newton-Cheh C, Chasman DI, et al.  Cohorts for Heart and Aging Research in Genomic Epidemiology Consortium; Global BPgen Consortium; Women’s Genome Health Study. Association of hypertension drug target genes with blood pressure and hypertension in 86,588 individuals. <span><span class="ref-journal">Hypertension. </span>2011;<span class="ref-vol">57</span>:903–910. doi: 10.1161/HYPERTENSIONAHA.110.158667.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3099407/">PMC free article</a>]</span> [<a href="/pubmed/21444836" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1161%2FHYPERTENSIONAHA.110.158667" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertension&amp;title=Cohorts+for+Heart+and+Aging+Research+in+Genomic+Epidemiology+Consortium;+Global+BPgen+Consortium;+Women’s+Genome+Health+Study.+Association+of+hypertension+drug+target+genes+with+blood+pressure+and+hypertension+in+86,588+individuals&amp;author=AD+Johnson&amp;author=C+Newton-Cheh&amp;author=DI+Chasman&amp;volume=57&amp;publication_year=2011&amp;pages=903-910&amp;pmid=21444836&amp;doi=10.1161/HYPERTENSIONAHA.110.158667&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Johnson T, Gaunt TR, Newhouse SJ, et al.  Cardiogenics Consortium; Global BPgen Consortium. Blood pressure loci identified with a gene-centric array. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">89</span>:688–700. doi: 10.1016/j.ajhg.2011.10.013.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3234370/">PMC free article</a>]</span> [<a href="/pubmed/22100073" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ajhg.2011.10.013" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Cardiogenics+Consortium;+Global+BPgen+Consortium.+Blood+pressure+loci+identified+with+a+gene-centric+array&amp;author=T+Johnson&amp;author=TR+Gaunt&amp;author=SJ+Newhouse&amp;volume=89&amp;publication_year=2011&amp;pages=688-700&amp;pmid=22100073&amp;doi=10.1016/j.ajhg.2011.10.013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Kato N, Takeuchi F, Tabara Y, et al.  Meta-analysis of genome-wide association studies identifies common variants associated with blood pressure variation in east Asians. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:531–538. doi: 10.1038/ng.834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3158568/">PMC free article</a>]</span> [<a href="/pubmed/21572416" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.834" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+studies+identifies+common+variants+associated+with+blood+pressure+variation+in+east+Asians&amp;author=N+Kato&amp;author=F+Takeuchi&amp;author=Y+Tabara&amp;volume=43&amp;publication_year=2011&amp;pages=531-538&amp;pmid=21572416&amp;doi=10.1038/ng.834&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Levy D, Ehret GB, Rice K, et al.  Genome-wide association study of blood pressure and hypertension. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:677–687. doi: 10.1038/ng.384.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2998712/">PMC free article</a>]</span> [<a href="/pubmed/19430479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.384" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+blood+pressure+and+hypertension&amp;author=D+Levy&amp;author=GB+Ehret&amp;author=K+Rice&amp;volume=41&amp;publication_year=2009&amp;pages=677-687&amp;pmid=19430479&amp;doi=10.1038/ng.384&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Newton-Cheh C, Johnson T, Gateva V, et al.  Wellcome Trust Case Control Consortium. Genome-wide association study identifies eight loci associated with blood pressure. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:666–676. doi: 10.1038/ng.361.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2891673/">PMC free article</a>]</span> [<a href="/pubmed/19430483" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.361" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Wellcome+Trust+Case+Control+Consortium.+Genome-wide+association+study+identifies+eight+loci+associated+with+blood+pressure&amp;author=C+Newton-Cheh&amp;author=T+Johnson&amp;author=V+Gateva&amp;volume=41&amp;publication_year=2009&amp;pages=666-676&amp;pmid=19430483&amp;doi=10.1038/ng.361&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Newton-Cheh C, Larson MG, Vasan RS, et al.  Association of common variants in NPPA and NPPB with circulating natriuretic peptides and blood pressure. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:348–353. doi: 10.1038/ng.328.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2664511/">PMC free article</a>]</span> [<a href="/pubmed/19219041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.328" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Association+of+common+variants+in+NPPA+and+NPPB+with+circulating+natriuretic+peptides+and+blood+pressure&amp;author=C+Newton-Cheh&amp;author=MG+Larson&amp;author=RS+Vasan&amp;volume=41&amp;publication_year=2009&amp;pages=348-353&amp;pmid=19219041&amp;doi=10.1038/ng.328&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Padmanabhan S, Melander O, Johnson T, et al.  Global BPgen Consortium. Genome-wide association study of blood pressure extremes identifies variant near UMOD associated with hypertension. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1001177. doi: 10.1371/journal.pgen.1001177.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2965757/">PMC free article</a>]</span> [<a href="/pubmed/21082022" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pgen.1001177" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Global+BPgen+Consortium.+Genome-wide+association+study+of+blood+pressure+extremes+identifies+variant+near+UMOD+associated+with+hypertension&amp;author=S+Padmanabhan&amp;author=O+Melander&amp;author=T+Johnson&amp;volume=6&amp;publication_year=2010&amp;pages=e1001177&amp;pmid=21082022&amp;doi=10.1371/journal.pgen.1001177&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Simino J, Shi G, Bis JC, et al.  LifeLines Cohort Study. Gene-age interactions in blood pressure regulation: a large-scale investigation with the CHARGE, Global BPgen, and ICBP Consortia. <span><span class="ref-journal">Am J Hum Genet. </span>2014;<span class="ref-vol">95</span>:24–38. doi: 10.1016/j.ajhg.2014.05.010.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4085636/">PMC free article</a>]</span> [<a href="/pubmed/24954895" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ajhg.2014.05.010" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=LifeLines+Cohort+Study.+Gene-age+interactions+in+blood+pressure+regulation:+a+large-scale+investigation+with+the+CHARGE,+Global+BPgen,+and+ICBP+Consortia&amp;author=J+Simino&amp;author=G+Shi&amp;author=JC+Bis&amp;volume=95&amp;publication_year=2014&amp;pages=24-38&amp;pmid=24954895&amp;doi=10.1016/j.ajhg.2014.05.010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Tragante V, Barnes MR, Ganesh SK, et al.  Gene-centric meta-analysis in 87,736 individuals of European ancestry identifies multiple blood-pressure-related loci. <span><span class="ref-journal">Am J Hum Genet. </span>2014;<span class="ref-vol">94</span>:349–360. doi: 10.1016/j.ajhg.2013.12.016.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3951943/">PMC free article</a>]</span> [<a href="/pubmed/24560520" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ajhg.2013.12.016" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Gene-centric+meta-analysis+in+87,736+individuals+of+European+ancestry+identifies+multiple+blood-pressure-related+loci&amp;author=V+Tragante&amp;author=MR+Barnes&amp;author=SK+Ganesh&amp;volume=94&amp;publication_year=2014&amp;pages=349-360&amp;pmid=24560520&amp;doi=10.1016/j.ajhg.2013.12.016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Wain LV, Verwoert GC, O’Reilly PF, et al.  LifeLines Cohort Study; EchoGen consortium; AortaGen Consortium; CHARGE Consortium Heart Failure Working Group; KidneyGen consortium; CKDGen consortium; Cardiogenics consortium; CardioGram. Genome-wide association study identifies six new loci influencing pulse pressure and mean arterial pressure. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:1005–1011. doi: 10.1038/ng.922.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3445021/">PMC free article</a>]</span> [<a href="/pubmed/21909110" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.922" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=LifeLines+Cohort+Study;+EchoGen+consortium;+AortaGen+Consortium;+CHARGE+Consortium+Heart+Failure+Working+Group;+KidneyGen+consortium;+CKDGen+consortium;+Cardiogenics+consortium;+CardioGram.+Genome-wide+association+study+identifies+six+new+loci+influencing+pulse+pressure+and+mean+arterial+pressure&amp;author=LV+Wain&amp;author=GC+Verwoert&amp;author=PF+O’Reilly&amp;volume=43&amp;publication_year=2011&amp;pages=1005-1011&amp;pmid=21909110&amp;doi=10.1038/ng.922&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Ehret GB, Ferreira T, Chasman DI, et al.  CHARGE-EchoGen Consortium; CHARGE-HF Consortium; Wellcome Trust Case Control Consortium. The genetics of blood pressure regulation and its target organs from association studies in 342,415 individuals. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:1171–1184. doi: 10.1038/ng.3667.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5042863/">PMC free article</a>]</span> [<a href="/pubmed/27618452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3667" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=CHARGE-EchoGen+Consortium;+CHARGE-HF+Consortium;+Wellcome+Trust+Case+Control+Consortium.+The+genetics+of+blood+pressure+regulation+and+its+target+organs+from+association+studies+in+342,415+individuals&amp;author=GB+Ehret&amp;author=T+Ferreira&amp;author=DI+Chasman&amp;volume=48&amp;publication_year=2016&amp;pages=1171-1184&amp;pmid=27618452&amp;doi=10.1038/ng.3667&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Liu C, Kraja AT, Smith JA, et al.  CHD Exome+ Consortium; ExomeBP Consortium; GoT2DGenes Consortium; T2D-GENES Consortium; Myocardial Infarction Genetics and CARDIoGRAM Exome Consortia; CKDGen Consortium. Meta-analysis identifies common and rare variants influencing blood pressure and overlapping with metabolic trait loci. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:1162–1170. doi: 10.1038/ng.3660.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5320952/">PMC free article</a>]</span> [<a href="/pubmed/27618448" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3660" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=CHD+Exome++Consortium;+ExomeBP+Consortium;+GoT2DGenes+Consortium;+T2D-GENES+Consortium;+Myocardial+Infarction+Genetics+and+CARDIoGRAM+Exome+Consortia;+CKDGen+Consortium.+Meta-analysis+identifies+common+and+rare+variants+influencing+blood+pressure+and+overlapping+with+metabolic+trait+loci&amp;author=C+Liu&amp;author=AT+Kraja&amp;author=JA+Smith&amp;volume=48&amp;publication_year=2016&amp;pages=1162-1170&amp;pmid=27618448&amp;doi=10.1038/ng.3660&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Hoffmann TJ, Ehret GB, Nandakumar P, Ranatunga D, Schaefer C, Kwok PY, Iribarren C, Chakravarti A, Risch N. Genome-wide association analyses using electronic health records identify new loci influencing blood pressure variation. <span><span class="ref-journal">Nat Genet. </span>2017;<span class="ref-vol">49</span>:54–64. doi: 10.1038/ng.3715.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5370207/">PMC free article</a>]</span> [<a href="/pubmed/27841878" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3715" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analyses+using+electronic+health+records+identify+new+loci+influencing+blood+pressure+variation&amp;author=TJ+Hoffmann&amp;author=GB+Ehret&amp;author=P+Nandakumar&amp;author=D+Ranatunga&amp;author=C+Schaefer&amp;volume=49&amp;publication_year=2017&amp;pages=54-64&amp;pmid=27841878&amp;doi=10.1038/ng.3715&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Surendran P, Drenos F, Young R, et al.  CHARGE-Heart Failure Consortium; EchoGen Consortium; METASTROKE Consortium; GIANT Consortium; EPIC-InterAct Consortium; Lifelines Cohort Study; Wellcome Trust Case Control Consortium; Understanding Society Scientific Group; EPIC-CVD Consortium; CHARGE+ Exome Chip Blood Pressure Consortium; T2DGENES Consortium; GoT2DGenes Consortium; ExomeBP Consortium; CHD Exome+ Consortium. Trans-ancestry meta-analyses identify rare and common variants associated with blood pressure and hypertension. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:1151–1161. doi: 10.1038/ng.3654.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5056636/">PMC free article</a>]</span> [<a href="/pubmed/27618447" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3654" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=CHARGE-Heart+Failure+Consortium;+EchoGen+Consortium;+METASTROKE+Consortium;+GIANT+Consortium;+EPIC-InterAct+Consortium;+Lifelines+Cohort+Study;+Wellcome+Trust+Case+Control+Consortium;+Understanding+Society+Scientific+Group;+EPIC-CVD+Consortium;+CHARGE++Exome+Chip+Blood+Pressure+Consortium;+T2DGENES+Consortium;+GoT2DGenes+Consortium;+ExomeBP+Consortium;+CHD+Exome++Consortium.+Trans-ancestry+meta-analyses+identify+rare+and+common+variants+associated+with+blood+pressure+and+hypertension&amp;author=P+Surendran&amp;author=F+Drenos&amp;author=R+Young&amp;volume=48&amp;publication_year=2016&amp;pages=1151-1161&amp;pmid=27618447&amp;doi=10.1038/ng.3654&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Warren HR, Evangelou E, Cabrera CP, et al.  International Consortium of Blood Pressure (ICBP) 1000G Analyses; BIOS Consortium; Lifelines Cohort Study; Understanding Society Scientific group; CHD Exome+ Consortium; ExomeBP Consortium; T2D-GENES Consortium; GoT2DGenes Consortium; Cohorts for Heart and Ageing Research in Genome Epidemiology (CHARGE) BP Exome Consortium; International Genomics of Blood Pressure (iGEN-BP) Consortium; UK Biobank CardioMetabolic Consortium BP working group. Genome-wide association analysis identifies novel blood pressure loci and offers biological insights into cardiovascular risk. <span><span class="ref-journal">Nat Genet. </span>2017;<span class="ref-vol">49</span>:403–415. doi: 10.1038/ng.3768.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5972004/">PMC free article</a>]</span> [<a href="/pubmed/28135244" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3768" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=International+Consortium+of+Blood+Pressure+(ICBP)+1000G+Analyses;+BIOS+Consortium;+Lifelines+Cohort+Study;+Understanding+Society+Scientific+group;+CHD+Exome++Consortium;+ExomeBP+Consortium;+T2D-GENES+Consortium;+GoT2DGenes+Consortium;+Cohorts+for+Heart+and+Ageing+Research+in+Genome+Epidemiology+(CHARGE)+BP+Exome+Consortium;+International+Genomics+of+Blood+Pressure+(iGEN-BP)+Consortium;+UK+Biobank+CardioMetabolic+Consortium+BP+working+group.+Genome-wide+association+analysis+identifies+novel+blood+pressure+loci+and+offers+biological+insights+into+cardiovascular+risk&amp;author=HR+Warren&amp;author=E+Evangelou&amp;author=CP+Cabrera&amp;volume=49&amp;publication_year=2017&amp;pages=403-415&amp;pmid=28135244&amp;doi=10.1038/ng.3768&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Safar ME, Nilsson PM, Blacher J, Mimran A. Pulse pressure, arterial stiffness, and end-organ damage. <span><span class="ref-journal">Curr Hypertens Rep. </span>2012;<span class="ref-vol">14</span>:339–344. doi: 10.1007/s11906-012-0272-9.</span> [<a href="/pubmed/22555981" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1007%2Fs11906-012-0272-9" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Hypertens+Rep&amp;title=Pulse+pressure,+arterial+stiffness,+and+end-organ+damage&amp;author=ME+Safar&amp;author=PM+Nilsson&amp;author=J+Blacher&amp;author=A+Mimran&amp;volume=14&amp;publication_year=2012&amp;pages=339-344&amp;pmid=22555981&amp;doi=10.1007/s11906-012-0272-9&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Tobin MD, Sheehan NA, Scurrah KJ, Burton PR. Adjusting for treatment effects in studies of quantitative traits: antihypertensive therapy and systolic blood pressure. <span><span class="ref-journal">Stat Med. </span>2005;<span class="ref-vol">24</span>:2911–2935. doi: 10.1002/sim.2165.</span> [<a href="/pubmed/16152135" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1002%2Fsim.2165" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=Adjusting+for+treatment+effects+in+studies+of+quantitative+traits:+antihypertensive+therapy+and+systolic+blood+pressure&amp;author=MD+Tobin&amp;author=NA+Sheehan&amp;author=KJ+Scurrah&amp;author=PR+Burton&amp;volume=24&amp;publication_year=2005&amp;pages=2911-2935&amp;pmid=16152135&amp;doi=10.1002/sim.2165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, Handsaker RE, Kang HM, Marth GT, McVean GA. 1000 Genomes Project Consortium. An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65. doi: 10.1038/nature11632.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnature11632" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=1000+Genomes+Project+Consortium.+An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=GR+Abecasis&amp;author=A+Auton&amp;author=LD+Brooks&amp;author=MA+DePristo&amp;author=RM+Durbin&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;doi=10.1038/nature11632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:76–82. doi: 10.1016/j.ajhg.2010.11.011.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ajhg.2010.11.011" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;doi=10.1016/j.ajhg.2010.11.011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Kato N, Loh M, Takeuchi F, et al.  BIOS-consortium; CARDIo GRAMplusCD; LifeLines Cohort Study; InterAct Consortium. Trans-ancestry genome-wide association study identifies 12 genetic loci influencing blood pressure and implicates a role for DNA methylation. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:1282–1293. doi: 10.1038/ng.3405.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4719169/">PMC free article</a>]</span> [<a href="/pubmed/26390057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3405" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=BIOS-consortium;+CARDIo+GRAMplusCD;+LifeLines+Cohort+Study;+InterAct+Consortium.+Trans-ancestry+genome-wide+association+study+identifies+12+genetic+loci+influencing+blood+pressure+and+implicates+a+role+for+DNA+methylation&amp;author=N+Kato&amp;author=M+Loh&amp;author=F+Takeuchi&amp;volume=47&amp;publication_year=2015&amp;pages=1282-1293&amp;pmid=26390057&amp;doi=10.1038/ng.3405&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">GTEx Consortium. Human genomics. The genotype-tissue expression (gtex) pilot analysis: Multitissue gene regulation in humans. <span><span class="ref-journal">Science. </span>2015;<span class="ref-vol">348</span>:648–660. doi: 10.1126/science.1262110.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4547484/">PMC free article</a>]</span> [<a href="/pubmed/25954001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1126%2Fscience.1262110" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Human+genomics.+The+genotype-tissue+expression+(gtex)+pilot+analysis:+Multitissue+gene+regulation+in+humans&amp;volume=348&amp;publication_year=2015&amp;pages=648-660&amp;pmid=25954001&amp;doi=10.1126/science.1262110&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Damman J, Bloks VW, Daha MR, van der Most PJ, Sanjabi B, van der Vlies P, Snieder H, Ploeg RJ, Krikke C, Leuvenink HG, Seelen MA. Hypoxia and complement-and-coagulation pathways in the deceased organ donor as the major target for intervention to improve renal allograft outcome. <span><span class="ref-journal">Transplantation. </span>2015;<span class="ref-vol">99</span>:1293–1300. doi: 10.1097/TP.0000000000000500.</span> [<a href="/pubmed/25427168" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1097%2FTP.0000000000000500" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transplantation&amp;title=Hypoxia+and+complement-and-coagulation+pathways+in+the+deceased+organ+donor+as+the+major+target+for+intervention+to+improve+renal+allograft+outcome&amp;author=J+Damman&amp;author=VW+Bloks&amp;author=MR+Daha&amp;author=PJ+van+der+Most&amp;author=B+Sanjabi&amp;volume=99&amp;publication_year=2015&amp;pages=1293-1300&amp;pmid=25427168&amp;doi=10.1097/TP.0000000000000500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Li H. Tabix: fast retrieval of sequence features from generic TAB-delimited files. <span><span class="ref-journal">Bioinformatics. </span>2011;<span class="ref-vol">27</span>:718–719. doi: 10.1093/bioinformatics/btq671.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042176/">PMC free article</a>]</span> [<a href="/pubmed/21208982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtq671" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Tabix:+fast+retrieval+of+sequence+features+from+generic+TAB-delimited+files&amp;author=H+Li&amp;volume=27&amp;publication_year=2011&amp;pages=718-719&amp;pmid=21208982&amp;doi=10.1093/bioinformatics/btq671&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575. doi: 10.1086/519795.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1086%2F519795" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;doi=10.1086/519795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. <span><span class="ref-journal">Nucleic Acids Res. </span>2010;<span class="ref-vol">38</span>:e164. doi: 10.1093/nar/gkq603.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2938201/">PMC free article</a>]</span> [<a href="/pubmed/20601685" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fnar%2Fgkq603" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=ANNOVAR:+functional+annotation+of+genetic+variants+from+high-throughput+sequencing+data&amp;author=K+Wang&amp;author=M+Li&amp;author=H+Hakonarson&amp;volume=38&amp;publication_year=2010&amp;pages=e164&amp;pmid=20601685&amp;doi=10.1093/nar/gkq603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Kamburov A, Stelzl U, Lehrach H, Herwig R. The ConsensusPathDB interaction database: 2013 update. <span><span class="ref-journal">Nucleic Acids Res. </span>2013;<span class="ref-vol">41</span>(database issue):D793–D800. doi: 10.1093/nar/gks1055.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3531102/">PMC free article</a>]</span> [<a href="/pubmed/23143270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fnar%2Fgks1055" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+ConsensusPathDB+interaction+database:+2013+update&amp;author=A+Kamburov&amp;author=U+Stelzl&amp;author=H+Lehrach&amp;author=R+Herwig&amp;volume=41&amp;issue=database+issue&amp;publication_year=2013&amp;pages=D793-D800&amp;pmid=23143270&amp;doi=10.1093/nar/gks1055&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Vaez A, Jansen R, Prins BP, Hottenga JJ, de Geus EJ, Boomsma DI, Penninx BW, Nolte IM, Snieder H, Alizadeh BZ. In silico post genome-wide association studies analysis of C-reactive protein loci suggests an important role for interferons. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2015;<span class="ref-vol">8</span>:487–497. doi: 10.1161/CIRCGENETICS.114.000714.</span> [<a href="/pubmed/25752597" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1161%2FCIRCGENETICS.114.000714" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=In+silico+post+genome-wide+association+studies+analysis+of+C-reactive+protein+loci+suggests+an+important+role+for+interferons&amp;author=A+Vaez&amp;author=R+Jansen&amp;author=BP+Prins&amp;author=JJ+Hottenga&amp;author=EJ+de+Geus&amp;volume=8&amp;publication_year=2015&amp;pages=487-497&amp;pmid=25752597&amp;doi=10.1161/CIRCGENETICS.114.000714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Saito R, Smoot ME, Ono K, Ruscheinski J, Wang PL, Lotia S, Pico AR, Bader GD, Ideker T. A travel guide to Cytoscape plugins. <span><span class="ref-journal">Nat Methods. </span>2012;<span class="ref-vol">9</span>:1069–1076. doi: 10.1038/nmeth.2212.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3649846/">PMC free article</a>]</span> [<a href="/pubmed/23132118" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnmeth.2212" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=A+travel+guide+to+Cytoscape+plugins&amp;author=R+Saito&amp;author=ME+Smoot&amp;author=K+Ono&amp;author=J+Ruscheinski&amp;author=PL+Wang&amp;volume=9&amp;publication_year=2012&amp;pages=1069-1076&amp;pmid=23132118&amp;doi=10.1038/nmeth.2212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Mostafavi S, Ray D, Warde-Farley D, Grouios C, Morris Q. GeneMANIA: a real-time multiple association network integration algorithm for predicting gene function. <span><span class="ref-journal">Genome Biol. </span>2008;<span class="ref-vol">9</span>(suppl 1):S4. doi: 10.1186/gb-2008-9-s1-s4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2447538/">PMC free article</a>]</span> [<a href="/pubmed/18613948" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1186%2Fgb-2008-9-s1-s4" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=GeneMANIA:+a+real-time+multiple+association+network+integration+algorithm+for+predicting+gene+function&amp;author=S+Mostafavi&amp;author=D+Ray&amp;author=D+Warde-Farley&amp;author=C+Grouios&amp;author=Q+Morris&amp;volume=9&amp;issue=suppl+1&amp;publication_year=2008&amp;pages=S4&amp;doi=10.1186/gb-2008-9-s1-s4&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Montojo J, Zuberi K, Rodriguez H, Kazi F, Wright G, Donaldson SL, Morris Q, Bader GD. GeneMANIA Cytoscape plugin: fast gene function predictions on the desktop. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2927–2928. doi: 10.1093/bioinformatics/btq562.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2971582/">PMC free article</a>]</span> [<a href="/pubmed/20926419" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtq562" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GeneMANIA+Cytoscape+plugin:+fast+gene+function+predictions+on+the+desktop&amp;author=J+Montojo&amp;author=K+Zuberi&amp;author=H+Rodriguez&amp;author=F+Kazi&amp;author=G+Wright&amp;volume=26&amp;publication_year=2010&amp;pages=2927-2928&amp;pmid=20926419&amp;doi=10.1093/bioinformatics/btq562&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Ashburner M, Ball CA, Blake JA, et al.  Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. <span><span class="ref-journal">Nat Genet. </span>2000;<span class="ref-vol">25</span>:25–29. doi: 10.1038/75556.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037419/">PMC free article</a>]</span> [<a href="/pubmed/10802651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2F75556" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Gene+ontology:+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium&amp;author=M+Ashburner&amp;author=CA+Ball&amp;author=JA+Blake&amp;volume=25&amp;publication_year=2000&amp;pages=25-29&amp;pmid=10802651&amp;doi=10.1038/75556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Dunham I, Kulesha E, Iotchkova V, Morganella S, Birney E. Forge: A tool to discover cell specific enrichments of gwas associated snps in regulatory regions. <span><span class="ref-journal">BioRxiv. </span>2014 doi: 10.1101/013045.</span> [<a href="//dx.doi.org/10.1101%2F013045" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BioRxiv&amp;title=Forge:+A+tool+to+discover+cell+specific+enrichments+of+gwas+associated+snps+in+regulatory+regions&amp;author=I+Dunham&amp;author=E+Kulesha&amp;author=V+Iotchkova&amp;author=S+Morganella&amp;author=E+Birney&amp;publication_year=2014&amp;doi=10.1101/013045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Wagner AH, Coffman AC, Ainscough BJ, Spies NC, Skidmore ZL, Campbell KM, Krysiak K, Pan D, McMichael JF, Eldred JM, Walker JR, Wilson RK, Mardis ER, Griffith M, Griffith OL. DGIdb 2.0: mining clinically relevant drug-gene interactions. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>(D1):D1036–D1044. doi: 10.1093/nar/gkv1165.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4702839/">PMC free article</a>]</span> [<a href="/pubmed/26531824" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fnar%2Fgkv1165" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=DGIdb+2.0:+mining+clinically+relevant+drug-gene+interactions&amp;author=AH+Wagner&amp;author=AC+Coffman&amp;author=BJ+Ainscough&amp;author=NC+Spies&amp;author=ZL+Skidmore&amp;volume=44&amp;issue=D1&amp;publication_year=2016&amp;pages=D1036-D1044&amp;pmid=26531824&amp;doi=10.1093/nar/gkv1165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Skol AD, Scott LJ, Abecasis GR, Boehnke M. Optimal designs for two-stage genome-wide association studies. <span><span class="ref-journal">Genet Epidemiol. </span>2007;<span class="ref-vol">31</span>:776–788. doi: 10.1002/gepi.20240.</span> [<a href="/pubmed/17549752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1002%2Fgepi.20240" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Optimal+designs+for+two-stage+genome-wide+association+studies&amp;author=AD+Skol&amp;author=LJ+Scott&amp;author=GR+Abecasis&amp;author=M+Boehnke&amp;volume=31&amp;publication_year=2007&amp;pages=776-788&amp;pmid=17549752&amp;doi=10.1002/gepi.20240&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Trevisan M, Ostrow D, Cooper R, Liu K, Sparks S, Okonek A, Stevens E, Marquardt J, Stamler J. Abnormal red blood cell ion transport and hypertension. The People’s Gas Company study. <span><span class="ref-journal">Hypertension. </span>1983;<span class="ref-vol">5</span>:363–367.</span> [<a href="/pubmed/6840823" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertension&amp;title=Abnormal+red+blood+cell+ion+transport+and+hypertension.+The+People’s+Gas+Company+study&amp;author=M+Trevisan&amp;author=D+Ostrow&amp;author=R+Cooper&amp;author=K+Liu&amp;author=S+Sparks&amp;volume=5&amp;publication_year=1983&amp;pages=363-367&amp;pmid=6840823&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Wright FA, Sullivan PF, Brooks AI, et al.  Heritability and genomics of gene expression in peripheral blood. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:430–437. doi: 10.1038/ng.2951.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4012342/">PMC free article</a>]</span> [<a href="/pubmed/24728292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.2951" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Heritability+and+genomics+of+gene+expression+in+peripheral+blood&amp;author=FA+Wright&amp;author=PF+Sullivan&amp;author=AI+Brooks&amp;volume=46&amp;publication_year=2014&amp;pages=430-437&amp;pmid=24728292&amp;doi=10.1038/ng.2951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Jansen R, Batista S, Brooks AI, et al.  Sex differences in the human peripheral blood transcriptome. <span><span class="ref-journal">BMC Genomics. </span>2014;<span class="ref-vol">15</span>:33. doi: 10.1186/1471-2164-15-33.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3904696/">PMC free article</a>]</span> [<a href="/pubmed/24438232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1186%2F1471-2164-15-33" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Sex+differences+in+the+human+peripheral+blood+transcriptome&amp;author=R+Jansen&amp;author=S+Batista&amp;author=AI+Brooks&amp;volume=15&amp;publication_year=2014&amp;pages=33&amp;pmid=24438232&amp;doi=10.1186/1471-2164-15-33&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Zhernakova DV, Deelen P, Vermaat M, et al.  Identification of context-dependent expression quantitative trait loci in whole blood. <span><span class="ref-journal">Nat Genet. </span>2017;<span class="ref-vol">49</span>:139–145. doi: 10.1038/ng.3737.</span> [<a href="/pubmed/27918533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3737" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Identification+of+context-dependent+expression+quantitative+trait+loci+in+whole+blood&amp;author=DV+Zhernakova&amp;author=P+Deelen&amp;author=M+Vermaat&amp;volume=49&amp;publication_year=2017&amp;pages=139-145&amp;pmid=27918533&amp;doi=10.1038/ng.3737&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Marusugi K, Nakano K, Sasaki H, Kimura J, Yanobu-Takanashi R, Okamura T, Sasaki N. Functional validation of tensin2 SH2-PTB domain by CRISPR/Cas9-mediated genome editing. <span><span class="ref-journal">J Vet Med Sci. </span>2016;<span class="ref-vol">78</span>:1413–1420. doi: 10.1292/jvms.16-0205.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5059368/">PMC free article</a>]</span> [<a href="/pubmed/27246398" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1292%2Fjvms.16-0205" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Vet+Med+Sci&amp;title=Functional+validation+of+tensin2+SH2-PTB+domain+by+CRISPR/Cas9-mediated+genome+editing&amp;author=K+Marusugi&amp;author=K+Nakano&amp;author=H+Sasaki&amp;author=J+Kimura&amp;author=R+Yanobu-Takanashi&amp;volume=78&amp;publication_year=2016&amp;pages=1413-1420&amp;pmid=27246398&amp;doi=10.1292/jvms.16-0205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Jeggle P, Smith ES, Stewart AP, Haerteis S, Korbmacher C, Edwardson JM. Atomic force microscopy imaging reveals the formation of ASIC/ENaC cross-clade ion channels. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>2015;<span class="ref-vol">464</span>:38–44. doi: 10.1016/j.bbrc.2015.05.091.</span> [<a href="/pubmed/26032502" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.bbrc.2015.05.091" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=Atomic+force+microscopy+imaging+reveals+the+formation+of+ASIC/ENaC+cross-clade+ion+channels&amp;author=P+Jeggle&amp;author=ES+Smith&amp;author=AP+Stewart&amp;author=S+Haerteis&amp;author=C+Korbmacher&amp;volume=464&amp;publication_year=2015&amp;pages=38-44&amp;pmid=26032502&amp;doi=10.1016/j.bbrc.2015.05.091&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Carrier L, Mearini G, Stathopoulou K, Cuello F. Cardiac myosin-binding protein C (MYBPC3) in cardiac pathophysiology. <span><span class="ref-journal">Gene. </span>2015;<span class="ref-vol">573</span>:188–197. doi: 10.1016/j.gene.2015.09.008.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6660134/">PMC free article</a>]</span> [<a href="/pubmed/26358504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.gene.2015.09.008" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Cardiac+myosin-binding+protein+C+(MYBPC3)+in+cardiac+pathophysiology&amp;author=L+Carrier&amp;author=G+Mearini&amp;author=K+Stathopoulou&amp;author=F+Cuello&amp;volume=573&amp;publication_year=2015&amp;pages=188-197&amp;pmid=26358504&amp;doi=10.1016/j.gene.2015.09.008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5783787&amp;issue-id=305348&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140304100444544title">Abstract</a></li><li><a href="#S1title">Materials and Methods</a></li><li><a href="#S14title">Results</a></li><li><a href="#S15title">Discussion</a></li><li><a href="#S16title">Perspectives</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#idm140304098893760title">Footnotes</a></li><li><a href="#idm140304095893488title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5783787/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5783787/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5783787/pdf/nihms902240.pdf">PDF (581K)</a></li> | <li><a href="#" data-citationid="PMC5783787" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5783787%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5783787%2F&amp;text=Novel%20Blood%20Pressure%20Locus%20and%20Gene%20Discovery%20Using%20Genome-Wide%20Association%20Study%20and%20Expression%20Data%20Sets%20From%20Blood%20and%20the%20Kidney" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5783787%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28739976" data-aiid="5783787" data-aid="5783787" data-iid="305348" data-domainid="319" data-domain="nihpa" data-accid="PMC5783787" data-md5="e1a4469a2a5030bc09b63e7deddba16e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:28:13-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A2BBCDD5A3E610000000007650454&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5783787%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5783787/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>