<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5255953/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide investigation into parent-of-origin effects in autism spectrum disorder identifies previously associated genes including SHANK3</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5255953/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="A genome-wide investigation into parent-of-origin effects in autism spectrum disorder identifies previously associated genes including SHANK3"><meta name="citation_authors" content="Siobhan Connolly, Richard Anney, Louise Gallagher, Elizabeth A Heron"><meta name="citation_date" content="February 2017"><meta name="citation_issue" content="2"><meta name="citation_volume" content="25"><meta name="citation_firstpage" content="234"><meta name="citation_doi" content="10.1038/ejhg.2016.153"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5255953/?report=abstract"><meta name="citation_pmid" content="27876814"><meta name="DC.Title" content="A genome-wide investigation into parent-of-origin effects in autism spectrum disorder identifies previously associated genes including SHANK3"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Siobhan Connolly"><meta name="DC.Contributor" content="Richard Anney"><meta name="DC.Contributor" content="Louise Gallagher"><meta name="DC.Contributor" content="Elizabeth A Heron"><meta name="DC.Date" content="2017 Feb"><meta name="DC.Identifier" content="10.1038/ejhg.2016.153"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide investigation into parent-of-origin effects in autism spectrum disorder identifies previously associated genes including SHANK3"><meta property="og:type" content="article"><meta property="og:description" content="Autism spectrum disorder (ASD) is known to be a heritable neurodevelopmental disorder affecting more than 1% of the population but in the majority of ASD cases, the genetic cause has not been identified. Parent-of-origin effects have been highlighted ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5255953/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5255953/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5255953/pdf/ejhg2016153a.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88C02CDD592CE1000000000911055E.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5255953%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/285295/">v.25(2); 2017 Feb</a></li><li class="accid">PMC5255953</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2017 Feb; 25(2): 234–239. </span></div><div><span class="fm-vol-iss-date">Published online 2016 Nov 23. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fejhg.2016.153" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ejhg.2016.153</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5255953</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27876814">27876814</a></div></div></div></div><h1 class="content-title">A genome-wide investigation into parent-of-origin effects in autism spectrum disorder identifies previously associated genes including <em>SHANK3</em></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Connolly%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27876814" class="affpopup" co-rid="_co_idm139831879324480" co-class="co-affbox">Siobhan Connolly</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Anney%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27876814" class="affpopup" co-rid="_co_idm139831879322368" co-class="co-affbox">Richard Anney</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Gallagher%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27876814" class="affpopup" co-rid="_co_idm139831886031104" co-class="co-affbox">Louise Gallagher</a>,<sup>1</sup> and  <a href="/pubmed/?term=Heron%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27876814" class="affpopup" co-rid="_co_idm139831886560992" co-class="co-affbox">Elizabeth A Heron</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139831879324480"><h3 class="no_margin">Siobhan Connolly</h3><p><sup>1</sup>Neuropsychiatric Genetics Research Group, Department of Psychiatry, Trinity College Dublin, Trinity Centre for Health Sciences, Dublin, Ireland</p><div>Find articles by <a href="/pubmed/?term=Connolly%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27876814">Siobhan Connolly</a></div></div><div id="_co_idm139831879322368"><h3 class="no_margin">Richard Anney</h3><p><sup>1</sup>Neuropsychiatric Genetics Research Group, Department of Psychiatry, Trinity College Dublin, Trinity Centre for Health Sciences, Dublin, Ireland</p><p><sup>2</sup>Division of Psychological Medicine and Clinical Neurosciences, Cardiff University School of Medicine, Hadyn Ellis Building, Cathays, Cardiff, UK</p><div>Find articles by <a href="/pubmed/?term=Anney%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27876814">Richard Anney</a></div></div><div id="_co_idm139831886031104"><h3 class="no_margin">Louise Gallagher</h3><p><sup>1</sup>Neuropsychiatric Genetics Research Group, Department of Psychiatry, Trinity College Dublin, Trinity Centre for Health Sciences, Dublin, Ireland</p><div>Find articles by <a href="/pubmed/?term=Gallagher%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27876814">Louise Gallagher</a></div></div><div id="_co_idm139831886560992"><h3 class="no_margin">Elizabeth A Heron</h3><p><sup>1</sup>Neuropsychiatric Genetics Research Group, Department of Psychiatry, Trinity College Dublin, Trinity Centre for Health Sciences, Dublin, Ireland</p><div>Find articles by <a href="/pubmed/?term=Heron%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27876814">Elizabeth A Heron</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139831884060176_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139831884060176_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139831884060176_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139831884060176_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Neuropsychiatric Genetics Research Group, Department of Psychiatry, Trinity College Dublin, Trinity Centre for Health Sciences, Dublin, Ireland</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Division of Psychological Medicine and Clinical Neurosciences, Cardiff University School of Medicine, Hadyn Ellis Building, Cathays, Cardiff, UK</div><div id="caf1"><sup>*</sup>Neuropsychiatric Genetics Research Group, Department of Psychiatry, Trinity College Dublin, Trinity Centre for Health Sciences, James's Street, Dublin, Ireland. Tel: +353 1 8963272; Fax: +353 1 8963405; E-mail: <a href="mailto:dev@null" data-email="moc.liamg@nahbois.yllonnoc" class="oemail">moc.liamg@nahbois.yllonnoc</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139831884060176_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 Feb 24; Revised 2016 Sep 28; Accepted 2016 Oct 7.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139831884060176_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2017 Macmillan Publishers Limited, part of Springer Nature.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5255953/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC5255953/bin/ejhg2016153x1.pdf" data-ga-action="click_feat_suppl">ejhg2016153x1.pdf</a><span style="color:gray"> (15M)</span></div><div class="small guid">GUID:&nbsp;D2C89284-89DC-49CC-9220-2BD8D8D1798C</div></div></dd></dl></div><div id="idm139831919422544" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139831919422544title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Autism spectrum disorder (ASD) is known to be a heritable neurodevelopmental disorder affecting more than 1% of the population but in the majority of ASD cases, the genetic cause has not been identified. Parent-of-origin effects have been highlighted as an important mechanism in the pathology of neurodevelopmental disorders such as Prader–Willi and Angelman syndrome, with individuals with these syndromes often exhibiting ASD symptoms. Consequently, systematic investigation of these effects in ASD is clearly an important line of investigation in elucidating the underlying genetic mechanisms. Using estimation of maternal, imprinting and interaction effects using multinomial modelling (EMIM), we simultaneously investigated imprinting, maternal genetic effects and associations in the Autism Genome Project and Simons Simplex Consortium genome-wide association data sets. To avoid using the overly stringent genome-wide association study significance level, we used a Bayesian threshold that takes into account the sample size, allele frequency and any available prior knowledge. Between the two data sets, we identified a total of 18 imprinting effects and 68 maternal genetic effects that met this Bayesian threshold criteria, but none met the threshold in both data sets. We identified imprinting and maternal genetic effects for regions that have previously shown evidence for parent-of-origin effects in ASD. Together with these findings, we have identified maternal genetic effects not previously identified in ASD at a locus in SHANK3 on chromosome 22 and a locus in WBSCR17 on chromosome 7 (associated with Williams syndrome). Both genes have previously been associated with ASD.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">Autism spectrum disorder (ASD) is a complex and heterogeneous neurodevelopmental disorder characterised by patterns of repetitive behaviours and deficits in language and social behaviour. ASD is known to be heritable, with earlier heritability estimates ranging from 0.85 to 0.92<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167927">1</a></sup> to a more current heritability estimate of 0.524,<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167900">2</a></sup> with most of this believed to be due to common rather than rare variation.<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167938">2</a></sup> Gaugler <em>et al</em><sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167990">2</a></sup> also estimated that 41% of the risk for ASD is due to environmental factors, which include prenatal, perinatal and postnatal environmental factors. Several genome-wide association studies (GWASs) have been performed on common variants in ASD using family studies, but the most significant results from these studies show modest effect size<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167893">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167923">4</a>, <a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167945">5</a></sup> and therefore, have low power to replicate.<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167992">4</a></sup> Despite growing evidence from investigations of rare and <em>de novo</em> structural and sequence variation in ASD,<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167907">6</a></sup> the aetiology of the majority of ASD cases remains unexplained, suggesting that there are complex genetic mechanisms underlying the disorder. There is also evidence to suggest that there is a complex relationship between these underlying genetic factors and environmental factors.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167935">7</a></sup></p><p id="__p3">Broadly speaking, parent-of-origin effects consist of genetic effects on the phenotype of an offspring that are dependent on the parental origin of the associated genetic variant(s). Parent-of-origin effects can occur through numerous mechanisms such as genomic imprinting and certain trans-generational effects (for example, maternal genetic effects).<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167910">8</a></sup> Genomic imprinting occurs when the allele from a particular parent is silenced and the gene is expressed only by the remaining allele that has been inherited from the other parent. Evidence for imprinting has been shown in Prader–Willi syndrome and Angleman syndrome,<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167967">9</a>, <a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167891">10</a></sup> both syndromes having autistic features and diagnoses.<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167915">11</a></sup> There has been strong suggestive evidence for imprinting in ASD in the 7q and 15q regions which warrant further investigation.<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167937">12</a></sup></p><p id="__p4">When statistically investigating GWAS data for imprinted genes, we examine the transmission of alleles from the parent to the offspring. If there is an over-transmission of the variant allele by a particular parent, then this might suggest that there is a difference in the expression of the alleles through epigenetic mechanisms (heritable changes that do not cause changes in the DNA sequence). For example, if the variant allele is over-transmitted from mothers only (ie, maternal over-transmission), this might suggest evidence for nonexpression of the paternally derived allele.</p><p id="__p5">In addition to imprinting, maternal genetic effects can also occur when the maternal genotype exerts an influence on the offspring's phenotype regardless of what genetic material has been passed from the mother to the offspring. One way in which this can occur is when the mother's genotype affects the development of the foetus through the intrauterine environment. There has been evidence of maternal genetic effects in ASD, for example, at the <em>GSTP*A</em> gene<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167991">13</a></sup> and the <em>HLA-DR4</em> gene.<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167899">14</a></sup> There has also been some evidence of possible prenatal environmental factors influencing the risk of ASD. For example, ASD is more likely in half siblings with a shared mother (recurrence rate estimates: 5.2 and 7.3%) compared with half siblings with a shared father (estimates: 0 and 3.2%, although sample sizes were small for paternal half siblings).<sup><a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167904">15</a></sup> However, more recent estimates based on a larger cohort from Sweden have not shown as large a difference<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167912">16</a></sup> (recurrence rates: 3.3% for half siblings with shared mother; 2.9% for half siblings with shared father).</p><p id="__p6">The study of parent-of-origin effects has the potential to identify genetic and environmental factors that may be contributing to a complex disorder such as ASD.<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167970">17</a></sup> Therefore, in order to help elucidate the genetic and epigenetic aetiology of ASD, we investigated two types of parent-of-origin effects, imprinting and maternal genetic effects, in ASD GWAS. Previous studies have investigated parent-of-origin effects in ASD.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167892">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167946">4</a>, <a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167903">18</a>, <a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167966">19</a></sup> Anney <em>et al</em><sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167971">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167986">4</a></sup> investigated imprinting in a secondary analysis in the Autism Genome Project (AGP) GWAS data sets using an in-house method reported to be similar to the method of Cordell <em>et al</em>,<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167969">20</a></sup> but findings were not considered to be statistically significant after correcting for multiple testing. Chaste <em>et al</em><sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167902">5</a></sup> also considered whether the transmission came from the mother or the father in their GWAS analysis of the Simons Simplex Collection (SSC) data although no parental-specific results are reported. Tsang <em>et al</em><sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167928">18</a></sup> and Yuan and Dougherty<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167951">19</a></sup> used GWAS data to investigate maternal genetic effects in ASD using a case–control type analysis. There were no genome-wide significant findings or replicated findings in either of these studies.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167977">18</a>, <a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167936">19</a></sup></p><p id="__p7" class="p p-last">Our analysis approach differs in that we investigated both imprinting and maternal genetic effects simultaneously as maternal genetic effects are known to mimic imprinting and vice versa.<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167953">21</a>, <a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167987">22</a></sup> We also included offspring genetic effects (associations) in our model to enable us to identify an imprinting effect associated with ASD. These parent-of-origin analyses were investigated using estimation of maternal, imprinting and interaction effects using multinomial modelling (EMIM),<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167979">23</a>, <a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167896">24</a></sup> which in comparison with other tests has been shown to be the most suitable in terms of power and type I errors for this type of data.<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167981">22</a></sup> We also adapted a Bayesian method<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167940">25</a></sup> to determine an appropriate noteworthy threshold at each locus (taking into account sample size, minor allele frequency (MAF) and prior knowledge) instead of using the genome-wide significance levels, which are known to be stringent.<sup><a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167947">26</a></sup></p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Materials and methods</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Data</h3><p id="__p8" class="p p-first">The AGP GWAS family trio data set is described elsewhere,<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167978">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167957">4</a></sup> and here we are using the Stage 2 data set consisting of 2931 families. This data set contains approximately one million SNPs genotyped on either the Illumina Infinium 1M-single or the Illumina 1M-duo microarray, see Acknowledgements for information on how to obtain the data.</p><p id="__p9">The SSC GWAS consists of data on 2591 simplex families that were genotyped for a million or more SNPs on one of three array versions—Illumina 1Mv1 (333 families), Illumina 1Mv3 Duo (1189 families) or Illumina HumanOmni2.5 M (1069 families) Therefore, since imputation is computationally intensive for trio data sets and we needed the parental genotype data, we combined the three data sets and investigated SNPs common to all three arrays, as was carried out in ref. <a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167901">5</a>. See refs <a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167921">5</a>, <a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167959">27</a>, <a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167984">28</a> for further details on the SSC data.</p><p id="__p10" class="p p-last">The AGP GWAS data includes families grouped into two nested diagnostic categories, Strict ASD (autism diagnoses met on both ADI and ADOS instruments) and spectrum ASD (autism-spectrum diagnoses met on either the ADI or ADOS instruments), as used in ref. <a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167948">3</a>. Although not considered in the analyses of the SSC data,<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167917">5</a></sup> we applied the same ASD phenotype criteria as was used in the AGP<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167976">3</a></sup> to define strict and spectrum ASD phenotypes within the SSC data. We focus our main analyses on the spectrum phenotype which provides the larger sample sizes for the analyses (secondary analyses on the strict phenotype are provided in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a>).</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Statistical model</h3><p id="__p11" class="p p-first-last">Following an extensive review of the parent-of-origin methodology, described elsewhere,<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167914">22</a></sup> we used the EMIM<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167909">23</a>, <a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167956">24</a></sup> approach for our analyses. EMIM is a multinomial model that directly maximises the multinomial likelihood to detect parent-of-origin effects and can incorporate missing data (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> for further details). We simultaneously investigated offspring genetic effects (associations), maternal genetic effects and imprinting effects using EMIM. We did not include mother/offspring interactions in our model as the inclusion of mother/offspring interaction parameters reduces the power of the model substantially, as is to be expected.<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167988">22</a></sup> Instead mother/offspring interaction effects were investigated subsequent to a SNP being identified as having an offspring effect and a maternal genetic effect. A multiplicative model was assumed for offspring and maternal genotype parameters. The benefit of this is twofold; firstly for ease of investigating the effects in an already complex model; and secondly, to reduce the number of parameters in the model to help increase power, see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> for more details. EMIM has also been extended to use haplotype estimates to help increase power for detecting imprinting but this method was not used in this paper.<sup><a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167911">29</a></sup></p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Quality control procedures</h3><p id="__p12" class="p p-first-last">The quality control (QC) follows a standard approach to trio GWAS QC, with individuals and SNPs removed when missingness &gt;0.05, MAF&lt;0.05 and Hardy–Weinberg equilibrium <em>P</em>-value&lt;0.00001. We limited our analyses to complete independent trios (both parents and offspring) to prevent the reduction in power that estimating missing data in our model would cause. Full details of the QC procedures are given in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a>. After QC, the AGP with a spectrum phenotype contains 2594 trios and 728 228 SNPs, and the SSC with a spectrum phenotype contains 2433 trios and 483 080 SNPs.</p></div><div id="__sec6" class="sec sec-last"><h3 id="__sec6title">Bayesian noteworthy threshold</h3><p id="__p13" class="p p-first">Power to detect parent-of-origin effects can be limited<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167980">22</a></sup> and the current genome-wide significance threshold guidelines are known to be very stringent.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167975">25</a>, <a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167994">26</a>, <a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167968">30</a></sup> These genome-wide significance levels are suggested for all sample sizes (and MAFs) and hence do not take into account the power at individual SNPs.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167964">25</a>, <a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167941">31</a></sup> To take these factors into account, we adopted a Bayesian method proposed by Wakefield<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_584167920">25</a></sup> to determine an appropriate threshold for identifying noteworthy findings.</p><p id="__p14">This threshold for <em>Z</em><sup>2</sup>-score is given by the following:</p><div class="disp-formula" id="equ1"><div class="f"><img src="/pmc/articles/PMC5255953/bin/ejhg2016153e1.jpg" class="eq-image" alt="equation image" title="equation image"></div></div><p></p><p id="__p16">where <em>V</em><sub><em>n</em></sub> is the standard error of the parameter (which is dependent on the sample size, <em>n</em>, and MAF), <em>W</em> is the prior variance for the log of the relative risk Δ (ie, Δ∼<em>N</em>(0,<em>W</em>)), <em>PO</em> is the prior odds (ie <em>π</em><sub><em>0</em></sub>/(1–<em>π</em><sub><em>0</em></sub>), where <em>π</em><sub><em>0</em></sub> is the prior probability that <em>H</em><sub>0</sub> is true), and <em>R</em> is the ratio of costs of type II to type I errors. See <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> and refs <a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359870">25</a>, <a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359884">31</a> for more information.</p><p id="__p17">We note that when we are investigating imprinting, it is necessary to have an offspring genotype effect present in addition to an imprinting effect. This is required to ensure that we do not identify non-disease related imprinted regions that would be observed in the general population. For maternal genetic effects, an association is not required and we investigate loci, where the mother's genotype exerts an influence on the offspring's phenotype, regardless of association being present or not. Therefore, we calculated separate <em>Z</em><sup>2</sup>-score thresholds for the Wald <em>Z</em>-score for the association parameter (<em>R1</em>) and for the maternal genetic effect parameter (<em>S1</em>). We detect a noteworthy imprinting result when both the association parameter and the imprinting parameter meet the threshold (see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2</a> in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a>), whereas we detect a noteworthy maternal genetic effect when the maternal genetic effect parameter meets the threshold (see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3</a> in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a>).</p><p id="__p18" class="p p-last">We chose the prior variance for the log of the relative risk to be <em>W</em>=[log(2)/1.645]2=0.422 (interpret this as a 5% chance that the relative risk will be larger than 2) to reflect the low effect sizes in GWAS.<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359851">32</a></sup> Because of the evidence that several hundred to thousands of loci are likely to contribute to the complex genetic heterogeneity of ASD,<sup><a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359873">33</a>, <a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359855">34</a></sup> we choose <em>π</em><sub><em>0</em></sub>=1–500/1 000 000=0.9995, which leads to a prior odds of H<sub>0</sub> being true of <em>PO</em>=1 999. Since the power is limited to detect parent-of-origin effects in ASD, and this in turn increases type II errors, we chose the ratio of cost of type II errors to type I errors, <em>R</em>, equal to 10, as false negatives cannot be followed up. For more details on the choice of these parameters and the sensitivity of the Bayesian threshold to these choices see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a>.</p></div></div><div id="__sec7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec7title">Results</h2><div id="__sec8" class="sec sec-first"><h3 id="__sec8title">Autism Genome Project (AGP)</h3><p id="__p19" class="p p-first">There were nine noteworthy independent loci showing imprinting effects and there were forty independent loci showing maternal genetic effects, see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S5</a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S4 and S5</a> in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> for all variants that were above the threshold for offspring genetic effect (<em>R1</em>) and imprinting (<em>IM</em>) or were above the threshold for maternal genetic effects (<em>S1</em>). <a href="/pmc/articles/PMC5255953/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> gives a summary of the results discussed below. Note: <em>IM</em>&gt;1 indicates a maternal over-transmission of the allele and <em>IM</em>&lt;1 indicates a paternal over-transmission of the allele, whereas <em>IM</em>=1 indicates no imprinting effect.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Main results from the AGP and SSC data set</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139831875401360" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th colspan="5" charoff="50" rowspan="1" valign="top" align="center"><em>AGP results</em></th><th colspan="2" charoff="50" rowspan="1" valign="top" align="center"><em>SSC results</em></th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>SNP</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Effect</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P-value</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Location</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Previous findings</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Effect</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P-value</em></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs675680</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sub><em>M</em></sub>=2.36</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.0 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 4p15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">None identified</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sub><em>M</em></sub>= 0.81</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.4 × 10<sup>−1</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10025482</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sub><em>M</em></sub> =0.59</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.2 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 4, <em>C4orf37</em> gene</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sub><em>M</em></sub> =1.19</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.5 × 10<sup>−1</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs4516878</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=1.40</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.2 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 5q34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">None identified</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=1.07</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.7 × 10<sup>−1</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs545208</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=0.74</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.0 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 11, <em>MAML2</em> gene</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=1.02</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8.1 × 10<sup>−1</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9870610</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=1.33</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7.2 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 3, <em>ROBO2</em> gene</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=1.16</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5.0 × 10<sup>−2</sup></td></tr><tr><td colspan="5" charoff="50" rowspan="1" valign="top" align="center"><em>SSC results</em></td><td colspan="2" charoff="50" rowspan="1" valign="top" align="center"><em>AGP results</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>SNP</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Effect</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P-value</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Location</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Previous findings</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Effect</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P-value</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9573533</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sub><em>M</em></sub>=0.59</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8.2 × 10<sup><em>−</em>6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 13, <em>TBC1D4</em> gene</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">None identified</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sub><em>M</em></sub>=0.98</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8.3 × 10<sup><em>−</em>1</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs16890706</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sub><em>M</em></sub>=1.86</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.1 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 6, <em>LRRC16A</em> gene</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><sup><a href="#bib35" rid="bib35" class=" bibr popnode">35</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>I</em><sub><em>M</em></sub>=1.18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.1 × 10<sup>−1</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6967953</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=1.38</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.0 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 7, <em>CHRM2</em> gene</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><sup><a href="#bib36" rid="bib36" class=" bibr popnode">36</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=0.92</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.0 × 10<sup>−1</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs5770820</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=1.25</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5.5 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 22, <em>SHANK3</em> gene</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><sup><a href="#bib37" rid="bib37" class=" bibr popnode">37</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=0.98</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8.2 × 10<sup>−1</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs4719103</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=1.41</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5.5 × 10<sup><em>−</em>5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 7, <em>WBSCR17</em> gene</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><sup><a href="#bib38" rid="bib38" class=" bibr popnode">38</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=0.96</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.3 × 10<sup><em>−</em>1</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs11075447</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=0.76</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.5 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Chr 22, <em>GNB1L</em> gene</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><sup><a href="#bib41" rid="bib41" class=" bibr popnode">41</a></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>S</em><sub><em>1</em></sub>=1.00</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.5 × 10<sup>−1</sup></td></tr></tbody></table></div><div id="largeobj_idm139831875401360" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5255953/table/tbl1/?report=objectonly">Open in a separate window</a></div></div><div id="__sec9" class="sec"><p></p><h4 id="__sec9title" class="inline">Autism Genome Project imprinting results </h4><p id="__p20" class="p p-first-last">The top imprinting result was a maternal over-transmission on chromosome 4, between <em>LOC391642</em> and <em>LOC645641</em> (rs675680, hg18 chr11:g.28082183A&gt;G, allele=G, <em>IM</em>=2.36, Wald <em>P</em>-value=3.02 × 10−6), which has not been previously associated with ASD. We also found a noteworthy paternal over-transmission in SNPs in the <em>STPG2</em> gene (<em>C4orf37</em> gene) on chromosome 4 (top SNP, rs10025482, hg18 chr4:g.99272299C&gt;T, allele=T, <em>IM</em>=0.59, Wald <em>P</em>-value=6.21 × 10−6, see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S7</a> in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a>) and this region was previously implicated in ASD,<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359854">18</a></sup> where a mother/offspring interaction effect was identified in the vicinity of the <em>STPG2</em> gene (rs28539905—not genotyped, <em>R</em><sup>2</sup>=0.005). We found no evidence of an interaction effect at rs10025482 (LRT <em>P</em>-value=0.33) or in this region.</p></div><div id="__sec10" class="sec sec-last"><p></p><h4 id="__sec10title" class="inline">Autism Genome Project maternal genetic results </h4><p id="__p21" class="p p-first-last">The top result for a maternal genetic effect was on chromosome 5 between the genes <em>LOC391845</em> and <em>LOC574080</em> (rs4516878, hg18 chr5:g.164271894T&gt;C, allele=C, <em>S1</em>=1.40, Wald <em>P</em>-value=1.16 × 10−5). This region has not been previously linked with ASD, to our knowledge. In our top results, we also identified two maternal genetic effects that were previously implicated as maternal genetic effects or mother/offspring interactions in the Early Markers for Autism data set.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359875">18</a></sup> The first hit is located on the <em>MAML2</em> gene on chromosome 11, which was identified as a maternal genetic effect in our analyses and in the same region in Tsang <em>et al</em><sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359874">18</a></sup> but in opposite directions (rs545208, hg18 chr11:g.95619756C&gt;T, allele=T, <em>S1</em>=0.74, Wald <em>P</em>-value=3 × 10−5, see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S8</a> in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a>). We also identified a maternal genetic effect (rs9870610, hg18 chr3:g.95619758C&gt;T, allele=T, <em>S1</em>=1.33, Wald <em>P</em>-value=7.24 × 10<sup>−5</sup>, see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S9</a>) on <em>ROBO2</em> on chromosome 3. An interaction effect was previously identified in the gene <em>ROBO2</em> in Tsang <em>et al</em>,<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359877">18</a></sup> but we found no evidence of an interaction (LRT <em>P</em>-value=0.34) at rs545208.</p></div></div><div id="__sec11" class="sec sec-last"><h3 id="__sec11title">Simons Simplex Collection (SSC)</h3><p id="__p22" class="p p-first">There were nine noteworthy imprinting results and there were 28 independent noteworthy loci with a maternal genetic effect in the SSC data, see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S16</a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S8 and S9</a> in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> and <a href="/pmc/articles/PMC5255953/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>.</p><div id="__sec12" class="sec"><p></p><h4 id="__sec12title" class="inline">Simons Simplex Collection imprinting results </h4><p id="__p23" class="p p-first-last">The top imprinting result was a paternal over-transmission on chromosome 13 in the <em>TBC1D4</em> gene (rs9573533, hg18 chr13:g.74853485G&gt;A, allele=A, <em>IM</em>=0.59, Wald <em>P</em>-value=8.17 × 10<sup>−6</sup>). To our knowledge, this area has not been previously linked with ASD. We identified a maternal over-transmission in the <em>LRRC16A</em> gene (near the <em>HLA</em> region) (rs16890706, hg18 chr6:g.25628073G&gt;A, allele=A, <em>IM</em>=1.86, Wald <em>P</em>-value=1.09 × 10−5), which was previously implicated in language deficits.<sup><a href="#bib35" rid="bib35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359861">35</a></sup></p></div><div id="__sec13" class="sec sec-last"><p></p><h4 id="__sec13title" class="inline">Simons Simplex Collection maternal genetic results </h4><p id="__p24" class="p p-first">The strongest association for a maternal genetic effect was on chromosome 7 in the <em>CHRM2</em> gene (rs6967953, hg18 chr7:g.136353916G&gt;A, allele=A, <em>S</em><sub><em>1</em></sub>=1.38, Wald <em>P</em>-value=6.01 × 10<sup>−6</sup>). This area has been previously linked with IQ and one of the strongest linkage signals reported for ASD occurred at 7q within 1.6 kb of the <em>CHRM2</em> gene.<sup><a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359893">36</a></sup> One of our top hits for maternal genetic effects was identified on chromosome 22 in the <em>SHANK3</em> gene (rs5770820, hg18 chr22:g.49497339G&gt;A, allele =A, <em>S</em><sub><em>1</em></sub>=1.25, Wald <em>P</em>-value=5.54 × 10<sup>−5</sup>, see <a href="/pmc/articles/PMC5255953/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>). Disruptions in the <em>SHANK3</em> gene have been associated with autistic traits and in particular, these disruptions are responsible for the development of Phelan–McDermid syndrome and other non-syndromic ASDs.<sup><a href="#bib37" rid="bib37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359890">37</a></sup>
<a href="/pmc/articles/PMC5255953/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a> shows that there are no SNPs in high LD with rs5770820 (the SNP in highest LD was rs739365, <em>R</em><sup><em>2</em></sup>=0.65) due to the limited number of SNPs common to all three arrays in the SSC data set.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC5255953/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139831884760288" class="figure"><a href="/pmc/articles/PMC5255953/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5255953_ejhg2016153f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg2016153f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5255953/bin/ejhg2016153f1.jpg"></a></div><div id="largeobj_idm139831884760288" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5255953/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC5255953/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p39">SSC spectrum chromosome 22, <em>SHANK3</em> gene, rs5770820 maternal genetic effect. Regional plot of SNPs highlighted in the SSC spectrum analysis for maternal genetic effects (<em>S</em><sub>1</sub>, triangles). Markers in linkage disequilibrium with the index SNP are shown and based on 1000 genomes CEU. Recombination rate plotted in black. The black dotted line represents the Bayesian threshold for <em>S</em><sub>1</sub>.</p></div></div></div><p id="__p25">We also detected a noteworthy maternal genetic effect on chromosome 7q11.23 in the <em>WBSCR17</em> gene (rs4719103, hg18 chr7:g.70395849G&gt;A, allele=A, <em>S</em><sub><em>1</em></sub>=1.41, Wald <em>P</em>-value=5.48 × 10<sup>−5</sup>, see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S18</a> in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a>), this region is deleted in Williams syndrome<sup><a href="#bib38" rid="bib38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359856">38</a></sup> and it is known that individuals with Williams syndrome exhibit autistic behaviours.<sup><a href="#bib39" rid="bib39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359860">39</a>, <a href="#bib40" rid="bib40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359857">40</a></sup> This region was strongly associated with ASD in a Copy Number Variant (CNV) study carried out on the SSC data set,<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359880">28</a></sup> and we acknowledge that there is a large overlap between the samples analysed here and those in the CNV study.</p><p id="__p26" class="p p-last">Another noteworthy result was found on chromosome 22q, which is a protective maternal genetic effect in the <em>GNB1L gene</em> (also known as <em>C22orf29</em> gene) (rs11075447, hg18 chr16:g.60560457A&gt;G, allele=G, <em>S</em><sub><em>1</em></sub>=0.76, Wald <em>P</em>-value=9.52 × 10<sup>−5</sup>), this gene has been linked to ASD and schizophrenia.<sup><a href="#bib41" rid="bib41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359885">41</a></sup></p></div></div></div><div id="__sec14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec14title">Discussion</h2><p id="__p27" class="p p-first">To our knowledge, this is the first genome-wide study to test for both imprinting and maternal genetic effects simultaneously in ASD. This is also the first study to implement the Bayesian thresholds that take into consideration the sample size and MAF at each SNP, and prior knowledge of effect size and prior odds of finding associations. We analysed the AGP and SSC ASD data sets for parent-of-origin effects, specifically imprinting and maternal genetic effects. Previous studies of parent-of-origin effects in ASD only investigated either imprinting effects or maternal genetic effects<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359894">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359879">4</a>, <a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359886">18</a>, <a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359892">19</a></sup> despite the fact that these effects are known to mimic each other. We identified a total of 18 imprinting effects and 68 maternal genetic effects that met this Bayesian threshold criteria in either the AGP or SSC data sets with a Spectrum phenotype. None of these results were identified in both data sets. The <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> contains further analyses of parent-of-origin effects in the AGP and SSC data sets for a Strict ASD phenotype, where we identified 10 imprinting effects and 72 maternal genetic effects that met the Bayesian threshold criteria in either data sets. A proportion (10–20%) of the results identified using a Strict ASD phenotype overlap with the results identified using the Spectrum ASD phenotype (see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> for further details).</p><p id="__p28">This model is complex as it includes three parameters, offspring genetic effects, imprinting effects and maternal genetic effects, which can reduce power and can lead to the results being harder to interpret. To help identify noteworthy findings, we adopted a Bayesian threshold proposed by Wakefield<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359852">25</a></sup> to investigate parent-of-origin effects as it facilitates ease of interpretation of an imprinting effect and a maternal genetic effect. In addition, the Bayesian threshold avoids the use of the overly stringent genome-wide significance threshold.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359891">25</a>, <a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359871">26</a></sup> The Bayesian threshold takes into account the sample size and MAF as well as other prior knowledge regarding ASD (for example, effect size) to allow for a more appropriate threshold for the model at each locus. The Bayesian threshold does not depend on the number of tests performed but instead depends on the prior odds. If a Bonferroni correction was employed, or the stringent GWAS threshold, then the noteworthy hits we identified would have been missed, but as we have shown, some of these hits show promise by being previously identified in ASD studies (see <a href="/pmc/articles/PMC5255953/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). In addition, we have accounted for the rate of true positives and true negatives in the prior odds in the Bayesian threshold, which is a superior method in comparison to using the Bonferroni correction or the GWAS thresholds, which do not account for these.</p><p id="__p29">Replicating results identified in a discovery analysis in an independent sample is the gold standard in GWAS analysis as it provides convincing statistical evidence for association, and has the potential to rule out associations due to biases.<sup><a href="#bib42" rid="bib42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359887">42</a></sup> Replication generally involves identifying the significant results in the discovery analysis and examining these in a replication data set that is as close to the ascertainment and design of the original GWAS as possible.<sup><a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359869">43</a></sup> It is important for a replication data set to be independent of the primary data set, have large enough sample sizes and have the same ascertainment and study design as the discovery GWAS.<sup><a href="#bib42" rid="bib42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359883">42</a>, <a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359889">43</a></sup></p><p id="__p30">The AGP and SSC data sets did not have the same ascertainment criteria and these differences have led to key differences in the AGP and SSC data sets. The AGP contains both simplex and multiplex families (approximately 38% of families are simplex<sup><a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359859">44</a></sup>) with the aim being to investigate common variation whereas the SSC data contains only simplex families, a design which inherently enriches for rare and <em>de novo</em> mutations.<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359868">28</a>, <a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359858">33</a></sup> In addition, the SSC data set excluded families with parents who met criteria for a spectrum diagnosis based on two instruments, thus further limiting the potential to discover heritable, penetrant genetic risk. This exclusion criterion did not apply in the AGP data set and a small proportion of parents included in the data set who had been screened using these instruments (which is only a small proportion of the sample) meet this criteria.<sup><a href="#bib45" rid="bib45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359864">45</a>, <a href="#bib46" rid="bib46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359878">46</a></sup></p><p id="__p31">From a phenotypic perspective, when compared with multiplex families, simplex family members share less ASD traits.<sup><a href="#bib47" rid="bib47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359850">47</a>, <a href="#bib48" rid="bib48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359882">48</a></sup> Klei <em>et al</em><sup><a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359888">43</a></sup> have shown that a lower proportion (&lt;40%) of the heritability of additive effects in ASD is explained in the SSC data set compared with the AGP (55–59%, 65% for AGP multiplex probands) and that family members in the AGP have elevated heritability estimates, which were not seen in the SSC. There has been evidence to suggest that genetic transmission mechanisms differ between multiplex families and simplex families.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359862">7</a>, <a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359881">44</a>, <a href="#bib48" rid="bib48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359863">48</a></sup> Even though both the AGP and SSC data sets are ASD data sets, and even though we would expect some shared common risk between the two in terms of associations, we do not believe this would be the case for parent-of-origin effects as the transmission mechanisms are the main focus. For these reasons, we felt it was not appropriate to treat either the AGP or SSC as a replication data set of the other. <a href="/pmc/articles/PMC5255953/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S4–S11</a> in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> show that the effects are often in different directions when comparing the AGP and SSC results to each other, possibly strengthening the theory that this difference in ascertainment leads to a different genetic aetiology for multiplex and simplex ASD families.</p><p id="__p32">Therefore, we have not replicated any of the findings we identified as we did not have an appropriate independent replication data set available. However, we did identify some potential parent-of-origin effects in ASD in regions that have been previously implicated in ASD. For example, one of the imprinting results (in the <em>STPG2</em> gene) and two of the maternal genetic effects (in the <em>MAML2</em> gene and the <em>ROBO2</em> gene) were previously implicated in an ASD study for maternal genetic effects by Tsang <em>et al.</em><sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359876">18</a></sup> We also identified a maternal genetic effect at rs5770820 in the <em>SHANK3</em> gene. <em>SHANK3</em> (<em>ProSAP2</em>) regulates the structural organisation of dendritic spines and is a binding partner of neuroligins. Mutations in <em>SHANK3</em> are well known risk factors for ASD.<sup><a href="#bib49" rid="bib49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359853">49</a></sup> Lebold <em>et al</em><sup><a href="#bib50" rid="bib50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359866">50</a></sup> estimated that 0.69% of cases with ASD had heterozygous truncating mutations in <em>SHANK3</em>. Sanders <em>et al</em><sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359865">6</a></sup> has identified <em>SHANK3</em> as one of the 71 risk loci in ASD, although we note that Sanders <em>et al</em><sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359872">6</a></sup> also used the SSC data set. We identified another maternal genetic effect on chromosome 7q11 in the <em>WBSCR17</em> gene, which is a deleted region in Williams syndrome. Williams syndrome has strong links with ASD as individuals with Williams syndrome often exhibit autistic traits.<sup><a href="#bib38" rid="bib38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_601359867">38</a></sup> Our findings suggest that mutations in a mother's <em>SHANK3</em> gene or the <em>WBSCR17</em> gene could increase the likelihood of the offspring having ASD. Although our findings are very promising, further investigation is necessary.</p><p id="__p33" class="p p-last">In conclusion, we set out to detect parent-of-origin effects in ASD using the AGP and the SSC GWAS data sets. We identified many regions with potential parent-of-origin effects. This study has also shown an approach to investigating both imprinting effects and maternal genetic effects in ASD family GWAS data sets using appropriate Bayesian thresholds that take into account the power of the test at each SNP. This approach can be used in future studies of ASD when there are larger and more appropriate replication data sets available in order to produce robust findings. This approach is not limited to ASD but is suitable for the examination of parent-of-origin effects in other phenotypes that have GWAS data sets with parental genotypes available.</p></div><div id="idm139831919357584" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139831919357584title">Acknowledgments</h2><div class="sec"><p id="__p34">The authors are grateful to all of the families for participating in the AGP, as well as the principal investigators, AGRE and the AGP funders: Autism Speaks (USA), the Medical Research Council (UK), the Health Research Board (Ireland), the National Institutes of Health (USA), Genome Canada, the Canadian Institutes of Health Research and the Hilibrand Foundation. The AGP data sets can be obtained from dbGaP at <a href="http://www.ncbi.nlm.nih.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gap</a>. The authors are grateful to all of the families at the participating SSC sites, as well as the principal investigators (A Beaudet, R Bernier, J Constantino, E Cook, E Fombonne, D Geschwind, R Goin-Kochel, E Hanson, D Grice, A Klin, D Ledbetter, C Lord, C Martin, D Martin, R Maxim, J Miles, O Ousley, K Pelphrey, B Peterson, J Piggot, C Saulnier, M State, W Stone, J Sutcliffe, C Walsh, Z Warren and E Wijsman), and appreciate obtaining access to phenotypic data on SFARI Base. Approved researchers can obtain the SSC population data set described in this study by applying at <a href="https://base.sfari.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://base.sfari.org</a>. In particular, We acknowledge S Sanders for his help with the SSC data. We acknowledge the support of the Trinity Centre for High Performance Computing. SC acknowledges the funding received from a Trinity College Dublin Studentship Award.</p><p id="__p35">Summary data results are available at <a href="http://www.gwascentral.org/study/HGVST1842" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gwascentral.org/study/HGVST1842</a>.</p></div></div><div id="idm139831884489584" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139831884489584title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm139831866126608"></p><p id="__p36" class="p p-first-last"><a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> accompanies this paper on European Journal of Human Genetics website (http://www.nature.com/ejhg)</p><p></p><p class="fn sec" id="idm139831866124976"></p><p id="__p37" class="p p-first-last">The authors declare no conflict of interest.</p><p></p></div></div><div id="sup1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sup1title">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="xob1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="idm139831882585872"><a href="/pmc/articles/PMC5255953/bin/ejhg2016153x1.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(15M, pdf)</sup></div></div></div><div id="idm139831866124016" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139831866124016title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="bib1"><span class="mixed-citation">Hallmayer J, Cleveland S, Torres A et al: <span class="ref-title">Genetic heritability and shared environmental factors among twin pairs with autism</span>. <span class="ref-journal">Arch Gen Psychiatry</span>
2011; <span class="ref-vol">68</span>: 1095–1102. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4440679/">PMC free article</a>]</span> [<a href="/pubmed/21727249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Genetic+heritability+and+shared+environmental+factors+among+twin+pairs+with+autism&amp;volume=68&amp;publication_year=2011&amp;pages=1095&amp;pmid=21727249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib2"><span class="mixed-citation">Gaugler T, Klei L, Sanders SJ et al: <span class="ref-title">Most genetic risk for autism resides with common variation</span>. <span class="ref-journal">Nat Genet</span>
2014; <span class="ref-vol">46</span>: 881–885. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4137411/">PMC free article</a>]</span> [<a href="/pubmed/25038753" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Most+genetic+risk+for+autism+resides+with+common+variation&amp;volume=46&amp;publication_year=2014&amp;pages=881&amp;pmid=25038753&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib3"><span class="mixed-citation">Anney R, Klei L, Pinto D et al: <span class="ref-title">A genome-wide scan for common alleles affecting risk for autism</span>. <span class="ref-journal">Hum Mol Genet</span>
2010; <span class="ref-vol">19</span>: 4072–4082. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947401/">PMC free article</a>]</span> [<a href="/pubmed/20663923" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=A+genome-wide+scan+for+common+alleles+affecting+risk+for+autism&amp;volume=19&amp;publication_year=2010&amp;pages=4072&amp;pmid=20663923&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib4"><span class="mixed-citation">Anney R, Klei L, Pinto D et al: <span class="ref-title">Individual common variants exert weak effects on the risk for autism spectrum disorderspi</span>. <span class="ref-journal">Hum Mol Genet</span>
2012; <span class="ref-vol">21</span>: 4781–4792. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3471395/">PMC free article</a>]</span> [<a href="/pubmed/22843504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Individual+common+variants+exert+weak+effects+on+the+risk+for+autism+spectrum+disorderspi&amp;volume=21&amp;publication_year=2012&amp;pages=4781&amp;pmid=22843504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib5"><span class="mixed-citation">Chaste P, Klei L, Sanders SJ et al: <span class="ref-title">A genomewide association study of autism using the Simons Simplex Collection: does reducing phenotypic heterogeneity in autism increase genetic homogeneity</span>? <span class="ref-journal">Biol Psychiatry</span>
2014; <span class="ref-vol">77</span>: 775–784. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4379124/">PMC free article</a>]</span> [<a href="/pubmed/25534755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=A+genomewide+association+study+of+autism+using+the+Simons+Simplex+Collection:+does+reducing+phenotypic+heterogeneity+in+autism+increase+genetic+homogeneity&amp;volume=77&amp;publication_year=2014&amp;pages=775&amp;pmid=25534755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib6"><span class="mixed-citation">Sanders S, He X, Willsey JA et al: <span class="ref-title">Insights into Autism Spectrum Disorder genomic architecture and biology from 71 risk loci</span>. <span class="ref-journal">Neuron</span>
2015; <span class="ref-vol">87</span>: 1215–1233. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4624267/">PMC free article</a>]</span> [<a href="/pubmed/26402605" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Insights+into+Autism+Spectrum+Disorder+genomic+architecture+and+biology+from+71+risk+loci&amp;volume=87&amp;publication_year=2015&amp;pages=1215&amp;pmid=26402605&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib7"><span class="mixed-citation">Chaste P, Leboyer M: <span class="ref-title">Autism risk factors: genes, environment, and gene-environment interactions</span>. <span class="ref-journal">Dialogues Clin Neurosci</span>
2012; <span class="ref-vol">14</span>: 281–292. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3513682/">PMC free article</a>]</span> [<a href="/pubmed/23226953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dialogues+Clin+Neurosci&amp;title=Autism+risk+factors:+genes,+environment,+and+gene-environment+interactions&amp;volume=14&amp;publication_year=2012&amp;pages=281&amp;pmid=23226953&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib8"><span class="mixed-citation">Guilmatre A, Sharp A: <span class="ref-title">Parent of origin effects</span>. <span class="ref-journal">Clin Genet</span>
2012; <span class="ref-vol">81</span>: 201–209. [<a href="/pubmed/21933173" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Genet&amp;title=Parent+of+origin+effects&amp;volume=81&amp;publication_year=2012&amp;pages=201&amp;pmid=21933173&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib9"><span class="mixed-citation">Falls J, Pulford D, Wylie A et al: <span class="ref-title">Genomic imprinting: implications for human disease</span>. <span class="ref-journal">Am J Pathol</span>
1999; <span class="ref-vol">154</span>: 635–647. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1866410/">PMC free article</a>]</span> [<a href="/pubmed/10079240" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Pathol&amp;title=Genomic+imprinting:+implications+for+human+disease&amp;volume=154&amp;publication_year=1999&amp;pages=635&amp;pmid=10079240&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib10"><span class="mixed-citation">Gurrier F, Accadia M: <span class="ref-title">Genetic imprinting: the paradigm of prader-willi and angelman syndromes</span>. <span class="ref-journal">Endocr Dev</span>
2009; <span class="ref-vol">14</span>: 20–28. [<a href="/pubmed/19293572" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocr+Dev&amp;title=Genetic+imprinting:+the+paradigm+of+prader-willi+and+angelman+syndromes&amp;volume=14&amp;publication_year=2009&amp;pages=20&amp;pmid=19293572&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib11"><span class="mixed-citation">Veltman MWM, Craig EE, Bolton PF: <span class="ref-title">Autism spectrum disorders in Prader-Willi and Angelman syndromes: a systematic review</span>. <span class="ref-journal">Psychiatr Genet</span>
2005; <span class="ref-vol">15</span>: 243–254. [<a href="/pubmed/16314754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychiatr+Genet&amp;title=Autism+spectrum+disorders+in+Prader-Willi+and+Angelman+syndromes:+a+systematic+review&amp;volume=15&amp;publication_year=2005&amp;pages=243&amp;pmid=16314754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib12"><span class="mixed-citation">Schanen NC: <span class="ref-title">Epigenetics of autism spectrum disorders</span>. <span class="ref-journal">Hum Mol Genet</span>
2006; <span class="ref-vol">15</span>: R138–R150. [<a href="/pubmed/16987877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Epigenetics+of+autism+spectrum+disorders&amp;volume=15&amp;publication_year=2006&amp;pages=R138&amp;pmid=16987877&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib13"><span class="mixed-citation">Williams TA, Mars AE, Buyske SG et al: <span class="ref-title">Risk of autistic disorder in affected offspring of mothers with a glutathione S-transferase P1 haplotype</span>. <span class="ref-journal">Arch Pediatr Adolesc Med</span>
2007; <span class="ref-vol">161</span>: 356–361. [<a href="/pubmed/17404132" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Pediatr+Adolesc+Med&amp;title=Risk+of+autistic+disorder+in+affected+offspring+of+mothers+with+a+glutathione+S-transferase+P1+haplotype&amp;volume=161&amp;publication_year=2007&amp;pages=356&amp;pmid=17404132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib14"><span class="mixed-citation">Johnson WG, Buyske S, Mars AE et al: <span class="ref-title">HLA-DR4 as a risk allele for autism acting in mothers of probands possibly during pregnancy</span>. <span class="ref-journal">Arch Pediatr Adolesc Med</span>
2009; <span class="ref-vol">163</span>: 542–546. [<a href="/pubmed/19487610" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Pediatr+Adolesc+Med&amp;title=HLA-DR4+as+a+risk+allele+for+autism+acting+in+mothers+of+probands+possibly+during+pregnancy&amp;volume=163&amp;publication_year=2009&amp;pages=542&amp;pmid=19487610&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib15"><span class="mixed-citation">Constantino JN, Todorov A, Hilton C et al: <span class="ref-title">Autism recurrence in half siblings: strong support for genetic mechanisms of transmission in ASD</span>. <span class="ref-journal">Mol Psychiatry</span>
2013; <span class="ref-vol">18</span>: 137–138. [<a href="/pubmed/22371046" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Autism+recurrence+in+half+siblings:+strong+support+for+genetic+mechanisms+of+transmission+in+ASD&amp;volume=18&amp;publication_year=2013&amp;pages=137&amp;pmid=22371046&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib16"><span class="mixed-citation">Sandin S, Lichtenstein P, Kuja-Halkola R, Larsson H, Hultman CM, Reichenberg A: <span class="ref-title">The familial risk of autism</span>. <span class="ref-journal">JAMA</span>
2014; <span class="ref-vol">311</span>: 1770–1777. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4381277/">PMC free article</a>]</span> [<a href="/pubmed/24794370" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=The+familial+risk+of+autism&amp;volume=311&amp;publication_year=2014&amp;pages=1770&amp;pmid=24794370&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib17"><span class="mixed-citation">Lawson Ha, Cheverud JM, Wolf JB: <span class="ref-title">Genomic imprinting and parent-of-origin effects on complex traits</span>. <span class="ref-journal">Nat Rev Genet</span>
2013; <span class="ref-vol">14</span>: 609–617. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3926806/">PMC free article</a>]</span> [<a href="/pubmed/23917626" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genomic+imprinting+and+parent-of-origin+effects+on+complex+traits&amp;volume=14&amp;publication_year=2013&amp;pages=609&amp;pmid=23917626&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib18"><span class="mixed-citation">Tsang KM, Croen LA, Torres AR et al: <span class="ref-title">A genome-wide survey of transgenerational genetic effects in autism</span>. <span class="ref-journal">PloS One</span>
2013; <span class="ref-vol">8</span>: e76978. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3811986/">PMC free article</a>]</span> [<a href="/pubmed/24204716" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+One&amp;title=A+genome-wide+survey+of+transgenerational+genetic+effects+in+autism&amp;volume=8&amp;publication_year=2013&amp;pages=e76978&amp;pmid=24204716&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib19"><span class="mixed-citation">Yuan H, Dougherty JD: <span class="ref-title">Investigation of maternal genotype effects in autism by genome- wide association</span>. <span class="ref-journal">Autism Res</span>
2014; <span class="ref-vol">7</span>: 245–253. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3989385/">PMC free article</a>]</span> [<a href="/pubmed/24574247" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autism+Res&amp;title=Investigation+of+maternal+genotype+effects+in+autism+by+genome-+wide+association&amp;volume=7&amp;publication_year=2014&amp;pages=245&amp;pmid=24574247&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib20"><span class="mixed-citation">Cordell H, Barratt B, Clayton D: <span class="ref-title">Case/pseudocontrol analysis in genetic association studies: a unified framework for detection of genotype and haplotype associations, gene-gene and gene-environment interactions, and parent-of-origin effects</span>. <span class="ref-journal">Genet Epidemiol</span>
2004; <span class="ref-vol">26</span>: 167–185. [<a href="/pubmed/15022205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Case/pseudocontrol+analysis+in+genetic+association+studies:+a+unified+framework+for+detection+of+genotype+and+haplotype+associations,+gene-gene+and+gene-environment+interactions,+and+parent-of-origin+effects&amp;volume=26&amp;publication_year=2004&amp;pages=167&amp;pmid=15022205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib21"><span class="mixed-citation">Wolf J, Wade M: <span class="ref-title">What are maternal effects (and what are they not)</span>? <span class="ref-journal">Philos Trans R Soc Lond B Biol Sci</span>
2009; <span class="ref-vol">10</span>: 1107–1115. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2666680/">PMC free article</a>]</span> [<a href="/pubmed/19324615" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Philos+Trans+R+Soc+Lond+B+Biol+Sci&amp;title=What+are+maternal+effects+(and+what+are+they+not)&amp;volume=10&amp;publication_year=2009&amp;pages=1107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib22"><span class="mixed-citation">Connolly S, Heron EA: <span class="ref-title">Review of statistical methodologies for the detection of parent- of-origin effects in family trio genome-wide association data with binary disease traits</span>. <span class="ref-journal">Brief Bioinform</span>
2014; <span class="ref-vol">16</span>: 429–448. [<a href="/pubmed/24903222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brief+Bioinform&amp;title=Review+of+statistical+methodologies+for+the+detection+of+parent-+of-origin+effects+in+family+trio+genome-wide+association+data+with+binary+disease+traits&amp;volume=16&amp;publication_year=2014&amp;pages=429&amp;pmid=24903222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib23"><span class="mixed-citation">Ainsworth H, Unwin J, Jamison D et al: <span class="ref-title">Investigation of maternal effects, maternal-fetal interactions and parent-of-origin effects (imprinting), using mothers and their offspring</span>. <span class="ref-journal">Genet Epidemiol</span>
2011; <span class="ref-vol">35</span>: 19–45. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025173/">PMC free article</a>]</span> [<a href="/pubmed/21181895" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Investigation+of+maternal+effects,+maternal-fetal+interactions+and+parent-of-origin+effects+(imprinting),+using+mothers+and+their+offspring&amp;volume=35&amp;publication_year=2011&amp;pages=19&amp;pmid=21181895&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib24"><span class="mixed-citation">Howey R, Cordell H: <span class="ref-title">PREMIM and EMIM: tools for estimation of maternal, imprinting and interaction effects using multinomial modelling</span>. <span class="ref-journal">BMC Bioinformatics</span>
2012; <span class="ref-vol">13</span>: 149. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3464602/">PMC free article</a>]</span> [<a href="/pubmed/22738121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=PREMIM+and+EMIM:+tools+for+estimation+of+maternal,+imprinting+and+interaction+effects+using+multinomial+modelling&amp;volume=13&amp;publication_year=2012&amp;pages=149&amp;pmid=22738121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib25"><span class="mixed-citation">Wakefield J: <span class="ref-title">Commentary: genome-wide significance thresholds via Bayes factors</span>. <span class="ref-journal">Int J Epidemiol</span>
2012; <span class="ref-vol">41</span>: 286–291. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3304534/">PMC free article</a>]</span> [<a href="/pubmed/22345299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=Commentary:+genome-wide+significance+thresholds+via+Bayes+factors&amp;volume=41&amp;publication_year=2012&amp;pages=286&amp;pmid=22345299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib26"><span class="mixed-citation">Ioannidis JPA, Thomas G, Daly MJ: <span class="ref-title">Validating, augmenting and refining genome-wide association signals</span>. <span class="ref-journal">Nat Rev Genet</span>
2009; <span class="ref-vol">10</span>: 318–329. [<a href="/pubmed/19373277" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Validating,+augmenting+and+refining+genome-wide+association+signals&amp;volume=10&amp;publication_year=2009&amp;pages=318&amp;pmid=19373277&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib27"><span class="mixed-citation">Fischbach GD, Lord C: <span class="ref-title">The Simons Simplex Collection: a resource for identification of autism genetic risk factors</span>. <span class="ref-journal">Neuron</span>
2010; <span class="ref-vol">68</span>: 192–195. [<a href="/pubmed/20955926" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=The+Simons+Simplex+Collection:+a+resource+for+identification+of+autism+genetic+risk+factors&amp;volume=68&amp;publication_year=2010&amp;pages=192&amp;pmid=20955926&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib28"><span class="mixed-citation">Sanders SJ, Ercan-Sencicek GA, Hus V et al: <span class="ref-title">Multiple recurrent <em>de novo</em> CNVs, including duplications of the 7q11.23 Williams syndrome region, are strongly associated with autism</span>. <span class="ref-journal">Neuron</span>
2011; <span class="ref-vol">70</span>: 863–885. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3939065/">PMC free article</a>]</span> [<a href="/pubmed/21658581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Multiple+recurrent+de+novo+CNVs,+including+duplications+of+the+7q11.23+Williams+syndrome+region,+are+strongly+associated+with+autism&amp;volume=70&amp;publication_year=2011&amp;pages=863&amp;pmid=21658581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib29"><span class="mixed-citation">Howey R, Mamasoula C, Töpf A et al: <span class="ref-title">Increased power for detection of parent-of-origin effects via the use of haplotype estimation</span>. <span class="ref-journal">Am J Hum Genet</span>
2015; <span class="ref-vol">97</span>: 419–434. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4564992/">PMC free article</a>]</span> [<a href="/pubmed/26320892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Increased+power+for+detection+of+parent-of-origin+effects+via+the+use+of+haplotype+estimation&amp;volume=97&amp;publication_year=2015&amp;pages=419&amp;pmid=26320892&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib30"><span class="mixed-citation">Johansen CT, Wang J, McIntyre AD et al: <span class="ref-title">Excess of rare variants in non-genome-wide association study candidate genes in patients with hypertriglyceridemia</span>. <span class="ref-journal">Circ Cardiovasc Genet</span>
2012; <span class="ref-vol">5</span>: 66–72. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3288444/">PMC free article</a>]</span> [<a href="/pubmed/22135386" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Excess+of+rare+variants+in+non-genome-wide+association+study+candidate+genes+in+patients+with+hypertriglyceridemia&amp;volume=5&amp;publication_year=2012&amp;pages=66&amp;pmid=22135386&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib31"><span class="mixed-citation">Wakefield J, Skrivankova V, Hsu FC, Sale M, Heagerty P: <span class="ref-title">Detecting signals in pharmacogenomic genome-wide association studies</span>. <span class="ref-journal">Pharmacogenomics J</span>
2014; <span class="ref-vol">14</span>: 309–315. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4085158/">PMC free article</a>]</span> [<a href="/pubmed/24394200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics+J&amp;title=Detecting+signals+in+pharmacogenomic+genome-wide+association+studies&amp;volume=14&amp;publication_year=2014&amp;pages=309&amp;pmid=24394200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib32"><span class="mixed-citation">Smith J, Newton-Cheh C: <span class="ref-title">Genome-wide association study in humans</span>. <span class="ref-journal">Methods Mol Biol</span>
2009; <span class="ref-vol">573</span>: 231–258. [<a href="/pubmed/19763932" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=Genome-wide+association+study+in+humans&amp;volume=573&amp;publication_year=2009&amp;pages=231&amp;pmid=19763932&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib33"><span class="mixed-citation">Sanders SJ, Murtha MT, Gupta AR et al: <span class="ref-title"><em>De novo</em> mutations revealed by whole-exome sequencing are strongly associated with autism</span>. <span class="ref-journal">Nature</span>
2012; <span class="ref-vol">485</span>: 237–241. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3667984/">PMC free article</a>]</span> [<a href="/pubmed/22495306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=De+novo+mutations+revealed+by+whole-exome+sequencing+are+strongly+associated+with+autism&amp;volume=485&amp;publication_year=2012&amp;pages=237&amp;pmid=22495306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib34"><span class="mixed-citation">Iossifov I, Ronemus M, Levy D et al: <span class="ref-title"><em>De novo</em> gene disruptions in children on the autistic spectrum</span>. <span class="ref-journal">Neuron</span>
2012; <span class="ref-vol">74</span>: 285–299. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3619976/">PMC free article</a>]</span> [<a href="/pubmed/22542183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=De+novo+gene+disruptions+in+children+on+the+autistic+spectrum&amp;volume=74&amp;publication_year=2012&amp;pages=285&amp;pmid=22542183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib35"><span class="mixed-citation">Nudel R, Simpson NH, Baird G et al: <span class="ref-title">Genome-wide association analyses of child genotype effects and parent-of-origin effects in specific language impairment</span>. <span class="ref-journal">Genes Brain Behav</span>
2014; <span class="ref-vol">13</span>: 418–429. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4114547/">PMC free article</a>]</span> [<a href="/pubmed/24571439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Brain+Behav&amp;title=Genome-wide+association+analyses+of+child+genotype+effects+and+parent-of-origin+effects+in+specific+language+impairment&amp;volume=13&amp;publication_year=2014&amp;pages=418&amp;pmid=24571439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib36"><span class="mixed-citation">Gosso MF, van Belzen M, de Geus EJC et al: <span class="ref-title">Association between the CHRM2 gene and intelligence in a sample of 304 Dutch families</span>. <span class="ref-journal">Genes Brain Behav</span>
2006; <span class="ref-vol">5</span>: 577–584. [<a href="/pubmed/17081262" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Brain+Behav&amp;title=Association+between+the+CHRM2+gene+and+intelligence+in+a+sample+of+304+Dutch+families&amp;volume=5&amp;publication_year=2006&amp;pages=577&amp;pmid=17081262&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib37"><span class="mixed-citation">Peça J, Feliciano C, Ting JT et al: <span class="ref-title">Shank3 mutant mice display autistic-like behaviours and striatal dysfunction</span>. <span class="ref-journal">Nature</span>
2011; <span class="ref-vol">472</span>: 437–442. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3090611/">PMC free article</a>]</span> [<a href="/pubmed/21423165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Shank3+mutant+mice+display+autistic-like+behaviours+and+striatal+dysfunction&amp;volume=472&amp;publication_year=2011&amp;pages=437&amp;pmid=21423165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib38"><span class="mixed-citation">Francke U: <span class="ref-title">Williams-Beuren syndrome:genes and mechanisms</span>. <span class="ref-journal">Hum Mol Genet</span>
1999; <span class="ref-vol">8</span>: 1947–1954. [<a href="/pubmed/10469848" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Williams-Beuren+syndrome:genes+and+mechanisms&amp;volume=8&amp;publication_year=1999&amp;pages=1947&amp;pmid=10469848&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib39"><span class="mixed-citation">Klein-Tasman BP, Phillips KD, Lord C, Mervis CB, Gallo FJ: <span class="ref-title">Overlap with the autism spectrum in young children with Williams syndrome</span>. <span class="ref-journal">J Dev Behav Pediatr</span>
2009; <span class="ref-vol">30</span>: 289–299. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2763277/">PMC free article</a>]</span> [<a href="/pubmed/19668090" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Dev+Behav+Pediatr&amp;title=Overlap+with+the+autism+spectrum+in+young+children+with+Williams+syndrome&amp;volume=30&amp;publication_year=2009&amp;pages=289&amp;pmid=19668090&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib40"><span class="mixed-citation">Tordjman S, Anderson GM, Botbol M et al: <span class="ref-title">Autistic disorder in patients with Williams-Beuren syndrome: a reconsideration of the Williams-Beuren syndrome phenotype</span>. <span class="ref-journal">PloS One</span>
2012; <span class="ref-vol">7</span>: e30778. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3295800/">PMC free article</a>]</span> [<a href="/pubmed/22412832" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PloS+One&amp;title=Autistic+disorder+in+patients+with+Williams-Beuren+syndrome:+a+reconsideration+of+the+Williams-Beuren+syndrome+phenotype&amp;volume=7&amp;publication_year=2012&amp;pages=e30778&amp;pmid=22412832&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib41"><span class="mixed-citation">Chen YZ, Matsushita M, Girirajan S et al: <span class="ref-title">Evidence for involvement of GNB1L in autism</span>. <span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet</span>
2012; <span class="ref-vol">159B</span>: 61–71. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3270696/">PMC free article</a>]</span> [<a href="/pubmed/22095694" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Evidence+for+involvement+of+GNB1L+in+autism&amp;volume=159B&amp;publication_year=2012&amp;pages=61&amp;pmid=22095694&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib42"><span class="mixed-citation">Kraft P, Zeggini E, Ioannidis JP: <span class="ref-title">Replication in genome-wide association studies</span>. <span class="ref-journal">Stat Sci</span>
2010; <span class="ref-vol">24</span>: 561. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2865141/">PMC free article</a>]</span> [<a href="/pubmed/20454541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Sci&amp;title=Replication+in+genome-wide+association+studies&amp;volume=24&amp;publication_year=2010&amp;pages=561&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib43"><span class="mixed-citation">Bush WS, Moore JH: <span class="ref-title">Chapter 11: Genome-wide association studies</span>. <span class="ref-journal">PLoS Comput Biol</span>
2012; <span class="ref-vol">8</span>: e1002822. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3531285/">PMC free article</a>]</span> [<a href="/pubmed/23300413" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput+Biol&amp;title=Chapter+11:+Genome-wide+association+studies&amp;volume=8&amp;publication_year=2012&amp;pages=e1002822&amp;pmid=23300413&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib44"><span class="mixed-citation">Klei L, Sanders SJ, Murtha MT et al: <span class="ref-title">Common genetic variants, acting additively, are a major source of risk for autism</span>. <span class="ref-journal">Mol Autism</span>
2012; <span class="ref-vol">3</span>: 9. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3579743/">PMC free article</a>]</span> [<a href="/pubmed/23067556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Autism&amp;title=Common+genetic+variants,+acting+additively,+are+a+major+source+of+risk+for+autism&amp;volume=3&amp;publication_year=2012&amp;pages=9&amp;pmid=23067556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib45"><span class="mixed-citation">Neale BM, Kou Y, Liu L et al: <span class="ref-title">Patterns and rates of exonic <em>de novo</em> mutations in autism spectrum disorders</span>. <span class="ref-journal">Nature</span>
2012; <span class="ref-vol">485</span>: 242–245. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3613847/">PMC free article</a>]</span> [<a href="/pubmed/22495311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Patterns+and+rates+of+exonic+de+novo+mutations+in+autism+spectrum+disorders&amp;volume=485&amp;publication_year=2012&amp;pages=242&amp;pmid=22495311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib46"><span class="mixed-citation">Pinto D, Pagnamenta AT, Klei L et al: <span class="ref-title">Functional impact of global rare copy number variation in autism spectrum disorders</span>. <span class="ref-journal">Nature</span>
2010; <span class="ref-vol">466</span>: 368–372. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3021798/">PMC free article</a>]</span> [<a href="/pubmed/20531469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Functional+impact+of+global+rare+copy+number+variation+in+autism+spectrum+disorders&amp;volume=466&amp;publication_year=2010&amp;pages=368&amp;pmid=20531469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib47"><span class="mixed-citation">Gerdts JA, Bernier R, Dawson G, Estes A: <span class="ref-title">The broader autism phenotype in simplex and multiplex families</span>. <span class="ref-journal">J Autism Dev Disord</span>
2013; <span class="ref-vol">43</span>: 1597–1605. [<a href="/pubmed/23117424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autism+Dev+Disord&amp;title=The+broader+autism+phenotype+in+simplex+and+multiplex+families&amp;volume=43&amp;publication_year=2013&amp;pages=1597&amp;pmid=23117424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib48"><span class="mixed-citation">Bernier R, Gerdts J, Munson J, Dawson G, Estes A: <span class="ref-title">Evidence for broader autism phenotype characteristics in parents from multiple-incidence autism families</span>. <span class="ref-journal">Autism Res</span>
2012; <span class="ref-vol">5</span>: 13–20. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3237782/">PMC free article</a>]</span> [<a href="/pubmed/21905246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autism+Res&amp;title=Evidence+for+broader+autism+phenotype+characteristics+in+parents+from+multiple-incidence+autism+families&amp;volume=5&amp;publication_year=2012&amp;pages=13&amp;pmid=21905246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib49"><span class="mixed-citation">Durand CM, Betancur C, Boeckers TM et al: <span class="ref-title">Mutations in the gene encoding the synaptic scaffolding protein SHANK3 are associated with autism spectrum disorders</span>. <span class="ref-journal">Nat Genet</span>
2007; <span class="ref-vol">39</span>: 25–27. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2082049/">PMC free article</a>]</span> [<a href="/pubmed/17173049" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Mutations+in+the+gene+encoding+the+synaptic+scaffolding+protein+SHANK3+are+associated+with+autism+spectrum+disorders&amp;volume=39&amp;publication_year=2007&amp;pages=25&amp;pmid=17173049&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib50"><span class="mixed-citation">Leblond CS, Nava C, Polge A et al: <span class="ref-title">Meta-analysis of SHANK Mutations in Autism Spectrum Disorders: a gradient of severity in cognitive impairments</span>. <span class="ref-journal">PLoS Genet</span>
2014; <span class="ref-vol">10</span>: e1004580. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4154644/">PMC free article</a>]</span> [<a href="/pubmed/25188300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Meta-analysis+of+SHANK+Mutations+in+Autism+Spectrum+Disorders:+a+gradient+of+severity+in+cognitive+impairments&amp;volume=10&amp;publication_year=2014&amp;pages=e1004580&amp;pmid=25188300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5255953&amp;issue-id=285295&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139831919422544title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Materials and methods</a></li><li><a href="#__sec7title">Results</a></li><li><a href="#__sec14title">Discussion</a></li><li><a href="#idm139831919357584title">Acknowledgments</a></li><li><a href="#idm139831884489584title">Footnotes</a></li><li><a href="#sup1title">Supplementary Material</a></li><li><a href="#idm139831866124016title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5255953/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5255953/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5255953/pdf/ejhg2016153a.pdf">PDF (426K)</a></li> | <li><a href="#" data-citationid="PMC5255953" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5255953%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5255953%2F&amp;text=A%20genome-wide%20investigation%20into%20parent-of-origin%20effects%20in%20autism%20spectrum%20disorder%20identifies%20previously%20associated%20genes%20including%20SHANK3" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5255953%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27876814" data-aiid="5255953" data-aid="5255953" data-iid="285295" data-domainid="1300" data-domain="ejhg" data-accid="PMC5255953" data-md5="cc93be3d2b86028e9c59e179ef697b0f"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:22:04-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88C02CDD592CE1000000000911055E&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5255953%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5255953/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>