<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5805394/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Investigating the genetic architecture of dementia with Lewy bodies: a two-stage genome-wide association study</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5805394/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="The Lancet. Neurology"><meta name="citation_title" content="Investigating the genetic architecture of dementia with Lewy bodies: a two-stage genome-wide association study"><meta name="citation_authors" content="Rita Guerreiro, Owen A Ross, Celia Kun-Rodrigues, Dena G Hernandez, Tatiana Orme, John D Eicher, Claire E Shepherd, Laura Parkkinen, Lee Darwent, Michael G Heckman, Sonja W Scholz, Prof Juan C Troncoso, Olga Pletnikova, Olaf Ansorge, Jordi Clarimon, Alberto Lleo, Estrella Morenas-Rodriguez, Lorraine Clark, Prof Lawrence S Honig, Prof Karen Marder, Afina Lemstra, Prof Ekaterina Rogaeva, Prof Peter St George-Hyslop, Elisabet Londos, Prof Henrik Zetterberg, Imelda Barber, Anne Braae, Kristelle Brown, Prof Kevin Morgan, Claire Troakes, Prof Safa Al-Sarraj, Tammaryn Lashley, Prof Janice Holton, Yaroslau Compta, Prof Vivianna Van Deerlin, Geidy E Serrano, Thomas G Beach, Suzanne Lesage, Prof Douglas Galasko, Prof Eliezer Masliah, Isabel Santana, Pau Pastor, Monica Diez-Fairen, Miquel Aguilar, Prof Pentti J Tienari, Liisa Myllykangas, Minna Oinas, Prof Tamas Revesz, Prof Andrew Lees, Prof Brad F Boeve, Prof Ronald C Petersen, Tanis J Ferman, Prof Valentina Escott-Price, Prof Neill Graff-Radford, Prof Nigel J Cairns, Prof John C Morris, Prof Stuart Pickering-Brown, Prof David Mann, Prof Glenda M Halliday, Prof John Hardy, Prof John Q Trojanowski, Prof Dennis W Dickson, Andrew Singleton, David J Stone, Jose Bras"><meta name="citation_date" content="January 2018"><meta name="citation_issue" content="1"><meta name="citation_volume" content="17"><meta name="citation_firstpage" content="64"><meta name="citation_doi" content="10.1016/S1474-4422(17)30400-3"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5805394/?report=abstract"><meta name="citation_pmid" content="29263008"><meta name="DC.Title" content="Investigating the genetic architecture of dementia with Lewy bodies: a two-stage genome-wide association study"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Rita Guerreiro"><meta name="DC.Contributor" content="Owen A Ross"><meta name="DC.Contributor" content="Celia Kun-Rodrigues"><meta name="DC.Contributor" content="Dena G Hernandez"><meta name="DC.Contributor" content="Tatiana Orme"><meta name="DC.Contributor" content="John D Eicher"><meta name="DC.Contributor" content="Claire E Shepherd"><meta name="DC.Contributor" content="Laura Parkkinen"><meta name="DC.Contributor" content="Lee Darwent"><meta name="DC.Contributor" content="Michael G Heckman"><meta name="DC.Contributor" content="Sonja W Scholz"><meta name="DC.Contributor" content="Prof Juan C Troncoso"><meta name="DC.Contributor" content="Olga Pletnikova"><meta name="DC.Contributor" content="Olaf Ansorge"><meta name="DC.Contributor" content="Jordi Clarimon"><meta name="DC.Contributor" content="Alberto Lleo"><meta name="DC.Contributor" content="Estrella Morenas-Rodriguez"><meta name="DC.Contributor" content="Lorraine Clark"><meta name="DC.Contributor" content="Prof Lawrence S Honig"><meta name="DC.Contributor" content="Prof Karen Marder"><meta name="DC.Contributor" content="Afina Lemstra"><meta name="DC.Contributor" content="Prof Ekaterina Rogaeva"><meta name="DC.Contributor" content="Prof Peter St George-Hyslop"><meta name="DC.Contributor" content="Elisabet Londos"><meta name="DC.Contributor" content="Prof Henrik Zetterberg"><meta name="DC.Contributor" content="Imelda Barber"><meta name="DC.Contributor" content="Anne Braae"><meta name="DC.Contributor" content="Kristelle Brown"><meta name="DC.Contributor" content="Prof Kevin Morgan"><meta name="DC.Contributor" content="Claire Troakes"><meta name="DC.Contributor" content="Prof Safa Al-Sarraj"><meta name="DC.Contributor" content="Tammaryn Lashley"><meta name="DC.Contributor" content="Prof Janice Holton"><meta name="DC.Contributor" content="Yaroslau Compta"><meta name="DC.Contributor" content="Prof Vivianna Van Deerlin"><meta name="DC.Contributor" content="Geidy E Serrano"><meta name="DC.Contributor" content="Thomas G Beach"><meta name="DC.Contributor" content="Suzanne Lesage"><meta name="DC.Contributor" content="Prof Douglas Galasko"><meta name="DC.Contributor" content="Prof Eliezer Masliah"><meta name="DC.Contributor" content="Isabel Santana"><meta name="DC.Contributor" content="Pau Pastor"><meta name="DC.Contributor" content="Monica Diez-Fairen"><meta name="DC.Contributor" content="Miquel Aguilar"><meta name="DC.Contributor" content="Prof Pentti J Tienari"><meta name="DC.Contributor" content="Liisa Myllykangas"><meta name="DC.Contributor" content="Minna Oinas"><meta name="DC.Contributor" content="Prof Tamas Revesz"><meta name="DC.Contributor" content="Prof Andrew Lees"><meta name="DC.Contributor" content="Prof Brad F Boeve"><meta name="DC.Contributor" content="Prof Ronald C Petersen"><meta name="DC.Contributor" content="Tanis J Ferman"><meta name="DC.Contributor" content="Prof Valentina Escott-Price"><meta name="DC.Contributor" content="Prof Neill Graff-Radford"><meta name="DC.Contributor" content="Prof Nigel J Cairns"><meta name="DC.Contributor" content="Prof John C Morris"><meta name="DC.Contributor" content="Prof Stuart Pickering-Brown"><meta name="DC.Contributor" content="Prof David Mann"><meta name="DC.Contributor" content="Prof Glenda M Halliday"><meta name="DC.Contributor" content="Prof John Hardy"><meta name="DC.Contributor" content="Prof John Q Trojanowski"><meta name="DC.Contributor" content="Prof Dennis W Dickson"><meta name="DC.Contributor" content="Andrew Singleton"><meta name="DC.Contributor" content="David J Stone"><meta name="DC.Contributor" content="Jose Bras"><meta name="DC.Date" content="2018 Jan"><meta name="DC.Identifier" content="10.1016/S1474-4422(17)30400-3"><meta name="DC.Language" content="en"><meta property="og:title" content="Investigating the genetic architecture of dementia with Lewy bodies: a two-stage genome-wide association study"><meta property="og:type" content="article"><meta property="og:description" content="Dementia with Lewy bodies is the second most common form of dementia in elderly people but has been overshadowed in the research field, partly because of similarities between dementia with Lewy bodies, Parkinson’s disease, and Alzheimer’s ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5805394/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5805394/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5805394/pdf/nihms938346.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C31B2DD5ABE510000000005320300.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5805394%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5805394</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Lancet Neurol</a></span>. Author manuscript; available in PMC 2018 Feb 8.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=29263008" target="pmc_ext" ref="reftype=publisher&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Lancet Neurol. 2018 Jan; 17(1): 64–74. </span></a></div><span class="fm-vol-iss-date">Published online 2017 Dec 16. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2FS1474-4422(17)30400-3" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/S1474-4422(17)30400-3</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5805394</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS938346</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29263008">29263008</a></div></div></div></div><h1 class="content-title">Investigating the genetic architecture of dementia with Lewy bodies: a two-stage genome-wide association study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Guerreiro%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Rita Guerreiro</a>, PhD,<sup>*</sup> <a href="/pubmed/?term=Ross%20OA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Owen A Ross</a>, PhD,<sup>*</sup> <a href="/pubmed/?term=Kun-Rodrigues%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Celia Kun-Rodrigues</a>, MSc, <a href="/pubmed/?term=Hernandez%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Dena G Hernandez</a>, PhD, <a href="/pubmed/?term=Orme%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Tatiana Orme</a>, BSc, <a href="/pubmed/?term=Eicher%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">John D Eicher</a>, PhD, <a href="/pubmed/?term=Shepherd%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Claire E Shepherd</a>, PhD, <a href="/pubmed/?term=Parkkinen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Laura Parkkinen</a>, PhD, <a href="/pubmed/?term=Darwent%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Lee Darwent</a>, MSc, <a href="/pubmed/?term=Heckman%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Michael G Heckman</a>, MS, <a href="/pubmed/?term=Scholz%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Sonja W Scholz</a>, PhD, <a href="/pubmed/?term=Troncoso%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Juan C Troncoso</a>, MD, <a href="/pubmed/?term=Pletnikova%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Olga Pletnikova</a>, MD, <a href="/pubmed/?term=Ansorge%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Olaf Ansorge</a>, MD, <a href="/pubmed/?term=Clarimon%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Jordi Clarimon</a>, PhD, <a href="/pubmed/?term=Lleo%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Alberto Lleo</a>, MD, <a href="/pubmed/?term=Morenas-Rodriguez%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Estrella Morenas-Rodriguez</a>, MD, <a href="/pubmed/?term=Clark%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Lorraine Clark</a>, PhD, <a href="/pubmed/?term=Honig%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Lawrence S Honig</a>, PhD, <a href="/pubmed/?term=Marder%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Karen Marder</a>, MD, <a href="/pubmed/?term=Lemstra%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Afina Lemstra</a>, PhD, <a href="/pubmed/?term=Rogaeva%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Ekaterina Rogaeva</a>, PhD, <a href="/pubmed/?term=St%20George-Hyslop%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Peter St George-Hyslop</a>, MD, <a href="/pubmed/?term=Londos%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Elisabet Londos</a>, MD, <a href="/pubmed/?term=Zetterberg%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Henrik Zetterberg</a>, PhD, <a href="/pubmed/?term=Barber%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Imelda Barber</a>, PhD, <a href="/pubmed/?term=Braae%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Anne Braae</a>, PhD, <a href="/pubmed/?term=Brown%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Kristelle Brown</a>, PhD, <a href="/pubmed/?term=Morgan%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Kevin Morgan</a>, PhD, <a href="/pubmed/?term=Troakes%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Claire Troakes</a>, PhD, <a href="/pubmed/?term=Al-Sarraj%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Safa Al-Sarraj</a>, FRCPath, <a href="/pubmed/?term=Lashley%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Tammaryn Lashley</a>, PhD, <a href="/pubmed/?term=Holton%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Janice Holton</a>, PhD, <a href="/pubmed/?term=Compta%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Yaroslau Compta</a>, PhD, <a href="/pubmed/?term=Van%20Deerlin%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Vivianna Van Deerlin</a>, PhD, <a href="/pubmed/?term=Serrano%20GE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Geidy E Serrano</a>, PhD, <a href="/pubmed/?term=Beach%20TG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Thomas G Beach</a>, <a href="/pubmed/?term=Lesage%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Suzanne Lesage</a>, PhD, <a href="/pubmed/?term=Galasko%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Douglas Galasko</a>, MD, <a href="/pubmed/?term=Masliah%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Eliezer Masliah</a>, MD, <a href="/pubmed/?term=Santana%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Isabel Santana</a>, PhD, <a href="/pubmed/?term=Pastor%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Pau Pastor</a>, MD, <a href="/pubmed/?term=Diez-Fairen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Monica Diez-Fairen</a>, BSc, <a href="/pubmed/?term=Aguilar%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Miquel Aguilar</a>, MD, <a href="/pubmed/?term=Tienari%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Pentti J Tienari</a>, PhD, <a href="/pubmed/?term=Myllykangas%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Liisa Myllykangas</a>, PhD, <a href="/pubmed/?term=Oinas%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Minna Oinas</a>, PhD, <a href="/pubmed/?term=Revesz%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Tamas Revesz</a>, PhD, <a href="/pubmed/?term=Lees%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Andrew Lees</a>, MD, <a href="/pubmed/?term=Boeve%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Brad F Boeve</a>, MD, <a href="/pubmed/?term=Petersen%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Ronald C Petersen</a>, PhD, <a href="/pubmed/?term=Ferman%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Tanis J Ferman</a>, PhD, <a href="/pubmed/?term=Escott-Price%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Valentina Escott-Price</a>, PhD, <a href="/pubmed/?term=Graff-Radford%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Neill Graff-Radford</a>, MD, <a href="/pubmed/?term=Cairns%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Nigel J Cairns</a>, PhD, <a href="/pubmed/?term=Morris%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof John C Morris</a>, MD, <a href="/pubmed/?term=Pickering-Brown%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Stuart Pickering-Brown</a>, PhD, <a href="/pubmed/?term=Mann%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof David Mann</a>, PhD, <a href="/pubmed/?term=Halliday%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Glenda M Halliday</a>, PhD, <a href="/pubmed/?term=Hardy%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof John Hardy</a>, PhD, <a href="/pubmed/?term=Trojanowski%20JQ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof John Q Trojanowski</a>, PhD, <a href="/pubmed/?term=Dickson%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Prof Dennis W Dickson</a>, MD, <a href="/pubmed/?term=Singleton%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Andrew Singleton</a>, PhD, <a href="/pubmed/?term=Stone%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">David J Stone</a>, PhD, and  <a href="/pubmed/?term=Bras%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29263008">Jose Bras</a>, PhD</div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140263512146960_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140263512146960_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140263512146960_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">UK Dementia Research Institute (R Guerreiro PhD, J Bras PhD, Prof J Hardy PhD, Prof H Zetterberg PhD), Department of Molecular Neuroscience, UCL Institute of Neurology (R Guerreiro, C Kun-Rodrigues MSc, T Orme BSc, L Darwent MSc, Prof J Hardy, J Bras, Prof H Zetterberg), and Queen Square Brain Bank, Department of Molecular Neuroscience, UCL Institute of Neurology (T Lashley PhD, Prof J Holton PhD, Prof T Revesz PhD, Prof A Lees MD, Y Compta PhD), University College London, London, UK; Department of Medical Sciences and Institute of Biomedicine, iBiMED, University of Aveiro, Aveiro, Portugal (R Guerreiro, J Bras); Department of Neuroscience (O A Ross PhD, Prof D W Dickson MD), Division of Biomedical Statistics and Informatics (M G Heckman MS), Department of Psychiatry (T J Ferman PhD), and Department of Neurology (Prof N Graff-Radford MD), Mayo Clinic, Jacksonville, FL, USA; Laboratory of Neurogenetics, National Institutes on Aging (D G Hernandez PhD, A Singleton PhD, Prof E Masliah MD), Neurodegenerative Diseases Research Unit, National Institute of Neurological Disorders and Stroke (S W Scholz PhD), and Division of Neurosciences (Prof E Masliah), National Institutes of Health, Bethesda, MD, USA; German Center for Neurodegenerative Diseases, Tubingen, Germany (D G Hernandez); Merck &amp; Co, Boston, MA, USA (J D Eicher PhD); Neuroscience Research Australia, Sydney, NSW, Australia (C E Shepherd PhD, Prof G M Halliday PhD); School of Medical Sciences, Faculty of Medicine, University of New South Wales, Sydney, NSW, Australia (C E Shepherd, Prof G M Halliday); Nuffield Department of Clinical Neurosciences, Oxford Parkinson’s Disease Centre, University of Oxford, Oxford, UK (L Parkkinen PhD, O Ansorge MD); Department of Pathology (Neuropathology), Johns Hopkins University School of Medicine, Baltimore, MD, USA (Prof J C Troncoso MD, O Pletnikova MD); Memory Unit, Department of Neurology, IIB Sant Pau, Hospital de la Santa Creu i Sant Pau, Universitat Autonoma de Barcelona, Barcelona, Spain (J Clarimon PhD, A Lleo MD, E Morenas-Rodriguez MD); Centro de Investigacion Biomedica en Red en Enfermedades Neurodegenerativas (CIBERNED), Instituto de Salud Carlos III, Madrid, Spain (J Clarimon, A Lleo, E Morenas-Rodriguez, P Pastor MD, M Diez-Fairen BSc, M Aguilar MD); Taub Institute for Alzheimer Disease and the Aging Brain and Department of Pathology and Cell Biology, Columbia University, New York, NY, USA (L Clark PhD, Prof L S Honig PhD, Prof K Marder MD); Department of Neurology and Alzheimer Center, Neuroscience Campus Amsterdam, VU University Medical Center, Amsterdam, Netherlands (A Lemstra PhD); Tanz Centre for Research in Neurodegenerative Diseases (Prof E Rogaeva PhD, Prof P St George-Hyslop MD) and Department of Medicine (Prof E Rogaeva, Prof P St George-Hyslop), University of Toronto, ON, Canada; Department of Clinical Neurosciences, Cambridge Institute for Medical Research, University of Cambridge, Cambridge, UK (Prof P St George-Hyslop); Clinical Memory Research Unit, Institution of Clinical Sciences Malmo, Lund University, Sweden (E Londos MD); Clinical Neurochemistry Laboratory, Institute of Neuroscience and Physiology, Sahlgrenska Academy, University of Gothenburg, Molndal, Sweden (Prof H Zetterberg); Human Genetics, School of Life Sciences, Queens Medical Centre, University of Nottingham, Nottingham, UK (I Barber PhD, A Braae PhD, K Brown PhD, Prof K Morgan PhD); Department of Basic and Clinical Neuroscience and Institute of Psychiatry, Psychology and Neuroscience, King’s College London, London, UK (C Troakes PhD, Prof S Al-Sarraj FRCPath); Parkinson’s Disease &amp; Movement Disorders Unit, Neurology Service, Hospital Clinic, IDIBAPS, CIBERNED, Department of Biomedicine (Y Compta) and Memory Unit, Department of Neurology, University Hospital Mutua de Terrassa (P Pastor, M Diez-Fairen, M Aguilar), University of Barcelona, Barcelona, Spain; Department of Pathology and Laboratory Medicine, Center for Neurodegenerative Disease Research, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA (Prof V Van Deerlin PhD, Prof J Q Trojanowski PhD); Banner Sun Health Research Institute, Sun City, AZ, USA (G E Serrano PhD, T G Beach); Inserm U1127, CNRS UMR7225, Sorbonne Universites, UPMC Univ Paris 06, UMR, Paris, France (S Lesage PhD); S1127, Institut du Cerveau et de la Moelle epiniere, Paris, France (S Lesage); Department of Neurosciences (Prof D Galasko MD), University of California, San Diego, La Jolla, CA, USA; Veterans Affairs San Diego Healthcare System, La Jolla, CA, USA (Prof D Galasko); Neurology Service, University of Coimbra Hospital, Coimbra, Portugal (I Santana PhD); Fundacio de Docencia I Recerca Mutua de Terrassa, Terrassa, Barcelona, Spain (P Pastor, M Diez-Fairen, M Aguilar); Molecular Neurology, Research Programs Unit (Prof P J Tienari PhD), Department of Pathology, Haartman Institute (L Myllykangas PhD), and Department of Neurosurgery (M Oinas PhD), University of Helsinki, Helsinki, Finland; Department of Neurology (Prof P J Tienari) and Department of Neuropathology and Neurosurgery (M Oinas), Helsinki University Hospital, Helsinki, Finland; HUSLAB, Helsinki, Finland (L Myllykangas); Department of Neurology, Mayo Clinic, Rochester, MN, USA (Prof B F Boeve MD, Prof R C Petersen PhD); MRC Centre for Neuropsychiatric Genetics and Genomics, School of Medicine, Cardiff University, Cardiff, UK (Prof V Escott-Price PhD); Knight Alzheimer’s Disease Research Center, Department of Neurology, Washington University School of Medicine, Saint Louis, MO, USA (Prof N J Cairns PhD, Prof J C Morris MD); Institute of Brain, Behaviour and Mental Health, Faculty of Medical and Human Sciences, University of Manchester, Manchester, UK (Prof S Pickering-Brown PhD, Prof D Mann PhD); Brain and Mind Centre, Sydney Medical School, University of Sydney, Sydney, NSW, Australia (Prof G M Halliday); and Merck &amp; Co, West Point, PA, USA (D J Stone PhD)</div><div id="FN1">Correspondence to: Dr Jose Bras, Department of Molecular Neuroscience, Institute of Neurology, London WC1N 1PJ, UK <a href="mailto:dev@null" data-email="ku.ca.lcu@sarb.j" class="oemail">ku.ca.lcu@sarb.j</a></div><div id="FN2"><sup>*</sup>Contributed equally</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140263512146960_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=29263008" target="pmc_ext" ref="reftype=publisher&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Lancet Neurol</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5805394/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplemental.</strong></div><div><a href="/pmc/articles/PMC5805394/bin/NIHMS938346-supplement-supplemental.pdf" data-ga-action="click_feat_suppl">NIHMS938346-supplement-supplemental.pdf</a><span style="color:gray"> (6.8M)</span></div><div class="small guid">GUID:&nbsp;836E9DED-4B17-467D-BB83-B534789D8B52</div></div></dd></dl></div><div id="idm140263506243552" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140263506243552title">Summary</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P2" class="p p-first-last">Dementia with Lewy bodies is the second most common form of dementia in elderly people but has been overshadowed in the research field, partly because of similarities between dementia with Lewy bodies, Parkinson’s disease, and Alzheimer’s disease. So far, to our knowledge, no large-scale genetic study of dementia with Lewy bodies has been done. To better understand the genetic basis of dementia with Lewy bodies, we have done a genome-wide association study with the aim of identifying genetic risk factors for this disorder.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P3" class="p p-first-last">In this two-stage genome-wide association study, we collected samples from white participants of European ancestry who had been diagnosed with dementia with Lewy bodies according to established clinical or pathological criteria. In the discovery stage (with the case cohort recruited from 22 centres in ten countries and the controls derived from two publicly available database of Genotypes and Phenotypes studies [phs000404.v1.p1 and phs000982.v1.p1] in the USA), we performed genotyping and exploited the recently established Haplotype Reference Consortium panel as the basis for imputation. Pathological samples were ascertained following autopsy in each individual brain bank, whereas clinical samples were collected after participant examination. There was no specific timeframe for collection of samples. We did association analyses in all participants with dementia with Lewy bodies, and also only in participants with pathological diagnosis. In the replication stage, we performed genotyping of significant and suggestive results from the discovery stage. Lastly, we did a meta-analysis of both stages under a fixed-effects model and used logistic regression to test for association in each stage.</p></div><div id="S3" class="sec"><h3 id="S3title">Findings</h3><p id="P4" class="p p-first-last">This study included 1743 patients with dementia with Lewy bodies (1324 with pathological diagnosis) and 4454 controls (1216 patients with dementia with Lewy bodies <em>vs</em> 3791 controls in the discovery stage; 527 <em>vs</em> 663 in the replication stage). Results confirm previously reported associations: <em>APOE</em> (rs429358; odds ratio [OR] 2·40, 95% CI 2·14–2·70; p=1·05 × 10<sup>−48</sup>), <em>SNCA</em> (rs7681440; OR 0·73, 0·66–0·81; p=6·39 × 10<sup>−10</sup>), and <em>GBA</em> (rs35749011; OR 2·55, 1·88–3·46; p=1·78 × 10<sup>−9</sup>). They also provide some evidence for a novel candidate locus, namely <em>CNTN1</em> (rs7314908; OR 1·51, 1·27–1·79; p=2·32 × 10<sup>−6</sup>); further replication will be important. Additionally, we estimate the heritable component of dementia with Lewy bodies to be about 36%.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Interpretation</h3><p id="P5" class="p p-first-last">Despite the small sample size for a genome-wide association study, and acknowledging the potential biases from ascertaining samples from multiple locations, we present the most comprehensive and well powered genetic study in dementia with Lewy bodies so far. These data show that common genetic variability has a role in the disease.</p></div></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P6" class="p p-first">Dementia with Lewy bodies is the second most common form of dementia after Alzheimer’s disease.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491542">1</a></sup> Despite this fact, very little attention has been devoted to understanding the pathogenesis of this disorder, particularly when compared with the other common neurodegenerative diseases such as Alzheimer’s disease and Parkinson’s disease.</p><p id="P7">So far, the only fully penetrant genetic variants that have been identified and replicated as a specific cause of dementia with Lewy bodies are <em>SNCA</em> point mutations and gene dosage. Three major factors might have contributed to this low number of causative mutations. First, dementia with Lewy bodies, often a disease of old age, is not commonly seen in multiplex kindreds, meaning that successful linkage studies have been rare.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491550">2</a></sup> Second, the accurate clinical diagnosis of dementia with Lewy bodies is complex, with a high rate of misdiagnosis.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491549">3</a></sup> Third, even the largest cohorts of dementia with Lewy body samples have been generally small, in many instances including as few as 100 patients.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491558">4</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491566">5</a></sup> However, the fact that dementia with Lewy bodies has a strong genetic component is currently indisputable. The ε4 allele of <em>APOE</em> is recognised to be a strong risk factor,<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491567">6</a>,<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491537">7</a></sup> as are heterozygous mutations and common polymorphisms in the glucocerebrosidase gene (<em>GBA</em>).<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491564">8</a></sup> Both of these results have stemmed from candidate gene association studies; <em>APOE</em> was known to be strongly associated with Alzheimer’s disease and <em>GBA</em> was known to be a strong risk factor for Parkinson’s disease and Lewy body disorders. In addition to these genetic associations with susceptibility, in 2016, our group provided evidence that dementia with Lewy bodies has a heritable component.<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664790304">9</a></sup></p><p id="P8">No overlap in common genetic risk has been shown to exist between Parkinson’s disease and Alzheimer’s disease,<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_720473529">10</a></sup> a fact that is not entirely surprising in view of the differences in phenotype. However, it is reasonable to hypothesise that the overlaps and differences in clinical and pathological presentation between dementia with Lewy bodies and both Parkinson’s disease and Alzheimer’s disease stem, at least in part, from aspects in their underlying genetic architecture and, consequently, disease pathobiology. Specific genes and loci associated with disease and the strength of association are factors that can be expected to modulate these phenotypic overlaps and differences. However, despite these encouraging findings, large-scale, unbiased genetic studies of dementia with Lewy bodies have not yet been done, which is probably due to the difficulty in identifying large, homogeneous cohorts of people with the disease.</p><p id="P9" class="p p-last">To address the need for more powerful and comprehensive genetic studies of dementia with Lewy bodies, we performed the first large-scale genome-wide association study in this disease.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Study design and participants</h3><p id="P10" class="p p-first-last">In this two-stage genome-wide association study, we examined data from white participants of European ancestry who had been diagnosed with dementia with Lewy bodies according to either clinical or pathological consensus criteria.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491524">11</a></sup> Most participants were diagnosed using pathological criteria and were included only when the likelihood of a diagnosis of dementia with Lewy bodies was “intermediate” or “high”.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491538">11</a></sup> Samples were collected at 22 different centres across ten countries in Europe, North America, and Australia. Pathological samples were ascertained following autopsy in each individual brain bank, whereas clinical samples were collected after participant examination. There was no specific timeframe for collection of samples. White control participants in the discovery stage are part of the “general research use” controls from the two studies publicly available at the database of Genotypes and Phenotypes (The Genetic Architecture of Smoking and Smoking Cessation [phs000404.v1.p1] and Genetic Analysis of Psoriasis and Psoriatic Arthritis [phs000982.v1.p1]). For the replication stage, white controls were from the Mayo Clinic Florida control database. Investigators at every site obtained written informed consent from patients and control individuals and approval from a local ethics committee.</p></div><div id="S8" class="sec"><h3 id="S8title">Discovery stage: genotyping, quality control, imputation, and statistical analysis</h3><p id="P11" class="p p-first">Participants with dementia with Lewy bodies were genotyped in either the Illumina Omni2.5M array or the Illumina OmniExpress genotyping array (Illumina, San Diego, CA, USA). Controls were genotyped in either the Illumina Omni2.5M array or the Illumina Omni1M array (Illumina, San Diego, CA, USA). Autosomal variants with GenTrain scores of more than 0·7 were included in the quality control stage. We removed single nucleotide polymorphisms (SNPs) with a call rate of less than 95%, a Hardy-Weinberg equilibrium p value in controls of less than 1 × 10<sup>−7</sup>, or a minor allele frequency of less than 0·01. Samples were removed if they had substantial non-European admixture, were duplicates or first-degree or second-degree relatives of other samples, had a genotype call rate of less than 98%, or had substantial cryptic relatedness scores (PI_HAT &gt;0·1).</p><p id="P12">We determined population outliers by principal components analysis, using SNPs passing the aforementioned quality-control filters. We used PLINK (version 1.9)<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491529">12</a></sup> to do linkage disequilibrium-based pruning. Genotypes for remaining SNPs were combined with 1000Genomes phase 3 genotypes for samples from the YRI, CEU, JPT, and CHB reference populations, and subjected to principal components analysis. Individuals lying farther than a quarter of the distance between CEU and JPT/CHB/YRI when plotted on the axes of the first two principal components were deemed to have substantial non-European admixture and were excluded (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 8</a>).</p><p id="P13">Because samples were genotyped in a variety of arrays, we selected only variants that intersected between all arrays to be included in the imputation stage. We performed imputation using the most recent reference panels provided by the Haplotype Reference Consortium (version 1.1, 2016). We used Eagle (version 2.3) to prephase haplotypes on the basis of genotype data.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491528">13</a>,<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491541">14</a></sup> We did the imputation using the Michigan Imputation Server.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491517">15</a></sup> Following imputation, we kept variants passing a standard imputation quality threshold (<em>R</em><sup>2</sup>≥0·3) for further analysis.</p><p id="P14">We used logistic regression, implemented in PLINK1.9,<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491551">12</a></sup> to test for association of hard-call variants with the binary case–control phenotype using sex as a covariate. We examined variants under an additive model (ie, effect of each minor allele) and estimated odds ratios (ORs) and 95% CIs. To control for population stratification, we used coordinates from the top six principal component dimensions as additional covariates in the logistic regression models. We used Q–Q plots and the genomic inflation factor (λ) to test for residual effects of population stratification not fully controlled for by the inclusion of the principal components analysis and cohort covariates in the regression model. Additionally, we have done a subanalysis in the discovery stage, including only participants with pathologically diagnosed dementia with Lewy bodies.</p><p id="P15">Moreover, to take into account the uncertainty of imputation, we have done the same association in PLINK1.9 using dosage data.</p><p id="P16" class="p p-last">We did gene-wise burden tests using all variants with an effect in protein sequence and a maximum minor allele frequency of 5%, using SKAT-O<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491532">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491533">17</a></sup> as implemented in EPACTS.<sup><a href="#R18" rid="R18" class=" bibr popnode">18</a></sup> We used the top six principal components and sex as covariates in the burden test.</p></div><div id="S9" class="sec"><h3 id="S9title">Replication stage: genotyping and power analysis</h3><p id="P17" class="p p-first">Replication was attempted for top variants showing a p value in the discovery stage of less than 5 × 10<sup>−6</sup>. We tested a total of 32 signals for replication using a Sequenom MassARRAY iPLEX SNP panel (Sequenom, San Diego, CA, USA; <a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 4</a>). We did power calculations for replication sample size selection using the R package RPower. We estimated a mean statistical power of 81% for the 32 signals on the basis of sample size, variant frequency, and effect size in the discovery stage, and used a replication p value threshold of 0·05. We tested associations in the replication stage using logistic regression models adjusted for age (age at onset for the patients with clinically diagnosed dementia with Lewy bodies, age at death for the patients with a high pathological likelihood of dementia with Lewy bodies, and age at recruitment to study for controls) and sex.</p><p id="P18" class="p p-last">We did a combined meta-analysis of stage 1 and 2 with GWAMA<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491540">19</a></sup> under a fixed-effects model, using estimates of the allelic OR and 95% CIs.</p></div><div id="S10" class="sec"><h3 id="S10title">Estimation of phenotypic variance</h3><p id="P19" class="p p-first-last">To estimate the phenotypic variance explained by the genotyped SNPs in this cohort, we used genetic restricted maximum likelihood analysis as implemented in the Genome-wide Complex Trait Analysis tool.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491534">20</a>,<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491545">21</a></sup> We used the first ten principal components as covariates and a disease prevalence of 0·1%.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491525">22</a></sup> We also estimated the partitioned heritability by chromosome, for which a separate genetic relationship matrix was generated for each chromosome. Each matrix was then run in a separate restricted maximum likelihood analysis. We applied linear regression to determine the relation between heritability and chromosome length.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Role of the funding source</h3><p id="P20" class="p p-first-last">The funders of the study had no role in study design, data collection, data analysis, data interpretation, or writing of the report. The corresponding author had full access to all the data in the study and had final responsibility for the decision to submit for publication.</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Results</h2><p id="P21" class="p p-first">This study included a total of 1743 patients with dementia with Lewy bodies and 4454 controls. The majority of patients with dementia with Lewy bodies were neuropathologically assessed (n=1324), providing a greater level of diagnostic detail. 987 participants with dementia with Lewy bodies were genotyped with the Illumina Omni2.5M array and 700 with the Illumina OmniExpress genotyping array. 1523 controls were genotyped with the Illumina Omni2.5M array and 2847 with the Illumina Omni1M array. Application of quality control filters to the dataset at the discovery stage yielded high-quality genotypes at 448 155 SNPs for 1216 participants with dementia with Lewy bodies and 3791 controls (<a href="/pmc/articles/PMC5805394/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>). A total of 52 participants with dementia with Lewy bodies were excluded for cryptic relatedness, 20 for genetic ancestry, and the remaining 399 for low call rates or poor genotyping. After imputation and quality control, genotypes for 8 397 716 variants were available for downstream analyses. After linkage disequilibrium-based pruning with PLINK (version 1.9)<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491546">12</a></sup> to quasi-independence (variance inflation factor=2), 130 715 SNPs remained in the dataset. The Q–Q plot and genomic inflation factor (λ=1·01) indicated good control of population stratification (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 9</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p19">Characteristics of the discovery cohort</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263507436544" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="top" align="left"></th><th rowspan="3" colspan="1" valign="top" align="right">N</th><th rowspan="3" colspan="1" valign="top" align="center">Neuropathological diagnosis</th><th rowspan="3" colspan="1" valign="top" align="left">Men:women ratio</th><th rowspan="3" colspan="1" valign="top" align="left">Age at onset (years)</th><th colspan="2" rowspan="1" valign="top" align="left">Passed quality controls</th></tr><tr><th colspan="2" rowspan="1" valign="bottom" align="left">
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="right">Total</th><th rowspan="1" colspan="1" valign="top" align="center">Neuropathological</th></tr></thead><tbody><tr><td colspan="7" rowspan="1" valign="top" align="left">Patients with dementia with Lewy bodies</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Australia</td><td rowspan="1" colspan="1" valign="top" align="right">79</td><td rowspan="1" colspan="1" valign="top" align="center">79 (100%)</td><td rowspan="1" colspan="1" valign="top" align="left">1·93</td><td rowspan="1" colspan="1" valign="top" align="left">65·2 (10·3)</td><td rowspan="1" colspan="1" valign="top" align="right">72 (91%)</td><td rowspan="1" colspan="1" valign="top" align="center">72 (91%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Canada</td><td rowspan="1" colspan="1" valign="top" align="right">29</td><td rowspan="1" colspan="1" valign="top" align="center">15 (52%)</td><td rowspan="1" colspan="1" valign="top" align="left">2·22</td><td rowspan="1" colspan="1" valign="top" align="left">67·9 (7·8)</td><td rowspan="1" colspan="1" valign="top" align="right">6 (21%)</td><td rowspan="1" colspan="1" valign="top" align="center">3 (10%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Finland</td><td rowspan="1" colspan="1" valign="top" align="right">34</td><td rowspan="1" colspan="1" valign="top" align="center">34 (100%)</td><td rowspan="1" colspan="1" valign="top" align="left">0·94</td><td rowspan="1" colspan="1" valign="top" align="left">94·3<sup>*</sup> (3·5)</td><td rowspan="1" colspan="1" valign="top" align="right">24 (71%)</td><td rowspan="1" colspan="1" valign="top" align="center">24 (71%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> France</td><td rowspan="1" colspan="1" valign="top" align="right">18</td><td rowspan="1" colspan="1" valign="top" align="center">18 (100%)</td><td rowspan="1" colspan="1" valign="top" align="left">3·5</td><td rowspan="1" colspan="1" valign="top" align="left">64·8 (10·3)</td><td rowspan="1" colspan="1" valign="top" align="right">16 (89%)</td><td rowspan="1" colspan="1" valign="top" align="center">16 (89%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Germany</td><td rowspan="1" colspan="1" valign="top" align="right">58</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">2·41</td><td rowspan="1" colspan="1" valign="top" align="left">67·8 (6·7)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> The Netherlands</td><td rowspan="1" colspan="1" valign="top" align="right">133</td><td rowspan="1" colspan="1" valign="top" align="center">133 (100%)</td><td rowspan="1" colspan="1" valign="top" align="left">1·71</td><td rowspan="1" colspan="1" valign="top" align="left">78·6<sup>*</sup> (7·4)</td><td rowspan="1" colspan="1" valign="top" align="right">132 (99%)</td><td rowspan="1" colspan="1" valign="top" align="center">132 (99%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Portugal</td><td rowspan="1" colspan="1" valign="top" align="right">13</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·63</td><td rowspan="1" colspan="1" valign="top" align="left">68·8 (8·2)</td><td rowspan="1" colspan="1" valign="top" align="right">11 (85%)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Spain</td><td rowspan="1" colspan="1" valign="top" align="right">133</td><td rowspan="1" colspan="1" valign="top" align="center">16 (12%)</td><td rowspan="1" colspan="1" valign="top" align="left">0·94</td><td rowspan="1" colspan="1" valign="top" align="left">73·3 (7·0)</td><td rowspan="1" colspan="1" valign="top" align="right">132 (99%)</td><td rowspan="1" colspan="1" valign="top" align="center">15 (11%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> UK</td><td rowspan="1" colspan="1" valign="top" align="right">404</td><td rowspan="1" colspan="1" valign="top" align="center">308 (76%)</td><td rowspan="1" colspan="1" valign="top" align="left">2·12</td><td rowspan="1" colspan="1" valign="top" align="left">69·7 (10·1)</td><td rowspan="1" colspan="1" valign="top" align="right">284 (70%)</td><td rowspan="1" colspan="1" valign="top" align="center">245 (61%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> USA</td><td rowspan="1" colspan="1" valign="top" align="right">786</td><td rowspan="1" colspan="1" valign="top" align="center">705 (90%)</td><td rowspan="1" colspan="1" valign="top" align="left">1·93</td><td rowspan="1" colspan="1" valign="top" align="left">71·2 (9·9)</td><td rowspan="1" colspan="1" valign="top" align="right">539 (69%)</td><td rowspan="1" colspan="1" valign="top" align="center">467 (59%)</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Total</td><td rowspan="1" colspan="1" valign="top" align="right">1687</td><td rowspan="1" colspan="1" valign="top" align="center">1308 (78%)</td><td rowspan="1" colspan="1" valign="top" align="left">1·83</td><td rowspan="1" colspan="1" valign="top" align="left">70·1 (9·5)</td><td rowspan="1" colspan="1" valign="top" align="right">1216 (72%)</td><td rowspan="1" colspan="1" valign="top" align="center">974 (58%)</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="7" rowspan="1" valign="top" align="left">Controls</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> USA (controls from PSA)</td><td rowspan="1" colspan="1" valign="top" align="right">2847</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·88</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="right">2832 (99%)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> USA (controls from SC)</td><td rowspan="1" colspan="1" valign="top" align="right">1523</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·78</td><td rowspan="1" colspan="1" valign="top" align="left">38 (5·7)</td><td rowspan="1" colspan="1" valign="top" align="right">959 (63%)</td><td rowspan="1" colspan="1" valign="top" align="center">··</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Total</td><td rowspan="1" colspan="1" valign="top" align="right">4370</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·83</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="right">3791 (87%)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr></tbody></table></div><div id="largeobj_idm140263507436544" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5805394/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p20" class="p p-first-last">Data are n (%) or mean (SD), unless stated otherwise. NA=not applicable. PSA=Genetic Analysis of Psoriasis and Psoriatic Arthritis database. SC=Genetic Architecture of Smoking and Smoking Cessation database.</p></div><div id="TFN2"><sup>*</sup>Represents age at death, which was available for these cohorts; these values were not used for calculation of the complete mean age at onset.</div></div></div><p id="P22">Five regions were associated with dementia with Lewy bodies risk at genome-wide significance (p&lt;5 × 10<sup>−8</sup>) in the discovery stage (<a href="/pmc/articles/PMC5805394/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>figure 1</span></a>; <a href="/pmc/articles/PMC5805394/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>). These regions included the previously described Alzheimer’s disease and Parkinson’s disease loci <em>APOE</em> (rs429358; OR 2·40, 95% CI 2·14–2·70; p=1·05 × 10<sup>−48</sup>), <em>SNCA</em> (rs7681440; OR 0·73, 0·66–0·81; p=6·39 × 10<sup>−10</sup>), and <em>GBA</em> (rs35749011; OR 2·55, 1·88–3·46; p=1·78 × 10<sup>−9</sup>). Additionally, loci overlapping <em>BCL7C/STX1B</em> (rs897984; OR 0·74, 0·67–0·82; p=3·30 × 10<sup>−9</sup>) and <em>GABRB3</em> (rs1426210; OR 1·34, 1·21–1·48; p=2·62 × 10<sup>−8</sup>) were also genome-wide significant. A subanalysis including only participants with pathologically diagnosed dementia with Lewy bodies revealed that all but <em>GABRB3</em> maintained their genome-wide significance in that smaller dataset (<a href="/pmc/articles/PMC5805394/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>; <a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 11</a>). Furthermore, when undertaking the same associations in PLINK1.9 to take into account the uncertainty of imputation, results were identical to the best-guess calls (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 8</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5805394/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263489262384" class="figure"><a href="/pmc/articles/PMC5805394/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5805394_nihms938346f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms938346f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5805394/bin/nihms938346f1.jpg"></a></div><div id="largeobj_idm140263489262384" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5805394/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5805394/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Manhattan plot showing genome-wide p values of association</strong><p id="__p15">The p values were obtained by logistic-regression analysis using the first six principal components and sex as covariates. The y axis shows −log<sub>10</sub> p values of 8 397 716 single nucleotide polymorphisms, and the x axis shows their chromosomal positions. The horizontal red dotted line represents the threshold of p=5 × 10<sup>−8</sup> for Bonferroni significance and the green dotted line represents the threshold of p=5 × 10<sup>−6</sup> for selecting single nucleotide polymorphisms for replication.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p22">Top signals of association at each locus that passed genome-wide threshold for significance and their replication and meta-analysis p values</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263508982528" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="left"></th><th colspan="5" rowspan="1" valign="bottom" align="left">General characteristics</th><th colspan="5" rowspan="1" valign="bottom" align="left">Discovery</th><th colspan="5" rowspan="1" valign="bottom" align="left">Replication</th><th colspan="2" rowspan="1" valign="bottom" align="left">Meta-analysis</th></tr><tr><th colspan="5" rowspan="1" valign="bottom" align="center">
<hr></th><th colspan="5" rowspan="1" valign="bottom" align="center">
<hr></th><th colspan="5" rowspan="1" valign="bottom" align="center">
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="right">Chromosome</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position</th><th rowspan="1" colspan="1" valign="bottom" align="left">Variant</th><th rowspan="1" colspan="1" valign="bottom" align="left"><em>R</em><sup>2</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Eur_AF<sup>*</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">MA</th><th rowspan="1" colspan="1" valign="bottom" align="left">MAF_A</th><th rowspan="1" colspan="1" valign="bottom" align="left">MAF_U</th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Power<sup>†</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">MAF_A</th><th rowspan="1" colspan="1" valign="bottom" align="left">MAF_U</th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value</th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value</th></tr></thead><tbody><tr><td colspan="18" rowspan="1" valign="top" align="left"><strong>Global cohort</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>APOE</em></td><td rowspan="1" colspan="1" valign="top" align="right">19</td><td rowspan="1" colspan="1" valign="top" align="center">45 411 941</td><td rowspan="1" colspan="1" valign="top" align="left">rs429358</td><td rowspan="1" colspan="1" valign="top" align="left">0·949</td><td rowspan="1" colspan="1" valign="top" align="left">0·149</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·283</td><td rowspan="1" colspan="1" valign="top" align="left">0·140</td><td rowspan="1" colspan="1" valign="top" align="left">2·40 (2·14–2·7)</td><td rowspan="1" colspan="1" valign="top" align="left">1·05 × 10<sup>−48</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">0·282</td><td rowspan="1" colspan="1" valign="top" align="left">0·148</td><td rowspan="1" colspan="1" valign="top" align="left">2·74 (2·15–3·49)</td><td rowspan="1" colspan="1" valign="top" align="left">4·00 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2·46 (2·22–2·74)</td><td rowspan="1" colspan="1" valign="top" align="left">3·31 × 10<sup>−64</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>SNCA</em></td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="center">90 756 550</td><td rowspan="1" colspan="1" valign="top" align="left">rs7681440<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·996</td><td rowspan="1" colspan="1" valign="top" align="left">0·52</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·411</td><td rowspan="1" colspan="1" valign="top" align="left">0·483</td><td rowspan="1" colspan="1" valign="top" align="left">0·73 (0·66–0·81)</td><td rowspan="1" colspan="1" valign="top" align="left">6·39 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·95</td><td rowspan="1" colspan="1" valign="top" align="left">0·38</td><td rowspan="1" colspan="1" valign="top" align="left">0·47</td><td rowspan="1" colspan="1" valign="top" align="left">0·68 (0·56–0·82)</td><td rowspan="1" colspan="1" valign="top" align="left">6·00 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·73 (0·67–0·79)</td><td rowspan="1" colspan="1" valign="top" align="left">9·22 × 10<sup>−13</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>GBA</em></td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="center">155 135 036</td><td rowspan="1" colspan="1" valign="top" align="left">rs35749011</td><td rowspan="1" colspan="1" valign="top" align="left">0·957</td><td rowspan="1" colspan="1" valign="top" align="left">0·014</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0·033</td><td rowspan="1" colspan="1" valign="top" align="left">0·014</td><td rowspan="1" colspan="1" valign="top" align="left">2·55 (1·88–3·46)</td><td rowspan="1" colspan="1" valign="top" align="left">1·78 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·83</td><td rowspan="1" colspan="1" valign="top" align="left">0·044</td><td rowspan="1" colspan="1" valign="top" align="left">0·022</td><td rowspan="1" colspan="1" valign="top" align="left">1·81 (1·05–3·11)</td><td rowspan="1" colspan="1" valign="top" align="left">0·033</td><td rowspan="1" colspan="1" valign="top" align="left">2·27 (1·75–2·95)</td><td rowspan="1" colspan="1" valign="top" align="left">6·57 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>BCL7C/ STX1B</em></td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="center">30 886 643</td><td rowspan="1" colspan="1" valign="top" align="left">rs897984<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·984</td><td rowspan="1" colspan="1" valign="top" align="left">0·609</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">0·334</td><td rowspan="1" colspan="1" valign="top" align="left">0·405</td><td rowspan="1" colspan="1" valign="top" align="left">0·74 (0·67–0·82)</td><td rowspan="1" colspan="1" valign="top" align="left">3·30 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·96</td><td rowspan="1" colspan="1" valign="top" align="left">0·368</td><td rowspan="1" colspan="1" valign="top" align="left">0·388</td><td rowspan="1" colspan="1" valign="top" align="left">0·98 (0·81–1·19)</td><td rowspan="1" colspan="1" valign="top" align="left">0·83</td><td rowspan="1" colspan="1" valign="top" align="left">0·77 (0·71–0·85)</td><td rowspan="1" colspan="1" valign="top" align="left">1·19 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>GABRB3</em></td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="center">26 840 998</td><td rowspan="1" colspan="1" valign="top" align="left">rs1426210</td><td rowspan="1" colspan="1" valign="top" align="left">0·982</td><td rowspan="1" colspan="1" valign="top" align="left">0·315</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0·348</td><td rowspan="1" colspan="1" valign="top" align="left">0·293</td><td rowspan="1" colspan="1" valign="top" align="left">1·34 (1·21–1·48)</td><td rowspan="1" colspan="1" valign="top" align="left">2·62 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·9</td><td rowspan="1" colspan="1" valign="top" align="left">0·281</td><td rowspan="1" colspan="1" valign="top" align="left">0·307</td><td rowspan="1" colspan="1" valign="top" align="left">0·84 (0·68–1·04)</td><td rowspan="1" colspan="1" valign="top" align="left">0·1</td><td rowspan="1" colspan="1" valign="top" align="left">1·22 (1·11–1·33)</td><td rowspan="1" colspan="1" valign="top" align="left">2·05 × 10<sup>−5</sup></td></tr><tr><td colspan="18" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="18" rowspan="1" valign="top" align="left"><strong>Neuropathologically diagnosed cases</strong></td></tr><tr><td colspan="18" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>APOE</em></td><td rowspan="1" colspan="1" valign="top" align="right">19</td><td rowspan="1" colspan="1" valign="top" align="center">45 411 941</td><td rowspan="1" colspan="1" valign="top" align="left">rs429358</td><td rowspan="1" colspan="1" valign="top" align="left">0·949</td><td rowspan="1" colspan="1" valign="top" align="left">0·149</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·292</td><td rowspan="1" colspan="1" valign="top" align="left">0·140</td><td rowspan="1" colspan="1" valign="top" align="left">2·52 (2·23–2·85)</td><td rowspan="1" colspan="1" valign="top" align="left">2·77 × 10<sup>−49</sup></td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>SNCA</em></td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="center">90 756 550</td><td rowspan="1" colspan="1" valign="top" align="left">rs7681440<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·996</td><td rowspan="1" colspan="1" valign="top" align="left">0·52</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·409</td><td rowspan="1" colspan="1" valign="top" align="left">0·483</td><td rowspan="1" colspan="1" valign="top" align="left">0·73 (0·66–0·81)</td><td rowspan="1" colspan="1" valign="top" align="left">2·82 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>GBA</em></td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="center">155 135 036</td><td rowspan="1" colspan="1" valign="top" align="left">rs35749011</td><td rowspan="1" colspan="1" valign="top" align="left">0·957</td><td rowspan="1" colspan="1" valign="top" align="left">0·014</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0·037</td><td rowspan="1" colspan="1" valign="top" align="left">0·014</td><td rowspan="1" colspan="1" valign="top" align="left">2·87 (2·10–3·90)</td><td rowspan="1" colspan="1" valign="top" align="left">2·67 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>BCL7C/STX1B</em></td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="center">30 886 643</td><td rowspan="1" colspan="1" valign="top" align="left">rs897984<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·984</td><td rowspan="1" colspan="1" valign="top" align="left">0·609</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">0·332</td><td rowspan="1" colspan="1" valign="top" align="left">0·405</td><td rowspan="1" colspan="1" valign="top" align="left">0·73 (0·65–0·81)</td><td rowspan="1" colspan="1" valign="top" align="left">4·32 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>GABRB3</em></td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="center">26 840 998</td><td rowspan="1" colspan="1" valign="top" align="left">rs1426210</td><td rowspan="1" colspan="1" valign="top" align="left">0·982</td><td rowspan="1" colspan="1" valign="top" align="left">0·315</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0·350</td><td rowspan="1" colspan="1" valign="top" align="left">0·293</td><td rowspan="1" colspan="1" valign="top" align="left">1·34 (1·20–1·45)</td><td rowspan="1" colspan="1" valign="top" align="left">1·21 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td><td rowspan="1" colspan="1" valign="top" align="left">··</td></tr></tbody></table></div><div id="largeobj_idm140263508982528" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5805394/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="__p23" class="p p-first-last"><em>R</em><sup>2</sup>=imputation <em>R</em><sup>2</sup> of each specific variant from Haplotype Reference Consortium. OR=odds ratio. MA=minor allele. MAF_A=minor allele frequency in cases. MAF_U=minor allele frequency in controls.</p></div><div id="TFN4"><sup>*</sup>Eur_AF is the alternative allele frequency derived from the European population of the Genome Aggregation Database (gnomAD).<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491526">23</a></sup></div><div id="TFN5"><sup>†</sup>Power refers to the calculated statistical power to replicate the discovery signal, taking into account the replication sample size, effect, and frequency in discovery and an association threshold of p&lt;0·05.</div><div id="TFN6"><sup>‡</sup>Represents variants for which the gnomAD allele frequency corresponds to the alternative allele and not the effect allele.</div></div></div><p id="P23">A total of 527 participants with dementia with Lewy bodies and 663 controls from the Mayo Clinic were included in the replication stage (<a href="/pmc/articles/PMC5805394/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>table 3</span></a>). The replication stage of the genome-wide association study design provided independent replication (p&lt;0·05) for three of the loci (<em>APOE</em>, <em>SNCA</em>, and <em>GBA</em>), all of which were also genome-wide significant in the combined analysis of both stages (<a href="/pmc/articles/PMC5805394/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>; <a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 4</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p27">Characteristics of the replication cohort</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263489648672" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center">Neuropathological diagnosis</th><th rowspan="1" colspan="1" valign="bottom" align="left">Men:women ratio</th><th rowspan="1" colspan="1" valign="bottom" align="left">Age at onset (years)<sup>*</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">USA: patients with dementia with Lewy bodies</td><td rowspan="1" colspan="1" valign="top" align="center">350/527 (66%)</td><td rowspan="1" colspan="1" valign="top" align="left">2·01</td><td rowspan="1" colspan="1" valign="top" align="left">76·3 (8·2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">USA: controls</td><td rowspan="1" colspan="1" valign="top" align="center">0/663</td><td rowspan="1" colspan="1" valign="top" align="left">0·75</td><td rowspan="1" colspan="1" valign="top" align="left">67·8 (10·0)</td></tr></tbody></table></div><div id="largeobj_idm140263489648672" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5805394/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN7"><p id="__p28" class="p p-first-last">Data are n (%) or mean (SD), unless stated otherwise.</p></div><div id="TFN8"><sup>*</sup>Denotes age at examination for controls; for patients with dementia with Lewy bodies, the age reflects age at onset for those diagnosed clinically and age at death for those pathologically diagnosed.</div></div></div><p id="P24">In the discovery stage, suggestive evidence of an association (p&lt;5 × 10<sup>−6</sup>) with dementia with Lewy bodies was also seen for two loci: <em>SOX17</em> and <em>CNTN1</em>. The association at <em>SOX17</em> did not replicate (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 4</a>). For <em>CNTN1</em>, the association with dementia with Lewy bodies (rs7314908; 1·51, 95% CI 1·27–1·79; p=2·32 × 10<sup>−6</sup>) improved slightly when performing the subanalysis on the participants with pathologically confirmed dementia with Lewy bodies (rs7314908; OR 1·58, 1·32–1·88; p=4·32 × 10<sup>−7</sup>), and this candidate locus showed evidence of replication with very similar effect size to that in the discovery stage (rs79329964; OR 1·54, 1·32–1·79; p=0·03; rs79329964 was used in replication as a proxy for rs7314908).</p><p id="P25">A systematic assessment of genetic loci previously associated with Alzheimer’s disease or Parkinson’s disease showed no evidence of other genome-wide significant associations in this dementia with Lewy bodies cohort (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 5</a>). These loci include the <em>TREM2</em> locus, where the p.Arg47His variant has been shown to have a strong effect in Alzheimer’s disease.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491554">24</a></sup> In our cohort this variant did not show genome-wide significant levels of association (OR 3·46, 95% CI 1·54–7·77; p=0·002), despite the over-representation in people with dementia with Lewy bodies compared with controls. Similarly, <em>MAPT</em>, which is strongly associated with Parkinson’s disease and has been previously linked to dementia with Lewy bodies,<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664790303">25</a></sup> shows no strong evidence of association in this study (rs17649553; OR 0·86, 0·76–0·96; p=0·0126).</p><p id="P26">To examine whether the association with <em>SNCA</em> is independent of that seen in Parkinson’s disease, we conditioned our analysis on the top Parkinson’s disease variant (rs356182), which showed only a negligible effect on the DLB association (conditioned OR 0·70, 95% CI 0·63–0·78; p=2·89 × 10<sup>−10</sup>; <a href="/pmc/articles/PMC5805394/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2</span></a>). To gain insight into potential regulatory effects of this distinct <em>SNCA</em> signal, we used expression quantitative trait loci (eQTL) data from the Genotype-Tissue Expression (GTEx) Project Consortium and the Harvard Brain Bank Resource Center to determine whether rs7681440 and rs7681154 (a variant shown to have an independent association for Parkinson’s disease that is in strong linkage disequilibrium [<em>R<sup>2</sup></em>=0·91] with the rs7681440 <em>SNCA</em> variant) affect gene expression as eQTLs. In the GTEx data, the most associated SNP in dementia with Lewy bodies is a strong eQTL in the cerebellum for <em>RP11-67M1.1</em>, a known antisense gene located at the 5′ end of <em>SNCA</em>, with the alternative allele showing a reduction in expression of <em>RP11-67M1.1</em> (<a href="/pmc/articles/PMC5805394/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>figure 3</span></a>). Additionally, rs7681154 was associated with <em>SNCA</em> expression in the cerebellum using the Harvard Brain Bank Resource Center results (p=2·87 × 10<sup>−11</sup>; <a href="/pmc/articles/PMC5805394/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>figure 3</span></a>), with the alternative allele associated with increased <em>SNCA</em> expression.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5805394/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263490156576" class="figure"><a href="/pmc/articles/PMC5805394/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5805394_nihms938346f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms938346f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5805394/bin/nihms938346f2.jpg"></a></div><div id="largeobj_idm140263490156576" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5805394/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5805394/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Regional association plot for the <em>SNCA</em> locus</strong><p id="__p16">Purple represents variant rs1372517, at chromosome 4, position 90755909, which is the most associated SNP at the locus also present in the 1000Genomes dataset. The variant rs1372517 is in complete linkage disequilibrium with rs7681440. Colours represent linkage disequilibrium derived from 1000Genomes between each variant and the most associated SNP. SNP=single nucleotide polymorphism. R<sup>2</sup> represents the level of pairwise linkage disequilibrium between the top variant and each other variant plotted, using data from the 1000 Genomes project.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC5805394/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263491746768" class="figure"><a href="/pmc/articles/PMC5805394/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5805394_nihms938346f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms938346f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5805394/bin/nihms938346f3.jpg"></a></div><div id="largeobj_idm140263491746768" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5805394/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC5805394/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Associations between genotypes and gene expression in the cerebellum of post-mortem controls</strong><p id="__p17">(A) Association between rs7681440 genotypes and <em>RP11-67M1.1</em> expression in 103 disease-free post-mortem cerebellum samples (p=2·00 × 10<sup>−7</sup>) from the Genotype-Tissue Expression (GTEx) Project Consortium. Carriers of the GG genotype (alternative allele) show the lowest levels of expression of the gene. Details on methods are on GTEx website. (B) Association between rs7681154 and <em>SNCA</em> expression (p=2·87 × 10<sup>−11</sup>) in 468 disease-free cerebellum samples from postmortem individuals from the Harvard Brain Bank Resource Center.<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491555">26</a></sup> Individuals with the alternative allele C had increased <em>SNCA</em> expression in the cerebellum, on average, compared with individuals with the reference allele G. Details on the subjects, experiments, and analytical methods of the expression quantitative trait loci study of the Harvard Brain Bank Resource Center samples are described by Zhang and colleagues<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491530">26</a></sup> and on the Harvard Brain Bank website. Boxplots for both panels show medians, IQRs, and individual data points.</p></div></div></div><p id="P27">We assessed linkage disequilibrium across the <em>LRRK2</em> locus region and that analysis revealed that rs79329964 is in equilibrium with both p.Gly2019Ser (<em>R</em><sup>2</sup>=0·000043) and with the Parkinson’s disease hit at this locus, rs76904798 (<em>R</em><sup>2</sup>=0·003), suggesting rs79329964 to be an independent association from the Parkinson’s disease risk. Although samples were not screened for p.Gly2019Ser directly, the variant was well imputed (<em>R</em><sup>2</sup>=0·94). The exclusion of all samples that carried the p.Gly2019Ser variant showed no significant effect on the association at the <em>CNTN1</em> locus. Notably, the p.Gly2019Ser variant showed a higher minor allele frequency in participants with dementia with Lewy bodies (0·0021) than in controls (0·0003).</p><p id="P28">Gene-based burden analysis of all low frequency and rare variants (minor allele frequency &lt;0·05) changing the aminoacid sequence, showed a single genome-wide significant result comprised of six variants at <em>GBA</em> (p.Asn409Ser, p.Thr408Met, p.Glu365Lys, p.Arg301His, p.Ile20Val, and p.Lys13Arg; p=1·29 × 10<sup>−13</sup>). No other gene showed evidence of strong association with disease or overlapped single variant analysis results (<a href="/pmc/articles/PMC5805394/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>table 4</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p30">Top gene burden results</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263492104864" class="xtable"><table frame="box" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center">Chromosome number</th><th rowspan="1" colspan="1" valign="top" align="left">Begin<sup>*</sup></th><th rowspan="1" colspan="1" valign="top" align="left">End<sup>*</sup></th><th rowspan="1" colspan="1" valign="top" align="left">Number of samples with non-missing genotypes</th><th rowspan="1" colspan="1" valign="top" align="left">Fraction with rare allele<sup>†</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Number of all variants defining the gene group</th><th rowspan="1" colspan="1" valign="top" align="center">Number of variants passing the frequency and call-rate thresholds</th><th rowspan="1" colspan="1" valign="top" align="left">Number of singletons among variants passing the frequency and call-rate thresholds</th><th rowspan="1" colspan="1" valign="top" align="left">p value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>GBA</em></td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="left">155 204 797</td><td rowspan="1" colspan="1" valign="top" align="left">155 210 498</td><td rowspan="1" colspan="1" valign="top" align="left">5 016</td><td rowspan="1" colspan="1" valign="top" align="left">0·05622</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">1·29 × 10<sup>−13</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>CBX6</em></td><td rowspan="1" colspan="1" valign="top" align="center">22</td><td rowspan="1" colspan="1" valign="top" align="left">39 262 224</td><td rowspan="1" colspan="1" valign="top" align="left">39 267 761</td><td rowspan="1" colspan="1" valign="top" align="left">5 016</td><td rowspan="1" colspan="1" valign="top" align="left">0·010965</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">1·66 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>ST14</em></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">130 058 428</td><td rowspan="1" colspan="1" valign="top" align="left">130 079 477</td><td rowspan="1" colspan="1" valign="top" align="left">5 016</td><td rowspan="1" colspan="1" valign="top" align="left">0·076754</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">4·29 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NPS</em></td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="left">129 347 767</td><td rowspan="1" colspan="1" valign="top" align="left">129 350 889</td><td rowspan="1" colspan="1" valign="top" align="left">5 016</td><td rowspan="1" colspan="1" valign="top" align="left">0·076555</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">6·74 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>RBM47</em></td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="left">40 428 010</td><td rowspan="1" colspan="1" valign="top" align="left">40 434 855</td><td rowspan="1" colspan="1" valign="top" align="left">5 016</td><td rowspan="1" colspan="1" valign="top" align="left">0·0099681</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">0·0001</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>TPH1</em></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">18 047 141</td><td rowspan="1" colspan="1" valign="top" align="left">18 057 637</td><td rowspan="1" colspan="1" valign="top" align="left">5 016</td><td rowspan="1" colspan="1" valign="top" align="left">0·0091707</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·0002</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-C</em></td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">31 237 124</td><td rowspan="1" colspan="1" valign="top" align="left">31 239 829</td><td rowspan="1" colspan="1" valign="top" align="left">5 016</td><td rowspan="1" colspan="1" valign="top" align="left">0·1262</td><td rowspan="1" colspan="1" valign="top" align="center">32</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">0·0003</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>FOSB</em></td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="left">45 971 941</td><td rowspan="1" colspan="1" valign="top" align="left">45 976 122</td><td rowspan="1" colspan="1" valign="top" align="left">5 016</td><td rowspan="1" colspan="1" valign="top" align="left">0·00079745</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">0·0004</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>DPH2</em></td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="left">44 435 905</td><td rowspan="1" colspan="1" valign="top" align="left">44 438 171</td><td rowspan="1" colspan="1" valign="top" align="left">5 016</td><td rowspan="1" colspan="1" valign="top" align="left">0·004386</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">0·0004</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>RAMP1</em></td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="left">238 785 923</td><td rowspan="1" colspan="1" valign="top" align="left">238 820 379</td><td rowspan="1" colspan="1" valign="top" align="left">5 016</td><td rowspan="1" colspan="1" valign="top" align="left">0·00079745</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">0·0005</td></tr></tbody></table></div><div id="largeobj_idm140263492104864" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5805394/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN9"><sup>*</sup>Begin and end refer to the locations defining each gene or marker group tested.</div><div id="TFN10"><sup>†</sup>Fraction of individuals carrying rare variants below the allele frequency threshold (0·05).</div></div></div><p id="P29">Using the first ten principal components as covariates and a disease prevalence of 0·1%, estimation of the phenotypic variance attributed to genetic variants showed a heritable component of dementia with Lewy bodies of 36% (SD 0·03). As expected for a common complex disease, we found a strong correlation between chromosome length and heritability (p=6·88 × 10<sup>−5</sup>; <a href="/pmc/articles/PMC5805394/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>figure 4</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC5805394/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263491212048" class="figure"><a href="/pmc/articles/PMC5805394/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5805394_nihms938346f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms938346f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5805394/bin/nihms938346f4.jpg"></a></div><div id="largeobj_idm140263491212048" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5805394/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC5805394/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Dementia with Lewy bodies heritability by chromosome</strong><p id="__p18">Heritability (y axis) per chromosome is plotted against chromosome length (x axis). The red line represents heritability regressed on chromosome length and the shaded area represents the 95% CI of the regression model.</p></div></div></div><p id="P30" class="p p-last">The heritability for dementia with Lewy bodies at chromosome 19 is much higher than what would be expected considering the chromosome’s size and probably reflects the role of <em>APOE</em>. Notably, chromosomes 5, 6, 7, and 13 all have higher heritability for dementia with Lewy bodies than expected, although none of them has variants with genome-wide significant results.</p></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Discussion</h2><p id="P31" class="p p-first">This is the first comprehensive, unbiased study of common and intermediate frequency genetic variability in dementia with Lewy bodies. We identified five genome-wide significant associations in the discovery stage (<em>APOE</em>, <em>BCL7C</em>/<em>STX1B</em>, <em>SNCA</em>, <em>GBA</em>, and <em>GABRB3</em>), with the associations regarding <em>APOE</em>, <em>SNCA</em>, and <em>GBA</em> being confirmed in the replication stage and in the combined analysis of both stages.</p><p id="P32">The most significant association signal is seen at the <em>APOE</em> locus (<em>APOE</em> ε4), which has been previously shown to be highly associated with dementia with Lewy bodies.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491553">6</a>,<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491515">7</a></sup> As described, <em>APOE</em> ε4 is the major genetic risk locus for Alzheimer’s disease and has been implicated in cognitive impairment within Parkinson’s disease, although not with the risk of Parkinson’s disease itself. The locus has also been reported to affect the levels of both β-amyloid and Lewy body pathology in brains of patients.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491557">27</a></sup> In a small Finnish dataset,<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491527">28</a></sup> the ε4 allele association with dementia with Lewy bodies was largely driven by the subgroup with concomitant Alzheimer’s disease pathology.</p><p id="P33">The second strongest association is seen at the <em>SNCA</em> locus. Results from our conditioned analysis confirmed the different association profile between dementia with Lewy bodies and Parkinson’s disease that we had previously reported.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491539">7</a></sup>
<em>SNCA</em> is the most significant common genetic risk factor for Parkinson’s disease, with rs356182 having a meta-analysis p value of 1·85 × 10<sup>−82</sup> (OR 1·34, 1·30–1·38) in PDGene. This variant is located 3’ to the gene,<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491535">29</a></sup> whereas in dementia with Lewy bodies, no association was found in that region (<a href="/pmc/articles/PMC5805394/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2</span></a>). The most associated dementia with Lewy bodies SNP for the <em>SNCA</em> locus (rs7681440) has a Parkinson’s disease meta-analysis p value of more than 0·05 in PDGene. When doing a conditional analysis on the top Parkinson’s disease SNP (rs356182), Nalls and colleagues<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491547">29</a></sup> reported an independent association at the 5′ region of the gene (rs7681154), and this variant is in strong linkage disequilibrium (<em>R</em><sup>2</sup>=0·91) with the rs7681440 <em>SNCA</em> variant identified in our study. It is tempting to speculate that these differences might reflect pathobiological differences between the two diseases, perhaps mediated by differential regulation of gene expression. The results in the GTEx data, showing that the most associated SNP in dementia with Lewy bodies is a strong eQTL in the cerebellum for <em>RP11-67M1.1</em>, are compatible with a model in which rs7681440 genotypes affect the expression levels of <em>SNCA</em> indirectly through the action of <em>RP11-67M1.1</em>. More specifically, the alternative allele associates with a decreased expression of <em>RP11-67M1.1</em> and consequently reduced repression of <em>SNCA</em> transcription (increased <em>SNCA</em> expression), which is in accordance with an increased frequency of the alternative allele in participants with dementia with Lewy bodies when compared with controls. Additionally, the relationship between rs7681154 and <em>SNCA</em> expression is supported by the high expression of <em>SNCA</em> in the brain and the association of rs7681440 with increased <em>SNCA</em> expression in whole blood (p=2·13 × 10<sup>−38</sup>).<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491559">30</a>,<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491523">31</a></sup> However, further investigation of the identified significant eQTLs is needed because the effect was seen for only one brain region. This localised effect could plausibly result from low overall expression of <em>RP11-67M1.1</em> and higher RNA quality in the cerebellum than in other assayed brain regions in these datasets. Notably, both eQTLs’ effects fit with a model of increased <em>SNCA</em> expression in participants with dementia with Lewy bodies compared with controls.</p><p id="P34">The most significant marker at the <em>GBA</em> locus (rs35682329) is located 85 781 base pairs downstream of the gene and is in high linkage disequilibrium (D′=0·9; <em>R</em><sup>2</sup>=0·8) with p.Glu365Lys (also reported in the scientific literature as E365K, E326K, and rs2230288), which has been suggested as a risk factor for dementia with Lewy bodies.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491520">8</a></sup> The top associated variant for Parkinson’s disease at this locus is the rs71628662 (PDGene meta-analysis OR 0·52 [95% CI 0·46–0·58; p=6·86 × 10<sup>−28</sup>]). This variant is also in high linkage disequilibrium with the top SNP identified here (D′=0·9 and <em>R</em><sup>2</sup>=0·8). In this study, we show similar effect sizes for <em>APOE</em> (OR 2·40) and <em>GBA</em> (OR 2·55) in dementia with Lewy bodies. Gene burden-based analysis showed <em>GBA</em> as the only genome-wide significant association with dementia with Lewy bodies risk. The inexistence of other associations should be interpreted with some caution. Because we were not ascertaining the complete spectrum of genetic variability, other genes could have had a significant burden of genetic variants that were simply not captured in our study design, despite our use of the most recent imputation panel.</p><p id="P35">Although in our meta-analysis we saw a genome-wide significant association with dementia with Lewy bodies at the <em>BCL7C/STX1B</em> locus, this association was mostly driven by the discovery-stage data (replication-stage results were OR 0·98; p=0·83) and further replication is needed. That being acknowledged, an association at the <em>BCL7C/ STX1B</em> locus has been previously reported for Parkinson’s disease.<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491563">29</a>,<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491562">32</a></sup> The top Parkinson’s disease-associated variants at this locus were rs14235 (synonymous; located at <em>BCKDK</em>) and rs4889603 (intronic; located at <em>SETD1A</em>). The top SNP identified in dementia with Lewy bodies at this locus (rs897984) shows the same direction of association seen in Parkinson’s disease (OR 0·93, 95% CI 0·90–0·96), a Parkinson’s disease meta-analysis p value of 1·34 × 10<sup>−5</sup> (data from PDgene), and strong linkage disequilibrium with both Parkinson’s disease hits (<em>R</em><sup>2</sup>=0·28–0·32; correlation p values&lt;0·0001). This is a gene-rich region of the genome (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 9</a>), making accurate nomination of the gene driving the association difficult. Mining data from the GTEx project showed that rs897984 is not an eQTL for any gene in the locus. Nonetheless, in both Parkinson’s disease studies, the nominated gene at the locus was <em>STX1B</em>, probably due to its function as a synaptic receptor.<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491531">33</a></sup> Additionally, <em>STX1B</em> has a distinctive pattern of expression across tissues, presenting the highest expression in the brain. In this tissue, when compared with the closest genes in the locus (<em>HSD3B7, BCL7C, ZNF668, MIR4519, CTF1, FBXL19, ORAI3, SETD1A, STX4</em>), <em>STX1B</em> also shows the highest levels of expression (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 10</a>). In 2014, mutations in <em>STX1B</em> were shown to cause fever-associated epileptic syndromes<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491565">34</a></sup> and myoclonic astatic epilepsy.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491544">35</a></sup></p><p id="P36">Although not quite genome-wide significant in the discovery stage, the association between <em>CNTN1</em> and dementia with Lewy bodies risk replicated with a very similar association OR as the discovery stage. Interestingly, the locus has been previously associated with Parkinson’s disease in a genome-wide study of identical-by-descent segments in an Ashkenazi cohort,<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491518">36</a></sup> and with cerebral amyloid deposition, assessed with PET imaging in <em>APOE</em> ε4 non-carriers.<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491536">37</a></sup> This locus also did not reach genome-wide significance with clinicopathological Alzheimer’s disease dementia (p=5·21 × 10<sup>−6</sup>).<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491543">38</a></sup> The contact in 1 protein, encoded by <em>CNTN1</em>, is a glycosylphosphatidylinositol-anchored neuronal membrane protein that functions as a cell-adhesion molecule with important roles in axonal function.<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491552">39</a>,<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664790305">40</a></sup> Mutations in <em>CNTN1</em> were found to cause a familial form of lethal congenital myopathy.<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491548">41</a></sup> Contactin 1 drives Notch-signalling activation and modulates neuroinflammation events, possibly participating in the pathogenesis of multiple sclerosis and other inflammatory disorders.<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491522">42</a></sup> A functional protein association network analysis of <em>CNTN1</em> using STRING shows contactin 1 is in the same network as <em>PSEN2</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 11</a>), supporting its potential role in neurodegeneration. Further replication will be important in view of the absence of a genome-wide significant association in the discovery stage; however, this association seems promising. Notably, <em>LRRK2</em> is located less than 500 000 base pairs away from the most associated SNP at this locus, which could suggest that the association might be driven by variation at the <em>LRRK2</em> locus. Further validation of the involvement of <em>CNTN1</em> variation in modifying risk of dementia with Lewy bodies will be important.</p><p id="P37">In addition to performing a genome-wide association study with clinicopathological Alzheimer’s disease dementia, Beecham and colleagues<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491521">38</a></sup> also analysed commonly comorbid neuropathological features seen in elderly individuals with dementia, including Lewy body disease. In this latter analysis, only the <em>APOE</em> locus was found to achieve genome-wide significance. However, when testing known common Alzheimer’s disease risk variants with coincident neuropathological features, Beecham and colleagues identified hits at <em>SORL1</em> and <em>MEF2C</em>, finding them to be nominally associated. In our cohort of participants with dementia with Lewy bodies, we found no genome-wide significant associations between these variants and disease. Similarly, we had previously reported an association at the <em>SCARB2</em> locus with dementia with Lewy bodies.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491561">7</a></sup> In the larger dataset of the present study, the association remained at the suggestive level and did not reach genome-wide significance (most significant SNP in the present study, rs13141895; p=9·58 × 10<sup>−4</sup>). No other variant previously reported to be significantly associated with Alzheimer’s disease or Parkinson’s disease in recent genome-wide association study meta-analyses showed a genome-wide significant association with dementia with Lewy bodies. The most significant Alzheimer’s disease or Parkinson’s disease variants at the following loci showed nominal (p&lt;0·05) association levels: <em>MAPT</em>, <em>BIN1</em>, <em>GAK</em>, <em>HLA-DBQB1</em>, <em>CD2AP</em>, <em>INPP5D</em>, <em>ECHDC3</em>, and <em>SCIMP</em>. Additionally, variants previously suggested to be associated with Lewy-related pathology in a Finnish cohort,<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491516">28</a></sup> did not show evidence of association in this study (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 5</a>). See <a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 12–70</a> for colocalisation plots of association between dementia with Lewy bodies and either Parkinson’s disease or Alzheimer’s disease.</p><p id="P38">This study has notable limitations. The control population is not perfectly matched to the case cohort because it was derived from publicly available data. To address this, we have used all available information (both clinical and genetic) to create a control cohort that is as similar as possible to the case cohort. Additionally, despite using the same diagnostic criteria for all included participants with dementia with Lewy bodies, diagnostic measurements were collected in a variety of locations, suggesting that diagnostic accuracy might have been variable, with contamination from participants with Parkinson’s disease or Alzheimer’s disease. Notably, we do not see an over-representation of genetic risk factors from those diseases in our results (eg, <em>MAPT</em>, <em>CLU</em>, or <em>CR1</em>), suggesting minimum inclusion. Similarly, population stratification could bias the results because samples were collected in various countries. In the present study, we have used standard methodology to correct for any such bias and, consequently, our results show no evidence of population stratification as evidenced by the Q–Q plot as defined by the acquired unbiased genotype data. Additionally, participants with dementia with Lewy bodies were genotyped at three locations and controls were all derived from publicly available datasets, using a mixture of genotyping arrays, which could provide a source of genotyping bias. However, our approach was to select variants that were at the intersection of all used arrays before imputation, which makes use, effectively, of the same genotyping probes for all samples. This approach has been shown to remove any bias from this type of result and any effects of using different array scanners are negligible for high-quality variants.<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491560">43</a></sup></p><p id="P39" class="p p-last">This is the first large-scale genome-wide association study in dementia with Lewy bodies. We estimate the heritability of dementia with Lewy bodies to be approximately 36%, which is similar to what is known to occur in Parkinson’s disease.<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_664491519">44</a></sup> This finding shows that, despite not having multiple causative genes identified so far, genetics has a relevant role in the common forms of dementia with Lewy bodies. Additionally, we provide evidence suggesting that novel dementia with Lewy bodies loci are likely to be found at chromosomes 5, 6, 7, and 13 in view of the high heritability estimates at these chromosomes. A significant majority of our case cohort in the present study was comprised of participants with neuropathological diagnoses, which provide a greater level of information for diagnostic accuracy. These results provide us with the first glimpse into the molecular pathogenesis of dementia with Lewy bodies; they reveal that this disorder has a strong genetic component and suggest a unique genetic risk profile. From a molecular perspective, dementia with Lewy bodies does not simply sit between Parkinson’s disease and Alzheimer’s disease; instead, the combination of risk alleles is unique, with loci that are established risk factors for those diseases having no clear role in dementia with Lewy bodies (eg, <em>MCCC1</em>, <em>STK39</em>, <em>CLU</em>, <em>CR1</em>, or <em>PICALM</em>). Further increases in the size of dementia with Lewy body cohorts will probably reveal additional common genetic risk loci, and these will, in turn, improve understanding of this disease, its commonalities, and differences with other neurodegenerative conditions, ultimately allowing the identification of disease-specific targets for future therapeutic approaches.</p></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="idm140263506498112title">Research in context</h3></div><div class="sec" id="S20"><h3 id="S20title">Evidence before this study</h3><p id="P41" class="p p-first-last">We searched PubMed using the keywords “dementia Lewy bodies” AND “genetics”, for manuscripts published in any language between database inception and June 21, 2017, and found no large-scale genome-wide studies of dementia with Lewy bodies. So far, most studies have focused on small cohorts and are frequently candidate gene association studies. However, in 2014, we showed that dementia with Lewy bodies has a genetic component, suggesting that a large unbiased genetic association study might provide novel loci that have a role in the disease.</p></div><div class="sec" id="S21"><h3 id="S21title">Added value of this study</h3><p id="P43" class="p p-first-last">To our knowledge, this is the first large-scale genome-wide association study in dementia with Lewy bodies. The discovery stage included 1216 patients with dementia with Lewy bodies and 3791 controls, and the replication stage included 527 patients with the disease and 663 controls. The vast majority of people with the disease from both stages were neuropathologically diagnosed. Furthermore, despite the comparatively smaller size of the replication cohort, all samples were ascertained at the same centre, which reduces diagnostic heterogeneity.</p></div><div class="sec" id="S22"><h3 id="S22title">Implications of all the available evidence</h3><p id="P45" class="p p-first-last">Our data show several genome-wide significant loci. Some of these loci had previously been implicated in Parkinson’s disease or Alzheimer’s disease, which could suggest that dementia with Lewy bodies is simply a combination of the genetic underpinnings underlying those diseases. However, our data suggest that dementia with Lewy bodies does not sit in the spectrum between Parkinson’s disease and Alzheimer’s disease, but instead, has a unique genetic profile. Additionally, we have also estimated the genetic heritability of dementia with Lewy bodies to be 36%, which is very close to what has been estimated for Parkinson’s disease, a disease now known to have a strong genetic component.</p></div></div></div></div><div id="S15" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>supplemental</h4><div class="sup-box half_rhythm" id="d36e1529"><a href="/pmc/articles/PMC5805394/bin/NIHMS938346-supplement-supplemental.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(6.8M, pdf)</sup></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Acknowledgments</h2><div class="sec"><p id="__p2"><strong>Funding:</strong> The Alzheimer’s Society and the Lewy Body Society.</p><p id="__p3">The authors would like to thank Ian McKeith for his continued support and encouragement. This study was supported in part by the National Institutes of Neurological Disease and Stroke. JB and RG’s work is funded by research fellowships from the Alzheimer’s Society. TO is supported by a scholarship from the Lewy Body Society. For the neuropathologically confirmed samples from Australia, tissues were received from the Sydney Brain Bank, which is supported by Neuroscience Research Australia and the University of New South Wales, and GMH is funded by a National Health and Medical Research Council senior principal research fellowship. We would like to thank the South West Dementia Brain Bank (SWDBB) for providing brain tissue for this study. The SWDBB is supported by BRACE (Bristol Research into Alzheimer’s and Care of the Elderly), Brains for Dementia Research, and the Medical Research Council. We acknowledge the Oxford Brain Bank, supported by the Medical Research Council, Brains for Dementia Research (Alzheimer’s Society and Alzheimer’s Research UK), Autistica UK, and the National Institute for Health Research Oxford Biomedical Research Centre. The brain samples and bio samples were obtained from The Netherlands Brain Bank, Netherlands Institute for Neuroscience, Amsterdam. All tissue samples were collected from donors with written informed consent for a brain autopsy, and the use of the material and clinical information for research purposes was obtained by the Netherlands Brain Bank. This study was also partly funded by the Wellcome Trust, the Medical Research Council, and the Canadian Institutes of Health Research (PStG-H). Research from YC was supported by the CERCA Programme, Generalitat de Catalunya, Barcelona, Catalonia, Spain. The Nottingham Genetics Group is supported by Alzheimer’s Research UK and The Big Lottery Fund. The contributions from Columbia University were supported by the Taub Institute, the Panasci Fund, the Parkinson’s Disease Foundation, and National Institutes of Health grants NS060113 (LC), P50AG008702, P50NS038370, and UL1TR000040. OAR is supported by the Michael J Fox Foundation for Parkinson’s Research (NINDS R01# NS078086). The Mayo Clinic Jacksonville is a Morris K Udall Parkinson’s Disease Research Center of Excellence (NINDS P50 #NS072187) and is supported by The Little Family Foundation, the Mangurian Foundation Program for Lewy Body Dementia research, and the Alzheimer Disease Research Center (P50 AG016547). The research from the Mayo Clinic Rochester is supported by the National Institute on Aging (P50 AG016574 and U01 <a id="__tag_813753988" class="tag_hotlink" href="/nuccore/AG006786" ref="/nuccore/AG006786"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG006786","term_id":"3063075","term_text":"AG006786"}}</span>AG006786</a>). This research has received support from The Queen Square Brain Bank at the University College London Institute of Neurology (where TL is funded by an Alzheimer’s Research UK senior fellowship). Some of the tissue samples studied were provided by the Medical Research Council London Neurodegenerative Diseases Brain Bank and the Brains for Dementia Research project (funded by Alzheimer’s Society and Alzheimer’s Research UK). This research was supported in part by both the National Institute for Health Research University College London Hospital Biomedical Research Centre and the Queen Square Dementia Biomedical Research Unit. This research was supported in part by the Intramural Research Program of the National Institute on Aging, National Institutes of Health, Department of Health and Human Services (project AG000951-12). The University of Pennsylvania case collection is funded by the Penn Alzheimer’s Disease Core Center (AG10124) and the Penn Morris K Udall Parkinson’s Disease Research Center (NS053488). Tissue samples from University of California San Diego are supported by National Institutes of Health grant AG05131. The authors thank the brain bank GIE NeuroCEB and the French program <em>Investissements d’avenir</em> (ANR-10-IAIHU-06). PJT and LM are supported by the Helsinki University Central Hospital, the Folkhälsan Research Foundation, and the Finnish Academy. This research was partly supported by the Canadian Consortium on Neurodegeneration in Aging (ER). The Genotype-Tissue Expression (GTEx) Project was supported by the Common Fund of the Office of the Director of the National Institutes of Health, and by the National Cancer Institute; the National Human Genome Research Institute; the National Heart, Lung, and Blood Institute; the National Institute on Drug Abuse; the National Institute of Mental Health; and the National Institute of Neurological Disorders and Stroke. The data used for the analyses described in this manuscript were obtained from the GTEx Portal on April 1, 2017. The authors acknowledge the contribution of data from Genetic Architecture of Smoking and Smoking Cessation accessed through the database of Genotypes and Phenotypes. Funding support for genotyping, which was done at the Center for Inherited Disease Research (CIDR), was provided by 1 X01 HG005274-01. CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University (contract number HHSN268200782096C). Assistance with genotype cleaning and general study coordination was provided by the Gene Environment Association Studies (GENEVA) Coordinating Center (U01 <a id="__tag_813753987" class="tag_hotlink" href="/nuccore/HG004446" ref="/nuccore/HG004446"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG004446","term_id":"585865293","term_text":"HG004446"}}</span>HG004446</a>). Funding support for collection of datasets and samples was provided by the Collaborative Genetic Study of Nicotine Dependence (COGEND; P01 CA089392) and the University of Wisconsin Transdisciplinary Tobacco Use Research Center (P50 DA019706, P50 CA084724). The data used for the analyses described in this Article were obtained from the database of Genotypes and Phenotypes. Genotype and phenotype data for the Genetic Analysis of Psoriasis and Psoriatic Arthritis study were provided by James T Elder (University of Michigan), with collaborators Dafna Gladman (University of Toronto) and Proton Rahman (Memorial University of Newfoundland) providing samples. This research was supported in part by the Intramural Research Program of the National Institutes of Health (National Institute of Neurological Disorders and Stroke; project ZIA NS003154). Tissue samples for genotyping were provided by the Johns Hopkins Morris K Udall Center of Excellence for Parkinson’s Disease Research (NIH P50 NS38377) and the Johns Hopkins Alzheimer’s Disease Research Center (NIH P50 AG05146). This study was supported by grants from the National Institutes of Health, the Canadian Institute for Health Research, and the Krembil Foundation. Additional support was provided by the Babcock Memorial Trust and by the Barbara and Neal Henschel Charitable Foundation. James T Elder is supported by the Ann Arbor Veterans Affairs Hospital (NIH R01 AR042742). The authors would like to thank the Genome Aggregation Database and the groups that provided exome and genome variant data to this resource. A full list of contributing groups can be found <a href="#SD1" rid="SD1" class=" supplementary-material">online</a>.</p></div></div><div id="idm140263489514336" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140263489514336title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="__p4" class="p p-first"><strong>Declaration of interests</strong></p><p id="__p5" class="p p-last">We declare no competing interests.</p><p></p><p class="fn sec" id="FN4"></p><p id="__p6" class="p p-first-last">See <a href="#SD1" rid="SD1" class=" supplementary-material">Online</a> for <a href="#SD1" rid="SD1" class=" supplementary-material">appendix</a></p><p></p><p class="fn sec" id="FN5"></p><p id="__p7" class="p p-first-last">For the <strong>GTEx website</strong> see <a href="https://www.gtexportal.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.gtexportal.org/</a></p><p></p><p class="fn sec" id="FN6"></p><p id="__p8" class="p p-first-last">For the <strong>Harvard Brain Bank</strong> website see <a href="http://www.brainbank.mclean.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.brainbank.mclean.org</a></p><p></p><p class="fn sec" id="FN7"></p><p id="__p9" class="p p-first-last">For the <strong>PDGene database</strong> see <a href="http://www.pdgene.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.pdgene.org/</a></p><p></p><p class="fn sec" id="FN8"></p><p id="__p10" class="p p-first-last">For <strong>The Netherlands Brain Bank</strong> see: <a href="http://www.brainbank.nl" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.brainbank.nl</a></p><p></p><p class="fn sec" id="FN9"></p><p id="__p11" class="p p-first-last">For the <strong>National Institutes of Health database of Genotypes and Phenotypes</strong> see: <a href="http://www.ncbi.nlm.nih.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gap</a></p><p></p><p class="fn sec" id="FN10"></p><p id="__p12" class="p p-first-last">For the full list of <strong>contributing authors</strong> see <a href="http://gnomad.broadinstitute.org/about" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://gnomad.broadinstitute.org/about</a></p><p></p><div><p id="__p13" class="p p-first"><strong>Contributors</strong></p>JB, RG, JH, DJS, and AS designed the study. JB, AS, DJS, and OAR obtained funding for the study. JB, RG, OAR, CK-R, LD, SWS, and DGH did the data acquisition. JB, RG, OAR, and CK-R analysed and interpreted the data. CES, LP, SWS, OA, JC, LC, LSH, KMa, ALee, ALem, ALl, EL, ER, PStG-H, EL, HZ, IB, AB, KB, KMo, CT, SA-S, TL, JH, YC, VVD, JQT, GES, TGB, SL, DG, EM, IS, PP, PJT, LM, MO, TR, BFB, RCP, TJF, VE-P, NG-R, NJC, JCM, DJS, SP-B, DM, DWD, and GMH collected and characterised samples. JB, RG, OAR, CK-R, and TO wrote the first draft of the manuscript. All other co-authors participated in preparation of the manuscript by reading and commenting on drafts before submission.</div></div></div><div id="idm140263546940960" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140263546940960title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Rahkonen T, Eloniemi-Sulkava U, Rissanen S, Vatanen A, Viramo P, Sulkava R. Dementia with Lewy bodies according to the consensus criteria in a general population aged 75 years or older. <span><span class="ref-journal">J Neurol Neurosurg Psychiatry. </span>2003;<span class="ref-vol">74</span>:720–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1738491/">PMC free article</a>]</span> [<a href="/pubmed/12754338" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurol+Neurosurg+Psychiatry&amp;title=Dementia+with+Lewy+bodies+according+to+the+consensus+criteria+in+a+general+population+aged+75+years+or+older&amp;author=T+Rahkonen&amp;author=U+Eloniemi-Sulkava&amp;author=S+Rissanen&amp;author=A+Vatanen&amp;author=P+Viramo&amp;volume=74&amp;publication_year=2003&amp;pages=720-24&amp;pmid=12754338&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Bogaerts V, Engelborghs S, Kumar-Singh S, et al.  A novel locus for dementia with Lewy bodies: a clinically and genetically heterogeneous disorder. <span><span class="ref-journal">Brain. </span>2007;<span class="ref-vol">130</span>:2277–91.</span> [<a href="/pubmed/17681982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=A+novel+locus+for+dementia+with+Lewy+bodies:+a+clinically+and+genetically+heterogeneous+disorder&amp;author=V+Bogaerts&amp;author=S+Engelborghs&amp;author=S+Kumar-Singh&amp;volume=130&amp;publication_year=2007&amp;pages=2277-91&amp;pmid=17681982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Walker Z, Possin KL, Boeve BF, Aarsland D. Lewy body dementias. <span><span class="ref-journal">Lancet. </span>2015;<span class="ref-vol">386</span>:1683–97.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5792067/">PMC free article</a>]</span> [<a href="/pubmed/26595642" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Lewy+body+dementias&amp;author=Z+Walker&amp;author=KL+Possin&amp;author=BF+Boeve&amp;author=D+Aarsland&amp;volume=386&amp;publication_year=2015&amp;pages=1683-97&amp;pmid=26595642&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Keogh MJ, Kurzawa-Akanbi M, Griffin H, et al.  Exome sequencing in dementia with Lewy bodies. <span><span class="ref-journal">Transl Psychiatry. </span>2016;<span class="ref-vol">6</span>:e728.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4872424/">PMC free article</a>]</span> [<a href="/pubmed/26836416" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transl+Psychiatry&amp;title=Exome+sequencing+in+dementia+with+Lewy+bodies&amp;author=MJ+Keogh&amp;author=M+Kurzawa-Akanbi&amp;author=H+Griffin&amp;volume=6&amp;publication_year=2016&amp;pages=e728&amp;pmid=26836416&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Geiger JT, Ding J, Crain B, et al.  Next-generation sequencing reveals substantial genetic contribution to dementia with Lewy bodies. <span><span class="ref-journal">Neurobiol Dis. </span>2016;<span class="ref-vol">94</span>:55–62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4983488/">PMC free article</a>]</span> [<a href="/pubmed/27312774" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol+Dis&amp;title=Next-generation+sequencing+reveals+substantial+genetic+contribution+to+dementia+with+Lewy+bodies&amp;author=JT+Geiger&amp;author=J+Ding&amp;author=B+Crain&amp;volume=94&amp;publication_year=2016&amp;pages=55-62&amp;pmid=27312774&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Tsuang D, Leverenz JB, Lopez OL, et al.  <em>APOE</em> ε4 increases risk for dementia in pure synucleinopathies. <span><span class="ref-journal">JAMA Neurol. </span>2013;<span class="ref-vol">70</span>:223–28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3580799/">PMC free article</a>]</span> [<a href="/pubmed/23407718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA+Neurol&amp;title=APOE+ε4+increases+risk+for+dementia+in+pure+synucleinopathies&amp;author=D+Tsuang&amp;author=JB+Leverenz&amp;author=OL+Lopez&amp;volume=70&amp;publication_year=2013&amp;pages=223-28&amp;pmid=23407718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Bras J, Guerreiro R, Darwent L, et al.  Genetic analysis implicates <em>APOE</em>, <em>SNCA</em> and suggests lysosomal dysfunction in the etiology of dementia with Lewy bodies. <span><span class="ref-journal">Hum Mol Genet. </span>2014;<span class="ref-vol">23</span>:6139–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4222357/">PMC free article</a>]</span> [<a href="/pubmed/24973356" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genetic+analysis+implicates+APOE,+SNCA+and+suggests+lysosomal+dysfunction+in+the+etiology+of+dementia+with+Lewy+bodies&amp;author=J+Bras&amp;author=R+Guerreiro&amp;author=L+Darwent&amp;volume=23&amp;publication_year=2014&amp;pages=6139-46&amp;pmid=24973356&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Nalls MA, Duran R, Lopez G, et al.  A multicenter study of glucocerebrosidase mutations in dementia with Lewy bodies. <span><span class="ref-journal">JAMA Neurol. </span>2013;<span class="ref-vol">70</span>:727–35.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3841974/">PMC free article</a>]</span> [<a href="/pubmed/23588557" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA+Neurol&amp;title=A+multicenter+study+of+glucocerebrosidase+mutations+in+dementia+with+Lewy+bodies&amp;author=MA+Nalls&amp;author=R+Duran&amp;author=G+Lopez&amp;volume=70&amp;publication_year=2013&amp;pages=727-35&amp;pmid=23588557&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Guerreiro R, Escott-Price V, Darwent L, et al.  Genome-wide analysis of genetic correlation in dementia with Lewy bodies, Parkinson’s and Alzheimer’s diseases. <span><span class="ref-journal">Neurobiol Aging. </span>2016;<span class="ref-vol">38</span>:214.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4759606/">PMC free article</a>]</span> [<a href="/pubmed/26643944" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol+Aging&amp;title=Genome-wide+analysis+of+genetic+correlation+in+dementia+with+Lewy+bodies,+Parkinson’s+and+Alzheimer’s+diseases&amp;author=R+Guerreiro&amp;author=V+Escott-Price&amp;author=L+Darwent&amp;volume=38&amp;publication_year=2016&amp;pages=214&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Moskvina V, Harold D, Russo G, et al.  Analysis of genome-wide association studies of Alzheimer disease and of Parkinson disease to determine if these 2 diseases share a common genetic risk. <span><span class="ref-journal">JAMA Neurol. </span>2013;<span class="ref-vol">70</span>:1268–76.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5978422/">PMC free article</a>]</span> [<a href="/pubmed/23921447" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA+Neurol&amp;title=Analysis+of+genome-wide+association+studies+of+Alzheimer+disease+and+of+Parkinson+disease+to+determine+if+these+2+diseases+share+a+common+genetic+risk&amp;author=V+Moskvina&amp;author=D+Harold&amp;author=G+Russo&amp;volume=70&amp;publication_year=2013&amp;pages=1268-76&amp;pmid=23921447&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">McKeith IG, Dickson DW, Lowe J, et al.  Diagnosis and management of dementia with Lewy bodies: third report of the DLB Consortium. <span><span class="ref-journal">Neurology. </span>2005;<span class="ref-vol">65</span>:1863–72.</span> [<a href="/pubmed/16237129" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Diagnosis+and+management+of+dementia+with+Lewy+bodies:+third+report+of+the+DLB+Consortium&amp;author=IG+McKeith&amp;author=DW+Dickson&amp;author=J+Lowe&amp;volume=65&amp;publication_year=2005&amp;pages=1863-72&amp;pmid=16237129&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. <span><span class="ref-journal">Gigascience. </span>2015;<span class="ref-vol">4</span>:7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4342193/">PMC free article</a>]</span> [<a href="/pubmed/25722852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Second-generation+PLINK:+rising+to+the+challenge+of+larger+and+richer+datasets&amp;author=CC+Chang&amp;author=CC+Chow&amp;author=LC+Tellier&amp;author=S+Vattikuti&amp;author=SM+Purcell&amp;volume=4&amp;publication_year=2015&amp;pages=7&amp;pmid=25722852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">McCarthy S, Das S, Kretzschmar W, et al.  A reference panel of 64 976 haplotypes for genotype imputation. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:1279–83.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5388176/">PMC free article</a>]</span> [<a href="/pubmed/27548312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+reference+panel+of+64+976+haplotypes+for+genotype+imputation&amp;author=S+McCarthy&amp;author=S+Das&amp;author=W+Kretzschmar&amp;volume=48&amp;publication_year=2016&amp;pages=1279-83&amp;pmid=27548312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Loh P-R, Danecek P, Palamara PF, et al.  Reference-based phasing using the Haplotype Reference Consortium panel. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:1443–48.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5096458/">PMC free article</a>]</span> [<a href="/pubmed/27694958" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Reference-based+phasing+using+the+Haplotype+Reference+Consortium+panel&amp;author=P-R+Loh&amp;author=P+Danecek&amp;author=PF+Palamara&amp;volume=48&amp;publication_year=2016&amp;pages=1443-48&amp;pmid=27694958&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Das S, Forer L, Schönherr S, et al.  Next-generation genotype imputation service and methods. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:1284–87.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5157836/">PMC free article</a>]</span> [<a href="/pubmed/27571263" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Next-generation+genotype+imputation+service+and+methods&amp;author=S+Das&amp;author=L+Forer&amp;author=S+Schönherr&amp;volume=48&amp;publication_year=2016&amp;pages=1284-87&amp;pmid=27571263&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Wu MC, Lee S, Cai T, Li Y, Boehnke M, Lin X. Rare-variant association testing for sequencing data with the sequence kernel association test. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">89</span>:82–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3135811/">PMC free article</a>]</span> [<a href="/pubmed/21737059" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Rare-variant+association+testing+for+sequencing+data+with+the+sequence+kernel+association+test&amp;author=MC+Wu&amp;author=S+Lee&amp;author=T+Cai&amp;author=Y+Li&amp;author=M+Boehnke&amp;volume=89&amp;publication_year=2011&amp;pages=82-93&amp;pmid=21737059&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Lee S, Emond MJ, Bamshad MJ, et al.  Optimal unified approach for rare-variant association testing with application to small-sample case-control whole-exome sequencing studies. <span><span class="ref-journal">Am J Hum Genet. </span>2012;<span class="ref-vol">91</span>:224–37.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3415556/">PMC free article</a>]</span> [<a href="/pubmed/22863193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Optimal+unified+approach+for+rare-variant+association+testing+with+application+to+small-sample+case-control+whole-exome+sequencing+studies&amp;author=S+Lee&amp;author=MJ+Emond&amp;author=MJ+Bamshad&amp;volume=91&amp;publication_year=2012&amp;pages=224-37&amp;pmid=22863193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Kang HM.  [accessed April 1, 2017];<span><span class="ref-journal">EPACTS: efficient and parallelizable association container toolbox. </span>2014 </span>  <a href="https://genome.sph.umich.edu/wiki/EPACTS" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://genome.sph.umich.edu/wiki/EPACTS</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Mägi R, Morris AP. GWAMA: software for genome-wide association meta-analysis. <span><span class="ref-journal">BMC Bioinformatics. </span>2010;<span class="ref-vol">11</span>:288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2893603/">PMC free article</a>]</span> [<a href="/pubmed/20509871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=GWAMA:+software+for+genome-wide+association+meta-analysis&amp;author=R+Mägi&amp;author=AP+Morris&amp;volume=11&amp;publication_year=2010&amp;pages=288&amp;pmid=20509871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Yang J, Benyamin B, McEvoy BP, et al.  Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:565–69.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J+Yang&amp;author=B+Benyamin&amp;author=BP+McEvoy&amp;volume=42&amp;publication_year=2010&amp;pages=565-69&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Lee SH, Wray NR, Goddard ME, Visscher PM. Estimating missing heritability for disease from genome-wide association studies. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:294–305.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3059431/">PMC free article</a>]</span> [<a href="/pubmed/21376301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Estimating+missing+heritability+for+disease+from+genome-wide+association+studies&amp;author=SH+Lee&amp;author=NR+Wray&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=294-305&amp;pmid=21376301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Zaccai J, McCracken C, Brayne C. A systematic review of prevalence and incidence studies of dementia with Lewy bodies. <span><span class="ref-journal">Age Ageing. </span>2005;<span class="ref-vol">34</span>:561–66.</span> [<a href="/pubmed/16267179" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Age+Ageing&amp;title=A+systematic+review+of+prevalence+and+incidence+studies+of+dementia+with+Lewy+bodies&amp;author=J+Zaccai&amp;author=C+McCracken&amp;author=C+Brayne&amp;volume=34&amp;publication_year=2005&amp;pages=561-66&amp;pmid=16267179&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Lek M, Karczewski KJ, Minikel EV, et al.  Analysis of protein-coding genetic variation in 60 706 humans. <span><span class="ref-journal">Nature. </span>2016;<span class="ref-vol">536</span>:285–91.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5018207/">PMC free article</a>]</span> [<a href="/pubmed/27535533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Analysis+of+protein-coding+genetic+variation+in+60+706+humans&amp;author=M+Lek&amp;author=KJ+Karczewski&amp;author=EV+Minikel&amp;volume=536&amp;publication_year=2016&amp;pages=285-91&amp;pmid=27535533&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Guerreiro R, Wojtas A, Bras J, et al.  TREM2 variants in Alzheimer’s disease. <span><span class="ref-journal">N Engl J Med. </span>2013;<span class="ref-vol">368</span>:117–27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3631573/">PMC free article</a>]</span> [<a href="/pubmed/23150934" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=TREM2+variants+in+Alzheimer’s+disease&amp;author=R+Guerreiro&amp;author=A+Wojtas&amp;author=J+Bras&amp;volume=368&amp;publication_year=2013&amp;pages=117-27&amp;pmid=23150934&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Labbé C, Heckman MG, Lorenzo-Betancor O, et al.  MAPT haplotype H1G is associated with increased risk of dementia with Lewy bodies. <span><span class="ref-journal">Alzheimers Dement. </span>2016 doi: 10.1016/j.jalz.2016.05.002.</span> published online June 7. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5143206/">PMC free article</a>]</span> [<a href="/pubmed/27287057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.jalz.2016.05.002" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alzheimers+Dement&amp;title=MAPT+haplotype+H1G+is+associated+with+increased+risk+of+dementia+with+Lewy+bodies&amp;author=C+Labbé&amp;author=MG+Heckman&amp;author=O+Lorenzo-Betancor&amp;publication_year=2016&amp;doi=10.1016/j.jalz.2016.05.002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Zhang B, Gaiteri C, Bodea L-G, et al.  Integrated systems approach identifies genetic nodes and networks in late-onset Alzheimer’s disease. <span><span class="ref-journal">Cell. </span>2013;<span class="ref-vol">153</span>:707–20.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3677161/">PMC free article</a>]</span> [<a href="/pubmed/23622250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Integrated+systems+approach+identifies+genetic+nodes+and+networks+in+late-onset+Alzheimer’s+disease&amp;author=B+Zhang&amp;author=C+Gaiteri&amp;author=L-G+Bodea&amp;volume=153&amp;publication_year=2013&amp;pages=707-20&amp;pmid=23622250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Tsuang D, Leverenz JB, Lopez OL, et al.  <em>APOE</em> ε4 increases risk for dementia in pure synucleinopathies. <span><span class="ref-journal">JAMA Neurol. </span>2013;<span class="ref-vol">70</span>:223–28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3580799/">PMC free article</a>]</span> [<a href="/pubmed/23407718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA+Neurol&amp;title=APOE+ε4+increases+risk+for+dementia+in+pure+synucleinopathies&amp;author=D+Tsuang&amp;author=JB+Leverenz&amp;author=OL+Lopez&amp;volume=70&amp;publication_year=2013&amp;pages=223-28&amp;pmid=23407718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Peuralinna T, Myllykangas L, Oinas M, et al.  Genome-wide association study of neocortical Lewy-related pathology. <span><span class="ref-journal">Ann Clin Transl Neurol. </span>2015;<span class="ref-vol">2</span>:920–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4574809/">PMC free article</a>]</span> [<a href="/pubmed/26401513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Clin+Transl+Neurol&amp;title=Genome-wide+association+study+of+neocortical+Lewy-related+pathology&amp;author=T+Peuralinna&amp;author=L+Myllykangas&amp;author=M+Oinas&amp;volume=2&amp;publication_year=2015&amp;pages=920-31&amp;pmid=26401513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Nalls MA, Pankratz N, Lill CM, et al.  Large-scale meta-analysis of genome-wide association data identifies six new risk loci for Parkinson’s disease. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:989–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4146673/">PMC free article</a>]</span> [<a href="/pubmed/25064009" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Large-scale+meta-analysis+of+genome-wide+association+data+identifies+six+new+risk+loci+for+Parkinson’s+disease&amp;author=MA+Nalls&amp;author=N+Pankratz&amp;author=CM+Lill&amp;volume=46&amp;publication_year=2014&amp;pages=989-93&amp;pmid=25064009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">GTEx Consortium. Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. <span><span class="ref-journal">Science. </span>2015;<span class="ref-vol">348</span>:648–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4547484/">PMC free article</a>]</span> [<a href="/pubmed/25954001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Human+genomics.+The+Genotype-Tissue+Expression+(GTEx)+pilot+analysis:+multitissue+gene+regulation+in+humans&amp;volume=348&amp;publication_year=2015&amp;pages=648-60&amp;pmid=25954001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Westra H-J, Peters MJ, Esko T, et al.  Systematic identification of trans eQTLs as putative drivers of known disease associations. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:1238–43.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3991562/">PMC free article</a>]</span> [<a href="/pubmed/24013639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Systematic+identification+of+trans+eQTLs+as+putative+drivers+of+known+disease+associations&amp;author=H-J+Westra&amp;author=MJ+Peters&amp;author=T+Esko&amp;volume=45&amp;publication_year=2013&amp;pages=1238-43&amp;pmid=24013639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">International Parkinson’s Disease Genomics Consortium (IPDGC), Wellcome Trust Case Control Consortium 2 (WTCCC2) A two-stage meta-analysis identifies several new loci for Parkinson’s disease. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002142.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128098/">PMC free article</a>]</span> [<a href="/pubmed/21738488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+two-stage+meta-analysis+identifies+several+new+loci+for+Parkinson’s+disease&amp;volume=7&amp;publication_year=2011&amp;pages=e1002142&amp;pmid=21738488&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Smirnova T, Stinnakre J, Mallet J. Characterization of a presynaptic glutamate receptor. <span><span class="ref-journal">Science. </span>1993;<span class="ref-vol">262</span>:430–33.</span> [<a href="/pubmed/8105537" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Characterization+of+a+presynaptic+glutamate+receptor&amp;author=T+Smirnova&amp;author=J+Stinnakre&amp;author=J+Mallet&amp;volume=262&amp;publication_year=1993&amp;pages=430-33&amp;pmid=8105537&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Schubert J, Siekierska A, Langlois M, et al.  Mutations in <em>STX1B</em>, encoding a presynaptic protein, cause fever-associated epilepsy syndromes. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:1327–32.</span> [<a href="/pubmed/25362483" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Mutations+in+STX1B,+encoding+a+presynaptic+protein,+cause+fever-associated+epilepsy+syndromes&amp;author=J+Schubert&amp;author=A+Siekierska&amp;author=M+Langlois&amp;volume=46&amp;publication_year=2014&amp;pages=1327-32&amp;pmid=25362483&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Vlaskamp DRM, Rump P, Callenbach PMC, et al.  Haploinsufficiency of the <em>STX1B</em> gene is associated with myoclonic astatic epilepsy. <span><span class="ref-journal">Eur J Paediatr Neurol. </span>2016;<span class="ref-vol">20</span>:489–92.</span> [<a href="/pubmed/26818399" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Paediatr+Neurol&amp;title=Haploinsufficiency+of+the+STX1B+gene+is+associated+with+myoclonic+astatic+epilepsy&amp;author=DRM+Vlaskamp&amp;author=P+Rump&amp;author=PMC+Callenbach&amp;volume=20&amp;publication_year=2016&amp;pages=489-92&amp;pmid=26818399&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Vacic V, Ozelius LJ, Clark LN, et al.  Genome-wide mapping of IBD segments in an Ashkenazi PD cohort identifies associated haplotypes. <span><span class="ref-journal">Hum Mol Genet. </span>2014;<span class="ref-vol">23</span>:4693–702.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4119402/">PMC free article</a>]</span> [<a href="/pubmed/24842889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+mapping+of+IBD+segments+in+an+Ashkenazi+PD+cohort+identifies+associated+haplotypes&amp;author=V+Vacic&amp;author=LJ+Ozelius&amp;author=LN+Clark&amp;volume=23&amp;publication_year=2014&amp;pages=4693-702&amp;pmid=24842889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Li QS, Parrado AR, Samtani MN, Narayan VA Alzheimer’s Disease Neuroimaging Initiative. Variations in the FRA10AC1 fragile site and 15q21 are associated with cerebrospinal fluid Aβ1–42 level. <span><span class="ref-journal">PLoS One. </span>2015;<span class="ref-vol">10</span>:e0134000.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4529186/">PMC free article</a>]</span> [<a href="/pubmed/26252872" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Variations+in+the+FRA10AC1+fragile+site+and+15q21+are+associated+with+cerebrospinal+fluid+Aβ1–42+level&amp;author=QS+Li&amp;author=AR+Parrado&amp;author=MN+Samtani&amp;author=VA+Narayan&amp;volume=10&amp;publication_year=2015&amp;pages=e0134000&amp;pmid=26252872&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Beecham GW, Hamilton K, Naj AC, et al.  Genome-wide association meta-analysis of neuropathologic features of Alzheimer’s disease and related dementias. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004606.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4154667/">PMC free article</a>]</span> [<a href="/pubmed/25188341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+meta-analysis+of+neuropathologic+features+of+Alzheimer’s+disease+and+related+dementias&amp;author=GW+Beecham&amp;author=K+Hamilton&amp;author=AC+Naj&amp;volume=10&amp;publication_year=2014&amp;pages=e1004606&amp;pmid=25188341&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Berglund E, Stigbrand T, Carlsson SR. Isolation and characterization of a membrane glycoprotein from human brain with sequence similarities to cell adhesion proteins from chicken and mouse. <span><span class="ref-journal">Eur J Biochem. </span>1991;<span class="ref-vol">197</span>:549–54.</span> [<a href="/pubmed/2026173" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Biochem&amp;title=Isolation+and+characterization+of+a+membrane+glycoprotein+from+human+brain+with+sequence+similarities+to+cell+adhesion+proteins+from+chicken+and+mouse&amp;author=E+Berglund&amp;author=T+Stigbrand&amp;author=SR+Carlsson&amp;volume=197&amp;publication_year=1991&amp;pages=549-54&amp;pmid=2026173&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Gennarini G, Bizzoca A, Picocci S, Puzzo D, Corsi P, Furley AJW. The role of Gpi-anchored axonal glycoproteins in neural development and neurological disorders. <span><span class="ref-journal">Mol Cell Neurosci. </span>2016 doi: 10.1016/j.mcn.2016.11.006.</span> published online Nov 18. [<a href="/pubmed/27871938" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.mcn.2016.11.006" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Neurosci&amp;title=The+role+of+Gpi-anchored+axonal+glycoproteins+in+neural+development+and+neurological+disorders&amp;author=G+Gennarini&amp;author=A+Bizzoca&amp;author=S+Picocci&amp;author=D+Puzzo&amp;author=P+Corsi&amp;publication_year=2016&amp;doi=10.1016/j.mcn.2016.11.006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Compton AG, Albrecht DE, Seto JT, et al.  Mutations in contactin-1, a neural adhesion and neuromuscular junction protein, cause a familial form of lethal congenital myopathy. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">83</span>:714–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668069/">PMC free article</a>]</span> [<a href="/pubmed/19026398" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Mutations+in+contactin-1,+a+neural+adhesion+and+neuromuscular+junction+protein,+cause+a+familial+form+of+lethal+congenital+myopathy&amp;author=AG+Compton&amp;author=DE+Albrecht&amp;author=JT+Seto&amp;volume=83&amp;publication_year=2008&amp;pages=714-24&amp;pmid=19026398&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Derfuss T, Parikh K, Velhin S, et al.  Contactin-2/TAG-1-directed autoimmunity is identified in multiple sclerosis patients and mediates gray matter pathology in animals. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2009;<span class="ref-vol">106</span>:8302–07.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2688870/">PMC free article</a>]</span> [<a href="/pubmed/19416878" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Contactin-2/TAG-1-directed+autoimmunity+is+identified+in+multiple+sclerosis+patients+and+mediates+gray+matter+pathology+in+animals&amp;author=T+Derfuss&amp;author=K+Parikh&amp;author=S+Velhin&amp;volume=106&amp;publication_year=2009&amp;pages=8302-07&amp;pmid=19416878&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Johnson EO, Hancock DB, Levy JL, et al.  Imputation across genotyping arrays for genome-wide association studies: assessment of bias and a correction strategy. <span><span class="ref-journal">Hum Genet. </span>2013;<span class="ref-vol">132</span>:509–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3628082/">PMC free article</a>]</span> [<a href="/pubmed/23334152" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Imputation+across+genotyping+arrays+for+genome-wide+association+studies:+assessment+of+bias+and+a+correction+strategy&amp;author=EO+Johnson&amp;author=DB+Hancock&amp;author=JL+Levy&amp;volume=132&amp;publication_year=2013&amp;pages=509-22&amp;pmid=23334152&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Keller MF, Saad M, Bras J, et al.  Using genome-wide complex trait analysis to quantify ‘missing heritability’ in Parkinson’s disease. <span><span class="ref-journal">Hum Mol Genet. </span>2012;<span class="ref-vol">21</span>:4996–5009.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3576713/">PMC free article</a>]</span> [<a href="/pubmed/22892372" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Using+genome-wide+complex+trait+analysis+to+quantify+‘missing+heritability’+in+Parkinson’s+disease&amp;author=MF+Keller&amp;author=M+Saad&amp;author=J+Bras&amp;volume=21&amp;publication_year=2012&amp;pages=4996-5009&amp;pmid=22892372&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5805394&amp;issue-id=306234&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140263506243552title">Summary</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Methods</a></li><li><a href="#S12title">Results</a></li><li><a href="#S13title">Discussion</a></li><li><a href="#S15title">Supplementary Material</a></li><li><a href="#S16title">Acknowledgments</a></li><li><a href="#idm140263489514336title">Footnotes</a></li><li><a href="#idm140263546940960title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5805394/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5805394/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5805394/pdf/nihms938346.pdf">PDF (889K)</a></li> | <li><a href="#" data-citationid="PMC5805394" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5805394%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5805394%2F&amp;text=Investigating%20the%20genetic%20architecture%20of%20dementia%20with%20Lewy%20bodies%3A%20a%20two-stage%20genome-wide%20association%20study" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5805394%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29263008" data-aiid="5805394" data-aid="5805394" data-iid="306234" data-domainid="319" data-domain="nihpa" data-accid="PMC5805394" data-md5="0d6b38b1a1f9061895c134212769e1bb"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:49:35-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C31B2DD5ABE510000000005320300&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5805394%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5805394/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>