<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5567150/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Variants in TTC25 affect autistic trait in patients with autism spectrum disorder and general population</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5567150/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="Variants in TTC25 affect autistic trait in patients with autism spectrum disorder and general population"><meta name="citation_authors" content="Dina Vojinovic, Nathalie Brison, Shahzad Ahmad, Ilse Noens, Irene Pappa, Lennart C Karssen, Henning Tiemeier, Cornelia M van Duijn, Hilde Peeters, Najaf Amin"><meta name="citation_date" content="August 2017"><meta name="citation_issue" content="8"><meta name="citation_volume" content="25"><meta name="citation_firstpage" content="982"><meta name="citation_doi" content="10.1038/ejhg.2017.82"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5567150/?report=abstract"><meta name="citation_pmid" content="28513607"><meta name="DC.Title" content="Variants in TTC25 affect autistic trait in patients with autism spectrum disorder and general population"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Dina Vojinovic"><meta name="DC.Contributor" content="Nathalie Brison"><meta name="DC.Contributor" content="Shahzad Ahmad"><meta name="DC.Contributor" content="Ilse Noens"><meta name="DC.Contributor" content="Irene Pappa"><meta name="DC.Contributor" content="Lennart C Karssen"><meta name="DC.Contributor" content="Henning Tiemeier"><meta name="DC.Contributor" content="Cornelia M van Duijn"><meta name="DC.Contributor" content="Hilde Peeters"><meta name="DC.Contributor" content="Najaf Amin"><meta name="DC.Date" content="2017 Aug"><meta name="DC.Identifier" content="10.1038/ejhg.2017.82"><meta name="DC.Language" content="en"><meta property="og:title" content="Variants in TTC25 affect autistic trait in patients with autism spectrum disorder and general population"><meta property="og:type" content="article"><meta property="og:description" content="Autism spectrum disorder (ASD) is a highly heritable neurodevelopmental disorder with a complex genetic architecture. To identify genetic variants underlying ASD, we performed single-variant and gene-based genome-wide association studies using a dense ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5567150/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5567150/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5567150/pdf/ejhg201782a.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B8216DD5A39F100000000060403A9.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5567150%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/297374/">v.25(8); 2017 Aug</a></li><li class="accid">PMC5567150</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2017 Aug; 25(8): 982–987. </span></div><div><span class="fm-vol-iss-date">Published online 2017 May 17. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fejhg.2017.82" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ejhg.2017.82</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5567150</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28513607">28513607</a></div></div></div></div><h1 class="content-title">Variants in <em>TTC25</em> affect autistic trait in patients with autism spectrum disorder and general population</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Vojinovic%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607" class="affpopup" co-rid="_co_idm139681654568944" co-class="co-affbox">Dina Vojinovic</a>,<sup>1</sup> <a href="/pubmed/?term=Brison%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607" class="affpopup" co-rid="_co_idm139681618823584" co-class="co-affbox">Nathalie Brison</a>,<sup>2</sup> <a href="/pubmed/?term=Ahmad%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607" class="affpopup" co-rid="_co_idm139681654565568" co-class="co-affbox">Shahzad Ahmad</a>,<sup>1</sup> <a href="/pubmed/?term=Noens%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607" class="affpopup" co-rid="_co_idm139681577594560" co-class="co-affbox">Ilse Noens</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Pappa%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607" class="affpopup" co-rid="_co_idm139681577591648" co-class="co-affbox">Irene Pappa</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Karssen%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607" class="affpopup" co-rid="_co_idm139681577588640" co-class="co-affbox">Lennart C Karssen</a>,<sup>1,</sup><sup>7</sup> <a href="/pubmed/?term=Tiemeier%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607" class="affpopup" co-rid="_co_idm139681577585040" co-class="co-affbox">Henning Tiemeier</a>,<sup>1,</sup><sup>8</sup> <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607" class="affpopup" co-rid="_co_idm139681577582128" co-class="co-affbox">Cornelia M van Duijn</a>,<sup>1,</sup><sup>9</sup> <a href="/pubmed/?term=Peeters%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607" class="affpopup" co-rid="_co_idm139681577579264" co-class="co-affbox">Hilde Peeters</a>,<sup>2,</sup><sup>3,</sup><sup>10</sup> and  <a href="/pubmed/?term=Amin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607" class="affpopup" co-rid="_co_idm139681577575488" co-class="co-affbox">Najaf Amin</a><sup>1,</sup><sup>10,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139681654568944"><h3 class="no_margin">Dina Vojinovic</h3><p><sup>1</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Vojinovic%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607">Dina Vojinovic</a></div></div><div id="_co_idm139681618823584"><h3 class="no_margin">Nathalie Brison</h3><p><sup>2</sup>Center for Human Genetics, University Hospitals Leuven, KU Leuven, Leuven, Belgium</p><div>Find articles by <a href="/pubmed/?term=Brison%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607">Nathalie Brison</a></div></div><div id="_co_idm139681654565568"><h3 class="no_margin">Shahzad Ahmad</h3><p><sup>1</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Ahmad%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607">Shahzad Ahmad</a></div></div><div id="_co_idm139681577594560"><h3 class="no_margin">Ilse Noens</h3><p><sup>3</sup>Leuven Autism Research (LAuRes), Leuven, Belgium</p><p><sup>4</sup>Parenting and Special Education Research Unit, KU Leuven, Leuven, Belgium</p><div>Find articles by <a href="/pubmed/?term=Noens%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607">Ilse Noens</a></div></div><div id="_co_idm139681577591648"><h3 class="no_margin">Irene Pappa</h3><p><sup>5</sup>School of Pedagogical and Educational Sciences, Erasmus University Rotterdam, Rotterdam, The Netherlands</p><p><sup>6</sup>Generation R Study Group, Erasmus University Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Pappa%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607">Irene Pappa</a></div></div><div id="_co_idm139681577588640"><h3 class="no_margin">Lennart C Karssen</h3><p><sup>1</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><p><sup>7</sup>PolyOmica, s-Hertogenbosch, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Karssen%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607">Lennart C Karssen</a></div></div><div id="_co_idm139681577585040"><h3 class="no_margin">Henning Tiemeier</h3><p><sup>1</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><p><sup>8</sup>Department of Child and Adolescent Psychiatry/Psychology, Erasmus University Medical Center-Sophia Children’s Hospital, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Tiemeier%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607">Henning Tiemeier</a></div></div><div id="_co_idm139681577582128"><h3 class="no_margin">Cornelia M van Duijn</h3><p><sup>1</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><p><sup>9</sup>Leiden Academic Centre for Drug Research (LACDR), Leiden University, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607">Cornelia M van Duijn</a></div></div><div id="_co_idm139681577579264"><h3 class="no_margin">Hilde Peeters</h3><p><sup>2</sup>Center for Human Genetics, University Hospitals Leuven, KU Leuven, Leuven, Belgium</p><p><sup>3</sup>Leuven Autism Research (LAuRes), Leuven, Belgium</p><div>Find articles by <a href="/pubmed/?term=Peeters%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607">Hilde Peeters</a></div></div><div id="_co_idm139681577575488"><h3 class="no_margin">Najaf Amin</h3><p><sup>1</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Amin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28513607">Najaf Amin</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139681618142336_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139681618142336_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139681618142336_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139681618142336_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Center for Human Genetics, University Hospitals Leuven, KU Leuven, Leuven, Belgium</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Leuven Autism Research (LAuRes), Leuven, Belgium</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Parenting and Special Education Research Unit, KU Leuven, Leuven, Belgium</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>School of Pedagogical and Educational Sciences, Erasmus University Rotterdam, Rotterdam, The Netherlands</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Generation R Study Group, Erasmus University Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>PolyOmica, s-Hertogenbosch, The Netherlands</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Department of Child and Adolescent Psychiatry/Psychology, Erasmus University Medical Center-Sophia Children’s Hospital, The Netherlands</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Leiden Academic Centre for Drug Research (LACDR), Leiden University, The Netherlands</div><div id="caf1"><sup>*</sup>Department of Epidemiology, Erasmus University Medical Center, PO Box 2040, Rotterdam 3000 CA, The Netherlands. Tel +31 10 7038125; Fax: +31 10 704 46 57; E-mail: <a href="mailto:dev@null" data-email="ln.cmsumsare@nima.n" class="oemail">ln.cmsumsare@nima.n</a></div><div id="note1"><sup>10</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139681618142336_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 Aug 29; Revised 2017 Apr 3; Accepted 2017 Apr 13.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139681618142336_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2017 Macmillan Publishers Limited, part of Springer Nature.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden"></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC5567150/bin/ejhg201782x1.docx" data-ga-action="click_feat_suppl">ejhg201782x1.docx</a><span style="color:gray"> (567K)</span></div><div class="small guid">GUID:&nbsp;69811A59-AC20-4D93-83E8-A4C164F228FA</div></div></dd></dl></div><div id="idm139681583964352" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681583964352title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Autism spectrum disorder (ASD) is a highly heritable neurodevelopmental disorder with a complex genetic architecture. To identify genetic variants underlying ASD, we performed single-variant and gene-based genome-wide association studies using a dense genotyping array containing over 2.3 million single-nucleotide variants in a discovery sample of 160 families with at least one child affected with non-syndromic ASD using a binary (ASD yes/no) phenotype and a quantitative autistic trait. Replication of the top findings was performed in Psychiatric Genomics Consortium and Erasmus Rucphen Family (ERF) cohort study. Significant association of quantitative autistic trait was observed with the <em>TTC25</em> gene at 17q21.2 (effect size=10.2, <em>P</em>-value=3.4 × 10<sup>−7</sup>) in the gene-based analysis. The gene also showed nominally significant association in the cohort-based ERF study (effect=1.75, <em>P</em>-value=0.05). Meta-analysis of discovery and replication improved the association signal (<em>P</em>-value<sub>meta</sub>=1.5 × 10<sup>−8</sup>). No genome-wide significant signal was observed in the single-variant analysis of either the binary ASD phenotype or the quantitative autistic trait. Our study has identified a novel gene <em>TTC25</em> to be associated with quantitative autistic trait in patients with ASD. The replication of association in a cohort-based study and the effect estimate suggest that variants in <em>TTC25</em> may also be relevant for broader ASD phenotype in the general population. <em>TTC25</em> is overexpressed in frontal cortex and testis and is known to be involved in cilium movement and thus an interesting candidate gene for autistic trait.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p3" class="p p-first">Autism spectrum disorder (ASD) is a neurodevelopmental disorder characterized by deficits in social communication and social interaction and restricted and repetitive patterns of activities and behaviour with an onset in early development.<sup><a href="#bib1" rid="bib1" class=" bibr popnode">1</a></sup> However ASD is a psychiatric diagnosis based on clinical criteria, and the severity of these characteristics can be measured as quantitative traits that represent a continuum that extends into the general population, with ASD at the extreme end of the distribution.<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966403">2</a></sup> Other associated, but not core features are intellectual disability, attention-deficit disorder and medical comorbidities.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966438">3</a></sup> The prevalence of ASD is estimated to be 62/10 000<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966402">4</a></sup> with boys-to-girls ratio of ~4:1.<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966404">5</a></sup> The importance of a genetic aetiology is established with heritability estimates ranging from 37 to 90%.<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966454">6</a>, <a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966433">7</a>, <a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966410">8</a>, <a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966474">9</a></sup> Despite genetic heterogeneity, considerable progress in understanding the genetic architecture of ASD has been made by identifying monogenetic causes through genetic syndromes,<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966407">10</a></sup> rare chromosomal abnormalities,<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966467">11</a>, <a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966429">12</a></sup> rare copy-number variants<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966439">13</a>, <a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966442">14</a>, <a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966397">15</a>, <a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966434">16</a></sup> and rare penetrant gene mutations.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966472">3</a></sup> Several genomic regions, including 2q, 3q25–27, 3p25, 6q14–21, 7q31–36 and 17q11–21<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966462">17</a></sup> have been linked to ASD. The role of rare genetic variants in the aetiology of ASD has been established by high-throughput technologies.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966453">18</a>, <a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966427">19</a></sup> More recently, the theory of excess of <em>de novo</em> loss-of-function variants in ASD patients has gained popularity after some initial successes.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966464">18</a>, <a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966452">19</a>, <a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966421">20</a></sup> Around 1000 genes have been identified to be enriched with <em>de novo</em> loss-of-function mutations in ASD patients.<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966430">21</a></sup> However, <em>de novo</em> genetic variants do not contribute to the estimated heritability as these are not inherited. On the other hand, most genetic variance in ASD is attributed to common genetic variants.<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966411">9</a>, <a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966456">22</a></sup> Their role has been demonstrated by several genome-wide association studies (GWAS)<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966450">23</a>, <a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966455">24</a>, <a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966425">25</a>, <a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966393">26</a>, <a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966389">27</a>, <a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966426">28</a>, <a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966415">29</a></sup> (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S1</a>). Even though not many common susceptibility variants have been identified, significant association has been reported at 5p14.1,<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966475">23</a></sup> at 5p15.31 between <em>SEMA5A</em> and <em>TAS2R1</em> genes,<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966459">28</a></sup> within <em>MACROD2</em> at 20p12.1<sup><a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966408">26</a></sup> and at 1p13.2.<sup><a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966418">29</a></sup> However, there is a significant overlap of the discovery samples used and little replication of specific loci between studies.<sup><a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966417">30</a></sup></p><p id="__p4" class="p p-last">Although the individual effect of common variants is modest, their joint effect may be substantial.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966405">25</a></sup> In this study besides assessing the effect of single variants on ASD, we evaluated the joint effect of multiple single variants in a gene in a genome-wide gene-based association analysis in patients with ASD from a Belgian Flemish cohort who were genotyped on a dense genotyping array.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Materials and methods</h2><p id="__p5" class="p p-first">General overview of the study design and work flow are illustrated in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1</a>.</p><div id="__sec3" class="sec"><h3 id="__sec3title">Discovery sample</h3><p id="__p6" class="p p-first">The discovery sample consisted of 160 nuclear Belgian Flemish families (657 individuals; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>). The families were recruited to participate in the prospective study through the Expert Center for Autism (ECA) Leuven. All probands had been seen multiple times as part of their clinical care program in the ECA before recruitment. The families were asked to participate if there was at least one child with the diagnosis of non-syndromic ASD of unknown origin after a clinical genetics workup. Out of the 160 families, 55 were multiplex (two or more siblings with ASD) and 105 simplex. In six families the father had also been diagnosed with ASD such that there were 77.7% affected males and 22.3% affected females with male-to-female ratio of 3.5:1. Among them, 88.4% had normal and high intelligence, whereas 11.6% had mild, moderate or severe intellectual disability.</p><p id="__p7" class="p p-last">Diagnoses of ASD were made by a multidisciplinary team in the ECA Leuven according to DSM-IV-TR (American Psychiatric Association, 2000) criteria. Additionally, participants were assessed for quantitative autistic trait using the Dutch version of the Social Responsiveness Scale (SRS) and the Social Responsiveness Scale for Adults (SRS-A) designed to measure social impairment associated with ASD across a wide range of severity.<sup><a href="#bib31" rid="bib31" class=" bibr popnode">31</a>, <a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> Completed questionnaires were obtained for 490 probands, parents and siblings. Among the affected patients that had the SRS score available, the majority had normal and high intelligence (86%). For all participants, we received written informed consent. This study was approved by the Medical Ethical Committee of the University Hospitals Leuven.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Genotyping</h3><p id="__p8" class="p p-first-last">Genotyping of 657 individuals from the discovery cohort was performed at the Center for Human Genetics at the KU Leuven, Belgium using the HumanOmni2.5-8 BeadChip, which contains more than 2.3 million common and less-frequent single-nucleotide polymorphisms (SNPs) from the 1000 Genome Project (minor allele frequency &gt;2.5%). SNP calling was performed in Genome Studio 2011.1 using the genotyping module v1.9. Markers with call rate &lt;95%, or which were monomorphic or which failed an exact test of Hardy–Weinberg equilibrium (HWE) (<em>P</em>-value&lt;1 × 10<sup>−7</sup>) were removed from the analysis. Samples with low call rate &lt;95% or high identity-by-state (≥95%) were also removed from the analysis. Ethnic outliers were determined using multidimensional scaling analysis with 1000 Genomes data set (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2</a>). All samples clustered tightly with the Europeans and no ethnic outlier was identified. In total 1 646 898 markers and 654 genotyped individuals were retained for further statistical analysis. Lastly, Mega2 tool v4.4<sup>(ref. <a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966392">33</a>)</sup> was used to identify Mendelian inconsistencies, which were later set to missing.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Statistical analysis</h3><p id="__p9" class="p p-first">Baseline descriptive analysis was performed with SPSS v21 (IBM Corporation, Armonk, NY, USA) and PEDSTATS v0.6.12.<sup><a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966468">34</a></sup> Genome-wide association analyses of the binary ASD phenotype were performed through joint modelling of linkage and association, using the LAMP software v0.0.9 (School of Public Health, Ann Arbor, MI, USA). LAMP uses a maximum likelihood model to extract information on genetic linkage and association from samples of unrelated individuals, sib pairs, trios and larger pedigrees in settings where population stratification is not a concern (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2</a>).<sup><a href="#bib35" rid="bib35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966399">35</a></sup> Odds ratios and 95% confidence intervals were estimated using PLINK v1.07.<sup><a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966398">36</a></sup> The association tests for markers on sex chromosomes were performed by transmission disequilibrium test for chromosome X and by logistic regression for chromosome Y. Single-variant and gene-based genome-wide association analyses of the quantitative autistic trait adjusted for age, gender and familial relationships were performed using the RVtests software tool version 20150630 (<a href="http://zhanxw.github.io/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://zhanxw.github.io/</a> rvtests/). The gene-based analysis included Combined Multivariate and Collapsing method which is robust and powerful in the presence of wide spectrum of variant allele frequencies.<sup><a href="#bib37" rid="bib37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966388">37</a></sup> The genes were defined according to human reference genome hg19. All association analyses were performed for entire discovery sample, and simplex and multiplex families separately. The standard genome-wide significance threshold of 5 × 10<sup>−8</sup> was used to declare significance in the single-variant analyses, while the genome-wide significance threshold for the gene-based analysis was set at 2.5 × 10<sup>−6</sup> based on 19 650 genes tested. PLINK/SEQ v0.10 (<a href="https://atgu.mgh.harvard.edu/plinkseq/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://atgu.mgh.harvard.edu/plinkseq/</a>) was used to convert PLINK files into variant call format files. All genome maps were updated to human genome build 19 (hg19). Gene pathway enrichment analysis of all nominally significant genes (<em>P</em>-value&lt;0.01) in the gene-based analysis was performed using the web-based gene network pathway enrichment tool (<a href="http://129.125.135.180:8080/GeneNetwork/pathway.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://129.125.135.180:8080/GeneNetwork/pathway.html</a>).</p><p id="__p10" class="p p-last">The data were deposited in the GWAS Central database (<a href="http://www.gwascentral.org/study/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gwascentral.org/study/HGVST1847</a>).</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Bioinformatic analysis</h3><p id="__p11" class="p p-first-last">To annotate SNPs with regulatory information, we used RegulomeDB v1.1 database (<a href="http://www.regulomedb.org/index" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.regulomedb.org/index</a>) that combines information from ENCODE and other sources, as well as computational predictions and manual annotations into a tool that classifies SNPs into six categories, where Category 1 variants are 'likely to affect binding and linked to expression of a gene target', whereas category 6 variants have 'minimal binding evidence'.<sup><a href="#bib38" rid="bib38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966431">38</a></sup> Furthermore, regulatory information on SNPs in haplotype blocks was explored using a HaploReg v4.1 tool.<sup><a href="#bib39" rid="bib39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966469">39</a></sup> For these analysis <em>r</em><sup>2</sup> was set to 1 and the population of European descent was chosen.</p></div><div id="__sec7" class="sec sec-last"><h3 id="__sec7title">Replication samples</h3><div id="__sec8" class="sec sec-first"><p></p><h4 id="__sec8title" class="inline">Psychiatric Genomics Consortium (PGC) </h4><p id="__p12" class="p p-first-last">A lookup of top findings from the single-variant analyses of the binary ASD phenotype and quantitative autistic trait was performed in the latest PGC GWAS. This data set consists of a total of 6495 parent-child trios who met diagnostic criteria for ASD and had genome-wide SNP data available (<a href="https://www.med.unc.edu/pgc" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.med.unc.edu/pgc</a>).</p></div><div id="__sec9" class="sec sec-last"><p></p><h4 id="__sec9title" class="inline">Erasmus Rucphen Family (ERF) study </h4><p id="__p13" class="p p-first">Replication of the gene-based analysis of quantitative autistic trait was performed in the ERF study as 1250 participants from this cohort have been assessed for quantitative autistic trait using Baron-Cohen’s Autism-Spectrum Quotient (AQ) test<sup><a href="#bib40" rid="bib40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966420">40</a></sup> and exomes of half of these participants (<em>n</em>=615) have been sequenced, thus providing a greater resolution at the gene level. Individuals whose exome were sequenced were selected based on having good quality phenotype information on a wide range of topics, and therefore random with regards to AQ scores (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>). ERF is a family-based cohort originating from 22 couples and spread over 23 generations.<sup><a href="#bib41" rid="bib41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966441">41</a>, <a href="#bib42" rid="bib42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966436">42</a></sup> The ERF study was approved by the Medical Ethics Committee of the Erasmus MC which is constituted according to the WMO (Wet Medisch-wetenschappelijk Onderzoek met mensen). A written informed consent was obtained from all study participants.</p><p id="__p14">Sequencing was done at a mean depth of 74 × using the Nimblegen SeqCap EZ V2 capture kit on an Illumina Hiseq2000 sequencer (Illumina, San Diego, CA, USA) using the TruSeq Version 3 protocol at the Human Genotyping facility of the Internal Medicine department, at the Erasmus MC, The Netherlands.<sup><a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966443">43</a>, <a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966457">44</a></sup> The sequence reads were aligned to the human genome build 19 (hg19), using Burrows-Wheeler Aligner and the NARWHAL pipeline.<sup><a href="#bib45" rid="bib45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966400">45</a>, <a href="#bib46" rid="bib46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966446">46</a></sup> After processing, genetic variants were called, using the Unified Genotyper tool from the GATK.<sup><a href="#bib47" rid="bib47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966414">47</a></sup> Variants with a low quality (QUAL&lt;150), which were out of HWE (<em>P</em>-value&lt;10<sup>−6</sup>) or with low call rate (&lt; 90%), as well as samples with a low call rate (&lt; 90%), and duplicates, were removed.<sup><a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966406">44</a></sup> Functional annotations were also performed using the SeattleSeq annotation 138 database (<a href="http://snp.gs.washington.edu/SeattleSeq" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://snp.gs.washington.edu/SeattleSeqAnnotation138/</a>).</p><p id="__p15" class="p p-last">Association analyses of the quantitative autistic trait adjusted for age, gender and familial relationship were performed using the RVtests software. Meta-analysis of gene-based results of discovery sample and ERF study was performed using Fisher’s combined probability test.</p></div></div></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">Results</h2><p id="__p16" class="p p-first">Results of genome-wide association analysis for the binary ASD phenotype and quantitative autistic trait are illustrated in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figures S3 and S4</a>. No single-variant surpassed the genome-wide significance threshold. Top findings from the association analyses are shown in <a href="/pmc/articles/PMC5567150/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Tables 1</span></a> and <a href="/pmc/articles/PMC5567150/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2</span></span><span>2</span></a> for the discovery sample, and in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S3 and S4</a> for simplex and multiplex families separately. Suggestive association of binary ASD phenotype was observed with two common variants (rs6452310; <em>P</em>-value=7.80 × 10<sup>−8</sup> and rs7700465; <em>P</em>-value=8.70 × 10<sup>−6</sup>; <a href="/pmc/articles/PMC5567150/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>) at chromosome 5p14.1—a region previously known to be associated with ASD (<a href="/pmc/articles/PMC5567150/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>). The two variants were in strong linkage disequilibrium (LD) (<em>r</em><sup><em>2</em></sup>=0.85) with each other but not in LD with any of the previously identified variants in this region associated with ASD<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966470">23</a>, <a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966458">24</a></sup> (<em>r</em><sup><em>2</em></sup> ranged from 0.002 to 0.009, <em>D’</em> ranged from 0.02 to 0.33). None of the top variants from this analysis showed evidence of association in the replication sample (<a href="/pmc/articles/PMC5567150/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC5567150/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681583300288" class="figure"><a href="/pmc/articles/PMC5567150/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5567150_ejhg201782f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg201782f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5567150/bin/ejhg201782f1.jpg"></a></div><div id="largeobj_idm139681583300288" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5567150/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC5567150/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p29">Regional association and recombination rate plot of the 5p14.1 region and the binary ASD phenotype in the discovery cohort. The left <em>y</em> axis represents –log10 <em>P</em>-values for association with binary ASD phenotype in the discovery cohort. The right <em>y</em> axis represents the recombination rate, and the <em>x</em> axis represents chromosomal position (genomic position is according to the hg19 assembly). The most significantly associated single SNP in this region (rs6452310) is denoted with a purple diamond. Surrounding SNPs are shaded according to their pairwise correlation (<em>r2</em>) with rs6452310. The gene annotations are shown below the figure.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Top findings (<em>P</em>-value&lt;1 × 10<sup>−5</sup>) from the binary ASD phenotype association analysis</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681583296944" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th colspan="6" charoff="50" rowspan="1" valign="top" align="center"><em>Discovery cohort</em></th><th colspan="2" charoff="50" rowspan="1" valign="top" align="center"><em>PGC</em></th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>SNP</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Chr</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Position</em><sup><a href="#t1-fn2" rid="t1-fn2" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Function</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>CADD</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Gene</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Reference allele</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Variant allele</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">N</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Call rate</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MAF</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>HWE</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>LOD</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">P</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>OR</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">P</th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6452310</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">25996104</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.58</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MSNP1AS</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">654</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.32</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.27</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7.80 × 10<sup>−8</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.996</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.9</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs72996425</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">59451206</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5.32</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>RFN125/CHD20</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">654</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.37</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7.20 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.96</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.7</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7147213</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">52932731</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Intron</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>TXNDC16</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">653</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.998</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.32</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8.20 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.34</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7700465</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">26003284</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MSNP1AS</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">654</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.28</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.89</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.29</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8.70 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.99</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.69</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7142973</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">52926750</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Intron</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.02</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>TXNDC16</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">654</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.26</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.50 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.04</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.36</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs4901281</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">52980190</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Intron</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7.66</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>TXNDC16</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">654</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.26</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.50 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.97</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.39</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs12435289</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">52981240</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Intron</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.54</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>TXNDC16</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">654</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.26</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.50 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.97</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.37</td></tr></tbody></table></div><div id="largeobj_idm139681583296944" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5567150/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1-fn1"><p id="__p30" class="p p-first-last">Abbreviations: ASD, autism spectrum disorder; Chr, chromosome; CADD, phred-like Combined Annotation Dependent Depletion scores from Kircher <em>et al</em>,<sup><a href="#bib70" rid="bib70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966412">70</a></sup> University of Washington; HWE, Hardy–Weinberg equilibrium in founders; MAF, minor allele frequency; <em>N</em>, number of individuals; OR, odds ratio; PGC, Psychiatric Genomics Consortium.</p></div><div id="t1-fn2"><sup>a</sup>Genomic position is according the hg19 assembly.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Top findings (<em>P</em>-value&lt;1 × 10<sup>−5</sup>) from the quantitative autistic trait association analysis</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681577666112" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th colspan="10" charoff="50" rowspan="1" valign="top" align="center"><em>Discovery cohort</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>PGC</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>SNP</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Chr</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Position</em><sup><a href="#t2-fn3" rid="t2-fn3" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Function</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>CADD</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Gene</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Reference allele</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Variant Allele</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">N</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>AF</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Call Rate</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>HWE</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>U_Stat</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Sqrt_V_Stat</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Effect</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">P</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>OR</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">P</th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs77782810</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">93169444</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Downstream</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.81</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">None</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.09</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.99</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.35</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.42</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">12.13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.28 × 10<sup>−7</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.54</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs75822891</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">551394</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">3-prime-UTR</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7.37</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>CCDC77</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.02</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.23</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">21.81</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8.78 × 10<sup>−7</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.89</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs61867303</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">10</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">106594496</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.92</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>SORCS3</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.04</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.44</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.30</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">16.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.19 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.70</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs202183827</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">93162861</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">None</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.10</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.99</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.45</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">10.25</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.30 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">—</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs75820392</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">526569</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.64</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>CCDC77</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.02</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.99</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.98</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">21.86</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.61 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.08</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.44</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs76459666</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">93161327</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.74</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">None</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.99</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.46</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">10.11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.88 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.01</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.85</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs12433198</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">78042307</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.51</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>SPTLC2</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.08</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.52</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.81</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.40</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">11.22</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.49 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.01</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs115601680</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">46378057</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intron</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>RCAN2</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.01</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.60</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">32.90</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8.99 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">—</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs77988177</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">20</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">21972232</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.47</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">None</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.27</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.06</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.24</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">18.52</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.18 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.94</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.52</td></tr></tbody></table></div><div id="largeobj_idm139681577666112" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5567150/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t2-fn1"><p id="__p32" class="p p-first-last">Abbreviations: AF, allele frequency; Chr, chromosome; CADD, phred-like Combined Annotation Dependent Depletion scores from Kircher <em>et al</em>,<sup><a href="#bib70" rid="bib70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966449">70</a></sup> University of Washington; Effect, effect of variant allele; <em>N</em>, number of individuals; HWE, Hardy–Weinberg equilibrium in founders; OR, odds ratio; PGC, Psychiatric Genomics Consortium; SNP, single-nucleotide polymorphism.</p></div><div id="t2-fn2"><p id="__p33" class="p p-first-last">U and V statistics are score statistics and their covariance matrix.</p></div><div id="t2-fn3"><sup>a</sup>Genomic position is according the hg19 assembly.</div></div></div><p id="__p17" class="p">Results of gene-based genome-wide association analysis are illustrated in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S5</a> for the discovery sample, and in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S5</a> for simplex and multiplex families separately. The gene-based association analysis revealed significant association of quantitative autistic trait and <em>TTC25</em> gene (<em>P</em>-value=3.4 × 10<sup>−7</sup>) on chromosome 17 (<a href="/pmc/articles/PMC5567150/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>). This association was not driven by any single variant but by nine variants, four of which showed nominally significant association with quantitative autistic trait (<em>P</em>-value&lt;0.05) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S6</a>). The combined effect of these variants on the SRS score was large (effect size=10.2). The functional annotation of nine variants revealed that five variants are likely to have regulatory functions (Category 1 RegulomeDB score; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S6</a>). Furthermore, the surrounding variants in strong LD (<em>r</em><sup>2</sup>=1) with nine variants lie in enhancer histone marks, and protein-binding regions and change regulatory motifs based on the variant allele changes (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S7</a>). The gene was also nominally associated with autistic trait in the ERF cohort (<em>P</em>-value=0.045). The combined effect of 15 variants in the <em>TTC25</em> gene (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S8</a>) in the replication sample was much smaller (effect size=1.75). Meta-analysis of discovery and replication samples resulted in an improved association signal (<em>P</em>-value=1.5 × 10<sup>−8</sup>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Top results (<em>P</em>-value&lt;1 × 10<sup>−3</sup>) from the gene-based association analysis of quantitative autistic trait in the discovery cohort</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681583049696" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Gene</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Chr</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">N</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>NumVar</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>NumSite</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>AF</em><sup><a href="#t3-fn2" rid="t3-fn2" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Effect</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">P</th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>TTC25</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">17</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.338</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">10.23</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.44 × 10<sup>−7</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>SH2D5</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.440</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−10.88</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.49 × 10<sup>−4</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>PSMC5</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">17</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.458</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−11.13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.81 × 10<sup>−4</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>LOC729177</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">486</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">22</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.434</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−9.15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.37 × 10<sup>−4</sup></td></tr></tbody></table></div><div id="largeobj_idm139681583049696" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5567150/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t3-fn1"><p id="__p35" class="p p-first-last">Abbreviations: AF, allele frequency; Chr, chromosome; Effect, effect size; <em>N</em>, sample size; NumVar, number of variants; NumSite, number of sites.</p></div><div id="t3-fn2"><sup>a</sup>Allele frequency is calculated as an average of individual allele frequency adjusted for relatedness (<a href="http://zhanxw.github.io/rvtests/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://zhanxw.github.io/rvtests/</a>).</div></div></div><div id="__sec11" class="sec sec-last"><h3 id="__sec11title">Pathway analysis</h3><p id="__p18" class="p p-first-last">Pathway enrichment analysis based on nominally significant genes in the gene-based association analysis (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S9</a>) showed significantly enriched SMAD protein signal transduction (<em>P</em>-value=2 × 10<sup>−5</sup>) pathway and series of digestive system development enrichment categories (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S10</a>).</p></div></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Discussion</h2><p id="__p19" class="p p-first">In this study, we have identified a novel gene <em>TTC25</em> associated with quantitative autistic trait. The association of the gene <em>TTC25</em> in a cohort-based study suggests that the gene may be relevant for broader ASD phenotype in the general population. Further, we identified SMAD protein signal transduction pathway and series of digestive system development categories as being significantly enriched with genes nominally associated with quantitative autistic trait. Moreover, our study provides additional evidence for the previously identified association of the intergenic loci at 5p14.1 with the binary ASD phenotype.</p><p id="__p20"><em>TTC25</em> gene is located on chromosome 17q21.2 and encodes Tetratricopeptide Repeat Domain 25. 17q21.2 locus has previously been linked to ASD in a genome-wide linkage scan,<sup><a href="#bib48" rid="bib48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966432">48</a>, <a href="#bib49" rid="bib49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966471">49</a></sup> although a gene was never implicated. <em>TTC25</em> is overexpressed in testis, frontal cortex and rectum (<a href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=TTC25" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genecards.org/cgi-bin/carddisp.pl?gene=TTC25</a>). <em>TTC25</em> is involved in cilium movement, organization and morphogenesis.<sup><a href="#bib50" rid="bib50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966413">50</a></sup> Cilia are specialized organelles protruding from the cell surface of almost all mammalian cells.<sup><a href="#bib51" rid="bib51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966444">51</a></sup> Mutations in ciliary proteins cause ciliopathies which can affect many organs at different levels of severity and are characterized by a wide spectrum of phenotypes.<sup><a href="#bib51" rid="bib51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966448">51</a></sup> In the vertebrate nervous system, the primary cilium is increasingly viewed as hub for certain neural developmental signalling pathways, and growing data suggest this is also true for several types of adult neuronal signalling.<sup><a href="#bib52" rid="bib52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966451">52</a></sup> The capacity of the brain to interpret the sensory input is often affected in ciliopathies, resulting in neurological disorders; cognitive impairment, anosmia, intellectual disability, ASD and obesity are apparent in various degrees in many of the ciliopathies.<sup><a href="#bib51" rid="bib51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966401">51</a>, <a href="#bib53" rid="bib53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966465">53</a></sup> Further Joubert syndrome (JS) is a well-known ciliopathy of the central nervous system.<sup><a href="#bib52" rid="bib52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966391">52</a>, <a href="#bib54" rid="bib54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966390">54</a></sup> Features of ASD, such as problems in social behaviour, communication problems and repetitive behaviours, have been described in up to 40% of JS patients<sup><a href="#bib55" rid="bib55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966440">55</a>, <a href="#bib56" rid="bib56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966461">56</a>, <a href="#bib57" rid="bib57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966424">57</a></sup> and about 25% of JS patients meet criteria for the DSM-IV diagnosis of ASD.<sup><a href="#bib55" rid="bib55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966466">55</a>, <a href="#bib58" rid="bib58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966473">58</a></sup> Multiple variants mapped to this gene in our sample appear to have a regulatory function.</p><p id="__p21">Our identification of SMAD protein signal transduction pathway as being significantly enriched with genes nominally associated with quantitative autistic trait reinforces the role of the transforming growth factor-β (TGFβ) in ASD. The Smad pathways are the major mediators of transcriptional responses induced by the TGFβ family, which control cell-fate determination, cell cycle arrest, apoptosis and actin rearrangements.<sup><a href="#bib59" rid="bib59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966463">59</a></sup> While decreased levels of TGFβ have been reported in blood samples from individuals with ASD<sup><a href="#bib60" rid="bib60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966447">60</a></sup> and associated with more severe behavioural scores in ASD children,<sup><a href="#bib61" rid="bib61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966396">61</a></sup> higher levels of TGFβ have been reported in postmortem brain and cerebrospinal fluid samples of ASD patients.<sup><a href="#bib62" rid="bib62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966416">62</a></sup> In addition, series of digestive system development categories were enriched with genes nominally associated with ASD. Gastrointestinal (GI) disturbances are 4-fold more common in ASD<sup><a href="#bib63" rid="bib63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966395">63</a></sup> and available scientific evidence supports combination of changes in the areas of immune function, gut microbiome and gut and brain signalling pathways.<sup><a href="#bib64" rid="bib64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966435">64</a></sup> Recent studies in animal models suggests that GI difficulties may originate from the same genetic changes that lead to the behavioural characteristics of ASD.<sup><a href="#bib65" rid="bib65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966419">65</a></sup></p><p id="__p22">In addition, we found the association of ASD with the known region on 5p14.1. This is one of the few replicated GWA regions that implicates long noncoding RNA gene, <em>MSNP1AS</em> (moesin pseudogene 1, antisense) in ASD risk.<sup><a href="#bib66" rid="bib66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966460">66</a></sup> This region has also been associated with social communication spectrum phenotypes in the general population supporting the role of 5p14.1 as a quantitative trait locus for ASD.<sup><a href="#bib49" rid="bib49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966437">49</a></sup>
<em>MSNP1AS</em> shows a very high sequence homology to the chromosome X transcript of <em>MSN</em> that is involved in brain development.<sup><a href="#bib66" rid="bib66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966394">66</a></sup>
<em>MSNP1AS</em> is highly overexpressed (12.7-fold) in the postmortem cerebral cortex of individuals with ASD.<sup><a href="#bib66" rid="bib66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966423">66</a></sup> Interestingly, our top hit did not replicate in the PGC sample and vice versa. The multiple different variants discovered in the 5p14.1 region<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966387">23</a>, <a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966428">24</a></sup> may suggest that multiple alleles in the same region are implicated in ASD.</p><p id="__p23">Although our study sample was small, the strength of our study is that we used a data set with high-quality phenotypes, in which participants were assessed for both binary ASD phenotype and a quantitative autistic trait. The use of quantitative endophenotype provides additional power to find genetic signals by focusing on less complex aspects of complex phenotypes such as ASD.<sup><a href="#bib67" rid="bib67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966422">67</a></sup> The identification of new loci associated with quantitative autistic trait in our study validates this approach. Another strength is phenotypic homogeneity in the sense that the majority of patients in the current study had a normal intelligence, unlike most ASD cohorts with typical rates of intellectual disability ranging from 30 to 50%.<sup><a href="#bib68" rid="bib68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966409">68</a></sup> Further, the genomes of our study participants were genotyped on a very dense SNP array that contains not only common but also less-frequent SNPs. This gave us the opportunity to make a comprehensive overview of how common and less-frequent variants, both individually and taken together, affect ASD. Our study shows the advantage of a gene-based test as the more powerful approach compared to single-variant analysis and demonstrated the use of gene-based pathway and enrichment analysis in understanding the molecular mechanisms of the disorder. One of the possible limitations of our study is that the assessment of quantitative autistic trait in the discovery and replication cohort used two different questionnaires, the SRS and the AQ. As the questionnaires have both been designed to measure the severity of social responsiveness problems across clinical cases and the general population and as their ratings are significantly correlated,<sup><a href="#bib69" rid="bib69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_632966445">69</a></sup> we were able to compare results from the two cohorts.</p><p id="__p24" class="p p-last">To conclude, our study has identified a novel gene <em>TTC25</em> to be associated with autistic trait in the ASD population where majority of patients have a normal intelligence. The replication of <em>TTC25</em> association in a cohort-based study suggests that this gene may also be relevant for broader ASD phenotype in the general population. However, whether these findings hold true also for ASD patients with intellectual disability remains to be evaluated. <em>TTC25</em> is overexpressed in frontal cortex and testis and is known to be involved in cilium movement and thus an interesting candidate gene for autistic trait. Furthermore, we discovered significantly enriched SMAD protein signal transduction pathway and series of digestive system development categories in the pathway analysis of quantitative autistic trait. Our finding provides new insights into the genetic background of quantitative autistic trait.</p></div><div id="idm139681656319888" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681656319888title">Acknowledgments</h2><div class="sec"><p id="__p25">The Erasmus Rucphen Family study (ERF) study as a part of EUROSPAN (European Special Populations Research Network) was supported by European Commission FP6 STRP grant number 018947 (LSHG-CT-2006-01947) and also received funding from the European Community's Seventh Framework Programme (FP7/2007-2013)/grant agreement HEALTH-F4-2007-201413 by the European Commission under the programme 'Quality of Life and Management of the Living Resources' of 5th Framework Programme (no. QLG2-CT-2002-01254). High-throughput analysis of the ERF data was supported by joint grant from Netherlands Organization for Scientific Research and the Russian Foundation for Basic Research (NWO-RFBR 047.017.043). Exome sequencing analysis in ERF was supported by the ZonMw grant (project 91111025). We are grateful to all study participants and their relatives, general practitioners and neurologists for their contributions and to P Veraart for her help in genealogy, J Vergeer for the supervision of the laboratory work and P Snijders for his help in the data collection. NA is supported by The Netherlands Brain Foundation (project number F2013(1)-28). HP is a Senior Clinical Investigator of The Research Foundation—Flanders (FWO). We acknowledge PGC consortium for providing the pre-publication results.</p></div></div><div id="idm139681618140032" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681618140032title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm139681618888256"></p><p id="__p26" class="p p-first-last"><a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> accompanies this paper on European Journal of Human Genetics website (http://www.nature.com/ejhg)</p><p></p><p class="fn sec" id="idm139681656317296"></p><p id="__p27" class="p p-first-last">IN receives royalties from Hogrefe Uitgeverij for the SRS-A (although not for the questionnaires used in this study). The remaining authors declare no conflict of interest.</p><p></p></div></div><div id="sup1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sup1title">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="xob1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="idm139681583307344"><a href="/pmc/articles/PMC5567150/bin/ejhg201782x1.docx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(567K, docx)</sup></div></div></div><div id="idm139681656316464" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681656316464title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="bib1"><span class="mixed-citation">Association AP<span class="ref-journal">Diagnostic and Statistical Manual of Mental Disorders</span>. 5th edn Arlington, VA: American Psychiatric Publishing, 2013. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Diagnostic+and+Statistical+Manual+of+Mental+Disorders&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib2"><span class="mixed-citation">Constantino JN, Todd RD: <span class="ref-title">Autistic traits in the general population: a twin study</span>. <span class="ref-journal">Arch Gen Psychiatry</span>
2003; <span class="ref-vol">60</span>: 524–530. [<a href="/pubmed/12742874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Autistic+traits+in+the+general+population:+a+twin+study&amp;volume=60&amp;publication_year=2003&amp;pages=524&amp;pmid=12742874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib3"><span class="mixed-citation">Devlin B, Scherer SW: <span class="ref-title">Genetic architecture in autism spectrum disorder</span>. <span class="ref-journal">Curr Opin Genet Dev</span>
2012; <span class="ref-vol">22</span>: 229–237. [<a href="/pubmed/22463983" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;title=Genetic+architecture+in+autism+spectrum+disorder&amp;volume=22&amp;publication_year=2012&amp;pages=229&amp;pmid=22463983&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib4"><span class="mixed-citation">Elsabbagh M, Divan G, Koh YJ et al: <span class="ref-title">Global prevalence of autism and other pervasive developmental disorders</span>. <span class="ref-journal">Autism Res</span>
2012; <span class="ref-vol">5</span>: 160–179. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3763210/">PMC free article</a>]</span> [<a href="/pubmed/22495912" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autism+Res&amp;title=Global+prevalence+of+autism+and+other+pervasive+developmental+disorders&amp;volume=5&amp;publication_year=2012&amp;pages=160&amp;pmid=22495912&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib5"><span class="mixed-citation">Werling DM, Geschwind DH: <span class="ref-title">Sex differences in autism spectrum disorders</span>. <span class="ref-journal">Curr Opin Neurol</span>
2013; <span class="ref-vol">26</span>: 146–153. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4164392/">PMC free article</a>]</span> [<a href="/pubmed/23406909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Neurol&amp;title=Sex+differences+in+autism+spectrum+disorders&amp;volume=26&amp;publication_year=2013&amp;pages=146&amp;pmid=23406909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib6"><span class="mixed-citation">Hallmayer J, Cleveland S, Torres A et al: <span class="ref-title">Genetic heritability and shared environmental factors among twin pairs with autism</span>. <span class="ref-journal">Arch Gen Psychiatry</span>
2011; <span class="ref-vol">68</span>: 1095–1102. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4440679/">PMC free article</a>]</span> [<a href="/pubmed/21727249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Genetic+heritability+and+shared+environmental+factors+among+twin+pairs+with+autism&amp;volume=68&amp;publication_year=2011&amp;pages=1095&amp;pmid=21727249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib7"><span class="mixed-citation">Bailey A, Le Couteur A, Gottesman I et al: <span class="ref-title">Autism as a strongly genetic disorder: evidence from a British twin study</span>. <span class="ref-journal">Psychol Med</span>
1995; <span class="ref-vol">25</span>: 63–77. [<a href="/pubmed/7792363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychol+Med&amp;title=Autism+as+a+strongly+genetic+disorder:+evidence+from+a+British+twin+study&amp;volume=25&amp;publication_year=1995&amp;pages=63&amp;pmid=7792363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib8"><span class="mixed-citation">Sandin S, Lichtenstein P, Kuja-Halkola R, Larsson H, Hultman CM, Reichenberg A: <span class="ref-title">The familial risk of autism</span>. <span class="ref-journal">JAMA</span>
2014; <span class="ref-vol">311</span>: 1770–1777. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4381277/">PMC free article</a>]</span> [<a href="/pubmed/24794370" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=The+familial+risk+of+autism&amp;volume=311&amp;publication_year=2014&amp;pages=1770&amp;pmid=24794370&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib9"><span class="mixed-citation">Gauglerl T, Klei L, Sanders SJ et al: <span class="ref-title">Most genetic risk for autism resides with common variation</span>. <span class="ref-journal">Nat Genet</span>
2014; <span class="ref-vol">46</span>: 881–885. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4137411/">PMC free article</a>]</span> [<a href="/pubmed/25038753" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Most+genetic+risk+for+autism+resides+with+common+variation&amp;volume=46&amp;publication_year=2014&amp;pages=881&amp;pmid=25038753&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib10"><span class="mixed-citation">Betancur C: <span class="ref-title">Etiological heterogeneity in autism spectrum disorders: more than 100 genetic and genomic disorders and still counting</span>. <span class="ref-journal">Brain Res</span>
2011; <span class="ref-vol">1380</span>: 42–77. [<a href="/pubmed/21129364" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res&amp;title=Etiological+heterogeneity+in+autism+spectrum+disorders:+more+than+100+genetic+and+genomic+disorders+and+still+counting&amp;volume=1380&amp;publication_year=2011&amp;pages=42&amp;pmid=21129364&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib11"><span class="mixed-citation">Marshall CR, Noor A, Vincent JB et al: <span class="ref-title">Structural variation of chromosomes in autism spectrum disorder</span>. <span class="ref-journal">Am J Hum Genet</span>
2008; <span class="ref-vol">82</span>: 477–488. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2426913/">PMC free article</a>]</span> [<a href="/pubmed/18252227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Structural+variation+of+chromosomes+in+autism+spectrum+disorder&amp;volume=82&amp;publication_year=2008&amp;pages=477&amp;pmid=18252227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib12"><span class="mixed-citation">Shen Y, Dies KA, Holm IA et al: <span class="ref-title">Clinical genetic testing for patients with autism spectrum disorders</span>. <span class="ref-journal">Pediatrics</span>
2010; <span class="ref-vol">125</span>: e727–e735. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4247857/">PMC free article</a>]</span> [<a href="/pubmed/20231187" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Clinical+genetic+testing+for+patients+with+autism+spectrum+disorders&amp;volume=125&amp;publication_year=2010&amp;pages=e727&amp;pmid=20231187&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib13"><span class="mixed-citation">Pinto D, Pagnamenta AT, Klei L et al: <span class="ref-title">Functional impact of global rare copy number variation in autism spectrum disorders</span>. <span class="ref-journal">Nature</span>
2010; <span class="ref-vol">466</span>: 368–372. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3021798/">PMC free article</a>]</span> [<a href="/pubmed/20531469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Functional+impact+of+global+rare+copy+number+variation+in+autism+spectrum+disorders&amp;volume=466&amp;publication_year=2010&amp;pages=368&amp;pmid=20531469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib14"><span class="mixed-citation">Gai X, Xie HM, Perin JC et al: <span class="ref-title">Rare structural variation of synapse and neurotransmission genes in autism</span>. <span class="ref-journal">Mol Psychiatry</span>
2012; <span class="ref-vol">17</span>: 402–411. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3314176/">PMC free article</a>]</span> [<a href="/pubmed/21358714" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Rare+structural+variation+of+synapse+and+neurotransmission+genes+in+autism&amp;volume=17&amp;publication_year=2012&amp;pages=402&amp;pmid=21358714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib15"><span class="mixed-citation">Sebat J, Lakshmi B, Malhotra D et al: <span class="ref-title">Strong association of <em>de novo</em> copy number mutations with autism</span>. <span class="ref-journal">Science</span>
2007; <span class="ref-vol">316</span>: 445–449. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2993504/">PMC free article</a>]</span> [<a href="/pubmed/17363630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Strong+association+of+de+novo+copy+number+mutations+with+autism&amp;volume=316&amp;publication_year=2007&amp;pages=445&amp;pmid=17363630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib16"><span class="mixed-citation">Girirajan S, Brkanac Z, Coe BP et al: <span class="ref-title">Relative burden of large CNVs on a range of neurodevelopmental phenotypes</span>. <span class="ref-journal">PLoS Genet</span>
2011; <span class="ref-vol">7</span>: e1002334. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3213131/">PMC free article</a>]</span> [<a href="/pubmed/22102821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Relative+burden+of+large+CNVs+on+a+range+of+neurodevelopmental+phenotypes&amp;volume=7&amp;publication_year=2011&amp;pages=e1002334&amp;pmid=22102821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib17"><span class="mixed-citation">Freitag CM: <span class="ref-title">The genetics of autistic disorders and its clinical relevance: a review of the literature</span>. <span class="ref-journal">Mol Psychiatry</span>
2007; <span class="ref-vol">12</span>: 2–22. [<a href="/pubmed/17033636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=The+genetics+of+autistic+disorders+and+its+clinical+relevance:+a+review+of+the+literature&amp;volume=12&amp;publication_year=2007&amp;pages=2&amp;pmid=17033636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib18"><span class="mixed-citation">Sanders SJ, Murtha MT, Gupta AR et al: <span class="ref-title"><em>De novo</em> mutations revealed by whole-exome sequencing are strongly associated with autism</span>. <span class="ref-journal">Nature</span>
2012; <span class="ref-vol">485</span>: 237–241. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3667984/">PMC free article</a>]</span> [<a href="/pubmed/22495306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=De+novo+mutations+revealed+by+whole-exome+sequencing+are+strongly+associated+with+autism&amp;volume=485&amp;publication_year=2012&amp;pages=237&amp;pmid=22495306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib19"><span class="mixed-citation">O'Roak BJ, Vives L, Girirajan S et al: <span class="ref-title">Sporadic autism exomes reveal a highly interconnected protein network of <em>de novo</em> mutations</span>. <span class="ref-journal">Nature</span>
2012; <span class="ref-vol">485</span>: 246–250. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3350576/">PMC free article</a>]</span> [<a href="/pubmed/22495309" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Sporadic+autism+exomes+reveal+a+highly+interconnected+protein+network+of+de+novo+mutations&amp;volume=485&amp;publication_year=2012&amp;pages=246&amp;pmid=22495309&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib20"><span class="mixed-citation">O'Roak BJ, Deriziotis P, Lee C et al: <span class="ref-title">Exome sequencing in sporadic autism spectrum disorders identifies severe de novo mutations</span>. <span class="ref-journal">Nat Genet</span>
2011; <span class="ref-vol">43</span>: 585–589. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3115696/">PMC free article</a>]</span> [<a href="/pubmed/21572417" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Exome+sequencing+in+sporadic+autism+spectrum+disorders+identifies+severe+de+novo+mutations&amp;volume=43&amp;publication_year=2011&amp;pages=585&amp;pmid=21572417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib21"><span class="mixed-citation">Samocha KE, Robinson EB, Sanders SJ et al: <span class="ref-title">A framework for the interpretation of de novo mutation in human disease</span>. <span class="ref-journal">Nat Genet</span>
2014; <span class="ref-vol">46</span>: 944–950. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4222185/">PMC free article</a>]</span> [<a href="/pubmed/25086666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+framework+for+the+interpretation+of+de+novo+mutation+in+human+disease&amp;volume=46&amp;publication_year=2014&amp;pages=944&amp;pmid=25086666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib22"><span class="mixed-citation">Cross-Disorder Group of the Psychiatric Genomics C: <span class="ref-title">Identification of risk loci with shared effects on five major psychiatric disorders: a genome-wide analysis</span>. <span class="ref-journal">Lancet</span>
2013; <span class="ref-vol">381</span>: 1371–1379. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3714010/">PMC free article</a>]</span> [<a href="/pubmed/23453885" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Identification+of+risk+loci+with+shared+effects+on+five+major+psychiatric+disorders:+a+genome-wide+analysis&amp;volume=381&amp;publication_year=2013&amp;pages=1371&amp;pmid=23453885&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib23"><span class="mixed-citation">Wang K, Zhang H, Ma D et al: <span class="ref-title">Common genetic variants on 5p14.1 associate with autism spectrum disorders</span>. <span class="ref-journal">Nature</span>
2009; <span class="ref-vol">459</span>: 528–533. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2943511/">PMC free article</a>]</span> [<a href="/pubmed/19404256" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+genetic+variants+on+5p14.1+associate+with+autism+spectrum+disorders&amp;volume=459&amp;publication_year=2009&amp;pages=528&amp;pmid=19404256&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib24"><span class="mixed-citation">Ma D, Salyakina D, Jaworski JM et al: <span class="ref-title">A genome-wide association study of autism reveals a common novel risk locus at 5p14.1</span>. <span class="ref-journal">Ann Hum Genet</span>
2009; <span class="ref-vol">73</span>: 263–273. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2918410/">PMC free article</a>]</span> [<a href="/pubmed/19456320" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=A+genome-wide+association+study+of+autism+reveals+a+common+novel+risk+locus+at+5p14.1&amp;volume=73&amp;publication_year=2009&amp;pages=263&amp;pmid=19456320&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib25"><span class="mixed-citation">Anney R, Klei L, Pinto D et al: <span class="ref-title">Individual common variants exert weak effects on the risk for autism spectrum disorderspi</span>. <span class="ref-journal">Hum Mol Genet</span>
2012; <span class="ref-vol">21</span>: 4781–4792. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3471395/">PMC free article</a>]</span> [<a href="/pubmed/22843504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Individual+common+variants+exert+weak+effects+on+the+risk+for+autism+spectrum+disorderspi&amp;volume=21&amp;publication_year=2012&amp;pages=4781&amp;pmid=22843504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib26"><span class="mixed-citation">Anney R, Klei L, Pinto D et al: <span class="ref-title">A genome-wide scan for common alleles affecting risk for autism</span>. <span class="ref-journal">Hum Mol Genet</span>
2010; <span class="ref-vol">19</span>: 4072–4082. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947401/">PMC free article</a>]</span> [<a href="/pubmed/20663923" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=A+genome-wide+scan+for+common+alleles+affecting+risk+for+autism&amp;volume=19&amp;publication_year=2010&amp;pages=4072&amp;pmid=20663923&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib27"><span class="mixed-citation">Connolly JJ, Glessner JT, Hakonarson H: <span class="ref-title">A genome-wide association study of autism incorporating autism diagnostic interview-revised, autism diagnostic observation schedule, and social responsiveness scale</span>. <span class="ref-journal">Child Dev</span>
2013; <span class="ref-vol">84</span>: 17–33. [<a href="/pubmed/22935194" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Child+Dev&amp;title=A+genome-wide+association+study+of+autism+incorporating+autism+diagnostic+interview-revised,+autism+diagnostic+observation+schedule,+and+social+responsiveness+scale&amp;volume=84&amp;publication_year=2013&amp;pages=17&amp;pmid=22935194&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib28"><span class="mixed-citation">Weiss LA, Arking DE, Daly MJ, Chakravarti A Gene Discovery Project of Johns H and the Autism C: <span class="ref-title">A genome-wide linkage and association scan reveals novel loci for autism</span>. <span class="ref-journal">Nature</span>
2009; <span class="ref-vol">461</span>: 802–808. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2772655/">PMC free article</a>]</span> [<a href="/pubmed/19812673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+linkage+and+association+scan+reveals+novel+loci+for+autism&amp;volume=461&amp;publication_year=2009&amp;pages=802&amp;pmid=19812673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib29"><span class="mixed-citation">Xia K, Guo H, Hu Z et al: <span class="ref-title">Common genetic variants on 1p13.2 associate with risk of autism</span>. <span class="ref-journal">Mol Psychiatry</span>
2014; <span class="ref-vol">19</span>: 1212–1219. [<a href="/pubmed/24189344" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Common+genetic+variants+on+1p13.2+associate+with+risk+of+autism&amp;volume=19&amp;publication_year=2014&amp;pages=1212&amp;pmid=24189344&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib30"><span class="mixed-citation">Holt R, Monaco AP: <span class="ref-title">Links between genetics and pathophysiology in the autism spectrum disorders</span>. <span class="ref-journal">EMBO Mol Med</span>
2011; <span class="ref-vol">3</span>: 438–450. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3377085/">PMC free article</a>]</span> [<a href="/pubmed/21805639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+Mol+Med&amp;title=Links+between+genetics+and+pathophysiology+in+the+autism+spectrum+disorders&amp;volume=3&amp;publication_year=2011&amp;pages=438&amp;pmid=21805639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib31"><span class="mixed-citation">Roeyers H, Thys M, Druart C, De Schryver M, Schittekatte M: <span class="ref-journal">SRS Screeningslijst Voor Autismespectrumstoornissen</span>. Amsterdam: Hogrefe Uitgevers B.V, 2011. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=SRS+Screeningslijst+Voor+Autismespectrumstoornissen&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib32"><span class="mixed-citation">Noens I, De la Marche W, Scholte E: <span class="ref-journal">SRS Screeningslijst Voor Autismespectrumstoornissen bij Volwassenen</span>. Amsterdam: Hogrefe Uitgevers BV, 2012. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=SRS+Screeningslijst+Voor+Autismespectrumstoornissen+bij+Volwassenen&amp;publication_year=2012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib33"><span class="mixed-citation">Mukhopadhyay N, Almasy L, Schroeder M, Mulvihill WP, Weeks DE: <span class="ref-title">Mega2: data-handling for facilitating genetic linkage and association analyses</span>. <span class="ref-journal">Bioinformatics</span>
2005; <span class="ref-vol">21</span>: 2556–2557. [<a href="/pubmed/15746282" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Mega2:+data-handling+for+facilitating+genetic+linkage+and+association+analyses&amp;volume=21&amp;publication_year=2005&amp;pages=2556&amp;pmid=15746282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib34"><span class="mixed-citation">Wigginton JE, Abecasis GR: <span class="ref-title">PEDSTATS: descriptive statistics, graphics and quality assessment for gene mapping data</span>. <span class="ref-journal">Bioinformatics</span>
2005; <span class="ref-vol">21</span>: 3445–3447. [<a href="/pubmed/15947021" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PEDSTATS:+descriptive+statistics,+graphics+and+quality+assessment+for+gene+mapping+data&amp;volume=21&amp;publication_year=2005&amp;pages=3445&amp;pmid=15947021&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib35"><span class="mixed-citation">Li M, Boehnke M, Abecasis GR: <span class="ref-title">Joint modeling of linkage and association: identifying SNPs responsible for a linkage signal</span>. <span class="ref-journal">Am J Hum Genet</span>
2005; <span class="ref-vol">76</span>: 934–949. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1196453/">PMC free article</a>]</span> [<a href="/pubmed/15877278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Joint+modeling+of+linkage+and+association:+identifying+SNPs+responsible+for+a+linkage+signal&amp;volume=76&amp;publication_year=2005&amp;pages=934&amp;pmid=15877278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib36"><span class="mixed-citation">Purcell S, Neale B, Todd-Brown K et al: <span class="ref-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <span class="ref-journal">Am J Hum Genet</span>
2007; <span class="ref-vol">81</span>: 559–575. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;volume=81&amp;publication_year=2007&amp;pages=559&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib37"><span class="mixed-citation">Li B, Leal SM: <span class="ref-title">Methods for detecting associations with rare variants for common diseases: application to analysis of sequence data</span>. <span class="ref-journal">Am J Hum Genet</span>
2008; <span class="ref-vol">83</span>: 311–321. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2842185/">PMC free article</a>]</span> [<a href="/pubmed/18691683" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Methods+for+detecting+associations+with+rare+variants+for+common+diseases:+application+to+analysis+of+sequence+data&amp;volume=83&amp;publication_year=2008&amp;pages=311&amp;pmid=18691683&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib38"><span class="mixed-citation">Boyle AP, Hong EL, Hariharan M et al: <span class="ref-title">Annotation of functional variation in personal genomes using RegulomeDB</span>. <span class="ref-journal">Genome Res</span>
2012; <span class="ref-vol">22</span>: 1790–1797. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3431494/">PMC free article</a>]</span> [<a href="/pubmed/22955989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Annotation+of+functional+variation+in+personal+genomes+using+RegulomeDB&amp;volume=22&amp;publication_year=2012&amp;pages=1790&amp;pmid=22955989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib39"><span class="mixed-citation">Ward LD, Kellis M: <span class="ref-title">HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</span>. <span class="ref-journal">Nucleic Acids Res</span>
2012; <span class="ref-vol">40</span>: D930–D934. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245002/">PMC free article</a>]</span> [<a href="/pubmed/22064851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=HaploReg:+a+resource+for+exploring+chromatin+states,+conservation,+and+regulatory+motif+alterations+within+sets+of+genetically+linked+variants&amp;volume=40&amp;publication_year=2012&amp;pages=D930&amp;pmid=22064851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib40"><span class="mixed-citation">Baron-Cohen S, Wheelwright S, Skinner R, Martin J, Clubley E: <span class="ref-title">The autism-spectrum quotient (AQ): evidence from Asperger syndrome/high-functioning autism, males and females, scientists and mathematicians</span>. <span class="ref-journal">J Autism Dev Disord</span>
2001; <span class="ref-vol">31</span>: 5–17. [<a href="/pubmed/11439754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autism+Dev+Disord&amp;title=The+autism-spectrum+quotient+(AQ):+evidence+from+Asperger+syndrome/high-functioning+autism,+males+and+females,+scientists+and+mathematicians&amp;volume=31&amp;publication_year=2001&amp;pages=5&amp;pmid=11439754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib41"><span class="mixed-citation">Aulchenko YS, Heutink P, Mackay I et al: <span class="ref-title">Linkage disequilibrium in young genetically isolated Dutch population</span>. <span class="ref-journal">Eur J Hum Genet</span>
2004; <span class="ref-vol">12</span>: 527–534. [<a href="/pubmed/15054401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Linkage+disequilibrium+in+young+genetically+isolated+Dutch+population&amp;volume=12&amp;publication_year=2004&amp;pages=527&amp;pmid=15054401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib42"><span class="mixed-citation">Pardo LM, MacKay I, Oostra B, van Duijn CM, Aulchenko YS: <span class="ref-title">The effect of genetic drift in a young genetically isolated population</span>. <span class="ref-journal">Ann Hum Genet</span>
2005; <span class="ref-vol">69</span>: 288–295. [<a href="/pubmed/15845033" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=The+effect+of+genetic+drift+in+a+young+genetically+isolated+population&amp;volume=69&amp;publication_year=2005&amp;pages=288&amp;pmid=15845033&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib43"><span class="mixed-citation">Vojinovic D, Adams HH, van der Lee SJ et al: <span class="ref-title">The dystrophin gene and cognitive function in the general population</span>. <span class="ref-journal">Eur J Hum Genet</span>
2015; <span class="ref-vol">23</span>: 837–843. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4795046/">PMC free article</a>]</span> [<a href="/pubmed/25227141" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=The+dystrophin+gene+and+cognitive+function+in+the+general+population&amp;volume=23&amp;publication_year=2015&amp;pages=837&amp;pmid=25227141&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib44"><span class="mixed-citation">Amin N, Jovanova O, Adams HH et al: <span class="ref-title">Exome-sequencing in a large population-based study reveals a rare Asn396Ser variant in the LIPG gene associated with depressive symptoms</span>. <span class="ref-journal">Mol Psychiatry</span>
2016; <span class="ref-vol">22</span>: 634. [<a href="/pubmed/27502476" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Exome-sequencing+in+a+large+population-based+study+reveals+a+rare+Asn396Ser+variant+in+the+LIPG+gene+associated+with+depressive+symptoms&amp;volume=22&amp;publication_year=2016&amp;pages=634&amp;pmid=27502476&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib45"><span class="mixed-citation">Brouwer RW, van den Hout MC, Grosveld FG, van Ijcken WF: <span class="ref-title">NARWHAL, a primary analysis pipeline for NGS data</span>. <span class="ref-journal">Bioinformatics</span>
2012; <span class="ref-vol">28</span>: 284–285. [<a href="/pubmed/22072383" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=NARWHAL,+a+primary+analysis+pipeline+for+NGS+data&amp;volume=28&amp;publication_year=2012&amp;pages=284&amp;pmid=22072383&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib46"><span class="mixed-citation">Bucan M, Abrahams BS, Wang K et al: <span class="ref-title">Genome-wide analyses of exonic copy number variants in a family-based study point to novel autism susceptibility genes</span>. <span class="ref-journal">PLoS Genet</span>
2009; <span class="ref-vol">5</span>: e1000536. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695001/">PMC free article</a>]</span> [<a href="/pubmed/19557195" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+analyses+of+exonic+copy+number+variants+in+a+family-based+study+point+to+novel+autism+susceptibility+genes&amp;volume=5&amp;publication_year=2009&amp;pages=e1000536&amp;pmid=19557195&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib47"><span class="mixed-citation">McKenna A, Hanna M, Banks E et al: <span class="ref-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span>. <span class="ref-journal">Genome Res</span>
2010; <span class="ref-vol">20</span>: 1297–1303. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928508/">PMC free article</a>]</span> [<a href="/pubmed/20644199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=The+Genome+Analysis+Toolkit:+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data&amp;volume=20&amp;publication_year=2010&amp;pages=1297&amp;pmid=20644199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib48"><span class="mixed-citation">McCauley JL, Li C, Jiang L et al: <span class="ref-title">Genome-wide and ordered-subset linkage analyses provide support for autism loci on 17q and 19p with evidence of phenotypic and interlocus genetic correlates</span>. <span class="ref-journal">BMC Med Genet</span>
2005; <span class="ref-vol">6</span>: 1. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC546213/">PMC free article</a>]</span> [<a href="/pubmed/15647115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=Genome-wide+and+ordered-subset+linkage+analyses+provide+support+for+autism+loci+on+17q+and+19p+with+evidence+of+phenotypic+and+interlocus+genetic+correlates&amp;volume=6&amp;publication_year=2005&amp;pages=1&amp;pmid=15647115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib49"><span class="mixed-citation">Lowe JK, Werling DM, Constantino JN, Cantor RM, Geschwind DH: <span class="ref-title">Social responsiveness, an autism endophenotype: genomewide significant linkage to two regions on chromosome 8</span>. <span class="ref-journal">Am J Psychiatry</span>
2015; <span class="ref-vol">172</span>: 266–275. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4523091/">PMC free article</a>]</span> [<a href="/pubmed/25727539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=Social+responsiveness,+an+autism+endophenotype:+genomewide+significant+linkage+to+two+regions+on+chromosome+8&amp;volume=172&amp;publication_year=2015&amp;pages=266&amp;pmid=25727539&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib50"><span class="mixed-citation">Xu Y, Cao J, Huang S et al: <span class="ref-title">Characterization of tetratricopeptide repeat-containing proteins critical for cilia formation and function</span>. <span class="ref-journal">PLoS One</span>
2015; <span class="ref-vol">10</span>: e0124378. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4393279/">PMC free article</a>]</span> [<a href="/pubmed/25860617" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Characterization+of+tetratricopeptide+repeat-containing+proteins+critical+for+cilia+formation+and+function&amp;volume=10&amp;publication_year=2015&amp;pages=e0124378&amp;pmid=25860617&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib51"><span class="mixed-citation">D'Angelo A, Franco B: <span class="ref-title">The dynamic cilium in human diseases</span>. <span class="ref-journal">Pathogenetics</span>
2009; <span class="ref-vol">2</span>: 3. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2694804/">PMC free article</a>]</span> [<a href="/pubmed/19439065" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pathogenetics&amp;title=The+dynamic+cilium+in+human+diseases&amp;volume=2&amp;publication_year=2009&amp;pages=3&amp;pmid=19439065&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib52"><span class="mixed-citation">Louvi A, Grove EA: <span class="ref-title">Cilia in the CNS: the quiet organelle claims center stage</span>. <span class="ref-journal">Neuron</span>
2011; <span class="ref-vol">69</span>: 1046–1060. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3070490/">PMC free article</a>]</span> [<a href="/pubmed/21435552" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Cilia+in+the+CNS:+the+quiet+organelle+claims+center+stage&amp;volume=69&amp;publication_year=2011&amp;pages=1046&amp;pmid=21435552&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib53"><span class="mixed-citation">Basten SG, Giles RH: <span class="ref-title">Functional aspects of primary cilia in signaling, cell cycle and tumorigenesis</span>. <span class="ref-journal">Cilia</span>
2013; <span class="ref-vol">2</span>: 6. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3662159/">PMC free article</a>]</span> [<a href="/pubmed/23628112" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cilia&amp;title=Functional+aspects+of+primary+cilia+in+signaling,+cell+cycle+and+tumorigenesis&amp;volume=2&amp;publication_year=2013&amp;pages=6&amp;pmid=23628112&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib54"><span class="mixed-citation">Doherty D: <span class="ref-title">Joubert syndrome: insights into brain development, cilium biology, and complex disease</span>. <span class="ref-journal">Semin Pediatr Neurol</span>
2009; <span class="ref-vol">16</span>: 143–154. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2804071/">PMC free article</a>]</span> [<a href="/pubmed/19778711" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin+Pediatr+Neurol&amp;title=Joubert+syndrome:+insights+into+brain+development,+cilium+biology,+and+complex+disease&amp;volume=16&amp;publication_year=2009&amp;pages=143&amp;pmid=19778711&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib55"><span class="mixed-citation">Ozonoff S, Williams BJ, Gale S, Miller JN: <span class="ref-title">Autism and autistic behavior in Joubert syndrome</span>. <span class="ref-journal">J Child Neurol</span>
1999; <span class="ref-vol">14</span>: 636–641. [<a href="/pubmed/10511335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Child+Neurol&amp;title=Autism+and+autistic+behavior+in+Joubert+syndrome&amp;volume=14&amp;publication_year=1999&amp;pages=636&amp;pmid=10511335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib56"><span class="mixed-citation">Artigas-Pallares J, Gabau-Vila E, Guitart-Feliubadalo M: <span class="ref-title">(Syndromic autism: II. Genetic syndromes associated with autism) El autismo sindromico: II. Sindromes de base genetica asociados a autismo</span>. <span class="ref-journal">Rev Neurol</span>
2005; <span class="ref-vol">40</span>: S151–S162. [<a href="/pubmed/15736079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Rev+Neurol&amp;title=(Syndromic+autism:+II.+Genetic+syndromes+associated+with+autism)+El+autismo+sindromico:+II.+Sindromes+de+base+genetica+asociados+a+autismo&amp;volume=40&amp;publication_year=2005&amp;pages=S151&amp;pmid=15736079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib57"><span class="mixed-citation">Holroyd S, Reiss AL, Bryan RN: <span class="ref-title">Autistic features in Joubert syndrome: a genetic disorder with agenesis of the cerebellar vermis</span>. <span class="ref-journal">Biol Psychiatry</span>
1991; <span class="ref-vol">29</span>: 287–294. [<a href="/pubmed/2015333" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Autistic+features+in+Joubert+syndrome:+a+genetic+disorder+with+agenesis+of+the+cerebellar+vermis&amp;volume=29&amp;publication_year=1991&amp;pages=287&amp;pmid=2015333&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib58"><span class="mixed-citation">Alvarez Retuerto AI, Cantor RM, Gleeson JG et al: <span class="ref-title">Association of common variants in the Joubert syndrome gene (AHI1) with autism</span>. <span class="ref-journal">Hum Mol Genet</span>
2008; <span class="ref-vol">17</span>: 3887–3896. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2638573/">PMC free article</a>]</span> [<a href="/pubmed/18782849" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Association+of+common+variants+in+the+Joubert+syndrome+gene+(AHI1)+with+autism&amp;volume=17&amp;publication_year=2008&amp;pages=3887&amp;pmid=18782849&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib59"><span class="mixed-citation">Wrana JL: <span class="ref-title">Signaling by the TGFbeta superfamily</span>. <span class="ref-journal">Cold Spring Harb Perspect Biol</span>
2013; <span class="ref-vol">5</span>: a011197. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3783053/">PMC free article</a>]</span> [<a href="/pubmed/24086041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cold+Spring+Harb+Perspect+Biol&amp;title=Signaling+by+the+TGFbeta+superfamily&amp;volume=5&amp;publication_year=2013&amp;pages=a011197&amp;pmid=24086041&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib60"><span class="mixed-citation">Okada K, Hashimoto K, Iwata Y et al: <span class="ref-title">Decreased serum levels of transforming growth factor-beta1 in patients with autism</span>. <span class="ref-journal">Prog Neuropsychopharmacol Biol Psychiatry</span>
2007; <span class="ref-vol">31</span>: 187–190. [<a href="/pubmed/17030376" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prog+Neuropsychopharmacol+Biol+Psychiatry&amp;title=Decreased+serum+levels+of+transforming+growth+factor-beta1+in+patients+with+autism&amp;volume=31&amp;publication_year=2007&amp;pages=187&amp;pmid=17030376&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib61"><span class="mixed-citation">Ashwood P, Enstrom A, Krakowiak P et al: <span class="ref-title">Decreased transforming growth factor beta1 in autism: a potential link between immune dysregulation and impairment in clinical behavioral outcomes</span>. <span class="ref-journal">J Neuroimmunol</span>
2008; <span class="ref-vol">204</span>: 149–153. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2615583/">PMC free article</a>]</span> [<a href="/pubmed/18762342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neuroimmunol&amp;title=Decreased+transforming+growth+factor+beta1+in+autism:+a+potential+link+between+immune+dysregulation+and+impairment+in+clinical+behavioral+outcomes&amp;volume=204&amp;publication_year=2008&amp;pages=149&amp;pmid=18762342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib62"><span class="mixed-citation">Vargas DL, Nascimbene C, Krishnan C, Zimmerman AW, Pardo CA: <span class="ref-title">Neuroglial activation and neuroinflammation in the brain of patients with autism</span>. <span class="ref-journal">Ann Neurol</span>
2005; <span class="ref-vol">57</span>: 67–81. [<a href="/pubmed/15546155" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Neuroglial+activation+and+neuroinflammation+in+the+brain+of+patients+with+autism&amp;volume=57&amp;publication_year=2005&amp;pages=67&amp;pmid=15546155&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib63"><span class="mixed-citation">McElhanon BO, McCracken C, Karpen S, Sharp WG: <span class="ref-title">Gastrointestinal symptoms in autism spectrum disorder: a meta-analysis</span>. <span class="ref-journal">Pediatrics</span>
2014; <span class="ref-vol">133</span>: 872–883. [<a href="/pubmed/24777214" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Gastrointestinal+symptoms+in+autism+spectrum+disorder:+a+meta-analysis&amp;volume=133&amp;publication_year=2014&amp;pages=872&amp;pmid=24777214&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib64"><span class="mixed-citation">Coury DL, Ashwood P, Fasano A et al: <span class="ref-title">Gastrointestinal conditions in children with autism spectrum disorder: developing a research agenda</span>. <span class="ref-journal">Pediatrics</span>
2012; <span class="ref-vol">130</span>: S160–S168. [<a href="/pubmed/23118247" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Gastrointestinal+conditions+in+children+with+autism+spectrum+disorder:+developing+a+research+agenda&amp;volume=130&amp;publication_year=2012&amp;pages=S160&amp;pmid=23118247&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib65"><span class="mixed-citation">Margolis KG, Li Z, Stevanovic K et al: <span class="ref-title">Serotonin transporter variant drives preventable gastrointestinal abnormalities in development and function</span>. <span class="ref-journal">J Clin Invest</span>
2016; <span class="ref-vol">126</span>: 2221–2235. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4887174/">PMC free article</a>]</span> [<a href="/pubmed/27111230" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Serotonin+transporter+variant+drives+preventable+gastrointestinal+abnormalities+in+development+and+function&amp;volume=126&amp;publication_year=2016&amp;pages=2221&amp;pmid=27111230&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib66"><span class="mixed-citation">Kerin T, Ramanathan A, Rivas K, Grepo N, Coetzee GA, Campbell DB: <span class="ref-title">A noncoding RNA antisense to moesin at 5p14.1 in autism</span>. <span class="ref-journal">Sci Transl Med</span>
2012; <span class="ref-vol">4</span>: 128ra140r. [<a href="/pubmed/22491950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Transl+Med&amp;title=A+noncoding+RNA+antisense+to+moesin+at+5p14.1+in+autism&amp;volume=4&amp;publication_year=2012&amp;pages=128ra140r&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib67"><span class="mixed-citation">Almasy L: <span class="ref-title">The role of phenotype in gene discovery in the whole genome sequencing era</span>. <span class="ref-journal">Hum Genet</span>
2012; <span class="ref-vol">131</span>: 1533–1540. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3525519/">PMC free article</a>]</span> [<a href="/pubmed/22722752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=The+role+of+phenotype+in+gene+discovery+in+the+whole+genome+sequencing+era&amp;volume=131&amp;publication_year=2012&amp;pages=1533&amp;pmid=22722752&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib68"><span class="mixed-citation">Geschwind DH: <span class="ref-title">Advances in autism</span>. <span class="ref-journal">Annu Rev Med</span>
2009; <span class="ref-vol">60</span>: 367–380. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3645857/">PMC free article</a>]</span> [<a href="/pubmed/19630577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Med&amp;title=Advances+in+autism&amp;volume=60&amp;publication_year=2009&amp;pages=367&amp;pmid=19630577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib69"><span class="mixed-citation">Armstrong K, Iarocci G: <span class="ref-title">Brief report: the autism spectrum quotient has convergent validity with the social responsiveness scale in a high-functioning sample</span>. <span class="ref-journal">J Autism Dev Disord</span>
2013; <span class="ref-vol">43</span>: 2228–2232. [<a href="/pubmed/23371508" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autism+Dev+Disord&amp;title=Brief+report:+the+autism+spectrum+quotient+has+convergent+validity+with+the+social+responsiveness+scale+in+a+high-functioning+sample&amp;volume=43&amp;publication_year=2013&amp;pages=2228&amp;pmid=23371508&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib70"><span class="mixed-citation">Kircher M, Witten DM, Jain P, O'Roak BJ, Cooper GM, Shendure J: <span class="ref-title">A general framework for estimating the relative pathogenicity of human genetic variants</span>. <span class="ref-journal">Nat Genet</span>
2014; <span class="ref-vol">46</span>: 310–315. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3992975/">PMC free article</a>]</span> [<a href="/pubmed/24487276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+general+framework+for+estimating+the+relative+pathogenicity+of+human+genetic+variants&amp;volume=46&amp;publication_year=2014&amp;pages=310&amp;pmid=24487276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5567150&amp;issue-id=297374&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139681583964352title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Materials and methods</a></li><li><a href="#__sec10title">Results</a></li><li><a href="#__sec12title">Discussion</a></li><li><a href="#idm139681656319888title">Acknowledgments</a></li><li><a href="#idm139681618140032title">Footnotes</a></li><li><a href="#sup1title">Supplementary Material</a></li><li><a href="#idm139681656316464title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5567150/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5567150/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5567150/pdf/ejhg201782a.pdf">PDF (346K)</a></li> | <li><a href="#" data-citationid="PMC5567150" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5567150%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5567150%2F&amp;text=Variants%20in%20TTC25%20affect%20autistic%20trait%20in%20patients%20with%20autism%20spectrum%20disorder%20and%20general%20population" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5567150%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28513607" data-aiid="5567150" data-aid="5567150" data-iid="297374" data-domainid="1300" data-domain="ejhg" data-accid="PMC5567150" data-md5="859e8de81148b5f3bc2fa99b9d8110d4"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:17:08-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B8216DD5A39F100000000060403A9&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5567150%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5567150/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>