<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5539991/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-Wide Association Studies and Meta-Analyses for Congenital Heart Defects</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5539991/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Circulation. Cardiovascular genetics"><meta name="citation_title" content="Genome-Wide Association Studies and Meta-Analyses for Congenital Heart Defects"><meta name="citation_authors" content="A.J. Agopian, Elizabeth Goldmuntz, Hakon Hakonarson, Anshuman Sewda, Deanne Taylor, Laura E. Mitchell, Pediatric Cardiac Genomics Consortium"><meta name="citation_date" content="June 2017"><meta name="citation_issue" content="3"><meta name="citation_volume" content="10"><meta name="citation_firstpage" content="e001449"><meta name="citation_doi" content="10.1161/CIRCGENETICS.116.001449"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5539991/?report=abstract"><meta name="citation_pmid" content="28468790"><meta name="DC.Title" content="Genome-Wide Association Studies and Meta-Analyses for Congenital Heart Defects"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="A.J. Agopian"><meta name="DC.Contributor" content="Elizabeth Goldmuntz"><meta name="DC.Contributor" content="Hakon Hakonarson"><meta name="DC.Contributor" content="Anshuman Sewda"><meta name="DC.Contributor" content="Deanne Taylor"><meta name="DC.Contributor" content="Laura E. Mitchell"><meta name="DC.Contributor" content="Pediatric Cardiac Genomics Consortium"><meta name="DC.Date" content="2017 Jun"><meta name="DC.Identifier" content="10.1161/CIRCGENETICS.116.001449"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-Wide Association Studies and Meta-Analyses for Congenital Heart Defects"><meta property="og:type" content="article"><meta property="og:description" content="Maternal and inherited (i.e., case) genetic factors likely contribute to the etiology of congenital heart defects, but it is unclear whether individual common variants confer a large risk.To evaluate the relationship between individual common maternal/inherited ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5539991/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5539991/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5539991/pdf/nihms864518.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B0FD8DD5AB1410000000000BF0070.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5539991%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5539991</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Circ Cardiovasc Genet</a></span>. Author manuscript; available in PMC 2018 Jun 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28468790" target="pmc_ext" ref="reftype=publisher&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Circ Cardiovasc Genet. 2017 Jun; 10(3): e001449. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1161%2FCIRCGENETICS.116.001449" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1161/CIRCGENETICS.116.001449</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5539991</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS864518</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28468790">28468790</a></div></div></div></div><h1 class="content-title">Genome-Wide Association Studies and Meta-Analyses for Congenital Heart Defects</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Agopian%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790" class="affpopup" co-rid="_co_idm140245428021632" co-class="co-affbox">A.J. Agopian</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Goldmuntz%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790" class="affpopup" co-rid="_co_idm140245427168672" co-class="co-affbox">Elizabeth Goldmuntz</a>, MD,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790" class="affpopup" co-rid="_co_idm140245418599616" co-class="co-affbox">Hakon Hakonarson</a>, MD, PhD,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Sewda%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790" class="affpopup" co-rid="_co_idm140245420990624" co-class="co-affbox">Anshuman Sewda</a>, MPH,<sup>1</sup> <a href="/pubmed/?term=Taylor%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790" class="affpopup" co-rid="_co_idm140245430524496" co-class="co-affbox">Deanne Taylor</a>, PhD,<sup>5</sup> <a href="/pubmed/?term=Mitchell%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790" class="affpopup" co-rid="_co_idm140245432275328" co-class="co-affbox">Laura E. Mitchell</a>, PhD,<sup>1</sup> and  Pediatric Cardiac Genomics Consortium</div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140245428021632"><h3 class="no_margin">A.J. Agopian</h3><p>  <sup>1</sup>Human Genetics Center, Department of Epidemiology, Human Genetics and Environmental Sciences, UTHealth School of Public Health, Houston, TX</p><div>Find articles by <a href="/pubmed/?term=Agopian%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790">A.J. Agopian</a></div></div><div id="_co_idm140245427168672"><h3 class="no_margin">Elizabeth Goldmuntz</h3><p>  <sup>2</sup>Department of Pediatrics, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA</p><p>  <sup>3</sup>Division of Cardiology, The Children’s Hospital of Philadelphia, Philadelphia, PA</p><div>Find articles by <a href="/pubmed/?term=Goldmuntz%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790">Elizabeth Goldmuntz</a></div></div><div id="_co_idm140245418599616"><h3 class="no_margin">Hakon Hakonarson</h3><p>  <sup>3</sup>Division of Cardiology, The Children’s Hospital of Philadelphia, Philadelphia, PA</p><p>  <sup>4</sup>Center for Applied Genomics, The Children’s Hospital of Philadelphia, Philadelphia, PA</p><div>Find articles by <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790">Hakon Hakonarson</a></div></div><div id="_co_idm140245420990624"><h3 class="no_margin">Anshuman Sewda</h3><p>  <sup>1</sup>Human Genetics Center, Department of Epidemiology, Human Genetics and Environmental Sciences, UTHealth School of Public Health, Houston, TX</p><div>Find articles by <a href="/pubmed/?term=Sewda%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790">Anshuman Sewda</a></div></div><div id="_co_idm140245430524496"><h3 class="no_margin">Deanne Taylor</h3><p>  <sup>5</sup>Department of Biomedical and Health Informatics, The Children’s Hospital of Philadelphia, Philadelphia, PA</p><div>Find articles by <a href="/pubmed/?term=Taylor%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790">Deanne Taylor</a></div></div><div id="_co_idm140245432275328"><h3 class="no_margin">Laura E. Mitchell</h3><p>  <sup>1</sup>Human Genetics Center, Department of Epidemiology, Human Genetics and Environmental Sciences, UTHealth School of Public Health, Houston, TX</p><div>Find articles by <a href="/pubmed/?term=Mitchell%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28468790">Laura E. Mitchell</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140245423127104_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140245423127104_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140245423127104_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">  <sup>1</sup>Human Genetics Center, Department of Epidemiology, Human Genetics and Environmental Sciences, UTHealth School of Public Health, Houston, TX</div><div class="fm-affl" id="A2" lang="en">  <sup>2</sup>Department of Pediatrics, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA</div><div class="fm-affl" id="A3" lang="en">  <sup>3</sup>Division of Cardiology, The Children’s Hospital of Philadelphia, Philadelphia, PA</div><div class="fm-affl" id="A4" lang="en">  <sup>4</sup>Center for Applied Genomics, The Children’s Hospital of Philadelphia, Philadelphia, PA</div><div class="fm-affl" id="A5" lang="en">  <sup>5</sup>Department of Biomedical and Health Informatics, The Children’s Hospital of Philadelphia, Philadelphia, PA</div><div id="FN1">Correspondence: Laura E. Mitchell, PhD, University of Texas School of Public Health, 1200 Pressler, Houston, TX 77030, Tel.: 713-500-9954, Fax: 713-500-9958, <a href="mailto:dev@null" data-email="ude.cmt.htu@llehctim.e.arual" class="oemail">ude.cmt.htu@llehctim.e.arual</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140245423127104_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28468790" target="pmc_ext" ref="reftype=publisher&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Circ Cardiovasc Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5539991/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>001449 - Supplemental Material.</strong></div><div><a href="/pmc/articles/PMC5539991/bin/NIHMS864518-supplement-001449_-_Supplemental_Material.pdf" data-ga-action="click_feat_suppl">NIHMS864518-supplement-001449_-_Supplemental_Material.pdf</a><span style="color:gray"> (8.0M)</span></div><div class="small guid">GUID:&nbsp;212A0AB0-06C8-43D1-B2E7-52DE038C3156</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>001449 - Supplemental Tables.</strong></div><div><a href="/pmc/articles/PMC5539991/bin/NIHMS864518-supplement-001449_-_Supplemental_Tables.xls" data-ga-action="click_feat_suppl">NIHMS864518-supplement-001449_-_Supplemental_Tables.xls</a><span style="color:gray"> (218K)</span></div><div class="small guid">GUID:&nbsp;04009D9D-5172-440B-91E7-05FCC1AC245A</div></div></dd></dl></div><div id="idm140245423864560" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140245423864560title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P1" class="p p-first-last">Maternal and inherited (i.e., case) genetic factors likely contribute to the etiology of congenital heart defects, but it is unclear whether individual common variants confer a large risk.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods and Results</h3><p id="P2" class="p p-first-last">To evaluate the relationship between individual common maternal/inherited genotypes and risk for heart defects, we conducted genome-wide association studies (GWAS) in five cohorts. Three cohorts were recruited at the Children’s Hospital of Philadelphia: 670 conotruncal heart defect (CTD) case-parent trios; 317 left ventricular obstructive tract defect (LVOTD) case-parent trios; and 406 CTD cases and 2,976 pediatric controls. Two cohorts were recruited through the Pediatric Cardiac Genomics Consortium: 355 CTD trios and 192 LVOTD trios. We also conducted meta-analyses using the GWAS results from the CTD cohorts, the LVOTD cohorts and from the combined CTD and LVOTD cohorts. In the individual GWAS, several genome-wide significant associations (p≤5×10<sup>−8</sup>) were observed. In our meta-analyses, one genome-wide significant association was detected: the case genotype for rs72820264, an intra-genetic SNP associated with LVOTDs (p=2.1×10<sup>−8</sup>).</p></div><div id="S3" class="sec sec-last"><h3 id="S3title">Conclusions</h3><p id="P3" class="p p-first-last">We identified one novel candidate region associated with LVOTDs and report on several additional regions with suggestive evidence for association with CTD and/or LVOTD. These studies were constrained by the relatively small samples sizes and thus have limited power to detect small to moderate associations. Approaches that minimize the multiple testing burden (e.g. gene- or pathway-based) may, therefore, be required to uncover common variants contributing to the risk of these relatively rare conditions.</p></div></div><div class="sec"><strong class="kwd-title">Journal Subject Terms: </strong><span class="kwd-text">Epidemiology, Genetic, Association Studies, Congenital Heart Disease, Functional Genomics</span></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Genome Wide Association Study, congenital cardiac defect, left ventricular outflow tract obstruction, epidemiology</span></div></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">Introduction</h2><p id="P4" class="p p-first">Congenital heart defects (CHDs) are the most common birth defect, with a prevalence of nearly 1 in 100 live births,<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079630">1</a></sup> and account for nearly 40% of infant deaths in North America.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079588">2</a></sup> Although few specific risk factors have been identified for CHDs, a genetic contribution to risk is expected based on narrow sense heritability estimates (h<sup>2</sup> ~0.8)<sup><a href="#R3" rid="R3" class=" bibr popnode">3</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079577">5</a></sup> and the occurrence of CHDs as part of specific genetic syndromes (e.g., 22q11 deletion syndrome).<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079596">6</a>, <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079618">7</a></sup> Moreover, familial recurrence patterns indicate that the genetic contribution to non-syndromic CHDs is likely to be complex and involve multiple, inherited variants of low to moderate effect.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079593">8</a>, <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079578">9</a></sup> Both maternal and inherited (i.e., case) genetic factors (as well as non-genetic factors) are thought to contribute to the risk of CHDs.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079600">10</a>–<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079579">12</a></sup> In addition, different groups of heart defects (e.g., conotruncal defects, left ventricular outflow tract defects) may share some genetic risk factors while also having group and even lesion-specific genetic etiologies.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079623">10</a>, <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079585">13</a>, <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079628">14</a></sup></p><p id="P5">While much remains to be learned about the genetic contribution to CHDs, genomic approaches, including genome-wide association studies (GWAS), have identified novel candidate genes and loci with relatively strong effects. For example, genome-wide significant and/or suggestive associations have been reported for conotruncal defects (CTDs),<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079580">15</a></sup> tetralogy of Fallot,<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079575">16</a></sup> left ventricular obstructive tract defects (LVOTDs)<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079607">17</a></sup> and septal defects.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079624">18</a>–<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079581">20</a></sup> Most of the reported associations have yet to be independently replicated. Hence, while GWAS have provided some evidence that the risk of CHDs is influenced by common genetic variants, further studies are warranted. Though the majority of GWAS of adult-onset traits have identified weak to moderate effects, stronger effects have been reported for birth defects and other early-onset traits.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079608">21</a></sup> Thus, we hypothesized that common individual variants contribute to the genetic architecture of CHD risk.</p><p id="P6" class="p p-last">To agnostically identify a subset of common candidate SNPs for future study, we conducted GWAS in multiple CHD cohorts, as well as meta-analyses of the combined data. Our analyses evaluated associations of CHD with both the maternal and inherited (case) genotypes.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Methods</h2><div id="S6" class="sec sec-first"><h3 id="S6title">Study Subjects</h3><div id="S7" class="sec sec-first"><p></p><h4 id="S7title" class="inline">Children’s Hospital of Philadelphia – CTD Trios </h4><p id="P7" class="p p-first-last">CTD case-parent trios were recruited from The Cardiac Center at The Children’s Hospital of Philadelphia (CHOP) from 1992–2010.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079597">15</a></sup> Individuals of all races/ethnicities were eligible to participate. All subjects provided informed consent under a protocol approved by the CHOP Institutional Review Board for the Protection of Human Subjects. The CTD diagnoses included: tetralogy of Fallot, D-transposition of the great arteries, ventricular septal defects (conoventricular, posterior malalignment and conoseptal hypoplasia), double outlet right ventricle, isolated aortic arch anomalies, truncus arteriosus, and interrupted aortic arch. Cardiac medical records were reviewed (e.g., echocardiography, cardiac MRI, cardiac catheterization, operative notes) to ensure accuracy of the cardiac phenotype, and cases with suspected genetic syndromes were excluded. Further, cases were screened for 22q11.2 deletion syndrome using fluorescence in situ hybridization and/or multiplex ligation-dependent probe amplification.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079599">22</a>, <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079614">23</a></sup></p></div><div id="S8" class="sec"><p></p><h4 id="S8title" class="inline">Children’s Hospital of Philadelphia – LVOTD Trios </h4><p id="P8" class="p p-first-last">LVOTD case-parent trios were recruited from the CHOP Cardiac Center, 1997–2007, using the same approach as described for the CTD trios.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079619">17</a></sup> Individuals of all races/ethnicities were eligible to participate. The LVOTD diagnoses included: hypoplastic left heart syndrome, coarctation of the aorta with or without bicuspid aortic valve, and aortic valve stenosis. We excluded those with variants of hypoplastic left heart syndrome, such as mal-aligned atrioventricular canal defects and double outlet right ventricle with mitral valve atresia. Cardiac medical records were reviewed to ensure accuracy of the cardiac phenotype. Cases with suspected genetic syndromes were excluded.</p></div><div id="S9" class="sec"><p></p><h4 id="S9title" class="inline">Children’s Hospital of Philadelphia – CTD cases and pediatric controls </h4><p id="P9" class="p p-first-last">Additional cases with CTDs were recruited through the CHOP Cardiac Center, 1999–2010, using the same approach as described for CTD trios. Controls were also recruited from CHOP, during well child visits, as previously described.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079617">24</a></sup> Cases and controls were Caucasian by self-report.</p></div><div id="S10" class="sec sec-last"><p></p><h4 id="S10title" class="inline">Pediatric Cardiac Genomics Consortium </h4><p id="P10" class="p p-first-last">CTD and LVOTD case-parent trios were recruited as part of the Pediatric Cardiac Genomics Consortium’s (PCGC) Congenital Heart Disease GEnetic NEtwork Study. The recruitment of these subjects has been described.<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079629">25</a></sup> Briefly, subjects provided informed consent and were recruited into the PCGC cohort at ten clinical study sites from 2010–2012. Individuals of all races/ethnicities were eligible to participate. Cardiac diagnoses were classified based on medical record abstraction by trained study coordinators. Although the PCGC cohort included a broad range of CHDs, our analyses were restricted to those with CTDs or LVOTDs.</p></div></div><div id="S11" class="sec"><h3 id="S11title">Genetic Methods</h3><p id="P11" class="p p-first">Study subjects in all cohorts provided either blood or saliva samples, and DNA was extracted directly from these samples (or from cell lines in a minority of subjects) using standard methods, as previously described. <sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079574">20</a>, <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079598">22</a>, <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079603">25</a></sup> All samples were array genotyped using Illumina arrays (<a href="/pmc/articles/PMC5539991/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) in the CHOP Center for Applied Genomics.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5539991/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140245421974208" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms864518f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms864518f1.jpg" src="/pmc/articles/PMC5539991/bin/nihms864518f1.jpg"></div></a><div id="largeobj_idm140245421974208" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC5539991/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC5539991/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5539991/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p4">Schematic of the five independent cohorts and summary of which cohorts were analyzed in each meta-analysis.</p></div></div></div><p id="P12">The array genotype data from the CHOP CTD and LVOTD trios have previously been used to impute additional genotypes against Hapmap reference data<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079583">26</a></sup> and these genotyped and imputed data were used in our previous GWAS.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079615">15</a>, <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079595">17</a></sup> For the analyses presented here, data for all cohorts were imputed (or re-imputed) using data from the 1,000 Genomes Project reference data.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079582">27</a></sup> Given differences in the timing of the availability of data from CHOP and PCGC, data from these two sources were imputed separately. However, data from all of the CHOP cohorts were imputed together. Likewise, data from PCGC CTD and LVOTD trios were imputed together. Both sets of imputations (i.e., CHOP and PCGC) were based on the genotype data for SNPs that overlapped across all genotyping platforms (i.e., SNPs in common to the 550K, 610K and 2.5M arrays for the CHOP imputation and the 1M and 2.5M arrays for the PCGC imputation). The overlapping set of SNPs was used to minimize differences in imputation accuracy across the cohorts that could arise if imputations were based on different SNPs for each cohort. The imputation and all subsequent analyses were restricted to autosomal variants.</p><p id="P13" class="p p-last">Before imputation, we used PLINK v1.06<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079604">28</a></sup> to perform standard quality control procedures on each study cohort (e.g., CHOP CTD trios). We excluded: trios with Mendelian error rate &gt;1%; SNPs with minor allele frequency &lt;1%, genotyping rate &lt;90%, or deviation from Hardy Weinberg equilibrium in controls/parents (p≤1×10<sup>−5</sup>); and suspected duplicate samples based on pairwise identity by descent pi-hat &gt;0.6. Following these exclusions, data from the three CHOP cohorts were combined, haplotypes were pre-phased using SHAPEIT2 v2.727<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630411771">29</a></sup> and genotypes for additional variants were imputed using Impute 2 v2.3.0,<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079602">30</a></sup> and pre-phased data from the 1,000 Genomes Project (Phase-I integrated v3 variants set).<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079591">27</a>, <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079616">31</a></sup> Post-imputation quality control procedures included removing variants that were: indels, poorly imputed (r<sup>2</sup>&lt; 0.8), did not have rs numbers, or rare (minor allele frequency &lt;5%). Variants and individuals with a genotyping rate &lt;90% were also removed. Data from the PCGC CTD and LVOTD trios were combined and processed using the same procedures.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">Statistical Analysis</h3><div id="S13" class="sec sec-first"><p></p><h4 id="S13title" class="inline">Analytic groups </h4><p id="P14" class="p p-first-last">We conducted separate GWAS for the five independent cohorts: CHOP CTD trios, CHOP LVOTD trios, CHOP CTD cases/controls, PCGC CTD trios, and PCGC LVOT trios (<a href="/pmc/articles/PMC5539991/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). We assessed the inherited (case) genotype in all cohorts, and the maternal genotype in the trios.</p></div><div id="S14" class="sec"><p></p><h4 id="S14title" class="inline">Family-based analyses </h4><p id="P15" class="p p-first">The analyses of case-parent trios were conducted using a multinomial likelihood approach<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079621">32</a></sup> implemented in the EMIM software package.<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079592">33</a></sup> Because the assessment of inherited genotypes using this approach is immune to population stratification bias,<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079613">34</a>, <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079587">35</a></sup> all trios were included in the analyses of inherited genotypes. However, the assessment of maternal genotypes using trio data can be biased by differences in the frequency of mating types as defined by maternal and paternal race/ethnicity (e.g. matings of non-Hispanic Caucasian males and Hispanic females are more frequent than the reciprocal mating type)<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079601">36</a></sup>. Consequently, analyses of maternal genotypes were restricted to trios in which both parents were reported to be non-Hispanic Caucasian.</p><p id="P16" class="p p-last">We used a likelihood ratio test to compare a full model, with terms for both the maternal and inherited genotypes, to a reduced model that excluded the term being tested. We used an additive (one degree of freedom) genetic risk model for the genotype being assessed (e.g., inherited genotype) and an unrestricted (two-degrees of freedom) model for the other genotype (e.g., maternal genotype). Manhattan plots were generated for each comparison using R version 3.03 (<a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>). SNPs with p-values ≤5×10<sup>−8</sup> were considered significantly associated, and SNPs with p-values between 10<sup>−5</sup> to 5×10<sup>−8</sup> were considered to have suggestive evidence of association with CTDs and/or LVOTDs.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079606">15</a>, <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079594">37</a></sup></p></div><div id="S15" class="sec"><p></p><h4 id="S15title" class="inline">Case-control analysis </h4><p id="P17" class="p p-first-last">Using data from the CHOP CTD cases and controls, the association between CTDs and the inherited genotype was assessed using logistic regression, based on an additive genetic risk model. These analyses were adjusted for the first two principal components of race/ethnicity and were conducted using Golden Helix v8.1 (Golden Helix, Inc., Bozeman, MT, <a href="http://www.goldenhelix.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.goldenhelix.com</a>), which implements principal component adjustment. The results from this analysis were visualized and interpreted using the same approach described for the family-based analyses.</p></div><div id="S16" class="sec"><p></p><h4 id="S16title" class="inline">Replication </h4><p id="P18" class="p p-first-last">For each SNP with at least a suggestive association with CHDs in one or more cohorts, we examined the p-values and effect size estimate from the additional, comparable cohorts. For example, if there was suggestive evidence for an association with the inherited genotype for a SNP in the CHOP CTD case-control cohort, we examined the p-values and effect size estimates for that SNP in the analyses of the CHOP and PCGC CTD trios. We deemed associations to be “replicated” if a SNP had at least suggestive evidence of association (p≤10<sup>−5</sup>) in more than one dataset assessing the same genetic effect (maternal or inherited) and phenotype and consistent direction of association in two or more comparable cohorts.</p></div><div id="S17" class="sec sec-last"><p></p><h4 id="S17title" class="inline">Meta-analysis </h4><p id="P19" class="p p-first-last">Because there is evidence that sub-groups of CHDs (e.g. CTDs and LVOTDs) are likely to have overlapping but not identical genetic profiles, we conducted three separate meta-analyses of inherited effects, based on the following groupings (<a href="/pmc/articles/PMC5539991/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>): CTDs only (CHOP CTD trios, CHOP CTD case-control, and PCGC CTD analyses); LVOTD only (CHOP and PCGC LVOTD trios); and CTD+LVOTD (all five cohorts). Likewise, we conducted three separate meta-analyses of maternal effects: CTDs only (CHOP and PCGC CTD trios); LVOTD only (CHOP and PCGC LVOTD trios); and, CTD+LVOTD (all four trio cohorts). Each meta-analysis was performed using a fixed-effects model, unless there was evidence of heterogeneity (based on Cochran’s heterogeneity p≤0.1), in which case a random-effects model was used. We conducted meta-analyses using GWAMA v2.1 (<a href="http://www.well.ox.ac.uk/gwama/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.well.ox.ac.uk/gwama/</a>).<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079609">38</a></sup> Manhattan plots were generated using R version 3.03 (<a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>) and regional association plots were constructed for top hits using Locus Zoom v1.1 (<a href="http://locuszoom.sph.umich.edu/locuszoom/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://locuszoom.sph.umich.edu/locuszoom/</a>).<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079586">39</a></sup> Variants were annotated with Combined Annotation Dependent Depletion (CADD) scores.<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079612">40</a></sup> CADD scores predict the potential deleteriousness of individual variants, such that variants with CADD&gt;10 are predicted to fall in the top 10% of the most deleterious variants in the genome. Variants were also annotated with GWAVA scores, which are scores for variants and corresponding regions that focus on predicting the functional impact of non-coding variants/regions <a href="https://www.sanger.ac.uk/sanger/StatGen_Gwava" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.sanger.ac.uk/sanger/StatGen_Gwava</a>).<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079605">41</a></sup></p></div></div></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Results</h2><p id="P20" class="p p-first">After quality control exclusions were applied, there were 670 CHOP CTD trios, 317 CHOP LVOTD trios, 406 CHOP CTD cases/2,976 pediatric controls, 355 PCGC CTD trios, and 192 PCGC LVOTD trios. The cohorts were characterized using counts and frequencies for race/ethnicity, sex, and heart defect lesion (<a href="/pmc/articles/PMC5539991/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The most frequent CTD was tetralogy of Fallot (30–40%) and the most common LVOTD was hypoplastic left heart syndrome (44–50%). Consistent with known epidemiological characteristics of LVOTDs and certain CTDs,<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079625">42</a></sup> there was an excess of males. After quality control exclusions, data were available for 4,756,722 SNPs (N=4,483,243 imputed) in the CHOP cohorts and for 5,737,343 SNPs (n=5,112,962 imputed) in the PCGC cohorts.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p6">Characteristics of cases by study sample.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140245424254224" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="9" rowspan="1" valign="bottom" align="center">N (%)
<hr></th></tr><tr><th colspan="9" rowspan="1" valign="bottom" align="left">
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th colspan="2" rowspan="1" valign="bottom" align="center">CHOP CTD trios(N=670 trios)</th><th colspan="2" rowspan="1" valign="bottom" align="center">CHOP LVOTD (N=317 trios)</th><th colspan="2" rowspan="1" valign="bottom" align="center">CHOP CTD cases<sup>*</sup>(N=406 cases)</th><th colspan="2" rowspan="1" valign="bottom" align="left">PCGC<sup>†</sup>(N=547 trios)</th></tr></thead><tbody><tr><td colspan="9" rowspan="1" valign="top" align="left">Race/ethnicity</td></tr><tr><td colspan="9" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Non-Hispanic Caucasian</td><td rowspan="1" colspan="1" valign="top" align="right">483</td><td rowspan="1" colspan="1" valign="top" align="left">(72.1)</td><td rowspan="1" colspan="1" valign="top" align="right">247</td><td rowspan="1" colspan="1" valign="top" align="left">(77.9)</td><td rowspan="1" colspan="1" valign="top" align="right">406</td><td rowspan="1" colspan="1" valign="top" align="left">(100.0)</td><td rowspan="1" colspan="1" valign="top" align="right">389</td><td rowspan="1" colspan="1" valign="top" align="left">(71.1)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Other</td><td rowspan="1" colspan="1" valign="top" align="right">187</td><td rowspan="1" colspan="1" valign="top" align="left">(27.9)</td><td rowspan="1" colspan="1" valign="top" align="right">70</td><td rowspan="1" colspan="1" valign="top" align="left">(22.1)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">(0.0)</td><td rowspan="1" colspan="1" valign="top" align="right">158</td><td rowspan="1" colspan="1" valign="top" align="left">(28.9)</td></tr><tr><td colspan="9" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="9" rowspan="1" valign="top" align="left">Sex</td></tr><tr><td colspan="9" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Male</td><td rowspan="1" colspan="1" valign="top" align="right">410</td><td rowspan="1" colspan="1" valign="top" align="left">(61.2)</td><td rowspan="1" colspan="1" valign="top" align="right">201</td><td rowspan="1" colspan="1" valign="top" align="left">(63.4)</td><td rowspan="1" colspan="1" valign="top" align="right">236</td><td rowspan="1" colspan="1" valign="top" align="left">(58.1)</td><td rowspan="1" colspan="1" valign="top" align="right">341</td><td rowspan="1" colspan="1" valign="top" align="left">(62.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Female</td><td rowspan="1" colspan="1" valign="top" align="right">260</td><td rowspan="1" colspan="1" valign="top" align="left">(38.8)</td><td rowspan="1" colspan="1" valign="top" align="right">116</td><td rowspan="1" colspan="1" valign="top" align="left">(36.6)</td><td rowspan="1" colspan="1" valign="top" align="right">170</td><td rowspan="1" colspan="1" valign="top" align="left">(41.9)</td><td rowspan="1" colspan="1" valign="top" align="right">206</td><td rowspan="1" colspan="1" valign="top" align="left">(37.7)</td></tr><tr><td colspan="9" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="9" rowspan="1" valign="top" align="left">Lesion</td></tr><tr><td colspan="9" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> CTDs</td><td rowspan="1" colspan="1" valign="top" align="right">670</td><td rowspan="1" colspan="1" valign="top" align="left">(100.0)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">406</td><td rowspan="1" colspan="1" valign="top" align="left">(100.0)</td><td rowspan="1" colspan="1" valign="top" align="right">355</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Tetralogy of Fallot</td><td rowspan="1" colspan="1" valign="top" align="right">267</td><td rowspan="1" colspan="1" valign="top" align="left">(39.9)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">134</td><td rowspan="1" colspan="1" valign="top" align="left">(33.0)</td><td rowspan="1" colspan="1" valign="top" align="right">106</td><td rowspan="1" colspan="1" valign="top" align="left">(29.9)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  D-transposition of the great arteries</td><td rowspan="1" colspan="1" valign="top" align="right">128</td><td rowspan="1" colspan="1" valign="top" align="left">(19.1)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">80</td><td rowspan="1" colspan="1" valign="top" align="left">(19.7)</td><td rowspan="1" colspan="1" valign="top" align="right">70</td><td rowspan="1" colspan="1" valign="top" align="left">(19.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Ventricular septal defects</td><td rowspan="1" colspan="1" valign="top" align="right">136</td><td rowspan="1" colspan="1" valign="top" align="left">(20.3)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">109</td><td rowspan="1" colspan="1" valign="top" align="left">(26.9)</td><td rowspan="1" colspan="1" valign="top" align="right">45</td><td rowspan="1" colspan="1" valign="top" align="left">(12.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Double outlet right ventricle</td><td rowspan="1" colspan="1" valign="top" align="right">76</td><td rowspan="1" colspan="1" valign="top" align="left">(11.3)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">25</td><td rowspan="1" colspan="1" valign="top" align="left">(6.2)</td><td rowspan="1" colspan="1" valign="top" align="right">57</td><td rowspan="1" colspan="1" valign="top" align="left">(16.1)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Isolated aortic arch anomalies</td><td rowspan="1" colspan="1" valign="top" align="right">30</td><td rowspan="1" colspan="1" valign="top" align="left">(4.5)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">22</td><td rowspan="1" colspan="1" valign="top" align="left">(5.4)</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="left">(2.0)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Truncus arteriosus</td><td rowspan="1" colspan="1" valign="top" align="right">20</td><td rowspan="1" colspan="1" valign="top" align="left">(3.0)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">19</td><td rowspan="1" colspan="1" valign="top" align="left">(4.7)</td><td rowspan="1" colspan="1" valign="top" align="right">13</td><td rowspan="1" colspan="1" valign="top" align="left">(3.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Interrupted aortic arch</td><td rowspan="1" colspan="1" valign="top" align="right">13</td><td rowspan="1" colspan="1" valign="top" align="left">(1.9)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">10</td><td rowspan="1" colspan="1" valign="top" align="left">(2.5)</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="left">(2.5)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Other</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">(0.0)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="left">(1.7)</td><td rowspan="1" colspan="1" valign="top" align="right">48</td><td rowspan="1" colspan="1" valign="top" align="left">(13.5)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> LVOTDs</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">317</td><td rowspan="1" colspan="1" valign="top" align="left">(100.0)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">192</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Hypoplastic left heart syndrome</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">157</td><td rowspan="1" colspan="1" valign="top" align="left">(49.5)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">85</td><td rowspan="1" colspan="1" valign="top" align="left">(44.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Coarctation of the aorta</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">97</td><td rowspan="1" colspan="1" valign="top" align="left">(30.6)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">67</td><td rowspan="1" colspan="1" valign="top" align="left">(34.9)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Aortic stenosis</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">63</td><td rowspan="1" colspan="1" valign="top" align="left">(19.9)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">28</td><td rowspan="1" colspan="1" valign="top" align="left">(14.6)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Other</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">(0.0)</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="left">(6.3)</td></tr></tbody></table></div><div id="largeobj_idm140245424254224" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5539991/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>From case-control analysis</div><div id="TFN2"><sup>†</sup>Percent of PCGC CTDs/LVOTs are listed for lesions</div></div></div><div id="S19" class="sec"><h3 id="S19title">GWAS of Individual Cohorts</h3><p id="P21" class="p p-first-last">A GWAS was conducted separately for each cohort (<a href="/pmc/articles/PMC5539991/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). The total numbers of genomewide significant and suggestive associations with SNPs observed in each cohort are summarized in <a href="/pmc/articles/PMC5539991/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. Genome-wide significant associations were only identified in the CHOP CTD case-control GWAS. In this analysis, 52 inherited genotypes were significantly associated with CTDs (<a href="/pmc/articles/PMC5539991/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table 1</a>), and 49 of these SNPs were in introns of <em>MGAT4C.</em> Multiple suggestive associations (p≤10<sup>−5</sup>) with the inherited and maternal genotypes were detected in the analysis of each cohort (<a href="/pmc/articles/PMC5539991/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). However, there was no overlap in the SNPs with at least suggestive evidence of association across comparable cohorts (data not shown). Thus, there was no evidence of replication for any of the suggestive or significant associations identified in the individual cohorts.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p9">Summary of analytic groups and results for inherited or maternal variants associated with heart defects</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140245424300320" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="top" align="left"></th><th colspan="4" rowspan="1" valign="bottom" align="center">CTD
<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">LVOTD
<hr></th><th rowspan="1" colspan="1" valign="bottom" align="center">CTD+LVOTD
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">CHOP trios</th><th rowspan="1" colspan="1" valign="top" align="center">CHOP case/control</th><th rowspan="1" colspan="1" valign="top" align="center">PCGC trios</th><th rowspan="1" colspan="1" valign="top" align="center">Meta-Analysis</th><th rowspan="1" colspan="1" valign="top" align="center">CHOP trios</th><th rowspan="1" colspan="1" valign="top" align="center">PCGC trios</th><th rowspan="1" colspan="1" valign="top" align="center">Meta-Analysis</th><th rowspan="1" colspan="1" valign="top" align="center">Meta-Analysis</th></tr></thead><tbody><tr><td colspan="9" rowspan="1" valign="top" align="left">Inherited</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Significant<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">52</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Suggestive<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="center">87</td><td rowspan="1" colspan="1" valign="top" align="center">45</td><td rowspan="1" colspan="1" valign="top" align="center">59</td><td rowspan="1" colspan="1" valign="top" align="center">36</td><td rowspan="1" colspan="1" valign="top" align="center">25</td><td rowspan="1" colspan="1" valign="top" align="center">40</td><td rowspan="1" colspan="1" valign="top" align="center">32</td><td rowspan="1" colspan="1" valign="top" align="center">70</td></tr><tr><td colspan="9" rowspan="1" valign="top" align="left">Maternal</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Significant<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Suggestive<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="center">80</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">39</td><td rowspan="1" colspan="1" valign="top" align="center">133</td><td rowspan="1" colspan="1" valign="top" align="center">122</td><td rowspan="1" colspan="1" valign="top" align="center">41</td><td rowspan="1" colspan="1" valign="top" align="center">67</td><td rowspan="1" colspan="1" valign="top" align="center">84</td></tr></tbody></table></div><div id="largeobj_idm140245424300320" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5539991/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>*</sup>p≤5×10<sup>−8</sup></div><div id="TFN4"><sup>†</sup>5×10<sup>−8</sup>&lt;p≤10<sup>−5</sup></div></div></div></div><div id="S20" class="sec sec-last"><h3 id="S20title">Meta-Analyses</h3><p id="P22" class="p p-first">There was suggestive evidence of associations with the maternal genotype for multiple SNPs in the meta-analysis of CTD only (133 SNPs in 11 regions), LVOTD only (67 SNPs in 10 regions, including one imputed SNP with borderline significant results, rs55788414, p=5.5×10<sup>−8</sup>, <a href="/pmc/articles/PMC5539991/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>) and CTD+LVOTD (84 SNPs in 20 regions) (<a href="/pmc/articles/PMC5539991/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>: SNPs with p&lt;10<sup>−6</sup>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Tables 2–4</a>: SNPs with p&lt;10<sup>−5</sup>). However, no genome-wide significant associations with the maternal genotype were identified (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 1</a>). Further, the maternal genotypes for SNPs in linkage disequilibrium with rs55788414, the SNP with the smallest association p-value, were not associated with LVOTD (<a href="/pmc/articles/PMC5539991/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5539991/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140245425189280" class="figure"><a href="/pmc/articles/PMC5539991/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5539991_nihms864518f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms864518f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5539991/bin/nihms864518f2.jpg"></a></div><div id="largeobj_idm140245425189280" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5539991/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5539991/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p5">Regional association plots for the meta-analysis between LVOTDs and: A) maternal SNPs near rs55788414 B) inherited SNPs near rs72820264. Each pane shows the association statistic from the meta-analysis (−log<sub>10</sub> p) on the left y-axis for the variant with the lowest combined p-value (purple diamond) and nearby markers (circles). The red shading indicates the amount of linkage disequilibrium (r<sup>2</sup>) between this variant and the nearby markers. The right y-axis indicates recombination rates from 1000 Genomes CEU data (blue lines). The x-axis indicates the chromosomal position (hg19) and the location of nearby genes.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p12">Summary data for variants in the meta-analysis with p&lt;10<sup>−6</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140245420755360" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="right">Chr<sup>*</sup></th><th rowspan="1" colspan="1" valign="top" align="left">Position<sup>†</sup> (bp)</th><th rowspan="1" colspan="1" valign="top" align="center">Reference allele</th><th rowspan="1" colspan="1" valign="top" align="center">Gene</th><th rowspan="1" colspan="1" valign="top" align="left">Function</th><th rowspan="1" colspan="1" valign="top" align="right"><em>P</em>-value<sup>‡</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Odds ratio<sup>§</sup>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="left">CADD<sup>||</sup></th><th rowspan="1" colspan="1" valign="top" align="left">GWAVA-TSS score<sup>#</sup></th></tr></thead><tbody><tr><td colspan="10" rowspan="1" valign="bottom" align="left"><strong>Inherited</strong>
<hr></td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left"> <strong>CTDs+LVODTs</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs11895588</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">141962157</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LRP1B</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="right">7.3 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.54 (0.43–0.69)</td><td rowspan="1" colspan="1" valign="top" align="left">0.85</td><td rowspan="1" colspan="1" valign="top" align="left">0.06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs56409046</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">9320306</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">2.7 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.34 (1.20–1.50)</td><td rowspan="1" colspan="1" valign="top" align="left">1.00</td><td rowspan="1" colspan="1" valign="top" align="left">0.23</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs72820285</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">9318701</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">6.6 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.38 (1.21–1.56)</td><td rowspan="1" colspan="1" valign="top" align="left">2.74</td><td rowspan="1" colspan="1" valign="top" align="left">0.22</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs56100674</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">9310391</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">7.8 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.37 (1.21–1.56)</td><td rowspan="1" colspan="1" valign="top" align="left">3.48</td><td rowspan="1" colspan="1" valign="top" align="left">0.41</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs111792270</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">9327938</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">8.1 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.38 (1.22–1.58)</td><td rowspan="1" colspan="1" valign="top" align="left">2.00</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs55720740</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">9309819</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">8.2 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.37 (1.21–1.55)</td><td rowspan="1" colspan="1" valign="top" align="left">1.18</td><td rowspan="1" colspan="1" valign="top" align="left">0.27</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs1176869</td><td rowspan="1" colspan="1" valign="top" align="right">14</td><td rowspan="1" colspan="1" valign="top" align="left">41276426</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">9.2 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.35 (1.20–1.52)</td><td rowspan="1" colspan="1" valign="top" align="left">5.41</td><td rowspan="1" colspan="1" valign="top" align="left">0.19</td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left"> <strong>CTDs</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs6886261</td><td rowspan="1" colspan="1" valign="top" align="right">5</td><td rowspan="1" colspan="1" valign="top" align="left">43022764</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LOC648987</em></td><td rowspan="1" colspan="1" valign="top" align="left">Non-coding</td><td rowspan="1" colspan="1" valign="top" align="right">1.7×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.78 (1.43–2.21)</td><td rowspan="1" colspan="1" valign="top" align="left">1.71</td><td rowspan="1" colspan="1" valign="top" align="left">0.20</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs585302</td><td rowspan="1" colspan="1" valign="top" align="right">5</td><td rowspan="1" colspan="1" valign="top" align="left">43034142</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LOC648987</em></td><td rowspan="1" colspan="1" valign="top" align="left">Non-coding</td><td rowspan="1" colspan="1" valign="top" align="right">1.7×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.78 (1.43–2.20)</td><td rowspan="1" colspan="1" valign="top" align="left">5.68</td><td rowspan="1" colspan="1" valign="top" align="left">0.55</td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left"> <strong>LVOTDs</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs11894932</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">99265186</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MGAT4A</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="right">1.5 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.44 (0.33–0.61)</td><td rowspan="1" colspan="1" valign="top" align="left">6.43</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs72820264</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">9291883</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.1 ×10<sup>−8</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center">2.05 (1.59–2.63)</td><td rowspan="1" colspan="1" valign="top" align="left">1.50</td><td rowspan="1" colspan="1" valign="top" align="left">0.19</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs72820266</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">9292316</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">4.9 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.86 (1.46–2.36)</td><td rowspan="1" colspan="1" valign="top" align="left">0.87</td><td rowspan="1" colspan="1" valign="top" align="left">0.35</td></tr><tr><td colspan="10" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left"><strong>Maternal</strong></td></tr><tr><td colspan="10" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left"> <strong>CTDs+LVOTDs</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs114952830</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">30850113</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">6.2 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.35 (1.20–1.52)</td><td rowspan="1" colspan="1" valign="top" align="left">5.08</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs75661265</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">60187791</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">5.0 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.17 (1.10–1.25)</td><td rowspan="1" colspan="1" valign="top" align="left">2.99</td><td rowspan="1" colspan="1" valign="top" align="left">0.47</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs55884872</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">60192814</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">5.5 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.18 (1.11–1.26)</td><td rowspan="1" colspan="1" valign="top" align="left">1.17</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs9643366</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">60190505</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">7.7 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.18 (1.10–1.25)</td><td rowspan="1" colspan="1" valign="top" align="left">11.35</td><td rowspan="1" colspan="1" valign="top" align="left">0.56</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs1941023</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="left">60275901</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">4.4 ×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.25 (1.15–1.36)</td><td rowspan="1" colspan="1" valign="top" align="left">1.10</td><td rowspan="1" colspan="1" valign="top" align="left">0.38</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs1245314</td><td rowspan="1" colspan="1" valign="top" align="right">14</td><td rowspan="1" colspan="1" valign="top" align="left">27492640</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">8.6×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.12 (1.07–1.17)</td><td rowspan="1" colspan="1" valign="top" align="left">0.80</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left"> <strong>CTDs</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs74461473</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="left">50250747</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SLC38A3</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="right">4.2×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.80 (0.73–0.87)</td><td rowspan="1" colspan="1" valign="top" align="left">14.85</td><td rowspan="1" colspan="1" valign="top" align="left">0.44</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs116734199</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">30808762</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Non-coding</td><td rowspan="1" colspan="1" valign="top" align="right">2.8×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.17 (1.10–1.25)</td><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs114952830</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">30850113</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">3.0×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.17 (1.10–1.25)</td><td rowspan="1" colspan="1" valign="top" align="left">5.08</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs114414726</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">30811164</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">5.2×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.11 (1.06–1.15)</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs2844660</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">30823760</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">7.8×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.41 (1.23–1.61)</td><td rowspan="1" colspan="1" valign="top" align="left">12.23</td><td rowspan="1" colspan="1" valign="top" align="left">0.46</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs115588720</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">30923527</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">8.5×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.04 (1.03–1.06)</td><td rowspan="1" colspan="1" valign="top" align="left">6.77</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs115946461</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">30837538</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">9.0×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.06 (1.03–1.08)</td><td rowspan="1" colspan="1" valign="top" align="left">0.71</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs116583468</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">30842674</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">9.7×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.07 (1.04–1.09)</td><td rowspan="1" colspan="1" valign="top" align="left">4.08</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs55884872</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">60192814</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">1.7×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.04 (1.02–1.06)</td><td rowspan="1" colspan="1" valign="top" align="left">1.18</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs75661265</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">60187791</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">2.2×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.19 (1.11–1.27)</td><td rowspan="1" colspan="1" valign="top" align="left">2.99</td><td rowspan="1" colspan="1" valign="top" align="left">0.47</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs9643366</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">60190505</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">3.4×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.39 (1.22–1.57)</td><td rowspan="1" colspan="1" valign="top" align="left">11.35</td><td rowspan="1" colspan="1" valign="top" align="left">0.56</td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left"> <strong>LVOTDs</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs1642645</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="left">42502281</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center"><em>HIVEP3</em></td><td rowspan="1" colspan="1" valign="top" align="left">5’ upstream</td><td rowspan="1" colspan="1" valign="top" align="right">7.8×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.53 (0.41–0.68)</td><td rowspan="1" colspan="1" valign="top" align="left">7.41</td><td rowspan="1" colspan="1" valign="top" align="left">0.55</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs141978609</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">168835335</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="right">8.6×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.50 (0.38–0.66)</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> rs55788414</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="left">81184939</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center"><em>PKD1L2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="right">5.5×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.82 (3.08–10.97)</td><td rowspan="1" colspan="1" valign="top" align="left">2.08</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td></tr></tbody></table></div><div id="largeobj_idm140245420755360" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5539991/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><sup>*</sup>Chromosome</div><div id="TFN6"><sup>†</sup>Hg19/NCBI build 37</div><div id="TFN7"><sup>‡</sup><em>P</em>-values only shown for top inherited or maternal effects (<em>p</em>&lt;1 × 10<sup>−6</sup>)</div><div id="TFN8"><sup>§</sup>Odds ratio estimate for carrying one copy of the high-risk allele compared to no copies, and corresponding 95% confidence interval</div><div id="TFN9"><sup>||</sup>Scaled Combined Annotation Dependent Depletion (CADD) score (variants with CADD&gt;10 are predicted to fall in the top 10% of the most deleterious variants in the genome)</div><div id="TFN10"><sup>#</sup>Genome Wide Annotation of VAriants (GWAVA) score for predicting the functional impact of non-coding variants/regions (range 0–1)</div></div></div><p id="P23">Meta-analyses of the inherited genotype for CTDs only and CTDs+LVOTDs also did not identify any genome-wide significant associations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 1</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 2</a>). However, there was suggestive evidence of associations with the inherited genotype for multiple SNPs in the meta-analysis of CTDs only (36 SNPs in 11 regions) and CTD+LVOTD (70 SNPs in 9 regions) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Tables 5–7</a>).</p><p id="P24" class="p p-last">One genome-wide significant association was identified in the meta-analysis of the inherited genotype for LVOTDs (rs72820264, p=2.1×10<sup>−8</sup>, minor allele frequency: 0.12 [1,000 Genomes release 17]). Seven additional SNPs in this region (6p24.3) also had suggestive evidence for association (p≤10<sup>−5</sup>) (<a href="/pmc/articles/PMC5539991/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). The LVOTD meta-analysis also identified 24 additional SNPs, in 14 regions, with suggestive evidence of associations (<a href="/pmc/articles/PMC5539991/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 1</a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table 7</a>).</p></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Discussion</h2><p id="P25" class="p p-first">Our GWAS of maternal and inherited genotypes and CTDs/LVOTDs in five independent cohorts identified several potential candidate regions. The majority of these signals were present in our case-control analysis, which was likely better powered than our family-based comparisons given the relatively large number of controls. It is noteworthy that nearly all of the genomewide significant SNPs in the CHOP case-control comparison were intronic variants in <em>MGAT4C,</em> which is involved in the biosynthesis of an N-glycan precursor. There are several congenital disorders of N-linked glycosylation, and there are reports of individuals with these conditions that have a CHD.<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079611">43</a>, <a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079622">44</a></sup> Two of the SNPs in <em>MGAT4C</em> had CADD scores &gt;10 (rs863392 and rs839163) and are thus predicted to be highly deleterious. However, the GWAVA-TSS scores for intronic SNPs in <em>MGAT4C</em> were all less than the median value for GWAS SNPs that have been externally replicated. Moreover, as these variants were not significantly related to CTDs in the meta-analysis, it may be that associations identified in our case-control represent false-positive findings arising from uncontrolled confounding (e.g., due to inadequate control for population stratification).</p><p id="P26">No association met our pairwise replication threshold of p≤10<sup>−5</sup> in more than one dataset assessing the same genetic effect (maternal or inherited) and phenotype. However, our meta-analyses identified several genes/regions of interest. We identified one inherited SNP (rs72820264, imputed) that was significantly associated with LVOTDs in our meta-analysis. This SNP is in an intergenic region on chromosome 6p24.3, 416kb from the nearest upstream gene (<em>MRDS1</em>/<em>OFCC1</em>) and 638kb from the nearest downstream gene (<em>HULC</em>). There were several additional SNPs in the region in strong linkage disequilibrium with rs72820264 that also had low p-values (<a href="/pmc/articles/PMC5539991/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>), including typed variants (e.g., rs7762096, p=6.2×10-5). There is a strong signal from an ENCODE clustered DNase I hypersensitivity site over 24 tissues, located at chr6:9293081-9293290, about 1.5kb away from this SNP (chr6:9291650). A DNAse I hypersensitivity site may indicate a promoter location within this primate-conserved region, because regulatory regions, especially promoters, tend to be DNase-sensitive.<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079620">45</a>, <a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079589">46</a></sup> However, GWAVA scores do not support a functional role for this SNP (0.2) or region (0.3). Also, the scaled CADD score was 1.5 for rs72820264, which suggests that this variant is not highly deleterious. The nearest downstream gene, <em>OFCC1</em> (Orofacial Cleft 1)<em>,</em> has been associated with orofacial clefts in several studies.<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079627">47</a>–<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079610">49</a></sup> Further, this region (6p24.3-21.2) has been linked to a rare autosomal dominant syndrome that includes heart anomalies (non-compaction of the ventricular myocardium, bradycardia, pulmonary valve stenosis, secundum atrial septal defect, left isomerism, heterotaxy).<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079626">50</a></sup> Thus, in silico data neither negates nor defines the potential contribution of this statistically associated SNP to disease risk.</p><p id="P27">The top suggestive associations from our meta-analyses included inherited variants in <em>LRP1B</em> (LDL receptor protein) and <em>MGAT4A</em> (regulates Golgi apparatus formation), as well as maternal variants in <em>SLC38A3</em> (glutamine transporter), <em>HIVEP3</em> (HIV enhancer-binding protein), and <em>PKD1L2</em> (polycystin protein involved in transmembrane domains). Of these, the intronic variant in <em>SLC38A3</em> was predicted to be within the most 10% of deleterious variants in the genome (scaled CADD score: 14.85) and had a GWAVA-TSS score that was within the third quartile of scores for GWAS SNPs that have been externally replicated. This gene is thought to play an important role in fetal development, as it likely is the main supplier of glutamine (necessary for fetal growth) to the fetus in early gestation.<sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079576">51</a></sup> The maternal intronic variant in <em>PKD1L2</em> had a borderline significant association with LVOTDs (p=5.5×10-8), but is not predicted to be deleterious based on CADD or GWAVA-TSS scores. However, no SNP in linkage disequilibrium with this PKD1L2 SNP was also associated.</p><p id="P28">The maternal SNP with the smallest association p-value, rs55788414, likely reflects a false positive finding, given that the maternal genotype for SNPs in linkage disequilibrium with the SNP with were not associated with LVOTD (<a href="/pmc/articles/PMC5539991/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>).</p><p id="P29">We had results for 16 SNPs that were previously identified in CHD GWAS, but none were significantly associated with the corresponding phenotype/effect in our meta-analysis (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 8</a>). However, some of the phenotypes included in previous GWAS did not directly correspond to those in our cohorts (e.g., tetralogy of Fallot versus conotruncal defects; septal defects versus CTDs+LVOTDs). For other traits (e.g., adult-onset conditions), it has not been uncommon for GWAS findings to fail to replicate in subsequent GWAS.<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079584">52</a></sup> Potential reasons for lack of replication across GWAS studies include population differences (e.g., three of the previous CHD GWAS were conducted among Chinese or European samples), genotyping platform/imputation differences, modeling differences, gene-environment or gene-gene interactions, and differences in case phenotypes (e.g., CTDs versus tetralogy of Fallot).</p><p id="P30">Our results should be considered in light of several limitations. We conducted nine GWAS analyses and six meta-analyses, using the typical GWAS threshold for significance of p&lt;5×10<sup>−8</sup> in each analysis. Given that we conducted multiple analyses, this threshold may have been too lenient. In addition, some of our cohorts were small, which limited our ability to detect SNPs that are only modestly associated with CHDs. In addition, our study subjects were recruited from a clinical rather than population-based setting and, thus, may not be representative of the general population of individuals affected by CHDs. Despite these limitations, our study also had several strengths including a relatively large sample size, as compared to other GWAS of CHDs, and the evaluation of both inherited and maternal genetic effects.</p><p id="P31" class="p p-last">In summary, we report on a number of potential candidate regions for some of the most common, severe heart defects. Our studies suggest that few, if any, single SNPs confer large risk of non-syndomic CTDs and/or LVOTDs. More work is needed to address the contribution of common SNPs that confer weak or moderate risk. For example, results from recent GWAS suggest that gene- or pathway-based studies may be required to identify genes that influence risk of CHDs via common variants.<sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_630079590">53</a></sup> Future work might therefore focus on the candidate loci highlighted by these meta-analyses and/or on gene-sets defined by functional or developmental pathways. Otherwise, substantially larger sample sets will be required to identify individual common variants that confer a small to moderate risk for CHDs and to identify risks specific to cardiac subtypes.</p></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="idm140245428890416title">Clinical Perspective</h3></div><p id="P32">Congenital heart defects (CHDs) are complex traits and familial recurrence patterns suggest that multiple, inherited genetic variants of low to moderate effect are likely to influence risk. The maternal genotype may also influence the risk of CHDs in offspring via an effect on the in utero environment. However, relatively little is known about the specific inherited or maternal genes that are associated with the risk of CHDs. In this study, we performed genome-wide association studies (GWAS) in five CHD cohorts, followed by meta-analyses of combined data. We evaluated inherited and maternal genotypes separately. In addition, cases with conotruncal defects and left ventricular obstructive defects were evaluated separately and together. While a number of suggestive associations were identified in the individual GWAS, none of these associations reached a GWAS level of significance. However, in our meta-analysis, one intra-genic, inherited SNP, rs72820264, was associated with left ventricular obstructive defects at a GWAS level of significance. The exact molecular mechanisms and pathways involved in this association are not yet clear. The results of this study will help guide future genomic efforts and will assist in narrowing down candidate regions of importance for the etiology of congenital heart defects.</p></div></div></div><div id="S23" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>001449 - Supplemental Material</h4><div class="sup-box half_rhythm" id="d36e743"><a href="/pmc/articles/PMC5539991/bin/NIHMS864518-supplement-001449_-_Supplemental_Material.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(8.0M, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>001449 - Supplemental Tables</h4><div class="sup-box half_rhythm" id="d36e747"><a href="/pmc/articles/PMC5539991/bin/NIHMS864518-supplement-001449_-_Supplemental_Tables.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(218K, xls)</sup></div></div></div><div id="S24" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S24title">Acknowledgments</h2><div class="sec"><p id="__p1">We thank the Pediatric Cardiac Genomics Consortium (members listed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Document 1</a>), Sharon Edman, Jennifer Garbarini, Stacy Woyciechowski, and Brande Latney for technical assistance, as well as the families who consented to participate in this study. Results for additional data (e.g., from the analyses within each individual cohort) may be requested by contacting the authors.</p><p id="__p2"><strong>Sources of Funding:</strong> This study was supported by grants from the Eunice Kennedy Shriver National Institute of Child Health and Human Development (P01HD070454), the National Heart, Lung, and Blood Institute including the Pediatric Cardiac Genomics Consortium (R21HL-098844, R01HL-076773, R01-HL74094, P50-HL74731, U01-<a id="__tag_802637515" class="tag_hotlink" href="/nuccore/HL098188" ref="/nuccore/HL098188"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL098188","term_id":"1051669497","term_text":"HL098188"}}</span>HL098188</a>, U01-<a id="__tag_802637514" class="tag_hotlink" href="/nuccore/HL098147" ref="/nuccore/HL098147"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL098147","term_id":"1051669456","term_text":"HL098147"}}</span>HL098147</a>, U01-<a id="__tag_802637513" class="tag_hotlink" href="/nuccore/HL098153" ref="/nuccore/HL098153"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL098153","term_id":"1051669462","term_text":"HL098153"}}</span>HL098153</a>, U01-<a id="__tag_802637519" class="tag_hotlink" href="/nuccore/HL098163" ref="/nuccore/HL098163"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL098163","term_id":"1051669472","term_text":"HL098163"}}</span>HL098163</a>, U01-<a id="__tag_802637518" class="tag_hotlink" href="/nuccore/HL098123" ref="/nuccore/HL098123"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL098123","term_id":"1051669432","term_text":"HL098123"}}</span>HL098123</a>, U01-<a id="__tag_802637517" class="tag_hotlink" href="/nuccore/HL098162" ref="/nuccore/HL098162"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL098162","term_id":"1051669471","term_text":"HL098162"}}</span>HL098162</a>), the Cardiovascular Development Consortium (U01-<a id="__tag_802637516" class="tag_hotlink" href="/nuccore/HL098166" ref="/nuccore/HL098166"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL098166","term_id":"1051669475","term_text":"HL098166"}}</span>HL098166</a>), the National Human Genome Research Institute (U54HG006504), the National Research Science Foundation (Award 1F30HL123238), and the National Center for Research Resources (M01-RR-000240, RR024134 [now the National Center for Advancing Translational Sciences (UL1TR000003)]). GWAS genotyping was funded by an Institutional Development Fund to The Center for Applied Genomics from The Children’s Hospital of Philadelphia. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH.</p></div></div><div id="idm140245431684688" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140245431684688title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p3" class="p p-first-last"><strong>Disclosures:</strong> None</p><p></p></div></div><div id="idm140245423318016" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140245423318016title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Botto LD, Lin AE, Riehle-Colarusso T, Malik S, Correa A. Seeking causes: Classifying and evaluating congenital heart defects in etiologic studies. <span><span class="ref-journal">Birth Defects Res A Clin Mol Teratol. </span>2007;<span class="ref-vol">79</span>:714–727.</span> [<a href="/pubmed/17729292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res+A+Clin+Mol+Teratol&amp;title=Seeking+causes:+Classifying+and+evaluating+congenital+heart+defects+in+etiologic+studies&amp;author=LD+Botto&amp;author=AE+Lin&amp;author=T+Riehle-Colarusso&amp;author=S+Malik&amp;author=A+Correa&amp;volume=79&amp;publication_year=2007&amp;pages=714-727&amp;pmid=17729292&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Rosano A, Botto LD, Botting B, Mastroiacovo P. Infant mortality and congenital anomalies from 1950 to 1994: An international perspective. <span><span class="ref-journal">J Epidemiol Community Health. </span>2000;<span class="ref-vol">54</span>:660–666.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1731756/">PMC free article</a>]</span> [<a href="/pubmed/10942444" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Epidemiol+Community+Health&amp;title=Infant+mortality+and+congenital+anomalies+from+1950+to+1994:+An+international+perspective&amp;author=A+Rosano&amp;author=LD+Botto&amp;author=B+Botting&amp;author=P+Mastroiacovo&amp;volume=54&amp;publication_year=2000&amp;pages=660-666&amp;pmid=10942444&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Perry LW, Neill CA, Ferencz C, Rubin JD, Loffredo CA.  Infants with congenital heart disease: The cases. In: Ferencz C, Rubin JD, Loffredo CA, Magee CA, editors. <span class="ref-journal">Epidemiology of congenital heart disease: The baltimore-washington infant study 1981–1989.</span> Mount Kisco: Futura Publishing Company, Inc; 1993. pp. 33–62. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Epidemiology+of+congenital+heart+disease:+The+baltimore-washington+infant+study+1981–1989&amp;author=LW+Perry&amp;author=CA+Neill&amp;author=C+Ferencz&amp;author=JD+Rubin&amp;author=CA+Loffredo&amp;publication_year=1993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Nora JJ, Nora AH. Genetic epidemiology of congenital heart diseases. <span><span class="ref-journal">Prog Med Genet. </span>1983;<span class="ref-vol">5</span>:91–137.</span> [<a href="/pubmed/6344145" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prog+Med+Genet&amp;title=Genetic+epidemiology+of+congenital+heart+diseases&amp;author=JJ+Nora&amp;author=AH+Nora&amp;volume=5&amp;publication_year=1983&amp;pages=91-137&amp;pmid=6344145&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Cripe L, Andelfinger G, Martin LJ, Shooner K, Benson DW. Bicuspid aortic valve is heritable. <span><span class="ref-journal">J Am Coll Cardiol. </span>2004;<span class="ref-vol">44</span>:138–143.</span> [<a href="/pubmed/15234422" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Coll+Cardiol&amp;title=Bicuspid+aortic+valve+is+heritable&amp;author=L+Cripe&amp;author=G+Andelfinger&amp;author=LJ+Martin&amp;author=K+Shooner&amp;author=DW+Benson&amp;volume=44&amp;publication_year=2004&amp;pages=138-143&amp;pmid=15234422&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Goldmuntz E, Clark BJ, Mitchell LE, Jawad AF, Cuneo BF, Reed L, et al.  Frequency of 22q11 deletions in patients with conotruncal defects. <span><span class="ref-journal">J Am Coll Cardiol. </span>1998;<span class="ref-vol">32</span>:492–498.</span> [<a href="/pubmed/9708481" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Coll+Cardiol&amp;title=Frequency+of+22q11+deletions+in+patients+with+conotruncal+defects&amp;author=E+Goldmuntz&amp;author=BJ+Clark&amp;author=LE+Mitchell&amp;author=AF+Jawad&amp;author=BF+Cuneo&amp;volume=32&amp;publication_year=1998&amp;pages=492-498&amp;pmid=9708481&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">McElhinney DB, Driscoll DA, Levin ER, Jawad AF, Emanuel BS, Goldmuntz E. Chromosome 22q11 deletion in patients with ventricular septal defect: Frequency and associated cardiovascular anomalies. <span><span class="ref-journal">Pediatrics. </span>2003;<span class="ref-vol">112</span>:e472.</span> [<a href="/pubmed/14654648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Chromosome+22q11+deletion+in+patients+with+ventricular+septal+defect:+Frequency+and+associated+cardiovascular+anomalies&amp;author=DB+McElhinney&amp;author=DA+Driscoll&amp;author=ER+Levin&amp;author=AF+Jawad&amp;author=BS+Emanuel&amp;volume=112&amp;publication_year=2003&amp;pages=e472&amp;pmid=14654648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Kwiatkowska J, Wierzba J, Aleszewicz-Baranowska J, Erecinski J. Genetic background of congenital conotruncal heart defects--a study of 45 families. <span><span class="ref-journal">Kardiol Pol. </span>2007;<span class="ref-vol">65</span>:32–37.</span> discussion 38–39. [<a href="/pubmed/17295158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kardiol+Pol&amp;title=Genetic+background+of+congenital+conotruncal+heart+defects--a+study+of+45+families&amp;author=J+Kwiatkowska&amp;author=J+Wierzba&amp;author=J+Aleszewicz-Baranowska&amp;author=J+Erecinski&amp;volume=65&amp;publication_year=2007&amp;pages=32-37&amp;pmid=17295158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Oyen N, Poulsen G, Boyd HA, Wohlfahrt J, Jensen PK, Melbye M. Recurrence of congenital heart defects in families. <span><span class="ref-journal">Circulation. </span>2009;<span class="ref-vol">120</span>:295–301.</span> [<a href="/pubmed/19597048" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Recurrence+of+congenital+heart+defects+in+families&amp;author=N+Oyen&amp;author=G+Poulsen&amp;author=HA+Boyd&amp;author=J+Wohlfahrt&amp;author=PK+Jensen&amp;volume=120&amp;publication_year=2009&amp;pages=295-301&amp;pmid=19597048&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Goldmuntz E, Woyciechowski S, Renstrom D, Lupo PJ, Mitchell LE. Variants of folate metabolism genes and the risk of conotruncal cardiac defects. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2008;<span class="ref-vol">1</span>:126–132.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3035562/">PMC free article</a>]</span> [<a href="/pubmed/20031554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Variants+of+folate+metabolism+genes+and+the+risk+of+conotruncal+cardiac+defects&amp;author=E+Goldmuntz&amp;author=S+Woyciechowski&amp;author=D+Renstrom&amp;author=PJ+Lupo&amp;author=LE+Mitchell&amp;volume=1&amp;publication_year=2008&amp;pages=126-132&amp;pmid=20031554&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Zhu H, Yang W, Lu W, Etheredge AJ, Lammer EJ, Finnell RH, et al.  Gene variants in the folate-mediated one-carbon metabolism (focm) pathway as risk factors for conotruncal heart defects. <span><span class="ref-journal">Am J Med Genet A. </span>2012;<span class="ref-vol">158A</span>:1124–1134.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3331895/">PMC free article</a>]</span> [<a href="/pubmed/22495907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=Gene+variants+in+the+folate-mediated+one-carbon+metabolism+(focm)+pathway+as+risk+factors+for+conotruncal+heart+defects&amp;author=H+Zhu&amp;author=W+Yang&amp;author=W+Lu&amp;author=AJ+Etheredge&amp;author=EJ+Lammer&amp;volume=158A&amp;publication_year=2012&amp;pages=1124-1134&amp;pmid=22495907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Hobbs CA, Cleves MA, Karim MA, Zhao W, MacLeod SL. Maternal folate-related gene environment interactions and congenital heart defects. <span><span class="ref-journal">Obstet Gynecol. </span>2010;<span class="ref-vol">116</span>:316–322.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3027124/">PMC free article</a>]</span> [<a href="/pubmed/20664391" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obstet+Gynecol&amp;title=Maternal+folate-related+gene+environment+interactions+and+congenital+heart+defects&amp;author=CA+Hobbs&amp;author=MA+Cleves&amp;author=MA+Karim&amp;author=W+Zhao&amp;author=SL+MacLeod&amp;volume=116&amp;publication_year=2010&amp;pages=316-322&amp;pmid=20664391&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Shaw GM, Lu W, Zhu H, Yang W, Briggs FB, Carmichael SL, et al.  118 snps of folate-related genes and risks of spina bifida and conotruncal heart defects. <span><span class="ref-journal">BMC Med Genet. </span>2009;<span class="ref-vol">10</span>:49.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2700092/">PMC free article</a>]</span> [<a href="/pubmed/19493349" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=118+snps+of+folate-related+genes+and+risks+of+spina+bifida+and+conotruncal+heart+defects&amp;author=GM+Shaw&amp;author=W+Lu&amp;author=H+Zhu&amp;author=W+Yang&amp;author=FB+Briggs&amp;volume=10&amp;publication_year=2009&amp;pages=49&amp;pmid=19493349&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Mitchell LE, Long J, Garbarini J, Paluru P, Goldmuntz E. Variants of folate metabolism genes and risk of left-sided cardiac defects. <span><span class="ref-journal">Birth Defects Res A Clin Mol Teratol. </span>2010;<span class="ref-vol">88</span>:48–53.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2860744/">PMC free article</a>]</span> [<a href="/pubmed/19777601" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res+A+Clin+Mol+Teratol&amp;title=Variants+of+folate+metabolism+genes+and+risk+of+left-sided+cardiac+defects&amp;author=LE+Mitchell&amp;author=J+Long&amp;author=J+Garbarini&amp;author=P+Paluru&amp;author=E+Goldmuntz&amp;volume=88&amp;publication_year=2010&amp;pages=48-53&amp;pmid=19777601&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Agopian AJ, Mitchell LE, Glessner J, Bhalla AD, Sewda A, Hakonarson H, et al.  Genome-wide association study of maternal and inherited loci for conotruncal heart defects. <span><span class="ref-journal">PLoS ONE. </span>2014;<span class="ref-vol">9</span>:e96057.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4011736/">PMC free article</a>]</span> [<a href="/pubmed/24800985" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Genome-wide+association+study+of+maternal+and+inherited+loci+for+conotruncal+heart+defects&amp;author=AJ+Agopian&amp;author=LE+Mitchell&amp;author=J+Glessner&amp;author=AD+Bhalla&amp;author=A+Sewda&amp;volume=9&amp;publication_year=2014&amp;pages=e96057&amp;pmid=24800985&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Cordell HJ, Topf A, Mamasoula C, Postma AV, Bentham J, Zelenika D, et al.  Genome-wide association study identifies loci on 12q24 and 13q32 associated with tetralogy of fallot. <span><span class="ref-journal">Hum Mol Genet. </span>2013;<span class="ref-vol">22</span>:1473–1481.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3596849/">PMC free article</a>]</span> [<a href="/pubmed/23297363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+study+identifies+loci+on+12q24+and+13q32+associated+with+tetralogy+of+fallot&amp;author=HJ+Cordell&amp;author=A+Topf&amp;author=C+Mamasoula&amp;author=AV+Postma&amp;author=J+Bentham&amp;volume=22&amp;publication_year=2013&amp;pages=1473-1481&amp;pmid=23297363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Mitchell LE, Agopian AJ, Bhalla A, Glessner JT, Kim CE, Swartz MD, et al.  Genome-wide association study of maternal and inherited effects on left-sided cardiac malformations. <span><span class="ref-journal">Hum Mol Genet. </span>2015;<span class="ref-vol">24</span>:265–73.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4326325/">PMC free article</a>]</span> [<a href="/pubmed/25138779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+study+of+maternal+and+inherited+effects+on+left-sided+cardiac+malformations&amp;author=LE+Mitchell&amp;author=AJ+Agopian&amp;author=A+Bhalla&amp;author=JT+Glessner&amp;author=CE+Kim&amp;volume=24&amp;publication_year=2015&amp;pages=265-73&amp;pmid=25138779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Cordell HJ, Bentham J, Topf A, Zelenika D, Heath S, Mamasoula C, et al.  Genome-wide association study of multiple congenital heart disease phenotypes identifies a susceptibility locus for atrial septal defect at chromosome 4p16. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:822–824.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3793630/">PMC free article</a>]</span> [<a href="/pubmed/23708191" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+multiple+congenital+heart+disease+phenotypes+identifies+a+susceptibility+locus+for+atrial+septal+defect+at+chromosome+4p16&amp;author=HJ+Cordell&amp;author=J+Bentham&amp;author=A+Topf&amp;author=D+Zelenika&amp;author=S+Heath&amp;volume=45&amp;publication_year=2013&amp;pages=822-824&amp;pmid=23708191&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Hu Z, Shi Y, Mo X, Xu J, Zhao B, Lin Y, et al.  A genome-wide association study identifies two risk loci for congenital heart malformations in han chinese populations. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:818–821.</span> [<a href="/pubmed/23708190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+identifies+two+risk+loci+for+congenital+heart+malformations+in+han+chinese+populations&amp;author=Z+Hu&amp;author=Y+Shi&amp;author=X+Mo&amp;author=J+Xu&amp;author=B+Zhao&amp;volume=45&amp;publication_year=2013&amp;pages=818-821&amp;pmid=23708190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Lin Y, Guo X, Zhao B, Liu J, Da M, Wen Y, et al.  Association analysis identifies new risk loci for congenital heart disease in chinese populations. <span><span class="ref-journal">Nat Commun. </span>2015;<span class="ref-vol">6</span>:8082.</span> [<a href="/pubmed/26283027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Association+analysis+identifies+new+risk+loci+for+congenital+heart+disease+in+chinese+populations&amp;author=Y+Lin&amp;author=X+Guo&amp;author=B+Zhao&amp;author=J+Liu&amp;author=M+Da&amp;volume=6&amp;publication_year=2015&amp;pages=8082&amp;pmid=26283027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Agopian AJ, Eastcott LM, Mitchell LE. Age of onset and effect size in genome-wide association studies. <span><span class="ref-journal">Birth Defects Res A Clin Mol Teratol. </span>2012;<span class="ref-vol">94</span>:908–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4219508/">PMC free article</a>]</span> [<a href="/pubmed/22933422" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res+A+Clin+Mol+Teratol&amp;title=Age+of+onset+and+effect+size+in+genome-wide+association+studies&amp;author=AJ+Agopian&amp;author=LM+Eastcott&amp;author=LE+Mitchell&amp;volume=94&amp;publication_year=2012&amp;pages=908-11&amp;pmid=22933422&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Goldmuntz E, Clark BJ, Mitchell LE, Jawad AF, Cuneo BF, Reed L, et al.  Frequency of 22q11 deletions in patients with conotruncal defects. <span><span class="ref-journal">J Am Coll Cardiol. </span>1998;<span class="ref-vol">32</span>:492–498.</span> [<a href="/pubmed/9708481" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Coll+Cardiol&amp;title=Frequency+of+22q11+deletions+in+patients+with+conotruncal+defects&amp;author=E+Goldmuntz&amp;author=BJ+Clark&amp;author=LE+Mitchell&amp;author=AF+Jawad&amp;author=BF+Cuneo&amp;volume=32&amp;publication_year=1998&amp;pages=492-498&amp;pmid=9708481&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Peyvandi S, Lupo PJ, Garbarini J, Woyciechowski S, Edman S, Emanuel BS, et al.  22q11.2 deletions in patients with conotruncal defects: Data from 1,610 consecutive cases. <span><span class="ref-journal">Pediatr Cardiol. </span>2013;<span class="ref-vol">34</span>:1687–1694.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4339067/">PMC free article</a>]</span> [<a href="/pubmed/23604262" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr+Cardiol&amp;title=22q11.2+deletions+in+patients+with+conotruncal+defects:+Data+from+1,610+consecutive+cases&amp;author=S+Peyvandi&amp;author=PJ+Lupo&amp;author=J+Garbarini&amp;author=S+Woyciechowski&amp;author=S+Edman&amp;volume=34&amp;publication_year=2013&amp;pages=1687-1694&amp;pmid=23604262&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">White PS, Xie HM, Werner P, Glessner J, Latney B, Hakonarson H, et al.  Analysis of chromosomal structural variation in patients with congenital left-sided cardiac lesions. <span><span class="ref-journal">Birth Defects Res A Clin Mol Teratol. </span>2014;<span class="ref-vol">100</span>:951–964.</span> [<a href="/pubmed/25066379" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res+A+Clin+Mol+Teratol&amp;title=Analysis+of+chromosomal+structural+variation+in+patients+with+congenital+left-sided+cardiac+lesions&amp;author=PS+White&amp;author=HM+Xie&amp;author=P+Werner&amp;author=J+Glessner&amp;author=B+Latney&amp;volume=100&amp;publication_year=2014&amp;pages=951-964&amp;pmid=25066379&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Gelb B, Brueckner M, Chung W, Goldmuntz E, Kaltman J, Kaski JP, et al.  The congenital heart disease genetic network study: Rationale, design, and early results. <span><span class="ref-journal">Circ Res. </span>2013;<span class="ref-vol">112</span>:698–706.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3679175/">PMC free article</a>]</span> [<a href="/pubmed/23410879" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=The+congenital+heart+disease+genetic+network+study:+Rationale,+design,+and+early+results&amp;author=B+Gelb&amp;author=M+Brueckner&amp;author=W+Chung&amp;author=E+Goldmuntz&amp;author=J+Kaltman&amp;volume=112&amp;publication_year=2013&amp;pages=698-706&amp;pmid=23410879&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">International HapMap Consortium. The international hapmap project. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">426</span>:789–796.</span> [<a href="/pubmed/14685227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+international+hapmap+project&amp;volume=426&amp;publication_year=2003&amp;pages=789-796&amp;pmid=14685227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, Handsaker RE, et al.  An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=GR+Abecasis&amp;author=A+Auton&amp;author=LD+Brooks&amp;author=MA+DePristo&amp;author=RM+Durbin&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  Plink: A tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Plink:+A+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Delaneau O, Marchini J, Zagury JF. A linear complexity phasing method for thousands of genomes. <span><span class="ref-journal">Nat Methods. </span>2012;<span class="ref-vol">9</span>:179–181.</span> [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes&amp;author=O+Delaneau&amp;author=J+Marchini&amp;author=JF+Zagury&amp;volume=9&amp;publication_year=2012&amp;pages=179-181&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Howie B, Marchini J, Stephens M. Genotype imputation with thousands of genomes. <span><span class="ref-journal">G3 (Bethesda) </span>2011;<span class="ref-vol">1</span>:457–470.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3276165/">PMC free article</a>]</span> [<a href="/pubmed/22384356" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=G3+(Bethesda)&amp;title=Genotype+imputation+with+thousands+of+genomes&amp;author=B+Howie&amp;author=J+Marchini&amp;author=M+Stephens&amp;volume=1&amp;publication_year=2011&amp;pages=457-470&amp;pmid=22384356&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, Gibbs RA, et al.  A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;author=GR+Abecasis&amp;author=D+Altshuler&amp;author=A+Auton&amp;author=LD+Brooks&amp;author=RM+Durbin&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Ainsworth HF, Unwin J, Jamison DL, Cordell HJ. Investigation of maternal effects, maternal-fetal interactions and parent-of-origin effects (imprinting), using mothers and their offspring. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>:19–45.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025173/">PMC free article</a>]</span> [<a href="/pubmed/21181895" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Investigation+of+maternal+effects,+maternal-fetal+interactions+and+parent-of-origin+effects+(imprinting),+using+mothers+and+their+offspring&amp;author=HF+Ainsworth&amp;author=J+Unwin&amp;author=DL+Jamison&amp;author=HJ+Cordell&amp;volume=35&amp;publication_year=2011&amp;pages=19-45&amp;pmid=21181895&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Howey R, Cordell HJ. Premim and emim: Tools for estimation of maternal, imprinting and interaction effects using multinomial modelling. <span><span class="ref-journal">BMC Bioinformatics. </span>2012;<span class="ref-vol">13</span>:149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3464602/">PMC free article</a>]</span> [<a href="/pubmed/22738121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=Premim+and+emim:+Tools+for+estimation+of+maternal,+imprinting+and+interaction+effects+using+multinomial+modelling&amp;author=R+Howey&amp;author=HJ+Cordell&amp;volume=13&amp;publication_year=2012&amp;pages=149&amp;pmid=22738121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Wilcox AJ, Weinberg CR, Lie RT. Distinguishing the effects of maternal and offspring genes through studies of “case-parent triads” <span><span class="ref-journal">Am J Epidemiol. </span>1998;<span class="ref-vol">148</span>:893–901.</span> [<a href="/pubmed/9801020" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Distinguishing+the+effects+of+maternal+and+offspring+genes+through+studies+of+“case-parent+triads”&amp;author=AJ+Wilcox&amp;author=CR+Weinberg&amp;author=RT+Lie&amp;volume=148&amp;publication_year=1998&amp;pages=893-901&amp;pmid=9801020&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Benyamin B, Visscher PM, McRae AF. Family-based genome-wide association studies. <span><span class="ref-journal">Pharmacogenomics. </span>2009;<span class="ref-vol">10</span>:181–190.</span> [<a href="/pubmed/19207019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&amp;title=Family-based+genome-wide+association+studies&amp;author=B+Benyamin&amp;author=PM+Visscher&amp;author=AF+McRae&amp;volume=10&amp;publication_year=2009&amp;pages=181-190&amp;pmid=19207019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Mitchell LE, Weinberg CR. Evaluation of offspring and maternal genetic effects on disease risk using a family-based approach: The “pent” design. <span><span class="ref-journal">Am J Epidemiol. </span>2005;<span class="ref-vol">162</span>:676–685.</span> [<a href="/pubmed/16093287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Evaluation+of+offspring+and+maternal+genetic+effects+on+disease+risk+using+a+family-based+approach:+The+“pent”+design&amp;author=LE+Mitchell&amp;author=CR+Weinberg&amp;volume=162&amp;publication_year=2005&amp;pages=676-685&amp;pmid=16093287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Duggal P, Gillanders EM, Holmes TN, Bailey-Wilson JE. Establishing an adjusted p-value threshold to control the family-wide type 1 error in genome wide association studies. <span><span class="ref-journal">BMC Genomics. </span>2008;<span class="ref-vol">9</span>:516.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2621212/">PMC free article</a>]</span> [<a href="/pubmed/18976480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Establishing+an+adjusted+p-value+threshold+to+control+the+family-wide+type+1+error+in+genome+wide+association+studies&amp;author=P+Duggal&amp;author=EM+Gillanders&amp;author=TN+Holmes&amp;author=JE+Bailey-Wilson&amp;volume=9&amp;publication_year=2008&amp;pages=516&amp;pmid=18976480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Magi R, Morris AP. Gwama: Software for genome-wide association meta-analysis. <span><span class="ref-journal">BMC Bioinformatics. </span>2010;<span class="ref-vol">11</span>:288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2893603/">PMC free article</a>]</span> [<a href="/pubmed/20509871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=Gwama:+Software+for+genome-wide+association+meta-analysis&amp;author=R+Magi&amp;author=AP+Morris&amp;volume=11&amp;publication_year=2010&amp;pages=288&amp;pmid=20509871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, et al.  Locuszoom: Regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Locuszoom:+Regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Kircher M, Witten DM, Jain P, O’Roak BJ, Cooper GM, Shendure J. A general framework for estimating the relative pathogenicity of human genetic variants. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:310–315.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3992975/">PMC free article</a>]</span> [<a href="/pubmed/24487276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+general+framework+for+estimating+the+relative+pathogenicity+of+human+genetic+variants&amp;author=M+Kircher&amp;author=DM+Witten&amp;author=P+Jain&amp;author=BJ+O’Roak&amp;author=GM+Cooper&amp;volume=46&amp;publication_year=2014&amp;pages=310-315&amp;pmid=24487276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Ritchie GR, Dunham I, Zeggini E, Flicek P. Functional annotation of noncoding sequence variants. <span><span class="ref-journal">Nat Methods. </span>2014;<span class="ref-vol">11</span>:294–296.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5015703/">PMC free article</a>]</span> [<a href="/pubmed/24487584" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Functional+annotation+of+noncoding+sequence+variants&amp;author=GR+Ritchie&amp;author=I+Dunham&amp;author=E+Zeggini&amp;author=P+Flicek&amp;volume=11&amp;publication_year=2014&amp;pages=294-296&amp;pmid=24487584&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Long J, Ramadhani T, Mitchell LE. Epidemiology of nonsyndromic conotruncal heart defects in texas, 1999–2004. <span><span class="ref-journal">Birth Defects Res A Clin Mol Teratol. </span>2010;<span class="ref-vol">88</span>:971–979.</span> [<a href="/pubmed/20878913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res+A+Clin+Mol+Teratol&amp;title=Epidemiology+of+nonsyndromic+conotruncal+heart+defects+in+texas,+1999–2004&amp;author=J+Long&amp;author=T+Ramadhani&amp;author=LE+Mitchell&amp;volume=88&amp;publication_year=2010&amp;pages=971-979&amp;pmid=20878913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Kranz C, Basinger AA, Gucsavas-Calikoglu M, Sun L, Powell CM, Henderson FW, et al.  Expanding spectrum of congenital disorder of glycosylation ig (cdg-ig): Sibs with a unique skeletal dysplasia, hypogammaglobulinemia, cardiomyopathy, genital malformations, and early lethality. <span><span class="ref-journal">Am J Med Genet A. </span>2007;<span class="ref-vol">143A</span>:1371–1378.</span> [<a href="/pubmed/17506107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+A&amp;title=Expanding+spectrum+of+congenital+disorder+of+glycosylation+ig+(cdg-ig):+Sibs+with+a+unique+skeletal+dysplasia,+hypogammaglobulinemia,+cardiomyopathy,+genital+malformations,+and+early+lethality&amp;author=C+Kranz&amp;author=AA+Basinger&amp;author=M+Gucsavas-Calikoglu&amp;author=L+Sun&amp;author=CM+Powell&amp;volume=143A&amp;publication_year=2007&amp;pages=1371-1378&amp;pmid=17506107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Cantagrel V, Lefeber DJ, Ng BG, Guan Z, Silhavy JL, Bielas SL, et al.  Srd5a3 is required for converting polyprenol to dolichol and is mutated in a congenital glycosylation disorder. <span><span class="ref-journal">Cell. </span>2010;<span class="ref-vol">142</span>:203–217.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2940322/">PMC free article</a>]</span> [<a href="/pubmed/20637498" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Srd5a3+is+required+for+converting+polyprenol+to+dolichol+and+is+mutated+in+a+congenital+glycosylation+disorder&amp;author=V+Cantagrel&amp;author=DJ+Lefeber&amp;author=BG+Ng&amp;author=Z+Guan&amp;author=JL+Silhavy&amp;volume=142&amp;publication_year=2010&amp;pages=203-217&amp;pmid=20637498&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:57–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3439153/">PMC free article</a>]</span> [<a href="/pubmed/22955616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+encyclopedia+of+DNA+elements+in+the+human+genome&amp;volume=489&amp;publication_year=2012&amp;pages=57-74&amp;pmid=22955616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Rosenbloom KR, Sloan CA, Malladi VS, Dreszer TR, Learned K, Kirkup VM, et al.  Encode data in the ucsc genome browser: Year 5 update. <span><span class="ref-journal">Nucleic Acids Res. </span>2013;<span class="ref-vol">41</span>:D56–63.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3531152/">PMC free article</a>]</span> [<a href="/pubmed/23193274" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Encode+data+in+the+ucsc+genome+browser:+Year+5+update&amp;author=KR+Rosenbloom&amp;author=CA+Sloan&amp;author=VS+Malladi&amp;author=TR+Dreszer&amp;author=K+Learned&amp;volume=41&amp;publication_year=2013&amp;pages=D56-63&amp;pmid=23193274&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Salahshourifar I, Halim AS, Sulaiman WA, Zilfalil BA. Contribution of 6p24 to non-syndromic cleft lip and palate in a malay population: Association of variants in ofc1. <span><span class="ref-journal">J Dent Res. </span>2011;<span class="ref-vol">90</span>:387–391.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3318024/">PMC free article</a>]</span> [<a href="/pubmed/21297019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Dent+Res&amp;title=Contribution+of+6p24+to+non-syndromic+cleft+lip+and+palate+in+a+malay+population:+Association+of+variants+in+ofc1&amp;author=I+Salahshourifar&amp;author=AS+Halim&amp;author=WA+Sulaiman&amp;author=BA+Zilfalil&amp;volume=90&amp;publication_year=2011&amp;pages=387-391&amp;pmid=21297019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Chen CP, Tzen CY, Chern SR, Tsai FJ, Hsu CY, Lee CC, et al.  A 12 mb deletion of 6p24.1--&gt;pter in an 18-gestational-week fetus with orofacial clefting, the dandy-walker malformation and bilateral multicystic kidneys. <span><span class="ref-journal">Eur J Med Genet. </span>2009;<span class="ref-vol">52</span>:59–61.</span> [<a href="/pubmed/19049908" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Med+Genet&amp;title=A+12+mb+deletion+of+6p24.1-->pter+in+an+18-gestational-week+fetus+with+orofacial+clefting,+the+dandy-walker+malformation+and+bilateral+multicystic+kidneys&amp;author=CP+Chen&amp;author=CY+Tzen&amp;author=SR+Chern&amp;author=FJ+Tsai&amp;author=CY+Hsu&amp;volume=52&amp;publication_year=2009&amp;pages=59-61&amp;pmid=19049908&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Scapoli L, Pezzetti F, Carinci F, Martinelli M, Carinci P, Tognon M. Evidence of linkage to 6p23 and genetic heterogeneity in nonsyndromic cleft lip with or without cleft palate. <span><span class="ref-journal">Genomics. </span>1997;<span class="ref-vol">43</span>:216–220.</span> [<a href="/pubmed/9244439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Evidence+of+linkage+to+6p23+and+genetic+heterogeneity+in+nonsyndromic+cleft+lip+with+or+without+cleft+palate&amp;author=L+Scapoli&amp;author=F+Pezzetti&amp;author=F+Carinci&amp;author=M+Martinelli&amp;author=P+Carinci&amp;volume=43&amp;publication_year=1997&amp;pages=216-220&amp;pmid=9244439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Wessels MW, De Graaf BM, Cohen-Overbeek TE, Spitaels SE, de Groot-de Laat LE, Ten Cate FJ, et al.  A new syndrome with noncompaction cardiomyopathy, bradycardia, pulmonary stenosis, atrial septal defect and heterotaxy with suggestive linkage to chromosome 6p. <span><span class="ref-journal">Hum Genet. </span>2008;<span class="ref-vol">122</span>:595–603.</span> [<a href="/pubmed/17938964" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=A+new+syndrome+with+noncompaction+cardiomyopathy,+bradycardia,+pulmonary+stenosis,+atrial+septal+defect+and+heterotaxy+with+suggestive+linkage+to+chromosome+6p&amp;author=MW+Wessels&amp;author=BM+De+Graaf&amp;author=TE+Cohen-Overbeek&amp;author=SE+Spitaels&amp;author=LE+de+Groot-de+Laat&amp;volume=122&amp;publication_year=2008&amp;pages=595-603&amp;pmid=17938964&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Balkrishna S, Broer A, Welford SM, Hatzoglou M, Broer S. Expression of glutamine transporter slc38a3 (snat3) during acidosis is mediated by a different mechanism than tissue-specific expression. <span><span class="ref-journal">Cell Physiol Biochem. </span>2014;<span class="ref-vol">33</span>:1591–1606.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4424794/">PMC free article</a>]</span> [<a href="/pubmed/24854847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Physiol+Biochem&amp;title=Expression+of+glutamine+transporter+slc38a3+(snat3)+during+acidosis+is+mediated+by+a+different+mechanism+than+tissue-specific+expression&amp;author=S+Balkrishna&amp;author=A+Broer&amp;author=SM+Welford&amp;author=M+Hatzoglou&amp;author=S+Broer&amp;volume=33&amp;publication_year=2014&amp;pages=1591-1606&amp;pmid=24854847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Greene CS, Penrod NM, Williams SM, Moore JH. Failure to replicate a genetic association may provide important clues about genetic architecture. <span><span class="ref-journal">PLoS ONE. </span>2009;<span class="ref-vol">4</span>:e5639.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2685469/">PMC free article</a>]</span> [<a href="/pubmed/19503614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Failure+to+replicate+a+genetic+association+may+provide+important+clues+about+genetic+architecture&amp;author=CS+Greene&amp;author=NM+Penrod&amp;author=SM+Williams&amp;author=JH+Moore&amp;volume=4&amp;publication_year=2009&amp;pages=e5639&amp;pmid=19503614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Hagg S, Ganna A, Van Der Laan SW, Esko T, Pers TH, Locke AE, et al.  Gene-based meta-analysis of genome-wide association studies implicates new loci involved in obesity. <span><span class="ref-journal">Hum Mol Genet. </span>2015;<span class="ref-vol">24</span>:6849–6860.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4643645/">PMC free article</a>]</span> [<a href="/pubmed/26376864" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Gene-based+meta-analysis+of+genome-wide+association+studies+implicates+new+loci+involved+in+obesity&amp;author=S+Hagg&amp;author=A+Ganna&amp;author=SW+Van+Der+Laan&amp;author=T+Esko&amp;author=TH+Pers&amp;volume=24&amp;publication_year=2015&amp;pages=6849-6860&amp;pmid=26376864&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5539991&amp;issue-id=296275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140245423864560title">Abstract</a></li><li><a href="#S4title">Introduction</a></li><li><a href="#S5title">Methods</a></li><li><a href="#S18title">Results</a></li><li><a href="#S21title">Discussion</a></li><li><a href="#S23title">Supplementary Material</a></li><li><a href="#S24title">Acknowledgments</a></li><li><a href="#idm140245431684688title">Footnotes</a></li><li><a href="#idm140245423318016title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5539991/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5539991/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5539991/pdf/nihms864518.pdf">PDF (468K)</a></li> | <li><a href="#" data-citationid="PMC5539991" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5539991%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5539991%2F&amp;text=Genome-Wide%20Association%20Studies%20and%20Meta-Analyses%20for%20Congenital%20Heart%20Defects" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5539991%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28468790" data-aiid="5539991" data-aid="5539991" data-iid="296275" data-domainid="319" data-domain="nihpa" data-accid="PMC5539991" data-md5="974b49c2dcb9635816fd743867c16fcf"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:12:41-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B0FD8DD5AB1410000000000BF0070&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5539991%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5539991/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>