<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5884124/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>An Unexpectedly Complex Architecture for Skin Pigmentation in Africans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5884124/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Cell"><meta name="citation_title" content="An Unexpectedly Complex Architecture for Skin Pigmentation in Africans"><meta name="citation_authors" content="Alicia R Martin, Meng Lin, Julie M Granka, Justin W Myrick, Xiaomin Liu, Alexandra Sockell, Elizabeth G Atkinson, Cedric J Werely, Marlo Möller, Manjinder S Sandhu, David M Kingsley, Eileen G Hoal, Xiao Liu, Mark J Daly, Marcus W Feldman, Christopher R Gignoux, Carlos D Bustamante, Brenna M Henn"><meta name="citation_date" content="30 November 2017"><meta name="citation_issue" content="6"><meta name="citation_volume" content="171"><meta name="citation_firstpage" content="1340"><meta name="citation_doi" content="10.1016/j.cell.2017.11.015"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5884124/?report=abstract"><meta name="citation_pmid" content="29195075"><meta name="DC.Title" content="An Unexpectedly Complex Architecture for Skin Pigmentation in Africans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Alicia R Martin"><meta name="DC.Contributor" content="Meng Lin"><meta name="DC.Contributor" content="Julie M Granka"><meta name="DC.Contributor" content="Justin W Myrick"><meta name="DC.Contributor" content="Xiaomin Liu"><meta name="DC.Contributor" content="Alexandra Sockell"><meta name="DC.Contributor" content="Elizabeth G Atkinson"><meta name="DC.Contributor" content="Cedric J Werely"><meta name="DC.Contributor" content="Marlo Möller"><meta name="DC.Contributor" content="Manjinder S Sandhu"><meta name="DC.Contributor" content="David M Kingsley"><meta name="DC.Contributor" content="Eileen G Hoal"><meta name="DC.Contributor" content="Xiao Liu"><meta name="DC.Contributor" content="Mark J Daly"><meta name="DC.Contributor" content="Marcus W Feldman"><meta name="DC.Contributor" content="Christopher R Gignoux"><meta name="DC.Contributor" content="Carlos D Bustamante"><meta name="DC.Contributor" content="Brenna M Henn"><meta name="DC.Date" content="2017 Nov 30"><meta name="DC.Identifier" content="10.1016/j.cell.2017.11.015"><meta name="DC.Language" content="en"><meta property="og:title" content="An Unexpectedly Complex Architecture for Skin Pigmentation in Africans"><meta property="og:type" content="article"><meta property="og:description" content="Approximately 15 genes have been directly associated with skin pigmentation variation in humans, leading to its characterization as a relatively simple trait. However, by assembling a global survey of quantitative skin pigmentation phenotypes, we demonstrate ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5884124/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5884124/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5884124/pdf/nihms921744.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BD7F6DD5A84C10000000006E00439.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/element/mml/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/input/MathML/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/jax.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5884124%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="mesh">MeSH</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5884124</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Cell</a></span>. Author manuscript; available in PMC 2018 Nov 30.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=29195075" target="pmc_ext" ref="reftype=publisher&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Cell. 2017 Nov 30; 171(6): 1340–1353.e14. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.cell.2017.11.015" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.cell.2017.11.015</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5884124</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS921744</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29195075">29195075</a></div></div></div></div><h1 class="content-title">An Unexpectedly Complex Architecture for Skin Pigmentation in Africans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Martin%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158324827408" co-class="co-affbox">Alicia R Martin</a>,<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>*</sup> <a href="/pubmed/?term=Lin%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158312311376" co-class="co-affbox">Meng Lin</a>,<sup>5</sup> <a href="/pubmed/?term=Granka%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158361364464" co-class="co-affbox">Julie M Granka</a>,<sup>6</sup> <a href="/pubmed/?term=Myrick%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158361362224" co-class="co-affbox">Justin W Myrick</a>,<sup>5</sup> <a href="/pubmed/?term=Liu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158363843472" co-class="co-affbox">Xiaomin Liu</a>,<sup>7</sup> <a href="/pubmed/?term=Sockell%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158314944848" co-class="co-affbox">Alexandra Sockell</a>,<sup>1</sup> <a href="/pubmed/?term=Atkinson%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158314942672" co-class="co-affbox">Elizabeth G Atkinson</a>,<sup>5</sup> <a href="/pubmed/?term=Werely%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158356309120" co-class="co-affbox">Cedric J Werely</a>,<sup>9</sup> <a href="/pubmed/?term=M%26%23x000f6%3Bller%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158312738864" co-class="co-affbox">Marlo Möller</a>,<sup>9</sup> <a href="/pubmed/?term=Sandhu%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158312736688" co-class="co-affbox">Manjinder S Sandhu</a>,<sup>10</sup> <a href="/pubmed/?term=Kingsley%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158363451936" co-class="co-affbox">David M Kingsley</a>,<sup>8</sup> <a href="/pubmed/?term=Hoal%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158316314160" co-class="co-affbox">Eileen G Hoal</a>,<sup>9</sup> <a href="/pubmed/?term=Liu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158316311984" co-class="co-affbox">Xiao Liu</a>,<sup>7</sup> <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158312660720" co-class="co-affbox">Mark J Daly</a>,<sup>2,</sup><sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Feldman%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158317577280" co-class="co-affbox">Marcus W Feldman</a>,<sup>6</sup> <a href="/pubmed/?term=Gignoux%20CR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158320930160" co-class="co-affbox">Christopher R Gignoux</a>,<sup>1</sup> <a href="/pubmed/?term=Bustamante%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158320927984" co-class="co-affbox">Carlos D Bustamante</a>,<sup>1</sup> and  <a href="/pubmed/?term=Henn%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075" class="affpopup" co-rid="_co_idm140158312416528" co-class="co-affbox">Brenna M Henn</a><sup>5,</sup><sup>11,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140158324827408"><h3 class="no_margin">Alicia R Martin</h3><p>  <sup>1</sup>Department of Genetics, Stanford University, Stanford, CA, 94305</p><p>  <sup>2</sup>Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114</p><p>  <sup>3</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, MA 02141</p><p>  <sup>4</sup>Stanley Center for Psychiatric Research, Broad Institute, Cambridge, MA 02141</p><div>Find articles by <a href="/pubmed/?term=Martin%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Alicia R Martin</a></div></div><div id="_co_idm140158312311376"><h3 class="no_margin">Meng Lin</h3><p>  <sup>5</sup>Department of Ecology and Evolution, SUNY Stony Brook, NY 11794</p><div>Find articles by <a href="/pubmed/?term=Lin%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Meng Lin</a></div></div><div id="_co_idm140158361364464"><h3 class="no_margin">Julie M Granka</h3><p>  <sup>6</sup>Department of Biological Sciences, Stanford University, Stanford, CA, 94305</p><div>Find articles by <a href="/pubmed/?term=Granka%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Julie M Granka</a></div></div><div id="_co_idm140158361362224"><h3 class="no_margin">Justin W Myrick</h3><p>  <sup>5</sup>Department of Ecology and Evolution, SUNY Stony Brook, NY 11794</p><div>Find articles by <a href="/pubmed/?term=Myrick%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Justin W Myrick</a></div></div><div id="_co_idm140158363843472"><h3 class="no_margin">Xiaomin Liu</h3><p>  <sup>7</sup>BGI-Shenzhen, Shenzhen, China</p><div>Find articles by <a href="/pubmed/?term=Liu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Xiaomin Liu</a></div></div><div id="_co_idm140158314944848"><h3 class="no_margin">Alexandra Sockell</h3><p>  <sup>1</sup>Department of Genetics, Stanford University, Stanford, CA, 94305</p><div>Find articles by <a href="/pubmed/?term=Sockell%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Alexandra Sockell</a></div></div><div id="_co_idm140158314942672"><h3 class="no_margin">Elizabeth G Atkinson</h3><p>  <sup>5</sup>Department of Ecology and Evolution, SUNY Stony Brook, NY 11794</p><div>Find articles by <a href="/pubmed/?term=Atkinson%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Elizabeth G Atkinson</a></div></div><div id="_co_idm140158356309120"><h3 class="no_margin">Cedric J Werely</h3><p>  <sup>9</sup>SA MRC Centre for Tuberculosis Research, DST/NRF Centre of Excellence for Biomedical Tuberculosis Research, Division of Molecular Biology and Human Genetics, Faculty of Medicine and Health Sciences, Stellenbosch University, Tygerberg, South Africa</p><div>Find articles by <a href="/pubmed/?term=Werely%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Cedric J Werely</a></div></div><div id="_co_idm140158312738864"><h3 class="no_margin">Marlo Möller</h3><p>  <sup>9</sup>SA MRC Centre for Tuberculosis Research, DST/NRF Centre of Excellence for Biomedical Tuberculosis Research, Division of Molecular Biology and Human Genetics, Faculty of Medicine and Health Sciences, Stellenbosch University, Tygerberg, South Africa</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000f6%3Bller%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Marlo Möller</a></div></div><div id="_co_idm140158312736688"><h3 class="no_margin">Manjinder S Sandhu</h3><p>  <sup>10</sup>Wellcome Trust Sanger Institute, Genome Campus, Hinxton, England</p><div>Find articles by <a href="/pubmed/?term=Sandhu%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Manjinder S Sandhu</a></div></div><div id="_co_idm140158363451936"><h3 class="no_margin">David M Kingsley</h3><p>  <sup>8</sup>Department of Developmental Biology, Stanford University, Stanford, California, USA</p><div>Find articles by <a href="/pubmed/?term=Kingsley%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">David M Kingsley</a></div></div><div id="_co_idm140158316314160"><h3 class="no_margin">Eileen G Hoal</h3><p>  <sup>9</sup>SA MRC Centre for Tuberculosis Research, DST/NRF Centre of Excellence for Biomedical Tuberculosis Research, Division of Molecular Biology and Human Genetics, Faculty of Medicine and Health Sciences, Stellenbosch University, Tygerberg, South Africa</p><div>Find articles by <a href="/pubmed/?term=Hoal%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Eileen G Hoal</a></div></div><div id="_co_idm140158316311984"><h3 class="no_margin">Xiao Liu</h3><p>  <sup>7</sup>BGI-Shenzhen, Shenzhen, China</p><div>Find articles by <a href="/pubmed/?term=Liu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Xiao Liu</a></div></div><div id="_co_idm140158312660720"><h3 class="no_margin">Mark J Daly</h3><p>  <sup>2</sup>Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114</p><p>  <sup>3</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, MA 02141</p><p>  <sup>4</sup>Stanley Center for Psychiatric Research, Broad Institute, Cambridge, MA 02141</p><div>Find articles by <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Mark J Daly</a></div></div><div id="_co_idm140158317577280"><h3 class="no_margin">Marcus W Feldman</h3><p>  <sup>6</sup>Department of Biological Sciences, Stanford University, Stanford, CA, 94305</p><div>Find articles by <a href="/pubmed/?term=Feldman%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Marcus W Feldman</a></div></div><div id="_co_idm140158320930160"><h3 class="no_margin">Christopher R Gignoux</h3><p>  <sup>1</sup>Department of Genetics, Stanford University, Stanford, CA, 94305</p><div>Find articles by <a href="/pubmed/?term=Gignoux%20CR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Christopher R Gignoux</a></div></div><div id="_co_idm140158320927984"><h3 class="no_margin">Carlos D Bustamante</h3><p>  <sup>1</sup>Department of Genetics, Stanford University, Stanford, CA, 94305</p><div>Find articles by <a href="/pubmed/?term=Bustamante%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Carlos D Bustamante</a></div></div><div id="_co_idm140158312416528"><h3 class="no_margin">Brenna M Henn</h3><p>  <sup>5</sup>Department of Ecology and Evolution, SUNY Stony Brook, NY 11794</p><div>Find articles by <a href="/pubmed/?term=Henn%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29195075">Brenna M Henn</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140158321944384_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140158321944384_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140158321944384_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">  <sup>1</sup>Department of Genetics, Stanford University, Stanford, CA, 94305</div><div class="fm-affl" id="A2" lang="en">  <sup>2</sup>Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114</div><div class="fm-affl" id="A3" lang="en">  <sup>3</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, MA 02141</div><div class="fm-affl" id="A4" lang="en">  <sup>4</sup>Stanley Center for Psychiatric Research, Broad Institute, Cambridge, MA 02141</div><div class="fm-affl" id="A5" lang="en">  <sup>5</sup>Department of Ecology and Evolution, SUNY Stony Brook, NY 11794</div><div class="fm-affl" id="A6" lang="en">  <sup>6</sup>Department of Biological Sciences, Stanford University, Stanford, CA, 94305</div><div class="fm-affl" id="A7" lang="en">  <sup>7</sup>BGI-Shenzhen, Shenzhen, China</div><div class="fm-affl" id="A8" lang="en">  <sup>8</sup>Department of Developmental Biology, Stanford University, Stanford, California, USA</div><div class="fm-affl" id="A9" lang="en">  <sup>9</sup>SA MRC Centre for Tuberculosis Research, DST/NRF Centre of Excellence for Biomedical Tuberculosis Research, Division of Molecular Biology and Human Genetics, Faculty of Medicine and Health Sciences, Stellenbosch University, Tygerberg, South Africa</div><div class="fm-affl" id="A10" lang="en">  <sup>10</sup>Wellcome Trust Sanger Institute, Genome Campus, Hinxton, England</div><div id="FN1"><sup>*</sup>Correspondence: <a href="mailto:dev@null" data-email="gro.etutitsnidaorb@nitramra" class="oemail">gro.etutitsnidaorb@nitramra</a> (A.R.M) or <a href="mailto:dev@null" data-email="ude.koorbynots@nneh.annerb" class="oemail">ude.koorbynots@nneh.annerb</a> (B.M.H)</div><div id="FN2"><sup>11</sup>Lead contact</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140158321944384_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN5">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=29195075" target="pmc_ext" ref="reftype=publisher&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Cell</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5884124/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1: </strong><strong>Figure S1</strong> – <strong>Pedigrees inferred from ethnographic information. (Related to </strong><a href="/pmc/articles/PMC5884124/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and Methods) Ethnographically inferred pedigrees for KhoeSan individuals are shown in: A) the ‡Khomani, and B) the Nama. Different shades represent whether the samples have been genotyped. Non-phenotyped individuals are crossed in pedigrees.<p><strong>Figure S2</strong> – <strong>Ancestry estimates in ‡Khomani and Nama samples. (Related to </strong><a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>) A) Admixture runs across K=3–7 for the ‡Khomani and Nama populations, using Namibian San, Hadza, Sandawe, Maasai, Kenyan Bantu, South African (SA) Bantu, Yoruba, Mozabite, Central Europeans (CEU), and French populations as a reference panel, as in <a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>. B) Minor ancestry component associations with M index, displayed with a square root x-axis to elongate the minor contributions. C) Pairwise ancestry component correlations at k=7 from A). Upper triangular matrix shows Pearson’s correlation coefficient between pairwise ancestry estimates. Lower triangular matrix shows scatter plots with ‡Khomani shown in blue and Nama shown in red. D-F) Principal Components Analysis (PCA) biplots for the same SNPs and individuals used in the ADMIXTURE analysis adjusted for relatedness using King and the PC-AiR approach. D) PC1 vs PC2, E) PC3 vs PC4, and F) PC5 vs PC6.</p><p><strong>Figure S3</strong> – <strong>Partitioned heritability by population. (Related to </strong><a href="/pmc/articles/PMC5884124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>) Proportion of heritable variation in baseline pigmentation explained by true pigmentation gene sets (dots) versus matched null distribution after accounting for number of SNPs in gene sets in two different candidate gene sets. Results are stratified by the combined (comb) data, ‡Khomani (khm), and Nama populations. * indicates FDR &lt; 0.1 (indicated by the top decile of the null distributions). Gene set labels are the same as in <a href="/pmc/articles/PMC5884124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>.</p><p><strong>Figure S4</strong> – <strong>Imputation and targeted resequencing quality control. (Related to STAR methods)</strong> A) Imputation quality was assessed via homozygous reference (HR), heterozygous (Het), and homozygous non-reference (HN) concordance for imputed dosages with high coverage exome sequencing data. Each concordance metric was computed as the ratio of dosages for a particular genotyping class (e.g. HR) in both genotype and exome datasets to the dosages of that total class in the genotype dataset with any call in the exomes. The colors indicate the different genotyping arrays, with the 550k array in red, OmniExpress array in blue, and OmniExpressPlus array in green. B–E) Targeted resequencing quality control. B) Depth of coverage versus array concordance. Dashed line at 10× indicates minimum depth accepted for a sample’s inclusion. C) Depth of coverage by targeted resequencing region. D) Resequencing coverage by sequencing library. Solid line indicates median value, and dashed lines indicate 25% and 75% quartiles. E) Fraction of reads in targeted resequencing regions by sequencing library. Sequencing libraries (seq1, seq2, seq3, seq4, seq5, new_seq1, new_seq2, new_seq3, new_seq4, and new_seq5) consisted of barcoded and pooled samples, with most samples sequenced in multiple libraries. Libraries with “new” in the name were sequenced after the last sample collection.</p><p><strong>Figure S5 – Pigmentation prediction from other models are inaccurate in the KhoeSan.</strong> (<strong>Related to </strong><a href="/pmc/articles/PMC5884124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) A) Prediction model of skin pigmentation based on seven SNPs. The model (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip">Hart et al., 2013b</a>; <a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip">Spichenok et al., 2011</a>) utilizes a bifurcating decision tree based on individuals’ genotypes at seven predictive loci. The condition of each step-wise decision is described, with arrows indicating the decision possibilities. Categorical predictions are shown at the bottom of the chart. B) Pigmentation prediction result is shown as individuals’ actual spectrometer measured of M index against predicted categorical pigmentation. Number of individuals assigned to each category is shown in the x axis labels. The prediction is based on a previously developed 7-SNP model (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip">Hart et al., 2013b</a>; <a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip">Spichenok et al., 2011</a>), with predicted output categorized as “Light”, “Medium”, “Dark”, or “No Call” if the model fails to unambiguously assign a prediction based on genotypes.</p><p><strong>Figure S6</strong> – <strong>Initial GWAS. (Related to </strong><a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>) Imputed variants were association with pigmentation and tanning status for 217 KhoeSan individuals (107 ‡Khomani and 110 Nama individuals). A) Study design overview. B) Manhattan plot for pigmentation, with LD assessed and clumped using best guess genotypes from MEGA data (i.e. largest sample size). LD clumps are shown in blue for top 25 loci and labeled with closest gene for top 5 loci. C) QQplot for pigmentation (λ<sub>GC</sub>=1.018), D) LocusZoom plot for <em>SLC24A5</em>, among top 5 independent loci with linkage signals, E) Manhattan plot for tanning status, F) QQplot for tanning status (λ<sub>GC</sub>=1.052), G) LocusZoom plot near <em>TYRP1</em>, among top 5 independent loci with linkage signals.</p></div><div><a href="/pmc/articles/PMC5884124/bin/NIHMS921744-supplement-1.pdf" data-ga-action="click_feat_suppl">NIHMS921744-supplement-1.pdf</a><span style="color:gray"> (137K)</span></div><div class="small guid">GUID:&nbsp;0B819F97-48B4-46C7-9748-AD66BAC9E6EC</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2: </strong><strong>Table S1</strong> – <strong>Summary statistics of pigmentation phenotypes for global population.</strong> (<strong>Related to </strong><a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) All statistics relate to M index for baseline pigmentation. Rows with no dataset listed refer to subsets of the dataset in prior rows, e.g. &gt;90% KhoeSan Nama refers to the Nama samples from this study. Cape Verdean samples from (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip">Beleza et al., 2013</a>) were broken into quantiles, labeled Q1 to Q4. μ = mean, σ = standard deviation, and c<sub>V</sub> = coefficient of variation (σ/μ), a standardized metric of dispersion.<p><strong>Table S2 – Reference and KhoeSan samples included in the ADMIXTURE analysis.</strong> (<strong>Related to </strong><a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>)</p><p><strong>Table S3</strong> – <strong>Heritability estimates by population using multiple approaches. (Related to </strong><a href="/pmc/articles/PMC5884124/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) Heritability was assessed separately and jointly in the ‡Khomani and Nama populations. Because of population structure, haplotypes were called separately, (i.e. even when populations were analyzed jointly). N = number of individuals. SNPs = number of SNPs included.</p><p><strong>Table S4 – Number of individuals from each population that were phenotyped, genotyped, exome sequenced, or with targeted resequencing data (Related to STAR Methods).</strong> Pheno = phenotyped, Geno=genotyped, Exome = exome sequenced, Reseq = targeted resequencing. We also indicate the overlap among different genotyping arrays, which are as follows: 550k = Illumina 550k array, OmniEx = Illumina OmniExpress array, OmniExPlus = Illumina OmniExpressPlus array, Omni2.5 = Illumina Omni2.5 array, Mega = Illumina Multiethnic genotyping array. Phase 2 of the GWAS analysis consisted of samples genotyped on the MEGA array.</p><p><strong>Table S5</strong> – <strong>Top 50 associations from initial GWAS. (Related to </strong><a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>) All associations shown are independent (R<sup>2</sup> ≤ 0.5, window size = 250 kb), and indicate the association between genotype dosages and skin pigmentation M index across the Nama and ‡Khomani datasets. The # tagging column indicates the number of SNPs with an rsID that are tagged by the independent signal.</p><p><strong>Table S6</strong> – <strong>Summary statistics and regulator information for known and novel pigmentation SNPs. (Related to </strong><a href="/pmc/articles/PMC5884124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>) A) P-value, effect size and standard error, and frequency are shown for known pigmentation loci (including skin, eye, and hair color phenotypes) in the ‡Khomani, Nama, and jointly analyzed samples. B) Regulatory information including conservation, functional class, motif changes, etc, indentified via HaploReg are shown for resequencing variants in 6 of the targeted regions.</p><p>*Provided as Excel spreadsheet due to size constraints</p><p><strong>Table S7</strong> – <strong>Regions selected for targeted resequencing. (Related to </strong><a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>) Resequencing was successfully performed for a total of 441 KhoeSan individuals.</p><p><strong>Table S8 – Power to detect loci with suggestive and genome-wide significance for effect sizes previously associated with skin pigmentation. (Related to STAR Methods)</strong> A previous study by <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip">Beleza et al, 2013</a> showed that 4 loci explained 35% of the variation in skin color. We show power for the approximate sample sizes in phase 1, phase 2, and the resequencing analysis.</p><p>*Provided as Excel file due to size constraints</p></div><div><a href="/pmc/articles/PMC5884124/bin/NIHMS921744-supplement-2.xlsx" data-ga-action="click_feat_suppl">NIHMS921744-supplement-2.xlsx</a><span style="color:gray"> (99K)</span></div><div class="small guid">GUID:&nbsp;5CD265B1-5558-44F9-B7E4-BAC4238AE323</div></div></dd></dl></div><div id="Abs1" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Abs1title">Summary</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Approximately 15 genes have been directly associated with skin pigmentation variation in humans, leading to its characterization as a relatively simple trait. However, by assembling a global survey of quantitative skin pigmentation phenotypes, we demonstrate that pigmentation is more complex than previously assumed with genetic architecture varying by latitude. We investigate polygenicity in the KhoeSan, populations indigenous to southern Africa, who have considerably lighter skin than equatorial Africans. We demonstrate that skin pigmentation is highly heritable, but known pigmentation loci explain only a small fraction of the variance. Rather, baseline skin pigmentation is a complex, polygenic trait in the KhoeSan. Despite this, we identify canonical and non-canonical skin pigmentation loci, including near <em>SLC24A5</em>, <em>TYRP1, SMARCA2</em>/<em>VLDLR</em>, and <em>SNX13</em> using a genome-wide association approach complemented by targeted resequencing. By considering diverse, under-studied African populations, we show how the architecture of skin pigmentation can vary across humans subject to different local evolutionary pressures.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">pigmentation, human evolution, Africa, heritability, population genetics</span></div></div><div id="Abs2" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="Abs2title">eTOC</h2><!--article-meta--><div><p id="P2" class="p p-first">The genetic architecture of skin pigmentation is highly complex and varies across human populations subjected to distinct geographical evolutionary pressures</p><p id="P3" class="p p-last">
</p><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140158321420544" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms921744u1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms921744u1.jpg" src="/pmc/articles/PMC5884124/bin/nihms921744u1.jpg"></div><div id="largeobj_idm140158321420544" class="largeobj-link align_right" style="display: none"><a target="object" href="">Open in a separate window</a></div><p></p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P4" class="p p-first">Skin pigmentation is one of the most strikingly variable and strongly selected phenotypes among human populations, with darker skin observed closer to the equator and lighter pigmentation observed at high latitudes (<a href="#R67" rid="R67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498082">Sturm and Duffy, 2012</a>). Researchers have hypothesized that variable exposure to ultra violet radiation (UVR) creates opposing selective forces for vitamin D production and folate protection, resulting in variable melanin production and global pigmentation differentiation (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498206">Chaplin and Jablonski, 2009</a>; <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498176">Jablonski and Chaplin, 2010</a>). Skin pigmentation differences at similar latitudes and UV exposures indicate that additional evolutionary forces, such as assortative mating, drift, and epistasis, are also likely to have affected global skin pigmentation (<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498162">Pospiech et al., 2014</a>; <a href="#R79" rid="R79" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498189">Wilde et al., 2014</a>). While ~171 genes have been implicated in variability across model organisms (e.g. the Color Genes database: <a href="http://www.espcr.org/micemut/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.espcr.org/micemut/</a>), only ~15 genes have been associated with skin color differences in humans (<a href="/pmc/articles/PMC5884124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The relative paucity of loci identified from GWAS efforts has led to the characterization of pigmentation variation as relatively simple, with only a handful of SNPs being highly predictive of skin, eye, and hair color across populations (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498164">Hart et al., 2013a</a>; <a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498129">Spichenok et al., 2011</a>; <a href="#R77" rid="R77" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498177">Walsh et al., 2013</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Replication of previously associated skin pigmentation variants in the joint ‡Khomani and Nama populations</strong><p id="__p15">P-value indicates the joint association across all KhoeSan individuals using a linear mixed model accounting for European and Bantu admixture as well as kinship. Beta values reflect the effect size of adding one derived allele, assuming an additive model, to the distribution of M index (see <a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). See also <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5</a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Table S6A</a>.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140158314940272" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">rsID</th><th rowspan="1" colspan="1" valign="bottom" align="left">P-value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Beta</th><th rowspan="1" colspan="1" valign="bottom" align="left">Derived frequency</th><th rowspan="1" colspan="1" valign="bottom" align="left">Allele number<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">San-specific frequency</th><th rowspan="1" colspan="1" valign="bottom" align="left">San 95% CI<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">W. AFR<sup>c</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">N. EUR<sup>d</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>UGT1A</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs6742078</td><td rowspan="1" colspan="1" valign="top" align="left">0.58</td><td rowspan="1" colspan="1" valign="top" align="left">−0.44</td><td rowspan="1" colspan="1" valign="top" align="left">0.54</td><td rowspan="1" colspan="1" valign="top" align="left">460</td><td rowspan="1" colspan="1" valign="top" align="left">0.60</td><td rowspan="1" colspan="1" valign="top" align="left">[0.54,0.69]</td><td rowspan="1" colspan="1" valign="top" align="left">0.47</td><td rowspan="1" colspan="1" valign="top" align="left">0.29</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC45A2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs35395</td><td rowspan="1" colspan="1" valign="top" align="left">0.98</td><td rowspan="1" colspan="1" valign="top" align="left">−0.02</td><td rowspan="1" colspan="1" valign="top" align="left">0.32</td><td rowspan="1" colspan="1" valign="top" align="left">882</td><td rowspan="1" colspan="1" valign="top" align="left">0.21</td><td rowspan="1" colspan="1" valign="top" align="left">[0.18,0.25]</td><td rowspan="1" colspan="1" valign="top" align="left">0.20</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC45A2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs16891982</td><td rowspan="1" colspan="1" valign="top" align="left">1.2E-03</td><td rowspan="1" colspan="1" valign="top" align="left">−2.84</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">882</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">[0.00,0.02]</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">0.98</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>IRF4</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12203592</td><td rowspan="1" colspan="1" valign="top" align="left">0.83</td><td rowspan="1" colspan="1" valign="top" align="left">−0.54</td><td rowspan="1" colspan="1" valign="top" align="left">0.01</td><td rowspan="1" colspan="1" valign="top" align="left">882</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">[0.00,0.00]</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">0.17</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>IRF4</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12202284</td><td rowspan="1" colspan="1" valign="top" align="left">0.51</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">0.04</td><td rowspan="1" colspan="1" valign="top" align="left">824</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">[0.00,0.01]</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">0.21</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>OPRM1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs6917661</td><td rowspan="1" colspan="1" valign="top" align="left">0.29</td><td rowspan="1" colspan="1" valign="top" align="left">−0.71</td><td rowspan="1" colspan="1" valign="top" align="left">0.66</td><td rowspan="1" colspan="1" valign="top" align="left">882</td><td rowspan="1" colspan="1" valign="top" align="left">0.71</td><td rowspan="1" colspan="1" valign="top" align="left">[0.67,0.79]</td><td rowspan="1" colspan="1" valign="top" align="left">0.61</td><td rowspan="1" colspan="1" valign="top" align="left">0.76</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>EGFR</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12668421</td><td rowspan="1" colspan="1" valign="top" align="left">0.65</td><td rowspan="1" colspan="1" valign="top" align="left">−0.49</td><td rowspan="1" colspan="1" valign="top" align="left">0.08</td><td rowspan="1" colspan="1" valign="top" align="left">882</td><td rowspan="1" colspan="1" valign="top" align="left">0.02</td><td rowspan="1" colspan="1" valign="top" align="left">[0.01,0.08]</td><td rowspan="1" colspan="1" valign="top" align="left">0.06</td><td rowspan="1" colspan="1" valign="top" align="left">0.27</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>TYRP1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs13289810</td><td rowspan="1" colspan="1" valign="top" align="left">0.61</td><td rowspan="1" colspan="1" valign="top" align="left">0.53</td><td rowspan="1" colspan="1" valign="top" align="left">0.19</td><td rowspan="1" colspan="1" valign="top" align="left">882</td><td rowspan="1" colspan="1" valign="top" align="left">0.18</td><td rowspan="1" colspan="1" valign="top" align="left">[0.11,0.25]</td><td rowspan="1" colspan="1" valign="top" align="left">0.24</td><td rowspan="1" colspan="1" valign="top" align="left">0.34</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>BNC2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs10756819</td><td rowspan="1" colspan="1" valign="top" align="left">0.51</td><td rowspan="1" colspan="1" valign="top" align="left">0.91</td><td rowspan="1" colspan="1" valign="top" align="left">0.08</td><td rowspan="1" colspan="1" valign="top" align="left">466</td><td rowspan="1" colspan="1" valign="top" align="left">0.02</td><td rowspan="1" colspan="1" valign="top" align="left">[0.00,0.05]</td><td rowspan="1" colspan="1" valign="top" align="left">0.07</td><td rowspan="1" colspan="1" valign="top" align="left">0.65</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>GATA3</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs376397</td><td rowspan="1" colspan="1" valign="top" align="left">0.91</td><td rowspan="1" colspan="1" valign="top" align="left">0.07</td><td rowspan="1" colspan="1" valign="top" align="left">0.65</td><td rowspan="1" colspan="1" valign="top" align="left">872</td><td rowspan="1" colspan="1" valign="top" align="left">0.79</td><td rowspan="1" colspan="1" valign="top" align="left">[0.75,0.82]</td><td rowspan="1" colspan="1" valign="top" align="left">0.31</td><td rowspan="1" colspan="1" valign="top" align="left">0.32</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>GRM5, TYR</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs10831496</td><td rowspan="1" colspan="1" valign="top" align="left">0.28</td><td rowspan="1" colspan="1" valign="top" align="left">−0.90</td><td rowspan="1" colspan="1" valign="top" align="left">0.52</td><td rowspan="1" colspan="1" valign="top" align="left">460</td><td rowspan="1" colspan="1" valign="top" align="left">0.63</td><td rowspan="1" colspan="1" valign="top" align="left">[0.57,0.70]</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.69</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>TYR</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs1042602</td><td rowspan="1" colspan="1" valign="top" align="left">0.74</td><td rowspan="1" colspan="1" valign="top" align="left">0.58</td><td rowspan="1" colspan="1" valign="top" align="left">0.06</td><td rowspan="1" colspan="1" valign="top" align="left">466</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">[0.00,0.02]</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">0.38</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KITLG</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12821256</td><td rowspan="1" colspan="1" valign="top" align="left">0.02</td><td rowspan="1" colspan="1" valign="top" align="left">−5.28</td><td rowspan="1" colspan="1" valign="top" align="left">0.02</td><td rowspan="1" colspan="1" valign="top" align="left">882</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">[0.00,0.01]</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">0.17</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>OCA2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs1800404</td><td rowspan="1" colspan="1" valign="top" align="left">0.53</td><td rowspan="1" colspan="1" valign="top" align="left">−0.40</td><td rowspan="1" colspan="1" valign="top" align="left">0.55</td><td rowspan="1" colspan="1" valign="top" align="left">854</td><td rowspan="1" colspan="1" valign="top" align="left">0.65</td><td rowspan="1" colspan="1" valign="top" align="left">[0.56,0.74]</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">0.81</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>OCA2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs7495174</td><td rowspan="1" colspan="1" valign="top" align="left">0.92</td><td rowspan="1" colspan="1" valign="top" align="left">−0.07</td><td rowspan="1" colspan="1" valign="top" align="left">0.71</td><td rowspan="1" colspan="1" valign="top" align="left">716</td><td rowspan="1" colspan="1" valign="top" align="left">0.61</td><td rowspan="1" colspan="1" valign="top" align="left">[0.55,0.69]</td><td rowspan="1" colspan="1" valign="top" align="left">0.26</td><td rowspan="1" colspan="1" valign="top" align="left">0.90</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HERC2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12913832</td><td rowspan="1" colspan="1" valign="top" align="left">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">−1.70</td><td rowspan="1" colspan="1" valign="top" align="left">0.10</td><td rowspan="1" colspan="1" valign="top" align="left">882</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">[0.00,0.02]</td><td rowspan="1" colspan="1" valign="top" align="left">0.01</td><td rowspan="1" colspan="1" valign="top" align="left">0.79</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>APBA2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs4424881</td><td rowspan="1" colspan="1" valign="top" align="left">0.25</td><td rowspan="1" colspan="1" valign="top" align="left">−1.24</td><td rowspan="1" colspan="1" valign="top" align="left">0.18</td><td rowspan="1" colspan="1" valign="top" align="left">440</td><td rowspan="1" colspan="1" valign="top" align="left">0.02</td><td rowspan="1" colspan="1" valign="top" align="left">[0.00,0.06]</td><td rowspan="1" colspan="1" valign="top" align="left">0.07</td><td rowspan="1" colspan="1" valign="top" align="left">0.86</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC24A5</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs1426654</td><td rowspan="1" colspan="1" valign="top" align="left">9.8E-09</td><td rowspan="1" colspan="1" valign="top" align="left">−3.58</td><td rowspan="1" colspan="1" valign="top" align="left">0.40</td><td rowspan="1" colspan="1" valign="top" align="left">882</td><td rowspan="1" colspan="1" valign="top" align="left">0.24</td><td rowspan="1" colspan="1" valign="top" align="left">[0.17,0.32]</td><td rowspan="1" colspan="1" valign="top" align="left">0.05</td><td rowspan="1" colspan="1" valign="top" align="left">1.00</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>MC1R</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs1805007</td><td rowspan="1" colspan="1" valign="top" align="left">0.80</td><td rowspan="1" colspan="1" valign="top" align="left">−0.64</td><td rowspan="1" colspan="1" valign="top" align="left">0.01</td><td rowspan="1" colspan="1" valign="top" align="left">630</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">[0.00,0.03]</td><td rowspan="1" colspan="1" valign="top" align="left">0.00</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td></tr></tbody></table></div><div id="largeobj_idm140158314940272" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5884124/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>a</sup>Allele number indicates the total number of alleles genotyped or sequenced across all KhoeSan samples.</div><div id="TFN4"><sup>b</sup>Confidence interval for the San-specific frequencies indicates the allele frequencies specifically on ‡Khomani haplotypes, assessed with local ancestry tracts.</div><div id="TFN5"><sup>c</sup>W. AFR (western African) allele frequencies were estimated from 405 ESN, GWD, YRI, and MSL populations in the phase 3 1000 Genomes project</div><div id="TFN6"><sup>d</sup>N. EUR (northern Europeans) allele frequencies were estimated from 190 GBR and CEU populations in the phase 3 1000 Genomes project</div></div></div><p id="P5">Despite being home to the greatest range of pigmentation globally, remarkably few genetic studies of pigmentation have been published to date in continental Africans (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498148">Jablonski and Chaplin, 2014</a>; <a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498160">Relethford, 2000</a>). Instead, the genetic basis of skin color has primarily been studied in Eurasians and admixed African Americans (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498117">Beleza et al., 2013</a>; <a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498181">2012</a>; <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498104">Candille et al., 2012</a>; <a href="#R67" rid="R67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498115">Sturm and Duffy, 2012</a>; <a href="#R68" rid="R68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498078">Sulem et al., 2008</a>; <a href="#R69" rid="R69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498193">2007</a>); selective sweeps in high latitude populations have been interpreted as resulting from strong environmental selection pressure. For example, the derived Ala111Thr allele (rs1426654) of <em>SLC24A5</em> that swept to near fixation in western Eurasian populations confers the largest known effect on skin color variability (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498151">Beleza et al., 2013</a>; <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498186">Lamason, 2005</a>). Loci in/near <em>SLC45A2</em>, <em>GRM5</em>/<em>TYR</em>, and <em>APBA2/OCA2</em> also have divergent allele frequencies between Europeans and Africans, with large derived lightening effects in Europeans (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498191">Beleza et al., 2013</a>; <a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498079">Norton et al., 2006</a>). Smaller effects contribute to the relatively narrow variation within Europeans, including associations in/near <em>MC1R</em>, <em>TYR</em>, <em>IRF4</em>, and <em>ASIP</em> (<a href="#R69" rid="R69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498197">Sulem et al., 2007</a>; <a href="#R68" rid="R68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498122">2008</a>). Several shared lightening effects derived in Eurasians arose through convergent evolution (e.g. rs1800414 in <em>OCA2</em> in East Asians), whereas other shared effects resulted from similar selective sweeps (e.g. <em>KITLG</em>) (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498132">Miller et al., 2007</a>; <a href="#R86" rid="R86" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498116">Yang et al., 2016</a>). Because African populations have been understudied, the genetic architecture and higher variability of skin pigmentation is poorly understood.</p><p id="P6">Strong positive selection acting on skin pigmentation has resulted in large effects that explain a large fraction of heritable variation. For example, a previous study in recently admixed Cape Verdeans with European and West African ancestors showed that only 4 loci explain 35% of the variation in skin pigmentation (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498083">Beleza et al., 2013</a>). In contrast, complex traits such as height and schizophrenia require ~10,000 independent SNPs derived from GWAS of &gt;100,000 individuals to build predictors that explain ~29% and ~20% of the variance in independent cohorts, respectively (<a href="#R59" rid="R59" class=" bibr popnode">Ripke et al., 2014</a>; <a href="#R81" rid="R81" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498156">Wood et al., 2014</a>). Previous studies of positively selected traits such as pigmentation, high altitude adaptation, and response to pathogens have repeatedly produced larger effect sizes than complex common disease; these large effect loci have typically been discovered with relatively small sample sizes (i.e. ~100s of individuals) compared to common diseases (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498125">Genovese et al., 2010</a>; <a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498134">Kenny et al., 2012a</a>; <a href="#R87" rid="R87" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498089">Yi et al., 2010</a>). While pigmentation loci are among the most diverged in the genome, it is noteworthy that effect size estimates of significant polymorphic GWAS loci tend to be directionally consistent across populations (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498103">Carlson et al., 2013</a>), but that aggregate prediction accuracy varies across populations (<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498185">Martin et al., 2017</a>).</p><p id="P7">Striking skin pigmentation variability among African populations has been underappreciated in genetic studies (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498118">Jablonski and Chaplin, 2014</a>; <a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498074">Relethford, 2000</a>). Light skin pigmentation is observed in the far southern latitudes of Africa among KhoeSan<sup><sup><a href="#FN4" rid="FN4" class=" fn">§</a></sup></sup> hunter-gatherers and pastoralists in and near the Kalahari Desert. The KhoeSan are unique in their early divergence from other populations, likely dating back at least ~100,000 years ago (<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498169">Schlebusch et al., 2012</a>; <a href="#R76" rid="R76" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498147">Veeramah et al., 2011</a>); they exhibit extraordinary levels of genetic diversity and low levels of linkage disequilibrium (LD) (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498142">Henn et al., 2011</a>). Previous work points to southern Africa as the point of origin for modern humans (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498101">Henn et al., 2011</a>; <a href="#R72" rid="R72" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498114">Tishkoff et al., 2009</a>), but it is unknown whether moderate to light skin pigmentation in the different KhoeSan groups is an example of convergent evolution with northern Europeans and Asians, or reflects the ancestral human phenotype. Previous studies have noted different pigmentation allele frequencies between the Ju|’hoansi San and other Africans, but these have been based on n&lt;7 individuals from the former population without measured phenotypes (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498099">Berg and Coop, 2014</a>; <a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498201">Norton et al., 2006</a>).</p><p id="P8" class="p p-last">Here we report an evolutionary and genetic study of skin pigmentation with a total of 465 genotyped KhoeSan individuals (278 ‡Khomani San and 187 Nama), with targeted resequencing at associated pigmentation loci and matched quantitative spectrophotometric phenotype data (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S4</a>). The ‡Khomani San are traditionally a N|u-speaking hunter-gatherer population living in the southern Kalahari Desert, while the Nama are traditionally a Khoekhoe-speaking semi-nomadic pastoralist group of KhoeSan ancestry. We investigate: i) the degree of polygenicity and heritability of skin pigmentation, ii) the extent of pigmentation variation explained by previously associated or canonical pigmentation genes, and iii) novel pigmentation alleles contributing to variation in the ‡Khomani San and Nama populations.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Results</h2><p id="P9" class="p p-first">We quantitatively phenotyped baseline skin color in 479 individuals (277 ‡Khomani, 202 Nama, <a href="#SD2" rid="SD2" class=" supplementary-material">Table S4</a>) via narrow-band reflectometry to measure hemoglobin and melanin of both the left and right upper inner arms (<strong>STAR Methods</strong>), with M index = log<sub>10</sub>(1 / % red reflectance). We sequenced and/or genotyped a subset of phenotyped samples (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S4</a>, Methods). Skin pigmentation is lighter in the KhoeSan than the majority of other African populations, with baseline upper arm M index = 57.57 ± 10.12 (mean ± sd) in the ‡Khomani San. Baseline upper arm pigmentation in the Nama is slightly lower, with M index = 52.12 ± 8.93. The ‡Khomani are on average significantly darker than the Nama (p=3.6e-10, <a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1C</span></a>), but the variance is not significantly different (p&gt;0.05). For comparison, we aggregated quantitative skin pigmentation across 32 globally diverse populations (4,712 individuals) assayed with a Derma Spectrometer (DSMI or DSMII) (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498165">Basu Mallick et al., 2013</a>; <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498200">Beleza et al., 2013</a>; <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498119">Candille et al., 2012</a>; <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498097">Coussens et al., 2015</a>; <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498128">Durazo-Arvizu et al., 2014</a>; <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498088">Edwards et al., 2010</a>; <a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498130">Norton et al., 2006</a>) (<a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A–B</span></a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>). Only four African populations are available for comparison; among these only the Ghanians represent an equatorial African population with minimal recent admixture. Skin color is substantially darker in equatorial Ghanaians, where M index reaches a mean of 96.04 ± 10.94; M index for Cape Verdeans, who have ~40% European admixture on average, have slightly lighter (55.39 ± 13.00, p=5.6e-3) and considerably more variable pigmentation (p=1.9e-6) than the KhoeSan. Two other populations living in South Africa, the Xhosa and admixed Coloured populations, have respectively darker (M index=67.1±7.5) and similar (M index=53.1±8.5) pigmentation compared to the KhoeSan populations (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498145">Coussens et al., 2015</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140158298696496" class="figure"><a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5884124_nihms921744f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms921744f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5884124/bin/nihms921744f1.jpg"></a></div><div id="largeobj_idm140158298696496" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5884124/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5884124/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Distributions of baseline pigmentation in globally diverse populations</strong><p id="__p7"><strong>A</strong>) Sample locations of skin pigmentation datasets where phenotypes were measured with a DSM I or DSM II. <strong>B</strong>) Violin plots of pigmentation distributions for 32 populations from 8 studies ordered by latitude; absolute latitudes provided on the right. Corresponding datasets are colored as in A). <a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a> provides summary statistics for each population. M indices are reflectance measures that approximate melanin content. <strong>C</strong>) A comparison of skin pigmentation distributions in ‡Khomani (top) and Nama populations (bottom). Dashed grey lines and labels indicate mean M index for the indicated other global populations. <strong>D</strong>) South African individuals in a household that exemplify the substantial skin pigmentation variability in the ‡Khomani and Nama populations. Picture taken with consent for publication. See also <a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>.</p></div></div></div><div id="S3" class="sec"><h3 id="S3title">Evidence of Increased Polygenicity in Skin Pigmentation Among Equatorial Populations</h3><p id="P10" class="p p-first">We tested whether the correlation between absolute latitude and pigmentation was significant with our large, quantitatively phenotyped sample of global populations. As previously observed (<a href="#R9" rid="R9" class=" bibr popnode">Byard, 1981</a>; <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498124">Jablonski and Chaplin, 2010</a>; <a href="#R88" rid="R88" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498196">Zaidi et al., 2017</a>), we find that skin pigmentation is strongly associated with absolute latitude (R<sup>2</sup>=0.53, β=−1.18 on M index scale, p&lt;2e-16); populations further from the equator have lighter skin pigmentation. We next tested whether variance in melanin within populations also varies across populations. Skin pigmentation has primarily been studied in lightly pigmented European and East Asian populations, where skin color varies minimally among individuals (<a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A–B</span></a>). Less-studied equatorial and admixed populations, including Melanesians, Ghanaians, Cape Verdeans, South African admixed Coloured, and South Asians vary considerably more in skin pigmentation (<a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>). We find that absolute latitude is also significantly negatively associated with the standard deviation in melanin (R<sup>2</sup>=0.41, p=5.0e-5). Further, melanin distributions are heteroskedastic (i.e. the variance is not constant—rather, it changes over the range of observed M index), with the coefficient of variation, a standardized metric of phenotypic dispersion, decreasing with increasing distance from the equator (cv=σ/μ, R<sup>2</sup>=0.14, p=0.03, <a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>).</p><p id="P11" class="p p-last">A sign test comparing variances in lighter versus darker population pairs within the same study indicates that populations with lighter skin have significantly reduced phenotypic variance than expected by chance (p=2.01e-8). These results suggest that there is reduced genetic heterogeneity and/or reduced variance in the population distribution of causal effect sizes contributing to lighter versus darker pigmentation. There is more than an order of magnitude difference in variance between the lightest and darkest populations (i.e. Irish vs Ghanaian F=0.03, p=6.7e-23). Europeans and East Asians have significantly less variation than South Asians (F=0.25, p=1.06e-14 and F=0.30, p=1.27e-10, respectively, <a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>). Cape Verdeans with the highest quartile of European admixture have lighter, less variable skin color than individuals with the lowest quartile of European ancestry (p=4.28e-9, although notably ancestry proportions are bimodal across individuals). Among Melanesians, islands at similar latitudes with more lightly pigmented individuals on average show less variance than those with more darkly pigmented individuals (e.g. one-sided F test comparing variance among more lightly pigmented New Britain individuals versus individuals from Bougainville, p=2.89e-9, <a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>). Among the ‡Khomani and Nama, comparing individuals with primarily European admixture (&gt;20%, N=124) to individuals with primarily Bantu admixture (&gt;20%, N=91), we find significantly greater melanin variation among KhoeSan individuals with more Bantu admixture (p=1.33e-4).</p></div><div id="S4" class="sec"><h3 id="S4title">Ancestry and Skin Pigmentation Variation in the KhoeSan</h3><p id="P12" class="p p-first">The ‡Khomani San and the Nama have both experienced admixture with neighboring darker-skinned Bantu-speaking groups beginning ~450 years ago, as well as with lighter-skinned European settlers who first arrived in the Northern Cape during the late 18<sup>th</sup> century (<a href="#R73" rid="R73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498179">Uren et al., 2016a</a>). We assessed these ancestry proportions using unsupervised allele frequency clustering with ADMIXTURE as well as principal components analysis (PCA, <strong>STAR Methods</strong>). At <em>k</em>=3, we observe distinct clustering between Europeans, Bantu-speaking and West African populations, and KhoeSan populations; both the Nama and the ‡Khomani have ~75–80% KhoeSan-specific ancestry. For <em>k</em>=7, which gives most stable ancestry estimates, we observe a partitioning of the KhoeSan ancestry into ‘northern Kalahari’ ancestry shared with Ju|’hoansi and a distinct southern or circum-Kalahari ancestry present in the Nama and the ‡Khomani. On average, in the ‡Khomani San we find 55% northern Kalahari KhoeSan ancestry, 21% southern Kalahari KhoeSan ancestry, 11% European ancestry (common in CEU and French individuals), 12% western African ancestry (common in Yoruba and Bantu-speaking populations), and 2% attributable to other African populations (Tanzanian hunter-gatherers, East African, and North African populations, <a href="#SD2" rid="SD2" class=" supplementary-material">Table S2</a>, <a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2A</a>). The Nama differ from the ‡Khomani in their proportion of northern versus southern Kalahari ancestry; they have 17% northern Kalahari ancestry, 62% southern Kalahari ancestry, 9% European ancestry, 10% western African ancestry, and 1% attributable to other African populations on average. The western African fraction in the Nama is significantly more variable among individuals (p=1.08e-5), resulting from recent Damara gene flow (<a href="#R73" rid="R73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498208">Uren et al., 2016a</a>). The partition of ancestry components occurs in the same order and is correlated between ADMIXTURE and PCA (<a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2A,D–F</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140158315427136" class="figure"><a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5884124_nihms921744f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms921744f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5884124/bin/nihms921744f2.jpg"></a></div><div id="largeobj_idm140158315427136" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5884124/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5884124/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Ancestry components in the KhoeSan and association with pigmentation</strong><p id="__p8"><strong>A</strong>) ADMIXTURE proportions at <em>k</em>=7 for the ‡Khomani and Nama populations, using Namibian San, Hadza, Sandawe, Maasai, Kenyan Bantu, South African (SA) Bantu, Yoruba, Mozabite, Central Europeans (CEU), and French populations as a reference panel <strong>B</strong>) Associations between substantial <em>k</em> ancestry clusters and average melanin (M index) baseline pigmentation value in the combined ‡Khomani and Nama populations. The Bantu and European components each constitute ≥ 5% of the total KhoeSan ancestry on average and have significant associations in the best multivariate model (<em>p</em> &lt;0.05). See also <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Table S2</a>.</p></div></div></div><p id="P13" class="p p-last">In a multivariate mixed model with the significant European and Bantu admixture components, European and Bantu ancestry are strongly correlated with light (β = −18.09, p=2.9e-03) and dark skin (β = 25.60, p=1.8e-09), respectively. Together, we estimate that fixed admixture effects explained 34% of the variation in skin color (adjusted R<sup>2</sup>); by comparison, 44% of pigmentation variation in Cape Verdeans is explained by admixture effects (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498133">Beleza et al., 2013</a>). Marginal associations are shown in <a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a>, with pairwise ancestry correlations shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2B</a>. Southern Kalahari ancestry frequent in the Nama is significantly anti-correlated with Bantu ancestry and is marginally predicted to lighten skin, but not when modeled jointly with Bantu ancestry in a multivariate model. Interestingly, the mean pigmentation of Nama and ‡Khomani individuals with &lt;90% KhoeSan ancestry is not significantly different from individuals with &gt;90% KhoeSan ancestry (p=0.94), although the variance is significantly greater in more admixed individuals (admixture from either/both European or Bantu ancestries, p=2.2e-3). These results suggest that while admixture increases phenotypic variance, pigmentation alleles on KhoeSan haplotypes contribute more to the overall heterogeneity than those on European or Bantu haplotypes. Consistent with this result, we observe substantial skin pigmentation variation among related individuals, which, coupled with high heritability (see below) suggests a role for large effect sizes of alleles contributing to pigmentation.</p></div><div id="S5" class="sec"><h3 id="S5title">Skin Pigmentation is Highly Heritable</h3><p id="P14" class="p p-first">We inferred narrow sense heritability for baseline skin pigmentation and tanning status in the KhoeSan with four methods: family pedigrees (
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">pedigree</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">pedigree</mi><mn>2</mn></msubsup></mrow></math></script>), SNP array similarity matrices (
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></script>), identity-by-descent (IBD) sharing matrices (
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">IBD</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">IBD</mi><mn>2</mn></msubsup></mrow></math></script>), and exome sequence variation (
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">exome</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">exome</mi><mn>2</mn></msubsup></mrow></math></script>, <a href="/pmc/articles/PMC5884124/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). While pedigree-based heritability estimates are not based on genetic data and therefore not strongly affected by admixture, we carefully considered it for SNP-based estimates, as described previously (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498140">Beleza et al., 2013</a>; <a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498150">Thornton et al., 2012a</a>; <a href="#R89" rid="R89" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498141">Zaitlen et al., 2013</a>; <a href="#R90" rid="R90" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498076">2014</a>). In each of the heritability estimates of baseline skin color and tanning status, we accounted for admixture proportions with European and Bantu ancestry as covariates, as well as familial relatedness via a kinship covariance matrix. Similarly for tanning status, we accounted for age, sex, and kinship. Previous family-based estimates for skin color heritability in other populations are high, ranging between 55–90% (<a href="#R9" rid="R9" class=" bibr popnode">Byard, 1981</a>; <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498149">Clark et al., 1981</a>; <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498105">Frisancho et al., 1981</a>; <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498086">Harrison and Owen, 1964</a>). Interestingly, published genetic estimates of skin pigmentation heritability in Europe are low and insignificant, potentially because of reduced genetic diversity at skin pigmentation loci due to positive selection (<a href="#R88" rid="R88" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498168">Zaidi et al., 2017</a>). Our heritability estimates in the KhoeSan are analogous to family-based estimates because of the elevated relatedness in our samples.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Heritability estimates contrasting baseline skin pigmentation with tanning status</strong><p id="__p12">SNP-based heritability estimates were computed with GCTA using genetic relationship matrices (GRMs) calculated from SNP gentoypes, an admixture-corrected GRM computed with REAP, and IBD segments. All models were unconstrained. See also <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S1</a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Table S3</a>.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140158306441232" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Method</th><th rowspan="1" colspan="1" valign="top" align="left">Dataset</th><th rowspan="1" colspan="1" valign="top" align="left">SNPs</th><th rowspan="1" colspan="1" valign="top" align="left">N</th><th rowspan="1" colspan="1" valign="top" align="left">h<sup>2</sup> (SE) baseline pigmentation<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">h<sup>2</sup> (SE) tanning status<sup>b</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">GCTA GRM</td><td rowspan="1" colspan="1" valign="top" align="left">genotype array</td><td rowspan="1" colspan="1" valign="top" align="left">286,026</td><td rowspan="1" colspan="1" valign="top" align="left">216</td><td rowspan="1" colspan="1" valign="top" align="left">0.90 (0.15)</td><td rowspan="1" colspan="1" valign="top" align="left">0.31 (0.19)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">REAP GRM</td><td rowspan="1" colspan="1" valign="top" align="left">genotype array</td><td rowspan="1" colspan="1" valign="top" align="left">286,026</td><td rowspan="1" colspan="1" valign="top" align="left">216</td><td rowspan="1" colspan="1" valign="top" align="left">0.97 (0.15)</td><td rowspan="1" colspan="1" valign="top" align="left">0.41 (0.21)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">K<sub>IBD</sub></td><td rowspan="1" colspan="1" valign="top" align="left">genotype array</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="left">216</td><td rowspan="1" colspan="1" valign="top" align="left">0.97 (0.16)</td><td rowspan="1" colspan="1" valign="top" align="left">0.45 (0.22)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">GCTA GRM</td><td rowspan="1" colspan="1" valign="top" align="left">exome</td><td rowspan="1" colspan="1" valign="top" align="left">117,132</td><td rowspan="1" colspan="1" valign="top" align="left">82</td><td rowspan="1" colspan="1" valign="top" align="left">0.95 (0.26)</td><td rowspan="1" colspan="1" valign="top" align="left">0.37 (0.37)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SOLAR</td><td rowspan="1" colspan="1" valign="top" align="left">pedigrees</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="left">477</td><td rowspan="1" colspan="1" valign="top" align="left">0.96 (0.12)</td><td rowspan="1" colspan="1" valign="top" align="left">0.19 (0.11)</td></tr></tbody></table></div><div id="largeobj_idm140158306441232" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5884124/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Bantu and European admixture proportions were included as covariates.</div><div id="TFN2"><sup>b</sup>Age and sex were included as significant covariates for tanning status (wrist minus baseline underarm pigmentation).</div></div></div><p id="P15">We first constructed pedigrees from ethnographic interviews for individuals within the ‡Khomani and Nama populations and verified relationships where possible with genetic data. 533 individuals (including parental individuals not sampled) could be assigned to a pedigree, resulting in 354 extended pedigrees and 470 nuclear families. Via traditional pedigree-based estimation, we estimate an 
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M5" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">pedigree</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M5" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">pedigree</mi><mn>2</mn></msubsup></mrow></math></script> of 0.96 ± 0.12 for baseline skin color (<strong>STAR Methods</strong>). We then asked whether variation present on the ascertained SNP arrays or from exome sequencing could explain a similar fraction of the pigmentation variation. Genetic heritability estimates inferred from recently admixed populations have two potential problems: 1) inferred familial relationships between individuals are less accurate (<a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498135">Thornton et al., 2012a</a>), and 2) environmental confounders (e.g. socioeconomic status) could be associated with the variance component attributed to additive genetic effects. In order to address the first issue, we use the proportion of KhoeSan, European and Bantu ancestry per individual to correct the SNP array genetic relatedness matrix (GRM) as described by the REAP approach (<a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498183">Thornton et al., 2012a</a>). The REAP matrix is also compared to the IBS matrix inferred using default GCTA parameters that do not account for stratification (<em>Methods</em>). We include European and Bantu ancestry as global covariates in the heritability estimation. All further estimation of 
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></script> was made using the unconstrained model in GCTA. Furthermore, we contrast baseline pigmentation with tanning status (i.e. sun exposed wrist – underarm melanin pigmentation); if our estimates were inflated by environmental confounders, we would also expect inflated heritability of tanning status.</p><p id="P16">The array-based heritability point estimates are consistently but not significantly higher when using a kinship matrix from Relatedness Estimation in Admixed Populations (REAP) than GCTA’s identity-by-state (IBS) GRM, both for the joint dataset and each population separately (<a href="/pmc/articles/PMC5884124/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Table S3</a>). We estimate 
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M7" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup><mo>=</mo><mn>0.97</mn><mo>±</mo><mn>0.15</mn></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M7" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup><mo>=</mo><mn>0.97</mn><mo>±</mo><mn>0.15</mn></mrow></math></script> (standard error) in an unconstrained model across both populations using the REAP GRM. We find consistent results from exome sequence data, where we estimate that 
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M8" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">exome</mi><mn>2</mn></msubsup><mo>=</mo><mn>0.95</mn><mo>±</mo><mn>0.26</mn></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M8" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">exome</mi><mn>2</mn></msubsup><mo>=</mo><mn>0.95</mn><mo>±</mo><mn>0.26</mn></mrow></math></script> in the ‡Khomani. We then used the familial relationships (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S1</a>) and population-level endogamy to estimate heritability from IBD sharing among all individuals in the ‡Khomani and Nama; we obtain a similar estimate of 
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M9" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">IBD</mi><mn>2</mn></msubsup><mo>=</mo><mn>0.97</mn><mo>±</mo><mn>0.15</mn></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M9" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">IBD</mi><mn>2</mn></msubsup><mo>=</mo><mn>0.97</mn><mo>±</mo><mn>0.15</mn></mrow></math></script> (<em>Methods</em>, see also (<a href="#R89" rid="R89" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498136">Zaitlen et al., 2013</a>)).</p><p id="P17" class="p p-last">We contrast the high heritability estimates for baseline pigmentation with estimates for tanning status. Tanning status is significantly associated with both sex (male β=6.2 increase in M index, p = 4.2e-4) and age (β=0.18 increase in M index per year, p=1.8e-4), but not with admixture proportions. None of the tanning status h<sup>2</sup> estimates, including pedigree-, IBD-, exome-, and SNP array-based estimates, are significantly greater than 0 (<a href="/pmc/articles/PMC5884124/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), consistent with previous observations that tanning status is largely environmentally determined by UV exposure (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498126">Clark et al., 1981</a>; <a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498091">Nan et al., 2009</a>). The stark contrast of the baseline pigmentation and tanning status heritability estimates, and the consistency of h<sup>2</sup> across methods indicates that our high baseline pigmentation heritability estimates do not simply arise from pedigree and population structure and that socioeconomic factors are unlikely to have significant effect on our heritability estimates.</p></div><div id="S6" class="sec"><h3 id="S6title">A Complex Genetic Architecture in the KhoeSan</h3><p id="P18" class="p p-first">The genetic architecture of skin pigmentation has been described as simpler than many other phenotypes, for which only a few genes explain ~35% of the total variation in a given population, and average genomic ancestry explains an additional ~44% of the variation, indicating a long tail of smaller effects (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498174">Beleza et al., 2013</a>; <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498085">Candille et al., 2012</a>). We investigated how much of the heritable variation in KhoeSan populations can be ascribed to previously annotated pigmentation gene sets (<a href="/pmc/articles/PMC5884124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3A</span></a>). The first gene set (GS1) consists of 14 genes containing or near previously discovered skin pigmentation genetic associations in Europeans, East Asians, Cape Verdeans, and Native Americans (<a href="/pmc/articles/PMC5884124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Table S6A</a>). The larger, second gene set (GS2) contains 50 genes compiled previously (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498153">Beleza et al., 2013</a>) from human pigmentation associations, positive selection scans, and model organism pigmentation loci. The third gene set (GS3) contained 50 loci most significantly associated with pigmentation in the KhoeSan (phase 1, see novel variants section below). We partitioned the genome into GS1, GS2, GS3, and the rest of the genome and performed four comparisons, computing the variance explained by: GS1 versus the rest of the genome, GS2 versus the rest of the genome, GS3 versus the rest of the genome, and GS1 versus GS2 versus the rest of the genome. For each comparison, we performed a restricted likelihood ratio test. The GS1 and GS2 gene sets do not explain a significant fraction of the heritability; that is, the heritability estimates overlap with zero. Rather, the remainder of the genome explains the overwhelming majority of the heritability (<a href="/pmc/articles/PMC5884124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3B</span></a>, σ<sup>2</sup><sub>GS1</sub>=0.08 vs σ<sup>2</sup><sub>Genome</sub>=0.82, p<sub>Genome</sub>=2.7e-5; σ<sup>2</sup><sub>GS2</sub>=0.09 vs σ<sup>2</sup><sub>Genome</sub>=0.79, p<sub>Genome</sub>=3.3e-4; and σ<sup>2</sup><sub>GS1</sub>=0.08 vs σ<sup>2</sup><sub>GS2</sub>=0.09 vs σ<sup>2</sup><sub>Genome</sub>=0.71, p<sub>Genome</sub>=2.5e-3, respectively). This result contrasts with conclusions from previous studies and indicates that the vast majority of variation in KhoeSan skin pigmentation arises from pigmentation genes yet to be discovered, providing strong evidence for a complex, polygenic architecture. GS3 explains a small but significant fraction of the heritability, as discussed below.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC5884124/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140158316006144" class="figure"><a href="/pmc/articles/PMC5884124/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5884124_nihms921744f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms921744f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5884124/bin/nihms921744f3.jpg"></a></div><div id="largeobj_idm140158316006144" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5884124/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC5884124/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Partitioned heritability across known and novel gene sets</strong><p id="__p9">Heritable variation in KhoeSan pigmentation is partially explained by previously associated loci, newly associated loci, and candidate genes discovered in divergence studies of other populations, and in animal models. <strong>A</strong>) Schema illustrating how heritability analyses were used to partition the phenotypic variance explained by candidate gene sets (GS1, GS2) and novel associations (GS3) compared to the rest of the genome. <strong>B)</strong> Variance components analysis in GCTA comparing pigmentation variability explained by GS1, GS2, and the rest of the genome. Error bars span ± 1 standard error. <strong>C</strong>) Heritability explained by estimated value observed in our data (dot and arrow) versus matched null distribution in the ‡Khomani and Nama after accounting for number of SNPs in GS1 gene sets containing 14 genes previously associated with skin pigmentation in other populations. <strong>D</strong>) As in C), where GS2 = gene set from Table S4 of (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498155">Beleza et al., 2013</a>) compiled based on pigmentation function. See also <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S3</a>.</p></div></div></div><p id="P19" class="p p-last">We further assessed whether GS1 and GS2 explain more of the heritable variation than a random sample of coding regions; genes tend to explain more phenotypic variation than noncoding regions (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498109">Gusev et al., 2014</a>). After regressing out the effect of variable numbers of SNPs per gene set (<strong>STAR Methods</strong>), we find that both GS1 and GS2 explain more than random genes with a 10% false discovery rate (FDR=0.016 and FDR = 0.079, <a href="/pmc/articles/PMC5884124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3C–D</span></a>, respectively) across both KhoeSan populations. This is not significant in the Nama alone (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S3</a>), likely because of ancestry heterogeneity between the two populations.</p></div><div id="S7" class="sec"><h3 id="S7title">Replication of Known Pigmentation Associations in the KhoeSan</h3><p id="P20" class="p p-first">Even though previously identified pigmentation loci explain little of the phenotypic variance in our samples, it is possible that these loci simply have small effect sizes in the KhoeSan. We used SNP array and/or resequencing data in a linear mixed model with ancestry covariates (see Methods and “<em>Novel Variants Influence Skin Pigmentation in KhoeSan Populations</em>”) to assess both the frequencies and effect sizes of 42 previously identified eye, skin, and hair pigmentation variants, some of which have been experimentally shown to be causal (<a href="/pmc/articles/PMC5884124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Table S6A</a>). To this end, we also deconvolved recent admixture into local ancestry tracts across the genome and estimated the allele frequencies specifically on KhoeSan haplotypes via expectation-maximization. Known pigmentation allele frequencies vary considerably between the ‡Khomani San, Europeans, and West Africans (<a href="/pmc/articles/PMC5884124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). However, most previously identified pigmentation associations do not replicate with genome-wide significance or nominally in the ‡Khomani and Nama, with a few exceptions (<strong>STAR Methods</strong>).</p><p id="P21" class="p p-last">Because high divergence in a segment of the genome can be a signature of selection (e.g. XP-EHH scans), we assessed genetic divergence between KhoeSan, West African, and European populations at SNPs and in sliding windows across the genome. We find considerable divergence in many canonical pigmentation genes when comparing regions of the genome across populations (<a href="/pmc/articles/PMC5884124/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4A–B</span></a>). We followed up our divergence scan by focusing on two outlier genes that were highly diverged among all three populations: <em>SLC24A5</em> and <em>OCA2</em> (<a href="/pmc/articles/PMC5884124/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). The divergence in <em>SLC24A5</em> is among the highest in the genome, especially between the KhoeSan and European populations (<a href="/pmc/articles/PMC5884124/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4D</span></a>). Interestingly, different regions of <em>OCA2</em> exhibit elevated divergence between the KhoeSan and European comparison versus the KhoeSan and West African comparison (<a href="/pmc/articles/PMC5884124/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4C</span></a>). A previous study suggested that the derived, synonymous T allele of rs1800404 in <em>OCA2</em> has been positively selected and is a candidate skin pigmentation variant conferring light skin in Europeans and KhoeSan populations based on its global allele frequency distribution (<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498127">Norton et al., 2006</a>). We confirm its elevated allele frequency on KhoeSan haplotypes (65%), but do not find an association with skin pigmentation (p=0.53). Variants in <em>OCA2</em> explain most of the variation in human eye color (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498093">Duffy et al., 2007</a>), and rs1800404 was later significantly associated with this phenotype (Eriksson et al., 2010); ‡Khomani and Nama individuals notably have heterogeneous eye color, with a range of brown, hazel, and green eyes. We identified a missense mutation in <em>OCA2</em> (rs1800417, ns with skin pigmentation: p=0.87) with a derived allele (G) frequency of 0.32 in the KhoeSan (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S6A</a>) that is at low frequency in all other populations surveyed (global allele frequency = 0.016 in Phase 3 1000 Genomes and 0.0058 in the Exome Aggregation Consortium, ExAC).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC5884124/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140158312450480" class="figure"><a href="/pmc/articles/PMC5884124/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5884124_nihms921744f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms921744f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5884124/bin/nihms921744f4.jpg"></a></div><div id="largeobj_idm140158312450480" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5884124/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC5884124/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic divergence in genes previously associated with pigmentation</strong><p id="__p10">A–B) Distribution of weighted F<sub>ST</sub> in 20 kb moving windows of SNPs across the genome with a step size of 5 kb. Labels indicate where the maximal F<sub>ST</sub> window from each canonical pigmentation gene lies in the distribution. Divergence depicted is between A) the KhoeSan and Europeans, and B) the KhoeSan and West African populations. C–D) F<sub>ST</sub> in canonical pigmentation genes. Dots indicate SNPs, lines indicate moving averages over 20 kb windows with a step size of 5 kb. Canonical pigmentation loci/genes are shown as: C) the <em>OCA2</em>-<em>HERC2</em> locus, and D) the <em>SLC24A5</em> gene.</p></div></div></div></div><div id="S8" class="sec sec-last"><h3 id="S8title">Novel Variants Influence Skin Pigmentation in KhoeSan Populations</h3><p id="P22" class="p p-first">To identify novel variants associated with skin pigmentation in the ‡Khomani and Nama, we performed a 2-stage study (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6A</a>), employing a linear mixed model approach including recent admixture covariates as fixed effects and covariance matrices adjusted for admixture (akin to a GRM in GCTA) as random effects to identify associations between pigmentation and high quality imputed variants. We assessed the quality of the imputation via homozygous reference, heterozygous, and homozygous non-reference concordance with high coverage exome sequencing data (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S4A</a>). We ran the initial GWAS (i.e. phase 1) with imputed variants from 107 ‡Khomani and 109 Nama individuals (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S4</a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Table S5</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6A–C</a>), and the genes closest to the strongest associations (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S5</a>) showed a significant enrichment in multiple mammalian phenotypes related to skin pigmentation (abnormal extracutaneous pigmentation p=2.3e-3, abnormal melanocyte morphology p=5.8e-3, abnormal skin morphology p=3.5e-2). Further, the strongest signals across the genotyped ‡Khomani and Nama cohorts were near canonical pigmentation genes (e.g. <em>TYRP1</em> and <em>SLC24A5</em>), genes associated with pigmentation-related disorders (e.g. <em>TYRP1</em>), or genes implicated in pigmentation in model organisms and <em>in vitro</em> studies (e.g. <em>VLDLR</em>, <em>SMARCA2</em>, and others) (<a href="#R66" rid="R66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498188">Sturm, 2009</a>) (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498108">Keenen et al., 2009a</a>; <a href="#R82" rid="R82" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498207">Xia et al., 2013a</a>). To assess the variation explained by the most significantly associated loci, we generated an additional gene set, referred to as “GS3”, using the 50 most significantly associated loci ± 10 kb. We find that the GS3 loci explain significantly more of the heritable variation in skin pigmentation than previously identified pigmentation candidate genes in the KhoeSan, but that the majority of heritable variation remains to be explained (<a href="/pmc/articles/PMC5884124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3B</span></a>, σ<sup>2</sup><sub>GS3</sub>=0.23±0.13, p<sub>GS3</sub>=0.027 vs σ<sup>2</sup><sub>Genome</sub>=0.64±0.08, p<sub>Genome</sub>&lt;1e-5).</p><p id="P23">Based on initial evidence from the imputed ‡Khomani pigmentation GWAS, we designed a targeted NGS capture and successfully resequenced 36 candidate pigmentation regions (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S7</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6</a>) across a larger set of 451 KhoeSan samples in order to improve power to detect associated loci (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S8</a>, <strong>STAR Methods</strong>), including 269 ‡Khomani, and 182 Nama individuals. In this larger sample, we observe more variants significantly associated with pigmentation than expected by chance in the resequencing regions (<a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5A</span></a>). The strongest signal comes from SNPs in <em>SLC24A5</em>, 8 of which are all in high pairwise LD (R<sup>2</sup> &gt; 0.6) on a high frequency haplotype (<a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5B</span></a>). We identify significant associations between lighter skin and derived <em>SLC24A5</em> SNPs, including the putatively causal p.Thr111Ala rs1426654 allele (β=−3.58 on M index scale, p=9.8e-9), which has previously been associated with skin pigmentation in Eurasians. The most strongly associated SNP (rs2555364, β=−3.58 on M index scale, p=6.7e-9) is tightly linked with rs1426654 (LD R<sup>2</sup>=0.81). These variants are strongly differentiated between Europeans and Africans, with rs1426654 having derived allele frequencies of 99.7% vs 5.5% in 1000 Genomes (excluding ASW and ACB populations with recent European admixture), respectively. The derived allele of rs1426654 has previously been observed in HGDP Ju|’hoansi San samples at 7% frequency, which have no detectable recent European admixture (Norton et al., 2007). The frequency of the derived rs1426654 allele is 40% in the combined Nama + ‡Khomani dataset, which is significantly greater than expected from ~11% European admixture alone (binomial test p=7.8e-52, <a href="#SD2" rid="SD2" class=" supplementary-material">Table S6A</a>, <strong>STAR Methods</strong>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140158310279888" class="figure"><a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5884124_nihms921744f5.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms921744f5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5884124/bin/nihms921744f5.jpg"></a></div><div id="largeobj_idm140158310279888" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5884124/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC5884124/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Associations between genetic data and baseline pigmentation</strong><p id="__p11">A) Targeted resequencing QQ plot. 95% confidence interval on the QQ plot is drawn assuming the <em>j</em><sup>th</sup> order statistic from a uniform sample follows a <em>Beta</em>(<em>j</em>, <em>n</em> − <em>j</em> + 1) distribution. Colors differentiate loci containing more than one variant associated more significantly than the 95% confidence interval in a region. B–C) LocusZoom plots of targeted resequencing genetic associations incorporating KhoeSan-specific LD. Recombination rates are from HapMap b37. Regions include: B) <em>SLC24A5</em>, and C) 5 independent signals associated with p &lt; 1e-3 in/near <em>SMARCA2</em> and <em>VLDLR</em>. D) LocusZoom plot of suggestive association in/near <em>SNX13</em> from meta-analysis of phase 1 and phase2 imputed associations with KhoeSan-specific LD. See also <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6</a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Table S5, Table S7, and Table S6B</a>.</p></div></div></div><p id="P24">Multiple low frequency (&lt;5%) SNPs near several additional genes, including <em>EPM2A</em>, <em>FREM1</em>, <em>SMARCA2</em>/<em>VLDLR</em>, and <em>TYRP1</em>, are above the 95% confidence interval of expected versus observed significance (<a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>). Two of these loci are near <em>EPM2A</em> and <em>FREM1</em>, neither of which have any known role in skin pigmentation in humans or model organisms. In contrast, there are &gt;5 independent low frequency signals with p&lt;1e-3 within/near <em>SMARCA2</em> and <em>VLDLR</em>, with rs7866411 (p=8.91e-5) and rs2093835 (p=1.17e-4) being most significantly associated with skin pigmentation. We used HaploReg to infer regulatory activity in/near these peaks and identify multiple enhancer and DNAse peaks identified in skin, including melanocytes and/or keratinocytes, overlapping top tag and/or perfectly linked SNPs (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S6B</a>). We also identify a low frequency association (rs34803545, p=3.7e-4) ~600 kb upstream of <em>TYRP1</em>. This variant is perfectly linked with multiple conserved variants, one of which exhibits enhancer activity and DNAse hypersensitivity specifically in skin (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S6B</a>).</p><p id="P25" class="p p-last">We performed a 2<sup>nd</sup> phase of GWAS, in which an additional 240 individuals were genotyped (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S4</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6A</a>) and meta-analyzed with phase 1 summary statistics. While two tanning status associations met genome-wide significance, none of the loci contained linkage peaks, suggesting that they are most likely spurious as expected from a phenotype with low heritability. As expected from the resequencing study, we identified a genome-wide significant association in <em>SLC24A5</em> (rs2470102 derived allele β = −3.4, p=3.6e-12) and a suggestive association upstream of <em>TYRP1</em> (chr9:12088112, frequency=0.014, β = −13.6, p=1.1e-07 <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6B–C</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6F–G</a>). We identified an additional suggestive novel association in and near <em>SNX13</em>, with common derived T alleles of rs2110015 associated with light skin (β = −3.1, p=1.3e-07, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6H</a>); <em>SNX13</em> regulates lysosomal degradation and G-protein signaling, but has not previously been associated with skin pigmentation.</p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Discussion</h2><p id="P26" class="p p-first">Pigmentation has been described previously as a relatively simple trait with few loci of large effect contributing to the phenotype (<a href="#R69" rid="R69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498152">Sulem et al., 2007</a>; <a href="#R77" rid="R77" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498199">Walsh et al., 2013</a>). However, populations living in continental Africa, where humans have the greatest genetic diversity and pigmentation variability, have been largely ignored in genetic studies with quantitative phenotypes. We investigated the genetic architecture of pigmentation in two KhoeSan populations: the ‡Khomani San and Nama, where baseline melanin variation is substantial. Southern African KhoeSan populations are the most polymorphic modern human populations yet studied (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498195">Henn et al., 2011</a>), and provide a unique glimpse into the evolution of pigmentation.</p><div id="S10" class="sec"><h3 id="S10title">Novel Genetic Associations with Pigmentation</h3><p id="P27" class="p p-first-last">We have performed the first genetic discovery effort for pigmentation loci in the Nama and ‡Khomani San populations. The strongest allelic associations include previously associated variants, noncoding regions near canonical pigmentation genes, and novel genes shown in model organisms to have a role in pigmentation. The strongest association is in <em>SLC24A5</em>, which is a well-known pigmentation gene (<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498096">Lamason, 2005</a>) and is among the most differentiated regions of the genome between European and African populations–indicative of strong positive selection in northern Europeans (<a href="#R67" rid="R67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498087">Sturm and Duffy, 2012</a>). We find that derived variants in <em>SLC24A5</em> are at high frequency in the KhoeSan, including missense mutations that influence skin and eye pigmentation (<a href="/pmc/articles/PMC5884124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Notably, these variants are segregating at higher frequency than expected by recent European admixture alone. Three possible evolutionary scenarios that may explain these elevated frequencies are: 1) these variants arose in southern Africa more than 100,000 years ago and were later selected for in Europeans after the out-of-Africa migration in response to northern UVR environments. Alternatively, 2) these variants arose in Europe/Near East, were introduced into KhoeSan populations via “back to Africa” migration into southern Africa predating 17<sup>th</sup> century European colonialism (<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498167">Pickrell et al., 2014</a>; <a href="#R73" rid="R73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498107">Uren et al., 2016a</a>), and have since been positively selected in the KhoeSan. Lastly, 3) a recurrent mutation (G to A transition at the CpG ancestral dinucleotide, a class of mutations shown to have elevated mutation rates) occurred. Considerable future work is needed to definitively disentangle these scenarios.</p></div><div id="S11" class="sec"><h3 id="S11title">The Polygenic Architecture of Pigmentation in Africa</h3><p id="P28" class="p p-first-last">We assessed the heritability of baseline skin pigmentation, and find that it is virtually completely heritable in our KhoeSan sample. In contrast, tanning status is primarily environmental, with heritability estimates which are not significantly different from zero. In European populations, predictive models based on only 9 SNPs capture up to 16% of the variance in skin pigmentation (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498111">Liu et al., 2015</a>), highlighting its relative simplicity. We applied a predictive model based on these SNPs to the Nama and ‡Khomani San populations, and find no significant association between predicted skin color and spectrophotometrically measured skin M index, showing that this estimation fails to capture the genetic variation driving the phenotype in the KhoeSan. Given the large effect sizes and high fraction of variation explained in Eurasian populations, we asked whether and how much of the phenotypic variation can be explained by previously identified genes. All gene sets, including previously associated loci, canonical pigmentation genes, and the most significantly associated variants in this study, explained a small fraction of the phenotypic variance (σ<sup>2</sup><sub>GS1</sub>=0.08, σ<sup>2</sup><sub>GS2</sub>=0.09, σ<sup>2</sup><sub>GS3</sub>=0.23, respectively). As expected from previous work (<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498180">Martin et al., 2017</a>), our results indicate that genetic risk prediction is strongly affected by population structure. Most of the pigmentation variability in KhoeSan populations is not explained by previously identified loci, suggesting that more than 50 loci (and indeed, likely far more, given our genomic heritability estimates) with a distribution of mostly small effects contribute to variation in pigmentation in the KhoeSan. This suggests that skin pigmentation is a far more complex trait than previously discussed, analogous to numerous other complex traits discussed in biomedical literature.</p></div><div id="S12" class="sec"><h3 id="S12title">The Evolution of Skin Pigmentation: Selection and Constraint</h3><p id="P29" class="p p-first">By aggregating a large set of quantitative skin pigmentation phenotypes (N=4,712) from globally diverse populations, we have demonstrated heteroskedasticity as a function of latitude. As observed previously, we find a strong correlation between absolute latitude and average skin pigmentation reflectance caused by melanin content. We also observe that populations with lighter skin have reduced variation within any given study: populations furthest from the equator have narrower distributions, while populations closest to the equator have wider distributions. These patterns suggest that selection is acting differently at different latitudes. In equatorial regions, strong directional selection for darker pigmentation has shifted the distribution means in some populations to M indices greater than 90, but with wide variances. This is consistent with a ‘threshold’ model (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498139">Chaplin, 2004</a>) in which the protective benefit of melanin needs to meet some minimum threshold but with no penalty to darker pigmentation; alternatively, diversifying selection could maintain the wide variance.</p><p id="P30">In stark contrast, pigmentation in far northern European and Asian populations has been under directional selection for decreased melanin production, reflected by very narrow distributions. There may be biological constraints on the lower boundary of skin pigmentation, and/or due to the strong positive selection acting on a few large-effect alleles, there is little genetic variability left at these pigmentation loci. This would simplify the genomic architecture, with relatively few alleles of large effect driving the phenotype, particularly alleles that lighten skin at extreme northern latitudes, and could explain why prior investigations observed an almost Mendelian inheritance of large effect light pigmentation alleles.</p><p id="P31" class="p p-last">Finally, populations at intermediate latitudes have increased variance and higher means than populations in northern Eurasia, but less than equatorial populations. The most parsimonious explanation for this pattern is that stabilizing selection affects the light and dark tails of the pigmentation distribution (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498121">Barton, 1999</a>). The Nama and ‡Khomani San appear to be two such instances of this intermediate variation within Africa, likely attributable to their geographic distance from the equator in far southern Africa (~24–29 degrees South). The observed mean and variance differences across the full spectrum of skin pigmentation by latitude may be driven by imbalanced opposing adaptive pressures, where selective forces to produce vitamin D and protect folate from photolysis are unequal and change in response to UV radiation exposure. Given our heritability results and the observed variability in baseline pigmentation; light skin pigmentation in the KhoeSan appears to be due to a combination of many small-effect mutations as well as some large-effect variants. The evolution of the pigmentation phenotype in these populations cannot be explained in terms of only a few variants segregating in Eurasians. A fuller characterization of the genes underlying the architecture in Africans is needed before we can distinguish between the hypothesis of directional versus stabilizing selection across different latitudes (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498154">Berg and Coop, 2014</a>).</p></div><div id="S13" class="sec sec-last"><h3 id="S13title">Conclusion</h3><p id="P32" class="p p-first-last">Because African populations often carry the ancestral (i.e. dark) allele for skin pigmentation genes identified in Eurasians, allusions to African skin pigmentation have ignored the great variability in this phenotype across Africa. Here, we reiterate that skin pigmentation varies more in Africa than any other continent, and we show that pigmentation in African populations cannot simply be explained by the small number of large effect alleles discovered in Eurasians. Even in light to moderately pigmented KhoeSan populations, the polygenicity of skin pigmentation is much greater than Eurasians, encompassing both known pigmentation genes as well as novel loci. We argue that the distributions of skin pigmentation globally suggest different forces of selection operating at various latitudes. To better understand baseline pigmentation, one of the most rapidly-evolving traits and strongest cases for positive selection in humans, it is essential to quantitatively measure and study pigmentation in a large set of genetically diverged populations that have historically been exposed to different levels of UV radiation. As human genetics moves to ever larger studies of complex traits, the full picture of genetic architecture will remain incomplete without representation from diverse worldwide populations.</p></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">STAR Methods</h2><div id="S15" class="sec sec-first"><h3 id="S15title">CONTACT FOR REAGENT AND RESOURCE SHARING</h3><p id="P33" class="p p-first-last">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Brenna Henn (<a href="mailto:dev@null" data-email="ude.koorbynots@nneh.annerb" class="oemail">ude.koorbynots@nneh.annerb</a>).</p></div><div id="S16" class="sec"><h3 id="S16title">EXPERIMENTAL MODEL AND SUBJECT DETAILS</h3><div id="S17" class="sec sec-first-last"><p></p><h4 id="S17title" class="inline">Sample collection and ethics approval </h4><p id="P34" class="p p-first-last">As described previously (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498137">Henn et al., 2011</a>; <a href="#R74" rid="R74" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498202">Uren et al., 2016b</a>), sampling of the ‡Khomani San took place in the Northern Cape of South Africa in the southern Kalahari Desert region (near Upington and neighboring villages) in 2006, 2010, 2011, 2013, and 2015. Sampling of the Nama took place in the Richtersveld in 2014 and 2015. Institutional review board (IRB) approval was obtained from Stanford University, Stony Brook University, and the University of Stellenbosch, South Africa. ‡Khomani N|u-speaking individuals, Nama individuals, local community leaders, traditional leaders, nonprofit organizations, and a legal counselor were all consulted regarding the aims of the research before collection of DNA (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498205">Henn et al., 2011</a>). Research was conducted with the permission of the Working Group of Indigenous Minorities in Southern Africa (WIMSA) and, subsequently, the South African San Council. All individuals gave signed written and verbal consent with a witness present before participating. Individuals collected in 2006 were re-consented under an updated protocol. Ethnographic interviews of all individuals were conducted, including questions about age, language, place of birth, and ethnic group of the individual and of his/her mother, maternal grandparents, father, and paternal grandparents. All individuals included in the study were adults (age range of 18 to 94, mean=53, sd=18). Both men and women were included in the study (311 females, 190 males). We recorded the relationships between any sampled individuals if revealed during the interview. Ages of older individuals were verified with separate interviews regarding reproductive history. DNA was obtained via saliva, collected using Oragene saliva collection kits (DNAGenotek, Ontario, Canada).</p></div></div><div id="S18" class="sec"><h3 id="S18title">METHOD DETAILS</h3><div id="S19" class="sec sec-first"><p></p><h4 id="S19title" class="inline">Skin reflectance measurements </h4><p id="P35" class="p p-first">A portable reflectance spectrophotometer (DermaSpectrometer DSMII ColorMeter, Cortex Technology, Hadsund, Denmark) was used to measure skin pigmentation. Similar devices have previously been shown to measure melanin and hemoglobin (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498092">Diffey et al., 1984</a>). The melanin content, <em>M</em> index is quantified as</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M10" display="block" overflow="scroll"><mrow><mi>M</mi><mo>=</mo><msub><mo>log</mo><mn>10</mn></msub><mspace width="0.16667em"></mspace><mrow><mo>(</mo><mfrac><mn>1</mn><mrow><mo>%</mo><mspace width="0.16667em"></mspace><mtext>red</mtext><mspace width="0.16667em"></mspace><mtext>reflectance</mtext></mrow></mfrac><mo>)</mo></mrow></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M10" display="block" overflow="scroll"><mrow><mi>M</mi><mo>=</mo><msub><mo>log</mo><mn>10</mn></msub><mspace width="0.16667em"></mspace><mrow><mo>(</mo><mfrac><mn>1</mn><mrow><mo>%</mo><mspace width="0.16667em"></mspace><mtext>red</mtext><mspace width="0.16667em"></mspace><mtext>reflectance</mtext></mrow></mfrac><mo>)</mo></mrow></mrow></math></script></div></div><p id="P36" class="p p-last">The device was calibrated to 0 as suggested by the manufacturer twice a day while sampling. Five measurements of M index were taken on each of the left and right upper inner arms to approximate baseline constitutive skin pigmentation. We also measured the dorsal side of the left or right wrist (i.e. an area exposed to sunlight) and subtracted the baseline pigmentation for a measure of tanning status (<a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498095">Shriver and Parra, 2000</a>). For the remainder of the analyses, we used the trimmed phenotype means (highest and lowest values removed); we also averaged the inner arm skin pigmentation measurements over the two arms.</p></div><div id="S20" class="sec"><p></p><h4 id="S20title" class="inline">Genotyping platforms </h4><p id="P37" class="p p-first-last">A total of 471 KhoeSan samples were genotyped across all arrays, including the Illumina 550k array, Illumina OmniExpress and OmniExpressPlus arrays, Illumina Omni2.5 array, and the Illumina MEGA array, some of which has been described individually previously. 35 ‡Khomani, 21 Hadza, and 35 Sandawe individuals were previously genotyped on the Illumina Beadchip 550K custom v2 chip (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498198">Henn et al., 2011</a>). 86 ‡Khomani and 13 Nama were genotyped on the Illumina OmniExpress and OmniExpressPlus arrays (same base content, additional exome content in the Plus version of the array) (<a href="#R74" rid="R74" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498143">Uren et al., 2016b</a>). 105 Nama individuals were genotyped on the Illumina Omni2.5 array as part of the African Genome Diversity Project. 185 ‡Khomani and 84 Nama individuals were genotyped on the Illumina MEGA array. <a href="#SD2" rid="SD2" class=" supplementary-material">Table S4</a> indicates the number of samples genotyped on each array platform, as well as the overlapping phenotypes, exome sequencing, and targeted resequencing. A small number of individuals overlapped between multiple arrays to perform QC.</p></div><div id="S21" class="sec"><p></p><h4 id="S21title" class="inline">Global ancestry estimation </h4><p id="P38" class="p p-first">We derived two sets of genome-wide ancestry estimates: one from a smaller set of genotyping array data and one from a larger set of samples with targeted resequencing data. For the former set of estimates, we included genotype data from the Human Genome Diversity Project (CEPH-HGDP, sample sizes in parentheses) as reference samples, including the South African Bantu (8), Kenyan Bantu (11), Namibian San (6), Mozabites (29), and French (28). We also included genotype data from 12 Namibian San individuals from <a href="#R62" rid="R62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498158">Schuster et al (2010)</a>, as well as individuals from the Hadza (17) and Sandawe (28) of Tanzania, described by <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498173">Henn et al (2011)</a>. We also included individuals genotyped in the HapMap Project, including Yoruba trio parents from Ibadan, Nigeria (YRI, 55), Centre d’Etude du Polymorphisme Humain (CEPH) Utah residents with ancestry from northern and western Europe (CEU) trio parents (86), and Maasai trio parents from Kinyawa, Kenya (MKK, 30) individuals (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498190">Altshuler et al., 2010</a>). Because of the high degree of relatedness in our dataset, we then split the merged ‡Khomani and Nama data into 11 groups of maximally unrelated KhoeSan individuals from this study based on ethnographic information, then merged in these samples as well, holding the reference panels constant. After merging the SNPs genotyped in the HapMap, CEPH-HGDP, and South African samples, and removing SNPs with any genotype missingness using PLINK 2, SNPs with minor allele frequency &lt;1%, and SNPs in high LD (r<sup>2</sup> &gt; 0.9) a total of 215,607 SNPs remained. All datasets were merged to Human Genome Build hg19 as above and dbSNP v138. We ran ADMIXTURE for <em>k</em> = 3–7 in unsupervised mode for each of the 11 groups, then matched clusters across runs. In runs where we identified multimodality, we further split running groups of KhoeSan samples resulting in a minor mode into two sets, which resulted in unimodality across all runs. After matching clusters, we merged ancestry estimates across all 11 running groups, averaging individuals that appeared in multiple running groups. We chose <em>k</em>=7 as the most stable and best representation of ancestry.</p><p id="P39">For the larger resequencing data, we extracted sequence data in the targeted resequencing intervals (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S7</a>) for 99 CEU and 99 LWK samples from the 1000 Genomes Project bam files and generated gVCF files, then called variants jointly with HaplotypeCaller. We included unrelated KhoeSan samples estimated to have &gt;90% KhoeSan ancestry from the genotype-based ancestry estimates as reference samples, then ran ADMIXTURE in supervised mode, projecting related and more admixed samples.</p><p id="P40" class="p p-last">Using the same data, we also estimated ancestry estimates using PCA. Because of the elevated relatedness and admixture in our data, we applied the PC-AiR and PC-Relate approaches (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498192">Conomos et al., 2016</a>).</p></div><div id="S22" class="sec"><p></p><h4 id="S22title" class="inline">Covariates </h4><p id="P41" class="p p-first-last">We performed forward stepwise regression using custom scripts in R to select the best multivariate mixed model of ancestry, age, and sex for pigmentation and tanning with a random effect accounting for the genetic relationships among individuals. Sex and age do not significantly correlate with baseline skin pigmentation, suggesting that our quantitative measure of underarm reflectance is not significantly affected by UV exposure. The best model fit, measured via AIC, included Bantu, European, East African, and Hadza ancestries, although the latter two components comprise ≤ 1% of individuals’ total ancestry on average and are likely imprecise.</p></div><div id="S23" class="sec"><p></p><h4 id="S23title" class="inline">Identity-by-descent (IBD) haplotype sharing </h4><p id="P42" class="p p-first-last">To estimate IBD, we phased intersected genotypes for the ‡Khomani and Nama populations both separately (number of SNPs = 300,370 in ‡Khomani, 525,934 in Nama) and jointly (number of SNPs = 241,929) using Beagle (v4.1) (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498131">Browning and Browning, 2007</a>). Adjusting for differences in SNP density, we used a sliding window size of 600 markers with 55 overlapping SNPs between each window for the ‡Khomani, a window size of 1000 markers with 90 overlapping SNPs between each window for the Nama, and a window size of 400 markers with 39 overlapping SNPs between each window for the joint ‡Khomani and Nama intersection, with 10 iterations per run. The phased data was then used to infer haplotypes shared via IBD with length ≥ 5 cM using Germline (v.1.5.1) with the following flags for the joint haplotype calls: “-w_extend -min_m 5 -err_hom 2 -err_het 5 -bits 60” (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498146">Gusev et al., 2009</a>). This allowed a mismatch of 2 homozygous and 5 heterozygous markers. We verified the total genomic length of the inferred cumulative IBD between pairs of individuals by comparing to pedigree relationships identified from ethnographic interviews and verified with IBD inferred here from the genotyping arrays.</p></div><div id="S24" class="sec"><p></p><h4 id="S24title" class="inline">Covariance matrices </h4><p id="P43" class="p p-first">To account for the considerable relatedness in our samples, which have variable degrees of admixture, we evaluated multiple covariance matrices: a Balding-Nichols matrix computed via EMMAX (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498178">Kang et al., 2010</a>), a genetic relationship matrix (GRM) computed via GCTA (<a href="#R84" rid="R84" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498203">Yang et al., 2010</a>), and a kinship matrix computed via REAP (<a href="#R71" rid="R71" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498084">Thornton et al., 2012b</a>). To generate the REAP matrix, we intersected genotype data for all individuals, then included P and Q matrices obtained from an ADMIXTURE run with k=3 (described in <em>Ancestry estimation</em>) to construct the ancestry-corrected kinship matrix. Briefly, this approach uses individual-specific allele frequencies at SNPs that are calculated on the basis of genome-wide ancestry. We compared inferred pairwise kinship values in all covariance matrices to ethnographically and genetically validated pedigree information. We used the REAP matrix to correct for kinship in all regression models unless otherwise noted because it correlated best with true relationships (e.g. heritability analyses with different kinship matrices).</p><p id="P44" class="p p-last">We also constructed a kinship matrix using pairwise IBD estimates (<em>K<sub>IBD</sub></em>), as previously (<a href="#R89" rid="R89" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498170">Zaitlen et al., 2013</a>), with haplotypes sharing calls computed as described above. We constructed a kinship covariance matrix based on IBD (<em>K<sub>IBD</sub></em>), where the entry for individuals <em>j</em> and <em>k</em> are defined as follows: 
</p><div class="disp-formula" id="FD2"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M11" display="block" overflow="scroll"><mrow><mfrac><mrow><msub><mo>∑</mo><mi>i</mi></msub><msub><mi>L</mi><mi>i</mi></msub></mrow><msub><mi>L</mi><mrow><mi mathvariant="italic">parent</mi><mo>-</mo><mi mathvariant="italic">offspring</mi></mrow></msub></mfrac></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M11" display="block" overflow="scroll"><mrow><mfrac><mrow><msub><mo>∑</mo><mi>i</mi></msub><msub><mi>L</mi><mi>i</mi></msub></mrow><msub><mi>L</mi><mrow><mi mathvariant="italic">parent</mi><mo>-</mo><mi mathvariant="italic">offspring</mi></mrow></msub></mfrac></mrow></math></script></div></div><p> where <em>L<sub>i</sub></em> is the genetic length in centimorgans of <em>i</em>th IBD segment between individual j and k, and <em>L<sub>parent–offspring</sub></em> is the total length of IBD in centimorgans shared between a parent and an offspring (i.e. the callable length of the haploid genome).</p></div><div id="S25" class="sec"><p></p><h4 id="S25title" class="inline">Heritability </h4><p id="P45" class="p p-first-last">Heritability estimates were calculated across the full KhoeSan sample (Nama and ‡Khomani) in addition to within each population separately. We used GCTA restricted maximum likelihood (REML) analysis to compute SNP-based heritability (
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M12" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M12" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></script>) in multiple ways with differing covariance matrices. For all heritability analyses of baseline pigmentation, we included European and Bantu ancestry proportion estimates at <em>k</em>=7 from ADMIXTURE as quantitative covariates. For tanning status, we included age as a quantitative covariate and sex as a binary covariate. We assessed 
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M13" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M13" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></script> by fitting an unconstrained linear mixed model (--reml-no-constrain) in GCTA (<a href="#R84" rid="R84" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498077">Yang et al., 2010</a>), once using a covariance matrix constructed with REAP, and once with a genetic relationship matrix (GRM) generated in GCTA. We estimated heritability from the exome data (
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M14" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">exome</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M14" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">exome</mi><mn>2</mn></msubsup></mrow></math></script>) similarly using a GRM generated in GCTA with exome sequencing data from 82 ‡Khomani individuals. We also estimated heritability (
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M15" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">IBD</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M15" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">IBD</mi><mn>2</mn></msubsup></mrow></math></script>) in our study using a kinship matrix constructed from pairwise IBD estimates (<em>K<sub>IBD</sub></em>), as previously (<a href="#R89" rid="R89" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498106">Zaitlen et al., 2013</a>). Lastly, we estimated narrow sense heritability using pedigree relationships (
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M16" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">pedigree</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M16" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">pedigree</mi><mn>2</mn></msubsup></mrow></math></script>) that were constructed from ethnographic interviews and subsequently genetically confirmed using the Sequential Oligogenic Linkage Analysis Routines (SOLAR) software (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498163">Almasy and Blangero, 1998</a>). SOLAR employs maximum likelihood variance decomposition to determine narrow-sense <em>h<sup>2</sup></em> assuming a normal distribution. SOLAR employs maximum likelihood variance decomposition to determine narrow-sense <em>h<sup>2</sup></em> assuming a normal distribution. It calculates heritability utilizing pairwise coefficients of genetic relatedness in the full pedigree, including dummy link individuals. The “polygenic” command was used to calculate trait polygenic heritability, significance of 
<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M17" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">pedigree</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M17" display="inline" overflow="scroll"><mrow><msubsup><mi>h</mi><mi mathvariant="italic">pedigree</mi><mn>2</mn></msubsup></mrow></math></script>, and the proportion of variance contributed by covariates, with the “screen” flag to assess the significance level of each covariate.</p></div><div id="S26" class="sec"><p></p><h4 id="S26title" class="inline">Variance partitioning </h4><p id="P46" class="p p-first">We partitioned heritability in two ways: 1) by comparing the heritability explained by candidate gene sets versus the rest of the genome, and 2) by comparing the heritability explained by candidate gene sets to randomly sampled genes. For the first type of analysis, we generated GRMs based on SNPs that fall within pigmentation candidate gene sets, including GS1 (genes in <a href="/pmc/articles/PMC5884124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), GS2 (Table S4 from (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498098">Beleza et al., 2013</a>)), GS3, and the rest of the genome. We performed a restricted likelihood ratio tests comparing the heritability explained by each gene set to the rest of the genome. We estimated partitioned heritability explained by different gene sets using joint linear mixed models, by including multiple genetic variance components as random effects.</p><p id="P47" class="p p-last">For the second type of analysis, we sought to determine how likely we are to find a candidate pigmentation gene set explaining more of the heritable variation than a random gene set. To do this, we matched both candidate gene sets by number of genes, length, and number of exons and permuted these matched samples 1000 times. Specifically, we generated a GRM in GCTA based on SNPs in these candidate genes, then calculated the heritability based on this GRM. Then, we binned all genes in the genome by the natural log of their lengths (absolute value of transcription end - transcription start) and number of exons. For each gene in the gene set, we sampled with replacement from its matched length and exon bin, and constructed 1000 matched gene sets. To create an empirical null distribution, for each of the 1000 matched gene sets, we constructed a GRM, and computed heritability. We then regressed out the effect of number of SNPs on heritability explained, then generated empirical false discovery rates by comparing the residual heritability of the true candidate gene set to the residual heritability of the matched empirical null distribution.</p></div><div id="S27" class="sec"><p></p><h4 id="S27title" class="inline">Categorical pigmentation prediction </h4><p id="P48" class="p p-first-last">As described previously (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498182">Hart et al., 2013b</a>; <a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498187">Spichenok et al., 2011</a>), published categorical skin color prediction models utilize 7 SNPs (rs12913832, rs1545397, rs16891982, rs1426654, rs885479, rs6119471, rs12203592) in or nearby pigmentation genes. The model follows a bifurcating decision tree, dependent on homozygous state at each locus. At any two loci except rs6119471, if both are homozygous derived, then the phenotype is predicted as “non-dark”, i.e. medium or light. Further “light” pigmentation is confirmed if all three loci: rs12913832, rs16891982, and rs1426654 are homozygous derived. A “non-light”, i.e. medium or dark, is predicted if rs6119471 is homozygous ancestral (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6</a>).</p></div><div id="S28" class="sec"><p></p><h4 id="S28title" class="inline">Replication of known pigmentation loci </h4><p id="P49" class="p p-first-last">Few known loci replicate with genome-wide significance or even marginally in the KhoeSan populations studied here. Four SNPs in the genes <em>SLC45A2</em> (rs16891982, p=1.2e-3), <em>KITLG</em> (rs12821256, p=0.02), and <em>SLC24A5</em> (rs1426654, p=9.8e-9 and rs2470102, p=1.1e-8) marginally replicate in the ‡Khomani + Nama under an additive model. The derived allele frequencies of the associated SNPs in <em>SLC45A2</em> and <em>KITLG</em> are low in the KhoeSan, consistent with ~10% admixture from recent European gene flow. Interestingly, however, SNPs in <em>OCA2</em>, <em>SLC24A5</em> and <em>GRM5</em>/<em>TYR</em> are at much higher frequencies in both the ‡Khomani and Nama than expected from European admixture alone, as estimated from global ancestry (see “<em>Global ancestry estimation</em>”). We do not replicate the vast majority of previously observed skin pigmentation associations in our dataset, potentially due to low frequencies in the KhoeSan, power limitations, differentiated LD structure in which the tag SNPs are non-causal pigmentation alleles, or epistatic effects. It is therefore unsurprising that when we applied forensic models based on only seven SNPs that claim very high prediction accuracy of skin color across populations (&gt;99%) (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498110">Hart et al., 2013b</a>; <a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498157">Spichenok et al., 2011</a>), we did not find a significant association with quantitatively measured M index (p=0.31, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5B</a>).</p></div><div id="S29" class="sec"><p></p><h4 id="S29title" class="inline">Phasing and imputation </h4><p id="P50" class="p p-first-last">We first ensured uniform SNP IDs by orienting all variants to dbSNP 138, then merged genotype data for KhoeSan individuals across all genotyping platforms (Illumina 550k, OmniExpress, OmniExpressPlus, and Omni2.5). We then phased all ‡Khomani (N=121) and Nama (N=112) individuals together with Shapeit2 (v2.r778) using all available genetic data. We used the full Phase 3 1000 Genomes reference panel, consisting of haplotypes from 2,535 individuals to aid phasing accuracy. Shapeit imputes missing genotypes, which can result in array-specific technical artifacts. We mitigated technical artifacts from individuals genotyped on different arrays by subsetting haplotypes to variants genotyped only the array, resulting in four sets of haplotypes. We then imputed variants in 5 Mb windows for all 4 sets using the full 1000 Genomes phase 3 reference panel as well as 53 HGDP medium coverage genomes (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498120">Henn et al., 2016</a>) with Impute2 (v2.2.2) for all runs. After imputing each array separately, we aggregated the data across windows and runs, including only sites that were imputed with an Impute2 info metric ≥ 0.8 across all sets and subset to sites with MAF ≥ 0.01. We assessed the accuracy of the imputation in three ways. First, we assessed the homozygous reference, heterozygous, and homozygous non-reference concordance between the imputed output and two low-pass genome sequences from individuals SA1000 and SA1025 at sites that passed variant call filters and with &gt; 5 reads. Next, we assessed concordance similarly across all 79 individuals for whom we have both genotype and high coverage exome sequencing data (see <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S4</a>). Finally, we ran PCA for 100,000 randomly selected imputed sites across all individuals as well as for the maximum number of unrelated individuals to test whether the primary source of aggregated imputed variation arose from technical artifacts or population/familial structure. By investigating the top PCs, we concluded the latter.</p></div><div id="S30" class="sec"><p></p><h4 id="S30title" class="inline">Local ancestry inference </h4><p id="P51" class="p p-first-last">To disentangle haplotypes specific to a given ancestry and estimate ancestry-specific allele frequencies, we inferred local ancestry along chromosomes for all the genotyped ‡Khomani (N=121) and Nama (N=112) individuals, as described in (<a href="#R74" rid="R74" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498075">Uren et al., 2016b</a>). We phased haplotypes as described above. As reference panels, we defined separate classes for European, Bantu, and KhoeSan ancestries respectively using CEU, LWK, and ‡Khomani individuals from this study as well as KhoeSan individuals from a previous study (<a href="#R62" rid="R62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498194">Schuster et al., 2010</a>) with &gt;90% KhoeSan ancestry as inferred via ADMIXTURE (see “<em>Ancestry estimation</em>”). We used RFMix (v1.5.4) to assess local ancestry at sites that intersected between the reference panels using an iterative expectation maximization (EM) approach with 0.2 cM windows, incorporating the reference panel throughout EM iterations and correcting potential phase errors. We used a node size of 5 to deal with class imbalances in our reference panels. For all individuals, we used calls from RFMix at the 1<sup>st</sup> iteration.</p></div><div id="S31" class="sec"><p></p><h4 id="S31title" class="inline">Allele frequency approximation </h4><p id="P52" class="p p-first-last">Because the ‡Khomani San are a recently admixed population, we estimated allele frequencies with consideration to local ancestry calls. We specifically estimated allele frequencies on KhoeSan haplotypes using an expectation maximization approach (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498138">Gravel et al., 2013</a>). Briefly, we used Bayes’ Rule to calculate the expected frequency given the observed genotype and diploid local ancestry calls. Ancestral/derived state were determined from great ape genome sequencing, where possible (<a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498175">Prado-Martinez et al., 2013</a>).</p></div><div id="S32" class="sec"><p></p><h4 id="S32title" class="inline">Mixed-model association approach </h4><p id="P53" class="p p-first-last">To identify loci significantly associated with baseline skin pigmentation and tanning status, we associated high quality imputed and resequenced SNPs and indels with these pigmentation phenotypes using a linear mixed model, with a covariance matrix of relatedness as a random effect. As with the heritability analyses, we used a covariance matrix constructed using REAP to account for admixture in the construction of the covariance matrix. We included the proportion of European and Bantu ancestry estimated via ADMIXTURE as fixed effect covariates for baseline pigmentation and tanning status, as chosen in forward stepwise regression. We also included age and sex covariates for tanning, which were significantly associated with the phenotype. We performed the association analysis using EMMAX (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498100">Kang et al., 2010</a>) for the imputed data and GCTA (<a href="#R84" rid="R84" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498094">Yang et al., 2010</a>) for the resequenced data, as both employ mixed model approaches and readily support different data formats.</p></div><div id="S33" class="sec"><p></p><h4 id="S33title" class="inline">Meta-analysis </h4><p id="P54" class="p p-first-last">We performed inverse variance weighted meta-analysis of summary statistics of the phase 1 and phase 2 summary statistics from imputed data using METAL (<a href="#R80" rid="R80" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498090">Willer et al., 2010</a>).</p></div><div id="S34" class="sec"><p></p><h4 id="S34title" class="inline">Association enrichment </h4><p id="P55" class="p p-first">Using the 50 most significant associations in the imputed dataset, we identified the closest genes using bedtools with gencode v19 gene annotations. We assessed enrichments using enrichR (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498113">Chen et al., 2013</a>), which computes enrichment in three ways: 1) the Fisher’s exact test, the standard method implemented in most enrichment analysis tools, assuming a binomial distribution (i.e. presence/absence of a gene in a gene set) and independence of a gene belonging to any set; 2) the deviation from the expected rank by the Fisher’s exact test given many random input gene lists is computed as a z-score, providing a correction to the Fisher’s exact test; and 3) multiplying the log of the p-value from the Fisher’s exact test by the z-score computed in the second test to generate a combined score. We investigated enrichment using the Mouse Genome Informatics (MGI) Mammalian Phenotype ontology rather than the Human Phenotype ontology because pigmentation is highly diverged across populations and has not been studied thoroughly across all populations.</p><p id="P56" class="p p-last">Across all GWAS efforts, we find a significant enrichment of genes related to melanogenesis. Specifically, we find several independent associations near <em>SMARCA2</em> and <em>VLDLR. SMARCA2</em> has a known role in folate biosynthesis, in vitamin D-coupled transcription regulation, and is differentially expressed across CEU and YRI populations in lymphoblastoid cell lines (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498171">Duan et al., 2009</a>). Additionally, previous functional studies have shown that <em>MITF,</em> the transcription factor known as the “master regulator of melanogenesis” due to its ability to activate many melanocyte-specific genes (<a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498081">Praetorius et al., 2013</a>), recruits critical components of the SWI/SNF chromatin remodeling complex (including <em>SMARCA2</em>), to the promoter region of its targets (<a href="#R75" rid="R75" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498184">Vachtenheim et al., 2010</a>). This recruitment is required for normal expression of many <em>MITF</em> target genes, including <em>TYR</em>, <em>TYRP1</em>, <em>DCT</em>, <em>RAB27A</em>, <em>BCL2</em>, among others (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498204">Keenen et al., 2009b</a>). Additionally, <em>VLDLR</em> knockout mice exhibit hypopigmented retinas (<a href="#R83" rid="R83" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498172">Xia et al., 2013b</a>). We also find a suggestive association upstream of <em>TYRP1</em> (<a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5A</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6G</a>). <em>TYRP1</em> mutations in humans have been associated with oculocutaneous albinism and shown to cause nearly Mendelian inheritance of blond hair in Solomon Islanders (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498159">Kenny et al., 2012b</a>; <a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498166">Sarangarajan and Boissy, 2001</a>). Thus, we observe enrichments of molecular pathways involved in pigmentation beyond those previously identified as associated with the phenotype in non-African populations.</p></div><div id="S35" class="sec"><p></p><h4 id="S35title" class="inline">Exome variant calling and annotation </h4><p id="P57" class="p p-first-last">Illumina sequencing reads from 91 KhoeSan DNA samples (of which 82 had pigmentation phenotypes) were captured with: 74 samples on an Agilent SureSelect Human All Exon V2 44Mb array (2 × 101 bp reads, sequenced at BGI on a HiSeq 2000), 8 samples on an Agilent SureSelect Human All Exon 50Mb array (2 × 101 bp reads, sequenced at BGI on a HiSeq 2000), and 8 samples on an Agilent SureSelect Human All Exon V4+UTRs 71Mb array (2 × 126 bp reads, sequenced at the New York Genome Center on a HiSeq 2500). Sequencing data was processed according to a standard pipeline informed by the 1000 Genomes Project. Briefly, we aligned reads to the hg19 reference genome using bwa-mem 0.7.10. We then sorted bam files and marked duplicate reads with Picard v1.92. We next ran RealignerTargetCreator, IndelRealigner, BaseRecalibrator, PrintReads, HaplotypeCaller, GenotypeGVCFs, and VariantRecalibrator, and ApplyRecalibration with GATK (v3.2.2). During the HaplotypeCaller step, we filtered reads down to the capture regions ± 100 bp of padding. We annotated exomic variants using the Variant Effect Prediction tool (VEP) using Ensembl version 75 annotations, which annotates variants using Gencode v19 gene set annotations. We also annotated loss-of-function variants using LOFTEE (<a href="https://github.com/konradjk/loftee" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/konradjk/loftee</a>). We calculated genotype concordance comparing passing variant calls to the corresponding Illumina 550k, Illumina OmniExpress, and Illumina OmniExpressPlus arrays. On target coverage was calculated using GATK’s DepthofCoverage tool.</p></div><div id="S36" class="sec"><p></p><h4 id="S36title" class="inline">Targeted resequencing </h4><p id="P58" class="p p-first-last">For 441 KhoeSan samples (N=269 ‡Khomani and N=172 Nama), we performed targeted resequencing. Older samples had a smaller quantity of DNA, so we first performed whole genome amplification (WGA) for a subset of samples. We chose resequencing targets based on the output of the ‡Khomani GWAS, enriching for strong associations, associations near genes with prior evidence for a role in pigmentation, and regions containing SNPs previously implicated in pigmentation in other populations. We chose 35 regions totaling 7.1 Mb and used the NimbleGen SeqCap EZ Choice Enrichment Kit to enrich for these loci. We barcoded then pooled 96 samples per sequencing run with the Illumina NextSeq.</p></div><div id="S37" class="sec"><p></p><h4 id="S37title" class="inline">Resequencing variant calling </h4><p id="P59" class="p p-first-last">Resequencing data was processed in the same way as exome variant calls up until the HaplotypeCaller step. Because there were not enough variants in the 7 Mb capture region to run VQSR, we applied hard filters to as quality control. We removed samples with &lt;10× mean coverage, samples with a ≥8% contamination rate measured by verify BamID, and highly discordant samples (concordance with genotyping array &lt;95%). We also removed variants with &lt;1% allele frequency, sites with &gt;50% missingness, and spanning (&lt;*:DEL&gt;) variants.</p></div><div id="S38" class="sec sec-last"><p></p><h4 id="S38title" class="inline">Resequencing QC </h4><p id="P60" class="p p-first-last">The resequencing design was successful for all targets, although the regions targeting SNPs previously associated with pigmentation in MC1R and TPCN2 had significantly lower coverage than the other regions. The resequencing efforts yielded large amounts of high-quality data for each pool with on average 84% of reads in target regions, achieving a median depth of coverage of 29× per sample. We compared variant calls and genotypes for samples that were genotyped on an array and achieved an average of &gt; 99% concordance (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S7</a>). We discovered a total of 46,429 SNPs and indels with a MAF &gt; 1% that passed our quality control filters, resulting in a Bonferoni threshold of 1.08e-6.</p></div></div><div id="S39" class="sec"><h3 id="S39title">QUANTIFICATION AND STATISTICAL ANALYSIS</h3><p id="P61" class="p p-first-last">All heritability analyses except for the pedigree-based analysis were performed with GCTA. Pedigree-based heritability analysis was performed with SOLAR. All mixed model association analyses with the imputed GWAS data were performed with EMMAX. All mixed model association analyses with resequenced data were performed with GCTA. All other statistical analyses, unless otherwise noted, were performed using R.</p></div><div id="S40" class="sec"><h3 id="S40title">DATA AND SOFTWARE AVAILABILITY</h3><p id="P62" class="p p-first">Processed data are available here: <a href="https://data.mendeley.com/datasets/98mh8z78m3/draft?a=4a3dc606-f854-4190-9f26-e5e07110349e" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://data.mendeley.com/datasets/98mh8z78m3/draft?a=4a3dc606-f854-4190-9f26-e5e07110349e</a>.</p><p id="P63">According to the newly issued San Code of Research Ethics, as published by the South African San Council, parties should first contact the South African San Council to request data access and submit a project proposal. Following local approval, Dr. Henn will release the appropriate SNP array, exome and/or phenotype data.</p><p id="P64" class="p p-last"><strong>Contact Details</strong>: <a href="mailto:dev@null" data-email="az.gro.isas@nimda" class="oemail">az.gro.isas@nimda</a>, South African San Institute, 4 Sanda Park, Platfontein Farm, Barkly-West / Kimberley Road, Kimberley, Northern Cape, South Africa</p></div><div id="S41" class="sec sec-last"><h3 id="S41title">KEY RESOURCES TABLE</h3><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140158311653472" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">REAGENT or RESOURCE</th><th rowspan="1" colspan="1" valign="bottom" align="left">SOURCE</th><th rowspan="1" colspan="1" valign="bottom" align="left">IDENTIFIER</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Biological Samples</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DNA from ‡Khomani and Nama individuals</td><td rowspan="1" colspan="1" valign="top" align="left">This study and (<a href="#R74" rid="R74" class=" bibr popnode">Uren et al., 2016b</a>)</td><td rowspan="1" colspan="1" valign="top" align="left">N/A</td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left">Critical Commercial Assays</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SeqCap EZ Choice XL Enrichment Kit</td><td rowspan="1" colspan="1" valign="top" align="left">Roche NimbleGen</td><td rowspan="1" colspan="1" valign="top" align="left">06266371001</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Kapa HyperPlus Library Prep Kit</td><td rowspan="1" colspan="1" valign="top" align="left">Roche / Kapa</td><td rowspan="1" colspan="1" valign="top" align="left">KK8514</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Oragene saliva collection kits</td><td rowspan="1" colspan="1" valign="top" align="left">DNAGenotek</td><td rowspan="1" colspan="1" valign="top" align="left">OGR-500</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">NextSeq Mid Output 300 cycle kit</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina</td><td rowspan="1" colspan="1" valign="top" align="left">FC-404-2004</td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left">Deposited Data</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">‡Khomani and Nama ancestry and summary statistics data</td><td rowspan="1" colspan="1" valign="top" align="left">This study</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://dx.doi.org/10.17632/98mh8z78m3.1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://dx.doi.org/10.17632/98mh8z78m3.1</a></td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left">Oligonucleotides</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">xGen Adapter Blocking Oligos</td><td rowspan="1" colspan="1" valign="top" align="left">xGen Adapter Blocking Oligos</td><td rowspan="1" colspan="1" valign="top" align="left">xGen Adapter Blocking Oligos</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8×12 TS HT Dual Index Duplex Mixed Adapter Plate</td><td rowspan="1" colspan="1" valign="top" align="left">8×12 TS HT Dual Index Duplex Mixed Adapter Plate</td><td rowspan="1" colspan="1" valign="top" align="left">8×12 TS HT Dual Index Duplex Mixed Adapter Plate</td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left">Software and Algorithms</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">METAL</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R80" rid="R80" class=" bibr popnode">Willer et al., 2010</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://csg.sph.umich.edu/abecasis/metal/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/abecasis/metal/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">EMMAX</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R37" rid="R37" class=" bibr popnode">Kang et al., 2010</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://genetics.cs.ucla.edu/emmax/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genetics.cs.ucla.edu/emmax/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ADMIXTURE</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R63" rid="R63" class=" bibr popnode">Shringarpure et al., 2016</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="https://www.genetics.ucla.edu/software/admixture/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.genetics.ucla.edu/software/admixture/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">PLINK</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R12" rid="R12" class=" bibr popnode">Chang et al., 2015</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="https://www.cog-genomics.org/plink2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.cog-genomics.org/plink2</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">VEP</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R49" rid="R49" class=" bibr popnode">McLaren et al., 2016</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="https://www.ensembl.org/vep" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ensembl.org/vep</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LOFTEE</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="https://github.com/konradjk/loftee" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/konradjk/loftee</a></td><td rowspan="1" colspan="1" valign="top" align="left"><a href="https://github.com/konradjk/loftee" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/konradjk/loftee</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SOLAR</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R1" rid="R1" class=" bibr popnode">Almasy and Blangero, 1998</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://www.biostat.wustl.edu/genetics/geneticssoft/manuals/solar210/00.contents.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.biostat.wustl.edu/genetics/geneticssoft/manuals/solar210/00.contents.html</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">GCTA</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R85" rid="R85" class=" bibr popnode">Yang et al., 2011</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://cnsgenomics.com/software/gcta/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cnsgenomics.com/software/gcta/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">BWA</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R43" rid="R43" class=" bibr popnode">Li and Durbin, 2009</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://bio-bwa.sourceforge.net/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://bio-bwa.sourceforge.net/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Picard</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://broadinstitute.github.io/picard/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://broadinstitute.github.io/picard/</a></td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://broadinstitute.github.io/picard/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://broadinstitute.github.io/picard/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">KING</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R45" rid="R45" class=" bibr popnode">Manichaikul et al., 2010</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://people.virginia.edu/~wc9c/KING/manual.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://people.virginia.edu/~wc9c/KING/manual.html</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">BEAGLE</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R8" rid="R8" class=" bibr popnode">Browning and Browning, 2007</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="https://faculty.washington.edu/browning/beagle/beagle.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://faculty.washington.edu/browning/beagle/beagle.html</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">R</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="https://www.rproject.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.rproject.org/</a></td><td rowspan="1" colspan="1" valign="top" align="left"><a href="https://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.r-project.org/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">GENESIS</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R17" rid="R17" class=" bibr popnode">Conomos et al., 2016</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://bioconductor.org/packages/release/bioc/html/GENESIS.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://bioconductor.org/packages/release/bioc/html/GENESIS.html</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">REAP</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R71" rid="R71" class=" bibr popnode">Thornton et al., 2012b</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://faculty.washington.edu/tathornt/software/REAP/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://faculty.washington.edu/tathornt/software/REAP/index.html</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">enrichR</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R15" rid="R15" class=" bibr popnode">Chen et al., 2013</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://amp.pharm.mssm.edu/Enrichr/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://amp.pharm.mssm.edu/Enrichr/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HaploReg</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R78" rid="R78" class=" bibr popnode">Ward and Kellis, 2011</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://archive.broadinstitute.org/mammals/haploreg/haploreg.php" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://archive.broadinstitute.org/mammals/haploreg/haploreg.php</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">RFMix</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R46" rid="R46" class=" bibr popnode">Maples et al., 2013</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="https://sites.google.com/site/rfmixlocalancestryinference/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank" role="button" aria-expanded="false" aria-haspopup="true">https://sites.google.com/site/rfmixlocalancestryinference/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">GERMLINE</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R27" rid="R27" class=" bibr popnode">Gusev et al., 2009</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://www.cs.columbia.edu/~gusev/germline/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cs.columbia.edu/~gusev/germline/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Shapeit2</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R53" rid="R53" class=" bibr popnode">O’Connell et al., 2014</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://www.shapeit.fr/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.shapeit.fr/</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Impute2</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R34" rid="R34" class=" bibr popnode">Howie et al., 2009</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="http://mathgen.stats.ox.ac.uk/impute/impute_v2.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mathgen.stats.ox.ac.uk/impute/impute_v2.html</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">GATK</td><td rowspan="1" colspan="1" valign="top" align="left">(<a href="#R48" rid="R48" class=" bibr popnode">McKenna et al., 2010</a>)</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="https://software.broadinstitute.org/gatk/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://software.broadinstitute.org/gatk/</a></td></tr></tbody></table></div><div id="largeobj_idm140158311653472" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5884124/table/T3/?report=objectonly">Open in a separate window</a></div></div></div></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="idm140158298860256title">Highlights</h3></div><ul class="unordered" style="list-style-type:disc" id="L1"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="P81">Skin pigmentation in Africans is far more polygenic than light skin in Eurasians</p></li><li><p id="P82">Early diverged KhoeSan populations have light skin compared to equatorial Africans</p></li><li><p id="P83">Highly heritable KhoeSan skin color variation is poorly explained by known genes</p></li><li><p id="P84">The study of African skin color identifies novel and canonical pigmentation genes</p></li></ul></div></div></div><div id="S42" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S42title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><p id="P65"><strong>Figure S1</strong> – <strong>Pedigrees inferred from ethnographic information. (Related to </strong><a href="/pmc/articles/PMC5884124/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and Methods) Ethnographically inferred pedigrees for KhoeSan individuals are shown in: A) the ‡Khomani, and B) the Nama. Different shades represent whether the samples have been genotyped. Non-phenotyped individuals are crossed in pedigrees.</p><p id="P66"><strong>Figure S2</strong> – <strong>Ancestry estimates in ‡Khomani and Nama samples. (Related to </strong><a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>) A) Admixture runs across K=3–7 for the ‡Khomani and Nama populations, using Namibian San, Hadza, Sandawe, Maasai, Kenyan Bantu, South African (SA) Bantu, Yoruba, Mozabite, Central Europeans (CEU), and French populations as a reference panel, as in <a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>. B) Minor ancestry component associations with M index, displayed with a square root x-axis to elongate the minor contributions. C) Pairwise ancestry component correlations at k=7 from A). Upper triangular matrix shows Pearson’s correlation coefficient between pairwise ancestry estimates. Lower triangular matrix shows scatter plots with ‡Khomani shown in blue and Nama shown in red. D-F) Principal Components Analysis (PCA) biplots for the same SNPs and individuals used in the ADMIXTURE analysis adjusted for relatedness using King and the PC-AiR approach. D) PC1 vs PC2, E) PC3 vs PC4, and F) PC5 vs PC6.</p><p id="P67"><strong>Figure S3</strong> – <strong>Partitioned heritability by population. (Related to </strong><a href="/pmc/articles/PMC5884124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>) Proportion of heritable variation in baseline pigmentation explained by true pigmentation gene sets (dots) versus matched null distribution after accounting for number of SNPs in gene sets in two different candidate gene sets. Results are stratified by the combined (comb) data, ‡Khomani (khm), and Nama populations. * indicates FDR &lt; 0.1 (indicated by the top decile of the null distributions). Gene set labels are the same as in <a href="/pmc/articles/PMC5884124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>.</p><p id="P68"><strong>Figure S4</strong> – <strong>Imputation and targeted resequencing quality control. (Related to STAR methods)</strong> A) Imputation quality was assessed via homozygous reference (HR), heterozygous (Het), and homozygous non-reference (HN) concordance for imputed dosages with high coverage exome sequencing data. Each concordance metric was computed as the ratio of dosages for a particular genotyping class (e.g. HR) in both genotype and exome datasets to the dosages of that total class in the genotype dataset with any call in the exomes. The colors indicate the different genotyping arrays, with the 550k array in red, OmniExpress array in blue, and OmniExpressPlus array in green. B–E) Targeted resequencing quality control. B) Depth of coverage versus array concordance. Dashed line at 10× indicates minimum depth accepted for a sample’s inclusion. C) Depth of coverage by targeted resequencing region. D) Resequencing coverage by sequencing library. Solid line indicates median value, and dashed lines indicate 25% and 75% quartiles. E) Fraction of reads in targeted resequencing regions by sequencing library. Sequencing libraries (seq1, seq2, seq3, seq4, seq5, new_seq1, new_seq2, new_seq3, new_seq4, and new_seq5) consisted of barcoded and pooled samples, with most samples sequenced in multiple libraries. Libraries with “new” in the name were sequenced after the last sample collection.</p><p id="P69"><strong>Figure S5 – Pigmentation prediction from other models are inaccurate in the KhoeSan.</strong> (<strong>Related to </strong><a href="/pmc/articles/PMC5884124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) A) Prediction model of skin pigmentation based on seven SNPs. The model (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498102">Hart et al., 2013b</a>; <a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498112">Spichenok et al., 2011</a>) utilizes a bifurcating decision tree based on individuals’ genotypes at seven predictive loci. The condition of each step-wise decision is described, with arrows indicating the decision possibilities. Categorical predictions are shown at the bottom of the chart. B) Pigmentation prediction result is shown as individuals’ actual spectrometer measured of M index against predicted categorical pigmentation. Number of individuals assigned to each category is shown in the x axis labels. The prediction is based on a previously developed 7-SNP model (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498161">Hart et al., 2013b</a>; <a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498123">Spichenok et al., 2011</a>), with predicted output categorized as “Light”, “Medium”, “Dark”, or “No Call” if the model fails to unambiguously assign a prediction based on genotypes.</p><p id="P70"><strong>Figure S6</strong> – <strong>Initial GWAS. (Related to </strong><a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>) Imputed variants were association with pigmentation and tanning status for 217 KhoeSan individuals (107 ‡Khomani and 110 Nama individuals). A) Study design overview. B) Manhattan plot for pigmentation, with LD assessed and clumped using best guess genotypes from MEGA data (i.e. largest sample size). LD clumps are shown in blue for top 25 loci and labeled with closest gene for top 5 loci. C) QQplot for pigmentation (λ<sub>GC</sub>=1.018), D) LocusZoom plot for <em>SLC24A5</em>, among top 5 independent loci with linkage signals, E) Manhattan plot for tanning status, F) QQplot for tanning status (λ<sub>GC</sub>=1.052), G) LocusZoom plot near <em>TYRP1</em>, among top 5 independent loci with linkage signals.</p><div class="sup-box half_rhythm" id="d36e2310"><a href="/pmc/articles/PMC5884124/bin/NIHMS921744-supplement-1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(137K, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><p id="P71"><strong>Table S1</strong> – <strong>Summary statistics of pigmentation phenotypes for global population.</strong> (<strong>Related to </strong><a href="/pmc/articles/PMC5884124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) All statistics relate to M index for baseline pigmentation. Rows with no dataset listed refer to subsets of the dataset in prior rows, e.g. &gt;90% KhoeSan Nama refers to the Nama samples from this study. Cape Verdean samples from (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498144">Beleza et al., 2013</a>) were broken into quantiles, labeled Q1 to Q4. μ = mean, σ = standard deviation, and c<sub>V</sub> = coefficient of variation (σ/μ), a standardized metric of dispersion.</p><p id="P72"><strong>Table S2 – Reference and KhoeSan samples included in the ADMIXTURE analysis.</strong> (<strong>Related to </strong><a href="/pmc/articles/PMC5884124/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>)</p><p id="P73"><strong>Table S3</strong> – <strong>Heritability estimates by population using multiple approaches. (Related to </strong><a href="/pmc/articles/PMC5884124/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) Heritability was assessed separately and jointly in the ‡Khomani and Nama populations. Because of population structure, haplotypes were called separately, (i.e. even when populations were analyzed jointly). N = number of individuals. SNPs = number of SNPs included.</p><p id="P74"><strong>Table S4 – Number of individuals from each population that were phenotyped, genotyped, exome sequenced, or with targeted resequencing data (Related to STAR Methods).</strong> Pheno = phenotyped, Geno=genotyped, Exome = exome sequenced, Reseq = targeted resequencing. We also indicate the overlap among different genotyping arrays, which are as follows: 550k = Illumina 550k array, OmniEx = Illumina OmniExpress array, OmniExPlus = Illumina OmniExpressPlus array, Omni2.5 = Illumina Omni2.5 array, Mega = Illumina Multiethnic genotyping array. Phase 2 of the GWAS analysis consisted of samples genotyped on the MEGA array.</p><p id="P75"><strong>Table S5</strong> – <strong>Top 50 associations from initial GWAS. (Related to </strong><a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>) All associations shown are independent (R<sup>2</sup> ≤ 0.5, window size = 250 kb), and indicate the association between genotype dosages and skin pigmentation M index across the Nama and ‡Khomani datasets. The # tagging column indicates the number of SNPs with an rsID that are tagged by the independent signal.</p><p id="P76"><strong>Table S6</strong> – <strong>Summary statistics and regulator information for known and novel pigmentation SNPs. (Related to </strong><a href="/pmc/articles/PMC5884124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>) A) P-value, effect size and standard error, and frequency are shown for known pigmentation loci (including skin, eye, and hair color phenotypes) in the ‡Khomani, Nama, and jointly analyzed samples. B) Regulatory information including conservation, functional class, motif changes, etc, indentified via HaploReg are shown for resequencing variants in 6 of the targeted regions.</p><p id="P77">*Provided as Excel spreadsheet due to size constraints</p><p id="P78"><strong>Table S7</strong> – <strong>Regions selected for targeted resequencing. (Related to </strong><a href="/pmc/articles/PMC5884124/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>) Resequencing was successfully performed for a total of 441 KhoeSan individuals.</p><p id="P79"><strong>Table S8 – Power to detect loci with suggestive and genome-wide significance for effect sizes previously associated with skin pigmentation. (Related to STAR Methods)</strong> A previous study by <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_687498080">Beleza et al, 2013</a> showed that 4 loci explained 35% of the variation in skin color. We show power for the approximate sample sizes in phase 1, phase 2, and the resequencing analysis.</p><p id="P80">*Provided as Excel file due to size constraints</p><div class="sup-box half_rhythm" id="d36e2399"><a href="/pmc/articles/PMC5884124/bin/NIHMS921744-supplement-2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(99K, xlsx)</sup></div></div></div><div id="S43" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S43title">Acknowledgments</h2><div class="sec"><p id="__p2">We thank the Beijing Genome Institute and Agilent Technologies for sequencing support. We thank Sandra Beleza, Heather Norton, Esteban Parra, Chandana Basu Mallick, Andres Moreno-Estrada and Karla Sandoval, Richard Cooper, Sophie Candille, George Chaplin, and Nina Jablonski and their colleagues for providing quantitative phenotypes or summaries from their pigmentation work. We thank Catherine Guenther for discussing molecular strategies for functional followup. We also thank the Working Group of Indigenous Minorities in Southern Africa (WIMSA), community leaders, and the South African San Institute (SASI) for their advice and oversight. Finally, we thank Richard Jacobs, Wilhelmina Mondzinger, Hans Padmaker, Willem de Klerk, Hendrik Kaiman, and the communities in which we have sampled; without their support, this study would not have been possible. Funding for this project was provided by a Stanford University CDEHA seed grant (NIH, NIA P30 AG017253-12) to BMH and by a trainee research grant to ARM by the Stanford Center for Computational, Evolutionary, and Human Genomics. Funding was also provided by the Morrison Institute for Population Studies, Stanford University. Funding for genotyping additional KhoeSan individuals was provided by the Stanford Center for Computational, Evolutionary, and Human Genomics. ARM was funded by NIH Genetics and Developmental Biology Training Program T32 GM07790. EA was supported by an NIH IRACDA fellowship to Stony Brook University. CRG was supported by the Stanford Genome Training Program 5T32HG000044. CJW, MM and EGH are partly supported by the South African Medical Research Council. The content is solely the responsibility of the authors and does not necessarily represent the official views of the South African Medical Research Council.</p></div></div><div id="idm140158316440752" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140158316440752title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><div><p id="__p3" class="p p-first"><strong>Author Contributions</strong></p>BMH, CRG, MWF, and CDB conceived of and designed the study. ARM and AS designed experiments. ARM, MS and AS performed experiments. ARM, JMG, ML, EGA and XL analyzed data. BMH, CRG, CJW, JWM, ARM, JMG, and MM collected samples and measured skin photometrics. EGH, CRG, DMK, MWF, CDB, and BMH supervised the study. ARM and BMH wrote the manuscript with input from ML, JMG, MM, DMK, EGH, MWF, and CRG. All authors read and approved of the final version of the manuscript.</div><p class="fn sec" id="FN4"><sup>§</sup>We use the term “KhoeSan” to refer to a diverse array of indigenous populations in southern Africa that carry KhoeSan ancestry and speak Khoe, !Ui-Tuu, or Kx’a languages. “KhoeSan” is not accepted by all such communities; where possible we refer to populations by their specific ethnic name. This grouping lumps together populations of different languages, cultures, and variable genetic diversity.</p><p class="fn sec" id="FN5"></p><p id="__p6" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="idm140158316209776" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140158316209776title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Almasy L, Blangero J. Multipoint Quantitative-Trait Linkage Analysis in General Pedigrees. <span><span class="ref-journal">The American Journal of Human Genetics. </span>1998;<span class="ref-vol">62</span>:1198–1211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377101/">PMC free article</a>]</span> [<a href="/pubmed/9545414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+American+Journal+of+Human+Genetics&amp;title=Multipoint+Quantitative-Trait+Linkage+Analysis+in+General+Pedigrees&amp;author=L+Almasy&amp;author=J+Blangero&amp;volume=62&amp;publication_year=1998&amp;pages=1198-1211&amp;pmid=9545414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Altshuler DM, Gibbs RA, Dermitzakis E, Peltonen L, Dermitzakis E, Bonnen PE, de Bakker PIW, Deloukas P, Gabriel SB, Gwilliam R, et al.  Integrating common and rare genetic variation in diverse human populations. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:52–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173859/">PMC free article</a>]</span> [<a href="/pubmed/20811451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrating+common+and+rare+genetic+variation+in+diverse+human+populations&amp;author=DM+Altshuler&amp;author=RA+Gibbs&amp;author=E+Dermitzakis&amp;author=L+Peltonen&amp;author=E+Dermitzakis&amp;volume=467&amp;publication_year=2010&amp;pages=52-58&amp;pmid=20811451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Barton NH. Clines in polygenic traits. <span><span class="ref-journal">Genet Res. </span>1999;<span class="ref-vol">74</span>:223–236.</span> [<a href="/pubmed/10689800" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Res&amp;title=Clines+in+polygenic+traits&amp;author=NH+Barton&amp;volume=74&amp;publication_year=1999&amp;pages=223-236&amp;pmid=10689800&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Basu Mallick C, Iliescu FM, Möls M, Hill S, Tamang R, Chaubey G, Goto R, Ho SYW, Gallego Romero I, Crivellaro F, et al.  The Light Skin Allele of SLC24A5 in South Asians and Europeans Shares Identity by Descent. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>:e1003912.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3820762/">PMC free article</a>]</span> [<a href="/pubmed/24244186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=The+Light+Skin+Allele+of+SLC24A5+in+South+Asians+and+Europeans+Shares+Identity+by+Descent&amp;author=C+Basu+Mallick&amp;author=FM+Iliescu&amp;author=M+Möls&amp;author=S+Hill&amp;author=R+Tamang&amp;volume=9&amp;publication_year=2013&amp;pages=e1003912&amp;pmid=24244186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Beleza S, Johnson NA, Candille SI, Absher DM, Coram MA, Lopes J, Campos J, Araújo II, Anderson TM, Vilhjálmsson BJ, et al.  Genetic Architecture of Skin and Eye Color in an African-European Admixed Population. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>:e1003372–15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3605137/">PMC free article</a>]</span> [<a href="/pubmed/23555287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetic+Architecture+of+Skin+and+Eye+Color+in+an+African-European+Admixed+Population&amp;author=S+Beleza&amp;author=NA+Johnson&amp;author=SI+Candille&amp;author=DM+Absher&amp;author=MA+Coram&amp;volume=9&amp;publication_year=2013&amp;pages=e1003372-15&amp;pmid=23555287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Beleza S, Santos AM, McEvoy B, Alves I, Martinho C, Cameron E, Shriver MD, Parra EJ, Rocha J. The Timing of Pigmentation Lightening in Europeans. <span><span class="ref-journal">Mol Biol Evol. </span>2012;<span class="ref-vol">30</span>:24–35.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3525146/">PMC free article</a>]</span> [<a href="/pubmed/22923467" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=The+Timing+of+Pigmentation+Lightening+in+Europeans&amp;author=S+Beleza&amp;author=AM+Santos&amp;author=B+McEvoy&amp;author=I+Alves&amp;author=C+Martinho&amp;volume=30&amp;publication_year=2012&amp;pages=24-35&amp;pmid=22923467&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Berg JJ, Coop G. A Population Genetic Signal of Polygenic Adaptation. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004412–e1004425.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4125079/">PMC free article</a>]</span> [<a href="/pubmed/25102153" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+Population+Genetic+Signal+of+Polygenic+Adaptation&amp;author=JJ+Berg&amp;author=G+Coop&amp;volume=10&amp;publication_year=2014&amp;pages=e1004412-e1004425&amp;pmid=25102153&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Browning SR, Browning BL. Rapid and Accurate Haplotype Phasing and Missing-Data Inference for Whole-Genome Association Studies By Use of Localized Haplotype Clustering. <span><span class="ref-journal">The American Journal of Human Genetics. </span>2007;<span class="ref-vol">81</span>:1084–1097.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265661/">PMC free article</a>]</span> [<a href="/pubmed/17924348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+American+Journal+of+Human+Genetics&amp;title=Rapid+and+Accurate+Haplotype+Phasing+and+Missing-Data+Inference+for+Whole-Genome+Association+Studies+By+Use+of+Localized+Haplotype+Clustering&amp;author=SR+Browning&amp;author=BL+Browning&amp;volume=81&amp;publication_year=2007&amp;pages=1084-1097&amp;pmid=17924348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Byard PJ. Quantitative genetics of human skin color. <span><span class="ref-journal">Am J Phys Anthropol. </span>1981;<span class="ref-vol">24</span>:123–137.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Phys+Anthropol&amp;title=Quantitative+genetics+of+human+skin+color&amp;author=PJ+Byard&amp;volume=24&amp;publication_year=1981&amp;pages=123-137&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Candille SI, Absher DM, Beleza S, Bauchet M, McEvoy B, Garrison NA, Li JZ, Myers RM, Barsh GS, Tang H, et al.  Genome-Wide Association Studies of Quantitatively Measured Skin, Hair, and Eye Pigmentation in Four European Populations. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e48294–12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3485197/">PMC free article</a>]</span> [<a href="/pubmed/23118974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Genome-Wide+Association+Studies+of+Quantitatively+Measured+Skin,+Hair,+and+Eye+Pigmentation+in+Four+European+Populations&amp;author=SI+Candille&amp;author=DM+Absher&amp;author=S+Beleza&amp;author=M+Bauchet&amp;author=B+McEvoy&amp;volume=7&amp;publication_year=2012&amp;pages=e48294-12&amp;pmid=23118974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Carlson CS, Matise TC, North KE, Haiman CA, Fesinmeyer MD, Buyske S, Schumacher FR, Peters U, Franceschini N, Ritchie MD, et al.  Generalization and Dilution of Association Results from European GWAS in Populations of Non-European Ancestry: The PAGE Study. <span><span class="ref-journal">PLoS Biol. </span>2013;<span class="ref-vol">11</span>:e1001661–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3775722/">PMC free article</a>]</span> [<a href="/pubmed/24068893" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Generalization+and+Dilution+of+Association+Results+from+European+GWAS+in+Populations+of+Non-European+Ancestry:+The+PAGE+Study&amp;author=CS+Carlson&amp;author=TC+Matise&amp;author=KE+North&amp;author=CA+Haiman&amp;author=MD+Fesinmeyer&amp;volume=11&amp;publication_year=2013&amp;pages=e1001661-11&amp;pmid=24068893&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. <span><span class="ref-journal">Giga Sci. </span>2015;<span class="ref-vol">4</span>:7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4342193/">PMC free article</a>]</span> [<a href="/pubmed/25722852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Giga+Sci&amp;title=Second-generation+PLINK:+rising+to+the+challenge+of+larger+and+richer+datasets&amp;author=CC+Chang&amp;author=CC+Chow&amp;author=LC+Tellier&amp;author=S+Vattikuti&amp;author=SM+Purcell&amp;volume=4&amp;publication_year=2015&amp;pages=7&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Chaplin G. Geographic distribution of environmental factors influencing human skin coloration. <span><span class="ref-journal">Am J Phys Anthropol. </span>2004;<span class="ref-vol">125</span>:292–302.</span> [<a href="/pubmed/15386260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Phys+Anthropol&amp;title=Geographic+distribution+of+environmental+factors+influencing+human+skin+coloration&amp;author=G+Chaplin&amp;volume=125&amp;publication_year=2004&amp;pages=292-302&amp;pmid=15386260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Chaplin G, Jablonski NG. Vitamin D and the evolution of human depigmentation. <span><span class="ref-journal">Am J Phys Anthropol. </span>2009;<span class="ref-vol">139</span>:451–461.</span> [<a href="/pubmed/19425101" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Phys+Anthropol&amp;title=Vitamin+D+and+the+evolution+of+human+depigmentation&amp;author=G+Chaplin&amp;author=NG+Jablonski&amp;volume=139&amp;publication_year=2009&amp;pages=451-461&amp;pmid=19425101&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Chen EY, Tan CM, Kou Y, Duan Q, Wang Z, Meirelles G, Clark NR, Ma’ayan A. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. <span><span class="ref-journal">BMC Bioinformatics. </span>2013;<span class="ref-vol">14</span>:128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3637064/">PMC free article</a>]</span> [<a href="/pubmed/23586463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=Enrichr:+interactive+and+collaborative+HTML5+gene+list+enrichment+analysis+tool&amp;author=EY+Chen&amp;author=CM+Tan&amp;author=Y+Kou&amp;author=Q+Duan&amp;author=Z+Wang&amp;volume=14&amp;publication_year=2013&amp;pages=128&amp;pmid=23586463&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Clark P, Stark AE, Walsh RJ, Jardine R, Martin NG. A twin study of skin reflectance. <span><span class="ref-journal">Annals of Human Biology. </span>1981;<span class="ref-vol">8</span>:529–541.</span> [<a href="/pubmed/7199888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+Human+Biology&amp;title=A+twin+study+of+skin+reflectance&amp;author=P+Clark&amp;author=AE+Stark&amp;author=RJ+Walsh&amp;author=R+Jardine&amp;author=NG+Martin&amp;volume=8&amp;publication_year=1981&amp;pages=529-541&amp;pmid=7199888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Conomos MP, Reiner AP, Weir BS, Thornton TA. Model-free Estimation of Recent Genetic Relatedness. <span><span class="ref-journal">The American Journal of Human Genetics. </span>2016;<span class="ref-vol">98</span>:127–148.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4716688/">PMC free article</a>]</span> [<a href="/pubmed/26748516" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+American+Journal+of+Human+Genetics&amp;title=Model-free+Estimation+of+Recent+Genetic+Relatedness&amp;author=MP+Conomos&amp;author=AP+Reiner&amp;author=BS+Weir&amp;author=TA+Thornton&amp;volume=98&amp;publication_year=2016&amp;pages=127-148&amp;pmid=26748516&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Coussens AK, Naude CE, Goliath R, Chaplin G, Wilkinson RJ, Jablonski NG. High-dose vitamin D 3reduces deficiency caused by low UVB exposure and limits HIV-1 replication in urban Southern Africans. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2015;<span class="ref-vol">112</span>:8052–8057.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4491791/">PMC free article</a>]</span> [<a href="/pubmed/26080414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=High-dose+vitamin+D+3reduces+deficiency+caused+by+low+UVB+exposure+and+limits+HIV-1+replication+in+urban+Southern+Africans&amp;author=AK+Coussens&amp;author=CE+Naude&amp;author=R+Goliath&amp;author=G+Chaplin&amp;author=RJ+Wilkinson&amp;volume=112&amp;publication_year=2015&amp;pages=8052-8057&amp;pmid=26080414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Diffey BL, OLIVER RJ, Farr PM. A portable instrument for quantifying erythema induced by ultraviolet radiation. <span><span class="ref-journal">British Journal of Dermatology. </span>1984;<span class="ref-vol">111</span>:663–672.</span> [<a href="/pubmed/6508999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=British+Journal+of+Dermatology&amp;title=A+portable+instrument+for+quantifying+erythema+induced+by+ultraviolet+radiation&amp;author=BL+Diffey&amp;author=RJ+OLIVER&amp;author=PM+Farr&amp;volume=111&amp;publication_year=1984&amp;pages=663-672&amp;pmid=6508999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Duan S, Huang RS, Zhang W, Mi S, Bleibel WK, Kistner EO, Cox NJ, Dolan ME. Expression and alternative splicing of folate pathway genes in HapMap lymphoblastoid cell lines. <span><span class="ref-journal">Pharmacogenomics. </span>2009;<span class="ref-vol">10</span>:549–563.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2709842/">PMC free article</a>]</span> [<a href="/pubmed/19374514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&amp;title=Expression+and+alternative+splicing+of+folate+pathway+genes+in+HapMap+lymphoblastoid+cell+lines&amp;author=S+Duan&amp;author=RS+Huang&amp;author=W+Zhang&amp;author=S+Mi&amp;author=WK+Bleibel&amp;volume=10&amp;publication_year=2009&amp;pages=549-563&amp;pmid=19374514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Duffy DL, Montgomery GW, Chen W, Zhao ZZ, Le L, James MR, Hayward NK, Martin NG, Sturm RA. A three-single-nucleotide polymorphism haplotype in intron 1 of OCA2 explains most human eye-color variation. <span><span class="ref-journal">The American Journal of Human Genetics. </span>2007;<span class="ref-vol">80</span>:241–252.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1785344/">PMC free article</a>]</span> [<a href="/pubmed/17236130" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+American+Journal+of+Human+Genetics&amp;title=A+three-single-nucleotide+polymorphism+haplotype+in+intron+1+of+OCA2+explains+most+human+eye-color+variation&amp;author=DL+Duffy&amp;author=GW+Montgomery&amp;author=W+Chen&amp;author=ZZ+Zhao&amp;author=L+Le&amp;volume=80&amp;publication_year=2007&amp;pages=241-252&amp;pmid=17236130&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Durazo-Arvizu RA, Camacho P, Bovet P, Forrester T, Lambert EV, Plange-Rhule J, Hoofnagle AN, Aloia J, Tayo B, Dugas LR, et al.  25-Hydroxyvitamin D in African-origin populations at varying latitudes challenges the construct of a physiologic norm. <span><span class="ref-journal">American Journal of Clinical Nutrition. </span>2014;<span class="ref-vol">100</span>:908–914.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4135499/">PMC free article</a>]</span> [<a href="/pubmed/25008852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Clinical+Nutrition&amp;title=25-Hydroxyvitamin+D+in+African-origin+populations+at+varying+latitudes+challenges+the+construct+of+a+physiologic+norm&amp;author=RA+Durazo-Arvizu&amp;author=P+Camacho&amp;author=P+Bovet&amp;author=T+Forrester&amp;author=EV+Lambert&amp;volume=100&amp;publication_year=2014&amp;pages=908-914&amp;pmid=25008852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Edwards M, Bigham A, Tan J, Li S, Gozdzik A, Ross K, Jin L, Parra EJ. Association of the OCA2 Polymorphism His615Arg with Melanin Content in East Asian Populations: Further Evidence of Convergent Evolution of Skin Pigmentation. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000867.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2832666/">PMC free article</a>]</span> [<a href="/pubmed/20221248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Association+of+the+OCA2+Polymorphism+His615Arg+with+Melanin+Content+in+East+Asian+Populations:+Further+Evidence+of+Convergent+Evolution+of+Skin+Pigmentation&amp;author=M+Edwards&amp;author=A+Bigham&amp;author=J+Tan&amp;author=S+Li&amp;author=A+Gozdzik&amp;volume=6&amp;publication_year=2010&amp;pages=e1000867&amp;pmid=20221248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Frisancho AR, Wainwright R, Way A. Heritability and components of phenotypic expression in skin reflectance of Mestizos from the Peruvian Lowlands. <span><span class="ref-journal">Am J Phys Anthropol. </span>1981;<span class="ref-vol">55</span>:203–208.</span> [<a href="/pubmed/7258341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Phys+Anthropol&amp;title=Heritability+and+components+of+phenotypic+expression+in+skin+reflectance+of+Mestizos+from+the+Peruvian+Lowlands&amp;author=AR+Frisancho&amp;author=R+Wainwright&amp;author=A+Way&amp;volume=55&amp;publication_year=1981&amp;pages=203-208&amp;pmid=7258341&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Genovese G, Friedman DJ, Ross MD, Lecordier L, Uzureau P, Freedman BI, Bowden DW, Langefeld CD, Oleksyk TK, Uscinski Knob AL, et al.  Association of trypanolytic ApoL1 variants with kidney disease in African Americans. <span><span class="ref-journal">Science. </span>2010;<span class="ref-vol">329</span>:841–845.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2980843/">PMC free article</a>]</span> [<a href="/pubmed/20647424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Association+of+trypanolytic+ApoL1+variants+with+kidney+disease+in+African+Americans&amp;author=G+Genovese&amp;author=DJ+Friedman&amp;author=MD+Ross&amp;author=L+Lecordier&amp;author=P+Uzureau&amp;volume=329&amp;publication_year=2010&amp;pages=841-845&amp;pmid=20647424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Gravel S, Zakharia F, Moreno-Estrada A, Byrnes JK, Muzzio M, Rodriguez-Flores JL, Kenny EE, Gignoux CR, Maples BK, Guiblet W, et al.  Reconstructing Native American Migrations from Whole-Genome and Whole-Exome Data. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>:e1004023–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3873240/">PMC free article</a>]</span> [<a href="/pubmed/24385924" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Reconstructing+Native+American+Migrations+from+Whole-Genome+and+Whole-Exome+Data&amp;author=S+Gravel&amp;author=F+Zakharia&amp;author=A+Moreno-Estrada&amp;author=JK+Byrnes&amp;author=M+Muzzio&amp;volume=9&amp;publication_year=2013&amp;pages=e1004023-14&amp;pmid=24385924&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">Gusev A, Lowe JK, Stoffel M, Daly MJ, Altshuler D, Breslow JL, Friedman JM, Pe’er I. Whole population, genome-wide mapping of hidden relatedness. <span><span class="ref-journal">Genome Research. </span>2009;<span class="ref-vol">19</span>:318–326.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2652213/">PMC free article</a>]</span> [<a href="/pubmed/18971310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Research&amp;title=Whole+population,+genome-wide+mapping+of+hidden+relatedness&amp;author=A+Gusev&amp;author=JK+Lowe&amp;author=M+Stoffel&amp;author=MJ+Daly&amp;author=D+Altshuler&amp;volume=19&amp;publication_year=2009&amp;pages=318-326&amp;pmid=18971310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Gusev A, Trynka G, Finucane H, Vilhjálmsson BJ, Xu H, Zang C, Stahl E, Hultman CM, Purcell SM, Daly M, et al.  Partitioning heritability of regulatory and cell-type-specific variants across 11 common diseases. <span><span class="ref-journal">Am J Hum Genet. </span>2014;<span class="ref-vol">95</span>:535–552.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4225595/">PMC free article</a>]</span> [<a href="/pubmed/25439723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Partitioning+heritability+of+regulatory+and+cell-type-specific+variants+across+11+common+diseases&amp;author=A+Gusev&amp;author=G+Trynka&amp;author=H+Finucane&amp;author=BJ+Vilhjálmsson&amp;author=H+Xu&amp;volume=95&amp;publication_year=2014&amp;pages=535-552&amp;pmid=25439723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Harrison GA, Owen JJT. Studies on the inheritance of human skin colour. <span><span class="ref-journal">Annals of Human Genetics. </span>1964;<span class="ref-vol">28</span>:27–37.</span> [<a href="/pubmed/14204850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+Human+Genetics&amp;title=Studies+on+the+inheritance+of+human+skin+colour&amp;author=GA+Harrison&amp;author=JJT+Owen&amp;volume=28&amp;publication_year=1964&amp;pages=27-37&amp;pmid=14204850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">Hart KL, Kimura SL, Mushailov V, Budimlija ZM, Prinz M, Wurmbach E. Improved eye- and skin-color prediction based on 8 SNPs. <span><span class="ref-journal">Croat Med J. </span>2013a;<span class="ref-vol">54</span>:248–256.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3694299/">PMC free article</a>]</span> [<a href="/pubmed/23771755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Croat+Med+J&amp;title=Improved+eye-+and+skin-color+prediction+based+on+8+SNPs&amp;author=KL+Hart&amp;author=SL+Kimura&amp;author=V+Mushailov&amp;author=ZM+Budimlija&amp;author=M+Prinz&amp;volume=54&amp;publication_year=2013a&amp;pages=248-256&amp;pmid=23771755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="element-citation">Hart KL, Kimura SL, Mushailov V, Budimlija ZM, Prinz M, Wurmbach E. Improved eye- and skin-color prediction based on 8 SNPs. <span><span class="ref-journal">Croat Med J. </span>2013b;<span class="ref-vol">54</span>:248–256.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3694299/">PMC free article</a>]</span> [<a href="/pubmed/23771755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Croat+Med+J&amp;title=Improved+eye-+and+skin-color+prediction+based+on+8+SNPs&amp;author=KL+Hart&amp;author=SL+Kimura&amp;author=V+Mushailov&amp;author=ZM+Budimlija&amp;author=M+Prinz&amp;volume=54&amp;publication_year=2013b&amp;pages=248-256&amp;pmid=23771755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="element-citation">Henn BM, Botigué LR, Peischl S, Dupanloup I, Lipatov M, Maples BK, Martin AR, Musharoff S, Cann H, Snyder MP, et al.  Distance from sub-Saharan Africa predicts mutational load in diverse human genomes. <span><span class="ref-journal">Proc Natl Acad Sci USa. </span>2016;<span class="ref-vol">113</span>:E440–E449.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4743782/">PMC free article</a>]</span> [<a href="/pubmed/26712023" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USa&amp;title=Distance+from+sub-Saharan+Africa+predicts+mutational+load+in+diverse+human+genomes&amp;author=BM+Henn&amp;author=LR+Botigué&amp;author=S+Peischl&amp;author=I+Dupanloup&amp;author=M+Lipatov&amp;volume=113&amp;publication_year=2016&amp;pages=E440-E449&amp;pmid=26712023&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="element-citation">Henn BM, Gignoux CR, Jobin M, Granka JM, Macpherson JM, Kidd JM, Rodríguez-Botigué L, Ramachandran S, Hon L, Brisbin A, et al.  Hunter-gatherer genomic diversity suggests a southern African origin for modern humans. <span><span class="ref-journal">Proc Natl Acad Sci USa. </span>2011;<span class="ref-vol">108</span>:5154–5162.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3069156/">PMC free article</a>]</span> [<a href="/pubmed/21383195" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USa&amp;title=Hunter-gatherer+genomic+diversity+suggests+a+southern+African+origin+for+modern+humans&amp;author=BM+Henn&amp;author=CR+Gignoux&amp;author=M+Jobin&amp;author=JM+Granka&amp;author=JM+Macpherson&amp;volume=108&amp;publication_year=2011&amp;pages=5154-5162&amp;pmid=21383195&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="element-citation">Howie BN, Donnelly P, Marchini J. A Flexible and Accurate Genotype Imputation Method for the Next Generation of Genome-Wide Association Studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+Flexible+and+Accurate+Genotype+Imputation+Method+for+the+Next+Generation+of+Genome-Wide+Association+Studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="element-citation">Jablonski NG, Chaplin G. Colloquium paper: human skin pigmentation as an adaptation to UV radiation. <span><span class="ref-journal">Proc Natl Acad Sci USa. </span>2010;<span class="ref-vol">107</span>(Suppl 2):8962–8968.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3024016/">PMC free article</a>]</span> [<a href="/pubmed/20445093" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USa&amp;title=Colloquium+paper:+human+skin+pigmentation+as+an+adaptation+to+UV+radiation&amp;author=NG+Jablonski&amp;author=G+Chaplin&amp;volume=107&amp;issue=Suppl+2&amp;publication_year=2010&amp;pages=8962-8968&amp;pmid=20445093&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="element-citation">Jablonski NG, Chaplin G. The Evolution of Skin Pigmentation and Hair Texture in People of African Ancestry. <span><span class="ref-journal">Dermatologic Clinics. </span>2014;<span class="ref-vol">32</span>:113–121.</span> [<a href="/pubmed/24679998" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dermatologic+Clinics&amp;title=The+Evolution+of+Skin+Pigmentation+and+Hair+Texture+in+People+of+African+Ancestry&amp;author=NG+Jablonski&amp;author=G+Chaplin&amp;volume=32&amp;publication_year=2014&amp;pages=113-121&amp;pmid=24679998&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="element-citation">Kang HM, Sul JH, Service SK, Zaitlen NA, Kong SY, Freimer NB, Sabatti C, Eskin E. Variance component model to account for sample structure in genome-wide association studies. <span><span class="ref-journal">Nature Publishing Group. </span>2010;<span class="ref-vol">42</span>:348–354.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3092069/">PMC free article</a>]</span> [<a href="/pubmed/20208533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Publishing+Group&amp;title=Variance+component+model+to+account+for+sample+structure+in+genome-wide+association+studies&amp;author=HM+Kang&amp;author=JH+Sul&amp;author=SK+Service&amp;author=NA+Zaitlen&amp;author=SY+Kong&amp;volume=42&amp;publication_year=2010&amp;pages=348-354&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="element-citation">Keenen B, Qi H, Saladi SV, Yeung M, de la Serna IL. Heterogeneous SWI/SNF chromatin remodeling complexes promote expression of microphthalmia-associated transcription factor target genes in melanoma. <span><span class="ref-journal">Oncogene. </span>2009a;<span class="ref-vol">29</span>:81–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2803337/">PMC free article</a>]</span> [<a href="/pubmed/19784067" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Heterogeneous+SWI/SNF+chromatin+remodeling+complexes+promote+expression+of+microphthalmia-associated+transcription+factor+target+genes+in+melanoma&amp;author=B+Keenen&amp;author=H+Qi&amp;author=SV+Saladi&amp;author=M+Yeung&amp;author=IL+de+la+Serna&amp;volume=29&amp;publication_year=2009a&amp;pages=81-92&amp;pmid=19784067&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R39"><span class="element-citation">Keenen B, Qi H, Saladi SV, Yeung M, de la Serna IL. Heterogeneous SWI/SNF chromatin remodeling complexes promote expression of microphthalmia-associated transcription factor target genes in melanoma. <span><span class="ref-journal">Oncogene. </span>2009b;<span class="ref-vol">29</span>:81–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2803337/">PMC free article</a>]</span> [<a href="/pubmed/19784067" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Heterogeneous+SWI/SNF+chromatin+remodeling+complexes+promote+expression+of+microphthalmia-associated+transcription+factor+target+genes+in+melanoma&amp;author=B+Keenen&amp;author=H+Qi&amp;author=SV+Saladi&amp;author=M+Yeung&amp;author=IL+de+la+Serna&amp;volume=29&amp;publication_year=2009b&amp;pages=81-92&amp;pmid=19784067&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="element-citation">Kenny EE, Timpson NJ, Sikora M, Yee MC, Moreno-Estrada A, Eng C, Huntsman S, Burchard EG, Stoneking M, Bustamante CD, et al.  Melanesian Blond Hair Is Caused by an Amino Acid Change in TYRP1. <span><span class="ref-journal">Science. </span>2012a;<span class="ref-vol">336</span>:554–554.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3481182/">PMC free article</a>]</span> [<a href="/pubmed/22556244" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Melanesian+Blond+Hair+Is+Caused+by+an+Amino+Acid+Change+in+TYRP1&amp;author=EE+Kenny&amp;author=NJ+Timpson&amp;author=M+Sikora&amp;author=MC+Yee&amp;author=A+Moreno-Estrada&amp;volume=336&amp;publication_year=2012a&amp;pages=554-554&amp;pmid=22556244&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R41"><span class="element-citation">Kenny EE, Timpson NJ, Sikora M, Yee MC, Moreno-Estrada A, Eng C, Huntsman S, Burchard EG, Stoneking M, Bustamante CD, et al.  Melanesian Blond Hair Is Caused by an Amino Acid Change in TYRP1. <span><span class="ref-journal">Science. </span>2012b;<span class="ref-vol">336</span>:554–554.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3481182/">PMC free article</a>]</span> [<a href="/pubmed/22556244" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Melanesian+Blond+Hair+Is+Caused+by+an+Amino+Acid+Change+in+TYRP1&amp;author=EE+Kenny&amp;author=NJ+Timpson&amp;author=M+Sikora&amp;author=MC+Yee&amp;author=A+Moreno-Estrada&amp;volume=336&amp;publication_year=2012b&amp;pages=554-554&amp;pmid=22556244&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R42"><span class="element-citation">Lamason RL. SLC24A5, a Putative Cation Exchanger, Affects Pigmentation in Zebrafish and Humans. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">310</span>:1782–1786.</span> [<a href="/pubmed/16357253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=SLC24A5,+a+Putative+Cation+Exchanger,+Affects+Pigmentation+in+Zebrafish+and+Humans&amp;author=RL+Lamason&amp;volume=310&amp;publication_year=2005&amp;pages=1782-1786&amp;pmid=16357253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R43"><span class="element-citation">Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. <span><span class="ref-journal">Bioinformatics. </span>2009;<span class="ref-vol">25</span>:1754–1760.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2705234/">PMC free article</a>]</span> [<a href="/pubmed/19451168" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Fast+and+accurate+short+read+alignment+with+Burrows-Wheeler+transform&amp;author=H+Li&amp;author=R+Durbin&amp;volume=25&amp;publication_year=2009&amp;pages=1754-1760&amp;pmid=19451168&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R44"><span class="element-citation">Liu F, Visser M, Duffy DL, Hysi PG, Jacobs LC, Lao O, Zhong K, Walsh S, Chaitanya L, Wollstein A, et al.  Genetics of skin color variation in Europeans: genome-wide association studies with functional follow-up. <span><span class="ref-journal">Human Genetics. </span>2015;<span class="ref-vol">134</span>:823–835.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4495261/">PMC free article</a>]</span> [<a href="/pubmed/25963972" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Genetics&amp;title=Genetics+of+skin+color+variation+in+Europeans:+genome-wide+association+studies+with+functional+follow-up&amp;author=F+Liu&amp;author=M+Visser&amp;author=DL+Duffy&amp;author=PG+Hysi&amp;author=LC+Jacobs&amp;volume=134&amp;publication_year=2015&amp;pages=823-835&amp;pmid=25963972&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R45"><span class="element-citation">Manichaikul A, Mychaleckyj JC, Rich SS, Daly K, Sale M, Chen WM. Robust relationship inference in genome-wide association studies. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2867–2873.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025716/">PMC free article</a>]</span> [<a href="/pubmed/20926424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Robust+relationship+inference+in+genome-wide+association+studies&amp;author=A+Manichaikul&amp;author=JC+Mychaleckyj&amp;author=SS+Rich&amp;author=K+Daly&amp;author=M+Sale&amp;volume=26&amp;publication_year=2010&amp;pages=2867-2873&amp;pmid=20926424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R46"><span class="element-citation">Maples BK, Gravel S, Kenny EE, Bustamante CD. RFMix: A Discriminative Modeling Approach for Rapid and Robust Local-Ancestry Inference. <span><span class="ref-journal">The American Journal of Human Genetics. </span>2013;<span class="ref-vol">93</span>:278–288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3738819/">PMC free article</a>]</span> [<a href="/pubmed/23910464" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+American+Journal+of+Human+Genetics&amp;title=RFMix:+A+Discriminative+Modeling+Approach+for+Rapid+and+Robust+Local-Ancestry+Inference&amp;author=BK+Maples&amp;author=S+Gravel&amp;author=EE+Kenny&amp;author=CD+Bustamante&amp;volume=93&amp;publication_year=2013&amp;pages=278-288&amp;pmid=23910464&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R47"><span class="element-citation">Martin AR, Gignoux CR, Walters RK, Wojcik GL, Neale BM, Gravel S, Daly MJ, Bustamante CD, Kenny EE. AR TICLE Human Demographic History Impacts Genetic Risk Prediction across Diverse Populations. <span><span class="ref-journal">The American Journal of Human Genetics. </span>2017:1–36.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5384097/">PMC free article</a>]</span> [<a href="/pubmed/28366442" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+American+Journal+of+Human+Genetics&amp;title=AR+TICLE+Human+Demographic+History+Impacts+Genetic+Risk+Prediction+across+Diverse+Populations&amp;author=AR+Martin&amp;author=CR+Gignoux&amp;author=RK+Walters&amp;author=GL+Wojcik&amp;author=BM+Neale&amp;publication_year=2017&amp;pages=1-36&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R48"><span class="element-citation">McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, Garimella K, Altshuler D, Gabriel S, Daly M, et al.  The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data. <span><span class="ref-journal">Genome Research. </span>2010;<span class="ref-vol">20</span>:1297–1303.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928508/">PMC free article</a>]</span> [<a href="/pubmed/20644199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Research&amp;title=The+Genome+Analysis+Toolkit:+A+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data&amp;author=A+McKenna&amp;author=M+Hanna&amp;author=E+Banks&amp;author=A+Sivachenko&amp;author=K+Cibulskis&amp;volume=20&amp;publication_year=2010&amp;pages=1297-1303&amp;pmid=20644199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R49"><span class="element-citation">McLaren W, Gil L, Hunt SE, Riat HS, Ritchie GRS, Thormann A, Flicek P, Cunningham F. The Ensembl Variant Effect Predictor. <span><span class="ref-journal">Genome Biol. </span>2016:1–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4893825/">PMC free article</a>]</span> [<a href="/pubmed/27268795" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=The+Ensembl+Variant+Effect+Predictor&amp;author=W+McLaren&amp;author=L+Gil&amp;author=SE+Hunt&amp;author=HS+Riat&amp;author=GRS+Ritchie&amp;publication_year=2016&amp;pages=1-14&amp;pmid=26753840&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R50"><span class="element-citation">Miller CT, Beleza S, Pollen AA, Schluter D, Kittles RA, Shriver MD, Kingsley DM. cis-Regulatory Changes in Kit Ligand Expression and Parallel Evolution of Pigmentation in Sticklebacks and Humans. <span><span class="ref-journal">Cell. </span>2007;<span class="ref-vol">131</span>:1179–1189.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2900316/">PMC free article</a>]</span> [<a href="/pubmed/18083106" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=cis-Regulatory+Changes+in+Kit+Ligand+Expression+and+Parallel+Evolution+of+Pigmentation+in+Sticklebacks+and+Humans&amp;author=CT+Miller&amp;author=S+Beleza&amp;author=AA+Pollen&amp;author=D+Schluter&amp;author=RA+Kittles&amp;volume=131&amp;publication_year=2007&amp;pages=1179-1189&amp;pmid=18083106&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R51"><span class="element-citation">Nan H, Kraft P, Qureshi AA, Guo Q, Chen C, Hankinson SE, Hu FB, Thomas G, Hoover RN, Chanock S, et al.  Genome-Wide Association Study of Tanning Phenotype in a Population of European Ancestry. <span>2009;<span class="ref-vol">129</span>:2250–2257.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2758927/">PMC free article</a>]</span> [<a href="/pubmed/19340012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Nan+H+Kraft+P+Qureshi+AA+Guo+Q+Chen+C+Hankinson+SE+Hu+FB+Thomas+G+Hoover+RN+Chanock+S+2009+Genome-Wide+Association+Study+of+Tanning+Phenotype+in+a+Population+of+European+Ancestry+129+2250+2257+" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R52"><span class="element-citation">Norton HL, Kittles RA, Parra E, McKeigue P, Mao X, Cheng K, Canfield VA, Bradley DG, McEvoy B, Shriver MD. Genetic Evidence for the Convergent Evolution of Light Skin in Europeans and East Asians. <span><span class="ref-journal">Mol Biol Evol. </span>2006;<span class="ref-vol">24</span>:710–722.</span> [<a href="/pubmed/17182896" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=Genetic+Evidence+for+the+Convergent+Evolution+of+Light+Skin+in+Europeans+and+East+Asians&amp;author=HL+Norton&amp;author=RA+Kittles&amp;author=E+Parra&amp;author=P+McKeigue&amp;author=X+Mao&amp;volume=24&amp;publication_year=2006&amp;pages=710-722&amp;pmid=17182896&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R53"><span class="element-citation">O’Connell J, Gurdasani D, Delaneau O, Pirastu N, Ulivi S, Cocca M, Traglia M, Huang J, Huffman JE, Rudan I, et al.  A General Approach for Haplotype Phasing across the Full Spectrum of Relatedness. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004234.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3990520/">PMC free article</a>]</span> [<a href="/pubmed/24743097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+General+Approach+for+Haplotype+Phasing+across+the+Full+Spectrum+of+Relatedness&amp;author=J+O’Connell&amp;author=D+Gurdasani&amp;author=O+Delaneau&amp;author=N+Pirastu&amp;author=S+Ulivi&amp;volume=10&amp;publication_year=2014&amp;pages=e1004234&amp;pmid=24743097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R54"><span class="element-citation">Pickrell JK, Patterson N, Loh PR, Lipson M, Berger B, Stoneking M, Pakendorf B, Reich D. Ancient west Eurasian ancestry in southern and eastern Africa. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2014;<span class="ref-vol">111</span>:2632–2637.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3932865/">PMC free article</a>]</span> [<a href="/pubmed/24550290" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Ancient+west+Eurasian+ancestry+in+southern+and+eastern+Africa&amp;author=JK+Pickrell&amp;author=N+Patterson&amp;author=PR+Loh&amp;author=M+Lipson&amp;author=B+Berger&amp;volume=111&amp;publication_year=2014&amp;pages=2632-2637&amp;pmid=24550290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R55"><span class="element-citation">Pospiech E, Wojas-Pelc A, Walsh S, Liu F, Maeda H, Ishikawa T, Skowron M, Kayser M, Branicki W. The common occurrence of epistasis in the determination of human pigmentation and its impact on DNA-based pigmentation phenotype prediction. <span><span class="ref-journal">Forensic Science International: Genetics. </span>2014;<span class="ref-vol">11</span>:64–72.</span> [<a href="/pubmed/24681889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Forensic+Science+International:+Genetics&amp;title=The+common+occurrence+of+epistasis+in+the+determination+of+human+pigmentation+and+its+impact+on+DNA-based+pigmentation+phenotype+prediction&amp;author=E+Pospiech&amp;author=A+Wojas-Pelc&amp;author=S+Walsh&amp;author=F+Liu&amp;author=H+Maeda&amp;volume=11&amp;publication_year=2014&amp;pages=64-72&amp;pmid=24681889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R56"><span class="element-citation">Prado-Martinez J, Sudmant PH, Kidd JM, Li H, Kelley JL, Lorente-Galdos B, Veeramah KR, Woerner AE, O’Connor TD, Santpere G, et al.  Great ape genetic diversity and population history. <span><span class="ref-journal">Nature. </span>2013;<span class="ref-vol">499</span>:471–475.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3822165/">PMC free article</a>]</span> [<a href="/pubmed/23823723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Great+ape+genetic+diversity+and+population+history&amp;author=J+Prado-Martinez&amp;author=PH+Sudmant&amp;author=JM+Kidd&amp;author=H+Li&amp;author=JL+Kelley&amp;volume=499&amp;publication_year=2013&amp;pages=471-475&amp;pmid=23823723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R57"><span class="element-citation">Praetorius C, Grill C, Stacey SN, Metcalf AM, Gorkin DU, Robinson KC, Van Otterloo E, Kim RSQ, Bergsteinsdottir K, Ogmundsdottir MH, et al.  A Polymorphism in IRF4 Affects Human Pigmentation through a Tyrosinase-Dependent MITF/TFAP2A Pathway. <span><span class="ref-journal">Cell. </span>2013;<span class="ref-vol">155</span>:1022–1033.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3873608/">PMC free article</a>]</span> [<a href="/pubmed/24267888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+Polymorphism+in+IRF4+Affects+Human+Pigmentation+through+a+Tyrosinase-Dependent+MITF/TFAP2A+Pathway&amp;author=C+Praetorius&amp;author=C+Grill&amp;author=SN+Stacey&amp;author=AM+Metcalf&amp;author=DU+Gorkin&amp;volume=155&amp;publication_year=2013&amp;pages=1022-1033&amp;pmid=24267888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R58"><span class="element-citation">Relethford JH. Human skin color diversity is highest in sub-Saharan African populations. <span><span class="ref-journal">Human Biology. </span>2000;<span class="ref-vol">72</span>:773–780.</span> [<a href="/pubmed/11126724" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Biology&amp;title=Human+skin+color+diversity+is+highest+in+sub-Saharan+African+populations&amp;author=JH+Relethford&amp;volume=72&amp;publication_year=2000&amp;pages=773-780&amp;pmid=11126724&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R59"><span class="element-citation">Ripke S, Neale BM, Corvin A, Walters JTR, Farh KH, Holmans PA, Lee P, Bulik-Sullivan B, Collier DA, Huang H, et al.  Biological insights from 108 schizophrenia-associated genetic loci. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">511</span>:421–427.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biological+insights+from+108+schizophrenia-associated+genetic+loci&amp;author=S+Ripke&amp;author=BM+Neale&amp;author=A+Corvin&amp;author=JTR+Walters&amp;author=KH+Farh&amp;volume=511&amp;publication_year=2014&amp;pages=421-427&amp;pmid=25056061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R60"><span class="element-citation">Sarangarajan R, Boissy RE. Tyrp1 and Oculocutaneous Albinism Type 3. <span><span class="ref-journal">Pigment Cell Res. </span>2001;<span class="ref-vol">14</span>:437–444.</span> [<a href="/pubmed/11775055" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pigment+Cell+Res&amp;title=Tyrp1+and+Oculocutaneous+Albinism+Type+3&amp;author=R+Sarangarajan&amp;author=RE+Boissy&amp;volume=14&amp;publication_year=2001&amp;pages=437-444&amp;pmid=11775055&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R61"><span class="element-citation">Schlebusch CM, Skoglund P, Sjodin P, Gattepaille LM, Hernandez D, Jay F, Li S, De Jongh M, Singleton A, Blum MGB, et al.  Genomic Variation in Seven Khoe-San Groups Reveals Adaptation and Complex African History. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">338</span>:374–379.</span> [<a href="/pubmed/22997136" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genomic+Variation+in+Seven+Khoe-San+Groups+Reveals+Adaptation+and+Complex+African+History&amp;author=CM+Schlebusch&amp;author=P+Skoglund&amp;author=P+Sjodin&amp;author=LM+Gattepaille&amp;author=D+Hernandez&amp;volume=338&amp;publication_year=2012&amp;pages=374-379&amp;pmid=22997136&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R62"><span class="element-citation">Schuster SC, Miller W, Ratan A, Tomsho LP, Giardine B, Kasson LR, Harris RS, Petersen DC, Zhao F, Qi J, et al.  Complete Khoisan and Bantu genomes from southern Africa. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">463</span>:943–947.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3890430/">PMC free article</a>]</span> [<a href="/pubmed/20164927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Complete+Khoisan+and+Bantu+genomes+from+southern+Africa&amp;author=SC+Schuster&amp;author=W+Miller&amp;author=A+Ratan&amp;author=LP+Tomsho&amp;author=B+Giardine&amp;volume=463&amp;publication_year=2010&amp;pages=943-947&amp;pmid=20164927&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R63"><span class="element-citation">Shringarpure SS, Bustamante CD, Lange K, Alexander DH. Efficient analysis of large datasets and sex bias with ADMIXTURE. <span><span class="ref-journal">BMC Bioinformatics. </span>2016;<span class="ref-vol">17</span>:1655.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4877806/">PMC free article</a>]</span> [<a href="/pubmed/27216439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=Efficient+analysis+of+large+datasets+and+sex+bias+with+ADMIXTURE&amp;author=SS+Shringarpure&amp;author=CD+Bustamante&amp;author=K+Lange&amp;author=DH+Alexander&amp;volume=17&amp;publication_year=2016&amp;pages=1655&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R64"><span class="element-citation">Shriver MD, Parra EJ. Comparison of narrow-band reflectance spectroscopy and tristimulus colorimetry for measurements of skin and hair color in persons of different biological ancestry. <span><span class="ref-journal">Am J Phys Anthropol. </span>2000;<span class="ref-vol">112</span>:17–27.</span> [<a href="/pubmed/10766940" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Phys+Anthropol&amp;title=Comparison+of+narrow-band+reflectance+spectroscopy+and+tristimulus+colorimetry+for+measurements+of+skin+and+hair+color+in+persons+of+different+biological+ancestry&amp;author=MD+Shriver&amp;author=EJ+Parra&amp;volume=112&amp;publication_year=2000&amp;pages=17-27&amp;pmid=10766940&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R65"><span class="element-citation">Spichenok O, Budimlija ZM, Mitchell AA, Jenny A, Kovacevic L, Marjanović D, Caragine T, Prinz M, Wurmbach E. Prediction of eye and skin color in diverse populations using seven SNPs. <span><span class="ref-journal">Forensic Science International: Genetics. </span>2011;<span class="ref-vol">5</span>:472–478.</span> [<a href="/pubmed/21050833" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Forensic+Science+International:+Genetics&amp;title=Prediction+of+eye+and+skin+color+in+diverse+populations+using+seven+SNPs&amp;author=O+Spichenok&amp;author=ZM+Budimlija&amp;author=AA+Mitchell&amp;author=A+Jenny&amp;author=L+Kovacevic&amp;volume=5&amp;publication_year=2011&amp;pages=472-478&amp;pmid=21050833&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R66"><span class="element-citation">Sturm RA. Molecular genetics of human pigmentation diversity. <span><span class="ref-journal">Human Molecular Genetics. </span>2009;<span class="ref-vol">18</span>:R9–R17.</span> [<a href="/pubmed/19297406" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Molecular+Genetics&amp;title=Molecular+genetics+of+human+pigmentation+diversity&amp;author=RA+Sturm&amp;volume=18&amp;publication_year=2009&amp;pages=R9-R17&amp;pmid=19297406&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R67"><span class="element-citation">Sturm RA, Duffy DL. Human pigmentation genes under environmental selection. <span><span class="ref-journal">Genome Biol. </span>2012;<span class="ref-vol">13</span>:248–15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3491390/">PMC free article</a>]</span> [<a href="/pubmed/23110848" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Human+pigmentation+genes+under+environmental+selection&amp;author=RA+Sturm&amp;author=DL+Duffy&amp;volume=13&amp;publication_year=2012&amp;pages=248-15&amp;pmid=23110848&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R68"><span class="element-citation">Sulem P, Gudbjartsson DF, Stacey SN, Helgason A, Rafnar T, Jakobsdottir M, Steinberg S, Gudjonsson SA, Palsson A, Thorleifsson G, et al.  Two newly identified genetic determinants of pigmentation in Europeans. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:835–837.</span> [<a href="/pubmed/18488028" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Two+newly+identified+genetic+determinants+of+pigmentation+in+Europeans&amp;author=P+Sulem&amp;author=DF+Gudbjartsson&amp;author=SN+Stacey&amp;author=A+Helgason&amp;author=T+Rafnar&amp;volume=40&amp;publication_year=2008&amp;pages=835-837&amp;pmid=18488028&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R69"><span class="element-citation">Sulem P, Gudbjartsson DF, Stacey SN, Helgason A, Rafnar T, Magnusson KP, Manolescu A, Karason A, Palsson A, Thorleifsson G, et al.  Genetic determinants of hair, eye and skin pigmentation in Europeans. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1443–1452.</span> [<a href="/pubmed/17952075" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+determinants+of+hair,+eye+and+skin+pigmentation+in+Europeans&amp;author=P+Sulem&amp;author=DF+Gudbjartsson&amp;author=SN+Stacey&amp;author=A+Helgason&amp;author=T+Rafnar&amp;volume=39&amp;publication_year=2007&amp;pages=1443-1452&amp;pmid=17952075&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R70"><span class="element-citation">Thornton T, Tang H, Hoffmann TJ, Ochs-Balcom HM, Caan BJ, Risch N. Estimating Kinship in Admixed Populations. <span><span class="ref-journal">The American Journal of Human Genetics. </span>2012a;<span class="ref-vol">91</span>:122–138.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3397261/">PMC free article</a>]</span> [<a href="/pubmed/22748210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+American+Journal+of+Human+Genetics&amp;title=Estimating+Kinship+in+Admixed+Populations&amp;author=T+Thornton&amp;author=H+Tang&amp;author=TJ+Hoffmann&amp;author=HM+Ochs-Balcom&amp;author=BJ+Caan&amp;volume=91&amp;publication_year=2012a&amp;pages=122-138&amp;pmid=22748210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R71"><span class="element-citation">Thornton T, Tang H, Hoffmann TJ, Ochs-Balcom HM, Caan BJ, Risch N. Estimating Kinship in Admixed Populations. <span><span class="ref-journal">The American Journal of Human Genetics. </span>2012b;<span class="ref-vol">91</span>:122–138.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3397261/">PMC free article</a>]</span> [<a href="/pubmed/22748210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+American+Journal+of+Human+Genetics&amp;title=Estimating+Kinship+in+Admixed+Populations&amp;author=T+Thornton&amp;author=H+Tang&amp;author=TJ+Hoffmann&amp;author=HM+Ochs-Balcom&amp;author=BJ+Caan&amp;volume=91&amp;publication_year=2012b&amp;pages=122-138&amp;pmid=22748210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R72"><span class="element-citation">Tishkoff SA, Reed FA, Friedlaender FR, Ehret C, Ranciaro A, Froment A, Hirbo JB, Awomoyi AA, Bodo JM, Doumbo O, et al.  The Genetic Structure and History of Africans and African Americans. <span><span class="ref-journal">Science. </span>2009;<span class="ref-vol">324</span>:1035–1044.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947357/">PMC free article</a>]</span> [<a href="/pubmed/19407144" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+Genetic+Structure+and+History+of+Africans+and+African+Americans&amp;author=SA+Tishkoff&amp;author=FA+Reed&amp;author=FR+Friedlaender&amp;author=C+Ehret&amp;author=A+Ranciaro&amp;volume=324&amp;publication_year=2009&amp;pages=1035-1044&amp;pmid=19407144&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R73"><span class="element-citation">Uren C, Kim M, Martin AR, Bobo D, Gignoux CR, van Helden PD, Möller M, Hoal EG, Henn BM. Fine-Scale Human Population Structure in Southern Africa Reflects Ecogeographic Boundaries. <span><span class="ref-journal">Genetics. </span>2016a;<span class="ref-vol">204</span>:303–314.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5012395/">PMC free article</a>]</span> [<a href="/pubmed/27474727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Fine-Scale+Human+Population+Structure+in+Southern+Africa+Reflects+Ecogeographic+Boundaries&amp;author=C+Uren&amp;author=M+Kim&amp;author=AR+Martin&amp;author=D+Bobo&amp;author=CR+Gignoux&amp;volume=204&amp;publication_year=2016a&amp;pages=303-314&amp;pmid=27474727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R74"><span class="element-citation">Uren C, Kim M, Martin AR, Bobo D, Gignoux CR, van Helden PD, Möller M, Hoal EG, Henn BM. Fine-Scale Human Population Structure in Southern Africa Reflects Ecogeographic Boundaries. <span><span class="ref-journal">Genetics. </span>2016b;<span class="ref-vol">204</span>:303–314.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5012395/">PMC free article</a>]</span> [<a href="/pubmed/27474727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Fine-Scale+Human+Population+Structure+in+Southern+Africa+Reflects+Ecogeographic+Boundaries&amp;author=C+Uren&amp;author=M+Kim&amp;author=AR+Martin&amp;author=D+Bobo&amp;author=CR+Gignoux&amp;volume=204&amp;publication_year=2016b&amp;pages=303-314&amp;pmid=27474727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R75"><span class="element-citation">Vachtenheim J, Ondrušová L, Borovanský J. SWI/SNF chromatin remodeling complex is critical for the expression of microphthalmia-associated transcription factor in melanoma cells. <span><span class="ref-journal">Biochemical and Biophysical Research Communications. </span>2010;<span class="ref-vol">392</span>:454–459.</span> [<a href="/pubmed/20083088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical+and+Biophysical+Research+Communications&amp;title=SWI/SNF+chromatin+remodeling+complex+is+critical+for+the+expression+of+microphthalmia-associated+transcription+factor+in+melanoma+cells&amp;author=J+Vachtenheim&amp;author=L+Ondrušová&amp;author=J+Borovanský&amp;volume=392&amp;publication_year=2010&amp;pages=454-459&amp;pmid=20083088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R76"><span class="element-citation">Veeramah KR, Wegmann D, Woerner A, Mendez FL, Watkins JC, Destro-Bisol G, Soodyall H, Louie L, Hammer MF. An Early Divergence of KhoeSan Ancestors from Those of Other Modern Humans Is Supported by an ABC-Based Analysis of Autosomal Resequencing Data. <span><span class="ref-journal">Mol Biol Evol. </span>2011;<span class="ref-vol">29</span>:617–630.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3258037/">PMC free article</a>]</span> [<a href="/pubmed/21890477" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=An+Early+Divergence+of+KhoeSan+Ancestors+from+Those+of+Other+Modern+Humans+Is+Supported+by+an+ABC-Based+Analysis+of+Autosomal+Resequencing+Data&amp;author=KR+Veeramah&amp;author=D+Wegmann&amp;author=A+Woerner&amp;author=FL+Mendez&amp;author=JC+Watkins&amp;volume=29&amp;publication_year=2011&amp;pages=617-630&amp;pmid=21890477&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R77"><span class="element-citation">Walsh S, Liu F, Wollstein A, Kovatsi L, Ralf A, Kosiniak-Kamysz A, Branicki W, Kayser M. The HIrisPlex system for simultaneous prediction of hair and eye colour from DNA. <span><span class="ref-journal">Forensic Science International: Genetics. </span>2013;<span class="ref-vol">7</span>:98–115.</span> [<a href="/pubmed/22917817" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Forensic+Science+International:+Genetics&amp;title=The+HIrisPlex+system+for+simultaneous+prediction+of+hair+and+eye+colour+from+DNA&amp;author=S+Walsh&amp;author=F+Liu&amp;author=A+Wollstein&amp;author=L+Kovatsi&amp;author=A+Ralf&amp;volume=7&amp;publication_year=2013&amp;pages=98-115&amp;pmid=22917817&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R78"><span class="element-citation">Ward LD, Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. <span><span class="ref-journal">Nucleic Acids Research. </span>2011;<span class="ref-vol">40</span>:D930–D934.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245002/">PMC free article</a>]</span> [<a href="/pubmed/22064851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Research&amp;title=HaploReg:+a+resource+for+exploring+chromatin+states,+conservation,+and+regulatory+motif+alterations+within+sets+of+genetically+linked+variants&amp;author=LD+Ward&amp;author=M+Kellis&amp;volume=40&amp;publication_year=2011&amp;pages=D930-D934&amp;pmid=22064851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R79"><span class="element-citation">Wilde S, Timpson A, Kirsanow K, Kaiser E, Kayser M, Unterländer M, Hollfelder N, Potekhina ID, Schier W, Thomas MG, et al.  Direct evidence for positive selection of skin, hair, and eye pigmentation in Europeans during the last 5,000 y. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2014;<span class="ref-vol">111</span>:4832–4837.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3977302/">PMC free article</a>]</span> [<a href="/pubmed/24616518" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Direct+evidence+for+positive+selection+of+skin,+hair,+and+eye+pigmentation+in+Europeans+during+the+last+5,000+y&amp;author=S+Wilde&amp;author=A+Timpson&amp;author=K+Kirsanow&amp;author=E+Kaiser&amp;author=M+Kayser&amp;volume=111&amp;publication_year=2014&amp;pages=4832-4837&amp;pmid=24616518&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R80"><span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R81"><span class="element-citation">Wood AR, Esko T, Yang J, Vedantam S, Pers TH, Gustafsson S, Chu AY, Estrada K, Luan J, Kutalik Z, et al.  Defining the role of common variation in the genomic and biological architecture of adult human height. <span><span class="ref-journal">Nature Publishing Group. </span>2014;<span class="ref-vol">46</span>:1173–1186.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4250049/">PMC free article</a>]</span> [<a href="/pubmed/25282103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Publishing+Group&amp;title=Defining+the+role+of+common+variation+in+the+genomic+and+biological+architecture+of+adult+human+height&amp;author=AR+Wood&amp;author=T+Esko&amp;author=J+Yang&amp;author=S+Vedantam&amp;author=TH+Pers&amp;volume=46&amp;publication_year=2014&amp;pages=1173-1186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R82"><span class="element-citation">Xia CH, Lu E, Zeng J, Gong X. Deletion of LRP5 in VLDLR Knockout Mice Inhibits Retinal Neovascularization. <span><span class="ref-journal">PLoS ONE. </span>2013a;<span class="ref-vol">8</span>:e75186–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772893/">PMC free article</a>]</span> [<a href="/pubmed/24058663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Deletion+of+LRP5+in+VLDLR+Knockout+Mice+Inhibits+Retinal+Neovascularization&amp;author=CH+Xia&amp;author=E+Lu&amp;author=J+Zeng&amp;author=X+Gong&amp;volume=8&amp;publication_year=2013a&amp;pages=e75186-11&amp;pmid=24058663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R83"><span class="element-citation">Xia CH, Lu E, Zeng J, Gong X. Deletion of LRP5 in VLDLR Knockout Mice Inhibits Retinal Neovascularization. <span><span class="ref-journal">PLoS ONE. </span>2013b;<span class="ref-vol">8</span>:e75186.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772893/">PMC free article</a>]</span> [<a href="/pubmed/24058663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Deletion+of+LRP5+in+VLDLR+Knockout+Mice+Inhibits+Retinal+Neovascularization&amp;author=CH+Xia&amp;author=E+Lu&amp;author=J+Zeng&amp;author=X+Gong&amp;volume=8&amp;publication_year=2013b&amp;pages=e75186&amp;pmid=24058663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R84"><span class="element-citation">Yang J, Benyamin B, McEvoy BP, Gordon S, Henders AK, Nyholt DR, Madden PA, Heath AC, Martin NG, Montgomery GW, et al.  Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:565–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J+Yang&amp;author=B+Benyamin&amp;author=BP+McEvoy&amp;author=S+Gordon&amp;author=AK+Henders&amp;volume=42&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R85"><span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: A Tool for Genome-wide Complex Trait Analysis. <span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Yang+J+Lee+SH+Goddard+ME+Visscher+PM+2011+GCTA:+A+Tool+for+Genome-wide+Complex+Trait+Analysis+88+76+82+" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R86"><span class="element-citation">Yang Z, Zhong H, Chen J, Zhang X, Zhang H, Luo X, Xu S, Chen H, Lu D, Han Y, et al.  A Genetic Mechanism for Convergent Skin Lightening during Recent Human Evolution. <span><span class="ref-journal">Mol Biol Evol. </span>2016;<span class="ref-vol">33</span>:1177–1187.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4839214/">PMC free article</a>]</span> [<a href="/pubmed/26744415" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=A+Genetic+Mechanism+for+Convergent+Skin+Lightening+during+Recent+Human+Evolution&amp;author=Z+Yang&amp;author=H+Zhong&amp;author=J+Chen&amp;author=X+Zhang&amp;author=H+Zhang&amp;volume=33&amp;publication_year=2016&amp;pages=1177-1187&amp;pmid=26744415&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R87"><span class="element-citation">Yi X, Liang Y, Huerta-Sanchez E, Jin X, Cuo ZXP, Pool JE, Xu X, Jiang H, Vinckenbosch N, Korneliussen TS, et al.  Sequencing of 50 Human Exomes Reveals Adaptation to High Altitude. <span><span class="ref-journal">Science. </span>2010;<span class="ref-vol">329</span>:75–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3711608/">PMC free article</a>]</span> [<a href="/pubmed/20595611" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Sequencing+of+50+Human+Exomes+Reveals+Adaptation+to+High+Altitude&amp;author=X+Yi&amp;author=Y+Liang&amp;author=E+Huerta-Sanchez&amp;author=X+Jin&amp;author=ZXP+Cuo&amp;volume=329&amp;publication_year=2010&amp;pages=75-78&amp;pmid=20595611&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R88"><span class="element-citation">Zaidi AA, Mattern BC, Claes P, McEcoy B, Hughes C, Shriver MD. Investigating the case of human nose shape and climate adaptation. <span><span class="ref-journal">PLoS Genet. </span>2017;<span class="ref-vol">13</span>:e1006616–e1006631.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5354252/">PMC free article</a>]</span> [<a href="/pubmed/28301464" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Investigating+the+case+of+human+nose+shape+and+climate+adaptation&amp;author=AA+Zaidi&amp;author=BC+Mattern&amp;author=P+Claes&amp;author=B+McEcoy&amp;author=C+Hughes&amp;volume=13&amp;publication_year=2017&amp;pages=e1006616-e1006631&amp;pmid=28301464&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R89"><span class="element-citation">Zaitlen N, Kraft P, Patterson N, Pasaniuc B, Bhatia G, Pollack S, Price AL. Using Extended Genealogy to Estimate Components of Heritability for 23 Quantitative and Dichotomous Traits. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>:e1003520–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3667752/">PMC free article</a>]</span> [<a href="/pubmed/23737753" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Using+Extended+Genealogy+to+Estimate+Components+of+Heritability+for+23+Quantitative+and+Dichotomous+Traits&amp;author=N+Zaitlen&amp;author=P+Kraft&amp;author=N+Patterson&amp;author=B+Pasaniuc&amp;author=G+Bhatia&amp;volume=9&amp;publication_year=2013&amp;pages=e1003520-11&amp;pmid=23737753&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R90"><span class="element-citation">Zaitlen N, Pasaniuc B, Sankararaman S, Bhatia G, Zhang J, Gusev A, Young T, Tandon A, Pollack S, Vilhjálmsson BJ, et al.  Leveraging population admixture to characterize the heritability of complex traits. <span><span class="ref-journal">Nature Publishing Group. </span>2014;<span class="ref-vol">46</span>:1356–1362.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4244251/">PMC free article</a>]</span> [<a href="/pubmed/25383972" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Publishing+Group&amp;title=Leveraging+population+admixture+to+characterize+the+heritability+of+complex+traits&amp;author=N+Zaitlen&amp;author=B+Pasaniuc&amp;author=S+Sankararaman&amp;author=G+Bhatia&amp;author=J+Zhang&amp;volume=46&amp;publication_year=2014&amp;pages=1356-1362&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5884124&amp;issue-id=309717&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#Abs1title">Summary</a></li><li><a href="#Abs2title">eTOC</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Results</a></li><li><a href="#S9title">Discussion</a></li><li><a href="#S14title">STAR Methods</a></li><li><a href="#S42title">Supplementary Material</a></li><li><a href="#S43title">Acknowledgments</a></li><li><a href="#idm140158316440752title">Footnotes</a></li><li><a href="#idm140158316209776title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5884124/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5884124/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5884124/pdf/nihms921744.pdf">PDF (2.1M)</a></li> | <li><a href="#" data-citationid="PMC5884124" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5884124%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5884124%2F&amp;text=An%20Unexpectedly%20Complex%20Architecture%20for%20Skin%20Pigmentation%20in%20Africans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5884124%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29195075" data-aiid="5884124" data-aid="5884124" data-iid="309717" data-domainid="319" data-domain="nihpa" data-accid="PMC5884124" data-md5="796766c1db5c3f5af4d5dea582ba3fbd"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:44:45-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BD7F6DD5A84C10000000006E00439&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5884124%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5884124/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>