<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5011075/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Trans-ethnic Fine Mapping Highlights Kidney-Function Genes Linked to Salt Sensitivity</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5011075/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Trans-ethnic Fine Mapping Highlights Kidney-Function Genes Linked to Salt Sensitivity"><meta name="citation_authors" content="Anubha Mahajan, Aylin&nbsp;R. Rodan, Thu&nbsp;H. Le, Kyle&nbsp;J. Gaulton, Jeffrey Haessler, Adrienne&nbsp;M. Stilp, Yoichiro Kamatani, Gu Zhu, Tamar Sofer, Sanjana Puri, Jeffrey&nbsp;N. Schellinger, Pei-Lun Chu, Sylvia Cechova, Natalie van&nbsp;Zuydam, the SUMMIT Consortium, the BioBank Japan Project, Johan Arnlov, Michael&nbsp;F. Flessner, Vilmantas Giedraitis, Andrew&nbsp;C. Heath, Michiaki Kubo, Anders Larsson, Cecilia&nbsp;M. Lindgren, Pamela&nbsp;A.F. Madden, Grant&nbsp;W. Montgomery, George&nbsp;J. Papanicolaou, Alex&nbsp;P. Reiner, Johan Sundström, Timothy&nbsp;A. Thornton, Lars Lind, Erik Ingelsson, Jianwen Cai, Nicholas&nbsp;G. Martin, Charles Kooperberg, Koichi Matsuda, John&nbsp;B. Whitfield, Yukinori Okada, Cathy&nbsp;C. Laurie, Andrew&nbsp;P. Morris, Nora Franceschini"><meta name="citation_date" content="1 September 2016"><meta name="citation_issue" content="3"><meta name="citation_volume" content="99"><meta name="citation_firstpage" content="636"><meta name="citation_doi" content="10.1016/j.ajhg.2016.07.012"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5011075/?report=abstract"><meta name="citation_pmid" content="27588450"><meta name="DC.Title" content="Trans-ethnic Fine Mapping Highlights Kidney-Function Genes Linked to Salt Sensitivity"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Anubha Mahajan"><meta name="DC.Contributor" content="Aylin&nbsp;R. Rodan"><meta name="DC.Contributor" content="Thu&nbsp;H. Le"><meta name="DC.Contributor" content="Kyle&nbsp;J. Gaulton"><meta name="DC.Contributor" content="Jeffrey Haessler"><meta name="DC.Contributor" content="Adrienne&nbsp;M. Stilp"><meta name="DC.Contributor" content="Yoichiro Kamatani"><meta name="DC.Contributor" content="Gu Zhu"><meta name="DC.Contributor" content="Tamar Sofer"><meta name="DC.Contributor" content="Sanjana Puri"><meta name="DC.Contributor" content="Jeffrey&nbsp;N. Schellinger"><meta name="DC.Contributor" content="Pei-Lun Chu"><meta name="DC.Contributor" content="Sylvia Cechova"><meta name="DC.Contributor" content="Natalie van&nbsp;Zuydam"><meta name="DC.Contributor" content="the SUMMIT Consortium"><meta name="DC.Contributor" content="the BioBank Japan Project"><meta name="DC.Contributor" content="Johan Arnlov"><meta name="DC.Contributor" content="Michael&nbsp;F. Flessner"><meta name="DC.Contributor" content="Vilmantas Giedraitis"><meta name="DC.Contributor" content="Andrew&nbsp;C. Heath"><meta name="DC.Contributor" content="Michiaki Kubo"><meta name="DC.Contributor" content="Anders Larsson"><meta name="DC.Contributor" content="Cecilia&nbsp;M. Lindgren"><meta name="DC.Contributor" content="Pamela&nbsp;A.F. Madden"><meta name="DC.Contributor" content="Grant&nbsp;W. Montgomery"><meta name="DC.Contributor" content="George&nbsp;J. Papanicolaou"><meta name="DC.Contributor" content="Alex&nbsp;P. Reiner"><meta name="DC.Contributor" content="Johan Sundström"><meta name="DC.Contributor" content="Timothy&nbsp;A. Thornton"><meta name="DC.Contributor" content="Lars Lind"><meta name="DC.Contributor" content="Erik Ingelsson"><meta name="DC.Contributor" content="Jianwen Cai"><meta name="DC.Contributor" content="Nicholas&nbsp;G. Martin"><meta name="DC.Contributor" content="Charles Kooperberg"><meta name="DC.Contributor" content="Koichi Matsuda"><meta name="DC.Contributor" content="John&nbsp;B. Whitfield"><meta name="DC.Contributor" content="Yukinori Okada"><meta name="DC.Contributor" content="Cathy&nbsp;C. Laurie"><meta name="DC.Contributor" content="Andrew&nbsp;P. Morris"><meta name="DC.Contributor" content="Nora Franceschini"><meta name="DC.Date" content="2016 Sep 1"><meta name="DC.Identifier" content="10.1016/j.ajhg.2016.07.012"><meta name="DC.Language" content="en"><meta property="og:title" content="Trans-ethnic Fine Mapping Highlights Kidney-Function Genes Linked to Salt Sensitivity"><meta property="og:type" content="article"><meta property="og:description" content="We analyzed genome-wide association studies (GWASs), including data from 71,638 individuals from four ancestries, for estimated glomerular filtration rate (eGFR), a measure of kidney function used to define chronic kidney disease (CKD). We identified ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5011075/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5011075/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5011075/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C2132DD58CE8100000000096F0588.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5011075%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/275125/">v.99(3); 2016 Sep 1</a></li><li class="accid">PMC5011075</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2016 Sep 1; 99(3): 636–646. </span></div><div><span class="fm-vol-iss-date">Published online 2016 Sep 1. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2016.07.012" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2016.07.012</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5011075</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27588450">27588450</a></div></div></div></div><h1 class="content-title">Trans-ethnic Fine Mapping Highlights Kidney-Function Genes Linked to Salt Sensitivity</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Mahajan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715216284784" co-class="co-affbox">Anubha Mahajan</a>,<sup>1</sup> <a href="/pubmed/?term=Rodan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715188834112" co-class="co-affbox">Aylin&nbsp;R. Rodan</a>,<sup>2</sup> <a href="/pubmed/?term=Le%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715185961888" co-class="co-affbox">Thu&nbsp;H. Le</a>,<sup>3</sup> <a href="/pubmed/?term=Gaulton%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715179845152" co-class="co-affbox">Kyle&nbsp;J. Gaulton</a>,<sup>4</sup> <a href="/pubmed/?term=Haessler%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715209471296" co-class="co-affbox">Jeffrey Haessler</a>,<sup>5</sup> <a href="/pubmed/?term=Stilp%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715216451664" co-class="co-affbox">Adrienne&nbsp;M. Stilp</a>,<sup>6</sup> <a href="/pubmed/?term=Kamatani%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715189071776" co-class="co-affbox">Yoichiro Kamatani</a>,<sup>7</sup> <a href="/pubmed/?term=Zhu%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715216433712" co-class="co-affbox">Gu Zhu</a>,<sup>8</sup> <a href="/pubmed/?term=Sofer%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715216431728" co-class="co-affbox">Tamar Sofer</a>,<sup>6</sup> <a href="/pubmed/?term=Puri%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715208488112" co-class="co-affbox">Sanjana Puri</a>,<sup>2</sup> <a href="/pubmed/?term=Schellinger%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715285317520" co-class="co-affbox">Jeffrey&nbsp;N. Schellinger</a>,<sup>2</sup> <a href="/pubmed/?term=Chu%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715212477680" co-class="co-affbox">Pei-Lun Chu</a>,<sup>3</sup> <a href="/pubmed/?term=Cechova%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715179344032" co-class="co-affbox">Sylvia Cechova</a>,<sup>3</sup> <a href="/pubmed/?term=van%26%23x000a0%3BZuydam%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715186924160" co-class="co-affbox">Natalie van&nbsp;Zuydam</a>,<sup>1</sup> the SUMMIT Consortium, the BioBank Japan Project, <a href="/pubmed/?term=Arnlov%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715216537024" co-class="co-affbox">Johan Arnlov</a>,<sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Flessner%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715189051552" co-class="co-affbox">Michael&nbsp;F. Flessner</a>,<sup>11</sup> <a href="/pubmed/?term=Giedraitis%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715211202224" co-class="co-affbox">Vilmantas Giedraitis</a>,<sup>12</sup> <a href="/pubmed/?term=Heath%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715210071808" co-class="co-affbox">Andrew&nbsp;C. Heath</a>,<sup>13</sup> <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715214444864" co-class="co-affbox">Michiaki Kubo</a>,<sup>14</sup> <a href="/pubmed/?term=Larsson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715214442880" co-class="co-affbox">Anders Larsson</a>,<sup>9</sup> <a href="/pubmed/?term=Lindgren%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715216101920" co-class="co-affbox">Cecilia&nbsp;M. Lindgren</a>,<sup>1,</sup><sup>15</sup> <a href="/pubmed/?term=Madden%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715202905376" co-class="co-affbox">Pamela&nbsp;A.F. Madden</a>,<sup>13</sup> <a href="/pubmed/?term=Montgomery%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715202587328" co-class="co-affbox">Grant&nbsp;W. Montgomery</a>,<sup>16</sup> <a href="/pubmed/?term=Papanicolaou%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715186965008" co-class="co-affbox">George&nbsp;J. Papanicolaou</a>,<sup>17</sup> <a href="/pubmed/?term=Reiner%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715188449184" co-class="co-affbox">Alex&nbsp;P. Reiner</a>,<sup>5</sup> <a href="/pubmed/?term=Sundstr%26%23x000f6%3Bm%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715203540656" co-class="co-affbox">Johan Sundström</a>,<sup>9</sup> <a href="/pubmed/?term=Thornton%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715203538672" co-class="co-affbox">Timothy&nbsp;A. Thornton</a>,<sup>6</sup> <a href="/pubmed/?term=Lind%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715207841248" co-class="co-affbox">Lars Lind</a>,<sup>9</sup> <a href="/pubmed/?term=Ingelsson%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715210659408" co-class="co-affbox">Erik Ingelsson</a>,<sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=Cai%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715197506496" co-class="co-affbox">Jianwen Cai</a>,<sup>20</sup> <a href="/pubmed/?term=Martin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715217205712" co-class="co-affbox">Nicholas&nbsp;G. Martin</a>,<sup>8</sup> <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715213108336" co-class="co-affbox">Charles Kooperberg</a>,<sup>5</sup> <a href="/pubmed/?term=Matsuda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715213106352" co-class="co-affbox">Koichi Matsuda</a>,<sup>21</sup> <a href="/pubmed/?term=Whitfield%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715213344768" co-class="co-affbox">John&nbsp;B. Whitfield</a>,<sup>8</sup> <a href="/pubmed/?term=Okada%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715210302128" co-class="co-affbox">Yukinori Okada</a>,<sup>7,</sup><sup>22</sup> <a href="/pubmed/?term=Laurie%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715213556080" co-class="co-affbox">Cathy&nbsp;C. Laurie</a>,<sup>6</sup> <a href="/pubmed/?term=Morris%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715187578528" co-class="co-affbox">Andrew&nbsp;P. Morris</a>,<sup>1,</sup><sup>23,</sup><sup>25,</sup><sup>∗∗</sup> and  <a href="/pubmed/?term=Franceschini%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450" class="affpopup" co-rid="_co_idm140715216248832" co-class="co-affbox">Nora Franceschini</a><sup>24,</sup><sup>25,</sup><sup>∗</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140715216284784"><h3 class="no_margin">Anubha Mahajan</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford OX3 7BN, UK</p><div>Find articles by <a href="/pubmed/?term=Mahajan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Anubha Mahajan</a></div></div><div id="_co_idm140715188834112"><h3 class="no_margin">Aylin&nbsp;R. Rodan</h3><p><sup>2</sup>Department of Internal Medicine, University of Texas Southwestern Medical Center, Dallas, TX 75229, USA</p><div>Find articles by <a href="/pubmed/?term=Rodan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Aylin&nbsp;R. Rodan</a></div></div><div id="_co_idm140715185961888"><h3 class="no_margin">Thu&nbsp;H. Le</h3><p><sup>3</sup>Department of Medicine, University of Virginia, Charlottesville, VA 22908, USA</p><div>Find articles by <a href="/pubmed/?term=Le%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Thu&nbsp;H. Le</a></div></div><div id="_co_idm140715179845152"><h3 class="no_margin">Kyle&nbsp;J. Gaulton</h3><p><sup>4</sup>Department of Pediatrics, University of California San Diego, La Jolla, CA 92093, USA</p><div>Find articles by <a href="/pubmed/?term=Gaulton%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Kyle&nbsp;J. Gaulton</a></div></div><div id="_co_idm140715209471296"><h3 class="no_margin">Jeffrey Haessler</h3><p><sup>5</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</p><div>Find articles by <a href="/pubmed/?term=Haessler%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Jeffrey Haessler</a></div></div><div id="_co_idm140715216451664"><h3 class="no_margin">Adrienne&nbsp;M. Stilp</h3><p><sup>6</sup>Department of Biostatistics, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Stilp%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Adrienne&nbsp;M. Stilp</a></div></div><div id="_co_idm140715189071776"><h3 class="no_margin">Yoichiro Kamatani</h3><p><sup>7</sup>Laboratory for Statistical Analysis, RIKEN Center for Integrative Medical Sciences, Yokohama 230-0045, Japan</p><div>Find articles by <a href="/pubmed/?term=Kamatani%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Yoichiro Kamatani</a></div></div><div id="_co_idm140715216433712"><h3 class="no_margin">Gu Zhu</h3><p><sup>8</sup>Genetic Epidemiology Laboratory, QIMR Berghofer Medical Research Institute, Brisbane 4006, Australia</p><div>Find articles by <a href="/pubmed/?term=Zhu%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Gu Zhu</a></div></div><div id="_co_idm140715216431728"><h3 class="no_margin">Tamar Sofer</h3><p><sup>6</sup>Department of Biostatistics, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Sofer%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Tamar Sofer</a></div></div><div id="_co_idm140715208488112"><h3 class="no_margin">Sanjana Puri</h3><p><sup>2</sup>Department of Internal Medicine, University of Texas Southwestern Medical Center, Dallas, TX 75229, USA</p><div>Find articles by <a href="/pubmed/?term=Puri%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Sanjana Puri</a></div></div><div id="_co_idm140715285317520"><h3 class="no_margin">Jeffrey&nbsp;N. Schellinger</h3><p><sup>2</sup>Department of Internal Medicine, University of Texas Southwestern Medical Center, Dallas, TX 75229, USA</p><div>Find articles by <a href="/pubmed/?term=Schellinger%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Jeffrey&nbsp;N. Schellinger</a></div></div><div id="_co_idm140715212477680"><h3 class="no_margin">Pei-Lun Chu</h3><p><sup>3</sup>Department of Medicine, University of Virginia, Charlottesville, VA 22908, USA</p><div>Find articles by <a href="/pubmed/?term=Chu%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Pei-Lun Chu</a></div></div><div id="_co_idm140715179344032"><h3 class="no_margin">Sylvia Cechova</h3><p><sup>3</sup>Department of Medicine, University of Virginia, Charlottesville, VA 22908, USA</p><div>Find articles by <a href="/pubmed/?term=Cechova%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Sylvia Cechova</a></div></div><div id="_co_idm140715186924160"><h3 class="no_margin">Natalie van&nbsp;Zuydam</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford OX3 7BN, UK</p><div>Find articles by <a href="/pubmed/?term=van%26%23x000a0%3BZuydam%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Natalie van&nbsp;Zuydam</a></div></div><div id="_co_idm140715216537024"><h3 class="no_margin">Johan Arnlov</h3><p><sup>9</sup>Department of Medical Sciences, Cardiovascular Epidemiology, Uppsala University, Uppsala 751 85, Sweden</p><p><sup>10</sup>School of Health and Social Studies, Dalarna University, Falun 791 88, Sweden</p><div>Find articles by <a href="/pubmed/?term=Arnlov%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Johan Arnlov</a></div></div><div id="_co_idm140715189051552"><h3 class="no_margin">Michael&nbsp;F. Flessner</h3><p><sup>11</sup>National Institute of Diabetes, Digestive, and Kidney Disease, NIH, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Flessner%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Michael&nbsp;F. Flessner</a></div></div><div id="_co_idm140715211202224"><h3 class="no_margin">Vilmantas Giedraitis</h3><p><sup>12</sup>Department of Public Health and Caring Sciences, Molecular Geriatrics, Uppsala University, Uppsala 752 37, Sweden</p><div>Find articles by <a href="/pubmed/?term=Giedraitis%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Vilmantas Giedraitis</a></div></div><div id="_co_idm140715210071808"><h3 class="no_margin">Andrew&nbsp;C. Heath</h3><p><sup>13</sup>Department of Psychiatry, Washington University in St. Louis, St. Louis, MO 63110, USA</p><div>Find articles by <a href="/pubmed/?term=Heath%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Andrew&nbsp;C. Heath</a></div></div><div id="_co_idm140715214444864"><h3 class="no_margin">Michiaki Kubo</h3><p><sup>14</sup>Laboratory for Genotyping Development, RIKEN Center for Integrative Medical Sciences, Yokohama 230-0045, Japan</p><div>Find articles by <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Michiaki Kubo</a></div></div><div id="_co_idm140715214442880"><h3 class="no_margin">Anders Larsson</h3><p><sup>9</sup>Department of Medical Sciences, Cardiovascular Epidemiology, Uppsala University, Uppsala 751 85, Sweden</p><div>Find articles by <a href="/pubmed/?term=Larsson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Anders Larsson</a></div></div><div id="_co_idm140715216101920"><h3 class="no_margin">Cecilia&nbsp;M. Lindgren</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford OX3 7BN, UK</p><p><sup>15</sup>Big Data Institute, Li Ka Shing Centre for Health Information and Discovery, University of Oxford, Oxford OX3 7BN, UK</p><div>Find articles by <a href="/pubmed/?term=Lindgren%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Cecilia&nbsp;M. Lindgren</a></div></div><div id="_co_idm140715202905376"><h3 class="no_margin">Pamela&nbsp;A.F. Madden</h3><p><sup>13</sup>Department of Psychiatry, Washington University in St. Louis, St. Louis, MO 63110, USA</p><div>Find articles by <a href="/pubmed/?term=Madden%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Pamela&nbsp;A.F. Madden</a></div></div><div id="_co_idm140715202587328"><h3 class="no_margin">Grant&nbsp;W. Montgomery</h3><p><sup>16</sup>Molecular Epidemiology Laboratory, QIMR Berghofer Medical Research Institute, Brisbane 4006, Australia</p><div>Find articles by <a href="/pubmed/?term=Montgomery%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Grant&nbsp;W. Montgomery</a></div></div><div id="_co_idm140715186965008"><h3 class="no_margin">George&nbsp;J. Papanicolaou</h3><p><sup>17</sup>Epidemiology Branch, Division of Cardiovascular Sciences, National Heart, Lung and Blood Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Papanicolaou%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">George&nbsp;J. Papanicolaou</a></div></div><div id="_co_idm140715188449184"><h3 class="no_margin">Alex&nbsp;P. Reiner</h3><p><sup>5</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</p><div>Find articles by <a href="/pubmed/?term=Reiner%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Alex&nbsp;P. Reiner</a></div></div><div id="_co_idm140715203540656"><h3 class="no_margin">Johan Sundström</h3><p><sup>9</sup>Department of Medical Sciences, Cardiovascular Epidemiology, Uppsala University, Uppsala 751 85, Sweden</p><div>Find articles by <a href="/pubmed/?term=Sundstr%26%23x000f6%3Bm%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Johan Sundström</a></div></div><div id="_co_idm140715203538672"><h3 class="no_margin">Timothy&nbsp;A. Thornton</h3><p><sup>6</sup>Department of Biostatistics, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Thornton%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Timothy&nbsp;A. Thornton</a></div></div><div id="_co_idm140715207841248"><h3 class="no_margin">Lars Lind</h3><p><sup>9</sup>Department of Medical Sciences, Cardiovascular Epidemiology, Uppsala University, Uppsala 751 85, Sweden</p><div>Find articles by <a href="/pubmed/?term=Lind%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Lars Lind</a></div></div><div id="_co_idm140715210659408"><h3 class="no_margin">Erik Ingelsson</h3><p><sup>18</sup>Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory, Uppsala University, Uppsala 752 37, Sweden</p><p><sup>19</sup>Department of Medicine, Division of Cardiovascular Medicine, Stanford University School of Medicine, Stanford, CA 94305, USA</p><div>Find articles by <a href="/pubmed/?term=Ingelsson%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Erik Ingelsson</a></div></div><div id="_co_idm140715197506496"><h3 class="no_margin">Jianwen Cai</h3><p><sup>20</sup>Collaborative Studies Coordinating Center, Department of Biostatistics, University of North&nbsp;Carolina at Chapel Hill, Chapel Hill, NC 27599, USA</p><div>Find articles by <a href="/pubmed/?term=Cai%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Jianwen Cai</a></div></div><div id="_co_idm140715217205712"><h3 class="no_margin">Nicholas&nbsp;G. Martin</h3><p><sup>8</sup>Genetic Epidemiology Laboratory, QIMR Berghofer Medical Research Institute, Brisbane 4006, Australia</p><div>Find articles by <a href="/pubmed/?term=Martin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Nicholas&nbsp;G. Martin</a></div></div><div id="_co_idm140715213108336"><h3 class="no_margin">Charles Kooperberg</h3><p><sup>5</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</p><div>Find articles by <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Charles Kooperberg</a></div></div><div id="_co_idm140715213106352"><h3 class="no_margin">Koichi Matsuda</h3><p><sup>21</sup>Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, University of Tokyo, Tokyo 108-8639, Japan</p><div>Find articles by <a href="/pubmed/?term=Matsuda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Koichi Matsuda</a></div></div><div id="_co_idm140715213344768"><h3 class="no_margin">John&nbsp;B. Whitfield</h3><p><sup>8</sup>Genetic Epidemiology Laboratory, QIMR Berghofer Medical Research Institute, Brisbane 4006, Australia</p><div>Find articles by <a href="/pubmed/?term=Whitfield%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">John&nbsp;B. Whitfield</a></div></div><div id="_co_idm140715210302128"><h3 class="no_margin">Yukinori Okada</h3><p><sup>7</sup>Laboratory for Statistical Analysis, RIKEN Center for Integrative Medical Sciences, Yokohama 230-0045, Japan</p><p><sup>22</sup>Department of Statistical Genetics, Osaka University Graduate School of Medicine, Osaka 565-0871, Japan</p><div>Find articles by <a href="/pubmed/?term=Okada%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Yukinori Okada</a></div></div><div id="_co_idm140715213556080"><h3 class="no_margin">Cathy&nbsp;C. Laurie</h3><p><sup>6</sup>Department of Biostatistics, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Laurie%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Cathy&nbsp;C. Laurie</a></div></div><div id="_co_idm140715187578528"><h3 class="no_margin">Andrew&nbsp;P. Morris</h3><p><sup>1</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford OX3 7BN, UK</p><p><sup>23</sup>Department of Biostatistics, University of Liverpool, Liverpool L69 3GL, UK</p><div>Find articles by <a href="/pubmed/?term=Morris%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Andrew&nbsp;P. Morris</a></div></div><div id="_co_idm140715216248832"><h3 class="no_margin">Nora Franceschini</h3><p><sup>24</sup>Department of Epidemiology, University of North Carolina, Chapel Hill, NC 27514, USA</p><div>Find articles by <a href="/pubmed/?term=Franceschini%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27588450">Nora Franceschini</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140715281540864_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140715281540864_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140715281540864_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140715281540864_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford OX3 7BN, UK</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Internal Medicine, University of Texas Southwestern Medical Center, Dallas, TX 75229, USA</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Department of Medicine, University of Virginia, Charlottesville, VA 22908, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Department of Pediatrics, University of California San Diego, La Jolla, CA 92093, USA</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Department of Biostatistics, University of Washington, Seattle, WA 98195, USA</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Laboratory for Statistical Analysis, RIKEN Center for Integrative Medical Sciences, Yokohama 230-0045, Japan</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Genetic Epidemiology Laboratory, QIMR Berghofer Medical Research Institute, Brisbane 4006, Australia</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Department of Medical Sciences, Cardiovascular Epidemiology, Uppsala University, Uppsala 751 85, Sweden</div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>School of Health and Social Studies, Dalarna University, Falun 791 88, Sweden</div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>National Institute of Diabetes, Digestive, and Kidney Disease, NIH, Bethesda, MD 20892, USA</div><div class="fm-affl" id="aff12" lang="en"><sup>12</sup>Department of Public Health and Caring Sciences, Molecular Geriatrics, Uppsala University, Uppsala 752 37, Sweden</div><div class="fm-affl" id="aff13" lang="en"><sup>13</sup>Department of Psychiatry, Washington University in St. Louis, St. Louis, MO 63110, USA</div><div class="fm-affl" id="aff14" lang="en"><sup>14</sup>Laboratory for Genotyping Development, RIKEN Center for Integrative Medical Sciences, Yokohama 230-0045, Japan</div><div class="fm-affl" id="aff15" lang="en"><sup>15</sup>Big Data Institute, Li Ka Shing Centre for Health Information and Discovery, University of Oxford, Oxford OX3 7BN, UK</div><div class="fm-affl" id="aff16" lang="en"><sup>16</sup>Molecular Epidemiology Laboratory, QIMR Berghofer Medical Research Institute, Brisbane 4006, Australia</div><div class="fm-affl" id="aff17" lang="en"><sup>17</sup>Epidemiology Branch, Division of Cardiovascular Sciences, National Heart, Lung and Blood Institute, Bethesda, MD 20892, USA</div><div class="fm-affl" id="aff18" lang="en"><sup>18</sup>Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory, Uppsala University, Uppsala 752 37, Sweden</div><div class="fm-affl" id="aff19" lang="en"><sup>19</sup>Department of Medicine, Division of Cardiovascular Medicine, Stanford University School of Medicine, Stanford, CA 94305, USA</div><div class="fm-affl" id="aff20" lang="en"><sup>20</sup>Collaborative Studies Coordinating Center, Department of Biostatistics, University of North&nbsp;Carolina at Chapel Hill, Chapel Hill, NC 27599, USA</div><div class="fm-affl" id="aff21" lang="en"><sup>21</sup>Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, University of Tokyo, Tokyo 108-8639, Japan</div><div class="fm-affl" id="aff22" lang="en"><sup>22</sup>Department of Statistical Genetics, Osaka University Graduate School of Medicine, Osaka 565-0871, Japan</div><div class="fm-affl" id="aff23" lang="en"><sup>23</sup>Department of Biostatistics, University of Liverpool, Liverpool L69 3GL, UK</div><div class="fm-affl" id="aff24" lang="en"><sup>24</sup>Department of Epidemiology, University of North Carolina, Chapel Hill, NC 27514, USA</div><div><span class="contrib-email" id="idm140715187578528">Andrew&nbsp;P. Morris: <a href="mailto:dev@null" data-email="ku.ca.looprevil@sirrompa" class="oemail">ku.ca.looprevil@sirrompa</a>; </span> <span class="contrib-email" id="idm140715216248832">Nora Franceschini: <a href="mailto:dev@null" data-email="ude.cnu@faron" class="oemail">ude.cnu@faron</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ude.cnu@faron" class="oemail">ude.cnu@faron</a></div><div id="cor2"><sup>∗∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ku.ca.looprevil@sirrompa" class="oemail">ku.ca.looprevil@sirrompa</a></div><div id="fn1"><sup>25</sup>These authors contributed equally to this work</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140715281540864_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 Feb 17; Accepted 2016 Jul 8.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140715281540864_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2016 The Author(s)</div><div class="license half_rhythm">This is an open access article under the CC BY license (http://creativecommons.org/licenses/by/4.0/).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5011075/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Figures S1–S5, Tables S1–S14, and Supplemental Acknowledgments</div><div><a href="/pmc/articles/PMC5011075/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (2.2M)</span></div><div class="small guid">GUID:&nbsp;FC39169B-F6F3-49EE-BE52-20E40550D7A2</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S2. Article plus Supplemental Data</div><div><a href="/pmc/articles/PMC5011075/bin/mmc2.pdf" data-ga-action="click_feat_suppl">mmc2.pdf</a><span style="color:gray"> (3.0M)</span></div><div class="small guid">GUID:&nbsp;B839FBDC-A4E6-4574-B92B-F2107581C8C0</div></div></dd></dl></div><div id="abs0010" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="abs0010title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">We analyzed genome-wide association studies (GWASs), including data from 71,638 individuals from four ancestries, for estimated glomerular filtration rate (eGFR), a measure of kidney function used to define chronic kidney disease (CKD). We identified 20 loci attaining genome-wide-significant evidence of association (p &lt; 5&nbsp;× 10<sup>−8</sup>) with kidney function and highlighted that allelic effects on eGFR at lead SNPs are homogeneous across ancestries. We leveraged differences in the pattern of linkage disequilibrium between diverse populations to fine-map the 20 loci through construction of “credible sets” of variants driving eGFR association signals. Credible variants at the 20 eGFR loci were enriched for DNase I hypersensitivity sites (DHSs) in human kidney cells. DHS credible variants were expression quantitative trait loci for <em>NFATC1</em> and <em>RGS14</em> (at the <em>SLC34A1</em> locus) in multiple tissues. Loss-of-function mutations in ancestral orthologs of both genes in <em>Drosophila melanogaster</em> were associated with altered sensitivity to salt stress. Renal mRNA expression of <em>Nfatc1</em> and <em>Rgs14</em> in a salt-sensitive mouse model was also reduced after exposure to a high-salt diet or induced CKD. Our study (1) demonstrates the utility of trans-ethnic fine mapping through integration of GWASs involving diverse populations with genomic annotation from relevant tissues to define molecular mechanisms by which association signals exert their effect and (2) suggests that salt sensitivity might be an important marker for biological processes that affect kidney function and CKD in humans.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Introduction</h2><p id="__p2" class="p p-first">Chronic kidney disease (CKD) is a major public health burden and affects nearly 10% of the global population.<sup><a href="#bib1" rid="bib1" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">1</a></sup> Reduced estimated glomerular filtration rate (eGFR), a measure of kidney function used to define CKD, is associated with premature cardiovascular disease and mortality, acute kidney injury, and progression to end stage renal disease (ESRD).<sup><a href="#bib2" rid="bib2" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">2</a></sup> Although individuals of African and Hispanic descent suffer the largest burden of CKD,<sup><a href="#bib3" rid="bib3" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">3</a></sup> the largest genome-wide association studies (GWASs) to search for kidney-function loci have been undertaken in populations of European and East Asian ancestry.<sup><a href="#bib4" rid="bib4" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">4</a></sup><sup>, </sup><sup><a href="#bib5" rid="bib5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a></sup><sup>, </sup><sup><a href="#bib6" rid="bib6" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">6</a></sup><sup>, </sup><sup><a href="#bib7" rid="bib7" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">7</a></sup><sup>, </sup><sup><a href="#bib8" rid="bib8" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">8</a></sup> Many of these loci are characterized by common variant association signals that map to large genomic intervals, which contain many possible causal genes for eGFR, thereby limiting understanding of the downstream pathogenesis of CKD.</p><p id="__p3" class="p p-last">To address this challenge, we have undertaken a trans-ethnic meta-analysis of nine GWASs comprising 71,638 individuals from four ancestries (African American, Hispanic, European, and East Asian), each imputed up to the&nbsp;phase&nbsp;1 integrated (March 2012 release) multi-ethnic&nbsp;reference panel from the 1000 Genomes Project<sup><a href="#bib9" rid="bib9" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">9</a></sup>, from the Continental Origins and Genetic Epidemiology Network (COGENT)-Kidney consortium. With these data, we aimed to (1) assess the evidence for heterogeneity in allelic effects on eGFR for lead SNPs at kidney-function loci across ethnic groups; (2) fine-map these loci by taking advantage of high-density imputation and by leveraging differences in the pattern of linkage disequilibrium (LD) between diverse populations to localize “credible sets” of variants driving eGFR association signals; (3) define potential molecular mechanisms through which eGFR association signals at these loci impact kidney function through overlap of credible variants with genomic annotation; and (4) assess possible markers for biological processes that impact kidney function and CKD in humans through targeted experimentation in model organisms.</p></div><div id="sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec2title">Subjects and Methods</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">Ethics Statement</h3><p id="__p4" class="p p-first-last">All human research was approved by the relevant institutional review boards and conducted according to the Declaration of Helsinki. All participants provided written informed consent.</p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">Study Overview</h3><p id="__p5" class="p p-first-last">We aggregated five GWASs of individuals of European ancestry (23,553 individuals from Europe, the USA, and Australia), two GWASs of Hispanic Americans (16,325 individuals from the USA), one GWAS of individuals of East Asian ancestry (23,536 individuals from Japan), and one GWAS of African Americans (8,224 individuals from the USA). Study sample characteristics are presented in <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S1</a>.</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">Genotyping, Quality Control, and Imputation</h3><p id="__p6" class="p p-first">Samples were genotyped with a variety of GWAS arrays, and quality control was undertaken within each study (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S2</a>). Sample quality control included exclusions on the basis of genome-wide call rate, extreme heterozygosity, sex discordance, cryptic relatedness, and outlying ethnicity. SNP quality control included exclusions on the basis of call rate across samples and extreme deviation from Hardy-Weinberg equilibrium. Non-autosomal SNPs were excluded from imputation and association analysis.</p><p id="__p7" class="p p-last">Within each study, the autosomal GWAS genotype scaffold was&nbsp;first pre-phased<sup><a href="#bib10" rid="bib10" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">10</a></sup><sup>, </sup><sup><a href="#bib11" rid="bib11" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">11</a></sup> with genetic maps from the International HapMap Consortium<sup><a href="#bib12" rid="bib12" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">12</a></sup> to model recombination rates. The scaffold was then imputed up to the phase 1 integrated (March 2012 release) multi-ethnic reference panel from the 1000 Genomes Project<sup><a href="#bib9" rid="bib9" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">9</a></sup> via&nbsp;IMPUTE2<sup><a href="#bib11" rid="bib11" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">11</a></sup><sup>, </sup><sup><a href="#bib13" rid="bib13" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">13</a></sup> or MaCH/Minimac<sup><a href="#bib11" rid="bib11" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">11</a></sup> (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S2</a>). Imputed variants were retained for downstream association analyses if they attained&nbsp;established GWAS quality control thresholds:<sup><a href="#bib14" rid="bib14" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">14</a></sup> IMPUTE2 info ≥ 0.4 or MaCH/Minimac <em>r</em><sup>2</sup> ≥ 0.3.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">Calculation of eGFR and Association Analysis</h3><p id="__p8" class="p p-first-last">Within each study, eGFR was calculated from serum creatinine (mg/dL), with adjustment for age, sex, and ethnicity by means of the four-variable MDRD (modification of diet in renal disease) equation<sup><a href="#bib15" rid="bib15" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">15</a></sup> to be comparable with published GWASs of kidney function.<sup><a href="#bib4" rid="bib4" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">4</a></sup><sup>, </sup><sup><a href="#bib5" rid="bib5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a></sup><sup>, </sup><sup><a href="#bib6" rid="bib6" class=" bibr popnode">6</a></sup><sup>, </sup><sup><a href="#bib7" rid="bib7" class=" bibr popnode">7</a></sup><sup>, </sup><sup><a href="#bib8" rid="bib8" class=" bibr popnode">8</a></sup> Within each study, we tested association of eGFR with each variant passing quality control in a linear regression framework under an additive dosage model and with adjustment for study-specific covariates to account for confounding due to population structure (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S2</a>). Association summary statistics were subsequently corrected in each study for residual population structure through a first round of genomic control<sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> where necessary (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S2</a>).</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">Trans-ethnic Meta-analysis</h3><p id="__p9" class="p p-first-last">Association summary statistics were combined across studies via fixed-effects meta-analysis (inverse-variance weighting) implemented in the GWAMA software.<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup> Variants passing quality control in fewer than 50% of the total sample size across studies were excluded from the meta-analysis. Association summary statistics from the meta-analysis were then corrected for a second round of genomic control<sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> (λ<sub>GC</sub> = 1.028). Heterogeneity in allelic effects between studies at each variant was assessed by means of Cochran’s Q statistic.<sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup> We extracted association summary statistics for eGFR from the trans-ethnic meta-analysis for previously reported lead SNPs at established GWAS loci.</p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">LD Calculations</h3><p id="__p10" class="p p-first-last">LD, as measured by the correlation coefficient <em>r</em><sup>2</sup>, was calculated on&nbsp;the basis of haplotypes in each ancestry group from the 1000 Genomes Project<sup><a href="#bib9" rid="bib9" class=" bibr popnode">9</a></sup> via LDlink.<sup><a href="#bib19" rid="bib19" class=" bibr popnode">19</a></sup></p></div><div id="sec2.7" class="sec"><h3 id="sec2.7title">Conditional Analyses</h3><p id="__p11" class="p p-first-last">To assess the evidence for distinct association signals at each locus attaining nominal significance (p<sub>COND</sub> &lt; 10<sup>−5</sup>, Bonferroni correction for ∼5,000 variants per locus) in our trans-ethnic meta-analysis, we performed conditional analysis in a 1 Mb genomic interval flanking the lead SNP. Within each study, we tested association of eGFR with each variant passing quality control in the flanking region in a linear regression framework under an additive dosage model and with adjustment for genotypes at the lead SNP, in addition to other study-specific covariates used in unconditional analysis (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S2</a>). Association summary statistics were subsequently corrected in each study for residual population structure, via the same genomic control<sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> correction employed for unconditional analysis (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S2</a>). These association summary statistics were combined across studies via fixed-effects meta-analysis (inverse-variance weighting) implemented in GWAMA.<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup> Variants passing quality control in less than 50% of the total sample size across studies were excluded from the meta-analysis. Association summary statistics from the conditional meta-analysis were then corrected for a second round of genomic control,<sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> making use of the same adjustment as defined in the unconditional analysis (λ<sub>GC</sub> = 1.028).</p></div><div id="sec2.8" class="sec"><h3 id="sec2.8title">Association with CKD</h3><p id="__p12" class="p p-first-last">We defined CKD by an eGFR &lt; 60&nbsp;mL/min/1.73 m<sup>2</sup> (calculated with the MDRD equation defined above) and/or incidence of ESRD, if available. Any individual who was prospectively initiated on dialysis or received a kidney transplant (self-reported or obtained from medical records or registries) was defined as having ESRD. Individuals who did not develop ESRD at follow-up were considered control subjects. We considered the lead eGFR SNP identified at each locus attaining genome-wide significance in our trans-ethnic meta-analysis. Within each study, we tested association of CKD with each SNP in a logistic regression framework under an additive dosage model and with adjustment for study-specific covariates to account for confounding due to population structure (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S2</a>). Association summary statistics were combined across studies via fixed-effects meta-analysis (sample size and inverse-variance weighting) implemented in METAL<sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup> and GWAMA.<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup></p></div><div id="sec2.9" class="sec"><h3 id="sec2.9title">Association with eGFR in Diabetic Individuals from the SUMMIT Consortium</h3><p id="__p13" class="p p-first">We considered the lead eGFR SNP at each locus attaining genome-wide significance in our trans-ethnic meta-analysis. We performed a look-up of association summary statistics for eGFR in 13,158 subjects with diabetes (9,197 with type 2 diabetes [T2D] and 3,961 with type 1 diabetes [T1D]) from five studies of individuals of European ancestry from the SUMMIT Consortium. Within each study, the outcome variable was defined as the last measured eGFR, calculated with the MDRD equation (defined above). Each study was imputed up to the phase 1 integrated (March 2012 release) multi-ethnic reference panel from the 1000 Genomes Project.<sup><a href="#bib9" rid="bib9" class=" bibr popnode">9</a></sup> Estimated allelic effects on eGFR were obtained from a linear mixed model and implemented in EMMAX<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup> with an empirical genetic relationship matrix, assuming an additive dosage of the minor allele and including sex, age at diabetes onset, and duration of diabetes as covariates. Association summary statistics for eGFR were combined across studies via fixed-effects meta-analysis (inverse-variance weighting) implemented in GWAMA.<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup> Combined allelic effect estimates across studies were reported for T1D and T2D subjects, both separately and for all diabetic individuals combined. Heterogeneity in allelic effects between T1D and T2D subjects at each variant was assessed by means of Cochran’s Q statistic,<sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup> as implemented in GWAMA.<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup></p><p id="__p14" class="p p-last">For lead SNPs, we tested for a difference in the allelic effect on eGFR in the general population (from our trans-ethnic meta-analysis) and in diabetic indivuduals (combined T1D and T2D from the SUMMIT Consortium) by using a two-sample Z-test.</p></div><div id="sec2.10" class="sec"><h3 id="sec2.10title">MANTRA Fine Mapping and Credible Set Construction</h3><p id="__p15" class="p p-first">We performed trans-ethnic fine mapping of each locus in a 1 Mb genomic interval flanking the lead SNP. Association summary statistics for each variant in the flanking region were combined across studies with a Bayesian hybrid of fixed- and random-effects meta-analysis, as implemented in MANTRA.<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup> MANTRA allows for heterogeneity in allelic effects between ancestry groups arising as a result of differences in the structure of LD between diverse populations by assigning studies to clusters according to a Bayesian partition model of relatedness between them, defined by pairwise genome-wide mean allele frequency differences (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S1</a>). MANTRA has been demonstrated, both empirically and by simulation, to improve fine-mapping resolution, as compared to either a fixed- or random-effects meta-analysis.<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup><sup>, </sup><sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup><sup>, </sup><sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup> Variants passing quality control in less than 50% of the total sample size across studies were excluded from the fine-mapping analysis.</p><p id="__p16" class="p p-last">We calculated the posterior probability that the <em>j</em><sup>th</sup> variant, π<sub>C<em>j</em></sub>, is driving the association signal at each locus by</p><div class="disp-formula" id="ufd1"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="M1" overflow="scroll" role="math" style="width: 7.516em; display: inline-block;"><span style="display: inline-block; position: relative; width: 6.198em; height: 0px; font-size: 121%;"><span style="position: absolute; clip: rect(0.587em, 1006.13em, 3.388em, -1000em); top: -2.221em; left: 0em;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="msub" id="MathJax-Span-4"><span style="display: inline-block; position: relative; width: 1.447em; height: 0px;"><span style="position: absolute; clip: rect(3.391em, 1000.57em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math; font-style: italic;">π<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.57em;"><span class="mrow" id="MathJax-Span-6"><span class="mi" id="MathJax-Span-7" style="font-size: 70.7%; font-family: MathJax_Main;">C</span><span class="mi" id="MathJax-Span-8" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-9" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mfrac" id="MathJax-Span-10" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.924em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.106em, 1001.06em, 4.426em, -1000em); top: -4.741em; left: 50%; margin-left: -0.53em;"><span class="mrow" id="MathJax-Span-11"><span class="msub" id="MathJax-Span-12"><span style="display: inline-block; position: relative; width: 1.06em; height: 0px;"><span style="position: absolute; clip: rect(3.106em, 1000.67em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-13" style="font-family: MathJax_Math; font-style: italic;">Λ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.694em;"><span class="mi" id="MathJax-Span-14" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.072em, 1002.8em, 4.425em, -1000em); top: -3.258em; left: 50%; margin-left: -1.402em;"><span class="mrow" id="MathJax-Span-15"><span class="msub" id="MathJax-Span-16"><span style="display: inline-block; position: relative; width: 1.499em; height: 0px;"><span style="position: absolute; clip: rect(3.072em, 1001em, 4.382em, -1000em); top: -3.977em; left: 0em;"><span class="mo" id="MathJax-Span-17" style="font-family: MathJax_Size1; vertical-align: 0em;">∑</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.692em; left: 1.056em;"><span class="mi" id="MathJax-Span-18" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mrow" id="MathJax-Span-19" style="padding-left: 0.167em;"><span class="msub" id="MathJax-Span-20"><span style="display: inline-block; position: relative; width: 1.137em; height: 0px;"><span style="position: absolute; clip: rect(3.106em, 1000.67em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-21" style="font-family: MathJax_Math; font-style: italic;">Λ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.694em;"><span class="mi" id="MathJax-Span-22" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.865em, 1002.92em, 1.24em, -1000em); top: -1.305em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 2.924em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span><span class="mo" id="MathJax-Span-23" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.221em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.287em; border-left: 0px solid; width: 0px; height: 3.14em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" altimg="si1.gif" overflow="scroll"><mrow><msub><mi mathvariant="normal">π</mi><mrow><mi mathvariant="normal">C</mi><mi>j</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mi mathvariant="italic">Λ</mi><mi>j</mi></msub></mrow><mrow><msub><mo>∑</mo><mi>k</mi></msub><mrow><msub><mi mathvariant="italic">Λ</mi><mi>k</mi></msub></mrow></mrow></mfrac><mo>,</mo></mrow></math></script></div></div><p>where the summation is over all variants in the flanking interval. In this expression, <em>Λ</em><sub><em>j</em></sub> is the MANTRA Bayes factor in favor of association from the trans-ethnic meta-analysis. For each distinct association signal, a 99% credible set<sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup> was then constructed by (1) ranking all variants according to their Bayes factor, <em>Λ</em><sub><em>j</em></sub>, and (2) including ranked variants until their cumulative posterior probability exceeds 0.99.</p></div><div id="sec2.11" class="sec"><h3 id="sec2.11title">Genomic Annotation</h3><p id="__p17" class="p p-first-last">For each locus attaining genome-wide significance in our trans-ethnic meta-analysis, we obtained genomic annotations of all single-nucleotide variants in a 1 Mb interval flanking the lead SNP. We utilized the Ensembl Variant Effect Predictor (VEP, version 2.7), based on the Ensembl transcript set (version 69). By default, the VEP reports all possible annotations (transcript- and gene-specific) for each variant. We therefore prioritized annotations by considering the most severe consequence of all those reported. We then calculated the total posterior probability of driving association signals for each consequence across loci.</p></div><div id="sec2.12" class="sec"><h3 id="sec2.12title">Regulatory Annotation</h3><p id="__p18" class="p p-first">We collected genomic annotations from three sources. First, we obtained regulatory chromatin states from the Epigenome Roadmap Project<sup><a href="#bib26" rid="bib26" class=" bibr popnode">26</a></sup> for 93 cell types after removing five cancer cell lines. For each cell type, we pooled enhancer (EnhA and EnhWk) and promoter (TssA and TssFlnk) elements into one annotation. Second, we obtained 145 non-redundant DNase I hypersensitivity sites (DHSs) from the ENCODE Project<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> by retaining only one dataset for cell types with multiple assayed samples. Third, we obtained chromatin immuno-precipitation sequence (ChIP-seq) binding sites for 165 transcription factors: 161 proteins from the ENCODE Project<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> and additional factors assayed in primary pancreatic islets.<sup><a href="#bib28" rid="bib28" class=" bibr popnode">28</a></sup> This resulted in a total of 403 annotations for downstream enrichment analyses. For each annotation, we considered variants passing quality control and mapping within 1 Mb of the lead SNP attaining genome-wide significance in the trans-ethnic meta-analysis.</p><p id="__p19">We first tested the effect of each annotation on the log odds of the posterior probability of driving eGFR association signals in a logistic regression model. For each variant, we encoded overlap with the tested annotation as a binary indicator (1 if variant overlaps annotation, 0 otherwise). The regression model also incorporated binary indicators of genic annotations as covariates, as well as a categorical variable for locus membership. Specifically,</p><div class="disp-formula" id="ufd2"><div class="f">logit(π<sub>C<em>j</em></sub>) =&nbsp;<em>α</em><sub><em>i</em></sub><em>L</em><sub><em>i</em><em>j</em></sub> +&nbsp;<em>β</em><sub><em>k</em></sub><em>x</em><sub><em>j</em><em>k</em></sub> +&nbsp;<em>γ</em><sub>3<sup>′</sup>UTR</sub><em>x</em><sub><em>j</em>3<sup>′</sup>UTR</sub> +&nbsp;<em>γ</em><sub>5<sup>′</sup>UTR</sub><em>x</em><sub><em>j</em>5<sup>′</sup>UTR</sub> +&nbsp;<em>γ</em><sub>EXON</sub><em>x</em><sub><em>j</em>EXON</sub> +&nbsp;<em>γ</em><sub>TSS</sub><em>x</em><sub><em>j</em>TSS</sub>,&nbsp;</div></div><p>where <em>π</em><sub><em>Cj</em></sub> is the posterior probability that the <em>j</em><sup>th</sup> variant drives the eGFR association; <em>α</em><sub><em>i</em></sub> denotes an intercept for the <em>i</em><sup>th</sup> locus and <em>L</em><sub><em>ij</em></sub> is a binary indicator of membership of the <em>j</em><sup>th</sup> variant in the <em>i</em><sup>th</sup> locus; <em>β</em><sub><em>k</em></sub> denotes the effect of the <em>k</em><sup>th</sup> annotation and <em>x</em><sub><em>kj</em></sub> is a binary indicator of overlap of the <em>j</em><sup>th</sup> variant with the <em>k</em><sup>th</sup> annotation; and <em>γ</em><sub>3′</sub>
<sub>UTR</sub>, <em>γ</em><sub>5′</sub>&nbsp;<sub>UTR</sub>, <em>γ</em><sub>EXON</sub> and <em>γ</em><sub>TSS</sub> denote the effects of 3′ UTRs, 5′ UTRs, coding exons, and the region within 1 kb upstream of GENCODE transcription start site (TSS) annotations, respectively, and <em>x</em><sub><em>j</em>3′</sub>
<sub>UTR</sub>, <em>x</em><sub><em>j</em>5′</sub>
<sub>UTR</sub>, <em>x</em><sub><em>j</em>EXON</sub> and <em>x</em><sub><em>j</em>TSS</sub> are binary indicators of overlap of the <em>j</em><sup>th</sup> variant with these annotations. The SE of the effect of the <em>k</em><sup>th</sup> annotation, <em>β</em><sub><em>k</em></sub>, was evaluated with a robust sandwich variance estimator.</p><p id="__p20" class="p p-last">Using fGWAS software, we then tested for the effect of each annotation by using the Bayes factor in favor of association.<sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup> We included coding exons, 3′ UTRs, 5′ UTRs, and the region within 1 kb upstream of the TSS in the model for each annotation. We obtained the estimated effect and 95% confidence interval (CI) from this model and considered an annotation enriched if the 95% CI did not overlap zero.</p></div><div id="sec2.13" class="sec"><h3 id="sec2.13title"><em>Drosophila melanogaster</em> Salt-Sensitivity Assay</h3><p id="__p21" class="p p-first">Four <em>y</em><sup><em>1</em></sup><em>w</em><sup><em>1</em></sup> virgin females were mated with two males each of the genotypes <em>y</em><sup><em>1</em></sup><em>w</em><sup><em>1</em></sup><em>/Y, y</em><sup><em>1</em></sup><em>w</em><sup><em>1</em></sup><em>/Y</em>; <em>loco</em><sup><em>EY-P283</em></sup><em>/TM3 Sb</em> or <em>y</em><sup><em>1</em></sup><em>w</em><sup><em>1</em></sup><em>/Y</em>; <em>loco</em><sup><em>d06164</em></sup> in rearing vials on standard cornmeal/yeast/molasses food (prepared in a central kitchen at University of Texas Southwestern Medical Center). The <em>y</em><sup>1</sup><em>w</em><sup>1</sup> isogenic control, in which all loci had been previously homozygosed, and to which <em>loco</em> mutants had been backcrossed for six generations, were obtained from Dr. Yongkyu Park (Rutgers New Jersey Medical School).<sup><a href="#bib30" rid="bib30" class=" bibr popnode">30</a></sup> Separately, to obtain highly heterozygous progeny (heterogenic), virgin females from the A.R.R. lab’s <em>wBerlin</em> strain were mated with males of genotypes <em>y</em><sup><em>1</em></sup><em>w</em><sup><em>1</em></sup><em>/Y</em>, <em>y</em><sup><em>1</em></sup><em>w</em><sup><em>1</em></sup><em>/Y</em>; <em>loco</em><sup><em>EY-P283</em></sup><em>/TM3 Sb</em> or <em>y</em><sup><em>1</em></sup><em>w</em><sup><em>1</em></sup><em>/Y</em>; <em>loco</em><sup><em>d06164</em></sup>, as above. Adults were cleared from rearing vials on rearing day eight. Ten female progeny from each vial were collected within 1–3&nbsp;days of eclosion and placed on food containing various concentrations of added NaCl. Each experimental vial contained flies from a single rearing vial. The number of dead flies in each vial was counted daily. Flies were transferred to fresh medium after day five, and again after day ten for the heterogenic flies. For each concentration of experimental medium, 225&nbsp;g Applied Scientific Jazz-Mix <em>Drosophila</em> Food (Fisher, cat. no. AS-153) was added to 500&nbsp;mL deionized water with constant stirring. Flasks were then placed on a hot plate at 350°C with constant stirring and heated to a slow boil (about 20–25&nbsp;min). The heat was then turned off, 4M NaCl was added to&nbsp;achieve varying concentrations of added NaCl, and total volume adjusted to 900&nbsp;mL with deinonized water. Medium was dispensed in 3–4&nbsp;mL aliquots in polystyrene vials. All crosses and assays were performed at room temperature (∼22°C–23°C) and ambient humidity.</p><p id="__p22" class="p p-last">We estimated the effect of the mutations on salt sensitivity by applying a Cox proportional hazards model on the fly survival data. The outcome was survival time, and at the end of the follow-up period, all living flies were censored. The data for each genetic background (heterogenic or isogenic) and NaCl concentration were analyzed separately. We estimated the effect on the hazard ratio of genotype (each mutation versus control as baseline). To account for intra-vial correlation, we used robust sandwich variance estimators in a generalized estimating equation (GEE)-like model that treats members of each vial as associated with a single cluster. Analyses were performed with the R “survival” package.</p></div><div id="sec2.14" class="sec sec-last"><h3 id="sec2.14title">Mouse Renal Expression Study</h3><p id="__p23" class="p p-first">129S6 mice were purchased from Taconic Biosciences and were maintained on a 12&nbsp;hr light-dark cycle with free access to standard chow and water in the animal facility of the University of Virginia. Only male mice at 12&nbsp;weeks of age were used. High-salt diet (HSD, 6% NaCl) in pellets was purchased from Harlan Teklad and administered in place of normal chow for two weeks. Experiments were carried out in accordance with local and NIH guidelines. To induce CKD, mice were subjected to sub-total nephrectomy (Nx) under 1.5% isoflurane anesthesia, the right kidney was removed, and the upper branch of the two main branches of the left renal artery were ligated to impede blood supply to the upper half of the kidney as previously reported.<sup><a href="#bib31" rid="bib31" class=" bibr popnode">31</a></sup></p><p id="__p24" class="p p-last">Renal mRNA was extracted at the end of 2&nbsp;weeks of HSD, or at 12&nbsp;weeks after sub-total Nx. Real-time RT-PCR was performed as&nbsp;previously described<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> with the primers listed in <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S3</a>. Fluorescence detection was accomplished with Sybr Green and the iCYcler system (Bio-Rad). mRNA expression was normalized against mRNA expression of the <em>Hprt</em> housekeeping gene, and the mean at baseline was used as the reference for determination of relative expression across conditions.</p></div></div><div id="sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">Identification of Loci Associated with Kidney Function across Ancestry Groups</h3><p id="__p25" class="p p-first">We identified 20 loci attaining genome-wide-significant evidence of association with eGFR (p &lt; 5&nbsp;× 10<sup>−8</sup>) in trans-ethnic meta-analysis (<a href="/pmc/articles/PMC5011075/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>, <a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S2</a>). These loci have been previously reported in ethnic-specific GWASs of individuals with European and East Asian ancestry<sup><a href="#bib4" rid="bib4" class=" bibr popnode">4</a></sup><sup>, </sup><sup><a href="#bib5" rid="bib5" class=" bibr popnode">5</a></sup><sup>, </sup><sup><a href="#bib6" rid="bib6" class=" bibr popnode">6</a></sup><sup>, </sup><sup><a href="#bib8" rid="bib8" class=" bibr popnode">8</a></sup> (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S4</a>). They include two loci discovered in a recently published meta-analysis of European ancestry GWASs: <em>LRP2</em> ([MIM: <a href="/omim/600073" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600073</a>] rs57989581, p = 5.6&nbsp;× 10<sup>−10</sup>) and <em>NFATC1</em> ([MIM: <a href="/omim/600489" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600489</a>] rs8096658, p = 1.3&nbsp;× 10<sup>−8</sup>). Previously reported lead SNPs at an additional 21 established kidney-function loci attained nominal evidence of association (p &lt; 0.05) with eGFR, with consistent direction of effect (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S4</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p66">Loci Attaining Genome-wide-Significant Evidence of Association (p &lt; 5&nbsp;× 10<sup>−8</sup>) with eGFR in Trans-ethnic Meta-analysis of 71,638 Individuals</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715183440464" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Locus</strong></th><th rowspan="1" colspan="1"><strong>Lead SNP</strong></th><th rowspan="1" colspan="1"><strong>Chr</strong></th><th rowspan="1" colspan="1"><strong>Position (bp, b37)</strong></th><th colspan="2" rowspan="1"><strong>Alleles</strong><hr></th><th colspan="5" rowspan="1"><strong>Trans-ethnic Meta-analysis</strong><hr></th></tr><tr><th rowspan="1" colspan="1"><strong>Effect</strong><sup>a</sup></th><th rowspan="1" colspan="1"><strong>Other</strong></th><th rowspan="1" colspan="1"><strong>Beta</strong></th><th rowspan="1" colspan="1"><strong>SE</strong></th><th rowspan="1" colspan="1"><strong>p Value</strong></th><th rowspan="1" colspan="1"><strong>Cochran’s Q p Value</strong></th><th rowspan="1" colspan="1"><strong>N</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>SLC43A1</em></td><td rowspan="1" colspan="1">rs35716097</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">176,806,636</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">−1.097</td><td rowspan="1" colspan="1" align="center">0.127</td><td rowspan="1" colspan="1" align="center">2.3&nbsp;× 10<sup>−17</sup></td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>SHROOM3</em></td><td rowspan="1" colspan="1">rs5020545</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">77,414,988</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">−0.969</td><td rowspan="1" colspan="1" align="center">0.119</td><td rowspan="1" colspan="1" align="center">1.3&nbsp;× 10<sup>−15</sup></td><td rowspan="1" colspan="1" align="center">0.010</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>PDILT-UMOD</em></td><td rowspan="1" colspan="1">rs77924615</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">20,392,332</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">−1.185</td><td rowspan="1" colspan="1" align="center">0.147</td><td rowspan="1" colspan="1" align="center">1.7&nbsp;× 10<sup>−15</sup></td><td rowspan="1" colspan="1" align="center">0.011</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>UNCX</em></td><td rowspan="1" colspan="1">rs62435145</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">1,286,567</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">−1.092</td><td rowspan="1" colspan="1" align="center">0.137</td><td rowspan="1" colspan="1" align="center">4.7&nbsp;× 10<sup>−15</sup></td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">59,865</td></tr><tr><td rowspan="1" colspan="1"><em>GCKR</em></td><td rowspan="1" colspan="1">rs1260326</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">27,730,940</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">−0.872</td><td rowspan="1" colspan="1" align="center">0.114</td><td rowspan="1" colspan="1" align="center">6.1&nbsp;× 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">0.069</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>BCAS3</em></td><td rowspan="1" colspan="1">rs9895661</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">59,456,589</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">−1.003</td><td rowspan="1" colspan="1" align="center">0.132</td><td rowspan="1" colspan="1" align="center">7.9&nbsp;× 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>SPATA5L1-GATM</em></td><td rowspan="1" colspan="1">rs2486288</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">45,712,339</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">−0.883</td><td rowspan="1" colspan="1" align="center">0.126</td><td rowspan="1" colspan="1" align="center">4.7&nbsp;× 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">0.76</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>ALMS1</em></td><td rowspan="1" colspan="1">rs7587577</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">73,832,786</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">−0.948</td><td rowspan="1" colspan="1" align="center">0.135</td><td rowspan="1" colspan="1" align="center">5.2&nbsp;× 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">0.098</td><td rowspan="1" colspan="1" align="center">48,102</td></tr><tr><td rowspan="1" colspan="1"><em>CPS1</em></td><td rowspan="1" colspan="1">rs715</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">211,543,055</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">−0.876</td><td rowspan="1" colspan="1" align="center">0.127</td><td rowspan="1" colspan="1" align="center">1.3&nbsp;× 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.21</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>WDR72</em></td><td rowspan="1" colspan="1">rs1031755</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">53,951,435</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">−0.860</td><td rowspan="1" colspan="1" align="center">0.127</td><td rowspan="1" colspan="1" align="center">2.2&nbsp;× 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.0013</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>PIP5K1B</em></td><td rowspan="1" colspan="1">rs4744712</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">71,434,707</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">−0.753</td><td rowspan="1" colspan="1" align="center">0.112</td><td rowspan="1" colspan="1" align="center">3.3&nbsp;× 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>PRKAG2</em></td><td rowspan="1" colspan="1">rs10265221</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">151,414,329</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">−0.963</td><td rowspan="1" colspan="1" align="center">0.146</td><td rowspan="1" colspan="1" align="center">7.3&nbsp;× 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>DAB2-C9</em></td><td rowspan="1" colspan="1">chr5: 39,404,526:D</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">39,404,526</td><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1">R</td><td rowspan="1" colspan="1" align="center">−0.817</td><td rowspan="1" colspan="1" align="center">0.126</td><td rowspan="1" colspan="1" align="center">1.5&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.80</td><td rowspan="1" colspan="1" align="center">48,102</td></tr><tr><td rowspan="1" colspan="1"><em>LRP2</em></td><td rowspan="1" colspan="1">rs57989581</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">170,194,459</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">−1.980</td><td rowspan="1" colspan="1" align="center">0.315</td><td rowspan="1" colspan="1" align="center">5.6&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>SLC22A2</em></td><td rowspan="1" colspan="1">rs316009</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">160,675,764</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">−1.193</td><td rowspan="1" colspan="1" align="center">0.192</td><td rowspan="1" colspan="1" align="center">1.0&nbsp;× 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.49</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>LOC100132354-VEGFA</em></td><td rowspan="1" colspan="1">rs881858</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">43,806,609</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">−0.772</td><td rowspan="1" colspan="1" align="center">0.127</td><td rowspan="1" colspan="1" align="center">2.0&nbsp;× 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.0020</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>DCDC5-MPPED2</em></td><td rowspan="1" colspan="1">rs963837</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">30,749,090</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">−0.685</td><td rowspan="1" colspan="1" align="center">0.114</td><td rowspan="1" colspan="1" align="center">3.7&nbsp;× 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.0034</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>NFATC1</em></td><td rowspan="1" colspan="1">rs8096658</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="center">77,156,537</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">−0.814</td><td rowspan="1" colspan="1" align="center">0.141</td><td rowspan="1" colspan="1" align="center">1.3&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.015</td><td rowspan="1" colspan="1" align="center">59,865</td></tr><tr><td rowspan="1" colspan="1"><em>PHTF2</em></td><td rowspan="1" colspan="1">rs848486</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">77,552,127</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">−0.643</td><td rowspan="1" colspan="1" align="center">0.113</td><td rowspan="1" colspan="1" align="center">2.0&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.83</td><td rowspan="1" colspan="1" align="center">71,638</td></tr><tr><td rowspan="1" colspan="1"><em>TFDP2</em></td><td rowspan="1" colspan="1">rs1511299</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">141,716,072</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">−0.727</td><td rowspan="1" colspan="1" align="center">0.131</td><td rowspan="1" colspan="1" align="center">4.4&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.55</td><td rowspan="1" colspan="1" align="center">71,638</td></tr></tbody></table></div><div id="largeobj_idm140715183440464" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5011075/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>Effect allele is eGFR-decreasing allele.</div></div></div><p id="__p26" class="p p-last">As expected, lead SNPs were common across ancestry groups at all 20 loci, with each displaying modest effects on eGFR (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S5</a>). Despite substantial variability in allele frequencies between ancestry groups, we observed no evidence of trans-ethnic heterogeneity in allelic effects on eGFR for any lead SNP (<a href="/pmc/articles/PMC5011075/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>, <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S5</a>). Through conditional analyses (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S6</a>), we observed no evidence of multiple distinct signals of association for eGFR at any locus (p<sub>COND</sub> &lt; 10<sup>−5</sup>, Bonferroni correction for ∼5,000 variants per locus). Taken together, these data are consistent with a single variant driving association signals in each locus; each variant is shared across ancestry groups and has homogeneous effects on eGFR in diverse populations. However, we recognize that larger multi-ethnic samples will be required to detect lower frequency, population-specific distinct association signals of modest effect on kidney function.</p></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">Impact of Lead eGFR SNPs on CKD and Kidney Function in Diabetic Individuals</h3><p id="__p27" class="p p-first">We assessed the impact on CKD of lead SNPs at the 20 eGFR loci in a subset of individuals (up to 3,976 cases and 55,904 controls) contributing to our trans-ethnic meta-analysis (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S7</a>). We defined CKD by eGFR &lt; 60&nbsp;mL/min/1.73 m<sup>2</sup> and/or incidence of ESRD. For all 20 lead SNPs, the eGFR-decreasing allele was associated with increased risk of CKD. Eleven of the lead SNPs demonstrated evidence of association with CKD at nominal significance (p &lt; 0.05), and the strongest signals were observed at <em>UNCX</em> (rs62435145, p = 2.2&nbsp;× 10<sup>−7</sup>), <em>ALMS1</em> ([MIM: <a href="/omim/606844" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606844</a>] rs7587577, p = 3.1&nbsp;× 10<sup>−6</sup>), and <em>PDILT-UMOD</em> ([MIM: <a href="/omim/191845" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">191845</a>] rs77924615, p = 4.0&nbsp;× 10<sup>−6</sup>).</p><p id="__p28" class="p p-last">We also investigated the impact of the lead SNPs on eGFR in GWASs of individuals with diabetes for whom there are different mechanisms for loss of renal function, such as diabetic nephropathy. We obtained association summary statistics for eGFR in 13,158 subjects of European ancestry with diabetes (9,197 with T2D and 3,961 with T1D) from the SUMMIT Consortium (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S8</a>). Consistent with previous reports,<sup><a href="#bib8" rid="bib8" class=" bibr popnode">8</a></sup><sup>, </sup><sup><a href="#bib33" rid="bib33" class=" bibr popnode">33</a></sup> allelic effects on eGFR in diabetic individuals and our trans-ethnic meta-analysis of individuals from the general population were homogeneous (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S3</a>). There was nominal evidence of association with eGFR (p &lt; 0.05), with the same direction of effect, at seven loci, and the strongest signals were observed at <em>PDILT-UMOD</em> (p = 6.9&nbsp;× 10<sup>−6</sup>), <em>PRKAG2</em> ([MIM: <a href="/omim/602743" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602743</a>] p = 0.00013) and <em>NFATC1</em> (p = 0.00045).</p></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">Fine Mapping of eGFR Loci</h3><p id="__p29" class="p p-first-last">We next sought to localize variants driving eGFR association signals in each of the 20 loci attaining genome-wide significance in our trans-ethnic meta-analysis. We utilized trans-ethnic fine mapping implemented in MANTRA,<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup> taking advantage of increased sample size and the expectation that patterns of LD vary between diverse populations. We derived credible sets of variants<sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup> mapping within 500&nbsp;kb of the lead SNP at each locus that together account for 99% of the posterior probability (π<sub>C</sub>) of driving the association signal (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S9</a>). Smaller credible sets, in terms of the number of SNPs they contain, or the genomic interval that they cover, thus correspond to more precise fine-mapping.&nbsp;The 99% credible set at the <em>PDILT-UMOD</em> locus included a single variant (rs77924615, π<sub>C</sub>&gt;0.999), which maps to an intron of <em>PDILT</em>. This variant has previously been reported as driving the primary association signal for CKD at the <em>PDILT-UMOD</em> locus through whole-genome sequencing and long-range haplotype imputation into 194,286 Icelandic individuals with serum creatinine measurements.<sup><a href="#bib34" rid="bib34" class=" bibr popnode">34</a></sup> We also observed precise localization, defined by a 99% credible set including no more than five variants (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S10</a>), at a five additional loci: <em>NFATC1</em> (two variants, mapping to 0.4 kb), <em>SLC34A1</em> ([MIM: <a href="/omim/182309" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">182309</a>] two variants,&nbsp;mapping to 0.6 kb), <em>GCKR</em> ([MIM: <a href="/omim/600842" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600842</a>] three variants,&nbsp;mapping to 11.7 kb), <em>DCDC5-MPPED2</em> ([MIM: <a href="/omim/612321" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612321</a>, <a href="/omim/600911" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600911</a>] four variants, mapping to 27.9 kb), and <em>PIP5K1B</em> ([MIM: <a href="/omim/602745" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602745</a>] five variants, mapping to 3.5 kb).</p></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">Integration of Genetic Fine-mapping and Genomic Annotation</h3><p id="__p30" class="p p-first">To gain insight into the mechanisms through which association signals at the 20 GWAS loci attaining genome-wide significance in our trans-ethnic meta-analysis impact eGFR, we began by obtaining genomic annotations for all single-nucleotide variants mapping within 500 kb of lead SNPs. Across all 20 loci, only 5.4% of the posterior probability of driving association signals was annotated to&nbsp;coding variants (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S11</a>), the majority of which was accounted for by <em>GCKR</em> p.Pro446Leu (rs1260326, π<sub>C</sub> = 0.938). This missense variant has been shown, functionally, to result in increased de novo triglyceride and cholesterol synthesis and export and decreased plasma glucose concentrations, all of which have been associated with risk of CKD,<sup><a href="#bib35" rid="bib35" class=" bibr popnode">35</a></sup><sup>, </sup><sup><a href="#bib36" rid="bib36" class=" bibr popnode">36</a></sup> making <em>GCKR</em> the likely effector transcript for eGFR at this locus. However, outside of the <em>GCKR</em> locus, variants mapping to non-coding sequence accounted for more than 99.4% of the probability of driving eGFR association, suggesting that these signals are most likely to be mediated by effects on gene regulation.</p><p id="__p31">We next investigated whether genomic annotations of regulatory chromatin state for 93 cell types,<sup><a href="#bib26" rid="bib26" class=" bibr popnode">26</a></sup> DHSs for 145 cell types,<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> and ChIP-seq binding sites for 165 transcription factors<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup><sup>, </sup><sup><a href="#bib28" rid="bib28" class=" bibr popnode">28</a></sup> were predictive of posterior probability of driving association signals across the 20 loci (<a href="/pmc/articles/PMC5011075/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>, <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S12</a>). We observed significant effects (p &lt; 0.00012, Bonferroni correction for 403 annotations) on posterior probability for variants in kidney DHSs, including adult&nbsp;renal proximal tubular epithelial cells (RPTECs; p = 3.4&nbsp;× 10<sup>−8</sup>), renal cortical epithelial cells (HRCEs; p = 4.7&nbsp;×10<sup>−7</sup>), and fetal kidney cells (p = 8.8&nbsp;× 10<sup>−6</sup>). We also observed significant effects on posterior probability for transcription-factor binding sites, most notably for HDAC8 (p = 1.1&nbsp;× 10<sup>−14</sup>). Histone deacetylases (HDACs) are involved in kidney function and development,<sup><a href="#bib37" rid="bib37" class=" bibr popnode">37</a></sup> and HDAC inhibitors could be promising in the treatment of kidney disease.<sup><a href="#bib38" rid="bib38" class=" bibr popnode">38</a></sup> We repeated our analyses by using fGWAS<sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup> (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S4</a>, <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S12</a>) and observed strong correlation in the ranking of enriched annotations (<em>r</em><sup>2</sup> = 0.93). These results highlight that variants driving association signals with eGFR are more likely to be co-localized with annotated elements in kidney cells, thereby suggesting that gene regulation in disease-relevant tissues is a likely mechanism by which GWAS loci impact CKD.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC5011075/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715202204592" class="figure"><a href="/pmc/articles/PMC5011075/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5011075_gr1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="Figure 1" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5011075/bin/gr1.jpg"></a></div><div id="largeobj_idm140715202204592" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5011075/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC5011075/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p60">DNase I Hypersensitivity Sites in Kidney Cells and HDAC8 Binding Sites are Predictive of Posterior Probability of Driving Association Signals at 20 eGFR Loci</p><p id="__p61">We tested whether genomic annotations of regulatory chromatin state for 93 cell types, DNase I hypersensitivity sites (DHSs) for 145 cell types, and chromatin immuno-precipitation sequence binding sites for 165 transcription factors were predictive of posterior probability of driving eGFR association signals. Each point corresponds to an annotation, plotted according to the effect size (log-odds ratio for driving association signal) on the x axis and ranked according to the significance of the association on the y&nbsp;axis. Significant association (p &lt; 0.00012, highlighted in red) was defined by Bonferroni correction for 403 tested annotations. The most significant effects included DHSs in kidney cells (RPTECs and HRCEs) and binding sites for HDAC8.</p></div></div></div><p id="__p32" class="p p-last">Lead SNPs that, by themselves, accounted for more than 80% of the posterior probability of driving association signals overlapped an enriched annotation at five loci (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S13</a>). In particular, at the <em>SLC34A1</em> locus, rs35716097 (π<sub>C</sub>&nbsp;= 0.946) overlapped DHSs in RPTECs and HRCEs, as well as a binding site for HDAC8, while at the <em>NFATC1</em> locus, rs8096658 (π<sub>C</sub> = 0.877) overlapped fetal kidney cell DHSs (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S5</a>). At both of these loci, the lead SNPs were also expression quantitative trait loci (eQTLs) for <em>NFATC1</em> and <em>RGS14</em> (MIM: <a href="/omim/602513" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602513</a>; at the <em>SLC34A1</em> locus) in multiple tissues (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S13</a>), highlighting these genes as likely effector transcripts through which eGFR association signals are mediated. <em>NFATC1</em> plays a central role in inducible gene transcription during immune response and is a downstream target of the transplant immunosuppression drug cyclosporine A. <em>RGS14</em> encodes a member of the regulator of G protein signaling family, which modulates downstream effects of Gα subunits and has unknown function in kidneys.</p></div><div id="sec3.5" class="sec sec-last"><h3 id="sec3.5title">Experimental Data in Model Organisms</h3><p id="__p33" class="p p-first">To provide insight into the role of <em>NFATC1</em> and <em>RGS14</em> (at the <em>SLC34A1</em> locus) in kidney physiology, we examined the function of ancestral orthologs in <em>Drosophila melanogaster</em>. The <em>Drosophila</em> genome encodes a single member of the <em>NFAT</em> family, and a previous report has demonstrated that flies with <em>NFAT</em> loss-of-function mutations have increased salt sensitivity, suggesting a role for this gene in ionic or osmotic regulation.<sup><a href="#bib39" rid="bib39" class=" bibr popnode">39</a></sup> The closest <em>RGS14</em> ortholog in <em>Drosophila melanogaster</em> is <em>loco</em>, for which reduced expression is associated with longer lifespan and stress resistance.<sup><a href="#bib30" rid="bib30" class=" bibr popnode">30</a></sup> We thus conducted experiments aimed at characterizing a role for <em>loco</em> loss-of-function variants in&nbsp;salt sensitivity. We compared survival of two independently derived heterozygous <em>loco</em> mutants (<em>y</em><sup><em>1</em></sup><em>w</em><sup><em>1</em></sup><em>; loco</em><sup><em>d06164</em></sup><em>/+</em> and <em>y</em><sup><em>1</em></sup><em>w</em><sup><em>1</em></sup><em>; loco</em><sup><em>EY-P283</em></sup><em>/+</em>) with isogenic <em>y</em><sup><em>1</em></sup><em>w</em><sup><em>1</em></sup> controls after supplementing their diet with varying NaCl concentrations for 8&nbsp;days (<a href="/pmc/articles/PMC5011075/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>). There was very little mortality of any of the genotypes on non-NaCl-supplemented&nbsp;food, indicating no baseline differences in&nbsp;viability over the time period tested. However, we observed significantly improved survival of the heterozygous <em>loco</em>&nbsp;mutants over controls on NaCl-supplemented food (<a href="/pmc/articles/PMC5011075/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>, <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S14</a>), thereby indicating a role for&nbsp;this gene in resistance to salt stress. To exclude the effects of inbreeding depression on our findings, we also repeated our experiments with the same strains on a heterogenic background. As expected, the hybrid heterogenic strains were less salt susceptible than the isogenic strains, but the <em>loco</em> mutants remained salt-resistant when compared to controls of a similar genetic background (<a href="/pmc/articles/PMC5011075/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>, <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S14</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC5011075/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715202364720" class="figure"><a href="/pmc/articles/PMC5011075/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5011075_gr2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="Figure 2" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5011075/bin/gr2.jpg"></a></div><div id="largeobj_idm140715202364720" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5011075/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC5011075/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p62"><em>Drosophila RGS14</em> Heterozygous Mutants Are Resistant to Salt Stress</p><p id="__p63">Survival of flies carrying heterozygous loss-of-function mutations in the <em>Drosophila melanogaster RGS14</em> homolog, <em>loco</em>, was compared to that of controls of the same genetic background. In the isogenic experiment, all genotypes were backcrossed to the control strain. In the heterogenic experiment, controls and <em>loco</em> mutants were crossed with the A.R.R. lab’s <em>wBerlin</em> strain to obtain highly heterozygous progeny. Kaplan-Meier plots demonstrated that flies heterozygous for two independently derived loco mutations, <em>loco</em><sup><em>EY-P283</em></sup> and <em>loco</em><sup><em>d06164</em></sup>, were resistant to salt stress across a range of NaCl concentrations when compared to controls. Cox-proportional hazards p values for each mutant, compared to those of controls, are presented and are calculated for each genetic background (isogenic or heterogenic) and NaCl concentration separately. Results are based on 170–200 flies per genotype for each NaCl concentration.</p></div></div></div><p id="__p34" class="p p-last">To further investigate the role of <em>NFATC1</em> and <em>RGS14</em> in kidney function, we used the 129S6 mouse strain that is salt-sensitive<sup><a href="#bib31" rid="bib31" class=" bibr popnode">31</a></sup> and susceptible to glomerulosclerosis.<sup><a href="#bib40" rid="bib40" class=" bibr popnode">40</a></sup> We compared the renal mRNA expression of <em>Nfatc1</em> and <em>Rgs14</em> at baseline versus (1) after a 2-week exposure to high-salt diet and (2) at 12&nbsp;weeks after CKD induced by sub-total nephrectomy. Compared to baseline condition, <em>Rgs14</em> was significantly decreased (∼75%, p = 0.01) during high-salt exposure (<a href="/pmc/articles/PMC5011075/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>). In the CKD model, <em>Rgs14</em> expression was also reduced and approached statistical significance (p = 0.06). The renal mRNA expression of <em>Nfatc1</em> was also significantly decreased (∼50%, p = 0.03) during high-salt exposure and trended down in CKD (p = 0.31). Although we cannot establish cause and effect, these data illustrate that the expression of both genes is altered during disease states.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC5011075/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140715202594672" class="figure"><a href="/pmc/articles/PMC5011075/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5011075_gr3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="Figure 3" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5011075/bin/gr3.jpg"></a></div><div id="largeobj_idm140715202594672" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5011075/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC5011075/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p64">Relative Renal mRNA Expression of <em>Rgs14</em> and <em>Nfatc1</em></p><p id="__p65">Expression of <em>Rgs14</em> is shown in (A) and <em>Nfatc1</em> in (B). n = 5 or 6 in each group. The empty triangle represents the median. According to the Mann-Whitney test, and compared to the baseline, <em>Rgs14</em> expression after exposure to a high-salt diet was significantly lower (p = 0.01), and was also lower in CKD (p = 0.06). <em>Nfatc1</em> expression after exposure to a high-salt diet was also significantly lower (p&nbsp;=&nbsp;0.03) and trended in the same direction in CKD (p = 0.31).</p></div></div></div></div></div><div id="sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec4title">Discussion</h2><p id="__p35" class="p p-first">We have undertaken a trans-ethnic meta-analysis of GWASs of eGFR, supplemented by imputation up to the phase 1 integrated (March 2012 release) multi-ethnic reference panel from the 1000 Genomes Project.<sup><a href="#bib9" rid="bib9" class=" bibr popnode">9</a></sup> With these high-density imputed data, we identified 20 loci at genome-wide significance for eGFR through trans-ethnic meta-analysis. Despite improved coverage of low-frequency variation offered by high-density imputation, lead SNPs were common across ancestry groups at all 20 of these kidney-function loci. There was also minimal evidence of trans-ethnic heterogeneity in allelic effects on eGFR at lead SNPs at kidney-function loci, thereby arguing against the “synthetic association” hypothesis.<sup><a href="#bib41" rid="bib41" class=" bibr popnode">41</a></sup> It is highly unlikely that eGFR association signals at these kidney-function loci reflect unobserved lower frequency causal alleles with larger effects because (1) rare variants are unlikely to have arisen before human population migration out of Africa and thus are not anticipated to be widely shared across diverse populations<sup><a href="#bib9" rid="bib9" class=" bibr popnode">9</a></sup><sup>, </sup><sup><a href="#bib42" rid="bib42" class=" bibr popnode">42</a></sup> and (2) LD with these variants is expected to be highly variable between ethnicities.</p><p id="__p36">Our conditional analyses did not provide evidence for multiple distinct eGFR association signals, which is consistent with a single causal variant at each of the 20 eGFR loci. However, we recognize that conditional analyses evaluate the evidence for residual association at the locus that cannot be ascribed to the lead SNP and do not provide a formal framework to test for the presence of multiple causal variants, for example, that are in strong LD with each other and reside on the same haplotype. Furthermore, larger sample sizes will be required to detect distinct association signals defined by common variants of modest effect or low-frequency variants that might be specific to particular ethnic groups.</p><p id="__p37">As with most previous GWASs of kidney function, our study was limited to a single measure of eGFR for each participant. We also did not adjust for diabetes or hypertension in our analyses given that these conditions are&nbsp;potential mediators or modifiers of the SNP-eGFR associations. However, despite ethnic differences in the prevalence of these conditions, we observed no evidence of heterogeneity in allelic effects on eGFR between ancestry groups. Exploration of context-dependent effects should be considered in future studies, for example, by using gene-environment interaction or mediation analyses.</p><p id="__p38">Given our observation that eGFR association signals are shared across ancestry groups, we next sought to take advantage of the differential patterns of LD across diverse populations to fine-map kidney-function loci. Credible-set variants mapped predominantly to non-coding sequence, suggesting that eGFR association signals are most likely to be mediated by effects on gene regulation, in agreement with previous reports for other complex human traits.<sup><a href="#bib43" rid="bib43" class=" bibr popnode">43</a></sup><sup>, </sup><sup><a href="#bib44" rid="bib44" class=" bibr popnode">44</a></sup><sup>, </sup><sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup> Through integration of genetic fine-mapping data with information from regulatory annotation resources, we have demonstrated significant enrichment of variants driving eGFR association signals with DHSs in multiple kidney cell types. Overlap with these enriched annotations could be used as a prior model for eGFR association signals, genome-wide, to improve power for discovery of additional kidney-function loci and further enhance trans-ethnic fine-mapping efforts.<sup><a href="#bib46" rid="bib46" class=" bibr popnode">46</a></sup></p><p id="__p39">Lead SNPs at kidney-function loci overlapping enriched annotations included eQTL for <em>NFATC1</em> and <em>RGS14</em> (at the&nbsp;<em>SLC34A1</em> locus) in multiple tissues, pointing to likely effector transcripts through which these eGFR association signals are mediated. We have established that loss-of-function mutations in ancestral orthologs of both genes in <em>Drosophila melanogaster</em> are associated with response to salt stress. Although salt sensitivity has not been directly correlated with variation in eGFR in humans, it has been associated with albuminuria,<sup><a href="#bib47" rid="bib47" class=" bibr popnode">47</a></sup><sup>, </sup><sup><a href="#bib48" rid="bib48" class=" bibr popnode">48</a></sup> elevated creatinine,<sup><a href="#bib48" rid="bib48" class=" bibr popnode">48</a></sup> and the subsequent development of hypertension,<sup><a href="#bib49" rid="bib49" class=" bibr popnode">49</a></sup> suggesting the relevance of this trait to kidney function. Indeed, in animal models, salt sensitivity is tightly linked with a blunted tubuloglomerular feedback (TGF) or impaired increase in GFR after salt loading.<sup><a href="#bib50" rid="bib50" class=" bibr popnode">50</a></sup><sup>, </sup><sup><a href="#bib51" rid="bib51" class=" bibr popnode">51</a></sup><sup>, </sup><sup><a href="#bib52" rid="bib52" class=" bibr popnode">52</a></sup><sup>, </sup><sup><a href="#bib53" rid="bib53" class=" bibr popnode">53</a></sup> Consistent with this, we demonstrated that renal mRNA expression of <em>Nfatc1</em> and <em>Rgs14</em> in a salt-sensitive mouse model was reduced after exposure to a high-salt diet and induced CKD. In parallel with the findings in <em>Drosophila melanogaster</em>, these results are consistent with the hypothesis that the capacity to reduce expression of <em>Rgs14</em> and <em>Nfatc1</em> determines the extent of the response to stress. Another possible mechanism suggested by our results in <em>Drosophila</em> is a role for oxidative stress, to which <em>RGS14</em> ortholog mutants are resistant,<sup><a href="#bib30" rid="bib30" class=" bibr popnode">30</a></sup> and which has been implicated in mammalian salt sensitivity.<sup><a href="#bib54" rid="bib54" class=" bibr popnode">54</a></sup><sup>, </sup><sup><a href="#bib55" rid="bib55" class=" bibr popnode">55</a></sup> Establishing the functional role of these genes in salt sensitivity, TGF, GFR, oxidative stress, and CKD will require targeted in&nbsp;vivo studies using knockout and/or transgenic mouse models.</p><p id="__p40" class="p p-last">In conclusion, our study demonstrates the utility of trans-ethnic fine mapping through integration of GWASs of diverse populations with genomic annotation from relevant tissues to define molecular mechanisms by which association signals exert their effect, thereby offering an exciting opportunity to elucidate the pathophysiology of complex human diseases.</p></div><div id="ack0010" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ack0010title">Acknowledgments</h2><div class="sec"><p id="__p58">A.R.R. and J.N.S. are supported by the US NIH (K08DK091316). T.H.L. is supported by the National Institute of Diabetes and Digestive and Kidney Diseases (1R01DK094907-01). S.P. is supported by the US NIH (R25DK101401). G.W.M. is supported by the Australian National Health and Medical Research Council Fellowship Scheme. Yukinori Okada was supported by the Japan Society for the Promotion of Science KAKENHI (15H05911, 15H05670, 15K14429), the Japan Science and Technology Agency, Mochida Memorial Foundation for Medical and Pharmaceutical Research, Takeda Science Foundation, Gout Research Foundation, the Tokyo Biochemical Research Foundation, and the Japan Rheumatism Foundation. A.P.M. is a Wellcome Trust Senior Fellow in Basic Biomedical Science (grant WT098017). N.F. is supported by the US NIH (5R21HL123677-02). The ESRD data reported here have been supplied by the United States Renal Data System. The interpretation and reporting of these data are the responsibility of the authors and in no way should be seen as official policy or interpretation of the US government. Additional acknowledgments are provided in the Supplemental Data.</p></div></div><div id="idm140715211864064" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140715211864064title">Notes</h2><p id="misc0010">Published: September 1, 2016</p></div><div id="idm140715217273808" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140715217273808title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="app1"></p><p id="__p59" class="p p-first-last">Supplemental Data include five figures, fourteen tables, and Supplemental Acknowledgments and can be found with this article online at <a href="//dx.doi.org/10.1016/j.ajhg.2016.07.012" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">http://dx.doi.org/10.1016/j.ajhg.2016.07.012</a>.</p><p></p></div></div><div id="app3" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><!--/article/back/sec/--><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li id="u9010"><p id="__p42">1000 Genomes, <a href="http://www.1000genomes.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.1000genomes.org</a></p></li><li id="u0010"><p id="__p43">ENCODE, <a href="https://www.encodeproject.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.encodeproject.org/</a></p></li><li id="u0015"><p id="__p44">Ensembl Genome Browser, <a href="http://www.ensembl.org/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ensembl.org/index.html</a></p></li><li id="u0020"><p id="__p45">EPACTS, <a href="http://genome.sph.umich.edu/wiki/EPACTS" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.sph.umich.edu/wiki/EPACTS</a></p></li><li id="u0025"><p id="__p46">fGWAS, <a href="https://github.com/joepickrell/fgwas" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/joepickrell/fgwas</a></p></li><li id="u0030"><p id="__p47">Gencode, <a href="http://www.gencodegenes.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gencodegenes.org</a></p></li><li id="u0035"><p id="__p48">GWAMA, <a href="http://www.geenivaramu.ee/en/tools/gwama" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.geenivaramu.ee/en/tools/gwama</a></p></li><li id="u0040"><p id="__p49">IMPUTE2, <a href="http://mathgen.stats.ox.ac.uk/impute/impute_v2.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mathgen.stats.ox.ac.uk/impute/impute_v2.html</a></p></li><li id="u0045"><p id="__p50">International HapMap Project, <a href="http://hapmap.ncbi.nlm.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hapmap.ncbi.nlm.nih.gov/</a></p></li><li id="u0050"><p id="__p51">LDlink, <a href="http://analysistools.nci.nih.gov/LDlink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://analysistools.nci.nih.gov/LDlink/</a></p></li><li id="u0055"><p id="__p52">METAL, <a href="http://www.sph.umich.edu/csg/abecasis/metal/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/metal/</a></p></li><li id="u0060"><p id="__p53">Minimac, <a href="http://genome.sph.umich.edu/wiki/Minimac" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.sph.umich.edu/wiki/Minimac</a></p></li><li id="u0065"><p id="__p54">OMIM, <a href="http://www.omim.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.omim.org/</a></p></li><li id="u0070"><p id="__p55">Roadmap, <a href="http://www.roadmapepigenomics.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.roadmapepigenomics.org/</a></p></li><li id="u0075"><p id="__p56">Variant Effect Predictor, <a href="http://useast.ensembl.org/Homo_sapiens/Tools/VEP" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://useast.ensembl.org/Homo_sapiens/Tools/VEP</a></p></li></ul><p></p></div><div id="app2" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Figures S1–S5, Tables S1–S14, and Supplemental Acknowledgments:</strong><div class="sup-box half_rhythm" id="idm140715214875072"><a href="/pmc/articles/PMC5011075/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.2M, pdf)</sup>Document S1. Figures S1–S5, Tables S1–S14, and Supplemental Acknowledgments</div></div><div class="sec suppmat" id="mmc2"><!--caption a9--><strong>Document S2. Article plus Supplemental Data:</strong><div class="sup-box half_rhythm" id="idm140715217276032"><a href="/pmc/articles/PMC5011075/bin/mmc2.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.0M, pdf)</sup>Document S2. Article plus Supplemental Data</div></div><p></p></div><div id="idm140715179312336" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140715179312336title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation" id="sref1">Levey A.S., Coresh J. Chronic kidney disease. <span><span class="ref-journal">Lancet. </span>2012;<span class="ref-vol">379</span>:165–180.</span> [<a href="/pubmed/21840587" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Chronic+kidney+disease&amp;author=A.S.+Levey&amp;author=J.+Coresh&amp;volume=379&amp;publication_year=2012&amp;pages=165-180&amp;pmid=21840587&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation" id="sref2">Collins A.J., Foley R.N., Gilbertson D.T., Chen S.-C. United States Renal Data System public health surveillance of chronic kidney disease and end-stage renal disease. <span><span class="ref-journal">Kidney Int Suppl (2011) </span>2015;<span class="ref-vol">5</span>:2–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4455192/">PMC free article</a>]</span> [<a href="/pubmed/26097778" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int+Suppl+(2011)&amp;title=United+States+Renal+Data+System+public+health+surveillance+of+chronic+kidney+disease+and+end-stage+renal+disease&amp;author=A.J.+Collins&amp;author=R.N.+Foley&amp;author=D.T.+Gilbertson&amp;author=S.-C.+Chen&amp;volume=5&amp;publication_year=2015&amp;pages=2-7&amp;pmid=26097778&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation" id="sref3">Xue J.L., Eggers P.W., Agodoa L.Y., Foley R.N., Collins A.J. Longitudinal study of racial and ethnic differences in developing end-stage renal disease among aged medicare beneficiaries. <span><span class="ref-journal">J.&nbsp;Am. Soc. Nephrol. </span>2007;<span class="ref-vol">18</span>:1299–1306.</span> [<a href="/pubmed/17329578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Am.+Soc.+Nephrol.&amp;title=Longitudinal+study+of+racial+and+ethnic+differences+in+developing+end-stage+renal+disease+among+aged+medicare+beneficiaries&amp;author=J.L.+Xue&amp;author=P.W.+Eggers&amp;author=L.Y.+Agodoa&amp;author=R.N.+Foley&amp;author=A.J.+Collins&amp;volume=18&amp;publication_year=2007&amp;pages=1299-1306&amp;pmid=17329578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation" id="sref4">Köttgen A., Pattaro C., Böger C.A., Fuchsberger C., Olden M., Glazer N.L., Parsa A., Gao X., Yang Q., Smith A.V. New loci associated with kidney function and chronic kidney disease. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:376–384.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2997674/">PMC free article</a>]</span> [<a href="/pubmed/20383146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=New+loci+associated+with+kidney+function+and+chronic+kidney+disease&amp;author=A.+Köttgen&amp;author=C.+Pattaro&amp;author=C.A.+Böger&amp;author=C.+Fuchsberger&amp;author=M.+Olden&amp;volume=42&amp;publication_year=2010&amp;pages=376-384&amp;pmid=20383146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation" id="sref5">Pattaro C., Köttgen A., Teumer A., Garnaas M., Böger C.A., Fuchsberger C., Olden M., Chen M.-H., Tin A., Taliun D., CARDIoGRAM Consortium. ICBP Consortium. CARe Consortium. Wellcome Trust Case Control Consortium 2 (WTCCC2)  Genome-wide association and functional follow-up reveals new loci for kidney function. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002584.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3315455/">PMC free article</a>]</span> [<a href="/pubmed/22479191" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+and+functional+follow-up+reveals+new+loci+for+kidney+function&amp;author=C.+Pattaro&amp;author=A.+Köttgen&amp;author=A.+Teumer&amp;author=M.+Garnaas&amp;author=C.A.+Böger&amp;volume=8&amp;publication_year=2012&amp;pages=e1002584&amp;pmid=22479191&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation" id="sref6">Okada Y., Sim X., Go M.J., Wu J.Y., Gu D., Takeuchi F., Takahashi A., Maeda S., Tsunoda T., Chen P., KidneyGen Consortium. CKDGen Consortium. GUGC consortium  Meta-analysis identifies multiple loci associated with kidney function-related traits in east Asian populations. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:904–909.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4737645/">PMC free article</a>]</span> [<a href="/pubmed/22797727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Meta-analysis+identifies+multiple+loci+associated+with+kidney+function-related+traits+in+east+Asian+populations&amp;author=Y.+Okada&amp;author=X.+Sim&amp;author=M.J.+Go&amp;author=J.Y.+Wu&amp;author=D.+Gu&amp;volume=44&amp;publication_year=2012&amp;pages=904-909&amp;pmid=22797727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation" id="sref7">Gorski M., Tin A., Garnaas M., McMahon G.M., Chu A.Y., Tayo B.O., Pattaro C., Teumer A., Chasman D.I., Chalmers J. Genome-wide association study of kidney function decline in individuals of European descent. <span><span class="ref-journal">Kidney Int. </span>2015;<span class="ref-vol">87</span>:1017–1029.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4425568/">PMC free article</a>]</span> [<a href="/pubmed/25493955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int.&amp;title=Genome-wide+association+study+of+kidney+function+decline+in+individuals+of+European+descent&amp;author=M.+Gorski&amp;author=A.+Tin&amp;author=M.+Garnaas&amp;author=G.M.+McMahon&amp;author=A.Y.+Chu&amp;volume=87&amp;publication_year=2015&amp;pages=1017-1029&amp;pmid=25493955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation" id="sref8">Pattaro C., Teumer A., Gorski M., Chu A.Y., Li M., Mijatovic V., Garnaas M., Tin A., Sorice R., Li Y., ICBP Consortium. AGEN Consortium. CARDIOGRAM. CHARGe-Heart Failure Group. ECHOGen Consortium  Genetic associations at 53 loci highlight cell types and biological pathways relevant for kidney function. <span><span class="ref-journal">Nat. Commun. </span>2016;<span class="ref-vol">7</span>:10023.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4735748/">PMC free article</a>]</span> [<a href="/pubmed/26831199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Genetic+associations+at+53+loci+highlight+cell+types+and+biological+pathways+relevant+for+kidney+function&amp;author=C.+Pattaro&amp;author=A.+Teumer&amp;author=M.+Gorski&amp;author=A.Y.+Chu&amp;author=M.+Li&amp;volume=7&amp;publication_year=2016&amp;pages=10023&amp;pmid=26831199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation" id="sref9">Abecasis G.R., Auton A., Brooks L.D., DePristo M.A., Durbin R.M., Handsaker R.E., Kang H.M., Marth G.T., McVean G.A., The 1000 Genomes Project Consortium  An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=G.R.+Abecasis&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=M.A.+DePristo&amp;author=R.M.+Durbin&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation" id="sref10">Delaneau O., Marchini J., Zagury J.F. A linear complexity phasing method for thousands of genomes. <span><span class="ref-journal">Nat. Methods. </span>2011;<span class="ref-vol">9</span>:179–181.</span> [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes&amp;author=O.+Delaneau&amp;author=J.+Marchini&amp;author=J.F.+Zagury&amp;volume=9&amp;publication_year=2011&amp;pages=179-181&amp;pmid=22138821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation" id="sref11">Howie B., Fuchsberger C., Stephens M., Marchini J., Abecasis G.R. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B.+Howie&amp;author=C.+Fuchsberger&amp;author=M.+Stephens&amp;author=J.+Marchini&amp;author=G.R.+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation" id="sref12">Frazer K.A., Ballinger D.G., Cox D.R., Hinds D.A., Stuve L.L., Gibbs R.A., Belmont J.W., Boudreau A., Hardenbol P., Leal S.M., International HapMap Consortium  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=K.A.+Frazer&amp;author=D.G.+Ballinger&amp;author=D.R.+Cox&amp;author=D.A.+Hinds&amp;author=L.L.+Stuve&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation" id="sref13">Howie B.N., Donnelly P., Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=B.N.+Howie&amp;author=P.+Donnelly&amp;author=J.+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation" id="sref14">Winkler T.W., Day F.R., Croteau-Chonka D.C., Wood A.R., Locke A.E., Mägi R., Ferreira T., Fall T., Graff M., Justice A.E., Genetic Investigation of Anthropometric Traits&nbsp;(GIANT) Consortium  Quality control and conduct of genome-wide association meta-analyses. <span><span class="ref-journal">Nat. Protoc. </span>2014;<span class="ref-vol">9</span>:1192–1212.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4083217/">PMC free article</a>]</span> [<a href="/pubmed/24762786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc.&amp;title=Quality+control+and+conduct+of+genome-wide+association+meta-analyses&amp;author=T.W.+Winkler&amp;author=F.R.+Day&amp;author=D.C.+Croteau-Chonka&amp;author=A.R.+Wood&amp;author=A.E.+Locke&amp;volume=9&amp;publication_year=2014&amp;pages=1192-1212&amp;pmid=24762786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation" id="sref15">National Kidney Foundation  K/DOQI clinical practice guidelines for chronic kidney disease: evaluation, classification, and stratification. <span><span class="ref-journal">Am. J. Kidney Dis. </span>2002;<span class="ref-vol">39</span>(2 Suppl 1):S1–S266.</span> [<a href="/pubmed/11904577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Kidney+Dis.&amp;title=K/DOQI+clinical+practice+guidelines+for+chronic+kidney+disease:+evaluation,+classification,+and+stratification&amp;volume=39&amp;issue=2+Suppl+1&amp;publication_year=2002&amp;pages=S1-S266&amp;pmid=11904577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation" id="sref16">Devlin B., Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation" id="sref17">Mägi R., Morris A.P. GWAMA: software for genome-wide association meta-analysis. <span><span class="ref-journal">BMC Bioinformatics. </span>2010;<span class="ref-vol">11</span>:288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2893603/">PMC free article</a>]</span> [<a href="/pubmed/20509871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=GWAMA:+software+for+genome-wide+association+meta-analysis&amp;author=R.+Mägi&amp;author=A.P.+Morris&amp;volume=11&amp;publication_year=2010&amp;pages=288&amp;pmid=20509871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation" id="sref18">Ioannidis J.P.A., Patsopoulos N.A., Evangelou E. Heterogeneity in meta-analyses of genome-wide association investigations. <span><span class="ref-journal">PLoS ONE. </span>2007;<span class="ref-vol">2</span>:e841.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950790/">PMC free article</a>]</span> [<a href="/pubmed/17786212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Heterogeneity+in+meta-analyses+of+genome-wide+association+investigations&amp;author=J.P.A.+Ioannidis&amp;author=N.A.+Patsopoulos&amp;author=E.+Evangelou&amp;volume=2&amp;publication_year=2007&amp;pages=e841&amp;pmid=17786212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation" id="sref19">Machiela M.J., Chanock S.J. LDlink: a web-based application for exploring population-specific haplotype structure and linking correlated alleles of possible functional variants. <span><span class="ref-journal">Bioinformatics. </span>2015;<span class="ref-vol">31</span>:3555–3557.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4626747/">PMC free article</a>]</span> [<a href="/pubmed/26139635" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LDlink:+a+web-based+application+for+exploring+population-specific+haplotype+structure+and+linking+correlated+alleles+of+possible+functional+variants&amp;author=M.J.+Machiela&amp;author=S.J.+Chanock&amp;volume=31&amp;publication_year=2015&amp;pages=3555-3557&amp;pmid=26139635&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation" id="sref20">Willer C.J., Li Y., Abecasis G.R. METAL: fast and&nbsp;efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and&nbsp;efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation" id="sref21">Kang H.M., Sul J.H., Service S.K., Zaitlen N.A., Kong S.Y., Freimer N.B., Sabatti C., Eskin E. Variance component model to account for sample structure in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:348–354.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3092069/">PMC free article</a>]</span> [<a href="/pubmed/20208533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Variance+component+model+to+account+for+sample+structure+in+genome-wide+association+studies&amp;author=H.M.+Kang&amp;author=J.H.+Sul&amp;author=S.K.+Service&amp;author=N.A.+Zaitlen&amp;author=S.Y.+Kong&amp;volume=42&amp;publication_year=2010&amp;pages=348-354&amp;pmid=20208533&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation" id="sref22">Morris A.P. Transethnic meta-analysis of genomewide association studies. <span><span class="ref-journal">Genet. Epidemiol. </span>2011;<span class="ref-vol">35</span>:809–822.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3460225/">PMC free article</a>]</span> [<a href="/pubmed/22125221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Transethnic+meta-analysis+of+genomewide+association+studies&amp;author=A.P.+Morris&amp;volume=35&amp;publication_year=2011&amp;pages=809-822&amp;pmid=22125221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation" id="sref23">Franceschini N., van Rooij F.J.A., Prins B.P., Feitosa M.F., Karakas M., Eckfeldt J.H., Folsom A.R., Kopp J., Vaez A., Andrews J.S., LifeLines Cohort Study  Discovery and fine mapping of serum protein loci through transethnic meta-analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2012;<span class="ref-vol">91</span>:744–753.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3484648/">PMC free article</a>]</span> [<a href="/pubmed/23022100" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Discovery+and+fine+mapping+of+serum+protein+loci+through+transethnic+meta-analysis&amp;author=N.+Franceschini&amp;author=F.J.A.+van+Rooij&amp;author=B.P.+Prins&amp;author=M.F.+Feitosa&amp;author=M.+Karakas&amp;volume=91&amp;publication_year=2012&amp;pages=744-753&amp;pmid=23022100&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation" id="sref24">Wang X., Chua H.X., Chen P., Ong R.T., Sim X., Zhang W., Takeuchi F., Liu X., Khor C.C., Tay W.T. Comparing methods for performing trans-ethnic meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum. Mol. Genet. </span>2013;<span class="ref-vol">22</span>:2303–2311.</span> [<a href="/pubmed/23406875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Comparing+methods+for+performing+trans-ethnic+meta-analysis+of+genome-wide+association+studies&amp;author=X.+Wang&amp;author=H.X.+Chua&amp;author=P.+Chen&amp;author=R.T.+Ong&amp;author=X.+Sim&amp;volume=22&amp;publication_year=2013&amp;pages=2303-2311&amp;pmid=23406875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation" id="sref25">Maller J.B., McVean G., Byrnes J., Vukcevic D., Palin K., Su Z., Howson J.M., Auton A., Myers S., Morris A., Wellcome Trust Case Control Consortium  Bayesian refinement of association signals for 14 loci in 3 common diseases. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:1294–1301.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3791416/">PMC free article</a>]</span> [<a href="/pubmed/23104008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Bayesian+refinement+of+association+signals+for+14+loci+in+3+common+diseases&amp;author=J.B.+Maller&amp;author=G.+McVean&amp;author=J.+Byrnes&amp;author=D.+Vukcevic&amp;author=K.+Palin&amp;volume=44&amp;publication_year=2012&amp;pages=1294-1301&amp;pmid=23104008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation" id="sref26">Bernstein B.E., Stamatoyannopoulos J.A., Costello J.F., Ren B., Milosavljevic A., Meissner A., Kellis M., Marra M.A., Beaudet A.L., Ecker J.R. The NIH Roadmap Epigenomics Mapping Consortium. <span><span class="ref-journal">Nat. Biotechnol. </span>2010;<span class="ref-vol">28</span>:1045–1048.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3607281/">PMC free article</a>]</span> [<a href="/pubmed/20944595" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=The+NIH+Roadmap+Epigenomics+Mapping+Consortium&amp;author=B.E.+Bernstein&amp;author=J.A.+Stamatoyannopoulos&amp;author=J.F.+Costello&amp;author=B.+Ren&amp;author=A.+Milosavljevic&amp;volume=28&amp;publication_year=2010&amp;pages=1045-1048&amp;pmid=20944595&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation" id="sref27">ENCODE Project Consortium  An integrated encyclopedia of DNA elements in the human genome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:57–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3439153/">PMC free article</a>]</span> [<a href="/pubmed/22955616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+encyclopedia+of+DNA+elements+in+the+human+genome&amp;volume=489&amp;publication_year=2012&amp;pages=57-74&amp;pmid=22955616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation" id="sref28">Pasquali L., Gaulton K.J., Rodríguez-Seguí S.A., Mularoni L., Miguel-Escalada I., Akerman I., Tena J.J., Morán I., Gómez-Marín C., van de Bunt M. Pancreatic islet enhancer clusters enriched in type 2 diabetes risk-associated variants. <span><span class="ref-journal">Nat. Genet. </span>2014;<span class="ref-vol">46</span>:136–143.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3935450/">PMC free article</a>]</span> [<a href="/pubmed/24413736" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Pancreatic+islet+enhancer+clusters+enriched+in+type+2+diabetes+risk-associated+variants&amp;author=L.+Pasquali&amp;author=K.J.+Gaulton&amp;author=S.A.+Rodríguez-Seguí&amp;author=L.+Mularoni&amp;author=I.+Miguel-Escalada&amp;volume=46&amp;publication_year=2014&amp;pages=136-143&amp;pmid=24413736&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation" id="sref29">Pickrell J.K. Joint analysis of functional genomic data and genome-wide association studies of 18 human traits. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2014;<span class="ref-vol">94</span>:559–573.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3980523/">PMC free article</a>]</span> [<a href="/pubmed/24702953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Joint+analysis+of+functional+genomic+data+and+genome-wide+association+studies+of+18+human+traits&amp;author=J.K.+Pickrell&amp;volume=94&amp;publication_year=2014&amp;pages=559-573&amp;pmid=24702953&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation" id="sref30">Lin Y.R., Kim K., Yang Y., Ivessa A., Sadoshima J., Park Y. Regulation of longevity by regulator of G-protein signaling protein, Loco. <span><span class="ref-journal">Aging Cell. </span>2011;<span class="ref-vol">10</span>:438–447.</span> [<a href="/pubmed/21255223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Aging+Cell&amp;title=Regulation+of+longevity+by+regulator+of+G-protein+signaling+protein,+Loco&amp;author=Y.R.+Lin&amp;author=K.+Kim&amp;author=Y.+Yang&amp;author=A.+Ivessa&amp;author=J.+Sadoshima&amp;volume=10&amp;publication_year=2011&amp;pages=438-447&amp;pmid=21255223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation" id="sref31">Cechova S., Zeng Q., Billaud M., Mutchler S., Rudy C.K., Straub A.C., Chi L., Chan F.R., Hu J., Griffiths R. Loss of collectrin, an angiotensin-converting enzyme 2 homolog, uncouples endothelial nitric oxide synthase and causes hypertension and vascular dysfunction. <span><span class="ref-journal">Circulation. </span>2013;<span class="ref-vol">128</span>:1770–1780.</span> [<a href="/pubmed/24048198" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Loss+of+collectrin,+an+angiotensin-converting+enzyme+2+homolog,+uncouples+endothelial+nitric+oxide+synthase+and+causes+hypertension+and+vascular+dysfunction&amp;author=S.+Cechova&amp;author=Q.+Zeng&amp;author=M.+Billaud&amp;author=S.+Mutchler&amp;author=C.K.+Rudy&amp;volume=128&amp;publication_year=2013&amp;pages=1770-1780&amp;pmid=24048198&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation" id="sref32">Yang Y., Parsons K.K., Chi L., Malakauskas S.M., Le T.H. Glutathione S-transferase-micro1 regulates vascular smooth muscle cell proliferation, migration, and oxidative stress. <span><span class="ref-journal">Hypertension. </span>2009;<span class="ref-vol">54</span>:1360–1368.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2783903/">PMC free article</a>]</span> [<a href="/pubmed/19822795" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertension&amp;title=Glutathione+S-transferase-micro1+regulates+vascular+smooth+muscle+cell+proliferation,+migration,+and+oxidative+stress&amp;author=Y.+Yang&amp;author=K.K.+Parsons&amp;author=L.+Chi&amp;author=S.M.+Malakauskas&amp;author=T.H.+Le&amp;volume=54&amp;publication_year=2009&amp;pages=1360-1368&amp;pmid=19822795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation" id="sref33">Deshmukh H.A., Palmer C.N., Morris A.D., Colhoun H.M. Investigation of known estimated glomerular filtration rate loci in patients with type 2 diabetes. <span><span class="ref-journal">Diabet. Med. </span>2013;<span class="ref-vol">30</span>:1230–1235.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4204276/">PMC free article</a>]</span> [<a href="/pubmed/23586973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabet.+Med.&amp;title=Investigation+of+known+estimated+glomerular+filtration+rate+loci+in+patients+with+type+2+diabetes&amp;author=H.A.+Deshmukh&amp;author=C.N.+Palmer&amp;author=A.D.+Morris&amp;author=H.M.+Colhoun&amp;volume=30&amp;publication_year=2013&amp;pages=1230-1235&amp;pmid=23586973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation" id="sref34">Sveinbjornsson G., Mikaelsdottir E., Palsson R., Indridason O.S., Holm H., Jonasdottir A., Helgason A., Sigurdsson S., Jonasdottir A., Sigurdsson A. Rare mutations associating with serum creatinine and chronic kidney disease. <span><span class="ref-journal">Hum. Mol. Genet. </span>2014;<span class="ref-vol">23</span>:6935–6943.</span> [<a href="/pubmed/25082825" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Rare+mutations+associating+with+serum+creatinine+and+chronic+kidney+disease&amp;author=G.+Sveinbjornsson&amp;author=E.+Mikaelsdottir&amp;author=R.+Palsson&amp;author=O.S.+Indridason&amp;author=H.+Holm&amp;volume=23&amp;publication_year=2014&amp;pages=6935-6943&amp;pmid=25082825&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation" id="sref35">Succurro E., Arturi F., Lugarà M., Grembiale A., Fiorentino T.V., Caruso V., Andreozzi F., Sciacqua A., Hribal M.L., Perticone F., Sesti G. One-hour postload plasma glucose levels are associated with kidney dysfunction. <span><span class="ref-journal">Clin. J. Am. Soc. Nephrol. </span>2010;<span class="ref-vol">5</span>:1922–1927.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3001771/">PMC free article</a>]</span> [<a href="/pubmed/20595688" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+J.+Am.+Soc.+Nephrol.&amp;title=One-hour+postload+plasma+glucose+levels+are+associated+with+kidney+dysfunction&amp;author=E.+Succurro&amp;author=F.+Arturi&amp;author=M.+Lugarà&amp;author=A.+Grembiale&amp;author=T.V.+Fiorentino&amp;volume=5&amp;publication_year=2010&amp;pages=1922-1927&amp;pmid=20595688&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation" id="sref36">Reiss A.B., Voloshyna I., De Leon J., Miyawaki N., Mattana J. Cholesterol Metabolism in CKD. <span><span class="ref-journal">Am. J. Kidney Dis. </span>2015;<span class="ref-vol">66</span>:1071–1082.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4658227/">PMC free article</a>]</span> [<a href="/pubmed/26337134" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Kidney+Dis.&amp;title=Cholesterol+Metabolism+in+CKD&amp;author=A.B.+Reiss&amp;author=I.+Voloshyna&amp;author=J.+De+Leon&amp;author=N.+Miyawaki&amp;author=J.+Mattana&amp;volume=66&amp;publication_year=2015&amp;pages=1071-1082&amp;pmid=26337134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation" id="sref37">Chen S., Bellew C., Yao X., Stefkova J., Dipp S., Saifudeen Z., Bachvarov D., El-Dahr S.S. Histone deacetylase (HDAC) activity is critical for embryonic kidney gene expression, growth, and differentiation. <span><span class="ref-journal">J.&nbsp;Biol. Chem. </span>2011;<span class="ref-vol">286</span>:32775–32789.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173185/">PMC free article</a>]</span> [<a href="/pubmed/21778236" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Biol.+Chem.&amp;title=Histone+deacetylase+(HDAC)+activity+is+critical+for+embryonic+kidney+gene+expression,+growth,+and+differentiation&amp;author=S.+Chen&amp;author=C.+Bellew&amp;author=X.+Yao&amp;author=J.+Stefkova&amp;author=S.+Dipp&amp;volume=286&amp;publication_year=2011&amp;pages=32775-32789&amp;pmid=21778236&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation" id="sref38">Brilli L.L., Swanhart L.M., de Caestecker M.P., Hukriede N.A. HDAC inhibitors in kidney development and disease. <span><span class="ref-journal">Pediatr. Nephrol. </span>2013;<span class="ref-vol">28</span>:1909–1921.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3751322/">PMC free article</a>]</span> [<a href="/pubmed/23052657" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.+Nephrol.&amp;title=HDAC+inhibitors+in+kidney+development+and+disease&amp;author=L.L.+Brilli&amp;author=L.M.+Swanhart&amp;author=M.P.+de+Caestecker&amp;author=N.A.+Hukriede&amp;volume=28&amp;publication_year=2013&amp;pages=1909-1921&amp;pmid=23052657&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation" id="sref39">Keyser P., Borge-Renberg K., Hultmark D. The Drosophila NFAT homolog is involved in salt stress tolerance. <span><span class="ref-journal">Insect Biochem. Mol. Biol. </span>2007;<span class="ref-vol">37</span>:356–362.</span> [<a href="/pubmed/17368199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Insect+Biochem.+Mol.+Biol.&amp;title=The+Drosophila+NFAT+homolog+is+involved+in+salt+stress+tolerance&amp;author=P.+Keyser&amp;author=K.+Borge-Renberg&amp;author=D.+Hultmark&amp;volume=37&amp;publication_year=2007&amp;pages=356-362&amp;pmid=17368199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation" id="sref40">Salzler H.R., Griffiths R., Ruiz P., Chi L., Frey C., Marchuk D.A., Rockman H.A., Le T.H. Hypertension and albuminuria in chronic kidney disease mapped to a mouse chromosome 11 locus. <span><span class="ref-journal">Kidney Int. </span>2007;<span class="ref-vol">72</span>:1226–1232.</span> [<a href="/pubmed/17851470" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int.&amp;title=Hypertension+and+albuminuria+in+chronic+kidney+disease+mapped+to+a+mouse+chromosome+11+locus&amp;author=H.R.+Salzler&amp;author=R.+Griffiths&amp;author=P.+Ruiz&amp;author=L.+Chi&amp;author=C.+Frey&amp;volume=72&amp;publication_year=2007&amp;pages=1226-1232&amp;pmid=17851470&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation" id="sref41">Dickson S.P., Wang K., Krantz I., Hakonarson H., Goldstein D.B. Rare variants create synthetic genome-wide associations. <span><span class="ref-journal">PLoS Biol. </span>2010;<span class="ref-vol">8</span>:e1000294.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2811148/">PMC free article</a>]</span> [<a href="/pubmed/20126254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol.&amp;title=Rare+variants+create+synthetic+genome-wide+associations&amp;author=S.P.+Dickson&amp;author=K.+Wang&amp;author=I.+Krantz&amp;author=H.+Hakonarson&amp;author=D.B.+Goldstein&amp;volume=8&amp;publication_year=2010&amp;pages=e1000294&amp;pmid=20126254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation" id="sref42">Abecasis G.R., Altshuler D., Auton A., Brooks L.D., Durbin R.M., Gibbs R.A., Hurles M.E., McVean G.A., The 1000 Genomes Project Consortium  A map of human genome variation from population scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population+scale+sequencing&amp;author=G.R.+Abecasis&amp;author=D.+Altshuler&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=R.M.+Durbin&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation" id="sref43">Trynka G., Sandor C., Han B., Xu H., Stranger B.E., Liu X.S., Raychaudhuri S. Chromatin marks identify critical cell types for fine mapping complex trait variants. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:124–130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3826950/">PMC free article</a>]</span> [<a href="/pubmed/23263488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Chromatin+marks+identify+critical+cell+types+for+fine+mapping+complex+trait+variants&amp;author=G.+Trynka&amp;author=C.+Sandor&amp;author=B.+Han&amp;author=H.+Xu&amp;author=B.E.+Stranger&amp;volume=45&amp;publication_year=2013&amp;pages=124-130&amp;pmid=23263488&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation" id="sref44">Parker S.C., Stitzel M.L., Taylor D.L., Orozco J.M., Erdos M.R., Akiyama J.A., van Bueren K.L., Chines P.S., Narisu N., Black B.L., NISC Comparative Sequencing Program. National Institutes of Health Intramural Sequencing Center Comparative Sequencing Program Authors. NISC Comparative Sequencing Program Authors  Chromatin stretch enhancer states drive cell-specific gene regulation and harbor human disease risk variants. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2013;<span class="ref-vol">110</span>:17921–17926.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3816444/">PMC free article</a>]</span> [<a href="/pubmed/24127591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Chromatin+stretch+enhancer+states+drive+cell-specific+gene+regulation+and+harbor+human+disease+risk+variants&amp;author=S.C.+Parker&amp;author=M.L.+Stitzel&amp;author=D.L.+Taylor&amp;author=J.M.+Orozco&amp;author=M.R.+Erdos&amp;volume=110&amp;publication_year=2013&amp;pages=17921-17926&amp;pmid=24127591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation" id="sref45">Gusev A., Lee S.H., Trynka G., Finucane H., Vilhjálmsson B.J., Xu H., Zang C., Ripke S., Bulik-Sullivan B., Stahl E., Schizophrenia Working Group of the Psychiatric Genomics Consortium. SWE-SCZ Consortium. Schizophrenia Working Group of the Psychiatric Genomics Consortium. SWE-SCZ Consortium  Partitioning heritability of regulatory and cell-type-specific variants across 11 common diseases. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2014;<span class="ref-vol">95</span>:535–552.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4225595/">PMC free article</a>]</span> [<a href="/pubmed/25439723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Partitioning+heritability+of+regulatory+and+cell-type-specific+variants+across+11+common+diseases&amp;author=A.+Gusev&amp;author=S.H.+Lee&amp;author=G.+Trynka&amp;author=H.+Finucane&amp;author=B.J.+Vilhjálmsson&amp;volume=95&amp;publication_year=2014&amp;pages=535-552&amp;pmid=25439723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation" id="sref46">Kichaev G., Pasaniuc B. Leveraging functional-annotation data in trans-ethnic fine-mapping studies. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2015;<span class="ref-vol">97</span>:260–271.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4573268/">PMC free article</a>]</span> [<a href="/pubmed/26189819" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Leveraging+functional-annotation+data+in+trans-ethnic+fine-mapping+studies&amp;author=G.+Kichaev&amp;author=B.+Pasaniuc&amp;volume=97&amp;publication_year=2015&amp;pages=260-271&amp;pmid=26189819&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation" id="sref47">Bigazzi R., Bianchi S., Baldari D., Sgherri G., Baldari G., Campese V.M. Microalbuminuria in salt-sensitive patients. A marker for renal and cardiovascular risk factors. <span><span class="ref-journal">Hypertension. </span>1994;<span class="ref-vol">23</span>:195–199.</span> [<a href="/pubmed/8307628" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertension&amp;title=Microalbuminuria+in+salt-sensitive+patients.+A+marker+for+renal+and+cardiovascular+risk+factors&amp;author=R.+Bigazzi&amp;author=S.+Bianchi&amp;author=D.+Baldari&amp;author=G.+Sgherri&amp;author=G.+Baldari&amp;volume=23&amp;publication_year=1994&amp;pages=195-199&amp;pmid=8307628&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation" id="sref48">Bihorac A., Tezcan H., Ozener C., Oktay A., Akoglu E. Association between salt sensitivity and target organ damage in essential hypertension. <span><span class="ref-journal">Am. J. Hypertens. </span>2000;<span class="ref-vol">13</span>:864–872.</span> [<a href="/pubmed/10950394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hypertens.&amp;title=Association+between+salt+sensitivity+and+target+organ+damage+in+essential+hypertension&amp;author=A.+Bihorac&amp;author=H.+Tezcan&amp;author=C.+Ozener&amp;author=A.+Oktay&amp;author=E.+Akoglu&amp;volume=13&amp;publication_year=2000&amp;pages=864-872&amp;pmid=10950394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation" id="sref49">Barba G., Galletti F., Cappuccio F.P., Siani A., Venezia A., Versiero M., Della Valle E., Sorrentino P., Tarantino G., Farinaro E., Strazzullo P. Incidence of hypertension in individuals with different blood pressure salt-sensitivity: results of a 15-year follow-up study. <span><span class="ref-journal">J.&nbsp;Hypertens. </span>2007;<span class="ref-vol">25</span>:1465–1471.</span> [<a href="/pubmed/17563570" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Hypertens.&amp;title=Incidence+of+hypertension+in+individuals+with+different+blood+pressure+salt-sensitivity:+results+of+a+15-year+follow-up+study&amp;author=G.+Barba&amp;author=F.+Galletti&amp;author=F.P.+Cappuccio&amp;author=A.+Siani&amp;author=A.+Venezia&amp;volume=25&amp;publication_year=2007&amp;pages=1465-1471&amp;pmid=17563570&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation" id="sref50">Thurau K., Schnermann J. The sodium concentration in the macula densa cells as a regulating factor for glomerular filtration (micropuncture experiments) <span><span class="ref-journal">Klin. Wochenschr. </span>1965;<span class="ref-vol">43</span>:410–413.</span> [<a href="/pubmed/14333330" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Klin.+Wochenschr.&amp;title=The+sodium+concentration+in+the+macula+densa+cells+as+a+regulating+factor+for+glomerular+filtration+(micropuncture+experiments)&amp;author=K.+Thurau&amp;author=J.+Schnermann&amp;volume=43&amp;publication_year=1965&amp;pages=410-413&amp;pmid=14333330&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation" id="sref51">Wilcox C.S., Welch W.J. TGF and nitric oxide: effects of salt intake and salt-sensitive hypertension. <span><span class="ref-journal">Kidney Int. Suppl. </span>1996;<span class="ref-vol">55</span>:S9–S13.</span> [<a href="/pubmed/8743503" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int.+Suppl.&amp;title=TGF+and+nitric+oxide:+effects+of+salt+intake+and+salt-sensitive+hypertension&amp;author=C.S.+Wilcox&amp;author=W.J.+Welch&amp;volume=55&amp;publication_year=1996&amp;pages=S9-S13&amp;pmid=8743503&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">52. <span class="element-citation" id="sref52">Persson A.E.G., Lai E.Y., Gao X., Carlström M., Patzak A. Interactions between adenosine, angiotensin II and nitric oxide on the afferent arteriole influence sensitivity of the tubuloglomerular feedback. <span><span class="ref-journal">Front. Physiol. </span>2013;<span class="ref-vol">4</span>:187.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3714451/">PMC free article</a>]</span> [<a href="/pubmed/23882224" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Physiol.&amp;title=Interactions+between+adenosine,+angiotensin+II+and+nitric+oxide+on+the+afferent+arteriole+influence+sensitivity+of+the+tubuloglomerular+feedback&amp;author=A.E.G.+Persson&amp;author=E.Y.+Lai&amp;author=X.+Gao&amp;author=M.+Carlström&amp;author=A.+Patzak&amp;volume=4&amp;publication_year=2013&amp;pages=187&amp;pmid=23882224&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">53. <span class="element-citation" id="sref53">Lu Y., Wei J., Stec D.E., Roman R.J., Ge Y., Cheng L., Liu E.Y., Zhang J., Hansen P.B., Fan F. Macula Densa Nitric Oxide Synthase 1β Protects against Salt-Sensitive Hypertension. <span><span class="ref-journal">J.&nbsp;Am. Soc. Nephrol. </span>2016;<span class="ref-vol">27</span>:2346–2356.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4978040/">PMC free article</a>]</span> [<a href="/pubmed/26647426" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Am.+Soc.+Nephrol.&amp;title=Macula+Densa+Nitric+Oxide+Synthase+1β+Protects+against+Salt-Sensitive+Hypertension&amp;author=Y.+Lu&amp;author=J.+Wei&amp;author=D.E.+Stec&amp;author=R.J.+Roman&amp;author=Y.+Ge&amp;volume=27&amp;publication_year=2016&amp;pages=2346-2356&amp;pmid=26647426&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib54">54. <span class="element-citation" id="sref54">Kanbay M., Chen Y., Solak Y., Sanders P.W. Mechanisms and consequences of salt sensitivity and dietary salt intake. <span><span class="ref-journal">Curr. Opin. Nephrol. Hypertens. </span>2011;<span class="ref-vol">20</span>:37–43.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3089903/">PMC free article</a>]</span> [<a href="/pubmed/21088577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Nephrol.+Hypertens.&amp;title=Mechanisms+and+consequences+of+salt+sensitivity+and+dietary+salt+intake&amp;author=M.+Kanbay&amp;author=Y.+Chen&amp;author=Y.+Solak&amp;author=P.W.+Sanders&amp;volume=20&amp;publication_year=2011&amp;pages=37-43&amp;pmid=21088577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib55">55. <span class="element-citation" id="sref55">Morris R.C., Jr., Schmidlin O., Sebastian A., Tanaka M., Kurtz T.W. Vasodysfunction that involves renal vasodysfunction, not abnormally increased renal retention of sodium, accounts for the initiation of salt-induced hypertension. <span><span class="ref-journal">Circulation. </span>2016;<span class="ref-vol">133</span>:881–893.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4778403/">PMC free article</a>]</span> [<a href="/pubmed/26927006" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Vasodysfunction+that+involves+renal+vasodysfunction,+not+abnormally+increased+renal+retention+of+sodium,+accounts+for+the+initiation+of+salt-induced+hypertension&amp;author=R.C.+Morris&amp;author=O.+Schmidlin&amp;author=A.+Sebastian&amp;author=M.+Tanaka&amp;author=T.W.+Kurtz&amp;volume=133&amp;publication_year=2016&amp;pages=881-893&amp;pmid=26927006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#abs0010title">Abstract</a></li><li><a href="#sec1title">Introduction</a></li><li><a href="#sec2title">Subjects and Methods</a></li><li><a href="#sec3title">Results</a></li><li><a href="#sec4title">Discussion</a></li><li><a href="#ack0010title">Acknowledgments</a></li><li><a href="#idm140715211864064title">Notes</a></li><li><a href="#idm140715217273808title">Footnotes</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#idm140715179312336title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5011075/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5011075/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5011075/pdf/main.pdf">PDF (823K)</a></li> | <li><a href="#" data-citationid="PMC5011075" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5011075%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5011075%2F&amp;text=Trans-ethnic%20Fine%20Mapping%20Highlights%20Kidney-Function%20Genes%20Linked%20to%20Salt%20Sensitivity" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5011075%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27588450" data-aiid="5011075" data-aid="5011075" data-iid="275125" data-domainid="203" data-domain="ajhg" data-accid="PMC5011075" data-md5="9309dce5e390b7cacd6d34383c0520a8">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input name="absid" id="absid" value="5011075" type="hidden"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:'#favUL',starExists:true,createCollectionUrl:'#',manageCollectionsUrl:'/myncbi/collections/'"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input class="button_apply" name="addtocollection" value="1" type="submit"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-6" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-6" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/30604766" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Trans-ethnic kidney function association study reveals putative causal genes and effects on kidney-specific disease aetiologies.</a><span class="source">[Nat Commun. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Morris AP, Le TH, Wu H, Akbarov A, van der Most PJ, Hemani G, Smith GD, Mahajan A, Gaulton KJ, Nadkarni GN, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nat Commun. 2019 Jan 3; 10(1):29. Epub 2019 Jan 3.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/28911207" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution.</a><span class="source">[Hum Mol Genet. 2017]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Mägi R, Horikoshi M, Sofer T, Mahajan A, Kitajima H, Franceschini N, McCarthy MI, COGENT-Kidney Consortium, T2D-GENES Consortium., Morris AP. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Hum Mol Genet. 2017 Sep 15; 26(18):3639-3650. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/31152163" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A catalog of genetic loci associated with kidney function from analyses of a million individuals.</a><span class="source">[Nat Genet. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Wuttke M, Li Y, Li M, Sieber KB, Feitosa MF, Gorski M, Tin A, Wang L, Chu AY, Hoppmann A, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nat Genet. 2019 Jun; 51(6):957-972. Epub 2019 May 31.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/29180396" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">The &lt;i&gt;UMOD&lt;/i&gt; Locus: Insights into the Pathogenesis and Prognosis of Kidney Disease.</a><span class="source">[J Am Soc Nephrol. 2018]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Devuyst O, Pattaro C. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">J Am Soc Nephrol. 2018 Mar; 29(3):713-726. Epub 2017 Nov 27.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/21691125" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Chronic kidney disease: novel insights from genome-wide association studies.</a><span class="source">[Kidney Blood Press Res. 2011]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-11" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Böger CA, Heid IM. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Kidney Blood Press Res. 2011; 34(4):225-34. Epub 2011 Jun 21.</em>
</div>
</li>
</ul>
<a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=27588450&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=27588450&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-12" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-12" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6698888/" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A catalog of genetic loci associated with kidney function from analyses of a million individuals</a><span class="source">[Nature genetics. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Wuttke M, Li Y, Li M, Sieber KB, Feitosa MF, Gorski M, Tin A, Wang L, Chu AY, Hoppmann A, Kirsten H, Giri A, Chai JF, Sveinbjornsson G, Tayo BO, Nutile T, Fuchsberger C, Marten J, Cocca M, Ghasemi S, Xu Y, Horn K, Noce D, van der Most PJ, Sedaghat S, Yu Z, Akiyama M, Afaq S, Ahluwalia TS, Almgren P, Amin N, Ärnlöv J, Bakker SJ, Bansal N, Baptista D, Bergmann S, Biggs ML, Biino G, Boehnke M, Boerwinkle E, Boissel M, Bottinger EP, Boutin TS, Brenner H, Brumat M, Burkhardt R, Butterworth AS, Campana E, Campbell A, Campbell H, Canouil M, Carroll RJ, Catamo E, Chambers JC, Chee ML, Chee ML, Chen X, Cheng CY, Cheng Y, Christensen K, Cifkova R, Ciullo M, Pina Concas M, Cook JP, Coresh J, Corre T, Sala CF, Cusi D, Danesh J, Daw EW, de Borst MH, De Grandi A, de Mutsert R, de Vries AP, Degenhardt F, Delgado G, Demirkan A, Di Angelantonio E, Dittrich K, Divers J, Dorajoo R, Eckardt KU, Ehret G, Elliott P, Endlich K, Evans MK, Felix JF, Foo VH, Franco OH, Franke A, Freedman BI, Freitag-Wolf S, Friedlander Y, Froguel P, Gansevoort RT, Gao H, Gasparini P, Gaziano JM, Giedraitis V, Gieger C, Girotto G, Giulianini F, Gögele M, Gordon SD, Gudbjartsson DF, Gudnason V, Haller T, Hamet P, Harris TB, Hartman CA, Hayward C, Hellwege JN, Heng CK, Hickst AA, Hofer E, Huang W, Hutri-Kähönen N, Hwang SJ, ikram MA, indridason OS, Ingelsson E, ising M, Jaddoe VW, Jakobsdottir J, Jonas JB, Joshi PK, Shilpa Josyula N, Jung B, Kähönen M, Kamatani Y, Kammerer CM, Kanai M, Kastarinen M, Kerr SM, Khor CC, Kiess W, Kleber ME, Koenig W, Kooner JS, Körner A, Kovacs P, Kraja AT, Krajcoviechova A, Kramer H, Krämer BK, Kronenberg F, Kubo M, Kühnel B, Kuokkanen M, Kuusisto J, La Bianca M, Laakso M, Lange LA, Langefeld CD, Jen-Mai Lee J, Lehne B, Lehtimäki T, Lieb W, Cohort Study L, Lim SC, Lind L, Lindgren CM, Liu J, Liu J, Loeffler M, Loos RJ, Lucae S, Ann Lukas M, Lyytikäinen LP, Mägi R, Magnusson PK, Mahajan A, Martin NG, Martins J, März W, Mascalzoni D, Matsuda K, Christa Meisinger, Meitinger T, Melander O, Metspalu A, Mikaelsdottir EK, Milaneschi Y, Miliku K, Mishra PP, Veteran Program VA, Mohlke KL, Mononen N, Montgomery GW, Mook-Kanamori DO, Mychaleckyj JC, Nadkarni GN, Nalls MA, Nauck M, Nikus K, Ning B, Nolte IM, Noordam R, O’Connell J, O’Donoghue ML, Olafsson I, Oldehinkel AJ, Orho-Melander M, Ouwehand WH, Padmanabhan S, Palmer ND, Palsson R, Penninx BW, Perls T, Perola M, Pirastu M, Pirastu N, Pistis G, Podgornaia AI, Polasek O, Ponte B, Porteous DJ, Poulain T, Pramstaller PP, Preuss MH, Prins BP, Province MA, Rabelink TJ, Raffield LM, Raitakari OT, Reilly DF, Rettig R, Rheinberger M, Rice KM, Ridker PM, Rivadeneira F, Rizzi F, Roberts DJ, Robino A, Rossing P, Rudan I, Rueedi R, Ruggiero D, Ryan KA, Saba Y, Sabanayagam C, Salomaa V, Salvi E, Saum KU, Schmidt H, Schmidt R, Schöttker B, Schulz CA, Schupf N, Shaffer CM, Shi Y, Smith AV, Smith BH, Soranzo N, Spracklen CN, Strauch K, Stringham HM, Stumvoll M, Svensson PO, Szymczak S, Tai ES, Tajuddin SM, Tan NY, Taylor KD, Teren A, Tham YC, Thiery J, Thio CH, Thomsen H, Thorleifsson G, Toniolo D, Tönjes A, Tremblay J, Tzoulaki I, Uitterlinden AG, Vaccargiu S, van Dam RM, van der Harst P, van Duijn CM, Velez Edward DR, Verweij N, Vogelezang S, Völker Ü, Vollenweider P, Waeber G, Waldenberger M, Wallentin L, Wang YX, Wang C, Waterworth DM, Bin Wei W, White H, Whitfield JB, Wild SH, Wilson JF, Wojczynski MK, Wong C, Wong TY, Xu L, Yang Q, Yasuda M, Yerges-Armstrong LM, Zhang W, Zonderman AB, Rotter JI, Bochud M, Psaty BM, Vitart V, Wilson JG, Dehghan A, Parsa A, Chasman DI, Ho K, Morris AP, Devuyst O, Akilesh S, Pendergrass SA, Sim X, Böger CA, Okada Y, Edwards TL, Snieder H, Stefansson K, Hung AM, Heid IM, Markus Scholz, Teumer A, Köttgen A, Pattaro C. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nature genetics. 2019 May 31; 51(6): 957-972</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6687539/" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Integrated analysis of genomics, longitudinal metabolomics, and Alzheimer’s risk factors among 1,111 cohort participants</a><span class="source">[Genetic epidemiology. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Darst BF, Lu Q, Johnson SC, Engelman CD. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Genetic epidemiology. 2019 May 18; 43(6): 657-674</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6521376/" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association study identifies novel loci for type 2 diabetes-attributed end-stage kidney disease in African Americans</a><span class="source">[Human Genomics. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Guan M, Keaton JM, Dimitrov L, Hicks PJ, Xu J, Palmer ND, Ma L, Das SK, Chen YD, Coresh J, Fornage M, Franceschini N, Kramer H, Langefeld CD, Mychaleckyj JC, Parekh RS, Post WS, Rasmussen-Torvik LJ, Rich SS, Rotter JI, Sedor JR, Thornley-Brown D, Tin A, Wilson JG, Freedman BI, Bowden DW, Ng MC, FIND ConsortiumBobeluA.AHorvathA.APickensA.AKessingB.BWasetaB.BFreedmanB. I.BIKasinathB. S.BSGarciaC.CJeffersonC.CLuetheC.CLangefeldC. D.CDBrownC. S.CSWinklerC.CThornley-BrownD.DWarnockD.DLeeheyD. J.DJBowdenD. W.DWHernandezE.EIppE.EHaririF.FThameemF.FBaroneG.GBrooksG.GJunG.GDunstonG. M.GMAbboudH.HSiliI.IChesterJ.JFondranJ.JLaPageJ.JMolinerosJ.JRiouxJ.JRotterJ.JSchellingJ.JSewemaenewaJ.JTayekJ.JGonzalezJ.JOlsonJ. M.JMBriggsJ. P.JPSedorJ. R.JRWolfordJ.JKrampK.KTaylorK.KGoddardK. A. B.KACardonL.LGetz-FradleyL.LHumbertL.LJonesL.LKaoL.LMeoniL.LPhillipsL. M.LMAguilarM.MBudgettM.MKlagM.MPahlM.MSpainhourM.MSternM.MThompsonM.MSaadM. F.MFSeldinM. F.MFScaviniM.MSmithM. W.MWArarN.NKohnO. F.OFZagerP.PKimmelP. L.PLChakrabortyR.RDuggiralaR.RHansonR.RIgoR.JrRJuanR.RLovelaceR.RParekhR.RRasoolyR.RSpielmanR.RYoungR.RElstonR. C.RCNelsonR. G.RGHansonR. L.RLAdlerS.SIalacciS.SSnyderS.SViveretteS.SWarrenS.SNicholasS. B.SBSatkoS. C.SCO’BrienS. J.SJIyengarS. K.SKRichS. S.SSHostetterT.TWhiteheadT.TShahV.VKnowlerW. C.WCGuoX.XChenY.-D. I.YD. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Human Genomics. 2019 May 15; 13: 21</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6478837/" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Sex-specific and pleiotropic effects underlying kidney function identified from GWAS meta-analysis</a><span class="source">[Nature Communications. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Graham SE, Nielsen JB, Zawistowski M, Zhou W, Fritsche LG, Gabrielsen ME, Skogholt AH, Surakka I, Hornsby WE, Fermin D, Larach DB, Kheterpal S, Brummett CM, Lee S, Kang HM, Abecasis GR, Romundstad S, Hallan S, Sampson MG, Hveem K, Willer CJ. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nature Communications. 2019 Apr 23; 10: 1847</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6467258/" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Multi-ancestry genome-wide gene-smoking interaction study of 387,272 individuals identifies new loci associated with serum lipids</a><span class="source">[Nature genetics. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-17" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Bentley AR, Sung YJ, Brown MR, Winkler TW, Kraja AT, Ntalla I, Schwander K, Chasman DI, Lim E, Deng X, Guo X, Liu J, Lu Y, Cheng CY, Sim X, Vojinovic D, Huffman JE, Musani SK, Li C, Feitosa MF, Richard MA, Noordam R, Baker J, Chen G, Aschard H, Bartz TM, Ding J, Dorajoo R, Manning AK, Rankinen T, Smith AV, Tajuddin SM, Zhao W, Graff M, Alver M, Boissel M, Chai JF, Chen X, Divers J, Evangelou E, Gao C, Goel A, Hagemeijer Y, Harris SE, Hartwig FP, He M, Horimoto AR, Hsu FC, Hung YJ, Jackson AU, Kasturiratne A, Komulainen P, Kühnel B, Leander K, Lin KH, Luan J, Lyytikäinen LP, Matoba N, Nolte IM, Pietzner M, Prins B, Riaz M, Robino A, Said MA, Schupf N, Scott RA, Sofer T, Stančáková A, Takeuchi F, Tayo BO, van der Most PJ, Varga TV, Wang TD, Wang Y, Ware EB, Wen W, Xiang YB, Yanek LR, Zhang W, Zhao JH, Adeyemo A, Afaq S, Amin N, Amini M, Arking DE, Arzumanyan Z, Aung T, Ballantyne C, Barr RG, Bielak LF, Boerwinkle E, Bottinger EP, Broeckel U, Brown M, Cade BE, Campbell A, Canouil M, Charumathi S, Chen YD, Christensen K, COGENT-Kidney Consortium, Concas MP, Connell JM, de las Fuentes L, de Silva HJ, de Vries PS, Doumatey A, Duan Q, Eaton CB, Eppinga RN, Faul JD, Floyd JS, Forouhi NG, Forrester T, Friedlander Y, Gandin I, Gao H, Ghanbari M, Gharib SA, Gigante B, Giulianini F, Grabe HJ, Gu CC, Harris TB, Heikkinen S, Heng CK, Hirata M, Hixson JE, Ikram MA, Consortium EI, Jia Y, Joehanes R, Johnson C, Jonas JB, Justice AE, Katsuya T, Khor CC, Kilpeläinen TO, Koh WP, Kolcic I, Kooperberg C, Krieger JE, Kritchevsky SB, Kubo M, Kuusisto J, Lakka TA, Langefeld CD, Langenberg C, Launer LJ, Lehne B, Lewis CE, Li Y, Liang J, Lin S, Liu CT, Liu J, Liu K, Loh M, Lohman KK, Louie T, Luzzi A, Mägi R, Mahajan A, Manichaikul AW, McKenzie CA, Meitinger T, Metspalu A, Milaneschi Y, Milani L, Mohlke KL, Momozawa Y, Morris AP, Murray AD, Nalls MA, Nauck M, Nelson CP, North KE, O'Connell JR, Palmer ND, Papanicolau GJ, Pedersen NL, Peters A, Peyser PA, Polasek O, Poulter N, Raitakari OT, Reiner AP, Renström F, Rice TK, Rich SS, Robinson JG, Rose LM, Rosendaal FR, Rudan I, Schmidt CO, Schreiner PJ, Scott WR, Sever P, Shi Y, Sidney S, Sims M, Smith JA, Snieder H, Starr JM, Strauch K, Stringham HM, Tan NY, Tang H, Taylor KD, Teo YY, Tham YC, Tiemeier H, Turner ST, Uitterlinden AG, Understanding Society Scientific Group, van Heemst D, Waldenberger M, Wang H, Wang L, Wang L, Wei WB, Williams CA, Wilson G Sr, Wojczynski MK, Yao J, Young K, Yu C, Yuan JM, Zhou J, Zonderman AB, Becker DM, Boehnke M, Bowden DW, Chambers JC, Cooper RS, de Faire U, Deary IJ, Elliott P, Esko T, Farrall M, Franks PW, Freedman BI, Froguel P, Gasparini P, Gieger C, Horta BL, Juang JM, Kamatani Y, Kammerer CM, Kato N, Kooner JS, Laakso M, Laurie CC, Lee IT, Lehtimäki T, Lifelines Cohort Study, Magnusson PK, Oldehinkel AJ, Penninx BW, Pereira AC, Rauramaa R, Redline S, Samani NJ, Scott J, Shu XO, van der Harst P, Wagenknecht LE, Wang JS, Wang YX, Wareham NJ, Watkins H, Weir DR, Wickremasinghe AR, Wu T, Zeggini E, Zheng W, Bouchard C, Evans MK, Gudnason V, Kardia SL, Liu Y, Psaty BM, Ridker PM, van Dam RM, Mook-Kanamori DO, Fornage M, Province MA, Kelly TN, Fox ER, Hayward C, van Duijn CM, Tai ES, Wong TY, Loos RJ, Franceschini N, Rotter JI, Zhu X, Bierut LJ, Gauderman WJ, Rice K, Munroe PB, Morrison AC, Rao DC, Rotimi CN, Cupples LA. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nature genetics. 2019 Mar 29; 51(4): 636-648</em>
</div>
</li>
</ul>
<a class="seemore" href="citedby/" ref="reftype=relart&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-18" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-18" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=pccompound&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_pccompound&amp;IdsFromResult=5011075" ref="reftype=Compound&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|PubChem%20Compound&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Compound</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Compound<div class="brieflinkpopdesc">PubChem Compound links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gene&amp;IdsFromResult=5011075" ref="reftype=Gene&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Gene&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Gene</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Gene<div class="brieflinkpopdesc">Gene links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=geoprofiles&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_geoprofiles&amp;IdsFromResult=5011075" ref="reftype=GEO%20Profiles&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">GEO Profiles</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-21" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">GEO Profiles<div class="brieflinkpopdesc">Related GEO records</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=5011075" ref="reftype=MedGen&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-22" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pubmed/27588450/" ref="reftype=PubMed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-23" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=5011075" ref="reftype=Taxonomy&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-24" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-25" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-25" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5dd59c16dde0893a8b769a6b">Trans-ethnic Fine Mapping Highlights Kidney-Function Genes Linked to Salt Sensit...</a><div class="ralinkpop offscreen_noflow">Trans-ethnic Fine Mapping Highlights Kidney-Function Genes Linked to Salt Sensitivity<div class="brieflinkpopdesc">American Journal of Human Genetics. 2016 Sep 1; 99(3)636</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=5dd59bdedde0893a8b6e2181">Meta-analysis of genome-wide association studies of HDL cholesterol response to ...</a><div class="ralinkpop offscreen_noflow">Meta-analysis of genome-wide association studies of HDL cholesterol response to statins<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2016 Dec; 53(12)835</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=5dd59ba2dd2142621d4863d0">Fine-mapping, novel loci identification, and SNP association transferability in ...</a><div class="ralinkpop offscreen_noflow">Fine-mapping, novel loci identification, and SNP association transferability in a genome-wide association study of QRS duration in African Americans<div class="brieflinkpopdesc">Human Molecular Genetics. 2016 Oct 1; 25(19)4350</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=5dd59b69dd2142621d3fb8eb">A Genome-Wide Association Study to Identify Single-Nucleotide Polymorphisms for ...</a><div class="ralinkpop offscreen_noflow">A Genome-Wide Association Study to Identify Single-Nucleotide Polymorphisms for Acute Kidney Injury<div class="brieflinkpopdesc">American Journal of Respiratory and Critical Care Medicine. 2017 Feb 15; 195(4)482</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=5dd59b40dd2142621d39372c">A Genome-wide Approach Validates that Thiopurine Methyltransferase Activity is a...</a><div class="ralinkpop offscreen_noflow">A Genome-wide Approach Validates that Thiopurine Methyltransferase Activity is a Monogenic Pharmacogenomic Trait<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2017 Mar; 101(3)373</div>
</div>
<div class="tertiary"></div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p>
<a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"><div id="body-link-popper-bib1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">1. <span class="element-citation" id="sref1">Levey A.S., Coresh J. Chronic kidney disease. <span><span class="ref-journal">Lancet. </span>2012;<span class="ref-vol">379</span>:165–180.</span> [<a href="/pubmed/21840587" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Chronic+kidney+disease&amp;author=A.S.+Levey&amp;author=J.+Coresh&amp;volume=379&amp;publication_year=2012&amp;pages=165-180&amp;pmid=21840587&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib1">Ref list</a>]</div><div id="body-link-popper-bib2" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">2. <span class="element-citation" id="sref2">Collins A.J., Foley R.N., Gilbertson D.T., Chen S.-C. United States Renal Data System public health surveillance of chronic kidney disease and end-stage renal disease. <span><span class="ref-journal">Kidney Int Suppl (2011) </span>2015;<span class="ref-vol">5</span>:2–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4455192/">PMC free article</a>]</span> [<a href="/pubmed/26097778" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int+Suppl+(2011)&amp;title=United+States+Renal+Data+System+public+health+surveillance+of+chronic+kidney+disease+and+end-stage+renal+disease&amp;author=A.J.+Collins&amp;author=R.N.+Foley&amp;author=D.T.+Gilbertson&amp;author=S.-C.+Chen&amp;volume=5&amp;publication_year=2015&amp;pages=2-7&amp;pmid=26097778&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib2">Ref list</a>]</div><div id="body-link-popper-bib3" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">3. <span class="element-citation" id="sref3">Xue J.L., Eggers P.W., Agodoa L.Y., Foley R.N., Collins A.J. Longitudinal study of racial and ethnic differences in developing end-stage renal disease among aged medicare beneficiaries. <span><span class="ref-journal">J.&nbsp;Am. Soc. Nephrol. </span>2007;<span class="ref-vol">18</span>:1299–1306.</span> [<a href="/pubmed/17329578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Am.+Soc.+Nephrol.&amp;title=Longitudinal+study+of+racial+and+ethnic+differences+in+developing+end-stage+renal+disease+among+aged+medicare+beneficiaries&amp;author=J.L.+Xue&amp;author=P.W.+Eggers&amp;author=L.Y.+Agodoa&amp;author=R.N.+Foley&amp;author=A.J.+Collins&amp;volume=18&amp;publication_year=2007&amp;pages=1299-1306&amp;pmid=17329578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib3">Ref list</a>]</div><div id="body-link-popper-bib4" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">4. <span class="element-citation" id="sref4">Köttgen A., Pattaro C., Böger C.A., Fuchsberger C., Olden M., Glazer N.L., Parsa A., Gao X., Yang Q., Smith A.V. New loci associated with kidney function and chronic kidney disease. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:376–384.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2997674/">PMC free article</a>]</span> [<a href="/pubmed/20383146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=New+loci+associated+with+kidney+function+and+chronic+kidney+disease&amp;author=A.+Köttgen&amp;author=C.+Pattaro&amp;author=C.A.+Böger&amp;author=C.+Fuchsberger&amp;author=M.+Olden&amp;volume=42&amp;publication_year=2010&amp;pages=376-384&amp;pmid=20383146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib4">Ref list</a>]</div><div id="body-link-popper-bib5" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">5. <span class="element-citation" id="sref5">Pattaro C., Köttgen A., Teumer A., Garnaas M., Böger C.A., Fuchsberger C., Olden M., Chen M.-H., Tin A., Taliun D., CARDIoGRAM Consortium. ICBP Consortium. CARe Consortium. Wellcome Trust Case Control Consortium 2 (WTCCC2)  Genome-wide association and functional follow-up reveals new loci for kidney function. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002584.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3315455/">PMC free article</a>]</span> [<a href="/pubmed/22479191" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+and+functional+follow-up+reveals+new+loci+for+kidney+function&amp;author=C.+Pattaro&amp;author=A.+Köttgen&amp;author=A.+Teumer&amp;author=M.+Garnaas&amp;author=C.A.+Böger&amp;volume=8&amp;publication_year=2012&amp;pages=e1002584&amp;pmid=22479191&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib5">Ref list</a>]</div><div id="body-link-popper-bib6" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">6. <span class="element-citation" id="sref6">Okada Y., Sim X., Go M.J., Wu J.Y., Gu D., Takeuchi F., Takahashi A., Maeda S., Tsunoda T., Chen P., KidneyGen Consortium. CKDGen Consortium. GUGC consortium  Meta-analysis identifies multiple loci associated with kidney function-related traits in east Asian populations. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:904–909.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4737645/">PMC free article</a>]</span> [<a href="/pubmed/22797727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Meta-analysis+identifies+multiple+loci+associated+with+kidney+function-related+traits+in+east+Asian+populations&amp;author=Y.+Okada&amp;author=X.+Sim&amp;author=M.J.+Go&amp;author=J.Y.+Wu&amp;author=D.+Gu&amp;volume=44&amp;publication_year=2012&amp;pages=904-909&amp;pmid=22797727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib6">Ref list</a>]</div><div id="body-link-popper-bib7" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">7. <span class="element-citation" id="sref7">Gorski M., Tin A., Garnaas M., McMahon G.M., Chu A.Y., Tayo B.O., Pattaro C., Teumer A., Chasman D.I., Chalmers J. Genome-wide association study of kidney function decline in individuals of European descent. <span><span class="ref-journal">Kidney Int. </span>2015;<span class="ref-vol">87</span>:1017–1029.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4425568/">PMC free article</a>]</span> [<a href="/pubmed/25493955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int.&amp;title=Genome-wide+association+study+of+kidney+function+decline+in+individuals+of+European+descent&amp;author=M.+Gorski&amp;author=A.+Tin&amp;author=M.+Garnaas&amp;author=G.M.+McMahon&amp;author=A.Y.+Chu&amp;volume=87&amp;publication_year=2015&amp;pages=1017-1029&amp;pmid=25493955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib7">Ref list</a>]</div><div id="body-link-popper-bib8" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">8. <span class="element-citation" id="sref8">Pattaro C., Teumer A., Gorski M., Chu A.Y., Li M., Mijatovic V., Garnaas M., Tin A., Sorice R., Li Y., ICBP Consortium. AGEN Consortium. CARDIOGRAM. CHARGe-Heart Failure Group. ECHOGen Consortium  Genetic associations at 53 loci highlight cell types and biological pathways relevant for kidney function. <span><span class="ref-journal">Nat. Commun. </span>2016;<span class="ref-vol">7</span>:10023.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4735748/">PMC free article</a>]</span> [<a href="/pubmed/26831199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Genetic+associations+at+53+loci+highlight+cell+types+and+biological+pathways+relevant+for+kidney+function&amp;author=C.+Pattaro&amp;author=A.+Teumer&amp;author=M.+Gorski&amp;author=A.Y.+Chu&amp;author=M.+Li&amp;volume=7&amp;publication_year=2016&amp;pages=10023&amp;pmid=26831199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib8">Ref list</a>]</div><div id="body-link-popper-bib9" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">9. <span class="element-citation" id="sref9">Abecasis G.R., Auton A., Brooks L.D., DePristo M.A., Durbin R.M., Handsaker R.E., Kang H.M., Marth G.T., McVean G.A., The 1000 Genomes Project Consortium  An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=G.R.+Abecasis&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=M.A.+DePristo&amp;author=R.M.+Durbin&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib9">Ref list</a>]</div><div id="body-link-popper-bib10" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">10. <span class="element-citation" id="sref10">Delaneau O., Marchini J., Zagury J.F. A linear complexity phasing method for thousands of genomes. <span><span class="ref-journal">Nat. Methods. </span>2011;<span class="ref-vol">9</span>:179–181.</span> [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes&amp;author=O.+Delaneau&amp;author=J.+Marchini&amp;author=J.F.+Zagury&amp;volume=9&amp;publication_year=2011&amp;pages=179-181&amp;pmid=22138821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib10">Ref list</a>]</div><div id="body-link-popper-bib11" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">11. <span class="element-citation" id="sref11">Howie B., Fuchsberger C., Stephens M., Marchini J., Abecasis G.R. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B.+Howie&amp;author=C.+Fuchsberger&amp;author=M.+Stephens&amp;author=J.+Marchini&amp;author=G.R.+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib11">Ref list</a>]</div><div id="body-link-popper-bib12" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">12. <span class="element-citation" id="sref12">Frazer K.A., Ballinger D.G., Cox D.R., Hinds D.A., Stuve L.L., Gibbs R.A., Belmont J.W., Boudreau A., Hardenbol P., Leal S.M., International HapMap Consortium  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=K.A.+Frazer&amp;author=D.G.+Ballinger&amp;author=D.R.+Cox&amp;author=D.A.+Hinds&amp;author=L.L.+Stuve&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib12">Ref list</a>]</div><div id="body-link-popper-bib13" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">13. <span class="element-citation" id="sref13">Howie B.N., Donnelly P., Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=B.N.+Howie&amp;author=P.+Donnelly&amp;author=J.+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib13">Ref list</a>]</div><div id="body-link-popper-bib14" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">14. <span class="element-citation" id="sref14">Winkler T.W., Day F.R., Croteau-Chonka D.C., Wood A.R., Locke A.E., Mägi R., Ferreira T., Fall T., Graff M., Justice A.E., Genetic Investigation of Anthropometric Traits&nbsp;(GIANT) Consortium  Quality control and conduct of genome-wide association meta-analyses. <span><span class="ref-journal">Nat. Protoc. </span>2014;<span class="ref-vol">9</span>:1192–1212.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4083217/">PMC free article</a>]</span> [<a href="/pubmed/24762786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc.&amp;title=Quality+control+and+conduct+of+genome-wide+association+meta-analyses&amp;author=T.W.+Winkler&amp;author=F.R.+Day&amp;author=D.C.+Croteau-Chonka&amp;author=A.R.+Wood&amp;author=A.E.+Locke&amp;volume=9&amp;publication_year=2014&amp;pages=1192-1212&amp;pmid=24762786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib14">Ref list</a>]</div><div id="body-link-popper-bib15" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">15. <span class="element-citation" id="sref15">National Kidney Foundation  K/DOQI clinical practice guidelines for chronic kidney disease: evaluation, classification, and stratification. <span><span class="ref-journal">Am. J. Kidney Dis. </span>2002;<span class="ref-vol">39</span>(2 Suppl 1):S1–S266.</span> [<a href="/pubmed/11904577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Kidney+Dis.&amp;title=K/DOQI+clinical+practice+guidelines+for+chronic+kidney+disease:+evaluation,+classification,+and+stratification&amp;volume=39&amp;issue=2+Suppl+1&amp;publication_year=2002&amp;pages=S1-S266&amp;pmid=11904577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5011075&amp;issue-id=275125&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib15">Ref list</a>]</div></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:03:34-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C2132DD58CE8100000000096F0588&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5011075%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5011075/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size1, sans-serif;"></div></div></body></html>