<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5117923/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Ancestry-based stratified analysis of Immunochip data identifies novel associations with celiac disease</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5117923/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="Ancestry-based stratified analysis of Immunochip data identifies novel associations with celiac disease"><meta name="citation_authors" content="Koldo Garcia-Etxebarria, Amaia Jauregi-Miguel, Irati Romero-Garmendia, Leticia Plaza-Izurieta, Maria Legarda, Iñaki Irastorza, Jose Ramon Bilbao"><meta name="citation_date" content="December 2016"><meta name="citation_issue" content="12"><meta name="citation_volume" content="24"><meta name="citation_firstpage" content="1831"><meta name="citation_doi" content="10.1038/ejhg.2016.120"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5117923/?report=abstract"><meta name="citation_pmid" content="27650971"><meta name="DC.Title" content="Ancestry-based stratified analysis of Immunochip data identifies novel associations with celiac disease"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Koldo Garcia-Etxebarria"><meta name="DC.Contributor" content="Amaia Jauregi-Miguel"><meta name="DC.Contributor" content="Irati Romero-Garmendia"><meta name="DC.Contributor" content="Leticia Plaza-Izurieta"><meta name="DC.Contributor" content="Maria Legarda"><meta name="DC.Contributor" content="Iñaki Irastorza"><meta name="DC.Contributor" content="Jose Ramon Bilbao"><meta name="DC.Date" content="2016 Dec"><meta name="DC.Identifier" content="10.1038/ejhg.2016.120"><meta name="DC.Language" content="en"><meta property="og:title" content="Ancestry-based stratified analysis of Immunochip data identifies novel associations with celiac disease"><meta property="og:type" content="article"><meta property="og:description" content="To identify candidate genes in celiac disease (CD), we reanalyzed the whole Immunochip CD cohort using a different approach that clusters individuals based on immunoancestry prior to disease association analysis, rather than by geographical origin. We ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5117923/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5117923/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5117923/pdf/ejhg2016120a.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8976F2DD59BFA100000000012200A1.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5117923%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/279467/">v.24(12); 2016 Dec</a></li><li class="accid">PMC5117923</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2016 Dec; 24(12): 1831–1834. </span></div><div><span class="fm-vol-iss-date">Published online 2016 Sep 21. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fejhg.2016.120" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ejhg.2016.120</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5117923</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27650971">27650971</a></div></div></div></div><h1 class="content-title">Ancestry-based stratified analysis of Immunochip data identifies novel associations with celiac disease</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Garcia-Etxebarria%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971" class="affpopup" co-rid="_co_idm140271918929664" co-class="co-affbox">Koldo Garcia-Etxebarria</a>,<sup>1</sup> <a href="/pubmed/?term=Jauregi-Miguel%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971" class="affpopup" co-rid="_co_idm140271919582944" co-class="co-affbox">Amaia Jauregi-Miguel</a>,<sup>1</sup> <a href="/pubmed/?term=Romero-Garmendia%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971" class="affpopup" co-rid="_co_idm140271922812752" co-class="co-affbox">Irati Romero-Garmendia</a>,<sup>1</sup> <a href="/pubmed/?term=Plaza-Izurieta%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971" class="affpopup" co-rid="_co_idm140271920875792" co-class="co-affbox">Leticia Plaza-Izurieta</a>,<sup>1</sup> <a href="/pubmed/?term=Legarda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971" class="affpopup" co-rid="_co_idm140271902308448" co-class="co-affbox">Maria Legarda</a>,<sup>2</sup> <a href="/pubmed/?term=Irastorza%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971" class="affpopup" co-rid="_co_idm140271925651392" co-class="co-affbox">Iñaki Irastorza</a>,<sup>2</sup> and  <a href="/pubmed/?term=Bilbao%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971" class="affpopup" co-rid="_co_idm140271927369792" co-class="co-affbox">Jose Ramon Bilbao</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140271918929664"><h3 class="no_margin">Koldo Garcia-Etxebarria</h3><p><sup>1</sup>Department of Genetics, Physical Anthropology and Animal Physiology, University of the Basque Country (UPV-EHU), BioCruces Health Research Institute, Leioa, Spain</p><div>Find articles by <a href="/pubmed/?term=Garcia-Etxebarria%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971">Koldo Garcia-Etxebarria</a></div></div><div id="_co_idm140271919582944"><h3 class="no_margin">Amaia Jauregi-Miguel</h3><p><sup>1</sup>Department of Genetics, Physical Anthropology and Animal Physiology, University of the Basque Country (UPV-EHU), BioCruces Health Research Institute, Leioa, Spain</p><div>Find articles by <a href="/pubmed/?term=Jauregi-Miguel%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971">Amaia Jauregi-Miguel</a></div></div><div id="_co_idm140271922812752"><h3 class="no_margin">Irati Romero-Garmendia</h3><p><sup>1</sup>Department of Genetics, Physical Anthropology and Animal Physiology, University of the Basque Country (UPV-EHU), BioCruces Health Research Institute, Leioa, Spain</p><div>Find articles by <a href="/pubmed/?term=Romero-Garmendia%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971">Irati Romero-Garmendia</a></div></div><div id="_co_idm140271920875792"><h3 class="no_margin">Leticia Plaza-Izurieta</h3><p><sup>1</sup>Department of Genetics, Physical Anthropology and Animal Physiology, University of the Basque Country (UPV-EHU), BioCruces Health Research Institute, Leioa, Spain</p><div>Find articles by <a href="/pubmed/?term=Plaza-Izurieta%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971">Leticia Plaza-Izurieta</a></div></div><div id="_co_idm140271902308448"><h3 class="no_margin">Maria Legarda</h3><p><sup>2</sup>Department of Pediatrics, Pediatric Gastroenterology Unit, Cruces University Hospital, University of the Basque Country (UPV-EHU), Barakaldo, Spain</p><div>Find articles by <a href="/pubmed/?term=Legarda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971">Maria Legarda</a></div></div><div id="_co_idm140271925651392"><h3 class="no_margin">Iñaki Irastorza</h3><p><sup>2</sup>Department of Pediatrics, Pediatric Gastroenterology Unit, Cruces University Hospital, University of the Basque Country (UPV-EHU), Barakaldo, Spain</p><div>Find articles by <a href="/pubmed/?term=Irastorza%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971">Iñaki Irastorza</a></div></div><div id="_co_idm140271927369792"><h3 class="no_margin">Jose Ramon Bilbao</h3><p><sup>1</sup>Department of Genetics, Physical Anthropology and Animal Physiology, University of the Basque Country (UPV-EHU), BioCruces Health Research Institute, Leioa, Spain</p><div>Find articles by <a href="/pubmed/?term=Bilbao%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27650971">Jose Ramon Bilbao</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140271970827648_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140271970827648_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140271970827648_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140271970827648_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Genetics, Physical Anthropology and Animal Physiology, University of the Basque Country (UPV-EHU), BioCruces Health Research Institute, Leioa, Spain</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Pediatrics, Pediatric Gastroenterology Unit, Cruces University Hospital, University of the Basque Country (UPV-EHU), Barakaldo, Spain</div><div id="caf1"><sup>*</sup>Department of Genetics, Physical Anthropology and Animal Physiology, University of the Basque Country (UPV-EHU), BioCruces Health Research Institute, Biscay Campus, Basque Country, Bizkaia, Leioa 48940, Spain. Tel: +34 94 601 5317; Fax: +34 94 601 3145; E-mail: <a href="mailto:dev@null" data-email="sue.uhe@oablib.nomaresoj" class="oemail">sue.uhe@oablib.nomaresoj</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140271970827648_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 Mar 8; Revised 2016 Jul 24; Accepted 2016 Aug 5.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140271970827648_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2016 Macmillan Publishers Limited, part of Springer Nature.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5117923/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC5117923/bin/ejhg2016120x1.pdf" data-ga-action="click_feat_suppl">ejhg2016120x1.pdf</a><span style="color:gray"> (1.4M)</span></div><div class="small guid">GUID:&nbsp;1CD2E2B0-DC04-4711-8093-96A3B794251F</div></div></dd></dl></div><div id="idm140271926828688" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140271926828688title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">To identify candidate genes in celiac disease (CD), we reanalyzed the whole Immunochip CD cohort using a different approach that clusters individuals based on immunoancestry prior to disease association analysis, rather than by geographical origin. We detected 636 new associated SNPs (<em>P&lt;</em>7.02 × 10<sup>−07</sup>) and identified 5 novel genomic regions, extended 8 others previously identified and also detected 18 isolated signals defined by one or very few significant SNPs. To test whether we could identify putative candidate genes, we performed expression analyses of several genes from the top novel region (chr2:134533564–136169524), from a previously identified locus that is now extended, and a gene marked by an isolated SNP, in duodenum biopsies of active and treated CD patients, and non-celiac controls. In the largest novel region, <em>CCNT2</em> and <em>R3HDM1</em> were constitutively underexpressed in disease, even after gluten removal. Moreover, several genes within this region were coexpressed in patients, but not in controls. Other novel genes like <em>KIF21B</em>, <em>REL</em> and <em>SORD</em> also showed altered expression in active disease. Apart from the identification of novel CD loci, these results suggest that ancestry-based stratified analysis is an efficient strategy for association studies in complex diseases.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">Celiac disease (CD, MIM: 212750) is a chronic, autoimmune disorder caused by intolerance to dietary gluten that develops in genetically susceptible individuals. It is a common disease (around 1% of the population) that is characterized by the presence of autoantibodies against tissue transglutaminase and villous atrophy, crypt hyperplasia and lymphocytic infiltration of the small intestinal mucosa. The major histocompatibility complex region on 6p21 harbors the major contributors to CD risk: in Caucasians, HLA-DQ2/-DQ8 heterodimers are present in &gt;90% of CD patients, but also in around 30% of the general population, so that HLA alone cannot explain all the genetic component.<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254809">1</a></sup> Genome-wide association studies (GWAS) and Immunochip project identified 57 association signals from 39 loci, that together contribute 5–7% to the genetic risk.<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254833">1</a>, <a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254813">2</a></sup> More recently, new association signals have been detected in the major histocompatibility complex region, increasing up to 48% proportion of the heritability that is known so far.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254823">3</a></sup> However, the effect of rare, coding variants within the Immunochip genes is minimal,<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254819">4</a></sup> and thus the remaining genetic component related to CD should still reside, in part, in common and known variants.</p><p id="__p3" class="p p-last">Despite the progress made, it has proven difficult to reconcile the results from association analyses across different populations, and to square SNP association results and expression levels of <em>cis-</em>located genes in patient tissues.<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254825">5</a>, <a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254812">6</a></sup> This limited success could be partly owing to certain genetic heterogeneity within CD, so that not every associated SNP is relevant to all CD cases. Random effects modeling has recently shown that SNPs reported to be associated with the disease (rs1050976C&gt;T in <em>IRF4</em>, h38 chr6:g.408079C&gt;T and rs11851414:C&gt;T in <em>ZFP36L1</em>, h38 chr14:g.68792785T&gt;C) would not have reached the significance threshold if heterogeneity among the different collections analyzed in the Immunochip had been accounted for.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254821">7</a></sup> In the original analysis, a covariate was introduced to indicate collection membership, but not the possible heterogeneity within. We believe that heterogeneity within each one of the Immunochip cohorts could be stronger than what has been assumed. In the present work, we propose taking into consideration the (immuno)genomic background of each individual (revealed by the Immunochip itself) rather than geographical origin, as an alternative strategy for disease association analysis of the Immunochip data.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Subjects and Methods</h2><p id="__p4" class="p p-first">We reanalyzed the 139 553 SNPs from the Immunochip in 12 041 CD patients and 12 228 non-celiac controls. To stratify individuals according to their genetic background, we first detected 8537 conserved LD blocks of SNPs using Plink<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254827">8</a></sup> and selected one random SNP from each block. These 8537 SNPs were used to calculate the possible number of ancestries using Admixture<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254820">9</a></sup> and the optimal number was set to 30 because it was the first K with a lower cross-validation value than the next K (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1</a>). We then assigned each individual to 1 of the 30 immunogroups (named this way because they are based on the Immunochip SNPs), according to their major ancestry component (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2</a>). Immunogroup sizes ranged from 19 to 4178 individuals (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S1</a>), and contained celiac and control individuals from different geographical origins, except for one where all the samples of Indian origin clustered (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3</a>), stressing the limitations of the Immunochip for the genetic analysis of non-European populations.<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254824">10</a></sup> Finally, we performed an association analysis, correcting for stratification of the 30 immunogroups, using a Cochran–Mantel–Haenszel test implemented in Plink.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254822">8</a></sup> We set the significance cutoff to <em>P&lt;</em>7.02 × 10<sup>−07</sup>, as there were 71 208 independent tests (8537 LD blocks plus 62 671 SNPs outside them), as calculated previously.<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254810">10</a></sup> The results of the association study are available at GWAS Central <a href="http://www.gwascentral.org/study/HGVST1839" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gwascentral.org/study/HGVST1839</a>.</p><p id="__p5" class="p p-last">The expression of 14 protein-coding genes was measured in intestinal biopsies from 15 CD patients at the time of diagnosis and after &gt;2 years on gluten-free diet (GFD), and from 15 non-celiac controls (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S3</a>). CD was diagnosed according to the ESPGHAN criteria. The study was approved by the Cruces University Hospital, and Basque Clinical Trials and Ethics Committees (CEIC- E09/10 and PI2013072), and biopsies of distal duodenum were obtained by endoscopy after informed consent from all subjects or their parents. Total RNA was extracted using the NucleoSpin microRNA kit (Macherey-Nagel, Düren, Germany) and converted to cDNA using the AffinityScript cDNA Synthesis kit (Agilent Technologies, Santa Clara, CA, USA). Gene expression was analyzed using Fluidigm Biomark 48.48 dynamic arrays (Fluidigm Corp., South San Francisco, CA, USA) and commercially available TaqMan Gene Expression assays, including <em>RPLPO</em> as an endogenous control of input RNA (Thermo Fisher Scientific Inc., Waltham, MA, USA). Relative expression was calculated using the accurate ΔΔCt method and normalized to the average expression value of the control samples. Difference between conditions was tested using nonparametric tests, paired in the case of the comparison between active and treated CD. Gene expression data are available at Gene Expression Omnibus (<a href="http://www.ncbi.nlm.nih.gov/geo" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/geo</a>) with accession number <a id="__tag_572254835" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE84729" ref="/geo/query/acc.cgi?acc=GSE84729"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE84729","term_id":"84729"}}</span>GSE84729</a>.</p></div><div id="__sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec3title">Results and Discussion</h2><p id="__p6" class="p p-first">There were 4881 SNPs significantly associated with CD (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S4</a>), of which 636 had not been detected in the original analysis (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S5</a>). Only one previously associated region was not detected: hg38 chr14:68,792,689–68,805,252 (<em>P</em>=3.146 × 10<sup>−06</sup>), where <em>ZFP36L1</em> had been proposed as the putative candidate gene, a region which was also ‘missed' by the Random Effects study.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254808">7</a></sup> A total of 500 novel SNPs were located in five previously unidentified genomic regions or close to or within four previously known loci, extending two of them; and there were also several isolated signals defined by one or few significant SNPs (<a href="/pmc/articles/PMC5117923/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>,<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>). The most strongly associated novel region (hg38 chr2:134,533,564-136,169,524) contains markers and genes that have been associated with type 2 diabetes<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254817">11</a></sup> (<a href="/pmc/articles/PMC5117923/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1a</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC5117923/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140271917540496" class="figure"><a href="/pmc/articles/PMC5117923/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5117923_ejhg2016120f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg2016120f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5117923/bin/ejhg2016120f1.jpg"></a></div><div id="largeobj_idm140271917540496" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5117923/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC5117923/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p16">The novel hg38 chr2:134,533,564–136,169,524 region that is associated with celiac disease. (<strong>a</strong>) Graphical representation of the region made using the Locuszoom online tool (<a href="http://locuszoom.sph.umich.edu/locuszoom/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://locuszoom.sph.umich.edu/locuszoom/</a>). (<strong>b</strong>) Expression analysis of selected genes from the region. REAC: expression relative to control average; below each gene, Mann–Whitney test <em>P</em>-values, when significant. (<strong>c</strong>) Spearman correlation analysis of expression levels.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Summary of relevant CD-associated regions identified or extended in this study (excluding the MHC)</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140271915581552" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Region covered by associated markers</em><sup><a href="#t1-fn2" rid="t1-fn2" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Previously reported region</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Number of significant new SNPs</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Number of SNPs in the region</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Top SNP (</em><em>P</em><em>-value; odds ratio)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Genes in the region</em><sup><a href="#t1-fn3" rid="t1-fn3" class=" fn">b</a></sup></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">chr1:66,920,521-67,373,890</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">489</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs17497947:G&gt;T; hg38 chr1:g.66991516G&gt;T (1.5 × 10<sup>−7</sup>; 0.88)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>SLC35D1, C1orf141, IL12RB2, IL23R</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">chr1:200,899,734-201,054,931</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">chr1:200,912,264</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">51</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">267</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7522462:A&gt;G; hg38 chr1:g.200912467G&gt;A (7.8 × 10<sup>−12</sup>; 0.86)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em><span class="underline">C1orf106</span>, KIF21B, CACNA1S</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">chr2:43,116,667-43,133,808</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">64</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs12466022:A&gt;C; hg38 chr2:g.43131922C&gt;A (2.5 × 10<sup>−8</sup>; 0.88)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">chr2:60,731,152-61,660,852</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">chr2:60,959,694-60,977,721</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">184</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">997</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs777585:C&gt;T; hg38 chr2:g.61185424C&gt;T (1.7 × 10<sup>−16</sup>; 0.84)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em><span class="underline">PUS10</span>, LINC01185, REL, PEX13, KIAA1841, USP34, XPO1</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">chr2:134,533,564-136,169,524</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">41</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">134</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6754311:C&gt;T; hg38 chr2:g.135950412T&gt;C (7.6 × 10<sup>−21</sup>; 1.23)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>TMEM163, ACMSD, CCNT2-AS1, CCNT2, MAP3K19, RAB3GAP1, ZRANB3, R3HDM1, LCT, MCM6, DARS</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">chr15:28,120,472-28,285,036</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs12913832:A&gt;G; hg38 chr15:g.28120472A&gt;G (1.4 × 10<sup>−15</sup>; 1.20)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>HERC2</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">chr16:27,403,018-27,414,415</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs12934152:C&gt;T; hg38 chr16:g.27403018T&gt;C (8.6 × 10<sup>−08</sup>; 1.129)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>IL21R</em></td></tr></tbody></table></div><div id="largeobj_idm140271915581552" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5117923/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1-fn1"><p id="__p17" class="p p-first-last">All positions correspond to the hg38 genome assembly.</p></div><div id="t1-fn2"><sup>a</sup>Nearby SNPs with <em>P</em>&lt;7.02 × 10<sup>−7</sup> in our analysis.</div><div id="t1-fn3"><sup>b</sup>Previously identified candidate genes are underlined.</div></div></div><p id="__p7">In this region, <em>CCNT2</em> and <em>R3HDM1</em> showed decreased expression between CD patients and controls, both at diagnosis and on GFD (<a href="/pmc/articles/PMC5117923/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1b</span></a>), pointing to a constitutive defect. <em>CCNT2</em> is a cyclin that is involved in cell cycle and RNA transcription. <em>R3HDM1</em> is a poorly characterized gene that could have a poly(A) RNA-binding function. The expression of the aspartate-tRNA ligase gene <em>DARS</em> was also reduced in patients, although it was not significant in active disease. In addition, the expression of genes within the region was strongly correlated among CD patients but not in controls (<a href="/pmc/articles/PMC5117923/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1c</span></a>), suggesting common, disease-dependent regulatory mechanisms in the region, as has been previously shown.<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254816">12</a>, <a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254826">13</a></sup> The lactase gene <em>LCT</em> showed a pronounced decrease in expression in active CD that recovered after GFD treatment (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S6</a>), indicative of the lactose intolerance observed in active CD. The other novel regions identified (<a href="/pmc/articles/PMC5117923/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>) contain genes relevant to the immune response associated with allergy (<em>IL21R</em>),<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254811">14</a></sup> Crohn's disease<sup><a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254815">15</a></sup> and psoriasis<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254834">16</a></sup> (<em>IL23R</em> and <em>IL12RB2</em>).</p><p id="__p8">Our analysis also identified novel SNPs in previously known regions, extending two of them (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>). In the hg38 chr1:200,901,626–201,054,931 region (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S7A</a>), <em>C1orf106</em> is the proposed candidate gene for CD,<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254831">2</a></sup> but there were several associated SNPs that extend it 3′-wards up to <em>CACNA1S</em>; including <em>KIF21B</em>, a kinesis related to immune-mediated chronic diseases like multiple sclerosis,<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254830">17</a></sup> whose expression was significantly increased in active CD (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S8A</a>).</p><p id="__p9">In the hg38 chr2:60,850,682–61,644,518 region (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S7B</a>), <em>PUS10</em> was the proposed candidate gene,<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254818">2</a></sup> but our results extend the region 5′-wards to <em>REL</em> and up to <em>XPO1</em> on the 3′side. Both genes participate in the NF<em>κ</em>B pathway, which is known to be altered in CD.<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254829">12</a>, <a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254828">18</a></sup> The expression of <em>REL</em> (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S8B</a>), a gene associated with CD,<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254832">19</a></sup> was reduced in active CD patients.</p><p id="__p10">There were also seven genes with only one significant SNP (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>) sometimes because those regions have a very low SNP density. From them, we analyzed the expression of <em>SORD</em>, a gene involved in the interconversion of polyols, that has been related to type 2 diabetic retinopathy,<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_572254814">20</a></sup> and its expression was significantly lower in CD individuals (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S8C</a>). Finally, a number of novel SNPs were located in intergenic regions, but functional analyses will be necessary to determine their possible role in disease susceptibility.</p><p id="__p11" class="p p-last">In conclusion, the immunoancestry-based analysis of the Immunochip data has allowed us to discover novel regions associated with CD that harbor genes that are functionally altered in patient intestinal mucosa. We believe that this type of stratified analysis is applicable to other large-scale genotype data from complex disease association studies and will help to find novel susceptibility genes, and to conciliate genotype and expression data.</p></div><div id="idm140271923177104" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140271923177104title">Acknowledgments</h2><div class="sec"><p id="__p12">JRB is funded by Research Grants PI13/01201 (within the National Plan for Scientific Research, Development and Technological Innovation 2013–2016, cofinanced by the Spanish Ministry Economy and Competitiveness, and the European Regional Development Fund) and 2011/111034 from the Basque Department of Health. IR-G and AJ-M are predoctoral fellows supported by grants from the University of the Basque Country (UPV/EHU) and the Basque Department of Education, respectively. This study makes use of data generated by the Wellcome Trust Case-Control Consortium (WTCC data sets EGAD00010000246, EGAD00010000248 and EGAD00010000250). A full list of the investigators who contributed to the generation of the data is available from <a href="http://www.wtccc.org.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.org.uk</a>. Funding for the project was provided by the Wellcome Trust under awards 076113, 085475 and 090355. We thank the technical and human support provided by SGIker of the UPV/EHU.</p></div></div><div id="idm140271900905824" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140271900905824title">Notes</h2><p id="__p14">The authors declare no conflict of interest.</p></div><div id="idm140271924518992" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140271924518992title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140271916871856"></p><p id="__p13" class="p p-first-last"><a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> accompanies this paper on European Journal of Human Genetics website (http://www.nature.com/ejhg)</p><p></p></div></div><div id="sup1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sup1title">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="xob1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="idm140271912089088"><a href="/pmc/articles/PMC5117923/bin/ejhg2016120x1.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.4M, pdf)</sup></div></div></div><div id="idm140271925646000" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140271925646000title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="bib1"><span class="mixed-citation">Ricaño-Ponce I, Wijmenga C, Gutierrez-Achury J: <span class="ref-title">Genetics of celiac disease</span>. <span class="ref-journal">Best Pract Res Clin Gastroenterol</span>
2015; <span class="ref-vol">29</span>: 399–412. [<a href="/pubmed/26060105" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Best+Pract+Res+Clin+Gastroenterol&amp;title=Genetics+of+celiac+disease&amp;volume=29&amp;publication_year=2015&amp;pages=399&amp;pmid=26060105&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib2"><span class="mixed-citation">Trynka G, Hunt K, Bockett N et al: <span class="ref-title">Dense genotyping identifies and localizes multiple common and rare variant association signals in celiac disease</span>. <span class="ref-journal">Nat Genet</span>
2011; <span class="ref-vol">43</span>: 1193–1201. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3242065/">PMC free article</a>]</span> [<a href="/pubmed/22057235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Dense+genotyping+identifies+and+localizes+multiple+common+and+rare+variant+association+signals+in+celiac+disease&amp;volume=43&amp;publication_year=2011&amp;pages=1193&amp;pmid=22057235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib3"><span class="mixed-citation">Gutierrez-Achury J, Zhernakova A, Pulit SL et al: <span class="ref-title">Fine mapping in the MHC region accounts for 18% additional genetic risk for celiac disease</span>. <span class="ref-journal">Nat Genet</span>
2015; <span class="ref-vol">47</span>: 577–578. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4449296/">PMC free article</a>]</span> [<a href="/pubmed/25894500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Fine+mapping+in+the+MHC+region+accounts+for+18%+additional+genetic+risk+for+celiac+disease&amp;volume=47&amp;publication_year=2015&amp;pages=577&amp;pmid=25894500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib4"><span class="mixed-citation">Hunt K, Mistry V, Bockett N et al: <span class="ref-title">Negligible impact of rare autoimmune-locus coding-region variants on missing heritability</span>. <span class="ref-journal">Nature</span>
2013; <span class="ref-vol">498</span>: 232–235. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3736321/">PMC free article</a>]</span> [<a href="/pubmed/23698362" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Negligible+impact+of+rare+autoimmune-locus+coding-region+variants+on+missing+heritability&amp;volume=498&amp;publication_year=2013&amp;pages=232&amp;pmid=23698362&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib5"><span class="mixed-citation">Plaza-Izurieta L, Castellanos-Rubio A, Irastorza I et al: <span class="ref-title">Revisiting genome wide association studies (GWAS) in coeliac disease: replication study in Spanish population and expression analysis of candidate genes</span>. <span class="ref-journal">J Med Genet</span>
2011; <span class="ref-vol">48</span>: 493–496. [<a href="/pubmed/21490378" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Revisiting+genome+wide+association+studies+(GWAS)+in+coeliac+disease:+replication+study+in+Spanish+population+and+expression+analysis+of+candidate+genes&amp;volume=48&amp;publication_year=2011&amp;pages=493&amp;pmid=21490378&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib6"><span class="mixed-citation">Plaza-Izurieta L, Fernandez-Jimenez N, Irastorza I et al: <span class="ref-title">Expression analysis in intestinal mucosa reveals complex relations among genes under the association peaks in celiac disease</span>. <span class="ref-journal">Eur J Hum Genet</span>
2015; <span class="ref-vol">23</span>: 1100–1105. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4795102/">PMC free article</a>]</span> [<a href="/pubmed/25388004" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Expression+analysis+in+intestinal+mucosa+reveals+complex+relations+among+genes+under+the+association+peaks+in+celiac+disease&amp;volume=23&amp;publication_year=2015&amp;pages=1100&amp;pmid=25388004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib7"><span class="mixed-citation">Ahn RS, Garner C: <span class="ref-title">A case study of fixed-effects and random-effects meta-analysis models for genome-wide association studies in celiac disease</span>. <span class="ref-journal">Hum Hered</span>
2015; <span class="ref-vol">80</span>: 51–61. [<a href="/pubmed/26436999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=A+case+study+of+fixed-effects+and+random-effects+meta-analysis+models+for+genome-wide+association+studies+in+celiac+disease&amp;volume=80&amp;publication_year=2015&amp;pages=51&amp;pmid=26436999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib8"><span class="mixed-citation">Purcell S, Neale B, Todd-Brown K et al: <span class="ref-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <span class="ref-journal">Am J Hum Genet</span>
2007; <span class="ref-vol">81</span>: 559–575. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;volume=81&amp;publication_year=2007&amp;pages=559&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib9"><span class="mixed-citation">Alexander DH, Novembre J, Lange K: <span class="ref-title">Fast model-based estimation of ancestry in unrelated individuals</span>. <span class="ref-journal">Genome Res</span>
2009; <span class="ref-vol">19</span>: 1655–1664. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2752134/">PMC free article</a>]</span> [<a href="/pubmed/19648217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Fast+model-based+estimation+of+ancestry+in+unrelated+individuals&amp;volume=19&amp;publication_year=2009&amp;pages=1655&amp;pmid=19648217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib10"><span class="mixed-citation">Senapati S, Gutierrez-Achury J, Sood A et al: <span class="ref-title">Evaluation of European coeliac disease risk variants in a north Indian population</span>. <span class="ref-journal">Eur J Hum Genet</span>
2014; <span class="ref-vol">23</span>: 530–535. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4666579/">PMC free article</a>]</span> [<a href="/pubmed/25052311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Evaluation+of+European+coeliac+disease+risk+variants+in+a+north+Indian+population&amp;volume=23&amp;publication_year=2014&amp;pages=530&amp;pmid=25052311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib11"><span class="mixed-citation">Tabassum R, Chauhan G, Dwivedi OP et al: <span class="ref-title">Genome-wide association study for type 2 diabetes in Indians identifies a new susceptibility locus at 2q21</span>. <span class="ref-journal">Diabetes</span>
2013; <span class="ref-vol">62</span>: 977–986. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3581193/">PMC free article</a>]</span> [<a href="/pubmed/23209189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Genome-wide+association+study+for+type+2+diabetes+in+Indians+identifies+a+new+susceptibility+locus+at+2q21&amp;volume=62&amp;publication_year=2013&amp;pages=977&amp;pmid=23209189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib12"><span class="mixed-citation">Fernandez-Jimenez N, Castellanos-Rubio A, Plaza-Izurieta L et al: <span class="ref-title">Coregulation and modulation of NFκB-related genes in celiac disease: uncovered aspects of gut mucosal inflammation</span>. <span class="ref-journal">Hum Mol Genet</span>
2014; <span class="ref-vol">23</span>: 1298–1310. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3919015/">PMC free article</a>]</span> [<a href="/pubmed/24163129" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Coregulation+and+modulation+of+NFκB-related+genes+in+celiac+disease:+uncovered+aspects+of+gut+mucosal+inflammation&amp;volume=23&amp;publication_year=2014&amp;pages=1298&amp;pmid=24163129&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib13"><span class="mixed-citation">Bondar C, Plaza-Izurieta L, Fernandez-Jimenez N et al: <span class="ref-title">THEMIS and PTPRK in celiac intestinal mucosa: coexpression in disease and after <em>in vitro</em> gliadin challenge</span>. <span class="ref-journal">Eur J Hum Genet</span>
2014; <span class="ref-vol">22</span>: 358–362. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3925264/">PMC free article</a>]</span> [<a href="/pubmed/23820479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=THEMIS+and+PTPRK+in+celiac+intestinal+mucosa:+coexpression+in+disease+and+after+in+vitro+gliadin+challenge&amp;volume=22&amp;publication_year=2014&amp;pages=358&amp;pmid=23820479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib14"><span class="mixed-citation">Hinds DA, McMahon G, Kiefer AK et al: <span class="ref-title">A genome-wide association meta-analysis of self-reported allergy identifies shared and allergy-specific susceptibility loci</span>. <span class="ref-journal">Nat Genet</span>
2013; <span class="ref-vol">45</span>: 907–911. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3753407/">PMC free article</a>]</span> [<a href="/pubmed/23817569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+meta-analysis+of+self-reported+allergy+identifies+shared+and+allergy-specific+susceptibility+loci&amp;volume=45&amp;publication_year=2013&amp;pages=907&amp;pmid=23817569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib15"><span class="mixed-citation">Julià A, Domènech E, Ricart E et al: <span class="ref-title">A genome-wide association study on a southern European population identifies a new Crohn's disease susceptibility locus at RBX1-EP300</span>. <span class="ref-journal">Gut</span>
2013; <span class="ref-vol">62</span>: 1440–1445. [<a href="/pubmed/22936669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=A+genome-wide+association+study+on+a+southern+European+population+identifies+a+new+Crohn's+disease+susceptibility+locus+at+RBX1-EP300&amp;volume=62&amp;publication_year=2013&amp;pages=1440&amp;pmid=22936669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib16"><span class="mixed-citation">Nair RP, Duffin KC, Helms C et al: <span class="ref-title">Genome-wide scan reveals association of psoriasis with IL-23 and NF-kappaB pathways</span>. <span class="ref-journal">Nat Genet</span>
2009; <span class="ref-vol">41</span>: 199–204. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745122/">PMC free article</a>]</span> [<a href="/pubmed/19169254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+scan+reveals+association+of+psoriasis+with+IL-23+and+NF-kappaB+pathways&amp;volume=41&amp;publication_year=2009&amp;pages=199&amp;pmid=19169254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib17"><span class="mixed-citation">Goris A, Boonen S, D'hooghe M-B, Dubois B: <span class="ref-title">Replication of KIF21B as a susceptibility locus for multiple sclerosis</span>. <span class="ref-journal">J Med Genet</span>
2010; <span class="ref-vol">47</span>: 775–776. [<a href="/pubmed/20587413" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Replication+of+KIF21B+as+a+susceptibility+locus+for+multiple+sclerosis&amp;volume=47&amp;publication_year=2010&amp;pages=775&amp;pmid=20587413&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib18"><span class="mixed-citation">Castellanos-Rubio A, Santin I, Martin-Pagola A et al: <span class="ref-title">Long-term and acute effects of gliadin on small intestine of patients on potentially pathogenic networks in celiac disease</span>. <span class="ref-journal">Autoimmunity</span>
2010; <span class="ref-vol">43</span>: 131–139. [<a href="/pubmed/19814655" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autoimmunity&amp;title=Long-term+and+acute+effects+of+gliadin+on+small+intestine+of+patients+on+potentially+pathogenic+networks+in+celiac+disease&amp;volume=43&amp;publication_year=2010&amp;pages=131&amp;pmid=19814655&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib19"><span class="mixed-citation">Trynka G, Zhernakova A, Romanos J et al: <span class="ref-title">Coeliac disease-associated risk variants in TNFAIP3 and REL implicate altered NF-kappaB signalling</span>. <span class="ref-journal">Gut</span>
2009; <span class="ref-vol">58</span>: 1078–1083. [<a href="/pubmed/19240061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Coeliac+disease-associated+risk+variants+in+TNFAIP3+and+REL+implicate+altered+NF-kappaB+signalling&amp;volume=58&amp;publication_year=2009&amp;pages=1078&amp;pmid=19240061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib20"><span class="mixed-citation">Szaflik JP, Majsterek I, Kowalski M et al: <span class="ref-title">Association between sorbitol dehydrogenase gene polymorphisms and type 2 diabetic retinopathy</span>. <span class="ref-journal">Exp Eye Res</span>
2008; <span class="ref-vol">86</span>: 647–652. [<a href="/pubmed/18289528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp+Eye+Res&amp;title=Association+between+sorbitol+dehydrogenase+gene+polymorphisms+and+type+2+diabetic+retinopathy&amp;volume=86&amp;publication_year=2008&amp;pages=647&amp;pmid=18289528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5117923&amp;issue-id=279467&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140271926828688title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Subjects and Methods</a></li><li><a href="#__sec3title">Results and Discussion</a></li><li><a href="#idm140271923177104title">Acknowledgments</a></li><li><a href="#idm140271900905824title">Notes</a></li><li><a href="#idm140271924518992title">Footnotes</a></li><li><a href="#sup1title">Supplementary Material</a></li><li><a href="#idm140271925646000title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5117923/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5117923/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5117923/pdf/ejhg2016120a.pdf">PDF (307K)</a></li> | <li><a href="#" data-citationid="PMC5117923" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5117923%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5117923%2F&amp;text=Ancestry-based%20stratified%20analysis%20of%20Immunochip%20data%20identifies%20novel%20associations%20with%20celiac%20disease" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5117923%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27650971" data-aiid="5117923" data-aid="5117923" data-iid="279467" data-domainid="1300" data-domain="ejhg" data-accid="PMC5117923" data-md5="63dea719674aa94ebeda8df431619bfe"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:10:32-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE8976F2DD59BFA100000000012200A1&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5117923%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5117923/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>