<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5703348/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study with additional genetic and post-transcriptional analyses reveals novel regulators of plasma factor XI levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5703348/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study with additional genetic and post-transcriptional analyses reveals novel regulators of plasma factor XI levels"><meta name="citation_authors" content="Bengt Sennblad, Saonli Basu, Johanna Mazur, Pierre Suchon, Angel Martinez-Perez, Astrid van Hylckama Vlieg, Vinh Truong, Yuhuang Li, Jesper R. Gådin, Weihong Tang, Vera Grossman, Hugoline G. de Haan, Niklas Handin, Angela Silveira, Juan Carlos Souto, Anders Franco-Cereceda, Pierre-Emmanuel Morange, France Gagnon, Jose Manuel Soria, Per Eriksson, Anders Hamsten, Lars Maegdefessel, Frits R. Rosendaal, Philipp Wild, Aaron R. Folsom, David-Alexandre Trégouët, Maria Sabater-Lleal"><meta name="citation_date" content="1 February 2017"><meta name="citation_issue" content="3"><meta name="citation_volume" content="26"><meta name="citation_firstpage" content="637"><meta name="citation_doi" content="10.1093/hmg/ddw401"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5703348/?report=abstract"><meta name="citation_pmid" content="28053049"><meta name="DC.Title" content="Genome-wide association study with additional genetic and post-transcriptional analyses reveals novel regulators of plasma factor XI levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Bengt Sennblad"><meta name="DC.Contributor" content="Saonli Basu"><meta name="DC.Contributor" content="Johanna Mazur"><meta name="DC.Contributor" content="Pierre Suchon"><meta name="DC.Contributor" content="Angel Martinez-Perez"><meta name="DC.Contributor" content="Astrid van Hylckama Vlieg"><meta name="DC.Contributor" content="Vinh Truong"><meta name="DC.Contributor" content="Yuhuang Li"><meta name="DC.Contributor" content="Jesper R. Gådin"><meta name="DC.Contributor" content="Weihong Tang"><meta name="DC.Contributor" content="Vera Grossman"><meta name="DC.Contributor" content="Hugoline G. de Haan"><meta name="DC.Contributor" content="Niklas Handin"><meta name="DC.Contributor" content="Angela Silveira"><meta name="DC.Contributor" content="Juan Carlos Souto"><meta name="DC.Contributor" content="Anders Franco-Cereceda"><meta name="DC.Contributor" content="Pierre-Emmanuel Morange"><meta name="DC.Contributor" content="France Gagnon"><meta name="DC.Contributor" content="Jose Manuel Soria"><meta name="DC.Contributor" content="Per Eriksson"><meta name="DC.Contributor" content="Anders Hamsten"><meta name="DC.Contributor" content="Lars Maegdefessel"><meta name="DC.Contributor" content="Frits R. Rosendaal"><meta name="DC.Contributor" content="Philipp Wild"><meta name="DC.Contributor" content="Aaron R. Folsom"><meta name="DC.Contributor" content="David-Alexandre Trégouët"><meta name="DC.Contributor" content="Maria Sabater-Lleal"><meta name="DC.Date" content="2017 Feb 1"><meta name="DC.Identifier" content="10.1093/hmg/ddw401"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study with additional genetic and post-transcriptional analyses reveals novel regulators of plasma factor XI levels"><meta property="og:type" content="article"><meta property="og:description" content="Coagulation factor XI (FXI) has become increasingly interesting for its role in pathogenesis of thrombosis. While elevated plasma levels of FXI have been associated with venous thromboembolism and ischemic stroke, its deficiency is associated with mild ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5703348/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5703348/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5703348/pdf/ddw401.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE883341DD5983910000000007AC0474.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/element/mml/optable/GeneralPunctuation.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5703348%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="mesh">MeSH</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC5703348</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2017 Feb 1; 26(3): 637–649. </span></div><div><span class="fm-vol-iss-date">Published online 2017 Jan 4. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddw401" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddw401</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5703348</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28053049">28053049</a></div></div></div></div><h1 class="content-title">Genome-wide association study with additional genetic and post-transcriptional analyses reveals novel regulators of plasma factor XI levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sennblad%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725700529952" co-class="co-affbox">Bengt Sennblad</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Basu%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725724547968" co-class="co-affbox">Saonli Basu</a>,<sup>3</sup> <a href="/pubmed/?term=Mazur%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725720707744" co-class="co-affbox">Johanna Mazur</a>,<sup>4</sup> <a href="/pubmed/?term=Suchon%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725703888384" co-class="co-affbox">Pierre Suchon</a>,<sup>5</sup> <a href="/pubmed/?term=Martinez-Perez%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725723524096" co-class="co-affbox">Angel Martinez-Perez</a>,<sup>6</sup> <a href="/pubmed/?term=van%20Hylckama%20Vlieg%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725723521824" co-class="co-affbox">Astrid van Hylckama Vlieg</a>,<sup>7</sup> <a href="/pubmed/?term=Truong%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725759116992" co-class="co-affbox">Vinh Truong</a>,<sup>8</sup> <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725716236640" co-class="co-affbox">Yuhuang Li</a>,<sup>1</sup> <a href="/pubmed/?term=G%26%23x000e5%3Bdin%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725723420928" co-class="co-affbox">Jesper R. Gådin</a>,<sup>1</sup> <a href="/pubmed/?term=Tang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725723418720" co-class="co-affbox">Weihong Tang</a>,<sup>9</sup> <a href="/pubmed/?term=Grossman%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725704317120" co-class="co-affbox">Vera Grossman</a>,<sup>4</sup> <a href="/pubmed/?term=de%20Haan%20HG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725723569232" co-class="co-affbox">Hugoline G. de Haan</a>,<sup>7</sup> <a href="/pubmed/?term=Handin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725722698816" co-class="co-affbox">Niklas Handin</a>,<sup>1</sup> <a href="/pubmed/?term=Silveira%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725722696608" co-class="co-affbox">Angela Silveira</a>,<sup>1</sup> <a href="/pubmed/?term=Souto%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725722079216" co-class="co-affbox">Juan Carlos Souto</a>,<sup>10</sup> <a href="/pubmed/?term=Franco-Cereceda%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725722448224" co-class="co-affbox">Anders Franco-Cereceda</a>,<sup>11</sup> <a href="/pubmed/?term=Morange%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725701495888" co-class="co-affbox">Pierre-Emmanuel Morange</a>,<sup>5</sup> <a href="/pubmed/?term=Gagnon%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725701493648" co-class="co-affbox">France Gagnon</a>,<sup>8</sup> <a href="/pubmed/?term=Soria%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725715433600" co-class="co-affbox">Jose Manuel Soria</a>,<sup>6</sup> <a href="/pubmed/?term=Eriksson%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725724205344" co-class="co-affbox">Per Eriksson</a>,<sup>1</sup> <a href="/pubmed/?term=Hamsten%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725706281504" co-class="co-affbox">Anders Hamsten</a>,<sup>1</sup> <a href="/pubmed/?term=Maegdefessel%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725706279296" co-class="co-affbox">Lars Maegdefessel</a>,<sup>1</sup> <a href="/pubmed/?term=Rosendaal%20FR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725704901200" co-class="co-affbox">Frits R. Rosendaal</a>,<sup>7</sup> <a href="/pubmed/?term=Wild%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725700748880" co-class="co-affbox">Philipp Wild</a>,<sup>4,</sup><sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=Folsom%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725704374736" co-class="co-affbox">Aaron R. Folsom</a>,<sup>9</sup> <a href="/pubmed/?term=Tr%26%23x000e9%3Bgou%26%23x000eb%3Bt%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725703734528" co-class="co-affbox">David-Alexandre Trégouët</a>,<sup>14,</sup><sup>15</sup> and  <a href="/pubmed/?term=Sabater-Lleal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049" class="affpopup" co-rid="_co_idm139725721720192" co-class="co-affbox">Maria Sabater-Lleal</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1</sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139725700529952"><h3 class="no_margin">Bengt Sennblad</h3><p>  <sup>1</sup>Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><p>  <sup>2</sup>Science for Life Laboratory, Department of Biochemistry and Biophysics, Stockholm University, Sweden</p><div>Find articles by <a href="/pubmed/?term=Sennblad%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Bengt Sennblad</a></div></div><div id="_co_idm139725724547968"><h3 class="no_margin">Saonli Basu</h3><p>  <sup>3</sup>University of Minnesota School of Public Health, Division of Biostatistics, MN, USA</p><div>Find articles by <a href="/pubmed/?term=Basu%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Saonli Basu</a></div></div><div id="_co_idm139725720707744"><h3 class="no_margin">Johanna Mazur</h3><p>  <sup>4</sup>University Medical Center Mainz, Johannes Gutenberg University Mainz, Mainz, Germany</p><div>Find articles by <a href="/pubmed/?term=Mazur%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Johanna Mazur</a></div></div><div id="_co_idm139725703888384"><h3 class="no_margin">Pierre Suchon</h3><p>  <sup>5</sup>Institut National pour la Santé et la Recherche Médicale (INSERM), Unité Mixte de Recherche en Santé (UMR_S) 1062, Nutrition Obesity and Risk of Thrombosis, Marseille, France; Aix-Marseille University</p><div>Find articles by <a href="/pubmed/?term=Suchon%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Pierre Suchon</a></div></div><div id="_co_idm139725723524096"><h3 class="no_margin">Angel Martinez-Perez</h3><p>  <sup>6</sup>Unitat de Genòmica de Malalties Complexes. Institut d'Investigació Biomèdica Sant Pau (IIB-Sant Pau), Barcelona, Spain</p><div>Find articles by <a href="/pubmed/?term=Martinez-Perez%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Angel Martinez-Perez</a></div></div><div id="_co_idm139725723521824"><h3 class="no_margin">Astrid van Hylckama Vlieg</h3><p>  <sup>7</sup>Department of Clinical Epidemiology, Leiden University Medical Center, Leiden, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Hylckama%20Vlieg%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Astrid van Hylckama Vlieg</a></div></div><div id="_co_idm139725759116992"><h3 class="no_margin">Vinh Truong</h3><p>  <sup>8</sup>Dalla Lana School of Public Health, University of Toronto, Toronto, Ontario, Canada</p><div>Find articles by <a href="/pubmed/?term=Truong%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Vinh Truong</a></div></div><div id="_co_idm139725716236640"><h3 class="no_margin">Yuhuang Li</h3><p>  <sup>1</sup>Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Yuhuang Li</a></div></div><div id="_co_idm139725723420928"><h3 class="no_margin">Jesper R. Gådin</h3><p>  <sup>1</sup>Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=G%26%23x000e5%3Bdin%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Jesper R. Gådin</a></div></div><div id="_co_idm139725723418720"><h3 class="no_margin">Weihong Tang</h3><p>  <sup>9</sup>University of Minnesota School of Public Health, Division of Epidemiology and Community Health, Minneapolis, MN, USA</p><div>Find articles by <a href="/pubmed/?term=Tang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Weihong Tang</a></div></div><div id="_co_idm139725704317120"><h3 class="no_margin">Vera Grossman</h3><p>  <sup>4</sup>University Medical Center Mainz, Johannes Gutenberg University Mainz, Mainz, Germany</p><div>Find articles by <a href="/pubmed/?term=Grossman%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Vera Grossman</a></div></div><div id="_co_idm139725723569232"><h3 class="no_margin">Hugoline G. de Haan</h3><p>  <sup>7</sup>Department of Clinical Epidemiology, Leiden University Medical Center, Leiden, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=de%20Haan%20HG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Hugoline G. de Haan</a></div></div><div id="_co_idm139725722698816"><h3 class="no_margin">Niklas Handin</h3><p>  <sup>1</sup>Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Handin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Niklas Handin</a></div></div><div id="_co_idm139725722696608"><h3 class="no_margin">Angela Silveira</h3><p>  <sup>1</sup>Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Silveira%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Angela Silveira</a></div></div><div id="_co_idm139725722079216"><h3 class="no_margin">Juan Carlos Souto</h3><p>  <sup>10</sup>Unitat de Trombosi i Hemostàsia, Hospital de Sant Pau, Barcelona, Spain</p><div>Find articles by <a href="/pubmed/?term=Souto%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Juan Carlos Souto</a></div></div><div id="_co_idm139725722448224"><h3 class="no_margin">Anders Franco-Cereceda</h3><p>  <sup>11</sup>Department of Molecular Medicine and Surgery, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Franco-Cereceda%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Anders Franco-Cereceda</a></div></div><div id="_co_idm139725701495888"><h3 class="no_margin">Pierre-Emmanuel Morange</h3><p>  <sup>5</sup>Institut National pour la Santé et la Recherche Médicale (INSERM), Unité Mixte de Recherche en Santé (UMR_S) 1062, Nutrition Obesity and Risk of Thrombosis, Marseille, France; Aix-Marseille University</p><div>Find articles by <a href="/pubmed/?term=Morange%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Pierre-Emmanuel Morange</a></div></div><div id="_co_idm139725701493648"><h3 class="no_margin">France Gagnon</h3><p>  <sup>8</sup>Dalla Lana School of Public Health, University of Toronto, Toronto, Ontario, Canada</p><div>Find articles by <a href="/pubmed/?term=Gagnon%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">France Gagnon</a></div></div><div id="_co_idm139725715433600"><h3 class="no_margin">Jose Manuel Soria</h3><p>  <sup>6</sup>Unitat de Genòmica de Malalties Complexes. Institut d'Investigació Biomèdica Sant Pau (IIB-Sant Pau), Barcelona, Spain</p><div>Find articles by <a href="/pubmed/?term=Soria%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Jose Manuel Soria</a></div></div><div id="_co_idm139725724205344"><h3 class="no_margin">Per Eriksson</h3><p>  <sup>1</sup>Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Eriksson%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Per Eriksson</a></div></div><div id="_co_idm139725706281504"><h3 class="no_margin">Anders Hamsten</h3><p>  <sup>1</sup>Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Hamsten%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Anders Hamsten</a></div></div><div id="_co_idm139725706279296"><h3 class="no_margin">Lars Maegdefessel</h3><p>  <sup>1</sup>Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Maegdefessel%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Lars Maegdefessel</a></div></div><div id="_co_idm139725704901200"><h3 class="no_margin">Frits R. Rosendaal</h3><p>  <sup>7</sup>Department of Clinical Epidemiology, Leiden University Medical Center, Leiden, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Rosendaal%20FR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Frits R. Rosendaal</a></div></div><div id="_co_idm139725700748880"><h3 class="no_margin">Philipp Wild</h3><p>  <sup>4</sup>University Medical Center Mainz, Johannes Gutenberg University Mainz, Mainz, Germany</p><p>  <sup>12</sup>Preventive Cardiology and Preventive Medicine, Center for Cardiology, University Medical Center of the Johannes Gutenberg-University Mainz, Mainz, Germany</p><p>  <sup>13</sup>DZHK (German Center for Cardiovascular Research), partner site RhineMain, Mainz, Germany</p><div>Find articles by <a href="/pubmed/?term=Wild%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Philipp Wild</a></div></div><div id="_co_idm139725704374736"><h3 class="no_margin">Aaron R. Folsom</h3><p>  <sup>9</sup>University of Minnesota School of Public Health, Division of Epidemiology and Community Health, Minneapolis, MN, USA</p><div>Find articles by <a href="/pubmed/?term=Folsom%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Aaron R. Folsom</a></div></div><div id="_co_idm139725703734528"><h3 class="no_margin">David-Alexandre Trégouët</h3><p>  <sup>14</sup>Sorbonne Universités, UPMC Univ. Paris 06, INSERM, UMR_S 1166, Team Genomics &amp; Pathophysiology of Cardiovascular Diseases, Paris, France</p><p>  <sup>15</sup>ICAN Institute for Cardiometabolism and Nutrition, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Tr%26%23x000e9%3Bgou%26%23x000eb%3Bt%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">David-Alexandre Trégouët</a></div></div><div id="_co_idm139725721720192"><h3 class="no_margin">Maria Sabater-Lleal</h3><p>  <sup>1</sup>Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Sabater-Lleal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28053049">Maria Sabater-Lleal</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139725724275104_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139725724275104_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139725724275104_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139725724275104_ai" style="display:none"><div class="fm-affl" id="ddw401-aff1" lang="en">  <sup>1</sup>Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden</div><div class="fm-affl" id="ddw401-aff2" lang="en">  <sup>2</sup>Science for Life Laboratory, Department of Biochemistry and Biophysics, Stockholm University, Sweden</div><div class="fm-affl" id="ddw401-aff3" lang="en">  <sup>3</sup>University of Minnesota School of Public Health, Division of Biostatistics, MN, USA</div><div class="fm-affl" id="ddw401-aff4" lang="en">  <sup>4</sup>University Medical Center Mainz, Johannes Gutenberg University Mainz, Mainz, Germany</div><div class="fm-affl" id="ddw401-aff5" lang="en">  <sup>5</sup>Institut National pour la Santé et la Recherche Médicale (INSERM), Unité Mixte de Recherche en Santé (UMR_S) 1062, Nutrition Obesity and Risk of Thrombosis, Marseille, France; Aix-Marseille University</div><div class="fm-affl" id="ddw401-aff6" lang="en">  <sup>6</sup>Unitat de Genòmica de Malalties Complexes. Institut d'Investigació Biomèdica Sant Pau (IIB-Sant Pau), Barcelona, Spain</div><div class="fm-affl" id="ddw401-aff7" lang="en">  <sup>7</sup>Department of Clinical Epidemiology, Leiden University Medical Center, Leiden, the Netherlands</div><div class="fm-affl" id="ddw401-aff8" lang="en">  <sup>8</sup>Dalla Lana School of Public Health, University of Toronto, Toronto, Ontario, Canada</div><div class="fm-affl" id="ddw401-aff9" lang="en">  <sup>9</sup>University of Minnesota School of Public Health, Division of Epidemiology and Community Health, Minneapolis, MN, USA</div><div class="fm-affl" id="ddw401-aff10" lang="en">  <sup>10</sup>Unitat de Trombosi i Hemostàsia, Hospital de Sant Pau, Barcelona, Spain</div><div class="fm-affl" id="ddw401-aff11" lang="en">  <sup>11</sup>Department of Molecular Medicine and Surgery, Karolinska Institutet, Stockholm, Sweden</div><div class="fm-affl" id="ddw401-aff12" lang="en">  <sup>12</sup>Preventive Cardiology and Preventive Medicine, Center for Cardiology, University Medical Center of the Johannes Gutenberg-University Mainz, Mainz, Germany</div><div class="fm-affl" id="ddw401-aff13" lang="en">  <sup>13</sup>DZHK (German Center for Cardiovascular Research), partner site RhineMain, Mainz, Germany</div><div class="fm-affl" id="ddw401-aff14" lang="en">  <sup>14</sup>Sorbonne Universités, UPMC Univ. Paris 06, INSERM, UMR_S 1166, Team Genomics &amp; Pathophysiology of Cardiovascular Diseases, Paris, France</div><div class="fm-affl" id="ddw401-aff15" lang="en">  <sup>15</sup>ICAN Institute for Cardiometabolism and Nutrition, Paris, France</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="ddw401-cor1"><sup>*</sup>To whom correspondence should be addressed at: Maria Sabater-Lleal, Cardiovascular Medicine Unit, Department of Medicine, Karolinska Institutet, Stockholm, Sweden. Tel: +46 (0)8-517 703 05; Fax: +46-(0)8-31 12 98; Email: <a href="mailto:dev@null" data-email="es.ik@laell.retabas.airam" class="oemail">es.ik@laell.retabas.airam</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139725724275104_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 May 20; Revised 2016 Oct 4; Accepted 2016 Nov 21.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139725724275104_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2017. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5703348/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Data.</strong></div><div><a href="/pmc/articles/PMC5703348/bin/ddw401_supp.zip" data-ga-action="click_feat_suppl">ddw401_supp.zip</a><span style="color:gray"> (1.0M)</span></div><div class="small guid">GUID:&nbsp;133890D0-2B96-4E50-B84C-9DEF2B05DD91</div></div></dd></dl></div><div id="idm139725723966288" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139725723966288title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first">Coagulation factor XI (FXI) has become increasingly interesting for its role in pathogenesis of thrombosis. While elevated plasma levels of FXI have been associated with venous thromboembolism and ischemic stroke, its deficiency is associated with mild bleeding. We aimed to determine novel genetic and post-transcriptional plasma FXI regulators.</p><p id="__p2">We performed a genome-wide association study (GWAS) for plasma FXI levels, using novel data imputed to the 1000 Genomes reference panel. Individual GWAS analyses, including a total of 16,169 European individuals from the ARIC, GHS, MARTHA and PROCARDIS studies, were meta-analysed and further replicated in 2,045 individuals from the F5L family, GAIT2 and MEGA studies. Additional association with activated partial thromboplastin time (aPTT) was tested for the top SNPs. In addition, a study on the effect of miRNA on FXI regulation was performed using <em>in silico</em> prediction tools and <em>in vitro</em> luciferase assays.</p><p id="__p3">Three loci showed robust, replicating association with circulating FXI levels: <em>KNG1</em> (rs710446, <em>P</em>-value = 2.07 × 10<sup>−302</sup>), <em>F11</em> (rs4253417, <em>P</em>-value  = 2.86 × 10<sup>−193</sup>), and a novel association in <em>GCKR (</em>rs780094, <em>P</em>-value  = 3.56 ×10<sup>−09</sup>), here for the first time implicated in FXI regulation. The two first SNPs (rs710446 and rs4253417) also associated with aPTT. Conditional and haplotype analyses demonstrated a complex association signal, with additional novel SNPs modulating plasma FXI levels in both the <em>F11</em> and <em>KNG1</em> loci. Finally, eight miRNAs were predicted to bind F11 mRNA. Over-expression of either miR-145 or miR-181 significantly reduced the luciferase activity in cells transfected with a plasmid containing FXI-3’UTR.</p><p id="__p4" class="p p-last">These results should open the door to new therapeutic targets for thrombosis prevention.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p5" class="p p-first">Coagulation factor XI (FXI) is a component of the contact pathway of coagulation with an important role in the propagation and stabilization of the thrombus and in the pathogenesis of thrombosis (<a href="#ddw401-B1" rid="ddw401-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601155">1–5</a>). Elevated plasma levels of FXI have been associated with increased risk of venous thromboembolism (VTE) and ischaemic stroke in most (<a href="#ddw401-B1" rid="ddw401-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601193">1</a>,<a href="#ddw401-B6" rid="ddw401-B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601208">6–8</a>) but not all (<a href="#ddw401-B9" rid="ddw401-B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601211">9</a>) studies. VTE and stroke constitute important, disabling and potentially fatal diseases with an incidence of 0.1–0.3% per year in Caucasians (<a href="#ddw401-B10" rid="ddw401-B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650597">10</a>,<a href="#ddw401-B11" rid="ddw401-B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601243">11</a>). Subjects with severe FXI deficiency have a reduced incidence of VTE and stroke (<a href="#ddw401-B7" rid="ddw401-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601247">7</a>,<a href="#ddw401-B12" rid="ddw401-B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601202">12</a>) and only mild bleeding, and several genetic studies have revealed that genetic variants in or close to the <em>F11</em> gene are associated with VTE (<a href="#ddw401-B13" rid="ddw401-B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650592">13–17</a>) and ischemic stroke (<a href="#ddw401-B18" rid="ddw401-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601138">18</a>). Moreover, an association has been shown between the activated partial thromboplastin time (aPTT), which measures the efficacy of the intrinsic coagulation pathway and has been shown to be a good predictor for VTE (<a href="#ddw401-B19" rid="ddw401-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601209">19</a>), and single nucleotide polymorphisms (SNPs) in the <em>F11</em> locus (<a href="#ddw401-B20" rid="ddw401-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650603">20</a>,<a href="#ddw401-B21" rid="ddw401-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601149">21</a>).</p><p id="__p6">Substantial evidence from mice and non-human primates indicates that FXI deficiency/inhibition is protective against experimentally induced thrombosis or stroke (<a href="#ddw401-B2" rid="ddw401-B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601238">2</a>,<a href="#ddw401-B22" rid="ddw401-B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601224">22</a>) without excessive bleeding. Moreover, emerging studies using antisense oligonucleotides that specifically reduce plasma FXI levels have proved effective for prevention of postoperative VTE without bleeding risk (<a href="#ddw401-B23" rid="ddw401-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601220">23</a>), suggesting that the plasma level of FXI is an important determinant of thrombosis and that understanding its regulation can have important implications in the diagnosis and management of serious disease outcomes such as VTE and stroke. Because of its implication in VTE and stroke, and the minor bleeding risk, FXI—as other components of the intrinsic coagulation pathway—have become increasingly interesting as therapeutic targets for thrombosis.</p><p id="__p7" class="p p-last">While the implications of changes in FXI plasma levels for disease have been widely studied, the genetic regulation of FXI remains incompletely understood. In a previous genome-wide association study (GWAS) using data imputed to the HapMap reference panel, we identified <em>KNG1</em> and <em>F11</em> as the two main loci determining plasma FXI levels (<a href="#ddw401-B24" rid="ddw401-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650595">24</a>). Recent studies have demonstrated that use of 1000 Genomes (1000G)-imputed data can substantially improve the ability of GWAS to discover novel loci (<a href="#ddw401-B25" rid="ddw401-B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601205">25</a>) and that the denser SNP coverage provided by imputation also allows us to study the genetic architecture of associated loci in greater detail, potentially revealing additional signals. Against this background, we have, in the present study, increased sample size 40-fold from the previous GWAS and have included novel data imputed to the 1000G reference panel in order to identify new loci that are important for FXI regulation and that could influence risk of venous and arterial thrombosis. In addition, we complement this genetic analysis with a study on the effect of microRNA (miRNA) on FXI regulation, with the overall aim to investigate novel regulatory mechanisms for FXI that could serve as therapeutic targets for treatment and prevention.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Results</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Three loci, <em>GCKR, KNG1</em> and <em>F11</em>, are robustly associated with circulating FXI levels</h3><p id="__p8" class="p p-first">In the discovery analysis, five studies (ARIC, GHS-1, GHS-2, MARTHA, and PROCARDIS), including 16,169 European individuals, performed GWAS analyses against natural-logarithm-transformed FXI levels (lnFXI). After quality control (QC) and filtering for minor allele frequency (MAF &gt; 0.01), 7,353,638 markers were included in a fixed-effect meta-analysis. Three loci contained associations with FXI plasma levels that were significant using the conventional genome-wide threshold of 5×10<sup>−</sup><sup>8</sup> (<a href="/pmc/articles/PMC5703348/figure/ddw401-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw401-F1" rid-ob="ob-ddw401-F1" co-legend-rid="lgnd_ddw401-F1"><span>Fig. 1</span></a>, <a href="/pmc/articles/PMC5703348/table/ddw401-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T1" rid-ob="ob-ddw401-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S2</a>). The most significant association (rs710446, Beta = −0.0888, <em>P</em>-value  = 2.07×10<sup>−</sup><sup>302</sup>) was located on chromosome 3, and created a non-synonymous substitution (Ile581Thr) in the <em>KNG1</em> gene; this confirms previous associations (<a href="#ddw401-B24" rid="ddw401-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650593">24</a>). A SNP on chromosome 4 (rs4253417, Beta = −0.0735, <em>P</em>-value  = 2.86×10<sup>−</sup><sup>193</sup>) was located in an intron on the structural gene coding for <em>F11</em>, which has previously been associated with FXI levels, albeit with different lead SNPs (<a href="#ddw401-B18" rid="ddw401-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601198">18</a>,<a href="#ddw401-B24" rid="ddw401-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650602">24</a>). Finally, a locus on chromosome 2 (rs780094, Beta = 0.0147, <em>P</em>-value  = 3.56×10<sup>−</sup><sup>09</sup>) was located in an intron of the glucokinase regulator (<em>GCKR)</em> gene, which has not previously been associated with FXI levels. The three lead SNPs were estimated, using GCTA (<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601213">26</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601167">27</a>), to explain 5.89% of the phenotypic variance in the PROCARDIS cohort. Summary results from the discovery meta-analysis have been deposited at the European Genome-phenome Archive (EGA), which is hosted by the EBI and the CRG, under accession number EGAS00001002123.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw401-F1" co-legend-rid="lgnd_ddw401-F1"><a href="/pmc/articles/PMC5703348/figure/ddw401-F1/" target="figure" rid-figpopup="ddw401-F1" rid-ob="ob-ddw401-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139725701295280" class="figure"><a href="/pmc/articles/PMC5703348/figure/ddw401-F1/" target="figure" rid-figpopup="ddw401-F1" rid-ob="ob-ddw401-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5703348_ddw401f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddw401f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5703348/bin/ddw401f1.jpg"></a></div><div id="largeobj_idm139725701295280" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5703348/figure/ddw401-F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw401-F1"><div><a class="figpopup" href="/pmc/articles/PMC5703348/figure/ddw401-F1/" target="figure" rid-figpopup="ddw401-F1" rid-ob="ob-ddw401-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p9">Results from the GWAS meta-analysis of natural-log-transformed plasma FXI levels in 16,169 European individuals (discovery cohort), adjusted for age, sex and population stratification; (<strong>a</strong>) Manhattan plot showing SNP <em>P</em>-values over chromosomes, (<strong>b</strong>) QQ-plot of expected and observed <em>P</em>-values (genomic control, Lambda = 1.006).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw401-T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p10"> Associations with lnFXI in the discovery and replication studies, fixed-effect meta-analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139725719836976" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">rs</th><th rowspan="1" colspan="1">Chr:pos</th><th rowspan="1" colspan="1">A1/A2</th><th rowspan="1" colspan="1">FreqA1</th><th rowspan="1" colspan="1">Beta<sub>disc</sub> (SE)</th><th rowspan="1" colspan="1">P<sub>disc</sub> (P<sub>het</sub>)</th><th rowspan="1" colspan="1">Beta<sub>repl</sub> (SE)</th><th rowspan="1" colspan="1">P<sub>repl</sub> (P<sub>het</sub>)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>Discovery</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs710446</td><td rowspan="1" colspan="1">3:186459927</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.590</td><td rowspan="1" colspan="1">−0.0888 (0.00239)</td><td rowspan="1" colspan="1">2.07 × 10<sup>−302</sup> (1.76 × 10<sup>−05</sup>)</td><td rowspan="1" colspan="1">−0.125 (0.00944)</td><td rowspan="1" colspan="1">9.74 × 10<sup>−40</sup> (5.55 × 10<sup>−16</sup>)</td></tr><tr><td rowspan="1" colspan="1">rs4253417</td><td rowspan="1" colspan="1">4:187199005</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.592</td><td rowspan="1" colspan="1">−0.0735 (0.00248)</td><td rowspan="1" colspan="1">2.86 × 10<sup>−193</sup> (2.23 × 10<sup>−07</sup>)</td><td rowspan="1" colspan="1">−0.0898 (0.00983)</td><td rowspan="1" colspan="1">6.9 × 10<sup>−20</sup> (1.11 × 10<sup>−15</sup>)</td></tr><tr><td rowspan="1" colspan="1">rs780094</td><td rowspan="1" colspan="1">2:27741237</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.407</td><td rowspan="1" colspan="1">0.0147 (0.00249)</td><td rowspan="1" colspan="1">3.56 × 10<sup>−09</sup> (0.0562)</td><td rowspan="1" colspan="1">0.0238 (0.00965)</td><td rowspan="1" colspan="1">0.0137 (0.262)</td></tr><tr><td rowspan="1" colspan="1"><em>Conditional</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4253421</td><td rowspan="1" colspan="1">4:187204937</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.118</td><td rowspan="1" colspan="1">−0.0498 (0.00373)</td><td rowspan="1" colspan="1">1.25 × 10<sup>−40</sup> (5.45 × 10<sup>−4</sup>)</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">rs76438938</td><td rowspan="1" colspan="1">3:186461524</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.971</td><td rowspan="1" colspan="1">−0.0765 (0.00711)</td><td rowspan="1" colspan="1">6.07 × 10<sup>−27</sup> (0.00211)</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td></tr></tbody></table></div><div id="largeobj_idm139725719836976" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5703348/table/ddw401-T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn1"><p id="__p11" class="p p-first-last">Results from the GWAMA analysis, number of tested SNPs in the discovery analysis are 7,353,638 and for the conditional analyses 7,353,629. A1 = allele 1 A2 = allele 2. For SNPs displaying inflated heterogeneity (rs710446, rs4253417 andrs4253421) in an analysis, a random-effect version of the analysis was performed; in all cases, these analyses confirmed the discovered associations for the investigated SNPs (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S2, S3, and S5</a>).</p></div></div></div><p id="__p12" class="p p-last">The top SNP from each of the three genome-wide significant loci were tested for replication in 2,058 European individuals from the French-Canadian family study on Factor V Leiden (F5L) study, GAIT2 and MEGA studies (see further information in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S1</a>) and the results were combined in a fixed-effect meta-analysis (<a href="/pmc/articles/PMC5703348/table/ddw401-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T1" rid-ob="ob-ddw401-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S3</a>). All three loci were significantly associated with lnFXI in our replication studies, <a href="/pmc/articles/PMC5703348/table/ddw401-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T1" rid-ob="ob-ddw401-T1" co-legend-rid=""><span>Table 1</span></a>), at the Bonferroni-adjusted threshold α = 0.0167 (<em>i.e.</em> 0.05/3).</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Conditional analysis identifies two independent signals at the <em>KNG1</em> and <em>F11</em> loci</h3><p id="__p13" class="p p-first-last">The associations of previously discovered SNPs in the loci with related phenotypes are listed in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S4</a>. In addition to the top SNPs (rs710446, rs4253417, rs780094), there was, at each locus, a set of additional SNPs that displayed significant association with FXI levels (<a href="/pmc/articles/PMC5703348/table/ddw401-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T2" rid-ob="ob-ddw401-T2" co-legend-rid=""><span>Table 2</span></a>). Regional plots (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Materials, Figs S3–S5</a>) suggested possible independent associations. To further investigate the association pattern at the three significantly associated loci, a secondary GWAS was performed in the discovery studies, conditioning on the three most significantly associated SNPs. The results of the fixed-effect meta-analysis of this conditional analysis are shown in <a href="/pmc/articles/PMC5703348/table/ddw401-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T1" rid-ob="ob-ddw401-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S5</a>. Additional, independently associated genome-wide significant associations were observed in the <em>F11</em> locus at rs4253421 (Beta = −0.0498, <em>P</em>-value  = 1.25×10<sup>−</sup><sup>40</sup>) and in the <em>KNG1</em> locus at rs76438938 (Beta = −0.0765, <em>P</em>-value  = 6.07×10<sup>−</sup><sup>27</sup>), to our knowledge previously not reported as associated with lnFXI. These SNPs were not in strong linkage disequilibrium (LD) with the lead <em>F11</em> (rs4253417, <em>r</em><sup>2 </sup>=<sup> </sup>0.084, D' = 0.99) and <em>KNG1</em> (rs710446, <em>r</em><sup>2 </sup>=<sup> </sup>0.033 and D' = 0.90) SNPs, respectively). Together with the three discovery SNPs, the two conditional SNPs were estimated, using GCTA (<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601219">26</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601214">27</a>), to explain 6.13% of the phenotypic variance in the PROCARDIS cohort. We used an approximate method, the stepwise selection procedure implemented in GCTA (<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601207">27</a>,<a href="#ddw401-B28" rid="ddw401-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601159">28</a>), to further investigate the signal pattern in the three loci. This analysis uses meta-analysis data combined with estimated pair-wise LD to identify the minimum subset of SNPs that jointly explains as much of the observed phenotype signal. For the <em>GCKR</em> and <em>F11</em> loci, this approximate stepwise analysis found no other signals than the lead SNPs from our discovery and conditional analyses (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S6</a>). However, for the <em>KNG1</em> locus, analyses suggested three additional signals, represented by rs5030095, rs9823431 and rs35575213, respectively, as lead SNPs (none of these is in strong LD with the lead SNPs from the discovery and conditional analyses, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S7</a>). We investigated these SNPs further in a haplotype analysis; however, the ensuing analyses focus on the five SNPs from our discovery and conditional analyses.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw401-T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p14">Summary of loci significantly associated with lnFXI in the discovery and conditional analyses.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139725723931712" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Start Chr:pos</th><th rowspan="1" colspan="1">Main gene</th><th rowspan="1" colspan="1">Discovery #GW-signif SNPs</th><th rowspan="1" colspan="1">Discovery top SNP</th><th rowspan="1" colspan="1">Conditional #GW-signif SNPs</th><th rowspan="1" colspan="1">Conditional top SNP</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">3:186000000</td><td rowspan="1" colspan="1">KNG1</td><td rowspan="1" colspan="1" align="center">470</td><td rowspan="1" colspan="1">rs710446</td><td rowspan="1" colspan="1" align="center">219</td><td rowspan="1" colspan="1">rs76438938</td></tr><tr><td rowspan="1" colspan="1">4:187000000</td><td rowspan="1" colspan="1">F11</td><td rowspan="1" colspan="1" align="center">395</td><td rowspan="1" colspan="1">rs4253417</td><td rowspan="1" colspan="1" align="center">84</td><td rowspan="1" colspan="1">rs4253421</td></tr><tr><td rowspan="1" colspan="1">2:27000000</td><td rowspan="1" colspan="1">GCKR</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1">rs780094</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1">–</td></tr></tbody></table></div><div id="largeobj_idm139725723931712" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5703348/table/ddw401-T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn2"><p id="__p15" class="p p-first-last">For convenience, a locus is here considered to be 2 Mbp long. For the <em>F11</em> locus, one additional SNP with genomic position 4:186990842 (<em>i.e</em>. just outside our conventional locus definition) was manually included.</p></div></div></div></div><div id="__sec5" class="sec"><h3 id="__sec5title">Haplotype analyses reveal complex association structures at the <em>F11</em> and <em>KNG1</em> loci</h3><p id="__p16" class="p p-first">To further resolve the LD patterns observed in the <em>F11</em> and <em>KNG1</em> loci, haplotype analyses were conducted in two of the discovery studies, PROCARDIS and MARTHA. For the <em>F11</em> locus, we performed a haplotype association analysis of five SNPs with lnFXI, including our two lead <em>F11</em> SNPs and three selected SNPs that have been associated with VTE in recent studies (rs3822057 and rs12186257 (<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601246">29–31</a>) and rs2036914 (<a href="#ddw401-B30" rid="ddw401-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601229">30–32</a>)). These five SNPs generated seven haplotypes with very similar distributions in PROCARDIS and MARTHA (<a href="/pmc/articles/PMC5703348/table/ddw401-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T3" rid-ob="ob-ddw401-T3" co-legend-rid=""><span>Table 3</span></a>). In both studies, all haplotypes carrying the T-allele at the lead rs4253417 were associated with decreased lnFXI levels. In addition, the single haplotype carrying the A-allele of the conditional rs4253421, and also carrying the rs4253417 T-allele, was associated with much lower lnFXI levels, supporting the result from our conditional analysis and suggesting that the rs4253417 and rs4253421 SNPs have additive independent effects on FXI levels.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw401-T3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p17">Results from haplotype analysis of the <em>F11</em> locus in the PROCARDIS and MARTHA studies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139725701367776" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">rs4253417</th><th rowspan="1" colspan="1">rs4253421</th><th rowspan="1" colspan="1">rs3822057</th><th rowspan="1" colspan="1">rs2036914</th><th rowspan="1" colspan="1">rs12186257</th><th rowspan="1" colspan="1">Haplotype frequencies</th><th rowspan="1" colspan="1">Additive effects on FXI levels Beta [CI] <em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">PROCARDIS</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.041</td><td rowspan="1" colspan="1">−0.017 [−0.059 to 0.025] p = 0.425</td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.308</td><td rowspan="1" colspan="1">−0.066 [−0.085 to 0.047] p = 1.18 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.122</td><td rowspan="1" colspan="1">−0.066 [−0.091 to 0.041] p = 1.92 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.108</td><td rowspan="1" colspan="1">−0.144 [-0.171 to 0.118] p = 5.93 10<sup>−27</sup></td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.019</td><td rowspan="1" colspan="1">−0.013 [−0.077 to 0.050] p = 0.681</td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.013</td><td rowspan="1" colspan="1">−0.013 [−0.100 to 0.072] p = 0.757</td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.355</td><td rowspan="1" colspan="1">Intercept</td></tr><tr><td rowspan="1" colspan="1">MARTHA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.042</td><td rowspan="1" colspan="1">−0.066 [−0.131 to 0.002] p = 0.044</td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.308</td><td rowspan="1" colspan="1">−0.093 [−0.122 to 0.065] p = 1.38 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.099</td><td rowspan="1" colspan="1">−0.072 [−0.120 to 0.025] p = 0.0028</td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.048</td><td rowspan="1" colspan="1">−0.136 [−0.185 to 0.088] p = 3.32 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.028</td><td rowspan="1" colspan="1">−0.061 [−0.140 to 0.018] p = 0.131</td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.013</td><td rowspan="1" colspan="1">−0.047 [−0.163 to 0.068] p = 0.421</td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.428</td><td rowspan="1" colspan="1">Intercept</td></tr></tbody></table></div><div id="largeobj_idm139725701367776" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5703348/table/ddw401-T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn3"><p id="__p18" class="p p-first-last">CI = confidence interval for Beta.</p></div></div></div><p id="__p19" class="p p-last">A similar haplotype analysis was performed on three SNPs from the <em>KNG1</em> locus, our lead discovery and conditional SNPs (rs710446 and rs76438938, respectively) and one of the SNPs suggested by the GCTA analysis (rs5030095). Again, haplotype results for the PROCARDIS and MARTHA studies (<a href="/pmc/articles/PMC5703348/table/ddw401-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T4" rid-ob="ob-ddw401-T4" co-legend-rid=""><span>Table 4</span></a>) were similar. The three SNPs generated five common haplotypes. The three haplotypes carrying the rs710446 C-allele consistently displayed a higher effect size than those carrying the T-allele. Moreover, the rs76438938 T-allele and the rs5030095 G-allele both, apparently independently, further increased the effect size (<a href="/pmc/articles/PMC5703348/table/ddw401-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T4" rid-ob="ob-ddw401-T4" co-legend-rid=""><span>Table 4</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw401-T4"><h3>Table 4.</h3><!--caption a7--><div class="caption"><p id="__p20">Results from haplotype analysis of the <em>KNG1</em> locus in the PROCARDIS and MARTHA studies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139725704275536" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">rs710446</th><th rowspan="1" colspan="1">rs76438938</th><th rowspan="1" colspan="1">rs5030095</th><th rowspan="1" colspan="1">Haplotype Frequencies</th><th rowspan="1" colspan="1">Additive effects on FXI levels Beta [CI] <em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">PROCARDIS</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.504</td><td rowspan="1" colspan="1">Intercept</td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.087</td><td rowspan="1" colspan="1">−0.011 [−0.046 to 0.019] <em>P = </em>0.425</td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.353</td><td rowspan="1" colspan="1">+0.073 [0.055 to 0.092] <em>P = </em>5.08 10<sup>−15</sup></td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.025</td><td rowspan="1" colspan="1">+0.045 [−0.019 to 0.107] <em>P = </em>0.162</td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.029</td><td rowspan="1" colspan="1">+0.147 [0.105 to 0.189] <em>P = </em>5.20 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1">MARTHA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.433</td><td rowspan="1" colspan="1">Intercept</td></tr><tr><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.101</td><td rowspan="1" colspan="1">−0.039 [−0.090 to 0.013] <em>P = </em>0.141</td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.412</td><td rowspan="1" colspan="1">+0.101 [0.073 to 0.128] <em>P = </em>1.27 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1">+0.087 [−0.028 to 0.202] <em>P = </em>0.137</td></tr><tr><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.031</td><td rowspan="1" colspan="1">+0.138 [0.076 to 0.200] <em>P = </em>1.13 10<sup>−5</sup></td></tr></tbody></table></div><div id="largeobj_idm139725704275536" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5703348/table/ddw401-T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn4"><p id="__p21" class="p p-first-last">CI = confidence interval for Beta.</p></div></div></div></div><div id="__sec6" class="sec"><h3 id="__sec6title">Associations with expression level in human liver suggest possible effects on transcription of the <em>GCKR</em> gene</h3><p id="__p22" class="p p-first-last">We investigated if any of the five lead SNPs from the discovery and conditional analyses could exert any <em>cis</em>-effect on expression of nearby genes (<a href="/pmc/articles/PMC5703348/table/ddw401-T5/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T5" rid-ob="ob-ddw401-T5" co-legend-rid=""><span>Table 5</span></a>). As FXI is almost exclusively expressed in the liver, we limited our analysis to liver tissue. No eQTL showed significant association with mRNA expression after adjustment for multiple testing (11 genes tested for the <em>KNG1</em> SNPs, 5 genes tested for the <em>F11</em> SNPs and 21 genes tested for the <em>GCKR</em> SNPs). The strongest association was observed between SNP rs780094 and the <em>GCKR</em> gene (<em>P</em>-value  = 0.00496, Bonferroni-adjusted significance threshold α = 0.05/21 = 0.00238).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw401-T5"><h3>Table 5.</h3><!--caption a7--><div class="caption"><p id="__p23">Results from eQTL associations of each of the three lead SNPs and the two conditional SNPs with the expression of neighboring genes (located within 500 Mbp) in human liver samples from the ASAP cohort.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139725705200272" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Gene</th><th rowspan="1" colspan="1">Fold change</th><th rowspan="1" colspan="1">Beta</th><th rowspan="1" colspan="1">SD</th><th rowspan="1" colspan="1"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs710446</td><td rowspan="1" colspan="1">TBCCD1</td><td rowspan="1" colspan="1" align="center">0.9924</td><td rowspan="1" colspan="1" align="center">−0.01104</td><td rowspan="1" colspan="1" align="center">0.03687</td><td rowspan="1" colspan="1" align="center">0.765</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">ST6GAL1</td><td rowspan="1" colspan="1" align="center">1.027</td><td rowspan="1" colspan="1" align="center">0.03888</td><td rowspan="1" colspan="1" align="center">0.03559</td><td rowspan="1" colspan="1" align="center">0.276</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">RFC4</td><td rowspan="1" colspan="1" align="center">1.003</td><td rowspan="1" colspan="1" align="center">0.004982</td><td rowspan="1" colspan="1" align="center">0.03113</td><td rowspan="1" colspan="1" align="center">0.873</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">CRYGS</td><td rowspan="1" colspan="1" align="center">0.9765</td><td rowspan="1" colspan="1" align="center">−0.03428</td><td rowspan="1" colspan="1" align="center">0.03117</td><td rowspan="1" colspan="1" align="center">0.273</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">DNAJB11</td><td rowspan="1" colspan="1" align="center">1.006</td><td rowspan="1" colspan="1" align="center">0.009068</td><td rowspan="1" colspan="1" align="center">0.05579</td><td rowspan="1" colspan="1" align="center">0.871</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">FETUB</td><td rowspan="1" colspan="1" align="center">1.035</td><td rowspan="1" colspan="1" align="center">0.04977</td><td rowspan="1" colspan="1" align="center">0.05485</td><td rowspan="1" colspan="1" align="center">0.365</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">KNG1</td><td rowspan="1" colspan="1" align="center">1.051</td><td rowspan="1" colspan="1" align="center">0.07149</td><td rowspan="1" colspan="1" align="center">0.03334</td><td rowspan="1" colspan="1" align="center">0.0332</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">HRG</td><td rowspan="1" colspan="1" align="center">1.021</td><td rowspan="1" colspan="1" align="center">0.02942</td><td rowspan="1" colspan="1" align="center">0.02593</td><td rowspan="1" colspan="1" align="center">0.258</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">EIF4A2</td><td rowspan="1" colspan="1" align="center">1.023</td><td rowspan="1" colspan="1" align="center">0.03331</td><td rowspan="1" colspan="1" align="center">0.03678</td><td rowspan="1" colspan="1" align="center">0.366</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">ADIPOQ</td><td rowspan="1" colspan="1" align="center">0.9744</td><td rowspan="1" colspan="1" align="center">−0.0374</td><td rowspan="1" colspan="1" align="center">0.02193</td><td rowspan="1" colspan="1" align="center">0.0897</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">AHSG</td><td rowspan="1" colspan="1" align="center">1.033</td><td rowspan="1" colspan="1" align="center">0.0466</td><td rowspan="1" colspan="1" align="center">0.03167</td><td rowspan="1" colspan="1" align="center">0.143</td></tr><tr><td rowspan="1" colspan="1">rs76438938</td><td rowspan="1" colspan="1">TBCCD1</td><td rowspan="1" colspan="1" align="center">0.9392</td><td rowspan="1" colspan="1" align="center">−0.09053</td><td rowspan="1" colspan="1" align="center">0.08882</td><td rowspan="1" colspan="1" align="center">0.309</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">ST6GAL1</td><td rowspan="1" colspan="1" align="center">1.038</td><td rowspan="1" colspan="1" align="center">0.05356</td><td rowspan="1" colspan="1" align="center">0.08612</td><td rowspan="1" colspan="1" align="center">0.535</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">RFC4</td><td rowspan="1" colspan="1" align="center">1.038</td><td rowspan="1" colspan="1" align="center">0.0536</td><td rowspan="1" colspan="1" align="center">0.07433</td><td rowspan="1" colspan="1" align="center">0.472</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">CRYGS</td><td rowspan="1" colspan="1" align="center">0.998</td><td rowspan="1" colspan="1" align="center">−0.002872</td><td rowspan="1" colspan="1" align="center">0.07547</td><td rowspan="1" colspan="1" align="center">0.97</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">DNAJB11</td><td rowspan="1" colspan="1" align="center">1.004</td><td rowspan="1" colspan="1" align="center">0.005966</td><td rowspan="1" colspan="1" align="center">0.1338</td><td rowspan="1" colspan="1" align="center">0.964</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">FETUB</td><td rowspan="1" colspan="1" align="center">0.9788</td><td rowspan="1" colspan="1" align="center">−0.03097</td><td rowspan="1" colspan="1" align="center">0.1322</td><td rowspan="1" colspan="1" align="center">0.815</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">KNG1</td><td rowspan="1" colspan="1" align="center">0.9837</td><td rowspan="1" colspan="1" align="center">−0.02375</td><td rowspan="1" colspan="1" align="center">0.08145</td><td rowspan="1" colspan="1" align="center">0.771</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">HRG</td><td rowspan="1" colspan="1" align="center">0.9874</td><td rowspan="1" colspan="1" align="center">−0.01829</td><td rowspan="1" colspan="1" align="center">0.06251</td><td rowspan="1" colspan="1" align="center">0.77</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">EIF4A2</td><td rowspan="1" colspan="1" align="center">0.9529</td><td rowspan="1" colspan="1" align="center">−0.06959</td><td rowspan="1" colspan="1" align="center">0.08888</td><td rowspan="1" colspan="1" align="center">0.435</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">ADIPOQ</td><td rowspan="1" colspan="1" align="center">1.092</td><td rowspan="1" colspan="1" align="center">0.1267</td><td rowspan="1" colspan="1" align="center">0.05254</td><td rowspan="1" colspan="1" align="center">0.0168</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">AHSG</td><td rowspan="1" colspan="1" align="center">0.9706</td><td rowspan="1" colspan="1" align="center">−0.04308</td><td rowspan="1" colspan="1" align="center">0.07623</td><td rowspan="1" colspan="1" align="center">0.573</td></tr><tr><td rowspan="1" colspan="1">rs4253417</td><td rowspan="1" colspan="1">TLR3</td><td rowspan="1" colspan="1" align="center">1.008</td><td rowspan="1" colspan="1" align="center">0.01143</td><td rowspan="1" colspan="1" align="center">0.05101</td><td rowspan="1" colspan="1" align="center">0.823</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">CYP4V2</td><td rowspan="1" colspan="1" align="center">0.964</td><td rowspan="1" colspan="1" align="center">−0.05288</td><td rowspan="1" colspan="1" align="center">0.04612</td><td rowspan="1" colspan="1" align="center">0.253</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">F11</td><td rowspan="1" colspan="1" align="center">0.9862</td><td rowspan="1" colspan="1" align="center">−0.02011</td><td rowspan="1" colspan="1" align="center">0.0353</td><td rowspan="1" colspan="1" align="center">0.57</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">KLKB1</td><td rowspan="1" colspan="1" align="center">1.015</td><td rowspan="1" colspan="1" align="center">0.02158</td><td rowspan="1" colspan="1" align="center">0.04275</td><td rowspan="1" colspan="1" align="center">0.614</td></tr><tr><td rowspan="1" colspan="1">rs4253421</td><td rowspan="1" colspan="1">TLR3</td><td rowspan="1" colspan="1" align="center">0.9976</td><td rowspan="1" colspan="1" align="center">−0.003492</td><td rowspan="1" colspan="1" align="center">0.07242</td><td rowspan="1" colspan="1" align="center">0.962</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">CYP4V2</td><td rowspan="1" colspan="1" align="center">0.8927</td><td rowspan="1" colspan="1" align="center">−0.1638</td><td rowspan="1" colspan="1" align="center">0.06467</td><td rowspan="1" colspan="1" align="center">0.0121</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">F11</td><td rowspan="1" colspan="1" align="center">1.018</td><td rowspan="1" colspan="1" align="center">0.02505</td><td rowspan="1" colspan="1" align="center">0.05012</td><td rowspan="1" colspan="1" align="center">0.618</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MTNR1A</td><td rowspan="1" colspan="1" align="center">1.033</td><td rowspan="1" colspan="1" align="center">0.04727</td><td rowspan="1" colspan="1" align="center">0.03058</td><td rowspan="1" colspan="1" align="center">0.124</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">KLKB1</td><td rowspan="1" colspan="1" align="center">1.046</td><td rowspan="1" colspan="1" align="center">0.06434</td><td rowspan="1" colspan="1" align="center">0.06055</td><td rowspan="1" colspan="1" align="center">0.289</td></tr><tr><td rowspan="1" colspan="1">rs780094</td><td rowspan="1" colspan="1">SNX17</td><td rowspan="1" colspan="1" align="center">1.007</td><td rowspan="1" colspan="1" align="center">0.0104</td><td rowspan="1" colspan="1" align="center">0.02593</td><td rowspan="1" colspan="1" align="center">0.689</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">PPM1G</td><td rowspan="1" colspan="1" align="center">0.982</td><td rowspan="1" colspan="1" align="center">−0.02613</td><td rowspan="1" colspan="1" align="center">0.02332</td><td rowspan="1" colspan="1" align="center">0.264</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">CCDC121</td><td rowspan="1" colspan="1" align="center">1.042</td><td rowspan="1" colspan="1" align="center">0.05874</td><td rowspan="1" colspan="1" align="center">0.03427</td><td rowspan="1" colspan="1" align="center">0.088</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">ZNF513</td><td rowspan="1" colspan="1" align="center">0.9945</td><td rowspan="1" colspan="1" align="center">−0.007998</td><td rowspan="1" colspan="1" align="center">0.01743</td><td rowspan="1" colspan="1" align="center">0.647</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">FNDC4</td><td rowspan="1" colspan="1" align="center">0.9462</td><td rowspan="1" colspan="1" align="center">−0.07973</td><td rowspan="1" colspan="1" align="center">0.03637</td><td rowspan="1" colspan="1" align="center">0.0295</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">IFT172</td><td rowspan="1" colspan="1" align="center">1.01</td><td rowspan="1" colspan="1" align="center">0.01428</td><td rowspan="1" colspan="1" align="center">0.02183</td><td rowspan="1" colspan="1" align="center">0.514</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MPV17</td><td rowspan="1" colspan="1" align="center">0.9983</td><td rowspan="1" colspan="1" align="center">−0.00252</td><td rowspan="1" colspan="1" align="center">0.03132</td><td rowspan="1" colspan="1" align="center">0.936</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">SUPT7L</td><td rowspan="1" colspan="1" align="center">0.9863</td><td rowspan="1" colspan="1" align="center">−0.01984</td><td rowspan="1" colspan="1" align="center">0.03583</td><td rowspan="1" colspan="1" align="center">0.58</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">DNAJC5G</td><td rowspan="1" colspan="1" align="center">1.016</td><td rowspan="1" colspan="1" align="center">0.02329</td><td rowspan="1" colspan="1" align="center">0.02236</td><td rowspan="1" colspan="1" align="center">0.299</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">KRTCAP3</td><td rowspan="1" colspan="1" align="center">1.013</td><td rowspan="1" colspan="1" align="center">0.01855</td><td rowspan="1" colspan="1" align="center">0.02715</td><td rowspan="1" colspan="1" align="center">0.495</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">C2orf16</td><td rowspan="1" colspan="1" align="center">1.028</td><td rowspan="1" colspan="1" align="center">0.03919</td><td rowspan="1" colspan="1" align="center">0.0212</td><td rowspan="1" colspan="1" align="center">0.0659</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">GTF3C2</td><td rowspan="1" colspan="1" align="center">0.9973</td><td rowspan="1" colspan="1" align="center">−0.003908</td><td rowspan="1" colspan="1" align="center">0.02108</td><td rowspan="1" colspan="1" align="center">0.853</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">GCKR</td><td rowspan="1" colspan="1" align="center">0.926</td><td rowspan="1" colspan="1" align="center">−0.1109</td><td rowspan="1" colspan="1" align="center">0.03904</td><td rowspan="1" colspan="1" align="center">0.00496</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">SLC4A1AP</td><td rowspan="1" colspan="1" align="center">1.008</td><td rowspan="1" colspan="1" align="center">0.01146</td><td rowspan="1" colspan="1" align="center">0.03296</td><td rowspan="1" colspan="1" align="center">0.728</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">NRBP1</td><td rowspan="1" colspan="1" align="center">0.9907</td><td rowspan="1" colspan="1" align="center">−0.01345</td><td rowspan="1" colspan="1" align="center">0.02053</td><td rowspan="1" colspan="1" align="center">0.513</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">GPN1</td><td rowspan="1" colspan="1" align="center">0.9959</td><td rowspan="1" colspan="1" align="center">−0.005879</td><td rowspan="1" colspan="1" align="center">0.03988</td><td rowspan="1" colspan="1" align="center">0.883</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">UCN</td><td rowspan="1" colspan="1" align="center">1.024</td><td rowspan="1" colspan="1" align="center">0.03448</td><td rowspan="1" colspan="1" align="center">0.03528</td><td rowspan="1" colspan="1" align="center">0.33</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">ZNF512</td><td rowspan="1" colspan="1" align="center">1.01</td><td rowspan="1" colspan="1" align="center">0.01398</td><td rowspan="1" colspan="1" align="center">0.0366</td><td rowspan="1" colspan="1" align="center">0.703</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">TRIM54</td><td rowspan="1" colspan="1" align="center">1.019</td><td rowspan="1" colspan="1" align="center">0.02703</td><td rowspan="1" colspan="1" align="center">0.02406</td><td rowspan="1" colspan="1" align="center">0.262</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">EIF2B4</td><td rowspan="1" colspan="1" align="center">0.9948</td><td rowspan="1" colspan="1" align="center">−0.007518</td><td rowspan="1" colspan="1" align="center">0.0214</td><td rowspan="1" colspan="1" align="center">0.726</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">SLC30A3</td><td rowspan="1" colspan="1" align="center">1.002</td><td rowspan="1" colspan="1" align="center">0.002337</td><td rowspan="1" colspan="1" align="center">0.02607</td><td rowspan="1" colspan="1" align="center">0.929</td></tr></tbody></table></div><div id="largeobj_idm139725705200272" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5703348/table/ddw401-T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn5"><p id="__p24" class="p p-first-last">Fold change is calculated from beta: fold change = 2<sup>beta</sup>.</p></div></div></div></div><div id="__sec7" class="sec"><h3 id="__sec7title">Association results are most likely linked to the roles of <em>KNG1</em> and <em>F11</em> in the contact coagulation pathway</h3><p id="__p25" class="p p-first">FXI has an important role in the propagation of the coagulation cascade by the contact pathway, which implicates the <em>F11</em> gene as a plausible causal gene for VTE. To investigate how the discovered associations relate to VTE, the five FXI-associated SNPs were also tested for association with aPTT in 10,908 individuals from the ARIC, F5L, GAIT2, and MARTHA studies, using fixed-effect meta-analysis. The aPTT test reflects the activity of the intrinsic pathway of coagulation and has been inversely associated with VTE risk (<a href="#ddw401-B33" rid="ddw401-B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601148">33</a>), To avoid bias due to technical differences in the measurements of aPTT across studies (see Material and Methods section), we performed a meta-analysis based on normalized effect sizes and standard errors. We additionally performed the meta-analysis based on sample-size-weighted <em>P</em>-values. We tested two models, model 1 was adjusted for age and sex covariates, and model 2 was additionally adjusted for lnFXI levels.</p><p id="__p26">The two meta-analysis models yielded consistent results (<a href="/pmc/articles/PMC5703348/table/ddw401-T6/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T6" rid-ob="ob-ddw401-T6" co-legend-rid=""><span>Table 6</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Materials, S8 and S9</a>). For model 1, the <em>F11</em> and <em>KNG1</em> SNPs all showed significant association in the expected directions with aPTT (<em>P</em>-value &lt; 0.01, Bonferroni-correction for five tests), but not the <em>GCKR</em> SNP (rs780094, <em>P</em>-value  = 0.125). With adjustment for lnFXI in model 2, the effect of all SNPs was strongly attenuated, and rs76438938 in the <em>KNG1</em> locus was no longer significant.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw401-T6"><h3>Table 6.</h3><!--caption a7--><div class="caption"><p id="__p27">Results of aPTT meta-analysis for models 1 and 2</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139725719846048" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">rs_locus</th><th rowspan="1" colspan="1">A1</th><th rowspan="1" colspan="1">A2</th><th rowspan="1" colspan="1">Freq1</th><th rowspan="1" colspan="1">Beta</th><th rowspan="1" colspan="1">SE</th><th rowspan="1" colspan="1"><em>P</em>-value</th><th rowspan="1" colspan="1">Direction</th><th rowspan="1" colspan="1"><em>P</em>_het</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Model 1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs710446</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">0.412164</td><td rowspan="1" colspan="1" align="center">−0.296423</td><td rowspan="1" colspan="1" align="center">0.009575</td><td rowspan="1" colspan="1" align="center">2.29 × 10<sup>−210</sup></td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1" align="center">6.10 × 10<sup>−04</sup></td></tr><tr><td rowspan="1" colspan="1">rs4253417</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">0.419123</td><td rowspan="1" colspan="1" align="center">−0.118784</td><td rowspan="1" colspan="1" align="center">0.009575</td><td rowspan="1" colspan="1" align="center">2.65 × 10<sup>−35</sup></td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1" align="center">0.194202</td></tr><tr><td rowspan="1" colspan="1">rs4253421</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.885623</td><td rowspan="1" colspan="1" align="center">−0.083696</td><td rowspan="1" colspan="1" align="center">0.009575</td><td rowspan="1" colspan="1" align="center">2.44 × 10<sup>−18</sup></td><td rowspan="1" colspan="1">−+–</td><td rowspan="1" colspan="1" align="center">0.130280</td></tr><tr><td rowspan="1" colspan="1">rs76438938</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.028115</td><td rowspan="1" colspan="1" align="center">−0.056257</td><td rowspan="1" colspan="1" align="center">0.009575</td><td rowspan="1" colspan="1" align="center">4.33 × 10<sup>−09</sup></td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1" align="center">0.877173</td></tr><tr><td rowspan="1" colspan="1">rs780094</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">0.592274</td><td rowspan="1" colspan="1" align="center">0.014700</td><td rowspan="1" colspan="1" align="center">0.009575</td><td rowspan="1" colspan="1" align="center">0.124693</td><td rowspan="1" colspan="1"> +++</td><td rowspan="1" colspan="1" align="center">0.770329</td></tr><tr><td rowspan="1" colspan="1">Model 2</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs710446</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">0.411999</td><td rowspan="1" colspan="1" align="center">−0.204035</td><td rowspan="1" colspan="1" align="center">0.009579</td><td rowspan="1" colspan="1" align="center">1.28 × 10<sup>−100</sup></td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1" align="center">4.54 × 10<sup>−05</sup></td></tr><tr><td rowspan="1" colspan="1">rs4253417</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">0.418964</td><td rowspan="1" colspan="1" align="center">−0.042991</td><td rowspan="1" colspan="1" align="center">0.009579</td><td rowspan="1" colspan="1" align="center">7.28 × 10<sup>−06</sup></td><td rowspan="1" colspan="1">−+–</td><td rowspan="1" colspan="1" align="center">0.614175</td></tr><tr><td rowspan="1" colspan="1">rs4253421</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.885574</td><td rowspan="1" colspan="1" align="center">−0.031183</td><td rowspan="1" colspan="1" align="center">0.009579</td><td rowspan="1" colspan="1" align="center">0.001138</td><td rowspan="1" colspan="1">− ++</td><td rowspan="1" colspan="1" align="center">0.255919</td></tr><tr><td rowspan="1" colspan="1">rs76438938</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.028138</td><td rowspan="1" colspan="1" align="center">−0.007322</td><td rowspan="1" colspan="1" align="center">0.009579</td><td rowspan="1" colspan="1" align="center">0.444662</td><td rowspan="1" colspan="1">− +++</td><td rowspan="1" colspan="1" align="center">0.280294</td></tr><tr><td rowspan="1" colspan="1">rs780094</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">0.592396</td><td rowspan="1" colspan="1" align="center">−0.004186</td><td rowspan="1" colspan="1" align="center">0.009579</td><td rowspan="1" colspan="1" align="center">0.662094</td><td rowspan="1" colspan="1">−+–</td><td rowspan="1" colspan="1" align="center">0.700256</td></tr></tbody></table></div><div id="largeobj_idm139725719846048" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5703348/table/ddw401-T6/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn6"><p id="__p28" class="p p-first-last">Results from the GWAMA fixed-effect meta-analysis based on normalized beta and standard errors. Because of the inflated heterogeneity for rs710446 in both model 1 and model 2 analyses, we performed a random-effect version of these analyses that confirmed the discovered association for this SNP (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S9</a>).</p></div></div></div><p id="__p29">Functional annotation suggests effects at the protein sequence level only for rs710446 and rs76438938, but possible regulatory effects for all five FXI-associated SNPs.</p><p id="__p30" class="p p-last">We used the HaploReg web-service (<a href="#ddw401-B34" rid="ddw401-B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601176">34</a>) to functionally annotate the five lead SNPs using information from publicly available genome annotation databases (<a href="/pmc/articles/PMC5703348/table/ddw401-T7/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T7" rid-ob="ob-ddw401-T7" co-legend-rid=""><span>Table 7</span></a>). The two <em>KNG1</em> SNPs are located in two distinct exons of the <em>KNG1</em> gene. The strongest associated SNP, rs710446, is a missense mutation (I581T), while rs76438938 is a nonsense mutation (R376*). The remaining three SNPs are all intronic and located in the <em>F11</em> and <em>GCKR</em> gene. There is substantial regulome information for the regions of interest. All SNPs, with the exception of rs4253417, were, based on liver reference epigenomes, reported to lie in predicted enhancer elements. These SNPs were also predicted to alter transcription factor binding motifs. However, only rs780094 has supporting ChIP-seq evidence for transcription factor and enhancer binding to this region in liver cell lines (<a href="/pmc/articles/PMC5703348/table/ddw401-T7/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T7" rid-ob="ob-ddw401-T7" co-legend-rid=""><span>Table 7</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw401-T7"><h3>Table 7.</h3><!--caption a7--><div class="caption"><p id="__p31">Summary of HaploReg results for the three discovery SNPs and the two conditional SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139725715357408" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Gene</th><th rowspan="1" colspan="1">Functional effect</th><th rowspan="1" colspan="1">Chromatin state in liver tissue</th><th rowspan="1" colspan="1">Proteins bound in liver cell lines</th><th rowspan="1" colspan="1">Transcription factor binding motifs changed</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs780094</td><td rowspan="1" colspan="1">GCKR</td><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1">Enhancers</td><td rowspan="1" colspan="1">FOXA2; MAFK; RXRA</td><td rowspan="1" colspan="1">AP-1; DBP; Foxi1; HNF4; Maf; Myf; RP58; TCF12; p300</td></tr><tr><td rowspan="1" colspan="1">rs710446</td><td rowspan="1" colspan="1">KNG1</td><td rowspan="1" colspan="1">missense; intron</td><td rowspan="1" colspan="1">Enhancers; Genic enhancers</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">AIRE; AP-1; CCNT2</td></tr><tr><td rowspan="1" colspan="1">rs76438938</td><td rowspan="1" colspan="1">KNG1</td><td rowspan="1" colspan="1">nonsense</td><td rowspan="1" colspan="1">Enhancers; Flanking Active TSS</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">GLI; LF-A1</td></tr><tr><td rowspan="1" colspan="1">rs4253417</td><td rowspan="1" colspan="1">F11</td><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">Gfi1</td></tr><tr><td rowspan="1" colspan="1">rs4253421</td><td rowspan="1" colspan="1">F11</td><td rowspan="1" colspan="1">intron</td><td rowspan="1" colspan="1">Enhancers</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">CHOP::CEBPalpha; NF-kappaB; RREB-1; Smad</td></tr></tbody></table></div><div id="largeobj_idm139725715357408" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5703348/table/ddw401-T7/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn7"><p id="__p32" class="p p-first-last">References for columns: Chromatin state in liver tissue (<a href="#ddw401-B73" rid="ddw401-B73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601181">73</a>), Proteins bound in liver cell lines (<a href="#ddw401-B74" rid="ddw401-B74" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601151">74</a>) and Transcription factor binding motifs changed (<a href="#ddw401-B75" rid="ddw401-B75" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601237">75</a>).</p></div></div></div></div><div id="__sec8" class="sec sec-last"><h3 id="__sec8title">Investigation of potential post-transcriptional regulation</h3><p id="__p33" class="p p-first-last">Previous results have indicated that post-transcriptional regulation of <em>F11</em> RNA expression in the liver can affect circulating FXI levels (<a href="#ddw401-B35" rid="ddw401-B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601164">35</a>). We therefore investigated a potential post-transcriptional mechanism for the regulation of FXI levels. Specifically, using an <em>in silico</em> consensus approach, we first predicted binding of miRNAs to the <em>F11</em> 3’ UTR, 5’ UTR or promoter regions (the top hits are shown in <a href="/pmc/articles/PMC5703348/table/ddw401-T8/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T8" rid-ob="ob-ddw401-T8" co-legend-rid=""><span>Table 8</span></a>). Eight miRNAs were predicted to target <em>F11</em> mRNA. Three of these have previously been experimentally tested (hsa-miR-181a-5p6, hsa-miR-16-5p6 and hsa-miR-23a-3p6; only the first was confirmed to bind to <em>F11</em>) (<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601174">36</a>), while the remaining five (hsa-miR-15a-5p, hsa-miR-15b-5p, hsa-miR-145-5p, hsa-miR-150-5p and hsa-miR-424-5p) remain to be experimentally validated. Of these latter five, miR-145 had the highest expression in liver (<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601221">36</a>) and was predicted to bind to the 3’ UTR region of <em>F11</em>, 383 bases from the <em>F11</em> stop codon. For this reason, we selected miR-145 for further <em>in vitro</em> validation. We used an <em>in vitro</em> luciferase assay in HEK293 cells to validate that miR-145 directly targets <em>F11</em> mRNA at its 3’-UTR; HEK293 cells, as the established cell source for luciferase reporter assays, were chosen for providing the highest possible transfection efficiency (<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601145">26</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601200">27</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601210">29</a>,<a href="#ddw401-B30" rid="ddw401-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601143">30</a>,<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601141">36</a>,<a href="#ddw401-B37" rid="ddw401-B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601163">37</a>). The previously validated miR-181 (<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601230">26</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601166">27</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601190">29–31</a>,<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601199">36</a>) was used as a positive miRNA control. Overexpression of either miR-145 or miR-181 significantly reduced the luciferase activity in cells transfected with pLS-FXI-3’UTR, but not the control pLS vector, suggesting that miR-145 (and miR-181) indeed targets <em>F11</em> mRNA through its 3’-UTR (<a href="/pmc/articles/PMC5703348/figure/ddw401-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw401-F2" rid-ob="ob-ddw401-F2" co-legend-rid="lgnd_ddw401-F2"><span>Fig. 2</span></a>) and regulate FXI levels. Finally, to connect this result to our genetic analysis, we investigated our significant <em>F11</em>-locus SNPs close to the miR-145 and miR-181binding sites. Three SNPs (rs1062547, rs4253430, and rs4253429) and a small deletion (4:187210318TA/A) were located within the reference 3’UTR for <em>F11</em> (see Supplementary Section 2); since all of them were significantly associated with FXI levels (<em>P</em>-values = 2.7×10<sup>−</sup><sup>41</sup>, 1.7×10<sup>−</sup><sup>41</sup>, 7.7×10<sup>−</sup><sup>27</sup>, and 5.1×10<sup>−</sup><sup>9</sup>, respectively), we first hypothesized that a possible effect of these SNPs could be to affect the binding of miR-145/miR-181 and thereby affect the regulation of FXI. To investigate if SNPs in the <em>F11</em> 3’UTR affect binding of miR-145 or miR-181 to the <em>F11</em> 3’ UTR, by changing the binding site sequence or through changes in secondary structure of the 3’UTR, we estimated haplotypes for these four SNPs in one of our cohorts (PROCARDIS, including 3525 individuals) and found four haplotypes. We investigated <em>in silico</em> the effect of miR-145 and miR-181a-5p binding in <em>F11</em> 3’UTR sequences corresponding to these four haplotypes using several public prediction tools that estimate the thermodynamic stability of the secondary structure of the putative miRNA-target pair, miRNA accessibility to binding site, and binding energy for the different sequences (See <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Section 2</a>). We did not find evidence that any of the SNPs had a functional effect on miRNA binding to the <em>F11</em> 3’ UTR, none of the investigated SNPs changed the binding target sequence, and there were no significant changes in miRNA binding energy between the different UTR haplotypes. Our results could not show any evidence of an effect of SNPs in 3’UTR on miRNA binding, suggesting that other mechanisms, probably the effect of LD with SNPs located somewhere else in the gene, might explain the association between these SNPs and FXI levels.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw401-F2" co-legend-rid="lgnd_ddw401-F2"><a href="/pmc/articles/PMC5703348/figure/ddw401-F2/" target="figure" rid-figpopup="ddw401-F2" rid-ob="ob-ddw401-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139725713827104" class="figure"><a href="/pmc/articles/PMC5703348/figure/ddw401-F2/" target="figure" rid-figpopup="ddw401-F2" rid-ob="ob-ddw401-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5703348_ddw401f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddw401f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5703348/bin/ddw401f2.jpg"></a></div><div id="largeobj_idm139725713827104" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5703348/figure/ddw401-F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw401-F2"><div><a class="figpopup" href="/pmc/articles/PMC5703348/figure/ddw401-F2/" target="figure" rid-figpopup="ddw401-F2" rid-ob="ob-ddw401-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p34">Results from the luciferase assay of miRNA binding to F11 mRNA. The plot shows luciferase activity after 24h of cells co-transfected with either an empty luciferase reporter plasmid (pLS), or a plasmid containing the FXI 3’UTR (pLS-FXI-3’UTR), together with scramble control, miR-145 or miR-181.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw401-T8"><h3>Table 8.</h3><!--caption a7--><div class="caption"><p id="__p35">Prediction of miRNAs binding to the F11 mRNA.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139725713824208" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">miRNA</th><th rowspan="1" colspan="1">MirWalk</th><th rowspan="1" colspan="1">TargetScan</th><th rowspan="1" colspan="1">miRanda</th><th rowspan="1" colspan="1">mirSVR</th><th rowspan="1" colspan="1">Tarbase</th><th rowspan="1" colspan="1">Predicted Binding</th><th rowspan="1" colspan="1">S<sub>i</sub> score</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">hsa-miR-15a-5p</td><td rowspan="1" colspan="1" align="center">6,0,3</td><td rowspan="1" colspan="1" align="center">−0.114</td><td rowspan="1" colspan="1" align="center">145</td><td rowspan="1" colspan="1" align="center">−0.1365</td><td rowspan="1" colspan="1" align="center">0.46</td><td rowspan="1" colspan="1" align="center">3' and promoter</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1">hsa-miR-15b-5p</td><td rowspan="1" colspan="1" align="center">5,0,3</td><td rowspan="1" colspan="1" align="center">−0.114</td><td rowspan="1" colspan="1" align="center">149</td><td rowspan="1" colspan="1" align="center">−0.1365</td><td rowspan="1" colspan="1" align="center">0.464</td><td rowspan="1" colspan="1" align="center">3' and promoter</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1">hsa-miR-145-5p</td><td rowspan="1" colspan="1" align="center">7,0,2</td><td rowspan="1" colspan="1" align="center">−0.287</td><td rowspan="1" colspan="1" align="center">155</td><td rowspan="1" colspan="1" align="center">−1.1535</td><td rowspan="1" colspan="1" align="center">0.751</td><td rowspan="1" colspan="1" align="center">3' and promoter</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1">hsa-miR-150-5p</td><td rowspan="1" colspan="1" align="center">6,0,4</td><td rowspan="1" colspan="1" align="center">−0.111</td><td rowspan="1" colspan="1" align="center">140</td><td rowspan="1" colspan="1" align="center">−0.2513</td><td rowspan="1" colspan="1" align="center">0.637</td><td rowspan="1" colspan="1" align="center">3' and promoter</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1">hsa-miR-424-5p</td><td rowspan="1" colspan="1" align="center">7,1,2</td><td rowspan="1" colspan="1" align="center">−0.133</td><td rowspan="1" colspan="1" align="center">157</td><td rowspan="1" colspan="1" align="center">−0.1338</td><td rowspan="1" colspan="1" align="center">0.555</td><td rowspan="1" colspan="1" align="center">3' and promoter</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1"><em>hsa-miR-181a-5p</em></td><td rowspan="1" colspan="1" align="center">6,3,2</td><td rowspan="1" colspan="1" align="center">−0.047</td><td rowspan="1" colspan="1" align="center">145</td><td rowspan="1" colspan="1" align="center">−0.3645</td><td rowspan="1" colspan="1" align="center">0.515</td><td rowspan="1" colspan="1" align="center">3', 5' and promoter</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1"><em>hsa-miR-16-5p</em></td><td rowspan="1" colspan="1" align="center">6,0,3</td><td rowspan="1" colspan="1" align="center">−0.114</td><td rowspan="1" colspan="1" align="center">146</td><td rowspan="1" colspan="1" align="center">−0.129</td><td rowspan="1" colspan="1" align="center">0.464</td><td rowspan="1" colspan="1" align="center">3’ and promoter</td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1"><em>hsa-miR-23a-3p</em></td><td rowspan="1" colspan="1" align="center">6,0,3</td><td rowspan="1" colspan="1" align="center">−0.073</td><td rowspan="1" colspan="1" align="center">140</td><td rowspan="1" colspan="1" align="center">−0.6707</td><td rowspan="1" colspan="1" align="center">0.664</td><td rowspan="1" colspan="1" align="center">3’ and promoter</td><td rowspan="1" colspan="1" align="center">4</td></tr></tbody></table></div><div id="largeobj_idm139725713824208" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5703348/table/ddw401-T8/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn8"><p id="__p36" class="p p-first-last">Italics denotes miRNA that previously identified to bind to F11 mRNA (<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601204">36</a>). For the MirWalk column, the comma-separated numbers denote the score for, in order, 3’ UTR,5’ UTR and Promoter region. References for columns: MirWalk (<a href="#ddw401-B67" rid="ddw401-B67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601196">67</a>), TargetScan (<a href="#ddw401-B61" rid="ddw401-B61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601165">61–64</a>), miRanda (<a href="#ddw401-B65" rid="ddw401-B65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601249">65</a>), mirSVR (<a href="#ddw401-B66" rid="ddw401-B66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650590">66</a>), Tarbase (<a href="#ddw401-B71" rid="ddw401-B71" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601147">71</a>).</p></div></div></div></div></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title">Discussion</h2><p id="__p37" class="p p-first">Coagulation factor XI is produced in the liver and circulates in the blood as a homodimer in complex with high molecular weight kininogen (HK or sometimes HMWK), also produced in the liver. HK also works as a co-factor in contact-induced activation of coagulation factor XII, which activates FXI; in this process, HK is cleaved by kallikrein to form bradykinin. Activated FXI, in turn, activates coagulation factor IX, which will start the common pathway of the coagulation cascade, leading to clot-formation.</p><p id="__p38">In this study, we have made a thorough investigation of genetic and potential post-transcriptional regulators of plasma FXI levels and their connection to activation of the contact coagulation pathway. In a previous study (<a href="#ddw401-B24" rid="ddw401-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650606">24</a>,<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601185">26</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601140">27</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601233">29</a>,<a href="#ddw401-B31" rid="ddw401-B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601175">31</a>,<a href="#ddw401-B32" rid="ddw401-B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601250">32</a>,<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601259">36</a>), we showed that the <em>KNG1</em> and <em>F11</em> loci are important determinants of plasma FXI levels based on a GWAS of a mere 398 individuals. The present study, representing a 40–fold increase in sample size (of note, the current sample is non-overlapping with that in the previous study), confirmed these associations, and identified additional independent signals in both loci. Moreover, we identified a novel third associated locus in the <em>GCKR</em> gene region, a gene that has been associated with multiple other traits, including coagulation factor VII, and Protein C (PC) levels (<a href="#ddw401-B1" rid="ddw401-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601160">1–5</a>,<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601215">26</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601225">27</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601146">29</a>,<a href="#ddw401-B30" rid="ddw401-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601223">30</a>,<a href="#ddw401-B32" rid="ddw401-B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601173">32</a>,<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601139">36</a>,<a href="#ddw401-B38" rid="ddw401-B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601231">38–40</a>). The <em>KNG1</em> and <em>F11</em> SNPs were also associated with aPTT, which support previously published associations of these loci with VTE (<a href="#ddw401-B1" rid="ddw401-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601177">1–5</a>,<a href="#ddw401-B13" rid="ddw401-B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650600">13</a>,<a href="#ddw401-B15" rid="ddw401-B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601171">15</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601184">27</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601222">29–31</a>,<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601179">36</a>,<a href="#ddw401-B41" rid="ddw401-B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601183">41</a>,<a href="#ddw401-B42" rid="ddw401-B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601162">42</a>). Finally, we confirmed the effect of miR-181 and demonstrated the effect of a novel miRNA, miR-145, in the regulation of plasma FXI concentration.</p><p id="__p39">Consistent with what was reported in the previous GWAS (<a href="#ddw401-B24" rid="ddw401-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650598">24</a>,<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601228">26</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601170">27</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601227">29–32</a>,<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601188">36</a>), the most robustly associated locus contained the <em>KNG1</em> gene. Several genes are located within the <em>KNG1</em> locus, including other biologically plausible genes such as those coding for histidine-rich glycoprotein (<em>HRG</em>) or adiponectin (<em>ADIPOQ</em>). Our results for the main discovery SNP, rs710446, as well as the main conditional SNP, rs76438938, point to <em>KNG1</em> as the main candidate gene in the region (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S10</a>). The C-allele of rs710446 causes an exonic, nonsynonymous mutation in <em>KNG1</em>, thus affecting the amino acid sequence of the protein, and therefore potentially also its function, while rs76438938 is a nonsense mutation. <em>KNG1</em> codes for HK, which has a central role in the transport and activation of FXI. FXI circulates as a homodimer in complex with HK. A probable mechanism for the effect of one or both SNPs on FXI levels could be by influence on the availability (or functionality) of HK for complex binding, thereby indirectly affecting the levels of circulating FXI. Our results show that the genetic structure underlying the <em>KNG1</em> association with FXI is more complex than previously thought. Our conditional and haplotype analyses, indeed, revealed that the T-allele of the nonsense rs76438938 was also associated with markedly increased FXI levels and that a third <em>KNG1</em> polymorphism could also be involved in the regulation of FXI levels. Further deep genetic investigations at the <em>KNG1</em> locus are required to better understand the <em>KNG1</em>-driven genetic regulation of FXI levels.</p><p id="__p40">The second strongly FXI-associated locus was the <em>F11</em> locus with two intronic SNPs, rs4253417 and rs4253421, that independently influenced FXI levels. The involvement of <em>F11</em>, the gene encoding FXI, could be expected. Neither rs4253417 nor rs4253421 (or SNPs in LD with these SNPs, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S11</a>) appears to have an effect on protein sequence, but functional annotation suggests potential regulatory functions; however, neither of the SNPs appears to have any effect on the mRNA expression of F11, nor on any surrounding gene. Consequently, we investigated if a post-transcriptional mechanism, specifically regulation by miRNA, could be operating. MiRNAs can have a regulating effect both on the transcriptional and post-transcriptional level. An increasing number of miRNAs have recently been involved in disease regulation, but the understanding of the role of miRNAs in the regulation of the coagulation pathway and their relevance to VTE remains in its infancy. A previous study (<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601189">26</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601245">27</a>,<a href="#ddw401-B30" rid="ddw401-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601212">30–32</a>,<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601182">36</a>) implicated a role for miR-181a-5p in FXI regulation. Our results confirm the effect of miR-181a-5 and combine bioinformatics-based prediction of miRNA binding to the 3’UTR region of F11 with validation using a luciferase assay to demonstrate a regulatory effect on FXI by the miRNA hsa-mir-145-5p. However, we could find no evidence supporting that the FXI associations in our study are mediated by SNPs affecting the binding of these two miRNAs.</p><p id="__p41">
<em>GCKR</em> codes for a glucokinase regulatory protein that inhibits glucokinase in liver and pancreatic-islet cells. This protein regulates the uptake and storage of dietary glucose and is important in the switch between feeding and fasting periods, and as such has been associated with several key metabolic pathways, especially for lipid and glucose metabolism. <em>GCKR</em> appears to be highly pleiotropic and genetic variations, specifically the SNP rs780094 and the nonsense SNP rs1260326, which are in LD with rs780094 (<em>r</em><sup>2</sup> = 0.92; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Materials, Tables S4 and S12</a>), have been associated with numerous metabolic traits, including type 2 diabetes (T2D) (<a href="#ddw401-B1" rid="ddw401-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601242">1–5</a>,<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601156">26</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601244">27</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601254">29–32</a>,<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601251">36</a>), maturity-onset diabetes of the young, nonalcoholic fatty liver disease, platelet count, and blood levels of glucose, fasting insulin, plasma-triglyceride, and total cholesterol (<a href="#ddw401-B1" rid="ddw401-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601217">1–5</a>,<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601157">26</a>,<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601252">27</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601253">29–32</a>,<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601144">36</a>), and also metabolites such as uric acid, creatinine and albumin (<a href="#ddw401-B27" rid="ddw401-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601180">27</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601206">29–32</a>). This is the first study associating <em>GCKR</em> with FXI levels, and the mechanism behind this association is therefore unclear. However, <em>GCKR</em> has also been associated with plasma viscosity (<a href="#ddw401-B24" rid="ddw401-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650599">24</a>,<a href="#ddw401-B26" rid="ddw401-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601192">26</a>,<a href="#ddw401-B30" rid="ddw401-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601232">30–32</a>) and coagulation proteins such as FVII (<a href="#ddw401-B1" rid="ddw401-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601258">1–</a><a href="#ddw401-B5" rid="ddw401-B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601216">5</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601169">29–32</a>,<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601154">36</a>,<a href="#ddw401-B43" rid="ddw401-B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601168">43</a>), and PC (<a href="#ddw401-B20" rid="ddw401-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650604">20</a>,<a href="#ddw401-B29" rid="ddw401-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601248">29–31</a>), and also with the inflammatory marker c-reactive protein (CRP) (<a href="#ddw401-B30" rid="ddw401-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601257">30–32</a>,<a href="#ddw401-B44" rid="ddw401-B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601255">44</a>,<a href="#ddw401-B45" rid="ddw401-B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601142">45</a>), which in turn has been shown to be associated with VTE (<a href="#ddw401-B33" rid="ddw401-B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601161">33</a>,<a href="#ddw401-B46" rid="ddw401-B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601241">46</a>). The <em>GCKR</em> gene is mainly expressed in liver and catalyzes the initial step in the utilization of glucose, providing glucose-6-phosphate. As suggested for PC and FVII (<a href="#ddw401-B39" rid="ddw401-B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601236">39</a>), <em>GCKR</em> could affect the function of several glycosylated proteins synthesized in the liver by altering the glucose required for their glycosylation. This could suggest a possible mechanism for the effect on FXI function—as FXI also has several N-glycosylation sites positioned at key areas of the glycoprotein that binds ligands involved in the coagulation cascade (<a href="#ddw401-B47" rid="ddw401-B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601194">47</a>)—but not its levels. However, a feedback regulatory loop on FXI could be mediated by FVII or PC within the coagulation cascade, both of which are regulated by <em>GCKR</em>. In our results, rs780094 displays no eQTL effects on either <em>F11</em> or <em>KNG1</em> expression. This strengthens the hypothesis of post-transcriptional or feedback regulatory mechanisms at a protein level or, alternatively, it might simply reflect that our study was underpowered to detect trans-effects.</p><p id="__p42">This might also be the reason why we could not see an association between rs780094 and aPTT. Plasma FXI has previously been robustly associated with aPTT and VTE. If FXI is causal in this association, we would therefore expect, from the Mendelian Randomization (MR) principle (<a href="#ddw401-B48" rid="ddw401-B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601201">48</a>), that genetic variants that regulate FXI levels are also associated with aPTT and VTE. We verified this for all SNPs in <em>KNG1</em> and <em>F11</em>, which had clear associations with aPTT, while the effect of the <em>GCKR</em> SNP rs780084 was much weaker and non-significant. When adjusting aPTT for FXI levels, all associations were markedly attenuated, indicating that, for all SNPs, the effect on aPTT is indeed mediated through their effect on FXI levels. As aPTT is a more complex phenotype, depending also on other factors than FXI, the power of the aPTT analysis would be expected to be lower than that of the lnFXI analysis (notice also that the two analyses are based on different sets of cohorts). The association <em>P</em>-values for aPTT are overall weaker than those for lnFXI and, in particular, the absence of a strong effect on FXI plasma levels for rs780084 resulted in an even weaker, insignificant, effect on aPTT. In addition, one of the criteria for MR to be applicable is the absence of pleiotropy, a condition that does not apply to <em>GCKR</em> (<a href="#ddw401-B31" rid="ddw401-B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601256">31</a>). Most likely, the effects of <em>GCKR</em> on other related phenotypes like FVII and PC levels act as confounders for the effects on FXI, and contribute to the loss of power to detect an association. Clearly, further studies are needed to understand the biological mechanism by which this regulatory protein influences plasma FXI.</p><p id="__p43">The use of 1000 genomes-imputed data has clear benefits compared with our previously published GWAS based on HapMap-imputed data: by allowing inclusion of more SNPs in the meta-analysis, it also enables discovery of additional signals with greater statistical significance—our top SNP in the <em>F11</em> locus rs4253417 has a <em>P</em>-value  = 2.86×10<sup>−</sup><sup>193</sup> in our discovery analysis, to be compared with the <em>P</em>-value  =1.02×10<sup>−</sup><sup>165</sup> for rs4253399 (in the present analysis), which was the top SNP in the analysis based on genotyped data in (<a href="#ddw401-B24" rid="ddw401-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650596">24</a>), cf. <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S4</a>. Moreover, the greater number of queried SNPs allowed a more comprehensive picture of the SNP structure of the locus (395 SNPs for the <em>KNG1</em> locus, 470 SNPs for the <em>F11</em> locus and five for the <em>GCKR</em> locus), and allowed the discovery of multiple signals in the locus.</p><p id="__p44" class="p p-last">To conclude, our results confirm <em>KNG1</em> and <em>F11</em> as important loci for FXI plasma level regulation, and discover <em>GCKR</em>, which has been reported to have pleiotropic effects on other coagulation proteins, as a novel interesting locus determining plasma FXI levels. Finally, we also demonstrate post-transcriptional regulation of FXI protein expression by a novel miRNA (miR-145-5p) binding to <em>F11</em>-UTR region. Moreover, our results suggest a potential mediating role of HK, the gene product of <em>KNG1</em>, and a future direction could therefore include studies on circulating HK levels. Given the role of FXI in determining risk of thrombosis, and the accumulating evidence suggesting that molecular determinants of FXI levels could be potential targets for anti-coagulation therapies that would carry a lesser bleeding risk compared with current anti-coagulant therapies, the novel genetic and post-transcriptional regulators of FXI in plasma found in the present study should open the door to new therapeutic targets for thrombosis prevention.</p></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">Materials and methods</h2><div id="__sec11" class="sec sec-first"><h3 id="__sec11title">Study sample</h3><p id="__p45" class="p p-first-last">The study consists of a discovery sample of 16,169 individuals of European ancestry from five studies (ARIC, PROCARDIS, GHS-1, GHS-2, and MARTHA) and a replication sample of 2,045 individuals from three studies (F5Leiden Cohort, MEGA, and GAIT2); for details, see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S1</a>. All studies were approved by the appropriate research ethics committees, and all respondents signed informed consent prior to participation.</p></div><div id="__sec12" class="sec"><h3 id="__sec12title">Phenotype determinations</h3><p id="__p46" class="p p-first">Plasma FXI concentration was determined using immunoassay (ELISA or electrochemiluminescence), coagulometry, a functional assay or a clotting assay. Results were expressed in U/ml, 1 unit being defined as the amount of activity of FXI in 1 ml of plasma. For details, please refer to <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S1</a>.</p><p id="__p47" class="p p-last">Activated partial thromboplastin time (aPTT) was measured in plasma, using automated coagulometry, either as the raw clotting time in seconds (three studies) or as a normalized ratio between the clotting time of the sample and the clotting of a laboratory reference standard (one cohort); for details see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S1</a>.</p></div><div id="__sec13" class="sec"><h3 id="__sec13title">Genotyping and imputations and QC</h3><p id="__p48" class="p p-first">Genome-wide genotyping chips from Illumina or Affymetrix were used by all studies (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S1</a>). All studies imputed variant dosages using the phase I version 3 or version 2 reference panels from the 1000 Genomes Project using MACH or IMPUTE (see further <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S1</a>) (<a href="#ddw401-B49" rid="ddw401-B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601235">49–52</a>).</p><p id="__p49" class="p p-last">Association analysis result files from the different studies were quality-controlled using the R package EasyQC (<a href="#ddw401-B53" rid="ddw401-B53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601152">53</a>) to harmonize any discrepancy in marker names between studies and to perform basic exclusions (incomplete and invalid data, monomorphic SNPs and SNPs with a sample size &lt; 30, a minor allele count &lt; 6, failed or low quality imputation, <em>i.e.</em> rsq &lt; 0.3 or allele mismatches). Duplicate SNPs (<em>i.e</em>. SNPs mapped to the same genomic position due to differences in the version of 1000G Phase I reference panel used for imputation) were excluded from analysis. The complete EasyQC summary output can be found in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S13</a>. A total of 9,966,893 SNPs were included in the meta-analysis. Only SNPs in autosomes were interrogated.</p></div><div id="__sec14" class="sec"><h3 id="__sec14title">Association analyses</h3><p id="__p50" class="p p-first">Detailed information about statistical software and cohort-specific analysis parameters for individual studies is given in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S1</a>. All meta-analyses were performed by two different analysts using two different softwares; fixed-effect meta-analysis was performed using METAL (<a href="#ddw401-B54" rid="ddw401-B54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601226">54</a>) and GWAMA (<a href="#ddw401-B55" rid="ddw401-B55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601234">55</a>), while random-effect analysis was performed with GWAMA only. Heterogeneity was assessed Cochran’s heterogeneity test as implemented in GWAMA. In all meta-analyses, adjustment for multiple testing was performed using the Bonferroni correction.</p><div id="__sec15" class="sec"><p></p><h4 id="__sec15title" class="inline">Discovery GWAS analysis </h4><p id="__p51" class="p p-first">For the discovery analysis, the ARIC, GHS1 and 2, GAIT2, Martha and PROCARDIS studies performed linear regression analyses using an additive model of inheritance between 1000 Genome-imputed SNPs and natural-logarithm-transformed FXI levels, with adjustments for age, sex, and accounting for population stratification and cryptic relatedness (adjustment for MDS or PCA components). PROCARDIS further used adjustment for coronary artery disease status and familial clustering to account for family structure. A total number of 16,169 European individuals were meta-analysed using an inverse-standard error model with fixed-effects. For significantly associated SNPs displaying significant heterogeneity among studies, a supplementary meta-analysis, using an inverse-standard error model with random-effects, was performed to control the discovered association. For convenience of analysis, we initially divided each chromosome into several 2 Mbp-sized ‘loci’ (while arbitrary, this preliminary locus definition worked well in practice for this study); each of these loci was represented by a <em>lead SNP</em>, defined as the most significantly associated SNP that fulfilled the following additional criteria of MAF &gt; 0.01 and consistent direction of effect size across discovery studies. We later refined these locus definitions using conditional analysis (see below).</p><p id="__p52" class="p p-last">Lead SNPs with <em>P</em>-values below the pre-defined genome-wide threshold of 5×10<sup>−</sup><sup>8</sup> (in total three SNPs) were chosen for replication. Replication was performed in a total of 2,045 individuals from the F5Leiden, GAIT2 and MEGA studies using, in each cohort, linear regression with natural-logarithm-transformed FXI, and adjusting for age, sex, and accounting for population stratification and cryptic relatedness (adjustment for MDS or PCA components). The cohort analysis results were combined using a fixed-effect meta-analysis; SNPs for which significant heterogeneity was indicated were confirmed using a random-effect meta-analysis.</p></div><div id="__sec16" class="sec"><p></p><h4 id="__sec16title" class="inline">Conditional analysis </h4><p id="__p53" class="p p-first">In order to investigate the existence of additional functional SNPs in the associated loci, all discovery studies (ARIC, GHS1 and 2, GAIT2, MARTHA and PROCARDIS) repeated the association analyses, using the same number of individuals and covariates, but now additionally conditioning on the three lead SNPs from the main analysis (rs710446, rs4253417 and rs780094). Meta-analysis and correction for multiple testing were performed as in the discovery analysis.</p><p id="__p54" class="p p-last">To investigate the existence of further independent signals present at the discovered loci, we used the approximate conditional joint selection (–<em>cojo-slct</em>) analysis implemented in the GCTA package (<a href="#ddw401-B28" rid="ddw401-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601240">28</a>). This analysis uses summary statistics from association analyses and an estimate of the LD structure of relevant regions in an approximate stepwise model selection of a set of independently associated SNPs for a pre-defined significance level (here <em>P</em>-value  &lt; 5×10<sup>−</sup><sup>8</sup>). We used the same summary results from the ARIC, GHS1 and 2, GAIT2, MARTHA and PROCARDIS studies as in discovery, and used the hard-called WTCCC data, described below, for a robust estimate of the LD-structure.</p></div><div id="__sec17" class="sec"><p></p><h4 id="__sec17title" class="inline">Haplotype analysis </h4><p id="__p55" class="p p-first-last">Haplotype analyses for the relevant associated SNPs were performed in the PROCARDIS and MARTHA studies using the THESIAS software (<a href="#ddw401-B56" rid="ddw401-B56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601195">56</a>), and association with lnFXI was adjusted for age, sex and population structure (principal components). Haplotype analyses were performed on best-guessed genotypes from imputed SNPs with imputation quality score greater than 0.8. Five SNPs were included in the haplotype analysis for the <em>F11</em> locus; the lead SNPs from the present study, rs4253417 and rs4253421, and three SNPs previously associated with VTE (rs3822057 and rs12186257 (<a href="#ddw401-B17" rid="ddw401-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650591">17</a>) and rs2036914 (<a href="#ddw401-B13" rid="ddw401-B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650589">13</a>)). These SNPs were all very well imputed in both studies. The SNPs were in strong LD and generated seven haplotypes with a frequency greater than 0.01 (<a href="/pmc/articles/PMC5703348/table/ddw401-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw401-T3" rid-ob="ob-ddw401-T3" co-legend-rid=""><span>Table 3</span></a>). For the <em>KNG1</em> locus, the aim was to include all SNPs identified in the discovery, conditional and the GCTA approximate conditional joint selection analyses; however, two of the SNPs from the GCTA analysis failed the imputation quality filter, so the final analysis comprised SNPs rs710446, rs76438938 and rs5030095, which generated five haplotypes.</p></div><div id="__sec18" class="sec sec-last"><p></p><h4 id="__sec18title" class="inline">Association with APTT </h4><p id="__p56" class="p p-first-last">The five candidate SNPs (<em>i.e</em>. the three lead SNPs from the main analysis plus the two additional SNPs from the conditional analyses, rs780094, rs710446, rs4253417, rs4253421 and rs76438938) were tested in the F5Leiden, GAIT2, MARTHA studies (a total of 10,908 subjects) for association with aPTT, adjusting for age, sex, and, if applicable, principal components (model 1), and also with additional adjustment for FXI levels (model 2). As aPTT was reported as the raw aPTT clotting time in three studies, but as a normalized aPTT ratio in one cohort, meta-analysis weighted by standard errors (<span id="IE1"><em>S</em><em>E</em></span>) was not applicable. We instead used two other meta-analysis approaches: In the first of these, we followed a previous approach to the same problem (<a href="#ddw401-B20" rid="ddw401-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650594">20</a>) and performed a sample-size-weighted meta-analysis on <em>P</em>-values weighted by the sample size (<em>N</em>); in the second, we created normalized aPTT measures (<span id="IE2">‖<em>a</em><em>P</em><em>T</em><em>T</em>‖</span>) by their standard deviation (<span id="IE3"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="MM3" overflow="scroll"><mi>σ</mi><mo>=</mo><mo></mo><mi>S</mi><mi>E</mi><mo></mo><mo>×</mo><mo></mo><msqrt><mi>N</mi></msqrt></math></span><span class="MathJax MathJax_Processed" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="MM3" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="mi" id="MathJax-Span-3" style="font-family: MathJax_Math; font-style: italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.001em;"></span></span><span class="mo" id="MathJax-Span-4" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mo" id="MathJax-Span-5"></span><span class="mi" id="MathJax-Span-6" style="font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.032em;"></span></span><span class="mi" id="MathJax-Span-7" style="font-family: MathJax_Math; font-style: italic;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span class="mo" id="MathJax-Span-8"></span><span class="mo" id="MathJax-Span-9" style="font-family: MathJax_Main; padding-left: 0.222em;">×</span><span class="mo" id="MathJax-Span-10"></span><span class="msqrt" id="MathJax-Span-11"><span style="display: inline-block; position: relative; width: 1.83em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0.833em;"><span class="mrow" id="MathJax-Span-12"><span class="mi" id="MathJax-Span-13" style="font-family: MathJax_Math; font-style: italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.552em, 1001em, 3.902em, -1000em); top: -4.56em; left: 0.833em;"><span style="display: inline-block; position: relative; width: 0.997em; height: 0px;"><span style="position: absolute; font-family: MathJax_Main; top: -3.977em; left: -0.084em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; font-family: MathJax_Main; top: -3.977em; left: 0.303em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.022em, 1000.85em, 4.332em, -1000em); top: -4.03em; left: 0em;"><span style="font-family: MathJax_Main;">√</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="MM3" overflow="scroll"><mi>σ</mi><mo>=</mo><mo></mo><mi>S</mi><mi>E</mi><mo></mo><mo>×</mo><mo></mo><msqrt><mi>N</mi></msqrt></math></script></span>), that is, <span id="IE4"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="MM4" overflow="scroll"><mfenced open="‖" close="‖" separators="|"><mrow><mi>a</mi><mi>P</mi><mi>T</mi><mi>T</mi></mrow></mfenced><mo>=</mo><mo></mo><mfrac><mrow><mi>a</mi><mi>P</mi><mi>T</mi><mi>T</mi></mrow><mrow><mi>σ</mi></mrow></mfrac></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-2-Frame" tabindex="0" style="font-size: 121%;"></span><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="MM4" overflow="scroll"><mfenced open="‖" close="‖" separators="|"><mrow><mi>a</mi><mi>P</mi><mi>T</mi><mi>T</mi></mrow></mfenced><mo>=</mo><mo></mo><mfrac><mrow><mi>a</mi><mi>P</mi><mi>T</mi><mi>T</mi></mrow><mrow><mi>σ</mi></mrow></mfrac></math></script></span>.</p></div></div><div id="__sec19" class="sec"><h3 id="__sec19title">eQTL analyses</h3><p id="__p57" class="p p-first-last">The five candidate SNPs derived from the discovery and conditional analyses (rs710446, rs4253417, rs780094, rs76438938 and rs4253421) were subsequently tested for association with expression levels of nearby genes (defined as located within 500 Mbp in both directions) in a total of 211 human liver samples obtained from the Advanced Study of Aortic Pathology (ASAP) study (<a href="#ddw401-B57" rid="ddw401-B57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601172">57</a>). The ASAP study is a prospective, single-centre, observational cohort study of patients with aortic valve and ascending aortic disease, undergoing elective open-heart surgery at the Cardiothoracic Surgery Unit, Karolinska University Hospital in Stockholm, Sweden. The inclusion criteria were patients aged 18 or above with aortic valve disease (<em>i.e</em>. aortic stenosis or regurgitation) and/or ascending aorta dilatation (aneurysm or ectasia of the ascending aorta including the aortic root) but devoid of coronary artery disease (defined as lacking significant stenosis on coronary angiogram) and primarily not planned for other concomitant valve surgery. Global gene expression in the ASAP cohort was measured using the Affymetrix GeneChip® Human Exon 1.0 ST array and the <em>core</em> subset of Affymetrix meta probe sets. For specific details on quality control procedures, see (<a href="#ddw401-B58" rid="ddw401-B58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601191">58</a>). SNPs rs710446 and rs780094 were genotyped in ASAP on the Illumina 1M chip, while rs4253417, rs76438938 and rs4253421 were imputed to 1000G. Genotypes of the five candidate SNPs were tested for association with gene expression levels using additive linear models in R, with each genotype being coded as 0, 1 or 2.</p></div><div id="__sec20" class="sec"><h3 id="__sec20title">Functional analysis and LD estimation</h3><p id="__p58" class="p p-first">We used HaploReg v4 (<a href="#ddw401-B34" rid="ddw401-B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601203">34</a>) to explore and summarize the functional annotation of associated SNPs from our analyses.</p><p id="__p59" class="p p-last">To obtain robust estimates of the LD structure, we used the controls from the Wellcome Trust Case Control Consortium (WTCCC). We imputed the WTCCC genotyped data for 5667 subjects into the phase I version 3 reference panels from the 1000 Genomes Project using IMPUTE. We then hard-called this data using an in-house C ++ program and a cutoff genotype probability &gt; 0.8 and converted the result to plink binary format. For estimation of pairwise LD between specified SNPs, we used the plink v1.90b2m (<a href="#ddw401-B59" rid="ddw401-B59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650605">59</a>,<a href="#ddw401-B60" rid="ddw401-B60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601187">60</a>).</p></div><div id="__sec21" class="sec sec-last"><h3 id="__sec21title">miRNA analysis</h3><div id="__sec22" class="sec sec-first"><p></p><h4 id="__sec22title" class="inline">Prediction of miRNA binding </h4><p id="__p60" class="p p-first-last">To find miRNA predicted to bind to the 3’ UTR, the 5’ UTR or the promoter region of the F11 mRNA, we devised a consensus algorithm combining the results from several public methods and databases for prediction of miRNA binding. In brief, an initial score <span id="IE5"><em>S</em><sub><em>i</em></sub> =&nbsp;0</span> was assigned to each miRNA; this score was then incremented by 1 for each of the following criteria that was true: <em>i</em>) TargetScan v 6.2 (<a href="#ddw401-B61" rid="ddw401-B61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601150">61–64</a>) context+ score &lt; 0.045; <em>ii</em>) miRanda v 3.3a (<a href="#ddw401-B65" rid="ddw401-B65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601186">65</a>) score &gt; 145; <em>iii</em>) mirSVR (<a href="#ddw401-B66" rid="ddw401-B66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651650601">66</a>) score ≤ 0.1; <em>iv</em>) positive prediction from miRWalk 2.0 (67) 5’ UTR algorithms ≥ 50%; <em>v</em>) positive prediction from miRWalk 2.0 3’ UTR algorithms &gt;50%; <em>vi</em>) positive prediction from miRWalk 2.0 promoter ≥ 50%. TargetScan and miRanda was set to use the mirBase release 21 database (<a href="#ddw401-B68" rid="ddw401-B68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601153">68</a>,<a href="#ddw401-B69" rid="ddw401-B69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601218">69</a>) and a cutoff <span id="IE6"><em>S</em><sub><em>i</em></sub>&nbsp;≥&nbsp;4</span> was required for a miRNA to be selected. We limited our search to miRNA that were expressed in the liver, which was defined by any of the following criteria were fulfilled: 1) Clone count &gt; 1 in SmirnaDB version 2009-05-08. 2) Normalized clone count &gt; 0.00072 in microRNA.org version 2010-11-01 (<a href="#ddw401-B70" rid="ddw401-B70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601239">70</a>). 3) Reported as expressed in the liver in (<a href="#ddw401-B36" rid="ddw401-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601178">36</a>): supporting <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S1</a>. Lastly, Tarbase (<a href="#ddw401-B71" rid="ddw401-B71" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601158">71</a>) was used to investigate if miRNAs found were already experimentally validated.</p></div><div id="__sec23" class="sec sec-last"><p></p><h4 id="__sec23title" class="inline">Luciferase reporter assay </h4><p id="__p61" class="p p-first-last">To validate if the miRNA of interest changed the expression of FXI, a luciferase report assay was performed as described (<a href="#ddw401-B72" rid="ddw401-B72" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_651601197">72</a>). In brief, HEK293 cells were seeded on 24-well plates (1 × 10<sup>5</sup> cells/well). At 50–60% confluence, cells were transfected with luciferase reporter plasmid pLS, pLS-FXI-3’UTR (100 ng/well, Active Motif, Switchgear Genomics), together with scramble control, miR-145 or miR-181 (10 nM final concentration) using DharmaFECT Duo Transfection Reagent (Switchgear) according to the manufacturer’s protocol. After a 24-hour transfection period, luciferase activity was quantified using the LightSwitch™ Luciferase Assay Kit (Switchgear) according to the manufacturer’s protocol.</p></div></div></div><div id="__sec24" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec24title">Supplementary Material</h2><!--/article/body/sec/--><p id="__p62" class="p p-first-last">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material</a> is available at <em>HMG</em> online.</p></div><div id="__sec25" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec25title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>Supplementary Data</h4><div class="sup-box half_rhythm" id="idm139725720200160"><a href="/pmc/articles/PMC5703348/bin/ddw401_supp.zip" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.0M, zip)</sup></div></div></div><div id="idm139725720198368" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139725720198368title">Acknowledgements</h2><div class="sec"><p id="__p64">We are grateful to Mary Cushman for advice in the project design and selection of samples and to Mattias Frånberg for valuable discussions on analysis methods. The authors thank the staff and participants of the ARIC study for their important contributions.</p><p id="__p65">
<em>Conflict of Interest statement</em>. None declared.</p><div id="__sec26" class="sec"><h3 id="__sec26title">Funding </h3><p id="__p66" class="p p-first-last">The Atherosclerosis Risk in Communities (ARIC) Study is carried out as a collaborative study supported by National Heart, Lung, and Blood Institute contracts (HHSN268201100005C, HHSN268201100006C, HHSN268201100007C, HHSN268201100008C, HHSN268201100009C, HHSN268201100010C, HHSN268201100011C, and HHSN268201100012C), R01HL087641, R01HL59367 and R01HL086694; National Human Genome Research Institute contract U01HG004402; and National Institutes of Health contract HHSN268200625226C. Infrastructure was partly supported by Grant Number UL1RR025005, a component of the National Institutes of Health and NIH Roadmap for Medical Research. PROCARDIS was supported by the European Community Sixth Framework Program (LSHM-CT- 2007-037273), AstraZeneca, the British Heart Foundation, the Wellcome Trust (Contract No. 075491/Z/04), the Swedish Research Council, the Knut and Alice Wallenberg Foundation, the Swedish Heart-Lung Foundation, the Torsten and Ragnar Söderberg Foundation, the Strategic Cardiovascular and Diabetes Programs of Karolinska Institutet and Stockholm County Council, the Foundation for Strategic Research and the Stockholm County Council. Bengt Sennblad acknowledges funding from the Magnus Bergvall foundation and the foundation for old servants. Maria Sabater-Lleal is recipient of the European Hematology Association and the International Society of Thrombosis and Hemostasis (EHA-ISTH) fellowship and acknowledges funding from Swedish Heart-Lung Foundation (20130399), Åke Wiberg (400546009), Lars Hiertas Minne (F02014-0357), and Magnus Bergvall's foundations (2014-00242). The Gutenberg Health Study is funded through the government of Rhineland-Palatinate (‘Stiftung Rheinland-Pfalz für Innovation’, contract AZ 961-386261/733), the research programs ‘Wissenschafft Zukunft’ and ‘Center for Translational Vascular Biology (CTVB)’ of the Johannes Gutenberg-University of Mainz, and its contract with Boehringer Ingelheim and PHILIPS Medical Systems, including an unrestricted grant for the Gutenberg Health Study. Philipp S. Wild is funded by the Federal Ministry of Education and Research (BMBF 01EO1503) and he is PI of the German Center for Cardiovascular Research (DZHK). The MARTHA project was supported by grants from the Program Hospitalier de Recherche Clinique. Statistical data analysis in MARTHA was performed on the C2BIG high-performance computing system funded by the Region Ile de France, the Université Pierre et Marie Curie and the ICAN Institute for Cardiometabolism and Nutrition (ANR-10-IAHU-05). GAIT2 was supported partially by grants PI-11/0184, Pi-14/0582, RD12/0042/0032 and UIN2013-50833 from the Instituto Carlos III (Fondo de Investigación Sanitaria—FIS). The French-Canadian family study on Factor V Leiden was partially supported by the Canadian Institutes of Health Research and the Heart and Stroke Foundation of Canada; France Gagnon is supported by a Canada Research Chair in Genetic Epidemiology.</p></div></div></div><div id="idm139725720191520" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139725720191520title">References </h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="ddw401-B1">1. <span class="mixed-citation">
Yang D.T., Flanders M.M., Kim H., Rodgers G.M. (2006) 
<span class="ref-title">Elevated factor XI activity levels are associated with an increased odds ratio for cerebrovascular events</span>. <span class="ref-journal">Am. J. Clin. Pathol</span>., <span class="ref-vol">126</span>, 411–415. [<a href="/pubmed/16880142" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Clin.+Pathol&amp;title=Elevated+factor+XI+activity+levels+are+associated+with+an+increased+odds+ratio+for+cerebrovascular+events&amp;author=D.T.+Yang&amp;author=M.M.+Flanders&amp;author=H.+Kim&amp;author=G.M.+Rodgers&amp;volume=126&amp;publication_year=2006&amp;pages=411-415&amp;pmid=16880142&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B2">2. <span class="mixed-citation">
Tucker E.I., Marzec U.M., White T.C., Hurst S., Rugonyi S., McCarty O.J.T., Gailani D., Gruber A., Hanson S.R. (2009) 
<span class="ref-title">Prevention of vascular graft occlusion and thrombus-associated thrombin generation by inhibition of factor XI</span>. <span class="ref-journal">Blood</span>, <span class="ref-vol">113</span>, 936–944. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2630279/">PMC free article</a>]</span> [<a href="/pubmed/18945968" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Prevention+of+vascular+graft+occlusion+and+thrombus-associated+thrombin+generation+by+inhibition+of+factor+XI&amp;author=E.I.+Tucker&amp;author=U.M.+Marzec&amp;author=T.C.+White&amp;author=S.+Hurst&amp;author=S.+Rugonyi&amp;volume=113&amp;publication_year=2009&amp;pages=936-944&amp;pmid=18945968&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B3">3. <span class="mixed-citation">
Meijers J.C., Tekelenburg W.L., Bouma B.N., Bertina R.M., Rosendaal F.R. (2000) 
<span class="ref-title">High levels of coagulation factor XI as a risk factor for venous thrombosis</span>. <span class="ref-journal">N. Engl. J. Med</span>., <span class="ref-vol">342</span>, 696–701. [<a href="/pubmed/10706899" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=High+levels+of+coagulation+factor+XI+as+a+risk+factor+for+venous+thrombosis&amp;author=J.C.+Meijers&amp;author=W.L.+Tekelenburg&amp;author=B.N.+Bouma&amp;author=R.M.+Bertina&amp;author=F.R.+Rosendaal&amp;volume=342&amp;publication_year=2000&amp;pages=696-701&amp;pmid=10706899&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B4">4. <span class="mixed-citation">
Gailani D., Smith S.B. (2009) 
<span class="ref-title">Structural and functional features of factor XI</span>. <span class="ref-journal">J. Thromb. Haemost</span>., <span class="ref-vol">7 Suppl 1</span>, 75–78. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2849299/">PMC free article</a>]</span> [<a href="/pubmed/19630773" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Thromb.+Haemost&amp;title=Structural+and+functional+features+of+factor+XI&amp;author=D.+Gailani&amp;author=S.B.+Smith&amp;volume=7+Suppl+1&amp;publication_year=2009&amp;pages=75-78&amp;pmid=19630773&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B5">5. <span class="mixed-citation">
Doggen C.J.M., Rosendaal F.R., Meijers J.C.M. (2006) 
<span class="ref-title">Levels of intrinsic coagulation factors and the risk of myocardial infarction among men: Opposite and synergistic effects of factors XI and XII</span>. <span class="ref-journal">Blood</span>, <span class="ref-vol">108</span>, 4045–4051. [<a href="/pubmed/16931632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Levels+of+intrinsic+coagulation+factors+and+the+risk+of+myocardial+infarction+among+men:+Opposite+and+synergistic+effects+of+factors+XI+and+XII&amp;author=C.J.M.+Doggen&amp;author=F.R.+Rosendaal&amp;author=J.C.M.+Meijers&amp;volume=108&amp;publication_year=2006&amp;pages=4045-4051&amp;pmid=16931632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B6">6. <span class="mixed-citation">
Siegerink B., Rosendaal F.R., Algra A. (2014) 
<span class="ref-title">Antigen levels of coagulation factor XII, coagulation factor XI and prekallikrein, and the risk of myocardial infarction and ischemic stroke in young women</span>. <span class="ref-journal">J. Thromb. Haemost</span>., <span class="ref-vol">12</span>, 606–613. [<a href="/pubmed/24977287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Thromb.+Haemost&amp;title=Antigen+levels+of+coagulation+factor+XII,+coagulation+factor+XI+and+prekallikrein,+and+the+risk+of+myocardial+infarction+and+ischemic+stroke+in+young+women&amp;author=B.+Siegerink&amp;author=F.R.+Rosendaal&amp;author=A.+Algra&amp;volume=12&amp;publication_year=2014&amp;pages=606-613&amp;pmid=24977287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B7">7. <span class="mixed-citation">
Salomon O., Steinberg D.M., Koren-Morag N., Tanne D., Seligsohn U. (2008) 
<span class="ref-title">Reduced incidence of ischemic stroke in patients with severe factor XI deficiency</span>. <span class="ref-journal">Blood</span>, <span class="ref-vol">111</span>, 4113–4117. [<a href="/pubmed/18268095" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Reduced+incidence+of+ischemic+stroke+in+patients+with+severe+factor+XI+deficiency&amp;author=O.+Salomon&amp;author=D.M.+Steinberg&amp;author=N.+Koren-Morag&amp;author=D.+Tanne&amp;author=U.+Seligsohn&amp;volume=111&amp;publication_year=2008&amp;pages=4113-4117&amp;pmid=18268095&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B8">8. <span class="mixed-citation">
Folsom A.R., Tang W., Roetker N.S., Heckbert S.R., Cushman M., Pankow J.S. (2015) 
<span class="ref-title">Prospective study of circulating factor XI and incident venous thromboembolism: The Longitudinal Investigation of Thromboembolism Etiology (LITE)</span>. <span class="ref-journal">Am. J. Hematol</span>., 10.1002/ajh.24168. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4618026/">PMC free article</a>]</span> [<a href="/pubmed/26260105" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hematol&amp;title=Prospective+study+of+circulating+factor+XI+and+incident+venous+thromboembolism:+The+Longitudinal+Investigation+of+Thromboembolism+Etiology+(LITE)&amp;author=A.R.+Folsom&amp;author=W.+Tang&amp;author=N.S.+Roetker&amp;author=S.R.+Heckbert&amp;author=M.+Cushman&amp;publication_year=2015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B9">9. <span class="mixed-citation">
Folsom A.R., George K.M., Appiah D. (2015) 
<span class="ref-title">Lack of association of plasma factor XI with incident stroke and coronary heart disease: The Atherosclerosis Risk in Communities (ARIC) Study</span>. <span class="ref-journal">Atherosclerosis</span>, <span class="ref-vol">243</span>, 181–185. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4620543/">PMC free article</a>]</span> [<a href="/pubmed/26386215" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Lack+of+association+of+plasma+factor+XI+with+incident+stroke+and+coronary+heart+disease:+The+Atherosclerosis+Risk+in+Communities+(ARIC)+Study&amp;author=A.R.+Folsom&amp;author=K.M.+George&amp;author=D.+Appiah&amp;volume=243&amp;publication_year=2015&amp;pages=181-185&amp;pmid=26386215&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B10">10. <span class="mixed-citation">
Holst A.G., Jensen G., Prescott E. (2010) 
<span class="ref-title">Risk factors for venous thromboembolism</span>. <span class="ref-journal">Circulation</span>, 4;121:1896–1903. [<a href="/pubmed/20404252" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Risk+factors+for+venous+thromboembolism&amp;author=A.G.+Holst&amp;author=G.+Jensen&amp;author=E.+Prescott&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B11">11. <span class="mixed-citation">
Naess I.A., Christiansen S.C., Romundstad P., Cannegieter S.C., Rosendaal F.R., Hammerstrøm J. (2007) 
<span class="ref-title">Incidence and mortality of venous thrombosis: a population-based study</span>. <span class="ref-journal">J. Thromb. Haemost</span>., <span class="ref-vol">5</span>, 692–699. [<a href="/pubmed/17367492" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Thromb.+Haemost&amp;title=Incidence+and+mortality+of+venous+thrombosis:+a+population-based+study&amp;author=I.A.+Naess&amp;author=S.C.+Christiansen&amp;author=P.+Romundstad&amp;author=S.C.+Cannegieter&amp;author=F.R.+Rosendaal&amp;volume=5&amp;publication_year=2007&amp;pages=692-699&amp;pmid=17367492&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B12">12. <span class="mixed-citation">
Salomon O., Steinberg D.M., Zucker M., Varon D., Zivelin A., Seligsohn U. (2011) 
<span class="ref-title">Patients with severe factor XI deficiency have a reduced incidence of deep-vein thrombosis</span>. <span class="ref-journal">Thromb. Haemost</span>., <span class="ref-vol">105</span>, 269–273. [<a href="/pubmed/21057700" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb.+Haemost&amp;title=Patients+with+severe+factor+XI+deficiency+have+a+reduced+incidence+of+deep-vein+thrombosis&amp;author=O.+Salomon&amp;author=D.M.+Steinberg&amp;author=M.+Zucker&amp;author=D.+Varon&amp;author=A.+Zivelin&amp;volume=105&amp;publication_year=2011&amp;pages=269-273&amp;pmid=21057700&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B13">13. <span class="mixed-citation">
Bezemer I.D., Bare L.A., Doggen C.J.M., Arellano A.R., Tong C., Rowland C.M., Catanese J., Young B.A., Reitsma P.H., Devlin J.J., et al. (2008) 
<span class="ref-title">Gene variants associated with deep vein thrombosis</span>. <span class="ref-journal">JAMA-J. Am. Med. Assoc</span>., <span class="ref-vol">299</span>, 1306–1314. [<a href="/pubmed/18349091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA-J.+Am.+Med.+Assoc&amp;title=Gene+variants+associated+with+deep+vein+thrombosis&amp;author=I.D.+Bezemer&amp;author=L.A.+Bare&amp;author=C.J.M.+Doggen&amp;author=A.R.+Arellano&amp;author=C.+Tong&amp;volume=299&amp;publication_year=2008&amp;pages=1306-1314&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B14">14. <span class="mixed-citation">
Li Y., Bezemer I.D., Rowland C.M., Tong C.H., Arellano A.R., Catanese J.J., Devlin J.J., Reitsma P.H., Bare L.A., Rosendaal F.R. (2009) 
<span class="ref-title">Genetic variants associated with deep vein thrombosis: the F11 locus</span>. <span class="ref-journal">J. Thromb. Haemost</span>., <span class="ref-vol">7</span>, 1802–1808. [<a href="/pubmed/19583818" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Thromb.+Haemost&amp;title=Genetic+variants+associated+with+deep+vein+thrombosis:+the+F11+locus&amp;author=Y.+Li&amp;author=I.D.+Bezemer&amp;author=C.M.+Rowland&amp;author=C.H.+Tong&amp;author=A.R.+Arellano&amp;volume=7&amp;publication_year=2009&amp;pages=1802-1808&amp;pmid=19583818&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B15">15. <span class="mixed-citation">
Austin H., De Staercke C., Lally C., Bezemer I.D., Rosendaal F.R., Hooper W.C. (2011) 
<span class="ref-title">New gene variants associated with venous thrombosis: a replication study in White and Black Americans</span>. <span class="ref-journal">J. Thromb. Haemost</span>., <span class="ref-vol">9</span>, 489–495. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4532311/">PMC free article</a>]</span> [<a href="/pubmed/21232005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Thromb.+Haemost&amp;title=New+gene+variants+associated+with+venous+thrombosis:+a+replication+study+in+White+and+Black+Americans&amp;author=H.+Austin&amp;author=C.+De+Staercke&amp;author=C.+Lally&amp;author=I.D.+Bezemer&amp;author=F.R.+Rosendaal&amp;volume=9&amp;publication_year=2011&amp;pages=489-495&amp;pmid=21232005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B16">16. <span class="mixed-citation">
Rovite V., Maurins U., Megnis K., Vaivade I., Pečulis R., Rits J., Prave S., Klovins J. (2014) 
<span class="ref-title">Association of F11 polymorphism rs2289252 with deep vein thrombosis and related phenotypes in population of Latvia</span>. <span class="ref-journal">Thromb. Res</span>., <span class="ref-vol">134</span>, 659–663. [<a href="/pubmed/25091233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb.+Res&amp;title=Association+of+F11+polymorphism+rs2289252+with+deep+vein+thrombosis+and+related+phenotypes+in+population+of+Latvia&amp;author=V.+Rovite&amp;author=U.+Maurins&amp;author=K.+Megnis&amp;author=I.+Vaivade&amp;author=R.+Pečulis&amp;volume=134&amp;publication_year=2014&amp;pages=659-663&amp;pmid=25091233&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B17">17. <span class="mixed-citation">
Germain M., Chasman D.I., de Haan H., Tang W., Lindström S., Weng L.C., de Andrade M., de Visser M.C.H., Wiggins K.L., Suchon P., et al. (2015) 
<span class="ref-title">Meta-analysis of 65,734 individuals identifies TSPAN15 and SLC44A2 as two susceptibility loci for venous thromboembolism. <em>Am</em></span>. <span class="ref-journal">J. Hum. Genet</span>., <span class="ref-vol">96</span>, 532–542. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4385184/">PMC free article</a>]</span> [<a href="/pubmed/25772935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hum.+Genet&amp;title=Meta-analysis+of+65,734+individuals+identifies+TSPAN15+and+SLC44A2+as+two+susceptibility+loci+for+venous+thromboembolism.+Am&amp;author=M.+Germain&amp;author=D.I.+Chasman&amp;author=H.+de+Haan&amp;author=W.+Tang&amp;author=S.+Lindström&amp;volume=96&amp;publication_year=2015&amp;pages=532-542&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B18">18. <span class="mixed-citation">
Hanson E., Nilsson S., Jood K., Norrving B., Engström G., Blomstrand C., Lindgren A., Melander O., Jern C. (2013) 
<span class="ref-title">Genetic variants of coagulation factor XI show association with ischemic stroke up to 70 years of age</span>. <span class="ref-journal">PLoS One</span>, <span class="ref-vol">8</span>, e75286.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3783404/">PMC free article</a>]</span> [<a href="/pubmed/24086496" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Genetic+variants+of+coagulation+factor+XI+show+association+with+ischemic+stroke+up+to+70+years+of+age&amp;author=E.+Hanson&amp;author=S.+Nilsson&amp;author=K.+Jood&amp;author=B.+Norrving&amp;author=G.+Engström&amp;volume=8&amp;publication_year=2013&amp;pages=e75286.&amp;pmid=24086496&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B19">19. <span class="mixed-citation">
Zakai N.A., Ohira T., White R., Folsom A.R., Cushman M. (2008) 
<span class="ref-title">Activated partial thromboplastin time and risk of future venous thromboembolism</span>. <span class="ref-journal">Am. J. Med</span>., <span class="ref-vol">121</span>, 231–238. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2295205/">PMC free article</a>]</span> [<a href="/pubmed/18328308" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med&amp;title=Activated+partial+thromboplastin+time+and+risk+of+future+venous+thromboembolism&amp;author=N.A.+Zakai&amp;author=T.+Ohira&amp;author=R.+White&amp;author=A.R.+Folsom&amp;author=M.+Cushman&amp;volume=121&amp;publication_year=2008&amp;pages=231-238&amp;pmid=18328308&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B20">20. <span class="mixed-citation">
Tang W., Schwienbacher C., Lopez L.M., Ben-Shlomo Y., Oudot-Mellakh T., Johnson A.D., Samani N.J., Basu S., Gögele M., Davies G., et al. (2012) 
<span class="ref-title">Genetic Associations for Activated Partial Thromboplastin Time and Prothrombin Time, their Gene Expression Profiles, and Risk of Coronary Artery Disease. <em>Am</em></span>. <span class="ref-journal">J. Hum. Genet</span>., <span class="ref-vol">91</span>, 152–162. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3397273/">PMC free article</a>]</span> [<a href="/pubmed/22703881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hum.+Genet&amp;title=Genetic+Associations+for+Activated+Partial+Thromboplastin+Time+and+Prothrombin+Time,+their+Gene+Expression+Profiles,+and+Risk+of+Coronary+Artery+Disease.+Am&amp;author=W.+Tang&amp;author=C.+Schwienbacher&amp;author=L.M.+Lopez&amp;author=Y.+Ben-Shlomo&amp;author=T.+Oudot-Mellakh&amp;volume=91&amp;publication_year=2012&amp;pages=152-162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B21">21. <span class="mixed-citation">
Gaunt T.R., Lowe G.D.O., Lawlor D.A., Casas J.P., Day I.N.M. (2013) 
<span class="ref-title">A gene-centric analysis of activated partial thromboplastin time and activated protein C resistance using the HumanCVD focused genotyping array</span>. <span class="ref-journal">Eur. J. Hum. Genet</span>., <span class="ref-vol">21</span>, 779–783. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3682876/">PMC free article</a>]</span> [<a href="/pubmed/23188048" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet&amp;title=A+gene-centric+analysis+of+activated+partial+thromboplastin+time+and+activated+protein+C+resistance+using+the+HumanCVD+focused+genotyping+array&amp;author=T.R.+Gaunt&amp;author=G.D.O.+Lowe&amp;author=D.A.+Lawlor&amp;author=J.P.+Casas&amp;author=I.N.M.+Day&amp;volume=21&amp;publication_year=2013&amp;pages=779-783&amp;pmid=23188048&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B22">22. <span class="mixed-citation">
Rosen E.D., Gailani D., Castellino F.J. (2002) 
<span class="ref-title">FXI is essential for thrombus formation following FeCl3-induced injury of the carotid artery in the mouse</span>. <span class="ref-journal">Thromb. Haemost</span>., <span class="ref-vol">87</span>, 774–776. [<a href="/pubmed/12008966" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb.+Haemost&amp;title=FXI+is+essential+for+thrombus+formation+following+FeCl3-induced+injury+of+the+carotid+artery+in+the+mouse&amp;author=E.D.+Rosen&amp;author=D.+Gailani&amp;author=F.J.+Castellino&amp;volume=87&amp;publication_year=2002&amp;pages=774-776&amp;pmid=12008966&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B23">23. <span class="mixed-citation">
Büller H.R., Bethune C., Bhanot S., Gailani D., Monia B.P., Raskob G.E., Segers A., Verhamme P., Weitz J.I.
FXI-ASO TKA Investigators (2015) 
<span class="ref-title">Factor XI antisense oligonucleotide for prevention of venous thrombosis</span>. <span class="ref-journal">N. Engl. J. Med</span>., <span class="ref-vol">372</span>, 232–240. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4367537/">PMC free article</a>]</span> [<a href="/pubmed/25482425" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Factor+XI+antisense+oligonucleotide+for+prevention+of+venous+thrombosis&amp;author=H.R.+Büller&amp;author=C.+Bethune&amp;author=S.+Bhanot&amp;author=D.+Gailani&amp;author=B.P.+Monia&amp;volume=372&amp;publication_year=2015&amp;pages=232-240&amp;pmid=25482425&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B24">24. <span class="mixed-citation">
Sabater-Lleal M., Martinez-Perez A., Buil A., Folkersen L., Souto J.C., Bruzelius M., Borrell M., Odeberg J., Silveira A., Eriksson P., et al. (2012) 
<span class="ref-title">A genome-wide association study identifies KNG1 as a genetic determinant of plasma factor XI Level and activated partial thromboplastin time</span>. <span class="ref-journal">Arterioscl. Throm. Vas</span>., <span class="ref-vol">32</span>, 2008–2016. [<a href="/pubmed/22701019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscl.+Throm.+Vas&amp;title=A+genome-wide+association+study+identifies+KNG1+as+a+genetic+determinant+of+plasma+factor+XI+Level+and+activated+partial+thromboplastin+time&amp;author=M.+Sabater-Lleal&amp;author=A.+Martinez-Perez&amp;author=A.+Buil&amp;author=L.+Folkersen&amp;author=J.C.+Souto&amp;volume=32&amp;publication_year=2012&amp;pages=2008-2016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B25">25. <span class="mixed-citation">
Nikpay M., Goel A., Won H.H., Hall L.M., Willenborg C., Kanoni S., Saleheen D., Kyriakou T., Nelson C.P., Hopewell J.C., et al. (2015) 
<span class="ref-title">A comprehensive 1,000 Genomes-based genome-wide association meta-analysis of coronary artery disease</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">47</span>, 1121–1130. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4589895/">PMC free article</a>]</span> [<a href="/pubmed/26343387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+comprehensive+1,000+Genomes-based+genome-wide+association+meta-analysis+of+coronary+artery+disease&amp;author=M.+Nikpay&amp;author=A.+Goel&amp;author=H.H.+Won&amp;author=L.M.+Hall&amp;author=C.+Willenborg&amp;volume=47&amp;publication_year=2015&amp;pages=1121-1130&amp;pmid=26343387&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B26">26. <span class="mixed-citation">
Yang J., Benyamin B., McEvoy B.P., Gordon S., Henders A.K., Nyholt D.R., Madden P.A., Heath A.C., Martin N.G., Montgomery G.W., et al. (2010) 
<span class="ref-title">Common SNPs explain a large proportion of the heritability for human height</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">42</span>, 565–569. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J.+Yang&amp;author=B.+Benyamin&amp;author=B.P.+McEvoy&amp;author=S.+Gordon&amp;author=A.K.+Henders&amp;volume=42&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B27">27. <span class="mixed-citation">
Yang J., Lee S.H., Goddard M.E., Visscher P.M. (2011) 
<span class="ref-title">GCTA: a tool for genome-wide complex trait analysis</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">88</span>, 76–82. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J.+Yang&amp;author=S.H.+Lee&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B28">28. <span class="mixed-citation">
Yang J., Ferreira T., Morris A.P., Medland S.E., Madden P.A.F., Heath A.C., Martin N.G., Montgomery G.W., Weedon M.N., Loos R.J., et al. (2012) 
<span class="ref-title">Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">44</span>, 369–375. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3593158/">PMC free article</a>]</span> [<a href="/pubmed/22426310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Conditional+and+joint+multiple-SNP+analysis+of+GWAS+summary+statistics+identifies+additional+variants+influencing+complex+traits&amp;author=J.+Yang&amp;author=T.+Ferreira&amp;author=A.P.+Morris&amp;author=S.E.+Medland&amp;author=P.A.F.+Madden&amp;volume=44&amp;publication_year=2012&amp;pages=369-375&amp;pmid=22426310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B29">29. <span class="mixed-citation">
Dupuis J., Langenberg C., Prokopenko I., Saxena R., Soranzo N., Jackson A.U., Wheeler E., Glazer N.L., Bouatia-Naji N., Gloyn A.L., et al. (2010) 
<span class="ref-title">New genetic loci implicated in fasting glucose homeostasis and their impact on type 2 diabetes risk</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">42</span>, 105–116. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3018764/">PMC free article</a>]</span> [<a href="/pubmed/20081858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=New+genetic+loci+implicated+in+fasting+glucose+homeostasis+and+their+impact+on+type+2+diabetes+risk&amp;author=J.+Dupuis&amp;author=C.+Langenberg&amp;author=I.+Prokopenko&amp;author=R.+Saxena&amp;author=N.+Soranzo&amp;volume=42&amp;publication_year=2010&amp;pages=105-116&amp;pmid=20081858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B30">30. <span class="mixed-citation">
Ng M.C.Y., Saxena R., Li J., Palmer N.D., Dimitrov L., Xu J., Rasmussen-Torvik L.J., Zmuda J.M., Siscovick D.S., Patel S.R., et al. (2013) 
<span class="ref-title">Transferability and fine mapping of type 2 diabetes loci in African Americans: the Candidate Gene Association Resource Plus Study</span>. <span class="ref-journal">Diabetes</span>, <span class="ref-vol">62</span>, 965–976. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3581206/">PMC free article</a>]</span> [<a href="/pubmed/23193183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Transferability+and+fine+mapping+of+type+2+diabetes+loci+in+African+Americans:+the+Candidate+Gene+Association+Resource+Plus+Study&amp;author=M.C.Y.+Ng&amp;author=R.+Saxena&amp;author=J.+Li&amp;author=N.D.+Palmer&amp;author=L.+Dimitrov&amp;volume=62&amp;publication_year=2013&amp;pages=965-976&amp;pmid=23193183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B31">31. <span class="mixed-citation">
Brouwers M.C.G.J., Jacobs C., Bast A., Stehouwer C.D.A., Schaper N.C. (2015) 
<span class="ref-title">Modulation of Glucokinase Regulatory Protein: A Double-Edged Sword?</span>. <span class="ref-journal">Trends Mol. Med</span>., <span class="ref-vol">21</span>, 583–594. [<a href="/pubmed/26432016" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Mol.+Med&amp;title=Modulation+of+Glucokinase+Regulatory+Protein:+A+Double-Edged+Sword?&amp;author=M.C.G.J.+Brouwers&amp;author=C.+Jacobs&amp;author=A.+Bast&amp;author=C.D.A.+Stehouwer&amp;author=N.C.+Schaper&amp;volume=21&amp;publication_year=2015&amp;pages=583-594&amp;pmid=26432016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B32">32. <span class="mixed-citation">
Hu Z., Liu J., Song Z., Hou Q., Fan X., Hou D. (2015) 
<span class="ref-title">Variants in the Atherogenic ALOX5AP, THBD, and KNG1 Genes Potentiate the Risk of Ischemic Stroke via a Genetic Main Effect and Epistatic Interactions in a Chinese Population</span>. <span class="ref-journal">J. Stroke Cerebrovasc. Dis</span>., <span class="ref-vol">24</span>, 2060–2068. [<a href="/pubmed/26159646" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Stroke+Cerebrovasc.+Dis&amp;title=Variants+in+the+Atherogenic+ALOX5AP,+THBD,+and+KNG1+Genes+Potentiate+the+Risk+of+Ischemic+Stroke+via+a+Genetic+Main+Effect+and+Epistatic+Interactions+in+a+Chinese+Population&amp;author=Z.+Hu&amp;author=J.+Liu&amp;author=Z.+Song&amp;author=Q.+Hou&amp;author=X.+Fan&amp;volume=24&amp;publication_year=2015&amp;pages=2060-2068&amp;pmid=26159646&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B33">33. <span class="mixed-citation">
Fabian M.R., Sonenberg N. (2012) 
<span class="ref-title">The mechanics of miRNA-mediated gene silencing: a look under the hood of miRISC</span>. <span class="ref-journal">Nat. Struct. Mol. Biol</span>., <span class="ref-vol">19</span>, 586–593. [<a href="/pubmed/22664986" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol&amp;title=The+mechanics+of+miRNA-mediated+gene+silencing:+a+look+under+the+hood+of+miRISC&amp;author=M.R.+Fabian&amp;author=N.+Sonenberg&amp;volume=19&amp;publication_year=2012&amp;pages=586-593&amp;pmid=22664986&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B34">34. <span class="mixed-citation">
Ward L.D., Kellis M. (2011) 
<span class="ref-title">HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</span>. <span class="ref-journal">Nucleic Acids Res</span>., <span class="ref-vol">40</span>, D930–D934. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245002/">PMC free article</a>]</span> [<a href="/pubmed/22064851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=HaploReg:+a+resource+for+exploring+chromatin+states,+conservation,+and+regulatory+motif+alterations+within+sets+of+genetically+linked+variants&amp;author=L.D.+Ward&amp;author=M.+Kellis&amp;volume=40&amp;publication_year=2011&amp;pages=D930-D934&amp;pmid=22064851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B35">35. <span class="mixed-citation">
Safdar H., Cleuren A.C.A., Cheung K.L., Gonzalez F.J., Vos H.L., Inoue Y., Reitsma P.H., van Vlijmen B.J.M. (2013) 
<span class="ref-title">Regulation of the F11, Klkb1, Cyp4v3 gene cluster in livers of metabolically challenged mice</span>. <span class="ref-journal">PLoS One</span>, <span class="ref-vol">8</span>, e74637.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3774739/">PMC free article</a>]</span> [<a href="/pubmed/24066149" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Regulation+of+the+F11,+Klkb1,+Cyp4v3+gene+cluster+in+livers+of+metabolically+challenged+mice&amp;author=H.+Safdar&amp;author=A.C.A.+Cleuren&amp;author=K.L.+Cheung&amp;author=F.J.+Gonzalez&amp;author=H.L.+Vos&amp;volume=8&amp;publication_year=2013&amp;pages=e74637.&amp;pmid=24066149&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B36">36. <span class="mixed-citation">
Salloum-Asfar S., Teruel-Montoya R., Arroyo A.B., García-Barberá N., Chaudhry A., Schuetz E., Luengo-Gil G., Vicente V., González-Conejero R., Martínez C. (2014) 
<span class="ref-title">Regulation of Coagulation Factor XI Expression by MicroRNAs in the Human Liver</span>. <span class="ref-journal">PLoS One</span>, <span class="ref-vol">9</span>, e111713.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4224396/">PMC free article</a>]</span> [<a href="/pubmed/25379760" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Regulation+of+Coagulation+Factor+XI+Expression+by+MicroRNAs+in+the+Human+Liver&amp;author=S.+Salloum-Asfar&amp;author=R.+Teruel-Montoya&amp;author=A.B.+Arroyo&amp;author=N.+García-Barberá&amp;author=A.+Chaudhry&amp;volume=9&amp;publication_year=2014&amp;pages=e111713.&amp;pmid=25379760&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B37">37. <span class="mixed-citation">
Remy I., Michnick S.W. (2006) 
<span class="ref-title">A highly sensitive protein-protein interaction assay based on Gaussia luciferase</span>. <span class="ref-journal">Nat. Methods</span>, <span class="ref-vol">3</span>, 977–979. [<a href="/pubmed/17099704" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=A+highly+sensitive+protein-protein+interaction+assay+based+on+Gaussia+luciferase&amp;author=I.+Remy&amp;author=S.W.+Michnick&amp;volume=3&amp;publication_year=2006&amp;pages=977-979&amp;pmid=17099704&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B38">38. <span class="mixed-citation">
Smith N.L., Chen M.H., Dehghan A., Strachan D.P., Basu S., Soranzo N., Hayward C., Rudan I., Sabater-Lleal M., Bis J.C., et al. (2010) 
<span class="ref-title">Novel associations of multiple genetic loci with plasma levels of factor VII, factor VIII, and von Willebrand factor: The CHARGE (Cohorts for Heart and Aging Research in Genome Epidemiology) Consortium</span>. <span class="ref-journal">Circulation</span>, <span class="ref-vol">121</span>, 1382–1392. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2861278/">PMC free article</a>]</span> [<a href="/pubmed/20231535" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Novel+associations+of+multiple+genetic+loci+with+plasma+levels+of+factor+VII,+factor+VIII,+and+von+Willebrand+factor:+The+CHARGE+(Cohorts+for+Heart+and+Aging+Research+in+Genome+Epidemiology)+Consortium&amp;author=N.L.+Smith&amp;author=M.H.+Chen&amp;author=A.+Dehghan&amp;author=D.P.+Strachan&amp;author=S.+Basu&amp;volume=121&amp;publication_year=2010&amp;pages=1382-1392&amp;pmid=20231535&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B39">39. <span class="mixed-citation">
Tang W., Basu S., Kong X., Pankow J.S., Aleksic N., Tan A., Cushman M., Boerwinkle E., Folsom A.R. (2010) 
<span class="ref-title">Genome-wide association study identifies novel loci for plasma levels of protein C: the ARIC study</span>. <span class="ref-journal">Blood</span>, <span class="ref-vol">116</span>, 5032–5036. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3012596/">PMC free article</a>]</span> [<a href="/pubmed/20802025" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Genome-wide+association+study+identifies+novel+loci+for+plasma+levels+of+protein+C:+the+ARIC+study&amp;author=W.+Tang&amp;author=S.+Basu&amp;author=X.+Kong&amp;author=J.S.+Pankow&amp;author=N.+Aleksic&amp;volume=116&amp;publication_year=2010&amp;pages=5032-5036&amp;pmid=20802025&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B40">40. <span class="mixed-citation">
Taylor K.C., Lange L.A., Zabaneh D., Lange E., Keating B.J., Tang W., Smith N.L., Delaney J.A., Kumari M., Hingorani A., et al. (2011) 
<span class="ref-title">A gene-centric association scan for Coagulation Factor VII levels in European and African Americans: the Candidate Gene Association Resource (CARe) Consortium</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">20</span>, 3525–3534. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3153310/">PMC free article</a>]</span> [<a href="/pubmed/21676895" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=A+gene-centric+association+scan+for+Coagulation+Factor+VII+levels+in+European+and+African+Americans:+the+Candidate+Gene+Association+Resource+(CARe)+Consortium&amp;author=K.C.+Taylor&amp;author=L.A.+Lange&amp;author=D.+Zabaneh&amp;author=E.+Lange&amp;author=B.J.+Keating&amp;volume=20&amp;publication_year=2011&amp;pages=3525-3534&amp;pmid=21676895&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B41">41. <span class="mixed-citation">
Bruzelius M., Bottai M., Sabater-Lleal M., Strawbridge R.J., Bergendal A., Silveira A., Sundström A., Kieler H., Hamsten A., Odeberg J. (2015) 
<span class="ref-title">Predicting venous thrombosis in women using a combination of genetic markers and clinical risk factors</span>. <span class="ref-journal">J. Thromb. Haemost</span>., <span class="ref-vol">13</span>, 219–227. [<a href="/pubmed/25472531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Thromb.+Haemost&amp;title=Predicting+venous+thrombosis+in+women+using+a+combination+of+genetic+markers+and+clinical+risk+factors&amp;author=M.+Bruzelius&amp;author=M.+Bottai&amp;author=M.+Sabater-Lleal&amp;author=R.J.+Strawbridge&amp;author=A.+Bergendal&amp;volume=13&amp;publication_year=2015&amp;pages=219-227&amp;pmid=25472531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B42">42. <span class="mixed-citation">
Germain M., Saut N., Greliche N., Dina C., Lambert J.C., Perret C., Cohen W., Oudot-Mellakh T., Antoni G., Alessi M.C., et al. (2011) 
<span class="ref-title">Genetics of venous thrombosis: insights from a new genome wide association study</span>. <span class="ref-journal">PLoS One</span>, <span class="ref-vol">6</span>, e25581.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3181335/">PMC free article</a>]</span> [<a href="/pubmed/21980494" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Genetics+of+venous+thrombosis:+insights+from+a+new+genome+wide+association+study&amp;author=M.+Germain&amp;author=N.+Saut&amp;author=N.+Greliche&amp;author=C.+Dina&amp;author=J.C.+Lambert&amp;volume=6&amp;publication_year=2011&amp;pages=e25581.&amp;pmid=21980494&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B43">43. <span class="mixed-citation">
Tang W., Cushman M., Green D., Rich S.S., Lange L.A., Yang Q., Tracy R.P., Tofler G.H., Basu S., Wilson J.G., et al. (2015) 
<span class="ref-title">Gene-centric approach identifies new and known loci for FVIII activity and VWF antigen levels in European Americans and African Americans</span>. <span class="ref-journal">Am. J. Hematol</span>., <span class="ref-vol">90</span>, 534–540. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4747096/">PMC free article</a>]</span> [<a href="/pubmed/25779970" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hematol&amp;title=Gene-centric+approach+identifies+new+and+known+loci+for+FVIII+activity+and+VWF+antigen+levels+in+European+Americans+and+African+Americans&amp;author=W.+Tang&amp;author=M.+Cushman&amp;author=D.+Green&amp;author=S.S.+Rich&amp;author=L.A.+Lange&amp;volume=90&amp;publication_year=2015&amp;pages=534-540&amp;pmid=25779970&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B44">44. <span class="mixed-citation">
Ellis J., Lange E.M., Li J., Dupuis J., Baumert J., Walston J.D., Keating B.J., Durda P., Fox E.R., Palmer C.D., et al. (2014) 
<span class="ref-title">Large multiethnic Candidate Gene Study for C-reactive protein levels: identification of a novel association at CD36 in African Americans</span>. <span class="ref-journal">Hum. Genet</span>., <span class="ref-vol">133</span>, 985–995. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4104766/">PMC free article</a>]</span> [<a href="/pubmed/24643644" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Large+multiethnic+Candidate+Gene+Study+for+C-reactive+protein+levels:+identification+of+a+novel+association+at+CD36+in+African+Americans&amp;author=J.+Ellis&amp;author=E.M.+Lange&amp;author=J.+Li&amp;author=J.+Dupuis&amp;author=J.+Baumert&amp;volume=133&amp;publication_year=2014&amp;pages=985-995&amp;pmid=24643644&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B45">45. <span class="mixed-citation">
Ridker P.M., Paré G., Parker A., Zee R.Y.L., Danik J.S., Buring J.E., Kwiatkowski D., Cook N.R., Miletich J.P., Chasman D.I. (2008) 
<span class="ref-title">Loci related to metabolic-syndrome pathways including LEPR, HNF1A, IL6R, and GCKR associate with plasma C-reactive protein: the Women's Genome Health Study</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">82</span>, 1185–1192. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427311/">PMC free article</a>]</span> [<a href="/pubmed/18439548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Loci+related+to+metabolic-syndrome+pathways+including+LEPR,+HNF1A,+IL6R,+and+GCKR+associate+with+plasma+C-reactive+protein:+the+Women's+Genome+Health+Study&amp;author=P.M.+Ridker&amp;author=G.+Paré&amp;author=A.+Parker&amp;author=R.Y.L.+Zee&amp;author=J.S.+Danik&amp;volume=82&amp;publication_year=2008&amp;pages=1185-1192&amp;pmid=18439548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B46">46. <span class="mixed-citation">
Folsom A.R., Lutsey P.L., Astor B.C., Cushman M. (2009) 
<span class="ref-title">C-reactive protein and venous thromboembolism. A prospective investigation in the ARIC cohort</span>. <span class="ref-journal">Thromb. Haemost</span>., <span class="ref-vol">102</span>, 615–619. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2810122/">PMC free article</a>]</span> [<a href="/pubmed/19806245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb.+Haemost&amp;title=C-reactive+protein+and+venous+thromboembolism.+A+prospective+investigation+in+the+ARIC+cohort&amp;author=A.R.+Folsom&amp;author=P.L.+Lutsey&amp;author=B.C.+Astor&amp;author=M.+Cushman&amp;volume=102&amp;publication_year=2009&amp;pages=615-619&amp;pmid=19806245&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B47">47. <span class="mixed-citation">
Faid V., Denguir N., Chapuis V., Bihoreau N., Chevreux G. (2014) 
<span class="ref-title">Site-specific N-glycosylation analysis of human factor XI: Identification of a noncanonical NXC glycosite</span>. <span class="ref-journal">Proteomics</span>, <span class="ref-vol">14</span>, 2460–2470. [<a href="/pubmed/25092234" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Site-specific+N-glycosylation+analysis+of+human+factor+XI:+Identification+of+a+noncanonical+NXC+glycosite&amp;author=V.+Faid&amp;author=N.+Denguir&amp;author=V.+Chapuis&amp;author=N.+Bihoreau&amp;author=G.+Chevreux&amp;volume=14&amp;publication_year=2014&amp;pages=2460-2470&amp;pmid=25092234&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B48">48. <span class="mixed-citation">
Smit R.A.J., Trompet S., De Craen A.J.M., Jukema J.W. (2014) 
<span class="ref-title">Using genetic variation for establishing causality of cardiovascular risk factors: overcoming confounding and reverse causality</span>. <span class="ref-journal">Neth. Heart J</span>., <span class="ref-vol">22</span>, 186–189. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3954928/">PMC free article</a>]</span> [<a href="/pubmed/24590770" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neth.+Heart+J&amp;title=Using+genetic+variation+for+establishing+causality+of+cardiovascular+risk+factors:+overcoming+confounding+and+reverse+causality&amp;author=R.A.J.+Smit&amp;author=S.+Trompet&amp;author=A.J.M.+De+Craen&amp;author=J.W.+Jukema&amp;volume=22&amp;publication_year=2014&amp;pages=186-189&amp;pmid=24590770&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B49">49. <span class="mixed-citation">
Li Y., Willer C., Sanna S., Abecasis G. (2009) 
<span class="ref-title">Genotype imputation</span>. <span class="ref-journal">Annu. Rev. Genomics Hum. Genet</span>., <span class="ref-vol">10</span>, 387–406. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Genomics+Hum.+Genet&amp;title=Genotype+imputation&amp;author=Y.+Li&amp;author=C.+Willer&amp;author=S.+Sanna&amp;author=G.+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B50">50. <span class="mixed-citation">
Howie B.N., Donnelly P., Marchini J. (2009) 
<span class="ref-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies</span>. <span class="ref-journal">PLoS Genetics</span>, <span class="ref-vol">5</span>, e1000529.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genetics&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=B.N.+Howie&amp;author=P.+Donnelly&amp;author=J.+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529.&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B51">51. <span class="mixed-citation">
Consortium, T.1.G.P., The 1000 Genomes Consortium Participants are arranged by project role, T.B.I.A.A.F.A.W.I.E.F.P.I.A.P.L.A.I., author, C., committee, S., Medicine, P.G.B.C.O., BGI-Shenzhen, Broad Institute of MIT and Harvard, European Bioinformatics Institute, Illumina, Max Planck Institute for Molecular Genetics., et al. (2012) 
<span class="ref-title">An integrated map of genetic variation from 1,092 human genomes</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">490</span>, 56–65. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;volume=490&amp;publication_year=2012&amp;pages=56-65&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B52">52. <span class="mixed-citation">
Li Y., Willer C.J., Ding J., Scheet P., Abecasis G.R. (2010) 
<span class="ref-title">MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">34</span>, 816–834. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y.+Li&amp;author=C.J.+Willer&amp;author=J.+Ding&amp;author=P.+Scheet&amp;author=G.R.+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B53">53. <span class="mixed-citation">
Winkler T.W., Day F.R., Croteau-Chonka D.C., Wood A.R., Locke A.E., Mägi R., Ferreira T., Fall T., Graff M., Justice A.E., et al. (2014) 
<span class="ref-title">Quality control and conduct of genome-wide association meta-analyses</span>. <span class="ref-journal">Nat. Protoc</span>., <span class="ref-vol">9</span>, 1192–1212. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4083217/">PMC free article</a>]</span> [<a href="/pubmed/24762786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc&amp;title=Quality+control+and+conduct+of+genome-wide+association+meta-analyses&amp;author=T.W.+Winkler&amp;author=F.R.+Day&amp;author=D.C.+Croteau-Chonka&amp;author=A.R.+Wood&amp;author=A.E.+Locke&amp;volume=9&amp;publication_year=2014&amp;pages=1192-1212&amp;pmid=24762786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B54">54. <span class="mixed-citation">
Willer C.J., Li Y., Abecasis G.R. (2010) 
<span class="ref-title">METAL: fast and efficient meta-analysis of genomewide association scans</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">26</span>, 2190–2191. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B55">55. <span class="mixed-citation">
Mägi R., Morris A.P. (2010) 
<span class="ref-title">GWAMA: software for genome-wide association meta-analysis</span>. <span class="ref-journal">BMC Bioinformatics</span>, <span class="ref-vol">11</span>, 288.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2893603/">PMC free article</a>]</span> [<a href="/pubmed/20509871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=GWAMA:+software+for+genome-wide+association+meta-analysis&amp;author=R.+Mägi&amp;author=A.P.+Morris&amp;volume=11&amp;publication_year=2010&amp;pages=288.&amp;pmid=20509871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B56">56. <span class="mixed-citation">
Tregouet D.A., Garelle V. (2007) 
<span class="ref-title">A new JAVA interface implementation of THESIAS: testing haplotype effects in association studies</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">23</span>, 1038–1039. [<a href="/pubmed/17308338" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+new+JAVA+interface+implementation+of+THESIAS:+testing+haplotype+effects+in+association+studies&amp;author=D.A.+Tregouet&amp;author=V.+Garelle&amp;volume=23&amp;publication_year=2007&amp;pages=1038-1039&amp;pmid=17308338&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B57">57. <span class="mixed-citation">
Jackson V., Petrini J., Caidahl K., Eriksson M.J., Liska J., Eriksson P., Franco-Cereceda A. (2011) 
<span class="ref-title">Bicuspid aortic valve leaflet morphology in relation to aortic root morphology: a study of 300 patients undergoing open-heart surgery</span>. <span class="ref-journal">Eur. J. Cardiothorac. Surg</span>., <span class="ref-vol">40</span>, e118–e124. [<a href="/pubmed/21620721" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Cardiothorac.+Surg&amp;title=Bicuspid+aortic+valve+leaflet+morphology+in+relation+to+aortic+root+morphology:+a+study+of+300+patients+undergoing+open-heart+surgery&amp;author=V.+Jackson&amp;author=J.+Petrini&amp;author=K.+Caidahl&amp;author=M.J.+Eriksson&amp;author=J.+Liska&amp;volume=40&amp;publication_year=2011&amp;pages=e118-e124&amp;pmid=21620721&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B58">58. <span class="mixed-citation">
Folkersen L., van't Hooft F., Chernogubova E., Agardh H.E., Hansson G.K., Hedin U., Liska J., Syvänen A.C., Paulsson-Berne G., Paulssson-Berne G., et al. (2010) 
<span class="ref-title">Association of genetic risk variants with expression of proximal genes identifies novel susceptibility genes for cardiovascular disease</span>. <span class="ref-journal">Circ. Cardiovasc. Genet</span>., <span class="ref-vol">3</span>, 365–373. [<a href="/pubmed/20562444" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet&amp;title=Association+of+genetic+risk+variants+with+expression+of+proximal+genes+identifies+novel+susceptibility+genes+for+cardiovascular+disease&amp;author=L.+Folkersen&amp;author=F.+van't+Hooft&amp;author=E.+Chernogubova&amp;author=H.E.+Agardh&amp;author=G.K.+Hansson&amp;volume=3&amp;publication_year=2010&amp;pages=365-373&amp;pmid=20562444&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B59">59. <span class="mixed-citation">
Chang C.C., Chow C.C., Tellier L.C., Vattikuti S., Purcell S.M., Lee J.J. (2015) 
<span class="ref-title">Second-generation PLINK: rising to the challenge of larger and richer datasets</span>. <span class="ref-journal">GigaScience</span>, <span class="ref-vol">4</span>, 559. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4342193/">PMC free article</a>]</span> [<a href="/pubmed/25722852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=Second-generation+PLINK:+rising+to+the+challenge+of+larger+and+richer+datasets&amp;author=C.C.+Chang&amp;author=C.C.+Chow&amp;author=L.C.+Tellier&amp;author=S.+Vattikuti&amp;author=S.M.+Purcell&amp;volume=4&amp;publication_year=2015&amp;pages=559.&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B60">60. <span class="mixed-citation">
Gaunt T.R., Rodríguez S., Day I.N. (2007) 
<span class="ref-title">Cubic exact solutions for the estimation of pairwise haplotype frequencies: implications for linkage disequilibrium analyses and a web tool ‘CubeX’</span>. <span class="ref-journal">BMC Bioinformatics</span>, <span class="ref-vol">8</span>, 428.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2180187/">PMC free article</a>]</span> [<a href="/pubmed/17980034" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=Cubic+exact+solutions+for+the+estimation+of+pairwise+haplotype+frequencies:+implications+for+linkage+disequilibrium+analyses+and+a+web+tool+‘CubeX’&amp;author=T.R.+Gaunt&amp;author=S.+Rodríguez&amp;author=I.N.+Day&amp;volume=8&amp;publication_year=2007&amp;pages=428.&amp;pmid=17980034&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B61">61. <span class="mixed-citation">
Lewis B.P., Burge C.B., Bartel D.P. (2005) 
<span class="ref-title">Conserved Seed Pairing, Often Flanked by Adenosines, Indicates that Thousands of Human Genes are MicroRNA Targets</span>. <span class="ref-journal">Cell</span>, <span class="ref-vol">120</span>, 15–20. [<a href="/pubmed/15652477" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Conserved+Seed+Pairing,+Often+Flanked+by+Adenosines,+Indicates+that+Thousands+of+Human+Genes+are+MicroRNA+Targets&amp;author=B.P.+Lewis&amp;author=C.B.+Burge&amp;author=D.P.+Bartel&amp;volume=120&amp;publication_year=2005&amp;pages=15-20&amp;pmid=15652477&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B62">62. <span class="mixed-citation">
Grimson A., Farh K.K.H., Johnston W.K., Garrett-Engele P., Lim L.P., Bartel D.P. (2007) 
<span class="ref-title">MicroRNA Targeting Specificity in Mammals: Determinants beyond Seed Pairing</span>. <span class="ref-journal">Molecular Cell</span>, <span class="ref-vol">27</span>, 91–105. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3800283/">PMC free article</a>]</span> [<a href="/pubmed/17612493" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecular+Cell&amp;title=MicroRNA+Targeting+Specificity+in+Mammals:+Determinants+beyond+Seed+Pairing&amp;author=A.+Grimson&amp;author=K.K.H.+Farh&amp;author=W.K.+Johnston&amp;author=P.+Garrett-Engele&amp;author=L.P.+Lim&amp;volume=27&amp;publication_year=2007&amp;pages=91-105&amp;pmid=17612493&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B63">63. <span class="mixed-citation">
Garcia D.M., Baek D., Shin C., Bell G.W., Grimson A., Bartel D.P. (2011) 
<span class="ref-title">Weak seed-pairing stability and high target-site abundance decrease the proficiency of lsy-6 and other microRNAs</span>. <span class="ref-journal">Nat. Struct. Mol. Biol</span>., <span class="ref-vol">18</span>, 1139–1146. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3190056/">PMC free article</a>]</span> [<a href="/pubmed/21909094" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol&amp;title=Weak+seed-pairing+stability+and+high+target-site+abundance+decrease+the+proficiency+of+lsy-6+and+other+microRNAs&amp;author=D.M.+Garcia&amp;author=D.+Baek&amp;author=C.+Shin&amp;author=G.W.+Bell&amp;author=A.+Grimson&amp;volume=18&amp;publication_year=2011&amp;pages=1139-1146&amp;pmid=21909094&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B64">64. <span class="mixed-citation">
Friedman R.C., Farh K.K.H., Burge C.B., Bartel D.P. (2009) 
<span class="ref-title">Most mammalian mRNAs are conserved targets of microRNAs</span>. <span class="ref-journal">Genome Res</span>., <span class="ref-vol">19</span>, 92–105. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2612969/">PMC free article</a>]</span> [<a href="/pubmed/18955434" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Most+mammalian+mRNAs+are+conserved+targets+of+microRNAs&amp;author=R.C.+Friedman&amp;author=K.K.H.+Farh&amp;author=C.B.+Burge&amp;author=D.P.+Bartel&amp;volume=19&amp;publication_year=2009&amp;pages=92-105&amp;pmid=18955434&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B65">65. <span class="mixed-citation">
John B., Enright A.J., Aravin A., Tuschl T., Sander C., Marks D.S. (2004) 
<span class="ref-title">Human MicroRNA targets</span>. <span class="ref-journal">PLoS Biol</span>., <span class="ref-vol">2</span>, e363.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC521178/">PMC free article</a>]</span> [<a href="/pubmed/15502875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Human+MicroRNA+targets&amp;author=B.+John&amp;author=A.J.+Enright&amp;author=A.+Aravin&amp;author=T.+Tuschl&amp;author=C.+Sander&amp;volume=2&amp;publication_year=2004&amp;pages=e363.&amp;pmid=15502875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B66">66. <span class="mixed-citation">
Betel D., Koppal A., Agius P., Sander C. (2010) 
<span class="ref-title">Comprehensive modeling of microRNA targets predicts functional non-conserved and non-canonical sites</span>. <span class="ref-journal">Genome Biol</span>., 11:R90. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2945792/">PMC free article</a>]</span> [<a href="/pubmed/20799968" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Comprehensive+modeling+of+microRNA+targets+predicts+functional+non-conserved+and+non-canonical+sites&amp;author=D.+Betel&amp;author=A.+Koppal&amp;author=P.+Agius&amp;author=C.+Sander&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B67">67. <span class="mixed-citation">
Dweep H., Sticht C., Pandey P., Gretz N. (2011) 
<span class="ref-title">miRWalk–database: prediction of possible miRNA binding sites by ‘walking’ the genes of three genomes</span>. <span class="ref-journal">J. Biomed. Inform</span>., <span class="ref-vol">44</span>, 839–847. [<a href="/pubmed/21605702" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biomed.+Inform&amp;title=miRWalk–database:+prediction+of+possible+miRNA+binding+sites+by+‘walking’+the+genes+of+three+genomes&amp;author=H.+Dweep&amp;author=C.+Sticht&amp;author=P.+Pandey&amp;author=N.+Gretz&amp;volume=44&amp;publication_year=2011&amp;pages=839-847&amp;pmid=21605702&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B68">68. <span class="mixed-citation">
Griffiths-Jones S., Saini H.K., van Dongen S., Enright A.J. (2007) 
<span class="ref-title">miRBase: tools for microRNA genomics</span>. <span class="ref-journal">Nucleic Acids Res</span>., <span class="ref-vol">36</span>, D154–D158. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2238936/">PMC free article</a>]</span> [<a href="/pubmed/17991681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=miRBase:+tools+for+microRNA+genomics&amp;author=S.+Griffiths-Jones&amp;author=H.K.+Saini&amp;author=S.+van+Dongen&amp;author=A.J.+Enright&amp;volume=36&amp;publication_year=2007&amp;pages=D154-D158&amp;pmid=17991681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B69">69. <span class="mixed-citation">
Kozomara A., Griffiths-Jones S. (2011) 
<span class="ref-title">miRBase: integrating microRNA annotation and deep-sequencing data</span>. <span class="ref-journal">Nucleic Acids Res</span>., <span class="ref-vol">39</span>, D152–D157. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3013655/">PMC free article</a>]</span> [<a href="/pubmed/21037258" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=miRBase:+integrating+microRNA+annotation+and+deep-sequencing+data&amp;author=A.+Kozomara&amp;author=S.+Griffiths-Jones&amp;volume=39&amp;publication_year=2011&amp;pages=D152-D157&amp;pmid=21037258&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B70">70. <span class="mixed-citation">
Landgraf P., Rusu M., Sheridan R., Sewer A., Iovino N., Aravin A., Pfeffer S., Rice A., Kamphorst A.O., Landthaler M., et al. (2007) 
<span class="ref-title">A mammalian microRNA expression atlas based on small RNA library sequencing</span>. <span class="ref-journal">Cell</span>, <span class="ref-vol">129</span>, 1401–1414. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681231/">PMC free article</a>]</span> [<a href="/pubmed/17604727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+mammalian+microRNA+expression+atlas+based+on+small+RNA+library+sequencing&amp;author=P.+Landgraf&amp;author=M.+Rusu&amp;author=R.+Sheridan&amp;author=A.+Sewer&amp;author=N.+Iovino&amp;volume=129&amp;publication_year=2007&amp;pages=1401-1414&amp;pmid=17604727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B71">71. <span class="mixed-citation">
Vlachos I.S., Paraskevopoulou M.D., Karagkouni D., Georgakilas G., Vergoulis T., Kanellos I., Anastasopoulos I.L., Maniou S., Karathanou K., Kalfakakou D., et al. (2015) 
<span class="ref-title">DIANA-TarBase v7.0: indexing more than half a million experimentally supported miRNA:mRNA interactions</span>. <span class="ref-journal">Nucleic Acids Res</span>., <span class="ref-vol">43</span>, D153–D159. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4383989/">PMC free article</a>]</span> [<a href="/pubmed/25416803" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=DIANA-TarBase+v7.0:+indexing+more+than+half+a+million+experimentally+supported+miRNA:mRNA+interactions&amp;author=I.S.+Vlachos&amp;author=M.D.+Paraskevopoulou&amp;author=D.+Karagkouni&amp;author=G.+Georgakilas&amp;author=T.+Vergoulis&amp;volume=43&amp;publication_year=2015&amp;pages=D153-D159&amp;pmid=25416803&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B72">72. <span class="mixed-citation">
Li Y., Challagundla K.B., Sun X.X., Zhang Q., Dai M.S. (2015) 
<span class="ref-title">MicroRNA-130a associates with ribosomal protein L11 to suppress c-Myc expression in response to UV irradiation</span>. <span class="ref-journal">Oncotarget</span>., <span class="ref-vol">6</span>, 1101–1114. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4359220/">PMC free article</a>]</span> [<a href="/pubmed/25544755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=MicroRNA-130a+associates+with+ribosomal+protein+L11+to+suppress+c-Myc+expression+in+response+to+UV+irradiation&amp;author=Y.+Li&amp;author=K.B.+Challagundla&amp;author=X.X.+Sun&amp;author=Q.+Zhang&amp;author=M.S.+Dai&amp;volume=6&amp;publication_year=2015&amp;pages=1101-1114&amp;pmid=25544755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B73">73. <span class="mixed-citation">
Consortium R.E., Kundaje A., Meuleman W., Ernst J., Bilenky M., Yen A., Heravi-Moussavi A., Kheradpour P., Zhang Z., Wang J., et al. (2015) 
<span class="ref-title">Integrative analysis of 111 reference human epigenomes</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">518</span>, 317–330. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4530010/">PMC free article</a>]</span> [<a href="/pubmed/25693563" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrative+analysis+of+111+reference+human+epigenomes&amp;author=R.E.+Consortium&amp;author=A.+Kundaje&amp;author=W.+Meuleman&amp;author=J.+Ernst&amp;author=M.+Bilenky&amp;volume=518&amp;publication_year=2015&amp;pages=317-330&amp;pmid=25693563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B74">74. <span class="mixed-citation">
Consortium E.P. (2011) 
<span class="ref-title">A user's guide to the encyclopedia of DNA elements (ENCODE)</span>. <span class="ref-journal">PLoS Biol</span>., <span class="ref-vol">9</span>, e1001046.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3079585/">PMC free article</a>]</span> [<a href="/pubmed/21526222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=A+user's+guide+to+the+encyclopedia+of+DNA+elements+(ENCODE)&amp;author=E.P.+Consortium&amp;volume=9&amp;publication_year=2011&amp;pages=e1001046.&amp;pmid=21526222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw401-B75">75. <span class="mixed-citation">
Kheradpour P., Kellis M. (2014) 
<span class="ref-title">Systematic discovery and characterization of regulatory motifs in ENCODE TF binding experiments</span>. <span class="ref-journal">Nucleic Acids Res</span>., <span class="ref-vol">42</span>, 2976–2987. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3950668/">PMC free article</a>]</span> [<a href="/pubmed/24335146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Systematic+discovery+and+characterization+of+regulatory+motifs+in+ENCODE+TF+binding+experiments&amp;author=P.+Kheradpour&amp;author=M.+Kellis&amp;volume=42&amp;publication_year=2014&amp;pages=2976-2987&amp;pmid=24335146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5703348&amp;issue-id=291614&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139725723966288title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Results</a></li><li><a href="#__sec9title">Discussion</a></li><li><a href="#__sec10title">Materials and methods</a></li><li><a href="#__sec24title">Supplementary Material</a></li><li><a href="#__sec25title">Supplementary Material</a></li><li><a href="#idm139725720198368title">Acknowledgements</a></li><li><a href="#idm139725720191520title">References </a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5703348/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5703348/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5703348/pdf/ddw401.pdf">PDF (260K)</a></li> | <li><a href="#" data-citationid="PMC5703348" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5703348%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5703348%2F&amp;text=Genome-wide%20association%20study%20with%20additional%20genetic%20and%20post-transcriptional%20analyses%20reveals%20novel%20regulators%20of%20plasma%20factor%20XI%20levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5703348%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28053049" data-aiid="5703348" data-aid="5703348" data-iid="291614" data-domainid="740" data-domain="hmg" data-accid="PMC5703348" data-md5="150f56d10235cc43df55f7a6f243e0cf"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:35:41-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE883341DD5983910000000007AC0474&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5703348%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5703348/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Main, sans-serif;"></div></div></body></html>