<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5433429/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-Wide Associations of CD46 and IFI44L Genetic Variants with Neutralizing Antibody Response to Measles Vaccine</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5433429/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human genetics"><meta name="citation_title" content="Genome-Wide Associations of CD46 and IFI44L Genetic Variants with Neutralizing Antibody Response to Measles Vaccine"><meta name="citation_authors" content="Iana H. Haralambieva, Inna G. Ovsyannikova, Richard B. Kennedy, Beth R. Larrabee, Michael T. Zimmermann, Diane E. Grill, Daniel J. Schaid, Gregory A. Poland"><meta name="citation_date" content="April 2017"><meta name="citation_issue" content="4"><meta name="citation_volume" content="136"><meta name="citation_firstpage" content="421"><meta name="citation_doi" content="10.1007/s00439-017-1768-9"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5433429/?report=abstract"><meta name="citation_pmid" content="28289848"><meta name="DC.Title" content="Genome-Wide Associations of CD46 and IFI44L Genetic Variants with Neutralizing Antibody Response to Measles Vaccine"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Iana H. Haralambieva"><meta name="DC.Contributor" content="Inna G. Ovsyannikova"><meta name="DC.Contributor" content="Richard B. Kennedy"><meta name="DC.Contributor" content="Beth R. Larrabee"><meta name="DC.Contributor" content="Michael T. Zimmermann"><meta name="DC.Contributor" content="Diane E. Grill"><meta name="DC.Contributor" content="Daniel J. Schaid"><meta name="DC.Contributor" content="Gregory A. Poland"><meta name="DC.Date" content="2017 Apr"><meta name="DC.Identifier" content="10.1007/s00439-017-1768-9"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-Wide Associations of CD46 and IFI44L Genetic Variants with Neutralizing Antibody Response to Measles Vaccine"><meta property="og:type" content="article"><meta property="og:description" content="Population-based studies have revealed 2 to 10% measles vaccine failure rate even after two vaccine doses. While the mechanisms behind this remain unknown, we hypothesized that host genetic factors are likely to be involved.We performed a genome-wide ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5433429/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5433429/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5433429/pdf/nihms859705.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE896FD6DD59B8F100000000077E0496.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathEvents.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathZoom.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathMenu.js?rev=2.6.1"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5433429%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5433429</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Genet</a></span>. Author manuscript; available in PMC 2018 Apr 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28289848" target="pmc_ext" ref="reftype=publisher&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Hum Genet. 2017 Apr; 136(4): 421–435. </span></a></div><span class="fm-vol-iss-date">Published online 2017 Mar 13. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1007%2Fs00439-017-1768-9" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s00439-017-1768-9</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5433429</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS859705</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28289848">28289848</a></div></div></div></div><h1 class="content-title">Genome-Wide Associations of CD46 and IFI44L Genetic Variants with Neutralizing Antibody Response to Measles Vaccine</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Haralambieva%20IH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848" class="affpopup" co-rid="_co_idm140355366716704" co-class="co-affbox">Iana H. Haralambieva</a>,<sup>1</sup> <a href="/pubmed/?term=Ovsyannikova%20IG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848" class="affpopup" co-rid="_co_idm140355356430960" co-class="co-affbox">Inna G. Ovsyannikova</a>,<sup>1</sup> <a href="/pubmed/?term=Kennedy%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848" class="affpopup" co-rid="_co_idm140355369304160" co-class="co-affbox">Richard B. Kennedy</a>,<sup>1</sup> <a href="/pubmed/?term=Larrabee%20BR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848" class="affpopup" co-rid="_co_idm140355366712992" co-class="co-affbox">Beth R. Larrabee</a>,<sup>2</sup> <a href="/pubmed/?term=Zimmermann%20MT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848" class="affpopup" co-rid="_co_idm140355366710752" co-class="co-affbox">Michael T. Zimmermann</a>,<sup>2</sup> <a href="/pubmed/?term=Grill%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848" class="affpopup" co-rid="_co_idm140355356572736" co-class="co-affbox">Diane E. Grill</a>,<sup>2</sup> <a href="/pubmed/?term=Schaid%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848" class="affpopup" co-rid="_co_idm140355356570560" co-class="co-affbox">Daniel J. Schaid</a>,<sup>2</sup> and  <a href="/pubmed/?term=Poland%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848" class="affpopup" co-rid="_co_idm140355356568384" co-class="co-affbox">Gregory A. Poland</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140355366716704"><h3 class="no_margin">Iana H. Haralambieva</h3><p><sup>1</sup>Mayo Clinic Vaccine Research Group, Mayo Clinic, Rochester, MN 55905 USA</p><div>Find articles by <a href="/pubmed/?term=Haralambieva%20IH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848">Iana H. Haralambieva</a></div></div><div id="_co_idm140355356430960"><h3 class="no_margin">Inna G. Ovsyannikova</h3><p><sup>1</sup>Mayo Clinic Vaccine Research Group, Mayo Clinic, Rochester, MN 55905 USA</p><div>Find articles by <a href="/pubmed/?term=Ovsyannikova%20IG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848">Inna G. Ovsyannikova</a></div></div><div id="_co_idm140355369304160"><h3 class="no_margin">Richard B. Kennedy</h3><p><sup>1</sup>Mayo Clinic Vaccine Research Group, Mayo Clinic, Rochester, MN 55905 USA</p><div>Find articles by <a href="/pubmed/?term=Kennedy%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848">Richard B. Kennedy</a></div></div><div id="_co_idm140355366712992"><h3 class="no_margin">Beth R. Larrabee</h3><p><sup>2</sup>Division of Biomedical Statistics and Informatics, Department of Health Science Research, Mayo Clinic, Rochester, MN 55905 USA</p><div>Find articles by <a href="/pubmed/?term=Larrabee%20BR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848">Beth R. Larrabee</a></div></div><div id="_co_idm140355366710752"><h3 class="no_margin">Michael T. Zimmermann</h3><p><sup>2</sup>Division of Biomedical Statistics and Informatics, Department of Health Science Research, Mayo Clinic, Rochester, MN 55905 USA</p><div>Find articles by <a href="/pubmed/?term=Zimmermann%20MT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848">Michael T. Zimmermann</a></div></div><div id="_co_idm140355356572736"><h3 class="no_margin">Diane E. Grill</h3><p><sup>2</sup>Division of Biomedical Statistics and Informatics, Department of Health Science Research, Mayo Clinic, Rochester, MN 55905 USA</p><div>Find articles by <a href="/pubmed/?term=Grill%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848">Diane E. Grill</a></div></div><div id="_co_idm140355356570560"><h3 class="no_margin">Daniel J. Schaid</h3><p><sup>2</sup>Division of Biomedical Statistics and Informatics, Department of Health Science Research, Mayo Clinic, Rochester, MN 55905 USA</p><div>Find articles by <a href="/pubmed/?term=Schaid%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848">Daniel J. Schaid</a></div></div><div id="_co_idm140355356568384"><h3 class="no_margin">Gregory A. Poland</h3><p><sup>1</sup>Mayo Clinic Vaccine Research Group, Mayo Clinic, Rochester, MN 55905 USA</p><div>Find articles by <a href="/pubmed/?term=Poland%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28289848">Gregory A. Poland</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140355370070848_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140355370070848_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140355370070848_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Mayo Clinic Vaccine Research Group, Mayo Clinic, Rochester, MN 55905 USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Division of Biomedical Statistics and Informatics, Department of Health Science Research, Mayo Clinic, Rochester, MN 55905 USA</div><div id="FN1">Address correspondence to: Gregory A. Poland, M.D., Director, Mayo Vaccine Research Group, Mayo Clinic, Guggenheim 611C, 200 First Street SW, Rochester, Minnesota 55905, Phone: (507) 284-4968; Fax: (507) 266-4716; <a href="mailto:dev@null" data-email="ude.oyam@yrogerg.dnalop" class="oemail">ude.oyam@yrogerg.dnalop</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140355370070848_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28289848" target="pmc_ext" ref="reftype=publisher&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Hum Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5433429/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="6" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>439_2017_1768_MOESM1_ESM: </strong>Supplemental Table 1 Top 20 genome-wide associations of SNPs with measles-specific IFNγ ELISPOT response after MMR vaccination (<em>Combined analysis, n=2872</em><sup>a</sup>)</div><div><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM1_ESM.docx" data-ga-action="click_feat_suppl">NIHMS859705-supplement-439_2017_1768_MOESM1_ESM.docx</a><span style="color:gray"> (34K)</span></div><div class="small guid">GUID:&nbsp;F22DC814-33C0-4D7C-A2D0-E6073882A3F8</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>439_2017_1768_MOESM2_ESM: </strong>Supplemental Table 2 Associations between chromosome 1 SNPs and measles virus-specific secreted IFNα (<em>Caucasians, n=625</em> subjects with available measles virus-specific cytokine data)</div><div><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM2_ESM.docx" data-ga-action="click_feat_suppl">NIHMS859705-supplement-439_2017_1768_MOESM2_ESM.docx</a><span style="color:gray"> (23K)</span></div><div class="small guid">GUID:&nbsp;B70F6CCE-3CC8-4642-9B87-2E31D44F273F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>439_2017_1768_MOESM3_ESM: </strong>Supplemental Fig. 1 Manhattan and Q-Q plots summary of GWAS associations between SNPs and measles-specific neutralizing antibody titers.<p><strong>A.</strong> Manhattan (left panel) and Q-Q (right panel) plots for the combined cohort analysis (n=2872, reduced to 2818 after excluding subjects with neutralizing antibody data, that failed QC).</p><p><strong>B.</strong> Manhattan (left panel) and Q-Q (right panel) plots for the subset analysis of subjects of Caucasian ancestry (n=2555, reduced to 2506 after excluding subjects with neutralizing antibody data, that failed QC).</p></div><div><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM3_ESM.tif" data-ga-action="click_feat_suppl">NIHMS859705-supplement-439_2017_1768_MOESM3_ESM.tif</a><span style="color:gray"> (453K)</span></div><div class="small guid">GUID:&nbsp;84565DC0-3551-4DB7-BC27-D806B33EEB88</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>439_2017_1768_MOESM4_ESM: </strong>Supplemental Fig. 2 Haplotype block structure of the significant IFI44L and CD46 SNPs associated with measles-specific antibody response (combined sample).<p>The schematic representation and LD block structure of <em>IFI44L</em> (left) and <em>CD46</em> (right) genetic regions are depicted (significantly associated SNPs only). The LD block structure was analyzed using Haploview software, version 4.2. The r<sup>2</sup> color scheme is: white (r<sup>2</sup>=0), shades of grey (0&lt; r<sup>2</sup>&lt;1), black (r<sup>2</sup>=1). The numbers report the r<sup>2</sup> value multiplied by 100.</p></div><div><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM4_ESM.tif" data-ga-action="click_feat_suppl">NIHMS859705-supplement-439_2017_1768_MOESM4_ESM.tif</a><span style="color:gray"> (1.1M)</span></div><div class="small guid">GUID:&nbsp;BF02FBF0-510E-40AE-91CF-30F7051C3B0B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>439_2017_1768_MOESM5_ESM: </strong>Supplemental Fig. 3 Manhattan plot summary of GWAS associations between SNPs and measles-specific IFNγ ELISPOT response in the combined cohort (n=2872<sup>a</sup>)<p><sup>a</sup>Reduced to 2618 after excluding subjects with immune outcome data that failed QC</p></div><div><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM5_ESM.tif" data-ga-action="click_feat_suppl">NIHMS859705-supplement-439_2017_1768_MOESM5_ESM.tif</a><span style="color:gray"> (199K)</span></div><div class="small guid">GUID:&nbsp;BB54CEB8-9F10-4D79-8D15-E794BBBC60E1</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>439_2017_1768_MOESM6_ESM: </strong>Supplemental Fig. 4 CD46 isoforms exhibit different flexibilities, specifically about the hinge between the SCR4 and STP domains.<p><strong>A.</strong> Molecular structure of the full length CD46, zoomed in to emphasize the differentially spliced exons.</p><p><strong>B.</strong> Using the first 3 modes of an ANM model (see Methods), we compute the mobility of each residue. There is increased mobility for the C1 isoform. (inset) The normal mode frequencies are plotted on a log-log scale and indicate a dramatically lower collectivity for the C1 isoform.</p><p><strong>C.</strong> Commute times are computed for each structure and show a decrease in C1 relative to BC1. (inset) Example matrix of commute times from the BC1 isoform with the N-terminus at the top left and C-terminus bottom right.</p><p><strong>D.</strong> and <strong>E.</strong> We choose representative C<sup>α</sup> atoms to define the hinge angle between the exon 6 subdomain and the subdomain comprised of the isoform-specific sequences. <strong>D.</strong> for BC1, relatively low mobility about this hinge region is observed (ANM mode 2), while greater flexibility is observed in C1 (ANM mode 2) in <strong>E.</strong> We show representative structures from the ANM modes, deformed to 2Å RMSD in both directions and superimposed about the sequences encoded by the variable exons.</p><p><strong>F.</strong> Across the first 5 low-frequency ANM modes, we indicate the change in this angle observed when deforming each structure to 2Å RMSD in each direction.</p></div><div><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM6_ESM.tif" data-ga-action="click_feat_suppl">NIHMS859705-supplement-439_2017_1768_MOESM6_ESM.tif</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID:&nbsp;8801A79D-2B1A-44D0-9A39-1F6BB0E24CDC</div></div></dd></dl></div><div id="idm140355357474256" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140355357474256title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P1" class="p p-first-last">Population-based studies have revealed 2 to 10% measles vaccine failure rate even after two vaccine doses. While the mechanisms behind this remain unknown, we hypothesized that host genetic factors are likely to be involved.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">We performed a genome-wide association study of measles specific neutralizing antibody and IFNγ ELISPOT response in a combined sample of 2,872 subjects.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">We identified two distinct chromosome 1 regions (previously associated with MMR-related febrile seizures), associated with vaccine-induced measles neutralizing antibody titers. The 1q32 region contained 20 significant SNPs in/around the measles virus receptor-encoding <em>CD46</em> gene, including the intronic rs2724384 (p-value = 2.64x10<sup>−09</sup>) and rs2724374 (p-value = 3.16x10<sup>−09</sup>) SNPs. The 1q31.1 region contained nine significant SNPs in/around <em>IFI44L</em>, including the intronic rs1333973 (p-value = 1.41x10<sup>−10</sup>) and the missense rs273259 (His73Arg, p-value = 2.87x10<sup>−10</sup>) SNPs. Analysis of differential exon usage with mRNA-Seq data and RT-PCR suggests the involvement of rs2724374 minor G allele in the <em>CD46</em> STP region exon B skipping, resulting in shorter CD46 isoforms.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions</h3><p id="P4" class="p p-first-last">Our study reveals common <em>CD46</em> and <em>IFI44L</em> SNPs associated with measles-specific humoral immunity, and highlights the importance of alternative splicing/virus cellular receptor isoform usage as a mechanism explaining inter-individual variation in immune response after live measles vaccine.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Genome-Wide Association Study, Measles, Measles Vaccine, Measles-Mumps-Rubella Vaccine, Immunity, Humoral, Immunity, Cellular, Polymorphism, Single Nucleotide, Alternative Splicing, Genetic Variation, CD46 protein, Human, Adult</span></div></div><div id="idm140355357027888" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">Measles still remains a disease of public health concern in the developing world and well-developed countries with multiple outbreaks even among populations with high vaccine coverage. From 2010 to date, the European region registered 135,600 measles cases, and the US experienced 1,381 measles cases in 27 states. (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116937">Haralambieva et al. 2015</a>; <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116952">Haralambieva et al. 2013</a>; <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116887">Poland and Jacobson 2012</a>; <a href="#R38" rid="R38" class=" bibr popnode">Prevention 2015</a>; <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116954">Whitaker and Poland 2014</a>) Several population-based studies have estimated that 2 to 10% of vaccine recipients do not develop or sustain measles-specific protective immunity after two doses of MMR vaccine. (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116971">Bednarczyk et al. 2016</a>; <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116970">Haralambieva et al. 2011b</a>; <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116926">Haralambieva et al. 2013</a>; <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116961">Poland and Jacobson 2012</a>; <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116973">Whitaker and Poland 2014</a>) The mechanisms behind vaccine failure are unknown. This knowledge gap is an impediment to controlling future outbreaks or designing improved vaccine candidates.</p><p id="P6">Measles vaccine-induced humoral immunity is reported to have an extremely high heritability of 88.5%. (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116904">Tan et al. 2001</a>) We have performed a series of candidate genetic association studies delineating the effect of HLA alleles and single nucleotide polymorphisms on measles humoral and cellular immune responses, but thus far only approximately 30% of the inter-individual variation in immune response to this vaccine can be explained.(<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116905">Dhiman et al. 2007</a>; <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116938">Haralambieva et al. 2015</a>; <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116881">Haralambieva et al. 2011a</a>; <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608117006">Haralambieva et al. 2013</a>; <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116906">Haralambieva et al. 2011c</a>; <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116894">Kennedy et al. 2012a</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116943">Ovsyannikova et al. 2011a</a>; <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116896">Ovsyannikova et al. 2011b</a>; <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116998">Ovsyannikova et al. 2012</a>)</p><p id="P7" class="p">We report the first GWAS study (on a sample of 2,872 subjects) of measles vaccine-induced humoral and cellular immune response outcomes in children and younger adults, that identify significant SNP associations (in the <em>CD46</em> and <em>IFI44L</em> genes) underlying the observed inter-individual variability in neutralizing antibody titers after vaccination.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Methods</h2><p id="P8" class="p p-first">Additional methods details are available in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary data</a>.</p><div id="S6" class="sec"><h3 id="S6title">Study Subjects</h3><p id="P9" class="p p-first">We analyzed a large sample of 3,191 healthy children, older adolescents, and healthy adults (age 11 to 40 years) consisting of three independent cohorts: a Rochester cohort (n=1,062); a San Diego cohort (n=1,071); and a US cohort (n=1,058). The demographic and clinical characteristics of these cohorts have been previously published (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116972">Haralambieva et al. 2011a</a>; <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116979">Haralambieva et al. 2011c</a>; <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116975">Kennedy et al. 2012a</a>; <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116923">Kennedy et al. 2012b</a>; <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608117008">Kennedy et al. 2012c</a>; <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116953">Lambert et al. 2015</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116955">Ovsyannikova et al. 2011a</a>; <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116999">Ovsyannikova et al. 2011b</a>; <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116867">Ovsyannikova et al. 2012</a>).</p><p id="P10" class="p p-last">The Institutional Review Boards of the Mayo Clinic (Rochester, MN) and the NHRC (San Diego, CA) approved the study, and written informed consent was obtained from each subject, i.e., from age-appropriate participants and the parents of all children who participated in the study.</p></div><div id="S7" class="sec"><h3 id="S7title">Genotyping and Immune Outcomes</h3><p id="P11" class="p p-first-last">The genome-wide SNP typing was performed using the Infinium Omni 1M-Quad SNP array (Illumina; San Diego, CA) for the Rochester cohort, Illumina Human Omni2.5–8 BeadChip array for the US cohort, and Illumina Infinium HumanHap550v3_A or HumanHap650Yv3 BeadChip arrays for the San Diego cohort. Measles-specific neutralizing antibody and cytokine responses were quantified using a fluorescence-based plaque reduction microneutralization assay (PRMN) and ELISPOT/ELISA assays, as previously described (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116948">Haralambieva et al. 2011b</a>). Our humoral immune response phenotype, the 50% end-point titer (Neutralizing Doze, ND<sub>50</sub>), was calculated using Karber’s formula and transformed into mIU/mL (using the 3<sup>rd</sup> WHO international measles antibody standard), as described previously (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116876">Haralambieva et al. 2011b</a>). The variability of the PRMN assay, calculated as a coefficient of variation (CV) based on the log-transformed ND<sub>50</sub> values of the third WHO standard, was 5.7%, as previously published (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116911">Haralambieva et al. 2011b</a>).</p></div><div id="S8" class="sec"><h3 id="S8title">Next Generation Sequencing (mRNA-Seq) and RT-PCR</h3><p id="P12" class="p p-first-last">Libraries were generated from total RNA (extracted from PBMCs of 30 subjects) using Illumina’s mRNA TruSeq (v1) kit and sequenced (paired end sequencing) on an Illumina HiSeq 2000 (Illumina; San Diego, CA) with Illumina’s TruSeq Cluster kit (v3-cBot-HS) and 51 Cycle Illumina TruSeq SBS Sequencing Kit (v3), as previously published (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116982">Haralambieva et al. 2016</a>). One-Step RT-PCR system with Platinum Taq<sup>®</sup> DNA polymerase (Invitrogen, Carlsbad, CA) and primers allowing CD46 isoform/isoforms discrimination were used, as previously described (<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116900">Wang et al. 2000</a>).</p></div><div id="S9" class="sec sec-last"><h3 id="S9title">Statistical methods</h3><div id="S10" class="sec sec-first"><p></p><h4 id="S10title" class="inline">GWAS Analysis </h4><p id="P13" class="p p-first-last">To achieve greatest power to detect SNPs associated with measles-specific immune response phenotypes, we pooled our data across genotyping platforms and the three cohorts. To perform the pooled analyses, we first accounted for the effects of potentially confounding factors that vary across ancestry/platform/cohort strata. After thoroughly evaluating the quality of the genotype data, we used the genetic data to define major ancestry groups. We then estimated eigenvectors within each ancestry group to account for the effects of population stratification within ancestry groups. Because the largest ancestry groups were Caucasian and African-American, we restricted our pooled analysis to these groups. After accounting for population stratification, we evaluated the covariates that were available within each of the ancestry-platform-cohort strata to determine if the covariates were associated with the phenotype, in order to regress out the effects of potential confounding factors. This produced residuals (adjusted traits) that were then used for the GWAS analyses. The immune response trait measles-specific IFNγ ELISPOT, as well as measles-specific secreted cytokines, were transformed by normal quantiles of the difference of the mean stimulated and mean unstimulated values. The immune response trait neutralizing antibody titer was transformed as the natural log of the PRMN mIU/mL value (for more details, refer to <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods</a>).</p></div><div id="S11" class="sec"><p></p><h4 id="S11title" class="inline">Analysis of mRNA-Seq Data for Differential Exon Usage </h4><p id="P14" class="p p-first-last">We tested for evidence of differential exon usage in the CD46 and IFI44L genes using the method of Anders <em>et al.</em>, implemented in the DEXSeq package (version 1.16.10) in the R programing language (version 3.2.3).(<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116934">Anders et al. 2012</a>; <a href="#R45" rid="R45" class=" bibr popnode">Team 2009</a>)</p></div><div id="S12" class="sec sec-last"><p></p><h4 id="S12title" class="inline">Molecular Modeling </h4><p id="P15" class="p p-first-last">In order to evaluate the effect of differential splicing on the dynamics of the CD46 molecular structure, we generated homology models (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116994">Roy et al. 2010</a>) of the extracellular domains (SCR1-4 and STP domains) for each isoform and analyzed each using Anisotropic Network Models (ANMs) (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116912">Atilgan et al. 2001</a>; <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116991">Chennubhotla and Bahar 2007</a>; <a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116871">Yang et al. 2009</a>) using multiple templates (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Data</a> for details). Interactions between CD46 and MV-H were modeled using the crystal structure PDB 3INB (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116958">Santiago et al. 2010</a>) as a template wherein dimeric MV-H is bound symmetrically by the SCR1 and SCR2 domains of two CD46 molecules (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Data</a> for details).</p></div></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Results</h2><div id="S14" class="sec sec-first"><h3 id="S14title">Genome-wide analysis results with humoral immunity</h3><div id="S15" class="sec sec-first"><p></p><h4 id="S15title" class="inline">Genetic regions associated with variations in measles-specific antibody response after vaccination </h4><p id="P16" class="p p-first-last">The demographic and immune characteristics of our study sample (n=2,872) are summarized in <a href="/pmc/articles/PMC5433429/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. We identified two independent gene regions on chromosome 1 associated with antibody response following measles vaccination (<a href="/pmc/articles/PMC5433429/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, <a href="#SD3" rid="SD3" class=" supplementary-material">Supplemental Fig. 1</a>). As depicted on the locus zoom plot (<a href="/pmc/articles/PMC5433429/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1A</span></a>), the right region on chromosome 1 contained multiple SNPs (n=20) in/around the MV receptor-encoding <em>CD46</em> gene and region (1q32, bp 207917499-208025926, NCBI Build 37/hg19). Analyzing associations between antibody response and SNPs in the two other MV receptors on chromosome 1, SLAM (<em>SLAMF1</em>) and nectin-4 (<em>NECTIN4/PVRL4</em>), did not result in significant findings. The left region (<a href="/pmc/articles/PMC5433429/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1C</span></a>) on chromosome 1 (1p31.1, bp 79082772-79110518) contained nine significant SNPs in/around the interferon-induced, protein 44-like gene <em>IFI44L</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5433429/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355360694800" class="figure"><a href="/pmc/articles/PMC5433429/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5433429_nihms859705f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms859705f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5433429/bin/nihms859705f1.jpg"></a></div><div id="largeobj_idm140355360694800" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5433429/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5433429/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Locus zoom plots of the chromosome 1 regions and effects of SNPs associated with measles-specific neutralizing antibody titers in the combined cohort<sup>a</sup></strong><p id="__p4"><strong>A and C.</strong> Locus zoom plots of the 1q32(<em>CD46</em>, <strong>A</strong>) and 1q31.1 (<em>IFI44L</em>, <strong>C</strong>) regions associated with neutralizing antibody titer after measles vaccination (combined cohort<sup>a</sup>). On the x-axis SNPs are plotted by chromosomal location. The left y-axis reflects the association (−log<sub>10</sub>P-value) with vaccine-induced measles-specific antibody titer, while the right y-axis reflects recombination rates and LD (r<sup>2</sup> color) of each plotted SNP with the most significant SNP (designated by a black diamond).</p><p id="__p5"><strong>B and D.</strong> Effect of top two <em>CD46</em> SNPs (<strong>B</strong>) and <em>IFI44L</em> SNPs (<strong>D</strong>) on measles-specific antibody response (effect in the combined cohort is presented as Turkey box-and-whisker plots). On the x-axis <strong>0</strong> designates subjects with homozygous major allele genotype, <strong>1</strong> designates heterozygous subjects and <strong>2</strong> designates subjects with homozygous minor allele genotype. On the y-axis neutralizing antibody titer is presented as the natural log of the PRMN mIU/mL value. The top (bottom) of the box indicates the 75th (25th) percentiles, respectively, while the bold line within the box indicates the median, and the whiskers indicate 1.5 times the IQR</p><p id="__p6"><sup>a</sup>n=2872, reduced to 2818 after excluding subjects with immune outcome data that failed QC</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p15">Demographic and immune characterization of the study population</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355369622656" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left">African American<sup>e</sup> (n=317)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Caucasian<sup>e</sup> (n=2555)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Total (n=2872)</th><th rowspan="1" colspan="1" valign="bottom" align="center">P- value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Neutralizing Ab titer</strong><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001<sup>f</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">N missing (failed assay QC)</td><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left">49</td><td rowspan="1" colspan="1" valign="top" align="left">54</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean (SD<sup>c</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">2065 (2821)</td><td rowspan="1" colspan="1" valign="top" align="left">1271 (1652)</td><td rowspan="1" colspan="1" valign="top" align="left">1359 (1835)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Median (IQR<sup>d</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">1180 (550, 2632)</td><td rowspan="1" colspan="1" valign="top" align="left">803 (383, 1607)</td><td rowspan="1" colspan="1" valign="top" align="left">845 (394, 1683)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>IFN</strong>γ <strong>ELISPOT response</strong><sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">0.427<sup>f</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">N missing (failed assay QC)</td><td rowspan="1" colspan="1" valign="top" align="left">26</td><td rowspan="1" colspan="1" valign="top" align="left">228</td><td rowspan="1" colspan="1" valign="top" align="left">254</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean (SD<sup>c</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">24.4 (26)</td><td rowspan="1" colspan="1" valign="top" align="left">25.2 (32)</td><td rowspan="1" colspan="1" valign="top" align="left">25.1 (31.4)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Median (IQR<sup>d</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">15 (7, 34.5)</td><td rowspan="1" colspan="1" valign="top" align="left">15 (6, 32.5)</td><td rowspan="1" colspan="1" valign="top" align="left">15 (6, 32.7)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Gender/sex</strong></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">0.082<sup>g</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Female</td><td rowspan="1" colspan="1" valign="top" align="left">73 (23%)</td><td rowspan="1" colspan="1" valign="top" align="left">707 (27.7%)</td><td rowspan="1" colspan="1" valign="top" align="left">780 (27.2%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Male</td><td rowspan="1" colspan="1" valign="top" align="left">244 (77%)</td><td rowspan="1" colspan="1" valign="top" align="left">1848 (72.3%)</td><td rowspan="1" colspan="1" valign="top" align="left">2092 (72.8%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Race (self-declared)</strong></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">NA<sup>h</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">White</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0%)</td><td rowspan="1" colspan="1" valign="top" align="left">2206 (86.3%)</td><td rowspan="1" colspan="1" valign="top" align="left">2206 (76.8%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Black or African American</td><td rowspan="1" colspan="1" valign="top" align="left">302 (95.3%)</td><td rowspan="1" colspan="1" valign="top" align="left">38 (1.49%)</td><td rowspan="1" colspan="1" valign="top" align="left">340 (11.8%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">American Indian/Alaska Native</td><td rowspan="1" colspan="1" valign="top" align="left">0 (0%)</td><td rowspan="1" colspan="1" valign="top" align="left">29 (1.14%)</td><td rowspan="1" colspan="1" valign="top" align="left">29 (1.01%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Asian/Hawaiian/Pacific Islander</td><td rowspan="1" colspan="1" valign="top" align="left">1 (0.315%)</td><td rowspan="1" colspan="1" valign="top" align="left">19 (0.744%)</td><td rowspan="1" colspan="1" valign="top" align="left">20 (0.696%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Multiple</td><td rowspan="1" colspan="1" valign="top" align="left">9 (2.84%)</td><td rowspan="1" colspan="1" valign="top" align="left">100 (3.91%)</td><td rowspan="1" colspan="1" valign="top" align="left">109 (3.8%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Other</td><td rowspan="1" colspan="1" valign="top" align="left">4 (1.26%)</td><td rowspan="1" colspan="1" valign="top" align="left">119 (4.66%)</td><td rowspan="1" colspan="1" valign="top" align="left">123 (4.28%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Unknown</td><td rowspan="1" colspan="1" valign="top" align="left">1 (0.315%)</td><td rowspan="1" colspan="1" valign="top" align="left">44 (1.72%)</td><td rowspan="1" colspan="1" valign="top" align="left">45 (1.57%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Ethnicity (self-declared)</strong></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001<sup>g</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Hispanic/Latino</td><td rowspan="1" colspan="1" valign="top" align="left">13 (4.1%)</td><td rowspan="1" colspan="1" valign="top" align="left">387 (15.1%)</td><td rowspan="1" colspan="1" valign="top" align="left">400 (13.9%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Not Hispanic/Latino</td><td rowspan="1" colspan="1" valign="top" align="left">285 (89.9%)</td><td rowspan="1" colspan="1" valign="top" align="left">2137 (83.6%)</td><td rowspan="1" colspan="1" valign="top" align="left">2422 (84.3%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Unknown</td><td rowspan="1" colspan="1" valign="top" align="left">19 (5.99%)</td><td rowspan="1" colspan="1" valign="top" align="left">31 (1.21%)</td><td rowspan="1" colspan="1" valign="top" align="left">50 (1.74%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Age at enrollment (years)</strong></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001<sup>f</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Number missing</td><td rowspan="1" colspan="1" valign="top" align="left">49</td><td rowspan="1" colspan="1" valign="top" align="left">196</td><td rowspan="1" colspan="1" valign="top" align="left">245</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean (SD<sup>c</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">24.5 (5.97)</td><td rowspan="1" colspan="1" valign="top" align="left">21.1 (6.06)</td><td rowspan="1" colspan="1" valign="top" align="left">21.4 (6.14)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Median (IQR<sup>d</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">24 (21, 28)</td><td rowspan="1" colspan="1" valign="top" align="left">22 (16, 25)</td><td rowspan="1" colspan="1" valign="top" align="left">22 (16, 25)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Age at last vaccination (years</strong>)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001<sup>f</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Number missing</td><td rowspan="1" colspan="1" valign="top" align="left">107</td><td rowspan="1" colspan="1" valign="top" align="left">577</td><td rowspan="1" colspan="1" valign="top" align="left">684</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean (SD<sup>c</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">19.8 (7.97)</td><td rowspan="1" colspan="1" valign="top" align="left">16.1 (8.38)</td><td rowspan="1" colspan="1" valign="top" align="left">16.4 (8.41)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Median (IQR<sup>d</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">20 (18, 24)</td><td rowspan="1" colspan="1" valign="top" align="left">18 (10, 22)</td><td rowspan="1" colspan="1" valign="top" align="left">18 (11, 23)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Time from last vaccination to enrollment (years)</strong></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">0.031<sup>f</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Number missing</td><td rowspan="1" colspan="1" valign="top" align="left">107</td><td rowspan="1" colspan="1" valign="top" align="left">577</td><td rowspan="1" colspan="1" valign="top" align="left">684</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean (SD<sup>c</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">3.7 (3.85)</td><td rowspan="1" colspan="1" valign="top" align="left">4.0 (3.43)</td><td rowspan="1" colspan="1" valign="top" align="left">4.0 (3.47)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Median (IQR<sup>d</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">2.6 (0.04, 5.5)</td><td rowspan="1" colspan="1" valign="top" align="left">3.5 (0.03, 6.4)</td><td rowspan="1" colspan="1" valign="top" align="left">3.4 (0.03, 6.4)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="5" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Cohort</strong></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001<sup>g</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Rochester</td><td rowspan="1" colspan="1" valign="top" align="left">40 (12.6%)</td><td rowspan="1" colspan="1" valign="top" align="left">942 (36.9%)</td><td rowspan="1" colspan="1" valign="top" align="left">982 (34.2%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">San Diego</td><td rowspan="1" colspan="1" valign="top" align="left">164 (51.7%)</td><td rowspan="1" colspan="1" valign="top" align="left">718 (28.1%)</td><td rowspan="1" colspan="1" valign="top" align="left">882 (30.7%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">US</td><td rowspan="1" colspan="1" valign="top" align="left">113 (35.6%)</td><td rowspan="1" colspan="1" valign="top" align="left">895 (35%)</td><td rowspan="1" colspan="1" valign="top" align="left">1008 (35.1%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140355369622656" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5433429/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Neutralizing antibody titer in mIU/mL, measured by the plaque reduction microneutralization assay (PRMN)</div><div id="TFN2"><sup>b</sup>IFNγ-positive spot-forming units (SFUs) per 2 x 10<sup>5</sup> cells (mean of measles virus-specific stimulated response, measured in triplicate, minus the mean unstimulated response, also measured in triplicate).</div><div id="TFN3"><sup>c</sup>Standard Deviation.</div><div id="TFN4"><sup>d</sup>IQR, inter-quartile range with 25% and 75% quartiles</div><div id="TFN5"><sup>e</sup>Genetically classified into African American or Caucasian ancestry group based on STRUCTURE (see Methods)</div><div id="TFN6"><sup>f</sup>Kruskal-Wallis Rank Test</div><div id="TFN7"><sup>g</sup>Fisher’s Exact Test</div><div id="TFN8"><sup>h</sup>Not applicable</div></div></div></div><div id="S16" class="sec"><p></p><h4 id="S16title" class="inline">CD46 region SNPs associated with variations in measles-specific antibody response after vaccination </h4><p id="P17" class="p p-first-last">The genetic association signal from the 1q32 region was linked to two blocks (97 kb and 8 kb) of 20 genetic variants in and around the <em>CD46</em> gene (seven intronic <em>CD46</em> SNPs, four SNPs in the uncharacterized <em>LOC101929385</em> [currently Gene ID 100128537, <em>C1orf132</em> chromosome 1 open reading frame 132], and nine intergenic SNPs, including the previously reported rs1318653 (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116903">Feenstra et al. 2014</a>), located between <em>CD46</em> and <em>CD34</em>), which were in high linkage disequilibrium (LD) (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplemental Fig. 2</a>). The most significant <em>CD46</em> SNPs, rs2724384 and rs2724374 (in high LD, r<sup>2</sup>=97), lie in intron 1, and near the boundary of intron 8 (of the reference sequence ENST00000358170, RefSeq <a id="__tag_666877306" class="tag_hotlink" href="/nuccore/NM_002389" ref="/nuccore/NM_002389"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_002389","term_id":"299522993","term_text":"NM_002389"}}</span>NM_002389</a>), respectively. The minor alleles (G) of rs2724384 and rs2724374 were significantly associated (<a href="/pmc/articles/PMC5433429/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) with an allele dose-related decrease in measles-specific neutralizing antibody titers after vaccination (46 – 47% decrease in antibody titer in homozygous minor allele genotype subjects compared to homozygous major allele genotype subjects, <a href="/pmc/articles/PMC5433429/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="/pmc/articles/PMC5433429/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig 1B</span></a>). Due to the high LD (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplemental Fig. 2</a>), all significant <em>CD46</em> SNPs displayed similar effects. Most of the1q32 associations remained genome-wide significant (p &lt; 5.0x10<sup>−08</sup>) in the subjects of Caucasian ancestry, where the most significant SNP was rs2724374 (p-value = 4.88x10<sup>−09</sup>, <a href="/pmc/articles/PMC5433429/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p24">Genome-wide significant associations of SNPs with measles-specific neutralizing antibody titers after MMR vaccination (<em>combined analysis, n=2872</em><sup>a</sup>)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355357736112" class="xtable"><table frame="hsides" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">SNP ID<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene/Location<sup>c</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Major allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Minor allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF<sup>d</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">P-value<sup>e</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Obs.0<sup>f</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Obs.1<sup>f</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Obs.2<sup>f</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Median (IQR) 0<sup>g</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Median (IQR) 1<sup>g</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Median (IQR) 2<sup>g</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1333973</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">1.41E-10</td><td rowspan="1" colspan="1" valign="top" align="center">1289</td><td rowspan="1" colspan="1" valign="top" align="center">1230</td><td rowspan="1" colspan="1" valign="top" align="center">299</td><td rowspan="1" colspan="1" valign="top" align="left">1010 (440, 1919)</td><td rowspan="1" colspan="1" valign="top" align="left">772 (363, 1491)</td><td rowspan="1" colspan="1" valign="top" align="left">625 (314, 1304)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273259</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, missense</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">2.87E-10</td><td rowspan="1" colspan="1" valign="top" align="center">1253</td><td rowspan="1" colspan="1" valign="top" align="center">1250</td><td rowspan="1" colspan="1" valign="top" align="center">315</td><td rowspan="1" colspan="1" valign="top" align="left">1003 (438, 1879)</td><td rowspan="1" colspan="1" valign="top" align="left">777 (364, 1520)</td><td rowspan="1" colspan="1" valign="top" align="left">631 (326, 1304)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2724384</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">2.64E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1680</td><td rowspan="1" colspan="1" valign="top" align="center">999</td><td rowspan="1" colspan="1" valign="top" align="center">139</td><td rowspan="1" colspan="1" valign="top" align="left">978 (429, 1882)</td><td rowspan="1" colspan="1" valign="top" align="left">710 (363, 1468)</td><td rowspan="1" colspan="1" valign="top" align="left">516 (272, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2761437</td><td rowspan="1" colspan="1" valign="top" align="left">.</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">3.14E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1671</td><td rowspan="1" colspan="1" valign="top" align="center">1003</td><td rowspan="1" colspan="1" valign="top" align="center">144</td><td rowspan="1" colspan="1" valign="top" align="left">981 (431, 1881)</td><td rowspan="1" colspan="1" valign="top" align="left">710 (359, 1475)</td><td rowspan="1" colspan="1" valign="top" align="left">517 (283, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2724374</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">3.16E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1666</td><td rowspan="1" colspan="1" valign="top" align="center">1006</td><td rowspan="1" colspan="1" valign="top" align="center">146</td><td rowspan="1" colspan="1" valign="top" align="left">978 (430, 1881)</td><td rowspan="1" colspan="1" valign="top" align="left">717 (361, 1470)</td><td rowspan="1" colspan="1" valign="top" align="left">526 (277, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2796265</td><td rowspan="1" colspan="1" valign="top" align="left">.</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">3.70E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1671</td><td rowspan="1" colspan="1" valign="top" align="center">1002</td><td rowspan="1" colspan="1" valign="top" align="center">145</td><td rowspan="1" colspan="1" valign="top" align="left">981 (431, 1881)</td><td rowspan="1" colspan="1" valign="top" align="left">710 (358, 1470)</td><td rowspan="1" colspan="1" valign="top" align="left">518 (286, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4650590</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, 3′UTR</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">3.72E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1266</td><td rowspan="1" colspan="1" valign="top" align="center">1226</td><td rowspan="1" colspan="1" valign="top" align="center">326</td><td rowspan="1" colspan="1" valign="top" align="left">1022 (450, 1932)</td><td rowspan="1" colspan="1" valign="top" align="left">755 (361, 1470)</td><td rowspan="1" colspan="1" valign="top" align="left">650 (329, 1327)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11118612</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC101929385</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">4.06E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1685</td><td rowspan="1" colspan="1" valign="top" align="center">991</td><td rowspan="1" colspan="1" valign="top" align="center">142</td><td rowspan="1" colspan="1" valign="top" align="left">978 (432, 1882)</td><td rowspan="1" colspan="1" valign="top" align="left">703 (347, 1435)</td><td rowspan="1" colspan="1" valign="top" align="left">545 (300, 972)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2761434</td><td rowspan="1" colspan="1" valign="top" align="left">.</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">4.97E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1671</td><td rowspan="1" colspan="1" valign="top" align="center">1002</td><td rowspan="1" colspan="1" valign="top" align="center">145</td><td rowspan="1" colspan="1" valign="top" align="left">981 (431, 1881)</td><td rowspan="1" colspan="1" valign="top" align="left">710 (358, 1463)</td><td rowspan="1" colspan="1" valign="top" align="left">518 (286, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4844392</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC101929385</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">5.26E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1688</td><td rowspan="1" colspan="1" valign="top" align="center">987</td><td rowspan="1" colspan="1" valign="top" align="center">143</td><td rowspan="1" colspan="1" valign="top" align="left">978 (432, 1882)</td><td rowspan="1" colspan="1" valign="top" align="left">707 (345, 1443)</td><td rowspan="1" colspan="1" valign="top" align="left">545 (301, 952)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4844619</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">5.40E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1685</td><td rowspan="1" colspan="1" valign="top" align="center">993</td><td rowspan="1" colspan="1" valign="top" align="center">140</td><td rowspan="1" colspan="1" valign="top" align="left">973 (430, 1879)</td><td rowspan="1" colspan="1" valign="top" align="left">712 (354, 1462)</td><td rowspan="1" colspan="1" valign="top" align="left">539 (291, 891)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2466572</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">6.33E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1668</td><td rowspan="1" colspan="1" valign="top" align="center">1005</td><td rowspan="1" colspan="1" valign="top" align="center">145</td><td rowspan="1" colspan="1" valign="top" align="left">978 (430, 1880)</td><td rowspan="1" colspan="1" valign="top" align="left">715 (361, 1473)</td><td rowspan="1" colspan="1" valign="top" align="left">533 (286, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2724360</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">6.51E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1664</td><td rowspan="1" colspan="1" valign="top" align="center">1009</td><td rowspan="1" colspan="1" valign="top" align="center">145</td><td rowspan="1" colspan="1" valign="top" align="left">980 (430, 1880)</td><td rowspan="1" colspan="1" valign="top" align="left">715 (360, 1473)</td><td rowspan="1" colspan="1" valign="top" align="left">533 (286, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6657476</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">6.51E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1687</td><td rowspan="1" colspan="1" valign="top" align="center">991</td><td rowspan="1" colspan="1" valign="top" align="center">140</td><td rowspan="1" colspan="1" valign="top" align="left">972 (430, 1878)</td><td rowspan="1" colspan="1" valign="top" align="left">710 (354, 1462)</td><td rowspan="1" colspan="1" valign="top" align="left">539 (291, 891)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs56075814</td><td rowspan="1" colspan="1" valign="top" align="left">.</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">9.15E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1682</td><td rowspan="1" colspan="1" valign="top" align="center">993</td><td rowspan="1" colspan="1" valign="top" align="center">143</td><td rowspan="1" colspan="1" valign="top" align="left">976 (431, 1879)</td><td rowspan="1" colspan="1" valign="top" align="left">708 (347, 1450)</td><td rowspan="1" colspan="1" valign="top" align="left">545 (301, 952)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6669384</td><td rowspan="1" colspan="1" valign="top" align="left">.</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.06E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1711</td><td rowspan="1" colspan="1" valign="top" align="center">980</td><td rowspan="1" colspan="1" valign="top" align="center">127</td><td rowspan="1" colspan="1" valign="top" align="left">970 (431, 1884)</td><td rowspan="1" colspan="1" valign="top" align="left">709 (349, 1420)</td><td rowspan="1" colspan="1" valign="top" align="left">545 (296, 992)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs55935450</td><td rowspan="1" colspan="1" valign="top" align="left">.</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.21E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1702</td><td rowspan="1" colspan="1" valign="top" align="center">981</td><td rowspan="1" colspan="1" valign="top" align="center">135</td><td rowspan="1" colspan="1" valign="top" align="left">978 (430, 1885)</td><td rowspan="1" colspan="1" valign="top" align="left">703 (348, 1417)</td><td rowspan="1" colspan="1" valign="top" align="left">552 (301, 1013)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6693207</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44, downstream</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">1.23E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1273</td><td rowspan="1" colspan="1" valign="top" align="center">1223</td><td rowspan="1" colspan="1" valign="top" align="center">322</td><td rowspan="1" colspan="1" valign="top" align="left">1023 (455, 1932)</td><td rowspan="1" colspan="1" valign="top" align="left">753 (354, 1479)</td><td rowspan="1" colspan="1" valign="top" align="left">650 (340, 1316)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273255</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">1.24E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1311</td><td rowspan="1" colspan="1" valign="top" align="center">1222</td><td rowspan="1" colspan="1" valign="top" align="center">285</td><td rowspan="1" colspan="1" valign="top" align="left">1005 (439, 1873)</td><td rowspan="1" colspan="1" valign="top" align="left">771 (362, 1504)</td><td rowspan="1" colspan="1" valign="top" align="left">637 (330, 1299)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11118668</td><td rowspan="1" colspan="1" valign="top" align="left">.</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.30E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1702</td><td rowspan="1" colspan="1" valign="top" align="center">981</td><td rowspan="1" colspan="1" valign="top" align="center">135</td><td rowspan="1" colspan="1" valign="top" align="left">978 (430, 1885)</td><td rowspan="1" colspan="1" valign="top" align="left">703 (348, 1417)</td><td rowspan="1" colspan="1" valign="top" align="left">552 (301, 1013)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs66532523</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC101929385</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.45E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1683</td><td rowspan="1" colspan="1" valign="top" align="center">992</td><td rowspan="1" colspan="1" valign="top" align="center">143</td><td rowspan="1" colspan="1" valign="top" align="left">974 (432, 1875)</td><td rowspan="1" colspan="1" valign="top" align="left">709 (347, 1455)</td><td rowspan="1" colspan="1" valign="top" align="left">545 (301, 952)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273261</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">2.03E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1223</td><td rowspan="1" colspan="1" valign="top" align="center">1251</td><td rowspan="1" colspan="1" valign="top" align="center">344</td><td rowspan="1" colspan="1" valign="top" align="left">997 (438, 1857)</td><td rowspan="1" colspan="1" valign="top" align="left">774 (363, 1522)</td><td rowspan="1" colspan="1" valign="top" align="left">662 (335, 1366)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4844620</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC101929385</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">2.18E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1762</td><td rowspan="1" colspan="1" valign="top" align="center">937</td><td rowspan="1" colspan="1" valign="top" align="center">119</td><td rowspan="1" colspan="1" valign="top" align="left">971 (430, 1876)</td><td rowspan="1" colspan="1" valign="top" align="left">701 (347, 1417)</td><td rowspan="1" colspan="1" valign="top" align="left">495 (280, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1318653</td><td rowspan="1" colspan="1" valign="top" align="left">.</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">2.94E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1708</td><td rowspan="1" colspan="1" valign="top" align="center">976</td><td rowspan="1" colspan="1" valign="top" align="center">134</td><td rowspan="1" colspan="1" valign="top" align="left">972 (430, 1882)</td><td rowspan="1" colspan="1" valign="top" align="left">705 (347, 1417)</td><td rowspan="1" colspan="1" valign="top" align="left">565 (303, 1023)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273256</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">3.15E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1166</td><td rowspan="1" colspan="1" valign="top" align="center">1277</td><td rowspan="1" colspan="1" valign="top" align="center">375</td><td rowspan="1" colspan="1" valign="top" align="left">1002 (445, 1898)</td><td rowspan="1" colspan="1" valign="top" align="left">774 (362, 1520)</td><td rowspan="1" colspan="1" valign="top" align="left">691 (349, 1398)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273244</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">3.30E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1162</td><td rowspan="1" colspan="1" valign="top" align="center">1280</td><td rowspan="1" colspan="1" valign="top" align="center">376</td><td rowspan="1" colspan="1" valign="top" align="left">1004 (445, 1898)</td><td rowspan="1" colspan="1" valign="top" align="left">772 (362, 1520)</td><td rowspan="1" colspan="1" valign="top" align="left">694 (349, 1417)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs61821293</td><td rowspan="1" colspan="1" valign="top" align="left">.</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">4.14E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1697</td><td rowspan="1" colspan="1" valign="top" align="center">984</td><td rowspan="1" colspan="1" valign="top" align="center">137</td><td rowspan="1" colspan="1" valign="top" align="left">970 (430, 1879)</td><td rowspan="1" colspan="1" valign="top" align="left">710 (347, 1426)</td><td rowspan="1" colspan="1" valign="top" align="left">561 (302, 992)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273238</td><td rowspan="1" colspan="1" valign="top" align="left">.</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">4.37E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1220</td><td rowspan="1" colspan="1" valign="top" align="center">1249</td><td rowspan="1" colspan="1" valign="top" align="center">349</td><td rowspan="1" colspan="1" valign="top" align="left">996 (438, 1845)</td><td rowspan="1" colspan="1" valign="top" align="left">779 (363, 1528)</td><td rowspan="1" colspan="1" valign="top" align="left">658 (338, 1359)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4844390</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">4.63E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1753</td><td rowspan="1" colspan="1" valign="top" align="center">940</td><td rowspan="1" colspan="1" valign="top" align="center">125</td><td rowspan="1" colspan="1" valign="top" align="left">974 (427, 1883)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (357, 1419)</td><td rowspan="1" colspan="1" valign="top" align="left">495 (274, 880)</td></tr></tbody></table></div><div id="largeobj_idm140355357736112" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5433429/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN9"><sup>a</sup>Reduced to 2818 after excluding subjects with immune outcome data that failed QC</div><div id="TFN10"><sup>b</sup>SNP identification number</div><div id="TFN11"><sup>c</sup>Gene/genetic region and SNP location relative to the gene</div><div id="TFN12"><sup>d</sup>Minor allele frequency</div><div id="TFN13"><sup>e</sup>P-values, as described in the Online Methods, Statistical Methods</div><div id="TFN14"><sup>f</sup>Number of subjects with homozygous major allele genotype (0), heterozygous (1) and homozygous minor allele genotype (2)</div><div id="TFN15"><sup>g</sup>Median neutralizing antibody titer (in miU/mL, with 25% and 75% inter-quartile range/IQR) for subjects with homozygous major allele genotype (0), heterozygous (1) and homozygous minor allele genotype (2)</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p32">Top significant associations of SNPs with measles-specific neutralizing antibody titers after MMR vaccination (<em>Caucasians, n=2555</em><sup>a</sup>)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355357761920" class="xtable"><table frame="hsides" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">SNP ID<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene/Location<sup>c</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Major allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Minor allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF<sup>d</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">P-value<sup>e</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Obs.0<sup>f</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Obs.1<sup>f</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Obs.2<sup>f</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Antibody titer Median (IQR) 0<sup>g</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Antibody titer Median (IQR) 1<sup>g</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Antibody titer Median (IQR) 2<sup>g</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2724374</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">4.88E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1462</td><td rowspan="1" colspan="1" valign="top" align="center">915</td><td rowspan="1" colspan="1" valign="top" align="center">129</td><td rowspan="1" colspan="1" valign="top" align="left">924 (414, 1799)</td><td rowspan="1" colspan="1" valign="top" align="left">703 (352, 1418)</td><td rowspan="1" colspan="1" valign="top" align="left">516 (274, 843)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2761437</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">6.98E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1467</td><td rowspan="1" colspan="1" valign="top" align="center">912</td><td rowspan="1" colspan="1" valign="top" align="center">127</td><td rowspan="1" colspan="1" valign="top" align="left">924 (414, 1797)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (349, 1422)</td><td rowspan="1" colspan="1" valign="top" align="left">515 (280, 825)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2796265</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">8.25E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1467</td><td rowspan="1" colspan="1" valign="top" align="center">911</td><td rowspan="1" colspan="1" valign="top" align="center">128</td><td rowspan="1" colspan="1" valign="top" align="left">924 (414, 1797)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (349, 1420)</td><td rowspan="1" colspan="1" valign="top" align="left">515 (283, 847)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2466572</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">9.49E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1464</td><td rowspan="1" colspan="1" valign="top" align="center">914</td><td rowspan="1" colspan="1" valign="top" align="center">128</td><td rowspan="1" colspan="1" valign="top" align="left">924 (414, 1798)</td><td rowspan="1" colspan="1" valign="top" align="left">703 (350, 1418)</td><td rowspan="1" colspan="1" valign="top" align="left">517 (283, 847)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4844619</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.05E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1468</td><td rowspan="1" colspan="1" valign="top" align="center">914</td><td rowspan="1" colspan="1" valign="top" align="center">124</td><td rowspan="1" colspan="1" valign="top" align="left">916 (414, 1794)</td><td rowspan="1" colspan="1" valign="top" align="left">705 (348, 1420)</td><td rowspan="1" colspan="1" valign="top" align="left">526 (283, 864)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6657476</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.06E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1468</td><td rowspan="1" colspan="1" valign="top" align="center">914</td><td rowspan="1" colspan="1" valign="top" align="center">124</td><td rowspan="1" colspan="1" valign="top" align="left">916 (414, 1794)</td><td rowspan="1" colspan="1" valign="top" align="left">705 (348, 1420)</td><td rowspan="1" colspan="1" valign="top" align="left">526 (283, 864)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2724360</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.18E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1461</td><td rowspan="1" colspan="1" valign="top" align="center">917</td><td rowspan="1" colspan="1" valign="top" align="center">128</td><td rowspan="1" colspan="1" valign="top" align="left">924 (414, 1798)</td><td rowspan="1" colspan="1" valign="top" align="left">703 (349, 1418)</td><td rowspan="1" colspan="1" valign="top" align="left">517 (283, 847)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2761434</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.19E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1467</td><td rowspan="1" colspan="1" valign="top" align="center">910</td><td rowspan="1" colspan="1" valign="top" align="center">129</td><td rowspan="1" colspan="1" valign="top" align="left">924 (414, 1797)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (349, 1418)</td><td rowspan="1" colspan="1" valign="top" align="left">516 (286, 858)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2724384</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.27E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1466</td><td rowspan="1" colspan="1" valign="top" align="center">914</td><td rowspan="1" colspan="1" valign="top" align="center">126</td><td rowspan="1" colspan="1" valign="top" align="left">924 (414, 1798)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (354, 1418)</td><td rowspan="1" colspan="1" valign="top" align="left">506 (277, 854)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4844620</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC101929385</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.64E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1490</td><td rowspan="1" colspan="1" valign="top" align="center">899</td><td rowspan="1" colspan="1" valign="top" align="center">117</td><td rowspan="1" colspan="1" valign="top" align="left">913 (416, 1772)</td><td rowspan="1" colspan="1" valign="top" align="left">701 (345, 1405)</td><td rowspan="1" colspan="1" valign="top" align="left">495 (286, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4844392</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC101929385</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.69E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1461</td><td rowspan="1" colspan="1" valign="top" align="center">917</td><td rowspan="1" colspan="1" valign="top" align="center">128</td><td rowspan="1" colspan="1" valign="top" align="left">919 (416, 1793)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (344, 1418)</td><td rowspan="1" colspan="1" valign="top" align="left">539 (291, 883)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11118612</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC101929385</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.98E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1461</td><td rowspan="1" colspan="1" valign="top" align="center">918</td><td rowspan="1" colspan="1" valign="top" align="center">127</td><td rowspan="1" colspan="1" valign="top" align="left">918 (416, 1793)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (346, 1417)</td><td rowspan="1" colspan="1" valign="top" align="left">533 (290, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1333973</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">2.10E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1122</td><td rowspan="1" colspan="1" valign="top" align="center">1106</td><td rowspan="1" colspan="1" valign="top" align="center">278</td><td rowspan="1" colspan="1" valign="top" align="left">942 (422, 1768)</td><td rowspan="1" colspan="1" valign="top" align="left">759 (361, 1448)</td><td rowspan="1" colspan="1" valign="top" align="left">616 (306, 1283)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs66532523</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC101929385</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">2.27E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1461</td><td rowspan="1" colspan="1" valign="top" align="center">917</td><td rowspan="1" colspan="1" valign="top" align="center">128</td><td rowspan="1" colspan="1" valign="top" align="left">918 (416, 1793)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (345, 1417)</td><td rowspan="1" colspan="1" valign="top" align="left">539 (291, 883)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs56075814</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">2.34E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1460</td><td rowspan="1" colspan="1" valign="top" align="center">918</td><td rowspan="1" colspan="1" valign="top" align="center">128</td><td rowspan="1" colspan="1" valign="top" align="left">917 (416, 1794)</td><td rowspan="1" colspan="1" valign="top" align="left">703 (346, 1417)</td><td rowspan="1" colspan="1" valign="top" align="left">539 (291, 883)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273259</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, missense</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">2.62E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1118</td><td rowspan="1" colspan="1" valign="top" align="center">1104</td><td rowspan="1" colspan="1" valign="top" align="center">284</td><td rowspan="1" colspan="1" valign="top" align="left">942 (424, 1765)</td><td rowspan="1" colspan="1" valign="top" align="left">757 (356, 1455)</td><td rowspan="1" colspan="1" valign="top" align="left">628 (319, 1284)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273261</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">3.41E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1110</td><td rowspan="1" colspan="1" valign="top" align="center">1106</td><td rowspan="1" colspan="1" valign="top" align="center">290</td><td rowspan="1" colspan="1" valign="top" align="left">950 (428, 1768)</td><td rowspan="1" colspan="1" valign="top" align="left">754 (355, 1441)</td><td rowspan="1" colspan="1" valign="top" align="left">631 (323, 1287)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4844390</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CD46, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">3.48E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1478</td><td rowspan="1" colspan="1" valign="top" align="center">907</td><td rowspan="1" colspan="1" valign="top" align="center">121</td><td rowspan="1" colspan="1" valign="top" align="left">919 (414, 1795)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (352, 1417)</td><td rowspan="1" colspan="1" valign="top" align="left">495 (274, 858)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6669384</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">3.75E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1484</td><td rowspan="1" colspan="1" valign="top" align="center">909</td><td rowspan="1" colspan="1" valign="top" align="center">113</td><td rowspan="1" colspan="1" valign="top" align="left">916 (414, 1797)</td><td rowspan="1" colspan="1" valign="top" align="left">703 (349, 1410)</td><td rowspan="1" colspan="1" valign="top" align="left">545 (293, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273256</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">5.10E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1052</td><td rowspan="1" colspan="1" valign="top" align="center">1134</td><td rowspan="1" colspan="1" valign="top" align="center">320</td><td rowspan="1" colspan="1" valign="top" align="left">958 (435, 1806)</td><td rowspan="1" colspan="1" valign="top" align="left">743 (347, 1448)</td><td rowspan="1" colspan="1" valign="top" align="left">646 (329, 1301)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273244</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">6.35E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1050</td><td rowspan="1" colspan="1" valign="top" align="center">1136</td><td rowspan="1" colspan="1" valign="top" align="center">320</td><td rowspan="1" colspan="1" valign="top" align="left">964 (434, 1807)</td><td rowspan="1" colspan="1" valign="top" align="left">740 (348, 1438)</td><td rowspan="1" colspan="1" valign="top" align="left">646(329, 1311)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4650590</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, 3′UTR</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">6.58E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1065</td><td rowspan="1" colspan="1" valign="top" align="center">1131</td><td rowspan="1" colspan="1" valign="top" align="center">310</td><td rowspan="1" colspan="1" valign="top" align="left">945 (432, 1808)</td><td rowspan="1" colspan="1" valign="top" align="left">739 (352, 1440)</td><td rowspan="1" colspan="1" valign="top" align="left">646 (329, 1316)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273238</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">6.91E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1108</td><td rowspan="1" colspan="1" valign="top" align="center">1105</td><td rowspan="1" colspan="1" valign="top" align="center">293</td><td rowspan="1" colspan="1" valign="top" align="left">950 (427, 1766)</td><td rowspan="1" colspan="1" valign="top" align="left">758 (355, 1436)</td><td rowspan="1" colspan="1" valign="top" align="left">630 (327, 1299)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs55935450</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">6.93E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1477</td><td rowspan="1" colspan="1" valign="top" align="center">910</td><td rowspan="1" colspan="1" valign="top" align="center">119</td><td rowspan="1" colspan="1" valign="top" align="left">916 (414, 1793)</td><td rowspan="1" colspan="1" valign="top" align="left">700 (347, 1407)</td><td rowspan="1" colspan="1" valign="top" align="left">545 (296, 898)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11118668</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">7.38E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1477</td><td rowspan="1" colspan="1" valign="top" align="center">910</td><td rowspan="1" colspan="1" valign="top" align="center">119</td><td rowspan="1" colspan="1" valign="top" align="left">916 (414, 1793)</td><td rowspan="1" colspan="1" valign="top" align="left">700 (347, 1407)</td><td rowspan="1" colspan="1" valign="top" align="left">545 (296, 898)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs61821293</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">8.51E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1475</td><td rowspan="1" colspan="1" valign="top" align="center">910</td><td rowspan="1" colspan="1" valign="top" align="center">121</td><td rowspan="1" colspan="1" valign="top" align="left">911 (414, 1790)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (345, 1415)</td><td rowspan="1" colspan="1" valign="top" align="left">561 (299, 884)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6693207</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, downstream</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">9.28E-08</td><td rowspan="1" colspan="1" valign="top" align="center">1069</td><td rowspan="1" colspan="1" valign="top" align="center">1131</td><td rowspan="1" colspan="1" valign="top" align="center">306</td><td rowspan="1" colspan="1" valign="top" align="left">956 (434, 1807)</td><td rowspan="1" colspan="1" valign="top" align="left">740 (346, 1447)</td><td rowspan="1" colspan="1" valign="top" align="left">643 (337, 1306)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs273255</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IFI44L, intron</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">1.01E-07</td><td rowspan="1" colspan="1" valign="top" align="center">1149</td><td rowspan="1" colspan="1" valign="top" align="center">1095</td><td rowspan="1" colspan="1" valign="top" align="center">262</td><td rowspan="1" colspan="1" valign="top" align="left">945 (426, 1769)</td><td rowspan="1" colspan="1" valign="top" align="left">735 (348, 1417)</td><td rowspan="1" colspan="1" valign="top" align="left">634 (328, 1287)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1318653</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.04E-07</td><td rowspan="1" colspan="1" valign="top" align="center">1480</td><td rowspan="1" colspan="1" valign="top" align="center">907</td><td rowspan="1" colspan="1" valign="top" align="center">119</td><td rowspan="1" colspan="1" valign="top" align="left">909 (414, 1776)</td><td rowspan="1" colspan="1" valign="top" align="left">702 (345, 1414)</td><td rowspan="1" colspan="1" valign="top" align="left">561 (301, 898)</td></tr></tbody></table></div><div id="largeobj_idm140355357761920" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5433429/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN16"><sup>a</sup>Reduced to 2506 after excluding subjects with immune outcome data that failed QC</div><div id="TFN17"><sup>b</sup>SNP identification number</div><div id="TFN18"><sup>c</sup>Gene/genetic region and SNP location relative to the gene</div><div id="TFN19"><sup>d</sup>Minor allele frequency</div><div id="TFN20"><sup>e</sup>P-values, as described in the Online Methods, Statistical Methods</div><div id="TFN21"><sup>f</sup>Number of subjects with homozygous major allele genotype (0), heterozygous (1) and homozygous minor allele genotype (2)</div><div id="TFN22"><sup>g</sup>Median neutralizing antibody titer (in miU/mL, with 25% and 75% inter-quartile range/IQR) for subjects with homozygous major allele genotype (0), heterozygous (1) and homozygous minor allele genotype (2)</div></div></div></div><div id="S17" class="sec sec-last"><p></p><h4 id="S17title" class="inline">IFI44L region SNPs associated with variations in measles-specific antibody response after vaccination </h4><p id="P18" class="p p-first">The 1p31.1 region signal was linked to a 21-kb block of 8 <em>IFI44L</em> SNPs and one intergenic SNP, in high LD (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplemental Fig. 2</a>). The two most significant <em>IFI44L</em> SNPs, rs1333973 and rs273259 (<a href="/pmc/articles/PMC5433429/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) were located in <em>IFI44L</em> intron 2 (boundary) and <em>IFI44L</em> exon 2, respectively. The missense SNP rs273259 (His73Arg, Ensembl transcript ENST00000370751), as well as the other significant SNPs, demonstrated an allele dose-related decrease in measles-specific neutralizing antibody titers (<a href="/pmc/articles/PMC5433429/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="/pmc/articles/PMC5433429/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig 1D</span></a>). Three of the nine SNPs remained genome-wide significant (p &lt; 5.0x10<sup>−08</sup>) in the subjects of Caucasian ancestry (<a href="/pmc/articles/PMC5433429/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><p id="P19" class="p p-last">To determine if there were multiple SNPs in each of the <em>CD46</em> and <em>IFI44L</em> regions associated with the neutralizing antibody, with the effects of the SNPs adjusted for each other, we used elastic net to select SNPs. An advantage of elastic net is that it can select highly correlated SNPs, although it cannot give p- values for selected SNPs. Hence, after selection, we evaluated the selected SNPs in linear regression models. This resulted in the selection of two SNPs in the <em>CD46</em> region and three SNPs in the <em>IFI44L</em> region, based on the Caucasian subjects. For the <em>CD46</em> region, we found rs2724374 to be the most significant and the primary driving SNP (p=4.3x10<sup>−8</sup>), with rs11806810 having much less association (p=0.02) when the effects of the SNPs were adjusted for each other. For the <em>IFI44L</em> region, the SNPs rs12026737, rs1333973, and rs273259 were selected. However, rs273259 and rs1333973 were highly correlated (r<sup>2</sup> = .99), making it difficult to disentangle their effects and fit both in a regression model. When choosing rs1333973 (top significant SNP) over rs273259, we found rs1333973 (p=2x10<sup>−8</sup>) and rs12026737 (p=6.0x10<sup>−4</sup>) to have strong statistical associations, with effects adjusted for each other. Furthermore, the <em>CD46</em> and <em>IFI44L</em> regions have associations that are independent of each other (since when the aforementioned SNPs were modeled together, the results were not significantly different from what was obtained when the regions were modeled separately).</p></div></div><div id="S18" class="sec"><h3 id="S18title">Genome-wide analysis results with measles vaccine cellular immunity</h3><p id="P20" class="p p-first-last">The GWAS analyses did not reveal significant SNP associations with cellular immunity after vaccination, as measured by MV-specific IFNγ ELISPOT (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 1</a>, <a href="#SD5" rid="SD5" class=" supplementary-material">Supplemental Fig. 3</a>). Analyses of all chromosome 1 SNPs with MV-specific secreted cytokines in 625 Caucasian subjects (for whom we had available cytokine data) demonstrate suggestive associations between <em>CD46</em> SNPs (including rs2724374 and rs2724384) and the secretion of IFNα; however, an allele-dose-dependency of IFNα secretion was not noted. (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table 2</a>).</p></div><div id="S19" class="sec"><h3 id="S19title">Analysis of differential usage of exons using mRNA-Seq data</h3><p id="P21" class="p p-first-last">In the CD46 analyses, we observed a highly significant per-exon estimate (q=2.96E<sup>−07</sup>) with lower exon B (in the <em>CD46</em> STP region) expression (exon skipping) in subjects with rs2724374 minor G allele (<a href="/pmc/articles/PMC5433429/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>). These results were further confirmed by RT-PCR analysis of common <em>CD46</em> isoforms in PBMCs. In this analysis, the predominant “lower band” CD46 isoform phenotype (C2, associated with exon B skipping) was clearly more pronounced in the homozygous minor G allele genotype subjects (<a href="/pmc/articles/PMC5433429/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>). We also observed differential exon usage based on <em>IFI44L</em> rs1333973/rs273259 genotypes (<a href="/pmc/articles/PMC5433429/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>)</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5433429/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355360684256" class="figure"><a href="/pmc/articles/PMC5433429/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5433429_nihms859705f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms859705f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5433429/bin/nihms859705f2.jpg"></a></div><div id="largeobj_idm140355360684256" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5433429/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5433429/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Differential exon/isoform <em>CD46</em> and <em>IFI44L</em> usage</strong><p id="__p7"><strong>A.</strong> Estimated exon usage in the CD46 gene comparing individuals with at least one minor allele (G for rs2724374, i.e., all the heterozygous subjects plus the one homozygous minor allele subject combined, total n=9) (blue) to individuals that are homozygous major (T for rs2724374) allele, n=19 (red). Even with a relatively small number of subjects possessing the minor allele genotype, differential exon usage with a highly significant p-value was observed for the STP exon B (genomic ID 207941124-207941168) (p=2.96E<sup>−07</sup>).</p><p id="__p8"><strong>B.</strong> RT-PCR analysis of common <em>CD46</em> isoforms was performed in PBMCs of rs2724374 homozygous major allele genotype individuals (n=10) compared to homozygous minor allele genotype individuals (n=10) and heterozygous individuals (n=10). The presented figure is representative of the patterns observed in all 30 subjects (10 subjects per genotype group) with the experiment replicated twice.</p><p id="__p9"><strong>C.</strong> Estimated exon usage in <em>IFI44L</em> rs1333973/rs273259 homozygous minor allele genotype subjects (A for rs1333973, n=5 [blue]) vs. homozygous major allele subjects (T for rs1333973, n=11 [red]). The results demonstrate significant per-exon estimates for several <em>IFI44L</em> exons, the most significant being exon 2 (genomic ID 79093591-79094078) (q=2.37E<sup>−150</sup>).</p></div></div></div></div><div id="S20" class="sec sec-last"><h3 id="S20title">Simulation of the structural differences between CD46 isoforms (with or without the STP exon B)</h3><p id="P22" class="p p-first-last">We have generated structural models of the predominant CD46 isoforms (extracellular portion): isoforms with the STP exons B and C (i.e., BC1 and BC2) and those skipping exon B (i.e., C1 and C2) (see <a href="/pmc/articles/PMC5433429/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>, <a href="/pmc/articles/PMC5433429/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>, <a href="#SD6" rid="SD6" class=" supplementary-material">Supplemental Fig. 4</a>). The longest isoform, ABC1, generates a high-quality homology model (<a href="#SD6" rid="SD6" class=" supplementary-material">Supplemental Fig. 4</a>). Interestingly, the protein sequences encoded by exons A (exon 7) and B (exon 8) have their N- and C-terminal ends close to each other. The amino acids encoded by exon A (exon 7) make a short beta strand within the STP domain. Deletion of these residues by exon skipping removes this strand from the beta-sheet, but the architecture of the domain is not significantly altered. Exon B (8) encodes amino acids making up two additional strands. In the isoforms C1 and C2, only two strands encoded by exons 9 and 10 remain. Computing the dynamics of each isoform’s atomic model, smaller STP domain isoforms exhibited greater flexibility and less collective motion (i.e., in the C1/C2 isoforms when compared to BC1/BC2 of the common isoforms). Monitoring the domain-domain angles about the SCR4-STP hinge as each structure is deformed about its low-frequency normal modes, the shorter isoform/isoforms exhibit a greater range of flexibility (<a href="#SD6" rid="SD6" class=" supplementary-material">Supplemental Fig. 4</a>). As illustrated in <a href="/pmc/articles/PMC5433429/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>, based on data from our molecular models, we propose that the differential isoform flexibility may influence the rate of formation of the CD46-MV-hemagglutinin (H) encounter complex, and/or influence the propensity for MV-H to undergo the conformational change necessary to trigger MV fusion protein.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC5433429/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355357871776" class="figure"><a href="/pmc/articles/PMC5433429/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5433429_nihms859705f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms859705f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5433429/bin/nihms859705f3.jpg"></a></div><div id="largeobj_idm140355357871776" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5433429/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC5433429/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Structure of CD46</strong><p id="__p10">The extracellular portion of CD46 consists of four N-glycosylated conserved short consensus repeats SCR1-4 (SCR1 and SCR2 containing binding sites for MV); a STP region that is O-glycosylated (encoded by exons 7, 8 and 9, designated as A, B and C); and a region of unknown function (U), followed by a hydrophobic transmembrane segment (H), basic amino acid anchor (A), and a cytoplasmic tail (CYT1 of 16 amino acids if exon 13 is present, or CYT2 of 23 amino acids if exon 13 is alternatively spliced).). Of the 14 known CD46 isoforms resulting from alternative splicing, four are commonly found in most human tissues and are designated based on the present STP exon/exons and the cytoplasmic tail: BC1 and BC2 (with B and C exons/domains in the STP and with either CYT1 or CYT2), and C1 and C2 (with C exon/domain in the STP and with either CYT1 or CYT2). The effect of <em>CD46</em> rs2724374 on CD46 isoform prevalence (exon B presence or skipping), interaction between CD46 and MV, and immune response following measles vaccination is also summarized.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC5433429/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355369625552" class="figure"><a href="/pmc/articles/PMC5433429/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5433429_nihms859705f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms859705f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5433429/bin/nihms859705f4.jpg"></a></div><div id="largeobj_idm140355369625552" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5433429/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC5433429/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Fig. 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Flexibility of CD46 isoforms and impact on MV-CD46 interactions and MV fusion</strong><p id="__p11"><strong>A.</strong> CD46 is a cell surface receptor affected by isoform differences. Due in part to its elongated structure, the extracellular domains will naturally exhibit flexibility within the physiologic environment (represented as rotational blurring of the reference isoform ABC1/ABC2). SCR domain residues are colored in yellow. The modeled segment of CD46 includes the extracellular domains and is shown in cartoon representation.</p><p id="__p12"><strong>B.</strong> Molecular modeling of the extracellular portion of BC1 (BC2) and C1 (C2) isoforms (the four most common isoforms) through SCR4, followed by generation of mechanics-based models (ANM; see Methods), indicates a difference in the intrinsic flexibility between the isoforms (see <a href="#SD6" rid="SD6" class=" supplementary-material">Supplemental Fig. 4</a> for details). The amino acids encoded by exons 8–10 are indicated by different colors. We show representative motion from our ANM models for the BC1 and C1 isoforms by showing multiple conformational states superimposed.</p><p id="__p13"><strong>C.</strong> CD46 on the cell surface interacts with MV-H on the virion surface. Two <em>CD46</em> molecules interacting with a MV-H dimer are depicted, as described previously. (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116993">Santiago et al. 2010</a>) Using ANM, we quantified the intrinsic flexibility of the MV-H dimer. The dominant motion is an anti-correlated twisting or ratcheting of the monomers with respect to one another. Projection the effect of this motion onto the bound CD46 molecules is represented by black arrows on each residue. As the C-terminus of CD46 is anchored in the cell membrane, activation of this motion would require flexibility of the CD46 molecule - flexibility that may differ by isoform (panel B). Presented is a second view of the motion rotated 90°, omitting the cartoon representation for clarity – only the motion-vectors are shown to emphasize the twisting of the MV-H dimer and its effect on CD46.</p><p id="__p14"><strong>D.</strong> Proposed molecular mechanism: MV-H and the fusion protein MV-F are normally associated with one another on the virus surface. Proteins are represented by smoothed molecular surfaces; see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods</a>. The encounter complex between CD46 and MV-H leads to MV-F disassociation. The disassociation is influenced by the MV-H conformational change and motion in the context of the CD46-H dimer complex.(<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116893">Navaratnarajah et al. 2011</a>) The free MV-F undergoes a substantial conformational change, the molecular details of which are not fully resolved, leading to bridging between the virus and target cell. We believe that the degree of flexibility exhibited by CD46 may influence 1) the ease with which the complex may undergo conformational changes (motion in the context of MV-H-CD46 complex) leading to MV-F triggering and fusion, and 2) the rate of encounter complex formation with MV-H.</p></div></div></div></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Discussion</h2><p id="P23" class="p p-first">This work is the first GWAS to reveal significant associations of two distinct chromosome 1 regions (<em>CD46</em> and <em>IFI44L</em>) with variations in neutralizing antibody response after measles vaccination.</p><p id="P24">The important implications of our findings are supported by the genome-wide identification of genetic variants in the same two loci (in an independent state-of-the-art GWAS study) as risk variants for the development of adverse events/febrile seizures following MMR vaccination, but not for MMR-unrelated febrile seizures (suggesting the influence of these loci on measles vaccine virus entry/propagation). (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116945">Feenstra et al. 2014</a>)</p><p id="P25">Most of the significant SNP-association signals with measles-specific antibody response were from a region in and around the <em>CD46</em> gene on 1q32. The encoded glycoprotein CD46 (structure is summarized in <a href="/pmc/articles/PMC5433429/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>) is ubiquitously expressed and serves as a regulator of complement activation (protecting the cells from complement and antibody-mediated lysis). CD46 also serves as a cellular receptor for measles virus attenuated (vaccine) strains, as well as for other pathogens (group B and D adenoviruses, human herpesvirus 6, bovine viral diarrhea virus, pathogenic <em>Neisseria</em> and <em>Streptococcus pyogenes</em>). (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116895">Cattaneo 2004</a>) Four CD46 isoforms, resulting from alternative splicing, are commonly found in most human tissues and are designated based on the present STP exon/exons and the cytoplasmic tail: BC1 and BC2 (with B and C exons/domains in the STP and with either CYT1 or CYT2), and C1 and C2 (with C exon/domain in the STP and with either CYT1 or CYT2). (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116920">Liszewski et al. 1994</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608117002">Post et al. 1991</a>; <a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116960">Russell et al. 1992</a>)(<a href="/pmc/articles/PMC5433429/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>)</p><p id="P26">Our significant GWAS findings in the <em>CD46</em> region consisted only of non-coding SNPs in high LD; the most significant were rs2724384 in <em>CD46</em> intron 1, and rs2724374 in <em>CD46</em> intron 8. The previously observed (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116885">Feenstra et al. 2014</a>) genetic association of intergenic rs1318653 with MMR-related febrile seizures is in the same region and was also significant in our genome-wide association study with measles-specific neutralizing antibody titers (p-value = 2.94x10<sup>−08</sup>, <a href="/pmc/articles/PMC5433429/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), although this association exhibited a slightly weaker signal in the subset analysis of subjects of Caucasian ancestry (p-value = 1.04x10<sup>−07</sup>, <a href="/pmc/articles/PMC5433429/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Feenstra <em>et al.</em> also reported the major <em>CD46</em> rs2724384 allele A as a risk allele for MMR-related febrile seizures (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116940">Feenstra et al. 2014</a>), which relates to higher measles vaccine-induced antibody titers in our GWAS and in three other candidate gene studies. (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116870">Clifford et al. 2012</a>; <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116914">Dhiman et al. 2007</a>; <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116922">Haralambieva et al. 2015</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116966">Ovsyannikova et al. 2011a</a>) Furthermore, <em>CD46</em> rs2724384 has been associated with overall <em>CD46</em> gene expression and isoform abundance in lymphoblastoid cell lines (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116950">Lappalainen et al. 2013</a>), and with variations in measles-specific IL-6, TNFα and IFNα secretion after <em>in vitro</em> viral stimulation of human PBMCs.(<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116873">Ovsyannikova et al. 2011a</a>) In this study, both of our top <em>CD46</em> GWAS hits (rs2724384 and rs2724374) exhibited suggestive associations with IFNα (see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table 2</a>).</p><p id="P27">The second <em>CD46</em> genetic variant with plausible functional consequences is rs2724374 (located in intron 8, near the intron-exon boundary with exon 8/STP B), which is the most significant SNP in our analysis of subjects of Caucasian ancestry and is identified by elastic net modeling as being predictive of neutralizing antibody response. <em>CD46</em> rs2724374 has been reported as a genetic variant, which is highly correlated with the splicing/skipping of exon B (i.e., a sQTL/splicing quantitative trait locus) in a study assessing the splicing patterns of 250 exons and their associations with DNA polymorphisms.(<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116932">Hull et al. 2007</a>) Similarly, a study assessing sQTLs from RNA-Seq data reported <em>CD46</em> rs2724374 as one of the most significant sQTLs (p=3.55x10<sup>−11</sup>) associated with alternative splicing (i.e., the skipping of <em>CD46</em> exon B) (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116892">Zhao et al. 2013</a>), a finding validated by our differential exon usage analysis of mRNA-Seq data and RT-PCR analysis of CD46 isoforms in PBMCs. Thus, the minor allele G of <em>CD46</em> rs2724374 is significantly associated (in a dose-response dependent manner) with lower measles-specific neutralizing antibody titer after vaccination (approximately 45% reduction in antibody titer), and with the skipping of exon B to preferentially yielding CD46 isoforms with a shorter (and less O-glycosylated) STP region (i.e., preferentially yielding C1/C2 vs. BC1/BC2 isoforms). Although some tissues preferentially express specific CD46 isoforms (sperm, brain, salivary gland, kidney, placenta), the abundance ratio of CD46 isoforms in most human tissues (e.g., PBMCs) is considered to be constant for each individual. (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116891">Liszewski et al. 1994</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116890">Post et al. 1991</a>; <a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116931">Russell et al. 1992</a>) These CD46 genetic and phenotypic inter-individual variations and their implications for host-pathogen interactions and vaccine/pathogen-induced immunity still remain unknown.</p><p id="P28">Since innate and adaptive immune responses rely on recognition, virus entry and propagation into susceptible cells, it is tempting to speculate about the mechanisms by which CD46 isoform usage can affect MV binding/fusion and propagation at the site of injection and the associated lymphoid tissue during measles vaccination and immune response priming. It is generally accepted that all CD46 isoforms can serve as MV receptors (primarily for attenuated MV strains) and confer susceptibility to infection (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608117001">Manchester et al. 1994</a>); however, several studies report differences in MV binding and fusion in <em>CD46</em> BC1/BC2 vs. C1/C2 isoforms, where longer BC isoforms support superior virus binding and shorter C isoforms support superior fusion competence. (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116976">Buchholz et al. 1996a</a>; <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116956">Buchholz et al. 1996b</a>; <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116984">Iwata et al. 1994</a>) It is likely that the complex dynamics of interactions between MV H dimers/tetramers, F trimers and cross-linked CD46 molecules at the virus-cell surface interface (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116913">Navaratnarajah et al. 2011</a>; <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116874">Persson et al. 2010</a>) are dependent in part on the varying flexibility conferred by different CD46 isoforms. Our molecular modeling (<a href="/pmc/articles/PMC5433429/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a> and <a href="#SD6" rid="SD6" class=" supplementary-material">Supplemental Fig. 4</a>) provides evidence for differences in the flexibilities of common CD46 isoforms, including the STP exon B (BC1/BC2 exhibiting decreased flexibility) and those excluding exon B (C1/C2 exhibiting increased flexibility). We hypothesize that the increased flexibility for the C1/C2 isoforms relative to BC1/BC2 isoforms allows increased motion between the components of the CD46-H dimer complex (needed for F triggering, as proposed in the model by Navaratnarajah <em>et al</em>., (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116875">Navaratnarajah et al. 2011</a>; <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116987">Persson et al. 2010</a>)) and increased fusogenicity. It is also possible that the increased flexibility of the C1/C2 isoforms (relative to BC1/BC2) may increase the encounter rate between CD46 and H protein. Thus, our proposed mechanistic hypothesis (informed by molecular modeling) is in concert with--and adds to--the prior literature, and suggests potential molecular mechanisms linking the top GWAS <em>CD46</em> hits to differences in splicing/CD46 isoform flexibility with potential effects on MV response.</p><p id="P29">The CYT1 and CYT2-containing CD46 isoforms (generated by alternative splicing of exon 13) are reported to differentially co-stimulate T helper 1 effector cells, affect T helper 1 switching to IL-10 producing T regulatory cells, and modulate cellular immunity, inflammation, B cell responses and signal transduction. (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116917">Fuchs et al. 2009</a>; <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116928">Marie et al. 2002</a>; <a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116888">Wang et al. 2000</a>) Although the differential usage of CYT1 vs. CYT2-containing isoforms cannot be inferred from our data, the involvement of such mechanisms in regulating vaccine-induced immunity cannot be excluded. Interestingly, we did not observe significant associations between polymorphisms in the other known MV receptor genes (SLAM and nectin-4) and vaccine-induced humoral immunity. This finding suggests the preferential usage of CD46 (over other receptors) by MV vaccine strains and/or insufficient expression of SLAM and Nectin-4 receptors at the site of injection and immune response priming during vaccination.</p><p id="P30">Even more interesting is the association of the 1p31.1 <em>IFI44L</em> genetic locus with measles-specific antibody titers, since the function of the encoded protein is largely unknown. Among the top significant SNPs in this region (grouped in a 21-kb LD block) are the intronic rs1333973 and the coding missense rs273259 (<a href="/pmc/articles/PMC5433429/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC5433429/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>); the latter has been reported to be a genetic variant significantly associated with febrile seizures after MMR vaccination. (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116951">Feenstra et al. 2014</a>) The rs273259 risk allele (A) for MMR-related febrile seizures was associated with increased measles-specific antibody titer in our study (while the minor allele G was associated with decreased antibody titer). In addition, rs273259 allele A was previously found to correlate with both the reduced expression of <em>IFI44L</em> exon 2 and differences in <em>IFI44L</em> isoform/isoforms abundance (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116959">Lappalainen et al. 2013</a>), a finding also confirmed by our mRNA-Seq differential exon usage analysis (<a href="/pmc/articles/PMC5433429/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>). In their study, Feenstra <em>et al.</em> did not observe direct and/or differential antiviral activity of <em>IFI44L</em> rs273259 allelic variant proteins against MV in human fibroblasts lacking STAT1.(<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116978">Feenstra et al. 2014</a>) It is likely that <em>IFI44L</em> requires a specific microenvironment (e.g., partners, signaling events, cell/tissue-specific milieu, etc.) to exert its antiviral action or its function is associated with specific isoforms, as suggested by our data and other studies (i.e., SNP rs1333973 has been reported as a significant sQTL for <em>IFI44L</em> (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116908">Coulombe-Huntington et al. 2009</a>; <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116878">Fraser and Xie 2009</a>; <a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116916">Zhao et al. 2013</a>)).</p><p id="P31">The proteins encoded by <em>IFI44L</em> and its partner <em>IFI44</em> are both stimulated by interferon type I and thus are likely to be involved in innate immunity. A screen of more than 380 interferon stimulated genes/ISGs for <em>in vitro</em> antiviral activity against different viruses identified <em>IFI44L</em> as an important antiviral effector of type I interferon response against hepatitis C virus. (<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116989">Schoggins et al. 2011</a>) Importantly, a study assessing susceptibility to viral myocarditis in mice (by Coxsackievirus B3) noted that genetic variants in the H28 (<em>IFI44L</em>) locus are likely involved in infection susceptibility.(<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116877">Wiltshire et al. 2011</a>) These data from the literature suggest that <em>IFI44L</em> is an important effector of innate antiviral immunity with a plausible role in immune response priming and protection against disease.</p><p id="P32">The strengths of our study include the use of a relatively large combined sample of three well-characterized cohorts (from different geographic areas) and the detailed and reliable demographic, clinical, immune phenotyping and genomic data to classify study participants into genetically defined ancestry categories. Despite the popularity of the discovery/replication study design, we analyzed available cohorts together (as this is the currently accepted approach (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116933">McCarthy et al. 2008</a>; <a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116925">Skol et al. 2006</a>), and taking advantage of the full set of genotyping data), which resulted in greater power and provided more accurate estimates of the magnitude of SNP associations and genomic localization. Although our significant findings (i.e., the involvement of <em>CD46</em> and <em>IFI44L</em> in the host response to measles vaccine) are supported by a different state-of-the-art GWAS study (Feenstra <em>et al.</em> GWAS study of febrile seizures (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116936">Feenstra et al. 2014</a>)), functional mechanistic studies are warranted to elucidate the underlying molecular mechanisms and the link between <em>CD46</em> and <em>IFI44L</em> genetic variants and humoral immunity to MV.</p><p id="P33" class="p p-last">In conclusion, our study identified common genetic variants associated with inter-individual variations in measles-specific antibody response following MMR vaccination. Ultimately, our study significantly advances the science of measles vaccine immunology/immunogenetics and provides knowledge of the most critical genomic features influencing measles vaccine immune response and their potential molecular mechanisms – a foundational information for the development of better measles vaccine candidates and vaccination approaches. Our study also provides more general insights and a proof of concept for the critical role of genomic variability in virus/pathogen cellular receptors and receptor isoform usage for the development of humoral immunity following vaccination.</p></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Detailed Methods</h2><p id="P34" class="p p-first">The methods described herein are similar or identical to those published for our previous studies. [1–12]</p><div id="S23" class="sec"><h3 id="S23title">Study Subjects</h3><p id="P35" class="p p-first">Subjects from previously described cohorts were used for this study [<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116941">1</a>–<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608117000">5</a>, <a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116968">9</a>–<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116880">11</a>, <a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116872">13</a>]</p><p id="P36">The study set was a large sample of 3,191 healthy children, older adolescents, and healthy adults (age 11 to 40 years) consisting of three independent cohorts: a Rochester cohort (n=1,062); a San Diego cohort (n=1,071); and a US cohort (n=1,058). The recruitment efforts, demographic and clinical characteristics of these cohorts have been previously published [<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116882">1</a>–<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116915">5</a>, <a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116921">9</a>–<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116963">11</a>, <a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116983">13</a>].</p><p id="P37">The Rochester cohort comprised 1,062 individuals enrolled into three age-stratified cohorts of healthy, school-age children and young adults from all socioeconomic strata in Rochester, MN, recruited between 2001–2009, as described previously.[<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116883">4</a>, <a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608117004">5</a>, <a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116995">7</a>, <a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116974">13</a>, <a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116988">14</a>] Parental consent was obtained for all participants and each subject had written records of having received two doses of MMR vaccine. Of the 1,062 individuals for this study, 982 (93%) were successfully genotyped and assayed for immune outcomes.</p><p id="P38" class="p p-last">The San Diego cohort consisted of 1,071 healthy older adolescents and healthy adults (age 18 to 40 years) from armed forces personnel in San Diego, CA, enrolled by the US Naval Health Research Center (NHRC) between 2005–2006, as previously published.[<a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116942">6</a>] The US cohort consisted of an additional 1,058 healthy older adolescents and healthy adults recruited from the US armed forces, enrolled between 2010–2011.[<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116962">8</a>]. Of the 1,071 and 1,058 subjects recruited into this study as the San Diego or US cohort (respectively), 882 (82%) subjects for the San Diego cohort and 1,008 (95%) subjects for the US cohort had provided consent for use of their samples/data in other studies, and were successfully genotyped and assayed for measles-specific immune outcomes for this study. The Institutional Review Boards of the Mayo Clinic (Rochester, MN) and the NHRC (San Diego, CA) approved the study, and written informed consent was obtained from each subject, from the parents of all children who participated in the study, as well as written assent from age-appropriate participants.</p></div><div id="S24" class="sec"><h3 id="S24title">Genome-wide SNP typing and QC</h3><p id="P39" class="p p-first">The methodology used for this study is identical to that used in our previously published papers. [<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116930">1</a>–<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116935">5</a>, <a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116898">9</a>–<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116957">11</a>, <a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116869">13</a>]</p><p id="P40">Briefly, DNA was extracted from each subject’s blood specimen using the Gentra Puregene Blood kit (Gentra Systems Inc.; Minneapolis, MN) and quantified by Picogreen (Molecular Probes; Carlsbad, CA). The genome-wide SNP typing was performed using the Infinium Omni 1M-Quad SNP array (Illumina; San Diego, CA) for the Rochester cohort, Illumina Human Omni2.5–8 BeadChip array for the US cohort, and Illumina Infinium HumanHap550v3_A or HumanHap650Yv3 BeadChip arrays for the San Diego cohort. DNA samples underwent amplification, fragmentation, and hybridization onto each BeadChip, which were imaged on an Illumina BeadArray reader. Genotype calls based on clustering of the raw intensity data were made using BeadStudio 2 software.</p><p id="P41" class="p p-last">For the 758 subjects in the San Diego cohort with the Illumina 550 array genotyping data, 54 subjects were removed due to high SNP missing rates (more than 5% of their SNPs missing). There were 561,303 unique measured SNPs, of which 508,199 SNPs passed QC. For the 313 subjects in the San Diego cohort with the Illumina 650 array genotyping data, 13 subjects were removed due to high SNP missing rates. There were 660,755 unique measured SNPs, of which 630,240 passed QC. In addition, subjects were further excluded from the San Diego cohort if not of Caucasian or African-American ancestry (determined by STRUCTURE), or if they were missing values for covariates used in the analysis. This resulted in a total of 882 subjects from this cohort used in the study (718 Caucasians and 164 African-Americans, see <a href="/pmc/articles/PMC5433429/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). For the 1,058 subjects in the US cohort with the Illumina Omni 2.5 array genotyping data, there were 2,376,105 measured SNPs, of which 2,116,447 passed QC. Subjects were excluded from the US cohort if not of Caucasian or African-American ancestry as determined by STRUCTURE, or had high SNP missing rates, or if values for covariates were missing; this resulted in a total of 1,008 subjects (895 Caucasians and 113 African Americans, <a href="/pmc/articles/PMC5433429/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). For the 1,062 subjects in the Rochester cohort with valid data, 10 subjects were removed due to high SNP missing rates. Subjects were further excluded from the cohort if not of Caucasian or African-American ancestry determined by STRUCTURE, or if values for covariates were missing; this resulted in a total of 982 subjects (942 Caucasians and 40 African Americans, <a href="/pmc/articles/PMC5433429/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). On the Omni 1 genotyping array, there were 1,134,514 unique measured SNPs, of which 887,889 passed QC.</p></div><div id="S25" class="sec"><h3 id="S25title">Next Generation Sequencing (mRNA-Seq)</h3><p id="P42" class="p p-first">The sequencing methods are similar or identical to those previously published. [<a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116897">15</a>, <a href="#R66" rid="R66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116944">16</a>] In summary, cryopreserved PBMCs from 30 Rochester cohort subjects (selected for an mRNA-Seq transcriptome profiling, based on their neutralizing antibody titers [15 highest and 15 lowest antibody responders] after two doses of MMR vaccine) were thawed and stimulated with live Edmonston measles virus (MV) at a multiplicity of infection (MOI) of 0.5 for 24 hours (for each subject, an aliquot of the cells was left unstimulated). The samples were randomly allocated to flow cells and lanes, balancing over the immune response and stimulation status.</p><p id="P43" class="p p-last">Cells were stabilized with RNAprotect cell reagent (Qiagen) and total RNA was extracted using RNeasy Plus mini kit (Qiagen). Quality and quantity of RNA was determined by Nanodrop spectrophotometry (Thermo Fisher Scientific). Libraries were generated using Illumina’s mRNA TruSeq (v1) kit and sequenced (paired end sequencing) on an Illumina HiSeq 2000 (Illumina; San Diego, CA) with Illumina’s TruSeq Cluster kit (v3-cBot-HS) and 51 Cycle Illumina TruSeq SBS Sequencing Kit (v3). The sequencing reads were aligned to the human genome build 37.1 using TopHat (1.3.3) and Bowtie (0.12.7). Gene counts were performed using HTSeq (0.5.3p3), while BEDTools software (2.7.1) was used to map normalized read count to individual exons. [<a href="#R67" rid="R67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116977">17</a>–<a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116879">20</a>]</p></div><div id="S26" class="sec"><h3 id="S26title">RT-PCR</h3><p id="P44" class="p p-first-last">Total RNA was extracted from PBMCs of 30 subjects (10 <em>CD46</em> rs2724374 homozygous major allele genotype subjects, 10 rs2724374 homozygous minor allele genotype subjects and 10 heterozygous subjects) using RNAprotect cell reagent and the RNAeasy Plus Mini kit (Qiagen, Valencia, CA). RT-PCR analysis of CD46 isoforms was performed with the SuperScript<sup>®</sup> One-Step RT-PCR system with Platinum Taq<sup>®</sup> DNA polymerase (Invitrogen, Carlsbad, CA), as previously described [<a href="#R71" rid="R71" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116939">21</a>] using the 5′ <em>CD46</em> SCR4 primer GTGGTCAAATGTCGATTTCCAGTAGTCG and the 3′ untranslated region primer CAAGCCACATTGCAATATTAGCTAAGCCACA, allowing CD46 isoform/isoforms discrimination/separation on a 3% agarose gel.</p></div><div id="S27" class="sec"><h3 id="S27title">Immune Outcome Phenotyping</h3><p id="P45" class="p p-first">The immune outcome assays described herein are similar or identical to those published for our previous studies. [<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116910">1</a>–<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116986">8</a>]</p><div id="S28" class="sec"><p></p><h4 id="S28title" class="inline">Neutralizing Antibody Assay </h4><p id="P46" class="p p-first-last">Measles-specific neutralizing antibody titers were quantified using a high-throughput, fluorescence-based plaque reduction microneutralization assay (PRMN), using a recombinant GFP-expressing measles virus, as previously described.[<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116949">3</a>–<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116965">8</a>] The plates were scanned and counted on an automated Olympus IX71 Fluorescent microscope using the Image-Pro Plus Software Version 6.3 (MediaCybernetics). The 50% end-point titer (Neutralizing Doze, ND<sub>50</sub>) was calculated automatically using Karber’s formula, and transformed into mIU/mL (using the 3<sup>rd</sup> WHO international anti-measles antibody standard), as described previously.[<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116964">3</a>–<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116996">8</a>] The variability of the PRMN assay, calculated as a coefficient of variation (CV) based on the log-transformed ND<sub>50</sub> values of the third WHO standard, was 5.7%.[<a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116969">7</a>]</p></div><div id="S29" class="sec"><p></p><h4 id="S29title" class="inline">IFNγ ELISPOT Assay </h4><p id="P47" class="p p-first-last">Measles-specific cellular immunity was quantified using commercial Human IFNγ ELISPOT kits (R&amp;D Systems; Minneapolis, MN) to measure the number of MV-specific IFNγ-producing cells, according to the manufacturer’s instructions and as previously described [<a href="#R72" rid="R72" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116981">22</a>]. We stimulated subjects’ peripheral blood mononuclear cells/PBMCs (or, alternatively, left them unstimulated) in triplicate with the Edmonston strain of MV (MOI=0.5), and developed the reaction after 42 hours incubation at 37°C, in 5 % CO<sub>2</sub>. PHA (5 μg/mL) was used as a positive control. All plates were scanned and analyzed using the same counting parameters on an ImmunoSpot<sup>®</sup> S4 Pro Analyzer (Cellular Technology Ltd.; Cleveland, OH) using ImmunoSpot<sup>®</sup> version 4.0 software (Cellular Technology Ltd.). The ELISPOT response is presented in spot-forming units (SFUs) per 2 x 10<sup>5</sup> cells. The intraclass correlation coefficients, comparing multiple observations per sample (stimulated and unstimulated condition), was 0.94 for the stimulated values and 0.85 for the unstimulated values, indicating low assay variability. [<a href="#R72" rid="R72" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116992">22</a>]</p></div><div id="S30" class="sec sec-last"><p></p><h4 id="S30title" class="inline">Measles-specific secreted cytokines </h4><p id="P48" class="p p-first-last">Secreted cytokines (IL-2, IL-6; IL-10; TNFα, IFNγ, IFNα and IFNλ1) were quantified in PBMC cultures after <em>in vitro</em> stimulation with live MV, as previously described. [<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116902">3</a>–<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116918">5</a>]</p></div></div><div id="S31" class="sec sec-last"><h3 id="S31title">Statistical methods</h3><div id="S32" class="sec sec-first"><p></p><h4 id="S32title" class="inline">Analysis Strategy </h4><p id="P49" class="p p-first-last">To achieve greatest power to detect SNPs associated with measles-specific immune response phenotypes, we pooled our data across genotyping platforms and the three cohorts. To perform the pooled analyses, we first accounted for the effects of potentially confounding factors that vary across ancestry/platform/cohort strata. After thoroughly evaluating the quality of the genotype data, we used the genetic data to define major ancestry groups. We then estimated eigenvectors within each ancestry group to account for the effects of population stratification within ancestry groups. Because the largest ancestry groups were Caucasian and African-American, we restricted our pooled analysis to these two groups. After accounting for population stratification, we evaluated the covariates that were available within each of the ancestry-platform-cohort strata to determine if the covariates were associated with the phenotype, in order to regress out the effects of potential confounding factors. This produced residuals (adjusted traits) that were then used for the GWAS analyses.</p></div><div id="S33" class="sec"><p></p><h4 id="S33title" class="inline">Genotype Quality Control and Imputation </h4><p id="P50" class="p p-first-last">Genotype quality control prior to imputation was conducted separately for four strata: subjects in the San Diego cohort with the Illumina 550 array genotyping data; subjects in the San Diego cohort with the Illumina 650 array genotyping data; subjects in the US cohort with the Omni 2.5 array genotyping data; and subjects in the Rochester cohort with the Omni 1M-Quad array genotyping data. SNPs on the Y chromosome and mitochondria were removed, and SNPs were eliminated if they were monomorphic or had a missing rate of 1% (5 % for the HumanHap550/650 arrays) or more. Subjects were eliminated if they had more than 5% of their SNPs missing (for the HumanHap550/650 arrays) or more than 1% missing (for the Omni 2.5 and Omni 1M-Quad arrays). The 1000 Genomes cosmopolitan samples (African, AFR; AMR; Asian, ASN; European, EUR) were used as a reference for imputation and were based on Build 37. SNPs that could not be converted to Build 37, or mapped to more than one position, or could not have their alleles verified for the forward strand, were eliminated. The reference genome was then filtered to include only those SNPs with a minor allele frequency (MAF) greater than 0.005. The data were then phased using SHAPEIT [<a href="#R73" rid="R73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116868">23</a>] and imputed via IMPUTE2. [<a href="#R74" rid="R74" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116919">24</a>] SNPs with an imputation dosage allele r<sup>2</sup> of at least 0.3, and a MAF of at least 0.01, were used in the pooled analyses.</p></div><div id="S34" class="sec"><p></p><h4 id="S34title" class="inline">Genetic Ancestry and Population Stratification </h4><p id="P51" class="p p-first">Genetic data was used to assign ancestry groups (African, Caucasian, or Asian) for individuals using the STRUCTURE software [<a href="#R75" rid="R75" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116907">25</a>], and using the 1000 Genomes data as a reference. These estimates were done within cohort and platform (San Diego/550, San Diego/650, US/Omni 2.5, Rochester/Omni 1). Others [<a href="#R76" rid="R76" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116901">26</a>] have shown that it is necessary to perform pruning of the SNPs to be used for eigenvectors in order to avoid having sample eigenvectors that are determined by small clusters of SNPs at specific locations, such as the lactose intolerance gene, or polymorphic inversion regions. Therefore, the SNPs used for STRUCTURE and for eigenvectors were selected by LD pruning from an initial pool consisting of all autosomal SNPs with the following filters: SNPs with a minor allele frequency (MAF) &lt; 5% were excluded; influential SNPs were removed (according to the following chromosome regions: chromosome 8 [bp 1-12700000]; chromosome 2 [bp 129900001-136800000, 5700000-33500000]; chromosome 4 [bp 0900001-44900000]); correlation (r<sup>2</sup>) pruning was used to subset to uncorrelated SNPs. SNPs passing these selection criteria were input to STRUCTURE [<a href="#R75" rid="R75" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116899">25</a>] to make ancestry “triangle” plots that depict the admixture proportions of ancestry groups for each subject. Subjects were classified into major ancestry groups based on the largest estimated STRUCTURE ancestry proportion.</p><p id="P52" class="p p-last">Eigenvectors were estimated within ancestry groups and platform strata for refined control of population stratification. For this step, SNPs with a MAF &lt; 0.01 were excluded, SNPs with a HWE p-value &lt; 0.001 were excluded, INDELS were removed, and pruning according to variance inflation factors was used. These data were then used with smartPCA to produce a set of eigenvectors using the normalization formulas of Price <em>et al</em>. [<a href="#R77" rid="R77" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116997">27</a>] following the procedures of EIGENSTRAT. Tracy-Widom statistics were computed to include eigenvectors as potential for adjusting covariates if they had a p-value &lt; 0.05.</p></div><div id="S35" class="sec"><p></p><h4 id="S35title" class="inline">Selection of Covariates to Adjust for Potential Confounders </h4><p id="P53" class="p p-first-last">To remove the effects of potential confounders so that the cohorts could be combined, we first screened for potential confounders relevant to each ancestry group and cohort. The immune response trait measles-specific IFNγ ELISPOT, as well as the measles-specific secreted cytokines, were transformed by normal quantiles of the difference of the mean stimulated and mean unstimulated values. Neutralizing antibody titer was transformed as the natural log of the PRMN mIU/mL value. Possible confounders were screened for their association with the trait using the following steps. Any categorical variable with a very large number of categories was binned using hierarchical clustering. This was achieved by using hierarchical clustering on the estimated regression coefficients for the different categories, binning categories with similar regression coefficients. All categorical variables were coded as dummy variables such that the most common category was used as baseline. Variables that were marginally associated with the trait with p-value &lt; 0.1 were then included in backwards selection with a p-value threshold of 0.1. This somewhat liberal threshold achieves the goal of controlling for potential confounding covariates. Residuals from the final “covariate models” were then used as the primary adjusted traits for GWAS analyses.</p></div><div id="S36" class="sec"><p></p><h4 id="S36title" class="inline">GWAS Pooled Analysis </h4><p id="P54" class="p p-first">The adjusted traits for IFNγ ELISPOT and neutralizing antibody were pooled over the three cohorts and genotype platforms to perform the pooled GWAS analysis. To pool the genotypes, we used the genotypes that were imputed separately within each platform. To analyze Caucasian and African-American ancestry together, we used a linear regression model that included a cohort indicator, ancestry indicator, dose of minor allele, and interaction of ancestry with dose of minor allele. This full model was compared with a reduced model that included a cohort indicator and ancestry indicator. The full and reduced models were used to create a likelihood ratio test for the effect of a SNP, allowing for the possible interaction of the SNP with ancestry, resulting in a test with 2 degrees of freedom (df). To test for the effect of a SNP within an ancestry group, we only used cohort indicator as an adjusting factor, resulting in a test with 1 df for the SNP effect. The measles-specific secreted cytokines were only measured in the Rochester cohort, so the resulting test for SNP effects was based on a 1 df statistic. All reported p-values are two-sided. To control for multiple testing, we used the standard p-value &lt; 5.0x10<sup>−08</sup> [<a href="#R78" rid="R78" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116924">28</a>, <a href="#R79" rid="R79" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608117005">29</a>] to determine genome-wide statistical significance. Statistical analyses were performed with the R 3.2.0 statistical software and PLINK. [<a href="#R80" rid="R80" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116886">30</a>]</p><div id="S37" class="sec sec-last"><p></p><h5 id="S37title" class="inline">Computer code availability </h5><p id="P55" class="p p-first-last">Computer code (linux shell scripts, R code) is available upon request from corresponding author.</p></div></div><div id="S38" class="sec"><p></p><h4 id="S38title" class="inline">Analysis of mRNA-Seq Data for Differential Exon Usage </h4><p id="P56" class="p p-first-last">We tested for evidence of differential exon usage in the CD46 and IFI44L genes using the method of Anders <em>et al.</em>, implemented in the DEXSeq package (version 1.16.10) in the R programing language (version 3.2.3). [<a href="#R81" rid="R81" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116927">31</a>, <a href="#R82" rid="R82" class=" bibr popnode">32</a>] This method utilizes generalized linear models (GLM) fitting per-gene negative binomial models.[<a href="#R83" rid="R83" class=" bibr popnode">33</a>] For the full model, we included main effects for the sample, exon, and an interaction between the exon and allele state/genotype (subjects’ homozygous minor allele compared to heterozygous and homozygous major allele). The likelihood estimate from the full model is compared to the likelihood from the model with the main effects to test the hypothesis for differential exon usage between the different genotypes/allele states. A significant test for a given exon provides evidence of differential exon usage (i.e., an alternative splicing event to produce different transcripts/isoforms). The per-exon estimates of the dispersion for the negative binomial GLMs were calculated using the Cox-Reid method [<a href="#R81" rid="R81" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116909">31</a>, <a href="#R84" rid="R84" class=" bibr popnode">34</a>, <a href="#R85" rid="R85" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116947">35</a>] applied to all of the observed exons from our experiment based on the full model. To limit the risk of false discovery, our analyses of differential exon usage were limited to the CD46 (n=14 exons) and IFI44L (n =9 exons) genes. The method of Benjamini and Hochberg was used to control the false discovery rate and the adjusted p-values are reported as q-values.[<a href="#R86" rid="R86" class=" bibr popnode">36</a>] We used mRNA-Seq paired-end sequencing data on 28 subjects (selected based on the extremes of the distribution of the antibody response, 14 high and 14 low antibody responders) to test the hypothesis of differential exon usage (exon expression) between different genotypes of interest (for <em>CD46</em> rs2724374 and for <em>IFI44L</em> rs1333973/rs273259).</p></div><div id="S39" class="sec sec-last"><p></p><h4 id="S39title" class="inline">Molecular Modeling </h4><p id="P57" class="p p-first">No full-length structure of CD46 exists, but partial structures of the extracellular domains have been solved. These cover the sequences encoded by the first six exons of the gene. Available experimental structures include the co-crystal structure between CD46 SCR1-4 and the adenovirus type 11 knob (3o8e [<a href="#R87" rid="R87" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116985">37</a>]), and a co-crystal structure of CD46 SCR1-2 bound to the MV-H globular head domain dimer (3inb [<a href="#R88" rid="R88" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608117007">38</a>]). Computationally determined models have also been deposited in MODBASE [<a href="#R89" rid="R89" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608117003">39</a>]. We generated homology models of the extracellular domains (SCR1-4 and STP domains) of ABC1, BC1, and C1 isoforms of CD46 (identical to ABC2, BC2, and C2, respectively) using I-TASSER [<a href="#R90" rid="R90" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116929">40</a>] and compared them to the MODBASE models and available crystal structures, in order to evaluate model quality and consistency.</p><p id="P58">In order to evaluate the effect of differential splicing on the dynamics of the CD46 structure, we generated Anisotropic Network Models (ANMs)[<a href="#R91" rid="R91" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116946">41</a>, <a href="#R92" rid="R92" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116967">42</a>] using a distance-dependent interaction strength [<a href="#R93" rid="R93" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116990">43</a>]. Models were evaluated using computed Mean Squared Fluctuations as a per-residue measure of mobility, commute time [<a href="#R94" rid="R94" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116980">44</a>] as a measure of efficiency with which information passes through the structure, and angle monitors to evaluate overall domain-domain orientations. The motions computed by ANM are theoretically local (small-scale), but often correlate strongly with large-scale functional motions. Thus, the functional magnitudes of ANM motions are not intrinsically defined. Therefore, we deformed each structure to 2Å RMSD in each direction of a given mode as a consistent and realistic extent of motion.</p><p id="P59">The crystal structure of the MV-H head domain dimer and its interactions with CD46 were taken from PDB 3INB [<a href="#R88" rid="R88" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116889">38</a>] and we modeled its dynamics using ANM. Motions of bound CD46 molecules induced by these intrinsic motions of MV-H were inferred by rigidly tethering SCR1 and SCR2 to their interacting residues in the crystal structure.</p><p id="P60" class="p p-last">Diagram mModels of MV-F were produced by Gaussian smoothing of the molecular surface of 1G5G [<a href="#R95" rid="R95" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_608116884">45</a>] – the fusion protein of Newcastle disease virus/NDV which, like MV, is a member of the <em>Paramyxoviridae</em> family.</p></div></div></div><div id="S40" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S40title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>439_2017_1768_MOESM1_ESM</h4><!--caption a9--><strong>Supplemental Table 1: </strong><p id="P65">Top 20 genome-wide associations of SNPs with measles-specific IFNγ ELISPOT response after MMR vaccination (<em>Combined analysis, n=2872</em><sup>a</sup>)</p><div class="sup-box half_rhythm" id="d36e1383"><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM1_ESM.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(34K, docx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>439_2017_1768_MOESM2_ESM</h4><!--caption a9--><strong>Supplemental Table 2: </strong><p id="P66">Associations between chromosome 1 SNPs and measles virus-specific secreted IFNα (<em>Caucasians, n=625</em> subjects with available measles virus-specific cytokine data)</p><div class="sup-box half_rhythm" id="d36e1395"><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM2_ESM.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(23K, docx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>439_2017_1768_MOESM3_ESM</h4><!--caption a9--><strong>Supplemental Fig. 1: </strong><p id="P61">Manhattan and Q-Q plots summary of GWAS associations between SNPs and measles-specific neutralizing antibody titers.</p><p id="P67"><strong>A.</strong> Manhattan (left panel) and Q-Q (right panel) plots for the combined cohort analysis (n=2872, reduced to 2818 after excluding subjects with neutralizing antibody data, that failed QC).</p><p id="P68"><strong>B.</strong> Manhattan (left panel) and Q-Q (right panel) plots for the subset analysis of subjects of Caucasian ancestry (n=2555, reduced to 2506 after excluding subjects with neutralizing antibody data, that failed QC).</p><div class="sup-box half_rhythm" id="d36e1412"><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM3_ESM.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(453K, tif)</sup></div></div><div class="sec suppmat" id="SD4"><h4>439_2017_1768_MOESM4_ESM</h4><!--caption a9--><strong>Supplemental Fig. 2: </strong><p id="P62">Haplotype block structure of the significant IFI44L and CD46 SNPs associated with measles-specific antibody response (combined sample).</p><p id="P69">The schematic representation and LD block structure of <em>IFI44L</em> (left) and <em>CD46</em> (right) genetic regions are depicted (significantly associated SNPs only). The LD block structure was analyzed using Haploview software, version 4.2. The r<sup>2</sup> color scheme is: white (r<sup>2</sup>=0), shades of grey (0&lt; r<sup>2</sup>&lt;1), black (r<sup>2</sup>=1). The numbers report the r<sup>2</sup> value multiplied by 100.</p><div class="sup-box half_rhythm" id="d36e1445"><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM4_ESM.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.1M, tif)</sup></div></div><div class="sec suppmat" id="SD5"><h4>439_2017_1768_MOESM5_ESM</h4><!--caption a9--><strong>Supplemental Fig. 3: </strong><p id="P63">Manhattan plot summary of GWAS associations between SNPs and measles-specific IFNγ ELISPOT response in the combined cohort (n=2872<sup>a</sup>)</p><p id="P76"><sup>a</sup>Reduced to 2618 after excluding subjects with immune outcome data that failed QC</p><div class="sup-box half_rhythm" id="d36e1461"><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM5_ESM.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(199K, tif)</sup></div></div><div class="sec suppmat" id="SD6"><h4>439_2017_1768_MOESM6_ESM</h4><!--caption a9--><strong>Supplemental Fig. 4: </strong><p id="P64">CD46 isoforms exhibit different flexibilities, specifically about the hinge between the SCR4 and STP domains.</p><p id="P70"><strong>A.</strong> Molecular structure of the full length CD46, zoomed in to emphasize the differentially spliced exons.</p><p id="P71"><strong>B.</strong> Using the first 3 modes of an ANM model (see Methods), we compute the mobility of each residue. There is increased mobility for the C1 isoform. (inset) The normal mode frequencies are plotted on a log-log scale and indicate a dramatically lower collectivity for the C1 isoform.</p><p id="P72"><strong>C.</strong> Commute times are computed for each structure and show a decrease in C1 relative to BC1. (inset) Example matrix of commute times from the BC1 isoform with the N-terminus at the top left and C-terminus bottom right.</p><p id="P73"><strong>D.</strong> and <strong>E.</strong> We choose representative C<sup>α</sup> atoms to define the hinge angle between the exon 6 subdomain and the subdomain comprised of the isoform-specific sequences. <strong>D.</strong> for BC1, relatively low mobility about this hinge region is observed (ANM mode 2), while greater flexibility is observed in C1 (ANM mode 2) in <strong>E.</strong> We show representative structures from the ANM modes, deformed to 2Å RMSD in both directions and superimposed about the sequences encoded by the variable exons.</p><p id="P74"><strong>F.</strong> Across the first 5 low-frequency ANM modes, we indicate the change in this angle observed when deforming each structure to 2Å RMSD in each direction.</p><div class="sup-box half_rhythm" id="d36e1502"><a href="/pmc/articles/PMC5433429/bin/NIHMS859705-supplement-439_2017_1768_MOESM6_ESM.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.3M, tif)</sup></div></div></div><div id="S41" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S41title">Acknowledgments</h2><div class="sec"><p id="__p1">We thank the Mayo Clinic Vaccine Research Group staff and the study participants. We wish to recognize Julie M. Cunningham and the Mayo Advanced Genomic Technology Center for the genotyping and next generation sequencing efforts, and Nathaniel D. Warner (Division of Biomedical Statistics and Informatics, Mayo Clinic Department of Health Science Research) for his programming assistance and contribution to statistical analysis. We thank Caroline L. Vitse for her editorial assistance with this manuscript. Research reported in this publication was supported by the National Institute of Allergy And Infectious Diseases of the National Institutes of Health under Award Number R01AI033144 and R37AI048793. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p></div></div><div id="idm140355367288448" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140355367288448title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p2" class="p p-first-last"><em>Presented in part</em>: Ovsyannikova IG, Haralambieva IH, Kennedy RB, Larrabee BR, Schaid DJ, Poland GA. A Genome-Wide Association Study Identifies Major Loci Associated with Measles Vaccine-Specific Immune Responses. The 19th Annual Conference on Vaccine Research, Baltimore, MD, April 18–20, 2016</p><p></p><p class="fn sec" id="FN3"></p><p id="__p3" class="p p-first-last"><strong>Competing Interests:</strong> Dr. Poland is the chair of a Safety Evaluation Committee for novel investigational vaccine trials being conducted by Merck Research Laboratories. Dr. Poland offers consultative advice on vaccine development to Merck &amp; Co. Inc., CSL Biotherapies, Avianax, Dynavax, Novartis Vaccines and Therapeutics, Emergent Biosolutions, Adjuvance, Microdermis, Seqirus, NewLink, Protein Sciences, GSK Vaccines, and Sanofi Pasteur. Drs. Poland and Ovsyannikova hold three patents related to measles and vaccinia peptide research. Dr. Kennedy has received funding from Merck Research Laboratories to study waning immunity to measles and mumps after immunization with MMR-II®. These activities have been reviewed by the Mayo Clinic Conflict of Interest Review Board and are conducted in compliance with Mayo Clinic Conflict of Interest policies. This research has been reviewed by the Mayo Clinic Conflict of Interest Review Board and was conducted in compliance with Mayo Clinic Conflict of Interest policies.</p><p></p></div></div><div id="idm140355367284736" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140355367284736title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Anders S, Reyes A, Huber W. Detecting differential usage of exons from RNA-seq data. <span><span class="ref-journal">Genome research. </span>2012;<span class="ref-vol">22</span>:2008–17. doi: 10.1101/gr.133744.111.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3460195/">PMC free article</a>]</span> [<a href="/pubmed/22722343" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1101%2Fgr.133744.111" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+research&amp;title=Detecting+differential+usage+of+exons+from+RNA-seq+data&amp;author=S+Anders&amp;author=A+Reyes&amp;author=W+Huber&amp;volume=22&amp;publication_year=2012&amp;pages=2008-17&amp;pmid=22722343&amp;doi=10.1101/gr.133744.111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Atilgan AR, Durell SR, Jernigan RL, Demirel MC, Keskin O, Bahar I. Anisotropy of fluctuation dynamics of proteins with an elastic network model. <span><span class="ref-journal">Biophysical journal. </span>2001;<span class="ref-vol">80</span>:505–15. doi: 10.1016/S0006-3495(01)76033-X.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1301252/">PMC free article</a>]</span> [<a href="/pubmed/11159421" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2FS0006-3495(01)76033-X" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophysical+journal&amp;title=Anisotropy+of+fluctuation+dynamics+of+proteins+with+an+elastic+network+model&amp;author=AR+Atilgan&amp;author=SR+Durell&amp;author=RL+Jernigan&amp;author=MC+Demirel&amp;author=O+Keskin&amp;volume=80&amp;publication_year=2001&amp;pages=505-15&amp;pmid=11159421&amp;doi=10.1016/S0006-3495(01)76033-X&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Bednarczyk RA, Orenstein WA, Omer SB. Estimating the Number of Measles-Susceptible Children and Adolescents in the United States Using Data From the National Immunization Survey-Teen (NIS-Teen) <span><span class="ref-journal">American Journal of Epidemiology. </span>2016;<span class="ref-vol">184</span>:148–56. doi: 10.1093/aje/kwv320.</span> [<a href="/pubmed/27338281" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Faje%2Fkwv320" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Epidemiology&amp;title=Estimating+the+Number+of+Measles-Susceptible+Children+and+Adolescents+in+the+United+States+Using+Data+From+the+National+Immunization+Survey-Teen+(NIS-Teen)&amp;author=RA+Bednarczyk&amp;author=WA+Orenstein&amp;author=SB+Omer&amp;volume=184&amp;publication_year=2016&amp;pages=148-56&amp;pmid=27338281&amp;doi=10.1093/aje/kwv320&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Buchholz CJ, Gerlier D, Hu A, Cathomen T, Liszewski MK, Atkinson JP, Cattaneo R. Selective expression of a subset of measles virus receptor-competent CD46 isoforms in human brain. <span><span class="ref-journal">Virology. </span>1996a;<span class="ref-vol">217</span>:349–355.</span> [<a href="/pubmed/8599221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=Selective+expression+of+a+subset+of+measles+virus+receptor-competent+CD46+isoforms+in+human+brain&amp;author=CJ+Buchholz&amp;author=D+Gerlier&amp;author=A+Hu&amp;author=T+Cathomen&amp;author=MK+Liszewski&amp;volume=217&amp;publication_year=1996a&amp;pages=349-355&amp;pmid=8599221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Buchholz CJ, Schneider U, Devaux P, Gerlier D, Cattaneo R. Cell entry by measles virus: Long hybrid receptors uncouple binding from membrane fusion. <span><span class="ref-journal">Virology. </span>1996b;<span class="ref-vol">70</span>:3716–3723.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC190247/">PMC free article</a>]</span> [<a href="/pubmed/8648706" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=Cell+entry+by+measles+virus:+Long+hybrid+receptors+uncouple+binding+from+membrane+fusion&amp;author=CJ+Buchholz&amp;author=U+Schneider&amp;author=P+Devaux&amp;author=D+Gerlier&amp;author=R+Cattaneo&amp;volume=70&amp;publication_year=1996b&amp;pages=3716-3723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Cattaneo R. Four viruses, two bacteria, and one receptor: membrane cofactor protein (CD46) as pathogens’ magnet. <span><span class="ref-journal">J Virol. </span>2004;<span class="ref-vol">78</span>:4385–4388.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC387720/">PMC free article</a>]</span> [<a href="/pubmed/15078919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Virol&amp;title=Four+viruses,+two+bacteria,+and+one+receptor:+membrane+cofactor+protein+(CD46)+as+pathogens’+magnet&amp;author=R+Cattaneo&amp;volume=78&amp;publication_year=2004&amp;pages=4385-4388&amp;pmid=15078919&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Chennubhotla C, Bahar I. Signal propagation in proteins and relation to equilibrium fluctuations. <span><span class="ref-journal">PLoS computational biology. </span>2007;<span class="ref-vol">3</span>:1716–26. doi: 10.1371/journal.pcbi.0030172.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1988854/">PMC free article</a>]</span> [<a href="/pubmed/17892319" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pcbi.0030172" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+computational+biology&amp;title=Signal+propagation+in+proteins+and+relation+to+equilibrium+fluctuations&amp;author=C+Chennubhotla&amp;author=I+Bahar&amp;volume=3&amp;publication_year=2007&amp;pages=1716-26&amp;pmid=17892319&amp;doi=10.1371/journal.pcbi.0030172&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Clifford HD, Hayden CM, Khoo SK, Zhang G, Le Souef PN, Richmond P. CD46 measles virus receptor polymorphisms influence receptor protein expression and primary measles vaccine responses in naive Australian children. <span><span class="ref-journal">Clin Vaccine Immunol. </span>2012;<span class="ref-vol">19</span>:704–710.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3346334/">PMC free article</a>]</span> [<a href="/pubmed/22357652" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Vaccine+Immunol&amp;title=CD46+measles+virus+receptor+polymorphisms+influence+receptor+protein+expression+and+primary+measles+vaccine+responses+in+naive+Australian+children&amp;author=HD+Clifford&amp;author=CM+Hayden&amp;author=SK+Khoo&amp;author=G+Zhang&amp;author=PN+Le+Souef&amp;volume=19&amp;publication_year=2012&amp;pages=704-710&amp;pmid=22357652&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Coulombe-Huntington J, Lam KC, Dias C, Majewski J. Fine-scale variation and genetic determinants of alternative splicing across individuals. <span><span class="ref-journal">PLoS genetics. </span>2009;<span class="ref-vol">5</span>:e1000766. doi: 10.1371/journal.pgen.1000766.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2780703/">PMC free article</a>]</span> [<a href="/pubmed/20011102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pgen.1000766" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=Fine-scale+variation+and+genetic+determinants+of+alternative+splicing+across+individuals&amp;author=J+Coulombe-Huntington&amp;author=KC+Lam&amp;author=C+Dias&amp;author=J+Majewski&amp;volume=5&amp;publication_year=2009&amp;pages=e1000766&amp;pmid=20011102&amp;doi=10.1371/journal.pgen.1000766&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Dhiman N, Cunningham JM, Jacobson RM, Vierkant RA, Wu Y, Ovsyannikova IG, Pankratz VS, Poland GA. Variations in measles vaccine-specific humoral immunity by polymorphisms in SLAM and CD46 measles virus receptors. <span><span class="ref-journal">Journal of Allergy and Clinical Immunology. </span>2007;<span class="ref-vol">120</span>:666–672.</span> [<a href="/pubmed/17560639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Allergy+and+Clinical+Immunology&amp;title=Variations+in+measles+vaccine-specific+humoral+immunity+by+polymorphisms+in+SLAM+and+CD46+measles+virus+receptors&amp;author=N+Dhiman&amp;author=JM+Cunningham&amp;author=RM+Jacobson&amp;author=RA+Vierkant&amp;author=Y+Wu&amp;volume=120&amp;publication_year=2007&amp;pages=666-672&amp;pmid=17560639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Feenstra B, Pasternak B, Geller F, Carstensen L, Wang T, Huang F, Eitson JL, Hollegaard MV, Svanstrom H, Vestergaard M, Hougaard DM, Schoggins JW, Jan LY, Melbye M, Hviid A. Common variants associated with general and MMR vaccine-related febrile seizures. <span><span class="ref-journal">Nature Genetics. </span>2014;<span class="ref-vol">46</span>:1274–82. doi: 10.1038/ng.3129.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4244308/">PMC free article</a>]</span> [<a href="/pubmed/25344690" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3129" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Common+variants+associated+with+general+and+MMR+vaccine-related+febrile+seizures&amp;author=B+Feenstra&amp;author=B+Pasternak&amp;author=F+Geller&amp;author=L+Carstensen&amp;author=T+Wang&amp;volume=46&amp;publication_year=2014&amp;pages=1274-82&amp;pmid=25344690&amp;doi=10.1038/ng.3129&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Fraser HB, Xie X. Common polymorphic transcript variation in human disease. <span><span class="ref-journal">Genome research. </span>2009;<span class="ref-vol">19</span>:567–75. doi: 10.1101/gr.083477.108.</span> [<a href="/pubmed/19189928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1101%2Fgr.083477.108" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+research&amp;title=Common+polymorphic+transcript+variation+in+human+disease&amp;author=HB+Fraser&amp;author=X+Xie&amp;volume=19&amp;publication_year=2009&amp;pages=567-75&amp;pmid=19189928&amp;doi=10.1101/gr.083477.108&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Fuchs A, Atkinson JP, Fremeaux-Bacchi V, Kemper C. CD46-induced human Treg enhance B-cell responses. <span><span class="ref-journal">European Journal of Immunology. </span>2009;<span class="ref-vol">39</span>:3097–109. doi: 10.1002/eji.200939392.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2856319/">PMC free article</a>]</span> [<a href="/pubmed/19784949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1002%2Feji.200939392" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=European+Journal+of+Immunology&amp;title=CD46-induced+human+Treg+enhance+B-cell+responses&amp;author=A+Fuchs&amp;author=JP+Atkinson&amp;author=V+Fremeaux-Bacchi&amp;author=C+Kemper&amp;volume=39&amp;publication_year=2009&amp;pages=3097-109&amp;pmid=19784949&amp;doi=10.1002/eji.200939392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Haralambieva IH, Kennedy RB, Ovsyannikova IG, Whitaker JA, Poland GA. Variability in Humoral Immunity to Measles Vaccine: New Developments. <span><span class="ref-journal">Trends in molecular medicine. </span>2015;<span class="ref-vol">21</span>:789–801. doi: 10.1016/j.molmed.2015.10.005.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4679650/">PMC free article</a>]</span> [<a href="/pubmed/26602762" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.molmed.2015.10.005" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+in+molecular+medicine&amp;title=Variability+in+Humoral+Immunity+to+Measles+Vaccine:+New+Developments&amp;author=IH+Haralambieva&amp;author=RB+Kennedy&amp;author=IG+Ovsyannikova&amp;author=JA+Whitaker&amp;author=GA+Poland&amp;volume=21&amp;publication_year=2015&amp;pages=789-801&amp;pmid=26602762&amp;doi=10.1016/j.molmed.2015.10.005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Haralambieva IH, Ovsyannikova IG, Kennedy RB, Vierkant RA, Pankratz SV, Jacobson RM, Poland GA. Associations between single nucleotide polymorphisms and haplotypes in cytokine and cytokine receptor genes and immunity to measles vaccination. <span><span class="ref-journal">Vaccine. </span>2011a;<span class="ref-vol">29</span>:7883–7895.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3191314/">PMC free article</a>]</span> [<a href="/pubmed/21875636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Associations+between+single+nucleotide+polymorphisms+and+haplotypes+in+cytokine+and+cytokine+receptor+genes+and+immunity+to+measles+vaccination&amp;author=IH+Haralambieva&amp;author=IG+Ovsyannikova&amp;author=RB+Kennedy&amp;author=RA+Vierkant&amp;author=SV+Pankratz&amp;volume=29&amp;publication_year=2011a&amp;pages=7883-7895&amp;pmid=21875636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Haralambieva IH, Ovsyannikova IG, O’Byrne M, Pankratz VS, Jacobson RM, Poland GA. A large observational study to concurrently assess persistence of measles specific B-cell and T-cell immunity in individuals following two doses of MMR vaccine. <span><span class="ref-journal">Vaccine. </span>2011b;<span class="ref-vol">29</span>:4485–4491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3117252/">PMC free article</a>]</span> [<a href="/pubmed/21539880" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=A+large+observational+study+to+concurrently+assess+persistence+of+measles+specific+B-cell+and+T-cell+immunity+in+individuals+following+two+doses+of+MMR+vaccine&amp;author=IH+Haralambieva&amp;author=IG+Ovsyannikova&amp;author=M+O’Byrne&amp;author=VS+Pankratz&amp;author=RM+Jacobson&amp;volume=29&amp;publication_year=2011b&amp;pages=4485-4491&amp;pmid=21539880&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Haralambieva IH, Ovsyannikova IG, Pankratz VS, Kennedy RB, Jacobson RM, Poland GA. The genetic basis for interindividual immune response variation to measles vaccine: new understanding and new vaccine approaches. <span><span class="ref-journal">Expert Review of Vaccines. </span>2013;<span class="ref-vol">12</span>:57–70. doi: 10.1586/erv.12.134.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3570049/">PMC free article</a>]</span> [<a href="/pubmed/23256739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1586%2Ferv.12.134" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Review+of+Vaccines&amp;title=The+genetic+basis+for+interindividual+immune+response+variation+to+measles+vaccine:+new+understanding+and+new+vaccine+approaches&amp;author=IH+Haralambieva&amp;author=IG+Ovsyannikova&amp;author=VS+Pankratz&amp;author=RB+Kennedy&amp;author=RM+Jacobson&amp;volume=12&amp;publication_year=2013&amp;pages=57-70&amp;pmid=23256739&amp;doi=10.1586/erv.12.134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Haralambieva IH, Ovsyannikova IG, Umlauf BJ, Vierkant RA, Pankratz SV, Jacobson RM, Poland GA. Genetic polymorphisms in host antiviral genes: associations with humoral and cellular immunity to measles vaccine. <span><span class="ref-journal">Vaccine. </span>2011c;<span class="ref-vol">29</span>:8988–8997.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3941984/">PMC free article</a>]</span> [<a href="/pubmed/21939710" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Genetic+polymorphisms+in+host+antiviral+genes:+associations+with+humoral+and+cellular+immunity+to+measles+vaccine&amp;author=IH+Haralambieva&amp;author=IG+Ovsyannikova&amp;author=BJ+Umlauf&amp;author=RA+Vierkant&amp;author=SV+Pankratz&amp;volume=29&amp;publication_year=2011c&amp;pages=8988-8997&amp;pmid=21939710&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Haralambieva IH, Zimmermann MT, Ovsyannikova IG, Grill DE, Oberg AL, Kennedy RB, Poland GA. Whole Transcriptome Profiling Identifies CD93 and Other Plasma Cell Survival Factor Genes Associated with Measles-Specific Antibody Response after Vaccination. <span><span class="ref-journal">PLos ONE. </span>2016;<span class="ref-vol">11</span>:e0160970. doi: 10.1371/journal.pone.0160970.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4987012/">PMC free article</a>]</span> [<a href="/pubmed/27529750" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pone.0160970" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLos+ONE&amp;title=Whole+Transcriptome+Profiling+Identifies+CD93+and+Other+Plasma+Cell+Survival+Factor+Genes+Associated+with+Measles-Specific+Antibody+Response+after+Vaccination&amp;author=IH+Haralambieva&amp;author=MT+Zimmermann&amp;author=IG+Ovsyannikova&amp;author=DE+Grill&amp;author=AL+Oberg&amp;volume=11&amp;publication_year=2016&amp;pages=e0160970&amp;pmid=27529750&amp;doi=10.1371/journal.pone.0160970&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Hull J, Campino S, Rowlands K, Chan MS, Copley RR, Taylor MS, Rockett K, Elvidge G, Keating B, Knight J, Kwiatkowski D. Identification of common genetic variation that modulates alternative splicing. <span><span class="ref-journal">PLoS genetics. </span>2007;<span class="ref-vol">3</span>:e99. doi: 10.1371/journal.pgen.0030099.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1904363/">PMC free article</a>]</span> [<a href="/pubmed/17571926" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pgen.0030099" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=Identification+of+common+genetic+variation+that+modulates+alternative+splicing&amp;author=J+Hull&amp;author=S+Campino&amp;author=K+Rowlands&amp;author=MS+Chan&amp;author=RR+Copley&amp;volume=3&amp;publication_year=2007&amp;pages=e99&amp;pmid=17571926&amp;doi=10.1371/journal.pgen.0030099&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Iwata K, Seya T, Ueda S, Ariga H, Nagasawa S. Modulation of complement regulatory function and measles virus receptor function by the serine-threonine-rich domains of membrane cofactor protein (CD46) <span><span class="ref-journal">The Biochemical journal. </span>1994;<span class="ref-vol">304</span>(Pt 1):169–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1137467/">PMC free article</a>]</span> [<a href="/pubmed/7998929" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Biochemical+journal&amp;title=Modulation+of+complement+regulatory+function+and+measles+virus+receptor+function+by+the+serine-threonine-rich+domains+of+membrane+cofactor+protein+(CD46)&amp;author=K+Iwata&amp;author=T+Seya&amp;author=S+Ueda&amp;author=H+Ariga&amp;author=S+Nagasawa&amp;volume=304&amp;issue=Pt+1&amp;publication_year=1994&amp;pages=169-75&amp;pmid=7998929&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Kennedy RB, Ovsyannikova IG, Haralambieva IH, O’Byrne MM, Jacobson RM, Pankratz VS, Poland GA. Multigenic control of measles vaccine immunity mediated by polymorphisms in measles receptor, innate pathway, and cytokine genes. <span><span class="ref-journal">Vaccine. </span>2012a;<span class="ref-vol">30</span>:2159–2167.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3288471/">PMC free article</a>]</span> [<a href="/pubmed/22265947" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Multigenic+control+of+measles+vaccine+immunity+mediated+by+polymorphisms+in+measles+receptor,+innate+pathway,+and+cytokine+genes&amp;author=RB+Kennedy&amp;author=IG+Ovsyannikova&amp;author=IH+Haralambieva&amp;author=MM+O’Byrne&amp;author=RM+Jacobson&amp;volume=30&amp;publication_year=2012a&amp;pages=2159-2167&amp;pmid=22265947&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Kennedy RB, Ovsyannikova IG, Pankratz VS, Haralambieva IH, Vierkant RA, Jacobson RM, Poland GA. Genome-wide genetic associations with IFNgamma response to smallpox vaccine. <span><span class="ref-journal">Human Genetics. </span>2012b;<span class="ref-vol">131</span>:1433–1451.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4170655/">PMC free article</a>]</span> [<a href="/pubmed/22661280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Genetics&amp;title=Genome-wide+genetic+associations+with+IFNgamma+response+to+smallpox+vaccine&amp;author=RB+Kennedy&amp;author=IG+Ovsyannikova&amp;author=VS+Pankratz&amp;author=IH+Haralambieva&amp;author=RA+Vierkant&amp;volume=131&amp;publication_year=2012b&amp;pages=1433-1451&amp;pmid=22661280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Kennedy RB, Ovsyannikova IG, Shane PV, Haralambieva IH, Vierkant RA, Poland GA. Genome-wide analysis of polymorphisms associated with cytokine responses in smallpox vaccine recipients. <span><span class="ref-journal">Human Genetics. </span>2012c;<span class="ref-vol">131</span>:1403–1421.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4170585/">PMC free article</a>]</span> [<a href="/pubmed/22610502" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Genetics&amp;title=Genome-wide+analysis+of+polymorphisms+associated+with+cytokine+responses+in+smallpox+vaccine+recipients&amp;author=RB+Kennedy&amp;author=IG+Ovsyannikova&amp;author=PV+Shane&amp;author=IH+Haralambieva&amp;author=RA+Vierkant&amp;volume=131&amp;publication_year=2012c&amp;pages=1403-1421&amp;pmid=22610502&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Lambert ND, Haralambieva IH, Kennedy RB, Ovsyannikova IG, Pankrantz VS, Poland GA. Polymorphisms in HLA-DPB1 are associated with differences in rubella-specific humoral immunity after vaccination. <span><span class="ref-journal">Journal of Infectious Diseases. </span>2015;<span class="ref-vol">211</span>:898–905.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4351364/">PMC free article</a>]</span> [<a href="/pubmed/25293367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Infectious+Diseases&amp;title=Polymorphisms+in+HLA-DPB1+are+associated+with+differences+in+rubella-specific+humoral+immunity+after+vaccination&amp;author=ND+Lambert&amp;author=IH+Haralambieva&amp;author=RB+Kennedy&amp;author=IG+Ovsyannikova&amp;author=VS+Pankrantz&amp;volume=211&amp;publication_year=2015&amp;pages=898-905&amp;pmid=25293367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Lappalainen T, Sammeth M, Friedlander MR, t Hoen PA, Monlong J, Rivas MA, Gonzalez-Porta M, Kurbatova N, Griebel T, Ferreira PG, Barann M, Wieland T, Greger L, van Iterson M, Almlof J, Ribeca P, Pulyakhina I, Esser D, Giger T, Tikhonov A, Sultan M, Bertier G, MacArthur DG, Lek M, Lizano E, Buermans HP, Padioleau I, Schwarzmayr T, Karlberg O, Ongen H, Kilpinen H, Beltran S, Gut M, Kahlem K, Amstislavskiy V, Stegle O, Pirinen M, Montgomery SB, Donnelly P, McCarthy MI, Flicek P, Strom TM, Lehrach H, Schreiber S, Sudbrak R, Carracedo A, Antonarakis SE, Hasler R, Syvanen AC, van Ommen GJ, Brazma A, Meitinger T, Rosenstiel P, Guigo R, Gut IG, Estivill X, Dermitzakis ET. Transcriptome and genome sequencing uncovers functional variation in humans. <span><span class="ref-journal">Nature. </span>2013;<span class="ref-vol">501</span>:506–11. doi: 10.1038/nature12531.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3918453/">PMC free article</a>]</span> [<a href="/pubmed/24037378" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnature12531" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Transcriptome+and+genome+sequencing+uncovers+functional+variation+in+humans&amp;author=T+Lappalainen&amp;author=M+Sammeth&amp;author=MR+Friedlander&amp;author=PA+t+Hoen&amp;author=J+Monlong&amp;volume=501&amp;publication_year=2013&amp;pages=506-11&amp;pmid=24037378&amp;doi=10.1038/nature12531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">Liszewski MK, Tedja I, Atkinson JP. Membrane cofactor protein (CD46) of complement. Processing differences related to alternatively spliced cytoplasmic domains. <span><span class="ref-journal">The Journal of Biological Chemistry. </span>1994;<span class="ref-vol">269</span>:10776–9.</span> [<a href="/pubmed/8144666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Journal+of+Biological+Chemistry&amp;title=Membrane+cofactor+protein+(CD46)+of+complement.+Processing+differences+related+to+alternatively+spliced+cytoplasmic+domains&amp;author=MK+Liszewski&amp;author=I+Tedja&amp;author=JP+Atkinson&amp;volume=269&amp;publication_year=1994&amp;pages=10776-9&amp;pmid=8144666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Manchester M, Liszewski MK, Atkinson JP, Oldstone MBA. Multiple isoforms of CD46 (membrane cofactor protein) serve as receptors for measles virus. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the United States of America. </span>1994;<span class="ref-vol">91</span>:2161–2165.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC43330/">PMC free article</a>]</span> [<a href="/pubmed/8134365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;title=Multiple+isoforms+of+CD46+(membrane+cofactor+protein)+serve+as+receptors+for+measles+virus&amp;author=M+Manchester&amp;author=MK+Liszewski&amp;author=JP+Atkinson&amp;author=MBA+Oldstone&amp;volume=91&amp;publication_year=1994&amp;pages=2161-2165&amp;pmid=8134365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Marie JC, Astier AL, Rivailler P, Rabourdin-Combe C, Wild TF, Horvat B. Linking innate and acquired immunity: divergent role of CD46 cytoplasmic domains in T cell induced inflammation. <span><span class="ref-journal">Nature immunology. </span>2002;<span class="ref-vol">3</span>:659–66. doi: 10.1038/ni810.</span> [<a href="/pubmed/12055630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fni810" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+immunology&amp;title=Linking+innate+and+acquired+immunity:+divergent+role+of+CD46+cytoplasmic+domains+in+T+cell+induced+inflammation&amp;author=JC+Marie&amp;author=AL+Astier&amp;author=P+Rivailler&amp;author=C+Rabourdin-Combe&amp;author=TF+Wild&amp;volume=3&amp;publication_year=2002&amp;pages=659-66&amp;pmid=12055630&amp;doi=10.1038/ni810&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">McCarthy MI, Abecasis GR, Cardon LR, Goldstein DB, Little J, Ioannidis JP, Hirschhorn JN. Genome-wide association studies for complex traits: consensus, uncertainty and challenges. <span><span class="ref-journal">Nat Rev Genet. </span>2008;<span class="ref-vol">9</span>:356–369.</span> [<a href="/pubmed/18398418" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+for+complex+traits:+consensus,+uncertainty+and+challenges&amp;author=MI+McCarthy&amp;author=GR+Abecasis&amp;author=LR+Cardon&amp;author=DB+Goldstein&amp;author=J+Little&amp;volume=9&amp;publication_year=2008&amp;pages=356-369&amp;pmid=18398418&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="element-citation">Navaratnarajah CK, Oezguen N, Rupp L, Kay L, Leonard VH, Braun W, Cattaneo R. The heads of the measles virus attachment protein move to transmit the fusion-triggering signal. <span><span class="ref-journal">Nat Struct Mol Biol. </span>2011;<span class="ref-vol">18</span>:128–134.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3059746/">PMC free article</a>]</span> [<a href="/pubmed/21217701" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Mol+Biol&amp;title=The+heads+of+the+measles+virus+attachment+protein+move+to+transmit+the+fusion-triggering+signal&amp;author=CK+Navaratnarajah&amp;author=N+Oezguen&amp;author=L+Rupp&amp;author=L+Kay&amp;author=VH+Leonard&amp;volume=18&amp;publication_year=2011&amp;pages=128-134&amp;pmid=21217701&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="element-citation">Ovsyannikova IG, Haralambieva IH, Vierkant RA, O’Byrne MM, Jacobson RM, Poland GA. The association of CD46, SLAM, and CD209 cellular receptor gene SNPs with variations in measles vaccine-induced immune responses--a replication study and examination of novel polymorphisms. <span><span class="ref-journal">Human Heredity. </span>2011a;<span class="ref-vol">72</span>:206–223.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3242703/">PMC free article</a>]</span> [<a href="/pubmed/22086389" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Heredity&amp;title=The+association+of+CD46,+SLAM,+and+CD209+cellular+receptor+gene+SNPs+with+variations+in+measles+vaccine-induced+immune+responses--a+replication+study+and+examination+of+novel+polymorphisms&amp;author=IG+Ovsyannikova&amp;author=IH+Haralambieva&amp;author=RA+Vierkant&amp;author=MM+O’Byrne&amp;author=RM+Jacobson&amp;volume=72&amp;publication_year=2011a&amp;pages=206-223&amp;pmid=22086389&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="element-citation">Ovsyannikova IG, Haralambieva IH, Vierkant RA, Pankratz VS, Poland GA. The role of polymorphisms in toll-like receptors and their associated intracellular signaling genes in measles vaccine immunity. <span><span class="ref-journal">Human Genetics. </span>2011b;<span class="ref-vol">130</span>:547–561.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3924423/">PMC free article</a>]</span> [<a href="/pubmed/21424379" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Genetics&amp;title=The+role+of+polymorphisms+in+toll-like+receptors+and+their+associated+intracellular+signaling+genes+in+measles+vaccine+immunity&amp;author=IG+Ovsyannikova&amp;author=IH+Haralambieva&amp;author=RA+Vierkant&amp;author=VS+Pankratz&amp;author=GA+Poland&amp;volume=130&amp;publication_year=2011b&amp;pages=547-561&amp;pmid=21424379&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="element-citation">Ovsyannikova IG, Pankratz VS, Vierkant RA, Jacobson RM, Poland GA. Consistency of HLA associations between two independent measles vaccine cohorts: a replication study. <span><span class="ref-journal">Vaccine. </span>2012;<span class="ref-vol">30</span>:2146–2152.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3288791/">PMC free article</a>]</span> [<a href="/pubmed/22285888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Consistency+of+HLA+associations+between+two+independent+measles+vaccine+cohorts:+a+replication+study&amp;author=IG+Ovsyannikova&amp;author=VS+Pankratz&amp;author=RA+Vierkant&amp;author=RM+Jacobson&amp;author=GA+Poland&amp;volume=30&amp;publication_year=2012&amp;pages=2146-2152&amp;pmid=22285888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="element-citation">Persson BD, Schmitz NB, Santiago C, Zocher G, Larvie M, Scheu U, Casasnovas JM, Stehle T. Structure of the extracellular portion of CD46 provides insights into its interactions with complement proteins and pathogens. <span><span class="ref-journal">PLoS Pathogens. </span>2010;<span class="ref-vol">6</span>:e1001122. doi: 10.1371/journal.ppat.1001122.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947992/">PMC free article</a>]</span> [<a href="/pubmed/20941397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.ppat.1001122" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathogens&amp;title=Structure+of+the+extracellular+portion+of+CD46+provides+insights+into+its+interactions+with+complement+proteins+and+pathogens&amp;author=BD+Persson&amp;author=NB+Schmitz&amp;author=C+Santiago&amp;author=G+Zocher&amp;author=M+Larvie&amp;volume=6&amp;publication_year=2010&amp;pages=e1001122&amp;pmid=20941397&amp;doi=10.1371/journal.ppat.1001122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="element-citation">Poland GA, Jacobson RM. The re-emergence of measles in developed countries: time to develop the next-generation measles vaccines? <span><span class="ref-journal">Vaccine. </span>2012;<span class="ref-vol">30</span>:103–104.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3905323/">PMC free article</a>]</span> [<a href="/pubmed/22196079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=The+re-emergence+of+measles+in+developed+countries:+time+to+develop+the+next-generation+measles+vaccines?&amp;author=GA+Poland&amp;author=RM+Jacobson&amp;volume=30&amp;publication_year=2012&amp;pages=103-104&amp;pmid=22196079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="element-citation">Post TW, Liszewski MK, Adams EM, Tedja I, Miller EA, Atkinson JP. Membrane cofactor protein of the complement system: alternative splicing of serine/threonine/proline-rich exons and cytoplasmic tails produces multiple isoforms that correlate with protein phenotype. <span><span class="ref-journal">J Exp Med. </span>1991;<span class="ref-vol">174</span>:93–102.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2118878/">PMC free article</a>]</span> [<a href="/pubmed/1711570" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=Membrane+cofactor+protein+of+the+complement+system:+alternative+splicing+of+serine/threonine/proline-rich+exons+and+cytoplasmic+tails+produces+multiple+isoforms+that+correlate+with+protein+phenotype&amp;author=TW+Post&amp;author=MK+Liszewski&amp;author=EM+Adams&amp;author=I+Tedja&amp;author=EA+Miller&amp;volume=174&amp;publication_year=1991&amp;pages=93-102&amp;pmid=1711570&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="element-citation">Prevention CfDCa.  [Accessed February 12, 2015 2015];<span><span class="ref-journal">Measles Cases and Outbreaks. </span>2015 </span>  <a href="http://www.cdc.gov/measles/cases-outbreaks.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cdc.gov/measles/cases-outbreaks.html</a>.  <a href="http://www.cdc.gov/measles/cases-outbreaks.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cdc.gov/measles/cases-outbreaks.html</a>.</span></li><li id="R39"><span class="element-citation">Roy A, Kucukural A, Zhang Y. I-TASSER: a unified platform for automated protein structure and function prediction. <span><span class="ref-journal">Nature protocols. </span>2010;<span class="ref-vol">5</span>:725–38. doi: 10.1038/nprot.2010.5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2849174/">PMC free article</a>]</span> [<a href="/pubmed/20360767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnprot.2010.5" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+protocols&amp;title=I-TASSER:+a+unified+platform+for+automated+protein+structure+and+function+prediction&amp;author=A+Roy&amp;author=A+Kucukural&amp;author=Y+Zhang&amp;volume=5&amp;publication_year=2010&amp;pages=725-38&amp;pmid=20360767&amp;doi=10.1038/nprot.2010.5&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="element-citation">Russell SM, Sparrow RL, McKenzie IFC, Purcell DFJ. Tissue-specific and allelic expression of the complement regulator CD46 is controlled by alternative splicing. <span><span class="ref-journal">European Journal of Immunology. </span>1992;<span class="ref-vol">22</span>:1513–1518.</span> [<a href="/pubmed/1601037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=European+Journal+of+Immunology&amp;title=Tissue-specific+and+allelic+expression+of+the+complement+regulator+CD46+is+controlled+by+alternative+splicing&amp;author=SM+Russell&amp;author=RL+Sparrow&amp;author=IFC+McKenzie&amp;author=DFJ+Purcell&amp;volume=22&amp;publication_year=1992&amp;pages=1513-1518&amp;pmid=1601037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R41"><span class="element-citation">Santiago C, Celma ML, Stehle T, Casasnovas JM. Structure of the measles virus hemagglutinin bound to the CD46 receptor. <span><span class="ref-journal">Nature structural &amp; molecular biology. </span>2010;<span class="ref-vol">17</span>:124–9. doi: 10.1038/nsmb.1726.</span> [<a href="/pubmed/20010840" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnsmb.1726" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+structural+&amp;+molecular+biology&amp;title=Structure+of+the+measles+virus+hemagglutinin+bound+to+the+CD46+receptor&amp;author=C+Santiago&amp;author=ML+Celma&amp;author=T+Stehle&amp;author=JM+Casasnovas&amp;volume=17&amp;publication_year=2010&amp;pages=124-9&amp;doi=10.1038/nsmb.1726&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R42"><span class="element-citation">Schoggins JW, Wilson SJ, Panis M, Murphy MY, Jones CT, Bieniasz P, Rice CM. A diverse range of gene products are effectors of the type I interferon antiviral response. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">472</span>:481–485.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3409588/">PMC free article</a>]</span> [<a href="/pubmed/21478870" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+diverse+range+of+gene+products+are+effectors+of+the+type+I+interferon+antiviral+response&amp;author=JW+Schoggins&amp;author=SJ+Wilson&amp;author=M+Panis&amp;author=MY+Murphy&amp;author=CT+Jones&amp;volume=472&amp;publication_year=2011&amp;pages=481-485&amp;pmid=21478870&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R43"><span class="element-citation">Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. <span><span class="ref-journal">Nature Genetics. </span>2006;<span class="ref-vol">38</span>:209–213.</span> [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=AD+Skol&amp;author=LJ+Scott&amp;author=GR+Abecasis&amp;author=M+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R44"><span class="element-citation">Tan PL, Jacobson RM, Poland GA, Jacobsen SJ, Pankratz SV. Twin studies of immunogenicity - determining the genetic contribution to vaccine failure. <span><span class="ref-journal">Vaccine. </span>2001;<span class="ref-vol">19</span>:2434–2439.</span> [<a href="/pubmed/11257374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Twin+studies+of+immunogenicity+-+determining+the+genetic+contribution+to+vaccine+failure&amp;author=PL+Tan&amp;author=RM+Jacobson&amp;author=GA+Poland&amp;author=SJ+Jacobsen&amp;author=SV+Pankratz&amp;volume=19&amp;publication_year=2001&amp;pages=2434-2439&amp;pmid=11257374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R45"><span class="element-citation">Team RDC. <span class="ref-journal">R: a language for statistical computing.</span> R Foundation for Statistical Computing; Vienna, Austria: 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R:+a+language+for+statistical+computing&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R46"><span class="element-citation">Wang G, Liszewski MK, Chan AC, Atkinson JP. Membrane cofactor protein (MCP; CD46): isoform-specific tyrosine phosphorylation. <span><span class="ref-journal">J Immunol. </span>2000;<span class="ref-vol">164</span>:1839–1846.</span> [<a href="/pubmed/10657632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Membrane+cofactor+protein+(MCP;+CD46):+isoform-specific+tyrosine+phosphorylation&amp;author=G+Wang&amp;author=MK+Liszewski&amp;author=AC+Chan&amp;author=JP+Atkinson&amp;volume=164&amp;publication_year=2000&amp;pages=1839-1846&amp;pmid=10657632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R47"><span class="element-citation">Whitaker JA, Poland GA. Measles and mumps outbreaks in the United States: Think globally, vaccinate locally. <span><span class="ref-journal">Vaccine. </span>2014;<span class="ref-vol">32</span>:4703–4. doi: 10.1016/j.vaccine.2014.06.088.</span> [<a href="/pubmed/24992719" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.vaccine.2014.06.088" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Measles+and+mumps+outbreaks+in+the+United+States:+Think+globally,+vaccinate+locally&amp;author=JA+Whitaker&amp;author=GA+Poland&amp;volume=32&amp;publication_year=2014&amp;pages=4703-4&amp;pmid=24992719&amp;doi=10.1016/j.vaccine.2014.06.088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R48"><span class="element-citation">Wiltshire SA, Leiva-Torres GA, Vidal SM. Quantitative trait locus analysis, pathway analysis, and consomic mapping show genetic variants of Tnni3k, Fpgt, or H28 control susceptibility to viral myocarditis. <span><span class="ref-journal">Journal of Immunology. </span>2011;<span class="ref-vol">186</span>:6398–405. doi: 10.4049/jimmunol.1100159.</span> [<a href="/pubmed/21525387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.4049%2Fjimmunol.1100159" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Immunology&amp;title=Quantitative+trait+locus+analysis,+pathway+analysis,+and+consomic+mapping+show+genetic+variants+of+Tnni3k,+Fpgt,+or+H28+control+susceptibility+to+viral+myocarditis&amp;author=SA+Wiltshire&amp;author=GA+Leiva-Torres&amp;author=SM+Vidal&amp;volume=186&amp;publication_year=2011&amp;pages=6398-405&amp;doi=10.4049/jimmunol.1100159&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R49"><span class="element-citation">Yang L, Song G, Jernigan RL. Protein elastic network models and the ranges of cooperativity. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the United States of America. </span>2009;<span class="ref-vol">106</span>:12347–52. doi: 10.1073/pnas.0902159106.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2718344/">PMC free article</a>]</span> [<a href="/pubmed/19617554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1073%2Fpnas.0902159106" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;title=Protein+elastic+network+models+and+the+ranges+of+cooperativity&amp;author=L+Yang&amp;author=G+Song&amp;author=RL+Jernigan&amp;volume=106&amp;publication_year=2009&amp;pages=12347-52&amp;pmid=19617554&amp;doi=10.1073/pnas.0902159106&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R50"><span class="element-citation">Zhao K, Lu ZX, Park JW, Zhou Q, Xing Y. GLiMMPS: robust statistical model for regulatory variation of alternative splicing using RNA-seq data. <span><span class="ref-journal">Genome biology. </span>2013;<span class="ref-vol">14</span>:R74. doi: 10.1186/gb-2013-14-7-r74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4054007/">PMC free article</a>]</span> [<a href="/pubmed/23876401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1186%2Fgb-2013-14-7-r74" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+biology&amp;title=GLiMMPS:+robust+statistical+model+for+regulatory+variation+of+alternative+splicing+using+RNA-seq+data&amp;author=K+Zhao&amp;author=ZX+Lu&amp;author=JW+Park&amp;author=Q+Zhou&amp;author=Y+Xing&amp;volume=14&amp;publication_year=2013&amp;pages=R74&amp;pmid=23876401&amp;doi=10.1186/gb-2013-14-7-r74&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div id="idm140355365459840" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140355365459840title">References for Detailed Methods</h2><div class="ref-list-sec sec"><div class="ref-cit-blk half_rhythm" id="R51">1. <span class="element-citation">Kennedy RB, Ovsyannikova IG, Haralambieva IH, et al.  Multigenic control of measles vaccine immunity mediated by polymorphisms in measles receptor, innate pathway, and cytokine genes. <span><span class="ref-journal">Vaccine. </span>2012;<span class="ref-vol">30</span>:2159–2167.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3288471/">PMC free article</a>]</span> [<a href="/pubmed/22265947" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Multigenic+control+of+measles+vaccine+immunity+mediated+by+polymorphisms+in+measles+receptor,+innate+pathway,+and+cytokine+genes&amp;author=RB+Kennedy&amp;author=IG+Ovsyannikova&amp;author=IH+Haralambieva&amp;volume=30&amp;publication_year=2012&amp;pages=2159-2167&amp;pmid=22265947&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">2. <span class="element-citation">Ovsyannikova IG, Haralambieva IH, Vierkant RA, Pankratz VS, Poland GA. The role of polymorphisms in toll-like receptors and their associated intracellular signaling genes in measles vaccine immunity. <span><span class="ref-journal">Human Genetics. </span>2011;<span class="ref-vol">130</span>:547–561.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3924423/">PMC free article</a>]</span> [<a href="/pubmed/21424379" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Genetics&amp;title=The+role+of+polymorphisms+in+toll-like+receptors+and+their+associated+intracellular+signaling+genes+in+measles+vaccine+immunity&amp;author=IG+Ovsyannikova&amp;author=IH+Haralambieva&amp;author=RA+Vierkant&amp;author=VS+Pankratz&amp;author=GA+Poland&amp;volume=130&amp;publication_year=2011&amp;pages=547-561&amp;pmid=21424379&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">3. <span class="element-citation">Ovsyannikova IG, Haralambieva IH, Vierkant RA, O’Byrne MM, Jacobson RM, Poland GA. The association of CD46, SLAM, and CD209 cellular receptor gene SNPs with variations in measles vaccine-induced immune responses--a replication study and examination of novel polymorphisms. <span><span class="ref-journal">Human Heredity. </span>2011;<span class="ref-vol">72</span>:206–223.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3242703/">PMC free article</a>]</span> [<a href="/pubmed/22086389" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Heredity&amp;title=The+association+of+CD46,+SLAM,+and+CD209+cellular+receptor+gene+SNPs+with+variations+in+measles+vaccine-induced+immune+responses--a+replication+study+and+examination+of+novel+polymorphisms&amp;author=IG+Ovsyannikova&amp;author=IH+Haralambieva&amp;author=RA+Vierkant&amp;author=MM+O’Byrne&amp;author=RM+Jacobson&amp;volume=72&amp;publication_year=2011&amp;pages=206-223&amp;pmid=22086389&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">4. <span class="element-citation">Haralambieva IH, Ovsyannikova IG, Umlauf BJ, et al.  Genetic polymorphisms in host antiviral genes: associations with humoral and cellular immunity to measles vaccine. <span><span class="ref-journal">Vaccine. </span>2011;<span class="ref-vol">29</span>:8988–8997.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3941984/">PMC free article</a>]</span> [<a href="/pubmed/21939710" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Genetic+polymorphisms+in+host+antiviral+genes:+associations+with+humoral+and+cellular+immunity+to+measles+vaccine&amp;author=IH+Haralambieva&amp;author=IG+Ovsyannikova&amp;author=BJ+Umlauf&amp;volume=29&amp;publication_year=2011&amp;pages=8988-8997&amp;pmid=21939710&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">5. <span class="element-citation">Haralambieva IH, Ovsyannikova IG, Kennedy RB, et al.  Associations between single nucleotide polymorphisms and haplotypes in cytokine and cytokine receptor genes and immunity to measles vaccination. <span><span class="ref-journal">Vaccine. </span>2011;<span class="ref-vol">29</span>:7883–7895.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3191314/">PMC free article</a>]</span> [<a href="/pubmed/21875636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Associations+between+single+nucleotide+polymorphisms+and+haplotypes+in+cytokine+and+cytokine+receptor+genes+and+immunity+to+measles+vaccination&amp;author=IH+Haralambieva&amp;author=IG+Ovsyannikova&amp;author=RB+Kennedy&amp;volume=29&amp;publication_year=2011&amp;pages=7883-7895&amp;pmid=21875636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">6. <span class="element-citation">Ovsyannikova IG, Kennedy RB, O’Byrne M, Jacobson RM, Pankratz VS, Poland GA. Genome-wide association study of antibody response to smallpox vaccine. <span><span class="ref-journal">Vaccine. </span>2012;<span class="ref-vol">30</span>:4182–4189.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3367131/">PMC free article</a>]</span> [<a href="/pubmed/22542470" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Genome-wide+association+study+of+antibody+response+to+smallpox+vaccine&amp;author=IG+Ovsyannikova&amp;author=RB+Kennedy&amp;author=M+O’Byrne&amp;author=RM+Jacobson&amp;author=VS+Pankratz&amp;volume=30&amp;publication_year=2012&amp;pages=4182-4189&amp;pmid=22542470&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">7. <span class="element-citation">Haralambieva IH, Ovsyannikova IG, O’Byrne M, Pankratz VS, Jacobson RM, Poland GA. A large observational study to concurrently assess persistence of measles specific B-cell and T-cell immunity in individuals following two doses of MMR vaccine. <span><span class="ref-journal">Vaccine. </span>2011;<span class="ref-vol">29</span>:4485–4491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3117252/">PMC free article</a>]</span> [<a href="/pubmed/21539880" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=A+large+observational+study+to+concurrently+assess+persistence+of+measles+specific+B-cell+and+T-cell+immunity+in+individuals+following+two+doses+of+MMR+vaccine&amp;author=IH+Haralambieva&amp;author=IG+Ovsyannikova&amp;author=M+O’Byrne&amp;author=VS+Pankratz&amp;author=RM+Jacobson&amp;volume=29&amp;publication_year=2011&amp;pages=4485-4491&amp;pmid=21539880&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">8. <span class="element-citation">Ovsyannikova IG, Pankrantz VS, Salk HM, Kennedy RB, Poland GA. HLA alleles associated with the adaptive immune response to smallpox vaccine: a replication study. <span><span class="ref-journal">Human Genetics. </span>2014;<span class="ref-vol">133</span>:1083–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4127812/">PMC free article</a>]</span> [<a href="/pubmed/24880604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Genetics&amp;title=HLA+alleles+associated+with+the+adaptive+immune+response+to+smallpox+vaccine:+a+replication+study&amp;author=IG+Ovsyannikova&amp;author=VS+Pankrantz&amp;author=HM+Salk&amp;author=RB+Kennedy&amp;author=GA+Poland&amp;volume=133&amp;publication_year=2014&amp;pages=1083-92&amp;pmid=24880604&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">9. <span class="element-citation">Kennedy RB, Ovsyannikova IG, Pankratz VS, et al.  Genome-wide genetic associations with IFNgamma response to smallpox vaccine. <span><span class="ref-journal">Human Genetics. </span>2012;<span class="ref-vol">131</span>:1433–1451.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4170655/">PMC free article</a>]</span> [<a href="/pubmed/22661280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Genetics&amp;title=Genome-wide+genetic+associations+with+IFNgamma+response+to+smallpox+vaccine&amp;author=RB+Kennedy&amp;author=IG+Ovsyannikova&amp;author=VS+Pankratz&amp;volume=131&amp;publication_year=2012&amp;pages=1433-1451&amp;pmid=22661280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">10. <span class="element-citation">Kennedy RB, Ovsyannikova IG, Shane PV, Haralambieva IH, Vierkant RA, Poland GA. Genome-wide analysis of polymorphisms associated with cytokine responses in smallpox vaccine recipients. <span><span class="ref-journal">Human Genetics. </span>2012;<span class="ref-vol">131</span>:1403–1421.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4170585/">PMC free article</a>]</span> [<a href="/pubmed/22610502" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Genetics&amp;title=Genome-wide+analysis+of+polymorphisms+associated+with+cytokine+responses+in+smallpox+vaccine+recipients&amp;author=RB+Kennedy&amp;author=IG+Ovsyannikova&amp;author=PV+Shane&amp;author=IH+Haralambieva&amp;author=RA+Vierkant&amp;volume=131&amp;publication_year=2012&amp;pages=1403-1421&amp;pmid=22610502&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">11. <span class="element-citation">Lambert ND, Haralambieva IH, Kennedy RB, Ovsyannikova IG, Pankrantz VS, Poland GA. Polymorphisms in HLA-DPB1 are associated with differences in rubella-specific humoral immunity after vaccination. <span><span class="ref-journal">Journal of Infectious Diseases. </span>2015;<span class="ref-vol">211</span>:898–905.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4351364/">PMC free article</a>]</span> [<a href="/pubmed/25293367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Infectious+Diseases&amp;title=Polymorphisms+in+HLA-DPB1+are+associated+with+differences+in+rubella-specific+humoral+immunity+after+vaccination&amp;author=ND+Lambert&amp;author=IH+Haralambieva&amp;author=RB+Kennedy&amp;author=IG+Ovsyannikova&amp;author=VS+Pankrantz&amp;volume=211&amp;publication_year=2015&amp;pages=898-905&amp;pmid=25293367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">12. <span class="element-citation">Haralambieva IH, Zimmermann MT, Ovsyannikova IG, et al.  Whole Transcriptome Profiling Identifies CD93 and Other Plasma Cell Survival Factor Genes Associated with Measles-Specific Antibody Response after Vaccination. <span><span class="ref-journal">PLos ONE. </span>2016;<span class="ref-vol">11</span>:e0160970.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4987012/">PMC free article</a>]</span> [<a href="/pubmed/27529750" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLos+ONE&amp;title=Whole+Transcriptome+Profiling+Identifies+CD93+and+Other+Plasma+Cell+Survival+Factor+Genes+Associated+with+Measles-Specific+Antibody+Response+after+Vaccination&amp;author=IH+Haralambieva&amp;author=MT+Zimmermann&amp;author=IG+Ovsyannikova&amp;volume=11&amp;publication_year=2016&amp;pages=e0160970&amp;pmid=27529750&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">13. <span class="element-citation">Ovsyannikova IG, Pankratz VS, Vierkant RA, Jacobson RM, Poland GA. Consistency of HLA associations between two independent measles vaccine cohorts: a replication study. <span><span class="ref-journal">Vaccine. </span>2012;<span class="ref-vol">30</span>:2146–2152.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3288791/">PMC free article</a>]</span> [<a href="/pubmed/22285888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Consistency+of+HLA+associations+between+two+independent+measles+vaccine+cohorts:+a+replication+study&amp;author=IG+Ovsyannikova&amp;author=VS+Pankratz&amp;author=RA+Vierkant&amp;author=RM+Jacobson&amp;author=GA+Poland&amp;volume=30&amp;publication_year=2012&amp;pages=2146-2152&amp;pmid=22285888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">14. <span class="element-citation">Ovsyannikova IG, Haralambieva IH, Vierkant RA, O’Byrne MM, Poland GA. Associations between polymorphisms in the antiviral TRIM genes and measles vaccine immunity. <span><span class="ref-journal">Human Immunology. </span>2013;<span class="ref-vol">74</span>:768–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3644318/">PMC free article</a>]</span> [<a href="/pubmed/23416095" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Immunology&amp;title=Associations+between+polymorphisms+in+the+antiviral+TRIM+genes+and+measles+vaccine+immunity&amp;author=IG+Ovsyannikova&amp;author=IH+Haralambieva&amp;author=RA+Vierkant&amp;author=MM+O’Byrne&amp;author=GA+Poland&amp;volume=74&amp;publication_year=2013&amp;pages=768-74&amp;pmid=23416095&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">15. <span class="element-citation">Haralambieva IH, Oberg AL, Ovsyannikova IG, et al.  Genome-wide characterization of transcriptional patterns in high and low antibody responders to rubella vaccination. <span><span class="ref-journal">PLos ONE. </span>2013;<span class="ref-vol">8</span>:e62149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3641062/">PMC free article</a>]</span> [<a href="/pubmed/23658707" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLos+ONE&amp;title=Genome-wide+characterization+of+transcriptional+patterns+in+high+and+low+antibody+responders+to+rubella+vaccination&amp;author=IH+Haralambieva&amp;author=AL+Oberg&amp;author=IG+Ovsyannikova&amp;volume=8&amp;publication_year=2013&amp;pages=e62149&amp;pmid=23658707&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R66">16. <span class="element-citation">Kennedy RB, Oberg AL, Ovsyannikova IG, Haralambieva IH, Grill DE, Poland GA. Transcriptomic profiles of high and low antibody responders to smallpox vaccine. <span><span class="ref-journal">Genes and Immunity. </span>2013;<span class="ref-vol">14</span>:277–285.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3723701/">PMC free article</a>]</span> [<a href="/pubmed/23594957" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+and+Immunity&amp;title=Transcriptomic+profiles+of+high+and+low+antibody+responders+to+smallpox+vaccine&amp;author=RB+Kennedy&amp;author=AL+Oberg&amp;author=IG+Ovsyannikova&amp;author=IH+Haralambieva&amp;author=DE+Grill&amp;volume=14&amp;publication_year=2013&amp;pages=277-285&amp;pmid=23594957&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R67">17. <span class="element-citation">Langmead B, Trapnell C, Pop M, Salzberg SL. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. <span><span class="ref-journal">Genome Biol. </span>2009;<span class="ref-vol">10</span>:R25.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2690996/">PMC free article</a>]</span> [<a href="/pubmed/19261174" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome&amp;author=B+Langmead&amp;author=C+Trapnell&amp;author=M+Pop&amp;author=SL+Salzberg&amp;volume=10&amp;publication_year=2009&amp;pages=R25&amp;pmid=19261174&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R68">18. <span class="element-citation">Quinlan AR, Hall IM. BEDTools: a flexible suite of utilities for comparing genomic features. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:841–2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2832824/">PMC free article</a>]</span> [<a href="/pubmed/20110278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BEDTools:+a+flexible+suite+of+utilities+for+comparing+genomic+features&amp;author=AR+Quinlan&amp;author=IM+Hall&amp;volume=26&amp;publication_year=2010&amp;pages=841-2&amp;pmid=20110278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R69">19. <span class="element-citation">Trapnell C, Pachter L, Salzberg SL. TopHat: discovering splice junctions with RNA-Seq. <span><span class="ref-journal">Bioinformatics. </span>2009;<span class="ref-vol">25</span>:1105–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672628/">PMC free article</a>]</span> [<a href="/pubmed/19289445" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=TopHat:+discovering+splice+junctions+with+RNA-Seq&amp;author=C+Trapnell&amp;author=L+Pachter&amp;author=SL+Salzberg&amp;volume=25&amp;publication_year=2009&amp;pages=1105-11&amp;pmid=19289445&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R70">20. <span class="element-citation">Anders S, Pyl PT, Huber W. HTSeq--a Python framework to work with high-throughput sequencing data. <span><span class="ref-journal">Bioinformatics. </span>2015;<span class="ref-vol">31</span>:166–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4287950/">PMC free article</a>]</span> [<a href="/pubmed/25260700" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=HTSeq--a+Python+framework+to+work+with+high-throughput+sequencing+data&amp;author=S+Anders&amp;author=PT+Pyl&amp;author=W+Huber&amp;volume=31&amp;publication_year=2015&amp;pages=166-9&amp;pmid=25260700&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R71">21. <span class="element-citation">Wang G, Liszewski MK, Chan AC, Atkinson JP. Membrane cofactor protein (MCP; CD46): isoform-specific tyrosine phosphorylation. <span><span class="ref-journal">J Immunol. </span>2000;<span class="ref-vol">164</span>:1839–1846.</span> [<a href="/pubmed/10657632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Membrane+cofactor+protein+(MCP;+CD46):+isoform-specific+tyrosine+phosphorylation&amp;author=G+Wang&amp;author=MK+Liszewski&amp;author=AC+Chan&amp;author=JP+Atkinson&amp;volume=164&amp;publication_year=2000&amp;pages=1839-1846&amp;pmid=10657632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R72">22. <span class="element-citation">Ovsyannikova IG, Haralambieva IH, Vierkant RA, O’Byrne MM, Jacobson RM, Poland GA. Effects of vitamin A and D receptor gene polymorphisms/haplotypes on immune responses to measles vaccine. <span><span class="ref-journal">Pharmacogenetics and Genomics. </span>2012;<span class="ref-vol">22</span>:20–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3237827/">PMC free article</a>]</span> [<a href="/pubmed/22082653" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics+and+Genomics&amp;title=Effects+of+vitamin+A+and+D+receptor+gene+polymorphisms/haplotypes+on+immune+responses+to+measles+vaccine&amp;author=IG+Ovsyannikova&amp;author=IH+Haralambieva&amp;author=RA+Vierkant&amp;author=MM+O’Byrne&amp;author=RM+Jacobson&amp;volume=22&amp;publication_year=2012&amp;pages=20-31&amp;pmid=22082653&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R73">23. <span class="element-citation">Delaneau O, Zagury JF, Marchini J. Improved whole-chromosome phasing for disease and population genetic studies. <span><span class="ref-journal">Nat Methods. </span>2013;<span class="ref-vol">10</span>:5–6.</span> [<a href="/pubmed/23269371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Improved+whole-chromosome+phasing+for+disease+and+population+genetic+studies&amp;author=O+Delaneau&amp;author=JF+Zagury&amp;author=J+Marchini&amp;volume=10&amp;publication_year=2013&amp;pages=5-6&amp;pmid=23269371&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R74">24. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R75">25. <span class="element-citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R76">26. <span class="element-citation">Novembre J, Johnson T, Bryc K, et al.  Genes mirror geography within Europe. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">456</span>:98–101.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2735096/">PMC free article</a>]</span> [<a href="/pubmed/18758442" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genes+mirror+geography+within+Europe&amp;author=J+Novembre&amp;author=T+Johnson&amp;author=K+Bryc&amp;volume=456&amp;publication_year=2008&amp;pages=98-101&amp;pmid=18758442&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R77">27. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nature Genetics. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R78">28. <span class="element-citation">Manolio TA. Genomewide association studies and assessment of the risk of disease. <span><span class="ref-journal">The New England Journal of Medicine. </span>2010;<span class="ref-vol">363</span>:166–76.</span> [<a href="/pubmed/20647212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+New+England+Journal+of+Medicine&amp;title=Genomewide+association+studies+and+assessment+of+the+risk+of+disease&amp;author=TA+Manolio&amp;volume=363&amp;publication_year=2010&amp;pages=166-76&amp;pmid=20647212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R79">29. <span class="element-citation">Pe’er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genetic Epidemiology. </span>2008;<span class="ref-vol">32</span>:381–5.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetic+Epidemiology&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe’er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-5&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R80">30. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">AmJ HumGenet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=AmJ+HumGenet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R81">31. <span class="element-citation">Anders S, Reyes A, Huber W. Detecting differential usage of exons from RNA-seq data. <span><span class="ref-journal">Genome research. </span>2012;<span class="ref-vol">22</span>:2008–17.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3460195/">PMC free article</a>]</span> [<a href="/pubmed/22722343" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+research&amp;title=Detecting+differential+usage+of+exons+from+RNA-seq+data&amp;author=S+Anders&amp;author=A+Reyes&amp;author=W+Huber&amp;volume=22&amp;publication_year=2012&amp;pages=2008-17&amp;pmid=22722343&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R82">32. <span class="element-citation">Team RDC. <span class="ref-journal">R: a language for statistical computing.</span> Vienna, Austria: R Foundation for Statistical Computing; 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R:+a+language+for+statistical+computing&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R83">33. <span class="element-citation">McCullagh Nelder.  <span class="ref-journal">Generalized Linear Models.</span> Chapman and Hall Lt; Boca Raton, FL: 1989.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Generalized+Linear+Models&amp;author=Nelder+McCullagh&amp;publication_year=1989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R84">34. <span class="element-citation">Cox DR, Reid N. Parameter orthogonality and approximate conditional inference. <span><span class="ref-journal">J R Stat Soc Ser B Methodol. </span>1987;<span class="ref-vol">49</span>:1–39.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+R+Stat+Soc+Ser+B+Methodol&amp;title=Parameter+orthogonality+and+approximate+conditional+inference&amp;author=DR+Cox&amp;author=N+Reid&amp;volume=49&amp;publication_year=1987&amp;pages=1-39&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R85">35. <span class="element-citation">McCarthy DJ, Chen Y, Smyth GK. Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation. <span><span class="ref-journal">Nucleic Acids Research. </span>2012;<span class="ref-vol">40</span>:4288–97.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3378882/">PMC free article</a>]</span> [<a href="/pubmed/22287627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Research&amp;title=Differential+expression+analysis+of+multifactor+RNA-Seq+experiments+with+respect+to+biological+variation&amp;author=DJ+McCarthy&amp;author=Y+Chen&amp;author=GK+Smyth&amp;volume=40&amp;publication_year=2012&amp;pages=4288-97&amp;pmid=22287627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R86">36. <span class="element-citation">Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. <span><span class="ref-journal">Journal of the Royal Statistical Society Series B. </span>1995;<span class="ref-vol">57</span>:289–300.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+the+Royal+Statistical+Society+Series+B&amp;title=Controlling+the+false+discovery+rate:+a+practical+and+powerful+approach+to+multiple+testing&amp;author=Y+Benjamini&amp;author=Y+Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R87">37. <span class="element-citation">Persson BD, Schmitz NB, Santiago C, et al.  Structure of the extracellular portion of CD46 provides insights into its interactions with complement proteins and pathogens. <span><span class="ref-journal">PLoS Pathogens. </span>2010;<span class="ref-vol">6</span>:e1001122.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947992/">PMC free article</a>]</span> [<a href="/pubmed/20941397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathogens&amp;title=Structure+of+the+extracellular+portion+of+CD46+provides+insights+into+its+interactions+with+complement+proteins+and+pathogens&amp;author=BD+Persson&amp;author=NB+Schmitz&amp;author=C+Santiago&amp;volume=6&amp;publication_year=2010&amp;pages=e1001122&amp;pmid=20941397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R88">38. <span class="element-citation">Santiago C, Celma ML, Stehle T, Casasnovas JM. Structure of the measles virus hemagglutinin bound to the CD46 receptor. <span><span class="ref-journal">Nat Struct Mol Biol. </span>2010;<span class="ref-vol">17</span>:124–9.</span> [<a href="/pubmed/20010840" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Mol+Biol&amp;title=Structure+of+the+measles+virus+hemagglutinin+bound+to+the+CD46+receptor&amp;author=C+Santiago&amp;author=ML+Celma&amp;author=T+Stehle&amp;author=JM+Casasnovas&amp;volume=17&amp;publication_year=2010&amp;pages=124-9&amp;pmid=20010840&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R89">39. <span class="element-citation">Pieper U, Eswar N, Braberg H, et al.  MODBASE, a database of annotated comparative protein structure models, and associated resources. <span><span class="ref-journal">Nucleic Acids Research. </span>2004;<span class="ref-vol">32</span>:D217–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC308829/">PMC free article</a>]</span> [<a href="/pubmed/14681398" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Research&amp;title=MODBASE,+a+database+of+annotated+comparative+protein+structure+models,+and+associated+resources&amp;author=U+Pieper&amp;author=N+Eswar&amp;author=H+Braberg&amp;volume=32&amp;publication_year=2004&amp;pages=D217-22&amp;pmid=14681398&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R90">40. <span class="element-citation">Roy A, Kucukural A, Zhang Y. I-TASSER: a unified platform for automated protein structure and function prediction. <span><span class="ref-journal">Nat Protoc. </span>2010;<span class="ref-vol">5</span>:725–38.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2849174/">PMC free article</a>]</span> [<a href="/pubmed/20360767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=I-TASSER:+a+unified+platform+for+automated+protein+structure+and+function+prediction&amp;author=A+Roy&amp;author=A+Kucukural&amp;author=Y+Zhang&amp;volume=5&amp;publication_year=2010&amp;pages=725-38&amp;pmid=20360767&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R91">41. <span class="element-citation">Zimmermann MT, Kloczkowski A, Jernigan RL. MAVENs: motion analysis and visualization of elastic networks and structural ensembles. <span><span class="ref-journal">BMC Bioinformatics. </span>2011;<span class="ref-vol">12</span>:264.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3213244/">PMC free article</a>]</span> [<a href="/pubmed/21711533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=MAVENs:+motion+analysis+and+visualization+of+elastic+networks+and+structural+ensembles&amp;author=MT+Zimmermann&amp;author=A+Kloczkowski&amp;author=RL+Jernigan&amp;volume=12&amp;publication_year=2011&amp;pages=264&amp;pmid=21711533&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R92">42. <span class="element-citation">Atilgan AR, Durell SR, Jernigan RL, Demirel MC, Keskin O, Bahar I. Anisotropy of fluctuation dynamics of proteins with an elastic network model. <span><span class="ref-journal">Biophys J. </span>2001;<span class="ref-vol">80</span>:505–15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1301252/">PMC free article</a>]</span> [<a href="/pubmed/11159421" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys+J&amp;title=Anisotropy+of+fluctuation+dynamics+of+proteins+with+an+elastic+network+model&amp;author=AR+Atilgan&amp;author=SR+Durell&amp;author=RL+Jernigan&amp;author=MC+Demirel&amp;author=O+Keskin&amp;volume=80&amp;publication_year=2001&amp;pages=505-15&amp;pmid=11159421&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R93">43. <span class="element-citation">Yang L, Song G, Jernigan RL. Protein elastic network models and the ranges of cooperativity. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the United States of America. </span>2009;<span class="ref-vol">106</span>:12347–52.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2718344/">PMC free article</a>]</span> [<a href="/pubmed/19617554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;title=Protein+elastic+network+models+and+the+ranges+of+cooperativity&amp;author=L+Yang&amp;author=G+Song&amp;author=RL+Jernigan&amp;volume=106&amp;publication_year=2009&amp;pages=12347-52&amp;pmid=19617554&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R94">44. <span class="element-citation">Chennubhotla C, Bahar I. Signal propagation in proteins and relation to equilibrium fluctuations. <span><span class="ref-journal">PLoS Comput Biol. </span>2007;<span class="ref-vol">3</span>:1716–26.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1988854/">PMC free article</a>]</span> [<a href="/pubmed/17892319" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput+Biol&amp;title=Signal+propagation+in+proteins+and+relation+to+equilibrium+fluctuations&amp;author=C+Chennubhotla&amp;author=I+Bahar&amp;volume=3&amp;publication_year=2007&amp;pages=1716-26&amp;pmid=17892319&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R95">45. <span class="element-citation">Chen L, Gorman JJ, McKimm-Breschkin J, et al.  The structure of the fusion glycoprotein of Newcastle disease virus suggests a novel paradigm for the molecular mechanism of membrane fusion. <span><span class="ref-journal">Structure. </span>2001;<span class="ref-vol">9</span>:255–66.</span> [<a href="/pubmed/11286892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=The+structure+of+the+fusion+glycoprotein+of+Newcastle+disease+virus+suggests+a+novel+paradigm+for+the+molecular+mechanism+of+membrane+fusion&amp;author=L+Chen&amp;author=JJ+Gorman&amp;author=J+McKimm-Breschkin&amp;volume=9&amp;publication_year=2001&amp;pages=255-66&amp;pmid=11286892&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5433429&amp;issue-id=292521&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140355357474256title">Abstract</a></li><li><a href="#S5title">Methods</a></li><li><a href="#S13title">Results</a></li><li><a href="#S21title">Discussion</a></li><li><a href="#S22title">Detailed Methods</a></li><li><a href="#S40title">Supplementary Material</a></li><li><a href="#S41title">Acknowledgments</a></li><li><a href="#idm140355367288448title">Footnotes</a></li><li><a href="#idm140355367284736title">References</a></li><li><a href="#idm140355365459840title">References for Detailed Methods</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5433429/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5433429/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5433429/pdf/nihms859705.pdf">PDF (1.2M)</a></li> | <li><a href="#" data-citationid="PMC5433429" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5433429%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5433429%2F&amp;text=Genome-Wide%20Associations%20of%20CD46%20and%20IFI44L%20Genetic%20Variants%20with%20Neutralizing%20Antibody%20Response%20to%20Measles%20Vaccine" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5433429%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28289848" data-aiid="5433429" data-aid="5433429" data-iid="292521" data-domainid="319" data-domain="nihpa" data-accid="PMC5433429" data-md5="e8813376aae2ce4c2d01d186f2ac72e0"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:53:02-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE896FD6DD59B8F100000000077E0496&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5433429%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5433429/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>