<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="pnas"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5402460/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genetic signatures of high-altitude adaptation in Tibetans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5402460/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America"><meta name="citation_title" content="Genetic signatures of high-altitude adaptation in Tibetans"><meta name="citation_authors" content="Jian Yang, Zi-Bing Jin, Jie Chen, Xiu-Feng Huang, Xiao-Man Li, Yuan-Bo Liang, Jian-Yang Mao, Xin Chen, Zhili Zheng, Andrew Bakshi, Dong-Dong Zheng, Mei-Qin Zheng, Naomi R. Wray, Peter M. Visscher, Fan Lu, Jia Qu"><meta name="citation_date" content="18 April 2017"><meta name="citation_issue" content="16"><meta name="citation_volume" content="114"><meta name="citation_firstpage" content="4189"><meta name="citation_doi" content="10.1073/pnas.1617042114"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5402460/?report=abstract"><meta name="citation_pmid" content="28373541"><meta name="DC.Title" content="Genetic signatures of high-altitude adaptation in Tibetans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="National Academy of Sciences"><meta name="DC.Contributor" content="Jian Yang"><meta name="DC.Contributor" content="Zi-Bing Jin"><meta name="DC.Contributor" content="Jie Chen"><meta name="DC.Contributor" content="Xiu-Feng Huang"><meta name="DC.Contributor" content="Xiao-Man Li"><meta name="DC.Contributor" content="Yuan-Bo Liang"><meta name="DC.Contributor" content="Jian-Yang Mao"><meta name="DC.Contributor" content="Xin Chen"><meta name="DC.Contributor" content="Zhili Zheng"><meta name="DC.Contributor" content="Andrew Bakshi"><meta name="DC.Contributor" content="Dong-Dong Zheng"><meta name="DC.Contributor" content="Mei-Qin Zheng"><meta name="DC.Contributor" content="Naomi R. Wray"><meta name="DC.Contributor" content="Peter M. Visscher"><meta name="DC.Contributor" content="Fan Lu"><meta name="DC.Contributor" content="Jia Qu"><meta name="DC.Date" content="2017 Apr 18"><meta name="DC.Identifier" content="10.1073/pnas.1617042114"><meta name="DC.Language" content="en"><meta property="og:title" content="Genetic signatures of high-altitude adaptation in Tibetans"><meta property="og:type" content="article"><meta property="og:description" content="The origin of Tibetans and the mechanism of how they adapted to the high-altitude environment remain mostly unknown. We conduct the largest genome-wide study in Tibetans to date. We detect signatures of natural selection at nine gene loci, two of which ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5402460/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-pnas.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5402460/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5402460/pdf/pnas.201617042.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C2789DD59A93100000000097F05A3.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5402460%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/291368/">v.114(16); 2017 Apr 18</a></li><li class="accid">PMC5402460</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-pnas.gif" alt="Logo of pnas" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="306,2,479,64" alt="PNAS" title="PNAS" href="http://www.pnas.org" ref="reftype=publisher&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="59,34,142,50" alt="Info for Authors" title="Info for Authors" href="http://www.pnas.org/site/misc/iforc.shtml" ref="reftype=publisher&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="143,34,200,50" alt="Subscriptions" title="Subscriptions" href="http://www.pnas.org/site/subscriptions/" ref="reftype=publisher&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="201,34,239,50" alt="About" title="About" href="http://www.pnas.org/site/misc/about.shtml" ref="reftype=publisher&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,34,57,50" alt="This Article" title="This Article" href="//dx.doi.org/10.1073/pnas.1617042114" ref="reftype=publisher&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Proc Natl Acad Sci U S A</a></span>. 2017 Apr 18; 114(16): 4189–4194. </span></div><div><span class="fm-vol-iss-date">Published online 2017 Apr 3. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1073%2Fpnas.1617042114" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.1617042114</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5402460</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28373541">28373541</a></div></div></div><div class="fm-doctopic no_margin">Genetics</div></div><h1 class="content-title">Genetic signatures of high-altitude adaptation in Tibetans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Yang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124590428896" co-class="co-affbox">Jian Yang</a>,<sup>a,</sup><sup>b,</sup><sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Jin%20ZB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124585923488" co-class="co-affbox">Zi-Bing Jin</a>,<sup>b,</sup><sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Chen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124591266976" co-class="co-affbox">Jie Chen</a>,<sup>b</sup> <a href="/pubmed/?term=Huang%20XF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124586880960" co-class="co-affbox">Xiu-Feng Huang</a>,<sup>b</sup> <a href="/pubmed/?term=Li%20XM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124592370736" co-class="co-affbox">Xiao-Man Li</a>,<sup>b</sup> <a href="/pubmed/?term=Liang%20YB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124589737024" co-class="co-affbox">Yuan-Bo Liang</a>,<sup>b</sup> <a href="/pubmed/?term=Mao%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124589734912" co-class="co-affbox">Jian-Yang Mao</a>,<sup>b</sup> <a href="/pubmed/?term=Chen%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124586893424" co-class="co-affbox">Xin Chen</a>,<sup>b</sup> <a href="/pubmed/?term=Zheng%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124592827232" co-class="co-affbox">Zhili Zheng</a>,<sup>a,</sup><sup>b</sup> <a href="/pubmed/?term=Bakshi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124586975984" co-class="co-affbox">Andrew Bakshi</a>,<sup>a</sup> <a href="/pubmed/?term=Zheng%20DD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124586973872" co-class="co-affbox">Dong-Dong Zheng</a>,<sup>b</sup> <a href="/pubmed/?term=Zheng%20MQ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124587040080" co-class="co-affbox">Mei-Qin Zheng</a>,<sup>b</sup> <a href="/pubmed/?term=Wray%20NR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124583435584" co-class="co-affbox">Naomi R. Wray</a>,<sup>a</sup> <a href="/pubmed/?term=Visscher%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124592033168" co-class="co-affbox">Peter M. Visscher</a>,<sup>a</sup> <a href="/pubmed/?term=Lu%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124592031056" co-class="co-affbox">Fan Lu</a>,<sup>b,</sup><sup>2</sup> and  <a href="/pubmed/?term=Qu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541" class="affpopup" co-rid="_co_idm140124593955552" co-class="co-affbox">Jia Qu</a><sup>b,</sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140124590428896"><h3 class="no_margin">Jian Yang</h3><p><sup>a</sup>Institute for Molecular Bioscience, Queensland Brain Institute, The University of Queensland, Brisbane, QLD 4072, Australia;</p><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Yang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Jian Yang</a></div></div><div id="_co_idm140124585923488"><h3 class="no_margin">Zi-Bing Jin</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Jin%20ZB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Zi-Bing Jin</a></div></div><div id="_co_idm140124591266976"><h3 class="no_margin">Jie Chen</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Chen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Jie Chen</a></div></div><div id="_co_idm140124586880960"><h3 class="no_margin">Xiu-Feng Huang</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Huang%20XF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Xiu-Feng Huang</a></div></div><div id="_co_idm140124592370736"><h3 class="no_margin">Xiao-Man Li</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Li%20XM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Xiao-Man Li</a></div></div><div id="_co_idm140124589737024"><h3 class="no_margin">Yuan-Bo Liang</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Liang%20YB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Yuan-Bo Liang</a></div></div><div id="_co_idm140124589734912"><h3 class="no_margin">Jian-Yang Mao</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Mao%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Jian-Yang Mao</a></div></div><div id="_co_idm140124586893424"><h3 class="no_margin">Xin Chen</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Chen%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Xin Chen</a></div></div><div id="_co_idm140124592827232"><h3 class="no_margin">Zhili Zheng</h3><p><sup>a</sup>Institute for Molecular Bioscience, Queensland Brain Institute, The University of Queensland, Brisbane, QLD 4072, Australia;</p><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Zheng%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Zhili Zheng</a></div></div><div id="_co_idm140124586975984"><h3 class="no_margin">Andrew Bakshi</h3><p><sup>a</sup>Institute for Molecular Bioscience, Queensland Brain Institute, The University of Queensland, Brisbane, QLD 4072, Australia;</p><div>Find articles by <a href="/pubmed/?term=Bakshi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Andrew Bakshi</a></div></div><div id="_co_idm140124586973872"><h3 class="no_margin">Dong-Dong Zheng</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Zheng%20DD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Dong-Dong Zheng</a></div></div><div id="_co_idm140124587040080"><h3 class="no_margin">Mei-Qin Zheng</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Zheng%20MQ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Mei-Qin Zheng</a></div></div><div id="_co_idm140124583435584"><h3 class="no_margin">Naomi R. Wray</h3><p><sup>a</sup>Institute for Molecular Bioscience, Queensland Brain Institute, The University of Queensland, Brisbane, QLD 4072, Australia;</p><div>Find articles by <a href="/pubmed/?term=Wray%20NR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Naomi R. Wray</a></div></div><div id="_co_idm140124592033168"><h3 class="no_margin">Peter M. Visscher</h3><p><sup>a</sup>Institute for Molecular Bioscience, Queensland Brain Institute, The University of Queensland, Brisbane, QLD 4072, Australia;</p><div>Find articles by <a href="/pubmed/?term=Visscher%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Peter M. Visscher</a></div></div><div id="_co_idm140124592031056"><h3 class="no_margin">Fan Lu</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Lu%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Fan Lu</a></div></div><div id="_co_idm140124593955552"><h3 class="no_margin">Jia Qu</h3><p><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</p><div>Find articles by <a href="/pubmed/?term=Qu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28373541">Jia Qu</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140124595103872_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140124595103872_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140124595103872_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>a</sup>Institute for Molecular Bioscience, Queensland Brain Institute, The University of Queensland, Brisbane, QLD 4072, Australia;</div><div class="fm-affl" id="aff2" lang="en"><sup>b</sup>The Eye Hospital, School of Ophthalmology &amp; Optometry, Wenzhou Medical University, China National Engineering Research Center of Ophthalmology and Optometry, State Key Laboratory Cultivation Base and Key Laboratory of Vision Science, Ministry of Health, Wenzhou 325027, China</div><div id="cor1"><sup>2</sup>To whom correspondence may be addressed. Email: <a href="mailto:dev@null" data-email="nc.ca.eye.liam@uqj" class="oemail">nc.ca.eye.liam@uqj</a>, <a href="mailto:dev@null" data-email="nc.ca.eye.liam@naful" class="oemail">nc.ca.eye.liam@naful</a>, <a href="mailto:dev@null" data-email="nc.ca.eye.liam@bznij" class="oemail">nc.ca.eye.liam@bznij</a>, or <a href="mailto:dev@null" data-email="ua.ude.qu@gnay.naij" class="oemail">ua.ude.qu@gnay.naij</a>.</div><div id="idm140124589981424">Edited by Marcus W. Feldman, Stanford University, Stanford, CA, and approved February 13, 2017 (received for review October 14, 2016)</div><div><p id="__p2" class="p p-first-last">Author contributions: J.Y., Z.-B.J., F.L., and J.Q. designed research; Z.-B.J., J.C., X.-F.H., X.-M.L., Y.-B.L., J.-Y.M., X.C., D.-D.Z., M.-Q.Z., F.L., and J.Q. performed research; J.Y., Z.Z., A.B., N.R.W., and P.M.V. analyzed data; F.L. and J.Q. jointly supervised the study; and J.Y. and Z.-B.J. wrote the paper.</p></div><div id="fn1"><sup>1</sup>J.Y. and Z.-B.J. contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140124595103872_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div class="license half_rhythm">Freely available online through the PNAS open access option.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5402460/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary File.</strong></div><div><a href="/pmc/articles/PMC5402460/bin/pnas.1617042114.sapp.pdf" data-ga-action="click_feat_suppl">pnas.1617042114.sapp.pdf</a><span style="color:gray"> (6.7M)</span></div><div class="small guid">GUID:&nbsp;30FE30D6-F413-447B-B1B8-29A38F893C9A</div></div></dd></dl></div><div id="idm140124589669648" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140124589669648title" style="text-transform: uppercase;">Significance</h2><!--article-meta--><div><p id="__p4" class="p p-first-last">The origin of Tibetans and the mechanism of how they adapted to the high-altitude environment remain mostly unknown. We conduct the largest genome-wide study in Tibetans to date. We detect signatures of natural selection at nine gene loci, two of which are strongly associated with blood phenotypes in present day Tibetans. We further show the genetic relatedness of Tibetans with other ethnic groups in China and estimate the divergence time between Tibetans and Han. These findings provide important knowledge to understand the genetic ancestry of Tibetans and the genetic basis of high-altitude adaptation.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">high-altitude adaptation, Tibetans, genome-wide association study, mixed linear model, polygenic selection</span></div></div><div id="idm140124667262720" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140124667262720title" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p id="__p5" class="p p-first-last">Indigenous Tibetan people have lived on the Tibetan Plateau for millennia. There is a long-standing question about the genetic basis of high-altitude adaptation in Tibetans. We conduct a genome-wide study of 7.3 million genotyped and imputed SNPs of 3,008 Tibetans and 7,287 non-Tibetan individuals of Eastern Asian ancestry. Using this large dataset, we detect signals of high-altitude adaptation at nine genomic loci, of which seven are unique. The alleles under natural selection at two of these loci [methylenetetrahydrofolate reductase (<em>MTHFR</em>) and <em>EPAS1</em>] are strongly associated with blood-related phenotypes, such as hemoglobin, homocysteine, and folate in Tibetans. The folate-increasing allele of rs1801133 at the <em>MTHFR</em> locus has an increased frequency in Tibetans more than expected under a drift model, which is probably a consequence of adaptation to high UV radiation. These findings provide important insights into understanding the genomic consequences of high-altitude adaptation in Tibetans.</p></div></div><div id="idm140124587930336" class="tsec sec headless whole_rhythm"><p id="__p6" class="p p-first">Genetic adaptation to a novel environment is a fundamental process for the survival and adaptation of a species. In humans, one of the most recent examples is adaptation to high altitude, such as the Tibetan highlands. The Tibetan Plateau (TP; also known as the Qinghai–Tibet Plateau in China) has an average elevation of ∼4,000 m above sea level, where the oxygen concentration is ∼40% lower (<a href="#r1" rid="r1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110493">1</a>) and UV radiation is ∼30% stronger (<a href="#r2" rid="r2" class=" bibr popnode">2</a>) than at sea level. The indigenous Tibetan people have developed a distinctive set of physiological characteristics to adapt to the extreme environmental conditions in the highlands (<a href="#r1" rid="r1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110494">1</a>). Previous population-based genetic studies have reported evidence that genetic variants at the <em>EPAS1</em> and <em>EGLN1</em> loci have been under positive natural selection (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110486">3</a><a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110509"></a><a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110503"></a><a href="#r6" rid="r6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110457"></a>–<a href="#r7" rid="r7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110537">7</a>). These genetic variants are associated with phenotypic variation of hemoglobin concentration (HGB) in Tibetans (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110484">3</a><a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110458"></a>–<a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110467">5</a>). The <em>EPAS1</em> gene, which encodes the hypoxia inducible factor-2α (HIF-2α) subunit of HIF complex, is a transcription factor involved in body response to hypoxia (<a href="#r8" rid="r8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110468">8</a>, <a href="#r9" rid="r9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110449">9</a>). <em>EGLN1</em> encodes PHD2, which is a major oxygen-dependent negative regulator of HIFs (<a href="#r10" rid="r10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110492">10</a>, <a href="#r11" rid="r11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110500">11</a>). Apart from these two known genes that have biological relevance to hypoxia adaptation (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110522">3</a><a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110542"></a><a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110515"></a><a href="#r6" rid="r6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110483"></a>–<a href="#r7" rid="r7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110501">7</a>, <a href="#r12" rid="r12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110465">12</a>), several other candidate gene loci (e.g., <em>PPARA</em> and <em>HBB</em>) have been highlighted in recent studies (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110510">3</a>, <a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110534">4</a>, <a href="#r13" rid="r13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110506">13</a><a href="#r14" rid="r14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110469"></a>–<a href="#r15" rid="r15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110459">15</a>). Genetic adaptation to high altitude, however, is likely to be a complex process, with a large number of genes involved in response to not only hypoxia but also, other extreme environmental conditions, such as low temperature, high UV radiation, and insufficient food supply. If the strength of natural selection at these gene loci has been small to moderate, these loci would not be detected in previous studies (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110517">3</a><a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110536"></a><a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110481"></a><a href="#r6" rid="r6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110479"></a>–<a href="#r7" rid="r7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110547">7</a>) of small sample size (typically <em>n</em> &lt; 150). In this study, we perform a large-scale genome-wide study to detect genetic signals of high-altitude adaptation in 3,008 Tibetans and 7,287 non-Tibetan individuals of Eastern Asian (EAS) ancestry. Using this large dataset, we identify signals of genetic adaptation.</p></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title" style="text-transform: uppercase;">Results</h2><div id="s2" class="sec sec-first"><p></p><h3 id="s2title" class="inline">Genetic Ancestry of Tibetans. </h3><p id="__p7" class="p p-first">There were 3,717 subjects collected from two sites (Seda and Litang) in the TP in China (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S1</a>). We extracted DNA from blood samples and performed genome-wide SNP genotyping assays using the Illumina CoreExome array, an SNP array with 264,909 tag SNPs with genome-wide coverage and 244,593 exome-focused SNPs (<a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a>). After standard quality control (QC) filtering of the genotype data, we retained 3,381 subjects and 287,691 SNPs (279,608 on autosomes), most of which were genome-wide tag SNPs.</p><p id="__p8">We performed a principal component analysis (PCA) of the subjects using all 279,608 autosomal SNPs after stringent QCs (<a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a>). There was no evidence of population stratification between the cohorts recruited from the two sites (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S2<em>A</em></a>), despite the fact that the Seda subjects were recruited from people who came from diverse regions of the TP to study or work at the Seda Larong Wuming Buddhist Institute and the Litang subjects were recruited from nomadic people who have lived in Litang and surrounding areas for many generations (<a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a>). We, therefore, combined the two cohorts for analysis. We showed by projecting the principal components (PCs) estimated from our samples on those from the 1000 Genome Projects (1000G) that all of our subjects were of EAS ancestry (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S2<em>B</em></a>). On a finer scale, the subjects are stratified along the first PC (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S2<em>C</em></a>), consistent with a few hundred self-reported Han in the sample. We classified our subjects into three groups (Tibetans, Han, and possibly admixed) (<a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a> and <a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S2<em>D</em></a>) and removed the possibly admixed subjects. There were 3,008 Tibetans and 373 Han retained for analysis.</p><p id="__p9">We projected the PCs of our subjects on the Chinese subjects from the Human Diversity Genome Project (HGDP) (<a href="#r16" rid="r16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110448">16</a>) and illustrated the genetic relatedness between Tibetans and other ethnic groups in China (<a href="/pmc/articles/PMC5402460/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>A</em></span></a>). Our result suggests that Tibetans show the nearest genetic relatedness to Yi, Tu, and Naxi ethnic minority populations (<a href="/pmc/articles/PMC5402460/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>A</em></span></a> and <a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Table S1</a>), consistent with these populations who reside in the neighboring regions of the TP (Yi and Naxi people are mainly distributed in Yunnan and Sichuan provinces, and most Tu people reside in Qinghai province) (<a href="/pmc/articles/PMC5402460/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>B</em></span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig01" co-legend-rid="lgnd_fig01"><a href="/pmc/articles/PMC5402460/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140124595619264" class="figure"><a href="/pmc/articles/PMC5402460/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5402460_pnas.1617042114fig01.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pnas.1617042114fig01.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5402460/bin/pnas.1617042114fig01.jpg"></a></div><div id="largeobj_idm140124595619264" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5402460/figure/fig01/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig01"><div><a class="figpopup" href="/pmc/articles/PMC5402460/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01">Fig. 1.</a></div><!--caption a7--><div class="caption"><p id="__p10">PCA of genetic ancestry in Chinese populations using genome-wide SNP data. (<em>A</em>) Result from a PCA in a combined sample of 3,381 genetically confirmed Tibetan and Han from this study and 180 Chinese subjects (multiple ethnic groups) from the HGDP. PC1 and PC2 represent the first two eigenvectors from PCA. Note that one of the Yi subjects from the HGDP seems to be of Tibetan ancestry. (<em>B</em>) Distribution of the ethnic groups in China. The blue circles represent the main distribution areas of the ethnic populations in the HGDP, and the red circle represents the Tibetan population. Note that many of the populations, such as Han, Mongola, Tibetan, and Uygur, are distributed widely in a range of regions rather than the specific areas labeled on the map. The green triangles represent the two areas (Seda and Litang) from which our Tibetan subjects were recruited (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S1</a>).</p></div></div></div><p id="__p11" class="p p-last">We estimated the divergence time between Tibetan and Han populations using the conventional <em>F</em><sub>ST</sub>-based approach (<a href="#r17" rid="r17" class=" bibr popnode">17</a>) (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Text S1</a>). As described above, there were 3,008 Tibetan and 373 Han subjects collected from the TP after QC. We included in this analysis an additional set of 1,726 Han subjects collected from the Eye Hospital of Wenzhou Medical University (WZ) after QC (<a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a>). We used GCTA-GRM to remove cryptic relatedness in the Tibetan and Han samples (note that the Han sample was a combined set of 373 Han subjects from the TP and 1,726 Han subjects from WZ) at a relatedness threshold of 0.05 and retained 1,998 unrelated Tibetan and 2,059 unrelated Han subjects. There was no genetic difference between WZ-Han and TP-Han as shown by PCA (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S3</a>), probably because most of the Han subjects, collected from either TP or WZ, were originally from diverse regions of China. The genome-wide mean <em>F</em><sub>ST</sub> between Tibetans and Han was 0.012 [using the method by Weir and Cockerham (<a href="#r18" rid="r18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_763246809">18</a>) implemented in GCTA], consistent with the estimate of the Han subjects from the HGDP (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Table S1</a>). Given the genome-wide mean <em>F</em><sub>ST</sub> value (<a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a>), we estimated that the divergence time between Tibetan and Han populations was 189 generations. Assuming an average generation time of 25 y as in previous studies (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110529">3</a>, <a href="#r19" rid="r19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110531">19</a>), this estimate suggests that Tibetans and Han split about 4,725 y ago, ∼2,000 y earlier than that estimated from whole-exome sequencing data (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110472">3</a>) but consistent with recent evidence from archeological studies (<a href="#r20" rid="r20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110499">20</a>, <a href="#r21" rid="r21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110487">21</a>).</p></div><div id="s3" class="sec"><p></p><h3 id="s3title" class="inline">Genome-Wide Analysis to Detect Genetic Signals of Adaptation. </h3><p id="__p12" class="p p-first">To detect genetic signals of high-altitude adaptation, we used a mixed linear model-based leave one chromosome out association (MLMA-LOCO) analysis approach [implemented in the BOLT-LMM software tool (<a href="#r22" rid="r22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110455">22</a>)] to test for allele frequency difference between Tibetans and non-Tibetans of EAS ancestry (<a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a>). We investigated the statistical properties of the method using simulations (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Table S2</a>). Similar approaches have been used in genome-wide association studies (GWASs) to control for population structure (<a href="#r22" rid="r22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110511">22</a>, <a href="#r23" rid="r23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110453">23</a>). In the MLMA-LOCO model, the target SNP to be tested is fitted as a fixed effect, and all SNPs on the other chromosomes are fitted as random effects (details about the model are in <a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a>). The underlying assumption is that, under a drift model, the random effects follow a normal distribution with the variance being proportional to <em>p</em><sub>0</sub>(1 – <em>p</em><sub>0</sub>)<em>F</em><sub>ST</sub>, where <em>p</em><sub>0</sub> is the allele frequency in the ancestral population and <em>F</em><sub>ST</sub> is the Wright’s fixation index between the two derived populations (<a href="#r24" rid="r24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110527">24</a>). If there are two diverged populations in the sample, even if neither of the populations have been under natural selection, SNPs on different chromosomes will be correlated because of the systematic difference in allele frequency between populations caused by cryptic relatedness in the samples, genetic drift, and/or possibly, admixture with other populations (see below for examples). We, therefore, can correct for the interchromosome correlations by modeling all of the SNPs on the other chromosomes (as random effects, because number of SNPs is usually larger than sample size) when testing for the association of an SNP. To maximize power, we included in the analysis all of the subjects collected from the TP and WZ in China (3,008 Tibetans and 2,099 Han) and an additional set of 5,188 subjects of EAS ancestry from the Genetic Epidemiology Research on Aging (GERA) Study (<a href="#r25" rid="r25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110498">25</a>) in the United States (<a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a>). Because the GERA-EAS subjects were genotyped on a different SNP array (Affymetrix Axiom), we imputed all of the genotype data to 1000G reference panels using IMPUTE2 (<a href="#r26" rid="r26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110525">26</a>). There were three ancestry outliers, which were excluded from analysis (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S4</a>). To exclude SNPs with allele frequency differences between cohorts caused by potential batch effects, we performed a “control–control” analysis using the MLMA-LOCO approach to test for difference in allele frequency between TP-Han and a combined set of WZ-Han and GERA-EAS and removed SNPs with <em>P</em> value &lt; 1 × 10<sup>−6</sup>. We then performed a “case–control” analysis using the MLMA-LOCO approach to test for difference in allele frequency between Tibetans (“cases”: <em>n</em> = 3,008) and EAS subjects (“controls”: TP-Han, WZ-Han, and GERA-EAS; <em>n</em> = 7,287) and identified nine loci that passed the genome-wide significance level (<em>P</em><sub>MLMA-LOCO</sub> &lt; 5e-8) (<a href="/pmc/articles/PMC5402460/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a> and <a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S5</a>). Of nine loci, two loci, <em>EPAS1</em> and <em>EGLN1</em>, which show the strongest signals in our analysis, are known (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110473">3</a><a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110544"></a><a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110464"></a><a href="#r6" rid="r6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110520"></a>–<a href="#r7" rid="r7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110545">7</a>), and the other seven loci are unique (<a href="/pmc/articles/PMC5402460/table/t01/" target="table" class="fig-table-link figpopup" rid-figpopup="t01" rid-ob="ob-t01" co-legend-rid=""><span>Table 1</span></a> and <a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S6</a>). Note that <em>FGF10</em> was one of a set of genes that showed large population branch statistic (PBS) values (Tibetans vs. Han vs. Europeans) in a recent study (<a href="#r15" rid="r15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110523">15</a>). We show by linkage disequilibrium (LD) score regression analysis (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Text S2</a>) that there is no inflation in the test statistic (an estimate of the regression intercept of 0.99 with an SE of 0.01, which is not significantly different from 1), suggesting that the sample structure has been well-controlled in the MLMA-LOCO analysis as expected from theory (<a href="#r23" rid="r23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110539">23</a>). We further divided the data into the Seda and Litang subsets and reran the analysis in each subset (<a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a>). Although all of nine loci remained highly significant, not all of them passed the genome-wide significance level in either subset (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Table S3</a>). This analysis shows the gain of power for detecting genetic signals of natural selection in a dataset of large sample size. In addition, we performed conditional analyses (<a href="#r27" rid="r27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110491">27</a>, <a href="#r28" rid="r28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110518">28</a>) at nine genome-wide significant loci and did not find evidence of multiple signals at any of these loci. We also performed the MLMA-LOCO analysis to detect signatures of genetic adaptation on the mitochondrial genome and did not observe any significant signal (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S7</a>). We replicated a number of candidate gene loci as reported in previous studies (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110548">3</a>, <a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110538">4</a>). The replication rate after correcting for multiple testing was ∼35.7% (= 5/14), much higher than expected by chance (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Table S4</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig02" co-legend-rid="lgnd_fig02"><a href="/pmc/articles/PMC5402460/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140124596170032" class="figure"><a href="/pmc/articles/PMC5402460/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5402460_pnas.1617042114fig02.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pnas.1617042114fig02.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5402460/bin/pnas.1617042114fig02.jpg"></a></div><div id="largeobj_idm140124596170032" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5402460/figure/fig02/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig02"><div><a class="figpopup" href="/pmc/articles/PMC5402460/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02">Fig. 2.</a></div><!--caption a7--><div class="caption"><p id="__p13">Genome-wide scan for genetic signatures of adaptation. Shown on the <em>y</em> axis are −log10 of <em>P</em> values from the tests of allele frequency difference between Tibetan Chinese (<em>n</em> = 3,008) and EASs (<em>n</em> = 7,287). The analysis was performed using the MLMA-LOCO method, which tests for difference in allele frequency between populations taking into account the difference caused by random drift. SNPs at the genome-wide significant loci are highlighted in red.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t01"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p14">Nine genetic loci with signals of natural selection</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140124584280720" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="2" colspan="1">Chromosome</td><td rowspan="2" colspan="1" align="center">SNP</td><td rowspan="2" colspan="1" align="center">bp</td><td rowspan="2" colspan="1" align="center">A1</td><td rowspan="2" colspan="1" align="center">A2</td><td colspan="2" rowspan="1" align="center">Frequency of A1</td><td rowspan="2" colspan="1" align="center"><em>P</em> value</td><td rowspan="2" colspan="1" align="center"><em>F</em><sub>ST</sub></td><td rowspan="2" colspan="1" align="center">Nearest gene</td></tr><tr><td rowspan="1" colspan="1" align="center">Tibetan</td><td rowspan="1" colspan="1" align="center">EAS</td></tr></thead><tbody><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1" align="center">rs1801133</td><td rowspan="1" colspan="1" align="center">11,856,378</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.238</td><td rowspan="1" colspan="1" align="center">0.333</td><td rowspan="1" colspan="1" align="center">6.3E-09</td><td rowspan="1" colspan="1" align="center">0.021</td><td rowspan="1" colspan="1" align="center"><em>MTHFR</em></td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1" align="center">rs71673426</td><td rowspan="1" colspan="1" align="center">112,159,304</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">0.102</td><td rowspan="1" colspan="1" align="center">0.013</td><td rowspan="1" colspan="1" align="center">1.5E-08</td><td rowspan="1" colspan="1" align="center">0.100</td><td rowspan="1" colspan="1" align="center"><em>RAP1A</em></td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1" align="center">rs78720557</td><td rowspan="1" colspan="1" align="center">198,096,548</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">0.498</td><td rowspan="1" colspan="1" align="center">0.201</td><td rowspan="1" colspan="1" align="center">4.7E-08</td><td rowspan="1" colspan="1" align="center">0.191</td><td rowspan="1" colspan="1" align="center"><em>NEK7</em></td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1" align="center">rs78561501</td><td rowspan="1" colspan="1" align="center">231,448,497</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.599</td><td rowspan="1" colspan="1" align="center">0.135</td><td rowspan="1" colspan="1" align="center">6.1E-15</td><td rowspan="1" colspan="1" align="center">0.414</td><td rowspan="1" colspan="1" align="center"><em>EGLN1</em></td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1" align="center">rs116611511</td><td rowspan="1" colspan="1" align="center">46,600,030</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.447</td><td rowspan="1" colspan="1" align="center">0.003</td><td rowspan="1" colspan="1" align="center">3.6E-19</td><td rowspan="1" colspan="1" align="center">0.570</td><td rowspan="1" colspan="1" align="center"><em>EPAS1</em></td></tr><tr><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1" align="center">rs2584462</td><td rowspan="1" colspan="1" align="center">100,324,464</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.211</td><td rowspan="1" colspan="1" align="center">0.549</td><td rowspan="1" colspan="1" align="center">3.9E-09</td><td rowspan="1" colspan="1" align="center">0.203</td><td rowspan="1" colspan="1" align="center"><em>ADH7</em></td></tr><tr><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1" align="center">rs4498258</td><td rowspan="1" colspan="1" align="center">44,325,322</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.586</td><td rowspan="1" colspan="1" align="center">0.287</td><td rowspan="1" colspan="1" align="center">1.7E-08</td><td rowspan="1" colspan="1" align="center">0.171</td><td rowspan="1" colspan="1" align="center"><em>FGF10</em></td></tr><tr><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1" align="center">rs9275281</td><td rowspan="1" colspan="1" align="center">32,662,920</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.095</td><td rowspan="1" colspan="1" align="center">0.365</td><td rowspan="1" colspan="1" align="center">1.1E-10</td><td rowspan="1" colspan="1" align="center">0.162</td><td rowspan="1" colspan="1" align="center"><em>HLA-DQB1</em></td></tr><tr><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1" align="center">rs139129572</td><td rowspan="1" colspan="1" align="center">123,178,478</td><td rowspan="1" colspan="1" align="center">GA</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.316</td><td rowspan="1" colspan="1" align="center">0.449</td><td rowspan="1" colspan="1" align="center">5.8E-09</td><td rowspan="1" colspan="1" align="center">0.036</td><td rowspan="1" colspan="1" align="center"><em>HCAR2</em></td></tr></tbody></table></div><div id="largeobj_idm140124584280720" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5402460/table/t01/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140124588169952"><p id="__p15" class="p p-first-last"><em>P</em> value indicates the <em>P</em> value from the MLMA-LOCO analysis. <em>F</em><sub>ST</sub> is the <em>F</em><sub>ST</sub> value between Tibetans and EASs. Nearest gene indicates the nearest annotated gene to the top differentiated SNP at each locus except <em>EGLN1</em>, which is known to be associated with high-altitude adaptation; rs139129572 is an insertion SNP with two alleles: GA and G. A1, allele 1; A2, allele 2.</p></div></div></div></div><div id="s4" class="sec sec-last"><p></p><h3 id="s4title" class="inline">Associations of the Loci Under Natural Selection with Phenotypes in Tibetans. </h3><p id="__p16" class="p p-first-last">Having identified nine genetic loci that have been under natural selection, we then asked whether these loci are associated with any phenotypes in Tibetans (<em>n</em> = 2,849). There were 91 quantitative traits measured on the Tibetan subjects (<a href="#s6" rid="s6" class=" sec"><em>Materials and Methods</em></a>), mainly morphological, blood biochemistry, and optometric measures (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Table S5</a>). The phenotypic correlation matrix of these traits is shown in <a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S8</a>. Most of the traits were highly heritable, with a substantial proportion of phenotypic variance explained by all SNPs in unrelated individuals (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S9 and Table S6</a>). We then performed GWAS analysis in Tibetans using the MLMA-LOCO approach described above to control for population structure. We found that the methylenetetrahydrofolate reductase (<em>MTHFR</em>) and <em>EPAS1</em> loci were associated with multiple traits (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S10</a>), and five of these traits were significant after correcting for multiple testing (<em>P</em><sub>GWAS</sub> &lt; 1.5 × 10<sup>−4</sup>) (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Table S7</a>). The <em>MTHFR</em> locus was strongly associated with folate (<em>b</em> = −0.34, <em>P</em><sub>GWAS</sub> = 6.5 × 10<sup>−27</sup>) and homocysteine (<em>b</em> = 0.54, <em>P</em><sub>GWAS</sub> = 1.1 × 10<sup>−69</sup>), where <em>b</em> is the effect size in SD units. This locus is known to be associated with homocysteine in Europeans (<a href="#r29" rid="r29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110521">29</a>). MTHFR is a key enzyme involved in the metabolic pathway of homocysteine and folate (<a href="#r30" rid="r30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110452">30</a>). The frequency of homocysteine-increasing allele was lower in Tibetans (0.238) than that in EAS (0.333) (<a href="/pmc/articles/PMC5402460/table/t01/" target="table" class="fig-table-link figpopup" rid-figpopup="t01" rid-ob="ob-t01" co-legend-rid=""><span>Table 1</span></a>), in line with homocysteine level in Tibetans (mean = 21.8, SE = 0.3) being lower than in Han (mean = 25.5, SE = 1.4), where SE represents SEM estimate. <em>EPAS1</em> is known to be associated with HGB (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110502">3</a>, <a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110454">5</a>). Our results suggest that <em>EPAS1</em> is strongly associated with HGB, red blood cell count, and hematocrit (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Table S7</a>) and that the HGB-decreasing allele is under very strong positive selection in Tibetans, with a frequency of 0.45 in Tibetans vs. 0.003 in EASs (<a href="/pmc/articles/PMC5402460/table/t01/" target="table" class="fig-table-link figpopup" rid-figpopup="t01" rid-ob="ob-t01" co-legend-rid=""><span>Table 1</span></a>). It is also interesting to note that the <em>ADH7</em> locus is associated with weight and body mass index (BMI) in Tibetans (<em>P</em><sub>GWAS</sub> = 7.1 × 10<sup>−4</sup> and <em>P</em><sub>GWAS</sub> = 4.9 × 10<sup>−4</sup>, respectively), although <em>ADH7</em> is not a known BMI-associated locus in Europeans (<a href="#r31" rid="r31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110535">31</a>). However, the associations are not significant after correcting for multiple testing. The <em>EGLN1</em> locus has been previously reported to be associated with HGB (<a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110489">4</a>). We found that the association between <em>EGLN1</em> and HGB was very weak (<em>P</em><sub>GWAS</sub> = 0.02,, not significant after correcting for multiple testing), and the effect size was stronger in males (−0.112 in SD unit, SE = 0.046) than in females [−0.037, SE = 0.036, <em>P</em><sub>difference</sub> = 0.01, consistent with the result from a previous study (<a href="#r12" rid="r12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110471">12</a>)].</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title" style="text-transform: uppercase;">Discussion</h2><p id="__p17" class="p p-first">We have performed a large-scale genetic study in 3,008 Tibetans and 7,287 non-Tibetans of EAS ancestry. We showed the genetic relatedness between Tibetans and a number of other ethnic groups in China and found that Yi, Tu, and Naxi people are genetically intermediate between Han and Tibetans (<a href="/pmc/articles/PMC5402460/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>A</em></span></a>). These people are also geographically distributed between major residential areas of Han and Tibetans (Yi, Tu, and Naxi people reside at the eastern border of the TP) (<a href="/pmc/articles/PMC5402460/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1<em>B</em></span></a>), suggesting potential routes of people migrating from the east to the TP. There has not been a consensus on the divergence time between Tibetans and Han (<a href="#r32" rid="r32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110497">32</a>). The estimates from different genetic studies are often inconsistent [varying from 2,750 (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110476">3</a>) to ∼8,000 (<a href="#r11" rid="r11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110532">11</a>, <a href="#r12" rid="r12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110504">12</a>) and ∼30,000 y B.P. (<a href="#r33" rid="r33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110541">33</a>)], even for studies using the same method [9,000–15,000 (<a href="#r34" rid="r34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110488">34</a>) vs. 20,000–40,000 y B.P. (<a href="#r35" rid="r35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110463">35</a>)]. Our estimate from, so far, the largest genetic data of Tibetans is that the divergence time between Tibetans and Han was ∼4,725 y B.P., which is consistent with the estimated permanent settlement time of ∼3,750–6,500 y B.P. from archaeological studies (<a href="#r20" rid="r20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110460">20</a>). Interestingly, a recent study (<a href="#r21" rid="r21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110482">21</a>) that investigates archaeological crop remains unearthed in the northeastern TP estimated that the first village was established 5,200 y B.P., which is highly concordant with our estimate. However, there is an important caveat in interpreting estimates from population genetics analyses. That is, if there is a constant gene flow from the founder population to the TP after initial settlement, then the estimate of divergence from a population genetic analysis will be biased downward. Therefore, our estimate should be interpreted as a lower limit of the permanent settlement time, implying that the actual settlement time of people in the TP is likely earlier than 4,725 y B.P.</p><p id="__p18">We applied the MLMA-LOCO method (<a href="#r27" rid="r27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110451">27</a>) as implemented in BOLT-LMM (<a href="#r22" rid="r22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110540">22</a>) to detect genetic signals of selection. Compared with the prevailing methods (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110519">3</a>, <a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110528">5</a>, <a href="#r6" rid="r6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110475">6</a>), the MLMA-LOCO performs statistical tests at a genome-wide significance level, controlling for locus-specific population differentiation and potential relatedness in the sample. It is expected that the analysis using unrelated individuals was, on average, less powerful than using all of the individuals, but overall, the results are highly consistent (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S11</a>). We show below an example of how the MLMA-LOCO controls for locus-specific population differentiation. There were three SNPs (on chromosomes 9, 20, and 22) that showed strong signals in linear regression (<a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110456">5</a>), <em>F</em><sub>ST</sub> (<a href="#r18" rid="r18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_763246808">18</a>, <a href="#r24" rid="r24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110514">24</a>, <a href="#r36" rid="r36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110546">36</a>, <a href="#r37" rid="r37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110470">37</a>), or PBS (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110496">3</a>) analysis but did not reach the genome-wide significance level in the MLMA-LOCO analysis (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S12</a>) because the three SNPs are located in regions with strong locus-specific population differentiation (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S13</a>). Using the MLMA-LOCO method, we identified nine gene loci that have been under selection as a consequence of adaptation to the high altitude (<a href="/pmc/articles/PMC5402460/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a> and <a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S5</a>), seven of which are unique. It is noteworthy that there are surprisingly few loci that have been identified given the large sample size of this study, consistent with a model of polygenic adaptation (<a href="#r38" rid="r38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110480">38</a>). The two known loci (<em>EPAS1</em> and <em>EGLN1</em>) showed the strongest signals in our analysis. The top signal (<em>EPAS1</em>) remained highly significant in the analysis of a small subset of data (150 Tibetans vs. 150 Han) (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S14</a>), which explains why the <em>EPAS1</em> locus can be detected in previous studies of small sample size (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110507">3</a>, <a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110512">5</a>, <a href="#r6" rid="r6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110462">6</a>). We further found that genetic variants at three of these loci (<em>MTHFR</em>, <em>EPAS1</em>, and <em>ADH7</em>) were associated with several phenotypes in Tibetans, with <em>MTHFR</em> being associated with folate and homocysteine levels and <em>EPAS1</em> being associated with HGB and hematocrit at an experimentwise significance level (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S10</a>). In addition, it was suggested in a previous study (<a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110474">4</a>) that the <em>PPARA</em> gene locus is associated with high-altitude adaptation and HGB level in Tibetans. In our study, we found that the signal of selection at the <em>PPARA</em> was not genome-wide significant (<em>P</em><sub>MLMA-LOCO</sub> = 9.1 × 10<sup>−5</sup> at the top SNP rs149670586) and did not find any evidence that rs149670586 is associated with HGB (<em>P</em><sub>GWAS</sub> = 0.20). There is a caveat in interpreting the MLMA-LOCO results. We found evidence of natural selection at nine gene loci by comparing the allele frequencies between Tibetans and EASs under the null hypothesis that there is no natural selection but a population differentiation caused by genetic drift and possibly, admixture with other populations. This result, however, does not necessarily mean that the selection has to relate to hypoxia. It could be adaptation to any of the extreme environmental or pathological conditions in the TP. For example, the folate-increasing allele of the SNP rs1801133 at the <em>MTHFR</em> locus (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Table S7</a>) has an increased frequency in the Tibetan population, more than expected under a drift model (<a href="/pmc/articles/PMC5402460/table/t01/" target="table" class="fig-table-link figpopup" rid-figpopup="t01" rid-ob="ob-t01" co-legend-rid=""><span>Table 1</span></a>), which is possibly a consequence of adaptation to high UV radiation, because the degradation of folate could be accelerated by UV exposure (<a href="#r39" rid="r39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110478">39</a>).</p><p id="__p19" class="p p-last">In summary, we performed a large-scale genetic study in Tibetans. We showed the genetic relatedness between Tibetans and other ethnic groups in China and estimated divergence time between Tibetans and Han (4,725 y B.P.). We identified genetic signatures of high-altitude adaptation at seven gene loci. These findings provide important insight into understanding of how the Tibetan genome has changed during high-altitude adaptation.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title" style="text-transform: uppercase;">Materials and Methods</h2><div id="s7" class="sec sec-first"><p></p><h3 id="s7title" class="inline">Sample Collection and Genotyping. </h3><p id="__p20" class="p p-first-last">The subjects were recruited at two separate sites (Seda and Litang) of the TP in Sichuan, western China (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S1</a>). Both sites are ∼4,100 m above sea level. The Seda subjects were recruited from people who were studying or working at the Seda Larong Wuming Buddhist Institute. These subjects are originally from diverse regions of the TP. The Litang subjects were recruited from nomadic people who have lived in Litang and surrounding areas for many generations. All of the subjects were recruited following the protocol approved by the Ethics Committee of the WZ. An informed written consent was obtained from each subject participating in this study. There were a total of 3,996 subjects with blood samples (3,142 from Seda and 854 from Litang). Peripheral blood sample was obtained from each subject for extraction of genomic DNA (Simgen Blood DNA Mini Kit; Simgen). DNA concentrations were subsequently determined using a NanoDrop 1000 spectrophotmeter (Thermo Scientific). DNA samples were subjected to array genomic hybridization using the HumanCoreExome-12 BeadChip (Illumina Inc.). An iScan Reader was used to scan the array slide (Illumina Inc.). SNP genotypes were called using the GenCall algorithm implemented in GenomeStudio (GenTrain Score threshold = 0.15).</p></div><div id="s8" class="sec"><p></p><h3 id="s8title" class="inline">QC. </h3><p id="__p21" class="p p-first">There were 3,717 subjects genotyped on 542,585 SNPs (526,123 on autosomes) before QC. We removed SNPs and individuals with missingness rate &gt; 5% and excluded SNPs with minor allele frequency (MAF) &lt; 3.3 × 10<sup>−4</sup> [equivalent to minor allele count (MAC) &lt; 3] or Hardy–Weinberg Equilibrium (HWE) test <em>P</em> value &lt; 1.0 × 10<sup>−6</sup>. We flipped strand for the SNPs that were not called on the forward strand using the latest annotation file from Illumina (<a href="https://support.illumina.com/downloads/humancoreexome-12v1-1_product_support_files.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://support.illumina.com/downloads/humancoreexome-12v1-1_product_support_files.html</a>) and removed SNPs with alleles called from the subjects that were inconsistent with those in the annotation file. We then used GCTA-GRM (<a href="#r27" rid="r27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110516">27</a>) to estimate the genetic relatedness between pairwise individuals using all of the common (MAF ≥ 0.01) SNPs on autosomes after QC and removed one of each pair of individuals (the one with higher missingness rate) with estimated genetic relatedness &gt; 0.8 (all were duplicated subjects, except one self-reported monozygotic twin pair).</p><p id="__p22" class="p p-last">We performed a PCA (<a href="#r40" rid="r40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110533">40</a>, <a href="#r41" rid="r41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110485">41</a>) in the sample using GCTA-PCA (<a href="#r27" rid="r27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110450">27</a>) on all common autosomal SNPs after QC and projected the PCs to the subjects from the 1000G (<a href="#r42" rid="r42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110490">42</a>). Our study subjects were stratified along PC1 (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S2<em>A</em></a>), because there were a few hundred self-reported Han Chinese in the sample. This information was confirmed by projecting the PCs estimated from our sample on the Han Chinese subjects from the 1000 Genome Project (1000G-Han) (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S2<em>C</em></a>). We stratified our subjects into three groups (Han, Tibetans, and possibly admixed individuals) using the following method. (<em>i</em>) We classified the individuals whose PC1s were less than four SDs away from mean PC1 of 1000G-Han (mean = −0.039 and SD = 0.0044) as Han. (<em>ii</em>) We then fitted a two-component mixture normal distribution to PC1 and classified the individuals whose PC1s were less than four SDs away from the mean of the second mixture component (mean = 0.0073 and SD = 0.0015) as Tibetans. (<em>iii</em>) The rest of the subjects were classified as possibly admixed individuals (or individuals from the other minority groups) and excluded from subsequent analyses (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S2<em>D</em></a>). After all of the QC filters, we retain 3,381 individuals (3,008 Tibetan and 373 Han) and 287,691 SNPs (279,608 autosomal SNPs). We then imputed the cleaned genotype data to 1000G reference panels (phase 1) (<a href="#r42" rid="r42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110530">42</a>) using IMPUTE2 (<a href="#r43" rid="r43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110524">43</a>). After imputation, we removed monomorphic SNPs, SNPs with HWE <em>P</em> value &lt; 1 × 10<sup>−6</sup>, SNPs with IMPUTE-INFO &lt; 0.3, or SNPs with MAF &lt; 0.01 in Tibetans and retained ∼7.8 million SNPs for analysis.</p></div><div id="s9" class="sec"><p></p><h3 id="s9title" class="inline">GWAS Data of Han from Wenzhou and EASs from the Database of Genotypes and Phenotypes. </h3><p id="__p23" class="p p-first">We had access to a GWAS dataset of 2,043 Han Chinese subjects who were recruited at the WZ using the same protocol. The genotyping experiments were performed on the same genotyping platform (Illumina CoreExome array) as described above. We removed SNPs and subjects with missingness rate &gt; 5% and excluded SNPs with HWE <em>P</em> value &lt; 1 × 10<sup>−6</sup> or MAC &lt; 3. After QC, there were 1,726 subjects and 270,630 SNPs (263,345 autosomal SNPs). The cleaned genotype data were imputed to 1000G reference panels (phase 1) using IMPUTE2. After imputation, monomorphic SNPs or SNPs with HWE <em>P</em> &lt; 1 × 10<sup>−6</sup> or IMPUTE-INFO score &lt; 0.3 were excluded from analysis. Although the subjects were recruited at the WZ, the individuals came from various different regions of China as confirmed by the genetic data (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S3</a>).</p><p id="__p24" class="p p-last">We also had access to GWAS data of the GERA Study (<a href="#r25" rid="r25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110543">25</a>) in the United States through the database of Genotypes and Phenotypes (accession no. phs000674.v2.p2). There were 5,188 EAS subjects genotyped on Affymetrix Axiom arrays. QCs of the genotype data have been detailed elsewhere (<a href="#r25" rid="r25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110505">25</a>). We followed the QC protocol provided by the GERA Study, and we further removed SNPs and individuals with missingness rate &gt; 2% and excluded SNPs with HWE <em>P</em> value &lt; 1 × 10<sup>−6</sup> or MAC &lt; 3. We imputed the genotype data to all of the 1000G reference panel (Phase 1) using the same imputation pipeline as described above and removed SNPs with HWE <em>P</em> value &lt; 1 × 10<sup>−6</sup> or IMPUTE-INFO &lt; 0.3 postimputation.</p></div><div id="s10" class="sec sec-last"><p></p><h3 id="s10title" class="inline">Genome-Wide Analysis to Detect Genetic Signals of Selection. </h3><p id="__p25" class="p p-first">We performed a genome-wide analysis to detect signals of selection by comparing allele frequency of each SNP between two diverged populations (e.g., Tibetan and Han) based on the following model:</p><div class="disp-formula" id="uneq1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="me1" overflow="scroll"><mrow><mi>y</mi><mo>=</mo><mi>μ</mi><mo>+</mo><mi>x</mi><mi>b</mi><mo>+</mo><munder><mstyle displaystyle="true"><mo>∑</mo></mstyle><mi>k</mi></munder><msub><mi>w</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub><msub><mi>u</mi><mi>k</mi></msub><mo>+</mo><mi>e</mi><mo>,</mo></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="me1" overflow="scroll"><mrow><mi>y</mi><mo>=</mo><mi>μ</mi><mo>+</mo><mi>x</mi><mi>b</mi><mo>+</mo><munder><mstyle displaystyle="true"><mo>∑</mo></mstyle><mi>k</mi></munder><msub><mi>w</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub><msub><mi>u</mi><mi>k</mi></msub><mo>+</mo><mi>e</mi><mo>,</mo></mrow></math></script></div></div><p></p><p id="__p26">where <em>y</em> is coded as one or zero to indicate population for an individual (e.g., one for Tibetan and zero for Han), <em>μ</em> is a fixed mean term, <em>x</em> is a genotype variable (coded as zero, one, or two for the three genotypes) for an SNP, <em>b</em> is a fixed effect that is a function of allele frequency difference between the two populations, ∑<sub><em>k</em></sub><em>w</em><sub><em>i</em><em>k</em></sub><em>u</em><sub><em>k</em></sub> is a term that fits all of the SNPs on the other chromosomes in the model to control for population differentiation, <em>w</em><sub><em>k</em></sub> is the standardized genotype variable for an SNP <em>k</em>, <em>u</em><sub><em>k</em></sub> is the corresponding effect size [<em>u</em><sub><em>k</em></sub> follows a normal distribution with variance being proportional to <em>p</em><sub>0</sub>(1 – <em>p</em><sub>0</sub>)<em>F</em><sub>ST</sub>, where <em>p</em><sub>0</sub> is the allele frequency in the ancestral population], and <em>e</em> is the residual. In GWASs, this analysis is called the MLMA-LOCO (<a href="#r23" rid="r23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110461">23</a>). We have shown previously by theory and simulations that this method not only controls for population structure but also, gains power compared with linear regression (<a href="#r23" rid="r23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110526">23</a>). The MLMA-LOCO method has been implemented in the GCTA-MLMA (<a href="#r23" rid="r23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110508">23</a>, <a href="#r27" rid="r27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110477">27</a>) and BOLT-LMM (<a href="#r22" rid="r22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110495">22</a>) software tools. We used BOLT-LMM in this study, because it is computationally more efficient than GCTA-MLMA when sample size is large. If we do not include the random effect term in the model; it will become a linear regression analysis (i.e., <em>y</em> =&nbsp;<em>μ</em> +&nbsp;<em>x</em><em>b</em> +&nbsp;<em>e</em>), analogous to an <em>F</em><sub>ST</sub> analysis (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S12</a> has a comparison between linear regression and <em>F</em><sub>ST</sub> analysis). The test statistics from linear regression will be inflated because of the population differentiation caused by genetic drift. Post hoc correction approaches, such as “Genomic Control” (<a href="#r44" rid="r44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110513">44</a>), can be used to correct for the inflation; however, Genomic Control correction could lead to overcorrection (<a href="#r23" rid="r23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_604110466">23</a>) (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Table S2</a>) and is not able to account for potential locus-specific differentiation (<a href="#d35e442" rid="d35e442" class=" supplementary-material"><em>SI Appendix</em>, Fig. S13</a>).</p><p id="__p27" class="p p-last">For data analysis, we used all of the Tibetan and Han subjects collected from the TP and WZ and all of the EAS subjects from the GERA cohort. All of the genotype data have been imputed to the 1000G and passed the QC steps as described above. We included only the SNPs in common among the datasets and SNPs with MAF ≥ 0.01 in Tibetans and the combined sample. There were ∼7.3 million SNPs on 10,292 individuals (3,008 Tibetans, 2,098 Han, and 5,186 EAS) included in analysis. The WZ-Han and GERA-EAS subjects were genotyped and imputed separately from the TP subjects, and the GERA-EAS subjects were genotyped on a different type of SNP array (the GERA-EAS subjects were genotyped on Affymetrix Axiom arrays, and all of the other subjects were genotyped on Illumina CoreExome arrays). To control for difference in allele frequency caused by genotyping or imputation artifacts, we performed a control–control analysis using the MLMA-LOCO method as described above to test for allele frequency (AF) difference between TP-Han (<em>n</em> = 373) and WZ-Han + GERA-EAS (<em>n</em> = 6,911) and removed SNPs at <em>P</em> &lt; 1 × 10<sup>−6</sup>. To detect genetic signals of high-altitude adaptation, we then performed an MLMA-LOCO analysis to test for AF difference between Tibetans (<em>n</em> = 3,008) and TP-Han + WZ-Han + GERA-EAS (<em>n</em> = 7,284). We further reran the analyses in two subsets of data to show the gain of power by combining all of the available samples [i.e., (<em>i</em>) Seda-Tibetan (<em>n</em> = 2,427) vs. Seda-Han + GERA-EAS (<em>n</em> = 5,548); (<em>ii</em>) Litang-Tibetan (<em>n</em> = 581) vs. Litang-Han + WZ-Han (<em>n</em> = 1,736), where Seda and Litang are two sites in the TP as described above]. Summary-level statistics from all of the MLMA-LOCO analyses are available at <a href="http://cnsgenomics.com/data/yang_et_al_2017_pnas.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">cnsgenomics.com/data/yang_et_al_2017_pnas.html</a>. The raw genotype and phenotype data of the Tibetan and Han subjects are available through application at <a href="https://www.wmubiobank.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.wmubiobank.org</a>.</p></div></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="d35e442"><h4>Supplementary File</h4><div class="sup-box half_rhythm" id="d35e442a1051501"><a href="/pmc/articles/PMC5402460/bin/pnas.1617042114.sapp.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(6.7M, pdf)</sup></div></div></div><div id="idm140124596822224" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140124596822224title" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p id="__p28">We thank Jonathan Pritchard for constructive and helpful comments on an earlier version of the manuscript. This study was supported by National Key Basic Research Program Grant 2013CB967502; National Natural Science Foundation of China Grants 81522014, 81371059, 81470659, 81500741, 81271039, and 81570880; Ministry of Science and Technology (MOST) Projects Grant 2012YQ12008004; National Key Clinical Specialty (Ophthalmology); the project of Zhejiang Provincial Top Key Discipline and Key Construction Discipline of Medicine; Wenzhou Science and Technology Innovation Team Project Grant C20150004; Zhejiang Provincial Natural Science Foundation of China Grants LR13H120001 and LQ14B020005; National Health and Family Planning Commission (NHFPC) Grant-in-Aid for Medical and Health Science 201472911; Australian National Health and Medical Research Council Grants 1078037, 1078901, 1107258, and 1113400; Australian Research Council Grant 160101343; and the Sylvia &amp; Charles Viertel Charitable Foundation.</p></div></div><div id="idm140124596821504" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140124596821504title" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140124596821376"></p><p id="__p29" class="p p-first-last">The authors declare no conflict of interest.</p><p></p><p class="fn sec" id="idm140124583348192"></p><p id="__p30" class="p p-first-last">This article is a PNAS Direct Submission.</p><p></p><p class="fn sec" id="idm140124583347232"></p><p id="__p31" class="p p-first-last">Data deposition: The summary-level statistics from all of the mixed linear model-based leave one chromosome out association analyses reported in the paper are available at <a href="http://cnsgenomics.com/data/yang_et_al_2017_pnas.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">cnsgenomics.com/data/yang_et_al_2017_pnas.html</a>. The raw genotype and phenotype data of the Tibetan and Han subjects are available through application at <a href="https://www.wmubiobank.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.wmubiobank.org</a>.</p><p></p><p class="fn sec" id="idm140124587447216"></p><p id="__p32" class="p p-first-last">This article contains supporting information online at <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1617042114/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.pnas.org/lookup/suppl/doi:10.1073/pnas.1617042114/-/DCSupplemental</a>.</p><p></p></div></div><div id="idm140124583136320" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140124583136320title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="r1">1. <span class="element-citation">Beall CM. Two routes to functional adaptation: Tibetan and Andean high-altitude natives. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2007;<span class="ref-vol">104</span>(Suppl 1):8655–8660.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1876443/">PMC free article</a>]</span> [<a href="/pubmed/17494744" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Two+routes+to+functional+adaptation:+Tibetan+and+Andean+high-altitude+natives&amp;author=CM+Beall&amp;volume=104&amp;issue=Suppl+1&amp;publication_year=2007&amp;pages=8655-8660&amp;pmid=17494744&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r2">2. <span class="element-citation">Dahlback A, Gelsor N, Stamnes JJ, Gjessing Y. UV measurements in the 3000–5000 m altitude region in Tibet. <span><span class="ref-journal">J Geophys Res Atmos. </span>2007;<span class="ref-vol">112</span>:D09308.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Geophys+Res+Atmos&amp;title=UV+measurements+in+the+3000–5000+m+altitude+region+in+Tibet&amp;author=A+Dahlback&amp;author=N+Gelsor&amp;author=JJ+Stamnes&amp;author=Y+Gjessing&amp;volume=112&amp;publication_year=2007&amp;pages=D09308&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r3">3. <span class="element-citation">Yi X, et al.  Sequencing of 50 human exomes reveals adaptation to high altitude. <span><span class="ref-journal">Science. </span>2010;<span class="ref-vol">329</span>:75–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3711608/">PMC free article</a>]</span> [<a href="/pubmed/20595611" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Sequencing+of+50+human+exomes+reveals+adaptation+to+high+altitude&amp;author=X+Yi&amp;volume=329&amp;publication_year=2010&amp;pages=75-78&amp;pmid=20595611&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r4">4. <span class="element-citation">Simonson TS, et al.  Genetic evidence for high-altitude adaptation in Tibet. <span><span class="ref-journal">Science. </span>2010;<span class="ref-vol">329</span>:72–75.</span> [<a href="/pubmed/20466884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic+evidence+for+high-altitude+adaptation+in+Tibet&amp;author=TS+Simonson&amp;volume=329&amp;publication_year=2010&amp;pages=72-75&amp;pmid=20466884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r5">5. <span class="element-citation">Beall CM, et al.  Natural selection on EPAS1 (HIF2alpha) associated with low hemoglobin concentration in Tibetan highlanders. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2010;<span class="ref-vol">107</span>:11459–11464.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2895075/">PMC free article</a>]</span> [<a href="/pubmed/20534544" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Natural+selection+on+EPAS1+(HIF2alpha)+associated+with+low+hemoglobin+concentration+in+Tibetan+highlanders&amp;author=CM+Beall&amp;volume=107&amp;publication_year=2010&amp;pages=11459-11464&amp;pmid=20534544&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r6">6. <span class="element-citation">Xu S, et al.  A genome-wide search for signals of high-altitude adaptation in Tibetans. <span><span class="ref-journal">Mol Biol Evol. </span>2011;<span class="ref-vol">28</span>:1003–1011.</span> [<a href="/pubmed/20961960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=A+genome-wide+search+for+signals+of+high-altitude+adaptation+in+Tibetans&amp;author=S+Xu&amp;volume=28&amp;publication_year=2011&amp;pages=1003-1011&amp;pmid=20961960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r7">7. <span class="element-citation">Peng Y, et al.  Genetic variations in Tibetan populations and high-altitude adaptation at the Himalayas. <span><span class="ref-journal">Mol Biol Evol. </span>2011;<span class="ref-vol">28</span>:1075–1081.</span> [<a href="/pubmed/21030426" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=Genetic+variations+in+Tibetan+populations+and+high-altitude+adaptation+at+the+Himalayas&amp;author=Y+Peng&amp;volume=28&amp;publication_year=2011&amp;pages=1075-1081&amp;pmid=21030426&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r8">8. <span class="element-citation">Tian H, McKnight SL, Russell DW. Endothelial PAS domain protein 1 (EPAS1), a transcription factor selectively expressed in endothelial cells. <span><span class="ref-journal">Genes Dev. </span>1997;<span class="ref-vol">11</span>:72–82.</span> [<a href="/pubmed/9000051" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=Endothelial+PAS+domain+protein+1+(EPAS1),+a+transcription+factor+selectively+expressed+in+endothelial+cells&amp;author=H+Tian&amp;author=SL+McKnight&amp;author=DW+Russell&amp;volume=11&amp;publication_year=1997&amp;pages=72-82&amp;pmid=9000051&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r9">9. <span class="element-citation">Tian H, Hammer RE, Matsumoto AM, Russell DW, McKnight SL. The hypoxia-responsive transcription factor EPAS1 is essential for catecholamine homeostasis and protection against heart failure during embryonic development. <span><span class="ref-journal">Genes Dev. </span>1998;<span class="ref-vol">12</span>:3320–3324.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC317225/">PMC free article</a>]</span> [<a href="/pubmed/9808618" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=The+hypoxia-responsive+transcription+factor+EPAS1+is+essential+for+catecholamine+homeostasis+and+protection+against+heart+failure+during+embryonic+development&amp;author=H+Tian&amp;author=RE+Hammer&amp;author=AM+Matsumoto&amp;author=DW+Russell&amp;author=SL+McKnight&amp;volume=12&amp;publication_year=1998&amp;pages=3320-3324&amp;pmid=9808618&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r10">10. <span class="element-citation">Lee FS, Percy MJ. The HIF pathway and erythrocytosis. <span><span class="ref-journal">Annu Rev Pathol. </span>2011;<span class="ref-vol">6</span>:165–192.</span> [<a href="/pubmed/20939709" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Pathol&amp;title=The+HIF+pathway+and+erythrocytosis&amp;author=FS+Lee&amp;author=MJ+Percy&amp;volume=6&amp;publication_year=2011&amp;pages=165-192&amp;pmid=20939709&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r11">11. <span class="element-citation">Lorenzo FR, et al.  A genetic mechanism for Tibetan high-altitude adaptation. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:951–956.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4473257/">PMC free article</a>]</span> [<a href="/pubmed/25129147" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genetic+mechanism+for+Tibetan+high-altitude+adaptation&amp;author=FR+Lorenzo&amp;volume=46&amp;publication_year=2014&amp;pages=951-956&amp;pmid=25129147&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r12">12. <span class="element-citation">Xiang K, et al.  Identification of a Tibetan-specific mutation in the hypoxic gene EGLN1 and its contribution to high-altitude adaptation. <span><span class="ref-journal">Mol Biol Evol. </span>2013;<span class="ref-vol">30</span>:1889–1898.</span> [<a href="/pubmed/23666208" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=Identification+of+a+Tibetan-specific+mutation+in+the+hypoxic+gene+EGLN1+and+its+contribution+to+high-altitude+adaptation&amp;author=K+Xiang&amp;volume=30&amp;publication_year=2013&amp;pages=1889-1898&amp;pmid=23666208&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r13">13. <span class="element-citation">Wang GD, et al.  Genetic convergence in the adaptation of dogs and humans to the high-altitude environment of the Tibetan plateau. <span><span class="ref-journal">Genome Biol Evol. </span>2014;<span class="ref-vol">6</span>:2122–2128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4231634/">PMC free article</a>]</span> [<a href="/pubmed/25091388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol+Evol&amp;title=Genetic+convergence+in+the+adaptation+of+dogs+and+humans+to+the+high-altitude+environment+of+the+Tibetan+plateau&amp;author=GD+Wang&amp;volume=6&amp;publication_year=2014&amp;pages=2122-2128&amp;pmid=25091388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r14">14. <span class="element-citation">Wuren T, et al.  Shared and unique signals of high-altitude adaptation in geographically distinct Tibetan populations. <span><span class="ref-journal">PLoS One. </span>2014;<span class="ref-vol">9</span>:e88252.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3958363/">PMC free article</a>]</span> [<a href="/pubmed/24642866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Shared+and+unique+signals+of+high-altitude+adaptation+in+geographically+distinct+Tibetan+populations&amp;author=T+Wuren&amp;volume=9&amp;publication_year=2014&amp;pages=e88252&amp;pmid=24642866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r15">15. <span class="element-citation">Jha AR, et al.  Shared genetic signals of hypoxia adaptation in Drosophila and in high-altitude human populations. <span><span class="ref-journal">Mol Biol Evol. </span>2016;<span class="ref-vol">33</span>:501–517.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4866538/">PMC free article</a>]</span> [<a href="/pubmed/26576852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=Shared+genetic+signals+of+hypoxia+adaptation+in+Drosophila+and+in+high-altitude+human+populations&amp;author=AR+Jha&amp;volume=33&amp;publication_year=2016&amp;pages=501-517&amp;pmid=26576852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r16">16. <span class="element-citation">Rosenberg NA, et al.  Genetic structure of human populations. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">298</span>:2381–2385.</span> [<a href="/pubmed/12493913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic+structure+of+human+populations&amp;author=NA+Rosenberg&amp;volume=298&amp;publication_year=2002&amp;pages=2381-2385&amp;pmid=12493913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r17">17. <span class="element-citation">Nei M.  <span class="ref-journal">Molecular Evolutionary Genetics.</span> Columbia Univ Press; New York: 1987.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Molecular+Evolutionary+Genetics&amp;author=M+Nei&amp;publication_year=1987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r18">18. <span class="element-citation">Weir BS, Cockerham CC. Estimating F-statistics for the analysis of population structure. <span><span class="ref-journal">Evolution. </span>1984;<span class="ref-vol">38</span>:1358–1370.</span> [<a href="/pubmed/28563791" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Estimating+F-statistics+for+the+analysis+of+population+structure&amp;author=BS+Weir&amp;author=CC+Cockerham&amp;volume=38&amp;publication_year=1984&amp;pages=1358-1370&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r19">19. <span class="element-citation">McEvoy BP, Powell JE, Goddard ME, Visscher PM. Human population dispersal “Out of Africa” estimated from linkage disequilibrium and allele frequencies of SNPs. <span><span class="ref-journal">Genome Res. </span>2011;<span class="ref-vol">21</span>:821–829.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3106315/">PMC free article</a>]</span> [<a href="/pubmed/21518737" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Human+population+dispersal+“Out+of+Africa”+estimated+from+linkage+disequilibrium+and+allele+frequencies+of+SNPs&amp;author=BP+McEvoy&amp;author=JE+Powell&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=21&amp;publication_year=2011&amp;pages=821-829&amp;pmid=21518737&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r20">20. <span class="element-citation">Aldenderfer M. Peopling the Tibetan plateau: Insights from archaeology. <span><span class="ref-journal">High Alt Med Biol. </span>2011;<span class="ref-vol">12</span>:141–147.</span> [<a href="/pubmed/21718162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=High+Alt+Med+Biol&amp;title=Peopling+the+Tibetan+plateau:+Insights+from+archaeology&amp;author=M+Aldenderfer&amp;volume=12&amp;publication_year=2011&amp;pages=141-147&amp;pmid=21718162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r21">21. <span class="element-citation">Chen FH, et al.  Agriculture facilitated permanent human occupation of the Tibetan Plateau after 3600 B.P. <span><span class="ref-journal">Science. </span>2015;<span class="ref-vol">347</span>:248–250.</span> [<a href="/pubmed/25593179" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Agriculture+facilitated+permanent+human+occupation+of+the+Tibetan+Plateau+after+3600+B.P&amp;author=FH+Chen&amp;volume=347&amp;publication_year=2015&amp;pages=248-250&amp;pmid=25593179&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r22">22. <span class="element-citation">Loh PR, et al.  Efficient Bayesian mixed-model analysis increases association power in large cohorts. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:284–290.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4342297/">PMC free article</a>]</span> [<a href="/pubmed/25642633" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Efficient+Bayesian+mixed-model+analysis+increases+association+power+in+large+cohorts&amp;author=PR+Loh&amp;volume=47&amp;publication_year=2015&amp;pages=284-290&amp;pmid=25642633&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r23">23. <span class="element-citation">Yang J, Zaitlen NA, Goddard ME, Visscher PM, Price AL. Advantages and pitfalls in the application of mixed-model association methods. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:100–106.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3989144/">PMC free article</a>]</span> [<a href="/pubmed/24473328" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Advantages+and+pitfalls+in+the+application+of+mixed-model+association+methods&amp;author=J+Yang&amp;author=NA+Zaitlen&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;author=AL+Price&amp;volume=46&amp;publication_year=2014&amp;pages=100-106&amp;pmid=24473328&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r24">24. <span class="element-citation">Bhatia G, Patterson N, Sankararaman S, Price AL. Estimating and interpreting FST: The impact of rare variants. <span><span class="ref-journal">Genome Res. </span>2013;<span class="ref-vol">23</span>:1514–1521.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3759727/">PMC free article</a>]</span> [<a href="/pubmed/23861382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Estimating+and+interpreting+FST:+The+impact+of+rare+variants&amp;author=G+Bhatia&amp;author=N+Patterson&amp;author=S+Sankararaman&amp;author=AL+Price&amp;volume=23&amp;publication_year=2013&amp;pages=1514-1521&amp;pmid=23861382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r25">25. <span class="element-citation">Banda Y, et al.  Characterizing race/ethnicity and genetic ancestry for 100,000 subjects in the Genetic Epidemiology Research on Adult Health and Aging (GERA) cohort. <span><span class="ref-journal">Genetics. </span>2015;<span class="ref-vol">200</span>:1285–1295.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4574246/">PMC free article</a>]</span> [<a href="/pubmed/26092716" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Characterizing+race/ethnicity+and+genetic+ancestry+for+100,000+subjects+in+the+Genetic+Epidemiology+Research+on+Adult+Health+and+Aging+(GERA)+cohort&amp;author=Y+Banda&amp;volume=200&amp;publication_year=2015&amp;pages=1285-1295&amp;pmid=26092716&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r26">26. <span class="element-citation">Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B+Howie&amp;author=C+Fuchsberger&amp;author=M+Stephens&amp;author=J+Marchini&amp;author=GR+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r27">27. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: A tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+A+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r28">28. <span class="element-citation">Yang J, et al.  Genetic Investigation of ANthropometric Traits (GIANT) Consortium; DIAbetes Genetics Replication And Meta-analysis (DIAGRAM) Consortium  Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:369–375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3593158/">PMC free article</a>]</span> [<a href="/pubmed/22426310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Conditional+and+joint+multiple-SNP+analysis+of+GWAS+summary+statistics+identifies+additional+variants+influencing+complex+traits&amp;author=J+Yang&amp;volume=44&amp;publication_year=2012&amp;pages=369-375&amp;pmid=22426310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r29">29. <span class="element-citation">van Meurs JB, et al.  Common genetic loci influencing plasma homocysteine concentrations and their effect on risk of coronary artery disease. <span><span class="ref-journal">Am J Clin Nutr. </span>2013;<span class="ref-vol">98</span>:668–676.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4321227/">PMC free article</a>]</span> [<a href="/pubmed/23824729" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Clin+Nutr&amp;title=Common+genetic+loci+influencing+plasma+homocysteine+concentrations+and+their+effect+on+risk+of+coronary+artery+disease&amp;author=JB+van+Meurs&amp;volume=98&amp;publication_year=2013&amp;pages=668-676&amp;pmid=23824729&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r30">30. <span class="element-citation">Jacques PF, et al.  Relation between folate status, a common mutation in methylenetetrahydrofolate reductase, and plasma homocysteine concentrations. <span><span class="ref-journal">Circulation. </span>1996;<span class="ref-vol">93</span>:7–9.</span> [<a href="/pubmed/8616944" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Relation+between+folate+status,+a+common+mutation+in+methylenetetrahydrofolate+reductase,+and+plasma+homocysteine+concentrations&amp;author=PF+Jacques&amp;volume=93&amp;publication_year=1996&amp;pages=7-9&amp;pmid=8616944&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r31">31. <span class="element-citation">Locke AE, et al.  LifeLines Cohort Study; ADIPOGen Consortium; AGEN-BMI Working Group; CARDIOGRAMplusC4D Consortium; CKDGen Consortium; GLGC; ICBP; MAGIC Investigators; MuTHER Consortium; MIGen Consortium; PAGE Consortium; ReproGen Consortium; GENIE Consortium; International Endogene Consortium  Genetic studies of body mass index yield new insights for obesity biology. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">518</span>:197–206.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4382211/">PMC free article</a>]</span> [<a href="/pubmed/25673413" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+studies+of+body+mass+index+yield+new+insights+for+obesity+biology&amp;author=AE+Locke&amp;volume=518&amp;publication_year=2015&amp;pages=197-206&amp;pmid=25673413&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r32">32. <span class="element-citation">Qiu J. Who are the Tibetans? <span><span class="ref-journal">Science. </span>2015;<span class="ref-vol">347</span>:708–711.</span> [<a href="/pubmed/25678642" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Who+are+the+Tibetans?&amp;author=J+Qiu&amp;volume=347&amp;publication_year=2015&amp;pages=708-711&amp;pmid=25678642&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r33">33. <span class="element-citation">Qi X, et al.  Genetic evidence of paleolithic colonization and neolithic expansion of modern humans on the tibetan plateau. <span><span class="ref-journal">Mol Biol Evol. </span>2013;<span class="ref-vol">30</span>:1761–1778.</span> [<a href="/pubmed/23682168" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Evol&amp;title=Genetic+evidence+of+paleolithic+colonization+and+neolithic+expansion+of+modern+humans+on+the+tibetan+plateau&amp;author=X+Qi&amp;volume=30&amp;publication_year=2013&amp;pages=1761-1778&amp;pmid=23682168&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r34">34. <span class="element-citation">Lu D, et al.  Ancestral origins and genetic history of Tibetan Highlanders. <span><span class="ref-journal">Am J Hum Genet. </span>2016;<span class="ref-vol">99</span>:580–594.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5011065/">PMC free article</a>]</span> [<a href="/pubmed/27569548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Ancestral+origins+and+genetic+history+of+Tibetan+Highlanders&amp;author=D+Lu&amp;volume=99&amp;publication_year=2016&amp;pages=580-594&amp;pmid=27569548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r35">35. <span class="element-citation">Jeong C, et al.  Admixture facilitates genetic adaptations to high altitude in Tibet. <span><span class="ref-journal">Nat Commun. </span>2014;<span class="ref-vol">5</span>:3281.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4643256/">PMC free article</a>]</span> [<a href="/pubmed/24513612" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Admixture+facilitates+genetic+adaptations+to+high+altitude+in+Tibet&amp;author=C+Jeong&amp;volume=5&amp;publication_year=2014&amp;pages=3281&amp;pmid=24513612&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r36">36. <span class="element-citation">Shriver MD, et al.  The genomic distribution of population substructure in four populations using 8,525 autosomal SNPs. <span><span class="ref-journal">Hum Genomics. </span>2004;<span class="ref-vol">1</span>:274–286.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3525267/">PMC free article</a>]</span> [<a href="/pubmed/15588487" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genomics&amp;title=The+genomic+distribution+of+population+substructure+in+four+populations+using+8,525+autosomal+SNPs&amp;author=MD+Shriver&amp;volume=1&amp;publication_year=2004&amp;pages=274-286&amp;pmid=15588487&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r37">37. <span class="element-citation">Raj SM, Pagani L, Gallego Romero I, Kivisild T, Amos W. A general linear model-based approach for inferring selection to climate. <span><span class="ref-journal">BMC Genet. </span>2013;<span class="ref-vol">14</span>:87.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3853933/">PMC free article</a>]</span> [<a href="/pubmed/24053227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet&amp;title=A+general+linear+model-based+approach+for+inferring+selection+to+climate&amp;author=SM+Raj&amp;author=L+Pagani&amp;author=I+Gallego+Romero&amp;author=T+Kivisild&amp;author=W+Amos&amp;volume=14&amp;publication_year=2013&amp;pages=87&amp;pmid=24053227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r38">38. <span class="element-citation">Pritchard JK, Pickrell JK, Coop G. The genetics of human adaptation: Hard sweeps, soft sweeps, and polygenic adaptation. <span><span class="ref-journal">Curr Biol. </span>2010;<span class="ref-vol">20</span>:R208–R215.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2994553/">PMC free article</a>]</span> [<a href="/pubmed/20178769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Biol&amp;title=The+genetics+of+human+adaptation:+Hard+sweeps,+soft+sweeps,+and+polygenic+adaptation&amp;author=JK+Pritchard&amp;author=JK+Pickrell&amp;author=G+Coop&amp;volume=20&amp;publication_year=2010&amp;pages=R208-R215&amp;pmid=20178769&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r39">39. <span class="element-citation">Borradale DC, Kimlin MG. Folate degradation due to ultraviolet radiation: Possible implications for human health and nutrition. <span><span class="ref-journal">Nutr Rev. </span>2012;<span class="ref-vol">70</span>:414–422.</span> [<a href="/pubmed/22747844" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nutr+Rev&amp;title=Folate+degradation+due+to+ultraviolet+radiation:+Possible+implications+for+human+health+and+nutrition&amp;author=DC+Borradale&amp;author=MG+Kimlin&amp;volume=70&amp;publication_year=2012&amp;pages=414-422&amp;pmid=22747844&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r40">40. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r41">41. <span class="element-citation">Galinsky KJ, et al.  Fast principal-component analysis reveals convergent evolution of ADH1B in Europe and East Asia. <span><span class="ref-journal">Am J Hum Genet. </span>2016;<span class="ref-vol">98</span>:456–472.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4827102/">PMC free article</a>]</span> [<a href="/pubmed/26924531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Fast+principal-component+analysis+reveals+convergent+evolution+of+ADH1B+in+Europe+and+East+Asia&amp;author=KJ+Galinsky&amp;volume=98&amp;publication_year=2016&amp;pages=456-472&amp;pmid=26924531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r42">42. <span class="element-citation">The 1000 Genomes Project Consortium  et al.  A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r43">43. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r44">44. <span class="element-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5402460&amp;issue-id=291368&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140124589669648title">Significance</a></li><li><a href="#idm140124667262720title">Abstract</a></li><li><a href="#s1title">Results</a></li><li><a href="#s5title">Discussion</a></li><li><a href="#s6title">Materials and Methods</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140124596822224title">Acknowledgments</a></li><li><a href="#idm140124596821504title">Footnotes</a></li><li><a href="#idm140124583136320title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5402460/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5402460/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5402460/pdf/pnas.201617042.pdf">PDF (815K)</a></li> | <li><a href="#" data-citationid="PMC5402460" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5402460%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5402460%2F&amp;text=Genetic%20signatures%20of%20high-altitude%20adaptation%20in%20Tibetans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5402460%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28373541" data-aiid="5402460" data-aid="5402460" data-iid="291368" data-domainid="2" data-domain="pnas" data-accid="PMC5402460" data-md5="e8a74196f223101c71639520c3ba4396"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:02:58-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C2789DD59A93100000000097F05A3&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5402460%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=pnas&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5402460/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size1, monospace;"></div></div></body></html>