<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5626933/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association analysis identifies variation in vitamin D receptor and other host factors influencing the gut microbiota</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5626933/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association analysis identifies variation in vitamin D receptor and other host factors influencing the gut microbiota"><meta name="citation_authors" content="Jun Wang, Louise B Thingholm, Jurgita Skiecevičienė, Philipp Rausch, Martin Kummen, Johannes R Hov, Frauke Degenhardt, Femke-Anouska Heinsen, Malte C Rühlemann, Silke Szymczak, Kristian Holm, Tönu Esko, Jun Sun, Mihaela Pricop-Jeckstadt, Samer Al-Dury, Pavol Bohov, Jörn Bethune, Felix Sommer, David Ellinghaus, Rolf K Berge, Matthias Hübenthal, Manja Koch, Karin Schwarz, Gerald Rimbach, Patricia Hübbe, Wei-Hung Pan, Raheleh Sheibani-Tezerji, Robert Häsler, Philipp Rosenstiel, Mauro D’Amato, Katja Cloppenborg-Schmidt, Sven Künzel, Matthias Laudes, Hanns-Ulrich Marschall, Wolfgang Lieb, Ute Nöthlings, Tom H Karlsen, John F Baines, Andre Franke"><meta name="citation_date" content="November 2016"><meta name="citation_issue" content="11"><meta name="citation_volume" content="48"><meta name="citation_firstpage" content="1396"><meta name="citation_doi" content="10.1038/ng.3695"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5626933/?report=abstract"><meta name="citation_pmid" content="27723756"><meta name="DC.Title" content="Genome-wide association analysis identifies variation in vitamin D receptor and other host factors influencing the gut microbiota"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Jun Wang"><meta name="DC.Contributor" content="Louise B Thingholm"><meta name="DC.Contributor" content="Jurgita Skiecevičienė"><meta name="DC.Contributor" content="Philipp Rausch"><meta name="DC.Contributor" content="Martin Kummen"><meta name="DC.Contributor" content="Johannes R Hov"><meta name="DC.Contributor" content="Frauke Degenhardt"><meta name="DC.Contributor" content="Femke-Anouska Heinsen"><meta name="DC.Contributor" content="Malte C Rühlemann"><meta name="DC.Contributor" content="Silke Szymczak"><meta name="DC.Contributor" content="Kristian Holm"><meta name="DC.Contributor" content="Tönu Esko"><meta name="DC.Contributor" content="Jun Sun"><meta name="DC.Contributor" content="Mihaela Pricop-Jeckstadt"><meta name="DC.Contributor" content="Samer Al-Dury"><meta name="DC.Contributor" content="Pavol Bohov"><meta name="DC.Contributor" content="Jörn Bethune"><meta name="DC.Contributor" content="Felix Sommer"><meta name="DC.Contributor" content="David Ellinghaus"><meta name="DC.Contributor" content="Rolf K Berge"><meta name="DC.Contributor" content="Matthias Hübenthal"><meta name="DC.Contributor" content="Manja Koch"><meta name="DC.Contributor" content="Karin Schwarz"><meta name="DC.Contributor" content="Gerald Rimbach"><meta name="DC.Contributor" content="Patricia Hübbe"><meta name="DC.Contributor" content="Wei-Hung Pan"><meta name="DC.Contributor" content="Raheleh Sheibani-Tezerji"><meta name="DC.Contributor" content="Robert Häsler"><meta name="DC.Contributor" content="Philipp Rosenstiel"><meta name="DC.Contributor" content="Mauro D’Amato"><meta name="DC.Contributor" content="Katja Cloppenborg-Schmidt"><meta name="DC.Contributor" content="Sven Künzel"><meta name="DC.Contributor" content="Matthias Laudes"><meta name="DC.Contributor" content="Hanns-Ulrich Marschall"><meta name="DC.Contributor" content="Wolfgang Lieb"><meta name="DC.Contributor" content="Ute Nöthlings"><meta name="DC.Contributor" content="Tom H Karlsen"><meta name="DC.Contributor" content="John F Baines"><meta name="DC.Contributor" content="Andre Franke"><meta name="DC.Date" content="2016 Nov"><meta name="DC.Identifier" content="10.1038/ng.3695"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association analysis identifies variation in vitamin D receptor and other host factors influencing the gut microbiota"><meta property="og:type" content="article"><meta property="og:description" content="Human gut microbiota is an important determinant for health and disease, and recent studies emphasize the numerous factors shaping its diversity. Here we performed a genome-wide association study (GWAS) of the gut microbiota using two cohorts from northern ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5626933/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5626933/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5626933/pdf/nihms904581.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D8AA4DD59DBC10000000000D8007F.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5626933%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5626933</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2017 Oct 4.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=27723756" target="pmc_ext" ref="reftype=publisher&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2016 Nov; 48(11): 1396–1406. </span></a></div><span class="fm-vol-iss-date">Published online 2016 Oct 10. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.3695" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.3695</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5626933</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS904581</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27723756">27723756</a></div></div></div></div><h1 class="content-title">Genome-wide association analysis identifies variation in vitamin D receptor and other host factors influencing the gut microbiota</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Wang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468841341168" co-class="co-affbox">Jun Wang</a>,<sup>1,</sup><sup>2,</sup><sup>21,</sup><sup>22</sup> <a href="/pubmed/?term=Thingholm%20LB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468842206688" co-class="co-affbox">Louise B Thingholm</a>,<sup>3,</sup><sup>22</sup> <a href="/pubmed/?term=Skiecevi%26%23x0010d%3Bien%26%23x00117%3B%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468842203968" co-class="co-affbox">Jurgita Skiecevičienė</a>,<sup>3,</sup><sup>22</sup> <a href="/pubmed/?term=Rausch%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468799166096" co-class="co-affbox">Philipp Rausch</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Kummen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468799163376" co-class="co-affbox">Martin Kummen</a>,<sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Hov%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468808372112" co-class="co-affbox">Johannes R Hov</a>,<sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Degenhardt%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468808468224" co-class="co-affbox">Frauke Degenhardt</a>,<sup>3</sup> <a href="/pubmed/?term=Heinsen%20FA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468808466240" co-class="co-affbox">Femke-Anouska Heinsen</a>,<sup>3</sup> <a href="/pubmed/?term=R%26%23x000fc%3Bhlemann%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468806265760" co-class="co-affbox">Malte C Rühlemann</a>,<sup>3</sup> <a href="/pubmed/?term=Szymczak%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468806263776" co-class="co-affbox">Silke Szymczak</a>,<sup>3,</sup><sup>21</sup> <a href="/pubmed/?term=Holm%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468804500704" co-class="co-affbox">Kristian Holm</a>,<sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468804496624" co-class="co-affbox">Tönu Esko</a>,<sup>9</sup> <a href="/pubmed/?term=Sun%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468805345776" co-class="co-affbox">Jun Sun</a>,<sup>10</sup> <a href="/pubmed/?term=Pricop-Jeckstadt%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468805343792" co-class="co-affbox">Mihaela Pricop-Jeckstadt</a>,<sup>11</sup> <a href="/pubmed/?term=Al-Dury%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468805341808" co-class="co-affbox">Samer Al-Dury</a>,<sup>12</sup> <a href="/pubmed/?term=Bohov%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468797874448" co-class="co-affbox">Pavol Bohov</a>,<sup>13</sup> <a href="/pubmed/?term=Bethune%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468797872464" co-class="co-affbox">Jörn Bethune</a>,<sup>3</sup> <a href="/pubmed/?term=Sommer%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468803265232" co-class="co-affbox">Felix Sommer</a>,<sup>3</sup> <a href="/pubmed/?term=Ellinghaus%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468803263360" co-class="co-affbox">David Ellinghaus</a>,<sup>3</sup> <a href="/pubmed/?term=Berge%20RK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468803261376" co-class="co-affbox">Rolf K Berge</a>,<sup>13,</sup><sup>14</sup> <a href="/pubmed/?term=H%26%23x000fc%3Bbenthal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468803041920" co-class="co-affbox">Matthias Hübenthal</a>,<sup>3</sup> <a href="/pubmed/?term=Koch%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468803039936" co-class="co-affbox">Manja Koch</a>,<sup>15</sup> <a href="/pubmed/?term=Schwarz%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468803037952" co-class="co-affbox">Karin Schwarz</a>,<sup>16</sup> <a href="/pubmed/?term=Rimbach%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468808678112" co-class="co-affbox">Gerald Rimbach</a>,<sup>16</sup> <a href="/pubmed/?term=H%26%23x000fc%3Bbbe%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468808676128" co-class="co-affbox">Patricia Hübbe</a>,<sup>16</sup> <a href="/pubmed/?term=Pan%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468800851600" co-class="co-affbox">Wei-Hung Pan</a>,<sup>3</sup> <a href="/pubmed/?term=Sheibani-Tezerji%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468800849616" co-class="co-affbox">Raheleh Sheibani-Tezerji</a>,<sup>3</sup> <a href="/pubmed/?term=H%26%23x000e4%3Bsler%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468800847632" co-class="co-affbox">Robert Häsler</a>,<sup>3</sup> <a href="/pubmed/?term=Rosenstiel%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468805618096" co-class="co-affbox">Philipp Rosenstiel</a>,<sup>3</sup> <a href="/pubmed/?term=D%26%23x02019%3BAmato%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468805616112" co-class="co-affbox">Mauro D’Amato</a>,<sup>17,</sup><sup>18</sup> <a href="/pubmed/?term=Cloppenborg-Schmidt%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468805613392" co-class="co-affbox">Katja Cloppenborg-Schmidt</a>,<sup>2</sup> <a href="/pubmed/?term=K%26%23x000fc%3Bnzel%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468796525120" co-class="co-affbox">Sven Künzel</a>,<sup>1</sup> <a href="/pubmed/?term=Laudes%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468796523136" co-class="co-affbox">Matthias Laudes</a>,<sup>19</sup> <a href="/pubmed/?term=Marschall%20HU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468877322896" co-class="co-affbox">Hanns-Ulrich Marschall</a>,<sup>12</sup> <a href="/pubmed/?term=Lieb%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468877321024" co-class="co-affbox">Wolfgang Lieb</a>,<sup>15</sup> <a href="/pubmed/?term=N%26%23x000f6%3Bthlings%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468877319040" co-class="co-affbox">Ute Nöthlings</a>,<sup>11</sup> <a href="/pubmed/?term=Karlsen%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468808016064" co-class="co-affbox">Tom H Karlsen</a>,<sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>7,</sup><sup>8,</sup><sup>20,</sup><sup>23</sup> <a href="/pubmed/?term=Baines%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468807820144" co-class="co-affbox">John F Baines</a>,<sup>1,</sup><sup>2,</sup><sup>23</sup> and  <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756" class="affpopup" co-rid="_co_idm140468807816688" co-class="co-affbox">Andre Franke</a><sup>3,</sup><sup>23</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140468841341168"><h3 class="no_margin">Jun Wang</h3><p>  <sup>1</sup>Evolutionary Genomics, Max Planck Institute for Evolutionary Biology, Plön, Germany</p><p>  <sup>2</sup>Institute for Experimental Medicine, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Wang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Jun Wang</a></div></div><div id="_co_idm140468842206688"><h3 class="no_margin">Louise B Thingholm</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Thingholm%20LB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Louise B Thingholm</a></div></div><div id="_co_idm140468842203968"><h3 class="no_margin">Jurgita Skiecevičienė</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Skiecevi%26%23x0010d%3Bien%26%23x00117%3B%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Jurgita Skiecevičienė</a></div></div><div id="_co_idm140468799166096"><h3 class="no_margin">Philipp Rausch</h3><p>  <sup>1</sup>Evolutionary Genomics, Max Planck Institute for Evolutionary Biology, Plön, Germany</p><p>  <sup>2</sup>Institute for Experimental Medicine, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Rausch%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Philipp Rausch</a></div></div><div id="_co_idm140468799163376"><h3 class="no_margin">Martin Kummen</h3><p>  <sup>4</sup>Norwegian PSC Research Center, Division of Surgery, Inflammatory Medicine and Transplantation, Oslo, Norway</p><p>  <sup>5</sup>Institute of Clinical Medicine, Faculty of Medicine, University of Oslo, Oslo, Norway</p><p>  <sup>6</sup>K.G. Jebsen Inflammation Research Centre, Institute of Clinical Medicine, University of Oslo, Oslo, Norway</p><p>  <sup>7</sup>Research Institute of Internal Medicine, Oslo University Hospital Rikshospitalet, Oslo, Norway</p><div>Find articles by <a href="/pubmed/?term=Kummen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Martin Kummen</a></div></div><div id="_co_idm140468808372112"><h3 class="no_margin">Johannes R Hov</h3><p>  <sup>4</sup>Norwegian PSC Research Center, Division of Surgery, Inflammatory Medicine and Transplantation, Oslo, Norway</p><p>  <sup>5</sup>Institute of Clinical Medicine, Faculty of Medicine, University of Oslo, Oslo, Norway</p><p>  <sup>6</sup>K.G. Jebsen Inflammation Research Centre, Institute of Clinical Medicine, University of Oslo, Oslo, Norway</p><p>  <sup>7</sup>Research Institute of Internal Medicine, Oslo University Hospital Rikshospitalet, Oslo, Norway</p><p>  <sup>8</sup>Section of Gastroenterology, Department of Transplantation Medicine, Oslo University Hospital Rikshospitalet, Oslo, Norway</p><div>Find articles by <a href="/pubmed/?term=Hov%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Johannes R Hov</a></div></div><div id="_co_idm140468808468224"><h3 class="no_margin">Frauke Degenhardt</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Degenhardt%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Frauke Degenhardt</a></div></div><div id="_co_idm140468808466240"><h3 class="no_margin">Femke-Anouska Heinsen</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Heinsen%20FA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Femke-Anouska Heinsen</a></div></div><div id="_co_idm140468806265760"><h3 class="no_margin">Malte C Rühlemann</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=R%26%23x000fc%3Bhlemann%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Malte C Rühlemann</a></div></div><div id="_co_idm140468806263776"><h3 class="no_margin">Silke Szymczak</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Szymczak%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Silke Szymczak</a></div></div><div id="_co_idm140468804500704"><h3 class="no_margin">Kristian Holm</h3><p>  <sup>4</sup>Norwegian PSC Research Center, Division of Surgery, Inflammatory Medicine and Transplantation, Oslo, Norway</p><p>  <sup>5</sup>Institute of Clinical Medicine, Faculty of Medicine, University of Oslo, Oslo, Norway</p><p>  <sup>6</sup>K.G. Jebsen Inflammation Research Centre, Institute of Clinical Medicine, University of Oslo, Oslo, Norway</p><p>  <sup>7</sup>Research Institute of Internal Medicine, Oslo University Hospital Rikshospitalet, Oslo, Norway</p><div>Find articles by <a href="/pubmed/?term=Holm%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Kristian Holm</a></div></div><div id="_co_idm140468804496624"><h3 class="no_margin">Tönu Esko</h3><p>  <sup>9</sup>Estonian Genome Center, University of Tartu, Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Tönu Esko</a></div></div><div id="_co_idm140468805345776"><h3 class="no_margin">Jun Sun</h3><p>  <sup>10</sup>Division of Gastroenterology and Hepatology, Department of Medicine, University of Illinois at Chicago, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Sun%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Jun Sun</a></div></div><div id="_co_idm140468805343792"><h3 class="no_margin">Mihaela Pricop-Jeckstadt</h3><p>  <sup>11</sup>Department of Nutrition and Food Sciences, Nutritional Epidemiology, University of Bonn, Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=Pricop-Jeckstadt%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Mihaela Pricop-Jeckstadt</a></div></div><div id="_co_idm140468805341808"><h3 class="no_margin">Samer Al-Dury</h3><p>  <sup>12</sup>Department of Molecular and Clinical Medicine, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden</p><div>Find articles by <a href="/pubmed/?term=Al-Dury%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Samer Al-Dury</a></div></div><div id="_co_idm140468797874448"><h3 class="no_margin">Pavol Bohov</h3><p>  <sup>13</sup>Department of Clinical Science, University of Bergen, Bergen, Norway</p><div>Find articles by <a href="/pubmed/?term=Bohov%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Pavol Bohov</a></div></div><div id="_co_idm140468797872464"><h3 class="no_margin">Jörn Bethune</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Bethune%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Jörn Bethune</a></div></div><div id="_co_idm140468803265232"><h3 class="no_margin">Felix Sommer</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Sommer%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Felix Sommer</a></div></div><div id="_co_idm140468803263360"><h3 class="no_margin">David Ellinghaus</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Ellinghaus%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">David Ellinghaus</a></div></div><div id="_co_idm140468803261376"><h3 class="no_margin">Rolf K Berge</h3><p>  <sup>13</sup>Department of Clinical Science, University of Bergen, Bergen, Norway</p><p>  <sup>14</sup>Department of Heart Disease, Haukeland University Hospital, Bergen, Norway</p><div>Find articles by <a href="/pubmed/?term=Berge%20RK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Rolf K Berge</a></div></div><div id="_co_idm140468803041920"><h3 class="no_margin">Matthias Hübenthal</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=H%26%23x000fc%3Bbenthal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Matthias Hübenthal</a></div></div><div id="_co_idm140468803039936"><h3 class="no_margin">Manja Koch</h3><p>  <sup>15</sup>Institute of Epidemiology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Koch%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Manja Koch</a></div></div><div id="_co_idm140468803037952"><h3 class="no_margin">Karin Schwarz</h3><p>  <sup>16</sup>Institute of Human Nutrition and Food Science, University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Schwarz%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Karin Schwarz</a></div></div><div id="_co_idm140468808678112"><h3 class="no_margin">Gerald Rimbach</h3><p>  <sup>16</sup>Institute of Human Nutrition and Food Science, University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Rimbach%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Gerald Rimbach</a></div></div><div id="_co_idm140468808676128"><h3 class="no_margin">Patricia Hübbe</h3><p>  <sup>16</sup>Institute of Human Nutrition and Food Science, University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=H%26%23x000fc%3Bbbe%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Patricia Hübbe</a></div></div><div id="_co_idm140468800851600"><h3 class="no_margin">Wei-Hung Pan</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Pan%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Wei-Hung Pan</a></div></div><div id="_co_idm140468800849616"><h3 class="no_margin">Raheleh Sheibani-Tezerji</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Sheibani-Tezerji%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Raheleh Sheibani-Tezerji</a></div></div><div id="_co_idm140468800847632"><h3 class="no_margin">Robert Häsler</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=H%26%23x000e4%3Bsler%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Robert Häsler</a></div></div><div id="_co_idm140468805618096"><h3 class="no_margin">Philipp Rosenstiel</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Rosenstiel%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Philipp Rosenstiel</a></div></div><div id="_co_idm140468805616112"><h3 class="no_margin">Mauro D’Amato</h3><p>  <sup>17</sup>BioDonostia Health Research Institute, San Sebastian and Ikerbasque, Basque Foundation for Science, Bilbao, Spain</p><p>  <sup>18</sup>Unit of Clinical Epidemiology, Department of Medicine Solna, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=D%26%23x02019%3BAmato%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Mauro D’Amato</a></div></div><div id="_co_idm140468805613392"><h3 class="no_margin">Katja Cloppenborg-Schmidt</h3><p>  <sup>2</sup>Institute for Experimental Medicine, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Cloppenborg-Schmidt%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Katja Cloppenborg-Schmidt</a></div></div><div id="_co_idm140468796525120"><h3 class="no_margin">Sven Künzel</h3><p>  <sup>1</sup>Evolutionary Genomics, Max Planck Institute for Evolutionary Biology, Plön, Germany</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000fc%3Bnzel%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Sven Künzel</a></div></div><div id="_co_idm140468796523136"><h3 class="no_margin">Matthias Laudes</h3><p>  <sup>19</sup>Department of Internal Medicine I, University Hospital S.-H. (UKSH, Campus Kiel), Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Laudes%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Matthias Laudes</a></div></div><div id="_co_idm140468877322896"><h3 class="no_margin">Hanns-Ulrich Marschall</h3><p>  <sup>12</sup>Department of Molecular and Clinical Medicine, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden</p><div>Find articles by <a href="/pubmed/?term=Marschall%20HU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Hanns-Ulrich Marschall</a></div></div><div id="_co_idm140468877321024"><h3 class="no_margin">Wolfgang Lieb</h3><p>  <sup>15</sup>Institute of Epidemiology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Lieb%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Wolfgang Lieb</a></div></div><div id="_co_idm140468877319040"><h3 class="no_margin">Ute Nöthlings</h3><p>  <sup>11</sup>Department of Nutrition and Food Sciences, Nutritional Epidemiology, University of Bonn, Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=N%26%23x000f6%3Bthlings%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Ute Nöthlings</a></div></div><div id="_co_idm140468808016064"><h3 class="no_margin">Tom H Karlsen</h3><p>  <sup>4</sup>Norwegian PSC Research Center, Division of Surgery, Inflammatory Medicine and Transplantation, Oslo, Norway</p><p>  <sup>5</sup>Institute of Clinical Medicine, Faculty of Medicine, University of Oslo, Oslo, Norway</p><p>  <sup>6</sup>K.G. Jebsen Inflammation Research Centre, Institute of Clinical Medicine, University of Oslo, Oslo, Norway</p><p>  <sup>7</sup>Research Institute of Internal Medicine, Oslo University Hospital Rikshospitalet, Oslo, Norway</p><p>  <sup>8</sup>Section of Gastroenterology, Department of Transplantation Medicine, Oslo University Hospital Rikshospitalet, Oslo, Norway</p><p>  <sup>20</sup>Department of Clinical Medicine, University of Bergen, Bergen, Norway</p><div>Find articles by <a href="/pubmed/?term=Karlsen%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Tom H Karlsen</a></div></div><div id="_co_idm140468807820144"><h3 class="no_margin">John F Baines</h3><p>  <sup>1</sup>Evolutionary Genomics, Max Planck Institute for Evolutionary Biology, Plön, Germany</p><p>  <sup>2</sup>Institute for Experimental Medicine, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Baines%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">John F Baines</a></div></div><div id="_co_idm140468807816688"><h3 class="no_margin">Andre Franke</h3><p>  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27723756">Andre Franke</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140468799855392_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140468799855392_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140468799855392_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">  <sup>1</sup>Evolutionary Genomics, Max Planck Institute for Evolutionary Biology, Plön, Germany</div><div class="fm-affl" id="A2" lang="en">  <sup>2</sup>Institute for Experimental Medicine, Christian Albrechts University of Kiel, Kiel, Germany</div><div class="fm-affl" id="A3" lang="en">  <sup>3</sup>Institute of Clinical Molecular Biology, Christian Albrechts University of Kiel, Kiel, Germany</div><div class="fm-affl" id="A4" lang="en">  <sup>4</sup>Norwegian PSC Research Center, Division of Surgery, Inflammatory Medicine and Transplantation, Oslo, Norway</div><div class="fm-affl" id="A5" lang="en">  <sup>5</sup>Institute of Clinical Medicine, Faculty of Medicine, University of Oslo, Oslo, Norway</div><div class="fm-affl" id="A6" lang="en">  <sup>6</sup>K.G. Jebsen Inflammation Research Centre, Institute of Clinical Medicine, University of Oslo, Oslo, Norway</div><div class="fm-affl" id="A7" lang="en">  <sup>7</sup>Research Institute of Internal Medicine, Oslo University Hospital Rikshospitalet, Oslo, Norway</div><div class="fm-affl" id="A8" lang="en">  <sup>8</sup>Section of Gastroenterology, Department of Transplantation Medicine, Oslo University Hospital Rikshospitalet, Oslo, Norway</div><div class="fm-affl" id="A9" lang="en">  <sup>9</sup>Estonian Genome Center, University of Tartu, Tartu, Estonia</div><div class="fm-affl" id="A10" lang="en">  <sup>10</sup>Division of Gastroenterology and Hepatology, Department of Medicine, University of Illinois at Chicago, Chicago, Illinois, USA</div><div class="fm-affl" id="A11" lang="en">  <sup>11</sup>Department of Nutrition and Food Sciences, Nutritional Epidemiology, University of Bonn, Bonn, Germany</div><div class="fm-affl" id="A12" lang="en">  <sup>12</sup>Department of Molecular and Clinical Medicine, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden</div><div class="fm-affl" id="A13" lang="en">  <sup>13</sup>Department of Clinical Science, University of Bergen, Bergen, Norway</div><div class="fm-affl" id="A14" lang="en">  <sup>14</sup>Department of Heart Disease, Haukeland University Hospital, Bergen, Norway</div><div class="fm-affl" id="A15" lang="en">  <sup>15</sup>Institute of Epidemiology, Christian Albrechts University of Kiel, Kiel, Germany</div><div class="fm-affl" id="A16" lang="en">  <sup>16</sup>Institute of Human Nutrition and Food Science, University of Kiel, Kiel, Germany</div><div class="fm-affl" id="A17" lang="en">  <sup>17</sup>BioDonostia Health Research Institute, San Sebastian and Ikerbasque, Basque Foundation for Science, Bilbao, Spain</div><div class="fm-affl" id="A18" lang="en">  <sup>18</sup>Unit of Clinical Epidemiology, Department of Medicine Solna, Karolinska Institutet, Stockholm, Sweden</div><div class="fm-affl" id="A19" lang="en">  <sup>19</sup>Department of Internal Medicine I, University Hospital S.-H. (UKSH, Campus Kiel), Kiel, Germany</div><div class="fm-affl" id="A20" lang="en">  <sup>20</sup>Department of Clinical Medicine, University of Bergen, Bergen, Norway</div><div id="FN1">Correspondence should be addressed to A.F. (<a href="mailto:dev@null" data-email="ed.asocum@eknarf.a" class="oemail">ed.asocum@eknarf.a</a>)</div><div id="FN2"><sup>21</sup>Present addresses: Department of Microbiology and Immunology, KU Leuven and Center for the Biology of Disease, VIB, Leuven, Belgium (J.W.) and Institute of Medical Informatics and Statistics, Christian Albrechts University of Kiel, Kiel, Germany (S.S.).</div><div id="FN3"><sup>22</sup>These authors contributed equally to this work.</div><div id="FN4"><sup>23</sup>These authors jointly directed this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140468799855392_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div class="license half_rhythm">Reprints and permissions information is available online at <a href="http://www.nature.com/reprints/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/reprints/index.html</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=27723756" target="pmc_ext" ref="reftype=publisher&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See the addendum "<a href="/pmc/articles/PMC5939986/">Application of the distance-based F test in an mGWAS investigating β diversity of intestinal microbiota identifies variants in <em>SLC9A8</em> (NHE8) and 3 other loci</a>" in <em>Gut Microbes</em>, volume 9 on&nbsp;page&nbsp;68.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC5626933/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC5626933/bin/NIHMS904581-supplement-2.pdf" data-ga-action="click_feat_suppl">NIHMS904581-supplement-2.pdf</a><span style="color:gray"> (13M)</span></div><div class="small guid">GUID:&nbsp;F8DDF049-EE0E-471E-8C6D-21E95F5D5DC7</div></div></dd></dl></div><div id="idm140468801850608" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140468801850608title">Abstract</h2><!--article-meta--><div><p id="P4" class="p p-first-last">Human gut microbiota is an important determinant for health and disease, and recent studies emphasize the numerous factors shaping its diversity. Here we performed a genome-wide association study (GWAS) of the gut microbiota using two cohorts from northern Germany totaling 1,812 individuals. Comprehensively controlling for diet and non-genetic parameters, we identify genome-wide significant associations for overall microbial variation and individual taxa at multiple genetic loci, including the <em>VDR</em> gene (encoding vitamin D receptor). We observe significant shifts in the microbiota of <em>Vdr</em><sup>−/−</sup> mice relative to control mice and correlations between the microbiota and serum measurements of selected bile and fatty acids in humans, including known ligands and downstream metabolites of VDR. Genome-wide significant (<em>P</em> &lt; 5 × 10<sup>−8</sup>) associations at multiple additional loci identify other important points of host–microbe intersection, notably several disease susceptibility genes and sterol metabolism pathway components. Non-genetic and genetic factors each account for approximately 10% of the variation in gut microbiota, whereby individual effects are relatively small.</p></div></div><div id="idm140468843369856" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">Microbes inhabiting the human intestine mediate key metabolic, physiological and immune functions<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112219">1</a>,<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112215">2</a></sup>, and perturbations of this ecosystem can profoundly influence health and disease<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112200">3</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112181">4</a></sup>. As disease states can also impose secondary changes to the gut microbiota, a fundamental understanding of the forces determining gut micro-bial composition in healthy individuals is essential for deciphering the nature of disease states and developing therapeutic strategies. Assemblage of the gut community begins at birth<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112225">5</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112216">6</a></sup>, and, once established, compositional features are resilient to perturbations<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112154">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112228">8</a></sup>. The composition of the gut microbiota is highly variable among adults<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112203">9</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112159">10</a></sup>, although family members tend to harbor more similar communities than unrelated individuals<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112202">11</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112214">12</a></sup>. Both genetic and environmental determinants may underlie this similarity among familial microbiomes. Diet is one of the major environmental drivers for microbial community structure<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112213">13</a>,<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112230">14</a></sup>, and other known factors include age and geography<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112152">11</a>,<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112170">15</a></sup> as well as the intake of medication<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112167">16</a></sup>.</p><p id="P6" class="p">There is increasing support for a host genetic component shaping and/or structuring between-individual variability in the gut microbiota. Using 416 twin pairs, Goodrich <em>et al</em>.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112175">12</a></sup> showed that monozygotic twins display greater overall similarity in their microbial communities than dizygotic twins and identified microbial taxa that were affected by host genetic variation. Influence of single candidate genes on the composition of the microbiome is also suggested by studies of the human gut mucosa (<em>FUT2</em>; ref. <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112232">17</a>) or in mouse models (<em>Nod2</em>; ref. <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112169">18</a>). A recent study using available Human Microbiome Project (HMP) metagen-omic sequencing data<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112227">19</a></sup> assessed associations between genome-wide genetic variation in humans and the microbiome and identified an association between the <em>LCT</em> gene and the abundance of bacteria in the <em>Bifidobacterium</em> genus. However, a small sample size (<em>n</em> = 93) and lack of thorough correction for known confounding factors (such as diet) represent drawbacks of this study. Here we report the results from a well-powered systematic host GWAS of the fecal microbiome in two independent but geographically matched cohorts totaling 1,812 individuals of European ancestry. A dense genomic marker set comprising a total of 6,344,846 genotyped and imputed SNPs and extensive metadata were included in the analyses, which enabled us to study the influence of host genotype, alongside dietary and other environmental factors, on between-individual variability in the gut microbiome.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Establishing covariables for the genetic analysis</h3><p id="P7" class="p p-first">Fecal samples were obtained from two independent cohorts of 914 individuals (PopGen<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112194">20</a></sup>) and 1,115 individuals (Food-Chain Plus; FoCus<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112205">21</a></sup>), both recruited at the University Hospital Schleswig-Holstein in the city area of Kiel, Germany, through the local Biobank PopGen<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112184">20</a></sup>. For each of the 2,029 samples, high-quality 16S rRNA gene sequence data (minimum of 10,000 reads/sample) were generated, yielding a total of 38 and 374 identified phyla and genera, respectively. The two cohorts exhibited similar taxon abundance at high (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>) and low (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>) taxo-nomic levels, although small differences in β diversity (Bray–Curtis) were present between the cohorts (<em>r</em><sup>2</sup> = 0.026; <em>P</em> = 1 × 10<sup>−3</sup>), which were due to differences in age, body mass index (BMI) and sex ratio (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). A subset of 1,812 of the 2,029 individuals had available SNP array data in addition to the 16S rRNA data. Unless otherwise noted, results are presented for the combined cohort of 1,812 individuals, that is, PopGen and FoCus (results for individual cohorts are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 1–3, Supplementary Table 1 and the Supplementary Note</a>).</p><p id="P8">Variables previously reported to influence the gut microbiota, including age, sex, BMI<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112189">11</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112178">12</a></sup> and smoking status<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112168">22</a></sup>, all displayed significant correlations with variability in the microbiome (<em>P</em> &lt; 0.05; <a href="/pmc/articles/PMC5626933/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4 and Supplementary Table 1</a>). In terms of the percentage of variation explained (as determined through principal-coordinate analysis (PCoA) applied to Bray–Curtis dissimilarity (BC), a β-diversity measure that reflects between-individual variability), age accounted for the greatest amount (4.74%) in the combined cohort, followed by BMI, smoking and sex (3.79%, 2.14% and 1.79%, respectively; <a href="/pmc/articles/PMC5626933/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC5626933/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140468807360480" class="figure"><a href="/pmc/articles/PMC5626933/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5626933_nihms904581f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms904581f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5626933/bin/nihms904581f1.jpg"></a></div><div id="largeobj_idm140468807360480" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5626933/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5626933/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Overview of variation in the gut microbiota and significantly associated non-genetic parameters</strong><p id="P54">(<strong>a</strong>) PCoA of the combined cohort using BC. Arrows represent increases in the eight most abundant genera (arrow length is proportional to mean abundance; scale bar); <em>n</em>= 1,812. Samples are colored according to cohort. MDS1 and MDS2 are the two major axes from PCoA. (<strong>b</strong>) Correlation of age, BMI and smoking status with microbiota. For age and BMI, green arrows denote effect size (variation in β diversity explained; scale bar). Differences in smoking status are depicted as two circles with different centroids, with the dashed lines containing 50% of the samples for each group (for visualization). (<strong>c</strong>) Correlation of major nutrients with microbiota, with red arrows denoting effect size (variation in β diversity explained; scale bar). As most individual nutrients are co-linear with total energy, all arrows, save for the one for total energy, show the increase in standardized nutrient values (calculated by nutrient/total energy).</p></div></div></div><p id="P9" class="p p-last">Moreover, using available food frequency data, we performed a systematic analysis of long-term diet and nutrients with respect to the microbiome. Using either the major food groups (for example, vegetables) or nutrients (for example, dietary protein content; <a href="/pmc/articles/PMC5626933/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4 and Supplementary Table 1</a>), we quantified the contribution to observed variability in the microbiome (PCoA applied to BC). We found that eight of the nine major nutrients and 12 of the 17 food groups displayed significant correlations in at least one cohort (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>), and overall diet was significantly associated with the landscape of the human gut microbiome and explained 5.79% of the variation in BC (<a href="/pmc/articles/PMC5626933/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Addition of the other significant covariates (age, sex, BMI and smoking status; <em>P</em> &lt; 0.05) resulted in a total of 8.87% of the variation in BC explained (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>). Given their detectable influence on between-individual variability, dietary variables, that is, water, alcohol and ‘total energy’ (as the best proxy for other co-linear nutrients with Pearson <em>r</em> &gt; 0.5), were included as covariates in the subsequent host SNP versus microbiome association analyses.</p></div><div id="S3" class="sec"><h3 id="S3title">Host genetic loci influence microbial β diversity</h3><p id="P10" class="p p-first">Between-individual variability is measured by β-diversity indices, which represent overall differences between microbial communities in the population and are driven by variation among multiple taxa. To identify individual loci contributing to βdiversity, we employed a multidimensional ANOVA approach, for which significance thresholds were determined for distinct classes of minor allele frequency (MAF) by performing &gt;2 × 10<sup>7</sup> permutations to simulate the largest possible effect size (percentage variation in β diversity explained) that can occur by chance (for details, see the Online Methods). After stringent filtering based on effect sizes in the cohorts separately as well as in combination (Online Methods), this analysis showed 42 loci to be associated with β diversity (<em>P</em> &lt; 5 × 10<sup>−8</sup>; <a href="/pmc/articles/PMC5626933/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="/pmc/articles/PMC5626933/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), each of which contributed from 0.65 to 0.97% of the variation in community structure (measured by BC) and additively explained 10.43% in the combined cohort (<a href="/pmc/articles/PMC5626933/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). Of these loci, 21 could be successfully replicated in a smaller, independent cohort composed of obese individuals (FoCus obesity, <em>n</em> = 371), recruited from the same geographic area (Online Methods and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC5626933/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140468808707216" class="figure"><a href="/pmc/articles/PMC5626933/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5626933_nihms904581f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms904581f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5626933/bin/nihms904581f2.jpg"></a></div><div id="largeobj_idm140468808707216" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5626933/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5626933/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Individual and combined effects of significant loci and overview of all significant loci identified in this study</strong><p id="P55">(<strong>a</strong>) Effect sizes (variation in β diversity explained) for the 42 significant loci (lead SNPs) are shown in decreasing order (left axis), and additive effects (Online Methods) are shown by the dashed line (right axis). (<strong>b</strong>) Chromosomes on the right side of the plot show the chromosomal position of genes significantly associated with β diversity (black) or an individual taxon (blue). The inner circle includes genes whose mouse homologs were implicated in one or more previously published mouse QTL studies<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112186">35</a>,<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112185">47</a>–<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112198">50</a></sup>(<a href="#SD1" rid="SD1" class=" supplementary-material">supplementary tables 6 and 7</a>), denoted by a link to the corresponding mouse chromosome and appearing in the same color as the human chromosome on which the gene is located. For genes located in the outer circle, either there is no mouse homolog or the mouse homolog does not fall within a QTL.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P57">Summary of loci significantly associated with β diversity</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140468843048208" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Locus</th><th rowspan="1" colspan="1" valign="bottom" align="right">SNP ID</th><th rowspan="1" colspan="1" valign="bottom" align="right">Chr.</th><th rowspan="1" colspan="1" valign="bottom" align="center">A1</th><th rowspan="1" colspan="1" valign="bottom" align="center">A2</th><th rowspan="1" colspan="1" valign="bottom" align="right">Locus start</th><th rowspan="1" colspan="1" valign="bottom" align="right">Locus end</th><th rowspan="1" colspan="1" valign="bottom" align="center">Nearest gene</th><th rowspan="1" colspan="1" valign="bottom" align="center">Genes in locus</th><th rowspan="1" colspan="1" valign="bottom" align="center">Effect<br>size (%)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">1</td><td rowspan="1" colspan="1" align="right">rs804427</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">33,538,964</td><td rowspan="1" colspan="1" align="right">33,623,510</td><td rowspan="1" colspan="1" align="center"><em>AK2</em></td><td rowspan="1" colspan="1" align="center"><em>ADC, TRIM62, AK2</em></td><td rowspan="1" colspan="1" align="center">0.79</td></tr><tr><td rowspan="1" colspan="1" align="left">2</td><td rowspan="1" colspan="1" align="right">rs1288616</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="right">53,885,577</td><td rowspan="1" colspan="1" align="right">53,965,248</td><td rowspan="1" colspan="1" align="center"><em>DMRTB1</em></td><td rowspan="1" colspan="1" align="center"><em>DMRTB1</em></td><td rowspan="1" colspan="1" align="center">0.76</td></tr><tr><td rowspan="1" colspan="1" align="left">3</td><td rowspan="1" colspan="1" align="right">rs1102737</td><td rowspan="1" colspan="1" align="right">1</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="right">172,700,868</td><td rowspan="1" colspan="1" align="right">172,779,833</td><td rowspan="1" colspan="1" align="center"><em>FASLG</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.66</td></tr><tr><td rowspan="1" colspan="1" align="left">4</td><td rowspan="1" colspan="1" align="right">rs72853661</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">25,323,083</td><td rowspan="1" colspan="1" align="right">25,453,968</td><td rowspan="1" colspan="1" align="center"><em>POMC</em></td><td rowspan="1" colspan="1" align="center"><em>POMC, EFR3B</em></td><td rowspan="1" colspan="1" align="center">0.79</td></tr><tr><td rowspan="1" colspan="1" align="left">5</td><td rowspan="1" colspan="1" align="right">rs7567349</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">61,384,324</td><td rowspan="1" colspan="1" align="right">61,853,037</td><td rowspan="1" colspan="1" align="center"><em>XPO1</em></td><td rowspan="1" colspan="1" align="center"><em>AHSA2, USP34, XPO1, KIAA1841</em></td><td rowspan="1" colspan="1" align="center">0.76</td></tr><tr><td rowspan="1" colspan="1" align="left">6</td><td rowspan="1" colspan="1" align="right">rs2010917</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">135,172,338</td><td rowspan="1" colspan="1" align="right">135,197,891</td><td rowspan="1" colspan="1" align="center"><em>MGAT 5</em></td><td rowspan="1" colspan="1" align="center"><em>MGAT 5</em></td><td rowspan="1" colspan="1" align="center">0.74</td></tr><tr><td rowspan="1" colspan="1" align="left">7</td><td rowspan="1" colspan="1" align="right">rs71415332</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="right">102,309,520</td><td rowspan="1" colspan="1" align="right">102,616,128</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center"><em>IL1R2, MAP4K4</em></td><td rowspan="1" colspan="1" align="center">0.68</td></tr><tr><td rowspan="1" colspan="1" align="left">8</td><td rowspan="1" colspan="1" align="right">rs4670302</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">33,808,725</td><td rowspan="1" colspan="1" align="right">34,068,392</td><td rowspan="1" colspan="1" align="center"><em>FAM98A</em></td><td rowspan="1" colspan="1" align="center"><em>FAM98A</em></td><td rowspan="1" colspan="1" align="center">0.92</td></tr><tr><td rowspan="1" colspan="1" align="left">9</td><td rowspan="1" colspan="1" align="right">rs6711771</td><td rowspan="1" colspan="1" align="right">2</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">34,339,420</td><td rowspan="1" colspan="1" align="right">34,491,584</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center">0.71</td></tr><tr><td rowspan="1" colspan="1" align="left">10</td><td rowspan="1" colspan="1" align="right">rs13099587</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="right">146,250,561</td><td rowspan="1" colspan="1" align="right">146,275,555</td><td rowspan="1" colspan="1" align="center"><em>PLSCR1</em></td><td rowspan="1" colspan="1" align="center"><em>PLSCR1</em></td><td rowspan="1" colspan="1" align="center">0.70</td></tr><tr><td rowspan="1" colspan="1" align="left">11</td><td rowspan="1" colspan="1" align="right">rs9647379</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">171,759,410</td><td rowspan="1" colspan="1" align="right">171,833,266</td><td rowspan="1" colspan="1" align="center"><em>FNDC3B</em></td><td rowspan="1" colspan="1" align="center"><em>FNDC3B</em></td><td rowspan="1" colspan="1" align="center">0.75</td></tr><tr><td rowspan="1" colspan="1" align="left">12</td><td rowspan="1" colspan="1" align="right">rs143050036</td><td rowspan="1" colspan="1" align="right">3</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="right">49,898,318</td><td rowspan="1" colspan="1" align="right">50,208,819</td><td rowspan="1" colspan="1" align="center"><em>SEMA3F</em></td><td rowspan="1" colspan="1" align="center"><em>RBM5, MST1R, CAMKV, MON1A, RBM6, SEMA3F</em></td><td rowspan="1" colspan="1" align="center">0.75</td></tr><tr><td rowspan="1" colspan="1" align="left">13</td><td rowspan="1" colspan="1" align="right">rs60500975</td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="right">102,769,693</td><td rowspan="1" colspan="1" align="right">102,929,034</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center"><em>BANK1</em></td><td rowspan="1" colspan="1" align="center">0.82</td></tr><tr><td rowspan="1" colspan="1" align="left">14</td><td rowspan="1" colspan="1" align="right">rs62367773</td><td rowspan="1" colspan="1" align="right">5</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">74,171,398</td><td rowspan="1" colspan="1" align="right">74,220,999</td><td rowspan="1" colspan="1" align="center"><em>FAM169A</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.67</td></tr><tr><td rowspan="1" colspan="1" align="left">15</td><td rowspan="1" colspan="1" align="right">rs1292672</td><td rowspan="1" colspan="1" align="right">6</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="right">87,217,958</td><td rowspan="1" colspan="1" align="right">87,509,434</td><td rowspan="1" colspan="1" align="center"><em>HTR1E</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.70</td></tr><tr><td rowspan="1" colspan="1" align="left">16</td><td rowspan="1" colspan="1" align="right">rs35148810</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="right">151,515,842</td><td rowspan="1" colspan="1" align="right">151,530,983</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center"><em>PRKAG2</em></td><td rowspan="1" colspan="1" align="center">0.83</td></tr><tr><td rowspan="1" colspan="1" align="left">17</td><td rowspan="1" colspan="1" align="right">rs12705241</td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">104,219,681</td><td rowspan="1" colspan="1" align="right">104,381,102</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center"><em>LHFPL3</em></td><td rowspan="1" colspan="1" align="center">0.76</td></tr><tr><td rowspan="1" colspan="1" align="left">18</td><td rowspan="1" colspan="1" align="right">rs13260600</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="right">3,705,807</td><td rowspan="1" colspan="1" align="right">3,713,004</td><td rowspan="1" colspan="1" align="center"><em>CSMD1</em></td><td rowspan="1" colspan="1" align="center"><em>CSMD1</em></td><td rowspan="1" colspan="1" align="center">0.77</td></tr><tr><td rowspan="1" colspan="1" align="left">19</td><td rowspan="1" colspan="1" align="right">rs138022915</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">19,815,256</td><td rowspan="1" colspan="1" align="right">19,939,049</td><td rowspan="1" colspan="1" align="center"><em>LPL</em></td><td rowspan="1" colspan="1" align="center"><em>LPL</em></td><td rowspan="1" colspan="1" align="center">0.73</td></tr><tr><td rowspan="1" colspan="1" align="left">20</td><td rowspan="1" colspan="1" align="right">rs11986935</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="right">10,576,753</td><td rowspan="1" colspan="1" align="right">10,732,050</td><td rowspan="1" colspan="1" align="center"><em>SOX7</em></td><td rowspan="1" colspan="1" align="center"><em>SOX7, PINX1</em></td><td rowspan="1" colspan="1" align="center">0.97</td></tr><tr><td rowspan="1" colspan="1" align="left">21</td><td rowspan="1" colspan="1" align="right">rs7818750</td><td rowspan="1" colspan="1" align="right">8</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="right">135,273,640</td><td rowspan="1" colspan="1" align="right">135,299,611</td><td rowspan="1" colspan="1" align="center"><em>ZFAT</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.74</td></tr><tr><td rowspan="1" colspan="1" align="left">22</td><td rowspan="1" colspan="1" align="right">rs1325919</td><td rowspan="1" colspan="1" align="right">9</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="right">37,626,956</td><td rowspan="1" colspan="1" align="right">37,650,386</td><td rowspan="1" colspan="1" align="center"><em>FRMPD1</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.67</td></tr><tr><td rowspan="1" colspan="1" align="left">23</td><td rowspan="1" colspan="1" align="right">rs7082134</td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">87,865,009</td><td rowspan="1" colspan="1" align="right">87,884,110</td><td rowspan="1" colspan="1" align="center"><em>GRID1</em></td><td rowspan="1" colspan="1" align="center"><em>GRID1</em></td><td rowspan="1" colspan="1" align="center">0.84</td></tr><tr><td rowspan="1" colspan="1" align="left">24</td><td rowspan="1" colspan="1" align="right">rs2251536</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">8,852,239</td><td rowspan="1" colspan="1" align="right">8,853,177</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center"><em>ST5</em></td><td rowspan="1" colspan="1" align="center">0.76</td></tr><tr><td rowspan="1" colspan="1" align="left">25</td><td rowspan="1" colspan="1" align="right">rs4472950</td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="right">120,798,714</td><td rowspan="1" colspan="1" align="right">120,853,675</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center"><em>GRIK4</em></td><td rowspan="1" colspan="1" align="center">0.69</td></tr><tr><td rowspan="1" colspan="1" align="left">26</td><td rowspan="1" colspan="1" align="right">rs7974353</td><td rowspan="1" colspan="1" align="right">12</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">48,256,280</td><td rowspan="1" colspan="1" align="right">48,270,596</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center"><em>VDR</em></td><td rowspan="1" colspan="1" align="center">0.75</td></tr><tr><td rowspan="1" colspan="1" align="left">27</td><td rowspan="1" colspan="1" align="right">rs4760399</td><td rowspan="1" colspan="1" align="right">12</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">93,011,759</td><td rowspan="1" colspan="1" align="right">93,081,307</td><td rowspan="1" colspan="1" align="center"><em>C12orf74</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.67</td></tr><tr><td rowspan="1" colspan="1" align="left">28</td><td rowspan="1" colspan="1" align="right">rs6573564</td><td rowspan="1" colspan="1" align="right">14</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="right">65,119,676</td><td rowspan="1" colspan="1" align="right">65,157,187</td><td rowspan="1" colspan="1" align="center"><em>PLEKHG3</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.73</td></tr><tr><td rowspan="1" colspan="1" align="left">29</td><td rowspan="1" colspan="1" align="right">rs12910631</td><td rowspan="1" colspan="1" align="right">15</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="right">26,603,288</td><td rowspan="1" colspan="1" align="right">26,622,999</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.79</td></tr><tr><td rowspan="1" colspan="1" align="left">30</td><td rowspan="1" colspan="1" align="right">rs8040493</td><td rowspan="1" colspan="1" align="right">15</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">101,414,167</td><td rowspan="1" colspan="1" align="right">101,418,682</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.65</td></tr><tr><td rowspan="1" colspan="1" align="left">31</td><td rowspan="1" colspan="1" align="right">rs293377</td><td rowspan="1" colspan="1" align="right">15</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">89,623,490</td><td rowspan="1" colspan="1" align="right">89,635,268</td><td rowspan="1" colspan="1" align="center"><em>ABHD2</em></td><td rowspan="1" colspan="1" align="center"><em>ABHD2</em></td><td rowspan="1" colspan="1" align="center">0.70</td></tr><tr><td rowspan="1" colspan="1" align="left">32</td><td rowspan="1" colspan="1" align="right">rs8055365</td><td rowspan="1" colspan="1" align="right">16</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">84,566,729</td><td rowspan="1" colspan="1" align="right">84,581,275</td><td rowspan="1" colspan="1" align="center"><em>KIAA1609</em></td><td rowspan="1" colspan="1" align="center"><em>KIAA1609</em></td><td rowspan="1" colspan="1" align="center">0.70</td></tr><tr><td rowspan="1" colspan="1" align="left">33</td><td rowspan="1" colspan="1" align="right">rs59986499</td><td rowspan="1" colspan="1" align="right">16</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="right">3,065,924</td><td rowspan="1" colspan="1" align="right">3,097,940</td><td rowspan="1" colspan="1" align="center"><em>CLDN6</em></td><td rowspan="1" colspan="1" align="center"><em>MMP25, TNFRSF12A, CLDN6, CCDC64B, HCFC1R1, THOC6</em></td><td rowspan="1" colspan="1" align="center">0.68</td></tr><tr><td rowspan="1" colspan="1" align="left">34</td><td rowspan="1" colspan="1" align="right">rs12931878</td><td rowspan="1" colspan="1" align="right">16</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">11,031,741</td><td rowspan="1" colspan="1" align="right">11,207,817</td><td rowspan="1" colspan="1" align="center"><em>CLEC16A</em></td><td rowspan="1" colspan="1" align="center"><em>DEXI, CLEC16A</em></td><td rowspan="1" colspan="1" align="center">0.65</td></tr><tr><td rowspan="1" colspan="1" align="left">35</td><td rowspan="1" colspan="1" align="right">rs62085746</td><td rowspan="1" colspan="1" align="right">17</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="right">66,166,300</td><td rowspan="1" colspan="1" align="right">66,213,540</td><td rowspan="1" colspan="1" align="center"><em>AMZ2</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.69</td></tr><tr><td rowspan="1" colspan="1" align="left">36</td><td rowspan="1" colspan="1" align="right">rs16969051</td><td rowspan="1" colspan="1" align="right">17</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="right">32,248,813</td><td rowspan="1" colspan="1" align="right">32,258,877</td><td rowspan="1" colspan="1" align="center"><em>ACCN1</em></td><td rowspan="1" colspan="1" align="center"><em>ACCN1</em></td><td rowspan="1" colspan="1" align="center">0.65</td></tr><tr><td rowspan="1" colspan="1" align="left">37</td><td rowspan="1" colspan="1" align="right">rs12601692</td><td rowspan="1" colspan="1" align="right">17</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">782,416</td><td rowspan="1" colspan="1" align="right">794,333</td><td rowspan="1" colspan="1" align="center">–</td><td rowspan="1" colspan="1" align="center"><em>NXN</em></td><td rowspan="1" colspan="1" align="center">0.68</td></tr><tr><td rowspan="1" colspan="1" align="left">38</td><td rowspan="1" colspan="1" align="right">rs2267922</td><td rowspan="1" colspan="1" align="right">19</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">18,217,350</td><td rowspan="1" colspan="1" align="right">18,289,634</td><td rowspan="1" colspan="1" align="center"><em>IFI30</em></td><td rowspan="1" colspan="1" align="center"><em>MAST3, IFI30, PIK3R2</em></td><td rowspan="1" colspan="1" align="center">0.77</td></tr><tr><td rowspan="1" colspan="1" align="left">39</td><td rowspan="1" colspan="1" align="right">rs273647</td><td rowspan="1" colspan="1" align="right">19</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">51,739,767</td><td rowspan="1" colspan="1" align="right">51,766,748</td><td rowspan="1" colspan="1" align="center"><em>C19orf75</em></td><td rowspan="1" colspan="1" align="center"><em>CD33, C19orf75</em></td><td rowspan="1" colspan="1" align="center">0.84</td></tr><tr><td rowspan="1" colspan="1" align="left">40</td><td rowspan="1" colspan="1" align="right">rs4809760</td><td rowspan="1" colspan="1" align="right">20</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">48,428,863</td><td rowspan="1" colspan="1" align="right">48,591,125</td><td rowspan="1" colspan="1" align="center"><em>SLC9A8</em></td><td rowspan="1" colspan="1" align="center"><em>RNF114, SLC9A8, SPATA2</em></td><td rowspan="1" colspan="1" align="center">0.85</td></tr><tr><td rowspan="1" colspan="1" align="left">41</td><td rowspan="1" colspan="1" align="right">rs2835692</td><td rowspan="1" colspan="1" align="right">21</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="right">38,657,572</td><td rowspan="1" colspan="1" align="right">38,704,886</td><td rowspan="1" colspan="1" align="center"><em>DSCR3</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.68</td></tr><tr><td rowspan="1" colspan="1" align="left">42</td><td rowspan="1" colspan="1" align="right">rs9917541</td><td rowspan="1" colspan="1" align="right">22</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="right">31,520,338</td><td rowspan="1" colspan="1" align="right">31,531,133</td><td rowspan="1" colspan="1" align="center"><em>PLA2G3</em></td><td rowspan="1" colspan="1" align="center"><em>PLA2G3, INPP5J</em></td><td rowspan="1" colspan="1" align="center">0.71</td></tr></tbody></table></div><div id="largeobj_idm140468843048208" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5626933/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P58" class="p p-first-last">All loci have effect sizes greater than the signifcance threshold calculated from the null distribution (<em>P</em> &lt; 5 × 10<sup>−8</sup>; Online Methods). The name of the lead SNP, chromosome, position, nearest gene and genes within a locus were determined using DEPICT software. A1 and A2 are the reference/alternative alleles based on the 1000 Genome Project. Chr., chromosome.</p></div></div></div><p id="P11">Interestingly, variants in the <em>VDR</em> gene (encoding vitamin D receptor) were among the 42 significant loci and accounted for 0.75% of the variation in the combined cohort (<a href="/pmc/articles/PMC5626933/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). <em>VDR</em> encodes a nuclear transcription factor, which through heterodimerization with the retinoid X receptor (RXR) exerts a range of physiological effects with many known exogenous and endogenous ligands. Besides vitamin D, both microbial (for example, secondary bile acids) and dietary (for example, fatty acids) metabolites act via the VDR–RXR heterodimer<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112190">23</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112165">24</a></sup>. To further explore this association, we analyzed gut microbiota data from a published <em>Vdr</em><sup>−/−</sup> mouse model<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112172">25</a></sup>, confirming that loss of <em>Vdr</em> in mice substantially affects β diversity (42% variation in BC explained in this controlled setting; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a>). Detailed exploration of parallels between human and mouse microbiota also showed that <em>VDR</em> consistently influences individual bacterial taxa such as <em>Parabacteroides</em> (<a href="/pmc/articles/PMC5626933/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3c,d</span></a>; additional taxa are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 6</a>). Incidentally, in another data set, we observed upregulation of VDR in the colonic biopsies of patients with acute inflammation, Crohn’s disease or ulcerative colitis as compared to healthy controls, accompanied by much lower abundance of <em>Parabacteroides</em>, thereby further supporting such interaction (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 7 and Supplementary Note</a>). Of note, enrichment analysis of genetic loci significantly associated with individual taxa (<a href="/pmc/articles/PMC5626933/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) showed vitamin D response as the fourth most significantly associated gene set (<a href="/pmc/articles/PMC5626933/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC5626933/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140468843050864" class="figure"><a href="/pmc/articles/PMC5626933/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5626933_nihms904581f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms904581f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5626933/bin/nihms904581f3.jpg"></a></div><div id="largeobj_idm140468843050864" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5626933/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC5626933/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong><em>VDR</em> and <em>POMC</em> as examples of genes associated with β diversity</strong><p id="P56">(<strong>a</strong>) LocusZoom plot of adjusted effect size (for each SNP, the actual effect size is divided by the significance threshold adjusted according to MAF category, represented by the dashed line; Online Methods) at the <em>VDR</em> locus, where two SNPs passed the significance threshold for association with β diversity (<em>P</em> &lt; 5 × 10<sup>−8</sup> for association with overall microbiome variation, measured by BC). (<strong>b</strong>) Association between genotypes at the lead SNP (rs7974353) and β diversity (BC). Microbiome data are shown in a PCoA plot; the dashed lines contain 50% of the samples for each group (for visualization) and show differences in the centroids for each genotype group; <em>n</em> = 1,812. (<strong>c</strong>) Meta-analysis in humans shows <em>Parabacteroides</em> to be the most significant taxon correlated with <em>VDR</em> using a GLM (Online Methods). The <em>x</em> axis shows the percentage of nonzero values for each genotype at rs7974353, and boxes and bars summarize 50% and 95% confidence intervals, respectively, for nonzero values; <em>n</em> = 1,812. (<strong>d</strong>) Knockout of <em>Vdr</em><sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112206">25</a></sup> in mice also leads to changes in <em>Parabacteroides</em> abundance. Error bars, 5–95% confidence intervals (<em>n</em> = 3 wild-type (WT) mice and <em>n</em> = 5 knockout mice; <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary Fig. 6 and supplementary Note</a>). (<strong>e</strong>) LocusZoom plot for adjusted effect size in the region upstream of <em>POMC</em>, where 78 SNPs passed the significance threshold. (<strong>f</strong>) Association between the genotypes of the lead SNP at <em>POMC</em> (rs72853661) and β diversity (BC). Microbiome data are shown in a PCoA plot; the dashed lines contain 50% of the samples for each group (for visualization) and show differences in the centroids for each genotype group; <em>n</em> = 1,812.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P59">Loci associated with bacterial abundance</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140468797789488" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Locus</th><th rowspan="1" colspan="1" valign="bottom" align="left">Bacteria</th><th rowspan="1" colspan="1" valign="bottom" align="right">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">A1</th><th rowspan="1" colspan="1" valign="bottom" align="center">A2</th><th rowspan="1" colspan="1" valign="bottom" align="center">Meta <em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">Meta <em>β</em></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>β</em>-div <em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">Chr.</th><th rowspan="1" colspan="1" valign="bottom" align="center">Locus start</th><th rowspan="1" colspan="1" valign="bottom" align="center">Locus end</th><th rowspan="1" colspan="1" valign="bottom" align="center">Nearest gene</th><th rowspan="1" colspan="1" valign="bottom" align="center">Genes in<br>locus</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">1</td><td rowspan="1" colspan="1" align="left">Unclassifed Enterobacteriaceae</td><td rowspan="1" colspan="1" align="right">rs938295</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">2.34 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.49</td><td rowspan="1" colspan="1" align="center">0.76</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">16,087,164</td><td rowspan="1" colspan="1" align="center">16,124,985</td><td rowspan="1" colspan="1" align="center"><em>FBLIM1</em></td><td rowspan="1" colspan="1" align="center"><em>FBLIM1</em></td></tr><tr><td rowspan="1" colspan="1" align="left">2</td><td rowspan="1" colspan="1" align="left">Unclassifed Acidaminococcaceae</td><td rowspan="1" colspan="1" align="right">rs75036654</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">4.94 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">−1.39</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">37,717,219</td><td rowspan="1" colspan="1" align="center">37,780,821</td><td rowspan="1" colspan="1" align="center"><em>LINC01137</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">3</td><td rowspan="1" colspan="1" align="left">OTU13305 <em>Fecalibacterium</em> Species-level OTU</td><td rowspan="1" colspan="1" align="right">rs597205</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">7.68 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−0.62</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">112,379,026</td><td rowspan="1" colspan="1" align="center">112,415,622</td><td rowspan="1" colspan="1" align="center"><em>C1orf183</em></td><td rowspan="1" colspan="1" align="center"><em>C1orf183</em></td></tr><tr><td rowspan="1" colspan="1" align="left">4</td><td rowspan="1" colspan="1" align="left"><em>Blautia</em> genus</td><td rowspan="1" colspan="1" align="right">rs4669413</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">1.2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.18</td><td rowspan="1" colspan="1" align="center">0.75</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">9,801,744</td><td rowspan="1" colspan="1" align="center">9,818,596</td><td rowspan="1" colspan="1" align="center"><em>RP11–521D12.1</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">5</td><td rowspan="1" colspan="1" align="left"><em>Blautia</em> genus</td><td rowspan="1" colspan="1" align="right">rs79387448</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">7.68 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">−0.31</td><td rowspan="1" colspan="1" align="center">0.66</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">103,099,953</td><td rowspan="1" colspan="1" align="center">103,239,356</td><td rowspan="1" colspan="1" align="center"><em>SLC9A2</em></td><td rowspan="1" colspan="1" align="center"><em>SLC9A2</em></td></tr><tr><td rowspan="1" colspan="1" align="left">6</td><td rowspan="1" colspan="1" align="left">Bacilli class</td><td rowspan="1" colspan="1" align="right">rs10928827</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">1.02 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.22</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">129,426,740</td><td rowspan="1" colspan="1" align="center">129,473,850</td><td rowspan="1" colspan="1" align="center"><em>HS6ST1</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Lactobacillales order</td><td rowspan="1" colspan="1" align="right">rs10928827</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">4.19 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−0.23</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">7</td><td rowspan="1" colspan="1" align="left">Gammaproteobacteria class</td><td rowspan="1" colspan="1" align="right">rs4621152</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">1.4 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.29</td><td rowspan="1" colspan="1" align="center">0.79</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">217,857,450</td><td rowspan="1" colspan="1" align="center">217,924,261</td><td rowspan="1" colspan="1" align="center"><em><a id="__tag_642112369" class="tag_hotlink" href="/nuccore/AC007557.1" ref="/nuccore/AC007557.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC007557.1","term_id":"4827319","term_text":"AC007557.1"}}</span>AC007557.1</a></em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">8</td><td rowspan="1" colspan="1" align="left">Unclassifed Acidaminococcaceae</td><td rowspan="1" colspan="1" align="right">rs56006724</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">6.35 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">−0.88</td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">228,486,044</td><td rowspan="1" colspan="1" align="center">228,523,585</td><td rowspan="1" colspan="1" align="center"><em>C2orf83</em></td><td rowspan="1" colspan="1" align="center"><em>C2orf83</em></td></tr><tr><td rowspan="1" colspan="1" align="left">9</td><td rowspan="1" colspan="1" align="left">Marinilabiliaceae family</td><td rowspan="1" colspan="1" align="right">rs11915634</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">2.99 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">−1.30</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">1,452,602</td><td rowspan="1" colspan="1" align="center">1,517,331</td><td rowspan="1" colspan="1" align="center"><em>CNTN6</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Unclassifed Marinilabiliaceae</td><td rowspan="1" colspan="1" align="right">rs11915634</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">2.99 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">−1.30</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">10</td><td rowspan="1" colspan="1" align="left">OTU10032 unclassifed Enterobacteriaceae Species-level OTU</td><td rowspan="1" colspan="1" align="right">rs3925158</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">6.29 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−1.00</td><td rowspan="1" colspan="1" align="center">0.78</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">38,161,078</td><td rowspan="1" colspan="1" align="center">38,313,688</td><td rowspan="1" colspan="1" align="center"><em>SLC22A13</em></td><td rowspan="1" colspan="1" align="center"><em>SLC22A13, MYD88, DLEC1, ACAA1, OXSR1</em></td></tr><tr><td rowspan="1" colspan="1" align="left">11</td><td rowspan="1" colspan="1" align="left">EscherichiaShigella</td><td rowspan="1" colspan="1" align="right">rs13096731</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">2.55 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.43</td><td rowspan="1" colspan="1" align="center">0.12</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">58,014,818</td><td rowspan="1" colspan="1" align="center">58,089,851</td><td rowspan="1" colspan="1" align="center"><em>FLNB</em></td><td rowspan="1" colspan="1" align="center"><em>FLNB</em></td></tr><tr><td rowspan="1" colspan="1" align="left">12</td><td rowspan="1" colspan="1" align="left">Lactobacillales order</td><td rowspan="1" colspan="1" align="right">rs59042687</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">6.22 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−0.23</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">95,359,287</td><td rowspan="1" colspan="1" align="center">95,823,523</td><td rowspan="1" colspan="1" align="center"><em>LINC00879</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">13</td><td rowspan="1" colspan="1" align="left">Unclassifed Marinilabiliaceae</td><td rowspan="1" colspan="1" align="right">rs9831278</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">2.53 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−1.16</td><td rowspan="1" colspan="1" align="center">0.49</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">98,879,786</td><td rowspan="1" colspan="1" align="center">98,942,990</td><td rowspan="1" colspan="1" align="center"><em>LINC00973</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Marinilabiliaceae family</td><td rowspan="1" colspan="1" align="right">rs9831278</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">2.53 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−1.16</td><td rowspan="1" colspan="1" align="center">0.49</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">14<sup>a</sup></td><td rowspan="1" colspan="1" align="left">Lactobacillales order</td><td rowspan="1" colspan="1" align="right">rs62295801</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">5.32 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">−0.27</td><td rowspan="1" colspan="1" align="center">0.21</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">162,444,724</td><td rowspan="1" colspan="1" align="center">163,236,170</td><td rowspan="1" colspan="1" align="center"><em>LINC01192</em></td><td rowspan="1" colspan="1" align="center"><em>LINC01192</em></td></tr><tr><td rowspan="1" colspan="1" align="left">15</td><td rowspan="1" colspan="1" align="left">Bacilli class</td><td rowspan="1" colspan="1" align="right">rs7646786</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">2.29 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.22</td><td rowspan="1" colspan="1" align="center">0.5</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">185,729,634</td><td rowspan="1" colspan="1" align="center">185,742,372</td><td rowspan="1" colspan="1" align="center"><em>LOC344887</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">16</td><td rowspan="1" colspan="1" align="left">Unclassifed Porphyromonadaceae</td><td rowspan="1" colspan="1" align="right">rs7656342</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">2.8 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.39</td><td rowspan="1" colspan="1" align="center">0.22</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">9,721,358</td><td rowspan="1" colspan="1" align="center">9,895,176</td><td rowspan="1" colspan="1" align="center"><em>DRD5</em></td><td rowspan="1" colspan="1" align="center"><em>SLC2A9, DRD5</em></td></tr><tr><td rowspan="1" colspan="1" align="left">17<sup>b</sup></td><td rowspan="1" colspan="1" align="left">Marinilabiliaceae family</td><td rowspan="1" colspan="1" align="right">rs11724031</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">2.44 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">−0.97</td><td rowspan="1" colspan="1" align="center">0.68</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">77,441,448</td><td rowspan="1" colspan="1" align="center">77,467,405</td><td rowspan="1" colspan="1" align="center"><em>SHROOM3</em></td><td rowspan="1" colspan="1" align="center"><em>SHROOM3</em></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Unclassifed Marinilabiliaceae</td><td rowspan="1" colspan="1" align="right">rs11724031</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">2.44 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">−0.97</td><td rowspan="1" colspan="1" align="center">0.68</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Marinilabiliaceae family</td><td rowspan="1" colspan="1" align="right">rs9996716</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">5.58 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−0.69</td><td rowspan="1" colspan="1" align="center">0.2</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Unclassifed Marinila-biliaceae</td><td rowspan="1" colspan="1" align="right">rs9996716</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">5.58 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−0.69</td><td rowspan="1" colspan="1" align="center">0.2</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">18</td><td rowspan="1" colspan="1" align="left">Erysipelotrichaceae family</td><td rowspan="1" colspan="1" align="right">rs17421787</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">3.6 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.30</td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">131,293,675</td><td rowspan="1" colspan="1" align="center">131,512,291</td><td rowspan="1" colspan="1" align="center"><em>RP11-22J15.1</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Erysipelotrichales order</td><td rowspan="1" colspan="1" align="right">rs17421787</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">3.6 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.30</td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Erysipelotrichia class</td><td rowspan="1" colspan="1" align="right">rs17421787</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">3.6 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.30</td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">19</td><td rowspan="1" colspan="1" align="left">Unclassifed Porphyromonadaceae</td><td rowspan="1" colspan="1" align="right">rs9291879</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">3.51 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−0.58</td><td rowspan="1" colspan="1" align="center">0.08</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">66,515,817</td><td rowspan="1" colspan="1" align="center">66,550,855</td><td rowspan="1" colspan="1" align="center"><em>CD180</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">20</td><td rowspan="1" colspan="1" align="left">OTU10032 unclassifed Enterobacteriaceae</td><td rowspan="1" colspan="1" align="right">rs249733</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">4.74 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">−0.65</td><td rowspan="1" colspan="1" align="center">0.68</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">141,877,862</td><td rowspan="1" colspan="1" align="center">141,911,748</td><td rowspan="1" colspan="1" align="center"><em>SPRY4</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">21</td><td rowspan="1" colspan="1" align="left">Unclassifed Acidaminococcaceae</td><td rowspan="1" colspan="1" align="right">rs17661843</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">3.72 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">−1.40</td><td rowspan="1" colspan="1" align="center">0.26</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">48,381,902</td><td rowspan="1" colspan="1" align="center">48,433,594</td><td rowspan="1" colspan="1" align="center"><em>ABCA13</em></td><td rowspan="1" colspan="1" align="center"><em>ABCA13</em></td></tr><tr><td rowspan="1" colspan="1" align="left">22</td><td rowspan="1" colspan="1" align="left">OTU10032 unclassifed Enterobacteriaceae</td><td rowspan="1" colspan="1" align="right">rs13276516</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">5.54 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−0.61</td><td rowspan="1" colspan="1" align="center">0.41</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center">56,589,428</td><td rowspan="1" colspan="1" align="center">56,596,140</td><td rowspan="1" colspan="1" align="center"><em>TGS1</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">23</td><td rowspan="1" colspan="1" align="left">OTU10032 unclassifed Enterobacteriaceae Species-level OTU</td><td rowspan="1" colspan="1" align="right">rs2318350</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">3.65 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−1.15</td><td rowspan="1" colspan="1" align="center">0.95</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center">139,889,972</td><td rowspan="1" colspan="1" align="center">139,942,500</td><td rowspan="1" colspan="1" align="center"><em>COL22A1</em></td><td rowspan="1" colspan="1" align="center"><em>COL22A1</em></td></tr><tr><td rowspan="1" colspan="1" align="left">24</td><td rowspan="1" colspan="1" align="left">OTU10032 unclassifed Enterobacteriaceae</td><td rowspan="1" colspan="1" align="right">rs17085775</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">2.06 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−1.03</td><td rowspan="1" colspan="1" align="center">0.54</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">71,165,704</td><td rowspan="1" colspan="1" align="center">71,167,878</td><td rowspan="1" colspan="1" align="center"><em>C9orf71</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">25</td><td rowspan="1" colspan="1" align="left">Lactobacillales order</td><td rowspan="1" colspan="1" align="right">rs7083345</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">2.89 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.24</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">7,020,329</td><td rowspan="1" colspan="1" align="center">7,044,987</td><td rowspan="1" colspan="1" align="center"><em>RP11-554I8.2</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Bacilli class</td><td rowspan="1" colspan="1" align="right">rs7083345</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">3.38 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.25</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">7,020,329</td><td rowspan="1" colspan="1" align="center">7,044,987</td><td rowspan="1" colspan="1" align="center"><em>RP11-554I8.2</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">26</td><td rowspan="1" colspan="1" align="left">Lactobacillales order</td><td rowspan="1" colspan="1" align="right">rs7113056</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">1.72 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">−0.50</td><td rowspan="1" colspan="1" align="center">0.07</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">122,091,502</td><td rowspan="1" colspan="1" align="center">122,154,110</td><td rowspan="1" colspan="1" align="center"><em>RP11-166D19.1</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">27</td><td rowspan="1" colspan="1" align="left">Bacilli class</td><td rowspan="1" colspan="1" align="right">rs479105</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">1.21 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.22</td><td rowspan="1" colspan="1" align="center">0.48</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">3,357,596</td><td rowspan="1" colspan="1" align="center">3,393,503</td><td rowspan="1" colspan="1" align="center"><em>PRMT8</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">28</td><td rowspan="1" colspan="1" align="left">OTU10032 unclassifed Enterobacteriaceae Species-level OTU</td><td rowspan="1" colspan="1" align="right">rs1009634</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">7.12 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−1.31</td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">4,779,313</td><td rowspan="1" colspan="1" align="center">4,900,344</td><td rowspan="1" colspan="1" align="center"><em>AKAP3</em></td><td rowspan="1" colspan="1" align="center"><em>NDUFA9, GALNT8, RP11-234B24.2</em></td></tr><tr><td rowspan="1" colspan="1" align="left">29</td><td rowspan="1" colspan="1" align="left">Gammaproteobacteria class</td><td rowspan="1" colspan="1" align="right">rs9300430</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">1.3 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−0.61</td><td rowspan="1" colspan="1" align="center">0.12</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="center">98,269,478</td><td rowspan="1" colspan="1" align="center">98,306,405</td><td rowspan="1" colspan="1" align="center"><em>RAP2A</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">30</td><td rowspan="1" colspan="1" align="left">Proteobacteria phylum</td><td rowspan="1" colspan="1" align="right">rs9323326</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">8.76 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">−0.21</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">58,476,448</td><td rowspan="1" colspan="1" align="center">58,532,709</td><td rowspan="1" colspan="1" align="center"><em>SLC35F4</em></td><td rowspan="1" colspan="1" align="center"><em>C14orf37</em></td></tr><tr><td rowspan="1" colspan="1" align="left">31</td><td rowspan="1" colspan="1" align="left">Unclassifed Acidaminococcaceae</td><td rowspan="1" colspan="1" align="right">rs986417</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">2.63 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−1.40</td><td rowspan="1" colspan="1" align="center">0.47</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">60,787,269</td><td rowspan="1" colspan="1" align="center">61,122,040</td><td rowspan="1" colspan="1" align="center"><em>SIX6</em></td><td rowspan="1" colspan="1" align="center"><em>SIX6, C14orf39, SIX1</em></td></tr><tr><td rowspan="1" colspan="1" align="left">32</td><td rowspan="1" colspan="1" align="left">Unclassifed Erysipelotrichaceae</td><td rowspan="1" colspan="1" align="right">rs11626933</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">1.83 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.24</td><td rowspan="1" colspan="1" align="center">0.55</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">90,681,816</td><td rowspan="1" colspan="1" align="center">90,810,659</td><td rowspan="1" colspan="1" align="center"><em>C14orf102</em></td><td rowspan="1" colspan="1" align="center"><em>C14orf102</em></td></tr><tr><td rowspan="1" colspan="1" align="left">33</td><td rowspan="1" colspan="1" align="left">OTU15355 <em>Dialister</em> Species-level OTU</td><td rowspan="1" colspan="1" align="right">rs12442649</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">3.72 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−1.49</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">37,968,393</td><td rowspan="1" colspan="1" align="center">38,035,538</td><td rowspan="1" colspan="1" align="center"><em>TMCO5A</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">34</td><td rowspan="1" colspan="1" align="left">Enterobacteriaceae family</td><td rowspan="1" colspan="1" align="right">rs35275482</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">3.72 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">−0.54</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">60,027,987</td><td rowspan="1" colspan="1" align="center">60,128,040</td><td rowspan="1" colspan="1" align="center"><em>BNIP2</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Enterobacteriales order</td><td rowspan="1" colspan="1" align="right">rs35275482</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">3.72 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">−0.54</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">35</td><td rowspan="1" colspan="1" align="left">OTU10032 unclassifed Enterobacteriaceae</td><td rowspan="1" colspan="1" align="right">rs12149695</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">1.82 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.61</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">27,205,994</td><td rowspan="1" colspan="1" align="center">27,293,886</td><td rowspan="1" colspan="1" align="center"><em>FLJ21408</em></td><td rowspan="1" colspan="1" align="center"><em>NSMCE1, FLJ21408, KDM8</em></td></tr><tr><td rowspan="1" colspan="1" align="left">36</td><td rowspan="1" colspan="1" align="left">Lactobacillales order</td><td rowspan="1" colspan="1" align="right">rs1362404</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">1.56 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">7.5 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">51,955,443</td><td rowspan="1" colspan="1" align="center">52,017,380</td><td rowspan="1" colspan="1" align="center"><em>TOX3</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">37</td><td rowspan="1" colspan="1" align="left">Erysipelotrichaceae family</td><td rowspan="1" colspan="1" align="right">rs11877825</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">2.82 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">−0.27</td><td rowspan="1" colspan="1" align="center">0.34</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="center">10,566,345</td><td rowspan="1" colspan="1" align="center">10,595,758</td><td rowspan="1" colspan="1" align="center"><em>NAPG</em></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Erysipelotrichia class</td><td rowspan="1" colspan="1" align="right">rs11877825</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">2.82 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">−0.27</td><td rowspan="1" colspan="1" align="center">0.34</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Erysipelotrichales order</td><td rowspan="1" colspan="1" align="right">rs11877825</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">2.82 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">−0.27</td><td rowspan="1" colspan="1" align="center">0.34</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">38</td><td rowspan="1" colspan="1" align="left">Bacilli class</td><td rowspan="1" colspan="1" align="right">rs148330122</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">1.32 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">−0.48</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">38,497,288</td><td rowspan="1" colspan="1" align="center">38,631,252</td><td rowspan="1" colspan="1" align="center"><em>SIPA1L3</em></td><td rowspan="1" colspan="1" align="center"><em>SIPA1L3</em></td></tr><tr><td rowspan="1" colspan="1" align="left">39</td><td rowspan="1" colspan="1" align="left">Bacilli class</td><td rowspan="1" colspan="1" align="right">rs2071199</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">1.24 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">−0.32</td><td rowspan="1" colspan="1" align="center">0.58</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="center">43,030,809</td><td rowspan="1" colspan="1" align="center">43,037,422</td><td rowspan="1" colspan="1" align="center"><em>HNF4A–AS1</em></td><td rowspan="1" colspan="1" align="center"><em>HNF4A</em></td></tr><tr><td rowspan="1" colspan="1" align="left">40</td><td rowspan="1" colspan="1" align="left">Actinobacteria class</td><td rowspan="1" colspan="1" align="right">rs34613612</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">6.34 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.25</td><td rowspan="1" colspan="1" align="center">9.87 × 10<sup>−3</sup> 21</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">32,184,901</td><td rowspan="1" colspan="1" align="center">32,204,347</td><td rowspan="1" colspan="1" align="center"><em>KRTAP8-1</em></td><td rowspan="1" colspan="1" align="center"><em>KRTAP8-1</em></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">Actinobacteria phylum</td><td rowspan="1" colspan="1" align="right">rs34613612</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">6.34 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.25</td><td rowspan="1" colspan="1" align="center">9.87 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140468797789488" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5626933/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="P60" class="p p-first-last">The 54 associations with bacterial abundance are grouped into 40 loci on the basis of LD. “Locus” corresponds to locus number, “Bacteria” corresponds to the trait associated with a locus, “SNP” corresponds to the tag SNP for a locus–trait pair, “A1” is the allele for which association is analyzed, “A2” is the opposite allele, “Meta <em>P</em>” is the meta-analysis <em>P</em> value for A1, “Meta <em>β</em>” is the meta-analysis coeffcient for A1, “β-div <em>P</em>” is the <em>P</em> value for association with β diversity (Online Methods), “Chr.” corresponds to the chromosome, “Locus start” is the genetic position at which the locus starts and “Locus end” is the genetic position at which the locus ends, “Nearest gene” is the nearest gene to the SNP according to DEPICT; “Genes in locus” includes genes found in the locus according to DEPICT.</p></div><div id="TFN3"><sup>a</sup>Locus 14 contains the rs9290183 hit in addition to rs62295801, although PLINK does not clump these SNPs together. <sup>b</sup>Locus 17 includes rs9996716, which is located 219 bp downstream of the end of the locus according to DEPICT.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="P62">Gene set and tissue enrichment results for associations with individual bacterial traits</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140468802487824" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="3" rowspan="1" valign="bottom" align="center">Top 20 enriched gene sets
<hr></th><th colspan="4" rowspan="1" valign="bottom" align="center">Top 20 Enriched Tissues
<hr></th><th rowspan="1" colspan="1" align="left"></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Original gene<br>set ID</th><th rowspan="1" colspan="1" valign="bottom" align="left">Original gene set description</th><th rowspan="1" colspan="1" valign="bottom" align="left">Nominal <em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">Name</th><th rowspan="1" colspan="1" valign="bottom" align="center">MeSH<br>first-level<br>term</th><th rowspan="1" colspan="1" valign="bottom" align="center">MeSH<br>second-level<br>term</th><th rowspan="1" colspan="1" valign="bottom" align="left">Nominal <em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">MeSH term</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">GO:0007566</td><td rowspan="1" colspan="1" align="left">Embryo implantation</td><td rowspan="1" colspan="1" align="left">2.29 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">Keratinocytes</td><td rowspan="1" colspan="1" align="center">Cells</td><td rowspan="1" colspan="1" align="center">Epithelial cells</td><td rowspan="1" colspan="1" align="left">2.85 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">A11.436.397</td></tr><tr><td rowspan="1" colspan="1" align="left">MP:0009402</td><td rowspan="1" colspan="1" align="left">Decreased skeletal muscle fber diameter</td><td rowspan="1" colspan="1" align="left">4.09 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">Intestines</td><td rowspan="1" colspan="1" align="center">Digestive system</td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="left">5.57 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">A03.556.124</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0033273</td><td rowspan="1" colspan="1" align="left">Response to vitamin</td><td rowspan="1" colspan="1" align="left">4.77 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="center">Digestive system</td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="left">6.7 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">A03.556</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0033280</td><td rowspan="1" colspan="1" align="left">Response to vitamin D</td><td rowspan="1" colspan="1" align="left">8.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">Lower<br>gastrointestinal<br>tract</td><td rowspan="1" colspan="1" align="center">Digestive system</td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="left">6.92 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">A03.556.249</td></tr><tr><td rowspan="1" colspan="1" align="left">MP:0006317</td><td rowspan="1" colspan="1" align="left">Decreased urine sodium level</td><td rowspan="1" colspan="1" align="left">1.19 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Colon</td><td rowspan="1" colspan="1" align="center">Digestive system</td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="left">7.07 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">A03.556.249.249.356</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0071496</td><td rowspan="1" colspan="1" align="left">Cellular response to external stimulus</td><td rowspan="1" colspan="1" align="left">1.73 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Intestine, large</td><td rowspan="1" colspan="1" align="center">Digestive system</td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="left">7.27 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">A03.556.249.249</td></tr><tr><td rowspan="1" colspan="1" align="left">MP:0010027</td><td rowspan="1" colspan="1" align="left">Increased liver cholesterol level</td><td rowspan="1" colspan="1" align="left">1.73 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Hepatocytes</td><td rowspan="1" colspan="1" align="center">Cells</td><td rowspan="1" colspan="1" align="center">Epithelial cells</td><td rowspan="1" colspan="1" align="left">9.19 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">A11.436.348</td></tr><tr><td rowspan="1" colspan="1" align="left">MP:0000221</td><td rowspan="1" colspan="1" align="left">Digestive system</td><td rowspan="1" colspan="1" align="left">1.75 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Ileum</td><td rowspan="1" colspan="1" align="center">Digestive system</td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="left">9.96 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">A03.556.249.124</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0007229</td><td rowspan="1" colspan="1" align="left">Integrin-mediated signaling pathway</td><td rowspan="1" colspan="1" align="left">1.97 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Rectum</td><td rowspan="1" colspan="1" align="center">Digestive system</td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="left">0.01</td><td rowspan="1" colspan="1" align="center">A03.556.124.526.767</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0031668</td><td rowspan="1" colspan="1" align="left">Cellular response to extracellular stimulus</td><td rowspan="1" colspan="1" align="left">2.33 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Intestinal mucosa</td><td rowspan="1" colspan="1" align="center">Digestive system</td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="left">0.02</td><td rowspan="1" colspan="1" align="center">A03.556.124.369</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0033189</td><td rowspan="1" colspan="1" align="left">Response to vitamin A</td><td rowspan="1" colspan="1" align="left">2.5 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Mucous membrane</td><td rowspan="1" colspan="1" align="center">Tissues</td><td rowspan="1" colspan="1" align="center">Membranes</td><td rowspan="1" colspan="1" align="left">0.02</td><td rowspan="1" colspan="1" align="center">A10.615.550</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0031669</td><td rowspan="1" colspan="1" align="left">Cellular response to nutrient levels</td><td rowspan="1" colspan="1" align="left">3.03 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Colon, sigmoid</td><td rowspan="1" colspan="1" align="center">Digestive system</td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="left">0.02</td><td rowspan="1" colspan="1" align="center">A03.556.249.249.356.668</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0055093</td><td rowspan="1" colspan="1" align="left">Response to hyperoxia</td><td rowspan="1" colspan="1" align="left">3.09 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Epithelial cells</td><td rowspan="1" colspan="1" align="center">Cells</td><td rowspan="1" colspan="1" align="center">Epithelial cells</td><td rowspan="1" colspan="1" align="left">0.03</td><td rowspan="1" colspan="1" align="center">A11.436</td></tr><tr><td rowspan="1" colspan="1" align="left">ENSG00000215328</td><td rowspan="1" colspan="1" align="left">HSPA1A PPI subnetwork</td><td rowspan="1" colspan="1" align="left">3.27 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Hypothalamo hypophyseal system</td><td rowspan="1" colspan="1" align="center">Nervous system</td><td rowspan="1" colspan="1" align="center">Central nervous system</td><td rowspan="1" colspan="1" align="left">0.03</td><td rowspan="1" colspan="1" align="center">A08.186.211.730.317.357. 352.435</td></tr><tr><td rowspan="1" colspan="1" align="left">ENSG00000143393</td><td rowspan="1" colspan="1" align="left">PI4KB PPI subnetwork</td><td rowspan="1" colspan="1" align="left">3.37 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Neurosecretory systems</td><td rowspan="1" colspan="1" align="center">Nervous system</td><td rowspan="1" colspan="1" align="center">Neurosecretory systems</td><td rowspan="1" colspan="1" align="left">0.03</td><td rowspan="1" colspan="1" align="center">A08.713</td></tr><tr><td rowspan="1" colspan="1" align="left">MP:0005266</td><td rowspan="1" colspan="1" align="left">Abnormal metabolism</td><td rowspan="1" colspan="1" align="left">3.51 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Hypothalamus, middle</td><td rowspan="1" colspan="1" align="center">Nervous system</td><td rowspan="1" colspan="1" align="center">Central nervous system</td><td rowspan="1" colspan="1" align="left">0.03</td><td rowspan="1" colspan="1" align="center">A08.186.211.730.317.35 7.352</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0048545</td><td rowspan="1" colspan="1" align="left">Response to steroid hormone stimulus</td><td rowspan="1" colspan="1" align="left">3.54 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Membranes</td><td rowspan="1" colspan="1" align="center">Tissues</td><td rowspan="1" colspan="1" align="center">Membranes</td><td rowspan="1" colspan="1" align="left">0.04</td><td rowspan="1" colspan="1" align="center">A10.615</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0009991</td><td rowspan="1" colspan="1" align="left">Response to extracellular stimulus</td><td rowspan="1" colspan="1" align="left">3.7 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Monocyte macrophage precursor cells</td><td rowspan="1" colspan="1" align="center">Cells</td><td rowspan="1" colspan="1" align="center">Myeloid cells</td><td rowspan="1" colspan="1" align="left">0.04</td><td rowspan="1" colspan="1" align="center">A11.627.624.249</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0033143</td><td rowspan="1" colspan="1" align="left">Regulation of intracellular steroid hormone receptor signaling pathway</td><td rowspan="1" colspan="1" align="left">4.29 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Urinary bladder</td><td rowspan="1" colspan="1" align="center">Urogenital system</td><td rowspan="1" colspan="1" align="center">Urinary tract</td><td rowspan="1" colspan="1" align="left">0.04</td><td rowspan="1" colspan="1" align="center">A05.810.890</td></tr><tr><td rowspan="1" colspan="1" align="left">GO:0031490</td><td rowspan="1" colspan="1" align="left">Chromatin DNA binding</td><td rowspan="1" colspan="1" align="left">4.59 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">Intestine, small</td><td rowspan="1" colspan="1" align="center">Digestive system</td><td rowspan="1" colspan="1" align="center">Gastrointestinal tract</td><td rowspan="1" colspan="1" align="left">0.04</td><td rowspan="1" colspan="1" align="center">A03.556.124.684</td></tr></tbody></table></div><div id="largeobj_idm140468802487824" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5626933/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><p id="P63" class="p p-first-last">Enrichment analysis was performed using DEPICT for the 40 loci associated with individual bacterial traits. The table shows the 20 most enriched gene sets (three left columns) and the 20 most enriched tissues or cell types (fve right columns). For tissue and cell type enrichment, headings are given for minimum two MeSH levels: frst- and second-level terms. The “Name” column contains the name for the lowest level term with enrichment. The MeSH codes for the hierarchical branch are given in the “MeSH term” column. Analysis for enriched genes and analysis for enriched tissues were independent of each other.</p></div></div></div><p id="P12">The gut microbiome is essential for bile acid metabolism, and bile acids act as both key VDR ligands and regulators of <em>VDR</em> expression<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112174">23</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112192">24</a>,<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112177">26</a></sup>. In addition, polyunsaturated fatty acids act as ligands for RXR, the heterodimeric partner of VDR, and were shown to compete for ligand binding to VDR<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112180">23</a></sup>. We therefore performed targeted measurement of bile acids and ω3 and ω6 polyunsaturated fatty acids in human serum in a subset of the PopGen cohort (<em>n</em> = 551). We found significant correlations between several bile acids and β diversity (BC), including taurochenodeoxycholic acid (TCDCA; 2.2% variation explained) and glycochenodeoxycholic acid (GCDCA; 1.4% variation explained; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 8 and Supplementary Table 3</a>). Bile acids also significantly associated with individual bacterial taxa, including the secondary bile acids lithocholic acid (LCA; a known VDR ligand) and deoxycholic acid (DCA; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>), both of which are produced by the gut microbiota<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112176">24</a></sup>. In addition, genomic analysis showed that <em>Parabacteroides</em> bacteria contain pathways involved in secondary bile acid metabolism (Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway pdi00121) and could thus indeed be associated with bile acid profiles, a hypothesis that is further supported by positive correlations between <em>Parabacteroides</em> abundance and LCA concentration (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). Furthermore, functional profiling of the gut microbiome via shotgun metagenomic analysis in a subset of the PopGen cohort (<em>n</em> = 122) also showed differences in bile acid–related gene pathways with respect to <em>VDR</em> genotype (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 9</a>). Finally, the above-mentioned data from colonic biopsies also suggested that the interplay between <em>VDR</em> and <em>Parabacteroides</em> involves two genes associated with bile acid metabolism (<em>CYP27A1</em>, encoding cytochrome P450 family 27 subfamily A member 1, and <em>NR5A2</em>, encoding nuclear receptor subfamily 5 group A member 2; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 7</a>), with interactions lost in the context of intestinal inflammation (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 7</a>). Together, these findings provide evidence that the gut microbiota significantly contributes to human bile acid profiles, as previously reported in mice<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112182">27</a></sup>. For fatty acids (false discovery rate (FDR) &lt; 0.05), we detected significant correlations between the gut microbiota and 7 of 15 polyunsaturated fatty acids, including arachidonic acid (an ω6 fatty acid that is capable of binding VDR), which correlated with β diversity (1.22% variation explained) and several specific taxa (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). Of note, two additional genome-wide significant associations with the gut microbiota are critically involved in bile acid (<em>HNF4A</em>; <a href="/pmc/articles/PMC5626933/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) and arachidonic acid (<em>PLA2G3</em>; <a href="/pmc/articles/PMC5626933/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) homeostasis<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112163">28</a>,<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112179">29</a></sup>. Finally, several loci identified in this study, in addition to <em>VDR</em>, were significantly correlated with bile acid profiles, as shown by regression analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>).</p><p id="P13">Many other interesting findings are found among the 42 significant loci (<a href="/pmc/articles/PMC5626933/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), in particular the <em>POMC</em> (proopiomelanocortin) gene (rs72853661, <em>P</em> &lt; 5 × 10<sup>−8</sup>; <a href="/pmc/articles/PMC5626933/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3e</span></a>). As an extremely functionally diverse protein, POMC participates in multiple physiological processes ranging from antimicrobial activity to appetite regulation (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>). Furthermore, this locus located upstream of the <em>POMC</em> gene is the largest we discovered (78 SNPs over 54.8 kb; <a href="/pmc/articles/PMC5626933/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3e</span></a>) and contains multiple SNPs that regulate the expression of <em>POMC</em> in multiple human tissues, as determined by expression quantitative trait locus (eQTL) studies (GTEx database). The associated SNP rs66589178 in particular is predicted to be a VDR binding site (RegulomeDB), and the TRAP analysis tool predicted an almost 200-fold difference in affinity for VDR between the two alleles (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 10</a>). Other findings include the <em>HTR1E</em> (serotonin receptor) and <em>GRID1</em> (glutamate ionotropic receptor) genes, which are potential components of the gut–brain axis<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112217">30</a></sup>, and genetic variation near <em>CLEC16A</em> (rs12931878, <em>P</em> &lt; 5 × 10<sup>−8</sup>), a gene associated with multiple autoimmune and inflammatory disorders involving alterations to gut microbiota (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>). A number of other regions are implicated in disease susceptibility as previously reported by case-control GWAS and can be found in <a href="/pmc/articles/PMC5626933/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6 and 7</a> (for example, <em>BANK1</em> close to <em>SCL39A8</em>).</p><p id="P14" class="p p-last">Finally, a targeted analysis was performed for the human leukocyte antigen (HLA) complex on chromosome 6. The HLA complex shapes the immune repertoire and may influence gut microbiome composition<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112151">31</a></sup>. Because SNPs do not capture the extreme polymorphism of the classical HLA genes, we imputed HLA alleles using SNP2HLA (Online Methods) and implemented a constrained ordination approach. This approach showed significant association of alleles at <em>HLA-B</em> (HLA-B*52:01) and <em>HLA-C</em> (HLA-C*12:02) in both cohorts (<em>P</em> &lt; 0.05; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 11 and Supplementary Table 8</a>). The associated alleles have been implicated in risk for ulcerative colitis in multiple ancestry groups<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112150">32</a>,<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112160">33</a></sup> and in Takayasu arteritis<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112149">34</a></sup>.</p></div><div id="S4" class="sec"><h3 id="S4title">Genetic associations with individual bacterial traits</h3><p id="P15" class="p p-first">To detect associations between genetic variants and specific bacterial traits, we first curated the microbiome data and removed rare bacteria by defining a ‘core measurable microbiota’ (ref. 35) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 12 and Supplementary Table 9</a>), which included 40 operational taxonomic units (OTUs) and 58 taxa ranging from the genus to the phylum level, and employed a generalized linear model (GLM) framework incorporating a negative binomial (negbin) distribution. Accordingly, we identified 54 significant associations involving 40 loci and 22 bacterial traits (meta-analysis <em>P</em> &lt; 5 × 10<sup>−8</sup> and single-cohort <em>P</em> &lt; 5 × 10<sup>−4</sup>; <a href="/pmc/articles/PMC5626933/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Of the 22 bacterial traits, the largest number belonged to Firmicutes (<em>n</em> = 10), followed by Proteobacteria (<em>n</em> = 7), Bacteroidetes (<em>n</em> = 3) and Actinobacteria (<em>n</em> = 2), at the phylum level. To identify the nearest and neighboring genes for each locus, we annotated the identified SNPs using DEPICT<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112187">36</a></sup> (<a href="/pmc/articles/PMC5626933/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><p id="P16">Among the 54 robust associations, the <em>SLC2A9</em> gene was associated with unclassified Porphyromonadaceae (rs7656342, meta-analysis <em>P</em> = 2.8 × 10<sup>−9</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 13</a>). The <em>SLC2A9</em> gene encodes a member of the glucose transporter family, which is important for maintaining glucose homeostasis<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112210">37</a></sup>. Furthermore, a number of long intergenic noncoding RNAs were among the 54 associations, including association of <em>LINC01192</em> with <em>Lactobacillales</em> (rs62295801, meta-analysis <em>P</em> = 5.32 × 10<sup>−10</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 13</a>). Of note, gene set enrichment analysis detected associations for <em>LINC01192</em> with ‘response to vitamin A’ and for <em>SLC2A9</em> with both ‘response to vitamin D’ and ‘increased liver cholesterol level’ (<a href="/pmc/articles/PMC5626933/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><p id="P17" class="p p-last">Next, we evaluated whether the genetic signal for β diversity is influenced by the abundance of individual bacterial taxa. Indeed, 37 loci that correlated with β diversity also correlated with the abundance of several core measurable microbiota taxa and OTUs (<em>P</em> &lt; 0.01), albeit not at the genome-wide significance level (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 14</a>). Conversely, the loci identified in association analyses for individual taxa explained a proportion of the variation in β diversity (six loci with <em>P</em> &lt; 0.05, effect size of 0.29–0.49%) but did not reach our conservative significance threshold of <em>P</em> &lt; 5 × 10<sup>−8</sup> (<a href="/pmc/articles/PMC5626933/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Thus, in conclusion, we found that genetic variants correlating with microbiome structure could be either strongly associated with an individual taxon or simultaneously associated with multiple taxa, with each association having a small effect size.</p></div><div id="S5" class="sec sec-last"><h3 id="S5title">Enrichment analysis of gene sets and tissues</h3><p id="P18" class="p p-first">To further assess the functional relevance of the 54 identified associations between genetic variants and specific bacterial traits, we used DEPICT<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112158">36</a></sup> to perform both gene set and tissue enrichment analyses (<a href="/pmc/articles/PMC5626933/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). DEPICT prioritizes genes in associated regions on the basis of functional relationships and linkage disequilibrium (LD) structure. Of interest, ‘response to vitamin D’ (original gene set ID GO:0033280, <em>P</em> = 8.8 × 10<sup>−5</sup>) was the fourth most enriched term. Enrichment of response to vitamins in general was also observed, including ‘response to vitamin A’, another fat-soluble vitamin binding to the retinoic acid receptor (RAR) and involved in bile acid homeostasis<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_781915479">38</a>,<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112211">39</a></sup>. The gene set for ‘response to vitamin D’ includes <em>SLC22A13, SLC2A9, COL22A1, ABCA13</em> and <em>KRTAP8-1</em> (<a href="/pmc/articles/PMC5626933/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The <em>VDR</em> gene locus itself, however, is not included, as the enrichment analysis was limited to loci associated with single bacterial taxa, and the association with <em>Parabacteroides</em> (<a href="/pmc/articles/PMC5626933/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3c</span></a>) did not reach the genome-wide significance threshold. Further, the term ‘increased liver cholesterol level’ was among the top enriched gene sets (original gene set ID MP:0010027, <em>P</em> = 1.7 × 10<sup>−4</sup>) and corresponds to one of the functions of the <em>POMC</em> gene locus identified in the above analysis. Among the bacterial taxa associated with ‘increased liver cholesterol level’ were Gammaproteobacteria, Bacilli, unclassified Porphyromonadaceae and an OTU belonging to Enterobacteriaceae. Furthermore, in a <em>trans</em>-eQTL analysis of the SNPs associated with β diversity or single bacterial taxa (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6 and 7</a>), <em>FDFT1</em>, which encodes the first specific enzyme in cholesterol synthesis, was among the top hits, further emphasizing the fact that several hits converge onto the sterol pathway.</p><p id="P19" class="p p-last">In the tissue enrichment analysis, the top 20 results with <em>P</em> &lt; 0.05 (<a href="/pmc/articles/PMC5626933/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>) showed the Medical Subject Heading (MeSH) terms ‘digestive system’ (10 occurrences), ‘nervous system’ (3 occurrences) and ‘cells’ (3 occurrences) as most significant. The best associated subcategories for ‘digestive system’ were ‘intestinal mucosa’ and ‘mucous membrane’, whereas the subcategories for ‘cells’ included ‘mono-cyte macrophage precursor cells’, ‘epithelial cells’ and ‘hepatocytes’. In sum, the tissue enrichment analysis relates microbial-associated host loci with gastrointestinal and immune-related tissues and cells, thus supporting the functional relevance of the identified loci.</p></div></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">DISCUSSION</h2><p id="P20" class="p p-first">We herein present a comprehensive analysis of genome-wide host–microbiota associations. We adhered to rigorous standards by including a large number of samples (1,812 SNP array–16S rRNA microbiome data set pairs) and considering important known and herein identified confounders of variation in the gut microbiome. As geography is a major factor contributing to microbiome composition<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112231">11</a>,<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112162">15</a></sup>, we used cohorts recruited from the same country and corrected for population stratification/ancestry in our genetic data set. We discovered genome-wide significant associations between gut microbial characteristics and the <em>VDR</em> gene, in addition to a large number of other host genetic factors, and eventually quantified the total contribution of host genetic loci to β diversity as 10.43%. The non-genetic factors examined (age, sex, BMI, smoking status and dietary patterns) explain 8.87% of the observed variation in the gut microbiome.</p><p id="P21">As shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 15</a>, the associations at the <em>VDR</em> locus with gut microbial community composition provide compelling follow-up to the finding by Makishima <em>et al</em>.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112153">24</a></sup> that secondary bile acids (bile acids transformed by gut microbial metabolism, that is, LCA, glycine-conjugated LCA and 3-keto-LCA from 7α-dehydroxylated primary CDCA) serve as ligands for VDR. Validation of a relationship between VDR alterations and the gut microbiota in the <em>Vdr</em><sup>−/−</sup> mouse model<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112164">25</a></sup> substantiates these observations. Results from gene set enrichment analysis and the observation that the bile acid profile in serum associates with variation in the gut microbiome further support this finding. The underlying mechanisms for the observed association between gut microbial profiles and the serum bile acid pool warrant further elaboration. The possibility that VDR-mediated signaling serves as a key mediator in the gut–liver signaling axis and microbial co-metabolism, as previously shown for <em>FXR</em> (farnesoid X receptor<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112193">27</a></sup>), motivates substantial new research directions. Although the lack of an association at the <em>FXR</em> locus (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 16</a>) does not signify the lack of FXR involvement in microbial bile acid co-regulation<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112226">23</a></sup> (for example, functional variation may simply not be present in our cohort), the <em>VDR</em> associations detected in the present study add another important player to this relationship.</p><p id="P22">Insight on interactions between the microbiome and bile acid homeostasis are mostly based on mouse studies<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112204">27</a>,<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112220">40</a>,<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642313271">41</a></sup>, for which the transfer of interpretations into the human setting may be considerably biased given the large differences in bile acid profiles between mice and humans. Additional data presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6 and 7</a> show cross-validation for a subset of the genes detected in the human analysis, including <em>VDR</em>, whereby differential expression in germ-free and conventionally raised mice further supports the roles of these genes in interacting with and/or maintaining the homeostasis of the gut microbiome. Such overlap between distantly related mammalian hosts provides strong support for our discoveries and, hence, the internal validity of our experiment. Genetic associations at the <em>VDR</em> locus were also detected in human inflammatory bowel disease and liver disease<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112195">42</a>,<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112183">43</a></sup>, for which the underlying mechanisms were proposed to be a perturbation of key aspects of host– microbe interactions<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112201">44</a></sup>. The multidimensional relationship of key factors involved in VDR signaling (bile acids and ω6 fatty acids in particular) and the gut microbiota is even supported by genetic associations at functionally related loci (<em>HNF4A</em> and <em>PLA2G3</em>).</p><p id="P23">The <em>POMC</em> locus gives rise to a number of proopiomelanocortin-derived peptides involved in various physiological processes, including blood sugar regulation, inflammation and energy intake<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112224">45</a></sup>, and association of SNP rs66589178, potentially affecting a VDR binding site (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 10</a>), is an additional interesting circumstantial observation for the <em>VDR</em> finding. On the basis of their broad influence on bacterial community structure (contribution to β diversity as measured by BC) in our cohorts, <em>VDR</em> and <em>POMC</em> (among other genes) could be major regulators of the gut microbiome. Given that <em>VDR</em> and <em>POMC</em> are further associated with numerous important phenotypes (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6 and 7</a>), our results provide a strong indication for genetic associations across phenotypes, including BMI, Crohn’s disease and the intestinal microbiome. However, further dedicated studies are still needed to link these pleiotropic signaling pathways and their associated biology<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112229">46</a></sup>. Finally, understanding the functional consequences of the genetic variants discovered in this study will also require in-depth exploration, as the functional consequences of the lead SNPs remain unknown (for example, <em>VDR</em> lead SNP rs7974353).</p><p id="P24">Genome-wide screening for host genetic associations with gut microbiome composition has mostly been performed in mice, for which environmental factors and genetic background are easy to control. Thus, to further validate our findings, we compared our results to previously published QTL studies for the mouse gut microbiome. We found that mouse homologs of numerous GWAS hits in our study are contained in the confidence intervals of mouse QTLs (<a href="/pmc/articles/PMC5626933/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>). One such overlap even involves association with an identical trait— between the <em>SLC9A2</em> gene and genus <em>Blautia</em>—in addition to traits at higher taxonomic levels (class or phylum). In addition, among all GWAS performed for human traits as determined by the National Human Genome Research Institute (NHGRI) GWAS Catalog, most loci and genes discovered in our study were previously associated with various traits, including diseases for which there is growing evidence of microbiome involvement in disease etiology (for example, inflammatory bowel disease, obesity and type 2 diabetes; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6 and 7</a>). Furthermore, specific associations of genes observed in previous studies (for instance, <em>FUT2, NOD2</em> and <em>LCT</em>) could be replicated in our data set, but with less contribution in terms of influencing overall microbial variation (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs. 16 and 17</a>).</p><p id="P25" class="p p-last">In summary, we identify several genetic and non-genetic factors that determine the composition of the human gut microbiome. We show that genetic variation at the <em>VDR</em> locus significantly influences micro-bial co-metabolism and the gut–liver axis. Multiple other findings highlight key aspects of the intersections of host physiology with the gut microbiota, including a number of disease susceptibility genes in complex human diseases and the gut–brain axis. Key non-genetic covariable parameters, including diet, cumulatively have a similar magnitude of influence on the microbiome as host genetics, highlighting the importance of controlling for these confounders. Our study also indicates that the effect of individual genes is small and emphasizes the need for adequate statistical power and large sample sizes in future assessments. Following a similar logic to that provided by the outcomes of GWAS, the underlying biology of our observations may far exceed the statistical estimates and is likely to provide a critical framework for future studies of host–microbe interactions in humans.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">METHODS</h2><p id="P26" class="p p-first-last">Methods and any associated references are available in the <a href="http://dx.doi.org/10.1038/ng.3695" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">online version of the paper</a>.</p></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">ONLINE METHODS</h2><div id="S9" class="sec sec-first"><h3 id="S9title">Study subjects and sample collection</h3><p id="P27" class="p p-first-last">Two population-based cohorts from Schleswig-Holstein (Germany) were included in the study. Nine hundred and fourteen individuals from the PopGen cohort and 1,115 individuals from the FoCus (Food Chain Plus) cohort were included. These two study cohorts were recruited independently from each other, and the maximum number of individuals available was included to increase statistical power for various analyses. All samples, as well as corresponding information on phenotype and dietary behavior, were obtained from the PopGen biobank (Schleswig-Holstein, Germany)<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112157">20</a></sup>. Study participants collected fecal samples at home in standard fecal tubes. Samples were shipped immediately at room temperature or brought to the collection center by the participants. Upon arrival into the study center (within 24 h), samples were stored at −80 °C until processing. Written, informed consent was obtained from all study participants, and all protocols were approved by the institutional ethical review committee in adherence with the Declaration of Helsinki Principles; investigators were blinded to sample identities. Sequence data for the 16S rRNA gene, genotype, nutritional and phenotype data used for the herein described study have been made available to other scientists through PopGen’s biobank general data transfer agreement. A summary of the phenotypes used in this paper is given in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>.</p></div><div id="S10" class="sec"><h3 id="S10title">Genotyping data</h3><p id="P28" class="p p-first-last">Samples of the PopGen and FoCus cohorts were geno-typed on different genotyping arrays. The PopGen samples were typed on the Affymetrix 6.0, Affymetrix Axiom, Illumina 550k, custom Illumina Immunochip and Illumina Metabochip arrays with sample sizes before quality control ranging from 678 to 1,218 and a variant coverage of 196,524 to 934,968 variants. The FoCus samples were typed on the custom Illumina Immunochip and the Omni Express Exome, with 1,024 and 1,713 samples overall before quality control and a variant coverage of 195,732 to 964,193 variants. For each cohort, genotype data for each array were quality controlled separately and then merged and imputed. In total, 17,017,474 single-nucleotide variants (SNVs) were included for the PopGen cohort and 17,340,550 SNVs were included for the FoCus cohort. Consequently, stringent quality filtering was performed for all genotyping data, with details provided in the accompanying <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>.</p></div><div id="S11" class="sec"><h3 id="S11title">Sequencing and processing of bacterial 16S rRNA sequences</h3><p id="P29" class="p p-first-last">Bacterial genomic DNA was extracted using the QIAamp DNA Stool Mini kit from Qiagen on a QIAcube system. For all samples, the V1–V2 region of the 16S rRNA gene was sequenced on the MiSeq platform, using the 27F-338R primer pair and dual MID indexing (8 nt each on the forward and reverse primers) as described by Kozich <em>et al</em>.<sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112173">51</a></sup>. Sequencing was performed with MiSeq Reagent Kit v2. After sequencing, MiSeq fastq files were derived from base calls for read 1 and 2 (R1 and R2), as well as both indices (I1 and I2), using the Bcl2fastq module in CASAVA 1.8.2. Stringent demultiplexing was carried out by allowing no mismatches in either index sequence (instead of the default of one mismatch allowed by MiSeq). Forward and reverse reads were merged with FLASH software (v1.2)<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112218">52</a></sup>, and quality filtering was subsequently performed with the fastx toolkit, excluding sequences with &gt;5% nucleotides with quality score &lt;30. Chimeras in sequences were removed using UCHIME (v6.0)<sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112156">53</a></sup>. After randomly selecting 10,000 reads for each sample, taxonomical classification and compositional matrices for each taxonomical level were carried out using the RDP classifier<sup><a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112155">54</a></sup> with the latest reference database (RDP14), where classifications with low confidence at the genus level (&lt;0.8) were organized in an arbitrary taxon of ‘unclassified family’. Species-level OTUs (97% similarity) were created using the UPARSE routine<sup><a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112188">55</a></sup>.</p></div><div id="S12" class="sec"><h3 id="S12title">Bile acid and fatty acid measurements on human serum samples</h3><p id="P30" class="p p-first-last">Serum bile acid and polyunsaturated fatty acid composition in plasma was analyzed for 551 PopGen samples by HPLC-MS/MS as recently described<sup><a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112166">56</a>,<a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112191">57</a></sup>. Five bile acids (cholic acid (CA), chenodeoxycholic acid (CDCA), lithocholic acid (LCA), deoxycholic acid (DCA) and ursodeoxycholic acid (UDCA)), including their taurinated (T) and glycinated (G) conjugates, were measured, as well as the following fatty acids: C18:2n-6 (linoleic acid), C18:3n-3 (α-linolenic acid), C18:3n-6 (γ-linolenic acid), C18:4n-3 (stearidonic acid), C20:2n-6 (eicosadienoic acid), C20:3n-6 (dihomo-γ-linolenic acid), C20:4n-3 (eicosatetraenoic acid), C20:4n-6 (arachidonic acid), C20:5n-3 (eicosapentaenoic acid), C21:5n-3 (heneicosapentaenoic acid), C22:2n-6 (docosadienoic acid), C22:4n-6 (adrenic acid), C22:5n-3 (docosapentaenoic acid), C22:5n-6 (docosapentaenoic acid), C22:6n-3 (docosahexaenoic acid).</p></div><div id="S13" class="sec"><h3 id="S13title">Statistical analysis</h3><div id="S14" class="sec sec-first"><p></p><h4 id="S14title" class="inline">Correlation between microbiome and metadata </h4><p id="P31" class="p p-first">In both cohorts, β-diversity measures based on genus-level composition were generated using the ‘vegdist’ function (Bray–Curtis and Jaccard dissimilarities). Community ordination was performed using PCoA based on the calculated dissimilarities using the ‘capscale’ function in ‘vegan’ (v2.3). The ‘envfit’ function in ‘vegan’ was used to correlate either categorical data, for which it performs multidimensional ANOVA on the ordination, or continuous variables, for which the function tests linear correlations between a given variable and the coordinates of microbial communities. This test does not assume a normal distribution, as the significance value is determined by a permutation test.</p><p id="P32" class="p p-last">We considered a range of reported confounding variables that could shape the human gut microbiome: age, sex, BMI, smoking and major nutritional components or food groups derived from diet patterns; similarly, the association analysis was performed for bile acid profiles and fatty acid composition. Dietary patterns were collected via a validated, self-administered, 112-item food frequency questionnaire established for German populations<sup><a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112209">58</a>,<a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112199">59</a></sup>. All participants were given the option of completing the questionnaire preferably as a web-based version and, optionally, on paper. Information on macro-and micronutrient intake was obtained by using the German Food Code and Nutrient Database (vII.3) and provided by the Department of Epidemiology of the German Institute of Human Nutrition Potsdam-Rehbruecke. Before association analysis, all individuals who took antibiotics less than 6 weeks before stool collection were excluded to remove the possible influences of antibiotic medication. The effect size and significance of the mentioned variables were estimated using ‘envfit’, and the variables with significant effects (<em>P</em> &lt; 0.05) were further used in the GWAS analysis as covariates (water, alcohol and all other highly correlated nutritional variables, which were collectively joined under the umbrella ‘total energy’). The combined effect of host metadata was estimated further using the ‘bioenv’ function in the ‘vegan’ package, which calculates the maximum Pearson correlation of microbial variation (Bray–Curtis dissimilarity) and combined dissimilarity in the selected subset of metadata (denoted by Gower distances). To reduce random errors in low-abundance taxa, the analysis focused on the ‘core measurable microbiota’, which was determined using technical replicates according to Benson <em>et al</em>.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112207">35</a></sup>. Only taxa with an average of &gt;40 reads per sample (and thus with less error introduced by random processes) were included (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 12</a>).</p></div><div id="S15" class="sec"><p></p><h4 id="S15title" class="inline">Association of individual bacterial traits with human genetic variation </h4><p id="P33" class="p p-first">To identify human genetic variation associated with the abundance of individual gut bacteria, a statistical test for each combination of SNP and taxon was performed. The abundance of bacteria in the human gut is characterized by an increasing number of zeros at lower taxonomic levels, a right-skewed distribution often with a long tail and only positive values. Thus, a model assuming a normal distribution of dependent variables could not be fitted to our data. The GLM with a negative binomial (negbin) distribution and log link was selected for the statistical analysis as the best-fitting model across all bacteria. The hurdle model with a negbin distribution showed increasingly good fit with increasing numbers of zeros. The GLM negbin model was therefore selected as a consistent model across all bacteria, while the analysis of species (97% similarity threshold OTUs) was supported with the hurdle model<sup><a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112223">60</a></sup>.</p><p id="P34">Our identified ‘core measurable microbiota’ (ref. 35) consists of 64 taxa across five levels (phylum, class, order, family and genus) and 42 species-level OTUs. Taxa with &gt;90% of their counts within the first 5% of the range of counts or with &gt;90% of above-zero counts within the first 5% of the above-zero range were excluded, as they performed poorly with the selected model(s). Forty OTUs and 58 taxa were used for association study with human SNPs. The analyses were preformed on both cohorts separately (986 samples in FoCus and 826 samples in PopGen). In the analyses, outliers defined as 5 s.d. were removed and genetic variants not overlapping in FoCus and PopGen were discarded, while variants with MAF &gt;0.05 and IMPUTE2 INFO criteria &gt;0.8 were included. No population stratification was observed between the two cohorts (λ<sub>GC</sub> = 1.00; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 18</a>) b<sup><a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642313270">61</a></sup>. The covariates BMI, age, sex, genetic principal components 1–3 and nutritional variables alcohol, water and ‘total energy’ intake were used. The analyses were performed using R Project version 3.2 and the GLM.nb function in ‘MASS’ package version 7.3 for the GLM negbin and the hurdle function in package ‘pscl’ (v1.4).</p><p id="P35" class="p p-last">A meta-analysis of GLM negbin hits across the two cohorts was performed using PLINK (v1.9 64-bit)<sup><a href="#R62" rid="R62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112212">62</a></sup>, with the command “--meta-analysis +qt”, including information on <em>β</em> coefficients and standard errors. Clumping was performed using PLINK v1.9 with the “--clump” command on SNPs meeting the following filtering criteria: meta-study fixed-effect <em>P</em> value &lt; 5 × 10<sup>−8</sup>, single-cohort <em>P</em> value &lt; 5 × 10<sup>−4</sup>, the same <em>β</em> value sign (same direction of association) and AIC (model fit parameter) &lt; 50,000. Clumps with at least two SNPs for which at least one SNP was genotyped were selected. For each selected clump, the SNP with the lowest meta-analysis <em>P</em> value was selected as the tag SNP, and for bacteria containing zero counts the hurdle model was applied as described above. All hits were confirmed to be supported by the count or zero part of the hurdle model with <em>P</em> &lt; 0.05 in both studies.</p></div><div id="S16" class="sec sec-last"><p></p><h4 id="S16title" class="inline">Genetic variation correlated with overall community differences </h4><p id="P36" class="p p-first">We also performed analyses aimed at identifying genetic variation that might not necessarily associate with individual bacterial taxa with genome-wide significance but might rather correlate with overall community differences (β diversity). We performed a simulation and treated genotype at each locus as categorical variables (the distribution of each genotype follows Hardy–Weinberg equilibrium). We measured the genotype association using the ‘envfit’ function in the ‘vegan’ R package (v2.3). This approach calculates the community differences associated with three different genotypes, by comparing the difference in the centroids of each group relative to the total variation, on the basis of the main axes of the PCoA. By shuffling the simulated genotype &gt;2 × 10<sup>7</sup> times, we effectively obtained a large enough null distribution of effect size. This was performed for six categories of MAF to represent loci with MAFs of 5%, 10%, 20%, 30%, 40% and 50% (whereas in case of a real SNP, it is compared to the category with the closest MAF value; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 19</a>), and if a certain locus displays greater effect sizes than the simulated maximum they are extremely unlikely to be observed by chance (<em>P</em> &lt; 5 × 10<sup>−8</sup>) and can be considered to be genome-wide significant. We have filtered SNPs in a similar fashion as the taxa associations mentioned above.</p><p id="P37">The additive effect of the significant loci from this analysis was then determined using redundancy analysis based on genus-level composition (‘rda’ in the ‘vegan’ package) and the ‘ordiR2step’ function in the ‘vegan’ package, which optimizes the order of loci in a linear model and sums up the variation of the ordination explained by each additional locus.</p><p id="P38" class="p p-last">HLA analyses were conducted on the respective HLA haplotypes within each locus, coded as carrier or non-carrier for each specific allele. We performed distance-based redundancy analysis after correction for host characteristics (see description of association analysis for factors). These models were then tested using a permutative ANOVA approach (5,000 permutations) as implemented in the ‘vegan’ function ‘anova.cca’, and the coefficients of determination were extracted via ‘RsquareAdj’.</p></div></div><div id="S17" class="sec"><h3 id="S17title">Annotation and enrichment</h3><p id="P39" class="p p-first-last">DEPICT<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112171">36</a></sup> was used to annotate and perform tissue and gene set enrichment analyses among the significant single-bacteria associations. DEPICT was used with the following settings: (i) association_pvalue_cutoff: 1 × 10<sup>−5</sup>, (ii) nr_repititions: 20, and (iii) nr_permutations: 500; all available analysis steps were performed. For genotype data, we used 1000_genomes_project_phase3_CEU/ALL.chr_merged.phase3_sha-peit2_mvncall_integrated_v5.2 0130502.genotypes; for the collection file, we specified ld0.5_collection_depict_150315.txt.gz and for the reconstituted gene sets file we specified <a id="__tag_642112233" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GPL570" ref="/geo/query/acc.cgi?acc=GPL570"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GPL570","term_id":"570"}}</span>GPL570</a>-<a id="__tag_642112234" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GPL96" ref="/geo/query/acc.cgi?acc=GPL96"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GPL96","term_id":"96"}}</span>GPL96</a>-<a id="__tag_642112235" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GPL1261" ref="/geo/query/acc.cgi?acc=GPL1261"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GPL1261","term_id":"1261"}}</span>GPL1261</a>-GPL1355TermGeneZScores-MGI_MF_CC_RT_IW_BP_KEGG_z_z.binary.</p></div><div id="S18" class="sec"><h3 id="S18title">Analysis of association between bile acids and lead SNPs identified in this study</h3><p id="P40" class="p p-first-last">To identify bile acids associated with lead SNPs identified to be associated with the microbiome in this study, a generalized linear model with an inverse Gaussian distribution and log link was applied. As a supporting model, a two-part model was used comprising a GLM with binomial distribution and logit link for zero versus nonzero values, and a linear regression on log-transformed concentrations plus a constant (<em>c</em> = 1) for nonzero values. For both models, outliers with bile acid levels more than 5 s.d. from the mean were excluded and the covariates age, sex, BMI, vitamin K, alcohol, bile acid batch number and PC1–3 were included. The analysis included 520 samples.</p></div><div id="S19" class="sec"><h3 id="S19title"><em>Cis</em>- and <em>trans</em>-eQTL analysis on human data</h3><p id="P41" class="p p-first-last">For SNPs identified as associated with β diversity and/or single bacterial traits, a <em>cis</em>- and <em>trans</em>-eQTL analysis was performed using data on 2,360 individuals. The analysis design and recourse are described in detail in previous studies<sup><a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112161">63</a>,<a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112208">64</a></sup>. In summary, <em>cis</em>-eQTL analysis was performed on SNP–probe pairs for cases where the distance was less than 1 Mb. To consider the effects of SNPs in LD with a disease-associated SNP (trait–SNP), a conditioned analysis was performed by first adjusting the probe expression level for the effect of the strongest associated local SNPs (eSNP) and then repeating the eQTL analysis. Likewise, the <em>P</em> value for the local best SNP was calculated with conditioning on the trait SNP. To control for FDR, sample labels were permuted 100 times to obtain a <em>P</em>-value distribution. Expression probes with a significant association (FDR &lt; 5%, two-way conditional analysis for <em>cis</em>-eQTL analysis) to a trait SNP are given in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6 and 7</a>.</p></div><div id="S20" class="sec"><h3 id="S20title">Analysis of gut microbiome data from <em>Vdr</em>-knockout mice</h3><p id="P42" class="p p-first-last">Gut microbiome data from Jin <em>et al</em>.<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112221">25</a></sup> include fecal samples from three wild-type and five <em>Vdr</em>-knockout mice for which the V4–V6 region of the 16S rRNA gene was sequenced on the 454 GS-FLX platform. Quality filtering, removal of chimeras and classification were performed according to the same procedure described in the previous section. Statistical tests for the effect of <em>Vdr</em> genotype on the microbiome were carried out with the ‘envfit’ function in ‘vegan’ as described for the analysis for human SNPs. Comparison of specific taxa was carried out by the Wilcoxon test. Results are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 5 and 6</a>.</p></div><div id="S21" class="sec"><h3 id="S21title">Analysis of association of bile acids and fatty acids with the microbiome</h3><p id="P43" class="p p-first-last">To identify bacteria associated with the concentration of measured bile acids, including total LCA (the sum of LCA, G.LCA and T.LCA) and total BA (sum of all 15 bile acids), a generalized linear model with an inverse Gaussian distribution and log link was applied, excluding outliers more than 5 s.d. from the mean for bacteria and bile acids, adding a constant (<em>c</em> = 1) to bile acid concentration and including the covariates age, sex, BMI, total energy intake, water, alcohol and bile acid batch number (<em>n</em> = 569). To identify bacteria associated with ω3 and ω6 fatty acids, a linear regression model was applied with a square root transformation of fatty acids, excluding outliers with values more than 5 s.d. from the mean for bacteria and including the covariates age, sex, BMI, total energy intake, water and alcohol. Two samples with negative concentrations found for C22.2n.6 were excluded, leaving 567 samples in the fatty acid analysis. Benjamini–Hochberg corrected <em>P</em> values were calculated for each dependent variable to determine significance (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>).</p></div><div id="S22" class="sec"><h3 id="S22title">Shotgun metagenomic analysis</h3><p id="P44" class="p p-first-last">For a subset of 197 individuals, the same DNA extracts used in 16S rRNA gene sequencing were subjected to shotgun metagenomic sequencing. Samples were prepared following the protocol for the Illumina Nextera DNA Library Preparation kit and sequenced on the HiSeq Platform as 2 × 125 bp paired-end reads. Nextera adaptor sequences were trimmed using Trimmomatic (v0.32)<sup><a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112197">65</a></sup>. Quality control of the sequencing reads was performed with sickle (v1.330), and parameters were set to a sliding-window quality threshold of 20 and a minimum length of 60 after quality trimming. DeconSeq<sup><a href="#R66" rid="R66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_642112196">66</a></sup> was run to identify and remove human reads from the sequencing file, using the hg19 human genome sequence as the reference database. If one of the reads belonging to a read pair was removed at any of the quality control steps, the respective paired read was discarded as well. Samples that passed quality control, with no diagnosed IBD, IBS or diabetes and with genetic data (<em>n</em> = 122), were analyzed using HUMAnN2 with default settings except ‘–bt2_ps sensitive’ for the analysis of pathway and gene family abundance. Tables were normalized to relative abundance using ‘humann2_renorm_table –units relab’. Gene families including the term ‘bile acid’ were selected, and four pathways relevant for bile acid metabolism were selected (bile acid degradation, iso-bile acid biosynthesis I + II, bile acid biosynthesis, neutral pathway and glycocholate metabolism (bacteria)). Association with <em>VDR</em> genotype (rs7974353) was evaluated using GLM with an inverse Gaussian distribution, the covariates BMI, age, sex, alcohol, water and total energy intake and removal of outliers more than 5 s.d. from the mean and a constant (<em>c</em> = 1) added to abundance followed by multiplication by 1 × 10<sup>6</sup>.</p></div><div id="S23" class="sec sec-last"><h3 id="S23title">Replication in the FoCus obesity cohort</h3><p id="P45" class="p p-first-last">SNPs found to be significantly associated with β diversity in this study were consequently replicated in an additional FoCus obesity cohort (<em>n</em> = 371). The FoCus obesity cohort was recruited from the Obesity Outpatient Centre at the University Hospital in Kiel, which offers both non-surgical and surgical obesity therapies. Similar phenotype and genotyping profiles were obtained for the FoCus control cohort. The recruitment of the FoCus obesity cohort was approved by the local Ethics Committee (A156/03), and each patient gave their informed consent. To replicate associations of lead SNPs with β diversity, the effect size of each SNP was calculated with ‘envfit’, and consequent <em>P</em> values were calculated on the basis of the same empirical null distributions described above; successful replications are defined as having <em>P</em> &lt; 0.05/42 (in total, 42 SNPs were included in the test).</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>2</h4><div class="sup-box half_rhythm" id="d36e1738"><a href="/pmc/articles/PMC5626933/bin/NIHMS904581-supplement-2.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(13M, pdf)</sup></div></div></div><div id="S24" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S24title">Acknowledgments</h2><div class="sec"><p id="P53">We thank A.D. Paterson and colleagues for support in selection of models for GWAS. We further thank Der Norddeutsche Verbund für Hoch- und Höchstleistungsrechnen (HLRN) and S. Knief and H. Marten for computational resources and support. This work was supported by German Research Foundation (DFG) Collaborative Research Center 1182, ‘Origin and Function of Metaorganisms’ (J.F.B. and A.F.) and Excellence Cluster 306, ‘Inflammation at Interfaces’ (J.F.B. and A.F.) and by German Federal Ministry of Education and Research (BMBF) project ‘SysINFLAME’ (J.F.B. and A.F.). Project support was also provided by the Norwegian PSC Research Center and the Western Norway Regional Health Authority (grant 911802) (T.H.K.). M.K. is the recipient of a Postdoctoral Research Fellowship from the German Research Foundation (DFG). J.R.H. was funded by the Norwegian Research Council (240787/F20).</p></div></div><div id="idm140468804423376" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140468804423376title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN5"></p><p id="P46" class="p p-first-last">URLs. PopGen Biobank, <a href="https://www.epidemiologie.uni-kiel.de/biobanking/biobank-popgen" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.epidemiologie.uni-kiel.de/biobanking/biobank-popgen</a>; GTEx, <a href="http://www.gtexportal.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gtexportal.org/</a>; RegulomeDB, <a href="http://www.regulomedb.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.regulomedb.org/</a>; TRAP, <a href="http://trap.molgen.mpg.de/cgi-bin/home.cgi" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://trap.molgen.mpg.de/cgi-bin/home.cgi</a>; GWAS Catalog, <a href="http://www.ebi.ac.uk/gwas" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ebi.ac.uk/gwas</a>; CASAVA, <a href="http://support.illumina.com/sequencing/sequencing_software/casava" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://support.illumina.com/sequencing/sequencing_software/casava</a>; FastqToolkit, <a href="http://hannonlab.cshl.edu/fastx_toolkit/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hannonlab.cshl.edu/fastx_toolkit/</a>; vegan R package, <a href="http://vegan.r-forge.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://vegan.r-forge.r-project.org/</a>; MASS R package, <a href="http://cran.r-project.org/package=MASS" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cran.r-project.org/package=MASS</a>; pscl R package, <a href="http://cran.r-project.org/package=pscl" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cran.r-project.org/package=pscl</a>; HUMAnN2, <a href="http://www.bitbucket.org/biobakery/humann2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.bitbucket.org/biobakery/humann2</a>; InnateDB, <a href="http://www.innatedb.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.innatedb.com/</a>; cisRED, <a href="http://www.cisred.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cisred.org/</a>; sickle, <a href="http://github.com/najoshi/sickle" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://github.com/najoshi/sickle</a>; German Food Code and Nutrient Database, <a href="http://www.mri.bund.de/de/service/datenbanken/bundeslebensmittelschluessel/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.mri.bund.de/de/service/datenbanken/bundeslebensmittelschluessel/</a>.</p><p></p><p class="fn sec" id="FN6"></p><p id="P47" class="p p-first-last"><strong>Data access.</strong> All samples and information on their corresponding phenotypes and dietary behavior were obtained from the PopGen Biobank (Schleswig-Holstein, Germany) and can be accessed through a Material Data Access Form. Information about the Material Data Access Form and how to apply can be found at <a href="http://www.uksh.de/p2n/Information+for+Researchers.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.uksh.de/p2n/Information+for+Researchers.html</a>.</p><p></p><p class="fn sec" id="FN7"></p><p id="P48" class="p p-first-last">Note: Any <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a> and Source Data files are available in the <a href="http://dx.doi.org/10.1038/ng.3695" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">online version of the paper</a>.</p><p></p><div><p id="P49" class="p p-first"><strong>AUTHOR CONTRIBUTIONS</strong></p>A.F., J.F.B. and T.H.K. conceived the project. U.N., W.L., M.L. and K.S. organized recruitment and sample collection for the PopGen and FoCus cohorts. Genotyping data were collected and processed by L.B.T., J. Skiecevicˇienė, J.R.H., F.D. and K.H.; nutritional data were generated and processed by S.S., M.P.-J., M. Koch and U.N.; microbiome data were generated and processed by J.W., P. Rausch, F.-A.H., M.C.R., P. Rosenstiel, K.C.-S., S.K. and J.F.B.; and bile acid and fatty acid data were generated and processed by S.A.-D., P.B., R.K.B., M.D’A. and H.-U.M. T.E., J. Sun, J.B., F.S., D.E., M.H., G.R., P.H., W.-H.P., R.S.-T., R.H. and P. Rosenstiel contributed to additional experiments and data for this study. Statistical analyses were performed by J.W., L.B.T., J. Skiecevicˇienė, P. Rausch and M. Kummen, and J.W., L.B.T., J. Skiecevicˇienė, P. Rausch, M. Kummen, J.R.H., M.D’A., H.-U.M., T.H.K., J.F.B. and A.F. interpreted the results. J.W., L.B.T., J. Skiecevicˇienė, P. Rausch, M. Kummen, J.R.H., T.H.K., J.F.B. and A.F. wrote the manuscript, with input from all other authors.</div><p class="fn sec" id="FN9"></p><p id="P51" class="p p-first"><strong>COMPETING FINANCIAL INTERESTS</strong></p><p id="P52" class="p p-last">The authors declare no competing financial interests.</p><p></p></div></div><div id="idm140468805214416" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140468805214416title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Ley RE, Peterson DA, Gordon JI. Ecological and evolutionary forces shaping microbial diversity in the human intestine. <span><span class="ref-journal">Cell. </span>2006;<span class="ref-vol">124</span>:837–848.</span> [<a href="/pubmed/16497592" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Ecological+and+evolutionary+forces+shaping+microbial+diversity+in+the+human+intestine&amp;author=RE+Ley&amp;author=DA+Peterson&amp;author=JI+Gordon&amp;volume=124&amp;publication_year=2006&amp;pages=837-848&amp;pmid=16497592&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Fraune S, Bosch TC. Why bacteria matter in animal development and evolution. <span><span class="ref-journal">BioEssays. </span>2010;<span class="ref-vol">32</span>:571–580.</span> [<a href="/pubmed/20544735" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BioEssays&amp;title=Why+bacteria+matter+in+animal+development+and+evolution&amp;author=S+Fraune&amp;author=TC+Bosch&amp;volume=32&amp;publication_year=2010&amp;pages=571-580&amp;pmid=20544735&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Sekirov I, Russell SL, Antunes LCBB, Finlay BB. Gut microbiota in health and disease. <span><span class="ref-journal">Physiol. Rev. </span>2010;<span class="ref-vol">90</span>:859–904.</span> [<a href="/pubmed/20664075" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol.+Rev&amp;title=Gut+microbiota+in+health+and+disease&amp;author=I+Sekirov&amp;author=SL+Russell&amp;author=LCBB+Antunes&amp;author=BB+Finlay&amp;volume=90&amp;publication_year=2010&amp;pages=859-904&amp;pmid=20664075&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Chow J, Mazmanian SK. A pathobiont of the microbiota balances host colonization and intestinal infammation. <span><span class="ref-journal">Cell Host Microbe. </span>2010;<span class="ref-vol">7</span>:265–276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2859213/">PMC free article</a>]</span> [<a href="/pubmed/20413095" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Host+Microbe&amp;title=A+pathobiont+of+the+microbiota+balances+host+colonization+and+intestinal+infammation&amp;author=J+Chow&amp;author=SK+Mazmanian&amp;volume=7&amp;publication_year=2010&amp;pages=265-276&amp;pmid=20413095&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Costello EK, Stagaman K, Dethlefsen L, Bohannan BJ, Relman DA. The application of ecological theory toward an understanding of the human microbiome. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">336</span>:1255–1262.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4208626/">PMC free article</a>]</span> [<a href="/pubmed/22674335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+application+of+ecological+theory+toward+an+understanding+of+the+human+microbiome&amp;author=EK+Costello&amp;author=K+Stagaman&amp;author=L+Dethlefsen&amp;author=BJ+Bohannan&amp;author=DA+Relman&amp;volume=336&amp;publication_year=2012&amp;pages=1255-1262&amp;pmid=22674335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Walter J, Ley R. The human gut microbiome: ecology and recent evolutionary changes. <span><span class="ref-journal">Annu. Rev. Microbiol. </span>2011;<span class="ref-vol">65</span>:411–429.</span> [<a href="/pubmed/21682646" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Microbiol&amp;title=The+human+gut+microbiome:+ecology+and+recent+evolutionary+changes&amp;author=J+Walter&amp;author=R+Ley&amp;volume=65&amp;publication_year=2011&amp;pages=411-429&amp;pmid=21682646&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Antonopoulos DA, et al.  Reproducible community dynamics of the gastrointestinal microbiota following antibiotic perturbation. <span><span class="ref-journal">Infect. Immun. </span>2009;<span class="ref-vol">77</span>:2367–2375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687343/">PMC free article</a>]</span> [<a href="/pubmed/19307217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect.+Immun&amp;title=Reproducible+community+dynamics+of+the+gastrointestinal+microbiota+following+antibiotic+perturbation&amp;author=DA+Antonopoulos&amp;volume=77&amp;publication_year=2009&amp;pages=2367-2375&amp;pmid=19307217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Caporaso JG, et al.  Moving pictures of the human microbiome. <span><span class="ref-journal">Genome Biol. </span>2011;<span class="ref-vol">12</span>:R50.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3271711/">PMC free article</a>]</span> [<a href="/pubmed/21624126" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Moving+pictures+of+the+human+microbiome&amp;author=JG+Caporaso&amp;volume=12&amp;publication_year=2011&amp;pages=R50&amp;pmid=21624126&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Eckburg PB, et al.  Diversity of the human intestinal microbial fora. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">308</span>:1635–1638.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1395357/">PMC free article</a>]</span> [<a href="/pubmed/15831718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Diversity+of+the+human+intestinal+microbial+fora&amp;author=PB+Eckburg&amp;volume=308&amp;publication_year=2005&amp;pages=1635-1638&amp;pmid=15831718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Qin J, et al.  A human gut microbial gene catalogue established by metagenomic sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">464</span>:59–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3779803/">PMC free article</a>]</span> [<a href="/pubmed/20203603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+human+gut+microbial+gene+catalogue+established+by+metagenomic+sequencing&amp;author=J+Qin&amp;volume=464&amp;publication_year=2010&amp;pages=59-65&amp;pmid=20203603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Yatsunenko T, et al.  Human gut microbiome viewed across age and geography. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">486</span>:222–227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3376388/">PMC free article</a>]</span> [<a href="/pubmed/22699611" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Human+gut+microbiome+viewed+across+age+and+geography&amp;author=T+Yatsunenko&amp;volume=486&amp;publication_year=2012&amp;pages=222-227&amp;pmid=22699611&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Goodrich JK, et al.  Human genetics shape the gut microbiome. <span><span class="ref-journal">Cell. </span>2014;<span class="ref-vol">159</span>:789–799.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4255478/">PMC free article</a>]</span> [<a href="/pubmed/25417156" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Human+genetics+shape+the+gut+microbiome&amp;author=JK+Goodrich&amp;volume=159&amp;publication_year=2014&amp;pages=789-799&amp;pmid=25417156&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Cotillard A, et al.  Dietary intervention impact on gut microbial gene richness. <span><span class="ref-journal">Nature. </span>2013;<span class="ref-vol">500</span>:585–588.</span> [<a href="/pubmed/23985875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Dietary+intervention+impact+on+gut+microbial+gene+richness&amp;author=A+Cotillard&amp;volume=500&amp;publication_year=2013&amp;pages=585-588&amp;pmid=23985875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">David LA, et al.  Diet rapidly and reproducibly alters the human gut microbiome. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">505</span>:559–563.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3957428/">PMC free article</a>]</span> [<a href="/pubmed/24336217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Diet+rapidly+and+reproducibly+alters+the+human+gut+microbiome&amp;author=LA+David&amp;volume=505&amp;publication_year=2014&amp;pages=559-563&amp;pmid=24336217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Rehman A, et al.  Geographical patterns of the standing and active human gut microbiome in health and IBD. <span><span class="ref-journal">Gut. </span>2016;<span class="ref-vol">65</span>:238–248.</span> [<a href="/pubmed/25567118" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Geographical+patterns+of+the+standing+and+active+human+gut+microbiome+in+health+and+IBD&amp;author=A+Rehman&amp;volume=65&amp;publication_year=2016&amp;pages=238-248&amp;pmid=25567118&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Maurice CF, Haiser HJ, Turnbaugh PJ. Xenobiotics shape the physiology and gene expression of the active human gut microbiome. <span><span class="ref-journal">Cell. </span>2013;<span class="ref-vol">152</span>:39–50.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3552296/">PMC free article</a>]</span> [<a href="/pubmed/23332745" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Xenobiotics+shape+the+physiology+and+gene+expression+of+the+active+human+gut+microbiome&amp;author=CF+Maurice&amp;author=HJ+Haiser&amp;author=PJ+Turnbaugh&amp;volume=152&amp;publication_year=2013&amp;pages=39-50&amp;pmid=23332745&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Rausch P, et al.  Colonic mucosa-associated microbiota is infuenced by an interaction of Crohn disease and <em>FUT2</em> (Secretor) genotype. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2011;<span class="ref-vol">108</span>:19030–19035.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3223430/">PMC free article</a>]</span> [<a href="/pubmed/22068912" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Colonic+mucosa-associated+microbiota+is+infuenced+by+an+interaction+of+Crohn+disease+and+FUT2+(Secretor)+genotype&amp;author=P+Rausch&amp;volume=108&amp;publication_year=2011&amp;pages=19030-19035&amp;pmid=22068912&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Rehman A, et al.  <em>Nod2</em> is essential for temporal development of intestinal microbial communities. <span><span class="ref-journal">Gut. </span>2011;<span class="ref-vol">60</span>:1354–1362.</span> [<a href="/pubmed/21421666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Nod2+is+essential+for+temporal+development+of+intestinal+microbial+communities&amp;author=A+Rehman&amp;volume=60&amp;publication_year=2011&amp;pages=1354-1362&amp;pmid=21421666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Blekhman R, et al.  Host genetic variation impacts microbiome composition across human body sites. <span><span class="ref-journal">Genome Biol. </span>2015;<span class="ref-vol">16</span>:191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4570153/">PMC free article</a>]</span> [<a href="/pubmed/26374288" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Host+genetic+variation+impacts+microbiome+composition+across+human+body+sites&amp;author=R+Blekhman&amp;volume=16&amp;publication_year=2015&amp;pages=191&amp;pmid=26374288&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Krawczak M, et al.  PopGen: population-based recruitment of patients and controls for the analysis of complex genotype-phenotype relationships. <span><span class="ref-journal">Community Genet. </span>2006;<span class="ref-vol">9</span>:55–61.</span> [<a href="/pubmed/16490960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Community+Genet&amp;title=PopGen:+population-based+recruitment+of+patients+and+controls+for+the+analysis+of+complex+genotype-phenotype+relationships&amp;author=M+Krawczak&amp;volume=9&amp;publication_year=2006&amp;pages=55-61&amp;pmid=16490960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Müller N, et al.  IL-6 blockade by monoclonal antibodies inhibits apolipoprotein (a) expression and lipoprotein (a) synthesis in humans. <span><span class="ref-journal">J. Lipid Res. </span>2015;<span class="ref-vol">56</span>:1034–1042.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4409280/">PMC free article</a>]</span> [<a href="/pubmed/25713100" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid+Res&amp;title=IL-6+blockade+by+monoclonal+antibodies+inhibits+apolipoprotein+(a)+expression+and+lipoprotein+(a)+synthesis+in+humans&amp;author=N+Müller&amp;volume=56&amp;publication_year=2015&amp;pages=1034-1042&amp;pmid=25713100&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Biedermann L, et al.  Smoking cessation induces profound changes in the composition of the intestinal microbiota in humans. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>:e59260.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3597605/">PMC free article</a>]</span> [<a href="/pubmed/23516617" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Smoking+cessation+induces+profound+changes+in+the+composition+of+the+intestinal+microbiota+in+humans&amp;author=L+Biedermann&amp;volume=8&amp;publication_year=2013&amp;pages=e59260&amp;pmid=23516617&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Haussler MR, et al.  Vitamin D receptor: molecular signaling and actions of nutritional ligands in disease prevention. <span><span class="ref-journal">Nutr. Rev. </span>2008;<span class="ref-vol">66</span>(Suppl. 2):S98–S112.</span> [<a href="/pubmed/18844852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nutr.+Rev&amp;title=Vitamin+D+receptor:+molecular+signaling+and+actions+of+nutritional+ligands+in+disease+prevention&amp;author=MR+Haussler&amp;volume=66&amp;issue=Suppl.+2&amp;publication_year=2008&amp;pages=S98-S112&amp;pmid=18844852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Makishima M, et al.  Vitamin D receptor as an intestinal bile acid sensor. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">296</span>:1313–1316.</span> [<a href="/pubmed/12016314" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Vitamin+D+receptor+as+an+intestinal+bile+acid+sensor&amp;author=M+Makishima&amp;volume=296&amp;publication_year=2002&amp;pages=1313-1316&amp;pmid=12016314&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Jin D, et al.  Lack ofes the functions of the murine intestinal microbiome. <span><span class="ref-journal">Clin. Ther. </span>2015;<span class="ref-vol">37</span>:996–1009. e7.</span> [<a href="/pubmed/26046242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Ther&amp;title=Lack+ofes+the+functions+of+the+murine+intestinal+microbiome&amp;author=D+Jin&amp;volume=37&amp;publication_year=2015&amp;pages=996-1009&amp;pmid=26046242&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">D’Aldebert E, et al.  Bile salts control the antimicrobial peptide cathelicidin through nuclear receptors in the human biliary epithelium. <span><span class="ref-journal">Gastroenterology. </span>2009;<span class="ref-vol">136</span>:1435–1443.</span> [<a href="/pubmed/19245866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Bile+salts+control+the+antimicrobial+peptide+cathelicidin+through+nuclear+receptors+in+the+human+biliary+epithelium&amp;author=E+D’Aldebert&amp;volume=136&amp;publication_year=2009&amp;pages=1435-1443&amp;pmid=19245866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Sayin SI, et al.  Gut microbiota regulates bile acid metabolism by reducing the levels of tauro-β-muricholic acid, a naturally occurring FXR antagonist. <span><span class="ref-journal">Cell Metab. </span>2013;<span class="ref-vol">17</span>:225–235.</span> [<a href="/pubmed/23395169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Metab&amp;title=Gut+microbiota+regulates+bile+acid+metabolism+by+reducing+the+levels+of+tauro-β-muricholic+acid,+a+naturally+occurring+FXR+antagonist&amp;author=SI+Sayin&amp;volume=17&amp;publication_year=2013&amp;pages=225-235&amp;pmid=23395169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Inoue Y, Yu AM, Inoue J, Gonzalez FJ. Hepatocyte nuclear factor 4α is a central regulator of bile acid conjugation. <span><span class="ref-journal">J. Biol. Chem. </span>2004;<span class="ref-vol">279</span>:2480–2489.</span> [<a href="/pubmed/14583614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Hepatocyte+nuclear+factor+4α+is+a+central+regulator+of+bile+acid+conjugation&amp;author=Y+Inoue&amp;author=AM+Yu&amp;author=J+Inoue&amp;author=FJ+Gonzalez&amp;volume=279&amp;publication_year=2004&amp;pages=2480-2489&amp;pmid=14583614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Sato H, et al.  Group III secreted phospholipase A2 transgenic mice spontaneously develop infammation. <span><span class="ref-journal">Biochem. J. </span>2009;<span class="ref-vol">421</span>:17–27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2708930/">PMC free article</a>]</span> [<a href="/pubmed/19371233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+J&amp;title=Group+III+secreted+phospholipase+A2+transgenic+mice+spontaneously+develop+infammation&amp;author=H+Sato&amp;volume=421&amp;publication_year=2009&amp;pages=17-27&amp;pmid=19371233&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Yano JM, et al.  Indigenous bacteria from the gut microbiota regulate host serotonin biosynthesis. <span><span class="ref-journal">Cell. </span>2015;<span class="ref-vol">161</span>:264–276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4393509/">PMC free article</a>]</span> [<a href="/pubmed/25860609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Indigenous+bacteria+from+the+gut+microbiota+regulate+host+serotonin+biosynthesis&amp;author=JM+Yano&amp;volume=161&amp;publication_year=2015&amp;pages=264-276&amp;pmid=25860609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Olivares M, et al.  The HLA-DQ2 genotype selects for early intestinal microbiota composition in infants at high risk of developing coeliac disease. <span><span class="ref-journal">Gut. </span>2015;<span class="ref-vol">64</span>:406–417.</span> [<a href="/pubmed/24939571" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=The+HLA-DQ2+genotype+selects+for+early+intestinal+microbiota+composition+in+infants+at+high+risk+of+developing+coeliac+disease&amp;author=M+Olivares&amp;volume=64&amp;publication_year=2015&amp;pages=406-417&amp;pmid=24939571&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Okada Y, et al.  HLA-Cw*1202–B*5201–DRB1*1502 haplotype increases risk for ulcerative colitis but reduces risk for Crohn’s disease. <span><span class="ref-journal">Gastroenterology. </span>2011;<span class="ref-vol">141</span>:864–871. e1.</span> 5. [<a href="/pubmed/21699788" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=HLA-Cw*1202–B*5201–DRB1*1502+haplotype+increases+risk+for+ulcerative+colitis+but+reduces+risk+for+Crohn’s+disease&amp;author=Y+Okada&amp;volume=141&amp;publication_year=2011&amp;pages=864-871&amp;pmid=21699788&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Arimura Y, et al.  Characteristics of Japanese infammatory bowel disease susceptibility loci. <span><span class="ref-journal">J. Gastroenterol. </span>2014;<span class="ref-vol">49</span>:1217–1230.</span> [<a href="/pubmed/23942620" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Gastroenterol&amp;title=Characteristics+of+Japanese+infammatory+bowel+disease+susceptibility+loci&amp;author=Y+Arimura&amp;volume=49&amp;publication_year=2014&amp;pages=1217-1230&amp;pmid=23942620&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Terao C, et al.  Two susceptibility loci to Takayasu arteritis reveal a synergistic role of the <em>IL12B</em> and <em>HLA-B</em> regions in a Japanese population. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2013;<span class="ref-vol">93</span>:289–297.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3738822/">PMC free article</a>]</span> [<a href="/pubmed/23830516" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Two+susceptibility+loci+to+Takayasu+arteritis+reveal+a+synergistic+role+of+the+IL12B+and+HLA-B+regions+in+a+Japanese+population&amp;author=C+Terao&amp;volume=93&amp;publication_year=2013&amp;pages=289-297&amp;pmid=23830516&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Benson AK, et al.  Individuality in gut microbiota composition is a complex polygenic trait shaped by multiple environmental and host genetic factors. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2010;<span class="ref-vol">107</span>:18933–18938.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2973891/">PMC free article</a>]</span> [<a href="/pubmed/20937875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Individuality+in+gut+microbiota+composition+is+a+complex+polygenic+trait+shaped+by+multiple+environmental+and+host+genetic+factors&amp;author=AK+Benson&amp;volume=107&amp;publication_year=2010&amp;pages=18933-18938&amp;pmid=20937875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Pers TH, et al.  Biological interpretation of genome-wide association studies using predicted gene functions. <span><span class="ref-journal">Nat. Commun. </span>2015;<span class="ref-vol">6</span>:5890.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4420238/">PMC free article</a>]</span> [<a href="/pubmed/25597830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Biological+interpretation+of+genome-wide+association+studies+using+predicted+gene+functions&amp;author=TH+Pers&amp;volume=6&amp;publication_year=2015&amp;pages=5890&amp;pmid=25597830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Phay JE, Hussain HB, Moley JF. Cloning and expression analysis of a novel member of the facilitative glucose transporter family, <em>SLC2A9</em> (<em>GLUT9</em>) <span><span class="ref-journal">Genomics. </span>2000;<span class="ref-vol">66</span>:217–220.</span> [<a href="/pubmed/10860667" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Cloning+and+expression+analysis+of+a+novel+member+of+the+facilitative+glucose+transporter+family,+SLC2A9+(GLUT9)&amp;author=JE+Phay&amp;author=HB+Hussain&amp;author=JF+Moley&amp;volume=66&amp;publication_year=2000&amp;pages=217-220&amp;pmid=10860667&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Kliewer SA, Umesono K, Noonan DJ, Heyman RA, Evans RM. Convergence of 9-<em>cis</em> retinoic acid and peroxisome proliferator signalling pathways through heterodimer formation of their receptors. <span><span class="ref-journal">Nature. </span>1992;<span class="ref-vol">358</span>:771–774.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6159883/">PMC free article</a>]</span> [<a href="/pubmed/1324435" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Convergence+of+9-cis+retinoic+acid+and+peroxisome+proliferator+signalling+pathways+through+heterodimer+formation+of+their+receptors&amp;author=SA+Kliewer&amp;author=K+Umesono&amp;author=DJ+Noonan&amp;author=RA+Heyman&amp;author=RM+Evans&amp;volume=358&amp;publication_year=1992&amp;pages=771-774&amp;pmid=1324435&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Repa JJ, et al.  Regulation of absorption and ABC1-mediated effux of cholesterol by RXR heterodimers. <span><span class="ref-journal">Science. </span>2000;<span class="ref-vol">289</span>:1524–1529.</span> [<a href="/pubmed/10968783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Regulation+of+absorption+and+ABC1-mediated+effux+of+cholesterol+by+RXR+heterodimers&amp;author=JJ+Repa&amp;volume=289&amp;publication_year=2000&amp;pages=1524-1529&amp;pmid=10968783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Wahlström A, Sayin SI, Marschall HU, Bäckhed F. Intestinal crosstalk between bile acids and microbiota and its impact on host metabolism. <span><span class="ref-journal">Cell Metab. </span>2016;<span class="ref-vol">24</span>:41–50.</span> [<a href="/pubmed/27320064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Metab&amp;title=Intestinal+crosstalk+between+bile+acids+and+microbiota+and+its+impact+on+host+metabolism&amp;author=A+Wahlström&amp;author=SI+Sayin&amp;author=HU+Marschall&amp;author=F+Bäckhed&amp;volume=24&amp;publication_year=2016&amp;pages=41-50&amp;pmid=27320064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Duparc T, et al.  Hepatocyte MyD88 affects bile acids, gut microbiota and metabolome contributing to regulate glucose and lipid metabolism. <span><span class="ref-journal">Gut. </span>2016</span>  <a href="http://dx.doi.org/10.1136/gutjnl-2015-310904" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://dx.doi.org/10.1136/gutjnl-2015-310904</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5529962/">PMC free article</a>]</span> [<a href="/pubmed/27196572" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Jostins L, et al.  Host-microbe interactions have shaped the genetic architecture of infammatory bowel disease. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:119–124.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3491803/">PMC free article</a>]</span> [<a href="/pubmed/23128233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Host-microbe+interactions+have+shaped+the+genetic+architecture+of+infammatory+bowel+disease&amp;author=L+Jostins&amp;volume=491&amp;publication_year=2012&amp;pages=119-124&amp;pmid=23128233&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Liu JZ, et al.  Dense genotyping of immune-related disease regions identifes nine new risk loci for primary sclerosing cholangitis. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:670–675.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3667736/">PMC free article</a>]</span> [<a href="/pubmed/23603763" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Dense+genotyping+of+immune-related+disease+regions+identifes+nine+new+risk+loci+for+primary+sclerosing+cholangitis&amp;author=JZ+Liu&amp;volume=45&amp;publication_year=2013&amp;pages=670-675&amp;pmid=23603763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Sun J. VDR/vitamin D receptor regulates autophagic activity through <em>ATG16L1</em>. <span><span class="ref-journal">Autophagy. </span>2016;<span class="ref-vol">12</span>:1057–1058.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4922437/">PMC free article</a>]</span> [<a href="/pubmed/26218741" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=VDR/vitamin+D+receptor+regulates+autophagic+activity+through+ATG16L1&amp;author=J+Sun&amp;volume=12&amp;publication_year=2016&amp;pages=1057-1058&amp;pmid=26218741&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Krude H, Biebermann H, Gruters A. Mutations in the human proopiomelanocortin gene. <span><span class="ref-journal">Ann. NY Acad. Sci. </span>2003;<span class="ref-vol">994</span>:233–239.</span> [<a href="/pubmed/12851321" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+NY+Acad.+Sci&amp;title=Mutations+in+the+human+proopiomelanocortin+gene&amp;author=H+Krude&amp;author=H+Biebermann&amp;author=A+Gruters&amp;volume=994&amp;publication_year=2003&amp;pages=233-239&amp;pmid=12851321&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Tuoresmäki P, Väisänen S, Neme A, Heikkinen S, Carlberg C. Patterns of genome-wide VDR locations. <span><span class="ref-journal">PLoS One. </span>2014;<span class="ref-vol">9</span>:e96105.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4005760/">PMC free article</a>]</span> [<a href="/pubmed/24787735" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Patterns+of+genome-wide+VDR+locations&amp;author=P+Tuoresmäki&amp;author=S+Väisänen&amp;author=A+Neme&amp;author=S+Heikkinen&amp;author=C+Carlberg&amp;volume=9&amp;publication_year=2014&amp;pages=e96105&amp;pmid=24787735&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Wang J, et al.  Analysis of intestinal microbiota in hybrid house mice reveals evolutionary divergence in a vertebrate hologenome. <span><span class="ref-journal">Nat. Commun. </span>2015;<span class="ref-vol">6</span>:6440.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4366507/">PMC free article</a>]</span> [<a href="/pubmed/25737238" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Analysis+of+intestinal+microbiota+in+hybrid+house+mice+reveals+evolutionary+divergence+in+a+vertebrate+hologenome&amp;author=J+Wang&amp;volume=6&amp;publication_year=2015&amp;pages=6440&amp;pmid=25737238&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Srinivas G, et al.  Genome-wide mapping of gene-microbiota interactions in susceptibility to autoimmune skin blistering. <span><span class="ref-journal">Nat. Commun. </span>2013;<span class="ref-vol">4</span>:2462.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3778513/">PMC free article</a>]</span> [<a href="/pubmed/24042968" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Genome-wide+mapping+of+gene-microbiota+interactions+in+susceptibility+to+autoimmune+skin+blistering&amp;author=G+Srinivas&amp;volume=4&amp;publication_year=2013&amp;pages=2462&amp;pmid=24042968&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">McKnite AM, et al.  Murine gut microbiota is defned by host genetics and modulates variation of metabolic traits. <span><span class="ref-journal">PLoS One. </span>2012;<span class="ref-vol">7</span>:e39191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3377628/">PMC free article</a>]</span> [<a href="/pubmed/22723961" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Murine+gut+microbiota+is+defned+by+host+genetics+and+modulates+variation+of+metabolic+traits&amp;author=AM+McKnite&amp;volume=7&amp;publication_year=2012&amp;pages=e39191&amp;pmid=22723961&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Leamy LJ, et al.  Host genetics and diet, but not immunoglobulin A expression, converge to shape compositional features of the gut microbiome in an advanced intercross population of mice. <span><span class="ref-journal">Genome Biol. </span>2014;<span class="ref-vol">15</span>:552.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4290092/">PMC free article</a>]</span> [<a href="/pubmed/25516416" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Host+genetics+and+diet,+but+not+immunoglobulin+A+expression,+converge+to+shape+compositional+features+of+the+gut+microbiome+in+an+advanced+intercross+population+of+mice&amp;author=LJ+Leamy&amp;volume=15&amp;publication_year=2014&amp;pages=552&amp;pmid=25516416&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Kozich JJ, Westcott SL, Baxter NT, Highlander SK, Schloss PD. Development of a dual-index sequencing strategy and curation pipeline for analyzing amplicon sequence data on the MiSeq Illumina sequencing platform. <span><span class="ref-journal">Appl. Environ.Microbiol. </span>2013;<span class="ref-vol">79</span>:5112–5120.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3753973/">PMC free article</a>]</span> [<a href="/pubmed/23793624" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.Microbiol&amp;title=Development+of+a+dual-index+sequencing+strategy+and+curation+pipeline+for+analyzing+amplicon+sequence+data+on+the+MiSeq+Illumina+sequencing+platform&amp;author=JJ+Kozich&amp;author=SL+Westcott&amp;author=NT+Baxter&amp;author=SK+Highlander&amp;author=PD+Schloss&amp;volume=79&amp;publication_year=2013&amp;pages=5112-5120&amp;pmid=23793624&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Magocˇ T, Salzberg SL. FLASH: fast length adjustment of short reads to improve genome assemblies. <span><span class="ref-journal">Bioinformatics. </span>2011;<span class="ref-vol">27</span>:2957–2963.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3198573/">PMC free article</a>]</span> [<a href="/pubmed/21903629" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=FLASH:+fast+length+adjustment+of+short+reads+to+improve+genome+assemblies&amp;author=T+Magocˇ&amp;author=SL+Salzberg&amp;volume=27&amp;publication_year=2011&amp;pages=2957-2963&amp;pmid=21903629&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Edgar RC, Haas BJ, Clemente JC, Quince C, Knight R. UCHIME improves sensitivity and speed of chimera detection. <span><span class="ref-journal">Bioinformatics. </span>2011;<span class="ref-vol">27</span>:2194–2200.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3150044/">PMC free article</a>]</span> [<a href="/pubmed/21700674" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=UCHIME+improves+sensitivity+and+speed+of+chimera+detection&amp;author=RC+Edgar&amp;author=BJ+Haas&amp;author=JC+Clemente&amp;author=C+Quince&amp;author=R+Knight&amp;volume=27&amp;publication_year=2011&amp;pages=2194-2200&amp;pmid=21700674&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Wang Q, Garrity GM, Tiedje JM, Cole JR. Naive Bayesian classifer for rapid assignment of rRNA sequences into the new bacterial taxonomy. <span><span class="ref-journal">Appl. Environ. Microbiol. </span>2007;<span class="ref-vol">73</span>:5261–5267.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950982/">PMC free article</a>]</span> [<a href="/pubmed/17586664" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol&amp;title=Naive+Bayesian+classifer+for+rapid+assignment+of+rRNA+sequences+into+the+new+bacterial+taxonomy&amp;author=Q+Wang&amp;author=GM+Garrity&amp;author=JM+Tiedje&amp;author=JR+Cole&amp;volume=73&amp;publication_year=2007&amp;pages=5261-5267&amp;pmid=17586664&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Edgar RC. UPARSE: highly accurate OTU sequences from microbial amplicon reads. <span><span class="ref-journal">Nat. Methods. </span>2013;<span class="ref-vol">10</span>:996–998.</span> [<a href="/pubmed/23955772" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=UPARSE:+highly+accurate+OTU+sequences+from+microbial+amplicon+reads&amp;author=RC+Edgar&amp;volume=10&amp;publication_year=2013&amp;pages=996-998&amp;pmid=23955772&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Abu-Hayyeh S, et al.  Prognostic and mechanistic potential of progesterone sulfates in intrahepatic cholestasis of pregnancy and pruritus gravidarum. <span><span class="ref-journal">Hepatology. </span>2016;<span class="ref-vol">63</span>:1287–1298.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4869673/">PMC free article</a>]</span> [<a href="/pubmed/26426865" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Prognostic+and+mechanistic+potential+of+progesterone+sulfates+in+intrahepatic+cholestasis+of+pregnancy+and+pruritus+gravidarum&amp;author=S+Abu-Hayyeh&amp;volume=63&amp;publication_year=2016&amp;pages=1287-1298&amp;pmid=26426865&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Bjørndal B, et al.  Krill powder increases liver lipid catabolism and reduces glucose mobilization in tumor necrosis factor-α transgenic mice fed a high-fat diet. <span><span class="ref-journal">Metabolism. </span>2012;<span class="ref-vol">61</span>:1461–1472.</span> [<a href="/pubmed/22538117" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metabolism&amp;title=Krill+powder+increases+liver+lipid+catabolism+and+reduces+glucose+mobilization+in+tumor+necrosis+factor-α+transgenic+mice+fed+a+high-fat+diet&amp;author=B+Bjørndal&amp;volume=61&amp;publication_year=2012&amp;pages=1461-1472&amp;pmid=22538117&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Nöthlings U, Hoffmann K, Bergmann MM, Boeing H. Fitting portion sizes in a self-administered food frequency questionnaire. <span><span class="ref-journal">J. Nutr. </span>2007;<span class="ref-vol">137</span>:2781–2786.</span> [<a href="/pubmed/18029499" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Nutr&amp;title=Fitting+portion+sizes+in+a+self-administered+food+frequency+questionnaire&amp;author=U+Nöthlings&amp;author=K+Hoffmann&amp;author=MM+Bergmann&amp;author=H+Boeing&amp;volume=137&amp;publication_year=2007&amp;pages=2781-2786&amp;pmid=18029499&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Dehne LI, Klemm C, Henseler G, Hermann-Kunz E. The German food code and nutrient data base (BLS II.2) <span><span class="ref-journal">Eur. J. Epidemiol. </span>1999;<span class="ref-vol">15</span>:355–359.</span> [<a href="/pubmed/10414376" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Epidemiol&amp;title=The+German+food+code+and+nutrient+data+base+(BLS+II.2)&amp;author=LI+Dehne&amp;author=C+Klemm&amp;author=G+Henseler&amp;author=E+Hermann-Kunz&amp;volume=15&amp;publication_year=1999&amp;pages=355-359&amp;pmid=10414376&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="element-citation">Xu L, Paterson AD, Turpin W, Xu W. Assessment and selection of competing models for zero-infated microbiome data. <span><span class="ref-journal">PLoS One. </span>2015;<span class="ref-vol">10</span>:e0129606.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4493133/">PMC free article</a>]</span> [<a href="/pubmed/26148172" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Assessment+and+selection+of+competing+models+for+zero-infated+microbiome+data&amp;author=L+Xu&amp;author=AD+Paterson&amp;author=W+Turpin&amp;author=W+Xu&amp;volume=10&amp;publication_year=2015&amp;pages=e0129606&amp;pmid=26148172&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="element-citation">Degenhardt F, et al.  Genome-wide association study of serum coenzyme Q10 levels identifes susceptibility loci linked to neuronal diseases. <span><span class="ref-journal">Hum. Mol. Genet. </span>2016</span>  <a href="http://dx.doi.org/10.1093/hmg/ddw134" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://dx.doi.org/10.1093/hmg/ddw134</a>. [<a href="/pubmed/27149984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="element-citation">Ellinghaus D, et al.  Analysis of fve chronic infammatory diseases identifes 27 new associations and highlights disease-specifc patterns at shared loci. <span><span class="ref-journal">Nat. Genet. </span>2016;<span class="ref-vol">48</span>:510–518.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4848113/">PMC free article</a>]</span> [<a href="/pubmed/26974007" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Analysis+of+fve+chronic+infammatory+diseases+identifes+27+new+associations+and+highlights+disease-specifc+patterns+at+shared+loci&amp;author=D+Ellinghaus&amp;volume=48&amp;publication_year=2016&amp;pages=510-518&amp;pmid=26974007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="element-citation">Wood AR, et al.  Defning the role of common variation in the genomic and biological architecture of adult human height. <span><span class="ref-journal">Nat. Genet. </span>2014;<span class="ref-vol">46</span>:1173–1186.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4250049/">PMC free article</a>]</span> [<a href="/pubmed/25282103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Defning+the+role+of+common+variation+in+the+genomic+and+biological+architecture+of+adult+human+height&amp;author=AR+Wood&amp;volume=46&amp;publication_year=2014&amp;pages=1173-1186&amp;pmid=25282103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="element-citation">Bolger AM, Lohse M, Usadel B. Trimmomatic: a fexible trimmer for Illumina sequence data. <span><span class="ref-journal">Bioinformatics. </span>2014;<span class="ref-vol">30</span>:2114–2120.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4103590/">PMC free article</a>]</span> [<a href="/pubmed/24695404" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:+a+fexible+trimmer+for+Illumina+sequence+data&amp;author=AM+Bolger&amp;author=M+Lohse&amp;author=B+Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R66">66. <span class="element-citation">Schmieder R, Edwards R. Fast identifcation and removal of sequence contamination from genomic and metagenomic datasets. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>:e17288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3052304/">PMC free article</a>]</span> [<a href="/pubmed/21408061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Fast+identifcation+and+removal+of+sequence+contamination+from+genomic+and+metagenomic+datasets&amp;author=R+Schmieder&amp;author=R+Edwards&amp;volume=6&amp;publication_year=2011&amp;pages=e17288&amp;pmid=21408061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5626933&amp;issue-id=299379&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140468801850608title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S6title">DISCUSSION</a></li><li><a href="#S7title">METHODS</a></li><li><a href="#S8title">ONLINE METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S24title">Acknowledgments</a></li><li><a href="#idm140468804423376title">Footnotes</a></li><li><a href="#idm140468805214416title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5626933/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5626933/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5626933/pdf/nihms904581.pdf">PDF (1.3M)</a></li> | <li><a href="#" data-citationid="PMC5626933" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5626933%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5626933%2F&amp;text=Genome-wide%20association%20analysis%20identifies%20variation%20in%20vitamin%20D%20receptor%20and%20other%20host%20factors%20influencing%20the%20gut%20microbiota" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5626933%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27723756" data-aiid="5626933" data-aid="5626933" data-iid="299379" data-domainid="319" data-domain="nihpa" data-accid="PMC5626933" data-md5="045f2bf8c37672b7d7e33b6c08dbc4f6"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:16:19-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D8AA4DD59DBC10000000000D8007F&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5626933%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5626933/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>