<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5027680/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genetic variants in RBFOX3 are associated with sleep latency</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5027680/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="Genetic variants in RBFOX3 are associated with sleep latency"><meta name="citation_authors" content="Najaf Amin, Karla V Allebrandt, Ashley van der Spek, Bertram Müller-Myhsok, Karin Hek, Maris Teder-Laving, Caroline Hayward, Tõnu Esko, Josine G van Mill, Hamdi Mbarek, Nathaniel F Watson, Scott A Melville, Fabiola M Del Greco, Enda M Byrne, Edwin Oole, Ivana Kolcic, Ting-hsu Chen, Daniel S Evans, Josef Coresh, Nicole Vogelzangs, Juha Karjalainen, Gonneke Willemsen, Sina A Gharib, Lina Zgaga, Evelin Mihailov, Katie L Stone, Harry Campbell, Rutger WW Brouwer, Ayse Demirkan, Aaron Isaacs, Zoran Dogas, Kristin D Marciante, Susan Campbell, Fran Borovecki, Annemarie I Luik, Man Li, Jouke Jan Hottenga, Jennifer E Huffman, Mirjam CGN van den Hout, Steven R Cummings, Yurii S Aulchenko, Philip R Gehrman, André G Uitterlinden, Heinz-Erich Wichmann, Martina Müller-Nurasyid, Rudolf SN Fehrmann, Grant W Montgomery, Albert Hofman, Wen Hong Linda Kao, Ben A Oostra, Alan F Wright, Jacqueline M Vink, James F Wilson, Peter P Pramstaller, Andrew A Hicks, Ozren Polasek, Naresh M Punjabi, Susan Redline, Bruce M Psaty, Andrew C Heath, Martha Merrow, Gregory J Tranah, Daniel J Gottlieb, Dorret I Boomsma, Nicholas G Martin, Igor Rudan, Henning Tiemeier, Wilfred FJ van IJcken, Brenda W Penninx, Andres Metspalu, Thomas Meitinger, Lude Franke, Till Roenneberg, Cornelia M van Duijn"><meta name="citation_date" content="October 2016"><meta name="citation_issue" content="10"><meta name="citation_volume" content="24"><meta name="citation_firstpage" content="1488"><meta name="citation_doi" content="10.1038/ejhg.2016.31"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5027680/?report=abstract"><meta name="citation_pmid" content="27142678"><meta name="DC.Title" content="Genetic variants in RBFOX3 are associated with sleep latency"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Najaf Amin"><meta name="DC.Contributor" content="Karla V Allebrandt"><meta name="DC.Contributor" content="Ashley van der Spek"><meta name="DC.Contributor" content="Bertram Müller-Myhsok"><meta name="DC.Contributor" content="Karin Hek"><meta name="DC.Contributor" content="Maris Teder-Laving"><meta name="DC.Contributor" content="Caroline Hayward"><meta name="DC.Contributor" content="Tõnu Esko"><meta name="DC.Contributor" content="Josine G van Mill"><meta name="DC.Contributor" content="Hamdi Mbarek"><meta name="DC.Contributor" content="Nathaniel F Watson"><meta name="DC.Contributor" content="Scott A Melville"><meta name="DC.Contributor" content="Fabiola M Del Greco"><meta name="DC.Contributor" content="Enda M Byrne"><meta name="DC.Contributor" content="Edwin Oole"><meta name="DC.Contributor" content="Ivana Kolcic"><meta name="DC.Contributor" content="Ting-hsu Chen"><meta name="DC.Contributor" content="Daniel S Evans"><meta name="DC.Contributor" content="Josef Coresh"><meta name="DC.Contributor" content="Nicole Vogelzangs"><meta name="DC.Contributor" content="Juha Karjalainen"><meta name="DC.Contributor" content="Gonneke Willemsen"><meta name="DC.Contributor" content="Sina A Gharib"><meta name="DC.Contributor" content="Lina Zgaga"><meta name="DC.Contributor" content="Evelin Mihailov"><meta name="DC.Contributor" content="Katie L Stone"><meta name="DC.Contributor" content="Harry Campbell"><meta name="DC.Contributor" content="Rutger WW Brouwer"><meta name="DC.Contributor" content="Ayse Demirkan"><meta name="DC.Contributor" content="Aaron Isaacs"><meta name="DC.Contributor" content="Zoran Dogas"><meta name="DC.Contributor" content="Kristin D Marciante"><meta name="DC.Contributor" content="Susan Campbell"><meta name="DC.Contributor" content="Fran Borovecki"><meta name="DC.Contributor" content="Annemarie I Luik"><meta name="DC.Contributor" content="Man Li"><meta name="DC.Contributor" content="Jouke Jan Hottenga"><meta name="DC.Contributor" content="Jennifer E Huffman"><meta name="DC.Contributor" content="Mirjam CGN van den Hout"><meta name="DC.Contributor" content="Steven R Cummings"><meta name="DC.Contributor" content="Yurii S Aulchenko"><meta name="DC.Contributor" content="Philip R Gehrman"><meta name="DC.Contributor" content="André G Uitterlinden"><meta name="DC.Contributor" content="Heinz-Erich Wichmann"><meta name="DC.Contributor" content="Martina Müller-Nurasyid"><meta name="DC.Contributor" content="Rudolf SN Fehrmann"><meta name="DC.Contributor" content="Grant W Montgomery"><meta name="DC.Contributor" content="Albert Hofman"><meta name="DC.Contributor" content="Wen Hong Linda Kao"><meta name="DC.Contributor" content="Ben A Oostra"><meta name="DC.Contributor" content="Alan F Wright"><meta name="DC.Contributor" content="Jacqueline M Vink"><meta name="DC.Contributor" content="James F Wilson"><meta name="DC.Contributor" content="Peter P Pramstaller"><meta name="DC.Contributor" content="Andrew A Hicks"><meta name="DC.Contributor" content="Ozren Polasek"><meta name="DC.Contributor" content="Naresh M Punjabi"><meta name="DC.Contributor" content="Susan Redline"><meta name="DC.Contributor" content="Bruce M Psaty"><meta name="DC.Contributor" content="Andrew C Heath"><meta name="DC.Contributor" content="Martha Merrow"><meta name="DC.Contributor" content="Gregory J Tranah"><meta name="DC.Contributor" content="Daniel J Gottlieb"><meta name="DC.Contributor" content="Dorret I Boomsma"><meta name="DC.Contributor" content="Nicholas G Martin"><meta name="DC.Contributor" content="Igor Rudan"><meta name="DC.Contributor" content="Henning Tiemeier"><meta name="DC.Contributor" content="Wilfred FJ van IJcken"><meta name="DC.Contributor" content="Brenda W Penninx"><meta name="DC.Contributor" content="Andres Metspalu"><meta name="DC.Contributor" content="Thomas Meitinger"><meta name="DC.Contributor" content="Lude Franke"><meta name="DC.Contributor" content="Till Roenneberg"><meta name="DC.Contributor" content="Cornelia M van Duijn"><meta name="DC.Date" content="2016 Oct"><meta name="DC.Identifier" content="10.1038/ejhg.2016.31"><meta name="DC.Language" content="en"><meta property="og:title" content="Genetic variants in RBFOX3 are associated with sleep latency"><meta property="og:type" content="article"><meta property="og:description" content="Time to fall asleep (sleep latency) is a major determinant of sleep quality. Chronic, long sleep latency is a major characteristic of sleep-onset insomnia and/or delayed sleep phase syndrome. In this study we aimed to discover common polymorphisms that ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5027680/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5027680/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5027680/pdf/ejhg201631a.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A8E4ADD58B78100000000075F046B.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5027680%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/275767/">v.24(10); 2016 Oct</a></li><li class="accid">PMC5027680</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2016 Oct; 24(10): 1488–1495. </span></div><div><span class="fm-vol-iss-date">Published online 2016 May 4. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fejhg.2016.31" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ejhg.2016.31</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5027680</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27142678">27142678</a></div></div></div></div><h1 class="content-title">Genetic variants in <em>RBFOX3</em> are associated with sleep latency</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Amin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144202615040" co-class="co-affbox">Najaf Amin</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Allebrandt%20KV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144202599824" co-class="co-affbox">Karla V Allebrandt</a>,<sup>2</sup> <a href="/pubmed/?term=van%20der%20Spek%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144202597696" co-class="co-affbox">Ashley van der Spek</a>,<sup>1</sup> <a href="/pubmed/?term=M%26%23x000fc%3Bller-Myhsok%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144202595664" co-class="co-affbox">Bertram Müller-Myhsok</a>,<sup>3</sup> <a href="/pubmed/?term=Hek%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144202593584" co-class="co-affbox">Karin Hek</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Teder-Laving%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144202590624" co-class="co-affbox">Maris Teder-Laving</a>,<sup>6</sup> <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144202588496" co-class="co-affbox">Caroline Hayward</a>,<sup>7</sup> <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144202586368" co-class="co-affbox">Tõnu Esko</a>,<sup>6</sup> <a href="/pubmed/?term=van%20Mill%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144202584336" co-class="co-affbox">Josine G van Mill</a>,<sup>8</sup> <a href="/pubmed/?term=Mbarek%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199407632" co-class="co-affbox">Hamdi Mbarek</a>,<sup>9</sup> <a href="/pubmed/?term=Watson%20NF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199405552" co-class="co-affbox">Nathaniel F Watson</a>,<sup>10,</sup><sup>11</sup> <a href="/pubmed/?term=Melville%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199402592" co-class="co-affbox">Scott A Melville</a>,<sup>12</sup> <a href="/pubmed/?term=Del%20Greco%20FM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199400512" co-class="co-affbox">Fabiola M Del Greco</a>,<sup>13</sup> <a href="/pubmed/?term=Byrne%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199398384" co-class="co-affbox">Enda M Byrne</a>,<sup>14,</sup><sup>15</sup> <a href="/pubmed/?term=Oole%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199395424" co-class="co-affbox">Edwin Oole</a>,<sup>16</sup> <a href="/pubmed/?term=Kolcic%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199393296" co-class="co-affbox">Ivana Kolcic</a>,<sup>17</sup> <a href="/pubmed/?term=Chen%20Th%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199391168" co-class="co-affbox">Ting-hsu Chen</a>,<sup>18</sup> <a href="/pubmed/?term=Evans%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199389040" co-class="co-affbox">Daniel S Evans</a>,<sup>19</sup> <a href="/pubmed/?term=Coresh%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199386912" co-class="co-affbox">Josef Coresh</a>,<sup>20</sup> <a href="/pubmed/?term=Vogelzangs%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199384784" co-class="co-affbox">Nicole Vogelzangs</a>,<sup>8</sup> <a href="/pubmed/?term=Karjalainen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199382752" co-class="co-affbox">Juha Karjalainen</a>,<sup>21</sup> <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199380624" co-class="co-affbox">Gonneke Willemsen</a>,<sup>9</sup> <a href="/pubmed/?term=Gharib%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199378592" co-class="co-affbox">Sina A Gharib</a>,<sup>11,</sup><sup>22</sup> <a href="/pubmed/?term=Zgaga%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199375728" co-class="co-affbox">Lina Zgaga</a>,<sup>7</sup> <a href="/pubmed/?term=Mihailov%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199373696" co-class="co-affbox">Evelin Mihailov</a>,<sup>6</sup> <a href="/pubmed/?term=Stone%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199371664" co-class="co-affbox">Katie L Stone</a>,<sup>19</sup> <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199369632" co-class="co-affbox">Harry Campbell</a>,<sup>23</sup> <a href="/pubmed/?term=Brouwer%20RW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199367504" co-class="co-affbox">Rutger WW Brouwer</a>,<sup>16</sup> <a href="/pubmed/?term=Demirkan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199365472" co-class="co-affbox">Ayse Demirkan</a>,<sup>1</sup> <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199363440" co-class="co-affbox">Aaron Isaacs</a>,<sup>1</sup> <a href="/pubmed/?term=Dogas%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199361408" co-class="co-affbox">Zoran Dogas</a>,<sup>24</sup> <a href="/pubmed/?term=Marciante%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199359280" co-class="co-affbox">Kristin D Marciante</a>,<sup>25</sup> <a href="/pubmed/?term=Campbell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199357152" co-class="co-affbox">Susan Campbell</a>,<sup>7</sup> <a href="/pubmed/?term=Borovecki%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199355120" co-class="co-affbox">Fran Borovecki</a>,<sup>26</sup> <a href="/pubmed/?term=Luik%20AI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199352992" co-class="co-affbox">Annemarie I Luik</a>,<sup>4</sup> <a href="/pubmed/?term=Li%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199350960" co-class="co-affbox">Man Li</a>,<sup>27</sup> <a href="/pubmed/?term=Hottenga%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233810576" co-class="co-affbox">Jouke Jan Hottenga</a>,<sup>9</sup> <a href="/pubmed/?term=Huffman%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233808544" co-class="co-affbox">Jennifer E Huffman</a>,<sup>7</sup> <a href="/pubmed/?term=van%20den%20Hout%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233806512" co-class="co-affbox">Mirjam CGN van den Hout</a>,<sup>16</sup> <a href="/pubmed/?term=Cummings%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233804480" co-class="co-affbox">Steven R Cummings</a>,<sup>19</sup> <a href="/pubmed/?term=Aulchenko%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233802448" co-class="co-affbox">Yurii S Aulchenko</a>,<sup>1</sup> <a href="/pubmed/?term=Gehrman%20PR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233800416" co-class="co-affbox">Philip R Gehrman</a>,<sup>28</sup> <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233798288" co-class="co-affbox">André G Uitterlinden</a>,<sup>4,</sup><sup>29,</sup><sup>30</sup> <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233794544" co-class="co-affbox">Heinz-Erich Wichmann</a>,<sup>31,</sup><sup>32,</sup><sup>33</sup> <a href="/pubmed/?term=M%26%23x000fc%3Bller-Nurasyid%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233790704" co-class="co-affbox">Martina Müller-Nurasyid</a>,<sup>31,</sup><sup>34,</sup><sup>35,</sup><sup>36</sup> <a href="/pubmed/?term=Fehrmann%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199424704" co-class="co-affbox">Rudolf SN Fehrmann</a>,<sup>21</sup> <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199422720" co-class="co-affbox">Grant W Montgomery</a>,<sup>15</sup> <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199420736" co-class="co-affbox">Albert Hofman</a>,<sup>4,</sup><sup>37</sup> <a href="/pubmed/?term=Kao%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199418016" co-class="co-affbox">Wen Hong Linda Kao</a>,<sup>27</sup> <a href="/pubmed/?term=Oostra%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199416032" co-class="co-affbox">Ben A Oostra</a>,<sup>1</sup> <a href="/pubmed/?term=Wright%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199414048" co-class="co-affbox">Alan F Wright</a>,<sup>7</sup> <a href="/pubmed/?term=Vink%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199412064" co-class="co-affbox">Jacqueline M Vink</a>,<sup>9</sup> <a href="/pubmed/?term=Wilson%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144199410080" co-class="co-affbox">James F Wilson</a>,<sup>7,</sup><sup>23</sup> <a href="/pubmed/?term=Pramstaller%20PP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198894368" co-class="co-affbox">Peter P Pramstaller</a>,<sup>13,</sup><sup>38,</sup><sup>39</sup> <a href="/pubmed/?term=Hicks%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198890912" co-class="co-affbox">Andrew A Hicks</a>,<sup>13</sup> <a href="/pubmed/?term=Polasek%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198888928" co-class="co-affbox">Ozren Polasek</a>,<sup>17,</sup><sup>40</sup> <a href="/pubmed/?term=Punjabi%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198886208" co-class="co-affbox">Naresh M Punjabi</a>,<sup>41</sup> <a href="/pubmed/?term=Redline%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198884224" co-class="co-affbox">Susan Redline</a>,<sup>42</sup> <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198882240" co-class="co-affbox">Bruce M Psaty</a>,<sup>43,</sup><sup>44</sup> <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198879520" co-class="co-affbox">Andrew C Heath</a>,<sup>45</sup> <a href="/pubmed/?term=Merrow%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198877536" co-class="co-affbox">Martha Merrow</a>,<sup>2</sup> <a href="/pubmed/?term=Tranah%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198875552" co-class="co-affbox">Gregory J Tranah</a>,<sup>19</sup> <a href="/pubmed/?term=Gottlieb%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198873568" co-class="co-affbox">Daniel J Gottlieb</a>,<sup>42,</sup><sup>46</sup> <a href="/pubmed/?term=Boomsma%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144198870848" co-class="co-affbox">Dorret I Boomsma</a>,<sup>9</sup> <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233784336" co-class="co-affbox">Nicholas G Martin</a>,<sup>15</sup> <a href="/pubmed/?term=Rudan%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233782352" co-class="co-affbox">Igor Rudan</a>,<sup>23</sup> <a href="/pubmed/?term=Tiemeier%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233780368" co-class="co-affbox">Henning Tiemeier</a>,<sup>4,</sup><sup>5,</sup><sup>47</sup> <a href="/pubmed/?term=van%20IJcken%20WF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233776912" co-class="co-affbox">Wilfred FJ van IJcken</a>,<sup>16</sup> <a href="/pubmed/?term=Penninx%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233774928" co-class="co-affbox">Brenda W Penninx</a>,<sup>8</sup> <a href="/pubmed/?term=Metspalu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233772944" co-class="co-affbox">Andres Metspalu</a>,<sup>6</sup> <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233770960" co-class="co-affbox">Thomas Meitinger</a>,<sup>48,</sup><sup>49</sup> <a href="/pubmed/?term=Franke%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233768240" co-class="co-affbox">Lude Franke</a>,<sup>21</sup> <a href="/pubmed/?term=Roenneberg%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233766256" co-class="co-affbox">Till Roenneberg</a>,<sup>2,</sup><sup>51</sup> and  <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678" class="affpopup" co-rid="_co_idm140144233763408" co-class="co-affbox">Cornelia M van Duijn</a><sup>1,</sup><sup>30,</sup><sup>50,</sup><sup>51</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140144202615040"><h3 class="no_margin">Najaf Amin</h3><p><sup>1</sup>Unit of Genetic Epidemiology, Department of Epidemiology, Erasmus University Medical Centre, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Amin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Najaf Amin</a></div></div><div id="_co_idm140144202599824"><h3 class="no_margin">Karla V Allebrandt</h3><p><sup>2</sup>Institute of Medical Psychology, Ludwig-Maximilians-University, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Allebrandt%20KV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Karla V Allebrandt</a></div></div><div id="_co_idm140144202597696"><h3 class="no_margin">Ashley van der Spek</h3><p><sup>1</sup>Unit of Genetic Epidemiology, Department of Epidemiology, Erasmus University Medical Centre, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20der%20Spek%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Ashley van der Spek</a></div></div><div id="_co_idm140144202595664"><h3 class="no_margin">Bertram Müller-Myhsok</h3><p><sup>3</sup>Max-Planck-Institute of Psychiatry, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000fc%3Bller-Myhsok%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Bertram Müller-Myhsok</a></div></div><div id="_co_idm140144202593584"><h3 class="no_margin">Karin Hek</h3><p><sup>4</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><p><sup>5</sup>Department of Psychiatry, Erasmus Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Hek%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Karin Hek</a></div></div><div id="_co_idm140144202590624"><h3 class="no_margin">Maris Teder-Laving</h3><p><sup>6</sup>Estonian Genome Center, University of Tartu and Estonian Biocenter, Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Teder-Laving%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Maris Teder-Laving</a></div></div><div id="_co_idm140144202588496"><h3 class="no_margin">Caroline Hayward</h3><p><sup>7</sup>Medical Research Council, Human Genetics Unit, IGMM, University of Edinburgh, Edinburgh, Scotland</p><div>Find articles by <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Caroline Hayward</a></div></div><div id="_co_idm140144202586368"><h3 class="no_margin">Tõnu Esko</h3><p><sup>6</sup>Estonian Genome Center, University of Tartu and Estonian Biocenter, Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Tõnu Esko</a></div></div><div id="_co_idm140144202584336"><h3 class="no_margin">Josine G van Mill</h3><p><sup>8</sup>Department of Psychiatry, VU University Medical Center Amsterdam, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Mill%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Josine G van Mill</a></div></div><div id="_co_idm140144199407632"><h3 class="no_margin">Hamdi Mbarek</h3><p><sup>9</sup>Department of Biological Psychology, VU University, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Mbarek%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Hamdi Mbarek</a></div></div><div id="_co_idm140144199405552"><h3 class="no_margin">Nathaniel F Watson</h3><p><sup>10</sup>Department of Neurology, University of Washington, Seattle, WA, USA</p><p><sup>11</sup>University of Washington Medicine Sleep Center, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Watson%20NF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Nathaniel F Watson</a></div></div><div id="_co_idm140144199402592"><h3 class="no_margin">Scott A Melville</h3><p><sup>12</sup>Department of Medicine (Biomedical Genetics), Boston University School of Medicine, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Melville%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Scott A Melville</a></div></div><div id="_co_idm140144199400512"><h3 class="no_margin">Fabiola M Del Greco</h3><p><sup>13</sup>Center for Biomedicine, European Academy of Bolzano, Bolzano, Italy - Affiliated Institute of the University of Lübeck, Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=Del%20Greco%20FM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Fabiola M Del Greco</a></div></div><div id="_co_idm140144199398384"><h3 class="no_margin">Enda M Byrne</h3><p><sup>14</sup>Queensland Brain Institute, University of Queensland, Brisbane, QLD, Australia</p><p><sup>15</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Byrne%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Enda M Byrne</a></div></div><div id="_co_idm140144199395424"><h3 class="no_margin">Edwin Oole</h3><p><sup>16</sup>Center for Biomics, Erasmus Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Oole%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Edwin Oole</a></div></div><div id="_co_idm140144199393296"><h3 class="no_margin">Ivana Kolcic</h3><p><sup>17</sup>School of Medicine, University of Split, Split, Croatia</p><div>Find articles by <a href="/pubmed/?term=Kolcic%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Ivana Kolcic</a></div></div><div id="_co_idm140144199391168"><h3 class="no_margin">Ting-hsu Chen</h3><p><sup>18</sup>VA Boston Healthcare System, Boston University, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20Th%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Ting-hsu Chen</a></div></div><div id="_co_idm140144199389040"><h3 class="no_margin">Daniel S Evans</h3><p><sup>19</sup>California Pacific Medical Center Research Institute, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Evans%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Daniel S Evans</a></div></div><div id="_co_idm140144199386912"><h3 class="no_margin">Josef Coresh</h3><p><sup>20</sup>Departments of Epidemiology, Biostatistics, and Medicine, Johns Hopkins University, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Coresh%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Josef Coresh</a></div></div><div id="_co_idm140144199384784"><h3 class="no_margin">Nicole Vogelzangs</h3><p><sup>8</sup>Department of Psychiatry, VU University Medical Center Amsterdam, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Vogelzangs%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Nicole Vogelzangs</a></div></div><div id="_co_idm140144199382752"><h3 class="no_margin">Juha Karjalainen</h3><p><sup>21</sup>Department of Genetics, University Medical Center Groningen and University of Groningen, Groningen, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Karjalainen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Juha Karjalainen</a></div></div><div id="_co_idm140144199380624"><h3 class="no_margin">Gonneke Willemsen</h3><p><sup>9</sup>Department of Biological Psychology, VU University, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Gonneke Willemsen</a></div></div><div id="_co_idm140144199378592"><h3 class="no_margin">Sina A Gharib</h3><p><sup>11</sup>University of Washington Medicine Sleep Center, Seattle, WA, USA</p><p><sup>22</sup>Department of Medicine, Division of Pulmonary &amp; Critical Care Medicine, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Gharib%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Sina A Gharib</a></div></div><div id="_co_idm140144199375728"><h3 class="no_margin">Lina Zgaga</h3><p><sup>7</sup>Medical Research Council, Human Genetics Unit, IGMM, University of Edinburgh, Edinburgh, Scotland</p><div>Find articles by <a href="/pubmed/?term=Zgaga%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Lina Zgaga</a></div></div><div id="_co_idm140144199373696"><h3 class="no_margin">Evelin Mihailov</h3><p><sup>6</sup>Estonian Genome Center, University of Tartu and Estonian Biocenter, Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Mihailov%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Evelin Mihailov</a></div></div><div id="_co_idm140144199371664"><h3 class="no_margin">Katie L Stone</h3><p><sup>19</sup>California Pacific Medical Center Research Institute, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Stone%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Katie L Stone</a></div></div><div id="_co_idm140144199369632"><h3 class="no_margin">Harry Campbell</h3><p><sup>23</sup>Centre for Global Health Research, Usher Institute for Population Health Sciences and Informatics, University of Edinburgh, Edinburgh, Scotland</p><div>Find articles by <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Harry Campbell</a></div></div><div id="_co_idm140144199367504"><h3 class="no_margin">Rutger WW Brouwer</h3><p><sup>16</sup>Center for Biomics, Erasmus Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Brouwer%20RW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Rutger WW Brouwer</a></div></div><div id="_co_idm140144199365472"><h3 class="no_margin">Ayse Demirkan</h3><p><sup>1</sup>Unit of Genetic Epidemiology, Department of Epidemiology, Erasmus University Medical Centre, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Demirkan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Ayse Demirkan</a></div></div><div id="_co_idm140144199363440"><h3 class="no_margin">Aaron Isaacs</h3><p><sup>1</sup>Unit of Genetic Epidemiology, Department of Epidemiology, Erasmus University Medical Centre, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Aaron Isaacs</a></div></div><div id="_co_idm140144199361408"><h3 class="no_margin">Zoran Dogas</h3><p><sup>24</sup>Department of Neuroscience and Sleep Medicine Centre, University of Split School of Medicine, Split, Croatia</p><div>Find articles by <a href="/pubmed/?term=Dogas%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Zoran Dogas</a></div></div><div id="_co_idm140144199359280"><h3 class="no_margin">Kristin D Marciante</h3><p><sup>25</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Marciante%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Kristin D Marciante</a></div></div><div id="_co_idm140144199357152"><h3 class="no_margin">Susan Campbell</h3><p><sup>7</sup>Medical Research Council, Human Genetics Unit, IGMM, University of Edinburgh, Edinburgh, Scotland</p><div>Find articles by <a href="/pubmed/?term=Campbell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Susan Campbell</a></div></div><div id="_co_idm140144199355120"><h3 class="no_margin">Fran Borovecki</h3><p><sup>26</sup>Centre for Functional Genomics and Department of Neurology, Faculty of Medicine, University of Zagreb, Zagreb, Croatia</p><div>Find articles by <a href="/pubmed/?term=Borovecki%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Fran Borovecki</a></div></div><div id="_co_idm140144199352992"><h3 class="no_margin">Annemarie I Luik</h3><p><sup>4</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Luik%20AI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Annemarie I Luik</a></div></div><div id="_co_idm140144199350960"><h3 class="no_margin">Man Li</h3><p><sup>27</sup>Department of Epidemiology, Johns Hopkins University, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Man Li</a></div></div><div id="_co_idm140144233810576"><h3 class="no_margin">Jouke Jan Hottenga</h3><p><sup>9</sup>Department of Biological Psychology, VU University, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Hottenga%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Jouke Jan Hottenga</a></div></div><div id="_co_idm140144233808544"><h3 class="no_margin">Jennifer E Huffman</h3><p><sup>7</sup>Medical Research Council, Human Genetics Unit, IGMM, University of Edinburgh, Edinburgh, Scotland</p><div>Find articles by <a href="/pubmed/?term=Huffman%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Jennifer E Huffman</a></div></div><div id="_co_idm140144233806512"><h3 class="no_margin">Mirjam CGN van den Hout</h3><p><sup>16</sup>Center for Biomics, Erasmus Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20den%20Hout%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Mirjam CGN van den Hout</a></div></div><div id="_co_idm140144233804480"><h3 class="no_margin">Steven R Cummings</h3><p><sup>19</sup>California Pacific Medical Center Research Institute, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Cummings%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Steven R Cummings</a></div></div><div id="_co_idm140144233802448"><h3 class="no_margin">Yurii S Aulchenko</h3><p><sup>1</sup>Unit of Genetic Epidemiology, Department of Epidemiology, Erasmus University Medical Centre, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Aulchenko%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Yurii S Aulchenko</a></div></div><div id="_co_idm140144233800416"><h3 class="no_margin">Philip R Gehrman</h3><p><sup>28</sup>Department of Psychiatry, University of Pennsylvania, Philadelphia, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Gehrman%20PR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Philip R Gehrman</a></div></div><div id="_co_idm140144233798288"><h3 class="no_margin">André G Uitterlinden</h3><p><sup>4</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><p><sup>29</sup>Department of Internal Medicine, Erasmus University Medical Center, Rotterdam, The Netherlands</p><p><sup>30</sup>Netherlands Consortium for Healthy Ageing and National Genomics Initiative, Leiden, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">André G Uitterlinden</a></div></div><div id="_co_idm140144233794544"><h3 class="no_margin">Heinz-Erich Wichmann</h3><p><sup>31</sup>Institute of Epidemiology I, Helmholtz Zentrum Munich-German Research Center for Environmental Health, Neuherberg, Germany</p><p><sup>32</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-University and Klinikum Grosshadern, Munich, Germany</p><p><sup>33</sup>Institute of Medical Statistics and Epidemiology, Technical University Munich, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Heinz-Erich Wichmann</a></div></div><div id="_co_idm140144233790704"><h3 class="no_margin">Martina Müller-Nurasyid</h3><p><sup>31</sup>Institute of Epidemiology I, Helmholtz Zentrum Munich-German Research Center for Environmental Health, Neuherberg, Germany</p><p><sup>34</sup>Department of Medicine I, University Hospital Grosshadern, Ludwig-Maximilians-Universität, Munich, Germany</p><p><sup>35</sup>Institute of Medical Informatics, Biometry and Epidemiology, Chair of Genetic Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany</p><p><sup>36</sup>DZHK (German Centre for Cardiovascular Research), Partner Site Munich Heart Alliance, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000fc%3Bller-Nurasyid%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Martina Müller-Nurasyid</a></div></div><div id="_co_idm140144199424704"><h3 class="no_margin">Rudolf SN Fehrmann</h3><p><sup>21</sup>Department of Genetics, University Medical Center Groningen and University of Groningen, Groningen, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Fehrmann%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Rudolf SN Fehrmann</a></div></div><div id="_co_idm140144199422720"><h3 class="no_margin">Grant W Montgomery</h3><p><sup>15</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Grant W Montgomery</a></div></div><div id="_co_idm140144199420736"><h3 class="no_margin">Albert Hofman</h3><p><sup>4</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><p><sup>37</sup>Department of Epidemiology, Harvard T.H. Chan School of Public Health, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Albert Hofman</a></div></div><div id="_co_idm140144199418016"><h3 class="no_margin">Wen Hong Linda Kao</h3><p><sup>27</sup>Department of Epidemiology, Johns Hopkins University, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Kao%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Wen Hong Linda Kao</a></div></div><div id="_co_idm140144199416032"><h3 class="no_margin">Ben A Oostra</h3><p><sup>1</sup>Unit of Genetic Epidemiology, Department of Epidemiology, Erasmus University Medical Centre, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Oostra%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Ben A Oostra</a></div></div><div id="_co_idm140144199414048"><h3 class="no_margin">Alan F Wright</h3><p><sup>7</sup>Medical Research Council, Human Genetics Unit, IGMM, University of Edinburgh, Edinburgh, Scotland</p><div>Find articles by <a href="/pubmed/?term=Wright%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Alan F Wright</a></div></div><div id="_co_idm140144199412064"><h3 class="no_margin">Jacqueline M Vink</h3><p><sup>9</sup>Department of Biological Psychology, VU University, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Vink%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Jacqueline M Vink</a></div></div><div id="_co_idm140144199410080"><h3 class="no_margin">James F Wilson</h3><p><sup>7</sup>Medical Research Council, Human Genetics Unit, IGMM, University of Edinburgh, Edinburgh, Scotland</p><p><sup>23</sup>Centre for Global Health Research, Usher Institute for Population Health Sciences and Informatics, University of Edinburgh, Edinburgh, Scotland</p><div>Find articles by <a href="/pubmed/?term=Wilson%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">James F Wilson</a></div></div><div id="_co_idm140144198894368"><h3 class="no_margin">Peter P Pramstaller</h3><p><sup>13</sup>Center for Biomedicine, European Academy of Bolzano, Bolzano, Italy - Affiliated Institute of the University of Lübeck, Lübeck, Germany</p><p><sup>38</sup>Department of Neurology, General Central Hospital, Bolzano, Italy</p><p><sup>39</sup>Department of Neurology, University of Lübeck, Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=Pramstaller%20PP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Peter P Pramstaller</a></div></div><div id="_co_idm140144198890912"><h3 class="no_margin">Andrew A Hicks</h3><p><sup>13</sup>Center for Biomedicine, European Academy of Bolzano, Bolzano, Italy - Affiliated Institute of the University of Lübeck, Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=Hicks%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Andrew A Hicks</a></div></div><div id="_co_idm140144198888928"><h3 class="no_margin">Ozren Polasek</h3><p><sup>17</sup>School of Medicine, University of Split, Split, Croatia</p><p><sup>40</sup>Centre for Global Health, University of Split School of Medicine, Split, Croatia</p><div>Find articles by <a href="/pubmed/?term=Polasek%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Ozren Polasek</a></div></div><div id="_co_idm140144198886208"><h3 class="no_margin">Naresh M Punjabi</h3><p><sup>41</sup>Department of Pulmonary Medicine and Epidemiology, Johns Hopkins University, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Punjabi%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Naresh M Punjabi</a></div></div><div id="_co_idm140144198884224"><h3 class="no_margin">Susan Redline</h3><p><sup>42</sup>Department of Medicine, Brigham and Women's Hospital and Beth Deaconess Medical Center, Harvard Medical School, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Redline%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Susan Redline</a></div></div><div id="_co_idm140144198882240"><h3 class="no_margin">Bruce M Psaty</h3><p><sup>43</sup>Cardiovascular Health Research Unit, Departments of Medicine, Epidemiology and Health Services, University of Washington, Seattle, WA, USA</p><p><sup>44</sup>Group Health Research Institute, Group Health Cooperative, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Bruce M Psaty</a></div></div><div id="_co_idm140144198879520"><h3 class="no_margin">Andrew C Heath</h3><p><sup>45</sup>Department of Psychiatry, Washington University, St Louis, MO, USA</p><div>Find articles by <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Andrew C Heath</a></div></div><div id="_co_idm140144198877536"><h3 class="no_margin">Martha Merrow</h3><p><sup>2</sup>Institute of Medical Psychology, Ludwig-Maximilians-University, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Merrow%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Martha Merrow</a></div></div><div id="_co_idm140144198875552"><h3 class="no_margin">Gregory J Tranah</h3><p><sup>19</sup>California Pacific Medical Center Research Institute, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Tranah%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Gregory J Tranah</a></div></div><div id="_co_idm140144198873568"><h3 class="no_margin">Daniel J Gottlieb</h3><p><sup>42</sup>Department of Medicine, Brigham and Women's Hospital and Beth Deaconess Medical Center, Harvard Medical School, Boston, MA, USA</p><p><sup>46</sup>VA Boston Healthcare System, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Gottlieb%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Daniel J Gottlieb</a></div></div><div id="_co_idm140144198870848"><h3 class="no_margin">Dorret I Boomsma</h3><p><sup>9</sup>Department of Biological Psychology, VU University, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Boomsma%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Dorret I Boomsma</a></div></div><div id="_co_idm140144233784336"><h3 class="no_margin">Nicholas G Martin</h3><p><sup>15</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Nicholas G Martin</a></div></div><div id="_co_idm140144233782352"><h3 class="no_margin">Igor Rudan</h3><p><sup>23</sup>Centre for Global Health Research, Usher Institute for Population Health Sciences and Informatics, University of Edinburgh, Edinburgh, Scotland</p><div>Find articles by <a href="/pubmed/?term=Rudan%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Igor Rudan</a></div></div><div id="_co_idm140144233780368"><h3 class="no_margin">Henning Tiemeier</h3><p><sup>4</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</p><p><sup>5</sup>Department of Psychiatry, Erasmus Medical Center, Rotterdam, The Netherlands</p><p><sup>47</sup>Department of Child and Adolescent Psychiatry, Erasmus MC, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Tiemeier%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Henning Tiemeier</a></div></div><div id="_co_idm140144233776912"><h3 class="no_margin">Wilfred FJ van IJcken</h3><p><sup>16</sup>Center for Biomics, Erasmus Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20IJcken%20WF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Wilfred FJ van IJcken</a></div></div><div id="_co_idm140144233774928"><h3 class="no_margin">Brenda W Penninx</h3><p><sup>8</sup>Department of Psychiatry, VU University Medical Center Amsterdam, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Penninx%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Brenda W Penninx</a></div></div><div id="_co_idm140144233772944"><h3 class="no_margin">Andres Metspalu</h3><p><sup>6</sup>Estonian Genome Center, University of Tartu and Estonian Biocenter, Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Metspalu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Andres Metspalu</a></div></div><div id="_co_idm140144233770960"><h3 class="no_margin">Thomas Meitinger</h3><p><sup>48</sup>Institute of Human Genetics, Helmholtz Zentrum München, Neuherberg, Germany</p><p><sup>49</sup>Institute of Human Genetics, Techinsche Universität München, München, Germany</p><div>Find articles by <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Thomas Meitinger</a></div></div><div id="_co_idm140144233768240"><h3 class="no_margin">Lude Franke</h3><p><sup>21</sup>Department of Genetics, University Medical Center Groningen and University of Groningen, Groningen, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Franke%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Lude Franke</a></div></div><div id="_co_idm140144233766256"><h3 class="no_margin">Till Roenneberg</h3><p><sup>2</sup>Institute of Medical Psychology, Ludwig-Maximilians-University, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Roenneberg%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Till Roenneberg</a></div></div><div id="_co_idm140144233763408"><h3 class="no_margin">Cornelia M van Duijn</h3><p><sup>1</sup>Unit of Genetic Epidemiology, Department of Epidemiology, Erasmus University Medical Centre, Rotterdam, The Netherlands</p><p><sup>30</sup>Netherlands Consortium for Healthy Ageing and National Genomics Initiative, Leiden, The Netherlands</p><p><sup>50</sup>Centre for Medical Systems Biology,,  Leiden, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27142678">Cornelia M van Duijn</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140144269549264_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140144269549264_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140144269549264_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140144269549264_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Unit of Genetic Epidemiology, Department of Epidemiology, Erasmus University Medical Centre, Rotterdam, The Netherlands</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Institute of Medical Psychology, Ludwig-Maximilians-University, Munich, Germany</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Max-Planck-Institute of Psychiatry, Munich, Germany</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Department of Psychiatry, Erasmus Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Estonian Genome Center, University of Tartu and Estonian Biocenter, Tartu, Estonia</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Medical Research Council, Human Genetics Unit, IGMM, University of Edinburgh, Edinburgh, Scotland</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Department of Psychiatry, VU University Medical Center Amsterdam, Amsterdam, The Netherlands</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Department of Biological Psychology, VU University, Amsterdam, The Netherlands</div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>Department of Neurology, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>University of Washington Medicine Sleep Center, Seattle, WA, USA</div><div class="fm-affl" id="aff12" lang="en"><sup>12</sup>Department of Medicine (Biomedical Genetics), Boston University School of Medicine, Boston, MA, USA</div><div class="fm-affl" id="aff13" lang="en"><sup>13</sup>Center for Biomedicine, European Academy of Bolzano, Bolzano, Italy - Affiliated Institute of the University of Lübeck, Lübeck, Germany</div><div class="fm-affl" id="aff14" lang="en"><sup>14</sup>Queensland Brain Institute, University of Queensland, Brisbane, QLD, Australia</div><div class="fm-affl" id="aff15" lang="en"><sup>15</sup>Queensland Institute of Medical Research, Brisbane, QLD, Australia</div><div class="fm-affl" id="aff16" lang="en"><sup>16</sup>Center for Biomics, Erasmus Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="aff17" lang="en"><sup>17</sup>School of Medicine, University of Split, Split, Croatia</div><div class="fm-affl" id="aff18" lang="en"><sup>18</sup>VA Boston Healthcare System, Boston University, Boston, MA, USA</div><div class="fm-affl" id="aff19" lang="en"><sup>19</sup>California Pacific Medical Center Research Institute, San Francisco, CA, USA</div><div class="fm-affl" id="aff20" lang="en"><sup>20</sup>Departments of Epidemiology, Biostatistics, and Medicine, Johns Hopkins University, Baltimore, MD, USA</div><div class="fm-affl" id="aff21" lang="en"><sup>21</sup>Department of Genetics, University Medical Center Groningen and University of Groningen, Groningen, The Netherlands</div><div class="fm-affl" id="aff22" lang="en"><sup>22</sup>Department of Medicine, Division of Pulmonary &amp; Critical Care Medicine, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="aff23" lang="en"><sup>23</sup>Centre for Global Health Research, Usher Institute for Population Health Sciences and Informatics, University of Edinburgh, Edinburgh, Scotland</div><div class="fm-affl" id="aff24" lang="en"><sup>24</sup>Department of Neuroscience and Sleep Medicine Centre, University of Split School of Medicine, Split, Croatia</div><div class="fm-affl" id="aff25" lang="en"><sup>25</sup>Cardiovascular Health Research Unit, Department of Medicine, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="aff26" lang="en"><sup>26</sup>Centre for Functional Genomics and Department of Neurology, Faculty of Medicine, University of Zagreb, Zagreb, Croatia</div><div class="fm-affl" id="aff27" lang="en"><sup>27</sup>Department of Epidemiology, Johns Hopkins University, Baltimore, MD, USA</div><div class="fm-affl" id="aff28" lang="en"><sup>28</sup>Department of Psychiatry, University of Pennsylvania, Philadelphia, PA, USA</div><div class="fm-affl" id="aff29" lang="en"><sup>29</sup>Department of Internal Medicine, Erasmus University Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="aff30" lang="en"><sup>30</sup>Netherlands Consortium for Healthy Ageing and National Genomics Initiative, Leiden, The Netherlands</div><div class="fm-affl" id="aff31" lang="en"><sup>31</sup>Institute of Epidemiology I, Helmholtz Zentrum Munich-German Research Center for Environmental Health, Neuherberg, Germany</div><div class="fm-affl" id="aff32" lang="en"><sup>32</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-University and Klinikum Grosshadern, Munich, Germany</div><div class="fm-affl" id="aff33" lang="en"><sup>33</sup>Institute of Medical Statistics and Epidemiology, Technical University Munich, Munich, Germany</div><div class="fm-affl" id="aff34" lang="en"><sup>34</sup>Department of Medicine I, University Hospital Grosshadern, Ludwig-Maximilians-Universität, Munich, Germany</div><div class="fm-affl" id="aff35" lang="en"><sup>35</sup>Institute of Medical Informatics, Biometry and Epidemiology, Chair of Genetic Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany</div><div class="fm-affl" id="aff36" lang="en"><sup>36</sup>DZHK (German Centre for Cardiovascular Research), Partner Site Munich Heart Alliance, Munich, Germany</div><div class="fm-affl" id="aff37" lang="en"><sup>37</sup>Department of Epidemiology, Harvard T.H. Chan School of Public Health, Boston, MA, USA</div><div class="fm-affl" id="aff38" lang="en"><sup>38</sup>Department of Neurology, General Central Hospital, Bolzano, Italy</div><div class="fm-affl" id="aff39" lang="en"><sup>39</sup>Department of Neurology, University of Lübeck, Lübeck, Germany</div><div class="fm-affl" id="aff40" lang="en"><sup>40</sup>Centre for Global Health, University of Split School of Medicine, Split, Croatia</div><div class="fm-affl" id="aff41" lang="en"><sup>41</sup>Department of Pulmonary Medicine and Epidemiology, Johns Hopkins University, Baltimore, MD, USA</div><div class="fm-affl" id="aff42" lang="en"><sup>42</sup>Department of Medicine, Brigham and Women's Hospital and Beth Deaconess Medical Center, Harvard Medical School, Boston, MA, USA</div><div class="fm-affl" id="aff43" lang="en"><sup>43</sup>Cardiovascular Health Research Unit, Departments of Medicine, Epidemiology and Health Services, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="aff44" lang="en"><sup>44</sup>Group Health Research Institute, Group Health Cooperative, Seattle, WA, USA</div><div class="fm-affl" id="aff45" lang="en"><sup>45</sup>Department of Psychiatry, Washington University, St Louis, MO, USA</div><div class="fm-affl" id="aff46" lang="en"><sup>46</sup>VA Boston Healthcare System, Boston, MA, USA</div><div class="fm-affl" id="aff47" lang="en"><sup>47</sup>Department of Child and Adolescent Psychiatry, Erasmus MC, Rotterdam, The Netherlands</div><div class="fm-affl" id="aff48" lang="en"><sup>48</sup>Institute of Human Genetics, Helmholtz Zentrum München, Neuherberg, Germany</div><div class="fm-affl" id="aff49" lang="en"><sup>49</sup>Institute of Human Genetics, Techinsche Universität München, München, Germany</div><div class="fm-affl" id="aff50" lang="en"><sup>50</sup>Centre for Medical Systems Biology,,  Leiden, The Netherlands</div><div id="caf1"><sup>*</sup>Unit of Genetic Epidemiology, Department of Epidemiology, Erasmus University Medical Centre, Wytemaweg 80, CN, Rotterdam 3015, The Netherlands. Tel: +31 10 7038125; E-mail: <a href="mailto:dev@null" data-email="ln.cmsumsare@nima.n" class="oemail">ln.cmsumsare@nima.n</a></div><div id="note1"><sup>51</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140144269549264_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2015 Aug 18; Revised 2016 Jan 13; Accepted 2016 Feb 1.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140144269549264_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2016 Macmillan Publishers Limited</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5027680/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="15" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 1.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x1.doc" data-ga-action="click_feat_suppl">ejhg201631x1.doc</a><span style="color:gray"> (156K)</span></div><div class="small guid">GUID:&nbsp;E34BF595-F8EF-472F-98A3-FBDA3CBD62B7</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 2.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x2.doc" data-ga-action="click_feat_suppl">ejhg201631x2.doc</a><span style="color:gray"> (114K)</span></div><div class="small guid">GUID:&nbsp;DCFC43F2-6818-42C7-A68D-EA6DA32C9412</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 3.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x3.doc" data-ga-action="click_feat_suppl">ejhg201631x3.doc</a><span style="color:gray"> (70K)</span></div><div class="small guid">GUID:&nbsp;247C2A9E-D54B-475E-8DC5-B0404C478CC4</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 4.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x4.doc" data-ga-action="click_feat_suppl">ejhg201631x4.doc</a><span style="color:gray"> (38K)</span></div><div class="small guid">GUID:&nbsp;B6A57B83-7D75-476A-8BF0-62F0845D820C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 5.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x5.doc" data-ga-action="click_feat_suppl">ejhg201631x5.doc</a><span style="color:gray"> (85K)</span></div><div class="small guid">GUID:&nbsp;E94A0432-1DDF-4412-BADD-DAD2857E6E94</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 6.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x6.doc" data-ga-action="click_feat_suppl">ejhg201631x6.doc</a><span style="color:gray"> (71K)</span></div><div class="small guid">GUID:&nbsp;DF990E63-3351-4743-9B62-5C490D025E2B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 7.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x7.doc" data-ga-action="click_feat_suppl">ejhg201631x7.doc</a><span style="color:gray"> (64K)</span></div><div class="small guid">GUID:&nbsp;73CEE947-A51E-4BE8-A7E3-B11209BA5607</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 8.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x8.doc" data-ga-action="click_feat_suppl">ejhg201631x8.doc</a><span style="color:gray"> (54K)</span></div><div class="small guid">GUID:&nbsp;077B9CBE-96E6-4135-9766-3C852C53E24F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 9.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x9.doc" data-ga-action="click_feat_suppl">ejhg201631x9.doc</a><span style="color:gray"> (176K)</span></div><div class="small guid">GUID:&nbsp;A231436E-EFE9-4947-8C0E-F2F31A3703F7</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Text.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x10.doc" data-ga-action="click_feat_suppl">ejhg201631x10.doc</a><span style="color:gray"> (33K)</span></div><div class="small guid">GUID:&nbsp;D564EA31-F4F5-4F00-90E7-EFB595A55F7A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure 1.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x11.pdf" data-ga-action="click_feat_suppl">ejhg201631x11.pdf</a><span style="color:gray"> (181K)</span></div><div class="small guid">GUID:&nbsp;C789444C-ACFF-4A6C-8660-A0F895FD5FA9</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure 2.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x12.pdf" data-ga-action="click_feat_suppl">ejhg201631x12.pdf</a><span style="color:gray"> (198K)</span></div><div class="small guid">GUID:&nbsp;AE7561AC-F120-4FCC-8EEF-822C16D50672</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure 3.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x13.pdf" data-ga-action="click_feat_suppl">ejhg201631x13.pdf</a><span style="color:gray"> (1.0M)</span></div><div class="small guid">GUID:&nbsp;A00AED5F-0887-4DFD-8BA9-822C710098BA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure 4.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x14.pdf" data-ga-action="click_feat_suppl">ejhg201631x14.pdf</a><span style="color:gray"> (66K)</span></div><div class="small guid">GUID:&nbsp;47082FFE-41B6-4EC3-A6FF-96BC1005C956</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figure 5.</strong></div><div><a href="/pmc/articles/PMC5027680/bin/ejhg201631x15.pdf" data-ga-action="click_feat_suppl">ejhg201631x15.pdf</a><span style="color:gray"> (59K)</span></div><div class="small guid">GUID:&nbsp;45E7C88F-6DCC-4285-9FAB-E95F3A8B4218</div></div></dd></dl></div><div id="idm140144232009408" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140144232009408title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Time to fall asleep (sleep latency) is a major determinant of sleep quality. Chronic, long sleep latency is a major characteristic of sleep-onset insomnia and/or delayed sleep phase syndrome. In this study we aimed to discover common polymorphisms that contribute to the genetics of sleep latency. We performed a meta-analysis of genome-wide association studies (GWAS) including 2 572 737 single nucleotide polymorphisms (SNPs) established in seven European cohorts including 4242 individuals. We found a cluster of three highly correlated variants (rs9900428, rs9907432 and rs7211029) in the RNA-binding protein fox-1 homolog 3 gene (<em>RBFOX3</em>) associated with sleep latency (<em>P</em>-values=5.77 × 10<sup>−08</sup>, 6.59 × 10<sup>−</sup><sup>08</sup> and 9.17 × 10<sup>−</sup><sup>08</sup>). These SNPs were replicated in up to 12 independent populations including 30 377 individuals (<em>P</em>-values=1.5 × 10<sup>−</sup><sup>02</sup>, 7.0 × 10<sup>−</sup><sup>03</sup> and 2.5 × 10<sup>−</sup><sup>03</sup>; combined meta-analysis <em>P</em>-values=5.5 × 10<sup>−07</sup>, 5.4 × 10<sup>−07</sup> and 1.0 × 10<sup>−07</sup>). A functional prediction of <em>RBFOX3</em> based on co-expression with other genes shows that this gene is predominantly expressed in brain (<em>P</em>-value=1.4 × 10<sup>−316</sup>) and the central nervous system (<em>P</em>-value=7.5 × 10<sup>−</sup><sup>321</sup>). The predicted function of <em>RBFOX3</em> based on co-expression analysis with other genes shows that this gene is significantly involved in the release cycle of neurotransmitters including gamma-aminobutyric acid and various monoamines (<em>P</em>-values&lt;2.9 × 10<sup>−11</sup>) that are crucial in triggering the onset of sleep. To conclude, in this first large-scale GWAS of sleep latency we report a novel association of variants in <em>RBFOX3</em> gene. Further, a functional prediction of <em>RBFOX3</em> supports the involvement of <em>RBFOX3</em> with sleep latency.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p3" class="p p-first">Sleep occurs during defined periods within the 24 h cycle.<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198694">1</a></sup> Its timing and duration is considered to be regulated by two processes; (i) a circadian process (ie, being awake during the day and asleep during night) and (ii) a homeostatic process that represents the sleep pressure accumulating during wakefulness, ie, the longer one is awake, the greater sleep pressure and the greater will be the duration of sleep when permitted.<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198701">2</a></sup> Sleep latency, ie, the duration it takes to fall asleep, is a measure of sleep quality computed as the time interval between ‘lights out' until the onset of sleep. Photic information from the retina is projected to the suprachiasmatic nucleus (SCN) via the retino-hypothalamic tract. In humans, the SCN is considered the pacemaker for the timing of daily sleep–wake behavior and consists of ~10 000 neurons located on both sides of the midline above the optic chiasma, ~3 cm behind the eyes.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198705">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198704">4</a></sup> At night, the SCN signals the release of melatonin – a hormone produced by the pineal gland that promotes sleep. Sleep latency may be assessed using self-reports, actigraphy or with polysomnography. Normal sleepers are known to provide estimates of sleep latency that correlate well with polysomnographic measures.<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198680">5</a>, <a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198713">6</a></sup> Previous studies show that sleep latency is largely independent of an individual's chronotype.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198669">7</a></sup></p><p id="__p4">Inter-individual differences in sleep latency are caused by both genetic and non-genetic factors. The latter include gender,<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198711">8</a>, <a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198698">9</a></sup> age,<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198659">8</a></sup> consumption of stimulants,<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198656">10</a></sup> dietary intake,<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198655">11</a></sup> sedentary life<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198662">12</a></sup> and illnesses, such as depression.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198649">13</a></sup> Persistent increased sleep latency is a major characteristic of delayed sleep phase syndrome<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198673">14</a></sup> and/or sleep-onset insomnia.<sup><a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198678">15</a>, <a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198686">16</a>, <a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198692">17</a>, <a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198696">18</a></sup> Prolonged sleep latency may shorten sleep duration and may lead to a wide range of problems including irritability, cognitive impairment, depression and loss of productivity as well as accident rates due to sleepiness. Increased sleep latency has also been associated with poor academic performance in children and adolescents.<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198667">19</a></sup> Although very long sleep latency can lead to sleep deprivation, very short sleep latency can reflect sleep deprivation (ie, ‘sleep debt' due to insufficient sleep normally shortens sleep latency). Unusually short sleep latency also may indicate disorders of excessive sleepiness (eg, narcolepsy). When giving individuals the opportunity to sleep during the day, sleep latency is used as an objective measure of daytime sleepiness in the diagnosis of sleep disorders.</p><p id="__p5" class="p p-last">Heritability of sleep latency is estimated to be ~17–44%.<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198695">20</a>, <a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198693">21</a>, <a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198703">22</a></sup> Although much is known about the environmental factors that can prolong sleep latency, we know little about the genetic influences on sleep latency. In this study, we performed a meta-analysis of unpublished genome-wide association studies (GWAS) on sleep latency in order to elucidate genetic associations with this trait.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Materials and methods</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Study populations (stage 1/discovery cohorts)</h3><p id="__p6" class="p p-first-last">We meta-analyzed data from seven GWAS comprising of 4242 subjects with European ancestry (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S1 and S2</a>). The participating cohorts in the gene discovery phase included the Erasmus Rucphen Family (ERF), Estonian Genome Center (EGP/EGCUT), CROATIA-Korcula, the Micro-isolates in South Tyrol Study (MICROS), Cooperative health research in the Region of Augsburg (KORA), the Netherlands Study of Depression and Anxiety (NESDA) and the Orkney Complex Disease Study (ORCADES) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S1 and S2</a>). A detailed description of the included studies is provided in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary text</a>. All studies in the discovery cohort used the Munich Chronotype Questionnaire (MCTQ)<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198681">23</a></sup> to assess sleep latency. Subjects were asked to report how long they take to fall asleep on free and workdays. Free days sleep latency was used in the analyses of those cohorts, where a person's sleep pattern is not influenced by professional duties<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198668">24</a></sup> (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1</a>). The question from the MCTQ used to assess sleep latency (in minutes): ‘I need … minutes to fall asleep'. Sleep duration was calculated by subtracting sleep onset from sleep end. Mid-sleep was calculated as the midpoint between sleep onset and waking on free days.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198653">25</a></sup> The quality control was centralized and the inclusion criteria were: (i) no use of an alarm clock on free days; (ii) no shift-work during the last 3 months; and (iii) no use of sleep medication (benzodiazepines and other pharmacological agents that influence sleep; see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S3</a>). Informed consent was obtained from all study participants and an appropriate local committee approved the study protocols.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Study populations (stage 2/replication cohorts)</h3><p id="__p7" class="p p-first-last">The replication stage included 12 independent cohorts (see <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary text</a>; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1</a>). The descriptive statistics are provided in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S1</a>. Inclusion criteria were: (1) sleep assessment with any available tool and (2) No use of sleep medication (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S1</a>). Five replication cohorts (MrOS, RS-I, RS-II, RS-III and SOF) used the Pittsburg Sleep Quality Index to assess sleep (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>), which uses only a single measure for sleep assessment and does not distinguish between sleep on free and working days.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Genotyping and imputation</h3><p id="__p8" class="p p-first-last">Both discovery and replication cohorts were genotyped on a variety of platforms (Affymetrix 250 K, Illumina 318 K, Illumina 370 K, Illumina 610 k; Perlegen 600 K; Affymetrix 1000 K). Quality control was done in each group separately. The overall criteria were to exclude individuals with low call rate, excess heterozygosity and gender mismatch, and exclude variants that were out of Hardy–Weinberg equilibrium, had low minor allele frequency (MAF) or low call rate (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>). In EGCUT1 study where the genome-wide data was not available, the two most significant single nucleotide polymorphisms (SNPs) for which a TaqMan assay was available were genotyped (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S4</a>). Imputations of non-genotyped SNPs in the discovery cohorts were carried out within each study using either MACH<sup><a href="#bib26" rid="bib26" class=" bibr popnode">26</a>, <a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198712">27</a></sup> or IMPUTE,<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198690">28</a>, <a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198709">29</a></sup> and HapMap CEU v21a or v22 as reference (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>). Genetic imputations in the replication cohorts were performed using MACH, IMPUTE, minimac or BimBam (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>). Of the three SNPS rs9907432 was genotyped in most replication cohorts, whereas rs9900428 was imputed in all replication cohorts (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S4</a>). The data are available in the GWAS Central database, under the accession number HGVST1836 (<a href="http://www.gwascentral.org/study/HGVST1836" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gwascentral.org/study/HGVST1836</a>).</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Methylation and expression</h3><p id="__p9" class="p p-first">The Infinium Human Methylation 450 array (Illumina, San Diego, CA, USA) was used to quantify genome-wide DNA methylation for 748 samples from the Rotterdam study (RS-III) covering 485 577 CpG sites in the genome. Bisulfite conversion followed by amplification, hybridization and imaging were performed according to standard protocols. Illumina GenomeStudio software was used to estimate <em>β</em> score from intensity. Quality-control (QC) steps included removal of signal probes with a detection of <em>P</em>-value&gt;0.01; and exclusion of probes with low intensity. Samples with a call rate&lt;99% were removed from the dataset. SWAN package for R software was used to normalize remaining CpG sites and correct for batch effects.</p><p id="__p10" class="p p-last">RNA from the same samples was obtained (PAXgene) and hybridized to Illumina HumanHT-12 arrays. Raw probe intensity was obtained using BeadStudio. Gene expression data were quantile-normalized to the median distribution, and subsequently log2 transformed. The probe and sample means were centered to zero. Probes that had a detection <em>P</em>-value&lt;0.05 in &gt;10% of the samples were removed from the analysis. The final analysis included 21 328 probes, which were significantly expressed in blood.</p></div><div id="__sec7" class="sec sec-last"><h3 id="__sec7title">Statistical analysis</h3><div id="__sec8" class="sec sec-first"><p></p><h4 id="__sec8title" class="inline">Heritability analysis </h4><p id="__p11" class="p p-first-last">Heritability analysis of sleep latency was performed in the family-based ERF cohort using Sequential Oligogenic Linkage Analysis Routines (SOLAR).<sup><a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198684">30</a></sup> SOLAR uses likelihood ratio tests to evaluate heritability by comparing a purely polygenic model with a sporadic model in the case of testing heritability. The ERF cohort forms one large family, which consists of &gt;23 000 members spanning over 23 generations. Since this uniquely large family is too large to be analyzed with SOLAR, we cut it into smaller pedigrees (three to five generations) using the Pedcut software<sup><a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198685">31</a></sup> for the heritability analysis. A natural log transformation was applied to the trait before estimating the heritability. The heritability was estimated with the ‘polygenic screen' option and using age and sex as covariates in the model. The estimate was then compared with the heritability estimate derived from the polygenic analysis of GenABEL that uses genome-wide genotype data instead of the pedigree to estimate heritability.<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198665">32</a></sup></p></div><div id="__sec9" class="sec"><p></p><h4 id="__sec9title" class="inline">Genome-wide association analysis </h4><p id="__p12" class="p p-first">Individual GWAS was performed using linear regression (under additive model), natural log of sleep latency as the dependent variable, SNP allele dosage as predictor and age and sex as covariates. CROATIA-Korcula and ORCADES additionally used first three principal components as covariates in the association model. The association analyses were conducted in ProbABEL<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198683">32</a>, <a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198708">33</a>, <a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198676">34</a></sup> or SNPTEST.<sup><a href="#bib35" rid="bib35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198675">35</a></sup> For cohorts with related individuals (ERF, MICROS, CROATIA-Korcula, ORCADES), a linear mixed model in ProbABEL using the ‘mmscore' option was used to account for familial relationships. The mmscore option performs the score test that uses the inverse variance-covariance object estimated from the genetic data and returned from the ‘polygenic' function of GenABEL<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198682">32</a></sup> to correct for familial relationships. This is a slightly modified FASTA method developed by Abecasis <em>et al.</em><sup><a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198660">36</a></sup></p><p id="__p13">As all three SNPs show no heterogeneity across the cohorts (<em>P</em>-values&gt;0.7) (<a href="/pmc/articles/PMC5027680/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>), a fixed effects meta-analysis was conducted using the inverse variance weighted method as implemented in METAL (<a href="http://www.sph.umich.edu/csg/abecasis/metal/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/metal/</a>). All SNPs that had a MAF&lt;0.01 and low imputation quality (Rsq/proper_info&lt;0.3) were dropped from the meta-analysis. Genomic control correction was also applied to all cohorts prior to the meta-analysis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Top SNPs from the genome-wide association analysis for sleep latency</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140144201215136" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>SNP</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>position(B36)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Allele</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Gene</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Maf</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Chrom</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Cohort</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>EGP</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>ERF</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>KORA</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>KORCULA</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MICROS</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>NESDA</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>ORCADES</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Meta-analysis</em></th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">N</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>933</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>740</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>548</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>610</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>693</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>540</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>206</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>4270</em></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9900428</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">74651323</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G&gt;A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">RBFOX3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.20–0.33</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">17</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>β</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.106</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.085</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.158</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.133</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.082</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.122</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.094</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">SE</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.051</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.024</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.057</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.067</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.055</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.064</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.104</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.017</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>P</em>-value</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.037</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.00036</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0057</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.045</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.137</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.632</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.239</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5.77 × 10<sup>−08</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Rsq</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.97</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.90</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.90</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.97</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.96</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.85</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.99</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>P</em><sub>HET</sub></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.823</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9907432</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">74651967</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G&gt;A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">RBFOX3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.20–0.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">17</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>β</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.106</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.083</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.153</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.127</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.084</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.0248</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">–0.1216</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.092</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">SE</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0501</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0234</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0551</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.066</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0543</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0616</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1031</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.017</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>P</em>-value</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0354</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.00038</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.00568</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.053</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1206</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.6874</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.2382</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.59 × 10<sup>−</sup><sup>08</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Rsq</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.94</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.95</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.99</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.90</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>P</em><sub>HET</sub></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.814</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7211029</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">74652903</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C&gt;T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">RBFOX3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.21–0.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">17</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>β</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.107</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.083</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.1499</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.127</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.083</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.0119</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.1215</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.091</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">SE</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.050</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0234</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0538</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.066</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0544</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0596</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1034</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.017</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>P</em>-value</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.035</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.00038</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0054</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.054</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1249</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.8416</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.2397</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.17 × 10<sup>−08</sup></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Rsq</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.97</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.93</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.98</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.98</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.96</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.99</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>P</em><sub>HET</sub></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.739</td></tr></tbody></table></div><div id="largeobj_idm140144201215136" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5027680/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1-fn1"><p id="__p48" class="p p-first-last">Abbreviations: <em>β,</em> effect of the genetic variant; SE, standard error of the effect estimate; <em>P</em><sub>HET</sub>, <em>P-value</em> for heterogeneity; Rsq, imputation quality.</p></div></div></div><p id="__p14" class="p p-last">For the SNPs that approached genome-wide significance (<em>P</em>-value&lt;5 × 10<sup>−08</sup>) we performed replication in up to 12 independent cohorts using the same model as in the initial GWAS. In the replication cohorts, among the family-based studies, ERF_ext used SOLAR, FHS used LMEKIN package of R software (<a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>), CROATIA-Split used the ‘mmscore' option in ProbABEL, whereas QIMR used MERLIN to account for family structure. Other population-based cohorts used SPSS, PLINK (<a href="http://pngu.mgh.harvard.edu/purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/purcell/plink/</a>)<sup><a href="#bib37" rid="bib37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198658">37</a></sup> or R software to perform association analysis (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>). Meta-analysis was performed using square-root of the sample size as weights.<sup><a href="#bib38" rid="bib38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198714">38</a></sup></p></div><div id="__sec10" class="sec"><p></p><h4 id="__sec10title" class="inline">Methylome-wide association analysis </h4><p id="__p15" class="p p-first-last">Top SNPs from the meta-analysis of GWAS were tested for association with genome-wide methylation (CpG) sites by performing linear regression analyses of methylation sites on each of the three SNPs while adjusting for age, sex, technical covariates including batch effects and blood cell counts (granulocytes, lymphocytes, monocytes, erythrocytes and platelets) in R software.</p></div><div id="__sec11" class="sec sec-last"><p></p><h4 id="__sec11title" class="inline">Gene functional prediction and network analysis </h4><p id="__p16" class="p p-first">Prediction of gene function can be conducted using a guilt-by-association approach: eg, if there are 100 genes that are known to be involved in apoptosis, identification of a gene that is strongly co-expressed with these 100 genes suggests that this gene is likely to be involved in apoptosis as well. As such co-expression data can be used to predict likely functions for genes. However, important to realize is that some phenomena exert very strong transcriptomic effects and therefore will overshadow more subtle effects. In order to be able to identify such subtle relationships as well, we conducted a principal component analysis (PCA) on an unprecedented scale.<sup><a href="#bib39" rid="bib39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198670">39</a></sup> We collected gene expression data for three different species (homo sapiens, mus musculus and rattus norvegicus) from the Gene Expression Omnibus. We confined analyses to four different Affymetrix expression platforms (Affymetrix Human Genome U133A Array, Affymetrix Human Genome U133 Plus 2.0 Array, Affymetrix Mouse Genome 430 2.0 Array and Affymetrix Rat Genome 230 2.0 Array). For each of these platforms we downloaded the raw CEL files (20 108, 43 278, 18 639 and 6124 arrays, respectively), and used RMA for normalization. We could run RMA on all samples at once for the 20 108 Human Genome U133A Array, 18 639 Mouse Genome 430 2.0 Array and 6123 Rat Genome 230 2.0 Array. For the 43 278 Human Genome U133 Plus 2.0 Array samples we ran RMA in eight batches due to its size, by randomly assigning the samples to one of these batches. We subsequently conducted QC on the data. We first removed duplicate samples, and then conducted a PCA on the sample correlation matrix. The first principal component (PCqc) on such a matrix describes nearly always a constant pattern (dominating the data), which explains ~80–90% of the total variance.<sup><a href="#bib40" rid="bib40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198702">40</a>, <a href="#bib41" rid="bib41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198657">41</a></sup> This pattern can be regarded as probe-specific variance, independent of the biological sample hybridized to the array. The correlation of each individual microarray with this PCqc can be used to detect outliers, as arrays of lesser quality will have a lower correlation with the PCqc. We removed samples that had a correlation <em>R</em>&lt;0.75. After QC in total 77 840 different samples remained for downstream analysis (54 736 human samples, 17 081 mouse samples, 6 023 rat samples). Although this QCed data set can be well used for the aforementioned guilt-by-association co-expression analysis, we reasoned that the presence of profound effects on many genes will make it difficult to identify the more subtle relationships that exist between genes. Therefore, we conducted a PCA on the probe correlation matrix, resulting in the identification of in total 2206 robustly estimated principal components (377 for Human Genome U133A, 777 for Human Genome U133 Plus 2.0, 677 for Mouse Genome 430 2.0 and 375 for Rat Genome 230 2.0) by requiring a Cronbach's alpha &gt;0.70 for each individual principal component. Jointly these components explain between 79% and 90% of the variance in the data per Affymetrix expression platform, and many of these are well conserved across the three species.</p><p id="__p17">Subsequent Gene Set Enrichment Analysis revealed that each of these 2206 components are significantly enriched (false discovery rate&lt;0.05) for at least one GO term, KEGG, BioCarta or Reactome pathway, indicating that these components are describing biologically relevant but often diverse phenomena. While per species the very first components describe profound effects on expression (ie, many enriched pathways and GO terms), the other components are potentially equally biologically relevant, as each of the components describe certain biological phenomena. We therefore used the individual components and integrated the different platforms and species by collapsing the probe identifiers to human Ensembl genes and used orthology information from Ensembl for the mouse and rat platform, resulting in a harmonized matrix of 19 997 unique Ensembl genes × 2206 principal components.</p><p id="__p18" class="p p-last">We subsequently predicted the most likely Gene Ontology (GO) biological process using the following strategy: (i) we first ascertained each individual GO term and assessed per PC whether the genes that were explicitly annotated with this GO term showed a significant difference from the genes that were not annotated with this GO term using a <em>t</em>-test; (ii) we converted the resulting <em>P</em>-value into an 'enrichment' <em>Z</em>-score (to ensure normality); and (iii) we subsequently investigated RNA-binding protein fox-1 homolog 3 gene (<em>RBFOX3</em>) and correlated the 2206 PC eigenvector coefficients of <em>RBFOX3</em> with each GO term by taking the 2206 'enrichment' <em>Z</em>-scores as the expression profile for that GO term. A significant positive correlation means <em>RBFOX3</em> has an expression profile that is comparable to the GO term. We have visualized this method at <a href="http://www.genenetwork.nl/genenetwork" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.genenetwork.nl/genenetwork</a> (click on ‘Method'). In order to correct for multiple testing, we permuted Ensembl gene identifiers: using permuted data we redid the ‘enrichment' <em>Z</em>-score calculation and investigated how strong <em>RBFOX3</em> correlated with permuted pathway. We repeated this analysis 100 times, allowing us to determine which of these predictions were significant (controlling to the false discovery rate of 0.05). We used the same procedure to predict in which BioCarta and Reactome pathways <em>RBFOX3</em> is involved.</p></div></div></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Results</h2><p id="__p19" class="p p-first">A basic description of the study populations is given in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S1</a>. The heritability of sleep latency using the pedigree data of the ERF study was estimated to be 0.18 (SE=0.112, <em>P</em>-value=0.05) (see methods section), which is consistent with earlier findings of heritability for this trait <sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198699">21</a></sup> and also comparable to the heritability estimate (<em>h</em><sup>2</sup>=0.20) derived from the genome-wide genotype data (see Materials and methods section for details) of the same cohort. The quantile–quantile plot of the meta-analysis shows no inflation of the chi-square statistic as the genomic control inflation factor (<em>λ</em>) is 1.01 (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S2</a>). The meta-analysis of GWAS produced a cluster of three borderline genome-wide significant SNPs (<a href="/pmc/articles/PMC5027680/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>; <a href="/pmc/articles/PMC5027680/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S3</a>) on chromosome 17q25. The highest-ranking SNP rs9900428 hg18.chr17:g.74651323G&gt;A) (<em>P</em>-value=5.7 × 10<sup>−08</sup>) had a MAF of 0.20–0.33, with the imputation quality ranging from 0.85 to 0.99 across all cohorts. The effect size (<em>β</em>=−0.094) and the direction of the risk were consistent across all cohorts (<a href="/pmc/articles/PMC5027680/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). The other two SNPs rs9907432 (hg18.chr17:g.74651967:G&gt;A) (<em>P</em>-value=6.5 × 10<sup>−</sup><sup>08</sup>) and rs7211029 (hg18.chr17:g.74652903C&gt;T) (<em>P</em>-value=9.1 × 10<sup>−08</sup>) were in linkage disequilibrium (<em>r</em><sup>2</sup>=0.95) with rs9900428 (<a href="/pmc/articles/PMC5027680/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>,<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S4</a>). <a href="/pmc/articles/PMC5027680/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a> shows the mean sleep latency per genotype category for all the discovery cohorts for the three SNPs unadjusted for age and gender. Compared with the homozygote carriers of the reference allele (pooled average sleep latency=16.4 min), the homozygote carriers (pooled average sleep latency=13.5 min) of the minor allele are estimated to have a sleep latency of on average approximately 3 min less (a reduction of 18%) and the heterozygote carriers (pooled average sleep latency=15.5 min) ~1 min less (a reduction of ~6%) (<a href="/pmc/articles/PMC5027680/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC5027680/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140144201227648" class="figure"><a href="/pmc/articles/PMC5027680/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5027680_ejhg201631f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg201631f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5027680/bin/ejhg201631f1.jpg"></a></div><div id="largeobj_idm140144201227648" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5027680/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC5027680/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p45">Regional association plot of sleep latency for the region 17q25. The plot was constructed using Locus Zoom (<a href="http://csg.sph.umich.edu/locuszoom/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/locuszoom/</a>). The most significant SNP is depicted as a diamond and other SNPs in the region are depicted by circles. Various colors represent the extent of linkage disequilibrium with most significant SNP. The <em>X</em> axis gives the position in mega bases and the <em>Y</em> axis shows the negative logarithm of the <em>P</em>-values from the meta-analysis.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC5027680/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140144201221152" class="figure"><a href="/pmc/articles/PMC5027680/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5027680_ejhg201631f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg201631f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5027680/bin/ejhg201631f2.jpg"></a></div><div id="largeobj_idm140144201221152" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5027680/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC5027680/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p46">Means per genotype category for all the discovery cohorts for the three most significant SNPs. Black bars represent the mean sleep latency for the homozygous carriers of the effect allele. <em>Y</em> axis depicts the sleep latency time in minutes. Panels <strong>a</strong>, <strong>b</strong> and <strong>c</strong> represent the effects of the SNPs in individual cohorts. Panel <strong>d</strong> represents the pooled average sleep latency per genotype category across all cohorts for the three SNPs (<em>X</em> axis). Data used to generate this figure were not adjusted for age and sex.</p></div></div></div><p id="__p20">We attempted to replicate these three SNPs in up to 12 independent cohorts (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S2 and S4</a>). SNP rs9900428 was available <em>in silico</em> in 11 cohorts, whereas rs9907432 and rs7211029 were available in all 12 cohorts. The replication analyses showed consistency in the direction of the effect across most replication cohorts (<a href="/pmc/articles/PMC5027680/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure 3</span></a>). The meta-analysis of the replication cohorts yielded significant evidence of association of rs9900428 (<em>P</em>-value=1.5 × 10<sup>−</sup><sup>02</sup>), rs9907432 (<em>P</em>-value=7.1 × 10<sup>−03</sup>) and rs7211029 (<em>P</em>-value=2.5 × 10<sup>−03</sup>) with sleep latency (<a href="/pmc/articles/PMC5027680/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). The three SNPs are intronic to the <em>RBFOX3</em> (also known as <em>HRNBP3</em>; <a href="/pmc/articles/PMC5027680/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure 1</span></a>) and lie in the region with active regulatory elements (ENCODE) containing the H3K27ac mark and methylation marks. H3K27ac is an important enhancer mark that can distinguish between active and poised enhancer elements. Such enhancer elements are known to affect the expression of proximal genes and cluster near the genes they regulate.<sup><a href="#bib42" rid="bib42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198687">42</a></sup> We investigated the three SNPs further using the methylome data of RS. The three SNPs were methylome-wide significantly associated (<em>P-</em>value=8.1 × 10<sup>−9</sup>, FDR=0.004) with the CpG site: cg16185152 in the RBFOX3 gene. We attempted to investigate the effect of methylation on gene expression, there was, however, only one probe for the RBFOX3 gene in the HumanHT-12_V4_Illumina 450K RNA expression array, which was removed in the quality control.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC5027680/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140144201217792" class="figure"><a href="/pmc/articles/PMC5027680/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5027680_ejhg201631f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ejhg201631f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5027680/bin/ejhg201631f3.jpg"></a></div><div id="largeobj_idm140144201217792" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5027680/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC5027680/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p47">Forest plot for the three SNPs at chromosome 17q25. On the left, the populations including the populations in the GWAS and the replication phase. The boxes represent the precision and horizontal lines representing the confidence intervals. Pooled estimate is the effect estimate from the meta-analysis of all cohorts. The horizontal axis represents the scale of effects.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Results of the replication of the top SNPs</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140144268553120" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Cohort</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>ARIC</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>CHS</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>EGCUT1</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>EGCUT2</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>ERF_ext</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>FHS</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MROS</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>NTR</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>QIMR</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>RS</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>SOF</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>SPLIT</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Meta-analysis replication</em></th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>SNP</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>N</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>3583</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>1533</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>5925</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>3540</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>143</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>2192</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>1849</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>1795</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>2280</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>5641</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>1480</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>416</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>30377</em></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9900428</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>β</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.006</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.011</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">NA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.02</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.128</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.109</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.012</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.014</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.034</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.015</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.006</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.095</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.026</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">SE</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.021</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.044</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">NA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.027</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.139</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.068</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.026</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.014</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.031</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.019</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.035</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.079</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.010</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P-</em>value</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.775</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.802</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">NA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.461</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.357</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.109</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.644</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.317</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.273</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.408</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.864</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.231</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.015</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em><sub>HET</sub></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.919</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9907432</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>β</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.007</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.009</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.027</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.022</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.09</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.109</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.012</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.012</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.034</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.016</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.005</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.102</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.026</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">SE</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.021</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.044</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.019</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.027</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.133</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.069</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.026</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.014</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.031</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0186</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.035</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.078</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0097</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em>-value</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.739</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.831</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.408</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.499</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.114</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.644</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.391</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.273</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.384</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.886</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.187</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.007</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em><sub>HET</sub></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.949</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7211029</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>β</em></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.007</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.009</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.028</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.023</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.223</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.109</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.012</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−.024</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.034</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.016</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.005</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.103</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−0.030</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">SE</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.021</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.044</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.019</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.027</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.139</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.069</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.026</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.016</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.031</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.019</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.035</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.078</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0098</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em>-value</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.739</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.831</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.142</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.398</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.109</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.116</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.644</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.133</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.272</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.384</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.886</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.186</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0025</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em><sub>HET</sub></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.792</td></tr></tbody></table></div><div id="largeobj_idm140144268553120" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5027680/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t2-fn1"><p id="__p49" class="p p-first-last">Abbreviations: <em>β</em>, effect of the genetic variant; SE, standard error of the effect estimate; <em>P</em><sub>HET</sub>, <em>P-</em>value for heterogeneity.</p></div></div></div><p id="__p21">A gene network analysis of <em>RBFOX3</em> using gene network tool (<a href="http://genenetwork.nl:8080/GeneNetwork/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genenetwork.nl:8080/GeneNetwork/</a>) shows strongest co-expression with the hippocalcin gene <em>HPCA</em> followed by <em>SNCB, CABP1, JPH3, CPLX2, GABRA6, GABRD, NRXN3, RBFOX1, RTN4R, CNTN2</em> and <em>WSCD2</em> among others (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure S5</a>). A functional prediction of the gene showed involvement in the biological processes of synaptic functioning, membrane depolarization, gamma-aminobutyric acid (GABA) signaling and nervous system development (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S5 and S6</a>) and in dopamine, serotonin and glutamate neurotransmitters release cycle (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S7</a>). <em>RBFOX3</em> is expressed most strongly in various parts of the brain (brain, <em>P</em>-value=1.4 × 10<sup>−316</sup>; central nervous system, <em>P</em>-value=7.6 × 10<sup>−</sup><sup>321</sup>; cerebral cortex, <em>P</em>-value=1.0 × 10<sup>−</sup><sup>174</sup>) including the hypothalamus (<em>P</em>-value=9.7 × 10<sup>−</sup><sup>10</sup>, AUC=0.96) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S8</a>), the locale of the SCN and a central part of the circadian clock.</p><p id="__p22" class="p p-last">None of the three SNPs showed strong association with sleep duration or mid-sleep, which is an indicator of the chronotype of an individual, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S9</a>.</p></div><div id="__sec13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec13title">Discussion</h2><p id="__p23" class="p p-first">In this first large-scale GWAS of sleep latency, we report the association of novel variants located in the gene <em>RBFOX3</em> with sleep latency. Our gene discovery phase consisted of a sample of 4242 individuals from seven European populations, where sleep latency was assessed according to a common protocol. With this sample we discovered a cluster of three borderline genome-wide significant SNPs that were intronic to the gene <em>RBFOX3</em>. The replication of the three SNPs in up to 30 377 individuals from 12 independent cohorts showed significant association of the three variants and consistency in the direction of the effect estimates across most cohorts. A functional prediction of <em>RBFOX3</em> based on a gene network analysis suggests significant involvement in the release cycle of neurotransmitters including GABA and various monoamines that are core to the human circadian clock, thus supporting the involvement of <em>RBFOX3</em> with sleep latency.</p><p id="__p24">The <em>RBFOX3</em> gene (also called <em>Fox-3, Hrnbp3, Neun</em>) is located on the long arm of chromosome 17 (17q25). It belongs to the Fox-1 family of genes and shows high homology to <em>RBFOX1</em> (also called <em>Fox-1, A2bp1, Hrnbp1</em>) and <em>RBFOX2</em> (also called <em>Fox-2, Rbm9, Hrnbp2</em>). <em>RBFOX3</em> is a relatively new member of this family and was recently identified to code for neuronal nuclei (NeuN) protein.<sup><a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198691">43</a></sup> The Fox proteins are a highly conserved family of tissue-specific splicing regulators.<sup><a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198666">44</a></sup> Although <em>RBFOX1</em> is expressed in neurons, muscles and heart, and <em>RBFOX2</em> in ovary, whole embryo, neurons and muscles,<sup><a href="#bib45" rid="bib45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198710">45</a>, <a href="#bib46" rid="bib46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198664">46</a></sup> the expression of <em>RBFOX3</em> was detected exclusively in the post-mitotic regions of embryonic mouse brain.<sup><a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198697">44</a>, <a href="#bib47" rid="bib47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198654">47</a></sup>
<em>RBFOX3</em> is believed to have a role in neuron-specific alternative splicing.<sup><a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198671">43</a></sup> Alternative splicing occurs most frequently in the brain,<sup><a href="#bib48" rid="bib48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198688">48</a>, <a href="#bib49" rid="bib49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198652">49</a></sup> presumably to generate large numbers of neuronal cell types and to support their diverse functions.<sup><a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198706">43</a></sup> A recent analysis of rodent SCN (the biological clock) anatomy using antibody against NeuN protein shows that <em>RBFOX3</em> has a unique distribution which is limited to a particular sector of the SCN.<sup><a href="#bib50" rid="bib50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198707">50</a></sup> Mutations in the <em>FOX-1</em> genes lead to severe neuro-developmental phenotypes exhibiting mental retardation, epilepsy and autism spectrum disorder.<sup><a href="#bib51" rid="bib51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198679">51</a>, <a href="#bib52" rid="bib52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198700">52</a>, <a href="#bib53" rid="bib53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198689">53</a>, <a href="#bib54" rid="bib54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198674">54</a></sup> Further, a very recent study on patients with developmental delay detected a translocation disrupting the intron 2 of the <em>RBFOX3</em> gene.<sup><a href="#bib55" rid="bib55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198651">55</a></sup> Interestingly, this patient additionally had sleeping difficulties.</p><p id="__p25">Gene-network analysis of <em>RBFOX3</em> showed strong co-expression with genes involved in calcium channel activity and GABA signaling. GABA-ergic sleep neurons of the ventro-lateral pre-optic nucleus are activated by the circadian clock and adenosine, which progressively accumulates in the brain during waking. In turn these sleep-active neurons begin to inhibit the wake-active neurons via the neurotransmitter GABA.<sup><a href="#bib56" rid="bib56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198672">56</a></sup>
<em>GABA-A</em> receptors are the site of action of a number of important pharmacologic agents including barbiturates, benzodiazepines (sleep inducing drugs) and ethanol.<sup><a href="#bib57" rid="bib57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198650">57</a>, <a href="#bib58" rid="bib58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198663">58</a></sup> Polymorphisms in <em>GABA-A</em> receptors have been associated with insomnia.<sup><a href="#bib59" rid="bib59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198677">59</a></sup> Functional prediction based on gene network analysis shows significant involvement of <em>RBFOX3</em> in the release cycle of various neurotransmitters including dopamine, serotonin and glutamate, <em>GABA-A</em> receptor activation and Ras-activation upon Ca<sup>2+</sup> influx through the NMDA receptor. Photic information is communicated to the molecular clockworks by release of glutamate from retino-hypothalamic nerve terminals and stimulation of glutamate receptors on SCN neurons.<sup><a href="#bib60" rid="bib60" class=" bibr popnode">60</a></sup> Glutamate stimulation is followed by intra-cellular increases of cyclic adenosine monophosphate and Ca<sup>2+</sup> and activation of a Ras-dependent signal cascade in the circadian clockwork. Non-photic signaling to the core subdivision of the SCN is conveyed through two major pathways including GABA-containing neurons derived from the thalamus and serotonin or 5-hydroxytryptamine-containing neurons derived from the midbrain. Serotonergic input to the SCN shifts the timing of the clock.<sup><a href="#bib60" rid="bib60" class=" bibr popnode">60</a></sup></p><p id="__p26" class="p p-last">Our study shows strong evidence of association of sleep latency with <em>RBFOX3</em>. The fact that the association signal was stronger in the discovery sample compared with the replication sample, even though the replication sample was sixfold larger, may be explained by the differences in phenotyping as well as by different cohort characteristics (eg, age). Notably, all discovery cohorts were European, whereas the replication cohorts were drawn additionally from North America and Asia; it is possible that socio-cultural factors may have influenced sleep latency or how sleep latency was reported across cohorts. For instance, the phenotyping and quality control for the discovery cohorts was synchronized and centralized; all cohorts were assessed with MCTQ, which, unlike other such instruments, assesses information separately for free days and working days. Our GWAS was based on sleep latency on free days, as sleep latency on workdays is heavily influenced by daily professional activities. Unfortunately, this distinction was not available for most of the replication cohorts. Moreover, exclusions in the GWAS discovery cohorts were based not only on sleep medication use but also on other drugs that are known to influence sleep. Sleep medication or for that matter any medication that has a sleep-inducing effect reduces sleep latency,<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_558198661">24</a></sup> thereby introducing a potential bias on genetic studies of sleep latency. We also removed shift-workers and those using alarm clocks from the analysis in the discovery phase, which was not done in most replication cohorts. All of these factors likely affected the results in the replication phase. Nevertheless, despite a small effect size, the direction of association signal was consistent in most replication cohorts indicating the robustness of our finding. Furthermore, gene functional prediction and network analysis support the association of <em>RBFOX3</em> variants with sleep latency. The predicted functioning of <em>RBFOX3</em>, including neurotransmitters' release cycle and GABA-receptor activation strongly implicates a chronobiological explanation. However, further tests of association with various sleep disorders and functional analyses will provide a better insight into the relationship between <em>RBFOX3</em> and sleep.</p></div><div id="idm140144201861712" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140144201861712title">Acknowledgments</h2><div class="sec"><p id="__p27">We thank all the participants and staff of all the studies for their co-operation and contribution. We are grateful to all participants and their relatives, general practitioners and neurologists for their contributions and to P Veraart for her help in genealogy, Jeannette Vergeer for the supervision of the laboratory work and P Snijders for his help in data collection for the ERF study. We thank Pascal Arp, Mila Jhamai, Marijn Verkerk, Lizbeth Herrera and Marjolein Peters for their help in creating the GWAS database, and Karol Estrada and Maksim V Struchalin for their support in creation and analysis of imputed data for the Rotterdam Study. We are grateful to the study participants, the staff from the Rotterdam Study and the participating general practitioners and pharmacists. We thank Dr Fernando Rivadeneira, Dr Tobias A Knoch, Anis Abuseiris, Luc V de Zeeuw and Rob de Graaf for their help in creating GRIMP, and BigGRID, MediGRID and Services@MediGRID/D-Grid. We acknowledge the invaluable contributions of the recruitment team from the Croatian Centre for Global Health, University of Split, the administrative teams in Croatia and Edinburgh and the people of Split. Our work was supported by the FP6 programme EUCLOCK, the Dutch Science Foundation (the NWO), the Hersenstichting Nederland, the Rosalind Franklin Fellowships of the University of Groningen, targeted Financing from the Estonian Government, the European Union through the European Regional Development Fund in the frame of Centre of Excellence in Genomics, FP7 Projects ECOGENE, BBMRI, ENGAGE and OPENGENE, the Geestkracht programme of the Dutch Scientific Organization (ZON-MW) and matching funds from participating universities and mental health care organizations, the Genetic Association Information Network (GAIN) of the Foundation for the US National Institutes of Health, the EUROSPAN (European Special Populations Research Network) through the European Commission FP6 STRP grant, the Chief Scientist Office of the Scottish Government, the Royal Society, the Medical Research Council, Erasmus University, Erasmus MC, the Centre for Medical Systems Biology (CMSB1 and CMSB2) and the Netherlands Genomics Initiative (NGI) and ALBAN. We acknowledge the High Performance Computing Center of University of Tartu, for space and facilities to conduct part of the EGCUT analysis. We also acknowledge the Netherlands Organization for Health Research and Development (ZonMw), the Research Institute for Diseases in the Elderly (RIDE), The Ministry of Education, Culture and Science, the Ministry of Health, Welfare and Sports of the Netherlands, the European Commission (DG XII) and the Municipality of Rotterdam. This study was supported by grants from the Netherlands Organization for Scientific Research (NWO-VENI grant 916.10.135 to LF) and a Horizon Breakthrough grant from the Netherlands Genomics Initiative (grant 92519031 to LF). The research leading to these results has received funding from the European Community's Health Seventh Framework Programme (FP7/2007–2013) under grant agreement 259867. Study-specific acknowledgments and the ethical statement are provided in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary text</a>.</p></div></div><div id="idm140144201857040" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140144201857040title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140144201856912"></p><p id="__p28" class="p p-first-last"><a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> accompanies this paper on European Journal of Human Genetics website (http://www.nature.com/ejhg)</p><p></p><p class="fn sec" id="idm140144201855440"></p><p id="__p29" class="p p-first-last">Dr Najaf Amin is supported by the Netherlands Brain Foundation (project number F2013(1)-28). Dr Gregory J Tranah was supported by NIA grant R01AG030474. Dr Henning Tiemeier was supported by the Vidi Grant of ZonMw (the Netherlands Organization for Health Research and Development, 2009-017.106.370). All other authors report no biomedical financial interests or potential conflicts of interest.</p><p></p></div></div><div id="sup1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sup1title">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="xob1"><h4>Supplementary Table 1</h4><div class="sup-box half_rhythm" id="idm140144198674288"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x1.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(156K, doc)</sup></div></div><div class="sec suppmat" id="xob2"><h4>Supplementary Table 2</h4><div class="sup-box half_rhythm" id="idm140144198671472"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x2.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(114K, doc)</sup></div></div><div class="sec suppmat" id="xob3"><h4>Supplementary Table 3</h4><div class="sup-box half_rhythm" id="idm140144198668608"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x3.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(70K, doc)</sup></div></div><div class="sec suppmat" id="xob4"><h4>Supplementary Table 4</h4><div class="sup-box half_rhythm" id="idm140144198665696"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x4.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(38K, doc)</sup></div></div><div class="sec suppmat" id="xob5"><h4>Supplementary Table 5</h4><div class="sup-box half_rhythm" id="idm140144198662736"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x5.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(85K, doc)</sup></div></div><div class="sec suppmat" id="xob6"><h4>Supplementary Table 6</h4><div class="sup-box half_rhythm" id="idm140144198659776"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x6.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(71K, doc)</sup></div></div><div class="sec suppmat" id="xob7"><h4>Supplementary Table 7</h4><div class="sup-box half_rhythm" id="idm140144198656864"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x7.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(64K, doc)</sup></div></div><div class="sec suppmat" id="xob8"><h4>Supplementary Table 8</h4><div class="sup-box half_rhythm" id="idm140144198653952"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x8.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(54K, doc)</sup></div></div><div class="sec suppmat" id="xob9"><h4>Supplementary Table 9</h4><div class="sup-box half_rhythm" id="idm140144198651040"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x9.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(176K, doc)</sup></div></div><div class="sec suppmat" id="xob10"><h4>Supplementary Text</h4><div class="sup-box half_rhythm" id="idm140144198648080"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x10.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(33K, doc)</sup></div></div><div class="sec suppmat" id="xob11"><h4>Supplementary Figure 1</h4><div class="sup-box half_rhythm" id="idm140144198645168"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x11.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(181K, pdf)</sup></div></div><div class="sec suppmat" id="xob12"><h4>Supplementary Figure 2</h4><div class="sup-box half_rhythm" id="idm140144198642208"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x12.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(198K, pdf)</sup></div></div><div class="sec suppmat" id="xob13"><h4>Supplementary Figure 3</h4><div class="sup-box half_rhythm" id="idm140144198639248"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x13.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.0M, pdf)</sup></div></div><div class="sec suppmat" id="xob14"><h4>Supplementary Figure 4</h4><div class="sup-box half_rhythm" id="idm140144198636288"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x14.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(66K, pdf)</sup></div></div><div class="sec suppmat" id="xob15"><h4>Supplementary Figure 5</h4><div class="sup-box half_rhythm" id="idm140144198633328"><a href="/pmc/articles/PMC5027680/bin/ejhg201631x15.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(59K, pdf)</sup></div></div></div><div id="idm140144201854160" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140144201854160title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="bib1"><span class="mixed-citation">Krauchi K, Wirz-Justice A: <span class="ref-title">Circadian clues to sleep onset mechanisms</span>. <span class="ref-journal">Neuropsychopharmacology</span>
2001; <span class="ref-vol">25</span>: S92–S96. [<a href="/pubmed/11682282" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&amp;title=Circadian+clues+to+sleep+onset+mechanisms&amp;volume=25&amp;publication_year=2001&amp;pages=S92&amp;pmid=11682282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib2"><span class="mixed-citation">Cirelli C: <span class="ref-title">The genetic and molecular regulation of sleep: from fruit flies to humans</span>. <span class="ref-journal">Nat Rev Neurosci</span>
2009; <span class="ref-vol">10</span>: 549–560. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2767184/">PMC free article</a>]</span> [<a href="/pubmed/19617891" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Neurosci&amp;title=The+genetic+and+molecular+regulation+of+sleep:+from+fruit+flies+to+humans&amp;volume=10&amp;publication_year=2009&amp;pages=549&amp;pmid=19617891&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib3"><span class="mixed-citation">Hastings M: <span class="ref-title">The brain, circadian rhythms, and clock genes</span>. <span class="ref-journal">BMJ</span>
1998; <span class="ref-vol">317</span>: 1704–1707. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1114487/">PMC free article</a>]</span> [<a href="/pubmed/9857134" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=The+brain,+circadian+rhythms,+and+clock+genes&amp;volume=317&amp;publication_year=1998&amp;pages=1704&amp;pmid=9857134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib4"><span class="mixed-citation">Hastings MH: <span class="ref-title">Central clocking</span>. <span class="ref-journal">Trends Neurosci</span>
1997; <span class="ref-vol">20</span>: 459–464. [<a href="/pubmed/9347613" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Neurosci&amp;title=Central+clocking&amp;volume=20&amp;publication_year=1997&amp;pages=459&amp;pmid=9347613&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib5"><span class="mixed-citation">Baekeland F, Hoy P: <span class="ref-title">Reported vs recorded sleep characteristics</span>. <span class="ref-journal">Arch Gen Psychiatry</span>
1971; <span class="ref-vol">24</span>: 548–551. [<a href="/pubmed/4325303" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Reported+vs+recorded+sleep+characteristics&amp;volume=24&amp;publication_year=1971&amp;pages=548&amp;pmid=4325303&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib6"><span class="mixed-citation">Hoch CC, Reynolds CF 3rd, Kupfer DJ, Berman SR, Houck PR, Stack JA: <span class="ref-title">Empirical note: self-report versus recorded sleep in healthy seniors</span>. <span class="ref-journal">Psychophysiology</span>
1987; <span class="ref-vol">24</span>: 293–299. [<a href="/pubmed/3602285" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychophysiology&amp;title=Empirical+note:+self-report+versus+recorded+sleep+in+healthy+seniors&amp;volume=24&amp;publication_year=1987&amp;pages=293&amp;pmid=3602285&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib7"><span class="mixed-citation">Lehnkering H, Siegmund R: <span class="ref-title">Influence of chronotype, season, and sex of subject on sleep behavior of young adults</span>. <span class="ref-journal">Chronobiol Int</span>
2007; <span class="ref-vol">24</span>: 875–888. [<a href="/pubmed/17994343" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chronobiol+Int&amp;title=Influence+of+chronotype,+season,+and+sex+of+subject+on+sleep+behavior+of+young+adults&amp;volume=24&amp;publication_year=2007&amp;pages=875&amp;pmid=17994343&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib8"><span class="mixed-citation">Geisler P, Tracik F, Cronlein T et al: <span class="ref-title">The influence of age and sex on sleep latency in the MSLT-30—a normative study</span>. <span class="ref-journal">Sleep</span>
2006; <span class="ref-vol">29</span>: 687–692. [<a href="/pubmed/16774159" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sleep&amp;title=The+influence+of+age+and+sex+on+sleep+latency+in+the+MSLT-30—a+normative+study&amp;volume=29&amp;publication_year=2006&amp;pages=687&amp;pmid=16774159&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib9"><span class="mixed-citation">Krishnan V, Collop NA: <span class="ref-title">Gender differences in sleep disorders</span>. <span class="ref-journal">Curr Opin Pulm Med</span>
2006; <span class="ref-vol">12</span>: 383–389. [<a href="/pubmed/17053485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Pulm+Med&amp;title=Gender+differences+in+sleep+disorders&amp;volume=12&amp;publication_year=2006&amp;pages=383&amp;pmid=17053485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib10"><span class="mixed-citation">Bonnet MH, Webb WB: <span class="ref-title">The return to sleep</span>. <span class="ref-journal">Biol Psychol</span>
1979; <span class="ref-vol">8</span>: 225–233. [<a href="/pubmed/497315" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychol&amp;title=The+return+to+sleep&amp;volume=8&amp;publication_year=1979&amp;pages=225&amp;pmid=497315&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib11"><span class="mixed-citation">Afaghi A, O'Connor H, Chow CM: <span class="ref-title">High-glycemic-index carbohydrate meals shorten sleep onset</span>. <span class="ref-journal">Am J Clin Nutr</span>
2007; <span class="ref-vol">85</span>: 426–430. [<a href="/pubmed/17284739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Clin+Nutr&amp;title=High-glycemic-index+carbohydrate+meals+shorten+sleep+onset&amp;volume=85&amp;publication_year=2007&amp;pages=426&amp;pmid=17284739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib12"><span class="mixed-citation">Nixon GM, Thompson JM, Han DY et al: <span class="ref-title">Falling asleep: the determinants of sleep latency</span>. <span class="ref-journal">Arch Dis Child</span>
2009; <span class="ref-vol">94</span>: 686–689. [<a href="/pubmed/19633062" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Dis+Child&amp;title=Falling+asleep:+the+determinants+of+sleep+latency&amp;volume=94&amp;publication_year=2009&amp;pages=686&amp;pmid=19633062&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib13"><span class="mixed-citation">Gupta R, Dahiya S, Bhatia MS: <span class="ref-title">Effect of depression on sleep: qualitative or quantitative</span>? <span class="ref-journal">Indian J Psychiatry</span>
2009; <span class="ref-vol">51</span>: 117–121. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2755175/">PMC free article</a>]</span> [<a href="/pubmed/19823630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Indian+J+Psychiatry&amp;title=Effect+of+depression+on+sleep:+qualitative+or+quantitative&amp;volume=51&amp;publication_year=2009&amp;pages=117&amp;pmid=19823630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib14"><span class="mixed-citation">Blader JC, Koplewicz HS, Abikoff H, Foley C: <span class="ref-title">Sleep problems of elementary school children. A community survey</span>. <span class="ref-journal">Arch Pediatr Adolesc Med</span>
1997; <span class="ref-vol">151</span>: 473–480. [<a href="/pubmed/9158439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Pediatr+Adolesc+Med&amp;title=Sleep+problems+of+elementary+school+children.+A+community+survey&amp;volume=151&amp;publication_year=1997&amp;pages=473&amp;pmid=9158439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib15"><span class="mixed-citation">Eddy M, Walbroehl GS: <span class="ref-title">Insomnia</span>. <span class="ref-journal">Am Fam Physician</span>
1999; <span class="ref-vol">59</span>: 1911–1916–1918. [<a href="/pubmed/10208709" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+Fam+Physician&amp;title=Insomnia&amp;volume=59&amp;publication_year=1999&amp;pages=1911–1916&amp;pmid=10208709&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib16"><span class="mixed-citation">Weitzman ED, Czeisler CA, Coleman RM et al: <span class="ref-title">Delayed sleep phase syndrome. A chronobiological disorder with sleep-onset insomnia</span>. <span class="ref-journal">Arch Gen Psychiatry</span>
1981; <span class="ref-vol">38</span>: 737–746. [<a href="/pubmed/7247637" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Delayed+sleep+phase+syndrome.+A+chronobiological+disorder+with+sleep-onset+insomnia&amp;volume=38&amp;publication_year=1981&amp;pages=737&amp;pmid=7247637&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib17"><span class="mixed-citation">Freedman RR: <span class="ref-title">EEG power spectra in sleep-onset insomnia</span>. <span class="ref-journal">Electroencephalogr Clin Neurophysiol</span>
1986; <span class="ref-vol">63</span>: 408–413. [<a href="/pubmed/2420556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Electroencephalogr+Clin+Neurophysiol&amp;title=EEG+power+spectra+in+sleep-onset+insomnia&amp;volume=63&amp;publication_year=1986&amp;pages=408&amp;pmid=2420556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib18"><span class="mixed-citation">van den Heuvel CJ, Lushington K: <span class="ref-title">Chronobiology and insomnia: pathophysiology and treatment of circadian rhythm sleep disorders</span>. <span class="ref-journal">Expert Rev Neurother</span>
2002; <span class="ref-vol">2</span>: 249–260. [<a href="/pubmed/19811006" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Rev+Neurother&amp;title=Chronobiology+and+insomnia:+pathophysiology+and+treatment+of+circadian+rhythm+sleep+disorders&amp;volume=2&amp;publication_year=2002&amp;pages=249&amp;pmid=19811006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib19"><span class="mixed-citation">Touchette E, Petit D, Seguin JR, Boivin M, Tremblay RE, Montplaisir JY: <span class="ref-title">Associations between sleep duration patterns and behavioral/cognitive functioning at school entry</span>. <span class="ref-journal">Sleep</span>
2007; <span class="ref-vol">30</span>: 1213–1219. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1978413/">PMC free article</a>]</span> [<a href="/pubmed/17910393" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sleep&amp;title=Associations+between+sleep+duration+patterns+and+behavioral/cognitive+functioning+at+school+entry&amp;volume=30&amp;publication_year=2007&amp;pages=1213&amp;pmid=17910393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib20"><span class="mixed-citation">Heath AC, Kendler KS, Eaves LJ, Martin NG: <span class="ref-title">Evidence for genetic influences on sleep disturbance and sleep pattern in twins</span>. <span class="ref-journal">Sleep</span>
1990; <span class="ref-vol">13</span>: 318–335. [<a href="/pubmed/2267475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sleep&amp;title=Evidence+for+genetic+influences+on+sleep+disturbance+and+sleep+pattern+in+twins&amp;volume=13&amp;publication_year=1990&amp;pages=318&amp;pmid=2267475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib21"><span class="mixed-citation">Klei L, Reitz P, Miller M et al: <span class="ref-title">Heritability of morningness-eveningness and self-report sleep measures in a family-based sample of 521 hutterites</span>. <span class="ref-journal">Chronobiol Int</span>
2005; <span class="ref-vol">22</span>: 1041–1054. [<a href="/pubmed/16393707" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chronobiol+Int&amp;title=Heritability+of+morningness-eveningness+and+self-report+sleep+measures+in+a+family-based+sample+of+521+hutterites&amp;volume=22&amp;publication_year=2005&amp;pages=1041&amp;pmid=16393707&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib22"><span class="mixed-citation">Partinen M, Kaprio J, Koskenvuo M, Putkonen P, Langinvainio H: <span class="ref-title">Genetic and environmental determination of human sleep</span>. <span class="ref-journal">Sleep</span>
1983; <span class="ref-vol">6</span>: 179–185. [<a href="/pubmed/6684786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sleep&amp;title=Genetic+and+environmental+determination+of+human+sleep&amp;volume=6&amp;publication_year=1983&amp;pages=179&amp;pmid=6684786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib23"><span class="mixed-citation">Kantermann T, Juda M, Merrow M, Roenneberg T: <span class="ref-title">The human circadian clock's seasonal adjustment is disrupted by daylight saving time</span>. <span class="ref-journal">Curr Biol</span>
2007; <span class="ref-vol">17</span>: 1996–2000. [<a href="/pubmed/17964164" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Biol&amp;title=The+human+circadian+clock's+seasonal+adjustment+is+disrupted+by+daylight+saving+time&amp;volume=17&amp;publication_year=2007&amp;pages=1996&amp;pmid=17964164&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib24"><span class="mixed-citation">DeMartinis NA, Winokur A: <span class="ref-title">Effects of psychiatric medications on sleep and sleep disorders</span>. <span class="ref-journal">CNS Neurol Disord Drug Targets</span>
2007; <span class="ref-vol">6</span>: 17–29. [<a href="/pubmed/17305551" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CNS+Neurol+Disord+Drug+Targets&amp;title=Effects+of+psychiatric+medications+on+sleep+and+sleep+disorders&amp;volume=6&amp;publication_year=2007&amp;pages=17&amp;pmid=17305551&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib25"><span class="mixed-citation">Wittmann M, Dinich J, Merrow M, Roenneberg T: <span class="ref-title">Social jetlag: misalignment of biological and social time</span>. <span class="ref-journal">Chronobiol Int</span>
2006; <span class="ref-vol">23</span>: 497–509. [<a href="/pubmed/16687322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chronobiol+Int&amp;title=Social+jetlag:+misalignment+of+biological+and+social+time&amp;volume=23&amp;publication_year=2006&amp;pages=497&amp;pmid=16687322&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib26"><span class="mixed-citation">Li Y, Abecasis GR: <span class="ref-title">Mach 1.0: rapid haplotype reconstruction and missing genotype inference</span>. <span class="ref-journal">Am J Hum Genet</span>
2006; <span class="ref-vol">S79</span>: 2290. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Mach+1.0:+rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib27"><span class="mixed-citation">Li Y, Willer C, Sanna S, Abecasis G: <span class="ref-title">Genotype imputation</span>. <span class="ref-journal">Annu Rev Genomics Hum Genet</span>
2009; <span class="ref-vol">10</span>: 387–406. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Genomics+Hum+Genet&amp;title=Genotype+imputation&amp;volume=10&amp;publication_year=2009&amp;pages=387&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib28"><span class="mixed-citation">Howie BN, Donnelly P, Marchini J: <span class="ref-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies</span>. <span class="ref-journal">PLoS Genet</span>
2009; <span class="ref-vol">5</span>: e1000529. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib29"><span class="mixed-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P: <span class="ref-title">A new multipoint method for genome-wide association studies by imputation of genotypes</span>. <span class="ref-journal">Nat Genet</span>
2007; <span class="ref-vol">39</span>: 906–913. [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;volume=39&amp;publication_year=2007&amp;pages=906&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib30"><span class="mixed-citation">Almasy L, Blangero J: <span class="ref-title">Multipoint quantitative-trait linkage analysis in general pedigrees</span>. <span class="ref-journal">Am J Hum Genet</span>
1998; <span class="ref-vol">62</span>: 1198–1211. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377101/">PMC free article</a>]</span> [<a href="/pubmed/9545414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Multipoint+quantitative-trait+linkage+analysis+in+general+pedigrees&amp;volume=62&amp;publication_year=1998&amp;pages=1198&amp;pmid=9545414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib31"><span class="mixed-citation">Liu F, Kirichenko A, Axenovich TI, van Duijn CM, Aulchenko YS: <span class="ref-title">An approach for cutting large and complex pedigrees for linkage analysis</span>. <span class="ref-journal">Eur J Hum Genet</span>
2008; <span class="ref-vol">16</span>: 854–860. [<a href="/pubmed/18301450" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=An+approach+for+cutting+large+and+complex+pedigrees+for+linkage+analysis&amp;volume=16&amp;publication_year=2008&amp;pages=854&amp;pmid=18301450&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib32"><span class="mixed-citation">Aulchenko YS, Ripke S, Isaacs A, van Duijn CM: <span class="ref-title">GenABEL: an R library for genome-wide association analysis</span>. <span class="ref-journal">Bioinformatics</span>
2007; <span class="ref-vol">23</span>: 1294–1296. [<a href="/pubmed/17384015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenABEL:+an+R+library+for+genome-wide+association+analysis&amp;volume=23&amp;publication_year=2007&amp;pages=1294&amp;pmid=17384015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib33"><span class="mixed-citation">Amin N, van Duijn CM, Aulchenko YS: <span class="ref-title">A genomic background based method for association analysis in related individuals</span>. <span class="ref-journal">PLoS One</span>
2007; <span class="ref-vol">2</span>: e1274. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2093991/">PMC free article</a>]</span> [<a href="/pubmed/18060068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=A+genomic+background+based+method+for+association+analysis+in+related+individuals&amp;volume=2&amp;publication_year=2007&amp;pages=e1274&amp;pmid=18060068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib34"><span class="mixed-citation">Aulchenko YS, Struchalin MV, van Duijn CM: <span class="ref-title">ProbABEL package for genome-wide association analysis of imputed data</span>. <span class="ref-journal">BMC Bioinformatics</span>
2010; <span class="ref-vol">11</span>: 134. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2846909/">PMC free article</a>]</span> [<a href="/pubmed/20233392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=ProbABEL+package+for+genome-wide+association+analysis+of+imputed+data&amp;volume=11&amp;publication_year=2010&amp;pages=134&amp;pmid=20233392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib35"><span class="mixed-citation">Wellcome Trust: <span class="ref-title">Case Control C. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls</span>. <span class="ref-journal">Nature</span>
2007; <span class="ref-vol">447</span>: 661–678. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Case+Control+C.+Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib36"><span class="mixed-citation">Chen WM, Abecasis GR: <span class="ref-title">Family-based association tests for genomewide association scans</span>. <span class="ref-journal">Am J Hum Genet</span>
2007; <span class="ref-vol">81</span>: 913–926. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265659/">PMC free article</a>]</span> [<a href="/pubmed/17924335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Family-based+association+tests+for+genomewide+association+scans&amp;volume=81&amp;publication_year=2007&amp;pages=913&amp;pmid=17924335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib37"><span class="mixed-citation">Orwoll E, Blank JB, Barrett-Connor E et al: <span class="ref-title">Design and baseline characteristics of the osteoporotic fractures in men (MrOS) study—a large observational study of the determinants of fracture in older men</span>. <span class="ref-journal">Contemp Clin Trials</span>
2005; <span class="ref-vol">26</span>: 569–585. [<a href="/pubmed/16084776" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Contemp+Clin+Trials&amp;title=Design+and+baseline+characteristics+of+the+osteoporotic+fractures+in+men+(MrOS)+study—a+large+observational+study+of+the+determinants+of+fracture+in+older+men&amp;volume=26&amp;publication_year=2005&amp;pages=569&amp;pmid=16084776&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib38"><span class="mixed-citation">Zaykin DV: <span class="ref-title">Optimally weighted Z-test is a powerful method for combining probabilities in meta-analysis</span>. <span class="ref-journal">J Evol Biol</span>
2011; <span class="ref-vol">24</span>: 1836–1841. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3135688/">PMC free article</a>]</span> [<a href="/pubmed/21605215" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Evol+Biol&amp;title=Optimally+weighted+Z-test+is+a+powerful+method+for+combining+probabilities+in+meta-analysis&amp;volume=24&amp;publication_year=2011&amp;pages=1836&amp;pmid=21605215&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib39"><span class="mixed-citation">Fehrmann RS, Karjalainen JM, Krajewska M et al: <span class="ref-title">Gene expression analysis identifies global gene dosage sensitivity in cancer</span>. <span class="ref-journal">Nat Genet</span>
2015; <span class="ref-vol">47</span>: 115–125. [<a href="/pubmed/25581432" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Gene+expression+analysis+identifies+global+gene+dosage+sensitivity+in+cancer&amp;volume=47&amp;publication_year=2015&amp;pages=115&amp;pmid=25581432&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib40"><span class="mixed-citation">Alter O, Brown PO, Botstein D: <span class="ref-title">Singular value decomposition for genome-wide expression data processing and modeling</span>. <span class="ref-journal">Proc Natl Acad Sci USA</span>
2000; <span class="ref-vol">97</span>: 10101–10106. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC27718/">PMC free article</a>]</span> [<a href="/pubmed/10963673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Singular+value+decomposition+for+genome-wide+expression+data+processing+and+modeling&amp;volume=97&amp;publication_year=2000&amp;pages=10101&amp;pmid=10963673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib41"><span class="mixed-citation">Sherlock G: <span class="ref-title">Analysis of large-scale gene expression data</span>. <span class="ref-journal">Brief Bioinform</span>
2001; <span class="ref-vol">2</span>: 350–362. [<a href="/pubmed/11808747" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brief+Bioinform&amp;title=Analysis+of+large-scale+gene+expression+data&amp;volume=2&amp;publication_year=2001&amp;pages=350&amp;pmid=11808747&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib42"><span class="mixed-citation">Creyghton MP, Cheng AW, Welstead GG et al: <span class="ref-title">Histone H3K27ac separates active from poised enhancers and predicts developmental state</span>. <span class="ref-journal">Proc Natl Acad Sci USA</span>
2010; <span class="ref-vol">107</span>: 21931–21936. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3003124/">PMC free article</a>]</span> [<a href="/pubmed/21106759" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Histone+H3K27ac+separates+active+from+poised+enhancers+and+predicts+developmental+state&amp;volume=107&amp;publication_year=2010&amp;pages=21931&amp;pmid=21106759&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib43"><span class="mixed-citation">Kim KK, Adelstein RS, Kawamoto S: <span class="ref-title">Identification of neuronal nuclei (NeuN) as Fox-3, a new member of the Fox-1 gene family of splicing factors</span>. <span class="ref-journal">J Biol Chem</span>
2009; <span class="ref-vol">284</span>: 31052–31061. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2781505/">PMC free article</a>]</span> [<a href="/pubmed/19713214" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Identification+of+neuronal+nuclei+(NeuN)+as+Fox-3,+a+new+member+of+the+Fox-1+gene+family+of+splicing+factors&amp;volume=284&amp;publication_year=2009&amp;pages=31052&amp;pmid=19713214&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib44"><span class="mixed-citation">Dredge BK, Jensen KB: <span class="ref-title">NeuN/Rbfox3 nuclear and cytoplasmic isoforms differentially regulate alternative splicing and nonsense-mediated decay of Rbfox2</span>. <span class="ref-journal">PLoS One</span>
2011; <span class="ref-vol">6</span>: e21585. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3126832/">PMC free article</a>]</span> [<a href="/pubmed/21747913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=NeuN/Rbfox3+nuclear+and+cytoplasmic+isoforms+differentially+regulate+alternative+splicing+and+nonsense-mediated+decay+of+Rbfox2&amp;volume=6&amp;publication_year=2011&amp;pages=e21585&amp;pmid=21747913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib45"><span class="mixed-citation">Kiehl TR, Shibata H, Vo T, Huynh DP, Pulst SM: <span class="ref-title">Identification and expression of a mouse ortholog of A2BP1</span>. <span class="ref-journal">Mamm Genome</span>
2001; <span class="ref-vol">12</span>: 595–601. [<a href="/pubmed/11471052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mamm+Genome&amp;title=Identification+and+expression+of+a+mouse+ortholog+of+A2BP1&amp;volume=12&amp;publication_year=2001&amp;pages=595&amp;pmid=11471052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib46"><span class="mixed-citation">Underwood JG, Boutz PL, Dougherty JD, Stoilov P, Black DL: <span class="ref-title">Homologues of the Caenorhabditis elegans Fox-1 protein are neuronal splicing regulators in mammals</span>. <span class="ref-journal">Mol Cell Biol</span>
2005; <span class="ref-vol">25</span>: 10005–10016. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1280273/">PMC free article</a>]</span> [<a href="/pubmed/16260614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Homologues+of+the+Caenorhabditis+elegans+Fox-1+protein+are+neuronal+splicing+regulators+in+mammals&amp;volume=25&amp;publication_year=2005&amp;pages=10005&amp;pmid=16260614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib47"><span class="mixed-citation">McKee AE, Minet E, Stern C, Riahi S, Stiles CD, Silver PA: <span class="ref-title">A genome-wide <em>in situ</em> hybridization map of RNA-binding proteins reveals anatomically restricted expression in the developing mouse brain</span>. <span class="ref-journal">BMC Dev Biol</span>
2005; <span class="ref-vol">5</span>: 14. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1199591/">PMC free article</a>]</span> [<a href="/pubmed/16033648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Dev+Biol&amp;title=A+genome-wide+in+situ+hybridization+map+of+RNA-binding+proteins+reveals+anatomically+restricted+expression+in+the+developing+mouse+brain&amp;volume=5&amp;publication_year=2005&amp;pages=14&amp;pmid=16033648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib48"><span class="mixed-citation">Modrek B, Resch A, Grasso C, Lee C: <span class="ref-title">Genome-wide detection of alternative splicing in expressed sequences of human genes</span>. <span class="ref-journal">Nucleic Acids Res</span>
2001; <span class="ref-vol">29</span>: 2850–2859. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC55780/">PMC free article</a>]</span> [<a href="/pubmed/11433032" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Genome-wide+detection+of+alternative+splicing+in+expressed+sequences+of+human+genes&amp;volume=29&amp;publication_year=2001&amp;pages=2850&amp;pmid=11433032&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib49"><span class="mixed-citation">Yeo G, Holste D, Kreiman G, Burge CB: <span class="ref-title">Variation in alternative splicing across human tissues</span>. <span class="ref-journal">Genome Biol</span>
2004; <span class="ref-vol">5</span>: R74. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC545594/">PMC free article</a>]</span> [<a href="/pubmed/15461793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Variation+in+alternative+splicing+across+human+tissues&amp;volume=5&amp;publication_year=2004&amp;pages=R74&amp;pmid=15461793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib50"><span class="mixed-citation">Morin LP, Hefton S, Studholme KM: <span class="ref-title">Neurons identified by NeuN/Fox-3 immunoreactivity have a novel distribution in the hamster and mouse suprachiasmatic nucleus</span>. <span class="ref-journal">Brain Res</span>
2011; <span class="ref-vol">1421</span>: 44–51. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3200441/">PMC free article</a>]</span> [<a href="/pubmed/21981805" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res&amp;title=Neurons+identified+by+NeuN/Fox-3+immunoreactivity+have+a+novel+distribution+in+the+hamster+and+mouse+suprachiasmatic+nucleus&amp;volume=1421&amp;publication_year=2011&amp;pages=44&amp;pmid=21981805&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib51"><span class="mixed-citation">Bhalla K, Phillips HA, Crawford J et al: <span class="ref-title">The de novo chromosome 16 translocations of two patients with abnormal phenotypes (mental retardation and epilepsy) disrupt the A2BP1 gene</span>. <span class="ref-journal">J Hum Genet</span>
2004; <span class="ref-vol">49</span>: 308–311. [<a href="/pubmed/15148587" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hum+Genet&amp;title=The+de+novo+chromosome+16+translocations+of+two+patients+with+abnormal+phenotypes+(mental+retardation+and+epilepsy)+disrupt+the+A2BP1+gene&amp;volume=49&amp;publication_year=2004&amp;pages=308&amp;pmid=15148587&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib52"><span class="mixed-citation">Barnby G, Abbott A, Sykes N et al: <span class="ref-title">Candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at GRIN2A and ABAT</span>. <span class="ref-journal">Am J Hum Genet</span>
2005; <span class="ref-vol">76</span>: 950–966. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1196454/">PMC free article</a>]</span> [<a href="/pubmed/15830322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Candidate-gene+screening+and+association+analysis+at+the+autism-susceptibility+locus+on+chromosome+16p:+evidence+of+association+at+GRIN2A+and+ABAT&amp;volume=76&amp;publication_year=2005&amp;pages=950&amp;pmid=15830322&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib53"><span class="mixed-citation">Martin CL, Duvall JA, Ilkin Y et al: <span class="ref-title">Cytogenetic and molecular characterization of A2BP1/FOX1 as a candidate gene for autism</span>. <span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet</span>
2007; <span class="ref-vol">144B</span>: 869–876. [<a href="/pubmed/17503474" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Cytogenetic+and+molecular+characterization+of+A2BP1/FOX1+as+a+candidate+gene+for+autism&amp;volume=144B&amp;publication_year=2007&amp;pages=869&amp;pmid=17503474&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib54"><span class="mixed-citation">Sebat J, Lakshmi B, Malhotra D et al: <span class="ref-title">Strong association of de novo copy number mutations with autism</span>. <span class="ref-journal">Science</span>
2007; <span class="ref-vol">316</span>: 445–449. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2993504/">PMC free article</a>]</span> [<a href="/pubmed/17363630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Strong+association+of+de+novo+copy+number+mutations+with+autism&amp;volume=316&amp;publication_year=2007&amp;pages=445&amp;pmid=17363630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib55"><span class="mixed-citation">Utami KH, Hillmer AM, Aksoy I et al: <span class="ref-title">Detection of chromosomal breakpoints in patients with developmental delay and speech disorders</span>. <span class="ref-journal">PLoS One</span>
2014; <span class="ref-vol">9</span>: e90852. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3946304/">PMC free article</a>]</span> [<a href="/pubmed/24603971" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Detection+of+chromosomal+breakpoints+in+patients+with+developmental+delay+and+speech+disorders&amp;volume=9&amp;publication_year=2014&amp;pages=e90852&amp;pmid=24603971&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib56"><span class="mixed-citation">Luppi PH: <span class="ref-title">Neurochemical aspects of sleep regulation with specific focus on slow-wave sleep</span>. <span class="ref-journal">World J Biol Psychiatry</span>
2010; <span class="ref-vol">11</span> (Suppl 1): 4–8. [<a href="/pubmed/20509826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=World+J+Biol+Psychiatry&amp;title=Neurochemical+aspects+of+sleep+regulation+with+specific+focus+on+slow-wave+sleep&amp;volume=11&amp;publication_year=2010&amp;pages=4&amp;pmid=20509826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib57"><span class="mixed-citation">Whiting PJ: <span class="ref-title">The GABA-A receptor gene family: new targets for therapeutic intervention</span>. <span class="ref-journal">Neurochem Int</span>
1999; <span class="ref-vol">34</span>: 387–390. [<a href="/pubmed/10397366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurochem+Int&amp;title=The+GABA-A+receptor+gene+family:+new+targets+for+therapeutic+intervention&amp;volume=34&amp;publication_year=1999&amp;pages=387&amp;pmid=10397366&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib58"><span class="mixed-citation">Whiting PJ, Bonnert TP, McKernan RM et al: <span class="ref-title">Molecular and functional diversity of the expanding GABA-A receptor gene family</span>. <span class="ref-journal">Ann N Y Acad Sci</span>
1999; <span class="ref-vol">868</span>: 645–653. [<a href="/pubmed/10414349" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+N+Y+Acad+Sci&amp;title=Molecular+and+functional+diversity+of+the+expanding+GABA-A+receptor+gene+family&amp;volume=868&amp;publication_year=1999&amp;pages=645&amp;pmid=10414349&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib59"><span class="mixed-citation">Buhr A, Bianchi MT, Baur R et al: <span class="ref-title">Functional characterization of the new human GABA(A) receptor mutation beta3(R192H)</span>. <span class="ref-journal">Hum Genet</span>
2002; <span class="ref-vol">111</span>: 154–160. [<a href="/pubmed/12189488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Functional+characterization+of+the+new+human+GABA(A)+receptor+mutation+beta3(R192H)&amp;volume=111&amp;publication_year=2002&amp;pages=154&amp;pmid=12189488&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib60"><span class="mixed-citation">Charney DS, Nestler EJ: <span class="ref-journal">Neurobiology of mental illness</span>. Oxford University Press: Oxford; New York, NY, USA, 2004. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Neurobiology+of+mental+illness&amp;publication_year=2004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5027680&amp;issue-id=275767&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140144232009408title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Materials and methods</a></li><li><a href="#__sec12title">Results</a></li><li><a href="#__sec13title">Discussion</a></li><li><a href="#idm140144201861712title">Acknowledgments</a></li><li><a href="#idm140144201857040title">Footnotes</a></li><li><a href="#sup1title">Supplementary Material</a></li><li><a href="#idm140144201854160title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5027680/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5027680/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5027680/pdf/ejhg201631a.pdf">PDF (356K)</a></li> | <li><a href="#" data-citationid="PMC5027680" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5027680%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5027680%2F&amp;text=Genetic%20variants%20in%20RBFOX3%20are%20associated%20with%20sleep%20latency" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5027680%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27142678" data-aiid="5027680" data-aid="5027680" data-iid="275767" data-domainid="1300" data-domain="ejhg" data-accid="PMC5027680" data-md5="b2cbbff1124bdd5260a9f41474cd23ca"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:41:58-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A8E4ADD58B78100000000075F046B&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5027680%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5027680/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>