<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ndt"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5837679/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genetic risk variants for membranous nephropathy: extension of and association with other chronic kidney disease aetiologies</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5837679/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nephrology Dialysis Transplantation"><meta name="citation_title" content="Genetic risk variants for membranous nephropathy: extension of and association with other chronic kidney disease aetiologies"><meta name="citation_authors" content="Peggy Sekula, Yong Li, Horia C. Stanescu, Matthias Wuttke, Arif B. Ekici, Detlef Bockenhauer, Gerd Walz, Stephen H. Powis, Jan T. Kielstein, Paul Brenchley, GCKD Investigators, Kai-Uwe Eckardt, Florian Kronenberg, Robert Kleta, Anna Köttgen"><meta name="citation_date" content="February 2017"><meta name="citation_issue" content="2"><meta name="citation_volume" content="32"><meta name="citation_firstpage" content="325"><meta name="citation_doi" content="10.1093/ndt/gfw001"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5837679/?report=abstract"><meta name="citation_pmid" content="27333618"><meta name="DC.Title" content="Genetic risk variants for membranous nephropathy: extension of and association with other chronic kidney disease aetiologies"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Peggy Sekula"><meta name="DC.Contributor" content="Yong Li"><meta name="DC.Contributor" content="Horia C. Stanescu"><meta name="DC.Contributor" content="Matthias Wuttke"><meta name="DC.Contributor" content="Arif B. Ekici"><meta name="DC.Contributor" content="Detlef Bockenhauer"><meta name="DC.Contributor" content="Gerd Walz"><meta name="DC.Contributor" content="Stephen H. Powis"><meta name="DC.Contributor" content="Jan T. Kielstein"><meta name="DC.Contributor" content="Paul Brenchley"><meta name="DC.Contributor" content="GCKD Investigators"><meta name="DC.Contributor" content="Kai-Uwe Eckardt"><meta name="DC.Contributor" content="Florian Kronenberg"><meta name="DC.Contributor" content="Robert Kleta"><meta name="DC.Contributor" content="Anna Köttgen"><meta name="DC.Date" content="2017 Feb"><meta name="DC.Identifier" content="10.1093/ndt/gfw001"><meta name="DC.Language" content="en"><meta property="og:title" content="Genetic risk variants for membranous nephropathy: extension of and association with other chronic kidney disease aetiologies"><meta property="og:type" content="article"><meta property="og:description" content="
Background: Membranous nephropathy (MN) is a common cause of nephrotic syndrome in adults. Previous genome-wide association studies (GWAS) of 300 000 genotyped variants identified MN-associated loci at HLA-DQA1 and PLA2R1.
Methods: We used a combined ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5837679/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ndt.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5837679/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5837679/pdf/gfw001.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89DA96DD593E310000000003540220.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5837679%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books">Books</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Nephrol Dial Transplant%22[journal]" class="navlink">Nephrol Dial Transplant</a></li><li class="accid">PMC5837679</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ndt.png" alt="Logo of ndt" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://ndt.oxfordjournals.org/" target="pmc_ext" ref="reftype=publisher&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nephrol Dial Transplant</a></span>. 2017 Feb; 32(2): 325–332. </span></div><div><span class="fm-vol-iss-date">Published online 2016 Feb 4. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fndt%2Fgfw001" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/ndt/gfw001</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5837679</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27333618">27333618</a></div></div></div></div><h1 class="content-title">Genetic risk variants for membranous nephropathy: extension of and association with other chronic kidney disease aetiologies</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sekula%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841093056272" co-class="co-affbox">Peggy Sekula</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841091851760" co-class="co-affbox">Yong Li</a>,<sup>1</sup> <a href="/pubmed/?term=Stanescu%20HC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841100362384" co-class="co-affbox">Horia C. Stanescu</a>,<sup>3</sup> <a href="/pubmed/?term=Wuttke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841103146368" co-class="co-affbox">Matthias Wuttke</a>,<sup>1</sup> <a href="/pubmed/?term=Ekici%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841089361984" co-class="co-affbox">Arif B. Ekici</a>,<sup>4</sup> <a href="/pubmed/?term=Bockenhauer%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841089360000" co-class="co-affbox">Detlef Bockenhauer</a>,<sup>3</sup> <a href="/pubmed/?term=Walz%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841103336144" co-class="co-affbox">Gerd Walz</a>,<sup>1</sup> <a href="/pubmed/?term=Powis%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841103334160" co-class="co-affbox">Stephen H. Powis</a>,<sup>3</sup> <a href="/pubmed/?term=Kielstein%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841099644816" co-class="co-affbox">Jan T. Kielstein</a>,<sup>5</sup> <a href="/pubmed/?term=Brenchley%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841100852288" co-class="co-affbox">Paul Brenchley</a>,<sup>6</sup> GCKD Investigators,<sup></sup> <a href="/pubmed/?term=Eckardt%20KU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841106710400" co-class="co-affbox">Kai-Uwe Eckardt</a>,<sup>7</sup> <a href="/pubmed/?term=Kronenberg%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841106708416" co-class="co-affbox">Florian Kronenberg</a>,<sup>8</sup> <a href="/pubmed/?term=Kleta%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841103589280" co-class="co-affbox">Robert Kleta</a>,<sup>3</sup> and  <a href="/pubmed/?term=K%26%23x000f6%3Bttgen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618" class="affpopup" co-rid="_co_idm139841103587296" co-class="co-affbox">Anna Köttgen</a><sup>1,</sup><sup>2</sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139841093056272"><h3 class="no_margin">Peggy Sekula</h3><p>  <sup>1</sup>Department of Internal Medicine IV, Medical Center—University of Freiburg, Berliner Allee 29, 79110 Freiburg, Germany</p><p>  <sup>2</sup>Center for Medical Biometry and Medical Informatics, Medical Center—University of Freiburg, Freiburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Sekula%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Peggy Sekula</a></div></div><div id="_co_idm139841091851760"><h3 class="no_margin">Yong Li</h3><p>  <sup>1</sup>Department of Internal Medicine IV, Medical Center—University of Freiburg, Berliner Allee 29, 79110 Freiburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Yong Li</a></div></div><div id="_co_idm139841100362384"><h3 class="no_margin">Horia C. Stanescu</h3><p>  <sup>3</sup>Centre for Nephrology, University College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Stanescu%20HC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Horia C. Stanescu</a></div></div><div id="_co_idm139841103146368"><h3 class="no_margin">Matthias Wuttke</h3><p>  <sup>1</sup>Department of Internal Medicine IV, Medical Center—University of Freiburg, Berliner Allee 29, 79110 Freiburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Wuttke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Matthias Wuttke</a></div></div><div id="_co_idm139841089361984"><h3 class="no_margin">Arif B. Ekici</h3><p>  <sup>4</sup>Institute of Human Genetics, Friedrich-Alexander University, Erlangen-Nürnberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Ekici%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Arif B. Ekici</a></div></div><div id="_co_idm139841089360000"><h3 class="no_margin">Detlef Bockenhauer</h3><p>  <sup>3</sup>Centre for Nephrology, University College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Bockenhauer%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Detlef Bockenhauer</a></div></div><div id="_co_idm139841103336144"><h3 class="no_margin">Gerd Walz</h3><p>  <sup>1</sup>Department of Internal Medicine IV, Medical Center—University of Freiburg, Berliner Allee 29, 79110 Freiburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Walz%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Gerd Walz</a></div></div><div id="_co_idm139841103334160"><h3 class="no_margin">Stephen H. Powis</h3><p>  <sup>3</sup>Centre for Nephrology, University College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Powis%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Stephen H. Powis</a></div></div><div id="_co_idm139841099644816"><h3 class="no_margin">Jan T. Kielstein</h3><p>  <sup>5</sup>Department of Nephrology and Hypertension, Hannover Medical School, Hannover, Germany</p><div>Find articles by <a href="/pubmed/?term=Kielstein%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Jan T. Kielstein</a></div></div><div id="_co_idm139841100852288"><h3 class="no_margin">Paul Brenchley</h3><p>  <sup>6</sup>Institute of Cardiovascular Sciences, University of Manchester, Manchester, UK</p><div>Find articles by <a href="/pubmed/?term=Brenchley%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Paul Brenchley</a></div></div><div id="_co_idm139841106710400"><h3 class="no_margin">Kai-Uwe Eckardt</h3><p>  <sup>7</sup>Department of Nephrology and Hypertension, Friedrich-Alexander University, Erlangen-Nürnberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Eckardt%20KU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Kai-Uwe Eckardt</a></div></div><div id="_co_idm139841106708416"><h3 class="no_margin">Florian Kronenberg</h3><p>  <sup>8</sup>Division of Genetic Epidemiology, Department of Medical Genetics, Molecular and Clinical Pharmacology, Medical University of Innsbruck, Innsbruck, Austria</p><div>Find articles by <a href="/pubmed/?term=Kronenberg%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Florian Kronenberg</a></div></div><div id="_co_idm139841103589280"><h3 class="no_margin">Robert Kleta</h3><p>  <sup>3</sup>Centre for Nephrology, University College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Kleta%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Robert Kleta</a></div></div><div id="_co_idm139841103587296"><h3 class="no_margin">Anna Köttgen</h3><p>  <sup>1</sup>Department of Internal Medicine IV, Medical Center—University of Freiburg, Berliner Allee 29, 79110 Freiburg, Germany</p><p>  <sup>2</sup>Center for Medical Biometry and Medical Informatics, Medical Center—University of Freiburg, Freiburg, Germany</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000f6%3Bttgen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27333618">Anna Köttgen</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139841105662384_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139841105662384_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139841105662384_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139841105662384_ai" style="display:none"><div class="fm-affl" id="af1" lang="en">  <sup>1</sup>Department of Internal Medicine IV, Medical Center—University of Freiburg, Berliner Allee 29, 79110 Freiburg, Germany</div><div class="fm-affl" id="af2" lang="en">  <sup>2</sup>Center for Medical Biometry and Medical Informatics, Medical Center—University of Freiburg, Freiburg, Germany</div><div class="fm-affl" id="af3" lang="en">  <sup>3</sup>Centre for Nephrology, University College London, London, UK</div><div class="fm-affl" id="af4" lang="en">  <sup>4</sup>Institute of Human Genetics, Friedrich-Alexander University, Erlangen-Nürnberg, Germany</div><div class="fm-affl" id="af5" lang="en">  <sup>5</sup>Department of Nephrology and Hypertension, Hannover Medical School, Hannover, Germany</div><div class="fm-affl" id="af6" lang="en">  <sup>6</sup>Institute of Cardiovascular Sciences, University of Manchester, Manchester, UK</div><div class="fm-affl" id="af7" lang="en">  <sup>7</sup>Department of Nephrology and Hypertension, Friedrich-Alexander University, Erlangen-Nürnberg, Germany</div><div class="fm-affl" id="af8" lang="en">  <sup>8</sup>Division of Genetic Epidemiology, Department of Medical Genetics, Molecular and Clinical Pharmacology, Medical University of Innsbruck, Innsbruck, Austria</div><div id="COR1">Correspondence and offprint requests to: Anna Köttgen; E-mail: <a href="mailto:dev@null" data-email="ed.grubierf-kinilkinu@negtteok.anna" class="oemail">ed.grubierf-kinilkinu@negtteok.anna</a></div><div>Contributed by <sup>*</sup>Members of the GCKD Investigators are listed in the Acknowledgements.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139841105662384_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2015 Sep 8; Accepted 2015 Dec 29.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139841105662384_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2016. Published by Oxford University Press on behalf of ERA-EDTA. All rights reserved.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5837679/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Data.</strong></div><div><a href="/pmc/articles/PMC5837679/bin/gfw001_supp.zip" data-ga-action="click_feat_suppl">gfw001_supp.zip</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID:&nbsp;C00383C8-F80A-4CE7-AC90-F0BB3E2A3EEA</div></div></dd></dl></div><div id="idm139841103667568" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139841103667568title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first">
<strong>Background:</strong> Membranous nephropathy (MN) is a common cause of nephrotic syndrome in adults. Previous genome-wide association studies (GWAS) of 300 000 genotyped variants identified MN-associated loci at <em>HLA-DQA1</em> and <em>PLA2R1</em>.</p><p id="__p3">
<strong>Methods:</strong> We used a combined approach of genotype imputation, GWAS, human leucocyte antigen (HLA) imputation and extension to other aetiologies of chronic kidney disease (CKD) to investigate genetic MN risk variants more comprehensively. GWAS using 9 million high-quality imputed genotypes and classical HLA alleles were conducted for 323 MN European-ancestry cases and 345 controls. Additionally, 4960 patients with different CKD aetiologies in the German Chronic Kidney Disease (GCKD) study were genotyped for risk variants at <em>HLA-DQA1</em> and <em>PLA2R1</em>.</p><p id="__p4">
<strong>Results:</strong> In GWAS, lead variants in known loci [rs9272729, <em>HLA-DQA1</em>, odds ratio (OR) = 7.3 per risk allele, P = 5.9 × 10<sup>−27</sup> and rs17830558, <em>PLA2R1</em>, OR = 2.2, P = 1.9 × 10<sup>−8</sup>] were significantly associated with MN. No novel signals emerged in GWAS of X-chromosomal variants or in sex-specific analyses. Classical HLA alleles (DRB1*0301-DQA1*0501-DQB1*0201 haplotype) were associated with MN but provided little additional information beyond rs9272729. Associations were replicated in 137 GCKD patients with MN (<em>HLA-DQA1</em>: P = 6.4 × 10<sup>−24</sup>; <em>PLA2R1</em>: P = 5.0 × 10<sup>−4</sup>). MN risk increased steeply for patients with high-risk genotype combinations (OR &gt; 79). While genetic variation in <em>PLA2R1</em> exclusively associated with MN across 19 CKD aetiologies, the <em>HLA-DQA1</em> risk allele was also associated with lupus nephritis (P = 2.8 × 10<sup>−6</sup>), type 1 diabetic nephropathy (P = 6.9 × 10<sup>−5</sup>) and focal segmental glomerulosclerosis (P = 5.1 × 10<sup>−5</sup>), but not with immunoglobulin A nephropathy.</p><p id="__p5" class="p p-last">
<strong>Conclusions:</strong>
<em>PLA2R1</em> and <em>HLA-DQA1</em> are the predominant risk loci for MN detected by GWAS. While <em>HLA-DQA1</em> risk variants show an association with other CKD aetiologies, <em>PLA2R1</em> variants are specific to MN.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">chronic kidney disease, genome-wide association study, membranous nephropathy</span></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p6" class="p p-first">Membranous nephropathy (MN) is one of the most common causes of nephrotic syndrome in adults [<a href="#gfw001C1" rid="gfw001C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227984">1</a>, <a href="#gfw001C2" rid="gfw001C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227987">2</a>]. It is considered an immune complex–mediated disease, with subepithelial deposits of immune complexes detectable using immunofluorescence or electron microscopy. In some patients, MN is secondary and consequential to tumours, infections or exposure to environmental factors. The majority of patients, however, suffer from so-called primary MN, termed ‘idiopathic’ MN, until its recent recognition as an autoimmune disease.</p><p id="__p7">A genetic component to MN has long been suspected based on familial clustering [<a href="#gfw001C3" rid="gfw001C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227992">3</a>]. The involvement of a risk allele at the human leucocyte antigen (HLA) locus, and more specifically of DQA1, had already been proposed in MN pathogenesis in the late 1980s [<a href="#gfw001C4" rid="gfw001C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228000">4</a>]. Additional insights into MN pathophysiology from animal experiments showed that immune complex formation can result from interactions of antibodies with a local antigen on podocytes [<a href="#gfw001C1" rid="gfw001C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228012">1</a>]. In humans, the M-type phospholipase A2 receptor protein at the podocyte surface was identified as the major autoantigen in patients with MN [<a href="#gfw001C5" rid="gfw001C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228001">5</a>]. More recently, thrombospondin type-1 domain-containing 7A (THSD7A) was identified as another autoantigen in phospholipase A2 receptor antibody–negative patients with MN [<a href="#gfw001C6" rid="gfw001C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228007">6</a>].</p><p id="__p8">Genome-wide association studies (GWAS) can be used to evaluate the association between genome-wide genetic markers and a disease in a comprehensive and unbiased way. The first meta-analysis of a GWAS of MN identified strong associations between MN and single nucleotide polymorphisms (SNPs) at the <em>HLA-DQA1</em> locus and in <em>PLA2R1</em>, encoding the M-type phospholipase A2 receptor [<a href="#gfw001C7" rid="gfw001C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228006">7</a>]. This study not only provided evidence for the influence of an individual's genetic make-up in shaping the interaction between the adaptive immune system and autoantigens, but also provided striking evidence that the combination of risk alleles at only two genetic loci was associated with large increases in disease risk of a magnitude that is typically only observed for single gene disorders.</p><p id="__p9" class="p p-last">However, previous GWAS of MN only evaluated a limited set of ∼300 000 genotyped genetic variants and did not use imputation methods [<a href="#gfw001C8" rid="gfw001C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228009">8</a>, <a href="#gfw001C9" rid="gfw001C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227979">9</a>] to increase marker coverage or to obtain information on classical HLA alleles [<a href="#gfw001C10" rid="gfw001C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670628546">10</a>]. Genotype imputation can increase the statistical power to detect genetic associations [<a href="#gfw001C9" rid="gfw001C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228003">9</a>]. Imputation to the 1000 Genomes reference panel in previous studies has led to the identification of additional risk loci that were missed in previous association studies of at least equal size, e.g. for body mass index, fasting glucose and coronary artery disease [<a href="#gfw001C11" rid="gfw001C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227998">11</a>, <a href="#gfw001C12" rid="gfw001C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227976">12</a>]. In addition, GWAS conditioning on the associated index variants can lead to the identification of secondary independent signals in associated regions [<a href="#gfw001C13" rid="gfw001C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670628547">13</a>]. Moreover, although MN shows a male predilection [<a href="#gfw001C1" rid="gfw001C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227999">1</a>], previous studies have combined both sexes to evaluate genetic associations and only studied autosomes. Finally, whether the previously identified genetic risk variants in the HLA locus and <em>PLA2R1</em> confer risk that is specific to MN or if they also increase the risk for other aetiologies of chronic kidney disease (CKD) has not been studied systematically. Therefore, the present study aimed to address these important gaps.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">MATERIALS AND METHODS</h2><p id="__p10" class="p p-first">For the GWAS of MN, DNA samples from 336 British MN cases who provided written informed consent were collected by the MRC/Kidney Research UK National DNA Bank for Glomerulonephritis and racially matched to samples from 349 control subjects of the 1958 UK birth control study (<a href="http://www.b58cgene.sgul.ac.uk">http://www.b58cgene.sgul.ac.uk</a>). Study protocol and sample characteristics have been described previously [<a href="#gfw001C7" rid="gfw001C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227997">7</a>]. On average, MN patients were 52.5 (±13.3) years old at the time of diagnosis and 69% of them were men.</p><p id="__p11">For replication and characterization, we examined patients taking part in the German Chronic Kidney Disease (GCKD) Study, a prospective observational study of patients with CKD with either an estimated glomerular filtration rate (eGFR) between 30 and 60 mL/min/1.73 m² or overt proteinuria (albumin:creatinine ratio &gt;300 mg/g or protein:creatinine ratio &gt;500 mg/g) upon study inclusion [<a href="#gfw001C14" rid="gfw001C14" class=" bibr popnode">14</a>]. The study was approved by local review boards at each participating academic institution and is described in detail elsewhere [<a href="#gfw001C15" rid="gfw001C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227994">15</a>]. Briefly, 5217 patients who gave their written informed consent were enrolled into the study between 2010 and 2012. The patients' long-term treating nephrologists were asked to select the leading, i.e. primary, cause of CKD for each patient. For disease categories in which biopsy was not performed for the majority of patients, such as diabetic or hypertensive kidney disease, diagnosis was established by the nephrologists based on clinical grounds. The assignment of these causes was consistent with comorbidities reported by the patients as well as with the presence of elevated biomarkers at the study visit (elevated haemoglobin A1c and blood pressure) and the intake of antidiabetic and anti-hypertensive medication, respectively. All patients were of Caucasian ancestry and 60% were men. At the time of entry, patients were on average 60.0 (±12.0) years old [<a href="#gfw001C16" rid="gfw001C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227991">16</a>]. MN as the leading cause of CKD was identified in 151 of 5217 patients, 144 of whom had a biopsy to confirm the diagnosis (80% men). In addition, other specific leading causes of CKD that comprised at least 40 patients were evaluated, as well as one smaller group of patients with rapidly progressive glomerulonephritis (GN) (pauci-immune; <em>n</em> = 28). Analyses were restricted to patients with a biopsy-proven diagnosis when the proportion of patients with biopsies in a given group was &gt;70%. Information on steroid sensitivity of focal segmental glomerulosclerosis (FSGS) was not available. As a control group, genotypes from 379 individuals (47% men) comprising the European subgroup of the 1000 Genomes project were used (<a href="http://www.1000genomes.org/home">http://www.1000genomes.org/home</a>; Phase 1, release v3, panel EUR). Data were downloaded from <a href="http://www.sph.umich.edu/csg/abecasis/MaCH/download/1000G.2012-03-14.html">http://www.sph.umich.edu/csg/abecasis/MaCH/download/1000G.2012-03-14.html</a> and genotypes for selected SNPs were subsequently extracted.</p><p id="__p12">Because of the complexity, detailed information on genotyping, quality control and imputation as well as on statistical analyses can be found in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary data, Methods</a>). In brief, GWAS samples were genotyped using the HumanCNV370-Quad SNP chip or HumanHap300 SNP chip. After stringent quality control using Plink and Eigensoft, the cleaned data set comprised 282 462 SNPs of 323 cases and 345 controls [<a href="#gfw001C17" rid="gfw001C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228013">17</a>, <a href="#gfw001C18" rid="gfw001C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228014">18</a>]. Subsequently, genotype imputation was carried out using Shapeit and Impute2 [<a href="#gfw001C19" rid="gfw001C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670628545">19</a>, <a href="#gfw001C20" rid="gfw001C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227983">20</a>]. Data from the 1000 Genomes project (Phase 1, release v3, panel ALL) served as a reference panel. The final data set contained 8.9 million SNPs of high quality. In the GCKD study, genotyping of index variants was carried out using an Agena (formerly Sequenom) iPLEX assay (<em>PLA2R1</em>) and using a Life Technologies TaqMan assay (<em>HLA-DQA1</em>) with call rates of 99% and genotype concordance of 100% among duplicate samples.</p><p id="__p13">Association tests relating MN case status to autosomal genetic variants were carried out using genotype probabilities (dosages) and an additive genetic model with the software SNPtest version 2.4.1 [<a href="#gfw001C21" rid="gfw001C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228002">21</a>]. The first five principle components were included as covariates. Statistical significance was set to the standard threshold of 5.0 × 10<sup>−8</sup>. Independent replication was defined as a one sided P-value &lt;0.05. Inverse-variance weighted, fixed-effect meta-analysis was conducted to pool results.</p><p id="__p14" class="p p-last">In GCKD, genotype information was also evaluated for association with CKD of 17 additional specific aetiologies as well as using all GCKD participants as a CKD case group, all in comparison with the 1000 Genomes control group. The statistical significance cut-off for these analyses was set to 2.6 × 10<sup>−3</sup>.</p></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">RESULTS</h2><div id="s3a" class="sec sec-first"><h3 id="s3atitle">GWAS of MN</h3><p id="__p15" class="p p-first">We carried out a GWAS among 323 MN cases and 345 controls of European ancestry using imputed genotypes as illustrated in the detailed analysis plan in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary data, Figure S1</a>. GWAS results did not show any evidence for inflation of the test statistics, indicating no systematic biases (quantile–quantile plot, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary data, Figure S2</a>). The two previously reported genetic regions, <em>PLA2R1</em> and <em>HLA-DQA1</em> [<a href="#gfw001C7" rid="gfw001C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228005">7</a>], contained SNPs associated with MN at genome-wide significance (Figure&nbsp;<a href="/pmc/articles/PMC5837679/figure/gfw001F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="gfw001F1" rid-ob="ob-gfw001F1" co-legend-rid="lgnd_gfw001F1"><span>1</span></a>). The index SNPs at the two loci used in subsequent analyses were rs9272729 in <em>HLA-DQA1</em> (P = 5.9 × 10<sup>−27</sup>) and rs17830558 in <em>PLA2R1</em> (P = 1.9 × 10<sup>−8</sup>, Table&nbsp;<a href="/pmc/articles/PMC5837679/table/gfw001TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="gfw001TB1" rid-ob="ob-gfw001TB1" co-legend-rid=""><span>1</span></a>). In both instances, the index SNPs were imputed with high imputation quality and were correlated with the previously published SNPs (Table&nbsp;<a href="/pmc/articles/PMC5837679/table/gfw001TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="gfw001TB1" rid-ob="ob-gfw001TB1" co-legend-rid=""><span>1</span></a>). <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary data, Figure S3A and B</a> display the two associated regions in more detail. The previously described extensive linkage disequilibrium in the HLA region is evident, extending several hundred kilobases around <em>HLA-DQA1</em> and including other HLA genes such as <em>HLA-DQB1</em>, <em>HLA-DRB1</em>, <em>HLA-DRB6</em> and <em>HLA-DRB5</em>.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="gfw001TB1"><h3>Table&nbsp;1</h3><!--caption a7--><div class="caption"><p id="__p16">Replicated SNPs associated with MN at genome-wide significance</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139841107315904" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left"></th><th colspan="2" rowspan="1" align="left">Gene<hr></th></tr><tr><th rowspan="1" colspan="1" align="left"><em>PLA2R1</em></th><th rowspan="1" colspan="1" align="left"><em>HLA-DQA1</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">SNP ID</td><td rowspan="1" colspan="1">rs17830558<sup>a</sup></td><td rowspan="1" colspan="1">rs9272729<sup>b</sup></td></tr><tr><td rowspan="1" colspan="1">chr:pos (hg19)</td><td rowspan="1" colspan="1">chr2:160878364</td><td rowspan="1" colspan="1">chr6:32609594</td></tr><tr><td rowspan="1" colspan="1">Function</td><td rowspan="1" colspan="1">Intronic</td><td rowspan="1" colspan="1">Intronic</td></tr><tr><td rowspan="1" colspan="1">Imputed (quality)</td><td rowspan="1" colspan="1">Yes (0.938)</td><td rowspan="1" colspan="1">Yes (0.901)</td></tr><tr><td rowspan="1" colspan="1">A<sub>1</sub>/A<sub>2</sub></td><td rowspan="1" colspan="1">T/G</td><td rowspan="1" colspan="1">A/G</td></tr><tr><td colspan="3" rowspan="1">Discovery analysis</td></tr><tr><td rowspan="1" colspan="1"> Allele frequency A<sub>1</sub></td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1"> OR</td><td rowspan="1" colspan="1">2.16</td><td rowspan="1" colspan="1">7.29</td></tr><tr><td rowspan="1" colspan="1"> 95% CI<sup>c</sup></td><td rowspan="1" colspan="1">1.64–2.85</td><td rowspan="1" colspan="1">4.82–11.04</td></tr><tr><td rowspan="1" colspan="1"> P-value<sup>c</sup></td><td rowspan="1" colspan="1">1.9 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">5.9 × 10<sup>−27</sup></td></tr><tr><td colspan="3" rowspan="1">Replication analysis</td></tr><tr><td rowspan="1" colspan="1"> Allele frequency A<sub>1</sub></td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">0.16</td></tr><tr><td rowspan="1" colspan="1"> OR</td><td rowspan="1" colspan="1">1.63</td><td rowspan="1" colspan="1">6.86</td></tr><tr><td rowspan="1" colspan="1"> 95% CI</td><td rowspan="1" colspan="1">1.23–2.15</td><td rowspan="1" colspan="1">4.55–10.34</td></tr><tr><td rowspan="1" colspan="1"> P-value</td><td rowspan="1" colspan="1">5.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">6.4 × 10<sup>−24</sup></td></tr><tr><td colspan="3" rowspan="1">Meta-analysis</td></tr><tr><td rowspan="1" colspan="1"> Allele frequency A<sub>1</sub></td><td rowspan="1" colspan="1">0.52</td><td rowspan="1" colspan="1">0.20</td></tr><tr><td rowspan="1" colspan="1"> OR</td><td rowspan="1" colspan="1">1.87</td><td rowspan="1" colspan="1">7.07</td></tr><tr><td rowspan="1" colspan="1"> 95% CI</td><td rowspan="1" colspan="1">1.54–2.28</td><td rowspan="1" colspan="1">5.28–9.47</td></tr><tr><td rowspan="1" colspan="1"> P-value</td><td rowspan="1" colspan="1">3.7 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">2.1 × 10<sup>−39</sup></td></tr></tbody></table></div><div id="largeobj_idm139841107315904" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5837679/table/gfw001TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn1"><p id="__p17" class="p p-first-last">OR refers to A1.</p></div><div id="tblfn2"><p id="__p18" class="p p-first-last"><sup>a</sup>SNP rs17830558 was used in all subsequent analyses of replication and meta-analysis as proxy for rs17830307, which showed the lowest P-value in the association analysis (results can be found in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary data, Table S1</a>).</p></div><div id="tblfn3"><p id="__p19" class="p p-first-last"><sup>b</sup>In GCKD, of the proxy rs2187668 was genotyped instead (pairwise correlations within our data) of rs9272729 and previously reported rs2187668 (<em>r</em><sup>2</sup> = 0.87) and of rs17830558 and previously reported rs4664308 (<em>r</em><sup>2</sup> = 0.40).</p></div><div id="tblfn4"><p id="__p20" class="p p-first-last"><sup>c</sup>GCKD-corrected values.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="gfw001F1" co-legend-rid="lgnd_gfw001F1"><a href="/pmc/articles/PMC5837679/figure/gfw001F1/" target="figure" rid-figpopup="gfw001F1" rid-ob="ob-gfw001F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139841106728032" class="figure"><a href="/pmc/articles/PMC5837679/figure/gfw001F1/" target="figure" rid-figpopup="gfw001F1" rid-ob="ob-gfw001F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5837679_gfw00101.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gfw00101.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5837679/bin/gfw00101.jpg"></a></div><div id="largeobj_idm139841106728032" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5837679/figure/gfw001F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_gfw001F1"><div><a class="figpopup" href="/pmc/articles/PMC5837679/figure/gfw001F1/" target="figure" rid-figpopup="gfw001F1" rid-ob="ob-gfw001F1">FIGURE&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p21">Results from the GWAS of MN using imputed genotype data (GCKD-corrected P-values, filtered by MAF &gt; 10%).</p></div></div></div><p id="__p22" class="p">No additional loci on the autosomes or the X chromosome contained SNPs associated with MN at genome-wide significance. <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary data, Table S1</a> contains all SNPs on autosomes and chromosome X with minor allele frequency (MAF) &gt;5% and corrected P &lt; 10<sup>−5</sup>.</p><div id="s3a1" class="sec sec-last"><p></p><h4 id="s3a1title" class="inline">Secondary analyses </h4><p id="__p23" class="p p-first">To further examine the genetic architecture of MN, several secondary analyses were carried out: first, additional GWAS were conducted separately for men (222 cases&nbsp;and 106 controls) and women (101 cases&nbsp;and 239 controls). These analyses were carried out because of the strong observed male predilection for MN, although the smaller sample sizes limit statistical power. No additional signals of genome-wide significance were identified in either men or women (data not shown). Second, we evaluated whether a recessive model instead of an additive model, which is typically used in discovery GWAS, better fit the data. Consistent with lower statistical power of a recessive model, P-values were larger than those obtained from the additive model for both <em>PLA2R1</em> and <em>HLA-DQA1</em>. No other genomic regions contained significantly associated markers (data not shown).</p><p id="__p24">We evaluated the identified regions containing <em>PLA2R1</em> and <em>HLA-DQA1</em> more closely in order to assess the presence of more than one independent signal of genome-wide significance. After conditioning on the index SNP of the corresponding region, no additional SNP across the respective chromosome was associated with MN at genome-wide significance. Thus, based on the variants assessed in this study, there was no evidence of multiple common independent variants in the known regions that contributed to the observed associations at genome-wide significance.</p><p id="__p25" class="p p-last">Finally, a very recent publication by Gbadegesin <em>et al</em>. [<a href="#gfw001C22" rid="gfw001C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227990">22</a>] reported on the association of an <em>HLA-DQA1</em> risk allele and steroid-sensitive nephrotic syndrome (SSNS) in children. We, therefore, assessed whether the reported index variant, rs1129740, was also associated with MN in our genome-wide data. The reported risk allele for SSNS, the major A allele, was significantly associated with MN risk in our data {odds ratio [OR] = 2.48 [95% confidence interval (CI) 1.81–3.40], P = 4.45 × 10<sup>−9</sup>}. The OR for MN was similar to the one published for SSNS. The SSNS index variant rs1129740 was correlated with the MN index SNP evaluated in this study, rs9272729 (<em>r</em><sup>2</sup> = 0.19, <em>D</em>′ = 1, based on the MN GWAS data). Besides MN, the SSNS risk allele at rs1129740 was also nominally associated with minimal change disease [OR = 1.64 (95% CI 1.09–2.45), P = 0.014] and FSGS [OR = 1.52 (95% CI 1.16–1.98), P = 0.002] in the GCKD study.</p></div></div><div id="s3b" class="sec"><h3 id="s3btitle">Replication and interactions</h3><div id="s3b1" class="sec sec-first"><p></p><h4 id="s3b1title" class="inline">Replication of SNP associations and meta-analysis </h4><p id="__p26" class="p p-first-last">We assessed evidence for replication of the association at the index variants in <em>PLA2R1</em> and <em>HLA-DQA1</em> in 137 participants of the GCKD study with biopsy-proven MN and 379 ethnically matched controls (see Materials and Methods). Both index SNPs were associated with MN after correction for multiple testing in this replication sample (Table&nbsp;<a href="/pmc/articles/PMC5837679/table/gfw001TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="gfw001TB1" rid-ob="ob-gfw001TB1" co-legend-rid=""><span>1</span></a>; P = 5.0 × 10<sup>−4</sup> for <em>PLA2R1</em> and 6.4 × 10<sup>−24</sup> for <em>HLA-DQA1</em>). The ORs for MN after meta-analysis of the discovery and replication samples were 1.87 (95% CI 1.54–2.28) for <em>PLA2R1</em> and 7.07 (95% CI 5.28–9.47) for <em>HLA-DQA1</em> for each additional copy of a risk allele.</p></div><div id="s3b2" class="sec sec-last"><p></p><h4 id="s3b2title" class="inline">Combined allelic effects and interactions </h4><p id="__p27" class="p p-first">Figure&nbsp;<a href="/pmc/articles/PMC5837679/figure/gfw001F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="gfw001F2" rid-ob="ob-gfw001F2" co-legend-rid="lgnd_gfw001F2"><span>2</span></a> presents the association with MN for the combination of risk alleles at <em>PLA2R1</em> and <em>HLA-DQA1</em>. The risk for MN increased dramatically with a higher number of risk alleles: almost all persons carrying two risk alleles at <em>HLA-DQA1</em> were affected with MN [14 of 16 (88%)] and all three individuals who carried two risk alleles at both loci were affected with MN (so that no risk estimates could be provided; see arrow in Figure&nbsp;<a href="/pmc/articles/PMC5837679/figure/gfw001F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="gfw001F2" rid-ob="ob-gfw001F2" co-legend-rid="lgnd_gfw001F2"><span>2</span></a>). The second-highest risk of MN was observed for individuals carrying two risk alleles at <em>HLA-DQA1</em> and one risk allele at <em>PLA2R1</em>. Compared with individuals who did not carry any risk alleles, their MN risk was increased almost 80-fold (OR = 79.4, P = 7.1 × 10<sup>−5</sup>). <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary data, Table S2</a> displays case and control numbers for all risk categories and, to obtain more stable numerical estimates, the corresponding results when combining individuals carrying one or two risk alleles at the <em>HLA-DQA1</em> locus into a single category.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="gfw001F2" co-legend-rid="lgnd_gfw001F2"><a href="/pmc/articles/PMC5837679/figure/gfw001F2/" target="figure" rid-figpopup="gfw001F2" rid-ob="ob-gfw001F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139841095435808" class="figure"><a href="/pmc/articles/PMC5837679/figure/gfw001F2/" target="figure" rid-figpopup="gfw001F2" rid-ob="ob-gfw001F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5837679_gfw00102.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gfw00102.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5837679/bin/gfw00102.jpg"></a></div><div id="largeobj_idm139841095435808" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5837679/figure/gfw001F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_gfw001F2"><div><a class="figpopup" href="/pmc/articles/PMC5837679/figure/gfw001F2/" target="figure" rid-figpopup="gfw001F2" rid-ob="ob-gfw001F2">FIGURE&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p28">Combined effect of genetic risk variants at replicated loci on risk of MN in the GCKD study. See Supplementary data, Table S2 for tabular presentation of the results.</p></div></div></div><p id="__p29" class="p p-last">Evidence for statistical interaction between the two index variants at <em>PLA2R1</em> and <em>HLA-DQA1</em> was assessed by simultaneous inclusion of the variants as well as of an interaction term into the regression model (see Materials and Methods). Compared with a model containing both variants but no interaction term, the inclusion of the interaction term lead to a decrease of Akaike's information criterion (AIC) from 491.9 to 487.2, which was statistically significant (likelihood ratio test P-value = 0.009). The OR of the interaction term in this model was 2.20 (P = 0.01), which together with the lower AIC supports the finding that the combination of risk alleles increases MN risk to a greater degree than expected based on the effect of the individual risk alleles at the two loci by themselves.</p></div></div><div id="s3c" class="sec"><h3 id="s3ctitle">Characterization across CKD aetiologies reveals shared associations at <em>HLA-DQA1</em></h3><p id="__p30" class="p p-first-last">Subsequently we evaluated whether the association at <em>PLA2R1</em> and <em>HLA-DQA1</em> is specific to MN or whether it extends to additional aetiologies of CKD in the large GCKD study, which collected information on the underlying cause of CKD (see Materials and Methods). Table&nbsp;<a href="/pmc/articles/PMC5837679/table/gfw001TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="gfw001TB2" rid-ob="ob-gfw001TB2" co-legend-rid=""><span>2</span></a> shows the association of the two index variants with CKD defined based on eGFR and/or albuminuria, as well as with 17 specific aetiologies of CKD. <em>PLA2R1</em> was only associated with MN but no other CKD aetiology after correction for multiple testing. Conversely, a significant association of the <em>HLA-DQA1</em> risk variant was observed not only with MN, but also with lupus nephritis (P = 2.8 × 10<sup>−6</sup>), CKD in type 1 diabetes mellitus (P = 6.9 × 10<sup>−5</sup>) and FSGS (P = 5.1 × 10<sup>−5</sup>). The association with FSGS remained significant even after the exclusion of eight patients with HIV and hepatitis, potential causes of secondary FSGS (P = 2.3 × 10<sup>−5</sup>). The observed association with CKD (P = 1.3 × 10<sup>−3</sup>) was mainly due to the inclusion of patients with MN, lupus nephritis, type 1 diabetes mellitus and FSGS.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="gfw001TB2"><h3>Table&nbsp;2</h3><!--caption a7--><div class="caption"><p id="__p31">Association of replicated SNPs with different CKD aetiologies in the GCKD study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139841101857072" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">CKD aetiology (leading cause)</th><th rowspan="2" colspan="1" align="left"><em>n</em></th><th rowspan="2" colspan="1" align="left">% biopsy proven</th><th colspan="3" rowspan="1" align="left">rs17830558 (<em>PLA2R1</em>)<hr></th><th colspan="3" rowspan="1" align="left">rs9272729 (<em>HLA-DQA1</em>)<sup>a</sup><hr></th></tr><tr><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left">95% CI</th><th rowspan="1" colspan="1" align="left">P-value</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left">95% CI</th><th rowspan="1" colspan="1" align="left">P-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">MN<sup>b</sup></td><td rowspan="1" colspan="1">137</td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">1.63</td><td rowspan="1" colspan="1">1.23–2.15</td><td rowspan="1" colspan="1"><strong>5.0 × 10<sup>−4</sup></strong></td><td rowspan="1" colspan="1">6.86</td><td rowspan="1" colspan="1">4.55–10.34</td><td rowspan="1" colspan="1"><strong>6.4 × 10<sup>−24</sup></strong></td></tr><tr><td rowspan="1" colspan="1">Lupus nephritis<sup>b</sup></td><td rowspan="1" colspan="1">104</td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">1.10</td><td rowspan="1" colspan="1">0.82–1.47</td><td rowspan="1" colspan="1">5.2 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">3.00</td><td rowspan="1" colspan="1">1.90–4.72</td><td rowspan="1" colspan="1"><strong>2.8 × 10<sup>−6</sup></strong></td></tr><tr><td rowspan="1" colspan="1">Diabetes mellitus, type 1</td><td rowspan="1" colspan="1">89</td><td rowspan="1" colspan="1">3.4</td><td rowspan="1" colspan="1">0.76</td><td rowspan="1" colspan="1">0.55–1.05</td><td rowspan="1" colspan="1">9.2 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">2.69</td><td rowspan="1" colspan="1">1.67–4.34</td><td rowspan="1" colspan="1"><strong>6.9 × 10<sup>−5</sup></strong></td></tr><tr><td rowspan="1" colspan="1">FSGS<sup>b</sup></td><td rowspan="1" colspan="1">139</td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">0.87</td><td rowspan="1" colspan="1">0.66–1.14</td><td rowspan="1" colspan="1">3.1 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">2.36</td><td rowspan="1" colspan="1">1.56–3.57</td><td rowspan="1" colspan="1"><strong>5.1 × 10<sup>−5</sup></strong></td></tr><tr><td rowspan="1" colspan="1">Membranoproliferative GN<sup>b</sup></td><td rowspan="1" colspan="1">36</td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">1.36</td><td rowspan="1" colspan="1">0.85–2.19</td><td rowspan="1" colspan="1">1.9 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">2.15</td><td rowspan="1" colspan="1">1.10–4.22</td><td rowspan="1" colspan="1">3.5 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1">Minimal change GN<sup>b</sup></td><td rowspan="1" colspan="1">53</td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">1.30</td><td rowspan="1" colspan="1">0.87–1.94</td><td rowspan="1" colspan="1">2.0 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.70</td><td rowspan="1" colspan="1">0.90–3.22</td><td rowspan="1" colspan="1">1.2 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">Interstitial nephritis</td><td rowspan="1" colspan="1">135</td><td rowspan="1" colspan="1">19.3</td><td rowspan="1" colspan="1">0.98</td><td rowspan="1" colspan="1">0.75–1.29</td><td rowspan="1" colspan="1">9.1 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.56</td><td rowspan="1" colspan="1">1.00–2.43</td><td rowspan="1" colspan="1">5.3 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1">Nephrosclerosis</td><td rowspan="1" colspan="1">1087</td><td rowspan="1" colspan="1">7.9</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">0.82–1.15</td><td rowspan="1" colspan="1">7.4 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.47</td><td rowspan="1" colspan="1">1.10–1.96</td><td rowspan="1" colspan="1">7.8 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1">Diabetes mellitus, type 2</td><td rowspan="1" colspan="1">649</td><td rowspan="1" colspan="1">4.2</td><td rowspan="1" colspan="1">0.89</td><td rowspan="1" colspan="1">0.74–1.07</td><td rowspan="1" colspan="1">2.0 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.38</td><td rowspan="1" colspan="1">1.02–1.87</td><td rowspan="1" colspan="1">3.3 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1">Renal artery stenosis</td><td rowspan="1" colspan="1">47</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">1.32</td><td rowspan="1" colspan="1">0.87–2.00</td><td rowspan="1" colspan="1">1.9 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.29</td><td rowspan="1" colspan="1">0.64–2.62</td><td rowspan="1" colspan="1">4.9 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">Wegener's granulomatosis<sup>b</sup></td><td rowspan="1" colspan="1">77</td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">0.93</td><td rowspan="1" colspan="1">0.67–1.31</td><td rowspan="1" colspan="1">6.9 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.18</td><td rowspan="1" colspan="1">0.66–2.13</td><td rowspan="1" colspan="1">5.8 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">IgA nephropathy<sup>b</sup></td><td rowspan="1" colspan="1">308</td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">1.02</td><td rowspan="1" colspan="1">0.83–1.25</td><td rowspan="1" colspan="1">8.6 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1">0.79–1.63</td><td rowspan="1" colspan="1">4.8 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">Autosomal dominant polycystic kidney disease</td><td rowspan="1" colspan="1">170</td><td rowspan="1" colspan="1">1.8</td><td rowspan="1" colspan="1">1.07</td><td rowspan="1" colspan="1">0.83–1.37</td><td rowspan="1" colspan="1">6.2 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.13</td><td rowspan="1" colspan="1">0.73–1.77</td><td rowspan="1" colspan="1">5.8 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">Tumour nephrectomy</td><td rowspan="1" colspan="1">61</td><td rowspan="1" colspan="1">11.5</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">0.63–1.32</td><td rowspan="1" colspan="1">6.2 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.07</td><td rowspan="1" colspan="1">0.55–2.08</td><td rowspan="1" colspan="1">8.4 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">Analgesic nephropathy</td><td rowspan="1" colspan="1">52</td><td rowspan="1" colspan="1">3.8</td><td rowspan="1" colspan="1">1.12</td><td rowspan="1" colspan="1">0.76–1.67</td><td rowspan="1" colspan="1">5.6 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.03</td><td rowspan="1" colspan="1">0.49–2.16</td><td rowspan="1" colspan="1">9.4 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">Acute kidney injury</td><td rowspan="1" colspan="1">59</td><td rowspan="1" colspan="1">10.2</td><td rowspan="1" colspan="1">0.90</td><td rowspan="1" colspan="1">0.61–1.32</td><td rowspan="1" colspan="1">5.9 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.02</td><td rowspan="1" colspan="1">0.50–2.08</td><td rowspan="1" colspan="1">9.5 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">Micropolyangiitis<sup>b</sup></td><td rowspan="1" colspan="1">54</td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">0.82</td><td rowspan="1" colspan="1">0.55–1.22</td><td rowspan="1" colspan="1">3.3 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">0.52</td><td rowspan="1" colspan="1">0.20–1.33</td><td rowspan="1" colspan="1">1.4 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">Rapidly progressive GN, pauci-immune<sup>b</sup></td><td rowspan="1" colspan="1">25</td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">0.72</td><td rowspan="1" colspan="1">0.41–1.29</td><td rowspan="1" colspan="1">2.7 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">0.11–1.88</td><td rowspan="1" colspan="1">2.2 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">KDIGO-defined CKD (all GCKD participants)</td><td rowspan="1" colspan="1">4960</td><td rowspan="1" colspan="1">25.8</td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.86–1.15</td><td rowspan="1" colspan="1">9.1 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">1.50</td><td rowspan="1" colspan="1">1.16–1.94</td><td rowspan="1" colspan="1"><strong>1.3 × 10<sup>−3</sup></strong></td></tr></tbody></table></div><div id="largeobj_idm139841101857072" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5837679/table/gfw001TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn5"><p id="__p32" class="p p-first-last">P-values that are statistically significant after correction for multiple testing are in bold type (P-value &lt; 2.6 × 10<sup>−3</sup> = 0.05/19).</p></div><div id="tblfn6"><p id="__p33" class="p p-first-last"><sup>a</sup>In GCKD, the proxy SNP rs2187668 was genotyped.</p></div><div id="tblfn7"><p id="__p34" class="p p-first-last"><sup>b</sup>Analysis based on patients with biopsy-proven (leading) cause only because the proportion of available biopsy results was &gt;70%.</p></div></div></div></div><div id="s3d" class="sec sec-last"><h3 id="s3dtitle">Associations with classical HLA alleles</h3><p id="__p35" class="p p-first">Lastly, we attempted to assess whether MN showed stronger associations with carrier status of classical HLA alleles than with the index variant identified in the GWAS, and thus imputed classical HLA alleles from the genotyped SNPs in the discovery sample (see Materials and Methods). MN was significantly associated with seven of the classical HLA alleles at genome-wide significance (Table&nbsp;<a href="/pmc/articles/PMC5837679/table/gfw001TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="gfw001TB3" rid-ob="ob-gfw001TB3" co-legend-rid=""><span>3</span></a>). Whereas the lowest P-value was observed for HLA-DRB1*0301 carrier status (OR = 6.15, P = 1.6 × 10<sup>−28</sup>), HLA-DQA1*0501 conferred the largest risk (OR = 6.23, P = 1.4 × 10<sup>−27</sup>). Carrier status at these two HLA alleles was almost perfectly correlated (Spearman <em>r</em> = 0.995), indicating a long shared risk haplotype, to which HLA-DQB1*0201 also belongs (Spearman correlation with HLA-DQA1*0501 <em>r</em> = 1). <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary data, Table S3</a> presents the associations of all imputed common HLA alleles (MAF &gt;5%) with MN.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="gfw001TB3"><h3>Table&nbsp;3</h3><!--caption a7--><div class="caption"><p id="__p36">Association of imputed classical HLA alleles with MN</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139841101525200" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">HLA allele</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left">95% CI</th><th rowspan="1" colspan="1" align="left">P-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">DRB1-0301</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">6.15</td><td rowspan="1" colspan="1">4.27–8.85</td><td rowspan="1" colspan="1">1.6 × 10<sup>−28</sup></td></tr><tr><td rowspan="1" colspan="1">DQA1-0501</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">6.23</td><td rowspan="1" colspan="1">4.29–9.04</td><td rowspan="1" colspan="1">1.4 × 10<sup>−27</sup></td></tr><tr><td rowspan="1" colspan="1">DQB1-0201</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">5.89</td><td rowspan="1" colspan="1">4.11–8.45</td><td rowspan="1" colspan="1">1.5 × 10<sup>−27</sup></td></tr><tr><td rowspan="1" colspan="1">B-0801</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">4.47</td><td rowspan="1" colspan="1">3.16–6.33</td><td rowspan="1" colspan="1">7.5 × 10<sup>−20</sup></td></tr><tr><td rowspan="1" colspan="1">DRB3-0100</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">3.98</td><td rowspan="1" colspan="1">2.83–5.59</td><td rowspan="1" colspan="1">1.3 × 10<sup>−17</sup></td></tr><tr><td rowspan="1" colspan="1">DRB3-9901</td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">0.21–0.39</td><td rowspan="1" colspan="1">1.6 × 10<sup>−17</sup></td></tr><tr><td rowspan="1" colspan="1">C-0701</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">2.94</td><td rowspan="1" colspan="1">2.14–4.03</td><td rowspan="1" colspan="1">2.4 × 10<sup>−12</sup></td></tr></tbody></table></div><div id="largeobj_idm139841101525200" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5837679/table/gfw001TB3/?report=objectonly">Open in a separate window</a></div></div><p id="__p37" class="p p-last">Using the classical HLA allele HLA-DQA1*0501 as a proxy for this risk haplotype, we assessed whether the index SNP at <em>HLA-DQA1</em> in the discovery GWAS, rs9272729, provided independent information. Individually, the association of rs9272729 with MN was of similar magnitude as the association of HLA-DQA1*0501 with MN: OR = 6.51 (P = 7.16 × 10<sup>−19</sup>) for the SNP versus OR = 5.87 (P = 1.25 × 10<sup>−18</sup>) for the HLA allele among individuals with both types of information available. Upon simultaneous inclusion of rs9272729 and HLA-DQA1*0501 into the model, the effects of both variables were attenuated but still significant. There was a significant improvement of model fit (likelihood ratio test P = 0.02), suggesting that the SNP captures most of the information but other variants in the genomic region and/or classical HLA alleles may still add some additional information.</p></div></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">DISCUSSION</h2><p id="__p38" class="p p-first">In this first GWAS of MN using imputed genotypes, we confirm the predominant importance of risk alleles at <em>PLA2R1</em> and <em>HLA-DQA1</em> in modulating MN risk. With the available number of cases, we did not find evidence for additional or independent common risk alleles or of sex-specific risk loci. While we found that genetic variation in <em>PLA2R1</em> exclusively associated with MN across a wide range of CKD aetiologies studied, the risk allele at <em>HLA-DQA1</em> was also associated with higher odds of a number of other autoimmunity-associated kidney diseases—lupus nephritis, type 1 diabetic nephropathy and FSGS—but not with immunoglobulin A (IgA) nephropathy.</p><p id="__p39">In accordance with other studies [<a href="#gfw001C23" rid="gfw001C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228004">23</a>], our results from the GCKD study confirmed the association between MN and previously reported risk variants or their proxies in <em>PLA2R1</em> and <em>HLA-DQA1</em>. In addition, in the GWAS of high coverage, we did not identify additional common risk variants of similar importance. While imputation to the 1000 Genomes reference panel has led to the identification of additional risk loci missed in earlier studies of similar size for a variety of traits and diseases [<a href="#gfw001C11" rid="gfw001C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227996">11</a>, <a href="#gfw001C12" rid="gfw001C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227982">12</a>], this was not the case for MN. Our study, therefore, highlights the oligogenic architecture of MN, informs future studies to detect genes encoding for additional antigens and confirms <em>PLA2R1</em> as a risk locus of major and specific importance. This is consistent with the presence of anti-PLA2R1 antibodies in ∼70–80% of MN patients [<a href="#gfw001C1" rid="gfw001C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227986">1</a>]. In the independent GCKD study, we also confirmed the large increase in MN risk conferred by the combination of risk variants at <em>PLA2R1</em> and <em>HLA-DQA1</em>, as well as a statistical interaction between the risk variants at both loci [<a href="#gfw001C24" rid="gfw001C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228011">24</a>]. In fact, all GCKD patients who were homozygous for the risk alleles at both loci were affected with MN, as were the great majority of persons homozygous for the risk allele at <em>HLA-DQA1</em> and heterozygous at <em>PLA2R1</em>. There are several potential explanations for the higher proportion of MN patients among carriers of high-risk genotype combinations in the GCKD study compared with the initial study [<a href="#gfw001C7" rid="gfw001C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227993">7</a>], including random fluctuations due to small numbers, differences in the prevalence of contributing environmental triggers and recruitment of patients under nephrological care with reduced eGFR into the GCKD study, which may increased the number of MN patients with advanced disease and high-risk genotypes. Only long-term prospective studies, if not lifetime studies, will be able to answer how many individuals carrying high-risk genotype combinations will not develop MN over their life course. Likewise, conclusive answers relating to long-term outcomes require standardized observational studies that follow patients for at least 10 years or until death while collecting information on immunosuppressive treatments and other comorbidities.</p><p id="__p40">Imputation of classical HLA alleles revealed the strongest associations with HLA-DQA1*0501, HLA-DRB1*0301 and HLA-DQB1*0201. The almost perfect correlation between these alleles implicates one long shared MN risk haplotype on which these alleles reside, DRB1*0301-DQA1*0501-DQB1*0201. This is supported by the reported finding that circulating anti-PLA2R1-antibody levels in MN cases are linked to the number of alleles of DQA1*0501 and DQB1*0201 [<a href="#gfw001C25" rid="gfw001C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227981">25</a>]. Therefore, although we refer to the HLA risk allele as the <em>HLA-DQA1</em> locus, the underlying causal variant could also be situated in <em>HLA-DQB1</em> or <em>HLA-DRB1</em>. Given the large inter-individual genetic variability of the HLA region, it is noteworthy that our analyses suggest that most of the strong association with MN at this locus can be captured by knowledge of the genotype at a single common SNP in the region, which may have clinical implications for risk prediction. Our observations are consistent with a relatively common genetic predisposition that controls antigen processing and/or presentation, that only a combination of additional factors, such as the presence of specific auto antigen-processed peptides from, for example, specific forms of PLA2R1 for MN—and/or certain environmental triggers can give rise to a specific autoantibody to cause disease [<a href="#gfw001C7" rid="gfw001C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227978">7</a>]. In support of this, the DRB1*0301-DQA1*0501-DQB1*0201 haplotype has also been described to increase the risk for other common autoimmune diseases, including type 1 diabetes [<a href="#gfw001C26" rid="gfw001C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227995">26</a>], systemic lupus erythematosus (SLE) [<a href="#gfw001C27" rid="gfw001C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228010">27</a>], coeliac disease [<a href="#gfw001C28" rid="gfw001C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227977">28</a>] and thyroid disease [<a href="#gfw001C29" rid="gfw001C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227989">29</a>].</p><p id="__p41">Our data indicate that this observation may extend to SSNS. As SSNS is commonly observed in children, it was not among the aetiologies that could be evaluated in the GCKD study. However, the association of the reported SSNS risk allele with MN risk in our GWAS and the high correlation of the risk allele of the reported SSNS variant and the risk allele of the MN index variant in our data suggest that the genetic predisposition at the <em>HLA-DQA1</em> locus shares at least some degree of overlap between SSNS and MN. The results from the GCKD study further suggest that the shared susceptibility between the SSNS variant extends to an association with minimal change disease and FSGS in adulthood. In contrast, there was no statistically significant association between the <em>HLA-DQA1</em> risk allele for MN and IgA nephropathy, another autoimmune kidney disease. This is consistent with the fact that previous GWAS of IgA nephropathy have highlighted other associated alleles in the <em>HLA-DQA1</em> region [<a href="#gfw001C30" rid="gfw001C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227980">30</a>].</p><p id="__p42">While genetic variants in the <em>HLA-DQA1</em> locus were associated with a variety of underlying causes of CKD, those in <em>PLA2R1</em> uniquely associated with MN across almost 20 different aetiologies of CKD. The genetic associations between HLA risk alleles and nephropathy resulting from SLE or type 1 diabetes are biologically plausible, given the shared genetic susceptibility to these autoimmune diseases. Because of the case-only design of the GCKD study, we could not specifically assess whether the HLA risk allele increases the risk for nephropathy in the setting of type 1 diabetes and SLE, or whether the observed associations are due to an association with the primary cause of the respective disease. The mechanisms underlying the observed association with FSGS need further study to delineate whether they are indicative of a common mechanism resulting in FSGS-defining features in the biopsies or whether some of the cases in whom the primary CKD aetiology was assigned as FSGS may in fact have additional specific causes contributing to their nephropathy, such as SLE. Although not significant after correction for multiple testing, the risk allele at <em>HLA-DQA1</em> also more than doubled the odds of membranoproliferative GN. This could be indicative of an autoimmune component specific to membranoproliferative GN or represent cases of secondary membranoproliferative GN such as those observed in SLE. Our approach highlights the power of collecting detailed information on kidney disease phenotypes, as the specific evaluation of associations across disease subtypes can reveal significant associations that would be missed in GWAS of these CKD aetiologies. A similar observation has been made for the <em>APOL1</em> risk alleles, where associations with FSGS and nondiabetic end-stage renal disease—originally identified using genome-wide approaches—were shown to extend to HIV-associated nephropathy without conducting a separate genome-wide genetic screen [<a href="#gfw001C31" rid="gfw001C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670228008">31</a>, <a href="#gfw001C32" rid="gfw001C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227985">32</a>].</p><p id="__p43">Strengths of our study include the availability of high-coverage genome-wide SNP data as well as the variety of CKD aetiologies in the GCKD study that allowed us to evaluate the presence of novel associations, carefully characterize known associations and identify a shared genetic predisposition to several autoimmune kidney diseases. Our study also has potential limitations: genotypes were generated on different platforms, which we addressed through careful data cleaning and combined imputation based on a common SNP set. Although—for a rare disease such as MN—we studied a sizeable number of cases, our study is still underpowered to identify associations with rare alleles or alleles conferring moderate or small increases in risk. For example, we did not find evidence for an association between the <em>THSD7A</em> variant and MN, which may not be surprising given the fact that this autoantigen has only been described in a minority of patients. In addition, because we evaluated chip-based and imputed genotypes, there may be rare risk variants that can only be discovered through sequencing-based approaches. At least for <em>PLA2R1</em>, however, targeted re-sequencing of the exons and canonical splice sites did not yield rare variants that could explain the observed association [<a href="#gfw001C33" rid="gfw001C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_670227988">33</a>]. In our study of prevalent MN cases, we did not have information on PLA2R1 antibody concentrations and could, therefore, not investigate antibody-negative patients separately. In any case, the statistical power to detect novel associated loci among antibody-negative patients in the current study, assuming 20% antibody-negative patients among the cases, would have been &lt;30% even for very common alleles. The assembly of larger collectives of MN patients with antibody-status and genome-wide genotypes is, therefore, an important future endeavour. Likewise, larger collectives of patients with other primary forms of specific CKD aetiologies such as FSGS are needed to confirm our findings or to establish statistical significance such as with membranoproliferative GN. Although in the GCKD study the patients' treating nephrologists were asked to identify the leading cause of CKD, misclassification can occur, especially for the leading causes of CKD that are typically not confirmed by kidney biopsy or for which no specific diagnostic criteria are available. Finally, as in all association studies, we cannot directly pinpoint causal variants, but associated variants may still be valuable for risk stratification.</p><p id="__p44" class="p p-last">In conclusion, <em>PLA2R1</em> and <em>HLA-DQA1</em> are the predominant risk loci for MN. Common variants in these two loci give rise to rare high-risk genotype combinations, which associate with disease risk of a magnitude otherwise observed for single-gene disorders. The risk allele at <em>HLA-DQA1</em>, but not <em>PLA2R1</em>, is also associated with nephropathy attributed to type 1 diabetes, SLE and FSGS, but not with IgA nephropathy, another autoimmune kidney disease. Future studies including those in PLA2R1 antibody–negative patients and sequence-based association studies should evaluate the presence of additional genetic risk variants for MN.</p></div><div id="s5" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY DATA</h2><!--/article/body/sec/--><p id="__p45" class="p p-first-last">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary data</a> are available online at <a href="http://ndt.oxfordjournals.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ndt.oxfordjournals.org</a>.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>Supplementary Data</h4><div class="sup-box half_rhythm" id="idm139841106685424"><a href="/pmc/articles/PMC5837679/bin/gfw001_supp.zip" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.3M, zip)</sup></div></div></div><div id="idm139841106683824" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139841106683824title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p47">P.S. was supported by the BMBF (Gerontosys II NephAge Project, 031 5896 A). The German Research Foundation funded the work of P.S., Y.L., M.W. and A.K. through the Emmy Noether Programme (KO 3598/2-1 to A.K.). The work of M.W., G.W. and A.K. was additionally supported by the German Research Foundation through CRC/SFB 1140.</p><p id="__p48">Work on the British study was supported in part by grants from the David and Elaine Potter Charitable Foundation (to S.H.P. and R.K.); the European Union, FP7 (EURenomics grant agreement 2013-305608; to H.C.S., D.B., S.H.P., P.B. and R.K.); St Peter's Trust for Kidney, Bladder and Prostate Research (to D.B., S.H.P. and R.K.); Kids Kidney Research UK (to D.B. and R.K.); Medical Research Council (MRC) at the Centre for Integrated Genomic Medical Research, University of Manchester; grants from the MRC (G0000934) and the Wellcome Trust (068545/Z/02); Manchester Academic Health Science Centre (MAHSC 186/200) and MRC project grant MR/J010847/1 (to P.B.). We thank the MRC and Kidney Research UK, as part of the MRC/Kidney Research UK National DNA Bank for Glomerulonephritis, for collection of the British idiopathic membranous nephropathy DNA samples.</p><p id="__p49">The GCKD study is funded by grants from the German Ministry of Education and Research (BMBF, grant number 01ER0804) and the KfH Foundation for Preventive Medicine. We are grateful for the willingness of the patients to participate in the GCKD study. The enormous effort of the study personnel of the various regional centres is highly appreciated. We thank the large number of nephrologists who provide routine care for the patients and collaborate with the GCKD study. A list of nephrologists currently collaborating with the GCKD study is available at <a href="http://www.gckd.org">http://www.gckd.org</a>.</p><p id="__p50">GCKD Investigators are as follows:</p><p id="__p51">University of Erlangen-Nürnberg, Germany: Kai-Uwe Eckardt, Stephanie Titze, Hans-Ulrich Prokosch, Barbara Bärthlein, André Reis, Arif B. Ekici, Olaf Gefeller, Karl F. Hilgers, Silvia Hübner, Susanne Avendaño, Dinah Becker-Grosspitsch, Nina Hauck, Susanne A. Seuchter, Birgit Hausknecht, Marion Rittmeier, Anke Weigel, Andreas Beck, Thomas Ganslandt, Sabine Knispel, Thomas Dressel&nbsp;and Martina Malzer.</p><p id="__p52">Technical University of Aachen, Germany: Jürgen Floege, Frank Eitner, Georg Schlieper, Katharina Findeisen, Elfriede Arweiler, Sabine Ernst, Mario Unger&nbsp;and Stefan Lipski.</p><p id="__p53">Charité, Humboldt-University of Berlin, Germany: Elke Schaeffner, Seema Baid-Agrawal, Kerstin Petzold&nbsp;and Ralf Schindler.</p><p id="__p54">University of Freiburg, Germany: Anna Köttgen, Ulla Schultheiss, Simone Meder, Erna Mitsch, Ursula Reinhard&nbsp;and Gerd Walz.</p><p id="__p55">Hannover Medical School, Germany: Hermann Haller, Johan Lorenzen, Jan T. Kielstein&nbsp;and Petra Otto.</p><p id="__p56">University of Heidelberg, Germany: Claudia Sommerer, Claudia Föllinger&nbsp;and Martin Zeier.</p><p id="__p57">University of Jena, Germany: Gunter Wolf, Martin Busch, Katharina Paul&nbsp;and Lisett Dittrich.</p><p id="__p58">Ludwig-Maximilians University of München, Germany: Thomas Sitter, Robert Hilge&nbsp;and Claudia Blank.</p><p id="__p59">University of Würzburg, Germany: Christoph Wanner, Vera Krane, Daniel Schmiedeke, Sebastian Toncar, Daniela Cavitt, Karina Schönowsky&nbsp;and Antje Börner-Klein.</p><p id="__p60">Medical University of Innsbruck, Austria: Florian Kronenberg, Julia Raschenberger, Barbara Kollerits, Lukas Forer, Sebastian Schönherr&nbsp;and Hansi Weißensteiner.</p><p id="__p61">University of Regensburg, Germany: Peter Oefner, Wolfram Gronwald&nbsp;and Helena Zacharias.</p><p id="__p62">Department of Medical Biometry, Informatics and Epidemiology (IMBIE), University of Bonn: Matthias Schmid.</p><div id="s6" class="sec"><h3 id="s6title">CONFLICT OF INTEREST STATEMENT</h3><p id="__p63" class="p p-first-last">None declared.</p></div></div></div><div id="idm139841093880912" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139841093880912title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="gfw001C1">1. <span class="mixed-citation">
Beck LH Jr, 
Salant DJ.
<span class="ref-title">Membranous nephropathy: from models to man</span>. <span class="ref-journal">J Clin Invest</span> 2014; <span class="ref-vol">124</span>: 2307–2314 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4089468/">PMC free article</a>]</span> [<a href="/pubmed/24892704" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Membranous+nephropathy:+from+models+to+man&amp;author=LH+Beck&amp;author=DJ+Salant&amp;volume=124&amp;publication_year=2014&amp;pages=2307-2314&amp;pmid=24892704&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C2">2. <span class="mixed-citation">
Debiec H, 
Ronco P.
<span class="ref-title">Immunopathogenesis of membranous nephropathy: an update</span>. <span class="ref-journal">Semin Immunopathol</span> 2014; <span class="ref-vol">36</span>: 381–397 [<a href="/pubmed/24715030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin+Immunopathol&amp;title=Immunopathogenesis+of+membranous+nephropathy:+an+update&amp;author=H+Debiec&amp;author=P+Ronco&amp;volume=36&amp;publication_year=2014&amp;pages=381-397&amp;pmid=24715030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C3">3. <span class="mixed-citation">
Short CD, 
Feehally J, 
Gokal R et al. 
<span class="ref-title">Familial membranous nephropathy</span>. <span class="ref-journal">Br Med J</span> 1984; <span class="ref-vol">289</span>: 1500. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1443752/">PMC free article</a>]</span> [<a href="/pubmed/6439290" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+Med+J&amp;title=Familial+membranous+nephropathy&amp;author=CD+Short&amp;author=J+Feehally&amp;author=R+Gokal&amp;volume=289&amp;publication_year=1984&amp;pages=1500&amp;pmid=6439290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C4">4. <span class="mixed-citation">
Vaughan RW, 
Demaine AG, 
Welsh KI.
<span class="ref-title">A DQA1 allele is strongly associated with idiopathic membranous nephropathy</span>. <span class="ref-journal">Tissue Antigens</span> 1989; <span class="ref-vol">34</span>: 261–269 [<a href="/pubmed/2576474" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Tissue+Antigens&amp;title=A+DQA1+allele+is+strongly+associated+with+idiopathic+membranous+nephropathy&amp;author=RW+Vaughan&amp;author=AG+Demaine&amp;author=KI+Welsh&amp;volume=34&amp;publication_year=1989&amp;pages=261-269&amp;pmid=2576474&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C5">5. <span class="mixed-citation">
Beck LH Jr, 
Bonegio RG, 
Lambeau G et al. 
<span class="ref-title">M-type phospholipase A2 receptor as target antigen in idiopathic membranous nephropathy</span>. <span class="ref-journal">N Engl J Med</span> 2009; <span class="ref-vol">361</span>: 11–21 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2762083/">PMC free article</a>]</span> [<a href="/pubmed/19571279" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=M-type+phospholipase+A2+receptor+as+target+antigen+in+idiopathic+membranous+nephropathy&amp;author=LH+Beck&amp;author=RG+Bonegio&amp;author=G+Lambeau&amp;volume=361&amp;publication_year=2009&amp;pages=11-21&amp;pmid=19571279&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C6">6. <span class="mixed-citation">
Tomas NM, 
Beck LH Jr, 
Meyer-Schwesinger C et al. 
<span class="ref-title">Thrombospondin type-1 domain-containing 7A in idiopathic membranous nephropathy</span>. <span class="ref-journal">N Engl J Med</span> 2014; <span class="ref-vol">371</span>: 2277–2287 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4278759/">PMC free article</a>]</span> [<a href="/pubmed/25394321" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Thrombospondin+type-1+domain-containing+7A+in+idiopathic+membranous+nephropathy&amp;author=NM+Tomas&amp;author=LH+Beck&amp;author=C+Meyer-Schwesinger&amp;volume=371&amp;publication_year=2014&amp;pages=2277-2287&amp;pmid=25394321&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C7">7. <span class="mixed-citation">
Stanescu HC, 
Arcos-Burgos M, 
Medlar A et al. 
<span class="ref-title">Risk HLA-DQA1 and PLA2R1 alleles in idiopathic membranous nephropathy</span>. <span class="ref-journal">N Engl J Med</span> 2011; <span class="ref-vol">364</span>: 616–626 [<a href="/pubmed/21323541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Risk+HLA-DQA1+and+PLA2R1+alleles+in+idiopathic+membranous+nephropathy&amp;author=HC+Stanescu&amp;author=M+Arcos-Burgos&amp;author=A+Medlar&amp;volume=364&amp;publication_year=2011&amp;pages=616-626&amp;pmid=21323541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C8">8. <span class="mixed-citation">
Leslie S, 
Donnelly P, 
McVean G.
<span class="ref-title">A statistical method for predicting classical HLA alleles from SNP data</span>. <span class="ref-journal">Am J Hum Genet</span> 2008; <span class="ref-vol">82</span>: 48–56 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253983/">PMC free article</a>]</span> [<a href="/pubmed/18179884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+statistical+method+for+predicting+classical+HLA+alleles+from+SNP+data&amp;author=S+Leslie&amp;author=P+Donnelly&amp;author=G+McVean&amp;volume=82&amp;publication_year=2008&amp;pages=48-56&amp;pmid=18179884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C9">9. <span class="mixed-citation">
Marchini J, 
Howie B.
<span class="ref-title">Genotype imputation for genome-wide association studies</span>. <span class="ref-journal">Nat Rev Genet</span> 2010; <span class="ref-vol">11</span>: 499–511 [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J+Marchini&amp;author=B+Howie&amp;volume=11&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C10">10. <span class="mixed-citation">
Fernando MM, 
Vyse TJ.
<span class="ref-title">Risk alleles in idiopathic membranous nephropathy</span>. <span class="ref-journal">N Engl J Med</span> 2011; <span class="ref-vol">364</span>: 2072; author reply 2073–2074 [<a href="/pubmed/21612489" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Risk+alleles+in+idiopathic+membranous+nephropathy&amp;author=MM+Fernando&amp;author=TJ+Vyse&amp;volume=364&amp;publication_year=2011&amp;pages=2072&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C11">11. <span class="mixed-citation">
Horikoshi M, 
Mgi R, 
van de Bunt M et al. 
<span class="ref-title">Discovery and fine-mapping of glycaemic and obesity-related trait loci using high-density imputation</span>. <span class="ref-journal">PLoS Genet</span> 2015; <span class="ref-vol">11</span>: e1005230. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4488845/">PMC free article</a>]</span> [<a href="/pubmed/26132169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Discovery+and+fine-mapping+of+glycaemic+and+obesity-related+trait+loci+using+high-density+imputation&amp;author=M+Horikoshi&amp;author=R+Mgi&amp;author=M+van+de+Bunt&amp;volume=11&amp;publication_year=2015&amp;pages=e1005230&amp;pmid=26132169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C12">12. <span class="mixed-citation">
Consortium CAD. 
<span class="ref-title">A comprehensive 1000 Genomes-based genome-wide association meta-analysis of coronary artery disease</span>. <span class="ref-journal">Nat Genet</span> 2015; <span class="ref-vol">47</span>: 1121–1130 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4589895/">PMC free article</a>]</span> [<a href="/pubmed/26343387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+comprehensive+1000+Genomes-based+genome-wide+association+meta-analysis+of+coronary+artery+disease&amp;volume=47&amp;publication_year=2015&amp;pages=1121-1130&amp;pmid=26343387&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C13">13. <span class="mixed-citation">
Kiryluk K.
<span class="ref-title">Risk alleles in idiopathic membranous nephropathy</span>. <span class="ref-journal">N Engl J Med</span> 2011; <span class="ref-vol">364</span>: 2072–2073; author reply 2073–2074 [<a href="/pubmed/21612488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Risk+alleles+in+idiopathic+membranous+nephropathy&amp;author=K+Kiryluk&amp;volume=364&amp;publication_year=2011&amp;pages=2072-2073&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C14">14. <span class="mixed-citation">
Kidney Disease: Improving Global Outcomes (KDIGO) CKD Work Group. 
<span class="ref-title">KDIGO 2012 clinical practice guideline for the evaluation and management of chronic kidney disease</span>. <span class="ref-journal">Kidney Int Suppl</span> 2013; <span class="ref-vol">3</span>: 1–150 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int+Suppl&amp;title=KDIGO+2012+clinical+practice+guideline+for+the+evaluation+and+management+of+chronic+kidney+disease&amp;volume=3&amp;publication_year=2013&amp;pages=1-150&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C15">15. <span class="mixed-citation">
Eckardt KU, 
Barthlein B, 
Baid-Agrawal S et al. 
<span class="ref-title">The German Chronic Kidney Disease (GCKD) study: design and methods</span>. <span class="ref-journal">Nephrol Dial Transplant</span> 2012; <span class="ref-vol">27</span>: 1454–1460 [<a href="/pubmed/21862458" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephrol+Dial+Transplant&amp;title=The+German+Chronic+Kidney+Disease+(GCKD)+study:+design+and+methods&amp;author=KU+Eckardt&amp;author=B+Barthlein&amp;author=S+Baid-Agrawal&amp;volume=27&amp;publication_year=2012&amp;pages=1454-1460&amp;pmid=21862458&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C16">16. <span class="mixed-citation">
Titze S, 
Schmid M, 
Kottgen A et al. 
<span class="ref-title">Disease burden and risk profile in referred patients with moderate chronic kidney disease: composition of the German Chronic Kidney Disease (GCKD) cohort</span>. <span class="ref-journal">Nephrol Dial Transplant</span> 2015; <span class="ref-vol">30</span>: 441–451 [<a href="/pubmed/25271006" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephrol+Dial+Transplant&amp;title=Disease+burden+and+risk+profile+in+referred+patients+with+moderate+chronic+kidney+disease:+composition+of+the+German+Chronic+Kidney+Disease+(GCKD)+cohort&amp;author=S+Titze&amp;author=M+Schmid&amp;author=A+Kottgen&amp;volume=30&amp;publication_year=2015&amp;pages=441-451&amp;pmid=25271006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C17">17. <span class="mixed-citation">
Purcell S, 
Neale B, 
Todd-Brown K et al. 
<span class="ref-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <span class="ref-journal">Am J Hum Genet</span> 2007; <span class="ref-vol">81</span>: 559–575 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C18">18. <span class="mixed-citation">
Price AL, 
Patterson NJ, 
Plenge RM et al. 
<span class="ref-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. <span class="ref-journal">Nat Genet</span> 2006; <span class="ref-vol">38</span>: 904–909 [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C19">19. <span class="mixed-citation">
Delaneau O, 
Marchini J, 
Zagury JF.
<span class="ref-title">A linear complexity phasing method for thousands of genomes</span>. <span class="ref-journal">Nat Methods</span> 2012; <span class="ref-vol">9</span>: 179–181 [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes&amp;author=O+Delaneau&amp;author=J+Marchini&amp;author=JF+Zagury&amp;volume=9&amp;publication_year=2012&amp;pages=179-181&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C20">20. <span class="mixed-citation">
Howie BN, 
Donnelly P, 
Marchini J.
<span class="ref-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies</span>. <span class="ref-journal">PLoS Genet</span> 2009; <span class="ref-vol">5</span>: e1000529. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C21">21. <span class="mixed-citation">
Marchini J, 
Howie B, 
Myers S et al. 
<span class="ref-title">A new multipoint method for genome-wide association studies by imputation of genotypes</span>. <span class="ref-journal">Nat Genet</span> 2007; <span class="ref-vol">39</span>: 906–913 [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C22">22. <span class="mixed-citation">
Gbadegesin RA, 
Adeyemo A, 
Webb NJ et al. 
<span class="ref-title">HLA-DQA1 and PLCG2 are candidate risk loci for childhood-onset steroid-sensitive nephrotic syndrome</span>. <span class="ref-journal">J Am Soc Nephrol</span> 2015; <span class="ref-vol">26</span>: 1701–1710 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4483579/">PMC free article</a>]</span> [<a href="/pubmed/25349203" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=HLA-DQA1+and+PLCG2+are+candidate+risk+loci+for+childhood-onset+steroid-sensitive+nephrotic+syndrome&amp;author=RA+Gbadegesin&amp;author=A+Adeyemo&amp;author=NJ+Webb&amp;volume=26&amp;publication_year=2015&amp;pages=1701-1710&amp;pmid=25349203&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C23">23. <span class="mixed-citation">
Bullich G, 
Ballarin J, 
Oliver A et al. 
<span class="ref-title">HLA-DQA1 and PLA2R1 polymorphisms and risk of idiopathic membranous nephropathy</span>. <span class="ref-journal">Clin J Am Soc Nephrol</span> 2014; <span class="ref-vol">9</span>: 335–343 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3913240/">PMC free article</a>]</span> [<a href="/pubmed/24262501" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+J+Am+Soc+Nephrol&amp;title=HLA-DQA1+and+PLA2R1+polymorphisms+and+risk+of+idiopathic+membranous+nephropathy&amp;author=G+Bullich&amp;author=J+Ballarin&amp;author=A+Oliver&amp;volume=9&amp;publication_year=2014&amp;pages=335-343&amp;pmid=24262501&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C24">24. <span class="mixed-citation">
Lv J, 
Hou W, 
Zhou X et al. 
<span class="ref-title">Interaction between PLA2R1 and HLA-DQA1 variants associates with anti-PLA2R antibodies and membranous nephropathy</span>. <span class="ref-journal">J Am Soc Nephrol</span> 2013; <span class="ref-vol">24</span>: 1323–1329 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3736719/">PMC free article</a>]</span> [<a href="/pubmed/23813219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=Interaction+between+PLA2R1+and+HLA-DQA1+variants+associates+with+anti-PLA2R+antibodies+and+membranous+nephropathy&amp;author=J+Lv&amp;author=W+Hou&amp;author=X+Zhou&amp;volume=24&amp;publication_year=2013&amp;pages=1323-1329&amp;pmid=23813219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C25">25. <span class="mixed-citation">
Kanigicherla D, 
Gummadova J, 
McKenzie EA et al. 
<span class="ref-title">Anti-PLA2R antibodies measured by ELISA predict long-term outcome in a prevalent population of patients with idiopathic membranous nephropathy</span>. <span class="ref-journal">Kidney Int</span> 2013; <span class="ref-vol">83</span>: 940–948 [<a href="/pubmed/23364522" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Anti-PLA2R+antibodies+measured+by+ELISA+predict+long-term+outcome+in+a+prevalent+population+of+patients+with+idiopathic+membranous+nephropathy&amp;author=D+Kanigicherla&amp;author=J+Gummadova&amp;author=EA+McKenzie&amp;volume=83&amp;publication_year=2013&amp;pages=940-948&amp;pmid=23364522&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C26">26. <span class="mixed-citation">
Erlich H, 
Valdes AM, 
Noble J et al. 
<span class="ref-title">HLA DR-DQ haplotypes and genotypes and type 1 diabetes risk: analysis of the type 1 diabetes genetics consortium families</span>. <span class="ref-journal">Diabetes</span> 2008; <span class="ref-vol">57</span>: 1084–1092 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4103420/">PMC free article</a>]</span> [<a href="/pubmed/18252895" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=HLA+DR-DQ+haplotypes+and+genotypes+and+type+1+diabetes+risk:+analysis+of+the+type+1+diabetes+genetics+consortium+families&amp;author=H+Erlich&amp;author=AM+Valdes&amp;author=J+Noble&amp;volume=57&amp;publication_year=2008&amp;pages=1084-1092&amp;pmid=18252895&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C27">27. <span class="mixed-citation">
Fernando MM, 
Stevens CR, 
Sabeti PC et al. 
<span class="ref-title">Identification of two independent risk factors for lupus within the MHC in United Kingdom families</span>. <span class="ref-journal">PLoS Genet</span> 2007; <span class="ref-vol">3</span>: e192. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2065882/">PMC free article</a>]</span> [<a href="/pubmed/17997607" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Identification+of+two+independent+risk+factors+for+lupus+within+the+MHC+in+United+Kingdom+families&amp;author=MM+Fernando&amp;author=CR+Stevens&amp;author=PC+Sabeti&amp;volume=3&amp;publication_year=2007&amp;pages=e192&amp;pmid=17997607&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C28">28. <span class="mixed-citation">
Spurkland A, 
Sollid LM, 
Ronningen KS et al. 
<span class="ref-title">Susceptibility to develop celiac disease is primarily associated with HLA-DQ alleles</span>. <span class="ref-journal">Hum Immunol</span> 1990; <span class="ref-vol">29</span>: 157–165 [<a href="/pubmed/1980918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Immunol&amp;title=Susceptibility+to+develop+celiac+disease+is+primarily+associated+with+HLA-DQ+alleles&amp;author=A+Spurkland&amp;author=LM+Sollid&amp;author=KS+Ronningen&amp;volume=29&amp;publication_year=1990&amp;pages=157-165&amp;pmid=1980918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C29">29. <span class="mixed-citation">
Yanagawa T, 
Mangklabruks A, 
Chang YB et al. 
<span class="ref-title">Human histocompatibility leukocyte antigen-DQA1*0501 allele associated with genetic susceptibility to Graves’ disease in a Caucasian population</span>. <span class="ref-journal">J Clin Endocrinol Metab</span> 1993; <span class="ref-vol">76</span>: 1569–1574 [<a href="/pubmed/8501164" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Endocrinol+Metab&amp;title=Human+histocompatibility+leukocyte+antigen-DQA1*0501+allele+associated+with+genetic+susceptibility+to+Graves’+disease+in+a+Caucasian+population&amp;author=T+Yanagawa&amp;author=A+Mangklabruks&amp;author=YB+Chang&amp;volume=76&amp;publication_year=1993&amp;pages=1569-1574&amp;pmid=8501164&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C30">30. <span class="mixed-citation">
Gharavi AG, 
Kiryluk K, 
Choi M et al. 
<span class="ref-title">Genome-wide association study identifies susceptibility loci for IgA nephropathy</span>. <span class="ref-journal">Nat Genet</span> 2011; <span class="ref-vol">43</span>: 321–327 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3412515/">PMC free article</a>]</span> [<a href="/pubmed/21399633" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+susceptibility+loci+for+IgA+nephropathy&amp;author=AG+Gharavi&amp;author=K+Kiryluk&amp;author=M+Choi&amp;volume=43&amp;publication_year=2011&amp;pages=321-327&amp;pmid=21399633&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C31">31. <span class="mixed-citation">
Behar DM, 
Kedem E, 
Rosset S et al. 
<span class="ref-title">Absence of APOL1 risk variants protects against HIV-associated nephropathy in the Ethiopian population</span>. <span class="ref-journal">Am J Nephrol</span> 2011; <span class="ref-vol">34</span>: 452–459 [<a href="/pubmed/21968148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Nephrol&amp;title=Absence+of+APOL1+risk+variants+protects+against+HIV-associated+nephropathy+in+the+Ethiopian+population&amp;author=DM+Behar&amp;author=E+Kedem&amp;author=S+Rosset&amp;volume=34&amp;publication_year=2011&amp;pages=452-459&amp;pmid=21968148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C32">32. <span class="mixed-citation">
Kopp JB, 
Nelson GW, 
Sampath K et al. 
<span class="ref-title">APOL1 genetic variants in focal segmental glomerulosclerosis and HIV-associated nephropathy</span>. <span class="ref-journal">J Am Soc Nephrol</span> 2011; <span class="ref-vol">22</span>: 2129–2137 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3231787/">PMC free article</a>]</span> [<a href="/pubmed/21997394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=APOL1+genetic+variants+in+focal+segmental+glomerulosclerosis+and+HIV-associated+nephropathy&amp;author=JB+Kopp&amp;author=GW+Nelson&amp;author=K+Sampath&amp;volume=22&amp;publication_year=2011&amp;pages=2129-2137&amp;pmid=21997394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="gfw001C33">33. <span class="mixed-citation">
Coenen MJ, 
Hofstra JM, 
Debiec H et al. 
<span class="ref-title">Phospholipase A2 receptor (PLA2R1) sequence variants in idiopathic membranous nephropathy</span>. <span class="ref-journal">J Am Soc Nephrol</span> 2013; <span class="ref-vol">24</span>: 677–683 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3609136/">PMC free article</a>]</span> [<a href="/pubmed/23431073" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=Phospholipase+A2+receptor+(PLA2R1)+sequence+variants+in+idiopathic+membranous+nephropathy&amp;author=MJ+Coenen&amp;author=JM+Hofstra&amp;author=H+Debiec&amp;volume=24&amp;publication_year=2013&amp;pages=677-683&amp;pmid=23431073&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5837679&amp;issue-id=291686&amp;journal-id=767&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139841103667568title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">MATERIALS AND METHODS</a></li><li><a href="#s3title">RESULTS</a></li><li><a href="#s4title">DISCUSSION</a></li><li><a href="#s5title">SUPPLEMENTARY DATA</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm139841106683824title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm139841093880912title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Nephrology Dialysis Transplantation</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5837679/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5837679/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5837679/pdf/gfw001.pdf">PDF (873K)</a></li> | <li><a href="#" data-citationid="PMC5837679" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5837679%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5837679%2F&amp;text=Genetic%20risk%20variants%20for%20membranous%20nephropathy%3A%20extension%20of%20and%20association%20with%20other%20chronic%20kidney%20disease%20aetiologies" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5837679%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27333618" data-aiid="5837679" data-aid="5837679" data-iid="291686" data-domainid="767" data-domain="ndt" data-accid="PMC5837679" data-md5="dd8dd82af19463f57784f9a0704dab98"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:49:47-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89DA96DD593E310000000003540220&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5837679%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ndt&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5837679/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>